<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="description" content="Methods to extract metadata and study the structure of scientific protocols based on all available online data and knowledge.">

<title>alhazen - Analyses of CryoET Knowledge</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="../styles.css">
<meta property="og:title" content="alhazen - Analyses of CryoET Knowledge">
<meta property="og:description" content="Methods to extract metadata and study the structure of scientific protocols based on all available online data and knowledge.">
<meta property="og:site-name" content="alhazen">
<meta name="twitter:title" content="alhazen - Analyses of CryoET Knowledge">
<meta name="twitter:description" content="Methods to extract metadata and study the structure of scientific protocols based on all available online data and knowledge.">
<meta name="twitter:card" content="summary">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">alhazen</span>
    </a>
  </div>
        <div class="quarto-navbar-tools ms-auto">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item">Analyses of CryoET Knowledge</li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Home - Alhazen</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../docnb1_use_cases.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Use Cases</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../docnb2_architecture.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Platform Architecture</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">API Docs</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../ceifns_db.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Database for Scientific Knowledge</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../apps.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Apps</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../toolkit.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Alhazen Toolkit</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../basic_tools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Tools for Alhazen</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../metadata_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Methods Metadata Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../paperqa_emulation_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Paper QA Emulation Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../search_engine_eutils.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Search Engine Tools</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../jats_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">JATS Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../pdf_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PDF Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../web_robot.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Web Robots</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#what-is-cryoet-why-is-it-interesting-what-does-the-future-hold-for-it" id="toc-what-is-cryoet-why-is-it-interesting-what-does-the-future-hold-for-it" class="nav-link active" data-scroll-target="#what-is-cryoet-why-is-it-interesting-what-does-the-future-hold-for-it">What is CryoET? Why is it interesting? What does the future hold for it?</a></li>
  <li><a href="#basics" id="toc-basics" class="nav-link" data-scroll-target="#basics">Basics</a>
  <ul class="collapse">
  <li><a href="#python-imports" id="toc-python-imports" class="nav-link" data-scroll-target="#python-imports">Python Imports</a></li>
  <li><a href="#environment-variables" id="toc-environment-variables" class="nav-link" data-scroll-target="#environment-variables">Environment Variables</a></li>
  <li><a href="#setup-utils-agents-and-tools" id="toc-setup-utils-agents-and-tools" class="nav-link" data-scroll-target="#setup-utils-agents-and-tools">Setup utils, agents, and tools</a></li>
  <li><a href="#set-evaluation-dataset" id="toc-set-evaluation-dataset" class="nav-link" data-scroll-target="#set-evaluation-dataset">Set Evaluation Dataset</a></li>
  </ul></li>
  <li><a href="#building-the-database" id="toc-building-the-database" class="nav-link" data-scroll-target="#building-the-database">Building the database</a>
  <ul class="collapse">
  <li><a href="#scripts-to-build-delete-the-database" id="toc-scripts-to-build-delete-the-database" class="nav-link" data-scroll-target="#scripts-to-build-delete-the-database">Scripts to Build / Delete the database</a></li>
  <li><a href="#build-ceifns-database-from-queries" id="toc-build-ceifns-database-from-queries" class="nav-link" data-scroll-target="#build-ceifns-database-from-queries">Build CEIFNS database from queries</a></li>
  </ul></li>
  <li><a href="#analyze-collections" id="toc-analyze-collections" class="nav-link" data-scroll-target="#analyze-collections">Analyze Collections</a></li>
  <li><a href="#run-metadata-extraction-chain-over-listed-papers" id="toc-run-metadata-extraction-chain-over-listed-papers" class="nav-link" data-scroll-target="#run-metadata-extraction-chain-over-listed-papers">Run MetaData Extraction Chain over listed papers</a></li>
  <li><a href="#protocol-modeling-extraction" id="toc-protocol-modeling-extraction" class="nav-link" data-scroll-target="#protocol-modeling-extraction">Protocol Modeling + Extraction</a></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/chanzuckerberg/alhazen/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Analyses of CryoET Knowledge</h1>
</div>

<div>
  <div class="description">
    Methods to extract metadata and study the structure of scientific protocols based on all available online data and knowledge.
  </div>
</div>


<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
<section id="what-is-cryoet-why-is-it-interesting-what-does-the-future-hold-for-it" class="level2">
<h2 class="anchored" data-anchor-id="what-is-cryoet-why-is-it-interesting-what-does-the-future-hold-for-it">What is CryoET? Why is it interesting? What does the future hold for it?</h2>
<p>Cryo-electron microscopy (cryo-EM) is a cutting-edge imaging technique that revolutionizes our understanding of molecular structures. Specifically, cryo-EM, or CryoET in tomography applications, involves rapidly freezing biological samples in their natural state to preserve their three-dimensional structure without the need for staining or crystallization. This methodology allows researchers to visualize proteins and other biomolecules at near-atomic resolution, leading to significant advancements in structural biology, drug discovery, and biotechnology.</p>
<p>CryoEM’s ability to produce high-resolution images of complex molecular structures is particularly interesting as it has unlocked new insights into cellular processes and the mechanisms behind various diseases. Researchers can now study protein interactions and conformational changes, providing critical information for developing targeted therapies and understanding fundamental biological phenomena. Moreover, this technique enables structural analysis of challenging samples that were previously inaccessible through X-ray crystallography or nuclear magnetic resonance (NMR) spectroscopy.</p>
<p>The future of cryo-EM is exceptionally promising as ongoing technological advancements continually improve the method’s resolution and efficiency. Ongoing research focuses on enhancing electron microscope capabilities, developing automated data collection systems, refining image processing algorithms, and integrating machine learning techniques to expedite the analysis workflow. As these innovations progress, cryo-EM will undoubtedly become an even more indispensable tool in structural biology, further transforming our understanding of life at the molecular level and driving advancements across various disciplines.</p>
<p><em>[text generated by Smuag-72b Quantised Model using Ollama]</em></p>
</section>
<section id="basics" class="level2">
<h2 class="anchored" data-anchor-id="basics">Basics</h2>
<section id="python-imports" class="level3">
<h3 class="anchored" data-anchor-id="python-imports">Python Imports</h3>
<p>Setting python imports, environment variables, and other crucial set up parameters here.</p>
<div class="cell" data-execution_count="6">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.apps.chat <span class="im">import</span>  AlhazenAgentChatBot</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.core <span class="im">import</span> get_langchain_chatmodel, MODEL_TYPE</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.schema_sqla <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.basic <span class="im">import</span> AddCollectionFromEPMCTool, DeleteCollectionTool</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.paperqa_emulation_tool <span class="im">import</span> PaperQAEmulationTool</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.metadata_extraction_tool <span class="im">import</span> <span class="op">*</span> </span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.protocol_extraction_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.toolkit <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.ceifns_db <span class="im">import</span> Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, restore_ceifns_database</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.searchEngineUtils <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.callbacks.tracers <span class="im">import</span> ConsoleCallbackHandler</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.docstore.document <span class="im">import</span> Document</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.text_splitter <span class="im">import</span> CharacterTextSplitter</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.vectorstores.pgvector <span class="im">import</span> PGVector</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_community.chat_models.ollama <span class="im">import</span> ChatOllama</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_google_vertexai <span class="im">import</span> ChatVertexAI</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_openai <span class="im">import</span> ChatOpenAI</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> bs4 <span class="im">import</span> BeautifulSoup,Tag,Comment,NavigableString</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> databricks <span class="im">import</span> sql</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> datetime <span class="im">import</span> datetime</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> importlib_resources <span class="im">import</span> files</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> re</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> requests</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy <span class="im">import</span> create_engine, exists, func, or_, and_, not_, desc, asc</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy.orm <span class="im">import</span> sessionmaker, aliased</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> time <span class="im">import</span> time,sleep</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> tqdm <span class="im">import</span> tqdm</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.request <span class="im">import</span> urlopen</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.parse <span class="im">import</span> quote_plus, quote, unquote</span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.error <span class="im">import</span> URLError, HTTPError</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> yaml</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Using Aliases like this massively simplifies the use of SQLAlchemy</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>IR <span class="op">=</span> aliased(InformationResource)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>SKC <span class="op">=</span> aliased(ScientificKnowledgeCollection)</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>SKC_HM <span class="op">=</span> aliased(ScientificKnowledgeCollectionHasMembers)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>SKE <span class="op">=</span> aliased(ScientificKnowledgeExpression)</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>SKE_XREF <span class="op">=</span> aliased(ScientificKnowledgeExpressionXref)</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>SKE_IRI <span class="op">=</span> aliased(ScientificKnowledgeExpressionIri)</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>SKE_HR <span class="op">=</span> aliased(ScientificKnowledgeExpressionHasRepresentation)</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>SKE_MO <span class="op">=</span> aliased(ScientificKnowledgeExpressionMemberOf)</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>SKI <span class="op">=</span> aliased(ScientificKnowledgeItem)</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>SKI_HP <span class="op">=</span> aliased(ScientificKnowledgeItemHasPart)</span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a>SKF <span class="op">=</span> aliased(ScientificKnowledgeFragment)</span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a>N <span class="op">=</span> aliased(Note)</span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a>NIA <span class="op">=</span> aliased(NoteIsAbout)</span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>SKC_HN <span class="op">=</span> aliased(ScientificKnowledgeCollectionHasNotes)</span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a>SKE_HN <span class="op">=</span> aliased(ScientificKnowledgeExpressionHasNotes)</span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a>SKI_HN <span class="op">=</span> aliased(ScientificKnowledgeItemHasNotes)</span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a>SKF_HN <span class="op">=</span> aliased(ScientificKnowledgeFragmentHasNotes)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="environment-variables" class="level3">
<h3 class="anchored" data-anchor-id="environment-variables">Environment Variables</h3>
<p>Remember to set environmental variables for this code:</p>
<ul>
<li><code>ALHAZEN_DB_NAME</code> - the name of the PostGresQL database you are storing information into</li>
<li><code>LOCAL_FILE_PATH</code> - the location on disk where you save temporary files, downloaded models or other data.</li>
</ul>
<div class="cell" data-execution_count="8">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>] <span class="op">=</span> <span class="st">'em_tech'</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>os.environ[<span class="st">'LOCAL_FILE_PATH'</span>] <span class="op">=</span> <span class="st">'/users/gully.burns/alhazen/'</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="9">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.path.exists(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]) <span class="kw">is</span> <span class="va">False</span>:</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>    os.makedirs(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>])</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.environ.get(<span class="st">'ALHAZEN_DB_NAME'</span>) <span class="kw">is</span> <span class="va">None</span>: </span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>    <span class="cf">raise</span> <span class="pp">Exception</span>(<span class="st">'Which database do you want to use for this application?'</span>)</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>db_name <span class="op">=</span> os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>]</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.environ.get(<span class="st">'LOCAL_FILE_PATH'</span>) <span class="kw">is</span> <span class="va">None</span>: </span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>    <span class="cf">raise</span> <span class="pp">Exception</span>(<span class="st">'Where are you storing your local literature database?'</span>)</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>loc <span class="op">=</span> os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="setup-utils-agents-and-tools" class="level3">
<h3 class="anchored" data-anchor-id="setup-utils-agents-and-tools">Setup utils, agents, and tools</h3>
<div class="cell" data-execution_count="5">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>ldb <span class="op">=</span> Ceifns_LiteratureDb(loc<span class="op">=</span>loc, name<span class="op">=</span>db_name)</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>llm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'mixtral:instruct'</span>) </span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>llm2 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-4-1106-preview'</span>) </span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>llm3 <span class="op">=</span> ChatVertexAI(model_name<span class="op">=</span><span class="st">"gemini-pro"</span>, convert_system_message_to_human<span class="op">=</span><span class="va">True</span>)</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a><span class="co">#cb = AlhazenAgentChatBot()</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a><span class="co">#print('AGENT TOOLS')</span></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a><span class="co">#for t in cb.tk.get_tools():</span></span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a><span class="co">#    print('\t'+type(t).__name__)</span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>test_tk <span class="op">=</span> MetadataExtractionToolkit(db<span class="op">=</span>ldb, llm<span class="op">=</span>llm2)</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'</span><span class="ch">\n</span><span class="st">TESTING TOOLS'</span>)</span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> t <span class="kw">in</span> test_tk.get_tools():</span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="st">'</span><span class="ch">\t</span><span class="st">'</span><span class="op">+</span><span class="bu">type</span>(t).<span class="va">__name__</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
TESTING TOOLS
    MetadataExtraction_EverythingEverywhere_Tool
    MetadataExtraction_RAGOnSections_Tool
    SimpleExtractionWithRAGTool</code></pre>
</div>
</div>
</section>
<section id="set-evaluation-dataset" class="level3">
<h3 class="anchored" data-anchor-id="set-evaluation-dataset">Set Evaluation Dataset</h3>
<p>These are cases directly taken from <code>*.yaml</code> files that</p>
<div class="cell" data-execution_count="6">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>dois <span class="op">=</span> {<span class="dv">10000</span>: [<span class="st">'10.1101/2022.04.12.488077'</span>], </span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10001</span>: [<span class="st">'10.1101/2022.04.12.488077'</span>], </span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10003</span>: [<span class="st">'10.1038/s41586-022-05255-2'</span>, <span class="st">'10.1038/s41592-020-01054-7'</span>], </span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10004</span>: [<span class="st">'10.1101/2023.04.28.538734'</span>], </span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10005</span>: [<span class="st">'10.1038/s41594-022-00861-0'</span>], </span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10006</span>: [<span class="st">'10.1038/s41586-020-2665-2'</span>], </span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10007</span>: [], </span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10008</span>: [<span class="st">'10.1038/s41586-022-04971-z'</span>], </span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10009</span>: [<span class="st">'10.1126/science.abm6704'</span>], </span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>        <span class="dv">10010</span>: [<span class="st">'10.1083/jcb.202204093'</span>, <span class="st">'10.1101/2022.01.23.477440'</span>]}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="building-the-database" class="level2">
<h2 class="anchored" data-anchor-id="building-the-database">Building the database</h2>
<section id="scripts-to-build-delete-the-database" class="level3">
<h3 class="anchored" data-anchor-id="scripts-to-build-delete-the-database">Scripts to Build / Delete the database</h3>
<p>If you need to restore a deleted database from backup, use the following shell commands:</p>
<pre><code>$ createdb em_tech
$ psql -d em_tech -f /local/file/path/em_tech/backup&lt;date_time&gt;.sql</code></pre>
<div class="cell" data-execution_count="6">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>drop_ceifns_database(os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Database has been backed up to /users/gully.burns/alhazen/em_tech/backup2024-02-23-22-23-08.sql
Database has been dropped successfully !!</code></pre>
</div>
</div>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>create_ceifns_database(os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 311/311 [00:00&lt;00:00, 3419.34it/s]</code></pre>
</div>
</div>
</section>
<section id="build-ceifns-database-from-queries" class="level3">
<h3 class="anchored" data-anchor-id="build-ceifns-database-from-queries">Build CEIFNS database from queries</h3>
<section id="import-papers-from-dois-pertaining-to-cryoet-portal-records-10000-10010" class="level4">
<h4 class="anchored" data-anchor-id="import-papers-from-dois-pertaining-to-cryoet-portal-records-10000-10010">Import papers from DOIs pertaining to CryoET-Portal records <code>10000-10010</code></h4>
<p>The <a href="https://chanzuckerberg.github.io/cryoet-data-portal/python-api.html">CryoET Data portal</a> system is based on submitted data to our curation team, accompanied by papers referenced by DOIs. Each dataset is assigned an ID value associated with DOIs.</p>
<div class="cell" data-execution_count="10">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># use the EMPCSearchTool to run a query for the dois mentioned</span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>query <span class="op">=</span> <span class="st">' OR '</span>.join([<span class="st">'doi:"'</span><span class="op">+</span>d<span class="op">+</span><span class="st">'"'</span> <span class="cf">for</span> d_id <span class="kw">in</span> dois <span class="cf">for</span> d <span class="kw">in</span> dois[d_id] ])</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool.run(tool_input<span class="op">=</span>{<span class="st">'id'</span>: <span class="st">'0'</span>, <span class="st">'name'</span>:<span class="st">'CryoET Portal (10000-10010)'</span>, <span class="st">'query'</span>:query, <span class="st">'full_text'</span>:<span class="va">True</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=doi:"10.1101/2022.04.12.488077" OR doi:"10.1101/2022.04.12.488077" OR doi:"10.1038/s41586-022-05255-2" OR doi:"10.1038/s41592-020-01054-7" OR doi:"10.1101/2023.04.28.538734" OR doi:"10.1038/s41594-022-00861-0" OR doi:"10.1038/s41586-020-2665-2" OR doi:"10.1038/s41586-022-04971-z" OR doi:"10.1126/science.abm6704" OR doi:"10.1083/jcb.202204093" OR doi:"10.1101/2022.01.23.477440", 10 European PMC PAPERS FOUND
 Returning 10</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 1/1 [00:01&lt;00:00,  1.34s/it]
100%|██████████| 10/10 [00:00&lt;00:00, 256.91it/s]</code></pre>
</div>
<div class="cell-output cell-output-display" data-execution_count="10">
<pre><code>{'report': 'We added a collection to the database called `CryoET Portal (10000-10010)` containing 0 papers from this query: `doi:"10.1101/2022.04.12.488077" OR doi:"10.1101/2022.04.12.488077" OR doi:"10.1038/s41586-022-05255-2" OR doi:"10.1038/s41592-020-01054-7" OR doi:"10.1101/2023.04.28.538734" OR doi:"10.1038/s41594-022-00861-0" OR doi:"10.1038/s41586-020-2665-2" OR doi:"10.1038/s41586-022-04971-z" OR doi:"10.1126/science.abm6704" OR doi:"10.1083/jcb.202204093" OR doi:"10.1101/2022.01.23.477440"`.'}</code></pre>
</div>
</div>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>cb.agent_executor.invoke({<span class="st">'input'</span>:<span class="st">'Describe the contents of the collection with id="base_cryoet_data".'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>

&gt; Entering new AgentExecutor chain...
{
    "action": "describe_collection_in_local_database",
    "action_input": {
        "id": "base_cryoet_data"
    }
}{'report': 'The collection with the id `base_cryoet_data` does not exist.'}


&gt; Finished chain.</code></pre>
</div>
<div class="cell-output cell-output-display" data-execution_count="7">
<pre><code>{'input': 'Describe the contents of the collection with id="base_cryoet_data".',
 'output': {'report': 'The collection with the id `base_cryoet_data` does not exist.'},
 'intermediate_steps': [(AgentAction(tool='describe_collection_in_local_database', tool_input={'id': 'base_cryoet_data'}, log='{\n    "action": "describe_collection_in_local_database",\n    "action_input": {\n        "id": "base_cryoet_data"\n    }\n}'),
   {'report': 'The collection with the id `base_cryoet_data` does not exist.'})]}</code></pre>
</div>
</div>
</section>
<section id="extend-database-to-include-all-cryoet-papers" class="level4">
<h4 class="anchored" data-anchor-id="extend-database-to-include-all-cryoet-papers">Extend Database to include all CryoET papers</h4>
<div class="cell" data-execution_count="29">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> local_resources.queries.em_tech <span class="im">as</span> em_tech_queries</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.queryTranslator <span class="im">import</span> QueryTranslator, QueryType</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>cols_to_include <span class="op">=</span> [<span class="st">'ID'</span>, <span class="st">'CORPUS_NAME'</span>, <span class="st">'QUERY'</span>]</span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.read_csv(files(em_tech_queries).joinpath(<span class="st">'EM_Methods.tsv'</span>), sep<span class="op">=</span><span class="st">'</span><span class="ch">\t</span><span class="st">'</span>)</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> df.drop(columns<span class="op">=</span>[c <span class="cf">for</span> c <span class="kw">in</span> df.columns <span class="cf">if</span> c <span class="kw">not</span> <span class="kw">in</span> cols_to_include])</span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="29">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">ID</th>
<th data-quarto-table-cell-role="th">CORPUS_NAME</th>
<th data-quarto-table-cell-role="th">QUERY</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>1</td>
<td>Hierarchical phase-contrast tomography</td>
<td>Hierarchical phase-contrast tomography | HIP-C...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td>2</td>
<td>Cryo-Electron Tomography</td>
<td>Cryoelectron Tomography | Cryo Electron Tomogr...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td>3</td>
<td>Volume Electron Microscopy</td>
<td>Volume Electron Microscopy | Volume EM | (seri...</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="cell" data-execution_count="9">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>qt <span class="op">=</span> QueryTranslator(df.sort_values(<span class="st">'ID'</span>), <span class="st">'ID'</span>, <span class="st">'QUERY'</span>, <span class="st">'CORPUS_NAME'</span>)</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>(corpus_ids, epmc_queries) <span class="op">=</span> qt.generate_queries(QueryType.epmc, sections<span class="op">=</span>[<span class="st">'TITLE_ABS'</span>, <span class="st">'METHODS'</span>])</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>corpus_names <span class="op">=</span> df[<span class="st">'CORPUS_NAME'</span>]</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (<span class="bu">id</span>, name, query) <span class="kw">in</span> <span class="bu">zip</span>(corpus_ids, corpus_names, epmc_queries):</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> <span class="bu">id</span> <span class="op">==</span> <span class="dv">3</span>:</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>        <span class="cf">continue</span></span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>    addEMPCCollection_tool.run(tool_input<span class="op">=</span>{<span class="st">'id'</span>: <span class="bu">id</span>, <span class="st">'name'</span>:name, <span class="st">'query'</span>:query, <span class="st">'full_text'</span>:<span class="va">False</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-error">
<pre><code>NameError: name 'QueryTranslator' is not defined</code></pre>
</div>
</div>
</section>
<section id="get-full-text-copies-of-all-the-papers-about-cryoet" class="level4">
<h4 class="anchored" data-anchor-id="get-full-text-copies-of-all-the-papers-about-cryoet">Get full text copies of all the papers about CryoET</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>cb.agent_executor.invoke({<span class="st">'input'</span>:<span class="st">'Get full text copies of all papers in the collection with id="1".'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
</section>
<section id="analyze-collections" class="level2">
<h2 class="anchored" data-anchor-id="analyze-collections">Analyze Collections</h2>
<div class="cell" data-execution_count="9">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKC.name, SKE.<span class="bu">id</span>, SKI.<span class="bu">type</span>) <span class="op">\</span></span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>SKE_HR.ScientificKnowledgeExpression_id) <span class="op">\</span></span>
<span id="cb24-5"><a href="#cb24-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) </span>
<span id="cb24-6"><a href="#cb24-6" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(q.<span class="bu">all</span>(), columns<span class="op">=</span>[<span class="st">'collection'</span>, <span class="st">'doi'</span>, <span class="st">'type'</span>])    </span>
<span id="cb24-7"><a href="#cb24-7" aria-hidden="true" tabindex="-1"></a>df.pivot_table(index<span class="op">=</span><span class="st">'collection'</span>, columns<span class="op">=</span><span class="st">'type'</span>, values<span class="op">=</span><span class="st">'doi'</span>, aggfunc<span class="op">=</span><span class="kw">lambda</span> x: <span class="bu">len</span>(x.unique()))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="9">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th">type</th>
<th data-quarto-table-cell-role="th">CitationRecord</th>
<th data-quarto-table-cell-role="th">JATSFullText</th>
<th data-quarto-table-cell-role="th">PDFFullText</th>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">collection</th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">Cryo-Electron Tomography</td>
<td>2545</td>
<td>1483</td>
<td>273</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">CryoET Portal (10000-10010)</td>
<td>10</td>
<td>6</td>
<td>3</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">Hierarchical phase-contrast tomography</td>
<td>134</td>
<td>47</td>
<td>1</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
</section>
<section id="run-metadata-extraction-chain-over-listed-papers" class="level2">
<h2 class="anchored" data-anchor-id="run-metadata-extraction-chain-over-listed-papers">Run MetaData Extraction Chain over listed papers</h2>
<p>Here, we run various versions of the metadata extraction tool to examine performance over the cryoet dataset.</p>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>t2 <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> test_tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, MetadataExtraction_EverythingEverywhere_Tool)][<span class="dv">0</span>]</span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>t2.compile_answers(<span class="st">'cryoet'</span>, <span class="st">'/Users/gully.burns/Documents/Coding/ChatGPT_etc/alzhazen/local_resources/data_files/cryoet_portal_metadata'</span>)</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame()</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d <span class="kw">in</span> [d <span class="cf">for</span> d_id <span class="kw">in</span> dois <span class="cf">for</span> d <span class="kw">in</span> dois[d_id]]:</span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>    item_types <span class="op">=</span> <span class="bu">set</span>()</span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>    d_id <span class="op">=</span> <span class="st">'doi:'</span><span class="op">+</span>d</span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, t2.build_report(d_id, <span class="st">'cryoet'</span>)]) </span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a>     </span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d <span class="kw">in</span> [d <span class="cf">for</span> d_id <span class="kw">in</span> dois <span class="cf">for</span> d <span class="kw">in</span> dois[d_id]]:</span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a>    item_types <span class="op">=</span> <span class="bu">set</span>()</span>
<span id="cb25-12"><a href="#cb25-12" aria-hidden="true" tabindex="-1"></a>    d_id <span class="op">=</span> <span class="st">'doi:'</span><span class="op">+</span>d</span>
<span id="cb25-13"><a href="#cb25-13" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> d_id <span class="kw">in</span> df.doi.unique():</span>
<span id="cb25-14"><a href="#cb25-14" aria-hidden="true" tabindex="-1"></a>        <span class="cf">continue</span></span>
<span id="cb25-15"><a href="#cb25-15" aria-hidden="true" tabindex="-1"></a>    t2.run(tool_input<span class="op">=</span>{<span class="st">'paper_id'</span>: d_id, <span class="st">'extraction_type'</span>: <span class="st">'cryoet'</span>})</span>
<span id="cb25-16"><a href="#cb25-16" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, t2.build_report(d_id, <span class="st">'cryoet'</span>)])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nSingle particle analysis (SPA) by cryo-transmission electron microscopy (cryo-TEM) has become a key technique to study the structure of isolated biological macromolecules at high-resolution (&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;). While SPA routinely reaches resolutions where protein side-chains can be fitted unambiguously, the reductionist approach of studying protein complexes in vitro loses all information concerning their molecular sociology: the interaction of the molecular complexes in their natural environment (&lt;&lt;REF:Beck-2016-26-825&gt;&gt;). Conversely, in situ cryo-electron tomography (cryo-ET) allows for the reconstruction of pleomorphic structures such as the crowded environment of the cell at molecular resolution, maintaining the interaction and localization of protein complexes within the biological system (&lt;&lt;REF:Dietrich-2022-607-823&gt;&gt;, &lt;&lt;REF:Gupta-2021-184-3643&gt;&gt;, &lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;, &lt;&lt;REF:Plitzko-2017-46-110&gt;&gt;, &lt;&lt;REF:Watanabe-2020-182-1508&gt;&gt;).\nOne of the primary factors limiting the resolution of cryo-TEM is inelastic scattering. As the mean free path of an electron in vitrified biological samples is about 300-400 nm, samples beyond the size of viruses and small prokaryotic cells are generally too thick for cryo-ET (&lt;&lt;REF:Liedtke-2022-6-100065&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;). In consequence, two main sample thinning methods to obtain electron-transparent specimens for cryo-ET have been developed: cryo-ultramicrotomy and cryo-FIB milling. Cryo-ultramicrotomy encompasses the thin-sectioning of vitreous cells and tissues with a diamond knife (&lt;&lt;REF:Al-Amoudi-2004-148-131&gt;&gt;). The shearing forces at the knife's edge, however, cause mechanical artifacts such as crevices and compression in the resulting sections (&lt;&lt;REF:Al-Amoudi-2003-9-372&gt;&gt;). More recently, the focused ion beam (FIB) instrument has been widely adopted for sample thinning at cryogenic temperatures. While not completely damage-free (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Lucas-2023-?-?&gt;&gt;), the technique bypasses the mechanical artifacts of cryo-ultramicrotomy (&lt;&lt;REF:Marko-2007-4-215&gt;&gt;, &lt;&lt;REF:Rigort-2012-109-4449&gt;&gt;) and has been shown to yield data that can allow for the elucidation of ribosomes to side-chain resolution (&lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;). The automation of lamella preparation by cryo-FIB milling has also reduced the need for user expertise and manual intervention (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Zachs-2020-9-?&gt;&gt;).\nPrior to thinning, the sample must be cryogenically fixed by cooling at a sufficiently high rate to prevent ice crystal formation, resulting in a vitrified sample. There are two main methods available for vitrification: plunge freezing, in which the sample is immersed at ambient pressure into liquid ethane or ethane-propane mixture (&lt;&lt;REF:Tivol-2008-14-375&gt;&gt;), and high pressure freezing (HPF), in which the sample is cooled with a jet of liquid nitrogen at a pressure of ~2000 bar. While the former yields samples that are easily FIB-milled, the sample thickness that can reliably be vitrified is limited to roughly 10 microm. HPF, on the other hand, allows for the vitrification of samples up to a thickness of roughly 200 microm (&lt;&lt;REF:Dubochet-1995-5-366&gt;&gt;).\nConsequently, HPF greatly expands the size range of biological samples that can be vitrified but comes at the cost of embedding the specimen in a thick layer of ice defined by the depth of the freezing receptacle. This increased sample thickness leads to longer milling times. While samples up to a thickness of about 50 microm can be prepared by milling lamellae directly on the grid following the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;), lamellae from thicker samples have, to date, only been prepared by cryo-lift-out (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Kuba-2021-281-112&gt;&gt;, &lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nCryo-lift-out refers to the extraction of the material for lamella preparation from bulk HPF sample and subsequent transfer and attachment to a lift-out receiver grid, conventionally a half-moon shaped grid (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;). Two main types of micromanipulator devices are currently available for cryo-lift-out: sharp needles and a cryo-gripper (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;). Initially, trenches are milled around the area of interest leaving it connected to the bulk material on a single side. For specimens in HPF sample carriers, the material must additionally be cleared from below. After these preparatory steps, the lift-out device is brought in contact with the volume to be extracted. The remaining connection to the bulk material is removed, the micromanipulator is used to transfer the extracted volume and redeposition milling (&lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) is used to attach the volume to the receiver grid. Finally, an electron transparent lamella is prepared (&lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;).\nWhile widely used in materials science at room temperature (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;), cryo-lift-out of biological samples has remained primarily proof-of-concept (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;). This is due to a number of factors, e.g. the need for cooled micromanipulator devices and accompanying workflow adaptations required resulting in limited throughput and problems with lamella loss during transfer to the TEM (&lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nAnother limitation to cryo-lift-out, as well as on-grid lamella preparation is the loss of contextual information. Only a fraction of the sample volume (~1% for larger eukaryotic cells, &lt; 1% for multicellular specimens) ends up inside the final lamella for cryo-ET data acquisition. Techniques exist that are capable of capturing larger volumetric data and tracking morphology at comparatively large volume scales. Examples of such techniques are X-ray tomography and various volume electron microscopy (vEM) techniques: serial FIB milling and scanning electron microscopy, and serial sectioning of plastic embedded samples imaged by TEM or STEM. These techniques, however, currently cannot achieve the resolution attainable by cryo-ET at an equivalent sample preservation state due to physical limitations in imaging or the necessity of fixation and contrasting (&lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;, &lt;&lt;REF:Spehner-2020-211-107528&gt;&gt;).\nIn this work, we describe a novel cryo-lift-out approach that creates a series of lamellae from one lift-out volume that we term Serial Lift-Out. Inspired by diamond knife serial sectioning, Serial Lift-Out retains more contextual information than previous procedures and increases the throughput of cryo-lift-out by an order of magnitude. It sets the stage for the study of multicellular organisms and tissues by cryo-ET - applications that previously seemed practically impossible (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). We demonstrate Serial Lift-Out on high-pressure frozen Caenorhabditis elegans L1 stage larvae, sampling their ultrastructure along the anterior-posterior axis by cryo-electron tomography. From the resulting dataset, we reconstruct the nematode's ribosome to 7 A resolution by subtomogram averaging, exemplifying the enormous potential of Serial Lift-Out for the study of the molecular anatomy of multicellular systems.\n\nResults\n\nThe concept behind Serial Lift-Out\nThe most time-consuming steps in cryo-lift-out are the preparation of the extraction sites and the transfer of the extracted volume to the receiving grid. Each repetition of the lift-out cycle therefore adds significant time investment with low yield. Scaling up the extracted volume and producing multiple lamellae from a single lift-out bypasses this repetitive, time-consuming trench milling and transfer, increases throughput and, foremost, provides more contextual information about the targeted material.\nA specific implementation of this concept is the extraction of an entire L1 larva of C. elegans, followed by repeated steps of attachment to the receiving grid, sectioning and transfer of the remaining volume to the next attachment site (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;). While material is still lost during sectioning and thinning, a large fraction of the worm is made readily accessible to cryo-TEM data acquisition.\nFigure 1:\nA schematic of Serial Lift-Out.\nAn illustration of the Serial Lift-Out method, exemplified by the process being performed on a C. elegans L1 larva embedded in an extracted volume of vitrified ice (blue). a, The extracted volume containing the larva. b, c, Schematic representation of the resulting Serial Lift-Out sections and lamellae, respectively. d, Illustration of the double-sided attachment Serial Lift-Out procedure. The extracted volume shown in panel a is attached to the lift-out needle via a copper block adaptor and transferred to a rectangular mesh receiver grid. Several sections are shown, obtained by the repetition of the attachment of the bottom part of the volume to the grid bars and subsequent sectioning.\nPrevious lift-out approaches extracted volumes approximately the size of the final lamella (Figure 1 - Figure Supplement 1a), with some excess material for stability during transfer and attachment. In order to section multiple lamellae from a single cryo-lift-out transfer, the volume extracted by cryo-lift-out needed to be increased. To ease the manipulation of such large volumes, we introduced a copper block that acts as an adapter between the needle and the extracted volume. This copper block is created from the receiver grid and attached to the needle before lift-out (Figure 1 - Figure Supplement 2, Figure 1 - Figure Supplement 3). As copper has higher redeposition rates than the tungsten needle, the copper block adapter results in significantly more resilient \"welding\" of the specimen to the micromanipulator.\nAnother volume limiting factor is the ablation rate of the ion beam. Most cryo-FIB machines are equipped with a gallium ion source rendering it impractical to mill beyond 50 microm in depth. In addition, for samples frozen in HPF carriers, preparation requires an undercut, the removal of material below the extracted volume, to detach the extraction volume from the bulk. In combination, these factors generally result in maximum extraction volumes of approximately 20 microm x 20 microm x 10-30 microm (length x width x height, Figure 1 - Figure Supplement 1b).\nIn order to extract larger volumes, we performed lift-out on an HPF 'waffle'-type sample. HPF 'waffle' samples are prepared by freezing sample on a grid that is sandwiched between HPF carriers (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The final thickness is therefore defined by the type of grid and spacer being used during freezing. For a 25 microm thick specimen, extraction volumes of up to 200 microm x 40 microm x 25 microm can easily be obtained by performing lift-out with the sample surface oriented perpendicular to the ion beam (Figure 1 - Figure Supplement 1c). The same orientation is used during trench milling and will be referred to as 'trench milling orientation' (Figure 1 - Figure Supplement 3c,f). Such large volumes extracted at the trench milling orientation can yield many sections from a single lift-out which can be subsequently thinned to lamellae for cryo-ET data acquisition (Figure 1b,c &lt;&lt;FIG:fig1&gt;&gt;).\n\nApplication of Serial Lift-Out to C. elegans L1 larvae\nTo assess the feasibility of obtaining serial lamellae from a single lift-out transfer, we performed the procedure on C. elegans L1 larvae that had been vitrified using a modified protocol of the 'waffle' method (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The sample contained many L1 larvae embedded in an approximately 25 microm thick layer of ice (Figure 2 - Figure Supplement 1). Two sites from two different grids were selected for preparation by correlating cryo-fluorescence light microscopy with SEM and FIB view images. Following the geometry described in Figure 1 &lt;&lt;FIG:fig1&gt;&gt; - Supplement 1c, the sites were prepared by milling trenches around the larva, yielding a 30 microm x 110 microm x 25 microm and a 40 microm x 180 microm x 25 microm extraction volume (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3a). Two different attachment strategies were explored, with resulting lamellae either being attached on one side (single-sided attachment) or two sides (double-sided attachment). To extract the volumes, the needle with the copper adapter (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3b) was attached to the sample using redeposition: copper material redepositing onto the surface of the extraction volume during milling. The extraction volume was subsequently milled free from the bulk material and lifted out.\nFigure 2:\nA workflow for double-sided attachment Serial Lift-Out.\na, FIB image of the prepared extraction site with overlaid correlated fluorescence data (green) indicating the larva being targeted. The micromanipulator is attached to the extraction volume by redeposition from the copper adapter (trench milling orientation). b, The extracted volume (orange dashed line) is lowered into position between two grid bars in lamella milling orientation. The lower front edge of the volume (yellow arrows) is aligned to the pre-milled line mark (black arrows). c, Double-sided attachment by redeposition from the grid bars (yellow arrows indicate direction of milling), followed by line pattern milling releasing the section of a desired thickness (dashed yellow line). Orange dashed line indicates the outline of the extracted volume. d, SEM image of a typical section after being released from the extracted volume. Black dashed line indicates the outline of the worm cross-section. e,f, SEM (e) and TEM (f) overview images of the 40 double-sided attached sections obtained. Figure 2 - Movie Supplement 1 summarizes the process.\nThe volume was then transferred to the receiver grid. For single-sided attachment, a modified grid based on customizing a standard 100 mesh copper grid (Figure 2 - Figure Supplement 2) was used. Removing every second row of grid bars yielded an array of trimmed grid bars that resemble the pins of a standard half-moon shaped lift-out grid (Figure 2 - Figure Supplement 2e). Alternatively, a copper grid with rectangular meshes can be used for double-sided attachment of volumes roughly as wide as the mesh (~40 microm for the 400/100 rectangular mesh grids used here, Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Double- sided attachment has the added benefit of increased section stability by eliminating the free-standing side (Figure 2 - Figure Supplement 3f).\nThe volume's lower front edge was precisely aligned to the front edge of the attachment pin (Figure 2 - Figure Supplement 3c) or a previously prepared alignment line pattern milled onto the receiver grid (Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Then, redeposition milling was used to attach the lower part of the volume to the grid bar(s) (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow arrows). After attachment, the lower part of the target material was separated from the extracted volume using a line pattern (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow dashed line), leaving a ~4 microm thick section (Figure 2d &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3e,f). Once sectioned, the remaining volume attached to the lift-out needle was transferred to the next attachment site. This procedure was iterated until no material remained. As a result, many sections were produced from a single cryo-lift-out transfer (Figure 2e-f &lt;&lt;FIG:fig2&gt;&gt;). To assess the general applicability of the approach to non-'waffle' type samples, an additional experiment was performed on D. melanogaster egg chambers, high-pressure frozen in standard sample carriers. From a ~20 microm long and ~15 microm deep extracted volume we produced sections of ~1-2 microm thickness (Figure 2 - Figure Supplement 4).\n\nSampling organismal cellular anatomy along the C. elegans L1 larva at molecular resolution\nWe prepared dozens of sections along the anterior-posterior axis of C. elegans L1 larvae: 12 single-sided attached and 40 double-sided attached lamellae (Figure 2 - Figure Supplement 3g, Figure 2e &lt;&lt;FIG:fig2&gt;&gt;). After transfer to the TEM, 8 out of 12 and 32 out of 40 lamellae were recovered, respectively (Figure 3 - Figure Supplement 1a, Figure 3 - Figure Supplement 2a). Lamella loss during transfer is common due to manual grid handling steps. The increased rate of successfully transferred lamellae from 66% for single-sided to 80% for double-sided attachment is indicative of the increased lamella stability of double-sided attachment.\nFigure 3:\nLamella TEM overviews sample the anatomy of a C. elegans L1 larva along the anterior-posterior body axis.\nNative tissue scattering contrast is sufficient to extract a considerable amount of anatomical information from low magnification TEM overviews of the lamellae generated in the double-sided attachment Serial Lift-Out experiment. a, Schematic representation of 29 body transverse section overviews obtained from the final lamellae along the anterior-posterior body axis of an L1 larva. Anterior is located to the left, posterior to the right. The OpenWorm project model of the adult C. elegans worm was used for illustrating the possibility of back mapping since a cellular model of the L1 larva only exists for its head and not the entire body (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;). The cross-sections were cropped from lamella overview images and mapped back to a position derived from the known sectioning distance and anatomical features discernible in the corresponding cross-section. The dashed white frames indicate overviews with corresponding magnified representations in panels b-g. b, This lamella originates from ~15 microm along the anterior-posterior axis. Clearly visible are the three lobes of the anterior pharyngeal lumen in the center of the worm cross-section (asterisk) and the relatively electron dense pharyngeal lining. c, Overview from the anterior part of the pharyngeal isthmus ~42 microm along the anterior-posterior axis. Note the nerve ring (dashed line) surrounding the central pharynx. Additionally, the alae (asterisks) running along the left and right lateral side of the worm become obvious. d, Overview of a lamella of roughly the center of the posterior pharyngeal bulb region. The central grinder organ is clearly discernible (asterisk). This section is positioned ~65 microm along the anterior-posterior axis. e, A section roughly mid-body. The intestinal lumen (dashed line) and intestinal cells (dotted line) are obvious. The darker cell slightly left of the body center is likely one of the gonadal primordial cells (asterisk). The section is from ~115 microm along the anterior-posterior axis. f, In this mid-body section, the intestinal lumen (dashed line) can again be clearly discerned. The section can be mapped to ~132 microm along the anterior-posterior axis. g, Section showing the intestinal lumen at ~155 microm along the anterior-posterior axis.\nFrom these successfully transferred lamellae, overview maps were recorded (approximately 20 microm x 25 microm in size, Figure 3 - Figure Supplement 1, Figure 3 - Figure Supplement 2) depicting clear larval cross-sections. These overviews allowed the identification and assignment of anatomical structures and tissues such as the pharynx, the body-wall muscle cells, neurons, the hypodermis and seam cells. Larger organelles such as nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, bundles of actin filaments and microtubules were clearly discernible (Figure 3b-g &lt;&lt;FIG:fig3&gt;&gt;, Figure3 - Movie Supplement 1). Given the stereotypic body plan of C. elegans, the information obtained from the overviews together with the sectioning thickness was used to determine the approximate location of the sections in the worm, as schematically shown in Figure 3a &lt;&lt;FIG:fig3&gt;&gt;.\nThe biological area within each lamella can, due to the circular nature of cross sections, be estimated by ?r2, where the average radius is around 5 microm, resulting in a mean of 78.5 microm2 imageable area per lamella. For the single-sided attachment experiment, the total imageable area was ~ 630 microm2 and allowed for the collection of 57 tilt series, each with a field of view of ~1.2 microm in size. The double-sided attachment experiment yielded ~2500 microm2 imageable area from which a total of 1012 tilt series with a field of view of ~750 nm in size were collected. The double-sided attachment experiment allowed us to sample a large fraction of all tissues and cell types along the anterior-posterior body axis of the C. elegans L1 larva.\nTo illustrate the information level already present at intermediate magnification (11,500x), we manually segmented a representative cross-section (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). The segmentation illustrates that the cuticle clearly delineates the body cross-section. Body-wall muscle cells are obvious due to their pronounced distal actomyosin pattern. The pharyngeal lumen's trilobal structure demarcates the body center and is surrounded by the three alternating pharyngeal muscle and marginal cells. Pharyngeal neurons and gland cell process cross-sections are embedded in the pharyngeal muscle cells. Nuclei are easy to discern due to their ribosome-decorated double-membrane and their denser, less granular interior. Neurons in general can be discerned by their appearance as round or tubular cells, often grouped in bundles. These clearly interpretable overviews allow targeting of tissue-specific cellular structures or cellular protein complexes with known location (e.g. sarcomeric proteins).\nFigure 4:\nRepresentative overview and tomograms of the L1 larval pharyngeal isthmus region.\na, Overview montage (magnification 11,500x) of the head region lamella. This section is located within the pharyngeal isthmus just posterior to the nerve ring. Cell types are colored according to the WormAtlas color code (&lt;&lt;REF:Altun-2002-?-?&gt;&gt;), for nuclei and mitochondria, arbitrary colors were chosen (HYP: hypodermis, S: seam cells, PM: pharyngeal muscle, MC: marginal cells, EXP: excretory pore, EXC: excretory canal, N: neuronal tissue, CUT: cuticle, n: nucleus. The white dashed rectangles show the positions of the tilt series acquired and corresponding reconstructions shown in b-d. Asterisks indicate ice contamination. The cross indicates the dorsal(D)-ventral(V) and left(L)-right(R) body axes. b, Tomographic slice of the perinuclear region of most likely a neuronal cell. The nucleus exhibits a different granularity and density than the cytoplasm (Cyto), from which it is separated by the nuclear envelope, which in turn is heavily decorated with ribosomes and contains nuclear pores (arrows). c, Tomographic slice of the pharynx. Central in the upper half is a marginal cell containing endoplasmatic reticulum (ER) and mitochondria (Mito). A neighboring neuronal cell body (CB) is separated from the pharynx by a diffuse density, which is the pharyngeal basal lamina (BL). d, Tomographic slice of a body wall muscle cross section (BWM). Clearly visible are the actin filaments in top view (arrowhead) surrounding a bundle of thick filaments (arrow). The thick filaments are partially interspersed with actin filaments. The muscle cell neighbors a hypodermal cell (HYP), from which it is separated by a space filled with a diffuse density, likely the body-wall muscle basal lamina. The next and last layer of the larval body wall is the cuticle (CUT). Most notably within this chitinous/collagenous structure, a fence-like array of denser structures can be discerned, likely cuticular collagen.\n\nQuality assessment and subtomogram analysis\nTo assess the quality of lamella thinning, tomogram thickness was measured for all 57 tomograms from the single-sided attachment and a random subset of 132 tomograms for the double-sided attachment experiment. The thickness for single-sided attachment was 253 nm +/- 125 nm. The thickness of the tomograms from the double-sided attachment experiment was more uniform, 303 +/- 40 nm (Figure 4 - Figure Supplement 1). The broader thickness distribution in single-sided attachment most likely stems from lamella bending and movement during milling due to the free-standing edge of the lamella.\nThe tilt series we obtained allowed us to reconstruct the 3D cellular architecture of cell types in different tissues such as the neuronal nuclear periphery (Figure 4b &lt;&lt;FIG:fig4&gt;&gt;), pharyngeal marginal cells and neuronal cell bodies (Figure 4c &lt;&lt;FIG:fig4&gt;&gt;) and body wall muscle, the hypodermis and the collagenous network of the cuticle (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nThe sectioning plane obtained when doing Serial Lift-Out from 'waffle'-type samples provides worm cross-sections. For a number of structures, such as nuclei, the difference between oblique longitudinal and transverse sectioning is minimal (Figure 4a &lt;&lt;FIG:fig4&gt;&gt;, Figure 4 - Figure Supplement 2). When investigating anisotropic structures such as the body wall muscle or pharynx, however, sectioning direction can greatly impact the interpretation of higher order structure. In longitudinal sectioned sarcomeres, located in body-wall muscle, appear as filaments running along the image plane. These filaments show an orientational distribution strongly biased towards the side view. In contrast, tomograms from Serial Lift-Out show transverse sections of the body-wall muscle. In this sectioning plane, actin and myosin run often perpendicular to the image plane and can be discerned as small and medium sized puncta. This more clearly reveals the organization of the acto-myosin bundling and holds the potential for further analysis of their packing (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nIn order to assess the quality of the data acquired, we picked 33,000 80S ribosome particles from 200 randomly selected tomograms from the double-sided attachment experiment. Subtomogram averaging and classification yielded a structure at a resolution of 6.9 A (GSFSC, Figure 5a &lt;&lt;FIG:fig5&gt;&gt;). The resolution was likely limited by lamella thickness, supported by the fact that the CTF could not be fit beyond 6 A.\nFigure 5:\nSubtomogram average reconstruction of the C. elegans 80S ribosome from in situ data.\na, The C. elegans ribosome to a resolution of 6.9 A (GSFSC). Density map is colored by local resolution. Note that protein alpha-helices and rRNA helices are clearly visible at this resolution. b, Four different ribosomal states obtained through subtomogram classification: Ribosome Class 1 with occupied A, P and EF sites, Ribosome Class 2 with an occupied P site, Ribosome Class 3 with occupied A and P sites, Ribosome Class with occupied A and EF sites.\nClassification yielded four different sub-populations in various translational states. When comparing these states to the recently published ribosome state landscape from D. discoideum [8], resemblances are apparent to the initiation state with an occupied P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 2), states with an occupied A and P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 3) and elongation factor bound states (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 1, Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 4).\n\nDiscussion\nIn the past, cryo-lift-out has been hampered by limited throughput and low overall success rate. Therefore, the technique has even been deemed 'practically impossible', or at least 'merely difficult' in the literature (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). The fact that the existing literature has not advanced beyond proof-of-principle experiments underlines this evaluation. The study of multicellular organisms and tissues by cryo-electron tomography, however, holds an enormous potential for biological discovery and technical advances are thus needed. We anticipate the combination of recent hardware and workflow improvements (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;, &lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) with the increased throughput of Serial Lift-Out to make cryo-ET data acquisition from high-pressure frozen material more attainable. In addition, plasma focused ion beam technology, while available for some years, was recently introduced to the cryo-FIB community (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Martynowycz-2023-14-1086&gt;&gt;, &lt;&lt;REF:Sergey-2018-?-?&gt;&gt;) and may further improve cryo-lift-out throughput by increased ablation rates.\nThe cryo-FIB lamella milling protocols developed to date remove most of the cell during lamella preparation. The final lamella represents only &lt;1 % of a eukaryotic cell and even less for multicellular organisms. While a high number of lamellae milled at different heights could restore that lost information through an ensemble average (&lt;&lt;REF:Ferguson-2017-147-653&gt;&gt;), Serial Lift-Out lamellae originate from a single specimen, yielding a more thorough characterization of its morphology. A variety of volume microscopy methods, e.g. volume electron microscopy or X-Ray tomography, retain this information. They remain, however, limited in resolution when compared to cryo-ET. Serial Lift-Out increases the contextual information retained in cryo-ET while preserving data acquisition at molecular resolution.\nHere, we have shown the ability to section in increments of one to four micrometers. This process has two steps that contribute to specimen loss: sectioning (~300-500 nm) and lamella milling. The latter can be minimized by finer sectioning. Preliminary experiments suggest that thinner sectioning down to 500 nm may be achievable. Reducing the section thickness even further in order to prepare lamellae directly from the extracted volume, though, will likely require major technological advances of the FIB instrument. Developments in ion beam shaping could become valuable for such endeavors, reducing the material lost during sectioning. Due to the physical basis of ablation in FIB preparation, however, material will always be lost to the milling process itself. Therefore, truly consecutive lamellae, similar to serial sections of plastic embedded samples, seem unlikely to be attainable.\nNevertheless, the creation of multiple sections within biological material increases the preparation throughput for HPF samples and adds contextual information. For model organisms with a stereotypic and well-described body plan such as C. elegans or Platynereis dumerilii (&lt;&lt;REF:Vergara-2021-184-4819&gt;&gt;), Serial Lift-Out sections and tomograms may be mapped back into context using other sources of volumetric data as illustrated in Figure 3a &lt;&lt;FIG:fig3&gt;&gt; (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;, &lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;). As a result, back-mapping analysis may enable label-free targeting of features and events that are tissue and cell-type specific.\nSerial Lift-Out also addresses the challenge that arises when dealing with anisotropic cells and tissues. As the sectioning plane of the specimen can be adapted, fluorescence or FIB/SEM data can inform the preparation of the lift-out site and, in turn, the sectioning angle. This adaptive preparation strategy can give new insights into the molecular architecture as illustrated by tomograms from transverse sections of muscle cells. When compared to previously obtained longitudinal sections (&lt;&lt;REF:Burbaum-2021-12-4086&gt;&gt;, &lt;&lt;REF:Davide-2023-?-?&gt;&gt;, &lt;&lt;REF:Wang-2021-184-2135&gt;&gt;), the transverse section shows actin-myosin packing from a novel angle revealing how actin filaments surround what is likely to be myosin heads. The combination of acquiring cryo-ET data on both the transverse and longitudinal sections also hold the potential to improve subtomogram analysis when facing structures with preferential orientation.\nIn addition to guiding site preparation for lift-out, cryo-correlative light and electron microscopy is more generally used to target specific subcellular events (&lt;&lt;REF:Arnold-2016-110-860&gt;&gt;, &lt;&lt;REF:Bieber-2022-119-e2209823119&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;). This technique, however, remains challenging for routine use within larger tissues. Improvements in the operation of integrated light microscopes are therefore necessary to streamline subcellular targeting in cryo-lift-out experiments. Serial Lift-Out combined with in-chamber light microscopy could increase the success rate of targeting by reducing the sample thickness used in the correlation, increasing the number of sections and in turn targeting attempts, and enabling to regularly check the fluorescence signal throughout the milling process.\nOne limitation of cryo-lift-out is the high rate of ice contamination during transfer. As can be seen in the TEM overviews (Figure 3 - Figure Supplement 1), large ice crystals tend to obstruct regions of the lamellae and prevent data acquisition. Serial Lift-Out, similar to on-grid preparations, compensates for the loss in imageable area through a higher yield of lamellae in comparison to previous lift-out techniques. Reducing transfer contamination, however, remains highly desirable and controlled environments, e.g. glove boxes (&lt;&lt;REF:Tacke-2021-213-107743&gt;&gt;), or other technological advances such as vacuum transfers will be needed to maximize the imageable area of cryo-FIB lamellae.\nFinally, the analysis of lamella thickness distributions and lamella survival rate suggest that double-sided attachment in cryo-lift-out is advantageous. This like likely due to the reduction of lamella bending during milling and greater stability during the transfer to the TEM.\nWith the methodological advances of Serial Lift-Out, the existing hurdles of lift-out have been greatly diminished, enabling data quality, throughput, and success rate in cryo-lift-out that permits the mapping of large tissue regions and whole organisms at the molecular level. Tomography on lamellae of an L1 larva obtained with our Serial Lift-Out method elucidated its ribosome to 6.9 A resolution and uncovered four different translational states. Taken together, Serial Lift-Out demonstrates enormous potential to discern and study the molecular anatomy of native tissues and small multicellular organisms.\n\nSupporting information\nExtended_Data_File\nFigure2_MovieSupplement1\nFigure2_MovieSupplement2\nFigure3_MovieSupplement1\n\nCompeting interests\nJMP holds a position on the advisory board of Thermo Fisher Scientific. CT is an employee of Thermo Fisher Scientific. The other authors declare that no competing interests exist.\n\nData availability\nTomograms have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes: EMD-17246, EMD-17247, EMD-17248, Subtomogram averages have been uploaded under accession numbers: EMD-17241, EMD-17242, EMD-17243, EMD-17244, EMD-17245 and will be released upon publication.\n\nMaterials and Methods\n\nSample Vitrification\nC. elegans strains AM140 (allele rmIs132[unc-54p::Q35::YFP]) and NK2476 (allele qy46[ina-1::mKate+loxP]) were cultivated according to standard methods on rich NGM (&lt;&lt;REF:Brenner-1974-77-71&gt;&gt;). To obtain a synchronous population of animals, gravid young adult worms were washed from five petri dishes (6 cm) with M9 medium. The worms were pooled into 15 mL centrifuge tubes and centrifuged at 175x g. Excess supernatant was removed and the worm suspension was mixed with bleach solution (3.75 mL 1M NaOH, 3.0 mL household bleach, 8.25 mL H2O (&lt;&lt;REF:Sulston-1988-?-587&gt;&gt;) and swiftly vortexed. Bleaching was continued with intermittent vortexing and checking on a stereomicroscope. As soon as roughly 50% of worms appeared broken, the bleaching procedure was stopped by the addition of egg-buffer (118 mM NaCl, 48 mM KCl, 2 mM CaCl2, 2 mM MgCl2, 25 mM HEPES, pH 7.3) and centrifugation at 175g for 1 minute. The supernatant was swiftly removed and the egg/worm carcass solution was washed another four times with egg-buffer. The egg/worm carcass mixture was then floated on 60 % sucrose solution and purified eggs were withdrawn from the surface (&lt;&lt;REF:Strange-2007-2-1003&gt;&gt;). The egg solution was washed in M9 buffer and larvae were allowed to hatch at 20 degreesC for 24 hours.\nThis population of developmentally arrested L1 larvae was used for vitrification, carried out following a modified version of Mahamid, et al. (&lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;) on a Leica EM-ICE (Leica Microsystems, Wetzlar, Germany). Sample carriers Type B (Leica Microsystems, Wetzlar, Germany) with a diameter of 6 mm were coated with a separating layer of cetyl palmitate solution (0.5% w/v in diethylether) by dipping the carriers briefly into the solution and placing them, cavity side down, on filter papers to let the solvent evaporate. A 2 microL drop of 20% (v/v) Ficoll 400 im M9 was applied to the flat side of the sample carrier and a Formvar-coated grid (50 mesh square or 75 mesh hexagonal mesh) was floated on this drop with the support film facing the liquid. Any excess cryoprotectant below the grid was wicked away, using pieces of Whatman No 1 filter paper (Whatman, Maidstone, UK). Next, synchronized L1 larvae were mixed with an equal amount of 40% Ficoll 400 in M9 medium to reach a final concentration of 20% cryoprotectant. 2microL of this sample solution was applied onto the grid and the sandwich was completed by addition of a second 6 mm sample carrier Type B, cavity-side up. The sample was immediately high-pressure-frozen. Grids were removed from the HPF sample carriers and stored in LN2 until further use. EM-grids were clipped into Thermo Fisher Scientific cartridges for FIB-milling. To be able to locate the biological material (L1 larvae) in the ice layer, clipped grids were either previously mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.\nDrosophila samples were prepared as previously described (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, fly strains were maintained at 22 degreesC on standard cornmeal agar. 24 hours prior to dissection of egg chambers, the flies were transferred into a new vial supplemented with yeast paste. Egg chambers were dissected in Schneider's medium. For high-pressure freezing, 3 mm HPF carriers were soaked in hexadecene and blotted on filter paper. Depending on the egg chamber stage, the dissected material was subsequently transferred into the 100 microm or 150 microm cavity of a 3 mm Type A or Type C HPF sample carrier (Engineering office M. Wohlwend, Sennwald, Switzerland), respectively. The filling medium was 20% Ficoll (70 kDa) in Schneider's medium. The flat surface of a Type B HPF Planchette was used to close the HPF assembly for freezing in the Leica EM ICE high-pressure freezer (Leica Microsystems, Wetzlar, Germany). The sample was subsequently pre-trimmed using a 45degrees diamond knife (Diatome, Nidau, Switzerland) in a cryo-microtome (EM UC6/FC6 cryo-microtome, Leica Microsystems) at a temperature of -170 degreesC and used for further FIB preparation.\n\nCryo-fluorescence microscopy\nGrids were clipped in Thermo Fisher Scientific (TFS) cartridges and imaged using a Leica TCS SP8 laser confocal microscope (Leica Microsystems, Wetzlar, Germany) fitted with a Leica cryo-stage (&lt;&lt;REF:Schorb-2017-197-83&gt;&gt;). Imaging was performed with a 50x 0.90 NA objective and HyD detectors.\nTile set montages of the NK2476 strain sample were collected using a pinhole size of 4.85 AU and a voxel size of 0.578 microm x 0.578 microm x 2.998 microm. The tile sets were merged and maximum intensity projections for each channel calculated using LAS X (Leica, Wetzlar, Germany). Autofluorescence imaging was performed for the double-sided attachment experiment. The 488 nm laser line at 2.5 % total power was used for excitation and the emission was detected from 501 nm to 535 nm. The reflection channel was excited at 552 nm and 2% total laser power and detected with an emission range of 547 nm to 557 nm.\nTile set montages of the AM140 strain were collected at a pinhole size of 1 AU and a voxel size of 0.578 microm x 0.578 microm x 1.03 microm. The tile sets were merged and maximum intensity projections for each channel were calculated using LAS X (Leica, Wetzlar, Germany). Q35-YFP was excited at 488 nm and 0.5 % total laser power and emission was detected from 509 nm to 551 nm. Reflection was recorded with 638 nm excitation at 0.83 % total laser power and the emission range set to 633 nm to 643 nm.\n\nSerial Lift Out - Single Side Attachment\n\nLift-In Grid Preparation\n100 square mesh Cu grids (Agar Scientific, Stansted, UK) were assembled into TFS cartridges. The cartridges were marked for manual alignment during sample loading. These markings were aligned to where the grid bars intersect the cartridge marks. The marks were also used for grid orientation during loading. The grids were then loaded into a FIB-SEM 45degrees pre-tilt cryo-shuttle such that the grid bars are aligned vertically and horizontally.\nOnce loaded, 14-pin grids were prepared by rotating the grid plane to be normal to the ion beam (trench milling orientation) and milling out two combs of horizontal grid bars. This yielded grids with 14 pins that can be used for 28 lamella attachments (Figure 2 - Figure Supplement 2). A horizontal grid bar was used to prepare 20 microm x 10 microm x 5 microm copper blocks for attachment to the lift-out needle by redeposition milling using single-pass regular cross-sections directed away from the needle (Figure 1 - Figure Supplement 2, Figure 2 - Figure Supplement 5a). Note, that the copper block attached to the needle can be re-used several times. The lower section of the copper block used for the previous attachment can be milled away leaving a clean surface for the next attachment.\nAs the FIB-SEM does not have to be cooled during lift-in grid milling, we recommend the preparation of several lift-in grids ahead of time in order to reduce workload during the serial lift-out session.\n\nLift-Out, Sectioning and Milling Procedure\nSerial lift-out was performed on an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. To facilitate lift-out, the EasyLift needle was modified by attaching a copper block adapter (see 'lift-in grid preparation') using redeposition. The copper adapter has a higher sputter yield and, thus, increased redeposition compared to tungsten and vitreous ice. Additionally, the block increases the surface area available for attachment to the biology, and reduces the wear of the needle over time.\nAfter copper block attachment, the volume to be extracted was prepared by milling trenches around the region of interest in trench milling orientation. The fluorescence data acquired of the L1 'waffle' samples was correlated to the ion beam images using MAPS version 3.14 (Thermo Fisher Scientific, Waltham, MA, U.S.A.). The grid was coated with a protective metal-organic platinum layer using a gas injection system (GIS) heated to 27degreesC at a stage working distance of 9.72 mm and a stage tilt of 45degrees for 90s. Trenches were milled with the grid perpendicular to the ion beam (trench milling orientation) after which the EasyLift needle was inserted. In order to increase the redeposition yield for attachment, the copper block needs to be aligned below the sample surface and flush to the surface exposed by trench milling. Redeposition from the copper block onto the extraction volume was achieved using single-pass patterns of regular cross-sections instead of the default multi-pass mode to avoid re-milling previously redeposited material. These patterns were placed at the interface of the two surfaces on the copper adapter with the milling direction for these patterns directed away from the extraction volume (Figure 2 - Figure Supplement 5b). The extraction volume was then released through milling the last connection to the bulk sample with a line pattern. The 30 microm x 110 microm x 25 microm extracted volume was subsequently lifted from the bulk sample.\nAfter the extraction of the volume, the shuttle was returned to the lamella milling orientation (18degrees in a system with a 45degrees pre-tilt) and the stage translated to the receiver grid. The lower edge of the extracted volume and the corner of the pin were aligned in both electron and ion beam images. Attachment was performed by redeposition, using single-pass regular cross-section patterns directed away from the extracted volume (Figure 2 - Figure Supplement 5d). Once attached, the EasyLift system was maneuvered by a 50 nm step horizontally and vertically away from the pin in horizontal direction in order to create a small amount of strain. Subsequently, the lower 4 microm of the volume were sectioned from the rest of the volume using a line pattern. This process was iterated until the entire extracted volume had been sectioned.\n\nSerial Lift-Out: Double-Sided Attachment\n\nLift-In Grid Preparation\nFor double-sided attachment, 100/400 rectangular mesh copper grids (Gilder, Grantham, UK) were used as the receiver grid. These were clipped into standard TFS cartridges that marked such that the 400 mesh bars were in line with the bottom mark and the 100 mesh bars were in line with the side markings. This grid was loaded into the FIB shuttle and 20 microm x 10 microm x 5 microm copper block adapters were prepared as described above.\nDuring the preparation of the receiver grid, roughly 5 microm deep line patterns were milled across the horizontal 400 mesh grid bars in trench milling orientation to divide the rectangles approximately (1/3) and (2/3). These marker lines are intended to guide the alignment of the extracted volume by providing a reference visible in both the electron and ion beams during the attachment process.\n\nLift Out, Sectioning and Milling Procedure\nLift-out site preparation for double-sided attachment was performed on an Aquilos 1 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with a METEOR in-chamber fluorescence light microscope (Delmic, Delft, Netherlands)(&lt;&lt;REF:Smeets-2021-29-20&gt;&gt;). The the GIS was employed for 90 s at 27degreesC with at a stage working distance set to 10.6 mm and the grid normal to the GIS needle to apply a layer of protective metal-organic platinum. The METEOR in-chamber microscope was used in conjunction with previously acquired cryo-confocal data to localize the larva of interest and produce an extraction volume of 42 microm x 180 microm x 20 microm. A width of 40 microm is necessary in order to span the space between the grid bars of a 100/400 rectangular mesh grid. A slight excess of width of the extraction volume is preferable, since a too narrow block will not allow for double-sided attachment, while material can be ablated during sectioning.\nThe lift-out and receiver grid were subsequently transferred into an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. Lift-out was otherwise performed as stated in the section for single-sided attachment. In brief, the volume was attached to the needle via the copper adapter, the volume was released, the needle retracted, and the shuttle was returned to the lamella milling orientation at the receiver grid. The extracted volume was re-inserted, and its lower edge was positioned in-between two grid bars. In the case when the volume was slightly too wide, material was ablated from its sides as necessary. To allow for proper attachment, the volume's lower edge was aligned to the reference line on the receiver grid in both, the electron and ion beam images and the volume was attached on both sides by redeposition from the grid bars. To achieve this, single-pass cross-section patterns were placed on the grid bars above the marker lines in in close proximity of the interface between the grid bars and extracted volume, while the milling was directed away from the volume (Figure 2 - Figure Supplement 5c). Once attached, the EasyLift system was maneuvered up in the z-direction by 50-100 nm in order to create a small amount of strain. Subsequently, a section was released from the extracted volume by milling a line pattern at 4 microm above the volume's lower edge. This process was iterated until the entire extracted volume had been sectioned.\nMilling pattern parameters for Serial Lift-Out.\nThe table summarizes the parameters used for the milling steps necessary to perform a Serial Lift-Out experiment. The corresponding pattern geometries are illustrated in Figure 2 - Figure Supplement 5. All patterns were milled at 30.00 kV FIB acceleration voltage. Pattern files for Thermo Fisher Scientific instruments are provided in Figure 2 &lt;&lt;FIG:fig2&gt;&gt; - Pattern Files Supplement. These templates and the milling parameters may need adjustment for different projects and for FIB-SEM instruments from other manufacturers, specifically concerning differing scanning strategies deployed by the microscope manufacturer.\n\nFine milling of lift-out sections\nFor both single-sided and double-sided attachment, the initial 4 microm sections were thinned in two steps: rough milling and fine milling. Rough milling was performed using regular cross-section patterns at a beam current of 1 nA to a thickness of 1.5 microm, 0.5 nA to 1.2 microm lamella thickness, 0.3 nA to 0.8 microm lamella thickness. After rough milling all of the sections, fine milling was performed using regular rectangle patterns at a beam current of 0.1 nA to 0.4 microm and 50 pA to final thickness. Over-tilting and under-tilting by up to 1degrees was used for beam convergence compensation. The double-sided attached lamellae were sputter coated with platinum after fine milling for 4 seconds, at a chamber pressure of 0.20 mbar and a current of 15.0 mA.\n\nSerial lift-out from high-pressure frozen D. melanogaster egg chambers\nLift-out experiments of high-pressure frozen D. melanogaster egg chambers were performed on a Scios FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift needle. The preparation of the lift-out volume as described previously (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, an approximately 20 microm x 20 microm volume of target material was prepared using regular cross-section patterns in trench milling orientation in a horseshoe-like shape. The trenches were ~20 microm wide, except for the region that needed to be accessed by the lift-out needle, which was ~35 microm wide. The extraction volume was undercut at a stage tilt of 45degrees, or the highest tilt reachable for the specific position. This preparation leaves the extraction volume attached to the bulk material on a single side. Subsequently, the procedure for single-sided attachment was performed as described above. Sectioning was performed in increments of 1-3 microm using a line pattern milled at a beam current of at 0.3 nA. After section preparation, fine milling was performed by sequentially decreasing the beam current: 0.3 nA to 800 nm, 0.1 nA to 500 nm, 50 pA to 350 nm and 30 pA to &lt; 300 nm. The final step was judged by the loss of contrast in SEM imaging at 3 kV acceleration voltage and a beam current of 13 pA.\n\nLamella preparation by the 'waffle' milling method\nFor preparation of lamellae directly on the high-pressure frozen grid, vitrified as described above, we followed a similar workflow as published as the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The grid was coated with a metal-organic platinum layer by GIS deposition for 20 s at a stage working distance of 10.6 mm. Milling was performed on a gallium FIB-SEM Aquilos 2 (Thermo Fisher Scientific, Waltham, MA, U.S.A.) instrument. Initial trenches were milled in trench milling orientation at a beam current of 3 nA using regular cross-section patterns. To avoid damaging the region of interest by milling at high beam currents, 2 microm of buffer distance were left around the region of interest. Trenches were extended to 15 microm on the backside of the subsequent lamella and 30 microm on the front side. In a next step, the trenches were extended to 1 microm closer to the region of interest at a beam current of 1 nA and in a last step the final dimension of the lamella region was defined milling at 0.5 nA. After trench milling, another layer of protective metal-organic platinum was deposited on the sample (20 s deposition time, 10.6 mm stage working distance, three consecutive times). The stage was rotated into lamella milling orientation and a notch was milled with line patterns at 0.3 nA as previously described (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The preparation of the final lamella was carried out by removing material above and below of the region of interest at a beam current of 0.3 nA, until a final lamella thickness of 2 microm was reached. In sequential steps of decreasing beam current (to 0.8 microm thickness at 0.1 nA beam current, to 0.4 microm at 50 pA), the remnant material was ablated. The lamella was polished to a final thickness of about 200-250 nm at 30 pA beam current.\n\nTEM Data Acquisition\nTEM data acquisition was performed on a Titan Krios G4 at 300 kV equipped with a Selectris X energy filter and a Falcon 4i camera (Thermo Fisher Scientific, Eindhoven, The Netherlands). Lamella overview montages were recorded by stage-driven tiling at 11,500x nominal magnification (pixel size 2.127 nm). Tomograms were recorded using the Tomo5 software package version 5.12.0 (Thermo Fisher Scientific, Eindhoven, The Netherlands) using the EER file format.\nTwo acquisition strategies were deployed. For the tomograms of the single-sided attachment and 'waffle' preparation tilt series were acquired at a nominal magnification of 42,000x resulting in a pixel size at the sample of 2.93 A. A dose-symmetric tilt scheme was used with an angular increment of 2degrees, a dose of 2 e-/A2 per tilt and a target defocus of -4 to -5.5 microm. Tilt series were collected in a tilt range of -70degrees to 50degrees due to the lamellae pre-tilt and with a total dose of 120 e-/A2.\nFor the tomograms collected of the double-sided attachment lamellae, that were subsequently used for subtomogram analysis, tilt series were acquired at a nominal magnification of 64,000x, resulting in a pixel size of 1.89 A. Data was collected using a dose-symmetric tilt scheme with an angular increment of 3degrees, a dose of 3.23 e-/A2 per tilt and a target defocus range of -1 to -4 microm. Angles of -70degrees to 50degrees were acquired, resulting in a total dose of 132 e-/A2.\n\nTomogram Reconstruction, Visualization and Subtomogram Analysis\nData was processed using the Tomoman version 0.7 pipeline (https://github.com/williamnwan/TOMOMAN). 14 frames with a dose of 0.23 e-/A2 per frame were rendered from the EER files. These were used for motion correction in MotionCor2 version 1.4.7 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;) and CTF estimation with CTFFIND4 version 4.14 (&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;). Bad tilts were removed after manual inspection using the Tomoman script. Dose-weighting was performed at 3.23 e-/A2 per tilt using either Tomoman or Warp. For denoising, tilt series were separated into odd and even tilts during motion correction and the resulting stacks were processed using Cryo-CARE (&lt;&lt;REF:Buchholz-2019-?-502&gt;&gt;). Tomogram reconstructions for visualization were done in IMOD version 4.12.32. Tilt series were aligned with AreTomo version 1.3.3. CTF-corrected tomograms for template matching were reconstructed in IMOD version 4.12.32 (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) at eight times binning, resulting in a pixel size of 15.6 A.\nInitial template matching was performed in STOPGAP version 0.7 (&lt;&lt;REF:Wan-2020-26-2516&gt;&gt;) on a subset of 70 tomograms at bin8 using PDB 4V4B as a reference filtered to 35 A (&lt;&lt;REF:Spahn-2004-23-1008&gt;&gt;). 16,420 particles were extracted and aligned in STOPGAP to generate the C. elegans 80S ribosome template. The template was subsequently used to repeat template matching on 200 tomograms at bin8. 65,451 particles were extracted and cleaned by projecting the subtomogram along the Z-axis and subsequent 2D classification in Relion version 4.0. The remaining data set contained 37,026 particles. The retained particles were reprocessed in Warp version 1.0.9, cleaned to remove particles with inadequate CTF resolution and astigmatism. The resulting 35,350 particles were extracted with a pixel size of 2.98 A and a boxsize of 160 pixels. Relion version 3.0 with a spherical mask of 340 A radius was used to align the subtomograms. Finally, the particles were imported into M version 1.0.9 and geometric and CTF parameters were sequentially refined. Corrected subtomograms were extracted from M and classified in Relion 3.0. Two rounds of classification with a spherical mask of 340 A in diameter resulted in 8,256 particles being removed. The second step was a focused classification with a spherical mask around the A/P/E-site. The resulting 5 classes were combined into 2 classes according to the small subunit rotation. Each of these 2 merged classes were subjected to another round of 3D classification with a mask around the elongation factor binding site. The final classes were manually pooled according to structural similarity yielding 5 classes with 2,175, 20,638, 877, 1,119, and 2,285 particles, of which the class with 877 particles was neglected. Segmentation of the elongation factor and tRNAs was done with the Segger tool in Chimera. ChimeraX version 1.3 was used for visualization (&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;).",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nSingle particle analysis (SPA) by cryo-transmission electron microscopy (cryo-TEM) has become a key technique to study the structure of isolated biological macromolecules at high-resolution (&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;). While SPA routinely reaches resolutions where protein side-chains can be fitted unambiguously, the reductionist approach of studying protein complexes in vitro loses all information concerning their molecular sociology: the interaction of the molecular complexes in their natural environment (&lt;&lt;REF:Beck-2016-26-825&gt;&gt;). Conversely, in situ cryo-electron tomography (cryo-ET) allows for the reconstruction of pleomorphic structures such as the crowded environment of the cell at molecular resolution, maintaining the interaction and localization of protein complexes within the biological system (&lt;&lt;REF:Dietrich-2022-607-823&gt;&gt;, &lt;&lt;REF:Gupta-2021-184-3643&gt;&gt;, &lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;, &lt;&lt;REF:Plitzko-2017-46-110&gt;&gt;, &lt;&lt;REF:Watanabe-2020-182-1508&gt;&gt;).\nOne of the primary factors limiting the resolution of cryo-TEM is inelastic scattering. As the mean free path of an electron in vitrified biological samples is about 300-400 nm, samples beyond the size of viruses and small prokaryotic cells are generally too thick for cryo-ET (&lt;&lt;REF:Liedtke-2022-6-100065&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;). In consequence, two main sample thinning methods to obtain electron-transparent specimens for cryo-ET have been developed: cryo-ultramicrotomy and cryo-FIB milling. Cryo-ultramicrotomy encompasses the thin-sectioning of vitreous cells and tissues with a diamond knife (&lt;&lt;REF:Al-Amoudi-2004-148-131&gt;&gt;). The shearing forces at the knife's edge, however, cause mechanical artifacts such as crevices and compression in the resulting sections (&lt;&lt;REF:Al-Amoudi-2003-9-372&gt;&gt;). More recently, the focused ion beam (FIB) instrument has been widely adopted for sample thinning at cryogenic temperatures. While not completely damage-free (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Lucas-2023-?-?&gt;&gt;), the technique bypasses the mechanical artifacts of cryo-ultramicrotomy (&lt;&lt;REF:Marko-2007-4-215&gt;&gt;, &lt;&lt;REF:Rigort-2012-109-4449&gt;&gt;) and has been shown to yield data that can allow for the elucidation of ribosomes to side-chain resolution (&lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;). The automation of lamella preparation by cryo-FIB milling has also reduced the need for user expertise and manual intervention (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Zachs-2020-9-?&gt;&gt;).\nPrior to thinning, the sample must be cryogenically fixed by cooling at a sufficiently high rate to prevent ice crystal formation, resulting in a vitrified sample. There are two main methods available for vitrification: plunge freezing, in which the sample is immersed at ambient pressure into liquid ethane or ethane-propane mixture (&lt;&lt;REF:Tivol-2008-14-375&gt;&gt;), and high pressure freezing (HPF), in which the sample is cooled with a jet of liquid nitrogen at a pressure of ~2000 bar. While the former yields samples that are easily FIB-milled, the sample thickness that can reliably be vitrified is limited to roughly 10 microm. HPF, on the other hand, allows for the vitrification of samples up to a thickness of roughly 200 microm (&lt;&lt;REF:Dubochet-1995-5-366&gt;&gt;).\nConsequently, HPF greatly expands the size range of biological samples that can be vitrified but comes at the cost of embedding the specimen in a thick layer of ice defined by the depth of the freezing receptacle. This increased sample thickness leads to longer milling times. While samples up to a thickness of about 50 microm can be prepared by milling lamellae directly on the grid following the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;), lamellae from thicker samples have, to date, only been prepared by cryo-lift-out (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Kuba-2021-281-112&gt;&gt;, &lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nCryo-lift-out refers to the extraction of the material for lamella preparation from bulk HPF sample and subsequent transfer and attachment to a lift-out receiver grid, conventionally a half-moon shaped grid (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;). Two main types of micromanipulator devices are currently available for cryo-lift-out: sharp needles and a cryo-gripper (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;). Initially, trenches are milled around the area of interest leaving it connected to the bulk material on a single side. For specimens in HPF sample carriers, the material must additionally be cleared from below. After these preparatory steps, the lift-out device is brought in contact with the volume to be extracted. The remaining connection to the bulk material is removed, the micromanipulator is used to transfer the extracted volume and redeposition milling (&lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) is used to attach the volume to the receiver grid. Finally, an electron transparent lamella is prepared (&lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;).\nWhile widely used in materials science at room temperature (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;), cryo-lift-out of biological samples has remained primarily proof-of-concept (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;). This is due to a number of factors, e.g. the need for cooled micromanipulator devices and accompanying workflow adaptations required resulting in limited throughput and problems with lamella loss during transfer to the TEM (&lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nAnother limitation to cryo-lift-out, as well as on-grid lamella preparation is the loss of contextual information. Only a fraction of the sample volume (~1% for larger eukaryotic cells, &lt; 1% for multicellular specimens) ends up inside the final lamella for cryo-ET data acquisition. Techniques exist that are capable of capturing larger volumetric data and tracking morphology at comparatively large volume scales. Examples of such techniques are X-ray tomography and various volume electron microscopy (vEM) techniques: serial FIB milling and scanning electron microscopy, and serial sectioning of plastic embedded samples imaged by TEM or STEM. These techniques, however, currently cannot achieve the resolution attainable by cryo-ET at an equivalent sample preservation state due to physical limitations in imaging or the necessity of fixation and contrasting (&lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;, &lt;&lt;REF:Spehner-2020-211-107528&gt;&gt;).\nIn this work, we describe a novel cryo-lift-out approach that creates a series of lamellae from one lift-out volume that we term Serial Lift-Out. Inspired by diamond knife serial sectioning, Serial Lift-Out retains more contextual information than previous procedures and increases the throughput of cryo-lift-out by an order of magnitude. It sets the stage for the study of multicellular organisms and tissues by cryo-ET - applications that previously seemed practically impossible (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). We demonstrate Serial Lift-Out on high-pressure frozen Caenorhabditis elegans L1 stage larvae, sampling their ultrastructure along the anterior-posterior axis by cryo-electron tomography. From the resulting dataset, we reconstruct the nematode's ribosome to 7 A resolution by subtomogram averaging, exemplifying the enormous potential of Serial Lift-Out for the study of the molecular anatomy of multicellular systems.\n\nResults\n\nThe concept behind Serial Lift-Out\nThe most time-consuming steps in cryo-lift-out are the preparation of the extraction sites and the transfer of the extracted volume to the receiving grid. Each repetition of the lift-out cycle therefore adds significant time investment with low yield. Scaling up the extracted volume and producing multiple lamellae from a single lift-out bypasses this repetitive, time-consuming trench milling and transfer, increases throughput and, foremost, provides more contextual information about the targeted material.\nA specific implementation of this concept is the extraction of an entire L1 larva of C. elegans, followed by repeated steps of attachment to the receiving grid, sectioning and transfer of the remaining volume to the next attachment site (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;). While material is still lost during sectioning and thinning, a large fraction of the worm is made readily accessible to cryo-TEM data acquisition.\nFigure 1:\nA schematic of Serial Lift-Out.\nAn illustration of the Serial Lift-Out method, exemplified by the process being performed on a C. elegans L1 larva embedded in an extracted volume of vitrified ice (blue). a, The extracted volume containing the larva. b, c, Schematic representation of the resulting Serial Lift-Out sections and lamellae, respectively. d, Illustration of the double-sided attachment Serial Lift-Out procedure. The extracted volume shown in panel a is attached to the lift-out needle via a copper block adaptor and transferred to a rectangular mesh receiver grid. Several sections are shown, obtained by the repetition of the attachment of the bottom part of the volume to the grid bars and subsequent sectioning.\nPrevious lift-out approaches extracted volumes approximately the size of the final lamella (Figure 1 - Figure Supplement 1a), with some excess material for stability during transfer and attachment. In order to section multiple lamellae from a single cryo-lift-out transfer, the volume extracted by cryo-lift-out needed to be increased. To ease the manipulation of such large volumes, we introduced a copper block that acts as an adapter between the needle and the extracted volume. This copper block is created from the receiver grid and attached to the needle before lift-out (Figure 1 - Figure Supplement 2, Figure 1 - Figure Supplement 3). As copper has higher redeposition rates than the tungsten needle, the copper block adapter results in significantly more resilient \"welding\" of the specimen to the micromanipulator.\nAnother volume limiting factor is the ablation rate of the ion beam. Most cryo-FIB machines are equipped with a gallium ion source rendering it impractical to mill beyond 50 microm in depth. In addition, for samples frozen in HPF carriers, preparation requires an undercut, the removal of material below the extracted volume, to detach the extraction volume from the bulk. In combination, these factors generally result in maximum extraction volumes of approximately 20 microm x 20 microm x 10-30 microm (length x width x height, Figure 1 - Figure Supplement 1b).\nIn order to extract larger volumes, we performed lift-out on an HPF 'waffle'-type sample. HPF 'waffle' samples are prepared by freezing sample on a grid that is sandwiched between HPF carriers (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The final thickness is therefore defined by the type of grid and spacer being used during freezing. For a 25 microm thick specimen, extraction volumes of up to 200 microm x 40 microm x 25 microm can easily be obtained by performing lift-out with the sample surface oriented perpendicular to the ion beam (Figure 1 - Figure Supplement 1c). The same orientation is used during trench milling and will be referred to as 'trench milling orientation' (Figure 1 - Figure Supplement 3c,f). Such large volumes extracted at the trench milling orientation can yield many sections from a single lift-out which can be subsequently thinned to lamellae for cryo-ET data acquisition (Figure 1b,c &lt;&lt;FIG:fig1&gt;&gt;).\n\nApplication of Serial Lift-Out to C. elegans L1 larvae\nTo assess the feasibility of obtaining serial lamellae from a single lift-out transfer, we performed the procedure on C. elegans L1 larvae that had been vitrified using a modified protocol of the 'waffle' method (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The sample contained many L1 larvae embedded in an approximately 25 microm thick layer of ice (Figure 2 - Figure Supplement 1). Two sites from two different grids were selected for preparation by correlating cryo-fluorescence light microscopy with SEM and FIB view images. Following the geometry described in Figure 1 &lt;&lt;FIG:fig1&gt;&gt; - Supplement 1c, the sites were prepared by milling trenches around the larva, yielding a 30 microm x 110 microm x 25 microm and a 40 microm x 180 microm x 25 microm extraction volume (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3a). Two different attachment strategies were explored, with resulting lamellae either being attached on one side (single-sided attachment) or two sides (double-sided attachment). To extract the volumes, the needle with the copper adapter (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3b) was attached to the sample using redeposition: copper material redepositing onto the surface of the extraction volume during milling. The extraction volume was subsequently milled free from the bulk material and lifted out.\nFigure 2:\nA workflow for double-sided attachment Serial Lift-Out.\na, FIB image of the prepared extraction site with overlaid correlated fluorescence data (green) indicating the larva being targeted. The micromanipulator is attached to the extraction volume by redeposition from the copper adapter (trench milling orientation). b, The extracted volume (orange dashed line) is lowered into position between two grid bars in lamella milling orientation. The lower front edge of the volume (yellow arrows) is aligned to the pre-milled line mark (black arrows). c, Double-sided attachment by redeposition from the grid bars (yellow arrows indicate direction of milling), followed by line pattern milling releasing the section of a desired thickness (dashed yellow line). Orange dashed line indicates the outline of the extracted volume. d, SEM image of a typical section after being released from the extracted volume. Black dashed line indicates the outline of the worm cross-section. e,f, SEM (e) and TEM (f) overview images of the 40 double-sided attached sections obtained. Figure 2 - Movie Supplement 1 summarizes the process.\nThe volume was then transferred to the receiver grid. For single-sided attachment, a modified grid based on customizing a standard 100 mesh copper grid (Figure 2 - Figure Supplement 2) was used. Removing every second row of grid bars yielded an array of trimmed grid bars that resemble the pins of a standard half-moon shaped lift-out grid (Figure 2 - Figure Supplement 2e). Alternatively, a copper grid with rectangular meshes can be used for double-sided attachment of volumes roughly as wide as the mesh (~40 microm for the 400/100 rectangular mesh grids used here, Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Double- sided attachment has the added benefit of increased section stability by eliminating the free-standing side (Figure 2 - Figure Supplement 3f).\nThe volume's lower front edge was precisely aligned to the front edge of the attachment pin (Figure 2 - Figure Supplement 3c) or a previously prepared alignment line pattern milled onto the receiver grid (Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Then, redeposition milling was used to attach the lower part of the volume to the grid bar(s) (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow arrows). After attachment, the lower part of the target material was separated from the extracted volume using a line pattern (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow dashed line), leaving a ~4 microm thick section (Figure 2d &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3e,f). Once sectioned, the remaining volume attached to the lift-out needle was transferred to the next attachment site. This procedure was iterated until no material remained. As a result, many sections were produced from a single cryo-lift-out transfer (Figure 2e-f &lt;&lt;FIG:fig2&gt;&gt;). To assess the general applicability of the approach to non-'waffle' type samples, an additional experiment was performed on D. melanogaster egg chambers, high-pressure frozen in standard sample carriers. From a ~20 microm long and ~15 microm deep extracted volume we produced sections of ~1-2 microm thickness (Figure 2 - Figure Supplement 4).\n\nSampling organismal cellular anatomy along the C. elegans L1 larva at molecular resolution\nWe prepared dozens of sections along the anterior-posterior axis of C. elegans L1 larvae: 12 single-sided attached and 40 double-sided attached lamellae (Figure 2 - Figure Supplement 3g, Figure 2e &lt;&lt;FIG:fig2&gt;&gt;). After transfer to the TEM, 8 out of 12 and 32 out of 40 lamellae were recovered, respectively (Figure 3 - Figure Supplement 1a, Figure 3 - Figure Supplement 2a). Lamella loss during transfer is common due to manual grid handling steps. The increased rate of successfully transferred lamellae from 66% for single-sided to 80% for double-sided attachment is indicative of the increased lamella stability of double-sided attachment.\nFigure 3:\nLamella TEM overviews sample the anatomy of a C. elegans L1 larva along the anterior-posterior body axis.\nNative tissue scattering contrast is sufficient to extract a considerable amount of anatomical information from low magnification TEM overviews of the lamellae generated in the double-sided attachment Serial Lift-Out experiment. a, Schematic representation of 29 body transverse section overviews obtained from the final lamellae along the anterior-posterior body axis of an L1 larva. Anterior is located to the left, posterior to the right. The OpenWorm project model of the adult C. elegans worm was used for illustrating the possibility of back mapping since a cellular model of the L1 larva only exists for its head and not the entire body (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;). The cross-sections were cropped from lamella overview images and mapped back to a position derived from the known sectioning distance and anatomical features discernible in the corresponding cross-section. The dashed white frames indicate overviews with corresponding magnified representations in panels b-g. b, This lamella originates from ~15 microm along the anterior-posterior axis. Clearly visible are the three lobes of the anterior pharyngeal lumen in the center of the worm cross-section (asterisk) and the relatively electron dense pharyngeal lining. c, Overview from the anterior part of the pharyngeal isthmus ~42 microm along the anterior-posterior axis. Note the nerve ring (dashed line) surrounding the central pharynx. Additionally, the alae (asterisks) running along the left and right lateral side of the worm become obvious. d, Overview of a lamella of roughly the center of the posterior pharyngeal bulb region. The central grinder organ is clearly discernible (asterisk). This section is positioned ~65 microm along the anterior-posterior axis. e, A section roughly mid-body. The intestinal lumen (dashed line) and intestinal cells (dotted line) are obvious. The darker cell slightly left of the body center is likely one of the gonadal primordial cells (asterisk). The section is from ~115 microm along the anterior-posterior axis. f, In this mid-body section, the intestinal lumen (dashed line) can again be clearly discerned. The section can be mapped to ~132 microm along the anterior-posterior axis. g, Section showing the intestinal lumen at ~155 microm along the anterior-posterior axis.\nFrom these successfully transferred lamellae, overview maps were recorded (approximately 20 microm x 25 microm in size, Figure 3 - Figure Supplement 1, Figure 3 - Figure Supplement 2) depicting clear larval cross-sections. These overviews allowed the identification and assignment of anatomical structures and tissues such as the pharynx, the body-wall muscle cells, neurons, the hypodermis and seam cells. Larger organelles such as nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, bundles of actin filaments and microtubules were clearly discernible (Figure 3b-g &lt;&lt;FIG:fig3&gt;&gt;, Figure3 - Movie Supplement 1). Given the stereotypic body plan of C. elegans, the information obtained from the overviews together with the sectioning thickness was used to determine the approximate location of the sections in the worm, as schematically shown in Figure 3a &lt;&lt;FIG:fig3&gt;&gt;.\nThe biological area within each lamella can, due to the circular nature of cross sections, be estimated by ?r2, where the average radius is around 5 microm, resulting in a mean of 78.5 microm2 imageable area per lamella. For the single-sided attachment experiment, the total imageable area was ~ 630 microm2 and allowed for the collection of 57 tilt series, each with a field of view of ~1.2 microm in size. The double-sided attachment experiment yielded ~2500 microm2 imageable area from which a total of 1012 tilt series with a field of view of ~750 nm in size were collected. The double-sided attachment experiment allowed us to sample a large fraction of all tissues and cell types along the anterior-posterior body axis of the C. elegans L1 larva.\nTo illustrate the information level already present at intermediate magnification (11,500x), we manually segmented a representative cross-section (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). The segmentation illustrates that the cuticle clearly delineates the body cross-section. Body-wall muscle cells are obvious due to their pronounced distal actomyosin pattern. The pharyngeal lumen's trilobal structure demarcates the body center and is surrounded by the three alternating pharyngeal muscle and marginal cells. Pharyngeal neurons and gland cell process cross-sections are embedded in the pharyngeal muscle cells. Nuclei are easy to discern due to their ribosome-decorated double-membrane and their denser, less granular interior. Neurons in general can be discerned by their appearance as round or tubular cells, often grouped in bundles. These clearly interpretable overviews allow targeting of tissue-specific cellular structures or cellular protein complexes with known location (e.g. sarcomeric proteins).\nFigure 4:\nRepresentative overview and tomograms of the L1 larval pharyngeal isthmus region.\na, Overview montage (magnification 11,500x) of the head region lamella. This section is located within the pharyngeal isthmus just posterior to the nerve ring. Cell types are colored according to the WormAtlas color code (&lt;&lt;REF:Altun-2002-?-?&gt;&gt;), for nuclei and mitochondria, arbitrary colors were chosen (HYP: hypodermis, S: seam cells, PM: pharyngeal muscle, MC: marginal cells, EXP: excretory pore, EXC: excretory canal, N: neuronal tissue, CUT: cuticle, n: nucleus. The white dashed rectangles show the positions of the tilt series acquired and corresponding reconstructions shown in b-d. Asterisks indicate ice contamination. The cross indicates the dorsal(D)-ventral(V) and left(L)-right(R) body axes. b, Tomographic slice of the perinuclear region of most likely a neuronal cell. The nucleus exhibits a different granularity and density than the cytoplasm (Cyto), from which it is separated by the nuclear envelope, which in turn is heavily decorated with ribosomes and contains nuclear pores (arrows). c, Tomographic slice of the pharynx. Central in the upper half is a marginal cell containing endoplasmatic reticulum (ER) and mitochondria (Mito). A neighboring neuronal cell body (CB) is separated from the pharynx by a diffuse density, which is the pharyngeal basal lamina (BL). d, Tomographic slice of a body wall muscle cross section (BWM). Clearly visible are the actin filaments in top view (arrowhead) surrounding a bundle of thick filaments (arrow). The thick filaments are partially interspersed with actin filaments. The muscle cell neighbors a hypodermal cell (HYP), from which it is separated by a space filled with a diffuse density, likely the body-wall muscle basal lamina. The next and last layer of the larval body wall is the cuticle (CUT). Most notably within this chitinous/collagenous structure, a fence-like array of denser structures can be discerned, likely cuticular collagen.\n\nQuality assessment and subtomogram analysis\nTo assess the quality of lamella thinning, tomogram thickness was measured for all 57 tomograms from the single-sided attachment and a random subset of 132 tomograms for the double-sided attachment experiment. The thickness for single-sided attachment was 253 nm +/- 125 nm. The thickness of the tomograms from the double-sided attachment experiment was more uniform, 303 +/- 40 nm (Figure 4 - Figure Supplement 1). The broader thickness distribution in single-sided attachment most likely stems from lamella bending and movement during milling due to the free-standing edge of the lamella.\nThe tilt series we obtained allowed us to reconstruct the 3D cellular architecture of cell types in different tissues such as the neuronal nuclear periphery (Figure 4b &lt;&lt;FIG:fig4&gt;&gt;), pharyngeal marginal cells and neuronal cell bodies (Figure 4c &lt;&lt;FIG:fig4&gt;&gt;) and body wall muscle, the hypodermis and the collagenous network of the cuticle (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nThe sectioning plane obtained when doing Serial Lift-Out from 'waffle'-type samples provides worm cross-sections. For a number of structures, such as nuclei, the difference between oblique longitudinal and transverse sectioning is minimal (Figure 4a &lt;&lt;FIG:fig4&gt;&gt;, Figure 4 - Figure Supplement 2). When investigating anisotropic structures such as the body wall muscle or pharynx, however, sectioning direction can greatly impact the interpretation of higher order structure. In longitudinal sectioned sarcomeres, located in body-wall muscle, appear as filaments running along the image plane. These filaments show an orientational distribution strongly biased towards the side view. In contrast, tomograms from Serial Lift-Out show transverse sections of the body-wall muscle. In this sectioning plane, actin and myosin run often perpendicular to the image plane and can be discerned as small and medium sized puncta. This more clearly reveals the organization of the acto-myosin bundling and holds the potential for further analysis of their packing (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nIn order to assess the quality of the data acquired, we picked 33,000 80S ribosome particles from 200 randomly selected tomograms from the double-sided attachment experiment. Subtomogram averaging and classification yielded a structure at a resolution of 6.9 A (GSFSC, Figure 5a &lt;&lt;FIG:fig5&gt;&gt;). The resolution was likely limited by lamella thickness, supported by the fact that the CTF could not be fit beyond 6 A.\nFigure 5:\nSubtomogram average reconstruction of the C. elegans 80S ribosome from in situ data.\na, The C. elegans ribosome to a resolution of 6.9 A (GSFSC). Density map is colored by local resolution. Note that protein alpha-helices and rRNA helices are clearly visible at this resolution. b, Four different ribosomal states obtained through subtomogram classification: Ribosome Class 1 with occupied A, P and EF sites, Ribosome Class 2 with an occupied P site, Ribosome Class 3 with occupied A and P sites, Ribosome Class with occupied A and EF sites.\nClassification yielded four different sub-populations in various translational states. When comparing these states to the recently published ribosome state landscape from D. discoideum [8], resemblances are apparent to the initiation state with an occupied P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 2), states with an occupied A and P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 3) and elongation factor bound states (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 1, Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 4).\n\nDiscussion\nIn the past, cryo-lift-out has been hampered by limited throughput and low overall success rate. Therefore, the technique has even been deemed 'practically impossible', or at least 'merely difficult' in the literature (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). The fact that the existing literature has not advanced beyond proof-of-principle experiments underlines this evaluation. The study of multicellular organisms and tissues by cryo-electron tomography, however, holds an enormous potential for biological discovery and technical advances are thus needed. We anticipate the combination of recent hardware and workflow improvements (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;, &lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) with the increased throughput of Serial Lift-Out to make cryo-ET data acquisition from high-pressure frozen material more attainable. In addition, plasma focused ion beam technology, while available for some years, was recently introduced to the cryo-FIB community (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Martynowycz-2023-14-1086&gt;&gt;, &lt;&lt;REF:Sergey-2018-?-?&gt;&gt;) and may further improve cryo-lift-out throughput by increased ablation rates.\nThe cryo-FIB lamella milling protocols developed to date remove most of the cell during lamella preparation. The final lamella represents only &lt;1 % of a eukaryotic cell and even less for multicellular organisms. While a high number of lamellae milled at different heights could restore that lost information through an ensemble average (&lt;&lt;REF:Ferguson-2017-147-653&gt;&gt;), Serial Lift-Out lamellae originate from a single specimen, yielding a more thorough characterization of its morphology. A variety of volume microscopy methods, e.g. volume electron microscopy or X-Ray tomography, retain this information. They remain, however, limited in resolution when compared to cryo-ET. Serial Lift-Out increases the contextual information retained in cryo-ET while preserving data acquisition at molecular resolution.\nHere, we have shown the ability to section in increments of one to four micrometers. This process has two steps that contribute to specimen loss: sectioning (~300-500 nm) and lamella milling. The latter can be minimized by finer sectioning. Preliminary experiments suggest that thinner sectioning down to 500 nm may be achievable. Reducing the section thickness even further in order to prepare lamellae directly from the extracted volume, though, will likely require major technological advances of the FIB instrument. Developments in ion beam shaping could become valuable for such endeavors, reducing the material lost during sectioning. Due to the physical basis of ablation in FIB preparation, however, material will always be lost to the milling process itself. Therefore, truly consecutive lamellae, similar to serial sections of plastic embedded samples, seem unlikely to be attainable.\nNevertheless, the creation of multiple sections within biological material increases the preparation throughput for HPF samples and adds contextual information. For model organisms with a stereotypic and well-described body plan such as C. elegans or Platynereis dumerilii (&lt;&lt;REF:Vergara-2021-184-4819&gt;&gt;), Serial Lift-Out sections and tomograms may be mapped back into context using other sources of volumetric data as illustrated in Figure 3a &lt;&lt;FIG:fig3&gt;&gt; (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;, &lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;). As a result, back-mapping analysis may enable label-free targeting of features and events that are tissue and cell-type specific.\nSerial Lift-Out also addresses the challenge that arises when dealing with anisotropic cells and tissues. As the sectioning plane of the specimen can be adapted, fluorescence or FIB/SEM data can inform the preparation of the lift-out site and, in turn, the sectioning angle. This adaptive preparation strategy can give new insights into the molecular architecture as illustrated by tomograms from transverse sections of muscle cells. When compared to previously obtained longitudinal sections (&lt;&lt;REF:Burbaum-2021-12-4086&gt;&gt;, &lt;&lt;REF:Davide-2023-?-?&gt;&gt;, &lt;&lt;REF:Wang-2021-184-2135&gt;&gt;), the transverse section shows actin-myosin packing from a novel angle revealing how actin filaments surround what is likely to be myosin heads. The combination of acquiring cryo-ET data on both the transverse and longitudinal sections also hold the potential to improve subtomogram analysis when facing structures with preferential orientation.\nIn addition to guiding site preparation for lift-out, cryo-correlative light and electron microscopy is more generally used to target specific subcellular events (&lt;&lt;REF:Arnold-2016-110-860&gt;&gt;, &lt;&lt;REF:Bieber-2022-119-e2209823119&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;). This technique, however, remains challenging for routine use within larger tissues. Improvements in the operation of integrated light microscopes are therefore necessary to streamline subcellular targeting in cryo-lift-out experiments. Serial Lift-Out combined with in-chamber light microscopy could increase the success rate of targeting by reducing the sample thickness used in the correlation, increasing the number of sections and in turn targeting attempts, and enabling to regularly check the fluorescence signal throughout the milling process.\nOne limitation of cryo-lift-out is the high rate of ice contamination during transfer. As can be seen in the TEM overviews (Figure 3 - Figure Supplement 1), large ice crystals tend to obstruct regions of the lamellae and prevent data acquisition. Serial Lift-Out, similar to on-grid preparations, compensates for the loss in imageable area through a higher yield of lamellae in comparison to previous lift-out techniques. Reducing transfer contamination, however, remains highly desirable and controlled environments, e.g. glove boxes (&lt;&lt;REF:Tacke-2021-213-107743&gt;&gt;), or other technological advances such as vacuum transfers will be needed to maximize the imageable area of cryo-FIB lamellae.\nFinally, the analysis of lamella thickness distributions and lamella survival rate suggest that double-sided attachment in cryo-lift-out is advantageous. This like likely due to the reduction of lamella bending during milling and greater stability during the transfer to the TEM.\nWith the methodological advances of Serial Lift-Out, the existing hurdles of lift-out have been greatly diminished, enabling data quality, throughput, and success rate in cryo-lift-out that permits the mapping of large tissue regions and whole organisms at the molecular level. Tomography on lamellae of an L1 larva obtained with our Serial Lift-Out method elucidated its ribosome to 6.9 A resolution and uncovered four different translational states. Taken together, Serial Lift-Out demonstrates enormous potential to discern and study the molecular anatomy of native tissues and small multicellular organisms.\n\nSupporting information\nExtended_Data_File\nFigure2_MovieSupplement1\nFigure2_MovieSupplement2\nFigure3_MovieSupplement1\n\nCompeting interests\nJMP holds a position on the advisory board of Thermo Fisher Scientific. CT is an employee of Thermo Fisher Scientific. The other authors declare that no competing interests exist.\n\nData availability\nTomograms have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes: EMD-17246, EMD-17247, EMD-17248, Subtomogram averages have been uploaded under accession numbers: EMD-17241, EMD-17242, EMD-17243, EMD-17244, EMD-17245 and will be released upon publication.\n\nMaterials and Methods\n\nSample Vitrification\nC. elegans strains AM140 (allele rmIs132[unc-54p::Q35::YFP]) and NK2476 (allele qy46[ina-1::mKate+loxP]) were cultivated according to standard methods on rich NGM (&lt;&lt;REF:Brenner-1974-77-71&gt;&gt;). To obtain a synchronous population of animals, gravid young adult worms were washed from five petri dishes (6 cm) with M9 medium. The worms were pooled into 15 mL centrifuge tubes and centrifuged at 175x g. Excess supernatant was removed and the worm suspension was mixed with bleach solution (3.75 mL 1M NaOH, 3.0 mL household bleach, 8.25 mL H2O (&lt;&lt;REF:Sulston-1988-?-587&gt;&gt;) and swiftly vortexed. Bleaching was continued with intermittent vortexing and checking on a stereomicroscope. As soon as roughly 50% of worms appeared broken, the bleaching procedure was stopped by the addition of egg-buffer (118 mM NaCl, 48 mM KCl, 2 mM CaCl2, 2 mM MgCl2, 25 mM HEPES, pH 7.3) and centrifugation at 175g for 1 minute. The supernatant was swiftly removed and the egg/worm carcass solution was washed another four times with egg-buffer. The egg/worm carcass mixture was then floated on 60 % sucrose solution and purified eggs were withdrawn from the surface (&lt;&lt;REF:Strange-2007-2-1003&gt;&gt;). The egg solution was washed in M9 buffer and larvae were allowed to hatch at 20 degreesC for 24 hours.\nThis population of developmentally arrested L1 larvae was used for vitrification, carried out following a modified version of Mahamid, et al. (&lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;) on a Leica EM-ICE (Leica Microsystems, Wetzlar, Germany). Sample carriers Type B (Leica Microsystems, Wetzlar, Germany) with a diameter of 6 mm were coated with a separating layer of cetyl palmitate solution (0.5% w/v in diethylether) by dipping the carriers briefly into the solution and placing them, cavity side down, on filter papers to let the solvent evaporate. A 2 microL drop of 20% (v/v) Ficoll 400 im M9 was applied to the flat side of the sample carrier and a Formvar-coated grid (50 mesh square or 75 mesh hexagonal mesh) was floated on this drop with the support film facing the liquid. Any excess cryoprotectant below the grid was wicked away, using pieces of Whatman No 1 filter paper (Whatman, Maidstone, UK). Next, synchronized L1 larvae were mixed with an equal amount of 40% Ficoll 400 in M9 medium to reach a final concentration of 20% cryoprotectant. 2microL of this sample solution was applied onto the grid and the sandwich was completed by addition of a second 6 mm sample carrier Type B, cavity-side up. The sample was immediately high-pressure-frozen. Grids were removed from the HPF sample carriers and stored in LN2 until further use. EM-grids were clipped into Thermo Fisher Scientific cartridges for FIB-milling. To be able to locate the biological material (L1 larvae) in the ice layer, clipped grids were either previously mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.\nDrosophila samples were prepared as previously described (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, fly strains were maintained at 22 degreesC on standard cornmeal agar. 24 hours prior to dissection of egg chambers, the flies were transferred into a new vial supplemented with yeast paste. Egg chambers were dissected in Schneider's medium. For high-pressure freezing, 3 mm HPF carriers were soaked in hexadecene and blotted on filter paper. Depending on the egg chamber stage, the dissected material was subsequently transferred into the 100 microm or 150 microm cavity of a 3 mm Type A or Type C HPF sample carrier (Engineering office M. Wohlwend, Sennwald, Switzerland), respectively. The filling medium was 20% Ficoll (70 kDa) in Schneider's medium. The flat surface of a Type B HPF Planchette was used to close the HPF assembly for freezing in the Leica EM ICE high-pressure freezer (Leica Microsystems, Wetzlar, Germany). The sample was subsequently pre-trimmed using a 45degrees diamond knife (Diatome, Nidau, Switzerland) in a cryo-microtome (EM UC6/FC6 cryo-microtome, Leica Microsystems) at a temperature of -170 degreesC and used for further FIB preparation.\n\nCryo-fluorescence microscopy\nGrids were clipped in Thermo Fisher Scientific (TFS) cartridges and imaged using a Leica TCS SP8 laser confocal microscope (Leica Microsystems, Wetzlar, Germany) fitted with a Leica cryo-stage (&lt;&lt;REF:Schorb-2017-197-83&gt;&gt;). Imaging was performed with a 50x 0.90 NA objective and HyD detectors.\nTile set montages of the NK2476 strain sample were collected using a pinhole size of 4.85 AU and a voxel size of 0.578 microm x 0.578 microm x 2.998 microm. The tile sets were merged and maximum intensity projections for each channel calculated using LAS X (Leica, Wetzlar, Germany). Autofluorescence imaging was performed for the double-sided attachment experiment. The 488 nm laser line at 2.5 % total power was used for excitation and the emission was detected from 501 nm to 535 nm. The reflection channel was excited at 552 nm and 2% total laser power and detected with an emission range of 547 nm to 557 nm.\nTile set montages of the AM140 strain were collected at a pinhole size of 1 AU and a voxel size of 0.578 microm x 0.578 microm x 1.03 microm. The tile sets were merged and maximum intensity projections for each channel were calculated using LAS X (Leica, Wetzlar, Germany). Q35-YFP was excited at 488 nm and 0.5 % total laser power and emission was detected from 509 nm to 551 nm. Reflection was recorded with 638 nm excitation at 0.83 % total laser power and the emission range set to 633 nm to 643 nm.\n\nSerial Lift Out - Single Side Attachment\n\nLift-In Grid Preparation\n100 square mesh Cu grids (Agar Scientific, Stansted, UK) were assembled into TFS cartridges. The cartridges were marked for manual alignment during sample loading. These markings were aligned to where the grid bars intersect the cartridge marks. The marks were also used for grid orientation during loading. The grids were then loaded into a FIB-SEM 45degrees pre-tilt cryo-shuttle such that the grid bars are aligned vertically and horizontally.\nOnce loaded, 14-pin grids were prepared by rotating the grid plane to be normal to the ion beam (trench milling orientation) and milling out two combs of horizontal grid bars. This yielded grids with 14 pins that can be used for 28 lamella attachments (Figure 2 - Figure Supplement 2). A horizontal grid bar was used to prepare 20 microm x 10 microm x 5 microm copper blocks for attachment to the lift-out needle by redeposition milling using single-pass regular cross-sections directed away from the needle (Figure 1 - Figure Supplement 2, Figure 2 - Figure Supplement 5a). Note, that the copper block attached to the needle can be re-used several times. The lower section of the copper block used for the previous attachment can be milled away leaving a clean surface for the next attachment.\nAs the FIB-SEM does not have to be cooled during lift-in grid milling, we recommend the preparation of several lift-in grids ahead of time in order to reduce workload during the serial lift-out session.\n\nLift-Out, Sectioning and Milling Procedure\nSerial lift-out was performed on an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. To facilitate lift-out, the EasyLift needle was modified by attaching a copper block adapter (see 'lift-in grid preparation') using redeposition. The copper adapter has a higher sputter yield and, thus, increased redeposition compared to tungsten and vitreous ice. Additionally, the block increases the surface area available for attachment to the biology, and reduces the wear of the needle over time.\nAfter copper block attachment, the volume to be extracted was prepared by milling trenches around the region of interest in trench milling orientation. The fluorescence data acquired of the L1 'waffle' samples was correlated to the ion beam images using MAPS version 3.14 (Thermo Fisher Scientific, Waltham, MA, U.S.A.). The grid was coated with a protective metal-organic platinum layer using a gas injection system (GIS) heated to 27degreesC at a stage working distance of 9.72 mm and a stage tilt of 45degrees for 90s. Trenches were milled with the grid perpendicular to the ion beam (trench milling orientation) after which the EasyLift needle was inserted. In order to increase the redeposition yield for attachment, the copper block needs to be aligned below the sample surface and flush to the surface exposed by trench milling. Redeposition from the copper block onto the extraction volume was achieved using single-pass patterns of regular cross-sections instead of the default multi-pass mode to avoid re-milling previously redeposited material. These patterns were placed at the interface of the two surfaces on the copper adapter with the milling direction for these patterns directed away from the extraction volume (Figure 2 - Figure Supplement 5b). The extraction volume was then released through milling the last connection to the bulk sample with a line pattern. The 30 microm x 110 microm x 25 microm extracted volume was subsequently lifted from the bulk sample.\nAfter the extraction of the volume, the shuttle was returned to the lamella milling orientation (18degrees in a system with a 45degrees pre-tilt) and the stage translated to the receiver grid. The lower edge of the extracted volume and the corner of the pin were aligned in both electron and ion beam images. Attachment was performed by redeposition, using single-pass regular cross-section patterns directed away from the extracted volume (Figure 2 - Figure Supplement 5d). Once attached, the EasyLift system was maneuvered by a 50 nm step horizontally and vertically away from the pin in horizontal direction in order to create a small amount of strain. Subsequently, the lower 4 microm of the volume were sectioned from the rest of the volume using a line pattern. This process was iterated until the entire extracted volume had been sectioned.\n\nSerial Lift-Out: Double-Sided Attachment\n\nLift-In Grid Preparation\nFor double-sided attachment, 100/400 rectangular mesh copper grids (Gilder, Grantham, UK) were used as the receiver grid. These were clipped into standard TFS cartridges that marked such that the 400 mesh bars were in line with the bottom mark and the 100 mesh bars were in line with the side markings. This grid was loaded into the FIB shuttle and 20 microm x 10 microm x 5 microm copper block adapters were prepared as described above.\nDuring the preparation of the receiver grid, roughly 5 microm deep line patterns were milled across the horizontal 400 mesh grid bars in trench milling orientation to divide the rectangles approximately (1/3) and (2/3). These marker lines are intended to guide the alignment of the extracted volume by providing a reference visible in both the electron and ion beams during the attachment process.\n\nLift Out, Sectioning and Milling Procedure\nLift-out site preparation for double-sided attachment was performed on an Aquilos 1 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with a METEOR in-chamber fluorescence light microscope (Delmic, Delft, Netherlands)(&lt;&lt;REF:Smeets-2021-29-20&gt;&gt;). The the GIS was employed for 90 s at 27degreesC with at a stage working distance set to 10.6 mm and the grid normal to the GIS needle to apply a layer of protective metal-organic platinum. The METEOR in-chamber microscope was used in conjunction with previously acquired cryo-confocal data to localize the larva of interest and produce an extraction volume of 42 microm x 180 microm x 20 microm. A width of 40 microm is necessary in order to span the space between the grid bars of a 100/400 rectangular mesh grid. A slight excess of width of the extraction volume is preferable, since a too narrow block will not allow for double-sided attachment, while material can be ablated during sectioning.\nThe lift-out and receiver grid were subsequently transferred into an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. Lift-out was otherwise performed as stated in the section for single-sided attachment. In brief, the volume was attached to the needle via the copper adapter, the volume was released, the needle retracted, and the shuttle was returned to the lamella milling orientation at the receiver grid. The extracted volume was re-inserted, and its lower edge was positioned in-between two grid bars. In the case when the volume was slightly too wide, material was ablated from its sides as necessary. To allow for proper attachment, the volume's lower edge was aligned to the reference line on the receiver grid in both, the electron and ion beam images and the volume was attached on both sides by redeposition from the grid bars. To achieve this, single-pass cross-section patterns were placed on the grid bars above the marker lines in in close proximity of the interface between the grid bars and extracted volume, while the milling was directed away from the volume (Figure 2 - Figure Supplement 5c). Once attached, the EasyLift system was maneuvered up in the z-direction by 50-100 nm in order to create a small amount of strain. Subsequently, a section was released from the extracted volume by milling a line pattern at 4 microm above the volume's lower edge. This process was iterated until the entire extracted volume had been sectioned.\nMilling pattern parameters for Serial Lift-Out.\nThe table summarizes the parameters used for the milling steps necessary to perform a Serial Lift-Out experiment. The corresponding pattern geometries are illustrated in Figure 2 - Figure Supplement 5. All patterns were milled at 30.00 kV FIB acceleration voltage. Pattern files for Thermo Fisher Scientific instruments are provided in Figure 2 &lt;&lt;FIG:fig2&gt;&gt; - Pattern Files Supplement. These templates and the milling parameters may need adjustment for different projects and for FIB-SEM instruments from other manufacturers, specifically concerning differing scanning strategies deployed by the microscope manufacturer.\n\nFine milling of lift-out sections\nFor both single-sided and double-sided attachment, the initial 4 microm sections were thinned in two steps: rough milling and fine milling. Rough milling was performed using regular cross-section patterns at a beam current of 1 nA to a thickness of 1.5 microm, 0.5 nA to 1.2 microm lamella thickness, 0.3 nA to 0.8 microm lamella thickness. After rough milling all of the sections, fine milling was performed using regular rectangle patterns at a beam current of 0.1 nA to 0.4 microm and 50 pA to final thickness. Over-tilting and under-tilting by up to 1degrees was used for beam convergence compensation. The double-sided attached lamellae were sputter coated with platinum after fine milling for 4 seconds, at a chamber pressure of 0.20 mbar and a current of 15.0 mA.\n\nSerial lift-out from high-pressure frozen D. melanogaster egg chambers\nLift-out experiments of high-pressure frozen D. melanogaster egg chambers were performed on a Scios FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift needle. The preparation of the lift-out volume as described previously (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, an approximately 20 microm x 20 microm volume of target material was prepared using regular cross-section patterns in trench milling orientation in a horseshoe-like shape. The trenches were ~20 microm wide, except for the region that needed to be accessed by the lift-out needle, which was ~35 microm wide. The extraction volume was undercut at a stage tilt of 45degrees, or the highest tilt reachable for the specific position. This preparation leaves the extraction volume attached to the bulk material on a single side. Subsequently, the procedure for single-sided attachment was performed as described above. Sectioning was performed in increments of 1-3 microm using a line pattern milled at a beam current of at 0.3 nA. After section preparation, fine milling was performed by sequentially decreasing the beam current: 0.3 nA to 800 nm, 0.1 nA to 500 nm, 50 pA to 350 nm and 30 pA to &lt; 300 nm. The final step was judged by the loss of contrast in SEM imaging at 3 kV acceleration voltage and a beam current of 13 pA.\n\nLamella preparation by the 'waffle' milling method\nFor preparation of lamellae directly on the high-pressure frozen grid, vitrified as described above, we followed a similar workflow as published as the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The grid was coated with a metal-organic platinum layer by GIS deposition for 20 s at a stage working distance of 10.6 mm. Milling was performed on a gallium FIB-SEM Aquilos 2 (Thermo Fisher Scientific, Waltham, MA, U.S.A.) instrument. Initial trenches were milled in trench milling orientation at a beam current of 3 nA using regular cross-section patterns. To avoid damaging the region of interest by milling at high beam currents, 2 microm of buffer distance were left around the region of interest. Trenches were extended to 15 microm on the backside of the subsequent lamella and 30 microm on the front side. In a next step, the trenches were extended to 1 microm closer to the region of interest at a beam current of 1 nA and in a last step the final dimension of the lamella region was defined milling at 0.5 nA. After trench milling, another layer of protective metal-organic platinum was deposited on the sample (20 s deposition time, 10.6 mm stage working distance, three consecutive times). The stage was rotated into lamella milling orientation and a notch was milled with line patterns at 0.3 nA as previously described (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The preparation of the final lamella was carried out by removing material above and below of the region of interest at a beam current of 0.3 nA, until a final lamella thickness of 2 microm was reached. In sequential steps of decreasing beam current (to 0.8 microm thickness at 0.1 nA beam current, to 0.4 microm at 50 pA), the remnant material was ablated. The lamella was polished to a final thickness of about 200-250 nm at 30 pA beam current.\n\nTEM Data Acquisition\nTEM data acquisition was performed on a Titan Krios G4 at 300 kV equipped with a Selectris X energy filter and a Falcon 4i camera (Thermo Fisher Scientific, Eindhoven, The Netherlands). Lamella overview montages were recorded by stage-driven tiling at 11,500x nominal magnification (pixel size 2.127 nm). Tomograms were recorded using the Tomo5 software package version 5.12.0 (Thermo Fisher Scientific, Eindhoven, The Netherlands) using the EER file format.\nTwo acquisition strategies were deployed. For the tomograms of the single-sided attachment and 'waffle' preparation tilt series were acquired at a nominal magnification of 42,000x resulting in a pixel size at the sample of 2.93 A. A dose-symmetric tilt scheme was used with an angular increment of 2degrees, a dose of 2 e-/A2 per tilt and a target defocus of -4 to -5.5 microm. Tilt series were collected in a tilt range of -70degrees to 50degrees due to the lamellae pre-tilt and with a total dose of 120 e-/A2.\nFor the tomograms collected of the double-sided attachment lamellae, that were subsequently used for subtomogram analysis, tilt series were acquired at a nominal magnification of 64,000x, resulting in a pixel size of 1.89 A. Data was collected using a dose-symmetric tilt scheme with an angular increment of 3degrees, a dose of 3.23 e-/A2 per tilt and a target defocus range of -1 to -4 microm. Angles of -70degrees to 50degrees were acquired, resulting in a total dose of 132 e-/A2.\n\nTomogram Reconstruction, Visualization and Subtomogram Analysis\nData was processed using the Tomoman version 0.7 pipeline (https://github.com/williamnwan/TOMOMAN). 14 frames with a dose of 0.23 e-/A2 per frame were rendered from the EER files. These were used for motion correction in MotionCor2 version 1.4.7 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;) and CTF estimation with CTFFIND4 version 4.14 (&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;). Bad tilts were removed after manual inspection using the Tomoman script. Dose-weighting was performed at 3.23 e-/A2 per tilt using either Tomoman or Warp. For denoising, tilt series were separated into odd and even tilts during motion correction and the resulting stacks were processed using Cryo-CARE (&lt;&lt;REF:Buchholz-2019-?-502&gt;&gt;). Tomogram reconstructions for visualization were done in IMOD version 4.12.32. Tilt series were aligned with AreTomo version 1.3.3. CTF-corrected tomograms for template matching were reconstructed in IMOD version 4.12.32 (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) at eight times binning, resulting in a pixel size of 15.6 A.\nInitial template matching was performed in STOPGAP version 0.7 (&lt;&lt;REF:Wan-2020-26-2516&gt;&gt;) on a subset of 70 tomograms at bin8 using PDB 4V4B as a reference filtered to 35 A (&lt;&lt;REF:Spahn-2004-23-1008&gt;&gt;). 16,420 particles were extracted and aligned in STOPGAP to generate the C. elegans 80S ribosome template. The template was subsequently used to repeat template matching on 200 tomograms at bin8. 65,451 particles were extracted and cleaned by projecting the subtomogram along the Z-axis and subsequent 2D classification in Relion version 4.0. The remaining data set contained 37,026 particles. The retained particles were reprocessed in Warp version 1.0.9, cleaned to remove particles with inadequate CTF resolution and astigmatism. The resulting 35,350 particles were extracted with a pixel size of 2.98 A and a boxsize of 160 pixels. Relion version 3.0 with a spherical mask of 340 A radius was used to align the subtomograms. Finally, the particles were imported into M version 1.0.9 and geometric and CTF parameters were sequentially refined. Corrected subtomograms were extracted from M and classified in Relion 3.0. Two rounds of classification with a spherical mask of 340 A in diameter resulted in 8,256 particles being removed. The second step was a focused classification with a spherical mask around the A/P/E-site. The resulting 5 classes were combined into 2 classes according to the small subunit rotation. Each of these 2 merged classes were subjected to another round of 3D classification with a mask around the elongation factor binding site. The final classes were manually pooled according to structural similarity yielding 5 classes with 2,175, 20,638, 877, 1,119, and 2,285 particles, of which the class with 877 particles was neglected. Segmentation of the elongation factor and tRNAs was done with the Segger tool in Chimera. ChimeraX version 1.3 was used for visualization (&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;).",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nSingle particle analysis (SPA) by cryo-transmission electron microscopy (cryo-TEM) has become a key technique to study the structure of isolated biological macromolecules at high-resolution (&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;). While SPA routinely reaches resolutions where protein side-chains can be fitted unambiguously, the reductionist approach of studying protein complexes in vitro loses all information concerning their molecular sociology: the interaction of the molecular complexes in their natural environment (&lt;&lt;REF:Beck-2016-26-825&gt;&gt;). Conversely, in situ cryo-electron tomography (cryo-ET) allows for the reconstruction of pleomorphic structures such as the crowded environment of the cell at molecular resolution, maintaining the interaction and localization of protein complexes within the biological system (&lt;&lt;REF:Dietrich-2022-607-823&gt;&gt;, &lt;&lt;REF:Gupta-2021-184-3643&gt;&gt;, &lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;, &lt;&lt;REF:Plitzko-2017-46-110&gt;&gt;, &lt;&lt;REF:Watanabe-2020-182-1508&gt;&gt;).\nOne of the primary factors limiting the resolution of cryo-TEM is inelastic scattering. As the mean free path of an electron in vitrified biological samples is about 300-400 nm, samples beyond the size of viruses and small prokaryotic cells are generally too thick for cryo-ET (&lt;&lt;REF:Liedtke-2022-6-100065&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;). In consequence, two main sample thinning methods to obtain electron-transparent specimens for cryo-ET have been developed: cryo-ultramicrotomy and cryo-FIB milling. Cryo-ultramicrotomy encompasses the thin-sectioning of vitreous cells and tissues with a diamond knife (&lt;&lt;REF:Al-Amoudi-2004-148-131&gt;&gt;). The shearing forces at the knife's edge, however, cause mechanical artifacts such as crevices and compression in the resulting sections (&lt;&lt;REF:Al-Amoudi-2003-9-372&gt;&gt;). More recently, the focused ion beam (FIB) instrument has been widely adopted for sample thinning at cryogenic temperatures. While not completely damage-free (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Lucas-2023-?-?&gt;&gt;), the technique bypasses the mechanical artifacts of cryo-ultramicrotomy (&lt;&lt;REF:Marko-2007-4-215&gt;&gt;, &lt;&lt;REF:Rigort-2012-109-4449&gt;&gt;) and has been shown to yield data that can allow for the elucidation of ribosomes to side-chain resolution (&lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;). The automation of lamella preparation by cryo-FIB milling has also reduced the need for user expertise and manual intervention (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Zachs-2020-9-?&gt;&gt;).\nPrior to thinning, the sample must be cryogenically fixed by cooling at a sufficiently high rate to prevent ice crystal formation, resulting in a vitrified sample. There are two main methods available for vitrification: plunge freezing, in which the sample is immersed at ambient pressure into liquid ethane or ethane-propane mixture (&lt;&lt;REF:Tivol-2008-14-375&gt;&gt;), and high pressure freezing (HPF), in which the sample is cooled with a jet of liquid nitrogen at a pressure of ~2000 bar. While the former yields samples that are easily FIB-milled, the sample thickness that can reliably be vitrified is limited to roughly 10 microm. HPF, on the other hand, allows for the vitrification of samples up to a thickness of roughly 200 microm (&lt;&lt;REF:Dubochet-1995-5-366&gt;&gt;).\nConsequently, HPF greatly expands the size range of biological samples that can be vitrified but comes at the cost of embedding the specimen in a thick layer of ice defined by the depth of the freezing receptacle. This increased sample thickness leads to longer milling times. While samples up to a thickness of about 50 microm can be prepared by milling lamellae directly on the grid following the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;), lamellae from thicker samples have, to date, only been prepared by cryo-lift-out (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Kuba-2021-281-112&gt;&gt;, &lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nCryo-lift-out refers to the extraction of the material for lamella preparation from bulk HPF sample and subsequent transfer and attachment to a lift-out receiver grid, conventionally a half-moon shaped grid (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;). Two main types of micromanipulator devices are currently available for cryo-lift-out: sharp needles and a cryo-gripper (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;). Initially, trenches are milled around the area of interest leaving it connected to the bulk material on a single side. For specimens in HPF sample carriers, the material must additionally be cleared from below. After these preparatory steps, the lift-out device is brought in contact with the volume to be extracted. The remaining connection to the bulk material is removed, the micromanipulator is used to transfer the extracted volume and redeposition milling (&lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) is used to attach the volume to the receiver grid. Finally, an electron transparent lamella is prepared (&lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;).\nWhile widely used in materials science at room temperature (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;), cryo-lift-out of biological samples has remained primarily proof-of-concept (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;). This is due to a number of factors, e.g. the need for cooled micromanipulator devices and accompanying workflow adaptations required resulting in limited throughput and problems with lamella loss during transfer to the TEM (&lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nAnother limitation to cryo-lift-out, as well as on-grid lamella preparation is the loss of contextual information. Only a fraction of the sample volume (~1% for larger eukaryotic cells, &lt; 1% for multicellular specimens) ends up inside the final lamella for cryo-ET data acquisition. Techniques exist that are capable of capturing larger volumetric data and tracking morphology at comparatively large volume scales. Examples of such techniques are X-ray tomography and various volume electron microscopy (vEM) techniques: serial FIB milling and scanning electron microscopy, and serial sectioning of plastic embedded samples imaged by TEM or STEM. These techniques, however, currently cannot achieve the resolution attainable by cryo-ET at an equivalent sample preservation state due to physical limitations in imaging or the necessity of fixation and contrasting (&lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;, &lt;&lt;REF:Spehner-2020-211-107528&gt;&gt;).\nIn this work, we describe a novel cryo-lift-out approach that creates a series of lamellae from one lift-out volume that we term Serial Lift-Out. Inspired by diamond knife serial sectioning, Serial Lift-Out retains more contextual information than previous procedures and increases the throughput of cryo-lift-out by an order of magnitude. It sets the stage for the study of multicellular organisms and tissues by cryo-ET - applications that previously seemed practically impossible (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). We demonstrate Serial Lift-Out on high-pressure frozen Caenorhabditis elegans L1 stage larvae, sampling their ultrastructure along the anterior-posterior axis by cryo-electron tomography. From the resulting dataset, we reconstruct the nematode's ribosome to 7 A resolution by subtomogram averaging, exemplifying the enormous potential of Serial Lift-Out for the study of the molecular anatomy of multicellular systems.\n\nResults\n\nThe concept behind Serial Lift-Out\nThe most time-consuming steps in cryo-lift-out are the preparation of the extraction sites and the transfer of the extracted volume to the receiving grid. Each repetition of the lift-out cycle therefore adds significant time investment with low yield. Scaling up the extracted volume and producing multiple lamellae from a single lift-out bypasses this repetitive, time-consuming trench milling and transfer, increases throughput and, foremost, provides more contextual information about the targeted material.\nA specific implementation of this concept is the extraction of an entire L1 larva of C. elegans, followed by repeated steps of attachment to the receiving grid, sectioning and transfer of the remaining volume to the next attachment site (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;). While material is still lost during sectioning and thinning, a large fraction of the worm is made readily accessible to cryo-TEM data acquisition.\nFigure 1:\nA schematic of Serial Lift-Out.\nAn illustration of the Serial Lift-Out method, exemplified by the process being performed on a C. elegans L1 larva embedded in an extracted volume of vitrified ice (blue). a, The extracted volume containing the larva. b, c, Schematic representation of the resulting Serial Lift-Out sections and lamellae, respectively. d, Illustration of the double-sided attachment Serial Lift-Out procedure. The extracted volume shown in panel a is attached to the lift-out needle via a copper block adaptor and transferred to a rectangular mesh receiver grid. Several sections are shown, obtained by the repetition of the attachment of the bottom part of the volume to the grid bars and subsequent sectioning.\nPrevious lift-out approaches extracted volumes approximately the size of the final lamella (Figure 1 - Figure Supplement 1a), with some excess material for stability during transfer and attachment. In order to section multiple lamellae from a single cryo-lift-out transfer, the volume extracted by cryo-lift-out needed to be increased. To ease the manipulation of such large volumes, we introduced a copper block that acts as an adapter between the needle and the extracted volume. This copper block is created from the receiver grid and attached to the needle before lift-out (Figure 1 - Figure Supplement 2, Figure 1 - Figure Supplement 3). As copper has higher redeposition rates than the tungsten needle, the copper block adapter results in significantly more resilient \"welding\" of the specimen to the micromanipulator.\nAnother volume limiting factor is the ablation rate of the ion beam. Most cryo-FIB machines are equipped with a gallium ion source rendering it impractical to mill beyond 50 microm in depth. In addition, for samples frozen in HPF carriers, preparation requires an undercut, the removal of material below the extracted volume, to detach the extraction volume from the bulk. In combination, these factors generally result in maximum extraction volumes of approximately 20 microm x 20 microm x 10-30 microm (length x width x height, Figure 1 - Figure Supplement 1b).\nIn order to extract larger volumes, we performed lift-out on an HPF 'waffle'-type sample. HPF 'waffle' samples are prepared by freezing sample on a grid that is sandwiched between HPF carriers (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The final thickness is therefore defined by the type of grid and spacer being used during freezing. For a 25 microm thick specimen, extraction volumes of up to 200 microm x 40 microm x 25 microm can easily be obtained by performing lift-out with the sample surface oriented perpendicular to the ion beam (Figure 1 - Figure Supplement 1c). The same orientation is used during trench milling and will be referred to as 'trench milling orientation' (Figure 1 - Figure Supplement 3c,f). Such large volumes extracted at the trench milling orientation can yield many sections from a single lift-out which can be subsequently thinned to lamellae for cryo-ET data acquisition (Figure 1b,c &lt;&lt;FIG:fig1&gt;&gt;).\n\nApplication of Serial Lift-Out to C. elegans L1 larvae\nTo assess the feasibility of obtaining serial lamellae from a single lift-out transfer, we performed the procedure on C. elegans L1 larvae that had been vitrified using a modified protocol of the 'waffle' method (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The sample contained many L1 larvae embedded in an approximately 25 microm thick layer of ice (Figure 2 - Figure Supplement 1). Two sites from two different grids were selected for preparation by correlating cryo-fluorescence light microscopy with SEM and FIB view images. Following the geometry described in Figure 1 &lt;&lt;FIG:fig1&gt;&gt; - Supplement 1c, the sites were prepared by milling trenches around the larva, yielding a 30 microm x 110 microm x 25 microm and a 40 microm x 180 microm x 25 microm extraction volume (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3a). Two different attachment strategies were explored, with resulting lamellae either being attached on one side (single-sided attachment) or two sides (double-sided attachment). To extract the volumes, the needle with the copper adapter (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3b) was attached to the sample using redeposition: copper material redepositing onto the surface of the extraction volume during milling. The extraction volume was subsequently milled free from the bulk material and lifted out.\nFigure 2:\nA workflow for double-sided attachment Serial Lift-Out.\na, FIB image of the prepared extraction site with overlaid correlated fluorescence data (green) indicating the larva being targeted. The micromanipulator is attached to the extraction volume by redeposition from the copper adapter (trench milling orientation). b, The extracted volume (orange dashed line) is lowered into position between two grid bars in lamella milling orientation. The lower front edge of the volume (yellow arrows) is aligned to the pre-milled line mark (black arrows). c, Double-sided attachment by redeposition from the grid bars (yellow arrows indicate direction of milling), followed by line pattern milling releasing the section of a desired thickness (dashed yellow line). Orange dashed line indicates the outline of the extracted volume. d, SEM image of a typical section after being released from the extracted volume. Black dashed line indicates the outline of the worm cross-section. e,f, SEM (e) and TEM (f) overview images of the 40 double-sided attached sections obtained. Figure 2 - Movie Supplement 1 summarizes the process.\nThe volume was then transferred to the receiver grid. For single-sided attachment, a modified grid based on customizing a standard 100 mesh copper grid (Figure 2 - Figure Supplement 2) was used. Removing every second row of grid bars yielded an array of trimmed grid bars that resemble the pins of a standard half-moon shaped lift-out grid (Figure 2 - Figure Supplement 2e). Alternatively, a copper grid with rectangular meshes can be used for double-sided attachment of volumes roughly as wide as the mesh (~40 microm for the 400/100 rectangular mesh grids used here, Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Double- sided attachment has the added benefit of increased section stability by eliminating the free-standing side (Figure 2 - Figure Supplement 3f).\nThe volume's lower front edge was precisely aligned to the front edge of the attachment pin (Figure 2 - Figure Supplement 3c) or a previously prepared alignment line pattern milled onto the receiver grid (Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Then, redeposition milling was used to attach the lower part of the volume to the grid bar(s) (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow arrows). After attachment, the lower part of the target material was separated from the extracted volume using a line pattern (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow dashed line), leaving a ~4 microm thick section (Figure 2d &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3e,f). Once sectioned, the remaining volume attached to the lift-out needle was transferred to the next attachment site. This procedure was iterated until no material remained. As a result, many sections were produced from a single cryo-lift-out transfer (Figure 2e-f &lt;&lt;FIG:fig2&gt;&gt;). To assess the general applicability of the approach to non-'waffle' type samples, an additional experiment was performed on D. melanogaster egg chambers, high-pressure frozen in standard sample carriers. From a ~20 microm long and ~15 microm deep extracted volume we produced sections of ~1-2 microm thickness (Figure 2 - Figure Supplement 4).\n\nSampling organismal cellular anatomy along the C. elegans L1 larva at molecular resolution\nWe prepared dozens of sections along the anterior-posterior axis of C. elegans L1 larvae: 12 single-sided attached and 40 double-sided attached lamellae (Figure 2 - Figure Supplement 3g, Figure 2e &lt;&lt;FIG:fig2&gt;&gt;). After transfer to the TEM, 8 out of 12 and 32 out of 40 lamellae were recovered, respectively (Figure 3 - Figure Supplement 1a, Figure 3 - Figure Supplement 2a). Lamella loss during transfer is common due to manual grid handling steps. The increased rate of successfully transferred lamellae from 66% for single-sided to 80% for double-sided attachment is indicative of the increased lamella stability of double-sided attachment.\nFigure 3:\nLamella TEM overviews sample the anatomy of a C. elegans L1 larva along the anterior-posterior body axis.\nNative tissue scattering contrast is sufficient to extract a considerable amount of anatomical information from low magnification TEM overviews of the lamellae generated in the double-sided attachment Serial Lift-Out experiment. a, Schematic representation of 29 body transverse section overviews obtained from the final lamellae along the anterior-posterior body axis of an L1 larva. Anterior is located to the left, posterior to the right. The OpenWorm project model of the adult C. elegans worm was used for illustrating the possibility of back mapping since a cellular model of the L1 larva only exists for its head and not the entire body (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;). The cross-sections were cropped from lamella overview images and mapped back to a position derived from the known sectioning distance and anatomical features discernible in the corresponding cross-section. The dashed white frames indicate overviews with corresponding magnified representations in panels b-g. b, This lamella originates from ~15 microm along the anterior-posterior axis. Clearly visible are the three lobes of the anterior pharyngeal lumen in the center of the worm cross-section (asterisk) and the relatively electron dense pharyngeal lining. c, Overview from the anterior part of the pharyngeal isthmus ~42 microm along the anterior-posterior axis. Note the nerve ring (dashed line) surrounding the central pharynx. Additionally, the alae (asterisks) running along the left and right lateral side of the worm become obvious. d, Overview of a lamella of roughly the center of the posterior pharyngeal bulb region. The central grinder organ is clearly discernible (asterisk). This section is positioned ~65 microm along the anterior-posterior axis. e, A section roughly mid-body. The intestinal lumen (dashed line) and intestinal cells (dotted line) are obvious. The darker cell slightly left of the body center is likely one of the gonadal primordial cells (asterisk). The section is from ~115 microm along the anterior-posterior axis. f, In this mid-body section, the intestinal lumen (dashed line) can again be clearly discerned. The section can be mapped to ~132 microm along the anterior-posterior axis. g, Section showing the intestinal lumen at ~155 microm along the anterior-posterior axis.\nFrom these successfully transferred lamellae, overview maps were recorded (approximately 20 microm x 25 microm in size, Figure 3 - Figure Supplement 1, Figure 3 - Figure Supplement 2) depicting clear larval cross-sections. These overviews allowed the identification and assignment of anatomical structures and tissues such as the pharynx, the body-wall muscle cells, neurons, the hypodermis and seam cells. Larger organelles such as nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, bundles of actin filaments and microtubules were clearly discernible (Figure 3b-g &lt;&lt;FIG:fig3&gt;&gt;, Figure3 - Movie Supplement 1). Given the stereotypic body plan of C. elegans, the information obtained from the overviews together with the sectioning thickness was used to determine the approximate location of the sections in the worm, as schematically shown in Figure 3a &lt;&lt;FIG:fig3&gt;&gt;.\nThe biological area within each lamella can, due to the circular nature of cross sections, be estimated by ?r2, where the average radius is around 5 microm, resulting in a mean of 78.5 microm2 imageable area per lamella. For the single-sided attachment experiment, the total imageable area was ~ 630 microm2 and allowed for the collection of 57 tilt series, each with a field of view of ~1.2 microm in size. The double-sided attachment experiment yielded ~2500 microm2 imageable area from which a total of 1012 tilt series with a field of view of ~750 nm in size were collected. The double-sided attachment experiment allowed us to sample a large fraction of all tissues and cell types along the anterior-posterior body axis of the C. elegans L1 larva.\nTo illustrate the information level already present at intermediate magnification (11,500x), we manually segmented a representative cross-section (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). The segmentation illustrates that the cuticle clearly delineates the body cross-section. Body-wall muscle cells are obvious due to their pronounced distal actomyosin pattern. The pharyngeal lumen's trilobal structure demarcates the body center and is surrounded by the three alternating pharyngeal muscle and marginal cells. Pharyngeal neurons and gland cell process cross-sections are embedded in the pharyngeal muscle cells. Nuclei are easy to discern due to their ribosome-decorated double-membrane and their denser, less granular interior. Neurons in general can be discerned by their appearance as round or tubular cells, often grouped in bundles. These clearly interpretable overviews allow targeting of tissue-specific cellular structures or cellular protein complexes with known location (e.g. sarcomeric proteins).\nFigure 4:\nRepresentative overview and tomograms of the L1 larval pharyngeal isthmus region.\na, Overview montage (magnification 11,500x) of the head region lamella. This section is located within the pharyngeal isthmus just posterior to the nerve ring. Cell types are colored according to the WormAtlas color code (&lt;&lt;REF:Altun-2002-?-?&gt;&gt;), for nuclei and mitochondria, arbitrary colors were chosen (HYP: hypodermis, S: seam cells, PM: pharyngeal muscle, MC: marginal cells, EXP: excretory pore, EXC: excretory canal, N: neuronal tissue, CUT: cuticle, n: nucleus. The white dashed rectangles show the positions of the tilt series acquired and corresponding reconstructions shown in b-d. Asterisks indicate ice contamination. The cross indicates the dorsal(D)-ventral(V) and left(L)-right(R) body axes. b, Tomographic slice of the perinuclear region of most likely a neuronal cell. The nucleus exhibits a different granularity and density than the cytoplasm (Cyto), from which it is separated by the nuclear envelope, which in turn is heavily decorated with ribosomes and contains nuclear pores (arrows). c, Tomographic slice of the pharynx. Central in the upper half is a marginal cell containing endoplasmatic reticulum (ER) and mitochondria (Mito). A neighboring neuronal cell body (CB) is separated from the pharynx by a diffuse density, which is the pharyngeal basal lamina (BL). d, Tomographic slice of a body wall muscle cross section (BWM). Clearly visible are the actin filaments in top view (arrowhead) surrounding a bundle of thick filaments (arrow). The thick filaments are partially interspersed with actin filaments. The muscle cell neighbors a hypodermal cell (HYP), from which it is separated by a space filled with a diffuse density, likely the body-wall muscle basal lamina. The next and last layer of the larval body wall is the cuticle (CUT). Most notably within this chitinous/collagenous structure, a fence-like array of denser structures can be discerned, likely cuticular collagen.\n\nQuality assessment and subtomogram analysis\nTo assess the quality of lamella thinning, tomogram thickness was measured for all 57 tomograms from the single-sided attachment and a random subset of 132 tomograms for the double-sided attachment experiment. The thickness for single-sided attachment was 253 nm +/- 125 nm. The thickness of the tomograms from the double-sided attachment experiment was more uniform, 303 +/- 40 nm (Figure 4 - Figure Supplement 1). The broader thickness distribution in single-sided attachment most likely stems from lamella bending and movement during milling due to the free-standing edge of the lamella.\nThe tilt series we obtained allowed us to reconstruct the 3D cellular architecture of cell types in different tissues such as the neuronal nuclear periphery (Figure 4b &lt;&lt;FIG:fig4&gt;&gt;), pharyngeal marginal cells and neuronal cell bodies (Figure 4c &lt;&lt;FIG:fig4&gt;&gt;) and body wall muscle, the hypodermis and the collagenous network of the cuticle (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nThe sectioning plane obtained when doing Serial Lift-Out from 'waffle'-type samples provides worm cross-sections. For a number of structures, such as nuclei, the difference between oblique longitudinal and transverse sectioning is minimal (Figure 4a &lt;&lt;FIG:fig4&gt;&gt;, Figure 4 - Figure Supplement 2). When investigating anisotropic structures such as the body wall muscle or pharynx, however, sectioning direction can greatly impact the interpretation of higher order structure. In longitudinal sectioned sarcomeres, located in body-wall muscle, appear as filaments running along the image plane. These filaments show an orientational distribution strongly biased towards the side view. In contrast, tomograms from Serial Lift-Out show transverse sections of the body-wall muscle. In this sectioning plane, actin and myosin run often perpendicular to the image plane and can be discerned as small and medium sized puncta. This more clearly reveals the organization of the acto-myosin bundling and holds the potential for further analysis of their packing (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nIn order to assess the quality of the data acquired, we picked 33,000 80S ribosome particles from 200 randomly selected tomograms from the double-sided attachment experiment. Subtomogram averaging and classification yielded a structure at a resolution of 6.9 A (GSFSC, Figure 5a &lt;&lt;FIG:fig5&gt;&gt;). The resolution was likely limited by lamella thickness, supported by the fact that the CTF could not be fit beyond 6 A.\nFigure 5:\nSubtomogram average reconstruction of the C. elegans 80S ribosome from in situ data.\na, The C. elegans ribosome to a resolution of 6.9 A (GSFSC). Density map is colored by local resolution. Note that protein alpha-helices and rRNA helices are clearly visible at this resolution. b, Four different ribosomal states obtained through subtomogram classification: Ribosome Class 1 with occupied A, P and EF sites, Ribosome Class 2 with an occupied P site, Ribosome Class 3 with occupied A and P sites, Ribosome Class with occupied A and EF sites.\nClassification yielded four different sub-populations in various translational states. When comparing these states to the recently published ribosome state landscape from D. discoideum [8], resemblances are apparent to the initiation state with an occupied P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 2), states with an occupied A and P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 3) and elongation factor bound states (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 1, Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 4).\n\nDiscussion\nIn the past, cryo-lift-out has been hampered by limited throughput and low overall success rate. Therefore, the technique has even been deemed 'practically impossible', or at least 'merely difficult' in the literature (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). The fact that the existing literature has not advanced beyond proof-of-principle experiments underlines this evaluation. The study of multicellular organisms and tissues by cryo-electron tomography, however, holds an enormous potential for biological discovery and technical advances are thus needed. We anticipate the combination of recent hardware and workflow improvements (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;, &lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) with the increased throughput of Serial Lift-Out to make cryo-ET data acquisition from high-pressure frozen material more attainable. In addition, plasma focused ion beam technology, while available for some years, was recently introduced to the cryo-FIB community (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Martynowycz-2023-14-1086&gt;&gt;, &lt;&lt;REF:Sergey-2018-?-?&gt;&gt;) and may further improve cryo-lift-out throughput by increased ablation rates.\nThe cryo-FIB lamella milling protocols developed to date remove most of the cell during lamella preparation. The final lamella represents only &lt;1 % of a eukaryotic cell and even less for multicellular organisms. While a high number of lamellae milled at different heights could restore that lost information through an ensemble average (&lt;&lt;REF:Ferguson-2017-147-653&gt;&gt;), Serial Lift-Out lamellae originate from a single specimen, yielding a more thorough characterization of its morphology. A variety of volume microscopy methods, e.g. volume electron microscopy or X-Ray tomography, retain this information. They remain, however, limited in resolution when compared to cryo-ET. Serial Lift-Out increases the contextual information retained in cryo-ET while preserving data acquisition at molecular resolution.\nHere, we have shown the ability to section in increments of one to four micrometers. This process has two steps that contribute to specimen loss: sectioning (~300-500 nm) and lamella milling. The latter can be minimized by finer sectioning. Preliminary experiments suggest that thinner sectioning down to 500 nm may be achievable. Reducing the section thickness even further in order to prepare lamellae directly from the extracted volume, though, will likely require major technological advances of the FIB instrument. Developments in ion beam shaping could become valuable for such endeavors, reducing the material lost during sectioning. Due to the physical basis of ablation in FIB preparation, however, material will always be lost to the milling process itself. Therefore, truly consecutive lamellae, similar to serial sections of plastic embedded samples, seem unlikely to be attainable.\nNevertheless, the creation of multiple sections within biological material increases the preparation throughput for HPF samples and adds contextual information. For model organisms with a stereotypic and well-described body plan such as C. elegans or Platynereis dumerilii (&lt;&lt;REF:Vergara-2021-184-4819&gt;&gt;), Serial Lift-Out sections and tomograms may be mapped back into context using other sources of volumetric data as illustrated in Figure 3a &lt;&lt;FIG:fig3&gt;&gt; (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;, &lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;). As a result, back-mapping analysis may enable label-free targeting of features and events that are tissue and cell-type specific.\nSerial Lift-Out also addresses the challenge that arises when dealing with anisotropic cells and tissues. As the sectioning plane of the specimen can be adapted, fluorescence or FIB/SEM data can inform the preparation of the lift-out site and, in turn, the sectioning angle. This adaptive preparation strategy can give new insights into the molecular architecture as illustrated by tomograms from transverse sections of muscle cells. When compared to previously obtained longitudinal sections (&lt;&lt;REF:Burbaum-2021-12-4086&gt;&gt;, &lt;&lt;REF:Davide-2023-?-?&gt;&gt;, &lt;&lt;REF:Wang-2021-184-2135&gt;&gt;), the transverse section shows actin-myosin packing from a novel angle revealing how actin filaments surround what is likely to be myosin heads. The combination of acquiring cryo-ET data on both the transverse and longitudinal sections also hold the potential to improve subtomogram analysis when facing structures with preferential orientation.\nIn addition to guiding site preparation for lift-out, cryo-correlative light and electron microscopy is more generally used to target specific subcellular events (&lt;&lt;REF:Arnold-2016-110-860&gt;&gt;, &lt;&lt;REF:Bieber-2022-119-e2209823119&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;). This technique, however, remains challenging for routine use within larger tissues. Improvements in the operation of integrated light microscopes are therefore necessary to streamline subcellular targeting in cryo-lift-out experiments. Serial Lift-Out combined with in-chamber light microscopy could increase the success rate of targeting by reducing the sample thickness used in the correlation, increasing the number of sections and in turn targeting attempts, and enabling to regularly check the fluorescence signal throughout the milling process.\nOne limitation of cryo-lift-out is the high rate of ice contamination during transfer. As can be seen in the TEM overviews (Figure 3 - Figure Supplement 1), large ice crystals tend to obstruct regions of the lamellae and prevent data acquisition. Serial Lift-Out, similar to on-grid preparations, compensates for the loss in imageable area through a higher yield of lamellae in comparison to previous lift-out techniques. Reducing transfer contamination, however, remains highly desirable and controlled environments, e.g. glove boxes (&lt;&lt;REF:Tacke-2021-213-107743&gt;&gt;), or other technological advances such as vacuum transfers will be needed to maximize the imageable area of cryo-FIB lamellae.\nFinally, the analysis of lamella thickness distributions and lamella survival rate suggest that double-sided attachment in cryo-lift-out is advantageous. This like likely due to the reduction of lamella bending during milling and greater stability during the transfer to the TEM.\nWith the methodological advances of Serial Lift-Out, the existing hurdles of lift-out have been greatly diminished, enabling data quality, throughput, and success rate in cryo-lift-out that permits the mapping of large tissue regions and whole organisms at the molecular level. Tomography on lamellae of an L1 larva obtained with our Serial Lift-Out method elucidated its ribosome to 6.9 A resolution and uncovered four different translational states. Taken together, Serial Lift-Out demonstrates enormous potential to discern and study the molecular anatomy of native tissues and small multicellular organisms.\n\nSupporting information\nExtended_Data_File\nFigure2_MovieSupplement1\nFigure2_MovieSupplement2\nFigure3_MovieSupplement1\n\nCompeting interests\nJMP holds a position on the advisory board of Thermo Fisher Scientific. CT is an employee of Thermo Fisher Scientific. The other authors declare that no competing interests exist.\n\nData availability\nTomograms have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes: EMD-17246, EMD-17247, EMD-17248, Subtomogram averages have been uploaded under accession numbers: EMD-17241, EMD-17242, EMD-17243, EMD-17244, EMD-17245 and will be released upon publication.\n\nMaterials and Methods\n\nSample Vitrification\nC. elegans strains AM140 (allele rmIs132[unc-54p::Q35::YFP]) and NK2476 (allele qy46[ina-1::mKate+loxP]) were cultivated according to standard methods on rich NGM (&lt;&lt;REF:Brenner-1974-77-71&gt;&gt;). To obtain a synchronous population of animals, gravid young adult worms were washed from five petri dishes (6 cm) with M9 medium. The worms were pooled into 15 mL centrifuge tubes and centrifuged at 175x g. Excess supernatant was removed and the worm suspension was mixed with bleach solution (3.75 mL 1M NaOH, 3.0 mL household bleach, 8.25 mL H2O (&lt;&lt;REF:Sulston-1988-?-587&gt;&gt;) and swiftly vortexed. Bleaching was continued with intermittent vortexing and checking on a stereomicroscope. As soon as roughly 50% of worms appeared broken, the bleaching procedure was stopped by the addition of egg-buffer (118 mM NaCl, 48 mM KCl, 2 mM CaCl2, 2 mM MgCl2, 25 mM HEPES, pH 7.3) and centrifugation at 175g for 1 minute. The supernatant was swiftly removed and the egg/worm carcass solution was washed another four times with egg-buffer. The egg/worm carcass mixture was then floated on 60 % sucrose solution and purified eggs were withdrawn from the surface (&lt;&lt;REF:Strange-2007-2-1003&gt;&gt;). The egg solution was washed in M9 buffer and larvae were allowed to hatch at 20 degreesC for 24 hours.\nThis population of developmentally arrested L1 larvae was used for vitrification, carried out following a modified version of Mahamid, et al. (&lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;) on a Leica EM-ICE (Leica Microsystems, Wetzlar, Germany). Sample carriers Type B (Leica Microsystems, Wetzlar, Germany) with a diameter of 6 mm were coated with a separating layer of cetyl palmitate solution (0.5% w/v in diethylether) by dipping the carriers briefly into the solution and placing them, cavity side down, on filter papers to let the solvent evaporate. A 2 microL drop of 20% (v/v) Ficoll 400 im M9 was applied to the flat side of the sample carrier and a Formvar-coated grid (50 mesh square or 75 mesh hexagonal mesh) was floated on this drop with the support film facing the liquid. Any excess cryoprotectant below the grid was wicked away, using pieces of Whatman No 1 filter paper (Whatman, Maidstone, UK). Next, synchronized L1 larvae were mixed with an equal amount of 40% Ficoll 400 in M9 medium to reach a final concentration of 20% cryoprotectant. 2microL of this sample solution was applied onto the grid and the sandwich was completed by addition of a second 6 mm sample carrier Type B, cavity-side up. The sample was immediately high-pressure-frozen. Grids were removed from the HPF sample carriers and stored in LN2 until further use. EM-grids were clipped into Thermo Fisher Scientific cartridges for FIB-milling. To be able to locate the biological material (L1 larvae) in the ice layer, clipped grids were either previously mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.\nDrosophila samples were prepared as previously described (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, fly strains were maintained at 22 degreesC on standard cornmeal agar. 24 hours prior to dissection of egg chambers, the flies were transferred into a new vial supplemented with yeast paste. Egg chambers were dissected in Schneider's medium. For high-pressure freezing, 3 mm HPF carriers were soaked in hexadecene and blotted on filter paper. Depending on the egg chamber stage, the dissected material was subsequently transferred into the 100 microm or 150 microm cavity of a 3 mm Type A or Type C HPF sample carrier (Engineering office M. Wohlwend, Sennwald, Switzerland), respectively. The filling medium was 20% Ficoll (70 kDa) in Schneider's medium. The flat surface of a Type B HPF Planchette was used to close the HPF assembly for freezing in the Leica EM ICE high-pressure freezer (Leica Microsystems, Wetzlar, Germany). The sample was subsequently pre-trimmed using a 45degrees diamond knife (Diatome, Nidau, Switzerland) in a cryo-microtome (EM UC6/FC6 cryo-microtome, Leica Microsystems) at a temperature of -170 degreesC and used for further FIB preparation.\n\nCryo-fluorescence microscopy\nGrids were clipped in Thermo Fisher Scientific (TFS) cartridges and imaged using a Leica TCS SP8 laser confocal microscope (Leica Microsystems, Wetzlar, Germany) fitted with a Leica cryo-stage (&lt;&lt;REF:Schorb-2017-197-83&gt;&gt;). Imaging was performed with a 50x 0.90 NA objective and HyD detectors.\nTile set montages of the NK2476 strain sample were collected using a pinhole size of 4.85 AU and a voxel size of 0.578 microm x 0.578 microm x 2.998 microm. The tile sets were merged and maximum intensity projections for each channel calculated using LAS X (Leica, Wetzlar, Germany). Autofluorescence imaging was performed for the double-sided attachment experiment. The 488 nm laser line at 2.5 % total power was used for excitation and the emission was detected from 501 nm to 535 nm. The reflection channel was excited at 552 nm and 2% total laser power and detected with an emission range of 547 nm to 557 nm.\nTile set montages of the AM140 strain were collected at a pinhole size of 1 AU and a voxel size of 0.578 microm x 0.578 microm x 1.03 microm. The tile sets were merged and maximum intensity projections for each channel were calculated using LAS X (Leica, Wetzlar, Germany). Q35-YFP was excited at 488 nm and 0.5 % total laser power and emission was detected from 509 nm to 551 nm. Reflection was recorded with 638 nm excitation at 0.83 % total laser power and the emission range set to 633 nm to 643 nm.\n\nSerial Lift Out - Single Side Attachment\n\nLift-In Grid Preparation\n100 square mesh Cu grids (Agar Scientific, Stansted, UK) were assembled into TFS cartridges. The cartridges were marked for manual alignment during sample loading. These markings were aligned to where the grid bars intersect the cartridge marks. The marks were also used for grid orientation during loading. The grids were then loaded into a FIB-SEM 45degrees pre-tilt cryo-shuttle such that the grid bars are aligned vertically and horizontally.\nOnce loaded, 14-pin grids were prepared by rotating the grid plane to be normal to the ion beam (trench milling orientation) and milling out two combs of horizontal grid bars. This yielded grids with 14 pins that can be used for 28 lamella attachments (Figure 2 - Figure Supplement 2). A horizontal grid bar was used to prepare 20 microm x 10 microm x 5 microm copper blocks for attachment to the lift-out needle by redeposition milling using single-pass regular cross-sections directed away from the needle (Figure 1 - Figure Supplement 2, Figure 2 - Figure Supplement 5a). Note, that the copper block attached to the needle can be re-used several times. The lower section of the copper block used for the previous attachment can be milled away leaving a clean surface for the next attachment.\nAs the FIB-SEM does not have to be cooled during lift-in grid milling, we recommend the preparation of several lift-in grids ahead of time in order to reduce workload during the serial lift-out session.\n\nLift-Out, Sectioning and Milling Procedure\nSerial lift-out was performed on an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. To facilitate lift-out, the EasyLift needle was modified by attaching a copper block adapter (see 'lift-in grid preparation') using redeposition. The copper adapter has a higher sputter yield and, thus, increased redeposition compared to tungsten and vitreous ice. Additionally, the block increases the surface area available for attachment to the biology, and reduces the wear of the needle over time.\nAfter copper block attachment, the volume to be extracted was prepared by milling trenches around the region of interest in trench milling orientation. The fluorescence data acquired of the L1 'waffle' samples was correlated to the ion beam images using MAPS version 3.14 (Thermo Fisher Scientific, Waltham, MA, U.S.A.). The grid was coated with a protective metal-organic platinum layer using a gas injection system (GIS) heated to 27degreesC at a stage working distance of 9.72 mm and a stage tilt of 45degrees for 90s. Trenches were milled with the grid perpendicular to the ion beam (trench milling orientation) after which the EasyLift needle was inserted. In order to increase the redeposition yield for attachment, the copper block needs to be aligned below the sample surface and flush to the surface exposed by trench milling. Redeposition from the copper block onto the extraction volume was achieved using single-pass patterns of regular cross-sections instead of the default multi-pass mode to avoid re-milling previously redeposited material. These patterns were placed at the interface of the two surfaces on the copper adapter with the milling direction for these patterns directed away from the extraction volume (Figure 2 - Figure Supplement 5b). The extraction volume was then released through milling the last connection to the bulk sample with a line pattern. The 30 microm x 110 microm x 25 microm extracted volume was subsequently lifted from the bulk sample.\nAfter the extraction of the volume, the shuttle was returned to the lamella milling orientation (18degrees in a system with a 45degrees pre-tilt) and the stage translated to the receiver grid. The lower edge of the extracted volume and the corner of the pin were aligned in both electron and ion beam images. Attachment was performed by redeposition, using single-pass regular cross-section patterns directed away from the extracted volume (Figure 2 - Figure Supplement 5d). Once attached, the EasyLift system was maneuvered by a 50 nm step horizontally and vertically away from the pin in horizontal direction in order to create a small amount of strain. Subsequently, the lower 4 microm of the volume were sectioned from the rest of the volume using a line pattern. This process was iterated until the entire extracted volume had been sectioned.\n\nSerial Lift-Out: Double-Sided Attachment\n\nLift-In Grid Preparation\nFor double-sided attachment, 100/400 rectangular mesh copper grids (Gilder, Grantham, UK) were used as the receiver grid. These were clipped into standard TFS cartridges that marked such that the 400 mesh bars were in line with the bottom mark and the 100 mesh bars were in line with the side markings. This grid was loaded into the FIB shuttle and 20 microm x 10 microm x 5 microm copper block adapters were prepared as described above.\nDuring the preparation of the receiver grid, roughly 5 microm deep line patterns were milled across the horizontal 400 mesh grid bars in trench milling orientation to divide the rectangles approximately (1/3) and (2/3). These marker lines are intended to guide the alignment of the extracted volume by providing a reference visible in both the electron and ion beams during the attachment process.\n\nLift Out, Sectioning and Milling Procedure\nLift-out site preparation for double-sided attachment was performed on an Aquilos 1 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with a METEOR in-chamber fluorescence light microscope (Delmic, Delft, Netherlands)(&lt;&lt;REF:Smeets-2021-29-20&gt;&gt;). The the GIS was employed for 90 s at 27degreesC with at a stage working distance set to 10.6 mm and the grid normal to the GIS needle to apply a layer of protective metal-organic platinum. The METEOR in-chamber microscope was used in conjunction with previously acquired cryo-confocal data to localize the larva of interest and produce an extraction volume of 42 microm x 180 microm x 20 microm. A width of 40 microm is necessary in order to span the space between the grid bars of a 100/400 rectangular mesh grid. A slight excess of width of the extraction volume is preferable, since a too narrow block will not allow for double-sided attachment, while material can be ablated during sectioning.\nThe lift-out and receiver grid were subsequently transferred into an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. Lift-out was otherwise performed as stated in the section for single-sided attachment. In brief, the volume was attached to the needle via the copper adapter, the volume was released, the needle retracted, and the shuttle was returned to the lamella milling orientation at the receiver grid. The extracted volume was re-inserted, and its lower edge was positioned in-between two grid bars. In the case when the volume was slightly too wide, material was ablated from its sides as necessary. To allow for proper attachment, the volume's lower edge was aligned to the reference line on the receiver grid in both, the electron and ion beam images and the volume was attached on both sides by redeposition from the grid bars. To achieve this, single-pass cross-section patterns were placed on the grid bars above the marker lines in in close proximity of the interface between the grid bars and extracted volume, while the milling was directed away from the volume (Figure 2 - Figure Supplement 5c). Once attached, the EasyLift system was maneuvered up in the z-direction by 50-100 nm in order to create a small amount of strain. Subsequently, a section was released from the extracted volume by milling a line pattern at 4 microm above the volume's lower edge. This process was iterated until the entire extracted volume had been sectioned.\nMilling pattern parameters for Serial Lift-Out.\nThe table summarizes the parameters used for the milling steps necessary to perform a Serial Lift-Out experiment. The corresponding pattern geometries are illustrated in Figure 2 - Figure Supplement 5. All patterns were milled at 30.00 kV FIB acceleration voltage. Pattern files for Thermo Fisher Scientific instruments are provided in Figure 2 &lt;&lt;FIG:fig2&gt;&gt; - Pattern Files Supplement. These templates and the milling parameters may need adjustment for different projects and for FIB-SEM instruments from other manufacturers, specifically concerning differing scanning strategies deployed by the microscope manufacturer.\n\nFine milling of lift-out sections\nFor both single-sided and double-sided attachment, the initial 4 microm sections were thinned in two steps: rough milling and fine milling. Rough milling was performed using regular cross-section patterns at a beam current of 1 nA to a thickness of 1.5 microm, 0.5 nA to 1.2 microm lamella thickness, 0.3 nA to 0.8 microm lamella thickness. After rough milling all of the sections, fine milling was performed using regular rectangle patterns at a beam current of 0.1 nA to 0.4 microm and 50 pA to final thickness. Over-tilting and under-tilting by up to 1degrees was used for beam convergence compensation. The double-sided attached lamellae were sputter coated with platinum after fine milling for 4 seconds, at a chamber pressure of 0.20 mbar and a current of 15.0 mA.\n\nSerial lift-out from high-pressure frozen D. melanogaster egg chambers\nLift-out experiments of high-pressure frozen D. melanogaster egg chambers were performed on a Scios FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift needle. The preparation of the lift-out volume as described previously (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, an approximately 20 microm x 20 microm volume of target material was prepared using regular cross-section patterns in trench milling orientation in a horseshoe-like shape. The trenches were ~20 microm wide, except for the region that needed to be accessed by the lift-out needle, which was ~35 microm wide. The extraction volume was undercut at a stage tilt of 45degrees, or the highest tilt reachable for the specific position. This preparation leaves the extraction volume attached to the bulk material on a single side. Subsequently, the procedure for single-sided attachment was performed as described above. Sectioning was performed in increments of 1-3 microm using a line pattern milled at a beam current of at 0.3 nA. After section preparation, fine milling was performed by sequentially decreasing the beam current: 0.3 nA to 800 nm, 0.1 nA to 500 nm, 50 pA to 350 nm and 30 pA to &lt; 300 nm. The final step was judged by the loss of contrast in SEM imaging at 3 kV acceleration voltage and a beam current of 13 pA.\n\nLamella preparation by the 'waffle' milling method\nFor preparation of lamellae directly on the high-pressure frozen grid, vitrified as described above, we followed a similar workflow as published as the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The grid was coated with a metal-organic platinum layer by GIS deposition for 20 s at a stage working distance of 10.6 mm. Milling was performed on a gallium FIB-SEM Aquilos 2 (Thermo Fisher Scientific, Waltham, MA, U.S.A.) instrument. Initial trenches were milled in trench milling orientation at a beam current of 3 nA using regular cross-section patterns. To avoid damaging the region of interest by milling at high beam currents, 2 microm of buffer distance were left around the region of interest. Trenches were extended to 15 microm on the backside of the subsequent lamella and 30 microm on the front side. In a next step, the trenches were extended to 1 microm closer to the region of interest at a beam current of 1 nA and in a last step the final dimension of the lamella region was defined milling at 0.5 nA. After trench milling, another layer of protective metal-organic platinum was deposited on the sample (20 s deposition time, 10.6 mm stage working distance, three consecutive times). The stage was rotated into lamella milling orientation and a notch was milled with line patterns at 0.3 nA as previously described (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The preparation of the final lamella was carried out by removing material above and below of the region of interest at a beam current of 0.3 nA, until a final lamella thickness of 2 microm was reached. In sequential steps of decreasing beam current (to 0.8 microm thickness at 0.1 nA beam current, to 0.4 microm at 50 pA), the remnant material was ablated. The lamella was polished to a final thickness of about 200-250 nm at 30 pA beam current.\n\nTEM Data Acquisition\nTEM data acquisition was performed on a Titan Krios G4 at 300 kV equipped with a Selectris X energy filter and a Falcon 4i camera (Thermo Fisher Scientific, Eindhoven, The Netherlands). Lamella overview montages were recorded by stage-driven tiling at 11,500x nominal magnification (pixel size 2.127 nm). Tomograms were recorded using the Tomo5 software package version 5.12.0 (Thermo Fisher Scientific, Eindhoven, The Netherlands) using the EER file format.\nTwo acquisition strategies were deployed. For the tomograms of the single-sided attachment and 'waffle' preparation tilt series were acquired at a nominal magnification of 42,000x resulting in a pixel size at the sample of 2.93 A. A dose-symmetric tilt scheme was used with an angular increment of 2degrees, a dose of 2 e-/A2 per tilt and a target defocus of -4 to -5.5 microm. Tilt series were collected in a tilt range of -70degrees to 50degrees due to the lamellae pre-tilt and with a total dose of 120 e-/A2.\nFor the tomograms collected of the double-sided attachment lamellae, that were subsequently used for subtomogram analysis, tilt series were acquired at a nominal magnification of 64,000x, resulting in a pixel size of 1.89 A. Data was collected using a dose-symmetric tilt scheme with an angular increment of 3degrees, a dose of 3.23 e-/A2 per tilt and a target defocus range of -1 to -4 microm. Angles of -70degrees to 50degrees were acquired, resulting in a total dose of 132 e-/A2.\n\nTomogram Reconstruction, Visualization and Subtomogram Analysis\nData was processed using the Tomoman version 0.7 pipeline (https://github.com/williamnwan/TOMOMAN). 14 frames with a dose of 0.23 e-/A2 per frame were rendered from the EER files. These were used for motion correction in MotionCor2 version 1.4.7 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;) and CTF estimation with CTFFIND4 version 4.14 (&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;). Bad tilts were removed after manual inspection using the Tomoman script. Dose-weighting was performed at 3.23 e-/A2 per tilt using either Tomoman or Warp. For denoising, tilt series were separated into odd and even tilts during motion correction and the resulting stacks were processed using Cryo-CARE (&lt;&lt;REF:Buchholz-2019-?-502&gt;&gt;). Tomogram reconstructions for visualization were done in IMOD version 4.12.32. Tilt series were aligned with AreTomo version 1.3.3. CTF-corrected tomograms for template matching were reconstructed in IMOD version 4.12.32 (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) at eight times binning, resulting in a pixel size of 15.6 A.\nInitial template matching was performed in STOPGAP version 0.7 (&lt;&lt;REF:Wan-2020-26-2516&gt;&gt;) on a subset of 70 tomograms at bin8 using PDB 4V4B as a reference filtered to 35 A (&lt;&lt;REF:Spahn-2004-23-1008&gt;&gt;). 16,420 particles were extracted and aligned in STOPGAP to generate the C. elegans 80S ribosome template. The template was subsequently used to repeat template matching on 200 tomograms at bin8. 65,451 particles were extracted and cleaned by projecting the subtomogram along the Z-axis and subsequent 2D classification in Relion version 4.0. The remaining data set contained 37,026 particles. The retained particles were reprocessed in Warp version 1.0.9, cleaned to remove particles with inadequate CTF resolution and astigmatism. The resulting 35,350 particles were extracted with a pixel size of 2.98 A and a boxsize of 160 pixels. Relion version 3.0 with a spherical mask of 340 A radius was used to align the subtomograms. Finally, the particles were imported into M version 1.0.9 and geometric and CTF parameters were sequentially refined. Corrected subtomograms were extracted from M and classified in Relion 3.0. Two rounds of classification with a spherical mask of 340 A in diameter resulted in 8,256 particles being removed. The second step was a focused classification with a spherical mask around the A/P/E-site. The resulting 5 classes were combined into 2 classes according to the small subunit rotation. Each of these 2 merged classes were subjected to another round of 3D classification with a mask around the elongation factor binding site. The final classes were manually pooled according to structural similarity yielding 5 classes with 2,175, 20,638, 877, 1,119, and 2,285 particles, of which the class with 877 particles was neglected. Segmentation of the elongation factor and tRNAs was done with the Segger tool in Chimera. ChimeraX version 1.3 was used for visualization (&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;).'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nSingle particle analysis (SPA) by cryo-transmission electron microscopy (cryo-TEM) has become a key technique to study the structure of isolated biological macromolecules at high-resolution (&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;). While SPA routinely reaches resolutions where protein side-chains can be fitted unambiguously, the reductionist approach of studying protein complexes in vitro loses all information concerning their molecular sociology: the interaction of the molecular complexes in their natural environment (&lt;&lt;REF:Beck-2016-26-825&gt;&gt;). Conversely, in situ cryo-electron tomography (cryo-ET) allows for the reconstruction of pleomorphic structures such as the crowded environment of the cell at molecular resolution, maintaining the interaction and localization of protein complexes within the biological system (&lt;&lt;REF:Dietrich-2022-607-823&gt;&gt;, &lt;&lt;REF:Gupta-2021-184-3643&gt;&gt;, &lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;, &lt;&lt;REF:Plitzko-2017-46-110&gt;&gt;, &lt;&lt;REF:Watanabe-2020-182-1508&gt;&gt;).\nOne of the primary factors limiting the resolution of cryo-TEM is inelastic scattering. As the mean free path of an electron in vitrified biological samples is about 300-400 nm, samples beyond the size of viruses and small prokaryotic cells are generally too thick for cryo-ET (&lt;&lt;REF:Liedtke-2022-6-100065&gt;&gt;, &lt;&lt;REF:O’Reilly-2020-369-554&gt;&gt;). In consequence, two main sample thinning methods to obtain electron-transparent specimens for cryo-ET have been developed: cryo-ultramicrotomy and cryo-FIB milling. Cryo-ultramicrotomy encompasses the thin-sectioning of vitreous cells and tissues with a diamond knife (&lt;&lt;REF:Al-Amoudi-2004-148-131&gt;&gt;). The shearing forces at the knife's edge, however, cause mechanical artifacts such as crevices and compression in the resulting sections (&lt;&lt;REF:Al-Amoudi-2003-9-372&gt;&gt;). More recently, the focused ion beam (FIB) instrument has been widely adopted for sample thinning at cryogenic temperatures. While not completely damage-free (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Lucas-2023-?-?&gt;&gt;), the technique bypasses the mechanical artifacts of cryo-ultramicrotomy (&lt;&lt;REF:Marko-2007-4-215&gt;&gt;, &lt;&lt;REF:Rigort-2012-109-4449&gt;&gt;) and has been shown to yield data that can allow for the elucidation of ribosomes to side-chain resolution (&lt;&lt;REF:Hoffmann-2022-13-7435&gt;&gt;). The automation of lamella preparation by cryo-FIB milling has also reduced the need for user expertise and manual intervention (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Zachs-2020-9-?&gt;&gt;).\nPrior to thinning, the sample must be cryogenically fixed by cooling at a sufficiently high rate to prevent ice crystal formation, resulting in a vitrified sample. There are two main methods available for vitrification: plunge freezing, in which the sample is immersed at ambient pressure into liquid ethane or ethane-propane mixture (&lt;&lt;REF:Tivol-2008-14-375&gt;&gt;), and high pressure freezing (HPF), in which the sample is cooled with a jet of liquid nitrogen at a pressure of ~2000 bar. While the former yields samples that are easily FIB-milled, the sample thickness that can reliably be vitrified is limited to roughly 10 microm. HPF, on the other hand, allows for the vitrification of samples up to a thickness of roughly 200 microm (&lt;&lt;REF:Dubochet-1995-5-366&gt;&gt;).\nConsequently, HPF greatly expands the size range of biological samples that can be vitrified but comes at the cost of embedding the specimen in a thick layer of ice defined by the depth of the freezing receptacle. This increased sample thickness leads to longer milling times. While samples up to a thickness of about 50 microm can be prepared by milling lamellae directly on the grid following the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;), lamellae from thicker samples have, to date, only been prepared by cryo-lift-out (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Kuba-2021-281-112&gt;&gt;, &lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nCryo-lift-out refers to the extraction of the material for lamella preparation from bulk HPF sample and subsequent transfer and attachment to a lift-out receiver grid, conventionally a half-moon shaped grid (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;). Two main types of micromanipulator devices are currently available for cryo-lift-out: sharp needles and a cryo-gripper (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;). Initially, trenches are milled around the area of interest leaving it connected to the bulk material on a single side. For specimens in HPF sample carriers, the material must additionally be cleared from below. After these preparatory steps, the lift-out device is brought in contact with the volume to be extracted. The remaining connection to the bulk material is removed, the micromanipulator is used to transfer the extracted volume and redeposition milling (&lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) is used to attach the volume to the receiver grid. Finally, an electron transparent lamella is prepared (&lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;).\nWhile widely used in materials science at room temperature (&lt;&lt;REF:Giannuzzi-2005-?-201&gt;&gt;), cryo-lift-out of biological samples has remained primarily proof-of-concept (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;, &lt;&lt;REF:Rubino-2012-180-572&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;). This is due to a number of factors, e.g. the need for cooled micromanipulator devices and accompanying workflow adaptations required resulting in limited throughput and problems with lamella loss during transfer to the TEM (&lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;).\nAnother limitation to cryo-lift-out, as well as on-grid lamella preparation is the loss of contextual information. Only a fraction of the sample volume (~1% for larger eukaryotic cells, &lt; 1% for multicellular specimens) ends up inside the final lamella for cryo-ET data acquisition. Techniques exist that are capable of capturing larger volumetric data and tracking morphology at comparatively large volume scales. Examples of such techniques are X-ray tomography and various volume electron microscopy (vEM) techniques: serial FIB milling and scanning electron microscopy, and serial sectioning of plastic embedded samples imaged by TEM or STEM. These techniques, however, currently cannot achieve the resolution attainable by cryo-ET at an equivalent sample preservation state due to physical limitations in imaging or the necessity of fixation and contrasting (&lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;, &lt;&lt;REF:Spehner-2020-211-107528&gt;&gt;).\nIn this work, we describe a novel cryo-lift-out approach that creates a series of lamellae from one lift-out volume that we term Serial Lift-Out. Inspired by diamond knife serial sectioning, Serial Lift-Out retains more contextual information than previous procedures and increases the throughput of cryo-lift-out by an order of magnitude. It sets the stage for the study of multicellular organisms and tissues by cryo-ET - applications that previously seemed practically impossible (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). We demonstrate Serial Lift-Out on high-pressure frozen Caenorhabditis elegans L1 stage larvae, sampling their ultrastructure along the anterior-posterior axis by cryo-electron tomography. From the resulting dataset, we reconstruct the nematode's ribosome to 7 A resolution by subtomogram averaging, exemplifying the enormous potential of Serial Lift-Out for the study of the molecular anatomy of multicellular systems.\n\nResults\n\nThe concept behind Serial Lift-Out\nThe most time-consuming steps in cryo-lift-out are the preparation of the extraction sites and the transfer of the extracted volume to the receiving grid. Each repetition of the lift-out cycle therefore adds significant time investment with low yield. Scaling up the extracted volume and producing multiple lamellae from a single lift-out bypasses this repetitive, time-consuming trench milling and transfer, increases throughput and, foremost, provides more contextual information about the targeted material.\nA specific implementation of this concept is the extraction of an entire L1 larva of C. elegans, followed by repeated steps of attachment to the receiving grid, sectioning and transfer of the remaining volume to the next attachment site (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;). While material is still lost during sectioning and thinning, a large fraction of the worm is made readily accessible to cryo-TEM data acquisition.\nFigure 1:\nA schematic of Serial Lift-Out.\nAn illustration of the Serial Lift-Out method, exemplified by the process being performed on a C. elegans L1 larva embedded in an extracted volume of vitrified ice (blue). a, The extracted volume containing the larva. b, c, Schematic representation of the resulting Serial Lift-Out sections and lamellae, respectively. d, Illustration of the double-sided attachment Serial Lift-Out procedure. The extracted volume shown in panel a is attached to the lift-out needle via a copper block adaptor and transferred to a rectangular mesh receiver grid. Several sections are shown, obtained by the repetition of the attachment of the bottom part of the volume to the grid bars and subsequent sectioning.\nPrevious lift-out approaches extracted volumes approximately the size of the final lamella (Figure 1 - Figure Supplement 1a), with some excess material for stability during transfer and attachment. In order to section multiple lamellae from a single cryo-lift-out transfer, the volume extracted by cryo-lift-out needed to be increased. To ease the manipulation of such large volumes, we introduced a copper block that acts as an adapter between the needle and the extracted volume. This copper block is created from the receiver grid and attached to the needle before lift-out (Figure 1 - Figure Supplement 2, Figure 1 - Figure Supplement 3). As copper has higher redeposition rates than the tungsten needle, the copper block adapter results in significantly more resilient \"welding\" of the specimen to the micromanipulator.\nAnother volume limiting factor is the ablation rate of the ion beam. Most cryo-FIB machines are equipped with a gallium ion source rendering it impractical to mill beyond 50 microm in depth. In addition, for samples frozen in HPF carriers, preparation requires an undercut, the removal of material below the extracted volume, to detach the extraction volume from the bulk. In combination, these factors generally result in maximum extraction volumes of approximately 20 microm x 20 microm x 10-30 microm (length x width x height, Figure 1 - Figure Supplement 1b).\nIn order to extract larger volumes, we performed lift-out on an HPF 'waffle'-type sample. HPF 'waffle' samples are prepared by freezing sample on a grid that is sandwiched between HPF carriers (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The final thickness is therefore defined by the type of grid and spacer being used during freezing. For a 25 microm thick specimen, extraction volumes of up to 200 microm x 40 microm x 25 microm can easily be obtained by performing lift-out with the sample surface oriented perpendicular to the ion beam (Figure 1 - Figure Supplement 1c). The same orientation is used during trench milling and will be referred to as 'trench milling orientation' (Figure 1 - Figure Supplement 3c,f). Such large volumes extracted at the trench milling orientation can yield many sections from a single lift-out which can be subsequently thinned to lamellae for cryo-ET data acquisition (Figure 1b,c &lt;&lt;FIG:fig1&gt;&gt;).\n\nApplication of Serial Lift-Out to C. elegans L1 larvae\nTo assess the feasibility of obtaining serial lamellae from a single lift-out transfer, we performed the procedure on C. elegans L1 larvae that had been vitrified using a modified protocol of the 'waffle' method (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The sample contained many L1 larvae embedded in an approximately 25 microm thick layer of ice (Figure 2 - Figure Supplement 1). Two sites from two different grids were selected for preparation by correlating cryo-fluorescence light microscopy with SEM and FIB view images. Following the geometry described in Figure 1 &lt;&lt;FIG:fig1&gt;&gt; - Supplement 1c, the sites were prepared by milling trenches around the larva, yielding a 30 microm x 110 microm x 25 microm and a 40 microm x 180 microm x 25 microm extraction volume (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3a). Two different attachment strategies were explored, with resulting lamellae either being attached on one side (single-sided attachment) or two sides (double-sided attachment). To extract the volumes, the needle with the copper adapter (Figure 2a &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3b) was attached to the sample using redeposition: copper material redepositing onto the surface of the extraction volume during milling. The extraction volume was subsequently milled free from the bulk material and lifted out.\nFigure 2:\nA workflow for double-sided attachment Serial Lift-Out.\na, FIB image of the prepared extraction site with overlaid correlated fluorescence data (green) indicating the larva being targeted. The micromanipulator is attached to the extraction volume by redeposition from the copper adapter (trench milling orientation). b, The extracted volume (orange dashed line) is lowered into position between two grid bars in lamella milling orientation. The lower front edge of the volume (yellow arrows) is aligned to the pre-milled line mark (black arrows). c, Double-sided attachment by redeposition from the grid bars (yellow arrows indicate direction of milling), followed by line pattern milling releasing the section of a desired thickness (dashed yellow line). Orange dashed line indicates the outline of the extracted volume. d, SEM image of a typical section after being released from the extracted volume. Black dashed line indicates the outline of the worm cross-section. e,f, SEM (e) and TEM (f) overview images of the 40 double-sided attached sections obtained. Figure 2 - Movie Supplement 1 summarizes the process.\nThe volume was then transferred to the receiver grid. For single-sided attachment, a modified grid based on customizing a standard 100 mesh copper grid (Figure 2 - Figure Supplement 2) was used. Removing every second row of grid bars yielded an array of trimmed grid bars that resemble the pins of a standard half-moon shaped lift-out grid (Figure 2 - Figure Supplement 2e). Alternatively, a copper grid with rectangular meshes can be used for double-sided attachment of volumes roughly as wide as the mesh (~40 microm for the 400/100 rectangular mesh grids used here, Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Double- sided attachment has the added benefit of increased section stability by eliminating the free-standing side (Figure 2 - Figure Supplement 3f).\nThe volume's lower front edge was precisely aligned to the front edge of the attachment pin (Figure 2 - Figure Supplement 3c) or a previously prepared alignment line pattern milled onto the receiver grid (Figure 2b &lt;&lt;FIG:fig2&gt;&gt;). Then, redeposition milling was used to attach the lower part of the volume to the grid bar(s) (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow arrows). After attachment, the lower part of the target material was separated from the extracted volume using a line pattern (Figure 2c &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3d, yellow dashed line), leaving a ~4 microm thick section (Figure 2d &lt;&lt;FIG:fig2&gt;&gt;, Figure 2 - Figure Supplement 3e,f). Once sectioned, the remaining volume attached to the lift-out needle was transferred to the next attachment site. This procedure was iterated until no material remained. As a result, many sections were produced from a single cryo-lift-out transfer (Figure 2e-f &lt;&lt;FIG:fig2&gt;&gt;). To assess the general applicability of the approach to non-'waffle' type samples, an additional experiment was performed on D. melanogaster egg chambers, high-pressure frozen in standard sample carriers. From a ~20 microm long and ~15 microm deep extracted volume we produced sections of ~1-2 microm thickness (Figure 2 - Figure Supplement 4).\n\nSampling organismal cellular anatomy along the C. elegans L1 larva at molecular resolution\nWe prepared dozens of sections along the anterior-posterior axis of C. elegans L1 larvae: 12 single-sided attached and 40 double-sided attached lamellae (Figure 2 - Figure Supplement 3g, Figure 2e &lt;&lt;FIG:fig2&gt;&gt;). After transfer to the TEM, 8 out of 12 and 32 out of 40 lamellae were recovered, respectively (Figure 3 - Figure Supplement 1a, Figure 3 - Figure Supplement 2a). Lamella loss during transfer is common due to manual grid handling steps. The increased rate of successfully transferred lamellae from 66% for single-sided to 80% for double-sided attachment is indicative of the increased lamella stability of double-sided attachment.\nFigure 3:\nLamella TEM overviews sample the anatomy of a C. elegans L1 larva along the anterior-posterior body axis.\nNative tissue scattering contrast is sufficient to extract a considerable amount of anatomical information from low magnification TEM overviews of the lamellae generated in the double-sided attachment Serial Lift-Out experiment. a, Schematic representation of 29 body transverse section overviews obtained from the final lamellae along the anterior-posterior body axis of an L1 larva. Anterior is located to the left, posterior to the right. The OpenWorm project model of the adult C. elegans worm was used for illustrating the possibility of back mapping since a cellular model of the L1 larva only exists for its head and not the entire body (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;). The cross-sections were cropped from lamella overview images and mapped back to a position derived from the known sectioning distance and anatomical features discernible in the corresponding cross-section. The dashed white frames indicate overviews with corresponding magnified representations in panels b-g. b, This lamella originates from ~15 microm along the anterior-posterior axis. Clearly visible are the three lobes of the anterior pharyngeal lumen in the center of the worm cross-section (asterisk) and the relatively electron dense pharyngeal lining. c, Overview from the anterior part of the pharyngeal isthmus ~42 microm along the anterior-posterior axis. Note the nerve ring (dashed line) surrounding the central pharynx. Additionally, the alae (asterisks) running along the left and right lateral side of the worm become obvious. d, Overview of a lamella of roughly the center of the posterior pharyngeal bulb region. The central grinder organ is clearly discernible (asterisk). This section is positioned ~65 microm along the anterior-posterior axis. e, A section roughly mid-body. The intestinal lumen (dashed line) and intestinal cells (dotted line) are obvious. The darker cell slightly left of the body center is likely one of the gonadal primordial cells (asterisk). The section is from ~115 microm along the anterior-posterior axis. f, In this mid-body section, the intestinal lumen (dashed line) can again be clearly discerned. The section can be mapped to ~132 microm along the anterior-posterior axis. g, Section showing the intestinal lumen at ~155 microm along the anterior-posterior axis.\nFrom these successfully transferred lamellae, overview maps were recorded (approximately 20 microm x 25 microm in size, Figure 3 - Figure Supplement 1, Figure 3 - Figure Supplement 2) depicting clear larval cross-sections. These overviews allowed the identification and assignment of anatomical structures and tissues such as the pharynx, the body-wall muscle cells, neurons, the hypodermis and seam cells. Larger organelles such as nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, bundles of actin filaments and microtubules were clearly discernible (Figure 3b-g &lt;&lt;FIG:fig3&gt;&gt;, Figure3 - Movie Supplement 1). Given the stereotypic body plan of C. elegans, the information obtained from the overviews together with the sectioning thickness was used to determine the approximate location of the sections in the worm, as schematically shown in Figure 3a &lt;&lt;FIG:fig3&gt;&gt;.\nThe biological area within each lamella can, due to the circular nature of cross sections, be estimated by ?r2, where the average radius is around 5 microm, resulting in a mean of 78.5 microm2 imageable area per lamella. For the single-sided attachment experiment, the total imageable area was ~ 630 microm2 and allowed for the collection of 57 tilt series, each with a field of view of ~1.2 microm in size. The double-sided attachment experiment yielded ~2500 microm2 imageable area from which a total of 1012 tilt series with a field of view of ~750 nm in size were collected. The double-sided attachment experiment allowed us to sample a large fraction of all tissues and cell types along the anterior-posterior body axis of the C. elegans L1 larva.\nTo illustrate the information level already present at intermediate magnification (11,500x), we manually segmented a representative cross-section (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). The segmentation illustrates that the cuticle clearly delineates the body cross-section. Body-wall muscle cells are obvious due to their pronounced distal actomyosin pattern. The pharyngeal lumen's trilobal structure demarcates the body center and is surrounded by the three alternating pharyngeal muscle and marginal cells. Pharyngeal neurons and gland cell process cross-sections are embedded in the pharyngeal muscle cells. Nuclei are easy to discern due to their ribosome-decorated double-membrane and their denser, less granular interior. Neurons in general can be discerned by their appearance as round or tubular cells, often grouped in bundles. These clearly interpretable overviews allow targeting of tissue-specific cellular structures or cellular protein complexes with known location (e.g. sarcomeric proteins).\nFigure 4:\nRepresentative overview and tomograms of the L1 larval pharyngeal isthmus region.\na, Overview montage (magnification 11,500x) of the head region lamella. This section is located within the pharyngeal isthmus just posterior to the nerve ring. Cell types are colored according to the WormAtlas color code (&lt;&lt;REF:Altun-2002-?-?&gt;&gt;), for nuclei and mitochondria, arbitrary colors were chosen (HYP: hypodermis, S: seam cells, PM: pharyngeal muscle, MC: marginal cells, EXP: excretory pore, EXC: excretory canal, N: neuronal tissue, CUT: cuticle, n: nucleus. The white dashed rectangles show the positions of the tilt series acquired and corresponding reconstructions shown in b-d. Asterisks indicate ice contamination. The cross indicates the dorsal(D)-ventral(V) and left(L)-right(R) body axes. b, Tomographic slice of the perinuclear region of most likely a neuronal cell. The nucleus exhibits a different granularity and density than the cytoplasm (Cyto), from which it is separated by the nuclear envelope, which in turn is heavily decorated with ribosomes and contains nuclear pores (arrows). c, Tomographic slice of the pharynx. Central in the upper half is a marginal cell containing endoplasmatic reticulum (ER) and mitochondria (Mito). A neighboring neuronal cell body (CB) is separated from the pharynx by a diffuse density, which is the pharyngeal basal lamina (BL). d, Tomographic slice of a body wall muscle cross section (BWM). Clearly visible are the actin filaments in top view (arrowhead) surrounding a bundle of thick filaments (arrow). The thick filaments are partially interspersed with actin filaments. The muscle cell neighbors a hypodermal cell (HYP), from which it is separated by a space filled with a diffuse density, likely the body-wall muscle basal lamina. The next and last layer of the larval body wall is the cuticle (CUT). Most notably within this chitinous/collagenous structure, a fence-like array of denser structures can be discerned, likely cuticular collagen.\n\nQuality assessment and subtomogram analysis\nTo assess the quality of lamella thinning, tomogram thickness was measured for all 57 tomograms from the single-sided attachment and a random subset of 132 tomograms for the double-sided attachment experiment. The thickness for single-sided attachment was 253 nm +/- 125 nm. The thickness of the tomograms from the double-sided attachment experiment was more uniform, 303 +/- 40 nm (Figure 4 - Figure Supplement 1). The broader thickness distribution in single-sided attachment most likely stems from lamella bending and movement during milling due to the free-standing edge of the lamella.\nThe tilt series we obtained allowed us to reconstruct the 3D cellular architecture of cell types in different tissues such as the neuronal nuclear periphery (Figure 4b &lt;&lt;FIG:fig4&gt;&gt;), pharyngeal marginal cells and neuronal cell bodies (Figure 4c &lt;&lt;FIG:fig4&gt;&gt;) and body wall muscle, the hypodermis and the collagenous network of the cuticle (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nThe sectioning plane obtained when doing Serial Lift-Out from 'waffle'-type samples provides worm cross-sections. For a number of structures, such as nuclei, the difference between oblique longitudinal and transverse sectioning is minimal (Figure 4a &lt;&lt;FIG:fig4&gt;&gt;, Figure 4 - Figure Supplement 2). When investigating anisotropic structures such as the body wall muscle or pharynx, however, sectioning direction can greatly impact the interpretation of higher order structure. In longitudinal sectioned sarcomeres, located in body-wall muscle, appear as filaments running along the image plane. These filaments show an orientational distribution strongly biased towards the side view. In contrast, tomograms from Serial Lift-Out show transverse sections of the body-wall muscle. In this sectioning plane, actin and myosin run often perpendicular to the image plane and can be discerned as small and medium sized puncta. This more clearly reveals the organization of the acto-myosin bundling and holds the potential for further analysis of their packing (Figure 4d &lt;&lt;FIG:fig4&gt;&gt;).\nIn order to assess the quality of the data acquired, we picked 33,000 80S ribosome particles from 200 randomly selected tomograms from the double-sided attachment experiment. Subtomogram averaging and classification yielded a structure at a resolution of 6.9 A (GSFSC, Figure 5a &lt;&lt;FIG:fig5&gt;&gt;). The resolution was likely limited by lamella thickness, supported by the fact that the CTF could not be fit beyond 6 A.\nFigure 5:\nSubtomogram average reconstruction of the C. elegans 80S ribosome from in situ data.\na, The C. elegans ribosome to a resolution of 6.9 A (GSFSC). Density map is colored by local resolution. Note that protein alpha-helices and rRNA helices are clearly visible at this resolution. b, Four different ribosomal states obtained through subtomogram classification: Ribosome Class 1 with occupied A, P and EF sites, Ribosome Class 2 with an occupied P site, Ribosome Class 3 with occupied A and P sites, Ribosome Class with occupied A and EF sites.\nClassification yielded four different sub-populations in various translational states. When comparing these states to the recently published ribosome state landscape from D. discoideum [8], resemblances are apparent to the initiation state with an occupied P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 2), states with an occupied A and P site (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 3) and elongation factor bound states (Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 1, Figure 5b &lt;&lt;FIG:fig5&gt;&gt; - Class 4).\n\nDiscussion\nIn the past, cryo-lift-out has been hampered by limited throughput and low overall success rate. Therefore, the technique has even been deemed 'practically impossible', or at least 'merely difficult' in the literature (&lt;&lt;REF:Parmenter-2016-79-298&gt;&gt;). The fact that the existing literature has not advanced beyond proof-of-principle experiments underlines this evaluation. The study of multicellular organisms and tissues by cryo-electron tomography, however, holds an enormous potential for biological discovery and technical advances are thus needed. We anticipate the combination of recent hardware and workflow improvements (&lt;&lt;REF:Klumpe-2022-30-42&gt;&gt;, &lt;&lt;REF:Parmenter-2021-281-157&gt;&gt;, &lt;&lt;REF:Schreiber-2018-194-89&gt;&gt;) with the increased throughput of Serial Lift-Out to make cryo-ET data acquisition from high-pressure frozen material more attainable. In addition, plasma focused ion beam technology, while available for some years, was recently introduced to the cryo-FIB community (&lt;&lt;REF:Berger-2023-14-629&gt;&gt;, &lt;&lt;REF:Martynowycz-2023-14-1086&gt;&gt;, &lt;&lt;REF:Sergey-2018-?-?&gt;&gt;) and may further improve cryo-lift-out throughput by increased ablation rates.\nThe cryo-FIB lamella milling protocols developed to date remove most of the cell during lamella preparation. The final lamella represents only &lt;1 % of a eukaryotic cell and even less for multicellular organisms. While a high number of lamellae milled at different heights could restore that lost information through an ensemble average (&lt;&lt;REF:Ferguson-2017-147-653&gt;&gt;), Serial Lift-Out lamellae originate from a single specimen, yielding a more thorough characterization of its morphology. A variety of volume microscopy methods, e.g. volume electron microscopy or X-Ray tomography, retain this information. They remain, however, limited in resolution when compared to cryo-ET. Serial Lift-Out increases the contextual information retained in cryo-ET while preserving data acquisition at molecular resolution.\nHere, we have shown the ability to section in increments of one to four micrometers. This process has two steps that contribute to specimen loss: sectioning (~300-500 nm) and lamella milling. The latter can be minimized by finer sectioning. Preliminary experiments suggest that thinner sectioning down to 500 nm may be achievable. Reducing the section thickness even further in order to prepare lamellae directly from the extracted volume, though, will likely require major technological advances of the FIB instrument. Developments in ion beam shaping could become valuable for such endeavors, reducing the material lost during sectioning. Due to the physical basis of ablation in FIB preparation, however, material will always be lost to the milling process itself. Therefore, truly consecutive lamellae, similar to serial sections of plastic embedded samples, seem unlikely to be attainable.\nNevertheless, the creation of multiple sections within biological material increases the preparation throughput for HPF samples and adds contextual information. For model organisms with a stereotypic and well-described body plan such as C. elegans or Platynereis dumerilii (&lt;&lt;REF:Vergara-2021-184-4819&gt;&gt;), Serial Lift-Out sections and tomograms may be mapped back into context using other sources of volumetric data as illustrated in Figure 3a &lt;&lt;FIG:fig3&gt;&gt; (&lt;&lt;REF:Britz-2021-?-?&gt;&gt;, &lt;&lt;REF:Dumoux-2023-12-?&gt;&gt;). As a result, back-mapping analysis may enable label-free targeting of features and events that are tissue and cell-type specific.\nSerial Lift-Out also addresses the challenge that arises when dealing with anisotropic cells and tissues. As the sectioning plane of the specimen can be adapted, fluorescence or FIB/SEM data can inform the preparation of the lift-out site and, in turn, the sectioning angle. This adaptive preparation strategy can give new insights into the molecular architecture as illustrated by tomograms from transverse sections of muscle cells. When compared to previously obtained longitudinal sections (&lt;&lt;REF:Burbaum-2021-12-4086&gt;&gt;, &lt;&lt;REF:Davide-2023-?-?&gt;&gt;, &lt;&lt;REF:Wang-2021-184-2135&gt;&gt;), the transverse section shows actin-myosin packing from a novel angle revealing how actin filaments surround what is likely to be myosin heads. The combination of acquiring cryo-ET data on both the transverse and longitudinal sections also hold the potential to improve subtomogram analysis when facing structures with preferential orientation.\nIn addition to guiding site preparation for lift-out, cryo-correlative light and electron microscopy is more generally used to target specific subcellular events (&lt;&lt;REF:Arnold-2016-110-860&gt;&gt;, &lt;&lt;REF:Bieber-2022-119-e2209823119&gt;&gt;, &lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;, &lt;&lt;REF:Schaffer-2019-16-757&gt;&gt;). This technique, however, remains challenging for routine use within larger tissues. Improvements in the operation of integrated light microscopes are therefore necessary to streamline subcellular targeting in cryo-lift-out experiments. Serial Lift-Out combined with in-chamber light microscopy could increase the success rate of targeting by reducing the sample thickness used in the correlation, increasing the number of sections and in turn targeting attempts, and enabling to regularly check the fluorescence signal throughout the milling process.\nOne limitation of cryo-lift-out is the high rate of ice contamination during transfer. As can be seen in the TEM overviews (Figure 3 - Figure Supplement 1), large ice crystals tend to obstruct regions of the lamellae and prevent data acquisition. Serial Lift-Out, similar to on-grid preparations, compensates for the loss in imageable area through a higher yield of lamellae in comparison to previous lift-out techniques. Reducing transfer contamination, however, remains highly desirable and controlled environments, e.g. glove boxes (&lt;&lt;REF:Tacke-2021-213-107743&gt;&gt;), or other technological advances such as vacuum transfers will be needed to maximize the imageable area of cryo-FIB lamellae.\nFinally, the analysis of lamella thickness distributions and lamella survival rate suggest that double-sided attachment in cryo-lift-out is advantageous. This like likely due to the reduction of lamella bending during milling and greater stability during the transfer to the TEM.\nWith the methodological advances of Serial Lift-Out, the existing hurdles of lift-out have been greatly diminished, enabling data quality, throughput, and success rate in cryo-lift-out that permits the mapping of large tissue regions and whole organisms at the molecular level. Tomography on lamellae of an L1 larva obtained with our Serial Lift-Out method elucidated its ribosome to 6.9 A resolution and uncovered four different translational states. Taken together, Serial Lift-Out demonstrates enormous potential to discern and study the molecular anatomy of native tissues and small multicellular organisms.\n\nSupporting information\nExtended_Data_File\nFigure2_MovieSupplement1\nFigure2_MovieSupplement2\nFigure3_MovieSupplement1\n\nCompeting interests\nJMP holds a position on the advisory board of Thermo Fisher Scientific. CT is an employee of Thermo Fisher Scientific. The other authors declare that no competing interests exist.\n\nData availability\nTomograms have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes: EMD-17246, EMD-17247, EMD-17248, Subtomogram averages have been uploaded under accession numbers: EMD-17241, EMD-17242, EMD-17243, EMD-17244, EMD-17245 and will be released upon publication.\n\nMaterials and Methods\n\nSample Vitrification\nC. elegans strains AM140 (allele rmIs132[unc-54p::Q35::YFP]) and NK2476 (allele qy46[ina-1::mKate+loxP]) were cultivated according to standard methods on rich NGM (&lt;&lt;REF:Brenner-1974-77-71&gt;&gt;). To obtain a synchronous population of animals, gravid young adult worms were washed from five petri dishes (6 cm) with M9 medium. The worms were pooled into 15 mL centrifuge tubes and centrifuged at 175x g. Excess supernatant was removed and the worm suspension was mixed with bleach solution (3.75 mL 1M NaOH, 3.0 mL household bleach, 8.25 mL H2O (&lt;&lt;REF:Sulston-1988-?-587&gt;&gt;) and swiftly vortexed. Bleaching was continued with intermittent vortexing and checking on a stereomicroscope. As soon as roughly 50% of worms appeared broken, the bleaching procedure was stopped by the addition of egg-buffer (118 mM NaCl, 48 mM KCl, 2 mM CaCl2, 2 mM MgCl2, 25 mM HEPES, pH 7.3) and centrifugation at 175g for 1 minute. The supernatant was swiftly removed and the egg/worm carcass solution was washed another four times with egg-buffer. The egg/worm carcass mixture was then floated on 60 % sucrose solution and purified eggs were withdrawn from the surface (&lt;&lt;REF:Strange-2007-2-1003&gt;&gt;). The egg solution was washed in M9 buffer and larvae were allowed to hatch at 20 degreesC for 24 hours.\nThis population of developmentally arrested L1 larvae was used for vitrification, carried out following a modified version of Mahamid, et al. (&lt;&lt;REF:Mahamid-2015-192-262&gt;&gt;) on a Leica EM-ICE (Leica Microsystems, Wetzlar, Germany). Sample carriers Type B (Leica Microsystems, Wetzlar, Germany) with a diameter of 6 mm were coated with a separating layer of cetyl palmitate solution (0.5% w/v in diethylether) by dipping the carriers briefly into the solution and placing them, cavity side down, on filter papers to let the solvent evaporate. A 2 microL drop of 20% (v/v) Ficoll 400 im M9 was applied to the flat side of the sample carrier and a Formvar-coated grid (50 mesh square or 75 mesh hexagonal mesh) was floated on this drop with the support film facing the liquid. Any excess cryoprotectant below the grid was wicked away, using pieces of Whatman No 1 filter paper (Whatman, Maidstone, UK). Next, synchronized L1 larvae were mixed with an equal amount of 40% Ficoll 400 in M9 medium to reach a final concentration of 20% cryoprotectant. 2microL of this sample solution was applied onto the grid and the sandwich was completed by addition of a second 6 mm sample carrier Type B, cavity-side up. The sample was immediately high-pressure-frozen. Grids were removed from the HPF sample carriers and stored in LN2 until further use. EM-grids were clipped into Thermo Fisher Scientific cartridges for FIB-milling. To be able to locate the biological material (L1 larvae) in the ice layer, clipped grids were either previously mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.\nDrosophila samples were prepared as previously described (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, fly strains were maintained at 22 degreesC on standard cornmeal agar. 24 hours prior to dissection of egg chambers, the flies were transferred into a new vial supplemented with yeast paste. Egg chambers were dissected in Schneider's medium. For high-pressure freezing, 3 mm HPF carriers were soaked in hexadecene and blotted on filter paper. Depending on the egg chamber stage, the dissected material was subsequently transferred into the 100 microm or 150 microm cavity of a 3 mm Type A or Type C HPF sample carrier (Engineering office M. Wohlwend, Sennwald, Switzerland), respectively. The filling medium was 20% Ficoll (70 kDa) in Schneider's medium. The flat surface of a Type B HPF Planchette was used to close the HPF assembly for freezing in the Leica EM ICE high-pressure freezer (Leica Microsystems, Wetzlar, Germany). The sample was subsequently pre-trimmed using a 45degrees diamond knife (Diatome, Nidau, Switzerland) in a cryo-microtome (EM UC6/FC6 cryo-microtome, Leica Microsystems) at a temperature of -170 degreesC and used for further FIB preparation.\n\nCryo-fluorescence microscopy\nGrids were clipped in Thermo Fisher Scientific (TFS) cartridges and imaged using a Leica TCS SP8 laser confocal microscope (Leica Microsystems, Wetzlar, Germany) fitted with a Leica cryo-stage (&lt;&lt;REF:Schorb-2017-197-83&gt;&gt;). Imaging was performed with a 50x 0.90 NA objective and HyD detectors.\nTile set montages of the NK2476 strain sample were collected using a pinhole size of 4.85 AU and a voxel size of 0.578 microm x 0.578 microm x 2.998 microm. The tile sets were merged and maximum intensity projections for each channel calculated using LAS X (Leica, Wetzlar, Germany). Autofluorescence imaging was performed for the double-sided attachment experiment. The 488 nm laser line at 2.5 % total power was used for excitation and the emission was detected from 501 nm to 535 nm. The reflection channel was excited at 552 nm and 2% total laser power and detected with an emission range of 547 nm to 557 nm.\nTile set montages of the AM140 strain were collected at a pinhole size of 1 AU and a voxel size of 0.578 microm x 0.578 microm x 1.03 microm. The tile sets were merged and maximum intensity projections for each channel were calculated using LAS X (Leica, Wetzlar, Germany). Q35-YFP was excited at 488 nm and 0.5 % total laser power and emission was detected from 509 nm to 551 nm. Reflection was recorded with 638 nm excitation at 0.83 % total laser power and the emission range set to 633 nm to 643 nm.\n\nSerial Lift Out - Single Side Attachment\n\nLift-In Grid Preparation\n100 square mesh Cu grids (Agar Scientific, Stansted, UK) were assembled into TFS cartridges. The cartridges were marked for manual alignment during sample loading. These markings were aligned to where the grid bars intersect the cartridge marks. The marks were also used for grid orientation during loading. The grids were then loaded into a FIB-SEM 45degrees pre-tilt cryo-shuttle such that the grid bars are aligned vertically and horizontally.\nOnce loaded, 14-pin grids were prepared by rotating the grid plane to be normal to the ion beam (trench milling orientation) and milling out two combs of horizontal grid bars. This yielded grids with 14 pins that can be used for 28 lamella attachments (Figure 2 - Figure Supplement 2). A horizontal grid bar was used to prepare 20 microm x 10 microm x 5 microm copper blocks for attachment to the lift-out needle by redeposition milling using single-pass regular cross-sections directed away from the needle (Figure 1 - Figure Supplement 2, Figure 2 - Figure Supplement 5a). Note, that the copper block attached to the needle can be re-used several times. The lower section of the copper block used for the previous attachment can be milled away leaving a clean surface for the next attachment.\nAs the FIB-SEM does not have to be cooled during lift-in grid milling, we recommend the preparation of several lift-in grids ahead of time in order to reduce workload during the serial lift-out session.\n\nLift-Out, Sectioning and Milling Procedure\nSerial lift-out was performed on an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. To facilitate lift-out, the EasyLift needle was modified by attaching a copper block adapter (see 'lift-in grid preparation') using redeposition. The copper adapter has a higher sputter yield and, thus, increased redeposition compared to tungsten and vitreous ice. Additionally, the block increases the surface area available for attachment to the biology, and reduces the wear of the needle over time.\nAfter copper block attachment, the volume to be extracted was prepared by milling trenches around the region of interest in trench milling orientation. The fluorescence data acquired of the L1 'waffle' samples was correlated to the ion beam images using MAPS version 3.14 (Thermo Fisher Scientific, Waltham, MA, U.S.A.). The grid was coated with a protective metal-organic platinum layer using a gas injection system (GIS) heated to 27degreesC at a stage working distance of 9.72 mm and a stage tilt of 45degrees for 90s. Trenches were milled with the grid perpendicular to the ion beam (trench milling orientation) after which the EasyLift needle was inserted. In order to increase the redeposition yield for attachment, the copper block needs to be aligned below the sample surface and flush to the surface exposed by trench milling. Redeposition from the copper block onto the extraction volume was achieved using single-pass patterns of regular cross-sections instead of the default multi-pass mode to avoid re-milling previously redeposited material. These patterns were placed at the interface of the two surfaces on the copper adapter with the milling direction for these patterns directed away from the extraction volume (Figure 2 - Figure Supplement 5b). The extraction volume was then released through milling the last connection to the bulk sample with a line pattern. The 30 microm x 110 microm x 25 microm extracted volume was subsequently lifted from the bulk sample.\nAfter the extraction of the volume, the shuttle was returned to the lamella milling orientation (18degrees in a system with a 45degrees pre-tilt) and the stage translated to the receiver grid. The lower edge of the extracted volume and the corner of the pin were aligned in both electron and ion beam images. Attachment was performed by redeposition, using single-pass regular cross-section patterns directed away from the extracted volume (Figure 2 - Figure Supplement 5d). Once attached, the EasyLift system was maneuvered by a 50 nm step horizontally and vertically away from the pin in horizontal direction in order to create a small amount of strain. Subsequently, the lower 4 microm of the volume were sectioned from the rest of the volume using a line pattern. This process was iterated until the entire extracted volume had been sectioned.\n\nSerial Lift-Out: Double-Sided Attachment\n\nLift-In Grid Preparation\nFor double-sided attachment, 100/400 rectangular mesh copper grids (Gilder, Grantham, UK) were used as the receiver grid. These were clipped into standard TFS cartridges that marked such that the 400 mesh bars were in line with the bottom mark and the 100 mesh bars were in line with the side markings. This grid was loaded into the FIB shuttle and 20 microm x 10 microm x 5 microm copper block adapters were prepared as described above.\nDuring the preparation of the receiver grid, roughly 5 microm deep line patterns were milled across the horizontal 400 mesh grid bars in trench milling orientation to divide the rectangles approximately (1/3) and (2/3). These marker lines are intended to guide the alignment of the extracted volume by providing a reference visible in both the electron and ion beams during the attachment process.\n\nLift Out, Sectioning and Milling Procedure\nLift-out site preparation for double-sided attachment was performed on an Aquilos 1 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with a METEOR in-chamber fluorescence light microscope (Delmic, Delft, Netherlands)(&lt;&lt;REF:Smeets-2021-29-20&gt;&gt;). The the GIS was employed for 90 s at 27degreesC with at a stage working distance set to 10.6 mm and the grid normal to the GIS needle to apply a layer of protective metal-organic platinum. The METEOR in-chamber microscope was used in conjunction with previously acquired cryo-confocal data to localize the larva of interest and produce an extraction volume of 42 microm x 180 microm x 20 microm. A width of 40 microm is necessary in order to span the space between the grid bars of a 100/400 rectangular mesh grid. A slight excess of width of the extraction volume is preferable, since a too narrow block will not allow for double-sided attachment, while material can be ablated during sectioning.\nThe lift-out and receiver grid were subsequently transferred into an Aquilos 2 FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift system. Lift-out was otherwise performed as stated in the section for single-sided attachment. In brief, the volume was attached to the needle via the copper adapter, the volume was released, the needle retracted, and the shuttle was returned to the lamella milling orientation at the receiver grid. The extracted volume was re-inserted, and its lower edge was positioned in-between two grid bars. In the case when the volume was slightly too wide, material was ablated from its sides as necessary. To allow for proper attachment, the volume's lower edge was aligned to the reference line on the receiver grid in both, the electron and ion beam images and the volume was attached on both sides by redeposition from the grid bars. To achieve this, single-pass cross-section patterns were placed on the grid bars above the marker lines in in close proximity of the interface between the grid bars and extracted volume, while the milling was directed away from the volume (Figure 2 - Figure Supplement 5c). Once attached, the EasyLift system was maneuvered up in the z-direction by 50-100 nm in order to create a small amount of strain. Subsequently, a section was released from the extracted volume by milling a line pattern at 4 microm above the volume's lower edge. This process was iterated until the entire extracted volume had been sectioned.\nMilling pattern parameters for Serial Lift-Out.\nThe table summarizes the parameters used for the milling steps necessary to perform a Serial Lift-Out experiment. The corresponding pattern geometries are illustrated in Figure 2 - Figure Supplement 5. All patterns were milled at 30.00 kV FIB acceleration voltage. Pattern files for Thermo Fisher Scientific instruments are provided in Figure 2 &lt;&lt;FIG:fig2&gt;&gt; - Pattern Files Supplement. These templates and the milling parameters may need adjustment for different projects and for FIB-SEM instruments from other manufacturers, specifically concerning differing scanning strategies deployed by the microscope manufacturer.\n\nFine milling of lift-out sections\nFor both single-sided and double-sided attachment, the initial 4 microm sections were thinned in two steps: rough milling and fine milling. Rough milling was performed using regular cross-section patterns at a beam current of 1 nA to a thickness of 1.5 microm, 0.5 nA to 1.2 microm lamella thickness, 0.3 nA to 0.8 microm lamella thickness. After rough milling all of the sections, fine milling was performed using regular rectangle patterns at a beam current of 0.1 nA to 0.4 microm and 50 pA to final thickness. Over-tilting and under-tilting by up to 1degrees was used for beam convergence compensation. The double-sided attached lamellae were sputter coated with platinum after fine milling for 4 seconds, at a chamber pressure of 0.20 mbar and a current of 15.0 mA.\n\nSerial lift-out from high-pressure frozen D. melanogaster egg chambers\nLift-out experiments of high-pressure frozen D. melanogaster egg chambers were performed on a Scios FIB-SEM instrument (Thermo Fisher Scientific, Waltham, MA, U.S.A.) equipped with an EasyLift needle. The preparation of the lift-out volume as described previously (&lt;&lt;REF:Klumpe-2021-10-?&gt;&gt;). In brief, an approximately 20 microm x 20 microm volume of target material was prepared using regular cross-section patterns in trench milling orientation in a horseshoe-like shape. The trenches were ~20 microm wide, except for the region that needed to be accessed by the lift-out needle, which was ~35 microm wide. The extraction volume was undercut at a stage tilt of 45degrees, or the highest tilt reachable for the specific position. This preparation leaves the extraction volume attached to the bulk material on a single side. Subsequently, the procedure for single-sided attachment was performed as described above. Sectioning was performed in increments of 1-3 microm using a line pattern milled at a beam current of at 0.3 nA. After section preparation, fine milling was performed by sequentially decreasing the beam current: 0.3 nA to 800 nm, 0.1 nA to 500 nm, 50 pA to 350 nm and 30 pA to &lt; 300 nm. The final step was judged by the loss of contrast in SEM imaging at 3 kV acceleration voltage and a beam current of 13 pA.\n\nLamella preparation by the 'waffle' milling method\nFor preparation of lamellae directly on the high-pressure frozen grid, vitrified as described above, we followed a similar workflow as published as the 'waffle' method (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The grid was coated with a metal-organic platinum layer by GIS deposition for 20 s at a stage working distance of 10.6 mm. Milling was performed on a gallium FIB-SEM Aquilos 2 (Thermo Fisher Scientific, Waltham, MA, U.S.A.) instrument. Initial trenches were milled in trench milling orientation at a beam current of 3 nA using regular cross-section patterns. To avoid damaging the region of interest by milling at high beam currents, 2 microm of buffer distance were left around the region of interest. Trenches were extended to 15 microm on the backside of the subsequent lamella and 30 microm on the front side. In a next step, the trenches were extended to 1 microm closer to the region of interest at a beam current of 1 nA and in a last step the final dimension of the lamella region was defined milling at 0.5 nA. After trench milling, another layer of protective metal-organic platinum was deposited on the sample (20 s deposition time, 10.6 mm stage working distance, three consecutive times). The stage was rotated into lamella milling orientation and a notch was milled with line patterns at 0.3 nA as previously described (&lt;&lt;REF:Kelley-2022-13-1857&gt;&gt;). The preparation of the final lamella was carried out by removing material above and below of the region of interest at a beam current of 0.3 nA, until a final lamella thickness of 2 microm was reached. In sequential steps of decreasing beam current (to 0.8 microm thickness at 0.1 nA beam current, to 0.4 microm at 50 pA), the remnant material was ablated. The lamella was polished to a final thickness of about 200-250 nm at 30 pA beam current.\n\nTEM Data Acquisition\nTEM data acquisition was performed on a Titan Krios G4 at 300 kV equipped with a Selectris X energy filter and a Falcon 4i camera (Thermo Fisher Scientific, Eindhoven, The Netherlands). Lamella overview montages were recorded by stage-driven tiling at 11,500x nominal magnification (pixel size 2.127 nm). Tomograms were recorded using the Tomo5 software package version 5.12.0 (Thermo Fisher Scientific, Eindhoven, The Netherlands) using the EER file format.\nTwo acquisition strategies were deployed. For the tomograms of the single-sided attachment and 'waffle' preparation tilt series were acquired at a nominal magnification of 42,000x resulting in a pixel size at the sample of 2.93 A. A dose-symmetric tilt scheme was used with an angular increment of 2degrees, a dose of 2 e-/A2 per tilt and a target defocus of -4 to -5.5 microm. Tilt series were collected in a tilt range of -70degrees to 50degrees due to the lamellae pre-tilt and with a total dose of 120 e-/A2.\nFor the tomograms collected of the double-sided attachment lamellae, that were subsequently used for subtomogram analysis, tilt series were acquired at a nominal magnification of 64,000x, resulting in a pixel size of 1.89 A. Data was collected using a dose-symmetric tilt scheme with an angular increment of 3degrees, a dose of 3.23 e-/A2 per tilt and a target defocus range of -1 to -4 microm. Angles of -70degrees to 50degrees were acquired, resulting in a total dose of 132 e-/A2.\n\nTomogram Reconstruction, Visualization and Subtomogram Analysis\nData was processed using the Tomoman version 0.7 pipeline (https://github.com/williamnwan/TOMOMAN). 14 frames with a dose of 0.23 e-/A2 per frame were rendered from the EER files. These were used for motion correction in MotionCor2 version 1.4.7 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;) and CTF estimation with CTFFIND4 version 4.14 (&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;). Bad tilts were removed after manual inspection using the Tomoman script. Dose-weighting was performed at 3.23 e-/A2 per tilt using either Tomoman or Warp. For denoising, tilt series were separated into odd and even tilts during motion correction and the resulting stacks were processed using Cryo-CARE (&lt;&lt;REF:Buchholz-2019-?-502&gt;&gt;). Tomogram reconstructions for visualization were done in IMOD version 4.12.32. Tilt series were aligned with AreTomo version 1.3.3. CTF-corrected tomograms for template matching were reconstructed in IMOD version 4.12.32 (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) at eight times binning, resulting in a pixel size of 15.6 A.\nInitial template matching was performed in STOPGAP version 0.7 (&lt;&lt;REF:Wan-2020-26-2516&gt;&gt;) on a subset of 70 tomograms at bin8 using PDB 4V4B as a reference filtered to 35 A (&lt;&lt;REF:Spahn-2004-23-1008&gt;&gt;). 16,420 particles were extracted and aligned in STOPGAP to generate the C. elegans 80S ribosome template. The template was subsequently used to repeat template matching on 200 tomograms at bin8. 65,451 particles were extracted and cleaned by projecting the subtomogram along the Z-axis and subsequent 2D classification in Relion version 4.0. The remaining data set contained 37,026 particles. The retained particles were reprocessed in Warp version 1.0.9, cleaned to remove particles with inadequate CTF resolution and astigmatism. The resulting 35,350 particles were extracted with a pixel size of 2.98 A and a boxsize of 160 pixels. Relion version 3.0 with a spherical mask of 340 A radius was used to align the subtomograms. Finally, the particles were imported into M version 1.0.9 and geometric and CTF parameters were sequentially refined. Corrected subtomograms were extracted from M and classified in Relion 3.0. Two rounds of classification with a spherical mask of 340 A in diameter resulted in 8,256 particles being removed. The second step was a focused classification with a spherical mask around the A/P/E-site. The resulting 5 classes were combined into 2 classes according to the small subunit rotation. Each of these 2 merged classes were subjected to another round of 3D classification with a mask around the elongation factor binding site. The final classes were manually pooled according to structural similarity yielding 5 classes with 2,175, 20,638, 877, 1,119, and 2,285 particles, of which the class with 877 particles was neglected. Segmentation of the elongation factor and tRNAs was done with the Segger tool in Chimera. ChimeraX version 1.3 was used for visualization (&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;).'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [16.13s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"whole organism\",\n  \"organism_name\": \"Caenorhabditis elegans, Drosophila melanogaster\",\n  \"cell_strain\": \"AM140, NK2476\",\n  \"sample_preparation\": \"L1 larvae were hatched and developmentally arrested. They were then vitrified using a modified 'waffle' method with high-pressure freezing on grids. Drosophila samples were prepared by dissecting egg chambers and high-pressure freezing them in standard sample carriers. Egg chambers were dissected in Schneider's medium.\",\n  \"grid_preparation\": \"Grids were coated with a separating layer, and a drop of sample solution was applied onto the grid. Clipped grids were mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.\",\n  \"cryoet_pixel_spacing\": \"1.89 A\",\n  \"cryoet_acceleration_voltage\": \"300 kV\",\n  \"microscope_name\": \"Titan Krios G4\",\n  \"microscope_setup_energy_filter\": \"Selectris X energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Thermo Fisher Scientific\",\n  \"camera_model\": \"Falcon 4i\",\n  \"tilt_minimum\": \"-70 degrees\",\n  \"tilt_maximum\": \"50 degrees\",\n  \"total_flux\": \"132 e-/A2\",\n  \"reconstruction_software\": \"Tomo5, IMOD, AreTomo, Warp, STOPGAP\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"not present\",\n  \"annotated_entities\": \"nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, actin filaments, microtubules, ribosome\",\n  \"annotation_methods\": \"manual segmentation\",\n  \"annotation_software\": \"Chimera, ChimeraX\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"whole organism\",\n  \"organism_name\": \"Caenorhabditis elegans, Drosophila melanogaster\",\n  \"cell_strain\": \"AM140, NK2476\",\n  \"sample_preparation\": \"L1 larvae were hatched and developmentally arrested. They were then vitrified using a modified 'waffle' method with high-pressure freezing on grids. Drosophila samples were prepared by dissecting egg chambers and high-pressure freezing them in standard sample carriers. Egg chambers were dissected in Schneider's medium.\",\n  \"grid_preparation\": \"Grids were coated with a separating layer, and a drop of sample solution was applied onto the grid. Clipped grids were mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.\",\n  \"cryoet_pixel_spacing\": \"1.89 A\",\n  \"cryoet_acceleration_voltage\": \"300 kV\",\n  \"microscope_name\": \"Titan Krios G4\",\n  \"microscope_setup_energy_filter\": \"Selectris X energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Thermo Fisher Scientific\",\n  \"camera_model\": \"Falcon 4i\",\n  \"tilt_minimum\": \"-70 degrees\",\n  \"tilt_maximum\": \"50 degrees\",\n  \"total_flux\": \"132 e-/A2\",\n  \"reconstruction_software\": \"Tomo5, IMOD, AreTomo, Warp, STOPGAP\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"not present\",\n  \"annotated_entities\": \"nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, actin filaments, microtubules, ribosome\",\n  \"annotation_methods\": \"manual segmentation\",\n  \"annotation_software\": \"Chimera, ChimeraX\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 437,
      "prompt_tokens": 14325,
      "total_tokens": 14762
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_021f187978"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [2ms] Exiting Parser run with output:
{
  "biological_sample_type": "whole organism",
  "organism_name": "Caenorhabditis elegans, Drosophila melanogaster",
  "cell_strain": "AM140, NK2476",
  "sample_preparation": "L1 larvae were hatched and developmentally arrested. They were then vitrified using a modified 'waffle' method with high-pressure freezing on grids. Drosophila samples were prepared by dissecting egg chambers and high-pressure freezing them in standard sample carriers. Egg chambers were dissected in Schneider's medium.",
  "grid_preparation": "Grids were coated with a separating layer, and a drop of sample solution was applied onto the grid. Clipped grids were mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.",
  "cryoet_pixel_spacing": "1.89 A",
  "cryoet_acceleration_voltage": "300 kV",
  "microscope_name": "Titan Krios G4",
  "microscope_setup_energy_filter": "Selectris X energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Thermo Fisher Scientific",
  "camera_model": "Falcon 4i",
  "tilt_minimum": "-70 degrees",
  "tilt_maximum": "50 degrees",
  "total_flux": "132 e-/A2",
  "reconstruction_software": "Tomo5, IMOD, AreTomo, Warp, STOPGAP",
  "tomogram_size": "not present",
  "reconstruction_method": "not present",
  "annotated_entities": "nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, actin filaments, microtubules, ribosome",
  "annotation_methods": "manual segmentation",
  "annotation_software": "Chimera, ChimeraX"
}
[chain/end] [1:chain:RunnableSequence] [16.14s] Exiting Chain run with output:
{
  "biological_sample_type": "whole organism",
  "organism_name": "Caenorhabditis elegans, Drosophila melanogaster",
  "cell_strain": "AM140, NK2476",
  "sample_preparation": "L1 larvae were hatched and developmentally arrested. They were then vitrified using a modified 'waffle' method with high-pressure freezing on grids. Drosophila samples were prepared by dissecting egg chambers and high-pressure freezing them in standard sample carriers. Egg chambers were dissected in Schneider's medium.",
  "grid_preparation": "Grids were coated with a separating layer, and a drop of sample solution was applied onto the grid. Clipped grids were mapped in a Leica SP8 confocal microscope equipped with a cryostage or directly transferred into a FIB-SEM instrument with an integrated light microscope.",
  "cryoet_pixel_spacing": "1.89 A",
  "cryoet_acceleration_voltage": "300 kV",
  "microscope_name": "Titan Krios G4",
  "microscope_setup_energy_filter": "Selectris X energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Thermo Fisher Scientific",
  "camera_model": "Falcon 4i",
  "tilt_minimum": "-70 degrees",
  "tilt_maximum": "50 degrees",
  "total_flux": "132 e-/A2",
  "reconstruction_software": "Tomo5, IMOD, AreTomo, Warp, STOPGAP",
  "tomogram_size": "not present",
  "reconstruction_method": "not present",
  "annotated_entities": "nuclei, mitochondria, Golgi cisternae, storage granules, junctional regions, actin filaments, microtubules, ribosome",
  "annotation_methods": "manual segmentation",
  "annotation_software": "Chimera, ChimeraX"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Main\nEukaryotic flagella and motile cilia are whip-like organelles, the rhythmic beating of which propels unicellular eukaryotes through fluids, clears dust particles in respiratory tracts and enables the swimming of sperm cells of various species&lt;&lt;REF:Wallmeier-2020-6-77&gt;&gt;-&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;. Most flagella from protozoa to mammals share a conserved core structure, the axoneme, composed of nine doublet microtubules (doublets) arranged in a circle around a central pair complex of two singlet microtubules (the 9 + 2 configuration, Fig. 1a &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Ishikawa-2017-9-a028076&gt;&gt;,&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;. Dyneins, microtubule-based molecular motors anchored on the nine doublets, drive the relative sliding of neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;,&lt;&lt;REF:Summers-1971-68-3092&gt;&gt;. However, if all dyneins were active at once, forces around the circle of the nine outer doublets would be canceled and the axoneme would not bend&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. To produce rhythmic beating motions, non-motor protein complexes are required to regulate dynein activities across the axoneme structure&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Witman-1978-76-729&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;. The largest and most critical of these regulatory complexes are the radial spokes (RSs) that bridge the outer doublets to the central pair complex and the nexin-dynein regulatory complexes (N-DRCs) that cross-link neighboring doublets and regulate dynein activities across the axoneme structure\nFig. 1\nThe consensus average of nine doublets in mouse sperm possesses unique features in non-motor protein complexes including MIPs, N-DRCs and RSs.\na, Schematic of a cross-section view of the conserved (9 + 2) configuration of axonemes in motile cilia. One doublet is highlighted, and its associated motor and non-motor protein complexes are labeled. The dyneins, N-DRCs, MIPs and RSs are colored in pink, green, orange and cyan, respectively. b-d, Three views of the consensus subvolume average of nine doublets in mouse sperm axonemes. Different protein complexes are highlighted as described in a. c, The clefts in the top of RS1 and RS2 and the 'S'-shaped head of RS3 are indicated by the red arrowheads and dashed line, respectively. e, Schematic of the doublet microtubule with individual protofilaments labeled. Two cross-sections of the A tubule and one longitudinal view of the isolated MIP densities are shown. The MIPs are colored in orange. The protofilaments connecting to the MIPs and the periodicities for the connections are indicated. f,g, Comparison of densities of N-DRCs and RSs from mouse sperm flagella (this study) and human respiratory cilia (EMD-5950)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. f, Additional densities in the mouse sperm N-DRCs are highlighted (light red). g, The unique densities of a barrel and an RS2-RS3 cross-linker in the mouse sperm axoneme are highlighted in blue and red, respectively.\nFlagella from different cells display a wide variety of beating patterns, from the planar and symmetric waveforms observed in flagella of unicellular organisms and sea urchin sperm, to the various non-planar and asymmetric waveforms displayed by different mammalian sperm&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. The structural and regulatory mechanisms underlying these different waveforms are poorly understood. Much of our current structural understanding of axonemes is derived from studies of Chlamydomonas and sea urchin sperm flagella using advanced cryo-electron tomography (cryo-ET) and image processing&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;. Apart from minor variations of the dyneins on a subset of the nine doublets, most of the other motor and non-motor protein complexes were found to be the same across the nine outer doublets. A unique bridge-like structure that cross-links two neighboring doublets is proposed to constrain the plane of bending&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Dutcher-2020-375-20190153&gt;&gt;. The pseudo-ninefold symmetry and the bridge structure are thought to be important for generating equivalent beating amplitudes in the opposite directions, leading to planar and symmetric waveforms.\nComparable structural information for mammalian sperm, which display varied non-planar asymmetric waveforms&lt;&lt;REF:Muschol-2018-13-e0199678&gt;&gt;-&lt;&lt;REF:Babcock-2014-12-67&gt;&gt;, has lagged behind. A technical challenge in using modern cryo-electron microscopy to investigate mammalian sperm flagella is their thickness (&gt;500 nm), which is close to the upper limit for the widely used 300-kV transmission electron microscopes (TEMs). Recently, cryogenic focused ion beam-scanning electron microscopy (cryo-FIB-SEM) and cryo-ET have been applied to study in situ macromolecular structures in sperm axonemes from mammals&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, the limited data obtained in the previous study precluded processing strategies to analyze individual microtubule structures within the axonemes and also their spatial relationships in situ.\nHere, we combined cryo-FIB-SEM and in situ cryo-ET with data-processing strategies to study the contextual assembly of different microtubule-based structures within mouse and human sperm flagella. Our data provide the highest-resolution information to date for mammalian sperm axonemes. Furthermore, our data reveal non-motor protein complexes in mammalian sperm that are not found in axonemes of other mammalian cilia and non-mammalian sperm. We show that each of the nine outer doublets is unique with regard to the composition of regulatory complexes including RSs and N-DRCs. The distribution of regulators varies between mouse and human sperm. We propose that the asymmetric distribution of these regulatory complexes across the axoneme could contribute to the asymmetric and non-planar beating waveforms of various mammalian sperm.\n\nResults\n\nSperm-specific features revealed by subvolume averaging\nFreshly extracted mouse sperm were vitrified on electron microscopy (EM) grids and loaded into a cryo-FIB-SEM device to generate lamellae of ~300 nm in thickness (Extended Data Fig. 1a,b &lt;&lt;FIG:Fig8&gt;&gt;). The lamellae were then imaged using a Krios 300-kV TEM, and dose-symmetric tomographic tilt series (+/-48degrees) around the axoneme were then acquired (Extended Data Fig. 1c &lt;&lt;FIG:Fig8&gt;&gt;). Our images showed detailed molecular features including the double-bilayer membranes of the surrounding mitochondria and individual microtubule protofilaments (Extended Data Fig. 1e,f &lt;&lt;FIG:Fig8&gt;&gt;). Three-dimensional (3D) tomograms were reconstructed from the tilt series, which revealed repetitive axonemal dyneins and RSs along the outer doublets as well as periodic protrusions from the singlet microtubules of the central pair complex (Extended Data Fig. 1h,i &lt;&lt;FIG:Fig8&gt;&gt;). The periodicities of the RSs and central pair protrusions are ~96 nm and ~32 nm, respectively (Extended Data Fig. 1i &lt;&lt;FIG:Fig8&gt;&gt;), consistent with those described in Chlamydomonas and sea urchin sperm&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;.\nTo overcome the low signal-to-noise ratio of raw cryo-ET data, subvolume averaging of 96-nm-repeating units was used to reconstruct consensus density maps (Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Our consensus maps of periodic units from all nine doublets revealed robust signals for individual microtubule protofilaments and other associated protein complexes that repeat every 96 nm (24 A at Fourier Shell Correlation (FSC) = 0.143; n = 9,055 subvolumes; N = 69 tomograms) (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Supplementary Figs. 1 and 2).\nInside the A tubule of the outer doublet, we observed a filamentous density of microtubule inner proteins (MIPs) that is very similar to but more extensive than that recently assigned as Tektin filaments in bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; (Fig. 1b &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The densities of MIPs have a periodicity of 48 nm, consistent with that of previously studied Chlamydomonas flagella and bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;,&lt;&lt;REF:Ma-2019-179-909&gt;&gt;. We thus calculated a subvolume average of the 48-nm-repeating doublets, focusing on the microtubule only (18 A at FSC = 0.143; n = 18,153 subvolumes; N = 69 tomograms) (Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The filamentous components have connections to 12 protofilaments of the A tubule in mouse sperm axonemes (except A4), in contrast to the more limited connections of the Tektin filaments to the A9-A13 and A1 protofilaments observed in bovine tracheal cilia (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and comparisons in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). We observed three different modes of interaction between the MIPs and the lumen of microtubules: (1) interaction with tubulins within a single protofilament, (2) connections to the inter-protofilament space across two neighboring protofilaments and (3) connections spanning multiple protofilaments (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3a,b &lt;&lt;FIG:Fig10&gt;&gt;). Notably, the A9-A10 junction is where the microtubule seam of the A tubule is located&lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;, and we observed several sperm-specific densities spanning protofilaments A9 and A10 that are absent in the map of bovine respiratory cilia (Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). In addition, we observed striated densities along the helical pitch of the microtubule inside the B tubule (Extended Data Fig. 3b,c &lt;&lt;FIG:Fig10&gt;&gt;). These striations are separated by 8 nm and cover the intradimeric interface between the alpha and beta tubulins. Together, our averages revealed sperm-specific MIPs that form an extensive interaction network inside the doublets.\nOur consensus map of the mouse sperm axoneme reveals outer and inner arm dyneins similar to those observed in sea urchin sperm (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 4a,b &lt;&lt;FIG:Fig11&gt;&gt;). However, we observed several unique non-motor protein complexes in the mouse sperm axoneme that do not have equivalent counterparts in the reported structures from Chlamydomonas, Tetrahymena and human respiratory cilia (Fig. 1f,g &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Bui-2012-198-913&gt;&gt;,&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;-&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. While the N-DRC in human respiratory cilia has a 'V' shape&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;, our consensus map reveals extra densities that extend to the microtubule surface, creating a square-shaped structure (Fig. 1f &lt;&lt;FIG:Fig1&gt;&gt;). The RSs are comprised of three tower-like densities, with two adopting similar morphology (RS1 and RS2) and a third, distinct RS3 (Fig. 1b,c,g &lt;&lt;FIG:Fig1&gt;&gt;). When viewed from the 'head' of the towers, RS1 and RS2 both exhibit a cleft and a C2 symmetry axis that extends through the 'tower', while RS3 has a distinctive S-shaped surface with two holes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), similar to the ones observed in human respiratory cilia&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. By contrast, the heads of RS1 and RS2 from Chlamydomonas and Tetrahymena do not have the deep cleft&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;,&lt;&lt;REF:Owa-2019-10-1143&gt;&gt;, the RS3 stump from Chlamydomonas is much shorter, and RS3 from Tetrahymena has a smaller surface area of the head&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). Multiple additional densities, not found in respiratory cilia or unicellular organisms, were observed between the three spokes in the mouse sperm axoneme (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). First, we observed an ~20-nm-sized barrel-shaped density between RS1 and RS2, consistent with extra densities in sperm axonemes reported recently&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. Our higher-resolution map revealed that the barrel is composed of ten rod-shaped strands arranged in a right-handed twist configuration (Supplementary Video 1). Furthermore, densities were found to cross-link RS2 and RS3, hereafter named the 'RS2-RS3 cross-linker' (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). Of note, all these extra densities in the MIPs, N-DRCs and RSs are apparent even when our maps were low-pass filtered to 50 A, a resolution lower than that of the published map of the human respiratory cilia axoneme that does not possess these features (Extended Data Fig. 4d-f &lt;&lt;FIG:Fig11&gt;&gt;). Therefore, the additional densities found in mouse sperm are not due to higher resolution in this work but most likely reflect the presence of additional sperm-specific proteins.\n\nThe outer doublets are arranged in fixed radial positions\nWe next calculated a subvolume average for the 32-nm-repeating units of the central pair complex (26 A at FSC = 0.143; n = 3,062 subvolumes; N = 69 tomograms) (Fig. 2a-c &lt;&lt;FIG:Fig2&gt;&gt;, Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt; and Supplementary Figs. 1 and 2). Individual protofilaments of the two singlet microtubules were clearly resolved (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). Various proteins protrude from both microtubules, giving rise to an asymmetric cross-section contour of the central pair complex (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). We observed two distinct sets of MIPs inside the two singlet microtubules, both of which repeat every 32 nm (Extended Data Fig. 5a &lt;&lt;FIG:Fig12&gt;&gt;). On the external surface of microtubules, we observed both 32-nm-repeating and 16-nm-repeating protrusions. Notably, compared to the central pair complex of sea urchin sperm, where MIPs were not observed, the overall shapes of the external protrusions are very similar (Extended Data Fig. 5b &lt;&lt;FIG:Fig12&gt;&gt;), while both are different compared to the central pair complex of Chlamydomonas flagella (Extended Data Fig. 5c &lt;&lt;FIG:Fig12&gt;&gt;)&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;,&lt;&lt;REF:Han-2022-29-472&gt;&gt;-&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. These comparisons suggest that the central pair complex is likely conserved from invertebrate to vertebrate sperm (animal sperm) but different from the ones from unicellular protists\nFig. 2\nThe central pair complex presents asymmetric surfaces in different directions.\na-c, Three views of the central pair complex of the mouse sperm axoneme. a,c, Periodic protrusions are indicated by red arrowheads and highlighted by light red lines, and their periodicities are labeled. d, An average of the entire axoneme was calculated by expanding the original subvolumes for the central pair complex threefold and averaging without further alignment. At high contour, densities of nine doublet microtubules are resolved at nine distinct radial positions. e, The same average as in d is shown at low contour. Note that the A tubules from doublet microtubules are distinguishable due to the presence of extensive MIP densities. Smeared densities corresponding to where dyneins, RSs and N-DRCs are located could be observed at lower occupancies than those for the doublet microtubules.\nTo understand how the outer doublets are arranged relative to the asymmetric central pair complex, we expanded our aligned subvolumes of the central pair complex three times to include the region where the outer doublet microtubules reside and then calculated an average without further alignment (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Although the alignment was only performed for the central pair complex in the expanded subvolumes, nine distinct doublet microtubule densities that are parallel to the singlet microtubules could be resolved, indicating a remarkably consistent radial arrangement of doublets in the axonemes. The A and B tubules of doublet microtubule were clearly distinguishable based on the stronger MIP signals in the former. By contrast, discrete external complexes such as dyneins, RSs and N-DRCs were poorly resolved. The lack of alignment along the longitudinal direction could be caused by mismatch of the 32-nm periodicity of the central pair complex and the 96-nm periodicity of the doublet microtubules (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;).\nTo better understand the spatial relationship between the central pair complex and the outer doublets, we performed multibody refinement by treating the central pair protrusions and the doublet structure as two rigid bodies, refining them separately and remapping them back to each raw subvolume&lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;. Their relative positions were then subjected to principal-component analysis (Methods and Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;). For all nine interfaces, we always observed that the first principal component, which explains most variations (40-50%), was parallel to the longitudinal axis of axonemes (Supplementary Video 2), suggesting that the doublets and the central pair complex from different tomograms meet at different longitudinal offsets.\nThe multibody refinement also yielded a map with the two rigid bodies placed at their average positions, allowing us to examine how the nine doublets interact with different protrusions of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). Interestingly, we observed that most RS heads were separated by a short distance from the central pair protrusions, without any resolved densities between them (Extended Data Fig. 5d &lt;&lt;FIG:Fig12&gt;&gt;). However, at the central pair interface with doublet 8, we observed protrusions from the central pair complex fit into the 'cleft' of RS1 and RS2 and also the two holes of the 'S curve' of the head of RS3 (Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Such complementary shapes may limit the sideways movement of doublet 8 and stabilize its radial position\nFig. 3\nThe diverse interfaces between RSs and the central pair.\na, The arrangement of the central pair complex and nine doublet microtubules (dmt1-9) is based on averaging relative positions deduced by multibody refinement. The different protrusions of the central pair complex are colored in accordance with data from Carbajal-Gonzalez et al.&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. The interface between the RSs from doublet 8 and the central pair complex is indicated (red rectangle). b-d, Three different views of the interface between the RSs from doublet 8 and the protrusions from the central pair complexes (P1, P2 and P3).\n\nAsymmetric distribution of sperm-specific regulators\nWe then sought to investigate whether the outer doublets themselves differ from each other. We grouped doublet subvolumes based on their radial positions relative to the central pair complex (numbered 1-9 as in refs. &lt;&lt;REF:Linck-2016-33-141&gt;&gt;,&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;). The subvolumes were aligned, and the averages were calculated for each of the nine doublets (n = 810-954 subvolumes; N = 58-64 tomograms). This processing strategy identified unique densities emerging from the inner arm dyneins of doublet 5 and connecting to the B tubule of doublet 6 (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;). These connecting densities are similar to the '5-6 bridge' observed in the sea urchin at lower resolutions&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;, validating our assignment of doublets and processing strategies. Interestingly, the RSs and other 96-nm-repeating features on both doublets 5 and 6 could be resolved concurrently after local refinement, indicating that there is a relatively consistent longitudinal offset (~20 nm) between these two doublet microtubules throughout different axonemes (Extended Data Fig. 6d &lt;&lt;FIG:Fig13&gt;&gt;). The correlation of the unique bridge densities and the consistent offset that is observed only between doublets 5 and 6 suggest that the bridge could limit the relative sliding between this outer doublet pair (in comparison to another doublet pair in Extended Data Fig. 6g,h &lt;&lt;FIG:Fig13&gt;&gt;).\nNext, we systematically compared motor and non-motor or regulatory protein complexes across the nine doublets. Outer arm dyneins across all nine doublets were indistinguishable from those of our consensus average (as shown in Fig. 1d &lt;&lt;FIG:Fig1&gt;&gt;). The densities of inner arm dyneins were also generally similar, with two exceptions. For doublet 5, densities corresponding to dyneins e and g (nomenclature defined in Chlamydomonas&lt;&lt;REF:Bui-2012-198-913&gt;&gt;) were shifted compared to those of the other doublets (Extended Data Fig. 7a &lt;&lt;FIG:Fig14&gt;&gt;), while, for doublet 9, densities for dynein b were not resolved (Extended Data Fig. 7b &lt;&lt;FIG:Fig14&gt;&gt;). These results indicate that the motor proteins are largely the same with only minor variations.\nWe next examined the RSs from each doublet. Strikingly, we observed that the sperm-specific features were asymmetrically distributed across the nine doublets (Fig. 4 &lt;&lt;FIG:Fig4&gt;&gt;). The barrel density was not observed in doublet 1 or 9 and was present at a lower occupancy in doublet 3. In the remaining six doublets, the occupancy of the barrel was comparable to that of other repeating structures (for example, RS1). The RS2-RS3 cross-linkers are absent in doublets 3 and 8 but present in the remaining doublets. In addition, for doublets 2 and 3, we resolved extra densities close to the base of RS3 (RS3 scaffolds) that were not observed in our consensus averages or previously reported consensus averages, in which subvolumes from all nine doublets were averaged together&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. This is likely because only one or two of the nine doublets possess these features (11-22%) and averaging smeared the signals. Our processing strategy allowed us to isolate these subvolumes based on cellular contextual information, and the high occupancies of these unique structures in the respective doublets suggest their consistent presence in these specific doublets\nFig. 4\nAsymmetric distribution of sperm-specific features in RSs from the nine doublet microtubules in mouse sperm.\nThree orthogonal views of the consensus average of doublets in mouse sperm are shown as in Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt;. At the bottom, densities corresponding to RSs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axoneme. Common features are colored in gray, while the barrel, the RS2-RS3 cross-linker and RS3 scaffolds are highlighted in cyan, red and orange, respectively.\nWe also examined the N-DRCs that cross-link neighboring doublets (Fig. 5 &lt;&lt;FIG:Fig5&gt;&gt;). All nine N-DRCs share the common 'V'-shape density, but the extra connections to the microtubule show heterogeneities. Interestingly, N-DRCs from doublets 2-4 share an arch-shaped density perpendicular to the microtubules, while the ones from doublets 6-8 have 45degrees-tilted thin strands. Doublets 1, 5 and 9 all have distinct densities, leading to five different N-DRCs. Note that all these features were observed at similar signal-to-noise levels, and because they resulted from averaging of more than 800 subvolumes sampled in ~60 different axoneme tomograms, they represent the commonly shared features within each individual doublet\nFig. 5\nAsymmetric distribution of sperm-specific features in N-DRCs from the nine doublet microtubules in mouse sperm.\nTop, two orthogonal views of the consensus average of doublets in mouse sperm. Bottom, densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axonemes. Common features among the N-DRCs are colored in gray, while the unique features are highlighted.\nTo test whether the bending states of the axonemes affect the features of RSs and N-DRCs, we curated subvolumes from tomograms with and without apparent curvatures and calculated per-doublet averages. The same set of features of RSs and N-DRCs were observed. In addition, we also collected a dataset of demembraned sperm axonemes (48 tomograms) that were not actively beating. The RSs and N-DRCs in these non-motile sperm have the same asymmetric features highlighted in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt; and 5 &lt;&lt;FIG:Fig5&gt;&gt;, suggesting that the asymmetric densities were not caused by bias in macroscopic curvatures but likely reflect intrinsic compositional heterogeneities in the nine doublets.\nOur in situ data also allowed us to separate axonemes of the midpiece and principal piece based on the presence of mitochondrial and fibrous sheaths, respectively. We averaged subvolumes from these two regions for each doublet and found only subtle differences in the base of RS3 of doublet 2 and also RS1 of doublet 7 (Extended Data Fig. 8a &lt;&lt;FIG:Fig15&gt;&gt;). In these per-doublet averages, we did not resolve robust densities corresponding to the connections between outer dense fibers and the respective doublets. Previous studies suggest that averages of all nine doublets in the proximal principal region from a few tomograms have such attachments&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, our raw tomograms of the proximal principal piece showed that some outer dense fibers are close to the corresponding doublets and some are further away (Extended Data Fig. 8b &lt;&lt;FIG:Fig15&gt;&gt;). Such variations require per-doublet averages to be considered. However, the subset of tomograms in the proximal principal region is small, and per-doublet averaging resulted in anisotropic 3D reconstructions. An even larger dataset is required to resolve structures with such a specific and complex distribution pattern. We also calculated an average of subvolumes from the nine doublets near the beginning of axonemes (within 2 microm) and found no significant difference when compared to the overall consensus average, suggesting that the sperm-specific features are established very close to the basal region of the flagella (Extended Data Fig. 8c,d &lt;&lt;FIG:Fig15&gt;&gt;). Together, these data highlighted the overall consistency of the axoneme structure along the flagella.\nOverall, our per-doublet averages showed that the distributions of various sperm-specific features for both the RS complexes and the N-DRCs follow distinct patterns, such that every doublet is decorated by a unique combination of non-motor proteins.\n\nA distinct asymmetric pattern in human sperm\nWe next examined whether the unique outer doublet features observed in mouse sperm were also conserved in other mammalian sperm. We collected tilt series of intact human sperm without milling, focusing on the thinner principal piece of the flagella (Extended Data Fig. 9a,b &lt;&lt;FIG:Fig16&gt;&gt; and Supplementary Figs. 1 and 2). We then calculated consensus averages of doublet microtubules and the central pair complex (23 A and 31 A at FSC = 0.143; n = 6,613 and 2,365 subvolumes, N = 56 and 59 tomograms, respectively) (Fig. 6a,b &lt;&lt;FIG:Fig6&gt;&gt;). These consensus averages of human sperm were similar to the ones from mice, with the notable exception that the relative occupancy of the barrel between RS1 and RS2 was much lower in human sperm. Using the per-doublet processing approach described above, we then calculated averages for each of the nine doublets individually. Although the per-doublet averages from human sperm were noisier than those from mouse sperm due to greater sample thickness (&gt;400 nm versus ~330 nm as shown in Extended Data Fig. 9c-e &lt;&lt;FIG:Fig16&gt;&gt;), they were sufficient to identify the sperm-specific features (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). In particular, the 5-6 bridge, RS2-RS3 cross-linkers and RS3 scaffolds show similar asymmetric distributions between human and mouse sperm. However, we observed that the barrel density only exists in four of nine doublets, in contrast to seven of nine doublets in mouse sperm axonemes (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). This is consistent with the lower occupancy of barrel densities in our consensus average (Fig. 6a &lt;&lt;FIG:Fig6&gt;&gt;). In particular, doublets 2-5 appear to be different in human and mouse axonemes in terms of the presence of the barrel, while the rest of the doublets are similar. We also examined the N-DRC from each doublet and found five distinct structures like those in mouse sperm (Extended Data Fig. 9f &lt;&lt;FIG:Fig16&gt;&gt;). In summary, our data indicate that the asymmetric architecture of axonemes is a general feature of mammalian sperm axonemes, although there are intriguing variations of distribution for the barrel in human and mouse sperm\nFig. 6\nHuman sperm have a different distribution of barrels compared to that in mouse sperm.\na, A consensus average map of the RS for human sperm axonemes. The barrel density appears to have lower occupancy than the RSs (indicated by the red arrowhead), which is not the case in mouse sperm shown in Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;. b, The consensus average map of the central pair complex for human sperm axonemes (gray) is overlaid with one from mouse sperm axonemes (blue). c, Densities corresponding to RSs from the per-doublet averages are shown around a schematic of the (9 + 2) axonemes. Common features among the RSs are colored in gray, while the barrel, the RS2-RS3 cross-linker and the RS3 scaffold are highlighted in cyan, red and orange, respectively.\n\nDiscussion\nOur in situ tomography studies of mouse and human sperm revealed a large ensemble of macromolecular complexes in their native cellular environment. In particular, we observed various sperm-specific features in the MIPs, RSs and N-DRCs that were not observed in mammalian respiratory cilia and non-mammalian sperm. Furthermore, we reconstructed the entire axoneme using cellular contextual information and uncovered the asymmetric architecture of the mammalian sperm axoneme, where every microtubule-based structure is decorated by a unique set of non-motor proteins (Fig. 7a,b &lt;&lt;FIG:Fig7&gt;&gt;). As these non-motor proteins regulate dynein activities based on previous studies, we propose that the asymmetries of non-motor protein complexes could modulate the sliding of the nine doublets individually to shape the species-specific asymmetric waveforms observed for mammalian sperm, as discussed below\nFig. 7\nModel: every outer doublet is surrounded by a different set of regulatory complexes in mammalian sperm.\nSchematic of the (9 + 2) axonemes of mouse (a) and human (b) sperm. The doublets are numbered, and the sperm-specific regulatory complexes are labeled for each of the nine outer doublets. In particular, the components from the RSs are shown for each doublet and the N-DRCs are colored differently depending on the extra density (as in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt;-6 &lt;&lt;FIG:Fig6&gt;&gt;). Note that only the barrel distribution is different in mouse and human sperm axonemes. c, RSs from different doublets interact with specific stripes of protrusions of the central pair complex. Our multibody refinement is consistent with the sliding hypothesis, and potential longitudinal movements of dmt1 are indicated by the two red arrows. The model is analogous to 'nine train moving on nine tracks'. d, Schematics showing gradual accumulations of offsets of periodic structure units between two filaments in a curved axoneme (left) and consistent offset between doublet 5-doublet 6 (right). e, The constant offset within tomograms and among tomograms (N = 63) would be consistent with limited horizontal bending in sperm flagella.\n\nSperm-specific structures added mechanical couplings\nMammalian sperm flagella are generally much longer and wider than flagella from unicellular organisms or respiratory cilia (lengths of &gt;45 microm versus ~10 microm and diameters of &gt;0.5-1 microm versus ~0.3 microm). Sperm axonemes are also surrounded by additional subcellular structures, such as outer dense fibers and mitochondrial and fibrous sheaths, which likely present additional mechanical challenges during rhythmic beatings. Previous studies also suggested that larger bending torques are associated with mammalian sperm compared to other cilia&lt;&lt;REF:Lindemann-1996-34-258&gt;&gt;, but it has remained unclear how sperm axonemes have evolved specific mechanisms to withstand the additional mechanical stress. Our averages of in situ cryo-ET revealed many additional non-motor proteins that cross-link the known axonemal components. We propose that these additional proteins function to strengthen the mechanical rigidity of the corresponding components to accommodate higher mechanical requirements of sperm axonemes.\nThe microtubules themselves must be able to withstand vigorous bending, and the seam is the weakest point&lt;&lt;REF:Kikkawa-1994-127-1965&gt;&gt;,&lt;&lt;REF:Zhang-2015-162-849&gt;&gt;. Our data revealed the most extensive MIP interaction network in microtubules observed in axonemes to date. The proteins between the A9 and A10 protofilaments could stabilize the lateral interfaces within the seam. Also, the proteins that form the striations inside the B tubule could cross-link the tubulins within B2-B6 protofilaments and also couple these protofilaments laterally along the helical pitch. We also observed extensive filamentous structures within the A tubule that likely provide additional mechanical stability. Tektin-1 to tektin-4 have been known to assemble into three-helix bundles that pack along one another laterally inside the microtubule doublets of bovine tracheal cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; and likely compose a part of the filamentous densities observed in sperm. Mammalian sperm also contain an additional tektin (tektin-5; Supplementary Table 1)&lt;&lt;REF:Firat-Karalar-2014-127-4128&gt;&gt;, which is a candidate for some of the additional sperm-specific densities (as shown in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;), although the assignment requires confirmation with higher-resolution reconstructions.\nWe also observed unique sperm-specific densities on the exterior of the microtubules, specifically the barrel and RS2-RS3 cross-linker between the three RSs, the RS3 scaffolds and extra densities in the N-DRC. The RSs were previously observed to tilt relative to the microtubule&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. Additional connections between the three RSs and the scaffolds at the bases would integrate them together into a more rigid unit. Another possibility is that the coupling could lead to coordinated movement of the three RSs. The N-DRCs regulate the sliding between the neighboring doublets and prevent splaying of axonemes&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;. Extra densities linking to the microtubules could improve the stability of N-DRCs under higher mechanical stress. Together, the additional protein complexes in mammalian sperm would help to maintain the geometric integrity of the (9 + 2) microtubule configuration of the axoneme under mechanical stress during vigorous beating motions.\n\nThe arrangement of the doublets and the central pair complex\nPrevious studies suggested that the central pair complex can twist radially relative to the nine outer doublets in Chlamydomonas flagella&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;. Our observation of densities corresponding to the nine doublet microtubules in the average of the entire (9 + 2) axoneme can exclude the possibility of such free twisting in mammalian sperm as the doublet densities would be smeared by averaging (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Interestingly, we observed a cleft between the two halves of RS1 and RS2 and holes in RS3 in mammalian sperm axonemes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), in contrast to the flatter surface of the RS heads from Chlamydomonas flagella&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). While the flat surfaces may enable twisting in Chlamydomonas&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;, the complementary shapes of RSs and protrusions from the central pair complex in mammalian sperm may restrict such radial movements (as shown in Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Fixation of radial positions of the nine doublets also orients each of the RS complexes in proximity to unique stripes of densities of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). This arrangement could allow functional specialization and divergent evolution of each doublet microtubule, such as the distinct sets of sperm-specific features in the nine doublets.\nOur multibody refinement suggests the existence of heterogeneities in the longitudinal offsets between doublets and the central pair complex in the randomly sampled axonemes that were combined in the averages (Supplementary Video 2). This observation is consistent with the sliding hypothesis for axonemal bending, in which active dyneins generate displacement between the neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;. Such movement would also lead to displacement of the doublets relative to the central pair complex along the longitudinal axis of axonemes, as though there were nine trains (doublets) moving along nine tracks (central pair protrusions) (Fig. 7c &lt;&lt;FIG:Fig7&gt;&gt;).\nThe 5-6 bridge is a sperm-specific feature that appears to be conserved between sea urchin&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt; and mammalian sperm. Previous studies pointed out that, for two parallel inelastic microtubule filaments, bending would lead to gradual accumulation of longitudinal offsets between them if they were at different radii of the bend (Fig. 7d &lt;&lt;FIG:Fig7&gt;&gt;)&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. As an estimate based on the reported curvature of mouse sperm&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;, the offsets between neighboring doublets could differ by as much as Delta~28 nm within one 2 microm-long tomogram (details in the Methods). More importantly, the initial offset of each tomogram varies depending on how much sliding has happened upstream or downstream of the imaged area of the flagella. On the other hand, the resolved periodic features in both doublets 5 and 6 suggest that there is a consistent offset between these two doublets (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;), not just within each tomogram but also among the 63 tomograms that contribute to the average. Due to the nature of random sampling of our imaging areas of different cells (N = 63 tomograms), this could happen if there is generally limited bending along the direction parallel to the plane of these two filaments (Fig. 7e &lt;&lt;FIG:Fig7&gt;&gt;). The bundling of these two doublets can create a filament stiffer than that of a single doublet, with distinct elastic properties or bending propensities in different directions. Such asymmetries of mechanical properties within the nine doublets could also contribute to asymmetric waveforms.\n\nAsymmetries of regulators may lead to asymmetric beatings\nOur in situ data and processing strategies based on the contextual information revealed that the axoneme itself, which appeared to have 'pseudo-ninefold symmetry' in classical TEM images&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;, is highly asymmetric at the molecular level. As the axoneme is the underlying engine that drives the flagellar beating motion, such asymmetries in structure could lead to asymmetric beating waveforms.\nFurthermore, the asymmetries lie mostly in the mammalian sperm-specific non-motor protein complexes, including the RSs and N-DRCs. These complexes are well-established regulators of dynein motor activities, and defects in individual protein components can lead to irregular beating&lt;&lt;REF:Witman-1978-76-729&gt;&gt;,&lt;&lt;REF:Huang-1981-88-80&gt;&gt;,&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;,&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;. In mammalian sperm, we show here that each of the nine doublet microtubules is decorated by a unique composition of sperm-specific regulators. We speculate that the distinct molecular composition could lead to differences in the sliding speeds or bending forces for each of the doublet pairs (Fig. 7 &lt;&lt;FIG:Fig7&gt;&gt;). The non-equivalent forces could lead to a deviation from the single plane of beating characteristic of more symmetric axonemes. Thus, we hypothesize that the non-uniform distribution of sperm-specific RS and N-DRC regulators are important for asymmetric and non-planar beating. Additionally, previous studies showed that human and mouse sperm have different swimming waveforms (movies in refs. &lt;&lt;REF:Babcock-2014-12-67&gt;&gt;,&lt;&lt;REF:Gadelha-2020-6-eaba5168&gt;&gt;). Our studies suggest that, although the RS barrels are conserved in human and mouse, their distribution varies and the variations could create diverse sperm swimming behaviors.\nThis study has revealed mammalian sperm-specific structures within the microtubules, RSs and N-DRCs that are absent in sea urchin and zebrafish sperm&lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Lin-2018-360-eaar1968&gt;&gt;. The still unknown proteins likely arose to serve functions required by natural fertilization in mammals. Sperm from different species are cast into a foreign environment and selected for their ability to reach and fertilize an egg. Sea urchin and zebrafish sperm swim freely in water, whereas mammalian sperm swim in a thin layer of viscous liquid on uneven surfaces of female reproductive tracts&lt;&lt;REF:Suarez-2016-363-185&gt;&gt;. The viscous environment brought more mechanical challenges to the axonemes and hence the microtubule filaments inside. In addition, asymmetric and non-planar waveforms of mammalian sperm could be beneficial to navigate around 3D obstacles in reproductive tracts. Furthermore, the dimensions and physical characteristics of reproductive tracts vary among different mammals; therefore, fine-tuning the underlying molecular features to produce specialized waveforms is likely under evolutionary selection. In the future, systematic genetic and proteomic analyses of mammalian sperm-specific proteins would be valuable to connect sperm-specific axonemal structures with their functions in sperm motility and reproduction.\n\nMethods\nAll chemicals were purchased from Sigma-Aldrich unless otherwise noted.\n\nSample preparation\nMouse sperm were collected from 10-16-week-old C57Bl/6J mice based on a published protocol&lt;&lt;REF:van der Spoel-2002-99-17173&gt;&gt;. Briefly, the sperm were stripped from vasa deferentia by applying pressure to the cauda epididymis in 1x Krebs buffer (1.2 mM KH2PO4, 120 mM NaCl, 1.2 mM MgSO4*7H2O, 14 mM dextrose, 1.2 mM CaCl2*2H2O, 5 mM KCl, 25 mM NaHCO3). The sperm were washed and resuspended in ~100 microl Krebs buffer for the following experiments.\nHuman sperm cells were collected by masturbation from healthy donors and visually inspected for normal morphology and motility before use. Spermatozoa were isolated by the swim-up procedure in HTF or HS solution as previously described&lt;&lt;REF:Skinner-2019-9-e3182&gt;&gt; and then concentrated by centrifuging for 5 min at 500g. The supernatant was reduced to 100 microl, and the cells were resuspended.\nAll experimental procedures using human-derived samples were approved by the Committee on Human Research at the University of California, Berkeley, under IRB protocol number &lt;&lt;REF:???-?-?-?&gt;&gt; -06-5395.\n\nGrid preparation\nEM grids (Quantifoil R 2/2 Au 200 mesh) were glow discharged to be hydrophilic using an easiGlow system (Pelco). The grid was then loaded onto a Leica GP cryo plunger (pre-equilibrated to 95% relative humidity at 25 degreesC). The mouse sperm suspension was then mixed with 10-nm gold beads (Electron Microscopy Sciences, 25487) to achieve final concentrations of 2-6 x 106 million cells per ml. For the demembraned mouse sperm, Triton X-100 was added to a final concentration of 0.1% in the final suspension. The human sperm suspension was mixed with 10-nm gold beads to achieve final concentrations of 0.5-2 x 106 million cells per ml. Next, 3.5 microl of sperm mixture was added to each grid, followed by incubation for 15 s. The grids were then blotted for 4 s and plunge-frozen in liquid ethane.\n\nCryogenic focused ion beam milling\nCryogenic focused ion beam milling was performed either manually using an Aquilos cryo-FIB-SEM microscope (Thermo Fisher Scientific) or automatically using an Aquilos II cryo-FIB-SEM microscope (Thermo Fisher Scientific). A panoramic SEM map of the whole grid was first taken at 377x magnification using an acceleration voltage of 5 kV with a beam current of 13 pA and a dwell time of 1 micros. Targets with appropriate thickness for milling were picked on the map. A platinum layer (~10 nm) was sputter coated, and a gas injection system was used to deposit the precursor compound trimethyl(methylcyclopentadienyl)platinum(IV). The stage was tilted to 15-20degrees, corresponding to a milling angle of 8-13degrees relative to the plane of grids. FIB milling was performed using stepwise decreasing current as the lamellae became thinner (1.0 nA-30 pA; final thickness, ~300 nm). The grids were then stored in liquid nitrogen before imaging.\n\nImage acquisition and tomogram reconstruction\nTilt series of mouse sperm were collected on a 300-kV Titan Krios TEM (Thermo Fisher Scientific) equipped with a high-brightness field emission gun (xFEG), a spherical aberration corrector, a Bioquantum energy filter (Gatan) and a K3 Summit detector (Gatan). The images were recorded at a nominal magnification of 19,500x in super-resolution counting mode using SerialEM&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;. After binning over 2 x 2 pixels, the calibrated pixel size was 3.53 A on the specimen level. For each tilt series, images were acquired using a modified dose-symmetric scheme between -48degrees and 48degrees relative to the lamella with 3degrees steps and grouping of two images on either side (0degrees, 3degrees, 6degrees, -3degrees, -6degrees, 9degrees, 12degrees, -9degrees, -12degrees, 15degrees...). At each tilt angle, the image was recorded as movies divided into eight subframes. The total electron dose applied to a tilt series was 100 e- A-2. The defocus target was set to be -4 to -7 microm.\nTilt series of human sperm were recorded at a nominal magnification of 33,000x in super-resolution counting mode. After binning over 2 x 2 pixels, the calibrated pixel size was 2.66 A on the specimen level. The total electron dose applied to a tilt series was 100 e- A-2. For each tilt series, images were acquired using a bidirectional scheme between -48degrees and 48degrees relative to the lamella starting from either 0degrees or 21degrees, with an incremental step of 3degrees. The defocus target was set to be ~2-5 microm. At each tilt, the image was recorded as movies divided into eight subframes.\nAll movie frames were corrected with a gain reference collected in the same EM session. Movement between frames was corrected using MotionCor2 without dose weighting&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;. Initial reconstructions of all tilt series were performed using AreTomo&lt;&lt;REF:Zheng-2022-6-100068&gt;&gt;, and these 3D tomograms were examined to exclude tomograms with bad target tracking, crystal ice, aberrant defocus, incomplete axonemes (less than five doublet microtubules caused by FIB-SEM milling) and unfavorable orientations (perpendicular to the tilt axis). Alignment of selected tilt series and tomographic reconstructions were then performed using Etomo&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. The aligned tilt series were then CTF corrected using TOMOCTF&lt;&lt;REF:Fernandez-2006-106-587&gt;&gt;, and the tomograms were generated using TOMO3D&lt;&lt;REF:Agulleiro-2015-189-147&gt;&gt; (bin2; pixel size, 7.06 A).\nIn total, we started with 24 milling grids of mouse sperm and obtained 200 lamellae. Tilt series with no crystal ice were kept and processed further. In some cases, parts of the (9 + 2) axoneme was milled away and we only processed the ones with at least five doublets and also enough space to include the full central pair complex for subvolume averaging. In the end, the final reconstructions of the consensus averages were from 69 usable tomograms. The per-doublet averages were based on subsets of the 69 tomograms (58-64 tomograms). For the human sperm cryo-ET dataset, the final reconstructions of the consensus averages are from 65 tilt series with the flagellar orientations within ~30degrees of the tilt axis. Note that the final tomograms used for different reconstructions vary as tomograms may be discarded due to poor alignment or may lack certain doublets.\n\nSubvolume averaging\nSubsequent subvolume extraction, classification and refinement were all performed using RELION3 (ref. &lt;&lt;REF:Bharat-2016-11-2054&gt;&gt;) (schematic workflow in Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Briefly, subvolumes from the doublets were manually picked every 24 nm and extracted with a box size large enough to accommodate a complete 96-nm-repeating unit (pixel size, 7.06 A; box size, 180 pixels; dimension, 127.08 nm). Initially, subvolumes were aligned to a map of Tetrahymena doublets (EMD-9023) low-pass filtered to 80 A, and the resulting map was used as the reference for further processing. Manual curation of the data was performed to check the alignment accuracy and data quality for each tomogram. Supervised 3D classification on RSs gave rise to four class averages of the 96-nm-repeating units at four different registries, staggered from one another by 24 nm. The pixel view in IMOD was used to determine the coordinates corresponding to the base of RS2 in all four class averages, and all subvolumes were re-extracted and recentered to the same point. All subvolumes were combined and aligned to one reference, and duplicate subvolumes were removed based on minimum distance (&lt;40 nm). The remaining subvolumes were aligned to yield the consensus average for all nine doublets. These subvolumes were later remapped and sorted to calculate per-doublet averages (the script is also available below). The MIPs inside microtubule doublets repeat every 48 nm. Subvolumes of the 96-nm-repeating units were recentered on MIP features that repeat every 48 nm and refined with a mask focusing on the microtubules only.\nTo generate an average for the central pair complex, subvolumes were picked and extracted from the central pair complex every 16 nm (pixel size, 7.06 A; box size, 160 pixels; dimension, 112.96 nm) (Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt;). Refinement of all subvolumes to an average of the sea urchin central pair (EMD-9385) resulted in a central pair with only 16-nm-repeating features. The alignment parameters were modified to reset all translations (x, y, z) to zero and then used for a second round of refinement with local search only. Focused classification was then performed on the microtubule-associated proteins on the C1 microtubule to separate the two populations of subvolumes (~50% each) corresponding to the central pair complex with 32-nm periodicity and an offset of 16 nm between them. The subvolumes were then recentered and extracted on the same protein features for the two class averages. All subvolumes were aligned to the same reference using local searches. Duplicate subvolumes were removed based on the minimum separating distance (&lt;30 nm). The remaining subvolumes were refined to generate the consensus average for the central pair complex.\nSubvolumes large enough to include all nine doublets were re-extracted, centering on the aligned subvolumes of the central pair complex (pixel size, 21.18 A; box size, 160 pixels; dimension, 338.88 nm). These subvolumes were averaged without further alignment and used to re-extract subvolumes corresponding to individual doublets. Subvolumes corresponding to a particular doublet were aligned and remapped back to the tomograms in three dimensions. However, these subvolumes do not have the correct alignment for the 96-nm-repeating features, but their centers nevertheless trace the axis of that microtubule doublet accurately when they are remapped in three dimensions (using the script available below). The 96-nm periodic subvolumes curated above to generate the consensus average of 96-nm repeats were then sorted. The subvolumes corresponding to a particular doublet were then aligned and averaged.\nFor multibody refinement (schematic workflow in Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;), the subvolumes corresponding to the specific RS-central pair interface were re-extracted based on the subvolumes of 96-nm units of individual doublet microtubules. The particles were recentered at the junction between the head and the stalk of RS2 to include enough features from both the doublets and the central pair complex. The RSs were then aligned with a mask. This mask and a mask covering the central pair complex were used for the multibody analysis implemented in RELION3 (ref. &lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;). Briefly, separate refinement of the RSs and central pair protrusions provided two sets of alignment parameters: three translational shifts (x, y, z) and three Euler angles required to align one subvolume to each reference. Thus, 12 parameters can be used to remap the two references back to each raw subvolume, and the spatial relationship of the two rigid bodies in each subvolume could be determined by these 12 parameters. Principal-component analyses essentially reprojected the original 12-dimensional data in a new 12-dimensional space with 12 new orthogonal eigenvectors. These 12 eigenvectors could be mathematically determined so that they represent decreasing variations of the entire data along the individual eigenvectors. Our analyses suggest that the first eigenvector or axis is parallel to the axoneme axis and the variations along this axis account for 40-50% of the total variations. When all subvolumes were divided into ten groups based on their projection values on the first axis (10% lowest, 10-20% lowest, ..., 90-100% highest). Each group was then represented by a snapshot depicting the averaging projection values of the group members, and ten of these snapshots were morphed to generate the animation (Supplementary Video 1).\nThe resolutions for maps were estimated based on FSC values of two independently refined half datasets (FSC = 0.143). Local-resolution maps for the consensus averages of doublets and the central pair complex of both human and mouse sperm were calculated by blocres in Bsoft (Supplementary Fig. 2). These local-resolution maps represent relative differences in resolution across the maps, but the absolute values may not be exact. IMOD was used to visualize the tomographic slices&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. UCSF Chimera was used to manually segment the maps for various structure features, and these maps were colored individually to prepare the figures using UCSF ChimeraX&lt;&lt;REF:Pettersen-2004-25-1605&gt;&gt;-&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;.\n\nEstimation of accumulated offsets in axonemes\nThe bending curvature of mouse sperm could be as large as 2 x 105 m-1 or 0.20 microm-1 based on literature (OD = 5 microm = 5,000 nm)&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;. The distance between the neighboring doublets is 72 nm (measured in the average of the axoneme shown in Fig. 2d &lt;&lt;FIG:Fig2&gt;&gt;, OB = 5,072 nm). If AB and CD represent 96-nm repeats from doublet 1 and doublet 2, respectively, offset Delta = CE =~ OC x (/_COD - /_AOB) = 5,000 x (96 / 5,000 - 96 / 5,072) = 1.4 nm.\nNote that the Delta represents the additional offset per 96-nm repeat. In a tomogram that contains an axoneme of ~2 microm, if the offset between the first pair of 96-nm repeats is 0 nm, the offset of the 20th pair is 28 nm. Note that this initial offset (0 nm) is tomogram specific depending on how much sliding has happened upstream or downstream of the imaging area and the imaged cell.\n\nReporting summary\nFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.\n\nOnline content\nAny methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41594-022-00861-0.\n\nSupplementary information\n\nSupplementary Information\nSupplementary Figs. 1 and 2 and Table 1\nReporting Summary\nPeer Review File\nSupplementary Video 1\nThe structure of the RS barrel.\nSupplementary Video 2\nThe movie contains ten frames, and each frame represents an average of one-tenth of the total populations distributed along the first component of the principal-component analysis. The different frames indicate that the RSs and the central pair complex can come together at different longitudinal offsets. The interface between doublet 6 and the central pair protrusion 1b is used as an example here.\nSupplementary Code\nSupplementary information\nThe online version contains supplementary material available at 10.1038/s41594-022-00861-0.\n\nExtended data\n\nExtended Data Fig. \nThe workflow of FIB-SEM/cryo-ET and representative images.\na, Schematic of EM grids with vitrified mouse sperm. An example for where the windows were milled by FIB-SEM is denoted by two yellow boxes. b, SEM (left) and FIB (right) images of a lamella are shown (scale bar: 5 microm, N = 200 lamellae). c, A low-magnification montage and its zoom-in view of a lamella acquired using TEM (scale bar: 1 microm, N = 200 lamellae). Axonemes and mitochondria are indicated by white arrowheads and an arrow, respectively. d, A representative image of non-milled sperm axoneme recorded by 300 kV Titan Krios TEM at a total dose of 3 e-/A2 (scale bar: 100 nm, N = 10 tilt series, not processed further). e, A representative image of milled lamella (thickness ~300 nm) is shown (scale bar: 50 nm, N = 69 tilt series). The zoom-in region in f is outlined (the red rectangle). f, Individual protofilaments of microtubules and double-bilayer membranes of the mitochondria are indicated by yellow arrows and arrowheads. g, Fourier transform of the image in e is shown and the upper limit of ctf fitting when using CTFFIND4 is indicated (at the pixel size of 3.53 A, N = 69 zero-tilt images). h, i, Two sections of 3D tomograms are shown. Outer arm dyneins (every 24 nm) anchored on doublet microtubules are indicated in h (red arrowheads). Protrusions (every 32 nm) extending from the singlet microtubule are indicated in i (black arrowheads). Periodic patterns of three radial spokes on the doublet microtubules (every 96 nm) are grouped and indicated in i (black brackets) (scale bar: 50 nm, N = 69 tomograms).\n\nExtended Data Fig. \nSchematics of data processing.\na, A schematic of the processing workflow of 96 nm-repeating units of doublets to achieve the consensus average combining data from all nine doublets. An example of remapping of the centers of the final subvolumes in three dimensions is shown. b, A schematic of the processing workflow of 32 nm-repeating units of central complex to achieve the subvolume average. Recentering of subvolumes of the central pair complex (every 32 nm) on the doublet 1 is shown. Remapping of these subvolumes helps to trace the trajectory of doublet 1 in three dimensions. The centers of these doublet 1 subvolumes are shown in the same tomogram as in a. c, A schematic of the workflow of multi-body refinement.\n\nExtended Data Fig. \nMIPs network in A-tubule of doublet microtubules.\na, Four cross-section views of the A tubule in the doublet microtubule are shown. The MIPs network is colored in orange. Three modes of interactions between the MIPs and the protofilaments are observed (see text). The protrusions from the filamentous core are indicated by empty red arrowheads and dashed cyan ovals, respectively. b, Six cross-section views of the overlay of the average of the 48-nm repeat of doublets of mouse sperm (orange, this study) and the 48-nm repeat of doublets of bovine trachea cilia (blue, EMD-24664)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;. The contour levels of the two maps are adjusted so that the densities of microtubules are matched. The microtubule seam, A9 and A10 protofilaments are labeled in all views. Note there are additional densities near A9-A10 region in the mouse sperm axoneme (black circles). We observed density striations in the B-tubule in some cross sections. c, An orthogonal view of the B-tubule striations along the helical pitch of the microtubule at a high and a low threshold, respectively. Note the striations are separated by 8 nm and they cover the intradimeric interface between the alpha- and beta-tubulins based on the model for the bovine doublet (PDB: 7rro)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;.\n\nExtended Data Fig. \nComparisons of dyneins and radial spokes of mouse sperm axonemes to published equivalent structures from other motile cilia.\na, Comparison of outer arm dyneins in mouse sperm axoneme (this study) and reported maps for those from sea urchin sperm (EMD-8835 and EMD-8836)&lt;&lt;REF:???-?-?-?&gt;&gt;. b, Comparison of inner arm dyneins in mouse sperm axoneme (this study) and published maps for those from sea urchin sperm (EMD-8837 and EMD-8838)&lt;&lt;REF:???-?-?-?&gt;&gt;. Note the barrel densities between RS1 and RS2 only exist in the mouse sperm are indicated (dashed ovals). c, Comparison of radial spokes of mouse sperm axoneme (this study) and the ones from motile cilia of Chlamydomonas (EMD-9768&lt;&lt;REF:Owa-2019-10-1143&gt;&gt; and EMD-22475&lt;&lt;REF:Gui-2021-28-29&gt;&gt;) and Tetrahymena (EMD-9023)&lt;&lt;REF:Song-2020-17-201&gt;&gt;. Note the RS3 stump in Chlamydomonas is much shorter than the others and the head of RS3 from Tetrahymena has a smaller surface area. d-f, The doublet average of mouse sperm axoneme is low-pass filtered to 50 A and viewed from different angles. Note this resolution is lower than published axoneme maps (for example 34 A for EMD-5950 shown in Fig. 1 &lt;&lt;FIG:Fig1&gt;&gt; for comparison)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. The extensive network of MIPs (d), extra densities at N-DRC (e) and radial spokes (f) remain apparent while comparing to human respiratory cilia, suggesting the presence of novel densities in our maps are not due to higher resolutions, but most likely the presence of additional proteins.\n\nExtended Data Fig. \nThe central pair complex in mouse axoneme possesses more MIPs but the overall shape is similar to the one from sea urchin sperm.\na, Two longitudinal views of the MIPs inside the two singlet microtubules are shown. Periodic features (every 32 nm) are indicated (red empty arrowheads). b, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from sea urchin sperm (EMD-9385) and they are colored in grey and cyan, respectively&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. Note most of the protrusions are very similar and the sea urchin-specific protrusion 1f is indicated in the middle panel (red arrowhead). c, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from Chlamydomonas (EMD-31143)&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. Note the maps are aligned based on the similar C1 protrusions 1d. Other protrusions are generally less similar (1a, 1f and 2d) and there appears to be an 8-nm shift for the C2 protrusions. The Chlamydomonas C2 microtubule was recently shown to possess two possible registers with 8-nm longitudinal differences&lt;&lt;REF:Han-2022-29-472&gt;&gt;. d, The spatial relationship between the nine radial spokes and the central pair complex are determined by multibody refinement and the 'average positioning' of these two rigid bodies are shown (RS-CP1 denotes the interface between radial spokes from doublet 1 and the central pair.\n\nExtended Data Fig. \nThe 5-6 bridge of mouse sperm axonemes.\na-d, Four views of the composite map for 5-6 bridge in mouse sperm axonemes combining maps for local refinement of the doublet 5, doublet 6 and the bridge. The inner arm dyneins, radial spokes, the barrel, N-DRC and extra bridge densities are colored in pink, different shades of cyan, light blue, green and blue, respectively. Note that the radial spokes which repeat every 96 nm on doublet 5 and doublet 6 are resolved and there appears to be a ~20 nm offset between their longitudinal registers, as indicated in the staggering distance of RS1 and RS2 from doublet 5 and doublet 6 in d. e, f, Two cut-in views of a and c are shown. Bridge-specific densities are highlighted in blue. These densities extend from the inner arm dyneins of doublet 5 towards the microtubule of doublet 6. Direct contacts of the bridge-specific densities to doublet 6 is indicated by the solid red arrowhead while densities in close proximity of the base of radial spoke 2 in doublet 6 are indicated by the empty red arrowhead. g, h, Subvolumes for doublet 1 was aligned and equivalent views of doublet 1 and 2 are shown (as in panel a and b). The expected positions for dyneins and radial spokes on doublet 2 are circled by dashed and solid red ovals. These 96 nm-repeating structures on doublet 2 are not resolved even after local refinement, suggesting the offsets between doublet 1 and doublet 2 are not consistent. The microtubule from doublet 2 is nevertheless partially resolved since variant longitudinal mismatching would not smear a tube of 'continuous densities' compared to 'discrete densities' like dyneins and radial spokes through averaging.\n\nExtended Data Fig. \nThe inner arm dyneins in doublet 5 and doublet 9 differ from the consensus average in mouse sperm axonemes.\na, An overlay of inner arm dyneins from doublet 5 and the concensus average. The assigned inner arm dyneins, N-DRC and 5-6 bridge densities from the doublet 5 are colored in pink, green and blue, respectively while the densities for the consensus average are in grey. Different inner arm dyneins are named according to the nomenclature defined by studies of Chlamydomonas flagella. The shifted positions of dynein e and dynein g are indicated by the two yellow arrows. b, An overlay of inner arm dyneins from doublet 9 and the consensus average were blue and grey, respectively. Note dynein b in doublet 9 is not resolved.\n\nExtended Data Fig. \nLongitudinal consistency of doublets in mouse sperm axonemes.\na, Schematic of a mouse sperm flagella with the basal region (&lt; 2 microm), midpiece and principal piece highlighted in the top panel. Subvolume averages of doublet 2 and doublet 7 from the midpiece and the principal piece are shown. The shapes of densities for the scaffolds at the base of radial spoke 3 appear different in doublet 2 and are circled. Additional densities at the base of radial spoke 1 in the doublet 7 in the midpiece region are observed and circled. b, Four xz slices of tomograms in different regions of the mouse sperm flagella are shown. Outer dense fibers that are in close proximity to the doublets and the ones with gaps in between are indicated (red and blue arrowheads, respectively) (scale bars: 100 nm). c, Two xy slices of two 3D tomograms of the basal region of mouse sperm axoneme are shown. The singlet microtubules of the central pair complex extend further into the cell bodies and the beginning of the doublet microtubules are indicated by the arrows and arrowheads, respectively (scale bars: 200 nm). d, Densities corresponding to different protein complexes from subvolume averages of the basal region (6 tomograms, N = 670 subtomograms) and the consensus average are overlaid and colored in blue and grey, respectively. Note the sperm-specific features in N-DRC and radial spokes exist in the average of doublets in the basal region and are highlighted using empty red arrowheads.\n\nExtended Data Fig. \nCryoET data of human sperm without milling and asymmetric distribution of sperm-specific features in N-DRCs in human sperm.\na, A representative xy slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 1, scale bar: 100 nm). Although the A- and B-tubule of the microtubule doublets are discernible, other features like axonemal dyneins and radial spokes are not clear in this slice (in contrast to Extended Data Fig. 1h-i &lt;&lt;FIG:Fig8&gt;&gt;). b, A representative xz slice of the same tomogram of human sperm (pixel size=10.648 A, image thickness: 5, displayed in IMOD) (scale bar: 100 nm). c, A representative xz slice of a tomogram of mouse sperm lamella (pixel size=14.12 A, image thickness: 10, displayed in IMOD). The red bar indicates how we measure the thickness of the lamella (scale bar: 100 nm). d, A representative xz slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 100, displayed in IMOD). Gold beads on the top and bottom surfaces are circled and the red bar indicates how we measure the thickness of the ice near the cell. The top and bottom membrane of the sperm cells is not resolved due to the missing wedge so the actual thickness is larger than the measured thickness of ice in the vicinity (scale bar: 100 nm). e, A dot plot showing the thickness of lamellae of mouse sperm and ice near human sperm as a lower bound estimate of the actual cells. The means and standard deviations are indicated (for N = 69 mouse sperm tomograms and N = 59 human sperm tomograms). f, Densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of the (9 + 2)- axoneme. Common features among the N-DRC are colored in grey, while the unique features are highlighted in various colors.\n\nExtended data\nis available for this paper at 10.1038/s41594-022-00861-0.\n\nPeer review information\nNature Structural &amp; Molecular Biology thanks Girish Mali and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Carolina Perdigoto and Florian Ullrich, in collaboration with the Nature Structural &amp; Molecular Biology team. Peer reviewer reports are available.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Main\nEukaryotic flagella and motile cilia are whip-like organelles, the rhythmic beating of which propels unicellular eukaryotes through fluids, clears dust particles in respiratory tracts and enables the swimming of sperm cells of various species&lt;&lt;REF:Wallmeier-2020-6-77&gt;&gt;-&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;. Most flagella from protozoa to mammals share a conserved core structure, the axoneme, composed of nine doublet microtubules (doublets) arranged in a circle around a central pair complex of two singlet microtubules (the 9 + 2 configuration, Fig. 1a &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Ishikawa-2017-9-a028076&gt;&gt;,&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;. Dyneins, microtubule-based molecular motors anchored on the nine doublets, drive the relative sliding of neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;,&lt;&lt;REF:Summers-1971-68-3092&gt;&gt;. However, if all dyneins were active at once, forces around the circle of the nine outer doublets would be canceled and the axoneme would not bend&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. To produce rhythmic beating motions, non-motor protein complexes are required to regulate dynein activities across the axoneme structure&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Witman-1978-76-729&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;. The largest and most critical of these regulatory complexes are the radial spokes (RSs) that bridge the outer doublets to the central pair complex and the nexin-dynein regulatory complexes (N-DRCs) that cross-link neighboring doublets and regulate dynein activities across the axoneme structure\nFig. 1\nThe consensus average of nine doublets in mouse sperm possesses unique features in non-motor protein complexes including MIPs, N-DRCs and RSs.\na, Schematic of a cross-section view of the conserved (9 + 2) configuration of axonemes in motile cilia. One doublet is highlighted, and its associated motor and non-motor protein complexes are labeled. The dyneins, N-DRCs, MIPs and RSs are colored in pink, green, orange and cyan, respectively. b-d, Three views of the consensus subvolume average of nine doublets in mouse sperm axonemes. Different protein complexes are highlighted as described in a. c, The clefts in the top of RS1 and RS2 and the 'S'-shaped head of RS3 are indicated by the red arrowheads and dashed line, respectively. e, Schematic of the doublet microtubule with individual protofilaments labeled. Two cross-sections of the A tubule and one longitudinal view of the isolated MIP densities are shown. The MIPs are colored in orange. The protofilaments connecting to the MIPs and the periodicities for the connections are indicated. f,g, Comparison of densities of N-DRCs and RSs from mouse sperm flagella (this study) and human respiratory cilia (EMD-5950)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. f, Additional densities in the mouse sperm N-DRCs are highlighted (light red). g, The unique densities of a barrel and an RS2-RS3 cross-linker in the mouse sperm axoneme are highlighted in blue and red, respectively.\nFlagella from different cells display a wide variety of beating patterns, from the planar and symmetric waveforms observed in flagella of unicellular organisms and sea urchin sperm, to the various non-planar and asymmetric waveforms displayed by different mammalian sperm&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. The structural and regulatory mechanisms underlying these different waveforms are poorly understood. Much of our current structural understanding of axonemes is derived from studies of Chlamydomonas and sea urchin sperm flagella using advanced cryo-electron tomography (cryo-ET) and image processing&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;. Apart from minor variations of the dyneins on a subset of the nine doublets, most of the other motor and non-motor protein complexes were found to be the same across the nine outer doublets. A unique bridge-like structure that cross-links two neighboring doublets is proposed to constrain the plane of bending&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Dutcher-2020-375-20190153&gt;&gt;. The pseudo-ninefold symmetry and the bridge structure are thought to be important for generating equivalent beating amplitudes in the opposite directions, leading to planar and symmetric waveforms.\nComparable structural information for mammalian sperm, which display varied non-planar asymmetric waveforms&lt;&lt;REF:Muschol-2018-13-e0199678&gt;&gt;-&lt;&lt;REF:Babcock-2014-12-67&gt;&gt;, has lagged behind. A technical challenge in using modern cryo-electron microscopy to investigate mammalian sperm flagella is their thickness (&gt;500 nm), which is close to the upper limit for the widely used 300-kV transmission electron microscopes (TEMs). Recently, cryogenic focused ion beam-scanning electron microscopy (cryo-FIB-SEM) and cryo-ET have been applied to study in situ macromolecular structures in sperm axonemes from mammals&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, the limited data obtained in the previous study precluded processing strategies to analyze individual microtubule structures within the axonemes and also their spatial relationships in situ.\nHere, we combined cryo-FIB-SEM and in situ cryo-ET with data-processing strategies to study the contextual assembly of different microtubule-based structures within mouse and human sperm flagella. Our data provide the highest-resolution information to date for mammalian sperm axonemes. Furthermore, our data reveal non-motor protein complexes in mammalian sperm that are not found in axonemes of other mammalian cilia and non-mammalian sperm. We show that each of the nine outer doublets is unique with regard to the composition of regulatory complexes including RSs and N-DRCs. The distribution of regulators varies between mouse and human sperm. We propose that the asymmetric distribution of these regulatory complexes across the axoneme could contribute to the asymmetric and non-planar beating waveforms of various mammalian sperm.\n\nResults\n\nSperm-specific features revealed by subvolume averaging\nFreshly extracted mouse sperm were vitrified on electron microscopy (EM) grids and loaded into a cryo-FIB-SEM device to generate lamellae of ~300 nm in thickness (Extended Data Fig. 1a,b &lt;&lt;FIG:Fig8&gt;&gt;). The lamellae were then imaged using a Krios 300-kV TEM, and dose-symmetric tomographic tilt series (+/-48degrees) around the axoneme were then acquired (Extended Data Fig. 1c &lt;&lt;FIG:Fig8&gt;&gt;). Our images showed detailed molecular features including the double-bilayer membranes of the surrounding mitochondria and individual microtubule protofilaments (Extended Data Fig. 1e,f &lt;&lt;FIG:Fig8&gt;&gt;). Three-dimensional (3D) tomograms were reconstructed from the tilt series, which revealed repetitive axonemal dyneins and RSs along the outer doublets as well as periodic protrusions from the singlet microtubules of the central pair complex (Extended Data Fig. 1h,i &lt;&lt;FIG:Fig8&gt;&gt;). The periodicities of the RSs and central pair protrusions are ~96 nm and ~32 nm, respectively (Extended Data Fig. 1i &lt;&lt;FIG:Fig8&gt;&gt;), consistent with those described in Chlamydomonas and sea urchin sperm&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;.\nTo overcome the low signal-to-noise ratio of raw cryo-ET data, subvolume averaging of 96-nm-repeating units was used to reconstruct consensus density maps (Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Our consensus maps of periodic units from all nine doublets revealed robust signals for individual microtubule protofilaments and other associated protein complexes that repeat every 96 nm (24 A at Fourier Shell Correlation (FSC) = 0.143; n = 9,055 subvolumes; N = 69 tomograms) (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Supplementary Figs. 1 and 2).\nInside the A tubule of the outer doublet, we observed a filamentous density of microtubule inner proteins (MIPs) that is very similar to but more extensive than that recently assigned as Tektin filaments in bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; (Fig. 1b &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The densities of MIPs have a periodicity of 48 nm, consistent with that of previously studied Chlamydomonas flagella and bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;,&lt;&lt;REF:Ma-2019-179-909&gt;&gt;. We thus calculated a subvolume average of the 48-nm-repeating doublets, focusing on the microtubule only (18 A at FSC = 0.143; n = 18,153 subvolumes; N = 69 tomograms) (Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The filamentous components have connections to 12 protofilaments of the A tubule in mouse sperm axonemes (except A4), in contrast to the more limited connections of the Tektin filaments to the A9-A13 and A1 protofilaments observed in bovine tracheal cilia (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and comparisons in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). We observed three different modes of interaction between the MIPs and the lumen of microtubules: (1) interaction with tubulins within a single protofilament, (2) connections to the inter-protofilament space across two neighboring protofilaments and (3) connections spanning multiple protofilaments (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3a,b &lt;&lt;FIG:Fig10&gt;&gt;). Notably, the A9-A10 junction is where the microtubule seam of the A tubule is located&lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;, and we observed several sperm-specific densities spanning protofilaments A9 and A10 that are absent in the map of bovine respiratory cilia (Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). In addition, we observed striated densities along the helical pitch of the microtubule inside the B tubule (Extended Data Fig. 3b,c &lt;&lt;FIG:Fig10&gt;&gt;). These striations are separated by 8 nm and cover the intradimeric interface between the alpha and beta tubulins. Together, our averages revealed sperm-specific MIPs that form an extensive interaction network inside the doublets.\nOur consensus map of the mouse sperm axoneme reveals outer and inner arm dyneins similar to those observed in sea urchin sperm (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 4a,b &lt;&lt;FIG:Fig11&gt;&gt;). However, we observed several unique non-motor protein complexes in the mouse sperm axoneme that do not have equivalent counterparts in the reported structures from Chlamydomonas, Tetrahymena and human respiratory cilia (Fig. 1f,g &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Bui-2012-198-913&gt;&gt;,&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;-&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. While the N-DRC in human respiratory cilia has a 'V' shape&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;, our consensus map reveals extra densities that extend to the microtubule surface, creating a square-shaped structure (Fig. 1f &lt;&lt;FIG:Fig1&gt;&gt;). The RSs are comprised of three tower-like densities, with two adopting similar morphology (RS1 and RS2) and a third, distinct RS3 (Fig. 1b,c,g &lt;&lt;FIG:Fig1&gt;&gt;). When viewed from the 'head' of the towers, RS1 and RS2 both exhibit a cleft and a C2 symmetry axis that extends through the 'tower', while RS3 has a distinctive S-shaped surface with two holes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), similar to the ones observed in human respiratory cilia&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. By contrast, the heads of RS1 and RS2 from Chlamydomonas and Tetrahymena do not have the deep cleft&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;,&lt;&lt;REF:Owa-2019-10-1143&gt;&gt;, the RS3 stump from Chlamydomonas is much shorter, and RS3 from Tetrahymena has a smaller surface area of the head&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). Multiple additional densities, not found in respiratory cilia or unicellular organisms, were observed between the three spokes in the mouse sperm axoneme (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). First, we observed an ~20-nm-sized barrel-shaped density between RS1 and RS2, consistent with extra densities in sperm axonemes reported recently&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. Our higher-resolution map revealed that the barrel is composed of ten rod-shaped strands arranged in a right-handed twist configuration (Supplementary Video 1). Furthermore, densities were found to cross-link RS2 and RS3, hereafter named the 'RS2-RS3 cross-linker' (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). Of note, all these extra densities in the MIPs, N-DRCs and RSs are apparent even when our maps were low-pass filtered to 50 A, a resolution lower than that of the published map of the human respiratory cilia axoneme that does not possess these features (Extended Data Fig. 4d-f &lt;&lt;FIG:Fig11&gt;&gt;). Therefore, the additional densities found in mouse sperm are not due to higher resolution in this work but most likely reflect the presence of additional sperm-specific proteins.\n\nThe outer doublets are arranged in fixed radial positions\nWe next calculated a subvolume average for the 32-nm-repeating units of the central pair complex (26 A at FSC = 0.143; n = 3,062 subvolumes; N = 69 tomograms) (Fig. 2a-c &lt;&lt;FIG:Fig2&gt;&gt;, Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt; and Supplementary Figs. 1 and 2). Individual protofilaments of the two singlet microtubules were clearly resolved (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). Various proteins protrude from both microtubules, giving rise to an asymmetric cross-section contour of the central pair complex (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). We observed two distinct sets of MIPs inside the two singlet microtubules, both of which repeat every 32 nm (Extended Data Fig. 5a &lt;&lt;FIG:Fig12&gt;&gt;). On the external surface of microtubules, we observed both 32-nm-repeating and 16-nm-repeating protrusions. Notably, compared to the central pair complex of sea urchin sperm, where MIPs were not observed, the overall shapes of the external protrusions are very similar (Extended Data Fig. 5b &lt;&lt;FIG:Fig12&gt;&gt;), while both are different compared to the central pair complex of Chlamydomonas flagella (Extended Data Fig. 5c &lt;&lt;FIG:Fig12&gt;&gt;)&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;,&lt;&lt;REF:Han-2022-29-472&gt;&gt;-&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. These comparisons suggest that the central pair complex is likely conserved from invertebrate to vertebrate sperm (animal sperm) but different from the ones from unicellular protists\nFig. 2\nThe central pair complex presents asymmetric surfaces in different directions.\na-c, Three views of the central pair complex of the mouse sperm axoneme. a,c, Periodic protrusions are indicated by red arrowheads and highlighted by light red lines, and their periodicities are labeled. d, An average of the entire axoneme was calculated by expanding the original subvolumes for the central pair complex threefold and averaging without further alignment. At high contour, densities of nine doublet microtubules are resolved at nine distinct radial positions. e, The same average as in d is shown at low contour. Note that the A tubules from doublet microtubules are distinguishable due to the presence of extensive MIP densities. Smeared densities corresponding to where dyneins, RSs and N-DRCs are located could be observed at lower occupancies than those for the doublet microtubules.\nTo understand how the outer doublets are arranged relative to the asymmetric central pair complex, we expanded our aligned subvolumes of the central pair complex three times to include the region where the outer doublet microtubules reside and then calculated an average without further alignment (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Although the alignment was only performed for the central pair complex in the expanded subvolumes, nine distinct doublet microtubule densities that are parallel to the singlet microtubules could be resolved, indicating a remarkably consistent radial arrangement of doublets in the axonemes. The A and B tubules of doublet microtubule were clearly distinguishable based on the stronger MIP signals in the former. By contrast, discrete external complexes such as dyneins, RSs and N-DRCs were poorly resolved. The lack of alignment along the longitudinal direction could be caused by mismatch of the 32-nm periodicity of the central pair complex and the 96-nm periodicity of the doublet microtubules (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;).\nTo better understand the spatial relationship between the central pair complex and the outer doublets, we performed multibody refinement by treating the central pair protrusions and the doublet structure as two rigid bodies, refining them separately and remapping them back to each raw subvolume&lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;. Their relative positions were then subjected to principal-component analysis (Methods and Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;). For all nine interfaces, we always observed that the first principal component, which explains most variations (40-50%), was parallel to the longitudinal axis of axonemes (Supplementary Video 2), suggesting that the doublets and the central pair complex from different tomograms meet at different longitudinal offsets.\nThe multibody refinement also yielded a map with the two rigid bodies placed at their average positions, allowing us to examine how the nine doublets interact with different protrusions of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). Interestingly, we observed that most RS heads were separated by a short distance from the central pair protrusions, without any resolved densities between them (Extended Data Fig. 5d &lt;&lt;FIG:Fig12&gt;&gt;). However, at the central pair interface with doublet 8, we observed protrusions from the central pair complex fit into the 'cleft' of RS1 and RS2 and also the two holes of the 'S curve' of the head of RS3 (Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Such complementary shapes may limit the sideways movement of doublet 8 and stabilize its radial position\nFig. 3\nThe diverse interfaces between RSs and the central pair.\na, The arrangement of the central pair complex and nine doublet microtubules (dmt1-9) is based on averaging relative positions deduced by multibody refinement. The different protrusions of the central pair complex are colored in accordance with data from Carbajal-Gonzalez et al.&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. The interface between the RSs from doublet 8 and the central pair complex is indicated (red rectangle). b-d, Three different views of the interface between the RSs from doublet 8 and the protrusions from the central pair complexes (P1, P2 and P3).\n\nAsymmetric distribution of sperm-specific regulators\nWe then sought to investigate whether the outer doublets themselves differ from each other. We grouped doublet subvolumes based on their radial positions relative to the central pair complex (numbered 1-9 as in refs. &lt;&lt;REF:Linck-2016-33-141&gt;&gt;,&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;). The subvolumes were aligned, and the averages were calculated for each of the nine doublets (n = 810-954 subvolumes; N = 58-64 tomograms). This processing strategy identified unique densities emerging from the inner arm dyneins of doublet 5 and connecting to the B tubule of doublet 6 (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;). These connecting densities are similar to the '5-6 bridge' observed in the sea urchin at lower resolutions&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;, validating our assignment of doublets and processing strategies. Interestingly, the RSs and other 96-nm-repeating features on both doublets 5 and 6 could be resolved concurrently after local refinement, indicating that there is a relatively consistent longitudinal offset (~20 nm) between these two doublet microtubules throughout different axonemes (Extended Data Fig. 6d &lt;&lt;FIG:Fig13&gt;&gt;). The correlation of the unique bridge densities and the consistent offset that is observed only between doublets 5 and 6 suggest that the bridge could limit the relative sliding between this outer doublet pair (in comparison to another doublet pair in Extended Data Fig. 6g,h &lt;&lt;FIG:Fig13&gt;&gt;).\nNext, we systematically compared motor and non-motor or regulatory protein complexes across the nine doublets. Outer arm dyneins across all nine doublets were indistinguishable from those of our consensus average (as shown in Fig. 1d &lt;&lt;FIG:Fig1&gt;&gt;). The densities of inner arm dyneins were also generally similar, with two exceptions. For doublet 5, densities corresponding to dyneins e and g (nomenclature defined in Chlamydomonas&lt;&lt;REF:Bui-2012-198-913&gt;&gt;) were shifted compared to those of the other doublets (Extended Data Fig. 7a &lt;&lt;FIG:Fig14&gt;&gt;), while, for doublet 9, densities for dynein b were not resolved (Extended Data Fig. 7b &lt;&lt;FIG:Fig14&gt;&gt;). These results indicate that the motor proteins are largely the same with only minor variations.\nWe next examined the RSs from each doublet. Strikingly, we observed that the sperm-specific features were asymmetrically distributed across the nine doublets (Fig. 4 &lt;&lt;FIG:Fig4&gt;&gt;). The barrel density was not observed in doublet 1 or 9 and was present at a lower occupancy in doublet 3. In the remaining six doublets, the occupancy of the barrel was comparable to that of other repeating structures (for example, RS1). The RS2-RS3 cross-linkers are absent in doublets 3 and 8 but present in the remaining doublets. In addition, for doublets 2 and 3, we resolved extra densities close to the base of RS3 (RS3 scaffolds) that were not observed in our consensus averages or previously reported consensus averages, in which subvolumes from all nine doublets were averaged together&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. This is likely because only one or two of the nine doublets possess these features (11-22%) and averaging smeared the signals. Our processing strategy allowed us to isolate these subvolumes based on cellular contextual information, and the high occupancies of these unique structures in the respective doublets suggest their consistent presence in these specific doublets\nFig. 4\nAsymmetric distribution of sperm-specific features in RSs from the nine doublet microtubules in mouse sperm.\nThree orthogonal views of the consensus average of doublets in mouse sperm are shown as in Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt;. At the bottom, densities corresponding to RSs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axoneme. Common features are colored in gray, while the barrel, the RS2-RS3 cross-linker and RS3 scaffolds are highlighted in cyan, red and orange, respectively.\nWe also examined the N-DRCs that cross-link neighboring doublets (Fig. 5 &lt;&lt;FIG:Fig5&gt;&gt;). All nine N-DRCs share the common 'V'-shape density, but the extra connections to the microtubule show heterogeneities. Interestingly, N-DRCs from doublets 2-4 share an arch-shaped density perpendicular to the microtubules, while the ones from doublets 6-8 have 45degrees-tilted thin strands. Doublets 1, 5 and 9 all have distinct densities, leading to five different N-DRCs. Note that all these features were observed at similar signal-to-noise levels, and because they resulted from averaging of more than 800 subvolumes sampled in ~60 different axoneme tomograms, they represent the commonly shared features within each individual doublet\nFig. 5\nAsymmetric distribution of sperm-specific features in N-DRCs from the nine doublet microtubules in mouse sperm.\nTop, two orthogonal views of the consensus average of doublets in mouse sperm. Bottom, densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axonemes. Common features among the N-DRCs are colored in gray, while the unique features are highlighted.\nTo test whether the bending states of the axonemes affect the features of RSs and N-DRCs, we curated subvolumes from tomograms with and without apparent curvatures and calculated per-doublet averages. The same set of features of RSs and N-DRCs were observed. In addition, we also collected a dataset of demembraned sperm axonemes (48 tomograms) that were not actively beating. The RSs and N-DRCs in these non-motile sperm have the same asymmetric features highlighted in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt; and 5 &lt;&lt;FIG:Fig5&gt;&gt;, suggesting that the asymmetric densities were not caused by bias in macroscopic curvatures but likely reflect intrinsic compositional heterogeneities in the nine doublets.\nOur in situ data also allowed us to separate axonemes of the midpiece and principal piece based on the presence of mitochondrial and fibrous sheaths, respectively. We averaged subvolumes from these two regions for each doublet and found only subtle differences in the base of RS3 of doublet 2 and also RS1 of doublet 7 (Extended Data Fig. 8a &lt;&lt;FIG:Fig15&gt;&gt;). In these per-doublet averages, we did not resolve robust densities corresponding to the connections between outer dense fibers and the respective doublets. Previous studies suggest that averages of all nine doublets in the proximal principal region from a few tomograms have such attachments&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, our raw tomograms of the proximal principal piece showed that some outer dense fibers are close to the corresponding doublets and some are further away (Extended Data Fig. 8b &lt;&lt;FIG:Fig15&gt;&gt;). Such variations require per-doublet averages to be considered. However, the subset of tomograms in the proximal principal region is small, and per-doublet averaging resulted in anisotropic 3D reconstructions. An even larger dataset is required to resolve structures with such a specific and complex distribution pattern. We also calculated an average of subvolumes from the nine doublets near the beginning of axonemes (within 2 microm) and found no significant difference when compared to the overall consensus average, suggesting that the sperm-specific features are established very close to the basal region of the flagella (Extended Data Fig. 8c,d &lt;&lt;FIG:Fig15&gt;&gt;). Together, these data highlighted the overall consistency of the axoneme structure along the flagella.\nOverall, our per-doublet averages showed that the distributions of various sperm-specific features for both the RS complexes and the N-DRCs follow distinct patterns, such that every doublet is decorated by a unique combination of non-motor proteins.\n\nA distinct asymmetric pattern in human sperm\nWe next examined whether the unique outer doublet features observed in mouse sperm were also conserved in other mammalian sperm. We collected tilt series of intact human sperm without milling, focusing on the thinner principal piece of the flagella (Extended Data Fig. 9a,b &lt;&lt;FIG:Fig16&gt;&gt; and Supplementary Figs. 1 and 2). We then calculated consensus averages of doublet microtubules and the central pair complex (23 A and 31 A at FSC = 0.143; n = 6,613 and 2,365 subvolumes, N = 56 and 59 tomograms, respectively) (Fig. 6a,b &lt;&lt;FIG:Fig6&gt;&gt;). These consensus averages of human sperm were similar to the ones from mice, with the notable exception that the relative occupancy of the barrel between RS1 and RS2 was much lower in human sperm. Using the per-doublet processing approach described above, we then calculated averages for each of the nine doublets individually. Although the per-doublet averages from human sperm were noisier than those from mouse sperm due to greater sample thickness (&gt;400 nm versus ~330 nm as shown in Extended Data Fig. 9c-e &lt;&lt;FIG:Fig16&gt;&gt;), they were sufficient to identify the sperm-specific features (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). In particular, the 5-6 bridge, RS2-RS3 cross-linkers and RS3 scaffolds show similar asymmetric distributions between human and mouse sperm. However, we observed that the barrel density only exists in four of nine doublets, in contrast to seven of nine doublets in mouse sperm axonemes (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). This is consistent with the lower occupancy of barrel densities in our consensus average (Fig. 6a &lt;&lt;FIG:Fig6&gt;&gt;). In particular, doublets 2-5 appear to be different in human and mouse axonemes in terms of the presence of the barrel, while the rest of the doublets are similar. We also examined the N-DRC from each doublet and found five distinct structures like those in mouse sperm (Extended Data Fig. 9f &lt;&lt;FIG:Fig16&gt;&gt;). In summary, our data indicate that the asymmetric architecture of axonemes is a general feature of mammalian sperm axonemes, although there are intriguing variations of distribution for the barrel in human and mouse sperm\nFig. 6\nHuman sperm have a different distribution of barrels compared to that in mouse sperm.\na, A consensus average map of the RS for human sperm axonemes. The barrel density appears to have lower occupancy than the RSs (indicated by the red arrowhead), which is not the case in mouse sperm shown in Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;. b, The consensus average map of the central pair complex for human sperm axonemes (gray) is overlaid with one from mouse sperm axonemes (blue). c, Densities corresponding to RSs from the per-doublet averages are shown around a schematic of the (9 + 2) axonemes. Common features among the RSs are colored in gray, while the barrel, the RS2-RS3 cross-linker and the RS3 scaffold are highlighted in cyan, red and orange, respectively.\n\nDiscussion\nOur in situ tomography studies of mouse and human sperm revealed a large ensemble of macromolecular complexes in their native cellular environment. In particular, we observed various sperm-specific features in the MIPs, RSs and N-DRCs that were not observed in mammalian respiratory cilia and non-mammalian sperm. Furthermore, we reconstructed the entire axoneme using cellular contextual information and uncovered the asymmetric architecture of the mammalian sperm axoneme, where every microtubule-based structure is decorated by a unique set of non-motor proteins (Fig. 7a,b &lt;&lt;FIG:Fig7&gt;&gt;). As these non-motor proteins regulate dynein activities based on previous studies, we propose that the asymmetries of non-motor protein complexes could modulate the sliding of the nine doublets individually to shape the species-specific asymmetric waveforms observed for mammalian sperm, as discussed below\nFig. 7\nModel: every outer doublet is surrounded by a different set of regulatory complexes in mammalian sperm.\nSchematic of the (9 + 2) axonemes of mouse (a) and human (b) sperm. The doublets are numbered, and the sperm-specific regulatory complexes are labeled for each of the nine outer doublets. In particular, the components from the RSs are shown for each doublet and the N-DRCs are colored differently depending on the extra density (as in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt;-6 &lt;&lt;FIG:Fig6&gt;&gt;). Note that only the barrel distribution is different in mouse and human sperm axonemes. c, RSs from different doublets interact with specific stripes of protrusions of the central pair complex. Our multibody refinement is consistent with the sliding hypothesis, and potential longitudinal movements of dmt1 are indicated by the two red arrows. The model is analogous to 'nine train moving on nine tracks'. d, Schematics showing gradual accumulations of offsets of periodic structure units between two filaments in a curved axoneme (left) and consistent offset between doublet 5-doublet 6 (right). e, The constant offset within tomograms and among tomograms (N = 63) would be consistent with limited horizontal bending in sperm flagella.\n\nSperm-specific structures added mechanical couplings\nMammalian sperm flagella are generally much longer and wider than flagella from unicellular organisms or respiratory cilia (lengths of &gt;45 microm versus ~10 microm and diameters of &gt;0.5-1 microm versus ~0.3 microm). Sperm axonemes are also surrounded by additional subcellular structures, such as outer dense fibers and mitochondrial and fibrous sheaths, which likely present additional mechanical challenges during rhythmic beatings. Previous studies also suggested that larger bending torques are associated with mammalian sperm compared to other cilia&lt;&lt;REF:Lindemann-1996-34-258&gt;&gt;, but it has remained unclear how sperm axonemes have evolved specific mechanisms to withstand the additional mechanical stress. Our averages of in situ cryo-ET revealed many additional non-motor proteins that cross-link the known axonemal components. We propose that these additional proteins function to strengthen the mechanical rigidity of the corresponding components to accommodate higher mechanical requirements of sperm axonemes.\nThe microtubules themselves must be able to withstand vigorous bending, and the seam is the weakest point&lt;&lt;REF:Kikkawa-1994-127-1965&gt;&gt;,&lt;&lt;REF:Zhang-2015-162-849&gt;&gt;. Our data revealed the most extensive MIP interaction network in microtubules observed in axonemes to date. The proteins between the A9 and A10 protofilaments could stabilize the lateral interfaces within the seam. Also, the proteins that form the striations inside the B tubule could cross-link the tubulins within B2-B6 protofilaments and also couple these protofilaments laterally along the helical pitch. We also observed extensive filamentous structures within the A tubule that likely provide additional mechanical stability. Tektin-1 to tektin-4 have been known to assemble into three-helix bundles that pack along one another laterally inside the microtubule doublets of bovine tracheal cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; and likely compose a part of the filamentous densities observed in sperm. Mammalian sperm also contain an additional tektin (tektin-5; Supplementary Table 1)&lt;&lt;REF:Firat-Karalar-2014-127-4128&gt;&gt;, which is a candidate for some of the additional sperm-specific densities (as shown in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;), although the assignment requires confirmation with higher-resolution reconstructions.\nWe also observed unique sperm-specific densities on the exterior of the microtubules, specifically the barrel and RS2-RS3 cross-linker between the three RSs, the RS3 scaffolds and extra densities in the N-DRC. The RSs were previously observed to tilt relative to the microtubule&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. Additional connections between the three RSs and the scaffolds at the bases would integrate them together into a more rigid unit. Another possibility is that the coupling could lead to coordinated movement of the three RSs. The N-DRCs regulate the sliding between the neighboring doublets and prevent splaying of axonemes&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;. Extra densities linking to the microtubules could improve the stability of N-DRCs under higher mechanical stress. Together, the additional protein complexes in mammalian sperm would help to maintain the geometric integrity of the (9 + 2) microtubule configuration of the axoneme under mechanical stress during vigorous beating motions.\n\nThe arrangement of the doublets and the central pair complex\nPrevious studies suggested that the central pair complex can twist radially relative to the nine outer doublets in Chlamydomonas flagella&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;. Our observation of densities corresponding to the nine doublet microtubules in the average of the entire (9 + 2) axoneme can exclude the possibility of such free twisting in mammalian sperm as the doublet densities would be smeared by averaging (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Interestingly, we observed a cleft between the two halves of RS1 and RS2 and holes in RS3 in mammalian sperm axonemes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), in contrast to the flatter surface of the RS heads from Chlamydomonas flagella&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). While the flat surfaces may enable twisting in Chlamydomonas&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;, the complementary shapes of RSs and protrusions from the central pair complex in mammalian sperm may restrict such radial movements (as shown in Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Fixation of radial positions of the nine doublets also orients each of the RS complexes in proximity to unique stripes of densities of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). This arrangement could allow functional specialization and divergent evolution of each doublet microtubule, such as the distinct sets of sperm-specific features in the nine doublets.\nOur multibody refinement suggests the existence of heterogeneities in the longitudinal offsets between doublets and the central pair complex in the randomly sampled axonemes that were combined in the averages (Supplementary Video 2). This observation is consistent with the sliding hypothesis for axonemal bending, in which active dyneins generate displacement between the neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;. Such movement would also lead to displacement of the doublets relative to the central pair complex along the longitudinal axis of axonemes, as though there were nine trains (doublets) moving along nine tracks (central pair protrusions) (Fig. 7c &lt;&lt;FIG:Fig7&gt;&gt;).\nThe 5-6 bridge is a sperm-specific feature that appears to be conserved between sea urchin&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt; and mammalian sperm. Previous studies pointed out that, for two parallel inelastic microtubule filaments, bending would lead to gradual accumulation of longitudinal offsets between them if they were at different radii of the bend (Fig. 7d &lt;&lt;FIG:Fig7&gt;&gt;)&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. As an estimate based on the reported curvature of mouse sperm&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;, the offsets between neighboring doublets could differ by as much as Delta~28 nm within one 2 microm-long tomogram (details in the Methods). More importantly, the initial offset of each tomogram varies depending on how much sliding has happened upstream or downstream of the imaged area of the flagella. On the other hand, the resolved periodic features in both doublets 5 and 6 suggest that there is a consistent offset between these two doublets (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;), not just within each tomogram but also among the 63 tomograms that contribute to the average. Due to the nature of random sampling of our imaging areas of different cells (N = 63 tomograms), this could happen if there is generally limited bending along the direction parallel to the plane of these two filaments (Fig. 7e &lt;&lt;FIG:Fig7&gt;&gt;). The bundling of these two doublets can create a filament stiffer than that of a single doublet, with distinct elastic properties or bending propensities in different directions. Such asymmetries of mechanical properties within the nine doublets could also contribute to asymmetric waveforms.\n\nAsymmetries of regulators may lead to asymmetric beatings\nOur in situ data and processing strategies based on the contextual information revealed that the axoneme itself, which appeared to have 'pseudo-ninefold symmetry' in classical TEM images&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;, is highly asymmetric at the molecular level. As the axoneme is the underlying engine that drives the flagellar beating motion, such asymmetries in structure could lead to asymmetric beating waveforms.\nFurthermore, the asymmetries lie mostly in the mammalian sperm-specific non-motor protein complexes, including the RSs and N-DRCs. These complexes are well-established regulators of dynein motor activities, and defects in individual protein components can lead to irregular beating&lt;&lt;REF:Witman-1978-76-729&gt;&gt;,&lt;&lt;REF:Huang-1981-88-80&gt;&gt;,&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;,&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;. In mammalian sperm, we show here that each of the nine doublet microtubules is decorated by a unique composition of sperm-specific regulators. We speculate that the distinct molecular composition could lead to differences in the sliding speeds or bending forces for each of the doublet pairs (Fig. 7 &lt;&lt;FIG:Fig7&gt;&gt;). The non-equivalent forces could lead to a deviation from the single plane of beating characteristic of more symmetric axonemes. Thus, we hypothesize that the non-uniform distribution of sperm-specific RS and N-DRC regulators are important for asymmetric and non-planar beating. Additionally, previous studies showed that human and mouse sperm have different swimming waveforms (movies in refs. &lt;&lt;REF:Babcock-2014-12-67&gt;&gt;,&lt;&lt;REF:Gadelha-2020-6-eaba5168&gt;&gt;). Our studies suggest that, although the RS barrels are conserved in human and mouse, their distribution varies and the variations could create diverse sperm swimming behaviors.\nThis study has revealed mammalian sperm-specific structures within the microtubules, RSs and N-DRCs that are absent in sea urchin and zebrafish sperm&lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Lin-2018-360-eaar1968&gt;&gt;. The still unknown proteins likely arose to serve functions required by natural fertilization in mammals. Sperm from different species are cast into a foreign environment and selected for their ability to reach and fertilize an egg. Sea urchin and zebrafish sperm swim freely in water, whereas mammalian sperm swim in a thin layer of viscous liquid on uneven surfaces of female reproductive tracts&lt;&lt;REF:Suarez-2016-363-185&gt;&gt;. The viscous environment brought more mechanical challenges to the axonemes and hence the microtubule filaments inside. In addition, asymmetric and non-planar waveforms of mammalian sperm could be beneficial to navigate around 3D obstacles in reproductive tracts. Furthermore, the dimensions and physical characteristics of reproductive tracts vary among different mammals; therefore, fine-tuning the underlying molecular features to produce specialized waveforms is likely under evolutionary selection. In the future, systematic genetic and proteomic analyses of mammalian sperm-specific proteins would be valuable to connect sperm-specific axonemal structures with their functions in sperm motility and reproduction.\n\nMethods\nAll chemicals were purchased from Sigma-Aldrich unless otherwise noted.\n\nSample preparation\nMouse sperm were collected from 10-16-week-old C57Bl/6J mice based on a published protocol&lt;&lt;REF:van der Spoel-2002-99-17173&gt;&gt;. Briefly, the sperm were stripped from vasa deferentia by applying pressure to the cauda epididymis in 1x Krebs buffer (1.2 mM KH2PO4, 120 mM NaCl, 1.2 mM MgSO4*7H2O, 14 mM dextrose, 1.2 mM CaCl2*2H2O, 5 mM KCl, 25 mM NaHCO3). The sperm were washed and resuspended in ~100 microl Krebs buffer for the following experiments.\nHuman sperm cells were collected by masturbation from healthy donors and visually inspected for normal morphology and motility before use. Spermatozoa were isolated by the swim-up procedure in HTF or HS solution as previously described&lt;&lt;REF:Skinner-2019-9-e3182&gt;&gt; and then concentrated by centrifuging for 5 min at 500g. The supernatant was reduced to 100 microl, and the cells were resuspended.\nAll experimental procedures using human-derived samples were approved by the Committee on Human Research at the University of California, Berkeley, under IRB protocol number &lt;&lt;REF:???-?-?-?&gt;&gt; -06-5395.\n\nGrid preparation\nEM grids (Quantifoil R 2/2 Au 200 mesh) were glow discharged to be hydrophilic using an easiGlow system (Pelco). The grid was then loaded onto a Leica GP cryo plunger (pre-equilibrated to 95% relative humidity at 25 degreesC). The mouse sperm suspension was then mixed with 10-nm gold beads (Electron Microscopy Sciences, 25487) to achieve final concentrations of 2-6 x 106 million cells per ml. For the demembraned mouse sperm, Triton X-100 was added to a final concentration of 0.1% in the final suspension. The human sperm suspension was mixed with 10-nm gold beads to achieve final concentrations of 0.5-2 x 106 million cells per ml. Next, 3.5 microl of sperm mixture was added to each grid, followed by incubation for 15 s. The grids were then blotted for 4 s and plunge-frozen in liquid ethane.\n\nCryogenic focused ion beam milling\nCryogenic focused ion beam milling was performed either manually using an Aquilos cryo-FIB-SEM microscope (Thermo Fisher Scientific) or automatically using an Aquilos II cryo-FIB-SEM microscope (Thermo Fisher Scientific). A panoramic SEM map of the whole grid was first taken at 377x magnification using an acceleration voltage of 5 kV with a beam current of 13 pA and a dwell time of 1 micros. Targets with appropriate thickness for milling were picked on the map. A platinum layer (~10 nm) was sputter coated, and a gas injection system was used to deposit the precursor compound trimethyl(methylcyclopentadienyl)platinum(IV). The stage was tilted to 15-20degrees, corresponding to a milling angle of 8-13degrees relative to the plane of grids. FIB milling was performed using stepwise decreasing current as the lamellae became thinner (1.0 nA-30 pA; final thickness, ~300 nm). The grids were then stored in liquid nitrogen before imaging.\n\nImage acquisition and tomogram reconstruction\nTilt series of mouse sperm were collected on a 300-kV Titan Krios TEM (Thermo Fisher Scientific) equipped with a high-brightness field emission gun (xFEG), a spherical aberration corrector, a Bioquantum energy filter (Gatan) and a K3 Summit detector (Gatan). The images were recorded at a nominal magnification of 19,500x in super-resolution counting mode using SerialEM&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;. After binning over 2 x 2 pixels, the calibrated pixel size was 3.53 A on the specimen level. For each tilt series, images were acquired using a modified dose-symmetric scheme between -48degrees and 48degrees relative to the lamella with 3degrees steps and grouping of two images on either side (0degrees, 3degrees, 6degrees, -3degrees, -6degrees, 9degrees, 12degrees, -9degrees, -12degrees, 15degrees...). At each tilt angle, the image was recorded as movies divided into eight subframes. The total electron dose applied to a tilt series was 100 e- A-2. The defocus target was set to be -4 to -7 microm.\nTilt series of human sperm were recorded at a nominal magnification of 33,000x in super-resolution counting mode. After binning over 2 x 2 pixels, the calibrated pixel size was 2.66 A on the specimen level. The total electron dose applied to a tilt series was 100 e- A-2. For each tilt series, images were acquired using a bidirectional scheme between -48degrees and 48degrees relative to the lamella starting from either 0degrees or 21degrees, with an incremental step of 3degrees. The defocus target was set to be ~2-5 microm. At each tilt, the image was recorded as movies divided into eight subframes.\nAll movie frames were corrected with a gain reference collected in the same EM session. Movement between frames was corrected using MotionCor2 without dose weighting&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;. Initial reconstructions of all tilt series were performed using AreTomo&lt;&lt;REF:Zheng-2022-6-100068&gt;&gt;, and these 3D tomograms were examined to exclude tomograms with bad target tracking, crystal ice, aberrant defocus, incomplete axonemes (less than five doublet microtubules caused by FIB-SEM milling) and unfavorable orientations (perpendicular to the tilt axis). Alignment of selected tilt series and tomographic reconstructions were then performed using Etomo&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. The aligned tilt series were then CTF corrected using TOMOCTF&lt;&lt;REF:Fernandez-2006-106-587&gt;&gt;, and the tomograms were generated using TOMO3D&lt;&lt;REF:Agulleiro-2015-189-147&gt;&gt; (bin2; pixel size, 7.06 A).\nIn total, we started with 24 milling grids of mouse sperm and obtained 200 lamellae. Tilt series with no crystal ice were kept and processed further. In some cases, parts of the (9 + 2) axoneme was milled away and we only processed the ones with at least five doublets and also enough space to include the full central pair complex for subvolume averaging. In the end, the final reconstructions of the consensus averages were from 69 usable tomograms. The per-doublet averages were based on subsets of the 69 tomograms (58-64 tomograms). For the human sperm cryo-ET dataset, the final reconstructions of the consensus averages are from 65 tilt series with the flagellar orientations within ~30degrees of the tilt axis. Note that the final tomograms used for different reconstructions vary as tomograms may be discarded due to poor alignment or may lack certain doublets.\n\nSubvolume averaging\nSubsequent subvolume extraction, classification and refinement were all performed using RELION3 (ref. &lt;&lt;REF:Bharat-2016-11-2054&gt;&gt;) (schematic workflow in Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Briefly, subvolumes from the doublets were manually picked every 24 nm and extracted with a box size large enough to accommodate a complete 96-nm-repeating unit (pixel size, 7.06 A; box size, 180 pixels; dimension, 127.08 nm). Initially, subvolumes were aligned to a map of Tetrahymena doublets (EMD-9023) low-pass filtered to 80 A, and the resulting map was used as the reference for further processing. Manual curation of the data was performed to check the alignment accuracy and data quality for each tomogram. Supervised 3D classification on RSs gave rise to four class averages of the 96-nm-repeating units at four different registries, staggered from one another by 24 nm. The pixel view in IMOD was used to determine the coordinates corresponding to the base of RS2 in all four class averages, and all subvolumes were re-extracted and recentered to the same point. All subvolumes were combined and aligned to one reference, and duplicate subvolumes were removed based on minimum distance (&lt;40 nm). The remaining subvolumes were aligned to yield the consensus average for all nine doublets. These subvolumes were later remapped and sorted to calculate per-doublet averages (the script is also available below). The MIPs inside microtubule doublets repeat every 48 nm. Subvolumes of the 96-nm-repeating units were recentered on MIP features that repeat every 48 nm and refined with a mask focusing on the microtubules only.\nTo generate an average for the central pair complex, subvolumes were picked and extracted from the central pair complex every 16 nm (pixel size, 7.06 A; box size, 160 pixels; dimension, 112.96 nm) (Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt;). Refinement of all subvolumes to an average of the sea urchin central pair (EMD-9385) resulted in a central pair with only 16-nm-repeating features. The alignment parameters were modified to reset all translations (x, y, z) to zero and then used for a second round of refinement with local search only. Focused classification was then performed on the microtubule-associated proteins on the C1 microtubule to separate the two populations of subvolumes (~50% each) corresponding to the central pair complex with 32-nm periodicity and an offset of 16 nm between them. The subvolumes were then recentered and extracted on the same protein features for the two class averages. All subvolumes were aligned to the same reference using local searches. Duplicate subvolumes were removed based on the minimum separating distance (&lt;30 nm). The remaining subvolumes were refined to generate the consensus average for the central pair complex.\nSubvolumes large enough to include all nine doublets were re-extracted, centering on the aligned subvolumes of the central pair complex (pixel size, 21.18 A; box size, 160 pixels; dimension, 338.88 nm). These subvolumes were averaged without further alignment and used to re-extract subvolumes corresponding to individual doublets. Subvolumes corresponding to a particular doublet were aligned and remapped back to the tomograms in three dimensions. However, these subvolumes do not have the correct alignment for the 96-nm-repeating features, but their centers nevertheless trace the axis of that microtubule doublet accurately when they are remapped in three dimensions (using the script available below). The 96-nm periodic subvolumes curated above to generate the consensus average of 96-nm repeats were then sorted. The subvolumes corresponding to a particular doublet were then aligned and averaged.\nFor multibody refinement (schematic workflow in Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;), the subvolumes corresponding to the specific RS-central pair interface were re-extracted based on the subvolumes of 96-nm units of individual doublet microtubules. The particles were recentered at the junction between the head and the stalk of RS2 to include enough features from both the doublets and the central pair complex. The RSs were then aligned with a mask. This mask and a mask covering the central pair complex were used for the multibody analysis implemented in RELION3 (ref. &lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;). Briefly, separate refinement of the RSs and central pair protrusions provided two sets of alignment parameters: three translational shifts (x, y, z) and three Euler angles required to align one subvolume to each reference. Thus, 12 parameters can be used to remap the two references back to each raw subvolume, and the spatial relationship of the two rigid bodies in each subvolume could be determined by these 12 parameters. Principal-component analyses essentially reprojected the original 12-dimensional data in a new 12-dimensional space with 12 new orthogonal eigenvectors. These 12 eigenvectors could be mathematically determined so that they represent decreasing variations of the entire data along the individual eigenvectors. Our analyses suggest that the first eigenvector or axis is parallel to the axoneme axis and the variations along this axis account for 40-50% of the total variations. When all subvolumes were divided into ten groups based on their projection values on the first axis (10% lowest, 10-20% lowest, ..., 90-100% highest). Each group was then represented by a snapshot depicting the averaging projection values of the group members, and ten of these snapshots were morphed to generate the animation (Supplementary Video 1).\nThe resolutions for maps were estimated based on FSC values of two independently refined half datasets (FSC = 0.143). Local-resolution maps for the consensus averages of doublets and the central pair complex of both human and mouse sperm were calculated by blocres in Bsoft (Supplementary Fig. 2). These local-resolution maps represent relative differences in resolution across the maps, but the absolute values may not be exact. IMOD was used to visualize the tomographic slices&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. UCSF Chimera was used to manually segment the maps for various structure features, and these maps were colored individually to prepare the figures using UCSF ChimeraX&lt;&lt;REF:Pettersen-2004-25-1605&gt;&gt;-&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;.\n\nEstimation of accumulated offsets in axonemes\nThe bending curvature of mouse sperm could be as large as 2 x 105 m-1 or 0.20 microm-1 based on literature (OD = 5 microm = 5,000 nm)&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;. The distance between the neighboring doublets is 72 nm (measured in the average of the axoneme shown in Fig. 2d &lt;&lt;FIG:Fig2&gt;&gt;, OB = 5,072 nm). If AB and CD represent 96-nm repeats from doublet 1 and doublet 2, respectively, offset Delta = CE =~ OC x (/_COD - /_AOB) = 5,000 x (96 / 5,000 - 96 / 5,072) = 1.4 nm.\nNote that the Delta represents the additional offset per 96-nm repeat. In a tomogram that contains an axoneme of ~2 microm, if the offset between the first pair of 96-nm repeats is 0 nm, the offset of the 20th pair is 28 nm. Note that this initial offset (0 nm) is tomogram specific depending on how much sliding has happened upstream or downstream of the imaging area and the imaged cell.\n\nReporting summary\nFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.\n\nOnline content\nAny methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41594-022-00861-0.\n\nSupplementary information\n\nSupplementary Information\nSupplementary Figs. 1 and 2 and Table 1\nReporting Summary\nPeer Review File\nSupplementary Video 1\nThe structure of the RS barrel.\nSupplementary Video 2\nThe movie contains ten frames, and each frame represents an average of one-tenth of the total populations distributed along the first component of the principal-component analysis. The different frames indicate that the RSs and the central pair complex can come together at different longitudinal offsets. The interface between doublet 6 and the central pair protrusion 1b is used as an example here.\nSupplementary Code\nSupplementary information\nThe online version contains supplementary material available at 10.1038/s41594-022-00861-0.\n\nExtended data\n\nExtended Data Fig. \nThe workflow of FIB-SEM/cryo-ET and representative images.\na, Schematic of EM grids with vitrified mouse sperm. An example for where the windows were milled by FIB-SEM is denoted by two yellow boxes. b, SEM (left) and FIB (right) images of a lamella are shown (scale bar: 5 microm, N = 200 lamellae). c, A low-magnification montage and its zoom-in view of a lamella acquired using TEM (scale bar: 1 microm, N = 200 lamellae). Axonemes and mitochondria are indicated by white arrowheads and an arrow, respectively. d, A representative image of non-milled sperm axoneme recorded by 300 kV Titan Krios TEM at a total dose of 3 e-/A2 (scale bar: 100 nm, N = 10 tilt series, not processed further). e, A representative image of milled lamella (thickness ~300 nm) is shown (scale bar: 50 nm, N = 69 tilt series). The zoom-in region in f is outlined (the red rectangle). f, Individual protofilaments of microtubules and double-bilayer membranes of the mitochondria are indicated by yellow arrows and arrowheads. g, Fourier transform of the image in e is shown and the upper limit of ctf fitting when using CTFFIND4 is indicated (at the pixel size of 3.53 A, N = 69 zero-tilt images). h, i, Two sections of 3D tomograms are shown. Outer arm dyneins (every 24 nm) anchored on doublet microtubules are indicated in h (red arrowheads). Protrusions (every 32 nm) extending from the singlet microtubule are indicated in i (black arrowheads). Periodic patterns of three radial spokes on the doublet microtubules (every 96 nm) are grouped and indicated in i (black brackets) (scale bar: 50 nm, N = 69 tomograms).\n\nExtended Data Fig. \nSchematics of data processing.\na, A schematic of the processing workflow of 96 nm-repeating units of doublets to achieve the consensus average combining data from all nine doublets. An example of remapping of the centers of the final subvolumes in three dimensions is shown. b, A schematic of the processing workflow of 32 nm-repeating units of central complex to achieve the subvolume average. Recentering of subvolumes of the central pair complex (every 32 nm) on the doublet 1 is shown. Remapping of these subvolumes helps to trace the trajectory of doublet 1 in three dimensions. The centers of these doublet 1 subvolumes are shown in the same tomogram as in a. c, A schematic of the workflow of multi-body refinement.\n\nExtended Data Fig. \nMIPs network in A-tubule of doublet microtubules.\na, Four cross-section views of the A tubule in the doublet microtubule are shown. The MIPs network is colored in orange. Three modes of interactions between the MIPs and the protofilaments are observed (see text). The protrusions from the filamentous core are indicated by empty red arrowheads and dashed cyan ovals, respectively. b, Six cross-section views of the overlay of the average of the 48-nm repeat of doublets of mouse sperm (orange, this study) and the 48-nm repeat of doublets of bovine trachea cilia (blue, EMD-24664)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;. The contour levels of the two maps are adjusted so that the densities of microtubules are matched. The microtubule seam, A9 and A10 protofilaments are labeled in all views. Note there are additional densities near A9-A10 region in the mouse sperm axoneme (black circles). We observed density striations in the B-tubule in some cross sections. c, An orthogonal view of the B-tubule striations along the helical pitch of the microtubule at a high and a low threshold, respectively. Note the striations are separated by 8 nm and they cover the intradimeric interface between the alpha- and beta-tubulins based on the model for the bovine doublet (PDB: 7rro)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;.\n\nExtended Data Fig. \nComparisons of dyneins and radial spokes of mouse sperm axonemes to published equivalent structures from other motile cilia.\na, Comparison of outer arm dyneins in mouse sperm axoneme (this study) and reported maps for those from sea urchin sperm (EMD-8835 and EMD-8836)&lt;&lt;REF:???-?-?-?&gt;&gt;. b, Comparison of inner arm dyneins in mouse sperm axoneme (this study) and published maps for those from sea urchin sperm (EMD-8837 and EMD-8838)&lt;&lt;REF:???-?-?-?&gt;&gt;. Note the barrel densities between RS1 and RS2 only exist in the mouse sperm are indicated (dashed ovals). c, Comparison of radial spokes of mouse sperm axoneme (this study) and the ones from motile cilia of Chlamydomonas (EMD-9768&lt;&lt;REF:Owa-2019-10-1143&gt;&gt; and EMD-22475&lt;&lt;REF:Gui-2021-28-29&gt;&gt;) and Tetrahymena (EMD-9023)&lt;&lt;REF:Song-2020-17-201&gt;&gt;. Note the RS3 stump in Chlamydomonas is much shorter than the others and the head of RS3 from Tetrahymena has a smaller surface area. d-f, The doublet average of mouse sperm axoneme is low-pass filtered to 50 A and viewed from different angles. Note this resolution is lower than published axoneme maps (for example 34 A for EMD-5950 shown in Fig. 1 &lt;&lt;FIG:Fig1&gt;&gt; for comparison)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. The extensive network of MIPs (d), extra densities at N-DRC (e) and radial spokes (f) remain apparent while comparing to human respiratory cilia, suggesting the presence of novel densities in our maps are not due to higher resolutions, but most likely the presence of additional proteins.\n\nExtended Data Fig. \nThe central pair complex in mouse axoneme possesses more MIPs but the overall shape is similar to the one from sea urchin sperm.\na, Two longitudinal views of the MIPs inside the two singlet microtubules are shown. Periodic features (every 32 nm) are indicated (red empty arrowheads). b, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from sea urchin sperm (EMD-9385) and they are colored in grey and cyan, respectively&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. Note most of the protrusions are very similar and the sea urchin-specific protrusion 1f is indicated in the middle panel (red arrowhead). c, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from Chlamydomonas (EMD-31143)&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. Note the maps are aligned based on the similar C1 protrusions 1d. Other protrusions are generally less similar (1a, 1f and 2d) and there appears to be an 8-nm shift for the C2 protrusions. The Chlamydomonas C2 microtubule was recently shown to possess two possible registers with 8-nm longitudinal differences&lt;&lt;REF:Han-2022-29-472&gt;&gt;. d, The spatial relationship between the nine radial spokes and the central pair complex are determined by multibody refinement and the 'average positioning' of these two rigid bodies are shown (RS-CP1 denotes the interface between radial spokes from doublet 1 and the central pair.\n\nExtended Data Fig. \nThe 5-6 bridge of mouse sperm axonemes.\na-d, Four views of the composite map for 5-6 bridge in mouse sperm axonemes combining maps for local refinement of the doublet 5, doublet 6 and the bridge. The inner arm dyneins, radial spokes, the barrel, N-DRC and extra bridge densities are colored in pink, different shades of cyan, light blue, green and blue, respectively. Note that the radial spokes which repeat every 96 nm on doublet 5 and doublet 6 are resolved and there appears to be a ~20 nm offset between their longitudinal registers, as indicated in the staggering distance of RS1 and RS2 from doublet 5 and doublet 6 in d. e, f, Two cut-in views of a and c are shown. Bridge-specific densities are highlighted in blue. These densities extend from the inner arm dyneins of doublet 5 towards the microtubule of doublet 6. Direct contacts of the bridge-specific densities to doublet 6 is indicated by the solid red arrowhead while densities in close proximity of the base of radial spoke 2 in doublet 6 are indicated by the empty red arrowhead. g, h, Subvolumes for doublet 1 was aligned and equivalent views of doublet 1 and 2 are shown (as in panel a and b). The expected positions for dyneins and radial spokes on doublet 2 are circled by dashed and solid red ovals. These 96 nm-repeating structures on doublet 2 are not resolved even after local refinement, suggesting the offsets between doublet 1 and doublet 2 are not consistent. The microtubule from doublet 2 is nevertheless partially resolved since variant longitudinal mismatching would not smear a tube of 'continuous densities' compared to 'discrete densities' like dyneins and radial spokes through averaging.\n\nExtended Data Fig. \nThe inner arm dyneins in doublet 5 and doublet 9 differ from the consensus average in mouse sperm axonemes.\na, An overlay of inner arm dyneins from doublet 5 and the concensus average. The assigned inner arm dyneins, N-DRC and 5-6 bridge densities from the doublet 5 are colored in pink, green and blue, respectively while the densities for the consensus average are in grey. Different inner arm dyneins are named according to the nomenclature defined by studies of Chlamydomonas flagella. The shifted positions of dynein e and dynein g are indicated by the two yellow arrows. b, An overlay of inner arm dyneins from doublet 9 and the consensus average were blue and grey, respectively. Note dynein b in doublet 9 is not resolved.\n\nExtended Data Fig. \nLongitudinal consistency of doublets in mouse sperm axonemes.\na, Schematic of a mouse sperm flagella with the basal region (&lt; 2 microm), midpiece and principal piece highlighted in the top panel. Subvolume averages of doublet 2 and doublet 7 from the midpiece and the principal piece are shown. The shapes of densities for the scaffolds at the base of radial spoke 3 appear different in doublet 2 and are circled. Additional densities at the base of radial spoke 1 in the doublet 7 in the midpiece region are observed and circled. b, Four xz slices of tomograms in different regions of the mouse sperm flagella are shown. Outer dense fibers that are in close proximity to the doublets and the ones with gaps in between are indicated (red and blue arrowheads, respectively) (scale bars: 100 nm). c, Two xy slices of two 3D tomograms of the basal region of mouse sperm axoneme are shown. The singlet microtubules of the central pair complex extend further into the cell bodies and the beginning of the doublet microtubules are indicated by the arrows and arrowheads, respectively (scale bars: 200 nm). d, Densities corresponding to different protein complexes from subvolume averages of the basal region (6 tomograms, N = 670 subtomograms) and the consensus average are overlaid and colored in blue and grey, respectively. Note the sperm-specific features in N-DRC and radial spokes exist in the average of doublets in the basal region and are highlighted using empty red arrowheads.\n\nExtended Data Fig. \nCryoET data of human sperm without milling and asymmetric distribution of sperm-specific features in N-DRCs in human sperm.\na, A representative xy slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 1, scale bar: 100 nm). Although the A- and B-tubule of the microtubule doublets are discernible, other features like axonemal dyneins and radial spokes are not clear in this slice (in contrast to Extended Data Fig. 1h-i &lt;&lt;FIG:Fig8&gt;&gt;). b, A representative xz slice of the same tomogram of human sperm (pixel size=10.648 A, image thickness: 5, displayed in IMOD) (scale bar: 100 nm). c, A representative xz slice of a tomogram of mouse sperm lamella (pixel size=14.12 A, image thickness: 10, displayed in IMOD). The red bar indicates how we measure the thickness of the lamella (scale bar: 100 nm). d, A representative xz slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 100, displayed in IMOD). Gold beads on the top and bottom surfaces are circled and the red bar indicates how we measure the thickness of the ice near the cell. The top and bottom membrane of the sperm cells is not resolved due to the missing wedge so the actual thickness is larger than the measured thickness of ice in the vicinity (scale bar: 100 nm). e, A dot plot showing the thickness of lamellae of mouse sperm and ice near human sperm as a lower bound estimate of the actual cells. The means and standard deviations are indicated (for N = 69 mouse sperm tomograms and N = 59 human sperm tomograms). f, Densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of the (9 + 2)- axoneme. Common features among the N-DRC are colored in grey, while the unique features are highlighted in various colors.\n\nExtended data\nis available for this paper at 10.1038/s41594-022-00861-0.\n\nPeer review information\nNature Structural &amp; Molecular Biology thanks Girish Mali and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Carolina Perdigoto and Florian Ullrich, in collaboration with the Nature Structural &amp; Molecular Biology team. Peer reviewer reports are available.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Main\nEukaryotic flagella and motile cilia are whip-like organelles, the rhythmic beating of which propels unicellular eukaryotes through fluids, clears dust particles in respiratory tracts and enables the swimming of sperm cells of various species&lt;&lt;REF:Wallmeier-2020-6-77&gt;&gt;-&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;. Most flagella from protozoa to mammals share a conserved core structure, the axoneme, composed of nine doublet microtubules (doublets) arranged in a circle around a central pair complex of two singlet microtubules (the 9 + 2 configuration, Fig. 1a &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Ishikawa-2017-9-a028076&gt;&gt;,&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;. Dyneins, microtubule-based molecular motors anchored on the nine doublets, drive the relative sliding of neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;,&lt;&lt;REF:Summers-1971-68-3092&gt;&gt;. However, if all dyneins were active at once, forces around the circle of the nine outer doublets would be canceled and the axoneme would not bend&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. To produce rhythmic beating motions, non-motor protein complexes are required to regulate dynein activities across the axoneme structure&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Witman-1978-76-729&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;. The largest and most critical of these regulatory complexes are the radial spokes (RSs) that bridge the outer doublets to the central pair complex and the nexin-dynein regulatory complexes (N-DRCs) that cross-link neighboring doublets and regulate dynein activities across the axoneme structure\nFig. 1\nThe consensus average of nine doublets in mouse sperm possesses unique features in non-motor protein complexes including MIPs, N-DRCs and RSs.\na, Schematic of a cross-section view of the conserved (9 + 2) configuration of axonemes in motile cilia. One doublet is highlighted, and its associated motor and non-motor protein complexes are labeled. The dyneins, N-DRCs, MIPs and RSs are colored in pink, green, orange and cyan, respectively. b-d, Three views of the consensus subvolume average of nine doublets in mouse sperm axonemes. Different protein complexes are highlighted as described in a. c, The clefts in the top of RS1 and RS2 and the 'S'-shaped head of RS3 are indicated by the red arrowheads and dashed line, respectively. e, Schematic of the doublet microtubule with individual protofilaments labeled. Two cross-sections of the A tubule and one longitudinal view of the isolated MIP densities are shown. The MIPs are colored in orange. The protofilaments connecting to the MIPs and the periodicities for the connections are indicated. f,g, Comparison of densities of N-DRCs and RSs from mouse sperm flagella (this study) and human respiratory cilia (EMD-5950)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. f, Additional densities in the mouse sperm N-DRCs are highlighted (light red). g, The unique densities of a barrel and an RS2-RS3 cross-linker in the mouse sperm axoneme are highlighted in blue and red, respectively.\nFlagella from different cells display a wide variety of beating patterns, from the planar and symmetric waveforms observed in flagella of unicellular organisms and sea urchin sperm, to the various non-planar and asymmetric waveforms displayed by different mammalian sperm&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. The structural and regulatory mechanisms underlying these different waveforms are poorly understood. Much of our current structural understanding of axonemes is derived from studies of Chlamydomonas and sea urchin sperm flagella using advanced cryo-electron tomography (cryo-ET) and image processing&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;. Apart from minor variations of the dyneins on a subset of the nine doublets, most of the other motor and non-motor protein complexes were found to be the same across the nine outer doublets. A unique bridge-like structure that cross-links two neighboring doublets is proposed to constrain the plane of bending&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Dutcher-2020-375-20190153&gt;&gt;. The pseudo-ninefold symmetry and the bridge structure are thought to be important for generating equivalent beating amplitudes in the opposite directions, leading to planar and symmetric waveforms.\nComparable structural information for mammalian sperm, which display varied non-planar asymmetric waveforms&lt;&lt;REF:Muschol-2018-13-e0199678&gt;&gt;-&lt;&lt;REF:Babcock-2014-12-67&gt;&gt;, has lagged behind. A technical challenge in using modern cryo-electron microscopy to investigate mammalian sperm flagella is their thickness (&gt;500 nm), which is close to the upper limit for the widely used 300-kV transmission electron microscopes (TEMs). Recently, cryogenic focused ion beam-scanning electron microscopy (cryo-FIB-SEM) and cryo-ET have been applied to study in situ macromolecular structures in sperm axonemes from mammals&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, the limited data obtained in the previous study precluded processing strategies to analyze individual microtubule structures within the axonemes and also their spatial relationships in situ.\nHere, we combined cryo-FIB-SEM and in situ cryo-ET with data-processing strategies to study the contextual assembly of different microtubule-based structures within mouse and human sperm flagella. Our data provide the highest-resolution information to date for mammalian sperm axonemes. Furthermore, our data reveal non-motor protein complexes in mammalian sperm that are not found in axonemes of other mammalian cilia and non-mammalian sperm. We show that each of the nine outer doublets is unique with regard to the composition of regulatory complexes including RSs and N-DRCs. The distribution of regulators varies between mouse and human sperm. We propose that the asymmetric distribution of these regulatory complexes across the axoneme could contribute to the asymmetric and non-planar beating waveforms of various mammalian sperm.\n\nResults\n\nSperm-specific features revealed by subvolume averaging\nFreshly extracted mouse sperm were vitrified on electron microscopy (EM) grids and loaded into a cryo-FIB-SEM device to generate lamellae of ~300 nm in thickness (Extended Data Fig. 1a,b &lt;&lt;FIG:Fig8&gt;&gt;). The lamellae were then imaged using a Krios 300-kV TEM, and dose-symmetric tomographic tilt series (+/-48degrees) around the axoneme were then acquired (Extended Data Fig. 1c &lt;&lt;FIG:Fig8&gt;&gt;). Our images showed detailed molecular features including the double-bilayer membranes of the surrounding mitochondria and individual microtubule protofilaments (Extended Data Fig. 1e,f &lt;&lt;FIG:Fig8&gt;&gt;). Three-dimensional (3D) tomograms were reconstructed from the tilt series, which revealed repetitive axonemal dyneins and RSs along the outer doublets as well as periodic protrusions from the singlet microtubules of the central pair complex (Extended Data Fig. 1h,i &lt;&lt;FIG:Fig8&gt;&gt;). The periodicities of the RSs and central pair protrusions are ~96 nm and ~32 nm, respectively (Extended Data Fig. 1i &lt;&lt;FIG:Fig8&gt;&gt;), consistent with those described in Chlamydomonas and sea urchin sperm&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;.\nTo overcome the low signal-to-noise ratio of raw cryo-ET data, subvolume averaging of 96-nm-repeating units was used to reconstruct consensus density maps (Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Our consensus maps of periodic units from all nine doublets revealed robust signals for individual microtubule protofilaments and other associated protein complexes that repeat every 96 nm (24 A at Fourier Shell Correlation (FSC) = 0.143; n = 9,055 subvolumes; N = 69 tomograms) (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Supplementary Figs. 1 and 2).\nInside the A tubule of the outer doublet, we observed a filamentous density of microtubule inner proteins (MIPs) that is very similar to but more extensive than that recently assigned as Tektin filaments in bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; (Fig. 1b &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The densities of MIPs have a periodicity of 48 nm, consistent with that of previously studied Chlamydomonas flagella and bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;,&lt;&lt;REF:Ma-2019-179-909&gt;&gt;. We thus calculated a subvolume average of the 48-nm-repeating doublets, focusing on the microtubule only (18 A at FSC = 0.143; n = 18,153 subvolumes; N = 69 tomograms) (Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The filamentous components have connections to 12 protofilaments of the A tubule in mouse sperm axonemes (except A4), in contrast to the more limited connections of the Tektin filaments to the A9-A13 and A1 protofilaments observed in bovine tracheal cilia (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and comparisons in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). We observed three different modes of interaction between the MIPs and the lumen of microtubules: (1) interaction with tubulins within a single protofilament, (2) connections to the inter-protofilament space across two neighboring protofilaments and (3) connections spanning multiple protofilaments (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3a,b &lt;&lt;FIG:Fig10&gt;&gt;). Notably, the A9-A10 junction is where the microtubule seam of the A tubule is located&lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;, and we observed several sperm-specific densities spanning protofilaments A9 and A10 that are absent in the map of bovine respiratory cilia (Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). In addition, we observed striated densities along the helical pitch of the microtubule inside the B tubule (Extended Data Fig. 3b,c &lt;&lt;FIG:Fig10&gt;&gt;). These striations are separated by 8 nm and cover the intradimeric interface between the alpha and beta tubulins. Together, our averages revealed sperm-specific MIPs that form an extensive interaction network inside the doublets.\nOur consensus map of the mouse sperm axoneme reveals outer and inner arm dyneins similar to those observed in sea urchin sperm (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 4a,b &lt;&lt;FIG:Fig11&gt;&gt;). However, we observed several unique non-motor protein complexes in the mouse sperm axoneme that do not have equivalent counterparts in the reported structures from Chlamydomonas, Tetrahymena and human respiratory cilia (Fig. 1f,g &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Bui-2012-198-913&gt;&gt;,&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;-&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. While the N-DRC in human respiratory cilia has a 'V' shape&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;, our consensus map reveals extra densities that extend to the microtubule surface, creating a square-shaped structure (Fig. 1f &lt;&lt;FIG:Fig1&gt;&gt;). The RSs are comprised of three tower-like densities, with two adopting similar morphology (RS1 and RS2) and a third, distinct RS3 (Fig. 1b,c,g &lt;&lt;FIG:Fig1&gt;&gt;). When viewed from the 'head' of the towers, RS1 and RS2 both exhibit a cleft and a C2 symmetry axis that extends through the 'tower', while RS3 has a distinctive S-shaped surface with two holes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), similar to the ones observed in human respiratory cilia&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. By contrast, the heads of RS1 and RS2 from Chlamydomonas and Tetrahymena do not have the deep cleft&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;,&lt;&lt;REF:Owa-2019-10-1143&gt;&gt;, the RS3 stump from Chlamydomonas is much shorter, and RS3 from Tetrahymena has a smaller surface area of the head&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). Multiple additional densities, not found in respiratory cilia or unicellular organisms, were observed between the three spokes in the mouse sperm axoneme (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). First, we observed an ~20-nm-sized barrel-shaped density between RS1 and RS2, consistent with extra densities in sperm axonemes reported recently&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. Our higher-resolution map revealed that the barrel is composed of ten rod-shaped strands arranged in a right-handed twist configuration (Supplementary Video 1). Furthermore, densities were found to cross-link RS2 and RS3, hereafter named the 'RS2-RS3 cross-linker' (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). Of note, all these extra densities in the MIPs, N-DRCs and RSs are apparent even when our maps were low-pass filtered to 50 A, a resolution lower than that of the published map of the human respiratory cilia axoneme that does not possess these features (Extended Data Fig. 4d-f &lt;&lt;FIG:Fig11&gt;&gt;). Therefore, the additional densities found in mouse sperm are not due to higher resolution in this work but most likely reflect the presence of additional sperm-specific proteins.\n\nThe outer doublets are arranged in fixed radial positions\nWe next calculated a subvolume average for the 32-nm-repeating units of the central pair complex (26 A at FSC = 0.143; n = 3,062 subvolumes; N = 69 tomograms) (Fig. 2a-c &lt;&lt;FIG:Fig2&gt;&gt;, Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt; and Supplementary Figs. 1 and 2). Individual protofilaments of the two singlet microtubules were clearly resolved (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). Various proteins protrude from both microtubules, giving rise to an asymmetric cross-section contour of the central pair complex (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). We observed two distinct sets of MIPs inside the two singlet microtubules, both of which repeat every 32 nm (Extended Data Fig. 5a &lt;&lt;FIG:Fig12&gt;&gt;). On the external surface of microtubules, we observed both 32-nm-repeating and 16-nm-repeating protrusions. Notably, compared to the central pair complex of sea urchin sperm, where MIPs were not observed, the overall shapes of the external protrusions are very similar (Extended Data Fig. 5b &lt;&lt;FIG:Fig12&gt;&gt;), while both are different compared to the central pair complex of Chlamydomonas flagella (Extended Data Fig. 5c &lt;&lt;FIG:Fig12&gt;&gt;)&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;,&lt;&lt;REF:Han-2022-29-472&gt;&gt;-&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. These comparisons suggest that the central pair complex is likely conserved from invertebrate to vertebrate sperm (animal sperm) but different from the ones from unicellular protists\nFig. 2\nThe central pair complex presents asymmetric surfaces in different directions.\na-c, Three views of the central pair complex of the mouse sperm axoneme. a,c, Periodic protrusions are indicated by red arrowheads and highlighted by light red lines, and their periodicities are labeled. d, An average of the entire axoneme was calculated by expanding the original subvolumes for the central pair complex threefold and averaging without further alignment. At high contour, densities of nine doublet microtubules are resolved at nine distinct radial positions. e, The same average as in d is shown at low contour. Note that the A tubules from doublet microtubules are distinguishable due to the presence of extensive MIP densities. Smeared densities corresponding to where dyneins, RSs and N-DRCs are located could be observed at lower occupancies than those for the doublet microtubules.\nTo understand how the outer doublets are arranged relative to the asymmetric central pair complex, we expanded our aligned subvolumes of the central pair complex three times to include the region where the outer doublet microtubules reside and then calculated an average without further alignment (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Although the alignment was only performed for the central pair complex in the expanded subvolumes, nine distinct doublet microtubule densities that are parallel to the singlet microtubules could be resolved, indicating a remarkably consistent radial arrangement of doublets in the axonemes. The A and B tubules of doublet microtubule were clearly distinguishable based on the stronger MIP signals in the former. By contrast, discrete external complexes such as dyneins, RSs and N-DRCs were poorly resolved. The lack of alignment along the longitudinal direction could be caused by mismatch of the 32-nm periodicity of the central pair complex and the 96-nm periodicity of the doublet microtubules (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;).\nTo better understand the spatial relationship between the central pair complex and the outer doublets, we performed multibody refinement by treating the central pair protrusions and the doublet structure as two rigid bodies, refining them separately and remapping them back to each raw subvolume&lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;. Their relative positions were then subjected to principal-component analysis (Methods and Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;). For all nine interfaces, we always observed that the first principal component, which explains most variations (40-50%), was parallel to the longitudinal axis of axonemes (Supplementary Video 2), suggesting that the doublets and the central pair complex from different tomograms meet at different longitudinal offsets.\nThe multibody refinement also yielded a map with the two rigid bodies placed at their average positions, allowing us to examine how the nine doublets interact with different protrusions of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). Interestingly, we observed that most RS heads were separated by a short distance from the central pair protrusions, without any resolved densities between them (Extended Data Fig. 5d &lt;&lt;FIG:Fig12&gt;&gt;). However, at the central pair interface with doublet 8, we observed protrusions from the central pair complex fit into the 'cleft' of RS1 and RS2 and also the two holes of the 'S curve' of the head of RS3 (Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Such complementary shapes may limit the sideways movement of doublet 8 and stabilize its radial position\nFig. 3\nThe diverse interfaces between RSs and the central pair.\na, The arrangement of the central pair complex and nine doublet microtubules (dmt1-9) is based on averaging relative positions deduced by multibody refinement. The different protrusions of the central pair complex are colored in accordance with data from Carbajal-Gonzalez et al.&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. The interface between the RSs from doublet 8 and the central pair complex is indicated (red rectangle). b-d, Three different views of the interface between the RSs from doublet 8 and the protrusions from the central pair complexes (P1, P2 and P3).\n\nAsymmetric distribution of sperm-specific regulators\nWe then sought to investigate whether the outer doublets themselves differ from each other. We grouped doublet subvolumes based on their radial positions relative to the central pair complex (numbered 1-9 as in refs. &lt;&lt;REF:Linck-2016-33-141&gt;&gt;,&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;). The subvolumes were aligned, and the averages were calculated for each of the nine doublets (n = 810-954 subvolumes; N = 58-64 tomograms). This processing strategy identified unique densities emerging from the inner arm dyneins of doublet 5 and connecting to the B tubule of doublet 6 (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;). These connecting densities are similar to the '5-6 bridge' observed in the sea urchin at lower resolutions&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;, validating our assignment of doublets and processing strategies. Interestingly, the RSs and other 96-nm-repeating features on both doublets 5 and 6 could be resolved concurrently after local refinement, indicating that there is a relatively consistent longitudinal offset (~20 nm) between these two doublet microtubules throughout different axonemes (Extended Data Fig. 6d &lt;&lt;FIG:Fig13&gt;&gt;). The correlation of the unique bridge densities and the consistent offset that is observed only between doublets 5 and 6 suggest that the bridge could limit the relative sliding between this outer doublet pair (in comparison to another doublet pair in Extended Data Fig. 6g,h &lt;&lt;FIG:Fig13&gt;&gt;).\nNext, we systematically compared motor and non-motor or regulatory protein complexes across the nine doublets. Outer arm dyneins across all nine doublets were indistinguishable from those of our consensus average (as shown in Fig. 1d &lt;&lt;FIG:Fig1&gt;&gt;). The densities of inner arm dyneins were also generally similar, with two exceptions. For doublet 5, densities corresponding to dyneins e and g (nomenclature defined in Chlamydomonas&lt;&lt;REF:Bui-2012-198-913&gt;&gt;) were shifted compared to those of the other doublets (Extended Data Fig. 7a &lt;&lt;FIG:Fig14&gt;&gt;), while, for doublet 9, densities for dynein b were not resolved (Extended Data Fig. 7b &lt;&lt;FIG:Fig14&gt;&gt;). These results indicate that the motor proteins are largely the same with only minor variations.\nWe next examined the RSs from each doublet. Strikingly, we observed that the sperm-specific features were asymmetrically distributed across the nine doublets (Fig. 4 &lt;&lt;FIG:Fig4&gt;&gt;). The barrel density was not observed in doublet 1 or 9 and was present at a lower occupancy in doublet 3. In the remaining six doublets, the occupancy of the barrel was comparable to that of other repeating structures (for example, RS1). The RS2-RS3 cross-linkers are absent in doublets 3 and 8 but present in the remaining doublets. In addition, for doublets 2 and 3, we resolved extra densities close to the base of RS3 (RS3 scaffolds) that were not observed in our consensus averages or previously reported consensus averages, in which subvolumes from all nine doublets were averaged together&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. This is likely because only one or two of the nine doublets possess these features (11-22%) and averaging smeared the signals. Our processing strategy allowed us to isolate these subvolumes based on cellular contextual information, and the high occupancies of these unique structures in the respective doublets suggest their consistent presence in these specific doublets\nFig. 4\nAsymmetric distribution of sperm-specific features in RSs from the nine doublet microtubules in mouse sperm.\nThree orthogonal views of the consensus average of doublets in mouse sperm are shown as in Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt;. At the bottom, densities corresponding to RSs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axoneme. Common features are colored in gray, while the barrel, the RS2-RS3 cross-linker and RS3 scaffolds are highlighted in cyan, red and orange, respectively.\nWe also examined the N-DRCs that cross-link neighboring doublets (Fig. 5 &lt;&lt;FIG:Fig5&gt;&gt;). All nine N-DRCs share the common 'V'-shape density, but the extra connections to the microtubule show heterogeneities. Interestingly, N-DRCs from doublets 2-4 share an arch-shaped density perpendicular to the microtubules, while the ones from doublets 6-8 have 45degrees-tilted thin strands. Doublets 1, 5 and 9 all have distinct densities, leading to five different N-DRCs. Note that all these features were observed at similar signal-to-noise levels, and because they resulted from averaging of more than 800 subvolumes sampled in ~60 different axoneme tomograms, they represent the commonly shared features within each individual doublet\nFig. 5\nAsymmetric distribution of sperm-specific features in N-DRCs from the nine doublet microtubules in mouse sperm.\nTop, two orthogonal views of the consensus average of doublets in mouse sperm. Bottom, densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axonemes. Common features among the N-DRCs are colored in gray, while the unique features are highlighted.\nTo test whether the bending states of the axonemes affect the features of RSs and N-DRCs, we curated subvolumes from tomograms with and without apparent curvatures and calculated per-doublet averages. The same set of features of RSs and N-DRCs were observed. In addition, we also collected a dataset of demembraned sperm axonemes (48 tomograms) that were not actively beating. The RSs and N-DRCs in these non-motile sperm have the same asymmetric features highlighted in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt; and 5 &lt;&lt;FIG:Fig5&gt;&gt;, suggesting that the asymmetric densities were not caused by bias in macroscopic curvatures but likely reflect intrinsic compositional heterogeneities in the nine doublets.\nOur in situ data also allowed us to separate axonemes of the midpiece and principal piece based on the presence of mitochondrial and fibrous sheaths, respectively. We averaged subvolumes from these two regions for each doublet and found only subtle differences in the base of RS3 of doublet 2 and also RS1 of doublet 7 (Extended Data Fig. 8a &lt;&lt;FIG:Fig15&gt;&gt;). In these per-doublet averages, we did not resolve robust densities corresponding to the connections between outer dense fibers and the respective doublets. Previous studies suggest that averages of all nine doublets in the proximal principal region from a few tomograms have such attachments&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, our raw tomograms of the proximal principal piece showed that some outer dense fibers are close to the corresponding doublets and some are further away (Extended Data Fig. 8b &lt;&lt;FIG:Fig15&gt;&gt;). Such variations require per-doublet averages to be considered. However, the subset of tomograms in the proximal principal region is small, and per-doublet averaging resulted in anisotropic 3D reconstructions. An even larger dataset is required to resolve structures with such a specific and complex distribution pattern. We also calculated an average of subvolumes from the nine doublets near the beginning of axonemes (within 2 microm) and found no significant difference when compared to the overall consensus average, suggesting that the sperm-specific features are established very close to the basal region of the flagella (Extended Data Fig. 8c,d &lt;&lt;FIG:Fig15&gt;&gt;). Together, these data highlighted the overall consistency of the axoneme structure along the flagella.\nOverall, our per-doublet averages showed that the distributions of various sperm-specific features for both the RS complexes and the N-DRCs follow distinct patterns, such that every doublet is decorated by a unique combination of non-motor proteins.\n\nA distinct asymmetric pattern in human sperm\nWe next examined whether the unique outer doublet features observed in mouse sperm were also conserved in other mammalian sperm. We collected tilt series of intact human sperm without milling, focusing on the thinner principal piece of the flagella (Extended Data Fig. 9a,b &lt;&lt;FIG:Fig16&gt;&gt; and Supplementary Figs. 1 and 2). We then calculated consensus averages of doublet microtubules and the central pair complex (23 A and 31 A at FSC = 0.143; n = 6,613 and 2,365 subvolumes, N = 56 and 59 tomograms, respectively) (Fig. 6a,b &lt;&lt;FIG:Fig6&gt;&gt;). These consensus averages of human sperm were similar to the ones from mice, with the notable exception that the relative occupancy of the barrel between RS1 and RS2 was much lower in human sperm. Using the per-doublet processing approach described above, we then calculated averages for each of the nine doublets individually. Although the per-doublet averages from human sperm were noisier than those from mouse sperm due to greater sample thickness (&gt;400 nm versus ~330 nm as shown in Extended Data Fig. 9c-e &lt;&lt;FIG:Fig16&gt;&gt;), they were sufficient to identify the sperm-specific features (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). In particular, the 5-6 bridge, RS2-RS3 cross-linkers and RS3 scaffolds show similar asymmetric distributions between human and mouse sperm. However, we observed that the barrel density only exists in four of nine doublets, in contrast to seven of nine doublets in mouse sperm axonemes (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). This is consistent with the lower occupancy of barrel densities in our consensus average (Fig. 6a &lt;&lt;FIG:Fig6&gt;&gt;). In particular, doublets 2-5 appear to be different in human and mouse axonemes in terms of the presence of the barrel, while the rest of the doublets are similar. We also examined the N-DRC from each doublet and found five distinct structures like those in mouse sperm (Extended Data Fig. 9f &lt;&lt;FIG:Fig16&gt;&gt;). In summary, our data indicate that the asymmetric architecture of axonemes is a general feature of mammalian sperm axonemes, although there are intriguing variations of distribution for the barrel in human and mouse sperm\nFig. 6\nHuman sperm have a different distribution of barrels compared to that in mouse sperm.\na, A consensus average map of the RS for human sperm axonemes. The barrel density appears to have lower occupancy than the RSs (indicated by the red arrowhead), which is not the case in mouse sperm shown in Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;. b, The consensus average map of the central pair complex for human sperm axonemes (gray) is overlaid with one from mouse sperm axonemes (blue). c, Densities corresponding to RSs from the per-doublet averages are shown around a schematic of the (9 + 2) axonemes. Common features among the RSs are colored in gray, while the barrel, the RS2-RS3 cross-linker and the RS3 scaffold are highlighted in cyan, red and orange, respectively.\n\nDiscussion\nOur in situ tomography studies of mouse and human sperm revealed a large ensemble of macromolecular complexes in their native cellular environment. In particular, we observed various sperm-specific features in the MIPs, RSs and N-DRCs that were not observed in mammalian respiratory cilia and non-mammalian sperm. Furthermore, we reconstructed the entire axoneme using cellular contextual information and uncovered the asymmetric architecture of the mammalian sperm axoneme, where every microtubule-based structure is decorated by a unique set of non-motor proteins (Fig. 7a,b &lt;&lt;FIG:Fig7&gt;&gt;). As these non-motor proteins regulate dynein activities based on previous studies, we propose that the asymmetries of non-motor protein complexes could modulate the sliding of the nine doublets individually to shape the species-specific asymmetric waveforms observed for mammalian sperm, as discussed below\nFig. 7\nModel: every outer doublet is surrounded by a different set of regulatory complexes in mammalian sperm.\nSchematic of the (9 + 2) axonemes of mouse (a) and human (b) sperm. The doublets are numbered, and the sperm-specific regulatory complexes are labeled for each of the nine outer doublets. In particular, the components from the RSs are shown for each doublet and the N-DRCs are colored differently depending on the extra density (as in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt;-6 &lt;&lt;FIG:Fig6&gt;&gt;). Note that only the barrel distribution is different in mouse and human sperm axonemes. c, RSs from different doublets interact with specific stripes of protrusions of the central pair complex. Our multibody refinement is consistent with the sliding hypothesis, and potential longitudinal movements of dmt1 are indicated by the two red arrows. The model is analogous to 'nine train moving on nine tracks'. d, Schematics showing gradual accumulations of offsets of periodic structure units between two filaments in a curved axoneme (left) and consistent offset between doublet 5-doublet 6 (right). e, The constant offset within tomograms and among tomograms (N = 63) would be consistent with limited horizontal bending in sperm flagella.\n\nSperm-specific structures added mechanical couplings\nMammalian sperm flagella are generally much longer and wider than flagella from unicellular organisms or respiratory cilia (lengths of &gt;45 microm versus ~10 microm and diameters of &gt;0.5-1 microm versus ~0.3 microm). Sperm axonemes are also surrounded by additional subcellular structures, such as outer dense fibers and mitochondrial and fibrous sheaths, which likely present additional mechanical challenges during rhythmic beatings. Previous studies also suggested that larger bending torques are associated with mammalian sperm compared to other cilia&lt;&lt;REF:Lindemann-1996-34-258&gt;&gt;, but it has remained unclear how sperm axonemes have evolved specific mechanisms to withstand the additional mechanical stress. Our averages of in situ cryo-ET revealed many additional non-motor proteins that cross-link the known axonemal components. We propose that these additional proteins function to strengthen the mechanical rigidity of the corresponding components to accommodate higher mechanical requirements of sperm axonemes.\nThe microtubules themselves must be able to withstand vigorous bending, and the seam is the weakest point&lt;&lt;REF:Kikkawa-1994-127-1965&gt;&gt;,&lt;&lt;REF:Zhang-2015-162-849&gt;&gt;. Our data revealed the most extensive MIP interaction network in microtubules observed in axonemes to date. The proteins between the A9 and A10 protofilaments could stabilize the lateral interfaces within the seam. Also, the proteins that form the striations inside the B tubule could cross-link the tubulins within B2-B6 protofilaments and also couple these protofilaments laterally along the helical pitch. We also observed extensive filamentous structures within the A tubule that likely provide additional mechanical stability. Tektin-1 to tektin-4 have been known to assemble into three-helix bundles that pack along one another laterally inside the microtubule doublets of bovine tracheal cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; and likely compose a part of the filamentous densities observed in sperm. Mammalian sperm also contain an additional tektin (tektin-5; Supplementary Table 1)&lt;&lt;REF:Firat-Karalar-2014-127-4128&gt;&gt;, which is a candidate for some of the additional sperm-specific densities (as shown in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;), although the assignment requires confirmation with higher-resolution reconstructions.\nWe also observed unique sperm-specific densities on the exterior of the microtubules, specifically the barrel and RS2-RS3 cross-linker between the three RSs, the RS3 scaffolds and extra densities in the N-DRC. The RSs were previously observed to tilt relative to the microtubule&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. Additional connections between the three RSs and the scaffolds at the bases would integrate them together into a more rigid unit. Another possibility is that the coupling could lead to coordinated movement of the three RSs. The N-DRCs regulate the sliding between the neighboring doublets and prevent splaying of axonemes&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;. Extra densities linking to the microtubules could improve the stability of N-DRCs under higher mechanical stress. Together, the additional protein complexes in mammalian sperm would help to maintain the geometric integrity of the (9 + 2) microtubule configuration of the axoneme under mechanical stress during vigorous beating motions.\n\nThe arrangement of the doublets and the central pair complex\nPrevious studies suggested that the central pair complex can twist radially relative to the nine outer doublets in Chlamydomonas flagella&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;. Our observation of densities corresponding to the nine doublet microtubules in the average of the entire (9 + 2) axoneme can exclude the possibility of such free twisting in mammalian sperm as the doublet densities would be smeared by averaging (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Interestingly, we observed a cleft between the two halves of RS1 and RS2 and holes in RS3 in mammalian sperm axonemes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), in contrast to the flatter surface of the RS heads from Chlamydomonas flagella&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). While the flat surfaces may enable twisting in Chlamydomonas&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;, the complementary shapes of RSs and protrusions from the central pair complex in mammalian sperm may restrict such radial movements (as shown in Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Fixation of radial positions of the nine doublets also orients each of the RS complexes in proximity to unique stripes of densities of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). This arrangement could allow functional specialization and divergent evolution of each doublet microtubule, such as the distinct sets of sperm-specific features in the nine doublets.\nOur multibody refinement suggests the existence of heterogeneities in the longitudinal offsets between doublets and the central pair complex in the randomly sampled axonemes that were combined in the averages (Supplementary Video 2). This observation is consistent with the sliding hypothesis for axonemal bending, in which active dyneins generate displacement between the neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;. Such movement would also lead to displacement of the doublets relative to the central pair complex along the longitudinal axis of axonemes, as though there were nine trains (doublets) moving along nine tracks (central pair protrusions) (Fig. 7c &lt;&lt;FIG:Fig7&gt;&gt;).\nThe 5-6 bridge is a sperm-specific feature that appears to be conserved between sea urchin&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt; and mammalian sperm. Previous studies pointed out that, for two parallel inelastic microtubule filaments, bending would lead to gradual accumulation of longitudinal offsets between them if they were at different radii of the bend (Fig. 7d &lt;&lt;FIG:Fig7&gt;&gt;)&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. As an estimate based on the reported curvature of mouse sperm&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;, the offsets between neighboring doublets could differ by as much as Delta~28 nm within one 2 microm-long tomogram (details in the Methods). More importantly, the initial offset of each tomogram varies depending on how much sliding has happened upstream or downstream of the imaged area of the flagella. On the other hand, the resolved periodic features in both doublets 5 and 6 suggest that there is a consistent offset between these two doublets (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;), not just within each tomogram but also among the 63 tomograms that contribute to the average. Due to the nature of random sampling of our imaging areas of different cells (N = 63 tomograms), this could happen if there is generally limited bending along the direction parallel to the plane of these two filaments (Fig. 7e &lt;&lt;FIG:Fig7&gt;&gt;). The bundling of these two doublets can create a filament stiffer than that of a single doublet, with distinct elastic properties or bending propensities in different directions. Such asymmetries of mechanical properties within the nine doublets could also contribute to asymmetric waveforms.\n\nAsymmetries of regulators may lead to asymmetric beatings\nOur in situ data and processing strategies based on the contextual information revealed that the axoneme itself, which appeared to have 'pseudo-ninefold symmetry' in classical TEM images&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;, is highly asymmetric at the molecular level. As the axoneme is the underlying engine that drives the flagellar beating motion, such asymmetries in structure could lead to asymmetric beating waveforms.\nFurthermore, the asymmetries lie mostly in the mammalian sperm-specific non-motor protein complexes, including the RSs and N-DRCs. These complexes are well-established regulators of dynein motor activities, and defects in individual protein components can lead to irregular beating&lt;&lt;REF:Witman-1978-76-729&gt;&gt;,&lt;&lt;REF:Huang-1981-88-80&gt;&gt;,&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;,&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;. In mammalian sperm, we show here that each of the nine doublet microtubules is decorated by a unique composition of sperm-specific regulators. We speculate that the distinct molecular composition could lead to differences in the sliding speeds or bending forces for each of the doublet pairs (Fig. 7 &lt;&lt;FIG:Fig7&gt;&gt;). The non-equivalent forces could lead to a deviation from the single plane of beating characteristic of more symmetric axonemes. Thus, we hypothesize that the non-uniform distribution of sperm-specific RS and N-DRC regulators are important for asymmetric and non-planar beating. Additionally, previous studies showed that human and mouse sperm have different swimming waveforms (movies in refs. &lt;&lt;REF:Babcock-2014-12-67&gt;&gt;,&lt;&lt;REF:Gadelha-2020-6-eaba5168&gt;&gt;). Our studies suggest that, although the RS barrels are conserved in human and mouse, their distribution varies and the variations could create diverse sperm swimming behaviors.\nThis study has revealed mammalian sperm-specific structures within the microtubules, RSs and N-DRCs that are absent in sea urchin and zebrafish sperm&lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Lin-2018-360-eaar1968&gt;&gt;. The still unknown proteins likely arose to serve functions required by natural fertilization in mammals. Sperm from different species are cast into a foreign environment and selected for their ability to reach and fertilize an egg. Sea urchin and zebrafish sperm swim freely in water, whereas mammalian sperm swim in a thin layer of viscous liquid on uneven surfaces of female reproductive tracts&lt;&lt;REF:Suarez-2016-363-185&gt;&gt;. The viscous environment brought more mechanical challenges to the axonemes and hence the microtubule filaments inside. In addition, asymmetric and non-planar waveforms of mammalian sperm could be beneficial to navigate around 3D obstacles in reproductive tracts. Furthermore, the dimensions and physical characteristics of reproductive tracts vary among different mammals; therefore, fine-tuning the underlying molecular features to produce specialized waveforms is likely under evolutionary selection. In the future, systematic genetic and proteomic analyses of mammalian sperm-specific proteins would be valuable to connect sperm-specific axonemal structures with their functions in sperm motility and reproduction.\n\nMethods\nAll chemicals were purchased from Sigma-Aldrich unless otherwise noted.\n\nSample preparation\nMouse sperm were collected from 10-16-week-old C57Bl/6J mice based on a published protocol&lt;&lt;REF:van der Spoel-2002-99-17173&gt;&gt;. Briefly, the sperm were stripped from vasa deferentia by applying pressure to the cauda epididymis in 1x Krebs buffer (1.2 mM KH2PO4, 120 mM NaCl, 1.2 mM MgSO4*7H2O, 14 mM dextrose, 1.2 mM CaCl2*2H2O, 5 mM KCl, 25 mM NaHCO3). The sperm were washed and resuspended in ~100 microl Krebs buffer for the following experiments.\nHuman sperm cells were collected by masturbation from healthy donors and visually inspected for normal morphology and motility before use. Spermatozoa were isolated by the swim-up procedure in HTF or HS solution as previously described&lt;&lt;REF:Skinner-2019-9-e3182&gt;&gt; and then concentrated by centrifuging for 5 min at 500g. The supernatant was reduced to 100 microl, and the cells were resuspended.\nAll experimental procedures using human-derived samples were approved by the Committee on Human Research at the University of California, Berkeley, under IRB protocol number &lt;&lt;REF:???-?-?-?&gt;&gt; -06-5395.\n\nGrid preparation\nEM grids (Quantifoil R 2/2 Au 200 mesh) were glow discharged to be hydrophilic using an easiGlow system (Pelco). The grid was then loaded onto a Leica GP cryo plunger (pre-equilibrated to 95% relative humidity at 25 degreesC). The mouse sperm suspension was then mixed with 10-nm gold beads (Electron Microscopy Sciences, 25487) to achieve final concentrations of 2-6 x 106 million cells per ml. For the demembraned mouse sperm, Triton X-100 was added to a final concentration of 0.1% in the final suspension. The human sperm suspension was mixed with 10-nm gold beads to achieve final concentrations of 0.5-2 x 106 million cells per ml. Next, 3.5 microl of sperm mixture was added to each grid, followed by incubation for 15 s. The grids were then blotted for 4 s and plunge-frozen in liquid ethane.\n\nCryogenic focused ion beam milling\nCryogenic focused ion beam milling was performed either manually using an Aquilos cryo-FIB-SEM microscope (Thermo Fisher Scientific) or automatically using an Aquilos II cryo-FIB-SEM microscope (Thermo Fisher Scientific). A panoramic SEM map of the whole grid was first taken at 377x magnification using an acceleration voltage of 5 kV with a beam current of 13 pA and a dwell time of 1 micros. Targets with appropriate thickness for milling were picked on the map. A platinum layer (~10 nm) was sputter coated, and a gas injection system was used to deposit the precursor compound trimethyl(methylcyclopentadienyl)platinum(IV). The stage was tilted to 15-20degrees, corresponding to a milling angle of 8-13degrees relative to the plane of grids. FIB milling was performed using stepwise decreasing current as the lamellae became thinner (1.0 nA-30 pA; final thickness, ~300 nm). The grids were then stored in liquid nitrogen before imaging.\n\nImage acquisition and tomogram reconstruction\nTilt series of mouse sperm were collected on a 300-kV Titan Krios TEM (Thermo Fisher Scientific) equipped with a high-brightness field emission gun (xFEG), a spherical aberration corrector, a Bioquantum energy filter (Gatan) and a K3 Summit detector (Gatan). The images were recorded at a nominal magnification of 19,500x in super-resolution counting mode using SerialEM&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;. After binning over 2 x 2 pixels, the calibrated pixel size was 3.53 A on the specimen level. For each tilt series, images were acquired using a modified dose-symmetric scheme between -48degrees and 48degrees relative to the lamella with 3degrees steps and grouping of two images on either side (0degrees, 3degrees, 6degrees, -3degrees, -6degrees, 9degrees, 12degrees, -9degrees, -12degrees, 15degrees...). At each tilt angle, the image was recorded as movies divided into eight subframes. The total electron dose applied to a tilt series was 100 e- A-2. The defocus target was set to be -4 to -7 microm.\nTilt series of human sperm were recorded at a nominal magnification of 33,000x in super-resolution counting mode. After binning over 2 x 2 pixels, the calibrated pixel size was 2.66 A on the specimen level. The total electron dose applied to a tilt series was 100 e- A-2. For each tilt series, images were acquired using a bidirectional scheme between -48degrees and 48degrees relative to the lamella starting from either 0degrees or 21degrees, with an incremental step of 3degrees. The defocus target was set to be ~2-5 microm. At each tilt, the image was recorded as movies divided into eight subframes.\nAll movie frames were corrected with a gain reference collected in the same EM session. Movement between frames was corrected using MotionCor2 without dose weighting&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;. Initial reconstructions of all tilt series were performed using AreTomo&lt;&lt;REF:Zheng-2022-6-100068&gt;&gt;, and these 3D tomograms were examined to exclude tomograms with bad target tracking, crystal ice, aberrant defocus, incomplete axonemes (less than five doublet microtubules caused by FIB-SEM milling) and unfavorable orientations (perpendicular to the tilt axis). Alignment of selected tilt series and tomographic reconstructions were then performed using Etomo&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. The aligned tilt series were then CTF corrected using TOMOCTF&lt;&lt;REF:Fernandez-2006-106-587&gt;&gt;, and the tomograms were generated using TOMO3D&lt;&lt;REF:Agulleiro-2015-189-147&gt;&gt; (bin2; pixel size, 7.06 A).\nIn total, we started with 24 milling grids of mouse sperm and obtained 200 lamellae. Tilt series with no crystal ice were kept and processed further. In some cases, parts of the (9 + 2) axoneme was milled away and we only processed the ones with at least five doublets and also enough space to include the full central pair complex for subvolume averaging. In the end, the final reconstructions of the consensus averages were from 69 usable tomograms. The per-doublet averages were based on subsets of the 69 tomograms (58-64 tomograms). For the human sperm cryo-ET dataset, the final reconstructions of the consensus averages are from 65 tilt series with the flagellar orientations within ~30degrees of the tilt axis. Note that the final tomograms used for different reconstructions vary as tomograms may be discarded due to poor alignment or may lack certain doublets.\n\nSubvolume averaging\nSubsequent subvolume extraction, classification and refinement were all performed using RELION3 (ref. &lt;&lt;REF:Bharat-2016-11-2054&gt;&gt;) (schematic workflow in Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Briefly, subvolumes from the doublets were manually picked every 24 nm and extracted with a box size large enough to accommodate a complete 96-nm-repeating unit (pixel size, 7.06 A; box size, 180 pixels; dimension, 127.08 nm). Initially, subvolumes were aligned to a map of Tetrahymena doublets (EMD-9023) low-pass filtered to 80 A, and the resulting map was used as the reference for further processing. Manual curation of the data was performed to check the alignment accuracy and data quality for each tomogram. Supervised 3D classification on RSs gave rise to four class averages of the 96-nm-repeating units at four different registries, staggered from one another by 24 nm. The pixel view in IMOD was used to determine the coordinates corresponding to the base of RS2 in all four class averages, and all subvolumes were re-extracted and recentered to the same point. All subvolumes were combined and aligned to one reference, and duplicate subvolumes were removed based on minimum distance (&lt;40 nm). The remaining subvolumes were aligned to yield the consensus average for all nine doublets. These subvolumes were later remapped and sorted to calculate per-doublet averages (the script is also available below). The MIPs inside microtubule doublets repeat every 48 nm. Subvolumes of the 96-nm-repeating units were recentered on MIP features that repeat every 48 nm and refined with a mask focusing on the microtubules only.\nTo generate an average for the central pair complex, subvolumes were picked and extracted from the central pair complex every 16 nm (pixel size, 7.06 A; box size, 160 pixels; dimension, 112.96 nm) (Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt;). Refinement of all subvolumes to an average of the sea urchin central pair (EMD-9385) resulted in a central pair with only 16-nm-repeating features. The alignment parameters were modified to reset all translations (x, y, z) to zero and then used for a second round of refinement with local search only. Focused classification was then performed on the microtubule-associated proteins on the C1 microtubule to separate the two populations of subvolumes (~50% each) corresponding to the central pair complex with 32-nm periodicity and an offset of 16 nm between them. The subvolumes were then recentered and extracted on the same protein features for the two class averages. All subvolumes were aligned to the same reference using local searches. Duplicate subvolumes were removed based on the minimum separating distance (&lt;30 nm). The remaining subvolumes were refined to generate the consensus average for the central pair complex.\nSubvolumes large enough to include all nine doublets were re-extracted, centering on the aligned subvolumes of the central pair complex (pixel size, 21.18 A; box size, 160 pixels; dimension, 338.88 nm). These subvolumes were averaged without further alignment and used to re-extract subvolumes corresponding to individual doublets. Subvolumes corresponding to a particular doublet were aligned and remapped back to the tomograms in three dimensions. However, these subvolumes do not have the correct alignment for the 96-nm-repeating features, but their centers nevertheless trace the axis of that microtubule doublet accurately when they are remapped in three dimensions (using the script available below). The 96-nm periodic subvolumes curated above to generate the consensus average of 96-nm repeats were then sorted. The subvolumes corresponding to a particular doublet were then aligned and averaged.\nFor multibody refinement (schematic workflow in Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;), the subvolumes corresponding to the specific RS-central pair interface were re-extracted based on the subvolumes of 96-nm units of individual doublet microtubules. The particles were recentered at the junction between the head and the stalk of RS2 to include enough features from both the doublets and the central pair complex. The RSs were then aligned with a mask. This mask and a mask covering the central pair complex were used for the multibody analysis implemented in RELION3 (ref. &lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;). Briefly, separate refinement of the RSs and central pair protrusions provided two sets of alignment parameters: three translational shifts (x, y, z) and three Euler angles required to align one subvolume to each reference. Thus, 12 parameters can be used to remap the two references back to each raw subvolume, and the spatial relationship of the two rigid bodies in each subvolume could be determined by these 12 parameters. Principal-component analyses essentially reprojected the original 12-dimensional data in a new 12-dimensional space with 12 new orthogonal eigenvectors. These 12 eigenvectors could be mathematically determined so that they represent decreasing variations of the entire data along the individual eigenvectors. Our analyses suggest that the first eigenvector or axis is parallel to the axoneme axis and the variations along this axis account for 40-50% of the total variations. When all subvolumes were divided into ten groups based on their projection values on the first axis (10% lowest, 10-20% lowest, ..., 90-100% highest). Each group was then represented by a snapshot depicting the averaging projection values of the group members, and ten of these snapshots were morphed to generate the animation (Supplementary Video 1).\nThe resolutions for maps were estimated based on FSC values of two independently refined half datasets (FSC = 0.143). Local-resolution maps for the consensus averages of doublets and the central pair complex of both human and mouse sperm were calculated by blocres in Bsoft (Supplementary Fig. 2). These local-resolution maps represent relative differences in resolution across the maps, but the absolute values may not be exact. IMOD was used to visualize the tomographic slices&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. UCSF Chimera was used to manually segment the maps for various structure features, and these maps were colored individually to prepare the figures using UCSF ChimeraX&lt;&lt;REF:Pettersen-2004-25-1605&gt;&gt;-&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;.\n\nEstimation of accumulated offsets in axonemes\nThe bending curvature of mouse sperm could be as large as 2 x 105 m-1 or 0.20 microm-1 based on literature (OD = 5 microm = 5,000 nm)&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;. The distance between the neighboring doublets is 72 nm (measured in the average of the axoneme shown in Fig. 2d &lt;&lt;FIG:Fig2&gt;&gt;, OB = 5,072 nm). If AB and CD represent 96-nm repeats from doublet 1 and doublet 2, respectively, offset Delta = CE =~ OC x (/_COD - /_AOB) = 5,000 x (96 / 5,000 - 96 / 5,072) = 1.4 nm.\nNote that the Delta represents the additional offset per 96-nm repeat. In a tomogram that contains an axoneme of ~2 microm, if the offset between the first pair of 96-nm repeats is 0 nm, the offset of the 20th pair is 28 nm. Note that this initial offset (0 nm) is tomogram specific depending on how much sliding has happened upstream or downstream of the imaging area and the imaged cell.\n\nReporting summary\nFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.\n\nOnline content\nAny methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41594-022-00861-0.\n\nSupplementary information\n\nSupplementary Information\nSupplementary Figs. 1 and 2 and Table 1\nReporting Summary\nPeer Review File\nSupplementary Video 1\nThe structure of the RS barrel.\nSupplementary Video 2\nThe movie contains ten frames, and each frame represents an average of one-tenth of the total populations distributed along the first component of the principal-component analysis. The different frames indicate that the RSs and the central pair complex can come together at different longitudinal offsets. The interface between doublet 6 and the central pair protrusion 1b is used as an example here.\nSupplementary Code\nSupplementary information\nThe online version contains supplementary material available at 10.1038/s41594-022-00861-0.\n\nExtended data\n\nExtended Data Fig. \nThe workflow of FIB-SEM/cryo-ET and representative images.\na, Schematic of EM grids with vitrified mouse sperm. An example for where the windows were milled by FIB-SEM is denoted by two yellow boxes. b, SEM (left) and FIB (right) images of a lamella are shown (scale bar: 5 microm, N = 200 lamellae). c, A low-magnification montage and its zoom-in view of a lamella acquired using TEM (scale bar: 1 microm, N = 200 lamellae). Axonemes and mitochondria are indicated by white arrowheads and an arrow, respectively. d, A representative image of non-milled sperm axoneme recorded by 300 kV Titan Krios TEM at a total dose of 3 e-/A2 (scale bar: 100 nm, N = 10 tilt series, not processed further). e, A representative image of milled lamella (thickness ~300 nm) is shown (scale bar: 50 nm, N = 69 tilt series). The zoom-in region in f is outlined (the red rectangle). f, Individual protofilaments of microtubules and double-bilayer membranes of the mitochondria are indicated by yellow arrows and arrowheads. g, Fourier transform of the image in e is shown and the upper limit of ctf fitting when using CTFFIND4 is indicated (at the pixel size of 3.53 A, N = 69 zero-tilt images). h, i, Two sections of 3D tomograms are shown. Outer arm dyneins (every 24 nm) anchored on doublet microtubules are indicated in h (red arrowheads). Protrusions (every 32 nm) extending from the singlet microtubule are indicated in i (black arrowheads). Periodic patterns of three radial spokes on the doublet microtubules (every 96 nm) are grouped and indicated in i (black brackets) (scale bar: 50 nm, N = 69 tomograms).\n\nExtended Data Fig. \nSchematics of data processing.\na, A schematic of the processing workflow of 96 nm-repeating units of doublets to achieve the consensus average combining data from all nine doublets. An example of remapping of the centers of the final subvolumes in three dimensions is shown. b, A schematic of the processing workflow of 32 nm-repeating units of central complex to achieve the subvolume average. Recentering of subvolumes of the central pair complex (every 32 nm) on the doublet 1 is shown. Remapping of these subvolumes helps to trace the trajectory of doublet 1 in three dimensions. The centers of these doublet 1 subvolumes are shown in the same tomogram as in a. c, A schematic of the workflow of multi-body refinement.\n\nExtended Data Fig. \nMIPs network in A-tubule of doublet microtubules.\na, Four cross-section views of the A tubule in the doublet microtubule are shown. The MIPs network is colored in orange. Three modes of interactions between the MIPs and the protofilaments are observed (see text). The protrusions from the filamentous core are indicated by empty red arrowheads and dashed cyan ovals, respectively. b, Six cross-section views of the overlay of the average of the 48-nm repeat of doublets of mouse sperm (orange, this study) and the 48-nm repeat of doublets of bovine trachea cilia (blue, EMD-24664)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;. The contour levels of the two maps are adjusted so that the densities of microtubules are matched. The microtubule seam, A9 and A10 protofilaments are labeled in all views. Note there are additional densities near A9-A10 region in the mouse sperm axoneme (black circles). We observed density striations in the B-tubule in some cross sections. c, An orthogonal view of the B-tubule striations along the helical pitch of the microtubule at a high and a low threshold, respectively. Note the striations are separated by 8 nm and they cover the intradimeric interface between the alpha- and beta-tubulins based on the model for the bovine doublet (PDB: 7rro)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;.\n\nExtended Data Fig. \nComparisons of dyneins and radial spokes of mouse sperm axonemes to published equivalent structures from other motile cilia.\na, Comparison of outer arm dyneins in mouse sperm axoneme (this study) and reported maps for those from sea urchin sperm (EMD-8835 and EMD-8836)&lt;&lt;REF:???-?-?-?&gt;&gt;. b, Comparison of inner arm dyneins in mouse sperm axoneme (this study) and published maps for those from sea urchin sperm (EMD-8837 and EMD-8838)&lt;&lt;REF:???-?-?-?&gt;&gt;. Note the barrel densities between RS1 and RS2 only exist in the mouse sperm are indicated (dashed ovals). c, Comparison of radial spokes of mouse sperm axoneme (this study) and the ones from motile cilia of Chlamydomonas (EMD-9768&lt;&lt;REF:Owa-2019-10-1143&gt;&gt; and EMD-22475&lt;&lt;REF:Gui-2021-28-29&gt;&gt;) and Tetrahymena (EMD-9023)&lt;&lt;REF:Song-2020-17-201&gt;&gt;. Note the RS3 stump in Chlamydomonas is much shorter than the others and the head of RS3 from Tetrahymena has a smaller surface area. d-f, The doublet average of mouse sperm axoneme is low-pass filtered to 50 A and viewed from different angles. Note this resolution is lower than published axoneme maps (for example 34 A for EMD-5950 shown in Fig. 1 &lt;&lt;FIG:Fig1&gt;&gt; for comparison)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. The extensive network of MIPs (d), extra densities at N-DRC (e) and radial spokes (f) remain apparent while comparing to human respiratory cilia, suggesting the presence of novel densities in our maps are not due to higher resolutions, but most likely the presence of additional proteins.\n\nExtended Data Fig. \nThe central pair complex in mouse axoneme possesses more MIPs but the overall shape is similar to the one from sea urchin sperm.\na, Two longitudinal views of the MIPs inside the two singlet microtubules are shown. Periodic features (every 32 nm) are indicated (red empty arrowheads). b, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from sea urchin sperm (EMD-9385) and they are colored in grey and cyan, respectively&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. Note most of the protrusions are very similar and the sea urchin-specific protrusion 1f is indicated in the middle panel (red arrowhead). c, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from Chlamydomonas (EMD-31143)&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. Note the maps are aligned based on the similar C1 protrusions 1d. Other protrusions are generally less similar (1a, 1f and 2d) and there appears to be an 8-nm shift for the C2 protrusions. The Chlamydomonas C2 microtubule was recently shown to possess two possible registers with 8-nm longitudinal differences&lt;&lt;REF:Han-2022-29-472&gt;&gt;. d, The spatial relationship between the nine radial spokes and the central pair complex are determined by multibody refinement and the 'average positioning' of these two rigid bodies are shown (RS-CP1 denotes the interface between radial spokes from doublet 1 and the central pair.\n\nExtended Data Fig. \nThe 5-6 bridge of mouse sperm axonemes.\na-d, Four views of the composite map for 5-6 bridge in mouse sperm axonemes combining maps for local refinement of the doublet 5, doublet 6 and the bridge. The inner arm dyneins, radial spokes, the barrel, N-DRC and extra bridge densities are colored in pink, different shades of cyan, light blue, green and blue, respectively. Note that the radial spokes which repeat every 96 nm on doublet 5 and doublet 6 are resolved and there appears to be a ~20 nm offset between their longitudinal registers, as indicated in the staggering distance of RS1 and RS2 from doublet 5 and doublet 6 in d. e, f, Two cut-in views of a and c are shown. Bridge-specific densities are highlighted in blue. These densities extend from the inner arm dyneins of doublet 5 towards the microtubule of doublet 6. Direct contacts of the bridge-specific densities to doublet 6 is indicated by the solid red arrowhead while densities in close proximity of the base of radial spoke 2 in doublet 6 are indicated by the empty red arrowhead. g, h, Subvolumes for doublet 1 was aligned and equivalent views of doublet 1 and 2 are shown (as in panel a and b). The expected positions for dyneins and radial spokes on doublet 2 are circled by dashed and solid red ovals. These 96 nm-repeating structures on doublet 2 are not resolved even after local refinement, suggesting the offsets between doublet 1 and doublet 2 are not consistent. The microtubule from doublet 2 is nevertheless partially resolved since variant longitudinal mismatching would not smear a tube of 'continuous densities' compared to 'discrete densities' like dyneins and radial spokes through averaging.\n\nExtended Data Fig. \nThe inner arm dyneins in doublet 5 and doublet 9 differ from the consensus average in mouse sperm axonemes.\na, An overlay of inner arm dyneins from doublet 5 and the concensus average. The assigned inner arm dyneins, N-DRC and 5-6 bridge densities from the doublet 5 are colored in pink, green and blue, respectively while the densities for the consensus average are in grey. Different inner arm dyneins are named according to the nomenclature defined by studies of Chlamydomonas flagella. The shifted positions of dynein e and dynein g are indicated by the two yellow arrows. b, An overlay of inner arm dyneins from doublet 9 and the consensus average were blue and grey, respectively. Note dynein b in doublet 9 is not resolved.\n\nExtended Data Fig. \nLongitudinal consistency of doublets in mouse sperm axonemes.\na, Schematic of a mouse sperm flagella with the basal region (&lt; 2 microm), midpiece and principal piece highlighted in the top panel. Subvolume averages of doublet 2 and doublet 7 from the midpiece and the principal piece are shown. The shapes of densities for the scaffolds at the base of radial spoke 3 appear different in doublet 2 and are circled. Additional densities at the base of radial spoke 1 in the doublet 7 in the midpiece region are observed and circled. b, Four xz slices of tomograms in different regions of the mouse sperm flagella are shown. Outer dense fibers that are in close proximity to the doublets and the ones with gaps in between are indicated (red and blue arrowheads, respectively) (scale bars: 100 nm). c, Two xy slices of two 3D tomograms of the basal region of mouse sperm axoneme are shown. The singlet microtubules of the central pair complex extend further into the cell bodies and the beginning of the doublet microtubules are indicated by the arrows and arrowheads, respectively (scale bars: 200 nm). d, Densities corresponding to different protein complexes from subvolume averages of the basal region (6 tomograms, N = 670 subtomograms) and the consensus average are overlaid and colored in blue and grey, respectively. Note the sperm-specific features in N-DRC and radial spokes exist in the average of doublets in the basal region and are highlighted using empty red arrowheads.\n\nExtended Data Fig. \nCryoET data of human sperm without milling and asymmetric distribution of sperm-specific features in N-DRCs in human sperm.\na, A representative xy slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 1, scale bar: 100 nm). Although the A- and B-tubule of the microtubule doublets are discernible, other features like axonemal dyneins and radial spokes are not clear in this slice (in contrast to Extended Data Fig. 1h-i &lt;&lt;FIG:Fig8&gt;&gt;). b, A representative xz slice of the same tomogram of human sperm (pixel size=10.648 A, image thickness: 5, displayed in IMOD) (scale bar: 100 nm). c, A representative xz slice of a tomogram of mouse sperm lamella (pixel size=14.12 A, image thickness: 10, displayed in IMOD). The red bar indicates how we measure the thickness of the lamella (scale bar: 100 nm). d, A representative xz slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 100, displayed in IMOD). Gold beads on the top and bottom surfaces are circled and the red bar indicates how we measure the thickness of the ice near the cell. The top and bottom membrane of the sperm cells is not resolved due to the missing wedge so the actual thickness is larger than the measured thickness of ice in the vicinity (scale bar: 100 nm). e, A dot plot showing the thickness of lamellae of mouse sperm and ice near human sperm as a lower bound estimate of the actual cells. The means and standard deviations are indicated (for N = 69 mouse sperm tomograms and N = 59 human sperm tomograms). f, Densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of the (9 + 2)- axoneme. Common features among the N-DRC are colored in grey, while the unique features are highlighted in various colors.\n\nExtended data\nis available for this paper at 10.1038/s41594-022-00861-0.\n\nPeer review information\nNature Structural &amp; Molecular Biology thanks Girish Mali and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Carolina Perdigoto and Florian Ullrich, in collaboration with the Nature Structural &amp; Molecular Biology team. Peer reviewer reports are available.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Main\nEukaryotic flagella and motile cilia are whip-like organelles, the rhythmic beating of which propels unicellular eukaryotes through fluids, clears dust particles in respiratory tracts and enables the swimming of sperm cells of various species&lt;&lt;REF:Wallmeier-2020-6-77&gt;&gt;-&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;. Most flagella from protozoa to mammals share a conserved core structure, the axoneme, composed of nine doublet microtubules (doublets) arranged in a circle around a central pair complex of two singlet microtubules (the 9 + 2 configuration, Fig. 1a &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Ishikawa-2017-9-a028076&gt;&gt;,&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;. Dyneins, microtubule-based molecular motors anchored on the nine doublets, drive the relative sliding of neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;,&lt;&lt;REF:Summers-1971-68-3092&gt;&gt;. However, if all dyneins were active at once, forces around the circle of the nine outer doublets would be canceled and the axoneme would not bend&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. To produce rhythmic beating motions, non-motor protein complexes are required to regulate dynein activities across the axoneme structure&lt;&lt;REF:Satir-2014-64-1073&gt;&gt;,&lt;&lt;REF:Witman-1978-76-729&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;. The largest and most critical of these regulatory complexes are the radial spokes (RSs) that bridge the outer doublets to the central pair complex and the nexin-dynein regulatory complexes (N-DRCs) that cross-link neighboring doublets and regulate dynein activities across the axoneme structure\nFig. 1\nThe consensus average of nine doublets in mouse sperm possesses unique features in non-motor protein complexes including MIPs, N-DRCs and RSs.\na, Schematic of a cross-section view of the conserved (9 + 2) configuration of axonemes in motile cilia. One doublet is highlighted, and its associated motor and non-motor protein complexes are labeled. The dyneins, N-DRCs, MIPs and RSs are colored in pink, green, orange and cyan, respectively. b-d, Three views of the consensus subvolume average of nine doublets in mouse sperm axonemes. Different protein complexes are highlighted as described in a. c, The clefts in the top of RS1 and RS2 and the 'S'-shaped head of RS3 are indicated by the red arrowheads and dashed line, respectively. e, Schematic of the doublet microtubule with individual protofilaments labeled. Two cross-sections of the A tubule and one longitudinal view of the isolated MIP densities are shown. The MIPs are colored in orange. The protofilaments connecting to the MIPs and the periodicities for the connections are indicated. f,g, Comparison of densities of N-DRCs and RSs from mouse sperm flagella (this study) and human respiratory cilia (EMD-5950)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. f, Additional densities in the mouse sperm N-DRCs are highlighted (light red). g, The unique densities of a barrel and an RS2-RS3 cross-linker in the mouse sperm axoneme are highlighted in blue and red, respectively.\nFlagella from different cells display a wide variety of beating patterns, from the planar and symmetric waveforms observed in flagella of unicellular organisms and sea urchin sperm, to the various non-planar and asymmetric waveforms displayed by different mammalian sperm&lt;&lt;REF:Lindemann-2021-78-36&gt;&gt;. The structural and regulatory mechanisms underlying these different waveforms are poorly understood. Much of our current structural understanding of axonemes is derived from studies of Chlamydomonas and sea urchin sperm flagella using advanced cryo-electron tomography (cryo-ET) and image processing&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;. Apart from minor variations of the dyneins on a subset of the nine doublets, most of the other motor and non-motor protein complexes were found to be the same across the nine outer doublets. A unique bridge-like structure that cross-links two neighboring doublets is proposed to constrain the plane of bending&lt;&lt;REF:Bui-2009-186-437&gt;&gt;-&lt;&lt;REF:Dutcher-2020-375-20190153&gt;&gt;. The pseudo-ninefold symmetry and the bridge structure are thought to be important for generating equivalent beating amplitudes in the opposite directions, leading to planar and symmetric waveforms.\nComparable structural information for mammalian sperm, which display varied non-planar asymmetric waveforms&lt;&lt;REF:Muschol-2018-13-e0199678&gt;&gt;-&lt;&lt;REF:Babcock-2014-12-67&gt;&gt;, has lagged behind. A technical challenge in using modern cryo-electron microscopy to investigate mammalian sperm flagella is their thickness (&gt;500 nm), which is close to the upper limit for the widely used 300-kV transmission electron microscopes (TEMs). Recently, cryogenic focused ion beam-scanning electron microscopy (cryo-FIB-SEM) and cryo-ET have been applied to study in situ macromolecular structures in sperm axonemes from mammals&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, the limited data obtained in the previous study precluded processing strategies to analyze individual microtubule structures within the axonemes and also their spatial relationships in situ.\nHere, we combined cryo-FIB-SEM and in situ cryo-ET with data-processing strategies to study the contextual assembly of different microtubule-based structures within mouse and human sperm flagella. Our data provide the highest-resolution information to date for mammalian sperm axonemes. Furthermore, our data reveal non-motor protein complexes in mammalian sperm that are not found in axonemes of other mammalian cilia and non-mammalian sperm. We show that each of the nine outer doublets is unique with regard to the composition of regulatory complexes including RSs and N-DRCs. The distribution of regulators varies between mouse and human sperm. We propose that the asymmetric distribution of these regulatory complexes across the axoneme could contribute to the asymmetric and non-planar beating waveforms of various mammalian sperm.\n\nResults\n\nSperm-specific features revealed by subvolume averaging\nFreshly extracted mouse sperm were vitrified on electron microscopy (EM) grids and loaded into a cryo-FIB-SEM device to generate lamellae of ~300 nm in thickness (Extended Data Fig. 1a,b &lt;&lt;FIG:Fig8&gt;&gt;). The lamellae were then imaged using a Krios 300-kV TEM, and dose-symmetric tomographic tilt series (+/-48degrees) around the axoneme were then acquired (Extended Data Fig. 1c &lt;&lt;FIG:Fig8&gt;&gt;). Our images showed detailed molecular features including the double-bilayer membranes of the surrounding mitochondria and individual microtubule protofilaments (Extended Data Fig. 1e,f &lt;&lt;FIG:Fig8&gt;&gt;). Three-dimensional (3D) tomograms were reconstructed from the tilt series, which revealed repetitive axonemal dyneins and RSs along the outer doublets as well as periodic protrusions from the singlet microtubules of the central pair complex (Extended Data Fig. 1h,i &lt;&lt;FIG:Fig8&gt;&gt;). The periodicities of the RSs and central pair protrusions are ~96 nm and ~32 nm, respectively (Extended Data Fig. 1i &lt;&lt;FIG:Fig8&gt;&gt;), consistent with those described in Chlamydomonas and sea urchin sperm&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;.\nTo overcome the low signal-to-noise ratio of raw cryo-ET data, subvolume averaging of 96-nm-repeating units was used to reconstruct consensus density maps (Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Our consensus maps of periodic units from all nine doublets revealed robust signals for individual microtubule protofilaments and other associated protein complexes that repeat every 96 nm (24 A at Fourier Shell Correlation (FSC) = 0.143; n = 9,055 subvolumes; N = 69 tomograms) (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Supplementary Figs. 1 and 2).\nInside the A tubule of the outer doublet, we observed a filamentous density of microtubule inner proteins (MIPs) that is very similar to but more extensive than that recently assigned as Tektin filaments in bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; (Fig. 1b &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The densities of MIPs have a periodicity of 48 nm, consistent with that of previously studied Chlamydomonas flagella and bovine trachea cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;,&lt;&lt;REF:Ma-2019-179-909&gt;&gt;. We thus calculated a subvolume average of the 48-nm-repeating doublets, focusing on the microtubule only (18 A at FSC = 0.143; n = 18,153 subvolumes; N = 69 tomograms) (Extended Data Fig. 3 &lt;&lt;FIG:Fig10&gt;&gt;). The filamentous components have connections to 12 protofilaments of the A tubule in mouse sperm axonemes (except A4), in contrast to the more limited connections of the Tektin filaments to the A9-A13 and A1 protofilaments observed in bovine tracheal cilia (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and comparisons in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). We observed three different modes of interaction between the MIPs and the lumen of microtubules: (1) interaction with tubulins within a single protofilament, (2) connections to the inter-protofilament space across two neighboring protofilaments and (3) connections spanning multiple protofilaments (Fig. 1e &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 3a,b &lt;&lt;FIG:Fig10&gt;&gt;). Notably, the A9-A10 junction is where the microtubule seam of the A tubule is located&lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;, and we observed several sperm-specific densities spanning protofilaments A9 and A10 that are absent in the map of bovine respiratory cilia (Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;). In addition, we observed striated densities along the helical pitch of the microtubule inside the B tubule (Extended Data Fig. 3b,c &lt;&lt;FIG:Fig10&gt;&gt;). These striations are separated by 8 nm and cover the intradimeric interface between the alpha and beta tubulins. Together, our averages revealed sperm-specific MIPs that form an extensive interaction network inside the doublets.\nOur consensus map of the mouse sperm axoneme reveals outer and inner arm dyneins similar to those observed in sea urchin sperm (Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt; and Extended Data Fig. 4a,b &lt;&lt;FIG:Fig11&gt;&gt;). However, we observed several unique non-motor protein complexes in the mouse sperm axoneme that do not have equivalent counterparts in the reported structures from Chlamydomonas, Tetrahymena and human respiratory cilia (Fig. 1f,g &lt;&lt;FIG:Fig1&gt;&gt;)&lt;&lt;REF:Bui-2012-198-913&gt;&gt;,&lt;&lt;REF:Nicastro-2006-313-944&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;-&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. While the N-DRC in human respiratory cilia has a 'V' shape&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;, our consensus map reveals extra densities that extend to the microtubule surface, creating a square-shaped structure (Fig. 1f &lt;&lt;FIG:Fig1&gt;&gt;). The RSs are comprised of three tower-like densities, with two adopting similar morphology (RS1 and RS2) and a third, distinct RS3 (Fig. 1b,c,g &lt;&lt;FIG:Fig1&gt;&gt;). When viewed from the 'head' of the towers, RS1 and RS2 both exhibit a cleft and a C2 symmetry axis that extends through the 'tower', while RS3 has a distinctive S-shaped surface with two holes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), similar to the ones observed in human respiratory cilia&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. By contrast, the heads of RS1 and RS2 from Chlamydomonas and Tetrahymena do not have the deep cleft&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt;,&lt;&lt;REF:Owa-2019-10-1143&gt;&gt;, the RS3 stump from Chlamydomonas is much shorter, and RS3 from Tetrahymena has a smaller surface area of the head&lt;&lt;REF:Pigino-2011-195-673&gt;&gt;,&lt;&lt;REF:Song-2020-17-201&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). Multiple additional densities, not found in respiratory cilia or unicellular organisms, were observed between the three spokes in the mouse sperm axoneme (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). First, we observed an ~20-nm-sized barrel-shaped density between RS1 and RS2, consistent with extra densities in sperm axonemes reported recently&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. Our higher-resolution map revealed that the barrel is composed of ten rod-shaped strands arranged in a right-handed twist configuration (Supplementary Video 1). Furthermore, densities were found to cross-link RS2 and RS3, hereafter named the 'RS2-RS3 cross-linker' (Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;). Of note, all these extra densities in the MIPs, N-DRCs and RSs are apparent even when our maps were low-pass filtered to 50 A, a resolution lower than that of the published map of the human respiratory cilia axoneme that does not possess these features (Extended Data Fig. 4d-f &lt;&lt;FIG:Fig11&gt;&gt;). Therefore, the additional densities found in mouse sperm are not due to higher resolution in this work but most likely reflect the presence of additional sperm-specific proteins.\n\nThe outer doublets are arranged in fixed radial positions\nWe next calculated a subvolume average for the 32-nm-repeating units of the central pair complex (26 A at FSC = 0.143; n = 3,062 subvolumes; N = 69 tomograms) (Fig. 2a-c &lt;&lt;FIG:Fig2&gt;&gt;, Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt; and Supplementary Figs. 1 and 2). Individual protofilaments of the two singlet microtubules were clearly resolved (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). Various proteins protrude from both microtubules, giving rise to an asymmetric cross-section contour of the central pair complex (Fig. 2b &lt;&lt;FIG:Fig2&gt;&gt;). We observed two distinct sets of MIPs inside the two singlet microtubules, both of which repeat every 32 nm (Extended Data Fig. 5a &lt;&lt;FIG:Fig12&gt;&gt;). On the external surface of microtubules, we observed both 32-nm-repeating and 16-nm-repeating protrusions. Notably, compared to the central pair complex of sea urchin sperm, where MIPs were not observed, the overall shapes of the external protrusions are very similar (Extended Data Fig. 5b &lt;&lt;FIG:Fig12&gt;&gt;), while both are different compared to the central pair complex of Chlamydomonas flagella (Extended Data Fig. 5c &lt;&lt;FIG:Fig12&gt;&gt;)&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;,&lt;&lt;REF:Han-2022-29-472&gt;&gt;-&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. These comparisons suggest that the central pair complex is likely conserved from invertebrate to vertebrate sperm (animal sperm) but different from the ones from unicellular protists\nFig. 2\nThe central pair complex presents asymmetric surfaces in different directions.\na-c, Three views of the central pair complex of the mouse sperm axoneme. a,c, Periodic protrusions are indicated by red arrowheads and highlighted by light red lines, and their periodicities are labeled. d, An average of the entire axoneme was calculated by expanding the original subvolumes for the central pair complex threefold and averaging without further alignment. At high contour, densities of nine doublet microtubules are resolved at nine distinct radial positions. e, The same average as in d is shown at low contour. Note that the A tubules from doublet microtubules are distinguishable due to the presence of extensive MIP densities. Smeared densities corresponding to where dyneins, RSs and N-DRCs are located could be observed at lower occupancies than those for the doublet microtubules.\nTo understand how the outer doublets are arranged relative to the asymmetric central pair complex, we expanded our aligned subvolumes of the central pair complex three times to include the region where the outer doublet microtubules reside and then calculated an average without further alignment (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Although the alignment was only performed for the central pair complex in the expanded subvolumes, nine distinct doublet microtubule densities that are parallel to the singlet microtubules could be resolved, indicating a remarkably consistent radial arrangement of doublets in the axonemes. The A and B tubules of doublet microtubule were clearly distinguishable based on the stronger MIP signals in the former. By contrast, discrete external complexes such as dyneins, RSs and N-DRCs were poorly resolved. The lack of alignment along the longitudinal direction could be caused by mismatch of the 32-nm periodicity of the central pair complex and the 96-nm periodicity of the doublet microtubules (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;).\nTo better understand the spatial relationship between the central pair complex and the outer doublets, we performed multibody refinement by treating the central pair protrusions and the doublet structure as two rigid bodies, refining them separately and remapping them back to each raw subvolume&lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;. Their relative positions were then subjected to principal-component analysis (Methods and Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;). For all nine interfaces, we always observed that the first principal component, which explains most variations (40-50%), was parallel to the longitudinal axis of axonemes (Supplementary Video 2), suggesting that the doublets and the central pair complex from different tomograms meet at different longitudinal offsets.\nThe multibody refinement also yielded a map with the two rigid bodies placed at their average positions, allowing us to examine how the nine doublets interact with different protrusions of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). Interestingly, we observed that most RS heads were separated by a short distance from the central pair protrusions, without any resolved densities between them (Extended Data Fig. 5d &lt;&lt;FIG:Fig12&gt;&gt;). However, at the central pair interface with doublet 8, we observed protrusions from the central pair complex fit into the 'cleft' of RS1 and RS2 and also the two holes of the 'S curve' of the head of RS3 (Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Such complementary shapes may limit the sideways movement of doublet 8 and stabilize its radial position\nFig. 3\nThe diverse interfaces between RSs and the central pair.\na, The arrangement of the central pair complex and nine doublet microtubules (dmt1-9) is based on averaging relative positions deduced by multibody refinement. The different protrusions of the central pair complex are colored in accordance with data from Carbajal-Gonzalez et al.&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. The interface between the RSs from doublet 8 and the central pair complex is indicated (red rectangle). b-d, Three different views of the interface between the RSs from doublet 8 and the protrusions from the central pair complexes (P1, P2 and P3).\n\nAsymmetric distribution of sperm-specific regulators\nWe then sought to investigate whether the outer doublets themselves differ from each other. We grouped doublet subvolumes based on their radial positions relative to the central pair complex (numbered 1-9 as in refs. &lt;&lt;REF:Linck-2016-33-141&gt;&gt;,&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;). The subvolumes were aligned, and the averages were calculated for each of the nine doublets (n = 810-954 subvolumes; N = 58-64 tomograms). This processing strategy identified unique densities emerging from the inner arm dyneins of doublet 5 and connecting to the B tubule of doublet 6 (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;). These connecting densities are similar to the '5-6 bridge' observed in the sea urchin at lower resolutions&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt;, validating our assignment of doublets and processing strategies. Interestingly, the RSs and other 96-nm-repeating features on both doublets 5 and 6 could be resolved concurrently after local refinement, indicating that there is a relatively consistent longitudinal offset (~20 nm) between these two doublet microtubules throughout different axonemes (Extended Data Fig. 6d &lt;&lt;FIG:Fig13&gt;&gt;). The correlation of the unique bridge densities and the consistent offset that is observed only between doublets 5 and 6 suggest that the bridge could limit the relative sliding between this outer doublet pair (in comparison to another doublet pair in Extended Data Fig. 6g,h &lt;&lt;FIG:Fig13&gt;&gt;).\nNext, we systematically compared motor and non-motor or regulatory protein complexes across the nine doublets. Outer arm dyneins across all nine doublets were indistinguishable from those of our consensus average (as shown in Fig. 1d &lt;&lt;FIG:Fig1&gt;&gt;). The densities of inner arm dyneins were also generally similar, with two exceptions. For doublet 5, densities corresponding to dyneins e and g (nomenclature defined in Chlamydomonas&lt;&lt;REF:Bui-2012-198-913&gt;&gt;) were shifted compared to those of the other doublets (Extended Data Fig. 7a &lt;&lt;FIG:Fig14&gt;&gt;), while, for doublet 9, densities for dynein b were not resolved (Extended Data Fig. 7b &lt;&lt;FIG:Fig14&gt;&gt;). These results indicate that the motor proteins are largely the same with only minor variations.\nWe next examined the RSs from each doublet. Strikingly, we observed that the sperm-specific features were asymmetrically distributed across the nine doublets (Fig. 4 &lt;&lt;FIG:Fig4&gt;&gt;). The barrel density was not observed in doublet 1 or 9 and was present at a lower occupancy in doublet 3. In the remaining six doublets, the occupancy of the barrel was comparable to that of other repeating structures (for example, RS1). The RS2-RS3 cross-linkers are absent in doublets 3 and 8 but present in the remaining doublets. In addition, for doublets 2 and 3, we resolved extra densities close to the base of RS3 (RS3 scaffolds) that were not observed in our consensus averages or previously reported consensus averages, in which subvolumes from all nine doublets were averaged together&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;,&lt;&lt;REF:???-?-?-?&gt;&gt;. This is likely because only one or two of the nine doublets possess these features (11-22%) and averaging smeared the signals. Our processing strategy allowed us to isolate these subvolumes based on cellular contextual information, and the high occupancies of these unique structures in the respective doublets suggest their consistent presence in these specific doublets\nFig. 4\nAsymmetric distribution of sperm-specific features in RSs from the nine doublet microtubules in mouse sperm.\nThree orthogonal views of the consensus average of doublets in mouse sperm are shown as in Fig. 1b-d &lt;&lt;FIG:Fig1&gt;&gt;. At the bottom, densities corresponding to RSs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axoneme. Common features are colored in gray, while the barrel, the RS2-RS3 cross-linker and RS3 scaffolds are highlighted in cyan, red and orange, respectively.\nWe also examined the N-DRCs that cross-link neighboring doublets (Fig. 5 &lt;&lt;FIG:Fig5&gt;&gt;). All nine N-DRCs share the common 'V'-shape density, but the extra connections to the microtubule show heterogeneities. Interestingly, N-DRCs from doublets 2-4 share an arch-shaped density perpendicular to the microtubules, while the ones from doublets 6-8 have 45degrees-tilted thin strands. Doublets 1, 5 and 9 all have distinct densities, leading to five different N-DRCs. Note that all these features were observed at similar signal-to-noise levels, and because they resulted from averaging of more than 800 subvolumes sampled in ~60 different axoneme tomograms, they represent the commonly shared features within each individual doublet\nFig. 5\nAsymmetric distribution of sperm-specific features in N-DRCs from the nine doublet microtubules in mouse sperm.\nTop, two orthogonal views of the consensus average of doublets in mouse sperm. Bottom, densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of a cross-section view of the (9 + 2) axonemes. Common features among the N-DRCs are colored in gray, while the unique features are highlighted.\nTo test whether the bending states of the axonemes affect the features of RSs and N-DRCs, we curated subvolumes from tomograms with and without apparent curvatures and calculated per-doublet averages. The same set of features of RSs and N-DRCs were observed. In addition, we also collected a dataset of demembraned sperm axonemes (48 tomograms) that were not actively beating. The RSs and N-DRCs in these non-motile sperm have the same asymmetric features highlighted in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt; and 5 &lt;&lt;FIG:Fig5&gt;&gt;, suggesting that the asymmetric densities were not caused by bias in macroscopic curvatures but likely reflect intrinsic compositional heterogeneities in the nine doublets.\nOur in situ data also allowed us to separate axonemes of the midpiece and principal piece based on the presence of mitochondrial and fibrous sheaths, respectively. We averaged subvolumes from these two regions for each doublet and found only subtle differences in the base of RS3 of doublet 2 and also RS1 of doublet 7 (Extended Data Fig. 8a &lt;&lt;FIG:Fig15&gt;&gt;). In these per-doublet averages, we did not resolve robust densities corresponding to the connections between outer dense fibers and the respective doublets. Previous studies suggest that averages of all nine doublets in the proximal principal region from a few tomograms have such attachments&lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;. However, our raw tomograms of the proximal principal piece showed that some outer dense fibers are close to the corresponding doublets and some are further away (Extended Data Fig. 8b &lt;&lt;FIG:Fig15&gt;&gt;). Such variations require per-doublet averages to be considered. However, the subset of tomograms in the proximal principal region is small, and per-doublet averaging resulted in anisotropic 3D reconstructions. An even larger dataset is required to resolve structures with such a specific and complex distribution pattern. We also calculated an average of subvolumes from the nine doublets near the beginning of axonemes (within 2 microm) and found no significant difference when compared to the overall consensus average, suggesting that the sperm-specific features are established very close to the basal region of the flagella (Extended Data Fig. 8c,d &lt;&lt;FIG:Fig15&gt;&gt;). Together, these data highlighted the overall consistency of the axoneme structure along the flagella.\nOverall, our per-doublet averages showed that the distributions of various sperm-specific features for both the RS complexes and the N-DRCs follow distinct patterns, such that every doublet is decorated by a unique combination of non-motor proteins.\n\nA distinct asymmetric pattern in human sperm\nWe next examined whether the unique outer doublet features observed in mouse sperm were also conserved in other mammalian sperm. We collected tilt series of intact human sperm without milling, focusing on the thinner principal piece of the flagella (Extended Data Fig. 9a,b &lt;&lt;FIG:Fig16&gt;&gt; and Supplementary Figs. 1 and 2). We then calculated consensus averages of doublet microtubules and the central pair complex (23 A and 31 A at FSC = 0.143; n = 6,613 and 2,365 subvolumes, N = 56 and 59 tomograms, respectively) (Fig. 6a,b &lt;&lt;FIG:Fig6&gt;&gt;). These consensus averages of human sperm were similar to the ones from mice, with the notable exception that the relative occupancy of the barrel between RS1 and RS2 was much lower in human sperm. Using the per-doublet processing approach described above, we then calculated averages for each of the nine doublets individually. Although the per-doublet averages from human sperm were noisier than those from mouse sperm due to greater sample thickness (&gt;400 nm versus ~330 nm as shown in Extended Data Fig. 9c-e &lt;&lt;FIG:Fig16&gt;&gt;), they were sufficient to identify the sperm-specific features (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). In particular, the 5-6 bridge, RS2-RS3 cross-linkers and RS3 scaffolds show similar asymmetric distributions between human and mouse sperm. However, we observed that the barrel density only exists in four of nine doublets, in contrast to seven of nine doublets in mouse sperm axonemes (Fig. 6c &lt;&lt;FIG:Fig6&gt;&gt;). This is consistent with the lower occupancy of barrel densities in our consensus average (Fig. 6a &lt;&lt;FIG:Fig6&gt;&gt;). In particular, doublets 2-5 appear to be different in human and mouse axonemes in terms of the presence of the barrel, while the rest of the doublets are similar. We also examined the N-DRC from each doublet and found five distinct structures like those in mouse sperm (Extended Data Fig. 9f &lt;&lt;FIG:Fig16&gt;&gt;). In summary, our data indicate that the asymmetric architecture of axonemes is a general feature of mammalian sperm axonemes, although there are intriguing variations of distribution for the barrel in human and mouse sperm\nFig. 6\nHuman sperm have a different distribution of barrels compared to that in mouse sperm.\na, A consensus average map of the RS for human sperm axonemes. The barrel density appears to have lower occupancy than the RSs (indicated by the red arrowhead), which is not the case in mouse sperm shown in Fig. 1g &lt;&lt;FIG:Fig1&gt;&gt;. b, The consensus average map of the central pair complex for human sperm axonemes (gray) is overlaid with one from mouse sperm axonemes (blue). c, Densities corresponding to RSs from the per-doublet averages are shown around a schematic of the (9 + 2) axonemes. Common features among the RSs are colored in gray, while the barrel, the RS2-RS3 cross-linker and the RS3 scaffold are highlighted in cyan, red and orange, respectively.\n\nDiscussion\nOur in situ tomography studies of mouse and human sperm revealed a large ensemble of macromolecular complexes in their native cellular environment. In particular, we observed various sperm-specific features in the MIPs, RSs and N-DRCs that were not observed in mammalian respiratory cilia and non-mammalian sperm. Furthermore, we reconstructed the entire axoneme using cellular contextual information and uncovered the asymmetric architecture of the mammalian sperm axoneme, where every microtubule-based structure is decorated by a unique set of non-motor proteins (Fig. 7a,b &lt;&lt;FIG:Fig7&gt;&gt;). As these non-motor proteins regulate dynein activities based on previous studies, we propose that the asymmetries of non-motor protein complexes could modulate the sliding of the nine doublets individually to shape the species-specific asymmetric waveforms observed for mammalian sperm, as discussed below\nFig. 7\nModel: every outer doublet is surrounded by a different set of regulatory complexes in mammalian sperm.\nSchematic of the (9 + 2) axonemes of mouse (a) and human (b) sperm. The doublets are numbered, and the sperm-specific regulatory complexes are labeled for each of the nine outer doublets. In particular, the components from the RSs are shown for each doublet and the N-DRCs are colored differently depending on the extra density (as in Figs. 4 &lt;&lt;FIG:Fig4&gt;&gt;-6 &lt;&lt;FIG:Fig6&gt;&gt;). Note that only the barrel distribution is different in mouse and human sperm axonemes. c, RSs from different doublets interact with specific stripes of protrusions of the central pair complex. Our multibody refinement is consistent with the sliding hypothesis, and potential longitudinal movements of dmt1 are indicated by the two red arrows. The model is analogous to 'nine train moving on nine tracks'. d, Schematics showing gradual accumulations of offsets of periodic structure units between two filaments in a curved axoneme (left) and consistent offset between doublet 5-doublet 6 (right). e, The constant offset within tomograms and among tomograms (N = 63) would be consistent with limited horizontal bending in sperm flagella.\n\nSperm-specific structures added mechanical couplings\nMammalian sperm flagella are generally much longer and wider than flagella from unicellular organisms or respiratory cilia (lengths of &gt;45 microm versus ~10 microm and diameters of &gt;0.5-1 microm versus ~0.3 microm). Sperm axonemes are also surrounded by additional subcellular structures, such as outer dense fibers and mitochondrial and fibrous sheaths, which likely present additional mechanical challenges during rhythmic beatings. Previous studies also suggested that larger bending torques are associated with mammalian sperm compared to other cilia&lt;&lt;REF:Lindemann-1996-34-258&gt;&gt;, but it has remained unclear how sperm axonemes have evolved specific mechanisms to withstand the additional mechanical stress. Our averages of in situ cryo-ET revealed many additional non-motor proteins that cross-link the known axonemal components. We propose that these additional proteins function to strengthen the mechanical rigidity of the corresponding components to accommodate higher mechanical requirements of sperm axonemes.\nThe microtubules themselves must be able to withstand vigorous bending, and the seam is the weakest point&lt;&lt;REF:Kikkawa-1994-127-1965&gt;&gt;,&lt;&lt;REF:Zhang-2015-162-849&gt;&gt;. Our data revealed the most extensive MIP interaction network in microtubules observed in axonemes to date. The proteins between the A9 and A10 protofilaments could stabilize the lateral interfaces within the seam. Also, the proteins that form the striations inside the B tubule could cross-link the tubulins within B2-B6 protofilaments and also couple these protofilaments laterally along the helical pitch. We also observed extensive filamentous structures within the A tubule that likely provide additional mechanical stability. Tektin-1 to tektin-4 have been known to assemble into three-helix bundles that pack along one another laterally inside the microtubule doublets of bovine tracheal cilia&lt;&lt;REF:Gui-2021-184-5791&gt;&gt; and likely compose a part of the filamentous densities observed in sperm. Mammalian sperm also contain an additional tektin (tektin-5; Supplementary Table 1)&lt;&lt;REF:Firat-Karalar-2014-127-4128&gt;&gt;, which is a candidate for some of the additional sperm-specific densities (as shown in Extended Data Fig. 3b &lt;&lt;FIG:Fig10&gt;&gt;), although the assignment requires confirmation with higher-resolution reconstructions.\nWe also observed unique sperm-specific densities on the exterior of the microtubules, specifically the barrel and RS2-RS3 cross-linker between the three RSs, the RS3 scaffolds and extra densities in the N-DRC. The RSs were previously observed to tilt relative to the microtubule&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. Additional connections between the three RSs and the scaffolds at the bases would integrate them together into a more rigid unit. Another possibility is that the coupling could lead to coordinated movement of the three RSs. The N-DRCs regulate the sliding between the neighboring doublets and prevent splaying of axonemes&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;. Extra densities linking to the microtubules could improve the stability of N-DRCs under higher mechanical stress. Together, the additional protein complexes in mammalian sperm would help to maintain the geometric integrity of the (9 + 2) microtubule configuration of the axoneme under mechanical stress during vigorous beating motions.\n\nThe arrangement of the doublets and the central pair complex\nPrevious studies suggested that the central pair complex can twist radially relative to the nine outer doublets in Chlamydomonas flagella&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;. Our observation of densities corresponding to the nine doublet microtubules in the average of the entire (9 + 2) axoneme can exclude the possibility of such free twisting in mammalian sperm as the doublet densities would be smeared by averaging (Fig. 2d,e &lt;&lt;FIG:Fig2&gt;&gt;). Interestingly, we observed a cleft between the two halves of RS1 and RS2 and holes in RS3 in mammalian sperm axonemes (Fig. 1c &lt;&lt;FIG:Fig1&gt;&gt;), in contrast to the flatter surface of the RS heads from Chlamydomonas flagella&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;-&lt;&lt;REF:Pigino-2011-195-673&gt;&gt; (Extended Data Fig. 4c &lt;&lt;FIG:Fig11&gt;&gt;). While the flat surfaces may enable twisting in Chlamydomonas&lt;&lt;REF:Mitchell-2004-166-709&gt;&gt;, the complementary shapes of RSs and protrusions from the central pair complex in mammalian sperm may restrict such radial movements (as shown in Fig. 3b-d &lt;&lt;FIG:Fig3&gt;&gt;). Fixation of radial positions of the nine doublets also orients each of the RS complexes in proximity to unique stripes of densities of the central pair complex (Fig. 3a &lt;&lt;FIG:Fig3&gt;&gt;). This arrangement could allow functional specialization and divergent evolution of each doublet microtubule, such as the distinct sets of sperm-specific features in the nine doublets.\nOur multibody refinement suggests the existence of heterogeneities in the longitudinal offsets between doublets and the central pair complex in the randomly sampled axonemes that were combined in the averages (Supplementary Video 2). This observation is consistent with the sliding hypothesis for axonemal bending, in which active dyneins generate displacement between the neighboring doublets&lt;&lt;REF:Satir-1968-39-77&gt;&gt;. Such movement would also lead to displacement of the doublets relative to the central pair complex along the longitudinal axis of axonemes, as though there were nine trains (doublets) moving along nine tracks (central pair protrusions) (Fig. 7c &lt;&lt;FIG:Fig7&gt;&gt;).\nThe 5-6 bridge is a sperm-specific feature that appears to be conserved between sea urchin&lt;&lt;REF:Lin-2012-7-e46494&gt;&gt; and mammalian sperm. Previous studies pointed out that, for two parallel inelastic microtubule filaments, bending would lead to gradual accumulation of longitudinal offsets between them if they were at different radii of the bend (Fig. 7d &lt;&lt;FIG:Fig7&gt;&gt;)&lt;&lt;REF:Warner-1974-63-35&gt;&gt;. As an estimate based on the reported curvature of mouse sperm&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;, the offsets between neighboring doublets could differ by as much as Delta~28 nm within one 2 microm-long tomogram (details in the Methods). More importantly, the initial offset of each tomogram varies depending on how much sliding has happened upstream or downstream of the imaged area of the flagella. On the other hand, the resolved periodic features in both doublets 5 and 6 suggest that there is a consistent offset between these two doublets (Extended Data Fig. 6 &lt;&lt;FIG:Fig13&gt;&gt;), not just within each tomogram but also among the 63 tomograms that contribute to the average. Due to the nature of random sampling of our imaging areas of different cells (N = 63 tomograms), this could happen if there is generally limited bending along the direction parallel to the plane of these two filaments (Fig. 7e &lt;&lt;FIG:Fig7&gt;&gt;). The bundling of these two doublets can create a filament stiffer than that of a single doublet, with distinct elastic properties or bending propensities in different directions. Such asymmetries of mechanical properties within the nine doublets could also contribute to asymmetric waveforms.\n\nAsymmetries of regulators may lead to asymmetric beatings\nOur in situ data and processing strategies based on the contextual information revealed that the axoneme itself, which appeared to have 'pseudo-ninefold symmetry' in classical TEM images&lt;&lt;REF:Fawcett-1975-44-394&gt;&gt;, is highly asymmetric at the molecular level. As the axoneme is the underlying engine that drives the flagellar beating motion, such asymmetries in structure could lead to asymmetric beating waveforms.\nFurthermore, the asymmetries lie mostly in the mammalian sperm-specific non-motor protein complexes, including the RSs and N-DRCs. These complexes are well-established regulators of dynein motor activities, and defects in individual protein components can lead to irregular beating&lt;&lt;REF:Witman-1978-76-729&gt;&gt;,&lt;&lt;REF:Huang-1981-88-80&gt;&gt;,&lt;&lt;REF:Bower-2013-24-1134&gt;&gt;,&lt;&lt;REF:Grossman-Haham-2021-28-20&gt;&gt;. In mammalian sperm, we show here that each of the nine doublet microtubules is decorated by a unique composition of sperm-specific regulators. We speculate that the distinct molecular composition could lead to differences in the sliding speeds or bending forces for each of the doublet pairs (Fig. 7 &lt;&lt;FIG:Fig7&gt;&gt;). The non-equivalent forces could lead to a deviation from the single plane of beating characteristic of more symmetric axonemes. Thus, we hypothesize that the non-uniform distribution of sperm-specific RS and N-DRC regulators are important for asymmetric and non-planar beating. Additionally, previous studies showed that human and mouse sperm have different swimming waveforms (movies in refs. &lt;&lt;REF:Babcock-2014-12-67&gt;&gt;,&lt;&lt;REF:Gadelha-2020-6-eaba5168&gt;&gt;). Our studies suggest that, although the RS barrels are conserved in human and mouse, their distribution varies and the variations could create diverse sperm swimming behaviors.\nThis study has revealed mammalian sperm-specific structures within the microtubules, RSs and N-DRCs that are absent in sea urchin and zebrafish sperm&lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Lin-2018-360-eaar1968&gt;&gt;. The still unknown proteins likely arose to serve functions required by natural fertilization in mammals. Sperm from different species are cast into a foreign environment and selected for their ability to reach and fertilize an egg. Sea urchin and zebrafish sperm swim freely in water, whereas mammalian sperm swim in a thin layer of viscous liquid on uneven surfaces of female reproductive tracts&lt;&lt;REF:Suarez-2016-363-185&gt;&gt;. The viscous environment brought more mechanical challenges to the axonemes and hence the microtubule filaments inside. In addition, asymmetric and non-planar waveforms of mammalian sperm could be beneficial to navigate around 3D obstacles in reproductive tracts. Furthermore, the dimensions and physical characteristics of reproductive tracts vary among different mammals; therefore, fine-tuning the underlying molecular features to produce specialized waveforms is likely under evolutionary selection. In the future, systematic genetic and proteomic analyses of mammalian sperm-specific proteins would be valuable to connect sperm-specific axonemal structures with their functions in sperm motility and reproduction.\n\nMethods\nAll chemicals were purchased from Sigma-Aldrich unless otherwise noted.\n\nSample preparation\nMouse sperm were collected from 10-16-week-old C57Bl/6J mice based on a published protocol&lt;&lt;REF:van der Spoel-2002-99-17173&gt;&gt;. Briefly, the sperm were stripped from vasa deferentia by applying pressure to the cauda epididymis in 1x Krebs buffer (1.2 mM KH2PO4, 120 mM NaCl, 1.2 mM MgSO4*7H2O, 14 mM dextrose, 1.2 mM CaCl2*2H2O, 5 mM KCl, 25 mM NaHCO3). The sperm were washed and resuspended in ~100 microl Krebs buffer for the following experiments.\nHuman sperm cells were collected by masturbation from healthy donors and visually inspected for normal morphology and motility before use. Spermatozoa were isolated by the swim-up procedure in HTF or HS solution as previously described&lt;&lt;REF:Skinner-2019-9-e3182&gt;&gt; and then concentrated by centrifuging for 5 min at 500g. The supernatant was reduced to 100 microl, and the cells were resuspended.\nAll experimental procedures using human-derived samples were approved by the Committee on Human Research at the University of California, Berkeley, under IRB protocol number &lt;&lt;REF:???-?-?-?&gt;&gt; -06-5395.\n\nGrid preparation\nEM grids (Quantifoil R 2/2 Au 200 mesh) were glow discharged to be hydrophilic using an easiGlow system (Pelco). The grid was then loaded onto a Leica GP cryo plunger (pre-equilibrated to 95% relative humidity at 25 degreesC). The mouse sperm suspension was then mixed with 10-nm gold beads (Electron Microscopy Sciences, 25487) to achieve final concentrations of 2-6 x 106 million cells per ml. For the demembraned mouse sperm, Triton X-100 was added to a final concentration of 0.1% in the final suspension. The human sperm suspension was mixed with 10-nm gold beads to achieve final concentrations of 0.5-2 x 106 million cells per ml. Next, 3.5 microl of sperm mixture was added to each grid, followed by incubation for 15 s. The grids were then blotted for 4 s and plunge-frozen in liquid ethane.\n\nCryogenic focused ion beam milling\nCryogenic focused ion beam milling was performed either manually using an Aquilos cryo-FIB-SEM microscope (Thermo Fisher Scientific) or automatically using an Aquilos II cryo-FIB-SEM microscope (Thermo Fisher Scientific). A panoramic SEM map of the whole grid was first taken at 377x magnification using an acceleration voltage of 5 kV with a beam current of 13 pA and a dwell time of 1 micros. Targets with appropriate thickness for milling were picked on the map. A platinum layer (~10 nm) was sputter coated, and a gas injection system was used to deposit the precursor compound trimethyl(methylcyclopentadienyl)platinum(IV). The stage was tilted to 15-20degrees, corresponding to a milling angle of 8-13degrees relative to the plane of grids. FIB milling was performed using stepwise decreasing current as the lamellae became thinner (1.0 nA-30 pA; final thickness, ~300 nm). The grids were then stored in liquid nitrogen before imaging.\n\nImage acquisition and tomogram reconstruction\nTilt series of mouse sperm were collected on a 300-kV Titan Krios TEM (Thermo Fisher Scientific) equipped with a high-brightness field emission gun (xFEG), a spherical aberration corrector, a Bioquantum energy filter (Gatan) and a K3 Summit detector (Gatan). The images were recorded at a nominal magnification of 19,500x in super-resolution counting mode using SerialEM&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;. After binning over 2 x 2 pixels, the calibrated pixel size was 3.53 A on the specimen level. For each tilt series, images were acquired using a modified dose-symmetric scheme between -48degrees and 48degrees relative to the lamella with 3degrees steps and grouping of two images on either side (0degrees, 3degrees, 6degrees, -3degrees, -6degrees, 9degrees, 12degrees, -9degrees, -12degrees, 15degrees...). At each tilt angle, the image was recorded as movies divided into eight subframes. The total electron dose applied to a tilt series was 100 e- A-2. The defocus target was set to be -4 to -7 microm.\nTilt series of human sperm were recorded at a nominal magnification of 33,000x in super-resolution counting mode. After binning over 2 x 2 pixels, the calibrated pixel size was 2.66 A on the specimen level. The total electron dose applied to a tilt series was 100 e- A-2. For each tilt series, images were acquired using a bidirectional scheme between -48degrees and 48degrees relative to the lamella starting from either 0degrees or 21degrees, with an incremental step of 3degrees. The defocus target was set to be ~2-5 microm. At each tilt, the image was recorded as movies divided into eight subframes.\nAll movie frames were corrected with a gain reference collected in the same EM session. Movement between frames was corrected using MotionCor2 without dose weighting&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;. Initial reconstructions of all tilt series were performed using AreTomo&lt;&lt;REF:Zheng-2022-6-100068&gt;&gt;, and these 3D tomograms were examined to exclude tomograms with bad target tracking, crystal ice, aberrant defocus, incomplete axonemes (less than five doublet microtubules caused by FIB-SEM milling) and unfavorable orientations (perpendicular to the tilt axis). Alignment of selected tilt series and tomographic reconstructions were then performed using Etomo&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. The aligned tilt series were then CTF corrected using TOMOCTF&lt;&lt;REF:Fernandez-2006-106-587&gt;&gt;, and the tomograms were generated using TOMO3D&lt;&lt;REF:Agulleiro-2015-189-147&gt;&gt; (bin2; pixel size, 7.06 A).\nIn total, we started with 24 milling grids of mouse sperm and obtained 200 lamellae. Tilt series with no crystal ice were kept and processed further. In some cases, parts of the (9 + 2) axoneme was milled away and we only processed the ones with at least five doublets and also enough space to include the full central pair complex for subvolume averaging. In the end, the final reconstructions of the consensus averages were from 69 usable tomograms. The per-doublet averages were based on subsets of the 69 tomograms (58-64 tomograms). For the human sperm cryo-ET dataset, the final reconstructions of the consensus averages are from 65 tilt series with the flagellar orientations within ~30degrees of the tilt axis. Note that the final tomograms used for different reconstructions vary as tomograms may be discarded due to poor alignment or may lack certain doublets.\n\nSubvolume averaging\nSubsequent subvolume extraction, classification and refinement were all performed using RELION3 (ref. &lt;&lt;REF:Bharat-2016-11-2054&gt;&gt;) (schematic workflow in Extended Data Fig. 2a &lt;&lt;FIG:Fig9&gt;&gt;). Briefly, subvolumes from the doublets were manually picked every 24 nm and extracted with a box size large enough to accommodate a complete 96-nm-repeating unit (pixel size, 7.06 A; box size, 180 pixels; dimension, 127.08 nm). Initially, subvolumes were aligned to a map of Tetrahymena doublets (EMD-9023) low-pass filtered to 80 A, and the resulting map was used as the reference for further processing. Manual curation of the data was performed to check the alignment accuracy and data quality for each tomogram. Supervised 3D classification on RSs gave rise to four class averages of the 96-nm-repeating units at four different registries, staggered from one another by 24 nm. The pixel view in IMOD was used to determine the coordinates corresponding to the base of RS2 in all four class averages, and all subvolumes were re-extracted and recentered to the same point. All subvolumes were combined and aligned to one reference, and duplicate subvolumes were removed based on minimum distance (&lt;40 nm). The remaining subvolumes were aligned to yield the consensus average for all nine doublets. These subvolumes were later remapped and sorted to calculate per-doublet averages (the script is also available below). The MIPs inside microtubule doublets repeat every 48 nm. Subvolumes of the 96-nm-repeating units were recentered on MIP features that repeat every 48 nm and refined with a mask focusing on the microtubules only.\nTo generate an average for the central pair complex, subvolumes were picked and extracted from the central pair complex every 16 nm (pixel size, 7.06 A; box size, 160 pixels; dimension, 112.96 nm) (Extended Data Fig. 2b &lt;&lt;FIG:Fig9&gt;&gt;). Refinement of all subvolumes to an average of the sea urchin central pair (EMD-9385) resulted in a central pair with only 16-nm-repeating features. The alignment parameters were modified to reset all translations (x, y, z) to zero and then used for a second round of refinement with local search only. Focused classification was then performed on the microtubule-associated proteins on the C1 microtubule to separate the two populations of subvolumes (~50% each) corresponding to the central pair complex with 32-nm periodicity and an offset of 16 nm between them. The subvolumes were then recentered and extracted on the same protein features for the two class averages. All subvolumes were aligned to the same reference using local searches. Duplicate subvolumes were removed based on the minimum separating distance (&lt;30 nm). The remaining subvolumes were refined to generate the consensus average for the central pair complex.\nSubvolumes large enough to include all nine doublets were re-extracted, centering on the aligned subvolumes of the central pair complex (pixel size, 21.18 A; box size, 160 pixels; dimension, 338.88 nm). These subvolumes were averaged without further alignment and used to re-extract subvolumes corresponding to individual doublets. Subvolumes corresponding to a particular doublet were aligned and remapped back to the tomograms in three dimensions. However, these subvolumes do not have the correct alignment for the 96-nm-repeating features, but their centers nevertheless trace the axis of that microtubule doublet accurately when they are remapped in three dimensions (using the script available below). The 96-nm periodic subvolumes curated above to generate the consensus average of 96-nm repeats were then sorted. The subvolumes corresponding to a particular doublet were then aligned and averaged.\nFor multibody refinement (schematic workflow in Extended Data Fig. 2c &lt;&lt;FIG:Fig9&gt;&gt;), the subvolumes corresponding to the specific RS-central pair interface were re-extracted based on the subvolumes of 96-nm units of individual doublet microtubules. The particles were recentered at the junction between the head and the stalk of RS2 to include enough features from both the doublets and the central pair complex. The RSs were then aligned with a mask. This mask and a mask covering the central pair complex were used for the multibody analysis implemented in RELION3 (ref. &lt;&lt;REF:Nakane-2018-7-e36861&gt;&gt;). Briefly, separate refinement of the RSs and central pair protrusions provided two sets of alignment parameters: three translational shifts (x, y, z) and three Euler angles required to align one subvolume to each reference. Thus, 12 parameters can be used to remap the two references back to each raw subvolume, and the spatial relationship of the two rigid bodies in each subvolume could be determined by these 12 parameters. Principal-component analyses essentially reprojected the original 12-dimensional data in a new 12-dimensional space with 12 new orthogonal eigenvectors. These 12 eigenvectors could be mathematically determined so that they represent decreasing variations of the entire data along the individual eigenvectors. Our analyses suggest that the first eigenvector or axis is parallel to the axoneme axis and the variations along this axis account for 40-50% of the total variations. When all subvolumes were divided into ten groups based on their projection values on the first axis (10% lowest, 10-20% lowest, ..., 90-100% highest). Each group was then represented by a snapshot depicting the averaging projection values of the group members, and ten of these snapshots were morphed to generate the animation (Supplementary Video 1).\nThe resolutions for maps were estimated based on FSC values of two independently refined half datasets (FSC = 0.143). Local-resolution maps for the consensus averages of doublets and the central pair complex of both human and mouse sperm were calculated by blocres in Bsoft (Supplementary Fig. 2). These local-resolution maps represent relative differences in resolution across the maps, but the absolute values may not be exact. IMOD was used to visualize the tomographic slices&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;. UCSF Chimera was used to manually segment the maps for various structure features, and these maps were colored individually to prepare the figures using UCSF ChimeraX&lt;&lt;REF:Pettersen-2004-25-1605&gt;&gt;-&lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;.\n\nEstimation of accumulated offsets in axonemes\nThe bending curvature of mouse sperm could be as large as 2 x 105 m-1 or 0.20 microm-1 based on literature (OD = 5 microm = 5,000 nm)&lt;&lt;REF:Lindemann-2016-73-652&gt;&gt;. The distance between the neighboring doublets is 72 nm (measured in the average of the axoneme shown in Fig. 2d &lt;&lt;FIG:Fig2&gt;&gt;, OB = 5,072 nm). If AB and CD represent 96-nm repeats from doublet 1 and doublet 2, respectively, offset Delta = CE =~ OC x (/_COD - /_AOB) = 5,000 x (96 / 5,000 - 96 / 5,072) = 1.4 nm.\nNote that the Delta represents the additional offset per 96-nm repeat. In a tomogram that contains an axoneme of ~2 microm, if the offset between the first pair of 96-nm repeats is 0 nm, the offset of the 20th pair is 28 nm. Note that this initial offset (0 nm) is tomogram specific depending on how much sliding has happened upstream or downstream of the imaging area and the imaged cell.\n\nReporting summary\nFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.\n\nOnline content\nAny methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41594-022-00861-0.\n\nSupplementary information\n\nSupplementary Information\nSupplementary Figs. 1 and 2 and Table 1\nReporting Summary\nPeer Review File\nSupplementary Video 1\nThe structure of the RS barrel.\nSupplementary Video 2\nThe movie contains ten frames, and each frame represents an average of one-tenth of the total populations distributed along the first component of the principal-component analysis. The different frames indicate that the RSs and the central pair complex can come together at different longitudinal offsets. The interface between doublet 6 and the central pair protrusion 1b is used as an example here.\nSupplementary Code\nSupplementary information\nThe online version contains supplementary material available at 10.1038/s41594-022-00861-0.\n\nExtended data\n\nExtended Data Fig. \nThe workflow of FIB-SEM/cryo-ET and representative images.\na, Schematic of EM grids with vitrified mouse sperm. An example for where the windows were milled by FIB-SEM is denoted by two yellow boxes. b, SEM (left) and FIB (right) images of a lamella are shown (scale bar: 5 microm, N = 200 lamellae). c, A low-magnification montage and its zoom-in view of a lamella acquired using TEM (scale bar: 1 microm, N = 200 lamellae). Axonemes and mitochondria are indicated by white arrowheads and an arrow, respectively. d, A representative image of non-milled sperm axoneme recorded by 300 kV Titan Krios TEM at a total dose of 3 e-/A2 (scale bar: 100 nm, N = 10 tilt series, not processed further). e, A representative image of milled lamella (thickness ~300 nm) is shown (scale bar: 50 nm, N = 69 tilt series). The zoom-in region in f is outlined (the red rectangle). f, Individual protofilaments of microtubules and double-bilayer membranes of the mitochondria are indicated by yellow arrows and arrowheads. g, Fourier transform of the image in e is shown and the upper limit of ctf fitting when using CTFFIND4 is indicated (at the pixel size of 3.53 A, N = 69 zero-tilt images). h, i, Two sections of 3D tomograms are shown. Outer arm dyneins (every 24 nm) anchored on doublet microtubules are indicated in h (red arrowheads). Protrusions (every 32 nm) extending from the singlet microtubule are indicated in i (black arrowheads). Periodic patterns of three radial spokes on the doublet microtubules (every 96 nm) are grouped and indicated in i (black brackets) (scale bar: 50 nm, N = 69 tomograms).\n\nExtended Data Fig. \nSchematics of data processing.\na, A schematic of the processing workflow of 96 nm-repeating units of doublets to achieve the consensus average combining data from all nine doublets. An example of remapping of the centers of the final subvolumes in three dimensions is shown. b, A schematic of the processing workflow of 32 nm-repeating units of central complex to achieve the subvolume average. Recentering of subvolumes of the central pair complex (every 32 nm) on the doublet 1 is shown. Remapping of these subvolumes helps to trace the trajectory of doublet 1 in three dimensions. The centers of these doublet 1 subvolumes are shown in the same tomogram as in a. c, A schematic of the workflow of multi-body refinement.\n\nExtended Data Fig. \nMIPs network in A-tubule of doublet microtubules.\na, Four cross-section views of the A tubule in the doublet microtubule are shown. The MIPs network is colored in orange. Three modes of interactions between the MIPs and the protofilaments are observed (see text). The protrusions from the filamentous core are indicated by empty red arrowheads and dashed cyan ovals, respectively. b, Six cross-section views of the overlay of the average of the 48-nm repeat of doublets of mouse sperm (orange, this study) and the 48-nm repeat of doublets of bovine trachea cilia (blue, EMD-24664)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;. The contour levels of the two maps are adjusted so that the densities of microtubules are matched. The microtubule seam, A9 and A10 protofilaments are labeled in all views. Note there are additional densities near A9-A10 region in the mouse sperm axoneme (black circles). We observed density striations in the B-tubule in some cross sections. c, An orthogonal view of the B-tubule striations along the helical pitch of the microtubule at a high and a low threshold, respectively. Note the striations are separated by 8 nm and they cover the intradimeric interface between the alpha- and beta-tubulins based on the model for the bovine doublet (PDB: 7rro)&lt;&lt;REF:Gui-2021-184-5791&gt;&gt;.\n\nExtended Data Fig. \nComparisons of dyneins and radial spokes of mouse sperm axonemes to published equivalent structures from other motile cilia.\na, Comparison of outer arm dyneins in mouse sperm axoneme (this study) and reported maps for those from sea urchin sperm (EMD-8835 and EMD-8836)&lt;&lt;REF:???-?-?-?&gt;&gt;. b, Comparison of inner arm dyneins in mouse sperm axoneme (this study) and published maps for those from sea urchin sperm (EMD-8837 and EMD-8838)&lt;&lt;REF:???-?-?-?&gt;&gt;. Note the barrel densities between RS1 and RS2 only exist in the mouse sperm are indicated (dashed ovals). c, Comparison of radial spokes of mouse sperm axoneme (this study) and the ones from motile cilia of Chlamydomonas (EMD-9768&lt;&lt;REF:Owa-2019-10-1143&gt;&gt; and EMD-22475&lt;&lt;REF:Gui-2021-28-29&gt;&gt;) and Tetrahymena (EMD-9023)&lt;&lt;REF:Song-2020-17-201&gt;&gt;. Note the RS3 stump in Chlamydomonas is much shorter than the others and the head of RS3 from Tetrahymena has a smaller surface area. d-f, The doublet average of mouse sperm axoneme is low-pass filtered to 50 A and viewed from different angles. Note this resolution is lower than published axoneme maps (for example 34 A for EMD-5950 shown in Fig. 1 &lt;&lt;FIG:Fig1&gt;&gt; for comparison)&lt;&lt;REF:Lin-2014-5-5727&gt;&gt;. The extensive network of MIPs (d), extra densities at N-DRC (e) and radial spokes (f) remain apparent while comparing to human respiratory cilia, suggesting the presence of novel densities in our maps are not due to higher resolutions, but most likely the presence of additional proteins.\n\nExtended Data Fig. \nThe central pair complex in mouse axoneme possesses more MIPs but the overall shape is similar to the one from sea urchin sperm.\na, Two longitudinal views of the MIPs inside the two singlet microtubules are shown. Periodic features (every 32 nm) are indicated (red empty arrowheads). b, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from sea urchin sperm (EMD-9385) and they are colored in grey and cyan, respectively&lt;&lt;REF:Carbajal-Gonzalez-2013-70-101&gt;&gt;. Note most of the protrusions are very similar and the sea urchin-specific protrusion 1f is indicated in the middle panel (red arrowhead). c, The average of the central pair complex in mouse sperm axonemes is overlaid on the map for the central pair complex from Chlamydomonas (EMD-31143)&lt;&lt;REF:Hou-2021-134-jcs258540&gt;&gt;. Note the maps are aligned based on the similar C1 protrusions 1d. Other protrusions are generally less similar (1a, 1f and 2d) and there appears to be an 8-nm shift for the C2 protrusions. The Chlamydomonas C2 microtubule was recently shown to possess two possible registers with 8-nm longitudinal differences&lt;&lt;REF:Han-2022-29-472&gt;&gt;. d, The spatial relationship between the nine radial spokes and the central pair complex are determined by multibody refinement and the 'average positioning' of these two rigid bodies are shown (RS-CP1 denotes the interface between radial spokes from doublet 1 and the central pair.\n\nExtended Data Fig. \nThe 5-6 bridge of mouse sperm axonemes.\na-d, Four views of the composite map for 5-6 bridge in mouse sperm axonemes combining maps for local refinement of the doublet 5, doublet 6 and the bridge. The inner arm dyneins, radial spokes, the barrel, N-DRC and extra bridge densities are colored in pink, different shades of cyan, light blue, green and blue, respectively. Note that the radial spokes which repeat every 96 nm on doublet 5 and doublet 6 are resolved and there appears to be a ~20 nm offset between their longitudinal registers, as indicated in the staggering distance of RS1 and RS2 from doublet 5 and doublet 6 in d. e, f, Two cut-in views of a and c are shown. Bridge-specific densities are highlighted in blue. These densities extend from the inner arm dyneins of doublet 5 towards the microtubule of doublet 6. Direct contacts of the bridge-specific densities to doublet 6 is indicated by the solid red arrowhead while densities in close proximity of the base of radial spoke 2 in doublet 6 are indicated by the empty red arrowhead. g, h, Subvolumes for doublet 1 was aligned and equivalent views of doublet 1 and 2 are shown (as in panel a and b). The expected positions for dyneins and radial spokes on doublet 2 are circled by dashed and solid red ovals. These 96 nm-repeating structures on doublet 2 are not resolved even after local refinement, suggesting the offsets between doublet 1 and doublet 2 are not consistent. The microtubule from doublet 2 is nevertheless partially resolved since variant longitudinal mismatching would not smear a tube of 'continuous densities' compared to 'discrete densities' like dyneins and radial spokes through averaging.\n\nExtended Data Fig. \nThe inner arm dyneins in doublet 5 and doublet 9 differ from the consensus average in mouse sperm axonemes.\na, An overlay of inner arm dyneins from doublet 5 and the concensus average. The assigned inner arm dyneins, N-DRC and 5-6 bridge densities from the doublet 5 are colored in pink, green and blue, respectively while the densities for the consensus average are in grey. Different inner arm dyneins are named according to the nomenclature defined by studies of Chlamydomonas flagella. The shifted positions of dynein e and dynein g are indicated by the two yellow arrows. b, An overlay of inner arm dyneins from doublet 9 and the consensus average were blue and grey, respectively. Note dynein b in doublet 9 is not resolved.\n\nExtended Data Fig. \nLongitudinal consistency of doublets in mouse sperm axonemes.\na, Schematic of a mouse sperm flagella with the basal region (&lt; 2 microm), midpiece and principal piece highlighted in the top panel. Subvolume averages of doublet 2 and doublet 7 from the midpiece and the principal piece are shown. The shapes of densities for the scaffolds at the base of radial spoke 3 appear different in doublet 2 and are circled. Additional densities at the base of radial spoke 1 in the doublet 7 in the midpiece region are observed and circled. b, Four xz slices of tomograms in different regions of the mouse sperm flagella are shown. Outer dense fibers that are in close proximity to the doublets and the ones with gaps in between are indicated (red and blue arrowheads, respectively) (scale bars: 100 nm). c, Two xy slices of two 3D tomograms of the basal region of mouse sperm axoneme are shown. The singlet microtubules of the central pair complex extend further into the cell bodies and the beginning of the doublet microtubules are indicated by the arrows and arrowheads, respectively (scale bars: 200 nm). d, Densities corresponding to different protein complexes from subvolume averages of the basal region (6 tomograms, N = 670 subtomograms) and the consensus average are overlaid and colored in blue and grey, respectively. Note the sperm-specific features in N-DRC and radial spokes exist in the average of doublets in the basal region and are highlighted using empty red arrowheads.\n\nExtended Data Fig. \nCryoET data of human sperm without milling and asymmetric distribution of sperm-specific features in N-DRCs in human sperm.\na, A representative xy slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 1, scale bar: 100 nm). Although the A- and B-tubule of the microtubule doublets are discernible, other features like axonemal dyneins and radial spokes are not clear in this slice (in contrast to Extended Data Fig. 1h-i &lt;&lt;FIG:Fig8&gt;&gt;). b, A representative xz slice of the same tomogram of human sperm (pixel size=10.648 A, image thickness: 5, displayed in IMOD) (scale bar: 100 nm). c, A representative xz slice of a tomogram of mouse sperm lamella (pixel size=14.12 A, image thickness: 10, displayed in IMOD). The red bar indicates how we measure the thickness of the lamella (scale bar: 100 nm). d, A representative xz slice of a tomogram of human sperm (pixel size=10.648 A, image thickness: 100, displayed in IMOD). Gold beads on the top and bottom surfaces are circled and the red bar indicates how we measure the thickness of the ice near the cell. The top and bottom membrane of the sperm cells is not resolved due to the missing wedge so the actual thickness is larger than the measured thickness of ice in the vicinity (scale bar: 100 nm). e, A dot plot showing the thickness of lamellae of mouse sperm and ice near human sperm as a lower bound estimate of the actual cells. The means and standard deviations are indicated (for N = 69 mouse sperm tomograms and N = 59 human sperm tomograms). f, Densities corresponding to N-DRCs from the nine per-doublet averages are shown around a schematic of the (9 + 2)- axoneme. Common features among the N-DRC are colored in grey, while the unique features are highlighted in various colors.\n\nExtended data\nis available for this paper at 10.1038/s41594-022-00861-0.\n\nPeer review information\nNature Structural &amp; Molecular Biology thanks Girish Mali and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Carolina Perdigoto and Florian Ullrich, in collaboration with the Nature Structural &amp; Molecular Biology team. Peer reviewer reports are available.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [21.76s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"organ\",\n  \"organism_name\": \"Mus musculus, Homo sapiens\",\n  \"cell_strain\": \"not present\",\n  \"sample_preparation\": \"Mouse sperm were collected, washed, resuspended and mixed with gold beads. Human sperm cells were collected, washed, concentrated by centrifuging, mixed with gold beads.\",\n  \"grid_preparation\": \"EM grids were glow discharged, loaded onto a cryo plunger, sperm mixture added, incubated, blotted and plunge-frozen in liquid ethane.\",\n  \"cryoet_pixel_spacing\": \"3.53 A, 2.66 A\",\n  \"cryoet_acceleration_voltage\": \"300 kV\",\n  \"microscope_name\": \"Titan Krios TEM\",\n  \"microscope_setup_energy_filter\": \"Bioquantum energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"spherical aberration corrector\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K3 Summit detector\",\n  \"tilt_minimum\": \"-48degrees\",\n  \"tilt_maximum\": \"48degrees\",\n  \"total_flux\": \"100 e- A-2\",\n  \"reconstruction_software\": \"AreTomo, Etomo, TOMOCTF, TOMO3D\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"not present\",\n  \"annotated_entities\": \"microtubule protofilaments, double-bilayer membranes of mitochondria, axonemal dyneins, radial spokes, periodic protrusions from singlet microtubules of the central pair complex\",\n  \"annotation_methods\": \"manual\",\n  \"annotation_software\": \"IMOD, UCSF Chimera, UCSF ChimeraX\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"organ\",\n  \"organism_name\": \"Mus musculus, Homo sapiens\",\n  \"cell_strain\": \"not present\",\n  \"sample_preparation\": \"Mouse sperm were collected, washed, resuspended and mixed with gold beads. Human sperm cells were collected, washed, concentrated by centrifuging, mixed with gold beads.\",\n  \"grid_preparation\": \"EM grids were glow discharged, loaded onto a cryo plunger, sperm mixture added, incubated, blotted and plunge-frozen in liquid ethane.\",\n  \"cryoet_pixel_spacing\": \"3.53 A, 2.66 A\",\n  \"cryoet_acceleration_voltage\": \"300 kV\",\n  \"microscope_name\": \"Titan Krios TEM\",\n  \"microscope_setup_energy_filter\": \"Bioquantum energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"spherical aberration corrector\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K3 Summit detector\",\n  \"tilt_minimum\": \"-48degrees\",\n  \"tilt_maximum\": \"48degrees\",\n  \"total_flux\": \"100 e- A-2\",\n  \"reconstruction_software\": \"AreTomo, Etomo, TOMOCTF, TOMO3D\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"not present\",\n  \"annotated_entities\": \"microtubule protofilaments, double-bilayer membranes of mitochondria, axonemal dyneins, radial spokes, periodic protrusions from singlet microtubules of the central pair complex\",\n  \"annotation_methods\": \"manual\",\n  \"annotation_software\": \"IMOD, UCSF Chimera, UCSF ChimeraX\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 390,
      "prompt_tokens": 17539,
      "total_tokens": 17929
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_dbe4f8ca8b"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "biological_sample_type": "organ",
  "organism_name": "Mus musculus, Homo sapiens",
  "cell_strain": "not present",
  "sample_preparation": "Mouse sperm were collected, washed, resuspended and mixed with gold beads. Human sperm cells were collected, washed, concentrated by centrifuging, mixed with gold beads.",
  "grid_preparation": "EM grids were glow discharged, loaded onto a cryo plunger, sperm mixture added, incubated, blotted and plunge-frozen in liquid ethane.",
  "cryoet_pixel_spacing": "3.53 A, 2.66 A",
  "cryoet_acceleration_voltage": "300 kV",
  "microscope_name": "Titan Krios TEM",
  "microscope_setup_energy_filter": "Bioquantum energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "spherical aberration corrector",
  "camera_manufacturer": "Gatan",
  "camera_model": "K3 Summit detector",
  "tilt_minimum": "-48degrees",
  "tilt_maximum": "48degrees",
  "total_flux": "100 e- A-2",
  "reconstruction_software": "AreTomo, Etomo, TOMOCTF, TOMO3D",
  "tomogram_size": "not present",
  "reconstruction_method": "not present",
  "annotated_entities": "microtubule protofilaments, double-bilayer membranes of mitochondria, axonemal dyneins, radial spokes, periodic protrusions from singlet microtubules of the central pair complex",
  "annotation_methods": "manual",
  "annotation_software": "IMOD, UCSF Chimera, UCSF ChimeraX"
}
[chain/end] [1:chain:RunnableSequence] [21.77s] Exiting Chain run with output:
{
  "biological_sample_type": "organ",
  "organism_name": "Mus musculus, Homo sapiens",
  "cell_strain": "not present",
  "sample_preparation": "Mouse sperm were collected, washed, resuspended and mixed with gold beads. Human sperm cells were collected, washed, concentrated by centrifuging, mixed with gold beads.",
  "grid_preparation": "EM grids were glow discharged, loaded onto a cryo plunger, sperm mixture added, incubated, blotted and plunge-frozen in liquid ethane.",
  "cryoet_pixel_spacing": "3.53 A, 2.66 A",
  "cryoet_acceleration_voltage": "300 kV",
  "microscope_name": "Titan Krios TEM",
  "microscope_setup_energy_filter": "Bioquantum energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "spherical aberration corrector",
  "camera_manufacturer": "Gatan",
  "camera_model": "K3 Summit detector",
  "tilt_minimum": "-48degrees",
  "tilt_maximum": "48degrees",
  "total_flux": "100 e- A-2",
  "reconstruction_software": "AreTomo, Etomo, TOMOCTF, TOMO3D",
  "tomogram_size": "not present",
  "reconstruction_method": "not present",
  "annotated_entities": "microtubule protofilaments, double-bilayer membranes of mitochondria, axonemal dyneins, radial spokes, periodic protrusions from singlet microtubules of the central pair complex",
  "annotation_methods": "manual",
  "annotation_software": "IMOD, UCSF Chimera, UCSF ChimeraX"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Structures and distributions of SARS-CoV-2 spike proteins on intact virions\nZunlong Ke 1,7 , Joaquin Oton 1,7 , Kun Qu 1,7 , Mirko Cortese 2 , Vojtech Zila 3 , Lesley McKeane 4 , Takanori Nakane 1 , Jasenko Zivanov 1 , Christopher J. Neufeldt 2 , Berati Cerikan 2 , John M. Lu 1 , Julia Peukes 1 , Xiaoli Xiong 1 , Hans-Georg Kräusslich 3,5 , Sjors H. W. Scheres 1 , Ralf Bartenschlager 2,5,6 &amp; John A. G. Briggs 1 ✉\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virions are surrounded by a lipid bilayer from which spike (S) protein trimers protrude 1 . Heavily glycosylated S trimers bind to&nbsp;the angiotensin-converting enzyme 2 receptor and mediate entry of virions into target cells 2–6 . S exhibits extensive conformational flexibility: it modulates exposure of its receptor-binding site and subsequently undergoes complete structural rearrangement to drive fusion of viral and cellular membranes 2,7,8 . The structures and conformations of soluble, overexpressed, purified S proteins have been studied in detail using cryo-electron microscopy 2,7,9–12 , but the structure and distribution of S on the virion surface remain unknown. Here we applied cryo-electron microscopy and tomography to image intact SARS-CoV-2 virions and determine the high-resolution structure, conformational flexibility and distribution of S trimers in&nbsp;situ on the virion surface. These results reveal the conformations of S on the virion, and provide a basis from which to understand interactions between S and neutralizing antibodies during infection or vaccination.\nThe Betacoronavirus SARS-CoV-2 13,14 is an enveloped virus containing a large nucleoprotein (N)-encapsidated positive-sense RNA genome 15 . Three transmembrane proteins are incorporated into the viral lipid envelope: S and two smaller proteins, membrane protein (M) and envelope protein (E) 1,15 . When imaged by cryo-electron microscopy (cryo-EM), betacoronaviruses appear as approximately spherical par- ticles with variable diameters centred around 100&nbsp;nm, containing a dense viroplasm and bounded by a lipid bilayer from which prominent S trimers protrude 16,17 . S trimers of SARS-CoV-2 bind to its receptor on the surface of target cells—angiotensin-converting enzyme 2 (ACE2)—and mediate subsequent viral uptake and fusion 2–5,7 . In so doing, S under- goes a substantial structural rearrangement from the prefusion form to the postfusion form 8 . The overall architectures of both prefusion and postfusion forms are highly conserved among coronaviruses 8,18,19 . During infection, coronaviruses extensively remodel the internal membrane organization of the cell, generating viral replication orga- nelles in which replication takes place 20–22 . The S, M and E proteins are inserted into membranes of the endoplasmic reticulum and traf- fic to the endoplasmic reticulum–Golgi intermediate compartment (ERGIC). The encapsidated genome buds into the ERGIC to form virions, which are then trafficked to the plasma membrane and released 20–22 . S is primed for membrane fusion by proteolytic cleavage at the S1–S2 site, and subsequently the S2′ site 23 .\nThe prefusion structures of S proteins from coronaviruses, including SARS-CoV-2 S, have been extensively studied using ectopic expres- sion of soluble, secreted forms of S, followed by purification and cryo-EM 2,7,9–11 . In the prefusion form, the receptor-binding domain (RBD)\nsits at the top of a broad, trimeric spike above the fusion core. Three copies of the RBD are surrounded by three copies of the N-terminal domain (NTD) which display some mobility 2,7–9 . In the closed prefu- sion conformation, all three copies of the RBD lie flat on the spike surface, largely occluding the receptor-binding site, whereas in the open prefusion conformation, one or multiple RBDs lift to expose the receptor-binding site 2,7,9,10 . The surface of the trimer is extensively glyco- sylated with 22 potential N-linked glycosylation sites per monomer 2,6,7 . After receptor binding, structural transition of the prefusion form to the postfusion form brings the fusion peptide and the transmembrane domain together at one end of a long, needle-like structure centred around a three-helix bundle 8 . Five N-linked glycans are spaced along the length of the postfusion spike 8 .\nFully understanding how S proteins function and how they interact with the immune system requires knowledge of the structures, con- formations and distributions of S trimers within virions. In this study, we apply cryo-EM methods to study the structure, conformations and distributions of S trimers in&nbsp;situ on the virion surface.\nTo avoid artefacts associated with virus concentration or puri- fication, we aimed to image SARS-CoV-2 virions from the superna- tant of infected cells without further concentrating or purifying the virus. Vero E6 cells were infected with SARS-CoV-2 (isolate Germany/ BavPat1/2020) 24 . At 48&nbsp;h after infection, the&nbsp;supernatant was clari- fied, inactivated by fixation with formaldehyde and stored at −80 °C. Western blot analysis revealed that approximately 45% of the total S protein monomers on virions had been cleaved at the multibasic cleav- age site, resulting in S1 and S2 peptides (Fig.&nbsp;1a). The fixed supernatant\nhttps://doi.org/10.1038/s41586-020-2665-2\nReceived: 2 July 2020\nAccepted: 10 August 2020\nPublished online: 17 August 2020\nCheck for updates\n1 Structural Studies Division, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 2 Department of Infectious Diseases, Molecular Virology, Heidelberg University, Heidelberg, Germany. 3 Department of Infectious Diseases, Virology, Heidelberg University, Heidelberg, Germany. 4 Visual Aids Department, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 5 German Center for Infection Research, Heidelberg Partner Site, Heidelberg, Germany. 6 Division of Virus-Associated Carcinogenesis, German Cancer Research Center (DKFZ), Heidelberg, Germany. 7 These authors contributed equally: Zunlong Ke, Joaquin Oton, Kun Qu. ✉ e-mail: jbriggs@mrc-lmb.cam.ac.uk\nwas vitrified by plunge freezing and imaged by cryo-EM. Fixation may help to stabilize some protein conformations by crosslinking, but is not expected to give rise to any new conformations. As expected, given the concentration of virus in cellular supernatants (around 10 7 plaque-forming units per ml), small numbers of individual virions were found scattered around the grid—these were imaged by cryo-electron tomography (cryo-ET) (Fig.&nbsp;1b).\nVirions were approximately spherical, with a diameter to the outside of the lipid bilayer of 91&nbsp;±&nbsp;11&nbsp;nm (n&nbsp;=&nbsp;179) (Extended Data Fig.&nbsp;1a). They contained granular densities corresponding to N, and were studded with S trimers (Fig.&nbsp;1b, c). These features are generally consistent with those of other coronaviruses imaged using cryo-EM 1,16,17 . S trimers pro- truding from the viral surface displayed two distinct morphologies—a minority were extended thin structures reminiscent of the postfusion form, whereas the majority were wider structures reminiscent of the prefusion form. This observation is in contrast to a study showing cryo-EM images of purified SARS-CoV-2 virions inactivated with the nucleic acid modifier β-propiolactone, in which only thin protrusions were seen on the viral surface 25 , but is consistent with in&nbsp;situ observa- tions of virus assembly 21 .\nWe also collected tomograms of SARS-CoV-2 virions produced by infection of Calu-3 cells, a human lung carcinoma cell line that supports virus production with a similar titre to Vero E6 cells. The morphology of the virions and the appearance of the S trimers on the surface were consistent with those for virions produced from Vero E6 cells (Extended Data Fig.&nbsp;2a–c). Western blot analysis showed that approximately 73% of S from Calu-3 supernatant was in the cleaved form (Extended Data Fig.&nbsp;2d).\nIndividual virions contained 24&nbsp;±&nbsp;9 S trimers (Extended Data Fig.&nbsp;1b). This is fewer than previous estimates that assumed a uniform distri- bution of S 21 , because S was not uniformly distributed over the virus surface. A small sub-population of virions contained only few S trim- ers whereas larger virions contained higher numbers of S trimers (Extended Data Fig.&nbsp;1b). We identified 4,104 wide-structure S trimers and 116 thin-structure S trimers from 179 virions and analysed them using subtomogram averaging. The averaged structures, at 7.7 and 22&nbsp;Å resolution, respectively, correspond closely to published structures of purified S trimers in the prefusion and postfusion forms 2,7,8 (Fig.&nbsp;2a).\nOverall, approximately 97% of S trimers were in the prefusion form and 3% were in the postfusion form. Prefusion and postfusion forms were evenly distributed among virions.\nPrefusion S trimers on the virus surface may be predominantly in the closed conformation, with the open conformation induced or stabilized only with ACE2 binding; alternatively, both closed and open prefu- sion trimers may be present in the absence of ACE2. Open or closed conformations may induce different antibodies responses when used as immunogens, and there are ongoing efforts to generate S protein constructs that are stabilized in specific conformations 9,11,12 . To assess whether S trimers are present in open and/or closed conformations, we classified the RBD regions of individual monomers within trimers. We identified three classes of RBD, one with the RBD in the closed position, one with the RBD in the open position, and one in which the RBD was predominantly in the closed position, but showed some weakening of the density, suggesting the presence of more mobile conformations (Extended Data Fig.&nbsp;3). On the basis of our assignment of individual monomers to these classes, we derived structures of fully closed trimers and of trimers in which one RBD was open, representing approximately 31% and 55% of 3,854 prefusion trimers, respectively (Fig.&nbsp;2b, Extended Data Fig.&nbsp;4). We also identified a small number of trimers (around 14% of 3,854 prefusion trimers) in which two RBDs were in the open con- formation (Fig.&nbsp;2b). These observations confirmed that the opening of the RBD observed in recombinant S trimers also takes place on the virus surface, and that artificial S protein constructs stabilized in the closed and open conformations both represent structures present in&nbsp;situ. The receptor-binding site is therefore stochastically exposed in&nbsp;situ and available to interact with ACE2 and with antibodies.\nThe trimers did not all protrude perpendicularly from the viral sur- face. They were tilted by up to 90° towards the membrane, although the frequency of observed tilt angles decreased above 50° (Extended Data Fig.&nbsp;1c, d). We grouped trimers according to their orientation relative to the membrane, and averaged these groups independently. The averaged structures reveal that the membrane-proximal stalk region acts as a hinge with sufficient flexibility to allow tilting in all directions (Fig.&nbsp;2c).\nWe generated models of individual virus particles, with S trim- ers located at the position, orientation and conformation that were\nFig. 1 | Characterization of virus production and images of SARS-CoV-2 virions. a, Western blot analysis of SARS-CoV-2 non-structural protein 3 (nsp3), S and N in lysates of Vero E6 cells and in virus preparations, representative of three experiments. In released virions, S is present in both cleaved and uncleaved forms. The positions of S0, S2 and the S2–S2′ cleavage product are marked. b, Four representative tomographic slices of SARS-CoV-2 virions from\nthe supernatant of infected cells. Virions are approximately spherical, contain granular density corresponding to N-packaged genome, and have S trimers protruding at variable angles from their surfaces. Scale bar, 50 nm. c, Three S trimers from the dataset, shown as projections through the trimer to illustrate variable tilt towards the membrane. Scale bars, 10 nm.\ndetermined by subtomogram averaging (Fig.&nbsp;2d). S trimers appear to be distributed randomly on the viral surface, with no obvious clus- tering or relationship between location, orientation and conforma- tion. There is approximately one trimer per 1,000&nbsp;nm 2 of membrane surface, compared with approximately one per 100&nbsp;nm 2 for influenza A virus 26 . The sparse distribution of S, together with the predominantly closed state, suggest that receptor binding may be less dependent on avidity effects than in the case of pandemic influenza viruses 27 . This is consistent with the higher affinity between S and ACE2 (in the nM range 2,7 ) than between haemagglutinin and sialic acid (mM range 27 ).\nThe low concentration of particles in supernatant makes high-resolution structure determination difficult. We therefore concen- trated the virus by pelleting through a sucrose cushion. Concentrated virions deviated from the spherical morphology (Extended Data Fig.&nbsp;5), but the overall features were preserved. We performed cryo-ET and subtomogram averaging on the particles and observed predominantly\nprefusion S trimers, with occasional postfusion S trimers. Upon classifi- cation of the prefusion S trimers, we were able to identify the RBD only in the closed position; weak RBD density was observed in monomers (Extended Data Fig.&nbsp;3c).\nVirions in the supernatant from infected cells exhibited primarily pre- fusion S trimers in either closed or open prefusion conformations. Viri- ons concentrated through a sucrose cushion also exhibited prefusion conformations, but we did not observe the open conformation. Previ- ous studies have shown that virions inactivated with β-propiolactone rather than formaldehyde are primarily in a postfusion state 25 . S trimers purified from membranes are found only in the closed prefusion and postfusion conformations 8 , whereas the open RBD in soluble S trimers has been found to exhibit a wide range of different conformations 28 . These observations suggest that the open prefusion conformations of the spike protein that we observed before, but not after concentration, are fragile (despite the fixation applied here) and may be affected by the purification procedures.\nFig. 2 | Structural analysis of SARS-CoV-2 S trimers on intact virions. a, Structures of prefusion (left) and postfusion (right) trimers from intact virions, determined by subtomogram averaging. Structures are shown as transparent grey isosurfaces fitted with structures of the closed, prefusion trimer (Protein Data Bank (PDB) 6VXX) and the postfusion trimer (PDB 6XRA). One prefusion monomer is coloured blue (N terminus) to red (C terminus). The NTD is blue, the RBD is cyan. The NTD does not fully occupy the cryo-EM density because some loops are not resolved or built in the template structure (PDB 6VXX). b, Three conformations of the prefusion trimer observed on intact virions: all RBDs in the closed position (left, fitted with PDB 6VXX); one RBD in the open position (centre, fitted with PDB 6VYB); two RBDs in the open position (right, fitted with PDB 6X2B, which lacks modelled glycans). The two-open\nconformation has only been observed in&nbsp;vitro after inserting multiple stabilizing mutations. S monomers with closed RBDs are green, and those with open RBDs are blue. c, Averaging of subsets of trimers grouped according to their orientation relative to the membrane shows flexibility in the stalk region. Examples are shown for pools centred at 0°, 30° and 60° from the perpendicular, and for two rotations of the trimer relative to the tilt direction. d, Three-dimensional models of two individual SARS-CoV-2 virions with a membrane (blue) of the measured radius, and all spike proteins shown in the conformations, positions and orientations determined by subtomogram averaging. Different S conformations are distributed over the virion surface and can be tilted by up to approximately 90° relative to the membrane (Extended Data Fig.&nbsp;1c, d).\nOur data indicate that inactivation and purification methods can alter the ratio of prefusion and postfusion forms, as well as the ratio of open and closed conformations. It has been suggested that sub- stantial amounts of postfusion S on the virus surface may protect the virus by shielding the prefusion form, or may shift the host response towards non-neutralizing antibodies 8 . Our observation that intact virions harbour only a small fraction of postfusion spikes suggest that this is unlikely to be an important defence mechanism for the virus during infection, but it may be an important consideration for vaccina- tion. Candidate vaccines based on inactivated virus particles are under development. These may present different S protein epitopes to the immune system depending on how they are prepared, and may there- fore differ in their ability to induce a neutralizing response. For exam- ple, β-propiolactone is frequently used in vaccine production (such as for influenza virus subunit vaccines 29 ), but if postfusion S induces\nnon-neutralizing responses, β-propiolactone may not be optimal for inactivation during SARS-CoV-2 S vaccine formulation.\nWe next imaged the concentrated virus in 2D using cryo-EM and per- formed single-particle analysis on prefusion S trimers that protruded from the sides of the virus particles, generating a consensus structure of the prefusion trimer at 3.4&nbsp;Å resolution. Focused classification with par- tial signal subtraction on individual RBD monomers led to two classes (Extended Data Fig.&nbsp;5). Consistent with the absence of open conforma- tions in this sample in cryo-ET images, 81% of the monomers showed the RBD in the closed conformation and 19% showed weaker density for the RBD, which remained predominantly in the closed position. We refined the structures of S trimers in which all three RBDs are in the closed conformation (53% of the data), and those in which at least one RBD has weak density (47% of the data), to resolutions of 3.5&nbsp;Å and 4.1&nbsp;Å, respectively (Fig.&nbsp;3a, Extended Data Fig.&nbsp;6). The two structures are highly similar, differing only in the density levels for one RBD. We used the structure with three closed RBDs to build and refine an atomic model of the S protein trimer in&nbsp;situ on the viral surface.\nThe positions of glycans on the surface of S are well resolved in our structure, with density at 17 of the 22 predicted N-glycosylation sites (Fig.&nbsp;3b). The other five glycosylation sites are in disordered NTD loops or in the stalk region and are not resolved at high resolution. At the base of the trimer, a clear ring of glycans forms a collar above the stalk region. The density for the stalk region is extended by two helical turns compared with published structures of soluble ectodomain, and fur- ther at lower resolution before fading owing to stalk flexibility (Fig.&nbsp;3c). SARS-CoV-2 S trimers are sparsely distributed and can be highly tilted towards the membrane. This implies that epitopes at the base of the head domain and in the stalk region would be accessible to antibodies where they are not protected by the extensive glycan shell.\nWe compared the in&nbsp;situ structure of the S trimer with structures previously obtained using exogenously expressed purified protein. A recent study of full-length trimeric S solubilized in detergent micelles 8 identified two features that are not seen in most structures of soluble S ectodomain trimers: well-defined density for residues 14–26 of S, and a folded loop between residues 833 and 853. This loop is folded in the structure of the ‘locked’ conformation of the ectodomain 9 , and may become folded in the low-pH conditions in the endosome 30 . We observed only weak density for residues 14–26, and&nbsp;no folded structure for the 833–853 region. The SARS-CoV-2 strain that we have imaged contains the widely circulating D614G substitution 31 , which abolishes a salt bridge to K854 (Fig.&nbsp;3d) and may reduce folding of the 833–854 loop 9,30 . We did not observe additional density that would correspond to bound lipids, as described recently 10 , or other bound co-factors. These may be present sub-stoichiometrically or in rare conformations, but are not a general feature of the S trimer in&nbsp;situ. Overall, our structure is very similar to that of the soluble trimeric ectodomain in the closed prefusion form stabilized by a double proline mutation 2,7 (Extended Data Fig.&nbsp;7). This validates the ongoing use of recombinant, purified S trimers for research, diagnostics and vaccination—they indeed represent the in&nbsp;situ structure of S. In this study, we have determined the structure of S trimers on the virion surface to 3.4&nbsp;Å resolution using cryo-EM (Fig.&nbsp;3), supporting the further use of this technique to study antibody binding to S in the context of the viral surface. Such studies could provide insights into how neutralizing antibodies block virus infection, particularly for antibodies against membrane-proximal regions of S, and thus can inform the design of immunogens for vaccination.\nOnline content\nAny methods, additional references, Nature Research reporting sum- maries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author con- tributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41586-020-2665-2.\nFig. 3 | Structures of SARS-CoV-2 S trimers on intact virions by single-particle reconstruction. a, Top and side views of trimers with three closed RBDs (left, 3.5&nbsp;Å resolution) and one RBD with weaker density (right, 4.1&nbsp;Å resolution). Individual monomers are coloured white, blue and green. Comparison of the left and right structures shows the weaker density for the RBD of the green monomer in the region indicated by the red arrowheads. b, Glycosylation profile of the S protein. Colour scheme as in a; glycans are shown in orange. Boxes indicates the regions shown in c and d. c, Enlarged view of the base of the trimer at lower isosurface threshold to highlight the glycan ring and the extended C-terminal density. d, Enlarged view of the region of the spike where the D614G variation abolishes a salt bridge to K854.\n1. Neuman, B. W. &amp; Buchmeier, M. J. Supramolecular architecture of the coronavirus particle. Adv. Virus Res. 96 , 1–27 (2016). 2. Walls, A. C. et&nbsp;al. Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein. Cell 181 , 281–292 (2020).\n3.  Hoffmann, M. et&nbsp;al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. Cell 181 , 271–280 (2020).\n4.  Shang, J. et&nbsp;al. Structural basis of receptor recognition by SARS-CoV-2. Nature 581 , 221–224 (2020).\n5. Wang, Q. et&nbsp;al. Structural and functional basis of SARS-CoV-2 entry by using human ACE2. Cell 181 , 894–904 (2020).\n6.  Watanabe, Y., Allen, J. D., Wrapp, D., McLellan, J. S. &amp; Crispin, M. Site-specific glycan analysis of the SARS-CoV-2 spike. Science 369 , 330–333 (2020). 7. Wrapp, D. et&nbsp;al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367 , 1260–1263 (2020). 8. Cai, Y. et&nbsp;al. Distinct conformational states of SARS-CoV-2 spike protein. Science 369 , 1586–1592 (2020).\n9. Xiong, X. et&nbsp;al. A thermostable, closed SARS-CoV-2 spike protein trimer. Nat. Struct. Mol. Biol. 27 , 934–941 (2020).\n10.  Toelzer, C. et&nbsp;al. Free fatty acid binding pocket in the locked structure of SARS-CoV-2 spike protein. Science eabd3255 (2020).\n11.  Henderson, R. et&nbsp;al. Controlling the SARS-CoV-2 spike glycoprotein conformation. Nat. Struct. Mol. Biol. 27 , 925–933&nbsp;(2020).\n12.  McCallum, M., Walls, A. C., Bowen, J. E., Corti, D. &amp; Veesler, D. Structure-guided covalent stabilization of coronavirus spike glycoprotein trimers in the closed conformation. Nat. Struct. Mol. Biol. 27 , 942–949&nbsp;(2020).\n13.  Zhu, N. et&nbsp;al. A novel coronavirus from patients with pneumonia in China, 2019. N. Engl. J. Med. 382 , 727–733 (2020).\n14.  Zhou, P. et&nbsp;al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579 , 270–273 (2020).\n15.  Masters, P. S. &amp; Perlman, S. in Fields Virology 6th edn (eds Knipe, D. M. &amp; Howley, P. M.) 826–858 (Elsevier, 2013).\n16.  Neuman, B. W. et&nbsp;al. Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy. J. Virol. 80 , 7918–7928 (2006). 17.  Bárcena, M. et&nbsp;al. Cryo-electron tomography of mouse hepatitis virus: insights into the structure of the coronavirion. Proc. Natl Acad. Sci. USA 106 , 582–587 (2009).\n18.  Walls, A. C. et&nbsp;al. Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion. Proc. Natl Acad. Sci. USA 114 , 11157–11162 (2017).\n19.  Yuan, Y. et&nbsp;al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat. Commun. 8 , 15092 (2017). 20.  Fehr, A. R. &amp; Perlman, S. Coronaviruses: an overview of their replication and pathogenesis. Methods Mol. Biol. 1282 , 1–23 (2015).\n21.  Klein, S. et&nbsp;al. SARS-CoV-2 structure and replication characterized by in&nbsp;situ cryo-electron tomography. Preprint at https://doi.org/10.1101/2020.06.23.167064 (2020).\n22.  Snijder, E. J. et&nbsp;al. A unifying structural and functional model of the coronavirus replication organelle: tracking down RNA synthesis. PLoS Biol. 18 , e3000715 (2020). 23.  Hoffmann, M., Hofmann-Winkler, H. &amp; Pöhlmann, S. in Activation of Viruses by Host Proteases (eds Böttcher-Friebertshäuser, E. et&nbsp;al.) 71–98 (2018).\n24.  Rothe, C. et&nbsp;al. Transmission of 2019-nCoV infection from an asymptomatic contact in Germany. N. Engl. J. Med. 382 , 970–971 (2020).\n25.  Liu, C. et&nbsp;al. Viral architecture of SARS-CoV-2 with post-fusion spike revealed by cryo-EM. Preprint at https://doi.org/10.1101/2020.03.02.972927 (2020).\n26.  Chlanda, P. et&nbsp;al. The hemifusion structure induced by influenza virus haemagglutinin is determined by physical properties of the target membranes. Nat. Microbiol. 1 , 16050 (2016).\n27.  Xiong, X. et&nbsp;al. Receptor binding by a ferret-transmissible H5 avian influenza virus. Nature 497 , 392–396 (2013).\n28.  Gui, M. et&nbsp;al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res. 27 , 119–129 (2017).\n29.  Bonnafous, P. et&nbsp;al. Treatment of influenza virus with beta-propiolactone alters viral membrane fusion. Biochim. Biophys. Acta 1838 , 355–363 (2014).\n30.  Zhou, T. et&nbsp;al. A pH-dependent switch mediates conformational masking of SARS-CoV-2 spike. Preprint at https://doi.org/10.1101/2020.07.04.187989 (2020).\n31.  Korber, B. et&nbsp;al. Tracking changes in SARS-CoV-2 spike: evidence that D614G increases infectivity of the COVID-19 virus. Cell 182 , 812–827 (2020).\nPublisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n© The Author(s), under exclusive licence to Springer Nature Limited 2020\n\n\nMethods\nNo statistical methods were used to predetermine sample size. For Cryo-ET, division of dataset into two random halves was done based on virus number. For Cryo-EM, division of datasets into two random halves was done based on the standard approach in RELION 3. Other experiments did not involve randomization. The investigators were not blinded to allocation during experiments and outcome assessment.\n\n\nCells and virus\nVero E6 cells were obtained from ATCC and were cultured in Dulbecco’s modified Eagle medium (DMEM, Life Technologies) containing 10% fetal bovine serum, 100 U&nbsp;ml −1 penicillin, 100 μg&nbsp;ml −1 streptomycin and 1% non-essential amino acids (complete medium). The Germany/Bav- Pat1/2020 SARS-CoV-2 strain was isolated by C. Drosten (Charité, Berlin, distributed by the European Virology Archive (Ref-SKU: 026V-03883)) at passage (P)2. A stock of SARS-CoV-2 was obtained by passaging the virus once in Vero E6 cells (P3). To produce SARS-CoV-2 virions, Vero E6 cells grown on 75 cm 2 side-bottom tissue culture flasks were infected with SARS-CoV-2 (P3) at a multiplicity of infection (MOI) of 0.5. Culture medium from infected cells was collected at 48 h after infection, clari- fied by centrifugation at 1,000g for 10 min, cleared through a 0.45-μm nitrocellulose filter and fixed with 4% formaldehyde for 30 min at room temperature. Culture medium was supplemented with 10 mM HEPES (pH 7.2) before fixation. Virus-containing medium was subsequently split into aliquots and stored at −80 °C. Infectious supernatants con- taining SARS-CoV-2 virions were obtained from Calu-3 cells infected with P3 virus at an MOI of 5 for 48 h and processed as described above. To obtain SARS-CoV-2 virions at high concentration, infection and collection of Vero E6 culture medium were performed as above, fol- lowed by concentration of fixation-inactivated virions from medium by ultracentrifugation through a 20% (wt/wt) sucrose cushion (120 min at 27,000 rpm in a Beckman SW32 rotor; Beckman Coulter Life Sciences). Pelleted particles were resuspended in PBS and stored in aliquots at −80 °C.\n\n\nWestern blot\nVero E6 or Calu-3 cells mock infected or infected for 48 h with SARS-CoV-2 (MOI&nbsp;=&nbsp;5) were washed twice in PBS, scraped, pelleted at 700g for 5 min and lysed in PBS pH 7.4, containing 1% Triton&nbsp;X-100 and protease inhibitors (Merck) for 30 min at 4 °C. Samples were centri- fuged at 4 °C for 30 min and supernatants were collected. Total pro- tein concentration was calculated using the Bio-Rad Protein Assay kit (Bio-Rad). Purified viruses were prepared for western blot by cen- trifugation of 32 ml of virus containing supernatants on a 10% sucrose cushion in a Beckmann J25 centrifuge. Centrifugation was performed at 10,000g for 4 h at 4 °C. Supernatants were discarded and purified virus pellet were resuspended in 500 μl PBS. For western blotting, 10 μg of total cell lysates and 5 μl purified viruses were diluted in Laemmli buffer and loaded on a pre-cast Criterion XT 4–12% gradient gel (Bio-Rad). Gels were transfered to PVDF membrane using a wet-electroblotting chamber system (Bio-Rad) in Towbin buffer containing 10% methanol. Transfer was performed overnight at 4 °C. Membranes were washed in PBS and blocked with 10% milk in PBS containing 0.2% tween-20 (PBS-T) for 1 h. Membranes were incubated for 1 h at room temperature with primary antibodies specific for an epitope in the C-terminal region of S (Abcam, ab252690; diluted 1:1,000 in PBS-T), for N protein (Sino Biological, 40143-MM05; diluted 1:1,000 in PBS-T) or nsp3 (Abcam, ab181620; diluted 1:500 in PBS-T). Next, the membranes were washed 3 times in PBS-T, incubated with horseradish peroxidase-conjugated anti-mouse or anti-rabbit antibodies at 1:10,000 dilution for 1 h, washed again 3 times in PBS-T, incubated with western Lightning Plus-ECL rea- gent (Perkin Elmer) and imaged using an Intas ChemoCam Imager 3.2 (Intas). Densitometric analysis of western blot assays was performed using LabImage 1D L340 software v.4.1 (Intas). The blot shown is\nrepresentative of three independent experiments. Uncropped, unpro- cessed blots are presented in the Source Data.\n\n\nPCR with reverse transcription and spike sequencing\nTotal RNA was isolated from infected Vero E6 cells 48 h after infec- tion with Germany/BavPat1/2020 SARS-CoV-2 (P2). Spike cDNA was produced from the total RNA using Superscript III (ThermoFisher) with specific reverse transcription primer (CAATTGTGAAGATTCTC ATA). The cDNA was amplified by PCR using specific primers (Fwrd, ATGTTTGTTTTTCTTGTTTTATT; rev, TTATGTGTAATGTAATTTGA) and the resulting amplicon was sent for Sanger sequencing. Sequences were compared to the Germany/BatPat1/2020 SARS-CoV-2 reference sequence (Ref-SKU: 026V-03883) and found to be identical. Specific sequencing primers: Fwrd1, ATGTTTGTTTTTCTTGTTTTATT; Fwrd2, GGTTGGACAGCTGGTGCT; Fwrd3, CCAACCATACAGAGTAGTAGTA; Rev1, GTAGCAGCAAGATTAGCAGAA; and Rev2, TTATGTGTAATGT AATTTGA.\n\n\nCryo-ET sample preparation\nFixed virus samples from the supernatant of infected cells without any concentration step (unconc) or concentrated by pelleting through a sucrose cushion (conc) were prepared, imaged, and processed in parallel. The virus suspension was mixed with 10-nm colloidal gold (in PBS solution) in 10:1 ratio. Then, 3 μl of the solution was added to a glow-discharged copper grid (C-Flat 2/2, Protochips). Grids were plunge frozen into liquid ethane by back-side blotting using a LeicaGP cryo plunger (Leica) and stored in liquid nitrogen until imaging.\n\n\nCryo-ET data collection\nCryo-ET data collection was performed essentially as described previ- ously 32 . Cryo-grids were loaded into an FEI Titan Krios transmission electron microscope operated at 300 kV and images were recorded on a Gatan K2 Summit direct detection camera in counting mode with a 20 eV energy slit in zero-loss mode. Tomographic tilt series between −60° and +60° were collected using SerialEM 3.8.0 software 33 in a dose-symmetric scheme 34 with a 3° angular increment. A total dose of 120 e − A −2 per tilt series was distributed evenly among 41 tilt images. The nominal mag- nification was 81,000×, giving a pixel size of 1.532 Å on the specimen. The defocus range was between −2 μm and −6 μm and 10 frames were saved for each tilt angle. All data acquisition parameters are listed in Extended Data Table&nbsp;1.\nFrames were motion-corrected in IMOD 4.10.30 35 and images were dose-filtered using the alignframes function in IMOD. Exposure fil- tering was implemented according to the cumulative dose per tilt as described elsewhere 36 . The contrast transfer function (CTF) was meas- ured using non-dose-filtered images using the ctfplotter package within IMOD 37 . Tilt series stacks were sorted using IMOD newstack function and fiducial-alignment of all tilt series was performed in IMOD/etomo. Tomograms with less than three trackable gold fiducials were discarded. Motion-corrected and dose-filtered tilt stacks were CTF-corrected by CTF multiplication and tomograms were reconstructed by weighted back-projection in novaCTF 38 . Tomograms were low-pass filtered to 50 Å for better visualization in EMAN2.2 39 and tomographic slices were visualized with IMOD.\n\n\nExtraction of S trimers from tomograms\nThe initial steps of subtomogram alignment and averaging were imple- mented using MATLAB (MathWorks) scripts, which were derived from the TOM 40 and AV3 41 packages as described previously 32 . The missing wedge was modelled as the summed amplitude spectrum of back- ground subtomograms for each tomogram, and was applied during alignment and averaging.\nTo generate an initial template model of the spike protein from the viral surface, 68 spikes were manually picked from four virions of tomograms that were down-scaled by 4× binning of the voxels.\nThe 68 spikes’ initial Euler angles (2 out of 3) were determined on the basis of the vector between two points, one on the head of the spike and one on the membrane where the spike anchors, respectively. The 68 spikes were iteratively aligned to one another for four iterations, applying three-fold symmetry to generate a low-resolution template that resembled a prefusion conformation of the spike. This template was used as an alignment reference for all virions (below). All postfu- sion spikes were manually identified and picked, and initial Euler angles were assigned in the same manner.\nThe centre of each virion was then marked manually using the Vol- ume Tracer function in UCSF Chimera (v.1.13.1) 42 and the radius of the virion was determined centred at the membrane using the Pick Particle Chimera Plugin 43 . An oversampled spherical grid of points was gener- ated on the virion surface with ~9 nm spacing, and subtomograms were extracted for all grid points with a box size of 96 pixels (approximately 60 nm) centred at a radius 14 nm above these grid positions (approxi- mately the radius of the expected centre of the spike). Initial Euler angles were assigned to each subtomogram based on the orientation of the normal vectors relative to the sphere surface.\nSubtomograms were aligned against the low-resolution template (from the above average of 68 spikes). During this alignment, subto- mogram positions converged onto clusters at the true spike positions. One subtomogram position was kept for each cluster, excluding parti- cles within a distance of ~60 Å (10 pixels at 4× binning) and removing particles with cross-correlation coefficients below 0.11. Subtomo- grams that had tilted by more than 90° relative to their perpendicular positions were excluded. Visual inspection of the tomograms using the Place Object Chimera Plugin 43 confirmed that subtomograms selected in this manner corresponded to S trimers on the viral surface (see below). Subtomograms were divided into two halves on the basis of virion number. From this point on the two halves were processed independently.\n\n\nSubtomogram averaging\nSubsequent processing was performed in RELION 44 . For this purpose, subtomograms were reconstructed from the original tilt series images after motion correction using relion_reconstruct. Using dedicated Python scripts, the S trimer positions in the 3D tomograms from the procedure outlined above were converted into 2D positions and defocus values in the corresponding tilt series images, as well as Euler angles in the RELION convention. Individual sub-tomograms were reconstructed at a 2× downscaled pixel size of 3.064 Å, by 3D insertion of Fourier slices of the cropped regions of the tilt series images, each multiplied by their corresponding CTF, which included the dose filter. Similarly, 3D-CTF volumes were generated by 3D insertion of the cor- responding CTF 2 slices. Subtomograms were reconstructed in a box size of 128 voxels for prefusion trimers and a box size of 192 voxels for averages of tilted prefusion trimers and for postfusion trimers.\nTo deal with the CTF pre-multiplied sub-tomograms, as well as the multiplicity of each 3D voxel in the tilt series, a modified version of RELION was used for subtomogram refinement and classification (details to be described elsewhere). Standard 3D auto-refinement was performed with C3 symmetry and a soft-edged mask around the trimers, using a 30 Å lowpass-filtered map as an initial reference. Using 3,854 of the 4,104 subtomograms (excluding those that had tilted by more than 90° relative to the perpendicular position (Extended Data Fig.&nbsp;1c)), a 7.7 Å consensus map was calculated for the prefusion trim- ers; 116 postfusion trimers led to a 22 Å map.\nAngular density (Extended Data Fig.&nbsp;1c) was illustrated by dividing the number of spikes per histogram bin by the sine of the tilt angle. From this histogram, the noise level in the distribution due to incorrectly aligned spikes was estimated assuming they are uniformly distrib- uted (red dashed line in Extended Data Fig.&nbsp;1c), and based on averaging over the range 140–180°. Such angles must be incorrectly alignments because they orient the spike towards the inside of the virus.\nBased on the subtomograms that contributed to the consensus map we estimated that there are 23&nbsp;±&nbsp;9 prefusion S trimers per virion. For a subset of 28 virions, we visually compared the trimers identified by subtomogram averaging with the tomograms to identify false positives or false negatives, finding that the number of S trimers identified by subtomogram averaging is an underestimate of the number of observed S trimers by 0.9 trimers per virion. On this basis we estimate there are 24&nbsp;±&nbsp;9 prefusion S trimers per virion.\nNext, we performed symmetry expansion 45 , followed by focused classification without alignment and with partial signal subtraction, while keeping the orientations from the consensus refinement fixed. The mask used for focused classification was generated manually and enclosed the RBD of one monomer (Extended Data Fig.&nbsp;3a), including the closest NTD of the neighbouring monomer. Classification of the primary dataset (unconc1) led to three different RBD states: closed (45% of the monomers); open (28%) and with weak density (27%) (Extended Data Fig.&nbsp;3a). Classification of a second, smaller dataset of virus parti- cles in supernatant (unconc2), comprising 1,224 trimers yielded 39% closed RBD, 22% open RBD and 39% weak RBD, in general agreement with the unconc1 results (Extended Data Fig.&nbsp;3b). Classification of a dataset of virus particles after concentration through a sucrose cush- ion (conc), comprising 3,788 trimers, yielded 67% of the monomers corresponding to closed RBDs and 33% to RBDs with weak density. For this dataset, no RBDs in the open conformation were identified (Extended Data Fig.&nbsp;3c).\nThe classification of the RBDs as closed, open or with weaker density was used to divide the unconc1 dataset into three subsets: 31% (1,175 trimers) have no open RBDs; 55% (2,121 trimers) have one RBD in the open state; and 14% (525 trimers) have two open RBDs. The remaining 33 trimers have three RBDs in the open state and not further processed. In the class with weaker density, the RBD appears to be predominantly in the closed state and was treated as closed for this assignment. For each of the three subsets, reconstruction of the two independently refined half sets was performed using the orientations from the consensus refinement that gave the 7.7 Å consensus map described above. Subse- quent standard post-processing procedures for resolution estimation, map sharpening and local-resolution filtering in RELION led to three final maps. The subset with no open RBDs gave a reconstruction with C3 symmetry in which all three RBDs were closed at 8.6 Å resolution. The other two subsets yielded structures with C1 symmetry and either one or two open RBDs, with resolutions of 8.6 Å and 9.9 Å, respectively (Extended Data Figs.&nbsp;3, 4). Note, that while fixation is not expected to give rise to any new conformations, we cannot rule out that it dif- ferentially stabilizes different conformations, and the position of the equilibrium between open and closed conformations in unfixed sam- ples may differ from those estimated from fixed samples.\nAverages of tilted prefusion trimers were generated by grouping according to the tilt and rotation of the subtomogram away from the normal vector to the membrane. Subtomograms were included in a group for averaging if they were within 15° of the displayed tilt (0°, 30° and 60°) and rotation (0°, 60°). The pools illustrated in Fig.&nbsp;2c contain 14.5, 14.1, 12.0 and 10.0% of the subtomograms.\n\n\nCryo-EM sample preparation and data collection\nVirus solution concentrated through a 20% (wt/wt) sucrose cushion was plunge-frozen on C-Flat 2/2 3C grids (Protochips) following the same procedure as for cryo-ET, but without adding gold fiducials. Grids were imaged on a Thermo Fisher Scientific Titan Krios transmission electron microscope that was operated at 300 kV, using a Gatan K3 direct electron detector and a Gatan BioQuantum energy filter with a 20 eV energy slit. Movies with 48 frames and an accumulated dose of 50 e − Å −2 were acquired in counting mode using SerialEM-3.8.0 33 at a nominal magnification of 81,000×, corresponding to a calibrated pixel size of 1.061 Å&nbsp;per&nbsp;pixel. Detailed data acquisition parameters are summarized in Extended Data Table&nbsp;2.\n\n\nCryo-EM image processing\nThe scheduler functionality in RELION-3.1 was used for fully automated real-time processing during data collection 9,46 . Mov- ies were motion-corrected and dose-weighted using the RELION implementation of the MotionCor2 algorithm 47 . Subsequently, non-dose-weighted sums were used to estimate the CTF in CTFFIND-4.1.13 48 . S trimers that were extending from the sides of virus particles were picked manually (4,493 particles from the first 100 micrographs) and then used as a training set for optimisation of the convolutional neural network in the automated particle picking software Topaz 49 . Extracted particles were subjected to 3D classification using a previously determined structure of the S trimer 9 , low-pass-filtered to 30 Å, as initial 3D reference. The selected 286,407 particles that contributed to 3D classes corresponding to S trimers were submitted to Bayesian polishing to correct for per-particle beam-induced motions and a second round of 3D clas- sification to select the 55,159 particles that contributed to the best class. This final consensus set of particles was subjected to CTF refinement of per-particle defocus, per-micrograph astigmatism and beam tilt, followed by a second round of Bayesian polishing. 3D auto-refinements were performed with the selected particles after each round of 3D classification, CTF refinement or Bayesian polishing. The consensus structure had a resolution of 3.4 Å.\nSubsequently, symmetry expansion 45 was performed, followed by focused 3D classification with partial signal subtraction 50 . Using a mask on a single RBD to focus classification into six classes while keeping the orientations of the last consensus refinement fixed resulted in the identification of two RBD states: closed or with weak density. S trimers with all three RBDs in the closed state were refined separately from S trimers with one RBD with weak density, resulting in two final maps with resolutions. Standard RELION post-processing was used for resolution estimation, map sharpening and local-resolution filtering. The C3 symmetric map with three closed RBDs had an estimated overall resolution of 3.5 Å; the C1 map with one weaker RBD extended to 4.1 Å resolution (Extended Data Fig.&nbsp;6b).\n\n\nModel building and refinement\nThe SARS-CoV-2 S trimer structure 9 (PDB 6ZP0) was used as an initial model for building into the model with three closed RBDs (Extended Data Fig.&nbsp;6a, middle). Residues were adjusted manually in Coot 0.9 51 . Steric clash and sidechain rotamer conformations were improved using the Namdinator web server 52 . After further iterations of manual adjustment, the structure was refined in PHENIX-1.18.2 53 . The geom- etry and statistics are given in Extended Data Table&nbsp;2. The unmasked model-to-map FSC was calculated in PHENIX for the refined model against the full reconstruction.\n\n\n3D model of spikes on authentic virions\nTo visualize the spike protein on the authentic SARS-CoV-2 virions, the coordinates, orientations and conformational classes determined by subtomogram averaging were converted into a format compat- ible with Maxon Cinema 4D (v.S22.116), and imported together with the 3D models of the different conformational states determined by subtomogram averaging. To generate representative virion images in Fig.&nbsp;2d we removed the three false positives from one virion and positioned the three false negatives on each virion that had been identified by visual inspection of the data (see above). The HR2 region was modelled as a cylinder. Images of individual virions from the dataset were rendered into Adobe Photoshop to generate images for presentation.\n\n\nReporting summary\nFurther information on research design is available in the&nbsp;Nature Research Reporting Summary linked to this paper.\n\n\nData availability\nPublished structures for comparison or initial model building were obtained from the PDB with accession codes 6VXX 2 , 6VYB 2 , 6X2B 11 , 6XRA 8 and 6ZP0 9 . The cryo-EM and cryo-ET structures determined here and representative tomograms are deposited in the Electron Micros- copy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (three closed RBDs from subtomo- gram averaging), EMD-11495 (one open RBD), EMD-11496 (two open RBDs), EMD-11497 (three closed RBDs from cryo-EM), and EMD-11498 (two open RBDs and one weak RBD). The associated molecular models are deposited in the PDB under accession code 6ZWV (three closed RBDs from cryo-EM). The raw micrographs are deposited in the Electron Microscopy Public Image Archive under accession codes EMPIAR-10492 (cryo-EM) and EMPIAR-10493 (cryo-ET).\n32.  Wan, W. et&nbsp;al. Structure and assembly of the Ebola virus nucleocapsid. Nature 551 , 394–397 (2017).\n33.  Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. J. Struct. Biol. 152 , 36–51 (2005).\n34.  Hagen, W. J. H., Wan, W. &amp; Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197 , 191–198 (2017).\n35.  Kremer, J. R., Mastronarde, D. N. &amp; McIntosh, J. R. Computer visualization of three-dimensional image data using IMOD. J. Struct. Biol. 116 , 71–76 (1996).\n36.  Grant, T. &amp; Grigorieff, N. Measuring the optimal exposure for single particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6. eLife 4 , e06980 (2015).\n37.  Xiong, Q., Morphew, M. K., Schwartz, C. L., Hoenger, A. H. &amp; Mastronarde, D. N. CTF determination and correction for low dose tomographic tilt series. J. Struct. Biol. 168 , 378–387 (2009).\n38.  Turoňová, B., Schur, F. K. M., Wan, W. &amp; Briggs, J. A. G. Efficient 3D-CTF correction for cryo-electron tomography using NovaCTF improves subtomogram averaging resolution to 3.4 Å. J. Struct. Biol. 199 , 187–195 (2017).\n39.  Galaz-Montoya, J. G., Flanagan, J., Schmid, M. F. &amp; Ludtke, S. J. Single particle tomography in EMAN2. J. Struct. Biol. 190 , 279–290 (2015).\n40.  Nickell, S. et&nbsp;al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149 , 227–234 (2005).\n41.  Förster, F., Medalia, O., Zauberman, N., Baumeister, W. &amp; Fass, D. Retrovirus envelope protein complex structure in&nbsp;situ studied by cryo-electron tomography. Proc. Natl Acad. Sci. USA 102 , 4729–4734 (2005).\n42.  Pettersen, E. F. et&nbsp;al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25 , 1605–1612 (2004).\n43.  Qu, K. et&nbsp;al. Structure and architecture of immature and mature murine leukemia virus capsids. Proc. Natl Acad. Sci. USA 115 , E11751–E11760 (2018).\n44.  Bharat, T. A. M., Russo, C. J., Löwe, J., Passmore, L. A. &amp; Scheres, S. H. W. Advances in single-particle electron cryomicroscopy structure determination applied to sub-tomogram averaging. Structure 23 , 1743–1753 (2015).\n45.  Scheres, S. H. W. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579 , 125–157 (2016).\n46.  Zivanov, J. et&nbsp;al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7 , e42166 (2018).\n47.  Zheng, S. Q. et&nbsp;al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14 , 331–332 (2017).\n48.  Rohou, A. &amp; Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192 , 216–221 (2015).\n49.  Bepler, T. et&nbsp;al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Nat. Methods 16 , 1153–1160 (2019).\n50.  Bai, X. C., Rajendra, E., Yang, G., Shi, Y. &amp; Scheres, S. H. W. Sampling the conformational space of the catalytic subunit of human γ-secretase. eLife 4 , e11182 (2015). 51.  Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. Acta Crystallogr. D 60 , 2126–2132 (2004).\n52.  Kidmose, R. T. et&nbsp;al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6 , 526–531 (2019).\n53.  Afonine, P. V. et&nbsp;al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr. D 74 , 531–544 (2018).\nAcknowledgements We thank the staff of the MRC-LMB for generous support during the COVID-19 pandemic lockdown; all of&nbsp;the staff of the MRC-LMB EM Facility, in particular A. Yeates, G. Sharov and G. Cannone, for supporting the EM experiments; J. Grimmett and T. Darling for supporting scientific computing; C. Drosten and EVAg for provision of the SARS-CoV-2 strain used here; and M. Frey for the Calu-3 cell line. We acknowledge microscopy support from the Infectious Diseases Imaging Platform (IDIP) at the Center for Integrative Infectious Disease Research Heidelberg. This study was supported by funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (ERC-CoG-648432 MEMBRANEFUSION to J.A.G.B.), the Medical Research Council as part of UK Research and Innovation (MC_UP_A025_1013 to S.H.W.S.; MC_UP_1201/16 to J.A.G.B.), the Deutsche Forschungsgemeinschaft (240245660–SFB 1129 project 5 to H.-G.K. and project 11 to R.B.) and the Japan Society for the Promotion of Science (overseas research fellowship to T.N.).\nAuthor contributions Z.K. and J.A.G.B. conceived the study. Z.K., J.O., K.Q., M.C., V.Z., H.-G.K., R.B. and J.A.G.B. designed the study. M.C., V.Z., C.J.N. and B.C. prepared and characterized virus samples with supervision from H.-G.K. and R.B. Z.K. and K.Q. collected cryo-EM data. Z.K., J.O. and K.Q. processed cryo-EM data with assistance from T.N., J.Z., J.M.L. and J.P., supervised by S.H.W.S. and J.A.G.B. Z.K., J.O., K.Q., M.C. and L.M. made figures. Z.K., J.O., K.Q., M.C., X.X. and J.A.G.B. interpreted data. J.A.G.B. prepared the initial draft and managed the project. Z.K., J.O., K.Q., M.C. and J.A.G.B. wrote the manuscript with input from all authors. S.H.W.S., H.-G.K., R.B. and J.A.G.B. obtained funding.\nCompeting interests The authors declare no competing interests.\nAdditional information Supplementary information is available for this paper at https://doi.org/10.1038/s41586-020- 2665-2.\nCorrespondence and requests for materials should be addressed to J.A.G.B. Peer review information Nature thanks Jason McLellan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Reprints and permissions information is available at http://www.nature.com/reprints.\nExtended Data Fig. 1 | Characterization of SARS-CoV-2 virion morphology. a, Histogram of virion diameters for unconcentrated extracellular virions in the supernatant of two independent preparations (top and middle), and for extracellular virions after concentration through a sucrose cushion (bottom). After concentration the virions become less spherical. Mean and standard deviation for diameters are 91&nbsp;±&nbsp;11 nm (n&nbsp;=&nbsp;179), 94&nbsp;±&nbsp;9 nm (n&nbsp;=&nbsp;68) and 92&nbsp;±&nbsp;8 nm (n&nbsp;=&nbsp;227) for the three preparations. b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. Visual inspection indicates that almost all spikes were identified for virions in the supernatant, but that not all spikes are identified in the concentrated preparation leading to an underestimate of the number of spikes. c, Histogram of spike tilt angle towards the membrane for the\nlarger supernatant virus dataset (unconc1). The vertical black dashed line indicates 90°. 97% of particles have tilts below 90°; particles with tilts above 90° were not included in image analysis. The angular density (right) is calculated by dividing the number of spikes by the sine of the determined angle. If spikes were unconstrained in tilt, this distribution would be uniform. The angular density decreases from ~50°, indicating that higher tilts are disfavoured. The horizontal red dashed line indicates the angular distribution of noise (spikes which have failed to align), estimated based on the angular density between 140° and 180°. d, Schematic diagram and examples of individual tilted spikes on virions. The schematic indicates the angle that was measured. Five examples of individual tilted spikes are marked on tomographic slices through an intact virion, with their associated angle. Scale bar 50 nm.\nExtended Data Fig. 2 | Morphology of SARS-CoV-2 virions released from infected Calu-3 cells. As in Extended Data Fig.&nbsp;1. a, Histogram of virion diameters. Mean and standard deviation for diameters are 104&nbsp;±&nbsp;13 nm (n&nbsp;=&nbsp;67). b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. c, Central slices through three representative viruses from 67 imaged in one experiment. Virions from Calu-3 cells had a slightly broader diameter distribution than those from Vero E6 cells. Scale bar 50 nm. d, Western blot analysis of SARS-CoV-2 nsp3, S and N in cell lysates and in virus preparations. In released virions, S is present in both cleaved (S2, 73%) and uncleaved forms (S0, 27%).\nExtended Data Fig. 3 | Classification of SARS-CoV-2 spike RBDs. a, Class averages obtained after focused classification on the RBD of the left monomer after symmetry expansion of the unconc1 dataset. Top views and side views are shown for closed, open and weak classes. The region subjected to classification is indicated by a transparent red mask surface in the left hand\npanel. b, Equivalent analysis for a smaller, independent dataset (unconc2). c, Equivalent analysis for a dataset obtained after concentrating virus through a sucrose cushion (conc). Only closed and weak classes were obtained. d–f, Cut-open local resolution maps for structures shown in a–c.\nExtended Data Fig. 4 | Resolution assessment of subtomogram averaging structures. a, Local resolution map for the consensus structure obtained for the prefusion S trimers. b, Local resolution maps for the prefusion S trimer in three different conformations. c, Global resolution assessment by Fourier shell correlation (FSC) at the 0.143 criterion for the four structures shown in a and b, as well as the postfusion S trimer.\nExtended Data Fig. 5 | Single particle cryo-EM image processing workflow. Automatically picked particles (green circles) were subjected to 3D classification. Scale bar 100 nm. Selected 3D classes are indicated by black boxes. RBDs from individual asymmetric units from the S trimer (red dashed circles) were locally classified to sort different conformations of RBD. The\nasymmetric unit subjected to local classification is shown in a top view, the RDB of the green monomer is weak in the right-hand class (red arrowhead). S trimers with all three RBDs in the closed state were further refined with C3 symmetry. S trimers where one RBD had weak density were refined with C1 symmetry. For further details see materials and methods.\nExtended Data Fig. 6 | Single particle Cryo-EM structure validation. a, Cut-open cryo-EM maps obtained using all prefusion S trimers, S trimers with 3 closed RBDs or S trimers with 2 closed and 1 weak RBDs, coloured according to the local resolution. b, FSC curves for the three structures in a, and for the atomic model against the map.\nExtended Data Fig. 7 | Structural comparison of in&nbsp;situ structure with recombinant soluble structure. Structural superposition of S trimer modelled into the structure of the trimer with three closed RBDs\n(green, this study) with the published structure of recombinant, soluble closed trimer (blue, PDB 6VXX). Top and side views are shown. The structures are very similar.\nExtended Data Table 1 | Cryo-ET data acquisition and image processing\n\n\nExtended Data Table 2 | Cryo-EM data collection, refinement and validation statistics\n\n\nReporting Summary\nNature Research wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see our Editorial Policies and the Editorial Policy Checklist .\n\n\nStatistics\nFor all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section.\nn/a Confirmed\nThe exact sample size ( n ) for each experimental group/condition, given as a discrete number and unit of measurement\nA statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly\nThe statistical test(s) used AND whether they are one- or two-sided\nOnly common tests should be described solely by name; describe more complex techniques in the Methods section.\n\n\nA description of all covariates tested\nA description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons\nA full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals)\nFor null hypothesis testing, the test statistic (e.g. F , t , r ) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable.\nFor Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings\nFor hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes\nEstimates of effect sizes (e.g. Cohen's d , Pearson's r ), indicating how they were calculated\nOur web collection on statistics for biologists contains articles on many of the points above.\n\n\nSoftware and code\nPolicy information about availability of computer code\nData collection SerialEM 3.8.0 for operation of electron microscope (available and referenced in the Methods section)\nData analysis CryoET data processing (all commercial or available and referenced in methods): CTFFIND4; IMOD 4.10; NovaCTF 1.0.0, EMAN2.2; Matlab R2016b; TOM (no version number); AV3 (no version number); Python 3.7; RELION 3.1.\nCryoEM data processing (all available and referenced in methods): CTFFIND4.1.13; RELION 3.1.\nModelling/structure refinement/visualization (all available and referenced in methods): Coot 0.9; Namdinator (no version numbering); PHENIX 1.18.2; Chimera 1.13.1; Maxon Cinema 4D version S22.116\nWestern blot quantification: LabImage 1D L340 software version 4.1 (Intas, Göttingen)\nFor manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors and reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Research guidelines for submitting code &amp; software for further information.\n\n\nPolicy information about availability of data\nAll manuscripts must include a data availability statement . This statement should provide the following information, where applicable: - Accession codes, unique identifiers, or web links for publicly available datasets - A list of figures that have associated raw data\n- A description of any restrictions on data availability\nPublished structures for comparison or initial model building were obtained from the protein data bank (PDB) with accession codes 6VXX, 6VYB, 6X2B, 6XRA and 6ZP0. The cryo-EM and cryo-ET structures determined here, and representative tomograms are deposited in the Electron Microscopy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (3 closed RBDs from subtomogram averaging), EMD-11495 (1 open RBD), EMD-11496 (2 open RBDs), EMD-11497 (3 closed RBDs from cryo-EM), and EMD-11498 (2 open and 1 weak RBDs). The associated molecular models are deposited in the PDB under accession codes 6ZWV (3 closed RBDs from cryo-EM).\n\n\nField-specific reporting\nPlease select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection.\nLife sciences Behavioural &amp; social sciences Ecological, evolutionary &amp; environmental sciences\nFor a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf\n\n\nLife sciences study design\n\n\nAll studies must disclose on these points even when the disclosure is negative.\nSample size Sample sizes were determined by available electron microscopy time and the number of virus particles on electron microscopy grids. The sample size is sufficient to obtain a structure at the reported resolution, as assessed by Fourier shell correlation.\nData exclusions For all cryo-ET experiments, tomograms which could not be used due to objects (contamination, ice, grid bars etc) obscuring the virus particle were discarded.\nReplication For cryoET and cryoEM, structures were determined from independent half datasets, which were compared to assess the resolution of the reconstruction, as described in the methods section. Two independent preparations of virus in supernatant were analysed as described in the methods section.\nWestern blots are representative of three independent experiments.\nRandomization For CryoET, division of dataset into two random halves was done based on virus number. For CryoEM, division of datasets into two random halves was done based on standard approach in RELION 3. Other experiments did not involve randomization.\n\n\nBlinding Blinding was not applicable to this study because this type of study does not use group allocation.\n\n\nReporting for specific materials, systems and methods\nWe require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.\nMaterials &amp; experimental systems n/a Involved in the study\nAntibodies\nEukaryotic cell lines\nPalaeontology and archaeology\nAnimals and other organisms\nHuman research participants\nClinical data\nDual use research of concern\nMRI-based neuroimaging\n\n\nAntibodies\nAntibodies used Anti-mouse HRP Sigma A4416 Anti-rabbit HRP Sigma A6154\nBeta actin Sigma A5441\nSpike Abcam Ab252690\n\n\nValidation\nCommercial antibodies validated as per manufacturers website: Beta actin Sigma A5441 Immunoblot on chicken fibroblast cell extracts Spike Abcam Ab252690 Validated by ELISA on free peptide from SARS-CoV-1 Nsp3 abcam ab181620 Validated by western blot on SARS-CoV-1 infected cells Nucleocapsid Sino Biological 40143-MM05 Validated by western blot with corresponding viruses\n\n\n245 190 135 100 80 58 46 32 25 22 17 11 58 46 200 Nucleoprotein Actin Nsp3 Spike Cell lysat e Virions b a c S2-S2 ′ S0 S2 SARS-CoV-2 – + + + kDa\n\n\n3 closed RBDs 1 open RBD 2 open RBDs a c b 90° 90° 90° 3 closed RBDs 1 open RBD 2 open RBDs Postfusion Postfusion Prefusion 0º 30º 60º d 60º, rotation 0º 60º, rotation 60º 90°\n\n\n90° a b 90° 3 closed RBDs 2 closed + 1 weak RBDs b G614 c d c d K854\n\n\nCorresponding author(s): John Briggs Last updated by author(s): Jul 28, 2020\n\n\nMethods n/a Involved in the study ChIP-seq Flow cytometry\n\n\n3 Closed RBDs (EMD-11497, PDB 6ZWV) 2 Closed + 1 Weak RBDs (EMD-11498) Data collection and processing Magnification 81,000 81,000 Voltage (kV) 300 300 Electron exposure (e – /Å 2 ) 50 50 Defocus range (µm) 1.0-3.0 1.0-3.0 Pixel size (Å) 1.061 1.061 Movies (no.) 7,982 7,982 Symmetry imposed C3 C1 Initial particle images (no.) 450,473 450,473 Final particle images (no.) 29,183 25,976 Map resolution (Å) 3.5 4.1 FSC threshold 0.143 0.143 Map resolution range (Å) 3.35-27.16 3.67-45.27 Refinement Initial model used PDB 6ZP0 Model resolution (Å) 3.5 FSC threshold 0.5 Map sharpening B factor (Å 2 ) -50 Model composition Non-hydrogen atoms 24036 Protein residues 2955 Ligands 63 B factors (Å 2 ) Protein 130.57 Ligand 142.74 R.m.s. deviations Bond lengths (Å) 0.009 Bond angles (°) 1.346 Validation MolProbity score 1.62 Clashscore 2.97 Poor rotamers (%) 1.74 Ramachandran plot Favored (%) 94.97 Allowed (%) 5.03 Disallowed (%) 0.00\n\n\nAuthentication Cells were not further authenticated\n",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Structures and distributions of SARS-CoV-2 spike proteins on intact virions\nZunlong Ke 1,7 , Joaquin Oton 1,7 , Kun Qu 1,7 , Mirko Cortese 2 , Vojtech Zila 3 , Lesley McKeane 4 , Takanori Nakane 1 , Jasenko Zivanov 1 , Christopher J. Neufeldt 2 , Berati Cerikan 2 , John M. Lu 1 , Julia Peukes 1 , Xiaoli Xiong 1 , Hans-Georg Kräusslich 3,5 , Sjors H. W. Scheres 1 , Ralf Bartenschlager 2,5,6 &amp; John A. G. Briggs 1 ✉\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virions are surrounded by a lipid bilayer from which spike (S) protein trimers protrude 1 . Heavily glycosylated S trimers bind to&nbsp;the angiotensin-converting enzyme 2 receptor and mediate entry of virions into target cells 2–6 . S exhibits extensive conformational flexibility: it modulates exposure of its receptor-binding site and subsequently undergoes complete structural rearrangement to drive fusion of viral and cellular membranes 2,7,8 . The structures and conformations of soluble, overexpressed, purified S proteins have been studied in detail using cryo-electron microscopy 2,7,9–12 , but the structure and distribution of S on the virion surface remain unknown. Here we applied cryo-electron microscopy and tomography to image intact SARS-CoV-2 virions and determine the high-resolution structure, conformational flexibility and distribution of S trimers in&nbsp;situ on the virion surface. These results reveal the conformations of S on the virion, and provide a basis from which to understand interactions between S and neutralizing antibodies during infection or vaccination.\nThe Betacoronavirus SARS-CoV-2 13,14 is an enveloped virus containing a large nucleoprotein (N)-encapsidated positive-sense RNA genome 15 . Three transmembrane proteins are incorporated into the viral lipid envelope: S and two smaller proteins, membrane protein (M) and envelope protein (E) 1,15 . When imaged by cryo-electron microscopy (cryo-EM), betacoronaviruses appear as approximately spherical par- ticles with variable diameters centred around 100&nbsp;nm, containing a dense viroplasm and bounded by a lipid bilayer from which prominent S trimers protrude 16,17 . S trimers of SARS-CoV-2 bind to its receptor on the surface of target cells—angiotensin-converting enzyme 2 (ACE2)—and mediate subsequent viral uptake and fusion 2–5,7 . In so doing, S under- goes a substantial structural rearrangement from the prefusion form to the postfusion form 8 . The overall architectures of both prefusion and postfusion forms are highly conserved among coronaviruses 8,18,19 . During infection, coronaviruses extensively remodel the internal membrane organization of the cell, generating viral replication orga- nelles in which replication takes place 20–22 . The S, M and E proteins are inserted into membranes of the endoplasmic reticulum and traf- fic to the endoplasmic reticulum–Golgi intermediate compartment (ERGIC). The encapsidated genome buds into the ERGIC to form virions, which are then trafficked to the plasma membrane and released 20–22 . S is primed for membrane fusion by proteolytic cleavage at the S1–S2 site, and subsequently the S2′ site 23 .\nThe prefusion structures of S proteins from coronaviruses, including SARS-CoV-2 S, have been extensively studied using ectopic expres- sion of soluble, secreted forms of S, followed by purification and cryo-EM 2,7,9–11 . In the prefusion form, the receptor-binding domain (RBD)\nsits at the top of a broad, trimeric spike above the fusion core. Three copies of the RBD are surrounded by three copies of the N-terminal domain (NTD) which display some mobility 2,7–9 . In the closed prefu- sion conformation, all three copies of the RBD lie flat on the spike surface, largely occluding the receptor-binding site, whereas in the open prefusion conformation, one or multiple RBDs lift to expose the receptor-binding site 2,7,9,10 . The surface of the trimer is extensively glyco- sylated with 22 potential N-linked glycosylation sites per monomer 2,6,7 . After receptor binding, structural transition of the prefusion form to the postfusion form brings the fusion peptide and the transmembrane domain together at one end of a long, needle-like structure centred around a three-helix bundle 8 . Five N-linked glycans are spaced along the length of the postfusion spike 8 .\nFully understanding how S proteins function and how they interact with the immune system requires knowledge of the structures, con- formations and distributions of S trimers within virions. In this study, we apply cryo-EM methods to study the structure, conformations and distributions of S trimers in&nbsp;situ on the virion surface.\nTo avoid artefacts associated with virus concentration or puri- fication, we aimed to image SARS-CoV-2 virions from the superna- tant of infected cells without further concentrating or purifying the virus. Vero E6 cells were infected with SARS-CoV-2 (isolate Germany/ BavPat1/2020) 24 . At 48&nbsp;h after infection, the&nbsp;supernatant was clari- fied, inactivated by fixation with formaldehyde and stored at −80 °C. Western blot analysis revealed that approximately 45% of the total S protein monomers on virions had been cleaved at the multibasic cleav- age site, resulting in S1 and S2 peptides (Fig.&nbsp;1a). The fixed supernatant\nhttps://doi.org/10.1038/s41586-020-2665-2\nReceived: 2 July 2020\nAccepted: 10 August 2020\nPublished online: 17 August 2020\nCheck for updates\n1 Structural Studies Division, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 2 Department of Infectious Diseases, Molecular Virology, Heidelberg University, Heidelberg, Germany. 3 Department of Infectious Diseases, Virology, Heidelberg University, Heidelberg, Germany. 4 Visual Aids Department, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 5 German Center for Infection Research, Heidelberg Partner Site, Heidelberg, Germany. 6 Division of Virus-Associated Carcinogenesis, German Cancer Research Center (DKFZ), Heidelberg, Germany. 7 These authors contributed equally: Zunlong Ke, Joaquin Oton, Kun Qu. ✉ e-mail: jbriggs@mrc-lmb.cam.ac.uk\nwas vitrified by plunge freezing and imaged by cryo-EM. Fixation may help to stabilize some protein conformations by crosslinking, but is not expected to give rise to any new conformations. As expected, given the concentration of virus in cellular supernatants (around 10 7 plaque-forming units per ml), small numbers of individual virions were found scattered around the grid—these were imaged by cryo-electron tomography (cryo-ET) (Fig.&nbsp;1b).\nVirions were approximately spherical, with a diameter to the outside of the lipid bilayer of 91&nbsp;±&nbsp;11&nbsp;nm (n&nbsp;=&nbsp;179) (Extended Data Fig.&nbsp;1a). They contained granular densities corresponding to N, and were studded with S trimers (Fig.&nbsp;1b, c). These features are generally consistent with those of other coronaviruses imaged using cryo-EM 1,16,17 . S trimers pro- truding from the viral surface displayed two distinct morphologies—a minority were extended thin structures reminiscent of the postfusion form, whereas the majority were wider structures reminiscent of the prefusion form. This observation is in contrast to a study showing cryo-EM images of purified SARS-CoV-2 virions inactivated with the nucleic acid modifier β-propiolactone, in which only thin protrusions were seen on the viral surface 25 , but is consistent with in&nbsp;situ observa- tions of virus assembly 21 .\nWe also collected tomograms of SARS-CoV-2 virions produced by infection of Calu-3 cells, a human lung carcinoma cell line that supports virus production with a similar titre to Vero E6 cells. The morphology of the virions and the appearance of the S trimers on the surface were consistent with those for virions produced from Vero E6 cells (Extended Data Fig.&nbsp;2a–c). Western blot analysis showed that approximately 73% of S from Calu-3 supernatant was in the cleaved form (Extended Data Fig.&nbsp;2d).\nIndividual virions contained 24&nbsp;±&nbsp;9 S trimers (Extended Data Fig.&nbsp;1b). This is fewer than previous estimates that assumed a uniform distri- bution of S 21 , because S was not uniformly distributed over the virus surface. A small sub-population of virions contained only few S trim- ers whereas larger virions contained higher numbers of S trimers (Extended Data Fig.&nbsp;1b). We identified 4,104 wide-structure S trimers and 116 thin-structure S trimers from 179 virions and analysed them using subtomogram averaging. The averaged structures, at 7.7 and 22&nbsp;Å resolution, respectively, correspond closely to published structures of purified S trimers in the prefusion and postfusion forms 2,7,8 (Fig.&nbsp;2a).\nOverall, approximately 97% of S trimers were in the prefusion form and 3% were in the postfusion form. Prefusion and postfusion forms were evenly distributed among virions.\nPrefusion S trimers on the virus surface may be predominantly in the closed conformation, with the open conformation induced or stabilized only with ACE2 binding; alternatively, both closed and open prefu- sion trimers may be present in the absence of ACE2. Open or closed conformations may induce different antibodies responses when used as immunogens, and there are ongoing efforts to generate S protein constructs that are stabilized in specific conformations 9,11,12 . To assess whether S trimers are present in open and/or closed conformations, we classified the RBD regions of individual monomers within trimers. We identified three classes of RBD, one with the RBD in the closed position, one with the RBD in the open position, and one in which the RBD was predominantly in the closed position, but showed some weakening of the density, suggesting the presence of more mobile conformations (Extended Data Fig.&nbsp;3). On the basis of our assignment of individual monomers to these classes, we derived structures of fully closed trimers and of trimers in which one RBD was open, representing approximately 31% and 55% of 3,854 prefusion trimers, respectively (Fig.&nbsp;2b, Extended Data Fig.&nbsp;4). We also identified a small number of trimers (around 14% of 3,854 prefusion trimers) in which two RBDs were in the open con- formation (Fig.&nbsp;2b). These observations confirmed that the opening of the RBD observed in recombinant S trimers also takes place on the virus surface, and that artificial S protein constructs stabilized in the closed and open conformations both represent structures present in&nbsp;situ. The receptor-binding site is therefore stochastically exposed in&nbsp;situ and available to interact with ACE2 and with antibodies.\nThe trimers did not all protrude perpendicularly from the viral sur- face. They were tilted by up to 90° towards the membrane, although the frequency of observed tilt angles decreased above 50° (Extended Data Fig.&nbsp;1c, d). We grouped trimers according to their orientation relative to the membrane, and averaged these groups independently. The averaged structures reveal that the membrane-proximal stalk region acts as a hinge with sufficient flexibility to allow tilting in all directions (Fig.&nbsp;2c).\nWe generated models of individual virus particles, with S trim- ers located at the position, orientation and conformation that were\nFig. 1 | Characterization of virus production and images of SARS-CoV-2 virions. a, Western blot analysis of SARS-CoV-2 non-structural protein 3 (nsp3), S and N in lysates of Vero E6 cells and in virus preparations, representative of three experiments. In released virions, S is present in both cleaved and uncleaved forms. The positions of S0, S2 and the S2–S2′ cleavage product are marked. b, Four representative tomographic slices of SARS-CoV-2 virions from\nthe supernatant of infected cells. Virions are approximately spherical, contain granular density corresponding to N-packaged genome, and have S trimers protruding at variable angles from their surfaces. Scale bar, 50 nm. c, Three S trimers from the dataset, shown as projections through the trimer to illustrate variable tilt towards the membrane. Scale bars, 10 nm.\ndetermined by subtomogram averaging (Fig.&nbsp;2d). S trimers appear to be distributed randomly on the viral surface, with no obvious clus- tering or relationship between location, orientation and conforma- tion. There is approximately one trimer per 1,000&nbsp;nm 2 of membrane surface, compared with approximately one per 100&nbsp;nm 2 for influenza A virus 26 . The sparse distribution of S, together with the predominantly closed state, suggest that receptor binding may be less dependent on avidity effects than in the case of pandemic influenza viruses 27 . This is consistent with the higher affinity between S and ACE2 (in the nM range 2,7 ) than between haemagglutinin and sialic acid (mM range 27 ).\nThe low concentration of particles in supernatant makes high-resolution structure determination difficult. We therefore concen- trated the virus by pelleting through a sucrose cushion. Concentrated virions deviated from the spherical morphology (Extended Data Fig.&nbsp;5), but the overall features were preserved. We performed cryo-ET and subtomogram averaging on the particles and observed predominantly\nprefusion S trimers, with occasional postfusion S trimers. Upon classifi- cation of the prefusion S trimers, we were able to identify the RBD only in the closed position; weak RBD density was observed in monomers (Extended Data Fig.&nbsp;3c).\nVirions in the supernatant from infected cells exhibited primarily pre- fusion S trimers in either closed or open prefusion conformations. Viri- ons concentrated through a sucrose cushion also exhibited prefusion conformations, but we did not observe the open conformation. Previ- ous studies have shown that virions inactivated with β-propiolactone rather than formaldehyde are primarily in a postfusion state 25 . S trimers purified from membranes are found only in the closed prefusion and postfusion conformations 8 , whereas the open RBD in soluble S trimers has been found to exhibit a wide range of different conformations 28 . These observations suggest that the open prefusion conformations of the spike protein that we observed before, but not after concentration, are fragile (despite the fixation applied here) and may be affected by the purification procedures.\nFig. 2 | Structural analysis of SARS-CoV-2 S trimers on intact virions. a, Structures of prefusion (left) and postfusion (right) trimers from intact virions, determined by subtomogram averaging. Structures are shown as transparent grey isosurfaces fitted with structures of the closed, prefusion trimer (Protein Data Bank (PDB) 6VXX) and the postfusion trimer (PDB 6XRA). One prefusion monomer is coloured blue (N terminus) to red (C terminus). The NTD is blue, the RBD is cyan. The NTD does not fully occupy the cryo-EM density because some loops are not resolved or built in the template structure (PDB 6VXX). b, Three conformations of the prefusion trimer observed on intact virions: all RBDs in the closed position (left, fitted with PDB 6VXX); one RBD in the open position (centre, fitted with PDB 6VYB); two RBDs in the open position (right, fitted with PDB 6X2B, which lacks modelled glycans). The two-open\nconformation has only been observed in&nbsp;vitro after inserting multiple stabilizing mutations. S monomers with closed RBDs are green, and those with open RBDs are blue. c, Averaging of subsets of trimers grouped according to their orientation relative to the membrane shows flexibility in the stalk region. Examples are shown for pools centred at 0°, 30° and 60° from the perpendicular, and for two rotations of the trimer relative to the tilt direction. d, Three-dimensional models of two individual SARS-CoV-2 virions with a membrane (blue) of the measured radius, and all spike proteins shown in the conformations, positions and orientations determined by subtomogram averaging. Different S conformations are distributed over the virion surface and can be tilted by up to approximately 90° relative to the membrane (Extended Data Fig.&nbsp;1c, d).\nOur data indicate that inactivation and purification methods can alter the ratio of prefusion and postfusion forms, as well as the ratio of open and closed conformations. It has been suggested that sub- stantial amounts of postfusion S on the virus surface may protect the virus by shielding the prefusion form, or may shift the host response towards non-neutralizing antibodies 8 . Our observation that intact virions harbour only a small fraction of postfusion spikes suggest that this is unlikely to be an important defence mechanism for the virus during infection, but it may be an important consideration for vaccina- tion. Candidate vaccines based on inactivated virus particles are under development. These may present different S protein epitopes to the immune system depending on how they are prepared, and may there- fore differ in their ability to induce a neutralizing response. For exam- ple, β-propiolactone is frequently used in vaccine production (such as for influenza virus subunit vaccines 29 ), but if postfusion S induces\nnon-neutralizing responses, β-propiolactone may not be optimal for inactivation during SARS-CoV-2 S vaccine formulation.\nWe next imaged the concentrated virus in 2D using cryo-EM and per- formed single-particle analysis on prefusion S trimers that protruded from the sides of the virus particles, generating a consensus structure of the prefusion trimer at 3.4&nbsp;Å resolution. Focused classification with par- tial signal subtraction on individual RBD monomers led to two classes (Extended Data Fig.&nbsp;5). Consistent with the absence of open conforma- tions in this sample in cryo-ET images, 81% of the monomers showed the RBD in the closed conformation and 19% showed weaker density for the RBD, which remained predominantly in the closed position. We refined the structures of S trimers in which all three RBDs are in the closed conformation (53% of the data), and those in which at least one RBD has weak density (47% of the data), to resolutions of 3.5&nbsp;Å and 4.1&nbsp;Å, respectively (Fig.&nbsp;3a, Extended Data Fig.&nbsp;6). The two structures are highly similar, differing only in the density levels for one RBD. We used the structure with three closed RBDs to build and refine an atomic model of the S protein trimer in&nbsp;situ on the viral surface.\nThe positions of glycans on the surface of S are well resolved in our structure, with density at 17 of the 22 predicted N-glycosylation sites (Fig.&nbsp;3b). The other five glycosylation sites are in disordered NTD loops or in the stalk region and are not resolved at high resolution. At the base of the trimer, a clear ring of glycans forms a collar above the stalk region. The density for the stalk region is extended by two helical turns compared with published structures of soluble ectodomain, and fur- ther at lower resolution before fading owing to stalk flexibility (Fig.&nbsp;3c). SARS-CoV-2 S trimers are sparsely distributed and can be highly tilted towards the membrane. This implies that epitopes at the base of the head domain and in the stalk region would be accessible to antibodies where they are not protected by the extensive glycan shell.\nWe compared the in&nbsp;situ structure of the S trimer with structures previously obtained using exogenously expressed purified protein. A recent study of full-length trimeric S solubilized in detergent micelles 8 identified two features that are not seen in most structures of soluble S ectodomain trimers: well-defined density for residues 14–26 of S, and a folded loop between residues 833 and 853. This loop is folded in the structure of the ‘locked’ conformation of the ectodomain 9 , and may become folded in the low-pH conditions in the endosome 30 . We observed only weak density for residues 14–26, and&nbsp;no folded structure for the 833–853 region. The SARS-CoV-2 strain that we have imaged contains the widely circulating D614G substitution 31 , which abolishes a salt bridge to K854 (Fig.&nbsp;3d) and may reduce folding of the 833–854 loop 9,30 . We did not observe additional density that would correspond to bound lipids, as described recently 10 , or other bound co-factors. These may be present sub-stoichiometrically or in rare conformations, but are not a general feature of the S trimer in&nbsp;situ. Overall, our structure is very similar to that of the soluble trimeric ectodomain in the closed prefusion form stabilized by a double proline mutation 2,7 (Extended Data Fig.&nbsp;7). This validates the ongoing use of recombinant, purified S trimers for research, diagnostics and vaccination—they indeed represent the in&nbsp;situ structure of S. In this study, we have determined the structure of S trimers on the virion surface to 3.4&nbsp;Å resolution using cryo-EM (Fig.&nbsp;3), supporting the further use of this technique to study antibody binding to S in the context of the viral surface. Such studies could provide insights into how neutralizing antibodies block virus infection, particularly for antibodies against membrane-proximal regions of S, and thus can inform the design of immunogens for vaccination.\nOnline content\nAny methods, additional references, Nature Research reporting sum- maries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author con- tributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41586-020-2665-2.\nFig. 3 | Structures of SARS-CoV-2 S trimers on intact virions by single-particle reconstruction. a, Top and side views of trimers with three closed RBDs (left, 3.5&nbsp;Å resolution) and one RBD with weaker density (right, 4.1&nbsp;Å resolution). Individual monomers are coloured white, blue and green. Comparison of the left and right structures shows the weaker density for the RBD of the green monomer in the region indicated by the red arrowheads. b, Glycosylation profile of the S protein. Colour scheme as in a; glycans are shown in orange. Boxes indicates the regions shown in c and d. c, Enlarged view of the base of the trimer at lower isosurface threshold to highlight the glycan ring and the extended C-terminal density. d, Enlarged view of the region of the spike where the D614G variation abolishes a salt bridge to K854.\n1. Neuman, B. W. &amp; Buchmeier, M. J. Supramolecular architecture of the coronavirus particle. Adv. Virus Res. 96 , 1–27 (2016). 2. Walls, A. C. et&nbsp;al. Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein. Cell 181 , 281–292 (2020).\n3.  Hoffmann, M. et&nbsp;al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. Cell 181 , 271–280 (2020).\n4.  Shang, J. et&nbsp;al. Structural basis of receptor recognition by SARS-CoV-2. Nature 581 , 221–224 (2020).\n5. Wang, Q. et&nbsp;al. Structural and functional basis of SARS-CoV-2 entry by using human ACE2. Cell 181 , 894–904 (2020).\n6.  Watanabe, Y., Allen, J. D., Wrapp, D., McLellan, J. S. &amp; Crispin, M. Site-specific glycan analysis of the SARS-CoV-2 spike. Science 369 , 330–333 (2020). 7. Wrapp, D. et&nbsp;al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367 , 1260–1263 (2020). 8. Cai, Y. et&nbsp;al. Distinct conformational states of SARS-CoV-2 spike protein. Science 369 , 1586–1592 (2020).\n9. Xiong, X. et&nbsp;al. A thermostable, closed SARS-CoV-2 spike protein trimer. Nat. Struct. Mol. Biol. 27 , 934–941 (2020).\n10.  Toelzer, C. et&nbsp;al. Free fatty acid binding pocket in the locked structure of SARS-CoV-2 spike protein. Science eabd3255 (2020).\n11.  Henderson, R. et&nbsp;al. Controlling the SARS-CoV-2 spike glycoprotein conformation. Nat. Struct. Mol. Biol. 27 , 925–933&nbsp;(2020).\n12.  McCallum, M., Walls, A. C., Bowen, J. E., Corti, D. &amp; Veesler, D. Structure-guided covalent stabilization of coronavirus spike glycoprotein trimers in the closed conformation. Nat. Struct. Mol. Biol. 27 , 942–949&nbsp;(2020).\n13.  Zhu, N. et&nbsp;al. A novel coronavirus from patients with pneumonia in China, 2019. N. Engl. J. Med. 382 , 727–733 (2020).\n14.  Zhou, P. et&nbsp;al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579 , 270–273 (2020).\n15.  Masters, P. S. &amp; Perlman, S. in Fields Virology 6th edn (eds Knipe, D. M. &amp; Howley, P. M.) 826–858 (Elsevier, 2013).\n16.  Neuman, B. W. et&nbsp;al. Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy. J. Virol. 80 , 7918–7928 (2006). 17.  Bárcena, M. et&nbsp;al. Cryo-electron tomography of mouse hepatitis virus: insights into the structure of the coronavirion. Proc. Natl Acad. Sci. USA 106 , 582–587 (2009).\n18.  Walls, A. C. et&nbsp;al. Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion. Proc. Natl Acad. Sci. USA 114 , 11157–11162 (2017).\n19.  Yuan, Y. et&nbsp;al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat. Commun. 8 , 15092 (2017). 20.  Fehr, A. R. &amp; Perlman, S. Coronaviruses: an overview of their replication and pathogenesis. Methods Mol. Biol. 1282 , 1–23 (2015).\n21.  Klein, S. et&nbsp;al. SARS-CoV-2 structure and replication characterized by in&nbsp;situ cryo-electron tomography. Preprint at https://doi.org/10.1101/2020.06.23.167064 (2020).\n22.  Snijder, E. J. et&nbsp;al. A unifying structural and functional model of the coronavirus replication organelle: tracking down RNA synthesis. PLoS Biol. 18 , e3000715 (2020). 23.  Hoffmann, M., Hofmann-Winkler, H. &amp; Pöhlmann, S. in Activation of Viruses by Host Proteases (eds Böttcher-Friebertshäuser, E. et&nbsp;al.) 71–98 (2018).\n24.  Rothe, C. et&nbsp;al. Transmission of 2019-nCoV infection from an asymptomatic contact in Germany. N. Engl. J. Med. 382 , 970–971 (2020).\n25.  Liu, C. et&nbsp;al. Viral architecture of SARS-CoV-2 with post-fusion spike revealed by cryo-EM. Preprint at https://doi.org/10.1101/2020.03.02.972927 (2020).\n26.  Chlanda, P. et&nbsp;al. The hemifusion structure induced by influenza virus haemagglutinin is determined by physical properties of the target membranes. Nat. Microbiol. 1 , 16050 (2016).\n27.  Xiong, X. et&nbsp;al. Receptor binding by a ferret-transmissible H5 avian influenza virus. Nature 497 , 392–396 (2013).\n28.  Gui, M. et&nbsp;al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res. 27 , 119–129 (2017).\n29.  Bonnafous, P. et&nbsp;al. Treatment of influenza virus with beta-propiolactone alters viral membrane fusion. Biochim. Biophys. Acta 1838 , 355–363 (2014).\n30.  Zhou, T. et&nbsp;al. A pH-dependent switch mediates conformational masking of SARS-CoV-2 spike. Preprint at https://doi.org/10.1101/2020.07.04.187989 (2020).\n31.  Korber, B. et&nbsp;al. Tracking changes in SARS-CoV-2 spike: evidence that D614G increases infectivity of the COVID-19 virus. Cell 182 , 812–827 (2020).\nPublisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n© The Author(s), under exclusive licence to Springer Nature Limited 2020\n\n\nMethods\nNo statistical methods were used to predetermine sample size. For Cryo-ET, division of dataset into two random halves was done based on virus number. For Cryo-EM, division of datasets into two random halves was done based on the standard approach in RELION 3. Other experiments did not involve randomization. The investigators were not blinded to allocation during experiments and outcome assessment.\n\n\nCells and virus\nVero E6 cells were obtained from ATCC and were cultured in Dulbecco’s modified Eagle medium (DMEM, Life Technologies) containing 10% fetal bovine serum, 100 U&nbsp;ml −1 penicillin, 100 μg&nbsp;ml −1 streptomycin and 1% non-essential amino acids (complete medium). The Germany/Bav- Pat1/2020 SARS-CoV-2 strain was isolated by C. Drosten (Charité, Berlin, distributed by the European Virology Archive (Ref-SKU: 026V-03883)) at passage (P)2. A stock of SARS-CoV-2 was obtained by passaging the virus once in Vero E6 cells (P3). To produce SARS-CoV-2 virions, Vero E6 cells grown on 75 cm 2 side-bottom tissue culture flasks were infected with SARS-CoV-2 (P3) at a multiplicity of infection (MOI) of 0.5. Culture medium from infected cells was collected at 48 h after infection, clari- fied by centrifugation at 1,000g for 10 min, cleared through a 0.45-μm nitrocellulose filter and fixed with 4% formaldehyde for 30 min at room temperature. Culture medium was supplemented with 10 mM HEPES (pH 7.2) before fixation. Virus-containing medium was subsequently split into aliquots and stored at −80 °C. Infectious supernatants con- taining SARS-CoV-2 virions were obtained from Calu-3 cells infected with P3 virus at an MOI of 5 for 48 h and processed as described above. To obtain SARS-CoV-2 virions at high concentration, infection and collection of Vero E6 culture medium were performed as above, fol- lowed by concentration of fixation-inactivated virions from medium by ultracentrifugation through a 20% (wt/wt) sucrose cushion (120 min at 27,000 rpm in a Beckman SW32 rotor; Beckman Coulter Life Sciences). Pelleted particles were resuspended in PBS and stored in aliquots at −80 °C.\n\n\nWestern blot\nVero E6 or Calu-3 cells mock infected or infected for 48 h with SARS-CoV-2 (MOI&nbsp;=&nbsp;5) were washed twice in PBS, scraped, pelleted at 700g for 5 min and lysed in PBS pH 7.4, containing 1% Triton&nbsp;X-100 and protease inhibitors (Merck) for 30 min at 4 °C. Samples were centri- fuged at 4 °C for 30 min and supernatants were collected. Total pro- tein concentration was calculated using the Bio-Rad Protein Assay kit (Bio-Rad). Purified viruses were prepared for western blot by cen- trifugation of 32 ml of virus containing supernatants on a 10% sucrose cushion in a Beckmann J25 centrifuge. Centrifugation was performed at 10,000g for 4 h at 4 °C. Supernatants were discarded and purified virus pellet were resuspended in 500 μl PBS. For western blotting, 10 μg of total cell lysates and 5 μl purified viruses were diluted in Laemmli buffer and loaded on a pre-cast Criterion XT 4–12% gradient gel (Bio-Rad). Gels were transfered to PVDF membrane using a wet-electroblotting chamber system (Bio-Rad) in Towbin buffer containing 10% methanol. Transfer was performed overnight at 4 °C. Membranes were washed in PBS and blocked with 10% milk in PBS containing 0.2% tween-20 (PBS-T) for 1 h. Membranes were incubated for 1 h at room temperature with primary antibodies specific for an epitope in the C-terminal region of S (Abcam, ab252690; diluted 1:1,000 in PBS-T), for N protein (Sino Biological, 40143-MM05; diluted 1:1,000 in PBS-T) or nsp3 (Abcam, ab181620; diluted 1:500 in PBS-T). Next, the membranes were washed 3 times in PBS-T, incubated with horseradish peroxidase-conjugated anti-mouse or anti-rabbit antibodies at 1:10,000 dilution for 1 h, washed again 3 times in PBS-T, incubated with western Lightning Plus-ECL rea- gent (Perkin Elmer) and imaged using an Intas ChemoCam Imager 3.2 (Intas). Densitometric analysis of western blot assays was performed using LabImage 1D L340 software v.4.1 (Intas). The blot shown is\nrepresentative of three independent experiments. Uncropped, unpro- cessed blots are presented in the Source Data.\n\n\nPCR with reverse transcription and spike sequencing\nTotal RNA was isolated from infected Vero E6 cells 48 h after infec- tion with Germany/BavPat1/2020 SARS-CoV-2 (P2). Spike cDNA was produced from the total RNA using Superscript III (ThermoFisher) with specific reverse transcription primer (CAATTGTGAAGATTCTC ATA). The cDNA was amplified by PCR using specific primers (Fwrd, ATGTTTGTTTTTCTTGTTTTATT; rev, TTATGTGTAATGTAATTTGA) and the resulting amplicon was sent for Sanger sequencing. Sequences were compared to the Germany/BatPat1/2020 SARS-CoV-2 reference sequence (Ref-SKU: 026V-03883) and found to be identical. Specific sequencing primers: Fwrd1, ATGTTTGTTTTTCTTGTTTTATT; Fwrd2, GGTTGGACAGCTGGTGCT; Fwrd3, CCAACCATACAGAGTAGTAGTA; Rev1, GTAGCAGCAAGATTAGCAGAA; and Rev2, TTATGTGTAATGT AATTTGA.\n\n\nCryo-ET sample preparation\nFixed virus samples from the supernatant of infected cells without any concentration step (unconc) or concentrated by pelleting through a sucrose cushion (conc) were prepared, imaged, and processed in parallel. The virus suspension was mixed with 10-nm colloidal gold (in PBS solution) in 10:1 ratio. Then, 3 μl of the solution was added to a glow-discharged copper grid (C-Flat 2/2, Protochips). Grids were plunge frozen into liquid ethane by back-side blotting using a LeicaGP cryo plunger (Leica) and stored in liquid nitrogen until imaging.\n\n\nCryo-ET data collection\nCryo-ET data collection was performed essentially as described previ- ously 32 . Cryo-grids were loaded into an FEI Titan Krios transmission electron microscope operated at 300 kV and images were recorded on a Gatan K2 Summit direct detection camera in counting mode with a 20 eV energy slit in zero-loss mode. Tomographic tilt series between −60° and +60° were collected using SerialEM 3.8.0 software 33 in a dose-symmetric scheme 34 with a 3° angular increment. A total dose of 120 e − A −2 per tilt series was distributed evenly among 41 tilt images. The nominal mag- nification was 81,000×, giving a pixel size of 1.532 Å on the specimen. The defocus range was between −2 μm and −6 μm and 10 frames were saved for each tilt angle. All data acquisition parameters are listed in Extended Data Table&nbsp;1.\nFrames were motion-corrected in IMOD 4.10.30 35 and images were dose-filtered using the alignframes function in IMOD. Exposure fil- tering was implemented according to the cumulative dose per tilt as described elsewhere 36 . The contrast transfer function (CTF) was meas- ured using non-dose-filtered images using the ctfplotter package within IMOD 37 . Tilt series stacks were sorted using IMOD newstack function and fiducial-alignment of all tilt series was performed in IMOD/etomo. Tomograms with less than three trackable gold fiducials were discarded. Motion-corrected and dose-filtered tilt stacks were CTF-corrected by CTF multiplication and tomograms were reconstructed by weighted back-projection in novaCTF 38 . Tomograms were low-pass filtered to 50 Å for better visualization in EMAN2.2 39 and tomographic slices were visualized with IMOD.\n\n\nExtraction of S trimers from tomograms\nThe initial steps of subtomogram alignment and averaging were imple- mented using MATLAB (MathWorks) scripts, which were derived from the TOM 40 and AV3 41 packages as described previously 32 . The missing wedge was modelled as the summed amplitude spectrum of back- ground subtomograms for each tomogram, and was applied during alignment and averaging.\nTo generate an initial template model of the spike protein from the viral surface, 68 spikes were manually picked from four virions of tomograms that were down-scaled by 4× binning of the voxels.\nThe 68 spikes’ initial Euler angles (2 out of 3) were determined on the basis of the vector between two points, one on the head of the spike and one on the membrane where the spike anchors, respectively. The 68 spikes were iteratively aligned to one another for four iterations, applying three-fold symmetry to generate a low-resolution template that resembled a prefusion conformation of the spike. This template was used as an alignment reference for all virions (below). All postfu- sion spikes were manually identified and picked, and initial Euler angles were assigned in the same manner.\nThe centre of each virion was then marked manually using the Vol- ume Tracer function in UCSF Chimera (v.1.13.1) 42 and the radius of the virion was determined centred at the membrane using the Pick Particle Chimera Plugin 43 . An oversampled spherical grid of points was gener- ated on the virion surface with ~9 nm spacing, and subtomograms were extracted for all grid points with a box size of 96 pixels (approximately 60 nm) centred at a radius 14 nm above these grid positions (approxi- mately the radius of the expected centre of the spike). Initial Euler angles were assigned to each subtomogram based on the orientation of the normal vectors relative to the sphere surface.\nSubtomograms were aligned against the low-resolution template (from the above average of 68 spikes). During this alignment, subto- mogram positions converged onto clusters at the true spike positions. One subtomogram position was kept for each cluster, excluding parti- cles within a distance of ~60 Å (10 pixels at 4× binning) and removing particles with cross-correlation coefficients below 0.11. Subtomo- grams that had tilted by more than 90° relative to their perpendicular positions were excluded. Visual inspection of the tomograms using the Place Object Chimera Plugin 43 confirmed that subtomograms selected in this manner corresponded to S trimers on the viral surface (see below). Subtomograms were divided into two halves on the basis of virion number. From this point on the two halves were processed independently.\n\n\nSubtomogram averaging\nSubsequent processing was performed in RELION 44 . For this purpose, subtomograms were reconstructed from the original tilt series images after motion correction using relion_reconstruct. Using dedicated Python scripts, the S trimer positions in the 3D tomograms from the procedure outlined above were converted into 2D positions and defocus values in the corresponding tilt series images, as well as Euler angles in the RELION convention. Individual sub-tomograms were reconstructed at a 2× downscaled pixel size of 3.064 Å, by 3D insertion of Fourier slices of the cropped regions of the tilt series images, each multiplied by their corresponding CTF, which included the dose filter. Similarly, 3D-CTF volumes were generated by 3D insertion of the cor- responding CTF 2 slices. Subtomograms were reconstructed in a box size of 128 voxels for prefusion trimers and a box size of 192 voxels for averages of tilted prefusion trimers and for postfusion trimers.\nTo deal with the CTF pre-multiplied sub-tomograms, as well as the multiplicity of each 3D voxel in the tilt series, a modified version of RELION was used for subtomogram refinement and classification (details to be described elsewhere). Standard 3D auto-refinement was performed with C3 symmetry and a soft-edged mask around the trimers, using a 30 Å lowpass-filtered map as an initial reference. Using 3,854 of the 4,104 subtomograms (excluding those that had tilted by more than 90° relative to the perpendicular position (Extended Data Fig.&nbsp;1c)), a 7.7 Å consensus map was calculated for the prefusion trim- ers; 116 postfusion trimers led to a 22 Å map.\nAngular density (Extended Data Fig.&nbsp;1c) was illustrated by dividing the number of spikes per histogram bin by the sine of the tilt angle. From this histogram, the noise level in the distribution due to incorrectly aligned spikes was estimated assuming they are uniformly distrib- uted (red dashed line in Extended Data Fig.&nbsp;1c), and based on averaging over the range 140–180°. Such angles must be incorrectly alignments because they orient the spike towards the inside of the virus.\nBased on the subtomograms that contributed to the consensus map we estimated that there are 23&nbsp;±&nbsp;9 prefusion S trimers per virion. For a subset of 28 virions, we visually compared the trimers identified by subtomogram averaging with the tomograms to identify false positives or false negatives, finding that the number of S trimers identified by subtomogram averaging is an underestimate of the number of observed S trimers by 0.9 trimers per virion. On this basis we estimate there are 24&nbsp;±&nbsp;9 prefusion S trimers per virion.\nNext, we performed symmetry expansion 45 , followed by focused classification without alignment and with partial signal subtraction, while keeping the orientations from the consensus refinement fixed. The mask used for focused classification was generated manually and enclosed the RBD of one monomer (Extended Data Fig.&nbsp;3a), including the closest NTD of the neighbouring monomer. Classification of the primary dataset (unconc1) led to three different RBD states: closed (45% of the monomers); open (28%) and with weak density (27%) (Extended Data Fig.&nbsp;3a). Classification of a second, smaller dataset of virus parti- cles in supernatant (unconc2), comprising 1,224 trimers yielded 39% closed RBD, 22% open RBD and 39% weak RBD, in general agreement with the unconc1 results (Extended Data Fig.&nbsp;3b). Classification of a dataset of virus particles after concentration through a sucrose cush- ion (conc), comprising 3,788 trimers, yielded 67% of the monomers corresponding to closed RBDs and 33% to RBDs with weak density. For this dataset, no RBDs in the open conformation were identified (Extended Data Fig.&nbsp;3c).\nThe classification of the RBDs as closed, open or with weaker density was used to divide the unconc1 dataset into three subsets: 31% (1,175 trimers) have no open RBDs; 55% (2,121 trimers) have one RBD in the open state; and 14% (525 trimers) have two open RBDs. The remaining 33 trimers have three RBDs in the open state and not further processed. In the class with weaker density, the RBD appears to be predominantly in the closed state and was treated as closed for this assignment. For each of the three subsets, reconstruction of the two independently refined half sets was performed using the orientations from the consensus refinement that gave the 7.7 Å consensus map described above. Subse- quent standard post-processing procedures for resolution estimation, map sharpening and local-resolution filtering in RELION led to three final maps. The subset with no open RBDs gave a reconstruction with C3 symmetry in which all three RBDs were closed at 8.6 Å resolution. The other two subsets yielded structures with C1 symmetry and either one or two open RBDs, with resolutions of 8.6 Å and 9.9 Å, respectively (Extended Data Figs.&nbsp;3, 4). Note, that while fixation is not expected to give rise to any new conformations, we cannot rule out that it dif- ferentially stabilizes different conformations, and the position of the equilibrium between open and closed conformations in unfixed sam- ples may differ from those estimated from fixed samples.\nAverages of tilted prefusion trimers were generated by grouping according to the tilt and rotation of the subtomogram away from the normal vector to the membrane. Subtomograms were included in a group for averaging if they were within 15° of the displayed tilt (0°, 30° and 60°) and rotation (0°, 60°). The pools illustrated in Fig.&nbsp;2c contain 14.5, 14.1, 12.0 and 10.0% of the subtomograms.\n\n\nCryo-EM sample preparation and data collection\nVirus solution concentrated through a 20% (wt/wt) sucrose cushion was plunge-frozen on C-Flat 2/2 3C grids (Protochips) following the same procedure as for cryo-ET, but without adding gold fiducials. Grids were imaged on a Thermo Fisher Scientific Titan Krios transmission electron microscope that was operated at 300 kV, using a Gatan K3 direct electron detector and a Gatan BioQuantum energy filter with a 20 eV energy slit. Movies with 48 frames and an accumulated dose of 50 e − Å −2 were acquired in counting mode using SerialEM-3.8.0 33 at a nominal magnification of 81,000×, corresponding to a calibrated pixel size of 1.061 Å&nbsp;per&nbsp;pixel. Detailed data acquisition parameters are summarized in Extended Data Table&nbsp;2.\n\n\nCryo-EM image processing\nThe scheduler functionality in RELION-3.1 was used for fully automated real-time processing during data collection 9,46 . Mov- ies were motion-corrected and dose-weighted using the RELION implementation of the MotionCor2 algorithm 47 . Subsequently, non-dose-weighted sums were used to estimate the CTF in CTFFIND-4.1.13 48 . S trimers that were extending from the sides of virus particles were picked manually (4,493 particles from the first 100 micrographs) and then used as a training set for optimisation of the convolutional neural network in the automated particle picking software Topaz 49 . Extracted particles were subjected to 3D classification using a previously determined structure of the S trimer 9 , low-pass-filtered to 30 Å, as initial 3D reference. The selected 286,407 particles that contributed to 3D classes corresponding to S trimers were submitted to Bayesian polishing to correct for per-particle beam-induced motions and a second round of 3D clas- sification to select the 55,159 particles that contributed to the best class. This final consensus set of particles was subjected to CTF refinement of per-particle defocus, per-micrograph astigmatism and beam tilt, followed by a second round of Bayesian polishing. 3D auto-refinements were performed with the selected particles after each round of 3D classification, CTF refinement or Bayesian polishing. The consensus structure had a resolution of 3.4 Å.\nSubsequently, symmetry expansion 45 was performed, followed by focused 3D classification with partial signal subtraction 50 . Using a mask on a single RBD to focus classification into six classes while keeping the orientations of the last consensus refinement fixed resulted in the identification of two RBD states: closed or with weak density. S trimers with all three RBDs in the closed state were refined separately from S trimers with one RBD with weak density, resulting in two final maps with resolutions. Standard RELION post-processing was used for resolution estimation, map sharpening and local-resolution filtering. The C3 symmetric map with three closed RBDs had an estimated overall resolution of 3.5 Å; the C1 map with one weaker RBD extended to 4.1 Å resolution (Extended Data Fig.&nbsp;6b).\n\n\nModel building and refinement\nThe SARS-CoV-2 S trimer structure 9 (PDB 6ZP0) was used as an initial model for building into the model with three closed RBDs (Extended Data Fig.&nbsp;6a, middle). Residues were adjusted manually in Coot 0.9 51 . Steric clash and sidechain rotamer conformations were improved using the Namdinator web server 52 . After further iterations of manual adjustment, the structure was refined in PHENIX-1.18.2 53 . The geom- etry and statistics are given in Extended Data Table&nbsp;2. The unmasked model-to-map FSC was calculated in PHENIX for the refined model against the full reconstruction.\n\n\n3D model of spikes on authentic virions\nTo visualize the spike protein on the authentic SARS-CoV-2 virions, the coordinates, orientations and conformational classes determined by subtomogram averaging were converted into a format compat- ible with Maxon Cinema 4D (v.S22.116), and imported together with the 3D models of the different conformational states determined by subtomogram averaging. To generate representative virion images in Fig.&nbsp;2d we removed the three false positives from one virion and positioned the three false negatives on each virion that had been identified by visual inspection of the data (see above). The HR2 region was modelled as a cylinder. Images of individual virions from the dataset were rendered into Adobe Photoshop to generate images for presentation.\n\n\nReporting summary\nFurther information on research design is available in the&nbsp;Nature Research Reporting Summary linked to this paper.\n\n\nData availability\nPublished structures for comparison or initial model building were obtained from the PDB with accession codes 6VXX 2 , 6VYB 2 , 6X2B 11 , 6XRA 8 and 6ZP0 9 . The cryo-EM and cryo-ET structures determined here and representative tomograms are deposited in the Electron Micros- copy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (three closed RBDs from subtomo- gram averaging), EMD-11495 (one open RBD), EMD-11496 (two open RBDs), EMD-11497 (three closed RBDs from cryo-EM), and EMD-11498 (two open RBDs and one weak RBD). The associated molecular models are deposited in the PDB under accession code 6ZWV (three closed RBDs from cryo-EM). The raw micrographs are deposited in the Electron Microscopy Public Image Archive under accession codes EMPIAR-10492 (cryo-EM) and EMPIAR-10493 (cryo-ET).\n32.  Wan, W. et&nbsp;al. Structure and assembly of the Ebola virus nucleocapsid. Nature 551 , 394–397 (2017).\n33.  Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. J. Struct. Biol. 152 , 36–51 (2005).\n34.  Hagen, W. J. H., Wan, W. &amp; Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197 , 191–198 (2017).\n35.  Kremer, J. R., Mastronarde, D. N. &amp; McIntosh, J. R. Computer visualization of three-dimensional image data using IMOD. J. Struct. Biol. 116 , 71–76 (1996).\n36.  Grant, T. &amp; Grigorieff, N. Measuring the optimal exposure for single particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6. eLife 4 , e06980 (2015).\n37.  Xiong, Q., Morphew, M. K., Schwartz, C. L., Hoenger, A. H. &amp; Mastronarde, D. N. CTF determination and correction for low dose tomographic tilt series. J. Struct. Biol. 168 , 378–387 (2009).\n38.  Turoňová, B., Schur, F. K. M., Wan, W. &amp; Briggs, J. A. G. Efficient 3D-CTF correction for cryo-electron tomography using NovaCTF improves subtomogram averaging resolution to 3.4 Å. J. Struct. Biol. 199 , 187–195 (2017).\n39.  Galaz-Montoya, J. G., Flanagan, J., Schmid, M. F. &amp; Ludtke, S. J. Single particle tomography in EMAN2. J. Struct. Biol. 190 , 279–290 (2015).\n40.  Nickell, S. et&nbsp;al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149 , 227–234 (2005).\n41.  Förster, F., Medalia, O., Zauberman, N., Baumeister, W. &amp; Fass, D. Retrovirus envelope protein complex structure in&nbsp;situ studied by cryo-electron tomography. Proc. Natl Acad. Sci. USA 102 , 4729–4734 (2005).\n42.  Pettersen, E. F. et&nbsp;al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25 , 1605–1612 (2004).\n43.  Qu, K. et&nbsp;al. Structure and architecture of immature and mature murine leukemia virus capsids. Proc. Natl Acad. Sci. USA 115 , E11751–E11760 (2018).\n44.  Bharat, T. A. M., Russo, C. J., Löwe, J., Passmore, L. A. &amp; Scheres, S. H. W. Advances in single-particle electron cryomicroscopy structure determination applied to sub-tomogram averaging. Structure 23 , 1743–1753 (2015).\n45.  Scheres, S. H. W. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579 , 125–157 (2016).\n46.  Zivanov, J. et&nbsp;al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7 , e42166 (2018).\n47.  Zheng, S. Q. et&nbsp;al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14 , 331–332 (2017).\n48.  Rohou, A. &amp; Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192 , 216–221 (2015).\n49.  Bepler, T. et&nbsp;al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Nat. Methods 16 , 1153–1160 (2019).\n50.  Bai, X. C., Rajendra, E., Yang, G., Shi, Y. &amp; Scheres, S. H. W. Sampling the conformational space of the catalytic subunit of human γ-secretase. eLife 4 , e11182 (2015). 51.  Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. Acta Crystallogr. D 60 , 2126–2132 (2004).\n52.  Kidmose, R. T. et&nbsp;al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6 , 526–531 (2019).\n53.  Afonine, P. V. et&nbsp;al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr. D 74 , 531–544 (2018).\nAcknowledgements We thank the staff of the MRC-LMB for generous support during the COVID-19 pandemic lockdown; all of&nbsp;the staff of the MRC-LMB EM Facility, in particular A. Yeates, G. Sharov and G. Cannone, for supporting the EM experiments; J. Grimmett and T. Darling for supporting scientific computing; C. Drosten and EVAg for provision of the SARS-CoV-2 strain used here; and M. Frey for the Calu-3 cell line. We acknowledge microscopy support from the Infectious Diseases Imaging Platform (IDIP) at the Center for Integrative Infectious Disease Research Heidelberg. This study was supported by funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (ERC-CoG-648432 MEMBRANEFUSION to J.A.G.B.), the Medical Research Council as part of UK Research and Innovation (MC_UP_A025_1013 to S.H.W.S.; MC_UP_1201/16 to J.A.G.B.), the Deutsche Forschungsgemeinschaft (240245660–SFB 1129 project 5 to H.-G.K. and project 11 to R.B.) and the Japan Society for the Promotion of Science (overseas research fellowship to T.N.).\nAuthor contributions Z.K. and J.A.G.B. conceived the study. Z.K., J.O., K.Q., M.C., V.Z., H.-G.K., R.B. and J.A.G.B. designed the study. M.C., V.Z., C.J.N. and B.C. prepared and characterized virus samples with supervision from H.-G.K. and R.B. Z.K. and K.Q. collected cryo-EM data. Z.K., J.O. and K.Q. processed cryo-EM data with assistance from T.N., J.Z., J.M.L. and J.P., supervised by S.H.W.S. and J.A.G.B. Z.K., J.O., K.Q., M.C. and L.M. made figures. Z.K., J.O., K.Q., M.C., X.X. and J.A.G.B. interpreted data. J.A.G.B. prepared the initial draft and managed the project. Z.K., J.O., K.Q., M.C. and J.A.G.B. wrote the manuscript with input from all authors. S.H.W.S., H.-G.K., R.B. and J.A.G.B. obtained funding.\nCompeting interests The authors declare no competing interests.\nAdditional information Supplementary information is available for this paper at https://doi.org/10.1038/s41586-020- 2665-2.\nCorrespondence and requests for materials should be addressed to J.A.G.B. Peer review information Nature thanks Jason McLellan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Reprints and permissions information is available at http://www.nature.com/reprints.\nExtended Data Fig. 1 | Characterization of SARS-CoV-2 virion morphology. a, Histogram of virion diameters for unconcentrated extracellular virions in the supernatant of two independent preparations (top and middle), and for extracellular virions after concentration through a sucrose cushion (bottom). After concentration the virions become less spherical. Mean and standard deviation for diameters are 91&nbsp;±&nbsp;11 nm (n&nbsp;=&nbsp;179), 94&nbsp;±&nbsp;9 nm (n&nbsp;=&nbsp;68) and 92&nbsp;±&nbsp;8 nm (n&nbsp;=&nbsp;227) for the three preparations. b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. Visual inspection indicates that almost all spikes were identified for virions in the supernatant, but that not all spikes are identified in the concentrated preparation leading to an underestimate of the number of spikes. c, Histogram of spike tilt angle towards the membrane for the\nlarger supernatant virus dataset (unconc1). The vertical black dashed line indicates 90°. 97% of particles have tilts below 90°; particles with tilts above 90° were not included in image analysis. The angular density (right) is calculated by dividing the number of spikes by the sine of the determined angle. If spikes were unconstrained in tilt, this distribution would be uniform. The angular density decreases from ~50°, indicating that higher tilts are disfavoured. The horizontal red dashed line indicates the angular distribution of noise (spikes which have failed to align), estimated based on the angular density between 140° and 180°. d, Schematic diagram and examples of individual tilted spikes on virions. The schematic indicates the angle that was measured. Five examples of individual tilted spikes are marked on tomographic slices through an intact virion, with their associated angle. Scale bar 50 nm.\nExtended Data Fig. 2 | Morphology of SARS-CoV-2 virions released from infected Calu-3 cells. As in Extended Data Fig.&nbsp;1. a, Histogram of virion diameters. Mean and standard deviation for diameters are 104&nbsp;±&nbsp;13 nm (n&nbsp;=&nbsp;67). b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. c, Central slices through three representative viruses from 67 imaged in one experiment. Virions from Calu-3 cells had a slightly broader diameter distribution than those from Vero E6 cells. Scale bar 50 nm. d, Western blot analysis of SARS-CoV-2 nsp3, S and N in cell lysates and in virus preparations. In released virions, S is present in both cleaved (S2, 73%) and uncleaved forms (S0, 27%).\nExtended Data Fig. 3 | Classification of SARS-CoV-2 spike RBDs. a, Class averages obtained after focused classification on the RBD of the left monomer after symmetry expansion of the unconc1 dataset. Top views and side views are shown for closed, open and weak classes. The region subjected to classification is indicated by a transparent red mask surface in the left hand\npanel. b, Equivalent analysis for a smaller, independent dataset (unconc2). c, Equivalent analysis for a dataset obtained after concentrating virus through a sucrose cushion (conc). Only closed and weak classes were obtained. d–f, Cut-open local resolution maps for structures shown in a–c.\nExtended Data Fig. 4 | Resolution assessment of subtomogram averaging structures. a, Local resolution map for the consensus structure obtained for the prefusion S trimers. b, Local resolution maps for the prefusion S trimer in three different conformations. c, Global resolution assessment by Fourier shell correlation (FSC) at the 0.143 criterion for the four structures shown in a and b, as well as the postfusion S trimer.\nExtended Data Fig. 5 | Single particle cryo-EM image processing workflow. Automatically picked particles (green circles) were subjected to 3D classification. Scale bar 100 nm. Selected 3D classes are indicated by black boxes. RBDs from individual asymmetric units from the S trimer (red dashed circles) were locally classified to sort different conformations of RBD. The\nasymmetric unit subjected to local classification is shown in a top view, the RDB of the green monomer is weak in the right-hand class (red arrowhead). S trimers with all three RBDs in the closed state were further refined with C3 symmetry. S trimers where one RBD had weak density were refined with C1 symmetry. For further details see materials and methods.\nExtended Data Fig. 6 | Single particle Cryo-EM structure validation. a, Cut-open cryo-EM maps obtained using all prefusion S trimers, S trimers with 3 closed RBDs or S trimers with 2 closed and 1 weak RBDs, coloured according to the local resolution. b, FSC curves for the three structures in a, and for the atomic model against the map.\nExtended Data Fig. 7 | Structural comparison of in&nbsp;situ structure with recombinant soluble structure. Structural superposition of S trimer modelled into the structure of the trimer with three closed RBDs\n(green, this study) with the published structure of recombinant, soluble closed trimer (blue, PDB 6VXX). Top and side views are shown. The structures are very similar.\nExtended Data Table 1 | Cryo-ET data acquisition and image processing\n\n\nExtended Data Table 2 | Cryo-EM data collection, refinement and validation statistics\n\n\nReporting Summary\nNature Research wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see our Editorial Policies and the Editorial Policy Checklist .\n\n\nStatistics\nFor all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section.\nn/a Confirmed\nThe exact sample size ( n ) for each experimental group/condition, given as a discrete number and unit of measurement\nA statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly\nThe statistical test(s) used AND whether they are one- or two-sided\nOnly common tests should be described solely by name; describe more complex techniques in the Methods section.\n\n\nA description of all covariates tested\nA description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons\nA full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals)\nFor null hypothesis testing, the test statistic (e.g. F , t , r ) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable.\nFor Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings\nFor hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes\nEstimates of effect sizes (e.g. Cohen's d , Pearson's r ), indicating how they were calculated\nOur web collection on statistics for biologists contains articles on many of the points above.\n\n\nSoftware and code\nPolicy information about availability of computer code\nData collection SerialEM 3.8.0 for operation of electron microscope (available and referenced in the Methods section)\nData analysis CryoET data processing (all commercial or available and referenced in methods): CTFFIND4; IMOD 4.10; NovaCTF 1.0.0, EMAN2.2; Matlab R2016b; TOM (no version number); AV3 (no version number); Python 3.7; RELION 3.1.\nCryoEM data processing (all available and referenced in methods): CTFFIND4.1.13; RELION 3.1.\nModelling/structure refinement/visualization (all available and referenced in methods): Coot 0.9; Namdinator (no version numbering); PHENIX 1.18.2; Chimera 1.13.1; Maxon Cinema 4D version S22.116\nWestern blot quantification: LabImage 1D L340 software version 4.1 (Intas, Göttingen)\nFor manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors and reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Research guidelines for submitting code &amp; software for further information.\n\n\nPolicy information about availability of data\nAll manuscripts must include a data availability statement . This statement should provide the following information, where applicable: - Accession codes, unique identifiers, or web links for publicly available datasets - A list of figures that have associated raw data\n- A description of any restrictions on data availability\nPublished structures for comparison or initial model building were obtained from the protein data bank (PDB) with accession codes 6VXX, 6VYB, 6X2B, 6XRA and 6ZP0. The cryo-EM and cryo-ET structures determined here, and representative tomograms are deposited in the Electron Microscopy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (3 closed RBDs from subtomogram averaging), EMD-11495 (1 open RBD), EMD-11496 (2 open RBDs), EMD-11497 (3 closed RBDs from cryo-EM), and EMD-11498 (2 open and 1 weak RBDs). The associated molecular models are deposited in the PDB under accession codes 6ZWV (3 closed RBDs from cryo-EM).\n\n\nField-specific reporting\nPlease select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection.\nLife sciences Behavioural &amp; social sciences Ecological, evolutionary &amp; environmental sciences\nFor a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf\n\n\nLife sciences study design\n\n\nAll studies must disclose on these points even when the disclosure is negative.\nSample size Sample sizes were determined by available electron microscopy time and the number of virus particles on electron microscopy grids. The sample size is sufficient to obtain a structure at the reported resolution, as assessed by Fourier shell correlation.\nData exclusions For all cryo-ET experiments, tomograms which could not be used due to objects (contamination, ice, grid bars etc) obscuring the virus particle were discarded.\nReplication For cryoET and cryoEM, structures were determined from independent half datasets, which were compared to assess the resolution of the reconstruction, as described in the methods section. Two independent preparations of virus in supernatant were analysed as described in the methods section.\nWestern blots are representative of three independent experiments.\nRandomization For CryoET, division of dataset into two random halves was done based on virus number. For CryoEM, division of datasets into two random halves was done based on standard approach in RELION 3. Other experiments did not involve randomization.\n\n\nBlinding Blinding was not applicable to this study because this type of study does not use group allocation.\n\n\nReporting for specific materials, systems and methods\nWe require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.\nMaterials &amp; experimental systems n/a Involved in the study\nAntibodies\nEukaryotic cell lines\nPalaeontology and archaeology\nAnimals and other organisms\nHuman research participants\nClinical data\nDual use research of concern\nMRI-based neuroimaging\n\n\nAntibodies\nAntibodies used Anti-mouse HRP Sigma A4416 Anti-rabbit HRP Sigma A6154\nBeta actin Sigma A5441\nSpike Abcam Ab252690\n\n\nValidation\nCommercial antibodies validated as per manufacturers website: Beta actin Sigma A5441 Immunoblot on chicken fibroblast cell extracts Spike Abcam Ab252690 Validated by ELISA on free peptide from SARS-CoV-1 Nsp3 abcam ab181620 Validated by western blot on SARS-CoV-1 infected cells Nucleocapsid Sino Biological 40143-MM05 Validated by western blot with corresponding viruses\n\n\n245 190 135 100 80 58 46 32 25 22 17 11 58 46 200 Nucleoprotein Actin Nsp3 Spike Cell lysat e Virions b a c S2-S2 ′ S0 S2 SARS-CoV-2 – + + + kDa\n\n\n3 closed RBDs 1 open RBD 2 open RBDs a c b 90° 90° 90° 3 closed RBDs 1 open RBD 2 open RBDs Postfusion Postfusion Prefusion 0º 30º 60º d 60º, rotation 0º 60º, rotation 60º 90°\n\n\n90° a b 90° 3 closed RBDs 2 closed + 1 weak RBDs b G614 c d c d K854\n\n\nCorresponding author(s): John Briggs Last updated by author(s): Jul 28, 2020\n\n\nMethods n/a Involved in the study ChIP-seq Flow cytometry\n\n\n3 Closed RBDs (EMD-11497, PDB 6ZWV) 2 Closed + 1 Weak RBDs (EMD-11498) Data collection and processing Magnification 81,000 81,000 Voltage (kV) 300 300 Electron exposure (e – /Å 2 ) 50 50 Defocus range (µm) 1.0-3.0 1.0-3.0 Pixel size (Å) 1.061 1.061 Movies (no.) 7,982 7,982 Symmetry imposed C3 C1 Initial particle images (no.) 450,473 450,473 Final particle images (no.) 29,183 25,976 Map resolution (Å) 3.5 4.1 FSC threshold 0.143 0.143 Map resolution range (Å) 3.35-27.16 3.67-45.27 Refinement Initial model used PDB 6ZP0 Model resolution (Å) 3.5 FSC threshold 0.5 Map sharpening B factor (Å 2 ) -50 Model composition Non-hydrogen atoms 24036 Protein residues 2955 Ligands 63 B factors (Å 2 ) Protein 130.57 Ligand 142.74 R.m.s. deviations Bond lengths (Å) 0.009 Bond angles (°) 1.346 Validation MolProbity score 1.62 Clashscore 2.97 Poor rotamers (%) 1.74 Ramachandran plot Favored (%) 94.97 Allowed (%) 5.03 Disallowed (%) 0.00\n\n\nAuthentication Cells were not further authenticated\n",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Structures and distributions of SARS-CoV-2 spike proteins on intact virions\nZunlong Ke 1,7 , Joaquin Oton 1,7 , Kun Qu 1,7 , Mirko Cortese 2 , Vojtech Zila 3 , Lesley McKeane 4 , Takanori Nakane 1 , Jasenko Zivanov 1 , Christopher J. Neufeldt 2 , Berati Cerikan 2 , John M. Lu 1 , Julia Peukes 1 , Xiaoli Xiong 1 , Hans-Georg Kräusslich 3,5 , Sjors H. W. Scheres 1 , Ralf Bartenschlager 2,5,6 &amp; John A. G. Briggs 1 ✉\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virions are surrounded by a lipid bilayer from which spike (S) protein trimers protrude 1 . Heavily glycosylated S trimers bind to&nbsp;the angiotensin-converting enzyme 2 receptor and mediate entry of virions into target cells 2–6 . S exhibits extensive conformational flexibility: it modulates exposure of its receptor-binding site and subsequently undergoes complete structural rearrangement to drive fusion of viral and cellular membranes 2,7,8 . The structures and conformations of soluble, overexpressed, purified S proteins have been studied in detail using cryo-electron microscopy 2,7,9–12 , but the structure and distribution of S on the virion surface remain unknown. Here we applied cryo-electron microscopy and tomography to image intact SARS-CoV-2 virions and determine the high-resolution structure, conformational flexibility and distribution of S trimers in&nbsp;situ on the virion surface. These results reveal the conformations of S on the virion, and provide a basis from which to understand interactions between S and neutralizing antibodies during infection or vaccination.\nThe Betacoronavirus SARS-CoV-2 13,14 is an enveloped virus containing a large nucleoprotein (N)-encapsidated positive-sense RNA genome 15 . Three transmembrane proteins are incorporated into the viral lipid envelope: S and two smaller proteins, membrane protein (M) and envelope protein (E) 1,15 . When imaged by cryo-electron microscopy (cryo-EM), betacoronaviruses appear as approximately spherical par- ticles with variable diameters centred around 100&nbsp;nm, containing a dense viroplasm and bounded by a lipid bilayer from which prominent S trimers protrude 16,17 . S trimers of SARS-CoV-2 bind to its receptor on the surface of target cells—angiotensin-converting enzyme 2 (ACE2)—and mediate subsequent viral uptake and fusion 2–5,7 . In so doing, S under- goes a substantial structural rearrangement from the prefusion form to the postfusion form 8 . The overall architectures of both prefusion and postfusion forms are highly conserved among coronaviruses 8,18,19 . During infection, coronaviruses extensively remodel the internal membrane organization of the cell, generating viral replication orga- nelles in which replication takes place 20–22 . The S, M and E proteins are inserted into membranes of the endoplasmic reticulum and traf- fic to the endoplasmic reticulum–Golgi intermediate compartment (ERGIC). The encapsidated genome buds into the ERGIC to form virions, which are then trafficked to the plasma membrane and released 20–22 . S is primed for membrane fusion by proteolytic cleavage at the S1–S2 site, and subsequently the S2′ site 23 .\nThe prefusion structures of S proteins from coronaviruses, including SARS-CoV-2 S, have been extensively studied using ectopic expres- sion of soluble, secreted forms of S, followed by purification and cryo-EM 2,7,9–11 . In the prefusion form, the receptor-binding domain (RBD)\nsits at the top of a broad, trimeric spike above the fusion core. Three copies of the RBD are surrounded by three copies of the N-terminal domain (NTD) which display some mobility 2,7–9 . In the closed prefu- sion conformation, all three copies of the RBD lie flat on the spike surface, largely occluding the receptor-binding site, whereas in the open prefusion conformation, one or multiple RBDs lift to expose the receptor-binding site 2,7,9,10 . The surface of the trimer is extensively glyco- sylated with 22 potential N-linked glycosylation sites per monomer 2,6,7 . After receptor binding, structural transition of the prefusion form to the postfusion form brings the fusion peptide and the transmembrane domain together at one end of a long, needle-like structure centred around a three-helix bundle 8 . Five N-linked glycans are spaced along the length of the postfusion spike 8 .\nFully understanding how S proteins function and how they interact with the immune system requires knowledge of the structures, con- formations and distributions of S trimers within virions. In this study, we apply cryo-EM methods to study the structure, conformations and distributions of S trimers in&nbsp;situ on the virion surface.\nTo avoid artefacts associated with virus concentration or puri- fication, we aimed to image SARS-CoV-2 virions from the superna- tant of infected cells without further concentrating or purifying the virus. Vero E6 cells were infected with SARS-CoV-2 (isolate Germany/ BavPat1/2020) 24 . At 48&nbsp;h after infection, the&nbsp;supernatant was clari- fied, inactivated by fixation with formaldehyde and stored at −80 °C. Western blot analysis revealed that approximately 45% of the total S protein monomers on virions had been cleaved at the multibasic cleav- age site, resulting in S1 and S2 peptides (Fig.&nbsp;1a). The fixed supernatant\nhttps://doi.org/10.1038/s41586-020-2665-2\nReceived: 2 July 2020\nAccepted: 10 August 2020\nPublished online: 17 August 2020\nCheck for updates\n1 Structural Studies Division, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 2 Department of Infectious Diseases, Molecular Virology, Heidelberg University, Heidelberg, Germany. 3 Department of Infectious Diseases, Virology, Heidelberg University, Heidelberg, Germany. 4 Visual Aids Department, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 5 German Center for Infection Research, Heidelberg Partner Site, Heidelberg, Germany. 6 Division of Virus-Associated Carcinogenesis, German Cancer Research Center (DKFZ), Heidelberg, Germany. 7 These authors contributed equally: Zunlong Ke, Joaquin Oton, Kun Qu. ✉ e-mail: jbriggs@mrc-lmb.cam.ac.uk\nwas vitrified by plunge freezing and imaged by cryo-EM. Fixation may help to stabilize some protein conformations by crosslinking, but is not expected to give rise to any new conformations. As expected, given the concentration of virus in cellular supernatants (around 10 7 plaque-forming units per ml), small numbers of individual virions were found scattered around the grid—these were imaged by cryo-electron tomography (cryo-ET) (Fig.&nbsp;1b).\nVirions were approximately spherical, with a diameter to the outside of the lipid bilayer of 91&nbsp;±&nbsp;11&nbsp;nm (n&nbsp;=&nbsp;179) (Extended Data Fig.&nbsp;1a). They contained granular densities corresponding to N, and were studded with S trimers (Fig.&nbsp;1b, c). These features are generally consistent with those of other coronaviruses imaged using cryo-EM 1,16,17 . S trimers pro- truding from the viral surface displayed two distinct morphologies—a minority were extended thin structures reminiscent of the postfusion form, whereas the majority were wider structures reminiscent of the prefusion form. This observation is in contrast to a study showing cryo-EM images of purified SARS-CoV-2 virions inactivated with the nucleic acid modifier β-propiolactone, in which only thin protrusions were seen on the viral surface 25 , but is consistent with in&nbsp;situ observa- tions of virus assembly 21 .\nWe also collected tomograms of SARS-CoV-2 virions produced by infection of Calu-3 cells, a human lung carcinoma cell line that supports virus production with a similar titre to Vero E6 cells. The morphology of the virions and the appearance of the S trimers on the surface were consistent with those for virions produced from Vero E6 cells (Extended Data Fig.&nbsp;2a–c). Western blot analysis showed that approximately 73% of S from Calu-3 supernatant was in the cleaved form (Extended Data Fig.&nbsp;2d).\nIndividual virions contained 24&nbsp;±&nbsp;9 S trimers (Extended Data Fig.&nbsp;1b). This is fewer than previous estimates that assumed a uniform distri- bution of S 21 , because S was not uniformly distributed over the virus surface. A small sub-population of virions contained only few S trim- ers whereas larger virions contained higher numbers of S trimers (Extended Data Fig.&nbsp;1b). We identified 4,104 wide-structure S trimers and 116 thin-structure S trimers from 179 virions and analysed them using subtomogram averaging. The averaged structures, at 7.7 and 22&nbsp;Å resolution, respectively, correspond closely to published structures of purified S trimers in the prefusion and postfusion forms 2,7,8 (Fig.&nbsp;2a).\nOverall, approximately 97% of S trimers were in the prefusion form and 3% were in the postfusion form. Prefusion and postfusion forms were evenly distributed among virions.\nPrefusion S trimers on the virus surface may be predominantly in the closed conformation, with the open conformation induced or stabilized only with ACE2 binding; alternatively, both closed and open prefu- sion trimers may be present in the absence of ACE2. Open or closed conformations may induce different antibodies responses when used as immunogens, and there are ongoing efforts to generate S protein constructs that are stabilized in specific conformations 9,11,12 . To assess whether S trimers are present in open and/or closed conformations, we classified the RBD regions of individual monomers within trimers. We identified three classes of RBD, one with the RBD in the closed position, one with the RBD in the open position, and one in which the RBD was predominantly in the closed position, but showed some weakening of the density, suggesting the presence of more mobile conformations (Extended Data Fig.&nbsp;3). On the basis of our assignment of individual monomers to these classes, we derived structures of fully closed trimers and of trimers in which one RBD was open, representing approximately 31% and 55% of 3,854 prefusion trimers, respectively (Fig.&nbsp;2b, Extended Data Fig.&nbsp;4). We also identified a small number of trimers (around 14% of 3,854 prefusion trimers) in which two RBDs were in the open con- formation (Fig.&nbsp;2b). These observations confirmed that the opening of the RBD observed in recombinant S trimers also takes place on the virus surface, and that artificial S protein constructs stabilized in the closed and open conformations both represent structures present in&nbsp;situ. The receptor-binding site is therefore stochastically exposed in&nbsp;situ and available to interact with ACE2 and with antibodies.\nThe trimers did not all protrude perpendicularly from the viral sur- face. They were tilted by up to 90° towards the membrane, although the frequency of observed tilt angles decreased above 50° (Extended Data Fig.&nbsp;1c, d). We grouped trimers according to their orientation relative to the membrane, and averaged these groups independently. The averaged structures reveal that the membrane-proximal stalk region acts as a hinge with sufficient flexibility to allow tilting in all directions (Fig.&nbsp;2c).\nWe generated models of individual virus particles, with S trim- ers located at the position, orientation and conformation that were\nFig. 1 | Characterization of virus production and images of SARS-CoV-2 virions. a, Western blot analysis of SARS-CoV-2 non-structural protein 3 (nsp3), S and N in lysates of Vero E6 cells and in virus preparations, representative of three experiments. In released virions, S is present in both cleaved and uncleaved forms. The positions of S0, S2 and the S2–S2′ cleavage product are marked. b, Four representative tomographic slices of SARS-CoV-2 virions from\nthe supernatant of infected cells. Virions are approximately spherical, contain granular density corresponding to N-packaged genome, and have S trimers protruding at variable angles from their surfaces. Scale bar, 50 nm. c, Three S trimers from the dataset, shown as projections through the trimer to illustrate variable tilt towards the membrane. Scale bars, 10 nm.\ndetermined by subtomogram averaging (Fig.&nbsp;2d). S trimers appear to be distributed randomly on the viral surface, with no obvious clus- tering or relationship between location, orientation and conforma- tion. There is approximately one trimer per 1,000&nbsp;nm 2 of membrane surface, compared with approximately one per 100&nbsp;nm 2 for influenza A virus 26 . The sparse distribution of S, together with the predominantly closed state, suggest that receptor binding may be less dependent on avidity effects than in the case of pandemic influenza viruses 27 . This is consistent with the higher affinity between S and ACE2 (in the nM range 2,7 ) than between haemagglutinin and sialic acid (mM range 27 ).\nThe low concentration of particles in supernatant makes high-resolution structure determination difficult. We therefore concen- trated the virus by pelleting through a sucrose cushion. Concentrated virions deviated from the spherical morphology (Extended Data Fig.&nbsp;5), but the overall features were preserved. We performed cryo-ET and subtomogram averaging on the particles and observed predominantly\nprefusion S trimers, with occasional postfusion S trimers. Upon classifi- cation of the prefusion S trimers, we were able to identify the RBD only in the closed position; weak RBD density was observed in monomers (Extended Data Fig.&nbsp;3c).\nVirions in the supernatant from infected cells exhibited primarily pre- fusion S trimers in either closed or open prefusion conformations. Viri- ons concentrated through a sucrose cushion also exhibited prefusion conformations, but we did not observe the open conformation. Previ- ous studies have shown that virions inactivated with β-propiolactone rather than formaldehyde are primarily in a postfusion state 25 . S trimers purified from membranes are found only in the closed prefusion and postfusion conformations 8 , whereas the open RBD in soluble S trimers has been found to exhibit a wide range of different conformations 28 . These observations suggest that the open prefusion conformations of the spike protein that we observed before, but not after concentration, are fragile (despite the fixation applied here) and may be affected by the purification procedures.\nFig. 2 | Structural analysis of SARS-CoV-2 S trimers on intact virions. a, Structures of prefusion (left) and postfusion (right) trimers from intact virions, determined by subtomogram averaging. Structures are shown as transparent grey isosurfaces fitted with structures of the closed, prefusion trimer (Protein Data Bank (PDB) 6VXX) and the postfusion trimer (PDB 6XRA). One prefusion monomer is coloured blue (N terminus) to red (C terminus). The NTD is blue, the RBD is cyan. The NTD does not fully occupy the cryo-EM density because some loops are not resolved or built in the template structure (PDB 6VXX). b, Three conformations of the prefusion trimer observed on intact virions: all RBDs in the closed position (left, fitted with PDB 6VXX); one RBD in the open position (centre, fitted with PDB 6VYB); two RBDs in the open position (right, fitted with PDB 6X2B, which lacks modelled glycans). The two-open\nconformation has only been observed in&nbsp;vitro after inserting multiple stabilizing mutations. S monomers with closed RBDs are green, and those with open RBDs are blue. c, Averaging of subsets of trimers grouped according to their orientation relative to the membrane shows flexibility in the stalk region. Examples are shown for pools centred at 0°, 30° and 60° from the perpendicular, and for two rotations of the trimer relative to the tilt direction. d, Three-dimensional models of two individual SARS-CoV-2 virions with a membrane (blue) of the measured radius, and all spike proteins shown in the conformations, positions and orientations determined by subtomogram averaging. Different S conformations are distributed over the virion surface and can be tilted by up to approximately 90° relative to the membrane (Extended Data Fig.&nbsp;1c, d).\nOur data indicate that inactivation and purification methods can alter the ratio of prefusion and postfusion forms, as well as the ratio of open and closed conformations. It has been suggested that sub- stantial amounts of postfusion S on the virus surface may protect the virus by shielding the prefusion form, or may shift the host response towards non-neutralizing antibodies 8 . Our observation that intact virions harbour only a small fraction of postfusion spikes suggest that this is unlikely to be an important defence mechanism for the virus during infection, but it may be an important consideration for vaccina- tion. Candidate vaccines based on inactivated virus particles are under development. These may present different S protein epitopes to the immune system depending on how they are prepared, and may there- fore differ in their ability to induce a neutralizing response. For exam- ple, β-propiolactone is frequently used in vaccine production (such as for influenza virus subunit vaccines 29 ), but if postfusion S induces\nnon-neutralizing responses, β-propiolactone may not be optimal for inactivation during SARS-CoV-2 S vaccine formulation.\nWe next imaged the concentrated virus in 2D using cryo-EM and per- formed single-particle analysis on prefusion S trimers that protruded from the sides of the virus particles, generating a consensus structure of the prefusion trimer at 3.4&nbsp;Å resolution. Focused classification with par- tial signal subtraction on individual RBD monomers led to two classes (Extended Data Fig.&nbsp;5). Consistent with the absence of open conforma- tions in this sample in cryo-ET images, 81% of the monomers showed the RBD in the closed conformation and 19% showed weaker density for the RBD, which remained predominantly in the closed position. We refined the structures of S trimers in which all three RBDs are in the closed conformation (53% of the data), and those in which at least one RBD has weak density (47% of the data), to resolutions of 3.5&nbsp;Å and 4.1&nbsp;Å, respectively (Fig.&nbsp;3a, Extended Data Fig.&nbsp;6). The two structures are highly similar, differing only in the density levels for one RBD. We used the structure with three closed RBDs to build and refine an atomic model of the S protein trimer in&nbsp;situ on the viral surface.\nThe positions of glycans on the surface of S are well resolved in our structure, with density at 17 of the 22 predicted N-glycosylation sites (Fig.&nbsp;3b). The other five glycosylation sites are in disordered NTD loops or in the stalk region and are not resolved at high resolution. At the base of the trimer, a clear ring of glycans forms a collar above the stalk region. The density for the stalk region is extended by two helical turns compared with published structures of soluble ectodomain, and fur- ther at lower resolution before fading owing to stalk flexibility (Fig.&nbsp;3c). SARS-CoV-2 S trimers are sparsely distributed and can be highly tilted towards the membrane. This implies that epitopes at the base of the head domain and in the stalk region would be accessible to antibodies where they are not protected by the extensive glycan shell.\nWe compared the in&nbsp;situ structure of the S trimer with structures previously obtained using exogenously expressed purified protein. A recent study of full-length trimeric S solubilized in detergent micelles 8 identified two features that are not seen in most structures of soluble S ectodomain trimers: well-defined density for residues 14–26 of S, and a folded loop between residues 833 and 853. This loop is folded in the structure of the ‘locked’ conformation of the ectodomain 9 , and may become folded in the low-pH conditions in the endosome 30 . We observed only weak density for residues 14–26, and&nbsp;no folded structure for the 833–853 region. The SARS-CoV-2 strain that we have imaged contains the widely circulating D614G substitution 31 , which abolishes a salt bridge to K854 (Fig.&nbsp;3d) and may reduce folding of the 833–854 loop 9,30 . We did not observe additional density that would correspond to bound lipids, as described recently 10 , or other bound co-factors. These may be present sub-stoichiometrically or in rare conformations, but are not a general feature of the S trimer in&nbsp;situ. Overall, our structure is very similar to that of the soluble trimeric ectodomain in the closed prefusion form stabilized by a double proline mutation 2,7 (Extended Data Fig.&nbsp;7). This validates the ongoing use of recombinant, purified S trimers for research, diagnostics and vaccination—they indeed represent the in&nbsp;situ structure of S. In this study, we have determined the structure of S trimers on the virion surface to 3.4&nbsp;Å resolution using cryo-EM (Fig.&nbsp;3), supporting the further use of this technique to study antibody binding to S in the context of the viral surface. Such studies could provide insights into how neutralizing antibodies block virus infection, particularly for antibodies against membrane-proximal regions of S, and thus can inform the design of immunogens for vaccination.\nOnline content\nAny methods, additional references, Nature Research reporting sum- maries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author con- tributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41586-020-2665-2.\nFig. 3 | Structures of SARS-CoV-2 S trimers on intact virions by single-particle reconstruction. a, Top and side views of trimers with three closed RBDs (left, 3.5&nbsp;Å resolution) and one RBD with weaker density (right, 4.1&nbsp;Å resolution). Individual monomers are coloured white, blue and green. Comparison of the left and right structures shows the weaker density for the RBD of the green monomer in the region indicated by the red arrowheads. b, Glycosylation profile of the S protein. Colour scheme as in a; glycans are shown in orange. Boxes indicates the regions shown in c and d. c, Enlarged view of the base of the trimer at lower isosurface threshold to highlight the glycan ring and the extended C-terminal density. d, Enlarged view of the region of the spike where the D614G variation abolishes a salt bridge to K854.\n1. Neuman, B. W. &amp; Buchmeier, M. J. Supramolecular architecture of the coronavirus particle. Adv. Virus Res. 96 , 1–27 (2016). 2. Walls, A. C. et&nbsp;al. Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein. Cell 181 , 281–292 (2020).\n3.  Hoffmann, M. et&nbsp;al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. Cell 181 , 271–280 (2020).\n4.  Shang, J. et&nbsp;al. Structural basis of receptor recognition by SARS-CoV-2. Nature 581 , 221–224 (2020).\n5. Wang, Q. et&nbsp;al. Structural and functional basis of SARS-CoV-2 entry by using human ACE2. Cell 181 , 894–904 (2020).\n6.  Watanabe, Y., Allen, J. D., Wrapp, D., McLellan, J. S. &amp; Crispin, M. Site-specific glycan analysis of the SARS-CoV-2 spike. Science 369 , 330–333 (2020). 7. Wrapp, D. et&nbsp;al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367 , 1260–1263 (2020). 8. Cai, Y. et&nbsp;al. Distinct conformational states of SARS-CoV-2 spike protein. Science 369 , 1586–1592 (2020).\n9. Xiong, X. et&nbsp;al. A thermostable, closed SARS-CoV-2 spike protein trimer. Nat. Struct. Mol. Biol. 27 , 934–941 (2020).\n10.  Toelzer, C. et&nbsp;al. Free fatty acid binding pocket in the locked structure of SARS-CoV-2 spike protein. Science eabd3255 (2020).\n11.  Henderson, R. et&nbsp;al. Controlling the SARS-CoV-2 spike glycoprotein conformation. Nat. Struct. Mol. Biol. 27 , 925–933&nbsp;(2020).\n12.  McCallum, M., Walls, A. C., Bowen, J. E., Corti, D. &amp; Veesler, D. Structure-guided covalent stabilization of coronavirus spike glycoprotein trimers in the closed conformation. Nat. Struct. Mol. Biol. 27 , 942–949&nbsp;(2020).\n13.  Zhu, N. et&nbsp;al. A novel coronavirus from patients with pneumonia in China, 2019. N. Engl. J. Med. 382 , 727–733 (2020).\n14.  Zhou, P. et&nbsp;al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579 , 270–273 (2020).\n15.  Masters, P. S. &amp; Perlman, S. in Fields Virology 6th edn (eds Knipe, D. M. &amp; Howley, P. M.) 826–858 (Elsevier, 2013).\n16.  Neuman, B. W. et&nbsp;al. Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy. J. Virol. 80 , 7918–7928 (2006). 17.  Bárcena, M. et&nbsp;al. Cryo-electron tomography of mouse hepatitis virus: insights into the structure of the coronavirion. Proc. Natl Acad. Sci. USA 106 , 582–587 (2009).\n18.  Walls, A. C. et&nbsp;al. Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion. Proc. Natl Acad. Sci. USA 114 , 11157–11162 (2017).\n19.  Yuan, Y. et&nbsp;al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat. Commun. 8 , 15092 (2017). 20.  Fehr, A. R. &amp; Perlman, S. Coronaviruses: an overview of their replication and pathogenesis. Methods Mol. Biol. 1282 , 1–23 (2015).\n21.  Klein, S. et&nbsp;al. SARS-CoV-2 structure and replication characterized by in&nbsp;situ cryo-electron tomography. Preprint at https://doi.org/10.1101/2020.06.23.167064 (2020).\n22.  Snijder, E. J. et&nbsp;al. A unifying structural and functional model of the coronavirus replication organelle: tracking down RNA synthesis. PLoS Biol. 18 , e3000715 (2020). 23.  Hoffmann, M., Hofmann-Winkler, H. &amp; Pöhlmann, S. in Activation of Viruses by Host Proteases (eds Böttcher-Friebertshäuser, E. et&nbsp;al.) 71–98 (2018).\n24.  Rothe, C. et&nbsp;al. Transmission of 2019-nCoV infection from an asymptomatic contact in Germany. N. Engl. J. Med. 382 , 970–971 (2020).\n25.  Liu, C. et&nbsp;al. Viral architecture of SARS-CoV-2 with post-fusion spike revealed by cryo-EM. Preprint at https://doi.org/10.1101/2020.03.02.972927 (2020).\n26.  Chlanda, P. et&nbsp;al. The hemifusion structure induced by influenza virus haemagglutinin is determined by physical properties of the target membranes. Nat. Microbiol. 1 , 16050 (2016).\n27.  Xiong, X. et&nbsp;al. Receptor binding by a ferret-transmissible H5 avian influenza virus. Nature 497 , 392–396 (2013).\n28.  Gui, M. et&nbsp;al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res. 27 , 119–129 (2017).\n29.  Bonnafous, P. et&nbsp;al. Treatment of influenza virus with beta-propiolactone alters viral membrane fusion. Biochim. Biophys. Acta 1838 , 355–363 (2014).\n30.  Zhou, T. et&nbsp;al. A pH-dependent switch mediates conformational masking of SARS-CoV-2 spike. Preprint at https://doi.org/10.1101/2020.07.04.187989 (2020).\n31.  Korber, B. et&nbsp;al. Tracking changes in SARS-CoV-2 spike: evidence that D614G increases infectivity of the COVID-19 virus. Cell 182 , 812–827 (2020).\nPublisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n© The Author(s), under exclusive licence to Springer Nature Limited 2020\n\n\nMethods\nNo statistical methods were used to predetermine sample size. For Cryo-ET, division of dataset into two random halves was done based on virus number. For Cryo-EM, division of datasets into two random halves was done based on the standard approach in RELION 3. Other experiments did not involve randomization. The investigators were not blinded to allocation during experiments and outcome assessment.\n\n\nCells and virus\nVero E6 cells were obtained from ATCC and were cultured in Dulbecco’s modified Eagle medium (DMEM, Life Technologies) containing 10% fetal bovine serum, 100 U&nbsp;ml −1 penicillin, 100 μg&nbsp;ml −1 streptomycin and 1% non-essential amino acids (complete medium). The Germany/Bav- Pat1/2020 SARS-CoV-2 strain was isolated by C. Drosten (Charité, Berlin, distributed by the European Virology Archive (Ref-SKU: 026V-03883)) at passage (P)2. A stock of SARS-CoV-2 was obtained by passaging the virus once in Vero E6 cells (P3). To produce SARS-CoV-2 virions, Vero E6 cells grown on 75 cm 2 side-bottom tissue culture flasks were infected with SARS-CoV-2 (P3) at a multiplicity of infection (MOI) of 0.5. Culture medium from infected cells was collected at 48 h after infection, clari- fied by centrifugation at 1,000g for 10 min, cleared through a 0.45-μm nitrocellulose filter and fixed with 4% formaldehyde for 30 min at room temperature. Culture medium was supplemented with 10 mM HEPES (pH 7.2) before fixation. Virus-containing medium was subsequently split into aliquots and stored at −80 °C. Infectious supernatants con- taining SARS-CoV-2 virions were obtained from Calu-3 cells infected with P3 virus at an MOI of 5 for 48 h and processed as described above. To obtain SARS-CoV-2 virions at high concentration, infection and collection of Vero E6 culture medium were performed as above, fol- lowed by concentration of fixation-inactivated virions from medium by ultracentrifugation through a 20% (wt/wt) sucrose cushion (120 min at 27,000 rpm in a Beckman SW32 rotor; Beckman Coulter Life Sciences). Pelleted particles were resuspended in PBS and stored in aliquots at −80 °C.\n\n\nWestern blot\nVero E6 or Calu-3 cells mock infected or infected for 48 h with SARS-CoV-2 (MOI&nbsp;=&nbsp;5) were washed twice in PBS, scraped, pelleted at 700g for 5 min and lysed in PBS pH 7.4, containing 1% Triton&nbsp;X-100 and protease inhibitors (Merck) for 30 min at 4 °C. Samples were centri- fuged at 4 °C for 30 min and supernatants were collected. Total pro- tein concentration was calculated using the Bio-Rad Protein Assay kit (Bio-Rad). Purified viruses were prepared for western blot by cen- trifugation of 32 ml of virus containing supernatants on a 10% sucrose cushion in a Beckmann J25 centrifuge. Centrifugation was performed at 10,000g for 4 h at 4 °C. Supernatants were discarded and purified virus pellet were resuspended in 500 μl PBS. For western blotting, 10 μg of total cell lysates and 5 μl purified viruses were diluted in Laemmli buffer and loaded on a pre-cast Criterion XT 4–12% gradient gel (Bio-Rad). Gels were transfered to PVDF membrane using a wet-electroblotting chamber system (Bio-Rad) in Towbin buffer containing 10% methanol. Transfer was performed overnight at 4 °C. Membranes were washed in PBS and blocked with 10% milk in PBS containing 0.2% tween-20 (PBS-T) for 1 h. Membranes were incubated for 1 h at room temperature with primary antibodies specific for an epitope in the C-terminal region of S (Abcam, ab252690; diluted 1:1,000 in PBS-T), for N protein (Sino Biological, 40143-MM05; diluted 1:1,000 in PBS-T) or nsp3 (Abcam, ab181620; diluted 1:500 in PBS-T). Next, the membranes were washed 3 times in PBS-T, incubated with horseradish peroxidase-conjugated anti-mouse or anti-rabbit antibodies at 1:10,000 dilution for 1 h, washed again 3 times in PBS-T, incubated with western Lightning Plus-ECL rea- gent (Perkin Elmer) and imaged using an Intas ChemoCam Imager 3.2 (Intas). Densitometric analysis of western blot assays was performed using LabImage 1D L340 software v.4.1 (Intas). The blot shown is\nrepresentative of three independent experiments. Uncropped, unpro- cessed blots are presented in the Source Data.\n\n\nPCR with reverse transcription and spike sequencing\nTotal RNA was isolated from infected Vero E6 cells 48 h after infec- tion with Germany/BavPat1/2020 SARS-CoV-2 (P2). Spike cDNA was produced from the total RNA using Superscript III (ThermoFisher) with specific reverse transcription primer (CAATTGTGAAGATTCTC ATA). The cDNA was amplified by PCR using specific primers (Fwrd, ATGTTTGTTTTTCTTGTTTTATT; rev, TTATGTGTAATGTAATTTGA) and the resulting amplicon was sent for Sanger sequencing. Sequences were compared to the Germany/BatPat1/2020 SARS-CoV-2 reference sequence (Ref-SKU: 026V-03883) and found to be identical. Specific sequencing primers: Fwrd1, ATGTTTGTTTTTCTTGTTTTATT; Fwrd2, GGTTGGACAGCTGGTGCT; Fwrd3, CCAACCATACAGAGTAGTAGTA; Rev1, GTAGCAGCAAGATTAGCAGAA; and Rev2, TTATGTGTAATGT AATTTGA.\n\n\nCryo-ET sample preparation\nFixed virus samples from the supernatant of infected cells without any concentration step (unconc) or concentrated by pelleting through a sucrose cushion (conc) were prepared, imaged, and processed in parallel. The virus suspension was mixed with 10-nm colloidal gold (in PBS solution) in 10:1 ratio. Then, 3 μl of the solution was added to a glow-discharged copper grid (C-Flat 2/2, Protochips). Grids were plunge frozen into liquid ethane by back-side blotting using a LeicaGP cryo plunger (Leica) and stored in liquid nitrogen until imaging.\n\n\nCryo-ET data collection\nCryo-ET data collection was performed essentially as described previ- ously 32 . Cryo-grids were loaded into an FEI Titan Krios transmission electron microscope operated at 300 kV and images were recorded on a Gatan K2 Summit direct detection camera in counting mode with a 20 eV energy slit in zero-loss mode. Tomographic tilt series between −60° and +60° were collected using SerialEM 3.8.0 software 33 in a dose-symmetric scheme 34 with a 3° angular increment. A total dose of 120 e − A −2 per tilt series was distributed evenly among 41 tilt images. The nominal mag- nification was 81,000×, giving a pixel size of 1.532 Å on the specimen. The defocus range was between −2 μm and −6 μm and 10 frames were saved for each tilt angle. All data acquisition parameters are listed in Extended Data Table&nbsp;1.\nFrames were motion-corrected in IMOD 4.10.30 35 and images were dose-filtered using the alignframes function in IMOD. Exposure fil- tering was implemented according to the cumulative dose per tilt as described elsewhere 36 . The contrast transfer function (CTF) was meas- ured using non-dose-filtered images using the ctfplotter package within IMOD 37 . Tilt series stacks were sorted using IMOD newstack function and fiducial-alignment of all tilt series was performed in IMOD/etomo. Tomograms with less than three trackable gold fiducials were discarded. Motion-corrected and dose-filtered tilt stacks were CTF-corrected by CTF multiplication and tomograms were reconstructed by weighted back-projection in novaCTF 38 . Tomograms were low-pass filtered to 50 Å for better visualization in EMAN2.2 39 and tomographic slices were visualized with IMOD.\n\n\nExtraction of S trimers from tomograms\nThe initial steps of subtomogram alignment and averaging were imple- mented using MATLAB (MathWorks) scripts, which were derived from the TOM 40 and AV3 41 packages as described previously 32 . The missing wedge was modelled as the summed amplitude spectrum of back- ground subtomograms for each tomogram, and was applied during alignment and averaging.\nTo generate an initial template model of the spike protein from the viral surface, 68 spikes were manually picked from four virions of tomograms that were down-scaled by 4× binning of the voxels.\nThe 68 spikes’ initial Euler angles (2 out of 3) were determined on the basis of the vector between two points, one on the head of the spike and one on the membrane where the spike anchors, respectively. The 68 spikes were iteratively aligned to one another for four iterations, applying three-fold symmetry to generate a low-resolution template that resembled a prefusion conformation of the spike. This template was used as an alignment reference for all virions (below). All postfu- sion spikes were manually identified and picked, and initial Euler angles were assigned in the same manner.\nThe centre of each virion was then marked manually using the Vol- ume Tracer function in UCSF Chimera (v.1.13.1) 42 and the radius of the virion was determined centred at the membrane using the Pick Particle Chimera Plugin 43 . An oversampled spherical grid of points was gener- ated on the virion surface with ~9 nm spacing, and subtomograms were extracted for all grid points with a box size of 96 pixels (approximately 60 nm) centred at a radius 14 nm above these grid positions (approxi- mately the radius of the expected centre of the spike). Initial Euler angles were assigned to each subtomogram based on the orientation of the normal vectors relative to the sphere surface.\nSubtomograms were aligned against the low-resolution template (from the above average of 68 spikes). During this alignment, subto- mogram positions converged onto clusters at the true spike positions. One subtomogram position was kept for each cluster, excluding parti- cles within a distance of ~60 Å (10 pixels at 4× binning) and removing particles with cross-correlation coefficients below 0.11. Subtomo- grams that had tilted by more than 90° relative to their perpendicular positions were excluded. Visual inspection of the tomograms using the Place Object Chimera Plugin 43 confirmed that subtomograms selected in this manner corresponded to S trimers on the viral surface (see below). Subtomograms were divided into two halves on the basis of virion number. From this point on the two halves were processed independently.\n\n\nSubtomogram averaging\nSubsequent processing was performed in RELION 44 . For this purpose, subtomograms were reconstructed from the original tilt series images after motion correction using relion_reconstruct. Using dedicated Python scripts, the S trimer positions in the 3D tomograms from the procedure outlined above were converted into 2D positions and defocus values in the corresponding tilt series images, as well as Euler angles in the RELION convention. Individual sub-tomograms were reconstructed at a 2× downscaled pixel size of 3.064 Å, by 3D insertion of Fourier slices of the cropped regions of the tilt series images, each multiplied by their corresponding CTF, which included the dose filter. Similarly, 3D-CTF volumes were generated by 3D insertion of the cor- responding CTF 2 slices. Subtomograms were reconstructed in a box size of 128 voxels for prefusion trimers and a box size of 192 voxels for averages of tilted prefusion trimers and for postfusion trimers.\nTo deal with the CTF pre-multiplied sub-tomograms, as well as the multiplicity of each 3D voxel in the tilt series, a modified version of RELION was used for subtomogram refinement and classification (details to be described elsewhere). Standard 3D auto-refinement was performed with C3 symmetry and a soft-edged mask around the trimers, using a 30 Å lowpass-filtered map as an initial reference. Using 3,854 of the 4,104 subtomograms (excluding those that had tilted by more than 90° relative to the perpendicular position (Extended Data Fig.&nbsp;1c)), a 7.7 Å consensus map was calculated for the prefusion trim- ers; 116 postfusion trimers led to a 22 Å map.\nAngular density (Extended Data Fig.&nbsp;1c) was illustrated by dividing the number of spikes per histogram bin by the sine of the tilt angle. From this histogram, the noise level in the distribution due to incorrectly aligned spikes was estimated assuming they are uniformly distrib- uted (red dashed line in Extended Data Fig.&nbsp;1c), and based on averaging over the range 140–180°. Such angles must be incorrectly alignments because they orient the spike towards the inside of the virus.\nBased on the subtomograms that contributed to the consensus map we estimated that there are 23&nbsp;±&nbsp;9 prefusion S trimers per virion. For a subset of 28 virions, we visually compared the trimers identified by subtomogram averaging with the tomograms to identify false positives or false negatives, finding that the number of S trimers identified by subtomogram averaging is an underestimate of the number of observed S trimers by 0.9 trimers per virion. On this basis we estimate there are 24&nbsp;±&nbsp;9 prefusion S trimers per virion.\nNext, we performed symmetry expansion 45 , followed by focused classification without alignment and with partial signal subtraction, while keeping the orientations from the consensus refinement fixed. The mask used for focused classification was generated manually and enclosed the RBD of one monomer (Extended Data Fig.&nbsp;3a), including the closest NTD of the neighbouring monomer. Classification of the primary dataset (unconc1) led to three different RBD states: closed (45% of the monomers); open (28%) and with weak density (27%) (Extended Data Fig.&nbsp;3a). Classification of a second, smaller dataset of virus parti- cles in supernatant (unconc2), comprising 1,224 trimers yielded 39% closed RBD, 22% open RBD and 39% weak RBD, in general agreement with the unconc1 results (Extended Data Fig.&nbsp;3b). Classification of a dataset of virus particles after concentration through a sucrose cush- ion (conc), comprising 3,788 trimers, yielded 67% of the monomers corresponding to closed RBDs and 33% to RBDs with weak density. For this dataset, no RBDs in the open conformation were identified (Extended Data Fig.&nbsp;3c).\nThe classification of the RBDs as closed, open or with weaker density was used to divide the unconc1 dataset into three subsets: 31% (1,175 trimers) have no open RBDs; 55% (2,121 trimers) have one RBD in the open state; and 14% (525 trimers) have two open RBDs. The remaining 33 trimers have three RBDs in the open state and not further processed. In the class with weaker density, the RBD appears to be predominantly in the closed state and was treated as closed for this assignment. For each of the three subsets, reconstruction of the two independently refined half sets was performed using the orientations from the consensus refinement that gave the 7.7 Å consensus map described above. Subse- quent standard post-processing procedures for resolution estimation, map sharpening and local-resolution filtering in RELION led to three final maps. The subset with no open RBDs gave a reconstruction with C3 symmetry in which all three RBDs were closed at 8.6 Å resolution. The other two subsets yielded structures with C1 symmetry and either one or two open RBDs, with resolutions of 8.6 Å and 9.9 Å, respectively (Extended Data Figs.&nbsp;3, 4). Note, that while fixation is not expected to give rise to any new conformations, we cannot rule out that it dif- ferentially stabilizes different conformations, and the position of the equilibrium between open and closed conformations in unfixed sam- ples may differ from those estimated from fixed samples.\nAverages of tilted prefusion trimers were generated by grouping according to the tilt and rotation of the subtomogram away from the normal vector to the membrane. Subtomograms were included in a group for averaging if they were within 15° of the displayed tilt (0°, 30° and 60°) and rotation (0°, 60°). The pools illustrated in Fig.&nbsp;2c contain 14.5, 14.1, 12.0 and 10.0% of the subtomograms.\n\n\nCryo-EM sample preparation and data collection\nVirus solution concentrated through a 20% (wt/wt) sucrose cushion was plunge-frozen on C-Flat 2/2 3C grids (Protochips) following the same procedure as for cryo-ET, but without adding gold fiducials. Grids were imaged on a Thermo Fisher Scientific Titan Krios transmission electron microscope that was operated at 300 kV, using a Gatan K3 direct electron detector and a Gatan BioQuantum energy filter with a 20 eV energy slit. Movies with 48 frames and an accumulated dose of 50 e − Å −2 were acquired in counting mode using SerialEM-3.8.0 33 at a nominal magnification of 81,000×, corresponding to a calibrated pixel size of 1.061 Å&nbsp;per&nbsp;pixel. Detailed data acquisition parameters are summarized in Extended Data Table&nbsp;2.\n\n\nCryo-EM image processing\nThe scheduler functionality in RELION-3.1 was used for fully automated real-time processing during data collection 9,46 . Mov- ies were motion-corrected and dose-weighted using the RELION implementation of the MotionCor2 algorithm 47 . Subsequently, non-dose-weighted sums were used to estimate the CTF in CTFFIND-4.1.13 48 . S trimers that were extending from the sides of virus particles were picked manually (4,493 particles from the first 100 micrographs) and then used as a training set for optimisation of the convolutional neural network in the automated particle picking software Topaz 49 . Extracted particles were subjected to 3D classification using a previously determined structure of the S trimer 9 , low-pass-filtered to 30 Å, as initial 3D reference. The selected 286,407 particles that contributed to 3D classes corresponding to S trimers were submitted to Bayesian polishing to correct for per-particle beam-induced motions and a second round of 3D clas- sification to select the 55,159 particles that contributed to the best class. This final consensus set of particles was subjected to CTF refinement of per-particle defocus, per-micrograph astigmatism and beam tilt, followed by a second round of Bayesian polishing. 3D auto-refinements were performed with the selected particles after each round of 3D classification, CTF refinement or Bayesian polishing. The consensus structure had a resolution of 3.4 Å.\nSubsequently, symmetry expansion 45 was performed, followed by focused 3D classification with partial signal subtraction 50 . Using a mask on a single RBD to focus classification into six classes while keeping the orientations of the last consensus refinement fixed resulted in the identification of two RBD states: closed or with weak density. S trimers with all three RBDs in the closed state were refined separately from S trimers with one RBD with weak density, resulting in two final maps with resolutions. Standard RELION post-processing was used for resolution estimation, map sharpening and local-resolution filtering. The C3 symmetric map with three closed RBDs had an estimated overall resolution of 3.5 Å; the C1 map with one weaker RBD extended to 4.1 Å resolution (Extended Data Fig.&nbsp;6b).\n\n\nModel building and refinement\nThe SARS-CoV-2 S trimer structure 9 (PDB 6ZP0) was used as an initial model for building into the model with three closed RBDs (Extended Data Fig.&nbsp;6a, middle). Residues were adjusted manually in Coot 0.9 51 . Steric clash and sidechain rotamer conformations were improved using the Namdinator web server 52 . After further iterations of manual adjustment, the structure was refined in PHENIX-1.18.2 53 . The geom- etry and statistics are given in Extended Data Table&nbsp;2. The unmasked model-to-map FSC was calculated in PHENIX for the refined model against the full reconstruction.\n\n\n3D model of spikes on authentic virions\nTo visualize the spike protein on the authentic SARS-CoV-2 virions, the coordinates, orientations and conformational classes determined by subtomogram averaging were converted into a format compat- ible with Maxon Cinema 4D (v.S22.116), and imported together with the 3D models of the different conformational states determined by subtomogram averaging. To generate representative virion images in Fig.&nbsp;2d we removed the three false positives from one virion and positioned the three false negatives on each virion that had been identified by visual inspection of the data (see above). The HR2 region was modelled as a cylinder. Images of individual virions from the dataset were rendered into Adobe Photoshop to generate images for presentation.\n\n\nReporting summary\nFurther information on research design is available in the&nbsp;Nature Research Reporting Summary linked to this paper.\n\n\nData availability\nPublished structures for comparison or initial model building were obtained from the PDB with accession codes 6VXX 2 , 6VYB 2 , 6X2B 11 , 6XRA 8 and 6ZP0 9 . The cryo-EM and cryo-ET structures determined here and representative tomograms are deposited in the Electron Micros- copy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (three closed RBDs from subtomo- gram averaging), EMD-11495 (one open RBD), EMD-11496 (two open RBDs), EMD-11497 (three closed RBDs from cryo-EM), and EMD-11498 (two open RBDs and one weak RBD). The associated molecular models are deposited in the PDB under accession code 6ZWV (three closed RBDs from cryo-EM). The raw micrographs are deposited in the Electron Microscopy Public Image Archive under accession codes EMPIAR-10492 (cryo-EM) and EMPIAR-10493 (cryo-ET).\n32.  Wan, W. et&nbsp;al. Structure and assembly of the Ebola virus nucleocapsid. Nature 551 , 394–397 (2017).\n33.  Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. J. Struct. Biol. 152 , 36–51 (2005).\n34.  Hagen, W. J. H., Wan, W. &amp; Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197 , 191–198 (2017).\n35.  Kremer, J. R., Mastronarde, D. N. &amp; McIntosh, J. R. Computer visualization of three-dimensional image data using IMOD. J. Struct. Biol. 116 , 71–76 (1996).\n36.  Grant, T. &amp; Grigorieff, N. Measuring the optimal exposure for single particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6. eLife 4 , e06980 (2015).\n37.  Xiong, Q., Morphew, M. K., Schwartz, C. L., Hoenger, A. H. &amp; Mastronarde, D. N. CTF determination and correction for low dose tomographic tilt series. J. Struct. Biol. 168 , 378–387 (2009).\n38.  Turoňová, B., Schur, F. K. M., Wan, W. &amp; Briggs, J. A. G. Efficient 3D-CTF correction for cryo-electron tomography using NovaCTF improves subtomogram averaging resolution to 3.4 Å. J. Struct. Biol. 199 , 187–195 (2017).\n39.  Galaz-Montoya, J. G., Flanagan, J., Schmid, M. F. &amp; Ludtke, S. J. Single particle tomography in EMAN2. J. Struct. Biol. 190 , 279–290 (2015).\n40.  Nickell, S. et&nbsp;al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149 , 227–234 (2005).\n41.  Förster, F., Medalia, O., Zauberman, N., Baumeister, W. &amp; Fass, D. Retrovirus envelope protein complex structure in&nbsp;situ studied by cryo-electron tomography. Proc. Natl Acad. Sci. USA 102 , 4729–4734 (2005).\n42.  Pettersen, E. F. et&nbsp;al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25 , 1605–1612 (2004).\n43.  Qu, K. et&nbsp;al. Structure and architecture of immature and mature murine leukemia virus capsids. Proc. Natl Acad. Sci. USA 115 , E11751–E11760 (2018).\n44.  Bharat, T. A. M., Russo, C. J., Löwe, J., Passmore, L. A. &amp; Scheres, S. H. W. Advances in single-particle electron cryomicroscopy structure determination applied to sub-tomogram averaging. Structure 23 , 1743–1753 (2015).\n45.  Scheres, S. H. W. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579 , 125–157 (2016).\n46.  Zivanov, J. et&nbsp;al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7 , e42166 (2018).\n47.  Zheng, S. Q. et&nbsp;al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14 , 331–332 (2017).\n48.  Rohou, A. &amp; Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192 , 216–221 (2015).\n49.  Bepler, T. et&nbsp;al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Nat. Methods 16 , 1153–1160 (2019).\n50.  Bai, X. C., Rajendra, E., Yang, G., Shi, Y. &amp; Scheres, S. H. W. Sampling the conformational space of the catalytic subunit of human γ-secretase. eLife 4 , e11182 (2015). 51.  Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. Acta Crystallogr. D 60 , 2126–2132 (2004).\n52.  Kidmose, R. T. et&nbsp;al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6 , 526–531 (2019).\n53.  Afonine, P. V. et&nbsp;al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr. D 74 , 531–544 (2018).\nAcknowledgements We thank the staff of the MRC-LMB for generous support during the COVID-19 pandemic lockdown; all of&nbsp;the staff of the MRC-LMB EM Facility, in particular A. Yeates, G. Sharov and G. Cannone, for supporting the EM experiments; J. Grimmett and T. Darling for supporting scientific computing; C. Drosten and EVAg for provision of the SARS-CoV-2 strain used here; and M. Frey for the Calu-3 cell line. We acknowledge microscopy support from the Infectious Diseases Imaging Platform (IDIP) at the Center for Integrative Infectious Disease Research Heidelberg. This study was supported by funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (ERC-CoG-648432 MEMBRANEFUSION to J.A.G.B.), the Medical Research Council as part of UK Research and Innovation (MC_UP_A025_1013 to S.H.W.S.; MC_UP_1201/16 to J.A.G.B.), the Deutsche Forschungsgemeinschaft (240245660–SFB 1129 project 5 to H.-G.K. and project 11 to R.B.) and the Japan Society for the Promotion of Science (overseas research fellowship to T.N.).\nAuthor contributions Z.K. and J.A.G.B. conceived the study. Z.K., J.O., K.Q., M.C., V.Z., H.-G.K., R.B. and J.A.G.B. designed the study. M.C., V.Z., C.J.N. and B.C. prepared and characterized virus samples with supervision from H.-G.K. and R.B. Z.K. and K.Q. collected cryo-EM data. Z.K., J.O. and K.Q. processed cryo-EM data with assistance from T.N., J.Z., J.M.L. and J.P., supervised by S.H.W.S. and J.A.G.B. Z.K., J.O., K.Q., M.C. and L.M. made figures. Z.K., J.O., K.Q., M.C., X.X. and J.A.G.B. interpreted data. J.A.G.B. prepared the initial draft and managed the project. Z.K., J.O., K.Q., M.C. and J.A.G.B. wrote the manuscript with input from all authors. S.H.W.S., H.-G.K., R.B. and J.A.G.B. obtained funding.\nCompeting interests The authors declare no competing interests.\nAdditional information Supplementary information is available for this paper at https://doi.org/10.1038/s41586-020- 2665-2.\nCorrespondence and requests for materials should be addressed to J.A.G.B. Peer review information Nature thanks Jason McLellan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Reprints and permissions information is available at http://www.nature.com/reprints.\nExtended Data Fig. 1 | Characterization of SARS-CoV-2 virion morphology. a, Histogram of virion diameters for unconcentrated extracellular virions in the supernatant of two independent preparations (top and middle), and for extracellular virions after concentration through a sucrose cushion (bottom). After concentration the virions become less spherical. Mean and standard deviation for diameters are 91&nbsp;±&nbsp;11 nm (n&nbsp;=&nbsp;179), 94&nbsp;±&nbsp;9 nm (n&nbsp;=&nbsp;68) and 92&nbsp;±&nbsp;8 nm (n&nbsp;=&nbsp;227) for the three preparations. b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. Visual inspection indicates that almost all spikes were identified for virions in the supernatant, but that not all spikes are identified in the concentrated preparation leading to an underestimate of the number of spikes. c, Histogram of spike tilt angle towards the membrane for the\nlarger supernatant virus dataset (unconc1). The vertical black dashed line indicates 90°. 97% of particles have tilts below 90°; particles with tilts above 90° were not included in image analysis. The angular density (right) is calculated by dividing the number of spikes by the sine of the determined angle. If spikes were unconstrained in tilt, this distribution would be uniform. The angular density decreases from ~50°, indicating that higher tilts are disfavoured. The horizontal red dashed line indicates the angular distribution of noise (spikes which have failed to align), estimated based on the angular density between 140° and 180°. d, Schematic diagram and examples of individual tilted spikes on virions. The schematic indicates the angle that was measured. Five examples of individual tilted spikes are marked on tomographic slices through an intact virion, with their associated angle. Scale bar 50 nm.\nExtended Data Fig. 2 | Morphology of SARS-CoV-2 virions released from infected Calu-3 cells. As in Extended Data Fig.&nbsp;1. a, Histogram of virion diameters. Mean and standard deviation for diameters are 104&nbsp;±&nbsp;13 nm (n&nbsp;=&nbsp;67). b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. c, Central slices through three representative viruses from 67 imaged in one experiment. Virions from Calu-3 cells had a slightly broader diameter distribution than those from Vero E6 cells. Scale bar 50 nm. d, Western blot analysis of SARS-CoV-2 nsp3, S and N in cell lysates and in virus preparations. In released virions, S is present in both cleaved (S2, 73%) and uncleaved forms (S0, 27%).\nExtended Data Fig. 3 | Classification of SARS-CoV-2 spike RBDs. a, Class averages obtained after focused classification on the RBD of the left monomer after symmetry expansion of the unconc1 dataset. Top views and side views are shown for closed, open and weak classes. The region subjected to classification is indicated by a transparent red mask surface in the left hand\npanel. b, Equivalent analysis for a smaller, independent dataset (unconc2). c, Equivalent analysis for a dataset obtained after concentrating virus through a sucrose cushion (conc). Only closed and weak classes were obtained. d–f, Cut-open local resolution maps for structures shown in a–c.\nExtended Data Fig. 4 | Resolution assessment of subtomogram averaging structures. a, Local resolution map for the consensus structure obtained for the prefusion S trimers. b, Local resolution maps for the prefusion S trimer in three different conformations. c, Global resolution assessment by Fourier shell correlation (FSC) at the 0.143 criterion for the four structures shown in a and b, as well as the postfusion S trimer.\nExtended Data Fig. 5 | Single particle cryo-EM image processing workflow. Automatically picked particles (green circles) were subjected to 3D classification. Scale bar 100 nm. Selected 3D classes are indicated by black boxes. RBDs from individual asymmetric units from the S trimer (red dashed circles) were locally classified to sort different conformations of RBD. The\nasymmetric unit subjected to local classification is shown in a top view, the RDB of the green monomer is weak in the right-hand class (red arrowhead). S trimers with all three RBDs in the closed state were further refined with C3 symmetry. S trimers where one RBD had weak density were refined with C1 symmetry. For further details see materials and methods.\nExtended Data Fig. 6 | Single particle Cryo-EM structure validation. a, Cut-open cryo-EM maps obtained using all prefusion S trimers, S trimers with 3 closed RBDs or S trimers with 2 closed and 1 weak RBDs, coloured according to the local resolution. b, FSC curves for the three structures in a, and for the atomic model against the map.\nExtended Data Fig. 7 | Structural comparison of in&nbsp;situ structure with recombinant soluble structure. Structural superposition of S trimer modelled into the structure of the trimer with three closed RBDs\n(green, this study) with the published structure of recombinant, soluble closed trimer (blue, PDB 6VXX). Top and side views are shown. The structures are very similar.\nExtended Data Table 1 | Cryo-ET data acquisition and image processing\n\n\nExtended Data Table 2 | Cryo-EM data collection, refinement and validation statistics\n\n\nReporting Summary\nNature Research wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see our Editorial Policies and the Editorial Policy Checklist .\n\n\nStatistics\nFor all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section.\nn/a Confirmed\nThe exact sample size ( n ) for each experimental group/condition, given as a discrete number and unit of measurement\nA statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly\nThe statistical test(s) used AND whether they are one- or two-sided\nOnly common tests should be described solely by name; describe more complex techniques in the Methods section.\n\n\nA description of all covariates tested\nA description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons\nA full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals)\nFor null hypothesis testing, the test statistic (e.g. F , t , r ) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable.\nFor Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings\nFor hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes\nEstimates of effect sizes (e.g. Cohen's d , Pearson's r ), indicating how they were calculated\nOur web collection on statistics for biologists contains articles on many of the points above.\n\n\nSoftware and code\nPolicy information about availability of computer code\nData collection SerialEM 3.8.0 for operation of electron microscope (available and referenced in the Methods section)\nData analysis CryoET data processing (all commercial or available and referenced in methods): CTFFIND4; IMOD 4.10; NovaCTF 1.0.0, EMAN2.2; Matlab R2016b; TOM (no version number); AV3 (no version number); Python 3.7; RELION 3.1.\nCryoEM data processing (all available and referenced in methods): CTFFIND4.1.13; RELION 3.1.\nModelling/structure refinement/visualization (all available and referenced in methods): Coot 0.9; Namdinator (no version numbering); PHENIX 1.18.2; Chimera 1.13.1; Maxon Cinema 4D version S22.116\nWestern blot quantification: LabImage 1D L340 software version 4.1 (Intas, Göttingen)\nFor manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors and reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Research guidelines for submitting code &amp; software for further information.\n\n\nPolicy information about availability of data\nAll manuscripts must include a data availability statement . This statement should provide the following information, where applicable: - Accession codes, unique identifiers, or web links for publicly available datasets - A list of figures that have associated raw data\n- A description of any restrictions on data availability\nPublished structures for comparison or initial model building were obtained from the protein data bank (PDB) with accession codes 6VXX, 6VYB, 6X2B, 6XRA and 6ZP0. The cryo-EM and cryo-ET structures determined here, and representative tomograms are deposited in the Electron Microscopy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (3 closed RBDs from subtomogram averaging), EMD-11495 (1 open RBD), EMD-11496 (2 open RBDs), EMD-11497 (3 closed RBDs from cryo-EM), and EMD-11498 (2 open and 1 weak RBDs). The associated molecular models are deposited in the PDB under accession codes 6ZWV (3 closed RBDs from cryo-EM).\n\n\nField-specific reporting\nPlease select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection.\nLife sciences Behavioural &amp; social sciences Ecological, evolutionary &amp; environmental sciences\nFor a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf\n\n\nLife sciences study design\n\n\nAll studies must disclose on these points even when the disclosure is negative.\nSample size Sample sizes were determined by available electron microscopy time and the number of virus particles on electron microscopy grids. The sample size is sufficient to obtain a structure at the reported resolution, as assessed by Fourier shell correlation.\nData exclusions For all cryo-ET experiments, tomograms which could not be used due to objects (contamination, ice, grid bars etc) obscuring the virus particle were discarded.\nReplication For cryoET and cryoEM, structures were determined from independent half datasets, which were compared to assess the resolution of the reconstruction, as described in the methods section. Two independent preparations of virus in supernatant were analysed as described in the methods section.\nWestern blots are representative of three independent experiments.\nRandomization For CryoET, division of dataset into two random halves was done based on virus number. For CryoEM, division of datasets into two random halves was done based on standard approach in RELION 3. Other experiments did not involve randomization.\n\n\nBlinding Blinding was not applicable to this study because this type of study does not use group allocation.\n\n\nReporting for specific materials, systems and methods\nWe require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.\nMaterials &amp; experimental systems n/a Involved in the study\nAntibodies\nEukaryotic cell lines\nPalaeontology and archaeology\nAnimals and other organisms\nHuman research participants\nClinical data\nDual use research of concern\nMRI-based neuroimaging\n\n\nAntibodies\nAntibodies used Anti-mouse HRP Sigma A4416 Anti-rabbit HRP Sigma A6154\nBeta actin Sigma A5441\nSpike Abcam Ab252690\n\n\nValidation\nCommercial antibodies validated as per manufacturers website: Beta actin Sigma A5441 Immunoblot on chicken fibroblast cell extracts Spike Abcam Ab252690 Validated by ELISA on free peptide from SARS-CoV-1 Nsp3 abcam ab181620 Validated by western blot on SARS-CoV-1 infected cells Nucleocapsid Sino Biological 40143-MM05 Validated by western blot with corresponding viruses\n\n\n245 190 135 100 80 58 46 32 25 22 17 11 58 46 200 Nucleoprotein Actin Nsp3 Spike Cell lysat e Virions b a c S2-S2 ′ S0 S2 SARS-CoV-2 – + + + kDa\n\n\n3 closed RBDs 1 open RBD 2 open RBDs a c b 90° 90° 90° 3 closed RBDs 1 open RBD 2 open RBDs Postfusion Postfusion Prefusion 0º 30º 60º d 60º, rotation 0º 60º, rotation 60º 90°\n\n\n90° a b 90° 3 closed RBDs 2 closed + 1 weak RBDs b G614 c d c d K854\n\n\nCorresponding author(s): John Briggs Last updated by author(s): Jul 28, 2020\n\n\nMethods n/a Involved in the study ChIP-seq Flow cytometry\n\n\n3 Closed RBDs (EMD-11497, PDB 6ZWV) 2 Closed + 1 Weak RBDs (EMD-11498) Data collection and processing Magnification 81,000 81,000 Voltage (kV) 300 300 Electron exposure (e – /Å 2 ) 50 50 Defocus range (µm) 1.0-3.0 1.0-3.0 Pixel size (Å) 1.061 1.061 Movies (no.) 7,982 7,982 Symmetry imposed C3 C1 Initial particle images (no.) 450,473 450,473 Final particle images (no.) 29,183 25,976 Map resolution (Å) 3.5 4.1 FSC threshold 0.143 0.143 Map resolution range (Å) 3.35-27.16 3.67-45.27 Refinement Initial model used PDB 6ZP0 Model resolution (Å) 3.5 FSC threshold 0.5 Map sharpening B factor (Å 2 ) -50 Model composition Non-hydrogen atoms 24036 Protein residues 2955 Ligands 63 B factors (Å 2 ) Protein 130.57 Ligand 142.74 R.m.s. deviations Bond lengths (Å) 0.009 Bond angles (°) 1.346 Validation MolProbity score 1.62 Clashscore 2.97 Poor rotamers (%) 1.74 Ramachandran plot Favored (%) 94.97 Allowed (%) 5.03 Disallowed (%) 0.00\n\n\nAuthentication Cells were not further authenticated\n'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Structures and distributions of SARS-CoV-2 spike proteins on intact virions\nZunlong Ke 1,7 , Joaquin Oton 1,7 , Kun Qu 1,7 , Mirko Cortese 2 , Vojtech Zila 3 , Lesley McKeane 4 , Takanori Nakane 1 , Jasenko Zivanov 1 , Christopher J. Neufeldt 2 , Berati Cerikan 2 , John M. Lu 1 , Julia Peukes 1 , Xiaoli Xiong 1 , Hans-Georg Kräusslich 3,5 , Sjors H. W. Scheres 1 , Ralf Bartenschlager 2,5,6 &amp; John A. G. Briggs 1 ✉\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virions are surrounded by a lipid bilayer from which spike (S) protein trimers protrude 1 . Heavily glycosylated S trimers bind to&nbsp;the angiotensin-converting enzyme 2 receptor and mediate entry of virions into target cells 2–6 . S exhibits extensive conformational flexibility: it modulates exposure of its receptor-binding site and subsequently undergoes complete structural rearrangement to drive fusion of viral and cellular membranes 2,7,8 . The structures and conformations of soluble, overexpressed, purified S proteins have been studied in detail using cryo-electron microscopy 2,7,9–12 , but the structure and distribution of S on the virion surface remain unknown. Here we applied cryo-electron microscopy and tomography to image intact SARS-CoV-2 virions and determine the high-resolution structure, conformational flexibility and distribution of S trimers in&nbsp;situ on the virion surface. These results reveal the conformations of S on the virion, and provide a basis from which to understand interactions between S and neutralizing antibodies during infection or vaccination.\nThe Betacoronavirus SARS-CoV-2 13,14 is an enveloped virus containing a large nucleoprotein (N)-encapsidated positive-sense RNA genome 15 . Three transmembrane proteins are incorporated into the viral lipid envelope: S and two smaller proteins, membrane protein (M) and envelope protein (E) 1,15 . When imaged by cryo-electron microscopy (cryo-EM), betacoronaviruses appear as approximately spherical par- ticles with variable diameters centred around 100&nbsp;nm, containing a dense viroplasm and bounded by a lipid bilayer from which prominent S trimers protrude 16,17 . S trimers of SARS-CoV-2 bind to its receptor on the surface of target cells—angiotensin-converting enzyme 2 (ACE2)—and mediate subsequent viral uptake and fusion 2–5,7 . In so doing, S under- goes a substantial structural rearrangement from the prefusion form to the postfusion form 8 . The overall architectures of both prefusion and postfusion forms are highly conserved among coronaviruses 8,18,19 . During infection, coronaviruses extensively remodel the internal membrane organization of the cell, generating viral replication orga- nelles in which replication takes place 20–22 . The S, M and E proteins are inserted into membranes of the endoplasmic reticulum and traf- fic to the endoplasmic reticulum–Golgi intermediate compartment (ERGIC). The encapsidated genome buds into the ERGIC to form virions, which are then trafficked to the plasma membrane and released 20–22 . S is primed for membrane fusion by proteolytic cleavage at the S1–S2 site, and subsequently the S2′ site 23 .\nThe prefusion structures of S proteins from coronaviruses, including SARS-CoV-2 S, have been extensively studied using ectopic expres- sion of soluble, secreted forms of S, followed by purification and cryo-EM 2,7,9–11 . In the prefusion form, the receptor-binding domain (RBD)\nsits at the top of a broad, trimeric spike above the fusion core. Three copies of the RBD are surrounded by three copies of the N-terminal domain (NTD) which display some mobility 2,7–9 . In the closed prefu- sion conformation, all three copies of the RBD lie flat on the spike surface, largely occluding the receptor-binding site, whereas in the open prefusion conformation, one or multiple RBDs lift to expose the receptor-binding site 2,7,9,10 . The surface of the trimer is extensively glyco- sylated with 22 potential N-linked glycosylation sites per monomer 2,6,7 . After receptor binding, structural transition of the prefusion form to the postfusion form brings the fusion peptide and the transmembrane domain together at one end of a long, needle-like structure centred around a three-helix bundle 8 . Five N-linked glycans are spaced along the length of the postfusion spike 8 .\nFully understanding how S proteins function and how they interact with the immune system requires knowledge of the structures, con- formations and distributions of S trimers within virions. In this study, we apply cryo-EM methods to study the structure, conformations and distributions of S trimers in&nbsp;situ on the virion surface.\nTo avoid artefacts associated with virus concentration or puri- fication, we aimed to image SARS-CoV-2 virions from the superna- tant of infected cells without further concentrating or purifying the virus. Vero E6 cells were infected with SARS-CoV-2 (isolate Germany/ BavPat1/2020) 24 . At 48&nbsp;h after infection, the&nbsp;supernatant was clari- fied, inactivated by fixation with formaldehyde and stored at −80 °C. Western blot analysis revealed that approximately 45% of the total S protein monomers on virions had been cleaved at the multibasic cleav- age site, resulting in S1 and S2 peptides (Fig.&nbsp;1a). The fixed supernatant\nhttps://doi.org/10.1038/s41586-020-2665-2\nReceived: 2 July 2020\nAccepted: 10 August 2020\nPublished online: 17 August 2020\nCheck for updates\n1 Structural Studies Division, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 2 Department of Infectious Diseases, Molecular Virology, Heidelberg University, Heidelberg, Germany. 3 Department of Infectious Diseases, Virology, Heidelberg University, Heidelberg, Germany. 4 Visual Aids Department, Medical Research Council Laboratory of Molecular Biology, Cambridge, UK. 5 German Center for Infection Research, Heidelberg Partner Site, Heidelberg, Germany. 6 Division of Virus-Associated Carcinogenesis, German Cancer Research Center (DKFZ), Heidelberg, Germany. 7 These authors contributed equally: Zunlong Ke, Joaquin Oton, Kun Qu. ✉ e-mail: jbriggs@mrc-lmb.cam.ac.uk\nwas vitrified by plunge freezing and imaged by cryo-EM. Fixation may help to stabilize some protein conformations by crosslinking, but is not expected to give rise to any new conformations. As expected, given the concentration of virus in cellular supernatants (around 10 7 plaque-forming units per ml), small numbers of individual virions were found scattered around the grid—these were imaged by cryo-electron tomography (cryo-ET) (Fig.&nbsp;1b).\nVirions were approximately spherical, with a diameter to the outside of the lipid bilayer of 91&nbsp;±&nbsp;11&nbsp;nm (n&nbsp;=&nbsp;179) (Extended Data Fig.&nbsp;1a). They contained granular densities corresponding to N, and were studded with S trimers (Fig.&nbsp;1b, c). These features are generally consistent with those of other coronaviruses imaged using cryo-EM 1,16,17 . S trimers pro- truding from the viral surface displayed two distinct morphologies—a minority were extended thin structures reminiscent of the postfusion form, whereas the majority were wider structures reminiscent of the prefusion form. This observation is in contrast to a study showing cryo-EM images of purified SARS-CoV-2 virions inactivated with the nucleic acid modifier β-propiolactone, in which only thin protrusions were seen on the viral surface 25 , but is consistent with in&nbsp;situ observa- tions of virus assembly 21 .\nWe also collected tomograms of SARS-CoV-2 virions produced by infection of Calu-3 cells, a human lung carcinoma cell line that supports virus production with a similar titre to Vero E6 cells. The morphology of the virions and the appearance of the S trimers on the surface were consistent with those for virions produced from Vero E6 cells (Extended Data Fig.&nbsp;2a–c). Western blot analysis showed that approximately 73% of S from Calu-3 supernatant was in the cleaved form (Extended Data Fig.&nbsp;2d).\nIndividual virions contained 24&nbsp;±&nbsp;9 S trimers (Extended Data Fig.&nbsp;1b). This is fewer than previous estimates that assumed a uniform distri- bution of S 21 , because S was not uniformly distributed over the virus surface. A small sub-population of virions contained only few S trim- ers whereas larger virions contained higher numbers of S trimers (Extended Data Fig.&nbsp;1b). We identified 4,104 wide-structure S trimers and 116 thin-structure S trimers from 179 virions and analysed them using subtomogram averaging. The averaged structures, at 7.7 and 22&nbsp;Å resolution, respectively, correspond closely to published structures of purified S trimers in the prefusion and postfusion forms 2,7,8 (Fig.&nbsp;2a).\nOverall, approximately 97% of S trimers were in the prefusion form and 3% were in the postfusion form. Prefusion and postfusion forms were evenly distributed among virions.\nPrefusion S trimers on the virus surface may be predominantly in the closed conformation, with the open conformation induced or stabilized only with ACE2 binding; alternatively, both closed and open prefu- sion trimers may be present in the absence of ACE2. Open or closed conformations may induce different antibodies responses when used as immunogens, and there are ongoing efforts to generate S protein constructs that are stabilized in specific conformations 9,11,12 . To assess whether S trimers are present in open and/or closed conformations, we classified the RBD regions of individual monomers within trimers. We identified three classes of RBD, one with the RBD in the closed position, one with the RBD in the open position, and one in which the RBD was predominantly in the closed position, but showed some weakening of the density, suggesting the presence of more mobile conformations (Extended Data Fig.&nbsp;3). On the basis of our assignment of individual monomers to these classes, we derived structures of fully closed trimers and of trimers in which one RBD was open, representing approximately 31% and 55% of 3,854 prefusion trimers, respectively (Fig.&nbsp;2b, Extended Data Fig.&nbsp;4). We also identified a small number of trimers (around 14% of 3,854 prefusion trimers) in which two RBDs were in the open con- formation (Fig.&nbsp;2b). These observations confirmed that the opening of the RBD observed in recombinant S trimers also takes place on the virus surface, and that artificial S protein constructs stabilized in the closed and open conformations both represent structures present in&nbsp;situ. The receptor-binding site is therefore stochastically exposed in&nbsp;situ and available to interact with ACE2 and with antibodies.\nThe trimers did not all protrude perpendicularly from the viral sur- face. They were tilted by up to 90° towards the membrane, although the frequency of observed tilt angles decreased above 50° (Extended Data Fig.&nbsp;1c, d). We grouped trimers according to their orientation relative to the membrane, and averaged these groups independently. The averaged structures reveal that the membrane-proximal stalk region acts as a hinge with sufficient flexibility to allow tilting in all directions (Fig.&nbsp;2c).\nWe generated models of individual virus particles, with S trim- ers located at the position, orientation and conformation that were\nFig. 1 | Characterization of virus production and images of SARS-CoV-2 virions. a, Western blot analysis of SARS-CoV-2 non-structural protein 3 (nsp3), S and N in lysates of Vero E6 cells and in virus preparations, representative of three experiments. In released virions, S is present in both cleaved and uncleaved forms. The positions of S0, S2 and the S2–S2′ cleavage product are marked. b, Four representative tomographic slices of SARS-CoV-2 virions from\nthe supernatant of infected cells. Virions are approximately spherical, contain granular density corresponding to N-packaged genome, and have S trimers protruding at variable angles from their surfaces. Scale bar, 50 nm. c, Three S trimers from the dataset, shown as projections through the trimer to illustrate variable tilt towards the membrane. Scale bars, 10 nm.\ndetermined by subtomogram averaging (Fig.&nbsp;2d). S trimers appear to be distributed randomly on the viral surface, with no obvious clus- tering or relationship between location, orientation and conforma- tion. There is approximately one trimer per 1,000&nbsp;nm 2 of membrane surface, compared with approximately one per 100&nbsp;nm 2 for influenza A virus 26 . The sparse distribution of S, together with the predominantly closed state, suggest that receptor binding may be less dependent on avidity effects than in the case of pandemic influenza viruses 27 . This is consistent with the higher affinity between S and ACE2 (in the nM range 2,7 ) than between haemagglutinin and sialic acid (mM range 27 ).\nThe low concentration of particles in supernatant makes high-resolution structure determination difficult. We therefore concen- trated the virus by pelleting through a sucrose cushion. Concentrated virions deviated from the spherical morphology (Extended Data Fig.&nbsp;5), but the overall features were preserved. We performed cryo-ET and subtomogram averaging on the particles and observed predominantly\nprefusion S trimers, with occasional postfusion S trimers. Upon classifi- cation of the prefusion S trimers, we were able to identify the RBD only in the closed position; weak RBD density was observed in monomers (Extended Data Fig.&nbsp;3c).\nVirions in the supernatant from infected cells exhibited primarily pre- fusion S trimers in either closed or open prefusion conformations. Viri- ons concentrated through a sucrose cushion also exhibited prefusion conformations, but we did not observe the open conformation. Previ- ous studies have shown that virions inactivated with β-propiolactone rather than formaldehyde are primarily in a postfusion state 25 . S trimers purified from membranes are found only in the closed prefusion and postfusion conformations 8 , whereas the open RBD in soluble S trimers has been found to exhibit a wide range of different conformations 28 . These observations suggest that the open prefusion conformations of the spike protein that we observed before, but not after concentration, are fragile (despite the fixation applied here) and may be affected by the purification procedures.\nFig. 2 | Structural analysis of SARS-CoV-2 S trimers on intact virions. a, Structures of prefusion (left) and postfusion (right) trimers from intact virions, determined by subtomogram averaging. Structures are shown as transparent grey isosurfaces fitted with structures of the closed, prefusion trimer (Protein Data Bank (PDB) 6VXX) and the postfusion trimer (PDB 6XRA). One prefusion monomer is coloured blue (N terminus) to red (C terminus). The NTD is blue, the RBD is cyan. The NTD does not fully occupy the cryo-EM density because some loops are not resolved or built in the template structure (PDB 6VXX). b, Three conformations of the prefusion trimer observed on intact virions: all RBDs in the closed position (left, fitted with PDB 6VXX); one RBD in the open position (centre, fitted with PDB 6VYB); two RBDs in the open position (right, fitted with PDB 6X2B, which lacks modelled glycans). The two-open\nconformation has only been observed in&nbsp;vitro after inserting multiple stabilizing mutations. S monomers with closed RBDs are green, and those with open RBDs are blue. c, Averaging of subsets of trimers grouped according to their orientation relative to the membrane shows flexibility in the stalk region. Examples are shown for pools centred at 0°, 30° and 60° from the perpendicular, and for two rotations of the trimer relative to the tilt direction. d, Three-dimensional models of two individual SARS-CoV-2 virions with a membrane (blue) of the measured radius, and all spike proteins shown in the conformations, positions and orientations determined by subtomogram averaging. Different S conformations are distributed over the virion surface and can be tilted by up to approximately 90° relative to the membrane (Extended Data Fig.&nbsp;1c, d).\nOur data indicate that inactivation and purification methods can alter the ratio of prefusion and postfusion forms, as well as the ratio of open and closed conformations. It has been suggested that sub- stantial amounts of postfusion S on the virus surface may protect the virus by shielding the prefusion form, or may shift the host response towards non-neutralizing antibodies 8 . Our observation that intact virions harbour only a small fraction of postfusion spikes suggest that this is unlikely to be an important defence mechanism for the virus during infection, but it may be an important consideration for vaccina- tion. Candidate vaccines based on inactivated virus particles are under development. These may present different S protein epitopes to the immune system depending on how they are prepared, and may there- fore differ in their ability to induce a neutralizing response. For exam- ple, β-propiolactone is frequently used in vaccine production (such as for influenza virus subunit vaccines 29 ), but if postfusion S induces\nnon-neutralizing responses, β-propiolactone may not be optimal for inactivation during SARS-CoV-2 S vaccine formulation.\nWe next imaged the concentrated virus in 2D using cryo-EM and per- formed single-particle analysis on prefusion S trimers that protruded from the sides of the virus particles, generating a consensus structure of the prefusion trimer at 3.4&nbsp;Å resolution. Focused classification with par- tial signal subtraction on individual RBD monomers led to two classes (Extended Data Fig.&nbsp;5). Consistent with the absence of open conforma- tions in this sample in cryo-ET images, 81% of the monomers showed the RBD in the closed conformation and 19% showed weaker density for the RBD, which remained predominantly in the closed position. We refined the structures of S trimers in which all three RBDs are in the closed conformation (53% of the data), and those in which at least one RBD has weak density (47% of the data), to resolutions of 3.5&nbsp;Å and 4.1&nbsp;Å, respectively (Fig.&nbsp;3a, Extended Data Fig.&nbsp;6). The two structures are highly similar, differing only in the density levels for one RBD. We used the structure with three closed RBDs to build and refine an atomic model of the S protein trimer in&nbsp;situ on the viral surface.\nThe positions of glycans on the surface of S are well resolved in our structure, with density at 17 of the 22 predicted N-glycosylation sites (Fig.&nbsp;3b). The other five glycosylation sites are in disordered NTD loops or in the stalk region and are not resolved at high resolution. At the base of the trimer, a clear ring of glycans forms a collar above the stalk region. The density for the stalk region is extended by two helical turns compared with published structures of soluble ectodomain, and fur- ther at lower resolution before fading owing to stalk flexibility (Fig.&nbsp;3c). SARS-CoV-2 S trimers are sparsely distributed and can be highly tilted towards the membrane. This implies that epitopes at the base of the head domain and in the stalk region would be accessible to antibodies where they are not protected by the extensive glycan shell.\nWe compared the in&nbsp;situ structure of the S trimer with structures previously obtained using exogenously expressed purified protein. A recent study of full-length trimeric S solubilized in detergent micelles 8 identified two features that are not seen in most structures of soluble S ectodomain trimers: well-defined density for residues 14–26 of S, and a folded loop between residues 833 and 853. This loop is folded in the structure of the ‘locked’ conformation of the ectodomain 9 , and may become folded in the low-pH conditions in the endosome 30 . We observed only weak density for residues 14–26, and&nbsp;no folded structure for the 833–853 region. The SARS-CoV-2 strain that we have imaged contains the widely circulating D614G substitution 31 , which abolishes a salt bridge to K854 (Fig.&nbsp;3d) and may reduce folding of the 833–854 loop 9,30 . We did not observe additional density that would correspond to bound lipids, as described recently 10 , or other bound co-factors. These may be present sub-stoichiometrically or in rare conformations, but are not a general feature of the S trimer in&nbsp;situ. Overall, our structure is very similar to that of the soluble trimeric ectodomain in the closed prefusion form stabilized by a double proline mutation 2,7 (Extended Data Fig.&nbsp;7). This validates the ongoing use of recombinant, purified S trimers for research, diagnostics and vaccination—they indeed represent the in&nbsp;situ structure of S. In this study, we have determined the structure of S trimers on the virion surface to 3.4&nbsp;Å resolution using cryo-EM (Fig.&nbsp;3), supporting the further use of this technique to study antibody binding to S in the context of the viral surface. Such studies could provide insights into how neutralizing antibodies block virus infection, particularly for antibodies against membrane-proximal regions of S, and thus can inform the design of immunogens for vaccination.\nOnline content\nAny methods, additional references, Nature Research reporting sum- maries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author con- tributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41586-020-2665-2.\nFig. 3 | Structures of SARS-CoV-2 S trimers on intact virions by single-particle reconstruction. a, Top and side views of trimers with three closed RBDs (left, 3.5&nbsp;Å resolution) and one RBD with weaker density (right, 4.1&nbsp;Å resolution). Individual monomers are coloured white, blue and green. Comparison of the left and right structures shows the weaker density for the RBD of the green monomer in the region indicated by the red arrowheads. b, Glycosylation profile of the S protein. Colour scheme as in a; glycans are shown in orange. Boxes indicates the regions shown in c and d. c, Enlarged view of the base of the trimer at lower isosurface threshold to highlight the glycan ring and the extended C-terminal density. d, Enlarged view of the region of the spike where the D614G variation abolishes a salt bridge to K854.\n1. Neuman, B. W. &amp; Buchmeier, M. J. Supramolecular architecture of the coronavirus particle. Adv. Virus Res. 96 , 1–27 (2016). 2. Walls, A. C. et&nbsp;al. Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein. Cell 181 , 281–292 (2020).\n3.  Hoffmann, M. et&nbsp;al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. Cell 181 , 271–280 (2020).\n4.  Shang, J. et&nbsp;al. Structural basis of receptor recognition by SARS-CoV-2. Nature 581 , 221–224 (2020).\n5. Wang, Q. et&nbsp;al. Structural and functional basis of SARS-CoV-2 entry by using human ACE2. Cell 181 , 894–904 (2020).\n6.  Watanabe, Y., Allen, J. D., Wrapp, D., McLellan, J. S. &amp; Crispin, M. Site-specific glycan analysis of the SARS-CoV-2 spike. Science 369 , 330–333 (2020). 7. Wrapp, D. et&nbsp;al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367 , 1260–1263 (2020). 8. Cai, Y. et&nbsp;al. Distinct conformational states of SARS-CoV-2 spike protein. Science 369 , 1586–1592 (2020).\n9. Xiong, X. et&nbsp;al. A thermostable, closed SARS-CoV-2 spike protein trimer. Nat. Struct. Mol. Biol. 27 , 934–941 (2020).\n10.  Toelzer, C. et&nbsp;al. Free fatty acid binding pocket in the locked structure of SARS-CoV-2 spike protein. Science eabd3255 (2020).\n11.  Henderson, R. et&nbsp;al. Controlling the SARS-CoV-2 spike glycoprotein conformation. Nat. Struct. Mol. Biol. 27 , 925–933&nbsp;(2020).\n12.  McCallum, M., Walls, A. C., Bowen, J. E., Corti, D. &amp; Veesler, D. Structure-guided covalent stabilization of coronavirus spike glycoprotein trimers in the closed conformation. Nat. Struct. Mol. Biol. 27 , 942–949&nbsp;(2020).\n13.  Zhu, N. et&nbsp;al. A novel coronavirus from patients with pneumonia in China, 2019. N. Engl. J. Med. 382 , 727–733 (2020).\n14.  Zhou, P. et&nbsp;al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579 , 270–273 (2020).\n15.  Masters, P. S. &amp; Perlman, S. in Fields Virology 6th edn (eds Knipe, D. M. &amp; Howley, P. M.) 826–858 (Elsevier, 2013).\n16.  Neuman, B. W. et&nbsp;al. Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy. J. Virol. 80 , 7918–7928 (2006). 17.  Bárcena, M. et&nbsp;al. Cryo-electron tomography of mouse hepatitis virus: insights into the structure of the coronavirion. Proc. Natl Acad. Sci. USA 106 , 582–587 (2009).\n18.  Walls, A. C. et&nbsp;al. Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion. Proc. Natl Acad. Sci. USA 114 , 11157–11162 (2017).\n19.  Yuan, Y. et&nbsp;al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat. Commun. 8 , 15092 (2017). 20.  Fehr, A. R. &amp; Perlman, S. Coronaviruses: an overview of their replication and pathogenesis. Methods Mol. Biol. 1282 , 1–23 (2015).\n21.  Klein, S. et&nbsp;al. SARS-CoV-2 structure and replication characterized by in&nbsp;situ cryo-electron tomography. Preprint at https://doi.org/10.1101/2020.06.23.167064 (2020).\n22.  Snijder, E. J. et&nbsp;al. A unifying structural and functional model of the coronavirus replication organelle: tracking down RNA synthesis. PLoS Biol. 18 , e3000715 (2020). 23.  Hoffmann, M., Hofmann-Winkler, H. &amp; Pöhlmann, S. in Activation of Viruses by Host Proteases (eds Böttcher-Friebertshäuser, E. et&nbsp;al.) 71–98 (2018).\n24.  Rothe, C. et&nbsp;al. Transmission of 2019-nCoV infection from an asymptomatic contact in Germany. N. Engl. J. Med. 382 , 970–971 (2020).\n25.  Liu, C. et&nbsp;al. Viral architecture of SARS-CoV-2 with post-fusion spike revealed by cryo-EM. Preprint at https://doi.org/10.1101/2020.03.02.972927 (2020).\n26.  Chlanda, P. et&nbsp;al. The hemifusion structure induced by influenza virus haemagglutinin is determined by physical properties of the target membranes. Nat. Microbiol. 1 , 16050 (2016).\n27.  Xiong, X. et&nbsp;al. Receptor binding by a ferret-transmissible H5 avian influenza virus. Nature 497 , 392–396 (2013).\n28.  Gui, M. et&nbsp;al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res. 27 , 119–129 (2017).\n29.  Bonnafous, P. et&nbsp;al. Treatment of influenza virus with beta-propiolactone alters viral membrane fusion. Biochim. Biophys. Acta 1838 , 355–363 (2014).\n30.  Zhou, T. et&nbsp;al. A pH-dependent switch mediates conformational masking of SARS-CoV-2 spike. Preprint at https://doi.org/10.1101/2020.07.04.187989 (2020).\n31.  Korber, B. et&nbsp;al. Tracking changes in SARS-CoV-2 spike: evidence that D614G increases infectivity of the COVID-19 virus. Cell 182 , 812–827 (2020).\nPublisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n© The Author(s), under exclusive licence to Springer Nature Limited 2020\n\n\nMethods\nNo statistical methods were used to predetermine sample size. For Cryo-ET, division of dataset into two random halves was done based on virus number. For Cryo-EM, division of datasets into two random halves was done based on the standard approach in RELION 3. Other experiments did not involve randomization. The investigators were not blinded to allocation during experiments and outcome assessment.\n\n\nCells and virus\nVero E6 cells were obtained from ATCC and were cultured in Dulbecco’s modified Eagle medium (DMEM, Life Technologies) containing 10% fetal bovine serum, 100 U&nbsp;ml −1 penicillin, 100 μg&nbsp;ml −1 streptomycin and 1% non-essential amino acids (complete medium). The Germany/Bav- Pat1/2020 SARS-CoV-2 strain was isolated by C. Drosten (Charité, Berlin, distributed by the European Virology Archive (Ref-SKU: 026V-03883)) at passage (P)2. A stock of SARS-CoV-2 was obtained by passaging the virus once in Vero E6 cells (P3). To produce SARS-CoV-2 virions, Vero E6 cells grown on 75 cm 2 side-bottom tissue culture flasks were infected with SARS-CoV-2 (P3) at a multiplicity of infection (MOI) of 0.5. Culture medium from infected cells was collected at 48 h after infection, clari- fied by centrifugation at 1,000g for 10 min, cleared through a 0.45-μm nitrocellulose filter and fixed with 4% formaldehyde for 30 min at room temperature. Culture medium was supplemented with 10 mM HEPES (pH 7.2) before fixation. Virus-containing medium was subsequently split into aliquots and stored at −80 °C. Infectious supernatants con- taining SARS-CoV-2 virions were obtained from Calu-3 cells infected with P3 virus at an MOI of 5 for 48 h and processed as described above. To obtain SARS-CoV-2 virions at high concentration, infection and collection of Vero E6 culture medium were performed as above, fol- lowed by concentration of fixation-inactivated virions from medium by ultracentrifugation through a 20% (wt/wt) sucrose cushion (120 min at 27,000 rpm in a Beckman SW32 rotor; Beckman Coulter Life Sciences). Pelleted particles were resuspended in PBS and stored in aliquots at −80 °C.\n\n\nWestern blot\nVero E6 or Calu-3 cells mock infected or infected for 48 h with SARS-CoV-2 (MOI&nbsp;=&nbsp;5) were washed twice in PBS, scraped, pelleted at 700g for 5 min and lysed in PBS pH 7.4, containing 1% Triton&nbsp;X-100 and protease inhibitors (Merck) for 30 min at 4 °C. Samples were centri- fuged at 4 °C for 30 min and supernatants were collected. Total pro- tein concentration was calculated using the Bio-Rad Protein Assay kit (Bio-Rad). Purified viruses were prepared for western blot by cen- trifugation of 32 ml of virus containing supernatants on a 10% sucrose cushion in a Beckmann J25 centrifuge. Centrifugation was performed at 10,000g for 4 h at 4 °C. Supernatants were discarded and purified virus pellet were resuspended in 500 μl PBS. For western blotting, 10 μg of total cell lysates and 5 μl purified viruses were diluted in Laemmli buffer and loaded on a pre-cast Criterion XT 4–12% gradient gel (Bio-Rad). Gels were transfered to PVDF membrane using a wet-electroblotting chamber system (Bio-Rad) in Towbin buffer containing 10% methanol. Transfer was performed overnight at 4 °C. Membranes were washed in PBS and blocked with 10% milk in PBS containing 0.2% tween-20 (PBS-T) for 1 h. Membranes were incubated for 1 h at room temperature with primary antibodies specific for an epitope in the C-terminal region of S (Abcam, ab252690; diluted 1:1,000 in PBS-T), for N protein (Sino Biological, 40143-MM05; diluted 1:1,000 in PBS-T) or nsp3 (Abcam, ab181620; diluted 1:500 in PBS-T). Next, the membranes were washed 3 times in PBS-T, incubated with horseradish peroxidase-conjugated anti-mouse or anti-rabbit antibodies at 1:10,000 dilution for 1 h, washed again 3 times in PBS-T, incubated with western Lightning Plus-ECL rea- gent (Perkin Elmer) and imaged using an Intas ChemoCam Imager 3.2 (Intas). Densitometric analysis of western blot assays was performed using LabImage 1D L340 software v.4.1 (Intas). The blot shown is\nrepresentative of three independent experiments. Uncropped, unpro- cessed blots are presented in the Source Data.\n\n\nPCR with reverse transcription and spike sequencing\nTotal RNA was isolated from infected Vero E6 cells 48 h after infec- tion with Germany/BavPat1/2020 SARS-CoV-2 (P2). Spike cDNA was produced from the total RNA using Superscript III (ThermoFisher) with specific reverse transcription primer (CAATTGTGAAGATTCTC ATA). The cDNA was amplified by PCR using specific primers (Fwrd, ATGTTTGTTTTTCTTGTTTTATT; rev, TTATGTGTAATGTAATTTGA) and the resulting amplicon was sent for Sanger sequencing. Sequences were compared to the Germany/BatPat1/2020 SARS-CoV-2 reference sequence (Ref-SKU: 026V-03883) and found to be identical. Specific sequencing primers: Fwrd1, ATGTTTGTTTTTCTTGTTTTATT; Fwrd2, GGTTGGACAGCTGGTGCT; Fwrd3, CCAACCATACAGAGTAGTAGTA; Rev1, GTAGCAGCAAGATTAGCAGAA; and Rev2, TTATGTGTAATGT AATTTGA.\n\n\nCryo-ET sample preparation\nFixed virus samples from the supernatant of infected cells without any concentration step (unconc) or concentrated by pelleting through a sucrose cushion (conc) were prepared, imaged, and processed in parallel. The virus suspension was mixed with 10-nm colloidal gold (in PBS solution) in 10:1 ratio. Then, 3 μl of the solution was added to a glow-discharged copper grid (C-Flat 2/2, Protochips). Grids were plunge frozen into liquid ethane by back-side blotting using a LeicaGP cryo plunger (Leica) and stored in liquid nitrogen until imaging.\n\n\nCryo-ET data collection\nCryo-ET data collection was performed essentially as described previ- ously 32 . Cryo-grids were loaded into an FEI Titan Krios transmission electron microscope operated at 300 kV and images were recorded on a Gatan K2 Summit direct detection camera in counting mode with a 20 eV energy slit in zero-loss mode. Tomographic tilt series between −60° and +60° were collected using SerialEM 3.8.0 software 33 in a dose-symmetric scheme 34 with a 3° angular increment. A total dose of 120 e − A −2 per tilt series was distributed evenly among 41 tilt images. The nominal mag- nification was 81,000×, giving a pixel size of 1.532 Å on the specimen. The defocus range was between −2 μm and −6 μm and 10 frames were saved for each tilt angle. All data acquisition parameters are listed in Extended Data Table&nbsp;1.\nFrames were motion-corrected in IMOD 4.10.30 35 and images were dose-filtered using the alignframes function in IMOD. Exposure fil- tering was implemented according to the cumulative dose per tilt as described elsewhere 36 . The contrast transfer function (CTF) was meas- ured using non-dose-filtered images using the ctfplotter package within IMOD 37 . Tilt series stacks were sorted using IMOD newstack function and fiducial-alignment of all tilt series was performed in IMOD/etomo. Tomograms with less than three trackable gold fiducials were discarded. Motion-corrected and dose-filtered tilt stacks were CTF-corrected by CTF multiplication and tomograms were reconstructed by weighted back-projection in novaCTF 38 . Tomograms were low-pass filtered to 50 Å for better visualization in EMAN2.2 39 and tomographic slices were visualized with IMOD.\n\n\nExtraction of S trimers from tomograms\nThe initial steps of subtomogram alignment and averaging were imple- mented using MATLAB (MathWorks) scripts, which were derived from the TOM 40 and AV3 41 packages as described previously 32 . The missing wedge was modelled as the summed amplitude spectrum of back- ground subtomograms for each tomogram, and was applied during alignment and averaging.\nTo generate an initial template model of the spike protein from the viral surface, 68 spikes were manually picked from four virions of tomograms that were down-scaled by 4× binning of the voxels.\nThe 68 spikes’ initial Euler angles (2 out of 3) were determined on the basis of the vector between two points, one on the head of the spike and one on the membrane where the spike anchors, respectively. The 68 spikes were iteratively aligned to one another for four iterations, applying three-fold symmetry to generate a low-resolution template that resembled a prefusion conformation of the spike. This template was used as an alignment reference for all virions (below). All postfu- sion spikes were manually identified and picked, and initial Euler angles were assigned in the same manner.\nThe centre of each virion was then marked manually using the Vol- ume Tracer function in UCSF Chimera (v.1.13.1) 42 and the radius of the virion was determined centred at the membrane using the Pick Particle Chimera Plugin 43 . An oversampled spherical grid of points was gener- ated on the virion surface with ~9 nm spacing, and subtomograms were extracted for all grid points with a box size of 96 pixels (approximately 60 nm) centred at a radius 14 nm above these grid positions (approxi- mately the radius of the expected centre of the spike). Initial Euler angles were assigned to each subtomogram based on the orientation of the normal vectors relative to the sphere surface.\nSubtomograms were aligned against the low-resolution template (from the above average of 68 spikes). During this alignment, subto- mogram positions converged onto clusters at the true spike positions. One subtomogram position was kept for each cluster, excluding parti- cles within a distance of ~60 Å (10 pixels at 4× binning) and removing particles with cross-correlation coefficients below 0.11. Subtomo- grams that had tilted by more than 90° relative to their perpendicular positions were excluded. Visual inspection of the tomograms using the Place Object Chimera Plugin 43 confirmed that subtomograms selected in this manner corresponded to S trimers on the viral surface (see below). Subtomograms were divided into two halves on the basis of virion number. From this point on the two halves were processed independently.\n\n\nSubtomogram averaging\nSubsequent processing was performed in RELION 44 . For this purpose, subtomograms were reconstructed from the original tilt series images after motion correction using relion_reconstruct. Using dedicated Python scripts, the S trimer positions in the 3D tomograms from the procedure outlined above were converted into 2D positions and defocus values in the corresponding tilt series images, as well as Euler angles in the RELION convention. Individual sub-tomograms were reconstructed at a 2× downscaled pixel size of 3.064 Å, by 3D insertion of Fourier slices of the cropped regions of the tilt series images, each multiplied by their corresponding CTF, which included the dose filter. Similarly, 3D-CTF volumes were generated by 3D insertion of the cor- responding CTF 2 slices. Subtomograms were reconstructed in a box size of 128 voxels for prefusion trimers and a box size of 192 voxels for averages of tilted prefusion trimers and for postfusion trimers.\nTo deal with the CTF pre-multiplied sub-tomograms, as well as the multiplicity of each 3D voxel in the tilt series, a modified version of RELION was used for subtomogram refinement and classification (details to be described elsewhere). Standard 3D auto-refinement was performed with C3 symmetry and a soft-edged mask around the trimers, using a 30 Å lowpass-filtered map as an initial reference. Using 3,854 of the 4,104 subtomograms (excluding those that had tilted by more than 90° relative to the perpendicular position (Extended Data Fig.&nbsp;1c)), a 7.7 Å consensus map was calculated for the prefusion trim- ers; 116 postfusion trimers led to a 22 Å map.\nAngular density (Extended Data Fig.&nbsp;1c) was illustrated by dividing the number of spikes per histogram bin by the sine of the tilt angle. From this histogram, the noise level in the distribution due to incorrectly aligned spikes was estimated assuming they are uniformly distrib- uted (red dashed line in Extended Data Fig.&nbsp;1c), and based on averaging over the range 140–180°. Such angles must be incorrectly alignments because they orient the spike towards the inside of the virus.\nBased on the subtomograms that contributed to the consensus map we estimated that there are 23&nbsp;±&nbsp;9 prefusion S trimers per virion. For a subset of 28 virions, we visually compared the trimers identified by subtomogram averaging with the tomograms to identify false positives or false negatives, finding that the number of S trimers identified by subtomogram averaging is an underestimate of the number of observed S trimers by 0.9 trimers per virion. On this basis we estimate there are 24&nbsp;±&nbsp;9 prefusion S trimers per virion.\nNext, we performed symmetry expansion 45 , followed by focused classification without alignment and with partial signal subtraction, while keeping the orientations from the consensus refinement fixed. The mask used for focused classification was generated manually and enclosed the RBD of one monomer (Extended Data Fig.&nbsp;3a), including the closest NTD of the neighbouring monomer. Classification of the primary dataset (unconc1) led to three different RBD states: closed (45% of the monomers); open (28%) and with weak density (27%) (Extended Data Fig.&nbsp;3a). Classification of a second, smaller dataset of virus parti- cles in supernatant (unconc2), comprising 1,224 trimers yielded 39% closed RBD, 22% open RBD and 39% weak RBD, in general agreement with the unconc1 results (Extended Data Fig.&nbsp;3b). Classification of a dataset of virus particles after concentration through a sucrose cush- ion (conc), comprising 3,788 trimers, yielded 67% of the monomers corresponding to closed RBDs and 33% to RBDs with weak density. For this dataset, no RBDs in the open conformation were identified (Extended Data Fig.&nbsp;3c).\nThe classification of the RBDs as closed, open or with weaker density was used to divide the unconc1 dataset into three subsets: 31% (1,175 trimers) have no open RBDs; 55% (2,121 trimers) have one RBD in the open state; and 14% (525 trimers) have two open RBDs. The remaining 33 trimers have three RBDs in the open state and not further processed. In the class with weaker density, the RBD appears to be predominantly in the closed state and was treated as closed for this assignment. For each of the three subsets, reconstruction of the two independently refined half sets was performed using the orientations from the consensus refinement that gave the 7.7 Å consensus map described above. Subse- quent standard post-processing procedures for resolution estimation, map sharpening and local-resolution filtering in RELION led to three final maps. The subset with no open RBDs gave a reconstruction with C3 symmetry in which all three RBDs were closed at 8.6 Å resolution. The other two subsets yielded structures with C1 symmetry and either one or two open RBDs, with resolutions of 8.6 Å and 9.9 Å, respectively (Extended Data Figs.&nbsp;3, 4). Note, that while fixation is not expected to give rise to any new conformations, we cannot rule out that it dif- ferentially stabilizes different conformations, and the position of the equilibrium between open and closed conformations in unfixed sam- ples may differ from those estimated from fixed samples.\nAverages of tilted prefusion trimers were generated by grouping according to the tilt and rotation of the subtomogram away from the normal vector to the membrane. Subtomograms were included in a group for averaging if they were within 15° of the displayed tilt (0°, 30° and 60°) and rotation (0°, 60°). The pools illustrated in Fig.&nbsp;2c contain 14.5, 14.1, 12.0 and 10.0% of the subtomograms.\n\n\nCryo-EM sample preparation and data collection\nVirus solution concentrated through a 20% (wt/wt) sucrose cushion was plunge-frozen on C-Flat 2/2 3C grids (Protochips) following the same procedure as for cryo-ET, but without adding gold fiducials. Grids were imaged on a Thermo Fisher Scientific Titan Krios transmission electron microscope that was operated at 300 kV, using a Gatan K3 direct electron detector and a Gatan BioQuantum energy filter with a 20 eV energy slit. Movies with 48 frames and an accumulated dose of 50 e − Å −2 were acquired in counting mode using SerialEM-3.8.0 33 at a nominal magnification of 81,000×, corresponding to a calibrated pixel size of 1.061 Å&nbsp;per&nbsp;pixel. Detailed data acquisition parameters are summarized in Extended Data Table&nbsp;2.\n\n\nCryo-EM image processing\nThe scheduler functionality in RELION-3.1 was used for fully automated real-time processing during data collection 9,46 . Mov- ies were motion-corrected and dose-weighted using the RELION implementation of the MotionCor2 algorithm 47 . Subsequently, non-dose-weighted sums were used to estimate the CTF in CTFFIND-4.1.13 48 . S trimers that were extending from the sides of virus particles were picked manually (4,493 particles from the first 100 micrographs) and then used as a training set for optimisation of the convolutional neural network in the automated particle picking software Topaz 49 . Extracted particles were subjected to 3D classification using a previously determined structure of the S trimer 9 , low-pass-filtered to 30 Å, as initial 3D reference. The selected 286,407 particles that contributed to 3D classes corresponding to S trimers were submitted to Bayesian polishing to correct for per-particle beam-induced motions and a second round of 3D clas- sification to select the 55,159 particles that contributed to the best class. This final consensus set of particles was subjected to CTF refinement of per-particle defocus, per-micrograph astigmatism and beam tilt, followed by a second round of Bayesian polishing. 3D auto-refinements were performed with the selected particles after each round of 3D classification, CTF refinement or Bayesian polishing. The consensus structure had a resolution of 3.4 Å.\nSubsequently, symmetry expansion 45 was performed, followed by focused 3D classification with partial signal subtraction 50 . Using a mask on a single RBD to focus classification into six classes while keeping the orientations of the last consensus refinement fixed resulted in the identification of two RBD states: closed or with weak density. S trimers with all three RBDs in the closed state were refined separately from S trimers with one RBD with weak density, resulting in two final maps with resolutions. Standard RELION post-processing was used for resolution estimation, map sharpening and local-resolution filtering. The C3 symmetric map with three closed RBDs had an estimated overall resolution of 3.5 Å; the C1 map with one weaker RBD extended to 4.1 Å resolution (Extended Data Fig.&nbsp;6b).\n\n\nModel building and refinement\nThe SARS-CoV-2 S trimer structure 9 (PDB 6ZP0) was used as an initial model for building into the model with three closed RBDs (Extended Data Fig.&nbsp;6a, middle). Residues were adjusted manually in Coot 0.9 51 . Steric clash and sidechain rotamer conformations were improved using the Namdinator web server 52 . After further iterations of manual adjustment, the structure was refined in PHENIX-1.18.2 53 . The geom- etry and statistics are given in Extended Data Table&nbsp;2. The unmasked model-to-map FSC was calculated in PHENIX for the refined model against the full reconstruction.\n\n\n3D model of spikes on authentic virions\nTo visualize the spike protein on the authentic SARS-CoV-2 virions, the coordinates, orientations and conformational classes determined by subtomogram averaging were converted into a format compat- ible with Maxon Cinema 4D (v.S22.116), and imported together with the 3D models of the different conformational states determined by subtomogram averaging. To generate representative virion images in Fig.&nbsp;2d we removed the three false positives from one virion and positioned the three false negatives on each virion that had been identified by visual inspection of the data (see above). The HR2 region was modelled as a cylinder. Images of individual virions from the dataset were rendered into Adobe Photoshop to generate images for presentation.\n\n\nReporting summary\nFurther information on research design is available in the&nbsp;Nature Research Reporting Summary linked to this paper.\n\n\nData availability\nPublished structures for comparison or initial model building were obtained from the PDB with accession codes 6VXX 2 , 6VYB 2 , 6X2B 11 , 6XRA 8 and 6ZP0 9 . The cryo-EM and cryo-ET structures determined here and representative tomograms are deposited in the Electron Micros- copy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (three closed RBDs from subtomo- gram averaging), EMD-11495 (one open RBD), EMD-11496 (two open RBDs), EMD-11497 (three closed RBDs from cryo-EM), and EMD-11498 (two open RBDs and one weak RBD). The associated molecular models are deposited in the PDB under accession code 6ZWV (three closed RBDs from cryo-EM). The raw micrographs are deposited in the Electron Microscopy Public Image Archive under accession codes EMPIAR-10492 (cryo-EM) and EMPIAR-10493 (cryo-ET).\n32.  Wan, W. et&nbsp;al. Structure and assembly of the Ebola virus nucleocapsid. Nature 551 , 394–397 (2017).\n33.  Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. J. Struct. Biol. 152 , 36–51 (2005).\n34.  Hagen, W. J. H., Wan, W. &amp; Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197 , 191–198 (2017).\n35.  Kremer, J. R., Mastronarde, D. N. &amp; McIntosh, J. R. Computer visualization of three-dimensional image data using IMOD. J. Struct. Biol. 116 , 71–76 (1996).\n36.  Grant, T. &amp; Grigorieff, N. Measuring the optimal exposure for single particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6. eLife 4 , e06980 (2015).\n37.  Xiong, Q., Morphew, M. K., Schwartz, C. L., Hoenger, A. H. &amp; Mastronarde, D. N. CTF determination and correction for low dose tomographic tilt series. J. Struct. Biol. 168 , 378–387 (2009).\n38.  Turoňová, B., Schur, F. K. M., Wan, W. &amp; Briggs, J. A. G. Efficient 3D-CTF correction for cryo-electron tomography using NovaCTF improves subtomogram averaging resolution to 3.4 Å. J. Struct. Biol. 199 , 187–195 (2017).\n39.  Galaz-Montoya, J. G., Flanagan, J., Schmid, M. F. &amp; Ludtke, S. J. Single particle tomography in EMAN2. J. Struct. Biol. 190 , 279–290 (2015).\n40.  Nickell, S. et&nbsp;al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149 , 227–234 (2005).\n41.  Förster, F., Medalia, O., Zauberman, N., Baumeister, W. &amp; Fass, D. Retrovirus envelope protein complex structure in&nbsp;situ studied by cryo-electron tomography. Proc. Natl Acad. Sci. USA 102 , 4729–4734 (2005).\n42.  Pettersen, E. F. et&nbsp;al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25 , 1605–1612 (2004).\n43.  Qu, K. et&nbsp;al. Structure and architecture of immature and mature murine leukemia virus capsids. Proc. Natl Acad. Sci. USA 115 , E11751–E11760 (2018).\n44.  Bharat, T. A. M., Russo, C. J., Löwe, J., Passmore, L. A. &amp; Scheres, S. H. W. Advances in single-particle electron cryomicroscopy structure determination applied to sub-tomogram averaging. Structure 23 , 1743–1753 (2015).\n45.  Scheres, S. H. W. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579 , 125–157 (2016).\n46.  Zivanov, J. et&nbsp;al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7 , e42166 (2018).\n47.  Zheng, S. Q. et&nbsp;al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14 , 331–332 (2017).\n48.  Rohou, A. &amp; Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192 , 216–221 (2015).\n49.  Bepler, T. et&nbsp;al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Nat. Methods 16 , 1153–1160 (2019).\n50.  Bai, X. C., Rajendra, E., Yang, G., Shi, Y. &amp; Scheres, S. H. W. Sampling the conformational space of the catalytic subunit of human γ-secretase. eLife 4 , e11182 (2015). 51.  Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. Acta Crystallogr. D 60 , 2126–2132 (2004).\n52.  Kidmose, R. T. et&nbsp;al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6 , 526–531 (2019).\n53.  Afonine, P. V. et&nbsp;al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr. D 74 , 531–544 (2018).\nAcknowledgements We thank the staff of the MRC-LMB for generous support during the COVID-19 pandemic lockdown; all of&nbsp;the staff of the MRC-LMB EM Facility, in particular A. Yeates, G. Sharov and G. Cannone, for supporting the EM experiments; J. Grimmett and T. Darling for supporting scientific computing; C. Drosten and EVAg for provision of the SARS-CoV-2 strain used here; and M. Frey for the Calu-3 cell line. We acknowledge microscopy support from the Infectious Diseases Imaging Platform (IDIP) at the Center for Integrative Infectious Disease Research Heidelberg. This study was supported by funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (ERC-CoG-648432 MEMBRANEFUSION to J.A.G.B.), the Medical Research Council as part of UK Research and Innovation (MC_UP_A025_1013 to S.H.W.S.; MC_UP_1201/16 to J.A.G.B.), the Deutsche Forschungsgemeinschaft (240245660–SFB 1129 project 5 to H.-G.K. and project 11 to R.B.) and the Japan Society for the Promotion of Science (overseas research fellowship to T.N.).\nAuthor contributions Z.K. and J.A.G.B. conceived the study. Z.K., J.O., K.Q., M.C., V.Z., H.-G.K., R.B. and J.A.G.B. designed the study. M.C., V.Z., C.J.N. and B.C. prepared and characterized virus samples with supervision from H.-G.K. and R.B. Z.K. and K.Q. collected cryo-EM data. Z.K., J.O. and K.Q. processed cryo-EM data with assistance from T.N., J.Z., J.M.L. and J.P., supervised by S.H.W.S. and J.A.G.B. Z.K., J.O., K.Q., M.C. and L.M. made figures. Z.K., J.O., K.Q., M.C., X.X. and J.A.G.B. interpreted data. J.A.G.B. prepared the initial draft and managed the project. Z.K., J.O., K.Q., M.C. and J.A.G.B. wrote the manuscript with input from all authors. S.H.W.S., H.-G.K., R.B. and J.A.G.B. obtained funding.\nCompeting interests The authors declare no competing interests.\nAdditional information Supplementary information is available for this paper at https://doi.org/10.1038/s41586-020- 2665-2.\nCorrespondence and requests for materials should be addressed to J.A.G.B. Peer review information Nature thanks Jason McLellan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Reprints and permissions information is available at http://www.nature.com/reprints.\nExtended Data Fig. 1 | Characterization of SARS-CoV-2 virion morphology. a, Histogram of virion diameters for unconcentrated extracellular virions in the supernatant of two independent preparations (top and middle), and for extracellular virions after concentration through a sucrose cushion (bottom). After concentration the virions become less spherical. Mean and standard deviation for diameters are 91&nbsp;±&nbsp;11 nm (n&nbsp;=&nbsp;179), 94&nbsp;±&nbsp;9 nm (n&nbsp;=&nbsp;68) and 92&nbsp;±&nbsp;8 nm (n&nbsp;=&nbsp;227) for the three preparations. b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. Visual inspection indicates that almost all spikes were identified for virions in the supernatant, but that not all spikes are identified in the concentrated preparation leading to an underestimate of the number of spikes. c, Histogram of spike tilt angle towards the membrane for the\nlarger supernatant virus dataset (unconc1). The vertical black dashed line indicates 90°. 97% of particles have tilts below 90°; particles with tilts above 90° were not included in image analysis. The angular density (right) is calculated by dividing the number of spikes by the sine of the determined angle. If spikes were unconstrained in tilt, this distribution would be uniform. The angular density decreases from ~50°, indicating that higher tilts are disfavoured. The horizontal red dashed line indicates the angular distribution of noise (spikes which have failed to align), estimated based on the angular density between 140° and 180°. d, Schematic diagram and examples of individual tilted spikes on virions. The schematic indicates the angle that was measured. Five examples of individual tilted spikes are marked on tomographic slices through an intact virion, with their associated angle. Scale bar 50 nm.\nExtended Data Fig. 2 | Morphology of SARS-CoV-2 virions released from infected Calu-3 cells. As in Extended Data Fig.&nbsp;1. a, Histogram of virion diameters. Mean and standard deviation for diameters are 104&nbsp;±&nbsp;13 nm (n&nbsp;=&nbsp;67). b, Scatter plot of number of spikes identified per virion during subtomogram averaging against virion diameter for the same virions shown in a. c, Central slices through three representative viruses from 67 imaged in one experiment. Virions from Calu-3 cells had a slightly broader diameter distribution than those from Vero E6 cells. Scale bar 50 nm. d, Western blot analysis of SARS-CoV-2 nsp3, S and N in cell lysates and in virus preparations. In released virions, S is present in both cleaved (S2, 73%) and uncleaved forms (S0, 27%).\nExtended Data Fig. 3 | Classification of SARS-CoV-2 spike RBDs. a, Class averages obtained after focused classification on the RBD of the left monomer after symmetry expansion of the unconc1 dataset. Top views and side views are shown for closed, open and weak classes. The region subjected to classification is indicated by a transparent red mask surface in the left hand\npanel. b, Equivalent analysis for a smaller, independent dataset (unconc2). c, Equivalent analysis for a dataset obtained after concentrating virus through a sucrose cushion (conc). Only closed and weak classes were obtained. d–f, Cut-open local resolution maps for structures shown in a–c.\nExtended Data Fig. 4 | Resolution assessment of subtomogram averaging structures. a, Local resolution map for the consensus structure obtained for the prefusion S trimers. b, Local resolution maps for the prefusion S trimer in three different conformations. c, Global resolution assessment by Fourier shell correlation (FSC) at the 0.143 criterion for the four structures shown in a and b, as well as the postfusion S trimer.\nExtended Data Fig. 5 | Single particle cryo-EM image processing workflow. Automatically picked particles (green circles) were subjected to 3D classification. Scale bar 100 nm. Selected 3D classes are indicated by black boxes. RBDs from individual asymmetric units from the S trimer (red dashed circles) were locally classified to sort different conformations of RBD. The\nasymmetric unit subjected to local classification is shown in a top view, the RDB of the green monomer is weak in the right-hand class (red arrowhead). S trimers with all three RBDs in the closed state were further refined with C3 symmetry. S trimers where one RBD had weak density were refined with C1 symmetry. For further details see materials and methods.\nExtended Data Fig. 6 | Single particle Cryo-EM structure validation. a, Cut-open cryo-EM maps obtained using all prefusion S trimers, S trimers with 3 closed RBDs or S trimers with 2 closed and 1 weak RBDs, coloured according to the local resolution. b, FSC curves for the three structures in a, and for the atomic model against the map.\nExtended Data Fig. 7 | Structural comparison of in&nbsp;situ structure with recombinant soluble structure. Structural superposition of S trimer modelled into the structure of the trimer with three closed RBDs\n(green, this study) with the published structure of recombinant, soluble closed trimer (blue, PDB 6VXX). Top and side views are shown. The structures are very similar.\nExtended Data Table 1 | Cryo-ET data acquisition and image processing\n\n\nExtended Data Table 2 | Cryo-EM data collection, refinement and validation statistics\n\n\nReporting Summary\nNature Research wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see our Editorial Policies and the Editorial Policy Checklist .\n\n\nStatistics\nFor all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section.\nn/a Confirmed\nThe exact sample size ( n ) for each experimental group/condition, given as a discrete number and unit of measurement\nA statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly\nThe statistical test(s) used AND whether they are one- or two-sided\nOnly common tests should be described solely by name; describe more complex techniques in the Methods section.\n\n\nA description of all covariates tested\nA description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons\nA full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals)\nFor null hypothesis testing, the test statistic (e.g. F , t , r ) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable.\nFor Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings\nFor hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes\nEstimates of effect sizes (e.g. Cohen's d , Pearson's r ), indicating how they were calculated\nOur web collection on statistics for biologists contains articles on many of the points above.\n\n\nSoftware and code\nPolicy information about availability of computer code\nData collection SerialEM 3.8.0 for operation of electron microscope (available and referenced in the Methods section)\nData analysis CryoET data processing (all commercial or available and referenced in methods): CTFFIND4; IMOD 4.10; NovaCTF 1.0.0, EMAN2.2; Matlab R2016b; TOM (no version number); AV3 (no version number); Python 3.7; RELION 3.1.\nCryoEM data processing (all available and referenced in methods): CTFFIND4.1.13; RELION 3.1.\nModelling/structure refinement/visualization (all available and referenced in methods): Coot 0.9; Namdinator (no version numbering); PHENIX 1.18.2; Chimera 1.13.1; Maxon Cinema 4D version S22.116\nWestern blot quantification: LabImage 1D L340 software version 4.1 (Intas, Göttingen)\nFor manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors and reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Research guidelines for submitting code &amp; software for further information.\n\n\nPolicy information about availability of data\nAll manuscripts must include a data availability statement . This statement should provide the following information, where applicable: - Accession codes, unique identifiers, or web links for publicly available datasets - A list of figures that have associated raw data\n- A description of any restrictions on data availability\nPublished structures for comparison or initial model building were obtained from the protein data bank (PDB) with accession codes 6VXX, 6VYB, 6X2B, 6XRA and 6ZP0. The cryo-EM and cryo-ET structures determined here, and representative tomograms are deposited in the Electron Microscopy Data Bank (EMDB) under accession codes EMD-11493 (prefusion consensus structure), EMD-11494 (3 closed RBDs from subtomogram averaging), EMD-11495 (1 open RBD), EMD-11496 (2 open RBDs), EMD-11497 (3 closed RBDs from cryo-EM), and EMD-11498 (2 open and 1 weak RBDs). The associated molecular models are deposited in the PDB under accession codes 6ZWV (3 closed RBDs from cryo-EM).\n\n\nField-specific reporting\nPlease select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection.\nLife sciences Behavioural &amp; social sciences Ecological, evolutionary &amp; environmental sciences\nFor a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf\n\n\nLife sciences study design\n\n\nAll studies must disclose on these points even when the disclosure is negative.\nSample size Sample sizes were determined by available electron microscopy time and the number of virus particles on electron microscopy grids. The sample size is sufficient to obtain a structure at the reported resolution, as assessed by Fourier shell correlation.\nData exclusions For all cryo-ET experiments, tomograms which could not be used due to objects (contamination, ice, grid bars etc) obscuring the virus particle were discarded.\nReplication For cryoET and cryoEM, structures were determined from independent half datasets, which were compared to assess the resolution of the reconstruction, as described in the methods section. Two independent preparations of virus in supernatant were analysed as described in the methods section.\nWestern blots are representative of three independent experiments.\nRandomization For CryoET, division of dataset into two random halves was done based on virus number. For CryoEM, division of datasets into two random halves was done based on standard approach in RELION 3. Other experiments did not involve randomization.\n\n\nBlinding Blinding was not applicable to this study because this type of study does not use group allocation.\n\n\nReporting for specific materials, systems and methods\nWe require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.\nMaterials &amp; experimental systems n/a Involved in the study\nAntibodies\nEukaryotic cell lines\nPalaeontology and archaeology\nAnimals and other organisms\nHuman research participants\nClinical data\nDual use research of concern\nMRI-based neuroimaging\n\n\nAntibodies\nAntibodies used Anti-mouse HRP Sigma A4416 Anti-rabbit HRP Sigma A6154\nBeta actin Sigma A5441\nSpike Abcam Ab252690\n\n\nValidation\nCommercial antibodies validated as per manufacturers website: Beta actin Sigma A5441 Immunoblot on chicken fibroblast cell extracts Spike Abcam Ab252690 Validated by ELISA on free peptide from SARS-CoV-1 Nsp3 abcam ab181620 Validated by western blot on SARS-CoV-1 infected cells Nucleocapsid Sino Biological 40143-MM05 Validated by western blot with corresponding viruses\n\n\n245 190 135 100 80 58 46 32 25 22 17 11 58 46 200 Nucleoprotein Actin Nsp3 Spike Cell lysat e Virions b a c S2-S2 ′ S0 S2 SARS-CoV-2 – + + + kDa\n\n\n3 closed RBDs 1 open RBD 2 open RBDs a c b 90° 90° 90° 3 closed RBDs 1 open RBD 2 open RBDs Postfusion Postfusion Prefusion 0º 30º 60º d 60º, rotation 0º 60º, rotation 60º 90°\n\n\n90° a b 90° 3 closed RBDs 2 closed + 1 weak RBDs b G614 c d c d K854\n\n\nCorresponding author(s): John Briggs Last updated by author(s): Jul 28, 2020\n\n\nMethods n/a Involved in the study ChIP-seq Flow cytometry\n\n\n3 Closed RBDs (EMD-11497, PDB 6ZWV) 2 Closed + 1 Weak RBDs (EMD-11498) Data collection and processing Magnification 81,000 81,000 Voltage (kV) 300 300 Electron exposure (e – /Å 2 ) 50 50 Defocus range (µm) 1.0-3.0 1.0-3.0 Pixel size (Å) 1.061 1.061 Movies (no.) 7,982 7,982 Symmetry imposed C3 C1 Initial particle images (no.) 450,473 450,473 Final particle images (no.) 29,183 25,976 Map resolution (Å) 3.5 4.1 FSC threshold 0.143 0.143 Map resolution range (Å) 3.35-27.16 3.67-45.27 Refinement Initial model used PDB 6ZP0 Model resolution (Å) 3.5 FSC threshold 0.5 Map sharpening B factor (Å 2 ) -50 Model composition Non-hydrogen atoms 24036 Protein residues 2955 Ligands 63 B factors (Å 2 ) Protein 130.57 Ligand 142.74 R.m.s. deviations Bond lengths (Å) 0.009 Bond angles (°) 1.346 Validation MolProbity score 1.62 Clashscore 2.97 Poor rotamers (%) 1.74 Ramachandran plot Favored (%) 94.97 Allowed (%) 5.03 Disallowed (%) 0.00\n\n\nAuthentication Cells were not further authenticated\n'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [17.39s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"virus\",\n  \"organism_name\": \"Severe acute respiratory syndrome coronavirus 2\",\n  \"cell_strain\": \"Vero E6\",\n  \"sample_preparation\": \"Virus-containing medium was collected at 48 h after infection, clarified by centrifugation, cleared through a 0.45-μm nitrocellulose filter, fixed with 4% formaldehyde, supplemented with 10 mM HEPES (pH 7.2) before fixation, split into aliquots and stored at −80 °C.\",\n  \"grid_preparation\": \"The virus suspension was mixed with 10-nm colloidal gold, added to a glow-discharged copper grid, and plunge frozen into liquid ethane using a LeicaGP cryo plunger.\",\n  \"cryoet_pixel_spacing\": \"1.532\",\n  \"cryoet_acceleration_voltage\": \"300\",\n  \"microscope_name\": \"FEI Titan Krios\",\n  \"microscope_setup_energy_filter\": \"20 eV energy slit in zero-loss mode\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K2 Summit\",\n  \"tilt_minimum\": \"-60\",\n  \"tilt_maximum\": \"+60\",\n  \"total_flux\": \"120 e- A-2\",\n  \"reconstruction_software\": \"IMOD, novaCTF\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"weighted back-projection\",\n  \"annotated_entities\": \"N (nucleoprotein)-encapsidated genome, S (spike) proteins\",\n  \"annotation_methods\": \"manual\",\n  \"annotation_software\": \"UCSF Chimera, Maxon Cinema 4D\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"virus\",\n  \"organism_name\": \"Severe acute respiratory syndrome coronavirus 2\",\n  \"cell_strain\": \"Vero E6\",\n  \"sample_preparation\": \"Virus-containing medium was collected at 48 h after infection, clarified by centrifugation, cleared through a 0.45-μm nitrocellulose filter, fixed with 4% formaldehyde, supplemented with 10 mM HEPES (pH 7.2) before fixation, split into aliquots and stored at −80 °C.\",\n  \"grid_preparation\": \"The virus suspension was mixed with 10-nm colloidal gold, added to a glow-discharged copper grid, and plunge frozen into liquid ethane using a LeicaGP cryo plunger.\",\n  \"cryoet_pixel_spacing\": \"1.532\",\n  \"cryoet_acceleration_voltage\": \"300\",\n  \"microscope_name\": \"FEI Titan Krios\",\n  \"microscope_setup_energy_filter\": \"20 eV energy slit in zero-loss mode\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K2 Summit\",\n  \"tilt_minimum\": \"-60\",\n  \"tilt_maximum\": \"+60\",\n  \"total_flux\": \"120 e- A-2\",\n  \"reconstruction_software\": \"IMOD, novaCTF\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"weighted back-projection\",\n  \"annotated_entities\": \"N (nucleoprotein)-encapsidated genome, S (spike) proteins\",\n  \"annotation_methods\": \"manual\",\n  \"annotation_software\": \"UCSF Chimera, Maxon Cinema 4D\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 399,
      "prompt_tokens": 18060,
      "total_tokens": 18459
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_021f187978"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "biological_sample_type": "virus",
  "organism_name": "Severe acute respiratory syndrome coronavirus 2",
  "cell_strain": "Vero E6",
  "sample_preparation": "Virus-containing medium was collected at 48 h after infection, clarified by centrifugation, cleared through a 0.45-μm nitrocellulose filter, fixed with 4% formaldehyde, supplemented with 10 mM HEPES (pH 7.2) before fixation, split into aliquots and stored at −80 °C.",
  "grid_preparation": "The virus suspension was mixed with 10-nm colloidal gold, added to a glow-discharged copper grid, and plunge frozen into liquid ethane using a LeicaGP cryo plunger.",
  "cryoet_pixel_spacing": "1.532",
  "cryoet_acceleration_voltage": "300",
  "microscope_name": "FEI Titan Krios",
  "microscope_setup_energy_filter": "20 eV energy slit in zero-loss mode",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Gatan",
  "camera_model": "K2 Summit",
  "tilt_minimum": "-60",
  "tilt_maximum": "+60",
  "total_flux": "120 e- A-2",
  "reconstruction_software": "IMOD, novaCTF",
  "tomogram_size": "not present",
  "reconstruction_method": "weighted back-projection",
  "annotated_entities": "N (nucleoprotein)-encapsidated genome, S (spike) proteins",
  "annotation_methods": "manual",
  "annotation_software": "UCSF Chimera, Maxon Cinema 4D"
}
[chain/end] [1:chain:RunnableSequence] [17.39s] Exiting Chain run with output:
{
  "biological_sample_type": "virus",
  "organism_name": "Severe acute respiratory syndrome coronavirus 2",
  "cell_strain": "Vero E6",
  "sample_preparation": "Virus-containing medium was collected at 48 h after infection, clarified by centrifugation, cleared through a 0.45-μm nitrocellulose filter, fixed with 4% formaldehyde, supplemented with 10 mM HEPES (pH 7.2) before fixation, split into aliquots and stored at −80 °C.",
  "grid_preparation": "The virus suspension was mixed with 10-nm colloidal gold, added to a glow-discharged copper grid, and plunge frozen into liquid ethane using a LeicaGP cryo plunger.",
  "cryoet_pixel_spacing": "1.532",
  "cryoet_acceleration_voltage": "300",
  "microscope_name": "FEI Titan Krios",
  "microscope_setup_energy_filter": "20 eV energy slit in zero-loss mode",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Gatan",
  "camera_model": "K2 Summit",
  "tilt_minimum": "-60",
  "tilt_maximum": "+60",
  "total_flux": "120 e- A-2",
  "reconstruction_software": "IMOD, novaCTF",
  "tomogram_size": "not present",
  "reconstruction_method": "weighted back-projection",
  "annotated_entities": "N (nucleoprotein)-encapsidated genome, S (spike) proteins",
  "annotation_methods": "manual",
  "annotation_software": "UCSF Chimera, Maxon Cinema 4D"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Supplementary information\nhttps://doi.org/10.1038/s41586-022-04971-z\nSupplementary Information for\nMembrane-anchored HDCR nanowires drive hydrogen-powered CO 2 fixation Helge M. Dietrich 1† , Ricardo D. Righetto 2,3† , Anuj Kumar 1,4 , Wojciech Wietrzynski 2,3 , Raphael\nTrischler 1 , Sandra K. Schuller 4 , Jonathan Wagner 5 , Fabian M. Schwarz 1 , Benjamin D. Engel 2,3 *, Volker Müller 1 *, Jan M. Schuller 4 *\n1 Molecular Microbiology &amp; Bioenergetics, Institute of Molecular Biosciences,\nJohann Wolfgang Goethe University; Frankfurt am Main, Germany\n2 Helmholtz Pioneer Campus, Helmholtz Munich; Neuherberg, Germany.\n3 Biozentrum, University of Basel; Basel, Switzerland\n4 CryoEM of Molecular Machines, SYNMIKRO Research Center and Department of\nChemistry, Philipps-University, Marburg, Germany\n5 Department of Molecular Structural Biology, Max Planck Institute of Biochemistry;\nMartinsried, Germany\n† Equal contribution\n*Correspondence to: ben.engel@unibas.ch , vmueller@bio.uni-frankfurt.de ,\njan.schuller@synmikro.uni-marburg.de\n\n\nSupplemental Information Table of Contents\nSupplementary Information\nSupplementary Table 1 - Coordination of the [4Fe4S]-clusters in HDCR subunits. p. 6\nSupplementary Table 2 - Conserved W- bis-pterin guanine dinucleotide (W-bisPGD)\ncoordinating residues. p. 7\nSupplementary Table 3 - T. kivui HDCR variants used in this study. p. 8\nSupplementary Table 4 - Plasmids used in this study.\nSupplementary Table 5 - Primers used in this study. p. 10\nSupplementary Figure 1 - Uncropped gel images. p. 11\nFigure legends for Supplementary Videos p. 12\nSupplementary Information:\n\n\nInteraction studies of HDCR subunits\nGenetic complementation of the Δ hdcr strain was used to probe the connectivity of the HDCR complex. When HycB4 was omitted from the overexpression (HDCR_ΔHycB4), only HydA2\nwas found in the eluate after purification with HydA2-His 6 (Fig. 2C). However, addition of a second  His 6 -tag  to  the  HDCR_ΔHycB4  construct  (HydA2-His 6 &amp;  His 6 -FdhF)  led  to\ncopurification of monomeric FdhF and the protein pair HycB3-HydA2, although there was no\nfunctional interaction between FdhF and HycB3-HydA2. This verifies HycB4 as the connecting\nsubunit between HydA2  and the protein  pair of FdhF and HycB3.  When  HycB3 was omitted\nfrom  the  overexpression  (HDCRΔHycB3),  HydA2-His 6 purification  yielded  a  complex containing  HydA2  and  HycB4,  but  not  FdhF.  This  proves  that  HycB3  is  the  subunit  linking\nHycB4 to FdhF, revealing the chain of protein interactions to be FdhF-HycB3-HycB4-HydA2.\nThis  conclusion  is  in  complete  accordance  with  the  cryo-EM  structure  (Fig.  1)  and  is  also\nsupported  by  the  genetic  organization  of  the hdcr operon,  where  the  genes  for  the  enzyme complex are arranged in the same order 1 .\nCatalytic centers operate independently, but HycB3 is vital for FdhF activity\nHydrogenase activity was independent of HycB3 and HycB4. When HydA2 was omitted from\noverexpression  (HDCR_∆HydA2)  or  the  H-cluster  coordinating  Cys387  was  substituted  with\nalanine (HDCR_HydA2 C387A), there was no hydrogenase activity (Extended Data Fig. 7F).\nFormate dehydrogenase activity was marginally affected by removal of HycB4 and HydA2, but\nSupplementary Methods 1: Additional methods for Extended Data Fig. 7.\nFor methylviologen-dependent hydrogenase activity, 1.1 x 10 5 Pa H 2 was used as electron donor.\nFor  methylviologen-dependent  formate  dehydrogenase  assay  and  analysis  of  hydrogen\nproduction from formate, 150 mM formate was used as electron donor. Formate production from\nH 2 + CO 2 was determined with a substrate gas composition of H 2 + CO 2 (80:20 [v:v], 1.1 x 10 5\nPa). The reaction buffer for pH-optima analysis (Extended Data Fig. 7a,b) composed of 50 mM\nMES, 50 mM  MOPS, 50 mM  HEPES, 50 mM  EPPS, 50 mM  CHES, 20 mM  MgSO 4 , 4 µM\nResazurin, 2 mM DTE, pH as indicated. The reaction buffer for Extended Data Fig. 7c-h was\n100 mM HEPES, 20 mM MgSO 4 , 4 µM Resazurin, 2 mM DTE, pH 7.0.\n\n\nSupplementary Discussion\nRelevance of HDCR filamentation to its activity in different organisms\nTo date, there are two organisms identified to produce an HDCR, T. kivui and the closely related\nAcetobacterium woodii , and characterization of these enzymes showed that both HDCRs form\nlarge  filaments 2,3 .  In  previous  studies,  filamentation  of A.  woodii HDCR  was  reported  to  be dependent  on  the  presence  of  divalent  cations,  and  in  accordance  with  our  findings,  the\nfilamentous form was the most active state of the enzyme 4 . While similar experiments with the\nT. kivui HDCR did not show a divalent cation dependency on filamentation, it must be noted that\nthe shortened protomers reported in the study of Schuchmann et al . 4 showed an elution profile corresponding to a molecular mass of about 3500 kDa. This indicates the reported enzyme still\nforms roughly a 14-mer (assuming the A. woodii HDCR forms hexamers as in T. kivui ) or a 20-\nmer (tetrameric form of the HDCR) and makes comparison of the depolymerized states of the\ntwo  enzymes  difficult.  However, A.  woodii HDCR  in  the  3500  kDa  state  showed  a  ~30  %\nreduction  in  activity  compared  to  a  similarly  treated  control  of  the  filamentous  form  (&gt;5000\nkDa), and therefore, is consistent with our findings of a stepwise reduction in HDCR activity,\ndepending on the number of connected active centers.\nSupplementary Table 1. Coordination of the [4Fe4S]-clusters in HDCR subunits.\nCys, cysteine; HC, hydrogen cluster.\nSupplementary  Table  2.  Conserved  W-  bis-pterin  guanine  dinucleotide  (W-bisPGD)\ncoordinating  residues. Amino  acids  interacting  with  the  W-bisPGD  cofactor  in  the  closest-\nrelated formate dehydrogenase (DgW-FDH from Desulfovibrio gigas , PDB: 1H0H) are mostly\nconserved in T. kivui FdhF. *, conserved amino acid.\n\n\nK968 K699 *\nSupplementary Table 3. T. kivui HDCR variants used in this study.\n\n\nCys, cysteine; Ala, alanine.\nSupplementary Table 4. Plasmids used in this study. The plasmids used in this study were\neither  used  for  PCR  (20  ng  per  25  µl  reaction)  or  for  transformation  of T.  kivui (1  mg).  C,\ncysteine; A, alanine; aa, amino acids.\npHD028 Overproduction of His 6 -tagged HDCR, HycB4 C114A, C117A, C120A This study pLR002 Plasmid containing HDCR without HydA2, His 6 at hycB4 This study\n\n\nSupplementary Table 5. Primers used in this study.\n\n\nPrimer Sequence (5´-3´)\n\n\nP1 GTGTAATTTTTTTATACAAATAATATTTCAATTCG\n\n\nP2 CCTGTTTACCATCTTTCATACAGTCAATCCTCCTCC\n\n\nP3 GAGGAGGATTGACTGTATGAAAGATGGTAAACAGGAAAAG\nP4 GATTTTTAATGGTGATGGTGATGGTGTACTTTTTTTCTCGGTGTATATTTAG\nP5 AGTACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAGTG\nP6 GGGTTTATCGACCTGCAGC\nP11 ATCCTCTAGAGTCGACCTG\nP12 GGTACCGAGCTCGAATTG\n\n\nP39 TAAGAGTGGTAAGATAAAAATCAAAAATTTTGTTGGTAGTG\n\n\nP40 ATTTTTGATTTTTATCTTACCACTCTTAAAAAACTC\n\n\nP47 GGGAGGAAAATGAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nP48 AATATTAGCAAAAATCATTCATTTTCCTCCCTTTTCCTTTGC\n\n\nP55 TCCAGAAGGAGCAATAAGCGGTGGTG\n\n\nP56 CAAGTCATCACTTCAACG\nP57 TAAGAGTGCTTTATTGGCATGTCCATTTG\nP58 GCGCCTATTGCCAACTTTTCATCTACTACG\n\n\nPLR3 GTAAGACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAG\n\n\nPLR4 TGATTTTTAATGGTGATGGTGATGGTGTCTTACCACTCTTAAAAAACTC\n\n\nPLR7 TCTATCTATTTGAGATTTTCATGTCTGCAAATAAAGCTA\n\n\nPLR8 TTATTTGCAGACATGAAAATCTCAAATAGATAGAAATATTAGC\n\n\nPLR11b GTATGCACCATCACCATCACCATAAAGATGGTAAACAGGAAAAGGTT\n\n\nPLR12b CTTTATGGTGATGGTGATGGTGCATACAGTCAATCCTCCTCCTTGTA\n\n\nPLR15 AAAGGGAGGAAAATGATTTATTAATTAATAATATTGAAAAGAAAGGGATGTG\n\n\nPLR16 ATATTATTAATTAATAAATCATTTTCCTCCCTTTTCCTTTG\n\n\nPRT7 TAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nPRT8 ACCATCAACCAATTTCAAAGCC\n\n\nPRT9 TAATTTATTAATTAATAATATTGAAAAGAAAGGGATG\nPRT10 CATAACCAAAGTCAACGCATTTTC\n\n\nPseq5 GAATCTTCAAATTCAGGCAATAAGC\nPseq6 TAGCCGACCCAAGAATGAAC\nPseq7 TTCAGCCAAATAGCCGAGAG\nPseq8 AGTAGCGGCTGCTAAAGTTG\nPseq9 AGGTGCGACGGTTGGTACTG\nPseq10 GCTCCGGCTATTAGAGTTTC\nSupplementary Figure 1. Uncropped gel images.\nSupplementary Video 1. Overall presentation of the segmented HDCR cryo-EM density\nand corresponding atomic model.\n\n\nColors are used as in Fig. 1.\nSupplementary Video 2. In situ visualization of bundled HDCR filaments inside a T. kivui\ncell.\nSequential slices back and forth through the cryo-ET volume, followed by reveal and tour of the\ncellular segmentation. Segmentation colored as described in Fig. 5.\n\n\nPage 1\n\n\nPage 2\n\n\nSupplementary References\n\n\nPage 4\n\n\nHycB3 [4Fe4S] I Cys13, Cys16, Cys19, Cys150 [4Fe4S] II Cys23, Cys134, Cys137, Cys146\n\n\nHydA2 [4Fe4S] I Cys15, Cys18, Cys21, Cys55 [4Fe4S] II Cys25, Cys45, Cys48, Cys51 [4Fe4S] (HC) Cys184, Cys239, Cys383, Cys387\n\n\nHDCR_His Δ hdcr strain + HDCR operon hydA2 C-term. This study HDCRΔHydA2 Δ hdcr strain + HDCR operon without hydA2 gene fdhF N-term. This study HDCRΔHycB4 Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. This study HDCRΔHycB3 Δ hdcr strain + HDCR operon without hycB3 gene hydA2 C-term. This study HDCRΔHycB3 ΔHycB4 Δ hdcr strain + HDCR operon without hycB3 and hycB4 genes hydA2 C-term. This study HDCRΔHycB4, His@Hyd &amp;\n\n\nHis@FdhF Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. and fdhF N-term. This study HDCR_HydA2 C387A Δ hdcr strain + HDCR operon, hydA2 _Cys387 substituted with Ala hydA2 C-term. This study HDCR_HycB4 Δ[4Fe4S] IV Δ hdcr strain + HDCR operon, hycB4 _Cys114, Cys117, Cys120\n\n\nsubstituted with Ala hydA2 C-term. This study HDCR_HycB3ΔC Δ hdcr strain + HDCR operon, deletion of hycB3 amino acids 160-184 hydA2 C-term. This study HDCR_HycB4ΔC Δ hdcr strain + HDCR operon, deletion\n\n\nof hycB4 amino acids 190-210 hydA2 C-term. This study\n\n\npHD026 Overproduction of His 6 -tagged HDCR, HydA2 C387A This study\n\n\np. 3 Supplementary Methods p. 4 Supplementary Discussion p. 5\n\n\np. 9\n\n\nSubunit [4Fe4S] -Number Coordinating amino acids FdhF [4Fe4S] I Cys13, Cys16, Cys20, Cys48\n\n\n[4Fe4S] III Cys52, Cys55, Cys61, Cys93 [4Fe4S] IV Cys64, Cys83, Cys86, Cys89 HycB4 [4Fe4S] I Cys34, Cys37, Cys40, Cys180 [4Fe4S] II Cys44, Cys165, Cys168, Cys176 [4Fe4S] III Cys83, Cys86, Cys91, Cys124 [4Fe4S] IV Cys95, Cys114, Cys117, Cys120\n\n\nW-bisPGD coordinating amino acid in DgW-FDH Related amino acid in T. kivui FdhF C20 C16 * C54 C48 * K56 K50 * Q154 C135 CSe158 C139 (*) E197 E179 * N198 C180 N194 N176 * D220 D202 * R222 R204 * G239 G221 * D241 N223 M371 M297 * W373 V299 N518 N403 * Q517 E402 S522 S407 * V542 Q427 N543 D428 K578 K450 * D605 missing T854 T600 * R856 R602 * V857 R603 T858 V604 H860 H606 * Q862 H608 T863 T609 * N951 N682 *\n\n\nStrain Genotype His 6 -tag location Reference Δhdcr hdcr operon deleted none ( 5 )\n\n\nPlasmid Purpose Reference pMU131 Shuttle vector ( 6 )\n\n\npPB5 Cloning vector for HDCR operon with S-layer promoter ( 7 )\n\n\npHD001 Overproduction of His 6 -tagged HDCR This study pHD015 Overproduction of His 6 -tagged HDCRΔHydA2 This study pHD020 Overproduction of His 6 -tagged HDCRΔHycB3 This study pHD024 Overproduction of His 6 -tagged HDCRΔHycB4, His 6 -tag at\n\n\nFdhF and His 6 -tag at HydA2 This study\n\n\npLR002c Plasmid containing HDCR without HydA2, His 6 at hycB4 and fdhF This study pLR003b Overproduction of His 6 -tagged HDCRΔHycB4 This study pLR004 Overproduction of His 6 -tagged HDCRΔHycB3ΔHycB4 This study pRT8 Overproduction of His 6 -tagged HDCR, HycB3Δaa160-184 This study pRT9 Overproduction of His 6 -tagged HDCR, HycB4Δaa190-210 This study\n",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Supplementary information\nhttps://doi.org/10.1038/s41586-022-04971-z\nSupplementary Information for\nMembrane-anchored HDCR nanowires drive hydrogen-powered CO 2 fixation Helge M. Dietrich 1† , Ricardo D. Righetto 2,3† , Anuj Kumar 1,4 , Wojciech Wietrzynski 2,3 , Raphael\nTrischler 1 , Sandra K. Schuller 4 , Jonathan Wagner 5 , Fabian M. Schwarz 1 , Benjamin D. Engel 2,3 *, Volker Müller 1 *, Jan M. Schuller 4 *\n1 Molecular Microbiology &amp; Bioenergetics, Institute of Molecular Biosciences,\nJohann Wolfgang Goethe University; Frankfurt am Main, Germany\n2 Helmholtz Pioneer Campus, Helmholtz Munich; Neuherberg, Germany.\n3 Biozentrum, University of Basel; Basel, Switzerland\n4 CryoEM of Molecular Machines, SYNMIKRO Research Center and Department of\nChemistry, Philipps-University, Marburg, Germany\n5 Department of Molecular Structural Biology, Max Planck Institute of Biochemistry;\nMartinsried, Germany\n† Equal contribution\n*Correspondence to: ben.engel@unibas.ch , vmueller@bio.uni-frankfurt.de ,\njan.schuller@synmikro.uni-marburg.de\n\n\nSupplemental Information Table of Contents\nSupplementary Information\nSupplementary Table 1 - Coordination of the [4Fe4S]-clusters in HDCR subunits. p. 6\nSupplementary Table 2 - Conserved W- bis-pterin guanine dinucleotide (W-bisPGD)\ncoordinating residues. p. 7\nSupplementary Table 3 - T. kivui HDCR variants used in this study. p. 8\nSupplementary Table 4 - Plasmids used in this study.\nSupplementary Table 5 - Primers used in this study. p. 10\nSupplementary Figure 1 - Uncropped gel images. p. 11\nFigure legends for Supplementary Videos p. 12\nSupplementary Information:\n\n\nInteraction studies of HDCR subunits\nGenetic complementation of the Δ hdcr strain was used to probe the connectivity of the HDCR complex. When HycB4 was omitted from the overexpression (HDCR_ΔHycB4), only HydA2\nwas found in the eluate after purification with HydA2-His 6 (Fig. 2C). However, addition of a second  His 6 -tag  to  the  HDCR_ΔHycB4  construct  (HydA2-His 6 &amp;  His 6 -FdhF)  led  to\ncopurification of monomeric FdhF and the protein pair HycB3-HydA2, although there was no\nfunctional interaction between FdhF and HycB3-HydA2. This verifies HycB4 as the connecting\nsubunit between HydA2  and the protein  pair of FdhF and HycB3.  When  HycB3 was omitted\nfrom  the  overexpression  (HDCRΔHycB3),  HydA2-His 6 purification  yielded  a  complex containing  HydA2  and  HycB4,  but  not  FdhF.  This  proves  that  HycB3  is  the  subunit  linking\nHycB4 to FdhF, revealing the chain of protein interactions to be FdhF-HycB3-HycB4-HydA2.\nThis  conclusion  is  in  complete  accordance  with  the  cryo-EM  structure  (Fig.  1)  and  is  also\nsupported  by  the  genetic  organization  of  the hdcr operon,  where  the  genes  for  the  enzyme complex are arranged in the same order 1 .\nCatalytic centers operate independently, but HycB3 is vital for FdhF activity\nHydrogenase activity was independent of HycB3 and HycB4. When HydA2 was omitted from\noverexpression  (HDCR_∆HydA2)  or  the  H-cluster  coordinating  Cys387  was  substituted  with\nalanine (HDCR_HydA2 C387A), there was no hydrogenase activity (Extended Data Fig. 7F).\nFormate dehydrogenase activity was marginally affected by removal of HycB4 and HydA2, but\nSupplementary Methods 1: Additional methods for Extended Data Fig. 7.\nFor methylviologen-dependent hydrogenase activity, 1.1 x 10 5 Pa H 2 was used as electron donor.\nFor  methylviologen-dependent  formate  dehydrogenase  assay  and  analysis  of  hydrogen\nproduction from formate, 150 mM formate was used as electron donor. Formate production from\nH 2 + CO 2 was determined with a substrate gas composition of H 2 + CO 2 (80:20 [v:v], 1.1 x 10 5\nPa). The reaction buffer for pH-optima analysis (Extended Data Fig. 7a,b) composed of 50 mM\nMES, 50 mM  MOPS, 50 mM  HEPES, 50 mM  EPPS, 50 mM  CHES, 20 mM  MgSO 4 , 4 µM\nResazurin, 2 mM DTE, pH as indicated. The reaction buffer for Extended Data Fig. 7c-h was\n100 mM HEPES, 20 mM MgSO 4 , 4 µM Resazurin, 2 mM DTE, pH 7.0.\n\n\nSupplementary Discussion\nRelevance of HDCR filamentation to its activity in different organisms\nTo date, there are two organisms identified to produce an HDCR, T. kivui and the closely related\nAcetobacterium woodii , and characterization of these enzymes showed that both HDCRs form\nlarge  filaments 2,3 .  In  previous  studies,  filamentation  of A.  woodii HDCR  was  reported  to  be dependent  on  the  presence  of  divalent  cations,  and  in  accordance  with  our  findings,  the\nfilamentous form was the most active state of the enzyme 4 . While similar experiments with the\nT. kivui HDCR did not show a divalent cation dependency on filamentation, it must be noted that\nthe shortened protomers reported in the study of Schuchmann et al . 4 showed an elution profile corresponding to a molecular mass of about 3500 kDa. This indicates the reported enzyme still\nforms roughly a 14-mer (assuming the A. woodii HDCR forms hexamers as in T. kivui ) or a 20-\nmer (tetrameric form of the HDCR) and makes comparison of the depolymerized states of the\ntwo  enzymes  difficult.  However, A.  woodii HDCR  in  the  3500  kDa  state  showed  a  ~30  %\nreduction  in  activity  compared  to  a  similarly  treated  control  of  the  filamentous  form  (&gt;5000\nkDa), and therefore, is consistent with our findings of a stepwise reduction in HDCR activity,\ndepending on the number of connected active centers.\nSupplementary Table 1. Coordination of the [4Fe4S]-clusters in HDCR subunits.\nCys, cysteine; HC, hydrogen cluster.\nSupplementary  Table  2.  Conserved  W-  bis-pterin  guanine  dinucleotide  (W-bisPGD)\ncoordinating  residues. Amino  acids  interacting  with  the  W-bisPGD  cofactor  in  the  closest-\nrelated formate dehydrogenase (DgW-FDH from Desulfovibrio gigas , PDB: 1H0H) are mostly\nconserved in T. kivui FdhF. *, conserved amino acid.\n\n\nK968 K699 *\nSupplementary Table 3. T. kivui HDCR variants used in this study.\n\n\nCys, cysteine; Ala, alanine.\nSupplementary Table 4. Plasmids used in this study. The plasmids used in this study were\neither  used  for  PCR  (20  ng  per  25  µl  reaction)  or  for  transformation  of T.  kivui (1  mg).  C,\ncysteine; A, alanine; aa, amino acids.\npHD028 Overproduction of His 6 -tagged HDCR, HycB4 C114A, C117A, C120A This study pLR002 Plasmid containing HDCR without HydA2, His 6 at hycB4 This study\n\n\nSupplementary Table 5. Primers used in this study.\n\n\nPrimer Sequence (5´-3´)\n\n\nP1 GTGTAATTTTTTTATACAAATAATATTTCAATTCG\n\n\nP2 CCTGTTTACCATCTTTCATACAGTCAATCCTCCTCC\n\n\nP3 GAGGAGGATTGACTGTATGAAAGATGGTAAACAGGAAAAG\nP4 GATTTTTAATGGTGATGGTGATGGTGTACTTTTTTTCTCGGTGTATATTTAG\nP5 AGTACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAGTG\nP6 GGGTTTATCGACCTGCAGC\nP11 ATCCTCTAGAGTCGACCTG\nP12 GGTACCGAGCTCGAATTG\n\n\nP39 TAAGAGTGGTAAGATAAAAATCAAAAATTTTGTTGGTAGTG\n\n\nP40 ATTTTTGATTTTTATCTTACCACTCTTAAAAAACTC\n\n\nP47 GGGAGGAAAATGAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nP48 AATATTAGCAAAAATCATTCATTTTCCTCCCTTTTCCTTTGC\n\n\nP55 TCCAGAAGGAGCAATAAGCGGTGGTG\n\n\nP56 CAAGTCATCACTTCAACG\nP57 TAAGAGTGCTTTATTGGCATGTCCATTTG\nP58 GCGCCTATTGCCAACTTTTCATCTACTACG\n\n\nPLR3 GTAAGACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAG\n\n\nPLR4 TGATTTTTAATGGTGATGGTGATGGTGTCTTACCACTCTTAAAAAACTC\n\n\nPLR7 TCTATCTATTTGAGATTTTCATGTCTGCAAATAAAGCTA\n\n\nPLR8 TTATTTGCAGACATGAAAATCTCAAATAGATAGAAATATTAGC\n\n\nPLR11b GTATGCACCATCACCATCACCATAAAGATGGTAAACAGGAAAAGGTT\n\n\nPLR12b CTTTATGGTGATGGTGATGGTGCATACAGTCAATCCTCCTCCTTGTA\n\n\nPLR15 AAAGGGAGGAAAATGATTTATTAATTAATAATATTGAAAAGAAAGGGATGTG\n\n\nPLR16 ATATTATTAATTAATAAATCATTTTCCTCCCTTTTCCTTTG\n\n\nPRT7 TAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nPRT8 ACCATCAACCAATTTCAAAGCC\n\n\nPRT9 TAATTTATTAATTAATAATATTGAAAAGAAAGGGATG\nPRT10 CATAACCAAAGTCAACGCATTTTC\n\n\nPseq5 GAATCTTCAAATTCAGGCAATAAGC\nPseq6 TAGCCGACCCAAGAATGAAC\nPseq7 TTCAGCCAAATAGCCGAGAG\nPseq8 AGTAGCGGCTGCTAAAGTTG\nPseq9 AGGTGCGACGGTTGGTACTG\nPseq10 GCTCCGGCTATTAGAGTTTC\nSupplementary Figure 1. Uncropped gel images.\nSupplementary Video 1. Overall presentation of the segmented HDCR cryo-EM density\nand corresponding atomic model.\n\n\nColors are used as in Fig. 1.\nSupplementary Video 2. In situ visualization of bundled HDCR filaments inside a T. kivui\ncell.\nSequential slices back and forth through the cryo-ET volume, followed by reveal and tour of the\ncellular segmentation. Segmentation colored as described in Fig. 5.\n\n\nPage 1\n\n\nPage 2\n\n\nSupplementary References\n\n\nPage 4\n\n\nHycB3 [4Fe4S] I Cys13, Cys16, Cys19, Cys150 [4Fe4S] II Cys23, Cys134, Cys137, Cys146\n\n\nHydA2 [4Fe4S] I Cys15, Cys18, Cys21, Cys55 [4Fe4S] II Cys25, Cys45, Cys48, Cys51 [4Fe4S] (HC) Cys184, Cys239, Cys383, Cys387\n\n\nHDCR_His Δ hdcr strain + HDCR operon hydA2 C-term. This study HDCRΔHydA2 Δ hdcr strain + HDCR operon without hydA2 gene fdhF N-term. This study HDCRΔHycB4 Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. This study HDCRΔHycB3 Δ hdcr strain + HDCR operon without hycB3 gene hydA2 C-term. This study HDCRΔHycB3 ΔHycB4 Δ hdcr strain + HDCR operon without hycB3 and hycB4 genes hydA2 C-term. This study HDCRΔHycB4, His@Hyd &amp;\n\n\nHis@FdhF Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. and fdhF N-term. This study HDCR_HydA2 C387A Δ hdcr strain + HDCR operon, hydA2 _Cys387 substituted with Ala hydA2 C-term. This study HDCR_HycB4 Δ[4Fe4S] IV Δ hdcr strain + HDCR operon, hycB4 _Cys114, Cys117, Cys120\n\n\nsubstituted with Ala hydA2 C-term. This study HDCR_HycB3ΔC Δ hdcr strain + HDCR operon, deletion of hycB3 amino acids 160-184 hydA2 C-term. This study HDCR_HycB4ΔC Δ hdcr strain + HDCR operon, deletion\n\n\nof hycB4 amino acids 190-210 hydA2 C-term. This study\n\n\npHD026 Overproduction of His 6 -tagged HDCR, HydA2 C387A This study\n\n\np. 3 Supplementary Methods p. 4 Supplementary Discussion p. 5\n\n\np. 9\n\n\nSubunit [4Fe4S] -Number Coordinating amino acids FdhF [4Fe4S] I Cys13, Cys16, Cys20, Cys48\n\n\n[4Fe4S] III Cys52, Cys55, Cys61, Cys93 [4Fe4S] IV Cys64, Cys83, Cys86, Cys89 HycB4 [4Fe4S] I Cys34, Cys37, Cys40, Cys180 [4Fe4S] II Cys44, Cys165, Cys168, Cys176 [4Fe4S] III Cys83, Cys86, Cys91, Cys124 [4Fe4S] IV Cys95, Cys114, Cys117, Cys120\n\n\nW-bisPGD coordinating amino acid in DgW-FDH Related amino acid in T. kivui FdhF C20 C16 * C54 C48 * K56 K50 * Q154 C135 CSe158 C139 (*) E197 E179 * N198 C180 N194 N176 * D220 D202 * R222 R204 * G239 G221 * D241 N223 M371 M297 * W373 V299 N518 N403 * Q517 E402 S522 S407 * V542 Q427 N543 D428 K578 K450 * D605 missing T854 T600 * R856 R602 * V857 R603 T858 V604 H860 H606 * Q862 H608 T863 T609 * N951 N682 *\n\n\nStrain Genotype His 6 -tag location Reference Δhdcr hdcr operon deleted none ( 5 )\n\n\nPlasmid Purpose Reference pMU131 Shuttle vector ( 6 )\n\n\npPB5 Cloning vector for HDCR operon with S-layer promoter ( 7 )\n\n\npHD001 Overproduction of His 6 -tagged HDCR This study pHD015 Overproduction of His 6 -tagged HDCRΔHydA2 This study pHD020 Overproduction of His 6 -tagged HDCRΔHycB3 This study pHD024 Overproduction of His 6 -tagged HDCRΔHycB4, His 6 -tag at\n\n\nFdhF and His 6 -tag at HydA2 This study\n\n\npLR002c Plasmid containing HDCR without HydA2, His 6 at hycB4 and fdhF This study pLR003b Overproduction of His 6 -tagged HDCRΔHycB4 This study pLR004 Overproduction of His 6 -tagged HDCRΔHycB3ΔHycB4 This study pRT8 Overproduction of His 6 -tagged HDCR, HycB3Δaa160-184 This study pRT9 Overproduction of His 6 -tagged HDCR, HycB4Δaa190-210 This study\n",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Supplementary information\nhttps://doi.org/10.1038/s41586-022-04971-z\nSupplementary Information for\nMembrane-anchored HDCR nanowires drive hydrogen-powered CO 2 fixation Helge M. Dietrich 1† , Ricardo D. Righetto 2,3† , Anuj Kumar 1,4 , Wojciech Wietrzynski 2,3 , Raphael\nTrischler 1 , Sandra K. Schuller 4 , Jonathan Wagner 5 , Fabian M. Schwarz 1 , Benjamin D. Engel 2,3 *, Volker Müller 1 *, Jan M. Schuller 4 *\n1 Molecular Microbiology &amp; Bioenergetics, Institute of Molecular Biosciences,\nJohann Wolfgang Goethe University; Frankfurt am Main, Germany\n2 Helmholtz Pioneer Campus, Helmholtz Munich; Neuherberg, Germany.\n3 Biozentrum, University of Basel; Basel, Switzerland\n4 CryoEM of Molecular Machines, SYNMIKRO Research Center and Department of\nChemistry, Philipps-University, Marburg, Germany\n5 Department of Molecular Structural Biology, Max Planck Institute of Biochemistry;\nMartinsried, Germany\n† Equal contribution\n*Correspondence to: ben.engel@unibas.ch , vmueller@bio.uni-frankfurt.de ,\njan.schuller@synmikro.uni-marburg.de\n\n\nSupplemental Information Table of Contents\nSupplementary Information\nSupplementary Table 1 - Coordination of the [4Fe4S]-clusters in HDCR subunits. p. 6\nSupplementary Table 2 - Conserved W- bis-pterin guanine dinucleotide (W-bisPGD)\ncoordinating residues. p. 7\nSupplementary Table 3 - T. kivui HDCR variants used in this study. p. 8\nSupplementary Table 4 - Plasmids used in this study.\nSupplementary Table 5 - Primers used in this study. p. 10\nSupplementary Figure 1 - Uncropped gel images. p. 11\nFigure legends for Supplementary Videos p. 12\nSupplementary Information:\n\n\nInteraction studies of HDCR subunits\nGenetic complementation of the Δ hdcr strain was used to probe the connectivity of the HDCR complex. When HycB4 was omitted from the overexpression (HDCR_ΔHycB4), only HydA2\nwas found in the eluate after purification with HydA2-His 6 (Fig. 2C). However, addition of a second  His 6 -tag  to  the  HDCR_ΔHycB4  construct  (HydA2-His 6 &amp;  His 6 -FdhF)  led  to\ncopurification of monomeric FdhF and the protein pair HycB3-HydA2, although there was no\nfunctional interaction between FdhF and HycB3-HydA2. This verifies HycB4 as the connecting\nsubunit between HydA2  and the protein  pair of FdhF and HycB3.  When  HycB3 was omitted\nfrom  the  overexpression  (HDCRΔHycB3),  HydA2-His 6 purification  yielded  a  complex containing  HydA2  and  HycB4,  but  not  FdhF.  This  proves  that  HycB3  is  the  subunit  linking\nHycB4 to FdhF, revealing the chain of protein interactions to be FdhF-HycB3-HycB4-HydA2.\nThis  conclusion  is  in  complete  accordance  with  the  cryo-EM  structure  (Fig.  1)  and  is  also\nsupported  by  the  genetic  organization  of  the hdcr operon,  where  the  genes  for  the  enzyme complex are arranged in the same order 1 .\nCatalytic centers operate independently, but HycB3 is vital for FdhF activity\nHydrogenase activity was independent of HycB3 and HycB4. When HydA2 was omitted from\noverexpression  (HDCR_∆HydA2)  or  the  H-cluster  coordinating  Cys387  was  substituted  with\nalanine (HDCR_HydA2 C387A), there was no hydrogenase activity (Extended Data Fig. 7F).\nFormate dehydrogenase activity was marginally affected by removal of HycB4 and HydA2, but\nSupplementary Methods 1: Additional methods for Extended Data Fig. 7.\nFor methylviologen-dependent hydrogenase activity, 1.1 x 10 5 Pa H 2 was used as electron donor.\nFor  methylviologen-dependent  formate  dehydrogenase  assay  and  analysis  of  hydrogen\nproduction from formate, 150 mM formate was used as electron donor. Formate production from\nH 2 + CO 2 was determined with a substrate gas composition of H 2 + CO 2 (80:20 [v:v], 1.1 x 10 5\nPa). The reaction buffer for pH-optima analysis (Extended Data Fig. 7a,b) composed of 50 mM\nMES, 50 mM  MOPS, 50 mM  HEPES, 50 mM  EPPS, 50 mM  CHES, 20 mM  MgSO 4 , 4 µM\nResazurin, 2 mM DTE, pH as indicated. The reaction buffer for Extended Data Fig. 7c-h was\n100 mM HEPES, 20 mM MgSO 4 , 4 µM Resazurin, 2 mM DTE, pH 7.0.\n\n\nSupplementary Discussion\nRelevance of HDCR filamentation to its activity in different organisms\nTo date, there are two organisms identified to produce an HDCR, T. kivui and the closely related\nAcetobacterium woodii , and characterization of these enzymes showed that both HDCRs form\nlarge  filaments 2,3 .  In  previous  studies,  filamentation  of A.  woodii HDCR  was  reported  to  be dependent  on  the  presence  of  divalent  cations,  and  in  accordance  with  our  findings,  the\nfilamentous form was the most active state of the enzyme 4 . While similar experiments with the\nT. kivui HDCR did not show a divalent cation dependency on filamentation, it must be noted that\nthe shortened protomers reported in the study of Schuchmann et al . 4 showed an elution profile corresponding to a molecular mass of about 3500 kDa. This indicates the reported enzyme still\nforms roughly a 14-mer (assuming the A. woodii HDCR forms hexamers as in T. kivui ) or a 20-\nmer (tetrameric form of the HDCR) and makes comparison of the depolymerized states of the\ntwo  enzymes  difficult.  However, A.  woodii HDCR  in  the  3500  kDa  state  showed  a  ~30  %\nreduction  in  activity  compared  to  a  similarly  treated  control  of  the  filamentous  form  (&gt;5000\nkDa), and therefore, is consistent with our findings of a stepwise reduction in HDCR activity,\ndepending on the number of connected active centers.\nSupplementary Table 1. Coordination of the [4Fe4S]-clusters in HDCR subunits.\nCys, cysteine; HC, hydrogen cluster.\nSupplementary  Table  2.  Conserved  W-  bis-pterin  guanine  dinucleotide  (W-bisPGD)\ncoordinating  residues. Amino  acids  interacting  with  the  W-bisPGD  cofactor  in  the  closest-\nrelated formate dehydrogenase (DgW-FDH from Desulfovibrio gigas , PDB: 1H0H) are mostly\nconserved in T. kivui FdhF. *, conserved amino acid.\n\n\nK968 K699 *\nSupplementary Table 3. T. kivui HDCR variants used in this study.\n\n\nCys, cysteine; Ala, alanine.\nSupplementary Table 4. Plasmids used in this study. The plasmids used in this study were\neither  used  for  PCR  (20  ng  per  25  µl  reaction)  or  for  transformation  of T.  kivui (1  mg).  C,\ncysteine; A, alanine; aa, amino acids.\npHD028 Overproduction of His 6 -tagged HDCR, HycB4 C114A, C117A, C120A This study pLR002 Plasmid containing HDCR without HydA2, His 6 at hycB4 This study\n\n\nSupplementary Table 5. Primers used in this study.\n\n\nPrimer Sequence (5´-3´)\n\n\nP1 GTGTAATTTTTTTATACAAATAATATTTCAATTCG\n\n\nP2 CCTGTTTACCATCTTTCATACAGTCAATCCTCCTCC\n\n\nP3 GAGGAGGATTGACTGTATGAAAGATGGTAAACAGGAAAAG\nP4 GATTTTTAATGGTGATGGTGATGGTGTACTTTTTTTCTCGGTGTATATTTAG\nP5 AGTACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAGTG\nP6 GGGTTTATCGACCTGCAGC\nP11 ATCCTCTAGAGTCGACCTG\nP12 GGTACCGAGCTCGAATTG\n\n\nP39 TAAGAGTGGTAAGATAAAAATCAAAAATTTTGTTGGTAGTG\n\n\nP40 ATTTTTGATTTTTATCTTACCACTCTTAAAAAACTC\n\n\nP47 GGGAGGAAAATGAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nP48 AATATTAGCAAAAATCATTCATTTTCCTCCCTTTTCCTTTGC\n\n\nP55 TCCAGAAGGAGCAATAAGCGGTGGTG\n\n\nP56 CAAGTCATCACTTCAACG\nP57 TAAGAGTGCTTTATTGGCATGTCCATTTG\nP58 GCGCCTATTGCCAACTTTTCATCTACTACG\n\n\nPLR3 GTAAGACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAG\n\n\nPLR4 TGATTTTTAATGGTGATGGTGATGGTGTCTTACCACTCTTAAAAAACTC\n\n\nPLR7 TCTATCTATTTGAGATTTTCATGTCTGCAAATAAAGCTA\n\n\nPLR8 TTATTTGCAGACATGAAAATCTCAAATAGATAGAAATATTAGC\n\n\nPLR11b GTATGCACCATCACCATCACCATAAAGATGGTAAACAGGAAAAGGTT\n\n\nPLR12b CTTTATGGTGATGGTGATGGTGCATACAGTCAATCCTCCTCCTTGTA\n\n\nPLR15 AAAGGGAGGAAAATGATTTATTAATTAATAATATTGAAAAGAAAGGGATGTG\n\n\nPLR16 ATATTATTAATTAATAAATCATTTTCCTCCCTTTTCCTTTG\n\n\nPRT7 TAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nPRT8 ACCATCAACCAATTTCAAAGCC\n\n\nPRT9 TAATTTATTAATTAATAATATTGAAAAGAAAGGGATG\nPRT10 CATAACCAAAGTCAACGCATTTTC\n\n\nPseq5 GAATCTTCAAATTCAGGCAATAAGC\nPseq6 TAGCCGACCCAAGAATGAAC\nPseq7 TTCAGCCAAATAGCCGAGAG\nPseq8 AGTAGCGGCTGCTAAAGTTG\nPseq9 AGGTGCGACGGTTGGTACTG\nPseq10 GCTCCGGCTATTAGAGTTTC\nSupplementary Figure 1. Uncropped gel images.\nSupplementary Video 1. Overall presentation of the segmented HDCR cryo-EM density\nand corresponding atomic model.\n\n\nColors are used as in Fig. 1.\nSupplementary Video 2. In situ visualization of bundled HDCR filaments inside a T. kivui\ncell.\nSequential slices back and forth through the cryo-ET volume, followed by reveal and tour of the\ncellular segmentation. Segmentation colored as described in Fig. 5.\n\n\nPage 1\n\n\nPage 2\n\n\nSupplementary References\n\n\nPage 4\n\n\nHycB3 [4Fe4S] I Cys13, Cys16, Cys19, Cys150 [4Fe4S] II Cys23, Cys134, Cys137, Cys146\n\n\nHydA2 [4Fe4S] I Cys15, Cys18, Cys21, Cys55 [4Fe4S] II Cys25, Cys45, Cys48, Cys51 [4Fe4S] (HC) Cys184, Cys239, Cys383, Cys387\n\n\nHDCR_His Δ hdcr strain + HDCR operon hydA2 C-term. This study HDCRΔHydA2 Δ hdcr strain + HDCR operon without hydA2 gene fdhF N-term. This study HDCRΔHycB4 Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. This study HDCRΔHycB3 Δ hdcr strain + HDCR operon without hycB3 gene hydA2 C-term. This study HDCRΔHycB3 ΔHycB4 Δ hdcr strain + HDCR operon without hycB3 and hycB4 genes hydA2 C-term. This study HDCRΔHycB4, His@Hyd &amp;\n\n\nHis@FdhF Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. and fdhF N-term. This study HDCR_HydA2 C387A Δ hdcr strain + HDCR operon, hydA2 _Cys387 substituted with Ala hydA2 C-term. This study HDCR_HycB4 Δ[4Fe4S] IV Δ hdcr strain + HDCR operon, hycB4 _Cys114, Cys117, Cys120\n\n\nsubstituted with Ala hydA2 C-term. This study HDCR_HycB3ΔC Δ hdcr strain + HDCR operon, deletion of hycB3 amino acids 160-184 hydA2 C-term. This study HDCR_HycB4ΔC Δ hdcr strain + HDCR operon, deletion\n\n\nof hycB4 amino acids 190-210 hydA2 C-term. This study\n\n\npHD026 Overproduction of His 6 -tagged HDCR, HydA2 C387A This study\n\n\np. 3 Supplementary Methods p. 4 Supplementary Discussion p. 5\n\n\np. 9\n\n\nSubunit [4Fe4S] -Number Coordinating amino acids FdhF [4Fe4S] I Cys13, Cys16, Cys20, Cys48\n\n\n[4Fe4S] III Cys52, Cys55, Cys61, Cys93 [4Fe4S] IV Cys64, Cys83, Cys86, Cys89 HycB4 [4Fe4S] I Cys34, Cys37, Cys40, Cys180 [4Fe4S] II Cys44, Cys165, Cys168, Cys176 [4Fe4S] III Cys83, Cys86, Cys91, Cys124 [4Fe4S] IV Cys95, Cys114, Cys117, Cys120\n\n\nW-bisPGD coordinating amino acid in DgW-FDH Related amino acid in T. kivui FdhF C20 C16 * C54 C48 * K56 K50 * Q154 C135 CSe158 C139 (*) E197 E179 * N198 C180 N194 N176 * D220 D202 * R222 R204 * G239 G221 * D241 N223 M371 M297 * W373 V299 N518 N403 * Q517 E402 S522 S407 * V542 Q427 N543 D428 K578 K450 * D605 missing T854 T600 * R856 R602 * V857 R603 T858 V604 H860 H606 * Q862 H608 T863 T609 * N951 N682 *\n\n\nStrain Genotype His 6 -tag location Reference Δhdcr hdcr operon deleted none ( 5 )\n\n\nPlasmid Purpose Reference pMU131 Shuttle vector ( 6 )\n\n\npPB5 Cloning vector for HDCR operon with S-layer promoter ( 7 )\n\n\npHD001 Overproduction of His 6 -tagged HDCR This study pHD015 Overproduction of His 6 -tagged HDCRΔHydA2 This study pHD020 Overproduction of His 6 -tagged HDCRΔHycB3 This study pHD024 Overproduction of His 6 -tagged HDCRΔHycB4, His 6 -tag at\n\n\nFdhF and His 6 -tag at HydA2 This study\n\n\npLR002c Plasmid containing HDCR without HydA2, His 6 at hycB4 and fdhF This study pLR003b Overproduction of His 6 -tagged HDCRΔHycB4 This study pLR004 Overproduction of His 6 -tagged HDCRΔHycB3ΔHycB4 This study pRT8 Overproduction of His 6 -tagged HDCR, HycB3Δaa160-184 This study pRT9 Overproduction of His 6 -tagged HDCR, HycB4Δaa190-210 This study\n'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Supplementary information\nhttps://doi.org/10.1038/s41586-022-04971-z\nSupplementary Information for\nMembrane-anchored HDCR nanowires drive hydrogen-powered CO 2 fixation Helge M. Dietrich 1† , Ricardo D. Righetto 2,3† , Anuj Kumar 1,4 , Wojciech Wietrzynski 2,3 , Raphael\nTrischler 1 , Sandra K. Schuller 4 , Jonathan Wagner 5 , Fabian M. Schwarz 1 , Benjamin D. Engel 2,3 *, Volker Müller 1 *, Jan M. Schuller 4 *\n1 Molecular Microbiology &amp; Bioenergetics, Institute of Molecular Biosciences,\nJohann Wolfgang Goethe University; Frankfurt am Main, Germany\n2 Helmholtz Pioneer Campus, Helmholtz Munich; Neuherberg, Germany.\n3 Biozentrum, University of Basel; Basel, Switzerland\n4 CryoEM of Molecular Machines, SYNMIKRO Research Center and Department of\nChemistry, Philipps-University, Marburg, Germany\n5 Department of Molecular Structural Biology, Max Planck Institute of Biochemistry;\nMartinsried, Germany\n† Equal contribution\n*Correspondence to: ben.engel@unibas.ch , vmueller@bio.uni-frankfurt.de ,\njan.schuller@synmikro.uni-marburg.de\n\n\nSupplemental Information Table of Contents\nSupplementary Information\nSupplementary Table 1 - Coordination of the [4Fe4S]-clusters in HDCR subunits. p. 6\nSupplementary Table 2 - Conserved W- bis-pterin guanine dinucleotide (W-bisPGD)\ncoordinating residues. p. 7\nSupplementary Table 3 - T. kivui HDCR variants used in this study. p. 8\nSupplementary Table 4 - Plasmids used in this study.\nSupplementary Table 5 - Primers used in this study. p. 10\nSupplementary Figure 1 - Uncropped gel images. p. 11\nFigure legends for Supplementary Videos p. 12\nSupplementary Information:\n\n\nInteraction studies of HDCR subunits\nGenetic complementation of the Δ hdcr strain was used to probe the connectivity of the HDCR complex. When HycB4 was omitted from the overexpression (HDCR_ΔHycB4), only HydA2\nwas found in the eluate after purification with HydA2-His 6 (Fig. 2C). However, addition of a second  His 6 -tag  to  the  HDCR_ΔHycB4  construct  (HydA2-His 6 &amp;  His 6 -FdhF)  led  to\ncopurification of monomeric FdhF and the protein pair HycB3-HydA2, although there was no\nfunctional interaction between FdhF and HycB3-HydA2. This verifies HycB4 as the connecting\nsubunit between HydA2  and the protein  pair of FdhF and HycB3.  When  HycB3 was omitted\nfrom  the  overexpression  (HDCRΔHycB3),  HydA2-His 6 purification  yielded  a  complex containing  HydA2  and  HycB4,  but  not  FdhF.  This  proves  that  HycB3  is  the  subunit  linking\nHycB4 to FdhF, revealing the chain of protein interactions to be FdhF-HycB3-HycB4-HydA2.\nThis  conclusion  is  in  complete  accordance  with  the  cryo-EM  structure  (Fig.  1)  and  is  also\nsupported  by  the  genetic  organization  of  the hdcr operon,  where  the  genes  for  the  enzyme complex are arranged in the same order 1 .\nCatalytic centers operate independently, but HycB3 is vital for FdhF activity\nHydrogenase activity was independent of HycB3 and HycB4. When HydA2 was omitted from\noverexpression  (HDCR_∆HydA2)  or  the  H-cluster  coordinating  Cys387  was  substituted  with\nalanine (HDCR_HydA2 C387A), there was no hydrogenase activity (Extended Data Fig. 7F).\nFormate dehydrogenase activity was marginally affected by removal of HycB4 and HydA2, but\nSupplementary Methods 1: Additional methods for Extended Data Fig. 7.\nFor methylviologen-dependent hydrogenase activity, 1.1 x 10 5 Pa H 2 was used as electron donor.\nFor  methylviologen-dependent  formate  dehydrogenase  assay  and  analysis  of  hydrogen\nproduction from formate, 150 mM formate was used as electron donor. Formate production from\nH 2 + CO 2 was determined with a substrate gas composition of H 2 + CO 2 (80:20 [v:v], 1.1 x 10 5\nPa). The reaction buffer for pH-optima analysis (Extended Data Fig. 7a,b) composed of 50 mM\nMES, 50 mM  MOPS, 50 mM  HEPES, 50 mM  EPPS, 50 mM  CHES, 20 mM  MgSO 4 , 4 µM\nResazurin, 2 mM DTE, pH as indicated. The reaction buffer for Extended Data Fig. 7c-h was\n100 mM HEPES, 20 mM MgSO 4 , 4 µM Resazurin, 2 mM DTE, pH 7.0.\n\n\nSupplementary Discussion\nRelevance of HDCR filamentation to its activity in different organisms\nTo date, there are two organisms identified to produce an HDCR, T. kivui and the closely related\nAcetobacterium woodii , and characterization of these enzymes showed that both HDCRs form\nlarge  filaments 2,3 .  In  previous  studies,  filamentation  of A.  woodii HDCR  was  reported  to  be dependent  on  the  presence  of  divalent  cations,  and  in  accordance  with  our  findings,  the\nfilamentous form was the most active state of the enzyme 4 . While similar experiments with the\nT. kivui HDCR did not show a divalent cation dependency on filamentation, it must be noted that\nthe shortened protomers reported in the study of Schuchmann et al . 4 showed an elution profile corresponding to a molecular mass of about 3500 kDa. This indicates the reported enzyme still\nforms roughly a 14-mer (assuming the A. woodii HDCR forms hexamers as in T. kivui ) or a 20-\nmer (tetrameric form of the HDCR) and makes comparison of the depolymerized states of the\ntwo  enzymes  difficult.  However, A.  woodii HDCR  in  the  3500  kDa  state  showed  a  ~30  %\nreduction  in  activity  compared  to  a  similarly  treated  control  of  the  filamentous  form  (&gt;5000\nkDa), and therefore, is consistent with our findings of a stepwise reduction in HDCR activity,\ndepending on the number of connected active centers.\nSupplementary Table 1. Coordination of the [4Fe4S]-clusters in HDCR subunits.\nCys, cysteine; HC, hydrogen cluster.\nSupplementary  Table  2.  Conserved  W-  bis-pterin  guanine  dinucleotide  (W-bisPGD)\ncoordinating  residues. Amino  acids  interacting  with  the  W-bisPGD  cofactor  in  the  closest-\nrelated formate dehydrogenase (DgW-FDH from Desulfovibrio gigas , PDB: 1H0H) are mostly\nconserved in T. kivui FdhF. *, conserved amino acid.\n\n\nK968 K699 *\nSupplementary Table 3. T. kivui HDCR variants used in this study.\n\n\nCys, cysteine; Ala, alanine.\nSupplementary Table 4. Plasmids used in this study. The plasmids used in this study were\neither  used  for  PCR  (20  ng  per  25  µl  reaction)  or  for  transformation  of T.  kivui (1  mg).  C,\ncysteine; A, alanine; aa, amino acids.\npHD028 Overproduction of His 6 -tagged HDCR, HycB4 C114A, C117A, C120A This study pLR002 Plasmid containing HDCR without HydA2, His 6 at hycB4 This study\n\n\nSupplementary Table 5. Primers used in this study.\n\n\nPrimer Sequence (5´-3´)\n\n\nP1 GTGTAATTTTTTTATACAAATAATATTTCAATTCG\n\n\nP2 CCTGTTTACCATCTTTCATACAGTCAATCCTCCTCC\n\n\nP3 GAGGAGGATTGACTGTATGAAAGATGGTAAACAGGAAAAG\nP4 GATTTTTAATGGTGATGGTGATGGTGTACTTTTTTTCTCGGTGTATATTTAG\nP5 AGTACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAGTG\nP6 GGGTTTATCGACCTGCAGC\nP11 ATCCTCTAGAGTCGACCTG\nP12 GGTACCGAGCTCGAATTG\n\n\nP39 TAAGAGTGGTAAGATAAAAATCAAAAATTTTGTTGGTAGTG\n\n\nP40 ATTTTTGATTTTTATCTTACCACTCTTAAAAAACTC\n\n\nP47 GGGAGGAAAATGAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nP48 AATATTAGCAAAAATCATTCATTTTCCTCCCTTTTCCTTTGC\n\n\nP55 TCCAGAAGGAGCAATAAGCGGTGGTG\n\n\nP56 CAAGTCATCACTTCAACG\nP57 TAAGAGTGCTTTATTGGCATGTCCATTTG\nP58 GCGCCTATTGCCAACTTTTCATCTACTACG\n\n\nPLR3 GTAAGACACCATCACCATCACCATTAAAAATCAAAAATTTTGTTGGTAG\n\n\nPLR4 TGATTTTTAATGGTGATGGTGATGGTGTCTTACCACTCTTAAAAAACTC\n\n\nPLR7 TCTATCTATTTGAGATTTTCATGTCTGCAAATAAAGCTA\n\n\nPLR8 TTATTTGCAGACATGAAAATCTCAAATAGATAGAAATATTAGC\n\n\nPLR11b GTATGCACCATCACCATCACCATAAAGATGGTAAACAGGAAAAGGTT\n\n\nPLR12b CTTTATGGTGATGGTGATGGTGCATACAGTCAATCCTCCTCCTTGTA\n\n\nPLR15 AAAGGGAGGAAAATGATTTATTAATTAATAATATTGAAAAGAAAGGGATGTG\n\n\nPLR16 ATATTATTAATTAATAAATCATTTTCCTCCCTTTTCCTTTG\n\n\nPRT7 TAATGATTTTTGCTAATATTTCTATCTATTTGAG\n\n\nPRT8 ACCATCAACCAATTTCAAAGCC\n\n\nPRT9 TAATTTATTAATTAATAATATTGAAAAGAAAGGGATG\nPRT10 CATAACCAAAGTCAACGCATTTTC\n\n\nPseq5 GAATCTTCAAATTCAGGCAATAAGC\nPseq6 TAGCCGACCCAAGAATGAAC\nPseq7 TTCAGCCAAATAGCCGAGAG\nPseq8 AGTAGCGGCTGCTAAAGTTG\nPseq9 AGGTGCGACGGTTGGTACTG\nPseq10 GCTCCGGCTATTAGAGTTTC\nSupplementary Figure 1. Uncropped gel images.\nSupplementary Video 1. Overall presentation of the segmented HDCR cryo-EM density\nand corresponding atomic model.\n\n\nColors are used as in Fig. 1.\nSupplementary Video 2. In situ visualization of bundled HDCR filaments inside a T. kivui\ncell.\nSequential slices back and forth through the cryo-ET volume, followed by reveal and tour of the\ncellular segmentation. Segmentation colored as described in Fig. 5.\n\n\nPage 1\n\n\nPage 2\n\n\nSupplementary References\n\n\nPage 4\n\n\nHycB3 [4Fe4S] I Cys13, Cys16, Cys19, Cys150 [4Fe4S] II Cys23, Cys134, Cys137, Cys146\n\n\nHydA2 [4Fe4S] I Cys15, Cys18, Cys21, Cys55 [4Fe4S] II Cys25, Cys45, Cys48, Cys51 [4Fe4S] (HC) Cys184, Cys239, Cys383, Cys387\n\n\nHDCR_His Δ hdcr strain + HDCR operon hydA2 C-term. This study HDCRΔHydA2 Δ hdcr strain + HDCR operon without hydA2 gene fdhF N-term. This study HDCRΔHycB4 Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. This study HDCRΔHycB3 Δ hdcr strain + HDCR operon without hycB3 gene hydA2 C-term. This study HDCRΔHycB3 ΔHycB4 Δ hdcr strain + HDCR operon without hycB3 and hycB4 genes hydA2 C-term. This study HDCRΔHycB4, His@Hyd &amp;\n\n\nHis@FdhF Δ hdcr strain + HDCR operon without hycB4 gene hydA2 C-term. and fdhF N-term. This study HDCR_HydA2 C387A Δ hdcr strain + HDCR operon, hydA2 _Cys387 substituted with Ala hydA2 C-term. This study HDCR_HycB4 Δ[4Fe4S] IV Δ hdcr strain + HDCR operon, hycB4 _Cys114, Cys117, Cys120\n\n\nsubstituted with Ala hydA2 C-term. This study HDCR_HycB3ΔC Δ hdcr strain + HDCR operon, deletion of hycB3 amino acids 160-184 hydA2 C-term. This study HDCR_HycB4ΔC Δ hdcr strain + HDCR operon, deletion\n\n\nof hycB4 amino acids 190-210 hydA2 C-term. This study\n\n\npHD026 Overproduction of His 6 -tagged HDCR, HydA2 C387A This study\n\n\np. 3 Supplementary Methods p. 4 Supplementary Discussion p. 5\n\n\np. 9\n\n\nSubunit [4Fe4S] -Number Coordinating amino acids FdhF [4Fe4S] I Cys13, Cys16, Cys20, Cys48\n\n\n[4Fe4S] III Cys52, Cys55, Cys61, Cys93 [4Fe4S] IV Cys64, Cys83, Cys86, Cys89 HycB4 [4Fe4S] I Cys34, Cys37, Cys40, Cys180 [4Fe4S] II Cys44, Cys165, Cys168, Cys176 [4Fe4S] III Cys83, Cys86, Cys91, Cys124 [4Fe4S] IV Cys95, Cys114, Cys117, Cys120\n\n\nW-bisPGD coordinating amino acid in DgW-FDH Related amino acid in T. kivui FdhF C20 C16 * C54 C48 * K56 K50 * Q154 C135 CSe158 C139 (*) E197 E179 * N198 C180 N194 N176 * D220 D202 * R222 R204 * G239 G221 * D241 N223 M371 M297 * W373 V299 N518 N403 * Q517 E402 S522 S407 * V542 Q427 N543 D428 K578 K450 * D605 missing T854 T600 * R856 R602 * V857 R603 T858 V604 H860 H606 * Q862 H608 T863 T609 * N951 N682 *\n\n\nStrain Genotype His 6 -tag location Reference Δhdcr hdcr operon deleted none ( 5 )\n\n\nPlasmid Purpose Reference pMU131 Shuttle vector ( 6 )\n\n\npPB5 Cloning vector for HDCR operon with S-layer promoter ( 7 )\n\n\npHD001 Overproduction of His 6 -tagged HDCR This study pHD015 Overproduction of His 6 -tagged HDCRΔHydA2 This study pHD020 Overproduction of His 6 -tagged HDCRΔHycB3 This study pHD024 Overproduction of His 6 -tagged HDCRΔHycB4, His 6 -tag at\n\n\nFdhF and His 6 -tag at HydA2 This study\n\n\npLR002c Plasmid containing HDCR without HydA2, His 6 at hycB4 and fdhF This study pLR003b Overproduction of His 6 -tagged HDCRΔHycB4 This study pLR004 Overproduction of His 6 -tagged HDCRΔHycB3ΔHycB4 This study pRT8 Overproduction of His 6 -tagged HDCR, HycB3Δaa160-184 This study pRT9 Overproduction of His 6 -tagged HDCR, HycB4Δaa190-210 This study\n'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [14.41s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"micro-organism\",\n  \"organism_name\": \"Thermoanaerobacter kivui\",\n  \"cell_strain\": \"Δhdcr\",\n  \"sample_preparation\": \"not present\",\n  \"grid_preparation\": \"not present\",\n  \"cryoet_pixel_spacing\": \"not present\",\n  \"cryoet_acceleration_voltage\": \"not present\",\n  \"microscope_name\": \"not present\",\n  \"microscope_setup_energy_filter\": \"not present\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"not present\",\n  \"camera_model\": \"not present\",\n  \"tilt_minimum\": \"not present\",\n  \"tilt_maximum\": \"not present\",\n  \"total_flux\": \"not present\",\n  \"reconstruction_software\": \"not present\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"not present\",\n  \"annotated_entities\": \"not present\",\n  \"annotation_methods\": \"not present\",\n  \"annotation_software\": \"not present\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"micro-organism\",\n  \"organism_name\": \"Thermoanaerobacter kivui\",\n  \"cell_strain\": \"Δhdcr\",\n  \"sample_preparation\": \"not present\",\n  \"grid_preparation\": \"not present\",\n  \"cryoet_pixel_spacing\": \"not present\",\n  \"cryoet_acceleration_voltage\": \"not present\",\n  \"microscope_name\": \"not present\",\n  \"microscope_setup_energy_filter\": \"not present\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"not present\",\n  \"camera_model\": \"not present\",\n  \"tilt_minimum\": \"not present\",\n  \"tilt_maximum\": \"not present\",\n  \"total_flux\": \"not present\",\n  \"reconstruction_software\": \"not present\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"not present\",\n  \"annotated_entities\": \"not present\",\n  \"annotation_methods\": \"not present\",\n  \"annotation_software\": \"not present\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 248,
      "prompt_tokens": 5030,
      "total_tokens": 5278
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_9e42505611"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "biological_sample_type": "micro-organism",
  "organism_name": "Thermoanaerobacter kivui",
  "cell_strain": "Δhdcr",
  "sample_preparation": "not present",
  "grid_preparation": "not present",
  "cryoet_pixel_spacing": "not present",
  "cryoet_acceleration_voltage": "not present",
  "microscope_name": "not present",
  "microscope_setup_energy_filter": "not present",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "not present",
  "camera_model": "not present",
  "tilt_minimum": "not present",
  "tilt_maximum": "not present",
  "total_flux": "not present",
  "reconstruction_software": "not present",
  "tomogram_size": "not present",
  "reconstruction_method": "not present",
  "annotated_entities": "not present",
  "annotation_methods": "not present",
  "annotation_software": "not present"
}
[chain/end] [1:chain:RunnableSequence] [14.42s] Exiting Chain run with output:
{
  "biological_sample_type": "micro-organism",
  "organism_name": "Thermoanaerobacter kivui",
  "cell_strain": "Δhdcr",
  "sample_preparation": "not present",
  "grid_preparation": "not present",
  "cryoet_pixel_spacing": "not present",
  "cryoet_acceleration_voltage": "not present",
  "microscope_name": "not present",
  "microscope_setup_energy_filter": "not present",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "not present",
  "camera_model": "not present",
  "tilt_minimum": "not present",
  "tilt_maximum": "not present",
  "total_flux": "not present",
  "reconstruction_software": "not present",
  "tomogram_size": "not present",
  "reconstruction_method": "not present",
  "annotated_entities": "not present",
  "annotation_methods": "not present",
  "annotation_software": "not present"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "CILIA AND FLAGELLA\nIn situ architecture of the ciliary base reveals the stepwise assembly of intraflagellar transport trains\nThe cilium is an antenna-like organelle that performs numerous cellular functions, including motility, sensing, and signaling. The base of the cilium contains a selective barrier that regulates the entry of large intraflagellar transport (IFT) trains, which carry cargo proteins required for ciliary assembly and maintenance. However, the native architecture of the ciliary base and the process of IFT train assembly remain unresolved. In this work, we used in situ cryo – electron tomography to reveal native structures of the transition zone region and assembling IFT trains at the ciliary base in Chlamydomonas . We combined this direct cellular visualization with ultrastructure expansion microscopy to describe the front-to-back stepwise assembly of IFT trains: IFT-B forms the backbone, onto which bind IFT-A, dynein-1b, and finally kinesin-2 before entry into the cilium.\nC ilia (and closely related flagella) are evo- lutionarily conserved eukaryotic organ- elles that extend from the cell surface and are involved in a wide variety of functions, including cell motility, fluid flow generation, sensing, and signaling ( 1 , 2 ). The cilium consists of an axoneme of micro- tubule doublets (MTDs), which extend from the microtubule triplets (MTTs) of the centri- ole (also called a basal body) and are sheathed in a ciliary membrane. At the base of the cilium, a specialized region known as the tran- sition zone (TZ) gates entry and exit of both membrane-bound and soluble ciliary proteins ( 3 – 5 ). Among its many gating functions, the TZ is believed to regulate entry of intra- flagellar transport (IFT), bidirectional traf- fic between the ciliary base and tip driven by kinesin and dynein motors ( 6 , 7 ). IFT is required for assembly and maintenance of the cilium, as well as for mediating many of its signaling functions ( 8 ). A pool of IFT pro- teins and motors resides at the ciliary base for several seconds before entry into the cilium ( 9 , 10 ). However, it remains to be characterized how these proteins assemble into the elaborate multi-megadalton IFT trains observed running along the ciliary axoneme ( 11 ). In this study,\nwe combined in situ cryo – electron tomogra- phy (cryo-ET) ( 12 ) with ultrastructure expan- sion microscopy (U-ExM) ( 13 ) of the green alga Chlamydomonas reinhardtii , a classic model for cilia research, to reveal native TZ structures and stepwise assembly of IFT trains at the ciliary base.\nVitrified Chlamydomonas cells were thinned with a focused ion beam (FIB) ( 14 ) then imaged by cryo-ET, revealing the ciliary base inside the native cellular environment (figs. S1 and S2). Focusing on the TZ, we observed several dif- ferent structures attached to the MTDs (Fig. 1, A and B), which we resolved in molecular de- tail by subtomogram averaging (Fig. 1, C to H, and figs. S3 and S4). The proximal ~180 nm of the TZ is occupied by peripheral Y-links (Fig. 1, turquoise) and luminal stellate fibers (Fig. 1, purple, and fig. S2), which resemble a nine- pointed star in cross section ( 15 ). Our structure reveals that the stellates form a helical cylinder with a six-start helix and a pitch of 49.2 nm (Fig. 1H). Notably, this matches the pitch of the inner scaffold, a three-start helical cylin- der in the lumen of the centriole ( 16 ). The nine points of the stellate star bind MTDs at protofilament A3, with a longitudinal perio- dicity of ~8.1 nm along each MTD (Fig. 1F and fig. S5), which matches the 8.2 nm length of one tubulin dimer.\nY-links are present in many species. They connect MTDs to the ciliary membrane and help gate transport into and out of the cilium ( 17 – 20 ). Our structure reveals a broad density attaching the Y-links to the MTD, spanning protofilaments A9, A10, and B1 to B4, with a longitudinal repeat of ~8.3 nm (Fig. 1F and fig. S5). However, the outer densities of the Y-links that connect to the ciliary membrane were not resolved, likely owing to flexibility of these filamentous structures. We observed that the Y-links extend along the MTDs prox-\nimal of the ciliary membrane (Fig. 1A). This suggests that connection to the membrane is not a prerequisite for MTD Y-link decoration.\nImmediately distal to the stellates and Y-links, we found a distinct helical density completely decorating the MTDs (Fig. 1, dark blue). This helical “ sleeve ” spans 76 ± 6 nm along the MTDs, with a periodicity of ~16.4 nm (fig. S5). Cryo-ET and U-ExM immediately after ciliary shedding (figs. S6 and S7) revealed that the MTD sleeve overlaps with the ciliary cleavage site, also known as the site of flagellar auto- tomy (SOFA) ( 21 ) (Fig. 1B). Consistent with this finding, the MTD sleeve was absent from the severed ends of MTDs both on the TZ (fig. S6, C and D) and isolated cilia ( 22 ). We therefore hypothesize that this structure might help reg- ulate axoneme severing and is lost during the process. Because the sleeve density covers every MTD protofilament, it should sterically hinder the attachment of molecular motors and thus may also play a role in regulating IFT entry or exit. The sleeve does not appear to be present on the TZ of motile cilia from bovine trachea ( 23 ). It has been known for decades that a pool of IFT proteins is localized near the base of the cilium ( 24 , 25 ), but the structural organization of this basal pool has remained a mystery. In our tomograms, we observed filamentous strings of particles with one end attached to the TZ and the other end splayed into the cytosol, passing between the transition fibers (Fig. 2A and fig. S8). These strings consisted of three layers, each with a different shape and periodicity (Fig. 2, B and C, and fig. S9). We performed subtomogram aver- aging of particles along each layer and combined them to produce a composite molecular struc- ture (Fig. 2D and fig. S10). Comparison to a previously published cryo-ET structure of mature anterograde IFT trains within the cilium ( 11 ) confirmed that the cytosolic strings are indeed IFT trains. Our observed cytosolic train struc- ture was similar to the anterograde train struc- ture (fig. S11), enabling us to assign densities to IFT-B complexes, IFT-A complexes, and dynein-1b motors, which have longitudinal periodicities of ∼ 6, ∼ 11, and ∼ 18 nm, respectively (fig. S12). As was shown for anterograde IFT, we observed that these cytosolic trains were attached to the MTD B-tubule (fig. S13D) ( 26 ) and that dynein-1b was loaded onto the trains as a cargo in an autoinhibited state ( 11 , 27 ). Thus, much of the IFT pool at the ciliary base consists of TZ- tethered anterograde IFT trains that are under- going assembly before entry into the cilium. We did not see retrograde trains at the ciliary base, which implies that they rapidly disassemble, perhaps even before exiting the TZ.\nWe observed two notable differences be- tween assembling and mature IFT trains. First, assembling trains are flexible and display re- gions of high curvature (Fig. 2, A to C, and fig. S9), whereas trains in the cilium have an extended straight conformation (fig. S12) ( 11 ),\nwhich is likely maintained by interactions with cargos and the ciliary membrane. Second, as- sembling IFT-B has a prominent extra density on the side opposite IFT-A (Fig. 2D and fig. S11; red asterisks). In regions where IFT trains are bound to the TZ, this extra density is positioned adjacent to the MTD. The density is not part of the MTD structure itself, because it is also\npresent on IFT-B in the sections of trains that dangle into the cytosol (fig. S13).\nIn situ cryo-ET combines native molecular structures with precise cellular localization, enabling us to plot the spatial relationship between assembling IFT trains and the TZ (Fig. 3, A and B). The front segments of IFT trains attach to MTDs (but never to the MTTs of the centriole) and then continue in the dis- tal direction along these microtubule tracks, while the back regions curve away from the TZ into the cytosol. The front ends of IFT trains were frequently observed at the Y-links, but only very rarely further distal at the MTD sleeve (fig. S9). By mapping the IFT-B, IFT-A, and dynein-1b structures along the IFT trains\n\n\nFig. 1. Cryo-ET structure of the ciliary transition zone within native\nChlamydomonas cells. ( A ) 2D slice through a cryo – electron tomogram, showing a TZ in longitudinal cross section. Stellate fibers, purple; Y-links, turquoise; MTD helical sleeve, dark blue; stellate plate, white asterisk (see fig. S2); and IFT, yellow (see Fig. 2). Scale bar: 100 nm. ( B ) Position along the TZ occupied by different structures. Distances are measured relative to the stellate plate (see fig. S6), defined as the “ 0 nm ” origin point. SOFA, site of flagellar autotomy, where the cilium is cleaved (see figs. S6 and S7). CP, central pair. Error bars indicate standard deviation. ( C to E ) Composite model of the TZ, combining averages of the stellate (purple), Y-links (turquoise), MTD sleeve (dark blue), and associated MTD (light blue), with schematic renderings of MTTs (dark gray), MTDs (light gray), and the CP (white). (C) Longitudinal section view of the complete composite model, assembled according to the measured\nlengths and positions of each component, with 21 Y-link repeats, 21 stellate repeats (7 proximal of the plate, 14 distal), and 5 MTD sleeve repeats. The model shows straight MTTs and MTDs, but, as seen in (A) and quantified in ( 16 ), the centriole is actually a slightly convex barrel. [(D) and (E)] Cross-sectional views through the indicated regions of the composite model, showing (D) MTDs encased in the helical sleeve, with the CP in the middle, and (E) the nine-pointed stellate cylinder attached to MTDs decorated with Y-links. ( F and G ) Side and cross-sectional views of a single MTD attached to (F) stellate fibers and Y-links and (G) the helical sleeve. Protofilaments of the A- and B-microtubules are numbered. ( H ) Unrolled composite model, viewed from the center of the TZ, looking outward toward the inner wall of the stellate cylinder. One continuous helical density of the six-start stellate helix is marked in pink (pitch: 49.2 nm). MTDs are numbered. See also movie S1.\n(as shown in Fig. 2A), we visualized the spa- tially defined order of train assembly. Anter- ograde trains always contain IFT-B, which forms a backbone scaffold upon which other components are attached. This train architec- ture is consistent with previous findings that IFT-B recruits IFT-A into cilia ( 28 , 29 ). We plotted the train assembly state relative to each IFT-B subunit, and although we observed variability between individual trains (Fig. 3A), the cumulative plot shows a clear distribution of IFT train components (Fig. 3B). The back seg-\nments of trains were in a less assembled state, with IFT-B extending the farthest, followed by IFT-A and then dynein-1b (we did note some assembling trains where dynein-1b extended far- ther than IFT-A). No gaps were observed in the regions occupied by IFT-A and dynein-1b, sug- gesting that these components linearly oligomer- ize from front to back on the IFT-B scaffold.\nNext, we compared the assembling cytosolic trains to mature anterograde trains within the cilium (Fig. 3, C and D). Mature trains ex- hibited a continuum of lengths from 223 to\n558 nm, comprising 36 to 90 IFT-B subunits with an average of 62 ± 16 IFT-B. In com- parison, the longest assembling train we ob- served contained 88 IFT-B subunits, and the average length was 44 ± 17 IFT-B. Thus, just one IFT train is assembled at a time per MTD. Cytosolic and ciliary IFT trains have similarly incomplete front ends, whereas the incomplete segment at the back of cytosolic trains is longer, as these regions are still undergoing assembly (Fig. 3D). Perhaps as IFT trains enter the cilium, part of the incompletely assembled\nFig. 2. Cryo-ET structures of assembling IFT trains attached to the transition zone. ( A ) Composite structures of assembling IFT trains (IFT-B, yellow; IFT-A, orange; and dynein-1b, red) engaging the ciliary TZ (colored as in Fig. 1). A schematic membrane has been added (gray double lines). Subtomogram averages have been mapped back into their positions inside a representative tomogram. (Inset) Rotated top view of the rightmost IFT train from (A). ( B and C ) 2D slices through tomograms, showing assembling IFT trains, and ( B ′ and C ′ ) overlaid 3D train structures, with their front ends attached to the TZ. (Inset) Enlarged view\nshowing the double AAA+ adenosine triphosphatase rings of the dynein-1b motors. Arrowhead colors correspond to (A): MTT/MTD, gray, and membrane, white. Scale bars: 100 nm in (B) and (C), 20 nm in the inset. ( D ) Different views of the cytosolic IFT train composite subtomogram average. Colored as in (A). In the lower right, IFT-A was removed to show details on IFT-B. Red asterisks indicate the extra IFT-B density, which was not observed on mature trains inside cilia (see fig. S11). Assembling trains were only found attached to the TZ, not in the surrounding cytosol farther from the ciliary base (see fig. S1). See also movie S2.\nback end is left behind at the TZ to nucleate the front of the next train (Fig. 4K). Because as- sembly of IFT-A and dynein-1b proceeds front to back, the front end of the new train would remain incomplete, as we see in mature anter- ograde trains inside the cilium (Fig. 3C).\nThe kinesin-2 anterograde IFT motor is dif- ficult to identify by cryo-ET owing to its rela- tively small size and flexible structure. We thus turned to U-ExM to examine the occupancy of kinesin-2 on assembling trains. We imaged components of kinesin-2, dynein-1b, IFT-B (subcomplexes B1 and B2), and IFT-A (Fig. 4,\nA to G, and fig. S14). Consistent with our cryo- ET data, all of these proteins formed filamen- tous strings at the ciliary base, with their distal ends in close proximity to the TZ (marked by centrin staining and the lack of glutamylated tubulin) (figs. S15 and S16) and their prox- imal ends curving into the cytoplasm. Using semiautomated tracing, we plotted the three- dimensional (3D) trajectories of each protein (fig. S15E). IFT-B traces were longer than traces of IFT-A and dynein-1b (Fig. 4H). Kinesin-2 traces were the shortest and were localized at the front of IFT trains close to the TZ (fig.\nS16). Double-labeling of individual IFT trains confirmed that IFT-A extends farther than kinesin-2 toward the back of the train, and IFT-B extends farther than IFT-A (Fig. 4, I and J, and fig. S17).\nWe observed ninefold occupancy of IFT around the majority of TZs (Fig. 4, F and G, and fig. S18). Measured by U-ExM, cryo-ExM, and cryo-ET, 93 to 96% of MTDs were bound by an assembling IFT train. Given that trains enter the cilium with a frequency of ~1 train/s ( 28 , 30 ), we predict that trains linger at the ciliary base for an average of 9 s during\nFig. 3. IFT trains undergo stepwise assembly from front to back. ( A and B ) Assembly states of IFT trains relative to structures of the TZ [shown at the bottom of (B)]. Distances are measured by the number of IFT-B subunits relative to the onset of the Y-links (defined as the “ 0 IFT-B ” origin point). (A) Position and train assembly state, measured for each IFT-B subunit of each assembling IFT train. Assembly states [color code in (A) applies to all panels] were determined on the basis of occupancy of IFT-A and/or dynein-1b on the IFT-B backbone. Black squares indicate train attachment points, that is, the most proximal IFT-B subunit that is close enough to the MTD (&lt;20 nm) to be considered bound to the microtubule. The three trains marked with black asterisks lack a complete “ middle ” (no dynein is bound) and thus were omitted from analysis in (D) because “ back-end B + A ” could not be split from “ front-end B + A. ” Of the 70 trains in our dataset, 30 had front ends cropped by the FIB milling and therefore were not quantified, as their starting point relative to the TZ could not be determined. Eight more trains had cropped back ends (indicated\nwith a red “ x ” ) but were included in the analysis. (B) Cumulative plot of all assembling IFT trains in (A), with positions in relation to the TZ (shown at bottom). Colored curves above the line show the summed abundance of each IFT assembly state. The black curve below the line shows the distribution of MTD attachment points, summed from the black squares in (A). ( C ) Distribution of IFT train lengths and completeness of assembly [colored as in (A)] for mature anterograde trains found in the cilium ( 11 ). ( D ) Comparison of the abundance of assembly states at the front and back ends of assembling ( “ A ” ) versus mature ( “ M ” ) trains [i.e., trains in (A) versus trains in (C)]. Box, median and 25 to 75% percentiles; X, mean; whiskers, 1.5× interquartile range; points, outliers. Statistical significance assessed by unpaired t test (**** P &lt; 0.0001; ** P = 0.002; * P = 0.03; ns, not significant, P &gt; 0.05). Because the backs of some assembling trains in this analysis were clipped by the FIB milling [ “ x ” in (A)], the “ IFT-B only ” region of assembling trains should be slightly more pronounced than the quantification shown here.\nassembly. Consistent with this hypothesis, photobleaching analysis of the Chlamydomonas basal pool showed that several green fluorescent protein (GFP) – tagged IFT proteins require ~9 s to recover peak fluorescence ( 9 ). Regardless, ciliary entry appears to be stochastic rather than sequential, with an avalanche-like relationship between lag time and IFT train size ( 31 ). Our combined cryo-ET and U-ExM mea- surements support a spatially defined sequence of IFT train assembly: IFT-B first forms the\nbackbone, which scaffolds the front-to-back attachment of IFT-A, dynein-1b, and finally kinesin-2 close to the TZ (Fig. 4K). A major remaining question is how IFT entry into the cilium is regulated. What prevents the front end of the train from entering the cilium as assembly proceeds toward the incomplete back of the train? Possible mechanisms may include controlled loading of kinesin-2 by factors at the TZ, the sleeve serving as a barrier to prevent entry of small trains, or the extra IFT-B – associated density acting as a molecular brake until train assembly is completed.\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T Downloaded from https://www.science.org on February 23, 2024\n\n\nHugo van den Hoek 1,2,3 † , Nikolai Klena 4,5 † , Mareike A. Jordan 6 , Gonzalo Alvarez Viar 5,6 , Ricardo D. Righetto 1,2 , Miroslava Schaffer 3 , Philipp S. Erdmann 5 , William Wan 7 , Stefan Geimer 8 , Jürgen M. Plitzko 3 , Wolfgang Baumeister 3 , Gaia Pigino 5,6 * , Virginie Hamel 4 * , Paul Guichard 4 * , Benjamin D. Engel 1,2 *\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 2 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 3 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 4 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 5 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\n7. K. G. Kozminski, K. A. Johnson, P. Forscher, J. L. Rosenbaum, Proc. Natl. Acad. Sci. U.S.A. 90 , 5519 – 5523 (1993). 8. H. Ishikawa, W. F. Marshall, Nat. Rev. Mol. Cell Biol. 12 , 222 – 234 (2011). 9. J. L. Wingfield et al ., eLife 6 , e26609 (2017). 10. J. V. K. Hibbard, N. Vazquez, R. Satija, J. B. Wallingford, Mol. Biol. Cell 32 , 1171 – 1180 (2021). 11. M. A. Jordan, D. R. Diener, L. Stepanek, G. Pigino, Nat. Cell Biol. 20 , 1250 – 1255 (2018). 12. S. Asano, B. D. Engel, W. Baumeister, J. Mol. Biol. 428 , 332 – 343 (2016). 13. D. Gambarotto et al ., Nat. Methods 16 , 71 – 74 (2019). 14. M. Schaffer et al ., J. Struct. Biol. 197 , 73 – 82 (2017). 15. S. Geimer, M. Melkonian, J. Cell Sci. 117 , 2663 – 2674 (2004). 16. M. Le Guennec et al ., Sci. Adv. 6 , eaaz4137 (2020). 17. B. Craige et al ., J. Cell Biol. 190 , 927 – 940 (2010). 18. C. L. Williams et al ., J. Cell Biol. 192 , 1023 – 1041 (2011). 19. F. R. Garcia-Gonzalo, J. F. Reiter, Cold Spring Harb. Perspect. Biol. 9 , a028134 (2017). 20. A. R. G. De-Castro et al ., J. Cell Biol. 221 , e202010178 (2022). 21. L. M. Quarmby, in The Chlamydomonas Sourcebook ( Second Edition ) , Volume 3 , E. H. Harris, D. B. Stern, G. B. Witman, Eds. (Academic Press, 2009), pp. 43 – 69.\n\n\n22. K. H. Bui, T. Yagi, R. Yamamoto, R. Kamiya, T. Ishikawa, J. Cell Biol. 198 , 913 – 925 (2012). 23. G. A. Greenan, R. D. Vale, D. A. Agard, J. Cell Biol. 219 , e201907060 (2020). 24. D. G. Cole et al ., J. Cell Biol. 141 , 993 – 1008 (1998). 25. Y. Hou et al ., J. Cell Biol. 176 , 653 – 665 (2007). 26. L. Stepanek, G. Pigino, Science 352 , 721 – 724 (2016). 27. K. Toropova et al ., Nat. Struct. Mol. Biol. 26 , 823 – 829 (2019). 28. C. Iomini, V. Babaev-Khaimov, M. Sassaroli, G. Piperno, J. Cell Biol. 153 , 13 – 24 (2001). 29. J. M. Brown, D. A. Cochran, B. Craige, T. Kubo, G. B. Witman, Curr. Biol. 25 , 1583 – 1593 (2015). 30. B. D. Engel, W. B. Ludington, W. F. Marshall, J. Cell Biol. 187 , 81 – 89 (2009).\n\n\n31. W. B. Ludington, K. A. Wemmer, K. F. Lechtreck, G. B. Witman, W. F. Marshall, Proc. Natl. Acad. Sci. U.S.A. 110 , 3925 – 3930 (2013).\n\n\nSUPPLEMENTARY MATERIALS science.org/doi/10.1126/science.abm6704 Materials and Methods Figs. S1 to S18 Tables S1 and S2 References ( 32 – 64 ) MDAR Reproducibility Checklist Movies S1 and S2 View/request a protocol for this paper from Bio-protocol .\n",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "CILIA AND FLAGELLA\nIn situ architecture of the ciliary base reveals the stepwise assembly of intraflagellar transport trains\nThe cilium is an antenna-like organelle that performs numerous cellular functions, including motility, sensing, and signaling. The base of the cilium contains a selective barrier that regulates the entry of large intraflagellar transport (IFT) trains, which carry cargo proteins required for ciliary assembly and maintenance. However, the native architecture of the ciliary base and the process of IFT train assembly remain unresolved. In this work, we used in situ cryo – electron tomography to reveal native structures of the transition zone region and assembling IFT trains at the ciliary base in Chlamydomonas . We combined this direct cellular visualization with ultrastructure expansion microscopy to describe the front-to-back stepwise assembly of IFT trains: IFT-B forms the backbone, onto which bind IFT-A, dynein-1b, and finally kinesin-2 before entry into the cilium.\nC ilia (and closely related flagella) are evo- lutionarily conserved eukaryotic organ- elles that extend from the cell surface and are involved in a wide variety of functions, including cell motility, fluid flow generation, sensing, and signaling ( 1 , 2 ). The cilium consists of an axoneme of micro- tubule doublets (MTDs), which extend from the microtubule triplets (MTTs) of the centri- ole (also called a basal body) and are sheathed in a ciliary membrane. At the base of the cilium, a specialized region known as the tran- sition zone (TZ) gates entry and exit of both membrane-bound and soluble ciliary proteins ( 3 – 5 ). Among its many gating functions, the TZ is believed to regulate entry of intra- flagellar transport (IFT), bidirectional traf- fic between the ciliary base and tip driven by kinesin and dynein motors ( 6 , 7 ). IFT is required for assembly and maintenance of the cilium, as well as for mediating many of its signaling functions ( 8 ). A pool of IFT pro- teins and motors resides at the ciliary base for several seconds before entry into the cilium ( 9 , 10 ). However, it remains to be characterized how these proteins assemble into the elaborate multi-megadalton IFT trains observed running along the ciliary axoneme ( 11 ). In this study,\nwe combined in situ cryo – electron tomogra- phy (cryo-ET) ( 12 ) with ultrastructure expan- sion microscopy (U-ExM) ( 13 ) of the green alga Chlamydomonas reinhardtii , a classic model for cilia research, to reveal native TZ structures and stepwise assembly of IFT trains at the ciliary base.\nVitrified Chlamydomonas cells were thinned with a focused ion beam (FIB) ( 14 ) then imaged by cryo-ET, revealing the ciliary base inside the native cellular environment (figs. S1 and S2). Focusing on the TZ, we observed several dif- ferent structures attached to the MTDs (Fig. 1, A and B), which we resolved in molecular de- tail by subtomogram averaging (Fig. 1, C to H, and figs. S3 and S4). The proximal ~180 nm of the TZ is occupied by peripheral Y-links (Fig. 1, turquoise) and luminal stellate fibers (Fig. 1, purple, and fig. S2), which resemble a nine- pointed star in cross section ( 15 ). Our structure reveals that the stellates form a helical cylinder with a six-start helix and a pitch of 49.2 nm (Fig. 1H). Notably, this matches the pitch of the inner scaffold, a three-start helical cylin- der in the lumen of the centriole ( 16 ). The nine points of the stellate star bind MTDs at protofilament A3, with a longitudinal perio- dicity of ~8.1 nm along each MTD (Fig. 1F and fig. S5), which matches the 8.2 nm length of one tubulin dimer.\nY-links are present in many species. They connect MTDs to the ciliary membrane and help gate transport into and out of the cilium ( 17 – 20 ). Our structure reveals a broad density attaching the Y-links to the MTD, spanning protofilaments A9, A10, and B1 to B4, with a longitudinal repeat of ~8.3 nm (Fig. 1F and fig. S5). However, the outer densities of the Y-links that connect to the ciliary membrane were not resolved, likely owing to flexibility of these filamentous structures. We observed that the Y-links extend along the MTDs prox-\nimal of the ciliary membrane (Fig. 1A). This suggests that connection to the membrane is not a prerequisite for MTD Y-link decoration.\nImmediately distal to the stellates and Y-links, we found a distinct helical density completely decorating the MTDs (Fig. 1, dark blue). This helical “ sleeve ” spans 76 ± 6 nm along the MTDs, with a periodicity of ~16.4 nm (fig. S5). Cryo-ET and U-ExM immediately after ciliary shedding (figs. S6 and S7) revealed that the MTD sleeve overlaps with the ciliary cleavage site, also known as the site of flagellar auto- tomy (SOFA) ( 21 ) (Fig. 1B). Consistent with this finding, the MTD sleeve was absent from the severed ends of MTDs both on the TZ (fig. S6, C and D) and isolated cilia ( 22 ). We therefore hypothesize that this structure might help reg- ulate axoneme severing and is lost during the process. Because the sleeve density covers every MTD protofilament, it should sterically hinder the attachment of molecular motors and thus may also play a role in regulating IFT entry or exit. The sleeve does not appear to be present on the TZ of motile cilia from bovine trachea ( 23 ). It has been known for decades that a pool of IFT proteins is localized near the base of the cilium ( 24 , 25 ), but the structural organization of this basal pool has remained a mystery. In our tomograms, we observed filamentous strings of particles with one end attached to the TZ and the other end splayed into the cytosol, passing between the transition fibers (Fig. 2A and fig. S8). These strings consisted of three layers, each with a different shape and periodicity (Fig. 2, B and C, and fig. S9). We performed subtomogram aver- aging of particles along each layer and combined them to produce a composite molecular struc- ture (Fig. 2D and fig. S10). Comparison to a previously published cryo-ET structure of mature anterograde IFT trains within the cilium ( 11 ) confirmed that the cytosolic strings are indeed IFT trains. Our observed cytosolic train struc- ture was similar to the anterograde train struc- ture (fig. S11), enabling us to assign densities to IFT-B complexes, IFT-A complexes, and dynein-1b motors, which have longitudinal periodicities of ∼ 6, ∼ 11, and ∼ 18 nm, respectively (fig. S12). As was shown for anterograde IFT, we observed that these cytosolic trains were attached to the MTD B-tubule (fig. S13D) ( 26 ) and that dynein-1b was loaded onto the trains as a cargo in an autoinhibited state ( 11 , 27 ). Thus, much of the IFT pool at the ciliary base consists of TZ- tethered anterograde IFT trains that are under- going assembly before entry into the cilium. We did not see retrograde trains at the ciliary base, which implies that they rapidly disassemble, perhaps even before exiting the TZ.\nWe observed two notable differences be- tween assembling and mature IFT trains. First, assembling trains are flexible and display re- gions of high curvature (Fig. 2, A to C, and fig. S9), whereas trains in the cilium have an extended straight conformation (fig. S12) ( 11 ),\nwhich is likely maintained by interactions with cargos and the ciliary membrane. Second, as- sembling IFT-B has a prominent extra density on the side opposite IFT-A (Fig. 2D and fig. S11; red asterisks). In regions where IFT trains are bound to the TZ, this extra density is positioned adjacent to the MTD. The density is not part of the MTD structure itself, because it is also\npresent on IFT-B in the sections of trains that dangle into the cytosol (fig. S13).\nIn situ cryo-ET combines native molecular structures with precise cellular localization, enabling us to plot the spatial relationship between assembling IFT trains and the TZ (Fig. 3, A and B). The front segments of IFT trains attach to MTDs (but never to the MTTs of the centriole) and then continue in the dis- tal direction along these microtubule tracks, while the back regions curve away from the TZ into the cytosol. The front ends of IFT trains were frequently observed at the Y-links, but only very rarely further distal at the MTD sleeve (fig. S9). By mapping the IFT-B, IFT-A, and dynein-1b structures along the IFT trains\n\n\nFig. 1. Cryo-ET structure of the ciliary transition zone within native\nChlamydomonas cells. ( A ) 2D slice through a cryo – electron tomogram, showing a TZ in longitudinal cross section. Stellate fibers, purple; Y-links, turquoise; MTD helical sleeve, dark blue; stellate plate, white asterisk (see fig. S2); and IFT, yellow (see Fig. 2). Scale bar: 100 nm. ( B ) Position along the TZ occupied by different structures. Distances are measured relative to the stellate plate (see fig. S6), defined as the “ 0 nm ” origin point. SOFA, site of flagellar autotomy, where the cilium is cleaved (see figs. S6 and S7). CP, central pair. Error bars indicate standard deviation. ( C to E ) Composite model of the TZ, combining averages of the stellate (purple), Y-links (turquoise), MTD sleeve (dark blue), and associated MTD (light blue), with schematic renderings of MTTs (dark gray), MTDs (light gray), and the CP (white). (C) Longitudinal section view of the complete composite model, assembled according to the measured\nlengths and positions of each component, with 21 Y-link repeats, 21 stellate repeats (7 proximal of the plate, 14 distal), and 5 MTD sleeve repeats. The model shows straight MTTs and MTDs, but, as seen in (A) and quantified in ( 16 ), the centriole is actually a slightly convex barrel. [(D) and (E)] Cross-sectional views through the indicated regions of the composite model, showing (D) MTDs encased in the helical sleeve, with the CP in the middle, and (E) the nine-pointed stellate cylinder attached to MTDs decorated with Y-links. ( F and G ) Side and cross-sectional views of a single MTD attached to (F) stellate fibers and Y-links and (G) the helical sleeve. Protofilaments of the A- and B-microtubules are numbered. ( H ) Unrolled composite model, viewed from the center of the TZ, looking outward toward the inner wall of the stellate cylinder. One continuous helical density of the six-start stellate helix is marked in pink (pitch: 49.2 nm). MTDs are numbered. See also movie S1.\n(as shown in Fig. 2A), we visualized the spa- tially defined order of train assembly. Anter- ograde trains always contain IFT-B, which forms a backbone scaffold upon which other components are attached. This train architec- ture is consistent with previous findings that IFT-B recruits IFT-A into cilia ( 28 , 29 ). We plotted the train assembly state relative to each IFT-B subunit, and although we observed variability between individual trains (Fig. 3A), the cumulative plot shows a clear distribution of IFT train components (Fig. 3B). The back seg-\nments of trains were in a less assembled state, with IFT-B extending the farthest, followed by IFT-A and then dynein-1b (we did note some assembling trains where dynein-1b extended far- ther than IFT-A). No gaps were observed in the regions occupied by IFT-A and dynein-1b, sug- gesting that these components linearly oligomer- ize from front to back on the IFT-B scaffold.\nNext, we compared the assembling cytosolic trains to mature anterograde trains within the cilium (Fig. 3, C and D). Mature trains ex- hibited a continuum of lengths from 223 to\n558 nm, comprising 36 to 90 IFT-B subunits with an average of 62 ± 16 IFT-B. In com- parison, the longest assembling train we ob- served contained 88 IFT-B subunits, and the average length was 44 ± 17 IFT-B. Thus, just one IFT train is assembled at a time per MTD. Cytosolic and ciliary IFT trains have similarly incomplete front ends, whereas the incomplete segment at the back of cytosolic trains is longer, as these regions are still undergoing assembly (Fig. 3D). Perhaps as IFT trains enter the cilium, part of the incompletely assembled\nFig. 2. Cryo-ET structures of assembling IFT trains attached to the transition zone. ( A ) Composite structures of assembling IFT trains (IFT-B, yellow; IFT-A, orange; and dynein-1b, red) engaging the ciliary TZ (colored as in Fig. 1). A schematic membrane has been added (gray double lines). Subtomogram averages have been mapped back into their positions inside a representative tomogram. (Inset) Rotated top view of the rightmost IFT train from (A). ( B and C ) 2D slices through tomograms, showing assembling IFT trains, and ( B ′ and C ′ ) overlaid 3D train structures, with their front ends attached to the TZ. (Inset) Enlarged view\nshowing the double AAA+ adenosine triphosphatase rings of the dynein-1b motors. Arrowhead colors correspond to (A): MTT/MTD, gray, and membrane, white. Scale bars: 100 nm in (B) and (C), 20 nm in the inset. ( D ) Different views of the cytosolic IFT train composite subtomogram average. Colored as in (A). In the lower right, IFT-A was removed to show details on IFT-B. Red asterisks indicate the extra IFT-B density, which was not observed on mature trains inside cilia (see fig. S11). Assembling trains were only found attached to the TZ, not in the surrounding cytosol farther from the ciliary base (see fig. S1). See also movie S2.\nback end is left behind at the TZ to nucleate the front of the next train (Fig. 4K). Because as- sembly of IFT-A and dynein-1b proceeds front to back, the front end of the new train would remain incomplete, as we see in mature anter- ograde trains inside the cilium (Fig. 3C).\nThe kinesin-2 anterograde IFT motor is dif- ficult to identify by cryo-ET owing to its rela- tively small size and flexible structure. We thus turned to U-ExM to examine the occupancy of kinesin-2 on assembling trains. We imaged components of kinesin-2, dynein-1b, IFT-B (subcomplexes B1 and B2), and IFT-A (Fig. 4,\nA to G, and fig. S14). Consistent with our cryo- ET data, all of these proteins formed filamen- tous strings at the ciliary base, with their distal ends in close proximity to the TZ (marked by centrin staining and the lack of glutamylated tubulin) (figs. S15 and S16) and their prox- imal ends curving into the cytoplasm. Using semiautomated tracing, we plotted the three- dimensional (3D) trajectories of each protein (fig. S15E). IFT-B traces were longer than traces of IFT-A and dynein-1b (Fig. 4H). Kinesin-2 traces were the shortest and were localized at the front of IFT trains close to the TZ (fig.\nS16). Double-labeling of individual IFT trains confirmed that IFT-A extends farther than kinesin-2 toward the back of the train, and IFT-B extends farther than IFT-A (Fig. 4, I and J, and fig. S17).\nWe observed ninefold occupancy of IFT around the majority of TZs (Fig. 4, F and G, and fig. S18). Measured by U-ExM, cryo-ExM, and cryo-ET, 93 to 96% of MTDs were bound by an assembling IFT train. Given that trains enter the cilium with a frequency of ~1 train/s ( 28 , 30 ), we predict that trains linger at the ciliary base for an average of 9 s during\nFig. 3. IFT trains undergo stepwise assembly from front to back. ( A and B ) Assembly states of IFT trains relative to structures of the TZ [shown at the bottom of (B)]. Distances are measured by the number of IFT-B subunits relative to the onset of the Y-links (defined as the “ 0 IFT-B ” origin point). (A) Position and train assembly state, measured for each IFT-B subunit of each assembling IFT train. Assembly states [color code in (A) applies to all panels] were determined on the basis of occupancy of IFT-A and/or dynein-1b on the IFT-B backbone. Black squares indicate train attachment points, that is, the most proximal IFT-B subunit that is close enough to the MTD (&lt;20 nm) to be considered bound to the microtubule. The three trains marked with black asterisks lack a complete “ middle ” (no dynein is bound) and thus were omitted from analysis in (D) because “ back-end B + A ” could not be split from “ front-end B + A. ” Of the 70 trains in our dataset, 30 had front ends cropped by the FIB milling and therefore were not quantified, as their starting point relative to the TZ could not be determined. Eight more trains had cropped back ends (indicated\nwith a red “ x ” ) but were included in the analysis. (B) Cumulative plot of all assembling IFT trains in (A), with positions in relation to the TZ (shown at bottom). Colored curves above the line show the summed abundance of each IFT assembly state. The black curve below the line shows the distribution of MTD attachment points, summed from the black squares in (A). ( C ) Distribution of IFT train lengths and completeness of assembly [colored as in (A)] for mature anterograde trains found in the cilium ( 11 ). ( D ) Comparison of the abundance of assembly states at the front and back ends of assembling ( “ A ” ) versus mature ( “ M ” ) trains [i.e., trains in (A) versus trains in (C)]. Box, median and 25 to 75% percentiles; X, mean; whiskers, 1.5× interquartile range; points, outliers. Statistical significance assessed by unpaired t test (**** P &lt; 0.0001; ** P = 0.002; * P = 0.03; ns, not significant, P &gt; 0.05). Because the backs of some assembling trains in this analysis were clipped by the FIB milling [ “ x ” in (A)], the “ IFT-B only ” region of assembling trains should be slightly more pronounced than the quantification shown here.\nassembly. Consistent with this hypothesis, photobleaching analysis of the Chlamydomonas basal pool showed that several green fluorescent protein (GFP) – tagged IFT proteins require ~9 s to recover peak fluorescence ( 9 ). Regardless, ciliary entry appears to be stochastic rather than sequential, with an avalanche-like relationship between lag time and IFT train size ( 31 ). Our combined cryo-ET and U-ExM mea- surements support a spatially defined sequence of IFT train assembly: IFT-B first forms the\nbackbone, which scaffolds the front-to-back attachment of IFT-A, dynein-1b, and finally kinesin-2 close to the TZ (Fig. 4K). A major remaining question is how IFT entry into the cilium is regulated. What prevents the front end of the train from entering the cilium as assembly proceeds toward the incomplete back of the train? Possible mechanisms may include controlled loading of kinesin-2 by factors at the TZ, the sleeve serving as a barrier to prevent entry of small trains, or the extra IFT-B – associated density acting as a molecular brake until train assembly is completed.\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T Downloaded from https://www.science.org on February 23, 2024\n\n\nHugo van den Hoek 1,2,3 † , Nikolai Klena 4,5 † , Mareike A. Jordan 6 , Gonzalo Alvarez Viar 5,6 , Ricardo D. Righetto 1,2 , Miroslava Schaffer 3 , Philipp S. Erdmann 5 , William Wan 7 , Stefan Geimer 8 , Jürgen M. Plitzko 3 , Wolfgang Baumeister 3 , Gaia Pigino 5,6 * , Virginie Hamel 4 * , Paul Guichard 4 * , Benjamin D. Engel 1,2 *\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 2 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 3 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 4 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 5 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\n7. K. G. Kozminski, K. A. Johnson, P. Forscher, J. L. Rosenbaum, Proc. Natl. Acad. Sci. U.S.A. 90 , 5519 – 5523 (1993). 8. H. Ishikawa, W. F. Marshall, Nat. Rev. Mol. Cell Biol. 12 , 222 – 234 (2011). 9. J. L. Wingfield et al ., eLife 6 , e26609 (2017). 10. J. V. K. Hibbard, N. Vazquez, R. Satija, J. B. Wallingford, Mol. Biol. Cell 32 , 1171 – 1180 (2021). 11. M. A. Jordan, D. R. Diener, L. Stepanek, G. Pigino, Nat. Cell Biol. 20 , 1250 – 1255 (2018). 12. S. Asano, B. D. Engel, W. Baumeister, J. Mol. Biol. 428 , 332 – 343 (2016). 13. D. Gambarotto et al ., Nat. Methods 16 , 71 – 74 (2019). 14. M. Schaffer et al ., J. Struct. Biol. 197 , 73 – 82 (2017). 15. S. Geimer, M. Melkonian, J. Cell Sci. 117 , 2663 – 2674 (2004). 16. M. Le Guennec et al ., Sci. Adv. 6 , eaaz4137 (2020). 17. B. Craige et al ., J. Cell Biol. 190 , 927 – 940 (2010). 18. C. L. Williams et al ., J. Cell Biol. 192 , 1023 – 1041 (2011). 19. F. R. Garcia-Gonzalo, J. F. Reiter, Cold Spring Harb. Perspect. Biol. 9 , a028134 (2017). 20. A. R. G. De-Castro et al ., J. Cell Biol. 221 , e202010178 (2022). 21. L. M. Quarmby, in The Chlamydomonas Sourcebook ( Second Edition ) , Volume 3 , E. H. Harris, D. B. Stern, G. B. Witman, Eds. (Academic Press, 2009), pp. 43 – 69.\n\n\n22. K. H. Bui, T. Yagi, R. Yamamoto, R. Kamiya, T. Ishikawa, J. Cell Biol. 198 , 913 – 925 (2012). 23. G. A. Greenan, R. D. Vale, D. A. Agard, J. Cell Biol. 219 , e201907060 (2020). 24. D. G. Cole et al ., J. Cell Biol. 141 , 993 – 1008 (1998). 25. Y. Hou et al ., J. Cell Biol. 176 , 653 – 665 (2007). 26. L. Stepanek, G. Pigino, Science 352 , 721 – 724 (2016). 27. K. Toropova et al ., Nat. Struct. Mol. Biol. 26 , 823 – 829 (2019). 28. C. Iomini, V. Babaev-Khaimov, M. Sassaroli, G. Piperno, J. Cell Biol. 153 , 13 – 24 (2001). 29. J. M. Brown, D. A. Cochran, B. Craige, T. Kubo, G. B. Witman, Curr. Biol. 25 , 1583 – 1593 (2015). 30. B. D. Engel, W. B. Ludington, W. F. Marshall, J. Cell Biol. 187 , 81 – 89 (2009).\n\n\n31. W. B. Ludington, K. A. Wemmer, K. F. Lechtreck, G. B. Witman, W. F. Marshall, Proc. Natl. Acad. Sci. U.S.A. 110 , 3925 – 3930 (2013).\n\n\nSUPPLEMENTARY MATERIALS science.org/doi/10.1126/science.abm6704 Materials and Methods Figs. S1 to S18 Tables S1 and S2 References ( 32 – 64 ) MDAR Reproducibility Checklist Movies S1 and S2 View/request a protocol for this paper from Bio-protocol .\n",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''CILIA AND FLAGELLA\nIn situ architecture of the ciliary base reveals the stepwise assembly of intraflagellar transport trains\nThe cilium is an antenna-like organelle that performs numerous cellular functions, including motility, sensing, and signaling. The base of the cilium contains a selective barrier that regulates the entry of large intraflagellar transport (IFT) trains, which carry cargo proteins required for ciliary assembly and maintenance. However, the native architecture of the ciliary base and the process of IFT train assembly remain unresolved. In this work, we used in situ cryo – electron tomography to reveal native structures of the transition zone region and assembling IFT trains at the ciliary base in Chlamydomonas . We combined this direct cellular visualization with ultrastructure expansion microscopy to describe the front-to-back stepwise assembly of IFT trains: IFT-B forms the backbone, onto which bind IFT-A, dynein-1b, and finally kinesin-2 before entry into the cilium.\nC ilia (and closely related flagella) are evo- lutionarily conserved eukaryotic organ- elles that extend from the cell surface and are involved in a wide variety of functions, including cell motility, fluid flow generation, sensing, and signaling ( 1 , 2 ). The cilium consists of an axoneme of micro- tubule doublets (MTDs), which extend from the microtubule triplets (MTTs) of the centri- ole (also called a basal body) and are sheathed in a ciliary membrane. At the base of the cilium, a specialized region known as the tran- sition zone (TZ) gates entry and exit of both membrane-bound and soluble ciliary proteins ( 3 – 5 ). Among its many gating functions, the TZ is believed to regulate entry of intra- flagellar transport (IFT), bidirectional traf- fic between the ciliary base and tip driven by kinesin and dynein motors ( 6 , 7 ). IFT is required for assembly and maintenance of the cilium, as well as for mediating many of its signaling functions ( 8 ). A pool of IFT pro- teins and motors resides at the ciliary base for several seconds before entry into the cilium ( 9 , 10 ). However, it remains to be characterized how these proteins assemble into the elaborate multi-megadalton IFT trains observed running along the ciliary axoneme ( 11 ). In this study,\nwe combined in situ cryo – electron tomogra- phy (cryo-ET) ( 12 ) with ultrastructure expan- sion microscopy (U-ExM) ( 13 ) of the green alga Chlamydomonas reinhardtii , a classic model for cilia research, to reveal native TZ structures and stepwise assembly of IFT trains at the ciliary base.\nVitrified Chlamydomonas cells were thinned with a focused ion beam (FIB) ( 14 ) then imaged by cryo-ET, revealing the ciliary base inside the native cellular environment (figs. S1 and S2). Focusing on the TZ, we observed several dif- ferent structures attached to the MTDs (Fig. 1, A and B), which we resolved in molecular de- tail by subtomogram averaging (Fig. 1, C to H, and figs. S3 and S4). The proximal ~180 nm of the TZ is occupied by peripheral Y-links (Fig. 1, turquoise) and luminal stellate fibers (Fig. 1, purple, and fig. S2), which resemble a nine- pointed star in cross section ( 15 ). Our structure reveals that the stellates form a helical cylinder with a six-start helix and a pitch of 49.2 nm (Fig. 1H). Notably, this matches the pitch of the inner scaffold, a three-start helical cylin- der in the lumen of the centriole ( 16 ). The nine points of the stellate star bind MTDs at protofilament A3, with a longitudinal perio- dicity of ~8.1 nm along each MTD (Fig. 1F and fig. S5), which matches the 8.2 nm length of one tubulin dimer.\nY-links are present in many species. They connect MTDs to the ciliary membrane and help gate transport into and out of the cilium ( 17 – 20 ). Our structure reveals a broad density attaching the Y-links to the MTD, spanning protofilaments A9, A10, and B1 to B4, with a longitudinal repeat of ~8.3 nm (Fig. 1F and fig. S5). However, the outer densities of the Y-links that connect to the ciliary membrane were not resolved, likely owing to flexibility of these filamentous structures. We observed that the Y-links extend along the MTDs prox-\nimal of the ciliary membrane (Fig. 1A). This suggests that connection to the membrane is not a prerequisite for MTD Y-link decoration.\nImmediately distal to the stellates and Y-links, we found a distinct helical density completely decorating the MTDs (Fig. 1, dark blue). This helical “ sleeve ” spans 76 ± 6 nm along the MTDs, with a periodicity of ~16.4 nm (fig. S5). Cryo-ET and U-ExM immediately after ciliary shedding (figs. S6 and S7) revealed that the MTD sleeve overlaps with the ciliary cleavage site, also known as the site of flagellar auto- tomy (SOFA) ( 21 ) (Fig. 1B). Consistent with this finding, the MTD sleeve was absent from the severed ends of MTDs both on the TZ (fig. S6, C and D) and isolated cilia ( 22 ). We therefore hypothesize that this structure might help reg- ulate axoneme severing and is lost during the process. Because the sleeve density covers every MTD protofilament, it should sterically hinder the attachment of molecular motors and thus may also play a role in regulating IFT entry or exit. The sleeve does not appear to be present on the TZ of motile cilia from bovine trachea ( 23 ). It has been known for decades that a pool of IFT proteins is localized near the base of the cilium ( 24 , 25 ), but the structural organization of this basal pool has remained a mystery. In our tomograms, we observed filamentous strings of particles with one end attached to the TZ and the other end splayed into the cytosol, passing between the transition fibers (Fig. 2A and fig. S8). These strings consisted of three layers, each with a different shape and periodicity (Fig. 2, B and C, and fig. S9). We performed subtomogram aver- aging of particles along each layer and combined them to produce a composite molecular struc- ture (Fig. 2D and fig. S10). Comparison to a previously published cryo-ET structure of mature anterograde IFT trains within the cilium ( 11 ) confirmed that the cytosolic strings are indeed IFT trains. Our observed cytosolic train struc- ture was similar to the anterograde train struc- ture (fig. S11), enabling us to assign densities to IFT-B complexes, IFT-A complexes, and dynein-1b motors, which have longitudinal periodicities of ∼ 6, ∼ 11, and ∼ 18 nm, respectively (fig. S12). As was shown for anterograde IFT, we observed that these cytosolic trains were attached to the MTD B-tubule (fig. S13D) ( 26 ) and that dynein-1b was loaded onto the trains as a cargo in an autoinhibited state ( 11 , 27 ). Thus, much of the IFT pool at the ciliary base consists of TZ- tethered anterograde IFT trains that are under- going assembly before entry into the cilium. We did not see retrograde trains at the ciliary base, which implies that they rapidly disassemble, perhaps even before exiting the TZ.\nWe observed two notable differences be- tween assembling and mature IFT trains. First, assembling trains are flexible and display re- gions of high curvature (Fig. 2, A to C, and fig. S9), whereas trains in the cilium have an extended straight conformation (fig. S12) ( 11 ),\nwhich is likely maintained by interactions with cargos and the ciliary membrane. Second, as- sembling IFT-B has a prominent extra density on the side opposite IFT-A (Fig. 2D and fig. S11; red asterisks). In regions where IFT trains are bound to the TZ, this extra density is positioned adjacent to the MTD. The density is not part of the MTD structure itself, because it is also\npresent on IFT-B in the sections of trains that dangle into the cytosol (fig. S13).\nIn situ cryo-ET combines native molecular structures with precise cellular localization, enabling us to plot the spatial relationship between assembling IFT trains and the TZ (Fig. 3, A and B). The front segments of IFT trains attach to MTDs (but never to the MTTs of the centriole) and then continue in the dis- tal direction along these microtubule tracks, while the back regions curve away from the TZ into the cytosol. The front ends of IFT trains were frequently observed at the Y-links, but only very rarely further distal at the MTD sleeve (fig. S9). By mapping the IFT-B, IFT-A, and dynein-1b structures along the IFT trains\n\n\nFig. 1. Cryo-ET structure of the ciliary transition zone within native\nChlamydomonas cells. ( A ) 2D slice through a cryo – electron tomogram, showing a TZ in longitudinal cross section. Stellate fibers, purple; Y-links, turquoise; MTD helical sleeve, dark blue; stellate plate, white asterisk (see fig. S2); and IFT, yellow (see Fig. 2). Scale bar: 100 nm. ( B ) Position along the TZ occupied by different structures. Distances are measured relative to the stellate plate (see fig. S6), defined as the “ 0 nm ” origin point. SOFA, site of flagellar autotomy, where the cilium is cleaved (see figs. S6 and S7). CP, central pair. Error bars indicate standard deviation. ( C to E ) Composite model of the TZ, combining averages of the stellate (purple), Y-links (turquoise), MTD sleeve (dark blue), and associated MTD (light blue), with schematic renderings of MTTs (dark gray), MTDs (light gray), and the CP (white). (C) Longitudinal section view of the complete composite model, assembled according to the measured\nlengths and positions of each component, with 21 Y-link repeats, 21 stellate repeats (7 proximal of the plate, 14 distal), and 5 MTD sleeve repeats. The model shows straight MTTs and MTDs, but, as seen in (A) and quantified in ( 16 ), the centriole is actually a slightly convex barrel. [(D) and (E)] Cross-sectional views through the indicated regions of the composite model, showing (D) MTDs encased in the helical sleeve, with the CP in the middle, and (E) the nine-pointed stellate cylinder attached to MTDs decorated with Y-links. ( F and G ) Side and cross-sectional views of a single MTD attached to (F) stellate fibers and Y-links and (G) the helical sleeve. Protofilaments of the A- and B-microtubules are numbered. ( H ) Unrolled composite model, viewed from the center of the TZ, looking outward toward the inner wall of the stellate cylinder. One continuous helical density of the six-start stellate helix is marked in pink (pitch: 49.2 nm). MTDs are numbered. See also movie S1.\n(as shown in Fig. 2A), we visualized the spa- tially defined order of train assembly. Anter- ograde trains always contain IFT-B, which forms a backbone scaffold upon which other components are attached. This train architec- ture is consistent with previous findings that IFT-B recruits IFT-A into cilia ( 28 , 29 ). We plotted the train assembly state relative to each IFT-B subunit, and although we observed variability between individual trains (Fig. 3A), the cumulative plot shows a clear distribution of IFT train components (Fig. 3B). The back seg-\nments of trains were in a less assembled state, with IFT-B extending the farthest, followed by IFT-A and then dynein-1b (we did note some assembling trains where dynein-1b extended far- ther than IFT-A). No gaps were observed in the regions occupied by IFT-A and dynein-1b, sug- gesting that these components linearly oligomer- ize from front to back on the IFT-B scaffold.\nNext, we compared the assembling cytosolic trains to mature anterograde trains within the cilium (Fig. 3, C and D). Mature trains ex- hibited a continuum of lengths from 223 to\n558 nm, comprising 36 to 90 IFT-B subunits with an average of 62 ± 16 IFT-B. In com- parison, the longest assembling train we ob- served contained 88 IFT-B subunits, and the average length was 44 ± 17 IFT-B. Thus, just one IFT train is assembled at a time per MTD. Cytosolic and ciliary IFT trains have similarly incomplete front ends, whereas the incomplete segment at the back of cytosolic trains is longer, as these regions are still undergoing assembly (Fig. 3D). Perhaps as IFT trains enter the cilium, part of the incompletely assembled\nFig. 2. Cryo-ET structures of assembling IFT trains attached to the transition zone. ( A ) Composite structures of assembling IFT trains (IFT-B, yellow; IFT-A, orange; and dynein-1b, red) engaging the ciliary TZ (colored as in Fig. 1). A schematic membrane has been added (gray double lines). Subtomogram averages have been mapped back into their positions inside a representative tomogram. (Inset) Rotated top view of the rightmost IFT train from (A). ( B and C ) 2D slices through tomograms, showing assembling IFT trains, and ( B ′ and C ′ ) overlaid 3D train structures, with their front ends attached to the TZ. (Inset) Enlarged view\nshowing the double AAA+ adenosine triphosphatase rings of the dynein-1b motors. Arrowhead colors correspond to (A): MTT/MTD, gray, and membrane, white. Scale bars: 100 nm in (B) and (C), 20 nm in the inset. ( D ) Different views of the cytosolic IFT train composite subtomogram average. Colored as in (A). In the lower right, IFT-A was removed to show details on IFT-B. Red asterisks indicate the extra IFT-B density, which was not observed on mature trains inside cilia (see fig. S11). Assembling trains were only found attached to the TZ, not in the surrounding cytosol farther from the ciliary base (see fig. S1). See also movie S2.\nback end is left behind at the TZ to nucleate the front of the next train (Fig. 4K). Because as- sembly of IFT-A and dynein-1b proceeds front to back, the front end of the new train would remain incomplete, as we see in mature anter- ograde trains inside the cilium (Fig. 3C).\nThe kinesin-2 anterograde IFT motor is dif- ficult to identify by cryo-ET owing to its rela- tively small size and flexible structure. We thus turned to U-ExM to examine the occupancy of kinesin-2 on assembling trains. We imaged components of kinesin-2, dynein-1b, IFT-B (subcomplexes B1 and B2), and IFT-A (Fig. 4,\nA to G, and fig. S14). Consistent with our cryo- ET data, all of these proteins formed filamen- tous strings at the ciliary base, with their distal ends in close proximity to the TZ (marked by centrin staining and the lack of glutamylated tubulin) (figs. S15 and S16) and their prox- imal ends curving into the cytoplasm. Using semiautomated tracing, we plotted the three- dimensional (3D) trajectories of each protein (fig. S15E). IFT-B traces were longer than traces of IFT-A and dynein-1b (Fig. 4H). Kinesin-2 traces were the shortest and were localized at the front of IFT trains close to the TZ (fig.\nS16). Double-labeling of individual IFT trains confirmed that IFT-A extends farther than kinesin-2 toward the back of the train, and IFT-B extends farther than IFT-A (Fig. 4, I and J, and fig. S17).\nWe observed ninefold occupancy of IFT around the majority of TZs (Fig. 4, F and G, and fig. S18). Measured by U-ExM, cryo-ExM, and cryo-ET, 93 to 96% of MTDs were bound by an assembling IFT train. Given that trains enter the cilium with a frequency of ~1 train/s ( 28 , 30 ), we predict that trains linger at the ciliary base for an average of 9 s during\nFig. 3. IFT trains undergo stepwise assembly from front to back. ( A and B ) Assembly states of IFT trains relative to structures of the TZ [shown at the bottom of (B)]. Distances are measured by the number of IFT-B subunits relative to the onset of the Y-links (defined as the “ 0 IFT-B ” origin point). (A) Position and train assembly state, measured for each IFT-B subunit of each assembling IFT train. Assembly states [color code in (A) applies to all panels] were determined on the basis of occupancy of IFT-A and/or dynein-1b on the IFT-B backbone. Black squares indicate train attachment points, that is, the most proximal IFT-B subunit that is close enough to the MTD (&lt;20 nm) to be considered bound to the microtubule. The three trains marked with black asterisks lack a complete “ middle ” (no dynein is bound) and thus were omitted from analysis in (D) because “ back-end B + A ” could not be split from “ front-end B + A. ” Of the 70 trains in our dataset, 30 had front ends cropped by the FIB milling and therefore were not quantified, as their starting point relative to the TZ could not be determined. Eight more trains had cropped back ends (indicated\nwith a red “ x ” ) but were included in the analysis. (B) Cumulative plot of all assembling IFT trains in (A), with positions in relation to the TZ (shown at bottom). Colored curves above the line show the summed abundance of each IFT assembly state. The black curve below the line shows the distribution of MTD attachment points, summed from the black squares in (A). ( C ) Distribution of IFT train lengths and completeness of assembly [colored as in (A)] for mature anterograde trains found in the cilium ( 11 ). ( D ) Comparison of the abundance of assembly states at the front and back ends of assembling ( “ A ” ) versus mature ( “ M ” ) trains [i.e., trains in (A) versus trains in (C)]. Box, median and 25 to 75% percentiles; X, mean; whiskers, 1.5× interquartile range; points, outliers. Statistical significance assessed by unpaired t test (**** P &lt; 0.0001; ** P = 0.002; * P = 0.03; ns, not significant, P &gt; 0.05). Because the backs of some assembling trains in this analysis were clipped by the FIB milling [ “ x ” in (A)], the “ IFT-B only ” region of assembling trains should be slightly more pronounced than the quantification shown here.\nassembly. Consistent with this hypothesis, photobleaching analysis of the Chlamydomonas basal pool showed that several green fluorescent protein (GFP) – tagged IFT proteins require ~9 s to recover peak fluorescence ( 9 ). Regardless, ciliary entry appears to be stochastic rather than sequential, with an avalanche-like relationship between lag time and IFT train size ( 31 ). Our combined cryo-ET and U-ExM mea- surements support a spatially defined sequence of IFT train assembly: IFT-B first forms the\nbackbone, which scaffolds the front-to-back attachment of IFT-A, dynein-1b, and finally kinesin-2 close to the TZ (Fig. 4K). A major remaining question is how IFT entry into the cilium is regulated. What prevents the front end of the train from entering the cilium as assembly proceeds toward the incomplete back of the train? Possible mechanisms may include controlled loading of kinesin-2 by factors at the TZ, the sleeve serving as a barrier to prevent entry of small trains, or the extra IFT-B – associated density acting as a molecular brake until train assembly is completed.\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T Downloaded from https://www.science.org on February 23, 2024\n\n\nHugo van den Hoek 1,2,3 † , Nikolai Klena 4,5 † , Mareike A. Jordan 6 , Gonzalo Alvarez Viar 5,6 , Ricardo D. Righetto 1,2 , Miroslava Schaffer 3 , Philipp S. Erdmann 5 , William Wan 7 , Stefan Geimer 8 , Jürgen M. Plitzko 3 , Wolfgang Baumeister 3 , Gaia Pigino 5,6 * , Virginie Hamel 4 * , Paul Guichard 4 * , Benjamin D. Engel 1,2 *\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 2 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 3 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 4 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 5 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\n7. K. G. Kozminski, K. A. Johnson, P. Forscher, J. L. Rosenbaum, Proc. Natl. Acad. Sci. U.S.A. 90 , 5519 – 5523 (1993). 8. H. Ishikawa, W. F. Marshall, Nat. Rev. Mol. Cell Biol. 12 , 222 – 234 (2011). 9. J. L. Wingfield et al ., eLife 6 , e26609 (2017). 10. J. V. K. Hibbard, N. Vazquez, R. Satija, J. B. Wallingford, Mol. Biol. Cell 32 , 1171 – 1180 (2021). 11. M. A. Jordan, D. R. Diener, L. Stepanek, G. Pigino, Nat. Cell Biol. 20 , 1250 – 1255 (2018). 12. S. Asano, B. D. Engel, W. Baumeister, J. Mol. Biol. 428 , 332 – 343 (2016). 13. D. Gambarotto et al ., Nat. Methods 16 , 71 – 74 (2019). 14. M. Schaffer et al ., J. Struct. Biol. 197 , 73 – 82 (2017). 15. S. Geimer, M. Melkonian, J. Cell Sci. 117 , 2663 – 2674 (2004). 16. M. Le Guennec et al ., Sci. Adv. 6 , eaaz4137 (2020). 17. B. Craige et al ., J. Cell Biol. 190 , 927 – 940 (2010). 18. C. L. Williams et al ., J. Cell Biol. 192 , 1023 – 1041 (2011). 19. F. R. Garcia-Gonzalo, J. F. Reiter, Cold Spring Harb. Perspect. Biol. 9 , a028134 (2017). 20. A. R. G. De-Castro et al ., J. Cell Biol. 221 , e202010178 (2022). 21. L. M. Quarmby, in The Chlamydomonas Sourcebook ( Second Edition ) , Volume 3 , E. H. Harris, D. B. Stern, G. B. Witman, Eds. (Academic Press, 2009), pp. 43 – 69.\n\n\n22. K. H. Bui, T. Yagi, R. Yamamoto, R. Kamiya, T. Ishikawa, J. Cell Biol. 198 , 913 – 925 (2012). 23. G. A. Greenan, R. D. Vale, D. A. Agard, J. Cell Biol. 219 , e201907060 (2020). 24. D. G. Cole et al ., J. Cell Biol. 141 , 993 – 1008 (1998). 25. Y. Hou et al ., J. Cell Biol. 176 , 653 – 665 (2007). 26. L. Stepanek, G. Pigino, Science 352 , 721 – 724 (2016). 27. K. Toropova et al ., Nat. Struct. Mol. Biol. 26 , 823 – 829 (2019). 28. C. Iomini, V. Babaev-Khaimov, M. Sassaroli, G. Piperno, J. Cell Biol. 153 , 13 – 24 (2001). 29. J. M. Brown, D. A. Cochran, B. Craige, T. Kubo, G. B. Witman, Curr. Biol. 25 , 1583 – 1593 (2015). 30. B. D. Engel, W. B. Ludington, W. F. Marshall, J. Cell Biol. 187 , 81 – 89 (2009).\n\n\n31. W. B. Ludington, K. A. Wemmer, K. F. Lechtreck, G. B. Witman, W. F. Marshall, Proc. Natl. Acad. Sci. U.S.A. 110 , 3925 – 3930 (2013).\n\n\nSUPPLEMENTARY MATERIALS science.org/doi/10.1126/science.abm6704 Materials and Methods Figs. S1 to S18 Tables S1 and S2 References ( 32 – 64 ) MDAR Reproducibility Checklist Movies S1 and S2 View/request a protocol for this paper from Bio-protocol .\n'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''CILIA AND FLAGELLA\nIn situ architecture of the ciliary base reveals the stepwise assembly of intraflagellar transport trains\nThe cilium is an antenna-like organelle that performs numerous cellular functions, including motility, sensing, and signaling. The base of the cilium contains a selective barrier that regulates the entry of large intraflagellar transport (IFT) trains, which carry cargo proteins required for ciliary assembly and maintenance. However, the native architecture of the ciliary base and the process of IFT train assembly remain unresolved. In this work, we used in situ cryo – electron tomography to reveal native structures of the transition zone region and assembling IFT trains at the ciliary base in Chlamydomonas . We combined this direct cellular visualization with ultrastructure expansion microscopy to describe the front-to-back stepwise assembly of IFT trains: IFT-B forms the backbone, onto which bind IFT-A, dynein-1b, and finally kinesin-2 before entry into the cilium.\nC ilia (and closely related flagella) are evo- lutionarily conserved eukaryotic organ- elles that extend from the cell surface and are involved in a wide variety of functions, including cell motility, fluid flow generation, sensing, and signaling ( 1 , 2 ). The cilium consists of an axoneme of micro- tubule doublets (MTDs), which extend from the microtubule triplets (MTTs) of the centri- ole (also called a basal body) and are sheathed in a ciliary membrane. At the base of the cilium, a specialized region known as the tran- sition zone (TZ) gates entry and exit of both membrane-bound and soluble ciliary proteins ( 3 – 5 ). Among its many gating functions, the TZ is believed to regulate entry of intra- flagellar transport (IFT), bidirectional traf- fic between the ciliary base and tip driven by kinesin and dynein motors ( 6 , 7 ). IFT is required for assembly and maintenance of the cilium, as well as for mediating many of its signaling functions ( 8 ). A pool of IFT pro- teins and motors resides at the ciliary base for several seconds before entry into the cilium ( 9 , 10 ). However, it remains to be characterized how these proteins assemble into the elaborate multi-megadalton IFT trains observed running along the ciliary axoneme ( 11 ). In this study,\nwe combined in situ cryo – electron tomogra- phy (cryo-ET) ( 12 ) with ultrastructure expan- sion microscopy (U-ExM) ( 13 ) of the green alga Chlamydomonas reinhardtii , a classic model for cilia research, to reveal native TZ structures and stepwise assembly of IFT trains at the ciliary base.\nVitrified Chlamydomonas cells were thinned with a focused ion beam (FIB) ( 14 ) then imaged by cryo-ET, revealing the ciliary base inside the native cellular environment (figs. S1 and S2). Focusing on the TZ, we observed several dif- ferent structures attached to the MTDs (Fig. 1, A and B), which we resolved in molecular de- tail by subtomogram averaging (Fig. 1, C to H, and figs. S3 and S4). The proximal ~180 nm of the TZ is occupied by peripheral Y-links (Fig. 1, turquoise) and luminal stellate fibers (Fig. 1, purple, and fig. S2), which resemble a nine- pointed star in cross section ( 15 ). Our structure reveals that the stellates form a helical cylinder with a six-start helix and a pitch of 49.2 nm (Fig. 1H). Notably, this matches the pitch of the inner scaffold, a three-start helical cylin- der in the lumen of the centriole ( 16 ). The nine points of the stellate star bind MTDs at protofilament A3, with a longitudinal perio- dicity of ~8.1 nm along each MTD (Fig. 1F and fig. S5), which matches the 8.2 nm length of one tubulin dimer.\nY-links are present in many species. They connect MTDs to the ciliary membrane and help gate transport into and out of the cilium ( 17 – 20 ). Our structure reveals a broad density attaching the Y-links to the MTD, spanning protofilaments A9, A10, and B1 to B4, with a longitudinal repeat of ~8.3 nm (Fig. 1F and fig. S5). However, the outer densities of the Y-links that connect to the ciliary membrane were not resolved, likely owing to flexibility of these filamentous structures. We observed that the Y-links extend along the MTDs prox-\nimal of the ciliary membrane (Fig. 1A). This suggests that connection to the membrane is not a prerequisite for MTD Y-link decoration.\nImmediately distal to the stellates and Y-links, we found a distinct helical density completely decorating the MTDs (Fig. 1, dark blue). This helical “ sleeve ” spans 76 ± 6 nm along the MTDs, with a periodicity of ~16.4 nm (fig. S5). Cryo-ET and U-ExM immediately after ciliary shedding (figs. S6 and S7) revealed that the MTD sleeve overlaps with the ciliary cleavage site, also known as the site of flagellar auto- tomy (SOFA) ( 21 ) (Fig. 1B). Consistent with this finding, the MTD sleeve was absent from the severed ends of MTDs both on the TZ (fig. S6, C and D) and isolated cilia ( 22 ). We therefore hypothesize that this structure might help reg- ulate axoneme severing and is lost during the process. Because the sleeve density covers every MTD protofilament, it should sterically hinder the attachment of molecular motors and thus may also play a role in regulating IFT entry or exit. The sleeve does not appear to be present on the TZ of motile cilia from bovine trachea ( 23 ). It has been known for decades that a pool of IFT proteins is localized near the base of the cilium ( 24 , 25 ), but the structural organization of this basal pool has remained a mystery. In our tomograms, we observed filamentous strings of particles with one end attached to the TZ and the other end splayed into the cytosol, passing between the transition fibers (Fig. 2A and fig. S8). These strings consisted of three layers, each with a different shape and periodicity (Fig. 2, B and C, and fig. S9). We performed subtomogram aver- aging of particles along each layer and combined them to produce a composite molecular struc- ture (Fig. 2D and fig. S10). Comparison to a previously published cryo-ET structure of mature anterograde IFT trains within the cilium ( 11 ) confirmed that the cytosolic strings are indeed IFT trains. Our observed cytosolic train struc- ture was similar to the anterograde train struc- ture (fig. S11), enabling us to assign densities to IFT-B complexes, IFT-A complexes, and dynein-1b motors, which have longitudinal periodicities of ∼ 6, ∼ 11, and ∼ 18 nm, respectively (fig. S12). As was shown for anterograde IFT, we observed that these cytosolic trains were attached to the MTD B-tubule (fig. S13D) ( 26 ) and that dynein-1b was loaded onto the trains as a cargo in an autoinhibited state ( 11 , 27 ). Thus, much of the IFT pool at the ciliary base consists of TZ- tethered anterograde IFT trains that are under- going assembly before entry into the cilium. We did not see retrograde trains at the ciliary base, which implies that they rapidly disassemble, perhaps even before exiting the TZ.\nWe observed two notable differences be- tween assembling and mature IFT trains. First, assembling trains are flexible and display re- gions of high curvature (Fig. 2, A to C, and fig. S9), whereas trains in the cilium have an extended straight conformation (fig. S12) ( 11 ),\nwhich is likely maintained by interactions with cargos and the ciliary membrane. Second, as- sembling IFT-B has a prominent extra density on the side opposite IFT-A (Fig. 2D and fig. S11; red asterisks). In regions where IFT trains are bound to the TZ, this extra density is positioned adjacent to the MTD. The density is not part of the MTD structure itself, because it is also\npresent on IFT-B in the sections of trains that dangle into the cytosol (fig. S13).\nIn situ cryo-ET combines native molecular structures with precise cellular localization, enabling us to plot the spatial relationship between assembling IFT trains and the TZ (Fig. 3, A and B). The front segments of IFT trains attach to MTDs (but never to the MTTs of the centriole) and then continue in the dis- tal direction along these microtubule tracks, while the back regions curve away from the TZ into the cytosol. The front ends of IFT trains were frequently observed at the Y-links, but only very rarely further distal at the MTD sleeve (fig. S9). By mapping the IFT-B, IFT-A, and dynein-1b structures along the IFT trains\n\n\nFig. 1. Cryo-ET structure of the ciliary transition zone within native\nChlamydomonas cells. ( A ) 2D slice through a cryo – electron tomogram, showing a TZ in longitudinal cross section. Stellate fibers, purple; Y-links, turquoise; MTD helical sleeve, dark blue; stellate plate, white asterisk (see fig. S2); and IFT, yellow (see Fig. 2). Scale bar: 100 nm. ( B ) Position along the TZ occupied by different structures. Distances are measured relative to the stellate plate (see fig. S6), defined as the “ 0 nm ” origin point. SOFA, site of flagellar autotomy, where the cilium is cleaved (see figs. S6 and S7). CP, central pair. Error bars indicate standard deviation. ( C to E ) Composite model of the TZ, combining averages of the stellate (purple), Y-links (turquoise), MTD sleeve (dark blue), and associated MTD (light blue), with schematic renderings of MTTs (dark gray), MTDs (light gray), and the CP (white). (C) Longitudinal section view of the complete composite model, assembled according to the measured\nlengths and positions of each component, with 21 Y-link repeats, 21 stellate repeats (7 proximal of the plate, 14 distal), and 5 MTD sleeve repeats. The model shows straight MTTs and MTDs, but, as seen in (A) and quantified in ( 16 ), the centriole is actually a slightly convex barrel. [(D) and (E)] Cross-sectional views through the indicated regions of the composite model, showing (D) MTDs encased in the helical sleeve, with the CP in the middle, and (E) the nine-pointed stellate cylinder attached to MTDs decorated with Y-links. ( F and G ) Side and cross-sectional views of a single MTD attached to (F) stellate fibers and Y-links and (G) the helical sleeve. Protofilaments of the A- and B-microtubules are numbered. ( H ) Unrolled composite model, viewed from the center of the TZ, looking outward toward the inner wall of the stellate cylinder. One continuous helical density of the six-start stellate helix is marked in pink (pitch: 49.2 nm). MTDs are numbered. See also movie S1.\n(as shown in Fig. 2A), we visualized the spa- tially defined order of train assembly. Anter- ograde trains always contain IFT-B, which forms a backbone scaffold upon which other components are attached. This train architec- ture is consistent with previous findings that IFT-B recruits IFT-A into cilia ( 28 , 29 ). We plotted the train assembly state relative to each IFT-B subunit, and although we observed variability between individual trains (Fig. 3A), the cumulative plot shows a clear distribution of IFT train components (Fig. 3B). The back seg-\nments of trains were in a less assembled state, with IFT-B extending the farthest, followed by IFT-A and then dynein-1b (we did note some assembling trains where dynein-1b extended far- ther than IFT-A). No gaps were observed in the regions occupied by IFT-A and dynein-1b, sug- gesting that these components linearly oligomer- ize from front to back on the IFT-B scaffold.\nNext, we compared the assembling cytosolic trains to mature anterograde trains within the cilium (Fig. 3, C and D). Mature trains ex- hibited a continuum of lengths from 223 to\n558 nm, comprising 36 to 90 IFT-B subunits with an average of 62 ± 16 IFT-B. In com- parison, the longest assembling train we ob- served contained 88 IFT-B subunits, and the average length was 44 ± 17 IFT-B. Thus, just one IFT train is assembled at a time per MTD. Cytosolic and ciliary IFT trains have similarly incomplete front ends, whereas the incomplete segment at the back of cytosolic trains is longer, as these regions are still undergoing assembly (Fig. 3D). Perhaps as IFT trains enter the cilium, part of the incompletely assembled\nFig. 2. Cryo-ET structures of assembling IFT trains attached to the transition zone. ( A ) Composite structures of assembling IFT trains (IFT-B, yellow; IFT-A, orange; and dynein-1b, red) engaging the ciliary TZ (colored as in Fig. 1). A schematic membrane has been added (gray double lines). Subtomogram averages have been mapped back into their positions inside a representative tomogram. (Inset) Rotated top view of the rightmost IFT train from (A). ( B and C ) 2D slices through tomograms, showing assembling IFT trains, and ( B ′ and C ′ ) overlaid 3D train structures, with their front ends attached to the TZ. (Inset) Enlarged view\nshowing the double AAA+ adenosine triphosphatase rings of the dynein-1b motors. Arrowhead colors correspond to (A): MTT/MTD, gray, and membrane, white. Scale bars: 100 nm in (B) and (C), 20 nm in the inset. ( D ) Different views of the cytosolic IFT train composite subtomogram average. Colored as in (A). In the lower right, IFT-A was removed to show details on IFT-B. Red asterisks indicate the extra IFT-B density, which was not observed on mature trains inside cilia (see fig. S11). Assembling trains were only found attached to the TZ, not in the surrounding cytosol farther from the ciliary base (see fig. S1). See also movie S2.\nback end is left behind at the TZ to nucleate the front of the next train (Fig. 4K). Because as- sembly of IFT-A and dynein-1b proceeds front to back, the front end of the new train would remain incomplete, as we see in mature anter- ograde trains inside the cilium (Fig. 3C).\nThe kinesin-2 anterograde IFT motor is dif- ficult to identify by cryo-ET owing to its rela- tively small size and flexible structure. We thus turned to U-ExM to examine the occupancy of kinesin-2 on assembling trains. We imaged components of kinesin-2, dynein-1b, IFT-B (subcomplexes B1 and B2), and IFT-A (Fig. 4,\nA to G, and fig. S14). Consistent with our cryo- ET data, all of these proteins formed filamen- tous strings at the ciliary base, with their distal ends in close proximity to the TZ (marked by centrin staining and the lack of glutamylated tubulin) (figs. S15 and S16) and their prox- imal ends curving into the cytoplasm. Using semiautomated tracing, we plotted the three- dimensional (3D) trajectories of each protein (fig. S15E). IFT-B traces were longer than traces of IFT-A and dynein-1b (Fig. 4H). Kinesin-2 traces were the shortest and were localized at the front of IFT trains close to the TZ (fig.\nS16). Double-labeling of individual IFT trains confirmed that IFT-A extends farther than kinesin-2 toward the back of the train, and IFT-B extends farther than IFT-A (Fig. 4, I and J, and fig. S17).\nWe observed ninefold occupancy of IFT around the majority of TZs (Fig. 4, F and G, and fig. S18). Measured by U-ExM, cryo-ExM, and cryo-ET, 93 to 96% of MTDs were bound by an assembling IFT train. Given that trains enter the cilium with a frequency of ~1 train/s ( 28 , 30 ), we predict that trains linger at the ciliary base for an average of 9 s during\nFig. 3. IFT trains undergo stepwise assembly from front to back. ( A and B ) Assembly states of IFT trains relative to structures of the TZ [shown at the bottom of (B)]. Distances are measured by the number of IFT-B subunits relative to the onset of the Y-links (defined as the “ 0 IFT-B ” origin point). (A) Position and train assembly state, measured for each IFT-B subunit of each assembling IFT train. Assembly states [color code in (A) applies to all panels] were determined on the basis of occupancy of IFT-A and/or dynein-1b on the IFT-B backbone. Black squares indicate train attachment points, that is, the most proximal IFT-B subunit that is close enough to the MTD (&lt;20 nm) to be considered bound to the microtubule. The three trains marked with black asterisks lack a complete “ middle ” (no dynein is bound) and thus were omitted from analysis in (D) because “ back-end B + A ” could not be split from “ front-end B + A. ” Of the 70 trains in our dataset, 30 had front ends cropped by the FIB milling and therefore were not quantified, as their starting point relative to the TZ could not be determined. Eight more trains had cropped back ends (indicated\nwith a red “ x ” ) but were included in the analysis. (B) Cumulative plot of all assembling IFT trains in (A), with positions in relation to the TZ (shown at bottom). Colored curves above the line show the summed abundance of each IFT assembly state. The black curve below the line shows the distribution of MTD attachment points, summed from the black squares in (A). ( C ) Distribution of IFT train lengths and completeness of assembly [colored as in (A)] for mature anterograde trains found in the cilium ( 11 ). ( D ) Comparison of the abundance of assembly states at the front and back ends of assembling ( “ A ” ) versus mature ( “ M ” ) trains [i.e., trains in (A) versus trains in (C)]. Box, median and 25 to 75% percentiles; X, mean; whiskers, 1.5× interquartile range; points, outliers. Statistical significance assessed by unpaired t test (**** P &lt; 0.0001; ** P = 0.002; * P = 0.03; ns, not significant, P &gt; 0.05). Because the backs of some assembling trains in this analysis were clipped by the FIB milling [ “ x ” in (A)], the “ IFT-B only ” region of assembling trains should be slightly more pronounced than the quantification shown here.\nassembly. Consistent with this hypothesis, photobleaching analysis of the Chlamydomonas basal pool showed that several green fluorescent protein (GFP) – tagged IFT proteins require ~9 s to recover peak fluorescence ( 9 ). Regardless, ciliary entry appears to be stochastic rather than sequential, with an avalanche-like relationship between lag time and IFT train size ( 31 ). Our combined cryo-ET and U-ExM mea- surements support a spatially defined sequence of IFT train assembly: IFT-B first forms the\nbackbone, which scaffolds the front-to-back attachment of IFT-A, dynein-1b, and finally kinesin-2 close to the TZ (Fig. 4K). A major remaining question is how IFT entry into the cilium is regulated. What prevents the front end of the train from entering the cilium as assembly proceeds toward the incomplete back of the train? Possible mechanisms may include controlled loading of kinesin-2 by factors at the TZ, the sleeve serving as a barrier to prevent entry of small trains, or the extra IFT-B – associated density acting as a molecular brake until train assembly is completed.\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T\n\n\nRESEARCH | R E P O R T Downloaded from https://www.science.org on February 23, 2024\n\n\nHugo van den Hoek 1,2,3 † , Nikolai Klena 4,5 † , Mareike A. Jordan 6 , Gonzalo Alvarez Viar 5,6 , Ricardo D. Righetto 1,2 , Miroslava Schaffer 3 , Philipp S. Erdmann 5 , William Wan 7 , Stefan Geimer 8 , Jürgen M. Plitzko 3 , Wolfgang Baumeister 3 , Gaia Pigino 5,6 * , Virginie Hamel 4 * , Paul Guichard 4 * , Benjamin D. Engel 1,2 *\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 2 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 3 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 4 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\nvan den Hoek et al ., Science 377 , 543 – 548 (2022) 29 July 2022 5 of 6\n\n\nDownloaded from https://www.science.org on February 23, 2024\n\n\n7. K. G. Kozminski, K. A. Johnson, P. Forscher, J. L. Rosenbaum, Proc. Natl. Acad. Sci. U.S.A. 90 , 5519 – 5523 (1993). 8. H. Ishikawa, W. F. Marshall, Nat. Rev. Mol. Cell Biol. 12 , 222 – 234 (2011). 9. J. L. Wingfield et al ., eLife 6 , e26609 (2017). 10. J. V. K. Hibbard, N. Vazquez, R. Satija, J. B. Wallingford, Mol. Biol. Cell 32 , 1171 – 1180 (2021). 11. M. A. Jordan, D. R. Diener, L. Stepanek, G. Pigino, Nat. Cell Biol. 20 , 1250 – 1255 (2018). 12. S. Asano, B. D. Engel, W. Baumeister, J. Mol. Biol. 428 , 332 – 343 (2016). 13. D. Gambarotto et al ., Nat. Methods 16 , 71 – 74 (2019). 14. M. Schaffer et al ., J. Struct. Biol. 197 , 73 – 82 (2017). 15. S. Geimer, M. Melkonian, J. Cell Sci. 117 , 2663 – 2674 (2004). 16. M. Le Guennec et al ., Sci. Adv. 6 , eaaz4137 (2020). 17. B. Craige et al ., J. Cell Biol. 190 , 927 – 940 (2010). 18. C. L. Williams et al ., J. Cell Biol. 192 , 1023 – 1041 (2011). 19. F. R. Garcia-Gonzalo, J. F. Reiter, Cold Spring Harb. Perspect. Biol. 9 , a028134 (2017). 20. A. R. G. De-Castro et al ., J. Cell Biol. 221 , e202010178 (2022). 21. L. M. Quarmby, in The Chlamydomonas Sourcebook ( Second Edition ) , Volume 3 , E. H. Harris, D. B. Stern, G. B. Witman, Eds. (Academic Press, 2009), pp. 43 – 69.\n\n\n22. K. H. Bui, T. Yagi, R. Yamamoto, R. Kamiya, T. Ishikawa, J. Cell Biol. 198 , 913 – 925 (2012). 23. G. A. Greenan, R. D. Vale, D. A. Agard, J. Cell Biol. 219 , e201907060 (2020). 24. D. G. Cole et al ., J. Cell Biol. 141 , 993 – 1008 (1998). 25. Y. Hou et al ., J. Cell Biol. 176 , 653 – 665 (2007). 26. L. Stepanek, G. Pigino, Science 352 , 721 – 724 (2016). 27. K. Toropova et al ., Nat. Struct. Mol. Biol. 26 , 823 – 829 (2019). 28. C. Iomini, V. Babaev-Khaimov, M. Sassaroli, G. Piperno, J. Cell Biol. 153 , 13 – 24 (2001). 29. J. M. Brown, D. A. Cochran, B. Craige, T. Kubo, G. B. Witman, Curr. Biol. 25 , 1583 – 1593 (2015). 30. B. D. Engel, W. B. Ludington, W. F. Marshall, J. Cell Biol. 187 , 81 – 89 (2009).\n\n\n31. W. B. Ludington, K. A. Wemmer, K. F. Lechtreck, G. B. Witman, W. F. Marshall, Proc. Natl. Acad. Sci. U.S.A. 110 , 3925 – 3930 (2013).\n\n\nSUPPLEMENTARY MATERIALS science.org/doi/10.1126/science.abm6704 Materials and Methods Figs. S1 to S18 Tables S1 and S2 References ( 32 – 64 ) MDAR Reproducibility Checklist Movies S1 and S2 View/request a protocol for this paper from Bio-protocol .\n'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [21.91s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"micro-organism\",\n  \"organism_name\": \"Chlamydomonas reinhardtii\",\n  \"cell_strain\": \"not present\",\n  \"sample_preparation\": \"Vitrified Chlamydomonas cells were thinned with a focused ion beam (FIB)\",\n  \"grid_preparation\": \"not present\",\n  \"cryoet_pixel_spacing\": \"not present\",\n  \"cryoet_acceleration_voltage\": \"not present\",\n  \"microscope_name\": \"not present\",\n  \"microscope_setup_energy_filter\": \"not present\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"not present\",\n  \"camera_model\": \"not present\",\n  \"tilt_minimum\": \"not present\",\n  \"tilt_maximum\": \"not present\",\n  \"total_flux\": \"not present\",\n  \"reconstruction_software\": \"not present\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"subtomogram averaging\",\n  \"annotated_entities\": \"MTDs (microtubule doublets), MTTs (microtubule triplets), peripheral Y-links, luminal stellate fibers, helical sleeve, IFT (intraflagellar transport) trains\",\n  \"annotation_methods\": \"cryo-ET (Cryo-Electron Tomography) combined with ultrastructure expansion microscopy and subtomogram averaging\",\n  \"annotation_software\": \"not present\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"micro-organism\",\n  \"organism_name\": \"Chlamydomonas reinhardtii\",\n  \"cell_strain\": \"not present\",\n  \"sample_preparation\": \"Vitrified Chlamydomonas cells were thinned with a focused ion beam (FIB)\",\n  \"grid_preparation\": \"not present\",\n  \"cryoet_pixel_spacing\": \"not present\",\n  \"cryoet_acceleration_voltage\": \"not present\",\n  \"microscope_name\": \"not present\",\n  \"microscope_setup_energy_filter\": \"not present\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"not present\",\n  \"camera_model\": \"not present\",\n  \"tilt_minimum\": \"not present\",\n  \"tilt_maximum\": \"not present\",\n  \"total_flux\": \"not present\",\n  \"reconstruction_software\": \"not present\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"subtomogram averaging\",\n  \"annotated_entities\": \"MTDs (microtubule doublets), MTTs (microtubule triplets), peripheral Y-links, luminal stellate fibers, helical sleeve, IFT (intraflagellar transport) trains\",\n  \"annotation_methods\": \"cryo-ET (Cryo-Electron Tomography) combined with ultrastructure expansion microscopy and subtomogram averaging\",\n  \"annotation_software\": \"not present\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 330,
      "prompt_tokens": 7200,
      "total_tokens": 7530
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_9e42505611"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "biological_sample_type": "micro-organism",
  "organism_name": "Chlamydomonas reinhardtii",
  "cell_strain": "not present",
  "sample_preparation": "Vitrified Chlamydomonas cells were thinned with a focused ion beam (FIB)",
  "grid_preparation": "not present",
  "cryoet_pixel_spacing": "not present",
  "cryoet_acceleration_voltage": "not present",
  "microscope_name": "not present",
  "microscope_setup_energy_filter": "not present",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "not present",
  "camera_model": "not present",
  "tilt_minimum": "not present",
  "tilt_maximum": "not present",
  "total_flux": "not present",
  "reconstruction_software": "not present",
  "tomogram_size": "not present",
  "reconstruction_method": "subtomogram averaging",
  "annotated_entities": "MTDs (microtubule doublets), MTTs (microtubule triplets), peripheral Y-links, luminal stellate fibers, helical sleeve, IFT (intraflagellar transport) trains",
  "annotation_methods": "cryo-ET (Cryo-Electron Tomography) combined with ultrastructure expansion microscopy and subtomogram averaging",
  "annotation_software": "not present"
}
[chain/end] [1:chain:RunnableSequence] [21.92s] Exiting Chain run with output:
{
  "biological_sample_type": "micro-organism",
  "organism_name": "Chlamydomonas reinhardtii",
  "cell_strain": "not present",
  "sample_preparation": "Vitrified Chlamydomonas cells were thinned with a focused ion beam (FIB)",
  "grid_preparation": "not present",
  "cryoet_pixel_spacing": "not present",
  "cryoet_acceleration_voltage": "not present",
  "microscope_name": "not present",
  "microscope_setup_energy_filter": "not present",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "not present",
  "camera_model": "not present",
  "tilt_minimum": "not present",
  "tilt_maximum": "not present",
  "total_flux": "not present",
  "reconstruction_software": "not present",
  "tomogram_size": "not present",
  "reconstruction_method": "subtomogram averaging",
  "annotated_entities": "MTDs (microtubule doublets), MTTs (microtubule triplets), peripheral Y-links, luminal stellate fibers, helical sleeve, IFT (intraflagellar transport) trains",
  "annotation_methods": "cryo-ET (Cryo-Electron Tomography) combined with ultrastructure expansion microscopy and subtomogram averaging",
  "annotation_software": "not present"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, 3D views of protein and organellar structure in unfixed and fully hydrated conditions (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;). By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;; &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;), it has become routine to collect large datasets of thin, cellular 3D reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells (&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;; &lt;&lt;REF:Himes-2018-15-955&gt;&gt;; &lt;&lt;REF:Chen-2019-16-1161&gt;&gt;), it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments (&lt;&lt;REF:Jiang-2017-7-45474&gt;&gt;; &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;). As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the 2D virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intramembrane distances through manual designation of membrane boundaries by locating distinct points along the membrane (&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;; &lt;&lt;REF:Ader-2019-8-?&gt;&gt;; &lt;&lt;REF:Tran-2021-374-52&gt;&gt;; &lt;&lt;REF:Mageswaran-2021-7-7&gt;&gt;). Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Toward a more accurate 3D representation, manual or automated (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvedness and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvedness. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvedness as well as the spatial relationship between individual triangles (i.e., membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e., boundary segmentations) were not reliable due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm (&lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;). In cryo-ET, even perfect segmentations will never show complete and closed membranes due to the substantial missing wedge (&lt;&lt;REF:Lučič-2013-202-407&gt;&gt;), making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e., compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time-intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-ET.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling (&lt;&lt;REF:Schirrmacher-2020-8-8&gt;&gt;; &lt;&lt;REF:West-2017-17-363&gt;&gt;; &lt;&lt;REF:Wai-2016-27-105&gt;&gt;). Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure; &lt;&lt;REF:Chan-2020-15-235&gt;&gt;). Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e., elongated) or disconnected (i.e., fragmented) networks that regulate prosurvival or proapoptotic mitochondrial functions (&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;; &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;). Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions (&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;).\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvedness required for efficient mitochondrial respiration, (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Blum-2019-116-4250&gt;&gt;). Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;; &lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;). Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;). It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-ET workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semiautomated workflow (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) to segment cellular membranes present in cryo-tomograms and applied a screened Poisson reconstruction strategy (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics pipeline to measure several parameters of membrane shape including inter- and intramembrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of ER stress, a condition known to promote adaptive remodeling of mitochondrial networks (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;), to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nResults\n\nCorrelative cryo-fluorescence microscopy and electron tomography enable association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP) such that the morphology of each cell's mitochondrial network could be readily assessed (Fig. 1, A-C &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification: Five researchers were given unlabeled fluorescence microscopy images of exemplar network morphologies (elongated and fragmented) as references to assign morphologies to the experimental fluorescence micrographs. We targeted cells with either fragmented or elongated mitochondrial networks for cryo-FIB milling to prepare thin (95-233 nm) lamellae (Fig. 1 D &lt;&lt;FIG:fig1&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Fig. 1 C &lt;&lt;FIG:fig1&gt;&gt;) to the ultrastructure of mitochondrial membranes (Fig. 1, F-H &lt;&lt;FIG:fig1&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). We applied our correlative approach to identify and target specific Tg-treated and vehicle (media with DMSO)-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). Notably, Tg treatment induced a shift toward regularly spaced disc-shaped (lamellar) cristae in the elongated population and toward a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-ET workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions. (A) MEFmtGFP cells were cultured on transmission EM grids (black mesh circle) and treated with vehicle or Tg (500 nM) for 8 h prior to vitrification via plunge freezing in ethane/propane mixture. (B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence microscopy (cryo-FM). (C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies. (D) These cells were targeted for cryo-FIB milling to generate thin sections (lamellae). (E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate 3D cryo-tomograms. (F) Voxel segmentations labeling distinct cellular membranes were generated using a semiautomated approach (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point (&lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; &lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). (G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: IMM, purple; OMM, blue; ER, teal. (H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP\ncells. (A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies from mixed populations of MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of 95-233 nm. MEFmtGFP cell periphery is outlined in dashed white line. (B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue. (C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backward 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\nFigure S1.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"UE1\" is untreated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with elongated mitochondrial networks, n = 10 tomograms.\nFigure S2.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TE1\" is Tg-treated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Tg-treated cells  with elongated mitochondrial networks, n = 13 tomograms.\nFigure S3.\nGallery of cellular membranes used for downstream quantifications from cells with fragmented mitochondrial networks in the absence and presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TF1\" is Tg-treated, fragmented, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with fragmented mitochondrial networks, n = 6 tomograms; and Tg-treated cells with fragmented mitochondrial networks, n = 5 tomograms.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semiautomated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;) that established a robust approach for estimating curvedness in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high-quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;; &lt;&lt;REF:Navarro-2018-5-82&gt;&gt;) and manual inpainting of entire compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;), both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semiautomated workflow to enable analysis at higher throughput (Fig. 1 F &lt;&lt;FIG:fig1&gt;&gt;). We used a segmentation strategy that takes advantage of the TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), which identifies and enhances regions of the tomogram with membrane-like voxel values. This was followed by manual labeling of membranes into mitochondrial IMM and OMM, ER, and multilamellar membrane based on cellular context, as well as manual cleanup of individual membrane segmentation using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 34 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;, Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). The segmentation output was fed into the fully automated surface morphometrics pipeline (Fig. 2 B &lt;&lt;FIG:fig2&gt;&gt;). The voxel segmentation was converted to high-quality membrane surfaces using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; and Video 1 &lt;&lt;FIG:video1&gt;&gt;). Next, these surfaces were converted into triangle graphs, and multiple membrane features were quantified including curvedness (estimated using pycurv15), distances within and between surfaces, and the relative orientations of different surfaces. Finally, the quantifications for each mitochondrion were combined into experiments to allow aggregate statistics and visualizations. Results were computed from 34 completed segmentations in a few hours using a cluster for parallelization (Table S1). This 3D surface morphometrics pipeline is configurable for any segmented membrane and is available at https://github.com/grotjahnlab/surface_morphometrics.\nFigure S4.\nComparison of surface reconstruction algorithms. The same IMM segmentation was subjected to the boundary segmentation surface reconstruction algorithm used in the pycurv software (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;; &lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;; left) as well as our new reconstruction approach based on the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; right) show the improvements in smoothness and completeness enabled by the new surface reconstruction algorithm. While the left algorithm was made part of the pycurv software, it was not used for extensive analysis in the cited manuscript, in favor of algorithms using compartment segmentations.\nVideo 1.\nComparison of voxel segmentation vs. surface reconstruction. An example tomogram is shown with and without a completed voxel segmentation (green). A zoom in reveals voxel stepping artifacts that impact measurement precision, as well as small holes that result from automated segmentation. Switching to the reconstructed surface shows the smooth triangle mesh and the hole-filling properties of the surface reconstruction algorithm. Surface colors: IMM in pink, OMM in purple, ER in blue.\n\nBulk analysis and assessment of statistical significance based on surface quantifications\nOur pipeline outputs multiple membrane feature quantifications (i.e., curvature, distance, and orientation) for each triangle, totaling ~500,000 triangles per tomogram. We set out to determine an appropriate method for bulk quantitative and statistical analyses that would enable detection of subtle albeit potentially significant differences in membrane ultrastructure across different mitochondrial morphologies and treatment groups. We plotted the overall distributions of each membrane feature quantification using histograms of triangle values that are weighted by the area of each triangle, both for individual mitochondrion (Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;) and for entire experiments. These histograms identify bulk differences in the membrane ultrastructure of mitochondria under different experimental conditions. For several membrane parameter quantifications, the observed histogram peak shifted visibly between conditions, suggesting potential differences between treatment and morphology groups. In order to test the statistical significance of these shifts, the weighted histogram peak for each membrane feature quantification for each individual mitochondrion was extracted and these peaks were used for statistical comparisons. In most cases, there was a statistically significant difference in the mean of the distributions, which was assessed via a non-parametric Mann-Whitney U test (Table S2). Differences in the variability of metrics between mitochondria within a given experimental condition were assessed with a two-sample Kolmogorov-Smirnoff test.\nFigure S5.\nGallery of IMM-OMM distance histograms for individual mitochondria. The distribution of distances between the IMM and OMM was plotted for each mitochondrion individually for all Tg-treated, elongated (TE), Tg-treated, fragmented (TF), vehicle (i.e., untreated), elongated (UE), and vehicle (i.e., untreated), fragmented (UF). The peak position is plotted for each and is used for the violin plot in Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;. The histograms are titled based on the measurement (i.e., IMM-IMM Dist) and correspond to the treatment conditions, morphologies, and individual mitochondria labels shown in Fig. S1 &lt;&lt;FIG:figS1&gt;&gt; (e.g., \"TE2a\" corresponds to a Tg-treated [T], elongated [E], tomogram \"2,\" mitochondrion labeled \"a\").\nIn some cases, there was an apparent change to the shape of distribution seen in quantification histograms rather than simply peak position. We primarily assessed the statistical significance of these changes by Mann-Whitney U tests and Kolmogorov-Smirnoff tests on the per-mitochondria standard deviation rather than histogram peak (Table S2). This metric captured changes to the variance of metrics. In order to quantify changes that combine variance and peak shifts, we used the two-sample Kolmogorov-Smirnoff test to test the statistical significance of changes to the distributions of mesh triangles in each condition. However, a major challenge of utilizing this statistical test for analysis of surface quantifications is assessing the true number of independent measurements being made. Triangles are highly correlated to their neighboring meshes, a correlation made even stronger by the vector voting approach employed by pycurv in order to better approximate a smooth membrane. When each triangle within a surface mesh is treated as an independent measurement (in some cases yielding sample sizes in the millions), even visibly identical distributions yield statistically significant P values. This is even further complicated because the number of triangles generated in a given surface mesh is arbitrary and not based on any physical constraint. In contrast, if each mitochondrion is treated as an independent measurement, the variance of features within the mitochondria is lost and there is essentially never statistical significance. We established a heuristic compromise based on the feature radius of 12 nm used in vector voting to optimally measure curvatures. The number of independent samples was assumed to be one per 452 nm2 of membrane area, corresponding to the 12-nm feature size. In practice, this area corresponds to the area of a single crista tip or junction and yields results with P values that might be effectively used comparatively between different measurements. Altering the feature size to 9 or 15 nm changes the number of independent samples and absolute P values; however the relative P value differences across groups do not change, demonstrating that absolute value for assessing statistical significance is limited by the artificial selection of feature radius, but the choice of feature size does not affect the overall result of the comparative analyses across different membrane feature parameters (Table S2).\nA challenge with using histogram peaks as a metric is determining the confidence interval associated with the metric. In order to assess the precision of the histogram peak measurements, we resampled each distribution using bootstrap sampling 1,000 times, and the histogram peak was extracted on each resampling. The 5th and 95th percentiles of these histogram peaks were used to represent the confidence interval of the histogram peak (Table S3).\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria is the juxtaposed inner and outer mitochondrial membranes. Cryo-ET is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh for each mitochondrion in our dataset (Fig. 3 A &lt;&lt;FIG:fig3&gt;&gt; and Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented mitochondria, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Fig. 3 C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in intermembrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak position and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each mitochondrion as independent observations and identified statistically significant increases in spacing in cells with fragmented vs. elongated mitochondria by 2.4 nm in the vehicle-treated condition and by 3.6 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance between OMM and IMM of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in cells with fragmented mitochondria after Tg treatment was not significant (Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significant differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nDistance between IMMs and OMMs is dependent on mitochondrial network morphology and presence or absence of ER stress. (A) Surface membrane reconstruction defining OMM (purple) and IMM (orange) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions. (C) Combined histogram of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Quantification of peak histogram values from each mitochondrion within the indicated treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;). We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intracrista and intrajunction distances, respectively), as well as the distance between adjacent crista bodies and junctions (intercrista and interjunction distances, respectively; Fig. 4 B &lt;&lt;FIG:fig4&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each mitochondria (Fig. 4, C and D &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intracrista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 5.3 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 1.8 nm upon Tg treatment. However, we did not observe significant changes in intercristae distance, intrajunction distance, or interjunction distance, with the exception of a 3.7 nm decrease in intrajunction spacing in the Tg-treated condition of elongated mitochondria relative to vehicle elongated mitochondria, suggesting that ER stress promotes rigidification of the junctions in elongated mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations. (A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with Tg (500 nM, 8 h) showing subdivision of IMM compartments (IBM, tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh. (B) Enlarged inset of boxed region in elongated example from A showing surface membrane reconstruction colored by inner membrane compartments defining intra- and intercrista and junction measurements. (C) Combined histogram of intracrista membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ***P &lt; 0.005. (D) Combined histogram of intrajunction membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05.\nFigure S6.\nAdditional quantifications of membrane ultrastructure. Additional quantifications output by the surface morphometrics pipeline, which showed non-significant changes or no apparent change. In all cases, quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. (A) Histograms and violin plot showing changes to spacing between closest neighbor segments of IMM. (B) Histograms and violin plot showing changes to spacing to the second nearest segments of IMM. (C) Histograms and violin plot showing changes to spacing between neighboring cristae. (D) Histograms and violin plot showing changes to spacing between closest neighbor segments of IBM. (E) Histograms and violin plot showing changes to spacing to the second nearest segments of IBM. (F) Histograms and violin plot showing changes to spacing between neighboring junctions. (G) Histograms and violin plot showing changes to curvedness of the IBM. (H) Histograms and violin plot showing changes to the angle between IMM and OMM. (I) Histograms and violin plot showing changes to the angle between IBM and OMM. (J) Histograms and violin plot showing changes to the angle between crista junctions and OMM. (K) Histograms and violin plot showing changes to the angle between IMM and the growth plane. (L) Histograms and violin plot showing changes to the angle between IBM and the growth plane. (M) Histograms and violin plot showing changes to the angle between crista junctions and the growth plane. *P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.005; ****P &lt; 0.001.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvedness (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;; &lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated cells with elongated mitochondria, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). A significant change was observed between the vehicle-elongated and vehicle-fragmented mitochondria, which established a basal distinction in curvedness among mitochondrial populations (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). As visual inspection revealed local changes in curvedness in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Fig. 5, C and D &lt;&lt;FIG:fig5&gt;&gt;).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principal components of curvedness and is used because the surface normal vectors do not have canonical sign. (B) Combined histogram of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01. (C) Combined histogram of cristae curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; **P &lt; 0.01. (D) Combined histogram of cristae junction curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\nLarger changes in curvature were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress. Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated cells with elongated mitochondria compared with both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria. This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nOMM curvedness is dependent on mitochondrial network morphology\nIn addition to the IMM, we calculated curvedness of the OMM for each mitochondrion. For both vehicle and Tg-treated cells, curvedness of the OMM is decreased in fragmented mitochondria relative to elongated networks (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;). This metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length.\nFigure S7.\nOMM curvature analysis. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by membrane curvedness and the IMM surface is represented as a transparent gray mesh. (B) Combined histogram of total OMM curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; ****P &lt; 0.001.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae was regulated and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (the plane of the EM grid substrate on which the cells are grown; Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg-treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Fig. 6 B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Fig. 6 C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position (Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Due to this, we quantified the standard deviation of these values for each mitochondrion in order to assess statistical significance of observed changes (Fig. 6, D and E &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells. Additionally, we observed a significant (based on the Mann-Whitney U test of standard deviations) increase in the variability of the angle of the cristae relative to the OMM for fragmented mitochondria in comparison with elongated mitochondria in Tg-treated cells (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt; and Table S2). We also identified a significant decrease in variability of the angle of the crista body relative to the growth plane in Tg-treated elongated relative to fragmented mitochondria (Fig. 6 E &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM or the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks. (A) Surface membrane reconstruction colored by inner membrane compartments (IBM, tan; junction, cyan; crista, purple) defining angular measurements between the surface and the nearest OMM or to the growth plane of the cell. (B) Two representative membrane surface reconstructions of lamellar Tg-treated elongated mitochondria, colored by the angle of IMM relative to OMM. (C) Two representative membrane surface reconstructions of a less rigidly oriented Tg-treated elongated mitochondria, colored by an angle of IMM relative to the growth plane of the cell. (D) Combined histogram of the angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; **P &lt; 0.01. (E) Combined histogram of the angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01.\n\nApplication of surface morphometrics and analysis to additional organellar membranes\nWe further demonstrated the generality of this approach by generating mesh surfaces for membranes of other organelles present in our data: the ER (Figs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;) and observed multilamellar membrane structures (Fig. S8 &lt;&lt;FIG:figS8&gt;&gt;). We applied our morphometrics pipeline to quantify parameters of membrane architecture of these organelles. The observed multilamellar structures are comprised of multiple membranes (&lt;&lt;REF:Bieber-2022-119-?&gt;&gt;), including the primary membrane and the secondary membrane, as well as interior membranes (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). We then visualized the spatial distributions of the curvatures, distances, and orientations of these distinct membranes on the generated surface reconstructions (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). Additionally, we calculated the curvedness of ER membranes in cells with differing mitochondrial populations and observed no significant changes in any of the conditions (Fig. S10 &lt;&lt;FIG:figS10&gt;&gt;).\nFigure S8.\nGallery of multilamellar membrane structures used for quantifications. Virtual slices of tomograms (left panels) and corresponding reconstructed surface mesh models (right panels) of multilamellar membrane structures. Primary membrane shown in red. Secondary membrane shown in yellow. Interior membranes shown in sage green. Meshes have been tilted 20degrees relative to tomography slice for visualization. Scale bar = 100 nm. n = 3 tomograms.\nFigure S9.\nMultilamellar membrane structure morphometric analysis. (A) Exemplar multilamellar membrane structure surfaces with membranes labeled. Primary membrane in red. Secondary membrane in yellow. Interior membranes in sage green. (B) Secondary and interior membrane surfaces colored by membrane curvedness; the primary membrane is represented as a transparent gray. (C) Primary surface is colored by membrane curvedness; the secondary and interior membranes are represented as a transparent gray. (D) Primary surface is colored by primary-to-secondary membrane distance; the secondary and interior membrane surfaces are shown in transparent gray. (E) Secondary surface colored by secondary to nearest interior (secondary-interior) membrane distance; the primary and interior membrane surfaces are shown in transparent gray. (F) Secondary and interior membranes surfaces are colored by angle relative to the primary membrane. Primary membrane surface is shown in transparent gray. (G) Secondary and interior membrane surfaces are colored by an angle relative to the growth plane of the cell. Primary membrane surface shown in transparent gray. (H) Primary membrane surface is colored by relative angle to the secondary membrane. Secondary and interior membrane surfaces are shown in transparent gray. (I) Primary membrane surface colored by relative angle to the growth plane of the cell. Secondary and interior membranes are shown in transparent gray.\nFigure S10.\nER membrane curvature analysis. (A) Representative membrane surface reconstructions of the ER in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h) and associated with defined mitochondrial populations (elongated and fragmented). The ER surface is colored by membrane curvedness, and the OMM surface is represented as a transparent gray mesh. (B) Combined histograms of total ER curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each ER membrane within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 9, Tg elongated n = 8, vehicle fragmented n = 3, and Tg fragmented n = 4 ER are shown.\n\nMonitoring ER-mitochondrial contacts by surface morphometrics\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission (&lt;&lt;REF:Friedman-2011-334-358&gt;&gt;) to autophagy (&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;). To investigate whether the remodeling we observed in response to Tg-induced ER stress was correlated with alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs. fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present and visualized the contact sites on the surfaces of the OMM (Fig. S11, A and B &lt;&lt;FIG:figS11&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Fig. S11 C &lt;&lt;FIG:figS11&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time point were not associated with the altered mitochondrial ultrastructure (Fig. S11 D &lt;&lt;FIG:figS11&gt;&gt;).\nFigure S11.\nSurface analysis of mitochondria-ER contact sites. (A) Surface membrane reconstruction defining OMM (blue-yellow gradient) and ER (transparent mesh) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM) colored by OMM to ER (OMM-ER) membrane distance. (C) Combined histogram of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 18, Tg elongated n = 11, vehicle fragmented n = 7, Tg fragmented n = 13 mitochondria are shown.\n\nDiscussion\nWe describe a correlative cellular cryo-ET workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influence mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis. The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-ET to reveal organellar membrane ultrastructures (Fig. 1, A-D &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semiautomated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high precision. We determined that the screened Poisson reconstruction approach (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) provides a more accurate estimation of the implicit membrane geometry of complex membranes and can be used to model and quantify complex membranes such as the IMM.\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions and quantified mitochondrial membrane parameters such as inter- and intramitochondrial membrane distances, curvedness, and orientation. We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps. This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions.\nWe first used our workflow to quantify the intermembrane distance between the inner and outer membranes of mitochondria (Fig. 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;; &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;), and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes as well as to quantify changes in the overall distributions. We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). The difference in distance, from 13.5 to 12.7 nm between the centers of the membranes, is small, but correcting for the ~7 nm thickness of mitochondrial membranes (&lt;&lt;REF:Perkins-1997-119-260&gt;&gt;) yields a change in intermembrane contact distance from 6.5 to 5.7 nm or a 12% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as phosphatidylserine decarboxylase, an IMM protein previously suggested to act on the OMM (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbations may impact intermembrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the translocase of the OMM complex (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;), which associate with the MICOS complex and may impact MICOS complex organization (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;).\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;). We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figs. 4 &lt;&lt;FIG:fig4&gt;&gt; and S6 &lt;&lt;FIG:figS6&gt;&gt;), quantified the curvedness of the membranes (Figs. 5 &lt;&lt;FIG:fig5&gt;&gt;, S6 &lt;&lt;FIG:figS6&gt;&gt;, S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, and S10 &lt;&lt;FIG:figS10&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; and 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress (&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt;). In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase--a mechanism similar to that responsible for ER-stress-induced mitochondrial elongation (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvedness. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvedness, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;), ATP synthase (&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;), and OPA1 (&lt;&lt;REF:Hu-2020-11-940&gt;&gt;), and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function.\nWe demonstrated the general utility of our workflow beyond mitochondrial membranes and analyzed the curvedness, intermembrane distances, and orientation of cellular membranes of the ER and observed multilamellar membrane structures (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, S10 &lt;&lt;FIG:figS10&gt;&gt;, and S11 &lt;&lt;FIG:figS11&gt;&gt;). Although our results did not reveal any statistically significant differences in the architecture of these cellular membranes upon Tg-induced ER stress, we anticipate that our workflow will facilitate future studies on the critical importance of membrane remodeling of these organelles induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;), which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) used in this study traces lower curved membranes more easily than high curved membranes, leading to limited segmentations of tight, high curved crista junctions and crista tips. We have mitigated these biases by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter (&lt;&lt;REF:Koning-2018-217-82&gt;&gt;) or phase plate (&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;), both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations (&lt;&lt;REF:Zhou-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Chen-2017-14-983&gt;&gt;; &lt;&lt;REF:Buchholz-2018-?-?&gt;&gt;; &lt;&lt;REF:Liu-2021-?-?&gt;&gt;\nPreprint). The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics (&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt;) and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template-matching approaches (&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;). These approaches have in the past depended on voxel segmentations, but moving to high-quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentation, which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well (&lt;&lt;REF:Basanta-2020-4-100041&gt;&gt;). Membrane meshes have been used very successfully to guide subtomogram averaging in the past (&lt;&lt;REF:Navarro-2018-5-82&gt;&gt;; &lt;&lt;REF:Burt-2021-?-?&gt;&gt;\nPreprint), but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu (&lt;&lt;REF:Erdmann-2021-12-5364&gt;&gt;; &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:O’Reilly-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Guo-2018-172-696&gt;&gt;; &lt;&lt;REF:Jordan-2018-20-1250&gt;&gt;; &lt;&lt;REF:Albert-2020-117-1069&gt;&gt;; &lt;&lt;REF:Lin-2018-360-360&gt;&gt;). Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nMaterials and methods\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM grids\nMouse embryonic fibroblasts with mitochondria-labeled GFP (MEFmtGFP; &lt;&lt;REF:Wang-2012-51-9302&gt;&gt;) were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 mug/ml; Corning) and UV-sterilized R (1/4) Carbon 200-mesh gold EM grids (Quantifoil Micro Tools). After 15-18 h of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group or fresh media supplemented with Thapsigargin (500 nM; Thermo Fisher Scientific) in the Tg treatment group. After 8 h of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific). The Vitrobot was set to 37degreesC and 100% relative humidity, and blotting was performed manually from the back side of grids using Whatman #1 filter paper strips through the Vitrobot humidity/temperature chamber side port. The Vitrobot settings used to disable automated blotting apparatus were as follows: Blot total: 0, 2; Blot force: 0, 3; Blot time: 0 s.\n\nCryo-fluorescence microscopy and mitochondria network morphology scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) and collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em: 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13; Thermo Fisher Scientific) for fluorescence-guided milling.\n\nFluorescence -guided milling\nCryo-FIB milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system for 3-4 s using an automation script (&lt;&lt;REF:Barad-2023-?-?&gt;&gt;), followed by a third and final layer of platinum sputter. Targeted cells were milled manually (&lt;&lt;REF:Schaffer-2015-5-5&gt;&gt;) or using an automated cryo preparation workflow (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;), and both methods used xT software with MAPS (Thermo Fisher Scientific). Minimal SEM imaging for monitoring was done at 2 keV to ensure thin lamella generation while avoiding radiation damage. A total of six treated grids and eight untreated grids were milled for further tomography analysis.\n\nTilt series data collection\nEM grids containing lamellae were transferred into a 300 keV Titan Krios microscope (Thermo Fisher Scientific) equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1 e-/A2) at high magnification to localize cellular regions containing mitochondria, which were identified by their distinctive IMMs and OMMs. Data were collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) with 2degrees steps between -60degrees and +60degrees. Data were collected with pixel sizes of 3.11 , 3.35 , or 3.55 A, and a nominal defocus range between -5 and -10 microm. A subset of data was collected with dose fractionation, with 10 0.1 e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2 and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose-fractionated tilt series micrograph movies underwent contrast transfer function estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series were then aligned using patch tracking in etomo (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;), with four times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into four times binned tomograms. Tomogram thicknesses ranged from 95 to 233 nm.\n\nSemiautomated segmentation\nAutomated detection and enhancement of mitochondrial membranes were performed using the TomoSegMemTV software package (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;). Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows: scale_space -s 3; dtvoting -s 12; surfaceness -m 0.17 -s 0.8 -p 0.8; dtvoting -w -s 10; surfaceness -S -s 0.75 -p 0.75 -l 20; thresholding -l 0.01-2 30; global analysis -3 100.\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher Scientific) for manual curation. Mitochondrial inner and outer membranes and ER were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nSoftware workflow\nThe surface morphometrics pipeline is a python-scripted workflow with requirements that can be installed as a conda environment contained in an \"environment.yml\" file. The workflow is fully scripted and configurable with a \"config.yml\" file and is run in three fully pipelined steps, with statistical analysis and visualization as an optional fourth step. First, a segmentation MRC file is converted automatically to a series of surface meshes formatted in the VTP file format. Second, for each mesh, the surface is converted to a graph (tg format) and curvature is estimated using pycurv. Third, orientations and distances between and within surfaces are calculated using the resulting graphs, and a CSV with quantifications as well as a final VTP surface file is output with all quantifications built in. Fourth, the outputs from multiple tomograms are combined for visualization and statistical analysis. Times and computational requirements are shown in Table S1.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab (&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;). First, the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;), with a reconstruction depth of 9, an interpolation weight of 0.7, and a minimum number of samples of 1.5. These settings were chosen to maximize correspondence to the data rather than smoothness. The resulting surface extended beyond the segmented region, so triangles more than 1.5 nm away from the point cloud were deleted. Interpolation weight (point_weight) and the mask distance (extrapolation_distance) are both configurable in the surface morphometrics pipeline if more aggressive smoothing and hole filling are desirable. The resulting mesh was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the \"segmentation_to_meshes.py\" script.\n\nRefinement of surface orientation and estimation of surface curvedness using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). This vector voting process also resulted in robust estimation of curvature metrics. As the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (Eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvedness metrics, which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the \"run_pycurv.py\" script.curvedness=kappa12+kappa222\n(1)\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated using a k-dimensional tree. Both the distance and identifier of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the \"measure_distances_orientations.py\" script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those &gt;40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nAs all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously (&lt;&lt;REF:Collado-2019-51-476&gt;&gt;). To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into shorter and longer distances. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector-based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector-based neighbor. Those triangles were filtered out before any further quantifications. These measurements are made in the \"measure_distances_orientations.py\" script.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (Eq. 2):verticality=arccos(|normalz|)\n(2)\nComparisons of orientations between membranes were made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector to its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (Eq. 3):relative_angle=arccos|self.neighbor|\n(3)\nBoth orientation calculations are determined in the \"measure_distances_orientations.py\" script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization pipeline \"vtp\" (&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt;) format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview (&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;). Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area (&lt;&lt;REF:Hunter-2007-9-90&gt;&gt;). Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests (&lt;&lt;REF:Mann-1947-18-50&gt;&gt;) on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram rather than the histogram peak. Statistical testing based on distributions, rather than peaks, is complicated by unbiased assessment of the number of independent samples in each tomogram. Aggregation, statistics, and visualization are accomplished by the \"morphometrics_stats.py\" classes and functions, and the mitochondria-specific statistics and visualizations are generated by the \"mitochondria_statistics.py\" script. The complete output of the per mitochondrion analyses for every measurement is available at https://zenodo.org/record/7510830.\n\nOnline supplemental material\nFigs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt; are a gallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress, cells with elongated mitochondrial networks in the presence of ER stress, and cells with fragmented mitochondrial networks in the presence and absence of ER stress, respectively. Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; shows a comparison of different surface reconstruction algorithms. Fig. S5 &lt;&lt;FIG:figS5&gt;&gt; is a gallery of histograms showing the distribution of IMM-OMM distances for each individual mitochondrion. Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; presents additional quantifications that do not show statistically significant differences. Fig. S7 &lt;&lt;FIG:figS7&gt;&gt; shows analysis of changes to OMM curvature. Fig. S8 &lt;&lt;FIG:figS8&gt;&gt; shows a gallery of multilamellar membrane bodies observed in the data and their segmentations. Fig. S9 &lt;&lt;FIG:figS9&gt;&gt; shows a gallery of surface morphometrics pipeline quantifications mapped onto the multilamellar membrane bodies. Fig. S10 &lt;&lt;FIG:figS10&gt;&gt; shows analysis of changes to ER curvature. Fig. S11 &lt;&lt;FIG:figS11&gt;&gt; shows quantification of mitochondria-ER contact sites. Table S1 shows typical computational requirements and time required for each step of the surface morphometrics pipeline. Table S2 shows additional statistical assessments for each quantification and each condition. Table S3 shows the confidence intervals derived from bootstrap analysis of each quantification. Video 1 &lt;&lt;FIG:video1&gt;&gt; shows a comparison of voxel segmentation and implicit surface reconstruction.\n\nSupplementary Material\nReview History\nClick here for additional data file.\nTable S1\nshows the approximate time for each step of the surface morphometrics workflow.\nClick here for additional data file.\nTable S2\nshows the statistical analyses of peaks and standard deviations.\nClick here for additional data file.\nTable S3\nshows the confidence intervals of quantifications with bootstrap analysis.\nClick here for additional data file.\n\nData availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Installation is simplified with a conda environment and a pip requirements file. Scripts are licensed under the GNU Public License version 3 (GPLv3). All tilt series, reconstructed tomograms, voxel segmentations, and reconstructed mesh surfaces used for quantifications were deposited in the Electron Microscopy Public Image Archive under accession code EMPIAR-11370.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, 3D views of protein and organellar structure in unfixed and fully hydrated conditions (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;). By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;; &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;), it has become routine to collect large datasets of thin, cellular 3D reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells (&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;; &lt;&lt;REF:Himes-2018-15-955&gt;&gt;; &lt;&lt;REF:Chen-2019-16-1161&gt;&gt;), it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments (&lt;&lt;REF:Jiang-2017-7-45474&gt;&gt;; &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;). As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the 2D virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intramembrane distances through manual designation of membrane boundaries by locating distinct points along the membrane (&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;; &lt;&lt;REF:Ader-2019-8-?&gt;&gt;; &lt;&lt;REF:Tran-2021-374-52&gt;&gt;; &lt;&lt;REF:Mageswaran-2021-7-7&gt;&gt;). Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Toward a more accurate 3D representation, manual or automated (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvedness and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvedness. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvedness as well as the spatial relationship between individual triangles (i.e., membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e., boundary segmentations) were not reliable due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm (&lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;). In cryo-ET, even perfect segmentations will never show complete and closed membranes due to the substantial missing wedge (&lt;&lt;REF:Lučič-2013-202-407&gt;&gt;), making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e., compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time-intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-ET.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling (&lt;&lt;REF:Schirrmacher-2020-8-8&gt;&gt;; &lt;&lt;REF:West-2017-17-363&gt;&gt;; &lt;&lt;REF:Wai-2016-27-105&gt;&gt;). Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure; &lt;&lt;REF:Chan-2020-15-235&gt;&gt;). Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e., elongated) or disconnected (i.e., fragmented) networks that regulate prosurvival or proapoptotic mitochondrial functions (&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;; &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;). Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions (&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;).\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvedness required for efficient mitochondrial respiration, (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Blum-2019-116-4250&gt;&gt;). Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;; &lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;). Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;). It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-ET workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semiautomated workflow (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) to segment cellular membranes present in cryo-tomograms and applied a screened Poisson reconstruction strategy (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics pipeline to measure several parameters of membrane shape including inter- and intramembrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of ER stress, a condition known to promote adaptive remodeling of mitochondrial networks (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;), to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nResults\n\nCorrelative cryo-fluorescence microscopy and electron tomography enable association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP) such that the morphology of each cell's mitochondrial network could be readily assessed (Fig. 1, A-C &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification: Five researchers were given unlabeled fluorescence microscopy images of exemplar network morphologies (elongated and fragmented) as references to assign morphologies to the experimental fluorescence micrographs. We targeted cells with either fragmented or elongated mitochondrial networks for cryo-FIB milling to prepare thin (95-233 nm) lamellae (Fig. 1 D &lt;&lt;FIG:fig1&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Fig. 1 C &lt;&lt;FIG:fig1&gt;&gt;) to the ultrastructure of mitochondrial membranes (Fig. 1, F-H &lt;&lt;FIG:fig1&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). We applied our correlative approach to identify and target specific Tg-treated and vehicle (media with DMSO)-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). Notably, Tg treatment induced a shift toward regularly spaced disc-shaped (lamellar) cristae in the elongated population and toward a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-ET workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions. (A) MEFmtGFP cells were cultured on transmission EM grids (black mesh circle) and treated with vehicle or Tg (500 nM) for 8 h prior to vitrification via plunge freezing in ethane/propane mixture. (B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence microscopy (cryo-FM). (C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies. (D) These cells were targeted for cryo-FIB milling to generate thin sections (lamellae). (E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate 3D cryo-tomograms. (F) Voxel segmentations labeling distinct cellular membranes were generated using a semiautomated approach (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point (&lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; &lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). (G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: IMM, purple; OMM, blue; ER, teal. (H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP\ncells. (A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies from mixed populations of MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of 95-233 nm. MEFmtGFP cell periphery is outlined in dashed white line. (B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue. (C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backward 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\nFigure S1.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"UE1\" is untreated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with elongated mitochondrial networks, n = 10 tomograms.\nFigure S2.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TE1\" is Tg-treated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Tg-treated cells  with elongated mitochondrial networks, n = 13 tomograms.\nFigure S3.\nGallery of cellular membranes used for downstream quantifications from cells with fragmented mitochondrial networks in the absence and presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TF1\" is Tg-treated, fragmented, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with fragmented mitochondrial networks, n = 6 tomograms; and Tg-treated cells with fragmented mitochondrial networks, n = 5 tomograms.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semiautomated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;) that established a robust approach for estimating curvedness in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high-quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;; &lt;&lt;REF:Navarro-2018-5-82&gt;&gt;) and manual inpainting of entire compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;), both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semiautomated workflow to enable analysis at higher throughput (Fig. 1 F &lt;&lt;FIG:fig1&gt;&gt;). We used a segmentation strategy that takes advantage of the TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), which identifies and enhances regions of the tomogram with membrane-like voxel values. This was followed by manual labeling of membranes into mitochondrial IMM and OMM, ER, and multilamellar membrane based on cellular context, as well as manual cleanup of individual membrane segmentation using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 34 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;, Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). The segmentation output was fed into the fully automated surface morphometrics pipeline (Fig. 2 B &lt;&lt;FIG:fig2&gt;&gt;). The voxel segmentation was converted to high-quality membrane surfaces using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; and Video 1 &lt;&lt;FIG:video1&gt;&gt;). Next, these surfaces were converted into triangle graphs, and multiple membrane features were quantified including curvedness (estimated using pycurv15), distances within and between surfaces, and the relative orientations of different surfaces. Finally, the quantifications for each mitochondrion were combined into experiments to allow aggregate statistics and visualizations. Results were computed from 34 completed segmentations in a few hours using a cluster for parallelization (Table S1). This 3D surface morphometrics pipeline is configurable for any segmented membrane and is available at https://github.com/grotjahnlab/surface_morphometrics.\nFigure S4.\nComparison of surface reconstruction algorithms. The same IMM segmentation was subjected to the boundary segmentation surface reconstruction algorithm used in the pycurv software (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;; &lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;; left) as well as our new reconstruction approach based on the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; right) show the improvements in smoothness and completeness enabled by the new surface reconstruction algorithm. While the left algorithm was made part of the pycurv software, it was not used for extensive analysis in the cited manuscript, in favor of algorithms using compartment segmentations.\nVideo 1.\nComparison of voxel segmentation vs. surface reconstruction. An example tomogram is shown with and without a completed voxel segmentation (green). A zoom in reveals voxel stepping artifacts that impact measurement precision, as well as small holes that result from automated segmentation. Switching to the reconstructed surface shows the smooth triangle mesh and the hole-filling properties of the surface reconstruction algorithm. Surface colors: IMM in pink, OMM in purple, ER in blue.\n\nBulk analysis and assessment of statistical significance based on surface quantifications\nOur pipeline outputs multiple membrane feature quantifications (i.e., curvature, distance, and orientation) for each triangle, totaling ~500,000 triangles per tomogram. We set out to determine an appropriate method for bulk quantitative and statistical analyses that would enable detection of subtle albeit potentially significant differences in membrane ultrastructure across different mitochondrial morphologies and treatment groups. We plotted the overall distributions of each membrane feature quantification using histograms of triangle values that are weighted by the area of each triangle, both for individual mitochondrion (Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;) and for entire experiments. These histograms identify bulk differences in the membrane ultrastructure of mitochondria under different experimental conditions. For several membrane parameter quantifications, the observed histogram peak shifted visibly between conditions, suggesting potential differences between treatment and morphology groups. In order to test the statistical significance of these shifts, the weighted histogram peak for each membrane feature quantification for each individual mitochondrion was extracted and these peaks were used for statistical comparisons. In most cases, there was a statistically significant difference in the mean of the distributions, which was assessed via a non-parametric Mann-Whitney U test (Table S2). Differences in the variability of metrics between mitochondria within a given experimental condition were assessed with a two-sample Kolmogorov-Smirnoff test.\nFigure S5.\nGallery of IMM-OMM distance histograms for individual mitochondria. The distribution of distances between the IMM and OMM was plotted for each mitochondrion individually for all Tg-treated, elongated (TE), Tg-treated, fragmented (TF), vehicle (i.e., untreated), elongated (UE), and vehicle (i.e., untreated), fragmented (UF). The peak position is plotted for each and is used for the violin plot in Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;. The histograms are titled based on the measurement (i.e., IMM-IMM Dist) and correspond to the treatment conditions, morphologies, and individual mitochondria labels shown in Fig. S1 &lt;&lt;FIG:figS1&gt;&gt; (e.g., \"TE2a\" corresponds to a Tg-treated [T], elongated [E], tomogram \"2,\" mitochondrion labeled \"a\").\nIn some cases, there was an apparent change to the shape of distribution seen in quantification histograms rather than simply peak position. We primarily assessed the statistical significance of these changes by Mann-Whitney U tests and Kolmogorov-Smirnoff tests on the per-mitochondria standard deviation rather than histogram peak (Table S2). This metric captured changes to the variance of metrics. In order to quantify changes that combine variance and peak shifts, we used the two-sample Kolmogorov-Smirnoff test to test the statistical significance of changes to the distributions of mesh triangles in each condition. However, a major challenge of utilizing this statistical test for analysis of surface quantifications is assessing the true number of independent measurements being made. Triangles are highly correlated to their neighboring meshes, a correlation made even stronger by the vector voting approach employed by pycurv in order to better approximate a smooth membrane. When each triangle within a surface mesh is treated as an independent measurement (in some cases yielding sample sizes in the millions), even visibly identical distributions yield statistically significant P values. This is even further complicated because the number of triangles generated in a given surface mesh is arbitrary and not based on any physical constraint. In contrast, if each mitochondrion is treated as an independent measurement, the variance of features within the mitochondria is lost and there is essentially never statistical significance. We established a heuristic compromise based on the feature radius of 12 nm used in vector voting to optimally measure curvatures. The number of independent samples was assumed to be one per 452 nm2 of membrane area, corresponding to the 12-nm feature size. In practice, this area corresponds to the area of a single crista tip or junction and yields results with P values that might be effectively used comparatively between different measurements. Altering the feature size to 9 or 15 nm changes the number of independent samples and absolute P values; however the relative P value differences across groups do not change, demonstrating that absolute value for assessing statistical significance is limited by the artificial selection of feature radius, but the choice of feature size does not affect the overall result of the comparative analyses across different membrane feature parameters (Table S2).\nA challenge with using histogram peaks as a metric is determining the confidence interval associated with the metric. In order to assess the precision of the histogram peak measurements, we resampled each distribution using bootstrap sampling 1,000 times, and the histogram peak was extracted on each resampling. The 5th and 95th percentiles of these histogram peaks were used to represent the confidence interval of the histogram peak (Table S3).\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria is the juxtaposed inner and outer mitochondrial membranes. Cryo-ET is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh for each mitochondrion in our dataset (Fig. 3 A &lt;&lt;FIG:fig3&gt;&gt; and Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented mitochondria, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Fig. 3 C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in intermembrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak position and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each mitochondrion as independent observations and identified statistically significant increases in spacing in cells with fragmented vs. elongated mitochondria by 2.4 nm in the vehicle-treated condition and by 3.6 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance between OMM and IMM of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in cells with fragmented mitochondria after Tg treatment was not significant (Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significant differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nDistance between IMMs and OMMs is dependent on mitochondrial network morphology and presence or absence of ER stress. (A) Surface membrane reconstruction defining OMM (purple) and IMM (orange) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions. (C) Combined histogram of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Quantification of peak histogram values from each mitochondrion within the indicated treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;). We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intracrista and intrajunction distances, respectively), as well as the distance between adjacent crista bodies and junctions (intercrista and interjunction distances, respectively; Fig. 4 B &lt;&lt;FIG:fig4&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each mitochondria (Fig. 4, C and D &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intracrista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 5.3 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 1.8 nm upon Tg treatment. However, we did not observe significant changes in intercristae distance, intrajunction distance, or interjunction distance, with the exception of a 3.7 nm decrease in intrajunction spacing in the Tg-treated condition of elongated mitochondria relative to vehicle elongated mitochondria, suggesting that ER stress promotes rigidification of the junctions in elongated mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations. (A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with Tg (500 nM, 8 h) showing subdivision of IMM compartments (IBM, tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh. (B) Enlarged inset of boxed region in elongated example from A showing surface membrane reconstruction colored by inner membrane compartments defining intra- and intercrista and junction measurements. (C) Combined histogram of intracrista membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ***P &lt; 0.005. (D) Combined histogram of intrajunction membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05.\nFigure S6.\nAdditional quantifications of membrane ultrastructure. Additional quantifications output by the surface morphometrics pipeline, which showed non-significant changes or no apparent change. In all cases, quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. (A) Histograms and violin plot showing changes to spacing between closest neighbor segments of IMM. (B) Histograms and violin plot showing changes to spacing to the second nearest segments of IMM. (C) Histograms and violin plot showing changes to spacing between neighboring cristae. (D) Histograms and violin plot showing changes to spacing between closest neighbor segments of IBM. (E) Histograms and violin plot showing changes to spacing to the second nearest segments of IBM. (F) Histograms and violin plot showing changes to spacing between neighboring junctions. (G) Histograms and violin plot showing changes to curvedness of the IBM. (H) Histograms and violin plot showing changes to the angle between IMM and OMM. (I) Histograms and violin plot showing changes to the angle between IBM and OMM. (J) Histograms and violin plot showing changes to the angle between crista junctions and OMM. (K) Histograms and violin plot showing changes to the angle between IMM and the growth plane. (L) Histograms and violin plot showing changes to the angle between IBM and the growth plane. (M) Histograms and violin plot showing changes to the angle between crista junctions and the growth plane. *P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.005; ****P &lt; 0.001.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvedness (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;; &lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated cells with elongated mitochondria, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). A significant change was observed between the vehicle-elongated and vehicle-fragmented mitochondria, which established a basal distinction in curvedness among mitochondrial populations (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). As visual inspection revealed local changes in curvedness in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Fig. 5, C and D &lt;&lt;FIG:fig5&gt;&gt;).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principal components of curvedness and is used because the surface normal vectors do not have canonical sign. (B) Combined histogram of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01. (C) Combined histogram of cristae curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; **P &lt; 0.01. (D) Combined histogram of cristae junction curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\nLarger changes in curvature were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress. Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated cells with elongated mitochondria compared with both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria. This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nOMM curvedness is dependent on mitochondrial network morphology\nIn addition to the IMM, we calculated curvedness of the OMM for each mitochondrion. For both vehicle and Tg-treated cells, curvedness of the OMM is decreased in fragmented mitochondria relative to elongated networks (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;). This metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length.\nFigure S7.\nOMM curvature analysis. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by membrane curvedness and the IMM surface is represented as a transparent gray mesh. (B) Combined histogram of total OMM curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; ****P &lt; 0.001.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae was regulated and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (the plane of the EM grid substrate on which the cells are grown; Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg-treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Fig. 6 B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Fig. 6 C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position (Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Due to this, we quantified the standard deviation of these values for each mitochondrion in order to assess statistical significance of observed changes (Fig. 6, D and E &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells. Additionally, we observed a significant (based on the Mann-Whitney U test of standard deviations) increase in the variability of the angle of the cristae relative to the OMM for fragmented mitochondria in comparison with elongated mitochondria in Tg-treated cells (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt; and Table S2). We also identified a significant decrease in variability of the angle of the crista body relative to the growth plane in Tg-treated elongated relative to fragmented mitochondria (Fig. 6 E &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM or the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks. (A) Surface membrane reconstruction colored by inner membrane compartments (IBM, tan; junction, cyan; crista, purple) defining angular measurements between the surface and the nearest OMM or to the growth plane of the cell. (B) Two representative membrane surface reconstructions of lamellar Tg-treated elongated mitochondria, colored by the angle of IMM relative to OMM. (C) Two representative membrane surface reconstructions of a less rigidly oriented Tg-treated elongated mitochondria, colored by an angle of IMM relative to the growth plane of the cell. (D) Combined histogram of the angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; **P &lt; 0.01. (E) Combined histogram of the angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01.\n\nApplication of surface morphometrics and analysis to additional organellar membranes\nWe further demonstrated the generality of this approach by generating mesh surfaces for membranes of other organelles present in our data: the ER (Figs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;) and observed multilamellar membrane structures (Fig. S8 &lt;&lt;FIG:figS8&gt;&gt;). We applied our morphometrics pipeline to quantify parameters of membrane architecture of these organelles. The observed multilamellar structures are comprised of multiple membranes (&lt;&lt;REF:Bieber-2022-119-?&gt;&gt;), including the primary membrane and the secondary membrane, as well as interior membranes (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). We then visualized the spatial distributions of the curvatures, distances, and orientations of these distinct membranes on the generated surface reconstructions (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). Additionally, we calculated the curvedness of ER membranes in cells with differing mitochondrial populations and observed no significant changes in any of the conditions (Fig. S10 &lt;&lt;FIG:figS10&gt;&gt;).\nFigure S8.\nGallery of multilamellar membrane structures used for quantifications. Virtual slices of tomograms (left panels) and corresponding reconstructed surface mesh models (right panels) of multilamellar membrane structures. Primary membrane shown in red. Secondary membrane shown in yellow. Interior membranes shown in sage green. Meshes have been tilted 20degrees relative to tomography slice for visualization. Scale bar = 100 nm. n = 3 tomograms.\nFigure S9.\nMultilamellar membrane structure morphometric analysis. (A) Exemplar multilamellar membrane structure surfaces with membranes labeled. Primary membrane in red. Secondary membrane in yellow. Interior membranes in sage green. (B) Secondary and interior membrane surfaces colored by membrane curvedness; the primary membrane is represented as a transparent gray. (C) Primary surface is colored by membrane curvedness; the secondary and interior membranes are represented as a transparent gray. (D) Primary surface is colored by primary-to-secondary membrane distance; the secondary and interior membrane surfaces are shown in transparent gray. (E) Secondary surface colored by secondary to nearest interior (secondary-interior) membrane distance; the primary and interior membrane surfaces are shown in transparent gray. (F) Secondary and interior membranes surfaces are colored by angle relative to the primary membrane. Primary membrane surface is shown in transparent gray. (G) Secondary and interior membrane surfaces are colored by an angle relative to the growth plane of the cell. Primary membrane surface shown in transparent gray. (H) Primary membrane surface is colored by relative angle to the secondary membrane. Secondary and interior membrane surfaces are shown in transparent gray. (I) Primary membrane surface colored by relative angle to the growth plane of the cell. Secondary and interior membranes are shown in transparent gray.\nFigure S10.\nER membrane curvature analysis. (A) Representative membrane surface reconstructions of the ER in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h) and associated with defined mitochondrial populations (elongated and fragmented). The ER surface is colored by membrane curvedness, and the OMM surface is represented as a transparent gray mesh. (B) Combined histograms of total ER curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each ER membrane within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 9, Tg elongated n = 8, vehicle fragmented n = 3, and Tg fragmented n = 4 ER are shown.\n\nMonitoring ER-mitochondrial contacts by surface morphometrics\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission (&lt;&lt;REF:Friedman-2011-334-358&gt;&gt;) to autophagy (&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;). To investigate whether the remodeling we observed in response to Tg-induced ER stress was correlated with alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs. fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present and visualized the contact sites on the surfaces of the OMM (Fig. S11, A and B &lt;&lt;FIG:figS11&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Fig. S11 C &lt;&lt;FIG:figS11&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time point were not associated with the altered mitochondrial ultrastructure (Fig. S11 D &lt;&lt;FIG:figS11&gt;&gt;).\nFigure S11.\nSurface analysis of mitochondria-ER contact sites. (A) Surface membrane reconstruction defining OMM (blue-yellow gradient) and ER (transparent mesh) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM) colored by OMM to ER (OMM-ER) membrane distance. (C) Combined histogram of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 18, Tg elongated n = 11, vehicle fragmented n = 7, Tg fragmented n = 13 mitochondria are shown.\n\nDiscussion\nWe describe a correlative cellular cryo-ET workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influence mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis. The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-ET to reveal organellar membrane ultrastructures (Fig. 1, A-D &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semiautomated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high precision. We determined that the screened Poisson reconstruction approach (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) provides a more accurate estimation of the implicit membrane geometry of complex membranes and can be used to model and quantify complex membranes such as the IMM.\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions and quantified mitochondrial membrane parameters such as inter- and intramitochondrial membrane distances, curvedness, and orientation. We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps. This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions.\nWe first used our workflow to quantify the intermembrane distance between the inner and outer membranes of mitochondria (Fig. 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;; &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;), and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes as well as to quantify changes in the overall distributions. We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). The difference in distance, from 13.5 to 12.7 nm between the centers of the membranes, is small, but correcting for the ~7 nm thickness of mitochondrial membranes (&lt;&lt;REF:Perkins-1997-119-260&gt;&gt;) yields a change in intermembrane contact distance from 6.5 to 5.7 nm or a 12% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as phosphatidylserine decarboxylase, an IMM protein previously suggested to act on the OMM (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbations may impact intermembrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the translocase of the OMM complex (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;), which associate with the MICOS complex and may impact MICOS complex organization (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;).\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;). We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figs. 4 &lt;&lt;FIG:fig4&gt;&gt; and S6 &lt;&lt;FIG:figS6&gt;&gt;), quantified the curvedness of the membranes (Figs. 5 &lt;&lt;FIG:fig5&gt;&gt;, S6 &lt;&lt;FIG:figS6&gt;&gt;, S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, and S10 &lt;&lt;FIG:figS10&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; and 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress (&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt;). In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase--a mechanism similar to that responsible for ER-stress-induced mitochondrial elongation (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvedness. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvedness, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;), ATP synthase (&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;), and OPA1 (&lt;&lt;REF:Hu-2020-11-940&gt;&gt;), and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function.\nWe demonstrated the general utility of our workflow beyond mitochondrial membranes and analyzed the curvedness, intermembrane distances, and orientation of cellular membranes of the ER and observed multilamellar membrane structures (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, S10 &lt;&lt;FIG:figS10&gt;&gt;, and S11 &lt;&lt;FIG:figS11&gt;&gt;). Although our results did not reveal any statistically significant differences in the architecture of these cellular membranes upon Tg-induced ER stress, we anticipate that our workflow will facilitate future studies on the critical importance of membrane remodeling of these organelles induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;), which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) used in this study traces lower curved membranes more easily than high curved membranes, leading to limited segmentations of tight, high curved crista junctions and crista tips. We have mitigated these biases by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter (&lt;&lt;REF:Koning-2018-217-82&gt;&gt;) or phase plate (&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;), both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations (&lt;&lt;REF:Zhou-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Chen-2017-14-983&gt;&gt;; &lt;&lt;REF:Buchholz-2018-?-?&gt;&gt;; &lt;&lt;REF:Liu-2021-?-?&gt;&gt;\nPreprint). The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics (&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt;) and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template-matching approaches (&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;). These approaches have in the past depended on voxel segmentations, but moving to high-quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentation, which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well (&lt;&lt;REF:Basanta-2020-4-100041&gt;&gt;). Membrane meshes have been used very successfully to guide subtomogram averaging in the past (&lt;&lt;REF:Navarro-2018-5-82&gt;&gt;; &lt;&lt;REF:Burt-2021-?-?&gt;&gt;\nPreprint), but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu (&lt;&lt;REF:Erdmann-2021-12-5364&gt;&gt;; &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:O’Reilly-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Guo-2018-172-696&gt;&gt;; &lt;&lt;REF:Jordan-2018-20-1250&gt;&gt;; &lt;&lt;REF:Albert-2020-117-1069&gt;&gt;; &lt;&lt;REF:Lin-2018-360-360&gt;&gt;). Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nMaterials and methods\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM grids\nMouse embryonic fibroblasts with mitochondria-labeled GFP (MEFmtGFP; &lt;&lt;REF:Wang-2012-51-9302&gt;&gt;) were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 mug/ml; Corning) and UV-sterilized R (1/4) Carbon 200-mesh gold EM grids (Quantifoil Micro Tools). After 15-18 h of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group or fresh media supplemented with Thapsigargin (500 nM; Thermo Fisher Scientific) in the Tg treatment group. After 8 h of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific). The Vitrobot was set to 37degreesC and 100% relative humidity, and blotting was performed manually from the back side of grids using Whatman #1 filter paper strips through the Vitrobot humidity/temperature chamber side port. The Vitrobot settings used to disable automated blotting apparatus were as follows: Blot total: 0, 2; Blot force: 0, 3; Blot time: 0 s.\n\nCryo-fluorescence microscopy and mitochondria network morphology scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) and collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em: 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13; Thermo Fisher Scientific) for fluorescence-guided milling.\n\nFluorescence -guided milling\nCryo-FIB milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system for 3-4 s using an automation script (&lt;&lt;REF:Barad-2023-?-?&gt;&gt;), followed by a third and final layer of platinum sputter. Targeted cells were milled manually (&lt;&lt;REF:Schaffer-2015-5-5&gt;&gt;) or using an automated cryo preparation workflow (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;), and both methods used xT software with MAPS (Thermo Fisher Scientific). Minimal SEM imaging for monitoring was done at 2 keV to ensure thin lamella generation while avoiding radiation damage. A total of six treated grids and eight untreated grids were milled for further tomography analysis.\n\nTilt series data collection\nEM grids containing lamellae were transferred into a 300 keV Titan Krios microscope (Thermo Fisher Scientific) equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1 e-/A2) at high magnification to localize cellular regions containing mitochondria, which were identified by their distinctive IMMs and OMMs. Data were collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) with 2degrees steps between -60degrees and +60degrees. Data were collected with pixel sizes of 3.11 , 3.35 , or 3.55 A, and a nominal defocus range between -5 and -10 microm. A subset of data was collected with dose fractionation, with 10 0.1 e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2 and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose-fractionated tilt series micrograph movies underwent contrast transfer function estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series were then aligned using patch tracking in etomo (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;), with four times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into four times binned tomograms. Tomogram thicknesses ranged from 95 to 233 nm.\n\nSemiautomated segmentation\nAutomated detection and enhancement of mitochondrial membranes were performed using the TomoSegMemTV software package (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;). Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows: scale_space -s 3; dtvoting -s 12; surfaceness -m 0.17 -s 0.8 -p 0.8; dtvoting -w -s 10; surfaceness -S -s 0.75 -p 0.75 -l 20; thresholding -l 0.01-2 30; global analysis -3 100.\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher Scientific) for manual curation. Mitochondrial inner and outer membranes and ER were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nSoftware workflow\nThe surface morphometrics pipeline is a python-scripted workflow with requirements that can be installed as a conda environment contained in an \"environment.yml\" file. The workflow is fully scripted and configurable with a \"config.yml\" file and is run in three fully pipelined steps, with statistical analysis and visualization as an optional fourth step. First, a segmentation MRC file is converted automatically to a series of surface meshes formatted in the VTP file format. Second, for each mesh, the surface is converted to a graph (tg format) and curvature is estimated using pycurv. Third, orientations and distances between and within surfaces are calculated using the resulting graphs, and a CSV with quantifications as well as a final VTP surface file is output with all quantifications built in. Fourth, the outputs from multiple tomograms are combined for visualization and statistical analysis. Times and computational requirements are shown in Table S1.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab (&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;). First, the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;), with a reconstruction depth of 9, an interpolation weight of 0.7, and a minimum number of samples of 1.5. These settings were chosen to maximize correspondence to the data rather than smoothness. The resulting surface extended beyond the segmented region, so triangles more than 1.5 nm away from the point cloud were deleted. Interpolation weight (point_weight) and the mask distance (extrapolation_distance) are both configurable in the surface morphometrics pipeline if more aggressive smoothing and hole filling are desirable. The resulting mesh was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the \"segmentation_to_meshes.py\" script.\n\nRefinement of surface orientation and estimation of surface curvedness using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). This vector voting process also resulted in robust estimation of curvature metrics. As the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (Eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvedness metrics, which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the \"run_pycurv.py\" script.curvedness=kappa12+kappa222\n(1)\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated using a k-dimensional tree. Both the distance and identifier of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the \"measure_distances_orientations.py\" script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those &gt;40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nAs all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously (&lt;&lt;REF:Collado-2019-51-476&gt;&gt;). To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into shorter and longer distances. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector-based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector-based neighbor. Those triangles were filtered out before any further quantifications. These measurements are made in the \"measure_distances_orientations.py\" script.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (Eq. 2):verticality=arccos(|normalz|)\n(2)\nComparisons of orientations between membranes were made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector to its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (Eq. 3):relative_angle=arccos|self.neighbor|\n(3)\nBoth orientation calculations are determined in the \"measure_distances_orientations.py\" script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization pipeline \"vtp\" (&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt;) format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview (&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;). Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area (&lt;&lt;REF:Hunter-2007-9-90&gt;&gt;). Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests (&lt;&lt;REF:Mann-1947-18-50&gt;&gt;) on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram rather than the histogram peak. Statistical testing based on distributions, rather than peaks, is complicated by unbiased assessment of the number of independent samples in each tomogram. Aggregation, statistics, and visualization are accomplished by the \"morphometrics_stats.py\" classes and functions, and the mitochondria-specific statistics and visualizations are generated by the \"mitochondria_statistics.py\" script. The complete output of the per mitochondrion analyses for every measurement is available at https://zenodo.org/record/7510830.\n\nOnline supplemental material\nFigs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt; are a gallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress, cells with elongated mitochondrial networks in the presence of ER stress, and cells with fragmented mitochondrial networks in the presence and absence of ER stress, respectively. Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; shows a comparison of different surface reconstruction algorithms. Fig. S5 &lt;&lt;FIG:figS5&gt;&gt; is a gallery of histograms showing the distribution of IMM-OMM distances for each individual mitochondrion. Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; presents additional quantifications that do not show statistically significant differences. Fig. S7 &lt;&lt;FIG:figS7&gt;&gt; shows analysis of changes to OMM curvature. Fig. S8 &lt;&lt;FIG:figS8&gt;&gt; shows a gallery of multilamellar membrane bodies observed in the data and their segmentations. Fig. S9 &lt;&lt;FIG:figS9&gt;&gt; shows a gallery of surface morphometrics pipeline quantifications mapped onto the multilamellar membrane bodies. Fig. S10 &lt;&lt;FIG:figS10&gt;&gt; shows analysis of changes to ER curvature. Fig. S11 &lt;&lt;FIG:figS11&gt;&gt; shows quantification of mitochondria-ER contact sites. Table S1 shows typical computational requirements and time required for each step of the surface morphometrics pipeline. Table S2 shows additional statistical assessments for each quantification and each condition. Table S3 shows the confidence intervals derived from bootstrap analysis of each quantification. Video 1 &lt;&lt;FIG:video1&gt;&gt; shows a comparison of voxel segmentation and implicit surface reconstruction.\n\nSupplementary Material\nReview History\nClick here for additional data file.\nTable S1\nshows the approximate time for each step of the surface morphometrics workflow.\nClick here for additional data file.\nTable S2\nshows the statistical analyses of peaks and standard deviations.\nClick here for additional data file.\nTable S3\nshows the confidence intervals of quantifications with bootstrap analysis.\nClick here for additional data file.\n\nData availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Installation is simplified with a conda environment and a pip requirements file. Scripts are licensed under the GNU Public License version 3 (GPLv3). All tilt series, reconstructed tomograms, voxel segmentations, and reconstructed mesh surfaces used for quantifications were deposited in the Electron Microscopy Public Image Archive under accession code EMPIAR-11370.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, 3D views of protein and organellar structure in unfixed and fully hydrated conditions (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;). By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;; &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;), it has become routine to collect large datasets of thin, cellular 3D reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells (&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;; &lt;&lt;REF:Himes-2018-15-955&gt;&gt;; &lt;&lt;REF:Chen-2019-16-1161&gt;&gt;), it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments (&lt;&lt;REF:Jiang-2017-7-45474&gt;&gt;; &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;). As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the 2D virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intramembrane distances through manual designation of membrane boundaries by locating distinct points along the membrane (&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;; &lt;&lt;REF:Ader-2019-8-?&gt;&gt;; &lt;&lt;REF:Tran-2021-374-52&gt;&gt;; &lt;&lt;REF:Mageswaran-2021-7-7&gt;&gt;). Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Toward a more accurate 3D representation, manual or automated (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvedness and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvedness. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvedness as well as the spatial relationship between individual triangles (i.e., membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e., boundary segmentations) were not reliable due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm (&lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;). In cryo-ET, even perfect segmentations will never show complete and closed membranes due to the substantial missing wedge (&lt;&lt;REF:Lučič-2013-202-407&gt;&gt;), making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e., compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time-intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-ET.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling (&lt;&lt;REF:Schirrmacher-2020-8-8&gt;&gt;; &lt;&lt;REF:West-2017-17-363&gt;&gt;; &lt;&lt;REF:Wai-2016-27-105&gt;&gt;). Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure; &lt;&lt;REF:Chan-2020-15-235&gt;&gt;). Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e., elongated) or disconnected (i.e., fragmented) networks that regulate prosurvival or proapoptotic mitochondrial functions (&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;; &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;). Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions (&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;).\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvedness required for efficient mitochondrial respiration, (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Blum-2019-116-4250&gt;&gt;). Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;; &lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;). Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;). It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-ET workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semiautomated workflow (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) to segment cellular membranes present in cryo-tomograms and applied a screened Poisson reconstruction strategy (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics pipeline to measure several parameters of membrane shape including inter- and intramembrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of ER stress, a condition known to promote adaptive remodeling of mitochondrial networks (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;), to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nResults\n\nCorrelative cryo-fluorescence microscopy and electron tomography enable association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP) such that the morphology of each cell's mitochondrial network could be readily assessed (Fig. 1, A-C &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification: Five researchers were given unlabeled fluorescence microscopy images of exemplar network morphologies (elongated and fragmented) as references to assign morphologies to the experimental fluorescence micrographs. We targeted cells with either fragmented or elongated mitochondrial networks for cryo-FIB milling to prepare thin (95-233 nm) lamellae (Fig. 1 D &lt;&lt;FIG:fig1&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Fig. 1 C &lt;&lt;FIG:fig1&gt;&gt;) to the ultrastructure of mitochondrial membranes (Fig. 1, F-H &lt;&lt;FIG:fig1&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). We applied our correlative approach to identify and target specific Tg-treated and vehicle (media with DMSO)-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). Notably, Tg treatment induced a shift toward regularly spaced disc-shaped (lamellar) cristae in the elongated population and toward a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-ET workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions. (A) MEFmtGFP cells were cultured on transmission EM grids (black mesh circle) and treated with vehicle or Tg (500 nM) for 8 h prior to vitrification via plunge freezing in ethane/propane mixture. (B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence microscopy (cryo-FM). (C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies. (D) These cells were targeted for cryo-FIB milling to generate thin sections (lamellae). (E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate 3D cryo-tomograms. (F) Voxel segmentations labeling distinct cellular membranes were generated using a semiautomated approach (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point (&lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; &lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). (G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: IMM, purple; OMM, blue; ER, teal. (H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP\ncells. (A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies from mixed populations of MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of 95-233 nm. MEFmtGFP cell periphery is outlined in dashed white line. (B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue. (C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backward 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\nFigure S1.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"UE1\" is untreated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with elongated mitochondrial networks, n = 10 tomograms.\nFigure S2.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TE1\" is Tg-treated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Tg-treated cells  with elongated mitochondrial networks, n = 13 tomograms.\nFigure S3.\nGallery of cellular membranes used for downstream quantifications from cells with fragmented mitochondrial networks in the absence and presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TF1\" is Tg-treated, fragmented, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with fragmented mitochondrial networks, n = 6 tomograms; and Tg-treated cells with fragmented mitochondrial networks, n = 5 tomograms.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semiautomated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;) that established a robust approach for estimating curvedness in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high-quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;; &lt;&lt;REF:Navarro-2018-5-82&gt;&gt;) and manual inpainting of entire compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;), both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semiautomated workflow to enable analysis at higher throughput (Fig. 1 F &lt;&lt;FIG:fig1&gt;&gt;). We used a segmentation strategy that takes advantage of the TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), which identifies and enhances regions of the tomogram with membrane-like voxel values. This was followed by manual labeling of membranes into mitochondrial IMM and OMM, ER, and multilamellar membrane based on cellular context, as well as manual cleanup of individual membrane segmentation using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 34 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;, Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). The segmentation output was fed into the fully automated surface morphometrics pipeline (Fig. 2 B &lt;&lt;FIG:fig2&gt;&gt;). The voxel segmentation was converted to high-quality membrane surfaces using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; and Video 1 &lt;&lt;FIG:video1&gt;&gt;). Next, these surfaces were converted into triangle graphs, and multiple membrane features were quantified including curvedness (estimated using pycurv15), distances within and between surfaces, and the relative orientations of different surfaces. Finally, the quantifications for each mitochondrion were combined into experiments to allow aggregate statistics and visualizations. Results were computed from 34 completed segmentations in a few hours using a cluster for parallelization (Table S1). This 3D surface morphometrics pipeline is configurable for any segmented membrane and is available at https://github.com/grotjahnlab/surface_morphometrics.\nFigure S4.\nComparison of surface reconstruction algorithms. The same IMM segmentation was subjected to the boundary segmentation surface reconstruction algorithm used in the pycurv software (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;; &lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;; left) as well as our new reconstruction approach based on the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; right) show the improvements in smoothness and completeness enabled by the new surface reconstruction algorithm. While the left algorithm was made part of the pycurv software, it was not used for extensive analysis in the cited manuscript, in favor of algorithms using compartment segmentations.\nVideo 1.\nComparison of voxel segmentation vs. surface reconstruction. An example tomogram is shown with and without a completed voxel segmentation (green). A zoom in reveals voxel stepping artifacts that impact measurement precision, as well as small holes that result from automated segmentation. Switching to the reconstructed surface shows the smooth triangle mesh and the hole-filling properties of the surface reconstruction algorithm. Surface colors: IMM in pink, OMM in purple, ER in blue.\n\nBulk analysis and assessment of statistical significance based on surface quantifications\nOur pipeline outputs multiple membrane feature quantifications (i.e., curvature, distance, and orientation) for each triangle, totaling ~500,000 triangles per tomogram. We set out to determine an appropriate method for bulk quantitative and statistical analyses that would enable detection of subtle albeit potentially significant differences in membrane ultrastructure across different mitochondrial morphologies and treatment groups. We plotted the overall distributions of each membrane feature quantification using histograms of triangle values that are weighted by the area of each triangle, both for individual mitochondrion (Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;) and for entire experiments. These histograms identify bulk differences in the membrane ultrastructure of mitochondria under different experimental conditions. For several membrane parameter quantifications, the observed histogram peak shifted visibly between conditions, suggesting potential differences between treatment and morphology groups. In order to test the statistical significance of these shifts, the weighted histogram peak for each membrane feature quantification for each individual mitochondrion was extracted and these peaks were used for statistical comparisons. In most cases, there was a statistically significant difference in the mean of the distributions, which was assessed via a non-parametric Mann-Whitney U test (Table S2). Differences in the variability of metrics between mitochondria within a given experimental condition were assessed with a two-sample Kolmogorov-Smirnoff test.\nFigure S5.\nGallery of IMM-OMM distance histograms for individual mitochondria. The distribution of distances between the IMM and OMM was plotted for each mitochondrion individually for all Tg-treated, elongated (TE), Tg-treated, fragmented (TF), vehicle (i.e., untreated), elongated (UE), and vehicle (i.e., untreated), fragmented (UF). The peak position is plotted for each and is used for the violin plot in Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;. The histograms are titled based on the measurement (i.e., IMM-IMM Dist) and correspond to the treatment conditions, morphologies, and individual mitochondria labels shown in Fig. S1 &lt;&lt;FIG:figS1&gt;&gt; (e.g., \"TE2a\" corresponds to a Tg-treated [T], elongated [E], tomogram \"2,\" mitochondrion labeled \"a\").\nIn some cases, there was an apparent change to the shape of distribution seen in quantification histograms rather than simply peak position. We primarily assessed the statistical significance of these changes by Mann-Whitney U tests and Kolmogorov-Smirnoff tests on the per-mitochondria standard deviation rather than histogram peak (Table S2). This metric captured changes to the variance of metrics. In order to quantify changes that combine variance and peak shifts, we used the two-sample Kolmogorov-Smirnoff test to test the statistical significance of changes to the distributions of mesh triangles in each condition. However, a major challenge of utilizing this statistical test for analysis of surface quantifications is assessing the true number of independent measurements being made. Triangles are highly correlated to their neighboring meshes, a correlation made even stronger by the vector voting approach employed by pycurv in order to better approximate a smooth membrane. When each triangle within a surface mesh is treated as an independent measurement (in some cases yielding sample sizes in the millions), even visibly identical distributions yield statistically significant P values. This is even further complicated because the number of triangles generated in a given surface mesh is arbitrary and not based on any physical constraint. In contrast, if each mitochondrion is treated as an independent measurement, the variance of features within the mitochondria is lost and there is essentially never statistical significance. We established a heuristic compromise based on the feature radius of 12 nm used in vector voting to optimally measure curvatures. The number of independent samples was assumed to be one per 452 nm2 of membrane area, corresponding to the 12-nm feature size. In practice, this area corresponds to the area of a single crista tip or junction and yields results with P values that might be effectively used comparatively between different measurements. Altering the feature size to 9 or 15 nm changes the number of independent samples and absolute P values; however the relative P value differences across groups do not change, demonstrating that absolute value for assessing statistical significance is limited by the artificial selection of feature radius, but the choice of feature size does not affect the overall result of the comparative analyses across different membrane feature parameters (Table S2).\nA challenge with using histogram peaks as a metric is determining the confidence interval associated with the metric. In order to assess the precision of the histogram peak measurements, we resampled each distribution using bootstrap sampling 1,000 times, and the histogram peak was extracted on each resampling. The 5th and 95th percentiles of these histogram peaks were used to represent the confidence interval of the histogram peak (Table S3).\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria is the juxtaposed inner and outer mitochondrial membranes. Cryo-ET is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh for each mitochondrion in our dataset (Fig. 3 A &lt;&lt;FIG:fig3&gt;&gt; and Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented mitochondria, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Fig. 3 C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in intermembrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak position and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each mitochondrion as independent observations and identified statistically significant increases in spacing in cells with fragmented vs. elongated mitochondria by 2.4 nm in the vehicle-treated condition and by 3.6 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance between OMM and IMM of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in cells with fragmented mitochondria after Tg treatment was not significant (Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significant differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nDistance between IMMs and OMMs is dependent on mitochondrial network morphology and presence or absence of ER stress. (A) Surface membrane reconstruction defining OMM (purple) and IMM (orange) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions. (C) Combined histogram of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Quantification of peak histogram values from each mitochondrion within the indicated treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;). We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intracrista and intrajunction distances, respectively), as well as the distance between adjacent crista bodies and junctions (intercrista and interjunction distances, respectively; Fig. 4 B &lt;&lt;FIG:fig4&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each mitochondria (Fig. 4, C and D &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intracrista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 5.3 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 1.8 nm upon Tg treatment. However, we did not observe significant changes in intercristae distance, intrajunction distance, or interjunction distance, with the exception of a 3.7 nm decrease in intrajunction spacing in the Tg-treated condition of elongated mitochondria relative to vehicle elongated mitochondria, suggesting that ER stress promotes rigidification of the junctions in elongated mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations. (A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with Tg (500 nM, 8 h) showing subdivision of IMM compartments (IBM, tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh. (B) Enlarged inset of boxed region in elongated example from A showing surface membrane reconstruction colored by inner membrane compartments defining intra- and intercrista and junction measurements. (C) Combined histogram of intracrista membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ***P &lt; 0.005. (D) Combined histogram of intrajunction membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05.\nFigure S6.\nAdditional quantifications of membrane ultrastructure. Additional quantifications output by the surface morphometrics pipeline, which showed non-significant changes or no apparent change. In all cases, quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. (A) Histograms and violin plot showing changes to spacing between closest neighbor segments of IMM. (B) Histograms and violin plot showing changes to spacing to the second nearest segments of IMM. (C) Histograms and violin plot showing changes to spacing between neighboring cristae. (D) Histograms and violin plot showing changes to spacing between closest neighbor segments of IBM. (E) Histograms and violin plot showing changes to spacing to the second nearest segments of IBM. (F) Histograms and violin plot showing changes to spacing between neighboring junctions. (G) Histograms and violin plot showing changes to curvedness of the IBM. (H) Histograms and violin plot showing changes to the angle between IMM and OMM. (I) Histograms and violin plot showing changes to the angle between IBM and OMM. (J) Histograms and violin plot showing changes to the angle between crista junctions and OMM. (K) Histograms and violin plot showing changes to the angle between IMM and the growth plane. (L) Histograms and violin plot showing changes to the angle between IBM and the growth plane. (M) Histograms and violin plot showing changes to the angle between crista junctions and the growth plane. *P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.005; ****P &lt; 0.001.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvedness (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;; &lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated cells with elongated mitochondria, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). A significant change was observed between the vehicle-elongated and vehicle-fragmented mitochondria, which established a basal distinction in curvedness among mitochondrial populations (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). As visual inspection revealed local changes in curvedness in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Fig. 5, C and D &lt;&lt;FIG:fig5&gt;&gt;).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principal components of curvedness and is used because the surface normal vectors do not have canonical sign. (B) Combined histogram of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01. (C) Combined histogram of cristae curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; **P &lt; 0.01. (D) Combined histogram of cristae junction curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\nLarger changes in curvature were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress. Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated cells with elongated mitochondria compared with both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria. This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nOMM curvedness is dependent on mitochondrial network morphology\nIn addition to the IMM, we calculated curvedness of the OMM for each mitochondrion. For both vehicle and Tg-treated cells, curvedness of the OMM is decreased in fragmented mitochondria relative to elongated networks (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;). This metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length.\nFigure S7.\nOMM curvature analysis. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by membrane curvedness and the IMM surface is represented as a transparent gray mesh. (B) Combined histogram of total OMM curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; ****P &lt; 0.001.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae was regulated and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (the plane of the EM grid substrate on which the cells are grown; Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg-treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Fig. 6 B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Fig. 6 C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position (Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Due to this, we quantified the standard deviation of these values for each mitochondrion in order to assess statistical significance of observed changes (Fig. 6, D and E &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells. Additionally, we observed a significant (based on the Mann-Whitney U test of standard deviations) increase in the variability of the angle of the cristae relative to the OMM for fragmented mitochondria in comparison with elongated mitochondria in Tg-treated cells (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt; and Table S2). We also identified a significant decrease in variability of the angle of the crista body relative to the growth plane in Tg-treated elongated relative to fragmented mitochondria (Fig. 6 E &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM or the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks. (A) Surface membrane reconstruction colored by inner membrane compartments (IBM, tan; junction, cyan; crista, purple) defining angular measurements between the surface and the nearest OMM or to the growth plane of the cell. (B) Two representative membrane surface reconstructions of lamellar Tg-treated elongated mitochondria, colored by the angle of IMM relative to OMM. (C) Two representative membrane surface reconstructions of a less rigidly oriented Tg-treated elongated mitochondria, colored by an angle of IMM relative to the growth plane of the cell. (D) Combined histogram of the angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; **P &lt; 0.01. (E) Combined histogram of the angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01.\n\nApplication of surface morphometrics and analysis to additional organellar membranes\nWe further demonstrated the generality of this approach by generating mesh surfaces for membranes of other organelles present in our data: the ER (Figs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;) and observed multilamellar membrane structures (Fig. S8 &lt;&lt;FIG:figS8&gt;&gt;). We applied our morphometrics pipeline to quantify parameters of membrane architecture of these organelles. The observed multilamellar structures are comprised of multiple membranes (&lt;&lt;REF:Bieber-2022-119-?&gt;&gt;), including the primary membrane and the secondary membrane, as well as interior membranes (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). We then visualized the spatial distributions of the curvatures, distances, and orientations of these distinct membranes on the generated surface reconstructions (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). Additionally, we calculated the curvedness of ER membranes in cells with differing mitochondrial populations and observed no significant changes in any of the conditions (Fig. S10 &lt;&lt;FIG:figS10&gt;&gt;).\nFigure S8.\nGallery of multilamellar membrane structures used for quantifications. Virtual slices of tomograms (left panels) and corresponding reconstructed surface mesh models (right panels) of multilamellar membrane structures. Primary membrane shown in red. Secondary membrane shown in yellow. Interior membranes shown in sage green. Meshes have been tilted 20degrees relative to tomography slice for visualization. Scale bar = 100 nm. n = 3 tomograms.\nFigure S9.\nMultilamellar membrane structure morphometric analysis. (A) Exemplar multilamellar membrane structure surfaces with membranes labeled. Primary membrane in red. Secondary membrane in yellow. Interior membranes in sage green. (B) Secondary and interior membrane surfaces colored by membrane curvedness; the primary membrane is represented as a transparent gray. (C) Primary surface is colored by membrane curvedness; the secondary and interior membranes are represented as a transparent gray. (D) Primary surface is colored by primary-to-secondary membrane distance; the secondary and interior membrane surfaces are shown in transparent gray. (E) Secondary surface colored by secondary to nearest interior (secondary-interior) membrane distance; the primary and interior membrane surfaces are shown in transparent gray. (F) Secondary and interior membranes surfaces are colored by angle relative to the primary membrane. Primary membrane surface is shown in transparent gray. (G) Secondary and interior membrane surfaces are colored by an angle relative to the growth plane of the cell. Primary membrane surface shown in transparent gray. (H) Primary membrane surface is colored by relative angle to the secondary membrane. Secondary and interior membrane surfaces are shown in transparent gray. (I) Primary membrane surface colored by relative angle to the growth plane of the cell. Secondary and interior membranes are shown in transparent gray.\nFigure S10.\nER membrane curvature analysis. (A) Representative membrane surface reconstructions of the ER in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h) and associated with defined mitochondrial populations (elongated and fragmented). The ER surface is colored by membrane curvedness, and the OMM surface is represented as a transparent gray mesh. (B) Combined histograms of total ER curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each ER membrane within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 9, Tg elongated n = 8, vehicle fragmented n = 3, and Tg fragmented n = 4 ER are shown.\n\nMonitoring ER-mitochondrial contacts by surface morphometrics\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission (&lt;&lt;REF:Friedman-2011-334-358&gt;&gt;) to autophagy (&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;). To investigate whether the remodeling we observed in response to Tg-induced ER stress was correlated with alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs. fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present and visualized the contact sites on the surfaces of the OMM (Fig. S11, A and B &lt;&lt;FIG:figS11&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Fig. S11 C &lt;&lt;FIG:figS11&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time point were not associated with the altered mitochondrial ultrastructure (Fig. S11 D &lt;&lt;FIG:figS11&gt;&gt;).\nFigure S11.\nSurface analysis of mitochondria-ER contact sites. (A) Surface membrane reconstruction defining OMM (blue-yellow gradient) and ER (transparent mesh) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM) colored by OMM to ER (OMM-ER) membrane distance. (C) Combined histogram of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 18, Tg elongated n = 11, vehicle fragmented n = 7, Tg fragmented n = 13 mitochondria are shown.\n\nDiscussion\nWe describe a correlative cellular cryo-ET workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influence mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis. The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-ET to reveal organellar membrane ultrastructures (Fig. 1, A-D &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semiautomated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high precision. We determined that the screened Poisson reconstruction approach (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) provides a more accurate estimation of the implicit membrane geometry of complex membranes and can be used to model and quantify complex membranes such as the IMM.\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions and quantified mitochondrial membrane parameters such as inter- and intramitochondrial membrane distances, curvedness, and orientation. We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps. This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions.\nWe first used our workflow to quantify the intermembrane distance between the inner and outer membranes of mitochondria (Fig. 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;; &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;), and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes as well as to quantify changes in the overall distributions. We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). The difference in distance, from 13.5 to 12.7 nm between the centers of the membranes, is small, but correcting for the ~7 nm thickness of mitochondrial membranes (&lt;&lt;REF:Perkins-1997-119-260&gt;&gt;) yields a change in intermembrane contact distance from 6.5 to 5.7 nm or a 12% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as phosphatidylserine decarboxylase, an IMM protein previously suggested to act on the OMM (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbations may impact intermembrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the translocase of the OMM complex (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;), which associate with the MICOS complex and may impact MICOS complex organization (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;).\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;). We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figs. 4 &lt;&lt;FIG:fig4&gt;&gt; and S6 &lt;&lt;FIG:figS6&gt;&gt;), quantified the curvedness of the membranes (Figs. 5 &lt;&lt;FIG:fig5&gt;&gt;, S6 &lt;&lt;FIG:figS6&gt;&gt;, S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, and S10 &lt;&lt;FIG:figS10&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; and 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress (&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt;). In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase--a mechanism similar to that responsible for ER-stress-induced mitochondrial elongation (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvedness. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvedness, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;), ATP synthase (&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;), and OPA1 (&lt;&lt;REF:Hu-2020-11-940&gt;&gt;), and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function.\nWe demonstrated the general utility of our workflow beyond mitochondrial membranes and analyzed the curvedness, intermembrane distances, and orientation of cellular membranes of the ER and observed multilamellar membrane structures (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, S10 &lt;&lt;FIG:figS10&gt;&gt;, and S11 &lt;&lt;FIG:figS11&gt;&gt;). Although our results did not reveal any statistically significant differences in the architecture of these cellular membranes upon Tg-induced ER stress, we anticipate that our workflow will facilitate future studies on the critical importance of membrane remodeling of these organelles induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;), which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) used in this study traces lower curved membranes more easily than high curved membranes, leading to limited segmentations of tight, high curved crista junctions and crista tips. We have mitigated these biases by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter (&lt;&lt;REF:Koning-2018-217-82&gt;&gt;) or phase plate (&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;), both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations (&lt;&lt;REF:Zhou-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Chen-2017-14-983&gt;&gt;; &lt;&lt;REF:Buchholz-2018-?-?&gt;&gt;; &lt;&lt;REF:Liu-2021-?-?&gt;&gt;\nPreprint). The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics (&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt;) and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template-matching approaches (&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;). These approaches have in the past depended on voxel segmentations, but moving to high-quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentation, which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well (&lt;&lt;REF:Basanta-2020-4-100041&gt;&gt;). Membrane meshes have been used very successfully to guide subtomogram averaging in the past (&lt;&lt;REF:Navarro-2018-5-82&gt;&gt;; &lt;&lt;REF:Burt-2021-?-?&gt;&gt;\nPreprint), but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu (&lt;&lt;REF:Erdmann-2021-12-5364&gt;&gt;; &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:O’Reilly-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Guo-2018-172-696&gt;&gt;; &lt;&lt;REF:Jordan-2018-20-1250&gt;&gt;; &lt;&lt;REF:Albert-2020-117-1069&gt;&gt;; &lt;&lt;REF:Lin-2018-360-360&gt;&gt;). Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nMaterials and methods\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM grids\nMouse embryonic fibroblasts with mitochondria-labeled GFP (MEFmtGFP; &lt;&lt;REF:Wang-2012-51-9302&gt;&gt;) were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 mug/ml; Corning) and UV-sterilized R (1/4) Carbon 200-mesh gold EM grids (Quantifoil Micro Tools). After 15-18 h of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group or fresh media supplemented with Thapsigargin (500 nM; Thermo Fisher Scientific) in the Tg treatment group. After 8 h of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific). The Vitrobot was set to 37degreesC and 100% relative humidity, and blotting was performed manually from the back side of grids using Whatman #1 filter paper strips through the Vitrobot humidity/temperature chamber side port. The Vitrobot settings used to disable automated blotting apparatus were as follows: Blot total: 0, 2; Blot force: 0, 3; Blot time: 0 s.\n\nCryo-fluorescence microscopy and mitochondria network morphology scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) and collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em: 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13; Thermo Fisher Scientific) for fluorescence-guided milling.\n\nFluorescence -guided milling\nCryo-FIB milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system for 3-4 s using an automation script (&lt;&lt;REF:Barad-2023-?-?&gt;&gt;), followed by a third and final layer of platinum sputter. Targeted cells were milled manually (&lt;&lt;REF:Schaffer-2015-5-5&gt;&gt;) or using an automated cryo preparation workflow (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;), and both methods used xT software with MAPS (Thermo Fisher Scientific). Minimal SEM imaging for monitoring was done at 2 keV to ensure thin lamella generation while avoiding radiation damage. A total of six treated grids and eight untreated grids were milled for further tomography analysis.\n\nTilt series data collection\nEM grids containing lamellae were transferred into a 300 keV Titan Krios microscope (Thermo Fisher Scientific) equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1 e-/A2) at high magnification to localize cellular regions containing mitochondria, which were identified by their distinctive IMMs and OMMs. Data were collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) with 2degrees steps between -60degrees and +60degrees. Data were collected with pixel sizes of 3.11 , 3.35 , or 3.55 A, and a nominal defocus range between -5 and -10 microm. A subset of data was collected with dose fractionation, with 10 0.1 e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2 and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose-fractionated tilt series micrograph movies underwent contrast transfer function estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series were then aligned using patch tracking in etomo (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;), with four times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into four times binned tomograms. Tomogram thicknesses ranged from 95 to 233 nm.\n\nSemiautomated segmentation\nAutomated detection and enhancement of mitochondrial membranes were performed using the TomoSegMemTV software package (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;). Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows: scale_space -s 3; dtvoting -s 12; surfaceness -m 0.17 -s 0.8 -p 0.8; dtvoting -w -s 10; surfaceness -S -s 0.75 -p 0.75 -l 20; thresholding -l 0.01-2 30; global analysis -3 100.\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher Scientific) for manual curation. Mitochondrial inner and outer membranes and ER were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nSoftware workflow\nThe surface morphometrics pipeline is a python-scripted workflow with requirements that can be installed as a conda environment contained in an \"environment.yml\" file. The workflow is fully scripted and configurable with a \"config.yml\" file and is run in three fully pipelined steps, with statistical analysis and visualization as an optional fourth step. First, a segmentation MRC file is converted automatically to a series of surface meshes formatted in the VTP file format. Second, for each mesh, the surface is converted to a graph (tg format) and curvature is estimated using pycurv. Third, orientations and distances between and within surfaces are calculated using the resulting graphs, and a CSV with quantifications as well as a final VTP surface file is output with all quantifications built in. Fourth, the outputs from multiple tomograms are combined for visualization and statistical analysis. Times and computational requirements are shown in Table S1.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab (&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;). First, the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;), with a reconstruction depth of 9, an interpolation weight of 0.7, and a minimum number of samples of 1.5. These settings were chosen to maximize correspondence to the data rather than smoothness. The resulting surface extended beyond the segmented region, so triangles more than 1.5 nm away from the point cloud were deleted. Interpolation weight (point_weight) and the mask distance (extrapolation_distance) are both configurable in the surface morphometrics pipeline if more aggressive smoothing and hole filling are desirable. The resulting mesh was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the \"segmentation_to_meshes.py\" script.\n\nRefinement of surface orientation and estimation of surface curvedness using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). This vector voting process also resulted in robust estimation of curvature metrics. As the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (Eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvedness metrics, which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the \"run_pycurv.py\" script.curvedness=kappa12+kappa222\n(1)\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated using a k-dimensional tree. Both the distance and identifier of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the \"measure_distances_orientations.py\" script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those &gt;40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nAs all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously (&lt;&lt;REF:Collado-2019-51-476&gt;&gt;). To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into shorter and longer distances. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector-based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector-based neighbor. Those triangles were filtered out before any further quantifications. These measurements are made in the \"measure_distances_orientations.py\" script.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (Eq. 2):verticality=arccos(|normalz|)\n(2)\nComparisons of orientations between membranes were made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector to its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (Eq. 3):relative_angle=arccos|self.neighbor|\n(3)\nBoth orientation calculations are determined in the \"measure_distances_orientations.py\" script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization pipeline \"vtp\" (&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt;) format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview (&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;). Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area (&lt;&lt;REF:Hunter-2007-9-90&gt;&gt;). Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests (&lt;&lt;REF:Mann-1947-18-50&gt;&gt;) on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram rather than the histogram peak. Statistical testing based on distributions, rather than peaks, is complicated by unbiased assessment of the number of independent samples in each tomogram. Aggregation, statistics, and visualization are accomplished by the \"morphometrics_stats.py\" classes and functions, and the mitochondria-specific statistics and visualizations are generated by the \"mitochondria_statistics.py\" script. The complete output of the per mitochondrion analyses for every measurement is available at https://zenodo.org/record/7510830.\n\nOnline supplemental material\nFigs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt; are a gallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress, cells with elongated mitochondrial networks in the presence of ER stress, and cells with fragmented mitochondrial networks in the presence and absence of ER stress, respectively. Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; shows a comparison of different surface reconstruction algorithms. Fig. S5 &lt;&lt;FIG:figS5&gt;&gt; is a gallery of histograms showing the distribution of IMM-OMM distances for each individual mitochondrion. Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; presents additional quantifications that do not show statistically significant differences. Fig. S7 &lt;&lt;FIG:figS7&gt;&gt; shows analysis of changes to OMM curvature. Fig. S8 &lt;&lt;FIG:figS8&gt;&gt; shows a gallery of multilamellar membrane bodies observed in the data and their segmentations. Fig. S9 &lt;&lt;FIG:figS9&gt;&gt; shows a gallery of surface morphometrics pipeline quantifications mapped onto the multilamellar membrane bodies. Fig. S10 &lt;&lt;FIG:figS10&gt;&gt; shows analysis of changes to ER curvature. Fig. S11 &lt;&lt;FIG:figS11&gt;&gt; shows quantification of mitochondria-ER contact sites. Table S1 shows typical computational requirements and time required for each step of the surface morphometrics pipeline. Table S2 shows additional statistical assessments for each quantification and each condition. Table S3 shows the confidence intervals derived from bootstrap analysis of each quantification. Video 1 &lt;&lt;FIG:video1&gt;&gt; shows a comparison of voxel segmentation and implicit surface reconstruction.\n\nSupplementary Material\nReview History\nClick here for additional data file.\nTable S1\nshows the approximate time for each step of the surface morphometrics workflow.\nClick here for additional data file.\nTable S2\nshows the statistical analyses of peaks and standard deviations.\nClick here for additional data file.\nTable S3\nshows the confidence intervals of quantifications with bootstrap analysis.\nClick here for additional data file.\n\nData availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Installation is simplified with a conda environment and a pip requirements file. Scripts are licensed under the GNU Public License version 3 (GPLv3). All tilt series, reconstructed tomograms, voxel segmentations, and reconstructed mesh surfaces used for quantifications were deposited in the Electron Microscopy Public Image Archive under accession code EMPIAR-11370.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, 3D views of protein and organellar structure in unfixed and fully hydrated conditions (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;). By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;; &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;), it has become routine to collect large datasets of thin, cellular 3D reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells (&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;; &lt;&lt;REF:Himes-2018-15-955&gt;&gt;; &lt;&lt;REF:Chen-2019-16-1161&gt;&gt;), it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments (&lt;&lt;REF:Jiang-2017-7-45474&gt;&gt;; &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;). As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the 2D virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intramembrane distances through manual designation of membrane boundaries by locating distinct points along the membrane (&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;; &lt;&lt;REF:Ader-2019-8-?&gt;&gt;; &lt;&lt;REF:Tran-2021-374-52&gt;&gt;; &lt;&lt;REF:Mageswaran-2021-7-7&gt;&gt;). Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Toward a more accurate 3D representation, manual or automated (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvedness and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvedness. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvedness as well as the spatial relationship between individual triangles (i.e., membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e., boundary segmentations) were not reliable due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm (&lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;). In cryo-ET, even perfect segmentations will never show complete and closed membranes due to the substantial missing wedge (&lt;&lt;REF:Lučič-2013-202-407&gt;&gt;), making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e., compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time-intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-ET.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling (&lt;&lt;REF:Schirrmacher-2020-8-8&gt;&gt;; &lt;&lt;REF:West-2017-17-363&gt;&gt;; &lt;&lt;REF:Wai-2016-27-105&gt;&gt;). Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure; &lt;&lt;REF:Chan-2020-15-235&gt;&gt;). Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e., elongated) or disconnected (i.e., fragmented) networks that regulate prosurvival or proapoptotic mitochondrial functions (&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;; &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;). Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions (&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;).\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvedness required for efficient mitochondrial respiration, (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Blum-2019-116-4250&gt;&gt;). Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;; &lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;). Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C (&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;; &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;). It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-ET workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semiautomated workflow (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) to segment cellular membranes present in cryo-tomograms and applied a screened Poisson reconstruction strategy (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics pipeline to measure several parameters of membrane shape including inter- and intramembrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of ER stress, a condition known to promote adaptive remodeling of mitochondrial networks (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;), to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nResults\n\nCorrelative cryo-fluorescence microscopy and electron tomography enable association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP) such that the morphology of each cell's mitochondrial network could be readily assessed (Fig. 1, A-C &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification: Five researchers were given unlabeled fluorescence microscopy images of exemplar network morphologies (elongated and fragmented) as references to assign morphologies to the experimental fluorescence micrographs. We targeted cells with either fragmented or elongated mitochondrial networks for cryo-FIB milling to prepare thin (95-233 nm) lamellae (Fig. 1 D &lt;&lt;FIG:fig1&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Fig. 1 C &lt;&lt;FIG:fig1&gt;&gt;) to the ultrastructure of mitochondrial membranes (Fig. 1, F-H &lt;&lt;FIG:fig1&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). We applied our correlative approach to identify and target specific Tg-treated and vehicle (media with DMSO)-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). Notably, Tg treatment induced a shift toward regularly spaced disc-shaped (lamellar) cristae in the elongated population and toward a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-ET workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions. (A) MEFmtGFP cells were cultured on transmission EM grids (black mesh circle) and treated with vehicle or Tg (500 nM) for 8 h prior to vitrification via plunge freezing in ethane/propane mixture. (B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence microscopy (cryo-FM). (C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies. (D) These cells were targeted for cryo-FIB milling to generate thin sections (lamellae). (E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate 3D cryo-tomograms. (F) Voxel segmentations labeling distinct cellular membranes were generated using a semiautomated approach (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point (&lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; &lt;&lt;REF:Cignoni-2008-?-?&gt;&gt;). (G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: IMM, purple; OMM, blue; ER, teal. (H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP\ncells. (A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies from mixed populations of MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of 95-233 nm. MEFmtGFP cell periphery is outlined in dashed white line. (B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue. (C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backward 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\nFigure S1.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"UE1\" is untreated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with elongated mitochondrial networks, n = 10 tomograms.\nFigure S2.\nGallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TE1\" is Tg-treated, elongated, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Tg-treated cells  with elongated mitochondrial networks, n = 13 tomograms.\nFigure S3.\nGallery of cellular membranes used for downstream quantifications from cells with fragmented mitochondrial networks in the absence and presence of ER stress. Virtual slices of tomograms (upper panels) and corresponding reconstructed surface mesh models (lower panels) of mitochondrial membranes (IMM, orange; OMM, purple) and ER membranes (blue). The surfaces (lower panels) are labeled according to treatment conditions (U, untreated/vehicle-treated), mitochondrial morphology (E, elongated), and tomogram number. For example, \"TF1\" is Tg-treated, fragmented, tomogram 1. Individual mitochondria in the tomograms are labeled by letters (e.g., a, b, c). These labels correspond to the title labels of the histograms of individual mitochondria shown in Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;. Meshes have been tilted backward by 20degrees for visualization. Scale bar = 100 nm. Vehicle-treated cells with fragmented mitochondrial networks, n = 6 tomograms; and Tg-treated cells with fragmented mitochondrial networks, n = 5 tomograms.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semiautomated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;) that established a robust approach for estimating curvedness in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high-quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;; &lt;&lt;REF:Navarro-2018-5-82&gt;&gt;) and manual inpainting of entire compartments (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;), both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semiautomated workflow to enable analysis at higher throughput (Fig. 1 F &lt;&lt;FIG:fig1&gt;&gt;). We used a segmentation strategy that takes advantage of the TomoSegMemTV program (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;), which identifies and enhances regions of the tomogram with membrane-like voxel values. This was followed by manual labeling of membranes into mitochondrial IMM and OMM, ER, and multilamellar membrane based on cellular context, as well as manual cleanup of individual membrane segmentation using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 34 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;, Fig. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;). The segmentation output was fed into the fully automated surface morphometrics pipeline (Fig. 2 B &lt;&lt;FIG:fig2&gt;&gt;). The voxel segmentation was converted to high-quality membrane surfaces using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; and Video 1 &lt;&lt;FIG:video1&gt;&gt;). Next, these surfaces were converted into triangle graphs, and multiple membrane features were quantified including curvedness (estimated using pycurv15), distances within and between surfaces, and the relative orientations of different surfaces. Finally, the quantifications for each mitochondrion were combined into experiments to allow aggregate statistics and visualizations. Results were computed from 34 completed segmentations in a few hours using a cluster for parallelization (Table S1). This 3D surface morphometrics pipeline is configurable for any segmented membrane and is available at https://github.com/grotjahnlab/surface_morphometrics.\nFigure S4.\nComparison of surface reconstruction algorithms. The same IMM segmentation was subjected to the boundary segmentation surface reconstruction algorithm used in the pycurv software (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;; &lt;&lt;REF:Hoppe-1992-26-71&gt;&gt;; left) as well as our new reconstruction approach based on the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;; right) show the improvements in smoothness and completeness enabled by the new surface reconstruction algorithm. While the left algorithm was made part of the pycurv software, it was not used for extensive analysis in the cited manuscript, in favor of algorithms using compartment segmentations.\nVideo 1.\nComparison of voxel segmentation vs. surface reconstruction. An example tomogram is shown with and without a completed voxel segmentation (green). A zoom in reveals voxel stepping artifacts that impact measurement precision, as well as small holes that result from automated segmentation. Switching to the reconstructed surface shows the smooth triangle mesh and the hole-filling properties of the surface reconstruction algorithm. Surface colors: IMM in pink, OMM in purple, ER in blue.\n\nBulk analysis and assessment of statistical significance based on surface quantifications\nOur pipeline outputs multiple membrane feature quantifications (i.e., curvature, distance, and orientation) for each triangle, totaling ~500,000 triangles per tomogram. We set out to determine an appropriate method for bulk quantitative and statistical analyses that would enable detection of subtle albeit potentially significant differences in membrane ultrastructure across different mitochondrial morphologies and treatment groups. We plotted the overall distributions of each membrane feature quantification using histograms of triangle values that are weighted by the area of each triangle, both for individual mitochondrion (Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;) and for entire experiments. These histograms identify bulk differences in the membrane ultrastructure of mitochondria under different experimental conditions. For several membrane parameter quantifications, the observed histogram peak shifted visibly between conditions, suggesting potential differences between treatment and morphology groups. In order to test the statistical significance of these shifts, the weighted histogram peak for each membrane feature quantification for each individual mitochondrion was extracted and these peaks were used for statistical comparisons. In most cases, there was a statistically significant difference in the mean of the distributions, which was assessed via a non-parametric Mann-Whitney U test (Table S2). Differences in the variability of metrics between mitochondria within a given experimental condition were assessed with a two-sample Kolmogorov-Smirnoff test.\nFigure S5.\nGallery of IMM-OMM distance histograms for individual mitochondria. The distribution of distances between the IMM and OMM was plotted for each mitochondrion individually for all Tg-treated, elongated (TE), Tg-treated, fragmented (TF), vehicle (i.e., untreated), elongated (UE), and vehicle (i.e., untreated), fragmented (UF). The peak position is plotted for each and is used for the violin plot in Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;. The histograms are titled based on the measurement (i.e., IMM-IMM Dist) and correspond to the treatment conditions, morphologies, and individual mitochondria labels shown in Fig. S1 &lt;&lt;FIG:figS1&gt;&gt; (e.g., \"TE2a\" corresponds to a Tg-treated [T], elongated [E], tomogram \"2,\" mitochondrion labeled \"a\").\nIn some cases, there was an apparent change to the shape of distribution seen in quantification histograms rather than simply peak position. We primarily assessed the statistical significance of these changes by Mann-Whitney U tests and Kolmogorov-Smirnoff tests on the per-mitochondria standard deviation rather than histogram peak (Table S2). This metric captured changes to the variance of metrics. In order to quantify changes that combine variance and peak shifts, we used the two-sample Kolmogorov-Smirnoff test to test the statistical significance of changes to the distributions of mesh triangles in each condition. However, a major challenge of utilizing this statistical test for analysis of surface quantifications is assessing the true number of independent measurements being made. Triangles are highly correlated to their neighboring meshes, a correlation made even stronger by the vector voting approach employed by pycurv in order to better approximate a smooth membrane. When each triangle within a surface mesh is treated as an independent measurement (in some cases yielding sample sizes in the millions), even visibly identical distributions yield statistically significant P values. This is even further complicated because the number of triangles generated in a given surface mesh is arbitrary and not based on any physical constraint. In contrast, if each mitochondrion is treated as an independent measurement, the variance of features within the mitochondria is lost and there is essentially never statistical significance. We established a heuristic compromise based on the feature radius of 12 nm used in vector voting to optimally measure curvatures. The number of independent samples was assumed to be one per 452 nm2 of membrane area, corresponding to the 12-nm feature size. In practice, this area corresponds to the area of a single crista tip or junction and yields results with P values that might be effectively used comparatively between different measurements. Altering the feature size to 9 or 15 nm changes the number of independent samples and absolute P values; however the relative P value differences across groups do not change, demonstrating that absolute value for assessing statistical significance is limited by the artificial selection of feature radius, but the choice of feature size does not affect the overall result of the comparative analyses across different membrane feature parameters (Table S2).\nA challenge with using histogram peaks as a metric is determining the confidence interval associated with the metric. In order to assess the precision of the histogram peak measurements, we resampled each distribution using bootstrap sampling 1,000 times, and the histogram peak was extracted on each resampling. The 5th and 95th percentiles of these histogram peaks were used to represent the confidence interval of the histogram peak (Table S3).\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria is the juxtaposed inner and outer mitochondrial membranes. Cryo-ET is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh for each mitochondrion in our dataset (Fig. 3 A &lt;&lt;FIG:fig3&gt;&gt; and Fig. S5 &lt;&lt;FIG:figS5&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Fig. 3 B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented mitochondria, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Fig. 3 C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in intermembrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak position and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each mitochondrion as independent observations and identified statistically significant increases in spacing in cells with fragmented vs. elongated mitochondria by 2.4 nm in the vehicle-treated condition and by 3.6 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance between OMM and IMM of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in cells with fragmented mitochondria after Tg treatment was not significant (Fig. 3 D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significant differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nDistance between IMMs and OMMs is dependent on mitochondrial network morphology and presence or absence of ER stress. (A) Surface membrane reconstruction defining OMM (purple) and IMM (orange) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions. (C) Combined histogram of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Quantification of peak histogram values from each mitochondrion within the indicated treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Hu-2020-11-940&gt;&gt;). We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Fig. 4 A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intracrista and intrajunction distances, respectively), as well as the distance between adjacent crista bodies and junctions (intercrista and interjunction distances, respectively; Fig. 4 B &lt;&lt;FIG:fig4&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each mitochondria (Fig. 4, C and D &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intracrista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 5.3 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 1.8 nm upon Tg treatment. However, we did not observe significant changes in intercristae distance, intrajunction distance, or interjunction distance, with the exception of a 3.7 nm decrease in intrajunction spacing in the Tg-treated condition of elongated mitochondria relative to vehicle elongated mitochondria, suggesting that ER stress promotes rigidification of the junctions in elongated mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations. (A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with Tg (500 nM, 8 h) showing subdivision of IMM compartments (IBM, tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh. (B) Enlarged inset of boxed region in elongated example from A showing surface membrane reconstruction colored by inner membrane compartments defining intra- and intercrista and junction measurements. (C) Combined histogram of intracrista membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ***P &lt; 0.005. (D) Combined histogram of intrajunction membrane distances of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05.\nFigure S6.\nAdditional quantifications of membrane ultrastructure. Additional quantifications output by the surface morphometrics pipeline, which showed non-significant changes or no apparent change. In all cases, quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. (A) Histograms and violin plot showing changes to spacing between closest neighbor segments of IMM. (B) Histograms and violin plot showing changes to spacing to the second nearest segments of IMM. (C) Histograms and violin plot showing changes to spacing between neighboring cristae. (D) Histograms and violin plot showing changes to spacing between closest neighbor segments of IBM. (E) Histograms and violin plot showing changes to spacing to the second nearest segments of IBM. (F) Histograms and violin plot showing changes to spacing between neighboring junctions. (G) Histograms and violin plot showing changes to curvedness of the IBM. (H) Histograms and violin plot showing changes to the angle between IMM and OMM. (I) Histograms and violin plot showing changes to the angle between IBM and OMM. (J) Histograms and violin plot showing changes to the angle between crista junctions and OMM. (K) Histograms and violin plot showing changes to the angle between IMM and the growth plane. (L) Histograms and violin plot showing changes to the angle between IBM and the growth plane. (M) Histograms and violin plot showing changes to the angle between crista junctions and the growth plane. *P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.005; ****P &lt; 0.001.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvedness (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;; &lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated cells with elongated mitochondria, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Fig. 5 A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). A significant change was observed between the vehicle-elongated and vehicle-fragmented mitochondria, which established a basal distinction in curvedness among mitochondrial populations (Fig. 5 B &lt;&lt;FIG:fig5&gt;&gt;). As visual inspection revealed local changes in curvedness in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Fig. 5, C and D &lt;&lt;FIG:fig5&gt;&gt;).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principal components of curvedness and is used because the surface normal vectors do not have canonical sign. (B) Combined histogram of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01. (C) Combined histogram of cristae curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; **P &lt; 0.01. (D) Combined histogram of cristae junction curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. *P &lt; 0.05; ****P &lt; 0.001.\nLarger changes in curvature were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress. Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated cells with elongated mitochondria compared with both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria. This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nOMM curvedness is dependent on mitochondrial network morphology\nIn addition to the IMM, we calculated curvedness of the OMM for each mitochondrion. For both vehicle and Tg-treated cells, curvedness of the OMM is decreased in fragmented mitochondria relative to elongated networks (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;). This metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length.\nFigure S7.\nOMM curvature analysis. (A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h). The OMM surface is colored by membrane curvedness and the IMM surface is represented as a transparent gray mesh. (B) Combined histogram of total OMM curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each mitochondrion within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; ****P &lt; 0.001.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae was regulated and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (the plane of the EM grid substrate on which the cells are grown; Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg-treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Fig. 6 B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Fig. 6 C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position (Fig. 6 A &lt;&lt;FIG:fig6&gt;&gt; and Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;). Due to this, we quantified the standard deviation of these values for each mitochondrion in order to assess statistical significance of observed changes (Fig. 6, D and E &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells. Additionally, we observed a significant (based on the Mann-Whitney U test of standard deviations) increase in the variability of the angle of the cristae relative to the OMM for fragmented mitochondria in comparison with elongated mitochondria in Tg-treated cells (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt; and Table S2). We also identified a significant decrease in variability of the angle of the crista body relative to the growth plane in Tg-treated elongated relative to fragmented mitochondria (Fig. 6 E &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM or the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks. (A) Surface membrane reconstruction colored by inner membrane compartments (IBM, tan; junction, cyan; crista, purple) defining angular measurements between the surface and the nearest OMM or to the growth plane of the cell. (B) Two representative membrane surface reconstructions of lamellar Tg-treated elongated mitochondria, colored by the angle of IMM relative to OMM. (C) Two representative membrane surface reconstructions of a less rigidly oriented Tg-treated elongated mitochondria, colored by an angle of IMM relative to the growth plane of the cell. (D) Combined histogram of the angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. *P &lt; 0.05; **P &lt; 0.01. (E) Combined histogram of the angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 20, Tg elongated n = 18, vehicle fragmented n = 11, and Tg fragmented n = 15 mitochondria are shown. P values from Mann-Whitney U test are indicated. **P &lt; 0.01.\n\nApplication of surface morphometrics and analysis to additional organellar membranes\nWe further demonstrated the generality of this approach by generating mesh surfaces for membranes of other organelles present in our data: the ER (Figs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt;) and observed multilamellar membrane structures (Fig. S8 &lt;&lt;FIG:figS8&gt;&gt;). We applied our morphometrics pipeline to quantify parameters of membrane architecture of these organelles. The observed multilamellar structures are comprised of multiple membranes (&lt;&lt;REF:Bieber-2022-119-?&gt;&gt;), including the primary membrane and the secondary membrane, as well as interior membranes (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). We then visualized the spatial distributions of the curvatures, distances, and orientations of these distinct membranes on the generated surface reconstructions (Fig. S9 &lt;&lt;FIG:figS9&gt;&gt;). Additionally, we calculated the curvedness of ER membranes in cells with differing mitochondrial populations and observed no significant changes in any of the conditions (Fig. S10 &lt;&lt;FIG:figS10&gt;&gt;).\nFigure S8.\nGallery of multilamellar membrane structures used for quantifications. Virtual slices of tomograms (left panels) and corresponding reconstructed surface mesh models (right panels) of multilamellar membrane structures. Primary membrane shown in red. Secondary membrane shown in yellow. Interior membranes shown in sage green. Meshes have been tilted 20degrees relative to tomography slice for visualization. Scale bar = 100 nm. n = 3 tomograms.\nFigure S9.\nMultilamellar membrane structure morphometric analysis. (A) Exemplar multilamellar membrane structure surfaces with membranes labeled. Primary membrane in red. Secondary membrane in yellow. Interior membranes in sage green. (B) Secondary and interior membrane surfaces colored by membrane curvedness; the primary membrane is represented as a transparent gray. (C) Primary surface is colored by membrane curvedness; the secondary and interior membranes are represented as a transparent gray. (D) Primary surface is colored by primary-to-secondary membrane distance; the secondary and interior membrane surfaces are shown in transparent gray. (E) Secondary surface colored by secondary to nearest interior (secondary-interior) membrane distance; the primary and interior membrane surfaces are shown in transparent gray. (F) Secondary and interior membranes surfaces are colored by angle relative to the primary membrane. Primary membrane surface is shown in transparent gray. (G) Secondary and interior membrane surfaces are colored by an angle relative to the growth plane of the cell. Primary membrane surface shown in transparent gray. (H) Primary membrane surface is colored by relative angle to the secondary membrane. Secondary and interior membrane surfaces are shown in transparent gray. (I) Primary membrane surface colored by relative angle to the growth plane of the cell. Secondary and interior membranes are shown in transparent gray.\nFigure S10.\nER membrane curvature analysis. (A) Representative membrane surface reconstructions of the ER in MEFmtGFP cells treated with vehicle and Tg (500 nM, 8 h) and associated with defined mitochondrial populations (elongated and fragmented). The ER surface is colored by membrane curvedness, and the OMM surface is represented as a transparent gray mesh. (B) Combined histograms of total ER curvedness of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. Graph (right) showing peak histogram values from each ER membrane within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 9, Tg elongated n = 8, vehicle fragmented n = 3, and Tg fragmented n = 4 ER are shown.\n\nMonitoring ER-mitochondrial contacts by surface morphometrics\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission (&lt;&lt;REF:Friedman-2011-334-358&gt;&gt;) to autophagy (&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;). To investigate whether the remodeling we observed in response to Tg-induced ER stress was correlated with alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs. fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present and visualized the contact sites on the surfaces of the OMM (Fig. S11, A and B &lt;&lt;FIG:figS11&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Fig. S11 C &lt;&lt;FIG:figS11&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time point were not associated with the altered mitochondrial ultrastructure (Fig. S11 D &lt;&lt;FIG:figS11&gt;&gt;).\nFigure S11.\nSurface analysis of mitochondria-ER contact sites. (A) Surface membrane reconstruction defining OMM (blue-yellow gradient) and ER (transparent mesh) distance measurement. (B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg (500 nM) colored by OMM to ER (OMM-ER) membrane distance. (C) Combined histogram of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Dashed vertical lines correspond to peak histogram values of pooled data. (D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Quantifications from vehicle elongated n = 18, Tg elongated n = 11, vehicle fragmented n = 7, Tg fragmented n = 13 mitochondria are shown.\n\nDiscussion\nWe describe a correlative cellular cryo-ET workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influence mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis. The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-ET to reveal organellar membrane ultrastructures (Fig. 1, A-D &lt;&lt;FIG:fig1&gt;&gt;, and Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semiautomated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high precision. We determined that the screened Poisson reconstruction approach (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;) provides a more accurate estimation of the implicit membrane geometry of complex membranes and can be used to model and quantify complex membranes such as the IMM.\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions and quantified mitochondrial membrane parameters such as inter- and intramitochondrial membrane distances, curvedness, and orientation. We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps. This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions.\nWe first used our workflow to quantify the intermembrane distance between the inner and outer membranes of mitochondria (Fig. 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;; &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;; &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;), and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes as well as to quantify changes in the overall distributions. We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). The difference in distance, from 13.5 to 12.7 nm between the centers of the membranes, is small, but correcting for the ~7 nm thickness of mitochondrial membranes (&lt;&lt;REF:Perkins-1997-119-260&gt;&gt;) yields a change in intermembrane contact distance from 6.5 to 5.7 nm or a 12% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as phosphatidylserine decarboxylase, an IMM protein previously suggested to act on the OMM (&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;). The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbations may impact intermembrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the translocase of the OMM complex (&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;), which associate with the MICOS complex and may impact MICOS complex organization (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;).\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex (&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;). We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figs. 4 &lt;&lt;FIG:fig4&gt;&gt; and S6 &lt;&lt;FIG:figS6&gt;&gt;), quantified the curvedness of the membranes (Figs. 5 &lt;&lt;FIG:fig5&gt;&gt;, S6 &lt;&lt;FIG:figS6&gt;&gt;, S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, and S10 &lt;&lt;FIG:figS10&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; and 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress (&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt;). In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase--a mechanism similar to that responsible for ER-stress-induced mitochondrial elongation (&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;). While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvedness. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvedness, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS (&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;), ATP synthase (&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;), and OPA1 (&lt;&lt;REF:Hu-2020-11-940&gt;&gt;), and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function.\nWe demonstrated the general utility of our workflow beyond mitochondrial membranes and analyzed the curvedness, intermembrane distances, and orientation of cellular membranes of the ER and observed multilamellar membrane structures (Fig. S7 &lt;&lt;FIG:figS7&gt;&gt;, S9 &lt;&lt;FIG:figS9&gt;&gt;, S10 &lt;&lt;FIG:figS10&gt;&gt;, and S11 &lt;&lt;FIG:figS11&gt;&gt;). Although our results did not reveal any statistically significant differences in the architecture of these cellular membranes upon Tg-induced ER stress, we anticipate that our workflow will facilitate future studies on the critical importance of membrane remodeling of these organelles induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series (&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;), which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Fig. 6 &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;) used in this study traces lower curved membranes more easily than high curved membranes, leading to limited segmentations of tight, high curved crista junctions and crista tips. We have mitigated these biases by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter (&lt;&lt;REF:Koning-2018-217-82&gt;&gt;) or phase plate (&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;), both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations (&lt;&lt;REF:Zhou-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Chen-2017-14-983&gt;&gt;; &lt;&lt;REF:Buchholz-2018-?-?&gt;&gt;; &lt;&lt;REF:Liu-2021-?-?&gt;&gt;\nPreprint). The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics (&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt;) and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template-matching approaches (&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;). These approaches have in the past depended on voxel segmentations, but moving to high-quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentation, which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well (&lt;&lt;REF:Basanta-2020-4-100041&gt;&gt;). Membrane meshes have been used very successfully to guide subtomogram averaging in the past (&lt;&lt;REF:Navarro-2018-5-82&gt;&gt;; &lt;&lt;REF:Burt-2021-?-?&gt;&gt;\nPreprint), but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu (&lt;&lt;REF:Erdmann-2021-12-5364&gt;&gt;; &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;; &lt;&lt;REF:O’Reilly-2020-?-?&gt;&gt;\nPreprint; &lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;; &lt;&lt;REF:Guo-2018-172-696&gt;&gt;; &lt;&lt;REF:Jordan-2018-20-1250&gt;&gt;; &lt;&lt;REF:Albert-2020-117-1069&gt;&gt;; &lt;&lt;REF:Lin-2018-360-360&gt;&gt;). Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nMaterials and methods\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM grids\nMouse embryonic fibroblasts with mitochondria-labeled GFP (MEFmtGFP; &lt;&lt;REF:Wang-2012-51-9302&gt;&gt;) were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 mug/ml; Corning) and UV-sterilized R (1/4) Carbon 200-mesh gold EM grids (Quantifoil Micro Tools). After 15-18 h of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group or fresh media supplemented with Thapsigargin (500 nM; Thermo Fisher Scientific) in the Tg treatment group. After 8 h of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific). The Vitrobot was set to 37degreesC and 100% relative humidity, and blotting was performed manually from the back side of grids using Whatman #1 filter paper strips through the Vitrobot humidity/temperature chamber side port. The Vitrobot settings used to disable automated blotting apparatus were as follows: Blot total: 0, 2; Blot force: 0, 3; Blot time: 0 s.\n\nCryo-fluorescence microscopy and mitochondria network morphology scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) and collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em: 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Fig. 2 A &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13; Thermo Fisher Scientific) for fluorescence-guided milling.\n\nFluorescence -guided milling\nCryo-FIB milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system for 3-4 s using an automation script (&lt;&lt;REF:Barad-2023-?-?&gt;&gt;), followed by a third and final layer of platinum sputter. Targeted cells were milled manually (&lt;&lt;REF:Schaffer-2015-5-5&gt;&gt;) or using an automated cryo preparation workflow (&lt;&lt;REF:Buckley-2020-210-107488&gt;&gt;), and both methods used xT software with MAPS (Thermo Fisher Scientific). Minimal SEM imaging for monitoring was done at 2 keV to ensure thin lamella generation while avoiding radiation damage. A total of six treated grids and eight untreated grids were milled for further tomography analysis.\n\nTilt series data collection\nEM grids containing lamellae were transferred into a 300 keV Titan Krios microscope (Thermo Fisher Scientific) equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1 e-/A2) at high magnification to localize cellular regions containing mitochondria, which were identified by their distinctive IMMs and OMMs. Data were collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;) with 2degrees steps between -60degrees and +60degrees. Data were collected with pixel sizes of 3.11 , 3.35 , or 3.55 A, and a nominal defocus range between -5 and -10 microm. A subset of data was collected with dose fractionation, with 10 0.1 e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2 and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose-fractionated tilt series micrograph movies underwent contrast transfer function estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series were then aligned using patch tracking in etomo (&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;), with four times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into four times binned tomograms. Tomogram thicknesses ranged from 95 to 233 nm.\n\nSemiautomated segmentation\nAutomated detection and enhancement of mitochondrial membranes were performed using the TomoSegMemTV software package (&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;). Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows: scale_space -s 3; dtvoting -s 12; surfaceness -m 0.17 -s 0.8 -p 0.8; dtvoting -w -s 10; surfaceness -S -s 0.75 -p 0.75 -l 20; thresholding -l 0.01-2 30; global analysis -3 100.\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher Scientific) for manual curation. Mitochondrial inner and outer membranes and ER were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nSoftware workflow\nThe surface morphometrics pipeline is a python-scripted workflow with requirements that can be installed as a conda environment contained in an \"environment.yml\" file. The workflow is fully scripted and configurable with a \"config.yml\" file and is run in three fully pipelined steps, with statistical analysis and visualization as an optional fourth step. First, a segmentation MRC file is converted automatically to a series of surface meshes formatted in the VTP file format. Second, for each mesh, the surface is converted to a graph (tg format) and curvature is estimated using pycurv. Third, orientations and distances between and within surfaces are calculated using the resulting graphs, and a CSV with quantifications as well as a final VTP surface file is output with all quantifications built in. Fourth, the outputs from multiple tomograms are combined for visualization and statistical analysis. Times and computational requirements are shown in Table S1.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab (&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;). First, the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm (&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;), with a reconstruction depth of 9, an interpolation weight of 0.7, and a minimum number of samples of 1.5. These settings were chosen to maximize correspondence to the data rather than smoothness. The resulting surface extended beyond the segmented region, so triangles more than 1.5 nm away from the point cloud were deleted. Interpolation weight (point_weight) and the mask distance (extrapolation_distance) are both configurable in the surface morphometrics pipeline if more aggressive smoothing and hole filling are desirable. The resulting mesh was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the \"segmentation_to_meshes.py\" script.\n\nRefinement of surface orientation and estimation of surface curvedness using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach (&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;). This vector voting process also resulted in robust estimation of curvature metrics. As the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (Eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvedness metrics, which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the \"run_pycurv.py\" script.curvedness=kappa12+kappa222\n(1)\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated using a k-dimensional tree. Both the distance and identifier of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the \"measure_distances_orientations.py\" script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those &gt;40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nAs all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously (&lt;&lt;REF:Collado-2019-51-476&gt;&gt;). To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into shorter and longer distances. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector-based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector-based neighbor. Those triangles were filtered out before any further quantifications. These measurements are made in the \"measure_distances_orientations.py\" script.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (Eq. 2):verticality=arccos(|normalz|)\n(2)\nComparisons of orientations between membranes were made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector to its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (Eq. 3):relative_angle=arccos|self.neighbor|\n(3)\nBoth orientation calculations are determined in the \"measure_distances_orientations.py\" script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization pipeline \"vtp\" (&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt;) format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview (&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;). Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area (&lt;&lt;REF:Hunter-2007-9-90&gt;&gt;). Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests (&lt;&lt;REF:Mann-1947-18-50&gt;&gt;) on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram rather than the histogram peak. Statistical testing based on distributions, rather than peaks, is complicated by unbiased assessment of the number of independent samples in each tomogram. Aggregation, statistics, and visualization are accomplished by the \"morphometrics_stats.py\" classes and functions, and the mitochondria-specific statistics and visualizations are generated by the \"mitochondria_statistics.py\" script. The complete output of the per mitochondrion analyses for every measurement is available at https://zenodo.org/record/7510830.\n\nOnline supplemental material\nFigs. S1 &lt;&lt;FIG:figS1&gt;&gt;, S2 &lt;&lt;FIG:figS2&gt;&gt;, and S3 &lt;&lt;FIG:figS3&gt;&gt; are a gallery of cellular membranes used for downstream quantifications from cells with elongated mitochondrial networks in the absence of ER stress, cells with elongated mitochondrial networks in the presence of ER stress, and cells with fragmented mitochondrial networks in the presence and absence of ER stress, respectively. Fig. S4 &lt;&lt;FIG:figS4&gt;&gt; shows a comparison of different surface reconstruction algorithms. Fig. S5 &lt;&lt;FIG:figS5&gt;&gt; is a gallery of histograms showing the distribution of IMM-OMM distances for each individual mitochondrion. Fig. S6 &lt;&lt;FIG:figS6&gt;&gt; presents additional quantifications that do not show statistically significant differences. Fig. S7 &lt;&lt;FIG:figS7&gt;&gt; shows analysis of changes to OMM curvature. Fig. S8 &lt;&lt;FIG:figS8&gt;&gt; shows a gallery of multilamellar membrane bodies observed in the data and their segmentations. Fig. S9 &lt;&lt;FIG:figS9&gt;&gt; shows a gallery of surface morphometrics pipeline quantifications mapped onto the multilamellar membrane bodies. Fig. S10 &lt;&lt;FIG:figS10&gt;&gt; shows analysis of changes to ER curvature. Fig. S11 &lt;&lt;FIG:figS11&gt;&gt; shows quantification of mitochondria-ER contact sites. Table S1 shows typical computational requirements and time required for each step of the surface morphometrics pipeline. Table S2 shows additional statistical assessments for each quantification and each condition. Table S3 shows the confidence intervals derived from bootstrap analysis of each quantification. Video 1 &lt;&lt;FIG:video1&gt;&gt; shows a comparison of voxel segmentation and implicit surface reconstruction.\n\nSupplementary Material\nReview History\nClick here for additional data file.\nTable S1\nshows the approximate time for each step of the surface morphometrics workflow.\nClick here for additional data file.\nTable S2\nshows the statistical analyses of peaks and standard deviations.\nClick here for additional data file.\nTable S3\nshows the confidence intervals of quantifications with bootstrap analysis.\nClick here for additional data file.\n\nData availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Installation is simplified with a conda environment and a pip requirements file. Scripts are licensed under the GNU Public License version 3 (GPLv3). All tilt series, reconstructed tomograms, voxel segmentations, and reconstructed mesh surfaces used for quantifications were deposited in the Electron Microscopy Public Image Archive under accession code EMPIAR-11370.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [18.61s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"cell culture\",\n  \"organism_name\": \"Mus musculus\",\n  \"cell_strain\": \"MEFmtGFP\",\n  \"sample_preparation\": \"Cells were cultured, treated with vehicle or Thapsigargin, and vitrified via plunge freezing in ethane/propane mixture.\",\n  \"grid_preparation\": \"Cultured on fibronectin-treated and UV-sterilized R(1/4) Carbon 200-mesh gold EM grids, coated with platinum before cryo-FIB milling.\",\n  \"cryoet_pixel_spacing\": \"3.11 to 3.55 A\",\n  \"cryoet_acceleration_voltage\": \"300 keV\",\n  \"microscope_name\": \"Titan Krios\",\n  \"microscope_setup_energy_filter\": \"BioQuantum energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K2 Summit\",\n  \"tilt_minimum\": \"-60 degrees\",\n  \"tilt_maximum\": \"60 degrees\",\n  \"total_flux\": \"under 70 e-/A2\",\n  \"reconstruction_software\": \"SerialEM, Warp, etomo\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"weighted back projection\",\n  \"annotated_entities\": \"mitochondrial inner and outer membranes, ER\",\n  \"annotation_methods\": \"automated detection and enhancement, manual curation\",\n  \"annotation_software\": \"TomoSegMemTV, AMIRA\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"cell culture\",\n  \"organism_name\": \"Mus musculus\",\n  \"cell_strain\": \"MEFmtGFP\",\n  \"sample_preparation\": \"Cells were cultured, treated with vehicle or Thapsigargin, and vitrified via plunge freezing in ethane/propane mixture.\",\n  \"grid_preparation\": \"Cultured on fibronectin-treated and UV-sterilized R(1/4) Carbon 200-mesh gold EM grids, coated with platinum before cryo-FIB milling.\",\n  \"cryoet_pixel_spacing\": \"3.11 to 3.55 A\",\n  \"cryoet_acceleration_voltage\": \"300 keV\",\n  \"microscope_name\": \"Titan Krios\",\n  \"microscope_setup_energy_filter\": \"BioQuantum energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K2 Summit\",\n  \"tilt_minimum\": \"-60 degrees\",\n  \"tilt_maximum\": \"60 degrees\",\n  \"total_flux\": \"under 70 e-/A2\",\n  \"reconstruction_software\": \"SerialEM, Warp, etomo\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"weighted back projection\",\n  \"annotated_entities\": \"mitochondrial inner and outer membranes, ER\",\n  \"annotation_methods\": \"automated detection and enhancement, manual curation\",\n  \"annotation_software\": \"TomoSegMemTV, AMIRA\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 345,
      "prompt_tokens": 18919,
      "total_tokens": 19264
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_7cc080b25b"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "biological_sample_type": "cell culture",
  "organism_name": "Mus musculus",
  "cell_strain": "MEFmtGFP",
  "sample_preparation": "Cells were cultured, treated with vehicle or Thapsigargin, and vitrified via plunge freezing in ethane/propane mixture.",
  "grid_preparation": "Cultured on fibronectin-treated and UV-sterilized R(1/4) Carbon 200-mesh gold EM grids, coated with platinum before cryo-FIB milling.",
  "cryoet_pixel_spacing": "3.11 to 3.55 A",
  "cryoet_acceleration_voltage": "300 keV",
  "microscope_name": "Titan Krios",
  "microscope_setup_energy_filter": "BioQuantum energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Gatan",
  "camera_model": "K2 Summit",
  "tilt_minimum": "-60 degrees",
  "tilt_maximum": "60 degrees",
  "total_flux": "under 70 e-/A2",
  "reconstruction_software": "SerialEM, Warp, etomo",
  "tomogram_size": "not present",
  "reconstruction_method": "weighted back projection",
  "annotated_entities": "mitochondrial inner and outer membranes, ER",
  "annotation_methods": "automated detection and enhancement, manual curation",
  "annotation_software": "TomoSegMemTV, AMIRA"
}
[chain/end] [1:chain:RunnableSequence] [18.62s] Exiting Chain run with output:
{
  "biological_sample_type": "cell culture",
  "organism_name": "Mus musculus",
  "cell_strain": "MEFmtGFP",
  "sample_preparation": "Cells were cultured, treated with vehicle or Thapsigargin, and vitrified via plunge freezing in ethane/propane mixture.",
  "grid_preparation": "Cultured on fibronectin-treated and UV-sterilized R(1/4) Carbon 200-mesh gold EM grids, coated with platinum before cryo-FIB milling.",
  "cryoet_pixel_spacing": "3.11 to 3.55 A",
  "cryoet_acceleration_voltage": "300 keV",
  "microscope_name": "Titan Krios",
  "microscope_setup_energy_filter": "BioQuantum energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Gatan",
  "camera_model": "K2 Summit",
  "tilt_minimum": "-60 degrees",
  "tilt_maximum": "60 degrees",
  "total_flux": "under 70 e-/A2",
  "reconstruction_software": "SerialEM, Warp, etomo",
  "tomogram_size": "not present",
  "reconstruction_method": "weighted back projection",
  "annotated_entities": "mitochondrial inner and outer membranes, ER",
  "annotation_methods": "automated detection and enhancement, manual curation",
  "annotation_software": "TomoSegMemTV, AMIRA"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "INTRODUCTION\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, three-dimensional (3D) views of protein and organellar structure in unfixed and fully hydrated conditions&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;. By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt;, &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;, it has become routine to collect large datasets of thin, cellular three-dimensional reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;-&lt;&lt;REF:Chen-2019-16-1161&gt;&gt;, it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments&lt;&lt;REF:Jiang-2017-7-?&gt;&gt;, &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;. As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the two-dimensional (2D) virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intra-membrane distances through manual designation of membrane boundaries, by locating distinct points along the membrane&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;-&lt;&lt;REF:Mageswaran-2021-7-?&gt;&gt;. Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user-bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Towards a more accurate 3D representation, manual or automated&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes&lt;&lt;REF:Chen-2017-14-983&gt;&gt;. Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvature and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvature. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvature as well as the spatial relationship between individual triangles (i.e. membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e. boundary segmentations) were not reliable, due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm&lt;&lt;REF:Hoppe-1992-?-71&gt;&gt;. In cryo-electron tomography, even perfect segmentations will never show complete and closed membranes, due to the substantial missing wedge&lt;&lt;REF:Lučič-2013-202-407&gt;&gt; making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e. compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-electron tomography.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling&lt;&lt;REF:Schirrmacher-2020-8-?&gt;&gt;-&lt;&lt;REF:Wai-2016-27-105&gt;&gt;. Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure)&lt;&lt;REF:Chan-2020-15-235&gt;&gt;. Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e. elongated) or disconnected (i.e. fragmented) networks that regulate pro-survival or pro-apoptotic mitochondrial functions&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;, &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;. Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;.\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvature required for efficient mitochondrial respiration,&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;-&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;. Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;-&lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;. Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;, &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;. It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-electron tomography workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semi-automated workflow&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; to segment cellular membranes present in cryo-tomograms, and applied a screened Poisson reconstruction strategy&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics toolkit to measure several parameters of membrane shape including inter- and intra-membrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of endoplasmic reticulum (ER) stress, a condition known to promote adaptive remodeling of mitochondrial networks&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;, to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nRESULTS\n\nCorrelative cryo fluorescence microscopy and electron tomography enables association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP), such that the morphology of each cell's mitochondrial network could be readily assessed (Figure 1A-C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification based on the fluorescence microscopy, then targeted cells with either fragmented or elongated mitochondrial networks for cryo-focused ion beam (cryo-FIB) milling to prepare thin (~100-200 nm) lamellae (Figure 1D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Figure 1C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;) to the ultrastructure of mitochondrial membranes (Figure 1 &lt;&lt;FIG:fig1&gt;&gt; F-H, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase (SERCA) and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (SIMH)&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. We applied our correlative approach to identify and target specific Tg-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Supplementary Figure 1). Notably, Tg treatment induced a shift towards regularly spaced disc-shaped (lamellar) cristae in the elongated population and towards a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-electron tomography workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions.\n(A) MEFmtGFP cells were cultured on transmission electron microscopy grids (black mesh circle) and treated with vehicle or thapsigargin (Tg, 500 nM) for 8 hours prior to vitrification via plunge freezing in ethane/propane mixture.\n(B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence (cryo-FM).\n(C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies.\n(D) These cells were targeted for cryo-focused ion beam (cryo-FIB) milling to generate thin sections (lamellae).\n(E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate three-dimensional (3D) cryo-tomograms.\n(F) Voxel segmentations labeling distinct cellular membranes were generated using a semi-automated approach&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point&lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;. Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, &lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;.\n(G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: inner mitochondrial membrane (IMM), purple; outer mitochondrial membrane (OMM), blue; endoplasmic reticulum (ER), teal.\n(H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP cells.\n(A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies in MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of ~150-200 nm. MEFmtGFP cell periphery is outlined in dashed white line.\n(B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue.\n(C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backwards 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semi-automated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work&lt;&lt;REF:Salfer-2020-16-?&gt;&gt; that established a robust approach for estimating curvature in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Navarro-2018-5-?&gt;&gt; and manual inpainting of entire compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;, both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semi-automated workflow to enable analysis at higher throughput (Figure 1F &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We used a segmentation strategy that takes advantage of the tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, which identifies and enhances regions of the tomogram with membrane-like voxel values. This allowed manual classification of membrane identity into mitochondrial IMM and OMM and ER membrane, as well as manual cleanup of individual membrane segmentations using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 32 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 1). In order to prepare high quality surface meshes appropriate for quantification, we established a fully automated surface reconstruction pipeline using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; (Figure 1G &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;, Supplementary figure 1). This allowed reconstruction of much higher quality membrane surface as compared to previously developed approaches for mesh reconstruction from membrane segmentations (Figure 2C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 2, Supplementary Movie 1). These surfaces were then passed into the 3D surface morphometrics workflow, generating quantifications such as inter- and intra-membrane distances, membrane curvature, and relative membrane orientations (Figure 1H &lt;&lt;FIG:fig1&gt;&gt;). This 3D surface morphometrics toolkit is available at https://github.com/grotjahnlab/surface_morphometrics.\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria are the juxtaposed inner and outer mitochondrial membranes. Cryo-electron tomography is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought out to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh (Figure 3A &lt;&lt;FIG:fig3&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions, but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented tomograms, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Figure 3C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in inter-membrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in the peak OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each tomogram as independent observations and identified statistically significant increases in spacing in fragmented vs elongated cells, by 1.7 nm in the vehicle-treated condition and by 3.9 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in fragmented cells after Tg treatment was not significant (Figure 3D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significance differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nInter-mitochondrial membrane distance is dependent on mitochondrial network morphology and presence or absence of ER stress.\n(A) Surface membrane reconstruction defining outer mitochondrial membrane (OMM, purple) and inner mitochondrial membrane (IMM, pink) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions.\n(C) Quantification of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Quantification of peak histogram values from each tomogram within the indicated treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, &lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;. We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intra-crista and intra-junction distances, respectively), as well as the distance between adjacent crista bodies and junctions (inter-crista and inter-junction distances, respectively) (Figure 4B &lt;&lt;FIG:fig4&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each tomogram (Figure 4C-F &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intra-crista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 4.5 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 3.2 nm upon Tg treatment (Figure 4C &lt;&lt;FIG:fig4&gt;&gt;). However, we did not observe significant changes in inter-cristae distance, intra-junction distance, or inter-junction distance (Figure 4D-F &lt;&lt;FIG:fig4&gt;&gt;), with the exception of a 9.4 nm increase in intra-junction spacing in the Tg-treated condition of elongated mitochondria relative to fragmented mitochondria (Figure 4E &lt;&lt;FIG:fig4&gt;&gt;), suggesting that ER stress promotes relaxation of the junction in fragmented mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations\n(A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with thapsigargin (Tg; 500 nM, 8h) showing subdivision of IMM compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh.\n(B) Enlarged inset of boxed region in elongated example from (A) showing surface membrane reconstruction colored by inner membrane compartments defining intra- and inter-crista and junction measurements.\n(C) Quantification of intra-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n(D) Quantification of inter-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(E) Quantification of intra-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(F) Quantification of inter-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvature (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;)&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated elongated cells, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Surprisingly, despite apparent visual difference, no significant changes were observed in IMM curvedness overall (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Because visual inspection revealed local changes in curvature in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;, Figure 5C-E &lt;&lt;FIG:fig5&gt;&gt;). Within the IBM, we observed a significant decrease in curvedness in fragmented relative to elongated mitochondrial populations in cells treated with vehicle (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Further, we found that Tg treatment significantly increased curvedness in elongated mitochondria, while no significant increase was observed in fragmented mitochondria from Tg-treated cells. This led to a significant increase in IBM curvedness in elongated mitochondria over fragmented mitochondria in Tg-treated cells (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Since the IBM's curvedness is constrained to be near-identical to that of the OMM, this metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length, while ER stress slightly reduces the radius of elongated mitochondria (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;, Supplemental Figure 1).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress.\n(A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principle components of curvature and is used because the surfaces normal vectors do not have canonical sign.\n(B) Quantification of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(C) Quantification of inner boundary membrane (IBM) curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\n(D) Quantification of cristae curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(E) Quantification of cristae junction curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\nIn contrast, larger changes were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress (Figure 5D &lt;&lt;FIG:fig5&gt;&gt;). Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated elongated cells compared to both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria (Figure 5E &lt;&lt;FIG:fig5&gt;&gt;). This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae were regulated, and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (Figure 6A &lt;&lt;FIG:fig6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Figure 6B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Figure 6C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position. Because of this, we quantified the standard deviation of these values for each tomogram in order to assess statistical significance of observed changes (Figure 6D-G &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells (Figure 6D &lt;&lt;FIG:fig6&gt;&gt;). We also identified a significant decrease in variability of the verticality of the crista body in Tg-treated elongated relative to fragmented mitochondria (Figure 6F &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM (Figure 6E &lt;&lt;FIG:fig6&gt;&gt;) or the growth plane (Figure 6G &lt;&lt;FIG:fig6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks\n(A) Surface membrane reconstruction colored by inner membrane compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple;) defining angular measurements, between the surface and the nearest OMM or to the growth plane of the cell.\n(B) Representative membrane surface reconstruction of a rigid and lamellar Tg-treated elongated mitochondrion, colored by angle of IMM relative to OMM.\n(C) Representative membrane surface reconstruction of a less rigidly oriented Tg-treated elongated mitochondrion, colored by angle of IMM relative to the growth plane of the cell.\n(D) Quantification of angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(E) Quantification of angle of junctions relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Violin plot (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(F) Quantification of angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(G) Quantification of angle of junctions relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nElongated and fragmented mitochondrial networks show differences in ER-mitochondrial contacts\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission&lt;&lt;REF:Friedman-2011-334-358&gt;&gt; to autophagy&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;. To investigate whether the remodeling we observed in response to Tg-induced ER stress was driven by alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present, and visualized the contact sites on the surfaces of the OMM (Figure 7A-B &lt;&lt;FIG:fig7&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Figure 7C &lt;&lt;FIG:fig7&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time-point were not driving the altered mitochondrial ultrastructure (Figure 7D &lt;&lt;FIG:fig7&gt;&gt;).\nFigure 7.\nMitochondria-ER contact sites can be localized in three dimensions on the outer membrane.\n(A) Surface membrane reconstruction defining OMM (purple) and ER (transparent) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Thapsigargin (Tg; 500 nM) colored by OMM to ER (OMM-ER) membrane distance.\n(C) Quantification of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-9 cryo-tomograms is shown.\n\nDISCUSSION\nWe describe a correlative cellular cryo-electron tomography workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular physiology conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influences mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;, Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-electron tomography to reveal organellar membrane ultrastructures (Figure 1A-D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semi-automated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high-precision (Figure 1E-H &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B,C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Movie 1). We determined that the screened Poisson reconstruction approach&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; provides a more accurate estimation of the implicit membrane geometry of complex membranes, and can be used to model and quantify complex membranes such as the IMM (Supplementary Figure 2, Supplementary Movie 1).\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions, and quantified mitochondrial membrane parameters such as inter- and intra-mitochondrial membrane distances (Figures 3 &lt;&lt;FIG:fig3&gt;&gt;-4 &lt;&lt;FIG:fig4&gt;&gt;), curvature (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and orientation (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps (Figures 3B &lt;&lt;FIG:fig3&gt;&gt;, 4A,B &lt;&lt;FIG:fig4&gt;&gt;, 5A &lt;&lt;FIG:fig5&gt;&gt;, 6B,C &lt;&lt;FIG:fig6&gt;&gt;). This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions (Figures 3C,D &lt;&lt;FIG:fig3&gt;&gt;, 4C-F &lt;&lt;FIG:fig4&gt;&gt;, 5C-E &lt;&lt;FIG:fig5&gt;&gt;, 6D-G &lt;&lt;FIG:fig6&gt;&gt;).\nWe first used our workflow to quantify the inter-membrane distance between the inner and outer membranes of mitochondria (Figure 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;, &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;, and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;. The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;) as well as to quantify changes in the overall distributions (Figure 3C-D &lt;&lt;FIG:fig3&gt;&gt;). We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion.&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt; The difference in distance, from 13.6 to 12.8 nm between the centers of the membranes, is small, but correcting for the approximately 7 nm thickness of mitochondrial membranes&lt;&lt;REF:Perkins-1997-119-260&gt;&gt; yields a change in intermembrane contact distance from 6.6 to 5.8 nm, or a 15% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as PISD, an IMM protein previously suggested to act on the OMM.&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt; The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbation may impact inter-membrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the TOM complex&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, which associate with the MICOS complex and may impact MICOS complex organization&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;.\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;. We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figure 4 &lt;&lt;FIG:fig4&gt;&gt;), quantified the curvedness of the membranes (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress.&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt; In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase - a mechanism similar to that responsible for ER-stress induced mitochondrial elongation&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvature. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvature, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;, ATP synthase&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;, and Opa1&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function. Our workflow will facilitate future studies on the critical importance of mitochondrial remodeling induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;, which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Figure 6F,G &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; used in this study traces lower curvature membranes more easily than high curvature membranes, leading to limited segmentations of tight, high curvature crista junctions and crista tips. We have mitigated these biases to the best of our ability by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter&lt;&lt;REF:Koning-2018-217-82&gt;&gt; or phase plate&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;, both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations&lt;&lt;REF:Chen-2017-14-983&gt;&gt;, &lt;&lt;REF:Zhou-2020-?-?&gt;&gt;-&lt;&lt;REF:Liu-2021-?-?&gt;&gt;. The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt; and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template matching approaches&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;. These approaches have in the past depended on voxel segmentations, but moving to high quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentations which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well&lt;&lt;REF:Basanta-2020-4-?&gt;&gt;. Membrane meshes have been used very successfully to guide subtomogram averaging in the past&lt;&lt;REF:Navarro-2018-5-?&gt;&gt;, &lt;&lt;REF:Burt-2021-?-?&gt;&gt;, but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and to much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu&lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;, &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, 55-&lt;&lt;REF:Lin-2018-360-?&gt;&gt;. Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nSupporting information\nSupplemental Figures\nSupplemental Movie 1\n\nCONFLICTS OF INTEREST\nThe authors declare that they have no conflict of interest.\n\nMATERIALS AND METHODS\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM Grids\nMouse embryonic fibroblasts with mitochondria labeled GFP (MEFmtGFP)&lt;&lt;REF:Wang-2012-51-9302&gt;&gt; were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 ug/ml, Corning) and UV sterilized R (1/4) Carbon 200-mesh gold electron microscopy (EM) grids (Quantifoil Micro Tools). After 15-18 hours of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group, or fresh media supplemented with Thapsigargin (500nM, Fisher Scientific) in the Tg treatment group. After 8 hours of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific) with manual back-blotting.\n\nCryo-fluorescence microscopy and Mitochondria Network Morphology Scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) were collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em : 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Figure 2a &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13) (Thermo Fisher) for fluorescence guided milling.\n\nFluorescence Guided Milling\nCryo-Focused Ion Beam milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to an SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and based on their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system (GIS) for 3-4 seconds using an automation script&lt;&lt;REF:Barad-?-?-?&gt;&gt;, followed by a third and final layer of platinum sputter was added. Targeted cells were milled manually&lt;&lt;REF:Schaffer-2015-5-?&gt;&gt; or using an automated cryo preparation workflow&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt; both methods used xT software with MAPS (Thermo Fisher). Minimal SEM imaging for monitoring was done at 2keV to ensure thin lamella generation while avoiding radiation damage. A total of 6 treated grids and 8 untreated grids were milled for further tomography analysis.\n\nTilt Series Data Collection\nEM grids containing lamellae were transferred into a 300keV Titan Krios microscope (Thermo Fisher Scientific), equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1e-/A2) at high magnification (53,000x nominal magnification) to localize cellular regions containing mitochondria, which were identified by their distinctive inner and outer mitochondrial membranes. Data was collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (Mastronarde &lt;&lt;REF:???-2007-9-90&gt;&gt; ) with 2 steps between -60 and +60. Data was collected with pixel sizes of 3.11A, 3.35 A, or 3.55 A, and a nominal defocus range between -5 to -10 microm. A subset of data was collected with dose fractionation, with 10 0.1e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2, and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose fractionated tilt series micrograph movies underwent CTF estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series then were aligned using patch tracking in etomo&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;, with 4 times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into 4 times binned tomograms. Tomogram thicknesses ranged from 95 nm to 233 nm.\n\nSemi-automated Segmentation\nAutomated detection and enhancement of mitochondrial membranes was performed using the TomoSegMemTV software package&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;. Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows:\nscale_space -s 3\ndtvoting -s 12\nsurfaceness -m 0.17 -s 0.8 -p 0.8\ndtvoting -w -s 10\nsurfaceness -S -s 0.75 -p 0.75 -l 20\nthresholding -l 0.01 -2 30\nglobal analysis -3 100\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher) for manual curation. Mitochondrial inner and outer membranes and endoplasmic reticulum were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;. First the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, with a reconstruction depth of 9, an interpolation weight of 4, and a minimum number of samples of 1.5. The resulting surface extended beyond the segmented region, so triangles more than 1 voxel away from the point cloud were deleted. In cases where the resulting surface was very complex, the surface was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the 'mrc2xyz.py' (for point cloud generation) and 'make_mesh.py' scripts.\n\nRefinement of surface orientation and estimation of surface curvature using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. This vector voting process also resulted in robust estimation of curvature metrics. Because the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvature metrics. which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the 'calculate_curvature_args.py' script, after file conversion using the 'mask_and_convert_ply.py' script.\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated. For surfaces with small numbers of triangles, this was accomplished using a distance matrix, while calculations on large surfaces, where the distance matrix would take up more than 32GB of system memory, were accomplished using a KD-tree. Both the distance and ID of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the 'surface_to_surface.py' script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those more than 40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nBecause all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph, rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously&lt;&lt;REF:Collado-2019-51-476&gt;&gt;. To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into a shorter and longer distance. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector based neighbor. Those triangles were filtered out before any further quantifications.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and to the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (eq. 2):\nComparisons of orientations between membranes was made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (eq. 3):\nBoth orientation calculations are determined in the 'surface_to_surface.py' script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization toolkit 'vtp'&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt; format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;. Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area.&lt;&lt;REF:???-2007-9-90&gt;&gt; Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests&lt;&lt;REF:Mann-1947-18-50&gt;&gt; on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram, rather than the histogram peak.\n\nCode Availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Scripts are licensed under the GNU Public License version 3 (GPLv3).",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "INTRODUCTION\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, three-dimensional (3D) views of protein and organellar structure in unfixed and fully hydrated conditions&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;. By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt;, &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;, it has become routine to collect large datasets of thin, cellular three-dimensional reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;-&lt;&lt;REF:Chen-2019-16-1161&gt;&gt;, it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments&lt;&lt;REF:Jiang-2017-7-?&gt;&gt;, &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;. As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the two-dimensional (2D) virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intra-membrane distances through manual designation of membrane boundaries, by locating distinct points along the membrane&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;-&lt;&lt;REF:Mageswaran-2021-7-?&gt;&gt;. Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user-bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Towards a more accurate 3D representation, manual or automated&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes&lt;&lt;REF:Chen-2017-14-983&gt;&gt;. Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvature and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvature. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvature as well as the spatial relationship between individual triangles (i.e. membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e. boundary segmentations) were not reliable, due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm&lt;&lt;REF:Hoppe-1992-?-71&gt;&gt;. In cryo-electron tomography, even perfect segmentations will never show complete and closed membranes, due to the substantial missing wedge&lt;&lt;REF:Lučič-2013-202-407&gt;&gt; making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e. compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-electron tomography.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling&lt;&lt;REF:Schirrmacher-2020-8-?&gt;&gt;-&lt;&lt;REF:Wai-2016-27-105&gt;&gt;. Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure)&lt;&lt;REF:Chan-2020-15-235&gt;&gt;. Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e. elongated) or disconnected (i.e. fragmented) networks that regulate pro-survival or pro-apoptotic mitochondrial functions&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;, &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;. Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;.\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvature required for efficient mitochondrial respiration,&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;-&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;. Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;-&lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;. Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;, &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;. It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-electron tomography workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semi-automated workflow&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; to segment cellular membranes present in cryo-tomograms, and applied a screened Poisson reconstruction strategy&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics toolkit to measure several parameters of membrane shape including inter- and intra-membrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of endoplasmic reticulum (ER) stress, a condition known to promote adaptive remodeling of mitochondrial networks&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;, to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nRESULTS\n\nCorrelative cryo fluorescence microscopy and electron tomography enables association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP), such that the morphology of each cell's mitochondrial network could be readily assessed (Figure 1A-C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification based on the fluorescence microscopy, then targeted cells with either fragmented or elongated mitochondrial networks for cryo-focused ion beam (cryo-FIB) milling to prepare thin (~100-200 nm) lamellae (Figure 1D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Figure 1C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;) to the ultrastructure of mitochondrial membranes (Figure 1 &lt;&lt;FIG:fig1&gt;&gt; F-H, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase (SERCA) and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (SIMH)&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. We applied our correlative approach to identify and target specific Tg-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Supplementary Figure 1). Notably, Tg treatment induced a shift towards regularly spaced disc-shaped (lamellar) cristae in the elongated population and towards a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-electron tomography workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions.\n(A) MEFmtGFP cells were cultured on transmission electron microscopy grids (black mesh circle) and treated with vehicle or thapsigargin (Tg, 500 nM) for 8 hours prior to vitrification via plunge freezing in ethane/propane mixture.\n(B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence (cryo-FM).\n(C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies.\n(D) These cells were targeted for cryo-focused ion beam (cryo-FIB) milling to generate thin sections (lamellae).\n(E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate three-dimensional (3D) cryo-tomograms.\n(F) Voxel segmentations labeling distinct cellular membranes were generated using a semi-automated approach&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point&lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;. Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, &lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;.\n(G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: inner mitochondrial membrane (IMM), purple; outer mitochondrial membrane (OMM), blue; endoplasmic reticulum (ER), teal.\n(H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP cells.\n(A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies in MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of ~150-200 nm. MEFmtGFP cell periphery is outlined in dashed white line.\n(B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue.\n(C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backwards 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semi-automated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work&lt;&lt;REF:Salfer-2020-16-?&gt;&gt; that established a robust approach for estimating curvature in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Navarro-2018-5-?&gt;&gt; and manual inpainting of entire compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;, both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semi-automated workflow to enable analysis at higher throughput (Figure 1F &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We used a segmentation strategy that takes advantage of the tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, which identifies and enhances regions of the tomogram with membrane-like voxel values. This allowed manual classification of membrane identity into mitochondrial IMM and OMM and ER membrane, as well as manual cleanup of individual membrane segmentations using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 32 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 1). In order to prepare high quality surface meshes appropriate for quantification, we established a fully automated surface reconstruction pipeline using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; (Figure 1G &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;, Supplementary figure 1). This allowed reconstruction of much higher quality membrane surface as compared to previously developed approaches for mesh reconstruction from membrane segmentations (Figure 2C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 2, Supplementary Movie 1). These surfaces were then passed into the 3D surface morphometrics workflow, generating quantifications such as inter- and intra-membrane distances, membrane curvature, and relative membrane orientations (Figure 1H &lt;&lt;FIG:fig1&gt;&gt;). This 3D surface morphometrics toolkit is available at https://github.com/grotjahnlab/surface_morphometrics.\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria are the juxtaposed inner and outer mitochondrial membranes. Cryo-electron tomography is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought out to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh (Figure 3A &lt;&lt;FIG:fig3&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions, but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented tomograms, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Figure 3C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in inter-membrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in the peak OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each tomogram as independent observations and identified statistically significant increases in spacing in fragmented vs elongated cells, by 1.7 nm in the vehicle-treated condition and by 3.9 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in fragmented cells after Tg treatment was not significant (Figure 3D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significance differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nInter-mitochondrial membrane distance is dependent on mitochondrial network morphology and presence or absence of ER stress.\n(A) Surface membrane reconstruction defining outer mitochondrial membrane (OMM, purple) and inner mitochondrial membrane (IMM, pink) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions.\n(C) Quantification of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Quantification of peak histogram values from each tomogram within the indicated treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, &lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;. We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intra-crista and intra-junction distances, respectively), as well as the distance between adjacent crista bodies and junctions (inter-crista and inter-junction distances, respectively) (Figure 4B &lt;&lt;FIG:fig4&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each tomogram (Figure 4C-F &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intra-crista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 4.5 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 3.2 nm upon Tg treatment (Figure 4C &lt;&lt;FIG:fig4&gt;&gt;). However, we did not observe significant changes in inter-cristae distance, intra-junction distance, or inter-junction distance (Figure 4D-F &lt;&lt;FIG:fig4&gt;&gt;), with the exception of a 9.4 nm increase in intra-junction spacing in the Tg-treated condition of elongated mitochondria relative to fragmented mitochondria (Figure 4E &lt;&lt;FIG:fig4&gt;&gt;), suggesting that ER stress promotes relaxation of the junction in fragmented mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations\n(A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with thapsigargin (Tg; 500 nM, 8h) showing subdivision of IMM compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh.\n(B) Enlarged inset of boxed region in elongated example from (A) showing surface membrane reconstruction colored by inner membrane compartments defining intra- and inter-crista and junction measurements.\n(C) Quantification of intra-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n(D) Quantification of inter-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(E) Quantification of intra-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(F) Quantification of inter-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvature (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;)&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated elongated cells, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Surprisingly, despite apparent visual difference, no significant changes were observed in IMM curvedness overall (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Because visual inspection revealed local changes in curvature in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;, Figure 5C-E &lt;&lt;FIG:fig5&gt;&gt;). Within the IBM, we observed a significant decrease in curvedness in fragmented relative to elongated mitochondrial populations in cells treated with vehicle (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Further, we found that Tg treatment significantly increased curvedness in elongated mitochondria, while no significant increase was observed in fragmented mitochondria from Tg-treated cells. This led to a significant increase in IBM curvedness in elongated mitochondria over fragmented mitochondria in Tg-treated cells (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Since the IBM's curvedness is constrained to be near-identical to that of the OMM, this metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length, while ER stress slightly reduces the radius of elongated mitochondria (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;, Supplemental Figure 1).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress.\n(A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principle components of curvature and is used because the surfaces normal vectors do not have canonical sign.\n(B) Quantification of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(C) Quantification of inner boundary membrane (IBM) curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\n(D) Quantification of cristae curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(E) Quantification of cristae junction curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\nIn contrast, larger changes were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress (Figure 5D &lt;&lt;FIG:fig5&gt;&gt;). Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated elongated cells compared to both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria (Figure 5E &lt;&lt;FIG:fig5&gt;&gt;). This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae were regulated, and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (Figure 6A &lt;&lt;FIG:fig6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Figure 6B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Figure 6C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position. Because of this, we quantified the standard deviation of these values for each tomogram in order to assess statistical significance of observed changes (Figure 6D-G &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells (Figure 6D &lt;&lt;FIG:fig6&gt;&gt;). We also identified a significant decrease in variability of the verticality of the crista body in Tg-treated elongated relative to fragmented mitochondria (Figure 6F &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM (Figure 6E &lt;&lt;FIG:fig6&gt;&gt;) or the growth plane (Figure 6G &lt;&lt;FIG:fig6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks\n(A) Surface membrane reconstruction colored by inner membrane compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple;) defining angular measurements, between the surface and the nearest OMM or to the growth plane of the cell.\n(B) Representative membrane surface reconstruction of a rigid and lamellar Tg-treated elongated mitochondrion, colored by angle of IMM relative to OMM.\n(C) Representative membrane surface reconstruction of a less rigidly oriented Tg-treated elongated mitochondrion, colored by angle of IMM relative to the growth plane of the cell.\n(D) Quantification of angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(E) Quantification of angle of junctions relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Violin plot (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(F) Quantification of angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(G) Quantification of angle of junctions relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nElongated and fragmented mitochondrial networks show differences in ER-mitochondrial contacts\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission&lt;&lt;REF:Friedman-2011-334-358&gt;&gt; to autophagy&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;. To investigate whether the remodeling we observed in response to Tg-induced ER stress was driven by alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present, and visualized the contact sites on the surfaces of the OMM (Figure 7A-B &lt;&lt;FIG:fig7&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Figure 7C &lt;&lt;FIG:fig7&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time-point were not driving the altered mitochondrial ultrastructure (Figure 7D &lt;&lt;FIG:fig7&gt;&gt;).\nFigure 7.\nMitochondria-ER contact sites can be localized in three dimensions on the outer membrane.\n(A) Surface membrane reconstruction defining OMM (purple) and ER (transparent) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Thapsigargin (Tg; 500 nM) colored by OMM to ER (OMM-ER) membrane distance.\n(C) Quantification of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-9 cryo-tomograms is shown.\n\nDISCUSSION\nWe describe a correlative cellular cryo-electron tomography workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular physiology conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influences mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;, Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-electron tomography to reveal organellar membrane ultrastructures (Figure 1A-D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semi-automated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high-precision (Figure 1E-H &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B,C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Movie 1). We determined that the screened Poisson reconstruction approach&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; provides a more accurate estimation of the implicit membrane geometry of complex membranes, and can be used to model and quantify complex membranes such as the IMM (Supplementary Figure 2, Supplementary Movie 1).\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions, and quantified mitochondrial membrane parameters such as inter- and intra-mitochondrial membrane distances (Figures 3 &lt;&lt;FIG:fig3&gt;&gt;-4 &lt;&lt;FIG:fig4&gt;&gt;), curvature (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and orientation (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps (Figures 3B &lt;&lt;FIG:fig3&gt;&gt;, 4A,B &lt;&lt;FIG:fig4&gt;&gt;, 5A &lt;&lt;FIG:fig5&gt;&gt;, 6B,C &lt;&lt;FIG:fig6&gt;&gt;). This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions (Figures 3C,D &lt;&lt;FIG:fig3&gt;&gt;, 4C-F &lt;&lt;FIG:fig4&gt;&gt;, 5C-E &lt;&lt;FIG:fig5&gt;&gt;, 6D-G &lt;&lt;FIG:fig6&gt;&gt;).\nWe first used our workflow to quantify the inter-membrane distance between the inner and outer membranes of mitochondria (Figure 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;, &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;, and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;. The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;) as well as to quantify changes in the overall distributions (Figure 3C-D &lt;&lt;FIG:fig3&gt;&gt;). We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion.&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt; The difference in distance, from 13.6 to 12.8 nm between the centers of the membranes, is small, but correcting for the approximately 7 nm thickness of mitochondrial membranes&lt;&lt;REF:Perkins-1997-119-260&gt;&gt; yields a change in intermembrane contact distance from 6.6 to 5.8 nm, or a 15% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as PISD, an IMM protein previously suggested to act on the OMM.&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt; The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbation may impact inter-membrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the TOM complex&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, which associate with the MICOS complex and may impact MICOS complex organization&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;.\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;. We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figure 4 &lt;&lt;FIG:fig4&gt;&gt;), quantified the curvedness of the membranes (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress.&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt; In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase - a mechanism similar to that responsible for ER-stress induced mitochondrial elongation&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvature. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvature, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;, ATP synthase&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;, and Opa1&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function. Our workflow will facilitate future studies on the critical importance of mitochondrial remodeling induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;, which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Figure 6F,G &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; used in this study traces lower curvature membranes more easily than high curvature membranes, leading to limited segmentations of tight, high curvature crista junctions and crista tips. We have mitigated these biases to the best of our ability by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter&lt;&lt;REF:Koning-2018-217-82&gt;&gt; or phase plate&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;, both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations&lt;&lt;REF:Chen-2017-14-983&gt;&gt;, &lt;&lt;REF:Zhou-2020-?-?&gt;&gt;-&lt;&lt;REF:Liu-2021-?-?&gt;&gt;. The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt; and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template matching approaches&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;. These approaches have in the past depended on voxel segmentations, but moving to high quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentations which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well&lt;&lt;REF:Basanta-2020-4-?&gt;&gt;. Membrane meshes have been used very successfully to guide subtomogram averaging in the past&lt;&lt;REF:Navarro-2018-5-?&gt;&gt;, &lt;&lt;REF:Burt-2021-?-?&gt;&gt;, but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and to much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu&lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;, &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, 55-&lt;&lt;REF:Lin-2018-360-?&gt;&gt;. Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nSupporting information\nSupplemental Figures\nSupplemental Movie 1\n\nCONFLICTS OF INTEREST\nThe authors declare that they have no conflict of interest.\n\nMATERIALS AND METHODS\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM Grids\nMouse embryonic fibroblasts with mitochondria labeled GFP (MEFmtGFP)&lt;&lt;REF:Wang-2012-51-9302&gt;&gt; were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 ug/ml, Corning) and UV sterilized R (1/4) Carbon 200-mesh gold electron microscopy (EM) grids (Quantifoil Micro Tools). After 15-18 hours of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group, or fresh media supplemented with Thapsigargin (500nM, Fisher Scientific) in the Tg treatment group. After 8 hours of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific) with manual back-blotting.\n\nCryo-fluorescence microscopy and Mitochondria Network Morphology Scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) were collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em : 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Figure 2a &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13) (Thermo Fisher) for fluorescence guided milling.\n\nFluorescence Guided Milling\nCryo-Focused Ion Beam milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to an SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and based on their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system (GIS) for 3-4 seconds using an automation script&lt;&lt;REF:Barad-?-?-?&gt;&gt;, followed by a third and final layer of platinum sputter was added. Targeted cells were milled manually&lt;&lt;REF:Schaffer-2015-5-?&gt;&gt; or using an automated cryo preparation workflow&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt; both methods used xT software with MAPS (Thermo Fisher). Minimal SEM imaging for monitoring was done at 2keV to ensure thin lamella generation while avoiding radiation damage. A total of 6 treated grids and 8 untreated grids were milled for further tomography analysis.\n\nTilt Series Data Collection\nEM grids containing lamellae were transferred into a 300keV Titan Krios microscope (Thermo Fisher Scientific), equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1e-/A2) at high magnification (53,000x nominal magnification) to localize cellular regions containing mitochondria, which were identified by their distinctive inner and outer mitochondrial membranes. Data was collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (Mastronarde &lt;&lt;REF:???-2007-9-90&gt;&gt; ) with 2 steps between -60 and +60. Data was collected with pixel sizes of 3.11A, 3.35 A, or 3.55 A, and a nominal defocus range between -5 to -10 microm. A subset of data was collected with dose fractionation, with 10 0.1e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2, and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose fractionated tilt series micrograph movies underwent CTF estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series then were aligned using patch tracking in etomo&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;, with 4 times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into 4 times binned tomograms. Tomogram thicknesses ranged from 95 nm to 233 nm.\n\nSemi-automated Segmentation\nAutomated detection and enhancement of mitochondrial membranes was performed using the TomoSegMemTV software package&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;. Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows:\nscale_space -s 3\ndtvoting -s 12\nsurfaceness -m 0.17 -s 0.8 -p 0.8\ndtvoting -w -s 10\nsurfaceness -S -s 0.75 -p 0.75 -l 20\nthresholding -l 0.01 -2 30\nglobal analysis -3 100\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher) for manual curation. Mitochondrial inner and outer membranes and endoplasmic reticulum were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;. First the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, with a reconstruction depth of 9, an interpolation weight of 4, and a minimum number of samples of 1.5. The resulting surface extended beyond the segmented region, so triangles more than 1 voxel away from the point cloud were deleted. In cases where the resulting surface was very complex, the surface was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the 'mrc2xyz.py' (for point cloud generation) and 'make_mesh.py' scripts.\n\nRefinement of surface orientation and estimation of surface curvature using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. This vector voting process also resulted in robust estimation of curvature metrics. Because the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvature metrics. which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the 'calculate_curvature_args.py' script, after file conversion using the 'mask_and_convert_ply.py' script.\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated. For surfaces with small numbers of triangles, this was accomplished using a distance matrix, while calculations on large surfaces, where the distance matrix would take up more than 32GB of system memory, were accomplished using a KD-tree. Both the distance and ID of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the 'surface_to_surface.py' script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those more than 40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nBecause all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph, rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously&lt;&lt;REF:Collado-2019-51-476&gt;&gt;. To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into a shorter and longer distance. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector based neighbor. Those triangles were filtered out before any further quantifications.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and to the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (eq. 2):\nComparisons of orientations between membranes was made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (eq. 3):\nBoth orientation calculations are determined in the 'surface_to_surface.py' script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization toolkit 'vtp'&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt; format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;. Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area.&lt;&lt;REF:???-2007-9-90&gt;&gt; Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests&lt;&lt;REF:Mann-1947-18-50&gt;&gt; on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram, rather than the histogram peak.\n\nCode Availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Scripts are licensed under the GNU Public License version 3 (GPLv3).",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "study the microscopic structure of a biological sample",
  "questions_output_specification": "1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''INTRODUCTION\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, three-dimensional (3D) views of protein and organellar structure in unfixed and fully hydrated conditions&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;. By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt;, &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;, it has become routine to collect large datasets of thin, cellular three-dimensional reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;-&lt;&lt;REF:Chen-2019-16-1161&gt;&gt;, it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments&lt;&lt;REF:Jiang-2017-7-?&gt;&gt;, &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;. As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the two-dimensional (2D) virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intra-membrane distances through manual designation of membrane boundaries, by locating distinct points along the membrane&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;-&lt;&lt;REF:Mageswaran-2021-7-?&gt;&gt;. Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user-bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Towards a more accurate 3D representation, manual or automated&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes&lt;&lt;REF:Chen-2017-14-983&gt;&gt;. Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvature and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvature. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvature as well as the spatial relationship between individual triangles (i.e. membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e. boundary segmentations) were not reliable, due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm&lt;&lt;REF:Hoppe-1992-?-71&gt;&gt;. In cryo-electron tomography, even perfect segmentations will never show complete and closed membranes, due to the substantial missing wedge&lt;&lt;REF:Lučič-2013-202-407&gt;&gt; making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e. compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-electron tomography.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling&lt;&lt;REF:Schirrmacher-2020-8-?&gt;&gt;-&lt;&lt;REF:Wai-2016-27-105&gt;&gt;. Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure)&lt;&lt;REF:Chan-2020-15-235&gt;&gt;. Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e. elongated) or disconnected (i.e. fragmented) networks that regulate pro-survival or pro-apoptotic mitochondrial functions&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;, &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;. Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;.\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvature required for efficient mitochondrial respiration,&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;-&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;. Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;-&lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;. Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;, &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;. It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-electron tomography workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semi-automated workflow&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; to segment cellular membranes present in cryo-tomograms, and applied a screened Poisson reconstruction strategy&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics toolkit to measure several parameters of membrane shape including inter- and intra-membrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of endoplasmic reticulum (ER) stress, a condition known to promote adaptive remodeling of mitochondrial networks&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;, to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nRESULTS\n\nCorrelative cryo fluorescence microscopy and electron tomography enables association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP), such that the morphology of each cell's mitochondrial network could be readily assessed (Figure 1A-C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification based on the fluorescence microscopy, then targeted cells with either fragmented or elongated mitochondrial networks for cryo-focused ion beam (cryo-FIB) milling to prepare thin (~100-200 nm) lamellae (Figure 1D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Figure 1C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;) to the ultrastructure of mitochondrial membranes (Figure 1 &lt;&lt;FIG:fig1&gt;&gt; F-H, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase (SERCA) and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (SIMH)&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. We applied our correlative approach to identify and target specific Tg-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Supplementary Figure 1). Notably, Tg treatment induced a shift towards regularly spaced disc-shaped (lamellar) cristae in the elongated population and towards a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-electron tomography workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions.\n(A) MEFmtGFP cells were cultured on transmission electron microscopy grids (black mesh circle) and treated with vehicle or thapsigargin (Tg, 500 nM) for 8 hours prior to vitrification via plunge freezing in ethane/propane mixture.\n(B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence (cryo-FM).\n(C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies.\n(D) These cells were targeted for cryo-focused ion beam (cryo-FIB) milling to generate thin sections (lamellae).\n(E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate three-dimensional (3D) cryo-tomograms.\n(F) Voxel segmentations labeling distinct cellular membranes were generated using a semi-automated approach&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point&lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;. Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, &lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;.\n(G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: inner mitochondrial membrane (IMM), purple; outer mitochondrial membrane (OMM), blue; endoplasmic reticulum (ER), teal.\n(H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP cells.\n(A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies in MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of ~150-200 nm. MEFmtGFP cell periphery is outlined in dashed white line.\n(B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue.\n(C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backwards 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semi-automated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work&lt;&lt;REF:Salfer-2020-16-?&gt;&gt; that established a robust approach for estimating curvature in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Navarro-2018-5-?&gt;&gt; and manual inpainting of entire compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;, both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semi-automated workflow to enable analysis at higher throughput (Figure 1F &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We used a segmentation strategy that takes advantage of the tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, which identifies and enhances regions of the tomogram with membrane-like voxel values. This allowed manual classification of membrane identity into mitochondrial IMM and OMM and ER membrane, as well as manual cleanup of individual membrane segmentations using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 32 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 1). In order to prepare high quality surface meshes appropriate for quantification, we established a fully automated surface reconstruction pipeline using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; (Figure 1G &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;, Supplementary figure 1). This allowed reconstruction of much higher quality membrane surface as compared to previously developed approaches for mesh reconstruction from membrane segmentations (Figure 2C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 2, Supplementary Movie 1). These surfaces were then passed into the 3D surface morphometrics workflow, generating quantifications such as inter- and intra-membrane distances, membrane curvature, and relative membrane orientations (Figure 1H &lt;&lt;FIG:fig1&gt;&gt;). This 3D surface morphometrics toolkit is available at https://github.com/grotjahnlab/surface_morphometrics.\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria are the juxtaposed inner and outer mitochondrial membranes. Cryo-electron tomography is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought out to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh (Figure 3A &lt;&lt;FIG:fig3&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions, but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented tomograms, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Figure 3C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in inter-membrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in the peak OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each tomogram as independent observations and identified statistically significant increases in spacing in fragmented vs elongated cells, by 1.7 nm in the vehicle-treated condition and by 3.9 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in fragmented cells after Tg treatment was not significant (Figure 3D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significance differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nInter-mitochondrial membrane distance is dependent on mitochondrial network morphology and presence or absence of ER stress.\n(A) Surface membrane reconstruction defining outer mitochondrial membrane (OMM, purple) and inner mitochondrial membrane (IMM, pink) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions.\n(C) Quantification of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Quantification of peak histogram values from each tomogram within the indicated treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, &lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;. We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intra-crista and intra-junction distances, respectively), as well as the distance between adjacent crista bodies and junctions (inter-crista and inter-junction distances, respectively) (Figure 4B &lt;&lt;FIG:fig4&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each tomogram (Figure 4C-F &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intra-crista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 4.5 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 3.2 nm upon Tg treatment (Figure 4C &lt;&lt;FIG:fig4&gt;&gt;). However, we did not observe significant changes in inter-cristae distance, intra-junction distance, or inter-junction distance (Figure 4D-F &lt;&lt;FIG:fig4&gt;&gt;), with the exception of a 9.4 nm increase in intra-junction spacing in the Tg-treated condition of elongated mitochondria relative to fragmented mitochondria (Figure 4E &lt;&lt;FIG:fig4&gt;&gt;), suggesting that ER stress promotes relaxation of the junction in fragmented mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations\n(A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with thapsigargin (Tg; 500 nM, 8h) showing subdivision of IMM compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh.\n(B) Enlarged inset of boxed region in elongated example from (A) showing surface membrane reconstruction colored by inner membrane compartments defining intra- and inter-crista and junction measurements.\n(C) Quantification of intra-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n(D) Quantification of inter-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(E) Quantification of intra-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(F) Quantification of inter-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvature (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;)&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated elongated cells, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Surprisingly, despite apparent visual difference, no significant changes were observed in IMM curvedness overall (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Because visual inspection revealed local changes in curvature in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;, Figure 5C-E &lt;&lt;FIG:fig5&gt;&gt;). Within the IBM, we observed a significant decrease in curvedness in fragmented relative to elongated mitochondrial populations in cells treated with vehicle (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Further, we found that Tg treatment significantly increased curvedness in elongated mitochondria, while no significant increase was observed in fragmented mitochondria from Tg-treated cells. This led to a significant increase in IBM curvedness in elongated mitochondria over fragmented mitochondria in Tg-treated cells (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Since the IBM's curvedness is constrained to be near-identical to that of the OMM, this metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length, while ER stress slightly reduces the radius of elongated mitochondria (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;, Supplemental Figure 1).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress.\n(A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principle components of curvature and is used because the surfaces normal vectors do not have canonical sign.\n(B) Quantification of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(C) Quantification of inner boundary membrane (IBM) curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\n(D) Quantification of cristae curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(E) Quantification of cristae junction curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\nIn contrast, larger changes were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress (Figure 5D &lt;&lt;FIG:fig5&gt;&gt;). Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated elongated cells compared to both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria (Figure 5E &lt;&lt;FIG:fig5&gt;&gt;). This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae were regulated, and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (Figure 6A &lt;&lt;FIG:fig6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Figure 6B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Figure 6C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position. Because of this, we quantified the standard deviation of these values for each tomogram in order to assess statistical significance of observed changes (Figure 6D-G &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells (Figure 6D &lt;&lt;FIG:fig6&gt;&gt;). We also identified a significant decrease in variability of the verticality of the crista body in Tg-treated elongated relative to fragmented mitochondria (Figure 6F &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM (Figure 6E &lt;&lt;FIG:fig6&gt;&gt;) or the growth plane (Figure 6G &lt;&lt;FIG:fig6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks\n(A) Surface membrane reconstruction colored by inner membrane compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple;) defining angular measurements, between the surface and the nearest OMM or to the growth plane of the cell.\n(B) Representative membrane surface reconstruction of a rigid and lamellar Tg-treated elongated mitochondrion, colored by angle of IMM relative to OMM.\n(C) Representative membrane surface reconstruction of a less rigidly oriented Tg-treated elongated mitochondrion, colored by angle of IMM relative to the growth plane of the cell.\n(D) Quantification of angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(E) Quantification of angle of junctions relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Violin plot (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(F) Quantification of angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(G) Quantification of angle of junctions relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nElongated and fragmented mitochondrial networks show differences in ER-mitochondrial contacts\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission&lt;&lt;REF:Friedman-2011-334-358&gt;&gt; to autophagy&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;. To investigate whether the remodeling we observed in response to Tg-induced ER stress was driven by alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present, and visualized the contact sites on the surfaces of the OMM (Figure 7A-B &lt;&lt;FIG:fig7&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Figure 7C &lt;&lt;FIG:fig7&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time-point were not driving the altered mitochondrial ultrastructure (Figure 7D &lt;&lt;FIG:fig7&gt;&gt;).\nFigure 7.\nMitochondria-ER contact sites can be localized in three dimensions on the outer membrane.\n(A) Surface membrane reconstruction defining OMM (purple) and ER (transparent) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Thapsigargin (Tg; 500 nM) colored by OMM to ER (OMM-ER) membrane distance.\n(C) Quantification of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-9 cryo-tomograms is shown.\n\nDISCUSSION\nWe describe a correlative cellular cryo-electron tomography workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular physiology conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influences mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;, Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-electron tomography to reveal organellar membrane ultrastructures (Figure 1A-D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semi-automated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high-precision (Figure 1E-H &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B,C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Movie 1). We determined that the screened Poisson reconstruction approach&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; provides a more accurate estimation of the implicit membrane geometry of complex membranes, and can be used to model and quantify complex membranes such as the IMM (Supplementary Figure 2, Supplementary Movie 1).\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions, and quantified mitochondrial membrane parameters such as inter- and intra-mitochondrial membrane distances (Figures 3 &lt;&lt;FIG:fig3&gt;&gt;-4 &lt;&lt;FIG:fig4&gt;&gt;), curvature (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and orientation (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps (Figures 3B &lt;&lt;FIG:fig3&gt;&gt;, 4A,B &lt;&lt;FIG:fig4&gt;&gt;, 5A &lt;&lt;FIG:fig5&gt;&gt;, 6B,C &lt;&lt;FIG:fig6&gt;&gt;). This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions (Figures 3C,D &lt;&lt;FIG:fig3&gt;&gt;, 4C-F &lt;&lt;FIG:fig4&gt;&gt;, 5C-E &lt;&lt;FIG:fig5&gt;&gt;, 6D-G &lt;&lt;FIG:fig6&gt;&gt;).\nWe first used our workflow to quantify the inter-membrane distance between the inner and outer membranes of mitochondria (Figure 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;, &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;, and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;. The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;) as well as to quantify changes in the overall distributions (Figure 3C-D &lt;&lt;FIG:fig3&gt;&gt;). We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion.&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt; The difference in distance, from 13.6 to 12.8 nm between the centers of the membranes, is small, but correcting for the approximately 7 nm thickness of mitochondrial membranes&lt;&lt;REF:Perkins-1997-119-260&gt;&gt; yields a change in intermembrane contact distance from 6.6 to 5.8 nm, or a 15% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as PISD, an IMM protein previously suggested to act on the OMM.&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt; The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbation may impact inter-membrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the TOM complex&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, which associate with the MICOS complex and may impact MICOS complex organization&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;.\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;. We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figure 4 &lt;&lt;FIG:fig4&gt;&gt;), quantified the curvedness of the membranes (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress.&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt; In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase - a mechanism similar to that responsible for ER-stress induced mitochondrial elongation&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvature. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvature, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;, ATP synthase&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;, and Opa1&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function. Our workflow will facilitate future studies on the critical importance of mitochondrial remodeling induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;, which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Figure 6F,G &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; used in this study traces lower curvature membranes more easily than high curvature membranes, leading to limited segmentations of tight, high curvature crista junctions and crista tips. We have mitigated these biases to the best of our ability by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter&lt;&lt;REF:Koning-2018-217-82&gt;&gt; or phase plate&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;, both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations&lt;&lt;REF:Chen-2017-14-983&gt;&gt;, &lt;&lt;REF:Zhou-2020-?-?&gt;&gt;-&lt;&lt;REF:Liu-2021-?-?&gt;&gt;. The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt; and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template matching approaches&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;. These approaches have in the past depended on voxel segmentations, but moving to high quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentations which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well&lt;&lt;REF:Basanta-2020-4-?&gt;&gt;. Membrane meshes have been used very successfully to guide subtomogram averaging in the past&lt;&lt;REF:Navarro-2018-5-?&gt;&gt;, &lt;&lt;REF:Burt-2021-?-?&gt;&gt;, but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and to much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu&lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;, &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, 55-&lt;&lt;REF:Lin-2018-360-?&gt;&gt;. Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nSupporting information\nSupplemental Figures\nSupplemental Movie 1\n\nCONFLICTS OF INTEREST\nThe authors declare that they have no conflict of interest.\n\nMATERIALS AND METHODS\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM Grids\nMouse embryonic fibroblasts with mitochondria labeled GFP (MEFmtGFP)&lt;&lt;REF:Wang-2012-51-9302&gt;&gt; were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 ug/ml, Corning) and UV sterilized R (1/4) Carbon 200-mesh gold electron microscopy (EM) grids (Quantifoil Micro Tools). After 15-18 hours of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group, or fresh media supplemented with Thapsigargin (500nM, Fisher Scientific) in the Tg treatment group. After 8 hours of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific) with manual back-blotting.\n\nCryo-fluorescence microscopy and Mitochondria Network Morphology Scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) were collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em : 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Figure 2a &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13) (Thermo Fisher) for fluorescence guided milling.\n\nFluorescence Guided Milling\nCryo-Focused Ion Beam milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to an SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and based on their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system (GIS) for 3-4 seconds using an automation script&lt;&lt;REF:Barad-?-?-?&gt;&gt;, followed by a third and final layer of platinum sputter was added. Targeted cells were milled manually&lt;&lt;REF:Schaffer-2015-5-?&gt;&gt; or using an automated cryo preparation workflow&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt; both methods used xT software with MAPS (Thermo Fisher). Minimal SEM imaging for monitoring was done at 2keV to ensure thin lamella generation while avoiding radiation damage. A total of 6 treated grids and 8 untreated grids were milled for further tomography analysis.\n\nTilt Series Data Collection\nEM grids containing lamellae were transferred into a 300keV Titan Krios microscope (Thermo Fisher Scientific), equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1e-/A2) at high magnification (53,000x nominal magnification) to localize cellular regions containing mitochondria, which were identified by their distinctive inner and outer mitochondrial membranes. Data was collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (Mastronarde &lt;&lt;REF:???-2007-9-90&gt;&gt; ) with 2 steps between -60 and +60. Data was collected with pixel sizes of 3.11A, 3.35 A, or 3.55 A, and a nominal defocus range between -5 to -10 microm. A subset of data was collected with dose fractionation, with 10 0.1e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2, and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose fractionated tilt series micrograph movies underwent CTF estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series then were aligned using patch tracking in etomo&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;, with 4 times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into 4 times binned tomograms. Tomogram thicknesses ranged from 95 nm to 233 nm.\n\nSemi-automated Segmentation\nAutomated detection and enhancement of mitochondrial membranes was performed using the TomoSegMemTV software package&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;. Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows:\nscale_space -s 3\ndtvoting -s 12\nsurfaceness -m 0.17 -s 0.8 -p 0.8\ndtvoting -w -s 10\nsurfaceness -S -s 0.75 -p 0.75 -l 20\nthresholding -l 0.01 -2 30\nglobal analysis -3 100\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher) for manual curation. Mitochondrial inner and outer membranes and endoplasmic reticulum were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;. First the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, with a reconstruction depth of 9, an interpolation weight of 4, and a minimum number of samples of 1.5. The resulting surface extended beyond the segmented region, so triangles more than 1 voxel away from the point cloud were deleted. In cases where the resulting surface was very complex, the surface was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the 'mrc2xyz.py' (for point cloud generation) and 'make_mesh.py' scripts.\n\nRefinement of surface orientation and estimation of surface curvature using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. This vector voting process also resulted in robust estimation of curvature metrics. Because the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvature metrics. which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the 'calculate_curvature_args.py' script, after file conversion using the 'mask_and_convert_ply.py' script.\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated. For surfaces with small numbers of triangles, this was accomplished using a distance matrix, while calculations on large surfaces, where the distance matrix would take up more than 32GB of system memory, were accomplished using a KD-tree. Both the distance and ID of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the 'surface_to_surface.py' script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those more than 40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nBecause all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph, rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously&lt;&lt;REF:Collado-2019-51-476&gt;&gt;. To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into a shorter and longer distance. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector based neighbor. Those triangles were filtered out before any further quantifications.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and to the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (eq. 2):\nComparisons of orientations between membranes was made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (eq. 3):\nBoth orientation calculations are determined in the 'surface_to_surface.py' script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization toolkit 'vtp'&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt; format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;. Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area.&lt;&lt;REF:???-2007-9-90&gt;&gt; Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests&lt;&lt;REF:Mann-1947-18-50&gt;&gt; on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram, rather than the histogram peak.\n\nCode Availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Scripts are licensed under the GNU Public License version 3 (GPLv3).'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''INTRODUCTION\nCellular cryo-electron tomography (cryo-ET) is a powerful technique that enables high-resolution, three-dimensional (3D) views of protein and organellar structure in unfixed and fully hydrated conditions&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;. By combining cryo-focused ion beam (cryo-FIB) milling with automated data collection schemes&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt;, &lt;&lt;REF:Schorb-2019-16-471&gt;&gt;, it has become routine to collect large datasets of thin, cellular three-dimensional reconstructions (tomograms) teeming with pristinely preserved proteins and organelles. Even as tremendous progress has been made in developing new algorithmic subtomogram averaging approaches to solve protein structures in cells&lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;-&lt;&lt;REF:Chen-2019-16-1161&gt;&gt;, it remains very challenging to quantitatively describe complex organellar membrane architectures (hereafter referred to as membrane ultrastructure) from cryo-tomograms. Organellar membrane remodeling plays a pivotal role in the cell's ability to adapt and respond to various changes in the physiological state of the cell (e.g., of cellular stress), and is mediated through changes in the molecular interactions of proteins that reside within membrane compartments&lt;&lt;REF:Jiang-2017-7-?&gt;&gt;, &lt;&lt;REF:Schwarz-2016-73-79&gt;&gt;. As such, an improved ability to quantify organellar architectures from cryo-tomograms would both enable contextualization of distinct protein structures revealed by subtomogram averaging and reveal subtle, yet functionally relevant, changes in organellar ultrastructures associated with distinct cellular physiologies.\nDespite the inherent 3D nature of cryo-tomograms, the two-dimensional (2D) virtual slices of cryo-tomograms are often used to quantify membrane ultrastructural parameters such as inter- and intra-membrane distances through manual designation of membrane boundaries, by locating distinct points along the membrane&lt;&lt;REF:Khanna-2021-10-?&gt;&gt;-&lt;&lt;REF:Mageswaran-2021-7-?&gt;&gt;. Although this can serve as a proxy to quantify 3D ultrastructure, this analysis is time-intensive and prone to both user-bias and inaccuracies in the precise location of membranes between 2D slices. Furthermore, the manual nature of this approach severely limits its applicability to perform ultrastructural quantification at a throughput and sample size sufficient to reveal substantial mechanistic insight. Towards a more accurate 3D representation, manual or automated&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; voxel-based segmentation can be used to assign and label individual voxels within cryo-tomograms to specific cellular membranes&lt;&lt;REF:Chen-2017-14-983&gt;&gt;. Although qualitatively informative, voxel-based segmentation methods do not encode for parameters defining membrane geometry (i.e. connectivity), a property essential for robust quantifications of ultrastructural features like membrane curvature and spacing.\nAn alternative to using voxel segmentations to represent membranes is representing them as a meshwork of triangles. Triangle mesh surfaces represent the connectivity and implicit geometry of the membrane itself, enabling direct measurement of metrics like orientation and curvature. A recently developed approach enables more accurate and robust estimation of each triangle's geometry, improving quantification of local curvature as well as the spatial relationship between individual triangles (i.e. membrane segments) both within the same (intra-) and between distinct (inter-) membrane compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Despite these major advances, surface reconstructions using segmentations of membranes themselves (i.e. boundary segmentations) were not reliable, due to the incomplete (open) nature of the membrane segmentations limiting the effectiveness of the reconstruction algorithm&lt;&lt;REF:Hoppe-1992-?-71&gt;&gt;. In cryo-electron tomography, even perfect segmentations will never show complete and closed membranes, due to the substantial missing wedge&lt;&lt;REF:Lučič-2013-202-407&gt;&gt; making horizontal membranes invisible. To address these issues, quantifications using this new approach depended on using voxel segmentations that \"fill\" the entire internal volume encompassed within a membrane (i.e. compartment segmentations), which are not amenable to automation. Manual segmentation of filled membrane compartments is feasible for small and simple compartments, but the time intensive process severely limits throughput and, thus, the ability to aggregate ultrastructural quantifications across large tomographic datasets. The low throughput is further exacerbated in the context of complex, highly variable organellar membranes, which can be very challenging to accurately fill in. Thus, new strategies are required to improve throughput, automation, and quantification of membrane ultrastructures for application to cryo-electron tomography.\nMitochondria are an ideal target for the development of this approach. Mitochondria are highly pleomorphic organelles that function simultaneously as an interconnected network population and as discrete organellar units involved in energy production, ion homeostasis, cellular stress pathway integration, and innate immune signaling&lt;&lt;REF:Schirrmacher-2020-8-?&gt;&gt;-&lt;&lt;REF:Wai-2016-27-105&gt;&gt;. Their ability to perform these essential eukaryotic functions fundamentally depends on their dynamic remodeling both within the entire cellular mitochondrial network population (hereafter referred to as bulk mitochondrial morphology) and within their distinct outer and inner mitochondrial membranes (hereafter referred to as mitochondrial membrane ultrastructure)&lt;&lt;REF:Chan-2020-15-235&gt;&gt;. Changes to bulk mitochondrial morphology are mediated through the opposing processes of mitochondrial fusion and fission, and imbalances in these processes can promote interconnected (i.e. elongated) or disconnected (i.e. fragmented) networks that regulate pro-survival or pro-apoptotic mitochondrial functions&lt;&lt;REF:Sabouny-2020-45-564&gt;&gt;, &lt;&lt;REF:Rambold-2011-108-10190&gt;&gt;. Within individual mitochondria, ultrastructures of both the outer and inner mitochondrial membranes (OMM and IMM, respectively) and the associations between these two membranes can exhibit dynamic remodeling to modulate metabolic and signaling functions&lt;&lt;REF:Nielsen-2017-595-2839&gt;&gt;.\nIn particular, the IMM contains functional membrane folds, called cristae, whose ultrastructure is intimately linked to nearly all aspects of mitochondrial function. The architecture of the cristae is maintained through the activity and localization of several proteins (termed \"shaping\" proteins), including components involved in cellular respiration such as ATP synthase, whose dimerization at the matrix-associated cristae \"tip\" regions induces membrane curvature required for efficient mitochondrial respiration,&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;-&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;. Cristae shape is also maintained through specialized regions called cristae junctions, formed by the interaction of mitochondrial contact site and cristae organizing system (MICOS) and Optic Atrophy 1 (OPA1), which act as dynamic molecular staples to compartmentalize the distinct internal cristae environments and associate IMM cristae with the OMM&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;-&lt;&lt;REF:Zhang-2008-283-3476&gt;&gt;. Alterations to the expression, localization, or activity of these shaping proteins disrupt mitochondrial respiration and can promote the induction of apoptotic signaling through the release of cytochrome C&lt;&lt;REF:Colina-Tenorio-2020-287-645&gt;&gt;, &lt;&lt;REF:Frezza-2006-126-177&gt;&gt;. It is therefore important to understand the relationship between mitochondrial network morphology and membrane ultrastructure, and how they are altered in response to cellular perturbations.\nHere, we sought to develop a correlative cryo-electron tomography workflow that enables quantitative analysis of complex membrane ultrastructures across distinct mitochondrial network morphologies and cellular physiologies with throughputs sufficient for statistical hypothesis testing. Specifically, we combined cryo-fluorescence microscopy with cellular cryo-ET imaging to correlate distinct mitochondrial network morphologies (e.g., elongated versus fragmented) to mitochondrial membrane ultrastructure. We developed and implemented a robust, semi-automated workflow&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; to segment cellular membranes present in cryo-tomograms, and applied a screened Poisson reconstruction strategy&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; to convert membrane voxel segmentations into implicit surface meshes that model complex cellular membrane geometries. To complement our surface generation workflow, we developed the surface morphometrics toolkit to measure several parameters of membrane shape including inter- and intra-membrane spacing, curvedness, and orientation across several hundred square microns of mitochondrial membranes. We applied this workflow to cells experiencing acute levels of endoplasmic reticulum (ER) stress, a condition known to promote adaptive remodeling of mitochondrial networks&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;, to explore how these changes influence membrane ultrastructure across distinct network morphologies. Our workflow for the first time allows us to distinguish statistically significant alterations in 3D ultrastructure in highly variable mitochondrial inner membranes.\n\nRESULTS\n\nCorrelative cryo fluorescence microscopy and electron tomography enables association of bulk mitochondrial morphology with local membrane ultrastructure\nWe set out to understand the connection between the bulk morphology of the mitochondrial network within a single cell and the local membrane ultrastructure of the mitochondrial membranes within that same cell. We used cryo-fluorescence microscopy to image vitrified mouse embryonic fibroblasts whose mitochondria were labeled with a mitochondria-targeted GFP (MEFmtGFP), such that the morphology of each cell's mitochondrial network could be readily assessed (Figure 1A-C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We categorized cells for mitochondrial network morphology by blinded manual classification based on the fluorescence microscopy, then targeted cells with either fragmented or elongated mitochondrial networks for cryo-focused ion beam (cryo-FIB) milling to prepare thin (~100-200 nm) lamellae (Figure 1D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). We collected tilt series of the mitochondria within these lamellae to directly correlate the cell's bulk mitochondrial morphology (Figure 1C &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;) to the ultrastructure of mitochondrial membranes (Figure 1 &lt;&lt;FIG:fig1&gt;&gt; F-H, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). To further probe the connection between morphology and ultrastructure, we treated MEFmtGFP cells with Thapsigargin (Tg), a small molecule that induces ER stress through inhibition of the sacro/ER Ca2+ ATPase (SERCA) and has been previously reported to promote mitochondrial elongation downstream of the ER stress-responsive kinase PERK through a process termed stress-induced mitochondrial hyperfusion (SIMH)&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. We applied our correlative approach to identify and target specific Tg-treated MEFmtGFP cells with either elongated or fragmented mitochondrial network morphologies for cryo-FIB milling and cryo-ET data acquisition and reconstruction. Visual inspection of the mitochondrial membranes within tomograms revealed variability in IMM architecture between mitochondria from different network morphologies and treatment groups, but some overall trends were apparent in elongated and fragmented mitochondrial populations in response to Tg-induced ER stress (Supplementary Figure 1). Notably, Tg treatment induced a shift towards regularly spaced disc-shaped (lamellar) cristae in the elongated population and towards a swollen tubular crista structure in the fragmented population. Due to the high degree of pleomorphism, with mixed populations in all conditions, we next established a workflow for ultrastructural quantification to determine the significance of these apparent changes.\nFigure 1.\nCorrelative cellular cryo-electron tomography workflow for robust quantitative analysis of membrane ultrastructures between distinct mitochondrial network morphologies and treatment conditions.\n(A) MEFmtGFP cells were cultured on transmission electron microscopy grids (black mesh circle) and treated with vehicle or thapsigargin (Tg, 500 nM) for 8 hours prior to vitrification via plunge freezing in ethane/propane mixture.\n(B) Vitrified MEFmtGFP cells were imaged using cryo-fluorescence (cryo-FM).\n(C) Cryo-FM facilitated the identification of single cells with distinct elongated or fragmented bulk mitochondrial network morphologies.\n(D) These cells were targeted for cryo-focused ion beam (cryo-FIB) milling to generate thin sections (lamellae).\n(E) Lamella were imaged using standard cryo-ET imaging parameters to generate tilt series that were further reconstructed to generate three-dimensional (3D) cryo-tomograms.\n(F) Voxel segmentations labeling distinct cellular membranes were generated using a semi-automated approach&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; prior to conversion to point clouds. Next, a normal-oriented vector is estimated for each point&lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;. Normal-oriented point clouds were used to generate surface meshes using a novel application of a screened Poisson reconstruction method followed by masking to the original voxel segmentation&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, &lt;&lt;REF:Cignoni-2008-2008-129&gt;&gt;.\n(G) Surface meshes model the implicit 3D ultrastructure of distinct organellar membranes. Surface coloring: inner mitochondrial membrane (IMM), purple; outer mitochondrial membrane (OMM), blue; endoplasmic reticulum (ER), teal.\n(H) Implicit geometries encoded within surface meshes are then used to perform 3D morphometrics to quantify several parameters that define membrane ultrastructure across mitochondria from distinct morphological and treatment conditions.\nFigure 2.\nApplication of correlative workflow to visualize cellular membrane ultrastructures in elongated and fragmented mitochondrial networks in MEFmtGFP cells.\n(A) Identification by cryo-FM of elongated (top; cyan outline) and fragmented (bottom; orange outline) mitochondrial morphologies in MEFmtGFP cells. Cryo-FM images of bulk mitochondrial morphology are then used for targeted cryo-FIB milling to generate thin lamellae of ~150-200 nm. MEFmtGFP cell periphery is outlined in dashed white line.\n(B) Virtual slides of tomograms of MEFmtGFP cells containing elongated (top) and fragmented (bottom) mitochondria are traced using the automated tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, followed by manual cleanup using AMIRA software to generate 3D voxel segmentations. 3D voxel segmentations are then converted into implicit surface meshes using the screened Poisson mesh reconstruction. Scale bars = 100 nm. Surface coloring: IMM in pink, OMM in purple, ER in blue.\n(C) Detailed views of voxel (green) and triangle surface mesh (IMM: pink, OMM: purple, ER: blue) show the improved smoothness and hole-filling of surfaces generated with the screened Poisson mesh approach. Zoomed insets are rotated backwards 20degrees to highlight features. Holes are outlined in red on the voxel segmentation.\n\nDevelopment of a framework to automate quantification of ultrastructural features of cellular membranes\nIn order to identify changes in different mitochondrial populations with and without induction of ER stress, we developed a semi-automated approach to quantify membrane features in three dimensions without the need for bias-prone manual measurements. We leveraged recent work&lt;&lt;REF:Salfer-2020-16-?&gt;&gt; that established a robust approach for estimating curvature in membranes and developed a framework for additional quantifications using triangular mesh models of membranes. However, the generation of high quality surface meshes has previously required labor-intensive manual segmentation approaches. These include manual contouring of membranes&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;, &lt;&lt;REF:Navarro-2018-5-?&gt;&gt; and manual inpainting of entire compartments&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;, both of which are especially challenging with the complex and highly curved IMM and ER membrane. With these methods, it would be impractical to analyze a sufficient number of tomograms to overcome the variability between mitochondria observed in these distinct mitochondrial populations (elongated versus fragmented) and upon treatment (vehicle versus Tg). Instead, we developed a semi-automated workflow to enable analysis at higher throughput (Figure 1F &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We used a segmentation strategy that takes advantage of the tomosegmemtv program&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;, which identifies and enhances regions of the tomogram with membrane-like voxel values. This allowed manual classification of membrane identity into mitochondrial IMM and OMM and ER membrane, as well as manual cleanup of individual membrane segmentations using AMIRA software (Thermo Fisher Scientific). With this strategy, we were able to segment 32 tomograms containing mitochondria, divided between the elongated and fragmented bulk morphology populations and the two treatment groups (Figure 2 &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 1). In order to prepare high quality surface meshes appropriate for quantification, we established a fully automated surface reconstruction pipeline using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; (Figure 1G &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;, Supplementary figure 1). This allowed reconstruction of much higher quality membrane surface as compared to previously developed approaches for mesh reconstruction from membrane segmentations (Figure 2C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Figure 2, Supplementary Movie 1). These surfaces were then passed into the 3D surface morphometrics workflow, generating quantifications such as inter- and intra-membrane distances, membrane curvature, and relative membrane orientations (Figure 1H &lt;&lt;FIG:fig1&gt;&gt;). This 3D surface morphometrics toolkit is available at https://github.com/grotjahnlab/surface_morphometrics.\n\nQuantification of IMM-OMM spacing in mitochondria identifies ultrastructural differences in elongated and fragmented mitochondrial networks\nThe most distinctive feature of mitochondria are the juxtaposed inner and outer mitochondrial membranes. Cryo-electron tomography is uniquely able to reveal the details of the architecture of these two membranes in 3D, and we sought out to use our tomograms to precisely measure their spacing. We measured the distance to the closest triangle on the IMM mesh for each triangle in the OMM mesh (Figure 3A &lt;&lt;FIG:fig3&gt;&gt;). We then visualized the spatial distribution of these distances directly on the generated surface mesh reconstructions of the OMM of individual mitochondria visible within tomograms (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). The resulting surface maps showed regions marked by the largest OMM-IMM distance representing the distribution of the crista junctions, but also revealed general increases in distance in the fragmented over elongated mitochondria in both vehicle- and Tg-treated conditions (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;). In order to determine how consistent these effects were across all of the segmented tomograms, we plotted a histogram of the combined distribution of distances for all surface mesh triangles within each condition (Figure 3C &lt;&lt;FIG:fig3&gt;&gt;). This revealed an overall increase in inter-membrane distance in fragmented mitochondria. Furthermore, we observed a subtle decrease in the peak OMM-IMM distance after Tg-induced ER stress in elongated mitochondria, contrasted by a Tg-dependent increase in peak and variability of distances in fragmented populations. We then assessed the peaks of the distance distribution for each tomogram as independent observations and identified statistically significant increases in spacing in fragmented vs elongated cells, by 1.7 nm in the vehicle-treated condition and by 3.9 nm in the Tg-treated condition. Additionally, we confirmed that the 0.8 nm decrease in distance of the elongated mitochondria after Tg treatment was statistically significant, while the increase in spacing in fragmented cells after Tg treatment was not significant (Figure 3D &lt;&lt;FIG:fig3&gt;&gt;), demonstrating the power of our approach to quantify and test the significance differences across distinct bulk mitochondrial network morphologies.\nFigure 3.\nInter-mitochondrial membrane distance is dependent on mitochondrial network morphology and presence or absence of ER stress.\n(A) Surface membrane reconstruction defining outer mitochondrial membrane (OMM, purple) and inner mitochondrial membrane (IMM, pink) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The OMM surface is colored by outer-to-inner (OMM-IMM) membrane distance, and the IMM surface is shown in transparent gray. Red arrows indicate regions on OMM with large OMM-IMM distances that correspond to cristae junctions.\n(C) Quantification of OMM-IMM distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Quantification of peak histogram values from each tomogram within the indicated treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n\nMitochondrial cristae and junction spacing in mitochondria are differentially impacted by Tg in elongated and fragmented mitochondrial networks\nThe IMM is divided between the inner boundary membrane (IBM), which is in close proximity to the OMM, and the invaginated mitochondrial cristae (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;), which are maintained as separate functional compartments by the organization of the MICOS complex and the protein OPA1 at crista junctions&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, &lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;. We sought to understand how the organization of mitochondrial cristae and their junctions differed between fragmented and elongated mitochondria populations in the presence and absence of the ER stressor, Tg. We classified the IMM into three compartments (IBM, crista junctions, and crista bodies) based on their distance from the OMM such that quantifications could be performed based on the functional components of the membrane (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;). We then measured the distance between the membrane segments on either side of the crista bodies and junctions (intra-crista and intra-junction distances, respectively), as well as the distance between adjacent crista bodies and junctions (inter-crista and inter-junction distances, respectively) (Figure 4B &lt;&lt;FIG:fig4&gt;&gt;). We quantified these effects as combined histograms showing the change in different populations upon treatment with Tg, and as violin plots of the histogram peaks for each tomogram (Figure 4C-F &lt;&lt;FIG:fig4&gt;&gt;). Based on these distributions, we identified statistically significant but opposing changes in intra-crista body spacing between the elongated and fragmented morphologies induced by Tg treatment. In elongated mitochondria, crista spacing was reduced by 4.5 nm upon Tg treatment, while in fragmented mitochondria the spacing increased by 3.2 nm upon Tg treatment (Figure 4C &lt;&lt;FIG:fig4&gt;&gt;). However, we did not observe significant changes in inter-cristae distance, intra-junction distance, or inter-junction distance (Figure 4D-F &lt;&lt;FIG:fig4&gt;&gt;), with the exception of a 9.4 nm increase in intra-junction spacing in the Tg-treated condition of elongated mitochondria relative to fragmented mitochondria (Figure 4E &lt;&lt;FIG:fig4&gt;&gt;), suggesting that ER stress promotes relaxation of the junction in fragmented mitochondria.\nFigure 4.\nSpacing within and between mitochondrial cristae is differentially altered by Tg-induced ER stress in fragmented and elongated populations\n(A) Representative membrane surface reconstructions of elongated (left) and fragmented (right) mitochondria in MEFmtGFP cells treated with thapsigargin (Tg; 500 nM, 8h) showing subdivision of IMM compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple) as defined by OMM-IMM distance. The OMM surface is represented as a transparent gray mesh.\n(B) Enlarged inset of boxed region in elongated example from (A) showing surface membrane reconstruction colored by inner membrane compartments defining intra- and inter-crista and junction measurements.\n(C) Quantification of intra-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05; **p&lt;0.01, ***p&lt;0.005.\n(D) Quantification of inter-crista membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(E) Quantification of intra-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(F) Quantification of inter-junction membrane distances of elongated (left) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (right) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nIMM curvedness is differentially sensitive to Tg treatment in elongated and fragmented mitochondrial networks\nWe next quantified the membrane curvedness locally at every mesh triangle position on each IMM surface to identify regions of local high and low curvature (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;)&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. Visual inspection of these surfaces suggested notable decreases in curvedness in the flat and lamellar cristae observed in the Tg-treated elongated cells, while in cells with fragmented mitochondria, Tg appeared to increase curvedness in the swollen and tubular cristae (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;). To understand the degree to which these effects differed across populations and not just within individual mitochondria, we quantified the total curvedness of the IMM across different mitochondrial network morphologies and treatment conditions (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Surprisingly, despite apparent visual difference, no significant changes were observed in IMM curvedness overall (Figure 5B &lt;&lt;FIG:fig5&gt;&gt;). Because visual inspection revealed local changes in curvature in the cristae and junctions, rather than uniformly across the IMM, we broke down these quantifications into the IMM's functional subcompartments (Figure 4A &lt;&lt;FIG:fig4&gt;&gt;, Figure 5C-E &lt;&lt;FIG:fig5&gt;&gt;). Within the IBM, we observed a significant decrease in curvedness in fragmented relative to elongated mitochondrial populations in cells treated with vehicle (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Further, we found that Tg treatment significantly increased curvedness in elongated mitochondria, while no significant increase was observed in fragmented mitochondria from Tg-treated cells. This led to a significant increase in IBM curvedness in elongated mitochondria over fragmented mitochondria in Tg-treated cells (Figure 5C &lt;&lt;FIG:fig5&gt;&gt;). Since the IBM's curvedness is constrained to be near-identical to that of the OMM, this metric likely reports primarily on the radius of the mitochondria as a whole, suggesting that fragmented mitochondria are larger in radius despite their apparent shorter overall length, while ER stress slightly reduces the radius of elongated mitochondria (Figure 5A &lt;&lt;FIG:fig5&gt;&gt;, Supplemental Figure 1).\nFigure 5.\nThe curvedness of mitochondrial cristae and crista junctions are dependent both on mitochondrial network morphology and Tg-induced ER stress.\n(A) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and thapsigargin (Tg; 500 nM, 8h). The IMM surface is colored by membrane curvedness and the OMM surface is represented as a transparent gray mesh. Curvedness is an unsigned combination of the two principle components of curvature and is used because the surfaces normal vectors do not have canonical sign.\n(B) Quantification of total IMM curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(C) Quantification of inner boundary membrane (IBM) curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\n(D) Quantification of cristae curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05.\n(E) Quantification of cristae junction curvedness of elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. *p&lt;0.05, ***p&lt;0.005.\nIn contrast, larger changes were observed in cristae and crista junctions. The cristae of fragmented mitochondria were less curved than those of elongated mitochondria in the absence of ER stress (Figure 5D &lt;&lt;FIG:fig5&gt;&gt;). Interestingly, Tg treatment induced significant but opposing changes in fragmented and elongated mitochondria. In elongated mitochondria, curvedness of cristae decreased upon Tg treatment, while in fragmented mitochondria, curvedness increased upon Tg treatment. In crista junctions, the most notable difference was an increase in curvedness in Tg-treated elongated cells compared to both the vehicle-treated elongated mitochondria and the Tg-treated fragmented mitochondria (Figure 5E &lt;&lt;FIG:fig5&gt;&gt;). This reflects and confirms the visual observation of crista ordering upon Tg treatment in elongated mitochondria in the tomograms.\n\nMembrane orientations in elongated and fragmented mitochondria differ upon Tg treatment\nWe next sought to understand to what degree the orientation of mitochondrial cristae were regulated, and how they changed in different mitochondrial morphologies as well as in response to Tg-induced ER stress. We took advantage of the 3D nature of our surface reconstructions to measure the orientation of each triangle in the surface mesh relative to the nearest point on the OMM, as well as to the cell's growth plane (Figure 6A &lt;&lt;FIG:fig6&gt;&gt;). Visual inspection revealed that the highly organized lamellar phenotype that was typical of elongated mitochondria in Tg treated cells was almost entirely held orthogonal to the OMM, forming the distinctive ladder-like phenotype (Figure 6B &lt;&lt;FIG:fig6&gt;&gt;). Similarly, most but not all of these lamellar cristae were remarkably vertical relative to the growth plane of the cell (Figure 6C &lt;&lt;FIG:fig6&gt;&gt;). We plotted the angles of the cristae and junctions relative to the OMM and the growth plane, revealing changes primarily in the variance of the angles rather than the peak position. Because of this, we quantified the standard deviation of these values for each tomogram in order to assess statistical significance of observed changes (Figure 6D-G &lt;&lt;FIG:fig6&gt;&gt;). This revealed a significant decrease in the variability of the angle of the crista bodies relative to the OMM for elongated mitochondria in Tg-treated cells (Figure 6D &lt;&lt;FIG:fig6&gt;&gt;). We also identified a significant decrease in variability of the verticality of the crista body in Tg-treated elongated relative to fragmented mitochondria (Figure 6F &lt;&lt;FIG:fig6&gt;&gt;). In both cases, these differences corresponded to increased control over the orientation of the ladder-like lamellar cristae observed in the elongated Tg-treated condition. Despite the changes in crista orientation, no significant changes were observed in the angle of junctions relative to the OMM (Figure 6E &lt;&lt;FIG:fig6&gt;&gt;) or the growth plane (Figure 6G &lt;&lt;FIG:fig6&gt;&gt;).\nFigure 6.\nTg-induced ER stress drives changes in crista orientation in elongated mitochondrial networks\n(A) Surface membrane reconstruction colored by inner membrane compartments (inner membrane boundary (IBM), tan; junction, cyan; crista, purple;) defining angular measurements, between the surface and the nearest OMM or to the growth plane of the cell.\n(B) Representative membrane surface reconstruction of a rigid and lamellar Tg-treated elongated mitochondrion, colored by angle of IMM relative to OMM.\n(C) Representative membrane surface reconstruction of a less rigidly oriented Tg-treated elongated mitochondrion, colored by angle of IMM relative to the growth plane of the cell.\n(D) Quantification of angle of cristae relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(E) Quantification of angle of junctions relative to OMM in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Violin plot (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n(F) Quantification of angle of cristae relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown. p-values from Mann-Whitney U test are indicated. ***p&lt;0.005.\n(G) Quantification of angle of junctions relative to growth plane in elongated (top) and fragmented (middle) mitochondria in MEFmtGFP cells treated with vehicle and Tg. Graph (bottom) showing distribution of standard deviations within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-12 cryo-tomograms is shown.\n\nElongated and fragmented mitochondrial networks show differences in ER-mitochondrial contacts\nMitochondria-ER contact sites are sites of significant exchange of lipids and proteins have been previously associated with a wide variety of critical regulatory functions for mitochondria, ranging from mitochondrial fission&lt;&lt;REF:Friedman-2011-334-358&gt;&gt; to autophagy&lt;&lt;REF:Bosc-2020-11-4056&gt;&gt;. To investigate whether the remodeling we observed in response to Tg-induced ER stress was driven by alterations to the ultrastructure of these contact sites, we leveraged our 3D morphometrics workflow to quantify changes in ER-mitochondrial contacts in elongated vs fragmented mitochondrial networks as well as in ER stress induced by Tg. Distances between different organelles can be as readily calculated as distances within mitochondria, with the only additional challenge that not all tomograms contained ER, and not all ER present was interacting with mitochondria. We measured the distance between the OMM and the nearest ER in all tomograms where the ER was present, and visualized the contact sites on the surfaces of the OMM (Figure 7A-B &lt;&lt;FIG:fig7&gt;&gt;). This showed detailed mapping of these functionally critical interactions in three dimensions. We also quantified the distribution of ER-OMM distances as a combined histogram (Figure 7C &lt;&lt;FIG:fig7&gt;&gt;). None of the conditions were statistically significant, suggesting that changes in ER-mitochondrial contact distance or area at the imaged time-point were not driving the altered mitochondrial ultrastructure (Figure 7D &lt;&lt;FIG:fig7&gt;&gt;).\nFigure 7.\nMitochondria-ER contact sites can be localized in three dimensions on the outer membrane.\n(A) Surface membrane reconstruction defining OMM (purple) and ER (transparent) distance measurement.\n(B) Representative membrane surface reconstructions of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Thapsigargin (Tg; 500 nM) colored by OMM to ER (OMM-ER) membrane distance.\n(C) Quantification of OMM-ER distances of elongated (top) and fragmented (bottom) mitochondria in MEFmtGFP cells treated with vehicle and Tg.\n(D) Graph showing peak histogram values from each tomogram within each corresponding treatment and mitochondrial morphology class. Individual quantification from n=5-9 cryo-tomograms is shown.\n\nDISCUSSION\nWe describe a correlative cellular cryo-electron tomography workflow for quantifying differences in membrane ultrastructure between distinct mitochondrial network morphologies and cellular physiology conditions. Our approach takes advantage of the distinct capabilities of multiple imaging modalities (fluorescence microscopy and cryo-ET) to correlate single-cell, organellar network morphologies to single-organellar membrane ultrastructures in varying cellular conditions. To our knowledge, this is the first application of a correlative approach that directly assesses how distinct organellar morphologies (e.g., elongated versus fragmented mitochondria) and cellular stress states (e.g., presence or absence of Tg-induced ER stress) influences mitochondrial membrane remodeling at nanometer-scale resolution. Our workflow can be divided into two functional steps: (1) single-cell targeting and (2) 3D surface morphometrics analysis (Figure 1 &lt;&lt;FIG:fig1&gt;&gt;, Figure 2 &lt;&lt;FIG:fig2&gt;&gt;). The first step involves identifying cells that exhibit a particular mitochondrial network morphology by cryo-fluorescence microscopy, targeting these cells for sample thinning to generate cellular lamellae by cryo-FIB milling, and imaging these cellular lamellae by cryo-electron tomography to reveal organellar membrane ultrastructures (Figure 1A-D &lt;&lt;FIG:fig1&gt;&gt;, Figure 2A &lt;&lt;FIG:fig2&gt;&gt;). The second step involves adapting semi-automated segmentation and surface mesh reconstruction algorithms to cryo-ET data, thereby enabling unbiased quantification of several membrane parameters at high-precision (Figure 1E-H &lt;&lt;FIG:fig1&gt;&gt;, Figure 2B,C &lt;&lt;FIG:fig2&gt;&gt;, Supplementary Movie 1). We determined that the screened Poisson reconstruction approach&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt; provides a more accurate estimation of the implicit membrane geometry of complex membranes, and can be used to model and quantify complex membranes such as the IMM (Supplementary Figure 2, Supplementary Movie 1).\nTo demonstrate proof-of-concept of our correlative workflow, we asked whether bulk changes in mitochondrial network morphologies were associated with ultrastructural remodeling of mitochondrial membranes, both in the presence and absence of ER stress. We identified cells with distinct mitochondrial morphologies (e.g., fragmented versus elongated) in both treatment conditions, and quantified mitochondrial membrane parameters such as inter- and intra-mitochondrial membrane distances (Figures 3 &lt;&lt;FIG:fig3&gt;&gt;-4 &lt;&lt;FIG:fig4&gt;&gt;), curvature (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and orientation (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We demonstrate that our approach can be used to identify changes to membrane ultrastructure in two ways. First, we can evaluate ultrastructural changes on an individual mitochondrion basis by overlaying values directly on individual triangles within our membrane surface maps (Figures 3B &lt;&lt;FIG:fig3&gt;&gt;, 4A,B &lt;&lt;FIG:fig4&gt;&gt;, 5A &lt;&lt;FIG:fig5&gt;&gt;, 6B,C &lt;&lt;FIG:fig6&gt;&gt;). This enables contextualization of membrane quantifications based on the specific membrane microenvironment. Second, we combine ultrastructural quantifications across many mitochondria within specific morphological or cellular physiological groups, thereby enabling us to establish statistical criteria for identifying trends even in the context of mitochondrial pleomorphism across all conditions (Figures 3C,D &lt;&lt;FIG:fig3&gt;&gt;, 4C-F &lt;&lt;FIG:fig4&gt;&gt;, 5C-E &lt;&lt;FIG:fig5&gt;&gt;, 6D-G &lt;&lt;FIG:fig6&gt;&gt;).\nWe first used our workflow to quantify the inter-membrane distance between the inner and outer membranes of mitochondria (Figure 3 &lt;&lt;FIG:fig3&gt;&gt;). These two membranes are mechanically linked by interactions between proteins on the two membranes&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, &lt;&lt;REF:Callegari-2019-431-2835&gt;&gt;, &lt;&lt;REF:Bauer-2000-10-25&gt;&gt;, and in some instances, there are reports of inner membrane-bound proteins acting in trans on the outer membrane&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt;. The uniquely high resolution and 3D information afforded by cryo-ET allowed us to visualize the fluctuations of distances between these membranes (Figure 3B &lt;&lt;FIG:fig3&gt;&gt;) as well as to quantify changes in the overall distributions (Figure 3C-D &lt;&lt;FIG:fig3&gt;&gt;). We identified significant increases in the IMM-OMM spacing of mitochondria in fragmented networks over those in elongated networks. This increase in spacing may reduce the effectiveness of protein interactions across the two membranes, thereby increasing isolation of the IMM from external signals. In contrast, within the elongated population, Tg treatment reduced IMM-OMM spacing, suggesting tighter interactions that may be critical for the protective function of stress-induced mitochondrial hyperfusion.&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt; The difference in distance, from 13.6 to 12.8 nm between the centers of the membranes, is small, but correcting for the approximately 7 nm thickness of mitochondrial membranes&lt;&lt;REF:Perkins-1997-119-260&gt;&gt; yields a change in intermembrane contact distance from 6.6 to 5.8 nm, or a 15% reduction. This could have a significant effect on the potential for transmembrane activity of proteins, such as PISD, an IMM protein previously suggested to act on the OMM.&lt;&lt;REF:Aaltonen-2016-213-525&gt;&gt; The ability to consistently and robustly resolve such small but physiologically significant differences in intermembrane spacing opens new avenues for understanding how different perturbation may impact inter-membrane communication in mitochondria. The spatially resolved high OMM-IMM distance proximal to crista junctions may also eventually prove to be an effective automated localization signal for particle picking and guided subtomogram averaging of proteins, such as the TOM complex&lt;&lt;REF:Bohnert-2012-23-3948&gt;&gt;, which associate with the MICOS complex and may impact MICOS complex organization&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;.\nMitochondrial inner membranes are functionally divided between the IBM and the mitochondrial cristae, with the cristae junctions that divide the two components organized by the MICOS complex&lt;&lt;REF:Wollweber-2017-1864-1481&gt;&gt;. We were able to automatically classify our inner membrane surface into the IBM, cristae, and junctions based on distance from the OMM, and using this classification were able to quantify specific architectural changes in cristae and junctions in different mitochondrial populations in the presence or absence of Tg-induced ER stress. We measured the membrane spacing within and between cristae and junctions (Figure 4 &lt;&lt;FIG:fig4&gt;&gt;), quantified the curvedness of the membranes (Figure 5 &lt;&lt;FIG:fig5&gt;&gt;), and measured their orientation relative to the OMM and the growth plane (Figure 6 &lt;&lt;FIG:fig6&gt;&gt;). We observed consistent but opposing changes between fragmented and elongated mitochondrial populations when exposed to Tg. In elongated populations of cells treated with Tg, cristae became narrower, less curved, and more vertical lamellar structures that were highly orthogonal to the OMM. This is consistent with previous reports of ultrastructural changes induced in brown adipocytes subjected to cold stress.&lt;&lt;REF:Latorre-Muro-2021-33-598&gt;&gt; In these conditions, modulations to the MICOS subunit Mic 19 remodels cristae architecture through the activity of the ER-stress responsive PERK kinase - a mechanism similar to that responsible for ER-stress induced mitochondrial elongation&lt;&lt;REF:Lebeau-2018-22-2827&gt;&gt;. While future work is needed to determine whether cristae remodeling and mitochondrial elongation are regulated by the same mechanism, our results highlight that these two changes in mitochondrial shape are correlated in response to Tg-induced ER stress.\nIn contrast, cristae in fragmented populations became more swollen upon induction of ER stress, as measured by increases in intra-crista membrane spacing and crista curvature. Of particular interest, our workflow will enable local analysis of individual cristae on the basis of their spacing, curvature, and orientation to isolate different functional microenvironments that can be studied for differences in protein composition and conformation. Crista ultrastructure is largely controlled by MICOS&lt;&lt;REF:Harner-2011-30-4356&gt;&gt;, ATP synthase&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;, and Opa1&lt;&lt;REF:Hu-2020-11-?&gt;&gt;, and we predict that the changes observed are driven by changes in composition, localization, or conformation of these proteins. Cryo-ET offers the unique combination of cellular and structural detail to pursue studies of how these proteins differ in distribution and conformation in different cristae even within the same mitochondrion, and how that is reflected in differences in mitochondrial architecture and function. Our workflow will facilitate future studies on the critical importance of mitochondrial remodeling induced by different types of cellular perturbations.\nA notable caveat of the methodology we have developed and presented is the potential for bias in which components of membrane are segmented and which are not. In addition to the components of membrane that are severed during the cryo-FIB-milling process, membranes fade and disappear when their normal vectors fall into the missing wedge of angles not collected in the tilt series&lt;&lt;REF:Turk-2020-594-3243&gt;&gt;, which can be seen in the lack of orientations less than 30degrees relative to the growth plane (Figure 6F,G &lt;&lt;FIG:fig6&gt;&gt;). Additionally, the segmentation algorithm&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt; used in this study traces lower curvature membranes more easily than high curvature membranes, leading to limited segmentations of tight, high curvature crista junctions and crista tips. We have mitigated these biases to the best of our ability by comparing data collected in similar ways and processed identically across different populations, and we anticipate that improvements to both data collection and data processing will further reduce these biases in the future. The majority of these data were collected without the use of a post-column energy filter&lt;&lt;REF:Koning-2018-217-82&gt;&gt; or phase plate&lt;&lt;REF:Imhof-2019-8-?&gt;&gt;, both of which improve contrast of visible features within tomograms. Additionally, there have been recent advances in algorithmic approaches to improve the quality and completeness of segmentations&lt;&lt;REF:Chen-2017-14-983&gt;&gt;, &lt;&lt;REF:Zhou-2020-?-?&gt;&gt;-&lt;&lt;REF:Liu-2021-?-?&gt;&gt;. The flexibility of the screened Poisson surface reconstruction workflow will enable easy adaptation of these morphometric approaches to new segmentation tools.\nBeyond incorporating new advances for ultrastructure, the automated surface reconstruction algorithm established in this paper for quantitative ultrastructural analysis presents additional potential for use in visual proteomics&lt;&lt;REF:Bäuerlein-2021-433-167187&gt;&gt; and subtomogram averaging. Finding proteins in noisy tomograms is challenging, and membrane segmentations have proven effective guides for localizing protein density that would otherwise be difficult to detect with traditional template matching approaches&lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, &lt;&lt;REF:Martinez-Sanchez-2020-17-209&gt;&gt;. These approaches have in the past depended on voxel segmentations, but moving to high quality meshes has the potential to improve particle picking further by constraining the orientation search to match the geometry of the membrane, as well as providing a more consistent central position than voxel segmentations which might have variable thickness. Prior knowledge of orientation has proven to be valuable for achieving successful initial alignments for subtomogram averaging as well&lt;&lt;REF:Basanta-2020-4-?&gt;&gt;. Membrane meshes have been used very successfully to guide subtomogram averaging in the past&lt;&lt;REF:Navarro-2018-5-?&gt;&gt;, &lt;&lt;REF:Burt-2021-?-?&gt;&gt;, but these have relied on manual segmentation and thus have been low throughput and restricted to relatively simple geometry. Surfaces generated automatically with our software have the potential to extend this guided approach to larger numbers of tomograms and to much more complex biological environments.\nThere is tremendous potential for cryo-ET to enable better understanding of function and regulation by associating changes in protein conformation with the local cellular milieu&lt;&lt;REF:Siegmund-2018-6-83&gt;&gt;, &lt;&lt;REF:Wietrzynski-2020-9-?&gt;&gt;, 55-&lt;&lt;REF:Lin-2018-360-?&gt;&gt;. Cryo-ET's ability to reveal both 3D protein conformation and the cellular ultrastructures surrounding it allows connections between the two that cannot be determined any other way. By using correlative light microscopy approaches to incorporate knowledge of the physiological state of the cell in combination with detailed local membrane morphometrics, it will be possible to gain a much more detailed understanding of how both global cellular state and local membrane ultrastructure correlate to protein localization and conformation. Moving from qualitative association to quantification with the measurement of statistical significance, as outlined here, will improve the power of structure-context mapping to understand how protein structure and function impact complex cellular processes.\n\nSupporting information\nSupplemental Figures\nSupplemental Movie 1\n\nCONFLICTS OF INTEREST\nThe authors declare that they have no conflict of interest.\n\nMATERIALS AND METHODS\n\nPreparation of vitrified mouse embryonic fibroblasts on cryo-EM Grids\nMouse embryonic fibroblasts with mitochondria labeled GFP (MEFmtGFP)&lt;&lt;REF:Wang-2012-51-9302&gt;&gt; were cultured in Dulbecco's Modified Eagle Medium + GlutaMAX (Gibco) additionally supplemented with HiFBS (10%) and glutamine (4 mM) on fibronectin-treated (500 ug/ml, Corning) and UV sterilized R (1/4) Carbon 200-mesh gold electron microscopy (EM) grids (Quantifoil Micro Tools). After 15-18 hours of culture, MEFmtGFP cells were transferred to fresh media supplemented with DMSO in the vehicle treatment group, or fresh media supplemented with Thapsigargin (500nM, Fisher Scientific) in the Tg treatment group. After 8 hours of incubation, samples were plunge-frozen in a liquid ethane/propane mixture using a Vitrobot Mark 4 (Thermo Fisher Scientific) with manual back-blotting.\n\nCryo-fluorescence microscopy and Mitochondria Network Morphology Scoring\nFluorescence and bright-field tiled image maps (atlases) of EM grids containing vitrified cellular samples were acquired with a Leica CryoCLEM microscope (Leica) were collected using Leica LAS X software (25 um Z stacks with system optimized steps, GFP channel ex: 470, em : 525). Z stacks were stitched together in LAS X navigator to provide a single mosaic of maximum projections of the GFP signal, enabling rapid identification of the bulk mitochondrial morphology for each cell. For classification of mitochondrial network morphologies, max projections of individual tiles within fluorescence atlases of MEFmtGFP cells were randomized and blinded. Five researchers classified the cells as containing primarily elongated or fragmented mitochondria (see Figure 2a &lt;&lt;FIG:fig2&gt;&gt; for examples of scoring). Atlases were then exported from LAS X and loaded into MAPS (3.13) (Thermo Fisher) for fluorescence guided milling.\n\nFluorescence Guided Milling\nCryo-Focused Ion Beam milling of lamellae was performed using an Aquilos dual-beam cryo-FIB/SEM instrument (Thermo Fisher Scientific). The fluorescence atlases were overlaid and aligned to an SEM atlases of the same grid to target milling of MEFmtGFP cells with distinct mitochondrial network morphologies, as determined during blind classification (described above). MEFmtGFP cell targets were chosen based on their position within grid squares, the thickness of the ice in their vicinity, and based on their bulk morphology as assessed by the GFP fluorescence channel. Prior to milling, EM grids were first coated with a thin platinum sputter, then coated with an organometallic platinum layer using a gas injection system (GIS) for 3-4 seconds using an automation script&lt;&lt;REF:Barad-?-?-?&gt;&gt;, followed by a third and final layer of platinum sputter was added. Targeted cells were milled manually&lt;&lt;REF:Schaffer-2015-5-?&gt;&gt; or using an automated cryo preparation workflow&lt;&lt;REF:Buckley-2020-107488-?&gt;&gt; both methods used xT software with MAPS (Thermo Fisher). Minimal SEM imaging for monitoring was done at 2keV to ensure thin lamella generation while avoiding radiation damage. A total of 6 treated grids and 8 untreated grids were milled for further tomography analysis.\n\nTilt Series Data Collection\nEM grids containing lamellae were transferred into a 300keV Titan Krios microscope (Thermo Fisher Scientific), equipped with a K2 Summit direct electron detector camera (Gatan). A small subset of data was collected after the installation of a BioQuantum energy filter (Gatan). Individual lamellae were montaged with low dose (1e-/A2) at high magnification (53,000x nominal magnification) to localize cellular regions containing mitochondria, which were identified by their distinctive inner and outer mitochondrial membranes. Data was collected to maximize the number of non-overlapping fields of view containing mitochondria, with no targeting of specific observed membrane ultrastructure. Tilt series were acquired using SerialEM software (Mastronarde &lt;&lt;REF:???-2007-9-90&gt;&gt; ) with 2 steps between -60 and +60. Data was collected with pixel sizes of 3.11A, 3.35 A, or 3.55 A, and a nominal defocus range between -5 to -10 microm. A subset of data was collected with dose fractionation, with 10 0.1e/A2 frames collected per second. The total dose per tilt was 0.9-1.2 e/A2, and the total accumulated dose for the tilt series was under 70 e/A2.\n\nTilt series processing and reconstruction\nDose fractionated tilt series micrograph movies underwent CTF estimation and motion correction in Warp and combined into averaged tilt series for alignment. Both non-fractionated and fractionated tilt series then were aligned using patch tracking in etomo&lt;&lt;REF:Mastronarde-2017-197-102&gt;&gt;, with 4 times binning and 400 binned pixel patches. Resulting contours were manually curated to remove poorly aligning patches, and the remaining contours were used for alignment and reconstruction with weighted back projection into 4 times binned tomograms. Tomogram thicknesses ranged from 95 nm to 233 nm.\n\nSemi-automated Segmentation\nAutomated detection and enhancement of mitochondrial membranes was performed using the TomoSegMemTV software package&lt;&lt;REF:Martinez-Sanchez-2014-186-49&gt;&gt;. Settings were optimized for individual tomograms to maximize the membrane quality. Typical parameters for membrane enhancement were as follows:\nscale_space -s 3\ndtvoting -s 12\nsurfaceness -m 0.17 -s 0.8 -p 0.8\ndtvoting -w -s 10\nsurfaceness -S -s 0.75 -p 0.75 -l 20\nthresholding -l 0.01 -2 30\nglobal analysis -3 100\nThe output correlation volumes containing high pixel intensity corresponding to membrane locations were imported into AMIRA (Thermo Fisher) for manual curation. Mitochondrial inner and outer membranes and endoplasmic reticulum were segmented and designated as separate labels in Amira using 3D threshold-based selection and the 3D magic wand tool, and very small gaps in segmentation were filled in manually using a paint tool.\n\nAutomated surface reconstruction\nThe process for conversion of voxel segmentations into smooth open surfaces fit for quantification was fully automated using python and pymeshlab&lt;&lt;REF:Muntoni-2021-?-?&gt;&gt;. First the individual classes in each segmentation volume were converted into unoriented point clouds, with a single point at the center of each segmented voxel. Next, the normal vectors for each point in the point cloud were estimated based on 120 neighbors with a single smoothing iteration. A surface mesh was calculated from the oriented point cloud using the screened Poisson algorithm&lt;&lt;REF:Kazhdan-2013-32-1&gt;&gt;, with a reconstruction depth of 9, an interpolation weight of 4, and a minimum number of samples of 1.5. The resulting surface extended beyond the segmented region, so triangles more than 1 voxel away from the point cloud were deleted. In cases where the resulting surface was very complex, the surface was simplified with quadric edge collapse decimation to produce a surface that represented the membrane with 150,000 triangles or fewer. This process was accomplished with the 'mrc2xyz.py' (for point cloud generation) and 'make_mesh.py' scripts.\n\nRefinement of surface orientation and estimation of surface curvature using pycurv\nOutput surfaces were processed with pycurv to remove extraneous and poorly modeled regions and to refine surface normals using an area-weighted tensor voting approach&lt;&lt;REF:Salfer-2020-16-?&gt;&gt;. This vector voting process also resulted in robust estimation of curvature metrics. Because the open surfaces generated from membrane segmentations do not have a meaningful inside or outside, signed metrics of curvature are not appropriate for analysis; instead, the unsigned curvedness metric (eq. 1) provides a single positive readout that can be used for quantification. Pycurv output triangle graph files containing the refined normal angles and curvature metrics. which enabled efficient computation of additional morphometric features, and the pycurv library was used as a base for the remaining quantifications in the manuscript. Pycurv calculations were done in the 'calculate_curvature_args.py' script, after file conversion using the 'mask_and_convert_ply.py' script.\n\nCalculation of distances between individual surfaces\nFor calculations of distances between respective surface meshes, the minimum distance from each triangle on one surface to the nearest triangle on the other surface was calculated. For surfaces with small numbers of triangles, this was accomplished using a distance matrix, while calculations on large surfaces, where the distance matrix would take up more than 32GB of system memory, were accomplished using a KD-tree. Both the distance and ID of the nearest triangle on the neighboring surface are recorded in the triangle to enable further analysis. This analysis is accomplished as part of the 'surface_to_surface.py' script.\n\nAutomated subclassification of the IMM\nIn order to understand how different components of the IMM vary with bulk morphology, triangles from inner membrane surfaces were classified into IBM, crista junction, or crista body based on their distance from the OMM. Triangles falling within 19 nm of the OMM are classified as IBM, those between 19 and 40 nm from the OMM as crista junction, and those more than 40 nm from the OMM are classified as crista body. The IBM distance was chosen as the smallest integer value above the peak OMM-IMM distance for all tomograms, while the crista body distance was selected based on visual inspection and to ensure enough triangles were assigned to the crista junctions for robust calculations.\n\nCalculation of distances between components within the inner membrane surface\nBecause all-to-all distance measurements within the same surface would trivially yield the spacing between neighboring triangles on the graph, rather than true distances between neighboring segments of membrane, measurements of distances within individual surfaces are made based on intersections along the normal vector of each triangle, similar to approaches used previously&lt;&lt;REF:Collado-2019-51-476&gt;&gt;. To modify this approach for measuring distances on surfaces without canonical normal directions, the first intersecting triangle was measured both along the normal vector and its inverse, and those distances were sorted into a shorter and longer distance. For cristae and junctions, the shorter distances were interpreted as the spacing between membranes within a crista or junction, whereas the longer distance of the two was interpreted as the distance between individual cristae or junctions. Normal-vector based distance measurements are sensitive to the completeness of the segmentation, and not all triangles have a normal-vector based neighbor. Those triangles were filtered out before any further quantifications.\n\nCalculation of relative orientations with the growth plane and neighboring regions\nThe smoothed normal vectors for each triangle, as estimated by pycurv, were used to estimate the orientation of mitochondrial membranes relative to each other and to the growth plane. Angles considered ranged only from 0degrees to 90degrees in the absence of canonical normal angle directions. For comparisons to the growth plane, the relative angle was calculated as follows (eq. 2):\nComparisons of orientations between membranes was made by comparison of the pycurv-smoothed normal vector between each triangle on a surface to the normal vector its nearest neighbor as determined previously. The angle between these two vectors was calculated as follows (eq. 3):\nBoth orientation calculations are determined in the 'surface_to_surface.py' script.\n\nVisualization and aggregate quantification of data and assessment of statistical significance\nThe per-triangle measurements are stored within the pycurv triangle graph file as well as being output both as surface meshes using the visualization toolkit 'vtp'&lt;&lt;REF:Schroeder-2006-?-?&gt;&gt; format as well as in csv files using existing pycurv functions. Visualizations were generated using the vtp files in paraview&lt;&lt;REF:Ahrens-2005-717-?&gt;&gt;. Aggregate histograms and empirical continuous distribution plots were plotted in matplotlib after aggregating all triangles across all tomograms within a condition and morphology group and weighting each triangle by its area.&lt;&lt;REF:???-2007-9-90&gt;&gt; Violin plots were generated by generating triangle-area weighted histograms for each tomogram with 100 bins and identifying the value of the most populated bin, followed by aggregation of peak values. Statistics were similarly calculated using Mann-Whitney U tests&lt;&lt;REF:Mann-1947-18-50&gt;&gt; on the peak values calculated for each tomogram to identify statistically significant differences in peak positions. For quantifying differences in distribution shape, a Mann-Whitney U test was applied to the standard deviations of each tomogram, rather than the histogram peak.\n\nCode Availability\nAll scripts used for surface reconstruction, ultrastructure quantifications, and plotting and statistical significance testing are available at https://github.com/grotjahnlab/surface_morphometrics. Scripts are licensed under the GNU Public License version 3 (GPLv3).'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to study the microscopic structure of a biological sample.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List the different types of biological sample being used? Select from the following list:- cell culture, tissue, organoid, organ, whole organism, virus, micro-organism, other. Record this value in the 'biological_sample_type' field of the output.\n2. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, return none. Record this value in the 'organism_name' field of the output.\n3. If the biological sample described in the text is a cell or cell culture, what was the strain? Record this value in the 'cell_strain' field of the output.\n4. Provide a summary of any actions performed on the biological sample. Record this value in the 'sample_preparation' field of the output.\n5. Provide a summary of how grids were prepared for electron microscopy. Record this value in the 'grid_preparation' field of the output.\n6. What was the pixel spacing the CryoET step? Record this value in the 'cryoet_pixel_spacing' field of the output.\n7. What was the acceleration voltage? Record this value in the 'cryoet_acceleration_voltage' field of the output.\n8. What was the name of the type of electron microscope being used? Record this value in the 'microscope_name' field of the output.\n9. If an energy filter was used (in the electron microscope setup), describe the type of filter used? Record this value in the 'microscope_setup_energy_filter' field of the output.\n10. If a phase plate was used (in the electron microscope setup), describe the type of phase plate used? Record this value in the 'microscope_setup_phase_plate' field of the output.\n11. If an image corrector was used (in the electron microscope setup), describe the type of image corrector used? Record this value in the 'microscope_setup_image_corrector' field of the output.\n12. Which company made the camera used to capture tilt images? Record this value in the 'camera_manufacturer' field of the output.\n13. What model of camera was used to capture tilt images? Record this value in the 'camera_model' field of the output.\n14. What was the minimum angle used in the tilt-series? Record this value in the 'tilt_minimum' field of the output.\n15. What was the maximum angle used in the tilt-series? Record this value in the 'tilt_maximum' field of the output.\n16. What was the total flux or total exposure dose? Record this value in the 'total_flux' field of the output.\n17. What software or computational methods were used to perform the tomogram reconstruction? Record this value in the 'reconstruction_software' field of the output.\n18. What are the dimensions of the tomogram dataset (measured in numbers of pixels)? Record this value in the 'tomogram_size' field of the output.\n19. What computational methods / algorithms were used (e.g., weighted back propagation, etc)? Record this value in the 'reconstruction_method' field of the output.\n20. What organelles or subcellular components were annotated in the images? Record this value in the 'annotated_entities' field of the output.\n21. List all the methods describing how annotations were generated (manually, computational analysis, machine learning, etc)? Record this value in the 'annotation_methods' field of the output.\n22. What software was used to make the annotations? Record this value in the 'annotation_software' field of the output.\nGenerate only JSON formatted output with 22 fields:\nbiological_sample_type, organism_name, cell_strain, sample_preparation, grid_preparation, cryoet_pixel_spacing, cryoet_acceleration_voltage, microscope_name, microscope_setup_energy_filter, microscope_setup_phase_plate, microscope_setup_image_corrector, camera_manufacturer, camera_model, tilt_minimum, tilt_maximum, total_flux, reconstruction_software, tomogram_size, reconstruction_method, annotated_entities, annotation_methods, annotation_software\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [20.24s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"biological_sample_type\": \"cell culture\",\n  \"organism_name\": \"Mus musculus\",\n  \"cell_strain\": \"Mouse embryonic fibroblasts\",\n  \"sample_preparation\": \"Cultured on fibronectin-treated and UV sterilized grids, treated with vehicle or thapsigargin for 8 hours, plunge-frozen in ethane/propane mixture\",\n  \"grid_preparation\": \"Cultured on fibronectin-treated and UV sterilized carbon 200-mesh gold EM grids, coated with platinum and plunge-frozen\",\n  \"cryoet_pixel_spacing\": \"3.11A, 3.35 A, or 3.55 A\",\n  \"cryoet_acceleration_voltage\": \"300keV\",\n  \"microscope_name\": \"Titan Krios\",\n  \"microscope_setup_energy_filter\": \"BioQuantum energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K2 Summit\",\n  \"tilt_minimum\": \"-60\",\n  \"tilt_maximum\": \"+60\",\n  \"total_flux\": \"under 70 e/A2\",\n  \"reconstruction_software\": \"SerialEM, Warp, etomo\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"weighted back projection\",\n  \"annotated_entities\": \"mitochondrial inner and outer membranes, endoplasmic reticulum\",\n  \"annotation_methods\": \"semi-automated approach using TomoSegMemTV and manual curation\",\n  \"annotation_software\": \"AMIRA\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"biological_sample_type\": \"cell culture\",\n  \"organism_name\": \"Mus musculus\",\n  \"cell_strain\": \"Mouse embryonic fibroblasts\",\n  \"sample_preparation\": \"Cultured on fibronectin-treated and UV sterilized grids, treated with vehicle or thapsigargin for 8 hours, plunge-frozen in ethane/propane mixture\",\n  \"grid_preparation\": \"Cultured on fibronectin-treated and UV sterilized carbon 200-mesh gold EM grids, coated with platinum and plunge-frozen\",\n  \"cryoet_pixel_spacing\": \"3.11A, 3.35 A, or 3.55 A\",\n  \"cryoet_acceleration_voltage\": \"300keV\",\n  \"microscope_name\": \"Titan Krios\",\n  \"microscope_setup_energy_filter\": \"BioQuantum energy filter\",\n  \"microscope_setup_phase_plate\": \"not present\",\n  \"microscope_setup_image_corrector\": \"not present\",\n  \"camera_manufacturer\": \"Gatan\",\n  \"camera_model\": \"K2 Summit\",\n  \"tilt_minimum\": \"-60\",\n  \"tilt_maximum\": \"+60\",\n  \"total_flux\": \"under 70 e/A2\",\n  \"reconstruction_software\": \"SerialEM, Warp, etomo\",\n  \"tomogram_size\": \"not present\",\n  \"reconstruction_method\": \"weighted back projection\",\n  \"annotated_entities\": \"mitochondrial inner and outer membranes, endoplasmic reticulum\",\n  \"annotation_methods\": \"semi-automated approach using TomoSegMemTV and manual curation\",\n  \"annotation_software\": \"AMIRA\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 360,
      "prompt_tokens": 14624,
      "total_tokens": 14984
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_7cc080b25b"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [2ms] Exiting Parser run with output:
{
  "biological_sample_type": "cell culture",
  "organism_name": "Mus musculus",
  "cell_strain": "Mouse embryonic fibroblasts",
  "sample_preparation": "Cultured on fibronectin-treated and UV sterilized grids, treated with vehicle or thapsigargin for 8 hours, plunge-frozen in ethane/propane mixture",
  "grid_preparation": "Cultured on fibronectin-treated and UV sterilized carbon 200-mesh gold EM grids, coated with platinum and plunge-frozen",
  "cryoet_pixel_spacing": "3.11A, 3.35 A, or 3.55 A",
  "cryoet_acceleration_voltage": "300keV",
  "microscope_name": "Titan Krios",
  "microscope_setup_energy_filter": "BioQuantum energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Gatan",
  "camera_model": "K2 Summit",
  "tilt_minimum": "-60",
  "tilt_maximum": "+60",
  "total_flux": "under 70 e/A2",
  "reconstruction_software": "SerialEM, Warp, etomo",
  "tomogram_size": "not present",
  "reconstruction_method": "weighted back projection",
  "annotated_entities": "mitochondrial inner and outer membranes, endoplasmic reticulum",
  "annotation_methods": "semi-automated approach using TomoSegMemTV and manual curation",
  "annotation_software": "AMIRA"
}
[chain/end] [1:chain:RunnableSequence] [20.25s] Exiting Chain run with output:
{
  "biological_sample_type": "cell culture",
  "organism_name": "Mus musculus",
  "cell_strain": "Mouse embryonic fibroblasts",
  "sample_preparation": "Cultured on fibronectin-treated and UV sterilized grids, treated with vehicle or thapsigargin for 8 hours, plunge-frozen in ethane/propane mixture",
  "grid_preparation": "Cultured on fibronectin-treated and UV sterilized carbon 200-mesh gold EM grids, coated with platinum and plunge-frozen",
  "cryoet_pixel_spacing": "3.11A, 3.35 A, or 3.55 A",
  "cryoet_acceleration_voltage": "300keV",
  "microscope_name": "Titan Krios",
  "microscope_setup_energy_filter": "BioQuantum energy filter",
  "microscope_setup_phase_plate": "not present",
  "microscope_setup_image_corrector": "not present",
  "camera_manufacturer": "Gatan",
  "camera_model": "K2 Summit",
  "tilt_minimum": "-60",
  "tilt_maximum": "+60",
  "total_flux": "under 70 e/A2",
  "reconstruction_software": "SerialEM, Warp, etomo",
  "tomogram_size": "not present",
  "reconstruction_method": "weighted back projection",
  "annotated_entities": "mitochondrial inner and outer membranes, endoplasmic reticulum",
  "annotation_methods": "semi-automated approach using TomoSegMemTV and manual curation",
  "annotation_software": "AMIRA"
}</code></pre>
</div>
</div>
<div class="cell" data-execution_count="22">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="co"># clear all notes from a paper</span></span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d <span class="kw">in</span> [d <span class="cf">for</span> d_id <span class="kw">in</span> dois <span class="cf">for</span> d <span class="kw">in</span> dois[d_id]]:</span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>    d_id <span class="op">=</span> <span class="st">'doi:'</span><span class="op">+</span>d</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>    e <span class="op">=</span> ldb.session.query(SKE).<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>d_id).first()</span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>    notes_to_delete <span class="op">=</span> []</span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> n <span class="kw">in</span> ldb.read_notes_about_x(e):</span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>        notes_to_delete.append(n.<span class="bu">id</span>)</span>
<span id="cb27-8"><a href="#cb27-8" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> n <span class="kw">in</span> notes_to_delete:</span>
<span id="cb27-9"><a href="#cb27-9" aria-hidden="true" tabindex="-1"></a>        ldb.delete_note(n)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="protocol-modeling-extraction" class="level2">
<h2 class="anchored" data-anchor-id="protocol-modeling-extraction">Protocol Modeling + Extraction</h2>
<div class="cell" data-execution_count="10">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>ldb <span class="op">=</span> Ceifns_LiteratureDb(loc<span class="op">=</span>loc, name<span class="op">=</span>db_name)</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>slm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'stablelm-zephyr'</span>) </span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>llm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'mixtral:instruct'</span>) </span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>llm2 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-4-1106-preview'</span>) </span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>d <span class="op">=</span> (<span class="st">"This tool attempts to draw a protocol design from the description of a scientific paper."</span>)</span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>t <span class="op">=</span> ProcotolExtractionTool(db<span class="op">=</span>ldb, llm<span class="op">=</span>llm2, description<span class="op">=</span>d)</span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a>t.run(tool_input<span class="op">=</span>{<span class="st">'paper_id'</span>: <span class="st">'doi:10.1101/2022.04.12.488077'</span>, <span class="st">'extraction_type'</span>: <span class="st">'cryoet'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-error">
<pre><code>KeyError: 'all entities'</code></pre>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>