{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# llama_cpp\n",
    "\n",
    "> Basic Experiments with Ollama using LangChain models, chains, and agents.    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sqlalchemy as db\n",
    "from sqlalchemy.engine import create_engine\n",
    "import json\n",
    "\n",
    "from langchain.llms.ollama import Ollama\n",
    "import os\n",
    "import pandas as pd\n",
    "from pprint import pprint\n",
    "import textwrap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Image Tech Text and Extractions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "engine = create_engine(\n",
    "    'databricks://token:'+os.environ['DB_TOKEN']+ \n",
    "    '@czi-shared-infra-czi-sci-general-prod-databricks.cloud.databricks.com'+\n",
    "    '?http_path=/sql/1.0/warehouses/1c4df94f2f1a6305')\n",
    "\n",
    "sql = '''\n",
    "SELECT * FROM gburns.nature_methods.papers WHERE LEN(ABSTRACT) > 0\n",
    "'''\n",
    "with engine.connect() as con:\n",
    "    stmt = db.text(sql)\n",
    "    rs = con.execute(stmt)\n",
    "    df = pd.DataFrame(rs.fetchall(), columns=rs.keys())\n",
    "df\n",
    "\n",
    "#metadata = db.MetaData()\n",
    "#metadata.reflect(bind=engine)\n",
    "#tbls = metadata.tables\n",
    "#print(json.dumps([str(t) for t in metadata.tables], indent=4))\n",
    "\n",
    "#logs = Table(\"corpus\", MetaData(bind=engine), autoload=True)\n",
    "sql = '''\n",
    "SELECT * FROM gburns.nature_methods.papers WHERE LEN(ABSTRACT) > 0\n",
    "\n",
    "'''\n",
    "with engine.connect() as con:\n",
    "    stmt = db.text(sql)\n",
    "    rs = con.execute(stmt)\n",
    "    df = pd.DataFrame(rs.fetchall(), columns=rs.keys())\n",
    "df\n",
    "#print(select([func.count(\"*\")], from_obj=logs).scalar())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>DOI</th>\n",
       "      <th>YEAR</th>\n",
       "      <th>ABSTRACT</th>\n",
       "      <th>TITLE</th>\n",
       "      <th>MESH_DESCRIPTORS</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>34811556</td>\n",
       "      <td>10.1038/s41592-021-01316-y</td>\n",
       "      <td>2022</td>\n",
       "      <td>Tissues and organs are composed of distinct ce...</td>\n",
       "      <td>Spatial mapping of protein composition and tis...</td>\n",
       "      <td>Antibodies | Diagnostic Imaging | Cell Communi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>34824477</td>\n",
       "      <td>10.1038/s41592-021-01308-y</td>\n",
       "      <td>2022</td>\n",
       "      <td>Highly multiplexed tissue imaging makes detail...</td>\n",
       "      <td>MCMICRO: a scalable, modular image-processing ...</td>\n",
       "      <td>Humans | Neoplasms | Diagnostic Imaging | Imag...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>34887550</td>\n",
       "      <td>10.1038/s41592-021-01335-9</td>\n",
       "      <td>2022</td>\n",
       "      <td>Mako is a software tool that converts microbio...</td>\n",
       "      <td>Fast and flexible analysis of linked microbiom...</td>\n",
       "      <td>Animals | Computational Biology | Computer Gra...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>34887551</td>\n",
       "      <td>10.1038/s41592-021-01334-w</td>\n",
       "      <td>2022</td>\n",
       "      <td>Recent whole-brain mapping projects are collec...</td>\n",
       "      <td>Cross-modal coherent registration of whole mou...</td>\n",
       "      <td>Brain | Animals | Mice, Inbred C57BL | Imaging...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>34916672</td>\n",
       "      <td>10.1038/s41592-021-01341-x</td>\n",
       "      <td>2022</td>\n",
       "      <td>Self-labeling protein tags such as HaloTag are...</td>\n",
       "      <td>Engineered HaloTag variants for fluorescence l...</td>\n",
       "      <td>Cell Line | Humans | Rhodamines | Hydrolases |...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6026</th>\n",
       "      <td>PPR587527</td>\n",
       "      <td>10.1101/2022.12.19.519427</td>\n",
       "      <td>2022</td>\n",
       "      <td>Cohort studies increasingly collect biosamples...</td>\n",
       "      <td>Omada: Robust clustering of transcriptomes thr...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6027</th>\n",
       "      <td>PPR587989</td>\n",
       "      <td>10.1101/2022.12.21.520923</td>\n",
       "      <td>2022</td>\n",
       "      <td>The two clathrin isoforms, CHC17 and CHC22, ge...</td>\n",
       "      <td>CHC22 clathrin membrane recruitment uses SNX5 ...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6028</th>\n",
       "      <td>PPR588013</td>\n",
       "      <td>10.1101/2022.12.21.521417</td>\n",
       "      <td>2022</td>\n",
       "      <td>With the recent growth in spectral flow cytome...</td>\n",
       "      <td>Back to the Future- Unleashing your cytometer’...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6029</th>\n",
       "      <td>PPR588026</td>\n",
       "      <td>10.1101/2022.12.21.521384</td>\n",
       "      <td>2022</td>\n",
       "      <td>The shape of cells is the outcome of the balan...</td>\n",
       "      <td>Friction patterns guide actin network contraction</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6030</th>\n",
       "      <td>PPR588048</td>\n",
       "      <td>10.1101/2022.12.21.521283</td>\n",
       "      <td>2022</td>\n",
       "      <td>Inflammation is essential for responding to in...</td>\n",
       "      <td>Inflammatory Synovial Fibroblast Culture in 3D...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>6031 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       ID_PAPER                         DOI  YEAR  \\\n",
       "0      34811556  10.1038/s41592-021-01316-y  2022   \n",
       "1      34824477  10.1038/s41592-021-01308-y  2022   \n",
       "2      34887550  10.1038/s41592-021-01335-9  2022   \n",
       "3      34887551  10.1038/s41592-021-01334-w  2022   \n",
       "4      34916672  10.1038/s41592-021-01341-x  2022   \n",
       "...         ...                         ...   ...   \n",
       "6026  PPR587527   10.1101/2022.12.19.519427  2022   \n",
       "6027  PPR587989   10.1101/2022.12.21.520923  2022   \n",
       "6028  PPR588013   10.1101/2022.12.21.521417  2022   \n",
       "6029  PPR588026   10.1101/2022.12.21.521384  2022   \n",
       "6030  PPR588048   10.1101/2022.12.21.521283  2022   \n",
       "\n",
       "                                               ABSTRACT  \\\n",
       "0     Tissues and organs are composed of distinct ce...   \n",
       "1     Highly multiplexed tissue imaging makes detail...   \n",
       "2     Mako is a software tool that converts microbio...   \n",
       "3     Recent whole-brain mapping projects are collec...   \n",
       "4     Self-labeling protein tags such as HaloTag are...   \n",
       "...                                                 ...   \n",
       "6026  Cohort studies increasingly collect biosamples...   \n",
       "6027  The two clathrin isoforms, CHC17 and CHC22, ge...   \n",
       "6028  With the recent growth in spectral flow cytome...   \n",
       "6029  The shape of cells is the outcome of the balan...   \n",
       "6030  Inflammation is essential for responding to in...   \n",
       "\n",
       "                                                  TITLE  \\\n",
       "0     Spatial mapping of protein composition and tis...   \n",
       "1     MCMICRO: a scalable, modular image-processing ...   \n",
       "2     Fast and flexible analysis of linked microbiom...   \n",
       "3     Cross-modal coherent registration of whole mou...   \n",
       "4     Engineered HaloTag variants for fluorescence l...   \n",
       "...                                                 ...   \n",
       "6026  Omada: Robust clustering of transcriptomes thr...   \n",
       "6027  CHC22 clathrin membrane recruitment uses SNX5 ...   \n",
       "6028  Back to the Future- Unleashing your cytometer’...   \n",
       "6029  Friction patterns guide actin network contraction   \n",
       "6030  Inflammatory Synovial Fibroblast Culture in 3D...   \n",
       "\n",
       "                                       MESH_DESCRIPTORS  \n",
       "0     Antibodies | Diagnostic Imaging | Cell Communi...  \n",
       "1     Humans | Neoplasms | Diagnostic Imaging | Imag...  \n",
       "2     Animals | Computational Biology | Computer Gra...  \n",
       "3     Brain | Animals | Mice, Inbred C57BL | Imaging...  \n",
       "4     Cell Line | Humans | Rhodamines | Hydrolases |...  \n",
       "...                                                 ...  \n",
       "6026                                                     \n",
       "6027                                                     \n",
       "6028                                                     \n",
       "6029                                                     \n",
       "6030                                                     \n",
       "\n",
       "[6031 rows x 6 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sql = '''\n",
    "SELECT p.* \n",
    "FROM gburns.imaging_tech.corpus_papers as cp\n",
    "    JOIN gburns.imaging_tech.papers as p on (p.ID_PAPER = cp.ID_PAPER)\n",
    "WHERE LEN(p.ABSTRACT)>0 AND p.YEAR>=2022\n",
    "'''\n",
    "with engine.connect() as con:\n",
    "    stmt = db.text(sql)\n",
    "    rs = con.execute(stmt)\n",
    "    df_2022 = pd.DataFrame(rs.fetchall(), columns=rs.keys())\n",
    "\n",
    "df_2022\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Starting analysis with Langchain"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "llama_model_loader: loaded meta data with 19 key-value pairs and 723 tensors from /Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf (version GGUF V2 (latest))\n",
      "llama_model_loader: - tensor    0:                token_embd.weight q5_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - tensor    1:           blk.0.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    2:            blk.0.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    3:            blk.0.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    4:              blk.0.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    5:            blk.0.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    6:              blk.0.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor    7:         blk.0.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    8:              blk.0.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    9:              blk.0.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   10:           blk.1.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   11:            blk.1.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   12:            blk.1.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   13:              blk.1.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   14:            blk.1.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   15:              blk.1.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   16:         blk.1.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   17:              blk.1.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   18:              blk.1.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   19:           blk.2.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   20:            blk.2.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   21:            blk.2.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   22:              blk.2.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   23:            blk.2.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   24:              blk.2.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   25:         blk.2.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   26:              blk.2.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   27:              blk.2.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   28:           blk.3.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   29:            blk.3.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   30:            blk.3.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   31:              blk.3.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   32:            blk.3.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   33:              blk.3.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   34:         blk.3.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   35:              blk.3.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   36:              blk.3.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   37:           blk.4.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   38:            blk.4.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   39:            blk.4.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   40:              blk.4.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   41:            blk.4.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   42:              blk.4.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   43:         blk.4.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   44:              blk.4.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   45:              blk.4.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   46:            blk.5.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   47:              blk.5.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   48:         blk.5.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   49:              blk.5.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   50:              blk.5.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   51:          blk.10.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   52:           blk.10.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   53:           blk.10.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   54:             blk.10.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   55:           blk.10.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   56:             blk.10.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   57:        blk.10.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   58:             blk.10.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   59:             blk.10.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   60:             blk.11.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   61:        blk.11.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   62:             blk.11.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   63:             blk.11.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   64:           blk.5.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   65:            blk.5.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   66:              blk.5.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   67:            blk.5.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   68:           blk.6.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   69:            blk.6.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   70:            blk.6.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   71:              blk.6.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   72:            blk.6.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   73:              blk.6.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   74:         blk.6.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   75:              blk.6.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   76:              blk.6.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   77:           blk.7.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   78:            blk.7.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   79:            blk.7.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   80:              blk.7.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   81:            blk.7.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   82:              blk.7.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   83:         blk.7.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   84:              blk.7.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   85:              blk.7.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   86:           blk.8.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   87:            blk.8.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   88:            blk.8.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   89:              blk.8.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   90:            blk.8.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   91:              blk.8.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   92:         blk.8.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   93:              blk.8.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   94:              blk.8.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   95:           blk.9.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   96:            blk.9.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   97:            blk.9.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   98:              blk.9.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   99:            blk.9.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  100:              blk.9.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  101:         blk.9.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  102:              blk.9.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  103:              blk.9.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  104:          blk.11.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  105:           blk.11.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  106:           blk.11.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  107:             blk.11.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  108:           blk.11.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  109:          blk.12.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  110:           blk.12.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  111:           blk.12.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  112:             blk.12.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  113:           blk.12.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  114:             blk.12.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  115:        blk.12.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  116:             blk.12.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  117:             blk.12.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  118:          blk.13.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  119:           blk.13.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  120:           blk.13.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  121:             blk.13.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  122:           blk.13.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  123:             blk.13.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  124:        blk.13.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  125:             blk.13.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  126:             blk.13.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  127:          blk.14.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  128:           blk.14.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  129:           blk.14.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  130:             blk.14.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  131:           blk.14.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  132:             blk.14.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  133:        blk.14.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  134:             blk.14.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  135:             blk.14.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  136:          blk.15.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  137:           blk.15.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  138:           blk.15.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  139:             blk.15.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  140:           blk.15.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  141:             blk.15.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  142:        blk.15.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  143:             blk.15.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  144:             blk.15.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  145:          blk.16.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  146:           blk.16.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  147:           blk.16.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  148:             blk.16.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  149:           blk.16.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  150:             blk.16.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  151:        blk.16.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  152:             blk.16.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  153:             blk.16.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  154:          blk.17.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  155:           blk.17.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  156:           blk.17.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  157:             blk.17.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  158:           blk.17.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  159:             blk.17.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  160:        blk.17.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  161:             blk.17.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  162:             blk.17.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  163:          blk.18.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  164:           blk.18.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  165:           blk.18.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  166:             blk.18.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  167:           blk.18.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  168:             blk.18.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  169:        blk.18.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  170:             blk.18.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  171:             blk.18.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  172:          blk.19.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  173:           blk.19.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  174:           blk.19.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  175:             blk.19.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  176:           blk.19.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  177:             blk.19.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  178:        blk.19.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  179:             blk.19.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  180:             blk.19.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  181:          blk.20.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  182:           blk.20.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  183:           blk.20.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  184:             blk.20.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  185:           blk.20.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  186:             blk.20.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  187:        blk.20.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  188:             blk.20.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  189:             blk.20.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  190:          blk.21.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  191:           blk.21.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  192:           blk.21.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  193:             blk.21.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  194:           blk.21.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  195:             blk.21.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  196:        blk.21.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  197:             blk.21.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  198:             blk.21.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  199:           blk.22.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  200:             blk.22.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  201:             blk.22.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  202:        blk.22.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  203:             blk.22.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  204:             blk.22.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  205:          blk.22.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  206:           blk.22.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  207:           blk.22.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  208:          blk.23.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  209:           blk.23.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  210:           blk.23.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  211:             blk.23.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  212:           blk.23.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  213:             blk.23.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  214:        blk.23.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  215:             blk.23.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  216:             blk.23.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  217:          blk.24.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  218:           blk.24.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  219:           blk.24.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  220:             blk.24.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  221:           blk.24.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  222:             blk.24.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  223:        blk.24.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  224:             blk.24.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  225:             blk.24.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  226:          blk.25.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  227:           blk.25.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  228:           blk.25.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  229:             blk.25.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  230:           blk.25.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  231:             blk.25.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  232:        blk.25.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  233:             blk.25.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  234:             blk.25.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  235:          blk.26.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  236:           blk.26.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  237:           blk.26.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  238:             blk.26.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  239:           blk.26.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  240:             blk.26.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  241:        blk.26.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  242:             blk.26.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  243:             blk.26.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  244:          blk.27.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  245:           blk.27.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  246:           blk.27.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  247:             blk.27.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  248:           blk.27.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  249:             blk.27.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  250:        blk.27.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  251:             blk.27.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  252:             blk.27.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  253:           blk.28.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  254:             blk.28.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  255:        blk.28.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  256:             blk.28.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  257:             blk.28.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  258:          blk.28.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  259:           blk.28.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  260:             blk.28.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  261:           blk.28.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  262:          blk.29.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  263:           blk.29.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  264:           blk.29.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  265:             blk.29.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  266:           blk.29.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  267:             blk.29.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  268:        blk.29.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  269:             blk.29.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  270:             blk.29.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  271:          blk.30.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  272:           blk.30.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  273:           blk.30.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  274:             blk.30.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  275:           blk.30.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  276:             blk.30.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  277:        blk.30.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  278:             blk.30.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  279:             blk.30.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  280:          blk.31.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  281:           blk.31.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  282:           blk.31.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  283:             blk.31.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  284:           blk.31.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  285:             blk.31.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  286:        blk.31.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  287:             blk.31.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  288:             blk.31.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  289:          blk.32.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  290:           blk.32.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  291:           blk.32.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  292:             blk.32.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  293:           blk.32.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  294:             blk.32.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  295:        blk.32.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  296:             blk.32.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  297:             blk.32.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  298:          blk.33.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  299:           blk.33.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  300:           blk.33.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  301:             blk.33.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  302:           blk.33.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  303:             blk.33.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  304:        blk.33.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  305:             blk.33.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  306:             blk.33.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  307:             blk.34.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  308:        blk.34.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  309:             blk.34.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  310:             blk.34.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  311:          blk.34.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  312:           blk.34.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  313:           blk.34.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  314:             blk.34.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  315:           blk.34.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  316:          blk.35.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  317:           blk.35.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  318:           blk.35.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  319:             blk.35.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  320:           blk.35.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  321:             blk.35.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  322:        blk.35.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  323:             blk.35.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  324:             blk.35.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  325:          blk.36.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  326:           blk.36.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  327:           blk.36.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  328:             blk.36.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  329:           blk.36.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  330:             blk.36.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  331:        blk.36.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  332:             blk.36.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  333:             blk.36.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  334:          blk.37.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  335:           blk.37.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  336:           blk.37.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  337:             blk.37.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  338:           blk.37.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  339:             blk.37.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  340:        blk.37.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  341:             blk.37.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  342:             blk.37.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  343:          blk.38.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  344:           blk.38.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  345:           blk.38.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  346:             blk.38.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  347:           blk.38.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  348:             blk.38.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  349:        blk.38.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  350:             blk.38.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  351:             blk.38.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  352:          blk.39.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  353:           blk.39.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  354:           blk.39.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  355:             blk.39.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  356:           blk.39.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  357:             blk.39.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  358:        blk.39.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  359:             blk.39.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  360:             blk.39.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  361:          blk.40.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  362:           blk.40.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  363:           blk.40.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  364:             blk.40.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  365:           blk.40.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  366:             blk.40.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  367:        blk.40.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  368:             blk.40.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  369:             blk.40.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  370:          blk.41.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  371:           blk.41.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  372:           blk.41.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  373:             blk.41.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  374:           blk.41.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  375:             blk.41.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  376:        blk.41.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  377:             blk.41.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  378:             blk.41.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  379:          blk.42.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  380:           blk.42.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  381:           blk.42.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  382:             blk.42.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  383:           blk.42.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  384:             blk.42.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  385:        blk.42.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  386:             blk.42.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  387:             blk.42.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  388:          blk.43.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  389:           blk.43.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  390:           blk.43.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  391:             blk.43.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  392:           blk.43.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  393:             blk.43.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  394:        blk.43.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  395:             blk.43.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  396:             blk.43.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  397:          blk.44.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  398:           blk.44.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  399:           blk.44.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  400:             blk.44.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  401:           blk.44.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  402:             blk.44.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  403:        blk.44.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  404:             blk.44.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  405:             blk.44.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  406:           blk.45.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  407:             blk.45.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  408:             blk.45.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  409:        blk.45.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  410:             blk.45.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  411:             blk.45.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  412:          blk.45.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  413:           blk.45.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  414:           blk.45.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  415:          blk.46.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  416:           blk.46.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  417:           blk.46.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  418:             blk.46.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  419:           blk.46.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  420:             blk.46.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  421:        blk.46.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  422:             blk.46.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  423:             blk.46.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  424:          blk.47.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  425:           blk.47.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  426:           blk.47.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  427:             blk.47.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  428:           blk.47.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  429:             blk.47.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  430:        blk.47.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  431:             blk.47.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  432:             blk.47.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  433:          blk.48.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  434:           blk.48.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  435:           blk.48.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  436:             blk.48.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  437:           blk.48.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  438:             blk.48.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  439:        blk.48.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  440:             blk.48.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  441:             blk.48.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  442:          blk.49.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  443:           blk.49.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  444:           blk.49.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  445:             blk.49.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  446:           blk.49.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  447:             blk.49.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  448:        blk.49.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  449:             blk.49.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  450:             blk.49.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  451:          blk.50.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  452:           blk.50.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  453:           blk.50.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  454:             blk.50.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  455:           blk.50.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  456:             blk.50.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  457:        blk.50.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  458:             blk.50.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  459:             blk.50.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  460:           blk.51.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  461:             blk.51.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  462:        blk.51.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  463:             blk.51.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  464:             blk.51.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  465:          blk.51.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  466:           blk.51.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  467:             blk.51.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  468:           blk.51.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  469:          blk.52.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  470:           blk.52.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  471:           blk.52.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  472:             blk.52.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  473:           blk.52.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  474:             blk.52.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  475:        blk.52.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  476:             blk.52.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  477:             blk.52.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  478:          blk.53.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  479:           blk.53.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  480:           blk.53.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  481:             blk.53.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  482:           blk.53.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  483:             blk.53.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  484:        blk.53.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  485:             blk.53.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  486:             blk.53.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  487:          blk.54.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  488:           blk.54.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  489:           blk.54.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  490:             blk.54.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  491:           blk.54.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  492:             blk.54.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  493:        blk.54.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  494:             blk.54.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  495:             blk.54.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  496:          blk.55.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  497:           blk.55.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  498:           blk.55.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  499:             blk.55.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  500:           blk.55.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  501:             blk.55.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  502:        blk.55.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  503:             blk.55.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  504:             blk.55.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  505:          blk.56.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  506:           blk.56.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  507:           blk.56.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  508:             blk.56.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  509:           blk.56.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  510:             blk.56.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  511:        blk.56.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  512:             blk.56.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  513:             blk.56.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  514:             blk.57.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  515:        blk.57.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  516:             blk.57.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  517:             blk.57.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  518:          blk.57.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  519:           blk.57.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  520:           blk.57.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  521:             blk.57.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  522:           blk.57.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  523:          blk.58.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  524:           blk.58.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  525:           blk.58.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  526:             blk.58.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  527:           blk.58.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  528:             blk.58.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  529:        blk.58.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  530:             blk.58.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  531:             blk.58.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  532:          blk.59.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  533:           blk.59.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  534:           blk.59.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  535:             blk.59.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  536:           blk.59.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  537:             blk.59.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  538:        blk.59.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  539:             blk.59.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  540:             blk.59.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  541:          blk.60.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  542:           blk.60.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  543:           blk.60.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  544:             blk.60.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  545:           blk.60.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  546:             blk.60.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  547:        blk.60.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  548:             blk.60.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  549:             blk.60.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  550:          blk.61.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  551:           blk.61.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  552:           blk.61.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  553:             blk.61.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  554:           blk.61.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  555:             blk.61.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  556:        blk.61.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  557:             blk.61.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  558:             blk.61.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  559:          blk.62.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  560:           blk.62.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  561:           blk.62.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  562:             blk.62.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  563:           blk.62.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  564:             blk.62.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  565:        blk.62.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  566:             blk.62.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  567:             blk.62.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  568:          blk.63.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  569:           blk.63.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  570:           blk.63.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  571:             blk.63.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  572:           blk.63.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  573:             blk.63.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  574:        blk.63.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  575:             blk.63.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  576:             blk.63.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  577:          blk.64.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  578:           blk.64.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  579:           blk.64.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  580:             blk.64.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  581:           blk.64.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  582:             blk.64.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  583:        blk.64.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  584:             blk.64.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  585:             blk.64.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  586:          blk.65.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  587:           blk.65.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  588:           blk.65.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  589:             blk.65.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  590:           blk.65.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  591:             blk.65.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  592:        blk.65.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  593:             blk.65.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  594:             blk.65.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  595:          blk.66.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  596:           blk.66.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  597:           blk.66.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  598:             blk.66.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  599:           blk.66.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  600:             blk.66.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  601:        blk.66.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  602:             blk.66.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  603:             blk.66.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  604:          blk.67.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  605:           blk.67.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  606:           blk.67.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  607:             blk.67.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  608:           blk.67.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  609:             blk.67.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  610:        blk.67.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  611:             blk.67.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  612:             blk.67.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  613:           blk.68.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  614:             blk.68.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  615:             blk.68.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  616:        blk.68.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  617:             blk.68.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  618:             blk.68.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  619:          blk.68.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  620:           blk.68.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  621:           blk.68.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  622:          blk.69.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  623:           blk.69.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  624:           blk.69.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  625:             blk.69.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  626:           blk.69.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  627:             blk.69.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  628:        blk.69.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  629:             blk.69.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  630:             blk.69.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  631:          blk.70.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  632:           blk.70.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  633:           blk.70.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  634:             blk.70.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  635:           blk.70.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  636:             blk.70.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  637:        blk.70.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  638:             blk.70.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  639:             blk.70.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  640:          blk.71.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  641:           blk.71.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  642:           blk.71.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  643:             blk.71.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  644:           blk.71.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  645:             blk.71.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  646:        blk.71.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  647:             blk.71.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  648:             blk.71.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  649:          blk.72.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  650:           blk.72.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  651:           blk.72.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  652:             blk.72.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  653:           blk.72.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  654:             blk.72.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  655:        blk.72.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  656:             blk.72.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  657:             blk.72.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  658:          blk.73.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  659:           blk.73.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  660:           blk.73.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  661:             blk.73.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  662:           blk.73.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  663:             blk.73.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  664:        blk.73.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  665:             blk.73.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  666:             blk.73.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  667:           blk.74.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  668:             blk.74.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  669:        blk.74.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  670:             blk.74.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  671:             blk.74.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  672:          blk.74.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  673:           blk.74.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  674:             blk.74.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  675:           blk.74.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  676:          blk.75.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  677:           blk.75.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  678:           blk.75.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  679:             blk.75.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  680:           blk.75.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  681:             blk.75.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  682:        blk.75.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  683:             blk.75.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  684:             blk.75.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  685:          blk.76.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  686:           blk.76.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  687:           blk.76.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  688:             blk.76.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  689:           blk.76.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  690:             blk.76.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  691:        blk.76.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  692:             blk.76.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  693:             blk.76.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  694:          blk.77.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  695:           blk.77.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  696:           blk.77.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  697:             blk.77.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  698:           blk.77.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  699:             blk.77.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  700:        blk.77.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  701:             blk.77.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  702:             blk.77.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  703:          blk.78.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  704:           blk.78.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  705:           blk.78.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  706:             blk.78.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  707:           blk.78.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  708:             blk.78.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  709:        blk.78.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  710:             blk.78.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  711:             blk.78.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  712:          blk.79.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  713:           blk.79.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  714:           blk.79.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  715:             blk.79.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  716:           blk.79.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  717:             blk.79.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  718:        blk.79.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  719:             blk.79.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  720:             blk.79.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  721:               output_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  722:                    output.weight q6_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - kv   0:                       general.architecture str     \n",
      "llama_model_loader: - kv   1:                               general.name str     \n",
      "llama_model_loader: - kv   2:                       llama.context_length u32     \n",
      "llama_model_loader: - kv   3:                     llama.embedding_length u32     \n",
      "llama_model_loader: - kv   4:                          llama.block_count u32     \n",
      "llama_model_loader: - kv   5:                  llama.feed_forward_length u32     \n",
      "llama_model_loader: - kv   6:                 llama.rope.dimension_count u32     \n",
      "llama_model_loader: - kv   7:                 llama.attention.head_count u32     \n",
      "llama_model_loader: - kv   8:              llama.attention.head_count_kv u32     \n",
      "llama_model_loader: - kv   9:     llama.attention.layer_norm_rms_epsilon f32     \n",
      "llama_model_loader: - kv  10:                          general.file_type u32     \n",
      "llama_model_loader: - kv  11:                       tokenizer.ggml.model str     \n",
      "llama_model_loader: - kv  12:                      tokenizer.ggml.tokens arr     \n",
      "llama_model_loader: - kv  13:                      tokenizer.ggml.scores arr     \n",
      "llama_model_loader: - kv  14:                  tokenizer.ggml.token_type arr     \n",
      "llama_model_loader: - kv  15:                tokenizer.ggml.bos_token_id u32     \n",
      "llama_model_loader: - kv  16:                tokenizer.ggml.eos_token_id u32     \n",
      "llama_model_loader: - kv  17:            tokenizer.ggml.unknown_token_id u32     \n",
      "llama_model_loader: - kv  18:               general.quantization_version u32     \n",
      "llama_model_loader: - type  f32:  161 tensors\n",
      "llama_model_loader: - type q5_K:  481 tensors\n",
      "llama_model_loader: - type q6_K:   81 tensors\n",
      "llm_load_print_meta: format         = GGUF V2 (latest)\n",
      "llm_load_print_meta: arch           = llama\n",
      "llm_load_print_meta: vocab type     = SPM\n",
      "llm_load_print_meta: n_vocab        = 32000\n",
      "llm_load_print_meta: n_merges       = 0\n",
      "llm_load_print_meta: n_ctx_train    = 4096\n",
      "llm_load_print_meta: n_ctx          = 4096\n",
      "llm_load_print_meta: n_embd         = 8192\n",
      "llm_load_print_meta: n_head         = 64\n",
      "llm_load_print_meta: n_head_kv      = 8\n",
      "llm_load_print_meta: n_layer        = 80\n",
      "llm_load_print_meta: n_rot          = 128\n",
      "llm_load_print_meta: n_gqa          = 8\n",
      "llm_load_print_meta: f_norm_eps     = 1.0e-05\n",
      "llm_load_print_meta: f_norm_rms_eps = 1.0e-05\n",
      "llm_load_print_meta: n_ff           = 28672\n",
      "llm_load_print_meta: freq_base      = 10000.0\n",
      "llm_load_print_meta: freq_scale     = 1\n",
      "llm_load_print_meta: model type     = 70B\n",
      "llm_load_print_meta: model ftype    = mostly Q5_K - Medium\n",
      "llm_load_print_meta: model size     = 68.98 B\n",
      "llm_load_print_meta: general.name   = LLaMA v2\n",
      "llm_load_print_meta: BOS token = 1 '<s>'\n",
      "llm_load_print_meta: EOS token = 2 '</s>'\n",
      "llm_load_print_meta: UNK token = 0 '<unk>'\n",
      "llm_load_print_meta: LF token  = 13 '<0x0A>'\n",
      "llm_load_tensors: ggml ctx size =    0.23 MB\n",
      "llm_load_tensors: mem required  = 46494.72 MB (+ 1280.00 MB per state)\n",
      "....................................................................................................\n",
      "llama_new_context_with_model: kv self size  = 1280.00 MB\n",
      "ggml_metal_init: allocating\n",
      "ggml_metal_init: found device: Apple M2 Max\n",
      "ggml_metal_init: picking default device: Apple M2 Max\n",
      "ggml_metal_init: loading '/Users/gburns/miniconda3/envs/alhazen/lib/python3.11/site-packages/llama_cpp/ggml-metal.metal'\n",
      "ggml_metal_init: loaded kernel_add                            0x16c7fdec0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_add_row                        0x16c7ffc10 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul                            0x16c7ff4a0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_row                        0x299404360 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_scale                          0x299404b20 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_silu                           0x299405180 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_relu                           0x16c7fe230 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_gelu                           0x299405e40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_soft_max                       0x2994064c0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_soft_max_4                     0x299406c50 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_diag_mask_inf                  0x299407380 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_diag_mask_inf_8                0x299408260 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_f16                   0x16c7fe490 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_0                  0x299408970 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_1                  0x299409070 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q8_0                  0x299409f00 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q2_K                  0x29940a650 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q3_K                  0x29940acf0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_K                  0x29940b470 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q5_K                  0x29940bbb0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q6_K                  0x29940c2f0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_rms_norm                       0x29940cef0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_norm                           0x29940d260 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32                0x29940dcc0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32_1row           0x29940e540 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32_l4             0x29940eeb0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_0_f32               0x29940f6d0 | th_max =  896 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_1_f32               0x299410000 | th_max =  896 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q8_0_f32               0x299410840 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q2_K_f32               0x299411390 | th_max =  640 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q3_K_f32               0x299411c00 | th_max =  576 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_K_f32               0x299412470 | th_max =  576 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q5_K_f32               0x299412c80 | th_max =  640 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q6_K_f32               0x2994134f0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_f16_f32                 0x299413d20 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_0_f32                0x299414550 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q8_0_f32                0x299414d80 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_1_f32                0x2994155b0 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q2_K_f32                0x299415e30 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q3_K_f32                0x299416660 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_K_f32                0x299416ee0 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q5_K_f32                0x299417710 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q6_K_f32                0x299417f80 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_rope                           0x2994185a0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_alibi_f32                      0x299418f40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f32_f16                    0x2994198b0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f32_f32                    0x29941a500 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f16_f16                    0x29941b010 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: hasUnifiedMemory              = true\n",
      "ggml_metal_init: recommendedMaxWorkingSetSize  = 73728.00 MB\n",
      "ggml_metal_init: maxTransferRate               = built-in GPU\n",
      "llama_new_context_with_model: compute buffer total size =  565.47 MB\n",
      "llama_new_context_with_model: max tensor size =   205.08 MB\n",
      "ggml_metal_add_buffer: allocated 'data            ' buffer, size = 46495.22 MB, (46495.72 / 73728.00)\n",
      "ggml_metal_add_buffer: allocated 'eval            ' buffer, size =     1.48 MB, (46497.20 / 73728.00)\n",
      "ggml_metal_add_buffer: allocated 'kv              ' buffer, size =  1282.00 MB, (47779.20 / 73728.00)\n",
      "AVX = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 0 | NEON = 1 | ARM_FMA = 1 | F16C = 0 | FP16_VA = 1 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 0 | SSSE3 = 0 | VSX = 0 | \n",
      "ggml_metal_add_buffer: allocated 'alloc           ' buffer, size =   564.02 MB, (48343.22 / 73728.00)\n"
     ]
    }
   ],
   "source": [
    "from langchain.llms import LlamaCpp\n",
    "from langchain import PromptTemplate, LLMChain\n",
    "from langchain.callbacks.manager import CallbackManager\n",
    "from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler\n",
    "\n",
    "template = \"\"\"Question: {question}\n",
    "\n",
    "Answer: Let's work this out in a step by step way to be sure we have the right answer.\"\"\"\n",
    "\n",
    "prompt = PromptTemplate(template=template, input_variables=[\"question\"])\n",
    "\n",
    "# Callbacks support token-wise streaming\n",
    "callback_manager = CallbackManager([StreamingStdOutCallbackHandler()])\n",
    "\n",
    "n_gpu_layers = 1  # Change this value based on your model and your GPU VRAM pool.\n",
    "n_batch = 512  # Should be between 1 and n_ctx, consider the amount of VRAM in your GPU.\n",
    "\n",
    "# Make sure the model path is correct for your system!\n",
    "llm = LlamaCpp(\n",
    "    model_path=\"/Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf\",\n",
    "    n_ctx=4096,\n",
    "    n_gpu_layers=n_gpu_layers,\n",
    "    n_batch=n_batch,\n",
    "    callback_manager=callback_manager,\n",
    "    verbose=True, # Verbose is required to pass to the callback manager\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from __future__ import annotations\n",
    "\n",
    "from abc import abstractmethod\n",
    "from typing import List\n",
    "import re\n",
    "\n",
    "from langchain.schema import BaseOutputParser, OutputParserException\n",
    "\n",
    "import json\n",
    "from json import JSONDecodeError\n",
    "from typing import Any, List\n",
    "\n",
    "class JsonEnclosedByTextOutputParser(BaseOutputParser[Any]):\n",
    "    \"\"\"Parse the output of an LLM call to a JSON object.\"\"\"\n",
    "\n",
    "    def parse(self, text: str) -> Any:\n",
    "        text = text.strip()\n",
    "        m = re.search('.*([\\[\\{](.|\\n)*[\\}\\]]).*', text, flags=re.M)\n",
    "        if m:\n",
    "            text1 = m.group(1)\n",
    "            try:\n",
    "                return json.loads(text1)\n",
    "            except JSONDecodeError as e:\n",
    "                return None\n",
    "                #raise OutputParserException(f\"Invalid json output: {text1} derived from {text}\") from e\n",
    "        else: \n",
    "            raise OutputParserException(f\"Could not find json-formatted data in: {text}\")\n",
    "\n",
    "    @property\n",
    "    def _type(self) -> str:\n",
    "        return \"json_enclosed_by_text_output_parser\"\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain import PromptTemplate\n",
    "from langchain.output_parsers import CommaSeparatedListOutputParser\n",
    "\n",
    "output_parser = JsonEnclosedByTextOutputParser()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t = \"\"\"Please identify any imaging techniques or imaging technologies described in the abstract as a list of short noun phrases. \n",
    "\n",
    "The article text is delimited with triple backticks. \n",
    "Format your response as a JSON list. \n",
    "If the information isn't present, use \"none\" as the value.\n",
    "Make your response as short as possible while still being accurate.\n",
    "Only include the method name. \n",
    "Do not include any other response other than the JSON list.\n",
    "\n",
    "Article text: '''{article_text}'''\n",
    "\"\"\"\n",
    "p = PromptTemplate(\n",
    "    template=t,\n",
    "    input_variables=[\"article_text\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t2 = \"\"\"\n",
    "You are an expert biological scientist trained in imaging and microscopy. \n",
    "Your knowledge is deep and wide. You are terse and precise in your language.\n",
    "\n",
    "Your job is to read the title and abstract of a scientific paper in detail, understand what you read, and perform the task described below based on what you read. \n",
    "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
    "\n",
    "First, break down the text of the title and abstract into separate sentences.\n",
    "\n",
    "Second, identify which sentences describe \n",
    "(A) background information about the work, \n",
    "(B) the goals of the work being described, \n",
    "(C) the methods used to achieve the goals, \n",
    "(D) the results of the work, and \n",
    "(E) the conclusions of the work. \n",
    "\n",
    "Third, use only sentences you identify as goals, methods, and results to identify the main contribution of the paper. \n",
    "Record this contribution as a complete noun phrase shorter than 12 words or so as main_contribution in the output.\n",
    "\n",
    "Fourth, decide if that contribution of the paper is developing an imaging technique or method. Be careful about this decision and only answer 'true' \n",
    "if you are confident that the contribution of the paper states that a new imaging technique or method was developed. Possible examples of imaging techniques\n",
    "could be new types of microscope, new imaging techniques, hardware, optics, or sample preparation for imaging. \n",
    "\n",
    "If the main contribution is the development of a new imaging method, record a true value in a variable called is_imaging_method in the output. \n",
    "\n",
    "If the main contribution is not the development of a new imaging method (such as the development of a genetic test, or a finding about a biological system), record a false value in a variable called is_imaging_method in the output. \n",
    "\n",
    "Format your response as a JSON object with \"main_contribution\" and \"is_imaging_method\" as the keys. \n",
    "\n",
    "Do not include any other response other than the JSON object. Remember, stop generating text after you finish the JSON object. Do not provide additional explanations of your answer.\n",
    "\n",
    "Article\n",
    "title: '''{title}'''\n",
    "abstract: '''{abstract}'''\n",
    "\"\"\"\n",
    "p2 = PromptTemplate(\n",
    "    template=t2,\n",
    "    input_variables=[\"title\", \"abstract\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t3 = \"\"\"\n",
    "Your job is to read the title and abstract of a scientific paper and perform the task described below. \n",
    "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
    "\n",
    "First, split the text of the title and abstract into separate sentences.\n",
    "\n",
    "Second, identify which sentences describe \n",
    "(A) background information about the work, \n",
    "(B) the goals of the work being described, \n",
    "(C) the methods used to achieve the goals, \n",
    "(D) the results of the work, and \n",
    "(E) the conclusions of the work. \n",
    "\n",
    "Third, use only sentences you identify as goals, methods, or results and write a description of what the authors did as the \n",
    "main contribution of the paper by saying 'The authors ...' and then describing what they did in a complete sentence shorter than 12 words. \n",
    "\n",
    "Fourth, decide if that contribution is developing an imaging technique or method. Be careful about this decision and only answer 'true' \n",
    "if you are confident that the main goal of the published work was to create or test this new imaging technique or method. Possible examples of imaging techniques\n",
    "could be new types of microscope, new imaging techniques, hardware, optics, or sample preparation for imaging. If the main contribution is the development \n",
    "of a new imaging method, record a true value in a variable called is_imaging_method in the output. \n",
    "\n",
    "Finally, if the paper provides a name for the technique being developed record it in a variable called 'tech_name' in the output.\n",
    "\n",
    "Format your response as a JSON object with \"main_contribution\", \"is_imaging_method\", and \"tech_name\" as the keys.  \n",
    "\n",
    "Stop generating text after you finish the JSON object. Do not provide additional explanations of your answer.\n",
    "\n",
    "Article\n",
    "title: '''{title}'''\n",
    "abstract: '''{abstract}'''\n",
    "\"\"\"\n",
    "p3 = PromptTemplate(\n",
    "    template=t3,\n",
    "    input_variables=[\"title\", \"abstract\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t3 = \"\"\"\n",
    "You are an expert biological scientist trained in imaging and microscopy. \n",
    "Your knowledge is deep and wide. You are terse and precise in your language.\n",
    "\n",
    "Your job is to read the title and abstract of a scientific paper delimited with triple backticks below. \n",
    "\n",
    "First, break down the text of the title and abstract into separate sentences.\n",
    "\n",
    "Second, identify which sentences describe \n",
    "* background information about the work, \n",
    "* the goals of the work being described, \n",
    "* the methods used to achieve the goals, \n",
    "* the results of the work, and \n",
    "* the conclusions of the work. \n",
    "\n",
    "Third, use only sentences you identify as goals, methods, or results and write a description of what the authors did as the main contribution of the paper by \n",
    "saying 'The authors ...' and then describing what they did in a complete sentence. \n",
    "\n",
    "Fourth, decide if the paper os the contribution of the paper is concerned with developing one of three possible types of biomedical imaging technology: probes, acquisition hardware, or data analysis. \n",
    "Be careful about this decision and only include the paper in one of the subtypes if you are confident that the contribution of the paper focuses \n",
    "on one of the specified types of biomedical imaging methods. \n",
    "\n",
    "If the main contribution is the development of molecular probes, stains, antibodies, or any other way of generating a visual signal from a biomedical sample, record a value of 'probe' in the 'technology_type' field of the output.\n",
    "\n",
    "If the main contribution is the development of methods to acquire biomedical images from biomedical samples in new ways (such as methods in microscopy, optics, spectroscopy, etc.), record a value of 'acquisition hardware' in the 'technology_type' field of the output.\n",
    "\n",
    "If the main contribution is the development of data analysis methods for imaging data, record a value of 'data analysis' in the 'technology_type' field of the output.\n",
    "\n",
    "If the main contribution is the development of technology but is not any of these types, record a value of 'other' in the 'technology_type' field of the output.\n",
    "\n",
    "If the main contribution is not the development of imaging technology (such as an experiment describing other phenomenon), record a value of 'none' in the 'technology_type' field of the output.\n",
    "\n",
    "Format your response as a JSON object with \"main_contribution\" and \"technology_type\" as the keys. Do not include any other response other than the JSON object. \n",
    "\n",
    "Remember, stop generating text after you finish the JSON object. Do not provide additional explanations of your answer.\n",
    "\n",
    "Article\n",
    "title: '''{title}'''\n",
    "abstract: '''{abstract}'''\n",
    "\"\"\"\n",
    "p3 = PromptTemplate(\n",
    "    template=t3,\n",
    "    input_variables=[\"title\", \"abstract\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "884a000e75774ffc8df10949defabbec",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Fetching 1 files:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "5cf26872bbd647ad9cee61e13867964c",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Downloading (…)90663ed1/config.json:   0%|          | 0.00/29.0 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "d8bb8290f91f4948957208d811ef39af",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Fetching 0 files: 0it [00:00, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "ename": "ValueError",
     "evalue": "Model file 'llama-2-70b-chat.q4_K_M.gguf' not found in '/Users/gburns/.cache/huggingface/hub/models--TheBloke--Llama-2-70B-chat-GGML/snapshots/2eb277ec3007dcd8e5978ab07dbb235c90663ed1'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[1;32m/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb Cell 15\u001b[0m line \u001b[0;36m4\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m \u001b[39mfrom\u001b[39;00m \u001b[39mctransformers\u001b[39;00m \u001b[39mimport\u001b[39;00m AutoModelForCausalLM\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=2'>3</a>\u001b[0m \u001b[39m# Set gpu_layers to the number of layers to offload to GPU. Set to 0 if no GPU acceleration is available on your system.\u001b[39;00m\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=3'>4</a>\u001b[0m llm \u001b[39m=\u001b[39m AutoModelForCausalLM\u001b[39m.\u001b[39mfrom_pretrained(\u001b[39m\"\u001b[39m\u001b[39mTheBloke/Llama-2-70B-chat-GGML\u001b[39m\u001b[39m\"\u001b[39m, \n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=4'>5</a>\u001b[0m                                            model_file\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mllama-2-70b-chat.q4_K_M.gguf\u001b[39m\u001b[39m\"\u001b[39m, \n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=5'>6</a>\u001b[0m                                            model_type\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mllama\u001b[39m\u001b[39m\"\u001b[39m, \n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=6'>7</a>\u001b[0m                                            gpu_layers\u001b[39m=\u001b[39m\u001b[39m50\u001b[39m)\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nbs/93_llama_cpp.ipynb#X54sZmlsZQ%3D%3D?line=8'>9</a>\u001b[0m \u001b[39mprint\u001b[39m(llm(\u001b[39m\"\u001b[39m\u001b[39mAI is going to\u001b[39m\u001b[39m\"\u001b[39m))\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/site-packages/ctransformers/hub.py:168\u001b[0m, in \u001b[0;36mAutoModelForCausalLM.from_pretrained\u001b[0;34m(cls, model_path_or_repo_id, model_type, model_file, config, lib, local_files_only, revision, hf, **kwargs)\u001b[0m\n\u001b[1;32m    164\u001b[0m     model_path \u001b[39m=\u001b[39m \u001b[39mcls\u001b[39m\u001b[39m.\u001b[39m_find_model_path_from_dir(\n\u001b[1;32m    165\u001b[0m         model_path_or_repo_id, model_file\n\u001b[1;32m    166\u001b[0m     )\n\u001b[1;32m    167\u001b[0m \u001b[39melif\u001b[39;00m path_type \u001b[39m==\u001b[39m \u001b[39m\"\u001b[39m\u001b[39mrepo\u001b[39m\u001b[39m\"\u001b[39m:\n\u001b[0;32m--> 168\u001b[0m     model_path \u001b[39m=\u001b[39m \u001b[39mcls\u001b[39m\u001b[39m.\u001b[39m_find_model_path_from_repo(\n\u001b[1;32m    169\u001b[0m         model_path_or_repo_id,\n\u001b[1;32m    170\u001b[0m         model_file,\n\u001b[1;32m    171\u001b[0m         local_files_only\u001b[39m=\u001b[39mlocal_files_only,\n\u001b[1;32m    172\u001b[0m         revision\u001b[39m=\u001b[39mrevision,\n\u001b[1;32m    173\u001b[0m     )\n\u001b[1;32m    175\u001b[0m llm \u001b[39m=\u001b[39m LLM(\n\u001b[1;32m    176\u001b[0m     model_path\u001b[39m=\u001b[39mmodel_path,\n\u001b[1;32m    177\u001b[0m     model_type\u001b[39m=\u001b[39mmodel_type,\n\u001b[1;32m    178\u001b[0m     config\u001b[39m=\u001b[39mconfig\u001b[39m.\u001b[39mconfig,\n\u001b[1;32m    179\u001b[0m     lib\u001b[39m=\u001b[39mlib,\n\u001b[1;32m    180\u001b[0m )\n\u001b[1;32m    181\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mnot\u001b[39;00m hf:\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/site-packages/ctransformers/hub.py:209\u001b[0m, in \u001b[0;36mAutoModelForCausalLM._find_model_path_from_repo\u001b[0;34m(cls, repo_id, filename, local_files_only, revision)\u001b[0m\n\u001b[1;32m    202\u001b[0m allow_patterns \u001b[39m=\u001b[39m filename \u001b[39mor\u001b[39;00m [\u001b[39m\"\u001b[39m\u001b[39m*.bin\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39m*.gguf\u001b[39m\u001b[39m\"\u001b[39m]\n\u001b[1;32m    203\u001b[0m path \u001b[39m=\u001b[39m snapshot_download(\n\u001b[1;32m    204\u001b[0m     repo_id\u001b[39m=\u001b[39mrepo_id,\n\u001b[1;32m    205\u001b[0m     allow_patterns\u001b[39m=\u001b[39mallow_patterns,\n\u001b[1;32m    206\u001b[0m     local_files_only\u001b[39m=\u001b[39mlocal_files_only,\n\u001b[1;32m    207\u001b[0m     revision\u001b[39m=\u001b[39mrevision,\n\u001b[1;32m    208\u001b[0m )\n\u001b[0;32m--> 209\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mcls\u001b[39m\u001b[39m.\u001b[39m_find_model_path_from_dir(path, filename\u001b[39m=\u001b[39mfilename)\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/site-packages/ctransformers/hub.py:242\u001b[0m, in \u001b[0;36mAutoModelForCausalLM._find_model_path_from_dir\u001b[0;34m(cls, path, filename)\u001b[0m\n\u001b[1;32m    240\u001b[0m     file \u001b[39m=\u001b[39m (path \u001b[39m/\u001b[39m filename)\u001b[39m.\u001b[39mresolve()\n\u001b[1;32m    241\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mnot\u001b[39;00m file\u001b[39m.\u001b[39mis_file():\n\u001b[0;32m--> 242\u001b[0m         \u001b[39mraise\u001b[39;00m \u001b[39mValueError\u001b[39;00m(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mModel file \u001b[39m\u001b[39m'\u001b[39m\u001b[39m{\u001b[39;00mfilename\u001b[39m}\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m not found in \u001b[39m\u001b[39m'\u001b[39m\u001b[39m{\u001b[39;00mpath\u001b[39m}\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[1;32m    243\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mstr\u001b[39m(file)\n\u001b[1;32m    245\u001b[0m files \u001b[39m=\u001b[39m [\n\u001b[1;32m    246\u001b[0m     (f\u001b[39m.\u001b[39mstat()\u001b[39m.\u001b[39mst_size, f)\n\u001b[1;32m    247\u001b[0m     \u001b[39mfor\u001b[39;00m f \u001b[39min\u001b[39;00m path\u001b[39m.\u001b[39miterdir()\n\u001b[1;32m    248\u001b[0m     \u001b[39mif\u001b[39;00m f\u001b[39m.\u001b[39mis_file() \u001b[39mand\u001b[39;00m (f\u001b[39m.\u001b[39mname\u001b[39m.\u001b[39mendswith(\u001b[39m\"\u001b[39m\u001b[39m.bin\u001b[39m\u001b[39m\"\u001b[39m) \u001b[39mor\u001b[39;00m f\u001b[39m.\u001b[39mname\u001b[39m.\u001b[39mendswith(\u001b[39m\"\u001b[39m\u001b[39m.gguf\u001b[39m\u001b[39m\"\u001b[39m))\n\u001b[1;32m    249\u001b[0m ]\n",
      "\u001b[0;31mValueError\u001b[0m: Model file 'llama-2-70b-chat.q4_K_M.gguf' not found in '/Users/gburns/.cache/huggingface/hub/models--TheBloke--Llama-2-70B-chat-GGML/snapshots/2eb277ec3007dcd8e5978ab07dbb235c90663ed1'"
     ]
    }
   ],
   "source": [
    "from ctransformers import AutoModelForCausalLM\n",
    "\n",
    "# Set gpu_layers to the number of layers to offload to GPU. Set to 0 if no GPU acceleration is available on your system.\n",
    "llm = AutoModelForCausalLM.from_pretrained(\"TheBloke/Llama-2-70B-chat-GGML\", \n",
    "                                           model_file=\"llama-2-70b-chat.q4_K_M.gguf\", \n",
    "                                           model_type=\"llama\", \n",
    "                                           gpu_layers=50)\n",
    "\n",
    "print(llm(\"AI is going to\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "73a74e873eaa4828ad8b6e2495a40d1b",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Fetching 1 files:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "56c50a15832443dca56b2952bf696844",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Fetching 1 files:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "80dced2e3e7e435285d608d03bb22609",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Downloading (…)chat.ggmlv3.q2_K.bin:   0%|          | 0.00/28.6G [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "from langchain.llms import CTransformers\n",
    "\n",
    "llm = CTransformers(model=\"TheBloke/Llama-2-70B-chat-GGML\")\n",
    "\n",
    "print(llm('AI is going to'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "You are an expert biological scientist trained in imaging and microscopy. \n",
      "Your knowledge is deep and wide. You are terse and precise in your language.\n",
      "\n",
      "Your job is to read the title and abstract of a scientific paper in detail, understand what you read, and perform the task described below based on what you read. \n",
      "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
      "\n",
      "First, break down the text of the title and abstract into separate sentences.\n",
      "\n",
      "Second, identify which sentences describe \n",
      "(A) background information about the work, \n",
      "(B) the goals of the work being described, \n",
      "(C) the methods used to achieve the goals, \n",
      "(D) the results of the work, and \n",
      "(E) the conclusions of the work. \n",
      "\n",
      "Third, use only sentences you identify as goals, methods, and results to identify the main contribution of the paper. \n",
      "Record this contribution as a complete noun phrase shorter than 12 words or so as main_contribution in the output.\n",
      "\n",
      "Fourth, decide if that contribution of the paper is developing an imaging technique or method. Be careful about this decision and only answer 'true' \n",
      "if you are confident that the contribution of the paper states that a new imaging technique or method was developed. Possible examples of imaging techniques\n",
      "could be new types of microscope, new imaging techniques, hardware, optics, or sample preparation for imaging. \n",
      "\n",
      "If the main contribution is the development of a new imaging method, record a true value in a variable called is_imaging_method in the output. \n",
      "\n",
      "If the main contribution is not the development of a new imaging method (such as the development of a genetic test, or a finding about a biological system), record a false value in a variable called is_imaging_method in the output. \n",
      "\n",
      "Format your response as a JSON object with \"main_contribution\" and \"is_imaging_method\" as the keys. \n",
      "\n",
      "Do not include any other response other than the JSON object. Remember, stop generating text after you finish the JSON object. Do not provide additional explanations of your answer.\n",
      "\n",
      "Article\n",
      "title: '''Multiplexing molecular tension sensors reveals piconewton force gradient across talin-1.'''\n",
      "abstract: '''Förster resonance energy transfer (FRET)-based tension sensor modules (TSMs) are available for investigating how distinct proteins bear mechanical forces in cells. Yet, forces in the single piconewton (pN) regime remain difficult to resolve, and tools for multiplexed tension sensing are lacking. Here, we report the generation and calibration of a genetically encoded, FRET-based biosensor called FL-TSM, which is characterized by a near-digital force response and increased sensitivity at 3-5 pN. In addition, we present a method allowing the simultaneous evaluation of coexpressed tension sensor constructs using two-color fluorescence lifetime microscopy. Finally, we introduce a procedure to calculate the fraction of mechanically engaged molecules within cells. Application of these techniques to new talin biosensors reveals an intramolecular tension gradient across talin-1 that is established upon integrin-mediated cell adhesion. The tension gradient is actomyosin- and vinculin-dependent and sensitive to the rigidity of the extracellular environment.'''\n",
      "\n",
      "Please process this article.Please process this article.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "llama_print_timings:        load time = 11699.64 ms\n",
      "llama_print_timings:      sample time =    14.70 ms /     6 runs   (    2.45 ms per token,   408.19 tokens per second)\n",
      "llama_print_timings: prompt eval time = 17669.69 ms /   777 tokens (   22.74 ms per token,    43.97 tokens per second)\n",
      "llama_print_timings:        eval time =   838.51 ms /     5 runs   (  167.70 ms per token,     5.96 tokens per second)\n",
      "llama_print_timings:       total time = 18558.42 ms\n"
     ]
    }
   ],
   "source": [
    "# check\n",
    "r = df[df.ID_PAPER==\"28945706\"]\n",
    "a = r.ABSTRACT.values[0]\n",
    "t = r.TITLE.values[0]\n",
    "prompt = p2.format(title=t, abstract=a)\n",
    "print(prompt)\n",
    "output = llm(p2.format(title=t, abstract=a))\n",
    "#m = output_parser.parse(output) \n",
    "print(output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('Förster resonance energy transfer (FRET)-based tension sensor modules (TSMs) '\n",
      " 'are available for investigating how distinct proteins bear mechanical forces '\n",
      " 'in cells. Yet, forces in the single piconewton (pN) regime remain difficult '\n",
      " 'to resolve, and tools for multiplexed tension sensing are lacking. Here, we '\n",
      " 'report the generation and calibration of a genetically encoded, FRET-based '\n",
      " 'biosensor called FL-TSM, which is characterized by a near-digital force '\n",
      " 'response and increased sensitivity at 3-5 pN. In addition, we present a '\n",
      " 'method allowing the simultaneous evaluation of coexpressed tension sensor '\n",
      " 'constructs using two-color fluorescence lifetime microscopy. Finally, we '\n",
      " 'introduce a procedure to calculate the fraction of mechanically engaged '\n",
      " 'molecules within cells. Application of these techniques to new talin '\n",
      " 'biosensors reveals an intramolecular tension gradient across talin-1 that is '\n",
      " 'established upon integrin-mediated cell adhesion. The tension gradient is '\n",
      " 'actomyosin- and vinculin-dependent and sensitive to the rigidity of the '\n",
      " 'extracellular environment.')\n"
     ]
    }
   ],
   "source": [
    "pprint(a)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "701\n",
      "19898486\n",
      "Optical interrogation of neural circuits in Caenorhabditis elegans.\n",
      "The nematode Caenorhabditis elegans has a compact nervous system with only 302 neurons. Whereas most\n",
      "of the synaptic connections between these neurons have been identified by electron microscopy serial\n",
      "reconstructions, functional connections have been inferred between only a few neurons through\n",
      "combinations of electrophysiology, cell ablation, in vivo calcium imaging and genetic analysis. To\n",
      "map functional connections between neurons, we combined in vivo optical stimulation with\n",
      "simultaneous calcium imaging. We analyzed the connections from the ASH sensory neurons and RIM\n",
      "interneurons to the command interneurons AVA and AVD. Stimulation of ASH or RIM neurons using\n",
      "channelrhodopsin-2 (ChR2) resulted in activation of AVA neurons, evoking an avoidance behavior. Our\n",
      "results demonstrate that we can excite specific neurons expressing ChR2 while simultaneously\n",
      "monitoring G-CaMP fluorescence in several other neurons, making it possible to rapidly decipher\n",
      "functional connections in C. elegans neural circuits.\n",
      " {\n",
      "\"main_contribution\": \"demonstrated the ability to excite specific neurons expressing ChR2 while simultaneously monitoring G-CaMP fluorescence in several other neurons, making it possible to rapidly decipher functional connections in C. elegans neural circuits\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows for the selective excitation of specific neurons while simultaneously monitoring the activity of other neurons in the Caenorhabditis elegans nervous system. This technique, which utilizes channelrhodopsin-2 (ChR2) and G-CaMP fluorescence, enables the rapid deciphering of functional connections within neural circuits. Therefore, the main contribution is classified as \"true\" for the variable is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "702\n",
      "19898487\n",
      "Automated high-throughput mapping of promoter-enhancer interactions in zebrafish embryos.\n",
      "Zebrafish embryos offer a unique combination of high-throughput capabilities and the complexity of\n",
      "the vertebrate animal for a variety of phenotypic screening applications. However, there is a need\n",
      "for automation of imaging technologies to exploit the potential of the transparent embryo. Here we\n",
      "report a high-throughput pipeline for registering domain-specific reporter expression in zebrafish\n",
      "embryos with the aim of mapping the interactions between cis-regulatory modules and core promoters.\n",
      "Automated microscopy coupled with custom-built embryo detection and segmentation software allowed\n",
      "the spatial registration of reporter activity for 202 enhancer-promoter combinations, based on\n",
      "images of thousands of embryos. The diversity of promoter-enhancer interaction specificities\n",
      "underscores the importance of the core promoter sequence in cis-regulatory interactions and provides\n",
      "a promoter resource for transgenic reporter studies. The technology described here is also suitable\n",
      "for the spatial analysis of fluorescence readouts in genetic, pharmaceutical or toxicological\n",
      "screens.\n",
      " {\n",
      "\"main_contribution\": \"Automated high-throughput mapping of promoter-enhancer interactions in zebrafish embryos\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a high-throughput pipeline for registering domain-specific reporter expression in zebrafish embryos, which includes automated microscopy and custom-built embryo detection and segmentation software. This imaging technique allows for the spatial registration of reporter activity for hundreds of enhancer-promoter combinations, providing a promoter resource for transgenic reporter studies and suitable for spatial analysis of fluorescence readouts in genetic, pharmaceutical or toxicological screens. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "703\n",
      "19915559\n",
      "Transgenic microRNA inhibition with spatiotemporal specificity in intact organisms.\n",
      "MicroRNAs are important regulators of gene expression, yet the functional outputs of most microRNA-\n",
      "target interactions remain elusive. Here we introduce the Drosophila melanogaster microRNA sponge\n",
      "(miR-SP) as a powerful transgenic technology to dissect the function of every microRNA with precise\n",
      "spatiotemporal resolution. miR-SPs can be used to characterize tissue-specific microRNA loss-of-\n",
      "function phenotypes, define the spatial regulation of their effectors and uncover interactions\n",
      "between microRNAs and other genes. Using themiR-SP system, we identified an essential role of the\n",
      "conserved microRNA miR-8, in neuromuscular junction formation. Tissue-specific silencing revealed\n",
      "that postsynaptic activity of miR-8 is important for normal neuromuscular junction morphogenesis.\n",
      "Given that miR-SPs rely on a bipartite modular expression system, they could be used to elucidate\n",
      "the endogenous function of microRNAs in any species in which conditional expression can be achieved.\n",
      " {\n",
      "\"main_contribution\": \"miR-SP system for transgenic microRNA inhibition with spatiotemporal specificity\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new transgenic technology called the Drosophila melanogaster microRNA sponge (miR-SP) that can be used to dissect the function of every microRNA with precise spatiotemporal resolution. The main contribution of the paper is the development of this system, which allows for tissue-specific silencing of microRNAs and can be used to elucidate the endogenous function of microRNAs in any species in which conditional expression can be achieved. Therefore, the main contribution is not an imaging method, and the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "704\n",
      "19966808\n",
      "Floxin, a resource for genetically engineering mouse ESCs.\n",
      "We describe a method for the highly efficient and precise targeted modification of gene trap loci in\n",
      "mouse embryonic stem cells (ESCs). Through the Floxin method, gene trap mutations were reverted and\n",
      "new DNA sequences inserted using Cre recombinase and a shuttle vector, pFloxin. Floxin technology is\n",
      "applicable to the existing collection of 24,149 compatible gene trap cell lines, which should enable\n",
      "high-throughput modification of many genes in mouse ESCs.\n",
      " {\n",
      "\"main_contribution\": \"Floxin technology for targeted modification of gene trap loci in mouse ESCs\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "Sentences identified as background information: None\n",
      "\n",
      "Sentences identified as goals: \"We describe a method for the highly efficient and precise targeted modification of gene trap loci in mouse embryonic stem cells (ESCs).\"\n",
      "\n",
      "Sentences identified as methods: \"Through the Floxin method, gene trap mutations were reverted and new DNA sequences inserted using Cre recombinase and a shuttle vector, pFloxin.\"\n",
      "\n",
      "Sentences identified as results: \"Floxin technology is applicable to the existing collection of 24,149 compatible gene trap cell lines, which should enable high-throughput modification of many genes in mouse ESCs.\"\n",
      "\n",
      "Main contribution: The main contribution of the paper is the development of Floxin technology for targeted modification of gene trap loci in mouse ESCs.\n",
      "\n",
      "Is imaging method: False. The paper does not describe the development of a new imaging technique or method, but rather a method for genetic engineering mouse ESCs. Therefore, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "705\n",
      "19966809\n",
      "Engineering a polarity-sensitive biosensor for time-lapse imaging of apoptotic processes and\n",
      "degeneration.\n",
      "Apoptosis is of central importance to many areas of biological research, but there is a lack of\n",
      "methods that permit continuous monitoring of apoptosis or cell viability in a nontoxic and\n",
      "noninvasive manner. Here we report the development of a tool applicable to live-cell imaging that\n",
      "facilitates the visualization of real-time apoptotic changes without perturbing the cellular\n",
      "environment. We designed a polarity-sensitive annexin-based biosensor (pSIVA) with switchable\n",
      "fluorescence states, which allows detection only when bound to apoptotic cells. Using pSIVA with\n",
      "live-cell imaging, we observed dynamic local changes in individual rat neurons during degeneration\n",
      "in vitro and in vivo. Furthermore, we observed that pSIVA binding was reversible and clearly defined\n",
      "the critical period for neurons to be rescued. We anticipate pSIVA can be widely applied to address\n",
      "questions concerning spatiotemporal events in apoptotic processes, its reversibility and the general\n",
      "viability of cells in culture.\n",
      " {\n",
      "\"main_contribution\": \"A tool for live-cell imaging that facilitates the visualization of real-time apoptotic changes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new tool for live-cell imaging that allows for the continuous monitoring of apoptosis or cell viability in a nontoxic and noninvasive manner. The tool is based on a polarity-sensitive annexin-based biosensor (pSIVA) with switchable fluorescence states, which only detects apoptotic cells. The authors used this tool to observe dynamic local changes in individual rat neurons during degeneration in vitro and in vivo.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new imaging technique or method, specifically the pSIVA biosensor, which enables real-time imaging of apoptotic processes without perturbing the cellular environment. This is a true value for the variable is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "706\n",
      "20010832\n",
      "A reprogrammable mouse strain from gene-targeted embryonic stem cells.\n",
      "The derivation of induced pluripotent stem cells (iPSCs) usually involves the viral introduction of\n",
      "reprogramming factors into somatic cells. Here we used gene targeting to generate a mouse strain\n",
      "with a single copy of an inducible, polycistronic reprogramming cassette, allowing for the induction\n",
      "of pluripotency in various somatic cell types. As these 'reprogrammable mice' can be easily bred,\n",
      "they are a useful tool to study the mechanisms underlying cellular reprogramming.\n",
      " {\n",
      "\"main_contribution\": \"A reprogrammable mouse strain from gene-targeted embryonic stem cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new mouse strain that can be easily reprogrammed to study the mechanisms underlying cellular reprogramming. The main contribution of the paper is the creation of this mouse strain, which is not an imaging method or technique. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "707\n",
      "20037591\n",
      "Chronic microsensors for longitudinal, subsecond dopamine detection in behaving animals.\n",
      "Neurotransmission operates on a millisecond timescale but is changed by normal experience or\n",
      "neuropathology over days to months. Despite the importance of long-term neurotransmitter dynamics,\n",
      "no technique exists to track these changes in a subject from day to day over extended periods of\n",
      "time. Here we describe and characterize a microsensor that can detect the neurotransmitter dopamine\n",
      "with subsecond temporal resolution over months in vivo in rats and mice.\n",
      " {\n",
      "\"main_contribution\": \"Chronic microsensors for longitudinal, subsecond dopamine detection in behaving animals\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, specifically a microsensor that can detect the neurotransmitter dopamine with subsecond temporal resolution over months in vivo in rats and mice. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "708\n",
      "20037592\n",
      "Adaptive optics via pupil segmentation for high-resolution imaging in biological tissues.\n",
      "Biological specimens are rife with optical inhomogeneities that seriously degrade imaging\n",
      "performance under all but the most ideal conditions. Measuring and then correcting for these\n",
      "inhomogeneities is the province of adaptive optics. Here we introduce an approach to adaptive optics\n",
      "in microscopy wherein the rear pupil of an objective lens is segmented into subregions, and light is\n",
      "directed individually to each subregion to measure, by image shift, the deflection faced by each\n",
      "group of rays as they emerge from the objective and travel through the specimen toward the focus.\n",
      "Applying our method to two-photon microscopy, we could recover near-diffraction-limited performance\n",
      "from a variety of biological and nonbiological samples exhibiting aberrations large or small and\n",
      "smoothly varying or abruptly changing. In particular, results from fixed mouse cortical slices\n",
      "illustrate our ability to improve signal and resolution to depths of 400 microm.\n",
      " {\n",
      "\"main_contribution\": \"Adaptive optics via pupil segmentation for high-resolution imaging in biological tissues\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "709\n",
      "20154677\n",
      "Temporal pixel multiplexing for simultaneous high-speed, high-resolution imaging.\n",
      "We introduce an imaging modality that, by offsetting pixel-exposure times during capture of a single\n",
      "image frame, embeds temporal information in each frame. This allows simultaneous acquisition of\n",
      "full-resolution images at native detector frame rates and high-speed image sequences at reduced\n",
      "resolution, without increasing bandwidth requirements. We demonstrate this method using macroscopic\n",
      "and microscopic examples, including imaging calcium transients in heart cells at 250 Hz using a\n",
      "10-Hz megapixel camera.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous high-speed, high-resolution imaging using temporal pixel multiplexing\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "710\n",
      "20190760\n",
      "Protein folding stability and dynamics imaged in a living cell.\n",
      "Biomolecular dynamics and stability are predominantly investigated in vitro and extrapolated to\n",
      "explain function in the living cell. We present fast relaxation imaging (FreI), which combines\n",
      "fluorescence microscopy and temperature jumps to probe biomolecular dynamics and stability inside a\n",
      "single living cell with high spatiotemporal resolution. We demonstrated the method by measuring the\n",
      "reversible fast folding kinetics as well as folding thermodynamics of a fluorescence resonance\n",
      "energy transfer (FRET) probe-labeled phosphoglycerate kinase construct in two human cell lines.\n",
      "Comparison with in vitro experiments at 23-49 degrees C showed that the cell environment influences\n",
      "protein stability and folding rate. FReI should also be applicable to the study of protein-protein\n",
      "interactions and heat-shock responses as well as to comparative studies of cell populations or whole\n",
      "organisms.\n",
      " {\n",
      "\"main_contribution\": \"Fast relaxation imaging (FreI) method that combines fluorescence microscopy and temperature jumps to probe biomolecular dynamics and stability inside a single living cell with high spatiotemporal resolution.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called Fast Relaxation Imaging (FreI), which allows for the study of biomolecular dynamics and stability inside a single living cell with high spatiotemporal resolution. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "711\n",
      "20195255\n",
      "Visualization of image data from cells to organisms.\n",
      "Advances in imaging techniques and high-throughput technologies are providing scientists with\n",
      "unprecedented possibilities to visualize internal structures of cells, organs and organisms and to\n",
      "collect systematic image data characterizing genes and proteins on a large scale. To make the best\n",
      "use of these increasingly complex and large image data resources, the scientific community must be\n",
      "provided with methods to query, analyze and crosslink these resources to give an intuitive visual\n",
      "representation of the data. This review gives an overview of existing methods and tools for this\n",
      "purpose and highlights some of their limitations and challenges.\n",
      " {\n",
      "\"main_contribution\": \"Visualization of image data from cells to organisms\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method for visualizing internal structures of cells, organs, and organisms. The paper reviews existing methods and tools for this purpose and highlights their limitations and challenges. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "712\n",
      "20228812\n",
      "Identifying single-cell molecular programs by stochastic profiling.\n",
      "Cells in tissues can be morphologically indistinguishable yet show molecular expression patterns\n",
      "that are remarkably heterogeneous. Here we describe an approach to comprehensively identify co-\n",
      "regulated, heterogeneously expressed genes among cells that otherwise appear identical. The\n",
      "technique, called stochastic profiling, involves repeated, random selection of very small cell\n",
      "populations via laser-capture microdissection followed by a customized single-cell amplification\n",
      "procedure and transcriptional profiling. Fluctuations in the resulting gene-expression measurements\n",
      "are then analyzed statistically to identify transcripts that are heterogeneously coexpressed. We\n",
      "stochastically profiled matrix-attached human epithelial cells in a three-dimensional culture model\n",
      "of mammary-acinar morphogenesis. Of 4,557 transcripts, we identified 547 genes with strong cell-to-\n",
      "cell expression differences. Clustering of this heterogeneous subset revealed several molecular\n",
      "'programs' implicated in protein biosynthesis, oxidative-stress responses and NF-kappaB signaling,\n",
      "which we independently confirmed by RNA fluorescence in situ hybridization. Thus, stochastic\n",
      "profiling can reveal single-cell heterogeneities without the need to measure expression in\n",
      "individual cells.\n",
      " {\n",
      "\"main_contribution\": \"identifying co-regulated, heterogeneously expressed genes among cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is identifying co-regulated, heterogeneously expressed genes among cells using a technique called stochastic profiling. This technique involves repeated, random selection of very small cell populations via laser-capture microdissection followed by a customized single-cell amplification procedure and transcriptional profiling. The fluctuations in the resulting gene-expression measurements are then analyzed statistically to identify transcripts that are heterogeneously coexpressed. Therefore, the main contribution is not the development of a new imaging method, and hence is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "713\n",
      "20228813\n",
      "Fluorescence fluctuations of quantum-dot sensors capture intracellular protein interaction dynamics.\n",
      "We extend the in vitro principle of co-immunoprecipitation to quantify dynamic protein interactions\n",
      "in living cells. Using a multiresolution implementation of fluorescence correlation spectroscopy to\n",
      "achieve maximal temporal resolution, we monitored the interactions of endogenous bait proteins,\n",
      "recruited by quantum dots, with fluorescently tagged prey. With this approach, we analyzed the rapid\n",
      "physiological regulation of protein kinase A.\n",
      " {\n",
      "\"main_contribution\": \"Fluorescence fluctuations of quantum-dot sensors capture intracellular protein interaction dynamics\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method using fluorescence correlation spectroscopy to monitor dynamic protein interactions in living cells. This method utilizes quantum dots to recruit endogenous bait proteins and fluorescently tagged prey, allowing for the analysis of rapid physiological regulation of protein kinase A. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "714\n",
      "20228814\n",
      "In vivo wide-area cellular imaging by side-view endomicroscopy.\n",
      "In vivo imaging of small animals offers several possibilities for studying normal and disease\n",
      "biology, but visualizing organs with single-cell resolution is challenging. We describe rotational\n",
      "side-view confocal endomicroscopy, which enables cellular imaging of gastrointestinal and\n",
      "respiratory tracts in mice and may be extensible to imaging organ parenchyma such as cerebral\n",
      "cortex. We monitored cell infiltration, vascular changes and tumor progression during inflammation\n",
      "and tumorigenesis in colon over several months.\n",
      " {\n",
      "\"main_contribution\": \"In vivo wide-area cellular imaging by side-view endomicroscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called rotational side-view confocal endomicroscopy, which enables cellular imaging of gastrointestinal and respiratory tracts in mice. This technique has the potential to be extended for imaging organ parenchyma such as cerebral cortex. Therefore, the main contribution is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "715\n",
      "20228815\n",
      "Imaging type-III secretion reveals dynamics and spatial segregation of Salmonella effectors.\n",
      "The type-III secretion system (T3SS) enables gram-negative bacteria to inject effector proteins into\n",
      "eukaryotic host cells. Upon entry, T3SS effectors work cooperatively to reprogram host cells,\n",
      "enabling bacterial survival. Progress in understanding when and where effectors localize in host\n",
      "cells has been hindered by a dearth of tools to study these proteins in the native cellular\n",
      "environment. We report a method to label and track T3SS effectors during infection using a split-GFP\n",
      "system. We demonstrate this technique by labeling three effectors from Salmonella enterica (PipB2,\n",
      "SteA and SteC) and characterizing their localization in host cells. PipB2 displayed highly dynamic\n",
      "behavior on tubules emanating from the Salmonella-containing vacuole labeled with both endo- and\n",
      "exocytic markers. SteA was preferentially enriched on tubules localizing with Golgi markers. This\n",
      "segregation suggests that effector targeting and localization may have a functional role during\n",
      "infection.\n",
      " {\n",
      "\"main_contribution\": \"The development of a method to label and track T3SS effectors during infection using a split-GFP system.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a method to label and track T3SS effectors during infection using a split-GFP system. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "716\n",
      "20354514\n",
      "Laser tricks without labels.\n",
      "Nonlinear optical microscopy lets researchers see chemical composition in living cells and\n",
      "organisms.\n",
      " {\n",
      "\"main_contribution\": \"nonlinear optical microscopy to see chemical composition in living cells and organisms\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "717\n",
      "20364146\n",
      "Fast, single-molecule localization that achieves theoretically minimum uncertainty.\n",
      "We describe an iterative algorithm that converges to the maximum likelihood estimate of the position\n",
      "and intensity of a single fluorophore. Our technique efficiently computes and achieves the Cramér-\n",
      "Rao lower bound, an essential tool for parameter estimation. An implementation of the algorithm on\n",
      "graphics processing unit hardware achieved more than 10(5) combined fits and Cramér-Rao lower bound\n",
      "calculations per second, enabling real-time data analysis for super-resolution imaging and other\n",
      "applications.\n",
      " {\n",
      "\"main_contribution\": \"iterative algorithm for fast, single-molecule localization\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the main contribution of the paper as an iterative algorithm for achieving theoretically minimum uncertainty in single-molecule localization. The abstract further explains that this technique efficiently computes and achieves the Cramér-Rao lower bound, which is essential for parameter estimation. Additionally, the authors implemented the algorithm on graphics processing unit hardware, enabling real-time data analysis for super-resolution imaging and other applications.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new imaging method, specifically an iterative algorithm for fast, single-molecule localization. This method improves upon existing techniques by achieving the Cramér-Rao lower bound, which is a theoretical limit on the accuracy of parameter estimation. As such, the contribution is classified as \"true\" for is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "718\n",
      "20364147\n",
      "Optimized localization analysis for single-molecule tracking and super-resolution microscopy.\n",
      "We optimally localized isolated fluorescent beads and molecules imaged as diffraction-limited spots,\n",
      "determined the orientation of molecules and present reliable formulas for the precision of various\n",
      "localization methods. Both theory and experimental data showed that unweighted least-squares fitting\n",
      "of a Gaussian squanders one-third of the available information, a popular formula for its precision\n",
      "exaggerates beyond Fisher's information limit, and weighted least-squares may do worse, whereas\n",
      "maximum-likelihood fitting is practically optimal.\n",
      " {\n",
      "\"main_contribution\": \"Optimized localization analysis for single-molecule tracking and super-resolution microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically an optimized localization analysis method for single-molecule tracking and super-resolution microscopy. This is described in the abstract as \"optimally localized isolated fluorescent beads and molecules imaged as diffraction-limited spots\". Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "719\n",
      "20383133\n",
      "A streptavidin variant with slower biotin dissociation and increased mechanostability.\n",
      "Streptavidin binds biotin conjugates with exceptional stability but dissociation does occur,\n",
      "limiting its use in imaging, DNA amplification and nanotechnology. We identified a mutant\n",
      "streptavidin, traptavidin, with more than tenfold slower biotin dissociation, increased mechanical\n",
      "strength and improved thermostability; this resilience should enable diverse applications. FtsK, a\n",
      "motor protein important in chromosome segregation, rapidly displaced streptavidin from biotinylated\n",
      "DNA, whereas traptavidin resisted displacement, indicating the force generated by Ftsk\n",
      "translocation.\n",
      " {\n",
      "\"main_contribution\": \"A streptavidin variant with slower biotin dissociation and increased mechanostability\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a streptavidin variant with slower biotin dissociation and increased mechanostability. This variant can be used in various applications such as imaging, DNA amplification, and nanotechnology. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "720\n",
      "20400966\n",
      "High-speed in vivo calcium imaging reveals neuronal network activity with near-millisecond\n",
      "precision.\n",
      "Two-photon calcium imaging of neuronal populations enables optical recording of spiking activity in\n",
      "living animals, but standard laser scanners are too slow to accurately determine spike times. Here\n",
      "we report in vivo imaging in mouse neocortex with greatly improved temporal resolution using random-\n",
      "access scanning with acousto-optic deflectors. We obtained fluorescence measurements from 34-91\n",
      "layer 2/3 neurons at a 180-490 Hz sampling rate. We detected single action potential-evoked calcium\n",
      "transients with signal-to-noise ratios of 2-5 and determined spike times with near-millisecond\n",
      "precision and 5-15 ms confidence intervals. An automated 'peeling' algorithm enabled reconstruction\n",
      "of complex spike trains from fluorescence traces up to 20-30 Hz frequency, uncovering spatiotemporal\n",
      "trial-to-trial variability of sensory responses in barrel cortex and visual cortex. By revealing\n",
      "spike sequences in neuronal populations on a fast time scale, high-speed calcium imaging will\n",
      "facilitate optical studies of information processing in brain microcircuits.\n",
      " {\n",
      "\"main_contribution\": \"High-speed in vivo calcium imaging reveals neuronal network activity with near-millisecond precision.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a high-speed in vivo calcium imaging technique that enables the optical recording of spiking activity in living animals with near-millisecond precision. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "721\n",
      "20418868\n",
      "Targeted gene deletions in C. elegans using transposon excision.\n",
      "We developed a method, MosDEL, to generate targeted knockouts of genes in Caenorhabditis elegans by\n",
      "injection. We generated a double-strand break by mobilizing a Mos1 transposon adjacent to the region\n",
      "to be deleted; the double-stranded break is repaired using injected DNA as a template. Repair can\n",
      "delete up to 25 kb of DNA and simultaneously insert a positive selection marker.\n",
      " {\n",
      "\"main_contribution\": \"A method for targeted gene deletions in C. elegans\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method called MosDEL for generating targeted knockouts of genes in Caenorhabditis elegans, a nematode worm widely used as a model organism in biological research. The method involves injecting DNA that mobilizes a transposon adjacent to the region to be deleted, creating a double-strand break that is repaired using the injected DNA as a template. This can result in the deletion of up to 25 kb of DNA and simultaneously insert a positive selection marker.\n",
      "\n",
      "The main contribution of the paper is the development of this new method for targeted gene deletions in C. elegans, which is described in sentences (B) and (C) of the abstract. Therefore, the main contribution is recorded as \"A method for targeted gene deletions in C. elegans\" in the JSON object.\n",
      "\n",
      "Since the paper does not describe the development of a new imaging technique or method, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "722\n",
      "20453865\n",
      "A photoconvertible reporter of the ubiquitin-proteasome system in vivo.\n",
      "The ubiquitin-proteasome system (UPS) orchestrates many cellular and tissue-specific processes by\n",
      "degrading damaged and key regulatory proteins. To enable investigation of UPS activity in different\n",
      "cell types in a living animal, we developed a photoconvertible fluorescent UPS reporter system for\n",
      "live imaging and quantification of protein degradation in Caenorhabditis elegans. Our reporter\n",
      "consists of the photoconvertible fluorescent protein Dendra2 targeted for proteasomal degradation by\n",
      "fusion to the UbG76V mutant form of ubiquitin. In contrast to previous reporters, this system\n",
      "permits quantification of UPS activity independently of protein synthesis. Our reporter revealed\n",
      "that UPS-mediated protein degradation varies in a cell type-specific and age-dependent manner in C.\n",
      "elegans.\n",
      " {\n",
      "\"main_contribution\": \"A photoconvertible reporter of the ubiquitin-proteasome system in vivo\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method, specifically a photoconvertible fluorescent UPS reporter system for live imaging and quantification of protein degradation in Caenorhabditis elegans. This system allows for the investigation of UPS activity in different cell types in a living animal, and it permits quantification of UPS activity independently of protein synthesis. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "723\n",
      "20453867\n",
      "In situ analysis of tyrosine phosphorylation networks by FLIM on cell arrays.\n",
      "Extracellular stimuli are transduced inside the cell by posttranslational modifications (PTMs), such\n",
      "as phosphorylation, of proteins in signaling networks. Insight into the structure of these networks\n",
      "requires quantification of PTM levels in individual cells. Fluorescence resonance energy transfer\n",
      "(FRET) measured by fluorescence lifetime imaging microscopy (FLIM) is a powerful tool to image PTM\n",
      "levels in situ. FLIM on cell arrays that express fluorescent protein fusions can quantify tyrosine\n",
      "phosphorylation patterns in large networks in individual cells. We identified tyrosine kinase\n",
      "substrates by imaging their phosphorylation levels after inhibition of protein tyrosine\n",
      "phosphatases. Analysis of the correlation between protein phosphorylation and expression levels at\n",
      "single cell resolution allowed us to identify positive feedback motifs. Using FLIM on cell arrays\n",
      "(CA-FLIM), we uncovered components that transduce signals from epidermal growth factor receptor.\n",
      " {\n",
      "\"main_contribution\": \"FLIM on cell arrays that express fluorescent protein fusions can quantify tyrosine phosphorylation patterns in large networks in individual cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development and application of FLIM (fluorescence lifetime imaging microscopy) on cell arrays to study tyrosine phosphorylation networks in individual cells. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "724\n",
      "20512143\n",
      "Enhanced neuronal RNAi in C. elegans using SID-1.\n",
      "We expressed SID-1, a transmembrane protein from Caenorhabditis elegans that is required for\n",
      "systemic RNA interference (RNAi), in C. elegans neurons. This expression increased the response of\n",
      "neurons to double-stranded (ds)RNA delivered by feeding. Mutations in the lin-15b and lin-35 genes\n",
      "enhanced this effect. Worms expressing neuronal SID-1 showed RNAi phenotypes when fed with bacteria\n",
      "expressing dsRNA for known neuronal genes and for uncharacterized genes with no previously known\n",
      "neuronal phenotypes. Neuronal expression of sid-1 decreased nonneuronal RNAi, suggesting that\n",
      "neurons expressing transgenic sid-1(+) served as a sink for dsRNA. This effect, or a sid-1(-)\n",
      "background, can be used to uncover neuronal defects for lethal genes. Expression of sid-1(+) from\n",
      "cell-specific promoters in sid-1 mutants results in cell-specific feeding RNAi. We used these\n",
      "strains to identify a role for integrin signaling genes in mechanosensation.\n",
      " {\n",
      "\"main_contribution\": \"Identification of a role for integrin signaling genes in mechanosensation using enhanced neuronal RNAi in C. elegans\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe research on enhancing neuronal RNAi in C. elegans using SID-1, a transmembrane protein. The main contribution of the paper is identifying a role for integrin signaling genes in mechanosensation using this enhanced neuronal RNAi technique. Therefore, the main contribution is not the development of a new imaging method, and hence \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "725\n",
      "20512144\n",
      "Probabilistic density maps to study global endomembrane organization.\n",
      "We developed a computational imaging approach that describes the three-dimensional spatial\n",
      "organization of endomembranes from micromanipulation-normalized mammalian cells with probabilistic\n",
      "density maps. Applied to several well-known marker proteins, this approach revealed the average\n",
      "steady-state organization of early endosomes, multivesicular bodies or lysosomes, endoplasmic\n",
      "reticulum exit sites, the Golgi apparatus and Golgi-derived transport carriers in crossbow-shaped\n",
      "cells. The steady-state organization of each tested endomembranous population was well-defined,\n",
      "unique and in some cases depended on the cellular adhesion geometry. Density maps of all\n",
      "endomembrane populations became stable when pooling several tens of cells only and were reproducible\n",
      "in independent experiments, allowing construction of a standardized cell model. We detected subtle\n",
      "changes in steady-state organization induced by disruption of the cellular cytoskeleton, with\n",
      "statistical significance observed for just 20 cells. Thus, combining micropatterning with\n",
      "construction of endomembrane density maps allows the systematic study of intracellular trafficking\n",
      "determinants.\n",
      " {\n",
      "\"main_contribution\": \"Probabilistic density maps to study global endomembrane organization\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "726\n",
      "20521367\n",
      "Discovering Mozart.\n",
      "High-throughput imaging of genome-wide RNA interference screens and systematic analysis of protein\n",
      "complexes involved in mitosis yield valuable resources and reveal new subunits of well-characterized\n",
      "complexes.\n",
      " {\n",
      "\"main_contribution\": \"high-throughput imaging of genome-wide RNA interference screens\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a high-throughput imaging technique for genome-wide RNA interference screens, which is an imaging method. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "727\n",
      "20526346\n",
      "Two-photon calcium imaging from head-fixed Drosophila during optomotor walking behavior.\n",
      "Drosophila melanogaster is a model organism rich in genetic tools to manipulate and identify neural\n",
      "circuits involved in specific behaviors. Here we present a technique for two-photon calcium imaging\n",
      "in the central brain of head-fixed Drosophila walking on an air-supported ball. The ball's motion is\n",
      "tracked at high resolution and can be treated as a proxy for the fly's own movements. We used the\n",
      "genetically encoded calcium sensor, GCaMP3.0, to record from important elements of the motion-\n",
      "processing pathway, the horizontal-system lobula plate tangential cells (LPTCs) in the fly optic\n",
      "lobe. We presented motion stimuli to the tethered fly and found that calcium transients in\n",
      "horizontal-system neurons correlated with robust optomotor behavior during walking. Our technique\n",
      "allows both behavior and physiology in identified neurons to be monitored in a genetic model\n",
      "organism with an extensive repertoire of walking behaviors.\n",
      " {\n",
      "\"main_contribution\": \"Two-photon calcium imaging technique for monitoring neural activity in Drosophila during optomotor behavior\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a two-photon calcium imaging technique, for monitoring neural activity in Drosophila during optomotor behavior. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "728\n",
      "20543845\n",
      "Imaging plant growth in 4D: robust tissue reconstruction and lineaging at cell resolution.\n",
      "Quantitative information on growing organs is required to better understand morphogenesis in both\n",
      "plants and animals. However, detailed analyses of growth patterns at cellular resolution have\n",
      "remained elusive. We developed an approach, multiangle image acquisition, three-dimensional\n",
      "reconstruction and cell segmentation-automated lineage tracking (MARS-ALT), in which we imaged whole\n",
      "organs from multiple angles, computationally merged and segmented these images to provide accurate\n",
      "cell identification in three dimensions and automatically tracked cell lineages through multiple\n",
      "rounds of cell division during development. Using these methods, we quantitatively analyzed\n",
      "Arabidopsis thaliana flower development at cell resolution, which revealed differential growth\n",
      "patterns of key regions during early stages of floral morphogenesis. Lastly, using rice roots, we\n",
      "demonstrated that this approach is both generic and scalable.\n",
      " {\n",
      "\"main_contribution\": \"Robust tissue reconstruction and lineaging at cell resolution for imaging plant growth in 4D.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "729\n",
      "20562866\n",
      "Noncontact microrheology at acoustic frequencies using frequency-modulated atomic force microscopy.\n",
      "We report an atomic force microscopy (AFM) method for assessing elastic and viscous properties of\n",
      "soft samples at acoustic frequencies under non-contact conditions. The method can be used to measure\n",
      "material properties via frequency modulation and is based on hydrodynamics theory of thin gaps we\n",
      "developed here. A cantilever with an attached microsphere is forced to oscillate tens of nanometers\n",
      "above a sample. The elastic modulus and viscosity of the sample are estimated by measuring the\n",
      "frequency-dependence of the phase lag between the oscillating microsphere and the driving piezo at\n",
      "various heights above the sample. This method features an effective area of pyramidal tips used in\n",
      "contact AFM but with only piconewton applied forces. Using this method, we analyzed polyacrylamide\n",
      "gels of different stiffness and assessed graded mechanical properties of guinea pig tectorial\n",
      "membrane. The technique enables the study of microrheology of biological tissues that produce or\n",
      "detect sound.\n",
      " {\n",
      "\"main_contribution\": \"A new method for assessing elastic and viscous properties of soft samples at acoustic frequencies under non-contact conditions using atomic force microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique or method, specifically a method for assessing elastic and viscous properties of soft samples at acoustic frequencies under non-contact conditions using atomic force microscopy. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "730\n",
      "20601949\n",
      "A photoactivatable marker protein for pulse-chase imaging with superresolution.\n",
      "IrisFP is a photoactivatable fluorescent protein that combines irreversible photoconversion from a\n",
      "green- to a red-emitting form with reversible photoswitching between a fluorescent and a\n",
      "nonfluorescent state in both forms. Here we introduce a monomeric variant, mIrisFP, and demonstrate\n",
      "how its multiple photoactivation modes can be used for pulse-chase experiments combined with\n",
      "subdiffraction-resolution imaging in living cells by using dual-color photoactivation localization\n",
      "microscopy (PALM).\n",
      " {\n",
      "\"main_contribution\": \"A photoactivatable marker protein for pulse-chase imaging with superresolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a photoactivatable marker protein, mIrisFP, which can be used for pulse-chase experiments combined with subdiffraction-resolution imaging in living cells using dual-color photoactivation localization microscopy (PALM). Therefore, the main contribution is the development of a new imaging method, and thus \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "731\n",
      "20601950\n",
      "Fast, high-contrast imaging of animal development with scanned light sheet-based structured-\n",
      "illumination microscopy.\n",
      "Recording light-microscopy images of large, nontransparent specimens, such as developing\n",
      "multicellular organisms, is complicated by decreased contrast resulting from light scattering. Early\n",
      "zebrafish development can be captured by standard light-sheet microscopy, but new imaging strategies\n",
      "are required to obtain high-quality data of late development or of less transparent organisms. We\n",
      "combined digital scanned laser light-sheet fluorescence microscopy with incoherent structured-\n",
      "illumination microscopy (DSLM-SI) and created structured-illumination patterns with continuously\n",
      "adjustable frequencies. Our method discriminates the specimen-related scattered background from\n",
      "signal fluorescence, thereby removing out-of-focus light and optimizing the contrast of in-focus\n",
      "structures. DSLM-SI provides rapid control of the illumination pattern, exceptional imaging quality\n",
      "and high imaging speeds. We performed long-term imaging of zebrafish development for 58 h and fast\n",
      "multiple-view imaging of early Drosophila melanogaster development. We reconstructed cell positions\n",
      "over time from the Drosophila DSLM-SI data and created a fly digital embryo.\n",
      " {\n",
      "\"main_contribution\": \"Development of a new imaging technique called scanned light sheet-based structured-illumination microscopy (DSLM-SI) for high-contrast imaging of animal development\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "732\n",
      "20622860\n",
      "Imaging brain electric signals with genetically targeted voltage-sensitive fluorescent proteins.\n",
      "Cortical information processing relies on synaptic interactions between diverse classes of neurons\n",
      "with distinct electrophysiological and connection properties. Uncovering the operational principles\n",
      "of these elaborate circuits requires the probing of electrical activity from selected populations of\n",
      "defined neurons. Here we show that genetically encoded voltage-sensitive fluorescent proteins\n",
      "(VSFPs) provide an optical voltage report from targeted neurons in culture, acute brain slices and\n",
      "living mice. By expressing VSFPs in pyramidal cells of mouse somatosensory cortex, we also\n",
      "demonstrate that these probes can report cortical electrical responses to single sensory stimuli in\n",
      "vivo. These protein-based voltage probes will facilitate the analysis of cortical circuits in\n",
      "genetically defined cell populations and are hence a valuable addition to the optogenetic toolbox.\n",
      " {\n",
      "\"main_contribution\": \"Genetically encoded voltage-sensitive fluorescent proteins (VSFPs) provide an optical voltage report from targeted neurons in culture, acute brain slices and living mice.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of genetically encoded voltage-sensitive fluorescent proteins (VSFPs) that can report optical activity from targeted neurons in various settings, including living mice. This is an imaging method, so the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "733\n",
      "20639868\n",
      "High-throughput in vivo vertebrate screening.\n",
      "We demonstrate a high-throughput platform for cellular-resolution in vivo chemical and genetic\n",
      "screens on zebrafish larvae. The system automatically loads zebrafish from reservoirs or multiwell\n",
      "plates, and positions and rotates them for high-speed confocal imaging and laser manipulation of\n",
      "both superficial and deep organs within 19 s without damage. We performed small-scale test screening\n",
      "of retinal axon guidance mutants and neuronal regeneration assays in combination with femtosecond\n",
      "laser microsurgery.\n",
      " {\n",
      "\"main_contribution\": \"high-throughput platform for cellular-resolution in vivo chemical and genetic screens on zebrafish larvae\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the development of a high-throughput screening platform for in vivo vertebrate screening. The abstract provides more detail, explaining that the system is for cellular-resolution chemical and genetic screens on zebrafish larvae, and that it includes high-speed confocal imaging and laser manipulation of both superficial and deep organs within 19 seconds without damage. Therefore, the main contribution of the paper is the development of this high-throughput platform, which includes advanced imaging techniques. Since the contribution is the development of an imaging method, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "734\n",
      "20676081\n",
      "Going deeper than microscopy: the optical imaging frontier in biology.\n",
      "Optical microscopy has been a fundamental tool of biological discovery for more than three\n",
      "centuries, but its in vivo tissue imaging ability has been restricted by light scattering to\n",
      "superficial investigations, even when confocal or multiphoton methods are used. Recent advances in\n",
      "optical and optoacoustic (photoacoustic) imaging now allow imaging at depths and resolutions\n",
      "unprecedented for optical methods. These abilities are increasingly important to understand the\n",
      "dynamic interactions of cellular processes at different systems levels, a major challenge of\n",
      "postgenome biology. This Review discusses promising photonic methods that have the ability to\n",
      "visualize cellular and subcellular components in tissues across different penetration scales. The\n",
      "methods are classified into microscopic, mesoscopic and macroscopic approaches, according to the\n",
      "tissue depth at which they operate. Key characteristics associated with different imaging\n",
      "implementations are described and the potential of these technologies in biological applications is\n",
      "discussed.\n",
      " {\n",
      "\"main_contribution\": \"Promising photonic methods that have the ability to visualize cellular and subcellular components in tissues across different penetration scales.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "735\n",
      "20676108\n",
      "Mechanical regulation of cell function with geometrically modulated elastomeric substrates.\n",
      "We report the establishment of a library of micromolded elastomeric micropost arrays to modulate\n",
      "substrate rigidity independently of effects on adhesive and other material surface properties. We\n",
      "demonstrated that micropost rigidity impacts cell morphology, focal adhesions, cytoskeletal\n",
      "contractility and stem cell differentiation. Furthermore, early changes in cytoskeletal\n",
      "contractility predicted later stem cell fate decisions in single cells.\n",
      " {\n",
      "\"main_contribution\": \"Mechanical regulation of cell function using elastomeric substrates\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the paper describe a new method for mechanically regulating cell function using geometrically modulated elastomeric substrates. The main contribution of the paper is not the development of a new imaging technique or method, but rather the establishment of a library of micromolded elastomeric micropost arrays to modulate substrate rigidity independently of effects on adhesive and other material surface properties. Therefore, the value of the \"is_imaging_method\" variable is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "736\n",
      "20676109\n",
      "Automated imaging with ScanLag reveals previously undetectable bacterial growth phenotypes.\n",
      "We developed an automated system, ScanLag, that measures in parallel the delay in growth (lag time)\n",
      "and growth rate of thousands of cells. Using ScanLag, we detected small subpopulations of bacteria\n",
      "with dramatically increased lag time upon starvation. By screening a library of Escherichia coli\n",
      "deletion mutants, we achieved two-dimensional mapping of growth characteristics, which showed that\n",
      "ScanLag enables multidimensional screens for quantitative characterization and identification of\n",
      "rare phenotypic variants.\n",
      " {\n",
      "\"main_contribution\": \"Automated imaging with ScanLag reveals previously undetectable bacterial growth phenotypes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an automated imaging system called ScanLag, which enables the measurement of lag time and growth rate of thousands of cells in parallel. This system allows for the detection of small subpopulations of bacteria with increased lag time upon starvation and enables multidimensional screens for quantitative characterization and identification of rare phenotypic variants. Therefore, the main contribution is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "737\n",
      "20693996\n",
      "CellCognition: time-resolved phenotype annotation in high-throughput live cell imaging.\n",
      "Fluorescence time-lapse imaging has become a powerful tool to investigate complex dynamic processes\n",
      "such as cell division or intracellular trafficking. Automated microscopes generate time-resolved\n",
      "imaging data at high throughput, yet tools for quantification of large-scale movie data are largely\n",
      "missing. Here we present CellCognition, a computational framework to annotate complex cellular\n",
      "dynamics. We developed a machine-learning method that combines state-of-the-art classification with\n",
      "hidden Markov modeling for annotation of the progression through morphologically distinct biological\n",
      "states. Incorporation of time information into the annotation scheme was essential to suppress\n",
      "classification noise at state transitions and confusion between different functional states with\n",
      "similar morphology. We demonstrate generic applicability in different assays and perturbation\n",
      "conditions, including a candidate-based RNA interference screen for regulators of mitotic exit in\n",
      "human cells. CellCognition is published as open source software, enabling live-cell imaging-based\n",
      "screening with assays that directly score cellular dynamics.\n",
      " {\n",
      "\"main_contribution\": \"computational framework to annotate complex cellular dynamics\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "738\n",
      "20693997\n",
      "Two-photon high-resolution measurement of partial pressure of oxygen in cerebral vasculature and\n",
      "tissue.\n",
      "Measurements of oxygen partial pressure (pO(2)) with high temporal and spatial resolution in three\n",
      "dimensions is crucial for understanding oxygen delivery and consumption in normal and diseased\n",
      "brain. Among existing pO(2) measurement methods, phosphorescence quenching is optimally suited for\n",
      "the task. However, previous attempts to couple phosphorescence with two-photon laser scanning\n",
      "microscopy have faced substantial difficulties because of extremely low two-photon absorption cross-\n",
      "sections of conventional phosphorescent probes. Here we report to our knowledge the first practical\n",
      "in vivo two-photon high-resolution pO(2) measurements in small rodents' cortical microvasculature\n",
      "and tissue, made possible by combining an optimized imaging system with a two-photon-enhanced\n",
      "phosphorescent nanoprobe. The method features a measurement depth of up to 250 microm, sub-second\n",
      "temporal resolution and requires low probe concentration. The properties of the probe allowed for\n",
      "direct high-resolution measurement of cortical extravascular (tissue) pO(2), opening many\n",
      "possibilities for functional metabolic brain studies.\n",
      " {\n",
      "\"main_contribution\": \"Two-photon high-resolution measurement of partial pressure of oxygen in cerebral vasculature and tissue\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a two-photon high-resolution measurement of partial pressure of oxygen in cerebral vasculature and tissue. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "739\n",
      "20693998\n",
      "Live-cell super-resolution imaging with trimethoprim conjugates.\n",
      "The spatiotemporal resolution of subdiffraction fluorescence imaging has been limited by the\n",
      "difficulty of labeling proteins in cells with suitable fluorophores. Here we report a chemical tag\n",
      "that allows proteins to be labeled with an organic fluorophore with high photon flux and fast\n",
      "photoswitching performance in live cells. This label allowed us to image the dynamics of human\n",
      "histone H2B protein in living cells at approximately 20 nm resolution.\n",
      " {\n",
      "\"main_contribution\": \"Live-cell super-resolution imaging with trimethoprim conjugates\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically live-cell super-resolution imaging with trimethoprim conjugates. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "740\n",
      "20729842\n",
      "Analysis of microtubule dynamic instability using a plus-end growth marker.\n",
      "Regulation of microtubule dynamics is essential for many cell biological processes and is likely to\n",
      "be variable between different subcellular regions. We describe a computational approach to analyze\n",
      "microtubule dynamics by detecting growing microtubule plus ends. Our algorithm tracked all EB1-EGFP\n",
      "comets visible in an image time-lapse sequence allowing the detection of spatial patterns of\n",
      "microtubule dynamics. We introduce spatiotemporal clustering of EB1-EGFP growth tracks to infer\n",
      "microtubule behaviors during phases of pause and shortening. We validated the algorithm by comparing\n",
      "the results to data for manually tracked, homogeneously labeled microtubules and by analyzing the\n",
      "effects of well-characterized inhibitors of microtubule polymerization dynamics. We used our method\n",
      "to analyze spatial variations of intracellular microtubule dynamics in migrating epithelial cells.\n",
      " {\n",
      "\"main_contribution\": \"spatiotemporal clustering of EB1-EGFP growth tracks to infer microtubule behaviors during phases of pause and shortening\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a computational approach to analyze microtubule dynamics by detecting growing microtubule plus ends, which is an imaging method. The algorithm tracked all EB1-EGFP comets visible in an image time-lapse sequence, allowing the detection of spatial patterns of microtubule dynamics. Therefore, the main contribution is related to imaging and microscopy, and the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "741\n",
      "20818379\n",
      "Near-infrared fluorescent proteins.\n",
      "Fluorescent proteins with emission wavelengths in the near-infrared and infrared range are in high\n",
      "demand for whole-body imaging techniques. Here we report near-infrared dimeric fluorescent proteins\n",
      "eqFP650 and eqFP670. To our knowledge, eqFP650 is the brightest fluorescent protein with emission\n",
      "maximum above 635 nm, and eqFP670 displays the most red-shifted emission maximum and high\n",
      "photostability.\n",
      " {\n",
      "\"main_contribution\": \"near-infrared dimeric fluorescent proteins eqFP650 and eqFP670\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of near-infrared dimeric fluorescent proteins eqFP650 and eqFP670, which can be used for whole-body imaging techniques. As the paper's contribution is the development of a new imaging technique, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "742\n",
      "20852649\n",
      "Scanless two-photon excitation of channelrhodopsin-2.\n",
      "Light-gated ion channels and pumps have made it possible to probe intact neural circuits by\n",
      "manipulating the activity of groups of genetically similar neurons. What is needed now is a method\n",
      "for precisely aiming the stimulating light at single neuronal processes, neurons or groups of\n",
      "neurons. We developed a method that combines generalized phase contrast with temporal focusing (TF-\n",
      "GPC) to shape two-photon excitation for this purpose. The illumination patterns are generated\n",
      "automatically from fluorescence images of neurons and shaped to cover the cell body or dendrites, or\n",
      "distributed groups of cells. The TF-GPC two-photon excitation patterns generated large photocurrents\n",
      "in Channelrhodopsin-2-expressing cultured cells and neurons and in mouse acute cortical slices. The\n",
      "amplitudes of the photocurrents can be precisely modulated by controlling the size and shape of the\n",
      "excitation volume and, thereby, be used to trigger single action potentials or trains of action\n",
      "potentials.\n",
      " {\n",
      "\"main_contribution\": \"development of a method for precisely aiming the stimulating light at single neuronal processes, neurons or groups of neurons using TF-GPC two-photon excitation patterns\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the combination of generalized phase contrast with temporal focusing (TF-GPC) to shape two-photon excitation for precisely aiming stimulating light at single neuronal processes, neurons or groups of neurons. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "743\n",
      "20871617\n",
      "Quantitative dynamic footprinting microscopy reveals mechanisms of neutrophil rolling.\n",
      "We introduce quantitative dynamic footprinting microscopy to resolve neutrophil rolling on\n",
      "P-selectin. We observed that the footprint of a rolling neutrophil was fourfold larger than\n",
      "previously thought, and that P-selectin-PSGL-1 bonds were relaxed at the leading edge of the rolling\n",
      "cell, compressed under the cell center, and stretched at the trailing edge. Each rolling neutrophil\n",
      "formed three to four long tethers that extended up to 16 μm behind the rolling cell.\n",
      " {\n",
      "\"main_contribution\": \"Quantitative dynamic footprinting microscopy reveals mechanisms of neutrophil rolling\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, quantitative dynamic footprinting microscopy, which allows for the study of neutrophil rolling on P-selectin. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "744\n",
      "20953177\n",
      "Dual RMCE for efficient re-engineering of mouse mutant alleles.\n",
      "We have developed dual recombinase-mediated cassette exchange (dRMCE) to efficiently re-engineer the\n",
      "thousands of available conditional alleles in mouse embryonic stem cells. dRMCE takes advantage of\n",
      "the wild-type loxP and FRT sites present in these conditional alleles and in many gene-trap lines.\n",
      "dRMCE is a scalable, flexible tool to introduce tags, reporters and mutant coding regions into an\n",
      "endogenous locus of interest in an easy and highly efficient manner.\n",
      " {\n",
      "\"main_contribution\": \"development of dual recombinase-mediated cassette exchange (dRMCE) to efficiently re-engineer conditional alleles in mouse embryonic stem cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new technique called dual recombinase-mediated cassette exchange (dRMCE) for re-engineering conditional alleles in mouse embryonic stem cells. The method takes advantage of wild-type loxP and FRT sites present in conditional alleles and gene-trap lines, and allows for the introduction of tags, reporters, and mutant coding regions into an endogenous locus of interest in a efficient manner.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of dRMCE, which is a new imaging method. The \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "745\n",
      "20966916\n",
      "Chronic optical access through a polished and reinforced thinned skull.\n",
      "We present a method to form an optical window in the mouse skull that spans millimeters and is\n",
      "stable for months without causing brain inflammation. This enabled us to repeatedly image blood flow\n",
      "in cortical capillaries of awake mice and determine long-range correlations in speed. We also\n",
      "repeatedly imaged dendritic spines, microglia and angioarchitecture, as well as used illumination to\n",
      "drive motor output via optogenetics and induce microstrokes via photosensitizers.\n",
      " {\n",
      "\"main_contribution\": \"chronic optical access through a polished and reinforced thinned skull\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a method for forming an optical window in the mouse skull that allows for chronic optical access and stable imaging over months. This method enables the imaging of blood flow in cortical capillaries, dendritic spines, microglia, and angioarchitecture, as well as the use of optogenetics and photosensitizers for various applications. Therefore, the main contribution is classified as an imaging method, and the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "746\n",
      "21037589\n",
      "Rapid blue-light-mediated induction of protein interactions in living cells.\n",
      "Dimerizers allowing inducible control of protein-protein interactions are powerful tools for\n",
      "manipulating biological processes. Here we describe genetically encoded light-inducible protein-\n",
      "interaction modules based on Arabidopsis thaliana cryptochrome 2 and CIB1 that require no exogenous\n",
      "ligands and dimerize on blue-light exposure with subsecond time resolution and subcellular spatial\n",
      "resolution. We demonstrate the utility of this system by inducing protein translocation,\n",
      "transcription and Cre recombinase-mediated DNA recombination using light.\n",
      " {\n",
      "\"main_contribution\": \"Rapid blue-light-mediated induction of protein interactions in living cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is described in the sentence: \"We demonstrate the utility of this system by inducing protein translocation, transcription and Cre recombinase-mediated DNA recombination using light.\" This indicates that the authors have developed a new method for manipulating biological processes using light-inducible protein-interaction modules. However, this method does not involve the development of a new imaging technique or method, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "747\n",
      "21049576\n",
      "Structural biology.  Protein structure gets exciting.\n",
      "Researchers determined the excited-state structure of a small protein using nuclear magnetic\n",
      "resonance spectroscopy.\n",
      " {\n",
      "\"main_contribution\": \"determined the excited-state structure of a small protein\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "748\n",
      "21049580\n",
      "Microscopy.  Self-healing light beams.\n",
      "The self-reconstructing properties of Bessel beams provide healing benefits in highly scattering\n",
      "media.\n",
      " {\n",
      "\"main_contribution\": \"self-healing light beams\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "749\n",
      "21076418\n",
      "A defined glycosaminoglycan-binding substratum for human pluripotent stem cells.\n",
      "To exploit the full potential of human pluripotent stem cells for regenerative medicine,\n",
      "developmental biology and drug discovery, defined culture conditions are needed. Media of known\n",
      "composition that maintain human embryonic stem (hES) cells have been developed, but finding\n",
      "chemically defined, robust substrata has proven difficult. We used an array of self-assembled\n",
      "monolayers to identify peptide surfaces that sustain pluripotent stem cell self-renewal. The\n",
      "effective substrates displayed heparin-binding peptides, which can interact with cell-surface\n",
      "glycosaminoglycans and could be used with a defined medium to culture hES cells for more than 3\n",
      "months. The resulting cells maintained a normal karyotype and had high levels of pluripotency\n",
      "markers. The peptides supported growth of eight pluripotent cell lines on a variety of scaffolds.\n",
      "Our results indicate that synthetic substrates that recognize cell-surface glycans can facilitate\n",
      "the long-term culture of pluripotent stem cells.\n",
      " {\n",
      "\"main_contribution\": \"A defined glycosaminoglycan-binding substratum for human pluripotent stem cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new substrate for culturing human pluripotent stem cells. The authors used an array of self-assembled monolayers to identify peptide surfaces that sustain pluripotent stem cell self-renewal, and found that heparin-binding peptides were effective in supporting the growth of pluripotent stem cells. They demonstrated that their method can maintain normal karyotype and high levels of pluripotency markers for over three months.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new substrate for culturing human pluripotent stem cells, which is not an imaging technique or method. Hence, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "750\n",
      "21076419\n",
      "Three-dimensional cellular ultrastructure resolved by X-ray microscopy.\n",
      "We developed an X-ray microscope using partially coherent object illumination instead of previously\n",
      "used quasi-incoherent illumination. The design permitted the incorporation of a cryogenic tilt\n",
      "stage, enabling tomography of frozen-hydrated, intact adherent cells. We obtained three-dimensional\n",
      "reconstructions of mouse adenocarcinoma cells at ∼36-nm (Rayleigh) and ∼70-nm (Fourier ring\n",
      "correlation) resolution, which allowed us to visualize the double nuclear membrane, nuclear pores,\n",
      "nuclear membrane channels, mitochondrial cristae and lysosomal inclusions.\n",
      " {\n",
      "\"main_contribution\": \"Three-dimensional reconstructions of mouse adenocarcinoma cells at ∼36-nm (Rayleigh) and ∼70-nm (Fourier ring correlation) resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically an X-ray microscope that uses partially coherent object illumination and a cryogenic tilt stage to obtain three-dimensional reconstructions of intact adherent cells at high resolution. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "751\n",
      "21102453\n",
      "Protein localization in electron micrographs using fluorescence nanoscopy.\n",
      "A complete portrait of a cell requires a detailed description of its molecular topography: proteins\n",
      "must be linked to particular organelles. Immunocytochemical electron microscopy can reveal locations\n",
      "of proteins with nanometer resolution but is limited by the quality of fixation, the paucity of\n",
      "antibodies and the inaccessibility of antigens. Here we describe correlative fluorescence electron\n",
      "microscopy for the nanoscopic localization of proteins in electron micrographs. We tagged proteins\n",
      "with the fluorescent proteins Citrine or tdEos and expressed them in Caenorhabditis elegans, fixed\n",
      "the worms and embedded them in plastic. We imaged the tagged proteins from ultrathin sections using\n",
      "stimulated emission depletion (STED) microscopy or photoactivated localization microscopy (PALM).\n",
      "Fluorescence correlated with organelles imaged in electron micrographs from the same sections. We\n",
      "used these methods to localize histones, a mitochondrial protein and a presynaptic dense projection\n",
      "protein in electron micrographs.\n",
      " {\n",
      "\"main_contribution\": \"nanoscopic localization of proteins in electron micrographs\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically correlative fluorescence electron microscopy for nanoscopic localization of proteins in electron micrographs. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "752\n",
      "21151136\n",
      "Stabilized imaging of immune surveillance in the mouse lung.\n",
      "Real-time imaging of cellular and subcellular dynamics in vascularized organs requires image\n",
      "resolution and image registration to be simultaneously optimized without perturbing normal\n",
      "physiology. This problem is particularly pronounced in the lung, in which cells may transit at\n",
      "speeds >1 mm s(-1) and in which normal respiration results in large-scale tissue movements that\n",
      "prevent image registration. Here we report video-rate, two-photon imaging of a physiologically\n",
      "intact preparation of the mouse lung that is stabilizing and nondisruptive. Using our method, we\n",
      "obtained evidence for differential trapping of T cells and neutrophils in mouse pulmonary\n",
      "capillaries, and observed neutrophil mobilization and dynamic vascular leak in response to stretch\n",
      "and inflammatory models of lung injury in mice. The system permits physiological measurement of\n",
      "motility rates of >1 mm s(-1), observation of detailed cellular morphology and could be applied in\n",
      "the future to other organs and tissues while maintaining intact physiology.\n",
      " {\n",
      "\"main_contribution\": \"Real-time imaging of cellular and subcellular dynamics in vascularized organs\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method for real-time imaging of cellular and subcellular dynamics in vascularized organs, specifically the mouse lung. This method allows for video-rate, two-photon imaging of a physiologically intact preparation of the mouse lung that is stabilizing and non-disruptive, enabling the observation of detailed cellular morphology and measurement of motility rates of >1 mm s(-1). Therefore, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "753\n",
      "21186361\n",
      "A microfluidic array for large-scale ordering and orientation of embryos.\n",
      "Quantitative studies of embryogenesis require the ability to monitor pattern formation and\n",
      "morphogenesis in large numbers of embryos, at multiple time points and in diverse genetic\n",
      "backgrounds. We describe a simple approach that greatly facilitates these tasks for Drosophila\n",
      "melanogaster embryos, one of the most advanced models of developmental genetics. Based on passive\n",
      "hydrodynamics, we developed a microfluidic embryo-trap array that can be used to rapidly order and\n",
      "vertically orient hundreds of embryos. We describe the physical principles of the design and used\n",
      "this platform to quantitatively analyze multiple morphogen gradients in the dorsoventral patterning\n",
      "system. Our approach can also be used for live imaging and, with slight modifications, could be\n",
      "adapted for studies of pattern formation and morphogenesis in other model organisms.\n",
      " {\n",
      "\"main_contribution\": \"a microfluidic array for large-scale ordering and orientation of embryos\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a microfluidic array for ordering and orienting embryos, which is not an imaging method. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "754\n",
      "21217749\n",
      "Simultaneous two-photon calcium imaging at different depths with spatiotemporal multiplexing.\n",
      "In vivo two-photon calcium imaging would benefit from the use of multiple excitation beams to\n",
      "increase scanning speed, signal-to-noise ratio and field of view or to image different axial planes\n",
      "simultaneously. Using spatiotemporal multiplexing we circumvented light-scattering ambiguity\n",
      "inherent to deep-tissue multifocal two-photon microscopy. We demonstrate calcium imaging at multiple\n",
      "axial planes in the intact mouse brain to monitor network activity of ensembles of cortical neurons\n",
      "in three spatial dimensions.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous two-photon calcium imaging at different depths with spatiotemporal multiplexing\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method that allows for simultaneous two-photon calcium imaging at different depths using spatiotemporal multiplexing. This technique enables the imaging of ensembles of cortical neurons in three spatial dimensions in the intact mouse brain, which was not possible before. Therefore, the main contribution is the development of a new imaging method, and the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "755\n",
      "21217750\n",
      "A photoprotection strategy for microsecond-resolution single-molecule fluorescence spectroscopy.\n",
      "Time resolution of current single-molecule fluorescence techniques is limited to milliseconds\n",
      "because of dye blinking and bleaching. Here we introduce a photoprotection strategy that affords\n",
      "microsecond resolution by combining efficient triplet quenching by oxygen and Trolox with minimized\n",
      "bleaching via the oxygen radical scavenger cysteamine. Using this approach we resolved the single-\n",
      "molecule microsecond conformational fluctuations of two proteins: the two-state folder α-spectrin\n",
      "SH3 domain and the ultrafast downhill folder BBL.\n",
      " {\n",
      "\"main_contribution\": \"A photoprotection strategy for microsecond-resolution single-molecule fluorescence spectroscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a photoprotection strategy that allows for microsecond-resolution single-molecule fluorescence spectroscopy, which is a new imaging method. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "756\n",
      "21240279\n",
      "Optogenetic manipulation of neural activity in freely moving Caenorhabditis elegans.\n",
      "We present an optogenetic illumination system capable of real-time light delivery with high spatial\n",
      "resolution to specified targets in freely moving Caenorhabditis elegans. A tracking microscope\n",
      "records the motion of an unrestrained worm expressing channelrhodopsin-2 or halorhodopsin in\n",
      "specific cell types. Image processing software analyzes the worm's position in each video frame,\n",
      "rapidly estimates the locations of targeted cells and instructs a digital micromirror device to\n",
      "illuminate targeted cells with laser light of the appropriate wavelengths to stimulate or inhibit\n",
      "activity. Because each cell in an unrestrained worm is a rapidly moving target, our system operates\n",
      "at high speed (∼50 frames per second) to provide high spatial resolution (∼30 μm). To test the\n",
      "accuracy, flexibility and utility of our system, we performed optogenetic analyses of the worm motor\n",
      "circuit, egg-laying circuit and mechanosensory circuits that have not been possible with previous\n",
      "methods.\n",
      " {\n",
      "\"main_contribution\": \"A tracking microscope records the motion of an unrestrained worm expressing channelrhodopsin-2 or halorhodopsin in specific cell types, and image processing software analyzes the worm's position in each video frame, rapidly estimates the locations of targeted cells and instructs a digital micromirror device to illuminate targeted cells with laser light of the appropriate wavelengths to stimulate or inhibit activity.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing an imaging technique or method, specifically an optogenetic illumination system capable of real-time light delivery with high spatial resolution to specified targets in freely moving Caenorhabditis elegans. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "757\n",
      "21240280\n",
      "A transgenic mouse for in vivo detection of endogenous labeled mRNA.\n",
      "Live-cell single mRNA imaging is a powerful tool but has been restricted in higher eukaryotes to\n",
      "artificial cell lines and reporter genes. We describe an approach that enables live-cell imaging of\n",
      "single endogenous labeled mRNA molecules transcribed in primary mammalian cells and tissue. We\n",
      "generated a knock-in mouse line with an MS2 binding site (MBS) cassette targeted to the 3'\n",
      "untranslated region of the essential β-actin gene. As β-actin-MBS was ubiquitously expressed, we\n",
      "could uniquely address endogenous mRNA regulation in any tissue or cell type. We simultaneously\n",
      "followed transcription from the β-actin alleles in real time and observed transcriptional bursting\n",
      "in response to serum stimulation with precise temporal resolution. We tracked single endogenous\n",
      "labeled mRNA particles being transported in primary hippocampal neurons. The MBS cassette also\n",
      "enabled high-sensitivity fluorescence in situ hybridization (FISH), allowing detection and\n",
      "localization of single β-actin mRNA molecules in various mouse tissues.\n",
      " {\n",
      "\"main_contribution\": \"A transgenic mouse for in vivo detection of endogenous labeled mRNA\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a transgenic mouse that allows for the in vivo detection of endogenous labeled mRNA. This is achieved through the use of an MS2 binding site (MBS) cassette targeted to the 3' untranslated region of the essential β-actin gene, which enables live-cell imaging of single endogenous labeled mRNA molecules transcribed in primary mammalian cells and tissue. The MBS cassette also enables high-sensitivity fluorescence in situ hybridization (FISH), allowing detection and localization of single β-actin mRNA molecules in various mouse tissues.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new imaging method, specifically a transgenic mouse that allows for the in vivo detection of endogenous labeled mRNA, and thus the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "758\n",
      "21240281\n",
      "RNAi screening for fat regulatory genes with SRS microscopy.\n",
      "Identification of genes regulating fat accumulation is important for basic and medical research;\n",
      "genetic screening for those genes in Caenorhabditis elegans, a widely used model organism, requires\n",
      "in vivo quantification of lipids. We demonstrated RNA interference screening based on quantitative\n",
      "imaging of lipids with label-free stimulated Raman scattering (SRS) microscopy, which overcomes\n",
      "major limitations of coherent anti-Stokes Raman scattering microscopy. Our screening yielded eight\n",
      "new genetic regulators of fat storage.\n",
      " {\n",
      "\"main_contribution\": \"identification of genes regulating fat accumulation\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a method for RNAi screening to identify genes that regulate fat accumulation in Caenorhabditis elegans using SRS microscopy. The main contribution of the paper is therefore the identification of these genetic regulators, which is described in sentence (D) of the abstract.\n",
      "\n",
      "The method used to achieve this goal is RNAi screening based on quantitative imaging of lipids with label-free stimulated Raman scattering (SRS) microscopy, which overcomes major limitations of coherent anti-Stokes Raman scattering microscopy. This is described in sentence (C) of the abstract.\n",
      "\n",
      "Therefore, the main contribution of the paper is the identification of genetic regulators of fat storage, and the method used to achieve this goal is a new imaging technique based on SRS microscopy, which makes the contribution an imaging method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "759\n",
      "21258339\n",
      "Micropilot: automation of fluorescence microscopy-based imaging for systems biology.\n",
      "Quantitative microscopy relies on imaging of large cell numbers but is often hampered by time-\n",
      "consuming manual selection of specific cells. The 'Micropilot' software automatically detects cells\n",
      "of interest and launches complex imaging experiments including three-dimensional multicolor time-\n",
      "lapse or fluorescence recovery after photobleaching in live cells. In three independent experimental\n",
      "setups this allowed us to statistically analyze biological processes in detail and is thus a\n",
      "powerful tool for systems biology.\n",
      " {\n",
      "\"main_contribution\": \"automation of fluorescence microscopy-based imaging for systems biology\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the development of a software called 'Micropilot' that automates fluorescence microscopy-based imaging for systems biology. The abstract explains how the software helps in quantitative microscopy by automatically detecting cells of interest and launching complex imaging experiments, including 3D multicolor time-lapse or fluorescence recovery after photobleaching in live cells.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of an imaging technique or method, specifically the automation of fluorescence microscopy-based imaging for systems biology. Hence, the value of 'is_imaging_method' is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "760\n",
      "21278722\n",
      "Five challenges to bringing single-molecule force spectroscopy into living cells.\n",
      "In recent years, single-molecule force spectroscopy techniques have been used to study how inter-\n",
      "and intramolecular interactions control the assembly and functional state of biomolecular machinery\n",
      "in vitro. Here we discuss the problems and challenges that need to be addressed to bring these\n",
      "technologies into living cells and to learn how cellular machinery is controlled in vivo.\n",
      " {\n",
      "\"main_contribution\": \"developing single-molecule force spectroscopy techniques for use in living cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method, specifically single-molecule force spectroscopy, for use in living cells. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "761\n",
      "21297620\n",
      "Visualizing a one-way protein encounter complex by ultrafast single-molecule mixing.\n",
      "We combined rapid microfluidic mixing with single-molecule fluorescence resonance energy transfer to\n",
      "study the folding kinetics of the intrinsically disordered human protein α-synuclein. The time-\n",
      "resolution of 0.2 ms revealed initial collapse of the unfolded protein induced by binding with lipid\n",
      "mimics and subsequent rapid formation of transient structures in the encounter complex. The method\n",
      "also enabled analysis of rapid dissociation and unfolding of weakly bound complexes triggered by\n",
      "massive dilution.\n",
      " {\n",
      "\"main_contribution\": \"study the folding kinetics of the intrinsically disordered human protein α-synuclein\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is studying the folding kinetics of a specific protein, which is achieved through the use of ultrafast single-molecule mixing. This technique allows for the analysis of rapid dissociation and unfolding of weakly bound complexes triggered by massive dilution, providing new insights into the folding kinetics of intrinsically disordered proteins. Therefore, the main contribution is related to the development of an imaging method, specifically ultrafast single-molecule mixing, which enables the study of protein folding kinetics at a high time resolution.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "762\n",
      "21297621\n",
      "Drosophila Brainbow: a recombinase-based fluorescence labeling technique to subdivide neural\n",
      "expression patterns.\n",
      "We developed a multicolor neuron labeling technique in Drosophila melanogaster that combines the\n",
      "power to specifically target different neural populations with the label diversity provided by\n",
      "stochastic color choice. This adaptation of vertebrate Brainbow uses recombination to select one of\n",
      "three epitope-tagged proteins detectable by immunofluorescence. Two copies of this construct yield\n",
      "six bright, separable colors. We used Drosophila Brainbow to study the innervation patterns of\n",
      "multiple antennal lobe projection neuron lineages in the same preparation and to observe the\n",
      "relative trajectories of individual aminergic neurons. Nerve bundles, and even individual neurites\n",
      "hundreds of micrometers long, can be followed with definitive color labeling. We traced motor\n",
      "neurons in the subesophageal ganglion and correlated them to neuromuscular junctions to identify\n",
      "their specific proboscis muscle targets. The ability to independently visualize multiple lineage or\n",
      "neuron projections in the same preparation greatly advances the goal of mapping how neurons connect\n",
      "into circuits.\n",
      " {\n",
      "\"main_contribution\": \"A multicolor neuron labeling technique in Drosophila melanogaster that combines the power to specifically target different neural populations with the label diversity provided by stochastic color choice.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method called Drosophila Brainbow, which uses recombination to select one of three epitope-tagged proteins detectable by immunofluorescence. This technique allows for the independent visualization of multiple lineage or neuron projections in the same preparation, greatly advancing the goal of mapping how neurons connect into circuits. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "763\n",
      "21317909\n",
      "Confined activation and subdiffractive localization enables whole-cell PALM with genetically\n",
      "expressed probes.\n",
      "We demonstrate three-dimensional (3D) super-resolution microscopy in whole fixed cells using\n",
      "photoactivated localization microscopy (PALM). The use of the bright, genetically expressed\n",
      "fluorescent marker photoactivatable monomeric (m)Cherry (PA-mCherry1) in combination with near\n",
      "diffraction-limited confinement of photoactivation using two-photon illumination and 3D localization\n",
      "methods allowed us to investigate a variety of cellular structures at <50 nm lateral and <100 nm\n",
      "axial resolution. Compared to existing methods, we have substantially reduced excitation and\n",
      "bleaching of unlocalized markers, which allows us to use 3D PALM imaging with high localization\n",
      "density in thick structures. Our 3D localization algorithms, which are based on cross-correlation,\n",
      "do not rely on idealized noise models or specific optical configurations. This allows instrument\n",
      "design to be flexible. By generating appropriate fusion constructs and expressing them in Cos7\n",
      "cells, we could image invaginations of the nuclear membrane, vimentin fibrils, the mitochondrial\n",
      "network and the endoplasmic reticulum at depths of greater than 8 μm.\n",
      " {\n",
      "\"main_contribution\": \"three-dimensional (3D) super-resolution microscopy in whole fixed cells using photoactivated localization microscopy (PALM)\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically three-dimensional (3D) super-resolution microscopy in whole fixed cells using photoactivated localization microscopy (PALM). Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "764\n",
      "21336286\n",
      "Ultrahigh-resolution optical trap with single-fluorophore sensitivity.\n",
      "We present a single-molecule instrument that combines a time-shared ultrahigh-resolution dual\n",
      "optical trap interlaced with a confocal fluorescence microscope. In a demonstration experiment, we\n",
      "observed individual single fluorophore-labeled DNA oligonucleotides to bind and unbind complementary\n",
      "DNA suspended between two trapped beads. Simultaneous with the single-fluorophore detection, we\n",
      "clearly observed coincident angstrom-scale changes in tether extension. Fluorescence readout allowed\n",
      "us to determine the duplex melting rate as a function of force. The new instrument will enable the\n",
      "simultaneous measurement of angstrom-scale mechanical motion of individual DNA-binding proteins (for\n",
      "example, single-base-pair stepping of DNA translocases) along with the detection of properties of\n",
      "fluorescently labeled protein (for example, internal configuration).\n",
      " {\n",
      "\"main_contribution\": \"A single-molecule instrument that combines a time-shared ultrahigh-resolution dual optical trap interlaced with a confocal fluorescence microscope.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique or method, specifically an ultrahigh-resolution optical trap with single-fluorophore sensitivity. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "765\n",
      "21378978\n",
      "Rapid three-dimensional isotropic imaging of living cells using Bessel beam plane illumination.\n",
      "A key challenge when imaging living cells is how to noninvasively extract the most spatiotemporal\n",
      "information possible. Unlike popular wide-field and confocal methods, plane-illumination microscopy\n",
      "limits excitation to the information-rich vicinity of the focal plane, providing effective optical\n",
      "sectioning and high speed while minimizing out-of-focus background and premature photobleaching.\n",
      "Here we used scanned Bessel beams in conjunction with structured illumination and/or two-photon\n",
      "excitation to create thinner light sheets (<0.5 μm) better suited to three-dimensional (3D)\n",
      "subcellular imaging. As demonstrated by imaging the dynamics of mitochondria, filopodia, membrane\n",
      "ruffles, intracellular vesicles and mitotic chromosomes in live cells, the microscope currently\n",
      "offers 3D isotropic resolution down to ∼0.3 μm, speeds up to nearly 200 image planes per second and\n",
      "the ability to noninvasively acquire hundreds of 3D data volumes from single living cells\n",
      "encompassing tens of thousands of image frames.\n",
      " {\n",
      "\"main_contribution\": \"Rapid three-dimensional isotropic imaging of living cells using Bessel beam plane illumination\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that uses Bessel beams and structured illumination to create thin light sheets for 3D subcellular imaging. This technique allows for high-speed imaging with minimal background and photobleaching, enabling the acquisition of hundreds of 3D data volumes from single living cells. Therefore, the main contribution is indeed the development of a new imaging method, and the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "766\n",
      "21399636\n",
      "Two-color nanoscopy of three-dimensional volumes by 4Pi detection of stochastically switched\n",
      "fluorophores.\n",
      "We demonstrate three-dimensional (3D) super-resolution imaging of stochastically switched\n",
      "fluorophores distributed across whole cells. By evaluating the higher moments of the diffraction\n",
      "spot provided by a 4Pi detection scheme, single markers can be simultaneously localized with <10 nm\n",
      "precision in three dimensions in a layer of 650 nm thickness at an arbitrarily selected depth in the\n",
      "sample. By splitting the fluorescence light into orthogonal polarization states, our 4Pi setup also\n",
      "facilitates the 3D nanoscopy of multiple fluorophores. Offering a combination of multicolor\n",
      "recording, nanoscale resolution and extended axial depth, our method substantially advances the\n",
      "noninvasive 3D imaging of cells and of other transparent materials.\n",
      " {\n",
      "\"main_contribution\": \"Three-dimensional super-resolution imaging of stochastically switched fluorophores distributed across whole cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows for three-dimensional super-resolution imaging of stochastically switched fluorophores distributed across whole cells. This method uses a 4Pi detection scheme to localize single markers with <10 nm precision in three dimensions in a layer of 650 nm thickness at an arbitrarily selected depth in the sample. Additionally, the method allows for the 3D nanoscopy of multiple fluorophores by splitting the fluorescence light into orthogonal polarization states. Therefore, the main contribution is the development of a new imaging technique, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "767\n",
      "21399637\n",
      "Simultaneous assessment of rodent behavior and neurochemistry using a miniature positron emission\n",
      "tomograph.\n",
      "Positron emission tomography (PET) neuroimaging and behavioral assays in rodents are widely used in\n",
      "neuroscience. PET gives insights into the molecular processes of neuronal communication, and\n",
      "behavioral methods analyze the actions that are associated with such processes. These methods have\n",
      "not been directly integrated, because PET studies in animals have until now required general\n",
      "anesthesia to immobilize the subject, which precludes behavioral studies. We present a method for\n",
      "imaging awake, behaving rats with PET that allows the simultaneous study of behavior. Key components\n",
      "include the 'rat conscious animal PET' or RatCAP, a miniature portable PET scanner that is mounted\n",
      "on the rat's head, a mobility system that allows considerable freedom of movement, radiotracer\n",
      "administration techniques and methods for quantifying behavior and correlating the two data sets.\n",
      "The simultaneity of the PET and behavioral data provides a multidimensional tool for studying the\n",
      "functions of different brain regions and their molecular constituents.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous assessment of rodent behavior and neurochemistry using a miniature positron emission tomograph\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a miniature positron emission tomograph (PET) scanner that can be used to image awake and behaving rats. This allows for the simultaneous study of behavior and neurochemistry, providing a multidimensional tool for studying the functions of different brain regions and their molecular constituents. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "768\n",
      "21423192\n",
      "High-resolution Fourier-transform infrared chemical imaging with multiple synchrotron beams.\n",
      "Conventional Fourier-transform infrared (FTIR) microspectroscopic systems are limited by an\n",
      "inevitable trade-off between spatial resolution, acquisition time, signal-to-noise ratio (SNR) and\n",
      "sample coverage. We present an FTIR imaging approach that substantially extends current capabilities\n",
      "by combining multiple synchrotron beams with wide-field detection. This advance allows truly\n",
      "diffraction-limited high-resolution imaging over the entire mid-infrared spectrum with high chemical\n",
      "sensitivity and fast acquisition speed while maintaining high-quality SNR.\n",
      " {\n",
      "\"main_contribution\": \"high-resolution Fourier-transform infrared chemical imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "769\n",
      "21451510\n",
      "Development and applications of single-cell transcriptome analysis.\n",
      "Dissecting the relationship between genotype and phenotype is one of the central goals in\n",
      "developmental biology and medicine. Transcriptome analysis is a powerful strategy to connect\n",
      "genotype to phenotype of a cell. Here we review the history, progress, potential applications and\n",
      "future developments of single-cell transcriptome analysis. In combination with live cell imaging and\n",
      "lineage tracing, it will be possible to decipher the full gene expression network underlying\n",
      "physiological functions of individual cells in embryos and adults, and to study diseases.\n",
      " {\n",
      "\"main_contribution\": \"Development of single-cell transcriptome analysis\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development and applications of single-cell transcriptome analysis. This technique allows researchers to study the relationship between genotype and phenotype at the level of individual cells, which is a central goal in developmental biology and medicine. While live cell imaging and lineage tracing are mentioned as potential complementary techniques, the main contribution of the paper is not the development of a new imaging method, but rather the development of a powerful strategy to connect genotype to phenotype using transcriptome analysis. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "770\n",
      "21451512\n",
      "Validating transcripts with probes and imaging technology.\n",
      "High-throughput gene expression screens provide a quantitative picture of the average expression\n",
      "signature of biological samples. However, the analysis of spatial gene expression patterns with\n",
      "single-cell resolution requires quantitative in situ measurement techniques. Here we describe recent\n",
      "technological advances in RNA fluorescence in situ hybridization (FISH) techniques that facilitate\n",
      "detection of individual fluorescently labeled mRNA molecules of practically any endogenous gene.\n",
      "These methods, which are based on advances in probe design, imaging technology and image processing,\n",
      "enable the absolute measurement of transcript abundance in individual cells with single-molecule\n",
      "resolution.\n",
      " {\n",
      "\"main_contribution\": \"absolute measurement of transcript abundance in individual cells with single-molecule resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that enables the absolute measurement of transcript abundance in individual cells with single-molecule resolution. This method is based on advances in probe design, imaging technology, and image processing. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "771\n",
      "21451514\n",
      "Long-term single-cell imaging of mammalian stem cells.\n",
      "Continuous long-term single-cell observation provides insight into the molecular control of cell\n",
      "fate. This is particularly important for rare and heterogeneous populations of cells, such as\n",
      "mammalian stem cells. The current lack of usable off-the-shelf hardware and software for such\n",
      "experiments makes their implementation technically challenging. Here I discuss the need for\n",
      "continuous single-cell quantification to understand molecular cell fate control as well as\n",
      "organizational and technical solutions for long-term imaging and tracking of stem cells.\n",
      " {\n",
      "\"main_contribution\": \"Long-term single-cell imaging of mammalian stem cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method for long-term single-cell imaging of mammalian stem cells, which is stated in the abstract as \"continuous long-term single-cell observation.\" The author highlights the need for this technique to understand molecular cell fate control and proposes solutions for long-term imaging and tracking of stem cells. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "772\n",
      "21460824\n",
      "A genome-scale shRNA resource for transgenic RNAi in Drosophila.\n",
      "Existing transgenic RNAi resources in Drosophila melanogaster based on long double-stranded hairpin\n",
      "RNAs are powerful tools for functional studies, but they are ineffective in gene knockdown during\n",
      "oogenesis, an important model system for the study of many biological questions. We show that\n",
      "shRNAs, modeled on an endogenous microRNA, are extremely effective at silencing gene expression\n",
      "during oogenesis. We also describe our progress toward building a genome-wide shRNA resource.\n",
      " {\n",
      "\"main_contribution\": \"A genome-scale shRNA resource for transgenic RNAi in Drosophila\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the paper describe the development of a new resource for transgenic RNAi in Drosophila using shRNAs. The main contribution of the paper is the creation of this genome-scale shRNA resource, which allows for effective gene knockdown during oogenesis. As this is not a development of a new imaging method, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "773\n",
      "21473015\n",
      "A versatile in vivo system for directed dissection of gene expression patterns.\n",
      "Tissue-specific gene expression using the upstream activating sequence (UAS)–GAL4 binary system has\n",
      "facilitated genetic dissection of many biological processes in Drosophila melanogaster. Refining\n",
      "GAL4 expression patterns or independently manipulating multiple cell populations using additional\n",
      "binary systems are common experimental goals. To simplify these processes, we developed a\n",
      "convertible genetic platform, the integrase swappable in vivo targeting element (InSITE) system.\n",
      "This approach allows GAL4 to be replaced with any other sequence, placing different genetic\n",
      "effectors under the control of the same regulatory elements. Using InSITE, GAL4 can be replaced with\n",
      "LexA or QF, allowing an expression pattern to be repurposed. GAL4 can also be replaced with GAL80 or\n",
      "split-GAL4 hemi-drivers, allowing intersectional approaches to refine expression patterns. The\n",
      "exchanges occur through efficient in vivo manipulations, making it possible to generate many swaps\n",
      "in parallel. This system is modular, allowing future genetic tools to be easily incorporated into\n",
      "the existing framework.\n",
      " {\n",
      "\"main_contribution\": \"A convertible genetic platform, the integrase swappable in vivo targeting element (InSITE) system, was developed to simplify refining GAL4 expression patterns or independently manipulating multiple cell populations using additional binary systems in Drosophila melanogaster.\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "774\n",
      "21473020\n",
      "Imaging impedance.\n",
      "A label-free microscopy technique based on electrochemical impedance offers a new way of studying\n",
      "electrochemical processes in single cells.\n",
      " {\n",
      "\"main_contribution\": \"a label-free microscopy technique\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique based on electrochemical impedance, which allows for the study of electrochemical processes in single cells. Therefore, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "775\n",
      "21478861\n",
      "A parallel microfluidic flow cytometer for high-content screening.\n",
      "A parallel microfluidic cytometer (PMC) uses a high-speed scanning photomultiplier-based detector to\n",
      "combine low-pixel-count, one-dimensional imaging with flow cytometry. The 384 parallel flow channels\n",
      "of the PMC decouple count rate from signal-to-noise ratio. Using six-pixel one-dimensional images,\n",
      "we investigated protein localization in a yeast model for human protein misfolding diseases and\n",
      "demonstrated the feasibility of a nuclear-translocation assay in Chinese hamster ovary (CHO) cells\n",
      "expressing an NFκB-EGFP reporter.\n",
      " {\n",
      "\"main_contribution\": \"A parallel microfluidic flow cytometer for high-content screening\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique, specifically a parallel microfluidic flow cytometer that uses a high-speed scanning photomultiplier-based detector to combine low-pixel-count, one-dimensional imaging with flow cytometry. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "776\n",
      "21516115\n",
      "Adaptive informatics for multifactorial and high-content biological data.\n",
      "Whereas genomic data are universally machine-readable, data from imaging, multiplex biochemistry,\n",
      "flow cytometry and other cell- and tissue-based assays usually reside in loosely organized files of\n",
      "poorly documented provenance. This arises because the relational databases used in genomic research\n",
      "are difficult to adapt to rapidly evolving experimental designs, data formats and analytic\n",
      "algorithms. Here we describe an adaptive approach to managing experimental data based on\n",
      "semantically typed data hypercubes (SDCubes) that combine hierarchical data format 5 (HDF5) and\n",
      "extensible markup language (XML) file types. We demonstrate the application of SDCube-based storage\n",
      "using ImageRail, a software package for high-throughput microscopy. Experimental design and its day-\n",
      "to-day evolution, not rigid standards, determine how ImageRail data are organized in SDCubes. We\n",
      "applied ImageRail to collect and analyze drug dose-response landscapes in human cell lines at\n",
      "single-cell resolution.\n",
      " {\n",
      "\"main_contribution\": \"Adaptive informatics for multifactorial and high-content biological data\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing an adaptive approach to managing experimental data based on semantically typed data hypercubes (SDCubes) that combine hierarchical data format 5 (HDF5) and extensible markup language (XML) file types. This new imaging method is described as being used in ImageRail, a software package for high-throughput microscopy. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "777\n",
      "21527931\n",
      "Two-photon absorption properties of fluorescent proteins.\n",
      "Two-photon excitation of fluorescent proteins is an attractive approach for imaging living systems.\n",
      "Today researchers are eager to know which proteins are the brightest and what the best excitation\n",
      "wavelengths are. Here we review the two-photon absorption properties of a wide variety of\n",
      "fluorescent proteins, including new far-red variants, to produce a comprehensive guide to choosing\n",
      "the right fluorescent protein and excitation wavelength for two-photon applications.\n",
      " {\n",
      "\"main_contribution\": \"comprehensive guide to choosing the right fluorescent protein and excitation wavelength for two-photon applications\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a comprehensive guide for choosing the right fluorescent protein and excitation wavelength for two-photon imaging applications. This guide is based on the two-photon absorption properties of various fluorescent proteins, including new far-red variants. As such, the main contribution is related to imaging technique development, specifically two-photon imaging. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "778\n",
      "21532582\n",
      "BrainAligner: 3D registration atlases of Drosophila brains.\n",
      "Analyzing Drosophila melanogaster neural expression patterns in thousands of three-dimensional image\n",
      "stacks of individual brains requires registering them into a canonical framework based on a fiducial\n",
      "reference of neuropil morphology. Given a target brain labeled with predefined landmarks, the\n",
      "BrainAligner program automatically finds the corresponding landmarks in a subject brain and maps it\n",
      "to the coordinate system of the target brain via a deformable warp. Using a neuropil marker (the\n",
      "antibody nc82) as a reference of the brain morphology and a target brain that is itself a\n",
      "statistical average of data for 295 brains, we achieved a registration accuracy of 2 μm on average,\n",
      "permitting assessment of stereotypy, potential connectivity and functional mapping of the adult\n",
      "fruit fly brain. We used BrainAligner to generate an image pattern atlas of 2954 registered brains\n",
      "containing 470 different expression patterns that cover all the major compartments of the fly brain.\n",
      " {\n",
      "\"main_contribution\": \"BrainAligner: 3D registration atlases of Drosophila brains\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called BrainAligner, which enables the registration of thousands of three-dimensional image stacks of individual brains into a canonical framework based on a fiducial reference of neuropil morphology. This technique allows for the assessment of stereotypy, potential connectivity, and functional mapping of the adult fruit fly brain. Therefore, the main contribution is identified as \"BrainAligner: 3D registration atlases of Drosophila brains\".\n",
      "\n",
      "Since the paper's main contribution is the development of a new imaging method, the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "779\n",
      "21552254\n",
      "Fast, three-dimensional super-resolution imaging of live cells.\n",
      "We report super-resolution fluorescence imaging of live cells with high spatiotemporal resolution\n",
      "using stochastic optical reconstruction microscopy (STORM). By labeling proteins either directly or\n",
      "via SNAP tags with photoswitchable dyes, we obtained two-dimensional (2D) and 3D super-resolution\n",
      "images of living cells, using clathrin-coated pits and the transferrin cargo as model systems.\n",
      "Bright, fast-switching probes enabled us to achieve 2D imaging at spatial resolutions of ∼25 nm and\n",
      "temporal resolutions as fast as 0.5 s. We also demonstrated live-cell 3D super-resolution imaging.\n",
      "We obtained 3D spatial resolution of ∼30 nm in the lateral direction and ∼50 nm in the axial\n",
      "direction at time resolutions as fast as 1-2 s with several independent snapshots. Using\n",
      "photoswitchable dyes with distinct emission wavelengths, we also demonstrated two-color 3D super-\n",
      "resolution imaging in live cells. These imaging capabilities open a new window for characterizing\n",
      "cellular structures in living cells at the ultrastructural level.\n",
      " {\n",
      "\"main_contribution\": \"Fast, three-dimensional super-resolution imaging of live cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method - stochastic optical reconstruction microscopy (STORM) - which enables fast, three-dimensional super-resolution imaging of live cells. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "780\n",
      "21602799\n",
      "High-throughput analysis of single hematopoietic stem cell proliferation in microfluidic cell\n",
      "culture arrays.\n",
      "Heterogeneity in cell populations poses a major obstacle to understanding complex biological\n",
      "processes. Here we present a microfluidic platform containing thousands of nanoliter-scale chambers\n",
      "suitable for live-cell imaging studies of clonal cultures of nonadherent cells with precise control\n",
      "of the conditions, capabilities for in situ immunostaining and recovery of viable cells. We show\n",
      "that this platform mimics conventional cultures in reproducing the responses of various types of\n",
      "primitive mouse hematopoietic cells with retention of their functional properties, as demonstrated\n",
      "by subsequent in vitro and in vivo (transplantation) assays of recovered cells. The automated medium\n",
      "exchange of this system made it possible to define when Steel factor stimulation is first required\n",
      "by adult hematopoietic stem cells in vitro as the point of exit from quiescence. This technology\n",
      "will offer many new avenues to interrogate otherwise inaccessible mechanisms governing mammalian\n",
      "cell growth and fate decisions.\n",
      " {\n",
      "\"main_contribution\": \"High-throughput analysis of single hematopoietic stem cell proliferation in microfluidic cell culture arrays\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a microfluidic platform for high-throughput analysis of single hematopoietic stem cell proliferation, which includes live-cell imaging studies and in situ immunostaining. This platform allows for precise control of conditions and recovery of viable cells, making it suitable for interrogating mechanisms governing mammalian cell growth and fate decisions. Therefore, the main contribution is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "781\n",
      "21642963\n",
      "Sharper low-power STED nanoscopy by time gating.\n",
      "Applying pulsed excitation together with time-gated detection improves the fluorescence on-off\n",
      "contrast in continuous-wave stimulated emission depletion (CW-STED) microscopy, thus revealing finer\n",
      "details in fixed and living cells using moderate light intensities. This method also enables super-\n",
      "resolution fluorescence correlation spectroscopy with CW-STED beams, as demonstrated by quantifying\n",
      "the dynamics of labeled lipid molecules in the plasma membrane of living cells.\n",
      " {\n",
      "\"main_contribution\": \"Sharper low-power STED nanoscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a modification to continuous-wave stimulated emission depletion (CW-STED) microscopy that improves the fluorescence on-off contrast and allows for finer details to be revealed in fixed and living cells using moderate light intensities. This method also enables super-resolution fluorescence correlation spectroscopy with CW-STED beams. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "782\n",
      "21642964\n",
      "High-throughput behavioral analysis in C. elegans.\n",
      "We designed a real-time computer vision system, the Multi-Worm Tracker (MWT), which can\n",
      "simultaneously quantify the behavior of dozens of Caenorhabditis elegans on a Petri plate at video\n",
      "rates. We examined three traditional behavioral paradigms using this system: spontaneous movement on\n",
      "food, where the behavior changes over tens of minutes; chemotaxis, where turning events must be\n",
      "detected accurately to determine strategy; and habituation of response to tap, where the response is\n",
      "stochastic and changes over time. In each case, manual analysis or automated single-worm tracking\n",
      "would be tedious and time-consuming, but the MWT system allowed rapid quantification of behavior\n",
      "with minimal human effort. Thus, this system will enable large-scale forward and reverse genetic\n",
      "screens for complex behaviors.\n",
      " {\n",
      "\"main_contribution\": \"A real-time computer vision system for high-throughput behavioral analysis in C. elegans\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a real-time computer vision system called the Multi-Worm Tracker (MWT) for high-throughput behavioral analysis in Caenorhabditis elegans. The MWT system is designed to simultaneously quantify the behavior of dozens of C. elegans on a Petri plate at video rates, allowing for rapid quantification of behavior with minimal human effort. The abstract specifically mentions that the system was used to examine three traditional behavioral paradigms: spontaneous movement on food, chemotaxis, and habituation of response to tap. Therefore, the main contribution of the paper is the development of a new imaging method, specifically the MWT system, for high-throughput behavioral analysis in C. elegans.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "783\n",
      "21642966\n",
      "Near-infrared branding efficiently correlates light and electron microscopy.\n",
      "The correlation of light and electron microscopy of complex tissues remains a major challenge. Here\n",
      "we report near-infrared branding (NIRB), which facilitates such correlation by using a pulsed, near-\n",
      "infrared laser to create defined fiducial marks in three dimensions in fixed tissue. As these marks\n",
      "are fluorescent and can be photo-oxidized to generate electron contrast, they can guide re-\n",
      "identification of previously imaged structures as small as dendritic spines by electron microscopy.\n",
      " {\n",
      "\"main_contribution\": \"Near-infrared branding technique to correlate light and electron microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called near-infrared branding (NIRB), which enables the correlation of light and electron microscopy of complex tissues. This technique uses a pulsed, near-infrared laser to create defined fiducial marks in three dimensions in fixed tissue, which can be fluorescent and photo-oxidized to generate electron contrast, allowing for the re-identification of previously imaged structures as small as dendritic spines by electron microscopy. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "784\n",
      "21666670\n",
      "Fluorogenic DNA sequencing in PDMS microreactors.\n",
      "We developed a multiplex sequencing-by-synthesis method combining terminal phosphate-labeled\n",
      "fluorogenic nucleotides (TPLFNs) and resealable polydimethylsiloxane (PDMS) microreactors. In the\n",
      "presence of phosphatase, primer extension by DNA polymerase using nonfluorescent TPLFNs generates\n",
      "fluorophores, which are confined in the microreactors and detected. We immobilized primed DNA\n",
      "templates in the microreactors, then sequentially introduced one of the four identically labeled\n",
      "TPLFNs, sealed the microreactors and recorded a fluorescence image after template-directed primer\n",
      "extension. With cycle times of <10 min, we demonstrate 30 base reads with ∼99% raw accuracy. Our\n",
      "'fluorogenic pyrosequencing' offers benefits of pyrosequencing, such as rapid turnaround, one-color\n",
      "detection and generation of native DNA, along with high detection sensitivity and simplicity of\n",
      "parallelization because simultaneous real-time monitoring of all microreactors is not required.\n",
      " {\n",
      "\"main_contribution\": \"Fluorogenic DNA sequencing in PDMS microreactors\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a multiplex sequencing-by-synthesis method combining terminal phosphate-labeled fluorogenic nucleotides (TPLFNs) and resealable polydimethylsiloxane (PDMS) microreactors. This is evident from the language used in the abstract, which highlights the novelty of the method and its advantages over existing techniques. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "785\n",
      "21725300\n",
      "Functional ultrasound imaging of the brain.\n",
      "We present functional ultrasound (fUS), a method for imaging transient changes in blood volume in\n",
      "the whole brain at better spatiotemporal resolution than with other functional brain imaging\n",
      "modalities. fUS uses plane-wave illumination at high frame rate and can measure blood volumes in\n",
      "smaller vessels than previous ultrasound methods. fUS identifies regions of brain activation and was\n",
      "used to image whisker-evoked cortical and thalamic responses and the propagation of epileptiform\n",
      "seizures in the rat brain.\n",
      " {\n",
      "\"main_contribution\": \"Functional ultrasound imaging of the brain\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "786\n",
      "21725301\n",
      "Two-photon polarization microscopy reveals protein structure and function.\n",
      "Membrane proteins are a large, diverse group of proteins, serving a multitude of cellular functions.\n",
      "They are difficult to study because of their requirement of a lipid membrane for function. Here we\n",
      "show that two-photon polarization microscopy can take advantage of the cell membrane requirement to\n",
      "yield insights into membrane protein structure and function, in living cells and organisms. The\n",
      "technique allows sensitive imaging of G-protein activation, changes in intracellular calcium\n",
      "concentration and other processes, and is not limited to membrane proteins. Conveniently, many\n",
      "suitable probes for two-photon polarization microscopy already exist.\n",
      " {\n",
      "\"main_contribution\": \"Two-photon polarization microscopy reveals protein structure and function\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the main contribution of the paper as revealing protein structure and function using two-photon polarization microscopy. The abstract provides background information about membrane proteins being difficult to study, and explains how two-photon polarization microscopy can be used to overcome this challenge. It also highlights the versatility of the technique and mentions that suitable probes already exist.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new imaging method (two-photon polarization microscopy) for studying membrane proteins in living cells and organisms. This method allows for sensitive imaging of various cellular processes, including G-protein activation and changes in intracellular calcium concentration. As such, the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "787\n",
      "21765409\n",
      "Deep and fast live imaging with two-photon scanned light-sheet microscopy.\n",
      "We implemented two-photon scanned light-sheet microscopy, combining nonlinear excitation with\n",
      "orthogonal illumination of light-sheet microscopy, and showed its excellent performance for in vivo,\n",
      "cellular-resolution, three-dimensional imaging of large biological samples. Live imaging of fruit\n",
      "fly and zebrafish embryos confirmed that the technique can be used to image up to twice deeper than\n",
      "with one-photon light-sheet microscopy and more than ten times faster than with point-scanning two-\n",
      "photon microscopy without compromising normal biology.\n",
      " {\n",
      "\"main_contribution\": \"Two-photon scanned light-sheet microscopy for deep and fast live imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work being discussed, specifically the development of a new imaging technique called two-photon scanned light-sheet microscopy. \n",
      "\n",
      "The abstract provides background information about the limitations of previous imaging techniques and explains how the new method combines nonlinear excitation with orthogonal illumination of light-sheet microscopy. It also describes the results of using this technique on live fruit fly and zebrafish embryos, which showed that it can image deeper and faster than previous methods without affecting normal biology. \n",
      "\n",
      "Therefore, the main contribution of the paper is the development of two-photon scanned light-sheet microscopy, which is an imaging technique. This is recorded as a true value in the \"is_imaging_method\" variable.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "788\n",
      "21804536\n",
      "A photoswitchable orange-to-far-red fluorescent protein, PSmOrange.\n",
      "We report a photoswitchable monomeric Orange (PSmOrange) protein that is initially orange\n",
      "(excitation, 548 nm; emission, 565 nm) but becomes far-red (excitation, 636 nm; emission, 662 nm)\n",
      "after irradiation with blue-green light. Compared to its parental orange proteins, PSmOrange has\n",
      "greater brightness, faster maturation, higher photoconversion contrast and better photostability.\n",
      "The red-shifted spectra of both forms of PSmOrange enable its simultaneous use with cyan-to-green\n",
      "photoswitchable proteins to study four intracellular populations. Photoconverted PSmOrange has, to\n",
      "our knowledge, the most far-red excitation peak of all GFP-like fluorescent proteins, provides\n",
      "diffraction-limited and super-resolution imaging in the far-red light range, is optimally excited\n",
      "with common red lasers, and can be photoconverted subcutaneously in a mouse. PSmOrange\n",
      "photoswitching occurs via a two-step photo-oxidation process, which causes cleavage of the\n",
      "polypeptide backbone. The far-red fluorescence of photoconverted PSmOrange results from a new\n",
      "chromophore containing N-acylimine with a co-planar carbon-oxygen double bond.\n",
      " {\n",
      "\"main_contribution\": \"A photoswitchable orange-to-far-red fluorescent protein, PSmOrange.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a photoswitchable fluorescent protein called PSmOrange. This protein can be used for super-resolution imaging in the far-red light range and can be photoconverted subcutaneously in a mouse. The far-red fluorescence of photoconverted PSmOrange results from a new chromophore containing N-acylimine with a co-planar carbon-oxygen double bond. Therefore, the is_imaging_method variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "789\n",
      "21822273\n",
      "Revealing off-target cleavage specificities of zinc-finger nucleases by in vitro selection.\n",
      "Engineered zinc-finger nucleases (ZFNs) are promising tools for genome manipulation, and determining\n",
      "off-target cleavage sites of these enzymes is of great interest. We developed an in vitro selection\n",
      "method that interrogates 10(11) DNA sequences for cleavage by active, dimeric ZFNs. The method\n",
      "revealed hundreds of thousands of DNA sequences, some present in the human genome, that can be\n",
      "cleaved in vitro by two ZFNs: CCR5-224 and VF2468, which target the endogenous human CCR5 and VEGFA\n",
      "genes, respectively. Analysis of identified sites in one cultured human cell line revealed\n",
      "CCR5-224-induced changes at nine off-target loci, though this remains to be tested in other relevant\n",
      "cell types. Similarly, we observed 31 off-target sites cleaved by VF2468 in cultured human cells.\n",
      "Our findings establish an energy compensation model of ZFN specificity in which excess binding\n",
      "energy contributes to off-target ZFN cleavage and suggest strategies for the improvement of future\n",
      "ZFN design.\n",
      " {\n",
      "\"main_contribution\": \"identifying off-target cleavage specificities of zinc-finger nucleases using an in vitro selection method\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is identifying off-target cleavage specificities of zinc-finger nucleases using an in vitro selection method. This is described in sentences (B) and (D) of the abstract. The method used to achieve this goal is described in sentence (C). Sentence (A) provides background information, while sentence (E) concludes that the findings establish an energy compensation model of ZFN specificity.\n",
      "\n",
      "Since the main contribution of the paper does not involve the development of a new imaging technique or method, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "790\n",
      "21878918\n",
      "Beyond the rainbow: new fluorescent proteins brighten the infrared scene.\n",
      "Two fluorescent proteins that emit in the far-red and infrared range for imaging applications in\n",
      "cells and in vivo are described.\n",
      " {\n",
      "\"main_contribution\": \"two fluorescent proteins for imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "791\n",
      "21909101\n",
      "Antiviral restriction factor transgenesis in the domestic cat.\n",
      "Studies of the domestic cat have contributed to many scientific advances, including the present\n",
      "understanding of the mammalian cerebral cortex. A practical capability for cat transgenesis is\n",
      "needed to realize the distinctive potential of research on this neurobehaviorally complex,\n",
      "accessible species for advancing human and feline health. For example, humans and cats are afflicted\n",
      "with pandemic AIDS lentiviruses that are susceptible to species-specific restriction factors. Here\n",
      "we introduced genes encoding such a factor, rhesus macaque TRIMCyp, and eGFP, into the cat germline.\n",
      "The method establishes gamete-targeted transgenesis for the first time in a carnivore. We observed\n",
      "uniformly transgenic outcomes, widespread expression, no mosaicism and no F1 silencing. TRIMCyp\n",
      "transgenic cat lymphocytes resisted feline immunodeficiency virus replication. This capability to\n",
      "experimentally manipulate the genome of an AIDS-susceptible species can be used to test the\n",
      "potential of restriction factors for HIV gene therapy and to build models of other infectious and\n",
      "noninfectious diseases.\n",
      " {\n",
      "\"main_contribution\": \"The development of a practical capability for cat transgenesis to realize the distinctive potential of research on this neurobehaviorally complex, accessible species for advancing human and feline health\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a practical capability for cat transgenesis, which is not an imaging method. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "792\n",
      "21909102\n",
      "Miniaturized integration of a fluorescence microscope.\n",
      "The light microscope is traditionally an instrument of substantial size and expense. Its\n",
      "miniaturized integration would enable many new applications based on mass-producible, tiny\n",
      "microscopes. Key prospective usages include brain imaging in behaving animals for relating cellular\n",
      "dynamics to animal behavior. Here we introduce a miniature (1.9 g) integrated fluorescence\n",
      "microscope made from mass-producible parts, including a semiconductor light source and sensor. This\n",
      "device enables high-speed cellular imaging across ∼0.5 mm2 areas in active mice. This capability\n",
      "allowed concurrent tracking of Ca2+ spiking in >200 Purkinje neurons across nine cerebellar\n",
      "microzones. During mouse locomotion, individual microzones exhibited large-scale, synchronized Ca2+\n",
      "spiking. This is a mesoscopic neural dynamic missed by prior techniques for studying the brain at\n",
      "other length scales. Overall, the integrated microscope is a potentially transformative technology\n",
      "that permits distribution to many animals and enables diverse usages, such as portable diagnostics\n",
      "or microscope arrays for large-scale screens.\n",
      " {\n",
      "\"main_contribution\": \"miniaturized integration of a fluorescence microscope\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a miniature fluorescence microscope that can be used for imaging in behaving animals, specifically for tracking cellular dynamics in the brain. This is an imaging method, so the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "793\n",
      "21926998\n",
      "Probing protein heterogeneity in the plasma membrane using PALM and pair correlation analysis.\n",
      "Photoactivated localization microscopy (PALM) is a powerful approach for investigating protein\n",
      "organization, yet tools for quantitative, spatial analysis of PALM datasets are largely missing.\n",
      "Combining pair-correlation analysis with PALM (PC-PALM), we provide a method to analyze complex\n",
      "patterns of protein organization across the plasma membrane without determination of absolute\n",
      "protein numbers. The approach uses an algorithm to distinguish a single protein with multiple\n",
      "appearances from clusters of proteins. This enables quantification of different parameters of\n",
      "spatial organization, including the presence of protein clusters, their size, density and abundance\n",
      "in the plasma membrane. Using this method, we demonstrate distinct nanoscale organization of plasma-\n",
      "membrane proteins with different membrane anchoring and lipid partitioning characteristics in COS-7\n",
      "cells, and show dramatic changes in glycosylphosphatidylinositol (GPI)-anchored protein arrangement\n",
      "under varying perturbations. PC-PALM is thus an effective tool with broad applicability for analysis\n",
      "of protein heterogeneity and function, adaptable to other single-molecule strategies.\n",
      " {\n",
      "\"main_contribution\": \"development of a method to analyze complex patterns of protein organization across the plasma membrane using PALM and pair correlation analysis\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method called PC-PALM, which combines pair-correlation analysis with PALM to analyze protein organization in the plasma membrane. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "794\n",
      "21946668\n",
      "Rapid and robust generation of functional oligodendrocyte progenitor cells from epiblast stem cells.\n",
      "Myelin-related disorders such as multiple sclerosis and leukodystrophies, for which restoration of\n",
      "oligodendrocyte function would be an effective treatment, are poised to benefit greatly from stem\n",
      "cell biology. Progress in myelin repair has been constrained by difficulties in generating pure\n",
      "populations of oligodendrocyte progenitor cells (OPCs) in sufficient quantities. Pluripotent stem\n",
      "cells theoretically provide an unlimited source of OPCs, but current differentiation strategies are\n",
      "poorly reproducible and generate heterogenous populations of cells. Here we provide a platform for\n",
      "the directed differentiation of pluripotent mouse epiblast stem cells (EpiSCs) through defined\n",
      "developmental transitions into a pure population of highly expandable OPCs in 10 d. These OPCs\n",
      "robustly differentiate into myelinating oligodendrocytes in vitro and in vivo. Our results\n",
      "demonstrate that mouse pluripotent stem cells provide a pure population of myelinogenic\n",
      "oligodendrocytes and offer a tractable platform for defining the molecular regulation of\n",
      "oligodendrocyte development and drug screening.\n",
      " {\n",
      "\"main_contribution\": \"a platform for the directed differentiation of pluripotent mouse epiblast stem cells (EpiSCs) through defined developmental transitions into a pure population of highly expandable OPCs in 10 d\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new method for generating functional oligodendrocyte progenitor cells (OPCs) from epiblast stem cells. This method allows for the directed differentiation of pluripotent mouse epiblast stem cells (EpiSCs) into a pure population of highly expandable OPCs in 10 days. The resulting OPCs are able to robustly differentiate into myelinating oligodendrocytes in vitro and in vivo, making them a promising tool for the study of oligodendrocyte development and drug screening.\n",
      "\n",
      "Since the main contribution of this paper is not the development of a new imaging method or technique, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "795\n",
      "21959136\n",
      "Optical sectioning microscopy with planar or structured illumination.\n",
      "A key requirement for performing three-dimensional (3D) imaging using optical microscopes is that\n",
      "they be capable of optical sectioning by distinguishing in-focus signal from out-of-focus\n",
      "background. Common techniques for fluorescence optical sectioning are confocal laser scanning\n",
      "microscopy and two-photon microscopy. But there is increasing interest in alternative optical\n",
      "sectioning techniques, particularly for applications involving high speeds, large fields of view or\n",
      "long-term imaging. In this Review, I examine two such techniques, based on planar illumination or\n",
      "structured illumination. The goal is to describe the advantages and disadvantages of these\n",
      "techniques.\n",
      " {\n",
      "\"main_contribution\": \"describing the advantages and disadvantages of optical sectioning microscopy techniques based on planar or structured illumination\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is to describe the advantages and disadvantages of two imaging techniques, which makes it an imaging method development paper.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "796\n",
      "21983925\n",
      "Live-cell 3D super-resolution imaging in thick biological samples.\n",
      "We demonstrate three-dimensional (3D) super-resolution live-cell imaging through thick specimens\n",
      "(50-150 μm), by coupling far-field individual molecule localization with selective plane\n",
      "illumination microscopy (SPIM). The improved signal-to-noise ratio of selective plane illumination\n",
      "allows nanometric localization of single molecules in thick scattering specimens without activating\n",
      "or exciting molecules outside the focal plane. We report 3D super-resolution imaging of cellular\n",
      "spheroids.\n",
      " {\n",
      "\"main_contribution\": \"three-dimensional (3D) super-resolution live-cell imaging through thick specimens\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically 3D super-resolution live-cell imaging through thick biological samples. This is achieved by coupling far-field individual molecule localization with selective plane illumination microscopy (SPIM). Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "797\n",
      "21983960\n",
      "Proteomic and phosphoproteomic comparison of human ES and iPS cells.\n",
      "Combining high-mass-accuracy mass spectrometry, isobaric tagging and software for multiplexed,\n",
      "large-scale protein quantification, we report deep proteomic coverage of four human embryonic stem\n",
      "cell and four induced pluripotent stem cell lines in biological triplicate. This 24-sample\n",
      "comparison resulted in a very large set of identified proteins and phosphorylation sites in\n",
      "pluripotent cells. The statistical analysis afforded by our approach revealed subtle but\n",
      "reproducible differences in protein expression and protein phosphorylation between embryonic stem\n",
      "cells and induced pluripotent cells. Merging these results with RNA-seq analysis data, we found\n",
      "functionally related differences across each tier of regulation. We also introduce the Stem Cell-\n",
      "Omics Repository (SCOR), a resource to collate and display quantitative information across multiple\n",
      "planes of measurement, including mRNA, protein and post-translational modifications.\n",
      " {\n",
      "\"main_contribution\": \"A resource to collate and display quantitative information across multiple planes of measurement, including mRNA, protein, and post-translational modifications.\"\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a resource called Stem Cell-Omics Repository (SCOR), which is a comprehensive database that combines quantitative information from different levels of molecular measurement, including mRNA, protein, and post-translational modifications. This database allows for the analysis and comparison of data from different samples and provides insights into the differences between embryonic stem cells and induced pluripotent stem cells.\n",
      "\n",
      "The paper does not describe the development of a new imaging technique or method; therefore, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "798\n",
      "21985008\n",
      "Cell type–specific channelrhodopsin-2 transgenic mice for optogenetic dissection of neural circuitry\n",
      "function.\n",
      "Optogenetic methods have emerged as powerful tools for dissecting neural circuit connectivity,\n",
      "function and dysfunction. We used a bacterial artificial chromosome (BAC) transgenic strategy to\n",
      "express the H134R variant of channelrhodopsin-2, ChR2(H134R), under the control of cell\n",
      "type–specific promoter elements. We performed an extensive functional characterization of the newly\n",
      "established VGAT-ChR2(H134R)-EYFP, ChAT-ChR2(H134R)-EYFP, Tph2-ChR2(H134R)-EYFP and\n",
      "Pvalb(H134R)-ChR2-EYFP BAC transgenic mouse lines and demonstrate the utility of these lines for\n",
      "precisely controlling action-potential firing of GABAergic, cholinergic, serotonergic and\n",
      "parvalbumin-expressing neuron subsets using blue light. This resource of cell type–specific\n",
      "ChR2(H134R) mouse lines will facilitate the precise mapping of neuronal connectivity and the\n",
      "dissection of the neural basis of behavior.\n",
      " {\n",
      "\"main_contribution\": \"The development of cell type-specific channelrhodopsin-2 transgenic mice for optogenetic dissection of neural circuitry function\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of transgenic mice that express a light-activated ion channel called ChR2(H134R) in specific cell types, such as GABAergic, cholinergic, serotonergic, and parvalbumin-expressing neurons. The main contribution of the paper is the creation of these transgenic mice, which will allow researchers to use optogenetics to precisely control action potential firing in specific neural populations and thereby map neuronal connectivity and dissect the neural basis of behavior.\n",
      "\n",
      "Since the development of these transgenic mice involves the use of imaging techniques (specifically, optogenetics) to study neural circuitry function, the answer to is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "799\n",
      "22002026\n",
      "Super-resolution 3D microscopy of live whole cells using structured illumination.\n",
      "Three-dimensional (3D) structured-illumination microscopy (SIM) can double the lateral and axial\n",
      "resolution of a wide-field fluorescence microscope but has been too slow for live imaging. Here we\n",
      "apply 3D SIM to living samples and record whole cells at up to 5 s per volume for >50 time points\n",
      "with 120-nm lateral and 360-nm axial resolution. We demonstrate the technique by imaging\n",
      "microtubules in S2 cells and mitochondria in HeLa cells.\n",
      " {\n",
      "\"main_contribution\": \"Imaging microtubules in S2 cells and mitochondria in HeLa cells using 3D structured-illumination microscopy (SIM)\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "800\n",
      "22036746\n",
      "Characterizing RNA dynamics at atomic resolution using solution-state NMR spectroscopy.\n",
      "Many recently discovered noncoding RNAs do not fold into a single native conformation but sample\n",
      "many different conformations along their free-energy landscape to carry out their biological\n",
      "function. Here we review solution-state NMR techniques that measure the structural, kinetic and\n",
      "thermodynamic characteristics of RNA motions spanning picosecond to second timescales at atomic\n",
      "resolution, allowing unprecedented insights into the RNA dynamic structure landscape. From these\n",
      "studies a basic description of the RNA dynamic structure landscape is emerging, bringing new\n",
      "insights into how RNA structures change to carry out their function as well as applications in RNA-\n",
      "targeted drug discovery and RNA bioengineering.\n",
      " {\n",
      "\"main_contribution\": \"Measuring the structural, kinetic and thermodynamic characteristics of RNA motions spanning picosecond to second timescales at atomic resolution using solution-state NMR techniques.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "801\n",
      "22037703\n",
      "Automated protein-DNA interaction screening of Drosophila regulatory elements.\n",
      "Drosophila melanogaster has one of the best characterized metazoan genomes in terms of functionally\n",
      "annotated regulatory elements. To explore how these elements contribute to gene regulation, we need\n",
      "convenient tools to identify the proteins that bind to them. Here we describe the development and\n",
      "validation of a high-throughput yeast one-hybrid platform, which enables screening of DNA elements\n",
      "versus an array of full-length, sequence-verified clones containing over 85% of predicted Drosophila\n",
      "transcription factors. Using six well-characterized regulatory elements, we identified 33\n",
      "transcription factor-DNA interactions of which 27 were previously unidentified. To simultaneously\n",
      "validate these interactions and locate the binding sites of involved transcription factors, we\n",
      "implemented a powerful microfluidics-based approach that enabled us to retrieve DNA-occupancy data\n",
      "for each transcription factor throughout the respective target DNA elements. Finally, we\n",
      "biologically validated several interactions and identified two new regulators of sine oculis gene\n",
      "expression and hence eye development.\n",
      " {\n",
      "\"main_contribution\": \"identification of 33 transcription factor-DNA interactions, including 27 previously unidentified interactions\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development and validation of a high-throughput yeast one-hybrid platform for identifying protein-DNA interactions in Drosophila. The main contribution of the paper is the identification of 33 transcription factor-DNA interactions, including 27 previously unidentified interactions, using six well-characterized regulatory elements. This is described in sentences (B) and (D) of the abstract.\n",
      "\n",
      "The method used to achieve this goal is the development and validation of a high-throughput yeast one-hybrid platform, which is described in sentences (C) of the abstract. However, this method is not an imaging technique or method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "802\n",
      "22037704\n",
      "Visualizing mechanical tension across membrane receptors with a fluorescent sensor.\n",
      "We report a fluorescence-based turn-on sensor for mapping the mechanical strain exerted by specific\n",
      "cell-surface proteins in living cells. The sensor generates force maps with high spatial and\n",
      "temporal resolution using conventional fluorescence microscopy. We demonstrate the approach by\n",
      "mapping mechanical forces during the early stages of regulatory endocytosis of the ligand-activated\n",
      "epidermal growth factor receptor (EGFR).\n",
      " {\n",
      "\"main_contribution\": \"Visualizing mechanical tension across membrane receptors with a fluorescent sensor\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on a fluorescence-based turn-on sensor for mapping mechanical strain exerted by specific cell-surface proteins in living cells. This is evident from the abstract, which describes the approach as generating force maps with high spatial and temporal resolution using conventional fluorescence microscopy. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "803\n",
      "22037705\n",
      "Enhanced yeast one-hybrid assays for high-throughput gene-centered regulatory network mapping.\n",
      "A major challenge in systems biology is to understand the gene regulatory networks that drive\n",
      "development, physiology and pathology. Interactions between transcription factors and regulatory\n",
      "genomic regions provide the first level of gene control. Gateway-compatible yeast one-hybrid (Y1H)\n",
      "assays present a convenient method to identify and characterize the repertoire of transcription\n",
      "factors that can bind a DNA sequence of interest. To delineate genome-scale regulatory networks,\n",
      "however, large sets of DNA fragments need to be processed at high throughput and high coverage. Here\n",
      "we present enhanced Y1H (eY1H) assays that use a robotic mating platform with a set of improved Y1H\n",
      "reagents and automated readout quantification. We demonstrate that eY1H assays provide excellent\n",
      "coverage and identify interacting transcription factors for multiple DNA fragments in a short time.\n",
      "eY1H assays will be an important tool for mapping gene regulatory networks in Caenorhabditis elegans\n",
      "and other model organisms as well as in humans.\n",
      " {\n",
      "\"main_contribution\": \"Enhanced yeast one-hybrid assays for high-throughput gene-centered regulatory network mapping\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for gene regulatory network mapping using enhanced yeast one-hybrid (Y1H) assays. The main contribution of the paper is the development of this new method, which is described in sentences (B) and (C). Sentence (A) provides background information about the challenge of understanding gene regulatory networks, while sentence (D) describes the results of using the enhanced Y1H assays. Sentence (E) discusses the potential impact of the method on mapping gene regulatory networks in various organisms.\n",
      "\n",
      "Since the main contribution of the paper is not the development of a new imaging technique or method, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "804\n",
      "22056676\n",
      "Evaluation of fluorophores for optimal performance in localization-based super-resolution imaging.\n",
      "One approach to super-resolution fluorescence imaging uses sequential activation and localization of\n",
      "individual fluorophores to achieve high spatial resolution. Essential to this technique is the\n",
      "choice of fluorescent probes; the properties of the probes, including photons per switching event,\n",
      "on-off duty cycle, photostability and number of switching cycles, largely dictate the quality of\n",
      "super-resolution images. Although many probes have been reported, a systematic characterization of\n",
      "the properties of these probes and their impact on super-resolution image quality has been described\n",
      "in only a few cases. Here we quantitatively characterized the switching properties of 26 organic\n",
      "dyes and directly related these properties to the quality of super-resolution images. This analysis\n",
      "provides guidelines for characterization of super-resolution probes and a resource for selecting\n",
      "probes based on performance. Our evaluation identified several photoswitchable dyes with good to\n",
      "excellent performance in four independent spectral ranges, with which we demonstrated low-cross-\n",
      "talk, four-color super-resolution imaging.\n",
      " {\n",
      "\"main_contribution\": \"Evaluation of fluorophores for optimal performance in localization-based super-resolution imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the evaluation of different fluorophores and their properties to achieve high spatial resolution in super-resolution imaging. This is a development of an imaging method, specifically a localization-based super-resolution technique that uses sequential activation and localization of individual fluorophores. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "805\n",
      "22081126\n",
      "Cyanine fluorophore derivatives with enhanced photostability.\n",
      "Fluorescence applications requiring high photostability often depend on the use of solution\n",
      "additives to enhance fluorophore performance. Here we demonstrate that the direct or proximal\n",
      "conjugation of cyclooctatetraene (COT), 4-nitrobenzyl alcohol (NBA) or Trolox to the cyanine\n",
      "fluorophore Cy5 dramatically enhanced fluorophore photostability without otherwise affecting its\n",
      "native spectral characteristics. Such conjugation is a powerful means of improving the robustness of\n",
      "fluorescence-based applications demanding long-lived, nonblinking fluorescence emission.\n",
      " {\n",
      "\"main_contribution\": \"Cyanine fluorophore derivatives with enhanced photostability\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of cyanine fluorophore derivatives with enhanced photostability, which can be used to improve the robustness of fluorescence-based applications demanding long-lived, nonblinking fluorescence emission. This is an imaging method, so the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "806\n",
      "22101853\n",
      "Sensitive and specific single-molecule sequencing of 5-hydroxymethylcytosine.\n",
      "We describe strand-specific, base-resolution detection of 5-hydroxymethylcytosine (5-hmC) in genomic\n",
      "DNA with single-molecule sensitivity, combining a bioorthogonal, selective chemical labeling method\n",
      "of 5-hmC with single-molecule, real-time (SMRT) DNA sequencing. The chemical labeling not only\n",
      "allows affinity enrichment of 5-hmC-containing DNA fragments but also enhances the kinetic signal of\n",
      "5-hmC during SMRT sequencing. We applied the approach to sequence 5-hmC in a genomic DNA sample with\n",
      "high confidence.\n",
      " {\n",
      "\"main_contribution\": \"Developing a method for sensitive and specific single-molecule sequencing of 5-hydroxymethylcytosine\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method, specifically a method for sensitive and specific single-molecule sequencing of 5-hydroxymethylcytosine. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "807\n",
      "22120467\n",
      "Optical recording of action potentials in mammalian neurons using a microbial rhodopsin.\n",
      "Reliable optical detection of single action potentials in mammalian neurons has been one of the\n",
      "longest-standing challenges in neuroscience. Here we achieved this goal by using the endogenous\n",
      "fluorescence of a microbial rhodopsin protein, Archaerhodopsin 3 (Arch) from Halorubrum sodomense,\n",
      "expressed in cultured rat hippocampal neurons. This genetically encoded voltage indicator exhibited\n",
      "an approximately tenfold improvement in sensitivity and speed over existing protein-based voltage\n",
      "indicators, with a roughly linear twofold increase in brightness between -150 mV and +150 mV and a\n",
      "sub-millisecond response time. Arch detected single electrically triggered action potentials with an\n",
      "optical signal-to-noise ratio >10. Arch(D95N) lacked endogenous proton pumping and had 50% greater\n",
      "sensitivity than wild type but had a slower response (41 ms). Nonetheless, Arch(D95N) also resolved\n",
      "individual action potentials. Microbial rhodopsin-based voltage indicators promise to enable optical\n",
      "interrogation of complex neural circuits and electrophysiology in systems for which electrode-based\n",
      "techniques are challenging.\n",
      " {\n",
      "\"main_contribution\": \"Optical recording of action potentials in mammalian neurons using a microbial rhodopsin.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method using microbial rhodopsin to optically record action potentials in mammalian neurons. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "808\n",
      "22127219\n",
      "Directed molecular evolution to design advanced red fluorescent proteins.\n",
      "Fluorescent proteins have become indispensable imaging tools for biomedical research. Continuing\n",
      "progress in fluorescence imaging, however, requires probes with additional colors and properties\n",
      "optimized for emerging techniques. Here we summarize strategies for development of red-shifted\n",
      "fluorescent proteins. We discuss possibilities for knowledge-based rational design based on the\n",
      "photochemistry of fluorescent proteins and the position of the chromophore in protein structure. We\n",
      "consider advances in library design by mutagenesis, protein expression systems and instrumentation\n",
      "for high-throughput screening that should yield improved fluorescent proteins for advanced imaging\n",
      "applications.\n",
      " {\n",
      "\"main_contribution\": \"Design of advanced red fluorescent proteins through directed molecular evolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "Sentences identified as background information:\n",
      "- Fluorescent proteins have become indispensable imaging tools for biomedical research. \n",
      "- Continuing progress in fluorescence imaging, however, requires probes with additional colors and properties optimized for emerging techniques.\n",
      "\n",
      "Sentences identified as goals:\n",
      "- Development of red-shifted fluorescent proteins.\n",
      "\n",
      "Sentences identified as methods:\n",
      "- Knowledge-based rational design based on the photochemistry of fluorescent proteins and the position of the chromophore in protein structure.\n",
      "- Library design by mutagenesis, protein expression systems and instrumentation for high-throughput screening.\n",
      "\n",
      "Sentences identified as results:\n",
      "- Improved fluorescent proteins for advanced imaging applications.\n",
      "\n",
      "Main contribution: Design of advanced red fluorescent proteins through directed molecular evolution.\n",
      "\n",
      "Since the main contribution is the development of a new imaging method (directed molecular evolution to design advanced red fluorescent proteins), the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "809\n",
      "22138823\n",
      "mGRASP enables mapping mammalian synaptic connectivity with light microscopy.\n",
      "The GFP reconstitution across synaptic partners (GRASP) technique, based on functional\n",
      "complementation between two nonfluorescent GFP fragments, can be used to detect the location of\n",
      "synapses quickly, accurately and with high spatial resolution. The method has been previously\n",
      "applied in the nematode and the fruit fly but requires substantial modification for use in the\n",
      "mammalian brain. We developed mammalian GRASP (mGRASP) by optimizing transmembrane split-GFP\n",
      "carriers for mammalian synapses. Using in silico protein design, we engineered chimeric synaptic\n",
      "mGRASP fragments that were efficiently delivered to synaptic locations and reconstituted GFP\n",
      "fluorescence in vivo. Furthermore, by integrating molecular and cellular approaches with a\n",
      "computational strategy for the three-dimensional reconstruction of neurons, we applied mGRASP to\n",
      "both long-range circuits and local microcircuits in the mouse hippocampus and thalamocortical\n",
      "regions, analyzing synaptic distribution in single neurons and in dendritic compartments.\n",
      " {\n",
      "\"main_contribution\": \"mammalian GRASP (mGRASP) technique for mapping mammalian synaptic connectivity with light microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new technique called mammalian GRASP (mGRASP) that enables the mapping of mammalian synaptic connectivity using light microscopy. The method is based on functional complementation between two non-fluorescent GFP fragments, which are optimized for mammalian synapses. The authors used in silico protein design to engineer chimeric synaptic mGRASP fragments that can be efficiently delivered to synaptic locations and reconstitute GFP fluorescence in vivo. The technique is applied to both long-range circuits and local microcircuits in the mouse hippocampus and thalamocortical regions, allowing for the analysis of synaptic distribution in single neurons and in dendritic compartments.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of the mGRASP technique, which is a new imaging method for mapping mammalian synaptic connectivity with high spatial resolution. Hence, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "810\n",
      "22138825\n",
      "Bayesian localization microscopy reveals nanoscale podosome dynamics.\n",
      "We describe a localization microscopy analysis method that is able to extract results in live cells\n",
      "using standard fluorescent proteins and xenon arc lamp illumination. Our Bayesian analysis of the\n",
      "blinking and bleaching (3B analysis) method models the entire dataset simultaneously as being\n",
      "generated by a number of fluorophores that may or may not be emitting light at any given time. The\n",
      "resulting technique allows many overlapping fluorophores in each frame and unifies the analysis of\n",
      "the localization from blinking and bleaching events. By modeling the entire dataset, we were able to\n",
      "use each reappearance of a fluorophore to improve the localization accuracy. The high performance of\n",
      "this technique allowed us to reveal the nanoscale dynamics of podosome formation and dissociation\n",
      "throughout an entire cell with a resolution of 50 nm on a 4-s timescale.\n",
      " {\n",
      "\"main_contribution\": \"Bayesian localization microscopy analysis method for live cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a Bayesian localization microscopy analysis method that can be used to study live cells with high resolution. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "811\n",
      "22205518\n",
      "Power tools for gene expression and clonal analysis in Drosophila.\n",
      "The development of two-component expression systems in Drosophila melanogaster, one of the most\n",
      "powerful genetic models, has allowed the precise manipulation of gene function in specific cell\n",
      "populations. These expression systems, in combination with site-specific recombination approaches,\n",
      "have also led to the development of new methods for clonal lineage analysis. We present a hands-on\n",
      "user guide to the techniques and approaches that have greatly increased resolution of genetic\n",
      "analysis in the fly, with a special focus on their application for lineage analysis. Our intention\n",
      "is to provide guidance and suggestions regarding which genetic tools are most suitable for\n",
      "addressing different developmental questions.\n",
      " {\n",
      "\"main_contribution\": \"A hands-on user guide to techniques and approaches for clonal lineage analysis in Drosophila\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is providing a user guide for techniques and approaches for clonal lineage analysis in Drosophila, which does not involve the development of a new imaging method. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "812\n",
      "22205519\n",
      "A guide to analysis of mouse energy metabolism.\n",
      "We present a consolidated view of the complexity and challenges of designing studies for measurement\n",
      "of energy metabolism in mouse models, including a practical guide to the assessment of energy\n",
      "expenditure, energy intake and body composition and statistical analysis thereof. We hope this guide\n",
      "will facilitate comparisons across studies and minimize spurious interpretations of data. We\n",
      "recommend that division of energy expenditure data by either body weight or lean body weight and\n",
      "that presentation of group effects as histograms should be replaced by plotting individual data and\n",
      "analyzing both group and body-composition effects using analysis of covariance (ANCOVA).\n",
      " {\n",
      "\"main_contribution\": \"practical guide to assessment of energy expenditure, energy intake, and body composition in mouse models,\"\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "813\n",
      "22231640\n",
      "Blotting protein complexes from native gels to electron microscopy grids.\n",
      "We report a simple and generic method for the direct transfer of protein complexes separated by\n",
      "native gel electrophoresis to electron microscopy grids. After transfer, sufficient material remains\n",
      "in the gel for identification and characterization by mass spectrometry. The method should\n",
      "facilitate higher-throughput single-particle analysis by substantially reducing the time needed for\n",
      "protein purification, as demonstrated for three complexes from Thermoplasma acidophilum.\n",
      " {\n",
      "\"main_contribution\": \"A simple and generic method for transferring protein complexes separated by native gel electrophoresis to electron microscopy grids\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, specifically a method for transferring protein complexes separated by native gel electrophoresis to electron microscopy grids. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "814\n",
      "22231641\n",
      "Fast two-photon in vivo imaging with three-dimensional random-access scanning in large tissue\n",
      "volumes.\n",
      "The understanding of brain computations requires methods that read out neural activity on different\n",
      "spatial and temporal scales. Following signal propagation and integration across a neuron and\n",
      "recording the concerted activity of hundreds of neurons pose distinct challenges, and the design of\n",
      "imaging systems has been mostly focused on tackling one of the two operations. We developed a high-\n",
      "resolution, acousto-optic two-photon microscope with continuous three-dimensional (3D) trajectory\n",
      "and random-access scanning modes that reaches near-cubic-millimeter scan range and can be adapted to\n",
      "imaging different spatial scales. We performed 3D calcium imaging of action potential\n",
      "backpropagation and dendritic spike forward propagation at sub-millisecond temporal resolution in\n",
      "mouse brain slices. We also performed volumetric random-access scanning calcium imaging of\n",
      "spontaneous and visual stimulation-evoked activity in hundreds of neurons of the mouse visual cortex\n",
      "in vivo. These experiments demonstrate the subcellular and network-scale imaging capabilities of our\n",
      "system.\n",
      " {\n",
      "\"main_contribution\": \"Development of a high-resolution, acousto-optic two-photon microscope with continuous three-dimensional (3D) trajectory and random-access scanning modes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a high-resolution, acousto-optic two-photon microscope with continuous 3D trajectory and random-access scanning modes. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "815\n",
      "22231642\n",
      "Dual-objective STORM reveals three-dimensional filament organization in the actin cytoskeleton.\n",
      "By combining astigmatism imaging with a dual-objective scheme, we improved the image resolution of\n",
      "stochastic optical reconstruction microscopy (STORM) and obtained <10-nm lateral resolution and\n",
      "<20-nm axial resolution when imaging biological specimens. Using this approach, we resolved\n",
      "individual actin filaments in cells and revealed three-dimensional ultrastructure of the actin\n",
      "cytoskeleton. We observed two vertically separated layers of actin networks with distinct structural\n",
      "organizations in sheet-like cell protrusions.\n",
      " {\n",
      "\"main_contribution\": \"Three-dimensional ultrastructure of the actin cytoskeleton\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging technique, specifically an improvement of stochastic optical reconstruction microscopy (STORM) with dual-objective imaging. This technique allows for higher resolution and better 3D structural organization of biological samples, such as actin filaments in cells. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "816\n",
      "22245808\n",
      "Controlling airborne cues to study small animal navigation.\n",
      "Small animals such as nematodes and insects analyze airborne chemical cues to infer the direction of\n",
      "favorable and noxious locations. In these animals, the study of navigational behavior evoked by\n",
      "airborne cues has been limited by the difficulty of precisely controlling stimuli. We present a\n",
      "system that can be used to deliver gaseous stimuli in defined spatial and temporal patterns to\n",
      "freely moving small animals. We used this apparatus, in combination with machine-vision algorithms,\n",
      "to assess and quantify navigational decision making of Drosophila melanogaster larvae in response to\n",
      "ethyl acetate (a volatile attractant) and carbon dioxide (a gaseous repellant).\n",
      " {\n",
      "\"main_contribution\": \"A system for delivering gaseous stimuli in defined spatial and temporal patterns to freely moving small animals, allowing for the study of navigational behavior evoked by airborne cues.\"\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "817\n",
      "22245809\n",
      "Serial two-photon tomography for automated ex vivo mouse brain imaging.\n",
      "Here we describe an automated method, named serial two-photon (STP) tomography, that achieves high-\n",
      "throughput fluorescence imaging of mouse brains by integrating two-photon microscopy and tissue\n",
      "sectioning. STP tomography generates high-resolution datasets that are free of distortions and can\n",
      "be readily warped in three dimensions, for example, for comparing multiple anatomical tracings. This\n",
      "method opens the door to routine systematic studies of neuroanatomy in mouse models of human brain\n",
      "disorders.\n",
      " {\n",
      "\"main_contribution\": \"Automated high-throughput fluorescence imaging method for mouse brains\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work as a method for automated ex vivo mouse brain imaging using serial two-photon tomography. The abstract goes further to explain that this method integrates two-photon microscopy and tissue sectioning, generating high-resolution datasets free of distortions and readily warped in three dimensions. This allows for comparing multiple anatomical tracings and opens up the possibility of routine systematic studies of neuroanatomy in mouse models of human brain disorders.\n",
      "\n",
      "Therefore, the main contribution of this paper is the development of an automated high-throughput fluorescence imaging method for mouse brains using serial two-photon tomography. As this is a new imaging technique, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "818\n",
      "22266542\n",
      "Chronic in vivo imaging in the mouse spinal cord using an implanted chamber.\n",
      "Understanding and treatment of spinal cord pathology is limited in part by a lack of time-lapse in\n",
      "vivo imaging strategies at the cellular level. We developed a chronically implanted spinal chamber\n",
      "and surgical procedure suitable for time-lapse in vivo multiphoton microscopy of mouse spinal cord\n",
      "without the need for repeat surgical procedures. We routinely imaged mice repeatedly for more than 5\n",
      "weeks postoperatively with up to ten separate imaging sessions and observed neither motor-function\n",
      "deficit nor neuropathology in the spinal cord as a result of chamber implantation. Using this\n",
      "chamber we quantified microglia and afferent axon dynamics after a laser-induced spinal cord lesion\n",
      "and observed massive microglia infiltration within 1 d along with a heterogeneous dieback of axon\n",
      "stumps. By enabling chronic imaging studies over timescales ranging from minutes to months, our\n",
      "method offers an ideal platform for understanding cellular dynamics in response to injury and\n",
      "therapeutic interventions.\n",
      " {\n",
      "\"main_contribution\": \"chronically implanted spinal chamber and surgical procedure suitable for time-lapse in vivo multiphoton microscopy of mouse spinal cord\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a chronically implanted spinal chamber and surgical procedure suitable for time-lapse in vivo multiphoton microscopy of mouse spinal cord. This is described in sentences (B) and (D) of the abstract. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "819\n",
      "22266543\n",
      "An artery-specific fluorescent dye for studying neurovascular coupling.\n",
      "We demonstrate that Alexa Fluor 633 hydrazide (Alexa Fluor 633) selectively labels neocortical\n",
      "arteries and arterioles by binding to elastin fibers. We measured sensory stimulus-evoked arteriole\n",
      "dilation dynamics in mouse, rat and cat visual cortex using Alexa Fluor 633 together with neuronal\n",
      "activity using calcium indicators or blood flow using fluorescein dextran. Arteriole dilation\n",
      "decreased fluorescence recorded from immediately underlying neurons, representing a potential\n",
      "artifact during neuronal functional imaging experiments.\n",
      " {\n",
      "\"main_contribution\": \"A fluorescent dye for studying neurovascular coupling\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a fluorescent dye that selectively labels neocortical arteries and arterioles by binding to elastin fibers. This dye can be used to study neurovascular coupling, which is the relationship between neural activity and blood flow in the brain. The paper demonstrates the use of this dye in mouse, rat, and cat visual cortex to measure sensory stimulus-evoked arteriole dilation dynamics, and how it can be used to study the potential artifacts that occur during neuronal functional imaging experiments. Therefore, the main contribution is the development of a new imaging technique, and the answer to is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "820\n",
      "22286384\n",
      "In vivo protein crystallization opens new routes in structural biology.\n",
      "Protein crystallization in cells has been observed several times in nature. However, owing to their\n",
      "small size these crystals have not yet been used for X-ray crystallographic analysis. We prepared\n",
      "nano-sized in vivo-grown crystals of Trypanosoma brucei enzymes and applied the emerging method of\n",
      "free-electron laser-based serial femtosecond crystallography to record interpretable diffraction\n",
      "data. This combined approach will open new opportunities in structural systems biology.\n",
      " {\n",
      "\"main_contribution\": \"Trypanosoma brucei enzymes nano-sized in vivo-grown crystals\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the use of free-electron laser-based serial femtosecond crystallography to record interpretable diffraction data from nano-sized in vivo-grown crystals of Trypanosoma brucei enzymes. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "821\n",
      "22286385\n",
      "Computational modeling of cellular signaling processes embedded into dynamic spatial contexts.\n",
      "Cellular signaling processes depend on spatiotemporal distributions of molecular components.\n",
      "Multicolor, high-resolution microscopy permits detailed assessment of such distributions, providing\n",
      "input for fine-grained computational models that explore mechanisms governing dynamic assembly of\n",
      "multimolecular complexes and their role in shaping cellular behavior. However, it is challenging to\n",
      "incorporate into such models both complex molecular reaction cascades and the spatial localization\n",
      "of signaling components in dynamic cellular morphologies. Here we introduce an approach to address\n",
      "these challenges by automatically generating computational representations of complex reaction\n",
      "networks based on simple bimolecular interaction rules embedded into detailed, adaptive models of\n",
      "cellular morphology. Using examples of receptor-mediated cellular adhesion and signal-induced\n",
      "localized mitogen-activated protein kinase (MAPK) activation in yeast, we illustrate the capacity of\n",
      "this simulation technique to provide insights into cell biological processes. The modeling\n",
      "algorithms, implemented in a new version of the Simmune toolset, are accessible through intuitive\n",
      "graphical interfaces and programming libraries.\n",
      " {\n",
      "\"main_contribution\": \"computational representations of complex reaction networks based on simple bimolecular interaction rules embedded into detailed, adaptive models of cellular morphology\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a computational modeling approach for cellular signaling processes that incorporates dynamic spatial contexts. The approach generates computational representations of complex reaction networks based on simple bimolecular interaction rules embedded into detailed, adaptive models of cellular morphology. The main contribution of the paper is the development of this computational modeling approach, which is not a new imaging technique or method. Therefore, the value of \"is_imaging_method\" is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "822\n",
      "22290187\n",
      "Immunolabeling artifacts and the need for live-cell imaging.\n",
      "Fluorescent fusion proteins have revolutionized examination of proteins in living cells. Still,\n",
      "studies using these proteins are met with criticism because proteins are modified and ectopically\n",
      "expressed, in contrast to immunofluorescence studies. However, introducing immunoreagents inside\n",
      "cells can cause protein extraction or relocalization, not reflecting the in vivo situation. Here we\n",
      "discuss pitfalls of immunofluorescence labeling that often receive little attention and argue that\n",
      "immunostaining experiments in dead, permeabilized cells should be complemented with live-cell\n",
      "imaging when scrutinizing protein localization.\n",
      " {\n",
      "\"main_contribution\": \"live-cell imaging to scrutinize protein localization\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title of the article highlights the need for live-cell imaging, while the abstract discusses the pitfalls of immunofluorescence labeling and argues for the use of live-cell imaging to complement immunostaining experiments in dead, permeabilized cells. Therefore, the main contribution of the paper is the development of a new imaging method, specifically live-cell imaging, to study protein localization. As this is a new technique, it falls under the category of \"is_imaging_method\" as true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "823\n",
      "22306808\n",
      "Polyubiquitin-sensor proteins reveal localization and linkage-type dependence of cellular ubiquitin\n",
      "signaling.\n",
      "Polyubiquitin chain topology is thought to direct modified substrates to specific fates, but this\n",
      "function-topology relationship is poorly understood, as are the dynamics and subcellular locations\n",
      "of specific polyubiquitin signals. Experimental access to these questions has been limited because\n",
      "linkage-specific inhibitors and in vivo sensors have been unavailable. Here we present a general\n",
      "strategy to track linkage-specific polyubiquitin signals in yeast and mammalian cells, and to probe\n",
      "their functions. We designed several high-affinity Lys63 polyubiquitin-binding proteins and\n",
      "demonstrate their specificity in vitro and in cells. We apply these tools as competitive inhibitors\n",
      "to dissect the polyubiquitin-linkage dependence of NF-κB activation in several cell types, inferring\n",
      "the essential role of Lys63 polyubiquitin for signaling via the IL-1β and TNF-related weak inducer\n",
      "of apoptosis (TWEAK) but not TNF-α receptors. We anticipate live-cell imaging, proteomic and\n",
      "biochemical applications for these tools and extension of the design strategy to other polymeric\n",
      "ubiquitin-like protein modifications.\n",
      " {\n",
      "\"main_contribution\": \"Polyubiquitin-sensor proteins reveal localization and linkage-type dependence of cellular ubiquitin signaling\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a general strategy to track linkage-specific polyubiquitin signals in yeast and mammalian cells, and to probe their functions. This is described in the abstract as \"a general strategy to track linkage-specific polyubiquitin signals in yeast and mammalian cells, and to probe their functions.\" Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "824\n",
      "22306809\n",
      "Interspecies activity correlations reveal functional correspondence between monkey and human brain\n",
      "areas.\n",
      "Evolution-driven functional changes in the primate brain are typically assessed by aligning monkey\n",
      "and human activation maps using cortical surface expansion models. These models use putative\n",
      "homologous areas as registration landmarks, assuming they are functionally correspondent. For cases\n",
      "in which functional changes have occurred in an area, this assumption prohibits to reveal whether\n",
      "other areas may have assumed lost functions. Here we describe a method to examine functional\n",
      "correspondences across species. Without making spatial assumptions, we assessed similarities in\n",
      "sensory-driven functional magnetic resonance imaging responses between monkey (Macaca mulatta) and\n",
      "human brain areas by temporal correlation. Using natural vision data, we revealed regions for which\n",
      "functional processing has shifted to topologically divergent locations during evolution. We conclude\n",
      "that substantial evolution-driven functional reorganizations have occurred, not always consistent\n",
      "with cortical expansion processes. This framework for evaluating changes in functional architecture\n",
      "is crucial to building more accurate evolutionary models.\n",
      " {\n",
      "\"main_contribution\": \"reveal regions for which functional processing has shifted to topologically divergent locations during evolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method to examine functional correspondences across species by temporal correlation, which is revealed in sentences (C) and (D). The method is used to assess similarities in sensory-driven functional magnetic resonance imaging responses between monkey and human brain areas. Therefore, the main contribution is recorded as \"reveal regions for which functional processing has shifted to topologically divergent locations during evolution\". Additionally, since the contribution is a new imaging method, the variable is_imaging_method is recorded as true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "825\n",
      "22343343\n",
      "A bioinformatics method identifies prominent off-targeted transcripts in RNAi screens.\n",
      "Because off-target effects hamper interpretation and validation of RNAi screen data, we developed a\n",
      "bioinformatics method, genome-wide enrichment of seed sequence matches (GESS), to identify candidate\n",
      "off-targeted transcripts in primary screening data. GESS analysis revealed a prominent off-targeted\n",
      "transcript in several screens, including MAD2 (MAD2L1) in a screen for genes required for the\n",
      "spindle assembly checkpoint. GESS analysis results can enhance the validation rate in RNAi screens.\n",
      " {\n",
      "\"main_contribution\": \"A bioinformatics method identifies prominent off-targeted transcripts in RNAi screens\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new bioinformatics method called GESS, which is used to identify candidate off-targeted transcripts in primary screening data from RNAi screens. The main contribution of the paper is the development of this method, which can enhance the validation rate in RNAi screens. Therefore, the main contribution is not an imaging technique or method, and is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "826\n",
      "22373911\n",
      "OMERO: flexible, model-driven data management for experimental biology.\n",
      "Data-intensive research depends on tools that manage multidimensional, heterogeneous datasets. We\n",
      "built OME Remote Objects (OMERO), a software platform that enables access to and use of a wide range\n",
      "of biological data. OMERO uses a server-based middleware application to provide a unified interface\n",
      "for images, matrices and tables. OMERO's design and flexibility have enabled its use for light-\n",
      "microscopy, high-content-screening, electron-microscopy and even non-image-genotype data. OMERO is\n",
      "open-source software, available at http://openmicroscopy.org/.\n",
      " {\n",
      "\"main_contribution\": \"OMERO: flexible, model-driven data management for experimental biology\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a software platform called OMERO that enables flexible and model-driven data management for experimental biology. The platform provides a unified interface for various types of biological data, including images, matrices, and tables. Since the paper describes the development of a new imaging technique or method, specifically a software platform for managing and analyzing imaging data, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "827\n",
      "22406855\n",
      "Controlling gene expression with the Q repressible binary expression system in Caenorhabditis\n",
      "elegans.\n",
      "We established a transcription-based binary gene expression system in Caenorhabditis elegans using\n",
      "the recently developed Q system. This system, derived from genes in Neurospora crassa, uses the\n",
      "transcriptional activator QF to induce the expression of target genes. Activation can be efficiently\n",
      "suppressed by the transcriptional repressor QS, and suppression can be relieved by the nontoxic\n",
      "small molecule quinic acid. We used QF, QS and quinic acid to achieve temporal and spatial control\n",
      "of transgene expression in various tissues in C. elegans. We also developed a split Q system, in\n",
      "which we separated QF into two parts encoding its DNA-binding and transcription-activation domains.\n",
      "Each domain showed negligible transcriptional activity when expressed alone, but expression of both\n",
      "reconstituted QF activity, providing additional combinatorial power to control gene expression.\n",
      " {\n",
      "\"main_contribution\": \"The development of a transcription-based binary gene expression system in Caenorhabditis elegans using the Q system, which allows for temporal and spatial control of transgene expression in various tissues.\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new gene expression system developed in Caenorhabditis elegans using the Q system, which allows for temporal and spatial control of transgene expression in various tissues. The main contribution of the paper is the development of this system, which is described in sentences (B) and (C). Sentence (A) provides background information about the work, while sentence (D) describes the results of the work. Sentence (E) concludes the abstract by stating that the split Q system provides additional combinatorial power to control gene expression.\n",
      "\n",
      "Since the main contribution of the paper is not the development of a new imaging method or technique, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "828\n",
      "22406856\n",
      "Synchronization of secretory protein traffic in populations of cells.\n",
      "To dissect secretory traffic, we developed the retention using selective hooks (RUSH) system. RUSH\n",
      "is a two-state assay based on the reversible interaction of a hook protein fused to core\n",
      "streptavidin and stably anchored in the donor compartment with a reporter protein of interest fused\n",
      "to streptavidin-binding peptide (SBP). Biotin addition causes a synchronous release of the reporter\n",
      "from the hook. Using the RUSH system, we analyzed different transport characteristics of various\n",
      "Golgi and plasma membrane reporters at physiological temperature in living cells. Using dual-color\n",
      "simultaneous live-cell imaging of two cargos, we observed intra- and post-Golgi segregation of cargo\n",
      "traffic, consistent with observation in other systems. We show preliminarily that the RUSH system is\n",
      "usable for automated screening. The system should help increase the understanding of the mechanisms\n",
      "of trafficking and enable screens for molecules that perturb pathological protein transport.\n",
      " {\n",
      "\"main_contribution\": \"development of a two-state assay called RUSH to study secretory traffic in populations of cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called RUSH, which allows for the study of secretory traffic in populations of cells. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "829\n",
      "22466793\n",
      "Integrated genetic and computation methods for in planta cytometry.\n",
      "We present the coupled use of specifically localized fluorescent gene markers and image processing\n",
      "for automated quantitative analysis of cell growth and genetic activity across living plant tissues.\n",
      "We used fluorescent protein markers to identify cells, create seeds and boundaries for the automatic\n",
      "segmentation of cell geometries and ratiometrically measure gene expression cell by cell in\n",
      "Arabidopsis thaliana.\n",
      " {\n",
      "\"main_contribution\": \"Automated quantitative analysis of cell growth and genetic activity across living plant tissues\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that uses specifically localized fluorescent gene markers and image processing for automated quantitative analysis of cell growth and genetic activity across living plant tissues. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "830\n",
      "22466794\n",
      "Direct visualization of specifically modified extracellular glycans in living animals.\n",
      "Modification patterns of heparan sulfate coordinate protein function in metazoans, yet in vivo\n",
      "imaging of such non-genetically encoded structures has been impossible. Here we report a transgenic\n",
      "method in Caenorhabditis elegans that allows direct live imaging of specific heparan sulfate\n",
      "modification patterns. This experimental approach reveals a dynamic and cell-specific heparan\n",
      "sulfate landscape and could in principle be adapted to visualize and analyze any extracellular\n",
      "molecule in vivo.\n",
      " {\n",
      "\"main_contribution\": \"direct live imaging of specific heparan sulfate modification patterns\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows direct live imaging of specific heparan sulfate modification patterns in living animals. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "831\n",
      "22466795\n",
      "Quantitative fluorescence labeling of aldehyde-tagged proteins for single-molecule imaging.\n",
      "A major hurdle for molecular mechanistic studies of many proteins is the lack of a general method\n",
      "for fluorescence labeling with high efficiency, specificity and speed. By incorporating an aldehyde\n",
      "motif genetically into a protein and improving the labeling kinetics substantially under mild\n",
      "conditions, we achieved fast, site-specific labeling of a protein with ∼100% efficiency while\n",
      "maintaining the biological function. We show that an aldehyde-tagged protein can be specifically\n",
      "labeled in cell extracts without protein purification and then can be used in single-molecule pull-\n",
      "down analysis. We also show the unique power of our method in single-molecule studies on the\n",
      "transient interactions and switching between two quantitatively labeled DNA polymerases on their\n",
      "processivity factor.\n",
      " {\n",
      "\"main_contribution\": \"A general method for fluorescence labeling of proteins for single-molecule imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "Sentences identified as background information: None\n",
      "\n",
      "Sentences identified as goals: \"A major hurdle for molecular mechanistic studies of many proteins is the lack of a general method for fluorescence labeling with high efficiency, specificity, and speed.\"\n",
      "\n",
      "Sentences identified as methods: \"By incorporating an aldehyde motif genetically into a protein and improving the labeling kinetics substantially under mild conditions, we achieved fast, site-specific labeling of a protein with ∼100% efficiency while maintaining the biological function.\"\n",
      "\n",
      "Sentences identified as results: \"We show that an aldehyde-tagged protein can be specifically labeled in cell extracts without protein purification and then can be used in single-molecule pull-down analysis. We also show the unique power of our method in single-molecule studies on the transient interactions and switching between two quantitatively labeled DNA polymerases on their processivity factor.\"\n",
      "\n",
      "Main contribution: A general method for fluorescence labeling of proteins for single-molecule imaging.\n",
      "\n",
      "Is imaging method: True, because the paper develops a new method for fluorescence labeling of proteins specifically designed for single-molecule imaging.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "832\n",
      "22484850\n",
      "Segregation of molecules at cell division reveals native protein localization.\n",
      "We introduce a nonintrusive method exploiting single-cell variability after cell division to\n",
      "validate protein localization. We found that Clp proteases, widely reported to form biologically\n",
      "relevant foci, were uniformly distributed in Escherichia coli cells, and that many commonly used\n",
      "fluorescent proteins caused severe mislocalization when fused to homo-oligomers. Retagging five\n",
      "other reportedly foci-forming proteins with the most monomeric fluorescent protein tested suggests\n",
      "that the foci were caused by the fluorescent tags.\n",
      " {\n",
      "\"main_contribution\": \"Segregation of molecules at cell division reveals native protein localization\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that exploits single-cell variability after cell division to validate protein localization. This method allows for the segregation of molecules at cell division, which in turn reveals native protein localization. Therefore, the main contribution is the development of an imaging technique, and is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "833\n",
      "22522656\n",
      "An image analysis toolbox for high-throughput C. elegans assays.\n",
      "We present a toolbox for high-throughput screening of image-based Caenorhabditis elegans phenotypes.\n",
      "The image analysis algorithms measure morphological phenotypes in individual worms and are effective\n",
      "for a variety of assays and imaging systems. This WormToolbox is available through the open-source\n",
      "CellProfiler project and enables objective scoring of whole-worm high-throughput image-based assays\n",
      "of C. elegans for the study of diverse biological pathways that are relevant to human disease.\n",
      " {\n",
      "\"main_contribution\": \"A toolbox for high-throughput screening of image-based Caenorhabditis elegans phenotypes.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an image analysis toolbox for high-throughput screening of image-based Caenorhabditis elegans phenotypes. This toolbox measures morphological phenotypes in individual worms and is effective for a variety of assays and imaging systems. Since the toolbox is designed to analyze images of C. elegans, it is considered an imaging method, and therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "834\n",
      "22522657\n",
      "Faster STORM using compressed sensing.\n",
      "In super-resolution microscopy methods based on single-molecule switching, the rate of accumulating\n",
      "single-molecule activation events often limits the time resolution. Here we developed a sparse-\n",
      "signal recovery technique using compressed sensing to analyze images with highly overlapping\n",
      "fluorescent spots. This method allows an activated fluorophore density an order of magnitude higher\n",
      "than what conventional single-molecule fitting methods can handle. Using this method, we\n",
      "demonstrated imaging microtubule dynamics in living cells with a time resolution of 3 s.\n",
      " {\n",
      "\"main_contribution\": \"Faster STORM using compressed sensing\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called Faster STORM, which uses compressed sensing to analyze images with highly overlapping fluorescent spots. This technique allows for an order of magnitude increase in activated fluorophore density compared to conventional single-molecule fitting methods and enables imaging microtubule dynamics in living cells with a time resolution of 3 seconds. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "835\n",
      "22543348\n",
      "A simple, versatile method for GFP-based super-resolution microscopy via nanobodies.\n",
      "We developed a method to use any GFP-tagged construct in single-molecule super-resolution\n",
      "microscopy. By targeting GFP with small, high-affinity antibodies coupled to organic dyes, we\n",
      "achieved nanometer spatial resolution and minimal linkage error when analyzing microtubules, living\n",
      "neurons and yeast cells. We show that in combination with libraries encoding GFP-tagged proteins,\n",
      "virtually any known protein can immediately be used in super-resolution microscopy and that\n",
      "simplified labeling schemes allow high-throughput super-resolution imaging.\n",
      " {\n",
      "\"main_contribution\": \"A simple, versatile method for GFP-based super-resolution microscopy via nanobodies\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method using nanobodies to achieve super-resolution microscopy with high spatial resolution and minimal linkage error. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "836\n",
      "22561987\n",
      "FMT-XCT: in vivo animal studies with hybrid fluorescence molecular tomography-X-ray computed\n",
      "tomography.\n",
      "The development of hybrid optical tomography methods to improve imaging performance has been\n",
      "suggested over a decade ago and has been experimentally demonstrated in animals and humans. Here we\n",
      "examined in vivo performance of a camera-based hybrid fluorescence molecular tomography (FMT) system\n",
      "for 360° imaging combined with X-ray computed tomography (XCT). Offering an accurately co-\n",
      "registered, information-rich hybrid data set, FMT-XCT has new imaging possibilities compared to\n",
      "stand-alone FMT and XCT. We applied FMT-XCT to a subcutaneous 4T1 tumor mouse model, an Aga2\n",
      "osteogenesis imperfecta model and a Kras lung cancer mouse model, using XCT information during FMT\n",
      "inversion. We validated in vivo imaging results against post-mortem planar fluorescence images of\n",
      "cryoslices and histology data. Besides offering concurrent anatomical and functional information,\n",
      "FMT-XCT resulted in the most accurate FMT performance to date. These findings indicate that addition\n",
      "of FMT optics into the XCT gantry may be a potent upgrade for small-animal XCT systems.\n",
      " {\n",
      "\"main_contribution\": \"FMT-XCT: in vivo animal studies with hybrid fluorescence molecular tomography-X-ray computed tomography\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development and in vivo testing of a new imaging method called FMT-XCT, which combines fluorescence molecular tomography (FMT) with X-ray computed tomography (XCT). Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "837\n",
      "22561988\n",
      "Automated whole-cell patch-clamp electrophysiology of neurons in vivo.\n",
      "Whole-cell patch-clamp electrophysiology of neurons is a gold-standard technique for high-fidelity\n",
      "analysis of the biophysical mechanisms of neural computation and pathology, but it requires great\n",
      "skill to perform. We have developed a robot that automatically performs patch clamping in vivo,\n",
      "algorithmically detecting cells by analyzing the temporal sequence of electrode impedance changes.\n",
      "We demonstrate good yield, throughput and quality of automated intracellular recording in mouse\n",
      "cortex and hippocampus.\n",
      " {\n",
      "\"main_contribution\": \"Automated whole-cell patch-clamp electrophysiology of neurons in vivo\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new technique for automating whole-cell patch-clamp electrophysiology of neurons in vivo. The main contribution of the paper is the development of a robot that can perform this technique, which allows for high-fidelity analysis of neural computation and pathology. Since this technique does not involve imaging, the answer to is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "838\n",
      "22561989\n",
      "Simultaneous BOLD fMRI and fiber-optic calcium recording in rat neocortex.\n",
      "Functional magnetic resonance imaging (fMRI) based on blood oxygen level-dependent (BOLD) contrast\n",
      "is widely used for probing brain activity, but its relationship to underlying neural activity\n",
      "remains elusive. Here, we combined fMRI with fiber-optic recordings of fluorescent calcium indicator\n",
      "signals to investigate this relationship in rat somatosensory cortex. Electrical forepaw stimulation\n",
      "(1-10 Hz) evoked fast calcium signals of neuronal origin that showed frequency-dependent adaptation.\n",
      "Additionally, slower calcium signals occurred in astrocyte networks, as verified by astrocyte-\n",
      "specific staining and two-photon microscopy. Without apparent glia activation, we could predict BOLD\n",
      "responses well from simultaneously recorded fiber-optic signals, assuming an impulse response\n",
      "function and taking into account neuronal adaptation. In cases with glia activation, we uncovered\n",
      "additional prolonged BOLD signal components. Our findings highlight the complexity of fMRI BOLD\n",
      "signals, involving both neuronal and glial activity. Combined fMRI and fiber-optic recordings should\n",
      "help to clarify cellular mechanisms underlying BOLD signals.\n",
      " {\n",
      "\"main_contribution\": \"Combined fMRI with fiber-optic recordings of fluorescent calcium indicator signals to investigate the relationship between BOLD contrast and underlying neural activity in rat somatosensory cortex.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the combination of fMRI with fiber-optic recordings of fluorescent calcium indicator signals to investigate the relationship between BOLD contrast and underlying neural activity in rat somatosensory cortex. This is a development of a new imaging method, hence the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "839\n",
      "22581370\n",
      "Rational design of true monomeric and bright photoactivatable fluorescent proteins.\n",
      "Monomeric (m)Eos2 is an engineered photoactivatable fluorescent protein widely used for super-\n",
      "resolution microscopy. We show that mEos2 forms oligomers at high concentrations and forms\n",
      "aggregates when labeling membrane proteins, limiting its application as a fusion partner. We solved\n",
      "the crystal structure of tetrameric mEos2 and rationally designed improved versions, mEos3.1 and\n",
      "mEos3.2, that are truly monomeric, are brighter, mature faster and exhibit higher photon budget and\n",
      "label density.\n",
      " {\n",
      "\"main_contribution\": \"rationally designed true monomeric and bright photoactivatable fluorescent proteins\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of new, improved versions of photoactivatable fluorescent proteins that are truly monomeric, brighter, mature faster, and exhibit a higher photon budget and label density. These proteins have the potential to be used in super-resolution microscopy and other imaging techniques. Therefore, the main contribution is the development of an imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "840\n",
      "22581371\n",
      "M-Track: detecting short-lived protein-protein interactions in vivo.\n",
      "We developed a protein-proximity assay in yeast based on fusing a histone lysine methyltransferase\n",
      "onto a bait and its substrate onto a prey. Upon binding, the prey is stably methylated and detected\n",
      "by methylation-specific antibodies. We applied this approach to detect varying interaction\n",
      "affinities among proteins in a mitogen-activated protein kinase pathway and to detect short-lived\n",
      "interactions between protein phosphatase 2A and its substrates that have so far escaped direct\n",
      "detection.\n",
      " {\n",
      "\"main_contribution\": \"detecting short-lived protein-protein interactions in vivo\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a protein-proximity assay in yeast that utilizes histone lysine methyltransferase and methylation-specific antibodies to detect short-lived protein-protein interactions. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "841\n",
      "22581372\n",
      "Resolution doubling in live, multicellular organisms via multifocal structured illumination\n",
      "microscopy.\n",
      "We demonstrate three-dimensional (3D) super-resolution in live multicellular organisms using\n",
      "structured illumination microscopy (SIM). Sparse multifocal illumination patterns generated by a\n",
      "digital micromirror device (DMD) allowed us to physically reject out-of-focus light, enabling 3D\n",
      "subdiffractive imaging in samples eightfold thicker than had been previously imaged with SIM. We\n",
      "imaged samples at one 2D image per second, at resolutions as low as 145 nm laterally and 400 nm\n",
      "axially. In addition to dual-labeled, whole fixed cells, we imaged GFP-labeled microtubules in live\n",
      "transgenic zebrafish embryos at depths >45 μm. We captured dynamic changes in the zebrafish lateral\n",
      "line primordium and observed interactions between myosin IIA and F-actin in cells encapsulated in\n",
      "collagen gels, obtaining two-color 4D super-resolution data sets spanning tens of time points and\n",
      "minutes without apparent phototoxicity. Our method uses commercially available parts and open-source\n",
      "software and is simpler than existing SIM implementations, allowing easy integration with wide-field\n",
      "microscopes.\n",
      " {\n",
      "\"main_contribution\": \"Three-dimensional (3D) super-resolution in live multicellular organisms using structured illumination microscopy (SIM).\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a method of three-dimensional (3D) super-resolution in live multicellular organisms using structured illumination microscopy (SIM). Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "842\n",
      "22609626\n",
      "Facile backbone structure determination of human membrane proteins by NMR spectroscopy.\n",
      "Although nearly half of today's major pharmaceutical drugs target human integral membrane proteins\n",
      "(hIMPs), only 30 hIMP structures are currently available in the Protein Data Bank, largely owing to\n",
      "inefficiencies in protein production. Here we describe a strategy for the rapid structure\n",
      "determination of hIMPs, using solution NMR spectroscopy with systematically labeled proteins\n",
      "produced via cell-free expression. We report new backbone structures of six hIMPs, solved in only 18\n",
      "months from 15 initial targets. Application of our protocols to an additional 135 hIMPs with\n",
      "molecular weight <30 kDa yielded 38 hIMPs suitable for structural characterization by solution NMR\n",
      "spectroscopy without additional optimization.\n",
      " {\n",
      "\"main_contribution\": \"rapid structure determination of hIMPs using solution NMR spectroscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a strategy for the rapid structure determination of human integral membrane proteins (hIMPs) using solution NMR spectroscopy. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "843\n",
      "22635062\n",
      "Unsupervised modeling of cell morphology dynamics for time-lapse microscopy.\n",
      "Analysis of cellular phenotypes in large imaging data sets conventionally involves supervised\n",
      "statistical methods, which require user-annotated training data. This paper introduces an\n",
      "unsupervised learning method, based on temporally constrained combinatorial clustering, for\n",
      "automatic prediction of cell morphology classes in time-resolved images. We applied the unsupervised\n",
      "method to diverse fluorescent markers and screening data and validated accurate classification of\n",
      "human cell phenotypes, demonstrating fully objective data labeling in image-based systems biology.\n",
      " {\n",
      "\"main_contribution\": \"Unsupervised modeling of cell morphology dynamics for time-lapse microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an unsupervised learning method for predicting cell morphology classes in time-resolved images, which is a new imaging technique. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "844\n",
      "22660739\n",
      "Multiview light-sheet microscope for rapid in toto imaging.\n",
      "We present a multiview selective-plane illumination microscope (MuVi-SPIM), comprising two detection\n",
      "and illumination objective lenses, that allows rapid in toto fluorescence imaging of biological\n",
      "specimens with subcellular resolution. The fixed geometrical arrangement of the imaging branches\n",
      "enables multiview data fusion in real time. The high speed of MuVi-SPIM allows faithful tracking of\n",
      "nuclei and cell shape changes, which we demonstrate through in toto imaging of the embryonic\n",
      "development of Drosophila melanogaster.\n",
      " {\n",
      "\"main_contribution\": \"rapid in toto imaging of biological specimens with subcellular resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a multiview light-sheet microscope that allows for rapid in toto imaging of biological specimens with subcellular resolution. This is described in the abstract as the ability to faithfully track nuclei and cell shape changes in real time. Therefore, the main contribution is the development of an imaging technique or method, which is recorded as true in the variable is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "845\n",
      "22660740\n",
      "Single-cell systems biology by super-resolution imaging and combinatorial labeling.\n",
      "Fluorescence microscopy is a powerful quantitative tool for exploring regulatory networks in single\n",
      "cells. However, the number of molecular species that can be measured simultaneously is limited by\n",
      "the spectral overlap between fluorophores. Here we demonstrate a simple but general strategy to\n",
      "drastically increase the capacity for multiplex detection of molecules in single cells by using\n",
      "optical super-resolution microscopy (SRM) and combinatorial labeling. As a proof of principle, we\n",
      "labeled mRNAs with unique combinations of fluorophores using fluorescence in situ hybridization\n",
      "(FISH), and resolved the sequences and combinations of fluorophores with SRM. We measured mRNA\n",
      "levels of 32 genes simultaneously in single Saccharomyces cerevisiae cells. These experiments\n",
      "demonstrate that combinatorial labeling and super-resolution imaging of single cells is a natural\n",
      "approach to bring systems biology into single cells.\n",
      " {\n",
      "\"main_contribution\": \"Combinatorial labeling and super-resolution imaging of single cells for systems biology\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "846\n",
      "22660741\n",
      "Quantitative high-speed imaging of entire developing embryos with simultaneous multiview light-sheet\n",
      "microscopy.\n",
      "Live imaging of large biological specimens is fundamentally limited by the short optical penetration\n",
      "depth of light microscopes. To maximize physical coverage, we developed the SiMView technology\n",
      "framework for high-speed in vivo imaging, which records multiple views of the specimen\n",
      "simultaneously. SiMView consists of a light-sheet microscope with four synchronized optical arms,\n",
      "real-time electronics for long-term sCMOS-based image acquisition at 175 million voxels per second,\n",
      "and computational modules for high-throughput image registration, segmentation, tracking and real-\n",
      "time management of the terabytes of multiview data recorded per specimen. We developed one-photon\n",
      "and multiphoton SiMView implementations and recorded cellular dynamics in entire Drosophila\n",
      "melanogaster embryos with 30-s temporal resolution throughout development. We furthermore performed\n",
      "high-resolution long-term imaging of the developing nervous system and followed neuroblast cell\n",
      "lineages in vivo. SiMView data sets provide quantitative morphological information even for fast\n",
      "global processes and enable accurate automated cell tracking in the entire early embryo.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous multiview light-sheet microscopy for high-speed imaging of entire developing embryos\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called SiMView, which uses simultaneous multiview light-sheet microscopy to image entire developing embryos at high speed. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "847\n",
      "22688414\n",
      "Elastic volume reconstruction from series of ultra-thin microscopy sections.\n",
      "Anatomy of large biological specimens is often reconstructed from serially sectioned volumes imaged\n",
      "by high-resolution microscopy. We developed a method to reassemble a continuous volume from such\n",
      "large section series that explicitly minimizes artificial deformation by applying a global elastic\n",
      "constraint. We demonstrate our method on a series of transmission electron microscopy sections\n",
      "covering the entire 558-cell Caenorhabditis elegans embryo and a segment of the Drosophila\n",
      "melanogaster larval ventral nerve cord.\n",
      " {\n",
      "\"main_contribution\": \"Elastic volume reconstruction from series of ultra-thin microscopy sections\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a method to reassemble a continuous volume from large section series that explicitly minimizes artificial deformation by applying a global elastic constraint. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "848\n",
      "22688415\n",
      "Rapid, accurate particle tracking by calculation of radial symmetry centers.\n",
      "I introduce an algorithm for subpixel localization of imaged objects based on an analytic, non-\n",
      "iterative calculation of the best-fit radial symmetry center. This approach yields tracking\n",
      "accuracies that are near theoretical limits, similarly to Gaussian fitting, but with orders-of-\n",
      "magnitude faster execution time, lower sensitivity to nearby particles and applicability to any\n",
      "radially symmetric intensity distribution. I demonstrate the method with several types of data,\n",
      "including super-resolution microscopy images.\n",
      " {\n",
      "\"main_contribution\": \"rapid, accurate particle tracking algorithm\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a rapid and accurate particle tracking algorithm based on an analytic, non-iterative calculation of the best-fit radial symmetry center. This algorithm is applicable to any radially symmetric intensity distribution and has been demonstrated with several types of data, including super-resolution microscopy images. Therefore, the main contribution is the development of a new imaging method, and is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "849\n",
      "22706672\n",
      "ViBE-Z: a framework for 3D virtual colocalization analysis in zebrafish larval brains.\n",
      "Precise three-dimensional (3D) mapping of a large number of gene expression patterns, neuronal types\n",
      "and connections to an anatomical reference helps us to understand the vertebrate brain and its\n",
      "development. We developed the Virtual Brain Explorer (ViBE-Z), a software that automatically maps\n",
      "gene expression data with cellular resolution to a 3D standard larval zebrafish (Danio rerio) brain.\n",
      "ViBE-Z enhances the data quality through fusion and attenuation correction of multiple confocal\n",
      "microscope stacks per specimen and uses a fluorescent stain of cell nuclei for image registration.\n",
      "It automatically detects 14 predefined anatomical landmarks for aligning new data with the reference\n",
      "brain. ViBE-Z performs colocalization analysis in expression databases for anatomical domains or\n",
      "subdomains defined by any specific pattern; here we demonstrate its utility for mapping neurons of\n",
      "the dopaminergic system. The ViBE-Z database, atlas and software are provided via a web interface.\n",
      " {\n",
      "\"main_contribution\": \"a software that automatically maps gene expression data with cellular resolution to a 3D standard larval zebrafish (Danio rerio) brain\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a software called Virtual Brain Explorer (ViBE-Z), which maps gene expression data with cellular resolution to a 3D standard larval zebrafish brain. This software enhances the data quality through fusion and attenuation correction of multiple confocal microscope stacks per specimen and uses a fluorescent stain of cell nuclei for image registration. Therefore, the main contribution is the development of a new imaging method, and thus, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "850\n",
      "22743769\n",
      "Why bioimage informatics matters.\n",
      "Driven by the importance of spatial and physical factors in cellular processes and the size and\n",
      "complexity of modern image data, computational analysis of biological imagery has become a vital\n",
      "emerging sub-discipline of bioinformatics and computer vision.\n",
      " {\n",
      "\"main_contribution\": \"computational analysis of biological imagery\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically computational analysis of biological imagery. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "851\n",
      "22743770\n",
      "Current challenges in open-source bioimage informatics.\n",
      "We discuss the advantages and challenges of the open-source strategy in biological image analysis\n",
      "and argue that its full impact will not be realized without better support and recognition of\n",
      "software engineers' contributions to the biological sciences and more support of this development\n",
      "model from funders and institutions.\n",
      " {\n",
      "\"main_contribution\": \"Better support and recognition of software engineers' contributions to the biological sciences\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "852\n",
      "22743772\n",
      "Fiji: an open-source platform for biological-image analysis.\n",
      "Fiji is a distribution of the popular open-source software ImageJ focused on biological-image\n",
      "analysis. Fiji uses modern software engineering practices to combine powerful software libraries\n",
      "with a broad range of scripting languages to enable rapid prototyping of image-processing\n",
      "algorithms. Fiji facilitates the transformation of new algorithms into ImageJ plugins that can be\n",
      "shared with end users through an integrated update system. We propose Fiji as a platform for\n",
      "productive collaboration between computer science and biology research communities.\n",
      " {\n",
      "\"main_contribution\": \"a distribution of open-source software ImageJ focused on biological-image analysis\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a platform for biological image analysis, specifically the Fiji software. This software enables rapid prototyping of image processing algorithms and facilitates the transformation of new algorithms into ImageJ plugins that can be shared with end users through an integrated update system. Therefore, the main contribution is the development of a new imaging method, and is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "853\n",
      "22743773\n",
      "BioImageXD: an open, general-purpose and high-throughput image-processing platform.\n",
      "BioImageXD puts open-source computer science tools for three-dimensional visualization and analysis\n",
      "into the hands of all researchers, through a user-friendly graphical interface tuned to the needs of\n",
      "biologists. BioImageXD has no restrictive licenses or undisclosed algorithms and enables publication\n",
      "of precise, reproducible and modifiable workflows. It allows simple construction of processing\n",
      "pipelines and should enable biologists to perform challenging analyses of complex processes. We\n",
      "demonstrate its performance in a study of integrin clustering in response to selected inhibitors.\n",
      " {\n",
      "\"main_contribution\": \"BioImageXD: an open, general-purpose and high-throughput image-processing platform\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of BioImageXD, an open-source image-processing platform for biologists. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "854\n",
      "22743774\n",
      "Icy: an open bioimage informatics platform for extended reproducible research.\n",
      "Current research in biology uses evermore complex computational and imaging tools. Here we describe\n",
      "Icy, a collaborative bioimage informatics platform that combines a community website for\n",
      "contributing and sharing tools and material, and software with a high-end visual programming\n",
      "framework for seamless development of sophisticated imaging workflows. Icy extends the reproducible\n",
      "research principles, by encouraging and facilitating the reusability, modularity, standardization\n",
      "and management of algorithms and protocols. Icy is free, open-source and available at\n",
      "http://icy.bioimageanalysis.org/.\n",
      " {\n",
      "\"main_contribution\": \"Icy, a collaborative bioimage informatics platform\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of Icy, a collaborative bioimage informatics platform that enables seamless development of sophisticated imaging workflows. Since Icy is an imaging informatics platform, it falls under the category of developing a new imaging method, and therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "855\n",
      "22743775\n",
      "Biological imaging software tools.\n",
      "Few technologies are more widespread in modern biological laboratories than imaging. Recent advances\n",
      "in optical technologies and instrumentation are providing hitherto unimagined capabilities. Almost\n",
      "all these advances have required the development of software to enable the acquisition, management,\n",
      "analysis and visualization of the imaging data. We review each computational step that biologists\n",
      "encounter when dealing with digital images, the inherent challenges and the overall status of\n",
      "available software for bioimage informatics, focusing on open-source options.\n",
      " {\n",
      "\"main_contribution\": \"Review of software tools for biological imaging\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "856\n",
      "22751201\n",
      "Scanning angle interference microscopy reveals cell dynamics at the nanoscale.\n",
      "Emerging questions in cell biology necessitate nanoscale imaging in live cells. Here we present\n",
      "scanning angle interference microscopy, which is capable of localizing fluorescent objects with\n",
      "nanoscale precision along the optical axis in motile cellular structures. We use this approach to\n",
      "resolve nanotopographical features of the cell membrane and cytoskeleton as well as the temporal\n",
      "evolution, three-dimensional architecture and nanoscale dynamics of focal adhesion complexes.\n",
      " {\n",
      "\"main_contribution\": \"Scanning angle interference microscopy reveals cell dynamics at the nanoscale\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging technique called scanning angle interference microscopy, which allows for nanoscale imaging in live cells. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "857\n",
      "22751204\n",
      "Targeted gene knockout by direct delivery of zinc-finger nuclease proteins.\n",
      "Zinc-finger nucleases (ZFNs) are versatile reagents that have redefined genome engineering.\n",
      "Realizing the full potential of this technology requires the development of safe and effective\n",
      "methods for delivering ZFNs into cells. We demonstrate the intrinsic cell-penetrating capabilities\n",
      "of the standard ZFN architecture and show that direct delivery of ZFNs as proteins leads to\n",
      "efficient endogenous gene disruption in various mammalian cell types with minimal off-target\n",
      "effects.\n",
      " {\n",
      "\"main_contribution\": \"Efficient endogenous gene disruption in various mammalian cell types with minimal off-target effects\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "Sentences describing background information about the work:\n",
      "- Zinc-finger nucleases (ZFNs) are versatile reagents that have redefined genome engineering. \n",
      "- Realizing the full potential of this technology requires the development of safe and effective methods for delivering ZFNs into cells.\n",
      "\n",
      "Sentences describing the goals of the work being described:\n",
      "- Show that direct delivery of ZFNs as proteins leads to efficient endogenous gene disruption in various mammalian cell types with minimal off-target effects.\n",
      "\n",
      "Sentences describing the methods used to achieve the goals: \n",
      "- We demonstrate the intrinsic cell-penetrating capabilities of the standard ZFN architecture.\n",
      "- Direct delivery of ZFNs as proteins.\n",
      "\n",
      "Sentences describing the results of the work:\n",
      "- Efficient endogenous gene disruption in various mammalian cell types with minimal off-target effects.\n",
      "\n",
      "Main contribution: \n",
      "Efficient endogenous gene disruption in various mammalian cell types with minimal off-target effects.\n",
      "\n",
      "Is imaging method: false\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "858\n",
      "22772730\n",
      "Multicolor two-photon tissue imaging by wavelength mixing.\n",
      "We achieve simultaneous two-photon excitation of three chromophores with distinct absorption spectra\n",
      "using synchronized pulses from a femtosecond laser and an optical parametric oscillator. The two\n",
      "beams generate separate multiphoton processes, and their spatiotemporal overlap provides an\n",
      "additional two-photon excitation route, with submicrometer overlay of the color channels. We report\n",
      "volume and live multicolor imaging of 'Brainbow'-labeled tissues as well as simultaneous three-color\n",
      "fluorescence and third-harmonic imaging of fly embryos.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous two-photon excitation of three chromophores with distinct absorption spectra using synchronized pulses from a femtosecond laser and an optical parametric oscillator.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that uses synchronized pulses from a femtosecond laser and an optical parametric oscillator to achieve simultaneous two-photon excitation of three chromophores with distinct absorption spectra. This technique allows for multicolor imaging of labeled tissues and live cells, as well as simultaneous three-color fluorescence and third-harmonic imaging of fly embryos. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "859\n",
      "22796663\n",
      "A culture system to study oligodendrocyte myelination processes using engineered nanofibers.\n",
      "Current methods for studying central nervous system myelination necessitate permissive axonal\n",
      "substrates conducive to myelin wrapping by oligodendrocytes. We have developed a neuron-free culture\n",
      "system in which electron-spun nanofibers of varying sizes substitute for axons as a substrate for\n",
      "oligodendrocyte myelination, thereby allowing manipulation of the biophysical elements of axonal-\n",
      "oligodendroglial interactions. To investigate axonal regulation of myelination, this system\n",
      "effectively uncouples the role of molecular (inductive) cues from that of biophysical properties of\n",
      "the axon. We use this method to uncover the causation and sufficiency of fiber diameter in the\n",
      "initiation of concentric wrapping by rat oligodendrocytes. We also show that oligodendrocyte\n",
      "precursor cells display sensitivity to the biophysical properties of fiber diameter and initiate\n",
      "membrane ensheathment before differentiation. The use of nanofiber scaffolds will enable screening\n",
      "for potential therapeutic agents that promote oligodendrocyte differentiation and myelination and\n",
      "will also provide valuable insight into the processes involved in remyelination.\n",
      " {\n",
      "\"main_contribution\": \"A culture system to study oligodendrocyte myelination processes using engineered nanofibers\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a culture system using engineered nanofibers to study oligodendrocyte myelination processes. This system allows for the manipulation of biophysical elements of axonal-oligodendroglial interactions and enables the investigation of axonal regulation of myelination. The use of this method uncovers the causation and sufficiency of fiber diameter in the initiation of concentric wrapping by rat oligodendrocytes. Therefore, the main contribution is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "860\n",
      "22863882\n",
      "Direct observation of mammalian cell growth and size regulation.\n",
      "We introduce a microfluidic system for simultaneously measuring single-cell mass and cell cycle\n",
      "progression over multiple generations. We use this system to obtain over 1,000 h of growth data from\n",
      "mouse lymphoblast and pro-B-cell lymphoid cell lines. Cell lineage analysis revealed a decrease in\n",
      "the growth rate variability at the G1-S phase transition, which suggests the presence of a growth\n",
      "rate threshold for maintaining size homeostasis.\n",
      " {\n",
      "\"main_contribution\": \"Development of a microfluidic system for measuring single-cell mass and cell cycle progression\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the paper describe the development of a microfluidic system for measuring single-cell mass and cell cycle progression, which is the main contribution of the paper. The system was used to obtain over 1,000 hours of growth data from mouse lymphoblast and pro-B-cell lymphoid cell lines, and the analysis of this data revealed a decrease in the growth rate variability at the G1-S phase transition, suggesting the presence of a growth rate threshold for maintaining size homeostasis. However, the paper does not describe the development of a new imaging method or technique, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "861\n",
      "22886092\n",
      "Fractional proliferation: a method to deconvolve cell population dynamics from single-cell data.\n",
      "We present an integrated method that uses extended time-lapse automated imaging to quantify the\n",
      "dynamics of cell proliferation. Cell counts are fit with a quiescence-growth model that estimates\n",
      "rates of cell division, entry into quiescence and death. The model is constrained with rates\n",
      "extracted experimentally from the behavior of tracked single cells over time. We visualize the\n",
      "output of the analysis in fractional proliferation graphs, which deconvolve dynamic proliferative\n",
      "responses to perturbations into the relative contributions of dividing, quiescent (nondividing) and\n",
      "dead cells. The method reveals that the response of 'oncogene-addicted' human cancer cells to\n",
      "tyrosine kinase inhibitors is a composite of altered rates of division, death and entry into\n",
      "quiescence, a finding that challenges the notion that such cells simply die in response to oncogene-\n",
      "targeted therapy.\n",
      " {\n",
      "\"main_contribution\": \"A method to deconvolve cell population dynamics from single-cell data\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method for analyzing single-cell data to deconvolve cell population dynamics, which is an imaging technique. Therefore, \"main_contribution\" is set to the title of the paper and \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "862\n",
      "22902935\n",
      "Autonomous screening of C. elegans identifies genes implicated in synaptogenesis.\n",
      "Morphometric studies in multicellular organisms are generally performed manually because of the\n",
      "complexity of multidimensional features and lack of appropriate tools for handling these organisms.\n",
      "Here we present an integrated system that identifies and sorts Caenorhabditis elegans mutants with\n",
      "altered subcellular traits in real time without human intervention. We performed self-directed\n",
      "screens 100 times faster than manual screens and identified both genes and phenotypic classes\n",
      "involved in synapse formation.\n",
      " {\n",
      "\"main_contribution\": \"identifies genes implicated in synaptogenesis\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for screening Caenorhabditis elegans mutants with altered subcellular traits in real-time without human intervention. The main contribution of the paper is identifying genes implicated in synaptogenesis, which is not a development of a new imaging method. Therefore, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "863\n",
      "22922501\n",
      "Site-specific genome editing in Plasmodium falciparum using engineered zinc-finger nucleases.\n",
      "Malaria afflicts over 200 million people worldwide, and its most lethal etiologic agent, Plasmodium\n",
      "falciparum, is evolving to resist even the latest-generation therapeutics. Efficient tools for\n",
      "genome-directed investigations of P. falciparum-induced pathogenesis, including drug-resistance\n",
      "mechanisms, are clearly required. Here we report rapid and targeted genetic engineering of this\n",
      "parasite using zinc-finger nucleases (ZFNs) that produce a double-strand break in a user-defined\n",
      "locus and trigger homology-directed repair. Targeting an integrated egfp locus, we obtained gene-\n",
      "deletion parasites with unprecedented speed (2 weeks), both with and without direct selection. ZFNs\n",
      "engineered against the parasite gene pfcrt, responsible for escape under chloroquine treatment,\n",
      "rapidly produced parasites that carried either an allelic replacement or a panel of specified point\n",
      "mutations. This method will enable a diverse array of genome-editing approaches to interrogate this\n",
      "human pathogen.\n",
      " {\n",
      "\"main_contribution\": \"rapid and targeted genetic engineering of Plasmodium falciparum using zinc-finger nucleases\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for genome editing in Plasmodium falciparum, the parasite that causes malaria, using engineered zinc-finger nucleases. The method allows for rapid and targeted genetic engineering of the parasite, enabling the creation of gene-deletion parasites and parasites with specified point mutations. This contribution is not related to imaging or microscopy techniques, so \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "864\n",
      "22930824\n",
      "The quest for quantitative microscopy.\n",
      "With the aid of informatics, microscopy is in the midst of a crucial evolution into a more\n",
      "quantitative and powerful technique.\n",
      " {\n",
      "\"main_contribution\": \"a more quantitative and powerful technique\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "865\n",
      "22930834\n",
      "NIH Image to ImageJ: 25 years of image analysis.\n",
      "For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the\n",
      "analysis of scientific images. We discuss the origins, challenges and solutions of these two\n",
      "programs, and how their history can serve to advise and inform other software projects.\n",
      " {\n",
      "\"main_contribution\": \"pioneers as open tools for the analysis of scientific images\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "866\n",
      "22936170\n",
      "Imaging without lenses: achievements and remaining challenges of wide-field on-chip microscopy.\n",
      "We discuss unique features of lens-free computational imaging tools and report some of their\n",
      "emerging results for wide-field on-chip microscopy, such as the achievement of a numerical aperture\n",
      "(NA) of ∼0.8-0.9 across a field of view (FOV) of more than 20 mm(2) or an NA of ∼0.1 across a FOV of\n",
      "∼18 cm(2), which corresponds to an image with more than 1.5 gigapixels. We also discuss the current\n",
      "challenges that these computational on-chip microscopes face, shedding light on their future\n",
      "directions and applications.\n",
      " {\n",
      "\"main_contribution\": \"Development of lens-free computational imaging tools for wide-field on-chip microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of lens-free computational imaging tools for wide-field on-chip microscopy, which is a new imaging method. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "867\n",
      "22941363\n",
      "Ultrafast force-clamp spectroscopy of single molecules reveals load dependence of myosin working\n",
      "stroke.\n",
      "We describe a dual-trap force-clamp configuration that applies constant loads between a binding\n",
      "protein and an intermittently interacting biological polymer. The method has a measurement delay of\n",
      "only ∼10 μs, allows detection of interactions as brief as ∼100 μs and probes sub-nanometer\n",
      "conformational changes with a time resolution of tens of microseconds. We tested our method on\n",
      "molecular motors and DNA-binding proteins. We could apply constant loads to a single motor domain of\n",
      "myosin before its working stroke was initiated (0.2-1 ms), thus directly measuring its load\n",
      "dependence. We found that, depending on the applied load, myosin weakly interacted (<1 ms) with\n",
      "actin without production of movement, fully developed its working stroke or prematurely detached (<5\n",
      "ms), thus reducing the working stroke size with load. Our technique extends single-molecule force-\n",
      "clamp spectroscopy and opens new avenues for investigating the effects of forces on biological\n",
      "processes.\n",
      " {\n",
      "\"main_contribution\": \"Ultrafast force-clamp spectroscopy of single molecules reveals load dependence of myosin working stroke\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new technique called ultrafast force-clamp spectroscopy that allows for the measurement of conformational changes in single molecules under constant loads. The method is used to study the load dependence of myosin working stroke, which is a biological process involving the interaction between myosin and actin.\n",
      "\n",
      "The main contribution of the paper is the development of this new technique, which is described in sentences (A) background information about the work, (B) the goals of the work being described, (C) the methods used to achieve the goals, and (D) the results of the work.\n",
      "\n",
      "Therefore, the main contribution of the paper is \"Ultrafast force-clamp spectroscopy of single molecules reveals load dependence of myosin working stroke\".\n",
      "\n",
      "Additionally, since the paper describes the development of a new imaging technique (force-clamp spectroscopy), the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "868\n",
      "22961245\n",
      "Improving FRET dynamic range with bright green and red fluorescent proteins.\n",
      "A variety of genetically encoded reporters use changes in fluorescence (or Förster) resonance energy\n",
      "transfer (FRET) to report on biochemical processes in living cells. The standard genetically encoded\n",
      "FRET pair consists of CFPs and YFPs, but many CFP-YFP reporters suffer from low FRET dynamic range,\n",
      "phototoxicity from the CFP excitation light and complex photokinetic events such as reversible\n",
      "photobleaching and photoconversion. We engineered two fluorescent proteins, Clover and mRuby2, which\n",
      "are the brightest green and red fluorescent proteins to date and have the highest Förster radius of\n",
      "any ratiometric FRET pair yet described. Replacement of CFP and YFP with these two proteins in\n",
      "reporters of kinase activity, small GTPase activity and transmembrane voltage significantly improves\n",
      "photostability, FRET dynamic range and emission ratio changes. These improvements enhance detection\n",
      "of transient biochemical events such as neuronal action-potential firing and RhoA activation in\n",
      "growth cones.\n",
      " {\n",
      "\"main_contribution\": \"Improving FRET dynamic range with bright green and red fluorescent proteins\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that improves the FRET dynamic range using bright green and red fluorescent proteins. Therefore, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "869\n",
      "22961247\n",
      "A general approach to break the concentration barrier in single-molecule imaging.\n",
      "Single-molecule fluorescence imaging is often incompatible with physiological protein\n",
      "concentrations, as fluorescence background overwhelms an individual molecule's signal. We solve this\n",
      "problem with a new imaging approach called PhADE (PhotoActivation, Diffusion and Excitation). A\n",
      "protein of interest is fused to a photoactivatable protein (mKikGR) and introduced to its surface-\n",
      "immobilized substrate. After photoactivation of mKikGR near the surface, rapid diffusion of the\n",
      "unbound mKikGR fusion out of the detection volume eliminates background fluorescence, whereupon the\n",
      "bound molecules are imaged. We labeled the eukaryotic DNA replication protein flap endonuclease 1\n",
      "with mKikGR and added it to replication-competent Xenopus laevis egg extracts. PhADE imaging of high\n",
      "concentrations of the fusion construct revealed its dynamics and micrometer-scale movements on\n",
      "individual, replicating DNA molecules. Because PhADE imaging is in principle compatible with any\n",
      "photoactivatable fluorophore, it should have broad applicability in revealing single-molecule\n",
      "dynamics and stoichiometry of macromolecular protein complexes at previously inaccessible\n",
      "fluorophore concentrations.\n",
      " {\n",
      "\"main_contribution\": \"A general approach to break the concentration barrier in single-molecule imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method called PhADE, which allows for single-molecule imaging at physiological protein concentrations. This method uses photoactivation and diffusion to eliminate background fluorescence and image individual molecules with high sensitivity and resolution. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "870\n",
      "22983458\n",
      "Reversible labeling of native and fusion-protein motifs.\n",
      "The reversible covalent attachment of chemical probes to proteins has long been sought as a means to\n",
      "visualize and manipulate proteins. Here we demonstrate the full reversibility of post-translational\n",
      "custom pantetheine modification of Escherichia coli acyl carrier protein for visualization and\n",
      "functional studies. We use this iterative enzymatic methodology in vitro to reversibly label acyl\n",
      "carrier protein variants and apply these tools to NMR structural studies of protein-substrate\n",
      "interactions.\n",
      " {\n",
      "\"main_contribution\": \"reversible labeling of native and fusion-protein motifs\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the reversible covalent attachment of chemical probes to proteins for visualization and manipulation. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "871\n",
      "23023597\n",
      "A microfluidic device and computational platform for high-throughput live imaging of gene\n",
      "expression.\n",
      "To fully describe gene expression dynamics requires the ability to quantitatively capture expression\n",
      "in individual cells over time. Automated systems for acquiring and analyzing real-time images are\n",
      "needed to obtain unbiased data across many samples and conditions. We developed a microfluidics\n",
      "device, the RootArray, in which 64 Arabidopsis thaliana seedlings can be grown and their roots\n",
      "imaged by confocal microscopy over several days without manual intervention. To achieve high\n",
      "throughput, we decoupled acquisition from analysis. In the acquisition phase, we obtain images at\n",
      "low resolution and segment to identify regions of interest. Coordinates are communicated to the\n",
      "microscope to record the regions of interest at high resolution. In the analysis phase, we\n",
      "reconstruct three-dimensional objects from stitched high-resolution images and extract quantitative\n",
      "measurements from a virtual medial section of the root. We tracked hundreds of roots to capture\n",
      "detailed expression patterns of 12 transgenic reporter lines under different conditions.\n",
      " {\n",
      "\"main_contribution\": \"A microfluidics device and computational platform for high-throughput live imaging of gene expression\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a microfluidic device and computational platform for high-throughput live imaging of gene expression. This is evident in the title and abstract, which describe the device and its ability to quantitatively capture expression in individual cells over time. The paper's focus on imaging and microscopy, as well as its mention of confocal microscopy, segmentation, and three-dimensional reconstruction, further support this conclusion. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "872\n",
      "23042453\n",
      "DNA template strand sequencing of single-cells maps genomic rearrangements at high resolution.\n",
      "DNA rearrangements such as sister chromatid exchanges (SCEs) are sensitive indicators of genomic\n",
      "stress and instability, but they are typically masked by single-cell sequencing techniques. We\n",
      "developed Strand-seq to independently sequence parental DNA template strands from single cells,\n",
      "making it possible to map SCEs at orders-of-magnitude greater resolution than was previously\n",
      "possible. On average, murine embryonic stem (mES) cells exhibit eight SCEs, which are detected at a\n",
      "resolution of up to 23 bp. Strikingly, Strand-seq of 62 single mES cells predicts that the mm 9\n",
      "mouse reference genome assembly contains at least 17 incorrectly oriented segments totaling nearly\n",
      "1% of the genome. These misoriented contigs and fragments have persisted through several iterations\n",
      "of the mouse reference genome and have been difficult to detect using conventional sequencing\n",
      "techniques. The ability to map SCE events at high resolution and fine-tune reference genomes by\n",
      "Strand-seq dramatically expands the scope of single-cell sequencing.\n",
      " {\n",
      "\"main_contribution\": \"DNA template strand sequencing of single-cells maps genomic rearrangements at high resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called Strand-seq, which allows for the mapping of genomic rearrangements at high resolution by independently sequencing parental DNA template strands from single cells. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "873\n",
      "23064517\n",
      "Magnetic nanoparticle-mediated massively parallel mechanical modulation of single-cell behavior.\n",
      "We report a technique for generating controllable, time-varying and localizable forces on arrays of\n",
      "cells in a massively parallel fashion. To achieve this, we grow magnetic nanoparticle-dosed cells in\n",
      "defined patterns on micromagnetic substrates. By manipulating and coalescing nanoparticles within\n",
      "cells, we apply localized nanoparticle-mediated forces approaching cellular yield tensions on the\n",
      "cortex of HeLa cells. We observed highly coordinated responses in cellular behavior, including the\n",
      "p21-activated kinase-dependent generation of active, leading edge-type filopodia and biasing of the\n",
      "metaphase plate during mitosis. The large sample size and rapid sample generation inherent to this\n",
      "approach allow the analysis of cells at an unprecedented rate: in a single experiment, potentially\n",
      "tens of thousands of cells can be stimulated for high statistical accuracy in measurements. This\n",
      "technique shows promise as a tool for both cell analysis and control.\n",
      " {\n",
      "\"main_contribution\": \"Generating controllable, time-varying, and localizable forces on arrays of cells in a massively parallel fashion using magnetic nanoparticle-dosed cells grown in defined patterns on micromagnetic substrates\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "874\n",
      "23064519\n",
      "Automated velocity mapping of migrating cell populations (AVeMap).\n",
      "Characterizing the migration of a population of cells remains laborious and somewhat subjective.\n",
      "Advances in genetics and robotics allow researchers to perform many experiments in parallel, but\n",
      "analyzing the large sets of data remains a bottleneck. Here we describe a rapid, fully automated\n",
      "correlation-based method for cell migration analysis, compatible with standard video microscopy.\n",
      "This method allows for the computation of quantitative migration parameters via an extensive dynamic\n",
      "mapping of cell displacements.\n",
      " {\n",
      "\"main_contribution\": \"rapid, fully automated correlation-based method for cell migration analysis\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a rapid, fully automated correlation-based method for cell migration analysis. This method is compatible with standard video microscopy and allows for the computation of quantitative migration parameters via an extensive dynamic mapping of cell displacements. Therefore, the main contribution is the development of a new imaging method, and is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "875\n",
      "23085613\n",
      "Staining and embedding the whole mouse brain for electron microscopy.\n",
      "The development of methods for imaging large contiguous volumes with the electron microscope could\n",
      "allow the complete mapping of a whole mouse brain at the single-axon level. We developed a method\n",
      "based on prolonged immersion that enables staining and embedding of the entire mouse brain with\n",
      "uniform myelin staining and a moderate preservation of the tissue's ultrastructure. We tested the\n",
      "ability to follow myelinated axons using serial block-face electron microscopy.\n",
      " {\n",
      "\"main_contribution\": \"development of a method for imaging large contiguous volumes with the electron microscope\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method based on prolonged immersion that enables staining and embedding of the entire mouse brain with uniform myelin staining and moderate preservation of the tissue's ultrastructure, which allows for the complete mapping of a whole mouse brain at the single-axon level. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "876\n",
      "23085614\n",
      "Membrane-protein binding measured with solution-phase plasmonic nanocube sensors.\n",
      "We describe a solution-phase sensor of lipid-protein binding based on localized surface plasmon\n",
      "resonance (LSPR) of silver nanocubes. When silica-coated nanocubes are mixed in a suspension of\n",
      "lipid vesicles, supported membranes spontaneously assemble on their surfaces. Using a standard\n",
      "laboratory spectrophotometer, we calibrated the LSPR peak shift due to protein binding to the\n",
      "membrane surface and then characterized the lipid-binding specificity of a pleckstrin homology\n",
      "domain protein.\n",
      " {\n",
      "\"main_contribution\": \"A solution-phase sensor of lipid-protein binding based on localized surface plasmon resonance (LSPR) of silver nanocubes.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on localized surface plasmon resonance (LSPR) of silver nanocubes, which allows measuring membrane-protein binding in a solution-phase sensor. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "877\n",
      "23103879\n",
      "Phase-gradient microscopy in thick tissue with oblique back-illumination.\n",
      "Phase-contrast techniques, such as differential interference contrast microscopy, are widely used to\n",
      "obtain morphological images of unstained biological samples. The transillumination geometry required\n",
      "for these techniques restricts their application to thin samples. We introduce oblique back-\n",
      "illumination microscopy, a method of collecting en face phase-gradient images of thick scattering\n",
      "samples, enabling near-video-rate in vivo phase imaging with a miniaturized probe suitable for\n",
      "endoscopy.\n",
      " {\n",
      "\"main_contribution\": \"near-video-rate in vivo phase imaging with a miniaturized probe suitable for endoscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically oblique back-illumination microscopy, which enables near-video-rate in vivo phase imaging with a miniaturized probe suitable for endoscopy. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "878\n",
      "23103881\n",
      "Ultrabright photoactivatable fluorophores created by reductive caging.\n",
      "Sub-diffraction-limit imaging can be achieved by sequential localization of photoactivatable\n",
      "fluorophores, for which the image resolution depends on the number of photons detected per\n",
      "localization. We report a strategy for fluorophore caging that creates photoactivatable probes with\n",
      "high photon yields. Upon photoactivation, these probes can provide 10(4)-10(6) photons per\n",
      "localization and allow imaging of fixed samples with resolutions of several nanometers. This\n",
      "strategy can be applied to many fluorophores across the visible spectrum.\n",
      " {\n",
      "\"main_contribution\": \"creating photoactivatable fluorophores by reductive caging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on photoactivatable fluorophores created by reductive caging, which allows for sub-diffraction-limit imaging with high resolution. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "879\n",
      "23142870\n",
      "Membrane-protein structure determination by solid-state NMR spectroscopy of microcrystals.\n",
      "Membrane proteins are largely underrepresented among available atomic-resolution structures. The use\n",
      "of detergents in protein purification procedures hinders the formation of well-ordered crystals for\n",
      "X-ray crystallography and leads to slower molecular tumbling, impeding the application of solution-\n",
      "state NMR. Solid-state magic-angle spinning NMR spectroscopy is an emerging method for membrane-\n",
      "protein structural biology that can overcome these technical problems. Here we present the solid-\n",
      "state NMR structure of the transmembrane domain of the Yersinia enterocolitica adhesin A (YadA). The\n",
      "sample was derived from crystallization trials that yielded only poorly diffracting microcrystals.\n",
      "We solved the structure using a single, uniformly (13)C- and (15)N-labeled sample. In addition,\n",
      "solid-state NMR allowed us to acquire information on the flexibility and mobility of parts of the\n",
      "structure, which, in combination with evolutionary conservation information, presents new insights\n",
      "into the autotransport mechanism of YadA.\n",
      " {\n",
      "\"main_contribution\": \"Solid-state NMR spectroscopy for membrane-protein structure determination\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically solid-state NMR spectroscopy, for determining the structure of membrane proteins. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "880\n",
      "23142872\n",
      "Systematic reconstruction of RNA functional motifs with high-throughput microfluidics.\n",
      "We present RNA-mechanically induced trapping of molecular interactions (RNA-MITOMI), a microfluidic\n",
      "platform that allows integrated synthesis and functional assays for programmable RNA libraries. The\n",
      "interaction of a comprehensive library of RNA mutants with stem-loop-binding protein precisely\n",
      "defined the RNA structural and sequence features that govern affinity. The functional motif\n",
      "reconstructed in a single experiment on our platform uncovers new binding specificities and enriches\n",
      "interpretation of phylogenetic data.\n",
      " {\n",
      "\"main_contribution\": \"Systematic reconstruction of RNA functional motifs with high-throughput microfluidics\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for reconstructing RNA functional motifs using high-throughput microfluidics. The main contribution of the paper is the development of this new method, which allows for the integrated synthesis and functional assays of programmable RNA libraries. This method is not an imaging technique or method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "881\n",
      "23142873\n",
      "Two-photon optogenetics of dendritic spines and neural circuits.\n",
      "We demonstrate a two-photon optogenetic method that generates action potentials in neurons with\n",
      "single-cell precision, using the red-shifted opsin C1V1(T). We applied the method to optically map\n",
      "synaptic circuits in mouse neocortical brain slices and to activate small dendritic regions and\n",
      "individual spines. Using a spatial light modulator, we split the laser beam onto several neurons and\n",
      "performed simultaneous optogenetic activation of selected neurons in three dimensions.\n",
      " {\n",
      "\"main_contribution\": \"two-photon optogenetic method for generating action potentials in neurons with single-cell precision\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a two-photon optogenetic method for generating action potentials in neurons with single-cell precision. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "882\n",
      "23169303\n",
      "Two-photon optogenetic toolbox for fast inhibition, excitation and bistable modulation.\n",
      "Optogenetics with microbial opsin genes has enabled high-speed control of genetically specified cell\n",
      "populations in intact tissue. However, it remains a challenge to independently control subsets of\n",
      "cells within the genetically targeted population. Although spatially precise excitation of target\n",
      "molecules can be achieved using two-photon laser-scanning microscopy (TPLSM) hardware, the\n",
      "integration of two-photon excitation with optogenetics has thus far required specialized equipment\n",
      "or scanning and has not yet been widely adopted. Here we take a complementary approach, developing\n",
      "opsins with custom kinetic, expression and spectral properties uniquely suited to scan times typical\n",
      "of the raster approach that is ubiquitous in TPLSMlaboratories. We use a range of culture, slice and\n",
      "mammalian in vivo preparations to demonstrate the versatility of this toolbox, and we quantitatively\n",
      "map parameter space for fast excitation, inhibition and bistable control. Together these advances\n",
      "may help enable broad adoption of integrated optogenetic and TPLSMtechnologies across experimental\n",
      "fields and systems.\n",
      " {\n",
      "\"main_contribution\": \"development of a two-photon optogenetic toolbox for fast inhibition, excitation, and bistable modulation\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a two-photon optogenetic toolbox for fast inhibition, excitation, and bistable modulation. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "883\n",
      "23202434\n",
      "Conversion of human fibroblasts to angioblast-like progenitor cells.\n",
      "Lineage conversion of one somatic cell type to another is an attractive approach for generating\n",
      "specific human cell types. Lineage conversion can be direct, in the absence of proliferation and\n",
      "multipotent progenitor generation, or indirect, by the generation of expandable multipotent\n",
      "progenitor states. We report the development of a reprogramming methodology in which cells\n",
      "transition through a plastic intermediate state, induced by brief exposure to reprogramming factors,\n",
      "followed by differentiation. We use this approach to convert human fibroblasts to mesodermal\n",
      "progenitor cells, including by non-integrative approaches. These progenitor cells demonstrated\n",
      "bipotent differentiation potential and could generate endothelial and smooth muscle lineages.\n",
      "Differentiated endothelial cells exhibited neo-angiogenesis and anastomosis in vivo. This\n",
      "methodology for indirect lineage conversion to angioblast-like cells adds to the armamentarium of\n",
      "reprogramming approaches aimed at the study and treatment of ischemic pathologies.\n",
      " {\n",
      "\"main_contribution\": \"Conversion of human fibroblasts to angioblast-like progenitor cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for converting human fibroblasts to mesodermal progenitor cells, including angioblast-like cells, which can differentiate into endothelial and smooth muscle lineages. The method involves brief exposure to reprogramming factors followed by differentiation, and does not involve the generation of expandable multipotent progenitor states or the use of integrative approaches.\n",
      "\n",
      "The main contribution of the paper is the development of this new reprogramming methodology, which is described in sentences (B) and (C) of the abstract. This methodology is not an imaging technique or method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "884\n",
      "23223154\n",
      "Fast multicolor 3D imaging using aberration-corrected multifocus microscopy.\n",
      "Conventional acquisition of three-dimensional (3D) microscopy data requires sequential z scanning\n",
      "and is often too slow to capture biological events. We report an aberration-corrected multifocus\n",
      "microscopy method capable of producing an instant focal stack of nine 2D images. Appended to an\n",
      "epifluorescence microscope, the multifocus system enables high-resolution 3D imaging in multiple\n",
      "colors with single-molecule sensitivity, at speeds limited by the camera readout time of a single\n",
      "image.\n",
      " {\n",
      "\"main_contribution\": \"Fast multicolor 3D imaging using aberration-corrected multifocus microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called aberration-corrected multifocus microscopy, which enables fast multicolor 3D imaging with single-molecule sensitivity. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "885\n",
      "23241632\n",
      "Quantitative optical trapping on single organelles in cell extract.\n",
      "We have developed an optical trapping method to precisely measure the force generated by motor\n",
      "proteins on single organelles of unknown size in cell extract. This approach, termed VMatch, permits\n",
      "the functional interrogation of native motor complexes. We apply VMatch to measure the force, number\n",
      "and activity of kinesin-1 on motile lipid droplets isolated from the liver of normally fed and food-\n",
      "deprived rats.\n",
      " {\n",
      "\"main_contribution\": \"measuring force generated by motor proteins on single organelles\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method, VMatch, which can measure the force generated by motor proteins on single organelles of unknown size in cell extract. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "886\n",
      "23263690\n",
      "Conditional genome engineering in Toxoplasma gondii uncovers alternative invasion mechanisms.\n",
      "We established a conditional site-specific recombination system based on dimerizable Cre\n",
      "recombinase-mediated recombination in the apicomplexan parasite Toxoplasma gondii. Using a new\n",
      "single-vector strategy that allows ligand-dependent, efficient removal of a gene of interest, we\n",
      "generated three knockouts of apicomplexan genes considered essential for host-cell invasion. Our\n",
      "findings uncovered the existence of an alternative invasion pathway in apicomplexan parasites.\n",
      " {\n",
      "\"main_contribution\": \"identification of an alternative invasion pathway in apicomplexan parasites\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe research on Toxoplasma gondii, an apicomplexan parasite. The main contribution of the paper is the identification of an alternative invasion pathway in these parasites, which is described in sentence (D) of the abstract. This contribution does not involve the development of a new imaging method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "887\n",
      "23263691\n",
      "Single-molecule analysis of gene expression using two-color RNA labeling in live yeast.\n",
      "Live-cell imaging of mRNA yields important insights into gene expression, but it has generally been\n",
      "limited to the labeling of one RNA species and has never been used to count single mRNAs over time\n",
      "in yeast. We demonstrate a two-color imaging system with single-molecule resolution using MS2 and\n",
      "PP7 RNA labeling. We use this methodology to measure intrinsic noise in mRNA levels and RNA\n",
      "polymerase II kinetics at a single gene.\n",
      " {\n",
      "\"main_contribution\": \"Measuring intrinsic noise in mRNA levels and RNA polymerase II kinetics at a single gene using a two-color imaging system with single-molecule resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that uses a two-color imaging system with single-molecule resolution to measure intrinsic noise in mRNA levels and RNA polymerase II kinetics at a single gene. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "888\n",
      "23291722\n",
      "iGLuc: a luciferase-based inflammasome and protease activity reporter.\n",
      "Measurement of protease activity in living cells or organisms remains a challenging task. We here\n",
      "present a transgene-encoded biosensor that reports the proteolytic activity of caspase-1 in the\n",
      "course of inflammasome activation and that of other proteases in a highly sensitive and specific\n",
      "manner. This protease reporter is based on the biological activity of a pro-interleukin\n",
      "(IL)-1β-Gaussia luciferase (iGLuc) fusion construct, in which pro-IL-1β-dependent formation of\n",
      "protein aggregates renders GLuc enzyme inactive. Cleavage leads to monomerization of this biosensor\n",
      "protein, resulting in a strong gain in luciferase activity. Exchange of the canonical caspase-1\n",
      "cleavage site in this reporter construct allows the generation of protease biosensors with\n",
      "additional specificities. The high sensitivity, signal-to-background ratio and specificity of the\n",
      "iGLuc system renders it a useful tool to study proteolytic events in mouse and human cells at high\n",
      "throughput and to monitor protease activity in mice in vivo.\n",
      " {\n",
      "\"main_contribution\": \"A luciferase-based inflammasome and protease activity reporter.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method based on a luciferase-based biosensor that reports proteolytic activity in living cells or organisms. This imaging technique allows for the measurement of protease activity in a highly sensitive and specific manner, making it a useful tool to study proteolytic events in mouse and human cells at high throughput and to monitor protease activity in mice in vivo. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "889\n",
      "23314171\n",
      "An optimized fluorescent probe for visualizing glutamate neurotransmission.\n",
      "We describe an intensity-based glutamate-sensing fluorescent reporter (iGluSnFR) with signal-to-\n",
      "noise ratio and kinetics appropriate for in vivo imaging. We engineered iGluSnFR in vitro to\n",
      "maximize its fluorescence change, and we validated its utility for visualizing glutamate release by\n",
      "neurons and astrocytes in increasingly intact neurological systems. In hippocampal culture, iGluSnFR\n",
      "detected single field stimulus-evoked glutamate release events. In pyramidal neurons in acute brain\n",
      "slices, glutamate uncaging at single spines showed that iGluSnFR responds robustly and specifically\n",
      "to glutamate in situ, and responses correlate with voltage changes. In mouse retina, iGluSnFR-\n",
      "expressing neurons showed intact light-evoked excitatory currents, and the sensor revealed tonic\n",
      "glutamate signaling in response to light stimuli. In worms, glutamate signals preceded and predicted\n",
      "postsynaptic calcium transients. In zebrafish, iGluSnFR revealed spatial organization of direction-\n",
      "selective synaptic activity in the optic tectum. Finally, in mouse forelimb motor cortex, iGluSnFR\n",
      "expression in layer V pyramidal neurons revealed task-dependent single-spine activity during\n",
      "running.\n",
      " {\n",
      "\"main_contribution\": \"An optimized fluorescent probe for visualizing glutamate neurotransmission\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically an intensity-based glutamate-sensing fluorescent reporter (iGluSnFR) for visualizing glutamate neurotransmission. This is evident in the title and abstract of the paper, which describe the probe and its ability to detect glutamate release events in various neurological systems. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "890\n",
      "23314172\n",
      "Detection of histone modifications at specific gene loci in single cells in histological sections.\n",
      "Chromatin immunoprecipitation assays have contributed greatly to our understanding of the role of\n",
      "histone modifications in gene regulation. However, they do not permit analysis with single-cell\n",
      "resolution, thus confounding analyses of heterogeneous cell populations. Here we present a method\n",
      "that permits visualization of histone modifications of single genomic loci with single-cell\n",
      "resolution in formaldehyde-fixed paraffin-embedded tissue sections based on combined use of in situ\n",
      "hybridization and proximity ligation assays. We show that dimethylation of lysine 4 of histone H3\n",
      "(H3K4me2) at the MYH11 locus is restricted to the smooth muscle cell (SMC) lineage in human and\n",
      "mouse tissue sections and that the mark persists even in phenotypically modulated SMC in\n",
      "atherosclerotic lesions that show no detectable expression of SMC marker genes. This methodology has\n",
      "promise for broad applications in the study of epigenetic mechanisms in complex multicellular\n",
      "tissues in development and disease.\n",
      " {\n",
      "\"main_contribution\": \"detection of histone modifications at specific gene loci in single cells in histological sections\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method for detecting histone modifications at specific gene loci in single cells in histological sections, which is an imaging technique. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "891\n",
      "23314173\n",
      "Lentiviral vector-based insertional mutagenesis identifies genes associated with liver cancer.\n",
      "Transposons and γ-retroviruses have been efficiently used as insertional mutagens in different\n",
      "tissues to identify molecular culprits of cancer. However, these systems are characterized by\n",
      "recurring integrations that accumulate in tumor cells and that hamper the identification of early\n",
      "cancer-driving events among bystander and progression-related events. We developed an insertional\n",
      "mutagenesis platform based on lentiviral vectors (LVVs) by which we could efficiently induce\n",
      "hepatocellular carcinoma (HCC) in three different mouse models. By virtue of the LVV's replication-\n",
      "deficient nature and broad genome-wide integration pattern, LVV-based insertional mutagenesis\n",
      "allowed identification of four previously unknown liver cancer-associated genes from a limited\n",
      "number of integrations. We validated the oncogenic potential of all the identified genes in vivo,\n",
      "with different levels of penetrance. The newly identified genes are likely to play a role in human\n",
      "cancer because they are upregulated, amplified and/or deleted in human HCCs and can predict clinical\n",
      "outcomes of patients.\n",
      " {\n",
      "\"main_contribution\": \"identification of four previously unknown liver cancer-associated genes\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the identification of four new genes associated with liver cancer using a lentiviral vector-based insertional mutagenesis platform. This does not involve the development of a new imaging method, so \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "892\n",
      "23407553\n",
      "Quantitative genetic-interaction mapping in mammalian cells.\n",
      "Mapping genetic interactions (GIs) by simultaneously perturbing pairs of genes is a powerful tool\n",
      "for understanding complex biological phenomena. Here we describe an experimental platform for\n",
      "generating quantitative GI maps in mammalian cells using a combinatorial RNA interference strategy.\n",
      "We performed ∼11,000 pairwise knockdowns in mouse fibroblasts, focusing on 130 factors involved in\n",
      "chromatin regulation to create a GI map. Comparison of the GI and protein-protein interaction (PPI)\n",
      "data revealed that pairs of genes exhibiting positive GIs and/or similar genetic profiles were\n",
      "predictive of the corresponding proteins being physically associated. The mammalian GI map\n",
      "identified pathways and complexes but also resolved functionally distinct submodules within larger\n",
      "protein complexes. By integrating GI and PPI data, we created a functional map of chromatin\n",
      "complexes in mouse fibroblasts, revealing that the PAF complex is a central player in the mammalian\n",
      "chromatin landscape.\n",
      " {\n",
      "\"main_contribution\": \"a quantitative genetic-interaction map in mammalian cells using a combinatorial RNA interference strategy\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a method for mapping genetic interactions in mammalian cells, which does not involve imaging. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "893\n",
      "23416756\n",
      "Single-chromosome transcriptional profiling reveals chromosomal gene expression regulation.\n",
      "We report intron chromosomal expression FISH (iceFISH), a multiplex imaging method for measuring\n",
      "gene expression and chromosome structure simultaneously on single chromosomes. We find substantial\n",
      "differences in transcriptional frequency between genes on a translocated chromosome and the same\n",
      "genes in their normal chromosomal context in the same cell. Correlations between genes on a single\n",
      "chromosome pointed toward a cis chromosome-level transcriptional interaction spanning 14.3\n",
      "megabases.\n",
      " {\n",
      "\"main_contribution\": \"measuring gene expression and chromosome structure simultaneously on single chromosomes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work as a whole, giving background information about what the work is about (\"Single-chromosome transcriptional profiling\") and stating the goal of the work (\"reveals chromosomal gene expression regulation\"). \n",
      "\n",
      "The abstract gives more detail about how the goal was achieved. It describes the method used (\"intron chromosomal expression FISH (iceFISH)\"), which is a multiplex imaging method, and states that this method was used to measure gene expression and chromosome structure simultaneously on single chromosomes. \n",
      "\n",
      "The results of the work are also described in the abstract (\"We find substantial differences in transcriptional frequency between genes...\"). However, the main contribution of the paper is the development of a new imaging method (iceFISH), which allows for measuring gene expression and chromosome structure simultaneously on single chromosomes. \n",
      "\n",
      "Therefore, the main contribution is recorded as \"measuring gene expression and chromosome structure simultaneously on single chromosomes\", and the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "894\n",
      "23435261\n",
      "Immunofluorescence and fluorescent-protein tagging show high correlation for protein localization in\n",
      "mammalian cells.\n",
      "Imaging techniques such as immunofluorescence (IF) and the expression of fluorescent protein (FP)\n",
      "fusions are widely used to investigate the subcellular distribution of proteins. Here we report a\n",
      "systematic analysis of >500 human proteins comparing the localizations obtained in live versus fixed\n",
      "cells using FPs and IF, respectively. We identify systematic discrepancies between IF and FPs as\n",
      "well as between FP tagging at the N and C termini. The analysis shows that for 80% of the proteins,\n",
      "IF and FPs yield the same subcellular distribution, and the locations of 250 previously unlocalized\n",
      "proteins were determined by the overlap between the two methods. Approximately 60% of proteins\n",
      "localize to multiple organelles for both methods, indicating a complex subcellular protein\n",
      "organization. These results show that both IF and FP tagging are reliable techniques and demonstrate\n",
      "the usefulness of an integrative approach for a complete investigation of the subcellular human\n",
      "proteome.\n",
      " {\n",
      "\"main_contribution\": \"An integrative approach for a complete investigation of the subcellular human proteome\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing an integrative approach using both immunofluorescence (IF) and fluorescent-protein tagging to investigate the subcellular distribution of proteins in mammalian cells. This approach allows for a more comprehensive understanding of protein localization, as it combines the strengths of both techniques and helps to identify systematic discrepancies between them. Therefore, the main contribution is the development of an imaging method, which is an integrative approach that combines IF and FP tagging.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "895\n",
      "23443633\n",
      "Zymography methods for visualizing hydrolytic enzymes.\n",
      "Zymography is a technique for studying hydrolytic enzymes on the basis of substrate degradation. It\n",
      "is a powerful, but often misinterpreted, tool yielding information on potential hydrolytic\n",
      "activities, enzyme forms and the locations of active enzymes. In this Review, zymography techniques\n",
      "are compared in terms of advantages, limitations and interpretations. With in gel zymography, enzyme\n",
      "forms are visualized according to their molecular weights. Proteolytic activities are localized in\n",
      "tissue sections with in situ zymography. In vivo zymography can pinpoint proteolytic activity to\n",
      "sites in an intact organism. Future development of novel substrate probes and improvement in\n",
      "detection and imaging methods will increase the applicability of zymography for (reverse)\n",
      "degradomics studies.\n",
      " {\n",
      "\"main_contribution\": \"zymography methods for visualizing hydrolytic enzymes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of zymography methods for visualizing hydrolytic enzymes, which is an imaging technique. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "896\n",
      "23455923\n",
      "Ultrahigh accuracy imaging modality for super-localization microscopy.\n",
      "Super-localization microscopy encompasses techniques that depend on the accurate localization of\n",
      "individual molecules from generally low-light images. The obtainable localization accuracies,\n",
      "however, are ultimately limited by the image detector's pixelation and noise. We present the\n",
      "ultrahigh accuracy imaging modality (UAIM), which allows users to obtain accuracies approaching the\n",
      "accuracy that is achievable only in the absence of detector pixelation and noise, and which we found\n",
      "can experimentally provide a >200% accuracy improvement over conventional low-light imaging.\n",
      " {\n",
      "\"main_contribution\": \"Ultrahigh accuracy imaging modality for super-localization microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an ultrahigh accuracy imaging modality (UAIM) for super-localization microscopy, which allows users to obtain accuracies approaching those achievable in the absence of detector pixelation and noise. This is a new imaging technique or method, so the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "897\n",
      "23503053\n",
      "Barcoding cells using cell-surface programmable DNA-binding domains.\n",
      "We report an approach to barcode cells through cell-surface expression of programmable zinc-finger\n",
      "DNA-binding domains (surface zinc fingers, sZFs). We show that sZFs enable sequence-specific\n",
      "labeling of living cells by dsDNA, and we develop a sequential labeling approach to image more than\n",
      "three cell types in mixed populations using three fluorophores. We demonstrate the versatility of\n",
      "sZFs through applications in which they serve as surrogate reporters, function as selective cell\n",
      "capture reagents and facilitate targeted cellular delivery of viruses.\n",
      " {\n",
      "\"main_contribution\": \"Barcoding cells using cell-surface programmable DNA-binding domains\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the use of cell-surface programmable zinc-finger DNA-binding domains (sZFs) for barcoding cells. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "898\n",
      "23524392\n",
      "A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum.\n",
      "We report a monomeric yellow-green fluorescent protein, mNeonGreen, derived from a tetrameric\n",
      "fluorescent protein from the cephalochordate Branchiostoma lanceolatum. mNeonGreen is the brightest\n",
      "monomeric green or yellow fluorescent protein yet described to our knowledge, performs exceptionally\n",
      "well as a fusion tag for traditional imaging as well as stochastic single-molecule superresolution\n",
      "imaging and is an excellent fluorescence resonance energy transfer (FRET) acceptor for the newest\n",
      "cyan fluorescent proteins.\n",
      " {\n",
      "\"main_contribution\": \"A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a monomeric yellow-green fluorescent protein called mNeonGreen, which is derived from a tetrameric fluorescent protein from the cephalochordate Branchiostoma lanceolatum. This protein performs exceptionally well as a fusion tag for traditional imaging and stochastic single-molecule superresolution imaging and is an excellent fluorescence resonance energy transfer (FRET) acceptor for new cyan fluorescent proteins. Therefore, the main contribution of the paper is related to imaging techniques, and thus, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "899\n",
      "23524393\n",
      "Whole-brain functional imaging at cellular resolution using light-sheet microscopy.\n",
      "Brain function relies on communication between large populations of neurons across multiple brain\n",
      "areas, a full understanding of which would require knowledge of the time-varying activity of all\n",
      "neurons in the central nervous system. Here we use light-sheet microscopy to record activity,\n",
      "reported through the genetically encoded calcium indicator GCaMP5G, from the entire volume of the\n",
      "brain of the larval zebrafish in vivo at 0.8 Hz, capturing more than 80% of all neurons at single-\n",
      "cell resolution. Demonstrating how this technique can be used to reveal functionally defined\n",
      "circuits across the brain, we identify two populations of neurons with correlated activity patterns.\n",
      "One circuit consists of hindbrain neurons functionally coupled to spinal cord neuropil. The other\n",
      "consists of an anatomically symmetric population in the anterior hindbrain, with activity in the\n",
      "left and right halves oscillating in antiphase, on a timescale of 20 s, and coupled to equally slow\n",
      "oscillations in the inferior olive.\n",
      " {\n",
      "\"main_contribution\": \"Whole-brain functional imaging at cellular resolution using light-sheet microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development and use of light-sheet microscopy to image the entire brain of a larval zebrafish in vivo at 0.8 Hz, capturing more than 80% of all neurons at single-cell resolution. This technique allows for the identification of functionally defined circuits across the brain and provides new insights into brain function. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "900\n",
      "23524394\n",
      "Single-molecule imaging of transcription factor binding to DNA in live mammalian cells.\n",
      "Imaging single fluorescent proteins in living mammalian cells is challenged by out-of-focus\n",
      "fluorescence excitation. To reduce out-of-focus fluorescence we developed reflected light-sheet\n",
      "microscopy (RLSM), a fluorescence microscopy method allowing selective plane illumination throughout\n",
      "the nuclei of living mammalian cells. A thin light sheet parallel to the imaging plane and close to\n",
      "the sample surface is generated by reflecting an elliptical laser beam incident from the top by 90°\n",
      "with a small mirror. The thin light sheet allows for an increased signal-to-background ratio\n",
      "superior to that in previous illumination schemes and enables imaging of single fluorescent proteins\n",
      "with up to 100-Hz time resolution. We demonstrated the single-molecule sensitivity of RLSM by\n",
      "measuring the DNA-bound fraction of glucocorticoid receptor (GR) and determining the residence times\n",
      "on DNA of various oligomerization states and mutants of GR and estrogen receptor-α (ER), which\n",
      "permitted us to resolve different modes of DNA binding of GR. We demonstrated two-color single-\n",
      "molecule imaging by observing the spatiotemporal colocalization of two different protein pairs. Our\n",
      "single-molecule measurements and statistical analysis revealed dynamic properties of transcription\n",
      "factors.\n",
      " {\n",
      "\"main_contribution\": \"The development of a new imaging technique called reflected light-sheet microscopy (RLSM) for imaging single fluorescent proteins in live mammalian cells.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "901\n",
      "23563794\n",
      "Mapping genetic interactions in human cancer cells with RNAi and multiparametric phenotyping.\n",
      "Genetic interactions influence many phenotypes and can be used as a powerful experimental tool to\n",
      "discover functional relationships between genes. Here we describe a robust and scalable method to\n",
      "systematically map genetic interactions in human cancer cells using combinatorial RNAi and high-\n",
      "throughput imaging. Through automated, single-cell phenotyping, we measured genetic interactions\n",
      "across a broad spectrum of phenotypes, including cell count, cell eccentricity and nuclear area. We\n",
      "mapped genetic interactions of epigenetic regulators in colon cancer cells, recovering known protein\n",
      "complexes. Our study also revealed the prospects and challenges of studying genetic interactions in\n",
      "human cells using multiparametric phenotyping.\n",
      " {\n",
      "\"main_contribution\": \"Systematic mapping of genetic interactions in human cancer cells using combinatorial RNAi and high-throughput imaging\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a systematic method to map genetic interactions in human cancer cells using combinatorial RNAi and high-throughput imaging. This method is not a new imaging technique or method, but rather a way of using existing imaging technology to gain insights into genetic interactions in cancer cells. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "902\n",
      "23563795\n",
      "Adhesion strength-based, label-free isolation of human pluripotent stem cells.\n",
      "We demonstrate substantial differences in 'adhesive signature' between human pluripotent stem cells\n",
      "(hPSCs), partially reprogrammed cells, somatic cells and hPSC-derived differentiated progeny. We\n",
      "exploited these differential adhesion strengths to rapidly (over ∼10 min) and efficiently isolate\n",
      "fully reprogrammed induced hPSCs (hiPSCs) as intact colonies from heterogeneous reprogramming\n",
      "cultures and from differentiated progeny using microfluidics. hiPSCs were isolated label free,\n",
      "enriched to 95%-99% purity with >80% survival, and had normal transcriptional profiles,\n",
      "differentiation potential and karyotypes. We also applied this strategy to isolate hPSCs (hiPSCs and\n",
      "human embryonic stem cells) during routine culture and show that it may be extended to isolate hPSC-\n",
      "derived lineage-specific stem cells or differentiated cells.\n",
      " {\n",
      "\"main_contribution\": \"Adhesion strength-based, label-free isolation of human pluripotent stem cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method for isolating human pluripotent stem cells based on their adhesion strength, which does not involve imaging techniques. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "903\n",
      "23584186\n",
      "A simple tool to improve pluripotent stem cell differentiation.\n",
      "We describe a method to help overcome restrictions on the differentiation propensities of human\n",
      "pluripotent stem cells. Culturing pluripotent stem cells in dimethylsulfoxide (DMSO) activates the\n",
      "retinoblastoma protein, increases the proportion of cells in the early G1 phase of the cell cycle\n",
      "and, in more than 25 embryonic and induced pluripotent stem cell lines, improves directed\n",
      "differentiation into multiple lineages. DMSO treatment also improves differentiation into terminal\n",
      "cell types in several cell lines.\n",
      " {\n",
      "\"main_contribution\": \"a method to improve pluripotent stem cell differentiation\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new method for improving pluripotent stem cell differentiation. This method involves culturing pluripotent stem cells in dimethylsulfoxide (DMSO), which activates the retinoblastoma protein and increases the proportion of cells in the early G1 phase of the cell cycle, leading to improved directed differentiation into multiple lineages. Therefore, the main contribution is not an imaging method, and the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "904\n",
      "23624665\n",
      "Measuring image resolution in optical nanoscopy.\n",
      "Resolution in optical nanoscopy (or super-resolution microscopy) depends on the localization\n",
      "uncertainty and density of single fluorescent labels and on the sample's spatial structure.\n",
      "Currently there is no integral, practical resolution measure that accounts for all factors. We\n",
      "introduce a measure based on Fourier ring correlation (FRC) that can be computed directly from an\n",
      "image. We demonstrate its validity and benefits on two-dimensional (2D) and 3D localization\n",
      "microscopy images of tubulin and actin filaments. Our FRC resolution method makes it possible to\n",
      "compare achieved resolutions in images taken with different nanoscopy methods, to optimize and rank\n",
      "different emitter localization and labeling strategies, to define a stopping criterion for data\n",
      "acquisition, to describe image anisotropy and heterogeneity, and even to estimate the average number\n",
      "of localizations per emitter. Our findings challenge the current focus on obtaining the best\n",
      "localization precision, showing instead how the best image resolution can be achieved as fast as\n",
      "possible.\n",
      " {\n",
      "\"main_contribution\": \"A measure based on Fourier ring correlation (FRC) that can be computed directly from an image\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "905\n",
      "23644547\n",
      "Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle\n",
      "cryo-EM.\n",
      "In recent work with large high-symmetry viruses, single-particle electron cryomicroscopy (cryo-EM)\n",
      "has achieved the determination of near-atomic-resolution structures by allowing direct fitting of\n",
      "atomic models into experimental density maps. However, achieving this goal with smaller particles of\n",
      "lower symmetry remains challenging. Using a newly developed single electron-counting detector, we\n",
      "confirmed that electron beam-induced motion substantially degrades resolution, and we showed that\n",
      "the combination of rapid readout and nearly noiseless electron counting allow image blurring to be\n",
      "corrected to subpixel accuracy, restoring intrinsic image information to high resolution (Thon rings\n",
      "visible to ∼3 Å). Using this approach, we determined a 3.3-Å-resolution structure of an ∼700-kDa\n",
      "protein with D7 symmetry, the Thermoplasma acidophilum 20S proteasome, showing clear side-chain\n",
      "density. Our method greatly enhances image quality and data acquisition efficiency-key bottlenecks\n",
      "in applying near-atomic-resolution cryo-EM to a broad range of protein samples.\n",
      " {\n",
      "\"main_contribution\": \"Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that enables near-atomic-resolution single-particle cryo-EM by combining rapid readout and nearly noiseless electron counting to correct image blurring caused by electron beam-induced motion. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "906\n",
      "23666410\n",
      "The Caenorhabditis elegans Lifespan Machine.\n",
      "The measurement of lifespan pervades aging research. Because lifespan results from complex\n",
      "interactions between genetic, environmental and stochastic factors, it varies widely even among\n",
      "isogenic individuals. The actions of molecular mechanisms on lifespan are therefore visible only\n",
      "through their statistical effects on populations. Indeed, survival assays in Caenorhabditis elegans\n",
      "have provided critical insights into evolutionarily conserved determinants of aging. To enable the\n",
      "rapid acquisition of survival curves at an arbitrary statistical resolution, we developed a scalable\n",
      "imaging and analysis platform to observe nematodes over multiple weeks across square meters of agar\n",
      "surface at 8-μm resolution. The automated method generates a permanent visual record of individual\n",
      "deaths from which survival curves are constructed and validated, producing data consistent with\n",
      "results from the manual method of survival curve acquisition for several mutants in both standard\n",
      "and stressful environments. Our approach permits rapid, detailed reverse-genetic and chemical\n",
      "screens for effects on survival and enables quantitative investigations into the statistical\n",
      "structure of aging.\n",
      " {\n",
      "\"main_contribution\": \"A scalable imaging and analysis platform to observe nematodes over multiple weeks across square meters of agar surface at 8-μm resolution.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "907\n",
      "23708387\n",
      "Video-rate nanoscopy using sCMOS camera-specific single-molecule localization algorithms.\n",
      "Newly developed scientific complementary metal-oxide semiconductor (sCMOS) cameras have the\n",
      "potential to dramatically accelerate data acquisition, enlarge the field of view and increase the\n",
      "effective quantum efficiency in single-molecule switching nanoscopy. However, sCMOS-intrinsic pixel-\n",
      "dependent readout noise substantially lowers the localization precision and introduces localization\n",
      "artifacts. We present algorithms that overcome these limitations and that provide unbiased, precise\n",
      "localization of single molecules at the theoretical limit. Using these in combination with a multi-\n",
      "emitter fitting algorithm, we demonstrate single-molecule localization super-resolution imaging at\n",
      "rates of up to 32 reconstructed images per second in fixed and living cells.\n",
      " {\n",
      "\"main_contribution\": \"single-molecule localization super-resolution imaging at rates of up to 32 reconstructed images per second\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows for single-molecule localization super-resolution imaging at high speeds, specifically up to 32 reconstructed images per second. This method utilizes sCMOS camera-specific single-molecule localization algorithms and overcomes the limitations of sCMOS-intrinsic pixel-dependent readout noise. Therefore, the main contribution is indeed the development of a new imaging technique or method, and thus \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "908\n",
      "23722209\n",
      "Cellular-resolution connectomics: challenges of dense neural circuit reconstruction.\n",
      "Neuronal networks are high-dimensional graphs that are packed into three-dimensional nervous tissue\n",
      "at extremely high density. Comprehensively mapping these networks is therefore a major challenge.\n",
      "Although recent developments in volume electron microscopy imaging have made data acquisition\n",
      "feasible for circuits comprising a few hundreds to a few thousands of neurons, data analysis is\n",
      "massively lagging behind. The aim of this perspective is to summarize and quantify the challenges\n",
      "for data analysis in cellular-resolution connectomics and describe current solutions involving\n",
      "online crowd-sourcing and machine-learning approaches.\n",
      " {\n",
      "\"main_contribution\": \"Comprehensively mapping high-dimensional graphs of neural networks\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the challenges of reconstructing dense neural circuits at the cellular level using connectomics. The main contribution of the paper is to summarize and quantify these challenges and propose solutions, including online crowd-sourcing and machine-learning approaches. Therefore, the main contribution is not the development of a new imaging method, but rather a contribution to the field of connectomics and neural network mapping.\n",
      "\n",
      "Sentence breakdown:\n",
      "\n",
      "Title:\n",
      "\n",
      "* Cellular-resolution connectomics: challenges of dense neural circuit reconstruction.\n",
      "\n",
      "Abstract:\n",
      "\n",
      "* Neuronal networks are high-dimensional graphs that are packed into three-dimensional nervous tissue at extremely high density.\n",
      "* Comprehensively mapping these networks is therefore a major challenge.\n",
      "* Although recent developments in volume electron microscopy imaging have made data acquisition feasible for circuits comprising a few hundreds to a few thousands of neurons, data analysis is massively lagging behind.\n",
      "* The aim of this perspective is to summarize and quantify the challenges for data analysis in cellular-resolution connectomics and describe current solutions involving online crowd-sourcing and machine-learning approaches.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "909\n",
      "23722211\n",
      "Mapping brain circuitry with a light microscope.\n",
      "The beginning of the 21st century has seen a renaissance in light microscopy and anatomical tract\n",
      "tracing that together are rapidly advancing our understanding of the form and function of neuronal\n",
      "circuits. The introduction of instruments for automated imaging of whole mouse brains, new cell\n",
      "type–specific and trans-synaptic tracers, and computational methods for handling the whole-brain\n",
      "data sets has opened the door to neuroanatomical studies at an unprecedented scale. We present an\n",
      "overview of the present state and future opportunities in charting long-range and local connectivity\n",
      "in the entire mouse brain and in linking brain circuits to function.\n",
      " {\n",
      "\"main_contribution\": \"charting long-range and local connectivity in the entire mouse brain\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "910\n",
      "23722212\n",
      "Imaging human connectomes at the macroscale.\n",
      "At macroscopic scales, the human connectome comprises anatomically distinct brain areas, the\n",
      "structural pathways connecting them and their functional interactions. Annotation of phenotypic\n",
      "associations with variation in the connectome and cataloging of neurophenotypes promise to transform\n",
      "our understanding of the human brain. In this Review, we provide a survey of magnetic resonance\n",
      "imaging–based measurements of functional and structural connectivity. We highlight emerging areas of\n",
      "development and inquiry and emphasize the importance of integrating structural and functional\n",
      "perspectives on brain architecture.\n",
      " {\n",
      "\"main_contribution\": \"annotation of phenotypic associations with variation in the connectome and cataloging of neurophenotypes\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work as imaging human connectomes at the macroscale. The abstract provides background information about the human connectome, its structural pathways, and functional interactions. It also mentions the importance of integrating structural and functional perspectives on brain architecture. However, it does not describe any new imaging techniques or methods. Therefore, the main contribution of the paper is the annotation of phenotypic associations with variation in the connectome and cataloging of neurophenotypes, which is not a new imaging method. Hence, is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "911\n",
      "23727986\n",
      "Enzymatic production of 'monoclonal stoichiometric' single-stranded DNA oligonucleotides.\n",
      "Single-stranded oligonucleotides are important as research tools, as diagnostic probes, in gene\n",
      "therapy and in DNA nanotechnology. Oligonucleotides are typically produced via solid-phase\n",
      "synthesis, using polymer chemistries that are limited relative to what biological systems produce.\n",
      "The number of errors in synthetic DNA increases with oligonucleotide length, and the resulting\n",
      "diversity of sequences can be a problem. Here we present the 'monoclonal stoichiometric' (MOSIC)\n",
      "method for enzyme-mediated production of DNA oligonucleotides. We amplified oligonucleotides from\n",
      "clonal templates derived from single bacterial colonies and then digested cutter hairpins in the\n",
      "products, which released pools of oligonucleotides with precisely controlled relative stoichiometric\n",
      "ratios. We prepared 14-378-nucleotide MOSIC oligonucleotides either by in vitro rolling-circle\n",
      "amplification or by amplification of phagemid DNA in Escherichia coli. Analyses of the formation of\n",
      "a DNA crystal and folding of DNA nanostructures confirmed the scalability, purity and stoichiometry\n",
      "of the produced oligonucleotides.\n",
      " {\n",
      "\"main_contribution\": \"Enzymatic production of monoclonal stoichiometric single-stranded DNA oligonucleotides\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for producing single-stranded DNA oligonucleotides using enzymatic synthesis. The method, called \"monoclonal stoichiometric\" (MOSIC), involves amplifying oligonucleotides from clonal templates derived from single bacterial colonies and then digesting cutter hairpins in the products to release pools of oligonucleotides with precisely controlled relative stoichiometric ratios. The paper describes the scalability, purity, and stoichiometry of the produced oligonucleotides, which are confirmed through analyses of DNA crystal formation and folding of DNA nanostructures.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new method for producing single-stranded DNA oligonucleotides using enzymatic synthesis, which is not an imaging technique or method. Hence, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "912\n",
      "23749299\n",
      "Hematopoietic progenitor cell lines with myeloid and lymphoid potential.\n",
      "Investigation of immune-cell differentiation and function is limited by shortcomings of suitable and\n",
      "scalable experimental systems. Here we show that retroviral delivery of an estrogen-regulated form\n",
      "of Hoxb8 into mouse bone marrow cells can be used along with Flt3 ligand to conditionally\n",
      "immortalize early hematopoietic progenitor cells (Hoxb8-FL cells). Hoxb8-FL cells have lost self-\n",
      "renewal capacity and potential to differentiate into megakaryocytes and erythrocytes but retain the\n",
      "potential to differentiate into myeloid and lymphoid cells. They differentiate in vitro and in vivo\n",
      "into macrophages, granulocytes, dendritic cells, B lymphocytes and T lymphocytes that are\n",
      "phenotypically and functionally indistinguishable from their primary counterparts. Quantitative in\n",
      "vitro assays indicate that myeloid and B-cell potential of Hoxb8-FL cells is comparable to that of\n",
      "primary lymphoid-primed multipotent progenitors, whereas T-cell potential is diminished. The\n",
      "simplicity of this system and the unlimited proliferative capacity of Hoxb8-FL cells will enable\n",
      "studies of immune-cell differentiation and function.\n",
      " {\n",
      "\"main_contribution\": \"Hematopoietic progenitor cell lines with myeloid and lymphoid potential\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the paper describe the development of a new experimental system for studying immune-cell differentiation and function. The main contribution of the paper is the establishment of a method for conditionally immortalizing early hematopoietic progenitor cells, which allows for the study of myeloid and lymphoid cell differentiation in vitro and in vivo. This system has the potential to overcome previous limitations in studying immune-cell differentiation and function, making it a significant contribution to the field.\n",
      "\n",
      "The sentences that describe the background information are:\n",
      "\n",
      "* \"Investigation of immune-cell differentiation and function is limited by shortcomings of suitable and scalable experimental systems.\"\n",
      "\n",
      "The sentences that describe the goals of the work are:\n",
      "\n",
      "* \"Here we show that retroviral delivery of an estrogen-regulated form of Hoxb8 into mouse bone marrow cells can be used along with Flt3 ligand to conditionally immortalize early hematopoietic progenitor cells (Hoxb8-FL cells).\"\n",
      "\n",
      "The sentences that describe the methods used are:\n",
      "\n",
      "* \"Hoxb8-FL cells have lost self-renewal capacity and potential to differentiate into megakaryocytes and erythrocytes but retain the potential to differentiate into myeloid and lymphoid cells.\"\n",
      "* \"They differentiate in vitro and in vivo into macrophages, granulocytes, dendritic cells, B lymphocytes and T lymphocytes that are phenotypically and functionally indistinguishable from their primary counterparts.\"\n",
      "\n",
      "The sentences that describe the results of the work are:\n",
      "\n",
      "* \"Quantitative in vitro assays indicate that myeloid and B-cell potential of Hoxb8-FL cells is comparable to that of primary lymphoid-primed multipotent progenitors, whereas T-cell potential is diminished.\"\n",
      "\n",
      "The sentences that describe the conclusions of the work are:\n",
      "\n",
      "* \"The simplicity of this system and the unlimited proliferative capacity of Hoxb8-FL cells will enable studies of immune-cell differentiation and function.\"\n",
      "\n",
      "Based on the above analysis, the main contribution of the paper is the development of a new experimental system for studying immune-cell differentiation and function. This system allows for the study of myeloid and lymphoid cell differentiation in vitro and in vivo, overcoming previous limitations in studying immune-cell differentiation and function. Therefore, the answer to the question is false, as the paper does not describe the development of a new imaging method or technique.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "913\n",
      "23770755\n",
      "Near-infrared fluorescent proteins for multicolor in vivo imaging.\n",
      "Near-infrared fluorescent proteins (FPs) are in high demand for in vivo imaging. We developed four\n",
      "spectrally distinct near-infrared FPs--iRFP670, iRFP682, iRFP702 and iRFP720--from bacterial\n",
      "phytochromes. iRFPs exhibit high brightness in mammalian cells and tissues and are suitable for\n",
      "long-term studies. iRFP670 and iRFP720 enable two-color imaging with standard approaches in living\n",
      "cells and mice. The four new iRFPs and the previously engineered iRFP713 allow multicolor imaging\n",
      "with spectral unmixing in living mice.\n",
      " {\n",
      "\"main_contribution\": \"Near-infrared fluorescent proteins for multicolor in vivo imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of near-infrared fluorescent proteins for in vivo imaging, which is an imaging method. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "914\n",
      "23817067\n",
      "Engineered nanostructured β-sheet peptides protect membrane proteins.\n",
      "We designed β-strand peptides that stabilize integral membrane proteins (IMPs). β-strand peptides\n",
      "self-assemble in solution as filaments and become restructured upon association with IMPs; resulting\n",
      "IMP-β-strand peptide complexes resisted aggregation when diluted in detergent-free buffer and were\n",
      "visible as stable, single particles with low detergent background in electron micrographs. β-strand\n",
      "peptides enabled clear visualization of flexible conformations in the highly dynamic ATP-binding\n",
      "cassette (ABC) transporter MsbA.\n",
      " {\n",
      "\"main_contribution\": \"Engineered nanostructured β-sheet peptides protect membrane proteins\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of engineered nanostructured β-sheet peptides that protect membrane proteins. The peptides were designed to stabilize integral membrane proteins (IMPs) and were found to resist aggregation and be visible as stable, single particles in electron micrographs. This contribution is related to imaging techniques, specifically electron microscopy, and thus the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "915\n",
      "23817068\n",
      "TALE-mediated modulation of transcriptional enhancers in vivo.\n",
      "We tested whether transcription activator-like effectors (TALEs) could mediate repression and\n",
      "activation of endogenous enhancers in the Drosophila genome. TALE repressors (TALERs) targeting each\n",
      "of the five even-skipped (eve) stripe enhancers generated repression specifically of the focal\n",
      "stripes. TALE activators (TALEAs) targeting the eve promoter or enhancers caused increased\n",
      "expression primarily in cells normally activated by the promoter or targeted enhancer, respectively.\n",
      "This effect supports the view that repression acts in a dominant fashion on transcriptional\n",
      "activators and that the activity state of an enhancer influences TALE binding or the ability of the\n",
      "VP16 domain to enhance transcription. In these assays, the Hairy repression domain did not exhibit\n",
      "previously described long-range transcriptional repression activity. The phenotypic effects of TALER\n",
      "and TALEA expression in larvae and adults are consistent with the observed modulations of eve\n",
      "expression. TALEs thus provide a novel tool for detection and functional modulation of\n",
      "transcriptional enhancers in their native genomic context.\n",
      " {\n",
      "\"main_contribution\": \"TALE-mediated modulation of transcriptional enhancers in vivo\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a novel tool for detection and functional modulation of transcriptional enhancers in their native genomic context using TALEs. This is described in the abstract as \"TALEs thus provide a novel tool for detection and functional modulation of transcriptional enhancers in their native genomic context.\"\n",
      "\n",
      "The method used to achieve this goal is the use of TALE repressors (TALERs) and activators (TALEAs) to target specific enhancers and promoters, which is described in the abstract as \"We tested whether transcription activator-like effectors (TALEs) could mediate repression and activation of endogenous enhancers in the Drosophila genome.\"\n",
      "\n",
      "The results of the work are that TALEs can indeed modulate transcriptional enhancers in vivo, as described in the abstract as \"This effect supports the view that repression acts in a dominant fashion on transcriptional activators and that the activity state of an enhancer influences TALE binding or the ability of the VP16 domain to enhance transcription.\"\n",
      "\n",
      "Therefore, the main contribution of the paper is not the development of a new imaging method, but rather the use of TALEs to modulate transcriptional enhancers in vivo. Therefore, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "916\n",
      "23817070\n",
      "Cell-selective labeling using amino acid precursors for proteomic studies of multicellular\n",
      "environments.\n",
      "We report a technique to selectively and continuously label the proteomes of individual cell types\n",
      "in coculture, named cell type-specific labeling using amino acid precursors (CTAP). Through\n",
      "transgenic expression of exogenous amino acid biosynthesis enzymes, vertebrate cells overcome their\n",
      "dependence on supplemented essential amino acids and can be selectively labeled through metabolic\n",
      "incorporation of amino acids produced from heavy isotope-labeled precursors. When testing CTAP in\n",
      "several human and mouse cell lines, we could differentially label the proteomes of distinct cell\n",
      "populations in coculture and determine the relative expression of proteins by quantitative mass\n",
      "spectrometry. In addition, using CTAP we identified the cell of origin of extracellular proteins\n",
      "secreted from cells in coculture. We believe that this method, which allows linking of proteins to\n",
      "their cell source, will be useful in studies of cell-cell communication and potentially for\n",
      "discovery of biomarkers.\n",
      " {\n",
      "\"main_contribution\": \"Cell-selective labeling using amino acid precursors for proteomic studies of multicellular environments\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new technique for selectively labeling proteomes of individual cell types in coculture, called cell type-specific labeling using amino acid precursors (CTAP). This technique allows for the differential labeling of proteins from distinct cell populations and enables the identification of the cell of origin of extracellular proteins. While this technique does not involve imaging methods, it has potential applications in studies of cell-cell communication and biomarker discovery. Therefore, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "917\n",
      "23817127\n",
      "Improved tools for the Brainbow toolbox.\n",
      "In the transgenic multicolor labeling strategy called 'Brainbow', Cre-loxP recombination is used to\n",
      "create a stochastic choice of expression among fluorescent proteins, resulting in the indelible\n",
      "marking of mouse neurons with multiple distinct colors. This method has been adapted to non-neuronal\n",
      "cells in mice and to neurons in fish and flies, but its full potential has yet to be realized in the\n",
      "mouse brain. Here we present several lines of mice that overcome limitations of the initial lines,\n",
      "and we report an adaptation of the method for use in adeno-associated viral vectors. We also provide\n",
      "technical advice about how best to image Brainbow-expressing tissue.\n",
      " {\n",
      "\"main_contribution\": \"Improved tools for the Brainbow toolbox\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new development in the Brainbow toolbox, which is a transgenic multicolor labeling strategy that uses Cre-loxP recombination to create a stochastic choice of expression among fluorescent proteins. The main contribution of the paper is improving the tools for this method, including adapting it for use in adeno-associated viral vectors and providing technical advice on imaging Brainbow-expressing tissue. Therefore, the main contribution is not the development of a new imaging method, but rather an adaptation and improvement of an existing technique. Hence, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "918\n",
      "23852451\n",
      "A database of Caenorhabditis elegans behavioral phenotypes.\n",
      "Using low-cost automated tracking microscopes, we have generated a behavioral database for 305\n",
      "Caenorhabditis elegans strains, including 76 mutants with no previously described phenotype. The\n",
      "growing database currently consists of 9,203 short videos segmented to extract behavior and\n",
      "morphology features, and these videos and feature data are available online for further analysis.\n",
      "The database also includes summary statistics for 702 measures with statistical comparisons to wild-\n",
      "type controls so that phenotypes can be identified and understood by users.\n",
      " {\n",
      "\"main_contribution\": \"A database of Caenorhabditis elegans behavioral phenotypes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a database of behavioral phenotypes for Caenorhabditis elegans strains, which includes 76 mutants with no previously described phenotype. The database consists of 9,203 short videos segmented to extract behavior and morphology features, and these videos and feature data are available online for further analysis. Therefore, the main contribution is the development of a new imaging method, specifically an automated tracking microscope, which enables the large-scale analysis of C. elegans behavioral phenotypes.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "919\n",
      "23852453\n",
      "Functional labeling of neurons and their projections using the synthetic activity-dependent promoter\n",
      "E-SARE.\n",
      "Identifying the neuronal ensembles that respond to specific stimuli and mapping their projection\n",
      "patterns in living animals are fundamental challenges in neuroscience. To this end, we engineered a\n",
      "synthetic promoter, the enhanced synaptic activity-responsive element (E-SARE), that drives neuronal\n",
      "activity-dependent gene expression more potently than other existing immediate-early gene promoters.\n",
      "Expression of a drug-inducible Cre recombinase downstream of E-SARE enabled imaging of neuronal\n",
      "populations that respond to monocular visual stimulation and tracking of their long-distance\n",
      "thalamocortical projections in living mice. Targeted cell-attached recordings and calcium imaging of\n",
      "neurons in sensory cortices revealed that E-SARE reporter expression correlates with sensory-evoked\n",
      "neuronal activity at the single-cell level and is highly specific to the type of stimuli presented\n",
      "to the animals. This activity-dependent promoter can expand the repertoire of genetic approaches for\n",
      "high-resolution anatomical and functional analysis of neural circuits.\n",
      " {\n",
      "\"main_contribution\": \"Engineering a synthetic promoter, the enhanced synaptic activity-responsive element (E-SARE), that drives neuronal activity-dependent gene expression more potently than other existing immediate-early gene promoters\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method, the enhanced synaptic activity-responsive element (E-SARE), which drives neuronal activity-dependent gene expression more potently than other existing immediate-early gene promoters. This allows for imaging of neuronal populations that respond to monocular visual stimulation and tracking of their long-distance thalamocortical projections in living mice. Therefore, the is_imaging_method variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "920\n",
      "23866336\n",
      "Improved tools for the Brainbow toolbox.\n",
      "In the transgenic multicolor labeling strategy called ‘Brainbow’, Cre-loxP recombination is used to\n",
      "create a stochastic choice of expression among fluorescent proteins, resulting in the indelible\n",
      "marking of mouse neurons with multiple distinct colors. This method has been adapted to non-neuronal\n",
      "cells in mice and to neurons in fish and flies, but its full potential has yet to be realized in the\n",
      "mouse brain. Here we present several lines of mice that overcome limitations of the initial lines,\n",
      "and we report an adaptation of the method for use in adeno-associated viral vectors. We also provide\n",
      "technical advice about how best to image Brainbow-expressing tissue.\n",
      " {\n",
      "\"main_contribution\": \"Improved tools for the Brainbow toolbox\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new development in the Brainbow toolbox, which is a transgenic multicolor labeling strategy that uses Cre-loxP recombination to create a stochastic choice of expression among fluorescent proteins. The main contribution of the paper is improving the tools for this method, including adapting it for use in adeno-associated viral vectors and providing technical advice on imaging Brainbow-expressing tissue. Therefore, the main contribution is not the development of a new imaging method, but rather an advancement of an existing technique. Hence, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "921\n",
      "23872791\n",
      "Imaging bacterial protein expression using genetically encoded RNA sensors.\n",
      "The difficulties in imaging the dynamics of protein expression in live bacterial cells can be\n",
      "overcome by using fluorescent sensors based on Spinach, an RNA that activates the fluorescence of a\n",
      "small-molecule fluorophore. These RNAs selectively bind target proteins and exhibit fluorescence\n",
      "increases that enable protein expression to be imaged in living Escherichia coli. These sensors are\n",
      "key components of a generalizable strategy to image protein expression in a single bacterium in real\n",
      "time.\n",
      " {\n",
      "\"main_contribution\": \"Imaging bacterial protein expression using genetically encoded RNA sensors\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that uses genetically encoded RNA sensors to image protein expression in live bacterial cells. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "922\n",
      "23892895\n",
      "RNA-guided gene activation by CRISPR-Cas9-based transcription factors.\n",
      "Technologies for engineering synthetic transcription factors have enabled many advances in medical\n",
      "and scientific research. In contrast to existing methods based on engineering of DNA-binding\n",
      "proteins, we created a Cas9-based transactivator that is targeted to DNA sequences by guide RNA\n",
      "molecules. Coexpression of this transactivator and combinations of guide RNAs in human cells induced\n",
      "specific expression of endogenous target genes, demonstrating a simple and versatile approach for\n",
      "RNA-guided gene activation.\n",
      " {\n",
      "\"main_contribution\": \"RNA-guided gene activation method using CRISPR-Cas9-based transcription factors\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the article describe a new method for RNA-guided gene activation using CRISPR-Cas9-based transcription factors. The main contribution of the paper is the development of this new method, which is described in sentences (B) and (D). Sentence (B) states that the authors created a Cas9-based transactivator targeted to DNA sequences by guide RNA molecules, while sentence (D) reports that coexpression of this transactivator and combinations of guide RNAs in human cells induced specific expression of endogenous target genes. Therefore, the main contribution of the paper is the development of a new imaging method, and is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "923\n",
      "23913258\n",
      "3D spectral imaging with synchrotron Fourier transform infrared spectro-microtomography.\n",
      "We report Fourier transform infrared spectro-microtomography, a nondestructive three-dimensional\n",
      "imaging approach that reveals the distribution of distinctive chemical compositions throughout an\n",
      "intact biological or materials sample. The method combines mid-infrared absorption contrast with\n",
      "computed tomographic data acquisition and reconstruction to enhance chemical and morphological\n",
      "localization by determining a complete infrared spectrum for every voxel (millions of spectra\n",
      "determined per sample).\n",
      " {\n",
      "\"main_contribution\": \"Fourier transform infrared spectro-microtomography\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called Fourier transform infrared spectro-microtomography, which combines mid-infrared absorption contrast with computed tomographic data acquisition and reconstruction to enhance chemical and morphological localization by determining a complete infrared spectrum for every voxel. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "924\n",
      "23913259\n",
      "Visualizing SNVs to quantify allele-specific expression in single cells.\n",
      "We present a FISH-based method for detecting single-nucleotide variants (SNVs) in exons and introns\n",
      "on individual RNA transcripts with high efficiency. We used this method to quantify allelic\n",
      "expression in cell populations and in single cells, and also to distinguish maternal from paternal\n",
      "chromosomes in single cells.\n",
      " {\n",
      "\"main_contribution\": \"a FISH-based method for detecting single-nucleotide variants (SNVs) in exons and introns on individual RNA transcripts\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on FISH (Fluorescence In Situ Hybridization) to detect single-nucleotide variants (SNVs) in individual RNA transcripts. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "925\n",
      "23934076\n",
      "Allele-specific detection of single mRNA molecules in situ.\n",
      "We describe a method for fluorescence in situ identification of individual mRNA molecules, allowing\n",
      "quantitative and accurate measurement, in single cells, of allele-specific transcripts that differ\n",
      "by only a few nucleotides. By using a combination of allele-specific and non-allele-specific probe\n",
      "libraries, we achieve >95% detection accuracy. We investigate the allele-specific stochastic\n",
      "expression of Nanog, which encodes a pluripotency factor, in murine embryonic stem cells.\n",
      " {\n",
      "\"main_contribution\": \"Allele-specific detection of single mRNA molecules in situ\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method for allele-specific detection of single mRNA molecules in situ, which is an imaging technique. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "926\n",
      "23934077\n",
      "STED nanoscopy combined with optical tweezers reveals protein dynamics on densely covered DNA.\n",
      "Dense coverage of DNA by proteins is a ubiquitous feature of cellular processes such as DNA\n",
      "organization, replication and repair. We present a single-molecule approach capable of visualizing\n",
      "individual DNA-binding proteins on densely covered DNA and in the presence of high protein\n",
      "concentrations. Our approach combines optical tweezers with multicolor confocal and stimulated\n",
      "emission depletion (STED) fluorescence microscopy. Proteins on DNA are visualized at a resolution of\n",
      "50 nm, a sixfold resolution improvement over that of confocal microscopy. High temporal resolution\n",
      "(<50 ms) is ensured by fast one-dimensional beam scanning. Individual trajectories of proteins\n",
      "translocating on DNA can thus be distinguished and tracked with high precision. We demonstrate our\n",
      "multimodal approach by visualizing the assembly of dense nucleoprotein filaments with unprecedented\n",
      "spatial resolution in real time. Experimental access to the force-dependent kinetics and motility of\n",
      "DNA-associating proteins at biologically relevant protein densities is essential for linking\n",
      "idealized in vitro experiments with the in vivo situation.\n",
      " {\n",
      "\"main_contribution\": \"Visualizing individual DNA-binding proteins on densely covered DNA and in the presence of high protein concentrations using a combination of optical tweezers, multicolor confocal microscopy, and STED fluorescence microscopy.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method that combines optical tweezers with multicolor confocal and STED fluorescence microscopy to visualize individual DNA-binding proteins on densely covered DNA. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "927\n",
      "23985731\n",
      "Multiparametric imaging of biological systems by force-distance curve-based AFM.\n",
      "A current challenge in the life sciences is to understand how biological systems change their\n",
      "structural, biophysical and chemical properties to adjust functionality. Addressing this issue has\n",
      "been severely hampered by the lack of methods capable of imaging biosystems at high resolution while\n",
      "simultaneously mapping their multiple properties. Recent developments in force-distance (FD) curve-\n",
      "based atomic force microscopy (AFM) now enable researchers to combine (sub)molecular imaging with\n",
      "quantitative mapping of physical, chemical and biological interactions. Here we discuss the\n",
      "principles and applications of advanced FD-based AFM tools for the quantitative multiparametric\n",
      "characterization of complex cellular and biomolecular systems under physiological conditions.\n",
      " {\n",
      "\"main_contribution\": \"Quantitative multiparametric characterization of complex cellular and biomolecular systems under physiological conditions using advanced force-distance curve-based AFM tools\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "928\n",
      "23995389\n",
      "Engineering the Caenorhabditis elegans genome using Cas9-triggered homologous recombination.\n",
      "Study of the nematode Caenorhabditis elegans has provided important insights in a wide range of\n",
      "fields in biology. The ability to precisely modify genomes is critical to fully realize the utility\n",
      "of model organisms. Here we report a method to edit the C. elegans genome using the clustered,\n",
      "regularly interspersed, short palindromic repeats (CRISPR) RNA-guided Cas9 nuclease and homologous\n",
      "recombination. We demonstrate that Cas9 is able to induce DNA double-strand breaks with specificity\n",
      "for targeted sites and that these breaks can be repaired efficiently by homologous recombination. By\n",
      "supplying engineered homologous repair templates, we generated gfp knock-ins and targeted mutations.\n",
      "Together our results outline a flexible methodology to produce essentially any desired modification\n",
      "in the C. elegans genome quickly and at low cost. This technology is an important addition to the\n",
      "array of genetic techniques already available in this experimentally tractable model organism.\n",
      " {\n",
      "\"main_contribution\": \"a method to edit the C. elegans genome using the CRISPR RNA-guided Cas9 nuclease and homologous recombination\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new genetic method for editing the genome of the nematode Caenorhabditis elegans using CRISPR-Cas9 technology. This method allows for precise and efficient modification of the genome, enabling researchers to generate targeted mutations and knock-ins quickly and at low cost. While this technique does not involve imaging or microscopy, it is an important tool for studying the biology of C. elegans, a model organism that has provided valuable insights in many fields of biology. Therefore, the main contribution of this paper is not an imaging method, and the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "929\n",
      "24013819\n",
      "Solid-state NMR spectroscopy structure determination of a lipid-embedded heptahelical membrane\n",
      "protein.\n",
      "Determination of structure of integral membrane proteins, especially in their native environment, is\n",
      "a formidable challenge in structural biology. Here we demonstrate that magic angle spinning solid-\n",
      "state NMR spectroscopy can be used to determine structures of membrane proteins reconstituted in\n",
      "synthetic lipids, an environment similar to the natural membrane. We combined a large number of\n",
      "experimentally determined interatomic distances and local torsional restraints to solve the\n",
      "structure of an oligomeric membrane protein of common seven-helical fold, Anabaena sensory rhodopsin\n",
      "(ASR). We determined the atomic resolution detail of the oligomerization interface of the ASR\n",
      "trimer, and the arrangement of helices, side chains and the retinal cofactor in the monomer.\n",
      " {\n",
      "\"main_contribution\": \"Determination of structure of integral membrane proteins in their native environment\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the use of magic angle spinning solid-state NMR spectroscopy to determine the structures of membrane proteins reconstituted in synthetic lipids. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "930\n",
      "24013820\n",
      "Brain-wide 3D imaging of neuronal activity in Caenorhabditis elegans with sculpted light.\n",
      "Recent efforts in neuroscience research have been aimed at obtaining detailed anatomical neuronal\n",
      "wiring maps as well as information on how neurons in these networks engage in dynamic activities.\n",
      "Although the entire connectivity map of the nervous system of Caenorhabditis elegans has been known\n",
      "for more than 25 years, this knowledge has not been sufficient to predict all functional connections\n",
      "underlying behavior. To approach this goal, we developed a two-photon technique for brain-wide\n",
      "calcium imaging in C. elegans, using wide-field temporal focusing (WF-TeFo). Pivotal to our results\n",
      "was the use of a nuclear-localized, genetically encoded calcium indicator, NLS-GCaMP5K, that permits\n",
      "unambiguous discrimination of individual neurons within the densely packed head ganglia of C.\n",
      "elegans. We demonstrate near-simultaneous recording of activity of up to 70% of all head neurons. In\n",
      "combination with a lab-on-a-chip device for stimulus delivery, this method provides an enabling\n",
      "platform for establishing functional maps of neuronal networks.\n",
      " {\n",
      "\"main_contribution\": \"Development of a two-photon technique for brain-wide calcium imaging in C. elegans using wide-field temporal focusing (WF-TeFo).\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a two-photon technique for brain-wide calcium imaging in C. elegans using wide-field temporal focusing (WF-TeFo). Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "931\n",
      "24056873\n",
      "Genetically encoded calcium indicator illuminates calcium dynamics in primary cilia.\n",
      "Visualization of signal transduction in live primary cilia constitutes a technical challenge owing\n",
      "to the organelle's submicrometer dimensions and close proximity to the cell body. Using a\n",
      "genetically encoded calcium indicator targeted to primary cilia, we visualized calcium signaling in\n",
      "cilia of mouse fibroblasts and kidney cells upon chemical or mechanical stimulation with high\n",
      "specificity, high sensitivity and wide dynamic range.\n",
      " {\n",
      "\"main_contribution\": \"Visualized calcium signaling in primary cilia with a genetically encoded calcium indicator\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a genetically encoded calcium indicator that allows for the visualization of calcium signaling in primary cilia. This is an imaging method, so \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "932\n",
      "24056874\n",
      "Inducible control of gene expression with destabilized Cre.\n",
      "Acute manipulation of gene and protein function in the brain is essential for understanding the\n",
      "mechanisms of nervous system development, plasticity and information processing. Here we describe a\n",
      "technique based on a destabilized Cre recombinase (DD-Cre) whose activity is controlled by the\n",
      "antibiotic trimethoprim (TMP). We show that DD-Cre triggers rapid TMP-dependent recombination of\n",
      "loxP-flanked ('floxed') alleles in mouse neurons in vivo and validate the use of this system for\n",
      "neurobehavioral research.\n",
      " {\n",
      "\"main_contribution\": \"inducible control of gene expression with destabilized Cre\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new technique for manipulating gene expression in the brain using a destabilized form of the Cre recombinase enzyme, which can be activated or inactivated using the antibiotic trimethoprim (TMP). The main contribution of the paper is the development of this technique, which allows for rapid and controlled manipulation of gene expression in neurons in vivo. This technique has potential applications in neurobehavioral research.\n",
      "\n",
      "Since the main contribution of the paper is not the development of a new imaging method or technique, but rather a new genetic manipulation technique, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "933\n",
      "24097269\n",
      "Image-based transcriptomics in thousands of single human cells at single-molecule resolution.\n",
      "Fluorescence in situ hybridization (FISH) is widely used to obtain information about transcript copy\n",
      "number and subcellular localization in single cells. However, current approaches do not readily\n",
      "scale to the analysis of whole transcriptomes. Here we show that branched DNA technology combined\n",
      "with automated liquid handling, high-content imaging and quantitative image analysis allows highly\n",
      "reproducible quantification of transcript abundance in thousands of single cells at single-molecule\n",
      "resolution. In addition, it allows extraction of a multivariate feature set quantifying subcellular\n",
      "patterning and spatial properties of transcripts and their cell-to-cell variability. This has\n",
      "multiple implications for the functional interpretation of cell-to-cell variability in gene\n",
      "expression and enables the unbiased identification of functionally relevant in situ signatures of\n",
      "the transcriptome without the need for perturbations. Because this method can be incorporated in a\n",
      "wide variety of high-throughput image-based approaches, we expect it to be broadly applicable.\n",
      " {\n",
      "\"main_contribution\": \"Highly reproducible quantification of transcript abundance in thousands of single cells at single-molecule resolution using branched DNA technology combined with automated liquid handling, high-content imaging and quantitative image analysis.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows for highly reproducible quantification of transcript abundance in thousands of single cells at single-molecule resolution. This method combines branched DNA technology with automated liquid handling, high-content imaging, and quantitative image analysis. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "934\n",
      "24097271\n",
      "Instant super-resolution imaging in live cells and embryos via analog image processing.\n",
      "Existing super-resolution fluorescence microscopes compromise acquisition speed to provide\n",
      "subdiffractive sample information. We report an analog implementation of structured illumination\n",
      "microscopy that enables three-dimensional (3D) super-resolution imaging with a lateral resolution of\n",
      "145 nm and an axial resolution of 350 nm at acquisition speeds up to 100 Hz. By using optical\n",
      "instead of digital image-processing operations, we removed the need to capture, store and combine\n",
      "multiple camera exposures, increasing data acquisition rates 10- to 100-fold over other super-\n",
      "resolution microscopes and acquiring and displaying super-resolution images in real time. Low\n",
      "excitation intensities allow imaging over hundreds of 2D sections, and combined physical and\n",
      "computational sectioning allow similar depth penetration to spinning-disk confocal microscopy. We\n",
      "demonstrate the capability of our system by imaging fine, rapidly moving structures including motor-\n",
      "driven organelles in human lung fibroblasts and the cytoskeleton of flowing blood cells within\n",
      "developing zebrafish embryos.\n",
      " {\n",
      "\"main_contribution\": \"Instant super-resolution imaging in live cells and embryos via analog image processing\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on analog image processing, which allows for real-time super-resolution imaging of live cells and embryos with high acquisition speeds. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "935\n",
      "24122039\n",
      "Formation of targeted monovalent quantum dots by steric exclusion.\n",
      "Precise control over interfacial chemistry between nanoparticles and other materials remains a major\n",
      "challenge that limits broad application of nanotechnology in biology. To address this challenge, we\n",
      "used 'steric exclusion' to completely convert commercial quantum dots (QDs) into monovalent imaging\n",
      "probes by wrapping each QD with a functionalized oligonucleotide. We demonstrated the utility of\n",
      "these QDs as modular and nonperturbing imaging probes by tracking individual Notch receptors on live\n",
      "cells.\n",
      " {\n",
      "\"main_contribution\": \"monovalent quantum dots\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of monovalent quantum dots, which can be used as imaging probes for tracking individual Notch receptors on live cells. Since the paper describes the development of a new imaging method (wrapping commercial quantum dots with functionalized oligonucleotides), the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "936\n",
      "24122040\n",
      "Detergent-free mass spectrometry of membrane protein complexes.\n",
      "We developed a method that allows release of intact membrane protein complexes from amphipols,\n",
      "bicelles and nanodiscs in the gas phase for observation by mass spectrometry (MS). Current methods\n",
      "involve release of membrane protein complexes from detergent micelles, which reveals subunit\n",
      "composition and lipid binding. We demonstrated that oligomeric complexes or proteins requiring\n",
      "defined lipid environments are stabilized to a greater extent in the absence of detergent.\n",
      " {\n",
      "\"main_contribution\": \"A method for releasing intact membrane protein complexes from amphipols, bicelles, and nanodiscs in the gas phase for observation by mass spectrometry\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the paper describe a new method for analyzing membrane protein complexes using mass spectrometry. The method developed in this work allows for the release of intact membrane protein complexes from amphipols, bicelles, and nanodiscs in the gas phase, which is an improvement over current methods that involve releasing membrane protein complexes from detergent micelles. This new method provides better stability for oligomeric complexes or proteins requiring defined lipid environments. Therefore, the main contribution of the paper is the development of a new technique for analyzing membrane protein complexes using mass spectrometry, which is not an imaging method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "937\n",
      "24141495\n",
      "High-throughput fingerprinting of human pluripotent stem cell fate responses and lineage bias.\n",
      "Populations of cells create local environments that lead to emergent heterogeneity. This is\n",
      "particularly evident with human pluripotent stem cells (hPSCs): microenvironmental heterogeneity\n",
      "limits hPSC cell fate control. We developed a high-throughput platform to screen hPSCs in\n",
      "configurable microenvironments in which we optimized colony size, cell density and other parameters\n",
      "to achieve rapid and robust cell fate responses to exogenous cues. We used this platform to perform\n",
      "single-cell protein expression profiling, revealing that Oct4 and Sox2 costaining discriminates\n",
      "pluripotent, neuroectoderm, primitive streak and extraembryonic cell fates. We applied this\n",
      "Oct4-Sox2 code to analyze dose responses of 27 developmental factors to obtain lineage-specific\n",
      "concentration optima and to quantify cell line-specific endogenous signaling pathway activation and\n",
      "differentiation bias. We demonstrated that short-term responses predict definitive endoderm\n",
      "induction efficiency and can be used to rescue differentiation of cell lines reticent to cardiac\n",
      "induction. This platform will facilitate high-throughput hPSC-based screening and quantification of\n",
      "lineage-induction bias.\n",
      " {\n",
      "\"main_contribution\": \"High-throughput fingerprinting of human pluripotent stem cell fate responses and lineage bias.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a high-throughput platform for screening human pluripotent stem cells in configurable microenvironments, which allows for rapid and robust cell fate responses to exogenous cues. This platform was used for single-cell protein expression profiling, revealing a code for discriminating between different cell fates. The authors also demonstrated that short-term responses can predict definitive endoderm induction efficiency and can be used to rescue differentiation of cell lines reticent to cardiac induction. Therefore, the main contribution is the development of an imaging method, specifically a high-throughput platform for screening human pluripotent stem cells.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "938\n",
      "24162923\n",
      "A superfolding Spinach2 reveals the dynamic nature of trinucleotide repeat-containing RNA.\n",
      "Imaging RNA in living cells is a challenging problem in cell biology. One strategy for genetically\n",
      "encoding fluorescent RNAs is to express them as fusions with Spinach, an 'RNA mimic of GFP'. We\n",
      "found that Spinach was dimmer than expected when used to tag constructs in living cells owing to a\n",
      "combination of thermal instability and a propensity for misfolding. Using systematic mutagenesis, we\n",
      "generated Spinach2 that overcomes these issues and can be used to image diverse RNAs. Using\n",
      "Spinach2, we detailed the dynamics of the CGG trinucleotide repeat-containing 'toxic RNA' associated\n",
      "with Fragile X-associated tremor/ataxia syndrome, and show that these RNAs form nuclear foci with\n",
      "unexpected morphological plasticity that is regulated by the cell cycle and by small molecules.\n",
      "Together, these data demonstrate that Spinach2 exhibits improved versatility for fluorescently\n",
      "labeling RNAs in living cells.\n",
      " {\n",
      "\"main_contribution\": \"Spinach2, a genetically encoded fluorescent RNA imaging technique\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of Spinach2, a new imaging method for visualizing RNAs in living cells. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "939\n",
      "24213166\n",
      "Quantifying the local resolution of cryo-EM density maps.\n",
      "We propose a definition of local resolution for three-dimensional electron cryo-microscopy (cryo-EM)\n",
      "density maps that uses local sinusoidal features. Our algorithm has no free parameters and is\n",
      "applicable to other imaging modalities, including tomography. By evaluating the local resolution of\n",
      "single-particle reconstructions and subtomogram averages for four example data sets, we report\n",
      "variable resolution across a 4- to 40-Å range.\n",
      " {\n",
      "\"main_contribution\": \"a definition of local resolution for three-dimensional electron cryo-microscopy (cryo-EM) density maps\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new method for quantifying the local resolution of cryo-EM density maps, which is an imaging technique. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "940\n",
      "24213167\n",
      "Parallel measurement of dynamic changes in translation rates in single cells.\n",
      "Protein concentrations are often regulated by dynamic changes in translation rates. Nevertheless, it\n",
      "has been challenging to directly monitor changes in translation in living cells. We have developed a\n",
      "reporter system to measure real-time changes of translation rates in human or mouse individual cells\n",
      "by conjugating translation regulatory motifs to sequences encoding a nuclear targeted fluorescent\n",
      "protein and a controllable destabilization domain. Application of the method showed that individual\n",
      "cells undergo marked fluctuations in the translation rate of mRNAs whose 5' terminal oligopyrimidine\n",
      "(5' TOP) motif regulates the synthesis of ribosomal proteins. Furthermore, we show that small\n",
      "reductions in amino acid levels signal through different mTOR-dependent pathways to control TOP mRNA\n",
      "translation, whereas larger reductions in amino acid levels control translation through eIF2A. Our\n",
      "study demonstrates that dynamic measurements of single-cell activities of translation regulatory\n",
      "motifs can be used to identify and investigate fundamental principles of translation.\n",
      " {\n",
      "\"main_contribution\": \"Measurement of dynamic changes in translation rates in single cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "941\n",
      "24240319\n",
      "Integrating protein-protein interaction networks with phenotypes reveals signs of interactions.\n",
      "A major objective of systems biology is to organize molecular interactions as networks and to\n",
      "characterize information flow within networks. We describe a computational framework to integrate\n",
      "protein-protein interaction (PPI) networks and genetic screens to predict the 'signs' of\n",
      "interactions (i.e., activation-inhibition relationships). We constructed a Drosophila melanogaster\n",
      "signed PPI network consisting of 6,125 signed PPIs connecting 3,352 proteins that can be used to\n",
      "identify positive and negative regulators of signaling pathways and protein complexes. We identified\n",
      "an unexpected role for the metabolic enzymes enolase and aldo-keto reductase as positive and\n",
      "negative regulators of proteolysis, respectively. Characterization of the activation-inhibition\n",
      "relationships between physically interacting proteins within signaling pathways will affect our\n",
      "understanding of many biological functions, including signal transduction and mechanisms of disease.\n",
      " {\n",
      "\"main_contribution\": \"Integrating protein-protein interaction networks with phenotypes reveals signs of interactions.\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a computational framework for integrating protein-protein interaction (PPI) networks and genetic screens to predict the signs of interactions. The main contribution of the paper is developing a computational method for understanding protein interactions, which is not an imaging technique or method. Therefore, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "942\n",
      "24240320\n",
      "Deciphering laminar-specific neural inputs with line-scanning fMRI.\n",
      "Using a line-scanning method during functional magnetic resonance imaging (fMRI), we obtained high\n",
      "temporal (50-ms) and spatial (50-μm) resolution information along the cortical thickness and showed\n",
      "that the laminar position of fMRI onset coincides with distinct neural inputs in rat somatosensory\n",
      "and motor cortices. This laminar-specific fMRI onset allowed us to identify the neural inputs\n",
      "underlying ipsilateral fMRI activation in the barrel cortex due to peripheral denervation-induced\n",
      "plasticity.\n",
      " {\n",
      "\"main_contribution\": \"identify the neural inputs underlying ipsilateral fMRI activation in the barrel cortex\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is identifying the neural inputs underlying ipsilateral fMRI activation in the barrel cortex, which is a development of a new imaging method, specifically a line-scanning method during functional magnetic resonance imaging (fMRI).\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "943\n",
      "24292484\n",
      "Niche-independent high-purity cultures of Lgr5+ intestinal stem cells and their progeny.\n",
      "Although Lgr5(+) intestinal stem cells have been expanded in vitro as organoids, homogeneous culture\n",
      "of these cells has not been possible thus far. Here we show that two small molecules, CHIR99021 and\n",
      "valproic acid, synergistically maintain self-renewal of mouse Lgr5(+) intestinal stem cells,\n",
      "resulting in nearly homogeneous cultures. The colony-forming efficiency of cells from these cultures\n",
      "is ~100-fold greater than that of cells cultured in the absence of CHIR99021 and valproic acid, and\n",
      "multilineage differentiation ability is preserved. We made use of these homogeneous cultures to\n",
      "identify conditions employing simultaneous modulation of Wnt and Notch signaling to direct lineage\n",
      "differentiation into mature enterocytes, goblet cells and Paneth cells. Expansion in these culture\n",
      "conditions may be feasible for Lgr5(+) cells from the mouse stomach and colon and from the human\n",
      "small intestine. These methods provide new tools for the study and application of multiple\n",
      "intestinal epithelial cell types.\n",
      " {\n",
      "\"main_contribution\": \"Niche-independent high-purity cultures of Lgr5+ intestinal stem cells and their progeny\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new method for culturing Lgr5+ intestinal stem cells and their progeny in a niche-independent manner, which allows for the expansion of these cells in vitro. This method utilizes two small molecules, CHIR99021 and valproic acid, to synergistically maintain the self-renewal of mouse Lgr5+ intestinal stem cells, resulting in nearly homogeneous cultures. The paper also describes the use of these homogeneous cultures to identify conditions for directing lineage differentiation into mature enterocytes, goblet cells, and Paneth cells.\n",
      "\n",
      "Therefore, the main contribution of the paper is not the development of a new imaging method or technique, but rather the development of a new method for culturing intestinal stem cells and their progeny. Therefore, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "944\n",
      "24292485\n",
      "A chemoproteomic platform to quantitatively map targets of lipid-derived electrophiles.\n",
      "Cells produce electrophilic products with the potential to modify and affect the function of\n",
      "proteins. Chemoproteomic methods have provided a means to qualitatively inventory proteins targeted\n",
      "by endogenous electrophiles; however, ascertaining the potency and specificity of these reactions to\n",
      "identify the sites in the proteome that are most sensitive to electrophilic modification requires\n",
      "more quantitative methods. Here we describe a competitive activity-based profiling method for\n",
      "quantifying the reactivity of electrophilic compounds against >1,000 cysteines in parallel in the\n",
      "human proteome. Using this approach, we identified a select set of proteins that constitute 'hot\n",
      "spots' for modification by various lipid-derived electrophiles, including the oxidative stress\n",
      "product 4-hydroxy-2-nonenal (HNE). We show that one of these proteins, ZAK kinase, is labeled by HNE\n",
      "on a conserved, active site-proximal cysteine and that the resulting enzyme inhibition creates a\n",
      "negative feedback mechanism that can suppress the activation of JNK pathways normally induced by\n",
      "oxidative stress.\n",
      " {\n",
      "\"main_contribution\": \"A chemoproteomic platform to quantitatively map targets of lipid-derived electrophiles\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a chemoproteomic platform for quantifying the reactivity of electrophilic compounds against over 1,000 cysteines in parallel in the human proteome. This is not a new imaging method or technique, so \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "945\n",
      "24317254\n",
      "Quantifying cell-generated mechanical forces within living embryonic tissues.\n",
      "Cell-generated mechanical forces play a critical role during tissue morphogenesis and organ\n",
      "formation in the embryo. Little is known about how these forces shape embryonic organs, mainly\n",
      "because it has not been possible to measure cellular forces within developing three-dimensional (3D)\n",
      "tissues in vivo. We present a method to quantify cell-generated mechanical stresses exerted locally\n",
      "within living embryonic tissues, using fluorescent, cell-sized oil microdroplets with defined\n",
      "mechanical properties and coated with adhesion receptor ligands. After a droplet is introduced\n",
      "between cells in a tissue, local stresses are determined from droplet shape deformations, measured\n",
      "using fluorescence microscopy and computerized image analysis. Using this method, we quantified the\n",
      "anisotropic stresses generated by mammary epithelial cells cultured within 3D aggregates, and we\n",
      "confirmed that these stresses (3.4 nN μm(-2)) are dependent on myosin II activity and are more than\n",
      "twofold larger than stresses generated by cells of embryonic tooth mesenchyme, either within\n",
      "cultured aggregates or in developing whole mouse mandibles.\n",
      " {\n",
      "\"main_contribution\": \"Quantifying cell-generated mechanical forces within living embryonic tissues\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method to quantify cell-generated mechanical stresses within living embryonic tissues using fluorescent, cell-sized oil microdroplets with defined mechanical properties and coated with adhesion receptor ligands. This method allows for the measurement of local stresses in vivo, which was previously not possible. Therefore, the main contribution is the development of a new imaging technique or method, and the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "946\n",
      "24363022\n",
      "Optogenetic control of Drosophila using a red-shifted channelrhodopsin reveals experience-dependent\n",
      "influences on courtship.\n",
      "Optogenetics allows the manipulation of neural activity in freely moving animals with millisecond\n",
      "precision, but its application in Drosophila melanogaster has been limited. Here we show that a\n",
      "recently described red activatable channelrhodopsin (ReaChR) permits control of complex behavior in\n",
      "freely moving adult flies, at wavelengths that are not thought to interfere with normal visual\n",
      "function. This tool affords the opportunity to control neural activity over a broad dynamic range of\n",
      "stimulation intensities. Using time-resolved activation, we show that the neural control of male\n",
      "courtship song can be separated into (i) probabilistic, persistent and (ii) deterministic, command-\n",
      "like components. The former, but not the latter, neurons are subject to functional modulation by\n",
      "social experience, which supports the idea that they constitute a locus of state-dependent\n",
      "influence. This separation is not evident using thermogenetic tools, a result underscoring the\n",
      "importance of temporally precise control of neuronal activation in the functional dissection of\n",
      "neural circuits in Drosophila.\n",
      " {\n",
      "\"main_contribution\": \"A red-shifted channelrhodopsin for optogenetic control of Drosophila\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, specifically a red-shifted channelrhodopsin for optogenetic control of Drosophila. This is described in the abstract as a tool that allows for the manipulation of neural activity in freely moving animals with millisecond precision, and is highlighted as a novel contribution in the field of optogenetics. Therefore, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "947\n",
      "24390439\n",
      "Single-molecule evaluation of fluorescent protein photoactivation efficiency using an in vivo\n",
      "nanotemplate.\n",
      "Photoswitchable fluorescent probes are central to localization-based super-resolution microscopy.\n",
      "Among these probes, fluorescent proteins are appealing because they are genetically encoded.\n",
      "Moreover, the ability to achieve a 1:1 labeling ratio between the fluorescent protein and the\n",
      "protein of interest makes these probes attractive for quantitative single-molecule counting. The\n",
      "percentage of fluorescent protein that is photoactivated into a fluorescently detectable form (i.e.,\n",
      "the photoactivation efficiency) plays a crucial part in properly interpreting the quantitative\n",
      "information. It is important to characterize the photoactivation efficiency at the single-molecule\n",
      "level under the conditions used in super-resolution imaging. Here, we used the human glycine\n",
      "receptor expressed in Xenopus oocytes and stepwise photobleaching or single-molecule counting\n",
      "photoactivated localization microcopy (PALM) to determine the photoactivation efficiency of\n",
      "fluorescent proteins mEos2, mEos3.1, mEos3.2, Dendra2, mClavGR2, mMaple, PA-GFP and PA-mCherry. This\n",
      "analysis provides important information that must be considered when using these fluorescent\n",
      "proteins in quantitative super-resolution microscopy.\n",
      " {\n",
      "\"main_contribution\": \"The photoactivation efficiency of fluorescent proteins mEos2, mEos3.1, mEos3.2, Dendra2, mClavGR2, mMaple, PA-GFP and PA-mCherry was determined using stepwise photobleaching or single-molecule counting photoactivated localization microscopy (PALM).\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the determination of the photoactivation efficiency of several fluorescent proteins using a novel imaging technique called single-molecule counting photoactivated localization microscopy (PALM). Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "948\n",
      "24390440\n",
      "Optimized ratiometric calcium sensors for functional in vivo imaging of neurons and T lymphocytes.\n",
      "The quality of genetically encoded calcium indicators (GECIs) has improved dramatically in recent\n",
      "years, but high-performing ratiometric indicators are still rare. Here we describe a series of\n",
      "fluorescence resonance energy transfer (FRET)-based calcium biosensors with a reduced number of\n",
      "calcium binding sites per sensor. These 'Twitch' sensors are based on the C-terminal domain of\n",
      "Opsanus troponin C. Their FRET responses were optimized by a large-scale functional screen in\n",
      "bacterial colonies, refined by a secondary screen in rat hippocampal neuron cultures. We tested the\n",
      "in vivo performance of the most sensitive variants in the brain and lymph nodes of mice. The\n",
      "sensitivity of the Twitch sensors matched that of synthetic calcium dyes and allowed visualization\n",
      "of tonic action potential firing in neurons and high resolution functional tracking of T\n",
      "lymphocytes. Given their ratiometric readout, their brightness, large dynamic range and linear\n",
      "response properties, Twitch sensors represent versatile tools for neuroscience and immunology.\n",
      " {\n",
      "\"main_contribution\": \"Twitch sensors\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title of the paper describes the development of \"ratiometric calcium sensors\" for in vivo imaging. The abstract further explains that these sensors are genetically encoded calcium indicators (GECIs) with a reduced number of calcium binding sites per sensor, based on the C-terminal domain of Opsanus troponin C. The authors optimized the FRET responses of these sensors through large-scale functional screens in bacterial colonies and rat hippocampal neuron cultures. They tested the in vivo performance of the most sensitive variants in mice, which showed a sensitivity matching that of synthetic calcium dyes and allowed visualization of tonic action potential firing in neurons and high-resolution functional tracking of T lymphocytes.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of Twitch sensors, which are ratiometric calcium sensors with improved sensitivity and brightness for in vivo imaging. Since the paper describes the development of a new imaging technique (the Twitch sensors), the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "949\n",
      "24412976\n",
      "Transcriptome in vivo analysis (TIVA) of spatially defined single cells in live tissue.\n",
      "Transcriptome profiling of single cells resident in their natural microenvironment depends upon RNA\n",
      "capture methods that are both noninvasive and spatially precise. We engineered a transcriptome in\n",
      "vivo analysis (TIVA) tag, which upon photoactivation enables mRNA capture from single cells in live\n",
      "tissue. Using the TIVA tag in combination with RNA sequencing (RNA-seq), we analyzed transcriptome\n",
      "variance among single neurons in culture and in mouse and human tissue in vivo. Our data showed that\n",
      "the tissue microenvironment shapes the transcriptomic landscape of individual cells. The TIVA\n",
      "methodology is, to our knowledge, the first noninvasive approach for capturing mRNA from live single\n",
      "cells in their natural microenvironment.\n",
      " {\n",
      "\"main_contribution\": \"The development of a noninvasive approach for capturing mRNA from live single cells in their natural microenvironment called Transcriptome in vivo analysis (TIVA).\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method called Transcriptome in vivo analysis (TIVA) for capturing mRNA from single cells in their natural microenvironment. This method is noninvasive, spatially precise, and enables transcriptome profiling of single cells resident in live tissue. Therefore, the main contribution of the paper is the development of this new imaging technique or method, which is identified as true for the variable is_imaging_method. The main contribution is summarized in a concise noun phrase as \"The development of a noninvasive approach for capturing mRNA from live single cells in their natural microenvironment called Transcriptome in vivo analysis (TIVA).\"\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "950\n",
      "24412977\n",
      "Scalable inference of heterogeneous reaction kinetics from pooled single-cell recordings.\n",
      "Mathematical methods combined with measurements of single-cell dynamics provide a means to\n",
      "reconstruct intracellular processes that are only partly or indirectly accessible experimentally. To\n",
      "obtain reliable reconstructions, the pooling of measurements from several cells of a clonal\n",
      "population is mandatory. However, cell-to-cell variability originating from diverse sources poses\n",
      "computational challenges for such process reconstruction. We introduce a scalable Bayesian inference\n",
      "framework that properly accounts for population heterogeneity. The method allows inference of\n",
      "inaccessible molecular states and kinetic parameters; computation of Bayes factors for model\n",
      "selection; and dissection of intrinsic, extrinsic and technical noise. We show how additional\n",
      "single-cell readouts such as morphological features can be included in the analysis. We use the\n",
      "method to reconstruct the expression dynamics of a gene under an inducible promoter in yeast from\n",
      "time-lapse microscopy data.\n",
      " {\n",
      "\"main_contribution\": \"Scalable inference of heterogeneous reaction kinetics from pooled single-cell recordings\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a scalable Bayesian inference framework for reconstructing intracellular processes from pooled single-cell recordings, which is an imaging method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "951\n",
      "24441936\n",
      "Objective comparison of particle tracking methods.\n",
      "Particle tracking is of key importance for quantitative analysis of intracellular dynamic processes\n",
      "from time-lapse microscopy image data. Because manually detecting and following large numbers of\n",
      "individual particles is not feasible, automated computational methods have been developed for these\n",
      "tasks by many groups. Aiming to perform an objective comparison of methods, we gathered the\n",
      "community and organized an open competition in which participating teams applied their own methods\n",
      "independently to a commonly defined data set including diverse scenarios. Performance was assessed\n",
      "using commonly defined measures. Although no single method performed best across all scenarios, the\n",
      "results revealed clear differences between the various approaches, leading to notable practical\n",
      "conclusions for users and developers.\n",
      " {\n",
      "\"main_contribution\": \"Objective comparison of particle tracking methods\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically an automated computational method for particle tracking in time-lapse microscopy image data. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "952\n",
      "24464288\n",
      "Correlative super-resolution fluorescence and metal-replica transmission electron microscopy.\n",
      "We combine super-resolution localization fluorescence microscopy with transmission electron\n",
      "microscopy of metal replicas to locate proteins on the landscape of the cellular plasma membrane at\n",
      "the nanoscale. We validate robust correlation on the scale of 20 nm by imaging endogenous clathrin\n",
      "(in two and three dimensions) and apply the method to find the previously unknown three-dimensional\n",
      "position of the endocytic protein epsin on clathrin-coated structures at the plasma membrane.\n",
      " {\n",
      "\"main_contribution\": \"Correlative super-resolution fluorescence and metal-replica transmission electron microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that combines super-resolution localization fluorescence microscopy with transmission electron microscopy of metal replicas. This method allows for the location of proteins on the cellular plasma membrane at the nanoscale. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "953\n",
      "24487583\n",
      "Multiplexed 3D cellular super-resolution imaging with DNA-PAINT and Exchange-PAINT.\n",
      "Super-resolution fluorescence microscopy is a powerful tool for biological research, but obtaining\n",
      "multiplexed images for a large number of distinct target species remains challenging. Here we use\n",
      "the transient binding of short fluorescently labeled oligonucleotides (DNA-PAINT, a variation of\n",
      "point accumulation for imaging in nanoscale topography) for simple and easy-to-implement multiplexed\n",
      "super-resolution imaging that achieves sub-10-nm spatial resolution in vitro on synthetic DNA\n",
      "structures. We also report a multiplexing approach (Exchange-PAINT) that allows sequential imaging\n",
      "of multiple targets using only a single dye and a single laser source. We experimentally demonstrate\n",
      "ten-color super-resolution imaging in vitro on synthetic DNA structures as well as four-color two-\n",
      "dimensional (2D) imaging and three-color 3D imaging of proteins in fixed cells.\n",
      " {\n",
      "\"main_contribution\": \"Multiplexed 3D cellular super-resolution imaging with DNA-PAINT and Exchange-PAINT\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called DNA-PAINT, which allows for multiplexed super-resolution imaging of a large number of distinct target species. This method uses the transient binding of short fluorescently labeled oligonucleotides to achieve sub-10-nm spatial resolution in vitro on synthetic DNA structures. Additionally, the paper reports a multiplexing approach called Exchange-PAINT that allows sequential imaging of multiple targets using only a single dye and a single laser source. Therefore, the main contribution of the paper is indeed the development of a new imaging method, and the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "954\n",
      "24509630\n",
      "A synthetic luciferin improves bioluminescence imaging in live mice.\n",
      "Firefly luciferase is the most widely used optical reporter for noninvasive bioluminescence imaging\n",
      "(BLI) in rodents. BLI relies on the ability of the injected luciferase substrate D-luciferin to\n",
      "access luciferase-expressing cells and tissues within the animal. Here we show that injection of\n",
      "mice with a synthetic luciferin, CycLuc1, improves BLI with existing luciferase reporters and\n",
      "enables imaging in the brain that could not be achieved with D-luciferin.\n",
      " {\n",
      "\"main_contribution\": \"A synthetic luciferin improves bioluminescence imaging in live mice\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a synthetic luciferin that improves bioluminescence imaging in live mice. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "955\n",
      "24509633\n",
      "Independent optical excitation of distinct neural populations.\n",
      "Optogenetic tools enable examination of how specific cell types contribute to brain circuit\n",
      "functions. A long-standing question is whether it is possible to independently activate two distinct\n",
      "neural populations in mammalian brain tissue. Such a capability would enable the study of how\n",
      "different synapses or pathways interact to encode information in the brain. Here we describe two\n",
      "channelrhodopsins, Chronos and Chrimson, discovered through sequencing and physiological\n",
      "characterization of opsins from over 100 species of alga. Chrimson's excitation spectrum is red\n",
      "shifted by 45 nm relative to previous channelrhodopsins and can enable experiments in which red\n",
      "light is preferred. We show minimal visual system-mediated behavioral interference when using\n",
      "Chrimson in neurobehavioral studies in Drosophila melanogaster. Chronos has faster kinetics than\n",
      "previous channelrhodopsins yet is effectively more light sensitive. Together these two reagents\n",
      "enable two-color activation of neural spiking and downstream synaptic transmission in independent\n",
      "neural populations without detectable cross-talk in mouse brain slice.\n",
      " {\n",
      "\"main_contribution\": \"Independent optical excitation of distinct neural populations\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "956\n",
      "24531421\n",
      "Cryo-scanning transmission electron tomography of vitrified cells.\n",
      "Cryo-electron tomography (CET) of fully hydrated, vitrified biological specimens has emerged as a\n",
      "vital tool for biological research. For cellular studies, the conventional imaging modality of\n",
      "transmission electron microscopy places stringent constraints on sample thickness because of its\n",
      "dependence on phase coherence for contrast generation. Here we demonstrate the feasibility of using\n",
      "scanning transmission electron microscopy for cryo-tomography of unstained vitrified specimens\n",
      "(CSTET). We compare CSTET and CET for the imaging of whole bacteria and human tissue culture cells,\n",
      "finding favorable contrast and detail in the CSTET reconstructions. Particularly at high sample\n",
      "tilts, the CSTET signals contain more informative data than energy-filtered CET phase contrast\n",
      "images, resulting in improved depth resolution. Careful control over dose delivery permits\n",
      "relatively high cumulative exposures before the onset of observable beam damage. The increase in\n",
      "acceptable specimen thickness broadens the applicability of electron cryo-tomography.\n",
      " {\n",
      "\"main_contribution\": \"Cryo-scanning transmission electron tomography of vitrified cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called cryo-scanning transmission electron tomography (CSTET) for imaging unstained vitrified specimens. This technique is an improvement over conventional transmission electron microscopy, which has limitations when it comes to imaging thick samples. The authors compare CSTET and CET (cryo-electron tomography) and find that CSTET provides better contrast and detail, particularly at high sample tilts. The paper demonstrates the feasibility of using CSTET for imaging whole bacteria and human tissue culture cells, and the authors suggest that this technique has the potential to broaden the applicability of electron cryo-tomography. Therefore, the main contribution of the paper is indeed the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "957\n",
      "24562422\n",
      "Gold rotor bead tracking for high-speed measurements of DNA twist, torque and extension.\n",
      "Single-molecule measurements of DNA twist and extension have been used to reveal physical properties\n",
      "of the double helix and to characterize structural dynamics and mechanochemistry in nucleoprotein\n",
      "complexes. However, the spatiotemporal resolution of twist measurements has been limited by the use\n",
      "of angular probes with high rotational drag, which prevents detection of short-lived intermediates\n",
      "or small angular steps. We introduce gold rotor bead tracking (AuRBT), which yields >100×\n",
      "improvement in time resolution over previous techniques. AuRBT employs gold nanoparticles as bright\n",
      "low-drag rotational and extensional probes, which are monitored by instrumentation that combines\n",
      "magnetic tweezers with objective-side evanescent darkfield microscopy. Our analysis of high-speed\n",
      "structural dynamics of DNA gyrase using AuRBT revealed an unanticipated transient intermediate.\n",
      "AuRBT also enables direct measurements of DNA torque with >50× shorter integration times than\n",
      "previous techniques; we demonstrated high-resolution torque spectroscopy by mapping the\n",
      "conformational landscape of a Z-forming DNA sequence.\n",
      " {\n",
      "\"main_contribution\": \"Gold rotor bead tracking for high-speed measurements of DNA twist, torque and extension\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called gold rotor bead tracking (AuRBT) that allows for high-speed measurements of DNA twist, torque, and extension. The method uses gold nanoparticles as low-drag rotational and extensional probes, which are monitored by instrumentation that combines magnetic tweezers with objective-side evanescent darkfield microscopy. Therefore, the is_imaging_method variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "958\n",
      "24562424\n",
      "Sleep-spindle detection: crowdsourcing and evaluating performance of experts, non-experts and\n",
      "automated methods.\n",
      "Sleep spindles are discrete, intermittent patterns of brain activity observed in human\n",
      "electroencephalographic data. Increasingly, these oscillations are of biological and clinical\n",
      "interest because of their role in development, learning and neurological disorders. We used an\n",
      "Internet interface to crowdsource spindle identification by human experts and non-experts, and we\n",
      "compared their performance with that of automated detection algorithms in data from middle- to\n",
      "older-aged subjects from the general population. We also refined methods for forming group consensus\n",
      "and evaluating the performance of event detectors in physiological data such as\n",
      "electroencephalographic recordings from polysomnography. Compared to the expert group consensus gold\n",
      "standard, the highest performance was by individual experts and the non-expert group consensus,\n",
      "followed by automated spindle detectors. This analysis showed that crowdsourcing the scoring of\n",
      "sleep data is an efficient method to collect large data sets, even for difficult tasks such as\n",
      "spindle identification. Further refinements to spindle detection algorithms are needed for middle-\n",
      "to older-aged subjects.\n",
      " {\n",
      "\"main_contribution\": \"Crowdsourcing the scoring of sleep data is an efficient method to collect large datasets, even for difficult tasks such as spindle identification\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is not the development of a new imaging method or technique. Rather, it's about using crowdsourcing as an efficient method to collect large datasets for sleep data, including difficult tasks like spindle identification. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "959\n",
      "24577276\n",
      "Precisely and accurately localizing single emitters in fluorescence microscopy.\n",
      "Methods based on single-molecule localization and photophysics have brought nanoscale imaging with\n",
      "visible light into reach. This has enabled single-particle tracking applications for studying the\n",
      "dynamics of molecules and nanoparticles and contributed to the recent revolution in super-resolution\n",
      "localization microscopy techniques. Crucial to the optimization of such methods are the precision\n",
      "and accuracy with which single fluorophores and nanoparticles can be localized. We present a lucid\n",
      "synthesis of the developments on this localization precision and accuracy and their practical\n",
      "implications in order to guide the increasing number of researchers using single-particle tracking\n",
      "and super-resolution localization microscopy.\n",
      " {\n",
      "\"main_contribution\": \"Precisely and accurately localizing single emitters in fluorescence microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method for precisely and accurately localizing single emitters in fluorescence microscopy. This is evident from the title and abstract, which highlight the importance of precision and accuracy in localization and the practical implications of this development for single-particle tracking and super-resolution localization microscopy techniques. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "960\n",
      "24577277\n",
      "Fluorophore localization algorithms for super-resolution microscopy.\n",
      "Super-resolution localization microscopy methods provide powerful new capabilities for probing\n",
      "biology at the nanometer scale via fluorescence. These methods rely on two key innovations:\n",
      "switchable fluorophores (which blink on and off and can be sequentially imaged) and powerful\n",
      "localization algorithms (which estimate the positions of the fluorophores in the images). These\n",
      "techniques have spurred a flurry of innovation in algorithm development over the last several years.\n",
      "In this Review, we survey the fundamental issues for single-fluorophore fitting routines,\n",
      "localization algorithms based on principles other than fitting, three-dimensional imaging, dipole\n",
      "imaging and techniques for estimating fluorophore positions from images of multiple activated\n",
      "fluorophores. We offer practical advice for users and adopters of algorithms, and we identify areas\n",
      "for further development.\n",
      " {\n",
      "\"main_contribution\": \"Fluorophore localization algorithms for super-resolution microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of fluorophore localization algorithms for super-resolution microscopy, which is an imaging technique. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "961\n",
      "24584193\n",
      "Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry.\n",
      "Mass cytometry enables high-dimensional, single-cell analysis of cell type and state. In mass\n",
      "cytometry, rare earth metals are used as reporters on antibodies. Analysis of metal abundances using\n",
      "the mass cytometer allows determination of marker expression in individual cells. Mass cytometry has\n",
      "previously been applied only to cell suspensions. To gain spatial information, we have coupled\n",
      "immunohistochemical and immunocytochemical methods with high-resolution laser ablation to CyTOF mass\n",
      "cytometry. This approach enables the simultaneous imaging of 32 proteins and protein modifications\n",
      "at subcellular resolution; with the availability of additional isotopes, measurement of over 100\n",
      "markers will be possible. We applied imaging mass cytometry to human breast cancer samples, allowing\n",
      "delineation of cell subpopulations and cell-cell interactions and highlighting tumor heterogeneity.\n",
      "Imaging mass cytometry complements existing imaging approaches. It will enable basic studies of\n",
      "tissue heterogeneity and function and support the transition of medicine toward individualized\n",
      "molecularly targeted diagnosis and therapies.\n",
      " {\n",
      "\"main_contribution\": \"Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the coupling of immunohistochemical and immunocytochemical methods with high-resolution laser ablation to CyTOF mass cytometry, which enables the simultaneous imaging of 32 proteins and protein modifications at subcellular resolution. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "962\n",
      "24584195\n",
      "Live-cell imaging of alkyne-tagged small biomolecules by stimulated Raman scattering.\n",
      "Sensitive and specific visualization of small biomolecules in living systems is highly challenging.\n",
      "We report stimulated Raman-scattering imaging of alkyne tags as a general strategy for studying a\n",
      "broad spectrum of small biomolecules in live cells and animals. We demonstrate this technique by\n",
      "tracking alkyne-bearing drugs in mouse tissues and visualizing de novo synthesis of DNA, RNA,\n",
      "proteins, phospholipids and triglycerides through metabolic incorporation of alkyne-tagged small\n",
      "precursors.\n",
      " {\n",
      "\"main_contribution\": \"stimulated Raman-scattering imaging of alkyne tags\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on stimulated Raman scattering, which allows for the visualization of small biomolecules in live cells and animals. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "963\n",
      "24608180\n",
      "Dynamic characterization of growth and gene expression using high-throughput automated flow\n",
      "cytometry.\n",
      "Cells adjust to changes in environmental conditions using complex regulatory programs. These\n",
      "cellular programs are the result of an intricate interplay between gene expression, cellular growth\n",
      "and protein degradation. Technologies that enable simultaneous and time-resolved measurements of\n",
      "these variables are necessary to dissect cellular homeostatic strategies. Here we report the\n",
      "development of an automated flow cytometry robotic setup that enables real-time measurement of\n",
      "precise and simultaneous relative growth and protein synthesis rates of multiplexed microbial\n",
      "populations across many conditions. These measurements generate quantitative profiles of dynamically\n",
      "evolving protein synthesis and degradation rates. We demonstrate this setup in the context of gene\n",
      "regulation of the unfolded protein response (UPR) of Saccharomyces cerevisiae and uncover a dynamic\n",
      "and complex landscape of gene expression, growth dynamics and proteolysis following perturbations.\n",
      " {\n",
      "\"main_contribution\": \"development of an automated flow cytometry robotic setup for real-time measurement of precise and simultaneous relative growth and protein synthesis rates of multiplexed microbial populations across many conditions\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new imaging technique using high-throughput automated flow cytometry to measure cellular growth, gene expression, and protein degradation rates in real-time. The main contribution of the paper is the development of this technique, which allows for simultaneous and time-resolved measurements of these variables. Therefore, the main contribution is identified as \"development of an automated flow cytometry robotic setup\" and is classified as true for the variable is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "964\n",
      "24633408\n",
      "Non-invasive intravital imaging of cellular differentiation with a bright red-excitable fluorescent\n",
      "protein.\n",
      "A method for non-invasive visualization of genetically labeled cells in animal disease models with\n",
      "micrometer-level resolution would greatly facilitate development of cell-based therapies. Imaging of\n",
      "fluorescent proteins (FPs) using red excitation light in the 'optical window' above 600 nm is one\n",
      "potential method for visualizing implanted cells. However, previous efforts to engineer FPs with\n",
      "peak excitation beyond 600 nm have resulted in undesirable reductions in brightness. Here we report\n",
      "three new red-excitable monomeric FPs obtained by structure-guided mutagenesis of mNeptune. Two of\n",
      "these, mNeptune2 and mNeptune2.5, demonstrate improved maturation and brighter fluorescence than\n",
      "mNeptune, whereas the third, mCardinal, has a red-shifted excitation spectrum without reduction in\n",
      "brightness. We show that mCardinal can be used to non-invasively and longitudinally visualize the\n",
      "differentiation of myoblasts into myocytes in living mice with high anatomical detail.\n",
      " {\n",
      "\"main_contribution\": \"non-invasive intravital imaging of cellular differentiation with a bright red-excitable fluorescent protein\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method using a bright red-excitable fluorescent protein for non-invasive intravital imaging of cellular differentiation. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "965\n",
      "24633409\n",
      "Accurate macromolecular structures using minimal measurements from X-ray free-electron lasers.\n",
      "X-ray free-electron laser (XFEL) sources enable the use of crystallography to solve three-\n",
      "dimensional macromolecular structures under native conditions and without radiation damage. Results\n",
      "to date, however, have been limited by the challenge of deriving accurate Bragg intensities from a\n",
      "heterogeneous population of microcrystals, while at the same time modeling the X-ray spectrum and\n",
      "detector geometry. Here we present a computational approach designed to extract meaningful high-\n",
      "resolution signals from fewer diffraction measurements.\n",
      " {\n",
      "\"main_contribution\": \"Accurate macromolecular structures using minimal measurements from X-ray free-electron lasers\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method using X-ray free-electron lasers to obtain accurate macromolecular structures. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "966\n",
      "24705472\n",
      "Fluorescence nanoscopy by polarization modulation and polarization angle narrowing.\n",
      "When excited with rotating linear polarized light, differently oriented fluorescent dyes emit\n",
      "periodic signals peaking at different times. We show that measurement of the average orientation of\n",
      "fluorescent dyes attached to rigid sample structures mapped to regularly defined (50 nm)(2) image\n",
      "nanoareas can provide subdiffraction resolution (super resolution by polarization demodulation,\n",
      "SPoD). Because the polarization angle range for effective excitation of an oriented molecule is\n",
      "rather broad and unspecific, we narrowed this range by simultaneous irradiation with a second, de-\n",
      "excitation, beam possessing a polarization perpendicular to the excitation beam (excitation\n",
      "polarization angle narrowing, ExPAN). This shortened the periodic emission flashes, allowing better\n",
      "discrimination between molecules or nanoareas. Our method requires neither the generation of\n",
      "nanometric interference structures nor the use of switchable or blinking fluorescent probes. We\n",
      "applied the method to standard wide-field microscopy with camera detection and to two-photon\n",
      "scanning microscopy, imaging the fine structural details of neuronal spines.\n",
      " {\n",
      "\"main_contribution\": \"Fluorescence nanoscopy by polarization modulation and polarization angle narrowing\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique called fluorescence nanoscopy, which uses polarization modulation and polarization angle narrowing to achieve super-resolution imaging. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "967\n",
      "24705473\n",
      "Light-sheet microscopy using an Airy beam.\n",
      "Light-sheet microscopy facilitates rapid, high-contrast, volumetric imaging with minimal sample\n",
      "exposure. However, the rapid divergence of a traditional Gaussian light sheet restricts the field of\n",
      "view (FOV) that provides innate subcellular resolution. We show that the Airy beam innately yields\n",
      "high contrast and resolution up to a tenfold larger FOV. In contrast to the Bessel beam, which also\n",
      "provides an increased FOV, the Airy beam's characteristic asymmetric excitation pattern results in\n",
      "all fluorescence contributing positively to the contrast, enabling a step change for light-sheet\n",
      "microscopy.\n",
      " {\n",
      "\"main_contribution\": \"Airy beam for light-sheet microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method using an Airy beam for light-sheet microscopy, which allows for a larger field of view and higher contrast resolution compared to traditional Gaussian light sheets. Therefore, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "968\n",
      "24727651\n",
      "Single-molecule analysis of cell surface dynamics in Caenorhabditis elegans embryos.\n",
      "We describe a general, versatile and minimally invasive method to image single molecules near the\n",
      "cell surface that can be applied to any GFP-tagged protein in Caenorhabditis elegans embryos. We\n",
      "exploited tunable expression via RNAi and a dynamically exchanging monomer pool to achieve fast,\n",
      "continuous single-molecule imaging at optimal densities with signal-to-noise ratios adequate for\n",
      "robust single-particle tracking (SPT). We introduce a method called smPReSS, single-molecule\n",
      "photobleaching relaxation to steady state, that infers exchange rates from quantitative analysis of\n",
      "single-molecule photobleaching kinetics without using SPT. Combining SPT and smPReSS allowed for\n",
      "spatially and temporally resolved measurements of protein mobility and exchange kinetics. We used\n",
      "these methods to (i) resolve distinct mobility states and spatial variation in exchange rates of the\n",
      "polarity protein PAR-6 and (ii) measure spatiotemporal modulation of actin filament assembly and\n",
      "disassembly. These methods offer a promising avenue to investigate dynamic mechanisms that pattern\n",
      "the embryonic cell surface.\n",
      " {\n",
      "\"main_contribution\": \"A versatile method for imaging single molecules near the cell surface in Caenorhabditis elegans embryos\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, as described in the abstract. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "969\n",
      "24727652\n",
      "Multiscale representation of genomic signals.\n",
      "Genomic information is encoded on a wide range of distance scales, ranging from tens of bases to\n",
      "megabases. We developed a multiscale framework to analyze and visualize the information content of\n",
      "genomic signals. Different types of signals, such as G+C content or DNA methylation, are\n",
      "characterized by distinct patterns of signal enrichment or depletion across scales spanning several\n",
      "orders of magnitude. These patterns are associated with a variety of genomic annotations. By\n",
      "integrating the information across all scales, we demonstrated improved prediction of gene\n",
      "expression from polymerase II chromatin immunoprecipitation sequencing (ChIP-seq) measurements, and\n",
      "we observed that gene expression differences in colorectal cancer are related to methylation\n",
      "patterns that extend beyond the single-gene scale. Our software is available at\n",
      "https://github.com/tknijnen/msr/.\n",
      " {\n",
      "\"main_contribution\": \"A multiscale framework to analyze and visualize the information content of genomic signals.\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a software for analyzing and visualizing genomic signals, specifically for predicting gene expression from ChIP-seq measurements. The method integrates information across multiple scales to improve prediction accuracy. While it's related to imaging in the sense that it can be used for visualization purposes, it doesn't seem to involve any novel imaging techniques or methods, so \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "970\n",
      "24727653\n",
      "Rapid adaptive optical recovery of optimal resolution over large volumes.\n",
      "Using a descanned, laser-induced guide star and direct wavefront sensing, we demonstrate adaptive\n",
      "correction of complex optical aberrations at high numerical aperture (NA) and a 14-ms update rate.\n",
      "This correction permits us to compensate for the rapid spatial variation in aberration often\n",
      "encountered in biological specimens and to recover diffraction-limited imaging over large volumes\n",
      "(>240 mm per side). We applied this to image fine neuronal processes and subcellular dynamics within\n",
      "the zebrafish brain.\n",
      " {\n",
      "\"main_contribution\": \"Rapid adaptive optical recovery of optimal resolution over large volumes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically an adaptive correction of complex optical aberrations at high numerical aperture (NA) and a 14-ms update rate. This method allows for the recovery of diffraction-limited imaging over large volumes, which is useful for imaging biological specimens. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "971\n",
      "24747812\n",
      "Efficient Bayesian-based multiview deconvolution.\n",
      "Light-sheet fluorescence microscopy is able to image large specimens with high resolution by\n",
      "capturing the samples from multiple angles. Multiview deconvolution can substantially improve the\n",
      "resolution and contrast of the images, but its application has been limited owing to the large size\n",
      "of the data sets. Here we present a Bayesian-based derivation of multiview deconvolution that\n",
      "drastically improves the convergence time, and we provide a fast implementation using graphics\n",
      "hardware.\n",
      " {\n",
      "\"main_contribution\": \"Bayesian-based multiview deconvolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for multiview deconvolution, which is a technique used to improve the resolution and contrast of images captured using light-sheet fluorescence microscopy. The authors claim that their Bayesian-based approach substantially improves the convergence time and have implemented it using graphics hardware. Therefore, the main contribution of the paper is the development of an imaging method, specifically a new approach to multiview deconvolution.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "972\n",
      "24747813\n",
      "Controlling protein adsorption on graphene for cryo-EM using low-energy hydrogen plasmas.\n",
      "Despite its many favorable properties as a sample support for biological electron microscopy,\n",
      "graphene is not widely used because its hydrophobicity precludes reliable protein deposition. We\n",
      "describe a method to modify graphene with a low-energy hydrogen plasma, which reduces hydrophobicity\n",
      "without degrading the graphene lattice. Use of plasma-treated graphene enables better control of\n",
      "protein distribution in ice for electron cryo-microscopy and improves image quality by reducing\n",
      "radiation-induced sample motion.\n",
      " {\n",
      "\"main_contribution\": \"a method to modify graphene with a low-energy hydrogen plasma for improved protein deposition in cryo-EM\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new method for modifying graphene to improve protein deposition in cryo-electron microscopy (cryo-EM) imaging. The method involves using low-energy hydrogen plasmas to reduce the hydrophobicity of graphene without degrading its lattice, allowing for better control of protein distribution in ice and improving image quality by reducing radiation-induced sample motion. Since this contribution is the development of a new imaging method, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "973\n",
      "24809628\n",
      "Comprehensive analysis of RNA-protein interactions by high-throughput sequencing-RNA affinity\n",
      "profiling.\n",
      "RNA-protein interactions play critical roles in gene regulation, but methods to quantitatively\n",
      "analyze these interactions at a large scale are lacking. We have developed a high-throughput\n",
      "sequencing-RNA affinity profiling (HiTS-RAP) assay by adapting a high-throughput DNA sequencer to\n",
      "quantify the binding of fluorescently labeled protein to millions of RNAs anchored to sequenced cDNA\n",
      "templates. Using HiTS-RAP, we measured the affinity of mutagenized libraries of GFP-binding and\n",
      "NELF-E-binding aptamers to their respective targets and identified critical regions of interaction.\n",
      "Mutations additively affected the affinity of the NELF-E-binding aptamer, whose interaction depended\n",
      "mainly on a single-stranded RNA motif, but not that of the GFP aptamer, whose interaction depended\n",
      "primarily on secondary structure.\n",
      " {\n",
      "\"main_contribution\": \"Comprehensive analysis of RNA-protein interactions by high-throughput sequencing-RNA affinity profiling\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for analyzing RNA-protein interactions called HiTS-RAP, which uses high-throughput sequencing to quantify the binding of fluorescently labeled proteins to millions of RNAs anchored to sequenced cDNA templates. The method is used to measure the affinity of mutagenized libraries of GFP-binding and NELF-E-binding aptamers to their respective targets and identify critical regions of interaction.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new method for analyzing RNA-protein interactions, which is not an imaging technique or method. Hence, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "974\n",
      "24813625\n",
      "Correlated cryogenic photoactivated localization microscopy and cryo-electron tomography.\n",
      "Cryo-electron tomography (CET) produces three-dimensional images of cells in a near-native state at\n",
      "macromolecular resolution, but identifying structures of interest can be challenging. Here we\n",
      "describe a correlated cryo-PALM (photoactivated localization microscopy)-CET method for localizing\n",
      "objects within cryo-tomograms to beyond the diffraction limit of the light microscope. Using cryo-\n",
      "PALM-CET, we identified multiple and new conformations of the dynamic type VI secretion system in\n",
      "the crowded interior of Myxococcus xanthus.\n",
      " {\n",
      "\"main_contribution\": \"identifying structures of interest can be challenging,\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "975\n",
      "24820376\n",
      "Random and targeted transgene insertion in Caenorhabditis elegans using a modified Mos1 transposon.\n",
      "We have generated a recombinant Mos1 transposon that can insert up to 45-kb transgenes into the\n",
      "Caenorhabditis elegans genome. The minimal Mos1 transposon (miniMos) is 550 bp long and inserts DNA\n",
      "into the genome at high frequency (~60% of injected animals). Genetic and antibiotic markers can be\n",
      "used for selection, and the transposon is active in C. elegans isolates and Caenorhabditis briggsae.\n",
      "We used the miniMos transposon to generate six universal Mos1-mediated single-copy insertion\n",
      "(mosSCI) landing sites that allow targeted transgene insertion with a single targeting vector into\n",
      "permissive expression sites on all autosomes. We also generated two collections of strains: a set of\n",
      "bright fluorescent insertions that are useful as dominant, genetic balancers and a set of lacO\n",
      "insertions to track genome position.\n",
      " {\n",
      "\"main_contribution\": \"generating a recombinant Mos1 transposon that can insert up to 45-kb transgenes into the Caenorhabditis elegans genome\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a modified Mos1 transposon that can insert up to 45-kb transgenes into the Caenorhabditis elegans genome. This is described in sentences (C) and (D) of the abstract. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "976\n",
      "24836920\n",
      "Simultaneous whole-animal 3D imaging of neuronal activity using light-field microscopy.\n",
      "High-speed, large-scale three-dimensional (3D) imaging of neuronal activity poses a major challenge\n",
      "in neuroscience. Here we demonstrate simultaneous functional imaging of neuronal activity at single-\n",
      "neuron resolution in an entire Caenorhabditis elegans and in larval zebrafish brain. Our technique\n",
      "captures the dynamics of spiking neurons in volumes of ∼700 μm × 700 μm × 200 μm at 20 Hz. Its\n",
      "simplicity makes it an attractive tool for high-speed volumetric calcium imaging.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous whole-animal 3D imaging of neuronal activity using light-field microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, simultaneous whole-animal 3D imaging of neuronal activity using light-field microscopy. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "977\n",
      "24859753\n",
      "Fluorogenic probes for live-cell imaging of the cytoskeleton.\n",
      "We introduce far-red, fluorogenic probes that combine minimal cytotoxicity with excellent brightness\n",
      "and photostability for fluorescence imaging of actin and tubulin in living cells. Applied in\n",
      "stimulated emission depletion (STED) microscopy, they reveal the ninefold symmetry of the centrosome\n",
      "and the spatial organization of actin in the axon of cultured rat neurons with a resolution\n",
      "unprecedented for imaging cytoskeletal structures in living cells.\n",
      " {\n",
      "\"main_contribution\": \"Fluorogenic probes for live-cell imaging of the cytoskeleton\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of fluorogenic probes for live-cell imaging of the cytoskeleton. These probes have minimal cytotoxicity and excellent brightness and photostability, making them ideal for use in stimulated emission depletion (STED) microscopy. The paper demonstrates the effectiveness of these probes by using them to image actin and tubulin in living cells with unprecedented resolution, revealing the ninefold symmetry of the centrosome and the spatial organization of actin in the axon of cultured rat neurons. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "978\n",
      "24880876\n",
      "Single-cell western blotting.\n",
      "To measure cell-to-cell variation in protein-mediated functions, we developed an approach to conduct\n",
      "∼10(3) concurrent single-cell western blots (scWesterns) in ∼4 h. A microscope slide supporting a\n",
      "30-μm-thick photoactive polyacrylamide gel enables western blotting: settling of single cells into\n",
      "microwells, lysis in situ, gel electrophoresis, photoinitiated blotting to immobilize proteins and\n",
      "antibody probing. We applied this scWestern method to monitor single-cell differentiation of rat\n",
      "neural stem cells and responses to mitogen stimulation. The scWestern quantified target proteins\n",
      "even with off-target antibody binding, multiplexed to 11 protein targets per single cell with\n",
      "detection thresholds of <30,000 molecules, and supported analyses of low starting cell numbers\n",
      "(∼200) when integrated with FACS. The scWestern overcomes limitations of antibody fidelity and\n",
      "sensitivity in other single-cell protein analysis methods and constitutes a versatile tool for the\n",
      "study of complex cell populations at single-cell resolution.\n",
      " {\n",
      "\"main_contribution\": \"A versatile tool for the study of complex cell populations at single-cell resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method called single-cell western blotting. This method allows for the measurement of cell-to-cell variation in protein-mediated functions by conducting concurrent single-cell western blots in a short amount of time. The method involves using a microscope slide with a 30-μm-thick photoactive polyacrylamide gel to enable western blotting, including settling of single cells into microwells, lysis in situ, gel electrophoresis, and photoinitiated blotting to immobilize proteins and antibody probing. Therefore, the main contribution is the development of an imaging technique, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "979\n",
      "24930129\n",
      "Protein delivery into live cells by incubation with an endosomolytic agent.\n",
      "We report that a tetramethylrhodamine-labeled dimer of the cell-penetrating peptide TAT, dfTAT,\n",
      "penetrates live cells by escaping from endosomes with high efficiency. By mediating endosomal\n",
      "leakage, dfTAT also delivers proteins into cultured cells after a simple co-incubation procedure. We\n",
      "achieved cytosolic delivery in several cell lines and primary cells and observed that only a\n",
      "relatively small amount of material remained trapped inside endosomes. Delivery did not require a\n",
      "binding interaction between dfTAT and a protein, multiple molecules could be delivered\n",
      "simultaneously, and delivery could be repeated. dfTAT-mediated delivery did not noticeably affect\n",
      "cell viability, cell proliferation or gene expression. dfTAT-based intracellular delivery should be\n",
      "useful for cell-based assays, cellular imaging applications and the ex vivo manipulation of cells.\n",
      " {\n",
      "\"main_contribution\": \"dfTAT-mediated delivery of proteins into live cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for delivering proteins into live cells using an endosomolytic agent. The main contribution of the paper is the development of this method, which is described in detail in the abstract. The method, called dfTAT-mediated delivery, involves using a tetramethylrhodamine-labeled dimer of the cell-penetrating peptide TAT to escape from endosomes and deliver proteins into cultured cells. This method does not require a binding interaction between dfTAT and a protein, and it can be used to deliver multiple molecules simultaneously. The abstract also mentions that the method did not noticeably affect cell viability, cell proliferation, or gene expression.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new method for delivering proteins into live cells, which is not an imaging technique or method. Hence, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "980\n",
      "24930130\n",
      "Chemically defined generation of human cardiomyocytes.\n",
      "Existing methods for human induced pluripotent stem cell (hiPSC) cardiac differentiation are\n",
      "efficient but require complex, undefined medium constituents that hinder further elucidation of the\n",
      "molecular mechanisms of cardiomyogenesis. Using hiPSCs derived under chemically defined conditions\n",
      "on synthetic matrices, we systematically developed an optimized cardiac differentiation strategy,\n",
      "using a chemically defined medium consisting of just three components: the basal medium RPMI 1640,\n",
      "L-ascorbic acid 2-phosphate and rice-derived recombinant human albumin. Along with small molecule-\n",
      "based induction of differentiation, this protocol produced contractile sheets of up to 95% TNNT2(+)\n",
      "cardiomyocytes at a yield of up to 100 cardiomyocytes for every input pluripotent cell and was\n",
      "effective in 11 hiPSC lines tested. This chemically defined platform for cardiac specification of\n",
      "hiPSCs will allow the elucidation of cardiomyocyte macromolecular and metabolic requirements and\n",
      "will provide a minimal system for the study of maturation and subtype specification.\n",
      " {\n",
      "\"main_contribution\": \"Chemically defined generation of human cardiomyocytes\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "981\n",
      "24952909\n",
      "Annotation of loci from genome-wide association studies using tissue-specific quantitative\n",
      "interaction proteomics.\n",
      "Genome-wide association studies (GWAS) have identified thousands of loci associated with complex\n",
      "traits, but it is challenging to pinpoint causal genes in these loci and to exploit subtle\n",
      "association signals. We used tissue-specific quantitative interaction proteomics to map a network of\n",
      "five genes involved in the Mendelian disorder long QT syndrome (LQTS). We integrated the LQTS\n",
      "network with GWAS loci from the corresponding common complex trait, QT-interval variation, to\n",
      "identify candidate genes that were subsequently confirmed in Xenopus laevis oocytes and zebrafish.\n",
      "We used the LQTS protein network to filter weak GWAS signals by identifying single-nucleotide\n",
      "polymorphisms (SNPs) in proximity to genes in the network supported by strong proteomic evidence.\n",
      "Three SNPs passing this filter reached genome-wide significance after replication genotyping.\n",
      "Overall, we present a general strategy to propose candidates in GWAS loci for functional studies and\n",
      "to systematically filter subtle association signals using tissue-specific quantitative interaction\n",
      "proteomics.\n",
      " {\n",
      "\"main_contribution\": \"identifying candidate genes involved in the Mendelian disorder long QT syndrome (LQTS) using tissue-specific quantitative interaction proteomics,\"\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a method for identifying candidate genes involved in a Mendelian disorder called long QT syndrome (LQTS) using tissue-specific quantitative interaction proteomics. The method involves integrating a protein network with GWAS loci from a corresponding common complex trait, QT-interval variation, to identify candidate genes that are subsequently confirmed in Xenopus laevis oocytes and zebrafish. The paper presents a general strategy for proposing candidates in GWAS loci for functional studies and for systematically filtering subtle association signals using tissue-specific quantitative interaction proteomics.\n",
      "\n",
      "Therefore, the main contribution of the paper is identifying candidate genes involved in LQTS using tissue-specific quantitative interaction proteomics, which is not a development of a new imaging method or technique. Hence, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "982\n",
      "24952910\n",
      "All-optical electrophysiology in mammalian neurons using engineered microbial rhodopsins.\n",
      "All-optical electrophysiology-spatially resolved simultaneous optical perturbation and measurement\n",
      "of membrane voltage-would open new vistas in neuroscience research. We evolved two archaerhodopsin-\n",
      "based voltage indicators, QuasAr1 and QuasAr2, which show improved brightness and voltage\n",
      "sensitivity, have microsecond response times and produce no photocurrent. We engineered a\n",
      "channelrhodopsin actuator, CheRiff, which shows high light sensitivity and rapid kinetics and is\n",
      "spectrally orthogonal to the QuasArs. A coexpression vector, Optopatch, enabled cross-talk-free\n",
      "genetically targeted all-optical electrophysiology. In cultured rat neurons, we combined Optopatch\n",
      "with patterned optical excitation to probe back-propagating action potentials (APs) in dendritic\n",
      "spines, synaptic transmission, subcellular microsecond-timescale details of AP propagation, and\n",
      "simultaneous firing of many neurons in a network. Optopatch measurements revealed homeostatic tuning\n",
      "of intrinsic excitability in human stem cell-derived neurons. In rat brain slices, Optopatch induced\n",
      "and reported APs and subthreshold events with high signal-to-noise ratios. The Optopatch platform\n",
      "enables high-throughput, spatially resolved electrophysiology without the use of conventional\n",
      "electrodes.\n",
      " {\n",
      "\"main_contribution\": \"All-optical electrophysiology in mammalian neurons using engineered microbial rhodopsins.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new imaging technique called all-optical electrophysiology, which uses engineered microbial rhodopsins to simultaneously perturb and measure membrane voltage in mammalian neurons. This technique allows for high-throughput, spatially resolved electrophysiology without the use of conventional electrodes. Therefore, the main contribution of the paper is the development of this new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "983\n",
      "24973948\n",
      "A method to recapitulate early embryonic spatial patterning in human embryonic stem cells.\n",
      "Embryos allocate cells to the three germ layers in a spatially ordered sequence. Human embryonic\n",
      "stem cells (hESCs) can generate the three germ layers in culture; however, differentiation is\n",
      "typically heterogeneous and spatially disordered. We show that geometric confinement is sufficient\n",
      "to trigger self-organized patterning in hESCs. In response to BMP4, colonies reproducibly\n",
      "differentiated to an outer trophectoderm-like ring, an inner ectodermal circle and a ring of\n",
      "mesendoderm expressing primitive-streak markers in between. Fates were defined relative to the\n",
      "boundary with a fixed length scale: small colonies corresponded to the outer layers of larger ones.\n",
      "Inhibitory signals limited the range of BMP4 signaling to the colony edge and induced a gradient of\n",
      "Activin-Nodal signaling that patterned mesendodermal fates. These results demonstrate that the\n",
      "intrinsic tendency of stem cells to make patterns can be harnessed by controlling colony geometries\n",
      "and provide a quantitative assay for studying paracrine signaling in early development.\n",
      " {\n",
      "\"main_contribution\": \"A method to recapitulate early embryonic spatial patterning in human embryonic stem cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a method to recapitulate early embryonic spatial patterning in human embryonic stem cells. This method involves geometric confinement and the use of BMP4 and Activin-Nodal signaling to pattern mesendodermal fates. The paper does not develop a new imaging technique or method, so is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "984\n",
      "24997861\n",
      "Accelerated chromatin biochemistry using DNA-barcoded nucleosome libraries.\n",
      "Elucidating the molecular details of how chromatin-associated factors deposit, remove and recognize\n",
      "histone post-translational modification (PTM) signatures remains a daunting task in the epigenetics\n",
      "field. We introduce a versatile platform that greatly accelerates biochemical investigations into\n",
      "chromatin recognition and signaling. This technology is based on the streamlined semisynthesis of\n",
      "DNA-barcoded nucleosome libraries with distinct combinations of PTMs. Chromatin immunoprecipitation\n",
      "of these libraries, once they have been treated with purified chromatin effectors or the combined\n",
      "chromatin recognizing and modifying activities of the nuclear proteome, is followed by multiplexed\n",
      "DNA-barcode sequencing. This ultrasensitive workflow allowed us to collect thousands of biochemical\n",
      "data points revealing the binding preferences of various nuclear factors for PTM patterns and how\n",
      "preexisting PTMs, alone or synergistically, affect further PTM deposition via cross-talk mechanisms.\n",
      "We anticipate that the high throughput and sensitivity of the technology will help accelerate the\n",
      "decryption of the diverse molecular controls that operate at the level of chromatin.\n",
      " {\n",
      "\"main_contribution\": \"accelerated biochemical investigations into chromatin recognition and signaling using a versatile platform based on DNA-barcoded nucleosome libraries\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new platform for accelerating biochemical investigations into chromatin recognition and signaling. The platform uses DNA-barcoded nucleosome libraries to streamline the process of chromatin immunoprecipitation and multiplexed DNA-barcode sequencing, allowing for the collection of thousands of biochemical data points revealing the binding preferences of various nuclear factors for PTM patterns and how preexisting PTMs affect further PTM deposition. While this platform is related to imaging in that it involves the use of DNA sequencing, it does not represent the development of a new imaging technique or method, and therefore the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "985\n",
      "25028895\n",
      "Isolation of rare recombinants without using selectable markers for one-step seamless BAC\n",
      "mutagenesis.\n",
      "Current methods to isolate rare (1:10,000-1:100,000) bacterial artificial chromosome (BAC)\n",
      "recombinants require selectable markers. For seamless BAC mutagenesis, selectable markers need to be\n",
      "removed after isolation of recombinants through counterselection. Here we illustrate founder\n",
      "principle-driven enrichment (FPE), a simple method to rapidly isolate rare recombinants without\n",
      "using selectable markers, allowing one-step seamless BAC mutagenesis. As proof of principle, we\n",
      "isolated 1:100,000 seamless fluorescent protein-modified Nodal BACs and confirmed BAC functionality\n",
      "by generating fluorescent reporter mice. We also isolated small indel P1 phage-derived artificial\n",
      "chromosome (PAC) and BAC recombinants. Statistical analysis revealed that 1:100,000 recombinants can\n",
      "be isolated with <40 PCRs, and we developed a web-based calculator to optimize FPE.\n",
      " {\n",
      "\"main_contribution\": \"seamless BAC mutagenesis without selectable markers\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for isolating rare recombinants without using selectable markers, which is the main contribution of the paper. The method, called founder principle-driven enrichment (FPE), allows for one-step seamless BAC mutagenesis. The authors prove the effectiveness of FPE by isolating 1:100,000 seamless fluorescent protein-modified Nodal BACs and generating fluorescent reporter mice. They also isolate small indel P1 phage-derived artificial chromosome (PAC) and BAC recombinants.\n",
      "\n",
      "The sentences that describe the background information are:\n",
      "\n",
      "* \"Current methods to isolate rare (1:10,000-1:100,000) bacterial artificial chromosome (BAC) recombinants require selectable markers.\"\n",
      "* \"For seamless BAC mutagenesis, selectable markers need to be removed after isolation of recombinants through counterselection.\"\n",
      "\n",
      "The sentences that describe the goals of the work are:\n",
      "\n",
      "* \"Here we illustrate founder principle-driven enrichment (FPE), a simple method to rapidly isolate rare recombinants without using selectable markers, allowing one-step seamless BAC mutagenesis.\"\n",
      "\n",
      "The sentences that describe the methods used are:\n",
      "\n",
      "* \"We isolated 1:100,000 seamless fluorescent protein-modified Nodal BACs and confirmed BAC functionality by generating fluorescent reporter mice.\"\n",
      "* \"We also isolated small indel P1 phage-derived artificial chromosome (PAC) and BAC recombinants.\"\n",
      "\n",
      "The sentences that describe the results are:\n",
      "\n",
      "* \"Statistical analysis revealed that 1:100,000 recombinants can be isolated with <40 PCRs, and we developed a web-based calculator to optimize FPE.\"\n",
      "\n",
      "The main contribution of the paper is the development of a new method for seamless BAC mutagenesis without selectable markers, which is not an imaging technique or method. Therefore, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "986\n",
      "25042784\n",
      "Optimized cell transplantation using adult rag2 mutant zebrafish.\n",
      "Cell transplantation into adult zebrafish has lagged behind mouse models owing to the lack of\n",
      "immunocompromised strains. Here we have created rag2(E450fs) mutant zebrafish that have reduced\n",
      "numbers of functional T and B cells but are viable and fecund. Mutant fish engraft muscle, blood\n",
      "stem cells and various cancers. rag2(E450fs) mutant zebrafish are the first immunocompromised\n",
      "zebrafish model that permits robust, long-term engraftment of multiple tissues and cancer.\n",
      " {\n",
      "\"main_contribution\": \"rag2(E450fs) mutant zebrafish\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new model organism, rag2(E450fs) mutant zebrafish, which has reduced numbers of functional T and B cells but is still viable and fecund. This model allows for robust, long-term engraftment of multiple tissues and cancer, making it the first immunocompromised zebrafish model that permits such engraftment. Therefore, the main contribution of the paper is the development of this new model organism, rag2(E450fs) mutant zebrafish.\n",
      "\n",
      "Since the paper does not describe the development of a new imaging technique or method, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "987\n",
      "25042785\n",
      "Fast, accurate reconstruction of cell lineages from large-scale fluorescence microscopy data.\n",
      "The comprehensive reconstruction of cell lineages in complex multicellular organisms is a central\n",
      "goal of developmental biology. We present an open-source computational framework for the\n",
      "segmentation and tracking of cell nuclei with high accuracy and speed. We demonstrate its (i)\n",
      "generality by reconstructing cell lineages in four-dimensional, terabyte-sized image data sets of\n",
      "fruit fly, zebrafish and mouse embryos acquired with three types of fluorescence microscopes, (ii)\n",
      "scalability by analyzing advanced stages of development with up to 20,000 cells per time point at\n",
      "26,000 cells min(-1) on a single computer workstation and (iii) ease of use by adjusting only two\n",
      "parameters across all data sets and providing visualization and editing tools for efficient data\n",
      "curation. Our approach achieves on average 97.0% linkage accuracy across all species and imaging\n",
      "modalities. Using our system, we performed the first cell lineage reconstruction of early Drosophila\n",
      "melanogaster nervous system development, revealing neuroblast dynamics throughout an entire embryo.\n",
      " {\n",
      "\"main_contribution\": \"Fast, accurate reconstruction of cell lineages from large-scale fluorescence microscopy data\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method for fast and accurate reconstruction of cell lineages from large-scale fluorescence microscopy data. The method is demonstrated to be effective in reconstructing cell lineages in four-dimensional, terabyte-sized image data sets of fruit fly, zebrafish, and mouse embryos acquired with three types of fluorescence microscopes. Therefore, the main contribution is the development of an imaging technique, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "988\n",
      "25042787\n",
      "High-resolution reconstruction of the beating zebrafish heart.\n",
      "The heart's continuous motion makes it difficult to capture high-resolution images of this organ in\n",
      "vivo. We developed tools based on high-speed selective plane illumination microscopy (SPIM),\n",
      "offering pristine views into the beating zebrafish heart. We captured three-dimensional cardiac\n",
      "dynamics with postacquisition synchronization of multiview movie stacks, obtained static high-\n",
      "resolution reconstructions by briefly stopping the heart with optogenetics and resolved nonperiodic\n",
      "phenomena by high-speed volume scanning with a liquid lens.\n",
      " {\n",
      "\"main_contribution\": \"High-resolution reconstruction of the beating zebrafish heart\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on high-speed selective plane illumination microscopy (SPIM) to capture high-resolution images of the beating zebrafish heart in vivo. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "989\n",
      "25068736\n",
      "Mapping brain activity at scale with cluster computing.\n",
      "Understanding brain function requires monitoring and interpreting the activity of large networks of\n",
      "neurons during behavior. Advances in recording technology are greatly increasing the size and\n",
      "complexity of neural data. Analyzing such data will pose a fundamental bottleneck for neuroscience.\n",
      "We present a library of analytical tools called Thunder built on the open-source Apache Spark\n",
      "platform for large-scale distributed computing. The library implements a variety of univariate and\n",
      "multivariate analyses with a modular, extendable structure well-suited to interactive exploration\n",
      "and analysis development. We demonstrate how these analyses find structure in large-scale neural\n",
      "data, including whole-brain light-sheet imaging data from fictively behaving larval zebrafish, and\n",
      "two-photon imaging data from behaving mouse. The analyses relate neuronal responses to sensory input\n",
      "and behavior, run in minutes or less and can be used on a private cluster or in the cloud. Our open-\n",
      "source framework thus holds promise for turning brain activity mapping efforts into biological\n",
      "insights.\n",
      " {\n",
      "\"main_contribution\": \"A library of analytical tools called Thunder built on the open-source Apache Spark platform for large-scale distributed computing\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a library of analytical tools called Thunder built on the open-source Apache Spark platform for large-scale distributed computing. This tool can be used for analyzing large-scale neural data, including whole-brain light-sheet imaging data from fictively behaving larval zebrafish, and two-photon imaging data from behaving mouse. Therefore, the is_imaging_method variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "990\n",
      "25086504\n",
      "Nanoscale high-content analysis using compositional heterogeneities of single proteoliposomes.\n",
      "Proteoliposome reconstitution is a standard method to stabilize purified transmembrane proteins in\n",
      "membranes for structural and functional assays. Here we quantified intrareconstitution\n",
      "heterogeneities in single proteoliposomes using fluorescence microscopy. Our results suggest that\n",
      "compositional heterogeneities can severely skew ensemble-average proteoliposome measurements but\n",
      "also enable ultraminiaturized high-content screens. We took advantage of this screening capability\n",
      "to map the oligomerization energy of the β2-adrenergic receptor using ∼10(9)-fold less protein than\n",
      "conventional assays.\n",
      " {\n",
      "\"main_contribution\": \"Nanoscale high-content analysis using compositional heterogeneities of single proteoliposomes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the use of compositional heterogeneities in single proteoliposomes for high-content analysis. This is described in the abstract as \"ultraminiaturized high-content screens\" and \"mapping the oligomerization energy of the β2-adrenergic receptor using ∼10(9)-fold less protein than conventional assays.\" Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "991\n",
      "25108686\n",
      "Visualizing a protein quake with time-resolved X-ray scattering at a free-electron laser.\n",
      "We describe a method to measure ultrafast protein structural changes using time-resolved wide-angle\n",
      "X-ray scattering at an X-ray free-electron laser. We demonstrated this approach using multiphoton\n",
      "excitation of the Blastochloris viridis photosynthetic reaction center, observing an ultrafast\n",
      "global conformational change that arises within picoseconds and precedes the propagation of heat\n",
      "through the protein. This provides direct structural evidence for a 'protein quake': the hypothesis\n",
      "that proteins rapidly dissipate energy through quake-like structural motions.\n",
      " {\n",
      "\"main_contribution\": \"direct structural evidence for a 'protein quake'\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "992\n",
      "25108687\n",
      "Efficient CRISPR-Cas9-mediated genome editing in Plasmodium falciparum.\n",
      "Malaria is a major cause of global morbidity and mortality, and new strategies for treating and\n",
      "preventing this disease are needed. Here we show that the Streptococcus pyogenes Cas9 DNA\n",
      "endonuclease and single guide RNAs (sgRNAs) produced using T7 RNA polymerase (T7 RNAP) efficiently\n",
      "edit the Plasmodium falciparum genome. Targeting the genes encoding native knob-associated\n",
      "histidine-rich protein (kahrp) and erythrocyte binding antigen 175 (eba-175), we achieved high (≥\n",
      "50-100%) gene disruption frequencies within the usual time frame for generating transgenic\n",
      "parasites.\n",
      " {\n",
      "\"main_contribution\": \"Efficient CRISPR-Cas9-mediated genome editing in Plasmodium falciparum\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for efficient genome editing in Plasmodium falciparum, the parasite that causes malaria, using CRISPR-Cas9. The main contribution of the paper is the development of this new method, which is not an imaging technique or method. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "993\n",
      "25128976\n",
      "Multiplexed aberration measurement for deep tissue imaging in vivo.\n",
      "We describe an adaptive optics method that modulates the intensity or phase of light rays at\n",
      "multiple pupil segments in parallel to determine the sample-induced aberration. Applicable to\n",
      "fluorescent protein-labeled structures of arbitrary complexity, it allowed us to obtain diffraction-\n",
      "limited resolution in various samples in vivo. For the strongly scattering mouse brain, a single\n",
      "aberration correction improved structural and functional imaging of fine neuronal processes over a\n",
      "large imaging volume.\n",
      " {\n",
      "\"main_contribution\": \"diffraction-limited resolution in various samples in vivo\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an adaptive optics method for achieving diffraction-limited resolution in deep tissue imaging in vivo. This method modulates the intensity or phase of light rays at multiple pupil segments in parallel to determine the sample-induced aberration, allowing for improved structural and functional imaging of fine neuronal processes over a large imaging volume in strongly scattering samples such as the mouse brain. Therefore, the main contribution is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "994\n",
      "25152083\n",
      "Systematic evaluation of quantotypic peptides for targeted analysis of the human kinome.\n",
      "In targeted proteomics it is critical that peptides are not only proteotypic but also accurately\n",
      "represent the level of the protein (quantotypic). Numerous approaches are used to identify\n",
      "proteotypic peptides, but quantotypic properties are rarely assessed. We show that measuring ratios\n",
      "of proteotypic peptides across biological samples can be used to empirically identify peptides with\n",
      "good quantotypic properties. We applied this technique to identify quantotypic peptides for 21% of\n",
      "the human kinome.\n",
      " {\n",
      "\"main_contribution\": \"Identification of quantotypic peptides for targeted analysis of the human kinome\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a systematic evaluation of quantotypic peptides for targeted analysis of the human kinome. The main contribution of the paper is the identification of quantotypic peptides, which accurately represent the level of protein, for 21% of the human kinome. This contribution is described in the abstract as \"empirically identifying peptides with good quantotypic properties\" and is not related to the development of a new imaging method, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "995\n",
      "25166869\n",
      "A critique of methods for temperature imaging in single cells.\n",
      "We argue that standard thermodynamic considerations and scaling laws show that a single cell cannot\n",
      "substantially raise its temperature by endogenous thermogenesis. This statement seriously questions\n",
      "the interpretations of recent work reporting temperature heterogeneities measured in single living\n",
      "cells.\n",
      " {\n",
      "\"main_contribution\": \"A critique of methods for temperature imaging in single cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "996\n",
      "25194847\n",
      "From genes to protein mechanics on a chip.\n",
      "Single-molecule force spectroscopy enables mechanical testing of individual proteins, but low\n",
      "experimental throughput limits the ability to screen constructs in parallel. We describe a\n",
      "microfluidic platform for on-chip expression, covalent surface attachment and measurement of single-\n",
      "molecule protein mechanical properties. A dockerin tag on each protein molecule allowed us to\n",
      "perform thousands of pulling cycles using a single cohesin-modified cantilever. The ability to\n",
      "synthesize and mechanically probe protein libraries enables high-throughput mechanical phenotyping.\n",
      " {\n",
      "\"main_contribution\": \"A microfluidic platform for on-chip expression, covalent surface attachment, and measurement of single-molecule protein mechanical properties\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work as going from genes to protein mechanics on a chip. The abstract describes the development of a microfluidic platform for expressing, attaching, and measuring single-molecule protein mechanical properties. This is the main contribution of the paper. The method used is single-molecule force spectroscopy, which is not an imaging technique or method. Therefore, is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "997\n",
      "25194848\n",
      "Evaluation and statistical inference for human connectomes.\n",
      "Diffusion-weighted imaging coupled with tractography is currently the only method for in vivo\n",
      "mapping of human white-matter fascicles. Tractography takes diffusion measurements as input and\n",
      "produces the connectome, a large collection of white-matter fascicles, as output. We introduce a\n",
      "method to evaluate the evidence supporting connectomes. Linear fascicle evaluation (LiFE) takes any\n",
      "connectome as input and predicts diffusion measurements as output, using the difference between the\n",
      "measured and predicted diffusion signals to quantify the prediction error. We use the prediction\n",
      "error to evaluate the evidence that supports the properties of the connectome, to compare\n",
      "tractography algorithms and to test hypotheses about tracts and connections.\n",
      " {\n",
      "\"main_contribution\": \"A method to evaluate the evidence supporting connectomes\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "998\n",
      "25218182\n",
      "Nanoparticle vesicle encoding for imaging and tracking cell populations.\n",
      "For phenotypic behavior to be understood in the context of cell lineage and local environment,\n",
      "properties of individual cells must be measured relative to population-wide traits. However, the\n",
      "inability to accurately identify, track and measure thousands of single cells via high-throughput\n",
      "microscopy has impeded dynamic studies of cell populations. We demonstrate unique labeling of cells,\n",
      "driven by the heterogeneous random uptake of fluorescent nanoparticles of different emission colors.\n",
      "By sequentially exposing a cell population to different particles, we generated a large number of\n",
      "unique digital codes, which corresponded to the cell-specific number of nanoparticle-loaded vesicles\n",
      "and were visible within a given fluorescence channel. When three colors are used, the assay can\n",
      "self-generate over 17,000 individual codes identifiable using a typical fluorescence microscope. The\n",
      "color-codes provided immediate visualization of cell identity and allowed us to track human cells\n",
      "with a success rate of 78% across image frames separated by 8 h.\n",
      " {\n",
      "\"main_contribution\": \"Nanoparticle vesicle encoding for imaging and tracking cell populations\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "999\n",
      "25282611\n",
      "Time-resolved crystallography using the Hadamard transform.\n",
      "We describe a method for performing time-resolved X-ray crystallographic experiments based on the\n",
      "Hadamard transform, in which time resolution is defined by the underlying periodicity of the probe\n",
      "pulse sequence, and signal/noise is greatly improved over that for the fastest pump-probe\n",
      "experiments depending on a single pulse. This approach should be applicable on standard synchrotron\n",
      "beamlines and will enable high-resolution measurements of protein and small-molecule structural\n",
      "dynamics. It is also applicable to other time-resolved measurements where a probe can be encoded,\n",
      "such as pump-probe spectroscopy.\n",
      " {\n",
      "\"main_contribution\": \"Time-resolved X-ray crystallographic experiments based on the Hadamard transform\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on the Hadamard transform, which enables time-resolved X-ray crystallographic experiments with improved signal/noise ratio and time resolution. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1000\n",
      "25306544\n",
      "An improved surface passivation method for single-molecule studies.\n",
      "We report a surface passivation method based on dichlorodimethylsilane (DDS)-Tween-20 for in vitro\n",
      "single-molecule studies, which, under the conditions tested here, more efficiently prevented\n",
      "nonspecific binding of biomolecules than the standard poly(ethylene glycol) surface. The DDS-\n",
      "Tween-20 surface was simple and inexpensive to prepare and did not perturb the behavior and\n",
      "activities of tethered biomolecules. It can also be used for single-molecule imaging in the presence\n",
      "of high concentrations of labeled species in solution.\n",
      " {\n",
      "\"main_contribution\": \"A surface passivation method for single-molecule studies\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new surface passivation method for single-molecule studies. This method is based on dichlorodimethylsilane (DDS)-Tween-20 and has been shown to be more efficient in preventing nonspecific binding of biomolecules than the standard poly(ethylene glycol) surface. However, this paper does not describe the development of a new imaging method or technique, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1001\n",
      "25326662\n",
      "Quantum dot-based multiphoton fluorescent pipettes for targeted neuronal electrophysiology.\n",
      "Targeting visually identified neurons for electrophysiological recording is a fundamental\n",
      "neuroscience technique; however, its potential is hampered by poor visualization of pipette tips in\n",
      "deep brain tissue. We describe quantum dot-coated glass pipettes that provide strong two-photon\n",
      "contrast at deeper penetration depths than those achievable with current methods. We demonstrated\n",
      "the pipettes' utility in targeted patch-clamp recording experiments and single-cell electroporation\n",
      "of identified rat and mouse neurons in vitro and in vivo.\n",
      " {\n",
      "\"main_contribution\": \"Quantum dot-based multiphoton fluorescent pipettes for targeted neuronal electrophysiology\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically quantum dot-coated glass pipettes that provide strong two-photon contrast at deeper penetration depths than current methods. This is described in sentences (C) and (D) of the abstract. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1002\n",
      "25344639\n",
      "Cell-based reporters reveal in vivo dynamics of dopamine and norepinephrine release in murine\n",
      "cortex.\n",
      "The neuronal coding of stimulus-to-action sequences is believed to involve the release of dopamine\n",
      "(DA) and norepinephrine (NE). The electrochemical similarity of these monoamines, however, confounds\n",
      "real-time measurements of their release. Here we report cell-based neurotransmitter fluorescent\n",
      "engineered reporters (CNiFERs) that use the specificity of G protein-coupled receptors (GPCRs) to\n",
      "discriminate nanomolar concentrations of DA and NE. CNiFERs were implanted into the frontal cortex\n",
      "of mice to measure the timing of neurotransmitter release during classical conditioning with the use\n",
      "of two-photon microscopy. The onset of DA release correlated with that of licking and shifted from\n",
      "the time of the reward toward that of the cue upon conditioning. In contrast, concurrent release of\n",
      "NE did not correlate with licking or the cue. This generation of CNiFERs provides unique tools to\n",
      "assess the release of monoamines. The molecular design of these CNiFERs may be generalized to\n",
      "realize CNiFERs for any molecule that activates a GPCR.\n",
      " {\n",
      "\"main_contribution\": \"The generation of cell-based neurotransmitter fluorescent engineered reporters (CNiFERs) that use the specificity of G protein-coupled receptors (GPCRs) to discriminate nanomolar concentrations of DA and NE.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, specifically the generation of cell-based neurotransmitter fluorescent engineered reporters (CNiFERs) that use the specificity of G protein-coupled receptors (GPCRs) to discriminate nanomolar concentrations of DA and NE. This is a true value for the variable is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1003\n",
      "25362362\n",
      "A robust pipeline for rapid production of versatile nanobody repertoires.\n",
      "Nanobodies are single-domain antibodies derived from the variable regions of Camelidae atypical\n",
      "immunoglobulins. They show promise as high-affinity reagents for research, diagnostics and\n",
      "therapeutics owing to their high specificity, small size (∼15 kDa) and straightforward bacterial\n",
      "expression. However, identification of repertoires with sufficiently high affinity has proven time\n",
      "consuming and difficult, hampering nanobody implementation. Our approach generates large repertoires\n",
      "of readily expressible recombinant nanobodies with high affinities and specificities against a given\n",
      "antigen. We demonstrate the efficacy of this approach through the production of large repertoires of\n",
      "nanobodies against two antigens, GFP and mCherry, with Kd values into the subnanomolar range. After\n",
      "mapping diverse epitopes on GFP, we were also able to design ultrahigh-affinity dimeric nanobodies\n",
      "with Kd values as low as ∼30 pM. The approach presented here is well suited for the routine\n",
      "production of high-affinity capture reagents for various biomedical applications.\n",
      " {\n",
      "\"main_contribution\": \"A robust pipeline for rapid production of versatile nanobody repertoires\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a pipeline for producing high-affinity nanobodies against specific antigens. The pipeline allows for the rapid production of large repertoires of recombinant nanobodies with high affinities and specificities, which can be used for various biomedical applications. While imaging techniques are not directly involved in this pipeline, the nanobodies produced could potentially be used as imaging agents in future studies. Therefore, the main contribution is not an imaging method, and the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1004\n",
      "25419959\n",
      "Rational design of a high-affinity, fast, red calcium indicator R-CaMP2.\n",
      "Fluorescent Ca(2+) reporters are widely used as readouts of neuronal activities. Here we designed\n",
      "R-CaMP2, a high-affinity red genetically encoded calcium indicator (GECI) with a Hill coefficient\n",
      "near 1. Use of the calmodulin-binding sequence of CaMKK-α and CaMKK-β in lieu of an M13 sequence\n",
      "resulted in threefold faster rise and decay times of Ca(2+) transients than R-CaMP1.07. These\n",
      "features allowed resolving single action potentials (APs) and recording fast AP trains up to 20-40\n",
      "Hz in cortical slices. Somatic and synaptic activities of a cortical neuronal ensemble in vivo were\n",
      "imaged with similar efficacy as with previously reported sensitive green GECIs. Combining green and\n",
      "red GECIs, we successfully achieved dual-color monitoring of neuronal activities of distinct cell\n",
      "types, both in the mouse cortex and in freely moving Caenorhabditis elegans. Dual imaging using\n",
      "R-CaMP2 and green GECIs provides a powerful means to interrogate orthogonal and hierarchical\n",
      "neuronal ensembles in vivo.\n",
      " {\n",
      "\"main_contribution\": \"Rational design of a high-affinity, fast, red calcium indicator R-CaMP2\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a high-affinity, fast, red calcium indicator R-CaMP2. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1005\n",
      "25419960\n",
      "Directed evolution of APEX2 for electron microscopy and proximity labeling.\n",
      "APEX is an engineered peroxidase that functions as an electron microscopy tag and a promiscuous\n",
      "labeling enzyme for live-cell proteomics. Because limited sensitivity precludes applications\n",
      "requiring low APEX expression, we used yeast-display evolution to improve its catalytic efficiency.\n",
      "APEX2 is far more active in cells, enabling the use of electron microscopy to resolve the\n",
      "submitochondrial localization of calcium uptake regulatory protein MICU1. APEX2 also permits\n",
      "superior enrichment of endogenous mitochondrial and endoplasmic reticulum membrane proteins.\n",
      " {\n",
      "\"main_contribution\": \"APEX2 for electron microscopy and proximity labeling\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of APEX2, a new imaging technique for electron microscopy and proximity labeling. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1006\n",
      "25419961\n",
      "Acoustic force spectroscopy.\n",
      "Force spectroscopy has become an indispensable tool to unravel the structural and mechanochemical\n",
      "properties of biomolecules. Here we extend the force spectroscopy toolbox with an acoustic\n",
      "manipulation device that can exert forces from subpiconewtons to hundreds of piconewtons on\n",
      "thousands of biomolecules in parallel, with submillisecond response time and inherent stability.\n",
      "This method can be readily integrated in lab-on-a-chip devices, allowing for cost-effective and\n",
      "massively parallel applications.\n",
      " {\n",
      "\"main_contribution\": \"acoustic force spectroscopy toolbox with an acoustic manipulation device\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an acoustic manipulation device that can exert forces on thousands of biomolecules in parallel, which is a new technique for studying the structural and mechanochemical properties of biomolecules. This device can be integrated into lab-on-a-chip devices, making it cost-effective and suitable for massively parallel applications. However, it is not a new imaging method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1007\n",
      "25437435\n",
      "cellPACK: a virtual mesoscope to model and visualize structural systems biology.\n",
      "cellPACK assembles computational models of the biological mesoscale, an intermediate scale (10-100\n",
      "nm) between molecular and cellular biology scales. cellPACK's modular architecture unites existing\n",
      "and novel packing algorithms to generate, visualize and analyze comprehensive three-dimensional\n",
      "models of complex biological environments that integrate data from multiple experimental systems\n",
      "biology and structural biology sources. cellPACK is available as open-source code, with tools for\n",
      "validation of models and with 'recipes' and models for five biological systems: blood plasma,\n",
      "cytoplasm, synaptic vesicles, HIV and a mycoplasma cell. We have applied cellPACK to model\n",
      "distributions of HIV envelope protein to test several hypotheses for consistency with experimental\n",
      "observations. Biologists, educators and outreach specialists can interact with cellPACK models,\n",
      "develop new recipes and perform packing experiments through scripting and graphical user interfaces\n",
      "at http://cellPACK.org/.\n",
      " {\n",
      "\"main_contribution\": \"Virtual mesoscope to model and visualize structural systems biology\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a virtual mesoscope, cellPACK, which is a new imaging technique that allows for the modeling and visualization of structural systems biology at an intermediate scale (10-100 nm) between molecular and cellular biology scales. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1008\n",
      "25486062\n",
      "DNA nanoswitches: a quantitative platform for gel-based biomolecular interaction analysis.\n",
      "We introduce a nanoscale experimental platform that enables kinetic and equilibrium measurements of\n",
      "a wide range of molecular interactions using a gel electrophoresis readout. Programmable, self-\n",
      "assembled DNA nanoswitches serve both as templates for positioning molecules and as sensitive,\n",
      "quantitative reporters of molecular association and dissociation. We demonstrated this low-cost,\n",
      "versatile, 'lab-on-a-molecule' system by characterizing ten different interactions, including a\n",
      "complex four-body interaction with five discernible states.\n",
      " {\n",
      "\"main_contribution\": \"A nanoscale experimental platform for measuring molecular interactions using gel electrophoresis and DNA nanoswitches\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new experimental platform for measuring molecular interactions, which does not involve imaging techniques. Therefore, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1009\n",
      "25506720\n",
      "Conformal nanopatterning of extracellular matrix proteins onto topographically complex surfaces.\n",
      "Our Patterning on Topography (PoT) printing technique enables fibronectin, laminin and other\n",
      "proteins to be applied to biomaterial surfaces in complex geometries that are inaccessible using\n",
      "traditional soft lithography techniques. Engineering combinatorial surfaces that integrate\n",
      "topographical and biochemical micropatterns enhances control of the biotic-abiotic interface. Here,\n",
      "we used this method to understand cardiomyocyte response to competing physical and chemical cues in\n",
      "the microenvironment.\n",
      " {\n",
      "\"main_contribution\": \"Conformal nanopatterning of extracellular matrix proteins onto topographically complex surfaces\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the main contribution of the paper as conformal nanopatterning of extracellular matrix proteins onto topographically complex surfaces. The abstract provides background information about the technique used, describes the goals of understanding cardiomyocyte response to competing physical and chemical cues in the microenvironment, and explains how the technique enables the application of proteins to biomaterial surfaces in complex geometries. However, it does not describe a new imaging method or technique, so is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1010\n",
      "25532138\n",
      "Simultaneous all-optical manipulation and recording of neural circuit activity with cellular\n",
      "resolution in vivo.\n",
      "We describe an all-optical strategy for simultaneously manipulating and recording the activity of\n",
      "multiple neurons with cellular resolution in vivo. We performed simultaneous two-photon optogenetic\n",
      "activation and calcium imaging by coexpression of a red-shifted opsin and a genetically encoded\n",
      "calcium indicator. A spatial light modulator allows tens of user-selected neurons to be targeted for\n",
      "spatiotemporally precise concurrent optogenetic activation, while simultaneous fast calcium imaging\n",
      "provides high-resolution network-wide readout of the manipulation with negligible optical cross-\n",
      "talk. Proof-of-principle experiments in mouse barrel cortex demonstrate interrogation of the same\n",
      "neuronal population during different behavioral states and targeting of neuronal ensembles based on\n",
      "their functional signature. This approach extends the optogenetic toolkit beyond the specificity\n",
      "obtained with genetic or viral approaches, enabling high-throughput, flexible and long-term optical\n",
      "interrogation of functionally defined neural circuits with single-cell and single-spike resolution\n",
      "in the mouse brain in vivo.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous all-optical manipulation and recording of neural circuit activity with cellular resolution in vivo\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an all-optical strategy for simultaneously manipulating and recording the activity of multiple neurons with cellular resolution in vivo. This technique allows for spatiotemporally precise concurrent optogenetic activation while simultaneous fast calcium imaging provides a high-resolution network-wide readout of the manipulation with negligible optical cross-talk. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1011\n",
      "25581799\n",
      "Fixation-resistant photoactivatable fluorescent proteins for CLEM.\n",
      "Fluorescent proteins facilitate a variety of imaging paradigms in live and fixed samples. However,\n",
      "they lose their fluorescence after heavy fixation, hindering applications such as correlative light\n",
      "and electron microscopy (CLEM). Here we report engineered variants of the photoconvertible Eos\n",
      "fluorescent protein that fluoresce and photoconvert normally in heavily fixed (0.5-1% OsO4), plastic\n",
      "resin-embedded samples, enabling correlative super-resolution fluorescence imaging and high-quality\n",
      "electron microscopy.\n",
      " {\n",
      "\"main_contribution\": \"engineered variants of the photoconvertible Eos fluorescent protein\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of engineered variants of the photoconvertible Eos fluorescent protein, which allows for correlative super-resolution fluorescence imaging and high-quality electron microscopy in heavily fixed and plastic resin-embedded samples. This is a new imaging method, therefore is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1012\n",
      "25581800\n",
      "Improved and expanded Q-system reagents for genetic manipulations.\n",
      "The Q system is a repressible binary expression system for transgenic manipulations in living\n",
      "organisms. Through protein engineering and in vivo functional tests, we report here variants of the\n",
      "Q-system transcriptional activator, including QF2, for driving strong and ubiquitous expression in\n",
      "all Drosophila tissues. Our QF2, Gal4QF and LexAQF chimeric transcriptional activators substantially\n",
      "enrich the toolkit available for transgenic regulation in Drosophila melanogaster.\n",
      " {\n",
      "\"main_contribution\": \"ubiquitous expression in all Drosophila tissues\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of new variants of the Q-system transcriptional activator for driving strong and ubiquitous expression in all Drosophila tissues. This is described in sentence (B) as a goal of the work being described. The methods used to achieve this goal are described in sentence (C), and the results of the work are described in sentence (D). However, the paper does not describe the development of a new imaging method, so is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1013\n",
      "25599549\n",
      "Sequence tag-based analysis of microbial population dynamics.\n",
      "We describe sequence tag-based analysis of microbial populations (STAMP) for characterization of\n",
      "pathogen population dynamics during infection. STAMP analyzes the frequency changes of genetically\n",
      "'barcoded' organisms to quantify population bottlenecks and infer the founding population size.\n",
      "Analyses of intraintestinal Vibrio cholerae revealed infection-stage and region-specific host\n",
      "barriers to infection and showed unexpected V. cholerae migration counter to intestinal flow. STAMP\n",
      "provides a robust, widely applicable analytical framework for high-confidence characterization of in\n",
      "vivo microbial dissemination.\n",
      " {\n",
      "\"main_contribution\": \"development of a sequence tag-based analysis method for characterizing microbial population dynamics\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1014\n",
      "25599551\n",
      "A general method to improve fluorophores for live-cell and single-molecule microscopy.\n",
      "Specific labeling of biomolecules with bright fluorophores is the keystone of fluorescence\n",
      "microscopy. Genetically encoded self-labeling tag proteins can be coupled to synthetic dyes inside\n",
      "living cells, resulting in brighter reporters than fluorescent proteins. Intracellular labeling\n",
      "using these techniques requires cell-permeable fluorescent ligands, however, limiting utility to a\n",
      "small number of classic fluorophores. Here we describe a simple structural modification that\n",
      "improves the brightness and photostability of dyes while preserving spectral properties and cell\n",
      "permeability. Inspired by molecular modeling, we replaced the N,N-dimethylamino substituents in\n",
      "tetramethylrhodamine with four-membered azetidine rings. This addition of two carbon atoms doubles\n",
      "the quantum efficiency and improves the photon yield of the dye in applications ranging from in\n",
      "vitro single-molecule measurements to super-resolution imaging. The novel substitution is\n",
      "generalizable, yielding a palette of chemical dyes with improved quantum efficiencies that spans the\n",
      "UV and visible range.\n",
      " {\n",
      "\"main_contribution\": \"A general method to improve fluorophores for live-cell and single-molecule microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method. The authors describe a simple structural modification that improves the brightness and photostability of dyes used in fluorescence microscopy, which doubles the quantum efficiency and improves the photon yield of the dye. This modification can be applied to a palette of chemical dyes spanning the UV and visible range, making it a general method for improving fluorophores for live-cell and single-molecule microscopy. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1015\n",
      "25622108\n",
      "Ratiometric biosensors based on dimerization-dependent fluorescent protein exchange.\n",
      "We have developed a versatile new class of genetically encoded fluorescent biosensor based on\n",
      "reversible exchange of the heterodimeric partners of green and red dimerization-dependent\n",
      "fluorescent proteins. We demonstrate the use of this strategy to construct both intermolecular and\n",
      "intramolecular ratiometric biosensors for qualitative imaging of caspase activity, Ca(2+)\n",
      "concentration dynamics and other second-messenger signaling activities.\n",
      " {\n",
      "\"main_contribution\": \"Development of a versatile new class of genetically encoded fluorescent biosensor\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on reversible exchange of the heterodimeric partners of green and red dimerization-dependent fluorescent proteins. This method is used to construct both intermolecular and intramolecular ratiometric biosensors for qualitative imaging of caspase activity, Ca(2+) concentration dynamics, and other second-messenger signaling activities. Therefore, the main contribution is identified as \"Development of a versatile new class of genetically encoded fluorescent biosensor\". Additionally, since the contribution is related to imaging, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1016\n",
      "25643149\n",
      "Reporters for sensitive and quantitative measurement of auxin response.\n",
      "The visualization of hormonal signaling input and output is key to understanding how multicellular\n",
      "development is regulated. The plant signaling molecule auxin triggers many growth and developmental\n",
      "responses, but current tools lack the sensitivity or precision to visualize these. We developed a\n",
      "set of fluorescent reporters that allow sensitive and semiquantitative readout of auxin responses at\n",
      "cellular resolution in Arabidopsis thaliana. These generic tools are suitable for any transformable\n",
      "plant species.\n",
      " {\n",
      "\"main_contribution\": \"set of fluorescent reporters for sensitive and semiquantitative readout of auxin responses\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a set of fluorescent reporters that allow for sensitive and semiquantitative measurement of auxin responses in Arabidopsis thaliana, which is an imaging method. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1017\n",
      "25686390\n",
      "Ultrastructurally smooth thick partitioning and volume stitching for large-scale connectomics.\n",
      "Focused-ion-beam scanning electron microscopy (FIB-SEM) has become an essential tool for studying\n",
      "neural tissue at resolutions below 10 nm × 10 nm × 10 nm, producing data sets optimized for\n",
      "automatic connectome tracing. We present a technical advance, ultrathick sectioning, which reliably\n",
      "subdivides embedded tissue samples into chunks (20 μm thick) optimally sized and mounted for\n",
      "efficient, parallel FIB-SEM imaging. These chunks are imaged separately and then 'volume stitched'\n",
      "back together, producing a final three-dimensional data set suitable for connectome tracing.\n",
      " {\n",
      "\"main_contribution\": \"Ultrastructurally smooth thick partitioning and volume stitching for large-scale connectomics\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically ultrathick sectioning, which allows for the efficient parallel imaging of embedded tissue samples using FIB-SEM. This technique enables the production of large-scale connectomics data sets. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1018\n",
      "25707028\n",
      "Image fusion of mass spectrometry and microscopy: a multimodality paradigm for molecular tissue\n",
      "mapping.\n",
      "We describe a predictive imaging modality created by 'fusing' two distinct technologies: imaging\n",
      "mass spectrometry (IMS) and microscopy. IMS-generated molecular maps, rich in chemical information\n",
      "but having coarse spatial resolution, are combined with optical microscopy maps, which have\n",
      "relatively low chemical specificity but high spatial information. The resulting images combine the\n",
      "advantages of both technologies, enabling prediction of a molecular distribution both at high\n",
      "spatial resolution and with high chemical specificity. Multivariate regression is used to model\n",
      "variables in one technology, using variables from the other technology. We demonstrate the potential\n",
      "of image fusion through several applications: (i) 'sharpening' of IMS images, which uses microscopy\n",
      "measurements to predict ion distributions at a spatial resolution that exceeds that of measured ion\n",
      "images by ten times or more; (ii) prediction of ion distributions in tissue areas that were not\n",
      "measured by IMS; and (iii) enrichment of biological signals and attenuation of instrumental\n",
      "artifacts, revealing insights not easily extracted from either microscopy or IMS individually.\n",
      " {\n",
      "\"main_contribution\": \"A predictive imaging modality created by fusing two distinct technologies: imaging mass spectrometry (IMS) and microscopy.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that combines the advantages of both imaging mass spectrometry (IMS) and microscopy. This is evident in the title and abstract, which describe the creation of a \"multimodality paradigm for molecular tissue mapping\" and the use of multivariate regression to model variables in one technology using variables from the other technology. The paper's focus on developing a new imaging technique and applying it to various applications further supports this conclusion. Therefore, the main contribution is recorded as \"A predictive imaging modality created by fusing two distinct technologies: imaging mass spectrometry (IMS) and microscopy.\" and is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1019\n",
      "25707029\n",
      "De novo protein structure determination from near-atomic-resolution cryo-EM maps.\n",
      "We present a de novo model-building approach that combines predicted backbone conformations with\n",
      "side-chain fit to density to accurately assign sequence into density maps. This method yielded\n",
      "accurate models for six of nine experimental maps at 3.3- to 4.8-Å resolution and produced a nearly\n",
      "complete model for an unsolved map containing a 660-residue heterodimeric protein. This method\n",
      "should enable rapid and reliable protein structure determination from near-atomic-resolution cryo-\n",
      "electron microscopy (cryo-EM) maps.\n",
      " {\n",
      "\"main_contribution\": \"de novo protein structure determination from near-atomic-resolution cryo-EM maps\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new imaging method, specifically a de novo model-building approach that combines predicted backbone conformations with side-chain fit to density to accurately assign sequence into density maps. This method allows for accurate protein structure determination from near-atomic-resolution cryo-electron microscopy (cryo-EM) maps. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1020\n",
      "25707030\n",
      "Atomic-accuracy models from 4.5-Å cryo-electron microscopy data with density-guided iterative local\n",
      "refinement.\n",
      "We describe a general approach for refining protein structure models on the basis of cryo-electron\n",
      "microscopy maps with near-atomic resolution. The method integrates Monte Carlo sampling with local\n",
      "density-guided optimization, Rosetta all-atom refinement and real-space B-factor fitting. In tests\n",
      "on experimental maps of three different systems with 4.5-Å resolution or better, the method\n",
      "consistently produced models with atomic-level accuracy largely independently of starting-model\n",
      "quality, and it outperformed the molecular dynamics-based MDFF method. Cross-validated model quality\n",
      "statistics correlated with model accuracy over the three test systems.\n",
      " {\n",
      "\"main_contribution\": \"atomic-accuracy models from 4.5-Å cryo-electron microscopy data\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a method for refining protein structure models based on cryo-electron microscopy maps with near-atomic resolution. This method integrates Monte Carlo sampling with local density-guided optimization, Rosetta all-atom refinement, and real-space B-factor fitting. The paper's focus on improving the accuracy of protein structure models using cryo-electron microscopy data makes it an imaging technique development paper. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1021\n",
      "25730490\n",
      "Highly efficient Cas9-mediated transcriptional programming.\n",
      "The RNA-guided nuclease Cas9 can be reengineered as a programmable transcription factor. However,\n",
      "modest levels of gene activation have limited potential applications. We describe an improved\n",
      "transcriptional regulator obtained through the rational design of a tripartite activator,\n",
      "VP64-p65-Rta (VPR), fused to nuclease-null Cas9. We demonstrate its utility in activating endogenous\n",
      "coding and noncoding genes, targeting several genes simultaneously and stimulating neuronal\n",
      "differentiation of human induced pluripotent stem cells (iPSCs).\n",
      " {\n",
      "\"main_contribution\": \"development of a programmable transcription factor using a tripartite activator fused to nuclease-null Cas9\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The title and abstract of the article describe the development of a new programmable transcription factor using a tripartite activator fused to nuclease-null Cas9. This is the main contribution of the paper, which is not an imaging method or technique. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1022\n",
      "25730493\n",
      "High-throughput RNA profiling via up-front sample parallelization.\n",
      "We describe a method called modular, early-tagged amplification (META) RNA profiling that can\n",
      "quantify a broad panel of microRNAs or mRNAs simultaneously across many samples and requires far\n",
      "less sequence depth than existing digital profiling technologies. The method assigns quantitative\n",
      "tags during reverse transcription to permit up-front sample pooling before competitive amplification\n",
      "and deep sequencing. This simple, scalable and inexpensive approach improves the practicality of\n",
      "large-scale gene expression studies.\n",
      " {\n",
      "\"main_contribution\": \"A method for high-throughput RNA profiling\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for high-throughput RNA profiling called META (modular, early-tagged amplification) RNA profiling. The method involves assigning quantitative tags during reverse transcription to permit up-front sample pooling before competitive amplification and deep sequencing. This approach improves the practicality of large-scale gene expression studies.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new method for high-throughput RNA profiling, which is not an imaging technique or method. Hence, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1023\n",
      "25751144\n",
      "Whole-body immunoPET reveals active SIV dynamics in viremic and antiretroviral therapy-treated\n",
      "macaques.\n",
      "The detection of viral dynamics and localization in the context of controlled HIV infection remains\n",
      "a challenge and is limited to blood and biopsies. We developed a method to capture total-body simian\n",
      "immunodeficiency virus (SIV) replication using immunoPET (antibody-targeted positron emission\n",
      "tomography). The administration of a poly(ethylene glycol)-modified, (64)Cu-labeled SIV\n",
      "Gp120-specific antibody led to readily detectable signals in the gastrointestinal and respiratory\n",
      "tract, lymphoid tissues and reproductive organs of viremic monkeys. Viral signals were reduced in\n",
      "aviremic antiretroviral-treated monkeys but detectable in colon, select lymph nodes, small bowel,\n",
      "nasal turbinates, the genital tract and lung. In elite controllers, virus was detected primarily in\n",
      "foci in the small bowel, select lymphoid areas and the male reproductive tract, as confirmed by\n",
      "quantitative reverse-transcription PCR (qRT-PCR) and immunohistochemistry. This real-time, in vivo\n",
      "viral imaging method has broad applications to the study of immunodeficiency virus pathogenesis,\n",
      "drug and vaccine development, and the potential for clinical translation.\n",
      " {\n",
      "\"main_contribution\": \"real-time, in vivo viral imaging method\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called immunoPET, which allows for the detection of viral dynamics and localization in the context of controlled HIV infection. This method uses a poly(ethylene glycol)-modified, (64)Cu-labeled SIV Gp120-specific antibody to capture total-body simian immunodeficiency virus (SIV) replication. The signals were readily detectable in various tissues and organs of viremic monkeys, and reduced but still detectable in aviremic antiretroviral-treated monkeys. This method has broad applications to the study of immunodeficiency virus pathogenesis, drug and vaccine development, and potentially clinical translation. Therefore, the main contribution is the development of a new imaging technique or method, and the answer to is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1024\n",
      "25775042\n",
      "Direct visualization of newly synthesized target proteins in situ.\n",
      "Protein synthesis is a dynamic process that tunes the cellular proteome in response to internal and\n",
      "external demands. Metabolic labeling approaches identify the general proteomic response but cannot\n",
      "visualize specific newly synthesized proteins within cells. Here we describe a technique that\n",
      "couples noncanonical amino acid tagging or puromycylation with the proximity ligation assay to\n",
      "visualize specific newly synthesized proteins and monitor their origin, redistribution and turnover\n",
      "in situ.\n",
      " {\n",
      "\"main_contribution\": \"Visualization of newly synthesized target proteins in situ\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new technique for visualizing specific, newly synthesized proteins within cells. This technique involves coupling noncanonical amino acid tagging or puromycylation with the proximity ligation assay. The main contribution of the paper is thus the development of this new imaging method, which allows researchers to monitor the origin, redistribution, and turnover of newly synthesized proteins in situ. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1025\n",
      "25775044\n",
      "CIDRE: an illumination-correction method for optical microscopy.\n",
      "Uneven illumination affects every image acquired by a microscope. It is often overlooked, but it can\n",
      "introduce considerable bias to image measurements. The most reliable correction methods require\n",
      "special reference images, and retrospective alternatives do not fully model the correction process.\n",
      "Our approach overcomes these issues for most optical microscopy applications without the need for\n",
      "reference images.\n",
      " {\n",
      "\"main_contribution\": \"an illumination-correction method for optical microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an illumination-correction method for optical microscopy, which is a new imaging technique. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1026\n",
      "25775045\n",
      "Catchup: a mouse model for imaging-based tracking and modulation of neutrophil granulocytes.\n",
      "Neutrophil granulocyte biology is a central issue of immunological research, but the lack of animal\n",
      "models that allow for neutrophil-selective genetic manipulation has delayed progress. By modulating\n",
      "the neutrophil-specific locus Ly6G with a knock-in allele expressing Cre recombinase and the\n",
      "fluorescent protein tdTomato, we generated a mouse model termed Catchup that exhibits strong\n",
      "neutrophil specificity. Transgene activity was found only in very few eosinophils and basophils and\n",
      "was undetectable in bone marrow precursors, including granulomonocytic progenitors (GMPs). Cre-\n",
      "mediated reporter-gene activation allowed for intravital two-photon microscopy of neutrophils\n",
      "without adoptive transfer. Homozygous animals were Ly6G deficient but showed normal leukocyte\n",
      "cellularity in all measured organs. Ly6G-deficient neutrophils were functionally normal in vitro and\n",
      "in multiple models of sterile or infectious inflammation in vivo. However, Cre-mediated deletion of\n",
      "FcγRIV in neutrophils reduced the cells' recruitment to immune-complex-mediated peritonitis,\n",
      "suggesting a cell-intrinsic role for activating Fc receptors in neutrophil trafficking.\n",
      " {\n",
      "\"main_contribution\": \"A mouse model for imaging-based tracking and modulation of neutrophil granulocytes was developed.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The title and abstract of the paper describe the development of a new mouse model, termed Catchup, which allows for neutrophil-selective genetic manipulation and imaging-based tracking and modulation of neutrophil granulocytes. The main contribution of the paper is the development of this mouse model, which is described in sentences (B) and (C). Sentence (A) provides background information about the importance of neutrophil biology in immunological research, while sentence (D) describes the results of the work, including the successful generation of a mouse model that exhibits strong neutrophil specificity and the functionality of Ly6G-deficient neutrophils in vitro and in vivo. Sentence (E) discusses the implications of the work, specifically the role of activating Fc receptors in neutrophil trafficking. Therefore, the main contribution of the paper is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1027\n",
      "25822799\n",
      "High-speed label-free functional photoacoustic microscopy of mouse brain in action.\n",
      "We present fast functional photoacoustic microscopy (PAM) for three-dimensional high-resolution,\n",
      "high-speed imaging of the mouse brain, complementary to other imaging modalities. We implemented a\n",
      "single-wavelength pulse-width-based method with a one-dimensional imaging rate of 100 kHz to image\n",
      "blood oxygenation with capillary-level resolution. We applied PAM to image the vascular morphology,\n",
      "blood oxygenation, blood flow and oxygen metabolism in both resting and stimulated states in the\n",
      "mouse brain.\n",
      " {\n",
      "\"main_contribution\": \"High-speed label-free functional photoacoustic microscopy of mouse brain in action\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a high-speed label-free functional photoacoustic microscopy technique for imaging the mouse brain in action. This technique allows for three-dimensional high-resolution and high-speed imaging of the brain, complementary to other imaging modalities. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1028\n",
      "25825836\n",
      "Integrative, dynamic structural biology at atomic resolution--it's about time.\n",
      "Biomolecules adopt a dynamic ensemble of conformations, each with the potential to interact with\n",
      "binding partners or perform the chemical reactions required for a multitude of cellular functions.\n",
      "Recent advances in X-ray crystallography, nuclear magnetic resonance (NMR) spectroscopy and other\n",
      "techniques are helping us realize the dream of seeing--in atomic detail--how different parts of\n",
      "biomolecules shift between functional substates using concerted motions. Integrative structural\n",
      "biology has advanced our understanding of the formation of large macromolecular complexes and how\n",
      "their components interact in assemblies by leveraging data from many low-resolution methods. Here,\n",
      "we review the growing opportunities for integrative, dynamic structural biology at the atomic scale,\n",
      "contending there is increasing synergistic potential between X-ray crystallography, NMR and computer\n",
      "simulations to reveal a structural basis for protein conformational dynamics at high resolution.\n",
      " {\n",
      "\"main_contribution\": \"a structural basis for protein conformational dynamics at high resolution,\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1029\n",
      "25849636\n",
      "Massively parallel delivery of large cargo into mammalian cells with light pulses.\n",
      "We report a high-throughput platform for delivering large cargo elements into 100,000 cells in 1\n",
      "min. Our biophotonic laser-assisted surgery tool (BLAST) generates an array of microcavitation\n",
      "bubbles that explode in response to laser pulsing, forming pores in adjacent cell membranes through\n",
      "which cargo is gently driven by pressurized flow. The platform delivers large items including\n",
      "bacteria, enzymes, antibodies and nanoparticles into diverse cell types with high efficiency and\n",
      "cell viability. We used this platform to explore the intracellular lifestyle of Francisella novicida\n",
      "and discovered that the iglC gene is unexpectedly required for intracellular replication even after\n",
      "phagosome escape into the cell cytosol.\n",
      " {\n",
      "\"main_contribution\": \"A high-throughput platform for delivering large cargo elements into 100,000 cells in 1 min\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a high-throughput platform for delivering large cargo elements into mammalian cells with light pulses. This platform uses microcavitation bubbles that explode in response to laser pulsing, forming pores in adjacent cell membranes through which cargo is gently driven by pressurized flow. The platform delivers large items including bacteria, enzymes, antibodies, and nanoparticles into diverse cell types with high efficiency and cell viability. Therefore, the main contribution is not the development of a new imaging method, but rather a new method for delivering cargo into cells, and hence is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1030\n",
      "25867848\n",
      "Rapid reverse genetic screening using CRISPR in zebrafish.\n",
      "Identifying genes involved in biological processes is critical for understanding the molecular\n",
      "building blocks of life. We used engineered CRISPR (clustered regularly interspaced short\n",
      "palindromic repeats) to efficiently mutate specific loci in zebrafish (Danio rerio) and screen for\n",
      "genes involved in vertebrate biological processes. We found that increasing CRISPR efficiency by\n",
      "injecting optimized amounts of Cas9-encoding mRNA and multiplexing single guide RNAs (sgRNAs)\n",
      "allowed for phenocopy of known mutants across many phenotypes in embryos. We performed a proof-of-\n",
      "concept screen in which we used intersecting, multiplexed pool injections to examine 48 loci and\n",
      "identified two new genes involved in electrical-synapse formation. By deep sequencing target loci,\n",
      "we found that 90% of the genes were effectively screened. We conclude that CRISPR can be used as a\n",
      "powerful reverse genetic screening strategy in vivo in a vertebrate system.\n",
      " {\n",
      "\"main_contribution\": \"Identifying genes involved in biological processes using CRISPR in zebrafish\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "Sentences describing background information:\n",
      "- \"Identifying genes involved in biological processes is critical for understanding the molecular building blocks of life.\"\n",
      "\n",
      "Sentences describing goals:\n",
      "- \"We used engineered CRISPR (clustered regularly interspaced short palindromic repeats) to efficiently mutate specific loci in zebrafish (Danio rerio) and screen for genes involved in vertebrate biological processes.\"\n",
      "\n",
      "Sentences describing methods:\n",
      "- \"We found that increasing CRISPR efficiency by injecting optimized amounts of Cas9-encoding mRNA and multiplexing single guide RNAs (sgRNAs) allowed for phenocopy of known mutants across many phenotypes in embryos.\"\n",
      "- \"We performed a proof-of-concept screen in which we used intersecting, multiplexed pool injections to examine 48 loci and identified two new genes involved in electrical-synapse formation.\"\n",
      "\n",
      "Sentences describing results:\n",
      "- \"By deep sequencing target loci, we found that 90% of the genes were effectively screened.\"\n",
      "\n",
      "Sentences describing conclusions:\n",
      "- \"We conclude that CRISPR can be used as a powerful reverse genetic screening strategy in vivo in a vertebrate system.\"\n",
      "\n",
      "The main contribution of the paper is identifying genes involved in biological processes using CRISPR in zebrafish. The method used is CRISPR, but it's not a new imaging method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1031\n",
      "25867849\n",
      "High-resolution whole-brain staining for electron microscopic circuit reconstruction.\n",
      "Currently only electron microscopy provides the resolution necessary to reconstruct neuronal\n",
      "circuits completely and with single-synapse resolution. Because almost all behaviors rely on neural\n",
      "computations widely distributed throughout the brain, a reconstruction of brain-wide circuits-and,\n",
      "ultimately, the entire brain-is highly desirable. However, these reconstructions require the\n",
      "undivided brain to be prepared for electron microscopic observation. Here we describe a preparation,\n",
      "BROPA (brain-wide reduced-osmium staining with pyrogallol-mediated amplification), that results in\n",
      "the preservation and staining of ultrastructural details throughout the brain at a resolution\n",
      "necessary for tracing neuronal processes and identifying synaptic contacts between them. Using\n",
      "serial block-face electron microscopy (SBEM), we tested human annotator ability to follow neural\n",
      "'wires' reliably and over long distances as well as the ability to detect synaptic contacts. Our\n",
      "results suggest that the BROPA method can produce a preparation suitable for the reconstruction of\n",
      "neural circuits spanning an entire mouse brain.\n",
      " {\n",
      "\"main_contribution\": \"Development of a high-resolution whole-brain staining method for electron microscopic circuit reconstruction\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a whole-brain staining method called BROPA (brain-wide reduced-osmium staining with pyrogallol-mediated amplification), which enables the preservation and staining of ultrastructural details throughout the brain at a resolution necessary for tracing neuronal processes and identifying synaptic contacts between them. This method is suitable for the reconstruction of neural circuits spanning an entire mouse brain, as demonstrated by the authors using serial block-face electron microscopy (SBEM). Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1032\n",
      "25867850\n",
      "In vivo cell-cycle profiling in xenograft tumors by quantitative intravital microscopy.\n",
      "Quantification of cell-cycle state at a single-cell level is essential to understand fundamental\n",
      "three-dimensional (3D) biological processes such as tissue development and cancer. Analysis of 3D in\n",
      "vivo images, however, is very challenging. Today's best practice, manual annotation of select image\n",
      "events, generates arbitrarily sampled data distributions, which are unsuitable for reliable\n",
      "mechanistic inferences. Here, we present an integrated workflow for quantitative in vivo cell-cycle\n",
      "profiling. It combines image analysis and machine learning methods for automated 3D segmentation and\n",
      "cell-cycle state identification of individual cell-nuclei with widely varying morphologies embedded\n",
      "in complex tumor environments. We applied our workflow to quantify cell-cycle effects of three\n",
      "antimitotic cancer drugs over 8 d in HT-1080 fibrosarcoma xenografts in living mice using a data set\n",
      "of 38,000 cells and compared the induced phenotypes. In contrast to results with 2D culture,\n",
      "observed mitotic arrest was relatively low, suggesting involvement of additional mechanisms in their\n",
      "antitumor effect in vivo.\n",
      " {\n",
      "\"main_contribution\": \"Quantitative in vivo cell-cycle profiling workflow using intravital microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, specifically an integrated workflow for quantitative in vivo cell-cycle profiling using intravital microscopy. This method combines image analysis and machine learning techniques for automated 3D segmentation and cell-cycle state identification of individual cell nuclei with varying morphologies embedded in complex tumor environments. Therefore, the is_imaging_method variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1033\n",
      "25894945\n",
      "Isotope-targeted glycoproteomics (IsoTaG): a mass-independent platform for intact N- and\n",
      "O-glycopeptide discovery and analysis.\n",
      "Protein glycosylation is a heterogeneous post-translational modification (PTM) that plays an\n",
      "essential role in biological regulation. However, the diversity found in glycoproteins has\n",
      "undermined efforts to describe the intact glycoproteome via mass spectrometry (MS). We present\n",
      "IsoTaG, a mass-independent chemical glycoproteomics platform for characterization of intact,\n",
      "metabolically labeled glycopeptides at the whole-proteome scale. In IsoTaG, metabolic labeling of\n",
      "the glycoproteome is combined with (i) chemical enrichment and isotopic recoding of glycopeptides to\n",
      "select peptides for targeted glycoproteomics using directed MS and (ii) mass-independent assignment\n",
      "of intact glycopeptides. We structurally assigned 32 N-glycopeptides and over 500 intact and fully\n",
      "elaborated O-glycopeptides from 250 proteins across three human cancer cell lines and also\n",
      "discovered unexpected peptide sequence polymorphisms (pSPs). The IsoTaG platform is broadly\n",
      "applicable to the discovery of PTM sites that are amenable to chemical labeling, as well as\n",
      "previously unknown protein isoforms including pSPs.\n",
      " {\n",
      "\"main_contribution\": \"A mass-independent platform for intact N- and O-glycopeptide discovery and analysis\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called IsoTaG, which is a mass-independent chemical glycoproteomics platform for characterizing intact, metabolically labeled glycopeptides at the whole-proteome scale. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1034\n",
      "25894946\n",
      "A fast- and positively photoswitchable fluorescent protein for ultralow-laser-power RESOLFT\n",
      "nanoscopy.\n",
      "Fluorescence nanoscopy has revolutionized our ability to visualize biological structures not\n",
      "resolvable by conventional microscopy. However, photodamage induced by intense light exposure has\n",
      "limited its use in live specimens. Here we describe Kohinoor, a fast-switching, positively\n",
      "photoswitchable fluorescent protein, and show that it has high photostability over many switching\n",
      "repeats. With Kohinoor, we achieved super-resolution imaging of live HeLa cells using biocompatible,\n",
      "ultralow laser intensity (0.004 J/cm(2)) in reversible saturable optical fluorescence transition\n",
      "(RESOLFT) nanoscopy.\n",
      " {\n",
      "\"main_contribution\": \"A fast- and positively photoswitchable fluorescent protein for ultralow-laser-power RESOLFT nanoscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a fast-switching, positively photoswitchable fluorescent protein called Kohinoor, which enables super-resolution imaging of live cells using ultralow laser intensity in RESOLFT nanoscopy. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1035\n",
      "25915120\n",
      "High-performance probes for light and electron microscopy.\n",
      "We describe an engineered family of highly antigenic molecules based on GFP-like fluorescent\n",
      "proteins. These molecules contain numerous copies of peptide epitopes and simultaneously bind IgG\n",
      "antibodies at each location. These 'spaghetti monster' fluorescent proteins (smFPs) distributed well\n",
      "in neurons, notably into small dendrites, spines and axons. smFP immunolabeling localized weakly\n",
      "expressed proteins not well resolved with traditional epitope tags. By varying epitope and scaffold,\n",
      "we generated a diverse family of mutually orthogonal antigens. In cultured neurons and mouse and fly\n",
      "brains, smFP probes allowed robust, orthogonal multicolor visualization of proteins, cell\n",
      "populations and neuropil. smFP variants complement existing tracers and greatly increase the number\n",
      "of simultaneous imaging channels, and they performed well in advanced preparations such as array\n",
      "tomography, super-resolution fluorescence imaging and electron microscopy. In living cells, the\n",
      "probes improved single-molecule image tracking and increased yield for RNA-seq. These probes\n",
      "facilitate new experiments in connectomics, transcriptomics and protein localization.\n",
      " {\n",
      "\"main_contribution\": \"High-performance probes for light and electron microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of high-performance probes for light and electron microscopy, which is a new imaging technique. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1036\n",
      "25938370\n",
      "Rapid, optimized interactomic screening.\n",
      "We must reliably map the interactomes of cellular macromolecular complexes in order to fully explore\n",
      "and understand biological systems. However, there are no methods to accurately predict how to\n",
      "capture a given macromolecular complex with its physiological binding partners. Here, we present a\n",
      "screening method that comprehensively explores the parameters affecting the stability of\n",
      "interactions in affinity-captured complexes, enabling the discovery of physiological binding\n",
      "partners in unparalleled detail. We have implemented this screen on several macromolecular complexes\n",
      "from a variety of organisms, revealing novel profiles for even well-studied proteins. Our approach\n",
      "is robust, economical and automatable, providing inroads to the rigorous, systematic dissection of\n",
      "cellular interactomes.\n",
      " {\n",
      "\"main_contribution\": \"a screening method that comprehensively explores the parameters affecting the stability of interactions in affinity-captured complexes\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for interactomic screening, which is a technique used to study the interactions between macromolecular complexes and their binding partners. The main contribution of the paper is the development of this screening method, which is described in the abstract as a way to comprehensively explore the parameters affecting the stability of interactions in affinity-captured complexes. This method is not an imaging technique or method, so the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1037\n",
      "25961413\n",
      "Structural analysis of multicellular organisms with cryo-electron tomography.\n",
      "We developed a method for visualizing tissues from multicellular organisms using cryo-electron\n",
      "tomography. Our protocol involves vitrifying samples with high-pressure freezing, thinning them with\n",
      "cryo-FIB-SEM (focused-ion-beam scanning electron microscopy) and applying fiducial gold markers\n",
      "under cryogenic conditions to the lamellae post-milling. We applied this protocol to acquire\n",
      "tomograms of vitrified Caenorhabditis elegans embryos and worms, which showed the intracellular\n",
      "organization of selected tissues at particular developmental stages in otherwise intact specimens.\n",
      " {\n",
      "\"main_contribution\": \"Visualizing tissues from multicellular organisms using cryo-electron tomography\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method for visualizing tissues from multicellular organisms using cryo-electron tomography, which is an imaging technique. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1038\n",
      "25961414\n",
      "3D high- and super-resolution imaging using single-objective SPIM.\n",
      "Single-objective selective-plane illumination microscopy (soSPIM) is achieved with micromirrored\n",
      "cavities combined with a laser beam-steering unit installed on a standard inverted microscope. The\n",
      "illumination and detection are done through the same objective. soSPIM can be used with standard\n",
      "sample preparations and features high background rejection and efficient photon collection, allowing\n",
      "for 3D single-molecule-based super-resolution imaging of whole cells or cell aggregates. Using\n",
      "larger mirrors enabled us to broaden the capabilities of our system to image Drosophila embryos.\n",
      " {\n",
      "\"main_contribution\": \"3D high- and super-resolution imaging using single-objective SPIM\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a novel application of selective-plane illumination microscopy (SPIM) that uses a single objective lens for both illumination and detection. This technique allows for 3D high- and super-resolution imaging of whole cells or cell aggregates with high background rejection and efficient photon collection. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1039\n",
      "25984698\n",
      "Probing a cell-embedded megadalton protein complex by DNP-supported solid-state NMR.\n",
      "Studying biomolecules at atomic resolution in their native environment is the ultimate aim of\n",
      "structural biology. We investigated the bacterial type IV secretion system core complex (T4SScc) by\n",
      "cellular dynamic nuclear polarization-based solid-state nuclear magnetic resonance spectroscopy to\n",
      "validate a structural model previously generated by combining in vitro and in silico data. Our\n",
      "results indicate that T4SScc is well folded in the cellular setting, revealing protein regions that\n",
      "had been elusive when studied in vitro.\n",
      " {\n",
      "\"main_contribution\": \"Studying biomolecules at atomic resolution in their native environment\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1040\n",
      "26005811\n",
      "Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture.\n",
      "The human cerebral cortex develops through an elaborate succession of cellular events that, when\n",
      "disrupted, can lead to neuropsychiatric disease. The ability to reprogram somatic cells into\n",
      "pluripotent cells that can be differentiated in vitro provides a unique opportunity to study normal\n",
      "and abnormal corticogenesis. Here, we present a simple and reproducible 3D culture approach for\n",
      "generating a laminated cerebral cortex-like structure, named human cortical spheroids (hCSs), from\n",
      "pluripotent stem cells. hCSs contain neurons from both deep and superficial cortical layers and map\n",
      "transcriptionally to in vivo fetal development. These neurons are electrophysiologically mature,\n",
      "display spontaneous activity, are surrounded by nonreactive astrocytes and form functional synapses.\n",
      "Experiments in acute hCS slices demonstrate that cortical neurons participate in network activity\n",
      "and produce complex synaptic events. These 3D cultures should allow a detailed interrogation of\n",
      "human cortical development, function and disease, and may prove a versatile platform for generating\n",
      "other neuronal and glial subtypes in vitro.\n",
      " {\n",
      "\"main_contribution\": \"Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a 3D culture approach for generating a laminated cerebral cortex-like structure from pluripotent stem cells. This is evident in the abstract, which highlights the ability to study normal and abnormal corticogenesis using this approach. Additionally, the use of the phrase \"a simple and reproducible 3D culture approach\" suggests that the method is novel and has not been previously described. Therefore, the main contribution of the paper is the development of a new imaging technique or method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1041\n",
      "26005812\n",
      "Dexterous robotic manipulation of alert adult Drosophila for high-content experimentation.\n",
      "We present a robot that enables high-content studies of alert adult Drosophila by combining\n",
      "operations including gentle picking; translations and rotations; characterizations of fly phenotypes\n",
      "and behaviors; microdissection; or release. To illustrate, we assessed fly morphology, tracked odor-\n",
      "evoked locomotion, sorted flies by sex, and dissected the cuticle to image neural activity. The\n",
      "robot's tireless capacity for precise manipulations enables a scalable platform for screening flies'\n",
      "complex attributes and behavioral patterns.\n",
      " {\n",
      "\"main_contribution\": \"A robot for high-content experimentation of alert adult Drosophila\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a robot that enables high-content experimentation of alert adult Drosophila, which includes imaging neural activity. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1042\n",
      "26020503\n",
      "Correlated light and electron microscopy: ultrastructure lights up!\n",
      "Microscopy has gone hand in hand with the study of living systems since van Leeuwenhoek observed\n",
      "living microorganisms and cells in 1674 using his light microscope. A spectrum of dyes and probes\n",
      "now enable the localization of molecules of interest within living cells by fluorescence microscopy.\n",
      "With electron microscopy (EM), cellular ultrastructure has been revealed. Bridging these two\n",
      "modalities, correlated light microscopy and EM (CLEM) opens new avenues. Studies of protein dynamics\n",
      "with fluorescent proteins (FPs), which leave the investigator 'in the dark' concerning cellular\n",
      "context, can be followed by EM examination. Rare events can be preselected at the light microscopy\n",
      "level before EM analysis. Ongoing development-including of dedicated probes, integrated microscopes,\n",
      "large-scale and three-dimensional EM and super-resolution fluorescence microscopy-now paves the way\n",
      "for broad CLEM implementation in biology.\n",
      " {\n",
      "\"main_contribution\": \"Correlated light and electron microscopy: ultrastructure lights up!\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically correlated light and electron microscopy (CLEM), which allows for the study of protein dynamics with fluorescent proteins and the examination of rare events in cellular ultrastructure. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1043\n",
      "26030444\n",
      "Multiplexable, locus-specific targeting of long RNAs with CRISPR-Display.\n",
      "Noncoding RNAs play diverse roles throughout biology and exhibit broad functional capacity. To\n",
      "investigate and harness these capabilities, we developed clustered regularly interspaced short\n",
      "palindromic repeats (CRISPR)-Display (CRISP-Disp), a targeted localization method that uses Cas9 to\n",
      "deploy large RNA cargos to DNA loci. We demonstrate that functional RNA domains up to at least 4.8\n",
      "kb long can be inserted in CRISPR guide RNA at multiple points, allowing the construction of Cas9\n",
      "complexes with protein-binding cassettes, artificial aptamers, pools of random sequences and natural\n",
      "long noncoding RNAs. A unique feature of CRISP-Disp is the multiplexing of distinct functions at\n",
      "multiple targets, limited only by the availability of functional RNA motifs. We anticipate the use\n",
      "of CRISP-Disp for ectopically targeting functional RNAs and ribonucleoprotein (RNP) complexes to\n",
      "genomic loci.\n",
      " {\n",
      "\"main_contribution\": \"targeted localization method that uses Cas9 to deploy large RNA cargos to DNA loci\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called CRISPR-Display (CRISP-Disp) that uses Cas9 to deploy large RNA cargos to DNA loci. This method allows for the multiplexing of distinct functions at multiple targets, limited only by the availability of functional RNA motifs. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1044\n",
      "26030446\n",
      "Traction microscopy to identify force modulation in subresolution adhesions.\n",
      "We present a reconstruction algorithm that resolves cellular tractions in diffraction-limited\n",
      "nascent adhesions (NAs). The enabling method is the introduction of sparsity regularization to the\n",
      "solution of the inverse problem, which suppresses noise without underestimating traction magnitude.\n",
      "We show that NAs transmit a distinguishable amount of traction and that NA maturation depends on\n",
      "traction growth rate. A software package implementing this numerical approach is provided.\n",
      " {\n",
      "\"main_contribution\": \"Traction microscopy to identify force modulation in subresolution adhesions\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called traction microscopy, which allows for the identification of force modulation in subresolution adhesions. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1045\n",
      "26030447\n",
      "Plasmodium falciparum genetic crosses in a humanized mouse model.\n",
      "Genetic crosses of phenotypically distinct strains of the human malaria parasite Plasmodium\n",
      "falciparum are a powerful tool for identifying genes controlling drug resistance and other key\n",
      "phenotypes. Previous studies relied on the isolation of recombinant parasites from splenectomized\n",
      "chimpanzees, a research avenue that is no longer available. Here we demonstrate that human-liver\n",
      "chimeric mice support recovery of recombinant progeny for the identification of genetic determinants\n",
      "of parasite traits and adaptations.\n",
      " {\n",
      "\"main_contribution\": \"identifying genes controlling drug resistance and other key phenotypes\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work being described as involving Plasmodium falciparum genetic crosses in a humanized mouse model. The abstract provides background information about previous studies that relied on isolating recombinant parasites from splenectomized chimpanzees, which is no longer available. It then describes the goal of the work as identifying genetic determinants of parasite traits and adaptations using human-liver chimeric mice to support recovery of recombinant progeny. The methods used are not explicitly stated in the abstract, but it can be inferred that they involve genetic crosses and recovery of recombinant progeny in human-liver chimeric mice. The results of the work are not explicitly stated either, but it is implied that the method allows for identifying genetic determinants of parasite traits and adaptations.\n",
      "\n",
      "Therefore, the main contribution of the paper is identifying genes controlling drug resistance and other key phenotypes, which is a goal described in the abstract. Since this contribution does not involve the development of a new imaging technique or method, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1046\n",
      "26030448\n",
      "Application of BRET to monitor ligand binding to GPCRs.\n",
      "Bioluminescence resonance energy transfer (BRET) is a well-established method for investigating\n",
      "protein-protein interactions. Here we present a BRET approach to monitor ligand binding to G\n",
      "protein-coupled receptors (GPCRs) on the surface of living cells made possible by the use of\n",
      "fluorescent ligands in combination with a bioluminescent protein (NanoLuc) that can be readily\n",
      "expressed on the N terminus of GPCRs.\n",
      " {\n",
      "\"main_contribution\": \"A BRET approach to monitor ligand binding to GPCRs on the surface of living cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method based on Bioluminescence Resonance Energy Transfer (BRET) to monitor ligand binding to G protein-coupled receptors (GPCRs) on the surface of living cells. This method utilizes fluorescent ligands in combination with a bioluminescent protein (NanoLuc) that can be readily expressed on the N terminus of GPCRs, allowing for real-time monitoring of ligand binding to GPCRs. Therefore, the main contribution is the development of an imaging technique or method, and the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1047\n",
      "26076424\n",
      "Quantitative evaluation of software packages for single-molecule localization microscopy.\n",
      "The quality of super-resolution images obtained by single-molecule localization microscopy (SMLM)\n",
      "depends largely on the software used to detect and accurately localize point sources. In this work,\n",
      "we focus on the computational aspects of super-resolution microscopy and present a comprehensive\n",
      "evaluation of localization software packages. Our philosophy is to evaluate each package as a whole,\n",
      "thus maintaining the integrity of the software. We prepared synthetic data that represent three-\n",
      "dimensional structures modeled after biological components, taking excitation parameters, noise\n",
      "sources, point-spread functions and pixelation into account. We then asked developers to run their\n",
      "software on our data; most responded favorably, allowing us to present a broad picture of the\n",
      "methods available. We evaluated their results using quantitative and user-interpretable criteria:\n",
      "detection rate, accuracy, quality of image reconstruction, resolution, software usability and\n",
      "computational resources. These metrics reflect the various tradeoffs of SMLM software packages and\n",
      "help users to choose the software that fits their needs.\n",
      " {\n",
      "\"main_contribution\": \"A comprehensive evaluation of localization software packages for single-molecule localization microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the evaluation of software packages for single-molecule localization microscopy, which falls under the category of imaging techniques. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1048\n",
      "26098019\n",
      "Single-cell magnetic imaging using a quantum diamond microscope.\n",
      "We apply a quantum diamond microscope for detection and imaging of immunomagnetically labeled cells.\n",
      "This instrument uses nitrogen-vacancy (NV) centers in diamond for correlated magnetic and\n",
      "fluorescence imaging. Our device provides single-cell resolution and a field of view (∼1 mm(2)) two\n",
      "orders of magnitude larger than that of previous NV imaging technologies, enabling practical\n",
      "applications. To illustrate, we quantified cancer biomarkers expressed by rare tumor cells in a\n",
      "large population of healthy cells.\n",
      " {\n",
      "\"main_contribution\": \"Quantum diamond microscope for single-cell magnetic imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method based on nitrogen-vacancy centers in diamond, which enables single-cell resolution and a larger field of view than previous technologies. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1049\n",
      "26098020\n",
      "A naturally monomeric infrared fluorescent protein for protein labeling in vivo.\n",
      "Infrared fluorescent proteins (IFPs) provide an additional color to GFP and its homologs in protein\n",
      "labeling. Drawing on structural analysis of the dimer interface, we identified a bacteriophytochrome\n",
      "in the sequence database that is monomeric in truncated form and engineered it into a naturally\n",
      "monomeric IFP (mIFP). We demonstrate that mIFP correctly labels proteins in live cells, Drosophila\n",
      "and zebrafish. It should be useful in molecular, cell and developmental biology.\n",
      " {\n",
      "\"main_contribution\": \"A naturally monomeric infrared fluorescent protein for protein labeling in vivo\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a naturally monomeric infrared fluorescent protein (mIFP) for protein labeling in vivo. This is described in the abstract as \"a bacteriophytochrome in the sequence database that is monomeric in truncated form and engineered it into a naturally monomeric IFP (mIFP)\". Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1050\n",
      "26098021\n",
      "Combining protein and mRNA quantification to decipher transcriptional regulation.\n",
      "We combine immunofluorescence and single-molecule fluorescence in situ hybridization (smFISH),\n",
      "followed by automated image analysis, to quantify the concentration of nuclear transcription\n",
      "factors, number of transcription factors bound, and number of nascent mRNAs synthesized at\n",
      "individual gene loci. A theoretical model is used to decipher how transcription factor binding\n",
      "modulates the stochastic kinetics of mRNA production. We demonstrate this approach by examining the\n",
      "regulation of hunchback in the early Drosophila embryo.\n",
      " {\n",
      "\"main_contribution\": \"Combining protein and mRNA quantification to decipher transcriptional regulation\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the combination of immunofluorescence and single-molecule fluorescence in situ hybridization (smFISH) to quantify various aspects of transcriptional regulation. This approach is a new imaging method, therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1051\n",
      "26121404\n",
      "Assembly and diploid architecture of an individual human genome via single-molecule technologies.\n",
      "We present the first comprehensive analysis of a diploid human genome that combines single-molecule\n",
      "sequencing with single-molecule genome maps. Our hybrid assembly markedly improves upon the\n",
      "contiguity observed from traditional shotgun sequencing approaches, with scaffold N50 values\n",
      "approaching 30 Mb, and we identified complex structural variants (SVs) missed by other high-\n",
      "throughput approaches. Furthermore, by combining Illumina short-read data with long reads, we phased\n",
      "both single-nucleotide variants and SVs, generating haplotypes with over 99% consistency with\n",
      "previous trio-based studies. Our work shows that it is now possible to integrate single-molecule and\n",
      "high-throughput sequence data to generate de novo assembled genomes that approach reference quality.\n",
      " {\n",
      "\"main_contribution\": \"Assembly and diploid architecture of an individual human genome via single-molecule technologies\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for assembling and analyzing the diploid genome of an individual human using single-molecule sequencing and mapping technologies. The main contribution of the paper is the development of this new method, which improves upon traditional shotgun sequencing approaches by generating haplotypes with high consistency and identifying complex structural variants missed by other high-throughput methods. Therefore, the main contribution is not the development of a new imaging method, and thus \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1052\n",
      "26121406\n",
      "Protein structure determination by combining sparse NMR data with evolutionary couplings.\n",
      "Accurate determination of protein structure by NMR spectroscopy is challenging for larger proteins,\n",
      "for which experimental data are often incomplete and ambiguous. Evolutionary sequence information\n",
      "together with advances in maximum entropy statistical methods provide a rich complementary source of\n",
      "structural constraints. We have developed a hybrid approach (evolutionary coupling-NMR spectroscopy;\n",
      "EC-NMR) combining sparse NMR data with evolutionary residue-residue couplings and demonstrate\n",
      "accurate structure determination for several proteins 6-41 kDa in size.\n",
      " {\n",
      "\"main_contribution\": \"Accurate determination of protein structure by NMR spectroscopy\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work as determining protein structure using a combination of sparse NMR data and evolutionary couplings. The abstract provides background information about the challenges of accurate structure determination for larger proteins, and then describes the development of a hybrid approach (EC-NMR) that combines sparse NMR data with evolutionary residue-residue couplings to achieve accurate structure determination for several proteins.\n",
      "\n",
      "Therefore, the main contribution of the paper is the development of a new method for accurately determining protein structure using a combination of NMR spectroscopy and evolutionary information. This method is not an imaging technique or method, so is_imaging_method is false. The main contribution is recorded as \"Accurate determination of protein structure by NMR spectroscopy\".\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1053\n",
      "26147917\n",
      "Multitarget super-resolution microscopy with high-density labeling by exchangeable probes.\n",
      "We have developed a multitarget super-resolution microscopy technique called image reconstruction by\n",
      "integrating exchangeable single-molecule localization (IRIS). IRIS uses protein fragment-based\n",
      "probes that directly associate with and dissociate from their targets over durations on the order of\n",
      "tens of milliseconds. By integrating single-molecule localization and sequential labeling, IRIS\n",
      "enables unprecedented labeling density along multiple cellular structures. IRIS can be used to\n",
      "discern the area-specific proximity between cytoskeletal components and focal adhesions within a\n",
      "single cell.\n",
      " {\n",
      "\"main_contribution\": \"Multitarget super-resolution microscopy technique called image reconstruction by integrating exchangeable single-molecule localization (IRIS).\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called IRIS, which stands for image reconstruction by integrating exchangeable single-molecule localization. IRIS is a multitarget super-resolution microscopy technique that uses protein fragment-based probes to label and image multiple cellular structures with high density. Therefore, the main contribution is indeed the development of a new imaging method, and the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1054\n",
      "26147918\n",
      "Single-molecule spectroscopy of protein conformational dynamics in live eukaryotic cells.\n",
      "Single-molecule methods have become widely used for quantifying the conformational heterogeneity and\n",
      "structural dynamics of biomolecules in vitro. Their application in vivo, however, has remained\n",
      "challenging owing to shortcomings in the design and reproducible delivery of labeled molecules, the\n",
      "range of applicable analysis methods, and suboptimal cell culture conditions. By addressing these\n",
      "limitations in an integrated approach, we demonstrate the feasibility of probing protein dynamics\n",
      "from milliseconds down to the nanosecond regime in live eukaryotic cells with confocal single-\n",
      "molecule Förster resonance energy transfer (FRET) spectroscopy. We illustrate the versatility of the\n",
      "approach by determining the dimensions and submicrosecond chain dynamics of an intrinsically\n",
      "disordered protein; by detecting even subtle changes in the temperature dependence of protein\n",
      "stability, including in-cell cold denaturation; and by quantifying the folding dynamics of a small\n",
      "protein. The methodology opens possibilities for assessing the effect of the cellular environment on\n",
      "biomolecular conformation, dynamics and function.\n",
      " {\n",
      "\"main_contribution\": \"probing protein dynamics from milliseconds down to the nanosecond regime in live eukaryotic cells with confocal single-molecule Förster resonance energy transfer (FRET) spectroscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for probing protein dynamics in live eukaryotic cells using single-molecule Förster resonance energy transfer (FRET) spectroscopy. This technique allows for the study of protein conformational heterogeneity and structural dynamics in vivo, which was previously challenging due to limitations in labeling molecules, analysis methods, and cell culture conditions. The main contribution of the paper is the development of this new imaging method, which is capable of detecting protein dynamics on timescales ranging from milliseconds to nanoseconds. Therefore, the main contribution is recorded as \"probing protein dynamics from milliseconds down to the nanosecond regime in live eukaryotic cells with confocal single-molecule Förster resonance energy transfer (FRET) spectroscopy\". Additionally, since this contribution is the development of a new imaging method, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1055\n",
      "26167640\n",
      "Photoactivatable genetically encoded calcium indicators for targeted neuronal imaging.\n",
      "Circuit mapping requires knowledge of both structural and functional connectivity between cells.\n",
      "Although optical tools have been made to assess either the morphology and projections of neurons or\n",
      "their activity and functional connections, few probes integrate this information. We have generated\n",
      "a family of photoactivatable genetically encoded Ca(2+) indicators that combines attributes of high-\n",
      "contrast photolabeling with high-sensitivity Ca(2+) detection in a single-color protein sensor. We\n",
      "demonstrated in cultured neurons and in fruit fly and zebrafish larvae how single cells could be\n",
      "selected out of dense populations for visualization of morphology and high signal-to-noise\n",
      "measurements of activity, synaptic transmission and connectivity. Our design strategy is\n",
      "transferrable to other sensors based on circularly permutated GFP (cpGFP).\n",
      " {\n",
      "\"main_contribution\": \"Photoactivatable genetically encoded calcium indicators for targeted neuronal imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically photoactivatable genetically encoded calcium indicators for targeted neuronal imaging. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1056\n",
      "26167641\n",
      "Continuous volumetric imaging via an optical phase-locked ultrasound lens.\n",
      "In vivo imaging at high spatiotemporal resolution is key to the understanding of complex biological\n",
      "systems. We integrated an optical phase-locked ultrasound lens into a two-photon fluorescence\n",
      "microscope and achieved microsecond-scale axial scanning, thus enabling volumetric imaging at tens\n",
      "of hertz. We applied this system to multicolor volumetric imaging of processes sensitive to motion\n",
      "artifacts, including calcium dynamics in behaving mouse brain and transient morphology changes and\n",
      "trafficking of immune cells.\n",
      " {\n",
      "\"main_contribution\": \"Continuous volumetric imaging via an optical phase-locked ultrasound lens\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically, the integration of an optical phase-locked ultrasound lens into a two-photon fluorescence microscope, enabling volumetric imaging at tens of hertz. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1057\n",
      "26167642\n",
      "Imaging G protein-coupled receptors while quantifying their ligand-binding free-energy landscape.\n",
      "Imaging native membrane receptors and testing how they interact with ligands is of fundamental\n",
      "interest in the life sciences but has proven remarkably difficult to accomplish. Here, we introduce\n",
      "an approach that uses force-distance curve-based atomic force microscopy to simultaneously image\n",
      "single native G protein-coupled receptors in membranes and quantify their dynamic binding strength\n",
      "to native and synthetic ligands. We measured kinetic and thermodynamic parameters for individual\n",
      "protease-activated receptor-1 (PAR1) molecules in the absence and presence of antagonists, and these\n",
      "measurements enabled us to describe PAR1's ligand-binding free-energy landscape with high accuracy.\n",
      "Our nanoscopic method opens an avenue to directly image and characterize ligand binding of native\n",
      "membrane receptors.\n",
      " {\n",
      "\"main_contribution\": \"Imaging native membrane receptors and quantifying their dynamic binding strength to native and synthetic ligands.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique that uses force-distance curve-based atomic force microscopy to simultaneously image single native G protein-coupled receptors in membranes and quantify their dynamic binding strength to native and synthetic ligands. This technique allows for the direct imaging and characterization of ligand binding of native membrane receptors, which has proven remarkably difficult to accomplish previously. Therefore, the main contribution is the development of a new imaging method, and the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1058\n",
      "26192083\n",
      "Inferring transient particle transport dynamics in live cells.\n",
      "Live-cell imaging and particle tracking provide rich information on mechanisms of intracellular\n",
      "transport. However, trajectory analysis procedures to infer complex transport dynamics involving\n",
      "stochastic switching between active transport and diffusive motion are lacking. We applied Bayesian\n",
      "model selection to hidden Markov modeling to infer transient transport states from trajectories of\n",
      "mRNA-protein complexes in live mouse hippocampal neurons and metaphase kinetochores in dividing\n",
      "human cells. The software is available at http://hmm-bayes.org/.\n",
      " {\n",
      "\"main_contribution\": \"Inferring transient particle transport dynamics in live cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a Bayesian model selection approach to hidden Markov modeling, which allows for the inference of transient transport states from trajectories of particles in live cells. Therefore, the value of \"is_imaging_method\" is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1059\n",
      "26192084\n",
      "Real-time imaging of brain activity in freely moving rats using functional ultrasound.\n",
      "Innovative imaging methods help to investigate the complex relationship between brain activity and\n",
      "behavior in freely moving animals. Functional ultrasound (fUS) is an imaging modality suitable for\n",
      "recording cerebral blood volume (CBV) dynamics in the whole brain but has so far been used only in\n",
      "head-fixed and anesthetized rodents. We designed a fUS device for tethered brain imaging in freely\n",
      "moving rats based on a miniaturized ultrasound probe and a custom-made ultrasound scanner. We\n",
      "monitored CBV changes in rats during various behavioral states such as quiet rest, after whisker or\n",
      "visual stimulations, and in a food-reinforced operant task. We show that fUS imaging in freely\n",
      "moving rats could efficiently decode brain activity in real time.\n",
      " {\n",
      "\"main_contribution\": \"Real-time imaging of brain activity in freely moving rats using functional ultrasound\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically functional ultrasound (fUS), for real-time imaging of brain activity in freely moving rats. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1060\n",
      "26237227\n",
      "ProteoPlex: stability optimization of macromolecular complexes by sparse-matrix screening of\n",
      "chemical space.\n",
      "Molecular machines or macromolecular complexes are supramolecular assemblies of biomolecules with a\n",
      "variety of functions. Structure determination of these complexes in a purified state is often\n",
      "tedious owing to their compositional complexity and the associated relative structural instability.\n",
      "To improve the stability of macromolecular complexes in vitro, we present a generic method that\n",
      "optimizes the stability, homogeneity and solubility of macromolecular complexes by sparse-matrix\n",
      "screening of their thermal unfolding behavior in the presence of various buffers and small\n",
      "molecules. The method includes the automated analysis of thermal unfolding curves based on a\n",
      "biophysical unfolding model for complexes. We found that under stabilizing conditions, even large\n",
      "multicomponent complexes reveal an almost ideal two-state unfolding behavior. We envisage an\n",
      "improved biochemical understanding of purified macromolecules as well as a substantial boost in\n",
      "successful macromolecular complex structure determination by both X-ray crystallography and cryo-\n",
      "electron microscopy.\n",
      " {\n",
      "\"main_contribution\": \"optimization of macromolecular complexes stability by sparse-matrix screening of chemical space\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a method for optimizing the stability of macromolecular complexes, which is not an imaging technique or method. Therefore, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1061\n",
      "26237228\n",
      "EEG and functional ultrasound imaging in mobile rats.\n",
      "We developed an integrated experimental framework that extends the brain exploration capabilities of\n",
      "functional ultrasound imaging to awake and mobile rats. In addition to acquiring hemodynamic data,\n",
      "this method further allows parallel access to electroencephalography (EEG) recordings of neuronal\n",
      "activity. We illustrate this approach with two proofs of concept: a behavioral study on theta rhythm\n",
      "activation in a maze running task and a disease-related study on spontaneous epileptic seizures.\n",
      " {\n",
      "\"main_contribution\": \"Integrated experimental framework for EEG and functional ultrasound imaging in mobile rats\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing an integrated experimental framework that extends the brain exploration capabilities of functional ultrasound imaging to awake and mobile rats, which includes parallel access to electroencephalography (EEG) recordings of neuronal activity. This framework enables the acquisition of both hemodynamic data and EEG recordings, allowing for a more comprehensive understanding of brain function in real-time. Therefore, the main contribution is classified as true for is_imaging_method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1062\n",
      "26280328\n",
      "EMRinger: side chain-directed model and map validation for 3D cryo-electron microscopy.\n",
      "Advances in high-resolution cryo-electron microscopy (cryo-EM) require the development of validation\n",
      "metrics to independently assess map quality and model geometry. We report EMRinger, a tool that\n",
      "assesses the precise fitting of an atomic model into the map during refinement and shows how\n",
      "radiation damage alters scattering from negatively charged amino acids. EMRinger\n",
      "(https://github.com/fraser-lab/EMRinger) will be useful for monitoring progress in resolving and\n",
      "modeling high-resolution features in cryo-EM.\n",
      " {\n",
      "\"main_contribution\": \"a tool that assesses the precise fitting of an atomic model into the map during refinement\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method called EMRinger, which assesses the precise fitting of an atomic model into the map during refinement. This tool is useful for monitoring progress in resolving and modeling high-resolution features in cryo-EM. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1063\n",
      "26280329\n",
      "Ultrahigh-throughput single-molecule spectroscopy and spectrally resolved super-resolution\n",
      "microscopy.\n",
      "By developing a wide-field scheme for spectral measurement and implementing photoswitching, we\n",
      "synchronously obtained the fluorescence spectra and positions of ∼10(6) single molecules in labeled\n",
      "cells in minutes, which consequently enabled spectrally resolved, 'true-color' super-resolution\n",
      "microscopy. The method, called spectrally resolved stochastic optical reconstruction microscopy (SR-\n",
      "STORM), achieved cross-talk-free three-dimensional (3D) imaging for four dyes 10 nm apart in\n",
      "emission spectrum. Excellent resolution was obtained for every channel, and 3D localizations of all\n",
      "molecules were automatically aligned within one imaging path.\n",
      " {\n",
      "\"main_contribution\": \"development of a wide-field scheme for spectral measurement and implementation of photoswitching to enable spectrally resolved, 'true-color' super-resolution microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe the development of a new imaging technique called spectrally resolved stochastic optical reconstruction microscopy (SR-STORM). This technique enables the simultaneous acquisition of fluorescence spectra and positions of approximately 10(6) single molecules in labeled cells within minutes. The main contribution of the paper is the development of this new imaging method, which is described in sentences (B), (C), and (D) of the abstract. Therefore, the main contribution is recorded as a true value in the variable is_imaging_method.\n",
      "\n",
      "The noun phrase \"development of a wide-field scheme for spectral measurement and implementation of photoswitching to enable spectrally resolved, 'true-color' super-resolution microscopy\" accurately summarizes the main contribution of the paper in 12 words or less.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1064\n",
      "26280330\n",
      "Wirelessly powered, fully internal optogenetics for brain, spinal and peripheral circuits in mice.\n",
      "To enable sophisticated optogenetic manipulation of neural circuits throughout the nervous system\n",
      "with limited disruption of animal behavior, light-delivery systems beyond fiber optic tethering and\n",
      "large, head-mounted wireless receivers are desirable. We report the development of an easy-to-\n",
      "construct, implantable wireless optogenetic device. Our smallest version (20 mg, 10 mm(3)) is two\n",
      "orders of magnitude smaller than previously reported wireless optogenetic systems, allowing the\n",
      "entire device to be implanted subcutaneously. With a radio-frequency (RF) power source and\n",
      "controller, this implant produces sufficient light power for optogenetic stimulation with minimal\n",
      "tissue heating (<1 °C). We show how three adaptations of the implant allow for untethered\n",
      "optogenetic control throughout the nervous system (brain, spinal cord and peripheral nerve endings)\n",
      "of behaving mice. This technology opens the door for optogenetic experiments in which animals are\n",
      "able to behave naturally with optogenetic manipulation of both central and peripheral targets.\n",
      " {\n",
      "\"main_contribution\": \"Wirelessly powered, fully internal optogenetics for brain, spinal, and peripheral circuits in mice\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a wireless optogenetic device that can be implanted subcutaneously in mice, allowing for untethered optogenetic control throughout the nervous system. This device is two orders of magnitude smaller than previously reported wireless optogenetic systems and produces sufficient light power for optogenetic stimulation with minimal tissue heating. The development of this device is a significant contribution to the field of imaging and microscopy, as it enables the manipulation of neural circuits in behaving animals without the need for fiber optic tethering or large, head-mounted wireless receivers. Therefore, the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1065\n",
      "26301841\n",
      "Oscope identifies oscillatory genes in unsynchronized single-cell RNA-seq experiments.\n",
      "Oscillatory gene expression is fundamental to development, but technologies for monitoring\n",
      "expression oscillations are limited. We have developed a statistical approach called Oscope to\n",
      "identify and characterize the transcriptional dynamics of oscillating genes in single-cell RNA-seq\n",
      "data from an unsynchronized cell population. Applying Oscope to a number of data sets, we\n",
      "demonstrated its utility and also identified a potential artifact in the Fluidigm C1 platform.\n",
      " {\n",
      "\"main_contribution\": \"identify and characterize transcriptional dynamics of oscillating genes\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work's goal (B) and method (C). The abstract provides background information (A), describes the method (C), and discusses the results (D) and conclusions (E).\n",
      "\n",
      "The main contribution of the paper is identifying and characterizing transcriptional dynamics of oscillating genes, which is not a new imaging method. Therefore, \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1066\n",
      "26301842\n",
      "Trajectories of cell-cycle progression from fixed cell populations.\n",
      "An accurate dissection of sources of cell-to-cell variability is crucial for quantitative biology at\n",
      "the single-cell level but has been challenging for the cell cycle. We present Cycler, a robust\n",
      "method that constructs a continuous trajectory of cell-cycle progression from images of fixed cells.\n",
      "Cycler handles heterogeneous microenvironments and does not require perturbations or genetic\n",
      "markers, making it generally applicable to quantifying multiple sources of cell-to-cell variability\n",
      "in mammalian cells.\n",
      " {\n",
      "\"main_contribution\": \"A robust method for constructing a continuous trajectory of cell-cycle progression from images of fixed cells\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1067\n",
      "26322836\n",
      "Programmed synthesis of three-dimensional tissues.\n",
      "Reconstituting tissues from their cellular building blocks facilitates the modeling of\n",
      "morphogenesis, homeostasis and disease in vitro. Here we describe DNA-programmed assembly of cells\n",
      "(DPAC), a method to reconstitute the multicellular organization of organoid-like tissues having\n",
      "programmed size, shape, composition and spatial heterogeneity. DPAC uses dissociated cells that are\n",
      "chemically functionalized with degradable oligonucleotide 'Velcro', allowing rapid, specific and\n",
      "reversible cell adhesion to other surfaces coated with complementary DNA sequences. DNA-patterned\n",
      "substrates function as removable and adhesive templates, and layer-by-layer DNA-programmed assembly\n",
      "builds arrays of tissues into the third dimension above the template. DNase releases completed\n",
      "arrays of organoid-like microtissues from the template concomitant with full embedding in a variety\n",
      "of extracellular matrix (ECM) gels. DPAC positions subpopulations of cells with single-cell spatial\n",
      "resolution and generates cultures several centimeters long. We used DPAC to explore the impact of\n",
      "ECM composition, heterotypic cell-cell interactions and patterns of signaling heterogeneity on\n",
      "collective cell behaviors.\n",
      " {\n",
      "\"main_contribution\": \"Reconstituting tissues from their cellular building blocks facilitates the modeling of morphogenesis, homeostasis and disease in vitro\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new method for reconstituting tissues from their cellular building blocks, which allows for the modeling of morphogenesis, homeostasis, and disease in vitro. This method, called DNA-programmed assembly of cells (DPAC), uses dissociated cells that are chemically functionalized with degradable oligonucleotide \"Velcro,\" allowing rapid, specific, and reversible cell adhesion to other surfaces coated with complementary DNA sequences. The method also uses DNA-patterned substrates as removable and adhesive templates, and layer-by-layer DNA-programmed assembly builds arrays of tissues into the third dimension above the template. DNase releases completed arrays of organoid-like microtissues from the template concomitant with full embedding in a variety of extracellular matrix (ECM) gels.\n",
      "\n",
      "While this paper does involve imaging techniques, specifically mentioning the use of DNA-patterned substrates as removable and adhesive templates, and the ability to generate cultures several centimeters long, it does not describe the development of a new imaging method or technique. Therefore, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1068\n",
      "26322839\n",
      "CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo.\n",
      "CRISPR-Cas9 technology provides a powerful system for genome engineering. However, variable activity\n",
      "across different single guide RNAs (sgRNAs) remains a significant limitation. We analyzed the\n",
      "molecular features that influence sgRNA stability, activity and loading into Cas9 in vivo. We\n",
      "observed that guanine enrichment and adenine depletion increased sgRNA stability and activity,\n",
      "whereas differential sgRNA loading, nucleosome positioning and Cas9 off-target binding were not\n",
      "major determinants. We also identified sgRNAs truncated by one or two nucleotides and containing 5'\n",
      "mismatches as efficient alternatives to canonical sgRNAs. On the basis of these results, we created\n",
      "a predictive sgRNA-scoring algorithm, CRISPRscan, that effectively captures the sequence features\n",
      "affecting the activity of CRISPR-Cas9 in vivo. Finally, we show that targeting Cas9 to the germ line\n",
      "using a Cas9-nanos 3' UTR led to the generation of maternal-zygotic mutants, as well as increased\n",
      "viability and decreased somatic mutations. These results identify determinants that influence Cas9\n",
      "activity and provide a framework for the design of highly efficient sgRNAs for genome targeting in\n",
      "vivo.\n",
      " {\n",
      "\"main_contribution\": \"identifying determinants that influence Cas9 activity and providing a framework for the design of highly efficient sgRNAs for genome targeting in vivo\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe research related to CRISPR-Cas9 technology and the development of a predictive algorithm called CRISPRscan to design highly efficient sgRNAs for genome targeting in vivo. The main contribution of the paper is identifying determinants that influence Cas9 activity and providing a framework for the design of highly efficient sgRNAs, which is not an imaging method. Therefore, the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1069\n",
      "26344046\n",
      "SR-Tesseler: a method to segment and quantify localization-based super-resolution microscopy data.\n",
      "Localization-based super-resolution techniques open the door to unprecedented analysis of molecular\n",
      "organization. This task often involves complex image processing adapted to the specific topology and\n",
      "quality of the image to be analyzed. Here we present a segmentation framework based on Voronoï\n",
      "tessellation constructed from the coordinates of localized molecules, implemented in freely\n",
      "available and open-source SR-Tesseler software. This method allows precise, robust and automatic\n",
      "quantification of protein organization at different scales, from the cellular level down to clusters\n",
      "of a few fluorescent markers. We validated our method on simulated data and on various biological\n",
      "experimental data of proteins labeled with genetically encoded fluorescent proteins or organic\n",
      "fluorophores. In addition to providing insight into complex protein organization, this polygon-based\n",
      "method should serve as a reference for the development of new types of quantifications, as well as\n",
      "for the optimization of existing ones.\n",
      " {\n",
      "\"main_contribution\": \"a segmentation framework based on Voronoï tessellation constructed from the coordinates of localized molecules\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a segmentation framework based on Voronoï tessellation constructed from the coordinates of localized molecules. This method allows for precise, robust, and automatic quantification of protein organization at different scales. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1070\n",
      "26436479\n",
      "Bayesian cluster identification in single-molecule localization microscopy data.\n",
      "Single-molecule localization-based super-resolution microscopy techniques such as photoactivated\n",
      "localization microscopy (PALM) and stochastic optical reconstruction microscopy (STORM) produce\n",
      "pointillist data sets of molecular coordinates. Although many algorithms exist for the\n",
      "identification and localization of molecules from raw image data, methods for analyzing the\n",
      "resulting point patterns for properties such as clustering have remained relatively under-studied.\n",
      "Here we present a model-based Bayesian approach to evaluate molecular cluster assignment proposals,\n",
      "generated in this study by analysis based on Ripley's K function. The method takes full account of\n",
      "the individual localization precisions calculated for each emitter. We validate the approach using\n",
      "simulated data, as well as experimental data on the clustering behavior of CD3ζ, a subunit of the\n",
      "CD3 T cell receptor complex, in resting and activated primary human T cells.\n",
      " {\n",
      "\"main_contribution\": \"Bayesian cluster identification in single-molecule localization microscopy data\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a Bayesian approach for analyzing pointillist data sets of molecular coordinates produced by single-molecule localization-based super-resolution microscopy techniques, such as PALM and STORM, to identify clustering behavior of molecules. This approach takes full account of the individual localization precisions calculated for each emitter and is validated using simulated and experimental data. Therefore, the main contribution is the development of a new imaging method, specifically an analysis technique for single-molecule localization microscopy data, which makes the \"is_imaging_method\" variable true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1071\n",
      "26436480\n",
      "A strategy for dissecting the architectures of native macromolecular assemblies.\n",
      "It remains particularly problematic to define the structures of native macromolecular assemblies,\n",
      "which are often of low abundance. Here we present a strategy for isolating complexes at endogenous\n",
      "levels from GFP-tagged transgenic cell lines. Using cross-linking mass spectrometry, we extracted\n",
      "distance restraints that allowed us to model the complexes' molecular architectures.\n",
      " {\n",
      "\"main_contribution\": \"A strategy for isolating complexes at endogenous levels from GFP-tagged transgenic cell lines\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new method for isolating complexes at endogenous levels from GFP-tagged transgenic cell lines. This method does not involve imaging techniques, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1072\n",
      "26436482\n",
      "Noncontact three-dimensional mapping of intracellular hydromechanical properties by Brillouin\n",
      "microscopy.\n",
      "Current measurements of the biomechanical properties of cells require physical contact with cells or\n",
      "lack subcellular resolution. Here we developed a label-free microscopy technique based on Brillouin\n",
      "light scattering that is capable of measuring an intracellular longitudinal modulus with optical\n",
      "resolution. The 3D Brillouin maps we obtained of cells in 2D and 3D microenvironments revealed\n",
      "mechanical changes due to cytoskeletal modulation and cell-volume regulation.\n",
      " {\n",
      "\"main_contribution\": \"Measuring intracellular longitudinal modulus with optical resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1073\n",
      "26457862\n",
      "Transparent intracortical microprobe array for simultaneous spatiotemporal optical stimulation and\n",
      "multichannel electrical recording.\n",
      "Optogenetics, the selective excitation or inhibition of neural circuits by light, has become a\n",
      "transformative approach for dissecting functional brain microcircuits, particularly in in vivo\n",
      "rodent models, owing to the expanding libraries of opsins and promoters. Yet there is a lack of\n",
      "versatile devices that can deliver spatiotemporally patterned light while performing simultaneous\n",
      "sensing to map the dynamics of perturbed neural populations at the network level. We have created\n",
      "optoelectronic actuator and sensor microarrays that can be used as monolithic intracortical\n",
      "implants, fabricated from an optically transparent, electrically highly conducting semiconductor ZnO\n",
      "crystal. The devices can perform simultaneous light delivery and electrical readout in precise\n",
      "spatial registry across the microprobe array. We applied the device technology in transgenic mice to\n",
      "study light-perturbed cortical microcircuit dynamics and their effects on behavior. The\n",
      "functionality of this device can be further expanded to optical imaging and patterned electrical\n",
      "microstimulation.\n",
      " {\n",
      "\"main_contribution\": \"Development of a transparent intracortical microprobe array for simultaneous spatiotemporal optical stimulation and multichannel electrical recording\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a transparent intracortical microprobe array for simultaneous spatiotemporal optical stimulation and multichannel electrical recording. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1074\n",
      "26480475\n",
      "Tissue matrix arrays for high-throughput screening and systems analysis of cell function.\n",
      "Cell and protein arrays have demonstrated remarkable utility in the high-throughput evaluation of\n",
      "biological responses; however, they lack the complexity of native tissue and organs. Here we spotted\n",
      "tissue extracellular matrix (ECM) particles as two-dimensional (2D) arrays or incorporated them with\n",
      "cells to generate three-dimensional (3D) cell-matrix microtissue arrays. We then investigated the\n",
      "responses of human stem, cancer and immune cells to tissue ECM arrays originating from 11 different\n",
      "tissues. We validated the 2D and 3D arrays as representative of the in vivo microenvironment by\n",
      "means of quantitative analysis of tissue-specific cellular responses, including matrix production,\n",
      "adhesion and proliferation, and morphological changes after culture. The biological outputs\n",
      "correlated with tissue proteomics, and network analysis identified several proteins linked to cell\n",
      "function. Our methodology enables broad screening of ECMs to connect tissue-specific composition\n",
      "with biological activity, providing a new resource for biomaterials research and further\n",
      "understanding of regeneration and disease mechanisms.\n",
      " {\n",
      "\"main_contribution\": \"Tissue matrix arrays for high-throughput screening and systems analysis of cell function\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of tissue matrix arrays for high-throughput screening and systems analysis of cell function. This is not a new imaging method, but rather a new technique for studying cellular behavior in a more native-like environment. Therefore, is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1075\n",
      "26501515\n",
      "Whole-animal functional and developmental imaging with isotropic spatial resolution.\n",
      "Imaging fast cellular dynamics across large specimens requires high resolution in all dimensions,\n",
      "high imaging speeds, good physical coverage and low photo-damage. To meet these requirements, we\n",
      "developed isotropic multiview (IsoView) light-sheet microscopy, which rapidly images large specimens\n",
      "via simultaneous light-sheet illumination and fluorescence detection along four orthogonal\n",
      "directions. Combining these four views by means of high-throughput multiview deconvolution yields\n",
      "images with high resolution in all three dimensions. We demonstrate whole-animal functional imaging\n",
      "of Drosophila larvae at a spatial resolution of 1.1-2.5 μm and temporal resolution of 2 Hz for\n",
      "several hours. We also present spatially isotropic whole-brain functional imaging in Danio rerio\n",
      "larvae and spatially isotropic multicolor imaging of fast cellular dynamics across gastrulating\n",
      "Drosophila embryos. Compared with conventional light-sheet microscopy, IsoView microscopy improves\n",
      "spatial resolution at least sevenfold and decreases resolution anisotropy at least threefold.\n",
      "Compared with existing high-resolution light-sheet techniques, IsoView microscopy effectively\n",
      "doubles the penetration depth and provides subsecond temporal resolution for specimens 400-fold\n",
      "larger than could previously be imaged.\n",
      " {\n",
      "\"main_contribution\": \"Whole-animal functional and developmental imaging with isotropic spatial resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1076\n",
      "26513553\n",
      "Focused ion beams in biology.\n",
      "A quiet revolution is under way in technologies used for nanoscale cellular imaging. Focused ion\n",
      "beams, previously restricted to the materials sciences and semiconductor fields, are rapidly\n",
      "becoming powerful tools for ultrastructural imaging of biological samples. Cell and tissue\n",
      "architecture, as preserved in plastic-embedded resin or in plunge-frozen form, can be investigated\n",
      "in three dimensions by scanning electron microscopy imaging of freshly created surfaces that result\n",
      "from the progressive removal of material using a focused ion beam. The focused ion beam can also be\n",
      "used as a sculpting tool to create specific specimen shapes such as lamellae or needles that can be\n",
      "analyzed further by transmission electron microscopy or by methods that probe chemical composition.\n",
      "Here we provide an in-depth primer to the application of focused ion beams in biology, including a\n",
      "guide to the practical aspects of using the technology, as well as selected examples of its\n",
      "contribution to the generation of new insights into subcellular architecture and mechanisms\n",
      "underlying host-pathogen interactions.\n",
      " {\n",
      "\"main_contribution\": \"A guide to the practical aspects of using focused ion beams in biology, including selected examples of its contribution to the generation of new insights into subcellular architecture and mechanisms underlying host-pathogen interactions.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is providing a guide to the practical aspects of using focused ion beams in biology, which includes examples of its contributions to the generation of new insights into subcellular architecture and mechanisms underlying host-pathogen interactions. Since the paper is discussing the use of focused ion beams as an imaging technique in biology, it is an imaging method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1077\n",
      "26524239\n",
      "Fixed single-cell transcriptomic characterization of human radial glial diversity.\n",
      "The diverse progenitors that give rise to the human neocortex have been difficult to characterize\n",
      "because progenitors, particularly radial glia (RG), are rare and are defined by a combination of\n",
      "intracellular markers, position and morphology. To circumvent these problems, we developed Fixed and\n",
      "Recovered Intact Single-cell RNA (FRISCR), a method for profiling the transcriptomes of individual\n",
      "fixed, stained and sorted cells. Using FRISCR, we profiled primary human RG that constitute only 1%\n",
      "of the midgestation cortex and classified them as ventricular zone-enriched RG (vRG) that express\n",
      "ANXA1 and CRYAB, and outer subventricular zone-localized RG (oRG) that express HOPX. Our study\n",
      "identified vRG and oRG markers and molecular profiles, an essential step for understanding human\n",
      "neocortical progenitor development. FRISCR allows targeted single-cell profiling of any tissues that\n",
      "lack live-cell markers.\n",
      " {\n",
      "\"main_contribution\": \"Fixed single-cell transcriptomic characterization of human radial glial diversity\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called Fixed and Recovered Intact Single-cell RNA (FRISCR), which allows for the profiling of individual fixed, stained, and sorted cells. This method was used to study the transcriptomes of primary human radial glia (RG) and classify them into two subtypes: ventricular zone-enriched RG (vRG) and outer subventricular zone-localized RG (oRG). Therefore, the main contribution is the development of an imaging technique, and the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1078\n",
      "26524242\n",
      "Tissue cartography: compressing bio-image data by dimensional reduction.\n",
      "The high volumes of data produced by state-of-the-art optical microscopes encumber research. We\n",
      "developed a method that reduces data size and processing time by orders of magnitude while\n",
      "disentangling signal by taking advantage of the laminar structure of many biological specimens. Our\n",
      "Image Surface Analysis Environment automatically constructs an atlas of 2D images for arbitrarily\n",
      "shaped, dynamic and possibly multilayered surfaces of interest. Built-in correction for cartographic\n",
      "distortion ensures that no information on the surface is lost, making the method suitable for\n",
      "quantitative analysis. We applied our approach to 4D imaging of a range of samples, including a\n",
      "Drosophila melanogaster embryo and a Danio rerio beating heart.\n",
      " {\n",
      "\"main_contribution\": \"A method that reduces data size and processing time by orders of magnitude while disentangling signal by taking advantage of the laminar structure of many biological specimens.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method that reduces data size and processing time while preserving information on the surface of biological specimens. This method takes advantage of the laminar structure of many biological specimens and uses dimensional reduction to compress bio-image data. Therefore, the variable \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1079\n",
      "26550774\n",
      "Multiscale photoacoustic tomography using reversibly switchable bacterial phytochrome as a near-\n",
      "infrared photochromic probe.\n",
      "Photoacoustic tomography (PAT) of genetically encoded probes allows for imaging of targeted\n",
      "biological processes deep in tissues with high spatial resolution; however, high background signals\n",
      "from blood can limit the achievable detection sensitivity. Here we describe a reversibly switchable\n",
      "nonfluorescent bacterial phytochrome for use in multiscale photoacoustic imaging, BphP1, with the\n",
      "most red-shifted absorption among genetically encoded probes. BphP1 binds a heme-derived biliverdin\n",
      "chromophore and is reversibly photoconvertible between red and near-infrared light-absorption\n",
      "states. We combined single-wavelength PAT with efficient BphP1 photoswitching, which enabled\n",
      "differential imaging with substantially decreased background signals, enhanced detection\n",
      "sensitivity, increased penetration depth and improved spatial resolution. We monitored tumor growth\n",
      "and metastasis with ∼ 100-μm resolution at depths approaching 10 mm using photoacoustic computed\n",
      "tomography, and we imaged individual cancer cells with a suboptical-diffraction resolution of ∼ 140\n",
      "nm using photoacoustic microscopy. This technology is promising for biomedical studies at several\n",
      "scales.\n",
      " {\n",
      "\"main_contribution\": \"Reversibly switchable nonfluorescent bacterial phytochrome for use in multiscale photoacoustic imaging, BphP1, with the most red-shifted absorption among genetically encoded probes.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method using a reversibly switchable nonfluorescent bacterial phytochrome probe, BphP1. This probe allows for multiscale photoacoustic imaging with high spatial resolution and low background signals. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1080\n",
      "26619013\n",
      "Concentric-flow electrokinetic injector enables serial crystallography of ribosome and photosystem\n",
      "II.\n",
      "We describe a concentric-flow electrokinetic injector for efficiently delivering microcrystals for\n",
      "serial femtosecond X-ray crystallography analysis that enables studies of challenging biological\n",
      "systems in their unadulterated mother liquor. We used the injector to analyze microcrystals of\n",
      "Geobacillus stearothermophilus thermolysin (2.2-Å structure), Thermosynechococcus elongatus\n",
      "photosystem II (<3-Å diffraction) and Thermus thermophilus small ribosomal subunit bound to the\n",
      "antibiotic paromomycin at ambient temperature (3.4-Å structure).\n",
      " {\n",
      "\"main_contribution\": \"development of a concentric-flow electrokinetic injector for delivering microcrystals for serial femtosecond X-ray crystallography analysis\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a concentric-flow electrokinetic injector for delivering microcrystals for serial femtosecond X-ray crystallography analysis. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1081\n",
      "26641311\n",
      "Three-dimensional force microscopy of cells in biopolymer networks.\n",
      "We describe a technique for the quantitative measurement of cell-generated forces in highly\n",
      "nonlinear three-dimensional biopolymer networks that mimic the physiological situation of living\n",
      "cells. We computed forces of MDA-MB-231 breast carcinoma cells from the measured network\n",
      "deformations around the cells using a finite-element approach based on a constitutive equation that\n",
      "captures the complex mechanical properties of diverse biopolymers such as collagen gels, fibrin gels\n",
      "and Matrigel. Our measurements show that breast carcinoma cells cultured in collagen gels generated\n",
      "nearly constant forces regardless of the collagen concentration and matrix stiffness. Furthermore,\n",
      "time-lapse force measurements showed that these cells migrated in a gliding motion with alternating\n",
      "phases of high and low contractility, elongation, migratory speed and persistence.\n",
      " {\n",
      "\"main_contribution\": \"The development of a technique for quantitative measurement of cell-generated forces in highly nonlinear three-dimensional biopolymer networks that mimic the physiological situation of living cells.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method, specifically a technique for measuring cell-generated forces in 3D biopolymer networks. This technique uses a finite-element approach based on a constitutive equation that captures the complex mechanical properties of diverse biopolymers, and it allows for quantitative measurement of cell forces in highly nonlinear 3D environments that mimic the physiological situation of living cells. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1082\n",
      "26657556\n",
      "TRP channel mediated neuronal activation and ablation in freely behaving zebrafish.\n",
      "The zebrafish (Danio rerio) is a useful vertebrate model system in which to study neural circuits\n",
      "and behavior, but tools to modulate neurons in freely behaving animals are limited. As poikilotherms\n",
      "that live in water, zebrafish are amenable to thermal and pharmacological perturbations. We exploit\n",
      "these properties by using transient receptor potential (TRP) channels to activate or ablate specific\n",
      "neuronal populations using the chemical and thermal agonists of heterologously expressed TRPV1,\n",
      "TRPM8 and TRPA1.\n",
      " {\n",
      "\"main_contribution\": \"The development of a method for activating or ablating specific neuronal populations in freely behaving zebrafish using TRP channels and chemical and thermal agonists.\"\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1083\n",
      "26657558\n",
      "Nanoscale optomechanical actuators for controlling mechanotransduction in living cells.\n",
      "To control receptor tension optically at the cell surface, we developed an approach involving\n",
      "optomechanical actuator nanoparticles that are controlled with near-infrared light. Illumination\n",
      "leads to particle collapse, delivering piconewton forces to specific cell surface receptors with\n",
      "high spatial and temporal resolution. We demonstrate optomechanical actuation by controlling\n",
      "integrin-based focal adhesion formation, cell protrusion and migration, and T cell receptor\n",
      "activation.\n",
      " {\n",
      "\"main_contribution\": \"Nanoscale optomechanical actuators for controlling mechanotransduction in living cells\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of nanoscale optomechanical actuators that can control mechanotransduction in living cells. This is described in the abstract as an \"approach involving optomechanical actuator nanoparticles that are controlled with near-infrared light.\" The methods used to achieve this goal are also described, including the use of illumination to deliver piconewton forces to specific cell surface receptors. However, the paper does not describe the development of a new imaging method or technique, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1084\n",
      "26657559\n",
      "Inverted light-sheet microscope for imaging mouse pre-implantation development.\n",
      "Despite its importance for understanding human infertility and congenital diseases, early mammalian\n",
      "development has remained inaccessible to in toto imaging. We developed an inverted light-sheet\n",
      "microscope that enabled us to image mouse embryos from zygote to blastocyst, computationally track\n",
      "all cells and reconstruct a complete lineage tree of mouse pre-implantation development. We used\n",
      "this unique data set to show that the first cell fate specification occurs at the 16-cell stage.\n",
      " {\n",
      "\"main_contribution\": \"imaging mouse pre-implantation development\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1085\n",
      "26716559\n",
      "How good can cryo-EM become?\n",
      "The suddenness with which single-particle cryo-electron microscopy (cryo-EM) has emerged as a method\n",
      "for determining high-resolution structures of biological macromolecules invites the questions, how\n",
      "much better can this technology get, and how fast is that likely to happen? Though we can rightly\n",
      "celebrate the maturation of cryo-EM as a high-resolution structure-determination tool, I believe\n",
      "there still are many developments to look forward to.\n",
      " {\n",
      "\"main_contribution\": \"determining high-resolution structures of biological macromolecules\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically the use of single-particle cryo-electron microscopy (cryo-EM) to determine high-resolution structures of biological macromolecules. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1086\n",
      "26778924\n",
      "Whole-brain activity mapping onto a zebrafish brain atlas.\n",
      "In order to localize the neural circuits involved in generating behaviors, it is necessary to assign\n",
      "activity onto anatomical maps of the nervous system. Using brain registration across hundreds of\n",
      "larval zebrafish, we have built an expandable open-source atlas containing molecular labels and\n",
      "definitions of anatomical regions, the Z-Brain. Using this platform and immunohistochemical\n",
      "detection of phosphorylated extracellular signal–regulated kinase (ERK) as a readout of neural\n",
      "activity, we have developed a system to create and contextualize whole-brain maps of stimulus- and\n",
      "behavior-dependent neural activity. This mitogen-activated protein kinase (MAP)-mapping assay is\n",
      "technically simple, and data analysis is completely automated. Because MAP-mapping is performed on\n",
      "freely swimming fish, it is applicable to studies of nearly any stimulus or behavior. Here we\n",
      "demonstrate our high-throughput approach using pharmacological, visual and noxious stimuli, as well\n",
      "as hunting and feeding. The resultant maps outline hundreds of areas associated with behaviors.\n",
      " {\n",
      "\"main_contribution\": \"Whole-brain activity mapping onto a zebrafish brain atlas\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically whole-brain activity mapping onto a zebrafish brain atlas. This is achieved through the use of brain registration across hundreds of larval zebrafish, resulting in an expandable open-source atlas containing molecular labels and definitions of anatomical regions, known as the Z-Brain. The authors also develop a system to create and contextualize whole-brain maps of stimulus- and behavior-dependent neural activity using immunohistochemical detection of phosphorylated extracellular signal–regulated kinase (ERK) as a readout of neural activity. Therefore, the is_imaging_method variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1087\n",
      "26780092\n",
      "Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis.\n",
      "The transcriptional state of a cell reflects a variety of biological factors, from cell-type-\n",
      "specific features to transient processes such as the cell cycle, all of which may be of interest.\n",
      "However, identifying such aspects from noisy single-cell RNA-seq data remains challenging. We\n",
      "developed pathway and gene set overdispersion analysis (PAGODA) to resolve multiple, potentially\n",
      "overlapping aspects of transcriptional heterogeneity by testing gene sets for coordinated\n",
      "variability among measured cells.\n",
      " {\n",
      "\"main_contribution\": \"Identifying transcriptional heterogeneity through pathway and gene set overdispersion analysis\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title describes the work being described as characterizing transcriptional heterogeneity. The abstract goes on to explain that the authors developed a method called PAGODA to identify aspects of transcriptional heterogeneity from noisy single-cell RNA-seq data. Therefore, the main contribution of the paper is identifying transcriptional heterogeneity through the development of the PAGODA method, which is not an imaging technique or method.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1088\n",
      "26808668\n",
      "Multi-target spectrally resolved fluorescence lifetime imaging microscopy.\n",
      "We introduce a pattern-matching technique for efficient identification of fluorophore ratios in\n",
      "complex multidimensional fluorescence signals using reference fluorescence decay and spectral\n",
      "signature patterns of individual fluorescent probes. Alternating pulsed laser excitation at three\n",
      "different wavelengths and time-resolved detection on 32 spectrally separated detection channels\n",
      "ensures efficient excitation of fluorophores and a maximum gain of fluorescence information. Using\n",
      "spectrally resolved fluorescence lifetime imaging microscopy (sFLIM), we were able to visualize up\n",
      "to nine different target molecules simultaneously in mouse C2C12 cells. By exploiting the\n",
      "sensitivity of fluorescence emission spectra and the lifetime of organic fluorophores on\n",
      "environmental factors, we carried out fluorescence imaging of three different target molecules in\n",
      "human U2OS cells with the same fluorophore. Our results demonstrate that sFLIM can be used for\n",
      "super-resolution multi-target imaging by stimulated emission depletion (STED).\n",
      " {\n",
      "\"main_contribution\": \"Multi-target spectrally resolved fluorescence lifetime imaging microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called spectrally resolved fluorescence lifetime imaging microscopy (sFLIM), which allows for multi-target imaging by stimulated emission depletion (STED). Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1089\n",
      "26808669\n",
      "A genetically targetable near-infrared photosensitizer.\n",
      "Upon illumination, photosensitizer molecules produce reactive oxygen species that can be used for\n",
      "functional manipulation of living cells, including protein inactivation, targeted-damage\n",
      "introduction and cellular ablation. Photosensitizers used to date have been either exogenous,\n",
      "resulting in delivery and removal challenges, or genetically encoded proteins that form or bind a\n",
      "native photosensitizing molecule, resulting in a constitutively active photosensitizer inside the\n",
      "cell. We describe a genetically encoded fluorogen-activating protein (FAP) that binds a heavy atom-\n",
      "substituted fluorogenic dye, forming an 'on-demand' activated photosensitizer that produces singlet\n",
      "oxygen and fluorescence when activated with near-infrared light. This targeted and activated\n",
      "photosensitizer (TAPs) approach enables protein inactivation, targeted cell killing and rapid\n",
      "targeted lineage ablation in living larval and adult zebrafish. The near-infrared excitation and\n",
      "emission of this FAP-TAPs provides a new spectral range for photosensitizer proteins that could be\n",
      "useful for imaging, manipulation and cellular ablation deep within living organisms.\n",
      " {\n",
      "\"main_contribution\": \"A genetically targetable near-infrared photosensitizer\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a genetically encoded fluorogen-activating protein (FAP) that binds a heavy atom-substituted fluorogenic dye, forming an 'on-demand' activated photosensitizer that produces singlet oxygen and fluorescence when activated with near-infrared light. This new method enables protein inactivation, targeted cell killing, and rapid targeted lineage ablation in living larval and adult zebrafish. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1090\n",
      "26808670\n",
      "Highly multiplexed simultaneous detection of RNAs and proteins in single cells.\n",
      "To enable the detection of expression signatures specific to individual cells, we developed PLAYR\n",
      "(proximity ligation assay for RNA), a method for highly multiplexed transcript quantification by\n",
      "flow and mass cytometry that is compatible with standard antibody staining. When used with mass\n",
      "cytometry, PLAYR allowed for the simultaneous quantification of more than 40 different mRNAs and\n",
      "proteins. In primary cells, we quantified multiple transcripts, with the identity and functional\n",
      "state of each analyzed cell defined on the basis of the expression of a separate set of transcripts\n",
      "or proteins. By expanding high-throughput deep phenotyping of cells beyond protein epitopes to\n",
      "include RNA expression, PLAYR opens a new avenue for the characterization of cellular metabolism.\n",
      " {\n",
      "\"main_contribution\": \"A method for highly multiplexed transcript quantification by flow and mass cytometry\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called PLAYR (proximity ligation assay for RNA), which enables the simultaneous quantification of multiple mRNAs and proteins in single cells using flow and mass cytometry. Therefore, the value of the \"is_imaging_method\" variable is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1091\n",
      "26855362\n",
      "Sensory and optogenetically driven single-vessel fMRI.\n",
      "Magnetic resonance imaging (MRI) sensitivity approaches vessel specificity. We developed a single-\n",
      "vessel functional MRI (fMRI) method to image the contribution of vascular components to blood\n",
      "oxygenation level-dependent (BOLD) and cerebral blood volume (CBV) fMRI signal. We mapped individual\n",
      "vessels penetrating the rat somatosensory cortex with 100-ms temporal resolution by MRI with sensory\n",
      "or optogenetic stimulation. The BOLD signal originated primarily from venules, and the CBV signal\n",
      "from arterioles. The single-vessel fMRI method and its combination with optogenetics provide a\n",
      "platform for mapping the hemodynamic signal through the neurovascular network with specificity at\n",
      "the level of individual arterioles and venules.\n",
      " {\n",
      "\"main_contribution\": \"A single-vessel functional MRI (fMRI) method to image the contribution of vascular components to blood oxygenation level-dependent (BOLD) and cerebral blood volume (CBV) fMRI signal.\"\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is developing a new imaging method, specifically a single-vessel functional MRI (fMRI) method. Therefore, the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1092\n",
      "26878381\n",
      "Simultaneous fast measurement of circuit dynamics at multiple sites across the mammalian brain.\n",
      "Real-time activity measurements from multiple specific cell populations and projections are likely\n",
      "to be important for understanding the brain as a dynamical system. Here we developed frame-projected\n",
      "independent-fiber photometry (FIP), which we used to record fluorescence activity signals from many\n",
      "brain regions simultaneously in freely behaving mice. We explored the versatility of the FIP\n",
      "microscope by quantifying real-time activity relationships among many brain regions during social\n",
      "behavior, simultaneously recording activity along multiple axonal pathways during sensory\n",
      "experience, performing simultaneous two-color activity recording, and applying optical perturbation\n",
      "tuned to elicit dynamics that match naturally occurring patterns observed during behavior.\n",
      " {\n",
      "\"main_contribution\": \"Simultaneous fast measurement of circuit dynamics at multiple sites across the mammalian brain\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new imaging method called frame-projected independent-fiber photometry (FIP), which allows for real-time activity measurements from multiple specific cell populations and projections in the brain. Therefore, \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1093\n",
      "26878382\n",
      "Single-molecule imaging of non-equilibrium molecular ensembles on the millisecond timescale.\n",
      "Single-molecule fluorescence microscopy is uniquely suited for detecting transient molecular\n",
      "recognition events, yet achieving the time resolution and statistics needed to realize this\n",
      "potential has proven challenging. Here we present a single-molecule imaging and analysis platform\n",
      "using scientific complementary metal-oxide semiconductor (sCMOS) detectors that enables imaging of\n",
      "15,000 individual molecules simultaneously at millisecond rates. This system enabled the detection\n",
      "of previously obscured processes relevant to the fidelity mechanism in protein synthesis.\n",
      " {\n",
      "\"main_contribution\": \"A single-molecule imaging and analysis platform using scientific complementary metal-oxide semiconductor (sCMOS) detectors\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging technique using sCMOS detectors, which allows for the simultaneous imaging of 15,000 individual molecules at millisecond rates. This technique enables the detection of previously obscured processes relevant to the fidelity mechanism in protein synthesis. Therefore, the main contribution is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1094\n",
      "26878383\n",
      "Apollo-NADP(+): a spectrally tunable family of genetically encoded sensors for NADP(+).\n",
      "NADPH-dependent antioxidant pathways have a critical role in scavenging hydrogen peroxide (H2O2)\n",
      "produced by oxidative phosphorylation. Inadequate scavenging results in H2O2 accumulation and can\n",
      "cause disease. To measure NADPH/NADP(+) redox states, we explored genetically encoded sensors based\n",
      "on steady-state fluorescence anisotropy due to FRET (fluorescence resonance energy transfer) between\n",
      "homologous fluorescent proteins (homoFRET); we refer to these sensors as Apollo sensors. We created\n",
      "an Apollo sensor for NADP(+) (Apollo-NADP(+)) that exploits NADP(+)-dependent homodimerization of\n",
      "enzymatically inactive glucose-6-phosphate dehydrogenase (G6PD). This sensor is reversible,\n",
      "responsive to glucose-stimulated metabolism and spectrally tunable for compatibility with many other\n",
      "sensors. We used Apollo-NADP(+) to study beta cells responding to oxidative stress and demonstrated\n",
      "that NADPH is significantly depleted before H2O2 accumulation by imaging a Cerulean-tagged version\n",
      "of Apollo-NADP(+) with the H2O2 sensor HyPer.\n",
      " {\n",
      "\"main_contribution\": \"Apollo-NADP(+): a spectrally tunable family of genetically encoded sensors for NADP(+).\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a genetically encoded sensor called Apollo-NADP(+) that can measure NADPH/NADP(+) redox states. This sensor is based on steady-state fluorescence anisotropy due to FRET (fluorescence resonance energy transfer) between homologous fluorescent proteins (homoFRET), and it is reversible, responsive to glucose-stimulated metabolism, and spectrally tunable for compatibility with many other sensors. The authors used Apollo-NADP(+) to study beta cells responding to oxidative stress and demonstrated that NADPH is significantly depleted before H2O2 accumulation by imaging a Cerulean-tagged version of Apollo-NADP(+) with the H2O2 sensor HyPer. Therefore, the main contribution of the paper is the development of a new imaging method, and the answer to the question is \"true\".\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1095\n",
      "26928761\n",
      "SuReSim: simulating localization microscopy experiments from ground truth models.\n",
      "Super-resolution fluorescence microscopy has become a widely used tool in many areas of research.\n",
      "However, designing and validating super-resolution experiments to address a research question in a\n",
      "technically feasible and scientifically rigorous manner remains a fundamental challenge. We\n",
      "developed SuReSim, a software tool that simulates localization data of arbitrary three-dimensional\n",
      "structures represented by ground truth models, allowing users to systematically explore how changing\n",
      "experimental parameters can affect potential imaging outcomes.\n",
      " {\n",
      "\"main_contribution\": \"a software tool that simulates localization data of arbitrary three-dimensional structures\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a software tool called SuReSim, which simulates localization data of arbitrary three-dimensional structures represented by ground truth models. This tool allows users to explore how changing experimental parameters can affect potential imaging outcomes. Since the paper's main contribution is the development of an imaging technique (simulation of localization microscopy experiments), the value of \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1096\n",
      "26928762\n",
      "One library to make them all: streamlining the creation of yeast libraries via a SWAp-Tag strategy.\n",
      "The yeast Saccharomyces cerevisiae is ideal for systematic studies relying on collections of\n",
      "modified strains (libraries). Despite the significance of yeast libraries and the immense variety of\n",
      "available tags and regulatory elements, only a few such libraries exist, as their construction is\n",
      "extremely expensive and laborious. To overcome these limitations, we developed a SWAp-Tag (SWAT)\n",
      "method that enables one parental library to be modified easily and efficiently to give rise to an\n",
      "endless variety of libraries of choice. To showcase the versatility of the SWAT approach, we\n",
      "constructed and investigated a library of ∼1,800 strains carrying SWAT-GFP modules at the amino\n",
      "termini of endomembrane proteins and then used it to create two new libraries (mCherry and seamless\n",
      "GFP). Our work demonstrates how the SWAT method allows fast and effortless creation of yeast\n",
      "libraries, opening the door to new ways of systematically studying cell biology.\n",
      " {\n",
      "\"main_contribution\": \"A streamlined method for creating yeast libraries using a SWAp-Tag strategy\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for creating yeast libraries using a SWAp-Tag strategy. This method is aimed at streamlining the process of creating modified strains (libraries) of yeast, which is important for systematic studies in cell biology. The main contribution of the paper is the development of this new method, which is described in sentences (B) and (C).\n",
      "\n",
      "The method involves using a parental library that can be easily modified to give rise to an endless variety of libraries of choice. The versatility of the SWAT approach is demonstrated by constructing and investigating a library of ∼1,800 strains carrying SWAT-GFP modules at the amino termini of endomembrane proteins, and then using it to create two new libraries (mCherry and seamless GFP).\n",
      "\n",
      "Since the main contribution of the paper is not the development of a new imaging method or technique, but rather a new method for creating yeast libraries, the value of is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1097\n",
      "26950744\n",
      "A saposin-lipoprotein nanoparticle system for membrane proteins.\n",
      "A limiting factor in membrane protein research is the ability to solubilize and stabilize such\n",
      "proteins. Detergents are used most often for solubilizing membrane proteins, but they are associated\n",
      "with protein instability and poor compatibility with structural and biophysical studies. Here we\n",
      "present a saposin-lipoprotein nanoparticle system, Salipro, which allows for the reconstitution of\n",
      "membrane proteins in a lipid environment that is stabilized by a scaffold of saposin proteins. We\n",
      "demonstrate the applicability of the method on two purified membrane protein complexes as well as by\n",
      "the direct solubilization and nanoparticle incorporation of a viral membrane protein complex from\n",
      "the virus membrane. Our approach facilitated high-resolution structural studies of the bacterial\n",
      "peptide transporter PeptTSo2 by single-particle cryo-electron microscopy (cryo-EM) and allowed us to\n",
      "stabilize the HIV envelope glycoprotein in a functional state.\n",
      " {\n",
      "\"main_contribution\": \"A saposin-lipoprotein nanoparticle system for membrane proteins\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically a saposin-lipoprotein nanoparticle system for membrane proteins. This system allows for the reconstitution of membrane proteins in a lipid environment that is stabilized by a scaffold of saposin proteins, which facilitates high-resolution structural studies of membrane proteins by single-particle cryo-electron microscopy (cryo-EM). Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1098\n",
      "26950745\n",
      "High-density three-dimensional localization microscopy across large volumes.\n",
      "Extending three-dimensional (3D) single-molecule localization microscopy away from the coverslip and\n",
      "into thicker specimens will greatly broaden its biological utility. However, because of the\n",
      "limitations of both conventional imaging modalities and conventional labeling techniques, it is a\n",
      "challenge to localize molecules in three dimensions with high precision in such samples while\n",
      "simultaneously achieving the labeling densities required for high resolution of densely crowded\n",
      "structures. Here we combined lattice light-sheet microscopy with newly developed, freely diffusing,\n",
      "cell-permeable chemical probes with targeted affinity for DNA, intracellular membranes or the plasma\n",
      "membrane. We used this combination to perform high-localization precision, ultrahigh-labeling\n",
      "density, multicolor localization microscopy in samples up to 20 μm thick, including dividing cells\n",
      "and the neuromast organ of a zebrafish embryo. We also demonstrate super-resolution correlative\n",
      "imaging with protein-specific photoactivable fluorophores, providing a mutually compatible, single-\n",
      "platform alternative to correlative light-electron microscopy over large volumes.\n",
      " {\n",
      "\"main_contribution\": \"High-localization precision, ultrahigh-labeling density, multicolor localization microscopy in samples up to 20 μm thick\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows for high-localization precision and ultrahigh labeling density in samples up to 20 μm thick. This method combines lattice light-sheet microscopy with newly developed, freely diffusing, cell-permeable chemical probes with targeted affinity for DNA, intracellular membranes or the plasma membrane. Therefore, the main contribution is indeed the development of a new imaging technique, and the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1099\n",
      "27018577\n",
      "Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP).\n",
      "As RNA-binding proteins (RBPs) play essential roles in cellular physiology by interacting with\n",
      "target RNA molecules, binding site identification by UV crosslinking and immunoprecipitation (CLIP)\n",
      "of ribonucleoprotein complexes is critical to understanding RBP function. However, current CLIP\n",
      "protocols are technically demanding and yield low-complexity libraries with high experimental\n",
      "failure rates. We have developed an enhanced CLIP (eCLIP) protocol that decreases requisite\n",
      "amplification by ∼1,000-fold, decreasing discarded PCR duplicate reads by ∼60% while maintaining\n",
      "single-nucleotide binding resolution. By simplifying the generation of paired IgG and size-matched\n",
      "input controls, eCLIP improves specificity in the discovery of authentic binding sites. We generated\n",
      "102 eCLIP experiments for 73 diverse RBPs in HepG2 and K562 cells (available at\n",
      "https://www.encodeproject.org), demonstrating that eCLIP enables large-scale and robust profiling,\n",
      "with amplification and sample requirements similar to those of ChIP-seq. eCLIP enables integrative\n",
      "analysis of diverse RBPs to reveal factor-specific profiles, common artifacts for CLIP and RNA-\n",
      "centric perspectives on RBP activity.\n",
      " {\n",
      "\"main_contribution\": \"eCLIP enables large-scale and robust profiling, with amplification and sample requirements similar to those of ChIP-seq.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an enhanced CLIP (eCLIP) protocol that simplifies the generation of paired IgG and size-matched input controls, improving specificity in the discovery of authentic binding sites. This is a new imaging method, as it involves UV crosslinking and immunoprecipitation of ribonucleoprotein complexes, which falls under the category of imaging techniques. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1100\n",
      "27018580\n",
      "Quantitative super-resolution imaging with qPAINT.\n",
      "Counting molecules in complexes is challenging, even with super-resolution microscopy. Here, we use\n",
      "the programmable and specific binding of dye-labeled DNA probes to count integer numbers of targets.\n",
      "This method, called quantitative points accumulation in nanoscale topography (qPAINT), works\n",
      "independently of dye photophysics for robust counting with high precision and accuracy over a wide\n",
      "dynamic range. qPAINT was benchmarked on DNA nanostructures and demonstrated for cellular\n",
      "applications by quantifying proteins in situ and the number of single-molecule FISH probes bound to\n",
      "an mRNA target.\n",
      " {\n",
      "\"main_contribution\": \"Quantitative super-resolution imaging method called qPAINT\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called qPAINT, which allows for quantitative super-resolution imaging. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1101\n",
      "27064647\n",
      "Expansion microscopy with conventional antibodies and fluorescent proteins.\n",
      "Expansion microscopy is a technique in which fluorophores on fixed specimens are linked to a\n",
      "swellable polymer that is physically expanded to enable super-resolution microscopy with ordinary\n",
      "microscopes. We have developed and characterized new methods for linking fluorophores to the polymer\n",
      "that now enable expansion microscopy with conventional fluorescently labeled antibodies and\n",
      "fluorescent proteins. Our methods simplify the procedure and expand the palette of compatible\n",
      "labels, allowing rapid dissemination of the technique.\n",
      " {\n",
      "\"main_contribution\": \"Expansion microscopy with conventional antibodies and fluorescent proteins\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically expansion microscopy, which enables super-resolution microscopy with ordinary microscopes. This technique involves linking fluorophores to a swellable polymer that is physically expanded, allowing for the use of conventional fluorescently labeled antibodies and fluorescent proteins. Therefore, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1102\n",
      "27088314\n",
      "siFLIM: single-image frequency-domain FLIM provides fast and photon-efficient lifetime data.\n",
      "We developed single-image fluorescence lifetime imaging microscopy (siFLIM), a method for acquiring\n",
      "quantitative lifetime images from a single exposure. siFLIM takes advantage of a new generation of\n",
      "dedicated cameras that simultaneously record two 180°-phase-shifted images, and it allows for video-\n",
      "rate lifetime imaging with minimal phototoxicity and bleaching. siFLIM is also inherently immune to\n",
      "artifacts stemming from rapid cellular movements and signal transients.\n",
      " {\n",
      "\"main_contribution\": \"siFLIM: single-image frequency-domain FLIM provides fast and photon-efficient lifetime data\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called siFLIM, which allows for fast and photon-efficient lifetime data acquisition. Therefore, the value of is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1103\n",
      "27110629\n",
      "The development of cryo-EM into a mainstream  structural biology technique.\n",
      "Single-particle cryo-electron microscopy (cryo-EM) has emerged over the last two decades as a\n",
      "technique capable of studying challenging systems that otherwise defy structural characterization.\n",
      "Recent technical advances have resulted in a 'quantum leap' in applicability, throughput and\n",
      "achievable resolution that has gained this technique worldwide attention. Here I discuss some of the\n",
      "major historical landmarks in the development of the cryo-EM field, ultimately leading to its\n",
      "present success.\n",
      " {\n",
      "\"main_contribution\": \"The development of cryo-EM into a mainstream structural biology technique\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1104\n",
      "27111507\n",
      "Automated structure modeling of large protein assemblies using crosslinks as distance restraints.\n",
      "Crosslinking mass spectrometry is increasingly used for structural characterization of multisubunit\n",
      "protein complexes. Chemical crosslinking captures conformational heterogeneity, which typically\n",
      "results in conflicting crosslinks that cannot be satisfied in a single model, making detailed\n",
      "modeling a challenging task. Here we introduce an automated modeling method dedicated to large\n",
      "protein assemblies ('XL-MOD' software is available at http://aria.pasteur.fr/supplementary-\n",
      "data/x-links) that (i) uses a form of spatial restraints that realistically reflects the\n",
      "distribution of experimentally observed crosslinked distances; (ii) automatically deals with\n",
      "ambiguous and/or conflicting crosslinks and identifies alternative conformations within a Bayesian\n",
      "framework; and (iii) allows subunit structures to be flexible during conformational sampling. We\n",
      "demonstrate our method by testing it on known structures and available crosslinking data. We also\n",
      "crosslinked and modeled the 17-subunit yeast RNA polymerase III at atomic resolution; the resulting\n",
      "model agrees remarkably well with recently published cryoelectron microscopy structures and provides\n",
      "additional insights into the polymerase structure.\n",
      " {\n",
      "\"main_contribution\": \"Automated structure modeling of large protein assemblies using crosslinks as distance restraints\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method, specifically an automated modeling method dedicated to large protein assemblies that uses crosslinks as distance restraints. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1105\n",
      "27123817\n",
      "Measuring cell-generated forces: a guide to the available tools.\n",
      "Forces generated by cells are critical regulators of cell adhesion, signaling, and function, and\n",
      "they are also essential drivers in the morphogenetic events of development. Over the past 20 years,\n",
      "several methods have been developed to measure these forces. However, despite recent substantial\n",
      "interest in understanding the contribution of these forces in biology, implementation and adoption\n",
      "of the developed methods by the broader biological community remain challenging because of the\n",
      "inherently multidisciplinary expertise required to conduct and interpret the measurements. In this\n",
      "review, we introduce the established methods and highlight the technical challenges associated with\n",
      "implementing each technique in a biological laboratory.\n",
      " {\n",
      "\"main_contribution\": \"A guide to available tools for measuring cell-generated forces\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1106\n",
      "27135972\n",
      "Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs.\n",
      "Drug sensitivity and resistance are conventionally quantified by IC50 or Emax values, but these\n",
      "metrics are highly sensitive to the number of divisions taking place over the course of a response\n",
      "assay. The dependency of IC50 and Emax on division rate creates artefactual correlations between\n",
      "genotype and drug sensitivity, while obscuring valuable biological insights and interfering with\n",
      "biomarker discovery. We derive alternative small molecule drug-response metrics that are insensitive\n",
      "to division number. These are based on estimation of the magnitude of drug-induced growth rate\n",
      "inhibition (GR) using endpoint or time-course assays. We show that GR50 and GRmax are superior to\n",
      "conventional metrics for assessing the effects of small molecule drugs in dividing cells. Moreover,\n",
      "adopting GR metrics requires only modest changes in experimental protocols. We expect GR metrics to\n",
      "improve the study of cell signaling and growth using small molecules and biologics and to facilitate\n",
      "the discovery of drug-response biomarkers and the identification of drugs effective against specific\n",
      "patient-derived tumor cells.\n",
      " {\n",
      "\"main_contribution\": \"Growth rate inhibition metrics for measuring sensitivity to cancer drugs\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new method for measuring drug sensitivity in cancer cells, specifically the use of growth rate inhibition metrics. This is described in the abstract as an alternative to conventional metrics such as IC50 and Emax, which are affected by the number of divisions taking place over the course of a response assay. The authors argue that their proposed method is superior to conventional metrics for assessing the effects of small molecule drugs in dividing cells. Therefore, the main contribution of the paper is not the development of an imaging technique or method, and hence is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1107\n",
      "27135974\n",
      "An unbiased metric of antiproliferative drug effect in vitro.\n",
      "In vitro cell proliferation assays are widely used in pharmacology, molecular biology, and drug\n",
      "discovery. Using theoretical modeling and experimentation, we show that current metrics of\n",
      "antiproliferative small molecule effect suffer from time-dependent bias, leading to inaccurate\n",
      "assessments of parameters such as drug potency and efficacy. We propose the drug-induced\n",
      "proliferation (DIP) rate, the slope of the line on a plot of cell population doublings versus time,\n",
      "as an alternative, time-independent metric.\n",
      " {\n",
      "\"main_contribution\": \"An unbiased metric of antiproliferative drug effect in vitro\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new method for measuring the effectiveness of antiproliferative drugs in vitro, specifically the drug-induced proliferation (DIP) rate. This is not an imaging technique or method, so is_imaging_method is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1108\n",
      "27159085\n",
      "A bacterial phytochrome-based optogenetic system controllable with near-infrared light.\n",
      "Light-mediated control of protein-protein interactions to regulate cellular pathways is an important\n",
      "application of optogenetics. Here, we report an optogenetic system based on the reversible light-\n",
      "induced binding between the bacterial phytochrome BphP1 and its natural partner PpsR2 from\n",
      "Rhodopseudomonas palustris bacteria. We extensively characterized the BphP1-PpsR2 interaction both\n",
      "in vitro and in mammalian cells and then used this interaction to translocate target proteins to\n",
      "specific cellular compartments, such as the plasma membrane and the nucleus. We showed light-\n",
      "inducible control of cell morphology that resulted in a substantial increase of the cell area. We\n",
      "demonstrated light-dependent gene expression with 40-fold contrast in cultured cells, 32-fold in\n",
      "subcutaneous mouse tissue, and 5.7-fold in deep tissues in mice. Characteristics of the BphP1-PpsR2\n",
      "optogenetic system include its sensitivity to 740- to 780-nm near-infrared light, its ability to\n",
      "utilize an endogenous biliverdin chromophore in eukaryotes (including mammals), and its spectral\n",
      "compatibility with blue-light-driven optogenetic systems.\n",
      " {\n",
      "\"main_contribution\": \"A bacterial phytochrome-based optogenetic system controllable with near-infrared light\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new optogenetic method based on the reversible light-induced binding between the bacterial phytochrome BphP1 and its natural partner PpsR2. This system utilizes near-infrared light to control protein-protein interactions and regulate cellular pathways, and it has been extensively characterized both in vitro and in mammalian cells. The method has been demonstrated to be effective in translocating target proteins to specific cellular compartments, controlling cell morphology, and regulating gene expression with high contrast. Therefore, the main contribution of the paper is the development of a new imaging method, and the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1109\n",
      "27183441\n",
      "Flyception: imaging brain activity in freely walking fruit flies.\n",
      "Genetically encoded calcium sensors have enabled monitoring of neural activity in vivo using optical\n",
      "imaging techniques. Linking neural activity to complex behavior remains challenging, however, as\n",
      "most imaging systems require tethering the animal, which can impact the animal's behavioral\n",
      "repertoire. Here, we report a method for monitoring the brain activity of untethered, freely walking\n",
      "Drosophila melanogaster during sensorially and socially evoked behaviors to facilitate the study of\n",
      "neural mechanisms that underlie naturalistic behaviors.\n",
      " {\n",
      "\"main_contribution\": \"monitoring brain activity in freely walking fruit flies\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is monitoring brain activity in freely walking fruit flies, which can be done using a new imaging method. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1110\n",
      "27240257\n",
      "Quantitative assessment of fluorescent proteins.\n",
      "The advent of fluorescent proteins (FPs) for genetic labeling of molecules and cells has\n",
      "revolutionized fluorescence microscopy. Genetic manipulations have created a vast array of bright\n",
      "and stable FPs spanning blue to red spectral regions. Common to autofluorescent FPs is their tight\n",
      "β-barrel structure, which provides the rigidity and chemical environment needed for effectual\n",
      "fluorescence. Despite the common structure, each FP has unique properties. Thus, there is no single\n",
      "'best' FP for every circumstance, and each FP has advantages and disadvantages. To guide decisions\n",
      "about which FP is right for a given application, we have quantitatively characterized the\n",
      "brightness, photostability, pH stability and monomeric properties of more than 40 FPs to enable\n",
      "straightforward and direct comparison between them. We focus on popular and/or top-performing FPs in\n",
      "each spectral region.\n",
      " {\n",
      "\"main_contribution\": \"Quantitative assessment of fluorescent proteins\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the quantitative assessment of fluorescent proteins, which falls under the category of imaging techniques. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1111\n",
      "27271196\n",
      "An E3-ligase-based method for ablating inhibitory synapses.\n",
      "Although neuronal activity can be modulated using a variety of techniques, there are currently few\n",
      "methods for controlling neuronal connectivity. We introduce a tool (GFE3) that mediates the fast,\n",
      "specific and reversible elimination of inhibitory synaptic inputs onto genetically determined\n",
      "neurons. GFE3 is a fusion between an E3 ligase, which mediates the ubiquitination and rapid\n",
      "degradation of proteins, and a recombinant, antibody-like protein (FingR) that binds to gephyrin.\n",
      "Expression of GFE3 leads to a strong and specific reduction of gephyrin in culture or in vivo and to\n",
      "a substantial decrease in phasic inhibition onto cells that express GFE3. By temporarily expressing\n",
      "GFE3 we showed that inhibitory synapses regrow following ablation. Thus, we have created a simple,\n",
      "reversible method for modulating inhibitory synaptic input onto genetically determined cells.\n",
      " {\n",
      "\"main_contribution\": \"A tool (GFE3) that mediates the fast, specific and reversible elimination of inhibitory synaptic inputs onto genetically determined neurons.\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for ablating inhibitory synapses using an E3-ligase-based tool called GFE3. The method is specific, fast, and reversible, and it allows for the modulation of inhibitory synaptic input onto genetically determined cells.\n",
      "\n",
      "The sentences that describe the background information are:\n",
      "\n",
      "* \"Although neuronal activity can be modulated using a variety of techniques, there are currently few methods for controlling neuronal connectivity.\"\n",
      "\n",
      "The sentences that describe the goals of the work are:\n",
      "\n",
      "* \"We introduce a tool (GFE3) that mediates the fast, specific and reversible elimination of inhibitory synaptic inputs onto genetically determined neurons.\"\n",
      "\n",
      "The sentences that describe the methods used to achieve the goals are:\n",
      "\n",
      "* \"GFE3 is a fusion between an E3 ligase, which mediates the ubiquitination and rapid degradation of proteins, and a recombinant, antibody-like protein (FingR) that binds to gephyrin.\"\n",
      "* \"Expression of GFE3 leads to a strong and specific reduction of gephyrin in culture or in vivo and to a substantial decrease in phasic inhibition onto cells that express GFE3.\"\n",
      "\n",
      "The sentences that describe the results of the work are:\n",
      "\n",
      "* \"By temporarily expressing GFE3 we showed that inhibitory synapses regrow following ablation.\"\n",
      "\n",
      "The sentence that describes the conclusion of the work is:\n",
      "\n",
      "* \"Thus, we have created a simple, reversible method for modulating inhibitory synaptic input onto genetically determined cells.\"\n",
      "\n",
      "Based on the information provided, the main contribution of the paper is the development of a new tool (GFE3) for ablating inhibitory synapses. This tool is specific, fast, and reversible, and it allows for the modulation of inhibitory synaptic input onto genetically determined cells. Therefore, the main contribution of the paper is not the development of a new imaging method, and the value of the variable is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1112\n",
      "27271198\n",
      "Dense transcript profiling in single cells by image correlation decoding.\n",
      "Sequential barcoded fluorescent in situ hybridization (seqFISH) allows large numbers of molecular\n",
      "species to be accurately detected in single cells, but multiplexing is limited by the density of\n",
      "barcoded objects. We present correlation FISH (corrFISH), a method to resolve dense temporal\n",
      "barcodes in sequential hybridization experiments. Using corrFISH, we quantified highly expressed\n",
      "ribosomal protein genes in single cultured cells and mouse thymus sections, revealing cell-type-\n",
      "specific gene expression.\n",
      " {\n",
      "\"main_contribution\": \"quantified highly expressed ribosomal protein genes in single cultured cells and mouse thymus sections\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called correlation FISH (corrFISH) that allows for the resolution of dense temporal barcodes in sequential hybridization experiments, enabling the quantification of highly expressed ribosomal protein genes in single cultured cells and mouse thymus sections. Therefore, is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1113\n",
      "27295310\n",
      "Varying label density allows artifact-free analysis of membrane-protein nanoclusters.\n",
      "We present a method to robustly discriminate clustered from randomly distributed molecules detected\n",
      "with techniques based on single-molecule localization microscopy, such as PALM and STORM. The\n",
      "approach is based on deliberate variation of labeling density, such as titration of fluorescent\n",
      "antibody, combined with quantitative cluster analysis, and it thereby circumvents the problem of\n",
      "cluster artifacts generated by overcounting of blinking fluorophores. The method was used to analyze\n",
      "nanocluster formation in resting and activated immune cells.\n",
      " {\n",
      "\"main_contribution\": \"a method to robustly discriminate clustered from randomly distributed molecules detected with techniques based on single-molecule localization microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method that allows for the robust discrimination of clustered vs. randomly distributed molecules in single-molecule localization microscopy techniques such as PALM and STORM. This method involves deliberate variation of labeling density, combined with quantitative cluster analysis, and thereby circumvents the problem of cluster artifacts generated by overcounting of blinking fluorophores. \n",
      "\n",
      "Therefore, the main contribution is the development of a new imaging technique, and thus \"is_imaging_method\" is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1114\n",
      "27376770\n",
      "Nanoscale imaging of RNA with expansion microscopy.\n",
      "The ability to image RNA identity and location with nanoscale precision in intact tissues is of\n",
      "great interest for defining cell types and states in normal and pathological biological settings.\n",
      "Here, we present a strategy for expansion microscopy of RNA. We developed a small-molecule linker\n",
      "that enables RNA to be covalently attached to a swellable polyelectrolyte gel synthesized throughout\n",
      "a biological specimen. Then, postexpansion, fluorescent in situ hybridization (FISH) imaging of RNA\n",
      "can be performed with high yield and specificity as well as single-molecule precision in both\n",
      "cultured cells and intact brain tissue. Expansion FISH (ExFISH) separates RNAs and supports\n",
      "amplification of single-molecule signals (i.e., via hybridization chain reaction) as well as\n",
      "multiplexed RNA FISH readout. ExFISH thus enables super-resolution imaging of RNA structure and\n",
      "location with diffraction-limited microscopes in thick specimens, such as intact brain tissue and\n",
      "other tissues of importance to biology and medicine.\n",
      " {\n",
      "\"main_contribution\": \"Nanoscale imaging of RNA with expansion microscopy\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called expansion microscopy, which enables nanoscale imaging of RNA in intact tissues. Therefore, the value of is_imaging_method is true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1115\n",
      "27400419\n",
      "Reversible cryo-arrest for imaging molecules in living cells at high spatial resolution.\n",
      "The dynamics of molecules in living cells hampers precise imaging of molecular patterns by\n",
      "functional and super-resolution microscopy. We developed a method that circumvents lethal chemical\n",
      "fixation and allows on-stage cryo-arrest for consecutive imaging of molecular patterns within the\n",
      "same living, but arrested, cells. The reversibility of consecutive cryo-arrests was demonstrated by\n",
      "the high survival rate of different cell lines and by intact growth factor signaling that was not\n",
      "perturbed by stress response. Reversible cryo-arrest was applied to study the evolution of ligand-\n",
      "induced receptor tyrosine kinase activation at different scales. The nanoscale clustering of\n",
      "epidermal growth factor receptor (EGFR) in the plasma membrane was assessed by single-molecule\n",
      "localization microscopy, and endosomal microscale activity patterns of ephrin receptor A2 (EphA2)\n",
      "were assessed by fluorescence lifetime imaging microscopy. Reversible cryo-arrest allows the precise\n",
      "determination of molecular patterns while conserving the dynamic capabilities of living cells.\n",
      " {\n",
      "\"main_contribution\": \"Reversible cryo-arrest for imaging molecules in living cells at high spatial resolution\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new imaging method called reversible cryo-arrest, which allows for consecutive imaging of molecular patterns within the same living cells while maintaining their dynamic capabilities. Therefore, is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1116\n",
      "27427858\n",
      "LOVTRAP: an optogenetic system for photoinduced protein dissociation.\n",
      "LOVTRAP is an optogenetic approach for reversible light-induced protein dissociation using protein A\n",
      "fragments that bind to the LOV domain only in the dark, with tunable kinetics and a >150-fold change\n",
      "in the dissociation constant (Kd). By reversibly sequestering proteins at mitochondria, we precisely\n",
      "modulated the proteins' access to the cell edge, demonstrating a naturally occurring 3-mHz cell-edge\n",
      "oscillation driven by interactions of Vav2, Rac1, and PI3K proteins.\n",
      " {\n",
      "\"main_contribution\": \"an optogenetic system for photoinduced protein dissociation\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of an optogenetic system called LOVTRAP, which enables reversible light-induced protein dissociation. This system uses protein A fragments that bind to the LOV domain only in the dark and has tunable kinetics and a significant change in the dissociation constant. The authors demonstrate the utility of this system by modulating the access of proteins to the cell edge and uncovering a naturally occurring oscillation driven by interactions of Vav2, Rac1, and PI3K proteins.\n",
      "\n",
      "Since the main contribution is not the development of a new imaging method, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1117\n",
      "27467726\n",
      "A practical guide to photoacoustic tomography in the life sciences.\n",
      "The life sciences can benefit greatly from imaging technologies that connect microscopic discoveries\n",
      "with macroscopic observations. One technology uniquely positioned to provide such benefits is\n",
      "photoacoustic tomography (PAT), a sensitive modality for imaging optical absorption contrast over a\n",
      "range of spatial scales at high speed. In PAT, endogenous contrast reveals a tissue's anatomical,\n",
      "functional, metabolic, and histologic properties, and exogenous contrast provides molecular and\n",
      "cellular specificity. The spatial scale of PAT covers organelles, cells, tissues, organs, and small\n",
      "animals. Consequently, PAT is complementary to other imaging modalities in contrast mechanism,\n",
      "penetration, spatial resolution, and temporal resolution. We review the fundamentals of PAT and\n",
      "provide practical guidelines for matching PAT systems with research needs. We also summarize the\n",
      "most promising biomedical applications of PAT, discuss related challenges, and envision PAT's\n",
      "potential to lead to further breakthroughs.\n",
      " {\n",
      "\"main_contribution\": \"A practical guide to photoacoustic tomography in the life sciences\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1118\n",
      "27467727\n",
      "Contrast agents for molecular photoacoustic imaging.\n",
      "Photoacoustic imaging (PAI) is an emerging tool that bridges the traditional depth limits of\n",
      "ballistic optical imaging and the resolution limits of diffuse optical imaging. Using the acoustic\n",
      "waves generated in response to the absorption of pulsed laser light, it provides noninvasive images\n",
      "of absorbed optical energy density at depths of several centimeters with a resolution of ∼100 μm.\n",
      "This versatile and scalable imaging modality has now shown potential for molecular imaging, which\n",
      "enables visualization of biological processes with systemically introduced contrast agents.\n",
      "Understanding the relative merits of the vast range of contrast agents available, from small-\n",
      "molecule dyes to gold and carbon nanostructures to liposome encapsulations, is a considerable\n",
      "challenge. Here we critically review the physical, chemical and biochemical characteristics of the\n",
      "existing photoacoustic contrast agents, highlighting key applications and present challenges for\n",
      "molecular PAI.\n",
      " {\n",
      "\"main_contribution\": \"Developing a new imaging technique or method\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1119\n",
      "27479327\n",
      "Revealing disease-associated pathways by network integration of untargeted metabolomics.\n",
      "Uncovering the molecular context of dysregulated metabolites is crucial to understand pathogenic\n",
      "pathways. However, their system-level analysis has been limited owing to challenges in global\n",
      "metabolite identification. Most metabolite features detected by untargeted metabolomics carried out\n",
      "by liquid-chromatography-mass spectrometry cannot be uniquely identified without additional, time-\n",
      "consuming experiments. We report a network-based approach, prize-collecting Steiner forest algorithm\n",
      "for integrative analysis of untargeted metabolomics (PIUMet), that infers molecular pathways and\n",
      "components via integrative analysis of metabolite features, without requiring their identification.\n",
      "We demonstrated PIUMet by analyzing changes in metabolism of sphingolipids, fatty acids and steroids\n",
      "in a Huntington's disease model. Additionally, PIUMet enabled us to elucidate putative identities of\n",
      "altered metabolite features in diseased cells, and infer experimentally undetected, disease-\n",
      "associated metabolites and dysregulated proteins. Finally, we established PIUMet's ability for\n",
      "integrative analysis of untargeted metabolomics data with proteomics data, demonstrating that this\n",
      "approach elicits disease-associated metabolites and proteins that cannot be inferred by individual\n",
      "analysis of these data.\n",
      " {\n",
      "\"main_contribution\": \"A network-based approach for integrative analysis of untargeted metabolomics data\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new approach for analyzing untargeted metabolomics data called PIUMet. This approach integrates metabolite features and infer molecular pathways and components without requiring their identification. The main contribution of the paper is the development of this network-based approach, which can analyze changes in metabolism and elucidate putative identities of altered metabolites features in diseased cells. Additionally, PIUMet can integrate proteomics data, demonstrating that this approach elicits disease-associated metabolites and proteins that cannot be inferred by individual analysis of these data. Therefore, the main contribution is not the development of a new imaging method, and thus, is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1120\n",
      "27479328\n",
      "A far-red fluorescent protein evolved from a cyanobacterial phycobiliprotein.\n",
      "Far-red fluorescent proteins (FPs) are desirable for in vivo imaging because with these molecules\n",
      "less light is scattered, absorbed, or re-emitted by endogenous biomolecules compared with cyan,\n",
      "green, yellow, and orange FPs. We developed a new class of FP from an allophycocyanin α-subunit\n",
      "(APCα). Native APC requires a lyase to incorporate phycocyanobilin. The evolved FP, which we named\n",
      "small ultra-red FP (smURFP), covalently attaches a biliverdin (BV) chromophore without a lyase, and\n",
      "has 642/670-nm excitation-emission peaks, a large extinction coefficient (180,000 M(-1)cm(-1)) and\n",
      "quantum yield (18%), and photostability comparable to that of eGFP. smURFP has significantly greater\n",
      "BV incorporation rate and protein stability than the bacteriophytochrome (BPH) FPs. Moreover, BV\n",
      "supply is limited by membrane permeability, and smURFPs (but not BPH FPs) can incorporate a more\n",
      "membrane-permeant BV analog, making smURFP fluorescence comparable to that of FPs from jellyfish or\n",
      "coral. A far-red and near-infrared fluorescent cell cycle indicator was created with smURFP and a\n",
      "BPH FP.\n",
      " {\n",
      "\"main_contribution\": \"A far-red fluorescent protein evolved from a cyanobacterial phycobiliprotein.\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a new far-red fluorescent protein, smURFP, which has improved properties compared to existing fluorescent proteins. The protein was developed from an allophycocyanin α-subunit (APCα) and incorporates a biliverdin (BV) chromophore without the need for a lyase. This allows for greater BV incorporation rate and protein stability than previous bacteriophytochrome (BPH) FPs. The smURFP has a large extinction coefficient, quantum yield, and photostability comparable to that of eGFP. Additionally, the smURFP can incorporate a more membrane-permeant BV analog, making it suitable for use as a far-red and near-infrared fluorescent cell cycle indicator. Therefore, the main contribution of the paper is the development of a new imaging method in the form of this novel fluorescent protein.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1121\n",
      "27479330\n",
      "Virtual microfluidics for digital quantification and single-cell sequencing.\n",
      "We have developed hydrogel-based virtual microfluidics as a simple and robust alternative to complex\n",
      "engineered microfluidic systems for the compartmentalization of nucleic acid amplification\n",
      "reactions. We applied in-gel digital multiple displacement amplification (dMDA) to purified DNA\n",
      "templates, cultured bacterial cells and human microbiome samples in the virtual microfluidics\n",
      "system, and demonstrated whole-genome sequencing of single-cell MDA products with excellent coverage\n",
      "uniformity and markedly reduced chimerism compared with products of liquid MDA reactions.\n",
      " {\n",
      "\"main_contribution\": \"Virtual microfluidics for digital quantification and single-cell sequencing\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of this paper is the development of a new method for compartmentalizing nucleic acid amplification reactions using hydrogel-based virtual microfluidics, which can be used for digital quantification and single-cell sequencing. This method is not an imaging technique or method, so \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1122\n",
      "27548807\n",
      "Shrinkage-mediated imaging of entire organs and organisms using uDISCO.\n",
      "Recent tissue-clearing approaches have become important alternatives to standard histology\n",
      "approaches. However, light scattering in thick tissues and the size restrictions on samples that can\n",
      "be imaged with standard light-sheet microscopy pose limitations for analyzing large samples such as\n",
      "an entire rodent body. We developed 'ultimate DISCO' (uDISCO) clearing to overcome these limitations\n",
      "in volumetric imaging. uDISCO preserves fluorescent proteins over months and renders intact organs\n",
      "and rodent bodies transparent while reducing their size up to 65%. We used uDISCO to image neuronal\n",
      "connections and vasculature from head to toe over 7 cm and to perform unbiased screening of\n",
      "transplanted stem cells within the entire body of adult mice. uDISCO is compatible with diverse\n",
      "labeling methods and archival human tissue, and it can readily be used in various biomedical\n",
      "applications to study organization of large organ systems throughout entire organisms.\n",
      " {\n",
      "\"main_contribution\": \"uDISCO clearing to overcome limitations in volumetric imaging\",\n",
      "\"is_imaging_method\": true\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new imaging technique called uDISCO, which enables the shrinkage-mediated imaging of entire organs and organisms. The main contribution of the paper is the development of this technique, which overcomes previous limitations in volumetric imaging by allowing for the imaging of large samples such as an entire rodent body. Therefore, the main contribution is identified as \"uDISCO clearing to overcome limitations in volumetric imaging\".\n",
      "\n",
      "Since uDISCO is a new imaging technique, the variable is_imaging_method is set to true.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1123\n",
      "27595403\n",
      "Augmenting CRISPR applications in Drosophila with tRNA-flanked sgRNAs.\n",
      "We present tRNA-based vectors for producing multiple clustered regularly interspaced short\n",
      "palindromic repeats (CRISPR) single guide RNAs (sgRNAs) from a single RNA polymerase II or III\n",
      "transcript in Drosophila. The system, which is based on liberation of sgRNAs by processing flanking\n",
      "tRNAs, permits highly efficient multiplexing of Cas9-based mutagenesis. We also demonstrate that the\n",
      "tRNA-sgRNA system markedly increases the efficacy of conditional gene disruption by Cas9 and can\n",
      "promote editing by the recently discovered RNA-guided endonuclease Cpf1.\n",
      " {\n",
      "\"main_contribution\": \"Augmenting CRISPR applications in Drosophila with tRNA-flanked sgRNAs\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "Explanation:\n",
      "\n",
      "The title and abstract of the paper describe a new method for augmenting CRISPR applications in Drosophila using tRNA-flanked sgRNAs. This method allows for highly efficient multiplexing of Cas9-based mutagenesis and can promote editing by the recently discovered RNA-guided endonuclease Cpf1.\n",
      "\n",
      "The main contribution of the paper is the development of a new method for CRISPR applications, which is described in the title and abstract. Therefore, the main contribution is recorded as \"Augmenting CRISPR applications in Drosophila with tRNA-flanked sgRNAs\".\n",
      "\n",
      "Since the main contribution is not the development of a new imaging technique or method, the value of \"is_imaging_method\" is set to false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1124\n",
      "27595405\n",
      "A multifunctional AAV-CRISPR-Cas9 and its host response.\n",
      "CRISPR-Cas9 delivery by adeno-associated virus (AAV) holds promise for gene therapy but faces\n",
      "critical barriers on account of its potential immunogenicity and limited payload capacity. Here, we\n",
      "demonstrate genome engineering in postnatal mice using AAV-split-Cas9, a multifunctional platform\n",
      "customizable for genome editing, transcriptional regulation, and other previously impracticable\n",
      "applications of AAV-CRISPR-Cas9. We identify crucial parameters that impact efficacy and clinical\n",
      "translation of our platform, including viral biodistribution, editing efficiencies in various\n",
      "organs, antigenicity, immunological reactions, and physiological outcomes. These results reveal that\n",
      "AAV-CRISPR-Cas9 evokes host responses with distinct cellular and molecular signatures, but unlike\n",
      "alternative delivery methods, does not induce extensive cellular damage in vivo. Our study provides\n",
      "a foundation for developing effective genome therapeutics.\n",
      " {\n",
      "\"main_contribution\": \"genome engineering in postnatal mice using AAV-split-Cas9\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is the development of a multifunctional platform for genome engineering in postnatal mice using AAV-split-Cas9. This platform allows for genome editing, transcriptional regulation, and other applications that were previously not possible with AAV-CRISPR-Cas9. The study identifies crucial parameters that impact the efficacy and clinical translation of the platform, including viral biodistribution, editing efficiencies in various organs, antigenicity, immunological reactions, and physiological outcomes. Therefore, the main contribution is not the development of a new imaging method, and hence the value of is_imaging_method is false.\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "1125\n",
      "27669326\n",
      "Enzyme-catalyzed expressed protein ligation.\n",
      "Expressed protein ligation is a valuable method for protein semisynthesis that involves the reaction\n",
      "of recombinant protein C-terminal thioesters with N-terminal cysteine (N-Cys)-containing peptides,\n",
      "but the requirement of a Cys residue at the ligation junction can limit the utility of this method.\n",
      "Here we employ subtiligase variants to efficiently ligate Cys-free peptides to protein thioesters.\n",
      "Using this method, we have more accurately determined the effect of C-terminal phosphorylation on\n",
      "the tumor suppressor protein PTEN.\n",
      " {\n",
      "\"main_contribution\": \"efficiently ligate Cys-free peptides to protein thioesters\",\n",
      "\"is_imaging_method\": false\n",
      "}\n",
      "\n",
      "The main contribution of the paper is developing a new method for protein semisynthesis, specifically a method for efficiently ligating Cys-free peptides to protein thioesters using subtiligase variants. This is described in the abstract as \"we employ subtiligase variants to efficiently ligate Cys-free peptides to protein thioesters.\" Therefore, the main contribution is not an imaging method, and is_imaging_method is set to false."
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>main_contribution</th>\n",
       "      <th>is_imaging_method</th>\n",
       "      <th>llm_output</th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>time</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>demonstrated the ability to excite specific ne...</td>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"main_contribution\": \"demonstrated the abi...</td>\n",
       "      <td>19898486</td>\n",
       "      <td>147.822898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Automated high-throughput mapping of promoter-...</td>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"main_contribution\": \"Automated high-throu...</td>\n",
       "      <td>19898487</td>\n",
       "      <td>77.961682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>miR-SP system for transgenic microRNA inhibiti...</td>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"main_contribution\": \"miR-SP system for tr...</td>\n",
       "      <td>19915559</td>\n",
       "      <td>82.453050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Floxin technology for targeted modification of...</td>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"main_contribution\": \"Floxin technology fo...</td>\n",
       "      <td>19966808</td>\n",
       "      <td>75.900989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>A tool for live-cell imaging that facilitates ...</td>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"main_contribution\": \"A tool for live-cell...</td>\n",
       "      <td>19966809</td>\n",
       "      <td>86.998896</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>Virtual microfluidics for digital quantificati...</td>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"main_contribution\": \"Virtual microfluidic...</td>\n",
       "      <td>27479330</td>\n",
       "      <td>52.461597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>uDISCO clearing to overcome limitations in vol...</td>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"main_contribution\": \"uDISCO clearing to o...</td>\n",
       "      <td>27548807</td>\n",
       "      <td>74.625292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>Augmenting CRISPR applications in Drosophila w...</td>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"main_contribution\": \"Augmenting CRISPR ap...</td>\n",
       "      <td>27595403</td>\n",
       "      <td>72.643309</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>genome engineering in postnatal mice using AAV...</td>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"main_contribution\": \"genome engineering i...</td>\n",
       "      <td>27595405</td>\n",
       "      <td>77.260483</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>efficiently ligate Cys-free peptides to protei...</td>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"main_contribution\": \"efficiently ligate C...</td>\n",
       "      <td>27669326</td>\n",
       "      <td>55.569566</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>425 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     main_contribution  is_imaging_method  \\\n",
       "0    demonstrated the ability to excite specific ne...               True   \n",
       "1    Automated high-throughput mapping of promoter-...               True   \n",
       "2    miR-SP system for transgenic microRNA inhibiti...              False   \n",
       "3    Floxin technology for targeted modification of...              False   \n",
       "4    A tool for live-cell imaging that facilitates ...               True   \n",
       "..                                                 ...                ...   \n",
       "420  Virtual microfluidics for digital quantificati...              False   \n",
       "421  uDISCO clearing to overcome limitations in vol...               True   \n",
       "422  Augmenting CRISPR applications in Drosophila w...              False   \n",
       "423  genome engineering in postnatal mice using AAV...              False   \n",
       "424  efficiently ligate Cys-free peptides to protei...              False   \n",
       "\n",
       "                                            llm_output  ID_PAPER        time  \n",
       "0     {\\n\"main_contribution\": \"demonstrated the abi...  19898486  147.822898  \n",
       "1     {\\n\"main_contribution\": \"Automated high-throu...  19898487   77.961682  \n",
       "2     {\\n\"main_contribution\": \"miR-SP system for tr...  19915559   82.453050  \n",
       "3     {\\n\"main_contribution\": \"Floxin technology fo...  19966808   75.900989  \n",
       "4     {\\n\"main_contribution\": \"A tool for live-cell...  19966809   86.998896  \n",
       "..                                                 ...       ...         ...  \n",
       "420   {\\n\"main_contribution\": \"Virtual microfluidic...  27479330   52.461597  \n",
       "421   {\\n\"main_contribution\": \"uDISCO clearing to o...  27548807   74.625292  \n",
       "422   {\\n\"main_contribution\": \"Augmenting CRISPR ap...  27595403   72.643309  \n",
       "423   {\\n\"main_contribution\": \"genome engineering i...  27595405   77.260483  \n",
       "424   {\\n\"main_contribution\": \"efficiently ligate C...  27669326   55.569566  \n",
       "\n",
       "[425 rows x 5 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import time\n",
    "\n",
    "def execute_llm(llm, p, i, t, a):\n",
    "    print(textwrap.fill(i, 100))\n",
    "    print(textwrap.fill(t, 100))\n",
    "    print(textwrap.fill(a, 100))\n",
    "    output = llm(p.format(title=t, abstract=a))\n",
    "    return output\n",
    "\n",
    "l = []\n",
    "for i, r in df[701:].iterrows():\n",
    "    if r.ABSTRACT is None: \n",
    "        continue\n",
    "    print('\\n-----------------------------------------------------------------\\n')\n",
    "    print(i)\n",
    "    # measure time to execute\n",
    "    t = time.time()\n",
    "    output = execute_llm(llm, p2, r.ID_PAPER, r.TITLE, r.ABSTRACT)\n",
    "    m = output_parser.parse(output)\n",
    "    if m is None: \n",
    "        m = {\n",
    "            'main_contribution': None,\n",
    "            'is_imaging_method': False,\n",
    "        } \n",
    "    m['llm_output'] = output\n",
    "    m['ID_PAPER'] = r.ID_PAPER\n",
    "    dt = time.time() - t    \n",
    "    m['time'] = dt\n",
    "    l.append(m)\n",
    "\n",
    "df1 = pd.DataFrame(l)\n",
    "df1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df1.to_csv('/Users/gburns/Documents/2023H2/image_tech_landscaping/alhazen_analysis/p2_08-25-23/df_200-700_70b.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df.to_csv('/Users/gburns/Documents/2023H2/image_tech_landscaping/alhazen_analysis/df.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# format df.ID_PAPER as int64\n",
    "df.ID_PAPER = df.ID_PAPER.astype('int64') \n",
    "df1.ID_PAPER = df1.ID_PAPER.astype('int64') \n",
    "df2 = df.set_index('ID_PAPER').join(df1.set_index('ID_PAPER'))\n",
    "df2.to_csv('/Users/gburns/Documents/2023H2/image_tech_landscaping/alhazen_analysis/p2_08-25-23/df_merged_200-500_70b.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fluorescent indicators for simultaneous reporting of all four cell cycle phases.\n",
      "A robust method for simultaneous visualization of all four cell cycle phases in living cells is\n",
      "highly desirable. We developed an intensiometric reporter of the transition from S to G2 phase and\n",
      "engineered a far-red fluorescent protein, mMaroon1, to visualize chromatin condensation in mitosis.\n",
      "We combined these new reporters with the previously described Fucci system to create Fucci4, a set\n",
      "of four orthogonal fluorescent indicators that together resolve all cell cycle phases.\n",
      " [ `'an intensiometric reporter of the transition from S to G2 phase', 'a far-red fluorescent protein, mMaroon1', 'the previously described Fucci system''](\" [ `'an intensiometric reporter of the transition from S to G2 phase', 'a \"\n",
      " \"far-red fluorescent protein, mMaroon1', 'the previously described Fucci \"\n",
      " \"system'']\")\n"
     ]
    }
   ],
   "source": [
    "\n",
    "\n",
    "article_text = df.iloc[5]['TITLE'] + ' ' + df.iloc[5]['ABSTRACT']\n",
    "print(textwrap.fill(df.iloc[5]['TITLE'], 100))\n",
    "print(textwrap.fill(df.iloc[5]['ABSTRACT'], 100))\n",
    "\n",
    "p2.format(article_text=article_text)\n",
    "\n",
    "output = llm(p2.format(article_text=article_text), )\n",
    "pprint(output, depth=2)\n",
    "#pprint(output_parser.parse(output), depth=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " Immuno-oncology (IO) is a field of cancer research and treatment that focuses on using the body's own immune system to fight cancer. The goal of IO is to enhance the immune system's ability to recognize, attack, and eliminate cancer cells.\n",
      "\n",
      "Cancer cells have ways to evade the immune system and avoid being targeted by immune cells, such as T cells. IO strategies aim to overcome these evasion mechanisms and restore the immune system's ability to recognize and attack cancer cells.\n",
      "\n",
      "There are several approaches to IO, including:\n",
      "\n",
      "1. Checkpoint inhibitors: These drugs work by blocking proteins on immune cells that normally prevent them from attacking cancer cells. Examples include pembrolizumab (Keytruda) and nivolumab (Opdivo).\n",
      "2. Immune stimulators: These drugs enhance the activity of immune cells, such as T cells, to recognize and attack cancer cells. Examples include talimogene laherparepvec (Imlygic) and CimaVax-EGF.\n",
      "3. Cancer vaccines: These vaccines aim to stimulate an immune response against cancer cells. Examples include sipuleucel-T (Provenge) and CimaVax-EGF.\n",
      "4. Adoptive T-cell therapy: This approach involves removing T cells from a patient's immune system, expanding or activating them in a lab, and then reinfusing them back into the patient to attack cancer cells. Examples include chimeric antigen receptor (CAR) T-cell therapy and T-cell receptor (TCR) therapy.\n",
      "5. Immunomodulators: These drugs work by modifying the immune system's response to cancer cells. Examples include lenalidomide (Revlimid) and imatinib (Gleevec).\n",
      "\n",
      "IO has shown significant promise in treating various types of cancer, including melanoma, lung cancer, kidney cancer, bladder cancer, and others. However, it's important to note that IO treatments can have side effects, such as inflammation of organs or tissues, and not all patients respond to these treatments.\n",
      "\n",
      "Overall, IO is a rapidly evolving field with ongoing research to develop new and more effective treatments to harness the power of the immune system against cancer."
     ]
    },
    {
     "data": {
      "text/plain": [
       "\" Immuno-oncology (IO) is a field of cancer research and treatment that focuses on using the body's own immune system to fight cancer. The goal of IO is to enhance the immune system's ability to recognize, attack, and eliminate cancer cells.\\n\\nCancer cells have ways to evade the immune system and avoid being targeted by immune cells, such as T cells. IO strategies aim to overcome these evasion mechanisms and restore the immune system's ability to recognize and attack cancer cells.\\n\\nThere are several approaches to IO, including:\\n\\n1. Checkpoint inhibitors: These drugs work by blocking proteins on immune cells that normally prevent them from attacking cancer cells. Examples include pembrolizumab (Keytruda) and nivolumab (Opdivo).\\n2. Immune stimulators: These drugs enhance the activity of immune cells, such as T cells, to recognize and attack cancer cells. Examples include talimogene laherparepvec (Imlygic) and CimaVax-EGF.\\n3. Cancer vaccines: These vaccines aim to stimulate an immune response against cancer cells. Examples include sipuleucel-T (Provenge) and CimaVax-EGF.\\n4. Adoptive T-cell therapy: This approach involves removing T cells from a patient's immune system, expanding or activating them in a lab, and then reinfusing them back into the patient to attack cancer cells. Examples include chimeric antigen receptor (CAR) T-cell therapy and T-cell receptor (TCR) therapy.\\n5. Immunomodulators: These drugs work by modifying the immune system's response to cancer cells. Examples include lenalidomide (Revlimid) and imatinib (Gleevec).\\n\\nIO has shown significant promise in treating various types of cancer, including melanoma, lung cancer, kidney cancer, bladder cancer, and others. However, it's important to note that IO treatments can have side effects, such as inflammation of organs or tissues, and not all patients respond to these treatments.\\n\\nOverall, IO is a rapidly evolving field with ongoing research to develop new and more effective treatments to harness the power of the immune system against cancer.\""
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "llm('What is immuno-oncology?')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "27749836\n",
      "Random-access scanning microscopy for 3D imaging in awake behaving animals.\n",
      "Understanding how neural circuits process information requires rapid measurements of activity from\n",
      "identified neurons distributed in 3D space. Here we describe an acousto-optic lens two-photon\n",
      "microscope that performs high-speed focusing and line scanning within a volume spanning hundreds of\n",
      "micrometers. We demonstrate its random-access functionality by selectively imaging cerebellar\n",
      "interneurons sparsely distributed in 3D space and by simultaneously recording from the soma,\n",
      "proximal and distal dendrites of neocortical pyramidal cells in awake behaving mice.\n",
      "['acousto-optic lens microscopy',\n",
      " 'two-photon microscopy',\n",
      " 'random-access scanning microscopy',\n",
      " 'line scanning']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "27749837\n",
      "ATAC-see reveals the accessible genome by transposase-mediated imaging and sequencing.\n",
      "Spatial organization of the genome plays a central role in gene expression, DNA replication, and\n",
      "repair. But current epigenomic approaches largely map DNA regulatory elements outside of the native\n",
      "context of the nucleus. Here we report assay of transposase-accessible chromatin with visualization\n",
      "(ATAC-see), a transposase-mediated imaging technology that employs direct imaging of the accessible\n",
      "genome in situ, cell sorting, and deep sequencing to reveal the identity of the imaged elements.\n",
      "ATAC-see revealed the cell-type-specific spatial organization of the accessible genome and the\n",
      "coordinated process of neutrophil chromatin extrusion, termed NETosis. Integration of ATAC-see with\n",
      "flow cytometry enables automated quantitation and prospective cell isolation as a function of\n",
      "chromatin accessibility, and it reveals a cell-cycle dependence of chromatin accessibility that is\n",
      "especially dynamic in G1 phase. The integration of imaging and epigenomics provides a general and\n",
      "scalable approach for deciphering the spatiotemporal architecture of gene control.\n",
      "['ATAC-see',\n",
      " 'transposase-mediated imaging',\n",
      " 'direct imaging',\n",
      " 'deep sequencing']\n",
      "Elapsed time: 11.28 seconds\n",
      "-------------------\n",
      "27776112\n",
      "Bright photoactivatable fluorophores for single-molecule imaging.\n",
      "Small-molecule fluorophores are important tools for advanced imaging experiments. We previously\n",
      "reported a general method to improve small, cell-permeable fluorophores which resulted in the\n",
      "azetidine-containing 'Janelia Fluor' (JF) dyes. Here, we refine and extend the utility of these dyes\n",
      "by synthesizing photoactivatable derivatives that are compatible with live-cell labeling strategies.\n",
      "Once activated, these derived compounds retain the superior brightness and photostability of the JF\n",
      "dyes, enabling improved single-particle tracking and facile localization microscopy experiments.\n",
      "['single-molecule imaging',\n",
      " 'live-cell labeling strategies',\n",
      " 'photostability',\n",
      " 'localization microscopy experiments']\n",
      "Elapsed time: 8.28 seconds\n",
      "-------------------\n",
      "27798609\n",
      "Simultaneous dual-color fluorescence lifetime imaging with novel red-shifted fluorescent proteins.\n",
      "We describe a red-shifted fluorescence resonance energy transfer (FRET) pair optimized for dual-\n",
      "color fluorescence lifetime imaging (FLIM). This pair utilizes a newly developed FRET donor,\n",
      "monomeric cyan-excitable red fluorescent protein (mCyRFP1), which has a large Stokes shift and a\n",
      "monoexponential fluorescence lifetime decay. When used together with EGFP-based biosensors, the new\n",
      "pair enables simultaneous imaging of the activities of two signaling molecules in single dendritic\n",
      "spines undergoing structural plasticity.\n",
      "['fluorescence lifetime imaging (FLIM)',\n",
      " 'red-shifted fluorescence resonance energy transfer (FRET)']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "27798610\n",
      "Fluorescent indicators for simultaneous reporting of all four cell cycle phases.\n",
      "A robust method for simultaneous visualization of all four cell cycle phases in living cells is\n",
      "highly desirable. We developed an intensiometric reporter of the transition from S to G2 phase and\n",
      "engineered a far-red fluorescent protein, mMaroon1, to visualize chromatin condensation in mitosis.\n",
      "We combined these new reporters with the previously described Fucci system to create Fucci4, a set\n",
      "of four orthogonal fluorescent indicators that together resolve all cell cycle phases.\n",
      "['intensiometric reporter',\n",
      " 'far-red fluorescent protein',\n",
      " 'Fucci system',\n",
      " 'four orthogonal fluorescent indicators']\n",
      "Elapsed time: 7.30 seconds\n",
      "-------------------\n",
      "27798611\n",
      "Directed evolution using dCas9-targeted somatic hypermutation in mammalian cells.\n",
      "Engineering and study of protein function by directed evolution has been limited by the technical\n",
      "requirement to use global mutagenesis or introduce DNA libraries. Here, we develop CRISPR-X, a\n",
      "strategy to repurpose the somatic hypermutation machinery for protein engineering in situ. Using\n",
      "catalytically inactive dCas9 to recruit variants of cytidine deaminase (AID) with MS2-modified\n",
      "sgRNAs, we can specifically mutagenize endogenous targets with limited off-target damage. This\n",
      "generates diverse libraries of localized point mutations and can target multiple genomic locations\n",
      "simultaneously. We mutagenize GFP and select for spectrum-shifted variants, including EGFP.\n",
      "Additionally, we mutate the target of the cancer therapeutic bortezomib, PSMB5, and identify known\n",
      "and novel mutations that confer bortezomib resistance. Finally, using a hyperactive AID variant, we\n",
      "mutagenize loci both upstream and downstream of transcriptional start sites. These experiments\n",
      "illustrate a powerful approach to create complex libraries of genetic variants in native context,\n",
      "which is broadly applicable to investigate and improve protein function.\n",
      "['CRISPR-X',\n",
      " 'directed evolution',\n",
      " 'somatic hypermutation',\n",
      " 'catalytically inactive dCas9',\n",
      " 'MS2-modified sgRNAs',\n",
      " 'localized point mutations',\n",
      " 'spectrum-shifted variants',\n",
      " 'EGFP',\n",
      " 'bortezomib resistance',\n",
      " 'hyperactive AID variant',\n",
      " 'native context']\n",
      "Elapsed time: 13.32 seconds\n",
      "-------------------\n",
      "27798612\n",
      "Fast volumetric calcium imaging across multiple cortical layers using sculpted light.\n",
      "Although whole-organism calcium imaging in small and semi-transparent animals has been demonstrated,\n",
      "capturing the functional dynamics of large-scale neuronal circuits in awake behaving mammals at high\n",
      "speed and resolution has remained one of the main frontiers in systems neuroscience. Here we present\n",
      "a method based on light sculpting that enables unbiased single- and dual-plane high-speed (up to 160\n",
      "Hz) calcium imaging as well as in vivo volumetric calcium imaging of a mouse cortical column (0.5 mm\n",
      "× 0.5 mm × 0.5 mm) at single-cell resolution and fast volume rates (3-6 Hz). We achieved this by\n",
      "tailoring the point-spread function of our microscope to the structures of interest while maximizing\n",
      "the signal-to-noise ratio using a home-built fiber laser amplifier with pulses that are synchronized\n",
      "to the imaging voxel speed. This enabled in vivo recording of calcium dynamics of several thousand\n",
      "neurons across cortical layers and in the hippocampus of awake behaving mice.\n",
      "['calcium imaging',\n",
      " 'whole-organism imaging',\n",
      " 'single-plane imaging',\n",
      " 'dual-plane imaging',\n",
      " 'volumetric imaging',\n",
      " 'light sculpting']\n",
      "Elapsed time: 11.59 seconds\n",
      "-------------------\n",
      "27819661\n",
      "DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo.\n",
      "Coupling of structure-specific in vivo chemical modification to next-generation sequencing is\n",
      "transforming RNA secondary structure studies in living cells. The dominant strategy for detecting in\n",
      "vivo chemical modifications uses reverse transcriptase truncation products, which introduce biases\n",
      "and necessitate population-average assessments of RNA structure. Here we present dimethyl sulfate\n",
      "(DMS) mutational profiling with sequencing (DMS-MaPseq), which encodes DMS modifications as\n",
      "mismatches using a thermostable group II intron reverse transcriptase. DMS-MaPseq yields a high\n",
      "signal-to-noise ratio, can report multiple structural features per molecule, and allows both genome-\n",
      "wide studies and focused in vivo investigations of even low-abundance RNAs. We apply DMS-MaPseq for\n",
      "the first analysis of RNA structure within an animal tissue and to identify a functional structure\n",
      "involved in noncanonical translation initiation. Additionally, we use DMS-MaPseq to compare the in\n",
      "vivo structure of pre-mRNAs with their mature isoforms. These applications illustrate DMS-MaPseq's\n",
      "capacity to dramatically expand in vivo analysis of RNA structure.\n",
      "['DMS-MaPseq',\n",
      " 'next-generation sequencing',\n",
      " 'reverse transcriptase truncation products',\n",
      " 'thermostable group II intron reverse transcriptase']\n",
      "Elapsed time: 12.00 seconds\n",
      "-------------------\n",
      "27842059\n",
      "FDR-controlled metabolite annotation for high-resolution imaging mass spectrometry.\n",
      "High-mass-resolution imaging mass spectrometry promises to localize hundreds of metabolites in\n",
      "tissues, cell cultures, and agar plates with cellular resolution, but it is hampered by the lack of\n",
      "bioinformatics tools for automated metabolite identification. We report pySM, a framework for false\n",
      "discovery rate (FDR)-controlled metabolite annotation at the level of the molecular sum formula, for\n",
      "high-mass-resolution imaging mass spectrometry (https://github.com/alexandrovteam/pySM). We\n",
      "introduce a metabolite-signal match score and a target-decoy FDR estimate for spatial metabolomics.\n",
      "['high-mass-resolution imaging mass spectrometry',\n",
      " 'false discovery rate (FDR)-controlled metabolite annotation',\n",
      " 'spatial metabolomics']\n",
      "Elapsed time: 8.57 seconds\n",
      "-------------------\n",
      "27842060\n",
      "Atmospheric pressure MALDI mass spectrometry imaging of tissues and cells at 1.4-μm lateral\n",
      "resolution.\n",
      "We report an atmospheric pressure (AP) matrix-assisted laser desorption/ionization (MALDI) mass\n",
      "spectrometry imaging (MSI) setup with a lateral resolution of 1.4 μm, a mass resolution greater than\n",
      "100,000, and accuracy below ±2 p.p.m. We achieved this by coupling a focusing objective with a\n",
      "numerical aperture (NA) of 0.9 at 337 nm and a free working distance of 18 mm in coaxial geometry to\n",
      "an orbitrap mass spectrometer and optimizing the matrix application. We demonstrate improvement in\n",
      "image contrast, lateral resolution, and ion yield per unit area compared with a state-of-the-art\n",
      "commercial MSI source. We show that our setup can be used to detect metabolites, lipids, and small\n",
      "peptides, as well as to perform tandem MS experiments with 1.5-μm<sup>2</sup> sampling areas. To\n",
      "showcase these capabilities, we identified subcellular lipid, metabolite, and peptide distributions\n",
      "that differentiate, for example, cilia and oral groove in Paramecium caudatum.\n",
      "['MALDI',\n",
      " 'mass spectrometry imaging',\n",
      " 'atmospheric pressure',\n",
      " 'matrix-assisted laser desorption/ionization',\n",
      " 'MSI']\n",
      "Elapsed time: 12.43 seconds\n",
      "-------------------\n",
      "27869813\n",
      "Covalently circularized nanodiscs for studying membrane proteins and viral entry.\n",
      "We engineered covalently circularized nanodiscs (cNDs) which, compared with standard nanodiscs,\n",
      "exhibit enhanced stability, defined diameter sizes and tunable shapes. Reconstitution into cNDs\n",
      "enhanced the quality of nuclear magnetic resonance spectra for both VDAC-1, a β-barrel membrane\n",
      "protein, and the G-protein-coupled receptor NTR1, an α-helical membrane protein. In addition, we\n",
      "used cNDs to visualize how simple, nonenveloped viruses translocate their genomes across membranes\n",
      "to initiate infection.\n",
      "['nuclear magnetic resonance spectra', 'electron microscopy']\n",
      "Elapsed time: 7.64 seconds\n",
      "-------------------\n",
      "27869814\n",
      "Photometry unlocks 3D information from 2D localization microscopy data.\n",
      "We developed a straightforward photometric method, temporal, radial-aperture-based intensity\n",
      "estimation (TRABI), that allows users to extract 3D information from existing 2D localization\n",
      "microscopy data. TRABI uses the accurate determination of photon numbers in different regions of the\n",
      "emission pattern of single emitters to generate a z-dependent photometric parameter. This method can\n",
      "determine fluorophore positions up to 600 nm from the focal plane and can be combined with biplane\n",
      "detection to further improve axial localization.\n",
      "['photometry',\n",
      " 'temporal radial-aperture-based intensity estimation (TRABI)',\n",
      " 'biplane detection']\n",
      "Elapsed time: 7.38 seconds\n",
      "-------------------\n",
      "27869816\n",
      "mScarlet: a bright monomeric red fluorescent protein for cellular imaging.\n",
      "We report the engineering of mScarlet, a truly monomeric red fluorescent protein with record\n",
      "brightness, quantum yield (70%) and fluorescence lifetime (3.9 ns). We developed mScarlet starting\n",
      "with a consensus synthetic template and using improved spectroscopic screening techniques;\n",
      "mScarlet's crystal structure reveals a planar and rigidified chromophore. mScarlet outperforms\n",
      "existing red fluorescent proteins as a fusion tag, and it is especially useful as a Förster\n",
      "resonance energy transfer (FRET) acceptor in ratiometric imaging.\n",
      "['ratiometric imaging',\n",
      " 'Förster resonance energy transfer (FRET) acceptor',\n",
      " 'cellular imaging']\n",
      "Elapsed time: 8.33 seconds\n",
      "-------------------\n",
      "27918540\n",
      "In vivo quantification of spatially varying mechanical properties in developing tissues.\n",
      "The mechanical properties of the cellular microenvironment and their spatiotemporal variations are\n",
      "thought to play a central role in sculpting embryonic tissues, maintaining organ architecture and\n",
      "controlling cell behavior, including cell differentiation. However, no direct in vivo and in situ\n",
      "measurement of mechanical properties within developing 3D tissues and organs has yet been performed.\n",
      "Here we introduce a technique that employs biocompatible, magnetically responsive ferrofluid\n",
      "microdroplets as local mechanical actuators and allows quantitative spatiotemporal measurements of\n",
      "mechanical properties in vivo. Using this technique, we show that vertebrate body elongation entails\n",
      "spatially varying tissue mechanics along the anteroposterior axis. Specifically, we find that the\n",
      "zebrafish tailbud is viscoelastic (elastic below a few seconds and fluid after just 1 min) and\n",
      "displays decreasing stiffness and increasing fluidity toward its posterior elongating region. This\n",
      "method opens new avenues to study mechanobiology in vivo, both in embryogenesis and in disease\n",
      "processes, including cancer.\n",
      "['magnetically responsive ferrofluid microdroplets',\n",
      " 'in vivo and in situ measurement of mechanical properties',\n",
      " 'spatiotemporal measurements of mechanical properties']\n",
      "Elapsed time: 10.50 seconds\n",
      "-------------------\n",
      "27941784\n",
      "Control of cerebral ischemia with magnetic nanoparticles.\n",
      "The precise manipulation of microcirculation in mice can facilitate mechanistic studies of brain\n",
      "injury and repair after ischemia, but this manipulation remains a technical challenge, particularly\n",
      "in conscious mice. We developed a technology that uses micromagnets to induce aggregation of\n",
      "magnetic nanoparticles to reversibly occlude blood flow in microvessels. This allowed induction of\n",
      "ischemia in a specific cortical region of conscious mice of any postnatal age, including perinatal\n",
      "and neonatal stages, with precise spatiotemporal control but without surgical intervention of the\n",
      "skull or artery. When combined with longitudinal live-imaging approaches, this technology\n",
      "facilitated the discovery of a feature of the ischemic cascade: selective loss of smooth muscle\n",
      "cells in juveniles but not adults shortly after onset of ischemia and during blood reperfusion.\n",
      "['micromagnets', 'longitudinal live-imaging approaches']\n",
      "Elapsed time: 8.87 seconds\n",
      "-------------------\n",
      "27992408\n",
      "cGAL, a temperature-robust GAL4-UAS system for Caenorhabditis elegans.\n",
      "The GAL4-UAS system is a powerful tool for manipulating gene expression, but its application in\n",
      "Caenorhabditis elegans has not been described. Here we systematically optimize the system's three\n",
      "main components to develop a temperature-optimized GAL4-UAS system (cGAL) that robustly controls\n",
      "gene expression in C. elegans from 15 to 25 °C. We demonstrate this system's utility in\n",
      "transcriptional reporter analysis, site-of-action experiments and exogenous transgene expression;\n",
      "and we provide a basic driver and effector toolkit.\n",
      "['GAL4-UAS system',\n",
      " 'temperature-optimized GAL4-UAS system (cGAL)',\n",
      " 'transcriptional reporter analysis',\n",
      " 'site-of-action experiments',\n",
      " 'exogenous transgene expression']\n",
      "Elapsed time: 9.18 seconds\n",
      "-------------------\n",
      "28068315\n",
      "Hyperspectral phasor analysis enables multiplexed 5D in vivo imaging.\n",
      "Time-lapse imaging of multiple labels is challenging for biological imaging as noise, photobleaching\n",
      "and phototoxicity compromise signal quality, while throughput can be limited by processing time.\n",
      "Here, we report software called Hyper-Spectral Phasors (HySP) for denoising and unmixing multiple\n",
      "spectrally overlapping fluorophores in a low signal-to-noise regime with fast analysis. We show that\n",
      "HySP enables unmixing of seven signals in time-lapse imaging of living zebrafish embryos.\n",
      "['hyperspectral phasor analysis', 'time-lapse imaging', 'denoising', 'unmixing']\n",
      "Elapsed time: 7.88 seconds\n",
      "-------------------\n",
      "28165473\n",
      "cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination.\n",
      "Single-particle electron cryomicroscopy (cryo-EM) is a powerful method for determining the\n",
      "structures of biological macromolecules. With automated microscopes, cryo-EM data can often be\n",
      "obtained in a few days. However, processing cryo-EM image data to reveal heterogeneity in the\n",
      "protein structure and to refine 3D maps to high resolution frequently becomes a severe bottleneck,\n",
      "requiring expert intervention, prior structural knowledge, and weeks of calculations on expensive\n",
      "computer clusters. Here we show that stochastic gradient descent (SGD) and branch-and-bound maximum\n",
      "likelihood optimization algorithms permit the major steps in cryo-EM structure determination to be\n",
      "performed in hours or minutes on an inexpensive desktop computer. Furthermore, SGD with Bayesian\n",
      "marginalization allows ab initio 3D classification, enabling automated analysis and discovery of\n",
      "unexpected structures without bias from a reference map. These algorithms are combined in a user-\n",
      "friendly computer program named cryoSPARC (http://www.cryosparc.com).\n",
      "['cryo-EM',\n",
      " 'stochastic gradient descent (SGD)',\n",
      " 'Bayesian marginalization',\n",
      " 'ab initio 3D classification']\n",
      "Elapsed time: 10.96 seconds\n",
      "-------------------\n",
      "28218899\n",
      "Prospective identification of hematopoietic lineage choice by deep learning.\n",
      "Differentiation alters molecular properties of stem and progenitor cells, leading to changes in\n",
      "their shape and movement characteristics. We present a deep neural network that prospectively\n",
      "predicts lineage choice in differentiating primary hematopoietic progenitors using image patches\n",
      "from brightfield microscopy and cellular movement. Surprisingly, lineage choice can be detected up\n",
      "to three generations before conventional molecular markers are observable. Our approach allows\n",
      "identification of cells with differentially expressed lineage-specifying genes without molecular\n",
      "labeling.\n",
      "['brightfield microscopy', 'cellular movement']\n",
      "Elapsed time: 7.11 seconds\n",
      "-------------------\n",
      "28218900\n",
      "In vivo three-photon imaging of activity of GCaMP6-labeled neurons deep in intact mouse brain.\n",
      "High-resolution optical imaging is critical to understanding brain function. We demonstrate that\n",
      "three-photon microscopy at 1,300-nm excitation enables functional imaging of GCaMP6s-labeled neurons\n",
      "beyond the depth limit of two-photon microscopy. We record spontaneous activity from up to 150\n",
      "neurons in the hippocampal stratum pyramidale at ∼1-mm depth within an intact mouse brain. Our\n",
      "method creates opportunities for noninvasive recording of neuronal activity with high spatial and\n",
      "temporal resolution deep within scattering brain tissues.\n",
      "['three-photon microscopy',\n",
      " 'high-resolution optical imaging',\n",
      " 'GCaMP6s-labeled neurons']\n",
      "Elapsed time: 8.41 seconds\n",
      "-------------------\n",
      "28250467\n",
      "Automated synaptic connectivity inference for volume electron microscopy.\n",
      "Teravoxel volume electron microscopy data sets from neural tissue can now be acquired in weeks, but\n",
      "data analysis requires years of manual labor. We developed the SyConn framework, which uses deep\n",
      "convolutional neural networks and random forest classifiers to infer a richly annotated synaptic\n",
      "connectivity matrix from manual neurite skeleton reconstructions by automatically identifying\n",
      "mitochondria, synapses and their types, axons, dendrites, spines, myelin, somata and cell types. We\n",
      "tested our approach on serial block-face electron microscopy data sets from zebrafish, mouse and\n",
      "zebra finch, and computed the synaptic wiring of songbird basal ganglia. We found that, for example,\n",
      "basal-ganglia cell types with high firing rates in vivo had higher densities of mitochondria and\n",
      "vesicles and that synapse sizes and quantities scaled systematically, depending on the innervated\n",
      "postsynaptic cell types.\n",
      "['electron microscopy',\n",
      " 'serial block-face electron microscopy',\n",
      " 'volume electron microscopy',\n",
      " 'mitochondria',\n",
      " 'synapses',\n",
      " 'axons',\n",
      " 'dendrites',\n",
      " 'spines',\n",
      " 'myelin',\n",
      " 'somata',\n",
      " 'cell types']\n",
      "Elapsed time: 62.35 seconds\n",
      "-------------------\n",
      "28250468\n",
      "Drop-on-demand sample delivery for studying biocatalysts in action at X-ray free-electron lasers.\n",
      "X-ray crystallography at X-ray free-electron laser sources is a powerful method for studying\n",
      "macromolecules at biologically relevant temperatures. Moreover, when combined with complementary\n",
      "techniques like X-ray emission spectroscopy, both global structures and chemical properties of\n",
      "metalloenzymes can be obtained concurrently, providing insights into the interplay between the\n",
      "protein structure and dynamics and the chemistry at an active site. The implementation of such a\n",
      "multimodal approach can be compromised by conflicting requirements to optimize each individual\n",
      "method. In particular, the method used for sample delivery greatly affects the data quality. We\n",
      "present here a robust way of delivering controlled sample amounts on demand using acoustic droplet\n",
      "ejection coupled with a conveyor belt drive that is optimized for crystallography and spectroscopy\n",
      "measurements of photochemical and chemical reactions over a wide range of time scales. Studies with\n",
      "photosystem II, the phytochrome photoreceptor, and ribonucleotide reductase R2 illustrate the power\n",
      "and versatility of this method.\n",
      "['X-ray crystallography',\n",
      " 'X-ray emission spectroscopy',\n",
      " 'acoustic droplet ejection']\n",
      "Elapsed time: 11.25 seconds\n",
      "-------------------\n",
      "28288122\n",
      "Genetically encoded biosensors for visualizing live-cell biochemical activity at super-resolution.\n",
      "Compartmentalized biochemical activities are essential to all cellular processes, but there is no\n",
      "generalizable method to visualize dynamic protein activities in living cells at a resolution\n",
      "commensurate with cellular compartmentalization. Here, we introduce a new class of fluorescent\n",
      "biosensors that detect biochemical activities in living cells at a resolution up to threefold better\n",
      "than the diffraction limit. These 'FLINC' biosensors use binding-induced changes in protein\n",
      "fluorescence dynamics to translate kinase activities or protein-protein interactions into changes in\n",
      "fluorescence fluctuations, which are quantifiable through stochastic optical fluctuation imaging. A\n",
      "protein kinase A (PKA) biosensor allowed us to resolve minute PKA activity microdomains on the\n",
      "plasma membranes of living cells and to uncover the role of clustered anchoring proteins in\n",
      "organizing these activity microdomains. Together, these findings suggest that biochemical activities\n",
      "of the cell are spatially organized into an activity architecture whose structural and functional\n",
      "characteristics can be revealed by these new biosensors.\n",
      "['stochastic optical fluctuation imaging',\n",
      " 'fluorescence dynamics',\n",
      " 'binding-induced changes']\n",
      "Elapsed time: 11.15 seconds\n",
      "-------------------\n",
      "28319111\n",
      "Volumetric two-photon imaging of neurons using stereoscopy (vTwINS).\n",
      "Two-photon laser scanning microscopy of calcium dynamics using fluorescent indicators is a widely\n",
      "used imaging method for large-scale recording of neural activity in vivo. Here, we introduce\n",
      "volumetric two-photon imaging of neurons using stereoscopy (vTwINS), a volumetric calcium imaging\n",
      "method that uses an elongated, V-shaped point spread function to image a 3D brain volume. Single\n",
      "neurons project to spatially displaced 'image pairs' in the resulting 2D image, and the separation\n",
      "distance between projections is proportional to depth in the volume. To demix the fluorescence time\n",
      "series of individual neurons, we introduce a modified orthogonal matching pursuit algorithm that\n",
      "also infers source locations within the 3D volume. We illustrated vTwINS by imaging neural\n",
      "population activity in the mouse primary visual cortex and hippocampus. Our results demonstrated\n",
      "that vTwINS provides an effective method for volumetric two-photon calcium imaging that increases\n",
      "the number of neurons recorded while maintaining a high frame rate.\n",
      "['two-photon laser scanning microscopy',\n",
      " 'fluorescent indicators',\n",
      " 'volumetric calcium imaging',\n",
      " 'stereoscopy',\n",
      " 'orthogonal matching pursuit algorithm']\n",
      "Elapsed time: 11.89 seconds\n",
      "-------------------\n",
      "28319112\n",
      "Accurate identification of single-nucleotide variants in whole-genome-amplified single cells.\n",
      "Mutation analysis in single-cell genomes is prone to artifacts associated with cell lysis and whole-\n",
      "genome amplification. Here we addressed these issues by developing single-cell multiple displacement\n",
      "amplification (SCMDA) and a general-purpose single-cell-variant caller, SCcaller\n",
      "(https://github.com/biosinodx/SCcaller/). By comparing SCMDA-amplified single cells with unamplified\n",
      "clones from the same population, we validated the procedure as a firm foundation for standardized\n",
      "somatic-mutation analysis in single-cell genomics.\n",
      "['single-cell multiple displacement amplification (SCMDA)',\n",
      " 'single-cell variant caller (SCcaller)']\n",
      "Elapsed time: 8.00 seconds\n",
      "-------------------\n",
      "28346450\n",
      "Structural modeling of protein-RNA complexes using crosslinking of segmentally isotope-labeled RNA\n",
      "and MS/MS.\n",
      "Ribonucleoproteins (RNPs) are key regulators of cellular function. We established an efficient\n",
      "approach, crosslinking of segmentally isotope-labeled RNA and tandem mass spectrometry (CLIR-MS/MS),\n",
      "to localize protein-RNA interactions simultaneously at amino acid and nucleotide resolution. The\n",
      "approach was tested on polypyrimidine tract binding protein 1 and U1 small nuclear RNP. Our method\n",
      "provides distance restraints to support integrative atomic-scale structural modeling and to gain\n",
      "mechanistic insights into RNP-regulated processes.\n",
      "['crosslinking of segmentally isotope-labeled RNA',\n",
      " 'tandem mass spectrometry (CLIR-MS/MS)',\n",
      " 'mass spectrometry (MS)']\n",
      "Elapsed time: 9.02 seconds\n",
      "-------------------\n",
      "28362435\n",
      "A guide to light-sheet fluorescence microscopy for multiscale imaging.\n",
      "The impact of light-sheet fluorescence microscopy (LSFM) is visible in fields as diverse as\n",
      "developmental and cell biology, anatomical science, biophysics and neuroscience. Although adoption\n",
      "among biologists has been steady, LSFM has not displaced more traditional imaging methods despite\n",
      "its often-superior performance. One reason for this is that the field has largely conformed to a do-\n",
      "it-yourself ethic, although the challenges of big image data cannot be overstated. With the most\n",
      "powerful implementations of LSFM available to only a few groups worldwide, the scope of this\n",
      "technique is unnecessarily limited. Here we elucidate the key developments and define a simple set\n",
      "of underlying principles governing LSFM. In doing so, we aim to clarify the decisions to be made for\n",
      "those who wish to develop and use bespoke light-sheet systems and to assist in identifying the best\n",
      "approaches to apply this powerful technique to myriad biological questions.\n",
      "['light-sheet fluorescence microscopy', 'LSFM']\n",
      "Elapsed time: 9.52 seconds\n",
      "-------------------\n",
      "28362436\n",
      "In vivo imaging of neural activity.\n",
      "Since the introduction of calcium imaging to monitor neuronal activity with single-cell resolution,\n",
      "optical imaging methods have revolutionized neuroscience by enabling systematic recordings of\n",
      "neuronal circuits in living animals. The plethora of methods for functional neural imaging can be\n",
      "daunting to the nonexpert to navigate. Here we review advanced microscopy techniques for in vivo\n",
      "functional imaging and offer guidelines for which technologies are best suited for particular\n",
      "applications.\n",
      "['calcium imaging', 'optical imaging', 'advanced microscopy techniques']\n",
      "Elapsed time: 6.76 seconds\n",
      "-------------------\n",
      "28362438\n",
      "Adaptive optical fluorescence microscopy.\n",
      "The past quarter century has witnessed rapid developments of fluorescence microscopy techniques that\n",
      "enable structural and functional imaging of biological specimens at unprecedented depth and\n",
      "resolution. The performance of these methods in multicellular organisms, however, is degraded by\n",
      "sample-induced optical aberrations. Here I review recent work on incorporating adaptive optics, a\n",
      "technology originally applied in astronomical telescopes to combat atmospheric aberrations, to\n",
      "improve image quality of fluorescence microscopy for biological imaging.\n",
      "['adaptive optical fluorescence microscopy',\n",
      " 'fluorescence microscopy',\n",
      " 'astronomical telescopes']\n",
      "Elapsed time: 7.20 seconds\n",
      "-------------------\n",
      "28369043\n",
      "Generation of mature T cells from human hematopoietic stem and progenitor cells in artificial thymic\n",
      "organoids.\n",
      "Studies of human T cell development require robust model systems that recapitulate the full span of\n",
      "thymopoiesis, from hematopoietic stem and progenitor cells (HSPCs) through to mature T cells.\n",
      "Existing in vitro models induce T cell commitment from human HSPCs; however, differentiation into\n",
      "mature CD3<sup>+</sup>TCR-αβ<sup>+</sup> single-positive CD8<sup>+</sup> or CD4<sup>+</sup> cells is\n",
      "limited. We describe here a serum-free, artificial thymic organoid (ATO) system that supports\n",
      "efficient and reproducible in vitro differentiation and positive selection of conventional human T\n",
      "cells from all sources of HSPCs. ATO-derived T cells exhibited mature naive phenotypes, a diverse T\n",
      "cell receptor (TCR) repertoire and TCR-dependent function. ATOs initiated with TCR-engineered HSPCs\n",
      "produced T cells with antigen-specific cytotoxicity and near-complete lack of endogenous TCR Vβ\n",
      "expression, consistent with allelic exclusion of Vβ-encoding loci. ATOs provide a robust tool for\n",
      "studying human T cell differentiation and for the future development of stem-cell-based engineered T\n",
      "cell therapies.\n",
      "['artificial thymic organoids', 'serum-free']\n",
      "Elapsed time: 13.95 seconds\n",
      "-------------------\n",
      "28394337\n",
      "DeActs: genetically encoded tools for perturbing the actin cytoskeleton in single cells.\n",
      "The actin cytoskeleton is essential for many fundamental biological processes, but tools for\n",
      "directly manipulating actin dynamics are limited to cell-permeable drugs that preclude single-cell\n",
      "perturbations. Here we describe DeActs, genetically encoded actin-modifying polypeptides, which\n",
      "effectively induce actin disassembly in eukaryotic cells. We demonstrate that DeActs are universal\n",
      "tools for studying the actin cytoskeleton in single cells in culture, tissues, and multicellular\n",
      "organisms including various neurodevelopmental model systems.\n",
      "['actin cytoskeleton', 'cell perturbations', 'single cells']\n",
      "Elapsed time: 7.83 seconds\n",
      "-------------------\n",
      "28417997\n",
      "Iterative expansion microscopy.\n",
      "We recently developed a method called expansion microscopy, in which preserved biological specimens\n",
      "are physically magnified by embedding them in a densely crosslinked polyelectrolyte gel, anchoring\n",
      "key labels or biomolecules to the gel, mechanically homogenizing the specimen, and then swelling the\n",
      "gel-specimen composite by ∼4.5× in linear dimension. Here we describe iterative expansion microscopy\n",
      "(iExM), in which a sample is expanded ∼20×. After preliminary expansion a second swellable polymer\n",
      "mesh is formed in the space newly opened up by the first expansion, and the sample is expanded\n",
      "again. iExM expands biological specimens ∼4.5 × 4.5, or ∼20×, and enables ∼25-nm-resolution imaging\n",
      "of cells and tissues on conventional microscopes. We used iExM to visualize synaptic proteins, as\n",
      "well as the detailed architecture of dendritic spines, in mouse brain circuitry.\n",
      "['expansion microscopy', 'iterative expansion microscopy', 'iExM']\n",
      "Elapsed time: 9.35 seconds\n",
      "-------------------\n",
      "28418000\n",
      "SCnorm: robust normalization of single-cell RNA-seq data.\n",
      "The normalization of RNA-seq data is essential for accurate downstream inference, but the\n",
      "assumptions upon which most normalization methods are based are not applicable in the single-cell\n",
      "setting. Consequently, applying existing normalization methods to single-cell RNA-seq data\n",
      "introduces artifacts that bias downstream analyses. To address this, we introduce SCnorm for\n",
      "accurate and efficient normalization of single-cell RNA-seq data.\n",
      "['SCnorm', 'single-cell RNA-seq data']\n",
      "Elapsed time: 6.14 seconds\n",
      "-------------------\n",
      "28481362\n",
      "Genetic interaction mapping in mammalian cells using CRISPR interference.\n",
      "We describe a combinatorial CRISPR interference (CRISPRi) screening platform for mapping genetic\n",
      "interactions in mammalian cells. We targeted 107 chromatin-regulation factors in human cells with\n",
      "pools of either single or double single guide RNAs (sgRNAs) to downregulate individual genes or gene\n",
      "pairs, respectively. Relative enrichment analysis of individual sgRNAs or sgRNA pairs allowed for\n",
      "quantitative characterization of genetic interactions, and comparison with protein-protein-\n",
      "interaction data revealed a functional map of chromatin regulation.\n",
      "['CRISPR interference',\n",
      " 'combinatorial CRISPR interference (CRISPRi) screening platform',\n",
      " 'pools of single guide RNAs (sgRNAs)',\n",
      " 'downregulate individual genes or gene pairs',\n",
      " 'relative enrichment analysis of individual sgRNAs or sgRNA pairs',\n",
      " 'quantitative characterization of genetic interactions']\n",
      "Elapsed time: 9.78 seconds\n",
      "-------------------\n",
      "28481364\n",
      "Large-field-of-view imaging by multi-pupil adaptive optics.\n",
      "Adaptive optics can correct for optical aberrations. We developed multi-pupil adaptive optics\n",
      "(MPAO), which enables simultaneous wavefront correction over a field of view of 450 × 450\n",
      "μm<sup>2</sup> and expands the correction area to nine times that of conventional methods. MPAO's\n",
      "ability to perform spatially independent wavefront control further enables 3D nonplanar imaging. We\n",
      "applied MPAO to in vivo structural and functional imaging in the mouse brain.\n",
      "[' Adaptive optics',\n",
      " ' multi-pupil adaptive optics (MPAO)',\n",
      " ' spatially independent wavefront control',\n",
      " ' 3D nonplanar imaging',\n",
      " ' in vivo structural and functional imaging']\n",
      "Elapsed time: 8.96 seconds\n",
      "-------------------\n",
      "28504681\n",
      "Fused cerebral organoids model interactions between brain regions.\n",
      "Human brain development involves complex interactions between different regions, including long-\n",
      "distance neuronal migration or formation of major axonal tracts. Different brain regions can be\n",
      "cultured in vitro within 3D cerebral organoids, but the random arrangement of regional identities\n",
      "limits the reliable analysis of complex phenotypes. Here, we describe a coculture method combining\n",
      "brain regions of choice within one organoid tissue. By fusing organoids of dorsal and ventral\n",
      "forebrain identities, we generate a dorsal-ventral axis. Using fluorescent reporters, we demonstrate\n",
      "CXCR4-dependent GABAergic interneuron migration from ventral to dorsal forebrain and describe\n",
      "methodology for time-lapse imaging of human interneuron migration. Our results demonstrate that\n",
      "cerebral organoid fusion cultures can model complex interactions between different brain regions.\n",
      "Combined with reprogramming technology, fusions should offer researchers the possibility to analyze\n",
      "complex neurodevelopmental defects using cells from neurological disease patients and to test\n",
      "potential therapeutic compounds.\n",
      "['time-lapse imaging', 'fluorescent reporters']\n",
      "Elapsed time: 10.25 seconds\n",
      "-------------------\n",
      "28530655\n",
      "A quantitative and multiplexed approach to uncover the fitness landscape of tumor suppression in\n",
      "vivo.\n",
      "Cancer growth is a multistage, stochastic evolutionary process. While cancer genome sequencing has\n",
      "been instrumental in identifying the genomic alterations that occur in human tumors, the\n",
      "consequences of these alterations on tumor growth remain largely unexplored. Conventional\n",
      "genetically engineered mouse models enable the study of tumor growth in vivo, but they are neither\n",
      "readily scalable nor sufficiently quantitative to unravel the magnitude and mode of action of many\n",
      "tumor-suppressor genes. Here, we present a method that integrates tumor barcoding with ultradeep\n",
      "barcode sequencing (Tuba-seq) to interrogate tumor-suppressor function in mouse models of human\n",
      "cancer. Tuba-seq uncovers genotype-dependent distributions of tumor sizes. By combining Tuba-seq\n",
      "with multiplexed CRISPR-Cas9-mediated genome editing, we quantified the effects of 11 tumor-\n",
      "suppressor pathways that are frequently altered in human lung adenocarcinoma. Tuba-seq enables the\n",
      "broad quantification of the function of tumor-suppressor genes with unprecedented resolution,\n",
      "parallelization, and precision.\n",
      "{'imagingTechniques': ['tumor barcoding',\n",
      "                       'ultradeep barcode sequencing (Tuba-seq)',\n",
      "                       'multiplexed CRISPR-Cas9-mediated genome editing']}\n",
      "Elapsed time: 12.40 seconds\n",
      "-------------------\n",
      "28553965\n",
      "Fast high-resolution miniature two-photon microscopy for brain imaging in freely behaving mice.\n",
      "Developments in miniaturized microscopes have enabled visualization of brain activities and\n",
      "structural dynamics in animals engaging in self-determined behaviors. However, it remains a\n",
      "challenge to resolve activity at single dendritic spines in freely behaving animals. Here, we report\n",
      "the design and application of a fast high-resolution, miniaturized two-photon microscope (FHIRM-TPM)\n",
      "that accomplishes this goal. With a headpiece weighing 2.15 g and a hollow-core photonic crystal\n",
      "fiber delivering 920-nm femtosecond laser pulses, the FHIRM-TPM is capable of imaging commonly used\n",
      "biosensors (GFP and GCaMP6) at high spatiotemporal resolution (0.64 μm laterally and 3.35 μm\n",
      "axially, 40 Hz at 256 × 256 pixels for raster scanning and 10,000 Hz for free-line scanning). We\n",
      "demonstrate the microscope's robustness with hour-long recordings of neuronal activities at the\n",
      "level of spines in mice experiencing vigorous body movements.\n",
      "['two-photon microscopy',\n",
      " 'miniature microscopes',\n",
      " 'high-resolution imaging',\n",
      " 'biosensors',\n",
      " 'GFP',\n",
      " 'GCaMP6',\n",
      " 'raster scanning',\n",
      " 'free-line scanning']\n",
      "Elapsed time: 13.08 seconds\n",
      "-------------------\n",
      "28581494\n",
      "Genetically encoded fluorescent sensors reveal dynamic regulation of NADPH metabolism.\n",
      "Reduced nicotinamide adenine dinucleotide phosphate (NADPH) is essential for biosynthetic reactions\n",
      "and antioxidant functions; however, detection of NADPH metabolism in living cells remains\n",
      "technically challenging. We develop and characterize ratiometric, pH-resistant, genetically encoded\n",
      "fluorescent indicators for NADPH (iNap sensors) with various affinities and wide dynamic range. iNap\n",
      "sensors enabled quantification of cytosolic and mitochondrial NADPH pools that are controlled by\n",
      "cytosolic NAD<sup>+</sup> kinase levels and revealed cellular NADPH dynamics under oxidative stress\n",
      "depending on glucose availability. We found that mammalian cells have a strong tendency to maintain\n",
      "physiological NADPH homeostasis, which is regulated by glucose-6-phosphate dehydrogenase and AMP\n",
      "kinase. Moreover, using the iNap sensors we monitor NADPH fluctuations during the activation of\n",
      "macrophage cells or wound response in vivo. These data demonstrate that the iNap sensors will be\n",
      "valuable tools for monitoring NADPH dynamics in live cells and gaining new insights into cell\n",
      "metabolism.\n",
      "[{'name': 'ratiometric', 'type': 'imaging technique'},\n",
      " {'name': 'pH-resistant', 'type': 'imaging technique'},\n",
      " {'name': 'genetically encoded', 'type': 'imaging technology'},\n",
      " {'name': 'fluorescent indicators', 'type': 'imaging technology'}]\n",
      "Elapsed time: 15.37 seconds\n",
      "-------------------\n",
      "28581495\n",
      "Quantitative mRNA imaging throughout the entire Drosophila brain.\n",
      "We describe a fluorescence in situ hybridization method that permits detection of the localization\n",
      "and abundance of single mRNAs (smFISH) in cleared whole-mount adult Drosophila brains. The approach\n",
      "is rapid and multiplexable and does not require molecular amplification; it allows facile\n",
      "quantification of mRNA expression with subcellular resolution on a standard confocal microscope. We\n",
      "further demonstrate single-mRNA detection across the entire brain using a custom Bessel beam\n",
      "structured illumination microscope (BB-SIM).\n",
      "['smFISH', 'confocal microscope', 'BB-SIM']\n",
      "Elapsed time: 7.55 seconds\n",
      "-------------------\n",
      "28604724\n",
      "Visualizing adenosine-to-inosine RNA editing in single mammalian cells.\n",
      "Conversion of adenosine to inosine is a frequent type of RNA editing, but important details about\n",
      "the biology of this conversion remain unknown because of a lack of imaging tools. We developed\n",
      "inoFISH to directly visualize and quantify adenosine-to-inosine-edited transcripts in situ. We found\n",
      "that editing of the GRIA2, EIF2AK2, and NUP43 transcripts is uncorrelated with nuclear localization\n",
      "and paraspeckle association. Further, NUP43 exhibits constant editing levels between single cells,\n",
      "while GRIA2 editing levels vary.\n",
      "['inoFISH', 'direct visualization', 'quantification']\n",
      "Elapsed time: 8.41 seconds\n",
      "-------------------\n",
      "28628127\n",
      "RosettaES: a sampling strategy enabling automated interpretation of difficult cryo-EM maps.\n",
      "Accurate atomic modeling of macromolecular structures into cryo-electron microscopy (cryo-EM) maps\n",
      "is a major challenge, as the moderate resolution makes accurate placement of atoms difficult. We\n",
      "present Rosetta enumerative sampling (RosettaES), an automated tool that uses a fragment-based\n",
      "sampling strategy for de novo model completion of macromolecular structures from cryo-EM density\n",
      "maps at 3-5-Å resolution. On a benchmark set of nine proteins, RosettaES was able to identify near-\n",
      "native conformations in 85% of segments. RosettaES was also used to determine models for three\n",
      "challenging macromolecular structures.\n",
      "['cryo-electron microscopy',\n",
      " 'fragment-based sampling strategy',\n",
      " 'de novo model completion']\n",
      "Elapsed time: 8.15 seconds\n",
      "-------------------\n",
      "28628128\n",
      "Adaptive optics improves multiphoton super-resolution imaging.\n",
      "We improve multiphoton structured illumination microscopy using a nonlinear guide star to determine\n",
      "optical aberrations and a deformable mirror to correct them. We demonstrate our method on bead\n",
      "phantoms, cells in collagen gels, nematode larvae and embryos, Drosophila brain, and zebrafish\n",
      "embryos. Peak intensity is increased (up to 40-fold) and resolution recovered (up to 176 ± 10 nm\n",
      "laterally, 729 ± 39 nm axially) at depths ∼250 μm from the coverslip surface.\n",
      "['multiphoton structured illumination microscopy',\n",
      " 'nonlinear guide star',\n",
      " 'deformable mirror',\n",
      " 'multiphoton super-resolution imaging']\n",
      "Elapsed time: 8.40 seconds\n",
      "-------------------\n",
      "28628129\n",
      "High-speed fixed-target serial virus crystallography.\n",
      "We report a method for serial X-ray crystallography at X-ray free-electron lasers (XFELs), which\n",
      "allows for full use of the current 120-Hz repetition rate of the Linear Coherent Light Source\n",
      "(LCLS). Using a micropatterned silicon chip in combination with the high-speed Roadrunner goniometer\n",
      "for sample delivery, we were able to determine the crystal structures of the picornavirus bovine\n",
      "enterovirus 2 (BEV2) and the cytoplasmic polyhedrosis virus type 18 polyhedrin, with total data\n",
      "collection times of less than 14 and 10 min, respectively. Our method requires only micrograms of\n",
      "sample and should therefore broaden the applicability of serial femtosecond crystallography to\n",
      "challenging projects for which only limited sample amounts are available. By synchronizing the\n",
      "sample exchange to the XFEL repetition rate, our method allows for most efficient use of the limited\n",
      "beam time available at XFELs and should enable a substantial increase in sample throughput at these\n",
      "facilities.\n",
      "['X-ray crystallography',\n",
      " 'serial X-ray crystallography',\n",
      " 'micropatterned silicon chip',\n",
      " 'Roadrunner goniometer',\n",
      " 'sample delivery']\n",
      "Elapsed time: 11.32 seconds\n",
      "-------------------\n",
      "28650477\n",
      "Video rate volumetric Ca<sup>2+</sup> imaging across cortex using seeded iterative demixing (SID)\n",
      "microscopy.\n",
      "Light-field microscopy (LFM) is a scalable approach for volumetric Ca<sup>2+</sup> imaging with high\n",
      "volumetric acquisition rates (up to 100 Hz). Although the technology has enabled whole-brain\n",
      "Ca<sup>2+</sup> imaging in semi-transparent specimens, tissue scattering has limited its application\n",
      "in the rodent brain. We introduce seeded iterative demixing (SID), a computational source-extraction\n",
      "technique that extends LFM to the mammalian cortex. SID can capture neuronal dynamics in vivo within\n",
      "a volume of 900 × 900 × 260 μm located as deep as 380 μm in the mouse cortex or hippocampus at a\n",
      "30-Hz volume rate while discriminating signals from neurons as close as 20 μm apart, at a\n",
      "computational cost three orders of magnitude less than that of frame-by-frame image reconstruction.\n",
      "We expect that the simplicity and scalability of LFM, coupled with the performance of SID, will open\n",
      "up a range of applications including closed-loop experiments.\n",
      "['light-field microscopy',\n",
      " 'seed iterative demixing (SID)',\n",
      " 'video rate volumetric Ca2+ imaging']\n",
      "Elapsed time: 13.84 seconds\n",
      "-------------------\n",
      "28650478\n",
      "A DNA origami platform for quantifying protein copy number in super-resolution.\n",
      "Single-molecule-based super-resolution microscopy offers researchers a unique opportunity to\n",
      "quantify protein copy number with nanoscale resolution. However, while fluorescent proteins have\n",
      "been characterized for quantitative imaging using calibration standards, similar calibration tools\n",
      "for immunofluorescence with small organic fluorophores are lacking. Here we show that DNA origami,\n",
      "in combination with GFP antibodies, is a versatile platform for calibrating fluorophore and antibody\n",
      "labeling efficiency to quantify protein copy number in cellular contexts using super-resolution\n",
      "microscopy.\n",
      "['DNA origami',\n",
      " 'single-molecule-based super-resolution microscopy',\n",
      " 'fluorescent proteins',\n",
      " 'immunofluorescence',\n",
      " 'GFP antibodies']\n",
      "Elapsed time: 9.24 seconds\n",
      "-------------------\n",
      "28661494\n",
      "Assessing phototoxicity in live fluorescence imaging.\n",
      "Are the answers to biological questions obtained via live fluorescence microscopy substantially\n",
      "affected by phototoxicity? Although a single set of standards for assessing phototoxicity cannot\n",
      "exist owing to the breadth of samples and experimental questions associated with biological imaging,\n",
      "we need quantitative, practical assessments and reporting standards to ensure that imaging has a\n",
      "minimal impact on observed biological processes and sample health. Here we discuss the problem of\n",
      "phototoxicity in biology and suggest guidelines to improve its reporting and assessment.\n",
      "['live fluorescence microscopy']\n",
      "Elapsed time: 6.73 seconds\n",
      "-------------------\n",
      "28671674\n",
      "Addressing preferred specimen orientation in single-particle cryo-EM through tilting.\n",
      "We present a strategy for tackling preferred specimen orientation in single-particle cryogenic\n",
      "electron microscopy by employing tilts during data collection. We also describe a tool to quantify\n",
      "the resulting directional resolution using 3D Fourier shell correlation volumes. We applied these\n",
      "methods to determine the structures at near-atomic resolution of the influenza hemagglutinin trimer,\n",
      "which adopts a highly preferred specimen orientation, and of ribosomal biogenesis intermediates,\n",
      "which adopt moderately preferred orientations.\n",
      "['cryo-EM', 'tilting', 'Fourier shell correlation volumes']\n",
      "Elapsed time: 7.23 seconds\n",
      "-------------------\n",
      "28775673\n",
      "The Image Data Resource: A Bioimage Data Integration and Publication Platform.\n",
      "Access to primary research data is vital for the advancement of science. To extend the data types\n",
      "supported by community repositories, we built a prototype Image Data Resource (IDR) that collects\n",
      "and integrates imaging data acquired across many different imaging modalities. IDR links data from\n",
      "several imaging modalities, including high-content screening, super-resolution and time-lapse\n",
      "microscopy, digital pathology, public genetic or chemical databases, and cell and tissue phenotypes\n",
      "expressed using controlled ontologies. Using this integration, IDR facilitates the analysis of gene\n",
      "networks and reveals functional interactions that are inaccessible to individual studies. To enable\n",
      "re-analysis, we also established a computational resource based on Jupyter notebooks that allows\n",
      "remote access to the entire IDR. IDR is also an open source platform that others can use to publish\n",
      "their own image data. Thus IDR provides both a novel on-line resource and a software infrastructure\n",
      "that promotes and extends publication and re-analysis of scientific image data.\n",
      "[{'name': 'high-content screening'},\n",
      " {'name': 'super-resolution microscopy'},\n",
      " {'name': 'time-lapse microscopy'},\n",
      " {'name': 'digital pathology'}]\n",
      "Elapsed time: 10.94 seconds\n",
      "-------------------\n",
      "28783155\n",
      "histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data.\n",
      "Single-cell, spatially resolved omics analysis of tissues is poised to transform biomedical research\n",
      "and clinical practice. We have developed an open-source, computational histology topography\n",
      "cytometry analysis toolbox (histoCAT) to enable interactive, quantitative, and comprehensive\n",
      "exploration of individual cell phenotypes, cell-cell interactions, microenvironments, and\n",
      "morphological structures within intact tissues. We highlight the unique abilities of histoCAT\n",
      "through analysis of highly multiplexed mass cytometry images of human breast cancer tissues.\n",
      "['histology',\n",
      " 'cytometry',\n",
      " 'mass cytometry',\n",
      " 'multiplex image cytometry',\n",
      " 'computational histology',\n",
      " 'topography analysis']\n",
      "Elapsed time: 7.66 seconds\n",
      "-------------------\n",
      "28825703\n",
      "Virtual reality for freely moving animals.\n",
      "Standard animal behavior paradigms incompletely mimic nature and thus limit our understanding of\n",
      "behavior and brain function. Virtual reality (VR) can help, but it poses challenges. Typical VR\n",
      "systems require movement restrictions but disrupt sensorimotor experience, causing neuronal and\n",
      "behavioral alterations. We report the development of FreemoVR, a VR system for freely moving\n",
      "animals. We validate immersive VR for mice, flies, and zebrafish. FreemoVR allows instant,\n",
      "disruption-free environmental reconfigurations and interactions between real organisms and computer-\n",
      "controlled agents. Using the FreemoVR platform, we established a height-aversion assay in mice and\n",
      "studied visuomotor effects in Drosophila and zebrafish. Furthermore, by photorealistically mimicking\n",
      "zebrafish we discovered that effective social influence depends on a prospective leader balancing\n",
      "its internally preferred directional choice with social interaction. FreemoVR technology facilitates\n",
      "detailed investigations into neural function and behavior through the precise manipulation of\n",
      "sensorimotor feedback loops in unrestrained animals.\n",
      "['Virtual reality',\n",
      " 'FreemoVR',\n",
      " 'height-aversion assay',\n",
      " 'visuomotor effects',\n",
      " 'photorealistic mimicking']\n",
      "Elapsed time: 11.04 seconds\n",
      "-------------------\n",
      "28825705\n",
      "Reversed graph embedding resolves complex single-cell trajectories.\n",
      "Single-cell trajectories can unveil how gene regulation governs cell fate decisions. However,\n",
      "learning the structure of complex trajectories with multiple branches remains a challenging\n",
      "computational problem. We present Monocle 2, an algorithm that uses reversed graph embedding to\n",
      "describe multiple fate decisions in a fully unsupervised manner. We applied Monocle 2 to two studies\n",
      "of blood development and found that mutations in the genes encoding key lineage transcription\n",
      "factors divert cells to alternative fates.\n",
      "['reversed graph embedding',\n",
      " 'single-cell trajectories',\n",
      " 'gene regulation',\n",
      " 'cell fate decisions']\n",
      "Elapsed time: 7.04 seconds\n",
      "-------------------\n",
      "28846087\n",
      "Convolutional neural networks for automated annotation of cellular cryo-electron tomograms.\n",
      "Cellular electron cryotomography offers researchers the ability to observe macromolecules frozen in\n",
      "action in situ, but a primary challenge with this technique is identifying molecular components\n",
      "within the crowded cellular environment. We introduce a method that uses neural networks to\n",
      "dramatically reduce the time and human effort required for subcellular annotation and feature\n",
      "extraction. Subsequent subtomogram classification and averaging yield in situ structures of\n",
      "molecular components of interest. The method is available in the EMAN2.2 software package.\n",
      "['convolutional neural networks',\n",
      " 'electron cryotomography',\n",
      " 'subcellular annotation',\n",
      " 'feature extraction',\n",
      " 'subtomogram classification',\n",
      " 'averaging']\n",
      "Elapsed time: 8.09 seconds\n",
      "-------------------\n",
      "28846089\n",
      "Oscillatory stimuli differentiate adapting circuit topologies.\n",
      "Biology emerges from interactions between molecules, which are challenging to elucidate with current\n",
      "techniques. An orthogonal approach is to probe for 'response signatures' that identify specific\n",
      "circuit motifs. For example, bistability, hysteresis, or irreversibility are used to detect positive\n",
      "feedback loops. For adapting systems, such signatures are not known. Only two circuit motifs\n",
      "generate adaptation: negative feedback loops (NFLs) and incoherent feed-forward loops (IFFLs). On\n",
      "the basis of computational testing and mathematical proofs, we propose differential signatures: in\n",
      "response to oscillatory stimulation, NFLs but not IFFLs show refractory-period stabilization\n",
      "(robustness to changes in stimulus duration) or period skipping. Applying this approach to yeast, we\n",
      "identified the circuit dominating cell cycle timing. In Caenorhabditis elegans AWA neurons, which\n",
      "are crucial for chemotaxis, we uncovered a Ca<sup>2+</sup> NFL leading to adaptation that would be\n",
      "difficult to find by other means. These response signatures allow direct access to the outlines of\n",
      "the wiring diagrams of adapting systems.\n",
      "['refractory-period stabilization', 'period skipping']\n",
      "Elapsed time: 11.16 seconds\n",
      "-------------------\n",
      "28858338\n",
      "Data-analysis strategies for image-based cell profiling.\n",
      "Image-based cell profiling is a high-throughput strategy for the quantification of phenotypic\n",
      "differences among a variety of cell populations. It paves the way to studying biological systems on\n",
      "a large scale by using chemical and genetic perturbations. The general workflow for this technology\n",
      "involves image acquisition with high-throughput microscopy systems and subsequent image processing\n",
      "and analysis. Here, we introduce the steps required to create high-quality image-based (i.e.,\n",
      "morphological) profiles from a collection of microscopy images. We recommend techniques that have\n",
      "proven useful in each stage of the data analysis process, on the basis of the experience of 20\n",
      "laboratories worldwide that are refining their image-based cell-profiling methodologies in pursuit\n",
      "of biological discovery. The recommended techniques cover alternatives that may suit various\n",
      "biological goals, experimental designs, and laboratories' preferences.\n",
      "['image acquisition', 'image processing', 'image analysis']\n",
      "Elapsed time: 8.66 seconds\n",
      "-------------------\n",
      "28869755\n",
      "Sampling strategies to capture single-cell heterogeneity.\n",
      "Advances in single-cell technologies have highlighted the prevalence and biological significance of\n",
      "cellular heterogeneity. A critical question researchers face is how to design experiments that\n",
      "faithfully capture the true range of heterogeneity from samples of cellular populations. Here we\n",
      "develop a data-driven approach, illustrated in the context of image data, that estimates the\n",
      "sampling depth required for prospective investigations of single-cell heterogeneity from an existing\n",
      "collection of samples.\n",
      "['image data', 'single-cell technologies']\n",
      "Elapsed time: 6.57 seconds\n",
      "-------------------\n",
      "28869756\n",
      "Red-shifted luciferase-luciferin pairs for enhanced bioluminescence imaging.\n",
      "Red-shifted bioluminescence reporters are desirable for biological imaging. We describe the\n",
      "development of red-shifted luciferins based on synthetic coelenterazine analogs and corresponding\n",
      "mutants of NanoLuc that enable bright bioluminescence. One pair in particular showed superior in\n",
      "vitro and in vivo sensitivity over commonly used bioluminescence reporters. We adapted this pair to\n",
      "develop a bioluminescence resonance-energy-based Antares reporter called Antares2, which offers\n",
      "improved signal from deep tissues.\n",
      "['bioluminescence',\n",
      " 'bioluminescence resonance-energy-based Antares reporter',\n",
      " 'in vitro',\n",
      " 'in vivo',\n",
      " 'sensitivity']\n",
      "Elapsed time: 8.44 seconds\n",
      "-------------------\n",
      "28869757\n",
      "A general method to fine-tune fluorophores for live-cell and in vivo imaging.\n",
      "Pushing the frontier of fluorescence microscopy requires the design of enhanced fluorophores with\n",
      "finely tuned properties. We recently discovered that incorporation of four-membered azetidine rings\n",
      "into classic fluorophore structures elicits substantial increases in brightness and photostability,\n",
      "resulting in the Janelia Fluor (JF) series of dyes. We refined and extended this strategy, finding\n",
      "that incorporation of 3-substituted azetidine groups allows rational tuning of the spectral and\n",
      "chemical properties of rhodamine dyes with unprecedented precision. This strategy allowed us to\n",
      "establish principles for fine-tuning the properties of fluorophores and to develop a palette of new\n",
      "fluorescent and fluorogenic labels with excitation ranging from blue to the far-red. Our results\n",
      "demonstrate the versatility of these new dyes in cells, tissues and animals.\n",
      "['fluorescence microscopy',\n",
      " 'live-cell imaging',\n",
      " 'in vivo imaging',\n",
      " 'blue excitation',\n",
      " 'far-red excitation']\n",
      "Elapsed time: 10.40 seconds\n",
      "-------------------\n",
      "28869758\n",
      "Internally ratiometric fluorescent sensors for evaluation of intracellular GTP levels and\n",
      "distribution.\n",
      "GTP is a major regulator of multiple cellular processes, but tools for quantitative evaluation of\n",
      "GTP levels in live cells have not been available. We report the development and characterization of\n",
      "genetically encoded GTP sensors, which we constructed by inserting a circularly permuted yellow\n",
      "fluorescent protein (cpYFP) into a region of the bacterial G protein FeoB that undergoes a GTP-\n",
      "driven conformational change. GTP binding to these sensors results in a ratiometric change in their\n",
      "fluorescence, thereby providing an internally normalized response to changes in GTP levels while\n",
      "minimally perturbing those levels. Mutations introduced into FeoB to alter its affinity for GTP\n",
      "created a series of sensors with a wide dynamic range. Critically, in mammalian cells the sensors\n",
      "showed consistent changes in ratiometric signal upon depletion or restoration of GTP pools. We show\n",
      "that these GTP evaluators (GEVALs) are suitable for detection of spatiotemporal changes in GTP\n",
      "levels in living cells and for high-throughput screening of molecules that modulate GTP levels.\n",
      "['fluorescence', 'ratiometric fluorescence', 'live cell imaging']\n",
      "Elapsed time: 10.36 seconds\n",
      "-------------------\n",
      "28892088\n",
      "Pan-neuronal calcium imaging with cellular resolution in freely swimming zebrafish.\n",
      "Calcium imaging with cellular resolution typically requires an animal to be tethered under a\n",
      "microscope, which substantially restricts the range of behaviors that can be studied. To expand the\n",
      "behavioral repertoire amenable to imaging, we have developed a tracking microscope that enables\n",
      "whole-brain calcium imaging with cellular resolution in freely swimming larval zebrafish. This\n",
      "microscope uses infrared imaging to track a target animal in a behavior arena. On the basis of the\n",
      "predicted trajectory of the animal, we applied optimal control theory to a motorized stage system to\n",
      "cancel brain motion in three dimensions. We combined this motion-cancellation system with\n",
      "differential illumination focal filtering, a variant of HiLo microscopy, which enabled us to image\n",
      "the brain of a freely swimming larval zebrafish for more than an hour. This work expands the\n",
      "repertoire of natural behaviors that can be studied with cellular-resolution calcium imaging to\n",
      "potentially include spatial navigation, social behavior, feeding and reward.\n",
      "['infrared imaging',\n",
      " 'optimal control theory',\n",
      " 'differential illumination focal filtering',\n",
      " 'HiLo microscopy']\n",
      "Elapsed time: 10.35 seconds\n",
      "-------------------\n",
      "28945703\n",
      "Autofocusing MALDI mass spectrometry imaging of tissue sections and 3D chemical topography of\n",
      "nonflat surfaces.\n",
      "We describe an atmospheric pressure matrix-assisted laser desorption-ionization mass spectrometry\n",
      "imaging system that uses long-distance laser triangulation on a micrometer scale to simultaneously\n",
      "obtain topographic and molecular information from 3D surfaces. We studied the topographic\n",
      "distribution of compounds on irregular 3D surfaces of plants and parasites, and we imaged nonplanar\n",
      "tissue sections with high lateral resolution, thereby eliminating height-related signal artifacts.\n",
      "['MALDI mass spectrometry imaging',\n",
      " 'long-distance laser triangulation',\n",
      " 'topographic and molecular information',\n",
      " 'atmospheric pressure matrix-assisted laser desorption-ionization']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "28945705\n",
      "Thiol-linked alkylation of RNA to assess expression dynamics.\n",
      "Gene expression profiling by high-throughput sequencing reveals qualitative and quantitative changes\n",
      "in RNA species at steady state but obscures the intracellular dynamics of RNA transcription,\n",
      "processing and decay. We developed thiol(SH)-linked alkylation for the metabolic sequencing of RNA\n",
      "(SLAM seq), an orthogonal-chemistry-based RNA sequencing technology that detects 4-thiouridine\n",
      "(s<sup>4</sup>U) incorporation in RNA species at single-nucleotide resolution. In combination with\n",
      "well-established metabolic RNA labeling protocols and coupled to standard, low-input, high-\n",
      "throughput RNA sequencing methods, SLAM seq enabled rapid access to RNA-polymerase-II-dependent gene\n",
      "expression dynamics in the context of total RNA. We validated the method in mouse embryonic stem\n",
      "cells by showing that the RNA-polymerase-II-dependent transcriptional output scaled with\n",
      "Oct4/Sox2/Nanog-defined enhancer activity, and we provide quantitative and mechanistic evidence for\n",
      "transcript-specific RNA turnover mediated by post-transcriptional gene regulatory pathways initiated\n",
      "by microRNAs and N<sup>6</sup>-methyladenosine. SLAM seq facilitates the dissection of fundamental\n",
      "mechanisms that control gene expression in an accessible, cost-effective and scalable manner.\n",
      "['RNA sequencing',\n",
      " 'high-throughput sequencing',\n",
      " 'metabolic sequencing',\n",
      " '4-thiouridine incorporation',\n",
      " 'post-transcriptional gene regulatory pathways',\n",
      " 'microRNAs',\n",
      " 'N&lt;sup&gt;6&lt;/sup&gt;-methyladenosine']\n",
      "Elapsed time: 15.35 seconds\n",
      "-------------------\n",
      "28945706\n",
      "Multiplexing molecular tension sensors reveals piconewton force gradient across talin-1.\n",
      "Förster resonance energy transfer (FRET)-based tension sensor modules (TSMs) are available for\n",
      "investigating how distinct proteins bear mechanical forces in cells. Yet, forces in the single\n",
      "piconewton (pN) regime remain difficult to resolve, and tools for multiplexed tension sensing are\n",
      "lacking. Here, we report the generation and calibration of a genetically encoded, FRET-based\n",
      "biosensor called FL-TSM, which is characterized by a near-digital force response and increased\n",
      "sensitivity at 3-5 pN. In addition, we present a method allowing the simultaneous evaluation of\n",
      "coexpressed tension sensor constructs using two-color fluorescence lifetime microscopy. Finally, we\n",
      "introduce a procedure to calculate the fraction of mechanically engaged molecules within cells.\n",
      "Application of these techniques to new talin biosensors reveals an intramolecular tension gradient\n",
      "across talin-1 that is established upon integrin-mediated cell adhesion. The tension gradient is\n",
      "actomyosin- and vinculin-dependent and sensitive to the rigidity of the extracellular environment.\n",
      "['fluorescence lifetime microscopy',\n",
      " 'two-color fluorescence lifetime microscopy',\n",
      " 'biosensor']\n",
      "Elapsed time: 11.38 seconds\n",
      "-------------------\n",
      "28967889\n",
      "NeuBtracker-imaging neurobehavioral dynamics in freely behaving fish.\n",
      "A long-standing objective in neuroscience has been to image distributed neuronal activity in freely\n",
      "behaving animals. Here we introduce NeuBtracker, a tracking microscope for simultaneous imaging of\n",
      "neuronal activity and behavior of freely swimming fluorescent reporter fish. We showcase the value\n",
      "of NeuBtracker for screening neurostimulants with respect to their combined neuronal and behavioral\n",
      "effects and for determining spontaneous and stimulus-induced spatiotemporal patterns of neuronal\n",
      "activation during naturalistic behavior.\n",
      "['simultaneous imaging',\n",
      " 'tracking microscope',\n",
      " 'freely swimming fluorescent reporter fish']\n",
      "Elapsed time: 7.22 seconds\n",
      "-------------------\n",
      "28991891\n",
      "Achieving better-than-3-Å resolution by single-particle cryo-EM at 200 keV.\n",
      "Nearly all single-particle cryo-EM structures resolved to better than 4-Å resolution have been\n",
      "determined using 300-keV transmission electron microscopes (TEMs). We demonstrate that it is\n",
      "possible to obtain reconstructions of macromolecular complexes of different sizes to better than 3-Å\n",
      "resolution using a 200-keV TEM. These structures are of sufficient quality to unambiguously assign\n",
      "amino acid rotameric conformations and identify ordered water molecules.\n",
      "['single-particle cryo-EM', 'transmission electron microscopes (TEMs)']\n",
      "Elapsed time: 6.96 seconds\n",
      "-------------------\n",
      "29039416\n",
      "Antibodies to biotin enable large-scale detection of biotinylation sites on proteins.\n",
      "Although purification of biotinylated molecules is highly efficient, identifying specific sites of\n",
      "biotinylation remains challenging. We show that anti-biotin antibodies enable unprecedented\n",
      "enrichment of biotinylated peptides from complex peptide mixtures. Live-cell proximity labeling\n",
      "using APEX peroxidase followed by anti-biotin enrichment and mass spectrometry yielded over 1,600\n",
      "biotinylation sites on hundreds of proteins, an increase of more than 30-fold in the number of\n",
      "biotinylation sites identified compared to streptavidin-based enrichment of proteins.\n",
      "['mass spectrometry',\n",
      " 'live-cell proximity labeling',\n",
      " 'APEX peroxidase',\n",
      " 'streptavidin-based enrichment']\n",
      "Elapsed time: 8.93 seconds\n",
      "-------------------\n",
      "29039418\n",
      "Rapid nonlinear image scanning microscopy.\n",
      "Image scanning microscopy (ISM) doubles the resolution of a conventional confocal microscope for\n",
      "super-resolution imaging. Here, we describe an all-optical ISM design based on rescanning microscopy\n",
      "for two-photon-excited fluorescence and second-harmonic generation that allows straightforward\n",
      "implementation into existing microscopes. The design offers improved sensitivity and high frame\n",
      "rates relative to those of existing systems. We demonstrate its utility using fixed and living\n",
      "specimens as well as collagen hydrogels.\n",
      "['Rapid nonlinear image scanning microscopy',\n",
      " 'all-optical ISM design',\n",
      " 'second-harmonic generation',\n",
      " 'two-photon-excited fluorescence']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "29083400\n",
      "Localization-based super-resolution imaging meets high-content screening.\n",
      "Single-molecule localization microscopy techniques have proven to be essential tools for\n",
      "quantitatively monitoring biological processes at unprecedented spatial resolution. However, these\n",
      "techniques are very low throughput and are not yet compatible with fully automated, multiparametric\n",
      "cellular assays. This shortcoming is primarily due to the huge amount of data generated during\n",
      "imaging and the lack of software for automation and dedicated data mining. We describe an automated\n",
      "quantitative single-molecule-based super-resolution methodology that operates in standard multiwell\n",
      "plates and uses analysis based on high-content screening and data-mining software. The workflow is\n",
      "compatible with fixed- and live-cell imaging and allows extraction of quantitative data like\n",
      "fluorophore photophysics, protein clustering or dynamic behavior of biomolecules. We demonstrate\n",
      "that the method is compatible with high-content screening using 3D dSTORM and DNA-PAINT based super-\n",
      "resolution microscopy as well as single-particle tracking.\n",
      "['single-molecule localization microscopy',\n",
      " 'high-content screening',\n",
      " 'data-mining software',\n",
      " 'fixed-cell imaging',\n",
      " 'live-cell imaging',\n",
      " '3D dSTORM microscopy',\n",
      " 'DNA-PAINT based super-resolution microscopy',\n",
      " 'single-particle tracking']\n",
      "Elapsed time: 11.89 seconds\n",
      "-------------------\n",
      "29083401\n",
      "High-throughput, image-based screening of pooled genetic-variant libraries.\n",
      "We report a high-throughput screening method that allows diverse genotypes and corresponding\n",
      "phenotypes to be imaged in individual cells. We achieve genotyping by introducing barcoded genetic\n",
      "variants into cells as pooled libraries and reading the barcodes out using massively multiplexed\n",
      "fluorescence in situ hybridization. To demonstrate the power of image-based pooled screening, we\n",
      "identified brighter and more photostable variants of the fluorescent protein YFAST among 60,000\n",
      "variants.\n",
      "['massively multiplexed fluorescence in situ hybridization',\n",
      " 'fluorescent protein YFAST']\n",
      "Elapsed time: 7.14 seconds\n",
      "-------------------\n",
      "29083403\n",
      "An objective comparison of cell-tracking algorithms.\n",
      "We present a combined report on the results of three editions of the Cell Tracking Challenge, an\n",
      "ongoing initiative aimed at promoting the development and objective evaluation of cell segmentation\n",
      "and tracking algorithms. With 21 participating algorithms and a data repository consisting of 13\n",
      "data sets from various microscopy modalities, the challenge displays today's state-of-the-art\n",
      "methodology in the field. We analyzed the challenge results using performance measures for\n",
      "segmentation and tracking that rank all participating methods. We also analyzed the performance of\n",
      "all of the algorithms in terms of biological measures and practical usability. Although some methods\n",
      "scored high in all technical aspects, none obtained fully correct solutions. We found that methods\n",
      "that either take prior information into account using learning strategies or analyze cells in a\n",
      "global spatiotemporal video context performed better than other methods under the segmentation and\n",
      "tracking scenarios included in the challenge.\n",
      "['cell segmentation',\n",
      " 'cell tracking',\n",
      " 'microscopy modalities',\n",
      " 'global spatiotemporal video context']\n",
      "Elapsed time: 9.08 seconds\n",
      "-------------------\n",
      "29131162\n",
      "The 3D OrbiSIMS-label-free metabolic imaging with subcellular lateral resolution and high mass-\n",
      "resolving power.\n",
      "We report the development of a 3D OrbiSIMS instrument for label-free biomedical imaging. It combines\n",
      "the high spatial resolution of secondary ion mass spectrometry (SIMS; under 200 nm for inorganic\n",
      "species and under 2 μm for biomolecules) with the high mass-resolving power of an Orbitrap (>240,000\n",
      "at m/z 200). This allows exogenous and endogenous metabolites to be visualized in 3D with\n",
      "subcellular resolution. We imaged the distribution of neurotransmitters-gamma-aminobutyric acid,\n",
      "dopamine and serotonin-with high spectroscopic confidence in the mouse hippocampus. We also\n",
      "putatively annotated and mapped the subcellular localization of 29 sulfoglycosphingolipids and 45\n",
      "glycerophospholipids, and we confirmed lipid identities with tandem mass spectrometry. We\n",
      "demonstrated single-cell metabolomic profiling using rat alveolar macrophage cells incubated with\n",
      "different concentrations of the drug amiodarone, and we observed that the upregulation of\n",
      "phospholipid species and cholesterol is correlated with the accumulation of amiodarone.\n",
      "['SIMS', 'Orbitrap', '3D OrbiSIMS']\n",
      "Elapsed time: 12.11 seconds\n",
      "-------------------\n",
      "29131163\n",
      "Profiling the transcriptome with RNA SPOTs.\n",
      "Single-molecule FISH (smFISH) has been the gold standard for quantifying individual transcript\n",
      "abundances. Here, we scale up multiplexed smFISH to the transcriptome level and profile 10,212\n",
      "different mRNAs from mouse fibroblast and embryonic stem cells. This method, called RNA sequential\n",
      "probing of targets (SPOTs), provides an accurate, flexible, and low-cost alternative to sequencing\n",
      "for profiling transcriptomes.\n",
      "['smFISH', 'RNA SPOTs', 'sequential probing of targets (SPOTs)']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "29131164\n",
      "An improved MS2 system for accurate reporting of the mRNA life cycle.\n",
      "The MS2-MCP system enables researchers to image multiple steps of the mRNA life cycle with high\n",
      "temporal and spatial resolution. However, for short-lived mRNAs, the tight binding of the MS2 coat\n",
      "protein (MCP) to the MS2 binding sites (MBS) protects the RNA from being efficiently degraded, and\n",
      "this confounds the study of mRNA regulation. Here, we describe a reporter system (MBSV6) with\n",
      "reduced affinity for the MCP, which allows mRNA degradation while preserving single-molecule\n",
      "detection determined by single-molecule FISH (smFISH) or live imaging. Constitutive mRNAs (MDN1 and\n",
      "DOA1) and highly-regulated mRNAs (GAL1 and ASH1) endogenously tagged with MBSV6 in Saccharomyces\n",
      "cerevisiae degrade normally. As a result, short-lived mRNAs were imaged throughout their complete\n",
      "life cycle. The MBSV6 reporter revealed that, in contrast to previous findings, coordinated\n",
      "recruitment of mRNAs at specialized structures such as P-bodies during stress did not occur, and\n",
      "mRNA degradation was heterogeneously distributed in the cytoplasm.\n",
      "['single-molecule FISH (smFISH)', 'live imaging']\n",
      "Elapsed time: 12.39 seconds\n",
      "-------------------\n",
      "29176592\n",
      "A permanent window for the murine lung enables high-resolution imaging of cancer metastasis.\n",
      "Stable, high-resolution intravital imaging of the lung has become possible through the utilization\n",
      "of vacuum-stabilized imaging windows. However, this technique is extremely invasive and limited to\n",
      "only hours in duration. Here we describe a minimally invasive, permanently implantable window for\n",
      "high-resolution intravital imaging of the murine lung that allows the mouse to survive surgery,\n",
      "recover from anesthesia, and breathe independently. Compared to vacuum-stabilized windows, this\n",
      "window produces the same high-quality images without vacuum-induced artifacts; it is also less\n",
      "invasive, which allows imaging of the same lung tissue over a period of weeks. We further adapt the\n",
      "technique of microcartography for reliable relocalization of the same cells longitudinally. Using\n",
      "commonly employed experimental, as well as more clinically relevant, spontaneous metastasis models,\n",
      "we visualize all stages of metastatic seeding, including: tumor cell arrival; extravasation; growth\n",
      "and progression to micrometastases; as well as tumor microenvironment of metastasis function, the\n",
      "hallmark of hematogenous dissemination of tumor cells.\n",
      "['vacuum-stabilized imaging windows', 'microcartography']\n",
      "Elapsed time: 11.07 seconds\n",
      "-------------------\n",
      "29256494\n",
      "Biotinylation by antibody recognition-a method for proximity labeling.\n",
      "The high-throughput detection of organelle composition and proteomic mapping of protein environment\n",
      "directly from primary tissue as well as the identification of interactors of insoluble proteins that\n",
      "form higher-order structures have remained challenges in biological research. We report a proximity-\n",
      "based labeling approach that uses an antibody to a target antigen to guide biotin deposition onto\n",
      "adjacent proteins in fixed cells and primary tissues, which allows proteins in close proximity to\n",
      "the target antigen to be captured and identified by mass spectrometry. We demonstrated the\n",
      "specificity and sensitivity of our method by examining the well-studied mitochondrial matrix. We\n",
      "then used the method to profile the dynamic interactome of lamin A/C in multiple cell and tissue\n",
      "types under various treatment conditions. The ability to detect proximal proteins and putative\n",
      "interactors in intact tissues, and to quantify changes caused by different conditions or in the\n",
      "presence of disease mutations, can provide a window into cell biology and disease pathogenesis.\n",
      "['mass spectrometry',\n",
      " 'biotin deposition',\n",
      " 'antibody recognition',\n",
      " 'proximity labeling']\n",
      "Elapsed time: 10.21 seconds\n",
      "-------------------\n",
      "29256495\n",
      "Mapping the 3D orientation of piconewton integrin traction forces.\n",
      "Mechanical forces are integral to many biological processes; however, current techniques cannot map\n",
      "the magnitude and direction of piconewton molecular forces. Here, we describe molecular force\n",
      "microscopy, leveraging molecular tension probes and fluorescence polarization microscopy to measure\n",
      "the magnitude and 3D orientation of cellular forces. We mapped the orientation of integrin-based\n",
      "traction forces in mouse fibroblasts and human platelets, revealing alignment between the\n",
      "organization of force-bearing structures and their force orientations.\n",
      "['molecular force microscopy', 'fluorescence polarization microscopy']\n",
      "Elapsed time: 6.93 seconds\n",
      "-------------------\n",
      "29298292\n",
      "Imaging organoids: a bright future ahead.\n",
      "Organogenesis, tissue homeostasis and organ function involve complex spatial cellular organization\n",
      "and tissue dynamics. The underlying mechanisms of these processes, and how they are disrupted in\n",
      "disease, are challenging to address in vivo and ethically impossible to study in human. Organoids,\n",
      "three-dimensional (3D) stem cell cultures that self-organize into ex vivo 'mini-organs', now open a\n",
      "new window onto cellular processes within tissue. Light microscopy is a powerful approach to probe\n",
      "the cellular complexity that can be modeled with organoids. This combination of tools is already\n",
      "leading to exciting synergies in stem cell and cancer research.\n",
      "['light microscopy', 'none']\n",
      "Elapsed time: 7.14 seconds\n",
      "-------------------\n",
      "29320486\n",
      "Systematic characterization of maturation time of fluorescent proteins in living cells.\n",
      "The slow maturation time of fluorescent proteins (FPs) limits the temporal accuracy of measurements\n",
      "of rapid processes such as gene expression dynamics and effectively reduces fluorescence signal in\n",
      "growing cells. We used high-precision time-lapse microscopy to characterize the maturation kinetics\n",
      "of 50 FPs that span the visible spectrum at two different temperatures in Escherichia coli cells. We\n",
      "identified fast-maturing FPs from this set that yielded the highest signal-to-noise ratio and\n",
      "temporal resolution in individual growing cells.\n",
      "['high-precision time-lapse microscopy', 'visible spectrum']\n",
      "Elapsed time: 6.98 seconds\n",
      "-------------------\n",
      "29320487\n",
      "Multicolor quantitative confocal imaging cytometry.\n",
      "Multicolor 3D quantitative imaging of large tissue volumes is necessary to understand tissue\n",
      "development and organization as well as interactions between distinct cell types in situ. However,\n",
      "tissue imaging remains technically challenging, particularly imaging of bone and marrow. Here, we\n",
      "describe a pipeline to reproducibly generate high-dimensional quantitative data from bone and bone\n",
      "marrow that may be extended to any tissue. We generate thick bone sections from adult mouse femurs\n",
      "with preserved tissue microarchitecture and demonstrate eight-color imaging using confocal\n",
      "microscopy without linear unmixing. We introduce XiT, an open-access software for fast and easy data\n",
      "curation, exploration and quantification of large imaging data sets with single-cell resolution. We\n",
      "describe how XiT can be used to correct for potential artifacts in quantitative 3D imaging, and we\n",
      "use the pipeline to measure the spatial relationship between hematopoietic cells, bone matrix and\n",
      "marrow Schwann cells.\n",
      "['confocal microscopy',\n",
      " 'quantitative 3D imaging',\n",
      " 'single-cell resolution',\n",
      " 'XiT software']\n",
      "Elapsed time: 9.59 seconds\n",
      "-------------------\n",
      "29334378\n",
      "Supermultiplexed optical imaging and barcoding with engineered polyynes.\n",
      "Optical multiplexing has a large impact in photonics, the life sciences and biomedicine. However,\n",
      "current technology is limited by a 'multiplexing ceiling' from existing optical materials. Here we\n",
      "engineered a class of polyyne-based materials for optical supermultiplexing. We achieved 20 distinct\n",
      "Raman frequencies, as 'Carbon rainbow', through rational engineering of conjugation length, bond-\n",
      "selective isotope doping and end-capping substitution of polyynes. With further probe\n",
      "functionalization, we demonstrated ten-color organelle imaging in individual living cells with high\n",
      "specificity, sensitivity and photostability. Moreover, we realized optical data storage and\n",
      "identification by combinatorial barcoding, yielding to our knowledge the largest number of distinct\n",
      "spectral barcodes to date. Therefore, these polyynes hold great promise in live-cell imaging and\n",
      "sorting as well as in high-throughput diagnostics and screening.\n",
      "['Raman frequencies',\n",
      " 'Carbon rainbow',\n",
      " 'organelle imaging',\n",
      " 'live-cell imaging',\n",
      " 'sorting',\n",
      " 'high-throughput diagnostics',\n",
      " 'screening']\n",
      "Elapsed time: 10.39 seconds\n",
      "-------------------\n",
      "29355846\n",
      "TimeLapse-seq: adding a temporal dimension to RNA sequencing through nucleoside recoding.\n",
      "RNA sequencing (RNA-seq) offers a snapshot of cellular RNA populations, but not temporal information\n",
      "about the sequenced RNA. Here we report TimeLapse-seq, which uses oxidative-nucleophilic-aromatic\n",
      "substitution to convert 4-thiouridine into cytidine analogs, yielding apparent U-to-C mutations that\n",
      "mark new transcripts upon sequencing. TimeLapse-seq is a single-molecule approach that is adaptable\n",
      "to many applications and reveals RNA dynamics and induced differential expression concealed in\n",
      "traditional RNA-seq.\n",
      "['RNA sequencing',\n",
      " 'TimeLapse-seq',\n",
      " 'oxidative-nucleophilic-aromatic substitution']\n",
      "Elapsed time: 8.53 seconds\n",
      "-------------------\n",
      "29355847\n",
      "EVIR: chimeric receptors that enhance dendritic cell cross-dressing with tumor antigens.\n",
      "We describe a lentivirus-encoded chimeric receptor, termed extracellular vesicle (EV)-internalizing\n",
      "receptor (EVIR), which enables the selective uptake of cancer-cell-derived EVs by dendritic cells\n",
      "(DCs). The EVIR enhances DC presentation of EV-associated tumor antigens to CD8<sup>+</sup> T cells\n",
      "primarily through MHCI recycling and cross-dressing. EVIRs should facilitate exploring the\n",
      "mechanisms and implications of horizontal transfer of tumor antigens to antigen-presenting cells.\n",
      "{'imagingTechniques': ['lentivirus-encoded chimeric receptor',\n",
      "                       'extracellular vesicle (EV)-internalizing receptor '\n",
      "                       '(EVIR)',\n",
      "                       'MHCI recycling',\n",
      "                       'cross-dressing']}\n",
      "Elapsed time: 9.72 seconds\n",
      "-------------------\n",
      "29377014\n",
      "STED super-resolved microscopy.\n",
      "Stimulated emission depletion (STED) microscopy provides subdiffraction resolution while preserving\n",
      "useful aspects of fluorescence microscopy, such as optical sectioning, and molecular specificity and\n",
      "sensitivity. However, sophisticated microscopy architectures and high illumination intensities have\n",
      "limited STED microscopy's widespread use in the past. Here we summarize the progress that is\n",
      "mitigating these problems and giving substantial momentum to STED microscopy applications. We\n",
      "discuss the future of this method in regard to spatiotemporal limits, live-cell imaging and\n",
      "combination with spectroscopy. Advances in these areas may elevate STED microscopy to a standard\n",
      "method for imaging in the life sciences.\n",
      "['STED microscopy',\n",
      " 'optical sectioning',\n",
      " 'molecular specificity and sensitivity']\n",
      "Elapsed time: 9.01 seconds\n",
      "-------------------\n",
      "29400715\n",
      "RNA-protein interaction detection in living cells.\n",
      "RNA-protein interactions play numerous roles in cellular function and disease. Here we describe RNA-\n",
      "protein interaction detection (RaPID), which uses proximity-dependent protein labeling, based on the\n",
      "BirA* biotin ligase, to rapidly identify the proteins that bind RNA sequences of interest in living\n",
      "cells. RaPID displays utility in multiple applications, including in evaluating protein binding to\n",
      "mutant RNA motifs in human genetic disorders, in uncovering potential post-transcriptional networks\n",
      "in breast cancer, and in discovering essential host proteins that interact with Zika virus RNA. To\n",
      "improve the BirA*-labeling component of RaPID, moreover, a new mutant BirA* was engineered from\n",
      "Bacillus subtilis, termed BASU, that enables >1,000-fold faster kinetics and >30-fold increased\n",
      "signal-to-noise ratio over the prior standard Escherichia coli BirA*, thereby enabling direct study\n",
      "of RNA-protein interactions in living cells on a timescale as short as 1 min.\n",
      "['proximity-dependent protein labeling',\n",
      " 'BirA* biotin ligase',\n",
      " 'BASU',\n",
      " 'direct study of RNA-protein interactions']\n",
      "Elapsed time: 11.00 seconds\n",
      "-------------------\n",
      "29457791\n",
      "Quantitative mapping and minimization of super-resolution optical imaging artifacts.\n",
      "Super-resolution microscopy depends on steps that can contribute to the formation of image\n",
      "artifacts, leading to misinterpretation of biological information. We present NanoJ-SQUIRREL, an\n",
      "ImageJ-based analytical approach that provides quantitative assessment of super-resolution image\n",
      "quality. By comparing diffraction-limited images and super-resolution equivalents of the same\n",
      "acquisition volume, this approach generates a quantitative map of super-resolution defects and can\n",
      "guide researchers in optimizing imaging parameters.\n",
      "['diffraction-limited images',\n",
      " 'super-resolution microscopy',\n",
      " 'ImageJ-based analytical approach']\n",
      "Elapsed time: 7.51 seconds\n",
      "-------------------\n",
      "29481550\n",
      "Cell-type specific sequencing of microRNAs from complex animal tissues.\n",
      "MicroRNAs (miRNAs) play an essential role in the post-transcriptional regulation of animal\n",
      "development and physiology. However, in vivo studies aimed at linking miRNA function to the biology\n",
      "of distinct cell types within complex tissues remain challenging, partly because in vivo miRNA-\n",
      "profiling methods lack cellular resolution. We report microRNome by methylation-dependent sequencing\n",
      "(mime-seq), an in vivo enzymatic small-RNA-tagging approach that enables high-throughput sequencing\n",
      "of tissue- and cell-type-specific miRNAs in animals. The method combines cell-type-specific\n",
      "3'-terminal 2'-O-methylation of animal miRNAs by a genetically encoded, plant-specific\n",
      "methyltransferase (HEN1), with chemoselective small-RNA cloning and high-throughput sequencing. We\n",
      "show that mime-seq uncovers the miRNomes of specific cells within Caenorhabditis elegans and\n",
      "Drosophila at unprecedented specificity and sensitivity, enabling miRNA profiling with single-cell\n",
      "resolution in whole animals. Mime-seq overcomes current challenges in cell-type-specific small-RNA\n",
      "profiling and provides novel entry points for understanding the function of miRNAs in spatially\n",
      "restricted physiological settings.\n",
      "['high-throughput sequencing',\n",
      " 'chemoselective small-RNA cloning',\n",
      " 'methylation-dependent sequencing (mime-seq)']\n",
      "Elapsed time: 14.56 seconds\n",
      "-------------------\n",
      "29481551\n",
      "A hybridization-chain-reaction-based method for amplifying immunosignals.\n",
      "Immunosignal hybridization chain reaction (isHCR) combines antibody-antigen interactions with\n",
      "hybridization chain reaction (HCR) technology, which results in amplification of immunofluorescence\n",
      "signals by up to two to three orders of magnitude with low background. isHCR's highly modular and\n",
      "easily adaptable design enables the technique to be applied broadly, and we further optimized its\n",
      "use in multiplexed imaging and in state-of-the-art tissue expansion and clearing techniques.\n",
      "['hybridization chain reaction',\n",
      " 'immunofluorescence',\n",
      " 'multiplexed imaging',\n",
      " 'tissue expansion and clearing']\n",
      "Elapsed time: 7.26 seconds\n",
      "-------------------\n",
      "29553578\n",
      "Identification of spatial expression trends in single-cell gene expression data.\n",
      "As methods for measuring spatial gene expression at single-cell resolution become available, there\n",
      "is a need for computational analysis strategies. We present trendsceek, a method based on marked\n",
      "point processes that identifies genes with statistically significant spatial expression trends.\n",
      "trendsceek finds these genes in spatial transcriptomic and sequential fluorescence in situ\n",
      "hybridization data, and also reveals significant gene expression gradients and hot spots in low-\n",
      "dimensional projections of dissociated single-cell RNA-seq data.\n",
      "['fluorescence in situ hybridization',\n",
      " 'spatial transcriptomic data',\n",
      " 'single-cell RNA-seq data']\n",
      "Elapsed time: 7.41 seconds\n",
      "-------------------\n",
      "29553579\n",
      "SpatialDE: identification of spatially variable genes.\n",
      "Technological advances have made it possible to measure spatially resolved gene expression at high\n",
      "throughput. However, methods to analyze these data are not established. Here we describe SpatialDE,\n",
      "a statistical test to identify genes with spatial patterns of expression variation from multiplexed\n",
      "imaging or spatial RNA-sequencing data. SpatialDE also implements 'automatic expression histology',\n",
      "a spatial gene-clustering approach that enables expression-based tissue histology.\n",
      "['SpatialDE', 'multiplexed imaging', 'spatial RNA-sequencing']\n",
      "Elapsed time: 7.18 seconds\n",
      "-------------------\n",
      "29578535\n",
      "NAMD goes quantum: an integrative suite for hybrid simulations.\n",
      "Hybrid methods that combine quantum mechanics (QM) and molecular mechanics (MM) can be applied to\n",
      "studies of reaction mechanisms in locations ranging from active sites of small enzymes to multiple\n",
      "sites in large bioenergetic complexes. By combining the widely used molecular dynamics and\n",
      "visualization programs NAMD and VMD with the quantum chemistry packages ORCA and MOPAC, we created\n",
      "an integrated, comprehensive, customizable, and easy-to-use suite\n",
      "(http://www.ks.uiuc.edu/Research/qmmm). Through the QwikMD interface, setup, execution,\n",
      "visualization, and analysis are streamlined for all levels of expertise.\n",
      "['NAMD', 'VMD', 'ORCA', 'MOPAC', 'QwikMD']\n",
      "Elapsed time: 8.48 seconds\n",
      "-------------------\n",
      "29578537\n",
      "Photoactivatable drugs for nicotinic optopharmacology.\n",
      "Photoactivatable pharmacological agents have revolutionized neuroscience, but the palette of\n",
      "available compounds is limited. We describe a general method for caging tertiary amines by using a\n",
      "stable quaternary ammonium linkage that elicits a red shift in the activation wavelength. We\n",
      "prepared a photoactivatable nicotine (PA-Nic), uncageable via one- or two-photon excitation, that is\n",
      "useful to study nicotinic acetylcholine receptors (nAChRs) in different experimental preparations\n",
      "and spatiotemporal scales.\n",
      "['one-photon excitation', 'two-photon excitation']\n",
      "Elapsed time: 7.42 seconds\n",
      "-------------------\n",
      "29608556\n",
      "Real-time fluorescence and deformability cytometry.\n",
      "The throughput of cell mechanical characterization has recently approached that of conventional flow\n",
      "cytometers. However, this very sensitive, label-free approach still lacks the specificity of\n",
      "molecular markers. Here we developed an approach that combines real-time 1D-imaging fluorescence and\n",
      "deformability cytometry in one instrument (RT-FDC), thus opening many new research avenues. We\n",
      "demonstrated its utility by using subcellular fluorescence localization to identify mitotic cells\n",
      "and test for mechanical changes in those cells in an RNA interference screen.\n",
      "['1D-imaging fluorescence', 'deformability cytometry']\n",
      "Elapsed time: 6.88 seconds\n",
      "-------------------\n",
      "29630062\n",
      "Real-time 3D single-molecule localization using experimental point spread functions.\n",
      "We present a real-time fitter for 3D single-molecule localization microscopy using experimental\n",
      "point spread functions (PSFs) that achieves minimal uncertainty in 3D on any microscope and is\n",
      "compatible with any PSF engineering approach. We used this method to image cellular structures and\n",
      "attained unprecedented image quality for astigmatic PSFs. The fitter compensates for most optical\n",
      "aberrations and makes accurate 3D super-resolution microscopy broadly accessible, even on standard\n",
      "microscopes without dedicated 3D optics.\n",
      "['real-time fitter',\n",
      " '3D single-molecule localization microscopy',\n",
      " 'experimental point spread functions (PSFs)',\n",
      " 'astigmatic PSFs',\n",
      " 'super-resolution microscopy']\n",
      "Elapsed time: 8.41 seconds\n",
      "-------------------\n",
      "29638227\n",
      "A toolbox of immunoprecipitation-grade monoclonal antibodies to human transcription factors.\n",
      "A key component of efforts to address the reproducibility crisis in biomedical research is the\n",
      "development of rigorously validated and renewable protein-affinity reagents. As part of the US\n",
      "National Institutes of Health (NIH) Protein Capture Reagents Program (PCRP), we have generated a\n",
      "collection of 1,406 highly validated immunoprecipitation- and/or immunoblotting-grade mouse\n",
      "monoclonal antibodies (mAbs) to 737 human transcription factors, using an integrated production and\n",
      "validation pipeline. We used HuProt human protein microarrays as a primary validation tool to\n",
      "identify mAbs with high specificity for their cognate targets. We further validated PCRP mAbs by\n",
      "means of multiple experimental applications, including immunoprecipitation, immunoblotting,\n",
      "chromatin immunoprecipitation followed by sequencing (ChIP-seq), and immunohistochemistry. We also\n",
      "conducted a meta-analysis that identified critical variables that contribute to the generation of\n",
      "high-quality mAbs. All validation data, protocols, and links to PCRP mAb suppliers are available at\n",
      "http://proteincapture.org.\n",
      "['immunoprecipitation',\n",
      " 'immunoblotting',\n",
      " 'chromatin immunoprecipitation followed by sequencing (ChIP-seq)',\n",
      " 'immunohistochemistry']\n",
      "Elapsed time: 12.33 seconds\n",
      "-------------------\n",
      "29713082\n",
      "Self-interference 3D super-resolution microscopy for deep tissue investigations.\n",
      "Fluorescence localization microscopy has achieved near-molecular resolution capable of revealing\n",
      "ultra-structures, with a broad range of applications, especially in cellular biology. However, it\n",
      "remains challenging to attain such resolution in three dimensions and inside biological tissues\n",
      "beyond the first cell layer. Here we introduce SELFI, a framework for 3D single-molecule\n",
      "localization within multicellular specimens and tissues. The approach relies on self-interference\n",
      "generated within the microscope's point spread function (PSF) to simultaneously encode equiphase and\n",
      "intensity fluorescence signals, which together provide the 3D position of an emitter. We combined\n",
      "SELFI with conventional localization microscopy to visualize F-actin 3D filament networks and reveal\n",
      "the spatial distribution of the transcription factor OCT4 in human induced pluripotent stem cells at\n",
      "depths up to 50 µm inside uncleared tissue spheroids. SELFI paves the way to nanoscale\n",
      "investigations of native cellular processes in intact tissues.\n",
      "['single-molecule localization microscopy',\n",
      " 'self-interference microscopy',\n",
      " 'fluorescence localization microscopy',\n",
      " '3D super-resolution microscopy']\n",
      "Elapsed time: 11.71 seconds\n",
      "-------------------\n",
      "29735996\n",
      "C-BERST: defining subnuclear proteomic landscapes at genomic elements with dCas9-APEX2.\n",
      "Mapping proteomic composition at distinct genomic loci in living cells has been a long-standing\n",
      "challenge. Here we report that dCas9-APEX2 biotinylation at genomic elements by restricted spatial\n",
      "tagging (C-BERST) allows the rapid, unbiased mapping of proteomes near defined genomic loci, as\n",
      "demonstrated for telomeres and centromeres. C-BERST enables the high-throughput identification of\n",
      "proteins associated with specific sequences, thereby facilitating annotation of these factors and\n",
      "their roles.\n",
      "['proteomic composition',\n",
      " 'restricted spatial tagging',\n",
      " 'rapid unbiased mapping',\n",
      " 'high-throughput identification']\n",
      "Elapsed time: 7.84 seconds\n",
      "-------------------\n",
      "29735999\n",
      "Single-shot super-resolution total internal reflection fluorescence microscopy.\n",
      "We combined instant structured illumination microscopy (iSIM) with total internal reflection\n",
      "fluorescence microscopy (TIRFM) in an approach referred to as instant TIRF-SIM, thereby improving\n",
      "the lateral spatial resolution of TIRFM to 115 ± 13 nm without compromising speed, and enabling\n",
      "imaging frame rates up to 100 Hz over hundreds of time points. We applied instant TIRF-SIM to\n",
      "multiple live samples and achieved rapid, high-contrast super-resolution imaging close to the\n",
      "coverslip surface.\n",
      "['iSIM', 'TIRFM', 'instant TIRF-SIM', 'super-resolution microscopy']\n",
      "Elapsed time: 10.30 seconds\n",
      "-------------------\n",
      "29736000\n",
      "High-speed volumetric imaging of neuronal activity in freely moving rodents.\n",
      "Thus far, optical recording of neuronal activity in freely behaving animals has been limited to a\n",
      "thin axial range. We present a head-mounted miniaturized light-field microscope (MiniLFM) capable of\n",
      "capturing neuronal network activity within a volume of 700 × 600 × 360 µm<sup>3</sup> at 16 Hz in\n",
      "the hippocampus of freely moving mice. We demonstrate that neurons separated by as little as ~15 µm\n",
      "and at depths up to 360 µm can be discriminated.\n",
      "['MiniLFM', 'light-field microscope', 'optical recording']\n",
      "Elapsed time: 7.58 seconds\n",
      "-------------------\n",
      "29786091\n",
      "Automated, parallel mass spectrometry imaging and structural identification of lipids.\n",
      "We report a method that enables automated data-dependent acquisition of lipid tandem mass\n",
      "spectrometry data in parallel with a high-resolution mass spectrometry imaging experiment. The\n",
      "method does not increase the total image acquisition time and is combined with automatic structural\n",
      "assignments. This lipidome-per-pixel approach automatically identified and validated 104 unique\n",
      "molecular lipids and their spatial locations from rat cerebellar tissue.\n",
      "['mass spectrometry imaging',\n",
      " 'data-dependent acquisition',\n",
      " 'high-resolution mass spectrometry',\n",
      " 'lipidome-per-pixel approach']\n",
      "Elapsed time: 7.44 seconds\n",
      "-------------------\n",
      "29786093\n",
      "Author Correction: High-speed volumetric imaging of neuronal activity in freely moving rodents.\n",
      "In the version of this Brief Communication originally published online, ref. 21 included details for\n",
      "a conference paper (Pegard, N. C. et al. Paper presented at Novel Techniques in Microscopy: Optics\n",
      "in the Life Sciences, Vancouver, BC, Canada, 12-15 April 2015). The correct reference is the\n",
      "following: Pégard, N. C. et al. Optica 3, 517-524 (2016). This error has been corrected in the\n",
      "print, HTML and PDF versions of the paper.\n",
      "['Optica', 'High-speed volumetric imaging', 'Microscopy']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "29808018\n",
      "Nanoparticles for super-resolution microscopy and single-molecule tracking.\n",
      "We review the use of luminescent nanoparticles in super-resolution imaging and single-molecule\n",
      "tracking, and showcase novel approaches to super-resolution imaging that leverage the brightness,\n",
      "stability, and unique optical-switching properties of these nanoparticles. We also discuss the\n",
      "challenges associated with their use in biological systems, including intracellular delivery and\n",
      "molecular targeting. In doing so, we hope to provide practical guidance for biologists and continue\n",
      "to bridge the fields of super-resolution imaging and nanoparticle engineering to support their\n",
      "mutual advancement.\n",
      "['super-resolution microscopy', 'single-molecule tracking']\n",
      "Elapsed time: 7.23 seconds\n",
      "-------------------\n",
      "29915189\n",
      "A comparison of methods to assess cell mechanical properties.\n",
      "The mechanical properties of cells influence their cellular and subcellular functions, including\n",
      "cell adhesion, migration, polarization, and differentiation, as well as organelle organization and\n",
      "trafficking inside the cytoplasm. Yet reported values of cell stiffness and viscosity vary\n",
      "substantially, which suggests differences in how the results of different methods are obtained or\n",
      "analyzed by different groups. To address this issue and illustrate the complementarity of certain\n",
      "approaches, here we present, analyze, and critically compare measurements obtained by means of some\n",
      "of the most widely used methods for cell mechanics: atomic force microscopy, magnetic twisting\n",
      "cytometry, particle-tracking microrheology, parallel-plate rheometry, cell monolayer rheology, and\n",
      "optical stretching. These measurements highlight how elastic and viscous moduli of MCF-7 breast\n",
      "cancer cells can vary 1,000-fold and 100-fold, respectively. We discuss the sources of these\n",
      "variations, including the level of applied mechanical stress, the rate of deformation, the geometry\n",
      "of the probe, the location probed in the cell, and the extracellular microenvironment.\n",
      "['atomic force microscopy',\n",
      " 'magnetic twisting cytometry',\n",
      " 'particle-tracking microrheology',\n",
      " 'parallel-plate rheometry',\n",
      " 'cell monolayer rheology',\n",
      " 'optical stretching']\n",
      "Elapsed time: 11.17 seconds\n",
      "-------------------\n",
      "29941872\n",
      "A reassessment of DNA-immunoprecipitation-based genomic profiling.\n",
      "DNA immunoprecipitation followed by sequencing (DIP-seq) is a common enrichment method for profiling\n",
      "DNA modifications in mammalian genomes. However, the results of independent DIP-seq studies often\n",
      "show considerable variation between profiles of the same genome and between profiles obtained by\n",
      "alternative methods. Here we show that these differences are primarily due to the intrinsic affinity\n",
      "of IgG for short unmodified DNA repeats. This pervasive experimental error accounts for 50-99% of\n",
      "regions identified as 'enriched' for DNA modifications in DIP-seq data. Correction of this error\n",
      "profoundly altered DNA-modification profiles for numerous cell types, including mouse embryonic stem\n",
      "cells, and subsequently revealed novel associations among DNA modifications, chromatin modifications\n",
      "and biological processes. We conclude that both matched input and IgG controls are essential in\n",
      "order for the results of DIP-based assays to be interpreted correctly, and that complementary, non-\n",
      "antibody-based techniques should be used to validate DIP-based findings to avoid further\n",
      "misinterpretation of genome-wide profiling data.\n",
      "['DIP-seq', 'IgG controls', 'complementary, non-antibody-based techniques']\n",
      "Elapsed time: 10.84 seconds\n",
      "-------------------\n",
      "29967493\n",
      "Efficient and precise editing of endogenous transcripts with SNAP-tagged ADARs.\n",
      "Molecular tools that target RNA at specific sites allow recoding of RNA information and processing.\n",
      "SNAP-tagged deaminases guided by a chemically stabilized guide RNA can edit targeted adenosine to\n",
      "inosine in several endogenous transcripts simultaneously, with high efficiency (up to 90%), high\n",
      "potency, sufficient editing duration, and high precision. We used adenosine deaminases acting on RNA\n",
      "(ADARs) fused to SNAP-tag for the efficient and concurrent editing of two disease-relevant signaling\n",
      "transcripts, KRAS and STAT1. We also demonstrate improved performance compared with that of the\n",
      "recently described Cas13b-ADAR.\n",
      "{'imagingTechniques': ['SNAP-tagged deaminases',\n",
      "                       'chemically stabilized guide RNA',\n",
      "                       'Cas13b-ADAR']}\n",
      "Elapsed time: 8.97 seconds\n",
      "-------------------\n",
      "29967495\n",
      "Chemogenetic control of gene expression and cell signaling with antiviral drugs.\n",
      "We developed a method in which the NS3 cis-protease from hepatitis C virus can be used as a ligand-\n",
      "inducible connection to control the function and localization of engineered proteins in mammalian\n",
      "cells. To demonstrate the versatility of this approach, we designed drug-sensitive transcription\n",
      "factors and transmembrane signaling proteins, the activities of which can be tightly and reversibly\n",
      "controlled through the use of clinically tested antiviral protease inhibitors.\n",
      "['ligand-inducible connection',\n",
      " 'drug-sensitive transcription factors',\n",
      " 'transmembrane signaling proteins']\n",
      "Elapsed time: 7.44 seconds\n",
      "-------------------\n",
      "29967496\n",
      "StaPLs: versatile genetically encoded modules for engineering drug-inducible proteins.\n",
      "Robust approaches for chemogenetic control of protein function would have many biological\n",
      "applications. We developed stabilizable polypeptide linkages (StaPLs) based on hepatitis C virus\n",
      "protease. StaPLs undergo autoproteolysis to cleave proteins by default, whereas protease inhibitors\n",
      "prevent cleavage and preserve protein function. We created StaPLs responsive to different clinically\n",
      "approved drugs to bidirectionally control transcription with zinc-finger-based effectors, and used\n",
      "StaPLs to create single-chain, drug-stabilizable variants of CRISPR-Cas9 and caspase-9.\n",
      "['StaPLs',\n",
      " 'autoproteolysis',\n",
      " 'protease inhibitors',\n",
      " 'zinc-finger-based effectors',\n",
      " 'CRISPR-Cas9',\n",
      " 'caspase-9']\n",
      "Elapsed time: 9.83 seconds\n",
      "-------------------\n",
      "29988094\n",
      "Genome-wide SWAp-Tag yeast libraries for proteome exploration.\n",
      "Yeast libraries revolutionized the systematic study of cell biology. To extensively increase the\n",
      "number of such libraries, we used our previously devised SWAp-Tag (SWAT) approach to construct a\n",
      "genome-wide library of ~5,500 strains carrying the SWAT NOP1promoter-GFP module at the N terminus of\n",
      "proteins. In addition, we created six diverse libraries that restored the native regulation, created\n",
      "an overexpression library with a Cherry tag, or enabled protein complementation assays from two\n",
      "fragments of an enzyme or fluorophore. We developed methods utilizing these SWAT collections to\n",
      "systematically characterize the yeast proteome for protein abundance, localization, topology, and\n",
      "interactions.\n",
      "['SWAT',\n",
      " 'NOP1promoter-GFP module',\n",
      " 'Cherry tag',\n",
      " 'protein complementation assays']\n",
      "Elapsed time: 8.81 seconds\n",
      "-------------------\n",
      "29988095\n",
      "Fast reversibly photoswitching red fluorescent proteins for live-cell RESOLFT nanoscopy.\n",
      "Reversibly photoswitchable fluorescent proteins (rsFPs) are gaining popularity as tags for optical\n",
      "nanoscopy because they make it possible to image with lower light doses. However, green rsFPs need\n",
      "violet-blue light for photoswitching, which is potentially phototoxic and highly scattering. We\n",
      "developed new rsFPs based on FusionRed that are reversibly photoswitchable with green-orange light.\n",
      "The rsFusionReds are bright and exhibit rapid photoswitching, thereby enabling nanoscale imaging of\n",
      "living cells.\n",
      "['RESOLFT nanoscopy',\n",
      " 'reversibly photoswitchable fluorescent proteins',\n",
      " 'optical nanoscopy']\n",
      "Elapsed time: 8.75 seconds\n",
      "-------------------\n",
      "30013046\n",
      "High-precision automated reconstruction of neurons with flood-filling networks.\n",
      "Reconstruction of neural circuits from volume electron microscopy data requires the tracing of cells\n",
      "in their entirety, including all their neurites. Automated approaches have been developed for\n",
      "tracing, but their error rates are too high to generate reliable circuit diagrams without extensive\n",
      "human proofreading. We present flood-filling networks, a method for automated segmentation that,\n",
      "similar to most previous efforts, uses convolutional neural networks, but contains in addition a\n",
      "recurrent pathway that allows the iterative optimization and extension of individual neuronal\n",
      "processes. We used flood-filling networks to trace neurons in a dataset obtained by serial block-\n",
      "face electron microscopy of a zebra finch brain. Using our method, we achieved a mean error-free\n",
      "neurite path length of 1.1 mm, and we observed only four mergers in a test set with a path length of\n",
      "97 mm. The performance of flood-filling networks was an order of magnitude better than that of\n",
      "previous approaches applied to this dataset, although with substantially increased computational\n",
      "costs.\n",
      "['convolutional neural networks',\n",
      " 'serial block-face electron microscopy',\n",
      " 'flood-filling networks']\n",
      "Elapsed time: 10.54 seconds\n",
      "-------------------\n",
      "30013047\n",
      "Active PSF shaping and adaptive optics enable volumetric localization microscopy through brain\n",
      "sections.\n",
      "Application of single-molecule switching nanoscopy (SMSN) beyond the coverslip surface poses\n",
      "substantial challenges due to sample-induced aberrations that distort and blur single-molecule\n",
      "emission patterns. We combined active shaping of point spread functions and efficient adaptive\n",
      "optics to enable robust 3D-SMSN imaging within tissues. This development allowed us to image through\n",
      "30-μm-thick brain sections to visualize and reconstruct the morphology and the nanoscale details of\n",
      "amyloid-β filaments in a mouse model of Alzheimer's disease.\n",
      "['active PSF shaping',\n",
      " 'adaptive optics',\n",
      " 'single-molecule switching nanoscopy (SMSN)']\n",
      "Elapsed time: 9.16 seconds\n",
      "-------------------\n",
      "30046099\n",
      "Induction of myelinating oligodendrocytes in human cortical spheroids.\n",
      "Cerebral organoids provide an accessible system for investigations of cellular composition,\n",
      "interactions, and organization but have lacked oligodendrocytes, the myelinating glia of the central\n",
      "nervous system. Here we reproducibly generated oligodendrocytes and myelin in 'oligocortical\n",
      "spheroids' derived from human pluripotent stem cells. Molecular features consistent with those of\n",
      "maturing oligodendrocytes and early myelin appeared by week 20 in culture, with further maturation\n",
      "and myelin compaction evident by week 30. Promyelinating drugs enhanced the rate and extent of\n",
      "oligodendrocyte generation and myelination, and spheroids generated from human subjects with a\n",
      "genetic myelin disorder recapitulated human disease phenotypes. Oligocortical spheroids provide a\n",
      "versatile platform for studies of myelination of the developing central nervous system and offer new\n",
      "opportunities for disease modeling and therapeutic development.\n",
      "['imaging', 'molecular features', 'myelin compaction']\n",
      "Elapsed time: 10.31 seconds\n",
      "-------------------\n",
      "30061677\n",
      "Artifact-free high-density localization microscopy analysis.\n",
      "High-density analysis methods for localization microscopy increase acquisition speed but produce\n",
      "artifacts. We demonstrate that these artifacts can be eliminated by the combination of Haar wavelet\n",
      "kernel (HAWK) analysis with standard single-frame fitting. We tested the performance of this method\n",
      "on synthetic, fixed-cell, and live-cell data, and found that HAWK preprocessing yielded\n",
      "reconstructions that reflected the structure of the sample, thus enabling high-speed, artifact-free\n",
      "super-resolution imaging of live cells.\n",
      "['HAWK analysis', 'single-frame fitting', 'super-resolution imaging']\n",
      "Elapsed time: 7.17 seconds\n",
      "-------------------\n",
      "30065368\n",
      "Quanti.us: a tool for rapid, flexible, crowd-based annotation of images.\n",
      "We describe Quanti.us , a crowd-based image-annotation platform that provides an accurate\n",
      "alternative to computational algorithms for difficult image-analysis problems. We used Quanti.us for\n",
      "a variety of medium-throughput image-analysis tasks and achieved 10-50× savings in analysis time\n",
      "compared with that required for the same task by a single expert annotator. We show equivalent deep\n",
      "learning performance for Quanti.us-derived and expert-derived annotations, which should allow\n",
      "scalable integration with tailored machine learning algorithms.\n",
      "['image-analysis', 'computational algorithms', 'crowd-based annotation']\n",
      "Elapsed time: 7.25 seconds\n",
      "-------------------\n",
      "30082898\n",
      "Single-DNA electron spin resonance spectroscopy in aqueous solutions.\n",
      "Magnetic resonance spectroscopy of single biomolecules under near-physiological conditions could\n",
      "substantially advance understanding of their biological function, but this approach remains very\n",
      "challenging. Here we used nitrogen-vacancy centers in diamonds to detect electron spin resonance\n",
      "spectra of individual, tethered DNA duplexes labeled with a nitroxide spin label in aqueous buffer\n",
      "solutions at ambient temperatures. This work paves the way for magnetic resonance studies on single\n",
      "biomolecules and their intermolecular interactions in native-like environments.\n",
      "['electron spin resonance spectroscopy',\n",
      " 'magnetic resonance spectroscopy',\n",
      " 'single-molecule']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "30108338\n",
      "Publisher Correction: Single-DNA electron spin resonance spectroscopy in aqueous solutions.\n",
      "In the version of this paper originally published online, the ORCID ID for Peter Z. Qin was\n",
      "incorrectly assigned to Zhuoyang Qin. In addition, the ORCID for Fazhan Shi was omitted. These\n",
      "errors have been corrected in the print, PDF, and HTML versions of the paper.\n",
      "['electron spin resonance spectroscopy', 'none', 'none']\n",
      "Elapsed time: 6.30 seconds\n",
      "-------------------\n",
      "30127504\n",
      "Modified aptamers enable quantitative sub-10-nm cellular DNA-PAINT imaging.\n",
      "Although current implementations of super-resolution microscopy are technically approaching true\n",
      "molecular-scale resolution, this has not translated to imaging of biological specimens, because of\n",
      "the large size of conventional affinity reagents. Here we introduce slow off-rate modified aptamers\n",
      "(SOMAmers) as small and specific labeling reagents for use with DNA points accumulation in nanoscale\n",
      "topography (DNA-PAINT). To demonstrate the achievable resolution, specificity, and multiplexing\n",
      "capability of SOMAmers, we labeled and imaged both transmembrane and intracellular targets in fixed\n",
      "and live cells.\n",
      "['DNA-PAINT', 'super-resolution microscopy', 'aptamers']\n",
      "Elapsed time: 8.06 seconds\n",
      "-------------------\n",
      "30127506\n",
      "Reduced MEK inhibition preserves genomic stability in naive human embryonic stem cells.\n",
      "Human embryonic stem cells (hESCs) can be captured in a primed state in which they resemble the\n",
      "postimplantation epiblast, or in a naive state where they resemble the preimplantation epiblast.\n",
      "Naive-cell-specific culture conditions allow the study of preimplantation development ex vivo but\n",
      "reportedly lead to chromosomal abnormalities, which compromises their utility in research and\n",
      "potential therapeutic applications. Although MEK inhibition is essential for the naive state, here\n",
      "we show that reduced MEK inhibition facilitated the establishment and maintenance of naive hESCs\n",
      "that retained naive-cell-specific features, including global DNA hypomethylation, HERVK expression,\n",
      "and two active X chromosomes. We further show that hESCs cultured under these modified conditions\n",
      "proliferated more rapidly; accrued fewer chromosomal abnormalities; and displayed changes in the\n",
      "phosphorylation levels of MAPK components, regulators of DNA damage/repair, and cell cycle. We thus\n",
      "provide a simple modification to current methods that can enable robust growth and reduced genomic\n",
      "instability in naive hESCs.\n",
      "['imaging', 'phosphorylation', 'DNA damage/repair']\n",
      "Elapsed time: 11.27 seconds\n",
      "-------------------\n",
      "30135560\n",
      "Author Correction: Genetically engineered cerebral organoids model brain tumor formation.\n",
      "In the originally published paper, the \"before\" image for the afatinib condition in Fig. 6c was\n",
      "incorrect. Instead of an image displaying a GBM-3 neoplastic organoid before afatinib treatment,\n",
      "this panel showed an image from the GBM-2 control (DMSO) group before treatment. This error has now\n",
      "been corrected in the HTML and PDF versions of the article; the \"before, afatinib\" panel in Fig. 6c\n",
      "now shows a representative image from the indicated experiment. The color of all error bars in Fig.\n",
      "6 has also been changed to black, for consistency. All statistical analysis and all conclusions\n",
      "presented in the article are unaffected by this error. Nevertheless, we apologize for the mistake.\n",
      "['before', 'afatinib', 'HTML', 'PDF']\n",
      "Elapsed time: 8.43 seconds\n",
      "-------------------\n",
      "30150754\n",
      "Trac-looping measures genome structure and chromatin accessibility.\n",
      "Long-range chromatin interactions play critical roles in genome organization and regulation of\n",
      "transcription. We now report transposase-mediated analysis of chromatin looping (Trac-looping) for\n",
      "simultaneous detection of multiscale genome-wide chromatin interactions among regulatory elements\n",
      "and chromatin accessibility. With this technique, a bivalent oligonucleotide linker is inserted\n",
      "between two interacting regions such that the chromatin interactions are captured without prior\n",
      "chromatin fragmentation and proximity-based ligation. Application of Trac-looping to human\n",
      "CD4<sup>+</sup> T cells revealed substantial reorganization of enhancer-promoter interactions\n",
      "associated with changes in gene expression after T cell receptor stimulation.\n",
      "['Trac-looping', 'transposase-mediated analysis of chromatin looping']\n",
      "Elapsed time: 8.35 seconds\n",
      "-------------------\n",
      "30171236\n",
      "CDeep3M-Plug-and-Play cloud-based deep learning for image segmentation.\n",
      "As biomedical imaging datasets expand, deep neural networks are considered vital for image\n",
      "processing, yet community access is still limited by setting up complex computational environments\n",
      "and availability of high-performance computing resources. We address these bottlenecks with CDeep3M,\n",
      "a ready-to-use image segmentation solution employing a cloud-based deep convolutional neural\n",
      "network. We benchmark CDeep3M on large and complex two-dimensional and three-dimensional imaging\n",
      "datasets from light, X-ray, and electron microscopy.\n",
      "['light microscopy', 'X-ray microscopy', 'electron microscopy']\n",
      "Elapsed time: 7.10 seconds\n",
      "-------------------\n",
      "30171252\n",
      "Precision and accuracy of single-molecule FRET measurements-a multi-laboratory benchmark study.\n",
      "Single-molecule Förster resonance energy transfer (smFRET) is increasingly being used to determine\n",
      "distances, structures, and dynamics of biomolecules in vitro and in vivo. However, generalized\n",
      "protocols and FRET standards to ensure the reproducibility and accuracy of measurements of FRET\n",
      "efficiencies are currently lacking. Here we report the results of a comparative blind study in which\n",
      "20 labs determined the FRET efficiencies (E) of several dye-labeled DNA duplexes. Using a unified,\n",
      "straightforward method, we obtained FRET efficiencies with s.d. between ±0.02 and ±0.05. We suggest\n",
      "experimental and computational procedures for converting FRET efficiencies into accurate distances,\n",
      "and discuss potential uncertainties in the experiment and the modeling. Our quantitative assessment\n",
      "of the reproducibility of intensity-based smFRET measurements and a unified correction procedure\n",
      "represents an important step toward the validation of distance networks, with the ultimate aim of\n",
      "achieving reliable structural models of biomolecular systems by smFRET-based hybrid methods.\n",
      "['smFRET', 'single-molecule Förster resonance energy transfer']\n",
      "Elapsed time: 10.91 seconds\n",
      "-------------------\n",
      "30202058\n",
      "COMRADES determines in vivo RNA structures and interactions.\n",
      "The structural flexibility of RNA underlies fundamental biological processes, but there are no\n",
      "methods for exploring the multiple conformations adopted by RNAs in vivo. We developed cross-linking\n",
      "of matched RNAs and deep sequencing (COMRADES) for in-depth RNA conformation capture, and a pipeline\n",
      "for the retrieval of RNA structural ensembles. Using COMRADES, we determined the architecture of the\n",
      "Zika virus RNA genome inside cells, and identified multiple site-specific interactions with human\n",
      "noncoding RNAs.\n",
      "['cross-linking of matched RNAs',\n",
      " 'deep sequencing',\n",
      " 'in-depth RNA conformation capture',\n",
      " 'site-specific interactions']\n",
      "Elapsed time: 7.56 seconds\n",
      "-------------------\n",
      "30202059\n",
      "Three-photon imaging of mouse brain structure and function through the intact skull.\n",
      "Optical imaging through the intact mouse skull is challenging because of skull-induced aberrations\n",
      "and scattering. We found that three-photon excitation provided improved optical sectioning compared\n",
      "with that obtained with two-photon excitation, even when we used the same excitation wavelength and\n",
      "imaging system. Here we demonstrate three-photon imaging of vasculature through the adult mouse\n",
      "skull at >500-μm depth, as well as GCaMP6s calcium imaging over weeks in cortical layers 2/3 and 4\n",
      "in awake mice, with 8.5 frames per second and a field of view spanning hundreds of micrometers.\n",
      "['three-photon imaging', 'optical imaging', 'calcium imaging', 'GCaMP6s']\n",
      "Elapsed time: 8.63 seconds\n",
      "-------------------\n",
      "30224671\n",
      "Template-free 2D particle fusion in localization microscopy.\n",
      "Methods that fuse multiple localization microscopy images of a single structure can improve signal-\n",
      "to-noise ratio and resolution, but they generally suffer from template bias or sensitivity to\n",
      "registration errors. We present a template-free particle-fusion approach based on an all-to-all\n",
      "registration that provides robustness against individual misregistrations and underlabeling. We\n",
      "achieved 3.3-nm Fourier ring correlation (FRC) image resolution by fusing 383 DNA origami\n",
      "nanostructures with 80% labeling density, and 5.0-nm resolution for structures with 30% labeling\n",
      "density.\n",
      "['all-to-all registration',\n",
      " 'Fourier ring correlation (FRC)',\n",
      " 'localization microscopy',\n",
      " 'particle fusion']\n",
      "Elapsed time: 8.54 seconds\n",
      "-------------------\n",
      "30224672\n",
      "Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy.\n",
      "Understanding cells as integrated systems is central to modern biology. Although fluorescence\n",
      "microscopy can resolve subcellular structure in living cells, it is expensive, is slow, and can\n",
      "damage cells. We present a label-free method for predicting three-dimensional fluorescence directly\n",
      "from transmitted-light images and demonstrate that it can be used to generate multi-structure,\n",
      "integrated images. The method can also predict immunofluorescence (IF) from electron micrograph (EM)\n",
      "inputs, extending the potential applications.\n",
      "['transmitted-light microscopy',\n",
      " 'fluorescence microscopy',\n",
      " 'immunofluorescence (IF)',\n",
      " 'electron micrograph (EM)']\n",
      "Elapsed time: 7.74 seconds\n",
      "-------------------\n",
      "30250056\n",
      "Reducing effects of particle adsorption to the air-water interface in cryo-EM.\n",
      "Most protein particles prepared in vitreous ice for single-particle cryo-electron microscopy (cryo-\n",
      "EM) are adsorbed to air-water or substrate-water interfaces, which can cause the particles to adopt\n",
      "preferred orientations. By using a rapid plunge-freezing robot and nanowire grids, we were able to\n",
      "reduce some of the deleterious effects of the air-water interface by decreasing the dwell time of\n",
      "particles in thin liquid films. We demonstrated this by using single-particle cryo-EM and cryo-\n",
      "electron tomography (cryo-ET) to examine hemagglutinin, insulin receptor complex, and apoferritin.\n",
      "['cryo-EM', 'cryo-ET']\n",
      "Elapsed time: 7.93 seconds\n",
      "-------------------\n",
      "30275574\n",
      "Multicolor single-particle reconstruction of protein complexes.\n",
      "Single-particle reconstruction (SPR) from electron microscopy (EM) images is widely used in\n",
      "structural biology, but it lacks direct information on protein identity. To address this limitation,\n",
      "we developed a computational and analytical framework that reconstructs and coaligns multiple\n",
      "proteins from 2D super-resolution fluorescence images. To demonstrate our method, we generated\n",
      "multicolor 3D reconstructions of several proteins within the human centriole, which revealed their\n",
      "relative locations, dimensions and orientations.\n",
      "['electron microscopy (EM)',\n",
      " 'single-particle reconstruction (SPR)',\n",
      " '2D super-resolution fluorescence images',\n",
      " 'multicolor 3D reconstructions']\n",
      "Elapsed time: 7.99 seconds\n",
      "-------------------\n",
      "30275587\n",
      "All-optical synaptic electrophysiology probes mechanism of ketamine-induced disinhibition.\n",
      "Optical assays of synaptic strength could facilitate studies of neuronal transmission and its\n",
      "dysregulation in disease. Here we introduce a genetic toolbox for all-optical interrogation of\n",
      "synaptic electrophysiology (synOptopatch) via mutually exclusive expression of a channelrhodopsin\n",
      "actuator and an archaerhodopsin-derived voltage indicator. Optically induced activity in the\n",
      "channelrhodopsin-expressing neurons generated excitatory and inhibitory postsynaptic potentials that\n",
      "we optically resolved in reporter-expressing neurons. We further developed a yellow spine-targeted\n",
      "Ca<sup>2+</sup> indicator to localize optogenetically triggered synaptic inputs. We demonstrated\n",
      "synOptopatch recordings in cultured rodent neurons and in acute rodent brain slice. In synOptopatch\n",
      "measurements of primary rodent cultures, acute ketamine administration suppressed disynaptic\n",
      "inhibitory feedbacks, mimicking the effect of this drug on network function in both rodents and\n",
      "humans. We localized this action of ketamine to excitatory synapses onto interneurons. These results\n",
      "establish an in vitro all-optical model of disynaptic disinhibition, a synaptic defect hypothesized\n",
      "in schizophrenia-associated psychosis.\n",
      "['optical assays',\n",
      " 'all-optical interrogation',\n",
      " 'channelrhodopsin actuator',\n",
      " 'archaerhodopsin-derived voltage indicator',\n",
      " 'yellow spine-targeted Ca2+ indicator']\n",
      "Elapsed time: 13.35 seconds\n",
      "-------------------\n",
      "30287931\n",
      "Publisher Correction: Image Data Resource: a bioimage data integration and publication platform.\n",
      "This paper was originally published under standard Nature America Inc. copyright. As of the date of\n",
      "this correction, the Resource is available online as an open-access paper with a CC-BY license. No\n",
      "other part of the paper has been changed.\n",
      "['Image Data Resource',\n",
      " 'CC-BY license',\n",
      " 'bioimage data integration and publication platform']\n",
      "Elapsed time: 5.76 seconds\n",
      "-------------------\n",
      "30349041\n",
      "emClarity: software for high-resolution cryo-electron tomography and subtomogram averaging.\n",
      "Macromolecular complexes are intrinsically flexible and often challenging to purify for structure\n",
      "determination by single-particle cryo-electron microscopy (cryo-EM). Such complexes can be studied\n",
      "by cryo-electron tomography (cryo-ET) combined with subtomogram alignment and classification, which\n",
      "in exceptional cases achieves subnanometer resolution, yielding insight into structure-function\n",
      "relationships. However, it remains challenging to apply this approach to specimens that exhibit\n",
      "conformational or compositional heterogeneity or are present in low abundance. To address this, we\n",
      "developed emClarity ( https://github.com/bHimes/emClarity/wiki ), a GPU-accelerated image-processing\n",
      "package featuring an iterative tomographic tilt-series refinement algorithm that uses subtomograms\n",
      "as fiducial markers and a 3D-sampling-function-compensated, multi-scale principal component analysis\n",
      "classification method. We demonstrate that our approach offers substantial improvement in the\n",
      "resolution of maps and in the separation of different functional states of macromolecular complexes\n",
      "compared with current state-of-the-art software.\n",
      "['cryo-electron microscopy',\n",
      " 'cryo-electron tomography',\n",
      " 'subtomogram alignment and classification',\n",
      " 'iterative tomographic tilt-series refinement algorithm',\n",
      " '3D-sampling-function-compensated, multi-scale principal component analysis '\n",
      " 'classification method']\n",
      "Elapsed time: 12.51 seconds\n",
      "-------------------\n",
      "30377346\n",
      "A fully automatic method yielding initial models from high-resolution cryo-electron microscopy maps.\n",
      "We report a fully automated procedure for the optimization and interpretation of reconstructions\n",
      "from cryo-electron microscopy (cryo-EM) data, available in Phenix as phenix.map_to_model. We applied\n",
      "our approach to 476 datasets with resolution of 4.5 Å or better, including reconstructions of 47\n",
      "ribosomes and 32 other protein-RNA complexes. The median fraction of residues in the deposited\n",
      "structures reproduced automatically was 71% for reconstructions determined at resolutions of 3 Å or\n",
      "better and 47% for those at resolutions worse than 3 Å.\n",
      "['cryo-electron microscopy',\n",
      " 'automatic procedure',\n",
      " 'optimization and interpretation of reconstructions']\n",
      "Elapsed time: 7.98 seconds\n",
      "-------------------\n",
      "30377349\n",
      "Analyzing complex single-molecule emission patterns with deep learning.\n",
      "A fluorescent emitter simultaneously transmits its identity, location, and cellular context through\n",
      "its emission pattern. We developed smNet, a deep neural network for multiplexed single-molecule\n",
      "analysis to retrieve such information with high accuracy. We demonstrate that smNet can extract\n",
      "three-dimensional molecule location, orientation, and wavefront distortion with precision\n",
      "approaching the theoretical limit, and therefore will allow multiplexed measurements through the\n",
      "emission pattern of a single molecule.\n",
      "['deep learning',\n",
      " 'single-molecule emission patterns',\n",
      " 'smNet',\n",
      " 'multiplexesed single-molecule analysis']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "30377360\n",
      "FLIRT: fast local infrared thermogenetics for subcellular control of protein function.\n",
      "FLIRT (fast local infrared thermogenetics) is a microscopy-based technology to locally and\n",
      "reversibly manipulate protein function while simultaneously monitoring the effects in vivo. FLIRT\n",
      "locally inactivates fast-acting temperature-sensitive mutant proteins. We demonstrate that FLIRT can\n",
      "control temperature-sensitive proteins required for cell division, Delta-Notch cell fate signaling,\n",
      "and germline structure in Caenorhabditis elegans with cell-specific and even subcellular precision.\n",
      "['microscopy-based technology', 'local infrared thermogenetics']\n",
      "Elapsed time: 7.04 seconds\n",
      "-------------------\n",
      "30377363\n",
      "Stability, affinity, and chromatic variants of the glutamate sensor iGluSnFR.\n",
      "Single-wavelength fluorescent reporters allow visualization of specific neurotransmitters with high\n",
      "spatial and temporal resolution. We report variants of intensity-based glutamate-sensing fluorescent\n",
      "reporter (iGluSnFR) that are functionally brighter; detect submicromolar to millimolar amounts of\n",
      "glutamate; and have blue, cyan, green, or yellow emission profiles. These variants could be imaged\n",
      "in vivo in cases where original iGluSnFR was too dim, resolved glutamate transients in dendritic\n",
      "spines and axonal boutons, and allowed imaging at kilohertz rates.\n",
      "['single-wavelength fluorescent reporters',\n",
      " 'intensity-based glutamate-sensing fluorescent reporter (iGluSnFR)',\n",
      " 'in vivo imaging',\n",
      " 'high spatial and temporal resolution',\n",
      " 'kilohertz rates']\n",
      "Elapsed time: 9.14 seconds\n",
      "-------------------\n",
      "30377372\n",
      "De novo computational RNA modeling into cryo-EM maps of large ribonucleoprotein complexes.\n",
      "Increasingly, cryo-electron microscopy (cryo-EM) is used to determine the structures of RNA-protein\n",
      "assemblies, but nearly all maps determined with this method have biologically important regions\n",
      "where the local resolution does not permit RNA coordinate tracing. To address these omissions, we\n",
      "present de novo ribonucleoprotein modeling in real space through assembly of fragments together with\n",
      "experimental density in Rosetta (DRRAFTER). We show that DRRAFTER recovers near-native models for a\n",
      "diverse benchmark set of RNA-protein complexes including the spliceosome, mitochondrial ribosome,\n",
      "and CRISPR-Cas9-sgRNA complexes; rigorous blind tests include yeast U1 snRNP and spliceosomal P\n",
      "complex maps. Additionally, to aid in model interpretation, we present a method for reliable in situ\n",
      "estimation of DRRAFTER model accuracy. Finally, we apply DRRAFTER to recently determined maps of\n",
      "telomerase, the HIV-1 reverse transcriptase initiation complex, and the packaged MS2 genome,\n",
      "demonstrating the acceleration of accurate model building in challenging cases.\n",
      "['cryo-electron microscopy', 'Rosetta', 'DRRAFTER']\n",
      "Elapsed time: 11.37 seconds\n",
      "-------------------\n",
      "30377374\n",
      "CRISPR-Sirius: RNA scaffolds for signal amplification in genome imaging.\n",
      "Clustered regularly interspaced short palindromic repeats (CRISPR) guide RNA scaffolds have been\n",
      "adapted to carry multiple binding sites for fluorescent proteins to enhance brightness for live cell\n",
      "imaging of genomic loci. However, many of these modifications result in guide RNA instability and\n",
      "thus produce lower genome-labeling efficiency than anticipated. Here we introduce CRISPR-Sirius,\n",
      "based on octet arrays of aptamers conferring both enhanced guide RNA stability and brightness, and\n",
      "provide initial biological applications of this platform.\n",
      "['CRISPR',\n",
      " 'live cell imaging',\n",
      " 'fluorescent proteins',\n",
      " 'guide RNA instability',\n",
      " 'genome-labeling efficiency']\n",
      "Elapsed time: 7.99 seconds\n",
      "-------------------\n",
      "30377376\n",
      "Species-level functional profiling of metagenomes and metatranscriptomes.\n",
      "Functional profiles of microbial communities are typically generated using comprehensive metagenomic\n",
      "or metatranscriptomic sequence read searches, which are time-consuming, prone to spurious mapping,\n",
      "and often limited to community-level quantification. We developed HUMAnN2, a tiered search strategy\n",
      "that enables fast, accurate, and species-resolved functional profiling of host-associated and\n",
      "environmental communities. HUMAnN2 identifies a community's known species, aligns reads to their\n",
      "pangenomes, performs translated search on unclassified reads, and finally quantifies gene families\n",
      "and pathways. Relative to pure translated search, HUMAnN2 is faster and produces more accurate gene\n",
      "family profiles. We applied HUMAnN2 to study clinal variation in marine metabolism, ecological\n",
      "contribution patterns among human microbiome pathways, variation in species' genomic versus\n",
      "transcriptional contributions, and strain profiling. Further, we introduce 'contributional\n",
      "diversity' to explain patterns of ecological assembly across different microbial community types.\n",
      "['metagenomics',\n",
      " 'metatranscriptomics',\n",
      " 'comprehensive sequence read searches',\n",
      " 'pangenomes',\n",
      " 'translated search',\n",
      " 'gene families',\n",
      " 'pathways']\n",
      "Elapsed time: 11.18 seconds\n",
      "-------------------\n",
      "30377377\n",
      "A light-gated potassium channel for sustained neuronal inhibition.\n",
      "Currently available inhibitory optogenetic tools provide short and transient silencing of neurons,\n",
      "but they cannot provide long-lasting inhibition because of the requirement for high light\n",
      "intensities. Here we present an optimized blue-light-sensitive synthetic potassium channel, BLINK2,\n",
      "which showed good expression in neurons in three species. The channel is activated by illumination\n",
      "with low doses of blue light, and in our experiments it remained active over (tens of) minutes in\n",
      "the dark after the illumination was stopped. This activation caused long periods of inhibition of\n",
      "neuronal firing in ex vivo recordings of mouse neurons and impaired motor neuron response in\n",
      "zebrafish in vivo. As a proof-of-concept application, we demonstrated that in a freely moving rat\n",
      "model of neuropathic pain, the activation of a small number of BLINK2 channels caused a long-lasting\n",
      "(>30 min) reduction in pain sensation.\n",
      "['BLINK2 channels']\n",
      "Elapsed time: 8.87 seconds\n",
      "-------------------\n",
      "30397326\n",
      "Nanobody immunostaining for correlated light and electron microscopy with preservation of\n",
      "ultrastructure.\n",
      "Morphological and molecular characteristics determine the function of biological tissues. Attempts\n",
      "to combine immunofluorescence and electron microscopy invariably compromise the quality of the\n",
      "ultrastructure of tissue sections. We developed NATIVE, a correlated light and electron microscopy\n",
      "approach that preserves ultrastructure while showing the locations of multiple molecular moieties,\n",
      "even deep within tissues. This technique allowed the large-scale 3D reconstruction of a volume of\n",
      "mouse hippocampal CA3 tissue at nanometer resolution.\n",
      "['electron microscopy',\n",
      " 'immunofluorescence',\n",
      " 'correlated light and electron microscopy',\n",
      " 'nanometer resolution']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "30420685\n",
      "Fast, in vivo voltage imaging using a red fluorescent indicator.\n",
      "Genetically encoded voltage indicators (GEVIs) are emerging optical tools for acquiring brain-wide\n",
      "cell-type-specific functional data at unparalleled temporal resolution. To broaden the application\n",
      "of GEVIs in high-speed multispectral imaging, we used a high-throughput strategy to develop voltage-\n",
      "activated red neuronal activity monitor (VARNAM), a fusion of the fast Acetabularia opsin and the\n",
      "bright red fluorophore mRuby3. Imageable under the modest illumination intensities required by\n",
      "bright green probes (<50 mW mm<sup>-2</sup>), VARNAM is readily usable in vivo. VARNAM can be\n",
      "combined with blue-shifted optical tools to enable cell-type-specific all-optical electrophysiology\n",
      "and dual-color spike imaging in acute brain slices and live Drosophila. With enhanced sensitivity to\n",
      "subthreshold voltages, VARNAM resolves postsynaptic potentials in slices and cortical and\n",
      "hippocampal rhythms in freely behaving mice. Together, VARNAM lends a new hue to the optical\n",
      "toolbox, opening the door to high-speed in vivo multispectral functional imaging.\n",
      "['high-speed multispectral imaging',\n",
      " 'in vivo voltage imaging',\n",
      " 'optical tools',\n",
      " 'cell-type-specific all-optical electrophysiology',\n",
      " 'dual-color spike imaging',\n",
      " 'postsynaptic potentials',\n",
      " 'cortical and hippocampal rhythms']\n",
      "Elapsed time: 13.69 seconds\n",
      "-------------------\n",
      "30420686\n",
      "Closed-loop all-optical interrogation of neural circuits in vivo.\n",
      "Understanding the causal relationship between neural activity and behavior requires the ability to\n",
      "perform rapid and targeted interventions in ongoing activity. Here we describe a closed-loop all-\n",
      "optical strategy for dynamically controlling neuronal activity patterns in awake mice. We rapidly\n",
      "tailored and delivered two-photon optogenetic stimulation based on online readout of activity using\n",
      "simultaneous two-photon imaging, thus enabling the manipulation of neural circuit activity 'on the\n",
      "fly' during behavior.\n",
      "['simultaneous two-photon imaging', 'two-photon optogenetic stimulation']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "30478322\n",
      "Faster, sharper, and deeper: structured illumination microscopy for biological imaging.\n",
      "Structured illumination microscopy (SIM) allows rapid, super-resolution (SR) imaging in live\n",
      "specimens. We review recent technical advances in SR-SIM, with emphasis on imaging speed,\n",
      "resolution, and depth. Since its introduction decades ago, the technique has grown to offer myriad\n",
      "implementations, each with its own strengths and weaknesses. We discuss these, aiming to provide a\n",
      "practical guide for biologists and to highlight which approach is best suited to a given\n",
      "application.\n",
      "['structured illumination microscopy',\n",
      " 'super-resolution (SR) imaging',\n",
      " 'live specimens',\n",
      " 'practical guide for biologists']\n",
      "Elapsed time: 7.80 seconds\n",
      "-------------------\n",
      "30478324\n",
      "Proximity-CLIP provides a snapshot of protein-occupied RNA elements in subcellular compartments.\n",
      "Methods for the systematic study of subcellular RNA localization are limited, and their development\n",
      "has lagged behind that of proteomic tools. We combined APEX2-mediated proximity biotinylation of\n",
      "proteins with photoactivatable ribonucleoside-enhanced crosslinking to simultaneously profile the\n",
      "proteome and the transcriptome bound by RNA-binding proteins in any given subcellular compartment.\n",
      "Our approach is fractionation independent and allows study of the localization of RNA processing\n",
      "intermediates, as well as the identification of regulatory RNA cis-acting elements occupied by\n",
      "proteins, in a cellular-compartment-specific manner. We used our method, Proximity-CLIP, to profile\n",
      "RNA and protein in the nucleus, in the cytoplasm, and at cell-cell interfaces. Among other insights,\n",
      "we observed frequent transcriptional readthrough continuing for several kilobases downstream of the\n",
      "canonical cleavage and polyadenylation site and a differential RBP occupancy pattern for mRNAs in\n",
      "the nucleus and cytoplasm. We observed that mRNAs localized to cell-cell interfaces often encoded\n",
      "regulatory proteins and contained protein-occupied CUG sequence elements in their 3' untranslated\n",
      "region.\n",
      "['Proximity-CLIP',\n",
      " 'photoactivatable ribonucleoside-enhanced crosslinking',\n",
      " 'APEX2-mediated proximity biotinylation of proteins']\n",
      "Elapsed time: 12.97 seconds\n",
      "-------------------\n",
      "30478326\n",
      "Content-aware image restoration: pushing the limits of fluorescence microscopy.\n",
      "Fluorescence microscopy is a key driver of discoveries in the life sciences, with observable\n",
      "phenomena being limited by the optics of the microscope, the chemistry of the fluorophores, and the\n",
      "maximum photon exposure tolerated by the sample. These limits necessitate trade-offs between imaging\n",
      "speed, spatial resolution, light exposure, and imaging depth. In this work we show how content-aware\n",
      "image restoration based on deep learning extends the range of biological phenomena observable by\n",
      "microscopy. We demonstrate on eight concrete examples how microscopy images can be restored even if\n",
      "60-fold fewer photons are used during acquisition, how near isotropic resolution can be achieved\n",
      "with up to tenfold under-sampling along the axial direction, and how tubular and granular structures\n",
      "smaller than the diffraction limit can be resolved at 20-times-higher frame rates compared to state-\n",
      "of-the-art methods. All developed image restoration methods are freely available as open source\n",
      "software in Python, FIJI, and KNIME.\n",
      "['content-aware image restoration', 'fluorescence microscopy', 'deep learning']\n",
      "Elapsed time: 10.24 seconds\n",
      "-------------------\n",
      "30504871\n",
      "A particle-filter framework for robust cryo-EM 3D reconstruction.\n",
      "Single-particle electron cryomicroscopy (cryo-EM) involves estimating a set of parameters for each\n",
      "particle image and reconstructing a 3D density map; robust algorithms with accurate parameter\n",
      "estimation are essential for high resolution and automation. We introduce a particle-filter\n",
      "algorithm for cryo-EM, which provides high-dimensional parameter estimation through a posterior\n",
      "probability density function (PDF) of the parameters given in the model and the experimental image.\n",
      "The framework uses a set of random support points to represent such a PDF and assigns weighting\n",
      "coefficients not only among the parameters of each particle but also among different particles. We\n",
      "implemented the algorithm in a new program named THUNDER, which features self-adaptive parameter\n",
      "adjustment, tolerance to bad particles, and per-particle defocus refinement. We tested the algorithm\n",
      "by using cryo-EM datasets for the cyclic-nucleotide-gated (CNG) channel, the proteasome,\n",
      "β-galactosidase, and an influenza hemagglutinin (HA) trimer, and observed substantial improvement in\n",
      "resolution.\n",
      "['cryo-EM',\n",
      " 'particle-filter algorithm',\n",
      " 'posterior probability density function (PDF)',\n",
      " 'random support points',\n",
      " 'self-adaptive parameter adjustment',\n",
      " 'tolerance to bad particles',\n",
      " 'per-particle defocus refinement']\n",
      "Elapsed time: 11.63 seconds\n",
      "-------------------\n",
      "30504888\n",
      "Brain-wide circuit interrogation at the cellular level guided by online analysis of neuronal\n",
      "function.\n",
      "Whole-brain imaging allows for comprehensive functional mapping of distributed neural pathways, but\n",
      "neuronal perturbation experiments are usually limited to targeting predefined regions or genetically\n",
      "identifiable cell types. To complement whole-brain measures of activity with brain-wide\n",
      "manipulations for testing causal interactions, we introduce a system that uses measured activity\n",
      "patterns to guide optical perturbations of any subset of neurons in the same fictively behaving\n",
      "larval zebrafish. First, a light-sheet microscope collects whole-brain data that are rapidly\n",
      "analyzed by a distributed computing system to generate functional brain maps. On the basis of these\n",
      "maps, the experimenter can then optically ablate neurons and image activity changes across the\n",
      "brain. We applied this method to characterize contributions of behaviorally tuned populations to the\n",
      "optomotor response. We extended the system to optogenetically stimulate arbitrary subsets of neurons\n",
      "during whole-brain imaging. These open-source methods enable delineating the contributions of\n",
      "neurons to brain-wide circuit dynamics and behavior in individual animals.\n",
      "['light-sheet microscope',\n",
      " 'whole-brain imaging',\n",
      " 'optical perturbations',\n",
      " 'optogenetically stimulate']\n",
      "Elapsed time: 10.38 seconds\n",
      "-------------------\n",
      "30504889\n",
      "An explant technique for high-resolution imaging and manipulation of mycobacterial granulomas.\n",
      "A central and critical structure in tuberculosis, the mycobacterial granuloma consists of highly\n",
      "organized immune cells, including macrophages that drive granuloma formation through a\n",
      "characteristic epithelioid transformation. Difficulties in imaging within intact animals and caveats\n",
      "associated with in vitro assembly models have severely limited the study and experimental\n",
      "manipulation of mature granulomas. Here we describe a new ex vivo culture technique, wherein mature,\n",
      "fully organized zebrafish granulomas are microdissected and maintained in three-dimensional (3D)\n",
      "culture. This approach enables high-resolution microscopy of granuloma macrophage dynamics,\n",
      "including epithelioid macrophage motility and granuloma consolidation, while retaining key bacterial\n",
      "and host characteristics. Using mass spectrometry, we find active production of key\n",
      "phosphotidylinositol species identified previously in human granulomas. We also describe a method to\n",
      "transfect isolated granulomas, enabling genetic manipulation, and provide proof-of-concept for host-\n",
      "directed small-molecule screens, identifying protein kinase C (PKC) signaling as an important\n",
      "regulator of granuloma macrophage organization.\n",
      "['high-resolution microscopy',\n",
      " 'mass spectrometry',\n",
      " 'genetic manipulation',\n",
      " 'host-directed small-molecule screens']\n",
      "Elapsed time: 11.64 seconds\n",
      "-------------------\n",
      "30532080\n",
      "fMRIPrep: a robust preprocessing pipeline for functional MRI.\n",
      "Preprocessing of functional magnetic resonance imaging (fMRI) involves numerous steps to clean and\n",
      "standardize the data before statistical analysis. Generally, researchers create ad hoc preprocessing\n",
      "workflows for each dataset, building upon a large inventory of available tools. The complexity of\n",
      "these workflows has snowballed with rapid advances in acquisition and processing. We introduce\n",
      "fMRIPrep, an analysis-agnostic tool that addresses the challenge of robust and reproducible\n",
      "preprocessing for fMRI data. fMRIPrep automatically adapts a best-in-breed workflow to the\n",
      "idiosyncrasies of virtually any dataset, ensuring high-quality preprocessing without manual\n",
      "intervention. By introducing visual assessment checkpoints into an iterative integration framework\n",
      "for software testing, we show that fMRIPrep robustly produces high-quality results on a diverse fMRI\n",
      "data collection. Additionally, fMRIPrep introduces less uncontrolled spatial smoothness than\n",
      "observed with commonly used preprocessing tools. fMRIPrep equips neuroscientists with an easy-to-use\n",
      "and transparent preprocessing workflow, which can help ensure the validity of inference and the\n",
      "interpretability of results.\n",
      "['fMRI', 'functional magnetic resonance imaging']\n",
      "Elapsed time: 10.14 seconds\n",
      "-------------------\n",
      "30532082\n",
      "A pH-correctable, DNA-based fluorescent reporter for organellar calcium.\n",
      "It is extremely challenging to quantitate lumenal Ca<sup>2+</sup> in acidic Ca<sup>2+</sup> stores\n",
      "of the cell because all Ca<sup>2+</sup> indicators are pH sensitive, and Ca<sup>2+</sup> transport\n",
      "is coupled to pH in acidic organelles. We have developed a fluorescent DNA-based reporter,\n",
      "CalipHluor, that is targetable to specific organelles. By ratiometrically reporting lumenal pH and\n",
      "Ca<sup>2+</sup> simultaneously, CalipHluor functions as a pH-correctable Ca<sup>2+</sup> reporter.\n",
      "By targeting CalipHluor to the endolysosomal pathway, we mapped lumenal Ca<sup>2+</sup> changes\n",
      "during endosomal maturation and found a surge in lumenal Ca<sup>2+</sup> specifically in lysosomes.\n",
      "Using lysosomal proteomics and genetic analysis, we found that catp-6, a Caenorhabditis elegans\n",
      "homolog of ATP13A2, was responsible for lysosomal Ca<sup>2+</sup> accumulation-an example of a\n",
      "lysosome-specific Ca<sup>2+</sup> importer in animals. By enabling the facile quantification of\n",
      "compartmentalized Ca<sup>2+</sup>, CalipHluor can expand the understanding of subcellular\n",
      "Ca<sup>2+</sup> importers.\n",
      "['fluorescent reporter',\n",
      " 'ratiometric reporting',\n",
      " 'endolysosomal pathway',\n",
      " 'lysosome-specific Ca2+ importer']\n",
      "Elapsed time: 22.50 seconds\n",
      "-------------------\n",
      "30559429\n",
      "U-Net: deep learning for cell counting, detection, and morphometry.\n",
      "U-Net is a generic deep-learning solution for frequently occurring quantification tasks such as cell\n",
      "detection and shape measurements in biomedical image data. We present an ImageJ plugin that enables\n",
      "non-machine-learning experts to analyze their data with U-Net on either a local computer or a remote\n",
      "server/cloud service. The plugin comes with pretrained models for single-cell segmentation and\n",
      "allows for U-Net to be adapted to new tasks on the basis of a few annotated samples.\n",
      "['cell detection',\n",
      " 'cell segmentation',\n",
      " 'shape measurements',\n",
      " 'biomedical image data']\n",
      "Elapsed time: 7.04 seconds\n",
      "-------------------\n",
      "30559430\n",
      "Imaging cellular ultrastructures using expansion microscopy (U-ExM).\n",
      "Determining the structure and composition of macromolecular assemblies is a major challenge in\n",
      "biology. Here we describe ultrastructure expansion microscopy (U-ExM), an extension of expansion\n",
      "microscopy that allows the visualization of preserved ultrastructures by optical microscopy. This\n",
      "method allows for near-native expansion of diverse structures in vitro and in cells; when combined\n",
      "with super-resolution microscopy, it unveiled details of ultrastructural organization, such as\n",
      "centriolar chirality, that could otherwise be observed only by electron microscopy.\n",
      "['expansion microscopy',\n",
      " 'ultrastructure expansion microscopy',\n",
      " 'super-resolution microscopy',\n",
      " 'electron microscopy']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "30559434\n",
      "Deep learning enables cross-modality super-resolution in fluorescence microscopy.\n",
      "We present deep-learning-enabled super-resolution across different fluorescence microscopy\n",
      "modalities. This data-driven approach does not require numerical modeling of the imaging process or\n",
      "the estimation of a point-spread-function, and is based on training a generative adversarial network\n",
      "(GAN) to transform diffraction-limited input images into super-resolved ones. Using this framework,\n",
      "we improve the resolution of wide-field images acquired with low-numerical-aperture objectives,\n",
      "matching the resolution that is acquired using high-numerical-aperture objectives. We also\n",
      "demonstrate cross-modality super-resolution, transforming confocal microscopy images to match the\n",
      "resolution acquired with a stimulated emission depletion (STED) microscope. We further demonstrate\n",
      "that total internal reflection fluorescence (TIRF) microscopy images of subcellular structures\n",
      "within cells and tissues can be transformed to match the results obtained with a TIRF-based\n",
      "structured illumination microscope. The deep network rapidly outputs these super-resolved images,\n",
      "without any iterations or parameter search, and could serve to democratize super-resolution imaging.\n",
      "['generative adversarial network (GAN)',\n",
      " 'stimulated emission depletion (STED) microscope',\n",
      " 'total internal reflection fluorescence (TIRF) microscopy',\n",
      " 'TIRF-based structured illumination microscope']\n",
      "Elapsed time: 12.24 seconds\n",
      "-------------------\n",
      "30573813\n",
      "Expansion microscopy: principles and uses in biological research.\n",
      "Many biological investigations require 3D imaging of cells or tissues with nanoscale spatial\n",
      "resolution. We recently discovered that preserved biological specimens can be physically expanded in\n",
      "an isotropic fashion through a chemical process. Expansion microscopy (ExM) allows nanoscale imaging\n",
      "of biological specimens with conventional microscopes, decrowds biomolecules in support of signal\n",
      "amplification and multiplexed readout chemistries, and makes specimens transparent. We review the\n",
      "principles of how ExM works, advances in the technology made by our group and others, and its\n",
      "applications throughout biology and medicine.\n",
      "['expansion microscopy',\n",
      " 'conventional microscopes',\n",
      " 'decrowding biomolecules',\n",
      " 'signal amplification',\n",
      " 'multiplexed readout chemistries',\n",
      " 'transparent specimens']\n",
      "Elapsed time: 8.70 seconds\n",
      "-------------------\n",
      "30573816\n",
      "Evaluation of variability in human kidney organoids.\n",
      "The utility of human pluripotent stem cell-derived kidney organoids relies implicitly on the\n",
      "robustness and transferability of the protocol. Here we analyze the sources of transcriptional\n",
      "variation in a specific kidney organoid protocol. Although individual organoids within a\n",
      "differentiation batch showed strong transcriptional correlation, we noted significant variation\n",
      "between experimental batches, particularly in genes associated with temporal maturation. Single-cell\n",
      "profiling revealed shifts in nephron patterning and proportions of component cells. Distinct induced\n",
      "pluripotent stem cell clones showed congruent transcriptional programs, with interexperimental and\n",
      "interclonal variation also strongly associated with nephron patterning. Epithelial cells isolated\n",
      "from organoids aligned with total organoids at the same day of differentiation, again implicating\n",
      "relative maturation as a confounder. This understanding of experimental variation facilitated an\n",
      "optimized analysis of organoid-based disease modeling, thereby increasing the utility of kidney\n",
      "organoids for personalized medicine and functional genomics.\n",
      "['single-cell profiling',\n",
      " 'nephron patterning',\n",
      " 'induced pluripotent stem cell clones']\n",
      "Elapsed time: 10.34 seconds\n",
      "-------------------\n",
      "30573820\n",
      "Fast animal pose estimation using deep neural networks.\n",
      "The need for automated and efficient systems for tracking full animal pose has increased with the\n",
      "complexity of behavioral data and analyses. Here we introduce LEAP (LEAP estimates animal pose), a\n",
      "deep-learning-based method for predicting the positions of animal body parts. This framework\n",
      "consists of a graphical interface for labeling of body parts and training the network. LEAP offers\n",
      "fast prediction on new data, and training with as few as 100 frames results in 95% of peak\n",
      "performance. We validated LEAP using videos of freely behaving fruit flies and tracked 32 distinct\n",
      "points to describe the pose of the head, body, wings and legs, with an error rate of <3% of body\n",
      "length. We recapitulated reported findings on insect gait dynamics and demonstrated LEAP's\n",
      "applicability for unsupervised behavioral classification. Finally, we extended the method to more\n",
      "challenging imaging situations and videos of freely moving mice.\n",
      "['LEAP',\n",
      " 'deep neural networks',\n",
      " 'graphical interface',\n",
      " 'body parts labeling',\n",
      " 'video training']\n",
      "Elapsed time: 9.65 seconds\n",
      "-------------------\n",
      "30573831\n",
      "A compressed sensing framework for efficient dissection of neural circuits.\n",
      "A fundamental question in neuroscience is how neural networks generate behavior. The lack of genetic\n",
      "tools and unique promoters to functionally manipulate specific neuronal subtypes makes it\n",
      "challenging to determine the roles of individual subtypes in behavior. We describe a compressed\n",
      "sensing-based framework in combination with non-specific genetic tools to infer candidate neurons\n",
      "controlling behaviors with fewer measurements than previously thought possible. We tested this\n",
      "framework by inferring interneuron subtypes regulating the speed of locomotion of the nematode\n",
      "Caenorhabditis elegans. We developed a real-time stabilization microscope for accurate long-term,\n",
      "high-magnification imaging and targeted perturbation of neural activity in freely moving animals to\n",
      "validate our inferences. We show that a circuit of three interconnected interneuron subtypes, RMG,\n",
      "AVB and SIA control different aspects of locomotion speed as the animal navigates its environment.\n",
      "Our work suggests that compressed sensing approaches can be used to identify key nodes in complex\n",
      "biological networks.\n",
      "['real-time stabilization microscope',\n",
      " 'targeted perturbation of neural activity',\n",
      " 'high-magnification imaging']\n",
      "Elapsed time: 10.19 seconds\n",
      "-------------------\n",
      "30594947\n",
      "Author Correction: Faster, sharper, and deeper: structured illumination microscopy for biological\n",
      "imaging.\n",
      "In the version of this Perspective originally published, Fig. 4g included an incorrect inset adapted\n",
      "from a different figure than the main image in the panel. This error has been corrected in the PDF\n",
      "and HTML versions of the paper.\n",
      "['structured illumination microscopy', 'PDF', 'HTML']\n",
      "Elapsed time: 5.73 seconds\n",
      "-------------------\n",
      "30602783\n",
      "Publisher Correction: Stability, affinity, and chromatic variants of the glutamate sensor iGluSnFR.\n",
      "In the version of this paper originally published, important figure labels in Fig. 3d were not\n",
      "visible. An image layer present in the authors' original figure that included two small dashed\n",
      "outlines and text labels indicating ROI 1 and ROI 2, as well as a scale bar and the name of the cell\n",
      "label, was erroneously altered during image processing. The figure has been corrected in the HTML\n",
      "and PDF versions of the paper.\n",
      "['image layer', 'scale bar', 'cell label']\n",
      "Elapsed time: 6.98 seconds\n",
      "-------------------\n",
      "30643212\n",
      "A robust and versatile platform for image scanning microscopy enabling super-resolution FLIM.\n",
      "Image scanning microscopy (ISM) can improve the effective spatial resolution of confocal microscopy\n",
      "to its theoretical limit. However, current implementations are not robust or versatile, and are\n",
      "incompatible with fluorescence lifetime imaging (FLIM). We describe an implementation of ISM based\n",
      "on a single-photon detector array that enables super-resolution FLIM and improves multicolor, live-\n",
      "cell and in-depth imaging, thereby paving the way for a massive transition from confocal microscopy\n",
      "to ISM.\n",
      "['image scanning microscopy',\n",
      " 'fluorescence lifetime imaging',\n",
      " 'confocal microscopy',\n",
      " 'single-photon detector array']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "30643213\n",
      "Metagenomic engineering of the mammalian gut microbiome in situ.\n",
      "Engineering of microbial communities in open environments remains challenging. Here we describe a\n",
      "platform used to identify and modify genetically tractable mammalian microbiota by engineering\n",
      "community-wide horizontal gene transfer events in situ. With this approach, we demonstrate that\n",
      "diverse taxa in the mouse gut microbiome can be modified directly with a desired genetic payload. In\n",
      "situ microbiome engineering in living animals allows novel capabilities to be introduced into\n",
      "established communities in their native milieu.\n",
      "['horizontal gene transfer events',\n",
      " 'in situ microbiome engineering',\n",
      " 'microbiota engineering']\n",
      "Elapsed time: 7.65 seconds\n",
      "-------------------\n",
      "30664776\n",
      "An online resource for GPCR structure determination and analysis.\n",
      "G-protein-coupled receptors (GPCRs) transduce physiological and sensory stimuli into appropriate\n",
      "cellular responses and mediate the actions of one-third of drugs. GPCR structural studies have\n",
      "revealed the general bases of receptor activation, signaling, drug action and allosteric modulation,\n",
      "but so far cover only 13% of nonolfactory receptors. We broadly surveyed the receptor\n",
      "modifications/engineering and methods used to produce all available GPCR crystal and cryo-electron\n",
      "microscopy (cryo-EM) structures, and present an interactive resource integrated in GPCRdb (\n",
      "http://www.gpcrdb.org ) to assist users in designing constructs and browsing appropriate\n",
      "experimental conditions for structure studies.\n",
      "['cryo-electron microscopy (cryo-EM)',\n",
      " 'crystal structures',\n",
      " 'interactive resource']\n",
      "Elapsed time: 8.93 seconds\n",
      "-------------------\n",
      "30664778\n",
      "A genetically encoded near-infrared fluorescent calcium ion indicator.\n",
      "We report an intensiometric, near-infrared fluorescent, genetically encoded calcium ion\n",
      "(Ca<sup>2+</sup>) indicator (GECI) with excitation and emission maxima at 678 and 704 nm,\n",
      "respectively. This GECI, designated NIR-GECO1, enables imaging of Ca<sup>2+</sup> transients in\n",
      "cultured mammalian cells and brain tissue with sensitivity comparable to that of currently available\n",
      "visible-wavelength GECIs. We demonstrate that NIR-GECO1 opens up new vistas for multicolor\n",
      "Ca<sup>2+</sup> imaging in combination with other optogenetic indicators and actuators.\n",
      "['intensiometric',\n",
      " 'near-infrared fluorescent',\n",
      " 'genetically encoded calcium ion (Ca2+) indicator (GECI)']\n",
      "Elapsed time: 9.70 seconds\n",
      "-------------------\n",
      "30742039\n",
      "Flow-enhanced vascularization and maturation of kidney organoids in vitro.\n",
      "Kidney organoids derived from human pluripotent stem cells have glomerular- and tubular-like\n",
      "compartments that are largely avascular and immature in static culture. Here we report an in vitro\n",
      "method for culturing kidney organoids under flow on millifluidic chips, which expands their\n",
      "endogenous pool of endothelial progenitor cells and generates vascular networks with perfusable\n",
      "lumens surrounded by mural cells. We found that vascularized kidney organoids cultured under flow\n",
      "had more mature podocyte and tubular compartments with enhanced cellular polarity and adult gene\n",
      "expression compared with that in static controls. Glomerular vascular development progressed through\n",
      "intermediate stages akin to those involved in the embryonic mammalian kidney's formation of\n",
      "capillary loops abutting foot processes. The association of vessels with these compartments was\n",
      "reduced after disruption of the endogenous VEGF gradient. The ability to induce substantial\n",
      "vascularization and morphological maturation of kidney organoids in vitro under flow opens new\n",
      "avenues for studies of kidney development, disease, and regeneration.\n",
      "['millifluidic chips', 'flow']\n",
      "Elapsed time: 11.10 seconds\n",
      "-------------------\n",
      "30742041\n",
      "Noninvasive monitoring of single-cell mechanics by acoustic scattering.\n",
      "The monitoring of mechanics in a single cell throughout the cell cycle has been hampered by the\n",
      "invasiveness of mechanical measurements. Here we quantify mechanical properties via acoustic\n",
      "scattering of waves from a cell inside a fluid-filled vibrating cantilever with a temporal\n",
      "resolution of < 1 min. Through simulations, experiments with hydrogels and the use of chemically\n",
      "perturbed cells, we show that our readout, the size-normalized acoustic scattering (SNACS), measures\n",
      "stiffness. To demonstrate the noninvasiveness of SNACS over successive cell cycles, we used\n",
      "measurements that resulted in deformations of < 15 nm. The cells maintained constant SNACS\n",
      "throughout interphase but showed dynamic changes during mitosis. Our work provides a basis for\n",
      "understanding how growing cells maintain mechanical integrity, and demonstrates that acoustic\n",
      "scattering can be used to noninvasively probe subtle and transient dynamics.\n",
      "['acoustic scattering',\n",
      " 'size-normalized acoustic scattering (SNACS)',\n",
      " 'temporal resolution']\n",
      "Elapsed time: 9.83 seconds\n",
      "-------------------\n",
      "30804550\n",
      "Universal light-sheet generation with field synthesis.\n",
      "We introduce field synthesis, a theorem and method that can be used to synthesize any scanned or\n",
      "dithered light sheet, including those used in lattice light-sheet microscopy (LLSM), from an\n",
      "incoherent superposition of one-dimensional intensity distributions. Compared to LLSM, this user-\n",
      "friendly and modular approach offers a simplified optical design, higher light throughput and\n",
      "simultaneous multicolor illumination. Further, field synthesis achieves lower rates of\n",
      "photobleaching than light sheets generated by lateral beam scanning.\n",
      "['lattice light-sheet microscopy (LLSM)',\n",
      " 'field synthesis',\n",
      " 'simultaneous multicolor illumination']\n",
      "Elapsed time: 7.51 seconds\n",
      "-------------------\n",
      "30804552\n",
      "Author Correction: U-Net: deep learning for cell counting, detection, and morphometry.\n",
      "In the version of this paper originally published, one of the affiliations for Dominic Mai was\n",
      "incorrect: \"Center for Biological Systems Analysis (ZBSA), Albert-Ludwigs-University, Freiburg,\n",
      "Germany\" should have been \"Life Imaging Center, Center for Biological Systems Analysis, Albert-\n",
      "Ludwigs-University, Freiburg, Germany.\" This change required some renumbering of subsequent author\n",
      "affiliations. These corrections have been made in the PDF and HTML versions of the article, as well\n",
      "as in any cover sheets for associated Supplementary Information.\n",
      "['Life Imaging Center',\n",
      " 'deep learning',\n",
      " 'cell counting',\n",
      " 'detection',\n",
      " 'morphometry']\n",
      "Elapsed time: 8.07 seconds\n",
      "-------------------\n",
      "30858598\n",
      "Aromatic <sup>19</sup>F-<sup>13</sup>C TROSY: a background-free approach to probe biomolecular\n",
      "structure, function, and dynamics.\n",
      "Atomic-level information about the structure and dynamics of biomolecules is critical for an\n",
      "understanding of their function. Nuclear magnetic resonance (NMR) spectroscopy provides unique\n",
      "insights into the dynamic nature of biomolecules and their interactions, capturing transient\n",
      "conformers and their features. However, relaxation-induced line broadening and signal overlap make\n",
      "it challenging to apply NMR spectroscopy to large biological systems. Here we took advantage of the\n",
      "high sensitivity and broad chemical shift range of <sup>19</sup>F nuclei and leveraged the\n",
      "remarkable relaxation properties of the aromatic <sup>19</sup>F-<sup>13</sup>C spin pair to disperse\n",
      "<sup>19</sup>F resonances in a two-dimensional transverse relaxation-optimized spectroscopy\n",
      "spectrum. We demonstrate the application of <sup>19</sup>F-<sup>13</sup>C transverse relaxation-\n",
      "optimized spectroscopy to investigate proteins and nucleic acids. This experiment expands the scope\n",
      "of <sup>19</sup>F NMR in the study of the structure, dynamics, and function of large and complex\n",
      "biological systems and provides a powerful background-free NMR probe.\n",
      "['NMR spectroscopy', 'TROSY', 'transverse relaxation-optimized spectroscopy']\n",
      "Elapsed time: 16.51 seconds\n",
      "-------------------\n",
      "30858600\n",
      "An active texture-based digital atlas enables automated mapping of structures and markers across\n",
      "brains.\n",
      "Brain atlases enable the mapping of labeled cells and projections from different brains onto a\n",
      "standard coordinate system. We address two issues in the construction and use of atlases. First,\n",
      "expert neuroanatomists ascertain the fine-scale pattern of brain tissue, the 'texture' formed by\n",
      "cellular organization, to define cytoarchitectural borders. We automate the processes of localizing\n",
      "landmark structures and alignment of brains to a reference atlas using machine learning and training\n",
      "data derived from expert annotations. Second, we construct an atlas that is active; that is,\n",
      "augmented with each use. We show that the alignment of new brains to a reference atlas can\n",
      "continuously refine the coordinate system and associated variance. We apply this approach to the\n",
      "adult murine brainstem and achieve a precise alignment of projections in cytoarchitecturally ill-\n",
      "defined regions across brains from different animals.\n",
      "['machine learning', 'training data', 'expert annotations']\n",
      "Elapsed time: 9.19 seconds\n",
      "-------------------\n",
      "30923372\n",
      "SNAC-tag for sequence-specific chemical protein cleavage.\n",
      "Site-specific protein cleavage is essential for many protein-production protocols and typically\n",
      "requires proteases. We report the development of a chemical protein-cleavage method that is achieved\n",
      "through the use of a sequence-specific nickel-assisted cleavage (SNAC)-tag. We demonstrate that the\n",
      "SNAC-tag can be inserted before both water-soluble and membrane proteins to achieve fusion protein\n",
      "cleavage under biocompatible conditions with efficiency comparable to that of enzymes, and that the\n",
      "method works even when enzymatic cleavages fail.\n",
      "['SNAC-tag',\n",
      " 'chemical protein cleavage',\n",
      " 'sequence-specific nickel-assisted cleavage']\n",
      "Elapsed time: 7.29 seconds\n",
      "-------------------\n",
      "30962622\n",
      "In vivo profiling of metastatic double knockouts through CRISPR-Cpf1 screens.\n",
      "Systematic investigation of the genetic interactions that influence metastatic potential has been\n",
      "challenging. Here we developed massively parallel CRISPR-Cpf1/Cas12a crRNA array profiling (MCAP),\n",
      "an approach for combinatorial interrogation of double knockouts in vivo. We designed an MCAP library\n",
      "of 11,934 arrays targeting 325 pairwise combinations of genes implicated in metastasis. By assessing\n",
      "the metastatic potential of the double knockouts in mice, we unveiled a quantitative landscape of\n",
      "genetic interactions that drive metastasis.\n",
      "[{'name': 'CRISPR-Cpf1 screens'},\n",
      " {'name': 'massively parallel CRISPR-Cpf1/Cas12a crRNA array profiling (MCAP)'}]\n",
      "Elapsed time: 9.18 seconds\n",
      "-------------------\n",
      "30962624\n",
      "Super-resolution fight club: assessment of 2D and 3D single-molecule localization microscopy\n",
      "software.\n",
      "With the widespread uptake of two-dimensional (2D) and three-dimensional (3D) single-molecule\n",
      "localization microscopy (SMLM), a large set of different data analysis packages have been developed\n",
      "to generate super-resolution images. In a large community effort, we designed a competition to\n",
      "extensively characterize and rank the performance of 2D and 3D SMLM software packages. We generated\n",
      "realistic simulated datasets for popular imaging modalities-2D, astigmatic 3D, biplane 3D and\n",
      "double-helix 3D-and evaluated 36 participant packages against these data. This provides the first\n",
      "broad assessment of 3D SMLM software and provides a holistic view of how the latest 2D and 3D SMLM\n",
      "packages perform in realistic conditions. This resource allows researchers to identify optimal\n",
      "analytical software for their experiments, allows 3D SMLM software developers to benchmark new\n",
      "software against the current state of the art, and provides insight into the current limits of the\n",
      "field.\n",
      "['two-dimensional (2D)',\n",
      " 'three-dimensional (3D)',\n",
      " 'single-molecule localization microscopy (SMLM)',\n",
      " 'astigmatic 3D',\n",
      " 'biplane 3D',\n",
      " 'double-helix 3D']\n",
      "Elapsed time: 11.46 seconds\n",
      "-------------------\n",
      "30988469\n",
      "A photocleavable surfactant for top-down proteomics.\n",
      "We report the identification of a photocleavable anionic surfactant, 4-hexylphenylazosulfonate\n",
      "(Azo), which can be rapidly degraded by ultraviolet irradiation, for top-down proteomics. Azo can\n",
      "effectively solubilize proteins with performance comparable to that of sodium dodecyl sulfate (SDS)\n",
      "and is compatible with mass spectrometry. Azo-aided top-down proteomics enables the solubilization\n",
      "of membrane proteins for comprehensive characterization of post-translational modifications.\n",
      "Moreover, Azo is simple to synthesize and can be used as a general SDS replacement in SDS-\n",
      "polyacrylamide gel electrophoresis.\n",
      "['mass spectrometry', 'SDS-polyacrylamide gel electrophoresis']\n",
      "Elapsed time: 8.49 seconds\n",
      "-------------------\n",
      "31011184\n",
      "Engineered peptide barcodes for in-depth analyses of binding protein libraries.\n",
      "Binding protein generation typically relies on laborious screening cascades that process candidate\n",
      "molecules individually. We have developed NestLink, a binder selection and identification technology\n",
      "able to biophysically characterize thousands of library members at once without the need to handle\n",
      "individual clones at any stage of the process. NestLink uses genetically encoded barcoding peptides\n",
      "termed flycodes, which were designed for maximal detectability by mass spectrometry and support\n",
      "accurate deep sequencing. We demonstrate NestLink's capacity to overcome the current limitations of\n",
      "binder-generation methods in three applications. First, we show that hundreds of binder candidates\n",
      "can be simultaneously ranked according to kinetic parameters. Next, we demonstrate deep mining of a\n",
      "nanobody immune repertoire for membrane protein binders, carried out entirely in solution without\n",
      "target immobilization. Finally, we identify rare binders against an integral membrane protein\n",
      "directly in the cellular environment of a human pathogen. NestLink opens avenues for the selection\n",
      "of tailored binder characteristics directly in tissues or in living organisms.\n",
      "['mass spectrometry', 'deep sequencing']\n",
      "Elapsed time: 10.01 seconds\n",
      "-------------------\n",
      "31036959\n",
      "Instantaneous isotropic volumetric imaging of fast biological processes.\n",
      "To capture highly dynamic biological processes at cellular resolution is a recurring challenge in\n",
      "biology. Here we show that combining selective-volume illumination with simultaneous acquisition of\n",
      "orthogonal light fields yields three-dimensional images with high, isotropic spatial resolution and\n",
      "a significant reduction of reconstruction artefacts, thereby overcoming current limitations of\n",
      "light-field microscopy implementations. We demonstrate medaka heart and blood flow imaging at\n",
      "single-cell resolution and free of motion artefacts at volume rates of up to 200 Hz.\n",
      "['selective-volume illumination',\n",
      " 'simultaneous acquisition of orthogonal light fields',\n",
      " 'light-field microscopy']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "31040436\n",
      "The cryo-EM method microcrystal electron diffraction (MicroED).\n",
      "In 2013 we established a cryo-electron microscopy (cryo-EM) technique called microcrystal electron\n",
      "diffraction (MicroED). Since that time, data collection and analysis schemes have been fine-tuned,\n",
      "and structures for more than 40 different proteins, oligopeptides and organic molecules have been\n",
      "determined. Here we review the MicroED technique and place it in context with other structure-\n",
      "determination methods. We showcase example structures solved by MicroED and provide practical advice\n",
      "to prospective users.\n",
      "['cryo-electron microscopy', 'microcrystal electron diffraction', 'MicroED']\n",
      "Elapsed time: 7.41 seconds\n",
      "-------------------\n",
      "31061492\n",
      "Epi-illumination SPIM for volumetric imaging with high spatial-temporal resolution.\n",
      "We designed an epi-illumination SPIM system that uses a single objective and has a sample interface\n",
      "identical to that of an inverted fluorescence microscope with no additional reflection elements. It\n",
      "achieves subcellular resolution and single-molecule sensitivity, and is compatible with common\n",
      "biological sample holders, including multi-well plates. We demonstrated multicolor fast volumetric\n",
      "imaging, single-molecule localization microscopy, parallel imaging of 16 cell lines and parallel\n",
      "recording of cellular responses to perturbations.\n",
      "['single-molecule localization microscopy',\n",
      " 'fast volumetric imaging',\n",
      " 'parallel imaging',\n",
      " 'multicolor imaging',\n",
      " 'parallel recording of cellular responses to perturbations']\n",
      "Elapsed time: 8.49 seconds\n",
      "-------------------\n",
      "31086341\n",
      "Bioluminescent-based imaging and quantification of glucose uptake in vivo.\n",
      "Glucose is a major source of energy for most living organisms, and its aberrant uptake is linked to\n",
      "many pathological conditions. However, our understanding of disease-associated glucose flux is\n",
      "limited owing to the lack of robust tools. To date, positron-emission tomography imaging remains the\n",
      "gold standard for measuring glucose uptake, and no optical tools exist for non-invasive longitudinal\n",
      "imaging of this important metabolite in in vivo settings. Here, we report the development of a\n",
      "bioluminescent glucose-uptake probe for real-time, non-invasive longitudinal imaging of glucose\n",
      "absorption both in vitro and in vivo. In addition, we demonstrate that the sensitivity of our method\n",
      "is comparable with that of commonly used <sup>18</sup>F-FDG-positron-emission-tomography tracers and\n",
      "validate the bioluminescent glucose-uptake probe as a tool for the identification of new glucose\n",
      "transport inhibitors. The new imaging reagent enables a wide range of applications in the fields of\n",
      "metabolism and drug development.\n",
      "['positron-emission tomography imaging', 'bioluminescent glucose-uptake probe']\n",
      "Elapsed time: 11.80 seconds\n",
      "-------------------\n",
      "31086342\n",
      "Light-sheet microscopy in the near-infrared II window.\n",
      "Non-invasive deep-tissue three-dimensional optical imaging of live mammals with high spatiotemporal\n",
      "resolution is challenging owing to light scattering. We developed near-infrared II (1,000-1,700 nm)\n",
      "light-sheet microscopy with excitation and emission of up to approximately 1,320 nm and 1,700 nm,\n",
      "respectively, for optical sectioning at a penetration depth of approximately 750 μm through live\n",
      "tissues without invasive surgery and at a depth of approximately 2 mm in glycerol-cleared brain\n",
      "tissues. Near-infrared II light-sheet microscopy in normal and oblique configurations enabled in\n",
      "vivo imaging of live mice through intact tissue, revealing abnormal blood flow and T-cell motion in\n",
      "tumor microcirculation and mapping out programmed-death ligand 1 and programmed cell death protein 1\n",
      "in tumors with cellular resolution. Three-dimensional imaging through the intact mouse head resolved\n",
      "vascular channels between the skull and brain cortex, and allowed monitoring of recruitment of\n",
      "macrophages and microglia to the traumatic brain injury site.\n",
      "['light-sheet microscopy',\n",
      " 'near-infrared II (1,000-1,700 nm) light-sheet microscopy',\n",
      " 'optical sectioning']\n",
      "Elapsed time: 12.32 seconds\n",
      "-------------------\n",
      "31086343\n",
      "Software tools for automated transmission electron microscopy.\n",
      "The demand for high-throughput data collection in electron microscopy is increasing for applications\n",
      "in structural and cellular biology. Here we present a combination of software tools that enable\n",
      "automated acquisition guided by image analysis for a variety of transmission electron microscopy\n",
      "acquisition schemes. SerialEM controls microscopes and detectors and can trigger automated tasks at\n",
      "multiple positions with high flexibility. Py-EM interfaces with SerialEM to enact specimen-specific\n",
      "image-analysis pipelines that enable feedback microscopy. As example applications, we demonstrate\n",
      "dose reduction in cryo-electron microscopy experiments, fully automated acquisition of every cell in\n",
      "a plastic section and automated targeting on serial sections for 3D volume imaging across multiple\n",
      "grids.\n",
      "['SerialEM',\n",
      " 'Py-EM',\n",
      " 'cryo-electron microscopy',\n",
      " 'fully automated acquisition',\n",
      " '3D volume imaging']\n",
      "Elapsed time: 8.54 seconds\n",
      "-------------------\n",
      "31097821\n",
      "Publisher Correction: Super-resolution fight club: assessment of 2D and 3D single-molecule\n",
      "localization microscopy software.\n",
      "In the version of this paper originally published, Figure 4a contained errors that were introduced\n",
      "during typesetting. The bottom 11° ThunderSTORM image is an xz view but was incorrectly labeled as\n",
      "xy, and the low x-axis value in the four line profiles was incorrectly set as -60 instead of -50.\n",
      "These errors have been corrected in the PDF and HTML versions of the paper.\n",
      "['ThunderSTORM', 'single-molecule localization microscopy']\n",
      "Elapsed time: 6.96 seconds\n",
      "-------------------\n",
      "31110282\n",
      "SABER amplifies FISH: enhanced multiplexed imaging of RNA and DNA in cells and tissues.\n",
      "Fluorescence in situ hybridization (FISH) reveals the abundance and positioning of nucleic acid\n",
      "sequences in fixed samples. Despite recent advances in multiplexed amplification of FISH signals, it\n",
      "remains challenging to achieve high levels of simultaneous amplification and sequential detection\n",
      "with high sampling efficiency and simple workflows. Here we introduce signal amplification by\n",
      "exchange reaction (SABER), which endows oligonucleotide-based FISH probes with long, single-stranded\n",
      "DNA concatemers that aggregate a multitude of short complementary fluorescent imager strands. We\n",
      "show that SABER amplified RNA and DNA FISH signals (5- to 450-fold) in fixed cells and tissues. We\n",
      "also applied 17 orthogonal amplifiers against chromosomal targets simultaneously and detected mRNAs\n",
      "with high efficiency. We then used 10-plex SABER-FISH to identify in vivo introduced enhancers with\n",
      "cell-type-specific activity in the mouse retina. SABER represents a simple and versatile molecular\n",
      "toolkit for rapid and cost-effective multiplexed imaging of nucleic acid targets.\n",
      "{'imaging techniques': ['fluorescence in situ hybridization (FISH)',\n",
      "                        'signal amplification by exchange reaction (SABER)',\n",
      "                        'oligonucleotide-based FISH probes',\n",
      "                        'chromosomal targets',\n",
      "                        'mRNAs',\n",
      "                        '10-plex SABER-FISH']}\n",
      "Elapsed time: 13.52 seconds\n",
      "-------------------\n",
      "31133758\n",
      "Deep learning for cellular image analysis.\n",
      "Recent advances in computer vision and machine learning underpin a collection of algorithms with an\n",
      "impressive ability to decipher the content of images. These deep learning algorithms are being\n",
      "applied to biological images and are transforming the analysis and interpretation of imaging data.\n",
      "These advances are positioned to render difficult analyses routine and to enable researchers to\n",
      "carry out new, previously impossible experiments. Here we review the intersection between deep\n",
      "learning and cellular image analysis and provide an overview of both the mathematical mechanics and\n",
      "the programming frameworks of deep learning that are pertinent to life scientists. We survey the\n",
      "field's progress in four key applications: image classification, image segmentation, object\n",
      "tracking, and augmented microscopy. Last, we relay our labs' experience with three key aspects of\n",
      "implementing deep learning in the laboratory: annotating training data, selecting and training a\n",
      "range of neural network architectures, and deploying solutions. We also highlight existing datasets\n",
      "and implementations for each surveyed application.\n",
      "['image classification',\n",
      " 'image segmentation',\n",
      " 'object tracking',\n",
      " 'augmented microscopy']\n",
      "Elapsed time: 9.54 seconds\n",
      "-------------------\n",
      "31209382\n",
      "High-performance calcium sensors for imaging activity in neuronal populations and microcompartments.\n",
      "Calcium imaging with genetically encoded calcium indicators (GECIs) is routinely used to measure\n",
      "neural activity in intact nervous systems. GECIs are frequently used in one of two different modes:\n",
      "to track activity in large populations of neuronal cell bodies, or to follow dynamics in subcellular\n",
      "compartments such as axons, dendrites and individual synaptic compartments. Despite major advances,\n",
      "calcium imaging is still limited by the biophysical properties of existing GECIs, including\n",
      "affinity, signal-to-noise ratio, rise and decay kinetics and dynamic range. Using structure-guided\n",
      "mutagenesis and neuron-based screening, we optimized the green fluorescent protein-based GECI GCaMP6\n",
      "for different modes of in vivo imaging. The resulting jGCaMP7 sensors provide improved detection of\n",
      "individual spikes (jGCaMP7s,f), imaging in neurites and neuropil (jGCaMP7b), and may allow tracking\n",
      "larger populations of neurons using two-photon (jGCaMP7s,f) or wide-field (jGCaMP7c) imaging.\n",
      "['calcium imaging',\n",
      " 'genetically encoded calcium indicators (GECIs)',\n",
      " 'two-photon imaging',\n",
      " 'wide-field imaging']\n",
      "Elapsed time: 11.50 seconds\n",
      "-------------------\n",
      "31209383\n",
      "Direct wavefront sensing enables functional imaging of infragranular axons and spines.\n",
      "We advance two-photon microscopy for near-diffraction-limited imaging up to 850 µm below the pia in\n",
      "awake mice. Our approach combines direct wavefront sensing of light from a guidestar (formed by\n",
      "descanned fluorescence from Cy5.5-conjugated dextran in brain microvessels) with adaptive optics to\n",
      "compensate for tissue-induced aberrations in the wavefront. We achieve high signal-to-noise ratios\n",
      "in recordings of glutamate release from thalamocortical axons and calcium transients in spines of\n",
      "layer 5b basal dendrites during active tactile sensing.\n",
      "['two-photon microscopy',\n",
      " 'direct wavefront sensing',\n",
      " 'adaptive optics',\n",
      " 'near-diffraction-limited imaging']\n",
      "Elapsed time: 8.77 seconds\n",
      "-------------------\n",
      "31263253\n",
      "Quantitative analysis of super-resolved structures using ASAP.\n",
      "Super-resolution microscopy allows imaging of cellular structures with high throughput and detail.\n",
      "However, the efficient and quantitative analysis of images generated is challenging with existing\n",
      "tools. Here, we develop ASAP (automated structures analysis program) to enable rapid and automated\n",
      "detection, classification and quantification of super-resolved structures. We validate ASAP on\n",
      "ground truth data and demonstrate its broad applicability by analyzing images of nucleoporins, TORC1\n",
      "complexes, endocytic vesicles and Bax pores.\n",
      "['super-resolution microscopy', 'quantitative analysis']\n",
      "Elapsed time: 6.70 seconds\n",
      "-------------------\n",
      "31285623\n",
      "Applications, promises, and pitfalls of deep learning for fluorescence image reconstruction.\n",
      "Deep learning is becoming an increasingly important tool for image reconstruction in fluorescence\n",
      "microscopy. We review state-of-the-art applications such as image restoration and super-resolution\n",
      "imaging, and discuss how the latest deep learning research could be applied to other image\n",
      "reconstruction tasks. Despite its successes, deep learning also poses substantial challenges and has\n",
      "limits. We discuss key questions, including how to obtain training data, whether discovery of\n",
      "unknown structures is possible, and the danger of inferring unsubstantiated image details.\n",
      "['image restoration', 'super-resolution imaging']\n",
      "Elapsed time: 6.67 seconds\n",
      "-------------------\n",
      "31285624\n",
      "Mechanistic investigation of mEos4b reveals a strategy to reduce track interruptions in sptPALM.\n",
      "Green-to-red photoconvertible fluorescent proteins repeatedly enter dark states, causing interrupted\n",
      "tracks in single-particle-tracking localization microscopy (sptPALM). We identified a long-lived\n",
      "dark state in photoconverted mEos4b that results from isomerization of the chromophore and\n",
      "efficiently absorbs cyan light. Addition of weak 488-nm light swiftly reverts this dark state to the\n",
      "fluorescent state. This strategy largely eliminates slow blinking and enables the recording of\n",
      "longer tracks in sptPALM with minimum effort.\n",
      "['sptPALM',\n",
      " 'single-particle-tracking localization microscopy',\n",
      " 'green-to-red photoconvertible fluorescent proteins',\n",
      " 'dark states',\n",
      " 'chromophore isomerization',\n",
      " '488-nm light']\n",
      "Elapsed time: 9.14 seconds\n",
      "-------------------\n",
      "31308547\n",
      "A genetically encoded fluorescent sensor for in vivo imaging of GABA.\n",
      "Current techniques for monitoring GABA (γ-aminobutyric acid), the primary inhibitory\n",
      "neurotransmitter in vertebrates, cannot follow transients in intact neural circuits. To develop a\n",
      "GABA sensor, we applied the design principles used to create the fluorescent glutamate receptor\n",
      "iGluSnFR. We used a protein derived from a previously unsequenced Pseudomonas fluorescens strain and\n",
      "performed structure-guided mutagenesis and library screening to obtain intensity-based GABA sensing\n",
      "fluorescence reporter (iGABASnFR) variants. iGABASnFR is genetically encoded, detects GABA release\n",
      "evoked by electric stimulation of afferent fibers in acute brain slices and produces readily\n",
      "detectable fluorescence increases in vivo in mice and zebrafish. We applied iGABASnFR to track\n",
      "mitochondrial GABA content and its modulation by an anticonvulsant, swimming-evoked, GABA-mediated\n",
      "transmission in zebrafish cerebellum, GABA release events during interictal spikes and seizures in\n",
      "awake mice, and found that GABA-mediated tone decreases during isoflurane anesthesia.\n",
      "['fluorescence reporter',\n",
      " 'electric stimulation',\n",
      " 'acute brain slices',\n",
      " 'in vivo',\n",
      " 'mice',\n",
      " 'zebrafish']\n",
      "Elapsed time: 12.35 seconds\n",
      "-------------------\n",
      "31308549\n",
      "High-affinity free ubiquitin sensors for quantifying ubiquitin homeostasis and deubiquitination.\n",
      "Ubiquitin (Ub) conjugation is an essential post-translational modification that affects nearly all\n",
      "proteins in eukaryotes. The functions and mechanisms of ubiquitination are areas of extensive study,\n",
      "and yet the dynamics and regulation of even free (that is, unconjugated) Ub are poorly understood. A\n",
      "major impediment has been the lack of simple and robust techniques to quantify Ub levels in cells\n",
      "and to monitor Ub release from conjugates. Here, we describe avidity-based fluorescent sensors that\n",
      "address this need. The sensors bind specifically to free Ub, have dissociation constant\n",
      "K<sub>d</sub> values down to 60 pM and, together with a newly developed workflow, allow us to\n",
      "distinguish and quantify the pools of free, protein-conjugated and thioesterified forms of Ub from\n",
      "cell lysates. Alternatively, free Ub in fixed cells can be visualized microscopically by staining\n",
      "with a sensor. Real-time assays using the sensors afford unprecedented flexibility and precision to\n",
      "measure deubiquitination of virtually any (poly)Ub conjugate.\n",
      "[['fluorescent sensors'], ['staining'], ['microscopy']]\n",
      "Elapsed time: 10.84 seconds\n",
      "-------------------\n",
      "31358979\n",
      "Protein secondary structure detection in intermediate-resolution cryo-EM maps using deep learning.\n",
      "Although structures determined at near-atomic resolution are now routinely reported by cryo-electron\n",
      "microscopy (cryo-EM), many density maps are determined at an intermediate resolution, and extracting\n",
      "structure information from these maps is still a challenge. We report a computational method,\n",
      "Emap2sec, that identifies the secondary structures of proteins (α-helices, β-sheets and other\n",
      "structures) in EM maps at resolutions of between 5 and 10 Å. Emap2sec uses a three-dimensional deep\n",
      "convolutional neural network to assign secondary structure to each grid point in an EM map. We\n",
      "tested Emap2sec on EM maps simulated from 34 structures at resolutions of 6.0 and 10.0 Å, as well as\n",
      "on 43 maps determined experimentally at resolutions of between 5.0 and 9.5 Å. Emap2sec was able to\n",
      "clearly identify the secondary structures in many maps tested, and showed substantially better\n",
      "performance than existing methods.\n",
      "['deep learning',\n",
      " 'cryo-electron microscopy (cryo-EM)',\n",
      " 'three-dimensional deep convolutional neural network']\n",
      "Elapsed time: 10.21 seconds\n",
      "-------------------\n",
      "31363221\n",
      "Gene expression atlas of a developing tissue by single cell expression correlation analysis.\n",
      "The Drosophila wing disc has been a fundamental model system for the discovery of key signaling\n",
      "pathways and for our understanding of developmental processes. However, a complete map of gene\n",
      "expression in this tissue is lacking. To obtain a gene expression atlas in the wing disc, we\n",
      "employed single cell RNA sequencing (scRNA-seq) and developed a method for analyzing scRNA-seq data\n",
      "based on gene expression correlations rather than cell mapping. This enables us to compute\n",
      "expression maps for all detected genes in the wing disc and to discover 824 genes with spatially\n",
      "restricted expression patterns. This approach identifies clusters of genes with similar expression\n",
      "patterns and functional relevance. As proof of concept, we characterize the previously unstudied\n",
      "gene CG5151 and show that it regulates Wnt signaling. Our method will enable the leveraging of\n",
      "scRNA-seq data for generating expression atlases of undifferentiated tissues during development.\n",
      "['scRNA-seq', 'gene expression correlations', 'cell mapping']\n",
      "Elapsed time: 9.56 seconds\n",
      "-------------------\n",
      "31363222\n",
      "Kilohertz frame-rate two-photon tomography.\n",
      "Point-scanning two-photon microscopy enables high-resolution imaging within scattering specimens\n",
      "such as the mammalian brain, but sequential acquisition of voxels fundamentally limits its speed. We\n",
      "developed a two-photon imaging technique that scans lines of excitation across a focal plane at\n",
      "multiple angles and computationally recovers high-resolution images, attaining voxel rates of over 1\n",
      "billion Hz in structured samples. Using a static image as a prior for recording neural activity, we\n",
      "imaged visually evoked and spontaneous glutamate release across hundreds of dendritic spines in mice\n",
      "at depths over 250 µm and frame rates over 1 kHz. Dendritic glutamate transients in anesthetized\n",
      "mice are synchronized within spatially contiguous domains spanning tens of micrometers at\n",
      "frequencies ranging from 1-100 Hz. We demonstrate millisecond-resolved recordings of acetylcholine\n",
      "and voltage indicators, three-dimensional single-particle tracking and imaging in densely labeled\n",
      "cortex. Our method surpasses limits on the speed of raster-scanned imaging imposed by fluorescence\n",
      "lifetime.\n",
      "['two-photon microscopy',\n",
      " 'point-scanning two-photon microscopy',\n",
      " 'two-photon imaging',\n",
      " 'raster-scanned imaging']\n",
      "Elapsed time: 11.71 seconds\n",
      "-------------------\n",
      "31363225\n",
      "Systems NMR: single-sample quantification of RNA, proteins and metabolites for biomolecular network\n",
      "analysis.\n",
      "Cellular behavior is controlled by the interplay of diverse biomolecules. Most experimental methods,\n",
      "however, can only monitor a single molecule class or reaction type at a time. We developed an in\n",
      "vitro nuclear magnetic resonance spectroscopy (NMR) approach, which permitted dynamic quantification\n",
      "of an entire 'heterotypic' network-simultaneously monitoring three distinct molecule classes\n",
      "(metabolites, proteins and RNA) and all elementary reaction types (bimolecular interactions,\n",
      "catalysis, unimolecular changes). Focusing on an eight-reaction co-transcriptional RNA folding\n",
      "network, in a single sample we recorded over 35 time points with over 170 observables each, and\n",
      "accurately determined five core reaction constants in multiplex. This reconstruction revealed\n",
      "unexpected cross-talk between the different reactions. We further observed dynamic phase-separation\n",
      "in a system of five distinct RNA-binding domains in the course of the RNA transcription reaction.\n",
      "Our Systems NMR approach provides a deeper understanding of biological network dynamics by combining\n",
      "the dynamic resolution of biochemical assays and the multiplexing ability of 'omics'.\n",
      "[{'name': 'Nuclear magnetic resonance spectroscopy'},\n",
      " {'name': 'In vitro NMR'},\n",
      " {'name': 'Single-sample quantification'}]\n",
      "Elapsed time: 12.25 seconds\n",
      "-------------------\n",
      "31384043\n",
      "High throughput discovery of functional protein modifications by Hotspot Thermal Profiling.\n",
      "Mass spectrometry enables global analysis of posttranslationally modified proteoforms from\n",
      "biological samples, yet we still lack methods to systematically predict, or even prioritize, which\n",
      "modification sites may perturb protein function. Here we describe a proteomic method, Hotspot\n",
      "Thermal Profiling, to detect the effects of site-specific protein phosphorylation on the thermal\n",
      "stability of thousands of native proteins in live cells. This massively parallel biophysical assay\n",
      "unveiled shifts in overall protein stability in response to site-specific phosphorylation sites, as\n",
      "well as trends related to protein function and structure. This method can detect intrinsic changes\n",
      "to protein structure as well as extrinsic changes to protein-protein and protein-metabolite\n",
      "interactions resulting from phosphorylation. Finally, we show that functional 'hotspot' protein\n",
      "modification sites can be discovered and prioritized for study in a high-throughput and unbiased\n",
      "fashion. This approach is applicable to diverse organisms, cell types and posttranslational\n",
      "modifications.\n",
      "['mass spectrometry', 'thermal profiling', 'hotspot thermal profiling']\n",
      "Elapsed time: 10.04 seconds\n",
      "-------------------\n",
      "31384047\n",
      "BigStitcher: reconstructing high-resolution image datasets of cleared and expanded samples.\n",
      "Light-sheet imaging of cleared and expanded samples creates terabyte-sized datasets that consist of\n",
      "many unaligned three-dimensional image tiles, which must be reconstructed before analysis. We\n",
      "developed the BigStitcher software to address this challenge. BigStitcher enables interactive\n",
      "visualization, fast and precise alignment, spatially resolved quality estimation, real-time fusion\n",
      "and deconvolution of dual-illumination, multitile, multiview datasets. The software also compensates\n",
      "for optical effects, thereby improving accuracy and enabling subsequent biological analysis.\n",
      "['Light-sheet imaging',\n",
      " 'Multitile imaging',\n",
      " 'Multiview imaging',\n",
      " 'Dual-illumination imaging',\n",
      " 'Real-time fusion and deconvolution',\n",
      " 'Optical effects compensation']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "31427757\n",
      "Oblique-plane single-molecule localization microscopy for tissues and small intact animals.\n",
      "Single-molecule localization microscopy (SMLM), while well established for cultured cells, is not\n",
      "yet fully compatible with tissue-scale samples. We introduce single-molecule oblique-plane\n",
      "microscopy (obSTORM), which by directly imaging oblique sections of samples with oblique light-sheet\n",
      "illumination offers a deep and volumetric SMLM platform that is convenient for standard tissue\n",
      "samples and small intact animals. We demonstrate super-resolution imaging at depths of up to 66 µm\n",
      "for cells, Caenorhabditis elegans gonads, Drosophila melanogaster larval brain, mouse retina and\n",
      "brain sections, and whole stickleback fish.\n",
      "['oblique-plane microscopy',\n",
      " 'single-molecule localization microscopy',\n",
      " 'SMLM',\n",
      " 'super-resolution imaging']\n",
      "Elapsed time: 8.89 seconds\n",
      "-------------------\n",
      "31451764\n",
      "Transmission-mode MALDI-2 mass spectrometry imaging of cells and tissues at subcellular resolution.\n",
      "Matrix-assisted laser desorption-ionization mass spectrometry imaging in transmission-mode geometry\n",
      "(t-MALDI-MSI) can provide molecular information with a pixel size of 1 µm and smaller, which makes\n",
      "this label-free method highly interesting for characterizing the chemical composition of tissues and\n",
      "cells on a (sub)cellular level. However, a major hindrance for wider use of the technology is the\n",
      "reduced ion abundance at small pixel sizes. Here we mitigate this problem by use of laser-induced\n",
      "post-ionization (MALDI-2) and by adapting a t-MALDI-2 ion source to an Orbitrap mass analyzer. We\n",
      "demonstrate the crucial sensitivity and accuracy boosts that are achieved with this combination by\n",
      "visualizing the distribution of numerous phospho- and glycolipids in mouse cerebellum and kidney\n",
      "slices, and in cultured Vero B4 cells. With brain tissue, a pixel size of 600 nm was achieved. Our\n",
      "method could constitute a valuable new tool for research in cell biology and biomedicine.\n",
      "['t-MALDI-MSI', 'laser-induced post-ionization', 'Orbitrap mass analyzer']\n",
      "Elapsed time: 11.27 seconds\n",
      "-------------------\n",
      "31451766\n",
      "Parameter-free image resolution estimation based on decorrelation analysis.\n",
      "Super-resolution microscopy opened diverse new avenues of research by overcoming the resolution\n",
      "limit imposed by diffraction. Exploitation of the fluorescent emission of individual fluorophores\n",
      "made it possible to reveal structures beyond the diffraction limit. To accurately determine the\n",
      "resolution achieved during imaging is challenging with existing metrics. Here, we propose a method\n",
      "for assessing the resolution of individual super-resolved images based on image partial phase\n",
      "autocorrelation. The algorithm is model-free and does not require any user-defined parameters. We\n",
      "demonstrate its performance on a wide variety of imaging modalities, including diffraction-limited\n",
      "techniques. Finally, we show how our method can be used to optimize image acquisition and post-\n",
      "processing in super-resolution microscopy.\n",
      "['diffraction-limited techniques',\n",
      " 'super-resolution microscopy',\n",
      " 'parameter-free image resolution estimation']\n",
      "Elapsed time: 8.48 seconds\n",
      "-------------------\n",
      "31471614\n",
      "Live imaging of mRNA using RNA-stabilized fluorogenic proteins.\n",
      "Fluorogenic RNA aptamers bind and activate the fluorescence of otherwise nonfluorescent dyes.\n",
      "However, fluorogenic aptamers are limited by the small number of fluorogenic dyes suitable for use\n",
      "in live cells. In this communication, fluorogenic proteins whose fluorescence is activated by RNA\n",
      "aptamers are described. Fluorogenic proteins are highly unstable until they bind RNA aptamers\n",
      "inserted into messenger RNAs, resulting in fluorescent RNA-protein complexes that enable live\n",
      "imaging of mRNA in living cells.\n",
      "['fluorogenic proteins',\n",
      " 'live imaging of mRNA',\n",
      " 'RNA-stabilized fluorogenic proteins']\n",
      "Elapsed time: 8.02 seconds\n",
      "-------------------\n",
      "31471616\n",
      "FRET as a biomolecular research tool - understanding its potential while avoiding pitfalls.\n",
      "The applications of Förster resonance energy transfer (FRET) grow with each year. However, different\n",
      "FRET techniques are not applied consistently, nor are results uniformly presented, which makes\n",
      "implementing and reproducing FRET experiments challenging. We discuss important considerations for\n",
      "designing and evaluating ensemble FRET experiments. Alongside a primer on FRET basics, we provide\n",
      "guidelines for making experimental design choices such as the donor-acceptor pair, instrumentation\n",
      "and labeling chemistries; selecting control experiments to unambiguously demonstrate FRET and\n",
      "validate that the experiments provide meaningful data about the biomolecular process in question;\n",
      "analyzing raw data and assessing the results; and reporting data and experimental details in a\n",
      "manner that easily allows for reproducibility. Some considerations are also given for FRET assays\n",
      "and FRET imaging, especially with fluorescent proteins. Our goal is to motivate and empower all\n",
      "biologists to consider FRET for the powerful research tool it can be.\n",
      "['FRET',\n",
      " 'fluorescent proteins',\n",
      " 'ensemble FRET experiments',\n",
      " 'instrumentation',\n",
      " 'labeling chemistries']\n",
      "Elapsed time: 10.57 seconds\n",
      "-------------------\n",
      "31471618\n",
      "Biological imaging of chemical bonds by stimulated Raman scattering microscopy.\n",
      "All molecules consist of chemical bonds, and much can be learned from mapping the spatiotemporal\n",
      "dynamics of these bonds. Since its invention a decade ago, stimulated Raman scattering (SRS)\n",
      "microscopy has become a powerful modality for imaging chemical bonds with high sensitivity,\n",
      "resolution, speed and specificity. We introduce the fundamentals of SRS microscopy and review\n",
      "innovations in SRS microscopes and imaging probes. We highlight examples of exciting biological\n",
      "applications, and share our vision for potential future breakthroughs for this technology.\n",
      "['stimulated Raman scattering microscopy']\n",
      "Elapsed time: 8.46 seconds\n",
      "-------------------\n",
      "31501547\n",
      "High-definition spatial transcriptomics for in situ tissue profiling.\n",
      "Spatial and molecular characteristics determine tissue function, yet high-resolution methods to\n",
      "capture both concurrently are lacking. Here, we developed high-definition spatial transcriptomics,\n",
      "which captures RNA from histological tissue sections on a dense, spatially barcoded bead array. Each\n",
      "experiment recovers several hundred thousand transcript-coupled spatial barcodes at 2-μm resolution,\n",
      "as demonstrated in mouse brain and primary breast cancer. This opens the way to high-resolution\n",
      "spatial analysis of cells and tissues.\n",
      "['spatial transcriptomics',\n",
      " 'histological tissue sections',\n",
      " 'dense, spatially barcoded bead array',\n",
      " 'high-resolution methods']\n",
      "Elapsed time: 7.85 seconds\n",
      "-------------------\n",
      "31501548\n",
      "Robust and automated detection of subcellular morphological motifs in 3D microscopy images.\n",
      "Rapid developments in live-cell three-dimensional (3D) microscopy enable imaging of cell morphology\n",
      "and signaling with unprecedented detail. However, tools to systematically measure and visualize the\n",
      "intricate relationships between intracellular signaling, cytoskeletal organization and downstream\n",
      "cell morphological outputs do not exist. Here, we introduce u-shape3D, a computer graphics and\n",
      "machine-learning pipeline to probe molecular mechanisms underlying 3D cell morphogenesis and to test\n",
      "the intriguing possibility that morphogenesis itself affects intracellular signaling. We demonstrate\n",
      "a generic morphological motif detector that automatically finds lamellipodia, filopodia, blebs and\n",
      "other motifs. Combining motif detection with molecular localization, we measure the differential\n",
      "association of PIP<sub>2</sub> and Kras<sup>V12</sup> with blebs. Both signals associate with bleb\n",
      "edges, as expected for membrane-localized proteins, but only PIP<sub>2</sub> is enhanced on blebs.\n",
      "This indicates that subcellular signaling processes are differentially modulated by local\n",
      "morphological motifs. Overall, our computational workflow enables the objective, 3D analysis of the\n",
      "coupling of cell shape and signaling.\n",
      "['lamellipodia', 'filopodia', 'blebs']\n",
      "Elapsed time: 13.34 seconds\n",
      "-------------------\n",
      "31501551\n",
      "Molecular resolution imaging by repetitive optical selective exposure.\n",
      "We introduce an interferometric single-molecule localization method for super-resolution\n",
      "fluorescence microscopy. Fluorescence molecules are located by the intensities of multiple\n",
      "excitation patterns of an interference fringe, providing around a twofold improvement in the\n",
      "localization precision compared with the conventional imaging with the same photon budget. We\n",
      "demonstrate this technique by resolving nanostructures down to 5 nm in size over a large\n",
      "25 × 25 μm<sup>2</sup> field of view.\n",
      "['interferometric single-molecule localization',\n",
      " 'super-resolution fluorescence microscopy',\n",
      " 'single-molecule localization method']\n",
      "Elapsed time: 8.15 seconds\n",
      "-------------------\n",
      "31501552\n",
      "Publisher Correction: Oblique-plane single-molecule localization microscopy for tissues and small\n",
      "intact animals.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['single-molecule localization microscopy',\n",
      " 'tissues',\n",
      " 'small intact animals',\n",
      " 'oblique-plane']\n",
      "Elapsed time: 5.81 seconds\n",
      "-------------------\n",
      "31520070\n",
      "Author Correction: Resolution upgrades for light-sheet microscopy.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['light-sheet microscopy', 'resolution upgrades']\n",
      "Elapsed time: 5.03 seconds\n",
      "-------------------\n",
      "31527839\n",
      "The mesoSPIM initiative: open-source light-sheet microscopes for imaging cleared tissue.\n",
      "Light-sheet microscopy is an ideal technique for imaging large cleared samples; however, the\n",
      "community is still lacking instruments capable of producing volumetric images of centimeter-sized\n",
      "cleared samples with near-isotropic resolution within minutes. Here, we introduce the mesoscale\n",
      "selective plane-illumination microscopy initiative, an open-hardware project for building and\n",
      "operating a light-sheet microscope that addresses these challenges and is compatible with any type\n",
      "of cleared or expanded sample ( www.mesospim.org ).\n",
      "['light-sheet microscopy',\n",
      " 'volumetric images',\n",
      " 'centimeter-sized cleared samples',\n",
      " 'near-isotropic resolution',\n",
      " 'selective plane-illumination microscopy']\n",
      "Elapsed time: 7.79 seconds\n",
      "-------------------\n",
      "31548704\n",
      "4D functional ultrasound imaging of whole-brain activity in rodents.\n",
      "We extended the capabilities of functional ultrasound to whole-brain four-dimensional (4D)\n",
      "neuroimaging. Our multiplane-wave transmission scheme on matrix arrays at thousands of frames per\n",
      "second provides volumetric recordings of cerebral blood volume changes at high spatiotemporal\n",
      "resolution. We illustrated the approach in rats while providing multiple sensory stimuli, for 4D\n",
      "functional connectivity and during instantaneous tracking of epileptiform events.\n",
      "['functional ultrasound',\n",
      " 'multiplane-wave transmission scheme',\n",
      " 'matrix arrays',\n",
      " 'volumetric recordings',\n",
      " 'cerebral blood volume changes',\n",
      " 'high spatiotemporal resolution']\n",
      "Elapsed time: 7.74 seconds\n",
      "-------------------\n",
      "31548706\n",
      "Comprehensive mapping of neurotransmitter networks by MALDI-MS imaging.\n",
      "We present a mass spectrometry imaging (MSI) approach for the comprehensive mapping of\n",
      "neurotransmitter networks in specific brain regions. Our fluoromethylpyridinium-based reactive\n",
      "matrices facilitate the covalent charge-tagging of molecules containing phenolic hydroxyl and/or\n",
      "primary or secondary amine groups, including dopaminergic and serotonergic neurotransmitters and\n",
      "their associated metabolites. These matrices improved the matrix-assisted laser\n",
      "desorption/ionization (MALDI)-MSI detection limit toward low-abundance neurotransmitters and\n",
      "facilitated the simultaneous imaging of neurotransmitters in fine structures of the brain at a\n",
      "lateral resolution of 10 µm. We demonstrate strategies for the identification of unknown molecular\n",
      "species using the innate chemoselectivity of the reactive matrices and the unique isotopic pattern\n",
      "of a brominated reactive matrix. We illustrate the capabilities of the developed method on\n",
      "Parkinsonian brain samples from human post-mortem tissue and animal models. The direct imaging of\n",
      "neurotransmitter systems provides a method for exploring how various neurological diseases affect\n",
      "specific brain regions through neurotransmitter modulation.\n",
      "['MALDI-MSI',\n",
      " 'mass spectrometry imaging',\n",
      " 'fluoromethylpyridinium-based reactive matrices',\n",
      " 'matrix-assisted laser desorption/ionization']\n",
      "Elapsed time: 12.24 seconds\n",
      "-------------------\n",
      "31548707\n",
      "Brillouin microscopy: an emerging tool for mechanobiology.\n",
      "The role and importance of mechanical properties of cells and tissues in cellular function,\n",
      "development and disease has widely been acknowledged, however standard techniques currently used to\n",
      "assess them exhibit intrinsic limitations. Recently, Brillouin microscopy, a type of optical\n",
      "elastography, has emerged as a non-destructive, label- and contact-free method that can probe the\n",
      "viscoelastic properties of biological samples with diffraction-limited resolution in 3D. This led to\n",
      "increased attention amongst the biological and medical research communities, but it also sparked\n",
      "debates about the interpretation and relevance of the measured physical quantities. Here, we review\n",
      "this emerging technology by describing the underlying biophysical principles and discussing the\n",
      "interpretation of Brillouin spectra arising from heterogeneous biological matter. We further\n",
      "elaborate on the technique's limitations, as well as its potential for gaining insights in biology,\n",
      "in order to guide interested researchers from various fields.\n",
      "['Brillouin microscopy', 'optical elastography']\n",
      "Elapsed time: 9.68 seconds\n",
      "-------------------\n",
      "31548708\n",
      "DART-seq: an antibody-free method for global m<sup>6</sup>A detection.\n",
      "N<sup>6</sup>-methyladenosine (m<sup>6</sup>A) is a widespread RNA modification that influences\n",
      "nearly every aspect of the messenger RNA lifecycle. Our understanding of m<sup>6</sup>A has been\n",
      "facilitated by the development of global m<sup>6</sup>A mapping methods, which use antibodies to\n",
      "immunoprecipitate methylated RNA. However, these methods have several limitations, including high\n",
      "input RNA requirements and cross-reactivity to other RNA modifications. Here, we present DART-seq\n",
      "(deamination adjacent to RNA modification targets), an antibody-free method for detecting\n",
      "m<sup>6</sup>A sites. In DART-seq, the cytidine deaminase APOBEC1 is fused to the\n",
      "m<sup>6</sup>A-binding YTH domain. APOBEC1-YTH expression in cells induces C-to-U deamination at\n",
      "sites adjacent to m<sup>6</sup>A residues, which are detected using standard RNA-seq. DART-seq\n",
      "identifies thousands of m<sup>6</sup>A sites in cells from as little as 10 ng of total RNA and can\n",
      "detect m<sup>6</sup>A accumulation in cells over time. Additionally, we use long-read DART-seq to\n",
      "gain insights into m<sup>6</sup>A distribution along the length of individual transcripts.\n",
      "{'imagingTechniques': ['RNA-seq', 'C-to-U deamination', 'long-read DART-seq']}\n",
      "Elapsed time: 18.99 seconds\n",
      "-------------------\n",
      "31562475\n",
      "Laser phase plate for transmission electron microscopy.\n",
      "Transmission electron microscopy (TEM) of rapidly frozen biological specimens, or cryo-EM, would\n",
      "benefit from the development of a phase plate for in-focus phase contrast imaging. Several types of\n",
      "phase plates have been investigated, but rapid electrostatic charging of all such devices has\n",
      "hindered these efforts. Here, we demonstrate electron phase manipulation with a high-intensity\n",
      "continuous-wave laser beam, and use it as a phase plate for TEM. We demonstrate the laser phase\n",
      "plate by imaging an amorphous carbon film. The laser phase plate provides a stable and tunable phase\n",
      "shift without electrostatic charging or unwanted electron scattering. These results suggest the\n",
      "possibility for dose-efficient imaging of unstained biological macromolecules and cells.\n",
      "['laser phase plate',\n",
      " 'transmission electron microscopy (TEM)',\n",
      " 'cryo-EM',\n",
      " 'in-focus phase contrast imaging']\n",
      "Elapsed time: 8.88 seconds\n",
      "-------------------\n",
      "31562488\n",
      "Nuclear pores as versatile reference standards for quantitative superresolution microscopy.\n",
      "Quantitative fluorescence and superresolution microscopy are often limited by insufficient data\n",
      "quality or artifacts. In this context, it is essential to have biologically relevant control samples\n",
      "to benchmark and optimize the quality of microscopes, labels and imaging conditions. Here, we\n",
      "exploit the stereotypic arrangement of proteins in the nuclear pore complex as in situ reference\n",
      "structures to characterize the performance of a variety of microscopy modalities. We created four\n",
      "genome edited cell lines in which we endogenously labeled the nucleoporin Nup96 with mEGFP, SNAP-\n",
      "tag, HaloTag or the photoconvertible fluorescent protein mMaple. We demonstrate their use (1) as\n",
      "three-dimensional resolution standards for calibration and quality control, (2) to quantify absolute\n",
      "labeling efficiencies and (3) as precise reference standards for molecular counting. These cell\n",
      "lines will enable the broader community to assess the quality of their microscopes and labels, and\n",
      "to perform quantitative, absolute measurements.\n",
      "['fluorescence microscopy',\n",
      " 'superresolution microscopy',\n",
      " 'quantitative microscopy',\n",
      " 'calibration and quality control']\n",
      "Elapsed time: 10.01 seconds\n",
      "-------------------\n",
      "31562489\n",
      "Real-time volumetric microscopy of in vivo dynamics and large-scale samples with SCAPE 2.0.\n",
      "The limited per-pixel bandwidth of most microscopy methods requires compromises between field of\n",
      "view, sampling density and imaging speed. This limitation constrains studies involving complex\n",
      "motion or fast cellular signaling, and presents a major bottleneck for high-throughput structural\n",
      "imaging. Here, we combine high-speed intensified camera technology with a versatile, reconfigurable\n",
      "and dramatically improved Swept, Confocally Aligned Planar Excitation (SCAPE) microscope design that\n",
      "can achieve high-resolution volumetric imaging at over 300 volumes per second and over 1.2 GHz pixel\n",
      "rates. We demonstrate near-isotropic sampling in freely moving Caenorhabditis elegans, and analyze\n",
      "real-time blood flow and calcium dynamics in the beating zebrafish heart. The same system also\n",
      "permits high-throughput structural imaging of mounted, intact, cleared and expanded samples. SCAPE\n",
      "2.0's significantly lower photodamage compared to point-scanning techniques is also confirmed. Our\n",
      "results demonstrate that SCAPE 2.0 is a powerful, yet accessible imaging platform for myriad\n",
      "emerging high-speed dynamic and high-throughput volumetric microscopy applications.\n",
      "['volumetric microscopy',\n",
      " 'intensified camera technology',\n",
      " 'SCAPE microscope design',\n",
      " 'high-resolution volumetric imaging',\n",
      " 'near-isotropic sampling',\n",
      " 'real-time blood flow dynamics',\n",
      " 'calcium dynamics',\n",
      " 'mounted, intact, cleared and expanded samples']\n",
      "Elapsed time: 13.60 seconds\n",
      "-------------------\n",
      "31570887\n",
      "ilastik: interactive machine learning for (bio)image analysis.\n",
      "We present ilastik, an easy-to-use interactive tool that brings machine-learning-based (bio)image\n",
      "analysis to end users without substantial computational expertise. It contains pre-defined workflows\n",
      "for image segmentation, object classification, counting and tracking. Users adapt the workflows to\n",
      "the problem at hand by interactively providing sparse training annotations for a nonlinear\n",
      "classifier. ilastik can process data in up to five dimensions (3D, time and number of channels). Its\n",
      "computational back end runs operations on-demand wherever possible, allowing for interactive\n",
      "prediction on data larger than RAM. Once the classifiers are trained, ilastik workflows can be\n",
      "applied to new data from the command line without further user interaction. We describe all ilastik\n",
      "workflows in detail, including three case studies and a discussion on the expected performance.\n",
      "['image segmentation', 'object classification', 'counting', 'tracking']\n",
      "Elapsed time: 8.34 seconds\n",
      "-------------------\n",
      "31591575\n",
      "Real-time cryo-electron microscopy data preprocessing with Warp.\n",
      "The acquisition of cryo-electron microscopy (cryo-EM) data from biological specimens must be tightly\n",
      "coupled to data preprocessing to ensure the best data quality and microscope usage. Here we describe\n",
      "Warp, a software that automates all preprocessing steps of cryo-EM data acquisition and enables\n",
      "real-time evaluation. Warp corrects micrographs for global and local motion, estimates the local\n",
      "defocus and monitors key parameters for each recorded micrograph or tomographic tilt series in real\n",
      "time. The software further includes deep-learning-based models for accurate particle picking and\n",
      "image denoising. The output from Warp can be fed into established programs for particle\n",
      "classification and 3D-map refinement. Our benchmarks show improvement in the nominal resolution,\n",
      "which went from 3.9 Å to 3.2 Å, of a published cryo-EM data set for influenza virus hemagglutinin.\n",
      "Warp is easy to install from http://github.com/cramerlab/warp and computationally inexpensive, and\n",
      "has an intuitive, streamlined user interface.\n",
      "['cryo-electron microscopy',\n",
      " 'real-time cryo-EM data preprocessing',\n",
      " 'particle picking',\n",
      " 'image denoising']\n",
      "Elapsed time: 10.98 seconds\n",
      "-------------------\n",
      "31591576\n",
      "An order of magnitude faster DNA-PAINT imaging by optimized sequence design and buffer conditions.\n",
      "DNA points accumulation in nanoscale topography (DNA-PAINT) is a relatively easy-to-implement super-\n",
      "resolution technique. However, image acquisition is slow compared to most other approaches. Here, we\n",
      "overcome this limitation by designing optimized DNA sequences and buffer conditions. We demonstrate\n",
      "our approach in vitro with DNA origami and in situ using cell samples, and achieve an order of\n",
      "magnitude faster imaging speeds without compromising image quality or spatial resolution. This\n",
      "improvement now makes DNA-PAINT applicable to high-throughput studies.\n",
      "['DNA-PAINT', 'super-resolution technique']\n",
      "Elapsed time: 6.78 seconds\n",
      "-------------------\n",
      "31591578\n",
      "Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs.\n",
      "Cryo-electron microscopy is a popular method for the determination of protein structures; however,\n",
      "identifying a sufficient number of particles for analysis can take months of manual effort. Current\n",
      "computational approaches find many false positives and require ad hoc postprocessing, especially for\n",
      "unusually shaped particles. To address these shortcomings, we develop Topaz, an efficient and\n",
      "accurate particle-picking pipeline using neural networks trained with a general-purpose positive-\n",
      "unlabeled learning method. This framework enables particle detection models to be trained with few\n",
      "sparsely labeled particles and no labeled negatives. Topaz retrieves many more real particles than\n",
      "conventional picking methods while maintaining low false-positive rates, is capable of picking\n",
      "challenging unusually shaped proteins (for example, small, non-globular and asymmetric particles),\n",
      "produces more representative particle sets and does not require post hoc curation. We demonstrate\n",
      "the performance of Topaz on two difficult datasets and three conventional datasets. Topaz is\n",
      "modular, standalone, free and open source ( http://topaz.csail.mit.edu ).\n",
      "['cryo-electron microscopy',\n",
      " 'convolutional neural networks',\n",
      " 'positive-unlabeled learning method']\n",
      "Elapsed time: 11.06 seconds\n",
      "-------------------\n",
      "31591580\n",
      "Engineering of human brain organoids with a functional vascular-like system.\n",
      "Human cortical organoids (hCOs), derived from human embryonic stem cells (hESCs), provide a platform\n",
      "to study human brain development and diseases in complex three-dimensional tissue. However, current\n",
      "hCOs lack microvasculature, resulting in limited oxygen and nutrient delivery to the inner-most\n",
      "parts of hCOs. We engineered hESCs to ectopically express human ETS variant 2 (ETV2).\n",
      "ETV2-expressing cells in hCOs contributed to forming a complex vascular-like network in hCOs.\n",
      "Importantly, the presence of vasculature-like structures resulted in enhanced functional maturation\n",
      "of organoids. We found that vascularized hCOs (vhCOs) acquired several blood-brain barrier\n",
      "characteristics, including an increase in the expression of tight junctions, nutrient transporters\n",
      "and trans-endothelial electrical resistance. Finally, ETV2-induced endothelium supported the\n",
      "formation of perfused blood vessels in vivo. These vhCOs form vasculature-like structures that\n",
      "resemble the vasculature in early prenatal brain, and they present a robust model to study brain\n",
      "disease in vitro.\n",
      "['vascular-like network',\n",
      " 'blood-brain barrier characteristics',\n",
      " 'tight junctions',\n",
      " 'nutrient transporters',\n",
      " 'trans-endothelial electrical resistance',\n",
      " 'perfused blood vessels']\n",
      "Elapsed time: 12.88 seconds\n",
      "-------------------\n",
      "31611690\n",
      "A complete data processing workflow for cryo-ET and subtomogram averaging.\n",
      "Electron cryotomography is currently the only method capable of visualizing cells in three\n",
      "dimensions at nanometer resolutions. While modern instruments produce massive amounts of tomography\n",
      "data containing extremely rich structural information, data processing is very labor intensive and\n",
      "the results are often limited by the skills of the personnel rather than the data. We present an\n",
      "integrated workflow that covers the entire tomography data processing pipeline, from automated tilt\n",
      "series alignment to subnanometer resolution subtomogram averaging. Resolution enhancement is made\n",
      "possible through the use of per-particle per-tilt contrast transfer function correction and\n",
      "alignment. The workflow greatly reduces human bias, increases throughput and more closely approaches\n",
      "data-limited resolution for subtomogram averaging in both purified macromolecules and cells.\n",
      "{'imaging techniques': ['cryo-ET',\n",
      "                        'subtomogram averaging',\n",
      "                        'electron cryotomography']}\n",
      "Elapsed time: 8.82 seconds\n",
      "-------------------\n",
      "31611691\n",
      "Optogenetic activation of intracellular antibodies for direct modulation of endogenous proteins.\n",
      "Intracellular antibodies have become powerful tools for imaging, modulating and neutralizing\n",
      "endogenous target proteins. Here, we describe an optogenetically activated intracellular antibody\n",
      "(optobody) consisting of split antibody fragments and blue-light inducible heterodimerization\n",
      "domains. We expanded this optobody platform by generating several optobodies from previously\n",
      "developed intracellular antibodies, and demonstrated that photoactivation of gelsolin and\n",
      "β2-adrenergic receptor (β2AR) optobodies suppressed endogenous gelsolin activity and β2AR signaling,\n",
      "respectively.\n",
      "{'imaging techniques': ['optogenetic activation',\n",
      "                        'intracellular antibodies',\n",
      "                        'split antibody fragments',\n",
      "                        'blue-light inducible heterodimerization domains']}\n",
      "Elapsed time: 8.67 seconds\n",
      "-------------------\n",
      "31611693\n",
      "mEosEM withstands osmium staining and Epon embedding for super-resolution CLEM.\n",
      "Super-resolution correlative light and electron microscopy (SR-CLEM) is a powerful approach for\n",
      "imaging specific molecules at the nanoscale in the context of the cellular ultrastructure. Epon\n",
      "epoxy resin embedding offers advantages for SR-CLEM, including ultrastructural preservation and high\n",
      "quality sectioning. However, Epon embedding eliminates fluorescence from most fluorescent proteins.\n",
      "We describe a photocontrollable fluorescent protein, mEosEM, that can survive Epon embedding after\n",
      "osmium tetroxide (OsO<sub>4</sub>) treatment for improved SR-CLEM.\n",
      "['Epon embedding',\n",
      " 'correlative light and electron microscopy (SR-CLEM)',\n",
      " 'super-resolution CLEM',\n",
      " 'fluorescent protein',\n",
      " 'photocontrollable fluorescent protein',\n",
      " 'mEosEM']\n",
      "Elapsed time: 9.43 seconds\n",
      "-------------------\n",
      "31611694\n",
      "Machine learning-guided channelrhodopsin engineering enables minimally invasive optogenetics.\n",
      "We engineered light-gated channelrhodopsins (ChRs) whose current strength and light sensitivity\n",
      "enable minimally invasive neuronal circuit interrogation. Current ChR tools applied to the mammalian\n",
      "brain require intracranial surgery for transgene delivery and implantation of fiber-optic cables to\n",
      "produce light-dependent activation of a small volume of tissue. To facilitate expansive optogenetics\n",
      "without the need for invasive implants, our engineering approach leverages the substantial\n",
      "literature of ChR variants to train statistical models for the design of high-performance ChRs. With\n",
      "Gaussian process models trained on a limited experimental set of 102 functionally characterized\n",
      "ChRs, we designed high-photocurrent ChRs with high light sensitivity. Three of these, ChRger1-3,\n",
      "enable optogenetic activation of the nervous system via systemic transgene delivery. ChRger2 enables\n",
      "light-induced neuronal excitation without fiber-optic implantation; that is, this opsin enables\n",
      "transcranial optogenetics.\n",
      "['Gaussian process models',\n",
      " 'systemic transgene delivery',\n",
      " 'transcranial optogenetics']\n",
      "Elapsed time: 10.54 seconds\n",
      "-------------------\n",
      "31636458\n",
      "A super-resolution platform for correlative live single-molecule imaging and STED microscopy.\n",
      "Super-resolution microscopy offers tremendous opportunities to unravel the complex and dynamic\n",
      "architecture of living cells. However, current super-resolution microscopes are well suited for\n",
      "revealing protein distributions or cell morphology, but not both. We present a super-resolution\n",
      "platform that permits correlative single-molecule imaging and stimulated emission depletion\n",
      "microscopy in live cells. It gives nanoscale access to the positions and movements of synaptic\n",
      "proteins within the morphological context of growth cones and dendritic spines.\n",
      "['single-molecule imaging', 'STED microscopy', 'super-resolution microscopy']\n",
      "Elapsed time: 7.88 seconds\n",
      "-------------------\n",
      "31636459\n",
      "Nucleus segmentation across imaging experiments: the 2018 Data Science Bowl.\n",
      "Segmenting the nuclei of cells in microscopy images is often the first step in the quantitative\n",
      "analysis of imaging data for biological and biomedical applications. Many bioimage analysis tools\n",
      "can segment nuclei in images but need to be selected and configured for every experiment. The 2018\n",
      "Data Science Bowl attracted 3,891 teams worldwide to make the first attempt to build a segmentation\n",
      "method that could be applied to any two-dimensional light microscopy image of stained nuclei across\n",
      "experiments, with no human interaction. Top participants in the challenge succeeded in this task,\n",
      "developing deep-learning-based models that identified cell nuclei across many image types and\n",
      "experimental conditions without the need to manually adjust segmentation parameters. This represents\n",
      "an important step toward configuration-free bioimage analysis software tools.\n",
      "['deep-learning-based models', 'microscopy images', 'light microscopy image']\n",
      "Elapsed time: 8.83 seconds\n",
      "-------------------\n",
      "31641286\n",
      "Author Correction: Molecular resolution imaging by repetitive optical selective exposure.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['optical selective exposure', 'molecular resolution imaging']\n",
      "Elapsed time: 4.99 seconds\n",
      "-------------------\n",
      "31653976\n",
      "Publisher Correction: Nuclear pores as versatile reference standards for quantitative\n",
      "superresolution microscopy.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['superresolution microscopy',\n",
      " 'quantitative superresolution microscopy',\n",
      " 'nuclear pores',\n",
      " 'reference standards',\n",
      " 'versatile reference standards']\n",
      "Elapsed time: 6.18 seconds\n",
      "-------------------\n",
      "31659327\n",
      "Kilohertz two-photon brain imaging in awake mice.\n",
      "Two-photon microscopy is a mainstay technique for imaging in scattering media and normally provides\n",
      "frame-acquisition rates of ~10-30 Hz. To track high-speed phenomena, we created a two-photon\n",
      "microscope with 400 illumination beams that collectively sample 95,000-211,000 µm<sup>2</sup> areas\n",
      "at rates up to 1 kHz. Using this microscope, we visualized microcirculatory flow, fast venous\n",
      "constrictions and neuronal Ca<sup>2+</sup> spiking with millisecond-scale timing resolution in the\n",
      "brains of awake mice.\n",
      "['two-photon microscopy',\n",
      " 'microcirculatory flow',\n",
      " 'fast venous constrictions',\n",
      " 'neuronal Ca2+ spiking']\n",
      "Elapsed time: 8.77 seconds\n",
      "-------------------\n",
      "31673159\n",
      "Light-sheet microscopy of cleared tissues with isotropic, subcellular resolution.\n",
      "We present cleared-tissue axially swept light-sheet microscopy (ctASLM), which enables isotropic,\n",
      "subcellular resolution imaging with high optical sectioning capability and a large field of view\n",
      "over a broad range of immersion media. ctASLM can image live, expanded, and both aqueous and non-\n",
      "aqueous chemically cleared tissue preparations. Depending on the optical configuration, ctASLM\n",
      "provides up to 260 nm of axial resolution, a three to tenfold improvement over confocal and other\n",
      "reported cleared-tissue light-sheet microscopes. We imaged millimeter-scale cleared tissues with\n",
      "subcellular three-dimensional resolution, which enabled automated detection of multicellular tissue\n",
      "architectures, individual cells, synaptic spines and rare cell-cell interactions.\n",
      "['confocal microscopy', 'cleared-tissue light-sheet microscopy', 'ctASLM']\n",
      "Elapsed time: 8.65 seconds\n",
      "-------------------\n",
      "31686039\n",
      "Three-dimensional virtual refocusing of fluorescence microscopy images using deep learning.\n",
      "We demonstrate that a deep neural network can be trained to virtually refocus a two-dimensional\n",
      "fluorescence image onto user-defined three-dimensional (3D) surfaces within the sample. Using this\n",
      "method, termed Deep-Z, we imaged the neuronal activity of a Caenorhabditis elegans worm in 3D using\n",
      "a time sequence of fluorescence images acquired at a single focal plane, digitally increasing the\n",
      "depth-of-field by 20-fold without any axial scanning, additional hardware or a trade-off of imaging\n",
      "resolution and speed. Furthermore, we demonstrate that this approach can correct for sample drift,\n",
      "tilt and other aberrations, all digitally performed after the acquisition of a single fluorescence\n",
      "image. This framework also cross-connects different imaging modalities to each other, enabling 3D\n",
      "refocusing of a single wide-field fluorescence image to match confocal microscopy images acquired at\n",
      "different sample planes. Deep-Z has the potential to improve volumetric imaging speed while reducing\n",
      "challenges relating to sample drift, aberration and defocusing that are associated with standard 3D\n",
      "fluorescence microscopy.\n",
      "['deep learning',\n",
      " 'fluorescence microscopy',\n",
      " 'confocal microscopy',\n",
      " 'wide-field fluorescence']\n",
      "Elapsed time: 11.15 seconds\n",
      "-------------------\n",
      "31686040\n",
      "Simultaneous mesoscopic and two-photon imaging of neuronal activity in cortical circuits.\n",
      "Spontaneous and sensory-evoked activity propagates across varying spatial scales in the mammalian\n",
      "cortex, but technical challenges have limited conceptual links between the function of local\n",
      "neuronal circuits and brain-wide network dynamics. We present a method for simultaneous cellular-\n",
      "resolution two-photon calcium imaging of a local microcircuit and mesoscopic widefield calcium\n",
      "imaging of the entire cortical mantle in awake mice. Our multi-scale approach involves a microscope\n",
      "with an orthogonal axis design where the mesoscopic objective is oriented above the brain and the\n",
      "two-photon objective is oriented horizontally, with imaging performed through a microprism. We also\n",
      "introduce a viral transduction method for robust and widespread gene delivery in the mouse brain.\n",
      "These approaches allow us to identify the behavioral state-dependent functional connectivity of\n",
      "pyramidal neurons and vasoactive intestinal peptide-expressing interneurons with long-range cortical\n",
      "networks. Our imaging system provides a powerful strategy for investigating cortical architecture\n",
      "across a wide range of spatial scales.\n",
      "['two-photon calcium imaging',\n",
      " 'cellular-resolution two-photon imaging',\n",
      " 'mesoscopic widefield calcium imaging',\n",
      " 'orthogonal axis design']\n",
      "Elapsed time: 11.45 seconds\n",
      "-------------------\n",
      "31712780\n",
      "Non-reversible tissue fixation retains extracellular vesicles for in situ imaging.\n",
      "Extracellular vesicles (EVs) are secreted nanosized particles with many biological functions and\n",
      "pathological associations. The inability to image EVs in fixed tissues has been a major limitation\n",
      "to understanding their role in healthy and diseased tissue microenvironments. Here, we show that\n",
      "crosslinking mammalian tissues with formaldehyde results in significant EV loss, which can be\n",
      "prevented by additional fixation with 1-ethyl-3-(3-dimethylaminopropyl) carbodiimide (EDC) for\n",
      "visualization of EVs in a range of normal and cancer tissues.\n",
      "['formaldehyde',\n",
      " '1-ethyl-3-(3-dimethylaminopropyl) carbodiimide (EDC)',\n",
      " 'visualization']\n",
      "Elapsed time: 8.77 seconds\n",
      "-------------------\n",
      "31740815\n",
      "Probabilistic cell typing enables fine mapping of closely related cell types in situ.\n",
      "Understanding the function of a tissue requires knowing the spatial organization of its constituent\n",
      "cell types. In the cerebral cortex, single-cell RNA sequencing (scRNA-seq) has revealed the genome-\n",
      "wide expression patterns that define its many, closely related neuronal types, but cannot reveal\n",
      "their spatial arrangement. Here we introduce probabilistic cell typing by in situ sequencing\n",
      "(pciSeq), an approach that leverages previous scRNA-seq classification to identify cell types using\n",
      "multiplexed in situ RNA detection. We applied this method by mapping the inhibitory neurons of mouse\n",
      "hippocampal area CA1, for which ground truth is available from extensive previous work identifying\n",
      "their laminar organization. Our method identified these neuronal classes in a spatial arrangement\n",
      "matching ground truth, and further identified multiple classes of isocortical pyramidal cell in a\n",
      "pattern matching their known organization. This method will allow identifying the spatial\n",
      "organization of closely related cell types across the brain and other tissues.\n",
      "['in situ sequencing',\n",
      " 'multiplexed in situ RNA detection',\n",
      " 'scRNA-seq classification']\n",
      "Elapsed time: 9.28 seconds\n",
      "-------------------\n",
      "31740816\n",
      "Time-resolved serial femtosecond crystallography at the European XFEL.\n",
      "The European XFEL (EuXFEL) is a 3.4-km long X-ray source, which produces femtosecond, ultrabrilliant\n",
      "and spatially coherent X-ray pulses at megahertz (MHz) repetition rates. This X-ray source has been\n",
      "designed to enable the observation of ultrafast processes with near-atomic spatial resolution. Time-\n",
      "resolved crystallographic investigations on biological macromolecules belong to an important class\n",
      "of experiments that explore fundamental and functional structural displacements in these molecules.\n",
      "Due to the unusual MHz X-ray pulse structure at the EuXFEL, these experiments are challenging. Here,\n",
      "we demonstrate how a biological reaction can be followed on ultrafast timescales at the EuXFEL. We\n",
      "investigate the picosecond time range in the photocycle of photoactive yellow protein (PYP) with MHz\n",
      "X-ray pulse rates. We show that difference electron density maps of excellent quality can be\n",
      "obtained. The results connect the previously explored femtosecond PYP dynamics to timescales\n",
      "accessible at synchrotrons. This opens the door to a wide range of time-resolved studies at the\n",
      "EuXFEL.\n",
      "['serial femtosecond crystallography',\n",
      " 'time-resolved crystallographic investigations',\n",
      " 'difference electron density maps']\n",
      "Elapsed time: 11.68 seconds\n",
      "-------------------\n",
      "31740817\n",
      "Time-resolved imaging-based CRISPRi screening.\n",
      "Our ability to connect genotypic variation to biologically important phenotypes has been seriously\n",
      "limited by the gap between live-cell microscopy and library-scale genomic engineering. Here, we show\n",
      "how in situ genotyping of a library of strains after time-lapse imaging in a microfluidic device\n",
      "overcomes this problem. We determine how 235 different CRISPR interference knockdowns impact the\n",
      "coordination of the replication and division cycles of Escherichia coli by monitoring the location\n",
      "of replication forks throughout on average >500 cell cycles per knockdown. Subsequent in situ\n",
      "genotyping allows us to map each phenotype distribution to a specific genetic perturbation to\n",
      "determine which genes are important for cell cycle control. The single-cell time-resolved assay\n",
      "allows us to determine the distribution of single-cell growth rates, cell division sizes and\n",
      "replication initiation volumes. The technology presented in this study enables genome-scale screens\n",
      "of most live-cell microscopy assays.\n",
      "['time-resolved imaging',\n",
      " 'in situ genotyping',\n",
      " 'single-cell time-resolved assay',\n",
      " 'live-cell microscopy']\n",
      "Elapsed time: 9.89 seconds\n",
      "-------------------\n",
      "31740819\n",
      "Fast, sensitive and accurate integration of single-cell data with Harmony.\n",
      "The emerging diversity of single-cell RNA-seq datasets allows for the full transcriptional\n",
      "characterization of cell types across a wide variety of biological and clinical conditions. However,\n",
      "it is challenging to analyze them together, particularly when datasets are assayed with different\n",
      "technologies, because biological and technical differences are interspersed. We present Harmony\n",
      "(https://github.com/immunogenomics/harmony), an algorithm that projects cells into a shared\n",
      "embedding in which cells group by cell type rather than dataset-specific conditions. Harmony\n",
      "simultaneously accounts for multiple experimental and biological factors. In six analyses, we\n",
      "demonstrate the superior performance of Harmony to previously published algorithms while requiring\n",
      "fewer computational resources. Harmony enables the integration of ~10<sup>6</sup> cells on a\n",
      "personal computer. We apply Harmony to peripheral blood mononuclear cells from datasets with large\n",
      "experimental differences, five studies of pancreatic islet cells, mouse embryogenesis datasets and\n",
      "the integration of scRNA-seq with spatial transcriptomics data.\n",
      "['scRNA-seq', 'spatial transcriptomics data']\n",
      "Elapsed time: 10.15 seconds\n",
      "-------------------\n",
      "31740820\n",
      "Gas cluster ion beam SEM for imaging of large tissue samples with 10 nm isotropic resolution.\n",
      "We demonstrate gas cluster ion beam scanning electron microscopy (SEM), in which wide-area ion\n",
      "milling is performed on a series of thick tissue sections. This three-dimensional electron\n",
      "microscopy technique acquires datasets with <10 nm isotropic resolution of each section, and these\n",
      "can then be stitched together to span the sectioned volume. Incorporating gas cluster ion beam SEM\n",
      "into existing single-beam and multibeam SEM workflows should be straightforward, increasing\n",
      "reliability while improving z resolution by a factor of three or more.\n",
      "['scanning electron microscopy',\n",
      " 'ion milling',\n",
      " 'single-beam SEM',\n",
      " 'multibeam SEM']\n",
      "Elapsed time: 7.84 seconds\n",
      "-------------------\n",
      "31740821\n",
      "Tailoring cryo-electron microscopy grids by photo-micropatterning for in-cell structural studies.\n",
      "Spatially controlled cell adhesion on electron microscopy supports remains a bottleneck in specimen\n",
      "preparation for cellular cryo-electron tomography. Here, we describe contactless and mask-free\n",
      "photo-micropatterning of electron microscopy grids for site-specific deposition of extracellular\n",
      "matrix-related proteins. We attained refined cell positioning for micromachining by cryo-focused ion\n",
      "beam milling. Complex micropatterns generated predictable intracellular organization, allowing\n",
      "direct correlation between cell architecture and in-cell three-dimensional structural\n",
      "characterization of the underlying molecular machinery.\n",
      "[['cryo-electron microscopy', 'electron microscopy', 'photo-micropatterning']]\n",
      "Elapsed time: 7.95 seconds\n",
      "-------------------\n",
      "31768058\n",
      "In situ structure determination at nanometer resolution using TYGRESS.\n",
      "The resolution of subtomogram averages calculated from cryo-electron tomograms (cryo-ET) of crowded\n",
      "cellular environments is often limited owing to signal loss in, and misalignment of, the\n",
      "subtomograms. By contrast, single-particle cryo-electron microscopy (SP-cryo-EM) routinely reaches\n",
      "near-atomic resolution of isolated complexes. We report a method called 'tomography-guided 3D\n",
      "reconstruction of subcellular structures' (TYGRESS) that is a hybrid of cryo-ET and SP-cryo-EM, and\n",
      "is able to achieve close-to-nanometer resolution of complexes inside crowded cellular environments.\n",
      "TYGRESS combines the advantages of SP-cryo-EM (images with good signal-to-noise ratio and contrast,\n",
      "as well as minimal radiation damage) and subtomogram averaging (three-dimensional alignment of\n",
      "macromolecules in a complex sample). Using TYGRESS, we determined the structure of the intact\n",
      "ciliary axoneme with up to resolution of 12 Å. These results reveal many structural details that\n",
      "were not visible by cryo-ET alone. TYGRESS is generally applicable to cellular complexes that are\n",
      "amenable to subtomogram averaging.\n",
      "{'imaging techniques': ['cryo-electron tomography (cryo-ET)',\n",
      "                        'single-particle cryo-electron microscopy (SP-cryo-EM)',\n",
      "                        'tomography-guided 3D reconstruction of subcellular '\n",
      "                        'structures (TYGRESS)']}\n",
      "Elapsed time: 13.57 seconds\n",
      "-------------------\n",
      "31768059\n",
      "A CMOS NMR needle for probing brain physiology with high spatial and temporal resolution.\n",
      "Magnetic resonance imaging and spectroscopy are versatile methods for probing brain physiology, but\n",
      "their intrinsically low sensitivity limits the achievable spatial and temporal resolution. Here, we\n",
      "introduce a monolithically integrated NMR-on-a-chip needle that combines an ultra-sensitive 300 µm\n",
      "NMR coil with a complete NMR transceiver, enabling in vivo measurements of blood oxygenation and\n",
      "flow in nanoliter volumes at a sampling rate of 200 Hz.\n",
      "['magnetic resonance imaging', 'spectroscopy', 'NMR-on-a-chip needle']\n",
      "Elapsed time: 7.80 seconds\n",
      "-------------------\n",
      "31768062\n",
      "Isolating live cells after high-throughput, long-term, time-lapse microscopy.\n",
      "Single-cell genetic screens can be incredibly powerful, but current high-throughput platforms do not\n",
      "track dynamic processes, and even for non-dynamic properties they struggle to separate mutants of\n",
      "interest from phenotypic outliers of the wild-type population. Here we introduce SIFT, single-cell\n",
      "isolation following time-lapse imaging, to address these limitations. After imaging and tracking\n",
      "individual bacteria for tens of consecutive generations under tightly controlled growth conditions,\n",
      "cells of interest are isolated and propagated for downstream analysis, free of contamination and\n",
      "without genetic or physiological perturbations. This platform can characterize tens of thousands of\n",
      "cell lineages per day, making it possible to accurately screen complex phenotypes without the need\n",
      "for barcoding or genetic modifications. We applied SIFT to identify a set of ultraprecise synthetic\n",
      "gene oscillators, with circuit variants spanning a 30-fold range of average periods. This revealed\n",
      "novel design principles in synthetic biology and demonstrated the power of SIFT to reliably screen\n",
      "diverse dynamic phenotypes.\n",
      "['time-lapse microscopy', 'single-cell isolation', 'synthetic gene oscillators']\n",
      "Elapsed time: 10.52 seconds\n",
      "-------------------\n",
      "31768063\n",
      "Bottom-up structural proteomics: cryoEM of protein complexes enriched from the cellular milieu.\n",
      "X-ray crystallography often requires non-native constructs involving mutations or truncations, and\n",
      "is challenged by membrane proteins and large multicomponent complexes. We present here a bottom-up\n",
      "endogenous structural proteomics approach whereby near-atomic-resolution cryo electron microscopy\n",
      "(cryoEM) maps are reconstructed ab initio from unidentified protein complexes enriched directly from\n",
      "the endogenous cellular milieu, followed by identification and atomic modeling of the proteins. The\n",
      "proteins in each complex are identified using cryoID, a program we developed to identify proteins in\n",
      "ab initio cryoEM maps. As a proof of principle, we applied this approach to the malaria-causing\n",
      "parasite Plasmodium falciparum, an organism that has resisted conventional structural-biology\n",
      "approaches, to obtain atomic models of multiple protein complexes implicated in intraerythrocytic\n",
      "survival of the parasite. Our approach is broadly applicable for determining structures of\n",
      "undiscovered protein complexes enriched directly from endogenous sources.\n",
      "['cryoEM', 'X-ray crystallography', 'cryoID']\n",
      "Elapsed time: 11.25 seconds\n",
      "-------------------\n",
      "31780840\n",
      "Analysis of the Human Protein Atlas Image Classification competition.\n",
      "Pinpointing subcellular protein localizations from microscopy images is easy to the trained eye, but\n",
      "challenging to automate. Based on the Human Protein Atlas image collection, we held a competition to\n",
      "identify deep learning solutions to solve this task. Challenges included training on highly\n",
      "imbalanced classes and predicting multiple labels per image. Over 3 months, 2,172 teams\n",
      "participated. Despite convergence on popular networks and training techniques, there was\n",
      "considerable variety among the solutions. Participants applied strategies for modifying neural\n",
      "networks and loss functions, augmenting data and using pretrained networks. The winning models far\n",
      "outperformed our previous effort at multi-label classification of protein localization patterns by\n",
      "~20%. These models can be used as classifiers to annotate new images, feature extractors to measure\n",
      "pattern similarity or pretrained networks for a wide range of biological applications.\n",
      "['deep learning solutions',\n",
      " 'multi-label classification',\n",
      " 'protein localization patterns',\n",
      " 'microscopy images']\n",
      "Elapsed time: 8.88 seconds\n",
      "-------------------\n",
      "31792434\n",
      "An adaptive excitation source for high-speed multiphoton microscopy.\n",
      "Optical imaging is important for understanding brain function. However, established methods with\n",
      "high spatiotemporal resolution are limited by the potential for laser damage to living tissues. We\n",
      "describe an adaptive femtosecond excitation source that only illuminates the region of interest,\n",
      "which leads to a 30-fold reduction in the power requirement for two- or three-photon imaging of\n",
      "brain activity in awake mice for improved high-speed longitudinal neuroimaging.\n",
      "['femtosecond excitation source', 'two-photon imaging', 'three-photon imaging']\n",
      "Elapsed time: 7.19 seconds\n",
      "-------------------\n",
      "31802009\n",
      "Publisher Correction: Gas cluster ion beam SEM for imaging of large tissue samples with 10 nm\n",
      "isotropic resolution.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['Gas cluster ion beam SEM',\n",
      " 'large tissue samples',\n",
      " '10 nm isotropic resolution']\n",
      "Elapsed time: 5.96 seconds\n",
      "-------------------\n",
      "31819263\n",
      "Localization microscopy at doubled precision with patterned illumination.\n",
      "MINFLUX offers a breakthrough in single molecule localization precision, but is limited in field of\n",
      "view. Here we combine centroid estimation and illumination pattern induced photon count variations\n",
      "in a conventional widefield imaging setup to extract position information over a typical micrometer-\n",
      "sized field of view. We show a near two-fold improvement in precision over standard localization\n",
      "with the same photon count on DNA-origami nanostructures and tubulin in cells, using DNA-PAINT and\n",
      "STORM imaging.\n",
      "['DNA-PAINT',\n",
      " 'STORM imaging',\n",
      " 'centroid estimation',\n",
      " 'illumination pattern induced photon count variations']\n",
      "Elapsed time: 7.54 seconds\n",
      "-------------------\n",
      "31819267\n",
      "Directed evolution improves the catalytic efficiency of TEV protease.\n",
      "Tobacco etch virus protease (TEV) is one of the most widely used proteases in biotechnology because\n",
      "of its exquisite sequence specificity. A limitation, however, is its slow catalytic rate. We\n",
      "developed a generalizable yeast-based platform for directed evolution of protease catalytic\n",
      "properties. Protease activity is read out via proteolytic release of a membrane-anchored\n",
      "transcription factor, and we temporally regulate access to TEV's cleavage substrate using a\n",
      "photosensory LOV domain. By gradually decreasing light exposure time, we enriched faster variants of\n",
      "TEV over multiple rounds of selection. Our TEV-S153N mutant (uTEV1Δ), when incorporated into the\n",
      "calcium integrator FLARE, improved the signal/background ratio by 27-fold, and enabled recording of\n",
      "neuronal activity in culture with 60-s temporal resolution. Given the widespread use of TEV in\n",
      "biotechnology, both our evolved TEV mutants and the directed-evolution platform used to generate\n",
      "them could be beneficial across a wide range of applications.\n",
      "['fluorescence microscopy',\n",
      " 'photosensory LOV domain',\n",
      " 'calcium integrator FLARE']\n",
      "Elapsed time: 11.17 seconds\n",
      "-------------------\n",
      "31822866\n",
      "Publisher Correction: Analysis of the Human Protein Atlas Image Classification competition.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['none', 'none', 'none']\n",
      "Elapsed time: 5.39 seconds\n",
      "-------------------\n",
      "31907446\n",
      "Template-free detection and classification of membrane-bound complexes in cryo-electron tomograms.\n",
      "With faithful sample preservation and direct imaging of fully hydrated biological material, cryo-\n",
      "electron tomography provides an accurate representation of molecular architecture of cells. However,\n",
      "detection and precise localization of macromolecular complexes within cellular environments is\n",
      "aggravated by the presence of many molecular species and molecular crowding. We developed a\n",
      "template-free image processing procedure for accurate tracing of complex networks of densities in\n",
      "cryo-electron tomograms, a comprehensive and automated detection of heterogeneous membrane-bound\n",
      "complexes and an unsupervised classification (PySeg). Applications to intact cells and isolated\n",
      "endoplasmic reticulum (ER) allowed us to detect and classify small protein complexes. This\n",
      "classification provided sufficiently homogeneous particle sets and initial references to allow\n",
      "subsequent de novo subtomogram averaging. Spatial distribution analysis showed that ER complexes\n",
      "have different localization patterns forming nanodomains. Therefore, this procedure allows a\n",
      "comprehensive detection and structural analysis of complexes in situ.\n",
      "['cryo-electron tomography',\n",
      " 'image processing',\n",
      " 'automated detection',\n",
      " 'heterogeneous membrane-bound complexes',\n",
      " 'unsupervised classification',\n",
      " 'de novo subtomogram averaging',\n",
      " 'spatial distribution analysis']\n",
      "Elapsed time: 11.24 seconds\n",
      "-------------------\n",
      "31907447\n",
      "Nanoscale subcellular architecture revealed by multicolor three-dimensional salvaged fluorescence\n",
      "imaging.\n",
      "Combining the molecular specificity of fluorescent probes with three-dimensional imaging at\n",
      "nanoscale resolution is critical for investigating the spatial organization and interactions of\n",
      "cellular organelles and protein complexes. We present a 4Pi single-molecule switching super-\n",
      "resolution microscope that enables ratiometric multicolor imaging of mammalian cells at 5-10-nm\n",
      "localization precision in three dimensions using 'salvaged fluorescence'. Imaging two or three\n",
      "fluorophores simultaneously, we show fluorescence images that resolve the highly convoluted Golgi\n",
      "apparatus and the close contacts between the endoplasmic reticulum and the plasma membrane,\n",
      "structures that have traditionally been the imaging realm of electron microscopy. The salvaged\n",
      "fluorescence approach is equally applicable in most single-objective microscopes.\n",
      "['super-resolution microscope',\n",
      " 'single-molecule switching',\n",
      " '4Pi microscope',\n",
      " 'multicolor imaging',\n",
      " 'ratiometric imaging',\n",
      " 'salvaged fluorescence']\n",
      "Elapsed time: 10.51 seconds\n",
      "-------------------\n",
      "31932776\n",
      "MINFLUX nanoscopy delivers 3D multicolor nanometer resolution in cells.\n",
      "The ultimate goal of biological super-resolution fluorescence microscopy is to provide three-\n",
      "dimensional resolution at the size scale of a fluorescent marker. Here we show that by localizing\n",
      "individual switchable fluorophores with a probing donut-shaped excitation beam, MINFLUX nanoscopy\n",
      "can provide resolutions in the range of 1 to 3 nm for structures in fixed and living cells. This\n",
      "progress has been facilitated by approaching each fluorophore iteratively with the probing-donut\n",
      "minimum, making the resolution essentially uniform and isotropic over scalable fields of view.\n",
      "MINFLUX imaging of nuclear pore complexes of a mammalian cell shows that this true nanometer-scale\n",
      "resolution is obtained in three dimensions and in two color channels. Relying on fewer detected\n",
      "photons than standard camera-based localization, MINFLUX nanoscopy is poised to open a new chapter\n",
      "in the imaging of protein complexes and distributions in fixed and living cells.\n",
      "['MINFLUX nanoscopy',\n",
      " 'fluorescence microscopy',\n",
      " 'localizing individual switchable fluorophores',\n",
      " 'probing-donut minimum']\n",
      "Elapsed time: 10.51 seconds\n",
      "-------------------\n",
      "31932777\n",
      "Uncovering axes of variation among single-cell cancer specimens.\n",
      "While several tools have been developed to map axes of variation among individual cells, no\n",
      "analogous approaches exist for identifying axes of variation among multicellular biospecimens\n",
      "profiled at single-cell resolution. For this purpose, we developed 'phenotypic earth mover's\n",
      "distance' (PhEMD). PhEMD is a general method for embedding a 'manifold of manifolds', in which each\n",
      "datapoint in the higher-level manifold (of biospecimens) represents a collection of points that span\n",
      "a lower-level manifold (of cells). We apply PhEMD to a newly generated drug-screen dataset and\n",
      "demonstrate that PhEMD uncovers axes of cell subpopulational variation among a large set of\n",
      "perturbation conditions. Moreover, we show that PhEMD can be used to infer the phenotypes of\n",
      "biospecimens not directly profiled. Applied to clinical datasets, PhEMD generates a map of the\n",
      "patient-state space that highlights sources of patient-to-patient variation. PhEMD is scalable,\n",
      "compatible with leading batch-effect correction techniques and generalizable to multiple\n",
      "experimental designs.\n",
      "{'imaging_techniques': [\"phenotypic earth mover's distance\"]}\n",
      "Elapsed time: 10.96 seconds\n",
      "-------------------\n",
      "31969730\n",
      "Publisher Correction: Nucleus segmentation across imaging experiments: the 2018 Data Science Bowl.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['nucleus segmentation', 'imaging experiments', 'data science bowl']\n",
      "Elapsed time: 5.67 seconds\n",
      "-------------------\n",
      "31969731\n",
      "Publisher Correction: Analysis of the Human Protein Atlas Image Classification competition.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['none', 'none', 'none']\n",
      "Elapsed time: 5.51 seconds\n",
      "-------------------\n",
      "32015543\n",
      "SciPy 1.0: fundamental algorithms for scientific computing in Python.\n",
      "SciPy is an open-source scientific computing library for the Python programming language. Since its\n",
      "initial release in 2001, SciPy has become a de facto standard for leveraging scientific algorithms\n",
      "in Python, with over 600 unique code contributors, thousands of dependent packages, over 100,000\n",
      "dependent repositories and millions of downloads per year. In this work, we provide an overview of\n",
      "the capabilities and development practices of SciPy 1.0 and highlight some recent technical\n",
      "developments.\n",
      "['none']\n",
      "Elapsed time: 6.51 seconds\n",
      "-------------------\n",
      "32015544\n",
      "An optimized chemical-genetic method for cell-specific metabolic labeling of RNA.\n",
      "Tissues and organs are composed of diverse cell types, which poses a major challenge for cell-type-\n",
      "specific profiling of gene expression. Current metabolic labeling methods rely on exogenous\n",
      "pyrimidine analogs that are only incorporated into RNA in cells expressing an exogenous enzyme. This\n",
      "approach assumes that off-target cells cannot incorporate these analogs. We disprove this assumption\n",
      "and identify and characterize the enzymatic pathways responsible for high background incorporation.\n",
      "We demonstrate that mammalian cells can incorporate uracil analogs and characterize the enzymatic\n",
      "pathways responsible for high background incorporation. To overcome these limitations, we developed\n",
      "a new small molecule-enzyme pair consisting of uridine/cytidine kinase 2 and 2'-azidouridine. We\n",
      "demonstrate that 2'-azidouridine is only incorporated in cells expressing uridine/cytidine kinase 2\n",
      "and characterize selectivity mechanisms using molecular dynamics and X-ray crystallography.\n",
      "Furthermore, this pair can be used to purify and track RNA from specific cellular populations,\n",
      "making it ideal for high-resolution cell-specific RNA labeling. Overall, these results reveal new\n",
      "aspects of mammalian salvage pathways and serve as a new benchmark for designing, characterizing and\n",
      "evaluating methodologies for cell-specific labeling of biomolecules.\n",
      "['molecular dynamics', 'X-ray crystallography']\n",
      "Elapsed time: 12.22 seconds\n",
      "-------------------\n",
      "32042186\n",
      "Simultaneous multiplane imaging with reverberation two-photon microscopy.\n",
      "Multiphoton microscopy has gained enormous popularity because of its unique capacity to provide\n",
      "high-resolution images from deep within scattering tissue. Here, we demonstrate video-rate\n",
      "multiplane imaging with two-photon microscopy by performing near-instantaneous axial scanning while\n",
      "maintaining three-dimensional micrometer-scale resolution. Our technique, termed reverberation\n",
      "microscopy, enables the monitoring of neuronal populations over large depth ranges and can be\n",
      "implemented as a simple add-on to a conventional design.\n",
      "['multiphoton microscopy',\n",
      " 'axial scanning',\n",
      " 'reverberation microscopy',\n",
      " 'three-dimensional micrometer-scale resolution']\n",
      "Elapsed time: 7.55 seconds\n",
      "-------------------\n",
      "32042190\n",
      "Measurement of atom resolvability in cryo-EM maps with Q-scores.\n",
      "Cryogenic electron microscopy (cryo-EM) maps are now at the point where resolvability of individual\n",
      "atoms can be achieved. However, resolvability is not necessarily uniform throughout the map. We\n",
      "introduce a quantitative parameter to characterize the resolvability of individual atoms in cryo-EM\n",
      "maps, the map Q-score. Q-scores can be calculated for atoms in proteins, nucleic acids, water,\n",
      "ligands and other solvent atoms, using models fitted to or derived from cryo-EM maps. Q-scores can\n",
      "also be averaged to represent larger features such as entire residues and nucleotides. Averaged over\n",
      "entire models, Q-scores correlate very well with the estimated resolution of cryo-EM maps for both\n",
      "protein and RNA. Assuming the models they are calculated from are well fitted to the map, Q-scores\n",
      "can be used as a measure of resolvability in cryo-EM maps at various scales, from entire\n",
      "macromolecules down to individual atoms. Q-score analysis of multiple cryo-EM maps of the same\n",
      "proteins derived from different laboratories confirms the reproducibility of structural features\n",
      "from side chains down to water and ion atoms.\n",
      "['cryogenic electron microscopy', 'cryo-EM maps']\n",
      "Elapsed time: 11.12 seconds\n",
      "-------------------\n",
      "32066960\n",
      "Cell-type-specific signaling networks in heterocellular organoids.\n",
      "Despite the widespread adoption of organoids as biomimetic tissue models, methods to comprehensively\n",
      "analyze cell-type-specific post-translational modification (PTM) signaling networks in organoids are\n",
      "absent. Here, we report multivariate single-cell analysis of such networks in organoids and organoid\n",
      "cocultures. Simultaneous analysis by mass cytometry of 28 PTMs in >1 million single cells derived\n",
      "from small intestinal organoids reveals cell-type- and cell-state-specific signaling networks in\n",
      "stem, Paneth, enteroendocrine, tuft and goblet cells, as well as enterocytes. Integrating single-\n",
      "cell PTM analysis with thiol-reactive organoid barcoding in situ (TOBis) enables high-throughput\n",
      "comparison of signaling networks between organoid cultures. Cell-type-specific PTM analysis of\n",
      "colorectal cancer organoid cocultures reveals that shApc, Kras<sup>G12D</sup> and\n",
      "Trp53<sup>R172H</sup> cell-autonomously mimic signaling states normally induced by stromal\n",
      "fibroblasts and macrophages. These results demonstrate how standard mass cytometry workflows can be\n",
      "modified to perform high-throughput multivariate cell-type-specific signaling analysis of healthy\n",
      "and cancerous organoids.\n",
      "['mass cytometry',\n",
      " 'thiol-reactive organoid barcoding in situ (TOBis)',\n",
      " 'single-cell analysis']\n",
      "Elapsed time: 13.59 seconds\n",
      "-------------------\n",
      "32123391\n",
      "Multiplexed mass spectrometry of individual ions improves measurement of proteoforms and their\n",
      "complexes.\n",
      "An Orbitrap-based ion analysis procedure determines the direct charge for numerous individual\n",
      "protein ions to generate true mass spectra. This individual ion mass spectrometry (I<sup>2</sup>MS)\n",
      "method for charge detection enables the characterization of highly complicated mixtures of\n",
      "proteoforms and their complexes in both denatured and native modes of operation, revealing\n",
      "information not obtainable by typical measurements of ensembles of ions.\n",
      "['Orbitrap-based ion analysis procedure',\n",
      " 'individual ion mass spectrometry (I<sup>2</sup>MS)']\n",
      "Elapsed time: 7.37 seconds\n",
      "-------------------\n",
      "32123392\n",
      "Kilohertz two-photon fluorescence microscopy imaging of neural activity in vivo.\n",
      "Understanding information processing in the brain requires monitoring neuronal activity at high\n",
      "spatiotemporal resolution. Using an ultrafast two-photon fluorescence microscope empowered by all-\n",
      "optical laser scanning, we imaged neuronal activity in vivo at up to 3,000 frames per second and\n",
      "submicrometer spatial resolution. This imaging method enabled monitoring of both supra- and\n",
      "subthreshold electrical activity down to 345 μm below the brain surface in head-fixed awake mice.\n",
      "['two-photon fluorescence microscopy',\n",
      " 'ultrafast two-photon fluorescence microscope',\n",
      " 'all-optical laser scanning']\n",
      "Elapsed time: 8.12 seconds\n",
      "-------------------\n",
      "32123393\n",
      "Rapid mesoscale volumetric imaging of neural activity with synaptic resolution.\n",
      "Imaging neurons and neural circuits over large volumes at high speed and subcellular resolution is a\n",
      "difficult task. Incorporating a Bessel focus module into a two-photon fluorescence mesoscope, we\n",
      "achieved rapid volumetric imaging of neural activity over the mesoscale with synaptic resolution. We\n",
      "applied the technology to calcium imaging of entire dendritic spans of neurons as well as neural\n",
      "ensembles within multiple cortical regions over two hemispheres of the awake mouse brain.\n",
      "['two-photon fluorescence mesoscope',\n",
      " 'Bessel focus module',\n",
      " 'rapid volumetric imaging']\n",
      "Elapsed time: 7.75 seconds\n",
      "-------------------\n",
      "32123394\n",
      "A virtual reality system to analyze neural activity and behavior in adult zebrafish.\n",
      "Virtual realities are powerful tools to analyze and manipulate interactions between animals and\n",
      "their environment and to enable measurements of neuronal activity during behavior. In many species,\n",
      "however, optical access to the brain and/or the behavioral repertoire are limited. We developed a\n",
      "high-resolution virtual reality for head-restrained adult zebrafish, which exhibit cognitive\n",
      "behaviors not shown by larvae. We noninvasively measured activity throughout the dorsal\n",
      "telencephalon by multiphoton calcium imaging. Fish in the virtual reality showed regular swimming\n",
      "patterns and were attracted to animations of conspecifics. Manipulations of visuo-motor feedback\n",
      "revealed neurons that responded selectively to the mismatch between the expected and the actual\n",
      "visual consequences of motor output. Such error signals were prominent in multiple telencephalic\n",
      "areas, consistent with models of predictive processing. A virtual reality system for adult zebrafish\n",
      "therefore provides opportunities to analyze neuronal processing mechanisms underlying higher brain\n",
      "functions including decision making, associative learning, and social interactions.\n",
      "['multiphoton calcium imaging', 'virtual reality']\n",
      "Elapsed time: 9.93 seconds\n",
      "-------------------\n",
      "32123395\n",
      "Visualizing the genome in high resolution challenges our textbook understanding.\n",
      "The relationship between the 4D folding of the genome and its function is an outstanding question in\n",
      "biology. A range of methods that probe the folding of the genome in space and time with\n",
      "unprecedented resolution have been developed. These methods, including chromosome conformation\n",
      "capture and high-resolution light and electron microscopy, are shedding new light on genome\n",
      "architecture and function. Here, we review the emerging picture of genome organization revealed by\n",
      "super-resolution and live-cell imaging. We compare and contrast population-based chromosome\n",
      "conformation capture approaches and imaging-based approaches and highlight future challenges.\n",
      "['chromosome conformation capture',\n",
      " 'high-resolution light microscopy',\n",
      " 'electron microscopy']\n",
      "Elapsed time: 8.38 seconds\n",
      "-------------------\n",
      "32161395\n",
      "Machine learning analysis of whole mouse brain vasculature.\n",
      "Tissue clearing methods enable the imaging of biological specimens without sectioning. However,\n",
      "reliable and scalable analysis of large imaging datasets in three dimensions remains a challenge.\n",
      "Here we developed a deep learning-based framework to quantify and analyze brain vasculature, named\n",
      "Vessel Segmentation & Analysis Pipeline (VesSAP). Our pipeline uses a convolutional neural network\n",
      "(CNN) with a transfer learning approach for segmentation and achieves human-level accuracy. By using\n",
      "VesSAP, we analyzed the vascular features of whole C57BL/6J, CD1 and BALB/c mouse brains at the\n",
      "micrometer scale after registering them to the Allen mouse brain atlas. We report evidence of\n",
      "secondary intracranial collateral vascularization in CD1 mice and find reduced vascularization of\n",
      "the brainstem in comparison to the cerebrum. Thus, VesSAP enables unbiased and scalable\n",
      "quantifications of the angioarchitecture of cleared mouse brains and yields biological insights into\n",
      "the vascular function of the brain.\n",
      "[{'key': 'tissue clearing', 'value': 'none'},\n",
      " {'key': 'convolutional neural network (CNN)', 'value': 'none'},\n",
      " {'key': 'transfer learning approach', 'value': 'none'}]\n",
      "Elapsed time: 12.23 seconds\n",
      "-------------------\n",
      "32203384\n",
      "3D ATAC-PALM: super-resolution imaging of the accessible genome.\n",
      "To image the accessible genome at nanometer scale in situ, we developed three-dimensional assay for\n",
      "transposase-accessible chromatin-photoactivated localization microscopy (3D ATAC-PALM) that\n",
      "integrates an assay for transposase-accessible chromatin with visualization, PALM super-resolution\n",
      "imaging and lattice light-sheet microscopy. Multiplexed with oligopaint DNA-fluorescence in situ\n",
      "hybridization (FISH), RNA-FISH and protein fluorescence, 3D ATAC-PALM connected microscopy and\n",
      "genomic data, revealing spatially segregated accessible chromatin domains (ACDs) that enclose active\n",
      "chromatin and transcribed genes. Using these methods to analyze genetically perturbed cells, we\n",
      "demonstrated that genome architectural protein CTCF prevents excessive clustering of accessible\n",
      "chromatin and decompacts ACDs. These results highlight 3D ATAC-PALM as a useful tool to probe the\n",
      "structure and organizing mechanism of the genome.\n",
      "['ATAC-PALM',\n",
      " 'lattice light-sheet microscopy',\n",
      " 'oligopaint DNA-fluorescence in situ hybridization (FISH)',\n",
      " 'RNA-FISH',\n",
      " 'protein fluorescence']\n",
      "Elapsed time: 11.79 seconds\n",
      "-------------------\n",
      "32203385\n",
      "A 6-nm ultra-photostable DNA FluoroCube for fluorescence imaging.\n",
      "Photobleaching limits extended imaging of fluorescent biological samples. We developed DNA-based\n",
      "'FluoroCubes' that are similar in size to the green fluorescent protein, have single-point\n",
      "attachment to proteins, have a ~54-fold higher photobleaching lifetime and emit ~43-fold more\n",
      "photons than single organic dyes. We demonstrate that DNA FluoroCubes provide outstanding tools for\n",
      "single-molecule imaging, allowing the tracking of single motor proteins for >800 steps with\n",
      "nanometer precision.\n",
      "['fluorescence imaging', 'single-molecule imaging', 'tracking']\n",
      "Elapsed time: 8.01 seconds\n",
      "-------------------\n",
      "32203387\n",
      "Single-molecule displacement mapping unveils nanoscale heterogeneities in intracellular diffusivity.\n",
      "Intracellular diffusion underlies vital cellular processes. However, it remains difficult to\n",
      "elucidate how an unbound protein diffuses inside the cell with good spatial resolution and\n",
      "sensitivity. Here we introduce single-molecule displacement/diffusivity mapping (SMdM), a super-\n",
      "resolution strategy that enables the nanoscale mapping of intracellular diffusivity through local\n",
      "statistics of the instantaneous displacements of freely diffusing single molecules. We thus show\n",
      "that the diffusion of an average-sized protein in the mammalian cytoplasm and nucleus is spatially\n",
      "heterogeneous at the nanoscale, and that variations in local diffusivity correlate with the\n",
      "ultrastructure of the actin cytoskeleton and the organization of the genome, respectively. SMdM of\n",
      "differently charged proteins further unveils that the possession of positive, but not negative, net\n",
      "charges drastically impedes diffusion, and that the rate is determined by the specific subcellular\n",
      "environments. We thus unveil rich heterogeneities and charge effects in intracellular diffusion at\n",
      "the nanoscale.\n",
      "[{'name': 'single-molecule displacement mapping'},\n",
      " {'name': 'super-resolution strategy'}]\n",
      "Elapsed time: 10.72 seconds\n",
      "-------------------\n",
      "32203389\n",
      "RecV recombinase system for in vivo targeted optogenomic modifications of single cells or cell\n",
      "populations.\n",
      "Brain circuits comprise vast numbers of interconnected neurons with diverse molecular, anatomical\n",
      "and physiological properties. To allow targeting of individual neurons for structural and functional\n",
      "studies, we created light-inducible site-specific DNA recombinases based on Cre, Dre and Flp\n",
      "(RecVs). RecVs can induce genomic modifications by one-photon or two-photon light induction in vivo.\n",
      "They can produce targeted, sparse and strong labeling of individual neurons by modifying multiple\n",
      "loci within mouse and zebrafish genomes. In combination with other genetic strategies, they allow\n",
      "intersectional targeting of different neuronal classes. In the mouse cortex they enable sparse\n",
      "labeling and whole-brain morphological reconstructions of individual neurons. Furthermore, these\n",
      "enzymes allow single-cell two-photon targeted genetic modifications and can be used in combination\n",
      "with functional optical indicators with minimal interference. In summary, RecVs enable\n",
      "spatiotemporally precise optogenomic modifications that can facilitate detailed single-cell analysis\n",
      "of neural circuits by linking genetic identity, morphology, connectivity and function.\n",
      "['one-photon', 'two-photon', 'functional optical indicators']\n",
      "Elapsed time: 11.00 seconds\n",
      "-------------------\n",
      "32284609\n",
      "Scanless two-photon excitation with temporal focusing.\n",
      "Temporal focusing, with its ability to focus light in time, enables scanless illumination of large\n",
      "surface areas at the sample with micrometer axial confinement and robust propagation through\n",
      "scattering tissue. In conventional two-photon microscopy, widely used for the investigation of\n",
      "intact tissue in live animals, images are formed by point scanning of a spatially focused pulsed\n",
      "laser beam, resulting in limited temporal resolution of the excitation. Replacing point scanning\n",
      "with temporally focused widefield illumination removes this limitation and represents an important\n",
      "milestone in two-photon microscopy. Temporal focusing uses a diffusive or dispersive optical element\n",
      "placed in a plane conjugate to the objective focal plane to generate position-dependent temporal\n",
      "pulse broadening that enables axially confined multiphoton absorption, without the need for tight\n",
      "spatial focusing. Many techniques have benefitted from temporal focusing, including scanless\n",
      "imaging, super-resolution imaging, photolithography, uncaging of caged neurotransmitters and control\n",
      "of neuronal activity via optogenetics.\n",
      "['scanless two-photon excitation',\n",
      " 'temporal focusing',\n",
      " 'super-resolution imaging',\n",
      " 'photolithography',\n",
      " 'uncaging of caged neurotransmitters',\n",
      " 'optogenetics']\n",
      "Elapsed time: 11.90 seconds\n",
      "-------------------\n",
      "32313222\n",
      "Multiplexed Cre-dependent selection yields systemic AAVs for targeting distinct brain cell types.\n",
      "Recombinant adeno-associated viruses (rAAVs) are efficient gene delivery vectors via intravenous\n",
      "delivery; however, natural serotypes display a finite set of tropisms. To expand their utility, we\n",
      "evolved AAV capsids to efficiently transduce specific cell types in adult mouse brains. Building\n",
      "upon our Cre-recombination-based AAV targeted evolution (CREATE) platform, we developed Multiplexed-\n",
      "CREATE (M-CREATE) to identify variants of interest in a given selection landscape through multiple\n",
      "positive and negative selection criteria. M-CREATE incorporates next-generation sequencing,\n",
      "synthetic library generation and a dedicated analysis pipeline. We have identified capsid variants\n",
      "that can transduce the central nervous system broadly, exhibit bias toward vascular cells and\n",
      "astrocytes, target neurons with greater specificity or cross the blood-brain barrier across diverse\n",
      "murine strains. Collectively, the M-CREATE methodology accelerates the discovery of capsids for use\n",
      "in neuroscience and gene-therapy applications.\n",
      "['next-generation sequencing',\n",
      " 'synthetic library generation',\n",
      " 'dedicated analysis pipeline']\n",
      "Elapsed time: 10.23 seconds\n",
      "-------------------\n",
      "32341543\n",
      "Deep three-photon imaging of the brain in intact adult zebrafish.\n",
      "Behaviors emerge from activity throughout the brain, but noninvasive optical access in adult\n",
      "vertebrate brains is limited. We show that three-photon (3P) imaging through the head of intact\n",
      "adult zebrafish allows structural and functional imaging at cellular resolution throughout the\n",
      "telencephalon and deep into the cerebellum and optic tectum. With 3P imaging, considerable portions\n",
      "of the brain become noninvasively accessible from embryo to sexually mature adult in a vertebrate\n",
      "model.\n",
      "['three-photon imaging',\n",
      " 'cellular resolution',\n",
      " 'structural and functional imaging']\n",
      "Elapsed time: 7.46 seconds\n",
      "-------------------\n",
      "32341544\n",
      "A comparison of microfluidic methods for high-throughput cell deformability measurements.\n",
      "The mechanical phenotype of a cell is an inherent biophysical marker of its state and function, with\n",
      "many applications in basic and applied biological research. Microfluidics-based methods have enabled\n",
      "single-cell mechanophenotyping at throughputs comparable to those of flow cytometry. Here, we\n",
      "present a standardized cross-laboratory study comparing three microfluidics-based approaches for\n",
      "measuring cell mechanical phenotype: constriction-based deformability cytometry (cDC), shear flow\n",
      "deformability cytometry (sDC) and extensional flow deformability cytometry (xDC). All three methods\n",
      "detect cell deformability changes induced by exposure to altered osmolarity. However, a dose-\n",
      "dependent deformability increase upon latrunculin B-induced actin disassembly was detected only with\n",
      "cDC and sDC, which suggests that when exposing cells to the higher strain rate imposed by xDC,\n",
      "cellular components other than the actin cytoskeleton dominate the response. The direct comparison\n",
      "presented here furthers our understanding of the applicability of the different deformability\n",
      "cytometry methods and provides context for the interpretation of deformability measurements\n",
      "performed using different platforms.\n",
      "['constriction-based deformability cytometry (cDC)',\n",
      " 'shear flow deformability cytometry (sDC)',\n",
      " 'extensional flow deformability cytometry (xDC)']\n",
      "Elapsed time: 12.36 seconds\n",
      "-------------------\n",
      "32371979\n",
      "Three-photon head-mounted microscope for imaging deep cortical layers in freely moving rats.\n",
      "We designed a head-mounted three-photon microscope for imaging deep cortical layer neuronal activity\n",
      "in a freely moving rat. Delivery of high-energy excitation pulses at 1,320 nm required both a\n",
      "hollow-core fiber whose transmission properties did not change with fiber movement and dispersion\n",
      "compensation. These developments enabled imaging at >1.1 mm below the cortical surface and stable\n",
      "imaging of layer 5 neuronal activity for >1 h in freely moving rats performing a range of behaviors.\n",
      "['three-photon microscope',\n",
      " 'imaging deep cortical layers',\n",
      " 'high-energy excitation pulses',\n",
      " 'hollow-core fiber',\n",
      " 'dispersion compensation']\n",
      "Elapsed time: 8.24 seconds\n",
      "-------------------\n",
      "32371980\n",
      "Three-dimensional nanoscopy of whole cells and tissues with in situ point spread function retrieval.\n",
      "Single-molecule localization microscopy is a powerful tool for visualizing subcellular structures,\n",
      "interactions and protein functions in biological research. However, inhomogeneous refractive indices\n",
      "inside cells and tissues distort the fluorescent signal emitted from single-molecule probes, which\n",
      "rapidly degrades resolution with increasing depth. We propose a method that enables the construction\n",
      "of an in situ 3D response of single emitters directly from single-molecule blinking datasets, and\n",
      "therefore allows their locations to be pinpointed with precision that achieves the Cramér-Rao lower\n",
      "bound and uncompromised fidelity. We demonstrate this method, named in situ PSF retrieval (INSPR),\n",
      "across a range of cellular and tissue architectures, from mitochondrial networks and nuclear pores\n",
      "in mammalian cells to amyloid-β plaques and dendrites in brain tissues and elastic fibers in\n",
      "developing cartilage of mice. This advancement expands the routine applicability of super-resolution\n",
      "microscopy from selected cellular targets near coverslips to intra- and extracellular targets deep\n",
      "inside tissues.\n",
      "{'imaging_techniques': ['single-molecule localization microscopy',\n",
      "                        'in situ point spread function retrieval',\n",
      "                        'super-resolution microscopy']}\n",
      "Elapsed time: 11.78 seconds\n",
      "-------------------\n",
      "32393832\n",
      "Pooled CRISPR screens with imaging on microraft arrays reveals stress granule-regulatory factors.\n",
      "Genetic screens using pooled CRISPR-based approaches are scalable and inexpensive, but restricted to\n",
      "standard readouts, including survival, proliferation and sortable markers. However, many\n",
      "biologically relevant cell states involve cellular and subcellular changes that are only accessible\n",
      "by microscopic visualization, and are currently impossible to screen with pooled methods. Here we\n",
      "combine pooled CRISPR-Cas9 screening with microraft array technology and high-content imaging to\n",
      "screen image-based phenotypes (CRaft-ID; CRISPR-based microRaft followed by guide RNA\n",
      "identification). By isolating microrafts that contain genetic clones harboring individual guide RNAs\n",
      "(gRNA), we identify RNA-binding proteins (RBPs) that influence the formation of stress granules, the\n",
      "punctate protein-RNA assemblies that form during stress. To automate hit identification, we\n",
      "developed a machine-learning model trained on nuclear morphology to remove unhealthy cells or\n",
      "imaging artifacts. In doing so, we identified and validated previously uncharacterized RBPs that\n",
      "modulate stress granule abundance, highlighting the applicability of our approach to facilitate\n",
      "image-based pooled CRISPR screens.\n",
      "['high-content imaging', 'microraft array technology']\n",
      "Elapsed time: 11.76 seconds\n",
      "-------------------\n",
      "32424271\n",
      "Elasticizing tissues for reversible shape transformation and accelerated molecular labeling.\n",
      "We developed entangled link-augmented stretchable tissue-hydrogel (ELAST), a technology that\n",
      "transforms tissues into elastic hydrogels to enhance macromolecular accessibility and mechanical\n",
      "stability simultaneously. ELASTicized tissues are highly stretchable and compressible, which enables\n",
      "reversible shape transformation and faster delivery of probes into intact tissue specimens via\n",
      "mechanical thinning. This universal platform may facilitate rapid and scalable molecular phenotyping\n",
      "of large-scale biological systems, such as human organs.\n",
      "['Elasticizing tissues',\n",
      " 'reversible shape transformation',\n",
      " 'accelerated molecular labeling']\n",
      "Elapsed time: 7.27 seconds\n",
      "-------------------\n",
      "32451475\n",
      "jYCaMP: an optimized calcium indicator for two-photon imaging at fiber laser wavelengths.\n",
      "Femtosecond lasers at fixed wavelengths above 1,000 nm are powerful, stable and inexpensive, making\n",
      "them promising sources for two-photon microscopy. Biosensors optimized for these wavelengths are\n",
      "needed for both next-generation microscopes and affordable turn-key systems. Here we report jYCaMP1,\n",
      "a yellow variant of the calcium indicator jGCaMP7 that outperforms its parent in mice and flies at\n",
      "excitation wavelengths above 1,000 nm and enables improved two-color calcium imaging with red\n",
      "fluorescent protein-based indicators.\n",
      "['two-photon microscopy', 'fiber lasers', 'biosensors']\n",
      "Elapsed time: 8.89 seconds\n",
      "-------------------\n",
      "32451476\n",
      "Intelligent image-based deformation-assisted cell sorting with molecular specificity.\n",
      "Although label-free cell sorting is desirable for providing pristine cells for further analysis or\n",
      "use, current approaches lack molecular specificity and speed. Here, we combine real-time\n",
      "fluorescence and deformability cytometry with sorting based on standing surface acoustic waves and\n",
      "transfer molecular specificity to image-based sorting using an efficient deep neural network. In\n",
      "addition to general performance, we demonstrate the utility of this method by sorting neutrophils\n",
      "from whole blood without labels.\n",
      "['real-time fluorescence',\n",
      " 'deformability cytometry',\n",
      " 'standing surface acoustic waves',\n",
      " 'deep neural network']\n",
      "Elapsed time: 7.30 seconds\n",
      "-------------------\n",
      "32483335\n",
      "Real-time 3D movement correction for two-photon imaging in behaving animals.\n",
      "Two-photon microscopy is widely used to investigate brain function across multiple spatial scales.\n",
      "However, measurements of neural activity are compromised by brain movement in behaving animals.\n",
      "Brain motion-induced artifacts are typically corrected using post hoc processing of two-dimensional\n",
      "images, but this approach is slow and does not correct for axial movements. Moreover, the\n",
      "deleterious effects of brain movement on high-speed imaging of small regions of interest and\n",
      "photostimulation cannot be corrected post hoc. To address this problem, we combined random-access\n",
      "three-dimensional (3D) laser scanning using an acousto-optic lens and rapid closed-loop field\n",
      "programmable gate array processing to track 3D brain movement and correct motion artifacts in real\n",
      "time at up to 1 kHz. Our recordings from synapses, dendrites and large neuronal populations in\n",
      "behaving mice and zebrafish demonstrate real-time movement-corrected 3D two-photon imaging with\n",
      "submicrometer precision.\n",
      "['two-photon microscopy',\n",
      " 'random-access three-dimensional (3D) laser scanning',\n",
      " 'real-time movement correction']\n",
      "Elapsed time: 10.38 seconds\n",
      "-------------------\n",
      "32541852\n",
      "Highly specific multiplexed RNA imaging in tissues with split-FISH.\n",
      "We present split-FISH, a multiplexed fluorescence in situ hybridization method that leverages a\n",
      "split-probe design to achieve enhanced specificity. Split-FISH reduces off-target background\n",
      "fluorescence, decreases false positives and enables accurate RNA profiling in uncleared tissues. We\n",
      "demonstrate the efficacy of split-FISH on various mouse tissues by quantifying the distribution and\n",
      "abundance of 317 genes in single cells and reveal diverse localization patterns for spatial\n",
      "regulation of the transcriptome in complex tissues.\n",
      "['fluorescence in situ hybridization', 'split-FISH', 'multiplexed']\n",
      "Elapsed time: 7.71 seconds\n",
      "-------------------\n",
      "32541853\n",
      "DeepSTORM3D: dense 3D localization microscopy and PSF design by deep learning.\n",
      "An outstanding challenge in single-molecule localization microscopy is the accurate and precise\n",
      "localization of individual point emitters in three dimensions in densely labeled samples. One\n",
      "established approach for three-dimensional single-molecule localization is point-spread-function\n",
      "(PSF) engineering, in which the PSF is engineered to vary distinctively with emitter depth using\n",
      "additional optical elements. However, images of dense emitters, which are desirable for improving\n",
      "temporal resolution, pose a challenge for algorithmic localization of engineered PSFs, due to\n",
      "lateral overlap of the emitter PSFs. Here we train a neural network to localize multiple emitters\n",
      "with densely overlapping Tetrapod PSFs over a large axial range. We then use the network to design\n",
      "the optimal PSF for the multi-emitter case. We demonstrate our approach experimentally with super-\n",
      "resolution reconstructions of mitochondria and volumetric imaging of fluorescently labeled telomeres\n",
      "in cells. Our approach, DeepSTORM3D, enables the study of biological processes in whole cells at\n",
      "timescales that are rarely explored in localization microscopy.\n",
      "['dense 3D localization microscopy',\n",
      " 'point-spread-function (PSF) engineering',\n",
      " 'algorithmic localization',\n",
      " 'neural network',\n",
      " 'super-resolution reconstruction',\n",
      " 'volumetric imaging']\n",
      "Elapsed time: 12.31 seconds\n",
      "-------------------\n",
      "32572233\n",
      "Homogeneous multifocal excitation for high-throughput super-resolution imaging.\n",
      "Super-resolution microscopies have become an established tool in biological research. However,\n",
      "imaging throughput remains a main bottleneck in acquiring large datasets required for quantitative\n",
      "biology. Here we describe multifocal flat illumination for field-independent imaging (mfFIFI). By\n",
      "integrating mfFIFI into an instant structured illumination microscope (iSIM), we extend the field of\n",
      "view (FOV) to >100 × 100 µm<sup>2</sup> while maintaining high-speed, multicolor, volumetric imaging\n",
      "at double the diffraction-limited resolution. We further extend the effective FOV by stitching\n",
      "adjacent images for fast live-cell super-resolution imaging of dozens of cells. Finally, we combine\n",
      "our flat-fielded iSIM with ultrastructure expansion microscopy to collect three-dimensional (3D)\n",
      "images of hundreds of centrioles in human cells, or thousands of purified Chlamydomonas reinhardtii\n",
      "centrioles, per hour at an effective resolution of ~35 nm. Classification and particle averaging of\n",
      "these large datasets enables 3D mapping of posttranslational modifications of centriolar\n",
      "microtubules, revealing differences in their coverage and positioning.\n",
      "['instant structured illumination microscope',\n",
      " 'multifocal flat illumination',\n",
      " 'live-cell super-resolution imaging',\n",
      " 'ultrastructure expansion microscopy']\n",
      "Elapsed time: 13.06 seconds\n",
      "-------------------\n",
      "32591761\n",
      "Publisher Correction: DeepSTORM3D: dense 3D localization microscopy and PSF design by deep learning.\n",
      "An amendment to this paper has been published and can be accessed via a link at the top of the\n",
      "paper.\n",
      "['DeepSTORM3D',\n",
      " 'dense 3D localization microscopy',\n",
      " 'PSF design by deep learning']\n",
      "Elapsed time: 6.00 seconds\n",
      "-------------------\n",
      "32601424\n",
      "Up to 100-fold speed-up and multiplexing in optimized DNA-PAINT.\n",
      "DNA-PAINT's imaging speed has recently been significantly enhanced by optimized sequence design and\n",
      "buffer conditions. However, this implementation has not reached an ultimate speed limit and is only\n",
      "applicable to imaging of single targets. To further improve acquisition speed, we introduce\n",
      "concatenated, periodic DNA sequence motifs, yielding up to 100-fold-faster sampling in comparison to\n",
      "traditional DNA-PAINT. We extend this approach to six orthogonal sequence motifs, now enabling\n",
      "speed-optimized multiplexed imaging.\n"
     ]
    },
    {
     "ename": "OutputParserException",
     "evalue": "Invalid json output: [\n\"DNA-PAINT\",\n\"traditional DNA-PAINT\",\n\"concatenated, periodic DNA sequence motifs\"\n\"multiplexed imaging\"\n] derived from Sure! Here is the list of imaging techniques or technologies described in the abstract:\n\n[\n\"DNA-PAINT\",\n\"traditional DNA-PAINT\",\n\"concatenated, periodic DNA sequence motifs\"\n\"multiplexed imaging\"\n]\n\nIs there anything else I can help you with?",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mJSONDecodeError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[26], line 22\u001b[0m, in \u001b[0;36mJsonEnclosedByTextOutputParser.parse\u001b[0;34m(self, text)\u001b[0m\n\u001b[1;32m     21\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m---> 22\u001b[0m     \u001b[39mreturn\u001b[39;00m json\u001b[39m.\u001b[39;49mloads(text1)\n\u001b[1;32m     23\u001b[0m \u001b[39mexcept\u001b[39;00m JSONDecodeError \u001b[39mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/json/__init__.py:346\u001b[0m, in \u001b[0;36mloads\u001b[0;34m(s, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)\u001b[0m\n\u001b[1;32m    343\u001b[0m \u001b[39mif\u001b[39;00m (\u001b[39mcls\u001b[39m \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mand\u001b[39;00m object_hook \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mand\u001b[39;00m\n\u001b[1;32m    344\u001b[0m         parse_int \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mand\u001b[39;00m parse_float \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mand\u001b[39;00m\n\u001b[1;32m    345\u001b[0m         parse_constant \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mand\u001b[39;00m object_pairs_hook \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mand\u001b[39;00m \u001b[39mnot\u001b[39;00m kw):\n\u001b[0;32m--> 346\u001b[0m     \u001b[39mreturn\u001b[39;00m _default_decoder\u001b[39m.\u001b[39;49mdecode(s)\n\u001b[1;32m    347\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mcls\u001b[39m \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/json/decoder.py:337\u001b[0m, in \u001b[0;36mJSONDecoder.decode\u001b[0;34m(self, s, _w)\u001b[0m\n\u001b[1;32m    333\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"Return the Python representation of ``s`` (a ``str`` instance\u001b[39;00m\n\u001b[1;32m    334\u001b[0m \u001b[39mcontaining a JSON document).\u001b[39;00m\n\u001b[1;32m    335\u001b[0m \n\u001b[1;32m    336\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[0;32m--> 337\u001b[0m obj, end \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mraw_decode(s, idx\u001b[39m=\u001b[39;49m_w(s, \u001b[39m0\u001b[39;49m)\u001b[39m.\u001b[39;49mend())\n\u001b[1;32m    338\u001b[0m end \u001b[39m=\u001b[39m _w(s, end)\u001b[39m.\u001b[39mend()\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/json/decoder.py:353\u001b[0m, in \u001b[0;36mJSONDecoder.raw_decode\u001b[0;34m(self, s, idx)\u001b[0m\n\u001b[1;32m    352\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m--> 353\u001b[0m     obj, end \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mscan_once(s, idx)\n\u001b[1;32m    354\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mStopIteration\u001b[39;00m \u001b[39mas\u001b[39;00m err:\n",
      "\u001b[0;31mJSONDecodeError\u001b[0m: Expecting ',' delimiter: line 5 column 1 (char 85)",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[0;31mOutputParserException\u001b[0m                     Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[28], line 14\u001b[0m\n\u001b[1;32m     12\u001b[0m p \u001b[39m=\u001b[39m prompt\u001b[39m.\u001b[39mformat(article_text\u001b[39m=\u001b[39m(row\u001b[39m.\u001b[39mTITLE\u001b[39m+\u001b[39m\u001b[39m'\u001b[39m\u001b[39m\\n\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m+\u001b[39mrow\u001b[39m.\u001b[39mABSTRACT))\n\u001b[1;32m     13\u001b[0m output_text \u001b[39m=\u001b[39m llm(p)\n\u001b[0;32m---> 14\u001b[0m output \u001b[39m=\u001b[39m output_parser\u001b[39m.\u001b[39;49mparse(output_text)\n\u001b[1;32m     15\u001b[0m pprint(output, depth\u001b[39m=\u001b[39m\u001b[39m2\u001b[39m)\n\u001b[1;32m     16\u001b[0m delta_t \u001b[39m=\u001b[39m time\u001b[39m.\u001b[39mtime() \u001b[39m-\u001b[39m start_time\n",
      "Cell \u001b[0;32mIn[26], line 24\u001b[0m, in \u001b[0;36mJsonEnclosedByTextOutputParser.parse\u001b[0;34m(self, text)\u001b[0m\n\u001b[1;32m     22\u001b[0m         \u001b[39mreturn\u001b[39;00m json\u001b[39m.\u001b[39mloads(text1)\n\u001b[1;32m     23\u001b[0m     \u001b[39mexcept\u001b[39;00m JSONDecodeError \u001b[39mas\u001b[39;00m e:\n\u001b[0;32m---> 24\u001b[0m         \u001b[39mraise\u001b[39;00m OutputParserException(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mInvalid json output: \u001b[39m\u001b[39m{\u001b[39;00mtext1\u001b[39m}\u001b[39;00m\u001b[39m derived from \u001b[39m\u001b[39m{\u001b[39;00mtext\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m) \u001b[39mfrom\u001b[39;00m \u001b[39me\u001b[39;00m\n\u001b[1;32m     25\u001b[0m \u001b[39melse\u001b[39;00m: \n\u001b[1;32m     26\u001b[0m     \u001b[39mraise\u001b[39;00m OutputParserException(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mCould not find json-formatted data in: \u001b[39m\u001b[39m{\u001b[39;00mtext\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m)\n",
      "\u001b[0;31mOutputParserException\u001b[0m: Invalid json output: [\n\"DNA-PAINT\",\n\"traditional DNA-PAINT\",\n\"concatenated, periodic DNA sequence motifs\"\n\"multiplexed imaging\"\n] derived from Sure! Here is the list of imaging techniques or technologies described in the abstract:\n\n[\n\"DNA-PAINT\",\n\"traditional DNA-PAINT\",\n\"concatenated, periodic DNA sequence motifs\"\n\"multiplexed imaging\"\n]\n\nIs there anything else I can help you with?"
     ]
    }
   ],
   "source": [
    "import time\n",
    "\n",
    "l = []\n",
    "t = []\n",
    "for i, row in df.iterrows():\n",
    "    if row.ABSTRACT == '':\n",
    "        continue\n",
    "    start_time = time.time()\n",
    "    print(row.ID_PAPER)\n",
    "    print(textwrap.fill(row.TITLE, 100))\n",
    "    print(textwrap.fill(row.ABSTRACT, 100))\n",
    "    p = prompt.format(article_text=(row.TITLE+'\\n'+row.ABSTRACT))\n",
    "    output_text = llm(p)\n",
    "    output = output_parser.parse(output_text)\n",
    "    pprint(output, depth=2)\n",
    "    delta_t = time.time() - start_time\n",
    "    t.append(delta_t)\n",
    "    print(f'Elapsed time: {delta_t:.2f} seconds')\n",
    "    print('-------------------')\n",
    "    for ner in output:\n",
    "        l.append((row.ID_PAPER, ner))\n",
    "l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'30061677', '28604724', '31932777', '33941937', '28783155', '28319112', '28967889', '28869755', '31548704', '31591578', '28394337', '31591580', '29298292', '31501548', '33398191', '30349041', '32066960', '30804550', '32123392', '27749837', '28417997', '31086341', '33589838', '34312566', '28650478', '29355847', '32661427', '32015543', '30013046', '31097821', '29334378', '29320487', '29039416', '27869816', '31086342', '30082898', '27776112', '34341582', '27798610', '27798611', '33875885', '34594032', '32616928', '28481362', '30643212', '29786093', '28869758', '34059826', '30397326', '29553579', '32895537', '30108338', '33633410', '30988469', '33408404', '30377376', '32958921', '28858338', '29131163', '30504871', '31501551', '28068315', '30573831', '32424271', '34608319', '28945705', '28846087', '29638227', '28250468', '30504889', '28628129', '31040436', '32203384', '30573813', '33963344', '34845388', '28661494', '33795876', '32807955', '31611690', '30559429', '32203385', '31308547', '30478324', '31768063', '31285624', '28418000', '31591576', '32123395', '29083401', '31501552', '34446922', '32737473', '27842059', '28628128', '29967495', '30573820', '31712780', '32989318', '30923372', '28362436', '34226721', '28165473', '34916672', '32719530', '30202058', '29553578', '31673159', '34059827', '33686300', '34354291', '33077967', '31969730', '28530655', '31740820', '34824475', '28581494', '34226720', '34092790', '32541853', '30377346', '31527839', '31562475', '32591761', '27749836', '33139894', '30858598', '31819267', '28991891', '34127855', '29256494', '31653976', '30532082', '30559434', '30559430', '34059828', '33649590', '30478322', '33633409', '31768058', '27869814', '31209383', '31363222', '30742041', '34168373', '33542511', '32778833', '27869813', '32451475', '30420686', '28346450', '34650233', '31285623', '34183831', '32451476', '30377377', '33479522', '32203389', '31768062', '31451764', '29735999', '30150754', '28581495', '28481364', '27798612', '27918540', '31548706', '29320486', '33046897', '28869757', '30478326', '31427757', '32713945', '28362435', '29083403', '32371979', '29915189', '32341543', '29735996', '31636458', '29355846', '29578535', '30250056', '27819661', '28218899', '31384043', '33046895', '32747769', '34737453', '31570887', '28504681', '32015544', '32820264', '33574612', '31548708', '29713082', '31780840', '32601424', '32042190', '34462592', '30171252', '29736000', '31932776', '29039418', '33820989', '28869756', '30275574', '30420685', '28553965', '33963348', '34811556', '31636459', '30224672', '31471616', '30377349', '34326541', '32929270', '28945703', '34594030', '30065368', '34711971', '32042186', '32778831', '34462594', '28825703', '31358979', '32807954', '27798609', '32807957', '30602783', '29608556', '31907446', '30127506', '30127504', '28825705', '30742039', '33408397', '33139893', '28319111', '33875887', '28671674', '34400836', '33318659', '31451766', '32632238', '31086343', '31133758', '27941784', '31740815', '28369043', '32371980', '30046099', '31686040', '30643213', '34059829', '34608310', '32929269', '31548707', '31969731', '29967496', '32203387', '32807956', '31641286', '31061492', '28945706', '31520070', '28775673', '32284609', '29967493', '31591575', '28218900', '31308549', '31907447', '29988094', '33963356', '31659327', '29481551', '31611693', '31611694', '34594033', '28362438', '28628127', '29808018', '32895538', '32393832', '32123393', '32341544', '30202059', '33408403', '29578537', '31740816', '32541852', '33288959', '32719532', '34608318', '31802009', '31792434', '30377372', '33820987', '30504888', '32123394', '34354294', '34862503', '33859440', '31384047', '32719531', '31501547', '30377360', '27842060', '31768059', '33542514', '30377374', '31209382', '29481550', '29941872', '30135560', '34887551', '28650477', '31822866', '31562488', '34341583', '30287931', '32483335', '34099930', '30013047', '31740821', '29786091', '33408407', '33963355', '32958922', '33077969', '30594947', '34155395', '32123391', '32572233', '30275587', '29988095', '31562489', '30532080', '32760039', '28288122', '32313222', '34725481', '32747768', '32778832', '33046896', '30224671', '31740819', '31611691', '34608315', '30664778', '29630062', '30171236', '34092791', '32601425', '29377014', '34608309', '34824474', '29131164', '33686301', '31819263', '31110282', '33169015', '30377363', '28250467', '30962624', '33649587', '31263253', '32161395', '29083400', '31363221', '29176592', '34480155', '33542510', '32989319', '34312564', '33542509', '29256495', '29131162', '28846089', '31011184', '31363225', '30858600', '31471618', '30573816', '34824477', '29400715', '30664776', '34354290', '30962622', '31686039', '31740817', '33257830', '28892088', '33288961', '31471614', '29457791', '31036959', '33087905', '34462591', '27992408', '30804552'}\n",
      "402\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "\n",
    "lset=set()\n",
    "for id in l:\n",
    "    lset.add(id[0])\n",
    "print(lset)\n",
    "print(len(lset))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "34949807\n",
      "Rapid, efficient and activation-neutral gene editing of polyclonal primary human resting\n",
      "CD4<sup>+</sup> T cells allows complex functional analyses.\n",
      "CD4<sup>+</sup> T cells are central mediators of adaptive and innate immune responses and constitute\n",
      "a major reservoir for human immunodeficiency virus (HIV) in vivo. Detailed investigations of resting\n",
      "human CD4<sup>+</sup> T cells have been precluded by the absence of efficient approaches for genetic\n",
      "manipulation limiting our understanding of HIV replication and restricting efforts to find a cure.\n",
      "Here we report a method for rapid, efficient, activation-neutral gene editing of resting, polyclonal\n",
      "human CD4<sup>+</sup> T cells using optimized cell cultivation and nucleofection conditions of\n",
      "Cas9-guide RNA ribonucleoprotein complexes. Up to six genes, including HIV dependency and\n",
      "restriction factors, were knocked out individually or simultaneously and functionally characterized.\n",
      "Moreover, we demonstrate the knock in of double-stranded DNA donor templates into different\n",
      "endogenous loci, enabling the study of the physiological interplay of cellular and viral components\n",
      "at single-cell resolution. Together, this technique allows improved molecular and functional\n",
      "characterizations of HIV biology and general immune functions in resting CD4<sup>+</sup> T cells.\n",
      "['nucleofection', 'cell cultivation', 'single-cell resolution']\n",
      "Elapsed time: 1.83 seconds\n",
      "-------------------\n",
      "34949809\n",
      "FlyWire: online community for whole-brain connectomics.\n",
      "Due to advances in automated image acquisition and analysis, whole-brain connectomes with 100,000 or\n",
      "more neurons are on the horizon. Proofreading of whole-brain automated reconstructions will require\n",
      "many person-years of effort, due to the huge volumes of data involved. Here we present FlyWire, an\n",
      "online community for proofreading neural circuits in a Drosophila melanogaster brain and explain how\n",
      "its computational and social structures are organized to scale up to whole-brain connectomics.\n",
      "Browser-based three-dimensional interactive segmentation by collaborative editing of a spatially\n",
      "chunked supervoxel graph makes it possible to distribute proofreading to individuals located\n",
      "virtually anywhere in the world. Information in the edit history is programmatically accessible for\n",
      "a variety of uses such as estimating proofreading accuracy or building incentive systems. An open\n",
      "community accelerates proofreading by recruiting more participants and accelerates scientific\n",
      "discovery by requiring information sharing. We demonstrate how FlyWire enables circuit analysis by\n",
      "reconstructing and analyzing the connectome of mechanosensory neurons.\n",
      "['browser-based three-dimensional interactive segmentation',\n",
      " 'collaborative editing of a spatially chunked supervoxel graph',\n",
      " 'automated image acquisition']\n",
      "Elapsed time: 11.05 seconds\n",
      "-------------------\n",
      "34949810\n",
      "Fast optical recording of neuronal activity by three-dimensional custom-access serial holography.\n",
      "Optical recording of neuronal activity in three-dimensional (3D) brain circuits at cellular and\n",
      "millisecond resolution in vivo is essential for probing information flow in the brain. While random-\n",
      "access multiphoton microscopy permits fast optical access to neuronal targets in three dimensions,\n",
      "the method is challenged by motion artifacts when recording from behaving animals. Therefore, we\n",
      "developed three-dimensional custom-access serial holography (3D-CASH). Built on a fast acousto-optic\n",
      "light modulator, 3D-CASH performs serial sampling at 40 kHz from neurons at freely selectable 3D\n",
      "locations. Motion artifacts are eliminated by targeting each neuron with a size-optimized pattern of\n",
      "excitation light covering the cell body and its anticipated displacement field. Spike rates inferred\n",
      "from GCaMP6f recordings in visual cortex of awake mice tracked the phase of a moving bar stimulus\n",
      "with higher spike correlation between intra compared to interlaminar neuron pairs. 3D-CASH offers\n",
      "access to the millisecond correlation structure of in vivo neuronal activity in 3D microcircuits.\n",
      "['three-dimensional custom-access serial holography',\n",
      " 'random-access multiphoton microscopy',\n",
      " 'acousto-optic light modulator']\n",
      "Elapsed time: 11.49 seconds\n",
      "-------------------\n",
      "34949811\n",
      "Caveat fluorophore: an insiders' guide to small-molecule fluorescent labels.\n",
      "The last three decades have brought a revolution in fluorescence microscopy. The development of new\n",
      "microscopes, fluorescent labels and analysis techniques has pushed the frontiers of biological\n",
      "imaging forward, moving from fixed to live cells, from diffraction-limited to super-resolution\n",
      "imaging and from simple cell culture systems to experiments in vivo. The large and ever-evolving\n",
      "collection of tools can be daunting for biologists, who must invest substantial time and effort in\n",
      "adopting new technologies to answer their specific questions. This is particularly relevant when\n",
      "working with small-molecule fluorescent labels, where users must navigate the jargon, idiosyncrasies\n",
      "and caveats of chemistry. Here, we present an overview of chemical dyes used in biology and provide\n",
      "frank advice from a chemist's perspective.\n",
      "['microscopes', 'fluorescent labels', 'super-resolution imaging', 'live cells']\n",
      "Elapsed time: 9.46 seconds\n",
      "-------------------\n",
      "34969984\n",
      "Lineage recording in human cerebral organoids.\n",
      "Induced pluripotent stem cell (iPSC)-derived organoids provide models to study human organ\n",
      "development. Single-cell transcriptomics enable highly resolved descriptions of cell states within\n",
      "these systems; however, approaches are needed to directly measure lineage relationships. Here we\n",
      "establish iTracer, a lineage recorder that combines reporter barcodes with inducible CRISPR-Cas9\n",
      "scarring and is compatible with single-cell and spatial transcriptomics. We apply iTracer to explore\n",
      "clonality and lineage dynamics during cerebral organoid development and identify a time window of\n",
      "fate restriction as well as variation in neurogenic dynamics between progenitor neuron families. We\n",
      "also establish long-term four-dimensional light-sheet microscopy for spatial lineage recording in\n",
      "cerebral organoids and confirm regional clonality in the developing neuroepithelium. We incorporate\n",
      "gene perturbation (iTracer-perturb) and assess the effect of mosaic TSC2 mutations on cerebral\n",
      "organoid development. Our data shed light on how lineages and fates are established during cerebral\n",
      "organoid formation. More broadly, our techniques can be adapted in any iPSC-derived culture system\n",
      "to dissect lineage alterations during normal or perturbed development.\n",
      "['single-cell transcriptomics',\n",
      " 'spatial transcriptomics',\n",
      " 'four-dimensional light-sheet microscopy',\n",
      " 'long-term four-dimensional light-sheet microscopy',\n",
      " 'gene perturbation']\n",
      "Elapsed time: 12.66 seconds\n",
      "-------------------\n",
      "35027766\n",
      "Visualizing the native cellular organization by coupling cryofixation with expansion microscopy\n",
      "(Cryo-ExM).\n",
      "Cryofixation has proven to be the gold standard for efficient preservation of native cell\n",
      "ultrastructure compared to chemical fixation, but this approach is not widely used in fluorescence\n",
      "microscopy owing to implementation challenges. Here, we develop Cryo-ExM, a method that preserves\n",
      "native cellular organization by coupling cryofixation with expansion microscopy. This method\n",
      "bypasses artifacts associated with chemical fixation and its simplicity will contribute to its\n",
      "widespread use in super-resolution microscopy.\n",
      "['cryofixation', 'expansion microscopy', 'super-resolution microscopy']\n",
      "Elapsed time: 7.21 seconds\n",
      "-------------------\n",
      "35027767\n",
      "CellRank for directed single-cell fate mapping.\n",
      "Computational trajectory inference enables the reconstruction of cell state dynamics from single-\n",
      "cell RNA sequencing experiments. However, trajectory inference requires that the direction of a\n",
      "biological process is known, largely limiting its application to differentiating systems in normal\n",
      "development. Here, we present CellRank ( https://cellrank.org ) for single-cell fate mapping in\n",
      "diverse scenarios, including regeneration, reprogramming and disease, for which direction is\n",
      "unknown. Our approach combines the robustness of trajectory inference with directional information\n",
      "from RNA velocity, taking into account the gradual and stochastic nature of cellular fate decisions,\n",
      "as well as uncertainty in velocity vectors. On pancreas development data, CellRank automatically\n",
      "detects initial, intermediate and terminal populations, predicts fate potentials and visualizes\n",
      "continuous gene expression trends along individual lineages. Applied to lineage-traced cellular\n",
      "reprogramming data, predicted fate probabilities correctly recover reprogramming outcomes. CellRank\n",
      "also predicts a new dedifferentiation trajectory during postinjury lung regeneration, including\n",
      "previously unknown intermediate cell states, which we confirm experimentally.\n",
      "{'imagingTechniques': ['trajectoryInference',\n",
      "                       'RNA velocity',\n",
      "                       'cellularFateDecisions',\n",
      "                       'geneExpressionTrends',\n",
      "                       'lineage-traced cellular reprogramming',\n",
      "                       'postinjury lung regeneration']}\n",
      "Elapsed time: 11.65 seconds\n",
      "-------------------\n",
      "35102346\n",
      "Squidpy: a scalable framework for spatial omics analysis.\n",
      "Spatial omics data are advancing the study of tissue organization and cellular communication at an\n",
      "unprecedented scale. Flexible tools are required to store, integrate and visualize the large\n",
      "diversity of spatial omics data. Here, we present Squidpy, a Python framework that brings together\n",
      "tools from omics and image analysis to enable scalable description of spatial molecular data, such\n",
      "as transcriptome or multivariate proteins. Squidpy provides efficient infrastructure and numerous\n",
      "analysis methods that allow to efficiently store, manipulate and interactively visualize spatial\n",
      "omics data. Squidpy is extensible and can be interfaced with a variety of already existing libraries\n",
      "for the scalable analysis of spatial omics data.\n",
      "['transcriptome', 'multivariate proteins', 'image analysis']\n",
      "Elapsed time: 8.15 seconds\n",
      "-------------------\n",
      "35115715\n",
      "Reprogramming the piRNA pathway for multiplexed and transgenerational gene silencing in C. elegans.\n",
      "Single-guide RNAs can target exogenous CRISPR-Cas proteins to unique DNA locations, enabling genetic\n",
      "tools that are efficient, specific and scalable. Here we show that short synthetic guide Piwi-\n",
      "interacting RNAs (piRNAs) (21-nucleotide sg-piRNAs) expressed from extrachromosomal transgenes can,\n",
      "analogously, reprogram the endogenous piRNA pathway for gene-specific silencing in the hermaphrodite\n",
      "germline, sperm and embryos of Caenorhabditis elegans. piRNA-mediated interference ('piRNAi') is\n",
      "more efficient than RNAi and can be multiplexed, and auxin-mediated degradation of the piRNA-\n",
      "specific Argonaute PRG-1 allows conditional gene silencing. Target-specific silencing results in\n",
      "decreased messenger RNA levels, amplification of secondary small interfering RNAs and repressive\n",
      "chromatin modifications. Short (300 base pairs) piRNAi transgenes amplified from arrayed\n",
      "oligonucleotide pools also induce silencing, potentially making piRNAi highly scalable. We show that\n",
      "piRNAi can induce transgenerational epigenetic silencing of two endogenous genes (him-5 and him-8).\n",
      "Silencing is inherited for four to six generations after target-specific sg-piRNAs are lost, whereas\n",
      "depleting PRG-1 leads to essentially permanent epigenetic silencing.\n",
      "{'imagingTechniques': ['single-guide RNAs',\n",
      "                       'synthetic guide Piwi-interacting RNAs (piRNAs)',\n",
      "                       'piRNA-mediated interference (piRNAi)',\n",
      "                       'auxin-mediated degradation of the piRNA-specific '\n",
      "                       'Argonaute PRG-1',\n",
      "                       'target-specific silencing']}\n",
      "Elapsed time: 16.87 seconds\n",
      "-------------------\n",
      "35132243\n",
      "Spatially resolved isotope tracing reveals tissue metabolic activity.\n",
      "Isotope tracing has helped to determine the metabolic activities of organs. Methods to probe\n",
      "metabolic heterogeneity within organs are less developed. We couple stable-isotope-labeled nutrient\n",
      "infusion to matrix-assisted laser desorption ionization imaging mass spectrometry (iso-imaging) to\n",
      "quantitate metabolic activity in mammalian tissues in a spatially resolved manner. In the kidney, we\n",
      "visualize gluconeogenic flux and glycolytic flux in the cortex and medulla, respectively.\n",
      "Tricarboxylic acid cycle substrate usage differs across kidney regions; glutamine and citrate are\n",
      "used preferentially in the cortex and fatty acids are used in the medulla. In the brain, we observe\n",
      "spatial gradations in carbon inputs to the tricarboxylic acid cycle and glutamate under a ketogenic\n",
      "diet. In a carbohydrate-rich diet, glucose predominates throughout but in a ketogenic diet,\n",
      "3-hydroxybutyrate contributes most strongly in the hippocampus and least in the midbrain. Brain\n",
      "nitrogen sources also vary spatially; branched-chain amino acids contribute most in the midbrain,\n",
      "whereas ammonia contributes in the thalamus. Thus, iso-imaging can reveal the spatial organization\n",
      "of metabolic activity.\n",
      "{'iso-imaging': 1,\n",
      " 'matrix-assisted laser desorption ionization imaging mass spectrometry': 1}\n",
      "Elapsed time: 14.56 seconds\n",
      "-------------------\n",
      "35132244\n",
      "CR-I-TASSER: assemble protein structures from cryo-EM density maps using deep convolutional neural\n",
      "networks.\n",
      "Cryo-electron microscopy (cryo-EM) has become a leading approach for protein structure\n",
      "determination, but it remains challenging to accurately model atomic structures with cryo-EM density\n",
      "maps. We propose a hybrid method, CR-I-TASSER (cryo-EM iterative threading assembly refinement),\n",
      "which integrates deep neural-network learning with I-TASSER assembly simulations for automated cryo-\n",
      "EM structure determination. The method is benchmarked on 778 proteins with simulated and\n",
      "experimental density maps, where CR-I-TASSER constructs models with a correct fold (template\n",
      "modeling (TM) score >0.5) for 643 targets that is 64% higher than the best of some other de novo and\n",
      "refinement-based approaches on high-resolution data samples. Detailed data analyses showed that the\n",
      "main advantage of CR-I-TASSER lies in the deep learning-based Cα position prediction, which\n",
      "significantly improves the threading template quality and therefore boosts the accuracy of final\n",
      "models through optimized fragment assembly simulations. These results demonstrate a new avenue to\n",
      "determine cryo-EM protein structures with high accuracy and robustness covering various target types\n",
      "and density map resolutions.\n",
      "['cryo-electron microscopy',\n",
      " 'deep convolutional neural networks',\n",
      " 'threading assembly refinement',\n",
      " 'template modeling']\n",
      "Elapsed time: 11.79 seconds\n",
      "-------------------\n",
      "35132245\n",
      "NTR 2.0: a rationally engineered prodrug-converting enzyme with substantially enhanced efficacy for\n",
      "targeted cell ablation.\n",
      "Transgenic expression of bacterial nitroreductase (NTR) enzymes sensitizes eukaryotic cells to\n",
      "prodrugs such as metronidazole (MTZ), enabling selective cell-ablation paradigms that have expanded\n",
      "studies of cell function and regeneration in vertebrates. However, first-generation NTRs required\n",
      "confoundingly toxic prodrug treatments to achieve effective cell ablation, and some cell types have\n",
      "proven resistant. Here we used rational engineering and cross-species screening to develop an NTR\n",
      "variant, NTR 2.0, which exhibits ~100-fold improvement in MTZ-mediated cell-specific ablation\n",
      "efficacy, eliminating the need for near-toxic prodrug treatment regimens. NTR 2.0 therefore enables\n",
      "sustained cell-loss paradigms and ablation of previously resistant cell types. These properties\n",
      "permit enhanced interrogations of cell function, extended challenges to the regenerative capacities\n",
      "of discrete stem cell niches, and novel modeling of chronic degenerative diseases. Accordingly, we\n",
      "have created a series of bipartite transgenic reporter/effector resources to facilitate\n",
      "dissemination of NTR 2.0 to the research community.\n",
      "['none']\n",
      "Elapsed time: 12.00 seconds\n",
      "-------------------\n",
      "35145319\n",
      "VascuViz: a multimodality and multiscale imaging and visualization pipeline for vascular systems\n",
      "biology.\n",
      "Despite advances in imaging, image-based vascular systems biology has remained challenging because\n",
      "blood vessel data are often available only from a single modality or at a given spatial scale, and\n",
      "cross-modality data are difficult to integrate. Therefore, there is an exigent need for a\n",
      "multimodality pipeline that enables ex vivo vascular imaging with magnetic resonance imaging,\n",
      "computed tomography and optical microscopy of the same sample, while permitting imaging with\n",
      "complementary contrast mechanisms from the whole-organ to endothelial cell spatial scales. To\n",
      "achieve this, we developed 'VascuViz'-an easy-to-use method for simultaneous three-dimensional\n",
      "imaging and visualization of the vascular microenvironment using magnetic resonance imaging,\n",
      "computed tomography and optical microscopy in the same intact, unsectioned tissue. The VascuViz\n",
      "workflow permits multimodal imaging with a single labeling step using commercial reagents and is\n",
      "compatible with diverse tissue types and protocols. VascuViz's interdisciplinary utility in\n",
      "conjunction with new data visualization approaches opens up new vistas in image-based vascular\n",
      "systems biology.\n",
      "['Magnetic Resonance Imaging', 'Computed Tomography', 'Optical Microscopy']\n",
      "Elapsed time: 11.39 seconds\n",
      "-------------------\n",
      "35145320\n",
      "A genetically encoded sensor for in vivo imaging of orexin neuropeptides.\n",
      "Orexins (also called hypocretins) are hypothalamic neuropeptides that carry out essential functions\n",
      "in the central nervous system; however, little is known about their release and range of action in\n",
      "vivo owing to the limited resolution of current detection technologies. Here we developed a\n",
      "genetically encoded orexin sensor (OxLight1) based on the engineering of circularly permutated green\n",
      "fluorescent protein into the human type-2 orexin receptor. In mice OxLight1 detects optogenetically\n",
      "evoked release of endogenous orexins in vivo with high sensitivity. Photometry recordings of\n",
      "OxLight1 in mice show rapid orexin release associated with spontaneous running behavior, acute\n",
      "stress and sleep-to-wake transitions in different brain areas. Moreover, two-photon imaging of\n",
      "OxLight1 reveals orexin release in layer 2/3 of the mouse somatosensory cortex during emergence from\n",
      "anesthesia. Thus, OxLight1 enables sensitive and direct optical detection of orexin neuropeptides\n",
      "with high spatiotemporal resolution in living animals.\n",
      "['photometry recordings', 'two-photon imaging']\n",
      "Elapsed time: 10.34 seconds\n",
      "-------------------\n",
      "35228725\n",
      "Targeted multicolor in vivo imaging over 1,000 nm enabled by nonamethine cyanines.\n",
      "Recent progress has shown that using wavelengths between 1,000 and 2,000 nm, referred to as the\n",
      "shortwave-infrared or near-infrared (NIR)-II range, can enable high-resolution in vivo imaging at\n",
      "depths not possible with conventional optical wavelengths. However, few bioconjugatable probes of\n",
      "the type that have proven invaluable for multiplexed imaging in the visible and NIR range are\n",
      "available for imaging these wavelengths. Using rational design, we have generated persulfonated\n",
      "indocyanine dyes with absorbance maxima at 872 and 1,072 nm through catechol-ring and aryl-ring\n",
      "fusion, respectively, onto the nonamethine scaffold. Multiplexed two-color and three-color in vivo\n",
      "imaging using monoclonal antibody and dextran conjugates in several tumor models illustrate the\n",
      "benefits of concurrent labeling of the tumor and healthy surrounding tissue and lymphatics. These\n",
      "efforts are enabled by complementary advances in a custom-built NIR/shortwave-infrared imaging setup\n",
      "and software package for multicolor real-time imaging.\n",
      "['multiplexed two-color imaging',\n",
      " 'three-color imaging',\n",
      " 'in vivo imaging',\n",
      " 'NIR-II range imaging',\n",
      " 'shortwave-infrared imaging',\n",
      " 'near-infrared imaging',\n",
      " 'custom-built NIR/shortwave-infrared imaging setup']\n",
      "Elapsed time: 13.71 seconds\n",
      "-------------------\n",
      "35277709\n",
      "Isotropic super-resolution light-sheet microscopy of dynamic intracellular structures at subsecond\n",
      "timescales.\n",
      "Long-term visualization of the dynamic interactions between intracellular structures throughout the\n",
      "three-dimensional space of whole live cells is essential to better understand their functions, but\n",
      "this task remains challenging due to the limitations of existing three-dimensional fluorescence\n",
      "microscopy techniques, such as an insufficient axial resolution, low volumetric imaging rate and\n",
      "photobleaching. Here, we present the combination of a progressive deep-learning super-resolution\n",
      "strategy with a double-ring-modulated selective plane illumination microscopy design capable of\n",
      "visualizing the dynamics of intracellular structures in live cells for hours at an isotropic spatial\n",
      "resolution of roughly 100 nm in three dimensions at speeds up to roughly 17 Hz. Using this approach,\n",
      "we reveal the complex spatial relationships and interactions between endoplasmic reticulum (ER) and\n",
      "mitochondria throughout live cells, providing new insights into ER-mediated mitochondrial division.\n",
      "We also examined the motion of Drp1 oligomers involved in mitochondrial fission and revealed the\n",
      "dynamic interactions between Drp1 and mitochondria in three dimensions.\n",
      "['progressive deep-learning super-resolution strategy',\n",
      " 'double-ring-modulated selective plane illumination microscopy design',\n",
      " 'isotropic spatial resolution of roughly 100\\u2009nm in three dimensions',\n",
      " 'speeds up to roughly 17\\u2009Hz']\n",
      "Elapsed time: 12.27 seconds\n",
      "-------------------\n",
      "35314838\n",
      "DaXi-high-resolution, large imaging volume and multi-view single-objective light-sheet microscopy.\n",
      "The promise of single-objective light-sheet microscopy is to combine the convenience of standard\n",
      "single-objective microscopes with the speed, coverage, resolution and gentleness of light-sheet\n",
      "microscopes. We present DaXi, a single-objective light-sheet microscope design based on oblique\n",
      "plane illumination that achieves: (1) a wider field of view and high-resolution imaging via a custom\n",
      "remote focusing objective; (2) fast volumetric imaging over larger volumes without compromising\n",
      "image quality or necessitating tiled acquisition; (3) fuller image coverage for large samples via\n",
      "multi-view imaging and (4) higher throughput multi-well imaging via remote coverslip placement. Our\n",
      "instrument achieves a resolution of 450 nm laterally and 2 μm axially over an imaging volume of\n",
      "3,000 × 800 × 300 μm. We demonstrate the speed, field of view, resolution and versatility of our\n",
      "instrument by imaging various systems, including Drosophila egg chamber development, zebrafish\n",
      "whole-brain activity and zebrafish embryonic development - up to nine embryos at a time.\n",
      "['single-objective light-sheet microscopy',\n",
      " 'remote focusing objective',\n",
      " 'oblique plane illumination',\n",
      " 'multi-view single-objective light-sheet microscopy',\n",
      " 'tiled acquisition',\n",
      " 'remote coverslip placement']\n",
      "Elapsed time: 15.22 seconds\n",
      "-------------------\n",
      "35347320\n",
      "Detecting and correcting false transients in calcium imaging.\n",
      "Population recordings of calcium activity are a major source of insight into neural function. Large\n",
      "datasets require automated processing, but this can introduce errors that are difficult to detect.\n",
      "Here we show that popular time course-estimation algorithms often contain substantial misattribution\n",
      "errors affecting 10-20% of transients. Misattribution, in which fluorescence is ascribed to the\n",
      "wrong cell, arises when overlapping cells and processes are imperfectly defined or not identified.\n",
      "To diagnose misattribution, we develop metrics and visualization tools for evaluating large\n",
      "datasets. To correct time courses, we introduce a robust estimator that explicitly accounts for\n",
      "contaminating signals. In one hippocampal dataset, removing contamination reduced the number of\n",
      "place cells by 15%, and 19% of place fields shifted by over 10 cm. Our methods are compatible with\n",
      "other cell-finding techniques, empowering users to diagnose and correct a potentially widespread\n",
      "problem that could alter scientific conclusions.\n",
      "{'imaging_techniques': ['calcium imaging',\n",
      "                        'time course-estimation algorithms',\n",
      "                        'cell-finding techniques']}\n",
      "Elapsed time: 9.86 seconds\n",
      "-------------------\n",
      "35347322\n",
      "HYBRiD: hydrogel-reinforced DISCO for clearing mammalian bodies.\n",
      "The recent development of solvent- and polymer-based brain-clearing techniques has advanced our\n",
      "ability to visualize the mammalian nervous system in three dimensions. However, it remains\n",
      "challenging to image the mammalian body en bloc. Here we developed HYBRiD (hydrogel-based\n",
      "reinforcement of three-dimensional imaging solvent-cleared organs (DISCO)), by recombining\n",
      "components of organic- and polymer-based clearing pipelines. We achieved high transparency and\n",
      "protein retention, as well as compatibility with direct fluorescent imaging and immunostaining in\n",
      "cleared mammalian bodies. Using parvalbumin- and somatostatin-Cre models, we demonstrated the\n",
      "utility of HYBRiD for whole-body imaging of genetically encoded fluorescent reporters without\n",
      "antibody enhancement of signals in newborn and juvenile mice. Using K18-hACE2 transgenic mice,\n",
      "HYBRiD enabled perfusion-free clearing and visualization of SARS-CoV-2 infection in a whole mouse\n",
      "chest, revealing macroscopic and microscopic features of viral pathology in the same sample. HYBRiD\n",
      "offers a simple and universal solution to visualize large heterogeneous body parts or entire animals\n",
      "for basic and translational research.\n",
      "['solvent-based brain-clearing techniques',\n",
      " 'polymer-based brain-clearing techniques',\n",
      " 'direct fluorescent imaging',\n",
      " 'immunostaining']\n",
      "Elapsed time: 13.07 seconds\n",
      "-------------------\n",
      "35379947\n",
      "SLEAP: A deep learning system for multi-animal pose tracking.\n",
      "The desire to understand how the brain generates and patterns behavior has driven rapid\n",
      "methodological innovation in tools to quantify natural animal behavior. While advances in deep\n",
      "learning and computer vision have enabled markerless pose estimation in individual animals,\n",
      "extending these to multiple animals presents unique challenges for studies of social behaviors or\n",
      "animals in their natural environments. Here we present Social LEAP Estimates Animal Poses (SLEAP), a\n",
      "machine learning system for multi-animal pose tracking. This system enables versatile workflows for\n",
      "data labeling, model training and inference on previously unseen data. SLEAP features an accessible\n",
      "graphical user interface, a standardized data model, a reproducible configuration system, over 30\n",
      "model architectures, two approaches to part grouping and two approaches to identity tracking. We\n",
      "applied SLEAP to seven datasets across flies, bees, mice and gerbils to systematically evaluate each\n",
      "approach and architecture, and we compare it with other existing approaches. SLEAP achieves greater\n",
      "accuracy and speeds of more than 800 frames per second, with latencies of less than 3.5 ms at full\n",
      "1,024 × 1,024 image resolution. This makes SLEAP usable for real-time applications, which we\n",
      "demonstrate by controlling the behavior of one animal on the basis of the tracking and detection of\n",
      "social interactions with another animal.\n",
      "Could not find json-formatted data in: Here is a list of imaging techniques or technologies described in the abstract:\n",
      "\n",
      "• markerless pose estimation\n",
      "• computer vision\n",
      "• deep learning\n",
      "• image resolution\n",
      "35396487\n",
      "DiMeLo-seq: a long-read, single-molecule method for mapping protein-DNA interactions genome wide.\n",
      "Studies of genome regulation routinely use high-throughput DNA sequencing approaches to determine\n",
      "where specific proteins interact with DNA, and they rely on DNA amplification and short-read\n",
      "sequencing, limiting their quantitative application in complex genomic regions. To address these\n",
      "limitations, we developed directed methylation with long-read sequencing (DiMeLo-seq), which uses\n",
      "antibody-tethered enzymes to methylate DNA near a target protein's binding sites in situ. These\n",
      "exogenous methylation marks are then detected simultaneously with endogenous CpG methylation on\n",
      "unamplified DNA using long-read, single-molecule sequencing technologies. We optimized and\n",
      "benchmarked DiMeLo-seq by mapping chromatin-binding proteins and histone modifications across the\n",
      "human genome. Furthermore, we identified where centromere protein A localizes within highly\n",
      "repetitive regions that were unmappable with short sequencing reads, and we estimated the density of\n",
      "centromere protein A molecules along single chromatin fibers. DiMeLo-seq is a versatile method that\n",
      "provides multimodal, genome-wide information for investigating protein-DNA interactions.\n",
      "['long-read sequencing',\n",
      " 'single-molecule sequencing',\n",
      " 'directed methylation',\n",
      " 'antibody-tethered enzymes']\n",
      "Elapsed time: 11.79 seconds\n",
      "-------------------\n",
      "35414125\n",
      "Multi-animal pose estimation, identification and tracking with DeepLabCut.\n",
      "Estimating the pose of multiple animals is a challenging computer vision problem: frequent\n",
      "interactions cause occlusions and complicate the association of detected keypoints to the correct\n",
      "individuals, as well as having highly similar looking animals that interact more closely than in\n",
      "typical multi-human scenarios. To take up this challenge, we build on DeepLabCut, an open-source\n",
      "pose estimation toolbox, and provide high-performance animal assembly and tracking-features required\n",
      "for multi-animal scenarios. Furthermore, we integrate the ability to predict an animal's identity to\n",
      "assist tracking (in case of occlusions). We illustrate the power of this framework with four\n",
      "datasets varying in complexity, which we release to serve as a benchmark for future algorithm\n",
      "development.\n",
      "['pose estimation', 'animal assembly', 'tracking', 'identity prediction']\n",
      "Elapsed time: 7.73 seconds\n",
      "-------------------\n",
      "35501384\n",
      "Sub-3-Å cryo-EM structure of RNA enabled by engineered homomeric self-assembly.\n",
      "High-resolution structural studies are essential for understanding the folding and function of\n",
      "diverse RNAs. Herein, we present a nanoarchitectural engineering strategy for efficient structural\n",
      "determination of RNA-only structures using single-particle cryogenic electron microscopy (cryo-EM).\n",
      "This strategy-ROCK (RNA oligomerization-enabled cryo-EM via installing kissing loops)-involves\n",
      "installing kissing-loop sequences onto the functionally nonessential stems of RNAs for homomeric\n",
      "self-assembly into closed rings with multiplied molecular weights and mitigated structural\n",
      "flexibility. ROCK enables cryo-EM reconstruction of the Tetrahymena group I intron at 2.98-Å\n",
      "resolution overall (2.85 Å for the core), allowing de novo model building of the complete RNA,\n",
      "including the previously unknown peripheral domains. ROCK is further applied to two smaller RNAs-the\n",
      "Azoarcus group I intron and the FMN riboswitch, revealing the conformational change of the former\n",
      "and the bound ligand in the latter. ROCK holds promise to greatly facilitate the use of cryo-EM in\n",
      "RNA structural studies.\n",
      "['cryogenic electron microscopy', 'cryo-EM', 'single-particle cryo-EM']\n",
      "Elapsed time: 11.41 seconds\n",
      "-------------------\n",
      "35501386\n",
      "Fluorogenic DNA-PAINT for faster, low-background super-resolution imaging.\n",
      "DNA-based points accumulation for imaging in nanoscale topography (DNA-PAINT) is a powerful super-\n",
      "resolution microscopy method that can acquire high-fidelity images at nanometer resolution. It\n",
      "suffers, however, from high background and slow imaging speed, both of which can be attributed to\n",
      "the presence of unbound fluorophores in solution. Here we present two-color fluorogenic DNA-PAINT,\n",
      "which uses improved imager probe and docking strand designs to solve these problems. These self-\n",
      "quenching single-stranded DNA probes are conjugated with a fluorophore and quencher at the\n",
      "terminals, which permits an increase in fluorescence by up to 57-fold upon binding and unquenching.\n",
      "In addition, the engineering of base pair mismatches between the fluorogenic imager probes and\n",
      "docking strands allowed us to achieve both high fluorogenicity and the fast binding kinetics\n",
      "required for fast imaging. We demonstrate a 26-fold increase in imaging speed over regular DNA-PAINT\n",
      "and show that our new implementation enables three-dimensional super-resolution DNA-PAINT imaging\n",
      "without optical sectioning.\n",
      "['DNA-PAINT',\n",
      " 'super-resolution microscopy',\n",
      " 'fluorogenic DNA-PAINT',\n",
      " 'two-color fluorogenic DNA-PAINT',\n",
      " 'imager probe',\n",
      " 'docking strands']\n",
      "Elapsed time: 12.78 seconds\n",
      "-------------------\n",
      "35534632\n",
      "Precision size and refractive index analysis of weakly scattering nanoparticles in polydispersions.\n",
      "Characterization of the size and material properties of particles in liquid suspensions is in very\n",
      "high demand, for example, in the analysis of colloidal samples or of bodily fluids such as urine or\n",
      "blood plasma. However, existing methods are limited in their ability to decipher the constituents of\n",
      "realistic samples. Here we introduce iNTA as a new method that combines interferometric detection of\n",
      "scattering with nanoparticle tracking analysis to reach unprecedented sensitivity and precision in\n",
      "determining the size and refractive index distributions of nanoparticles in suspensions. After\n",
      "benchmarking iNTA with samples of colloidal gold, we present its remarkable ability to resolve the\n",
      "constituents of various multicomponent and polydisperse samples of known origin. Furthermore, we\n",
      "showcase the method by elucidating the refractive index and size distributions of extracellular\n",
      "vesicles from Leishmania parasites and human urine. The current performance of iNTA already enables\n",
      "advances in several important applications, but we also discuss possible improvements.\n",
      "['interferometric detection of scattering', 'nanoparticle tracking analysis']\n",
      "Elapsed time: 10.00 seconds\n",
      "-------------------\n",
      "35545712\n",
      "Correction of multiple-blinking artifacts in photoactivated localization microscopy.\n",
      "Photoactivated localization microscopy (PALM) produces an array of localization coordinates by means\n",
      "of photoactivatable fluorescent proteins. However, observations are subject to fluorophore multiple\n",
      "blinking and each protein is included in the dataset an unknown number of times at different\n",
      "positions, due to localization error. This causes artificial clustering to be observed in the data.\n",
      "We present a 'model-based correction' (MBC) workflow using calibration-free estimation of blinking\n",
      "dynamics and model-based clustering to produce a corrected set of localization coordinates\n",
      "representing the true underlying fluorophore locations with enhanced localization precision,\n",
      "outperforming the state of the art. The corrected data can be reliably tested for spatial randomness\n",
      "or analyzed by other clustering approaches, and descriptors such as the absolute number of\n",
      "fluorophores per cluster are now quantifiable, which we validate with simulated data and\n",
      "experimental data with known ground truth. Using MBC, we confirm that the adapter protein, the\n",
      "linker for activation of T cells, is clustered at the T cell immunological synapse.\n",
      "['PALM',\n",
      " 'photoactivatable fluorescent proteins',\n",
      " 'model-based correction',\n",
      " 'calibration-free estimation',\n",
      " 'blinking dynamics',\n",
      " 'localization error',\n",
      " 'clustering approaches',\n",
      " 'spatial randomness',\n",
      " 'simulated data',\n",
      " 'experimental data']\n",
      "Elapsed time: 11.56 seconds\n",
      "-------------------\n",
      "35545715\n",
      "A hybrid open-top light-sheet microscope for versatile multi-scale imaging of cleared tissues.\n",
      "Light-sheet microscopy has emerged as the preferred means for high-throughput volumetric imaging of\n",
      "cleared tissues. However, there is a need for a flexible system that can address imaging\n",
      "applications with varied requirements in terms of resolution, sample size, tissue-clearing protocol,\n",
      "and transparent sample-holder material. Here, we present a 'hybrid' system that combines a unique\n",
      "non-orthogonal dual-objective and conventional (orthogonal) open-top light-sheet (OTLS) architecture\n",
      "for versatile multi-scale volumetric imaging. We demonstrate efficient screening and targeted sub-\n",
      "micrometer imaging of sparse axons within an intact, cleared mouse brain. The same system enables\n",
      "high-throughput automated imaging of multiple specimens, as spotlighted by a quantitative multi-\n",
      "scale analysis of brain metastases. Compared with existing academic and commercial light-sheet\n",
      "microscopy systems, our hybrid OTLS system provides a unique combination of versatility and\n",
      "performance necessary to satisfy the diverse requirements of a growing number of cleared-tissue\n",
      "imaging applications.\n",
      "['light-sheet microscopy',\n",
      " 'open-top light-sheet (OTLS) architecture',\n",
      " 'non-orthogonal dual-objective']\n",
      "Elapsed time: 11.07 seconds\n",
      "-------------------\n",
      "35577958\n",
      "Optimal precision and accuracy in 4Pi-STORM using dynamic spline PSF models.\n",
      "Coherent fluorescence imaging with two objective lenses (4Pi detection) enables single-molecule\n",
      "localization microscopy with sub-10 nm spatial resolution in three dimensions. Despite its\n",
      "outstanding sensitivity, wider application of this technique has been hindered by complex\n",
      "instrumentation and the challenging nature of the data analysis. Here we report the development of a\n",
      "4Pi-STORM microscope, which obtains optimal resolution and accuracy by modeling the 4Pi point spread\n",
      "function (PSF) dynamically while also using a simpler optical design. Dynamic spline PSF models\n",
      "incorporate fluctuations in the modulation phase of the experimentally determined PSF, capturing the\n",
      "temporal evolution of the optical system. Our method reaches the theoretical limits for precision\n",
      "and minimizes phase-wrapping artifacts by making full use of the information content of the data.\n",
      "4Pi-STORM achieves a near-isotropic three-dimensional localization precision of 2-3 nm, and we\n",
      "demonstrate its capabilities by investigating protein and nucleic acid organization in primary\n",
      "neurons and mammalian mitochondria.\n",
      "['4Pi-STORM',\n",
      " 'coherent fluorescence imaging',\n",
      " 'single-molecule localization microscopy',\n",
      " 'dynamic spline PSF models']\n",
      "Elapsed time: 10.82 seconds\n",
      "-------------------\n",
      "35606446\n",
      "Single-domain near-infrared protein provides a scaffold for antigen-dependent fluorescent\n",
      "nanobodies.\n",
      "Small near-infrared (NIR) fluorescent proteins (FPs) are much needed as protein tags for imaging\n",
      "applications. We developed a 17 kDa NIR FP, called miRFP670nano3, which brightly fluoresces in\n",
      "mammalian cells and enables deep-brain imaging. By exploring miRFP670nano3 as an internal tag, we\n",
      "engineered 32 kDa NIR fluorescent nanobodies, termed NIR-Fbs, whose stability and fluorescence\n",
      "strongly depend on the presence of specific intracellular antigens. NIR-Fbs allowed background-free\n",
      "visualization of endogenous proteins, detection of viral antigens, labeling of cells expressing\n",
      "target molecules and identification of double-positive cell populations with bispecific NIR-Fbs\n",
      "against two antigens. Applying NIR-Fbs as destabilizing fusion partners, we developed molecular\n",
      "tools for directed degradation of targeted proteins, controllable protein expression and modulation\n",
      "of enzymatic activities. Altogether, NIR-Fbs enable the detection and manipulation of a variety of\n",
      "cellular processes based on the intracellular protein profile.\n",
      "['fluorescent nanobodies',\n",
      " 'deep-brain imaging',\n",
      " 'directed degradation of targeted proteins',\n",
      " 'controllable protein expression',\n",
      " 'modulation of enzymatic activities']\n",
      "Elapsed time: 12.74 seconds\n",
      "-------------------\n",
      "35637302\n",
      "Ab initio phasing macromolecular structures using electron-counted MicroED data.\n",
      "Structures of two globular proteins were determined ab initio using microcrystal electron\n",
      "diffraction (MicroED) data that were collected on a direct electron detector in counting mode.\n",
      "Microcrystals were identified using a scanning electron microscope (SEM) and thinned with a focused\n",
      "ion beam (FIB) to produce crystalline lamellae of ideal thickness. Continuous-rotation data were\n",
      "collected using an ultra-low exposure rate to enable electron counting in diffraction. For the first\n",
      "sample, triclinic lysozyme extending to a resolution of 0.87 Å, an ideal helical fragment of only\n",
      "three alanine residues provided initial phases. These phases were improved using density\n",
      "modification, allowing the entire atomic structure to be built automatically. A similar approach was\n",
      "successful on a second macromolecular sample, proteinase K, which is much larger and diffracted to a\n",
      "resolution of 1.5 Å. These results demonstrate that macromolecules can be determined to sub-ångström\n",
      "resolution by MicroED and that ab initio phasing can be successfully applied to counting data.\n",
      "['scanning electron microscope (SEM)',\n",
      " 'focused ion beam (FIB)',\n",
      " 'microcrystal electron diffraction (MicroED)',\n",
      " 'ultra-low exposure rate',\n",
      " 'density modification']\n",
      "Elapsed time: 11.95 seconds\n",
      "-------------------\n",
      "35637303\n",
      "Label-free nanofluidic scattering microscopy of size and mass of single diffusing molecules and\n",
      "nanoparticles.\n",
      "Label-free characterization of single biomolecules aims to complement fluorescence microscopy in\n",
      "situations where labeling compromises data interpretation, is technically challenging or even\n",
      "impossible. However, existing methods require the investigated species to bind to a surface to be\n",
      "visible, thereby leaving a large fraction of analytes undetected. Here, we present nanofluidic\n",
      "scattering microscopy (NSM), which overcomes these limitations by enabling label-free, real-time\n",
      "imaging of single biomolecules diffusing inside a nanofluidic channel. NSM facilitates accurate\n",
      "determination of molecular weight from the measured optical contrast and of the hydrodynamic radius\n",
      "from the measured diffusivity, from which information about the conformational state can be\n",
      "inferred. Furthermore, we demonstrate its applicability to the analysis of a complex biofluid, using\n",
      "conditioned cell culture medium containing extracellular vesicles as an example. We foresee the\n",
      "application of NSM to monitor conformational changes, aggregation and interactions of single\n",
      "biomolecules, and to analyze single-cell secretomes.\n",
      "['nanofluidic scattering microscopy', 'fluorescence microscopy']\n",
      "Elapsed time: 10.26 seconds\n",
      "-------------------\n",
      "35637304\n",
      "MSNovelist: de novo structure generation from mass spectra.\n",
      "Current methods for structure elucidation of small molecules rely on finding similarity with spectra\n",
      "of known compounds, but do not predict structures de novo for unknown compound classes. We present\n",
      "MSNovelist, which combines fingerprint prediction with an encoder-decoder neural network to generate\n",
      "structures de novo solely from tandem mass spectrometry (MS<sup>2</sup>) spectra. In an evaluation\n",
      "with 3,863 MS<sup>2</sup> spectra from the Global Natural Product Social Molecular Networking site,\n",
      "MSNovelist predicted 25% of structures correctly on first rank, retrieved 45% of structures overall\n",
      "and reproduced 61% of correct database annotations, without having ever seen the structure in the\n",
      "training phase. Similarly, for the CASMI 2016 challenge, MSNovelist correctly predicted 26% and\n",
      "retrieved 57% of structures, recovering 64% of correct database annotations. Finally, we illustrate\n",
      "the application of MSNovelist in a bryophyte MS<sup>2</sup> dataset, in which de novo structure\n",
      "prediction substantially outscored the best database candidate for seven spectra. MSNovelist is\n",
      "ideally suited to complement library-based annotation in the case of poorly represented analyte\n",
      "classes and novel compounds.\n",
      "['fingerprint prediction',\n",
      " 'encoder-decoder neural network',\n",
      " 'tandem mass spectrometry (MS<sup>2</sup>)',\n",
      " 'mass spectrometry (MS)']\n",
      "Elapsed time: 13.35 seconds\n",
      "-------------------\n",
      "35654950\n",
      "TrackMate 7: integrating state-of-the-art segmentation algorithms into tracking pipelines.\n",
      "TrackMate is an automated tracking software used to analyze bioimages and is distributed as a Fiji\n",
      "plugin. Here, we introduce a new version of TrackMate. TrackMate 7 is built to address the broad\n",
      "spectrum of modern challenges researchers face by integrating state-of-the-art segmentation\n",
      "algorithms into tracking pipelines. We illustrate qualitatively and quantitatively that these new\n",
      "capabilities function effectively across a wide range of bio-imaging experiments.\n",
      "['segmentation algorithms', 'tracking pipelines']\n",
      "Elapsed time: 6.62 seconds\n",
      "-------------------\n",
      "35654951\n",
      "Identification of cell types in multiplexed in situ images by combining protein expression and\n",
      "spatial information using CELESTA.\n",
      "Advances in multiplexed in situ imaging are revealing important insights in spatial biology.\n",
      "However, cell type identification remains a major challenge in imaging analysis, with most existing\n",
      "methods involving substantial manual assessment and subjective decisions for thousands of cells. We\n",
      "developed an unsupervised machine learning algorithm, CELESTA, which identifies the cell type of\n",
      "each cell, individually, using the cell's marker expression profile and, when needed, its spatial\n",
      "information. We demonstrate the performance of CELESTA on multiplexed immunofluorescence images of\n",
      "colorectal cancer and head and neck squamous cell carcinoma (HNSCC). Using the cell types identified\n",
      "by CELESTA, we identify tissue architecture associated with lymph node metastasis in HNSCC, and\n",
      "validate our findings in an independent cohort. By coupling our spatial analysis with single-cell\n",
      "RNA-sequencing data on proximal sections of the same specimens, we identify cell-cell crosstalk\n",
      "associated with lymph node metastasis, demonstrating the power of CELESTA to facilitate\n",
      "identification of clinically relevant interactions.\n",
      "['multiplexed in situ imaging',\n",
      " 'immunofluorescence',\n",
      " 'single-cell RNA-sequencing']\n",
      "Elapsed time: 11.26 seconds\n",
      "-------------------\n",
      "35689029\n",
      "ASLPrep: a platform for processing of arterial spin labeled MRI and quantification of regional brain\n",
      "perfusion.\n",
      "Arterial spin labeled (ASL) magnetic resonance imaging (MRI) is the primary method for noninvasively\n",
      "measuring regional brain perfusion in humans. We introduce ASLPrep, a suite of software pipelines\n",
      "that ensure the reproducible and generalizable processing of ASL MRI data.\n",
      "['ASL MRI', 'MRI']\n",
      "Elapsed time: 6.66 seconds\n",
      "-------------------\n",
      "35697835\n",
      "Automated high-speed 3D imaging of organoid cultures with multi-scale phenotypic quantification.\n",
      "Current imaging approaches limit the ability to perform multi-scale characterization of three-\n",
      "dimensional (3D) organotypic cultures (organoids) in large numbers. Here, we present an automated\n",
      "multi-scale 3D imaging platform synergizing high-density organoid cultures with rapid and live 3D\n",
      "single-objective light-sheet imaging. It is composed of disposable microfabricated organoid culture\n",
      "chips, termed JeWells, with embedded optical components and a laser beam-steering unit coupled to a\n",
      "commercial inverted microscope. It permits streamlining organoid culture and high-content 3D imaging\n",
      "on a single user-friendly instrument with minimal manipulations and a throughput of 300 organoids\n",
      "per hour. We demonstrate that the large number of 3D stacks that can be collected via our platform\n",
      "allows training deep learning-based algorithms to quantify morphogenetic organizations of organoids\n",
      "at multi-scales, ranging from the subcellular scale to the whole organoid level. We validated the\n",
      "versatility and robustness of our approach on intestine, hepatic, neuroectoderm organoids and\n",
      "oncospheres.\n",
      "['light-sheet imaging',\n",
      " 'single-objective light-sheet imaging',\n",
      " 'deep learning-based algorithms']\n",
      "Elapsed time: 11.36 seconds\n",
      "-------------------\n",
      "35739310\n",
      "Multiplexed bioluminescence microscopy via phasor analysis.\n",
      "Bioluminescence imaging with luciferase-luciferin pairs is a well-established technique for\n",
      "visualizing biological processes across tissues and whole organisms. Applications at the microscale,\n",
      "by contrast, have been hindered by a lack of detection platforms and easily resolved probes. We\n",
      "addressed this limitation by combining bioluminescence with phasor analysis, a method commonly used\n",
      "to distinguish spectrally similar fluorophores. We built a camera-based microscope equipped with\n",
      "special optical filters to directly assign phasor locations to unique luciferase-luciferin pairs.\n",
      "Six bioluminescent reporters were easily resolved in live cells, and the readouts were quantitative\n",
      "and instantaneous. Multiplexed imaging was also performed over extended time periods. Bioluminescent\n",
      "phasor further provided direct measures of resonance energy transfer in single cells, setting the\n",
      "stage for dynamic measures of cellular and molecular features. The merger of bioluminescence with\n",
      "phasor analysis fills a long-standing void in imaging capabilities, and will bolster future efforts\n",
      "to visualize biological events in real time and over multiple length scales.\n",
      "['bioluminescence imaging',\n",
      " 'phasor analysis',\n",
      " 'camera-based microscope',\n",
      " 'special optical filters']\n",
      "Elapsed time: 11.19 seconds\n",
      "-------------------\n",
      "35817938\n",
      "Integrative genome modeling platform reveals essentiality of rare contact events in 3D genome\n",
      "organizations.\n",
      "A multitude of sequencing-based and microscopy technologies provide the means to unravel the\n",
      "relationship between the three-dimensional organization of genomes and key regulatory processes of\n",
      "genome function. Here, we develop a multimodal data integration approach to produce populations of\n",
      "single-cell genome structures that are highly predictive for nuclear locations of genes and nuclear\n",
      "bodies, local chromatin compaction and spatial segregation of functionally related chromatin. We\n",
      "demonstrate that multimodal data integration can compensate for systematic errors in some of the\n",
      "data and can greatly increase accuracy and coverage of genome structure models. We also show that\n",
      "alternative combinations of different orthogonal data sources can converge to models with similar\n",
      "predictive power. Moreover, our study reveals the key contributions of low-frequency ('rare')\n",
      "interchromosomal contacts to accurately predicting the global nuclear architecture, including the\n",
      "positioning of genes and chromosomes. Overall, our results highlight the benefits of multimodal data\n",
      "integration for genome structure analysis, available through the Integrative Genome Modeling\n",
      "software package.\n",
      "['sequencing-based technologies',\n",
      " 'microscopy technologies',\n",
      " 'multimodal data integration approach']\n",
      "Elapsed time: 10.58 seconds\n",
      "-------------------\n",
      "35879607\n",
      "Directed evolution of adeno-associated virus for efficient gene delivery to microglia.\n",
      "As the resident immune cells in the central nervous system (CNS), microglia orchestrate immune\n",
      "responses and dynamically sculpt neural circuits in the CNS. Microglial dysfunction and mutations of\n",
      "microglia-specific genes have been implicated in many diseases of the CNS. Developing effective and\n",
      "safe vehicles for transgene delivery into microglia will facilitate the studies of microglia biology\n",
      "and microglia-associated disease mechanisms. Here, we report the discovery of adeno-associated virus\n",
      "(AAV) variants that mediate efficient in vitro and in vivo microglial transduction via directed\n",
      "evolution of the AAV capsid protein. These AAV-cMG and AAV-MG variants are capable of delivering\n",
      "various genetic payloads into microglia with high efficiency, and enable sufficient transgene\n",
      "expression to support fluorescent labeling, Ca<sup>2+</sup> and neurotransmitter imaging and genome\n",
      "editing in microglia in vivo. Furthermore, single-cell RNA sequencing shows that the AAV-MG variants\n",
      "mediate in vivo transgene delivery without inducing microglia immune activation. These AAV variants\n",
      "should facilitate the use of various genetically encoded sensors and effectors in the study of\n",
      "microglia-related biology.\n",
      "['single-cell RNA sequencing',\n",
      " 'fluorescent labeling',\n",
      " 'Ca2+ imaging',\n",
      " 'neurotransmitter imaging',\n",
      " 'genome editing']\n",
      "Elapsed time: 13.10 seconds\n",
      "-------------------\n",
      "35879608\n",
      "Self-supervised deep learning encodes high-resolution features of protein subcellular localization.\n",
      "Explaining the diversity and complexity of protein localization is essential to fully understand\n",
      "cellular architecture. Here we present cytoself, a deep-learning approach for fully self-supervised\n",
      "protein localization profiling and clustering. Cytoself leverages a self-supervised training scheme\n",
      "that does not require preexisting knowledge, categories or annotations. Training cytoself on images\n",
      "of 1,311 endogenously labeled proteins from the OpenCell database reveals a highly resolved protein\n",
      "localization atlas that recapitulates major scales of cellular organization, from coarse classes,\n",
      "such as nuclear and cytoplasmic, to the subtle localization signatures of individual protein\n",
      "complexes. We quantitatively validate cytoself's ability to cluster proteins into organelles and\n",
      "protein complexes, showing that cytoself outperforms previous self-supervised approaches. Moreover,\n",
      "to better understand the inner workings of our model, we dissect the emergent features from which\n",
      "our clustering is derived, interpret them in the context of the fluorescence images, and analyze the\n",
      "performance contributions of each component of our approach.\n",
      "['fluorescence images', 'OpenCell database']\n",
      "Elapsed time: 10.37 seconds\n",
      "-------------------\n",
      "35915194\n",
      "Photoswitching fingerprint analysis bypasses the 10-nm resolution barrier.\n",
      "Advances in super-resolution microscopy have demonstrated single-molecule localization precisions of\n",
      "a few nanometers. However, translation of such high localization precisions into sub-10-nm spatial\n",
      "resolution in biological samples remains challenging. Here we show that resonance energy transfer\n",
      "between fluorophores separated by less than 10 nm results in accelerated fluorescence blinking and\n",
      "consequently lower localization probabilities impeding sub-10-nm fluorescence imaging. We\n",
      "demonstrate that time-resolved fluorescence detection in combination with photoswitching fingerprint\n",
      "analysis can be used to determine the number and distance even of spatially unresolvable\n",
      "fluorophores in the sub-10-nm range. In combination with genetic code expansion with unnatural amino\n",
      "acids and bioorthogonal click labeling with small fluorophores, photoswitching fingerprint analysis\n",
      "can be used advantageously to reveal information about the number of fluorophores present and their\n",
      "distances in the sub-10-nm range in cells.\n",
      "['fluorescence imaging',\n",
      " 'super-resolution microscopy',\n",
      " 'photoswitching fingerprint analysis',\n",
      " 'time-resolved fluorescence detection']\n",
      "Elapsed time: 11.24 seconds\n",
      "-------------------\n",
      "35927475\n",
      "Functional ultrasound localization microscopy reveals brain-wide neurovascular activity on a\n",
      "microscopic scale.\n",
      "The advent of neuroimaging has increased our understanding of brain function. While most brain-wide\n",
      "functional imaging modalities exploit neurovascular coupling to map brain activity at millimeter\n",
      "resolutions, the recording of functional responses at microscopic scale in mammals remains the\n",
      "privilege of invasive electrophysiological or optical approaches, but is mostly restricted to either\n",
      "the cortical surface or the vicinity of implanted sensors. Ultrasound localization microscopy (ULM)\n",
      "has achieved transcranial imaging of cerebrovascular flow, up to micrometre scales, by localizing\n",
      "intravenously injected microbubbles; however, the long acquisition time required to detect\n",
      "microbubbles within microscopic vessels has so far restricted ULM application mainly to\n",
      "microvasculature structural imaging. Here we show how ULM can be modified to quantify functional\n",
      "hyperemia dynamically during brain activation reaching a 6.5-µm spatial and 1-s temporal resolution\n",
      "in deep regions of the rat brain.\n",
      "{'imagingTechniques': ['functional ultrasound localization microscopy',\n",
      "                       'neurovascular coupling',\n",
      "                       'transcranial imaging',\n",
      "                       'microscopic scale imaging',\n",
      "                       'intravenously injected microbubbles']}\n",
      "Elapsed time: 11.55 seconds\n",
      "-------------------\n",
      "35927477\n",
      "Sprod for de-noising spatially resolved transcriptomics data based on position and image\n",
      "information.\n",
      "Spatially resolved transcriptomics (SRT) provide gene expression close to, or even superior to,\n",
      "single-cell resolution while retaining the physical locations of sequencing and often also providing\n",
      "matched pathology images. However, SRT expression data suffer from high noise levels, due to the\n",
      "shallow coverage in each sequencing unit and the extra experimental steps required to preserve the\n",
      "locations of sequencing. Fortunately, such noise can be removed by leveraging information from the\n",
      "physical locations of sequencing, and the tissue organization reflected in corresponding pathology\n",
      "images. In this work, we developed Sprod, based on latent graph learning of matched location and\n",
      "imaging data, to impute accurate SRT gene expression. We validated Sprod comprehensively and\n",
      "demonstrated its advantages over previous methods for removing drop-outs in single-cell RNA-\n",
      "sequencing data. We showed that, after imputation by Sprod, differential expression analyses,\n",
      "pathway enrichment and cell-to-cell interaction inferences are more accurate. Overall, we envision\n",
      "de-noising by Sprod to become a key first step towards empowering SRT technologies for biomedical\n",
      "discoveries.\n",
      "['latent graph learning', 'matched location and imaging data']\n",
      "Elapsed time: 10.59 seconds\n",
      "-------------------\n",
      "35927480\n",
      "PROBER identifies proteins associated with programmable sequence-specific DNA in living cells.\n",
      "DNA-protein interactions mediate physiologic gene regulation and may be altered by DNA variants\n",
      "linked to polygenic disease. To enhance the speed and signal-to-noise ratio (SNR) in the\n",
      "identification and quantification of proteins associated with specific DNA sequences in living\n",
      "cells, we developed proximal biotinylation by episomal recruitment (PROBER). PROBER uses high-copy\n",
      "episomes to amplify SNR, and proximity proteomics (BioID) to identify the transcription factors and\n",
      "additional gene regulators associated with short DNA sequences of interest. PROBER quantified both\n",
      "constitutive and inducible association of transcription factors and corresponding chromatin\n",
      "regulators to target DNA sequences and binding quantitative trait loci due to single-nucleotide\n",
      "variants. PROBER identified alterations in regulator associations due to cancer hotspot mutations in\n",
      "the hTERT promoter, indicating that these mutations increase promoter association with specific gene\n",
      "activators. PROBER provides an approach to rapidly identify proteins associated with specific DNA\n",
      "sequences and their variants in living cells.\n",
      "['proximity proteomics',\n",
      " ' BioID',\n",
      " 'high-copy episomes',\n",
      " 'proximal biotinylation']\n",
      "Elapsed time: 10.78 seconds\n",
      "-------------------\n",
      "35953671\n",
      "Residue-wise local quality estimation for protein models from cryo-EM maps.\n",
      "An increasing number of protein structures are being determined by cryogenic electron microscopy\n",
      "(cryo-EM). Although the resolution of determined cryo-EM density maps is improving in general, there\n",
      "are still many cases where amino acids of a protein are assigned with different levels of\n",
      "confidence. Here we developed a method that identifies potential misassignment of residues in the\n",
      "map, including residue shifts along an otherwise correct main-chain trace. The score, named DAQ,\n",
      "computes the likelihood that the local density corresponds to different amino acids, atoms, and\n",
      "secondary structures, estimated via deep learning, and assesses the consistency of the amino acid\n",
      "assignment in the protein structure model with that likelihood. When DAQ was applied to different\n",
      "versions of model structures in the Protein Data Bank that were derived from the same density maps,\n",
      "a clear improvement in the DAQ score was observed in the newer versions of the models. DAQ also\n",
      "found potential misassignment errors in a substantial number of deposited protein structure models\n",
      "built into cryo-EM maps.\n",
      "['DAQ', 'cryogenic electron microscopy (cryo-EM)', 'deep learning']\n",
      "Elapsed time: 10.20 seconds\n",
      "-------------------\n",
      "36008629\n",
      "Advances in measuring cancer cell metabolism with subcellular resolution.\n",
      "Characterizing metabolism in cancer is crucial for understanding tumor biology and for developing\n",
      "potential therapies. Although most metabolic investigations analyze averaged metabolite levels from\n",
      "all cell compartments, subcellular metabolomics can provide more detailed insight into the\n",
      "biochemical processes associated with the disease. Methodological limitations have historically\n",
      "prevented the wider application of subcellular metabolomics in cancer research. Recently, however,\n",
      "ways to distinguish and identify metabolic pathways within organelles have been developed, including\n",
      "state-of-the-art methods to monitor metabolism in situ (such as mass spectrometry-based imaging,\n",
      "Raman spectroscopy and fluorescence microscopy), to isolate key organelles via new approaches and to\n",
      "use tailored isotope-tracing strategies. Herein, we examine the advantages and limitations of these\n",
      "developments and look to the future of this field of research.\n",
      "['mass spectrometry-based imaging',\n",
      " 'Raman spectroscopy',\n",
      " 'fluorescence microscopy']\n",
      "Elapsed time: 9.75 seconds\n",
      "-------------------\n",
      "36050489\n",
      "Antibody stabilization for thermally accelerated deep immunostaining.\n",
      "Antibodies have diverse applications due to their high reaction specificities but are sensitive to\n",
      "denaturation when a higher working temperature is required. We have developed a simple, highly\n",
      "scalable and generalizable chemical approach for stabilizing off-the-shelf antibodies against\n",
      "thermal and chemical denaturation. We demonstrate that the stabilized antibodies (termed SPEARs) can\n",
      "withstand up to 4 weeks of continuous heating at 55 °C and harsh denaturants, and apply our method\n",
      "to 33 tested antibodies. SPEARs enable flexible applications of thermocycling and denaturants to\n",
      "dynamically modulate their binding kinetics, reaction equilibrium, macromolecular diffusivity and\n",
      "aggregation propensity. In particular, we show that SPEARs permit the use of a thermally facilitated\n",
      "three-dimensional immunolabeling strategy (termed ThICK staining), achieving whole mouse brain\n",
      "immunolabeling within 72 h, as well as nearly fourfold deeper penetration with threefold less\n",
      "antibodies in human brain tissue. With faster deep-tissue immunolabeling and broad compatibility\n",
      "with tissue processing and clearing methods without the need for any specialized equipment, we\n",
      "anticipate the wide applicability of ThICK staining with SPEARs for deep immunostaining.\n",
      "['immunolabeling', 'thermal facilitation', 'three-dimensional staining']\n",
      "Elapsed time: 12.34 seconds\n",
      "-------------------\n",
      "36050490\n",
      "DNA-PAINT MINFLUX nanoscopy.\n",
      "MINimal fluorescence photon FLUXes (MINFLUX) nanoscopy, providing photon-efficient fluorophore\n",
      "localizations, has brought about three-dimensional resolution at nanometer scales. However, by using\n",
      "an intrinsic on-off switching process for single fluorophore separation, initial MINFLUX\n",
      "implementations have been limited to two color channels. Here we show that MINFLUX can be\n",
      "effectively combined with sequentially multiplexed DNA-based labeling (DNA-PAINT), expanding MINFLUX\n",
      "nanoscopy to multiple molecular targets. Our method is exemplified with three-color recordings of\n",
      "mitochondria in human cells.\n",
      "['DNA-PAINT', 'MINFLUX', 'sequentially multiplexed DNA-based labeling']\n",
      "Elapsed time: 8.35 seconds\n",
      "-------------------\n",
      "36064775\n",
      "Single-particle cryo-EM structures from iDPC-STEM at near-atomic resolution.\n",
      "In electron cryomicroscopy (cryo-EM), molecular images of vitrified biological samples are obtained\n",
      "by conventional transmission microscopy (CTEM) using large underfocuses and subsequently\n",
      "computationally combined into a high-resolution three-dimensional structure. Here, we apply scanning\n",
      "transmission electron microscopy (STEM) using the integrated differential phase contrast mode also\n",
      "known as iDPC-STEM to two cryo-EM test specimens, keyhole limpet hemocyanin (KLH) and tobacco mosaic\n",
      "virus (TMV). The micrographs show complete contrast transfer to high resolution and enable the cryo-\n",
      "EM structure determination for KLH at 6.5 Å resolution, as well as for TMV at 3.5 Å resolution using\n",
      "single-particle reconstruction methods, which share identical features with maps obtained by CTEM of\n",
      "a previously acquired same-sized TMV data set. These data show that STEM imaging in general, and in\n",
      "particular the iDPC-STEM approach, can be applied to vitrified single-particle specimens to\n",
      "determine near-atomic resolution cryo-EM structures of biological macromolecules.\n",
      "['cryo-EM',\n",
      " 'scanning transmission electron microscopy (STEM)',\n",
      " 'integrated differential phase contrast mode (iDPC-STEM)',\n",
      " 'conventional transmission microscopy (CTEM)']\n",
      "Elapsed time: 12.23 seconds\n",
      "-------------------\n",
      "36076037\n",
      "Event-triggered STED imaging.\n",
      "Monitoring the proteins and lipids that mediate all cellular processes requires imaging methods with\n",
      "increased spatial and temporal resolution. STED (stimulated emission depletion) nanoscopy enables\n",
      "fast imaging of nanoscale structures in living cells but is limited by photobleaching. Here, we\n",
      "present event-triggered STED, an automated multiscale method capable of rapidly initiating two-\n",
      "dimensional (2D) and 3D STED imaging after detecting cellular events such as protein recruitment,\n",
      "vesicle trafficking and second messengers activity using biosensors. STED is applied in the vicinity\n",
      "of detected events to maximize the temporal resolution. We imaged synaptic vesicle dynamics at up to\n",
      "24 Hz, 40 ms after local calcium activity; endocytosis and exocytosis events at up to 11 Hz, 40 ms\n",
      "after local protein recruitment or pH changes; and the interaction between endosomal vesicles at up\n",
      "to 3 Hz, 70 ms after approaching one another. Event-triggered STED extends the capabilities of live\n",
      "nanoscale imaging, enabling novel biological observations in real time.\n",
      "['STED', '2D STED', '3D STED', 'event-triggered STED']\n",
      "Elapsed time: 11.10 seconds\n",
      "-------------------\n",
      "36076039\n",
      "Event-driven acquisition for content-enriched microscopy.\n",
      "A common goal of fluorescence microscopy is to collect data on specific biological events. Yet, the\n",
      "event-specific content that can be collected from a sample is limited, especially for rare or\n",
      "stochastic processes. This is due in part to photobleaching and phototoxicity, which constrain\n",
      "imaging speed and duration. We developed an event-driven acquisition framework, in which neural-\n",
      "network-based recognition of specific biological events triggers real-time control in an instant\n",
      "structured illumination microscope. Our setup adapts acquisitions on-the-fly by switching between a\n",
      "slow imaging rate while detecting the onset of events, and a fast imaging rate during their\n",
      "progression. Thus, we capture mitochondrial and bacterial divisions at imaging rates that match\n",
      "their dynamic timescales, while extending overall imaging durations. Because event-driven\n",
      "acquisition allows the microscope to respond specifically to complex biological events, it acquires\n",
      "data enriched in relevant content.\n",
      "['instant structured illumination microscope',\n",
      " 'slow imaging rate',\n",
      " 'fast imaging rate']\n",
      "Elapsed time: 9.68 seconds\n",
      "-------------------\n",
      "36109679\n",
      "SVision: a deep learning approach to resolve complex structural variants.\n",
      "Complex structural variants (CSVs) encompass multiple breakpoints and are often missed or\n",
      "misinterpreted. We developed SVision, a deep-learning-based multi-object-recognition framework, to\n",
      "automatically detect and haracterize CSVs from long-read sequencing data. SVision outperforms\n",
      "current callers at identifying the internal structure of complex events and has revealed 80 high-\n",
      "quality CSVs with 25 distinct structures from an individual genome. SVision directly detects CSVs\n",
      "without matching known structures, allowing sensitive detection of both common and previously\n",
      "uncharacterized complex rearrangements.\n",
      "['deep learning', 'long-read sequencing', 'multi-object recognition framework']\n",
      "Elapsed time: 7.71 seconds\n",
      "-------------------\n",
      "36138173\n",
      "Cell region fingerprints enable highly precise single-cell tracking and lineage reconstruction.\n",
      "Experimental studies of cell growth, inheritance and their associated processes by microscopy\n",
      "require accurate single-cell observations of sufficient duration to reconstruct the genealogy.\n",
      "However, cell tracking-assigning identical cells on consecutive images to a track-is often\n",
      "challenging, resulting in laborious manual verification. Here, we propose fingerprints to identify\n",
      "problematic assignments rapidly. A fingerprint distance compares the structural information\n",
      "contained in the low frequencies of a Fourier transform to measure the similarity between cells in\n",
      "two consecutive images. We show that fingerprints are broadly applicable across cell types and image\n",
      "modalities, provided the image has sufficient structural information. Our tracker (Trac<sup>X</sup>)\n",
      "uses fingerprints to reject unlikely assignments, thereby increasing tracking performance on\n",
      "published and newly generated long-term data sets. For Saccharomyces cerevisiae, we propose a\n",
      "comprehensive model for cell size control at the single-cell and population level centered on the\n",
      "Whi5 regulator, demonstrating how precise tracking can help uncover previously undescribed single-\n",
      "cell biology.\n",
      "['microscopy', 'Fourier transform', 'tracking']\n",
      "Elapsed time: 10.35 seconds\n",
      "-------------------\n",
      "36138174\n",
      "A fluorescent sensor for real-time measurement of extracellular oxytocin dynamics in the brain.\n",
      "Oxytocin (OT), a hypothalamic neuropeptide that acts as a neuromodulator in the brain, orchestrates\n",
      "a variety of animal behaviors. However, the relationship between brain OT dynamics and complex\n",
      "animal behaviors remains largely elusive, partly because of the lack of a suitable technique for its\n",
      "real-time recording in vivo. Here, we describe MTRIA<sub>OT</sub>, a G-protein-coupled receptor-\n",
      "based green fluorescent OT sensor that has a large dynamic range, suitable affinity, ligand\n",
      "specificity for OT orthologs, minimal effects on downstream signaling and long-term fluorescence\n",
      "stability. By combining viral gene delivery and fiber photometry-mediated fluorescence measurements,\n",
      "we demonstrate the utility of MTRIA<sub>OT</sub> for real-time detection of brain OT dynamics in\n",
      "living mice. MTRIA<sub>OT</sub>-mediated measurements indicate variability of OT dynamics depending\n",
      "on the behavioral context and physical condition of an animal. MTRIA<sub>OT</sub> will likely enable\n",
      "the analysis of OT dynamics in a variety of physiological and pathological processes.\n",
      "['green fluorescent', 'fiber photometry-mediated fluorescence measurements']\n",
      "Elapsed time: 12.45 seconds\n",
      "-------------------\n",
      "36175767\n",
      "Analysis of the Human Protein Atlas Weakly Supervised Single-Cell Classification competition.\n",
      "While spatial proteomics by fluorescence imaging has quickly become an essential discovery tool for\n",
      "researchers, fast and scalable methods to classify and embed single-cell protein distributions in\n",
      "such images are lacking. Here, we present the design and analysis of the results from the\n",
      "competition Human Protein Atlas - Single-Cell Classification hosted on the Kaggle platform. This\n",
      "represents a crowd-sourced competition to develop machine learning models trained on limited\n",
      "annotations to label single-cell protein patterns in fluorescent images. The particular challenges\n",
      "of this competition include class imbalance, weak labels and multi-label classification, prompting\n",
      "competitors to apply a wide range of approaches in their solutions. The winning models serve as the\n",
      "first subcellular omics tools that can annotate single-cell locations, extract single-cell features\n",
      "and capture cellular dynamics.\n",
      "{'imagingTechniques': ['fluorescenceImaging', 'spatialProteomics']}\n",
      "Elapsed time: 9.12 seconds\n",
      "-------------------\n",
      "36203018\n",
      "neuromaps: structural and functional interpretation of brain maps.\n",
      "Imaging technologies are increasingly used to generate high-resolution reference maps of brain\n",
      "structure and function. Comparing experimentally generated maps to these reference maps facilitates\n",
      "cross-disciplinary scientific discovery. Although recent data sharing initiatives increase the\n",
      "accessibility of brain maps, data are often shared in disparate coordinate systems, precluding\n",
      "systematic and accurate comparisons. Here we introduce neuromaps, a toolbox for accessing,\n",
      "transforming and analyzing structural and functional brain annotations. We implement functionalities\n",
      "for generating high-quality transformations between four standard coordinate systems. The toolbox\n",
      "includes curated reference maps and biological ontologies of the human brain, such as molecular,\n",
      "microstructural, electrophysiological, developmental and functional ontologies. Robust quantitative\n",
      "assessment of map-to-map similarity is enabled via a suite of spatial autocorrelation-preserving\n",
      "null models. neuromaps combines open-access data with transparent functionality for standardizing\n",
      "and comparing brain maps, providing a systematic workflow for comprehensive structural and\n",
      "functional annotation enrichment analysis of the human brain.\n",
      "['electrophysiological',\n",
      " 'developmental',\n",
      " 'functional',\n",
      " 'molecular',\n",
      " 'microstructural']\n",
      "Elapsed time: 10.35 seconds\n",
      "-------------------\n",
      "36216958\n",
      "Light-Seq: light-directed in situ barcoding of biomolecules in fixed cells and tissues for spatially\n",
      "indexed sequencing.\n",
      "We present Light-Seq, an approach for multiplexed spatial indexing of intact biological samples\n",
      "using light-directed DNA barcoding in fixed cells and tissues followed by ex situ sequencing. Light-\n",
      "Seq combines spatially targeted, rapid photocrosslinking of DNA barcodes onto complementary DNAs in\n",
      "situ with a one-step DNA stitching reaction to create pooled, spatially indexed sequencing\n",
      "libraries. This light-directed barcoding enables in situ selection of multiple cell populations in\n",
      "intact fixed tissue samples for full-transcriptome sequencing based on location, morphology or\n",
      "protein stains, without cellular dissociation. Applying Light-Seq to mouse retinal sections, we\n",
      "recovered thousands of differentially enriched transcripts from three cellular layers and discovered\n",
      "biomarkers for a very rare neuronal subtype, dopaminergic amacrine cells, from only four to eight\n",
      "individual cells per section. Light-Seq provides an accessible workflow to combine in situ imaging\n",
      "and protein staining with next generation sequencing of the same cells, leaving the sample intact\n",
      "for further analysis post-sequencing.\n",
      "[' Light-Directed DNA Barcoding',\n",
      " ' Spatially Targeted Rapid Photocrosslinking',\n",
      " ' One-Step DNA Stitching Reaction',\n",
      " ' In Situ Selection of Multiple Cell Populations',\n",
      " ' Full-Transcriptome Sequencing',\n",
      " ' In Situ Imaging and Protein Staining with Next Generation Sequencing']\n",
      "Elapsed time: 12.74 seconds\n",
      "-------------------\n",
      "36253643\n",
      "Omnipose: a high-precision morphology-independent solution for bacterial cell segmentation.\n",
      "Advances in microscopy hold great promise for allowing quantitative and precise measurement of\n",
      "morphological and molecular phenomena at the single-cell level in bacteria; however, the potential\n",
      "of this approach is ultimately limited by the availability of methods to faithfully segment cells\n",
      "independent of their morphological or optical characteristics. Here, we present Omnipose, a deep\n",
      "neural network image-segmentation algorithm. Unique network outputs such as the gradient of the\n",
      "distance field allow Omnipose to accurately segment cells on which current algorithms, including its\n",
      "predecessor, Cellpose, produce errors. We show that Omnipose achieves unprecedented segmentation\n",
      "performance on mixed bacterial cultures, antibiotic-treated cells and cells of elongated or branched\n",
      "morphology. Furthermore, the benefits of Omnipose extend to non-bacterial subjects, varied imaging\n",
      "modalities and three-dimensional objects. Finally, we demonstrate the utility of Omnipose in the\n",
      "characterization of extreme morphological phenotypes that arise during interbacterial antagonism.\n",
      "Our results distinguish Omnipose as a powerful tool for characterizing diverse and arbitrarily\n",
      "shaped cell types from imaging data.\n",
      "['deep neural network image-segmentation algorithm',\n",
      " 'microscopy',\n",
      " 'three-dimensional objects']\n",
      "Elapsed time: 11.51 seconds\n",
      "-------------------\n",
      "36253644\n",
      "Estimation of skeletal kinematics in freely moving rodents.\n",
      "Forming a complete picture of the relationship between neural activity and skeletal kinematics\n",
      "requires quantification of skeletal joint biomechanics during free behavior; however, without\n",
      "detailed knowledge of the underlying skeletal motion, inferring limb kinematics using surface-\n",
      "tracking approaches is difficult, especially for animals where the relationship between the surface\n",
      "and underlying skeleton changes during motion. Here we developed a videography-based method enabling\n",
      "detailed three-dimensional kinematic quantification of an anatomically defined skeleton in\n",
      "untethered freely behaving rats and mice. This skeleton-based model was constrained using anatomical\n",
      "principles and joint motion limits and provided skeletal pose estimates for a range of body sizes,\n",
      "even when limbs were occluded. Model-inferred limb positions and joint kinematics during gait and\n",
      "gap-crossing behaviors were verified by direct measurement of either limb placement or limb\n",
      "kinematics using inertial measurement units. Together we show that complex decision-making behaviors\n",
      "can be accurately reconstructed at the level of skeletal kinematics using our anatomically\n",
      "constrained model.\n",
      "['videography-based method', 'inertial measurement units']\n",
      "Elapsed time: 10.06 seconds\n",
      "-------------------\n",
      "36266465\n",
      "Improved AlphaFold modeling with implicit experimental information.\n",
      "Machine-learning prediction algorithms such as AlphaFold and RoseTTAFold can create remarkably\n",
      "accurate protein models, but these models usually have some regions that are predicted with low\n",
      "confidence or poor accuracy. We hypothesized that by implicitly including new experimental\n",
      "information such as a density map, a greater portion of a model could be predicted accurately, and\n",
      "that this might synergistically improve parts of the model that were not fully addressed by either\n",
      "machine learning or experiment alone. An iterative procedure was developed in which AlphaFold models\n",
      "are automatically rebuilt on the basis of experimental density maps and the rebuilt models are used\n",
      "as templates in new AlphaFold predictions. We show that including experimental information improves\n",
      "prediction beyond the improvement obtained with simple rebuilding guided by the experimental data.\n",
      "This procedure for AlphaFold modeling with density has been incorporated into an automated procedure\n",
      "for interpretation of crystallographic and electron cryo-microscopy maps.\n",
      "['density map', 'crystallographic maps', 'electron cryo-microscopy maps']\n",
      "Elapsed time: 937.60 seconds\n",
      "-------------------\n",
      "36266466\n",
      "Practical considerations for quantitative light sheet fluorescence microscopy.\n",
      "Fluorescence microscopy has evolved from a purely observational tool to a platform for quantitative,\n",
      "hypothesis-driven research. As such, the demand for faster and less phototoxic imaging modalities\n",
      "has spurred a rapid growth in light sheet fluorescence microscopy (LSFM). By restricting the\n",
      "excitation to a thin plane, LSFM reduces the overall light dose to a specimen while simultaneously\n",
      "improving image contrast. However, the defining characteristics of light sheet microscopes\n",
      "subsequently warrant unique considerations in their use for quantitative experiments. In this\n",
      "Perspective, we outline many of the pitfalls in LSFM that can compromise analysis and confound\n",
      "interpretation. Moreover, we offer guidance in addressing these caveats when possible. In doing so,\n",
      "we hope to provide a useful resource for life scientists seeking to adopt LSFM to quantitatively\n",
      "address complex biological hypotheses.\n",
      "['LSFM', 'light sheet fluorescence microscopy', 'fluorescence microscopy']\n",
      "Elapsed time: 8.68 seconds\n",
      "-------------------\n",
      "36280715\n",
      "SyConn2: dense synaptic connectivity inference for volume electron microscopy.\n",
      "The ability to acquire ever larger datasets of brain tissue using volume electron microscopy leads\n",
      "to an increasing demand for the automated extraction of connectomic information. We introduce\n",
      "SyConn2, an open-source connectome analysis toolkit, which works with both on-site high-performance\n",
      "compute environments and rentable cloud computing clusters. SyConn2 was tested on connectomic\n",
      "datasets with more than 10 million synapses, provides a web-based visualization interface and makes\n",
      "these data amenable to complex anatomical and neuronal connectivity queries.\n",
      "['SyConn2', 'volume electron microscopy', 'connectome analysis toolkit']\n",
      "Elapsed time: 6.97 seconds\n",
      "-------------------\n",
      "36280717\n",
      "Automated synapse-level reconstruction of neural circuits in the larval zebrafish brain.\n",
      "Dense reconstruction of synaptic connectivity requires high-resolution electron microscopy images of\n",
      "entire brains and tools to efficiently trace neuronal wires across the volume. To generate such a\n",
      "resource, we sectioned and imaged a larval zebrafish brain by serial block-face electron microscopy\n",
      "at a voxel size of 14 × 14 × 25 nm<sup>3</sup>. We segmented the resulting dataset with the flood-\n",
      "filling network algorithm, automated the detection of chemical synapses and validated the results by\n",
      "comparisons to transmission electron microscopic images and light-microscopic reconstructions.\n",
      "Neurons and their connections are stored in the form of a queryable and expandable digital address\n",
      "book. We reconstructed a network of 208 neurons involved in visual motion processing, most of them\n",
      "located in the pretectum, which had been functionally characterized in the same specimen by two-\n",
      "photon calcium imaging. Moreover, we mapped all 407 presynaptic and postsynaptic partners of two\n",
      "superficial interneurons in the tectum. The resource developed here serves as a foundation for\n",
      "synaptic-resolution circuit analyses in the zebrafish nervous system.\n",
      "[{'name': 'serial block-face electron microscopy'},\n",
      " {'name': 'flood-filling network algorithm'},\n",
      " {'name': 'transmission electron microscopic imaging'},\n",
      " {'name': 'light-microscopic reconstructions'}]\n",
      "Elapsed time: 13.83 seconds\n",
      "-------------------\n",
      "36280718\n",
      "Resolution doubling in light-sheet microscopy via oblique plane structured illumination.\n",
      "Structured illumination microscopy (SIM) doubles the spatial resolution of a fluorescence microscope\n",
      "without requiring high laser powers or specialized fluorophores. However, the excitation of out-of-\n",
      "focus fluorescence can accelerate photobleaching and phototoxicity. In contrast, light-sheet\n",
      "fluorescence microscopy (LSFM) largely avoids exciting out-of-focus fluorescence, thereby enabling\n",
      "volumetric imaging with low photobleaching and intrinsic optical sectioning. Combining SIM with LSFM\n",
      "would enable gentle three-dimensional (3D) imaging at doubled resolution. However, multiple\n",
      "orientations of the illumination pattern, which are needed for isotropic resolution doubling in SIM,\n",
      "are challenging to implement in a light-sheet format. Here we show that multidirectional structured\n",
      "illumination can be implemented in oblique plane microscopy, an LSFM technique that uses a single\n",
      "objective for excitation and detection, in a straightforward manner. We demonstrate isotropic\n",
      "lateral resolution below 150 nm, combined with lower phototoxicity compared to traditional SIM\n",
      "systems and volumetric acquisition speed exceeding 1 Hz.\n",
      "['structured illumination microscopy (SIM)',\n",
      " 'light-sheet fluorescence microscopy (LSFM)',\n",
      " 'oblique plane microscopy']\n",
      "Elapsed time: 11.25 seconds\n",
      "-------------------\n",
      "36280720\n",
      "Annotation of spatially resolved single-cell data with STELLAR.\n",
      "Accurate cell-type annotation from spatially resolved single cells is crucial to understand\n",
      "functional spatial biology that is the basis of tissue organization. However, current computational\n",
      "methods for annotating spatially resolved single-cell data are typically based on techniques\n",
      "established for dissociated single-cell technologies and thus do not take spatial organization into\n",
      "account. Here we present STELLAR, a geometric deep learning method for cell-type discovery and\n",
      "identification in spatially resolved single-cell datasets. STELLAR automatically assigns cells to\n",
      "cell types present in the annotated reference dataset and discovers novel cell types and cell\n",
      "states. STELLAR transfers annotations across different dissection regions, different tissues and\n",
      "different donors, and learns cell representations that capture higher-order tissue structures. We\n",
      "successfully applied STELLAR to CODEX multiplexed fluorescent microscopy data and multiplexed RNA\n",
      "imaging datasets. Within the Human BioMolecular Atlas Program, STELLAR has annotated 2.6 million\n",
      "spatially resolved single cells with dramatic time savings.\n",
      "['fluorescent microscopy', 'RNA imaging', 'multiplexed RNA imaging']\n",
      "Elapsed time: 855.80 seconds\n",
      "-------------------\n",
      "36280722\n",
      "Geometric engineering of organoid culture for enhanced organogenesis in a dish.\n",
      "Here, we introduce a facile, scalable engineering approach to enable long-term development and\n",
      "maturation of organoids. We have redesigned the configuration of conventional organoid culture to\n",
      "develop a platform that converts single injections of stem cell suspensions to radial arrays of\n",
      "organoids that can be maintained for extended periods without the need for passaging. Using this\n",
      "system, we demonstrate accelerated production of intestinal organoids with significantly enhanced\n",
      "structural and functional maturity, and their continuous development for over 4 weeks. Furthermore,\n",
      "we present a patient-derived organoid model of inflammatory bowel disease (IBD) and its\n",
      "interrogation using single-cell RNA sequencing to demonstrate its ability to reproduce key\n",
      "pathological features of IBD. Finally, we describe the extension of our approach to engineer\n",
      "vascularized, perfusable human enteroids, which can be used to model innate immune responses in IBD.\n",
      "This work provides an immediately deployable platform technology toward engineering more realistic\n",
      "organ-like structures in a dish.\n",
      "['single-cell RNA sequencing',\n",
      " 'radial arrays',\n",
      " 'patient-derived organoid model',\n",
      " 'vascularized, perfusable human enteroids']\n",
      "Elapsed time: 10.16 seconds\n",
      "-------------------\n",
      "36280724\n",
      "ClampFISH 2.0 enables rapid, scalable amplified RNA detection in situ.\n",
      "RNA labeling in situ has enormous potential to visualize transcripts and quantify their levels in\n",
      "single cells, but it remains challenging to produce high levels of signal while also enabling\n",
      "multiplexed detection of multiple RNA species simultaneously. Here, we describe clampFISH 2.0, a\n",
      "method that uses an inverted padlock design to efficiently detect many RNA species and exponentially\n",
      "amplify their signals at once, while also reducing the time and cost compared with the prior\n",
      "clampFISH method. We leverage the increased throughput afforded by multiplexed signal amplification\n",
      "and sequential detection to detect 10 different RNA species in more than 1 million cells. We also\n",
      "show that clampFISH 2.0 works in tissue sections. We expect that the advantages offered by clampFISH\n",
      "2.0 will enable many applications in spatial transcriptomics.\n",
      "{'imaging techniques': ['ClampFISH', 'multiplexed signal amplification']}\n",
      "Elapsed time: 9.18 seconds\n",
      "-------------------\n",
      "36303019\n",
      "Sensitive genetically encoded sensors for population and subcellular imaging of cAMP in vivo.\n",
      "Cyclic adenosine monophosphate (cAMP) signaling integrates information from diverse G-protein-\n",
      "coupled receptors, such as neuromodulator receptors, to regulate pivotal biological processes in a\n",
      "cellular-specific and subcellular-specific manner. However, in vivo cellular-resolution imaging of\n",
      "cAMP dynamics remains challenging. Here, we screen existing genetically encoded cAMP sensors and\n",
      "further develop the best performer to derive three improved variants, called cAMPFIREs. Compared\n",
      "with their parental sensor, these sensors exhibit up to 10-fold increased sensitivity to cAMP and a\n",
      "cytosolic distribution. cAMPFIREs are compatible with both ratiometric and fluorescence lifetime\n",
      "imaging and can detect cAMP dynamics elicited by norepinephrine at physiologically relevant,\n",
      "nanomolar concentrations. Imaging of cAMPFIREs in awake mice reveals tonic levels of cAMP in\n",
      "cortical neurons that are associated with wakefulness, modulated by opioids, and differentially\n",
      "regulated across subcellular compartments. Furthermore, enforced locomotion elicits neuron-specific,\n",
      "bidirectional cAMP dynamics. cAMPFIREs also function in Drosophila. Overall, cAMPFIREs may have\n",
      "broad applicability for studying intracellular signaling in vivo.\n",
      "['ratiometric imaging',\n",
      " 'fluorescence lifetime imaging',\n",
      " 'awake mice',\n",
      " 'locomotion']\n",
      "Elapsed time: 13.85 seconds\n",
      "-------------------\n",
      "36316562\n",
      "Unsupervised discovery of tissue architecture in multiplexed imaging.\n",
      "Multiplexed imaging and spatial transcriptomics enable highly resolved spatial characterization of\n",
      "cellular phenotypes, but still largely depend on laborious manual annotation to understand higher-\n",
      "order patterns of tissue organization. As a result, higher-order patterns of tissue organization are\n",
      "poorly understood and not systematically connected to disease pathology or clinical outcomes. To\n",
      "address this gap, we developed an approach called UTAG to identify and quantify microanatomical\n",
      "tissue structures in multiplexed images without human intervention. Our method combines information\n",
      "on cellular phenotypes with the physical proximity of cells to accurately identify organ-specific\n",
      "microanatomical domains in healthy and diseased tissue. We apply our method to various types of\n",
      "images across healthy and disease states to show that it can consistently detect higher-level\n",
      "architectures in human tissues, quantify structural differences between healthy and diseased tissue,\n",
      "and reveal tissue organization patterns at the organ scale.\n",
      "['multiplexed imaging', 'spatial transcriptomics', 'untag']\n",
      "Elapsed time: 710.56 seconds\n",
      "-------------------\n",
      "36316563\n",
      "Incorporating the image formation process into deep learning improves network performance.\n",
      "We present Richardson-Lucy network (RLN), a fast and lightweight deep learning method for three-\n",
      "dimensional fluorescence microscopy deconvolution. RLN combines the traditional Richardson-Lucy\n",
      "iteration with a fully convolutional network structure, establishing a connection to the image\n",
      "formation process and thereby improving network performance. Containing only roughly 16,000\n",
      "parameters, RLN enables four- to 50-fold faster processing than purely data-driven networks with\n",
      "many more parameters. By visual and quantitative analysis, we show that RLN provides better\n",
      "deconvolution, better generalizability and fewer artifacts than other networks, especially along the\n",
      "axial dimension. RLN outperforms classic Richardson-Lucy deconvolution on volumes contaminated with\n",
      "severe out of focus fluorescence or noise and provides four- to sixfold faster reconstructions of\n",
      "large, cleared-tissue datasets than classic multi-view pipelines. We demonstrate RLN's performance\n",
      "on cells, tissues and embryos imaged with widefield-, light-sheet-, confocal- and super-resolution\n",
      "microscopy.\n",
      "['widefield microscopy',\n",
      " 'light-sheet microscopy',\n",
      " 'confocal microscopy',\n",
      " 'super-resolution microscopy']\n",
      "Elapsed time: 11.20 seconds\n",
      "-------------------\n",
      "36344832\n",
      "Cellpose 2.0: how to train your own model.\n",
      "Pretrained neural network models for biological segmentation can provide good out-of-the-box results\n",
      "for many image types. However, such models do not allow users to adapt the segmentation style to\n",
      "their specific needs and can perform suboptimally for test images that are very different from the\n",
      "training images. Here we introduce Cellpose 2.0, a new package that includes an ensemble of diverse\n",
      "pretrained models as well as a human-in-the-loop pipeline for rapid prototyping of new custom\n",
      "models. We show that models pretrained on the Cellpose dataset can be fine-tuned with only 500-1,000\n",
      "user-annotated regions of interest (ROI) to perform nearly as well as models trained on entire\n",
      "datasets with up to 200,000 ROI. A human-in-the-loop approach further reduced the required user\n",
      "annotation to 100-200 ROI, while maintaining high-quality segmentations. We provide software tools\n",
      "such as an annotation graphical user interface, a model zoo and a human-in-the-loop pipeline to\n",
      "facilitate the adoption of Cellpose 2.0.\n",
      "['ensemble of diverse pretrained models',\n",
      " 'human-in-the-loop pipeline',\n",
      " 'rapid prototyping of new custom models',\n",
      " 'user-annotated regions of interest (ROI)',\n",
      " 'model fine-tuning']\n",
      "Elapsed time: 11.30 seconds\n",
      "-------------------\n",
      "36344833\n",
      "Chemically stable fluorescent proteins for advanced microscopy.\n",
      "We report the rational engineering of a remarkably stable yellow fluorescent protein (YFP),\n",
      "'hyperfolder YFP' (hfYFP), that withstands chaotropic conditions that denature most biological\n",
      "structures within seconds, including superfolder green fluorescent protein (GFP). hfYFP contains no\n",
      "cysteines, is chloride insensitive and tolerates aldehyde and osmium tetroxide fixation better than\n",
      "common fluorescent proteins, enabling its use in expansion and electron microscopies. We solved\n",
      "crystal structures of hfYFP (to 1.7-Å resolution), a monomeric variant, monomeric hyperfolder YFP\n",
      "(1.6 Å) and an mGreenLantern mutant (1.2 Å), and then rationally engineered highly stable 405-nm-\n",
      "excitable GFPs, large Stokes shift (LSS) monomeric GFP (LSSmGFP) and LSSA12 from these structures.\n",
      "Lastly, we directly exploited the chemical stability of hfYFP and LSSmGFP by devising a\n",
      "fluorescence-assisted protein purification strategy enabling all steps of denaturing affinity\n",
      "chromatography to be visualized using ultraviolet or blue light. hfYFP and LSSmGFP represent a new\n",
      "generation of robustly stable fluorescent proteins developed for advanced biotechnological\n",
      "applications.\n",
      "['electron microscopy',\n",
      " 'expansion microscopy',\n",
      " 'fluorescence-assisted protein purification',\n",
      " 'ultraviolet light',\n",
      " 'blue light']\n",
      "Elapsed time: 13.31 seconds\n",
      "-------------------\n",
      "36344834\n",
      "High-dimensional gene expression and morphology profiles of cells across 28,000 genetic and chemical\n",
      "perturbations.\n",
      "Cells can be perturbed by various chemical and genetic treatments and the impact on gene expression\n",
      "and morphology can be measured via transcriptomic profiling and image-based assays, respectively.\n",
      "The patterns observed in these high-dimensional profile data can power a dozen applications in drug\n",
      "discovery and basic biology research, but both types of profiles are rarely available for large-\n",
      "scale experiments. Here, we provide a collection of four datasets with both gene expression and\n",
      "morphological profile data useful for developing and testing multimodal methodologies. Roughly a\n",
      "thousand features are measured for each of the two data types, across more than 28,000 chemical and\n",
      "genetic perturbations. We define biological problems that use the shared and complementary\n",
      "information in these two data modalities, provide baseline analysis and evaluation metrics for\n",
      "multi-omic applications, and make the data resource publicly available ( https://broad.io/rosetta/\n",
      ").\n",
      "['transcriptomic profiling', 'image-based assays']\n",
      "Elapsed time: 9.03 seconds\n",
      "-------------------\n",
      "36357694\n",
      "Capturing the start point of the virus-cell interaction with high-speed 3D single-virus tracking.\n",
      "The early stages of the virus-cell interaction have long evaded observation by existing microscopy\n",
      "methods due to the rapid diffusion of virions in the extracellular space and the large three-\n",
      "dimensional cellular structures involved. Here we present an active-feedback single-particle\n",
      "tracking method with simultaneous volumetric imaging of the live cell environment called 3D-TrIm to\n",
      "address this knowledge gap. 3D-TrIm captures the extracellular phase of the infectious cycle in what\n",
      "we believe is unprecedented detail. We report what are, to our knowledge, previously unobserved\n",
      "phenomena in the early stages of the virus-cell interaction, including skimming contact events at\n",
      "the millisecond timescale, orders of magnitude change in diffusion coefficient upon binding and\n",
      "cylindrical and linear diffusion modes along cellular protrusions. Finally, we demonstrate how this\n",
      "method can move single-particle tracking from simple monolayer culture toward more tissue-like\n",
      "conditions by tracking single virions in tightly packed epithelial cells. This multiresolution\n",
      "method presents opportunities for capturing fast, three-dimensional processes in biological systems.\n",
      "['active-feedback single-particle tracking',\n",
      " 'simultaneous volumetric imaging',\n",
      " '3D TrIm']\n",
      "Elapsed time: 11.01 seconds\n",
      "-------------------\n",
      "36357695\n",
      "An improved imaging system that corrects MS2-induced RNA destabilization.\n",
      "The MS2 and MS2-coat protein (MS2-MCP) imaging system is widely used to study messenger RNA (mRNA)\n",
      "spatial distribution in living cells. Here, we report that the MS2-MCP system destabilizes some\n",
      "tagged mRNAs by activating the nonsense-mediated mRNA decay pathway. We introduce an improved\n",
      "version, which counteracts this effect by increasing the efficiency of translation termination of\n",
      "the tagged mRNAs. Improved versions were developed for both yeast and mammalian systems.\n",
      "['MS2-MCP system',\n",
      " 'translation termination',\n",
      " 'nonsense-mediated mRNA decay pathway']\n",
      "Elapsed time: 7.72 seconds\n",
      "-------------------\n",
      "36396787\n",
      "RS-FISH: precise, interactive, fast, and scalable FISH spot detection.\n",
      "Fluorescent in-situ hybridization (FISH)-based methods extract spatially resolved genetic and\n",
      "epigenetic information from biological samples by detecting fluorescent spots in microscopy images,\n",
      "an often challenging task. We present Radial Symmetry-FISH (RS-FISH), an accurate, fast, and user-\n",
      "friendly software for spot detection in two- and three-dimensional images. RS-FISH offers\n",
      "interactive parameter tuning and readily scales to large datasets and image volumes of cleared or\n",
      "expanded samples using distributed processing on workstations, clusters, or the cloud. RS-FISH\n",
      "maintains high detection accuracy and low localization error across a wide range of signal-to-noise\n",
      "ratios, a key feature for single-molecule FISH, spatial transcriptomics, or spatial genomics\n",
      "applications.\n",
      "['RS-FISH', 'microscopy images', 'fluorescent spots']\n",
      "Elapsed time: 9.02 seconds\n",
      "-------------------\n",
      "36424441\n",
      "Image-seq: spatially resolved single-cell sequencing guided by in situ and in vivo imaging.\n",
      "Tissue function depends on cellular organization. While the properties of individual cells are\n",
      "increasingly being deciphered using powerful single-cell sequencing technologies, understanding\n",
      "their spatial organization and temporal evolution remains a major challenge. Here, we present Image-\n",
      "seq, a technology that provides single-cell transcriptional data on cells that are isolated from\n",
      "specific spatial locations under image guidance, thus preserving the spatial information of the\n",
      "target cells. It is compatible with in situ and in vivo imaging and can document the temporal and\n",
      "dynamic history of the cells being analyzed. Cell samples are isolated from intact tissue and\n",
      "processed with state-of-the-art library preparation protocols. The technique therefore combines\n",
      "spatial information with highly sensitive RNA sequencing readouts from individual, intact cells. We\n",
      "have used both high-throughput, droplet-based sequencing as well as SMARTseq-v4 library preparation\n",
      "to demonstrate its application to bone marrow and leukemia biology. We discovered that DPP4 is a\n",
      "highly upregulated gene during early progression of acute myeloid leukemia and that it marks a more\n",
      "proliferative subpopulation that is confined to specific bone marrow microenvironments. Furthermore,\n",
      "the ability of Image-seq to isolate viable, intact cells should make it compatible with a range of\n",
      "downstream single-cell analysis tools including multi-omics protocols.\n",
      "{'imaging techniques': ['in situ imaging',\n",
      "                        'in vivo imaging',\n",
      "                        'high-throughput sequencing',\n",
      "                        'droplet-based sequencing',\n",
      "                        'SMARTseq-v4 library preparation']}\n",
      "Elapsed time: 13.13 seconds\n",
      "-------------------\n",
      "36443485\n",
      "A three-photon head-mounted microscope for imaging all layers of visual cortex in freely moving\n",
      "mice.\n",
      "Advances in head-mounted microscopes have enabled imaging of neuronal activity using genetic tools\n",
      "in freely moving mice but these microscopes are restricted to recording in minimally lit arenas and\n",
      "imaging upper cortical layers. Here we built a 2-g, three-photon excitation-based microscope,\n",
      "containing a z-drive that enabled access to all cortical layers while mice freely behaved in a fully\n",
      "lit environment. The microscope had on-board photon detectors, robust to environmental light, and\n",
      "the arena lighting was timed to the end of each line-scan, enabling functional imaging of activity\n",
      "from cortical layer 4 and layer 6 neurons expressing jGCaMP7f in mice roaming a fully lit or dark\n",
      "arena. By comparing the neuronal activity measured from populations in these layers we show that\n",
      "activity in cortical layer 4 and layer 6 is differentially modulated by lit and dark conditions\n",
      "during free exploration.\n",
      "['three-photon microscopy', 'head-mounted microscope', 'functional imaging']\n",
      "Elapsed time: 9.98 seconds\n",
      "-------------------\n",
      "36456783\n",
      "Parallel cryo electron tomography on in situ lamellae.\n",
      "In situ cryo electron tomography of cryo focused ion beam milled samples has emerged in recent years\n",
      "as a powerful technique for structural studies of macromolecular complexes in their native cellular\n",
      "environment. However, the possibilities for recording tomographic tilt series in a high-throughput\n",
      "manner are limited, in part by the lamella-shaped samples. Here we utilize a geometrical sample\n",
      "model and optical image shift to record tens of tilt series in parallel, thereby saving time and\n",
      "gaining access to sample areas conventionally used for tracking specimen movement. The parallel cryo\n",
      "electron tomography (PACE-tomo) method achieves a throughput faster than 5 min per tilt series and\n",
      "allows for the collection of sample areas that were previously unreachable, thus maximizing the\n",
      "amount of data from each lamella. Performance testing with ribosomes in vitro and in situ on state-\n",
      "of-the-art and general-purpose microscopes demonstrated the high throughput and quality of PACE-\n",
      "tomo.\n",
      "['electron tomography',\n",
      " 'cryo electron tomography',\n",
      " 'parallel cryo electron tomography',\n",
      " 'PACE-tomo']\n",
      "Elapsed time: 9.68 seconds\n",
      "-------------------\n",
      "36456785\n",
      "Deep-tissue SWIR imaging using rationally designed small red-shifted near-infrared fluorescent\n",
      "protein.\n",
      "Applying rational design, we developed 17 kDa cyanobacteriochrome-based near-infrared (NIR-I)\n",
      "fluorescent protein, miRFP718nano. miRFP718nano efficiently binds endogenous biliverdin chromophore\n",
      "and brightly fluoresces in mammalian cells and tissues. miRFP718nano has maximal emission at 718 nm\n",
      "and an emission tail in the short-wave infrared (SWIR) region, allowing deep-penetrating off-peak\n",
      "fluorescence imaging in vivo. The miRFP718nano structure reveals the molecular basis of its red\n",
      "shift. We demonstrate superiority of miRFP718nano-enabled SWIR imaging over NIR-I imaging of\n",
      "microbes in the mouse digestive tract, mammalian cells injected into the mouse mammary gland and NF-\n",
      "kB activity in a mouse model of liver inflammation.\n",
      "['SWIR imaging', 'NIR-I imaging', 'off-peak fluorescence imaging']\n",
      "Elapsed time: 10.88 seconds\n",
      "-------------------\n",
      "36456786\n",
      "TemplateFlow: FAIR-sharing of multi-scale, multi-species brain models.\n",
      "Reference anatomies of the brain ('templates') and corresponding atlases are the foundation for\n",
      "reporting standardized neuroimaging results. Currently, there is no registry of templates and\n",
      "atlases; therefore, the redistribution of these resources occurs either bundled within existing\n",
      "software or in ad hoc ways such as downloads from institutional sites and general-purpose data\n",
      "repositories. We introduce TemplateFlow as a publicly available framework for human and non-human\n",
      "brain models. The framework combines an open database with software for access, management, and\n",
      "vetting, allowing scientists to share their resources under FAIR-findable, accessible,\n",
      "interoperable, and reusable-principles. TemplateFlow enables multifaceted insights into brains\n",
      "across species, and supports multiverse analyses testing whether results generalize across standard\n",
      "references, scales, and in the long term, species.\n",
      "['imaging results',\n",
      " 'software for access',\n",
      " 'management',\n",
      " 'vetting',\n",
      " 'multiverse analyses']\n",
      "Elapsed time: 9.37 seconds\n",
      "-------------------\n",
      "36481965\n",
      "A general method for chemogenetic control of peptide function.\n",
      "Natural or engineered peptides serve important biological functions. A general approach to achieve\n",
      "chemical-dependent activation of short peptides will be valuable for spatial and temporal control of\n",
      "cellular processes. Here we present a pair of chemically activated protein domains (CAPs) for\n",
      "controlling the accessibility of both the N- and C-terminal portion of a peptide. CAPs were\n",
      "developed through directed evolution of an FK506-binding protein. By fusing a peptide to one or both\n",
      "CAPs, the function of the peptide is blocked until a small molecule displaces them from the\n",
      "FK506-binding protein ligand-binding site. We demonstrate that CAPs are generally applicable to a\n",
      "range of short peptides, including a protease cleavage site, a dimerization-inducing heptapeptide, a\n",
      "nuclear localization signal peptide, and an opioid peptide, with a chemical dependence up to\n",
      "156-fold. We show that the CAPs system can be utilized in cell cultures and multiple organs in\n",
      "living animals.\n",
      "['chemical-dependent activation',\n",
      " 'directed evolution',\n",
      " 'FK506-binding protein ligand-binding site',\n",
      " 'protease cleavage site',\n",
      " 'dimerization-inducing heptapeptide',\n",
      " 'nuclear localization signal peptide',\n",
      " 'opioid peptide']\n",
      "Elapsed time: 12.12 seconds\n",
      "-------------------\n",
      "36522500\n",
      "Maximum-likelihood model fitting for quantitative analysis of SMLM data.\n",
      "Quantitative data analysis is important for any single-molecule localization microscopy (SMLM)\n",
      "workflow to extract biological insights from the coordinates of the single fluorophores. However,\n",
      "current approaches are restricted to simple geometries or require identical structures. Here, we\n",
      "present LocMoFit (Localization Model Fit), an open-source framework to fit an arbitrary model to\n",
      "localization coordinates. It extracts meaningful parameters from individual structures and can\n",
      "select the most suitable model. In addition to analyzing complex, heterogeneous and dynamic\n",
      "structures for in situ structural biology, we demonstrate how LocMoFit can assemble multi-protein\n",
      "distribution maps of six nuclear pore components, calculate single-particle averages without any\n",
      "assumption about geometry or symmetry, and perform a time-resolved reconstruction of the highly\n",
      "dynamic endocytic process from static snapshots. We provide extensive simulation and visualization\n",
      "routines to validate the robustness of LocMoFit and tutorials to enable any user to increase the\n",
      "information content they can extract from their SMLM data.\n",
      "['Localization Microscopy',\n",
      " 'Single-molecule Localization Microscopy (SMLM)',\n",
      " 'Quantitative Data Analysis',\n",
      " 'Open-source Framework',\n",
      " 'Time-resolved Reconstruction']\n",
      "Elapsed time: 10.93 seconds\n",
      "-------------------\n",
      "36522503\n",
      "Uniform thin ice on ultraflat graphene for high-resolution cryo-EM.\n",
      "Cryo-electron microscopy (cryo-EM) visualizes the atomic structure of macromolecules that are\n",
      "embedded in vitrified thin ice at their close-to-native state. However, the homogeneity of ice\n",
      "thickness, a key factor to ensure high image quality, is poorly controlled during specimen\n",
      "preparation and has become one of the main challenges for high-resolution cryo-EM. Here we found\n",
      "that the uniformity of thin ice relies on the surface flatness of the supporting film, and developed\n",
      "a method to use ultraflat graphene (UFG) as the support for cryo-EM specimen preparation to achieve\n",
      "better control of vitreous ice thickness. We show that the uniform thin ice on UFG improves the\n",
      "image quality of vitrified specimens. Using such a method we successfully determined the three-\n",
      "dimensional structures of hemoglobin (64 kDa), α-fetoprotein (67 kDa) with no symmetry, and\n",
      "streptavidin (52 kDa) at a resolution of 3.5 Å, 2.6 Å and 2.2 Å, respectively. Furthermore, our\n",
      "results demonstrate the potential of UFG for the fields of cryo-electron tomography and structure-\n",
      "based drug discovery.\n",
      "[['cryo-electron microscopy (cryo-EM)'],\n",
      " ['ultraflat graphene (UFG)'],\n",
      " ['vitrified thin ice']]\n",
      "Elapsed time: 12.35 seconds\n",
      "-------------------\n",
      "36585454\n",
      "Multifaceted atlases of the human brain in its infancy.\n",
      "Brain atlases are spatial references for integrating, processing, and analyzing brain features\n",
      "gathered from different individuals, sources, and scales. Here we introduce a collection of joint\n",
      "surface-volume atlases that chart postnatal development of the human brain in a spatiotemporally\n",
      "dense manner from two weeks to two years of age. Our month-specific atlases chart normative patterns\n",
      "and capture key traits of early brain development and are therefore conducive to identifying\n",
      "aberrations from normal developmental trajectories. These atlases will enhance our understanding of\n",
      "early structural and functional development by facilitating the mapping of diverse features of the\n",
      "infant brain to a common reference frame for precise multifaceted quantification of cortical and\n",
      "subcortical changes.\n",
      "['surface-volume atlases',\n",
      " 'joint atlases',\n",
      " 'month-specific atlases',\n",
      " 'normative patterns',\n",
      " 'key traits of early brain development']\n",
      "Elapsed time: 9.05 seconds\n",
      "-------------------\n",
      "36585455\n",
      "Local shape descriptors for neuron segmentation.\n",
      "We present an auxiliary learning task for the problem of neuron segmentation in electron microscopy\n",
      "volumes. The auxiliary task consists of the prediction of local shape descriptors (LSDs), which we\n",
      "combine with conventional voxel-wise direct neighbor affinities for neuron boundary detection. The\n",
      "shape descriptors capture local statistics about the neuron to be segmented, such as diameter,\n",
      "elongation, and direction. On a study comparing several existing methods across various specimen,\n",
      "imaging techniques, and resolutions, auxiliary learning of LSDs consistently increases segmentation\n",
      "accuracy of affinity-based methods over a range of metrics. Furthermore, the addition of LSDs\n",
      "promotes affinity-based segmentation methods to be on par with the current state of the art for\n",
      "neuron segmentation (flood-filling networks), while being two orders of magnitudes more efficient-a\n",
      "critical requirement for the processing of future petabyte-sized datasets.\n",
      "{'imaging_techniques': ['electron microscopy', 'flood-filling networks']}\n",
      "Elapsed time: 9.45 seconds\n",
      "-------------------\n",
      "36585456\n",
      "Optimizing multiplexed imaging experimental design through tissue spatial segregation estimation.\n",
      "Recent advances in multiplexed imaging methods allow simultaneous detection of dozens of proteins\n",
      "and hundreds of RNAs, enabling deep spatial characterization of both healthy and diseased tissues.\n",
      "Parameters for the design of optimal multiplex imaging studies, especially those estimating how much\n",
      "area has to be imaged to capture all cell phenotype clusters, are lacking. Here, using a spatial\n",
      "transcriptomic atlas of healthy and tumor human tissues, we developed a statistical framework that\n",
      "determines the number and area of fields of view necessary to accurately identify all cell\n",
      "phenotypes that are part of a tissue. Using this strategy on imaging mass cytometry data, we\n",
      "identified a measurement of tissue spatial segregation that enables optimal experimental design.\n",
      "This strategy will enable an improved design of multiplexed imaging studies.\n",
      "['imaging mass cytometry',\n",
      " 'multiplexed imaging methods',\n",
      " 'spatial transcriptomic atlas']\n",
      "Elapsed time: 8.66 seconds\n",
      "-------------------\n",
      "36624211\n",
      "Video-based pooled screening yields improved far-red genetically encoded voltage indicators.\n",
      "Video-based screening of pooled libraries is a powerful approach for directed evolution of\n",
      "biosensors because it enables selection along multiple dimensions simultaneously from large\n",
      "libraries. Here we develop a screening platform, Photopick, which achieves precise phenotype-\n",
      "activated photoselection over a large field of view (2.3 × 2.3 mm, containing >10<sup>3</sup> cells,\n",
      "per shot). We used the Photopick platform to evolve archaerhodopsin-derived genetically encoded\n",
      "voltage indicators (GEVIs) with improved signal-to-noise ratio (QuasAr6a) and kinetics (QuasAr6b).\n",
      "These GEVIs gave improved signals in cultured neurons and in live mouse brains. By combining\n",
      "targeted in vivo optogenetic stimulation with high-precision voltage imaging, we characterized\n",
      "inhibitory synaptic coupling between individual cortical NDNF (neuron-derived neurotrophic factor)\n",
      "interneurons, and excitatory electrical synapses between individual hippocampal parvalbumin neurons.\n",
      "The QuasAr6 GEVIs are powerful tools for all-optical electrophysiology and the Photopick approach\n",
      "could be adapted to evolve a broad range of biosensors.\n",
      "['Video-based screening',\n",
      " 'photoselection',\n",
      " 'large field of view',\n",
      " 'high-precision voltage imaging']\n",
      "Elapsed time: 13.00 seconds\n",
      "-------------------\n",
      "36624212\n",
      "Robust single-cell matching and multimodal analysis using shared and distinct features.\n",
      "The ability to align individual cellular information from multiple experimental sources is\n",
      "fundamental for a systems-level understanding of biological processes. However, currently available\n",
      "tools are mainly designed for single-cell transcriptomics matching and integration, and generally\n",
      "rely on a large number of shared features across datasets for cell matching. This approach\n",
      "underperforms when applied to single-cell proteomic datasets due to the limited number of parameters\n",
      "simultaneously accessed and lack of shared markers across these experiments. Here, we introduce a\n",
      "cell-matching algorithm, matching with partial overlap (MARIO) that accounts for both shared and\n",
      "distinct features, while consisting of vital filtering steps to avoid suboptimal matching. MARIO\n",
      "accurately matches and integrates data from different single-cell proteomic and multimodal methods,\n",
      "including spatial techniques and has cross-species capabilities. MARIO robustly matched tissue\n",
      "macrophages identified from COVID-19 lung autopsies via codetection by indexing imaging to\n",
      "macrophages recovered from COVID-19 bronchoalveolar lavage fluid by cellular indexing of\n",
      "transcriptomes and epitopes by sequencing, revealing unique immune responses within the lung\n",
      "microenvironment of patients with COVID.\n",
      "['spatial techniques',\n",
      " 'cellular indexing of transcriptomes and epitopes by sequencing']\n",
      "Elapsed time: 10.23 seconds\n",
      "-------------------\n",
      "36646896\n",
      "Integrated multimodality microscope for accurate and efficient target-guided cryo-lamellae\n",
      "preparation.\n",
      "Cryo-electron tomography (cryo-ET) is a revolutionary technique for resolving the structure of\n",
      "subcellular organelles and macromolecular complexes in their cellular context. However, the\n",
      "application of the cryo-ET is hampered by the sample preparation step. Performing cryo-focused ion\n",
      "beam milling at an arbitrary position on the sample is inefficient, and the target of interest is\n",
      "not guaranteed to be preserved when thinning the cell from several micrometers to less than 300 nm\n",
      "thick. Here, we report a cryogenic correlated light, ion and electron microscopy (cryo-CLIEM)\n",
      "technique that is capable of preparing cryo-lamellae under the guidance of three-dimensional\n",
      "confocal imaging. Moreover, we demonstrate a workflow to preselect and preserve nanoscale target\n",
      "regions inside the finished cryo-lamellae. By successfully preparing cryo-lamellae that contain a\n",
      "single centriole or contact sites between subcellular organelles, we show that this approach is\n",
      "generally applicable, and shall help in innovating more applications of cryo-ET.\n",
      "['cryo-electron tomography',\n",
      " 'cryo-focused ion beam milling',\n",
      " 'confocal imaging',\n",
      " 'correlated light, ion and electron microscopy']\n",
      "Elapsed time: 11.84 seconds\n",
      "-------------------\n",
      "36646897\n",
      "ELI trifocal microscope: a precise system to prepare target cryo-lamellae for in situ cryo-ET study.\n",
      "Cryo-electron tomography (cryo-ET) has become a powerful approach to study the high-resolution\n",
      "structure of cellular macromolecular machines in situ. However, the current correlative cryo-\n",
      "fluorescence and electron microscopy lacks sufficient accuracy and efficiency to precisely prepare\n",
      "cryo-lamellae of target locations for subsequent cryo-ET. Here we describe a precise cryogenic\n",
      "fabrication system, ELI-TriScope, which sets electron (E), light (L) and ion (I) beams at the same\n",
      "focal point to achieve accurate and efficient preparation of a target cryo-lamella. ELI-TriScope\n",
      "uses a commercial dual-beam scanning electron microscope modified to incorporate a cryo-holder-based\n",
      "transfer system and embed an optical imaging system just underneath the vitrified specimen. Cryo-\n",
      "focused ion beam milling can be accurately navigated by monitoring the real-time fluorescence signal\n",
      "of the target molecule. Using ELI-TriScope, we prepared a batch of cryo-lamellae of HeLa cells\n",
      "targeting the centrosome with a success rate of ~91% and discovered new in situ structural features\n",
      "of the human centrosome by cryo-ET.\n",
      "['electron microscopy', 'cryo-fluorescence microscopy', 'ion beam milling']\n",
      "Elapsed time: 12.11 seconds\n",
      "-------------------\n",
      "36658278\n",
      "HyU: Hybrid Unmixing for longitudinal in vivo imaging of low signal-to-noise fluorescence.\n",
      "The expansion of fluorescence bioimaging toward more complex systems and geometries requires\n",
      "analytical tools capable of spanning widely varying timescales and length scales, cleanly separating\n",
      "multiple fluorescent labels and distinguishing these labels from background autofluorescence. Here\n",
      "we meet these challenging objectives for multispectral fluorescence microscopy, combining\n",
      "hyperspectral phasors and linear unmixing to create Hybrid Unmixing (HyU). HyU is efficient and\n",
      "robust, capable of quantitative signal separation even at low illumination levels. In dynamic\n",
      "imaging of developing zebrafish embryos and in mouse tissue, HyU was able to cleanly and efficiently\n",
      "unmix multiple fluorescent labels, even in demanding volumetric timelapse imaging settings. HyU\n",
      "permits high dynamic range imaging, allowing simultaneous imaging of bright exogenous labels and dim\n",
      "endogenous labels. This enables coincident studies of tagged components, cellular behaviors and\n",
      "cellular metabolism within the same specimen, providing more accurate insights into the orchestrated\n",
      "complexity of biological systems.\n",
      "['hybrid unmixing',\n",
      " 'hyperspectral phasors',\n",
      " 'linear unmixing',\n",
      " 'multispectral fluorescence microscopy']\n",
      "Elapsed time: 11.25 seconds\n",
      "-------------------\n",
      "36690741\n",
      "Convolutional networks for supervised mining of molecular patterns within cellular context.\n",
      "Cryo-electron tomograms capture a wealth of structural information on the molecular constituents of\n",
      "cells and tissues. We present DeePiCt (deep picker in context), an open-source deep-learning\n",
      "framework for supervised segmentation and macromolecular complex localization in cryo-electron\n",
      "tomography. To train and benchmark DeePiCt on experimental data, we comprehensively annotated 20\n",
      "tomograms of Schizosaccharomyces pombe for ribosomes, fatty acid synthases, membranes, nuclear pore\n",
      "complexes, organelles, and cytosol. By comparing DeePiCt to state-of-the-art approaches on this\n",
      "dataset, we show its unique ability to identify low-abundance and low-density complexes. We use\n",
      "DeePiCt to study compositionally distinct subpopulations of cellular ribosomes, with emphasis on\n",
      "their contextual association with mitochondria and the endoplasmic reticulum. Finally, applying pre-\n",
      "trained networks to a HeLa cell tomogram demonstrates that DeePiCt achieves high-quality predictions\n",
      "in unseen datasets from different biological species in a matter of minutes. The comprehensively\n",
      "annotated experimental data and pre-trained networks are provided for immediate use by the\n",
      "community.\n",
      "['cryo-electron tomography',\n",
      " 'deep learning',\n",
      " 'supervised segmentation',\n",
      " 'macromolecular complex localization']\n",
      "Elapsed time: 12.00 seconds\n",
      "-------------------\n",
      "36717692\n",
      "LILAC: enhanced actin imaging with an optogenetic Lifeact.\n",
      "Lifeact is a popular peptide-based label of actin filaments in live cells. We have designed an\n",
      "improved Lifeact variant, LILAC, that binds to actin in light using the LOV2 protein. Light control\n",
      "allows the user to modulate actin labeling, enabling image analysis that leverages modulation for an\n",
      "enhanced view of F-actin dynamics in cells. Furthermore, the tool reduces actin perturbations and\n",
      "cell sickness caused by Lifeact overexpression.\n",
      "['LOV2 protein', 'optogenetic']\n",
      "Elapsed time: 6.66 seconds\n",
      "-------------------\n",
      "36765136\n",
      "A framework for evaluating the performance of SMLM cluster analysis algorithms.\n",
      "Single-molecule localization microscopy (SMLM) generates data in the form of coordinates of\n",
      "localized fluorophores. Cluster analysis is an attractive route for extracting biologically\n",
      "meaningful information from such data and has been widely applied. Despite a range of cluster\n",
      "analysis algorithms, there exists no consensus framework for the evaluation of their performance.\n",
      "Here, we use a systematic approach based on two metrics to score the success of clustering\n",
      "algorithms in simulated conditions mimicking experimental data. We demonstrate the framework using\n",
      "seven diverse analysis algorithms: DBSCAN, ToMATo, KDE, FOCAL, CAML, ClusterViSu and SR-Tesseler.\n",
      "Given that the best performer depended on the underlying distribution of localizations, we\n",
      "demonstrate an analysis pipeline based on statistical similarity measures that enables the selection\n",
      "of the most appropriate algorithm, and the optimized analysis parameters for real SMLM data. We\n",
      "propose that these standard simulated conditions, metrics and analysis pipeline become the basis for\n",
      "future analysis algorithm development and evaluation.\n",
      "['DBSCAN',\n",
      " 'ToMATo',\n",
      " 'KDE',\n",
      " 'FOCAL',\n",
      " 'CAML',\n",
      " 'ClusterViSu',\n",
      " 'SR-Tesseler',\n",
      " 'single-molecule localization microscopy']\n",
      "Elapsed time: 11.11 seconds\n",
      "-------------------\n",
      "36797410\n",
      "Super-resolution SRS microscopy with A-PoD.\n",
      "Stimulated Raman scattering (SRS) offers the ability to image metabolic dynamics with high signal-\n",
      "to-noise ratio. However, its spatial resolution is limited by the numerical aperture of the imaging\n",
      "objective and the scattering cross-section of molecules. To achieve super-resolved SRS imaging, we\n",
      "developed a deconvolution algorithm, adaptive moment estimation (Adam) optimization-based\n",
      "pointillism deconvolution (A-PoD) and demonstrated a spatial resolution of lower than 59 nm on the\n",
      "membrane of a single lipid droplet (LD). We applied A-PoD to spatially correlated multiphoton\n",
      "fluorescence imaging and deuterium oxide (D<sub>2</sub>O)-probed SRS (DO-SRS) imaging from diverse\n",
      "samples to compare nanoscopic distributions of proteins and lipids in cells and subcellular\n",
      "organelles. We successfully differentiated newly synthesized lipids in LDs using A-PoD-coupled DO-\n",
      "SRS. The A-PoD-enhanced DO-SRS imaging method was also applied to reveal metabolic changes in brain\n",
      "samples from Drosophila on different diets. This new approach allows us to quantitatively measure\n",
      "the nanoscopic colocalization of biomolecules and metabolic dynamics in organelles.\n",
      "['stimulated Raman scattering (SRS)',\n",
      " 'deconvolution algorithm',\n",
      " 'adaptive moment estimation (Adam)',\n",
      " 'pointillism deconvolution (A-PoD)',\n",
      " 'multiphoton fluorescence imaging',\n",
      " 'deuterium oxide (D2O)-probed SRS (DO-SRS) imaging']\n",
      "Elapsed time: 14.34 seconds\n",
      "-------------------\n",
      "36823329\n",
      "Miniature three-photon microscopy maximized for scattered fluorescence collection.\n",
      "In deep-tissue multiphoton microscopy, diffusion and scattering of fluorescent photons, rather than\n",
      "ballistic emanation from the focal point, have been a confounding factor. Here we report on a 2.17-g\n",
      "miniature three-photon microscope (m3PM) with a configuration that maximizes fluorescence collection\n",
      "when imaging in highly scattering regimes. We demonstrate its capability by imaging calcium activity\n",
      "throughout the entire cortex and dorsal hippocampal CA1, up to 1.2 mm depth, at a safe laser power.\n",
      "It also enables the detection of sensorimotor behavior-correlated activities of layer 6 neurons in\n",
      "the posterior parietal cortex in freely moving mice during single-pellet reaching tasks. Thus, m3PM-\n",
      "empowered imaging allows the study of neural mechanisms in deep cortex and subcortical structures,\n",
      "like the dorsal hippocampus and dorsal striatum, in freely behaving animals.\n",
      "['multiphoton microscopy', 'three-photon microscopy', 'fluorescence collection']\n",
      "Elapsed time: 10.41 seconds\n",
      "-------------------\n",
      "36823333\n",
      "Mesoscale volumetric light-field (MesoLF) imaging of neuroactivity across cortical areas at 18 Hz.\n",
      "Various implementations of mesoscopes provide optical access for calcium imaging across multi-\n",
      "millimeter fields of view in the mammalian brain; however, capturing the activity of the neuronal\n",
      "population within such fields of view near-simultaneously and in a volumetric fashion has remained\n",
      "challenging as approaches for imaging scattering brain tissues typically are based on sequential\n",
      "acquisition. Here we present a modular, mesoscale light-field (MesoLF) imaging hardware and software\n",
      "solution that allows recording from thousands of neurons within volumes of ⌀ 4 × 0.2 mm, located at\n",
      "up to 350 µm depth in the mouse cortex, at 18 volumes per second and an effective voxel rate of ~40\n",
      "megavoxels per second. Using our optical design and computational approach we show recording of\n",
      "~10,000 neurons across multiple cortical areas in mice using workstation-grade computing resources.\n",
      "['Mesoscale volumetric light-field (MesoLF) imaging',\n",
      " 'calcium imaging',\n",
      " 'modular, mesoscale light-field (MesoLF) imaging hardware and software '\n",
      " 'solution',\n",
      " 'sequential acquisition']\n",
      "Elapsed time: 11.41 seconds\n",
      "-------------------\n",
      "36823335\n",
      "Field-dependent deep learning enables high-throughput whole-cell 3D super-resolution imaging.\n",
      "Single-molecule localization microscopy in a typical wide-field setup has been widely used for\n",
      "investigating subcellular structures with super resolution; however, field-dependent aberrations\n",
      "restrict the field of view (FOV) to only tens of micrometers. Here, we present a deep-learning\n",
      "method for precise localization of spatially variant point emitters (FD-DeepLoc) over a large FOV\n",
      "covering the full chip of a modern sCMOS camera. Using a graphic processing unit-based vectorial\n",
      "point spread function (PSF) fitter, we can fast and accurately model the spatially variant PSF of a\n",
      "high numerical aperture objective in the entire FOV. Combined with deformable mirror-based optimal\n",
      "PSF engineering, we demonstrate high-accuracy three-dimensional single-molecule localization\n",
      "microscopy over a volume of ~180 × 180 × 5 μm<sup>3</sup>, allowing us to image mitochondria and\n",
      "nuclear pore complexes in entire cells in a single imaging cycle without hardware scanning; a\n",
      "100-fold increase in throughput compared to the state of the art.\n",
      "['single-molecule localization microscopy',\n",
      " 'wide-field setup',\n",
      " 'deformable mirror-based optimal PSF engineering',\n",
      " 'vectorial point spread function (PSF) fitter']\n",
      "Elapsed time: 11.94 seconds\n",
      "-------------------\n",
      "36849549\n",
      "Self-supervised machine learning pushes the sensitivity limit in label-free detection of single\n",
      "proteins below 10 kDa.\n",
      "Interferometric scattering (iSCAT) microscopy is a label-free optical method capable of detecting\n",
      "single proteins, localizing their binding positions with nanometer precision, and measuring their\n",
      "mass. In the ideal case, iSCAT is limited by shot noise such that collection of more photons should\n",
      "extend its detection sensitivity to biomolecules of arbitrarily low mass. However, a number of\n",
      "technical noise sources combined with speckle-like background fluctuations have restricted the\n",
      "detection limit in iSCAT. Here, we show that an unsupervised machine learning isolation forest\n",
      "algorithm for anomaly detection pushes the mass sensitivity limit by a factor of 4 to below 10 kDa.\n",
      "We implement this scheme both with a user-defined feature matrix and a self-supervised FastDVDNet\n",
      "and validate our results with correlative fluorescence images recorded in total internal reflection\n",
      "mode. Our work opens the door to optical investigations of small traces of biomolecules and disease\n",
      "markers such as α-synuclein, chemokines and cytokines.\n",
      "{'imagingTechniques': ['interferometric scattering (iSCAT) microscopy',\n",
      "                       'anomaly detection using unsupervised machine learning',\n",
      "                       'FastDVDNet']}\n",
      "Elapsed time: 11.51 seconds\n",
      "-------------------\n",
      "36864197\n",
      "In silico tissue generation and power analysis for spatial omics.\n",
      "As spatially resolved multiplex profiling of RNA and proteins becomes more prominent, it is\n",
      "increasingly important to understand the statistical power available to test specific hypotheses\n",
      "when designing and interpreting such experiments. Ideally, it would be possible to create an oracle\n",
      "that predicts sampling requirements for generalized spatial experiments. However, the unknown number\n",
      "of relevant spatial features and the complexity of spatial data analysis make this challenging.\n",
      "Here, we enumerate multiple parameters of interest that should be considered in the design of a\n",
      "properly powered spatial omics study. We introduce a method for tunable in silico tissue (IST)\n",
      "generation and use it with spatial profiling data sets to construct an exploratory computational\n",
      "framework for spatial power analysis. Finally, we demonstrate that our framework can be applied\n",
      "across diverse spatial data modalities and tissues of interest. While we demonstrate ISTs in the\n",
      "context of spatial power analysis, these simulated tissues have other potential use cases, including\n",
      "spatial method benchmarking and optimization.\n",
      "['in silico tissue generation',\n",
      " 'spatial profiling',\n",
      " 'computational framework',\n",
      " 'tunable in silico tissue (IST) generation']\n",
      "Elapsed time: 9.67 seconds\n",
      "-------------------\n",
      "36894684\n",
      "Rapid biomechanical imaging at low irradiation level via dual line-scanning Brillouin microscopy.\n",
      "Brillouin microscopy is a technique for mechanical characterization of biological material without\n",
      "contact at high three-dimensional resolution. Here, we introduce dual line-scanning Brillouin\n",
      "microscopy (dLSBM), which improves acquisition speed and reduces irradiation dose by more than one\n",
      "order of magnitude with selective illumination and single-shot analysis of hundreds of points along\n",
      "the incident beam axis. Using tumor spheroids, we demonstrate the ability to capture the sample\n",
      "response to rapid mechanical perturbations as well as the spatially resolved evolution of the\n",
      "mechanical properties in growing spheroids.\n",
      "['Brillouin microscopy',\n",
      " 'dual line-scanning Brillouin microscopy (dLSBM)',\n",
      " 'selective illumination',\n",
      " 'single-shot analysis',\n",
      " 'rapid mechanical perturbations']\n",
      "Elapsed time: 9.05 seconds\n",
      "-------------------\n",
      "36914814\n",
      "Cryo-electron tomography on focused ion beam lamellae transforms structural cell biology.\n",
      "Cryogenic electron microscopy and data processing enable the determination of structures of isolated\n",
      "macromolecules to near-atomic resolution. However, these data do not provide structural information\n",
      "in the cellular environment where macromolecules perform their native functions, and vital molecular\n",
      "interactions can be lost during the isolation process. Cryogenic focused ion beam (FIB) fabrication\n",
      "generates thin lamellae of cellular samples and tissues, enabling structural studies on the near-\n",
      "native cellular interior and its surroundings by cryogenic electron tomography (cryo-ET). Cellular\n",
      "cryo-ET benefits from the technological developments in electron microscopes, detectors and data\n",
      "processing, and more in situ structures are being obtained and at increasingly higher resolution. In\n",
      "this Review, we discuss recent studies employing cryo-ET on FIB-generated lamellae and the\n",
      "technological developments in ultrarapid sample freezing, FIB fabrication of lamellae, tomography,\n",
      "data processing and correlative light and electron microscopy that have enabled these studies.\n",
      "Finally, we explore the future of cryo-ET in terms of both methods development and biological\n",
      "application.\n",
      "['cryogenic electron microscopy',\n",
      " 'electron tomography',\n",
      " 'ultrarapid sample freezing',\n",
      " 'focused ion beam fabrication',\n",
      " 'correlative light and electron microscopy']\n",
      "Elapsed time: 11.81 seconds\n",
      "-------------------\n",
      "36928075\n",
      "An optical design enabling lightweight and large field-of-view head-mounted microscopes.\n",
      "Here we present a fluorescence microscope light path that enables imaging, during free behavior, of\n",
      "thousands of neurons in mice and hundreds of neurons in juvenile songbirds. The light path\n",
      "eliminates traditional illumination optics, allowing for head-mounted microscopes that have both a\n",
      "lower weight and a larger field of view (FOV) than previously possible. Using this light path, we\n",
      "designed two microscopes: one optimized for FOV (~4 mm FOV; 1.4 g), and the other optimized for\n",
      "weight (1.0 mm FOV; 1.0 g).\n",
      "['fluorescence microscope', 'head-mounted microscopes', 'light path']\n",
      "Elapsed time: 8.59 seconds\n",
      "-------------------\n",
      "36959322\n",
      "Cue: a deep-learning framework for structural variant discovery and genotyping.\n",
      "Structural variants (SVs) are a major driver of genetic diversity and disease in the human genome\n",
      "and their discovery is imperative to advances in precision medicine. Existing SV callers rely on\n",
      "hand-engineered features and heuristics to model SVs, which cannot scale to the vast diversity of\n",
      "SVs nor fully harness the information available in sequencing datasets. Here we propose an\n",
      "extensible deep-learning framework, Cue, to call and genotype SVs that can learn complex SV\n",
      "abstractions directly from the data. At a high level, Cue converts alignments to images that encode\n",
      "SV-informative signals and uses a stacked hourglass convolutional neural network to predict the\n",
      "type, genotype and genomic locus of the SVs captured in each image. We show that Cue outperforms the\n",
      "state of the art in the detection of several classes of SVs on synthetic and real short-read data\n",
      "and that it can be easily extended to other sequencing platforms, while achieving competitive\n",
      "performance.\n",
      "['images',\n",
      " 'convolutional neural networks',\n",
      " 'stacked hourglass convolutional neural network']\n",
      "Elapsed time: 9.56 seconds\n",
      "-------------------\n",
      "36973547\n",
      "High-speed low-light in vivo two-photon voltage imaging of large neuronal populations.\n",
      "Monitoring spiking activity across large neuronal populations at behaviorally relevant timescales is\n",
      "critical for understanding neural circuit function. Unlike calcium imaging, voltage imaging requires\n",
      "kilohertz sampling rates that reduce fluorescence detection to near shot-noise levels. High-photon\n",
      "flux excitation can overcome photon-limited shot noise, but photobleaching and photodamage restrict\n",
      "the number and duration of simultaneously imaged neurons. We investigated an alternative approach\n",
      "aimed at low two-photon flux, which is voltage imaging below the shot-noise limit. This framework\n",
      "involved developing positive-going voltage indicators with improved spike detection (SpikeyGi and\n",
      "SpikeyGi2); a two-photon microscope ('SMURF') for kilohertz frame rate imaging across a\n",
      "0.4 mm × 0.4 mm field of view; and a self-supervised denoising algorithm (DeepVID) for inferring\n",
      "fluorescence from shot-noise-limited signals. Through these combined advances, we achieved\n",
      "simultaneous high-speed deep-tissue imaging of more than 100 densely labeled neurons over 1 hour in\n",
      "awake behaving mice. This demonstrates a scalable approach for voltage imaging across increasing\n",
      "neuronal populations.\n",
      "['two-photon microscopy',\n",
      " 'high-speed imaging',\n",
      " 'low-light imaging',\n",
      " 'voltage imaging',\n",
      " 'fluorescence detection']\n",
      "Elapsed time: 13.15 seconds\n",
      "-------------------\n",
      "36997816\n",
      "ERnet: a tool for the semantic segmentation and quantitative analysis of endoplasmic reticulum\n",
      "topology.\n",
      "The ability to quantify structural changes of the endoplasmic reticulum (ER) is crucial for\n",
      "understanding the structure and function of this organelle. However, the rapid movement and complex\n",
      "topology of ER networks make this challenging. Here, we construct a state-of-the-art semantic\n",
      "segmentation method that we call ERnet for the automatic classification of sheet and tubular ER\n",
      "domains inside individual cells. Data are skeletonized and represented by connectivity graphs,\n",
      "enabling precise and efficient quantification of network connectivity. ERnet generates metrics on\n",
      "topology and integrity of ER structures and quantifies structural change in response to genetic or\n",
      "metabolic manipulation. We validate ERnet using data obtained by various ER-imaging methods from\n",
      "different cell types as well as ground truth images of synthetic ER structures. ERnet can be\n",
      "deployed in an automatic high-throughput and unbiased fashion and identifies subtle changes in ER\n",
      "phenotypes that may inform on disease progression and response to therapy.\n",
      "[{'name': 'synthetic ER structures'}, {'name': 'ER-imaging methods'}]\n",
      "Elapsed time: 9.93 seconds\n",
      "-------------------\n",
      "36997817\n",
      "High-resolution line-scan Brillouin microscopy for live imaging of mechanical properties during\n",
      "embryo development.\n",
      "Brillouin microscopy can assess mechanical properties of biological samples in a three-dimensional\n",
      "(3D), all-optical and hence non-contact fashion, but its weak signals often lead to long imaging\n",
      "times and require an illumination dosage harmful for living organisms. Here, we present a high-\n",
      "resolution line-scanning Brillouin microscope for multiplexed and hence fast 3D imaging of dynamic\n",
      "biological processes with low phototoxicity. The improved background suppression and resolution, in\n",
      "combination with fluorescence light-sheet imaging, enables the visualization of the mechanical\n",
      "properties of cells and tissues over space and time in living organism models such as fruit flies,\n",
      "ascidians and mouse embryos.\n",
      "['Brillouin microscopy',\n",
      " 'high-resolution line-scanning Brillouin microscopy',\n",
      " 'fluorescence light-sheet imaging']\n",
      "Elapsed time: 9.05 seconds\n",
      "-------------------\n",
      "37002377\n",
      "Rapid detection of neurons in widefield calcium imaging datasets after training with synthetic data.\n",
      "Widefield microscopy can provide optical access to multi-millimeter fields of view and thousands of\n",
      "neurons in mammalian brains at video rate. However, tissue scattering and background contamination\n",
      "results in signal deterioration, making the extraction of neuronal activity challenging, laborious\n",
      "and time consuming. Here we present our deep-learning-based widefield neuron finder (DeepWonder),\n",
      "which is trained by simulated functional recordings and effectively works on experimental data to\n",
      "achieve high-fidelity neuronal extraction. Equipped with systematic background contribution priors,\n",
      "DeepWonder conducts neuronal inference with an order-of-magnitude-faster speed and improved accuracy\n",
      "compared with alternative approaches. DeepWonder removes background contaminations and is\n",
      "computationally efficient. Specifically, DeepWonder accomplishes 50-fold signal-to-background ratio\n",
      "enhancement when processing terabytes-scale cortex-wide functional recordings, with over 14,000\n",
      "neurons extracted in 17 h.\n",
      "['widefield microscopy',\n",
      " 'synthetic data',\n",
      " 'deep-learning-based widefield neuron finder']\n",
      "Elapsed time: 10.30 seconds\n",
      "-------------------\n",
      "37024649\n",
      "Julia for biologists.\n",
      "Major computational challenges exist in relation to the collection, curation, processing and\n",
      "analysis of large genomic and imaging datasets, as well as the simulation of larger and more\n",
      "realistic models in systems biology. Here we discuss how a relative newcomer among programming\n",
      "languages-Julia-is poised to meet the current and emerging demands in the computational biosciences\n",
      "and beyond. Speed, flexibility, a thriving package ecosystem and readability are major factors that\n",
      "make high-performance computing and data analysis available to an unprecedented degree. We highlight\n",
      "how Julia's design is already enabling new ways of analyzing biological data and systems, and we\n",
      "provide a list of resources that can facilitate the transition into Julian computing.\n",
      "['imaging datasets', 'simulation', 'models in systems biology']\n",
      "Elapsed time: 7.72 seconds\n",
      "-------------------\n",
      "37024650\n",
      "CD-CODE: crowdsourcing condensate database and encyclopedia.\n",
      "The discovery of biomolecular condensates transformed our understanding of intracellular\n",
      "compartmentalization of molecules. To integrate interdisciplinary scientific knowledge about the\n",
      "function and composition of biomolecular condensates, we developed the crowdsourcing condensate\n",
      "database and encyclopedia ( cd-code.org ). CD-CODE is a community-editable platform, which includes\n",
      "a database of biomolecular condensates based on the literature, an encyclopedia of relevant\n",
      "scientific terms and a crowdsourcing web application. Our platform will accelerate the discovery and\n",
      "validation of biomolecular condensates, and facilitate efforts to understand their role in disease\n",
      "and as therapeutic targets.\n",
      "['database', 'encyclopedia', 'web application']\n",
      "Elapsed time: 7.46 seconds\n",
      "-------------------\n",
      "37024653\n",
      "Improving the sensitivity of in vivo CRISPR off-target detection with DISCOVER-Seq.\n",
      "Discovery of off-target CRISPR-Cas activity in patient-derived cells and animal models is crucial\n",
      "for genome editing applications, but currently exhibits low sensitivity. We demonstrate that\n",
      "inhibition of DNA-dependent protein kinase catalytic subunit accumulates the repair protein MRE11 at\n",
      "CRISPR-Cas-targeted sites, enabling high-sensitivity mapping of off-target sites to positions of\n",
      "MRE11 binding using chromatin immunoprecipitation followed by sequencing. This technique, termed\n",
      "DISCOVER-Seq+, discovered up to fivefold more CRISPR off-target sites in immortalized cell lines,\n",
      "primary human cells and mice compared with previous methods. We demonstrate applicability to ex vivo\n",
      "knock-in of a cancer-directed transgenic T cell receptor in primary human T cells and in vivo\n",
      "adenovirus knock-out of cardiovascular risk gene PCSK9 in mice. Thus, DISCOVER-Seq+ is, to our\n",
      "knowledge, the most sensitive method to-date for discovering off-target genome editing in vivo.\n",
      "['chromatin immunoprecipitation', 'sequencing']\n",
      "Elapsed time: 10.74 seconds\n",
      "-------------------\n",
      "37024654\n",
      "Virtual-scanning light-field microscopy for robust snapshot high-resolution volumetric imaging.\n",
      "High-speed three-dimensional (3D) intravital imaging in animals is useful for studying transient\n",
      "subcellular interactions and functions in health and disease. Light-field microscopy (LFM) provides\n",
      "a computational solution for snapshot 3D imaging with low phototoxicity but is restricted by low\n",
      "resolution and reconstruction artifacts induced by optical aberrations, motion and noise. Here, we\n",
      "propose virtual-scanning LFM (VsLFM), a physics-based deep learning framework to increase the\n",
      "resolution of LFM up to the diffraction limit within a snapshot. By constructing a 40 GB high-\n",
      "resolution scanning LFM dataset across different species, we exploit physical priors between phase-\n",
      "correlated angular views to address the frequency aliasing problem. This enables us to bypass\n",
      "hardware scanning and associated motion artifacts. Here, we show that VsLFM achieves ultrafast 3D\n",
      "imaging of diverse processes such as the beating heart in embryonic zebrafish, voltage activity in\n",
      "Drosophila brains and neutrophil migration in the mouse liver at up to 500 volumes per second.\n",
      "['light-field microscopy (LFM)',\n",
      " 'virtual-scanning light-field microscopy (VsLFM)']\n",
      "Elapsed time: 11.42 seconds\n",
      "-------------------\n",
      "37038000\n",
      "Spatiotemporally resolved transcriptomics reveals the subcellular RNA kinetic landscape.\n",
      "Spatiotemporal regulation of the cellular transcriptome is crucial for proper protein expression and\n",
      "cellular function. However, the intricate subcellular dynamics of RNA remain obscured due to the\n",
      "limitations of existing transcriptomics methods. Here, we report TEMPOmap-a method that uncovers\n",
      "subcellular RNA profiles across time and space at the single-cell level. TEMPOmap integrates pulse-\n",
      "chase metabolic labeling with highly multiplexed three-dimensional in situ sequencing to\n",
      "simultaneously profile the age and location of individual RNA molecules. Using TEMPOmap, we\n",
      "constructed the subcellular RNA kinetic landscape in various human cells from transcription and\n",
      "translocation to degradation. Clustering analysis of RNA kinetic parameters across single cells\n",
      "revealed 'kinetic gene clusters' whose expression patterns were shaped by multistep kinetic\n",
      "sculpting. Importantly, these kinetic gene clusters are functionally segregated, suggesting that\n",
      "subcellular RNA kinetics are differentially regulated in a cell-state- and cell-type-dependent\n",
      "manner. Spatiotemporally resolved transcriptomics provides a gateway to uncovering new\n",
      "spatiotemporal gene regulation principles.\n",
      "['TEMPOmap',\n",
      " 'pulse-chase metabolic labeling',\n",
      " 'three-dimensional in situ sequencing']\n",
      "Elapsed time: 11.30 seconds\n",
      "-------------------\n",
      "37069271\n",
      "BigNeuron: a resource to benchmark and predict performance of algorithms for automated tracing of\n",
      "neurons in light microscopy datasets.\n",
      "BigNeuron is an open community bench-testing platform with the goal of setting open standards for\n",
      "accurate and fast automatic neuron tracing. We gathered a diverse set of image volumes across\n",
      "several species that is representative of the data obtained in many neuroscience laboratories\n",
      "interested in neuron tracing. Here, we report generated gold standard manual annotations for a\n",
      "subset of the available imaging datasets and quantified tracing quality for 35 automatic tracing\n",
      "algorithms. The goal of generating such a hand-curated diverse dataset is to advance the development\n",
      "of tracing algorithms and enable generalizable benchmarking. Together with image quality features,\n",
      "we pooled the data in an interactive web application that enables users and developers to perform\n",
      "principal component analysis, t-distributed stochastic neighbor embedding, correlation and\n",
      "clustering, visualization of imaging and tracing data, and benchmarking of automatic tracing\n",
      "algorithms in user-defined data subsets. The image quality metrics explain most of the variance in\n",
      "the data, followed by neuromorphological features related to neuron size. We observed that diverse\n",
      "algorithms can provide complementary information to obtain accurate results and developed a method\n",
      "to iteratively combine methods and generate consensus reconstructions. The consensus trees obtained\n",
      "provide estimates of the neuron structure ground truth that typically outperform single algorithms\n",
      "in noisy datasets. However, specific algorithms may outperform the consensus tree strategy in\n",
      "specific imaging conditions. Finally, to aid users in predicting the most accurate automatic tracing\n",
      "results without manual annotations for comparison, we used support vector machine regression to\n",
      "predict reconstruction quality given an image volume and a set of automatic tracings.\n",
      "['image volumes',\n",
      " 'light microscopy datasets',\n",
      " 'manual annotations',\n",
      " 'automatic tracing algorithms',\n",
      " 'principal component analysis',\n",
      " 't-distributed stochastic neighbor embedding',\n",
      " 'correlation and clustering',\n",
      " 'imaging and tracing data',\n",
      " 'support vector machine regression']\n",
      "Elapsed time: 15.18 seconds\n",
      "-------------------\n",
      "37081094\n",
      "Engineering of NEMO as calcium indicators with large dynamics and high sensitivity.\n",
      "Genetically encoded calcium indicators (GECIs) are indispensable tools for real-time monitoring of\n",
      "intracellular calcium signals and cellular activities in living organisms. Current GECIs face the\n",
      "challenge of suboptimal peak signal-to-baseline ratio (SBR) with limited resolution for reporting\n",
      "subtle calcium transients. We report herein the development of a suite of calcium sensors,\n",
      "designated NEMO, with fast kinetics and wide dynamic ranges (>100-fold). NEMO indicators report\n",
      "Ca<sup>2+</sup> transients with peak SBRs around 20-fold larger than the top-of-the-range GCaMP6\n",
      "series. NEMO sensors further enable the quantification of absolution calcium concentration with\n",
      "ratiometric or photochromic imaging. Compared with GCaMP6s, NEMOs could detect single action\n",
      "potentials in neurons with a peak SBR two times higher and a median peak SBR four times larger in\n",
      "vivo, thereby outperforming most existing state-of-the-art GECIs. Given their high sensitivity and\n",
      "resolution to report intracellular Ca<sup>2+</sup> signals, NEMO sensors may find broad applications\n",
      "in monitoring neuronal activities and other Ca<sup>2+</sup>-modulated physiological processes in\n",
      "both mammals and plants.\n",
      "['ratiometric imaging',\n",
      " 'photochromic imaging',\n",
      " 'single action potentials in neurons']\n",
      "Elapsed time: 13.26 seconds\n",
      "-------------------\n",
      "37142767\n",
      "Glutamate indicators with improved activation kinetics and localization for imaging synaptic\n",
      "transmission.\n",
      "The fluorescent glutamate indicator iGluSnFR enables imaging of neurotransmission with genetic and\n",
      "molecular specificity. However, existing iGluSnFR variants exhibit low in vivo signal-to-noise\n",
      "ratios, saturating activation kinetics and exclusion from postsynaptic densities. Using a multiassay\n",
      "screen in bacteria, soluble protein and cultured neurons, we generated variants with improved\n",
      "signal-to-noise ratios and kinetics. We developed surface display constructs that improve iGluSnFR's\n",
      "nanoscopic localization to postsynapses. The resulting indicator iGluSnFR3 exhibits rapid\n",
      "nonsaturating activation kinetics and reports synaptic glutamate release with decreased saturation\n",
      "and increased specificity versus extrasynaptic signals in cultured neurons. Simultaneous imaging and\n",
      "electrophysiology at individual boutons in mouse visual cortex showed that iGluSnFR3 transients\n",
      "report single action potentials with high specificity. In vibrissal sensory cortex layer 4, we used\n",
      "iGluSnFR3 to characterize distinct patterns of touch-evoked feedforward input from thalamocortical\n",
      "boutons and both feedforward and recurrent input onto L4 cortical neuron dendritic spines.\n",
      "['imaging synaptic transmission',\n",
      " 'fluorescent glutamate indicator',\n",
      " 'genetic and molecular specificity',\n",
      " 'multiassay screen',\n",
      " 'surface display constructs',\n",
      " 'nanoscopic localization to postsynapses',\n",
      " 'rapid nonsaturating activation kinetics']\n",
      "Elapsed time: 13.93 seconds\n",
      "-------------------\n",
      "37156842\n",
      "EmbryoNet: using deep learning to link embryonic phenotypes to signaling pathways.\n",
      "Evolutionarily conserved signaling pathways are essential for early embryogenesis, and reducing or\n",
      "abolishing their activity leads to characteristic developmental defects. Classification of\n",
      "phenotypic defects can identify the underlying signaling mechanisms, but this requires expert\n",
      "knowledge and the classification schemes have not been standardized. Here we use a machine learning\n",
      "approach for automated phenotyping to train a deep convolutional neural network, EmbryoNet, to\n",
      "accurately identify zebrafish signaling mutants in an unbiased manner. Combined with a model of\n",
      "time-dependent developmental trajectories, this approach identifies and classifies with high\n",
      "precision phenotypic defects caused by loss of function of the seven major signaling pathways\n",
      "relevant for vertebrate development. Our classification algorithms have wide applications in\n",
      "developmental biology and robustly identify signaling defects in evolutionarily distant species.\n",
      "Furthermore, using automated phenotyping in high-throughput drug screens, we show that EmbryoNet can\n",
      "resolve the mechanism of action of pharmaceutical substances. As part of this work, we freely\n",
      "provide more than 2 million images that were used to train and test EmbryoNet.\n",
      "['deep learning',\n",
      " 'convolutional neural network',\n",
      " 'automated phenotyping',\n",
      " 'time-dependent developmental trajectories']\n",
      "Elapsed time: 11.45 seconds\n",
      "-------------------\n",
      "37156843\n",
      "High-contrast en bloc staining of mouse whole-brain and human brain samples for EM-based\n",
      "connectomics.\n",
      "Connectomes of human cortical gray matter require high-contrast homogeneously stained samples sized\n",
      "at least 2 mm on a side, and a mouse whole-brain connectome requires samples sized at least 5-10 mm\n",
      "on a side. Here we report en bloc staining and embedding protocols for these and other applications,\n",
      "removing a key obstacle for connectomic analyses at the mammalian whole-brain level.\n",
      "['en bloc staining', 'embedding', 'electron microscopy (EM)']\n",
      "Elapsed time: 7.33 seconds\n",
      "-------------------\n",
      "37169928\n",
      "Cross-modality supervised image restoration enables nanoscale tracking of synaptic plasticity in\n",
      "living mice.\n",
      "Learning is thought to involve changes in glutamate receptors at synapses, submicron structures that\n",
      "mediate communication between neurons in the central nervous system. Due to their small size and\n",
      "high density, synapses are difficult to resolve in vivo, limiting our ability to directly relate\n",
      "receptor dynamics to animal behavior. Here we developed a combination of computational and\n",
      "biological methods to overcome these challenges. First, we trained a deep-learning image-restoration\n",
      "algorithm that combines the advantages of ex vivo super-resolution and in vivo imaging modalities to\n",
      "overcome limitations specific to each optical system. When applied to in vivo images from transgenic\n",
      "mice expressing fluorescently labeled glutamate receptors, this restoration algorithm super-resolved\n",
      "synapses, enabling the tracking of behavior-associated synaptic plasticity with high spatial\n",
      "resolution. This method demonstrates the capabilities of image enhancement to learn from ex vivo\n",
      "data and imaging techniques to improve in vivo imaging resolution.\n",
      "['ex vivo super-resolution',\n",
      " 'in vivo imaging modalities',\n",
      " 'deep-learning image-restoration algorithm']\n",
      "Elapsed time: 10.34 seconds\n",
      "-------------------\n",
      "37169929\n",
      "3DFlex: determining structure and motion of flexible proteins from cryo-EM.\n",
      "Modeling flexible macromolecules is one of the foremost challenges in single-particle cryogenic-\n",
      "electron microscopy (cryo-EM), with the potential to illuminate fundamental questions in structural\n",
      "biology. We introduce Three-Dimensional Flexible Refinement (3DFlex), a motion-based neural network\n",
      "model for continuous molecular heterogeneity for cryo-EM data. 3DFlex exploits knowledge that\n",
      "conformational variability of a protein is often the result of physical processes that transport\n",
      "density over space and tend to preserve local geometry. From two-dimensional image data, 3DFlex\n",
      "enables the determination of high-resolution 3D density, and provides an explicit model of a\n",
      "flexible protein's motion over its conformational landscape. Experimentally, for large molecular\n",
      "machines (tri-snRNP spliceosome complex, translocating ribosome) and small flexible proteins (TRPV1\n",
      "ion channel, αVβ8 integrin, SARS-CoV-2 spike), 3DFlex learns nonrigid molecular motions while\n",
      "resolving details of moving secondary structure elements. 3DFlex can improve 3D density resolution\n",
      "beyond the limits of existing methods because particle images contribute coherent signal over the\n",
      "conformational landscape.\n",
      "['cryo-EM',\n",
      " 'neural network model',\n",
      " 'motion-based model',\n",
      " 'continuous molecular heterogeneity']\n",
      "Elapsed time: 11.80 seconds\n",
      "-------------------\n",
      "37188953\n",
      "TomoTwin: generalized 3D localization of macromolecules in cryo-electron tomograms with structural\n",
      "data mining.\n",
      "Cryogenic-electron tomography enables the visualization of cellular environments in extreme detail,\n",
      "however, tools to analyze the full amount of information contained within these densely packed\n",
      "volumes are still needed. Detailed analysis of macromolecules through subtomogram averaging requires\n",
      "particles to first be localized within the tomogram volume, a task complicated by several factors\n",
      "including a low signal to noise ratio and crowding of the cellular space. Available methods for this\n",
      "task suffer either from being error prone or requiring manual annotation of training data. To assist\n",
      "in this crucial particle picking step, we present TomoTwin: an open source general picking model for\n",
      "cryogenic-electron tomograms based on deep metric learning. By embedding tomograms in an\n",
      "information-rich, high-dimensional space that separates macromolecules according to their three-\n",
      "dimensional structure, TomoTwin allows users to identify proteins in tomograms de novo without\n",
      "manually creating training data or retraining the network to locate new proteins.\n",
      "['cryo-electron tomograms', 'subtomogram averaging', 'deep metric learning']\n",
      "Elapsed time: 10.45 seconds\n",
      "-------------------\n",
      "37248386\n",
      "Machine learning in rare disease.\n",
      "High-throughput profiling methods (such as genomics or imaging) have accelerated basic research and\n",
      "made deep molecular characterization of patient samples routine. These approaches provide a rich\n",
      "portrait of genes, molecular pathways and cell types involved in disease phenotypes. Machine\n",
      "learning (ML) can be a useful tool for extracting disease-relevant patterns from high-dimensional\n",
      "datasets. However, depending upon the complexity of the biological question, machine learning often\n",
      "requires many samples to identify recurrent and biologically meaningful patterns. Rare diseases are\n",
      "inherently limited in clinical cases, leading to few samples to study. In this Perspective, we\n",
      "outline the challenges and emerging solutions for using ML for small sample sets, specifically in\n",
      "rare diseases. Advances in ML methods for rare diseases are likely to be informative for\n",
      "applications beyond rare diseases for which few samples exist with high-dimensional data. We propose\n",
      "that the method community prioritize the development of ML techniques for rare disease research.\n",
      "{'imaging techniques': ['genomics', 'imaging']}\n",
      "Elapsed time: 10.02 seconds\n",
      "-------------------\n",
      "37248387\n",
      "New measures of anisotropy of cryo-EM maps.\n",
      "We propose two new measures of resolution anisotropy for cryogenic electron microscopy maps: Fourier\n",
      "shell occupancy (FSO), and the Bingham test (BT). FSO varies from 1 to 0, with 1 representing\n",
      "perfect isotropy, and lower values indicating increasing anisotropy. The threshold FSO = 0.5 occurs\n",
      "at Fourier shell correlation resolution. BT is a hypothesis test that complements the FSO to ensure\n",
      "the existence of anisotropy. FSO and BT allow visualization of resolution anisotropy. We illustrate\n",
      "their use with different experimental cryogenic electron microscopy maps.\n",
      "['Fourier shell occupancy (FSO)', 'Bingham test (BT)']\n",
      "Elapsed time: 7.64 seconds\n",
      "-------------------\n",
      "37248388\n",
      "Learning consistent subcellular landmarks to quantify changes in multiplexed protein maps.\n",
      "Highly multiplexed imaging holds enormous promise for understanding how spatial context shapes the\n",
      "activity of the genome and its products at multiple length scales. Here, we introduce a deep\n",
      "learning framework called CAMPA (Conditional Autoencoder for Multiplexed Pixel Analysis), which uses\n",
      "a conditional variational autoencoder to learn representations of molecular pixel profiles that are\n",
      "consistent across heterogeneous cell populations and experimental perturbations. Clustering these\n",
      "pixel-level representations identifies consistent subcellular landmarks, which can be quantitatively\n",
      "compared in terms of their size, shape, molecular composition and relative spatial organization.\n",
      "Using high-resolution multiplexed immunofluorescence, this reveals how subcellular organization\n",
      "changes upon perturbation of RNA synthesis, RNA processing or cell size, and uncovers links between\n",
      "the molecular composition of membraneless organelles and cell-to-cell variability in bulk RNA\n",
      "synthesis rates. By capturing interpretable cellular phenotypes, we anticipate that CAMPA will\n",
      "greatly accelerate the systematic mapping of multiscale atlases of biological organization to\n",
      "identify the rules by which context shapes physiology and disease.\n",
      "['high-resolution multiplexed immunofluorescence',\n",
      " 'conditional variational autoencoder']\n",
      "Elapsed time: 10.98 seconds\n",
      "-------------------\n",
      "37429962\n",
      "A positively tuned voltage indicator for extended electrical recordings in the brain.\n",
      "Genetically encoded voltage indicators (GEVIs) enable optical recording of electrical signals in the\n",
      "brain, providing subthreshold sensitivity and temporal resolution not possible with calcium\n",
      "indicators. However, one- and two-photon voltage imaging over prolonged periods with the same GEVI\n",
      "has not yet been demonstrated. Here, we report engineering of ASAP family GEVIs to enhance\n",
      "photostability by inversion of the fluorescence-voltage relationship. Two of the resulting GEVIs,\n",
      "ASAP4b and ASAP4e, respond to 100-mV depolarizations with ≥180% fluorescence increases, compared\n",
      "with the 50% fluorescence decrease of the parental ASAP3. With standard microscopy equipment, ASAP4e\n",
      "enables single-trial detection of spikes in mice over the course of minutes. Unlike GEVIs previously\n",
      "used for one-photon voltage recordings, ASAP4b and ASAP4e also perform well under two-photon\n",
      "illumination. By imaging voltage and calcium simultaneously, we show that ASAP4b and ASAP4e can\n",
      "identify place cells and detect voltage spikes with better temporal resolution than commonly used\n",
      "calcium indicators. Thus, ASAP4b and ASAP4e extend the capabilities of voltage imaging to standard\n",
      "one- and two-photon microscopes while improving the duration of voltage recordings.\n",
      "['one-photon voltage recordings',\n",
      " 'two-photon voltage recordings',\n",
      " 'calcium imaging']\n",
      "Elapsed time: 12.38 seconds\n",
      "-------------------\n",
      "37429992\n",
      "SCS: cell segmentation for high-resolution spatial transcriptomics.\n",
      "Spatial transcriptomics promises to greatly improve our understanding of tissue organization and\n",
      "cell-cell interactions. While most current platforms for spatial transcriptomics only offer multi-\n",
      "cellular resolution, with 10-15 cells per spot, recent technologies provide a much denser spot\n",
      "placement leading to subcellular resolution. A key challenge for these newer methods is cell\n",
      "segmentation and the assignment of spots to cells. Traditional image-based segmentation methods are\n",
      "limited and do not make full use of the information profiled by spatial transcriptomics. Here we\n",
      "present subcellular spatial transcriptomics cell segmentation (SCS), which combines imaging data\n",
      "with sequencing data to improve cell segmentation accuracy. SCS assigns spots to cells by adaptively\n",
      "learning the position of each spot relative to the center of its cell using a transformer neural\n",
      "network. SCS was tested on two new subcellular spatial transcriptomics technologies and outperformed\n",
      "traditional image-based segmentation methods. SCS achieved better accuracy, identified more cells\n",
      "and provided more realistic cell size estimation. Subcellular analysis of RNAs using SCS spot\n",
      "assignments provides information on RNA localization and further supports the segmentation results.\n",
      "['transformer neural network', 'spatial transcriptomics']\n",
      "Elapsed time: 10.87 seconds\n",
      "-------------------\n",
      "37429995\n",
      "Dense 4D nanoscale reconstruction of living brain tissue.\n",
      "Three-dimensional (3D) reconstruction of living brain tissue down to an individual synapse level\n",
      "would create opportunities for decoding the dynamics and structure-function relationships of the\n",
      "brain's complex and dense information processing network; however, this has been hindered by\n",
      "insufficient 3D resolution, inadequate signal-to-noise ratio and prohibitive light burden in optical\n",
      "imaging, whereas electron microscopy is inherently static. Here we solved these challenges by\n",
      "developing an integrated optical/machine-learning technology, LIONESS (live information-optimized\n",
      "nanoscopy enabling saturated segmentation). This leverages optical modifications to stimulated\n",
      "emission depletion microscopy in comprehensively, extracellularly labeled tissue and previous\n",
      "information on sample structure via machine learning to simultaneously achieve isotropic super-\n",
      "resolution, high signal-to-noise ratio and compatibility with living tissue. This allows dense deep-\n",
      "learning-based instance segmentation and 3D reconstruction at a synapse level, incorporating\n",
      "molecular, activity and morphodynamic information. LIONESS opens up avenues for studying the dynamic\n",
      "functional (nano-)architecture of living brain tissue.\n",
      "['stimulated emission depletion microscopy',\n",
      " 'machine learning',\n",
      " 'isotropic super-resolution',\n",
      " 'high signal-to-noise ratio']\n",
      "Elapsed time: 11.67 seconds\n",
      "-------------------\n",
      "37468619\n",
      "Organ Mapping Antibody Panels: a community resource for standardized multiplexed tissue imaging.\n",
      "Multiplexed antibody-based imaging enables the detailed characterization of molecular and cellular\n",
      "organization in tissues. Advances in the field now allow high-parameter data collection (>60\n",
      "targets); however, considerable expertise and capital are needed to construct the antibody panels\n",
      "employed by these methods. Organ mapping antibody panels are community-validated resources that save\n",
      "time and money, increase reproducibility, accelerate discovery and support the construction of a\n",
      "Human Reference Atlas.\n",
      "['Organ Mapping Antibody Panels',\n",
      " 'multiplexed antibody-based imaging',\n",
      " 'high-parameter data collection']\n",
      "Elapsed time: 7.60 seconds\n",
      "-------------------\n",
      "37474807\n",
      "Sensitive multicolor indicators for monitoring norepinephrine in vivo.\n",
      "Genetically encoded indicators engineered from G-protein-coupled receptors are important tools that\n",
      "enable high-resolution in vivo neuromodulator imaging. Here, we introduce a family of sensitive\n",
      "multicolor norepinephrine (NE) indicators, which includes nLightG (green) and nLightR (red). These\n",
      "tools report endogenous NE release in vitro, ex vivo and in vivo with improved sensitivity, ligand\n",
      "selectivity and kinetics, as well as a distinct pharmacological profile compared with previous\n",
      "state-of-the-art GRAB<sub>NE</sub> indicators. Using in vivo multisite fiber photometry recordings\n",
      "of nLightG, we could simultaneously monitor optogenetically evoked NE release in the mouse locus\n",
      "coeruleus and hippocampus. Two-photon imaging of nLightG revealed locomotion and reward-related NE\n",
      "transients in the dorsal CA1 area of the hippocampus. Thus, the sensitive NE indicators introduced\n",
      "here represent an important addition to the current repertoire of indicators and provide the means\n",
      "for a thorough investigation of the NE system.\n",
      "['in vivo neuromodulator imaging',\n",
      " 'genetically encoded indicators',\n",
      " 'two-photon imaging',\n",
      " 'fiber photometry recordings',\n",
      " 'optogenetically evoked NE release']\n",
      "Elapsed time: 12.50 seconds\n",
      "-------------------\n",
      "37474808\n",
      "Voltage-Seq: all-optical postsynaptic connectome-guided single-cell transcriptomics.\n",
      "Understanding the routing of neuronal information requires the functional characterization of\n",
      "connections. Neuronal projections recruit large postsynaptic ensembles with distinct postsynaptic\n",
      "response types (PRTs). PRT is typically probed by low-throughput whole-cell electrophysiology and is\n",
      "not a selection criterion for single-cell RNA-sequencing (scRNA-seq). To overcome these limitations\n",
      "and target neurons based on specific PRTs for soma harvesting and subsequent scRNA-seq, we created\n",
      "Voltage-Seq. We established all-optical voltage imaging and recorded the PRT of 8,347 neurons in the\n",
      "mouse periaqueductal gray (PAG) evoked by the optogenetic activation of ventromedial hypothalamic\n",
      "(VMH) terminals. PRTs were classified and spatially resolved in the entire VMH-PAG connectome. We\n",
      "built an onsite analysis tool named VoltView to navigate soma harvesting towards target PRTs guided\n",
      "by a classifier that used the VMH-PAG connectome database as a reference. We demonstrated Voltage-\n",
      "seq by locating VMH-driven γ-aminobutyric acid-ergic neurons in the PAG, guided solely by the onsite\n",
      "classification in VoltView.\n",
      "{'imagingTechniques': ['all-optical voltage imaging',\n",
      "                       'whole-cell electrophysiology',\n",
      "                       'single-cell RNA-sequencing']}\n",
      "Elapsed time: 13.54 seconds\n",
      "-------------------\n",
      "37474809\n",
      "Open-3DSIM: an open-source three-dimensional structured illumination microscopy reconstruction\n",
      "platform.\n",
      "Open-3DSIM is an open-source reconstruction platform for three-dimensional structured illumination\n",
      "microscopy. We demonstrate its superior performance for artifact suppression and high-fidelity\n",
      "reconstruction relative to other algorithms on various specimens and over a range of signal-to-noise\n",
      "levels. Open-3DSIM also offers the capacity to extract dipole orientation, paving a new avenue for\n",
      "interpreting subcellular structures in six dimensions (xyzθλt). The platform is available as MATLAB\n",
      "code, a Fiji plugin and an Exe application to maximize user-friendliness.\n",
      "['three-dimensional structured illumination microscopy',\n",
      " 'MATLAB code',\n",
      " 'Fiji plugin',\n",
      " 'Exe application']\n",
      "Elapsed time: 8.33 seconds\n",
      "-------------------\n",
      "37500758\n",
      "Segmentation metric misinterpretations in bioimage analysis.\n",
      "Quantitative evaluation of image segmentation algorithms is crucial in the field of bioimage\n",
      "analysis. The most common assessment scores, however, are often misinterpreted and multiple\n",
      "definitions coexist with the same name. Here we present the ambiguities of evaluation metrics for\n",
      "segmentation algorithms and show how these misinterpretations can alter leaderboards of influential\n",
      "competitions. We also propose guidelines for how the currently existing problems could be tackled.\n",
      "['evaluation metrics', 'image segmentation algorithms', 'bioimage analysis']\n",
      "Elapsed time: 6.25 seconds\n",
      "-------------------\n",
      "37500760\n",
      "DBlink: dynamic localization microscopy in super spatiotemporal resolution via deep learning.\n",
      "Single-molecule localization microscopy (SMLM) has revolutionized biological imaging, improving the\n",
      "spatial resolution of traditional microscopes by an order of magnitude. However, SMLM techniques\n",
      "require long acquisition times, typically a few minutes, to yield a single super-resolved image,\n",
      "because they depend on accumulation of many localizations over thousands of recorded frames. Hence,\n",
      "the capability of SMLM to observe dynamics at high temporal resolution has always been limited. In\n",
      "this work, we present DBlink, a deep-learning-based method for super spatiotemporal resolution\n",
      "reconstruction from SMLM data. The input to DBlink is a recorded video of SMLM data and the output\n",
      "is a super spatiotemporal resolution video reconstruction. We use a convolutional neural network\n",
      "combined with a bidirectional long short-term memory network architecture, designed for capturing\n",
      "long-term dependencies between different input frames. We demonstrate DBlink performance on\n",
      "simulated filaments and mitochondria-like structures, on experimental SMLM data under controlled\n",
      "motion conditions and on live-cell dynamic SMLM. DBlink's spatiotemporal interpolation constitutes\n",
      "an important advance in super-resolution imaging of dynamic processes in live cells.\n",
      "['single-molecule localization microscopy (SMLM)',\n",
      " 'deep learning',\n",
      " 'super spatiotemporal resolution reconstruction',\n",
      " 'convolutional neural network',\n",
      " 'bidirectional long short-term memory network',\n",
      " 'spatiotemporal interpolation']\n",
      "Elapsed time: 13.42 seconds\n",
      "-------------------\n",
      "15782150\n",
      "Sulforhodamine 101 as a specific marker of astroglia in the neocortex in vivo.\n",
      "Glial cells have been identified as key signaling components in the brain; however, methods to\n",
      "investigate their structure and function in vivo have been lacking. Here, we describe a new, highly\n",
      "selective approach for labeling astrocytes in intact rodent neocortex that allows in vivo imaging\n",
      "using two-photon microscopy. The red fluorescent dye sulforhodamine 101 (SR101) was specifically\n",
      "taken up by protoplasmic astrocytes after brief exposure to the brain surface. Specificity was\n",
      "confirmed by immunohistochemistry. In addition, SR101 labeled enhanced green fluorescent protein\n",
      "(EGFP)-expressing astrocytes but not microglial cells in transgenic mice. We used SR101 labeling to\n",
      "quantify morphological characteristics of astrocytes and to visualize their close association with\n",
      "the cortical microvasculature. Furthermore, by combining this method with calcium indicator loading\n",
      "of cell populations, we demonstrated distinct calcium dynamics in astroglial and neuronal networks.\n",
      "We expect SR101 staining to become a principal tool for investigating astroglia in vivo.\n",
      "['two-photon microscopy',\n",
      " 'sulforhodamine 101 (SR101)',\n",
      " 'immunohistochemistry',\n",
      " 'calcium indicator loading']\n",
      "Elapsed time: 11.99 seconds\n",
      "-------------------\n",
      "15782159\n",
      "Polyene-lipids: a new tool to image lipids.\n",
      "Microscopy of lipids in living cells is currently hampered by a lack of adequate fluorescent tags.\n",
      "The most frequently used tags, NBD and BODIPY, strongly influence the properties of lipids, yielding\n",
      "analogs with quite different characteristics. Here, we introduce polyene-lipids containing five\n",
      "conjugated double bonds as a new type of lipid tag. Polyene-lipids exhibit a unique structural\n",
      "similarity to natural lipids, which results in minimal effects on the lipid properties. Analyzing\n",
      "membrane phase partitioning, an important biophysical and biological property of lipids, we\n",
      "demonstrated the superiority of polyene-lipids to both NBD- and BODIPY-tagged lipids. Cells readily\n",
      "take up various polyene-lipid precursors and generate the expected end products with no apparent\n",
      "disturbance by the tag. Applying two-photon excitation microscopy, we imaged the distribution of\n",
      "polyene-lipids in living mammalian cells. For the first time, ether lipids, important for the\n",
      "function of the brain, were successfully visualized.\n",
      "['microscopy', 'two-photon excitation microscopy']\n",
      "Elapsed time: 10.65 seconds\n",
      "-------------------\n",
      "15782161\n",
      "LAMP, a new imaging assay of gap junctional communication unveils that Ca2+ influx inhibits cell\n",
      "coupling.\n",
      "Using a new class of photo-activatible fluorophores, we have developed a new imaging technique for\n",
      "measuring molecular transfer rates across gap junction connexin channels in intact living cells.\n",
      "This technique, named LAMP, involves local activation of a molecular fluorescent probe, NPE-\n",
      "HCCC2/AM, to optically label a cell. Subsequent dye transfer through gap junctions from labeled to\n",
      "unlabeled cells was quantified by fluorescence microscopy. Additional uncagings after prior dye\n",
      "transfers reached equilibrium enabled multiple measurements of dye transfer rates in the same\n",
      "coupled cell pair. Measurements in the same cell pair minimized variation due to differences in cell\n",
      "volume and number of gap junctions, allowing us to track acute changes in gap junction permeability.\n",
      "We applied the technique to study the regulation of gap junction coupling by intracellular Ca(2+)\n",
      "([Ca(2+)](i)). Although agonist or ionomycin exposure can raise bulk [Ca(2+)](i) to levels higher\n",
      "than those caused by capacitative Ca(2+) influx, the LAMP assay revealed that only Ca(2+) influx\n",
      "through the plasma membrane store-operated Ca(2+) channels strongly reduced gap junction coupling.\n",
      "The noninvasive and quantitative nature of this imaging technique should facilitate future\n",
      "investigations of the dynamic regulation of gap junction communication.\n",
      "['LAMP', 'fluorescence microscopy']\n",
      "Elapsed time: 13.67 seconds\n",
      "-------------------\n",
      "15782196\n",
      "Three-chromophore FRET microscopy to analyze multiprotein interactions in living cells.\n",
      "Nearly every major process in a cell is carried out by assemblies of multiple dynamically\n",
      "interacting protein molecules. To study multi-protein interactions within such molecular\n",
      "machineries, we have developed a fluorescence microscopy method called three-chromophore\n",
      "fluorescence resonance energy transfer (3-FRET). This method allows analysis of three mutually\n",
      "dependent energy transfer processes between the fluorescent labels, such as cyan, yellow and\n",
      "monomeric red fluorescent proteins. Here, we describe both theoretical and experimental approaches\n",
      "that discriminate the parallel versus the sequential energy transfer processes in the 3-FRET system.\n",
      "These approaches were established in vitro and in cultured mammalian cells, using chimeric proteins\n",
      "consisting of two or three fluorescent proteins linked together. The 3-FRET microscopy was further\n",
      "applied to the analysis of three-protein interactions in the constitutive and activation-dependent\n",
      "complexes in single endosomal compartments. These data highlight the potential of 3-FRET microscopy\n",
      "in studies of spatial and temporal regulation of signaling processes in living cells.\n",
      "['fluorescence microscopy',\n",
      " 'fluorescence resonance energy transfer (FRET)',\n",
      " 'three-chromophore FRET microscopy']\n",
      "Elapsed time: 10.67 seconds\n",
      "-------------------\n",
      "15973422\n",
      "Nanoscale mapping and functional analysis of individual adhesins on living bacteria.\n",
      "Although much progress has been made in the identification and characterization of adhesins borne by\n",
      "pathogenic bacteria, the molecular details underlying their interaction with host receptors remain\n",
      "largely unknown owing to the lack of appropriate probing techniques. Here we report a method, based\n",
      "on atomic force microscopy (AFM) with tips bearing biologically active molecules, for measuring the\n",
      "specific binding forces of individual adhesins and for mapping their distribution on the surface of\n",
      "living bacteria. First, we determined the adhesion forces between the heparin-binding haemagglutinin\n",
      "adhesin (HBHA) produced by Mycobacterium tuberculosis and heparin, used as a model sulphated\n",
      "glycoconjugate receptor. Both the adhesion frequency and adhesion force increased with contact time,\n",
      "indicating that the HBHA-heparin complex is formed via multiple intermolecular bridges. We then\n",
      "mapped the distribution of single HBHA molecules on the surface of living mycobacteria and found\n",
      "that the adhesin is not randomly distributed over the mycobacterial surface, but concentrated into\n",
      "nanodomains.\n",
      "[{'name': 'atomic force microscopy (AFM)'},\n",
      " {'name': 'biologically active molecules'}]\n",
      "Elapsed time: 11.41 seconds\n",
      "-------------------\n",
      "16094385\n",
      "A microfluidic culture platform for CNS axonal injury, regeneration and transport.\n",
      "Investigation of axonal biology in the central nervous system (CNS) is hindered by a lack of an\n",
      "appropriate in vitro method to probe axons independently from cell bodies. Here we describe a\n",
      "microfluidic culture platform that polarizes the growth of CNS axons into a fluidically isolated\n",
      "environment without the use of targeting neurotrophins. In addition to its compatibility with live\n",
      "cell imaging, the platform can be used to (i) isolate CNS axons without somata or dendrites,\n",
      "facilitating biochemical analyses of pure axonal fractions and (ii) localize physical and chemical\n",
      "treatments to axons or somata. We report the first evidence that presynaptic (Syp) but not\n",
      "postsynaptic (Camk2a) mRNA is localized to developing rat cortical and hippocampal axons. The\n",
      "platform also serves as a straightforward, reproducible method to model CNS axonal injury and\n",
      "regeneration. The results presented here demonstrate several experimental paradigms using the\n",
      "microfluidic platform, which can greatly facilitate future studies in axonal biology.\n",
      "['live cell imaging',\n",
      " 'fluorescence microscopy',\n",
      " 'microfluidic culture platform']\n",
      "Elapsed time: 10.70 seconds\n",
      "-------------------\n",
      "16094386\n",
      "Real-time imaging of ligand-induced IKK activation in intact cells and in living mice.\n",
      "The transcription factor NF-kappaB is a key regulator of cellular activation, proliferation and\n",
      "apoptosis. Defects in the NF-kappaB pathway contribute to a broad array of malignant,\n",
      "neurodegenerative and chronic inflammatory diseases. IKK-dependent IkappaB alpha degradation by the\n",
      "26S proteasome is a critical NF-kappaB regulatory control point, which is emerging as an important\n",
      "target for drug development. To directly monitor regulation of IKK activation in intact organisms,\n",
      "we engineered an IkappaB alpha-firefly luciferase (IkappaB alpha-FLuc) fusion reporter. In cultured\n",
      "cells and living animals, the reporter provided a continuous, noninvasive readout of the kinetics of\n",
      "ligand-induced IKK activation and the pharmacodynamics of selective inhibitors of both IKK and the\n",
      "26S proteasome. This IkappaB alpha-FLuc reporter now permits continuous readout of IKK activation in\n",
      "vivo, facilitates development and validation of target-specific therapeutics, and complements\n",
      "conventional NF-kappaB transcriptional reporters for more complete temporal and regional\n",
      "investigations of the NF-kappaB signaling pathway in health and disease.\n",
      "['real-time imaging', 'living mice', 'intact cells']\n",
      "Elapsed time: 13.03 seconds\n",
      "-------------------\n",
      "16179919\n",
      "Molecular dynamics imaging in micropatterned living cells.\n",
      "Micropatterning approaches using self-assembled monolayers of alkyl thiols on gold are not optimal\n",
      "for important imaging modalities in cell biology because of absorption of light and scattering of\n",
      "electrons by the gold layer. We report here an anisotropic solid microetching (ASOMIC) procedure\n",
      "that overcomes these limitations. The method allows molecular dynamics imaging by wide-field and\n",
      "total internal reflection fluorescence (TIRF) microscopy of living mammalian cells and correlative\n",
      "platinum replica electron microscopy.\n",
      "['wide-field microscopy',\n",
      " 'total internal reflection fluorescence (TIRF) microscopy',\n",
      " 'platinum replica electron microscopy']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "16179920\n",
      "Correlated light and electron microscopic imaging of multiple endogenous proteins using Quantum\n",
      "dots.\n",
      "The importance of locating proteins in their context within cells has been heightened recently by\n",
      "the accomplishments in molecular structure and systems biology. Although light microscopy (LM) has\n",
      "been extensively used for mapping protein localization, many studies require the additional\n",
      "resolution of the electron microscope. Here we report the application of small nanocrystals (Quantum\n",
      "dots; QDs) to specifically and efficiently label multiple distinct endogenous proteins. QDs are both\n",
      "fluorescent and electron dense, facilitating their use for correlated microscopic analysis.\n",
      "Furthermore, QDs can be discriminated optically by their emission wavelength and physically by size,\n",
      "making them invaluable for multilabeling analysis. We developed pre-embedding labeling criteria\n",
      "using QDs that allows optimization at the light level, before continuing with electron microscopy\n",
      "(EM). We provide examples of double and triple immunolabeling using light, electron and correlated\n",
      "microscopy in rat cells and mouse tissue. We conclude that QDs aid precise high-throughput\n",
      "determination of protein distribution.\n",
      "['light microscopy', 'electron microscopy', 'correlated microscopy']\n",
      "Elapsed time: 10.05 seconds\n",
      "-------------------\n",
      "16278653\n",
      "One-step analysis of protein complexes in microliters of cell lysate.\n",
      "We present 'mix and measure' procedures for the analysis of protein complexes in microliters of\n",
      "crude human and mouse cell lysates using fluorescence correlation and crosscorrelation spectroscopy.\n",
      "We labeled interacting endogenous proteins by indirect immunofluorescence with all primary and\n",
      "secondary reagents added in one step. Especially for the screening of compounds interfering with\n",
      "interactions that depend on signaling-induced posttranslational modifications, the approach\n",
      "represents a major advance over existing protocols.\n",
      "['fluorescence correlation spectroscopy',\n",
      " 'crosscorrelation spectroscopy',\n",
      " 'indirect immunofluorescence']\n",
      "Elapsed time: 8.07 seconds\n",
      "-------------------\n",
      "16278654\n",
      "Rapid neurotransmitter uncaging in spatially defined patterns.\n",
      "Light-sensitive 'caged' molecules provide a means of rapidly and noninvasively manipulating\n",
      "biochemical signals with submicron spatial resolution. Here we describe a new optical system for\n",
      "rapid uncaging in arbitrary patterns to emulate complex neural activity. This system uses TeO(2)\n",
      "acousto-optical deflectors to steer an ultraviolet beam rapidly and can uncage at over 20,000\n",
      "locations per second. The uncaging beam is projected into the focal plane of a two-photon\n",
      "microscope, allowing us to combine patterned uncaging with imaging and electrophysiology. By\n",
      "photolyzing caged neurotransmitter in brain slices we can generate precise, complex activity\n",
      "patterns for dendritic integration. The method can also be used to activate many presynaptic neurons\n",
      "at once. Patterned uncaging opens new vistas in the study of signal integration and plasticity in\n",
      "neuronal circuits and other biological systems.\n",
      "['two-photon microscopy', 'electrophysiology', 'imaging']\n",
      "Elapsed time: 9.97 seconds\n",
      "-------------------\n",
      "16278657\n",
      "Correlative microscopy and electron tomography of GFP through photooxidation.\n",
      "We have developed a simple correlative photooxidation method that allows for the direct\n",
      "ultrastructural visualization of the green fluorescent protein (GFP) upon illumination. The method,\n",
      "termed GRAB for GFP recognition after bleaching, uses oxygen radicals generated during the GFP\n",
      "bleaching process to photooxidize 3,3'-diaminobenzidine (DAB) into an electron-dense precipitate\n",
      "that can be visualized by routine electron microscopy and electron tomography. The amount of DAB\n",
      "product produced by the GRAB method appears to be linear with the initial fluorescence, and the\n",
      "resulting images are of sufficient quality to reveal detailed spatial information. This is\n",
      "exemplified by the observed intra-Golgi stack and intracisternal distribution of a human Golgi\n",
      "resident glycosylation enzyme, N-acetylgalactosaminyltransferase-2 fused either to enhanced GFP or\n",
      "CFP.\n",
      "['electron microscopy',\n",
      " 'electron tomography',\n",
      " 'routine electron microscopy',\n",
      " 'fluorescence microscopy',\n",
      " 'photooxidation method']\n",
      "Elapsed time: 10.21 seconds\n",
      "-------------------\n",
      "16281357\n",
      "Labeling on the surface.\n",
      "A strategy for labeling cell-surface proteins in living cells with small-molecule fluorophores\n",
      "allowed scientists to study receptor trafficking by single-cell FRET imaging in real time.\n",
      "['FRET imaging',\n",
      " 'single-cell imaging',\n",
      " 'live cell imaging',\n",
      " 'small-molecule fluorophores']\n",
      "Elapsed time: 6.00 seconds\n",
      "-------------------\n",
      "16299474\n",
      "Fluorescence microscopy today.\n",
      "Fluorescence microscopy has undergone a renaissance in the last decade. The introduction of green\n",
      "fluorescent protein (GFP) and two-photon microscopy has allowed systematic imaging studies of\n",
      "protein localization in living cells and of the structure and function of living tissues. The impact\n",
      "of these and other new imaging methods in biophysics, neuroscience, and developmental and cell\n",
      "biology has been remarkable. Further advances in fluorophore design, molecular biological tools and\n",
      "nonlinear and hyper-resolution microscopies are poised to profoundly transform many fields of\n",
      "biological research.\n",
      "['fluorescence microscopy',\n",
      " 'two-photon microscopy',\n",
      " 'green fluorescent protein (GFP)',\n",
      " 'nonlinear microscopies',\n",
      " 'hyper-resolution microscopies']\n",
      "Elapsed time: 8.24 seconds\n",
      "-------------------\n",
      "16299475\n",
      "A guide to choosing fluorescent proteins.\n",
      "The recent explosion in the diversity of available fluorescent proteins (FPs) promises a wide\n",
      "variety of new tools for biological imaging. With no unified standard for assessing these tools,\n",
      "however, a researcher is faced with difficult questions. Which FPs are best for general use? Which\n",
      "are the brightest? What additional factors determine which are best for a given experiment? Although\n",
      "in many cases, a trial-and-error approach may still be necessary in determining the answers to these\n",
      "questions, a unified characterization of the best available FPs provides a useful guide in narrowing\n",
      "down the options.\n",
      "['brightest', 'best for general use', 'trial-and-error approach']\n",
      "Elapsed time: 7.19 seconds\n",
      "-------------------\n",
      "16299476\n",
      "Fluorescence microscopy.\n",
      "Although fluorescence microscopy permeates all of cell and molecular biology, most biologists have\n",
      "little experience with the underlying photophysical phenomena. Understanding the principles\n",
      "underlying fluorescence microscopy is useful when attempting to solve imaging problems.\n",
      "Additionally, fluorescence microscopy is in a state of rapid evolution, with new techniques, probes\n",
      "and equipment appearing almost daily. Familiarity with fluorescence is a prerequisite for taking\n",
      "advantage of many of these developments. This review attempts to provide a framework for\n",
      "understanding excitation of and emission by fluorophores, the way fluorescence microscopes work, and\n",
      "some of the ways fluorescence can be optimized.\n",
      "['fluorescence microscopy', 'optimization']\n",
      "Elapsed time: 7.36 seconds\n",
      "-------------------\n",
      "16299477\n",
      "Optical sectioning microscopy.\n",
      "Confocal scanning microscopy, a form of optical sectioning microscopy, has radically transformed\n",
      "optical imaging in biology. These devices provide a powerful means to eliminate from images the\n",
      "background caused by out-of-focus light and scatter. Confocal techniques can also improve the\n",
      "resolution of a light microscope image beyond what is achievable with widefield fluorescence\n",
      "microscopy. The quality of the images obtained, however, depends on the user's familiarity with the\n",
      "optical and fluorescence concepts that underlie this approach. We describe the core concepts of\n",
      "confocal microscopes and important variables that adversely affect confocal images. We also discuss\n",
      "data-processing methods for confocal microscopy and computational optical sectioning techniques that\n",
      "can perform optical sectioning without a confocal microscope.\n",
      "['optical sectioning microscopy',\n",
      " 'confocal scanning microscopy',\n",
      " 'widefield fluorescence microscopy']\n",
      "Elapsed time: 8.61 seconds\n",
      "-------------------\n",
      "16299478\n",
      "Deep tissue two-photon microscopy.\n",
      "With few exceptions biological tissues strongly scatter light, making high-resolution deep imaging\n",
      "impossible for traditional-including confocal-fluorescence microscopy. Nonlinear optical microscopy,\n",
      "in particular two photon-excited fluorescence microscopy, has overcome this limitation, providing\n",
      "large depth penetration mainly because even multiply scattered signal photons can be assigned to\n",
      "their origin as the result of localized nonlinear signal generation. Two-photon microscopy thus\n",
      "allows cellular imaging several hundred microns deep in various organs of living animals. Here we\n",
      "review fundamental concepts of nonlinear microscopy and discuss conditions relevant for achieving\n",
      "large imaging depths in intact tissue.\n",
      "['two-photon microscopy',\n",
      " 'confocal fluorescence microscopy',\n",
      " 'nonlinear optical microscopy']\n",
      "Elapsed time: 8.32 seconds\n",
      "-------------------\n",
      "16299479\n",
      "Fiber-optic fluorescence imaging.\n",
      "Optical fibers guide light between separate locations and enable new types of fluorescence imaging.\n",
      "Fiber-optic fluorescence imaging systems include portable handheld microscopes, flexible endoscopes\n",
      "well suited for imaging within hollow tissue cavities and microendoscopes that allow minimally\n",
      "invasive high-resolution imaging deep within tissue. A challenge in the creation of such devices is\n",
      "the design and integration of miniaturized optical and mechanical components. Until recently, fiber-\n",
      "based fluorescence imaging was mainly limited to epifluorescence and scanning confocal modalities.\n",
      "Two new classes of photonic crystal fiber facilitate ultrashort pulse delivery for fiber-optic two-\n",
      "photon fluorescence imaging. An upcoming generation of fluorescence imaging devices will be based on\n",
      "microfabricated device components.\n",
      "['fiber-optic fluorescence imaging',\n",
      " 'epifluorescence',\n",
      " 'scanning confocal modalities',\n",
      " 'two-photon fluorescence imaging',\n",
      " 'microfabricated device components']\n",
      "Elapsed time: 10.11 seconds\n",
      "-------------------\n",
      "16299482\n",
      "Secretion of type III effectors into host cells in real time.\n",
      "Type III secretion (T3S) systems are key features of many gram-negative bacteria that translocate\n",
      "T3S effector proteins directly into eukaryotic cells. There, T3S effectors exert many effects, such\n",
      "as cellular invasion or modulation of host immune responses. Studying spatiotemporal orchestrated\n",
      "secretion of various effectors has been difficult without disrupting their functions. Here we\n",
      "developed a new approach using Shigella flexneri T3S as a model to investigate bacterial\n",
      "translocation of individual effectors via multidimensional time-lapse microscopy. We demonstrate\n",
      "that direct fluorescent labeling of tetracysteine motif-tagged effectors IpaB and IpaC is possible\n",
      "in situ without loss of function. Studying the T3S kinetics of IpaB and IpaC ejection from\n",
      "individual bacteria, we found that the entire pools of IpaB and IpaC were released concurrently upon\n",
      "host cell contact, and that 50% of each effector was secreted in 240 s. This method allows an\n",
      "unprecedented analysis of the spatiotemporal events during T3S.\n",
      "['multidimensional time-lapse microscopy',\n",
      " 'direct fluorescent labeling',\n",
      " 'tetracysteine motif-tagging']\n",
      "Elapsed time: 11.60 seconds\n",
      "-------------------\n",
      "16369548\n",
      "Fluorescent sensors for rapid monitoring of intracellular cGMP.\n",
      "Sensors based on fluorescence resonance energy transfer (FRET) are powerful tools to monitor\n",
      "signaling events in living mammalian cells. Here we describe development and use of new sensors for\n",
      "cyclic GMP (cGMP) based on cGMP binding domains from cGMP-dependent protein kinase I (GKI) and from\n",
      "phosphodiesterases (PDEs). The temporal and spatial resolution attained with the new sensors is\n",
      "superior to that of existing techniques, and permits direct recording and imaging of rapid cGMP-\n",
      "signaling events.\n",
      "['Fluorescence resonance energy transfer (FRET)', 'Direct recording', 'Imaging']\n",
      "Elapsed time: 7.58 seconds\n",
      "-------------------\n",
      "16369553\n",
      "Imaging lipid bodies in cells and tissues using third-harmonic generation microscopy.\n",
      "Lipid bodies have an important role in energy storage and lipid regulation. Here we show that lipid\n",
      "bodies are a major source of contrast in third-harmonic generation (THG) microscopy of cells and\n",
      "tissues. In hepatocytes, micrometer-sized lipid bodies produce a THG signal 1-2 orders of magnitude\n",
      "larger than other structures, which allows one to image them with high specificity. THG microscopy\n",
      "with approximately 1,200 nm excitation can be used to follow the distribution of lipid bodies in a\n",
      "variety of unstained samples including insect embryos, plant seeds and intact mammalian tissue\n",
      "(liver, lung). We found that epi-THG imaging is possible in weakly absorbing tissues because bulk\n",
      "scattering redirects a substantial fraction of the forward-generated harmonic light toward the\n",
      "objective. Finally, we show that the combination of THG microscopy with two-photon and second-\n",
      "harmonic imaging provides a new tool for exploring the interactions between lipid bodies,\n",
      "extracellular matrix and fluorescent compounds (vitamin A, NADH and others) in tissues.\n",
      "['third-harmonic generation microscopy',\n",
      " 'two-photon imaging',\n",
      " 'second-harmonic imaging']\n",
      "Elapsed time: 10.99 seconds\n",
      "-------------------\n",
      "16432516\n",
      "Fluorescence cross-correlation spectroscopy in living cells.\n",
      "Cell biologists strive to characterize molecular interactions directly in the intracellular\n",
      "environment. The intrinsic resolution of optical microscopy, however, allows visualization of only\n",
      "coarse subcellular localization. By extracting information from molecular dynamics, fluorescence\n",
      "cross-correlation spectroscopy (FCCS) grants access to processes on a molecular scale, such as\n",
      "diffusion, binding, enzymatic reactions and codiffusion, and has become a valuable tool for studies\n",
      "in living cells. Here we review basic principles of FCCS and focus on seminal applications,\n",
      "including examples of intracellular signaling and trafficking. We consider FCCS in the context of\n",
      "fluorescence resonance energy transfer and multicolor imaging techniques and discuss application\n",
      "strategies and recent technical advances.\n",
      "{'imaging techniques': ['fluorescence cross-correlation spectroscopy',\n",
      "                        'fluorescence resonance energy transfer',\n",
      "                        'multicolor imaging techniques']}\n",
      "Elapsed time: 9.51 seconds\n",
      "-------------------\n",
      "16432517\n",
      "Mapping structural interactions using in-cell NMR spectroscopy (STINT-NMR).\n",
      "We describe a high-throughput in-cell nuclear magnetic resonance (NMR)-based method for mapping the\n",
      "structural changes that accompany protein-protein interactions (STINT-NMR). The method entails\n",
      "sequentially expressing two (or more) proteins within a single bacterial cell in a time-controlled\n",
      "manner and monitoring the protein interactions using in-cell NMR spectroscopy. The resulting spectra\n",
      "provide a complete titration of the interaction and define structural details of the interacting\n",
      "surfaces at atomic resolution.\n",
      "['in-cell NMR spectroscopy', 'STINT-NMR']\n",
      "Elapsed time: 6.94 seconds\n",
      "-------------------\n",
      "16432519\n",
      "Targeted insult to subsurface cortical blood vessels using ultrashort laser pulses: three models of\n",
      "stroke.\n",
      "We present a method to produce vascular disruptions within rat brain parenchyma that targets single\n",
      "microvessels. We used two-photon microscopy to image vascular architecture, to select a vessel for\n",
      "injury and to measure blood-flow dynamics. We irradiated the vessel with high-fluence, ultrashort\n",
      "laser pulses and achieved three forms of vascular insult. (i) Vessel rupture was induced at the\n",
      "highest optical energies; this provides a model for hemorrhage. (ii) Extravasation of blood\n",
      "components was induced near the lowest energies and was accompanied by maintained flow in the target\n",
      "vessel. (iii) An intravascular clot evolved when an extravasated vessel was further irradiated. Such\n",
      "clots dramatically impaired blood flow in downstream vessels, in which speeds dropped to as low as\n",
      "approximately 10% of baseline values. This demonstrates that a single blockage to a microvessel can\n",
      "lead to local cortical ischemia. Lastly, we show that hemodilution leads to a restoration of flow in\n",
      "secondary downstream vessels.\n",
      "['two-photon microscopy', 'high-fluence ultrashort laser pulses']\n",
      "Elapsed time: 10.92 seconds\n",
      "-------------------\n",
      "16432523\n",
      "A visual thalamocortical slice.\n",
      "We describe a thalamocortical slice preparation in which connectivity between the mouse lateral\n",
      "geniculate nucleus (LGN) and primary visual cortex (V1) is preserved. Through DiI injections in\n",
      "fixed brains we traced and created a three-dimensional model of the mouse visual pathways. From this\n",
      "computer model we designed a slice preparation that contains a projection from LGN to V1. We\n",
      "prepared brain slices with these predicted coordinates and demonstrated anatomical LGN-V1\n",
      "connectivity in these slices after LGN tracer injections. We also revealed functional LGN-V1\n",
      "connectivity by stimulating LGN electrically and detecting responses in layer 4 of V1 using calcium\n",
      "imaging, field potential recordings and whole-cell recordings. We also identified layer-4 neurons\n",
      "that receive direct thalamocortical input. Finally, we compared cortical activity after LGN\n",
      "stimulation with spontaneous cortical activity and found significant overlap of the spatiotemporal\n",
      "dynamics generated by both types of events.\n",
      "['calcium imaging', 'field potential recordings', 'whole-cell recordings']\n",
      "Elapsed time: 9.69 seconds\n",
      "-------------------\n",
      "16489335\n",
      "A pooling-deconvolution strategy for biological network elucidation.\n",
      "The generation of large-scale data sets is a fundamental requirement of systems biology. But despite\n",
      "recent advances, generation of such high-coverage data remains a major challenge. We developed a\n",
      "pooling-deconvolution strategy that can dramatically decrease the effort required. This strategy,\n",
      "pooling with imaginary tags followed by deconvolution (PI-deconvolution), allows the screening of\n",
      "2(n) probe proteins (baits) in 2 x n pools, with n replicates for each bait. Deconvolution of baits\n",
      "with their binding partners (preys) can be achieved by reading the prey's profile from the 2 x n\n",
      "experiments. We validated this strategy for protein-protein interaction mapping using both proteome\n",
      "microarrays and a yeast two-hybrid array, demonstrating that PI-deconvolution can be used to\n",
      "identify interactions accurately with fewer experiments and better coverage. We also show that PI-\n",
      "deconvolution can be used to identify protein-small molecule interactions inferred from profiling\n",
      "the yeast deletion collection. PI-deconvolution should be applicable to a wide range of library-\n",
      "against-library approaches and can also be used to optimize array designs.\n",
      "['proteome microarrays', 'yeast two-hybrid array']\n",
      "Elapsed time: 10.83 seconds\n",
      "-------------------\n",
      "16554829\n",
      "Imaging specific cell-surface proteolytic activity in single living cells.\n",
      "We describe a simple, sensitive and noninvasive assay that uses nontoxic, reengineered anthrax\n",
      "toxin-beta-lactamase fusion proteins with altered protease cleavage specificity to visualize\n",
      "specific cell-surface proteolytic activity in single living cells. The assay could be used to\n",
      "specifically image endogenous cell-surface furin, urokinase plasminogen activator and\n",
      "metalloprotease activity. We have adapted the assay for fluorescence microscopy, flow cytometry and\n",
      "fluorescent plate reader formats, and it is amenable for automation and high-throughput analysis.\n",
      "['fluorescence microscopy', 'flow cytometry', 'fluorescent plate reader']\n",
      "Elapsed time: 8.11 seconds\n",
      "-------------------\n",
      "16554831\n",
      "A monovalent streptavidin with a single femtomolar biotin binding site.\n",
      "Streptavidin and avidin are used ubiquitously because of the remarkable affinity of their biotin\n",
      "binding, but they are tetramers, which disrupts many of their applications. Making either protein\n",
      "monomeric reduces affinity by at least 10(4)-fold because part of the binding site comes from a\n",
      "neighboring subunit. Here we engineered a streptavidin tetramer with only one functional biotin\n",
      "binding subunit that retained the affinity, off rate and thermostability of wild-type streptavidin.\n",
      "In denaturant, we mixed a streptavidin variant containing three mutations that block biotin binding\n",
      "with wild-type streptavidin in a 3:1 ratio. Then we generated monovalent streptavidin by refolding\n",
      "and nickel-affinity purification. Similarly, we purified defined tetramers with two or three biotin\n",
      "binding subunits. Labeling of site-specifically biotinylated neuroligin-1 with monovalent\n",
      "streptavidin allowed stable neuroligin-1 tracking without cross-linking, whereas wild-type\n",
      "streptavidin aggregated neuroligin-1 and disrupted presynaptic contacts. Monovalent streptavidin\n",
      "should find general application in biomolecule labeling, single-particle tracking and\n",
      "nanotechnology.\n",
      "['refolding',\n",
      " 'nickel-affinity purification',\n",
      " 'site-specifically biotinylated',\n",
      " 'stable neuroligin-1 tracking']\n",
      "Elapsed time: 13.07 seconds\n",
      "-------------------\n",
      "16554835\n",
      "Luminescent imaging of beta-galactosidase activity in living subjects using sequential reporter-\n",
      "enzyme luminescence.\n",
      "We generated a sequential reporter-enzyme luminescence (SRL) technology for in vivo detection of\n",
      "beta-galactosidase (beta-gal) activity. The substrate, a caged D-luciferin-galactoside conjugate,\n",
      "must first be cleaved by beta-gal before it can be catalyzed by firefly luciferase (FLuc) to\n",
      "generate light. As a result, luminescence is dependent on beta-gal activity. Using this technology,\n",
      "constitutive beta-gal activity in engineered cells and inducible tissue-specific beta-gal expression\n",
      "in transgenic mice can now be visualized noninvasively over time. A substantial advantage of beta-\n",
      "gal as a bioluminescent probe is that the enzyme retains full activity outside of cells, unlike\n",
      "FLuc, which requires intracellular cofactors. As a result, antibodies conjugated to the recombinant\n",
      "beta-gal enzyme can be used to detect endogenous cells and extracellular antigens in vivo. Thus,\n",
      "coupling the properties of FLuc to the advantages of beta-gal permits bioluminescent imaging\n",
      "applications that previously were not possible.\n",
      "['sequential reporter-enzyme luminescence (SRL)',\n",
      " 'bioluminescent imaging',\n",
      " 'in vivo detection']\n",
      "Elapsed time: 11.86 seconds\n",
      "-------------------\n",
      "16578927\n",
      "Bioluminescent quantum dots.\n",
      "Bioluminescence is widely used for in vivo imaging of nude mice. By conjugating luciferase protein\n",
      "to quantum dots, bioluminescence resonance energy transfer (BRET) turns these useful fluorophores\n",
      "into a new class of bioluminescent probe.\n",
      "['bioluminescence', 'resonance energy transfer', 'BRET']\n",
      "Elapsed time: 6.06 seconds\n",
      "-------------------\n",
      "16628204\n",
      "Detection and localization of single molecular recognition events using atomic force microscopy.\n",
      "Because of its piconewton force sensitivity and nanometer positional accuracy, the atomic force\n",
      "microscope (AFM) has emerged as a powerful tool for exploring the forces and the dynamics of the\n",
      "interaction between individual ligands and receptors, either on isolated molecules or on cellular\n",
      "surfaces. These studies require attaching specific biomolecules or cells on AFM tips and on solid\n",
      "supports and measuring the unbinding forces between the modified surfaces using AFM force\n",
      "spectroscopy. In this review, we describe the current methodology for molecular recognition studies\n",
      "using the AFM, with an emphasis on strategies available for preparing AFM tips and samples, and on\n",
      "procedures for detecting and localizing single molecular recognition events.\n",
      "['atomic force microscopy', 'AFM force spectroscopy']\n",
      "Elapsed time: 8.19 seconds\n",
      "-------------------\n",
      "16628208\n",
      "Reconstruction of firing rate changes across neuronal populations by temporally deconvolved Ca2+\n",
      "imaging.\n",
      "Methods to record action potential (AP) firing in many individual neurons are essential to unravel\n",
      "the function of complex neuronal circuits in the brain. A promising approach is bolus loading of\n",
      "Ca(2+) indicators combined with multiphoton microscopy. Currently, however, this technique lacks\n",
      "cell-type specificity, has low temporal resolution and cannot resolve complex temporal firing\n",
      "patterns. Here we present simple solutions to these problems. We identified neuron types by\n",
      "colocalizing Ca(2+) signals of a red-fluorescing indicator with genetically encoded markers. We\n",
      "reconstructed firing rate changes from Ca(2+) signals by temporal deconvolution. This technique is\n",
      "efficient, dramatically enhances temporal resolution, facilitates data interpretation and permits\n",
      "analysis of odor-response patterns across thousands of neurons in the zebrafish olfactory bulb.\n",
      "Hence, temporally deconvolved Ca(2+) imaging (TDCa imaging) resolves limitations of current optical\n",
      "recording techniques and is likely to be widely applicable because of its simplicity, robustness and\n",
      "generic principle.\n",
      "[{'label': 'Ca2+ imaging', 'value': 'none'},\n",
      " {'label': 'multiphoton microscopy', 'value': 'none'},\n",
      " {'label': 'temporal deconvolution', 'value': 'TDCa imaging'}]\n",
      "Elapsed time: 11.91 seconds\n",
      "-------------------\n",
      "16628209\n",
      "High-throughput RNAi screening by time-lapse imaging of live human cells.\n",
      "RNA interference (RNAi) is a powerful tool to study gene function in cultured cells. Transfected\n",
      "cell microarrays in principle allow high-throughput phenotypic analysis after gene knockdown by\n",
      "microscopy. But bottlenecks in imaging and data analysis have limited such high-content screens to\n",
      "endpoint assays in fixed cells and determination of global parameters such as viability. Here we\n",
      "have overcome these limitations and developed an automated platform for high-content RNAi screening\n",
      "by time-lapse fluorescence microscopy of live HeLa cells expressing histone-GFP to report on\n",
      "chromosome segregation and structure. We automated all steps, including printing transfection-ready\n",
      "small interfering RNA (siRNA) microarrays, fluorescence imaging and computational phenotyping of\n",
      "digital images, in a high-throughput workflow. We validated this method in a pilot screen assaying\n",
      "cell division and delivered a sensitive, time-resolved phenoprint for each of the 49 endogenous\n",
      "genes we suppressed. This modular platform is scalable and makes the power of time-lapse microscopy\n",
      "available for genome-wide RNAi screens.\n",
      "['fluorescence microscopy',\n",
      " 'time-lapse microscopy',\n",
      " 'digital imaging',\n",
      " 'computational phenotyping']\n",
      "Elapsed time: 11.42 seconds\n",
      "-------------------\n",
      "16628210\n",
      "Metabolic biotinylation of cell surface receptors for in vivo imaging.\n",
      "We have developed a versatile, potent technique for imaging cells in culture and in vivo by\n",
      "expressing a metabolically biotinylated cell-surface receptor and visualizing it with labeled\n",
      "streptavidin moieties. The recombinant reporter protein, which incorporates a biotin acceptor\n",
      "peptide (BAP) between an N-terminal signal sequence and a transmembrane domain, (BAP-TM) was\n",
      "efficiently biotinylated by endogenous biotin ligase in mammalian cells with the biotin displayed on\n",
      "the cell surface. Tumors expressing the BAP-TM have high sensitivity for magnetic resonance and\n",
      "fluorescence tomographic imaging in vivo after intravascular injection of streptavidin conjugated to\n",
      "magnetic nanoparticles or fluorochromes, respectively. Moreover, streptavidin-horseradish peroxidase\n",
      "conjugates in conjunction with a peroxidase-sensitive gadolinium agent further increased and\n",
      "prolonged the magnetic resonance signal. This BAP-TM allows noninvasive real-time imaging of any\n",
      "cell type transduced to express this reporter protein in culture or in vivo.\n",
      "['magnetic resonance imaging',\n",
      " 'fluorescence tomographic imaging',\n",
      " 'peroxidase-sensitive gadolinium agent']\n",
      "Elapsed time: 11.79 seconds\n",
      "-------------------\n",
      "16685789\n",
      "SAILing toward larger protein structures.\n",
      "With a new method for optimizing the amino acid isotopic labeling pattern, stereo-array isotope\n",
      "labeling (SAIL), structures of large proteins (up to 50 kDa) can now be solved with ease with\n",
      "nuclear magnetic resonance (NMR) spectroscopy.\n",
      "{'methods': ['nuclear magnetic resonance (NMR)',\n",
      "             'stereo-array isotope labeling (SAIL)']}\n",
      "Elapsed time: 6.20 seconds\n",
      "-------------------\n",
      "16721376\n",
      "Protein ligation: an enabling technology for the biophysical analysis of proteins.\n",
      "Biophysical techniques such as fluorescence spectroscopy and nuclear magnetic resonance (NMR)\n",
      "spectroscopy provide a window into the inner workings of proteins. These approaches make use of\n",
      "probes that can either be naturally present within the protein or introduced through a labeling\n",
      "procedure. In general, the more control one has over the type, location and number of probes in a\n",
      "protein, then the more information one can extract from a given biophysical analysis. Recently, two\n",
      "related approaches have emerged that allow proteins to be labeled with a broad range of physical\n",
      "probes. Expressed protein ligation (EPL) and protein trans-splicing (PTS) are both intein-based\n",
      "approaches that permit the assembly of a protein from smaller synthetic and/or recombinant pieces.\n",
      "Here we provide some guidelines for the use of EPL and PTS, and highlight how the dovetailing of\n",
      "these new protein chemistry methods with standard biophysical techniques has improved our ability to\n",
      "interrogate protein function, structure and folding.\n",
      "['fluorescence spectroscopy', 'nuclear magnetic resonance (NMR) spectroscopy']\n",
      "Elapsed time: 10.29 seconds\n",
      "-------------------\n",
      "16721377\n",
      "Assaying chromosomal inversions by single-molecule haplotyping.\n",
      "Inversions are an important form of structural variation, but they are difficult to characterize, as\n",
      "their breakpoints often fall within inverted repeats. We have developed a method called 'haplotype\n",
      "fusion' in which an inversion breakpoint is genotyped by performing fusion PCR on single molecules\n",
      "of human genomic DNA. Fusing single-copy sequences bracketing an inversion breakpoint generates\n",
      "orientation-specific PCR products, exemplified by a genotyping assay for the int22 hemophilia A\n",
      "inversion on Xq28. Furthermore, we demonstrated that inversion events with breakpoints embedded\n",
      "within long (>100 kb) inverted repeats can be genotyped by haplotype-fusion PCR followed by bead-\n",
      "based single-molecule haplotyping on repeat-specific markers bracketing the inversion breakpoint. We\n",
      "illustrate this method by genotyping a Yp paracentric inversion sponsored by >300-kb-long inverted\n",
      "repeats. The generality of our methods to survey for, and genotype chromosomal inversions should\n",
      "help our understanding of the contribution of inversions to genomic variation, inherited diseases\n",
      "and cancer.\n",
      "['single-molecule haplotyping',\n",
      " 'fusion PCR',\n",
      " 'bead-based single-molecule haplotyping']\n",
      "Elapsed time: 11.92 seconds\n",
      "-------------------\n",
      "16791209\n",
      "Monitoring dynamic protein interactions with photoquenching FRET.\n",
      "The mammalian cell nucleus is a dynamic and highly organized structure. Most proteins are mobile\n",
      "within the nuclear compartment, and this mobility reflects transient interactions with chromatin, as\n",
      "well as network interactions with a variety of protein partners. To study these dynamic processes in\n",
      "living cells, we developed an imaging method that combines the photoactivated green fluorescent\n",
      "protein (PA-GFP) and fluorescence resonance energy transfer (FRET) microscopy. We used this new\n",
      "method, photoquenching FRET (PQ-FRET), to define the dynamic interactions of the heterochromatin\n",
      "protein-1 alpha (HP1alpha) and the transcription factor CCAAT/enhancer binding protein alpha\n",
      "(C/EBPalpha) in regions of centromeric heterochromatin in mouse pituitary cells. The advantage of\n",
      "the PQ-FRET assay is that it provides simultaneous measurement of a protein's mobility, its exchange\n",
      "within macromolecular complexes and its interactions with other proteins in the living cell without\n",
      "the need for corrections based on reference images acquired from control cells.\n",
      "['fluorescence resonance energy transfer (FRET) microscopy',\n",
      " 'photoactivated green fluorescent protein (PA-GFP)',\n",
      " \"simultaneous measurement of a protein's mobility, its exchange within \"\n",
      " 'macromolecular complexes and its interactions with other proteins in the '\n",
      " 'living cell']\n",
      "Elapsed time: 12.11 seconds\n",
      "-------------------\n",
      "16791210\n",
      "Dynamic proteomics in individual human cells uncovers widespread cell-cycle dependence of nuclear\n",
      "proteins.\n",
      "We examined cell cycle-dependent changes in the proteome of human cells by systematically measuring\n",
      "protein dynamics in individual living cells. We used time-lapse microscopy to measure the dynamics\n",
      "of a random subset of 20 nuclear proteins, each tagged with yellow fluorescent protein (YFP) at its\n",
      "endogenous chromosomal location. We synchronized the cells in silico by aligning protein dynamics in\n",
      "each cell between consecutive divisions. We observed widespread (40%) cell-cycle dependence of\n",
      "nuclear protein levels and detected previously unknown cell cycle-dependent localization changes.\n",
      "This approach to dynamic proteomics can aid in discovery and accurate quantification of the\n",
      "extensive regulation of protein concentration and localization in individual living cells.\n",
      "{'imagingTechniques': ['time-lapse microscopy',\n",
      "                       'yellow fluorescent protein (YFP)']}\n",
      "Elapsed time: 8.79 seconds\n",
      "-------------------\n",
      "16862134\n",
      "Removal of a time barrier for high-resolution multidimensional NMR spectroscopy.\n",
      "We introduce the recursive multidimensional decomposition (R-MDD) method to speed recording of high-\n",
      "resolution NMR spectra. The measurement time is logarithmically dependent on the sizes of indirect\n",
      "spectral dimensions. R-MDD has the sensitivity and resolution advantages of optimized nonuniform\n",
      "acquisition schemes and is applicable to all types of biomolecular spectra. We demonstrated it for\n",
      "triple resonance experiments on three globular proteins (ubiquitin, azurin and the barstar-barnase\n",
      "complex) of 8-22 kDa.\n",
      "['R-MDD', 'nonuniform acquisition schemes', 'triple resonance experiments']\n",
      "Elapsed time: 7.70 seconds\n",
      "-------------------\n",
      "16862136\n",
      "Efficient site-specific integration in Plasmodium falciparum chromosomes mediated by\n",
      "mycobacteriophage Bxb1 integrase.\n",
      "Here we report an efficient, site-specific system of genetic integration into Plasmodium falciparum\n",
      "malaria parasite chromosomes. This is mediated by mycobacteriophage Bxb1 integrase, which catalyzes\n",
      "recombination between an incoming attP and a chromosomal attB site. We developed P. falciparum lines\n",
      "with the attB site integrated into the glutaredoxin-like cg6 gene. Transfection of these attB(+)\n",
      "lines with a dual-plasmid system, expressing a transgene on an attP-containing plasmid together with\n",
      "a drug resistance gene and the integrase on a separate plasmid, produced recombinant parasites\n",
      "within 2 to 4 weeks that were genetically uniform for single-copy plasmid integration. Integrase-\n",
      "mediated recombination resulted in proper targeting of parasite proteins to intra-erythrocytic\n",
      "compartments, including the apicoplast, a plastid-like organelle. Recombinant attB x attP parasites\n",
      "were genetically stable in the absence of drug and were phenotypically homogeneous. This system can\n",
      "be exploited for rapid genetic integration and complementation analyses at any stage of the P.\n",
      "falciparum life cycle, and it illustrates the utility of Bxb1-based integrative recombination for\n",
      "genetic studies of intracellular eukaryotic organisms.\n",
      "['integrase-mediated recombination', 'apicoplast', 'plastid-like organelle']\n",
      "Elapsed time: 15.73 seconds\n",
      "-------------------\n",
      "16896339\n",
      "Sub-diffraction-limit imaging by stochastic optical reconstruction microscopy (STORM).\n",
      "We have developed a high-resolution fluorescence microscopy method based on high-accuracy\n",
      "localization of photoswitchable fluorophores. In each imaging cycle, only a fraction of the\n",
      "fluorophores were turned on, allowing their positions to be determined with nanometer accuracy. The\n",
      "fluorophore positions obtained from a series of imaging cycles were used to reconstruct the overall\n",
      "image. We demonstrated an imaging resolution of 20 nm. This technique can, in principle, reach\n",
      "molecular-scale resolution.\n",
      "['STORM',\n",
      " 'fluorescence microscopy',\n",
      " 'localization',\n",
      " 'photoswitchable fluorophores']\n",
      "Elapsed time: 7.99 seconds\n",
      "-------------------\n",
      "16896340\n",
      "Nanoscale resolution in GFP-based microscopy.\n",
      "We report attainment of subdiffraction resolution using stimulated emission depletion (STED)\n",
      "microscopy with GFP-labeled samples. The approximately 70 nm lateral resolution attained in this\n",
      "study is demonstrated by imaging GFP-labeled viruses and the endoplasmic reticulum (ER) of a\n",
      "mammalian cell. Our results mark the advent of nanoscale biological microscopy with genetically\n",
      "encoded markers.\n",
      "[{'name': 'stimulated emission depletion (STED)'},\n",
      " {'name': 'GFP-based microscopy'}]\n",
      "Elapsed time: 7.23 seconds\n",
      "-------------------\n",
      "16967551\n",
      "STEDy progress.\n",
      "Continuing progress in the development of stimulated emission depletion microscopy (STED) is slowly\n",
      "but surely fulfilling the method's promise of ultrahigh-resolution fluorescence microscopy and may\n",
      "bring unexpected benefits to other imaging modalities as well.\n",
      "['STED', 'fluorescence microscopy', 'ultrahigh-resolution imaging']\n",
      "Elapsed time: 5.83 seconds\n",
      "-------------------\n",
      "16990811\n",
      "Imaging diacylglycerol dynamics at organelle membranes.\n",
      "Fluorescence imaging is a powerful technique to visualize spatiotemporal dynamics of biomolecules in\n",
      "living cells. We describe fluorescent indicators for a lipid second messenger, diacylglycerol (DAG),\n",
      "which allow the localized analysis of DAG dynamics at subcellular membranes. We have thus pinpointed\n",
      "that DAG concentrations increase and/or decrease at not only the plasma membrane but also organelle\n",
      "membranes such as endomembranes and mitochondrial outer membranes.\n",
      "Invalid json output: [\n",
      "\"fluorescence imaging\",\n",
      "\"spatiotemporal dynamics\"\n",
      "\"subcellular membranes\"\n",
      "\"endomembranes\"\n",
      "\"mitochondrial outer membranes\"\n",
      "] derived from Sure! Here is the list of imaging techniques described in the abstract:\n",
      "\n",
      "[\n",
      "\"fluorescence imaging\",\n",
      "\"spatiotemporal dynamics\"\n",
      "\"subcellular membranes\"\n",
      "\"endomembranes\"\n",
      "\"mitochondrial outer membranes\"\n",
      "]\n",
      "16990812\n",
      "Defining the actual sensitivity and specificity of the neurosphere assay in stem cell biology.\n",
      "For more than a decade the 'neurosphere assay' has been used to define and measure neural stem cell\n",
      "(NSC) behavior, with similar assays now used in other organ systems and in cancer. We asked whether\n",
      "neurospheres are clonal structures whose diameter, number and composition accurately reflect the\n",
      "proliferation, self-renewal and multipotency of a single founding NSC. Using time-lapse video\n",
      "microscopy, coculture experiments with genetically labeled cells, and analysis of the volume of\n",
      "spheres, we observed that neurospheres are highly motile structures prone to fuse even under\n",
      "ostensibly 'clonal' culture conditions. Chimeric neurospheres were prevalent independent of ages,\n",
      "species and neural structures. Thus, the intrinsic dynamic of neurospheres, as conventionally\n",
      "assayed, introduces confounders. More accurate conditions (for example, plating a single cell per\n",
      "miniwell) will be crucial for assessing clonality, number and fate of stem cells. These cautions\n",
      "probably have implications for the use of 'cytospheres' as an assay in other organ systems and with\n",
      "other cell types, both normal and neoplastic.\n",
      "['time-lapse video microscopy',\n",
      " 'coculture experiments',\n",
      " 'analysis of the volume of spheres']\n",
      "Elapsed time: 11.85 seconds\n",
      "-------------------\n",
      "16990814\n",
      "Quantitative four-dimensional tracking of cytoplasmic and nuclear HIV-1 complexes.\n",
      "Emerging real-time techniques for imaging viral infections provide powerful tools for understanding\n",
      "the dynamics of virus-host cell interactions. Here we labeled human immunodeficiency virus-1 (HIV-1)\n",
      "integrase with a small tetracysteine tag, which preserved the virus' infectivity while allowing it\n",
      "to be labeled with the bis-arsenical fluorescein derivative FlAsH. This labeling allowed us to image\n",
      "both intracytoplasmic and intranuclear HIV-1 complexes in three dimensions over time (4D) in human\n",
      "cells and enabled us to analyze HIV-1 kinetics by automated 4D quantitative particle tracking. In\n",
      "the cytoplasm, HIV-1 complexes underwent directed movements toward the nuclear compartment,\n",
      "kinetically characteristic of both microtubule- and actin-dependent transport. The complexes then\n",
      "adopted smaller movements in a very confined volume once associated with the nuclear membrane and\n",
      "more diffuse movements once inside the nucleus. This work contributes new insight into the various\n",
      "movements of HIV-1 complexes within infected cells and provides a useful tool for the study of\n",
      "virus-host cell interactions during infection.\n",
      "['fluorescein derivative FlAsH',\n",
      " '4D quantitative particle tracking',\n",
      " 'automated 4D quantitative particle tracking']\n",
      "Elapsed time: 11.54 seconds\n",
      "-------------------\n",
      "17013382\n",
      "Nonblinking and long-lasting single-molecule fluorescence imaging.\n",
      "Photobleaching and blinking of fluorophores pose fundamental limitations on the information content\n",
      "of single-molecule fluorescence measurements. Photoinduced blinking of Cy5 has hampered many\n",
      "previous investigations using this popular fluorophore. Here we show that Trolox in combination with\n",
      "the enzymatic oxygen-scavenging system eliminates Cy5 blinking, dramatically reduces photobleaching\n",
      "and improves the signal linearity at high excitation rates, significantly extending the\n",
      "applicability of single-molecule fluorescence techniques.\n",
      "['single-molecule fluorescence imaging',\n",
      " 'photobleaching',\n",
      " 'blinking of fluorophores',\n",
      " 'Cy5 blinking',\n",
      " 'Trolox']\n",
      "Elapsed time: 8.30 seconds\n",
      "-------------------\n",
      "17060909\n",
      "Two-chamber AFM: probing membrane proteins separating two aqueous compartments.\n",
      "Biological membranes compartmentalize and define physical borders of cells. They are crowded with\n",
      "membrane proteins that fulfill diverse crucial functions. About one-third of all genes in organisms\n",
      "code for, and the majority of drugs target, membrane proteins. To combine structure and function\n",
      "analysis of membrane proteins, we designed a two-chamber atomic force microscopy (AFM) setup that\n",
      "allows investigation of membranes spanned over nanowells, therefore separating two aqueous chambers.\n",
      "We imaged nonsupported surface layers (S layers) of Corynebacterium glutamicum at sufficient\n",
      "resolution to delineate a 15 A-wide protein pore. We probed the elastic and yield moduli of\n",
      "nonsupported membranes, giving access to the lateral interaction energy between proteins. We\n",
      "combined AFM and fluorescence microscopy to demonstrate the functionality of proteins in the setup\n",
      "by documenting proton pumping by Halobacterium salinarium purple membranes.\n",
      "['atomic force microscopy (AFM)', 'fluorescence microscopy']\n",
      "Elapsed time: 10.44 seconds\n",
      "-------------------\n",
      "17060917\n",
      "A new strategy for structure determination of large proteins in solution without deuteration.\n",
      "So far high-resolution structure determination by nuclear magnetic resonance (NMR) spectroscopy has\n",
      "been limited to proteins <30 kDa, although global fold determination is possible for substantially\n",
      "larger proteins. Here we present a strategy for assigning backbone and side-chain resonances of\n",
      "large proteins without deuteration, with which one can obtain high-resolution structures from\n",
      "(1)H-(1)H distance restraints. The strategy uses information from through-bond correlation\n",
      "experiments to filter intraresidue and sequential correlations from through-space correlation\n",
      "experiments, and then matches the filtered correlations to obtain sequential assignment. We\n",
      "demonstrate this strategy on three proteins ranging from 24 to 65 kDa for resonance assignment and\n",
      "on maltose binding protein (42 kDa) and hemoglobin (65 kDa) for high-resolution structure\n",
      "determination. The strategy extends the size limit for structure determination by NMR spectroscopy\n",
      "to 42 kDa for monomeric proteins and to 65 kDa for differentially labeled multimeric proteins\n",
      "without the need for deuteration or selective labeling.\n",
      "{'methods': ['nuclear magnetic resonance (NMR) spectroscopy',\n",
      "             'through-bond correlation experiments',\n",
      "             'through-space correlation experiments']}\n",
      "Elapsed time: 11.37 seconds\n",
      "-------------------\n",
      "17143280\n",
      "Imaging cellular network dynamics in three dimensions using fast 3D laser scanning.\n",
      "Spatiotemporal activity patterns in three-dimensionally organized cellular networks are fundamental\n",
      "to the function of the nervous system. Despite advances in functional imaging of cell populations, a\n",
      "method to resolve local network activity in three dimensions has been lacking. Here we introduce a\n",
      "three-dimensional (3D) line-scan technology for two-photon microscopy that permits fast fluorescence\n",
      "measurements from several hundred cells distributed in 3D space. We combined sinusoidal vibration of\n",
      "the microscope objective at 10 Hz with 'smart' movements of galvanometric x-y scanners to repeatedly\n",
      "scan the laser focus along a closed 3D trajectory. More than 90% of cell somata were sampled by the\n",
      "scan line within volumes of 250 microm side length. Using bulk-loading of calcium indicator, we\n",
      "applied this method to reveal spatiotemporal activity patterns in neuronal and astrocytic networks\n",
      "in the rat neocortex in vivo. Two-photon population imaging using 3D scanning opens the field for\n",
      "comprehensive studies of local network dynamics in intact tissue.\n",
      "['two-photon microscopy',\n",
      " 'three-dimensional (3D) line-scan technology',\n",
      " 'sinusoidal vibration of the microscope objective',\n",
      " 'galvanometric x-y scanners',\n",
      " 'bulk-loading of calcium indicator']\n",
      "Elapsed time: 12.33 seconds\n",
      "-------------------\n",
      "17143281\n",
      "Tomographic molecular imaging and 3D quantification within adult mouse organs.\n",
      "A convenient technology to quantify three-dimensional (3D) morphological features would have\n",
      "widespread applications in biomedical research. Based on combined improvements in sample\n",
      "preparation, tomographic imaging and computational processing, we present a procedure for high-\n",
      "resolution 3D quantification of structures within intact adult mouse organs. Using the nonobese\n",
      "diabetic (NOD) mouse model, we demonstrate a correlation between total islet beta-cell volume and\n",
      "the onset of type-1 diabetes.\n",
      "['tomographic molecular imaging',\n",
      " 'computational processing',\n",
      " 'high-resolution 3D quantification']\n",
      "Elapsed time: 7.21 seconds\n",
      "-------------------\n",
      "17179934\n",
      "Disentangling conformational states of macromolecules in 3D-EM through likelihood optimization.\n",
      "Although three-dimensional electron microscopy (3D-EM) permits structural characterization of\n",
      "macromolecular assemblies in distinct functional states, the inability to classify projections from\n",
      "structurally heterogeneous samples has severely limited its application. We present a maximum\n",
      "likelihood-based classification method that does not depend on prior knowledge about the structural\n",
      "variability, and demonstrate its effectiveness for two macromolecular assemblies with different\n",
      "types of conformational variability: the Escherichia coli ribosome and Simian virus 40 (SV40) large\n",
      "T-antigen.\n",
      "['electron microscopy',\n",
      " 'three-dimensional electron microscopy',\n",
      " 'structural characterization']\n",
      "Elapsed time: 7.71 seconds\n",
      "-------------------\n",
      "17179936\n",
      "MRI-based localization of electrophysiological recording sites within the cerebral cortex at single-\n",
      "voxel accuracy.\n",
      "The localization of microelectrode recording sites in the layers of primate cerebral cortex permits\n",
      "the analysis of relationships between recorded neuronal activities and underlying anatomical\n",
      "connections. We present a magnetic resonance imaging method for precise in vivo localization of\n",
      "cortical recording sites. In this method, the susceptibility-induced effect thickens the appearance\n",
      "of the microelectrode and enhances the detectability of the microelectrode tip, which usually\n",
      "occupies less than a few percent of the volume of an image voxel. In a phantom study, the optimized\n",
      "susceptibility-induced effect allowed tip detection with single-voxel accuracy (in-plane resolution,\n",
      "50 mum). We applied this method to recording microelectrodes inserted into the brains of macaque\n",
      "monkeys, and localized the microelectrode tip at an in-plane resolution of 150 mum within the cortex\n",
      "of 2-3 mm in thickness. Subsequent histological analyses validated the single-voxel accuracy of the\n",
      "in vivo tip localization. This method opens up a way to investigate information flow during\n",
      "cognitive processes in the brain.\n",
      "['MRI', 'susceptibility-induced effect', 'microelectrode', 'tip detection']\n",
      "Elapsed time: 11.29 seconds\n",
      "-------------------\n",
      "17179937\n",
      "Major signal increase in fluorescence microscopy through dark-state relaxation.\n",
      "We report a substantial signal gain in fluorescence microscopy by ensuring that transient molecular\n",
      "dark states with lifetimes >1 micros, such as the triplet state relax between two molecular\n",
      "absorption events. For GFP and Rhodamine dye Atto532, we observed a 5-25-fold increase in total\n",
      "fluorescence yield before molecular bleaching when strong continuous-wave or high-repetition-rate\n",
      "pulsed illumination was replaced with pulses featuring temporal pulse separation >1 micros. The\n",
      "signal gain was observed both for one- and two-photon excitation. Obeying dark or triplet state\n",
      "relaxation in the illumination process signifies a major step toward imaging with low photobleaching\n",
      "and strong fluorescence fluxes.\n",
      "['fluorescence microscopy',\n",
      " 'one-photon excitation',\n",
      " 'two-photon excitation',\n",
      " 'strong continuous-wave illumination',\n",
      " 'high-repetition-rate pulsed illumination']\n",
      "Elapsed time: 9.55 seconds\n",
      "-------------------\n",
      "17195846\n",
      "Optical induction of synaptic plasticity using a light-sensitive channel.\n",
      "We have combined millisecond activation of channelrhodopsin-2 (ChR2), a light-gated ion channel,\n",
      "with two-photon calcium imaging to investigate active synaptic contacts in rat hippocampal slice\n",
      "cultures. Calcium influx was larger during light-induced action potentials than during action\n",
      "potentials induced by somatic current injection, leading to highly reproducible synaptic\n",
      "transmission. Pairing of light stimulation with postsynaptic depolarization induced long-term\n",
      "potentiation, making this technique ideal for genetic and pharmacological dissection of synaptic\n",
      "plasticity.\n",
      "['calcium imaging', 'two-photon calcium imaging', 'light-sensitive channel']\n",
      "Elapsed time: 7.75 seconds\n",
      "-------------------\n",
      "17259991\n",
      "Improved calcium imaging in transgenic mice expressing a troponin C-based biosensor.\n",
      "Fluorescent Ca(2+) indicator proteins (FCIPs) are attractive tools for studying Ca(2+) dynamics in\n",
      "live cells. Here we describe transgenic mouse lines expressing a troponin C (TnC)-based biosensor.\n",
      "The biosensor is widely expressed in neurons and has improved Ca(2+) sensitivity both in vitro and\n",
      "in vivo. This allows FCIP-based two-photon Ca(2+) imaging of distinct neurons and their dendrites in\n",
      "vivo, and opens a new avenue for structure-function analysis of intact neuronal circuits.\n",
      "['two-photon Ca(2+) imaging',\n",
      " 'FCIP-based two-photon Ca(2+) imaging',\n",
      " 'Ca(2+) sensitivity']\n",
      "Elapsed time: 8.92 seconds\n",
      "-------------------\n",
      "17277781\n",
      "Focused-ion-beam thinning of frozen-hydrated biological specimens for cryo-electron microscopy.\n",
      "Cryo-electron microscopy can provide high-resolution structural information about cells and\n",
      "organelles in the nearly native, frozen-hydrated state. Applicability, however, is limited by\n",
      "difficulties encountered in preparing suitably thin, vitreously frozen biological specimens. We\n",
      "demonstrate, by cryo-electron tomography of Escherichia coli cells, that a focused ion beam (FIB)\n",
      "can be used to thin whole frozen-hydrated cells in a convenient and essentially artifact-free way.\n",
      "['focused ion beam', 'cryo-electron microscopy', 'cryo-electron tomography']\n",
      "Elapsed time: 7.90 seconds\n",
      "-------------------\n",
      "17322889\n",
      "Imaging receptor microdomains on leukocyte subsets in live mice.\n",
      "We present a simple method to identify the recruitment of leukocyte subsets and determine concurrent\n",
      "surface-receptor clustering in live mice. We show that CD45+ F4/80- Gr-1+ neutrophils are robustly\n",
      "recruited in surgery-activated cremasteric venules, whereas adherent CD45+ B220+ B lymphocytes were\n",
      "dominant in bone marrow venules. Most adherent Gr-1+ leukocytes are not firmly stationary but\n",
      "actively migrate on TNF-alpha-activated cremasteric venular endothelium and exhibit marked\n",
      "polarization of surface PSGL-1, but not LFA-1, to the trailing edge.\n",
      "['live mice imaging',\n",
      " 'surface-receptor clustering',\n",
      " 'TNF-alpha-activated cremasteric venular endothelium',\n",
      " 'trailing edge polarization']\n",
      "Elapsed time: 9.72 seconds\n",
      "-------------------\n",
      "17326268\n",
      "Ultrafast spectroscopy: timing is everything.\n",
      "Two groups present applications of two-dimensional optical spectroscopic methods for investigating\n",
      "dynamic biological processes on pico- to femtosecond timescales.\n",
      "['two-dimensional optical spectroscopic methods', 'ultrafast spectroscopy']\n",
      "Elapsed time: 5.64 seconds\n",
      "-------------------\n",
      "17339846\n",
      "Single-molecule analysis of DNA-protein complexes using nanopores.\n",
      "We present a method for rapid measurement of DNA-protein interactions using voltage-driven threading\n",
      "of single DNA molecules through a protein nanopore. Electrical force applied to individual ssDNA-\n",
      "exonuclease I complexes pulls the two molecules apart, while ion current probes the dissociation\n",
      "rate of the complex. Nanopore force spectroscopy (NFS) reveals energy barriers affecting complex\n",
      "dissociation. This method can be applied to other nucleic acid-protein complexes, using protein or\n",
      "solid-state nanopore devices.\n",
      "['nanopore force spectroscopy',\n",
      " 'electrical force',\n",
      " 'ion current probes',\n",
      " 'voltage-driven threading']\n",
      "Elapsed time: 7.37 seconds\n",
      "-------------------\n",
      "17339847\n",
      "High-resolution three-dimensional imaging of large specimens with light sheet-based microscopy.\n",
      "We report that single (or selective) plane illumination microscopy (SPIM), combined with a new\n",
      "deconvolution algorithm, provides a three-dimensional spatial resolution exceeding that of confocal\n",
      "fluorescence microscopy in large samples. We demonstrate this by imaging large living multicellular\n",
      "specimens obtained in a three-dimensional cell culture. The ability to rapidly image large samples\n",
      "at high resolution with minimal photodamage provides new opportunities especially for the study of\n",
      "subcellular processes in large living specimens.\n",
      "['SPIM', 'deconvolution algorithm', 'confocal fluorescence microscopy']\n",
      "Elapsed time: 7.00 seconds\n",
      "-------------------\n",
      "17351620\n",
      "Unitary permeability of gap junction channels to second messengers measured by FRET microscopy.\n",
      "Gap junction channels assembled from connexin protein subunits mediate intercellular transfer of\n",
      "ions and metabolites. Impaired channel function is implicated in several hereditary human diseases.\n",
      "In particular, defective permeation of cAMP or inositol-1,4,5-trisphosphate (InsP(3)) through\n",
      "connexin channels is associated with peripheral neuropathies and deafness, respectively. Here we\n",
      "present a method to estimate the permeability of single gap junction channels to second messengers.\n",
      "Using HeLa cells that overexpressed wild-type human connexin 26 (HCx26wt) as a model system, we\n",
      "combined measurements of junctional conductance and fluorescence resonance energy transfer (FRET)\n",
      "emission ratio of biosensors selective for cAMP and InsP(3). The unitary permeabilities to cAMP (47\n",
      "x 10(-3) +/- 15 x 10(-3) microm(3)/s) and InsP(3) (60 x 10(-3) +/- 12 x 10(-3) microm(3)/s) were\n",
      "similar, but substantially larger than the unitary permeability to lucifer yellow (LY; 7 +/- 3 x\n",
      "10(-3) microm(3)/s), an exogenous tracer. This method permits quantification of defects of metabolic\n",
      "coupling and can be used to investigate interdependence of intercellular diffusion and cross-talk\n",
      "between diverse signaling pathways.\n",
      "['FRET microscopy', 'biosensors', 'junctional conductance']\n",
      "Elapsed time: 15.19 seconds\n",
      "-------------------\n",
      "17351621\n",
      "Fluorescence imaging of amyloid formation in living cells by a functional, tetracysteine-tagged\n",
      "alpha-synuclein.\n",
      "Alpha-synuclein is a major component of intraneuronal protein aggregates constituting a distinctive\n",
      "feature of Parkinson disease. To date, fluorescence imaging of dynamic processes leading to such\n",
      "amyloid deposits in living cells has not been feasible. To address this need, we generated a\n",
      "recombinant alpha-synuclein (alpha-synuclein-C4) bearing a tetracysteine target for fluorogenic\n",
      "biarsenical compounds. The biophysical, biochemical and aggregation properties of alpha-synuclein-C4\n",
      "matched those of the wild-type protein in vitro and in living cells. We observed aggregation of\n",
      "alpha-synuclein-C4 transfected or microinjected into cells, particularly under oxidative stress\n",
      "conditions. Fluorescence resonance energy transfer (FRET) between FlAsH and ReAsH confirmed the\n",
      "close association of fibrillized alpha-synuclein-C4 molecules. Alpha-synuclein-C4 offers the means\n",
      "for directly probing amyloid formation and interactions of alpha-synuclein with other proteins in\n",
      "living cells, the response to cellular stress and screening drugs for Parkinson disease.\n",
      "['fluorescence imaging',\n",
      " 'fluorescence resonance energy transfer (FRET)',\n",
      " 'biarsenical compounds']\n",
      "Elapsed time: 12.22 seconds\n",
      "-------------------\n",
      "17384643\n",
      "Ultramicroscopy: three-dimensional visualization of neuronal networks in the whole mouse brain.\n",
      "Visualizing entire neuronal networks for analysis in the intact brain has been impossible up to now.\n",
      "Techniques like computer tomography or magnetic resonance imaging (MRI) do not yield cellular\n",
      "resolution, and mechanical slicing procedures are insufficient to achieve high-resolution\n",
      "reconstructions in three dimensions. Here we present an approach that allows imaging of whole fixed\n",
      "mouse brains. We modified 'ultramicroscopy' by combining it with a special procedure to clear\n",
      "tissue. We show that this new technique allows optical sectioning of fixed mouse brains with\n",
      "cellular resolution and can be used to detect single GFP-labeled neurons in excised mouse\n",
      "hippocampi. We obtained three-dimensional (3D) images of dendritic trees and spines of populations\n",
      "of CA1 neurons in isolated hippocampi. Also in fruit flies and in mouse embryos, we were able to\n",
      "visualize details of the anatomy by imaging autofluorescence. Our method is ideally suited for high-\n",
      "throughput phenotype screening of transgenic mice and thus will benefit the investigation of disease\n",
      "models.\n",
      "['ultramicroscopy',\n",
      " 'computer tomography',\n",
      " 'magnetic resonance imaging',\n",
      " 'optical sectioning']\n",
      "Elapsed time: 10.89 seconds\n",
      "-------------------\n",
      "17401369\n",
      "Image-based multivariate profiling of drug responses from single cells.\n",
      "Quantitative analytical approaches for discovering new compound mechanisms are required for\n",
      "summarizing high-throughput, image-based drug screening data. Here we present a multivariate method\n",
      "for classifying untreated and treated human cancer cells based on approximately 300 single-cell\n",
      "phenotypic measurements. This classification provides a score, measuring the magnitude of the drug\n",
      "effect, and a vector, indicating the simultaneous phenotypic changes induced by the drug. These two\n",
      "quantities were used to characterize compound activities and identify dose-dependent multiphasic\n",
      "responses. A systematic survey of profiles extracted from a 100-compound compendium of image data\n",
      "revealed that only 10-15% of the original features were required to detect a compound effect. We\n",
      "report the most informative image features for each compound and fluorescence marker set using a\n",
      "method that will be useful for determining minimal collections of readouts for drug screens. Our\n",
      "approach provides human-interpretable profiles and automatic determination of on- and off-target\n",
      "effects.\n",
      "['image-based drug screening',\n",
      " 'single-cell phenotypic measurements',\n",
      " 'multivariate method',\n",
      " 'dose-dependent multiphasic responses',\n",
      " 'systematic survey',\n",
      " 'image features',\n",
      " 'fluorescence marker set',\n",
      " 'human-interpretable profiles']\n",
      "Elapsed time: 11.37 seconds\n",
      "-------------------\n",
      "17401370\n",
      "Imaging dynamics of endogenous mitochondrial RNA in single living cells.\n",
      "We developed genetically encoded RNA probes for characterizing localization and dynamics of\n",
      "mitochondrial RNA (mtRNA) in single living cells. The probes consist of two RNA-binding domains of\n",
      "PUMILIO1, each connected with split fragments of a fluorescent protein capable of reconstituting\n",
      "upon binding to a target RNA. We designed the probes to specifically recognize a 16-base sequence of\n",
      "mtRNA encoding NADH dehydrogenase subunit 6 (ND6) and to be targeted into the mitochondrial matrix,\n",
      "which allowed real-time imaging of ND6 mtRNA localization in living cells. We showed that ND6 mtRNA\n",
      "is localized within mitochondria and concentrated particularly on mitochondrial DNA (mtDNA).\n",
      "Movement of the ND6 mtRNA is restricted but oxidative stress induces the mtRNA to disperse in the\n",
      "mitochondria and gradually decompose. These probes provide a means to study spatial and temporal\n",
      "mRNA dynamics in intracellular compartments in living mammalian cells.\n",
      "['live cell imaging',\n",
      " 'genetically encoded RNA probes',\n",
      " 'split fragments of a fluorescent protein',\n",
      " 'real-time imaging',\n",
      " 'mitochondrial matrix',\n",
      " 'oxidative stress']\n",
      "Elapsed time: 11.26 seconds\n",
      "-------------------\n",
      "17486085\n",
      "High-pressure treatment of polytene chromosomes improves structural resolution.\n",
      "The exceptional cytology provided by polytene chromosomes has made Drosophila melanogaster a premier\n",
      "model for chromosome studies, but full exploitation of polytene cytology is impeded by the\n",
      "difficulty in preparing high-quality chromosome spreads. Here we describe use of high pressure to\n",
      "produce formaldehyde-fixed chromosome spreads, which upon light-microscopy examination reveal\n",
      "structural detail previously observed only in electron microscopy preparations. We demonstrate\n",
      "applications to immunofluorescence and in situ hybridization.\n",
      "['light microscopy', 'electron microscopy']\n",
      "Elapsed time: 7.32 seconds\n",
      "-------------------\n",
      "17486087\n",
      "Generation of functional hemangioblasts from human embryonic stem cells.\n",
      "Recent evidence suggests the existence of progenitor cells in adult tissues that are capable of\n",
      "differentiating into vascular structures as well as into all hematopoietic cell lineages. Here we\n",
      "describe an efficient and reproducible method for generating large numbers of these bipotential\n",
      "progenitors-known as hemangioblasts-from human embryonic stem (hES) cells using an in vitro\n",
      "differentiation system. Blast cells expressed gene signatures characteristic of hemangioblasts, and\n",
      "could be expanded, cryopreserved and differentiated into multiple hematopoietic lineages as well as\n",
      "into endothelial cells. When we injected these cells into rats with diabetes or into mice with\n",
      "ischemia-reperfusion injury of the retina, they localized to the site of injury in the damaged\n",
      "vasculature and appeared to participate in repair. Injection of the cells also reduced the mortality\n",
      "rate after myocardial infarction and restored blood flow in hind limb ischemia in mouse models. Our\n",
      "data suggest that hES-derived blast cells (hES-BCs) could be important in vascular repair.\n",
      "['in vitro differentiation system',\n",
      " 'endothelial cells',\n",
      " 'hematopoietic lineages',\n",
      " 'vascular repair']\n",
      "Elapsed time: 11.27 seconds\n",
      "-------------------\n",
      "17546037\n",
      "Nucleic acid and protein mass mapping by live-cell deep-ultraviolet microscopy.\n",
      "We developed a deep-ultraviolet (UV) microscope capable of imaging cell mitosis and motility at 280\n",
      "nm for 45 min with minimal UV-induced toxicity, and for 6 h before the onset of visible cell death\n",
      "in cultured human and mouse cells. Combined with computational methods that convert the intensity of\n",
      "each pixel into an estimate of mass, deep-UV microscopy images generate maps of nucleic acid mass,\n",
      "protein mass and fluorescence yield in unlabeled cells.\n",
      "['deep-ultraviolet microscopy', 'computational methods']\n",
      "Elapsed time: 7.33 seconds\n",
      "-------------------\n",
      "17558414\n",
      "Imaging axonal transport of mitochondria in vivo.\n",
      "Neuronal mitochondria regulate synaptic physiology and cellular survival, and disruption of their\n",
      "function or transport causes neurological disease. We present a fluorescence method to selectively\n",
      "image mitochondrial dynamics in the mouse nervous system, in both live mice and acute explants. We\n",
      "show that axon damage and recovery lead to early and sustained changes in anterograde and retrograde\n",
      "transport. In vivo imaging of mitochondria will be a useful tool to analyze this essential\n",
      "organelle.\n",
      "['fluorescence method', 'live mice', 'acute explants']\n",
      "Elapsed time: 7.12 seconds\n",
      "-------------------\n",
      "17572680\n",
      "Bright monomeric red fluorescent protein with an extended fluorescence lifetime.\n",
      "Fluorescent proteins have become extremely popular tools for in vivo imaging and especially for the\n",
      "study of localization, motility and interaction of proteins in living cells. Here we report TagRFP,\n",
      "a monomeric red fluorescent protein, which is characterized by high brightness, complete chromophore\n",
      "maturation, prolonged fluorescence lifetime and high pH-stability. These properties make TagRFP an\n",
      "excellent tag for protein localization studies and fluorescence resonance energy transfer (FRET)\n",
      "applications.\n",
      "['in vivo imaging',\n",
      " 'localization',\n",
      " 'motility',\n",
      " 'interaction of proteins',\n",
      " 'protein localization studies',\n",
      " 'fluorescence resonance energy transfer (FRET)']\n",
      "Elapsed time: 7.90 seconds\n",
      "-------------------\n",
      "17572681\n",
      "Ultrasound biomicroscopy permits in vivo characterization of zebrafish liver tumors.\n",
      "Zebrafish are a valuable vertebrate model to study carcinogenesis, but noninvasive imaging is\n",
      "challenging because adult fish are not transparent. Here we show that tumors could be readily\n",
      "detected in vivo using high-resolution microscopic ultrasound in zebrafish. We successfully obtained\n",
      "tissue perfusion calculations and cellular aspirates, and analyzed tumor progression and response to\n",
      "treatment. Ultrasound biomicroscopy allows longitudinal studies of tumor development and real-time\n",
      "assessment of therapeutic effects in zebrafish.\n",
      "['ultrasound biomicroscopy',\n",
      " 'microscopic ultrasound',\n",
      " 'tissue perfusion calculations',\n",
      " 'cellular aspirates']\n",
      "Elapsed time: 7.91 seconds\n",
      "-------------------\n",
      "17603490\n",
      "LambdaN-GFP: an RNA reporter system for live-cell imaging.\n",
      "We describe a GFP-based RNA reporter system (lambdaN-GFP) to visualize RNA molecules in live\n",
      "mammalian cells. It consists of GFP fused to an arginine-rich peptide derived from the phage lambda\n",
      "N protein, lambdaN22, which binds a unique minimal RNA motif and can be used to tag any RNA\n",
      "molecule. LambdaN-GFP uses a small and easy to engineer RNA tag, reducing the likelihood of\n",
      "perturbing the function of the tagged RNA molecule.\n",
      "['GFP-based RNA reporter system',\n",
      " 'live-cell imaging',\n",
      " 'arginine-rich peptide',\n",
      " 'small and easy to engineer RNA tag']\n",
      "Elapsed time: 8.05 seconds\n",
      "-------------------\n",
      "17618292\n",
      "Red-shifted Renilla reniformis luciferase variants for imaging in living subjects.\n",
      "The use of R. reniformis luciferase (RLuc) as a reporter gene in small-animal imaging has been\n",
      "hampered by its 481 nm peaked emission spectrum, as blue wavelengths are strongly attenuated in\n",
      "biological tissues. To overcome this, we generated variants of RLuc with bathochromic (red) shifts\n",
      "of up to 66 nm (547 nm peak) that also had greater stability and higher light emission than native\n",
      "RLuc.\n",
      "['RLuc', 'small-animal imaging', 'bathochromic shift', 'stability']\n",
      "Elapsed time: 7.35 seconds\n",
      "-------------------\n",
      "17618293\n",
      "Luciferase-YFP fusion tag with enhanced emission for single-cell luminescence imaging.\n",
      "Taking advantage of the phenomenon of bioluminescence resonance energy transfer (BRET), we developed\n",
      "a bioluminescent probe composed of EYFP and Renilla reniformis luciferase (RLuc)--BRET-based\n",
      "autoilluminated fluorescent protein on EYFP (BAF-Y)--for near-real-time single-cell imaging. We show\n",
      "that BAF-Y exhibits enhanced RLuc luminescence intensity and appropriate subcellular distribution\n",
      "when it was fused to targeting-signal peptides or histone H2AX, thus allowing high spatial and\n",
      "temporal resolution microscopy of living cells.\n",
      "['luminescence imaging',\n",
      " 'single-cell imaging',\n",
      " 'microscopy',\n",
      " 'high spatial and temporal resolution microscopy',\n",
      " 'bioluminescence resonance energy transfer (BRET)']\n",
      "Elapsed time: 8.59 seconds\n",
      "-------------------\n",
      "17664946\n",
      "Caged compounds: photorelease technology for control of cellular chemistry and physiology.\n",
      "Caged compounds are light-sensitive probes that functionally encapsulate biomolecules in an inactive\n",
      "form. Irradiation liberates the trapped molecule, permitting targeted perturbation of a biological\n",
      "process. Uncaging technology and fluorescence microscopy are 'optically orthogonal': the former\n",
      "allows control, and the latter, observation of cellular function. Used in conjunction with other\n",
      "technologies (for example, patch clamp and/or genetics), the light beam becomes a uniquely powerful\n",
      "tool to stimulate a selected biological target in space or time. Here I describe important examples\n",
      "of widely used caged compounds, their design features and synthesis, as well as practical details of\n",
      "how to use them with living cells.\n",
      "['fluorescence microscopy', 'uncaging technology', 'patch clamp', 'genetics']\n",
      "Elapsed time: 8.85 seconds\n",
      "-------------------\n",
      "17694065\n",
      "Tomographic phase microscopy.\n",
      "We report a technique for quantitative three-dimensional (3D) mapping of refractive index in live\n",
      "cells and tissues using a phase-shifting laser interferometric microscope with variable illumination\n",
      "angle. We demonstrate tomographic imaging of cells and multicellular organisms, and time-dependent\n",
      "changes in cell structure. Our results will permit quantitative characterization of specimen-induced\n",
      "aberrations in high-resolution microscopy and have multiple applications in tissue light scattering.\n",
      "['tomographic phase microscopy',\n",
      " 'phase-shifting laser interferometric microscope',\n",
      " 'time-dependent changes in cell structure']\n",
      "Elapsed time: 7.08 seconds\n",
      "-------------------\n",
      "17704783\n",
      "Microfluidics for in vivo imaging of neuronal and behavioral activity in Caenorhabditis elegans.\n",
      "The nematode C. elegans is an excellent model organism for studying behavior at the neuronal level.\n",
      "Because of the organism's small size, it is challenging to deliver stimuli to C. elegans and monitor\n",
      "neuronal activity in a controlled environment. To address this problem, we developed two\n",
      "microfluidic chips, the 'behavior' chip and the 'olfactory' chip for imaging of neuronal and\n",
      "behavioral responses in C. elegans. We used the behavior chip to correlate the activity of AVA\n",
      "command interneurons with the worm locomotion pattern. We used the olfactory chip to record\n",
      "responses from ASH sensory neurons exposed to high-osmotic-strength stimulus. Observation of\n",
      "neuronal responses in these devices revealed previously unknown properties of AVA and ASH neurons.\n",
      "The use of these chips can be extended to correlate the activity of sensory neurons, interneurons\n",
      "and motor neurons with the worm's behavior.\n",
      "['behavior chip', 'olfactory chip', 'microfluidics']\n",
      "Elapsed time: 10.56 seconds\n",
      "-------------------\n",
      "17721542\n",
      "Bright far-red fluorescent protein for whole-body imaging.\n",
      "For deep imaging of animal tissues, the optical window favorable for light penetration is in near-\n",
      "infrared wavelengths, which requires proteins with emission spectra in the far-red wavelengths. Here\n",
      "we report a far-red fluorescent protein, named Katushka, which is seven- to tenfold brighter\n",
      "compared to the spectrally close HcRed or mPlum, and is characterized by fast maturation as well as\n",
      "a high pH-stability and photostability. These unique characteristics make Katushka the protein of\n",
      "choice for visualization in living tissues. We demonstrate superiority of Katushka for whole-body\n",
      "imaging by direct comparison with other red and far-red fluorescent proteins. We also describe a\n",
      "monomeric version of Katushka, named mKate, which is characterized by high brightness and\n",
      "photostability, and should be an excellent fluorescent label for protein tagging in the far-red part\n",
      "of the spectrum.\n",
      "['whole-body imaging', 'deep imaging', 'far-red fluorescent protein']\n",
      "Elapsed time: 9.49 seconds\n",
      "-------------------\n",
      "17762877\n",
      "Solution NMR of supramolecular complexes: providing new insights into function.\n",
      "Solution NMR spectroscopy is an extremely powerful technology for the study of biomolecular dynamics\n",
      "and site-specific molecular interactions. An important limitation in the past has been molecule\n",
      "size, with molecular weights of targets seldom exceeding 50 kDa. New labeling technology and NMR\n",
      "experiments are changing this paradigm so that applications for investigating supramolecular\n",
      "complexes are starting to become feasible. Here we describe a strategy developed in our laboratory\n",
      "that involves the use of labeled methyl groups of isoleucine, leucine and valine residues in\n",
      "proteins as probes, along with experiments that significantly enhance the lifetimes of the resulting\n",
      "signals. We describe the application of these methods to a number of systems with molecular weights\n",
      "in the hundreds of kilodaltons.\n",
      "['NMR spectroscopy', 'labeling technology']\n",
      "Elapsed time: 8.23 seconds\n",
      "-------------------\n",
      "17828271\n",
      "Three-dimensional preparation and imaging reveal intrinsic microtubule properties.\n",
      "We present an experimental investigation of microtubule dynamic instability in three dimensions,\n",
      "based on laser light-sheet fluorescence microscopy. We introduce three-dimensional (3D) preparation\n",
      "of Xenopus laevis egg extracts in Teflon-based cylinders and provide algorithms for 3D image\n",
      "processing. Our approach gives experimental access to the intrinsic dynamic properties of\n",
      "microtubules and to microtubule population statistics in single asters. We obtain evidence for a\n",
      "stochastic nature of microtubule pausing.\n",
      "['laser light-sheet fluorescence microscopy',\n",
      " 'three-dimensional preparation',\n",
      " 'image processing algorithms']\n",
      "Elapsed time: 7.64 seconds\n",
      "-------------------\n",
      "17901873\n",
      "MALDI imaging mass spectrometry: molecular snapshots of biochemical systems.\n",
      "Matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS) is emerging as a\n",
      "powerful tool for investigating the distribution of molecules within biological systems through the\n",
      "direct analysis of thin tissue sections. Unique among imaging methods, MALDI-IMS can determine the\n",
      "distribution of hundreds of unknown compounds in a single measurement. We discuss the current state\n",
      "of the art of MALDI-IMS along with some recent applications and technological developments that\n",
      "illustrate not only its current capabilities but also the future potential of the technique to\n",
      "provide a better understanding of the underlying molecular mechanisms of biological processes.\n",
      "['MALDI imaging mass spectrometry',\n",
      " 'matrix-assisted laser desorption/ionization',\n",
      " 'imaging methods']\n",
      "Elapsed time: 8.37 seconds\n",
      "-------------------\n",
      "17952088\n",
      "STED microscopy with continuous wave beams.\n",
      "We report stimulated emission depletion (STED) fluorescence microscopy with continuous wave (CW)\n",
      "laser beams. Lateral fluorescence confinement from the scanning focal spot delivered a resolution of\n",
      "29-60 nm in the focal plane, corresponding to a 5-8-fold improvement over the diffraction barrier.\n",
      "Axial spot confinement increased the axial resolution by 3.5-fold. We observed three-dimensional\n",
      "(3D) subdiffraction resolution in 3D image stacks. Viable for fluorophores with low triplet yield,\n",
      "the use of CW light sources greatly simplifies the implementation of this concept of far-field\n",
      "fluorescence nanoscopy.\n",
      "['STED microscopy', 'continuous wave beams', 'far-field fluorescence nanoscopy']\n",
      "Elapsed time: 8.85 seconds\n",
      "-------------------\n",
      "17965719\n",
      "Two-photon photostimulation and imaging of neural circuits.\n",
      "We introduce an optical method to stimulate individual neurons in brain slices in any arbitrary\n",
      "spatiotemporal pattern, using two-photon uncaging of MNI-glutamate with beam multiplexing. This\n",
      "method has single-cell and three-dimensional precision. By sequentially stimulating up to a thousand\n",
      "potential presynaptic neurons, we generated detailed functional maps of inputs to a cell. We\n",
      "combined this approach with two-photon calcium imaging in an all-optical method to image and\n",
      "manipulate circuit activity.\n",
      "['two-photon photostimulation',\n",
      " 'two-photon calcium imaging',\n",
      " 'beam multiplexing']\n",
      "Elapsed time: 7.01 seconds\n",
      "-------------------\n",
      "17971780\n",
      "An evaluation of 3C-based methods to capture DNA interactions.\n",
      "The shape of the genome is thought to play an important part in the coordination of transcription\n",
      "and other DNA-metabolic processes. Chromosome conformation capture (3C) technology allows us to\n",
      "analyze the folding of chromatin in the native cellular state at a resolution beyond that provided\n",
      "by current microscopy techniques. It has been used, for example, to demonstrate that regulatory DNA\n",
      "elements communicate with distant target genes through direct physical interactions that loop out\n",
      "the intervening chromatin fiber. Here we discuss the intricacies of 3C and new 3C-based methods\n",
      "including the 4C, 5C and ChIP-loop assay.\n",
      "['3C', '4C', '5C', 'ChIP-loop assay']\n",
      "Elapsed time: 7.83 seconds\n",
      "-------------------\n",
      "17971781\n",
      "Fluorescence correlation spectroscopy in living cells.\n",
      "Fluorescence correlation spectroscopy (FCS) is an ideal analytical tool for studying concentrations,\n",
      "propagation, interactions and internal dynamics of molecules at nanomolar concentrations in living\n",
      "cells. FCS analyzes minute fluorescence-intensity fluctuations about the equilibrium of a small\n",
      "ensemble (<10(3)) of molecules. These fluctuations act like a 'fingerprint' of a molecular species\n",
      "detected when entering and leaving a femtoliter-sized optically defined observation volume created\n",
      "by a focused laser beam. In FCS the fluorescence fluctuations are recorded as a function of time and\n",
      "then statistically analyzed by autocorrelation analysis. The resulting autocorrelation curve yields\n",
      "a measure of self-similarity of the system after a certain time delay, and its amplitude describes\n",
      "the normalized variance of the fluorescence fluctuations. By fitting the curves to an appropriate\n",
      "physical model, this method provides precise information about a multitude of measurement\n",
      "parameters, including diffusion coefficients, local concentration, states of aggregation and\n",
      "molecular interactions. FCS operates in real time with diffraction-limited spatial and sub-\n",
      "microsecond temporal resolution. Assessing diverse molecular dynamics within the living cell is a\n",
      "challenge well met by FCS because of its single-molecule sensitivity and high dynamic resolution.\n",
      "For these same reasons, however, intracellular FCS measurements also harbor the large risk of\n",
      "collecting artifacts and thus producing erroneous data. Here we provide a step-by-step guide to the\n",
      "application of FCS to cellular systems, including methods for minimizing artifacts, optimizing\n",
      "measurement conditions and obtaining parameter values in the face of diverse and complex conditions\n",
      "of the living cell. A discussion of advantages and disadvantages of one-photon versus two-photon\n",
      "excitation for FCS is available in Supplementary Methods online.\n",
      "['fluorescence correlation spectroscopy',\n",
      " 'autocorrelation analysis',\n",
      " 'one-photon excitation',\n",
      " 'two-photon excitation']\n",
      "Elapsed time: 17.91 seconds\n",
      "-------------------\n",
      "18066071\n",
      "In vivo imaging of Drosophila melanogaster pupae with mesoscopic fluorescence tomography.\n",
      "We report a technique for fluorescence tomography that operates beyond the penetration limits of\n",
      "tissue-sectioning fluorescence microscopy. The method uses multi-projection illumination and photon\n",
      "transport description in opaque tissues. We demonstrate whole-body three-dimensional visualization\n",
      "of the morphogenesis of GFP-expressing salivary glands and wing imaginal discs in living Drosophila\n",
      "melanogaster pupae in vivo and over time.\n",
      "['fluorescence tomography',\n",
      " 'multi-projection illumination',\n",
      " 'photon transport description in opaque tissues']\n",
      "Elapsed time: 7.34 seconds\n",
      "-------------------\n",
      "18084298\n",
      "Integrating spatially resolved three-dimensional MALDI IMS with in vivo magnetic resonance imaging.\n",
      "We have developed a method for integrating three dimensional-volume reconstructions of spatially\n",
      "resolved matrix-assisted laser desorption/ionization imaging mass spectrometry (MALDI IMS) ion\n",
      "images of whole mouse heads with high-resolution images from other modalities in an animal-specific\n",
      "manner. This approach enabled us to analyze proteomic profiles from MALDI IMS data with\n",
      "corresponding in vivo data provided by magnetic resonance imaging.\n",
      "['MALDI IMS', 'magnetic resonance imaging']\n",
      "Elapsed time: 6.73 seconds\n",
      "-------------------\n",
      "18084300\n",
      "Tracking transmitter-gated P2X cation channel activation in vitro and in vivo.\n",
      "We present a noninvasive approach to track activation of ATP-gated P2X receptors and potentially\n",
      "other transmitter-gated cation channels that show calcium fluxes. We genetically engineered rat P2X\n",
      "receptors to carry calcium sensors near the channel pore and tested this as a reporter for P2X(2)\n",
      "receptor opening. The method has several advantages over previous attempts to image P2X channel\n",
      "activation by fluorescence resonance energy transfer (FRET): notably, it reports channel opening\n",
      "rather than a conformation change in the receptor protein. Our FRET-based imaging approach can be\n",
      "used as a general method to track, in real time, the location, regional expression variation,\n",
      "mobility and activation of transmitter-gated P2X channels in living neurons in vitro and in vivo.\n",
      "This approach should help to determine when, where and how different receptors are activated during\n",
      "physiological processes.\n",
      "['FRET', 'real-time', 'in vitro', 'in vivo']\n",
      "Elapsed time: 9.21 seconds\n",
      "-------------------\n",
      "18157136\n",
      "Targeted patch-clamp recordings and single-cell electroporation of unlabeled neurons in vivo.\n",
      "Here we describe an approach for making targeted patch-clamp recordings from single neurons in vivo,\n",
      "visualized by two-photon microscopy. A patch electrode is used to perfuse the extracellular space\n",
      "surrounding the neuron of interest with a fluorescent dye, thus enabling the neuron to be visualized\n",
      "as a negative image ('shadow') and identified on the basis of its somatodendritic structure. The\n",
      "same electrode is then placed on the neuron under visual control to allow formation of a gigaseal\n",
      "('shadowpatching'). We demonstrate the reliability and versatility of shadowpatching by performing\n",
      "whole-cell recordings from visually identified neurons in the neocortex and cerebellum of rat and\n",
      "mouse. We also show that the method can be used for targeted in vivo single-cell electroporation of\n",
      "plasmid DNA into identified cell types, leading to stable transgene expression. This approach\n",
      "facilitates the recording, labeling and genetic manipulation of single neurons in the intact native\n",
      "mammalian brain without the need to pre-label neuronal populations.\n",
      "{'imagingTechniques': ['two-photon microscopy',\n",
      "                       'patch-clamp recordings',\n",
      "                       'single-cell electroporation',\n",
      "                       'shadow patching']}\n",
      "Elapsed time: 12.01 seconds\n",
      "-------------------\n",
      "18165805\n",
      "Live-cell imaging of cAMP dynamics.\n",
      "Spatial and temporal compartmentalization of cAMP (and its target proteins) is central to the\n",
      "ability of this second messenger to govern cellular activity over timescales ranging from\n",
      "milliseconds to several hours. Recent years have witnessed a burgeoning of methodologies that enable\n",
      "researchers to directly monitor rapid subcellular cAMP dynamics, which are unobtainable by\n",
      "traditional cAMP assays. In this review, we examine cAMP biosensors that are currently available for\n",
      "measuring cAMP at the single-cell level, compare their various operating principles and discuss\n",
      "their applications.\n",
      "['live-cell imaging', 'single-cell level', 'cAMP biosensors']\n",
      "Elapsed time: 6.86 seconds\n",
      "-------------------\n",
      "18165806\n",
      "Imaging mass spectrometry of proteins and peptides: 3D volume reconstruction.\n",
      "As large genomic and proteomic datasets are generated from homogenates of various tissues, the need\n",
      "for information on the spatial localization of their encoded products has become more pressing.\n",
      "Matrix-assisted laser desorption-ionization (MALDI) imaging mass spectrometry (IMS) offers\n",
      "investigators the means with which to unambiguously study peptides and proteins with molecular\n",
      "specificity, and to determine their distribution in two and three dimensions. In the past few years,\n",
      "several parameters have been optimized for IMS, including sample preparation, matrix application and\n",
      "instrumental acquisition parameters (Box 1). These developments have resulted in a high degree of\n",
      "reproducibility in mass accuracy and peak intensities (Supplementary Fig. 1 online). Recently, we\n",
      "have optimized our protocol to be able to increase the number of molecular species analyzed by\n",
      "collecting two sets of sections, covering one set of sections with sinapinic acid for optimal\n",
      "detection of proteins and adjacent sections with 2,5-dihydroxybenzoic acid (DHB) matrix for the\n",
      "optimal detection of low-mass species, including peptides. Approximately 1,000 peaks can be observed\n",
      "in each dataset (Fig. 1). Furthermore, the sections are collected at an equal distance, 200 mum\n",
      "instead of 400-500 mum used previously, thus enabling the use of virtual z-stacks and three-\n",
      "dimensional (3D) volume renderings to investigate differential localization patterns in much smaller\n",
      "brain structures such as the substantia nigra and the interpeduncular nucleus. Here we present our\n",
      "optimized step-by-step procedure based on previous work in our laboratory, describing how to make 3D\n",
      "volume reconstructions of MALDI IMS data, as applied to the rat brain.\n",
      "['MALDI', 'IMS', '3D volume reconstruction']\n",
      "Elapsed time: 16.86 seconds\n",
      "-------------------\n",
      "18176568\n",
      "Highly inclined thin illumination enables clear single-molecule imaging in cells.\n",
      "We describe a simple illumination method of fluorescence microscopy for molecular imaging.\n",
      "Illumination by a highly inclined and thin beam increases image intensity and decreases background\n",
      "intensity, yielding a signal/background ratio about eightfold greater than that of epi-illumination.\n",
      "A high ratio yielded clear single-molecule images and three-dimensional images using cultured\n",
      "mammalian cells, enabling one to visualize and quantify molecular dynamics, interactions and\n",
      "kinetics in cells for molecular systems biology.\n",
      "['fluorescence microscopy',\n",
      " 'epi-illumination',\n",
      " 'highly inclined thin illumination']\n",
      "Elapsed time: 7.23 seconds\n",
      "-------------------\n",
      "18193054\n",
      "High-density mapping of single-molecule trajectories with photoactivated localization microscopy.\n",
      "We combined photoactivated localization microscopy (PALM) with live-cell single-particle tracking to\n",
      "create a new method termed sptPALM. We created spatially resolved maps of single-molecule motions by\n",
      "imaging the membrane proteins Gag and VSVG, and obtained several orders of magnitude more\n",
      "trajectories per cell than traditional single-particle tracking enables. By probing distinct subsets\n",
      "of molecules, sptPALM can provide insight into the origins of spatial and temporal heterogeneities\n",
      "in membranes.\n",
      "['PALM', 'live-cell single-particle tracking', 'single-molecule trajectories']\n",
      "Elapsed time: 7.48 seconds\n",
      "-------------------\n",
      "18204457\n",
      "A secreted luciferase for ex vivo monitoring of in vivo processes.\n",
      "Luciferases are widely used to monitor biological processes. Here we describe the naturally secreted\n",
      "Gaussia princeps luciferase (Gluc) as a highly sensitive reporter for quantitative assessment of\n",
      "cells in vivo by measuring its concentration in blood. The Gluc blood assay complements in vivo\n",
      "bioluminescence imaging, which has the ability to localize the signal and provides a multifaceted\n",
      "assessment of cell viability, proliferation and location in experimental disease and therapy models.\n",
      "['bioluminescence imaging', 'in vivo imaging', 'blood assay']\n",
      "Elapsed time: 7.40 seconds\n",
      "-------------------\n",
      "18204458\n",
      "High-speed, low-photodamage nonlinear imaging using passive pulse splitters.\n",
      "Pulsed lasers are key elements in nonlinear bioimaging techniques such as two-photon fluorescence\n",
      "excitation (TPE) microscopy. Typically, however, only a percent or less of the laser power available\n",
      "can be delivered to the sample before photoinduced damage becomes excessive. Here we describe a\n",
      "passive pulse splitter that converts each laser pulse into a fixed number of sub-pulses of equal\n",
      "energy. We applied the splitter to TPE imaging of fixed mouse brain slices labeled with GFP and show\n",
      "that, in different power regimes, the splitter can be used either to increase the signal rate more\n",
      "than 100-fold or to reduce the rate of photobleaching by over fourfold. In living specimens, the\n",
      "gains were even greater: a ninefold reduction in photobleaching during in vivo imaging of\n",
      "Caenorhabditis elegans larvae, and a six- to 20-fold decrease in the rate of photodamage during\n",
      "calcium imaging of rat hippocampal brain slices.\n",
      "['two-photon fluorescence excitation (TPE) microscopy',\n",
      " 'passive pulse splitter']\n",
      "Elapsed time: 10.44 seconds\n",
      "-------------------\n",
      "18246076\n",
      "Extending the NMR spatial resolution limit for RNA by motional couplings.\n",
      "Experimental resolution of distinct dynamical processes in molecules can prove impossible when they\n",
      "are correlated to one another. In nuclear magnetic resonance (NMR) spectroscopy, couplings between\n",
      "internal and overall motions lead to intractable complexity, depriving insights into functionally\n",
      "important motions. Here we demonstrate that motional couplings can be used to anchor NMR frames of\n",
      "reference onto different parts of an RNA molecule, thus extending the spatial resolution limit for\n",
      "dynamical characterization.\n",
      "['NMR spectroscopy', 'NMR frames of reference']\n",
      "Elapsed time: 6.74 seconds\n",
      "-------------------\n",
      "18309310\n",
      "Induction of linear tracks of DNA double-strand breaks by alpha-particle irradiation of cells.\n",
      "Understanding how cells maintain genome integrity when challenged with DNA double-strand breaks\n",
      "(DSBs) is of major importance, particularly since the discovery of multiple links of DSBs with\n",
      "genome instability and cancer-predisposition disorders. Ionizing radiation is the agent of choice to\n",
      "produce DSBs in cells; however, targeting DSBs and monitoring changes in their position over time\n",
      "can be difficult. Here we describe a procedure for induction of easily recognizable linear arrays of\n",
      "DSBs in nuclei of adherent eukaryotic cells by exposing the cells to alpha particles from a small\n",
      "Americium source (Box 1). Each alpha particle traversing the cell nucleus induces a linear array of\n",
      "DSBs, typically 10-20 DSBs per 10 mum track length. Because alpha particles cannot penetrate cell-\n",
      "culture plastic or coverslips, it is necessary to irradiate cells through a Mylar membrane. We\n",
      "describe setup and irradiation procedures for two types of experiments: immunodetection of DSB\n",
      "response proteins in fixed cells grown in Mylar-bottom culture dishes (Option A) and detection of\n",
      "fluorescently labeled DSB-response proteins in living cells irradiated through a Mylar membrane\n",
      "placed on top of the cells (Option B). Using immunodetection, recruitment of repair proteins to\n",
      "individual DSB sites as early as 30 s after irradiation can be detected. Furthermore, combined with\n",
      "fluorescence live-cell microscopy of fluorescently tagged DSB-response proteins, this technique\n",
      "allows spatiotemporal analysis of the DSB repair response in living cells. Although the procedures\n",
      "might seem a bit intimidating, in our experience, once the source and the setup are ready, it is\n",
      "easy to obtain results. Because the live-cell procedure requires more hands-on experience, we\n",
      "recommend starting with the fixed-cell application.\n",
      "['immunodetection',\n",
      " 'fluorescently labeled DSB-response proteins',\n",
      " 'fluorescence live-cell microscopy']\n",
      "Elapsed time: 18.48 seconds\n",
      "-------------------\n",
      "18345005\n",
      "In vivo immunogold labeling confirms large-scale chromatin folding motifs.\n",
      "The difficulty in localizing specific cellular proteins by immuno-electron microscopy techniques\n",
      "limits applications of electron microscopy to cell biology. We found that in vivo immunogold\n",
      "labeling improves epitope accessibility, ultrastructural preservation and three-dimensional\n",
      "visualization, and allows correlated light and electron microscopy. We detected large-scale\n",
      "chromatin folding motifs within intact interphase nuclei of CHO cells and visualized the\n",
      "ultrastructure of DNA replication 'factories' labeled with GFP-proliferating cell nuclear antigen\n",
      "(PCNA).\n",
      "['in vivo immunogold labeling',\n",
      " 'immuno-electron microscopy',\n",
      " 'correlated light and electron microscopy']\n",
      "Elapsed time: 8.33 seconds\n",
      "-------------------\n",
      "18345008\n",
      "Direct measurement of protein dynamics inside cells using a rationally designed photoconvertible\n",
      "protein.\n",
      "All biological reactions depend on the diffusion and re-localization of biomolecules. Our\n",
      "understanding of biological processes requires accurate measurement of biomolecule mobility in\n",
      "living cells. Currently, approaches for investigating the mobility of biomolecules are generally\n",
      "restricted to measuring either fast or slow diffusion kinetics. We describe the development and\n",
      "application of a photoconvertible fluorescent protein, Phamret, that can be highlighted by UV light\n",
      "stimulation inducing a change in fluorescence emission from cyan fluorescent protein (CFP) to\n",
      "photoactivated GFP (PA-GFP). Phamret can be monitored by single excitation-dual emission mode for\n",
      "visualization of molecular dynamics for a broad range of kinetics. We also devised a microscopy-\n",
      "based method to measure the diffusion coefficient from the fluorescence decay after photostimulation\n",
      "of Phamret, enabling analysis of diffusion kinetics ranging from less than 0.1 microm2/s up to\n",
      "approximately 100 microm2/s, and found significant changes in free protein movement during cell-\n",
      "cycle progression.\n",
      "['single excitation-dual emission mode',\n",
      " 'microscopy-based method',\n",
      " 'photoconvertible protein']\n",
      "Elapsed time: 10.58 seconds\n",
      "-------------------\n",
      "18408726\n",
      "Live-cell photoactivated localization microscopy of nanoscale adhesion dynamics.\n",
      "We demonstrate live-cell super-resolution imaging using photoactivated localization microscopy\n",
      "(PALM). The use of photon-tolerant cell lines in combination with the high resolution and molecular\n",
      "sensitivity of PALM permitted us to investigate the nanoscale dynamics within individual adhesion\n",
      "complexes (ACs) in living cells under physiological conditions for as long as 25 min, with half of\n",
      "the time spent collecting the PALM images at spatial resolutions down to approximately 60 nm and\n",
      "frame rates as short as 25 s. We visualized the formation of ACs and measured the fractional gain\n",
      "and loss of individual paxillin molecules as each AC evolved. By allowing observation of a wide\n",
      "variety of nanoscale dynamics, live-cell PALM provides insights into molecular assembly during the\n",
      "initiation, maturation and dissolution of cellular processes.\n",
      "['PALM',\n",
      " 'live-cell super-resolution imaging',\n",
      " 'photoactivated localization microscopy']\n",
      "Elapsed time: 9.21 seconds\n",
      "-------------------\n",
      "18425137\n",
      "Fluorescent protein FRET pairs for ratiometric imaging of dual biosensors.\n",
      "Fluorescence resonance energy transfer (FRET) with fluorescent proteins is a powerful method for\n",
      "detection of protein-protein interactions, enzyme activities and small molecules in the\n",
      "intracellular milieu. Aided by a new violet-excitable yellow-fluorescing variant of Aequorea\n",
      "victoria GFP, we developed dual FRET-based caspase-3 biosensors. Owing to their distinct excitation\n",
      "profiles, each FRET biosensor can be ratiometrically imaged in the presence of the other.\n",
      "['Fluorescence resonance energy transfer',\n",
      " 'fluorescent proteins',\n",
      " 'ratiometric imaging']\n",
      "Elapsed time: 7.68 seconds\n",
      "-------------------\n",
      "18425138\n",
      "Monovalent, reduced-size quantum dots for imaging receptors on living cells.\n",
      "We describe a method to generate monovalent quantum dots (QDs) using agarose gel electrophoresis. We\n",
      "passivated QDs with a carboxy-terminated polyethylene-glycol ligand, yielding particles with half\n",
      "the diameter of commercial QDs, which we conjugated to a single copy of a high-affinity targeting\n",
      "moiety (monovalent streptavidin or antibody to carcinoembryonic antigen) to label cell-surface\n",
      "proteins. The small size improved access of QD-labeled glutamate receptors to neuronal synapses, and\n",
      "monovalency prevented EphA3 tyrosine kinase activation.\n",
      "['agarose gel electrophoresis',\n",
      " 'polyethylene-glycol ligand',\n",
      " 'monovalent streptavidin',\n",
      " 'antibody to carcinoembryonic antigen']\n",
      "Elapsed time: 9.08 seconds\n",
      "-------------------\n",
      "18425139\n",
      "Modeling lymphangiogenesis in a three-dimensional culture system.\n",
      "A lack of appropriate in vitro models of three-dimensional lymph vessel growth hampers the study of\n",
      "lymphangiogenesis. We developed a lymphatic ring assay--a potent, reproducible and quantifiable\n",
      "three-dimensional culture system for lymphatic endothelial cells that reproduces spreading of\n",
      "endothelial cells from a pre-existing vessel, cell proliferation, migration and differentiation into\n",
      "capillaries. In the assay, mouse thoracic duct fragments are embedded in a collagen gel, leading to\n",
      "the formation of lumen-containing lymphatic capillaries, which we assessed by electron microscopy\n",
      "and immunostaining. We developed a computerized method to quantify the lymphatic network. By\n",
      "applying this model to gene-deficient mice, we found evidence for involvement of the matrix\n",
      "metalloproteinase, MMP-2, in lymphangiogenesis. The lymphatic ring assay bridges the gap between\n",
      "two-dimensional in vitro models and in vivo models of lymphangiogenesis, can be used to exploit the\n",
      "potential of existing transgenic mouse models, and rapidly identify regulators of lymphangiogenesis.\n",
      "['electron microscopy', 'immunostaining', 'computerized method']\n",
      "Elapsed time: 11.70 seconds\n",
      "-------------------\n",
      "18469822\n",
      "Real-time imaging of the intracellular glutathione redox potential.\n",
      "Dynamic analysis of redox-based processes in living cells is now restricted by the lack of\n",
      "appropriate redox biosensors. Conventional redox-sensitive GFPs (roGFPs) are limited by undefined\n",
      "specificity and slow response to changes in redox potential. In this study we demonstrate that the\n",
      "fusion of human glutaredoxin-1 (Grx1) to roGFP2 facilitates specific real-time equilibration between\n",
      "the sensor protein and the glutathione redox couple. The Grx1-roGFP2 fusion protein allowed dynamic\n",
      "live imaging of the glutathione redox potential (E(GSH)) in different cellular compartments with\n",
      "high sensitivity and temporal resolution. The biosensor detected nanomolar changes in oxidized\n",
      "glutathione (GSSG) against a backdrop of millimolar reduced glutathione (GSH) on a scale of seconds\n",
      "to minutes. It facilitated the observation of redox changes associated with growth factor\n",
      "availability, cell density, mitochondrial depolarization, respiratory burst activity and immune\n",
      "receptor stimulation.\n",
      "['live imaging', 'real-time imaging', 'dynamic analysis', 'biosensor']\n",
      "Elapsed time: 10.85 seconds\n",
      "-------------------\n",
      "18469823\n",
      "Three-dimensional sub-100 nm resolution fluorescence microscopy of thick samples.\n",
      "Imaging volumes as thick as whole cells at three-dimensional (3D) super-resolution is required to\n",
      "reveal unknown features of cellular organization. We report a light microscope that generates images\n",
      "with translationally invariant 30 x 30 x 75 nm resolution over a depth of several micrometers. This\n",
      "method, named biplane (BP) FPALM, combines a double-plane detection scheme with fluorescence\n",
      "photoactivation localization microscopy (FPALM) enabling 3D sub-diffraction resolution without\n",
      "compromising speed or sensitivity.\n",
      "['biplane FPALM',\n",
      " 'fluorescence photoactivation localization microscopy',\n",
      " 'FPALM',\n",
      " 'three-dimensional super-resolution microscopy']\n",
      "Elapsed time: 8.00 seconds\n",
      "-------------------\n",
      "18488034\n",
      "Spherical nanosized focal spot unravels the interior of cells.\n",
      "The resolution of any linear imaging system is given by its point spread function (PSF) that\n",
      "quantifies the blur of an object point in the image. The sharper the PSF, the better the resolution\n",
      "is. In standard fluorescence microscopy, however, diffraction dictates a PSF with a cigar-shaped\n",
      "main maximum, called the focal spot, which extends over at least half the wavelength of light\n",
      "(lambda = 400-700 nm) in the focal plane and >lambda along the optical axis (z). Although concepts\n",
      "have been developed to sharpen the focal spot both laterally and axially, none of them has reached\n",
      "their ultimate goal: a spherical spot that can be arbitrarily downscaled in size. Here we introduce\n",
      "a fluorescence microscope that creates nearly spherical focal spots of 40-45 nm (lambda/16) in\n",
      "diameter. Fully relying on focused light, this lens-based fluorescence nanoscope unravels the\n",
      "interior of cells noninvasively, uniquely dissecting their sub-lambda-sized organelles.\n",
      "['fluorescence microscopy',\n",
      " 'linear imaging system',\n",
      " 'point spread function (PSF)',\n",
      " 'standard fluorescence microscopy',\n",
      " 'diffraction',\n",
      " 'spherical focal spot']\n",
      "Elapsed time: 11.63 seconds\n",
      "-------------------\n",
      "18488035\n",
      "Cell-surface protein-protein interaction analysis with time-resolved FRET and snap-tag technologies:\n",
      "application to GPCR oligomerization.\n",
      "Cell-surface proteins are important in cell-cell communication. They assemble into heterocomplexes\n",
      "that include different receptors and effectors. Elucidation and manipulation of such protein\n",
      "complexes offers new therapeutic possibilities. We describe a methodology combining time-resolved\n",
      "fluorescence resonance energy transfer (FRET) with snap-tag technology to quantitatively analyze\n",
      "protein-protein interactions at the surface of living cells, in a high throughput-compatible format.\n",
      "Using this approach, we examined whether G protein-coupled receptors (GPCRs) are monomers or\n",
      "assemble into dimers or larger oligomers--a matter of intense debate. We obtained evidence for the\n",
      "oligomeric state of both class A and class C GPCRs. We also observed different quaternary structure\n",
      "of GPCRs for the neurotransmitters glutamate and gamma-aminobutyric acid (GABA): whereas\n",
      "metabotropic glutamate receptors assembled into strict dimers, the GABA(B) receptors spontaneously\n",
      "formed dimers of heterodimers, offering a way to modulate G-protein coupling efficacy. This approach\n",
      "will be useful in systematic analysis of cell-surface protein interaction in living cells.\n",
      "['FRET',\n",
      " 'snap-tag technology',\n",
      " 'time-resolved fluorescence resonance energy transfer']\n",
      "Elapsed time: 12.87 seconds\n",
      "-------------------\n",
      "18511916\n",
      "Do-it-yourself guide: how to use the modern single-molecule toolkit.\n",
      "Single-molecule microscopy has evolved into the ultimate-sensitivity toolkit to study systems from\n",
      "small molecules to living cells, with the prospect of revolutionizing the modern biosciences. Here\n",
      "we survey the current state of the art in single-molecule tools including fluorescence spectroscopy,\n",
      "tethered particle microscopy, optical and magnetic tweezers, and atomic force microscopy. We also\n",
      "provide guidelines for choosing the right approach from the available single-molecule toolkit for\n",
      "applications as diverse as structural biology, enzymology, nanotechnology and systems biology.\n",
      "['fluorescence spectroscopy',\n",
      " 'tethered particle microscopy',\n",
      " 'optical tweezers',\n",
      " 'magnetic tweezers',\n",
      " 'atomic force microscopy']\n",
      "Elapsed time: 8.67 seconds\n",
      "-------------------\n",
      "18511917\n",
      "Single-molecule force spectroscopy: optical tweezers, magnetic tweezers and atomic force microscopy.\n",
      "Single-molecule force spectroscopy has emerged as a powerful tool to investigate the forces and\n",
      "motions associated with biological molecules and enzymatic activity. The most common force\n",
      "spectroscopy techniques are optical tweezers, magnetic tweezers and atomic force microscopy. Here we\n",
      "describe these techniques and illustrate them with examples highlighting current capabilities and\n",
      "limitations.\n",
      "['optical tweezers', 'magnetic tweezers', 'atomic force microscopy']\n",
      "Elapsed time: 6.42 seconds\n",
      "-------------------\n",
      "18511918\n",
      "A practical guide to single-molecule FRET.\n",
      "Single-molecule fluorescence resonance energy transfer (smFRET) is one of the most general and\n",
      "adaptable single-molecule techniques. Despite the explosive growth in the application of smFRET to\n",
      "answer biological questions in the last decade, the technique has been practiced mostly by\n",
      "biophysicists. We provide a practical guide to using smFRET, focusing on the study of immobilized\n",
      "molecules that allow measurements of single-molecule reaction trajectories from 1 ms to many\n",
      "minutes. We discuss issues a biologist must consider to conduct successful smFRET experiments,\n",
      "including experimental design, sample preparation, single-molecule detection and data analysis. We\n",
      "also describe how a smFRET-capable instrument can be built at a reasonable cost with off-the-shelf\n",
      "components and operated reliably using well-established protocols and freely available software.\n",
      "['smFRET',\n",
      " 'single-molecule fluorescence resonance energy transfer',\n",
      " 'single-molecule techniques',\n",
      " 'immobilized molecules',\n",
      " 'single-molecule reaction trajectories',\n",
      " 'biologist',\n",
      " 'experimental design',\n",
      " 'sample preparation',\n",
      " 'single-molecule detection',\n",
      " 'data analysis']\n",
      "Elapsed time: 10.98 seconds\n",
      "-------------------\n",
      "18511919\n",
      "Laminar flow cells for single-molecule studies of DNA-protein interactions.\n",
      "Microfluidic flow cells are used in single-molecule experiments, enabling measurements to be made\n",
      "with high spatial and temporal resolution. We discuss the fundamental processes affecting flow cell\n",
      "operation and describe the flow cells in use at present for studying the interaction of optically\n",
      "trapped or mechanically isolated, single DNA molecules with proteins. To assist the experimentalist\n",
      "in flow cell selection, we review the construction techniques and materials used to fabricate both\n",
      "single- and multiple-channel flow cells and the advantages of each design for different experiments.\n",
      "['microfluidic flow cells',\n",
      " 'optically trapped DNA molecules',\n",
      " 'mechanically isolated DNA molecules',\n",
      " 'single-channel flow cells',\n",
      " 'multiple-channel flow cells']\n",
      "Elapsed time: 8.07 seconds\n",
      "-------------------\n",
      "18516047\n",
      "In vitro whole-organ imaging: 4D quantification of growing mouse limb buds.\n",
      "Quantitative mapping of the normal tissue dynamics of an entire developing mammalian organ has not\n",
      "been achieved so far but is essential to understand developmental processes and to provide\n",
      "quantitative data for computational modeling. We developed a four-dimensional (4D) imaging technique\n",
      "that can be used to quantitatively image tissue movements and dynamic GFP expression domains in a\n",
      "growing transgenic mouse limb by time-lapse optical projection tomography (OPT).\n",
      "['optical projection tomography', 'time-lapse imaging', '4D imaging']\n",
      "Elapsed time: 6.91 seconds\n",
      "-------------------\n",
      "18536722\n",
      "Lifeact: a versatile marker to visualize F-actin.\n",
      "Live imaging of the actin cytoskeleton is crucial for the study of many fundamental biological\n",
      "processes, but current approaches to visualize actin have several limitations. Here we describe\n",
      "Lifeact, a 17-amino-acid peptide, which stained filamentous actin (F-actin) structures in eukaryotic\n",
      "cells and tissues. Lifeact did not interfere with actin dynamics in vitro and in vivo and in its\n",
      "chemically modified peptide form allowed visualization of actin dynamics in nontransfectable cells.\n",
      "['live imaging', 'visualize actin', 'filamentous actin (F-actin)']\n",
      "Elapsed time: 7.36 seconds\n",
      "-------------------\n",
      "18568029\n",
      "Automated on-chip rapid microscopy, phenotyping and sorting of C. elegans.\n",
      "Microscopy, phenotyping and visual screens are frequently applied to model organisms in combination\n",
      "with genetics. Although widely used, these techniques for multicellular organisms have mostly\n",
      "remained manual and low-throughput. Here we report the complete automation of sample handling, high-\n",
      "resolution microscopy, phenotyping and sorting of Caenorhabditis elegans. The engineered\n",
      "microfluidic system, coupled with customized software, has enabled high-throughput, high-resolution\n",
      "microscopy and sorting with no human intervention and may be combined with any microscopy setup. The\n",
      "microchip is capable of robust local temperature control, self-regulated sample-loading and\n",
      "automatic sample-positioning, while the integrated software performs imaging and classification of\n",
      "worms based on morphological and intensity features. We demonstrate the ability to perform sensitive\n",
      "and quantitative screens based on cellular and subcellular phenotypes with over 95% accuracy per\n",
      "round and a rate of several hundred worms per hour. Screening time can be reduced by orders of\n",
      "magnitude; moreover, screening is completely automated.\n",
      "['microscopy',\n",
      " 'phenotyping',\n",
      " 'sorting',\n",
      " 'high-resolution microscopy',\n",
      " 'automated microscopy']\n",
      "Elapsed time: 10.85 seconds\n",
      "-------------------\n",
      "18587405\n",
      "Automated analysis of embryonic gene expression with cellular resolution in C. elegans.\n",
      "We describe a system that permits the automated analysis of reporter gene expression in\n",
      "Caenorhabditis elegans with cellular resolution continuously during embryogenesis. We demonstrate\n",
      "its utility by defining the expression patterns of reporters for several embryonically expressed\n",
      "transcription factors. The invariant cell lineage permits the automated alignment of multiple\n",
      "expression profiles, allowing direct comparison of the expression of different genes' reporters. We\n",
      "also used this system to monitor perturbations to normal development involving changes both in cell-\n",
      "division timing and in cell fate. Systematic application of this system could reveal the gene\n",
      "activity of each cell throughout development.\n",
      "['expression profiles',\n",
      " 'cellular resolution',\n",
      " 'automated analysis',\n",
      " 'reporter gene expression',\n",
      " 'invariant cell lineage']\n",
      "Elapsed time: 8.15 seconds\n",
      "-------------------\n",
      "18641657\n",
      "Robust single-particle tracking in live-cell time-lapse sequences.\n",
      "Single-particle tracking (SPT) is often the rate-limiting step in live-cell imaging studies of\n",
      "subcellular dynamics. Here we present a tracking algorithm that addresses the principal challenges\n",
      "of SPT, namely high particle density, particle motion heterogeneity, temporary particle\n",
      "disappearance, and particle merging and splitting. The algorithm first links particles between\n",
      "consecutive frames and then links the resulting track segments into complete trajectories. Both\n",
      "steps are formulated as global combinatorial optimization problems whose solution identifies the\n",
      "overall most likely set of particle trajectories throughout a movie. Using this approach, we show\n",
      "that the GTPase dynamin differentially affects the kinetics of long- and short-lived endocytic\n",
      "structures and that the motion of CD36 receptors along cytoskeleton-mediated linear tracks increases\n",
      "their aggregation probability. Both applications indicate the requirement for robust and complete\n",
      "tracking of dense particle fields to dissect the mechanisms of receptor organization at the level of\n",
      "the plasma membrane.\n",
      "['single-particle tracking',\n",
      " 'live-cell time-lapse sequences',\n",
      " 'global combinatorial optimization problems']\n",
      "Elapsed time: 9.57 seconds\n",
      "-------------------\n",
      "18641658\n",
      "Colloid-guided assembly of oriented 3D neuronal networks.\n",
      "A central challenge in neuroscience is to understand the formation and function of three-dimensional\n",
      "(3D) neuronal networks. In vitro studies have been mainly limited to measurements of small numbers\n",
      "of neurons connected in two dimensions. Here we demonstrate the use of colloids as moveable supports\n",
      "for neuronal growth, maturation, transfection and manipulation, where the colloids serve as guides\n",
      "for the assembly of controlled 3D, millimeter-sized neuronal networks. Process growth can be guided\n",
      "into layered connectivity with a density similar to what is found in vivo. The colloidal\n",
      "superstructures are optically transparent, enabling remote stimulation and recording of neuronal\n",
      "activity using layer-specific expression of light-activated channels and indicator dyes. The modular\n",
      "approach toward in vitro circuit construction provides a stepping stone for applications ranging\n",
      "from basic neuroscience to neuron-based screening of targeted drugs.\n",
      "['optical imaging', 'light-activated channels', 'indicator dyes']\n",
      "Elapsed time: 10.00 seconds\n",
      "-------------------\n",
      "18756197\n",
      "Quantum dots versus organic dyes as fluorescent labels.\n",
      "Suitable labels are at the core of Luminescence and fluorescence imaging and sensing. One of the\n",
      "most exciting, yet also controversial, advances in label technology is the emerging development of\n",
      "quantum dots (QDs)--inorganic nanocrystals with unique optical and chemical properties but\n",
      "complicated surface chemistry--as in vitro and in vivo fluorophores. Here we compare and evaluate\n",
      "the differences in physicochemical properties of common fluorescent labels, focusing on traditional\n",
      "organic dyes and QDs. Our aim is to provide a better understanding of the advantages and limitations\n",
      "of both classes of chromophores, to facilitate label choice and to address future challenges in the\n",
      "rational design and manipulation of QD labels.\n",
      "['fluorescence imaging', 'luminescence imaging', 'quantum dots', 'organic dyes']\n",
      "Elapsed time: 8.90 seconds\n",
      "-------------------\n",
      "18794860\n",
      "Super-resolution imaging in live Caulobacter crescentus cells using photoswitchable EYFP.\n",
      "The commonly used, monomeric EYFP enabled imaging of intracellular protein structures beyond the\n",
      "optical resolution limit ('super-resolution' imaging) in living cells. By combining photoinduced\n",
      "activation of single EYFP fusions and time-lapse imaging, we obtained sub-40 nm resolution images of\n",
      "the filamentous superstructure of the bacterial actin protein MreB in live Caulobacter crescentus\n",
      "cells. These studies demonstrated that EYFP is a useful emitter for in vivo super-resolution\n",
      "imaging.\n",
      "['super-resolution', 'time-lapse', 'photoswitchable EYFP']\n",
      "Elapsed time: 7.66 seconds\n",
      "-------------------\n",
      "18794862\n",
      "Optogenetic analysis of synaptic function.\n",
      "We introduce optogenetic investigation of neurotransmission (OptIoN) for time-resolved and\n",
      "quantitative assessment of synaptic function via behavioral and electrophysiological analyses. We\n",
      "photo-triggered release of acetylcholine or gamma-aminobutyric acid at Caenorhabditis elegans\n",
      "neuromuscular junctions using targeted expression of Chlamydomonas reinhardtii Channelrhodopsin-2.\n",
      "In intact Channelrhodopsin-2 transgenic worms, photostimulation instantly induced body elongation\n",
      "(for gamma-aminobutyric acid) or contraction (for acetylcholine), which we analyzed acutely, or\n",
      "during sustained activation with automated image analysis, to assess synaptic efficacy. In dissected\n",
      "worms, photostimulation evoked neurotransmitter-specific postsynaptic currents that could be\n",
      "triggered repeatedly and at various frequencies. Light-evoked behaviors and postsynaptic currents\n",
      "were significantly (P <or= 0.05) altered in mutants with pre- or postsynaptic defects, although the\n",
      "behavioral phenotypes did not unambiguously report on synaptic function in all cases tested. OptIoN\n",
      "facilitates the analysis of neurotransmission with high temporal precision, in a neurotransmitter-\n",
      "selective manner, possibly allowing future investigation of synaptic plasticity in C. elegans.\n",
      "Invalid json output: [{\"name\": \"photostimulation\"}]\n",
      "[{\"name\": \"image analysis\"}]\n",
      "[{\"name\": \"electrophysiological analyses\"}] derived from Sure! Here is the list of imaging techniques or technologies described in the abstract:\n",
      "\n",
      "[{\"name\": \"photostimulation\"}]\n",
      "[{\"name\": \"image analysis\"}]\n",
      "[{\"name\": \"electrophysiological analyses\"}]\n",
      "\n",
      "Please note that there are only three imaging techniques or technologies described in the abstract, so the list is limited to these three items.\n",
      "18806790\n",
      "Tracking the structural dynamics of proteins in solution using time-resolved wide-angle X-ray\n",
      "scattering.\n",
      "We demonstrate tracking of protein structural changes with time-resolved wide-angle X-ray scattering\n",
      "(TR-WAXS) with nanosecond time resolution. We investigated the tertiary and quaternary\n",
      "conformational changes of human hemoglobin under nearly physiological conditions triggered by laser-\n",
      "induced ligand photolysis. We also report data on optically induced tertiary relaxations of\n",
      "myoglobin and refolding of cytochrome c to illustrate the wide applicability of the technique. By\n",
      "providing insights into the structural dynamics of proteins functioning in their natural\n",
      "environment, TR-WAXS complements and extends results obtained with time-resolved optical\n",
      "spectroscopy and X-ray crystallography.\n",
      "['TR-WAXS', 'X-ray crystallography', 'time-resolved optical spectroscopy']\n",
      "Elapsed time: 8.73 seconds\n",
      "-------------------\n",
      "18806792\n",
      "Imaging individual mRNA molecules using multiple singly labeled probes.\n",
      "We describe a method for imaging individual mRNA molecules in fixed cells by probing each mRNA\n",
      "species with 48 or more short, singly labeled oligonucleotide probes. This makes each mRNA molecule\n",
      "visible as a computationally identifiable fluorescent spot by fluorescence microscopy. We\n",
      "demonstrate simultaneous detection of three mRNA species in single cells and mRNA detection in\n",
      "yeast, nematodes, fruit fly wing discs, and mammalian cell lines and neurons.\n",
      "['fluorescence microscopy', 'oligonucleotide probes', 'singly labeled probes']\n",
      "Elapsed time: 7.44 seconds\n",
      "-------------------\n",
      "18836457\n",
      "High-speed, miniaturized fluorescence microscopy in freely moving mice.\n",
      "A central goal in biomedicine is to explain organismic behavior in terms of causal cellular\n",
      "processes. However, concurrent observation of mammalian behavior and underlying cellular dynamics\n",
      "has been a longstanding challenge. We describe a miniaturized (1.1 g mass) epifluorescence\n",
      "microscope for cellular-level brain imaging in freely moving mice, and its application to imaging\n",
      "microcirculation and neuronal Ca(2+) dynamics.\n",
      "['epifluorescence microscopy', 'miniaturized microscope']\n",
      "Elapsed time: 7.08 seconds\n",
      "-------------------\n",
      "18978785\n",
      "High-resolution statistical mapping reveals gene territories in live yeast.\n",
      "The nonrandom positioning of genes inside eukaryotic cell nuclei is implicated in central nuclear\n",
      "functions. However, the spatial organization of the genome remains largely uncharted, owing to\n",
      "limited resolution of optical microscopy, paucity of nuclear landmarks and moderate cell sampling.\n",
      "We developed a computational imaging approach that creates high-resolution probabilistic maps of\n",
      "subnuclear domains occupied by individual loci in budding yeast through automated analysis of\n",
      "thousands of living cells. After validation, we applied the technique to genes involved in galactose\n",
      "metabolism and ribosome biogenesis. We found that genomic loci are confined to 'gene territories'\n",
      "much smaller than the nucleus, which can be remodeled during transcriptional activation, and that\n",
      "the nucleolus is an important landmark for gene positioning. The technique can be used to visualize\n",
      "and quantify territory positions relative to each other and to nuclear landmarks, and should advance\n",
      "studies of nuclear architecture and function.\n",
      "['computational imaging approach',\n",
      " 'automated analysis',\n",
      " 'high-resolution probabilistic maps',\n",
      " 'visualize and quantify territory positions']\n",
      "Elapsed time: 9.65 seconds\n",
      "-------------------\n",
      "18997781\n",
      "Intravital imaging of metastatic behavior through a mammary imaging window.\n",
      "We report a technique to evaluate the same tumor microenvironment over multiple intravital imaging\n",
      "sessions in living mice. We optically marked individual tumor cells expressing photoswitchable\n",
      "proteins in an orthotopic mammary carcinoma and followed them for extended periods through a mammary\n",
      "imaging window. We found that two distinct microenvironments in the same orthotopic mammary tumor\n",
      "affected differently the invasion and intravasation of tumor cells.\n",
      "['intravital imaging', 'optical marking', 'photoswitchable proteins']\n",
      "Elapsed time: 6.90 seconds\n",
      "-------------------\n",
      "18997782\n",
      "Micropatterning for quantitative analysis of protein-protein interactions in living cells.\n",
      "We present a method to identify and characterize interactions between a fluorophore-labeled protein\n",
      "('prey') and a membrane protein ('bait') in live mammalian cells. Cells are plated on micropatterned\n",
      "surfaces functionalized with antibodies to the bait extracellular domain. Bait-prey interactions are\n",
      "assayed through the redistribution of the fluorescent prey. We used the method to characterize the\n",
      "interaction between human CD4, the major co-receptor in T-cell activation, and human Lck, the\n",
      "protein tyrosine kinase essential for early T-cell signaling. We measured equilibrium associations\n",
      "by quantifying Lck redistribution to CD4 micropatterns and studied interaction dynamics by\n",
      "photobleaching experiments and single-molecule imaging. In addition to the known zinc clasp\n",
      "structure, the Lck membrane anchor in particular had a major impact on the Lck-CD4 interaction,\n",
      "mediating direct binding and further stabilizing the interaction of other Lck domains. In total,\n",
      "membrane anchorage increased the interaction lifetime by two orders of magnitude.\n",
      "['single-molecule imaging',\n",
      " 'photobleaching experiments',\n",
      " 'micropatterned surfaces']\n",
      "Elapsed time: 11.05 seconds\n",
      "-------------------\n",
      "19011626\n",
      "Nanoscale imaging of molecular positions and anisotropies.\n",
      "Knowledge of the orientation of molecules within biological structures is crucial to understanding\n",
      "the mechanisms of cell function. We present a method to image simultaneously the positions and\n",
      "fluorescence anisotropies of large numbers of single molecules with nanometer lateral resolution\n",
      "within a sample. Based on a simple modification of fluorescence photoactivation localization\n",
      "microscopy (FPALM), polarization (P)-FPALM does not compromise speed or sensitivity. We show results\n",
      "for mouse fibroblasts expressing Dendra2-actin or Dendra2-hemagglutinin.\n",
      "['FPALM', 'polarization (P)-FPALM']\n",
      "Elapsed time: 7.36 seconds\n",
      "-------------------\n",
      "19029906\n",
      "Whole-cell 3D STORM reveals interactions between cellular structures with nanometer-scale\n",
      "resolution.\n",
      "The ability to directly visualize nanoscopic cellular structures and their spatial relationship in\n",
      "all three dimensions will greatly enhance our understanding of molecular processes in cells. Here we\n",
      "demonstrated multicolor three-dimensional (3D) stochastic optical reconstruction microscopy (STORM)\n",
      "as a tool to quantitatively probe cellular structures and their interactions. To facilitate STORM\n",
      "imaging, we generated photoswitchable probes in several distinct colors by covalently linking a\n",
      "photoswitchable cyanine reporter and an activator molecule to assist bioconjugation. We performed 3D\n",
      "localization in conjunction with focal plane scanning and correction for refractive index mismatch\n",
      "to obtain whole-cell images with a spatial resolution of 20-30 nm and 60-70 nm in the lateral and\n",
      "axial dimensions, respectively. Using this approach, we imaged the entire mitochondrial network in\n",
      "fixed monkey kidney BS-C-1 cells, and studied the spatial relationship between mitochondria and\n",
      "microtubules. The 3D STORM images resolved mitochondrial morphologies as well as mitochondria-\n",
      "microtubule contacts that were obscured in conventional fluorescence images.\n",
      "Could not find json-formatted data in: Sure! Here is the list of imaging techniques described in the abstract:\n",
      "\n",
      "• 3D STORM\n",
      "• multicolor 3D STORM\n",
      "• focal plane scanning\n",
      "• refractive index mismatch correction\n",
      "\n",
      "Note that there are no other imaging techniques mentioned in the abstract, so this list only contains the three techniques mentioned.\n",
      "19116611\n",
      "Microscopy and its focal switch.\n",
      "Until not very long ago, it was widely accepted that lens-based (far-field) optical microscopes\n",
      "cannot visualize details much finer than about half the wavelength of light. The advent of viable\n",
      "physical concepts for overcoming the limiting role of diffraction in the early 1990s set off a quest\n",
      "that has led to readily applicable and widely accessible fluorescence microscopes with nanoscale\n",
      "spatial resolution. Here I discuss the principles of these methods together with their differences\n",
      "in implementation and operation. Finally, I outline potential developments.\n",
      "['fluorescence microscopes',\n",
      " 'lens-based (far-field) optical microscopes',\n",
      " 'nanoscale spatial resolution']\n",
      "Elapsed time: 7.48 seconds\n",
      "-------------------\n",
      "19122668\n",
      "Microfluidic control of cell pairing and fusion.\n",
      "Cell fusion has been used for many different purposes, including generation of hybridomas and\n",
      "reprogramming of somatic cells. The fusion step is the key event in initiation of these procedures.\n",
      "Standard fusion techniques, however, provide poor and random cell contact, leading to low yields. We\n",
      "present here a microfluidic device to trap and properly pair thousands of cells. Using this device,\n",
      "we paired different cell types, including fibroblasts, mouse embryonic stem cells and myeloma cells,\n",
      "achieving pairing efficiencies up to 70%. The device is compatible with both chemical and electrical\n",
      "fusion protocols. We observed that electrical fusion was more efficient than chemical fusion, with\n",
      "membrane reorganization efficiencies of up to 89%. We achieved greater than 50% properly paired and\n",
      "fused cells over the entire device, fivefold greater than with a commercial electrofusion chamber\n",
      "and observed reprogramming in hybrids between mouse embryonic stem cells and mouse embryonic\n",
      "fibroblasts.\n",
      "['microfluidic device',\n",
      " 'chemical fusion protocols',\n",
      " 'electrical fusion protocols',\n",
      " 'membrane reorganization']\n",
      "Elapsed time: 10.54 seconds\n",
      "-------------------\n",
      "19122669\n",
      "A genetically encoded fluorescent reporter of ATP:ADP ratio.\n",
      "We constructed a fluorescent sensor of adenylate nucleotides by combining a circularly permuted\n",
      "variant of GFP with a bacterial regulatory protein, GlnK1, from Methanococcus jannaschii. The\n",
      "sensor's affinity for Mg-ATP was <100 nM, as seen for other members of the bacterial PII regulator\n",
      "family, a surprisingly high affinity given that normal intracellular ATP concentration is in the\n",
      "millimolar range. ADP bound the same site of the sensor as Mg-ATP, competing with it, but produced a\n",
      "smaller change in fluorescence. At physiological ATP and ADP concentrations, the binding site is\n",
      "saturated, but competition between the two substrates causes the sensor to behave as a nearly ideal\n",
      "reporter of the ATP:ADP concentration ratio. This principle for sensing the ratio of two analytes by\n",
      "competition at a high-affinity site probably underlies the normal functioning of PII regulatory\n",
      "proteins. The engineered sensor, Perceval, can be used to monitor the ATP:ADP ratio during live-cell\n",
      "imaging.\n",
      "['live-cell imaging']\n",
      "Elapsed time: 10.66 seconds\n",
      "-------------------\n",
      "19151719\n",
      "Probing the mechanical architecture of the vertebrate meiotic spindle.\n",
      "Accurate chromosome segregation during meiosis depends on the assembly of a microtubule-based\n",
      "spindle of proper shape and size. Current models for spindle-size control focus on reaction\n",
      "diffusion-based chemical regulation and balance in activities of motor proteins. Although several\n",
      "molecular perturbations have been used to test these models, controlled mechanical perturbations\n",
      "have not been possible. Here we report a piezoresistive dual cantilever-based system to test models\n",
      "for spindle-size control and examine the mechanical features, such as deformability and stiffness,\n",
      "of the vertebrate meiotic spindle. We found that meiotic spindles prepared in Xenopus laevis egg\n",
      "extracts were viscoelastic and recovered their original shape in response to small compression.\n",
      "Larger compression resulted in plastic deformation, but the spindle adapted to this change,\n",
      "establishing a stable mechanical architecture at different sizes. The technique we describe here may\n",
      "also be useful for examining the micromechanics of other cellular organelles.\n",
      "['piezoresistive dual cantilever-based system',\n",
      " 'compression',\n",
      " 'deformability',\n",
      " 'stiffness']\n",
      "Elapsed time: 10.10 seconds\n",
      "-------------------\n",
      "19151720\n",
      "In vivo imaging of extracellular matrix remodeling by tumor-associated fibroblasts.\n",
      "Here we integrated multiphoton laser scanning microscopy and the registration of second harmonic\n",
      "generation images of collagen fibers to overcome difficulties in tracking stromal cell-matrix\n",
      "interactions for several days in live mice. We show that the matrix-modifying hormone relaxin\n",
      "increased tumor-associated fibroblast (TAF) interaction with collagen fibers by stimulating\n",
      "beta1-integrin activity, which is necessary for fiber remodeling by matrix metalloproteinases.\n",
      "['multiphoton laser scanning microscopy',\n",
      " 'second harmonic generation imaging',\n",
      " 'beta1-integrin activity']\n",
      "Elapsed time: 7.70 seconds\n",
      "-------------------\n",
      "19160515\n",
      "A genetically encoded calcium indicator for chronic in vivo two-photon imaging.\n",
      "Neurons in the nervous system can change their functional properties over time. At present, there\n",
      "are no techniques that allow reliable monitoring of changes within identified neurons over repeated\n",
      "experimental sessions. We increased the signal strength of troponin C-based calcium biosensors in\n",
      "the low-calcium regime by mutagenesis and domain rearrangement within the troponin C calcium binding\n",
      "moiety to generate the indicator TN-XXL. Using in vivo two-photon ratiometric imaging, we show that\n",
      "TN-XXL exhibits enhanced fluorescence changes in neurons of flies and mice. TN-XXL could be used to\n",
      "obtain tuning curves of orientation-selective neurons in mouse visual cortex measured repeatedly\n",
      "over days and weeks. Thus, the genetically encoded calcium indicator TN-XXL allows repeated imaging\n",
      "of response properties from individual, identified neurons in vivo, which will be crucial for\n",
      "gaining new insights into cellular mechanisms of plasticity, regeneration and disease.\n",
      "['in vivo two-photon imaging', 'ratiometric imaging']\n",
      "Elapsed time: 10.46 seconds\n",
      "-------------------\n",
      "19160517\n",
      "Holographic photolysis of caged neurotransmitters.\n",
      "Stimulation of light-sensitive chemical probes has become a powerful tool for the study of dynamic\n",
      "signaLing processes in living tissue. Classically, this approach has been constrained by limitations\n",
      "of lens-based and point-scanning illumination systems. Here we describe a microscope configuration\n",
      "that incorporates a nematic liquid-crystal spatial light modulator to generate holographic patterns\n",
      "of illumination. This microscope can produce illumination spots of variable size and number, and in\n",
      "patterns shaped to precisely match user-defined elements in a specimen. Using holographic\n",
      "illumination to photolyze caged glutamate in brain slices, we show that shaped excitation on\n",
      "segments of neuronal dendrites and simultaneous, multispot excitation of different dendrites enables\n",
      "precise spatial and rapid temporal control of glutamate receptor activation. By allowing the\n",
      "excitation volume shape to be tailored precisely, the holographic microscope provides an extremely\n",
      "flexible method for activation of various photosensitive proteins and small molecules.\n",
      "['holographic photolysis',\n",
      " 'lens-based illumination systems',\n",
      " 'point-scanning illumination systems',\n",
      " 'nematic liquid-crystal spatial light modulator',\n",
      " 'holographic illumination']\n",
      "Elapsed time: 11.53 seconds\n",
      "-------------------\n",
      "19160519\n",
      "Imaging dynamic cell-cell junctional coupling in vivo using Trojan-LAMP.\n",
      "To study the physiological regulation and function of cell-cell gap junction communication in vivo,\n",
      "we developed a bioconjugate of caged dye, named dextran-CANPE-HCC, for imaging cell coupling in\n",
      "small model organisms. In vitro, the compound was photolyzed efficiently with robust fluorescence\n",
      "enhancement. Dextran-CANPE-HCC delivered into Caenorhabditis elegans oocytes was retained in cells\n",
      "throughout development. Using local uncaging, we photolyzed dextran-CANPE-HCC to release the small\n",
      "HCC dye and imaged the dynamics of intercellular dye transfer through gap junction channels, a\n",
      "technique we named Trojan-local activation of molecular fluorescent probes (LAMP). Early during\n",
      "embryonic development, the pattern of cell coupling undergoes dramatic remodeling and imaging\n",
      "revealed that the germ cell precursors, P2, P3 and P4, were isolated from the somatic cell\n",
      "communication compartment. As dextran-CANPE-HCC is chemically and metabolically stable, Labeled\n",
      "worms showed very bright signal upon photoactivation after hatching, which allowed us to examine\n",
      "cell coupling in living worms noninvasively.\n",
      "['Trojan-LAMP', 'local uncaging', 'molecular fluorescent probes']\n",
      "Elapsed time: 12.17 seconds\n",
      "-------------------\n",
      "19169259\n",
      "Photoactivatable mCherry for high-resolution two-color fluorescence microscopy.\n",
      "The reliance of modern microscopy techniques on photoactivatable fluorescent proteins prompted\n",
      "development of mCherry variants that are initially dark but become red fluorescent after violet-\n",
      "light irradiation. Using ensemble and single-molecule characteristics as selection criteria, we\n",
      "developed PAmCherry1 with excitation/emission maxima at 564/595 nm. Compared to other monomeric red\n",
      "photoactivatable proteins, it has faster maturation, better pH stability, faster photoactivation,\n",
      "higher photoactivation contrast and better photostability. Lack of green fluorescence and single-\n",
      "molecule behavior make monomeric PAmCherry1 a preferred tag for two-color diffraction-limited\n",
      "photoactivation imaging and for super-resolution techniques such as one- and two-color\n",
      "photoactivated localization microscopy (PALM). We performed PALM imaging using PAmCherry1-tagged\n",
      "transferrin receptor expressed alone or with photoactivatable GFP-tagged clathrin light chain. Pair\n",
      "correlation and cluster analyses of the resulting PALM images identified < or =200 nm clusters of\n",
      "transferrin receptor and clathrin light chain at < or =25 nm resolution and confirmed the utility of\n",
      "PAmCherry1 as an intracellular probe.\n",
      "{'imaging techniques': ['two-color fluorescence microscopy',\n",
      "                        'photoactivated localization microscopy (PALM)',\n",
      "                        'one- and two-color photoactivated localization '\n",
      "                        'microscopy (PALM)',\n",
      "                        'diffraction-limited photoactivation imaging']}\n",
      "Elapsed time: 16.52 seconds\n",
      "-------------------\n",
      "19169260\n",
      "A bright and photostable photoconvertible fluorescent protein.\n",
      "Photoconvertible fluorescent proteins are potential tools for investigating dynamic processes in\n",
      "living cells and for emerging super-resolution microscopy techniques. Unfortunately, most probes in\n",
      "this class are hampered by oligomerization, small photon budgets or poor photostability. Here we\n",
      "report an EosFP variant that functions well in a broad range of protein fusions for dynamic\n",
      "investigations, exhibits high photostability and preserves the approximately 10-nm localization\n",
      "precision of its parent.\n",
      "['super-resolution microscopy', 'dynamic investigations', 'photostability']\n",
      "Elapsed time: 7.45 seconds\n",
      "-------------------\n",
      "19198595\n",
      "Direct determination of haplotypes from single DNA molecules.\n",
      "Determining the long-range haplotypes in a diploid individual is a major technical challenge. Here\n",
      "we report a method of molecular haplotyping by directly imaging multiple polymorphic sites on\n",
      "individual human DNA molecules simultaneously. We demonstrate the utility of this technology by\n",
      "accurately determining the haplotypes consisting of up to 16 single-nucleotide polymorphisms in\n",
      "genomic regions up to 50 kilobases.\n",
      "['direct imaging',\n",
      " 'molecular haplotyping',\n",
      " 'single-nucleotide polymorphisms',\n",
      " 'long-range haplotypes']\n",
      "Elapsed time: 7.43 seconds\n",
      "-------------------\n",
      "19234451\n",
      "Analysis of receptor oligomerization by FRAP microscopy.\n",
      "Here we describe an approach to investigate di- or oligomerization of transmembrane receptors in\n",
      "living cells with fluorescence recovery after photobleaching (FRAP). We immobilized a defined\n",
      "fraction of receptors with antibodies and then measured lateral mobility of the nonimmobilized\n",
      "fraction by FRAP. We validated this approach with CD86 and CD28 as monomeric and dimeric reference\n",
      "proteins, respectively. Di- or oligomerization of G protein-coupled receptors is strongly debated.\n",
      "We studied human beta-adrenergic receptors as prototypical G protein-coupled receptors and found\n",
      "that beta(1)-AR shows transient interactions whereas beta(2)-AR can form stable oligomers. We\n",
      "propose that this FRAP method can be widely applied to study di- or oligomerization of cell-surface\n",
      "proteins.\n",
      "['FRAP microscopy', 'fluorescence recovery after photobleaching']\n",
      "Elapsed time: 9.05 seconds\n",
      "-------------------\n",
      "19252505\n",
      "Nanoscale live-cell imaging using hopping probe ion conductance microscopy.\n",
      "We describe hopping mode scanning ion conductance microscopy that allows noncontact imaging of the\n",
      "complex three-dimensional surfaces of live cells with resolution better than 20 nm. We tested the\n",
      "effectiveness of this technique by imaging networks of cultured rat hippocampal neurons and\n",
      "mechanosensory stereocilia of mouse cochlear hair cells. The technique allowed examination of\n",
      "nanoscale phenomena on the surface of live cells under physiological conditions.\n",
      "['hopping probe ion conductance microscopy',\n",
      " 'live-cell imaging',\n",
      " 'noncontact imaging']\n",
      "Elapsed time: 7.23 seconds\n",
      "-------------------\n",
      "19349978\n",
      "An ultramarine fluorescent protein with increased photostability and pH insensitivity.\n",
      "We report a pH-insensitive and photostable ultramarine fluorescent protein, Sirius, with an emission\n",
      "peak at 424 nm, the shortest emission wavelength among fluorescent proteins reported to date. The\n",
      "pH-insensitivity of Sirius allowed prolonged visualization of biological events in an acidic\n",
      "environment. Two fluorescence resonance energy transfer (FRET) pairs, Sirius-mseCFP and Sapphire-\n",
      "DsRed, allowed dual-FRET imaging with single-wavelength excitation, enabling detection of Ca(2+)\n",
      "concentration and caspase-3 activation in the same apoptotic cells.\n",
      "['fluorescent protein',\n",
      " 'prolonged visualization',\n",
      " 'acidic environment',\n",
      " 'fluorescence resonance energy transfer (FRET)',\n",
      " 'dual-FRET imaging',\n",
      " 'single-wavelength excitation']\n",
      "Elapsed time: 9.31 seconds\n",
      "-------------------\n",
      "19349979\n",
      "Single molecule-sensitive probes for imaging RNA in live cells.\n",
      "To visualize native or non-engineered RNA in live cells with single-molecule sensitivity, we\n",
      "developed multiply labeled tetravalent RNA imaging probes (MTRIPs). When delivered with streptolysin\n",
      "O into living human epithelial cancer cells and primary chicken fibroblasts, MTRIPs allowed the\n",
      "accurate imaging of native mRNAs and a non-engineered viral RNA, of RNA co-localization with known\n",
      "RNA-binding proteins, and of RNA dynamics and interactions with stress granules.\n",
      "['single molecule sensitivity',\n",
      " 'live cells',\n",
      " 'native RNA imaging',\n",
      " 'non-engineered RNA imaging',\n",
      " 'RNA co-localization imaging',\n",
      " 'stress granule imaging']\n",
      "Elapsed time: 8.47 seconds\n",
      "-------------------\n",
      "19363493\n",
      "Combined atomic force microscopy and side-view optical imaging for mechanical studies of cells.\n",
      "The mechanical rigidity of cells and adhesion forces between cells are important in various\n",
      "biological processes, including cell differentiation, proliferation and tissue organization. Atomic\n",
      "force microscopy has emerged as a powerful tool to quantify the mechanical properties of individual\n",
      "cells and adhesion forces between cells. Here we demonstrate an instrument that combines atomic\n",
      "force microscopy with a side-view fluorescent imaging path that enables direct imaging of cellular\n",
      "deformation and cytoskeletal rearrangements along the axis of loading. With this instrument, we\n",
      "directly observed cell shape under mechanical load, correlated changes in shape with force-induced\n",
      "ruptures and imaged formation of membrane tethers during cell-cell adhesion measurements.\n",
      "Additionally, we observed cytoskeletal reorganization and stress-fiber formation while measuring the\n",
      "contractile force of an individual cell. This instrument can be a useful tool for understanding the\n",
      "role of mechanics in biological processes.\n",
      "['atomic force microscopy', 'side-view optical imaging']\n",
      "Elapsed time: 9.21 seconds\n",
      "-------------------\n",
      "19404252\n",
      "Imaging intracellular RNA distribution and dynamics in living cells.\n",
      "Powerful methods now allow the imaging of specific mRNAs in living cells. These methods enlist\n",
      "fluorescent proteins to illuminate mRNAs, use labeled oligonucleotide probes and exploit aptamers\n",
      "that render organic dyes fluorescent. The intracellular dynamics of mRNA synthesis, transport and\n",
      "localization can be analyzed at higher temporal resolution with these methods than has been possible\n",
      "with traditional fixed-cell or biochemical approaches. These methods have also been adopted to\n",
      "visualize and track single mRNA molecules in real time. This review explores the promises and\n",
      "limitations of these methods.\n",
      "['fluorescent proteins', 'labeled oligonucleotide probes', 'aptamers']\n",
      "Elapsed time: 7.96 seconds\n",
      "-------------------\n",
      "19404253\n",
      "Super-resolution video microscopy of live cells by structured illumination.\n",
      "Structured-illumination microscopy can double the resolution of the widefield fluorescence\n",
      "microscope but has previously been too slow for dynamic live imaging. Here we demonstrate a high-\n",
      "speed structured-illumination microscope that is capable of 100-nm resolution at frame rates up to\n",
      "11 Hz for several hundred time points. We demonstrate the microscope by video imaging of tubulin and\n",
      "kinesin dynamics in living Drosophila melanogaster S2 cells in the total internal reflection mode.\n",
      "['structured illumination microscopy',\n",
      " 'widefield fluorescence microscope',\n",
      " 'high-speed structured-illumination microscope']\n",
      "Elapsed time: 7.35 seconds\n",
      "-------------------\n",
      "19412169\n",
      "High-throughput ethomics in large groups of Drosophila.\n",
      "We present a camera-based method for automatically quantifying the individual and social behaviors\n",
      "of fruit flies, Drosophila melanogaster, interacting in a planar arena. Our system includes machine-\n",
      "vision algorithms that accurately track many individuals without swapping identities and\n",
      "classification algorithms that detect behaviors. The data may be represented as an ethogram that\n",
      "plots the time course of behaviors exhibited by each fly or as a vector that concisely captures the\n",
      "statistical properties of all behaviors displayed in a given period. We found that behavioral\n",
      "differences between individuals were consistent over time and were sufficient to accurately predict\n",
      "gender and genotype. In addition, we found that the relative positions of flies during social\n",
      "interactions vary according to gender, genotype and social environment. We expect that our software,\n",
      "which permits high-throughput screening, will complement existing molecular methods available in\n",
      "Drosophila, facilitating new investigations into the genetic and cellular basis of behavior.\n",
      "['machine-vision algorithms', 'classification algorithms']\n",
      "Elapsed time: 9.13 seconds\n",
      "-------------------\n",
      "19412170\n",
      "Tissue tectonics: morphogenetic strain rates, cell shape change and intercalation.\n",
      "The dynamic reshaping of tissues during morphogenesis results from a combination of individual cell\n",
      "behaviors and collective cell rearrangements. However, a comprehensive framework to unambiguously\n",
      "measure and link cell behavior to tissue morphogenesis is lacking. Here we introduce such a\n",
      "kinematic framework, bridging cell and tissue behaviors at an intermediate, mesoscopic, level of\n",
      "cell clusters or domains. By measuring domain deformation in terms of the relative motion of cell\n",
      "positions and the evolution of their shapes, we characterized the basic invariant quantities that\n",
      "measure fundamental classes of cell behavior, namely tensorial rates of cell shape change and cell\n",
      "intercalation. In doing so we introduce an explicit definition of cell intercalation as a continuous\n",
      "process. We mapped strain rates spatiotemporally in three models of tissue morphogenesis, gaining\n",
      "insight into morphogenetic mechanisms. Our quantitative approach has broad relevance for the precise\n",
      "characterization and comparison of morphogenetic phenotypes.\n",
      "['mesoscopic imaging',\n",
      " 'cell cluster imaging',\n",
      " 'domain deformation imaging',\n",
      " 'cell shape change imaging',\n",
      " 'intercalation imaging']\n",
      "Elapsed time: 9.95 seconds\n",
      "-------------------\n",
      "19465918\n",
      "A toolkit for high-throughput, cross-species gene engineering in Drosophila.\n",
      "We generated two complementary genomic fosmid libraries for Drosophila melanogaster and Drosophila\n",
      "pseudoobscura that permit seamless modification of large genomic clones by high-throughput\n",
      "recombineering and direct transgenesis. The fosmid transgenes recapitulated endogenous gene\n",
      "expression patterns. These libraries, in combination with recombineering technology, will be useful\n",
      "to rescue mutant phenotypes, allow imaging of gene products in living flies and enable systematic\n",
      "analysis and manipulation of gene activity across species.\n",
      "['imaging of gene products in living flies',\n",
      " 'systematic analysis and manipulation of gene activity across species']\n",
      "Elapsed time: 7.11 seconds\n",
      "-------------------\n",
      "19525958\n",
      "Mapping the structure and conformational movements of proteins with transition metal ion FRET.\n",
      "Visualizing conformational dynamics in proteins has been difficult, and the atomic-scale motions\n",
      "responsible for the behavior of most allosteric proteins are unknown. Here we report that\n",
      "fluorescence resonance energy transfer (FRET) between a small fluorescent dye and a nickel ion bound\n",
      "to a dihistidine motif can be used to monitor small structural rearrangements in proteins. This\n",
      "method provides several key advantages over classical FRET, including the ability to measure the\n",
      "dynamics of close-range interactions, the use of small probes with short linkers, a low orientation\n",
      "dependence, and the ability to add and remove unique tunable acceptors. We used this 'transition\n",
      "metal ion FRET' approach along with X-ray crystallography to determine the structural changes of the\n",
      "gating ring of the mouse hyperpolarization-activated cyclic nucleotide-regulated ion channel HCN2.\n",
      "Our results suggest a general model for the conformational switch in the cyclic nucleotide-binding\n",
      "site of cyclic nucleotide-regulated ion channels.\n",
      "['fluorescence resonance energy transfer (FRET)', 'X-ray crystallography']\n",
      "Elapsed time: 10.86 seconds\n",
      "-------------------\n",
      "19525959\n",
      "In vivo fluorescence imaging with high-resolution microlenses.\n",
      "Micro-optics are increasingly used for minimally invasive in vivo imaging, in miniaturized\n",
      "microscopes and in lab-on-a-chip devices. Owing to optical aberrations and lower numerical\n",
      "apertures, a main class of microlens, gradient refractive index lenses, has not achieved resolution\n",
      "comparable to conventional microscopy. Here we describe high-resolution microlenses, and illustrate\n",
      "two-photon imaging of dendritic spines on hippocampal neurons and dual-color nonlinear optical\n",
      "imaging of neuromuscular junctions in live mice.\n",
      "['two-photon imaging',\n",
      " 'nonlinear optical imaging',\n",
      " 'high-resolution microlenses']\n",
      "Elapsed time: 7.71 seconds\n",
      "-------------------\n",
      "19648917\n",
      "Modular scanning FCS quantifies receptor-ligand interactions in living multicellular organisms.\n",
      "Analysis of receptor-ligand interactions in vivo is key to biology but poses a considerable\n",
      "challenge to quantitative microscopy. Here we combine static-volume, two-focus and dual-color\n",
      "scanning fluorescence correlation spectroscopy to solve this task at cellular resolution in complex\n",
      "biological environments. We quantified the mobility of fibroblast growth factor receptors Fgfr1 and\n",
      "Fgfr4 in cell membranes of living zebrafish embryos and determined their in vivo binding affinities\n",
      "to their ligand Fgf8.\n",
      "['scanning fluorescence correlation spectroscopy',\n",
      " 'static-volume scanning',\n",
      " 'two-focus scanning',\n",
      " 'dual-color scanning']\n",
      "Elapsed time: 8.01 seconds\n",
      "-------------------\n",
      "19684595\n",
      "A 3D digital atlas of C. elegans and its application to single-cell analyses.\n",
      "We built a digital nuclear atlas of the newly hatched, first larval stage (L1) of the wild-type\n",
      "hermaphrodite of Caenorhabditis elegans at single-cell resolution from confocal image stacks of 15\n",
      "individual worms. The atlas quantifies the stereotypy of nuclear locations and provides other\n",
      "statistics on the spatial patterns of the 357 nuclei that could be faithfully segmented and\n",
      "annotated out of the 558 present at this developmental stage. We then developed an automated\n",
      "approach to assign cell names to each nucleus in a three-dimensional image of an L1 worm. We\n",
      "achieved 86% accuracy in identifying the 357 nuclei automatically. This computational method will\n",
      "allow high-throughput single-cell analyses of the post-embryonic worm, such as gene expression\n",
      "analysis, or ablation or stimulation of cells under computer control in a high-throughput functional\n",
      "screen.\n",
      "['confocal image stacks', 'single-cell resolution']\n",
      "Elapsed time: 9.84 seconds\n",
      "-------------------\n",
      "19718032\n",
      "Genetically encoded FRET sensors to monitor intracellular Zn2+ homeostasis.\n",
      "We developed genetically encoded fluorescence resonance energy transfer (FRET)-based sensors that\n",
      "display a large ratiometric change upon Zn(2+) binding, have affinities that span the pico- to\n",
      "nanomolar range and can readily be targeted to subcellular organelles. Using this sensor toolbox we\n",
      "found that cytosolic Zn(2+) was buffered at 0.4 nM in pancreatic beta cells, and we found\n",
      "substantially higher Zn(2+) concentrations in insulin-containing secretory vesicles.\n",
      "['fluorescence resonance energy transfer (FRET)',\n",
      " 'ratiometric change',\n",
      " 'pico- to nanomolar range']\n",
      "Elapsed time: 8.27 seconds\n",
      "-------------------\n",
      "19718033\n",
      "Nanoscale 3D cellular imaging by axial scanning transmission electron tomography.\n",
      "Electron tomography provides three-dimensional structural information about supramolecular\n",
      "assemblies and organelles in a cellular context, but image degradation, caused by scattering of\n",
      "transmitted electrons, limits applicability in specimens thicker than 300 nm. We found that scanning\n",
      "transmission electron tomography of 1,000-nm-thick samples using axial detection provided resolution\n",
      "comparable to that of conventional electron tomography. We demonstrated the method by reconstructing\n",
      "a human erythrocyte infected with the malaria parasite Plasmodium falciparum.\n",
      "['scanning transmission electron tomography',\n",
      " 'axial detection',\n",
      " 'conventional electron tomography']\n",
      "Elapsed time: 7.73 seconds\n",
      "-------------------\n",
      "19767759\n",
      "An approach for extensibly profiling the molecular states of cellular subpopulations.\n",
      "Microscopy often reveals the existence of phenotypically distinct cellular subpopulations. However,\n",
      "additional characterization of observed subpopulations can be limited by the number of biomolecular\n",
      "markers that can be simultaneously monitored. Here we present a computational approach for\n",
      "extensibly profiling cellular subpopulations by freeing one or more imaging channels to monitor\n",
      "additional probes. In our approach, we trained classifiers to re-identify subpopulations accurately\n",
      "based on an enhanced collection of phenotypic features extracted from only a subset of the original\n",
      "markers. Then we constructed subpopulation profiles step-wise from replicate experiments, in which\n",
      "cells were labeled with different but overlapping marker sets. We applied our approach to identify\n",
      "molecular differences among subpopulations and to identify functional groupings of markers, in\n",
      "populations of differentiating mouse preadipocytes, polarizing human neutrophil-like cells and\n",
      "dividing human cancer cells.\n",
      "['microscopy',\n",
      " 'computational approach',\n",
      " 'freeing one or more imaging channels',\n",
      " 'enhanced collection of phenotypic features',\n",
      " 're-identify subpopulations accurately']\n",
      "Elapsed time: 10.19 seconds\n",
      "-------------------\n",
      "19801991\n",
      "High-resolution, long-term characterization of bacterial motility using optical tweezers.\n",
      "We present a single-cell motility assay, which allows the quantification of bacterial swimming in a\n",
      "well-controlled environment, for durations of up to an hour and with a temporal resolution greater\n",
      "than the flagellar rotation rates of approximately 100 Hz. The assay is based on an instrument\n",
      "combining optical tweezers, light and fluorescence microscopy, and a microfluidic chamber. Using\n",
      "this device we characterized the long-term statistics of the run-tumble time series in individual\n",
      "Escherichia coli cells. We also quantified higher-order features of bacterial swimming, such as\n",
      "changes in velocity and reversals of swimming direction.\n",
      "['optical tweezers',\n",
      " 'light microscopy',\n",
      " 'fluorescence microscopy',\n",
      " 'microfluidic chamber']\n",
      "Elapsed time: 8.27 seconds\n",
      "-------------------\n",
      "19838168\n",
      "A chemical platform for improved induction of human iPSCs.\n",
      "The slow kinetics and low efficiency of reprogramming methods to generate human induced pluripotent\n",
      "stem cells (iPSCs) impose major limitations on their utility in biomedical applications. Here we\n",
      "describe a chemical approach that dramatically improves (200-fold) the efficiency of iPSC generation\n",
      "from human fibroblasts, within seven days of treatment. This will provide a basis for developing\n",
      "safer, more efficient, nonviral methods for reprogramming human somatic cells.\n",
      "['chemical approach', 'reprogramming methods']\n",
      "Elapsed time: 6.43 seconds\n",
      "-------------------\n",
      "19838170\n",
      "Visual proteomics of the human pathogen Leptospira interrogans.\n",
      "Systems biology conceptualizes biological systems as dynamic networks of interacting elements,\n",
      "whereby functionally important properties are thought to emerge from the structure of such networks.\n",
      "Owing to the ubiquitous role of complexes of interacting proteins in biological systems, their\n",
      "subunit composition and temporal and spatial arrangement within the cell are of particular interest.\n",
      "'Visual proteomics' attempts to localize individual macromolecular complexes inside of intact cells\n",
      "by template matching reference structures into cryo-electron tomograms. Here we combined\n",
      "quantitative mass spectrometry and cryo-electron tomography to detect, count and localize specific\n",
      "protein complexes in the cytoplasm of the human pathogen Leptospira interrogans. We describe a\n",
      "scoring function for visual proteomics and assess its performance and accuracy under realistic\n",
      "conditions. We discuss current and general limitations of the approach, as well as expected\n",
      "improvements in the future.\n",
      "[{'name': 'cryo-electron tomography'},\n",
      " {'name': 'quantitative mass spectrometry'}]\n",
      "Elapsed time: 10.40 seconds\n",
      "-------------------\n",
      "19881510\n",
      "High-speed nanoscopic tracking of the position and orientation of a single virus.\n",
      "Optical studies have revealed that, after binding, virions move laterally on the plasma membrane,\n",
      "but the complexity of the cellular environment and the drawbacks of fluorescence microscopy have\n",
      "prevented access to the molecular dynamics of early virus-host couplings, which are important for\n",
      "cell infection. Here we present a colocalization methodology that combines scattering interferometry\n",
      "and single-molecule fluorescence microscopy to visualize both position and orientation of single\n",
      "quantum dot-labeled Simian virus 40 (SV40) particles. By achieving nanometer spatial and 8 ms\n",
      "temporal resolution, we observed sliding and tumbling motions during rapid lateral diffusion on\n",
      "supported lipid bilayers, and repeated back and forth rocking between nanoscopic regions separated\n",
      "by 9 nm. Our findings suggest recurrent swap of receptors and viral pentamers as well as receptor\n",
      "aggregation in nanodomains. We discuss the prospects of our technique for studying virus-membrane\n",
      "interactions and for resolving nanoscopic dynamics of individual biological nano-objects.\n",
      "{'imagingTechniques': ['scattering interferometry',\n",
      "                       'single-molecule fluorescence microscopy']}\n",
      "Elapsed time: 10.64 seconds\n",
      "-------------------\n",
      "19898484\n",
      "A genetically encoded reporter of synaptic activity in vivo.\n",
      "To image synaptic activity within neural circuits, we tethered the genetically encoded calcium\n",
      "indicator (GECI) GCaMP2 to synaptic vesicles by fusion to synaptophysin. The resulting reporter,\n",
      "SyGCaMP2, detected the electrical activity of neurons with two advantages over existing cytoplasmic\n",
      "GECIs: it identified the locations of synapses and had a linear response over a wider range of spike\n",
      "frequencies. Simulations and experimental measurements indicated that linearity arises because\n",
      "SyGCaMP2 samples the brief calcium transient passing through the presynaptic compartment close to\n",
      "voltage-sensitive calcium channels rather than changes in bulk calcium concentration. In vivo\n",
      "imaging in zebrafish demonstrated that SyGCaMP2 can assess electrical activity in conventional\n",
      "synapses of spiking neurons in the optic tectum and graded voltage signals transmitted by ribbon\n",
      "synapses of retinal bipolar cells. Localizing a GECI to synaptic terminals provides a strategy for\n",
      "monitoring activity across large groups of neurons at the level of individual synapses.\n",
      "['in vivo imaging', 'simulations', 'experimental measurements']\n",
      "Elapsed time: 9.98 seconds\n",
      "-------------------\n",
      "19898485\n",
      "Imaging neural activity in worms, flies and mice with improved GCaMP calcium indicators.\n",
      "Genetically encoded calcium indicators (GECIs) can be used to image activity in defined neuronal\n",
      "populations. However, current GECIs produce inferior signals compared to synthetic indicators and\n",
      "recording electrodes, precluding detection of low firing rates. We developed a single-wavelength\n",
      "GCaMP2-based GECI (GCaMP3), with increased baseline fluorescence (3-fold), increased dynamic range\n",
      "(3-fold) and higher affinity for calcium (1.3-fold). We detected GCaMP3 fluorescence changes\n",
      "triggered by single action potentials in pyramidal cell dendrites, with signal-to-noise ratio and\n",
      "photostability substantially better than those of GCaMP2, D3cpVenus and TN-XXL. In Caenorhabditis\n",
      "elegans chemosensory neurons and the Drosophila melanogaster antennal lobe, sensory stimulation-\n",
      "evoked fluorescence responses were significantly enhanced with GCaMP3 (4-6-fold). In somatosensory\n",
      "and motor cortical neurons in the intact mouse, GCaMP3 detected calcium transients with amplitudes\n",
      "linearly dependent on action potential number. Long-term imaging in the motor cortex of behaving\n",
      "mice revealed large fluorescence changes in imaged neurons over months.\n",
      "['imaging neural activity',\n",
      " 'defined neuronal populations',\n",
      " 'synthetic indicators',\n",
      " 'recording electrodes',\n",
      " 'single-wavelength GCaMP2-based GECI',\n",
      " 'baseline fluorescence',\n",
      " 'dynamic range',\n",
      " 'affinity for calcium',\n",
      " 'photostability',\n",
      " 'action potentials',\n",
      " 'dendrites',\n",
      " 'sensory stimulation-evoked fluorescence responses',\n",
      " 'chemosensory neurons',\n",
      " 'antennal lobe',\n",
      " 'somatosensory cortical neurons',\n",
      " 'motor cortical neurons',\n",
      " 'long-term imaging']\n",
      "Elapsed time: 16.88 seconds\n",
      "-------------------\n",
      "19898486\n",
      "Optical interrogation of neural circuits in Caenorhabditis elegans.\n",
      "The nematode Caenorhabditis elegans has a compact nervous system with only 302 neurons. Whereas most\n",
      "of the synaptic connections between these neurons have been identified by electron microscopy serial\n",
      "reconstructions, functional connections have been inferred between only a few neurons through\n",
      "combinations of electrophysiology, cell ablation, in vivo calcium imaging and genetic analysis. To\n",
      "map functional connections between neurons, we combined in vivo optical stimulation with\n",
      "simultaneous calcium imaging. We analyzed the connections from the ASH sensory neurons and RIM\n",
      "interneurons to the command interneurons AVA and AVD. Stimulation of ASH or RIM neurons using\n",
      "channelrhodopsin-2 (ChR2) resulted in activation of AVA neurons, evoking an avoidance behavior. Our\n",
      "results demonstrate that we can excite specific neurons expressing ChR2 while simultaneously\n",
      "monitoring G-CaMP fluorescence in several other neurons, making it possible to rapidly decipher\n",
      "functional connections in C. elegans neural circuits.\n",
      "['in vivo calcium imaging',\n",
      " 'optical stimulation',\n",
      " 'simultaneous calcium imaging']\n",
      "Elapsed time: 10.63 seconds\n",
      "-------------------\n",
      "19898487\n",
      "Automated high-throughput mapping of promoter-enhancer interactions in zebrafish embryos.\n",
      "Zebrafish embryos offer a unique combination of high-throughput capabilities and the complexity of\n",
      "the vertebrate animal for a variety of phenotypic screening applications. However, there is a need\n",
      "for automation of imaging technologies to exploit the potential of the transparent embryo. Here we\n",
      "report a high-throughput pipeline for registering domain-specific reporter expression in zebrafish\n",
      "embryos with the aim of mapping the interactions between cis-regulatory modules and core promoters.\n",
      "Automated microscopy coupled with custom-built embryo detection and segmentation software allowed\n",
      "the spatial registration of reporter activity for 202 enhancer-promoter combinations, based on\n",
      "images of thousands of embryos. The diversity of promoter-enhancer interaction specificities\n",
      "underscores the importance of the core promoter sequence in cis-regulatory interactions and provides\n",
      "a promoter resource for transgenic reporter studies. The technology described here is also suitable\n",
      "for the spatial analysis of fluorescence readouts in genetic, pharmaceutical or toxicological\n",
      "screens.\n",
      "['microscopy', 'embryo detection and segmentation software']\n",
      "Elapsed time: 10.12 seconds\n",
      "-------------------\n",
      "19915559\n",
      "Transgenic microRNA inhibition with spatiotemporal specificity in intact organisms.\n",
      "MicroRNAs are important regulators of gene expression, yet the functional outputs of most microRNA-\n",
      "target interactions remain elusive. Here we introduce the Drosophila melanogaster microRNA sponge\n",
      "(miR-SP) as a powerful transgenic technology to dissect the function of every microRNA with precise\n",
      "spatiotemporal resolution. miR-SPs can be used to characterize tissue-specific microRNA loss-of-\n",
      "function phenotypes, define the spatial regulation of their effectors and uncover interactions\n",
      "between microRNAs and other genes. Using themiR-SP system, we identified an essential role of the\n",
      "conserved microRNA miR-8, in neuromuscular junction formation. Tissue-specific silencing revealed\n",
      "that postsynaptic activity of miR-8 is important for normal neuromuscular junction morphogenesis.\n",
      "Given that miR-SPs rely on a bipartite modular expression system, they could be used to elucidate\n",
      "the endogenous function of microRNAs in any species in which conditional expression can be achieved.\n",
      "['miR-SP',\n",
      " 'spatiotemporal resolution',\n",
      " 'tissue-specific silencing',\n",
      " 'postsynaptic activity']\n",
      "Elapsed time: 10.81 seconds\n",
      "-------------------\n",
      "19966808\n",
      "Floxin, a resource for genetically engineering mouse ESCs.\n",
      "We describe a method for the highly efficient and precise targeted modification of gene trap loci in\n",
      "mouse embryonic stem cells (ESCs). Through the Floxin method, gene trap mutations were reverted and\n",
      "new DNA sequences inserted using Cre recombinase and a shuttle vector, pFloxin. Floxin technology is\n",
      "applicable to the existing collection of 24,149 compatible gene trap cell lines, which should enable\n",
      "high-throughput modification of many genes in mouse ESCs.\n",
      "['Cre recombinase', 'shuttle vector', 'gene trap loci', 'pFloxin']\n",
      "Elapsed time: 7.39 seconds\n",
      "-------------------\n",
      "19966809\n",
      "Engineering a polarity-sensitive biosensor for time-lapse imaging of apoptotic processes and\n",
      "degeneration.\n",
      "Apoptosis is of central importance to many areas of biological research, but there is a lack of\n",
      "methods that permit continuous monitoring of apoptosis or cell viability in a nontoxic and\n",
      "noninvasive manner. Here we report the development of a tool applicable to live-cell imaging that\n",
      "facilitates the visualization of real-time apoptotic changes without perturbing the cellular\n",
      "environment. We designed a polarity-sensitive annexin-based biosensor (pSIVA) with switchable\n",
      "fluorescence states, which allows detection only when bound to apoptotic cells. Using pSIVA with\n",
      "live-cell imaging, we observed dynamic local changes in individual rat neurons during degeneration\n",
      "in vitro and in vivo. Furthermore, we observed that pSIVA binding was reversible and clearly defined\n",
      "the critical period for neurons to be rescued. We anticipate pSIVA can be widely applied to address\n",
      "questions concerning spatiotemporal events in apoptotic processes, its reversibility and the general\n",
      "viability of cells in culture.\n",
      "['live-cell imaging', 'fluorescence microscopy']\n",
      "Elapsed time: 10.02 seconds\n",
      "-------------------\n",
      "20010832\n",
      "A reprogrammable mouse strain from gene-targeted embryonic stem cells.\n",
      "The derivation of induced pluripotent stem cells (iPSCs) usually involves the viral introduction of\n",
      "reprogramming factors into somatic cells. Here we used gene targeting to generate a mouse strain\n",
      "with a single copy of an inducible, polycistronic reprogramming cassette, allowing for the induction\n",
      "of pluripotency in various somatic cell types. As these 'reprogrammable mice' can be easily bred,\n",
      "they are a useful tool to study the mechanisms underlying cellular reprogramming.\n",
      "['induced pluripotent stem cells (iPSCs)',\n",
      " 'viral introduction',\n",
      " 'gene targeting']\n",
      "Elapsed time: 7.38 seconds\n",
      "-------------------\n",
      "20037591\n",
      "Chronic microsensors for longitudinal, subsecond dopamine detection in behaving animals.\n",
      "Neurotransmission operates on a millisecond timescale but is changed by normal experience or\n",
      "neuropathology over days to months. Despite the importance of long-term neurotransmitter dynamics,\n",
      "no technique exists to track these changes in a subject from day to day over extended periods of\n",
      "time. Here we describe and characterize a microsensor that can detect the neurotransmitter dopamine\n",
      "with subsecond temporal resolution over months in vivo in rats and mice.\n",
      "['microsensor',\n",
      " 'in vivo',\n",
      " 'over months',\n",
      " 'subsecond temporal resolution',\n",
      " 'longitudinal']\n",
      "Elapsed time: 7.15 seconds\n",
      "-------------------\n",
      "20037592\n",
      "Adaptive optics via pupil segmentation for high-resolution imaging in biological tissues.\n",
      "Biological specimens are rife with optical inhomogeneities that seriously degrade imaging\n",
      "performance under all but the most ideal conditions. Measuring and then correcting for these\n",
      "inhomogeneities is the province of adaptive optics. Here we introduce an approach to adaptive optics\n",
      "in microscopy wherein the rear pupil of an objective lens is segmented into subregions, and light is\n",
      "directed individually to each subregion to measure, by image shift, the deflection faced by each\n",
      "group of rays as they emerge from the objective and travel through the specimen toward the focus.\n",
      "Applying our method to two-photon microscopy, we could recover near-diffraction-limited performance\n",
      "from a variety of biological and nonbiological samples exhibiting aberrations large or small and\n",
      "smoothly varying or abruptly changing. In particular, results from fixed mouse cortical slices\n",
      "illustrate our ability to improve signal and resolution to depths of 400 microm.\n",
      "['two-photon microscopy', 'adaptive optics', 'pupil segmentation']\n",
      "Elapsed time: 9.63 seconds\n",
      "-------------------\n",
      "20154677\n",
      "Temporal pixel multiplexing for simultaneous high-speed, high-resolution imaging.\n",
      "We introduce an imaging modality that, by offsetting pixel-exposure times during capture of a single\n",
      "image frame, embeds temporal information in each frame. This allows simultaneous acquisition of\n",
      "full-resolution images at native detector frame rates and high-speed image sequences at reduced\n",
      "resolution, without increasing bandwidth requirements. We demonstrate this method using macroscopic\n",
      "and microscopic examples, including imaging calcium transients in heart cells at 250 Hz using a\n",
      "10-Hz megapixel camera.\n",
      "['temporal pixel multiplexing',\n",
      " 'high-speed imaging',\n",
      " 'high-resolution imaging',\n",
      " 'native detector frame rates']\n",
      "Elapsed time: 7.59 seconds\n",
      "-------------------\n",
      "20190760\n",
      "Protein folding stability and dynamics imaged in a living cell.\n",
      "Biomolecular dynamics and stability are predominantly investigated in vitro and extrapolated to\n",
      "explain function in the living cell. We present fast relaxation imaging (FreI), which combines\n",
      "fluorescence microscopy and temperature jumps to probe biomolecular dynamics and stability inside a\n",
      "single living cell with high spatiotemporal resolution. We demonstrated the method by measuring the\n",
      "reversible fast folding kinetics as well as folding thermodynamics of a fluorescence resonance\n",
      "energy transfer (FRET) probe-labeled phosphoglycerate kinase construct in two human cell lines.\n",
      "Comparison with in vitro experiments at 23-49 degrees C showed that the cell environment influences\n",
      "protein stability and folding rate. FReI should also be applicable to the study of protein-protein\n",
      "interactions and heat-shock responses as well as to comparative studies of cell populations or whole\n",
      "organisms.\n",
      "['fluorescence microscopy',\n",
      " 'temperature jumps',\n",
      " 'frequency relaxation imaging (FreI)']\n",
      "Elapsed time: 9.47 seconds\n",
      "-------------------\n",
      "20195255\n",
      "Visualization of image data from cells to organisms.\n",
      "Advances in imaging techniques and high-throughput technologies are providing scientists with\n",
      "unprecedented possibilities to visualize internal structures of cells, organs and organisms and to\n",
      "collect systematic image data characterizing genes and proteins on a large scale. To make the best\n",
      "use of these increasingly complex and large image data resources, the scientific community must be\n",
      "provided with methods to query, analyze and crosslink these resources to give an intuitive visual\n",
      "representation of the data. This review gives an overview of existing methods and tools for this\n",
      "purpose and highlights some of their limitations and challenges.\n",
      "['imaging techniques',\n",
      " 'high-throughput technologies',\n",
      " 'cells',\n",
      " 'organs',\n",
      " 'organisms',\n",
      " 'query',\n",
      " 'analyze',\n",
      " 'crosslink',\n",
      " 'visual representation']\n",
      "Elapsed time: 7.65 seconds\n",
      "-------------------\n",
      "20228812\n",
      "Identifying single-cell molecular programs by stochastic profiling.\n",
      "Cells in tissues can be morphologically indistinguishable yet show molecular expression patterns\n",
      "that are remarkably heterogeneous. Here we describe an approach to comprehensively identify co-\n",
      "regulated, heterogeneously expressed genes among cells that otherwise appear identical. The\n",
      "technique, called stochastic profiling, involves repeated, random selection of very small cell\n",
      "populations via laser-capture microdissection followed by a customized single-cell amplification\n",
      "procedure and transcriptional profiling. Fluctuations in the resulting gene-expression measurements\n",
      "are then analyzed statistically to identify transcripts that are heterogeneously coexpressed. We\n",
      "stochastically profiled matrix-attached human epithelial cells in a three-dimensional culture model\n",
      "of mammary-acinar morphogenesis. Of 4,557 transcripts, we identified 547 genes with strong cell-to-\n",
      "cell expression differences. Clustering of this heterogeneous subset revealed several molecular\n",
      "'programs' implicated in protein biosynthesis, oxidative-stress responses and NF-kappaB signaling,\n",
      "which we independently confirmed by RNA fluorescence in situ hybridization. Thus, stochastic\n",
      "profiling can reveal single-cell heterogeneities without the need to measure expression in\n",
      "individual cells.\n",
      "['laser-capture microdissection',\n",
      " 'single-cell amplification',\n",
      " 'transcriptional profiling',\n",
      " 'RNA fluorescence in situ hybridization']\n",
      "Elapsed time: 11.83 seconds\n",
      "-------------------\n",
      "20228813\n",
      "Fluorescence fluctuations of quantum-dot sensors capture intracellular protein interaction dynamics.\n",
      "We extend the in vitro principle of co-immunoprecipitation to quantify dynamic protein interactions\n",
      "in living cells. Using a multiresolution implementation of fluorescence correlation spectroscopy to\n",
      "achieve maximal temporal resolution, we monitored the interactions of endogenous bait proteins,\n",
      "recruited by quantum dots, with fluorescently tagged prey. With this approach, we analyzed the rapid\n",
      "physiological regulation of protein kinase A.\n",
      "['fluorescence correlation spectroscopy', 'co-immunoprecipitation']\n",
      "Elapsed time: 6.91 seconds\n",
      "-------------------\n",
      "20228814\n",
      "In vivo wide-area cellular imaging by side-view endomicroscopy.\n",
      "In vivo imaging of small animals offers several possibilities for studying normal and disease\n",
      "biology, but visualizing organs with single-cell resolution is challenging. We describe rotational\n",
      "side-view confocal endomicroscopy, which enables cellular imaging of gastrointestinal and\n",
      "respiratory tracts in mice and may be extensible to imaging organ parenchyma such as cerebral\n",
      "cortex. We monitored cell infiltration, vascular changes and tumor progression during inflammation\n",
      "and tumorigenesis in colon over several months.\n",
      "['confocal endomicroscopy',\n",
      " 'rotational side-view confocal endomicroscopy',\n",
      " 'cellular imaging',\n",
      " 'single-cell resolution']\n",
      "Elapsed time: 7.85 seconds\n",
      "-------------------\n",
      "20228815\n",
      "Imaging type-III secretion reveals dynamics and spatial segregation of Salmonella effectors.\n",
      "The type-III secretion system (T3SS) enables gram-negative bacteria to inject effector proteins into\n",
      "eukaryotic host cells. Upon entry, T3SS effectors work cooperatively to reprogram host cells,\n",
      "enabling bacterial survival. Progress in understanding when and where effectors localize in host\n",
      "cells has been hindered by a dearth of tools to study these proteins in the native cellular\n",
      "environment. We report a method to label and track T3SS effectors during infection using a split-GFP\n",
      "system. We demonstrate this technique by labeling three effectors from Salmonella enterica (PipB2,\n",
      "SteA and SteC) and characterizing their localization in host cells. PipB2 displayed highly dynamic\n",
      "behavior on tubules emanating from the Salmonella-containing vacuole labeled with both endo- and\n",
      "exocytic markers. SteA was preferentially enriched on tubules localizing with Golgi markers. This\n",
      "segregation suggests that effector targeting and localization may have a functional role during\n",
      "infection.\n",
      "['split-GFP system', 'endo- and exocytic markers', 'Golgi markers']\n",
      "Elapsed time: 10.47 seconds\n",
      "-------------------\n",
      "20354514\n",
      "Laser tricks without labels.\n",
      "Nonlinear optical microscopy lets researchers see chemical composition in living cells and\n",
      "organisms.\n",
      "['nonlinear optical microscopy', 'laser tricks without labels']\n",
      "Elapsed time: 4.99 seconds\n",
      "-------------------\n",
      "20364146\n",
      "Fast, single-molecule localization that achieves theoretically minimum uncertainty.\n",
      "We describe an iterative algorithm that converges to the maximum likelihood estimate of the position\n",
      "and intensity of a single fluorophore. Our technique efficiently computes and achieves the Cramér-\n",
      "Rao lower bound, an essential tool for parameter estimation. An implementation of the algorithm on\n",
      "graphics processing unit hardware achieved more than 10(5) combined fits and Cramér-Rao lower bound\n",
      "calculations per second, enabling real-time data analysis for super-resolution imaging and other\n",
      "applications.\n",
      "['single-molecule localization',\n",
      " 'Cramér-Rao lower bound',\n",
      " 'super-resolution imaging',\n",
      " 'real-time data analysis']\n",
      "Elapsed time: 7.55 seconds\n",
      "-------------------\n",
      "20364147\n",
      "Optimized localization analysis for single-molecule tracking and super-resolution microscopy.\n",
      "We optimally localized isolated fluorescent beads and molecules imaged as diffraction-limited spots,\n",
      "determined the orientation of molecules and present reliable formulas for the precision of various\n",
      "localization methods. Both theory and experimental data showed that unweighted least-squares fitting\n",
      "of a Gaussian squanders one-third of the available information, a popular formula for its precision\n",
      "exaggerates beyond Fisher's information limit, and weighted least-squares may do worse, whereas\n",
      "maximum-likelihood fitting is practically optimal.\n",
      "['localization analysis',\n",
      " 'single-molecule tracking',\n",
      " 'super-resolution microscopy',\n",
      " 'isolated fluorescent beads',\n",
      " 'diffraction-limited spots',\n",
      " 'unweighted least-squares fitting',\n",
      " 'Gaussian squanders',\n",
      " 'maximum-likelihood fitting']\n",
      "Elapsed time: 8.69 seconds\n",
      "-------------------\n",
      "20383133\n",
      "A streptavidin variant with slower biotin dissociation and increased mechanostability.\n",
      "Streptavidin binds biotin conjugates with exceptional stability but dissociation does occur,\n",
      "limiting its use in imaging, DNA amplification and nanotechnology. We identified a mutant\n",
      "streptavidin, traptavidin, with more than tenfold slower biotin dissociation, increased mechanical\n",
      "strength and improved thermostability; this resilience should enable diverse applications. FtsK, a\n",
      "motor protein important in chromosome segregation, rapidly displaced streptavidin from biotinylated\n",
      "DNA, whereas traptavidin resisted displacement, indicating the force generated by Ftsk\n",
      "translocation.\n",
      "['imaging', 'DNA amplification', 'nanotechnology']\n",
      "Elapsed time: 8.41 seconds\n",
      "-------------------\n",
      "20400966\n",
      "High-speed in vivo calcium imaging reveals neuronal network activity with near-millisecond\n",
      "precision.\n",
      "Two-photon calcium imaging of neuronal populations enables optical recording of spiking activity in\n",
      "living animals, but standard laser scanners are too slow to accurately determine spike times. Here\n",
      "we report in vivo imaging in mouse neocortex with greatly improved temporal resolution using random-\n",
      "access scanning with acousto-optic deflectors. We obtained fluorescence measurements from 34-91\n",
      "layer 2/3 neurons at a 180-490 Hz sampling rate. We detected single action potential-evoked calcium\n",
      "transients with signal-to-noise ratios of 2-5 and determined spike times with near-millisecond\n",
      "precision and 5-15 ms confidence intervals. An automated 'peeling' algorithm enabled reconstruction\n",
      "of complex spike trains from fluorescence traces up to 20-30 Hz frequency, uncovering spatiotemporal\n",
      "trial-to-trial variability of sensory responses in barrel cortex and visual cortex. By revealing\n",
      "spike sequences in neuronal populations on a fast time scale, high-speed calcium imaging will\n",
      "facilitate optical studies of information processing in brain microcircuits.\n",
      "Invalid json output: [\n",
      "\"two-photon calcium imaging\",\n",
      "\"random-access scanning with acousto-optic deflectors\",\n",
      "\"fluorescence measurements\"\n",
      "\"automated 'peeling' algorithm\"\n",
      "] derived from Sure! Here is the list of imaging techniques described in the abstract:\n",
      "\n",
      "[\n",
      "\"two-photon calcium imaging\",\n",
      "\"random-access scanning with acousto-optic deflectors\",\n",
      "\"fluorescence measurements\"\n",
      "\"automated 'peeling' algorithm\"\n",
      "]\n",
      "20418868\n",
      "Targeted gene deletions in C. elegans using transposon excision.\n",
      "We developed a method, MosDEL, to generate targeted knockouts of genes in Caenorhabditis elegans by\n",
      "injection. We generated a double-strand break by mobilizing a Mos1 transposon adjacent to the region\n",
      "to be deleted; the double-stranded break is repaired using injected DNA as a template. Repair can\n",
      "delete up to 25 kb of DNA and simultaneously insert a positive selection marker.\n",
      "['MosDEL', 'transposon excision']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "20453865\n",
      "A photoconvertible reporter of the ubiquitin-proteasome system in vivo.\n",
      "The ubiquitin-proteasome system (UPS) orchestrates many cellular and tissue-specific processes by\n",
      "degrading damaged and key regulatory proteins. To enable investigation of UPS activity in different\n",
      "cell types in a living animal, we developed a photoconvertible fluorescent UPS reporter system for\n",
      "live imaging and quantification of protein degradation in Caenorhabditis elegans. Our reporter\n",
      "consists of the photoconvertible fluorescent protein Dendra2 targeted for proteasomal degradation by\n",
      "fusion to the UbG76V mutant form of ubiquitin. In contrast to previous reporters, this system\n",
      "permits quantification of UPS activity independently of protein synthesis. Our reporter revealed\n",
      "that UPS-mediated protein degradation varies in a cell type-specific and age-dependent manner in C.\n",
      "elegans.\n",
      "['live imaging',\n",
      " 'quantification of protein degradation',\n",
      " 'photoconvertible fluorescent protein Dendra2',\n",
      " 'proteasomal degradation']\n",
      "Elapsed time: 9.90 seconds\n",
      "-------------------\n",
      "20453867\n",
      "In situ analysis of tyrosine phosphorylation networks by FLIM on cell arrays.\n",
      "Extracellular stimuli are transduced inside the cell by posttranslational modifications (PTMs), such\n",
      "as phosphorylation, of proteins in signaling networks. Insight into the structure of these networks\n",
      "requires quantification of PTM levels in individual cells. Fluorescence resonance energy transfer\n",
      "(FRET) measured by fluorescence lifetime imaging microscopy (FLIM) is a powerful tool to image PTM\n",
      "levels in situ. FLIM on cell arrays that express fluorescent protein fusions can quantify tyrosine\n",
      "phosphorylation patterns in large networks in individual cells. We identified tyrosine kinase\n",
      "substrates by imaging their phosphorylation levels after inhibition of protein tyrosine\n",
      "phosphatases. Analysis of the correlation between protein phosphorylation and expression levels at\n",
      "single cell resolution allowed us to identify positive feedback motifs. Using FLIM on cell arrays\n",
      "(CA-FLIM), we uncovered components that transduce signals from epidermal growth factor receptor.\n",
      "{'imagingTechniques': ['fluorescence resonance energy transfer (FRET)',\n",
      "                       'fluorescence lifetime imaging microscopy (FLIM)',\n",
      "                       'in situ analysis']}\n",
      "Elapsed time: 10.72 seconds\n",
      "-------------------\n",
      "20512143\n",
      "Enhanced neuronal RNAi in C. elegans using SID-1.\n",
      "We expressed SID-1, a transmembrane protein from Caenorhabditis elegans that is required for\n",
      "systemic RNA interference (RNAi), in C. elegans neurons. This expression increased the response of\n",
      "neurons to double-stranded (ds)RNA delivered by feeding. Mutations in the lin-15b and lin-35 genes\n",
      "enhanced this effect. Worms expressing neuronal SID-1 showed RNAi phenotypes when fed with bacteria\n",
      "expressing dsRNA for known neuronal genes and for uncharacterized genes with no previously known\n",
      "neuronal phenotypes. Neuronal expression of sid-1 decreased nonneuronal RNAi, suggesting that\n",
      "neurons expressing transgenic sid-1(+) served as a sink for dsRNA. This effect, or a sid-1(-)\n",
      "background, can be used to uncover neuronal defects for lethal genes. Expression of sid-1(+) from\n",
      "cell-specific promoters in sid-1 mutants results in cell-specific feeding RNAi. We used these\n",
      "strains to identify a role for integrin signaling genes in mechanosensation.\n",
      "['cell-specific promoters', 'dsRNA delivery by feeding', 'mechanosensation']\n",
      "Elapsed time: 11.86 seconds\n",
      "-------------------\n",
      "20512144\n",
      "Probabilistic density maps to study global endomembrane organization.\n",
      "We developed a computational imaging approach that describes the three-dimensional spatial\n",
      "organization of endomembranes from micromanipulation-normalized mammalian cells with probabilistic\n",
      "density maps. Applied to several well-known marker proteins, this approach revealed the average\n",
      "steady-state organization of early endosomes, multivesicular bodies or lysosomes, endoplasmic\n",
      "reticulum exit sites, the Golgi apparatus and Golgi-derived transport carriers in crossbow-shaped\n",
      "cells. The steady-state organization of each tested endomembranous population was well-defined,\n",
      "unique and in some cases depended on the cellular adhesion geometry. Density maps of all\n",
      "endomembrane populations became stable when pooling several tens of cells only and were reproducible\n",
      "in independent experiments, allowing construction of a standardized cell model. We detected subtle\n",
      "changes in steady-state organization induced by disruption of the cellular cytoskeleton, with\n",
      "statistical significance observed for just 20 cells. Thus, combining micropatterning with\n",
      "construction of endomembrane density maps allows the systematic study of intracellular trafficking\n",
      "determinants.\n",
      "['micromanipulation-normalized mammalian cells',\n",
      " 'computational imaging approach',\n",
      " 'probabilistic density maps',\n",
      " 'early endosomes',\n",
      " 'multivesicular bodies',\n",
      " 'lysosomes',\n",
      " 'endoplasmic reticulum exit sites',\n",
      " 'Golgi apparatus',\n",
      " 'Golgi-derived transport carriers']\n",
      "Elapsed time: 12.76 seconds\n",
      "-------------------\n",
      "20521367\n",
      "Discovering Mozart.\n",
      "High-throughput imaging of genome-wide RNA interference screens and systematic analysis of protein\n",
      "complexes involved in mitosis yield valuable resources and reveal new subunits of well-characterized\n",
      "complexes.\n",
      "['high-throughput imaging',\n",
      " 'genome-wide RNA interference screens',\n",
      " 'systematic analysis of protein complexes involved in mitosis']\n",
      "Elapsed time: 6.16 seconds\n",
      "-------------------\n",
      "20526346\n",
      "Two-photon calcium imaging from head-fixed Drosophila during optomotor walking behavior.\n",
      "Drosophila melanogaster is a model organism rich in genetic tools to manipulate and identify neural\n",
      "circuits involved in specific behaviors. Here we present a technique for two-photon calcium imaging\n",
      "in the central brain of head-fixed Drosophila walking on an air-supported ball. The ball's motion is\n",
      "tracked at high resolution and can be treated as a proxy for the fly's own movements. We used the\n",
      "genetically encoded calcium sensor, GCaMP3.0, to record from important elements of the motion-\n",
      "processing pathway, the horizontal-system lobula plate tangential cells (LPTCs) in the fly optic\n",
      "lobe. We presented motion stimuli to the tethered fly and found that calcium transients in\n",
      "horizontal-system neurons correlated with robust optomotor behavior during walking. Our technique\n",
      "allows both behavior and physiology in identified neurons to be monitored in a genetic model\n",
      "organism with an extensive repertoire of walking behaviors.\n",
      "['two-photon calcium imaging',\n",
      " 'high resolution tracking',\n",
      " 'genetically encoded calcium sensor',\n",
      " 'GCaMP3.0']\n",
      "Elapsed time: 10.98 seconds\n",
      "-------------------\n",
      "20543845\n",
      "Imaging plant growth in 4D: robust tissue reconstruction and lineaging at cell resolution.\n",
      "Quantitative information on growing organs is required to better understand morphogenesis in both\n",
      "plants and animals. However, detailed analyses of growth patterns at cellular resolution have\n",
      "remained elusive. We developed an approach, multiangle image acquisition, three-dimensional\n",
      "reconstruction and cell segmentation-automated lineage tracking (MARS-ALT), in which we imaged whole\n",
      "organs from multiple angles, computationally merged and segmented these images to provide accurate\n",
      "cell identification in three dimensions and automatically tracked cell lineages through multiple\n",
      "rounds of cell division during development. Using these methods, we quantitatively analyzed\n",
      "Arabidopsis thaliana flower development at cell resolution, which revealed differential growth\n",
      "patterns of key regions during early stages of floral morphogenesis. Lastly, using rice roots, we\n",
      "demonstrated that this approach is both generic and scalable.\n",
      "['multiangle image acquisition',\n",
      " 'three-dimensional reconstruction',\n",
      " 'cell segmentation',\n",
      " 'automated lineage tracking']\n",
      "Elapsed time: 8.84 seconds\n",
      "-------------------\n",
      "20562866\n",
      "Noncontact microrheology at acoustic frequencies using frequency-modulated atomic force microscopy.\n",
      "We report an atomic force microscopy (AFM) method for assessing elastic and viscous properties of\n",
      "soft samples at acoustic frequencies under non-contact conditions. The method can be used to measure\n",
      "material properties via frequency modulation and is based on hydrodynamics theory of thin gaps we\n",
      "developed here. A cantilever with an attached microsphere is forced to oscillate tens of nanometers\n",
      "above a sample. The elastic modulus and viscosity of the sample are estimated by measuring the\n",
      "frequency-dependence of the phase lag between the oscillating microsphere and the driving piezo at\n",
      "various heights above the sample. This method features an effective area of pyramidal tips used in\n",
      "contact AFM but with only piconewton applied forces. Using this method, we analyzed polyacrylamide\n",
      "gels of different stiffness and assessed graded mechanical properties of guinea pig tectorial\n",
      "membrane. The technique enables the study of microrheology of biological tissues that produce or\n",
      "detect sound.\n",
      "['atomic force microscopy',\n",
      " 'frequency-modulated atomic force microscopy',\n",
      " 'microrheology']\n",
      "Elapsed time: 10.29 seconds\n",
      "-------------------\n",
      "20601949\n",
      "A photoactivatable marker protein for pulse-chase imaging with superresolution.\n",
      "IrisFP is a photoactivatable fluorescent protein that combines irreversible photoconversion from a\n",
      "green- to a red-emitting form with reversible photoswitching between a fluorescent and a\n",
      "nonfluorescent state in both forms. Here we introduce a monomeric variant, mIrisFP, and demonstrate\n",
      "how its multiple photoactivation modes can be used for pulse-chase experiments combined with\n",
      "subdiffraction-resolution imaging in living cells by using dual-color photoactivation localization\n",
      "microscopy (PALM).\n",
      "['PALM',\n",
      " 'dual-color photoactivation localization microscopy',\n",
      " 'subdiffraction-resolution imaging',\n",
      " 'living cells']\n",
      "Elapsed time: 7.54 seconds\n",
      "-------------------\n",
      "20601950\n",
      "Fast, high-contrast imaging of animal development with scanned light sheet-based structured-\n",
      "illumination microscopy.\n",
      "Recording light-microscopy images of large, nontransparent specimens, such as developing\n",
      "multicellular organisms, is complicated by decreased contrast resulting from light scattering. Early\n",
      "zebrafish development can be captured by standard light-sheet microscopy, but new imaging strategies\n",
      "are required to obtain high-quality data of late development or of less transparent organisms. We\n",
      "combined digital scanned laser light-sheet fluorescence microscopy with incoherent structured-\n",
      "illumination microscopy (DSLM-SI) and created structured-illumination patterns with continuously\n",
      "adjustable frequencies. Our method discriminates the specimen-related scattered background from\n",
      "signal fluorescence, thereby removing out-of-focus light and optimizing the contrast of in-focus\n",
      "structures. DSLM-SI provides rapid control of the illumination pattern, exceptional imaging quality\n",
      "and high imaging speeds. We performed long-term imaging of zebrafish development for 58 h and fast\n",
      "multiple-view imaging of early Drosophila melanogaster development. We reconstructed cell positions\n",
      "over time from the Drosophila DSLM-SI data and created a fly digital embryo.\n",
      "['digital scanned laser light-sheet fluorescence microscopy',\n",
      " 'incoherent structured-illumination microscopy (DSLM-SI)',\n",
      " 'structured-illumination patterns with continuously adjustable frequencies']\n",
      "Elapsed time: 12.57 seconds\n",
      "-------------------\n",
      "20622860\n",
      "Imaging brain electric signals with genetically targeted voltage-sensitive fluorescent proteins.\n",
      "Cortical information processing relies on synaptic interactions between diverse classes of neurons\n",
      "with distinct electrophysiological and connection properties. Uncovering the operational principles\n",
      "of these elaborate circuits requires the probing of electrical activity from selected populations of\n",
      "defined neurons. Here we show that genetically encoded voltage-sensitive fluorescent proteins\n",
      "(VSFPs) provide an optical voltage report from targeted neurons in culture, acute brain slices and\n",
      "living mice. By expressing VSFPs in pyramidal cells of mouse somatosensory cortex, we also\n",
      "demonstrate that these probes can report cortical electrical responses to single sensory stimuli in\n",
      "vivo. These protein-based voltage probes will facilitate the analysis of cortical circuits in\n",
      "genetically defined cell populations and are hence a valuable addition to the optogenetic toolbox.\n",
      "['voltage-sensitive fluorescent proteins',\n",
      " 'optical voltage report',\n",
      " 'genetically encoded voltage probes',\n",
      " 'protein-based voltage probes',\n",
      " 'optogenetic toolbox']\n",
      "Elapsed time: 9.69 seconds\n",
      "-------------------\n",
      "20639868\n",
      "High-throughput in vivo vertebrate screening.\n",
      "We demonstrate a high-throughput platform for cellular-resolution in vivo chemical and genetic\n",
      "screens on zebrafish larvae. The system automatically loads zebrafish from reservoirs or multiwell\n",
      "plates, and positions and rotates them for high-speed confocal imaging and laser manipulation of\n",
      "both superficial and deep organs within 19 s without damage. We performed small-scale test screening\n",
      "of retinal axon guidance mutants and neuronal regeneration assays in combination with femtosecond\n",
      "laser microsurgery.\n",
      "{'imaging techniques': ['confocal imaging',\n",
      "                        'laser manipulation',\n",
      "                        'high-speed confocal imaging']}\n",
      "Elapsed time: 7.49 seconds\n",
      "-------------------\n",
      "20676081\n",
      "Going deeper than microscopy: the optical imaging frontier in biology.\n",
      "Optical microscopy has been a fundamental tool of biological discovery for more than three\n",
      "centuries, but its in vivo tissue imaging ability has been restricted by light scattering to\n",
      "superficial investigations, even when confocal or multiphoton methods are used. Recent advances in\n",
      "optical and optoacoustic (photoacoustic) imaging now allow imaging at depths and resolutions\n",
      "unprecedented for optical methods. These abilities are increasingly important to understand the\n",
      "dynamic interactions of cellular processes at different systems levels, a major challenge of\n",
      "postgenome biology. This Review discusses promising photonic methods that have the ability to\n",
      "visualize cellular and subcellular components in tissues across different penetration scales. The\n",
      "methods are classified into microscopic, mesoscopic and macroscopic approaches, according to the\n",
      "tissue depth at which they operate. Key characteristics associated with different imaging\n",
      "implementations are described and the potential of these technologies in biological applications is\n",
      "discussed.\n",
      "['microscopy',\n",
      " 'confocal microscopy',\n",
      " 'multiphoton microscopy',\n",
      " 'optical imaging',\n",
      " 'optoacoustic imaging',\n",
      " 'photoacoustic imaging']\n",
      "Elapsed time: 10.42 seconds\n",
      "-------------------\n",
      "20676108\n",
      "Mechanical regulation of cell function with geometrically modulated elastomeric substrates.\n",
      "We report the establishment of a library of micromolded elastomeric micropost arrays to modulate\n",
      "substrate rigidity independently of effects on adhesive and other material surface properties. We\n",
      "demonstrated that micropost rigidity impacts cell morphology, focal adhesions, cytoskeletal\n",
      "contractility and stem cell differentiation. Furthermore, early changes in cytoskeletal\n",
      "contractility predicted later stem cell fate decisions in single cells.\n",
      "['microscope',\n",
      " 'confocal microscopy',\n",
      " 'fluorescence microscopy',\n",
      " 'live cell imaging',\n",
      " 'cell morphology',\n",
      " 'focal adhesions',\n",
      " 'cytoskeletal contractility',\n",
      " 'stem cell differentiation']\n",
      "Elapsed time: 8.13 seconds\n",
      "-------------------\n",
      "20676109\n",
      "Automated imaging with ScanLag reveals previously undetectable bacterial growth phenotypes.\n",
      "We developed an automated system, ScanLag, that measures in parallel the delay in growth (lag time)\n",
      "and growth rate of thousands of cells. Using ScanLag, we detected small subpopulations of bacteria\n",
      "with dramatically increased lag time upon starvation. By screening a library of Escherichia coli\n",
      "deletion mutants, we achieved two-dimensional mapping of growth characteristics, which showed that\n",
      "ScanLag enables multidimensional screens for quantitative characterization and identification of\n",
      "rare phenotypic variants.\n",
      "['ScanLag', 'growth rate', 'lag time']\n",
      "Elapsed time: 7.02 seconds\n",
      "-------------------\n",
      "20693996\n",
      "CellCognition: time-resolved phenotype annotation in high-throughput live cell imaging.\n",
      "Fluorescence time-lapse imaging has become a powerful tool to investigate complex dynamic processes\n",
      "such as cell division or intracellular trafficking. Automated microscopes generate time-resolved\n",
      "imaging data at high throughput, yet tools for quantification of large-scale movie data are largely\n",
      "missing. Here we present CellCognition, a computational framework to annotate complex cellular\n",
      "dynamics. We developed a machine-learning method that combines state-of-the-art classification with\n",
      "hidden Markov modeling for annotation of the progression through morphologically distinct biological\n",
      "states. Incorporation of time information into the annotation scheme was essential to suppress\n",
      "classification noise at state transitions and confusion between different functional states with\n",
      "similar morphology. We demonstrate generic applicability in different assays and perturbation\n",
      "conditions, including a candidate-based RNA interference screen for regulators of mitotic exit in\n",
      "human cells. CellCognition is published as open source software, enabling live-cell imaging-based\n",
      "screening with assays that directly score cellular dynamics.\n",
      "['fluorescence time-lapse imaging',\n",
      " 'high-throughput live cell imaging',\n",
      " 'automated microscopes']\n",
      "Elapsed time: 10.13 seconds\n",
      "-------------------\n",
      "20693997\n",
      "Two-photon high-resolution measurement of partial pressure of oxygen in cerebral vasculature and\n",
      "tissue.\n",
      "Measurements of oxygen partial pressure (pO(2)) with high temporal and spatial resolution in three\n",
      "dimensions is crucial for understanding oxygen delivery and consumption in normal and diseased\n",
      "brain. Among existing pO(2) measurement methods, phosphorescence quenching is optimally suited for\n",
      "the task. However, previous attempts to couple phosphorescence with two-photon laser scanning\n",
      "microscopy have faced substantial difficulties because of extremely low two-photon absorption cross-\n",
      "sections of conventional phosphorescent probes. Here we report to our knowledge the first practical\n",
      "in vivo two-photon high-resolution pO(2) measurements in small rodents' cortical microvasculature\n",
      "and tissue, made possible by combining an optimized imaging system with a two-photon-enhanced\n",
      "phosphorescent nanoprobe. The method features a measurement depth of up to 250 microm, sub-second\n",
      "temporal resolution and requires low probe concentration. The properties of the probe allowed for\n",
      "direct high-resolution measurement of cortical extravascular (tissue) pO(2), opening many\n",
      "possibilities for functional metabolic brain studies.\n",
      "['two-photon laser scanning microscopy',\n",
      " 'phosphorescence quenching',\n",
      " 'two-photon high-resolution pO(2) measurements']\n",
      "Elapsed time: 12.49 seconds\n",
      "-------------------\n",
      "20693998\n",
      "Live-cell super-resolution imaging with trimethoprim conjugates.\n",
      "The spatiotemporal resolution of subdiffraction fluorescence imaging has been limited by the\n",
      "difficulty of labeling proteins in cells with suitable fluorophores. Here we report a chemical tag\n",
      "that allows proteins to be labeled with an organic fluorophore with high photon flux and fast\n",
      "photoswitching performance in live cells. This label allowed us to image the dynamics of human\n",
      "histone H2B protein in living cells at approximately 20 nm resolution.\n",
      "['fluorescence imaging',\n",
      " 'super-resolution imaging',\n",
      " 'subdiffraction fluorescence imaging']\n",
      "Elapsed time: 7.15 seconds\n",
      "-------------------\n",
      "20729842\n",
      "Analysis of microtubule dynamic instability using a plus-end growth marker.\n",
      "Regulation of microtubule dynamics is essential for many cell biological processes and is likely to\n",
      "be variable between different subcellular regions. We describe a computational approach to analyze\n",
      "microtubule dynamics by detecting growing microtubule plus ends. Our algorithm tracked all EB1-EGFP\n",
      "comets visible in an image time-lapse sequence allowing the detection of spatial patterns of\n",
      "microtubule dynamics. We introduce spatiotemporal clustering of EB1-EGFP growth tracks to infer\n",
      "microtubule behaviors during phases of pause and shortening. We validated the algorithm by comparing\n",
      "the results to data for manually tracked, homogeneously labeled microtubules and by analyzing the\n",
      "effects of well-characterized inhibitors of microtubule polymerization dynamics. We used our method\n",
      "to analyze spatial variations of intracellular microtubule dynamics in migrating epithelial cells.\n",
      "['time-lapse sequence',\n",
      " 'EB1-EGFP growth tracks',\n",
      " 'spatiotemporal clustering',\n",
      " 'manual tracking',\n",
      " 'homogeneously labeled microtubules']\n",
      "Elapsed time: 10.00 seconds\n",
      "-------------------\n",
      "20818379\n",
      "Near-infrared fluorescent proteins.\n",
      "Fluorescent proteins with emission wavelengths in the near-infrared and infrared range are in high\n",
      "demand for whole-body imaging techniques. Here we report near-infrared dimeric fluorescent proteins\n",
      "eqFP650 and eqFP670. To our knowledge, eqFP650 is the brightest fluorescent protein with emission\n",
      "maximum above 635 nm, and eqFP670 displays the most red-shifted emission maximum and high\n",
      "photostability.\n",
      "['whole-body imaging',\n",
      " 'infrared range',\n",
      " 'dimeric fluorescent proteins',\n",
      " 'eqFP650',\n",
      " 'eqFP670',\n",
      " 'photostability']\n",
      "Elapsed time: 8.00 seconds\n",
      "-------------------\n",
      "20852649\n",
      "Scanless two-photon excitation of channelrhodopsin-2.\n",
      "Light-gated ion channels and pumps have made it possible to probe intact neural circuits by\n",
      "manipulating the activity of groups of genetically similar neurons. What is needed now is a method\n",
      "for precisely aiming the stimulating light at single neuronal processes, neurons or groups of\n",
      "neurons. We developed a method that combines generalized phase contrast with temporal focusing (TF-\n",
      "GPC) to shape two-photon excitation for this purpose. The illumination patterns are generated\n",
      "automatically from fluorescence images of neurons and shaped to cover the cell body or dendrites, or\n",
      "distributed groups of cells. The TF-GPC two-photon excitation patterns generated large photocurrents\n",
      "in Channelrhodopsin-2-expressing cultured cells and neurons and in mouse acute cortical slices. The\n",
      "amplitudes of the photocurrents can be precisely modulated by controlling the size and shape of the\n",
      "excitation volume and, thereby, be used to trigger single action potentials or trains of action\n",
      "potentials.\n",
      "['generalized phase contrast', 'temporal focusing', 'two-photon excitation']\n",
      "Elapsed time: 9.94 seconds\n",
      "-------------------\n",
      "20871617\n",
      "Quantitative dynamic footprinting microscopy reveals mechanisms of neutrophil rolling.\n",
      "We introduce quantitative dynamic footprinting microscopy to resolve neutrophil rolling on\n",
      "P-selectin. We observed that the footprint of a rolling neutrophil was fourfold larger than\n",
      "previously thought, and that P-selectin-PSGL-1 bonds were relaxed at the leading edge of the rolling\n",
      "cell, compressed under the cell center, and stretched at the trailing edge. Each rolling neutrophil\n",
      "formed three to four long tethers that extended up to 16 μm behind the rolling cell.\n",
      "['quantitative dynamic footprinting microscopy',\n",
      " 'P-selectin-PSGL-1 bonds',\n",
      " 'tethers']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "20953177\n",
      "Dual RMCE for efficient re-engineering of mouse mutant alleles.\n",
      "We have developed dual recombinase-mediated cassette exchange (dRMCE) to efficiently re-engineer the\n",
      "thousands of available conditional alleles in mouse embryonic stem cells. dRMCE takes advantage of\n",
      "the wild-type loxP and FRT sites present in these conditional alleles and in many gene-trap lines.\n",
      "dRMCE is a scalable, flexible tool to introduce tags, reporters and mutant coding regions into an\n",
      "endogenous locus of interest in an easy and highly efficient manner.\n",
      "['dRMCE', 'recombinase-mediated cassette exchange', 'loxP', 'FRT']\n",
      "Elapsed time: 7.69 seconds\n",
      "-------------------\n",
      "20966916\n",
      "Chronic optical access through a polished and reinforced thinned skull.\n",
      "We present a method to form an optical window in the mouse skull that spans millimeters and is\n",
      "stable for months without causing brain inflammation. This enabled us to repeatedly image blood flow\n",
      "in cortical capillaries of awake mice and determine long-range correlations in speed. We also\n",
      "repeatedly imaged dendritic spines, microglia and angioarchitecture, as well as used illumination to\n",
      "drive motor output via optogenetics and induce microstrokes via photosensitizers.\n",
      "['optical window',\n",
      " 'cortical capillaries',\n",
      " 'awake mice',\n",
      " 'dendritic spines',\n",
      " 'microglia',\n",
      " 'angioarchitecture',\n",
      " 'optogenetics',\n",
      " 'photosensitizers']\n",
      "Elapsed time: 8.41 seconds\n",
      "-------------------\n",
      "21037589\n",
      "Rapid blue-light-mediated induction of protein interactions in living cells.\n",
      "Dimerizers allowing inducible control of protein-protein interactions are powerful tools for\n",
      "manipulating biological processes. Here we describe genetically encoded light-inducible protein-\n",
      "interaction modules based on Arabidopsis thaliana cryptochrome 2 and CIB1 that require no exogenous\n",
      "ligands and dimerize on blue-light exposure with subsecond time resolution and subcellular spatial\n",
      "resolution. We demonstrate the utility of this system by inducing protein translocation,\n",
      "transcription and Cre recombinase-mediated DNA recombination using light.\n",
      "['cryptochrome 2', 'CIB1', 'blue-light exposure']\n",
      "Elapsed time: 7.20 seconds\n",
      "-------------------\n",
      "21049576\n",
      "Structural biology.  Protein structure gets exciting.\n",
      "Researchers determined the excited-state structure of a small protein using nuclear magnetic\n",
      "resonance spectroscopy.\n",
      "['nuclear magnetic resonance spectroscopy', 'none']\n",
      "Elapsed time: 5.12 seconds\n",
      "-------------------\n",
      "21049580\n",
      "Microscopy.  Self-healing light beams.\n",
      "The self-reconstructing properties of Bessel beams provide healing benefits in highly scattering\n",
      "media.\n",
      "['microscopy', 'self-healing light beams']\n",
      "Elapsed time: 5.48 seconds\n",
      "-------------------\n",
      "21076418\n",
      "A defined glycosaminoglycan-binding substratum for human pluripotent stem cells.\n",
      "To exploit the full potential of human pluripotent stem cells for regenerative medicine,\n",
      "developmental biology and drug discovery, defined culture conditions are needed. Media of known\n",
      "composition that maintain human embryonic stem (hES) cells have been developed, but finding\n",
      "chemically defined, robust substrata has proven difficult. We used an array of self-assembled\n",
      "monolayers to identify peptide surfaces that sustain pluripotent stem cell self-renewal. The\n",
      "effective substrates displayed heparin-binding peptides, which can interact with cell-surface\n",
      "glycosaminoglycans and could be used with a defined medium to culture hES cells for more than 3\n",
      "months. The resulting cells maintained a normal karyotype and had high levels of pluripotency\n",
      "markers. The peptides supported growth of eight pluripotent cell lines on a variety of scaffolds.\n",
      "Our results indicate that synthetic substrates that recognize cell-surface glycans can facilitate\n",
      "the long-term culture of pluripotent stem cells.\n",
      "['heparin-binding peptides',\n",
      " 'self-assembled monolayers',\n",
      " 'synthetic substrates']\n",
      "Elapsed time: 10.94 seconds\n",
      "-------------------\n",
      "21076419\n",
      "Three-dimensional cellular ultrastructure resolved by X-ray microscopy.\n",
      "We developed an X-ray microscope using partially coherent object illumination instead of previously\n",
      "used quasi-incoherent illumination. The design permitted the incorporation of a cryogenic tilt\n",
      "stage, enabling tomography of frozen-hydrated, intact adherent cells. We obtained three-dimensional\n",
      "reconstructions of mouse adenocarcinoma cells at ∼36-nm (Rayleigh) and ∼70-nm (Fourier ring\n",
      "correlation) resolution, which allowed us to visualize the double nuclear membrane, nuclear pores,\n",
      "nuclear membrane channels, mitochondrial cristae and lysosomal inclusions.\n",
      "['X-ray microscopy',\n",
      " 'tomography',\n",
      " 'quasi-incoherent illumination',\n",
      " 'cryogenic tilt stage']\n",
      "Elapsed time: 8.13 seconds\n",
      "-------------------\n",
      "21102453\n",
      "Protein localization in electron micrographs using fluorescence nanoscopy.\n",
      "A complete portrait of a cell requires a detailed description of its molecular topography: proteins\n",
      "must be linked to particular organelles. Immunocytochemical electron microscopy can reveal locations\n",
      "of proteins with nanometer resolution but is limited by the quality of fixation, the paucity of\n",
      "antibodies and the inaccessibility of antigens. Here we describe correlative fluorescence electron\n",
      "microscopy for the nanoscopic localization of proteins in electron micrographs. We tagged proteins\n",
      "with the fluorescent proteins Citrine or tdEos and expressed them in Caenorhabditis elegans, fixed\n",
      "the worms and embedded them in plastic. We imaged the tagged proteins from ultrathin sections using\n",
      "stimulated emission depletion (STED) microscopy or photoactivated localization microscopy (PALM).\n",
      "Fluorescence correlated with organelles imaged in electron micrographs from the same sections. We\n",
      "used these methods to localize histones, a mitochondrial protein and a presynaptic dense projection\n",
      "protein in electron micrographs.\n",
      "['STED microscopy', 'PALM', 'immunocytochemical electron microscopy']\n",
      "Elapsed time: 10.37 seconds\n",
      "-------------------\n",
      "21151136\n",
      "Stabilized imaging of immune surveillance in the mouse lung.\n",
      "Real-time imaging of cellular and subcellular dynamics in vascularized organs requires image\n",
      "resolution and image registration to be simultaneously optimized without perturbing normal\n",
      "physiology. This problem is particularly pronounced in the lung, in which cells may transit at\n",
      "speeds >1 mm s(-1) and in which normal respiration results in large-scale tissue movements that\n",
      "prevent image registration. Here we report video-rate, two-photon imaging of a physiologically\n",
      "intact preparation of the mouse lung that is stabilizing and nondisruptive. Using our method, we\n",
      "obtained evidence for differential trapping of T cells and neutrophils in mouse pulmonary\n",
      "capillaries, and observed neutrophil mobilization and dynamic vascular leak in response to stretch\n",
      "and inflammatory models of lung injury in mice. The system permits physiological measurement of\n",
      "motility rates of >1 mm s(-1), observation of detailed cellular morphology and could be applied in\n",
      "the future to other organs and tissues while maintaining intact physiology.\n",
      "['two-photon imaging', 'video-rate imaging', 'stabilizing imaging']\n",
      "Elapsed time: 10.10 seconds\n",
      "-------------------\n",
      "21186361\n",
      "A microfluidic array for large-scale ordering and orientation of embryos.\n",
      "Quantitative studies of embryogenesis require the ability to monitor pattern formation and\n",
      "morphogenesis in large numbers of embryos, at multiple time points and in diverse genetic\n",
      "backgrounds. We describe a simple approach that greatly facilitates these tasks for Drosophila\n",
      "melanogaster embryos, one of the most advanced models of developmental genetics. Based on passive\n",
      "hydrodynamics, we developed a microfluidic embryo-trap array that can be used to rapidly order and\n",
      "vertically orient hundreds of embryos. We describe the physical principles of the design and used\n",
      "this platform to quantitatively analyze multiple morphogen gradients in the dorsoventral patterning\n",
      "system. Our approach can also be used for live imaging and, with slight modifications, could be\n",
      "adapted for studies of pattern formation and morphogenesis in other model organisms.\n",
      "['microfluidic array', 'live imaging']\n",
      "Elapsed time: 26.92 seconds\n",
      "-------------------\n",
      "21217749\n",
      "Simultaneous two-photon calcium imaging at different depths with spatiotemporal multiplexing.\n",
      "In vivo two-photon calcium imaging would benefit from the use of multiple excitation beams to\n",
      "increase scanning speed, signal-to-noise ratio and field of view or to image different axial planes\n",
      "simultaneously. Using spatiotemporal multiplexing we circumvented light-scattering ambiguity\n",
      "inherent to deep-tissue multifocal two-photon microscopy. We demonstrate calcium imaging at multiple\n",
      "axial planes in the intact mouse brain to monitor network activity of ensembles of cortical neurons\n",
      "in three spatial dimensions.\n",
      "['two-photon calcium imaging',\n",
      " 'spatiotemporal multiplexing',\n",
      " 'deep-tissue multifocal two-photon microscopy']\n",
      "Elapsed time: 8.54 seconds\n",
      "-------------------\n",
      "21217750\n",
      "A photoprotection strategy for microsecond-resolution single-molecule fluorescence spectroscopy.\n",
      "Time resolution of current single-molecule fluorescence techniques is limited to milliseconds\n",
      "because of dye blinking and bleaching. Here we introduce a photoprotection strategy that affords\n",
      "microsecond resolution by combining efficient triplet quenching by oxygen and Trolox with minimized\n",
      "bleaching via the oxygen radical scavenger cysteamine. Using this approach we resolved the single-\n",
      "molecule microsecond conformational fluctuations of two proteins: the two-state folder α-spectrin\n",
      "SH3 domain and the ultrafast downhill folder BBL.\n",
      "['single-molecule fluorescence spectroscopy', 'triplet quenching', 'bleaching']\n",
      "Elapsed time: 7.91 seconds\n",
      "-------------------\n",
      "21240279\n",
      "Optogenetic manipulation of neural activity in freely moving Caenorhabditis elegans.\n",
      "We present an optogenetic illumination system capable of real-time light delivery with high spatial\n",
      "resolution to specified targets in freely moving Caenorhabditis elegans. A tracking microscope\n",
      "records the motion of an unrestrained worm expressing channelrhodopsin-2 or halorhodopsin in\n",
      "specific cell types. Image processing software analyzes the worm's position in each video frame,\n",
      "rapidly estimates the locations of targeted cells and instructs a digital micromirror device to\n",
      "illuminate targeted cells with laser light of the appropriate wavelengths to stimulate or inhibit\n",
      "activity. Because each cell in an unrestrained worm is a rapidly moving target, our system operates\n",
      "at high speed (∼50 frames per second) to provide high spatial resolution (∼30 μm). To test the\n",
      "accuracy, flexibility and utility of our system, we performed optogenetic analyses of the worm motor\n",
      "circuit, egg-laying circuit and mechanosensory circuits that have not been possible with previous\n",
      "methods.\n",
      "['tracking microscope',\n",
      " 'digital micromirror device',\n",
      " 'laser light',\n",
      " 'image processing software']\n",
      "Elapsed time: 10.25 seconds\n",
      "-------------------\n",
      "21240280\n",
      "A transgenic mouse for in vivo detection of endogenous labeled mRNA.\n",
      "Live-cell single mRNA imaging is a powerful tool but has been restricted in higher eukaryotes to\n",
      "artificial cell lines and reporter genes. We describe an approach that enables live-cell imaging of\n",
      "single endogenous labeled mRNA molecules transcribed in primary mammalian cells and tissue. We\n",
      "generated a knock-in mouse line with an MS2 binding site (MBS) cassette targeted to the 3'\n",
      "untranslated region of the essential β-actin gene. As β-actin-MBS was ubiquitously expressed, we\n",
      "could uniquely address endogenous mRNA regulation in any tissue or cell type. We simultaneously\n",
      "followed transcription from the β-actin alleles in real time and observed transcriptional bursting\n",
      "in response to serum stimulation with precise temporal resolution. We tracked single endogenous\n",
      "labeled mRNA particles being transported in primary hippocampal neurons. The MBS cassette also\n",
      "enabled high-sensitivity fluorescence in situ hybridization (FISH), allowing detection and\n",
      "localization of single β-actin mRNA molecules in various mouse tissues.\n",
      "['live-cell single mRNA imaging',\n",
      " 'fluorescence in situ hybridization (FISH)',\n",
      " 'high-sensitivity fluorescence in situ hybridization (FISH)']\n",
      "Elapsed time: 12.08 seconds\n",
      "-------------------\n",
      "21240281\n",
      "RNAi screening for fat regulatory genes with SRS microscopy.\n",
      "Identification of genes regulating fat accumulation is important for basic and medical research;\n",
      "genetic screening for those genes in Caenorhabditis elegans, a widely used model organism, requires\n",
      "in vivo quantification of lipids. We demonstrated RNA interference screening based on quantitative\n",
      "imaging of lipids with label-free stimulated Raman scattering (SRS) microscopy, which overcomes\n",
      "major limitations of coherent anti-Stokes Raman scattering microscopy. Our screening yielded eight\n",
      "new genetic regulators of fat storage.\n",
      "['SRS microscopy',\n",
      " 'label-free stimulated Raman scattering',\n",
      " 'coherent anti-Stokes Raman scattering']\n",
      "Elapsed time: 7.95 seconds\n",
      "-------------------\n",
      "21258339\n",
      "Micropilot: automation of fluorescence microscopy-based imaging for systems biology.\n",
      "Quantitative microscopy relies on imaging of large cell numbers but is often hampered by time-\n",
      "consuming manual selection of specific cells. The 'Micropilot' software automatically detects cells\n",
      "of interest and launches complex imaging experiments including three-dimensional multicolor time-\n",
      "lapse or fluorescence recovery after photobleaching in live cells. In three independent experimental\n",
      "setups this allowed us to statistically analyze biological processes in detail and is thus a\n",
      "powerful tool for systems biology.\n",
      "['fluorescence microscopy-based imaging',\n",
      " 'three-dimensional multicolor time-lapse',\n",
      " 'fluorescence recovery after photobleaching']\n",
      "Elapsed time: 7.61 seconds\n",
      "-------------------\n",
      "21278722\n",
      "Five challenges to bringing single-molecule force spectroscopy into living cells.\n",
      "In recent years, single-molecule force spectroscopy techniques have been used to study how inter-\n",
      "and intramolecular interactions control the assembly and functional state of biomolecular machinery\n",
      "in vitro. Here we discuss the problems and challenges that need to be addressed to bring these\n",
      "technologies into living cells and to learn how cellular machinery is controlled in vivo.\n",
      "['single-molecule force spectroscopy',\n",
      " 'inter- and intramolecular interactions',\n",
      " 'assembly and functional state of biomolecular machinery',\n",
      " 'cellular machinery']\n",
      "Elapsed time: 6.99 seconds\n",
      "-------------------\n",
      "21297620\n",
      "Visualizing a one-way protein encounter complex by ultrafast single-molecule mixing.\n",
      "We combined rapid microfluidic mixing with single-molecule fluorescence resonance energy transfer to\n",
      "study the folding kinetics of the intrinsically disordered human protein α-synuclein. The time-\n",
      "resolution of 0.2 ms revealed initial collapse of the unfolded protein induced by binding with lipid\n",
      "mimics and subsequent rapid formation of transient structures in the encounter complex. The method\n",
      "also enabled analysis of rapid dissociation and unfolding of weakly bound complexes triggered by\n",
      "massive dilution.\n",
      "['rapid microfluidic mixing',\n",
      " 'single-molecule fluorescence resonance energy transfer',\n",
      " 'time-resolution']\n",
      "Elapsed time: 7.32 seconds\n",
      "-------------------\n",
      "21297621\n",
      "Drosophila Brainbow: a recombinase-based fluorescence labeling technique to subdivide neural\n",
      "expression patterns.\n",
      "We developed a multicolor neuron labeling technique in Drosophila melanogaster that combines the\n",
      "power to specifically target different neural populations with the label diversity provided by\n",
      "stochastic color choice. This adaptation of vertebrate Brainbow uses recombination to select one of\n",
      "three epitope-tagged proteins detectable by immunofluorescence. Two copies of this construct yield\n",
      "six bright, separable colors. We used Drosophila Brainbow to study the innervation patterns of\n",
      "multiple antennal lobe projection neuron lineages in the same preparation and to observe the\n",
      "relative trajectories of individual aminergic neurons. Nerve bundles, and even individual neurites\n",
      "hundreds of micrometers long, can be followed with definitive color labeling. We traced motor\n",
      "neurons in the subesophageal ganglion and correlated them to neuromuscular junctions to identify\n",
      "their specific proboscis muscle targets. The ability to independently visualize multiple lineage or\n",
      "neuron projections in the same preparation greatly advances the goal of mapping how neurons connect\n",
      "into circuits.\n",
      "['recombinase-based fluorescence labeling',\n",
      " 'immunofluorescence',\n",
      " 'stochastic color choice']\n",
      "Elapsed time: 10.92 seconds\n",
      "-------------------\n",
      "21317909\n",
      "Confined activation and subdiffractive localization enables whole-cell PALM with genetically\n",
      "expressed probes.\n",
      "We demonstrate three-dimensional (3D) super-resolution microscopy in whole fixed cells using\n",
      "photoactivated localization microscopy (PALM). The use of the bright, genetically expressed\n",
      "fluorescent marker photoactivatable monomeric (m)Cherry (PA-mCherry1) in combination with near\n",
      "diffraction-limited confinement of photoactivation using two-photon illumination and 3D localization\n",
      "methods allowed us to investigate a variety of cellular structures at <50 nm lateral and <100 nm\n",
      "axial resolution. Compared to existing methods, we have substantially reduced excitation and\n",
      "bleaching of unlocalized markers, which allows us to use 3D PALM imaging with high localization\n",
      "density in thick structures. Our 3D localization algorithms, which are based on cross-correlation,\n",
      "do not rely on idealized noise models or specific optical configurations. This allows instrument\n",
      "design to be flexible. By generating appropriate fusion constructs and expressing them in Cos7\n",
      "cells, we could image invaginations of the nuclear membrane, vimentin fibrils, the mitochondrial\n",
      "network and the endoplasmic reticulum at depths of greater than 8 μm.\n",
      "['photoactivated localization microscopy (PALM',\n",
      " 'two-photon illumination',\n",
      " '3D localization methods',\n",
      " 'cross-correlation algorithms']\n",
      "Elapsed time: 11.99 seconds\n",
      "-------------------\n",
      "21336286\n",
      "Ultrahigh-resolution optical trap with single-fluorophore sensitivity.\n",
      "We present a single-molecule instrument that combines a time-shared ultrahigh-resolution dual\n",
      "optical trap interlaced with a confocal fluorescence microscope. In a demonstration experiment, we\n",
      "observed individual single fluorophore-labeled DNA oligonucleotides to bind and unbind complementary\n",
      "DNA suspended between two trapped beads. Simultaneous with the single-fluorophore detection, we\n",
      "clearly observed coincident angstrom-scale changes in tether extension. Fluorescence readout allowed\n",
      "us to determine the duplex melting rate as a function of force. The new instrument will enable the\n",
      "simultaneous measurement of angstrom-scale mechanical motion of individual DNA-binding proteins (for\n",
      "example, single-base-pair stepping of DNA translocases) along with the detection of properties of\n",
      "fluorescently labeled protein (for example, internal configuration).\n",
      "['ultrahigh-resolution dual optical trap',\n",
      " 'confocal fluorescence microscope',\n",
      " 'single-fluorophore sensitivity']\n",
      "Elapsed time: 9.66 seconds\n",
      "-------------------\n",
      "21378978\n",
      "Rapid three-dimensional isotropic imaging of living cells using Bessel beam plane illumination.\n",
      "A key challenge when imaging living cells is how to noninvasively extract the most spatiotemporal\n",
      "information possible. Unlike popular wide-field and confocal methods, plane-illumination microscopy\n",
      "limits excitation to the information-rich vicinity of the focal plane, providing effective optical\n",
      "sectioning and high speed while minimizing out-of-focus background and premature photobleaching.\n",
      "Here we used scanned Bessel beams in conjunction with structured illumination and/or two-photon\n",
      "excitation to create thinner light sheets (<0.5 μm) better suited to three-dimensional (3D)\n",
      "subcellular imaging. As demonstrated by imaging the dynamics of mitochondria, filopodia, membrane\n",
      "ruffles, intracellular vesicles and mitotic chromosomes in live cells, the microscope currently\n",
      "offers 3D isotropic resolution down to ∼0.3 μm, speeds up to nearly 200 image planes per second and\n",
      "the ability to noninvasively acquire hundreds of 3D data volumes from single living cells\n",
      "encompassing tens of thousands of image frames.\n",
      "['plane-illumination microscopy',\n",
      " 'wide-field microscopy',\n",
      " 'confocal microscopy',\n",
      " 'scanned Bessel beams',\n",
      " 'structured illumination',\n",
      " 'two-photon excitation']\n",
      "Elapsed time: 11.79 seconds\n",
      "-------------------\n",
      "21399636\n",
      "Two-color nanoscopy of three-dimensional volumes by 4Pi detection of stochastically switched\n",
      "fluorophores.\n",
      "We demonstrate three-dimensional (3D) super-resolution imaging of stochastically switched\n",
      "fluorophores distributed across whole cells. By evaluating the higher moments of the diffraction\n",
      "spot provided by a 4Pi detection scheme, single markers can be simultaneously localized with <10 nm\n",
      "precision in three dimensions in a layer of 650 nm thickness at an arbitrarily selected depth in the\n",
      "sample. By splitting the fluorescence light into orthogonal polarization states, our 4Pi setup also\n",
      "facilitates the 3D nanoscopy of multiple fluorophores. Offering a combination of multicolor\n",
      "recording, nanoscale resolution and extended axial depth, our method substantially advances the\n",
      "noninvasive 3D imaging of cells and of other transparent materials.\n",
      "['4Pi detection',\n",
      " 'stochastically switched fluorophores',\n",
      " 'nanoscopy',\n",
      " 'multicolor recording',\n",
      " 'noninvasive 3D imaging']\n",
      "Elapsed time: 9.28 seconds\n",
      "-------------------\n",
      "21399637\n",
      "Simultaneous assessment of rodent behavior and neurochemistry using a miniature positron emission\n",
      "tomograph.\n",
      "Positron emission tomography (PET) neuroimaging and behavioral assays in rodents are widely used in\n",
      "neuroscience. PET gives insights into the molecular processes of neuronal communication, and\n",
      "behavioral methods analyze the actions that are associated with such processes. These methods have\n",
      "not been directly integrated, because PET studies in animals have until now required general\n",
      "anesthesia to immobilize the subject, which precludes behavioral studies. We present a method for\n",
      "imaging awake, behaving rats with PET that allows the simultaneous study of behavior. Key components\n",
      "include the 'rat conscious animal PET' or RatCAP, a miniature portable PET scanner that is mounted\n",
      "on the rat's head, a mobility system that allows considerable freedom of movement, radiotracer\n",
      "administration techniques and methods for quantifying behavior and correlating the two data sets.\n",
      "The simultaneity of the PET and behavioral data provides a multidimensional tool for studying the\n",
      "functions of different brain regions and their molecular constituents.\n",
      "['PET', 'positron emission tomography', 'rat conscious animal PET']\n",
      "Elapsed time: 10.24 seconds\n",
      "-------------------\n",
      "21423192\n",
      "High-resolution Fourier-transform infrared chemical imaging with multiple synchrotron beams.\n",
      "Conventional Fourier-transform infrared (FTIR) microspectroscopic systems are limited by an\n",
      "inevitable trade-off between spatial resolution, acquisition time, signal-to-noise ratio (SNR) and\n",
      "sample coverage. We present an FTIR imaging approach that substantially extends current capabilities\n",
      "by combining multiple synchrotron beams with wide-field detection. This advance allows truly\n",
      "diffraction-limited high-resolution imaging over the entire mid-infrared spectrum with high chemical\n",
      "sensitivity and fast acquisition speed while maintaining high-quality SNR.\n",
      "['FTIR microspectroscopic systems',\n",
      " 'synchrotron beams',\n",
      " 'wide-field detection',\n",
      " 'diffraction-limited high-resolution imaging',\n",
      " 'mid-infrared spectrum']\n",
      "Elapsed time: 8.32 seconds\n",
      "-------------------\n",
      "21451510\n",
      "Development and applications of single-cell transcriptome analysis.\n",
      "Dissecting the relationship between genotype and phenotype is one of the central goals in\n",
      "developmental biology and medicine. Transcriptome analysis is a powerful strategy to connect\n",
      "genotype to phenotype of a cell. Here we review the history, progress, potential applications and\n",
      "future developments of single-cell transcriptome analysis. In combination with live cell imaging and\n",
      "lineage tracing, it will be possible to decipher the full gene expression network underlying\n",
      "physiological functions of individual cells in embryos and adults, and to study diseases.\n",
      "['live cell imaging', 'lineage tracing']\n",
      "Elapsed time: 6.99 seconds\n",
      "-------------------\n",
      "21451512\n",
      "Validating transcripts with probes and imaging technology.\n",
      "High-throughput gene expression screens provide a quantitative picture of the average expression\n",
      "signature of biological samples. However, the analysis of spatial gene expression patterns with\n",
      "single-cell resolution requires quantitative in situ measurement techniques. Here we describe recent\n",
      "technological advances in RNA fluorescence in situ hybridization (FISH) techniques that facilitate\n",
      "detection of individual fluorescently labeled mRNA molecules of practically any endogenous gene.\n",
      "These methods, which are based on advances in probe design, imaging technology and image processing,\n",
      "enable the absolute measurement of transcript abundance in individual cells with single-molecule\n",
      "resolution.\n",
      "['imaging technology',\n",
      " 'probes',\n",
      " 'RNA fluorescence in situ hybridization (FISH)',\n",
      " 'single-molecule resolution']\n",
      "Elapsed time: 7.98 seconds\n",
      "-------------------\n",
      "21451514\n",
      "Long-term single-cell imaging of mammalian stem cells.\n",
      "Continuous long-term single-cell observation provides insight into the molecular control of cell\n",
      "fate. This is particularly important for rare and heterogeneous populations of cells, such as\n",
      "mammalian stem cells. The current lack of usable off-the-shelf hardware and software for such\n",
      "experiments makes their implementation technically challenging. Here I discuss the need for\n",
      "continuous single-cell quantification to understand molecular cell fate control as well as\n",
      "organizational and technical solutions for long-term imaging and tracking of stem cells.\n",
      "['single-cell imaging',\n",
      " 'long-term imaging',\n",
      " 'continuous imaging',\n",
      " 'quantification']\n",
      "Elapsed time: 7.11 seconds\n",
      "-------------------\n",
      "21460824\n",
      "A genome-scale shRNA resource for transgenic RNAi in Drosophila.\n",
      "Existing transgenic RNAi resources in Drosophila melanogaster based on long double-stranded hairpin\n",
      "RNAs are powerful tools for functional studies, but they are ineffective in gene knockdown during\n",
      "oogenesis, an important model system for the study of many biological questions. We show that\n",
      "shRNAs, modeled on an endogenous microRNA, are extremely effective at silencing gene expression\n",
      "during oogenesis. We also describe our progress toward building a genome-wide shRNA resource.\n",
      "['shRNAs', 'long double-stranded hairpin RNAs']\n",
      "Elapsed time: 6.83 seconds\n",
      "-------------------\n",
      "21473015\n",
      "A versatile in vivo system for directed dissection of gene expression patterns.\n",
      "Tissue-specific gene expression using the upstream activating sequence (UAS)–GAL4 binary system has\n",
      "facilitated genetic dissection of many biological processes in Drosophila melanogaster. Refining\n",
      "GAL4 expression patterns or independently manipulating multiple cell populations using additional\n",
      "binary systems are common experimental goals. To simplify these processes, we developed a\n",
      "convertible genetic platform, the integrase swappable in vivo targeting element (InSITE) system.\n",
      "This approach allows GAL4 to be replaced with any other sequence, placing different genetic\n",
      "effectors under the control of the same regulatory elements. Using InSITE, GAL4 can be replaced with\n",
      "LexA or QF, allowing an expression pattern to be repurposed. GAL4 can also be replaced with GAL80 or\n",
      "split-GAL4 hemi-drivers, allowing intersectional approaches to refine expression patterns. The\n",
      "exchanges occur through efficient in vivo manipulations, making it possible to generate many swaps\n",
      "in parallel. This system is modular, allowing future genetic tools to be easily incorporated into\n",
      "the existing framework.\n",
      "['in vivo manipulations',\n",
      " 'GAL4 expression patterns',\n",
      " 'expression patterns',\n",
      " 'intersectional approaches',\n",
      " 'swaps']\n",
      "Elapsed time: 10.67 seconds\n",
      "-------------------\n",
      "21473020\n",
      "Imaging impedance.\n",
      "A label-free microscopy technique based on electrochemical impedance offers a new way of studying\n",
      "electrochemical processes in single cells.\n",
      "['electrochemical impedance', 'label-free microscopy']\n",
      "Elapsed time: 5.16 seconds\n",
      "-------------------\n",
      "21478861\n",
      "A parallel microfluidic flow cytometer for high-content screening.\n",
      "A parallel microfluidic cytometer (PMC) uses a high-speed scanning photomultiplier-based detector to\n",
      "combine low-pixel-count, one-dimensional imaging with flow cytometry. The 384 parallel flow channels\n",
      "of the PMC decouple count rate from signal-to-noise ratio. Using six-pixel one-dimensional images,\n",
      "we investigated protein localization in a yeast model for human protein misfolding diseases and\n",
      "demonstrated the feasibility of a nuclear-translocation assay in Chinese hamster ovary (CHO) cells\n",
      "expressing an NFκB-EGFP reporter.\n",
      "{'imaging_techniques': ['one-dimensional imaging',\n",
      "                        'flow cytometry',\n",
      "                        'scanning photomultiplier-based detector']}\n",
      "Elapsed time: 8.47 seconds\n",
      "-------------------\n",
      "21516115\n",
      "Adaptive informatics for multifactorial and high-content biological data.\n",
      "Whereas genomic data are universally machine-readable, data from imaging, multiplex biochemistry,\n",
      "flow cytometry and other cell- and tissue-based assays usually reside in loosely organized files of\n",
      "poorly documented provenance. This arises because the relational databases used in genomic research\n",
      "are difficult to adapt to rapidly evolving experimental designs, data formats and analytic\n",
      "algorithms. Here we describe an adaptive approach to managing experimental data based on\n",
      "semantically typed data hypercubes (SDCubes) that combine hierarchical data format 5 (HDF5) and\n",
      "extensible markup language (XML) file types. We demonstrate the application of SDCube-based storage\n",
      "using ImageRail, a software package for high-throughput microscopy. Experimental design and its day-\n",
      "to-day evolution, not rigid standards, determine how ImageRail data are organized in SDCubes. We\n",
      "applied ImageRail to collect and analyze drug dose-response landscapes in human cell lines at\n",
      "single-cell resolution.\n",
      "['high-throughput microscopy', 'ImageRail']\n",
      "Elapsed time: 9.69 seconds\n",
      "-------------------\n",
      "21527931\n",
      "Two-photon absorption properties of fluorescent proteins.\n",
      "Two-photon excitation of fluorescent proteins is an attractive approach for imaging living systems.\n",
      "Today researchers are eager to know which proteins are the brightest and what the best excitation\n",
      "wavelengths are. Here we review the two-photon absorption properties of a wide variety of\n",
      "fluorescent proteins, including new far-red variants, to produce a comprehensive guide to choosing\n",
      "the right fluorescent protein and excitation wavelength for two-photon applications.\n",
      "['two-photon absorption', 'two-photon excitation', 'fluorescent proteins']\n",
      "Elapsed time: 6.97 seconds\n",
      "-------------------\n",
      "21532582\n",
      "BrainAligner: 3D registration atlases of Drosophila brains.\n",
      "Analyzing Drosophila melanogaster neural expression patterns in thousands of three-dimensional image\n",
      "stacks of individual brains requires registering them into a canonical framework based on a fiducial\n",
      "reference of neuropil morphology. Given a target brain labeled with predefined landmarks, the\n",
      "BrainAligner program automatically finds the corresponding landmarks in a subject brain and maps it\n",
      "to the coordinate system of the target brain via a deformable warp. Using a neuropil marker (the\n",
      "antibody nc82) as a reference of the brain morphology and a target brain that is itself a\n",
      "statistical average of data for 295 brains, we achieved a registration accuracy of 2 μm on average,\n",
      "permitting assessment of stereotypy, potential connectivity and functional mapping of the adult\n",
      "fruit fly brain. We used BrainAligner to generate an image pattern atlas of 2954 registered brains\n",
      "containing 470 different expression patterns that cover all the major compartments of the fly brain.\n",
      "['deformable warp', 'nc82', 'three-dimensional image stacks']\n",
      "Elapsed time: 9.54 seconds\n",
      "-------------------\n",
      "21552254\n",
      "Fast, three-dimensional super-resolution imaging of live cells.\n",
      "We report super-resolution fluorescence imaging of live cells with high spatiotemporal resolution\n",
      "using stochastic optical reconstruction microscopy (STORM). By labeling proteins either directly or\n",
      "via SNAP tags with photoswitchable dyes, we obtained two-dimensional (2D) and 3D super-resolution\n",
      "images of living cells, using clathrin-coated pits and the transferrin cargo as model systems.\n",
      "Bright, fast-switching probes enabled us to achieve 2D imaging at spatial resolutions of ∼25 nm and\n",
      "temporal resolutions as fast as 0.5 s. We also demonstrated live-cell 3D super-resolution imaging.\n",
      "We obtained 3D spatial resolution of ∼30 nm in the lateral direction and ∼50 nm in the axial\n",
      "direction at time resolutions as fast as 1-2 s with several independent snapshots. Using\n",
      "photoswitchable dyes with distinct emission wavelengths, we also demonstrated two-color 3D super-\n",
      "resolution imaging in live cells. These imaging capabilities open a new window for characterizing\n",
      "cellular structures in living cells at the ultrastructural level.\n",
      "['STORM',\n",
      " 'two-dimensional (2D)',\n",
      " 'three-dimensional (3D)',\n",
      " 'super-resolution fluorescence imaging',\n",
      " 'live cells']\n",
      "Elapsed time: 11.45 seconds\n",
      "-------------------\n",
      "21602799\n",
      "High-throughput analysis of single hematopoietic stem cell proliferation in microfluidic cell\n",
      "culture arrays.\n",
      "Heterogeneity in cell populations poses a major obstacle to understanding complex biological\n",
      "processes. Here we present a microfluidic platform containing thousands of nanoliter-scale chambers\n",
      "suitable for live-cell imaging studies of clonal cultures of nonadherent cells with precise control\n",
      "of the conditions, capabilities for in situ immunostaining and recovery of viable cells. We show\n",
      "that this platform mimics conventional cultures in reproducing the responses of various types of\n",
      "primitive mouse hematopoietic cells with retention of their functional properties, as demonstrated\n",
      "by subsequent in vitro and in vivo (transplantation) assays of recovered cells. The automated medium\n",
      "exchange of this system made it possible to define when Steel factor stimulation is first required\n",
      "by adult hematopoietic stem cells in vitro as the point of exit from quiescence. This technology\n",
      "will offer many new avenues to interrogate otherwise inaccessible mechanisms governing mammalian\n",
      "cell growth and fate decisions.\n",
      "['live-cell imaging', 'in situ immunostaining', 'recovery of viable cells']\n",
      "Elapsed time: 9.92 seconds\n",
      "-------------------\n",
      "21642963\n",
      "Sharper low-power STED nanoscopy by time gating.\n",
      "Applying pulsed excitation together with time-gated detection improves the fluorescence on-off\n",
      "contrast in continuous-wave stimulated emission depletion (CW-STED) microscopy, thus revealing finer\n",
      "details in fixed and living cells using moderate light intensities. This method also enables super-\n",
      "resolution fluorescence correlation spectroscopy with CW-STED beams, as demonstrated by quantifying\n",
      "the dynamics of labeled lipid molecules in the plasma membrane of living cells.\n",
      "['STED microscopy',\n",
      " 'CW-STED microscopy',\n",
      " 'super-resolution fluorescence correlation spectroscopy']\n",
      "Elapsed time: 7.47 seconds\n",
      "-------------------\n",
      "21642964\n",
      "High-throughput behavioral analysis in C. elegans.\n",
      "We designed a real-time computer vision system, the Multi-Worm Tracker (MWT), which can\n",
      "simultaneously quantify the behavior of dozens of Caenorhabditis elegans on a Petri plate at video\n",
      "rates. We examined three traditional behavioral paradigms using this system: spontaneous movement on\n",
      "food, where the behavior changes over tens of minutes; chemotaxis, where turning events must be\n",
      "detected accurately to determine strategy; and habituation of response to tap, where the response is\n",
      "stochastic and changes over time. In each case, manual analysis or automated single-worm tracking\n",
      "would be tedious and time-consuming, but the MWT system allowed rapid quantification of behavior\n",
      "with minimal human effort. Thus, this system will enable large-scale forward and reverse genetic\n",
      "screens for complex behaviors.\n",
      "['spontaneous movement on food', 'chemotaxis', 'habituation of response to tap']\n",
      "Elapsed time: 8.69 seconds\n",
      "-------------------\n",
      "21642966\n",
      "Near-infrared branding efficiently correlates light and electron microscopy.\n",
      "The correlation of light and electron microscopy of complex tissues remains a major challenge. Here\n",
      "we report near-infrared branding (NIRB), which facilitates such correlation by using a pulsed, near-\n",
      "infrared laser to create defined fiducial marks in three dimensions in fixed tissue. As these marks\n",
      "are fluorescent and can be photo-oxidized to generate electron contrast, they can guide re-\n",
      "identification of previously imaged structures as small as dendritic spines by electron microscopy.\n",
      "['electron microscopy',\n",
      " 'near-infrared laser',\n",
      " 'fluorescent marks',\n",
      " 'photo-oxidation']\n",
      "Elapsed time: 7.08 seconds\n",
      "-------------------\n",
      "21666670\n",
      "Fluorogenic DNA sequencing in PDMS microreactors.\n",
      "We developed a multiplex sequencing-by-synthesis method combining terminal phosphate-labeled\n",
      "fluorogenic nucleotides (TPLFNs) and resealable polydimethylsiloxane (PDMS) microreactors. In the\n",
      "presence of phosphatase, primer extension by DNA polymerase using nonfluorescent TPLFNs generates\n",
      "fluorophores, which are confined in the microreactors and detected. We immobilized primed DNA\n",
      "templates in the microreactors, then sequentially introduced one of the four identically labeled\n",
      "TPLFNs, sealed the microreactors and recorded a fluorescence image after template-directed primer\n",
      "extension. With cycle times of <10 min, we demonstrate 30 base reads with ∼99% raw accuracy. Our\n",
      "'fluorogenic pyrosequencing' offers benefits of pyrosequencing, such as rapid turnaround, one-color\n",
      "detection and generation of native DNA, along with high detection sensitivity and simplicity of\n",
      "parallelization because simultaneous real-time monitoring of all microreactors is not required.\n",
      "['fluorogenic DNA sequencing',\n",
      " 'terminal phosphate-labeled fluorogenic nucleotides (TPLFNs)',\n",
      " 'polydimethylsiloxane (PDMS) microreactors',\n",
      " 'fluorescence imaging']\n",
      "Elapsed time: 12.23 seconds\n",
      "-------------------\n",
      "21725300\n",
      "Functional ultrasound imaging of the brain.\n",
      "We present functional ultrasound (fUS), a method for imaging transient changes in blood volume in\n",
      "the whole brain at better spatiotemporal resolution than with other functional brain imaging\n",
      "modalities. fUS uses plane-wave illumination at high frame rate and can measure blood volumes in\n",
      "smaller vessels than previous ultrasound methods. fUS identifies regions of brain activation and was\n",
      "used to image whisker-evoked cortical and thalamic responses and the propagation of epileptiform\n",
      "seizures in the rat brain.\n",
      "['functional ultrasound',\n",
      " 'plane-wave illumination',\n",
      " 'high frame rate',\n",
      " 'blood volume imaging']\n",
      "Elapsed time: 7.38 seconds\n",
      "-------------------\n",
      "21725301\n",
      "Two-photon polarization microscopy reveals protein structure and function.\n",
      "Membrane proteins are a large, diverse group of proteins, serving a multitude of cellular functions.\n",
      "They are difficult to study because of their requirement of a lipid membrane for function. Here we\n",
      "show that two-photon polarization microscopy can take advantage of the cell membrane requirement to\n",
      "yield insights into membrane protein structure and function, in living cells and organisms. The\n",
      "technique allows sensitive imaging of G-protein activation, changes in intracellular calcium\n",
      "concentration and other processes, and is not limited to membrane proteins. Conveniently, many\n",
      "suitable probes for two-photon polarization microscopy already exist.\n",
      "['two-photon polarization microscopy',\n",
      " 'G-protein activation',\n",
      " 'changes in intracellular calcium concentration']\n",
      "Elapsed time: 8.09 seconds\n",
      "-------------------\n",
      "21765409\n",
      "Deep and fast live imaging with two-photon scanned light-sheet microscopy.\n",
      "We implemented two-photon scanned light-sheet microscopy, combining nonlinear excitation with\n",
      "orthogonal illumination of light-sheet microscopy, and showed its excellent performance for in vivo,\n",
      "cellular-resolution, three-dimensional imaging of large biological samples. Live imaging of fruit\n",
      "fly and zebrafish embryos confirmed that the technique can be used to image up to twice deeper than\n",
      "with one-photon light-sheet microscopy and more than ten times faster than with point-scanning two-\n",
      "photon microscopy without compromising normal biology.\n",
      "['two-photon scanned light-sheet microscopy',\n",
      " 'nonlinear excitation',\n",
      " 'orthogonal illumination',\n",
      " 'live imaging',\n",
      " 'three-dimensional imaging']\n",
      "Elapsed time: 8.19 seconds\n",
      "-------------------\n",
      "21804536\n",
      "A photoswitchable orange-to-far-red fluorescent protein, PSmOrange.\n",
      "We report a photoswitchable monomeric Orange (PSmOrange) protein that is initially orange\n",
      "(excitation, 548 nm; emission, 565 nm) but becomes far-red (excitation, 636 nm; emission, 662 nm)\n",
      "after irradiation with blue-green light. Compared to its parental orange proteins, PSmOrange has\n",
      "greater brightness, faster maturation, higher photoconversion contrast and better photostability.\n",
      "The red-shifted spectra of both forms of PSmOrange enable its simultaneous use with cyan-to-green\n",
      "photoswitchable proteins to study four intracellular populations. Photoconverted PSmOrange has, to\n",
      "our knowledge, the most far-red excitation peak of all GFP-like fluorescent proteins, provides\n",
      "diffraction-limited and super-resolution imaging in the far-red light range, is optimally excited\n",
      "with common red lasers, and can be photoconverted subcutaneously in a mouse. PSmOrange\n",
      "photoswitching occurs via a two-step photo-oxidation process, which causes cleavage of the\n",
      "polypeptide backbone. The far-red fluorescence of photoconverted PSmOrange results from a new\n",
      "chromophore containing N-acylimine with a co-planar carbon-oxygen double bond.\n",
      "['fluorescent protein',\n",
      " 'photoswitchable',\n",
      " 'orange-to-far-red',\n",
      " 'diffraction-limited',\n",
      " 'super-resolution',\n",
      " 'common red lasers']\n",
      "Elapsed time: 13.26 seconds\n",
      "-------------------\n",
      "21822273\n",
      "Revealing off-target cleavage specificities of zinc-finger nucleases by in vitro selection.\n",
      "Engineered zinc-finger nucleases (ZFNs) are promising tools for genome manipulation, and determining\n",
      "off-target cleavage sites of these enzymes is of great interest. We developed an in vitro selection\n",
      "method that interrogates 10(11) DNA sequences for cleavage by active, dimeric ZFNs. The method\n",
      "revealed hundreds of thousands of DNA sequences, some present in the human genome, that can be\n",
      "cleaved in vitro by two ZFNs: CCR5-224 and VF2468, which target the endogenous human CCR5 and VEGFA\n",
      "genes, respectively. Analysis of identified sites in one cultured human cell line revealed\n",
      "CCR5-224-induced changes at nine off-target loci, though this remains to be tested in other relevant\n",
      "cell types. Similarly, we observed 31 off-target sites cleaved by VF2468 in cultured human cells.\n",
      "Our findings establish an energy compensation model of ZFN specificity in which excess binding\n",
      "energy contributes to off-target ZFN cleavage and suggest strategies for the improvement of future\n",
      "ZFN design.\n",
      "['in vitro selection', 'DNA sequences', 'cultured human cell line']\n",
      "Elapsed time: 12.44 seconds\n",
      "-------------------\n",
      "21878918\n",
      "Beyond the rainbow: new fluorescent proteins brighten the infrared scene.\n",
      "Two fluorescent proteins that emit in the far-red and infrared range for imaging applications in\n",
      "cells and in vivo are described.\n",
      "['fluorescent proteins', 'infrared scene']\n",
      "Elapsed time: 5.37 seconds\n",
      "-------------------\n",
      "21909101\n",
      "Antiviral restriction factor transgenesis in the domestic cat.\n",
      "Studies of the domestic cat have contributed to many scientific advances, including the present\n",
      "understanding of the mammalian cerebral cortex. A practical capability for cat transgenesis is\n",
      "needed to realize the distinctive potential of research on this neurobehaviorally complex,\n",
      "accessible species for advancing human and feline health. For example, humans and cats are afflicted\n",
      "with pandemic AIDS lentiviruses that are susceptible to species-specific restriction factors. Here\n",
      "we introduced genes encoding such a factor, rhesus macaque TRIMCyp, and eGFP, into the cat germline.\n",
      "The method establishes gamete-targeted transgenesis for the first time in a carnivore. We observed\n",
      "uniformly transgenic outcomes, widespread expression, no mosaicism and no F1 silencing. TRIMCyp\n",
      "transgenic cat lymphocytes resisted feline immunodeficiency virus replication. This capability to\n",
      "experimentally manipulate the genome of an AIDS-susceptible species can be used to test the\n",
      "potential of restriction factors for HIV gene therapy and to build models of other infectious and\n",
      "noninfectious diseases.\n",
      "['gamete-targeted transgenesis', 'eGFP']\n",
      "Elapsed time: 10.66 seconds\n",
      "-------------------\n",
      "21909102\n",
      "Miniaturized integration of a fluorescence microscope.\n",
      "The light microscope is traditionally an instrument of substantial size and expense. Its\n",
      "miniaturized integration would enable many new applications based on mass-producible, tiny\n",
      "microscopes. Key prospective usages include brain imaging in behaving animals for relating cellular\n",
      "dynamics to animal behavior. Here we introduce a miniature (1.9 g) integrated fluorescence\n",
      "microscope made from mass-producible parts, including a semiconductor light source and sensor. This\n",
      "device enables high-speed cellular imaging across ∼0.5 mm2 areas in active mice. This capability\n",
      "allowed concurrent tracking of Ca2+ spiking in >200 Purkinje neurons across nine cerebellar\n",
      "microzones. During mouse locomotion, individual microzones exhibited large-scale, synchronized Ca2+\n",
      "spiking. This is a mesoscopic neural dynamic missed by prior techniques for studying the brain at\n",
      "other length scales. Overall, the integrated microscope is a potentially transformative technology\n",
      "that permits distribution to many animals and enables diverse usages, such as portable diagnostics\n",
      "or microscope arrays for large-scale screens.\n",
      "['fluorescence microscope', 'light microscope']\n",
      "Elapsed time: 10.50 seconds\n",
      "-------------------\n",
      "21926998\n",
      "Probing protein heterogeneity in the plasma membrane using PALM and pair correlation analysis.\n",
      "Photoactivated localization microscopy (PALM) is a powerful approach for investigating protein\n",
      "organization, yet tools for quantitative, spatial analysis of PALM datasets are largely missing.\n",
      "Combining pair-correlation analysis with PALM (PC-PALM), we provide a method to analyze complex\n",
      "patterns of protein organization across the plasma membrane without determination of absolute\n",
      "protein numbers. The approach uses an algorithm to distinguish a single protein with multiple\n",
      "appearances from clusters of proteins. This enables quantification of different parameters of\n",
      "spatial organization, including the presence of protein clusters, their size, density and abundance\n",
      "in the plasma membrane. Using this method, we demonstrate distinct nanoscale organization of plasma-\n",
      "membrane proteins with different membrane anchoring and lipid partitioning characteristics in COS-7\n",
      "cells, and show dramatic changes in glycosylphosphatidylinositol (GPI)-anchored protein arrangement\n",
      "under varying perturbations. PC-PALM is thus an effective tool with broad applicability for analysis\n",
      "of protein heterogeneity and function, adaptable to other single-molecule strategies.\n",
      "['PALM', 'pair correlation analysis', 'PC-PALM']\n",
      "Elapsed time: 10.12 seconds\n",
      "-------------------\n",
      "21946668\n",
      "Rapid and robust generation of functional oligodendrocyte progenitor cells from epiblast stem cells.\n",
      "Myelin-related disorders such as multiple sclerosis and leukodystrophies, for which restoration of\n",
      "oligodendrocyte function would be an effective treatment, are poised to benefit greatly from stem\n",
      "cell biology. Progress in myelin repair has been constrained by difficulties in generating pure\n",
      "populations of oligodendrocyte progenitor cells (OPCs) in sufficient quantities. Pluripotent stem\n",
      "cells theoretically provide an unlimited source of OPCs, but current differentiation strategies are\n",
      "poorly reproducible and generate heterogenous populations of cells. Here we provide a platform for\n",
      "the directed differentiation of pluripotent mouse epiblast stem cells (EpiSCs) through defined\n",
      "developmental transitions into a pure population of highly expandable OPCs in 10 d. These OPCs\n",
      "robustly differentiate into myelinating oligodendrocytes in vitro and in vivo. Our results\n",
      "demonstrate that mouse pluripotent stem cells provide a pure population of myelinogenic\n",
      "oligodendrocytes and offer a tractable platform for defining the molecular regulation of\n",
      "oligodendrocyte development and drug screening.\n",
      "['defined developmental transitions',\n",
      " 'directed differentiation',\n",
      " 'heterogenous populations',\n",
      " 'in vitro differentiation',\n",
      " 'in vivo differentiation',\n",
      " 'myelin repair',\n",
      " 'oligodendrocyte development',\n",
      " 'oligodendrocyte progenitor cells',\n",
      " 'pluripotent stem cells',\n",
      " 'pure population']\n",
      "Elapsed time: 13.19 seconds\n",
      "-------------------\n",
      "21959136\n",
      "Optical sectioning microscopy with planar or structured illumination.\n",
      "A key requirement for performing three-dimensional (3D) imaging using optical microscopes is that\n",
      "they be capable of optical sectioning by distinguishing in-focus signal from out-of-focus\n",
      "background. Common techniques for fluorescence optical sectioning are confocal laser scanning\n",
      "microscopy and two-photon microscopy. But there is increasing interest in alternative optical\n",
      "sectioning techniques, particularly for applications involving high speeds, large fields of view or\n",
      "long-term imaging. In this Review, I examine two such techniques, based on planar illumination or\n",
      "structured illumination. The goal is to describe the advantages and disadvantages of these\n",
      "techniques.\n",
      "['confocal laser scanning microscopy',\n",
      " 'two-photon microscopy',\n",
      " 'planar illumination',\n",
      " 'structured illumination']\n",
      "Elapsed time: 8.42 seconds\n",
      "-------------------\n",
      "21983925\n",
      "Live-cell 3D super-resolution imaging in thick biological samples.\n",
      "We demonstrate three-dimensional (3D) super-resolution live-cell imaging through thick specimens\n",
      "(50-150 μm), by coupling far-field individual molecule localization with selective plane\n",
      "illumination microscopy (SPIM). The improved signal-to-noise ratio of selective plane illumination\n",
      "allows nanometric localization of single molecules in thick scattering specimens without activating\n",
      "or exciting molecules outside the focal plane. We report 3D super-resolution imaging of cellular\n",
      "spheroids.\n",
      "['far-field individual molecule localization',\n",
      " 'selective plane illumination microscopy (SPIM)',\n",
      " '3D super-resolution live-cell imaging']\n",
      "Elapsed time: 7.97 seconds\n",
      "-------------------\n",
      "21983960\n",
      "Proteomic and phosphoproteomic comparison of human ES and iPS cells.\n",
      "Combining high-mass-accuracy mass spectrometry, isobaric tagging and software for multiplexed,\n",
      "large-scale protein quantification, we report deep proteomic coverage of four human embryonic stem\n",
      "cell and four induced pluripotent stem cell lines in biological triplicate. This 24-sample\n",
      "comparison resulted in a very large set of identified proteins and phosphorylation sites in\n",
      "pluripotent cells. The statistical analysis afforded by our approach revealed subtle but\n",
      "reproducible differences in protein expression and protein phosphorylation between embryonic stem\n",
      "cells and induced pluripotent cells. Merging these results with RNA-seq analysis data, we found\n",
      "functionally related differences across each tier of regulation. We also introduce the Stem Cell-\n",
      "Omics Repository (SCOR), a resource to collate and display quantitative information across multiple\n",
      "planes of measurement, including mRNA, protein and post-translational modifications.\n",
      "['mass spectrometry', 'isobaric tagging', 'protein quantification']\n",
      "Elapsed time: 9.85 seconds\n",
      "-------------------\n",
      "21985008\n",
      "Cell type–specific channelrhodopsin-2 transgenic mice for optogenetic dissection of neural circuitry\n",
      "function.\n",
      "Optogenetic methods have emerged as powerful tools for dissecting neural circuit connectivity,\n",
      "function and dysfunction. We used a bacterial artificial chromosome (BAC) transgenic strategy to\n",
      "express the H134R variant of channelrhodopsin-2, ChR2(H134R), under the control of cell\n",
      "type–specific promoter elements. We performed an extensive functional characterization of the newly\n",
      "established VGAT-ChR2(H134R)-EYFP, ChAT-ChR2(H134R)-EYFP, Tph2-ChR2(H134R)-EYFP and\n",
      "Pvalb(H134R)-ChR2-EYFP BAC transgenic mouse lines and demonstrate the utility of these lines for\n",
      "precisely controlling action-potential firing of GABAergic, cholinergic, serotonergic and\n",
      "parvalbumin-expressing neuron subsets using blue light. This resource of cell type–specific\n",
      "ChR2(H134R) mouse lines will facilitate the precise mapping of neuronal connectivity and the\n",
      "dissection of the neural basis of behavior.\n",
      "['bacterial artificial chromosome (BAC)',\n",
      " 'blue light',\n",
      " 'functional characterization',\n",
      " 'optogenetic',\n",
      " 'precise mapping',\n",
      " 'serotonergic',\n",
      " 'GABAergic',\n",
      " 'cholinergic',\n",
      " 'parvalbumin-expressing',\n",
      " 'imaging']\n",
      "Elapsed time: 12.91 seconds\n",
      "-------------------\n",
      "22002026\n",
      "Super-resolution 3D microscopy of live whole cells using structured illumination.\n",
      "Three-dimensional (3D) structured-illumination microscopy (SIM) can double the lateral and axial\n",
      "resolution of a wide-field fluorescence microscope but has been too slow for live imaging. Here we\n",
      "apply 3D SIM to living samples and record whole cells at up to 5 s per volume for >50 time points\n",
      "with 120-nm lateral and 360-nm axial resolution. We demonstrate the technique by imaging\n",
      "microtubules in S2 cells and mitochondria in HeLa cells.\n",
      "['3D structured-illumination microscopy (SIM)',\n",
      " 'wide-field fluorescence microscope',\n",
      " 'whole cells']\n",
      "Elapsed time: 8.63 seconds\n",
      "-------------------\n",
      "22036746\n",
      "Characterizing RNA dynamics at atomic resolution using solution-state NMR spectroscopy.\n",
      "Many recently discovered noncoding RNAs do not fold into a single native conformation but sample\n",
      "many different conformations along their free-energy landscape to carry out their biological\n",
      "function. Here we review solution-state NMR techniques that measure the structural, kinetic and\n",
      "thermodynamic characteristics of RNA motions spanning picosecond to second timescales at atomic\n",
      "resolution, allowing unprecedented insights into the RNA dynamic structure landscape. From these\n",
      "studies a basic description of the RNA dynamic structure landscape is emerging, bringing new\n",
      "insights into how RNA structures change to carry out their function as well as applications in RNA-\n",
      "targeted drug discovery and RNA bioengineering.\n",
      "['solution-state NMR spectroscopy',\n",
      " 'structural NMR',\n",
      " 'kinetic NMR',\n",
      " 'thermodynamic NMR',\n",
      " 'RNA dynamic structure landscape']\n",
      "Elapsed time: 8.25 seconds\n",
      "-------------------\n",
      "22037703\n",
      "Automated protein-DNA interaction screening of Drosophila regulatory elements.\n",
      "Drosophila melanogaster has one of the best characterized metazoan genomes in terms of functionally\n",
      "annotated regulatory elements. To explore how these elements contribute to gene regulation, we need\n",
      "convenient tools to identify the proteins that bind to them. Here we describe the development and\n",
      "validation of a high-throughput yeast one-hybrid platform, which enables screening of DNA elements\n",
      "versus an array of full-length, sequence-verified clones containing over 85% of predicted Drosophila\n",
      "transcription factors. Using six well-characterized regulatory elements, we identified 33\n",
      "transcription factor-DNA interactions of which 27 were previously unidentified. To simultaneously\n",
      "validate these interactions and locate the binding sites of involved transcription factors, we\n",
      "implemented a powerful microfluidics-based approach that enabled us to retrieve DNA-occupancy data\n",
      "for each transcription factor throughout the respective target DNA elements. Finally, we\n",
      "biologically validated several interactions and identified two new regulators of sine oculis gene\n",
      "expression and hence eye development.\n",
      "['yeast one-hybrid platform', 'microfluidics-based approach']\n",
      "Elapsed time: 10.02 seconds\n",
      "-------------------\n",
      "22037704\n",
      "Visualizing mechanical tension across membrane receptors with a fluorescent sensor.\n",
      "We report a fluorescence-based turn-on sensor for mapping the mechanical strain exerted by specific\n",
      "cell-surface proteins in living cells. The sensor generates force maps with high spatial and\n",
      "temporal resolution using conventional fluorescence microscopy. We demonstrate the approach by\n",
      "mapping mechanical forces during the early stages of regulatory endocytosis of the ligand-activated\n",
      "epidermal growth factor receptor (EGFR).\n",
      "['fluorescence-based turn-on sensor', 'conventional fluorescence microscopy']\n",
      "Elapsed time: 6.83 seconds\n",
      "-------------------\n",
      "22037705\n",
      "Enhanced yeast one-hybrid assays for high-throughput gene-centered regulatory network mapping.\n",
      "A major challenge in systems biology is to understand the gene regulatory networks that drive\n",
      "development, physiology and pathology. Interactions between transcription factors and regulatory\n",
      "genomic regions provide the first level of gene control. Gateway-compatible yeast one-hybrid (Y1H)\n",
      "assays present a convenient method to identify and characterize the repertoire of transcription\n",
      "factors that can bind a DNA sequence of interest. To delineate genome-scale regulatory networks,\n",
      "however, large sets of DNA fragments need to be processed at high throughput and high coverage. Here\n",
      "we present enhanced Y1H (eY1H) assays that use a robotic mating platform with a set of improved Y1H\n",
      "reagents and automated readout quantification. We demonstrate that eY1H assays provide excellent\n",
      "coverage and identify interacting transcription factors for multiple DNA fragments in a short time.\n",
      "eY1H assays will be an important tool for mapping gene regulatory networks in Caenorhabditis elegans\n",
      "and other model organisms as well as in humans.\n",
      "['yeast one-hybrid assays',\n",
      " 'robotic mating platform',\n",
      " 'improved Y1H reagents',\n",
      " 'automated readout quantification']\n",
      "Elapsed time: 10.77 seconds\n",
      "-------------------\n",
      "22056676\n",
      "Evaluation of fluorophores for optimal performance in localization-based super-resolution imaging.\n",
      "One approach to super-resolution fluorescence imaging uses sequential activation and localization of\n",
      "individual fluorophores to achieve high spatial resolution. Essential to this technique is the\n",
      "choice of fluorescent probes; the properties of the probes, including photons per switching event,\n",
      "on-off duty cycle, photostability and number of switching cycles, largely dictate the quality of\n",
      "super-resolution images. Although many probes have been reported, a systematic characterization of\n",
      "the properties of these probes and their impact on super-resolution image quality has been described\n",
      "in only a few cases. Here we quantitatively characterized the switching properties of 26 organic\n",
      "dyes and directly related these properties to the quality of super-resolution images. This analysis\n",
      "provides guidelines for characterization of super-resolution probes and a resource for selecting\n",
      "probes based on performance. Our evaluation identified several photoswitchable dyes with good to\n",
      "excellent performance in four independent spectral ranges, with which we demonstrated low-cross-\n",
      "talk, four-color super-resolution imaging.\n",
      "['sequential activation and localization',\n",
      " 'photoswitchable dyes',\n",
      " 'low-cross-talk, four-color super-resolution imaging']\n",
      "Elapsed time: 11.19 seconds\n",
      "-------------------\n",
      "22081126\n",
      "Cyanine fluorophore derivatives with enhanced photostability.\n",
      "Fluorescence applications requiring high photostability often depend on the use of solution\n",
      "additives to enhance fluorophore performance. Here we demonstrate that the direct or proximal\n",
      "conjugation of cyclooctatetraene (COT), 4-nitrobenzyl alcohol (NBA) or Trolox to the cyanine\n",
      "fluorophore Cy5 dramatically enhanced fluorophore photostability without otherwise affecting its\n",
      "native spectral characteristics. Such conjugation is a powerful means of improving the robustness of\n",
      "fluorescence-based applications demanding long-lived, nonblinking fluorescence emission.\n",
      "['fluorescence applications',\n",
      " 'solution additives',\n",
      " 'photostability',\n",
      " 'cyclooctatetraene (COT)',\n",
      " '4-nitrobenzyl alcohol (NBA)',\n",
      " 'Trolox',\n",
      " 'cyanine fluorophore Cy5']\n",
      "Elapsed time: 9.17 seconds\n",
      "-------------------\n",
      "22101853\n",
      "Sensitive and specific single-molecule sequencing of 5-hydroxymethylcytosine.\n",
      "We describe strand-specific, base-resolution detection of 5-hydroxymethylcytosine (5-hmC) in genomic\n",
      "DNA with single-molecule sensitivity, combining a bioorthogonal, selective chemical labeling method\n",
      "of 5-hmC with single-molecule, real-time (SMRT) DNA sequencing. The chemical labeling not only\n",
      "allows affinity enrichment of 5-hmC-containing DNA fragments but also enhances the kinetic signal of\n",
      "5-hmC during SMRT sequencing. We applied the approach to sequence 5-hmC in a genomic DNA sample with\n",
      "high confidence.\n",
      "['SMRT sequencing', 'chemical labeling', 'affinity enrichment']\n",
      "Elapsed time: 8.50 seconds\n",
      "-------------------\n",
      "22120467\n",
      "Optical recording of action potentials in mammalian neurons using a microbial rhodopsin.\n",
      "Reliable optical detection of single action potentials in mammalian neurons has been one of the\n",
      "longest-standing challenges in neuroscience. Here we achieved this goal by using the endogenous\n",
      "fluorescence of a microbial rhodopsin protein, Archaerhodopsin 3 (Arch) from Halorubrum sodomense,\n",
      "expressed in cultured rat hippocampal neurons. This genetically encoded voltage indicator exhibited\n",
      "an approximately tenfold improvement in sensitivity and speed over existing protein-based voltage\n",
      "indicators, with a roughly linear twofold increase in brightness between -150 mV and +150 mV and a\n",
      "sub-millisecond response time. Arch detected single electrically triggered action potentials with an\n",
      "optical signal-to-noise ratio >10. Arch(D95N) lacked endogenous proton pumping and had 50% greater\n",
      "sensitivity than wild type but had a slower response (41 ms). Nonetheless, Arch(D95N) also resolved\n",
      "individual action potentials. Microbial rhodopsin-based voltage indicators promise to enable optical\n",
      "interrogation of complex neural circuits and electrophysiology in systems for which electrode-based\n",
      "techniques are challenging.\n",
      "['optical recording',\n",
      " 'microbial rhodopsin',\n",
      " 'Archaeorubrum sodomense',\n",
      " 'genetically encoded voltage indicator',\n",
      " 'single action potentials',\n",
      " 'endogenous fluorescence']\n",
      "Elapsed time: 12.88 seconds\n",
      "-------------------\n",
      "22127219\n",
      "Directed molecular evolution to design advanced red fluorescent proteins.\n",
      "Fluorescent proteins have become indispensable imaging tools for biomedical research. Continuing\n",
      "progress in fluorescence imaging, however, requires probes with additional colors and properties\n",
      "optimized for emerging techniques. Here we summarize strategies for development of red-shifted\n",
      "fluorescent proteins. We discuss possibilities for knowledge-based rational design based on the\n",
      "photochemistry of fluorescent proteins and the position of the chromophore in protein structure. We\n",
      "consider advances in library design by mutagenesis, protein expression systems and instrumentation\n",
      "for high-throughput screening that should yield improved fluorescent proteins for advanced imaging\n",
      "applications.\n",
      "['directed molecular evolution',\n",
      " 'fluorescence imaging',\n",
      " 'high-throughput screening',\n",
      " 'library design by mutagenesis',\n",
      " 'protein expression systems']\n",
      "Elapsed time: 8.39 seconds\n",
      "-------------------\n",
      "22138823\n",
      "mGRASP enables mapping mammalian synaptic connectivity with light microscopy.\n",
      "The GFP reconstitution across synaptic partners (GRASP) technique, based on functional\n",
      "complementation between two nonfluorescent GFP fragments, can be used to detect the location of\n",
      "synapses quickly, accurately and with high spatial resolution. The method has been previously\n",
      "applied in the nematode and the fruit fly but requires substantial modification for use in the\n",
      "mammalian brain. We developed mammalian GRASP (mGRASP) by optimizing transmembrane split-GFP\n",
      "carriers for mammalian synapses. Using in silico protein design, we engineered chimeric synaptic\n",
      "mGRASP fragments that were efficiently delivered to synaptic locations and reconstituted GFP\n",
      "fluorescence in vivo. Furthermore, by integrating molecular and cellular approaches with a\n",
      "computational strategy for the three-dimensional reconstruction of neurons, we applied mGRASP to\n",
      "both long-range circuits and local microcircuits in the mouse hippocampus and thalamocortical\n",
      "regions, analyzing synaptic distribution in single neurons and in dendritic compartments.\n",
      "['GFP reconstitution across synaptic partners (GRASP)',\n",
      " 'in silico protein design',\n",
      " 'mammalian GRASP (mGRASP)']\n",
      "Elapsed time: 10.81 seconds\n",
      "-------------------\n",
      "22138825\n",
      "Bayesian localization microscopy reveals nanoscale podosome dynamics.\n",
      "We describe a localization microscopy analysis method that is able to extract results in live cells\n",
      "using standard fluorescent proteins and xenon arc lamp illumination. Our Bayesian analysis of the\n",
      "blinking and bleaching (3B analysis) method models the entire dataset simultaneously as being\n",
      "generated by a number of fluorophores that may or may not be emitting light at any given time. The\n",
      "resulting technique allows many overlapping fluorophores in each frame and unifies the analysis of\n",
      "the localization from blinking and bleaching events. By modeling the entire dataset, we were able to\n",
      "use each reappearance of a fluorophore to improve the localization accuracy. The high performance of\n",
      "this technique allowed us to reveal the nanoscale dynamics of podosome formation and dissociation\n",
      "throughout an entire cell with a resolution of 50 nm on a 4-s timescale.\n",
      "['localization microscopy',\n",
      " 'Bayesian analysis',\n",
      " 'blinking and bleaching (3B) method',\n",
      " 'fluorescent proteins']\n",
      "Elapsed time: 9.41 seconds\n",
      "-------------------\n",
      "22205518\n",
      "Power tools for gene expression and clonal analysis in Drosophila.\n",
      "The development of two-component expression systems in Drosophila melanogaster, one of the most\n",
      "powerful genetic models, has allowed the precise manipulation of gene function in specific cell\n",
      "populations. These expression systems, in combination with site-specific recombination approaches,\n",
      "have also led to the development of new methods for clonal lineage analysis. We present a hands-on\n",
      "user guide to the techniques and approaches that have greatly increased resolution of genetic\n",
      "analysis in the fly, with a special focus on their application for lineage analysis. Our intention\n",
      "is to provide guidance and suggestions regarding which genetic tools are most suitable for\n",
      "addressing different developmental questions.\n",
      "['expression systems',\n",
      " 'site-specific recombination approaches',\n",
      " 'clonal lineage analysis']\n",
      "Elapsed time: 7.52 seconds\n",
      "-------------------\n",
      "22205519\n",
      "A guide to analysis of mouse energy metabolism.\n",
      "We present a consolidated view of the complexity and challenges of designing studies for measurement\n",
      "of energy metabolism in mouse models, including a practical guide to the assessment of energy\n",
      "expenditure, energy intake and body composition and statistical analysis thereof. We hope this guide\n",
      "will facilitate comparisons across studies and minimize spurious interpretations of data. We\n",
      "recommend that division of energy expenditure data by either body weight or lean body weight and\n",
      "that presentation of group effects as histograms should be replaced by plotting individual data and\n",
      "analyzing both group and body-composition effects using analysis of covariance (ANCOVA).\n",
      "['body composition',\n",
      " 'energy expenditure',\n",
      " 'energy intake',\n",
      " 'lean body weight',\n",
      " 'statistical analysis']\n",
      "Elapsed time: 7.81 seconds\n",
      "-------------------\n",
      "22231640\n",
      "Blotting protein complexes from native gels to electron microscopy grids.\n",
      "We report a simple and generic method for the direct transfer of protein complexes separated by\n",
      "native gel electrophoresis to electron microscopy grids. After transfer, sufficient material remains\n",
      "in the gel for identification and characterization by mass spectrometry. The method should\n",
      "facilitate higher-throughput single-particle analysis by substantially reducing the time needed for\n",
      "protein purification, as demonstrated for three complexes from Thermoplasma acidophilum.\n",
      "['electron microscopy', 'native gel electrophoresis']\n",
      "Elapsed time: 6.44 seconds\n",
      "-------------------\n",
      "22231641\n",
      "Fast two-photon in vivo imaging with three-dimensional random-access scanning in large tissue\n",
      "volumes.\n",
      "The understanding of brain computations requires methods that read out neural activity on different\n",
      "spatial and temporal scales. Following signal propagation and integration across a neuron and\n",
      "recording the concerted activity of hundreds of neurons pose distinct challenges, and the design of\n",
      "imaging systems has been mostly focused on tackling one of the two operations. We developed a high-\n",
      "resolution, acousto-optic two-photon microscope with continuous three-dimensional (3D) trajectory\n",
      "and random-access scanning modes that reaches near-cubic-millimeter scan range and can be adapted to\n",
      "imaging different spatial scales. We performed 3D calcium imaging of action potential\n",
      "backpropagation and dendritic spike forward propagation at sub-millisecond temporal resolution in\n",
      "mouse brain slices. We also performed volumetric random-access scanning calcium imaging of\n",
      "spontaneous and visual stimulation-evoked activity in hundreds of neurons of the mouse visual cortex\n",
      "in vivo. These experiments demonstrate the subcellular and network-scale imaging capabilities of our\n",
      "system.\n",
      "['two-photon microscopy',\n",
      " 'acousto-optic scanning',\n",
      " 'random-access scanning',\n",
      " 'calcium imaging',\n",
      " 'spontaneous activity',\n",
      " 'visual stimulation-evoked activity']\n",
      "Elapsed time: 11.17 seconds\n",
      "-------------------\n",
      "22231642\n",
      "Dual-objective STORM reveals three-dimensional filament organization in the actin cytoskeleton.\n",
      "By combining astigmatism imaging with a dual-objective scheme, we improved the image resolution of\n",
      "stochastic optical reconstruction microscopy (STORM) and obtained <10-nm lateral resolution and\n",
      "<20-nm axial resolution when imaging biological specimens. Using this approach, we resolved\n",
      "individual actin filaments in cells and revealed three-dimensional ultrastructure of the actin\n",
      "cytoskeleton. We observed two vertically separated layers of actin networks with distinct structural\n",
      "organizations in sheet-like cell protrusions.\n",
      "['astigmatism imaging',\n",
      " 'dual-objective scheme',\n",
      " 'stochastic optical reconstruction microscopy (STORM)',\n",
      " 'serial timing optimization by rotating the microscope (STORM)']\n",
      "Elapsed time: 8.32 seconds\n",
      "-------------------\n",
      "22245808\n",
      "Controlling airborne cues to study small animal navigation.\n",
      "Small animals such as nematodes and insects analyze airborne chemical cues to infer the direction of\n",
      "favorable and noxious locations. In these animals, the study of navigational behavior evoked by\n",
      "airborne cues has been limited by the difficulty of precisely controlling stimuli. We present a\n",
      "system that can be used to deliver gaseous stimuli in defined spatial and temporal patterns to\n",
      "freely moving small animals. We used this apparatus, in combination with machine-vision algorithms,\n",
      "to assess and quantify navigational decision making of Drosophila melanogaster larvae in response to\n",
      "ethyl acetate (a volatile attractant) and carbon dioxide (a gaseous repellant).\n",
      "['machine-vision algorithms',\n",
      " 'gaseous stimuli',\n",
      " 'defined spatial patterns',\n",
      " 'temporal patterns']\n",
      "Elapsed time: 7.93 seconds\n",
      "-------------------\n",
      "22245809\n",
      "Serial two-photon tomography for automated ex vivo mouse brain imaging.\n",
      "Here we describe an automated method, named serial two-photon (STP) tomography, that achieves high-\n",
      "throughput fluorescence imaging of mouse brains by integrating two-photon microscopy and tissue\n",
      "sectioning. STP tomography generates high-resolution datasets that are free of distortions and can\n",
      "be readily warped in three dimensions, for example, for comparing multiple anatomical tracings. This\n",
      "method opens the door to routine systematic studies of neuroanatomy in mouse models of human brain\n",
      "disorders.\n",
      "['two-photon microscopy', 'tissue sectioning']\n",
      "Elapsed time: 6.80 seconds\n",
      "-------------------\n",
      "22266542\n",
      "Chronic in vivo imaging in the mouse spinal cord using an implanted chamber.\n",
      "Understanding and treatment of spinal cord pathology is limited in part by a lack of time-lapse in\n",
      "vivo imaging strategies at the cellular level. We developed a chronically implanted spinal chamber\n",
      "and surgical procedure suitable for time-lapse in vivo multiphoton microscopy of mouse spinal cord\n",
      "without the need for repeat surgical procedures. We routinely imaged mice repeatedly for more than 5\n",
      "weeks postoperatively with up to ten separate imaging sessions and observed neither motor-function\n",
      "deficit nor neuropathology in the spinal cord as a result of chamber implantation. Using this\n",
      "chamber we quantified microglia and afferent axon dynamics after a laser-induced spinal cord lesion\n",
      "and observed massive microglia infiltration within 1 d along with a heterogeneous dieback of axon\n",
      "stumps. By enabling chronic imaging studies over timescales ranging from minutes to months, our\n",
      "method offers an ideal platform for understanding cellular dynamics in response to injury and\n",
      "therapeutic interventions.\n",
      "['multiphoton microscopy',\n",
      " 'time-lapse in vivo imaging',\n",
      " 'chronic in vivo imaging']\n",
      "Elapsed time: 10.21 seconds\n",
      "-------------------\n",
      "22266543\n",
      "An artery-specific fluorescent dye for studying neurovascular coupling.\n",
      "We demonstrate that Alexa Fluor 633 hydrazide (Alexa Fluor 633) selectively labels neocortical\n",
      "arteries and arterioles by binding to elastin fibers. We measured sensory stimulus-evoked arteriole\n",
      "dilation dynamics in mouse, rat and cat visual cortex using Alexa Fluor 633 together with neuronal\n",
      "activity using calcium indicators or blood flow using fluorescein dextran. Arteriole dilation\n",
      "decreased fluorescence recorded from immediately underlying neurons, representing a potential\n",
      "artifact during neuronal functional imaging experiments.\n",
      "['fluorescence imaging',\n",
      " 'calcium indicators',\n",
      " 'blood flow imaging',\n",
      " 'fluorescein dextran']\n",
      "Elapsed time: 8.22 seconds\n",
      "-------------------\n",
      "22286384\n",
      "In vivo protein crystallization opens new routes in structural biology.\n",
      "Protein crystallization in cells has been observed several times in nature. However, owing to their\n",
      "small size these crystals have not yet been used for X-ray crystallographic analysis. We prepared\n",
      "nano-sized in vivo-grown crystals of Trypanosoma brucei enzymes and applied the emerging method of\n",
      "free-electron laser-based serial femtosecond crystallography to record interpretable diffraction\n",
      "data. This combined approach will open new opportunities in structural systems biology.\n",
      "['X-ray crystallography',\n",
      " 'free-electron laser-based serial femtosecond crystallography',\n",
      " 'serial femtosecond crystallography']\n",
      "Elapsed time: 7.64 seconds\n",
      "-------------------\n",
      "22286385\n",
      "Computational modeling of cellular signaling processes embedded into dynamic spatial contexts.\n",
      "Cellular signaling processes depend on spatiotemporal distributions of molecular components.\n",
      "Multicolor, high-resolution microscopy permits detailed assessment of such distributions, providing\n",
      "input for fine-grained computational models that explore mechanisms governing dynamic assembly of\n",
      "multimolecular complexes and their role in shaping cellular behavior. However, it is challenging to\n",
      "incorporate into such models both complex molecular reaction cascades and the spatial localization\n",
      "of signaling components in dynamic cellular morphologies. Here we introduce an approach to address\n",
      "these challenges by automatically generating computational representations of complex reaction\n",
      "networks based on simple bimolecular interaction rules embedded into detailed, adaptive models of\n",
      "cellular morphology. Using examples of receptor-mediated cellular adhesion and signal-induced\n",
      "localized mitogen-activated protein kinase (MAPK) activation in yeast, we illustrate the capacity of\n",
      "this simulation technique to provide insights into cell biological processes. The modeling\n",
      "algorithms, implemented in a new version of the Simmune toolset, are accessible through intuitive\n",
      "graphical interfaces and programming libraries.\n",
      "{'imagingTechniques': ['multicolor high-resolution microscopy',\n",
      "                       'computational models',\n",
      "                       'simulation technique',\n",
      "                       'bimolecular interaction rules']}\n",
      "Elapsed time: 10.96 seconds\n",
      "-------------------\n",
      "22290187\n",
      "Immunolabeling artifacts and the need for live-cell imaging.\n",
      "Fluorescent fusion proteins have revolutionized examination of proteins in living cells. Still,\n",
      "studies using these proteins are met with criticism because proteins are modified and ectopically\n",
      "expressed, in contrast to immunofluorescence studies. However, introducing immunoreagents inside\n",
      "cells can cause protein extraction or relocalization, not reflecting the in vivo situation. Here we\n",
      "discuss pitfalls of immunofluorescence labeling that often receive little attention and argue that\n",
      "immunostaining experiments in dead, permeabilized cells should be complemented with live-cell\n",
      "imaging when scrutinizing protein localization.\n",
      "['immunofluorescence', 'live-cell imaging', 'dead, permeabilized cells']\n",
      "Elapsed time: 7.75 seconds\n",
      "-------------------\n",
      "22306808\n",
      "Polyubiquitin-sensor proteins reveal localization and linkage-type dependence of cellular ubiquitin\n",
      "signaling.\n",
      "Polyubiquitin chain topology is thought to direct modified substrates to specific fates, but this\n",
      "function-topology relationship is poorly understood, as are the dynamics and subcellular locations\n",
      "of specific polyubiquitin signals. Experimental access to these questions has been limited because\n",
      "linkage-specific inhibitors and in vivo sensors have been unavailable. Here we present a general\n",
      "strategy to track linkage-specific polyubiquitin signals in yeast and mammalian cells, and to probe\n",
      "their functions. We designed several high-affinity Lys63 polyubiquitin-binding proteins and\n",
      "demonstrate their specificity in vitro and in cells. We apply these tools as competitive inhibitors\n",
      "to dissect the polyubiquitin-linkage dependence of NF-κB activation in several cell types, inferring\n",
      "the essential role of Lys63 polyubiquitin for signaling via the IL-1β and TNF-related weak inducer\n",
      "of apoptosis (TWEAK) but not TNF-α receptors. We anticipate live-cell imaging, proteomic and\n",
      "biochemical applications for these tools and extension of the design strategy to other polymeric\n",
      "ubiquitin-like protein modifications.\n",
      "['live-cell imaging', 'proteomic', 'biochemical']\n",
      "Elapsed time: 11.27 seconds\n",
      "-------------------\n",
      "22306809\n",
      "Interspecies activity correlations reveal functional correspondence between monkey and human brain\n",
      "areas.\n",
      "Evolution-driven functional changes in the primate brain are typically assessed by aligning monkey\n",
      "and human activation maps using cortical surface expansion models. These models use putative\n",
      "homologous areas as registration landmarks, assuming they are functionally correspondent. For cases\n",
      "in which functional changes have occurred in an area, this assumption prohibits to reveal whether\n",
      "other areas may have assumed lost functions. Here we describe a method to examine functional\n",
      "correspondences across species. Without making spatial assumptions, we assessed similarities in\n",
      "sensory-driven functional magnetic resonance imaging responses between monkey (Macaca mulatta) and\n",
      "human brain areas by temporal correlation. Using natural vision data, we revealed regions for which\n",
      "functional processing has shifted to topologically divergent locations during evolution. We conclude\n",
      "that substantial evolution-driven functional reorganizations have occurred, not always consistent\n",
      "with cortical expansion processes. This framework for evaluating changes in functional architecture\n",
      "is crucial to building more accurate evolutionary models.\n",
      "['functional magnetic resonance imaging',\n",
      " 'temporal correlation',\n",
      " 'sensory-driven functional magnetic resonance imaging responses']\n",
      "Elapsed time: 10.01 seconds\n",
      "-------------------\n",
      "22343343\n",
      "A bioinformatics method identifies prominent off-targeted transcripts in RNAi screens.\n",
      "Because off-target effects hamper interpretation and validation of RNAi screen data, we developed a\n",
      "bioinformatics method, genome-wide enrichment of seed sequence matches (GESS), to identify candidate\n",
      "off-targeted transcripts in primary screening data. GESS analysis revealed a prominent off-targeted\n",
      "transcript in several screens, including MAD2 (MAD2L1) in a screen for genes required for the\n",
      "spindle assembly checkpoint. GESS analysis results can enhance the validation rate in RNAi screens.\n",
      "['GESS', 'RNAi screen data']\n",
      "Elapsed time: 6.81 seconds\n",
      "-------------------\n",
      "22373911\n",
      "OMERO: flexible, model-driven data management for experimental biology.\n",
      "Data-intensive research depends on tools that manage multidimensional, heterogeneous datasets. We\n",
      "built OME Remote Objects (OMERO), a software platform that enables access to and use of a wide range\n",
      "of biological data. OMERO uses a server-based middleware application to provide a unified interface\n",
      "for images, matrices and tables. OMERO's design and flexibility have enabled its use for light-\n",
      "microscopy, high-content-screening, electron-microscopy and even non-image-genotype data. OMERO is\n",
      "open-source software, available at http://openmicroscopy.org/.\n",
      "['images', 'matrices', 'tables']\n",
      "Elapsed time: 7.81 seconds\n",
      "-------------------\n",
      "22406855\n",
      "Controlling gene expression with the Q repressible binary expression system in Caenorhabditis\n",
      "elegans.\n",
      "We established a transcription-based binary gene expression system in Caenorhabditis elegans using\n",
      "the recently developed Q system. This system, derived from genes in Neurospora crassa, uses the\n",
      "transcriptional activator QF to induce the expression of target genes. Activation can be efficiently\n",
      "suppressed by the transcriptional repressor QS, and suppression can be relieved by the nontoxic\n",
      "small molecule quinic acid. We used QF, QS and quinic acid to achieve temporal and spatial control\n",
      "of transgene expression in various tissues in C. elegans. We also developed a split Q system, in\n",
      "which we separated QF into two parts encoding its DNA-binding and transcription-activation domains.\n",
      "Each domain showed negligible transcriptional activity when expressed alone, but expression of both\n",
      "reconstituted QF activity, providing additional combinatorial power to control gene expression.\n",
      "['Q system',\n",
      " 'quinic acid',\n",
      " 'temporal and spatial control of transgene expression']\n",
      "Elapsed time: 9.52 seconds\n",
      "-------------------\n",
      "22406856\n",
      "Synchronization of secretory protein traffic in populations of cells.\n",
      "To dissect secretory traffic, we developed the retention using selective hooks (RUSH) system. RUSH\n",
      "is a two-state assay based on the reversible interaction of a hook protein fused to core\n",
      "streptavidin and stably anchored in the donor compartment with a reporter protein of interest fused\n",
      "to streptavidin-binding peptide (SBP). Biotin addition causes a synchronous release of the reporter\n",
      "from the hook. Using the RUSH system, we analyzed different transport characteristics of various\n",
      "Golgi and plasma membrane reporters at physiological temperature in living cells. Using dual-color\n",
      "simultaneous live-cell imaging of two cargos, we observed intra- and post-Golgi segregation of cargo\n",
      "traffic, consistent with observation in other systems. We show preliminarily that the RUSH system is\n",
      "usable for automated screening. The system should help increase the understanding of the mechanisms\n",
      "of trafficking and enable screens for molecules that perturb pathological protein transport.\n",
      "['live-cell imaging',\n",
      " 'dual-color simultaneous live-cell imaging',\n",
      " 'automated screening']\n",
      "Elapsed time: 10.22 seconds\n",
      "-------------------\n",
      "22466793\n",
      "Integrated genetic and computation methods for in planta cytometry.\n",
      "We present the coupled use of specifically localized fluorescent gene markers and image processing\n",
      "for automated quantitative analysis of cell growth and genetic activity across living plant tissues.\n",
      "We used fluorescent protein markers to identify cells, create seeds and boundaries for the automatic\n",
      "segmentation of cell geometries and ratiometrically measure gene expression cell by cell in\n",
      "Arabidopsis thaliana.\n",
      "['fluorescent protein markers',\n",
      " 'image processing',\n",
      " 'automatic segmentation',\n",
      " 'ratiometric measurement']\n",
      "Elapsed time: 6.59 seconds\n",
      "-------------------\n",
      "22466794\n",
      "Direct visualization of specifically modified extracellular glycans in living animals.\n",
      "Modification patterns of heparan sulfate coordinate protein function in metazoans, yet in vivo\n",
      "imaging of such non-genetically encoded structures has been impossible. Here we report a transgenic\n",
      "method in Caenorhabditis elegans that allows direct live imaging of specific heparan sulfate\n",
      "modification patterns. This experimental approach reveals a dynamic and cell-specific heparan\n",
      "sulfate landscape and could in principle be adapted to visualize and analyze any extracellular\n",
      "molecule in vivo.\n",
      "['live imaging', 'transgenic method', 'direct visualization']\n",
      "Elapsed time: 7.04 seconds\n",
      "-------------------\n",
      "22466795\n",
      "Quantitative fluorescence labeling of aldehyde-tagged proteins for single-molecule imaging.\n",
      "A major hurdle for molecular mechanistic studies of many proteins is the lack of a general method\n",
      "for fluorescence labeling with high efficiency, specificity and speed. By incorporating an aldehyde\n",
      "motif genetically into a protein and improving the labeling kinetics substantially under mild\n",
      "conditions, we achieved fast, site-specific labeling of a protein with ∼100% efficiency while\n",
      "maintaining the biological function. We show that an aldehyde-tagged protein can be specifically\n",
      "labeled in cell extracts without protein purification and then can be used in single-molecule pull-\n",
      "down analysis. We also show the unique power of our method in single-molecule studies on the\n",
      "transient interactions and switching between two quantitatively labeled DNA polymerases on their\n",
      "processivity factor.\n",
      "['single-molecule imaging',\n",
      " 'fluorescence labeling',\n",
      " 'quantitative fluorescence labeling',\n",
      " 'pull-down analysis']\n",
      "Elapsed time: 9.57 seconds\n",
      "-------------------\n",
      "22484850\n",
      "Segregation of molecules at cell division reveals native protein localization.\n",
      "We introduce a nonintrusive method exploiting single-cell variability after cell division to\n",
      "validate protein localization. We found that Clp proteases, widely reported to form biologically\n",
      "relevant foci, were uniformly distributed in Escherichia coli cells, and that many commonly used\n",
      "fluorescent proteins caused severe mislocalization when fused to homo-oligomers. Retagging five\n",
      "other reportedly foci-forming proteins with the most monomeric fluorescent protein tested suggests\n",
      "that the foci were caused by the fluorescent tags.\n",
      "['Clp proteases', 'fluorescent proteins', 'retagging']\n",
      "Elapsed time: 7.06 seconds\n",
      "-------------------\n",
      "22522656\n",
      "An image analysis toolbox for high-throughput C. elegans assays.\n",
      "We present a toolbox for high-throughput screening of image-based Caenorhabditis elegans phenotypes.\n",
      "The image analysis algorithms measure morphological phenotypes in individual worms and are effective\n",
      "for a variety of assays and imaging systems. This WormToolbox is available through the open-source\n",
      "CellProfiler project and enables objective scoring of whole-worm high-throughput image-based assays\n",
      "of C. elegans for the study of diverse biological pathways that are relevant to human disease.\n",
      "['image-based assays',\n",
      " 'high-throughput screening',\n",
      " 'morphological phenotypes',\n",
      " 'whole-worm imaging']\n",
      "Elapsed time: 7.66 seconds\n",
      "-------------------\n",
      "22522657\n",
      "Faster STORM using compressed sensing.\n",
      "In super-resolution microscopy methods based on single-molecule switching, the rate of accumulating\n",
      "single-molecule activation events often limits the time resolution. Here we developed a sparse-\n",
      "signal recovery technique using compressed sensing to analyze images with highly overlapping\n",
      "fluorescent spots. This method allows an activated fluorophore density an order of magnitude higher\n",
      "than what conventional single-molecule fitting methods can handle. Using this method, we\n",
      "demonstrated imaging microtubule dynamics in living cells with a time resolution of 3 s.\n",
      "['compressed sensing',\n",
      " 'single-molecule switching',\n",
      " 'sparse-signal recovery',\n",
      " 'super-resolution microscopy']\n",
      "Elapsed time: 7.60 seconds\n",
      "-------------------\n",
      "22543348\n",
      "A simple, versatile method for GFP-based super-resolution microscopy via nanobodies.\n",
      "We developed a method to use any GFP-tagged construct in single-molecule super-resolution\n",
      "microscopy. By targeting GFP with small, high-affinity antibodies coupled to organic dyes, we\n",
      "achieved nanometer spatial resolution and minimal linkage error when analyzing microtubules, living\n",
      "neurons and yeast cells. We show that in combination with libraries encoding GFP-tagged proteins,\n",
      "virtually any known protein can immediately be used in super-resolution microscopy and that\n",
      "simplified labeling schemes allow high-throughput super-resolution imaging.\n",
      "['single-molecule super-resolution microscopy',\n",
      " 'organic dyes',\n",
      " 'nanometer spatial resolution',\n",
      " 'minimal linkage error',\n",
      " 'high-throughput super-resolution imaging']\n",
      "Elapsed time: 8.56 seconds\n",
      "-------------------\n",
      "22561987\n",
      "FMT-XCT: in vivo animal studies with hybrid fluorescence molecular tomography-X-ray computed\n",
      "tomography.\n",
      "The development of hybrid optical tomography methods to improve imaging performance has been\n",
      "suggested over a decade ago and has been experimentally demonstrated in animals and humans. Here we\n",
      "examined in vivo performance of a camera-based hybrid fluorescence molecular tomography (FMT) system\n",
      "for 360° imaging combined with X-ray computed tomography (XCT). Offering an accurately co-\n",
      "registered, information-rich hybrid data set, FMT-XCT has new imaging possibilities compared to\n",
      "stand-alone FMT and XCT. We applied FMT-XCT to a subcutaneous 4T1 tumor mouse model, an Aga2\n",
      "osteogenesis imperfecta model and a Kras lung cancer mouse model, using XCT information during FMT\n",
      "inversion. We validated in vivo imaging results against post-mortem planar fluorescence images of\n",
      "cryoslices and histology data. Besides offering concurrent anatomical and functional information,\n",
      "FMT-XCT resulted in the most accurate FMT performance to date. These findings indicate that addition\n",
      "of FMT optics into the XCT gantry may be a potent upgrade for small-animal XCT systems.\n",
      "['fluorescence molecular tomography',\n",
      " 'X-ray computed tomography',\n",
      " 'hybrid fluorescence molecular tomography-X-ray computed tomography',\n",
      " 'camera-based hybrid fluorescence molecular tomography',\n",
      " 'FMT-XCT']\n",
      "Elapsed time: 13.03 seconds\n",
      "-------------------\n",
      "22561988\n",
      "Automated whole-cell patch-clamp electrophysiology of neurons in vivo.\n",
      "Whole-cell patch-clamp electrophysiology of neurons is a gold-standard technique for high-fidelity\n",
      "analysis of the biophysical mechanisms of neural computation and pathology, but it requires great\n",
      "skill to perform. We have developed a robot that automatically performs patch clamping in vivo,\n",
      "algorithmically detecting cells by analyzing the temporal sequence of electrode impedance changes.\n",
      "We demonstrate good yield, throughput and quality of automated intracellular recording in mouse\n",
      "cortex and hippocampus.\n",
      "['patch-clamp electrophysiology', 'whole-cell patch-clamp electrophysiology']\n",
      "Elapsed time: 6.81 seconds\n",
      "-------------------\n",
      "22561989\n",
      "Simultaneous BOLD fMRI and fiber-optic calcium recording in rat neocortex.\n",
      "Functional magnetic resonance imaging (fMRI) based on blood oxygen level-dependent (BOLD) contrast\n",
      "is widely used for probing brain activity, but its relationship to underlying neural activity\n",
      "remains elusive. Here, we combined fMRI with fiber-optic recordings of fluorescent calcium indicator\n",
      "signals to investigate this relationship in rat somatosensory cortex. Electrical forepaw stimulation\n",
      "(1-10 Hz) evoked fast calcium signals of neuronal origin that showed frequency-dependent adaptation.\n",
      "Additionally, slower calcium signals occurred in astrocyte networks, as verified by astrocyte-\n",
      "specific staining and two-photon microscopy. Without apparent glia activation, we could predict BOLD\n",
      "responses well from simultaneously recorded fiber-optic signals, assuming an impulse response\n",
      "function and taking into account neuronal adaptation. In cases with glia activation, we uncovered\n",
      "additional prolonged BOLD signal components. Our findings highlight the complexity of fMRI BOLD\n",
      "signals, involving both neuronal and glial activity. Combined fMRI and fiber-optic recordings should\n",
      "help to clarify cellular mechanisms underlying BOLD signals.\n",
      "['fMRI', 'BOLD fMRI', 'fiber-optic recording', 'two-photon microscopy']\n",
      "Elapsed time: 11.52 seconds\n",
      "-------------------\n",
      "22581370\n",
      "Rational design of true monomeric and bright photoactivatable fluorescent proteins.\n",
      "Monomeric (m)Eos2 is an engineered photoactivatable fluorescent protein widely used for super-\n",
      "resolution microscopy. We show that mEos2 forms oligomers at high concentrations and forms\n",
      "aggregates when labeling membrane proteins, limiting its application as a fusion partner. We solved\n",
      "the crystal structure of tetrameric mEos2 and rationally designed improved versions, mEos3.1 and\n",
      "mEos3.2, that are truly monomeric, are brighter, mature faster and exhibit higher photon budget and\n",
      "label density.\n",
      "['crystal structure',\n",
      " 'super-resolution microscopy',\n",
      " 'photoactivatable fluorescent protein']\n",
      "Elapsed time: 7.75 seconds\n",
      "-------------------\n",
      "22581371\n",
      "M-Track: detecting short-lived protein-protein interactions in vivo.\n",
      "We developed a protein-proximity assay in yeast based on fusing a histone lysine methyltransferase\n",
      "onto a bait and its substrate onto a prey. Upon binding, the prey is stably methylated and detected\n",
      "by methylation-specific antibodies. We applied this approach to detect varying interaction\n",
      "affinities among proteins in a mitogen-activated protein kinase pathway and to detect short-lived\n",
      "interactions between protein phosphatase 2A and its substrates that have so far escaped direct\n",
      "detection.\n",
      "['histone lysine methyltransferase', 'methylation-specific antibodies']\n",
      "Elapsed time: 7.81 seconds\n",
      "-------------------\n",
      "22581372\n",
      "Resolution doubling in live, multicellular organisms via multifocal structured illumination\n",
      "microscopy.\n",
      "We demonstrate three-dimensional (3D) super-resolution in live multicellular organisms using\n",
      "structured illumination microscopy (SIM). Sparse multifocal illumination patterns generated by a\n",
      "digital micromirror device (DMD) allowed us to physically reject out-of-focus light, enabling 3D\n",
      "subdiffractive imaging in samples eightfold thicker than had been previously imaged with SIM. We\n",
      "imaged samples at one 2D image per second, at resolutions as low as 145 nm laterally and 400 nm\n",
      "axially. In addition to dual-labeled, whole fixed cells, we imaged GFP-labeled microtubules in live\n",
      "transgenic zebrafish embryos at depths >45 μm. We captured dynamic changes in the zebrafish lateral\n",
      "line primordium and observed interactions between myosin IIA and F-actin in cells encapsulated in\n",
      "collagen gels, obtaining two-color 4D super-resolution data sets spanning tens of time points and\n",
      "minutes without apparent phototoxicity. Our method uses commercially available parts and open-source\n",
      "software and is simpler than existing SIM implementations, allowing easy integration with wide-field\n",
      "microscopes.\n",
      "['SIM',\n",
      " 'DMD',\n",
      " 'whole fixed cells',\n",
      " 'GFP-labeled microtubules',\n",
      " 'live transgenic zebrafish embryos',\n",
      " 'collagen gels']\n",
      "Elapsed time: 12.50 seconds\n",
      "-------------------\n",
      "22609626\n",
      "Facile backbone structure determination of human membrane proteins by NMR spectroscopy.\n",
      "Although nearly half of today's major pharmaceutical drugs target human integral membrane proteins\n",
      "(hIMPs), only 30 hIMP structures are currently available in the Protein Data Bank, largely owing to\n",
      "inefficiencies in protein production. Here we describe a strategy for the rapid structure\n",
      "determination of hIMPs, using solution NMR spectroscopy with systematically labeled proteins\n",
      "produced via cell-free expression. We report new backbone structures of six hIMPs, solved in only 18\n",
      "months from 15 initial targets. Application of our protocols to an additional 135 hIMPs with\n",
      "molecular weight <30 kDa yielded 38 hIMPs suitable for structural characterization by solution NMR\n",
      "spectroscopy without additional optimization.\n",
      "{'methods': ['NMR spectroscopy',\n",
      "             'solution NMR spectroscopy',\n",
      "             'cell-free expression']}\n",
      "Elapsed time: 9.17 seconds\n",
      "-------------------\n",
      "22635062\n",
      "Unsupervised modeling of cell morphology dynamics for time-lapse microscopy.\n",
      "Analysis of cellular phenotypes in large imaging data sets conventionally involves supervised\n",
      "statistical methods, which require user-annotated training data. This paper introduces an\n",
      "unsupervised learning method, based on temporally constrained combinatorial clustering, for\n",
      "automatic prediction of cell morphology classes in time-resolved images. We applied the unsupervised\n",
      "method to diverse fluorescent markers and screening data and validated accurate classification of\n",
      "human cell phenotypes, demonstrating fully objective data labeling in image-based systems biology.\n",
      "['time-lapse microscopy',\n",
      " 'fluorescent markers',\n",
      " 'screening data',\n",
      " 'image-based systems biology']\n",
      "Elapsed time: 7.51 seconds\n",
      "-------------------\n",
      "22660739\n",
      "Multiview light-sheet microscope for rapid in toto imaging.\n",
      "We present a multiview selective-plane illumination microscope (MuVi-SPIM), comprising two detection\n",
      "and illumination objective lenses, that allows rapid in toto fluorescence imaging of biological\n",
      "specimens with subcellular resolution. The fixed geometrical arrangement of the imaging branches\n",
      "enables multiview data fusion in real time. The high speed of MuVi-SPIM allows faithful tracking of\n",
      "nuclei and cell shape changes, which we demonstrate through in toto imaging of the embryonic\n",
      "development of Drosophila melanogaster.\n",
      "['fluorescence imaging',\n",
      " 'selective-plane illumination microscope',\n",
      " 'multiview light-sheet microscope',\n",
      " 'real-time data fusion']\n",
      "Elapsed time: 8.77 seconds\n",
      "-------------------\n",
      "22660740\n",
      "Single-cell systems biology by super-resolution imaging and combinatorial labeling.\n",
      "Fluorescence microscopy is a powerful quantitative tool for exploring regulatory networks in single\n",
      "cells. However, the number of molecular species that can be measured simultaneously is limited by\n",
      "the spectral overlap between fluorophores. Here we demonstrate a simple but general strategy to\n",
      "drastically increase the capacity for multiplex detection of molecules in single cells by using\n",
      "optical super-resolution microscopy (SRM) and combinatorial labeling. As a proof of principle, we\n",
      "labeled mRNAs with unique combinations of fluorophores using fluorescence in situ hybridization\n",
      "(FISH), and resolved the sequences and combinations of fluorophores with SRM. We measured mRNA\n",
      "levels of 32 genes simultaneously in single Saccharomyces cerevisiae cells. These experiments\n",
      "demonstrate that combinatorial labeling and super-resolution imaging of single cells is a natural\n",
      "approach to bring systems biology into single cells.\n",
      "['fluorescence microscopy',\n",
      " 'optical super-resolution microscopy (SRM)',\n",
      " 'fluorescence in situ hybridization (FISH)']\n",
      "Elapsed time: 9.99 seconds\n",
      "-------------------\n",
      "22660741\n",
      "Quantitative high-speed imaging of entire developing embryos with simultaneous multiview light-sheet\n",
      "microscopy.\n",
      "Live imaging of large biological specimens is fundamentally limited by the short optical penetration\n",
      "depth of light microscopes. To maximize physical coverage, we developed the SiMView technology\n",
      "framework for high-speed in vivo imaging, which records multiple views of the specimen\n",
      "simultaneously. SiMView consists of a light-sheet microscope with four synchronized optical arms,\n",
      "real-time electronics for long-term sCMOS-based image acquisition at 175 million voxels per second,\n",
      "and computational modules for high-throughput image registration, segmentation, tracking and real-\n",
      "time management of the terabytes of multiview data recorded per specimen. We developed one-photon\n",
      "and multiphoton SiMView implementations and recorded cellular dynamics in entire Drosophila\n",
      "melanogaster embryos with 30-s temporal resolution throughout development. We furthermore performed\n",
      "high-resolution long-term imaging of the developing nervous system and followed neuroblast cell\n",
      "lineages in vivo. SiMView data sets provide quantitative morphological information even for fast\n",
      "global processes and enable accurate automated cell tracking in the entire early embryo.\n",
      "['light-sheet microscopy',\n",
      " 'sCMOS-based image acquisition',\n",
      " 'one-photon SiMView implementations',\n",
      " 'multiphoton SiMView implementations']\n",
      "Elapsed time: 11.53 seconds\n",
      "-------------------\n",
      "22688414\n",
      "Elastic volume reconstruction from series of ultra-thin microscopy sections.\n",
      "Anatomy of large biological specimens is often reconstructed from serially sectioned volumes imaged\n",
      "by high-resolution microscopy. We developed a method to reassemble a continuous volume from such\n",
      "large section series that explicitly minimizes artificial deformation by applying a global elastic\n",
      "constraint. We demonstrate our method on a series of transmission electron microscopy sections\n",
      "covering the entire 558-cell Caenorhabditis elegans embryo and a segment of the Drosophila\n",
      "melanogaster larval ventral nerve cord.\n",
      "['high-resolution microscopy', 'transmission electron microscopy']\n",
      "Elapsed time: 7.13 seconds\n",
      "-------------------\n",
      "22688415\n",
      "Rapid, accurate particle tracking by calculation of radial symmetry centers.\n",
      "I introduce an algorithm for subpixel localization of imaged objects based on an analytic, non-\n",
      "iterative calculation of the best-fit radial symmetry center. This approach yields tracking\n",
      "accuracies that are near theoretical limits, similarly to Gaussian fitting, but with orders-of-\n",
      "magnitude faster execution time, lower sensitivity to nearby particles and applicability to any\n",
      "radially symmetric intensity distribution. I demonstrate the method with several types of data,\n",
      "including super-resolution microscopy images.\n",
      "['subpixel localization',\n",
      " 'radial symmetry centers',\n",
      " 'Gaussian fitting',\n",
      " 'super-resolution microscopy']\n",
      "Elapsed time: 7.44 seconds\n",
      "-------------------\n",
      "22706672\n",
      "ViBE-Z: a framework for 3D virtual colocalization analysis in zebrafish larval brains.\n",
      "Precise three-dimensional (3D) mapping of a large number of gene expression patterns, neuronal types\n",
      "and connections to an anatomical reference helps us to understand the vertebrate brain and its\n",
      "development. We developed the Virtual Brain Explorer (ViBE-Z), a software that automatically maps\n",
      "gene expression data with cellular resolution to a 3D standard larval zebrafish (Danio rerio) brain.\n",
      "ViBE-Z enhances the data quality through fusion and attenuation correction of multiple confocal\n",
      "microscope stacks per specimen and uses a fluorescent stain of cell nuclei for image registration.\n",
      "It automatically detects 14 predefined anatomical landmarks for aligning new data with the reference\n",
      "brain. ViBE-Z performs colocalization analysis in expression databases for anatomical domains or\n",
      "subdomains defined by any specific pattern; here we demonstrate its utility for mapping neurons of\n",
      "the dopaminergic system. The ViBE-Z database, atlas and software are provided via a web interface.\n",
      "['confocal microscopy', 'image registration', 'anatomical reference']\n",
      "Elapsed time: 10.25 seconds\n",
      "-------------------\n",
      "22743769\n",
      "Why bioimage informatics matters.\n",
      "Driven by the importance of spatial and physical factors in cellular processes and the size and\n",
      "complexity of modern image data, computational analysis of biological imagery has become a vital\n",
      "emerging sub-discipline of bioinformatics and computer vision.\n",
      "['bioimage informatics',\n",
      " 'computational analysis',\n",
      " 'biological imagery',\n",
      " 'spatial and physical factors',\n",
      " 'cellular processes',\n",
      " 'modern image data']\n",
      "Elapsed time: 6.07 seconds\n",
      "-------------------\n",
      "22743770\n",
      "Current challenges in open-source bioimage informatics.\n",
      "We discuss the advantages and challenges of the open-source strategy in biological image analysis\n",
      "and argue that its full impact will not be realized without better support and recognition of\n",
      "software engineers' contributions to the biological sciences and more support of this development\n",
      "model from funders and institutions.\n",
      "['none']\n",
      "Elapsed time: 5.08 seconds\n",
      "-------------------\n",
      "22743772\n",
      "Fiji: an open-source platform for biological-image analysis.\n",
      "Fiji is a distribution of the popular open-source software ImageJ focused on biological-image\n",
      "analysis. Fiji uses modern software engineering practices to combine powerful software libraries\n",
      "with a broad range of scripting languages to enable rapid prototyping of image-processing\n",
      "algorithms. Fiji facilitates the transformation of new algorithms into ImageJ plugins that can be\n",
      "shared with end users through an integrated update system. We propose Fiji as a platform for\n",
      "productive collaboration between computer science and biology research communities.\n",
      "['ImageJ',\n",
      " 'modern software engineering practices',\n",
      " 'scripting languages',\n",
      " 'image-processing algorithms',\n",
      " 'integrated update system']\n",
      "Elapsed time: 6.92 seconds\n",
      "-------------------\n",
      "22743773\n",
      "BioImageXD: an open, general-purpose and high-throughput image-processing platform.\n",
      "BioImageXD puts open-source computer science tools for three-dimensional visualization and analysis\n",
      "into the hands of all researchers, through a user-friendly graphical interface tuned to the needs of\n",
      "biologists. BioImageXD has no restrictive licenses or undisclosed algorithms and enables publication\n",
      "of precise, reproducible and modifiable workflows. It allows simple construction of processing\n",
      "pipelines and should enable biologists to perform challenging analyses of complex processes. We\n",
      "demonstrate its performance in a study of integrin clustering in response to selected inhibitors.\n",
      "['3D visualization',\n",
      " 'image processing',\n",
      " 'general-purpose platform',\n",
      " 'high-throughput analysis']\n",
      "Elapsed time: 7.48 seconds\n",
      "-------------------\n",
      "22743774\n",
      "Icy: an open bioimage informatics platform for extended reproducible research.\n",
      "Current research in biology uses evermore complex computational and imaging tools. Here we describe\n",
      "Icy, a collaborative bioimage informatics platform that combines a community website for\n",
      "contributing and sharing tools and material, and software with a high-end visual programming\n",
      "framework for seamless development of sophisticated imaging workflows. Icy extends the reproducible\n",
      "research principles, by encouraging and facilitating the reusability, modularity, standardization\n",
      "and management of algorithms and protocols. Icy is free, open-source and available at\n",
      "http://icy.bioimageanalysis.org/.\n",
      "['visual programming framework',\n",
      " 'imaging workflows',\n",
      " 'reproducible research principles']\n",
      "Elapsed time: 7.44 seconds\n",
      "-------------------\n",
      "22743775\n",
      "Biological imaging software tools.\n",
      "Few technologies are more widespread in modern biological laboratories than imaging. Recent advances\n",
      "in optical technologies and instrumentation are providing hitherto unimagined capabilities. Almost\n",
      "all these advances have required the development of software to enable the acquisition, management,\n",
      "analysis and visualization of the imaging data. We review each computational step that biologists\n",
      "encounter when dealing with digital images, the inherent challenges and the overall status of\n",
      "available software for bioimage informatics, focusing on open-source options.\n",
      "['acquisition', 'management', 'analysis', 'visualization']\n",
      "Elapsed time: 6.94 seconds\n",
      "-------------------\n",
      "22751201\n",
      "Scanning angle interference microscopy reveals cell dynamics at the nanoscale.\n",
      "Emerging questions in cell biology necessitate nanoscale imaging in live cells. Here we present\n",
      "scanning angle interference microscopy, which is capable of localizing fluorescent objects with\n",
      "nanoscale precision along the optical axis in motile cellular structures. We use this approach to\n",
      "resolve nanotopographical features of the cell membrane and cytoskeleton as well as the temporal\n",
      "evolution, three-dimensional architecture and nanoscale dynamics of focal adhesion complexes.\n",
      "['scanning angle interference microscopy',\n",
      " 'live cell imaging',\n",
      " 'fluorescent object localization',\n",
      " 'nanoscale precision',\n",
      " 'optical axis']\n",
      "Elapsed time: 7.49 seconds\n",
      "-------------------\n",
      "22751204\n",
      "Targeted gene knockout by direct delivery of zinc-finger nuclease proteins.\n",
      "Zinc-finger nucleases (ZFNs) are versatile reagents that have redefined genome engineering.\n",
      "Realizing the full potential of this technology requires the development of safe and effective\n",
      "methods for delivering ZFNs into cells. We demonstrate the intrinsic cell-penetrating capabilities\n",
      "of the standard ZFN architecture and show that direct delivery of ZFNs as proteins leads to\n",
      "efficient endogenous gene disruption in various mammalian cell types with minimal off-target\n",
      "effects.\n",
      "['zinc-finger nucleases',\n",
      " 'direct delivery',\n",
      " 'cell penetrating capabilities',\n",
      " 'minimal off-target effects']\n",
      "Elapsed time: 7.38 seconds\n",
      "-------------------\n",
      "22772730\n",
      "Multicolor two-photon tissue imaging by wavelength mixing.\n",
      "We achieve simultaneous two-photon excitation of three chromophores with distinct absorption spectra\n",
      "using synchronized pulses from a femtosecond laser and an optical parametric oscillator. The two\n",
      "beams generate separate multiphoton processes, and their spatiotemporal overlap provides an\n",
      "additional two-photon excitation route, with submicrometer overlay of the color channels. We report\n",
      "volume and live multicolor imaging of 'Brainbow'-labeled tissues as well as simultaneous three-color\n",
      "fluorescence and third-harmonic imaging of fly embryos.\n",
      "['two-photon tissue imaging',\n",
      " 'wavelength mixing',\n",
      " 'simultaneous two-photon excitation',\n",
      " 'multicolor imaging',\n",
      " 'live multicolor imaging',\n",
      " 'volume imaging',\n",
      " 'third-harmonic imaging']\n",
      "Elapsed time: 8.58 seconds\n",
      "-------------------\n",
      "22796663\n",
      "A culture system to study oligodendrocyte myelination processes using engineered nanofibers.\n",
      "Current methods for studying central nervous system myelination necessitate permissive axonal\n",
      "substrates conducive to myelin wrapping by oligodendrocytes. We have developed a neuron-free culture\n",
      "system in which electron-spun nanofibers of varying sizes substitute for axons as a substrate for\n",
      "oligodendrocyte myelination, thereby allowing manipulation of the biophysical elements of axonal-\n",
      "oligodendroglial interactions. To investigate axonal regulation of myelination, this system\n",
      "effectively uncouples the role of molecular (inductive) cues from that of biophysical properties of\n",
      "the axon. We use this method to uncover the causation and sufficiency of fiber diameter in the\n",
      "initiation of concentric wrapping by rat oligodendrocytes. We also show that oligodendrocyte\n",
      "precursor cells display sensitivity to the biophysical properties of fiber diameter and initiate\n",
      "membrane ensheathment before differentiation. The use of nanofiber scaffolds will enable screening\n",
      "for potential therapeutic agents that promote oligodendrocyte differentiation and myelination and\n",
      "will also provide valuable insight into the processes involved in remyelination.\n",
      "[{'key': 'electron-spun nanofibers', 'value': 'nanofibers'},\n",
      " {'key': 'membrane ensheathment', 'value': 'ensheathment'},\n",
      " {'key': 'concentric wrapping', 'value': 'wrapping'}]\n",
      "Elapsed time: 12.73 seconds\n",
      "-------------------\n",
      "22863882\n",
      "Direct observation of mammalian cell growth and size regulation.\n",
      "We introduce a microfluidic system for simultaneously measuring single-cell mass and cell cycle\n",
      "progression over multiple generations. We use this system to obtain over 1,000 h of growth data from\n",
      "mouse lymphoblast and pro-B-cell lymphoid cell lines. Cell lineage analysis revealed a decrease in\n",
      "the growth rate variability at the G1-S phase transition, which suggests the presence of a growth\n",
      "rate threshold for maintaining size homeostasis.\n",
      "['microfluidic system',\n",
      " 'single-cell mass measurement',\n",
      " 'cell cycle progression measurement',\n",
      " 'growth rate variability measurement']\n",
      "Elapsed time: 7.34 seconds\n",
      "-------------------\n",
      "22886092\n",
      "Fractional proliferation: a method to deconvolve cell population dynamics from single-cell data.\n",
      "We present an integrated method that uses extended time-lapse automated imaging to quantify the\n",
      "dynamics of cell proliferation. Cell counts are fit with a quiescence-growth model that estimates\n",
      "rates of cell division, entry into quiescence and death. The model is constrained with rates\n",
      "extracted experimentally from the behavior of tracked single cells over time. We visualize the\n",
      "output of the analysis in fractional proliferation graphs, which deconvolve dynamic proliferative\n",
      "responses to perturbations into the relative contributions of dividing, quiescent (nondividing) and\n",
      "dead cells. The method reveals that the response of 'oncogene-addicted' human cancer cells to\n",
      "tyrosine kinase inhibitors is a composite of altered rates of division, death and entry into\n",
      "quiescence, a finding that challenges the notion that such cells simply die in response to oncogene-\n",
      "targeted therapy.\n",
      "['extended time-lapse automated imaging',\n",
      " 'quiescence-growth model',\n",
      " 'fractional proliferation graphs']\n",
      "Elapsed time: 9.92 seconds\n",
      "-------------------\n",
      "22902935\n",
      "Autonomous screening of C. elegans identifies genes implicated in synaptogenesis.\n",
      "Morphometric studies in multicellular organisms are generally performed manually because of the\n",
      "complexity of multidimensional features and lack of appropriate tools for handling these organisms.\n",
      "Here we present an integrated system that identifies and sorts Caenorhabditis elegans mutants with\n",
      "altered subcellular traits in real time without human intervention. We performed self-directed\n",
      "screens 100 times faster than manual screens and identified both genes and phenotypic classes\n",
      "involved in synapse formation.\n",
      "['real-time screening',\n",
      " 'automated morphometric analysis',\n",
      " 'self-directed screens',\n",
      " 'synaptogenesis']\n",
      "Elapsed time: 7.53 seconds\n",
      "-------------------\n",
      "22922501\n",
      "Site-specific genome editing in Plasmodium falciparum using engineered zinc-finger nucleases.\n",
      "Malaria afflicts over 200 million people worldwide, and its most lethal etiologic agent, Plasmodium\n",
      "falciparum, is evolving to resist even the latest-generation therapeutics. Efficient tools for\n",
      "genome-directed investigations of P. falciparum-induced pathogenesis, including drug-resistance\n",
      "mechanisms, are clearly required. Here we report rapid and targeted genetic engineering of this\n",
      "parasite using zinc-finger nucleases (ZFNs) that produce a double-strand break in a user-defined\n",
      "locus and trigger homology-directed repair. Targeting an integrated egfp locus, we obtained gene-\n",
      "deletion parasites with unprecedented speed (2 weeks), both with and without direct selection. ZFNs\n",
      "engineered against the parasite gene pfcrt, responsible for escape under chloroquine treatment,\n",
      "rapidly produced parasites that carried either an allelic replacement or a panel of specified point\n",
      "mutations. This method will enable a diverse array of genome-editing approaches to interrogate this\n",
      "human pathogen.\n",
      "['zinc-finger nucleases',\n",
      " 'double-strand break',\n",
      " 'homology-directed repair',\n",
      " 'gene-deletion parasites',\n",
      " 'allelic replacement',\n",
      " 'point mutations']\n",
      "Elapsed time: 11.97 seconds\n",
      "-------------------\n",
      "22930824\n",
      "The quest for quantitative microscopy.\n",
      "With the aid of informatics, microscopy is in the midst of a crucial evolution into a more\n",
      "quantitative and powerful technique.\n",
      "['informatics', 'microscopy']\n",
      "Elapsed time: 4.62 seconds\n",
      "-------------------\n",
      "22930834\n",
      "NIH Image to ImageJ: 25 years of image analysis.\n",
      "For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the\n",
      "analysis of scientific images. We discuss the origins, challenges and solutions of these two\n",
      "programs, and how their history can serve to advise and inform other software projects.\n",
      "['NIH Image', 'ImageJ', 'open tools']\n",
      "Elapsed time: 5.57 seconds\n",
      "-------------------\n",
      "22936170\n",
      "Imaging without lenses: achievements and remaining challenges of wide-field on-chip microscopy.\n",
      "We discuss unique features of lens-free computational imaging tools and report some of their\n",
      "emerging results for wide-field on-chip microscopy, such as the achievement of a numerical aperture\n",
      "(NA) of ∼0.8-0.9 across a field of view (FOV) of more than 20 mm(2) or an NA of ∼0.1 across a FOV of\n",
      "∼18 cm(2), which corresponds to an image with more than 1.5 gigapixels. We also discuss the current\n",
      "challenges that these computational on-chip microscopes face, shedding light on their future\n",
      "directions and applications.\n",
      "[' numerical aperture (NA)', 'field of view (FOV)', 'gigapixels']\n",
      "Elapsed time: 8.03 seconds\n",
      "-------------------\n",
      "22941363\n",
      "Ultrafast force-clamp spectroscopy of single molecules reveals load dependence of myosin working\n",
      "stroke.\n",
      "We describe a dual-trap force-clamp configuration that applies constant loads between a binding\n",
      "protein and an intermittently interacting biological polymer. The method has a measurement delay of\n",
      "only ∼10 μs, allows detection of interactions as brief as ∼100 μs and probes sub-nanometer\n",
      "conformational changes with a time resolution of tens of microseconds. We tested our method on\n",
      "molecular motors and DNA-binding proteins. We could apply constant loads to a single motor domain of\n",
      "myosin before its working stroke was initiated (0.2-1 ms), thus directly measuring its load\n",
      "dependence. We found that, depending on the applied load, myosin weakly interacted (<1 ms) with\n",
      "actin without production of movement, fully developed its working stroke or prematurely detached (<5\n",
      "ms), thus reducing the working stroke size with load. Our technique extends single-molecule force-\n",
      "clamp spectroscopy and opens new avenues for investigating the effects of forces on biological\n",
      "processes.\n",
      "[{'name': 'single-molecule force-clamp spectroscopy'},\n",
      " {'name': 'dual-trap force-clamp configuration'},\n",
      " {'name': 'measuring delay'},\n",
      " {'name': 'microsecond resolution'}]\n",
      "Elapsed time: 12.31 seconds\n",
      "-------------------\n",
      "22961245\n",
      "Improving FRET dynamic range with bright green and red fluorescent proteins.\n",
      "A variety of genetically encoded reporters use changes in fluorescence (or Förster) resonance energy\n",
      "transfer (FRET) to report on biochemical processes in living cells. The standard genetically encoded\n",
      "FRET pair consists of CFPs and YFPs, but many CFP-YFP reporters suffer from low FRET dynamic range,\n",
      "phototoxicity from the CFP excitation light and complex photokinetic events such as reversible\n",
      "photobleaching and photoconversion. We engineered two fluorescent proteins, Clover and mRuby2, which\n",
      "are the brightest green and red fluorescent proteins to date and have the highest Förster radius of\n",
      "any ratiometric FRET pair yet described. Replacement of CFP and YFP with these two proteins in\n",
      "reporters of kinase activity, small GTPase activity and transmembrane voltage significantly improves\n",
      "photostability, FRET dynamic range and emission ratio changes. These improvements enhance detection\n",
      "of transient biochemical events such as neuronal action-potential firing and RhoA activation in\n",
      "growth cones.\n",
      "['FRET',\n",
      " 'fluorescence resonance energy transfer',\n",
      " 'CFP',\n",
      " 'YFP',\n",
      " 'Clover',\n",
      " 'mRuby2',\n",
      " 'photostability',\n",
      " 'emission ratio changes']\n",
      "Elapsed time: 11.18 seconds\n",
      "-------------------\n",
      "22961247\n",
      "A general approach to break the concentration barrier in single-molecule imaging.\n",
      "Single-molecule fluorescence imaging is often incompatible with physiological protein\n",
      "concentrations, as fluorescence background overwhelms an individual molecule's signal. We solve this\n",
      "problem with a new imaging approach called PhADE (PhotoActivation, Diffusion and Excitation). A\n",
      "protein of interest is fused to a photoactivatable protein (mKikGR) and introduced to its surface-\n",
      "immobilized substrate. After photoactivation of mKikGR near the surface, rapid diffusion of the\n",
      "unbound mKikGR fusion out of the detection volume eliminates background fluorescence, whereupon the\n",
      "bound molecules are imaged. We labeled the eukaryotic DNA replication protein flap endonuclease 1\n",
      "with mKikGR and added it to replication-competent Xenopus laevis egg extracts. PhADE imaging of high\n",
      "concentrations of the fusion construct revealed its dynamics and micrometer-scale movements on\n",
      "individual, replicating DNA molecules. Because PhADE imaging is in principle compatible with any\n",
      "photoactivatable fluorophore, it should have broad applicability in revealing single-molecule\n",
      "dynamics and stoichiometry of macromolecular protein complexes at previously inaccessible\n",
      "fluorophore concentrations.\n",
      "['PhotoActivation', 'Diffusion', 'Excitation']\n",
      "Elapsed time: 11.56 seconds\n",
      "-------------------\n",
      "22983458\n",
      "Reversible labeling of native and fusion-protein motifs.\n",
      "The reversible covalent attachment of chemical probes to proteins has long been sought as a means to\n",
      "visualize and manipulate proteins. Here we demonstrate the full reversibility of post-translational\n",
      "custom pantetheine modification of Escherichia coli acyl carrier protein for visualization and\n",
      "functional studies. We use this iterative enzymatic methodology in vitro to reversibly label acyl\n",
      "carrier protein variants and apply these tools to NMR structural studies of protein-substrate\n",
      "interactions.\n",
      "['NMR', 'pantetheine modification', 'reversible labeling']\n",
      "Elapsed time: 7.07 seconds\n",
      "-------------------\n",
      "23023597\n",
      "A microfluidic device and computational platform for high-throughput live imaging of gene\n",
      "expression.\n",
      "To fully describe gene expression dynamics requires the ability to quantitatively capture expression\n",
      "in individual cells over time. Automated systems for acquiring and analyzing real-time images are\n",
      "needed to obtain unbiased data across many samples and conditions. We developed a microfluidics\n",
      "device, the RootArray, in which 64 Arabidopsis thaliana seedlings can be grown and their roots\n",
      "imaged by confocal microscopy over several days without manual intervention. To achieve high\n",
      "throughput, we decoupled acquisition from analysis. In the acquisition phase, we obtain images at\n",
      "low resolution and segment to identify regions of interest. Coordinates are communicated to the\n",
      "microscope to record the regions of interest at high resolution. In the analysis phase, we\n",
      "reconstruct three-dimensional objects from stitched high-resolution images and extract quantitative\n",
      "measurements from a virtual medial section of the root. We tracked hundreds of roots to capture\n",
      "detailed expression patterns of 12 transgenic reporter lines under different conditions.\n",
      "['confocal microscopy',\n",
      " 'high-resolution imaging',\n",
      " 'low-resolution imaging',\n",
      " 'virtual medial sectioning']\n",
      "Elapsed time: 9.86 seconds\n",
      "-------------------\n",
      "23042453\n",
      "DNA template strand sequencing of single-cells maps genomic rearrangements at high resolution.\n",
      "DNA rearrangements such as sister chromatid exchanges (SCEs) are sensitive indicators of genomic\n",
      "stress and instability, but they are typically masked by single-cell sequencing techniques. We\n",
      "developed Strand-seq to independently sequence parental DNA template strands from single cells,\n",
      "making it possible to map SCEs at orders-of-magnitude greater resolution than was previously\n",
      "possible. On average, murine embryonic stem (mES) cells exhibit eight SCEs, which are detected at a\n",
      "resolution of up to 23 bp. Strikingly, Strand-seq of 62 single mES cells predicts that the mm 9\n",
      "mouse reference genome assembly contains at least 17 incorrectly oriented segments totaling nearly\n",
      "1% of the genome. These misoriented contigs and fragments have persisted through several iterations\n",
      "of the mouse reference genome and have been difficult to detect using conventional sequencing\n",
      "techniques. The ability to map SCE events at high resolution and fine-tune reference genomes by\n",
      "Strand-seq dramatically expands the scope of single-cell sequencing.\n",
      "['Strand-seq', 'single-cell sequencing', 'parental DNA template strands']\n",
      "Elapsed time: 10.35 seconds\n",
      "-------------------\n",
      "23064517\n",
      "Magnetic nanoparticle-mediated massively parallel mechanical modulation of single-cell behavior.\n",
      "We report a technique for generating controllable, time-varying and localizable forces on arrays of\n",
      "cells in a massively parallel fashion. To achieve this, we grow magnetic nanoparticle-dosed cells in\n",
      "defined patterns on micromagnetic substrates. By manipulating and coalescing nanoparticles within\n",
      "cells, we apply localized nanoparticle-mediated forces approaching cellular yield tensions on the\n",
      "cortex of HeLa cells. We observed highly coordinated responses in cellular behavior, including the\n",
      "p21-activated kinase-dependent generation of active, leading edge-type filopodia and biasing of the\n",
      "metaphase plate during mitosis. The large sample size and rapid sample generation inherent to this\n",
      "approach allow the analysis of cells at an unprecedented rate: in a single experiment, potentially\n",
      "tens of thousands of cells can be stimulated for high statistical accuracy in measurements. This\n",
      "technique shows promise as a tool for both cell analysis and control.\n",
      "['micromagnetic substrates',\n",
      " 'nanoparticle-mediated forces',\n",
      " 'cellular yield tensions',\n",
      " 'filopodia',\n",
      " 'metaphase plate during mitosis']\n",
      "Elapsed time: 10.21 seconds\n",
      "-------------------\n",
      "23064519\n",
      "Automated velocity mapping of migrating cell populations (AVeMap).\n",
      "Characterizing the migration of a population of cells remains laborious and somewhat subjective.\n",
      "Advances in genetics and robotics allow researchers to perform many experiments in parallel, but\n",
      "analyzing the large sets of data remains a bottleneck. Here we describe a rapid, fully automated\n",
      "correlation-based method for cell migration analysis, compatible with standard video microscopy.\n",
      "This method allows for the computation of quantitative migration parameters via an extensive dynamic\n",
      "mapping of cell displacements.\n",
      "['video microscopy',\n",
      " 'correlation-based method',\n",
      " 'quantitative migration parameters']\n",
      "Elapsed time: 7.27 seconds\n",
      "-------------------\n",
      "23085613\n",
      "Staining and embedding the whole mouse brain for electron microscopy.\n",
      "The development of methods for imaging large contiguous volumes with the electron microscope could\n",
      "allow the complete mapping of a whole mouse brain at the single-axon level. We developed a method\n",
      "based on prolonged immersion that enables staining and embedding of the entire mouse brain with\n",
      "uniform myelin staining and a moderate preservation of the tissue's ultrastructure. We tested the\n",
      "ability to follow myelinated axons using serial block-face electron microscopy.\n",
      "['electron microscope',\n",
      " 'prolonged immersion',\n",
      " 'serial block-face electron microscopy']\n",
      "Elapsed time: 6.78 seconds\n",
      "-------------------\n",
      "23085614\n",
      "Membrane-protein binding measured with solution-phase plasmonic nanocube sensors.\n",
      "We describe a solution-phase sensor of lipid-protein binding based on localized surface plasmon\n",
      "resonance (LSPR) of silver nanocubes. When silica-coated nanocubes are mixed in a suspension of\n",
      "lipid vesicles, supported membranes spontaneously assemble on their surfaces. Using a standard\n",
      "laboratory spectrophotometer, we calibrated the LSPR peak shift due to protein binding to the\n",
      "membrane surface and then characterized the lipid-binding specificity of a pleckstrin homology\n",
      "domain protein.\n",
      "['localized surface plasmon resonance (LSPR)',\n",
      " 'silver nanocubes',\n",
      " 'spectrophotometer']\n",
      "Elapsed time: 7.83 seconds\n",
      "-------------------\n",
      "23103879\n",
      "Phase-gradient microscopy in thick tissue with oblique back-illumination.\n",
      "Phase-contrast techniques, such as differential interference contrast microscopy, are widely used to\n",
      "obtain morphological images of unstained biological samples. The transillumination geometry required\n",
      "for these techniques restricts their application to thin samples. We introduce oblique back-\n",
      "illumination microscopy, a method of collecting en face phase-gradient images of thick scattering\n",
      "samples, enabling near-video-rate in vivo phase imaging with a miniaturized probe suitable for\n",
      "endoscopy.\n",
      "['differential interference contrast microscopy',\n",
      " 'oblique back-illumination microscopy',\n",
      " 'en face phase-gradient images']\n",
      "Elapsed time: 7.28 seconds\n",
      "-------------------\n",
      "23103881\n",
      "Ultrabright photoactivatable fluorophores created by reductive caging.\n",
      "Sub-diffraction-limit imaging can be achieved by sequential localization of photoactivatable\n",
      "fluorophores, for which the image resolution depends on the number of photons detected per\n",
      "localization. We report a strategy for fluorophore caging that creates photoactivatable probes with\n",
      "high photon yields. Upon photoactivation, these probes can provide 10(4)-10(6) photons per\n",
      "localization and allow imaging of fixed samples with resolutions of several nanometers. This\n",
      "strategy can be applied to many fluorophores across the visible spectrum.\n",
      "[' photoactivatable fluorophores',\n",
      " ' sub-diffraction-limit imaging',\n",
      " ' sequential localization',\n",
      " ' reductive caging']\n",
      "Elapsed time: 8.04 seconds\n",
      "-------------------\n",
      "23142870\n",
      "Membrane-protein structure determination by solid-state NMR spectroscopy of microcrystals.\n",
      "Membrane proteins are largely underrepresented among available atomic-resolution structures. The use\n",
      "of detergents in protein purification procedures hinders the formation of well-ordered crystals for\n",
      "X-ray crystallography and leads to slower molecular tumbling, impeding the application of solution-\n",
      "state NMR. Solid-state magic-angle spinning NMR spectroscopy is an emerging method for membrane-\n",
      "protein structural biology that can overcome these technical problems. Here we present the solid-\n",
      "state NMR structure of the transmembrane domain of the Yersinia enterocolitica adhesin A (YadA). The\n",
      "sample was derived from crystallization trials that yielded only poorly diffracting microcrystals.\n",
      "We solved the structure using a single, uniformly (13)C- and (15)N-labeled sample. In addition,\n",
      "solid-state NMR allowed us to acquire information on the flexibility and mobility of parts of the\n",
      "structure, which, in combination with evolutionary conservation information, presents new insights\n",
      "into the autotransport mechanism of YadA.\n",
      "['solid-state NMR spectroscopy', 'X-ray crystallography', 'solution-state NMR']\n",
      "Elapsed time: 11.35 seconds\n",
      "-------------------\n",
      "23142872\n",
      "Systematic reconstruction of RNA functional motifs with high-throughput microfluidics.\n",
      "We present RNA-mechanically induced trapping of molecular interactions (RNA-MITOMI), a microfluidic\n",
      "platform that allows integrated synthesis and functional assays for programmable RNA libraries. The\n",
      "interaction of a comprehensive library of RNA mutants with stem-loop-binding protein precisely\n",
      "defined the RNA structural and sequence features that govern affinity. The functional motif\n",
      "reconstructed in a single experiment on our platform uncovers new binding specificities and enriches\n",
      "interpretation of phylogenetic data.\n",
      "['microfluidics',\n",
      " 'synthesis',\n",
      " 'functional assays',\n",
      " 'programmable RNA libraries',\n",
      " 'stem-loop-binding protein']\n",
      "Elapsed time: 7.47 seconds\n",
      "-------------------\n",
      "23142873\n",
      "Two-photon optogenetics of dendritic spines and neural circuits.\n",
      "We demonstrate a two-photon optogenetic method that generates action potentials in neurons with\n",
      "single-cell precision, using the red-shifted opsin C1V1(T). We applied the method to optically map\n",
      "synaptic circuits in mouse neocortical brain slices and to activate small dendritic regions and\n",
      "individual spines. Using a spatial light modulator, we split the laser beam onto several neurons and\n",
      "performed simultaneous optogenetic activation of selected neurons in three dimensions.\n",
      "['two-photon optogenetics',\n",
      " 'red-shifted opsin C1V1(T)',\n",
      " 'spatial light modulator']\n",
      "Elapsed time: 7.35 seconds\n",
      "-------------------\n",
      "23169303\n",
      "Two-photon optogenetic toolbox for fast inhibition, excitation and bistable modulation.\n",
      "Optogenetics with microbial opsin genes has enabled high-speed control of genetically specified cell\n",
      "populations in intact tissue. However, it remains a challenge to independently control subsets of\n",
      "cells within the genetically targeted population. Although spatially precise excitation of target\n",
      "molecules can be achieved using two-photon laser-scanning microscopy (TPLSM) hardware, the\n",
      "integration of two-photon excitation with optogenetics has thus far required specialized equipment\n",
      "or scanning and has not yet been widely adopted. Here we take a complementary approach, developing\n",
      "opsins with custom kinetic, expression and spectral properties uniquely suited to scan times typical\n",
      "of the raster approach that is ubiquitous in TPLSMlaboratories. We use a range of culture, slice and\n",
      "mammalian in vivo preparations to demonstrate the versatility of this toolbox, and we quantitatively\n",
      "map parameter space for fast excitation, inhibition and bistable control. Together these advances\n",
      "may help enable broad adoption of integrated optogenetic and TPLSMtechnologies across experimental\n",
      "fields and systems.\n",
      "['two-photon laser-scanning microscopy (TPLSM)',\n",
      " 'raster approach',\n",
      " 'optogenetics',\n",
      " 'microbial opsin genes']\n",
      "Elapsed time: 11.31 seconds\n",
      "-------------------\n",
      "23202434\n",
      "Conversion of human fibroblasts to angioblast-like progenitor cells.\n",
      "Lineage conversion of one somatic cell type to another is an attractive approach for generating\n",
      "specific human cell types. Lineage conversion can be direct, in the absence of proliferation and\n",
      "multipotent progenitor generation, or indirect, by the generation of expandable multipotent\n",
      "progenitor states. We report the development of a reprogramming methodology in which cells\n",
      "transition through a plastic intermediate state, induced by brief exposure to reprogramming factors,\n",
      "followed by differentiation. We use this approach to convert human fibroblasts to mesodermal\n",
      "progenitor cells, including by non-integrative approaches. These progenitor cells demonstrated\n",
      "bipotent differentiation potential and could generate endothelial and smooth muscle lineages.\n",
      "Differentiated endothelial cells exhibited neo-angiogenesis and anastomosis in vivo. This\n",
      "methodology for indirect lineage conversion to angioblast-like cells adds to the armamentarium of\n",
      "reprogramming approaches aimed at the study and treatment of ischemic pathologies.\n",
      "['reprogramming factors',\n",
      " 'brief exposure',\n",
      " 'differentiation',\n",
      " 'non-integrative approaches',\n",
      " 'neo-angiogenesis',\n",
      " 'anastomosis']\n",
      "Elapsed time: 10.31 seconds\n",
      "-------------------\n",
      "23223154\n",
      "Fast multicolor 3D imaging using aberration-corrected multifocus microscopy.\n",
      "Conventional acquisition of three-dimensional (3D) microscopy data requires sequential z scanning\n",
      "and is often too slow to capture biological events. We report an aberration-corrected multifocus\n",
      "microscopy method capable of producing an instant focal stack of nine 2D images. Appended to an\n",
      "epifluorescence microscope, the multifocus system enables high-resolution 3D imaging in multiple\n",
      "colors with single-molecule sensitivity, at speeds limited by the camera readout time of a single\n",
      "image.\n",
      "['multifocus microscopy',\n",
      " 'aberration-corrected multifocus microscopy',\n",
      " 'epifluorescence microscope',\n",
      " 'single-molecule sensitivity']\n",
      "Elapsed time: 8.20 seconds\n",
      "-------------------\n",
      "23241632\n",
      "Quantitative optical trapping on single organelles in cell extract.\n",
      "We have developed an optical trapping method to precisely measure the force generated by motor\n",
      "proteins on single organelles of unknown size in cell extract. This approach, termed VMatch, permits\n",
      "the functional interrogation of native motor complexes. We apply VMatch to measure the force, number\n",
      "and activity of kinesin-1 on motile lipid droplets isolated from the liver of normally fed and food-\n",
      "deprived rats.\n",
      "['VMatch', 'optical trapping', 'single organelles', 'cell extract']\n",
      "Elapsed time: 6.38 seconds\n",
      "-------------------\n",
      "23263690\n",
      "Conditional genome engineering in Toxoplasma gondii uncovers alternative invasion mechanisms.\n",
      "We established a conditional site-specific recombination system based on dimerizable Cre\n",
      "recombinase-mediated recombination in the apicomplexan parasite Toxoplasma gondii. Using a new\n",
      "single-vector strategy that allows ligand-dependent, efficient removal of a gene of interest, we\n",
      "generated three knockouts of apicomplexan genes considered essential for host-cell invasion. Our\n",
      "findings uncovered the existence of an alternative invasion pathway in apicomplexan parasites.\n",
      "['conditional site-specific recombination system',\n",
      " 'dimerizable Cre recombinase-mediated recombination',\n",
      " 'ligand-dependent efficient removal of a gene of interest']\n",
      "Elapsed time: 7.60 seconds\n",
      "-------------------\n",
      "23263691\n",
      "Single-molecule analysis of gene expression using two-color RNA labeling in live yeast.\n",
      "Live-cell imaging of mRNA yields important insights into gene expression, but it has generally been\n",
      "limited to the labeling of one RNA species and has never been used to count single mRNAs over time\n",
      "in yeast. We demonstrate a two-color imaging system with single-molecule resolution using MS2 and\n",
      "PP7 RNA labeling. We use this methodology to measure intrinsic noise in mRNA levels and RNA\n",
      "polymerase II kinetics at a single gene.\n",
      "['live-cell imaging', 'single-molecule resolution', 'two-color imaging system']\n",
      "Elapsed time: 7.49 seconds\n",
      "-------------------\n",
      "23291722\n",
      "iGLuc: a luciferase-based inflammasome and protease activity reporter.\n",
      "Measurement of protease activity in living cells or organisms remains a challenging task. We here\n",
      "present a transgene-encoded biosensor that reports the proteolytic activity of caspase-1 in the\n",
      "course of inflammasome activation and that of other proteases in a highly sensitive and specific\n",
      "manner. This protease reporter is based on the biological activity of a pro-interleukin\n",
      "(IL)-1β-Gaussia luciferase (iGLuc) fusion construct, in which pro-IL-1β-dependent formation of\n",
      "protein aggregates renders GLuc enzyme inactive. Cleavage leads to monomerization of this biosensor\n",
      "protein, resulting in a strong gain in luciferase activity. Exchange of the canonical caspase-1\n",
      "cleavage site in this reporter construct allows the generation of protease biosensors with\n",
      "additional specificities. The high sensitivity, signal-to-background ratio and specificity of the\n",
      "iGLuc system renders it a useful tool to study proteolytic events in mouse and human cells at high\n",
      "throughput and to monitor protease activity in mice in vivo.\n",
      "['luciferase-based inflammasome and protease activity reporter',\n",
      " 'bioluminescence',\n",
      " 'live cell imaging',\n",
      " 'in vivo imaging']\n",
      "Elapsed time: 10.99 seconds\n",
      "-------------------\n",
      "23314171\n",
      "An optimized fluorescent probe for visualizing glutamate neurotransmission.\n",
      "We describe an intensity-based glutamate-sensing fluorescent reporter (iGluSnFR) with signal-to-\n",
      "noise ratio and kinetics appropriate for in vivo imaging. We engineered iGluSnFR in vitro to\n",
      "maximize its fluorescence change, and we validated its utility for visualizing glutamate release by\n",
      "neurons and astrocytes in increasingly intact neurological systems. In hippocampal culture, iGluSnFR\n",
      "detected single field stimulus-evoked glutamate release events. In pyramidal neurons in acute brain\n",
      "slices, glutamate uncaging at single spines showed that iGluSnFR responds robustly and specifically\n",
      "to glutamate in situ, and responses correlate with voltage changes. In mouse retina, iGluSnFR-\n",
      "expressing neurons showed intact light-evoked excitatory currents, and the sensor revealed tonic\n",
      "glutamate signaling in response to light stimuli. In worms, glutamate signals preceded and predicted\n",
      "postsynaptic calcium transients. In zebrafish, iGluSnFR revealed spatial organization of direction-\n",
      "selective synaptic activity in the optic tectum. Finally, in mouse forelimb motor cortex, iGluSnFR\n",
      "expression in layer V pyramidal neurons revealed task-dependent single-spine activity during\n",
      "running.\n",
      "['hippocampal culture',\n",
      " 'acute brain slices',\n",
      " 'mouse retina',\n",
      " 'worms',\n",
      " 'zebrafish',\n",
      " 'mouse forelimb motor cortex']\n",
      "Elapsed time: 13.37 seconds\n",
      "-------------------\n",
      "23314172\n",
      "Detection of histone modifications at specific gene loci in single cells in histological sections.\n",
      "Chromatin immunoprecipitation assays have contributed greatly to our understanding of the role of\n",
      "histone modifications in gene regulation. However, they do not permit analysis with single-cell\n",
      "resolution, thus confounding analyses of heterogeneous cell populations. Here we present a method\n",
      "that permits visualization of histone modifications of single genomic loci with single-cell\n",
      "resolution in formaldehyde-fixed paraffin-embedded tissue sections based on combined use of in situ\n",
      "hybridization and proximity ligation assays. We show that dimethylation of lysine 4 of histone H3\n",
      "(H3K4me2) at the MYH11 locus is restricted to the smooth muscle cell (SMC) lineage in human and\n",
      "mouse tissue sections and that the mark persists even in phenotypically modulated SMC in\n",
      "atherosclerotic lesions that show no detectable expression of SMC marker genes. This methodology has\n",
      "promise for broad applications in the study of epigenetic mechanisms in complex multicellular\n",
      "tissues in development and disease.\n",
      "['in situ hybridization', 'proximity ligation assays']\n",
      "Elapsed time: 10.14 seconds\n",
      "-------------------\n",
      "23314173\n",
      "Lentiviral vector-based insertional mutagenesis identifies genes associated with liver cancer.\n",
      "Transposons and γ-retroviruses have been efficiently used as insertional mutagens in different\n",
      "tissues to identify molecular culprits of cancer. However, these systems are characterized by\n",
      "recurring integrations that accumulate in tumor cells and that hamper the identification of early\n",
      "cancer-driving events among bystander and progression-related events. We developed an insertional\n",
      "mutagenesis platform based on lentiviral vectors (LVVs) by which we could efficiently induce\n",
      "hepatocellular carcinoma (HCC) in three different mouse models. By virtue of the LVV's replication-\n",
      "deficient nature and broad genome-wide integration pattern, LVV-based insertional mutagenesis\n",
      "allowed identification of four previously unknown liver cancer-associated genes from a limited\n",
      "number of integrations. We validated the oncogenic potential of all the identified genes in vivo,\n",
      "with different levels of penetrance. The newly identified genes are likely to play a role in human\n",
      "cancer because they are upregulated, amplified and/or deleted in human HCCs and can predict clinical\n",
      "outcomes of patients.\n",
      "['lentiviral vectors',\n",
      " 'transposons',\n",
      " 'γ-retroviruses',\n",
      " 'insertional mutagenesis']\n",
      "Elapsed time: 11.09 seconds\n",
      "-------------------\n",
      "23407553\n",
      "Quantitative genetic-interaction mapping in mammalian cells.\n",
      "Mapping genetic interactions (GIs) by simultaneously perturbing pairs of genes is a powerful tool\n",
      "for understanding complex biological phenomena. Here we describe an experimental platform for\n",
      "generating quantitative GI maps in mammalian cells using a combinatorial RNA interference strategy.\n",
      "We performed ∼11,000 pairwise knockdowns in mouse fibroblasts, focusing on 130 factors involved in\n",
      "chromatin regulation to create a GI map. Comparison of the GI and protein-protein interaction (PPI)\n",
      "data revealed that pairs of genes exhibiting positive GIs and/or similar genetic profiles were\n",
      "predictive of the corresponding proteins being physically associated. The mammalian GI map\n",
      "identified pathways and complexes but also resolved functionally distinct submodules within larger\n",
      "protein complexes. By integrating GI and PPI data, we created a functional map of chromatin\n",
      "complexes in mouse fibroblasts, revealing that the PAF complex is a central player in the mammalian\n",
      "chromatin landscape.\n",
      "['combinatorial RNA interference strategy',\n",
      " 'protein-protein interaction (PPI) data']\n",
      "Elapsed time: 10.07 seconds\n",
      "-------------------\n",
      "23416756\n",
      "Single-chromosome transcriptional profiling reveals chromosomal gene expression regulation.\n",
      "We report intron chromosomal expression FISH (iceFISH), a multiplex imaging method for measuring\n",
      "gene expression and chromosome structure simultaneously on single chromosomes. We find substantial\n",
      "differences in transcriptional frequency between genes on a translocated chromosome and the same\n",
      "genes in their normal chromosomal context in the same cell. Correlations between genes on a single\n",
      "chromosome pointed toward a cis chromosome-level transcriptional interaction spanning 14.3\n",
      "megabases.\n",
      "['iceFISH', 'fluorescence in situ hybridization']\n",
      "Elapsed time: 7.02 seconds\n",
      "-------------------\n",
      "23435261\n",
      "Immunofluorescence and fluorescent-protein tagging show high correlation for protein localization in\n",
      "mammalian cells.\n",
      "Imaging techniques such as immunofluorescence (IF) and the expression of fluorescent protein (FP)\n",
      "fusions are widely used to investigate the subcellular distribution of proteins. Here we report a\n",
      "systematic analysis of >500 human proteins comparing the localizations obtained in live versus fixed\n",
      "cells using FPs and IF, respectively. We identify systematic discrepancies between IF and FPs as\n",
      "well as between FP tagging at the N and C termini. The analysis shows that for 80% of the proteins,\n",
      "IF and FPs yield the same subcellular distribution, and the locations of 250 previously unlocalized\n",
      "proteins were determined by the overlap between the two methods. Approximately 60% of proteins\n",
      "localize to multiple organelles for both methods, indicating a complex subcellular protein\n",
      "organization. These results show that both IF and FP tagging are reliable techniques and demonstrate\n",
      "the usefulness of an integrative approach for a complete investigation of the subcellular human\n",
      "proteome.\n",
      "['immunofluorescence', 'fluorescent-protein tagging', 'live versus fixed cells']\n",
      "Elapsed time: 10.38 seconds\n",
      "-------------------\n",
      "23443633\n",
      "Zymography methods for visualizing hydrolytic enzymes.\n",
      "Zymography is a technique for studying hydrolytic enzymes on the basis of substrate degradation. It\n",
      "is a powerful, but often misinterpreted, tool yielding information on potential hydrolytic\n",
      "activities, enzyme forms and the locations of active enzymes. In this Review, zymography techniques\n",
      "are compared in terms of advantages, limitations and interpretations. With in gel zymography, enzyme\n",
      "forms are visualized according to their molecular weights. Proteolytic activities are localized in\n",
      "tissue sections with in situ zymography. In vivo zymography can pinpoint proteolytic activity to\n",
      "sites in an intact organism. Future development of novel substrate probes and improvement in\n",
      "detection and imaging methods will increase the applicability of zymography for (reverse)\n",
      "degradomics studies.\n",
      "Invalid json output: {\n",
      "\"in gel zymography\",\n",
      "\"in situ zymography\",\n",
      "\"in vivo zymography\"\n",
      "} derived from Sure! Here is the list of imaging techniques described in the abstract as a JSON list:\n",
      "\n",
      "{\n",
      "\"in gel zymography\",\n",
      "\"in situ zymography\",\n",
      "\"in vivo zymography\"\n",
      "}\n",
      "23455923\n",
      "Ultrahigh accuracy imaging modality for super-localization microscopy.\n",
      "Super-localization microscopy encompasses techniques that depend on the accurate localization of\n",
      "individual molecules from generally low-light images. The obtainable localization accuracies,\n",
      "however, are ultimately limited by the image detector's pixelation and noise. We present the\n",
      "ultrahigh accuracy imaging modality (UAIM), which allows users to obtain accuracies approaching the\n",
      "accuracy that is achievable only in the absence of detector pixelation and noise, and which we found\n",
      "can experimentally provide a >200% accuracy improvement over conventional low-light imaging.\n",
      "['low-light images',\n",
      " 'super-localization microscopy',\n",
      " 'ultrahigh accuracy imaging modality']\n",
      "Elapsed time: 7.75 seconds\n",
      "-------------------\n",
      "23503053\n",
      "Barcoding cells using cell-surface programmable DNA-binding domains.\n",
      "We report an approach to barcode cells through cell-surface expression of programmable zinc-finger\n",
      "DNA-binding domains (surface zinc fingers, sZFs). We show that sZFs enable sequence-specific\n",
      "labeling of living cells by dsDNA, and we develop a sequential labeling approach to image more than\n",
      "three cell types in mixed populations using three fluorophores. We demonstrate the versatility of\n",
      "sZFs through applications in which they serve as surrogate reporters, function as selective cell\n",
      "capture reagents and facilitate targeted cellular delivery of viruses.\n",
      "['dsDNA', 'fluorophores', 'surface zinc fingers']\n",
      "Elapsed time: 7.45 seconds\n",
      "-------------------\n",
      "23524392\n",
      "A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum.\n",
      "We report a monomeric yellow-green fluorescent protein, mNeonGreen, derived from a tetrameric\n",
      "fluorescent protein from the cephalochordate Branchiostoma lanceolatum. mNeonGreen is the brightest\n",
      "monomeric green or yellow fluorescent protein yet described to our knowledge, performs exceptionally\n",
      "well as a fusion tag for traditional imaging as well as stochastic single-molecule superresolution\n",
      "imaging and is an excellent fluorescence resonance energy transfer (FRET) acceptor for the newest\n",
      "cyan fluorescent proteins.\n",
      "['traditional imaging',\n",
      " 'stochastic single-molecule superresolution imaging',\n",
      " 'fluorescence resonance energy transfer (FRET)']\n",
      "Elapsed time: 8.00 seconds\n",
      "-------------------\n",
      "23524393\n",
      "Whole-brain functional imaging at cellular resolution using light-sheet microscopy.\n",
      "Brain function relies on communication between large populations of neurons across multiple brain\n",
      "areas, a full understanding of which would require knowledge of the time-varying activity of all\n",
      "neurons in the central nervous system. Here we use light-sheet microscopy to record activity,\n",
      "reported through the genetically encoded calcium indicator GCaMP5G, from the entire volume of the\n",
      "brain of the larval zebrafish in vivo at 0.8 Hz, capturing more than 80% of all neurons at single-\n",
      "cell resolution. Demonstrating how this technique can be used to reveal functionally defined\n",
      "circuits across the brain, we identify two populations of neurons with correlated activity patterns.\n",
      "One circuit consists of hindbrain neurons functionally coupled to spinal cord neuropil. The other\n",
      "consists of an anatomically symmetric population in the anterior hindbrain, with activity in the\n",
      "left and right halves oscillating in antiphase, on a timescale of 20 s, and coupled to equally slow\n",
      "oscillations in the inferior olive.\n",
      "['light-sheet microscopy',\n",
      " 'whole-brain functional imaging',\n",
      " 'single-cell resolution']\n",
      "Elapsed time: 10.20 seconds\n",
      "-------------------\n",
      "23524394\n",
      "Single-molecule imaging of transcription factor binding to DNA in live mammalian cells.\n",
      "Imaging single fluorescent proteins in living mammalian cells is challenged by out-of-focus\n",
      "fluorescence excitation. To reduce out-of-focus fluorescence we developed reflected light-sheet\n",
      "microscopy (RLSM), a fluorescence microscopy method allowing selective plane illumination throughout\n",
      "the nuclei of living mammalian cells. A thin light sheet parallel to the imaging plane and close to\n",
      "the sample surface is generated by reflecting an elliptical laser beam incident from the top by 90°\n",
      "with a small mirror. The thin light sheet allows for an increased signal-to-background ratio\n",
      "superior to that in previous illumination schemes and enables imaging of single fluorescent proteins\n",
      "with up to 100-Hz time resolution. We demonstrated the single-molecule sensitivity of RLSM by\n",
      "measuring the DNA-bound fraction of glucocorticoid receptor (GR) and determining the residence times\n",
      "on DNA of various oligomerization states and mutants of GR and estrogen receptor-α (ER), which\n",
      "permitted us to resolve different modes of DNA binding of GR. We demonstrated two-color single-\n",
      "molecule imaging by observing the spatiotemporal colocalization of two different protein pairs. Our\n",
      "single-molecule measurements and statistical analysis revealed dynamic properties of transcription\n",
      "factors.\n",
      "['reflected light-sheet microscopy (RLSM)',\n",
      " 'fluorescence microscopy',\n",
      " 'single-molecule imaging']\n",
      "Elapsed time: 13.14 seconds\n",
      "-------------------\n",
      "23563794\n",
      "Mapping genetic interactions in human cancer cells with RNAi and multiparametric phenotyping.\n",
      "Genetic interactions influence many phenotypes and can be used as a powerful experimental tool to\n",
      "discover functional relationships between genes. Here we describe a robust and scalable method to\n",
      "systematically map genetic interactions in human cancer cells using combinatorial RNAi and high-\n",
      "throughput imaging. Through automated, single-cell phenotyping, we measured genetic interactions\n",
      "across a broad spectrum of phenotypes, including cell count, cell eccentricity and nuclear area. We\n",
      "mapped genetic interactions of epigenetic regulators in colon cancer cells, recovering known protein\n",
      "complexes. Our study also revealed the prospects and challenges of studying genetic interactions in\n",
      "human cells using multiparametric phenotyping.\n",
      "['high-throughput imaging',\n",
      " 'single-cell phenotyping',\n",
      " 'automated phenotyping',\n",
      " 'multiparametric phenotyping']\n",
      "Elapsed time: 8.49 seconds\n",
      "-------------------\n",
      "23563795\n",
      "Adhesion strength-based, label-free isolation of human pluripotent stem cells.\n",
      "We demonstrate substantial differences in 'adhesive signature' between human pluripotent stem cells\n",
      "(hPSCs), partially reprogrammed cells, somatic cells and hPSC-derived differentiated progeny. We\n",
      "exploited these differential adhesion strengths to rapidly (over ∼10 min) and efficiently isolate\n",
      "fully reprogrammed induced hPSCs (hiPSCs) as intact colonies from heterogeneous reprogramming\n",
      "cultures and from differentiated progeny using microfluidics. hiPSCs were isolated label free,\n",
      "enriched to 95%-99% purity with >80% survival, and had normal transcriptional profiles,\n",
      "differentiation potential and karyotypes. We also applied this strategy to isolate hPSCs (hiPSCs and\n",
      "human embryonic stem cells) during routine culture and show that it may be extended to isolate hPSC-\n",
      "derived lineage-specific stem cells or differentiated cells.\n",
      "['microfluidics', 'label-free isolation']\n",
      "Elapsed time: 9.58 seconds\n",
      "-------------------\n",
      "23584186\n",
      "A simple tool to improve pluripotent stem cell differentiation.\n",
      "We describe a method to help overcome restrictions on the differentiation propensities of human\n",
      "pluripotent stem cells. Culturing pluripotent stem cells in dimethylsulfoxide (DMSO) activates the\n",
      "retinoblastoma protein, increases the proportion of cells in the early G1 phase of the cell cycle\n",
      "and, in more than 25 embryonic and induced pluripotent stem cell lines, improves directed\n",
      "differentiation into multiple lineages. DMSO treatment also improves differentiation into terminal\n",
      "cell types in several cell lines.\n",
      "['dimethylsulfoxide (DMSO)', 'cell cycle', 'retinoblastoma protein']\n",
      "Elapsed time: 7.23 seconds\n",
      "-------------------\n",
      "23624665\n",
      "Measuring image resolution in optical nanoscopy.\n",
      "Resolution in optical nanoscopy (or super-resolution microscopy) depends on the localization\n",
      "uncertainty and density of single fluorescent labels and on the sample's spatial structure.\n",
      "Currently there is no integral, practical resolution measure that accounts for all factors. We\n",
      "introduce a measure based on Fourier ring correlation (FRC) that can be computed directly from an\n",
      "image. We demonstrate its validity and benefits on two-dimensional (2D) and 3D localization\n",
      "microscopy images of tubulin and actin filaments. Our FRC resolution method makes it possible to\n",
      "compare achieved resolutions in images taken with different nanoscopy methods, to optimize and rank\n",
      "different emitter localization and labeling strategies, to define a stopping criterion for data\n",
      "acquisition, to describe image anisotropy and heterogeneity, and even to estimate the average number\n",
      "of localizations per emitter. Our findings challenge the current focus on obtaining the best\n",
      "localization precision, showing instead how the best image resolution can be achieved as fast as\n",
      "possible.\n",
      "[{'name': 'Fourier ring correlation (FRC)'},\n",
      " {'name': 'localization microscopy'},\n",
      " {'name': 'super-resolution microscopy'},\n",
      " {'name': 'nanoscopy methods'}]\n",
      "Elapsed time: 11.04 seconds\n",
      "-------------------\n",
      "23644547\n",
      "Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle\n",
      "cryo-EM.\n",
      "In recent work with large high-symmetry viruses, single-particle electron cryomicroscopy (cryo-EM)\n",
      "has achieved the determination of near-atomic-resolution structures by allowing direct fitting of\n",
      "atomic models into experimental density maps. However, achieving this goal with smaller particles of\n",
      "lower symmetry remains challenging. Using a newly developed single electron-counting detector, we\n",
      "confirmed that electron beam-induced motion substantially degrades resolution, and we showed that\n",
      "the combination of rapid readout and nearly noiseless electron counting allow image blurring to be\n",
      "corrected to subpixel accuracy, restoring intrinsic image information to high resolution (Thon rings\n",
      "visible to ∼3 Å). Using this approach, we determined a 3.3-Å-resolution structure of an ∼700-kDa\n",
      "protein with D7 symmetry, the Thermoplasma acidophilum 20S proteasome, showing clear side-chain\n",
      "density. Our method greatly enhances image quality and data acquisition efficiency-key bottlenecks\n",
      "in applying near-atomic-resolution cryo-EM to a broad range of protein samples.\n",
      "['electron counting',\n",
      " 'beam-induced motion correction',\n",
      " 'single-particle electron cryomicroscopy (cryo-EM)',\n",
      " 'rapid readout',\n",
      " 'noiseless electron counting']\n",
      "Elapsed time: 11.78 seconds\n",
      "-------------------\n",
      "23666410\n",
      "The Caenorhabditis elegans Lifespan Machine.\n",
      "The measurement of lifespan pervades aging research. Because lifespan results from complex\n",
      "interactions between genetic, environmental and stochastic factors, it varies widely even among\n",
      "isogenic individuals. The actions of molecular mechanisms on lifespan are therefore visible only\n",
      "through their statistical effects on populations. Indeed, survival assays in Caenorhabditis elegans\n",
      "have provided critical insights into evolutionarily conserved determinants of aging. To enable the\n",
      "rapid acquisition of survival curves at an arbitrary statistical resolution, we developed a scalable\n",
      "imaging and analysis platform to observe nematodes over multiple weeks across square meters of agar\n",
      "surface at 8-μm resolution. The automated method generates a permanent visual record of individual\n",
      "deaths from which survival curves are constructed and validated, producing data consistent with\n",
      "results from the manual method of survival curve acquisition for several mutants in both standard\n",
      "and stressful environments. Our approach permits rapid, detailed reverse-genetic and chemical\n",
      "screens for effects on survival and enables quantitative investigations into the statistical\n",
      "structure of aging.\n",
      "['imaging',\n",
      " 'survival curves',\n",
      " 'agar surface',\n",
      " '8-μm resolution',\n",
      " 'automated method',\n",
      " 'visual record',\n",
      " 'reverse-genetic screens',\n",
      " 'chemical screens']\n",
      "Elapsed time: 10.63 seconds\n",
      "-------------------\n",
      "23708387\n",
      "Video-rate nanoscopy using sCMOS camera-specific single-molecule localization algorithms.\n",
      "Newly developed scientific complementary metal-oxide semiconductor (sCMOS) cameras have the\n",
      "potential to dramatically accelerate data acquisition, enlarge the field of view and increase the\n",
      "effective quantum efficiency in single-molecule switching nanoscopy. However, sCMOS-intrinsic pixel-\n",
      "dependent readout noise substantially lowers the localization precision and introduces localization\n",
      "artifacts. We present algorithms that overcome these limitations and that provide unbiased, precise\n",
      "localization of single molecules at the theoretical limit. Using these in combination with a multi-\n",
      "emitter fitting algorithm, we demonstrate single-molecule localization super-resolution imaging at\n",
      "rates of up to 32 reconstructed images per second in fixed and living cells.\n",
      "['sCMOS camera',\n",
      " 'single-molecule localization algorithms',\n",
      " 'single-molecule switching nanoscopy',\n",
      " 'pixel-dependent readout noise',\n",
      " 'localization artifacts',\n",
      " 'multi-emitter fitting algorithm',\n",
      " 'single-molecule localization super-resolution imaging']\n",
      "Elapsed time: 10.27 seconds\n",
      "-------------------\n",
      "23722209\n",
      "Cellular-resolution connectomics: challenges of dense neural circuit reconstruction.\n",
      "Neuronal networks are high-dimensional graphs that are packed into three-dimensional nervous tissue\n",
      "at extremely high density. Comprehensively mapping these networks is therefore a major challenge.\n",
      "Although recent developments in volume electron microscopy imaging have made data acquisition\n",
      "feasible for circuits comprising a few hundreds to a few thousands of neurons, data analysis is\n",
      "massively lagging behind. The aim of this perspective is to summarize and quantify the challenges\n",
      "for data analysis in cellular-resolution connectomics and describe current solutions involving\n",
      "online crowd-sourcing and machine-learning approaches.\n",
      "['volume electron microscopy imaging',\n",
      " 'online crowd-sourcing',\n",
      " 'machine-learning approaches']\n",
      "Elapsed time: 7.42 seconds\n",
      "-------------------\n",
      "23722211\n",
      "Mapping brain circuitry with a light microscope.\n",
      "The beginning of the 21st century has seen a renaissance in light microscopy and anatomical tract\n",
      "tracing that together are rapidly advancing our understanding of the form and function of neuronal\n",
      "circuits. The introduction of instruments for automated imaging of whole mouse brains, new cell\n",
      "type–specific and trans-synaptic tracers, and computational methods for handling the whole-brain\n",
      "data sets has opened the door to neuroanatomical studies at an unprecedented scale. We present an\n",
      "overview of the present state and future opportunities in charting long-range and local connectivity\n",
      "in the entire mouse brain and in linking brain circuits to function.\n",
      "['light microscope',\n",
      " 'automated imaging',\n",
      " 'whole-brain data sets',\n",
      " 'cell type–specific tracers',\n",
      " 'trans-synaptic tracers']\n",
      "Elapsed time: 8.39 seconds\n",
      "-------------------\n",
      "23722212\n",
      "Imaging human connectomes at the macroscale.\n",
      "At macroscopic scales, the human connectome comprises anatomically distinct brain areas, the\n",
      "structural pathways connecting them and their functional interactions. Annotation of phenotypic\n",
      "associations with variation in the connectome and cataloging of neurophenotypes promise to transform\n",
      "our understanding of the human brain. In this Review, we provide a survey of magnetic resonance\n",
      "imaging–based measurements of functional and structural connectivity. We highlight emerging areas of\n",
      "development and inquiry and emphasize the importance of integrating structural and functional\n",
      "perspectives on brain architecture.\n",
      "['magnetic resonance imaging',\n",
      " 'functional connectivity',\n",
      " 'structural connectivity']\n",
      "Elapsed time: 6.94 seconds\n",
      "-------------------\n",
      "23727986\n",
      "Enzymatic production of 'monoclonal stoichiometric' single-stranded DNA oligonucleotides.\n",
      "Single-stranded oligonucleotides are important as research tools, as diagnostic probes, in gene\n",
      "therapy and in DNA nanotechnology. Oligonucleotides are typically produced via solid-phase\n",
      "synthesis, using polymer chemistries that are limited relative to what biological systems produce.\n",
      "The number of errors in synthetic DNA increases with oligonucleotide length, and the resulting\n",
      "diversity of sequences can be a problem. Here we present the 'monoclonal stoichiometric' (MOSIC)\n",
      "method for enzyme-mediated production of DNA oligonucleotides. We amplified oligonucleotides from\n",
      "clonal templates derived from single bacterial colonies and then digested cutter hairpins in the\n",
      "products, which released pools of oligonucleotides with precisely controlled relative stoichiometric\n",
      "ratios. We prepared 14-378-nucleotide MOSIC oligonucleotides either by in vitro rolling-circle\n",
      "amplification or by amplification of phagemid DNA in Escherichia coli. Analyses of the formation of\n",
      "a DNA crystal and folding of DNA nanostructures confirmed the scalability, purity and stoichiometry\n",
      "of the produced oligonucleotides.\n",
      "['in vitro rolling-circle amplification',\n",
      " 'phagemid DNA amplification in Escherichia coli',\n",
      " 'DNA crystal formation analysis',\n",
      " 'DNA nanostructure folding analysis']\n",
      "Elapsed time: 13.28 seconds\n",
      "-------------------\n",
      "23749299\n",
      "Hematopoietic progenitor cell lines with myeloid and lymphoid potential.\n",
      "Investigation of immune-cell differentiation and function is limited by shortcomings of suitable and\n",
      "scalable experimental systems. Here we show that retroviral delivery of an estrogen-regulated form\n",
      "of Hoxb8 into mouse bone marrow cells can be used along with Flt3 ligand to conditionally\n",
      "immortalize early hematopoietic progenitor cells (Hoxb8-FL cells). Hoxb8-FL cells have lost self-\n",
      "renewal capacity and potential to differentiate into megakaryocytes and erythrocytes but retain the\n",
      "potential to differentiate into myeloid and lymphoid cells. They differentiate in vitro and in vivo\n",
      "into macrophages, granulocytes, dendritic cells, B lymphocytes and T lymphocytes that are\n",
      "phenotypically and functionally indistinguishable from their primary counterparts. Quantitative in\n",
      "vitro assays indicate that myeloid and B-cell potential of Hoxb8-FL cells is comparable to that of\n",
      "primary lymphoid-primed multipotent progenitors, whereas T-cell potential is diminished. The\n",
      "simplicity of this system and the unlimited proliferative capacity of Hoxb8-FL cells will enable\n",
      "studies of immune-cell differentiation and function.\n",
      "['retroviral delivery', 'Flt3 ligand', 'in vitro assays']\n",
      "Elapsed time: 12.00 seconds\n",
      "-------------------\n",
      "23770755\n",
      "Near-infrared fluorescent proteins for multicolor in vivo imaging.\n",
      "Near-infrared fluorescent proteins (FPs) are in high demand for in vivo imaging. We developed four\n",
      "spectrally distinct near-infrared FPs--iRFP670, iRFP682, iRFP702 and iRFP720--from bacterial\n",
      "phytochromes. iRFPs exhibit high brightness in mammalian cells and tissues and are suitable for\n",
      "long-term studies. iRFP670 and iRFP720 enable two-color imaging with standard approaches in living\n",
      "cells and mice. The four new iRFPs and the previously engineered iRFP713 allow multicolor imaging\n",
      "with spectral unmixing in living mice.\n",
      "['in vivo imaging',\n",
      " 'multicolor imaging',\n",
      " 'spectral unmixing',\n",
      " 'standard approaches']\n",
      "Elapsed time: 8.92 seconds\n",
      "-------------------\n",
      "23817067\n",
      "Engineered nanostructured β-sheet peptides protect membrane proteins.\n",
      "We designed β-strand peptides that stabilize integral membrane proteins (IMPs). β-strand peptides\n",
      "self-assemble in solution as filaments and become restructured upon association with IMPs; resulting\n",
      "IMP-β-strand peptide complexes resisted aggregation when diluted in detergent-free buffer and were\n",
      "visible as stable, single particles with low detergent background in electron micrographs. β-strand\n",
      "peptides enabled clear visualization of flexible conformations in the highly dynamic ATP-binding\n",
      "cassette (ABC) transporter MsbA.\n",
      "['electron micrographs', 'detergent-free buffer']\n",
      "Elapsed time: 7.74 seconds\n",
      "-------------------\n",
      "23817068\n",
      "TALE-mediated modulation of transcriptional enhancers in vivo.\n",
      "We tested whether transcription activator-like effectors (TALEs) could mediate repression and\n",
      "activation of endogenous enhancers in the Drosophila genome. TALE repressors (TALERs) targeting each\n",
      "of the five even-skipped (eve) stripe enhancers generated repression specifically of the focal\n",
      "stripes. TALE activators (TALEAs) targeting the eve promoter or enhancers caused increased\n",
      "expression primarily in cells normally activated by the promoter or targeted enhancer, respectively.\n",
      "This effect supports the view that repression acts in a dominant fashion on transcriptional\n",
      "activators and that the activity state of an enhancer influences TALE binding or the ability of the\n",
      "VP16 domain to enhance transcription. In these assays, the Hairy repression domain did not exhibit\n",
      "previously described long-range transcriptional repression activity. The phenotypic effects of TALER\n",
      "and TALEA expression in larvae and adults are consistent with the observed modulations of eve\n",
      "expression. TALEs thus provide a novel tool for detection and functional modulation of\n",
      "transcriptional enhancers in their native genomic context.\n",
      "['TALE-mediated modulation',\n",
      " 'repression',\n",
      " 'activation',\n",
      " 'transcriptional enhancers',\n",
      " 'native genomic context']\n",
      "Elapsed time: 11.49 seconds\n",
      "-------------------\n",
      "23817070\n",
      "Cell-selective labeling using amino acid precursors for proteomic studies of multicellular\n",
      "environments.\n",
      "We report a technique to selectively and continuously label the proteomes of individual cell types\n",
      "in coculture, named cell type-specific labeling using amino acid precursors (CTAP). Through\n",
      "transgenic expression of exogenous amino acid biosynthesis enzymes, vertebrate cells overcome their\n",
      "dependence on supplemented essential amino acids and can be selectively labeled through metabolic\n",
      "incorporation of amino acids produced from heavy isotope-labeled precursors. When testing CTAP in\n",
      "several human and mouse cell lines, we could differentially label the proteomes of distinct cell\n",
      "populations in coculture and determine the relative expression of proteins by quantitative mass\n",
      "spectrometry. In addition, using CTAP we identified the cell of origin of extracellular proteins\n",
      "secreted from cells in coculture. We believe that this method, which allows linking of proteins to\n",
      "their cell source, will be useful in studies of cell-cell communication and potentially for\n",
      "discovery of biomarkers.\n",
      "['mass spectrometry',\n",
      " 'quantitative mass spectrometry',\n",
      " 'cell type-specific labeling using amino acid precursors (CTAP)']\n",
      "Elapsed time: 10.43 seconds\n",
      "-------------------\n",
      "23817127\n",
      "Improved tools for the Brainbow toolbox.\n",
      "In the transgenic multicolor labeling strategy called 'Brainbow', Cre-loxP recombination is used to\n",
      "create a stochastic choice of expression among fluorescent proteins, resulting in the indelible\n",
      "marking of mouse neurons with multiple distinct colors. This method has been adapted to non-neuronal\n",
      "cells in mice and to neurons in fish and flies, but its full potential has yet to be realized in the\n",
      "mouse brain. Here we present several lines of mice that overcome limitations of the initial lines,\n",
      "and we report an adaptation of the method for use in adeno-associated viral vectors. We also provide\n",
      "technical advice about how best to image Brainbow-expressing tissue.\n",
      "['imaging', 'viral vectors', 'Technical advice']\n",
      "Elapsed time: 7.67 seconds\n",
      "-------------------\n",
      "23852451\n",
      "A database of Caenorhabditis elegans behavioral phenotypes.\n",
      "Using low-cost automated tracking microscopes, we have generated a behavioral database for 305\n",
      "Caenorhabditis elegans strains, including 76 mutants with no previously described phenotype. The\n",
      "growing database currently consists of 9,203 short videos segmented to extract behavior and\n",
      "morphology features, and these videos and feature data are available online for further analysis.\n",
      "The database also includes summary statistics for 702 measures with statistical comparisons to wild-\n",
      "type controls so that phenotypes can be identified and understood by users.\n",
      "['low-cost automated tracking microscopes',\n",
      " 'short videos',\n",
      " 'behavior and morphology features',\n",
      " 'online analysis',\n",
      " 'statistical comparisons']\n",
      "Elapsed time: 7.60 seconds\n",
      "-------------------\n",
      "23852453\n",
      "Functional labeling of neurons and their projections using the synthetic activity-dependent promoter\n",
      "E-SARE.\n",
      "Identifying the neuronal ensembles that respond to specific stimuli and mapping their projection\n",
      "patterns in living animals are fundamental challenges in neuroscience. To this end, we engineered a\n",
      "synthetic promoter, the enhanced synaptic activity-responsive element (E-SARE), that drives neuronal\n",
      "activity-dependent gene expression more potently than other existing immediate-early gene promoters.\n",
      "Expression of a drug-inducible Cre recombinase downstream of E-SARE enabled imaging of neuronal\n",
      "populations that respond to monocular visual stimulation and tracking of their long-distance\n",
      "thalamocortical projections in living mice. Targeted cell-attached recordings and calcium imaging of\n",
      "neurons in sensory cortices revealed that E-SARE reporter expression correlates with sensory-evoked\n",
      "neuronal activity at the single-cell level and is highly specific to the type of stimuli presented\n",
      "to the animals. This activity-dependent promoter can expand the repertoire of genetic approaches for\n",
      "high-resolution anatomical and functional analysis of neural circuits.\n",
      "['calcium imaging', 'cell-attached recordings', 'imaging of neurons']\n",
      "Elapsed time: 11.07 seconds\n",
      "-------------------\n",
      "23866336\n",
      "Improved tools for the Brainbow toolbox.\n",
      "In the transgenic multicolor labeling strategy called ‘Brainbow’, Cre-loxP recombination is used to\n",
      "create a stochastic choice of expression among fluorescent proteins, resulting in the indelible\n",
      "marking of mouse neurons with multiple distinct colors. This method has been adapted to non-neuronal\n",
      "cells in mice and to neurons in fish and flies, but its full potential has yet to be realized in the\n",
      "mouse brain. Here we present several lines of mice that overcome limitations of the initial lines,\n",
      "and we report an adaptation of the method for use in adeno-associated viral vectors. We also provide\n",
      "technical advice about how best to image Brainbow-expressing tissue.\n",
      "['Brainbow',\n",
      " 'Adeno-associated viral vectors',\n",
      " 'Technical advice for imaging Brainbow-expressing tissue']\n",
      "Elapsed time: 8.56 seconds\n",
      "-------------------\n",
      "23872791\n",
      "Imaging bacterial protein expression using genetically encoded RNA sensors.\n",
      "The difficulties in imaging the dynamics of protein expression in live bacterial cells can be\n",
      "overcome by using fluorescent sensors based on Spinach, an RNA that activates the fluorescence of a\n",
      "small-molecule fluorophore. These RNAs selectively bind target proteins and exhibit fluorescence\n",
      "increases that enable protein expression to be imaged in living Escherichia coli. These sensors are\n",
      "key components of a generalizable strategy to image protein expression in a single bacterium in real\n",
      "time.\n",
      "['fluorescent sensors', 'Spinach', 'small-molecule fluorophore']\n",
      "Elapsed time: 7.05 seconds\n",
      "-------------------\n",
      "23892895\n",
      "RNA-guided gene activation by CRISPR-Cas9-based transcription factors.\n",
      "Technologies for engineering synthetic transcription factors have enabled many advances in medical\n",
      "and scientific research. In contrast to existing methods based on engineering of DNA-binding\n",
      "proteins, we created a Cas9-based transactivator that is targeted to DNA sequences by guide RNA\n",
      "molecules. Coexpression of this transactivator and combinations of guide RNAs in human cells induced\n",
      "specific expression of endogenous target genes, demonstrating a simple and versatile approach for\n",
      "RNA-guided gene activation.\n",
      "['CRISPR-Cas9', 'guide RNA molecules', 'coexpression']\n",
      "Elapsed time: 7.21 seconds\n",
      "-------------------\n",
      "23913258\n",
      "3D spectral imaging with synchrotron Fourier transform infrared spectro-microtomography.\n",
      "We report Fourier transform infrared spectro-microtomography, a nondestructive three-dimensional\n",
      "imaging approach that reveals the distribution of distinctive chemical compositions throughout an\n",
      "intact biological or materials sample. The method combines mid-infrared absorption contrast with\n",
      "computed tomographic data acquisition and reconstruction to enhance chemical and morphological\n",
      "localization by determining a complete infrared spectrum for every voxel (millions of spectra\n",
      "determined per sample).\n",
      "['Fourier transform infrared spectro-microtomography',\n",
      " 'computed tomographic data acquisition and reconstruction',\n",
      " 'mid-infrared absorption contrast']\n",
      "Elapsed time: 7.56 seconds\n",
      "-------------------\n",
      "23913259\n",
      "Visualizing SNVs to quantify allele-specific expression in single cells.\n",
      "We present a FISH-based method for detecting single-nucleotide variants (SNVs) in exons and introns\n",
      "on individual RNA transcripts with high efficiency. We used this method to quantify allelic\n",
      "expression in cell populations and in single cells, and also to distinguish maternal from paternal\n",
      "chromosomes in single cells.\n",
      "['FISH-based method',\n",
      " 'single-nucleotide variants (SNVs)',\n",
      " 'allelic expression',\n",
      " 'cell populations',\n",
      " 'single cells',\n",
      " 'maternal chromosomes',\n",
      " 'paternal chromosomes']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "23934076\n",
      "Allele-specific detection of single mRNA molecules in situ.\n",
      "We describe a method for fluorescence in situ identification of individual mRNA molecules, allowing\n",
      "quantitative and accurate measurement, in single cells, of allele-specific transcripts that differ\n",
      "by only a few nucleotides. By using a combination of allele-specific and non-allele-specific probe\n",
      "libraries, we achieve >95% detection accuracy. We investigate the allele-specific stochastic\n",
      "expression of Nanog, which encodes a pluripotency factor, in murine embryonic stem cells.\n",
      "['fluorescence in situ', 'allele-specific detection', 'single mRNA molecules']\n",
      "Elapsed time: 7.60 seconds\n",
      "-------------------\n",
      "23934077\n",
      "STED nanoscopy combined with optical tweezers reveals protein dynamics on densely covered DNA.\n",
      "Dense coverage of DNA by proteins is a ubiquitous feature of cellular processes such as DNA\n",
      "organization, replication and repair. We present a single-molecule approach capable of visualizing\n",
      "individual DNA-binding proteins on densely covered DNA and in the presence of high protein\n",
      "concentrations. Our approach combines optical tweezers with multicolor confocal and stimulated\n",
      "emission depletion (STED) fluorescence microscopy. Proteins on DNA are visualized at a resolution of\n",
      "50 nm, a sixfold resolution improvement over that of confocal microscopy. High temporal resolution\n",
      "(<50 ms) is ensured by fast one-dimensional beam scanning. Individual trajectories of proteins\n",
      "translocating on DNA can thus be distinguished and tracked with high precision. We demonstrate our\n",
      "multimodal approach by visualizing the assembly of dense nucleoprotein filaments with unprecedented\n",
      "spatial resolution in real time. Experimental access to the force-dependent kinetics and motility of\n",
      "DNA-associating proteins at biologically relevant protein densities is essential for linking\n",
      "idealized in vitro experiments with the in vivo situation.\n",
      "['STED fluorescence microscopy',\n",
      " 'optical tweezers',\n",
      " 'confocal microscopy',\n",
      " 'multicolor confocal microscopy']\n",
      "Elapsed time: 11.44 seconds\n",
      "-------------------\n",
      "23985731\n",
      "Multiparametric imaging of biological systems by force-distance curve-based AFM.\n",
      "A current challenge in the life sciences is to understand how biological systems change their\n",
      "structural, biophysical and chemical properties to adjust functionality. Addressing this issue has\n",
      "been severely hampered by the lack of methods capable of imaging biosystems at high resolution while\n",
      "simultaneously mapping their multiple properties. Recent developments in force-distance (FD) curve-\n",
      "based atomic force microscopy (AFM) now enable researchers to combine (sub)molecular imaging with\n",
      "quantitative mapping of physical, chemical and biological interactions. Here we discuss the\n",
      "principles and applications of advanced FD-based AFM tools for the quantitative multiparametric\n",
      "characterization of complex cellular and biomolecular systems under physiological conditions.\n",
      "['atomic force microscopy', 'submolecular imaging', 'quantitative mapping']\n",
      "Elapsed time: 8.64 seconds\n",
      "-------------------\n",
      "23995389\n",
      "Engineering the Caenorhabditis elegans genome using Cas9-triggered homologous recombination.\n",
      "Study of the nematode Caenorhabditis elegans has provided important insights in a wide range of\n",
      "fields in biology. The ability to precisely modify genomes is critical to fully realize the utility\n",
      "of model organisms. Here we report a method to edit the C. elegans genome using the clustered,\n",
      "regularly interspersed, short palindromic repeats (CRISPR) RNA-guided Cas9 nuclease and homologous\n",
      "recombination. We demonstrate that Cas9 is able to induce DNA double-strand breaks with specificity\n",
      "for targeted sites and that these breaks can be repaired efficiently by homologous recombination. By\n",
      "supplying engineered homologous repair templates, we generated gfp knock-ins and targeted mutations.\n",
      "Together our results outline a flexible methodology to produce essentially any desired modification\n",
      "in the C. elegans genome quickly and at low cost. This technology is an important addition to the\n",
      "array of genetic techniques already available in this experimentally tractable model organism.\n",
      "['CRISPR',\n",
      " 'RNA-guided Cas9 nuclease',\n",
      " 'homologous recombination',\n",
      " 'DNA double-strand breaks',\n",
      " 'gfp knock-ins',\n",
      " 'targeted mutations']\n",
      "Elapsed time: 11.09 seconds\n",
      "-------------------\n",
      "24013819\n",
      "Solid-state NMR spectroscopy structure determination of a lipid-embedded heptahelical membrane\n",
      "protein.\n",
      "Determination of structure of integral membrane proteins, especially in their native environment, is\n",
      "a formidable challenge in structural biology. Here we demonstrate that magic angle spinning solid-\n",
      "state NMR spectroscopy can be used to determine structures of membrane proteins reconstituted in\n",
      "synthetic lipids, an environment similar to the natural membrane. We combined a large number of\n",
      "experimentally determined interatomic distances and local torsional restraints to solve the\n",
      "structure of an oligomeric membrane protein of common seven-helical fold, Anabaena sensory rhodopsin\n",
      "(ASR). We determined the atomic resolution detail of the oligomerization interface of the ASR\n",
      "trimer, and the arrangement of helices, side chains and the retinal cofactor in the monomer.\n",
      "['solid-state NMR spectroscopy',\n",
      " 'magic angle spinning solid-state NMR spectroscopy']\n",
      "Elapsed time: 9.12 seconds\n",
      "-------------------\n",
      "24013820\n",
      "Brain-wide 3D imaging of neuronal activity in Caenorhabditis elegans with sculpted light.\n",
      "Recent efforts in neuroscience research have been aimed at obtaining detailed anatomical neuronal\n",
      "wiring maps as well as information on how neurons in these networks engage in dynamic activities.\n",
      "Although the entire connectivity map of the nervous system of Caenorhabditis elegans has been known\n",
      "for more than 25 years, this knowledge has not been sufficient to predict all functional connections\n",
      "underlying behavior. To approach this goal, we developed a two-photon technique for brain-wide\n",
      "calcium imaging in C. elegans, using wide-field temporal focusing (WF-TeFo). Pivotal to our results\n",
      "was the use of a nuclear-localized, genetically encoded calcium indicator, NLS-GCaMP5K, that permits\n",
      "unambiguous discrimination of individual neurons within the densely packed head ganglia of C.\n",
      "elegans. We demonstrate near-simultaneous recording of activity of up to 70% of all head neurons. In\n",
      "combination with a lab-on-a-chip device for stimulus delivery, this method provides an enabling\n",
      "platform for establishing functional maps of neuronal networks.\n",
      "[{'name': 'two-photon technique'},\n",
      " {'name': 'wide-field temporal focusing (WF-TeFo)'},\n",
      " {'name': 'nuclear-localized, genetically encoded calcium indicator, '\n",
      "          'NLS-GCaMP5K'}]\n",
      "Elapsed time: 12.94 seconds\n",
      "-------------------\n",
      "24056873\n",
      "Genetically encoded calcium indicator illuminates calcium dynamics in primary cilia.\n",
      "Visualization of signal transduction in live primary cilia constitutes a technical challenge owing\n",
      "to the organelle's submicrometer dimensions and close proximity to the cell body. Using a\n",
      "genetically encoded calcium indicator targeted to primary cilia, we visualized calcium signaling in\n",
      "cilia of mouse fibroblasts and kidney cells upon chemical or mechanical stimulation with high\n",
      "specificity, high sensitivity and wide dynamic range.\n",
      "['genetically encoded calcium indicator',\n",
      " 'live primary cilia',\n",
      " 'chemical stimulation',\n",
      " 'mechanical stimulation']\n",
      "Elapsed time: 7.09 seconds\n",
      "-------------------\n",
      "24056874\n",
      "Inducible control of gene expression with destabilized Cre.\n",
      "Acute manipulation of gene and protein function in the brain is essential for understanding the\n",
      "mechanisms of nervous system development, plasticity and information processing. Here we describe a\n",
      "technique based on a destabilized Cre recombinase (DD-Cre) whose activity is controlled by the\n",
      "antibiotic trimethoprim (TMP). We show that DD-Cre triggers rapid TMP-dependent recombination of\n",
      "loxP-flanked ('floxed') alleles in mouse neurons in vivo and validate the use of this system for\n",
      "neurobehavioral research.\n",
      "['recombinase', 'loxp-flanked alleles', 'neurobehavioral research']\n",
      "Elapsed time: 7.53 seconds\n",
      "-------------------\n",
      "24097269\n",
      "Image-based transcriptomics in thousands of single human cells at single-molecule resolution.\n",
      "Fluorescence in situ hybridization (FISH) is widely used to obtain information about transcript copy\n",
      "number and subcellular localization in single cells. However, current approaches do not readily\n",
      "scale to the analysis of whole transcriptomes. Here we show that branched DNA technology combined\n",
      "with automated liquid handling, high-content imaging and quantitative image analysis allows highly\n",
      "reproducible quantification of transcript abundance in thousands of single cells at single-molecule\n",
      "resolution. In addition, it allows extraction of a multivariate feature set quantifying subcellular\n",
      "patterning and spatial properties of transcripts and their cell-to-cell variability. This has\n",
      "multiple implications for the functional interpretation of cell-to-cell variability in gene\n",
      "expression and enables the unbiased identification of functionally relevant in situ signatures of\n",
      "the transcriptome without the need for perturbations. Because this method can be incorporated in a\n",
      "wide variety of high-throughput image-based approaches, we expect it to be broadly applicable.\n",
      "['fluorescence in situ hybridization (FISH)',\n",
      " 'high-content imaging',\n",
      " 'automated liquid handling',\n",
      " 'quantitative image analysis']\n",
      "Elapsed time: 10.65 seconds\n",
      "-------------------\n",
      "24097271\n",
      "Instant super-resolution imaging in live cells and embryos via analog image processing.\n",
      "Existing super-resolution fluorescence microscopes compromise acquisition speed to provide\n",
      "subdiffractive sample information. We report an analog implementation of structured illumination\n",
      "microscopy that enables three-dimensional (3D) super-resolution imaging with a lateral resolution of\n",
      "145 nm and an axial resolution of 350 nm at acquisition speeds up to 100 Hz. By using optical\n",
      "instead of digital image-processing operations, we removed the need to capture, store and combine\n",
      "multiple camera exposures, increasing data acquisition rates 10- to 100-fold over other super-\n",
      "resolution microscopes and acquiring and displaying super-resolution images in real time. Low\n",
      "excitation intensities allow imaging over hundreds of 2D sections, and combined physical and\n",
      "computational sectioning allow similar depth penetration to spinning-disk confocal microscopy. We\n",
      "demonstrate the capability of our system by imaging fine, rapidly moving structures including motor-\n",
      "driven organelles in human lung fibroblasts and the cytoskeleton of flowing blood cells within\n",
      "developing zebrafish embryos.\n",
      "['structured illumination microscopy',\n",
      " 'analog image processing',\n",
      " 'super-resolution fluorescence microscopes',\n",
      " 'spinning-disk confocal microscopy']\n",
      "Elapsed time: 11.76 seconds\n",
      "-------------------\n",
      "24122039\n",
      "Formation of targeted monovalent quantum dots by steric exclusion.\n",
      "Precise control over interfacial chemistry between nanoparticles and other materials remains a major\n",
      "challenge that limits broad application of nanotechnology in biology. To address this challenge, we\n",
      "used 'steric exclusion' to completely convert commercial quantum dots (QDs) into monovalent imaging\n",
      "probes by wrapping each QD with a functionalized oligonucleotide. We demonstrated the utility of\n",
      "these QDs as modular and nonperturbing imaging probes by tracking individual Notch receptors on live\n",
      "cells.\n",
      "['steric exclusion', 'functionalized oligonucleotide', 'live cells']\n",
      "Elapsed time: 7.27 seconds\n",
      "-------------------\n",
      "24122040\n",
      "Detergent-free mass spectrometry of membrane protein complexes.\n",
      "We developed a method that allows release of intact membrane protein complexes from amphipols,\n",
      "bicelles and nanodiscs in the gas phase for observation by mass spectrometry (MS). Current methods\n",
      "involve release of membrane protein complexes from detergent micelles, which reveals subunit\n",
      "composition and lipid binding. We demonstrated that oligomeric complexes or proteins requiring\n",
      "defined lipid environments are stabilized to a greater extent in the absence of detergent.\n",
      "['mass spectrometry (MS)',\n",
      " 'detergent-free mass spectrometry',\n",
      " 'amphipols',\n",
      " 'bicelles',\n",
      " 'nanodiscs']\n",
      "Elapsed time: 7.41 seconds\n",
      "-------------------\n",
      "24141495\n",
      "High-throughput fingerprinting of human pluripotent stem cell fate responses and lineage bias.\n",
      "Populations of cells create local environments that lead to emergent heterogeneity. This is\n",
      "particularly evident with human pluripotent stem cells (hPSCs): microenvironmental heterogeneity\n",
      "limits hPSC cell fate control. We developed a high-throughput platform to screen hPSCs in\n",
      "configurable microenvironments in which we optimized colony size, cell density and other parameters\n",
      "to achieve rapid and robust cell fate responses to exogenous cues. We used this platform to perform\n",
      "single-cell protein expression profiling, revealing that Oct4 and Sox2 costaining discriminates\n",
      "pluripotent, neuroectoderm, primitive streak and extraembryonic cell fates. We applied this\n",
      "Oct4-Sox2 code to analyze dose responses of 27 developmental factors to obtain lineage-specific\n",
      "concentration optima and to quantify cell line-specific endogenous signaling pathway activation and\n",
      "differentiation bias. We demonstrated that short-term responses predict definitive endoderm\n",
      "induction efficiency and can be used to rescue differentiation of cell lines reticent to cardiac\n",
      "induction. This platform will facilitate high-throughput hPSC-based screening and quantification of\n",
      "lineage-induction bias.\n",
      "['single-cell protein expression profiling',\n",
      " 'high-throughput platform',\n",
      " 'configurable microenvironments',\n",
      " 'colony size optimization',\n",
      " 'cell density optimization']\n",
      "Elapsed time: 11.96 seconds\n",
      "-------------------\n",
      "24162923\n",
      "A superfolding Spinach2 reveals the dynamic nature of trinucleotide repeat-containing RNA.\n",
      "Imaging RNA in living cells is a challenging problem in cell biology. One strategy for genetically\n",
      "encoding fluorescent RNAs is to express them as fusions with Spinach, an 'RNA mimic of GFP'. We\n",
      "found that Spinach was dimmer than expected when used to tag constructs in living cells owing to a\n",
      "combination of thermal instability and a propensity for misfolding. Using systematic mutagenesis, we\n",
      "generated Spinach2 that overcomes these issues and can be used to image diverse RNAs. Using\n",
      "Spinach2, we detailed the dynamics of the CGG trinucleotide repeat-containing 'toxic RNA' associated\n",
      "with Fragile X-associated tremor/ataxia syndrome, and show that these RNAs form nuclear foci with\n",
      "unexpected morphological plasticity that is regulated by the cell cycle and by small molecules.\n",
      "Together, these data demonstrate that Spinach2 exhibits improved versatility for fluorescently\n",
      "labeling RNAs in living cells.\n",
      "['fluorescent RNAs',\n",
      " 'Spinach',\n",
      " 'thermal instability',\n",
      " 'misfolding',\n",
      " 'systematic mutagenesis',\n",
      " 'Spinach2']\n",
      "Elapsed time: 11.73 seconds\n",
      "-------------------\n",
      "24213166\n",
      "Quantifying the local resolution of cryo-EM density maps.\n",
      "We propose a definition of local resolution for three-dimensional electron cryo-microscopy (cryo-EM)\n",
      "density maps that uses local sinusoidal features. Our algorithm has no free parameters and is\n",
      "applicable to other imaging modalities, including tomography. By evaluating the local resolution of\n",
      "single-particle reconstructions and subtomogram averages for four example data sets, we report\n",
      "variable resolution across a 4- to 40-Å range.\n",
      "['cryo-EM',\n",
      " 'sinusoidal features',\n",
      " 'tomography',\n",
      " 'single-particle reconstructions',\n",
      " 'subtomogram averages']\n",
      "Elapsed time: 7.25 seconds\n",
      "-------------------\n",
      "24213167\n",
      "Parallel measurement of dynamic changes in translation rates in single cells.\n",
      "Protein concentrations are often regulated by dynamic changes in translation rates. Nevertheless, it\n",
      "has been challenging to directly monitor changes in translation in living cells. We have developed a\n",
      "reporter system to measure real-time changes of translation rates in human or mouse individual cells\n",
      "by conjugating translation regulatory motifs to sequences encoding a nuclear targeted fluorescent\n",
      "protein and a controllable destabilization domain. Application of the method showed that individual\n",
      "cells undergo marked fluctuations in the translation rate of mRNAs whose 5' terminal oligopyrimidine\n",
      "(5' TOP) motif regulates the synthesis of ribosomal proteins. Furthermore, we show that small\n",
      "reductions in amino acid levels signal through different mTOR-dependent pathways to control TOP mRNA\n",
      "translation, whereas larger reductions in amino acid levels control translation through eIF2A. Our\n",
      "study demonstrates that dynamic measurements of single-cell activities of translation regulatory\n",
      "motifs can be used to identify and investigate fundamental principles of translation.\n",
      "['fluorescent protein', 'destabilization domain', 'nuclear targeted']\n",
      "Elapsed time: 9.79 seconds\n",
      "-------------------\n",
      "24240319\n",
      "Integrating protein-protein interaction networks with phenotypes reveals signs of interactions.\n",
      "A major objective of systems biology is to organize molecular interactions as networks and to\n",
      "characterize information flow within networks. We describe a computational framework to integrate\n",
      "protein-protein interaction (PPI) networks and genetic screens to predict the 'signs' of\n",
      "interactions (i.e., activation-inhibition relationships). We constructed a Drosophila melanogaster\n",
      "signed PPI network consisting of 6,125 signed PPIs connecting 3,352 proteins that can be used to\n",
      "identify positive and negative regulators of signaling pathways and protein complexes. We identified\n",
      "an unexpected role for the metabolic enzymes enolase and aldo-keto reductase as positive and\n",
      "negative regulators of proteolysis, respectively. Characterization of the activation-inhibition\n",
      "relationships between physically interacting proteins within signaling pathways will affect our\n",
      "understanding of many biological functions, including signal transduction and mechanisms of disease.\n",
      "[{'name': 'protein-protein interaction networks'},\n",
      " {'name': 'genetic screens'},\n",
      " {'name': 'signed PPI network'}]\n",
      "Elapsed time: 10.09 seconds\n",
      "-------------------\n",
      "24240320\n",
      "Deciphering laminar-specific neural inputs with line-scanning fMRI.\n",
      "Using a line-scanning method during functional magnetic resonance imaging (fMRI), we obtained high\n",
      "temporal (50-ms) and spatial (50-μm) resolution information along the cortical thickness and showed\n",
      "that the laminar position of fMRI onset coincides with distinct neural inputs in rat somatosensory\n",
      "and motor cortices. This laminar-specific fMRI onset allowed us to identify the neural inputs\n",
      "underlying ipsilateral fMRI activation in the barrel cortex due to peripheral denervation-induced\n",
      "plasticity.\n",
      "['functional magnetic resonance imaging',\n",
      " 'line-scanning fMRI',\n",
      " 'high temporal resolution',\n",
      " 'high spatial resolution']\n",
      "Elapsed time: 7.84 seconds\n",
      "-------------------\n",
      "24292484\n",
      "Niche-independent high-purity cultures of Lgr5+ intestinal stem cells and their progeny.\n",
      "Although Lgr5(+) intestinal stem cells have been expanded in vitro as organoids, homogeneous culture\n",
      "of these cells has not been possible thus far. Here we show that two small molecules, CHIR99021 and\n",
      "valproic acid, synergistically maintain self-renewal of mouse Lgr5(+) intestinal stem cells,\n",
      "resulting in nearly homogeneous cultures. The colony-forming efficiency of cells from these cultures\n",
      "is ~100-fold greater than that of cells cultured in the absence of CHIR99021 and valproic acid, and\n",
      "multilineage differentiation ability is preserved. We made use of these homogeneous cultures to\n",
      "identify conditions employing simultaneous modulation of Wnt and Notch signaling to direct lineage\n",
      "differentiation into mature enterocytes, goblet cells and Paneth cells. Expansion in these culture\n",
      "conditions may be feasible for Lgr5(+) cells from the mouse stomach and colon and from the human\n",
      "small intestine. These methods provide new tools for the study and application of multiple\n",
      "intestinal epithelial cell types.\n",
      "['organoids', 'simultaneous modulation of Wnt and Notch signaling']\n",
      "Elapsed time: 11.61 seconds\n",
      "-------------------\n",
      "24292485\n",
      "A chemoproteomic platform to quantitatively map targets of lipid-derived electrophiles.\n",
      "Cells produce electrophilic products with the potential to modify and affect the function of\n",
      "proteins. Chemoproteomic methods have provided a means to qualitatively inventory proteins targeted\n",
      "by endogenous electrophiles; however, ascertaining the potency and specificity of these reactions to\n",
      "identify the sites in the proteome that are most sensitive to electrophilic modification requires\n",
      "more quantitative methods. Here we describe a competitive activity-based profiling method for\n",
      "quantifying the reactivity of electrophilic compounds against >1,000 cysteines in parallel in the\n",
      "human proteome. Using this approach, we identified a select set of proteins that constitute 'hot\n",
      "spots' for modification by various lipid-derived electrophiles, including the oxidative stress\n",
      "product 4-hydroxy-2-nonenal (HNE). We show that one of these proteins, ZAK kinase, is labeled by HNE\n",
      "on a conserved, active site-proximal cysteine and that the resulting enzyme inhibition creates a\n",
      "negative feedback mechanism that can suppress the activation of JNK pathways normally induced by\n",
      "oxidative stress.\n",
      "['competitive activity-based profiling',\n",
      " 'chemoproteomic methods',\n",
      " 'activity-based profiling']\n",
      "Elapsed time: 11.69 seconds\n",
      "-------------------\n",
      "24317254\n",
      "Quantifying cell-generated mechanical forces within living embryonic tissues.\n",
      "Cell-generated mechanical forces play a critical role during tissue morphogenesis and organ\n",
      "formation in the embryo. Little is known about how these forces shape embryonic organs, mainly\n",
      "because it has not been possible to measure cellular forces within developing three-dimensional (3D)\n",
      "tissues in vivo. We present a method to quantify cell-generated mechanical stresses exerted locally\n",
      "within living embryonic tissues, using fluorescent, cell-sized oil microdroplets with defined\n",
      "mechanical properties and coated with adhesion receptor ligands. After a droplet is introduced\n",
      "between cells in a tissue, local stresses are determined from droplet shape deformations, measured\n",
      "using fluorescence microscopy and computerized image analysis. Using this method, we quantified the\n",
      "anisotropic stresses generated by mammary epithelial cells cultured within 3D aggregates, and we\n",
      "confirmed that these stresses (3.4 nN μm(-2)) are dependent on myosin II activity and are more than\n",
      "twofold larger than stresses generated by cells of embryonic tooth mesenchyme, either within\n",
      "cultured aggregates or in developing whole mouse mandibles.\n",
      "[{'name': 'fluorescence microscopy'}, {'name': 'computerized image analysis'}]\n",
      "Elapsed time: 11.36 seconds\n",
      "-------------------\n",
      "24363022\n",
      "Optogenetic control of Drosophila using a red-shifted channelrhodopsin reveals experience-dependent\n",
      "influences on courtship.\n",
      "Optogenetics allows the manipulation of neural activity in freely moving animals with millisecond\n",
      "precision, but its application in Drosophila melanogaster has been limited. Here we show that a\n",
      "recently described red activatable channelrhodopsin (ReaChR) permits control of complex behavior in\n",
      "freely moving adult flies, at wavelengths that are not thought to interfere with normal visual\n",
      "function. This tool affords the opportunity to control neural activity over a broad dynamic range of\n",
      "stimulation intensities. Using time-resolved activation, we show that the neural control of male\n",
      "courtship song can be separated into (i) probabilistic, persistent and (ii) deterministic, command-\n",
      "like components. The former, but not the latter, neurons are subject to functional modulation by\n",
      "social experience, which supports the idea that they constitute a locus of state-dependent\n",
      "influence. This separation is not evident using thermogenetic tools, a result underscoring the\n",
      "importance of temporally precise control of neuronal activation in the functional dissection of\n",
      "neural circuits in Drosophila.\n",
      "['optogenetics',\n",
      " 'channelrhodopsin',\n",
      " 'red-shifted channelrhodopsin',\n",
      " 'time-resolved activation']\n",
      "Elapsed time: 11.21 seconds\n",
      "-------------------\n",
      "24390439\n",
      "Single-molecule evaluation of fluorescent protein photoactivation efficiency using an in vivo\n",
      "nanotemplate.\n",
      "Photoswitchable fluorescent probes are central to localization-based super-resolution microscopy.\n",
      "Among these probes, fluorescent proteins are appealing because they are genetically encoded.\n",
      "Moreover, the ability to achieve a 1:1 labeling ratio between the fluorescent protein and the\n",
      "protein of interest makes these probes attractive for quantitative single-molecule counting. The\n",
      "percentage of fluorescent protein that is photoactivated into a fluorescently detectable form (i.e.,\n",
      "the photoactivation efficiency) plays a crucial part in properly interpreting the quantitative\n",
      "information. It is important to characterize the photoactivation efficiency at the single-molecule\n",
      "level under the conditions used in super-resolution imaging. Here, we used the human glycine\n",
      "receptor expressed in Xenopus oocytes and stepwise photobleaching or single-molecule counting\n",
      "photoactivated localization microcopy (PALM) to determine the photoactivation efficiency of\n",
      "fluorescent proteins mEos2, mEos3.1, mEos3.2, Dendra2, mClavGR2, mMaple, PA-GFP and PA-mCherry. This\n",
      "analysis provides important information that must be considered when using these fluorescent\n",
      "proteins in quantitative super-resolution microscopy.\n",
      "['single-molecule counting photoactivated localization microcopy (PALM',\n",
      " 'stepwise photobleaching',\n",
      " 'fluorescent protein photoactivation efficiency']\n",
      "Elapsed time: 13.50 seconds\n",
      "-------------------\n",
      "24390440\n",
      "Optimized ratiometric calcium sensors for functional in vivo imaging of neurons and T lymphocytes.\n",
      "The quality of genetically encoded calcium indicators (GECIs) has improved dramatically in recent\n",
      "years, but high-performing ratiometric indicators are still rare. Here we describe a series of\n",
      "fluorescence resonance energy transfer (FRET)-based calcium biosensors with a reduced number of\n",
      "calcium binding sites per sensor. These 'Twitch' sensors are based on the C-terminal domain of\n",
      "Opsanus troponin C. Their FRET responses were optimized by a large-scale functional screen in\n",
      "bacterial colonies, refined by a secondary screen in rat hippocampal neuron cultures. We tested the\n",
      "in vivo performance of the most sensitive variants in the brain and lymph nodes of mice. The\n",
      "sensitivity of the Twitch sensors matched that of synthetic calcium dyes and allowed visualization\n",
      "of tonic action potential firing in neurons and high resolution functional tracking of T\n",
      "lymphocytes. Given their ratiometric readout, their brightness, large dynamic range and linear\n",
      "response properties, Twitch sensors represent versatile tools for neuroscience and immunology.\n",
      "['fluorescence resonance energy transfer (FRET)',\n",
      " 'functional screen',\n",
      " 'ratiometric calcium biosensors',\n",
      " 'brightness',\n",
      " 'large dynamic range',\n",
      " 'linear response properties']\n",
      "Elapsed time: 11.75 seconds\n",
      "-------------------\n",
      "24412976\n",
      "Transcriptome in vivo analysis (TIVA) of spatially defined single cells in live tissue.\n",
      "Transcriptome profiling of single cells resident in their natural microenvironment depends upon RNA\n",
      "capture methods that are both noninvasive and spatially precise. We engineered a transcriptome in\n",
      "vivo analysis (TIVA) tag, which upon photoactivation enables mRNA capture from single cells in live\n",
      "tissue. Using the TIVA tag in combination with RNA sequencing (RNA-seq), we analyzed transcriptome\n",
      "variance among single neurons in culture and in mouse and human tissue in vivo. Our data showed that\n",
      "the tissue microenvironment shapes the transcriptomic landscape of individual cells. The TIVA\n",
      "methodology is, to our knowledge, the first noninvasive approach for capturing mRNA from live single\n",
      "cells in their natural microenvironment.\n",
      "['RNA sequencing (RNA-seq)',\n",
      " 'transcriptome profiling',\n",
      " 'noninvasive mRNA capture methods',\n",
      " 'spatially precise mRNA capture methods',\n",
      " 'photactivation']\n",
      "Elapsed time: 9.49 seconds\n",
      "-------------------\n",
      "24412977\n",
      "Scalable inference of heterogeneous reaction kinetics from pooled single-cell recordings.\n",
      "Mathematical methods combined with measurements of single-cell dynamics provide a means to\n",
      "reconstruct intracellular processes that are only partly or indirectly accessible experimentally. To\n",
      "obtain reliable reconstructions, the pooling of measurements from several cells of a clonal\n",
      "population is mandatory. However, cell-to-cell variability originating from diverse sources poses\n",
      "computational challenges for such process reconstruction. We introduce a scalable Bayesian inference\n",
      "framework that properly accounts for population heterogeneity. The method allows inference of\n",
      "inaccessible molecular states and kinetic parameters; computation of Bayes factors for model\n",
      "selection; and dissection of intrinsic, extrinsic and technical noise. We show how additional\n",
      "single-cell readouts such as morphological features can be included in the analysis. We use the\n",
      "method to reconstruct the expression dynamics of a gene under an inducible promoter in yeast from\n",
      "time-lapse microscopy data.\n",
      "['Bayesian inference framework', 'time-lapse microscopy data']\n",
      "Elapsed time: 9.09 seconds\n",
      "-------------------\n",
      "24441936\n",
      "Objective comparison of particle tracking methods.\n",
      "Particle tracking is of key importance for quantitative analysis of intracellular dynamic processes\n",
      "from time-lapse microscopy image data. Because manually detecting and following large numbers of\n",
      "individual particles is not feasible, automated computational methods have been developed for these\n",
      "tasks by many groups. Aiming to perform an objective comparison of methods, we gathered the\n",
      "community and organized an open competition in which participating teams applied their own methods\n",
      "independently to a commonly defined data set including diverse scenarios. Performance was assessed\n",
      "using commonly defined measures. Although no single method performed best across all scenarios, the\n",
      "results revealed clear differences between the various approaches, leading to notable practical\n",
      "conclusions for users and developers.\n",
      "['particle tracking',\n",
      " 'time-lapse microscopy',\n",
      " 'computational methods',\n",
      " 'automated detection']\n",
      "Elapsed time: 7.81 seconds\n",
      "-------------------\n",
      "24464288\n",
      "Correlative super-resolution fluorescence and metal-replica transmission electron microscopy.\n",
      "We combine super-resolution localization fluorescence microscopy with transmission electron\n",
      "microscopy of metal replicas to locate proteins on the landscape of the cellular plasma membrane at\n",
      "the nanoscale. We validate robust correlation on the scale of 20 nm by imaging endogenous clathrin\n",
      "(in two and three dimensions) and apply the method to find the previously unknown three-dimensional\n",
      "position of the endocytic protein epsin on clathrin-coated structures at the plasma membrane.\n",
      "['super-resolution fluorescence microscopy',\n",
      " 'transmission electron microscopy',\n",
      " 'metal replicas']\n",
      "Elapsed time: 7.14 seconds\n",
      "-------------------\n",
      "24487583\n",
      "Multiplexed 3D cellular super-resolution imaging with DNA-PAINT and Exchange-PAINT.\n",
      "Super-resolution fluorescence microscopy is a powerful tool for biological research, but obtaining\n",
      "multiplexed images for a large number of distinct target species remains challenging. Here we use\n",
      "the transient binding of short fluorescently labeled oligonucleotides (DNA-PAINT, a variation of\n",
      "point accumulation for imaging in nanoscale topography) for simple and easy-to-implement multiplexed\n",
      "super-resolution imaging that achieves sub-10-nm spatial resolution in vitro on synthetic DNA\n",
      "structures. We also report a multiplexing approach (Exchange-PAINT) that allows sequential imaging\n",
      "of multiple targets using only a single dye and a single laser source. We experimentally demonstrate\n",
      "ten-color super-resolution imaging in vitro on synthetic DNA structures as well as four-color two-\n",
      "dimensional (2D) imaging and three-color 3D imaging of proteins in fixed cells.\n",
      "['DNA-PAINT', 'Exchange-PAINT', 'super-resolution fluorescence microscopy']\n",
      "Elapsed time: 9.66 seconds\n",
      "-------------------\n",
      "24509630\n",
      "A synthetic luciferin improves bioluminescence imaging in live mice.\n",
      "Firefly luciferase is the most widely used optical reporter for noninvasive bioluminescence imaging\n",
      "(BLI) in rodents. BLI relies on the ability of the injected luciferase substrate D-luciferin to\n",
      "access luciferase-expressing cells and tissues within the animal. Here we show that injection of\n",
      "mice with a synthetic luciferin, CycLuc1, improves BLI with existing luciferase reporters and\n",
      "enables imaging in the brain that could not be achieved with D-luciferin.\n",
      "['bioluminescence imaging', 'noninvasive bioluminescence imaging', 'BLI']\n",
      "Elapsed time: 7.80 seconds\n",
      "-------------------\n",
      "24509633\n",
      "Independent optical excitation of distinct neural populations.\n",
      "Optogenetic tools enable examination of how specific cell types contribute to brain circuit\n",
      "functions. A long-standing question is whether it is possible to independently activate two distinct\n",
      "neural populations in mammalian brain tissue. Such a capability would enable the study of how\n",
      "different synapses or pathways interact to encode information in the brain. Here we describe two\n",
      "channelrhodopsins, Chronos and Chrimson, discovered through sequencing and physiological\n",
      "characterization of opsins from over 100 species of alga. Chrimson's excitation spectrum is red\n",
      "shifted by 45 nm relative to previous channelrhodopsins and can enable experiments in which red\n",
      "light is preferred. We show minimal visual system-mediated behavioral interference when using\n",
      "Chrimson in neurobehavioral studies in Drosophila melanogaster. Chronos has faster kinetics than\n",
      "previous channelrhodopsins yet is effectively more light sensitive. Together these two reagents\n",
      "enable two-color activation of neural spiking and downstream synaptic transmission in independent\n",
      "neural populations without detectable cross-talk in mouse brain slice.\n",
      "['optogenetic tools',\n",
      " 'two-color activation',\n",
      " 'independent neural populations',\n",
      " 'minimal visual system-mediated behavioral interference',\n",
      " 'neurobehavioral studies',\n",
      " 'mouse brain slice']\n",
      "Elapsed time: 11.50 seconds\n",
      "-------------------\n",
      "24531421\n",
      "Cryo-scanning transmission electron tomography of vitrified cells.\n",
      "Cryo-electron tomography (CET) of fully hydrated, vitrified biological specimens has emerged as a\n",
      "vital tool for biological research. For cellular studies, the conventional imaging modality of\n",
      "transmission electron microscopy places stringent constraints on sample thickness because of its\n",
      "dependence on phase coherence for contrast generation. Here we demonstrate the feasibility of using\n",
      "scanning transmission electron microscopy for cryo-tomography of unstained vitrified specimens\n",
      "(CSTET). We compare CSTET and CET for the imaging of whole bacteria and human tissue culture cells,\n",
      "finding favorable contrast and detail in the CSTET reconstructions. Particularly at high sample\n",
      "tilts, the CSTET signals contain more informative data than energy-filtered CET phase contrast\n",
      "images, resulting in improved depth resolution. Careful control over dose delivery permits\n",
      "relatively high cumulative exposures before the onset of observable beam damage. The increase in\n",
      "acceptable specimen thickness broadens the applicability of electron cryo-tomography.\n",
      "['scanning transmission electron microscopy',\n",
      " 'cryo-electron tomography',\n",
      " 'transmission electron microscopy']\n",
      "Elapsed time: 10.18 seconds\n",
      "-------------------\n",
      "24562422\n",
      "Gold rotor bead tracking for high-speed measurements of DNA twist, torque and extension.\n",
      "Single-molecule measurements of DNA twist and extension have been used to reveal physical properties\n",
      "of the double helix and to characterize structural dynamics and mechanochemistry in nucleoprotein\n",
      "complexes. However, the spatiotemporal resolution of twist measurements has been limited by the use\n",
      "of angular probes with high rotational drag, which prevents detection of short-lived intermediates\n",
      "or small angular steps. We introduce gold rotor bead tracking (AuRBT), which yields >100×\n",
      "improvement in time resolution over previous techniques. AuRBT employs gold nanoparticles as bright\n",
      "low-drag rotational and extensional probes, which are monitored by instrumentation that combines\n",
      "magnetic tweezers with objective-side evanescent darkfield microscopy. Our analysis of high-speed\n",
      "structural dynamics of DNA gyrase using AuRBT revealed an unanticipated transient intermediate.\n",
      "AuRBT also enables direct measurements of DNA torque with >50× shorter integration times than\n",
      "previous techniques; we demonstrated high-resolution torque spectroscopy by mapping the\n",
      "conformational landscape of a Z-forming DNA sequence.\n",
      "['magnetic tweezers',\n",
      " 'objective-side evanescent darkfield microscopy',\n",
      " 'angular probes']\n",
      "Elapsed time: 11.24 seconds\n",
      "-------------------\n",
      "24562424\n",
      "Sleep-spindle detection: crowdsourcing and evaluating performance of experts, non-experts and\n",
      "automated methods.\n",
      "Sleep spindles are discrete, intermittent patterns of brain activity observed in human\n",
      "electroencephalographic data. Increasingly, these oscillations are of biological and clinical\n",
      "interest because of their role in development, learning and neurological disorders. We used an\n",
      "Internet interface to crowdsource spindle identification by human experts and non-experts, and we\n",
      "compared their performance with that of automated detection algorithms in data from middle- to\n",
      "older-aged subjects from the general population. We also refined methods for forming group consensus\n",
      "and evaluating the performance of event detectors in physiological data such as\n",
      "electroencephalographic recordings from polysomnography. Compared to the expert group consensus gold\n",
      "standard, the highest performance was by individual experts and the non-expert group consensus,\n",
      "followed by automated spindle detectors. This analysis showed that crowdsourcing the scoring of\n",
      "sleep data is an efficient method to collect large data sets, even for difficult tasks such as\n",
      "spindle identification. Further refinements to spindle detection algorithms are needed for middle-\n",
      "to older-aged subjects.\n",
      "['electroencephalographic data', 'polysomnography']\n",
      "Elapsed time: 10.73 seconds\n",
      "-------------------\n",
      "24577276\n",
      "Precisely and accurately localizing single emitters in fluorescence microscopy.\n",
      "Methods based on single-molecule localization and photophysics have brought nanoscale imaging with\n",
      "visible light into reach. This has enabled single-particle tracking applications for studying the\n",
      "dynamics of molecules and nanoparticles and contributed to the recent revolution in super-resolution\n",
      "localization microscopy techniques. Crucial to the optimization of such methods are the precision\n",
      "and accuracy with which single fluorophores and nanoparticles can be localized. We present a lucid\n",
      "synthesis of the developments on this localization precision and accuracy and their practical\n",
      "implications in order to guide the increasing number of researchers using single-particle tracking\n",
      "and super-resolution localization microscopy.\n",
      "['single-molecule localization',\n",
      " 'photophysics',\n",
      " 'super-resolution localization microscopy',\n",
      " 'single-particle tracking']\n",
      "Elapsed time: 8.30 seconds\n",
      "-------------------\n",
      "24577277\n",
      "Fluorophore localization algorithms for super-resolution microscopy.\n",
      "Super-resolution localization microscopy methods provide powerful new capabilities for probing\n",
      "biology at the nanometer scale via fluorescence. These methods rely on two key innovations:\n",
      "switchable fluorophores (which blink on and off and can be sequentially imaged) and powerful\n",
      "localization algorithms (which estimate the positions of the fluorophores in the images). These\n",
      "techniques have spurred a flurry of innovation in algorithm development over the last several years.\n",
      "In this Review, we survey the fundamental issues for single-fluorophore fitting routines,\n",
      "localization algorithms based on principles other than fitting, three-dimensional imaging, dipole\n",
      "imaging and techniques for estimating fluorophore positions from images of multiple activated\n",
      "fluorophores. We offer practical advice for users and adopters of algorithms, and we identify areas\n",
      "for further development.\n",
      "Invalid json output: {\n",
      "\"fluorophore localization algorithms\",\n",
      "\"single-fluorophore fitting routines\",\n",
      "\"localization algorithms based on principles other than fitting\",\n",
      "\"three-dimensional imaging\",\n",
      "\"dipole imaging\",\n",
      "\"estimating fluorophore positions from images of multiple activated fluorophores\"\n",
      "} derived from Sure! Here is the list of imaging techniques or technologies described in the abstract as a JSON list:\n",
      "\n",
      "{\n",
      "\"fluorophore localization algorithms\",\n",
      "\"single-fluorophore fitting routines\",\n",
      "\"localization algorithms based on principles other than fitting\",\n",
      "\"three-dimensional imaging\",\n",
      "\"dipole imaging\",\n",
      "\"estimating fluorophore positions from images of multiple activated fluorophores\"\n",
      "}\n",
      "24584193\n",
      "Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry.\n",
      "Mass cytometry enables high-dimensional, single-cell analysis of cell type and state. In mass\n",
      "cytometry, rare earth metals are used as reporters on antibodies. Analysis of metal abundances using\n",
      "the mass cytometer allows determination of marker expression in individual cells. Mass cytometry has\n",
      "previously been applied only to cell suspensions. To gain spatial information, we have coupled\n",
      "immunohistochemical and immunocytochemical methods with high-resolution laser ablation to CyTOF mass\n",
      "cytometry. This approach enables the simultaneous imaging of 32 proteins and protein modifications\n",
      "at subcellular resolution; with the availability of additional isotopes, measurement of over 100\n",
      "markers will be possible. We applied imaging mass cytometry to human breast cancer samples, allowing\n",
      "delineation of cell subpopulations and cell-cell interactions and highlighting tumor heterogeneity.\n",
      "Imaging mass cytometry complements existing imaging approaches. It will enable basic studies of\n",
      "tissue heterogeneity and function and support the transition of medicine toward individualized\n",
      "molecularly targeted diagnosis and therapies.\n",
      "['mass cytometry',\n",
      " 'immunohistochemical methods',\n",
      " 'immunocytochemical methods',\n",
      " 'high-resolution laser ablation',\n",
      " 'CyTOF mass cytometry']\n",
      "Elapsed time: 11.79 seconds\n",
      "-------------------\n",
      "24584195\n",
      "Live-cell imaging of alkyne-tagged small biomolecules by stimulated Raman scattering.\n",
      "Sensitive and specific visualization of small biomolecules in living systems is highly challenging.\n",
      "We report stimulated Raman-scattering imaging of alkyne tags as a general strategy for studying a\n",
      "broad spectrum of small biomolecules in live cells and animals. We demonstrate this technique by\n",
      "tracking alkyne-bearing drugs in mouse tissues and visualizing de novo synthesis of DNA, RNA,\n",
      "proteins, phospholipids and triglycerides through metabolic incorporation of alkyne-tagged small\n",
      "precursors.\n",
      "['stimulated Raman scattering', 'live-cell imaging']\n",
      "Elapsed time: 7.36 seconds\n",
      "-------------------\n",
      "24608180\n",
      "Dynamic characterization of growth and gene expression using high-throughput automated flow\n",
      "cytometry.\n",
      "Cells adjust to changes in environmental conditions using complex regulatory programs. These\n",
      "cellular programs are the result of an intricate interplay between gene expression, cellular growth\n",
      "and protein degradation. Technologies that enable simultaneous and time-resolved measurements of\n",
      "these variables are necessary to dissect cellular homeostatic strategies. Here we report the\n",
      "development of an automated flow cytometry robotic setup that enables real-time measurement of\n",
      "precise and simultaneous relative growth and protein synthesis rates of multiplexed microbial\n",
      "populations across many conditions. These measurements generate quantitative profiles of dynamically\n",
      "evolving protein synthesis and degradation rates. We demonstrate this setup in the context of gene\n",
      "regulation of the unfolded protein response (UPR) of Saccharomyces cerevisiae and uncover a dynamic\n",
      "and complex landscape of gene expression, growth dynamics and proteolysis following perturbations.\n",
      "['flow cytometry',\n",
      " 'automated flow cytometry robotic setup',\n",
      " 'real-time measurement']\n",
      "Elapsed time: 8.70 seconds\n",
      "-------------------\n",
      "24633408\n",
      "Non-invasive intravital imaging of cellular differentiation with a bright red-excitable fluorescent\n",
      "protein.\n",
      "A method for non-invasive visualization of genetically labeled cells in animal disease models with\n",
      "micrometer-level resolution would greatly facilitate development of cell-based therapies. Imaging of\n",
      "fluorescent proteins (FPs) using red excitation light in the 'optical window' above 600 nm is one\n",
      "potential method for visualizing implanted cells. However, previous efforts to engineer FPs with\n",
      "peak excitation beyond 600 nm have resulted in undesirable reductions in brightness. Here we report\n",
      "three new red-excitable monomeric FPs obtained by structure-guided mutagenesis of mNeptune. Two of\n",
      "these, mNeptune2 and mNeptune2.5, demonstrate improved maturation and brighter fluorescence than\n",
      "mNeptune, whereas the third, mCardinal, has a red-shifted excitation spectrum without reduction in\n",
      "brightness. We show that mCardinal can be used to non-invasively and longitudinally visualize the\n",
      "differentiation of myoblasts into myocytes in living mice with high anatomical detail.\n",
      "['optical window',\n",
      " 'red excitation light',\n",
      " 'non-invasive visualization',\n",
      " 'longitudinal visualization']\n",
      "Elapsed time: 11.36 seconds\n",
      "-------------------\n",
      "24633409\n",
      "Accurate macromolecular structures using minimal measurements from X-ray free-electron lasers.\n",
      "X-ray free-electron laser (XFEL) sources enable the use of crystallography to solve three-\n",
      "dimensional macromolecular structures under native conditions and without radiation damage. Results\n",
      "to date, however, have been limited by the challenge of deriving accurate Bragg intensities from a\n",
      "heterogeneous population of microcrystals, while at the same time modeling the X-ray spectrum and\n",
      "detector geometry. Here we present a computational approach designed to extract meaningful high-\n",
      "resolution signals from fewer diffraction measurements.\n",
      "['crystallography', 'X-ray free-electron lasers', 'Bragg intensities']\n",
      "Elapsed time: 7.37 seconds\n",
      "-------------------\n",
      "24705472\n",
      "Fluorescence nanoscopy by polarization modulation and polarization angle narrowing.\n",
      "When excited with rotating linear polarized light, differently oriented fluorescent dyes emit\n",
      "periodic signals peaking at different times. We show that measurement of the average orientation of\n",
      "fluorescent dyes attached to rigid sample structures mapped to regularly defined (50 nm)(2) image\n",
      "nanoareas can provide subdiffraction resolution (super resolution by polarization demodulation,\n",
      "SPoD). Because the polarization angle range for effective excitation of an oriented molecule is\n",
      "rather broad and unspecific, we narrowed this range by simultaneous irradiation with a second, de-\n",
      "excitation, beam possessing a polarization perpendicular to the excitation beam (excitation\n",
      "polarization angle narrowing, ExPAN). This shortened the periodic emission flashes, allowing better\n",
      "discrimination between molecules or nanoareas. Our method requires neither the generation of\n",
      "nanometric interference structures nor the use of switchable or blinking fluorescent probes. We\n",
      "applied the method to standard wide-field microscopy with camera detection and to two-photon\n",
      "scanning microscopy, imaging the fine structural details of neuronal spines.\n",
      "['fluorescence nanoscopy',\n",
      " 'polarization modulation',\n",
      " 'polarization angle narrowing',\n",
      " 'super resolution by polarization demodulation',\n",
      " 'excitation polarization angle narrowing']\n",
      "Elapsed time: 11.04 seconds\n",
      "-------------------\n",
      "24705473\n",
      "Light-sheet microscopy using an Airy beam.\n",
      "Light-sheet microscopy facilitates rapid, high-contrast, volumetric imaging with minimal sample\n",
      "exposure. However, the rapid divergence of a traditional Gaussian light sheet restricts the field of\n",
      "view (FOV) that provides innate subcellular resolution. We show that the Airy beam innately yields\n",
      "high contrast and resolution up to a tenfold larger FOV. In contrast to the Bessel beam, which also\n",
      "provides an increased FOV, the Airy beam's characteristic asymmetric excitation pattern results in\n",
      "all fluorescence contributing positively to the contrast, enabling a step change for light-sheet\n",
      "microscopy.\n",
      "['light-sheet microscopy', 'Gaussian light sheet', 'Airy beam', 'Bessel beam']\n",
      "Elapsed time: 7.77 seconds\n",
      "-------------------\n",
      "24727651\n",
      "Single-molecule analysis of cell surface dynamics in Caenorhabditis elegans embryos.\n",
      "We describe a general, versatile and minimally invasive method to image single molecules near the\n",
      "cell surface that can be applied to any GFP-tagged protein in Caenorhabditis elegans embryos. We\n",
      "exploited tunable expression via RNAi and a dynamically exchanging monomer pool to achieve fast,\n",
      "continuous single-molecule imaging at optimal densities with signal-to-noise ratios adequate for\n",
      "robust single-particle tracking (SPT). We introduce a method called smPReSS, single-molecule\n",
      "photobleaching relaxation to steady state, that infers exchange rates from quantitative analysis of\n",
      "single-molecule photobleaching kinetics without using SPT. Combining SPT and smPReSS allowed for\n",
      "spatially and temporally resolved measurements of protein mobility and exchange kinetics. We used\n",
      "these methods to (i) resolve distinct mobility states and spatial variation in exchange rates of the\n",
      "polarity protein PAR-6 and (ii) measure spatiotemporal modulation of actin filament assembly and\n",
      "disassembly. These methods offer a promising avenue to investigate dynamic mechanisms that pattern\n",
      "the embryonic cell surface.\n",
      "['single-molecule imaging',\n",
      " 'GFP-tagged protein',\n",
      " 'RNAi',\n",
      " 'dynamic exchanging monomer pool',\n",
      " 'single-particle tracking (SPT)',\n",
      " 'single-molecule photobleaching relaxation to steady state (smPReSS)']\n",
      "Elapsed time: 12.80 seconds\n",
      "-------------------\n",
      "24727652\n",
      "Multiscale representation of genomic signals.\n",
      "Genomic information is encoded on a wide range of distance scales, ranging from tens of bases to\n",
      "megabases. We developed a multiscale framework to analyze and visualize the information content of\n",
      "genomic signals. Different types of signals, such as G+C content or DNA methylation, are\n",
      "characterized by distinct patterns of signal enrichment or depletion across scales spanning several\n",
      "orders of magnitude. These patterns are associated with a variety of genomic annotations. By\n",
      "integrating the information across all scales, we demonstrated improved prediction of gene\n",
      "expression from polymerase II chromatin immunoprecipitation sequencing (ChIP-seq) measurements, and\n",
      "we observed that gene expression differences in colorectal cancer are related to methylation\n",
      "patterns that extend beyond the single-gene scale. Our software is available at\n",
      "https://github.com/tknijnen/msr/.\n",
      "['ChIP-seq', 'polymerase II chromatin immunoprecipitation sequencing']\n",
      "Elapsed time: 8.71 seconds\n",
      "-------------------\n",
      "24727653\n",
      "Rapid adaptive optical recovery of optimal resolution over large volumes.\n",
      "Using a descanned, laser-induced guide star and direct wavefront sensing, we demonstrate adaptive\n",
      "correction of complex optical aberrations at high numerical aperture (NA) and a 14-ms update rate.\n",
      "This correction permits us to compensate for the rapid spatial variation in aberration often\n",
      "encountered in biological specimens and to recover diffraction-limited imaging over large volumes\n",
      "(>240 mm per side). We applied this to image fine neuronal processes and subcellular dynamics within\n",
      "the zebrafish brain.\n",
      "['laser-induced guide star',\n",
      " 'direct wavefront sensing',\n",
      " 'adaptive correction of complex optical aberrations',\n",
      " 'diffraction-limited imaging']\n",
      "Elapsed time: 7.32 seconds\n",
      "-------------------\n",
      "24747812\n",
      "Efficient Bayesian-based multiview deconvolution.\n",
      "Light-sheet fluorescence microscopy is able to image large specimens with high resolution by\n",
      "capturing the samples from multiple angles. Multiview deconvolution can substantially improve the\n",
      "resolution and contrast of the images, but its application has been limited owing to the large size\n",
      "of the data sets. Here we present a Bayesian-based derivation of multiview deconvolution that\n",
      "drastically improves the convergence time, and we provide a fast implementation using graphics\n",
      "hardware.\n",
      "['light-sheet fluorescence microscopy', 'multiview deconvolution']\n",
      "Elapsed time: 6.83 seconds\n",
      "-------------------\n",
      "24747813\n",
      "Controlling protein adsorption on graphene for cryo-EM using low-energy hydrogen plasmas.\n",
      "Despite its many favorable properties as a sample support for biological electron microscopy,\n",
      "graphene is not widely used because its hydrophobicity precludes reliable protein deposition. We\n",
      "describe a method to modify graphene with a low-energy hydrogen plasma, which reduces hydrophobicity\n",
      "without degrading the graphene lattice. Use of plasma-treated graphene enables better control of\n",
      "protein distribution in ice for electron cryo-microscopy and improves image quality by reducing\n",
      "radiation-induced sample motion.\n",
      "['electron microscopy', 'cryo-EM', 'radiation-induced sample motion']\n",
      "Elapsed time: 7.19 seconds\n",
      "-------------------\n",
      "24809628\n",
      "Comprehensive analysis of RNA-protein interactions by high-throughput sequencing-RNA affinity\n",
      "profiling.\n",
      "RNA-protein interactions play critical roles in gene regulation, but methods to quantitatively\n",
      "analyze these interactions at a large scale are lacking. We have developed a high-throughput\n",
      "sequencing-RNA affinity profiling (HiTS-RAP) assay by adapting a high-throughput DNA sequencer to\n",
      "quantify the binding of fluorescently labeled protein to millions of RNAs anchored to sequenced cDNA\n",
      "templates. Using HiTS-RAP, we measured the affinity of mutagenized libraries of GFP-binding and\n",
      "NELF-E-binding aptamers to their respective targets and identified critical regions of interaction.\n",
      "Mutations additively affected the affinity of the NELF-E-binding aptamer, whose interaction depended\n",
      "mainly on a single-stranded RNA motif, but not that of the GFP aptamer, whose interaction depended\n",
      "primarily on secondary structure.\n",
      "[{'key': 'HiTS-RAP',\n",
      "  'value': 'high-throughput sequencing-RNA affinity profiling'},\n",
      " {'key': 'DNA sequencer', 'value': 'high-throughput DNA sequencer'}]\n",
      "Elapsed time: 10.99 seconds\n",
      "-------------------\n",
      "24813625\n",
      "Correlated cryogenic photoactivated localization microscopy and cryo-electron tomography.\n",
      "Cryo-electron tomography (CET) produces three-dimensional images of cells in a near-native state at\n",
      "macromolecular resolution, but identifying structures of interest can be challenging. Here we\n",
      "describe a correlated cryo-PALM (photoactivated localization microscopy)-CET method for localizing\n",
      "objects within cryo-tomograms to beyond the diffraction limit of the light microscope. Using cryo-\n",
      "PALM-CET, we identified multiple and new conformations of the dynamic type VI secretion system in\n",
      "the crowded interior of Myxococcus xanthus.\n",
      "Invalid json output: {\n",
      "\"correlated cryo-PALM\",\n",
      "\"cryo-electron tomography\",\n",
      "\"photoactivated localization microscopy\"\n",
      "} derived from Sure thing! Here is the list of imaging techniques described in the abstract:\n",
      "\n",
      "{\n",
      "\"correlated cryo-PALM\",\n",
      "\"cryo-electron tomography\",\n",
      "\"photoactivated localization microscopy\"\n",
      "}\n",
      "24820376\n",
      "Random and targeted transgene insertion in Caenorhabditis elegans using a modified Mos1 transposon.\n",
      "We have generated a recombinant Mos1 transposon that can insert up to 45-kb transgenes into the\n",
      "Caenorhabditis elegans genome. The minimal Mos1 transposon (miniMos) is 550 bp long and inserts DNA\n",
      "into the genome at high frequency (~60% of injected animals). Genetic and antibiotic markers can be\n",
      "used for selection, and the transposon is active in C. elegans isolates and Caenorhabditis briggsae.\n",
      "We used the miniMos transposon to generate six universal Mos1-mediated single-copy insertion\n",
      "(mosSCI) landing sites that allow targeted transgene insertion with a single targeting vector into\n",
      "permissive expression sites on all autosomes. We also generated two collections of strains: a set of\n",
      "bright fluorescent insertions that are useful as dominant, genetic balancers and a set of lacO\n",
      "insertions to track genome position.\n",
      "['bright fluorescent insertions',\n",
      " 'genetic balancers',\n",
      " 'lacO insertions',\n",
      " 'single-copy insertion (mosSCI)',\n",
      " 'targeted transgene insertion']\n",
      "Elapsed time: 10.76 seconds\n",
      "-------------------\n",
      "24836920\n",
      "Simultaneous whole-animal 3D imaging of neuronal activity using light-field microscopy.\n",
      "High-speed, large-scale three-dimensional (3D) imaging of neuronal activity poses a major challenge\n",
      "in neuroscience. Here we demonstrate simultaneous functional imaging of neuronal activity at single-\n",
      "neuron resolution in an entire Caenorhabditis elegans and in larval zebrafish brain. Our technique\n",
      "captures the dynamics of spiking neurons in volumes of ∼700 μm × 700 μm × 200 μm at 20 Hz. Its\n",
      "simplicity makes it an attractive tool for high-speed volumetric calcium imaging.\n",
      "['light-field microscopy',\n",
      " 'whole-animal 3D imaging',\n",
      " 'functional imaging',\n",
      " 'simultaneous imaging']\n",
      "Elapsed time: 8.39 seconds\n",
      "-------------------\n",
      "24859753\n",
      "Fluorogenic probes for live-cell imaging of the cytoskeleton.\n",
      "We introduce far-red, fluorogenic probes that combine minimal cytotoxicity with excellent brightness\n",
      "and photostability for fluorescence imaging of actin and tubulin in living cells. Applied in\n",
      "stimulated emission depletion (STED) microscopy, they reveal the ninefold symmetry of the centrosome\n",
      "and the spatial organization of actin in the axon of cultured rat neurons with a resolution\n",
      "unprecedented for imaging cytoskeletal structures in living cells.\n",
      "['STED microscopy', 'fluorescence imaging', 'live-cell imaging']\n",
      "Elapsed time: 7.16 seconds\n",
      "-------------------\n",
      "24880876\n",
      "Single-cell western blotting.\n",
      "To measure cell-to-cell variation in protein-mediated functions, we developed an approach to conduct\n",
      "∼10(3) concurrent single-cell western blots (scWesterns) in ∼4 h. A microscope slide supporting a\n",
      "30-μm-thick photoactive polyacrylamide gel enables western blotting: settling of single cells into\n",
      "microwells, lysis in situ, gel electrophoresis, photoinitiated blotting to immobilize proteins and\n",
      "antibody probing. We applied this scWestern method to monitor single-cell differentiation of rat\n",
      "neural stem cells and responses to mitogen stimulation. The scWestern quantified target proteins\n",
      "even with off-target antibody binding, multiplexed to 11 protein targets per single cell with\n",
      "detection thresholds of <30,000 molecules, and supported analyses of low starting cell numbers\n",
      "(∼200) when integrated with FACS. The scWestern overcomes limitations of antibody fidelity and\n",
      "sensitivity in other single-cell protein analysis methods and constitutes a versatile tool for the\n",
      "study of complex cell populations at single-cell resolution.\n",
      "['single-cell western blotting',\n",
      " 'microwells',\n",
      " 'gel electrophoresis',\n",
      " 'photoinitiated blotting',\n",
      " 'antibody probing']\n",
      "Elapsed time: 10.94 seconds\n",
      "-------------------\n",
      "24930129\n",
      "Protein delivery into live cells by incubation with an endosomolytic agent.\n",
      "We report that a tetramethylrhodamine-labeled dimer of the cell-penetrating peptide TAT, dfTAT,\n",
      "penetrates live cells by escaping from endosomes with high efficiency. By mediating endosomal\n",
      "leakage, dfTAT also delivers proteins into cultured cells after a simple co-incubation procedure. We\n",
      "achieved cytosolic delivery in several cell lines and primary cells and observed that only a\n",
      "relatively small amount of material remained trapped inside endosomes. Delivery did not require a\n",
      "binding interaction between dfTAT and a protein, multiple molecules could be delivered\n",
      "simultaneously, and delivery could be repeated. dfTAT-mediated delivery did not noticeably affect\n",
      "cell viability, cell proliferation or gene expression. dfTAT-based intracellular delivery should be\n",
      "useful for cell-based assays, cellular imaging applications and the ex vivo manipulation of cells.\n",
      "Invalid json output: [\"none\"], [\"none\"], [\"none\"] derived from Sure! Here is the list of imaging techniques described in the abstract:\n",
      "\n",
      "[\"none\"], [\"none\"], [\"none\"]\n",
      "24930130\n",
      "Chemically defined generation of human cardiomyocytes.\n",
      "Existing methods for human induced pluripotent stem cell (hiPSC) cardiac differentiation are\n",
      "efficient but require complex, undefined medium constituents that hinder further elucidation of the\n",
      "molecular mechanisms of cardiomyogenesis. Using hiPSCs derived under chemically defined conditions\n",
      "on synthetic matrices, we systematically developed an optimized cardiac differentiation strategy,\n",
      "using a chemically defined medium consisting of just three components: the basal medium RPMI 1640,\n",
      "L-ascorbic acid 2-phosphate and rice-derived recombinant human albumin. Along with small molecule-\n",
      "based induction of differentiation, this protocol produced contractile sheets of up to 95% TNNT2(+)\n",
      "cardiomyocytes at a yield of up to 100 cardiomyocytes for every input pluripotent cell and was\n",
      "effective in 11 hiPSC lines tested. This chemically defined platform for cardiac specification of\n",
      "hiPSCs will allow the elucidation of cardiomyocyte macromolecular and metabolic requirements and\n",
      "will provide a minimal system for the study of maturation and subtype specification.\n",
      "['contractile sheets',\n",
      " 'TNNT2(+) cardiomyocytes',\n",
      " 'small molecule-based induction of differentiation']\n",
      "Elapsed time: 11.13 seconds\n",
      "-------------------\n",
      "24952909\n",
      "Annotation of loci from genome-wide association studies using tissue-specific quantitative\n",
      "interaction proteomics.\n",
      "Genome-wide association studies (GWAS) have identified thousands of loci associated with complex\n",
      "traits, but it is challenging to pinpoint causal genes in these loci and to exploit subtle\n",
      "association signals. We used tissue-specific quantitative interaction proteomics to map a network of\n",
      "five genes involved in the Mendelian disorder long QT syndrome (LQTS). We integrated the LQTS\n",
      "network with GWAS loci from the corresponding common complex trait, QT-interval variation, to\n",
      "identify candidate genes that were subsequently confirmed in Xenopus laevis oocytes and zebrafish.\n",
      "We used the LQTS protein network to filter weak GWAS signals by identifying single-nucleotide\n",
      "polymorphisms (SNPs) in proximity to genes in the network supported by strong proteomic evidence.\n",
      "Three SNPs passing this filter reached genome-wide significance after replication genotyping.\n",
      "Overall, we present a general strategy to propose candidates in GWAS loci for functional studies and\n",
      "to systematically filter subtle association signals using tissue-specific quantitative interaction\n",
      "proteomics.\n",
      "['proteomics',\n",
      " 'quantitative interaction proteomics',\n",
      " 'tissue-specific quantitative interaction proteomics']\n",
      "Elapsed time: 10.95 seconds\n",
      "-------------------\n",
      "24952910\n",
      "All-optical electrophysiology in mammalian neurons using engineered microbial rhodopsins.\n",
      "All-optical electrophysiology-spatially resolved simultaneous optical perturbation and measurement\n",
      "of membrane voltage-would open new vistas in neuroscience research. We evolved two archaerhodopsin-\n",
      "based voltage indicators, QuasAr1 and QuasAr2, which show improved brightness and voltage\n",
      "sensitivity, have microsecond response times and produce no photocurrent. We engineered a\n",
      "channelrhodopsin actuator, CheRiff, which shows high light sensitivity and rapid kinetics and is\n",
      "spectrally orthogonal to the QuasArs. A coexpression vector, Optopatch, enabled cross-talk-free\n",
      "genetically targeted all-optical electrophysiology. In cultured rat neurons, we combined Optopatch\n",
      "with patterned optical excitation to probe back-propagating action potentials (APs) in dendritic\n",
      "spines, synaptic transmission, subcellular microsecond-timescale details of AP propagation, and\n",
      "simultaneous firing of many neurons in a network. Optopatch measurements revealed homeostatic tuning\n",
      "of intrinsic excitability in human stem cell-derived neurons. In rat brain slices, Optopatch induced\n",
      "and reported APs and subthreshold events with high signal-to-noise ratios. The Optopatch platform\n",
      "enables high-throughput, spatially resolved electrophysiology without the use of conventional\n",
      "electrodes.\n",
      "{'imagingTechniques': ['all-optical electrophysiology',\n",
      "                       'spatially resolved simultaneous optical perturbation '\n",
      "                       'and measurement of membrane voltage',\n",
      "                       'engineered microbial rhodopsins']}\n",
      "Elapsed time: 13.48 seconds\n",
      "-------------------\n",
      "24973948\n",
      "A method to recapitulate early embryonic spatial patterning in human embryonic stem cells.\n",
      "Embryos allocate cells to the three germ layers in a spatially ordered sequence. Human embryonic\n",
      "stem cells (hESCs) can generate the three germ layers in culture; however, differentiation is\n",
      "typically heterogeneous and spatially disordered. We show that geometric confinement is sufficient\n",
      "to trigger self-organized patterning in hESCs. In response to BMP4, colonies reproducibly\n",
      "differentiated to an outer trophectoderm-like ring, an inner ectodermal circle and a ring of\n",
      "mesendoderm expressing primitive-streak markers in between. Fates were defined relative to the\n",
      "boundary with a fixed length scale: small colonies corresponded to the outer layers of larger ones.\n",
      "Inhibitory signals limited the range of BMP4 signaling to the colony edge and induced a gradient of\n",
      "Activin-Nodal signaling that patterned mesendodermal fates. These results demonstrate that the\n",
      "intrinsic tendency of stem cells to make patterns can be harnessed by controlling colony geometries\n",
      "and provide a quantitative assay for studying paracrine signaling in early development.\n",
      "['geometric confinement',\n",
      " 'colonies',\n",
      " 'outer trophectoderm-like ring',\n",
      " 'inner ectodermal circle',\n",
      " 'mesendoderm expressing primitive-streak markers',\n",
      " 'fixed length scale',\n",
      " 'inhibitory signals',\n",
      " 'Activin-Nodal signaling']\n",
      "Elapsed time: 12.07 seconds\n",
      "-------------------\n",
      "24997861\n",
      "Accelerated chromatin biochemistry using DNA-barcoded nucleosome libraries.\n",
      "Elucidating the molecular details of how chromatin-associated factors deposit, remove and recognize\n",
      "histone post-translational modification (PTM) signatures remains a daunting task in the epigenetics\n",
      "field. We introduce a versatile platform that greatly accelerates biochemical investigations into\n",
      "chromatin recognition and signaling. This technology is based on the streamlined semisynthesis of\n",
      "DNA-barcoded nucleosome libraries with distinct combinations of PTMs. Chromatin immunoprecipitation\n",
      "of these libraries, once they have been treated with purified chromatin effectors or the combined\n",
      "chromatin recognizing and modifying activities of the nuclear proteome, is followed by multiplexed\n",
      "DNA-barcode sequencing. This ultrasensitive workflow allowed us to collect thousands of biochemical\n",
      "data points revealing the binding preferences of various nuclear factors for PTM patterns and how\n",
      "preexisting PTMs, alone or synergistically, affect further PTM deposition via cross-talk mechanisms.\n",
      "We anticipate that the high throughput and sensitivity of the technology will help accelerate the\n",
      "decryption of the diverse molecular controls that operate at the level of chromatin.\n",
      "[['chromatin immunoprecipitation'], ['multiplexed DNA-barcode sequencing']]\n",
      "Elapsed time: 10.60 seconds\n",
      "-------------------\n",
      "25028895\n",
      "Isolation of rare recombinants without using selectable markers for one-step seamless BAC\n",
      "mutagenesis.\n",
      "Current methods to isolate rare (1:10,000-1:100,000) bacterial artificial chromosome (BAC)\n",
      "recombinants require selectable markers. For seamless BAC mutagenesis, selectable markers need to be\n",
      "removed after isolation of recombinants through counterselection. Here we illustrate founder\n",
      "principle-driven enrichment (FPE), a simple method to rapidly isolate rare recombinants without\n",
      "using selectable markers, allowing one-step seamless BAC mutagenesis. As proof of principle, we\n",
      "isolated 1:100,000 seamless fluorescent protein-modified Nodal BACs and confirmed BAC functionality\n",
      "by generating fluorescent reporter mice. We also isolated small indel P1 phage-derived artificial\n",
      "chromosome (PAC) and BAC recombinants. Statistical analysis revealed that 1:100,000 recombinants can\n",
      "be isolated with <40 PCRs, and we developed a web-based calculator to optimize FPE.\n",
      "['PCR',\n",
      " 'fluorescent protein-modified Nodal BACs',\n",
      " 'fluorescent reporter mice',\n",
      " 'P1 phage-derived artificial chromosome (PAC) recombinants']\n",
      "Elapsed time: 11.29 seconds\n",
      "-------------------\n",
      "25042784\n",
      "Optimized cell transplantation using adult rag2 mutant zebrafish.\n",
      "Cell transplantation into adult zebrafish has lagged behind mouse models owing to the lack of\n",
      "immunocompromised strains. Here we have created rag2(E450fs) mutant zebrafish that have reduced\n",
      "numbers of functional T and B cells but are viable and fecund. Mutant fish engraft muscle, blood\n",
      "stem cells and various cancers. rag2(E450fs) mutant zebrafish are the first immunocompromised\n",
      "zebrafish model that permits robust, long-term engraftment of multiple tissues and cancer.\n",
      "['muscle engraftment', 'blood stem cells engraftment', 'cancer engraftment']\n",
      "Elapsed time: 7.78 seconds\n",
      "-------------------\n",
      "25042785\n",
      "Fast, accurate reconstruction of cell lineages from large-scale fluorescence microscopy data.\n",
      "The comprehensive reconstruction of cell lineages in complex multicellular organisms is a central\n",
      "goal of developmental biology. We present an open-source computational framework for the\n",
      "segmentation and tracking of cell nuclei with high accuracy and speed. We demonstrate its (i)\n",
      "generality by reconstructing cell lineages in four-dimensional, terabyte-sized image data sets of\n",
      "fruit fly, zebrafish and mouse embryos acquired with three types of fluorescence microscopes, (ii)\n",
      "scalability by analyzing advanced stages of development with up to 20,000 cells per time point at\n",
      "26,000 cells min(-1) on a single computer workstation and (iii) ease of use by adjusting only two\n",
      "parameters across all data sets and providing visualization and editing tools for efficient data\n",
      "curation. Our approach achieves on average 97.0% linkage accuracy across all species and imaging\n",
      "modalities. Using our system, we performed the first cell lineage reconstruction of early Drosophila\n",
      "melanogaster nervous system development, revealing neuroblast dynamics throughout an entire embryo.\n",
      "['fluorescence microscopy',\n",
      " 'terabyte-sized image data sets',\n",
      " 'three types of fluorescence microscopes']\n",
      "Elapsed time: 10.92 seconds\n",
      "-------------------\n",
      "25042787\n",
      "High-resolution reconstruction of the beating zebrafish heart.\n",
      "The heart's continuous motion makes it difficult to capture high-resolution images of this organ in\n",
      "vivo. We developed tools based on high-speed selective plane illumination microscopy (SPIM),\n",
      "offering pristine views into the beating zebrafish heart. We captured three-dimensional cardiac\n",
      "dynamics with postacquisition synchronization of multiview movie stacks, obtained static high-\n",
      "resolution reconstructions by briefly stopping the heart with optogenetics and resolved nonperiodic\n",
      "phenomena by high-speed volume scanning with a liquid lens.\n",
      "['selective plane illumination microscopy (SPIM)',\n",
      " 'optogenetics',\n",
      " 'high-speed volume scanning with a liquid lens']\n",
      "Elapsed time: 7.59 seconds\n",
      "-------------------\n",
      "25068736\n",
      "Mapping brain activity at scale with cluster computing.\n",
      "Understanding brain function requires monitoring and interpreting the activity of large networks of\n",
      "neurons during behavior. Advances in recording technology are greatly increasing the size and\n",
      "complexity of neural data. Analyzing such data will pose a fundamental bottleneck for neuroscience.\n",
      "We present a library of analytical tools called Thunder built on the open-source Apache Spark\n",
      "platform for large-scale distributed computing. The library implements a variety of univariate and\n",
      "multivariate analyses with a modular, extendable structure well-suited to interactive exploration\n",
      "and analysis development. We demonstrate how these analyses find structure in large-scale neural\n",
      "data, including whole-brain light-sheet imaging data from fictively behaving larval zebrafish, and\n",
      "two-photon imaging data from behaving mouse. The analyses relate neuronal responses to sensory input\n",
      "and behavior, run in minutes or less and can be used on a private cluster or in the cloud. Our open-\n",
      "source framework thus holds promise for turning brain activity mapping efforts into biological\n",
      "insights.\n",
      "['whole-brain light-sheet imaging', 'two-photon imaging']\n",
      "Elapsed time: 9.33 seconds\n",
      "-------------------\n",
      "25086504\n",
      "Nanoscale high-content analysis using compositional heterogeneities of single proteoliposomes.\n",
      "Proteoliposome reconstitution is a standard method to stabilize purified transmembrane proteins in\n",
      "membranes for structural and functional assays. Here we quantified intrareconstitution\n",
      "heterogeneities in single proteoliposomes using fluorescence microscopy. Our results suggest that\n",
      "compositional heterogeneities can severely skew ensemble-average proteoliposome measurements but\n",
      "also enable ultraminiaturized high-content screens. We took advantage of this screening capability\n",
      "to map the oligomerization energy of the β2-adrenergic receptor using ∼10(9)-fold less protein than\n",
      "conventional assays.\n",
      "{'imagingTechniques': ['fluorescence microscopy', 'none']}\n",
      "Elapsed time: 7.93 seconds\n",
      "-------------------\n",
      "25108686\n",
      "Visualizing a protein quake with time-resolved X-ray scattering at a free-electron laser.\n",
      "We describe a method to measure ultrafast protein structural changes using time-resolved wide-angle\n",
      "X-ray scattering at an X-ray free-electron laser. We demonstrated this approach using multiphoton\n",
      "excitation of the Blastochloris viridis photosynthetic reaction center, observing an ultrafast\n",
      "global conformational change that arises within picoseconds and precedes the propagation of heat\n",
      "through the protein. This provides direct structural evidence for a 'protein quake': the hypothesis\n",
      "that proteins rapidly dissipate energy through quake-like structural motions.\n",
      "{'imagingTechniques': ['time-resolved X-ray scattering',\n",
      "                       'wide-angle X-ray scattering',\n",
      "                       'multiphoton excitation']}\n",
      "Elapsed time: 9.29 seconds\n",
      "-------------------\n",
      "25108687\n",
      "Efficient CRISPR-Cas9-mediated genome editing in Plasmodium falciparum.\n",
      "Malaria is a major cause of global morbidity and mortality, and new strategies for treating and\n",
      "preventing this disease are needed. Here we show that the Streptococcus pyogenes Cas9 DNA\n",
      "endonuclease and single guide RNAs (sgRNAs) produced using T7 RNA polymerase (T7 RNAP) efficiently\n",
      "edit the Plasmodium falciparum genome. Targeting the genes encoding native knob-associated\n",
      "histidine-rich protein (kahrp) and erythrocyte binding antigen 175 (eba-175), we achieved high (≥\n",
      "50-100%) gene disruption frequencies within the usual time frame for generating transgenic\n",
      "parasites.\n",
      "['sgRNAs', 'T7 RNAP', 'CRISPR-Cas9']\n",
      "Elapsed time: 8.68 seconds\n",
      "-------------------\n",
      "25128976\n",
      "Multiplexed aberration measurement for deep tissue imaging in vivo.\n",
      "We describe an adaptive optics method that modulates the intensity or phase of light rays at\n",
      "multiple pupil segments in parallel to determine the sample-induced aberration. Applicable to\n",
      "fluorescent protein-labeled structures of arbitrary complexity, it allowed us to obtain diffraction-\n",
      "limited resolution in various samples in vivo. For the strongly scattering mouse brain, a single\n",
      "aberration correction improved structural and functional imaging of fine neuronal processes over a\n",
      "large imaging volume.\n",
      "['intensity modulation',\n",
      " 'phase modulation',\n",
      " 'adaptive optics',\n",
      " 'diffraction-limited resolution']\n",
      "Elapsed time: 7.01 seconds\n",
      "-------------------\n",
      "25152083\n",
      "Systematic evaluation of quantotypic peptides for targeted analysis of the human kinome.\n",
      "In targeted proteomics it is critical that peptides are not only proteotypic but also accurately\n",
      "represent the level of the protein (quantotypic). Numerous approaches are used to identify\n",
      "proteotypic peptides, but quantotypic properties are rarely assessed. We show that measuring ratios\n",
      "of proteotypic peptides across biological samples can be used to empirically identify peptides with\n",
      "good quantotypic properties. We applied this technique to identify quantotypic peptides for 21% of\n",
      "the human kinome.\n",
      "['proteotypic peptides', 'quantotypic properties', 'empirical identification']\n",
      "Elapsed time: 7.80 seconds\n",
      "-------------------\n",
      "25166869\n",
      "A critique of methods for temperature imaging in single cells.\n",
      "We argue that standard thermodynamic considerations and scaling laws show that a single cell cannot\n",
      "substantially raise its temperature by endogenous thermogenesis. This statement seriously questions\n",
      "the interpretations of recent work reporting temperature heterogeneities measured in single living\n",
      "cells.\n",
      "['thermodynamic considerations',\n",
      " 'scaling laws',\n",
      " 'temperature heterogeneities',\n",
      " 'endogenous thermogenesis']\n",
      "Elapsed time: 6.08 seconds\n",
      "-------------------\n",
      "25194847\n",
      "From genes to protein mechanics on a chip.\n",
      "Single-molecule force spectroscopy enables mechanical testing of individual proteins, but low\n",
      "experimental throughput limits the ability to screen constructs in parallel. We describe a\n",
      "microfluidic platform for on-chip expression, covalent surface attachment and measurement of single-\n",
      "molecule protein mechanical properties. A dockerin tag on each protein molecule allowed us to\n",
      "perform thousands of pulling cycles using a single cohesin-modified cantilever. The ability to\n",
      "synthesize and mechanically probe protein libraries enables high-throughput mechanical phenotyping.\n",
      "['single-molecule force spectroscopy',\n",
      " 'microfluidic platform',\n",
      " 'covalent surface attachment',\n",
      " 'cohesin-modified cantilever']\n",
      "Elapsed time: 7.73 seconds\n",
      "-------------------\n",
      "25194848\n",
      "Evaluation and statistical inference for human connectomes.\n",
      "Diffusion-weighted imaging coupled with tractography is currently the only method for in vivo\n",
      "mapping of human white-matter fascicles. Tractography takes diffusion measurements as input and\n",
      "produces the connectome, a large collection of white-matter fascicles, as output. We introduce a\n",
      "method to evaluate the evidence supporting connectomes. Linear fascicle evaluation (LiFE) takes any\n",
      "connectome as input and predicts diffusion measurements as output, using the difference between the\n",
      "measured and predicted diffusion signals to quantify the prediction error. We use the prediction\n",
      "error to evaluate the evidence that supports the properties of the connectome, to compare\n",
      "tractography algorithms and to test hypotheses about tracts and connections.\n",
      "['diffusion-weighted imaging',\n",
      " 'tractography',\n",
      " 'linear fascicle evaluation (LiFE)']\n",
      "Elapsed time: 7.95 seconds\n",
      "-------------------\n",
      "25218182\n",
      "Nanoparticle vesicle encoding for imaging and tracking cell populations.\n",
      "For phenotypic behavior to be understood in the context of cell lineage and local environment,\n",
      "properties of individual cells must be measured relative to population-wide traits. However, the\n",
      "inability to accurately identify, track and measure thousands of single cells via high-throughput\n",
      "microscopy has impeded dynamic studies of cell populations. We demonstrate unique labeling of cells,\n",
      "driven by the heterogeneous random uptake of fluorescent nanoparticles of different emission colors.\n",
      "By sequentially exposing a cell population to different particles, we generated a large number of\n",
      "unique digital codes, which corresponded to the cell-specific number of nanoparticle-loaded vesicles\n",
      "and were visible within a given fluorescence channel. When three colors are used, the assay can\n",
      "self-generate over 17,000 individual codes identifiable using a typical fluorescence microscope. The\n",
      "color-codes provided immediate visualization of cell identity and allowed us to track human cells\n",
      "with a success rate of 78% across image frames separated by 8 h.\n",
      "['fluorescence microscopy', 'nanoparticle-loaded vesicles', 'color-codes']\n",
      "Elapsed time: 9.55 seconds\n",
      "-------------------\n",
      "25282611\n",
      "Time-resolved crystallography using the Hadamard transform.\n",
      "We describe a method for performing time-resolved X-ray crystallographic experiments based on the\n",
      "Hadamard transform, in which time resolution is defined by the underlying periodicity of the probe\n",
      "pulse sequence, and signal/noise is greatly improved over that for the fastest pump-probe\n",
      "experiments depending on a single pulse. This approach should be applicable on standard synchrotron\n",
      "beamlines and will enable high-resolution measurements of protein and small-molecule structural\n",
      "dynamics. It is also applicable to other time-resolved measurements where a probe can be encoded,\n",
      "such as pump-probe spectroscopy.\n",
      "['time-resolved crystallography',\n",
      " 'Hadamard transform',\n",
      " 'pump-probe experiments',\n",
      " 'spectroscopy']\n",
      "Elapsed time: 8.06 seconds\n",
      "-------------------\n",
      "25306544\n",
      "An improved surface passivation method for single-molecule studies.\n",
      "We report a surface passivation method based on dichlorodimethylsilane (DDS)-Tween-20 for in vitro\n",
      "single-molecule studies, which, under the conditions tested here, more efficiently prevented\n",
      "nonspecific binding of biomolecules than the standard poly(ethylene glycol) surface. The DDS-\n",
      "Tween-20 surface was simple and inexpensive to prepare and did not perturb the behavior and\n",
      "activities of tethered biomolecules. It can also be used for single-molecule imaging in the presence\n",
      "of high concentrations of labeled species in solution.\n",
      "{'imagingTechniques': ['single-molecule studies',\n",
      "                       'in vitro studies',\n",
      "                       'high concentrations of labeled species in solution']}\n",
      "Elapsed time: 9.13 seconds\n",
      "-------------------\n",
      "25326662\n",
      "Quantum dot-based multiphoton fluorescent pipettes for targeted neuronal electrophysiology.\n",
      "Targeting visually identified neurons for electrophysiological recording is a fundamental\n",
      "neuroscience technique; however, its potential is hampered by poor visualization of pipette tips in\n",
      "deep brain tissue. We describe quantum dot-coated glass pipettes that provide strong two-photon\n",
      "contrast at deeper penetration depths than those achievable with current methods. We demonstrated\n",
      "the pipettes' utility in targeted patch-clamp recording experiments and single-cell electroporation\n",
      "of identified rat and mouse neurons in vitro and in vivo.\n",
      "['quantum dot-based multiphoton fluorescent pipettes',\n",
      " 'two-photon contrast',\n",
      " 'patch-clamp recording',\n",
      " 'single-cell electroporation']\n",
      "Elapsed time: 8.08 seconds\n",
      "-------------------\n",
      "25344639\n",
      "Cell-based reporters reveal in vivo dynamics of dopamine and norepinephrine release in murine\n",
      "cortex.\n",
      "The neuronal coding of stimulus-to-action sequences is believed to involve the release of dopamine\n",
      "(DA) and norepinephrine (NE). The electrochemical similarity of these monoamines, however, confounds\n",
      "real-time measurements of their release. Here we report cell-based neurotransmitter fluorescent\n",
      "engineered reporters (CNiFERs) that use the specificity of G protein-coupled receptors (GPCRs) to\n",
      "discriminate nanomolar concentrations of DA and NE. CNiFERs were implanted into the frontal cortex\n",
      "of mice to measure the timing of neurotransmitter release during classical conditioning with the use\n",
      "of two-photon microscopy. The onset of DA release correlated with that of licking and shifted from\n",
      "the time of the reward toward that of the cue upon conditioning. In contrast, concurrent release of\n",
      "NE did not correlate with licking or the cue. This generation of CNiFERs provides unique tools to\n",
      "assess the release of monoamines. The molecular design of these CNiFERs may be generalized to\n",
      "realize CNiFERs for any molecule that activates a GPCR.\n",
      "[{'name': 'two-photon microscopy'}, {'name': 'cell-based reporters'}]\n",
      "Elapsed time: 12.47 seconds\n",
      "-------------------\n",
      "25362362\n",
      "A robust pipeline for rapid production of versatile nanobody repertoires.\n",
      "Nanobodies are single-domain antibodies derived from the variable regions of Camelidae atypical\n",
      "immunoglobulins. They show promise as high-affinity reagents for research, diagnostics and\n",
      "therapeutics owing to their high specificity, small size (∼15 kDa) and straightforward bacterial\n",
      "expression. However, identification of repertoires with sufficiently high affinity has proven time\n",
      "consuming and difficult, hampering nanobody implementation. Our approach generates large repertoires\n",
      "of readily expressible recombinant nanobodies with high affinities and specificities against a given\n",
      "antigen. We demonstrate the efficacy of this approach through the production of large repertoires of\n",
      "nanobodies against two antigens, GFP and mCherry, with Kd values into the subnanomolar range. After\n",
      "mapping diverse epitopes on GFP, we were also able to design ultrahigh-affinity dimeric nanobodies\n",
      "with Kd values as low as ∼30 pM. The approach presented here is well suited for the routine\n",
      "production of high-affinity capture reagents for various biomedical applications.\n",
      "[{'key': 'bacterial expression', 'value': 'none'},\n",
      " {'key': 'high-affinity capture reagents', 'value': 'none'},\n",
      " {'key': 'dimeric nanobodies', 'value': 'none'},\n",
      " {'key': 'ultrahigh-affinity dimeric nanobodies', 'value': 'none'},\n",
      " {'key': 'GFP', 'value': ' none'},\n",
      " {'key': 'mCherry', 'value': ' none'}]\n",
      "Elapsed time: 13.73 seconds\n",
      "-------------------\n",
      "25419959\n",
      "Rational design of a high-affinity, fast, red calcium indicator R-CaMP2.\n",
      "Fluorescent Ca(2+) reporters are widely used as readouts of neuronal activities. Here we designed\n",
      "R-CaMP2, a high-affinity red genetically encoded calcium indicator (GECI) with a Hill coefficient\n",
      "near 1. Use of the calmodulin-binding sequence of CaMKK-α and CaMKK-β in lieu of an M13 sequence\n",
      "resulted in threefold faster rise and decay times of Ca(2+) transients than R-CaMP1.07. These\n",
      "features allowed resolving single action potentials (APs) and recording fast AP trains up to 20-40\n",
      "Hz in cortical slices. Somatic and synaptic activities of a cortical neuronal ensemble in vivo were\n",
      "imaged with similar efficacy as with previously reported sensitive green GECIs. Combining green and\n",
      "red GECIs, we successfully achieved dual-color monitoring of neuronal activities of distinct cell\n",
      "types, both in the mouse cortex and in freely moving Caenorhabditis elegans. Dual imaging using\n",
      "R-CaMP2 and green GECIs provides a powerful means to interrogate orthogonal and hierarchical\n",
      "neuronal ensembles in vivo.\n",
      "['fluorescent Ca(2+) reporters',\n",
      " 'genetically encoded calcium indicator (GECI)',\n",
      " 'calmodulin-binding sequence',\n",
      " 'CaMKK-α',\n",
      " 'CaMKK-β',\n",
      " 'green GECIs',\n",
      " 'red GECIs',\n",
      " 'dual-color monitoring']\n",
      "Elapsed time: 12.87 seconds\n",
      "-------------------\n",
      "25419960\n",
      "Directed evolution of APEX2 for electron microscopy and proximity labeling.\n",
      "APEX is an engineered peroxidase that functions as an electron microscopy tag and a promiscuous\n",
      "labeling enzyme for live-cell proteomics. Because limited sensitivity precludes applications\n",
      "requiring low APEX expression, we used yeast-display evolution to improve its catalytic efficiency.\n",
      "APEX2 is far more active in cells, enabling the use of electron microscopy to resolve the\n",
      "submitochondrial localization of calcium uptake regulatory protein MICU1. APEX2 also permits\n",
      "superior enrichment of endogenous mitochondrial and endoplasmic reticulum membrane proteins.\n",
      "['electron microscopy', 'proximity labeling']\n",
      "Elapsed time: 7.88 seconds\n",
      "-------------------\n",
      "25419961\n",
      "Acoustic force spectroscopy.\n",
      "Force spectroscopy has become an indispensable tool to unravel the structural and mechanochemical\n",
      "properties of biomolecules. Here we extend the force spectroscopy toolbox with an acoustic\n",
      "manipulation device that can exert forces from subpiconewtons to hundreds of piconewtons on\n",
      "thousands of biomolecules in parallel, with submillisecond response time and inherent stability.\n",
      "This method can be readily integrated in lab-on-a-chip devices, allowing for cost-effective and\n",
      "massively parallel applications.\n",
      "['acoustic force spectroscopy', 'force spectroscopy']\n",
      "Elapsed time: 7.20 seconds\n",
      "-------------------\n",
      "25437435\n",
      "cellPACK: a virtual mesoscope to model and visualize structural systems biology.\n",
      "cellPACK assembles computational models of the biological mesoscale, an intermediate scale (10-100\n",
      "nm) between molecular and cellular biology scales. cellPACK's modular architecture unites existing\n",
      "and novel packing algorithms to generate, visualize and analyze comprehensive three-dimensional\n",
      "models of complex biological environments that integrate data from multiple experimental systems\n",
      "biology and structural biology sources. cellPACK is available as open-source code, with tools for\n",
      "validation of models and with 'recipes' and models for five biological systems: blood plasma,\n",
      "cytoplasm, synaptic vesicles, HIV and a mycoplasma cell. We have applied cellPACK to model\n",
      "distributions of HIV envelope protein to test several hypotheses for consistency with experimental\n",
      "observations. Biologists, educators and outreach specialists can interact with cellPACK models,\n",
      "develop new recipes and perform packing experiments through scripting and graphical user interfaces\n",
      "at http://cellPACK.org/.\n",
      "['computational models',\n",
      " 'virtual mesoscope',\n",
      " 'three-dimensional models',\n",
      " 'experimental systems biology',\n",
      " 'structural biology',\n",
      " 'open-source code',\n",
      " 'validation of models',\n",
      " 'recipes',\n",
      " 'models']\n",
      "Elapsed time: 11.29 seconds\n",
      "-------------------\n",
      "25486062\n",
      "DNA nanoswitches: a quantitative platform for gel-based biomolecular interaction analysis.\n",
      "We introduce a nanoscale experimental platform that enables kinetic and equilibrium measurements of\n",
      "a wide range of molecular interactions using a gel electrophoresis readout. Programmable, self-\n",
      "assembled DNA nanoswitches serve both as templates for positioning molecules and as sensitive,\n",
      "quantitative reporters of molecular association and dissociation. We demonstrated this low-cost,\n",
      "versatile, 'lab-on-a-molecule' system by characterizing ten different interactions, including a\n",
      "complex four-body interaction with five discernible states.\n",
      "['gel electrophoresis', 'DNA nanoswitches', 'lab-on-a-molecule']\n",
      "Elapsed time: 7.86 seconds\n",
      "-------------------\n",
      "25506720\n",
      "Conformal nanopatterning of extracellular matrix proteins onto topographically complex surfaces.\n",
      "Our Patterning on Topography (PoT) printing technique enables fibronectin, laminin and other\n",
      "proteins to be applied to biomaterial surfaces in complex geometries that are inaccessible using\n",
      "traditional soft lithography techniques. Engineering combinatorial surfaces that integrate\n",
      "topographical and biochemical micropatterns enhances control of the biotic-abiotic interface. Here,\n",
      "we used this method to understand cardiomyocyte response to competing physical and chemical cues in\n",
      "the microenvironment.\n",
      "['PoT printing', 'traditional soft lithography techniques']\n",
      "Elapsed time: 6.86 seconds\n",
      "-------------------\n",
      "25532138\n",
      "Simultaneous all-optical manipulation and recording of neural circuit activity with cellular\n",
      "resolution in vivo.\n",
      "We describe an all-optical strategy for simultaneously manipulating and recording the activity of\n",
      "multiple neurons with cellular resolution in vivo. We performed simultaneous two-photon optogenetic\n",
      "activation and calcium imaging by coexpression of a red-shifted opsin and a genetically encoded\n",
      "calcium indicator. A spatial light modulator allows tens of user-selected neurons to be targeted for\n",
      "spatiotemporally precise concurrent optogenetic activation, while simultaneous fast calcium imaging\n",
      "provides high-resolution network-wide readout of the manipulation with negligible optical cross-\n",
      "talk. Proof-of-principle experiments in mouse barrel cortex demonstrate interrogation of the same\n",
      "neuronal population during different behavioral states and targeting of neuronal ensembles based on\n",
      "their functional signature. This approach extends the optogenetic toolkit beyond the specificity\n",
      "obtained with genetic or viral approaches, enabling high-throughput, flexible and long-term optical\n",
      "interrogation of functionally defined neural circuits with single-cell and single-spike resolution\n",
      "in the mouse brain in vivo.\n",
      "[{'name': 'two-photon optogenetic activation'},\n",
      " {'name': 'calcium imaging'},\n",
      " {'name': 'spatial light modulator'}]\n",
      "Elapsed time: 11.80 seconds\n",
      "-------------------\n",
      "25581799\n",
      "Fixation-resistant photoactivatable fluorescent proteins for CLEM.\n",
      "Fluorescent proteins facilitate a variety of imaging paradigms in live and fixed samples. However,\n",
      "they lose their fluorescence after heavy fixation, hindering applications such as correlative light\n",
      "and electron microscopy (CLEM). Here we report engineered variants of the photoconvertible Eos\n",
      "fluorescent protein that fluoresce and photoconvert normally in heavily fixed (0.5-1% OsO4), plastic\n",
      "resin-embedded samples, enabling correlative super-resolution fluorescence imaging and high-quality\n",
      "electron microscopy.\n",
      "['fluorescent proteins',\n",
      " 'correlative light and electron microscopy (CLEM)',\n",
      " 'super-resolution fluorescence imaging',\n",
      " 'electron microscopy']\n",
      "Elapsed time: 8.18 seconds\n",
      "-------------------\n",
      "25581800\n",
      "Improved and expanded Q-system reagents for genetic manipulations.\n",
      "The Q system is a repressible binary expression system for transgenic manipulations in living\n",
      "organisms. Through protein engineering and in vivo functional tests, we report here variants of the\n",
      "Q-system transcriptional activator, including QF2, for driving strong and ubiquitous expression in\n",
      "all Drosophila tissues. Our QF2, Gal4QF and LexAQF chimeric transcriptional activators substantially\n",
      "enrich the toolkit available for transgenic regulation in Drosophila melanogaster.\n",
      "['Q-system', 'protein engineering', 'in vivo functional tests']\n",
      "Elapsed time: 6.93 seconds\n",
      "-------------------\n",
      "25599549\n",
      "Sequence tag-based analysis of microbial population dynamics.\n",
      "We describe sequence tag-based analysis of microbial populations (STAMP) for characterization of\n",
      "pathogen population dynamics during infection. STAMP analyzes the frequency changes of genetically\n",
      "'barcoded' organisms to quantify population bottlenecks and infer the founding population size.\n",
      "Analyses of intraintestinal Vibrio cholerae revealed infection-stage and region-specific host\n",
      "barriers to infection and showed unexpected V. cholerae migration counter to intestinal flow. STAMP\n",
      "provides a robust, widely applicable analytical framework for high-confidence characterization of in\n",
      "vivo microbial dissemination.\n",
      "['STAMP', 'genetically barcoded organisms', 'frequency changes']\n",
      "Elapsed time: 7.90 seconds\n",
      "-------------------\n",
      "25599551\n",
      "A general method to improve fluorophores for live-cell and single-molecule microscopy.\n",
      "Specific labeling of biomolecules with bright fluorophores is the keystone of fluorescence\n",
      "microscopy. Genetically encoded self-labeling tag proteins can be coupled to synthetic dyes inside\n",
      "living cells, resulting in brighter reporters than fluorescent proteins. Intracellular labeling\n",
      "using these techniques requires cell-permeable fluorescent ligands, however, limiting utility to a\n",
      "small number of classic fluorophores. Here we describe a simple structural modification that\n",
      "improves the brightness and photostability of dyes while preserving spectral properties and cell\n",
      "permeability. Inspired by molecular modeling, we replaced the N,N-dimethylamino substituents in\n",
      "tetramethylrhodamine with four-membered azetidine rings. This addition of two carbon atoms doubles\n",
      "the quantum efficiency and improves the photon yield of the dye in applications ranging from in\n",
      "vitro single-molecule measurements to super-resolution imaging. The novel substitution is\n",
      "generalizable, yielding a palette of chemical dyes with improved quantum efficiencies that spans the\n",
      "UV and visible range.\n",
      "['single-molecule microscopy',\n",
      " 'live-cell microscopy',\n",
      " 'super-resolution imaging',\n",
      " 'in vitro single-molecule measurements']\n",
      "Elapsed time: 11.88 seconds\n",
      "-------------------\n",
      "25622108\n",
      "Ratiometric biosensors based on dimerization-dependent fluorescent protein exchange.\n",
      "We have developed a versatile new class of genetically encoded fluorescent biosensor based on\n",
      "reversible exchange of the heterodimeric partners of green and red dimerization-dependent\n",
      "fluorescent proteins. We demonstrate the use of this strategy to construct both intermolecular and\n",
      "intramolecular ratiometric biosensors for qualitative imaging of caspase activity, Ca(2+)\n",
      "concentration dynamics and other second-messenger signaling activities.\n",
      "['fluorescent protein exchange',\n",
      " 'ratiometric biosensors',\n",
      " 'intermolecular ratiometric biosensors',\n",
      " 'intramolecular ratiometric biosensors',\n",
      " 'qualitative imaging',\n",
      " 'caspase activity',\n",
      " 'Ca(2+) concentration dynamics',\n",
      " 'second-messenger signaling activities']\n",
      "Elapsed time: 8.68 seconds\n",
      "-------------------\n",
      "25643149\n",
      "Reporters for sensitive and quantitative measurement of auxin response.\n",
      "The visualization of hormonal signaling input and output is key to understanding how multicellular\n",
      "development is regulated. The plant signaling molecule auxin triggers many growth and developmental\n",
      "responses, but current tools lack the sensitivity or precision to visualize these. We developed a\n",
      "set of fluorescent reporters that allow sensitive and semiquantitative readout of auxin responses at\n",
      "cellular resolution in Arabidopsis thaliana. These generic tools are suitable for any transformable\n",
      "plant species.\n",
      "['fluorescent reporters',\n",
      " 'sensitive readout',\n",
      " 'semiquantitative readout',\n",
      " 'cellular resolution',\n",
      " 'generic tools']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "25686390\n",
      "Ultrastructurally smooth thick partitioning and volume stitching for large-scale connectomics.\n",
      "Focused-ion-beam scanning electron microscopy (FIB-SEM) has become an essential tool for studying\n",
      "neural tissue at resolutions below 10 nm × 10 nm × 10 nm, producing data sets optimized for\n",
      "automatic connectome tracing. We present a technical advance, ultrathick sectioning, which reliably\n",
      "subdivides embedded tissue samples into chunks (20 μm thick) optimally sized and mounted for\n",
      "efficient, parallel FIB-SEM imaging. These chunks are imaged separately and then 'volume stitched'\n",
      "back together, producing a final three-dimensional data set suitable for connectome tracing.\n",
      "['focused-ion-beam scanning electron microscopy (FIB-SEM)', 'volume stitching']\n",
      "Elapsed time: 8.69 seconds\n",
      "-------------------\n",
      "25707028\n",
      "Image fusion of mass spectrometry and microscopy: a multimodality paradigm for molecular tissue\n",
      "mapping.\n",
      "We describe a predictive imaging modality created by 'fusing' two distinct technologies: imaging\n",
      "mass spectrometry (IMS) and microscopy. IMS-generated molecular maps, rich in chemical information\n",
      "but having coarse spatial resolution, are combined with optical microscopy maps, which have\n",
      "relatively low chemical specificity but high spatial information. The resulting images combine the\n",
      "advantages of both technologies, enabling prediction of a molecular distribution both at high\n",
      "spatial resolution and with high chemical specificity. Multivariate regression is used to model\n",
      "variables in one technology, using variables from the other technology. We demonstrate the potential\n",
      "of image fusion through several applications: (i) 'sharpening' of IMS images, which uses microscopy\n",
      "measurements to predict ion distributions at a spatial resolution that exceeds that of measured ion\n",
      "images by ten times or more; (ii) prediction of ion distributions in tissue areas that were not\n",
      "measured by IMS; and (iii) enrichment of biological signals and attenuation of instrumental\n",
      "artifacts, revealing insights not easily extracted from either microscopy or IMS individually.\n",
      "['imaging mass spectrometry', 'microscopy', 'image fusion']\n",
      "Elapsed time: 10.89 seconds\n",
      "-------------------\n",
      "25707029\n",
      "De novo protein structure determination from near-atomic-resolution cryo-EM maps.\n",
      "We present a de novo model-building approach that combines predicted backbone conformations with\n",
      "side-chain fit to density to accurately assign sequence into density maps. This method yielded\n",
      "accurate models for six of nine experimental maps at 3.3- to 4.8-Å resolution and produced a nearly\n",
      "complete model for an unsolved map containing a 660-residue heterodimeric protein. This method\n",
      "should enable rapid and reliable protein structure determination from near-atomic-resolution cryo-\n",
      "electron microscopy (cryo-EM) maps.\n",
      "['cryo-EM',\n",
      " 'predicted backbone conformations',\n",
      " 'side-chain fit to density',\n",
      " 'de novo model-building approach']\n",
      "Elapsed time: 7.87 seconds\n",
      "-------------------\n",
      "25707030\n",
      "Atomic-accuracy models from 4.5-Å cryo-electron microscopy data with density-guided iterative local\n",
      "refinement.\n",
      "We describe a general approach for refining protein structure models on the basis of cryo-electron\n",
      "microscopy maps with near-atomic resolution. The method integrates Monte Carlo sampling with local\n",
      "density-guided optimization, Rosetta all-atom refinement and real-space B-factor fitting. In tests\n",
      "on experimental maps of three different systems with 4.5-Å resolution or better, the method\n",
      "consistently produced models with atomic-level accuracy largely independently of starting-model\n",
      "quality, and it outperformed the molecular dynamics-based MDFF method. Cross-validated model quality\n",
      "statistics correlated with model accuracy over the three test systems.\n",
      "['Monte Carlo sampling',\n",
      " 'local density-guided optimization',\n",
      " 'Rosetta all-atom refinement',\n",
      " 'real-space B-factor fitting']\n",
      "Elapsed time: 8.16 seconds\n",
      "-------------------\n",
      "25730490\n",
      "Highly efficient Cas9-mediated transcriptional programming.\n",
      "The RNA-guided nuclease Cas9 can be reengineered as a programmable transcription factor. However,\n",
      "modest levels of gene activation have limited potential applications. We describe an improved\n",
      "transcriptional regulator obtained through the rational design of a tripartite activator,\n",
      "VP64-p65-Rta (VPR), fused to nuclease-null Cas9. We demonstrate its utility in activating endogenous\n",
      "coding and noncoding genes, targeting several genes simultaneously and stimulating neuronal\n",
      "differentiation of human induced pluripotent stem cells (iPSCs).\n",
      "['Cas9',\n",
      " 'RNA-guided nuclease',\n",
      " 'transcriptional regulator',\n",
      " 'VP64-p65-Rta (VPR)']\n",
      "Elapsed time: 7.95 seconds\n",
      "-------------------\n",
      "25730493\n",
      "High-throughput RNA profiling via up-front sample parallelization.\n",
      "We describe a method called modular, early-tagged amplification (META) RNA profiling that can\n",
      "quantify a broad panel of microRNAs or mRNAs simultaneously across many samples and requires far\n",
      "less sequence depth than existing digital profiling technologies. The method assigns quantitative\n",
      "tags during reverse transcription to permit up-front sample pooling before competitive amplification\n",
      "and deep sequencing. This simple, scalable and inexpensive approach improves the practicality of\n",
      "large-scale gene expression studies.\n",
      "['META RNA profiling',\n",
      " 'reverse transcription',\n",
      " 'competitive amplification',\n",
      " 'deep sequencing']\n",
      "Elapsed time: 7.39 seconds\n",
      "-------------------\n",
      "25751144\n",
      "Whole-body immunoPET reveals active SIV dynamics in viremic and antiretroviral therapy-treated\n",
      "macaques.\n",
      "The detection of viral dynamics and localization in the context of controlled HIV infection remains\n",
      "a challenge and is limited to blood and biopsies. We developed a method to capture total-body simian\n",
      "immunodeficiency virus (SIV) replication using immunoPET (antibody-targeted positron emission\n",
      "tomography). The administration of a poly(ethylene glycol)-modified, (64)Cu-labeled SIV\n",
      "Gp120-specific antibody led to readily detectable signals in the gastrointestinal and respiratory\n",
      "tract, lymphoid tissues and reproductive organs of viremic monkeys. Viral signals were reduced in\n",
      "aviremic antiretroviral-treated monkeys but detectable in colon, select lymph nodes, small bowel,\n",
      "nasal turbinates, the genital tract and lung. In elite controllers, virus was detected primarily in\n",
      "foci in the small bowel, select lymphoid areas and the male reproductive tract, as confirmed by\n",
      "quantitative reverse-transcription PCR (qRT-PCR) and immunohistochemistry. This real-time, in vivo\n",
      "viral imaging method has broad applications to the study of immunodeficiency virus pathogenesis,\n",
      "drug and vaccine development, and the potential for clinical translation.\n",
      "{'imagingTechniques': ['immunoPET',\n",
      "                       'positron emission tomography',\n",
      "                       'antibody-targeted positron emission tomography',\n",
      "                       'quantitative reverse-transcription PCR',\n",
      "                       'immunohistochemistry']}\n",
      "Elapsed time: 13.96 seconds\n",
      "-------------------\n",
      "25775042\n",
      "Direct visualization of newly synthesized target proteins in situ.\n",
      "Protein synthesis is a dynamic process that tunes the cellular proteome in response to internal and\n",
      "external demands. Metabolic labeling approaches identify the general proteomic response but cannot\n",
      "visualize specific newly synthesized proteins within cells. Here we describe a technique that\n",
      "couples noncanonical amino acid tagging or puromycylation with the proximity ligation assay to\n",
      "visualize specific newly synthesized proteins and monitor their origin, redistribution and turnover\n",
      "in situ.\n",
      "['noncanonical amino acid tagging',\n",
      " 'puromycylation',\n",
      " 'proximity ligation assay']\n",
      "Elapsed time: 7.11 seconds\n",
      "-------------------\n",
      "25775044\n",
      "CIDRE: an illumination-correction method for optical microscopy.\n",
      "Uneven illumination affects every image acquired by a microscope. It is often overlooked, but it can\n",
      "introduce considerable bias to image measurements. The most reliable correction methods require\n",
      "special reference images, and retrospective alternatives do not fully model the correction process.\n",
      "Our approach overcomes these issues for most optical microscopy applications without the need for\n",
      "reference images.\n",
      "['microscope', 'optical microscopy', 'illumination correction']\n",
      "Elapsed time: 6.33 seconds\n",
      "-------------------\n",
      "25775045\n",
      "Catchup: a mouse model for imaging-based tracking and modulation of neutrophil granulocytes.\n",
      "Neutrophil granulocyte biology is a central issue of immunological research, but the lack of animal\n",
      "models that allow for neutrophil-selective genetic manipulation has delayed progress. By modulating\n",
      "the neutrophil-specific locus Ly6G with a knock-in allele expressing Cre recombinase and the\n",
      "fluorescent protein tdTomato, we generated a mouse model termed Catchup that exhibits strong\n",
      "neutrophil specificity. Transgene activity was found only in very few eosinophils and basophils and\n",
      "was undetectable in bone marrow precursors, including granulomonocytic progenitors (GMPs). Cre-\n",
      "mediated reporter-gene activation allowed for intravital two-photon microscopy of neutrophils\n",
      "without adoptive transfer. Homozygous animals were Ly6G deficient but showed normal leukocyte\n",
      "cellularity in all measured organs. Ly6G-deficient neutrophils were functionally normal in vitro and\n",
      "in multiple models of sterile or infectious inflammation in vivo. However, Cre-mediated deletion of\n",
      "FcγRIV in neutrophils reduced the cells' recruitment to immune-complex-mediated peritonitis,\n",
      "suggesting a cell-intrinsic role for activating Fc receptors in neutrophil trafficking.\n",
      "{'imagingTechniques': ['two-photon microscopy',\n",
      "                       'intravital microscopy',\n",
      "                       'adoptive transfer']}\n",
      "Elapsed time: 14.27 seconds\n",
      "-------------------\n",
      "25822799\n",
      "High-speed label-free functional photoacoustic microscopy of mouse brain in action.\n",
      "We present fast functional photoacoustic microscopy (PAM) for three-dimensional high-resolution,\n",
      "high-speed imaging of the mouse brain, complementary to other imaging modalities. We implemented a\n",
      "single-wavelength pulse-width-based method with a one-dimensional imaging rate of 100 kHz to image\n",
      "blood oxygenation with capillary-level resolution. We applied PAM to image the vascular morphology,\n",
      "blood oxygenation, blood flow and oxygen metabolism in both resting and stimulated states in the\n",
      "mouse brain.\n",
      "['PAM',\n",
      " 'functional photoacoustic microscopy',\n",
      " 'high-resolution',\n",
      " 'high-speed imaging',\n",
      " 'vascular morphology',\n",
      " 'blood oxygenation',\n",
      " 'blood flow',\n",
      " 'oxygen metabolism']\n",
      "Elapsed time: 8.85 seconds\n",
      "-------------------\n",
      "25825836\n",
      "Integrative, dynamic structural biology at atomic resolution--it's about time.\n",
      "Biomolecules adopt a dynamic ensemble of conformations, each with the potential to interact with\n",
      "binding partners or perform the chemical reactions required for a multitude of cellular functions.\n",
      "Recent advances in X-ray crystallography, nuclear magnetic resonance (NMR) spectroscopy and other\n",
      "techniques are helping us realize the dream of seeing--in atomic detail--how different parts of\n",
      "biomolecules shift between functional substates using concerted motions. Integrative structural\n",
      "biology has advanced our understanding of the formation of large macromolecular complexes and how\n",
      "their components interact in assemblies by leveraging data from many low-resolution methods. Here,\n",
      "we review the growing opportunities for integrative, dynamic structural biology at the atomic scale,\n",
      "contending there is increasing synergistic potential between X-ray crystallography, NMR and computer\n",
      "simulations to reveal a structural basis for protein conformational dynamics at high resolution.\n",
      "['X-ray crystallography', 'NMR spectroscopy', 'computer simulations']\n",
      "Elapsed time: 9.64 seconds\n",
      "-------------------\n",
      "25849636\n",
      "Massively parallel delivery of large cargo into mammalian cells with light pulses.\n",
      "We report a high-throughput platform for delivering large cargo elements into 100,000 cells in 1\n",
      "min. Our biophotonic laser-assisted surgery tool (BLAST) generates an array of microcavitation\n",
      "bubbles that explode in response to laser pulsing, forming pores in adjacent cell membranes through\n",
      "which cargo is gently driven by pressurized flow. The platform delivers large items including\n",
      "bacteria, enzymes, antibodies and nanoparticles into diverse cell types with high efficiency and\n",
      "cell viability. We used this platform to explore the intracellular lifestyle of Francisella novicida\n",
      "and discovered that the iglC gene is unexpectedly required for intracellular replication even after\n",
      "phagosome escape into the cell cytosol.\n",
      "['biophotonic laser-assisted surgery tool (BLAST)',\n",
      " 'microcavitation bubbles',\n",
      " 'laser pulsing']\n",
      "Elapsed time: 8.87 seconds\n",
      "-------------------\n",
      "25867848\n",
      "Rapid reverse genetic screening using CRISPR in zebrafish.\n",
      "Identifying genes involved in biological processes is critical for understanding the molecular\n",
      "building blocks of life. We used engineered CRISPR (clustered regularly interspaced short\n",
      "palindromic repeats) to efficiently mutate specific loci in zebrafish (Danio rerio) and screen for\n",
      "genes involved in vertebrate biological processes. We found that increasing CRISPR efficiency by\n",
      "injecting optimized amounts of Cas9-encoding mRNA and multiplexing single guide RNAs (sgRNAs)\n",
      "allowed for phenocopy of known mutants across many phenotypes in embryos. We performed a proof-of-\n",
      "concept screen in which we used intersecting, multiplexed pool injections to examine 48 loci and\n",
      "identified two new genes involved in electrical-synapse formation. By deep sequencing target loci,\n",
      "we found that 90% of the genes were effectively screened. We conclude that CRISPR can be used as a\n",
      "powerful reverse genetic screening strategy in vivo in a vertebrate system.\n",
      "['CRISPR', 'zebrafish', 'embryos', 'phenocopy', 'electrical-synapse formation']\n",
      "Elapsed time: 10.13 seconds\n",
      "-------------------\n",
      "25867849\n",
      "High-resolution whole-brain staining for electron microscopic circuit reconstruction.\n",
      "Currently only electron microscopy provides the resolution necessary to reconstruct neuronal\n",
      "circuits completely and with single-synapse resolution. Because almost all behaviors rely on neural\n",
      "computations widely distributed throughout the brain, a reconstruction of brain-wide circuits-and,\n",
      "ultimately, the entire brain-is highly desirable. However, these reconstructions require the\n",
      "undivided brain to be prepared for electron microscopic observation. Here we describe a preparation,\n",
      "BROPA (brain-wide reduced-osmium staining with pyrogallol-mediated amplification), that results in\n",
      "the preservation and staining of ultrastructural details throughout the brain at a resolution\n",
      "necessary for tracing neuronal processes and identifying synaptic contacts between them. Using\n",
      "serial block-face electron microscopy (SBEM), we tested human annotator ability to follow neural\n",
      "'wires' reliably and over long distances as well as the ability to detect synaptic contacts. Our\n",
      "results suggest that the BROPA method can produce a preparation suitable for the reconstruction of\n",
      "neural circuits spanning an entire mouse brain.\n",
      "['electron microscopy', 'serial block-face electron microscopy (SBEM)']\n",
      "Elapsed time: 9.96 seconds\n",
      "-------------------\n",
      "25867850\n",
      "In vivo cell-cycle profiling in xenograft tumors by quantitative intravital microscopy.\n",
      "Quantification of cell-cycle state at a single-cell level is essential to understand fundamental\n",
      "three-dimensional (3D) biological processes such as tissue development and cancer. Analysis of 3D in\n",
      "vivo images, however, is very challenging. Today's best practice, manual annotation of select image\n",
      "events, generates arbitrarily sampled data distributions, which are unsuitable for reliable\n",
      "mechanistic inferences. Here, we present an integrated workflow for quantitative in vivo cell-cycle\n",
      "profiling. It combines image analysis and machine learning methods for automated 3D segmentation and\n",
      "cell-cycle state identification of individual cell-nuclei with widely varying morphologies embedded\n",
      "in complex tumor environments. We applied our workflow to quantify cell-cycle effects of three\n",
      "antimitotic cancer drugs over 8 d in HT-1080 fibrosarcoma xenografts in living mice using a data set\n",
      "of 38,000 cells and compared the induced phenotypes. In contrast to results with 2D culture,\n",
      "observed mitotic arrest was relatively low, suggesting involvement of additional mechanisms in their\n",
      "antitumor effect in vivo.\n",
      "['quantitative intravital microscopy',\n",
      " 'machine learning methods',\n",
      " '3D segmentation',\n",
      " 'cell-cycle state identification']\n",
      "Elapsed time: 11.43 seconds\n",
      "-------------------\n",
      "25894945\n",
      "Isotope-targeted glycoproteomics (IsoTaG): a mass-independent platform for intact N- and\n",
      "O-glycopeptide discovery and analysis.\n",
      "Protein glycosylation is a heterogeneous post-translational modification (PTM) that plays an\n",
      "essential role in biological regulation. However, the diversity found in glycoproteins has\n",
      "undermined efforts to describe the intact glycoproteome via mass spectrometry (MS). We present\n",
      "IsoTaG, a mass-independent chemical glycoproteomics platform for characterization of intact,\n",
      "metabolically labeled glycopeptides at the whole-proteome scale. In IsoTaG, metabolic labeling of\n",
      "the glycoproteome is combined with (i) chemical enrichment and isotopic recoding of glycopeptides to\n",
      "select peptides for targeted glycoproteomics using directed MS and (ii) mass-independent assignment\n",
      "of intact glycopeptides. We structurally assigned 32 N-glycopeptides and over 500 intact and fully\n",
      "elaborated O-glycopeptides from 250 proteins across three human cancer cell lines and also\n",
      "discovered unexpected peptide sequence polymorphisms (pSPs). The IsoTaG platform is broadly\n",
      "applicable to the discovery of PTM sites that are amenable to chemical labeling, as well as\n",
      "previously unknown protein isoforms including pSPs.\n",
      "['mass spectrometry',\n",
      " 'chemical enrichment',\n",
      " 'isotopic recoding',\n",
      " 'directed MS',\n",
      " 'structural assignment']\n",
      "Elapsed time: 12.96 seconds\n",
      "-------------------\n",
      "25894946\n",
      "A fast- and positively photoswitchable fluorescent protein for ultralow-laser-power RESOLFT\n",
      "nanoscopy.\n",
      "Fluorescence nanoscopy has revolutionized our ability to visualize biological structures not\n",
      "resolvable by conventional microscopy. However, photodamage induced by intense light exposure has\n",
      "limited its use in live specimens. Here we describe Kohinoor, a fast-switching, positively\n",
      "photoswitchable fluorescent protein, and show that it has high photostability over many switching\n",
      "repeats. With Kohinoor, we achieved super-resolution imaging of live HeLa cells using biocompatible,\n",
      "ultralow laser intensity (0.004 J/cm(2)) in reversible saturable optical fluorescence transition\n",
      "(RESOLFT) nanoscopy.\n",
      "['fluorescence nanoscopy', 'RESOLFT nanoscopy', 'super-resolution imaging']\n",
      "Elapsed time: 8.77 seconds\n",
      "-------------------\n",
      "25915120\n",
      "High-performance probes for light and electron microscopy.\n",
      "We describe an engineered family of highly antigenic molecules based on GFP-like fluorescent\n",
      "proteins. These molecules contain numerous copies of peptide epitopes and simultaneously bind IgG\n",
      "antibodies at each location. These 'spaghetti monster' fluorescent proteins (smFPs) distributed well\n",
      "in neurons, notably into small dendrites, spines and axons. smFP immunolabeling localized weakly\n",
      "expressed proteins not well resolved with traditional epitope tags. By varying epitope and scaffold,\n",
      "we generated a diverse family of mutually orthogonal antigens. In cultured neurons and mouse and fly\n",
      "brains, smFP probes allowed robust, orthogonal multicolor visualization of proteins, cell\n",
      "populations and neuropil. smFP variants complement existing tracers and greatly increase the number\n",
      "of simultaneous imaging channels, and they performed well in advanced preparations such as array\n",
      "tomography, super-resolution fluorescence imaging and electron microscopy. In living cells, the\n",
      "probes improved single-molecule image tracking and increased yield for RNA-seq. These probes\n",
      "facilitate new experiments in connectomics, transcriptomics and protein localization.\n",
      "['fluorescence imaging',\n",
      " 'electron microscopy',\n",
      " 'array tomography',\n",
      " 'super-resolution fluorescence imaging']\n",
      "Elapsed time: 11.91 seconds\n",
      "-------------------\n",
      "25938370\n",
      "Rapid, optimized interactomic screening.\n",
      "We must reliably map the interactomes of cellular macromolecular complexes in order to fully explore\n",
      "and understand biological systems. However, there are no methods to accurately predict how to\n",
      "capture a given macromolecular complex with its physiological binding partners. Here, we present a\n",
      "screening method that comprehensively explores the parameters affecting the stability of\n",
      "interactions in affinity-captured complexes, enabling the discovery of physiological binding\n",
      "partners in unparalleled detail. We have implemented this screen on several macromolecular complexes\n",
      "from a variety of organisms, revealing novel profiles for even well-studied proteins. Our approach\n",
      "is robust, economical and automatable, providing inroads to the rigorous, systematic dissection of\n",
      "cellular interactomes.\n",
      "['screening method', 'affinity-captured complexes', 'novel profiles']\n",
      "Elapsed time: 8.39 seconds\n",
      "-------------------\n",
      "25961413\n",
      "Structural analysis of multicellular organisms with cryo-electron tomography.\n",
      "We developed a method for visualizing tissues from multicellular organisms using cryo-electron\n",
      "tomography. Our protocol involves vitrifying samples with high-pressure freezing, thinning them with\n",
      "cryo-FIB-SEM (focused-ion-beam scanning electron microscopy) and applying fiducial gold markers\n",
      "under cryogenic conditions to the lamellae post-milling. We applied this protocol to acquire\n",
      "tomograms of vitrified Caenorhabditis elegans embryos and worms, which showed the intracellular\n",
      "organization of selected tissues at particular developmental stages in otherwise intact specimens.\n",
      "['cryo-electron tomography',\n",
      " 'high-pressure freezing',\n",
      " 'cryo-FIB-SEM (focused-ion-beam scanning electron microscopy)',\n",
      " 'lamellae post-milling']\n",
      "Elapsed time: 8.67 seconds\n",
      "-------------------\n",
      "25961414\n",
      "3D high- and super-resolution imaging using single-objective SPIM.\n",
      "Single-objective selective-plane illumination microscopy (soSPIM) is achieved with micromirrored\n",
      "cavities combined with a laser beam-steering unit installed on a standard inverted microscope. The\n",
      "illumination and detection are done through the same objective. soSPIM can be used with standard\n",
      "sample preparations and features high background rejection and efficient photon collection, allowing\n",
      "for 3D single-molecule-based super-resolution imaging of whole cells or cell aggregates. Using\n",
      "larger mirrors enabled us to broaden the capabilities of our system to image Drosophila embryos.\n",
      "['SPIM',\n",
      " 'soSPIM',\n",
      " 'single-molecule-based super-resolution imaging',\n",
      " 'whole cells or cell aggregates',\n",
      " 'Drosophila embryos']\n",
      "Elapsed time: 8.51 seconds\n",
      "-------------------\n",
      "25984698\n",
      "Probing a cell-embedded megadalton protein complex by DNP-supported solid-state NMR.\n",
      "Studying biomolecules at atomic resolution in their native environment is the ultimate aim of\n",
      "structural biology. We investigated the bacterial type IV secretion system core complex (T4SScc) by\n",
      "cellular dynamic nuclear polarization-based solid-state nuclear magnetic resonance spectroscopy to\n",
      "validate a structural model previously generated by combining in vitro and in silico data. Our\n",
      "results indicate that T4SScc is well folded in the cellular setting, revealing protein regions that\n",
      "had been elusive when studied in vitro.\n",
      "['solid-state NMR', 'cellular dynamic nuclear polarization']\n",
      "Elapsed time: 7.08 seconds\n",
      "-------------------\n",
      "26005811\n",
      "Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture.\n",
      "The human cerebral cortex develops through an elaborate succession of cellular events that, when\n",
      "disrupted, can lead to neuropsychiatric disease. The ability to reprogram somatic cells into\n",
      "pluripotent cells that can be differentiated in vitro provides a unique opportunity to study normal\n",
      "and abnormal corticogenesis. Here, we present a simple and reproducible 3D culture approach for\n",
      "generating a laminated cerebral cortex-like structure, named human cortical spheroids (hCSs), from\n",
      "pluripotent stem cells. hCSs contain neurons from both deep and superficial cortical layers and map\n",
      "transcriptionally to in vivo fetal development. These neurons are electrophysiologically mature,\n",
      "display spontaneous activity, are surrounded by nonreactive astrocytes and form functional synapses.\n",
      "Experiments in acute hCS slices demonstrate that cortical neurons participate in network activity\n",
      "and produce complex synaptic events. These 3D cultures should allow a detailed interrogation of\n",
      "human cortical development, function and disease, and may prove a versatile platform for generating\n",
      "other neuronal and glial subtypes in vitro.\n",
      "[{'name': 'electrophysiology'},\n",
      " {'name': 'spontaneous activity'},\n",
      " {'name': 'functional synapses'},\n",
      " {'name': 'acute slices'}]\n",
      "Elapsed time: 12.89 seconds\n",
      "-------------------\n",
      "26005812\n",
      "Dexterous robotic manipulation of alert adult Drosophila for high-content experimentation.\n",
      "We present a robot that enables high-content studies of alert adult Drosophila by combining\n",
      "operations including gentle picking; translations and rotations; characterizations of fly phenotypes\n",
      "and behaviors; microdissection; or release. To illustrate, we assessed fly morphology, tracked odor-\n",
      "evoked locomotion, sorted flies by sex, and dissected the cuticle to image neural activity. The\n",
      "robot's tireless capacity for precise manipulations enables a scalable platform for screening flies'\n",
      "complex attributes and behavioral patterns.\n",
      "['gentle picking',\n",
      " 'translations and rotations',\n",
      " 'microdissection',\n",
      " 'image neural activity']\n",
      "Elapsed time: 7.71 seconds\n",
      "-------------------\n",
      "26020503\n",
      "Correlated light and electron microscopy: ultrastructure lights up!\n",
      "Microscopy has gone hand in hand with the study of living systems since van Leeuwenhoek observed\n",
      "living microorganisms and cells in 1674 using his light microscope. A spectrum of dyes and probes\n",
      "now enable the localization of molecules of interest within living cells by fluorescence microscopy.\n",
      "With electron microscopy (EM), cellular ultrastructure has been revealed. Bridging these two\n",
      "modalities, correlated light microscopy and EM (CLEM) opens new avenues. Studies of protein dynamics\n",
      "with fluorescent proteins (FPs), which leave the investigator 'in the dark' concerning cellular\n",
      "context, can be followed by EM examination. Rare events can be preselected at the light microscopy\n",
      "level before EM analysis. Ongoing development-including of dedicated probes, integrated microscopes,\n",
      "large-scale and three-dimensional EM and super-resolution fluorescence microscopy-now paves the way\n",
      "for broad CLEM implementation in biology.\n",
      "['fluorescence microscopy',\n",
      " 'electron microscopy',\n",
      " 'correlated light and electron microscopy',\n",
      " 'super-resolution fluorescence microscopy']\n",
      "Elapsed time: 10.63 seconds\n",
      "-------------------\n",
      "26030444\n",
      "Multiplexable, locus-specific targeting of long RNAs with CRISPR-Display.\n",
      "Noncoding RNAs play diverse roles throughout biology and exhibit broad functional capacity. To\n",
      "investigate and harness these capabilities, we developed clustered regularly interspaced short\n",
      "palindromic repeats (CRISPR)-Display (CRISP-Disp), a targeted localization method that uses Cas9 to\n",
      "deploy large RNA cargos to DNA loci. We demonstrate that functional RNA domains up to at least 4.8\n",
      "kb long can be inserted in CRISPR guide RNA at multiple points, allowing the construction of Cas9\n",
      "complexes with protein-binding cassettes, artificial aptamers, pools of random sequences and natural\n",
      "long noncoding RNAs. A unique feature of CRISP-Disp is the multiplexing of distinct functions at\n",
      "multiple targets, limited only by the availability of functional RNA motifs. We anticipate the use\n",
      "of CRISP-Disp for ectopically targeting functional RNAs and ribonucleoprotein (RNP) complexes to\n",
      "genomic loci.\n",
      "Invalid json output: [\n",
      "\"CRISPR-Display\",\n",
      "\"Cas9\",\n",
      "\"long RNAs\"\n",
      "\"functional RNA domains\"\n",
      "\"artificial aptamers\"\n",
      "\"pools of random sequences\"\n",
      "\"natural long noncoding RNAs\"\n",
      "] derived from Sure! Here is the list of imaging techniques or technologies described in the abstract:\n",
      "\n",
      "[\n",
      "\"CRISPR-Display\",\n",
      "\"Cas9\",\n",
      "\"long RNAs\"\n",
      "\"functional RNA domains\"\n",
      "\"artificial aptamers\"\n",
      "\"pools of random sequences\"\n",
      "\"natural long noncoding RNAs\"\n",
      "]\n",
      "26030446\n",
      "Traction microscopy to identify force modulation in subresolution adhesions.\n",
      "We present a reconstruction algorithm that resolves cellular tractions in diffraction-limited\n",
      "nascent adhesions (NAs). The enabling method is the introduction of sparsity regularization to the\n",
      "solution of the inverse problem, which suppresses noise without underestimating traction magnitude.\n",
      "We show that NAs transmit a distinguishable amount of traction and that NA maturation depends on\n",
      "traction growth rate. A software package implementing this numerical approach is provided.\n",
      "['traction microscopy',\n",
      " 'diffraction-limited nascent adhesions',\n",
      " 'sparsity regularization',\n",
      " 'inverse problem']\n",
      "Elapsed time: 7.27 seconds\n",
      "-------------------\n",
      "26030447\n",
      "Plasmodium falciparum genetic crosses in a humanized mouse model.\n",
      "Genetic crosses of phenotypically distinct strains of the human malaria parasite Plasmodium\n",
      "falciparum are a powerful tool for identifying genes controlling drug resistance and other key\n",
      "phenotypes. Previous studies relied on the isolation of recombinant parasites from splenectomized\n",
      "chimpanzees, a research avenue that is no longer available. Here we demonstrate that human-liver\n",
      "chimeric mice support recovery of recombinant progeny for the identification of genetic determinants\n",
      "of parasite traits and adaptations.\n",
      "['genetic crosses', 'recombinant parasites', 'splenectomized chimpanzees']\n",
      "Elapsed time: 7.79 seconds\n",
      "-------------------\n",
      "26030448\n",
      "Application of BRET to monitor ligand binding to GPCRs.\n",
      "Bioluminescence resonance energy transfer (BRET) is a well-established method for investigating\n",
      "protein-protein interactions. Here we present a BRET approach to monitor ligand binding to G\n",
      "protein-coupled receptors (GPCRs) on the surface of living cells made possible by the use of\n",
      "fluorescent ligands in combination with a bioluminescent protein (NanoLuc) that can be readily\n",
      "expressed on the N terminus of GPCRs.\n",
      "['BRET',\n",
      " 'bioluminescence resonance energy transfer',\n",
      " 'fluorescent ligands',\n",
      " 'NanoLuc']\n",
      "Elapsed time: 7.49 seconds\n",
      "-------------------\n",
      "26076424\n",
      "Quantitative evaluation of software packages for single-molecule localization microscopy.\n",
      "The quality of super-resolution images obtained by single-molecule localization microscopy (SMLM)\n",
      "depends largely on the software used to detect and accurately localize point sources. In this work,\n",
      "we focus on the computational aspects of super-resolution microscopy and present a comprehensive\n",
      "evaluation of localization software packages. Our philosophy is to evaluate each package as a whole,\n",
      "thus maintaining the integrity of the software. We prepared synthetic data that represent three-\n",
      "dimensional structures modeled after biological components, taking excitation parameters, noise\n",
      "sources, point-spread functions and pixelation into account. We then asked developers to run their\n",
      "software on our data; most responded favorably, allowing us to present a broad picture of the\n",
      "methods available. We evaluated their results using quantitative and user-interpretable criteria:\n",
      "detection rate, accuracy, quality of image reconstruction, resolution, software usability and\n",
      "computational resources. These metrics reflect the various tradeoffs of SMLM software packages and\n",
      "help users to choose the software that fits their needs.\n",
      "['detection rate',\n",
      " 'accuracy',\n",
      " 'quality of image reconstruction',\n",
      " 'resolution',\n",
      " 'software usability',\n",
      " 'computational resources']\n",
      "Elapsed time: 10.53 seconds\n",
      "-------------------\n",
      "26098019\n",
      "Single-cell magnetic imaging using a quantum diamond microscope.\n",
      "We apply a quantum diamond microscope for detection and imaging of immunomagnetically labeled cells.\n",
      "This instrument uses nitrogen-vacancy (NV) centers in diamond for correlated magnetic and\n",
      "fluorescence imaging. Our device provides single-cell resolution and a field of view (∼1 mm(2)) two\n",
      "orders of magnitude larger than that of previous NV imaging technologies, enabling practical\n",
      "applications. To illustrate, we quantified cancer biomarkers expressed by rare tumor cells in a\n",
      "large population of healthy cells.\n",
      "['magnetic imaging',\n",
      " 'fluorescence imaging',\n",
      " 'correlated magnetic and fluorescence imaging',\n",
      " 'NV centers imaging',\n",
      " 'quantum diamond microscope']\n",
      "Elapsed time: 7.98 seconds\n",
      "-------------------\n",
      "26098020\n",
      "A naturally monomeric infrared fluorescent protein for protein labeling in vivo.\n",
      "Infrared fluorescent proteins (IFPs) provide an additional color to GFP and its homologs in protein\n",
      "labeling. Drawing on structural analysis of the dimer interface, we identified a bacteriophytochrome\n",
      "in the sequence database that is monomeric in truncated form and engineered it into a naturally\n",
      "monomeric IFP (mIFP). We demonstrate that mIFP correctly labels proteins in live cells, Drosophila\n",
      "and zebrafish. It should be useful in molecular, cell and developmental biology.\n",
      "['infrared fluorescent proteins',\n",
      " 'mIFP',\n",
      " 'live cells',\n",
      " 'Drosophila',\n",
      " 'zebrafish']\n",
      "Elapsed time: 7.67 seconds\n",
      "-------------------\n",
      "26098021\n",
      "Combining protein and mRNA quantification to decipher transcriptional regulation.\n",
      "We combine immunofluorescence and single-molecule fluorescence in situ hybridization (smFISH),\n",
      "followed by automated image analysis, to quantify the concentration of nuclear transcription\n",
      "factors, number of transcription factors bound, and number of nascent mRNAs synthesized at\n",
      "individual gene loci. A theoretical model is used to decipher how transcription factor binding\n",
      "modulates the stochastic kinetics of mRNA production. We demonstrate this approach by examining the\n",
      "regulation of hunchback in the early Drosophila embryo.\n",
      "['immunofluorescence',\n",
      " 'single-molecule fluorescence in situ hybridization (smFISH)',\n",
      " 'automated image analysis']\n",
      "Elapsed time: 8.45 seconds\n",
      "-------------------\n",
      "26121404\n",
      "Assembly and diploid architecture of an individual human genome via single-molecule technologies.\n",
      "We present the first comprehensive analysis of a diploid human genome that combines single-molecule\n",
      "sequencing with single-molecule genome maps. Our hybrid assembly markedly improves upon the\n",
      "contiguity observed from traditional shotgun sequencing approaches, with scaffold N50 values\n",
      "approaching 30 Mb, and we identified complex structural variants (SVs) missed by other high-\n",
      "throughput approaches. Furthermore, by combining Illumina short-read data with long reads, we phased\n",
      "both single-nucleotide variants and SVs, generating haplotypes with over 99% consistency with\n",
      "previous trio-based studies. Our work shows that it is now possible to integrate single-molecule and\n",
      "high-throughput sequence data to generate de novo assembled genomes that approach reference quality.\n",
      "{'imagingTechniques': ['single-molecule sequencing',\n",
      "                       'single-molecule genome maps',\n",
      "                       'Illumina short-read data',\n",
      "                       'long reads']}\n",
      "Elapsed time: 9.89 seconds\n",
      "-------------------\n",
      "26121406\n",
      "Protein structure determination by combining sparse NMR data with evolutionary couplings.\n",
      "Accurate determination of protein structure by NMR spectroscopy is challenging for larger proteins,\n",
      "for which experimental data are often incomplete and ambiguous. Evolutionary sequence information\n",
      "together with advances in maximum entropy statistical methods provide a rich complementary source of\n",
      "structural constraints. We have developed a hybrid approach (evolutionary coupling-NMR spectroscopy;\n",
      "EC-NMR) combining sparse NMR data with evolutionary residue-residue couplings and demonstrate\n",
      "accurate structure determination for several proteins 6-41 kDa in size.\n",
      "['NMR spectroscopy',\n",
      " 'evolutionary sequence information',\n",
      " 'maximum entropy statistical methods']\n",
      "Elapsed time: 7.31 seconds\n",
      "-------------------\n",
      "26147917\n",
      "Multitarget super-resolution microscopy with high-density labeling by exchangeable probes.\n",
      "We have developed a multitarget super-resolution microscopy technique called image reconstruction by\n",
      "integrating exchangeable single-molecule localization (IRIS). IRIS uses protein fragment-based\n",
      "probes that directly associate with and dissociate from their targets over durations on the order of\n",
      "tens of milliseconds. By integrating single-molecule localization and sequential labeling, IRIS\n",
      "enables unprecedented labeling density along multiple cellular structures. IRIS can be used to\n",
      "discern the area-specific proximity between cytoskeletal components and focal adhesions within a\n",
      "single cell.\n",
      "['single-molecule localization',\n",
      " 'sequential labeling',\n",
      " 'multitarget super-resolution microscopy']\n",
      "Elapsed time: 8.13 seconds\n",
      "-------------------\n",
      "26147918\n",
      "Single-molecule spectroscopy of protein conformational dynamics in live eukaryotic cells.\n",
      "Single-molecule methods have become widely used for quantifying the conformational heterogeneity and\n",
      "structural dynamics of biomolecules in vitro. Their application in vivo, however, has remained\n",
      "challenging owing to shortcomings in the design and reproducible delivery of labeled molecules, the\n",
      "range of applicable analysis methods, and suboptimal cell culture conditions. By addressing these\n",
      "limitations in an integrated approach, we demonstrate the feasibility of probing protein dynamics\n",
      "from milliseconds down to the nanosecond regime in live eukaryotic cells with confocal single-\n",
      "molecule Förster resonance energy transfer (FRET) spectroscopy. We illustrate the versatility of the\n",
      "approach by determining the dimensions and submicrosecond chain dynamics of an intrinsically\n",
      "disordered protein; by detecting even subtle changes in the temperature dependence of protein\n",
      "stability, including in-cell cold denaturation; and by quantifying the folding dynamics of a small\n",
      "protein. The methodology opens possibilities for assessing the effect of the cellular environment on\n",
      "biomolecular conformation, dynamics and function.\n",
      "{'methods': ['single-molecule spectroscopy',\n",
      "             'confocal single-molecule Förster resonance energy transfer '\n",
      "             '(FRET) spectroscopy',\n",
      "             'temperature dependence of protein stability',\n",
      "             'folding dynamics']}\n",
      "Elapsed time: 11.61 seconds\n",
      "-------------------\n",
      "26167640\n",
      "Photoactivatable genetically encoded calcium indicators for targeted neuronal imaging.\n",
      "Circuit mapping requires knowledge of both structural and functional connectivity between cells.\n",
      "Although optical tools have been made to assess either the morphology and projections of neurons or\n",
      "their activity and functional connections, few probes integrate this information. We have generated\n",
      "a family of photoactivatable genetically encoded Ca(2+) indicators that combines attributes of high-\n",
      "contrast photolabeling with high-sensitivity Ca(2+) detection in a single-color protein sensor. We\n",
      "demonstrated in cultured neurons and in fruit fly and zebrafish larvae how single cells could be\n",
      "selected out of dense populations for visualization of morphology and high signal-to-noise\n",
      "measurements of activity, synaptic transmission and connectivity. Our design strategy is\n",
      "transferrable to other sensors based on circularly permutated GFP (cpGFP).\n",
      "['photoactivatable genetically encoded calcium indicators',\n",
      " 'high-contrast photolabeling',\n",
      " 'high-sensitivity Ca(2+) detection',\n",
      " 'single-color protein sensor']\n",
      "Elapsed time: 9.22 seconds\n",
      "-------------------\n",
      "26167641\n",
      "Continuous volumetric imaging via an optical phase-locked ultrasound lens.\n",
      "In vivo imaging at high spatiotemporal resolution is key to the understanding of complex biological\n",
      "systems. We integrated an optical phase-locked ultrasound lens into a two-photon fluorescence\n",
      "microscope and achieved microsecond-scale axial scanning, thus enabling volumetric imaging at tens\n",
      "of hertz. We applied this system to multicolor volumetric imaging of processes sensitive to motion\n",
      "artifacts, including calcium dynamics in behaving mouse brain and transient morphology changes and\n",
      "trafficking of immune cells.\n",
      "['two-photon fluorescence microscopy',\n",
      " 'optical phase-locked ultrasound lens',\n",
      " 'microsecond-scale axial scanning',\n",
      " 'volumetric imaging',\n",
      " 'multicolor volumetric imaging']\n",
      "Elapsed time: 8.58 seconds\n",
      "-------------------\n",
      "26167642\n",
      "Imaging G protein-coupled receptors while quantifying their ligand-binding free-energy landscape.\n",
      "Imaging native membrane receptors and testing how they interact with ligands is of fundamental\n",
      "interest in the life sciences but has proven remarkably difficult to accomplish. Here, we introduce\n",
      "an approach that uses force-distance curve-based atomic force microscopy to simultaneously image\n",
      "single native G protein-coupled receptors in membranes and quantify their dynamic binding strength\n",
      "to native and synthetic ligands. We measured kinetic and thermodynamic parameters for individual\n",
      "protease-activated receptor-1 (PAR1) molecules in the absence and presence of antagonists, and these\n",
      "measurements enabled us to describe PAR1's ligand-binding free-energy landscape with high accuracy.\n",
      "Our nanoscopic method opens an avenue to directly image and characterize ligand binding of native\n",
      "membrane receptors.\n",
      "['atomic force microscopy',\n",
      " 'force-distance curve-based atomic force microscopy',\n",
      " 'nanoscopic method']\n",
      "Elapsed time: 9.09 seconds\n",
      "-------------------\n",
      "26192083\n",
      "Inferring transient particle transport dynamics in live cells.\n",
      "Live-cell imaging and particle tracking provide rich information on mechanisms of intracellular\n",
      "transport. However, trajectory analysis procedures to infer complex transport dynamics involving\n",
      "stochastic switching between active transport and diffusive motion are lacking. We applied Bayesian\n",
      "model selection to hidden Markov modeling to infer transient transport states from trajectories of\n",
      "mRNA-protein complexes in live mouse hippocampal neurons and metaphase kinetochores in dividing\n",
      "human cells. The software is available at http://hmm-bayes.org/.\n",
      "['live-cell imaging',\n",
      " 'particle tracking',\n",
      " 'Bayesian model selection',\n",
      " 'hidden Markov modeling']\n",
      "Elapsed time: 7.50 seconds\n",
      "-------------------\n",
      "26192084\n",
      "Real-time imaging of brain activity in freely moving rats using functional ultrasound.\n",
      "Innovative imaging methods help to investigate the complex relationship between brain activity and\n",
      "behavior in freely moving animals. Functional ultrasound (fUS) is an imaging modality suitable for\n",
      "recording cerebral blood volume (CBV) dynamics in the whole brain but has so far been used only in\n",
      "head-fixed and anesthetized rodents. We designed a fUS device for tethered brain imaging in freely\n",
      "moving rats based on a miniaturized ultrasound probe and a custom-made ultrasound scanner. We\n",
      "monitored CBV changes in rats during various behavioral states such as quiet rest, after whisker or\n",
      "visual stimulations, and in a food-reinforced operant task. We show that fUS imaging in freely\n",
      "moving rats could efficiently decode brain activity in real time.\n",
      "['functional ultrasound',\n",
      " 'cerebral blood volume dynamics',\n",
      " 'miniaturized ultrasound probe',\n",
      " 'custom-made ultrasound scanner']\n",
      "Elapsed time: 9.55 seconds\n",
      "-------------------\n",
      "26237227\n",
      "ProteoPlex: stability optimization of macromolecular complexes by sparse-matrix screening of\n",
      "chemical space.\n",
      "Molecular machines or macromolecular complexes are supramolecular assemblies of biomolecules with a\n",
      "variety of functions. Structure determination of these complexes in a purified state is often\n",
      "tedious owing to their compositional complexity and the associated relative structural instability.\n",
      "To improve the stability of macromolecular complexes in vitro, we present a generic method that\n",
      "optimizes the stability, homogeneity and solubility of macromolecular complexes by sparse-matrix\n",
      "screening of their thermal unfolding behavior in the presence of various buffers and small\n",
      "molecules. The method includes the automated analysis of thermal unfolding curves based on a\n",
      "biophysical unfolding model for complexes. We found that under stabilizing conditions, even large\n",
      "multicomponent complexes reveal an almost ideal two-state unfolding behavior. We envisage an\n",
      "improved biochemical understanding of purified macromolecules as well as a substantial boost in\n",
      "successful macromolecular complex structure determination by both X-ray crystallography and cryo-\n",
      "electron microscopy.\n",
      "['X-ray crystallography', 'cryo-electron microscopy']\n",
      "Elapsed time: 10.80 seconds\n",
      "-------------------\n",
      "26237228\n",
      "EEG and functional ultrasound imaging in mobile rats.\n",
      "We developed an integrated experimental framework that extends the brain exploration capabilities of\n",
      "functional ultrasound imaging to awake and mobile rats. In addition to acquiring hemodynamic data,\n",
      "this method further allows parallel access to electroencephalography (EEG) recordings of neuronal\n",
      "activity. We illustrate this approach with two proofs of concept: a behavioral study on theta rhythm\n",
      "activation in a maze running task and a disease-related study on spontaneous epileptic seizures.\n",
      "['EEG', 'functional ultrasound imaging']\n",
      "Elapsed time: 6.68 seconds\n",
      "-------------------\n",
      "26280328\n",
      "EMRinger: side chain-directed model and map validation for 3D cryo-electron microscopy.\n",
      "Advances in high-resolution cryo-electron microscopy (cryo-EM) require the development of validation\n",
      "metrics to independently assess map quality and model geometry. We report EMRinger, a tool that\n",
      "assesses the precise fitting of an atomic model into the map during refinement and shows how\n",
      "radiation damage alters scattering from negatively charged amino acids. EMRinger\n",
      "(https://github.com/fraser-lab/EMRinger) will be useful for monitoring progress in resolving and\n",
      "modeling high-resolution features in cryo-EM.\n",
      "['cryo-electron microscopy',\n",
      " 'atomic model fitting',\n",
      " 'map validation',\n",
      " 'radiation damage assessment']\n",
      "Elapsed time: 7.62 seconds\n",
      "-------------------\n",
      "26280329\n",
      "Ultrahigh-throughput single-molecule spectroscopy and spectrally resolved super-resolution\n",
      "microscopy.\n",
      "By developing a wide-field scheme for spectral measurement and implementing photoswitching, we\n",
      "synchronously obtained the fluorescence spectra and positions of ∼10(6) single molecules in labeled\n",
      "cells in minutes, which consequently enabled spectrally resolved, 'true-color' super-resolution\n",
      "microscopy. The method, called spectrally resolved stochastic optical reconstruction microscopy (SR-\n",
      "STORM), achieved cross-talk-free three-dimensional (3D) imaging for four dyes 10 nm apart in\n",
      "emission spectrum. Excellent resolution was obtained for every channel, and 3D localizations of all\n",
      "molecules were automatically aligned within one imaging path.\n",
      "['spectral measurement',\n",
      " 'photoswitching',\n",
      " 'stochastic optical reconstruction microscopy (SR-STORM)',\n",
      " 'cross-talk-free three-dimensional (3D) imaging']\n",
      "Elapsed time: 9.48 seconds\n",
      "-------------------\n",
      "26280330\n",
      "Wirelessly powered, fully internal optogenetics for brain, spinal and peripheral circuits in mice.\n",
      "To enable sophisticated optogenetic manipulation of neural circuits throughout the nervous system\n",
      "with limited disruption of animal behavior, light-delivery systems beyond fiber optic tethering and\n",
      "large, head-mounted wireless receivers are desirable. We report the development of an easy-to-\n",
      "construct, implantable wireless optogenetic device. Our smallest version (20 mg, 10 mm(3)) is two\n",
      "orders of magnitude smaller than previously reported wireless optogenetic systems, allowing the\n",
      "entire device to be implanted subcutaneously. With a radio-frequency (RF) power source and\n",
      "controller, this implant produces sufficient light power for optogenetic stimulation with minimal\n",
      "tissue heating (<1 °C). We show how three adaptations of the implant allow for untethered\n",
      "optogenetic control throughout the nervous system (brain, spinal cord and peripheral nerve endings)\n",
      "of behaving mice. This technology opens the door for optogenetic experiments in which animals are\n",
      "able to behave naturally with optogenetic manipulation of both central and peripheral targets.\n",
      "['implantable wireless optogenetic device',\n",
      " 'radio-frequency (RF) power source and controller']\n",
      "Elapsed time: 10.79 seconds\n",
      "-------------------\n",
      "26301841\n",
      "Oscope identifies oscillatory genes in unsynchronized single-cell RNA-seq experiments.\n",
      "Oscillatory gene expression is fundamental to development, but technologies for monitoring\n",
      "expression oscillations are limited. We have developed a statistical approach called Oscope to\n",
      "identify and characterize the transcriptional dynamics of oscillating genes in single-cell RNA-seq\n",
      "data from an unsynchronized cell population. Applying Oscope to a number of data sets, we\n",
      "demonstrated its utility and also identified a potential artifact in the Fluidigm C1 platform.\n",
      "['Oscope', 'single-cell RNA-seq', 'Fluidigm C1 platform']\n",
      "Elapsed time: 6.93 seconds\n",
      "-------------------\n",
      "26301842\n",
      "Trajectories of cell-cycle progression from fixed cell populations.\n",
      "An accurate dissection of sources of cell-to-cell variability is crucial for quantitative biology at\n",
      "the single-cell level but has been challenging for the cell cycle. We present Cycler, a robust\n",
      "method that constructs a continuous trajectory of cell-cycle progression from images of fixed cells.\n",
      "Cycler handles heterogeneous microenvironments and does not require perturbations or genetic\n",
      "markers, making it generally applicable to quantifying multiple sources of cell-to-cell variability\n",
      "in mammalian cells.\n",
      "['Cycler', 'images of fixed cells']\n",
      "Elapsed time: 6.67 seconds\n",
      "-------------------\n",
      "26322836\n",
      "Programmed synthesis of three-dimensional tissues.\n",
      "Reconstituting tissues from their cellular building blocks facilitates the modeling of\n",
      "morphogenesis, homeostasis and disease in vitro. Here we describe DNA-programmed assembly of cells\n",
      "(DPAC), a method to reconstitute the multicellular organization of organoid-like tissues having\n",
      "programmed size, shape, composition and spatial heterogeneity. DPAC uses dissociated cells that are\n",
      "chemically functionalized with degradable oligonucleotide 'Velcro', allowing rapid, specific and\n",
      "reversible cell adhesion to other surfaces coated with complementary DNA sequences. DNA-patterned\n",
      "substrates function as removable and adhesive templates, and layer-by-layer DNA-programmed assembly\n",
      "builds arrays of tissues into the third dimension above the template. DNase releases completed\n",
      "arrays of organoid-like microtissues from the template concomitant with full embedding in a variety\n",
      "of extracellular matrix (ECM) gels. DPAC positions subpopulations of cells with single-cell spatial\n",
      "resolution and generates cultures several centimeters long. We used DPAC to explore the impact of\n",
      "ECM composition, heterotypic cell-cell interactions and patterns of signaling heterogeneity on\n",
      "collective cell behaviors.\n",
      "[['DNA-programmed assembly of cells (DPAC)'],\n",
      " ['single-cell spatial resolution'],\n",
      " ['ECM composition']]\n",
      "Elapsed time: 11.05 seconds\n",
      "-------------------\n",
      "26322839\n",
      "CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo.\n",
      "CRISPR-Cas9 technology provides a powerful system for genome engineering. However, variable activity\n",
      "across different single guide RNAs (sgRNAs) remains a significant limitation. We analyzed the\n",
      "molecular features that influence sgRNA stability, activity and loading into Cas9 in vivo. We\n",
      "observed that guanine enrichment and adenine depletion increased sgRNA stability and activity,\n",
      "whereas differential sgRNA loading, nucleosome positioning and Cas9 off-target binding were not\n",
      "major determinants. We also identified sgRNAs truncated by one or two nucleotides and containing 5'\n",
      "mismatches as efficient alternatives to canonical sgRNAs. On the basis of these results, we created\n",
      "a predictive sgRNA-scoring algorithm, CRISPRscan, that effectively captures the sequence features\n",
      "affecting the activity of CRISPR-Cas9 in vivo. Finally, we show that targeting Cas9 to the germ line\n",
      "using a Cas9-nanos 3' UTR led to the generation of maternal-zygotic mutants, as well as increased\n",
      "viability and decreased somatic mutations. These results identify determinants that influence Cas9\n",
      "activity and provide a framework for the design of highly efficient sgRNAs for genome targeting in\n",
      "vivo.\n",
      "Invalid json output: [ \"CRISPR-Cas9 technology\" ]\n",
      "[ \"single guide RNAs (sgRNAs)\" ]\n",
      "[ \"genome engineering\" ] derived from Sure! Here is the list of imaging techniques or technologies described in the abstract:\n",
      "\n",
      "[ \"CRISPR-Cas9 technology\" ]\n",
      "[ \"single guide RNAs (sgRNAs)\" ]\n",
      "[ \"genome engineering\" ]\n",
      "\n",
      "I hope this helps! Let me know if you have any further questions.\n",
      "26344046\n",
      "SR-Tesseler: a method to segment and quantify localization-based super-resolution microscopy data.\n",
      "Localization-based super-resolution techniques open the door to unprecedented analysis of molecular\n",
      "organization. This task often involves complex image processing adapted to the specific topology and\n",
      "quality of the image to be analyzed. Here we present a segmentation framework based on Voronoï\n",
      "tessellation constructed from the coordinates of localized molecules, implemented in freely\n",
      "available and open-source SR-Tesseler software. This method allows precise, robust and automatic\n",
      "quantification of protein organization at different scales, from the cellular level down to clusters\n",
      "of a few fluorescent markers. We validated our method on simulated data and on various biological\n",
      "experimental data of proteins labeled with genetically encoded fluorescent proteins or organic\n",
      "fluorophores. In addition to providing insight into complex protein organization, this polygon-based\n",
      "method should serve as a reference for the development of new types of quantifications, as well as\n",
      "for the optimization of existing ones.\n",
      "['SR-Tesseler',\n",
      " 'localization-based super-resolution microscopy',\n",
      " 'polygon-based method']\n",
      "Elapsed time: 9.87 seconds\n",
      "-------------------\n",
      "26436479\n",
      "Bayesian cluster identification in single-molecule localization microscopy data.\n",
      "Single-molecule localization-based super-resolution microscopy techniques such as photoactivated\n",
      "localization microscopy (PALM) and stochastic optical reconstruction microscopy (STORM) produce\n",
      "pointillist data sets of molecular coordinates. Although many algorithms exist for the\n",
      "identification and localization of molecules from raw image data, methods for analyzing the\n",
      "resulting point patterns for properties such as clustering have remained relatively under-studied.\n",
      "Here we present a model-based Bayesian approach to evaluate molecular cluster assignment proposals,\n",
      "generated in this study by analysis based on Ripley's K function. The method takes full account of\n",
      "the individual localization precisions calculated for each emitter. We validate the approach using\n",
      "simulated data, as well as experimental data on the clustering behavior of CD3ζ, a subunit of the\n",
      "CD3 T cell receptor complex, in resting and activated primary human T cells.\n",
      "['PALM', 'STORM', 'single-molecule localization microscopy']\n",
      "Elapsed time: 9.64 seconds\n",
      "-------------------\n",
      "26436480\n",
      "A strategy for dissecting the architectures of native macromolecular assemblies.\n",
      "It remains particularly problematic to define the structures of native macromolecular assemblies,\n",
      "which are often of low abundance. Here we present a strategy for isolating complexes at endogenous\n",
      "levels from GFP-tagged transgenic cell lines. Using cross-linking mass spectrometry, we extracted\n",
      "distance restraints that allowed us to model the complexes' molecular architectures.\n",
      "['cross-linking mass spectrometry',\n",
      " 'GFP-tagged transgenic cell lines',\n",
      " 'isolating complexes at endogenous levels']\n",
      "Elapsed time: 6.76 seconds\n",
      "-------------------\n",
      "26436482\n",
      "Noncontact three-dimensional mapping of intracellular hydromechanical properties by Brillouin\n",
      "microscopy.\n",
      "Current measurements of the biomechanical properties of cells require physical contact with cells or\n",
      "lack subcellular resolution. Here we developed a label-free microscopy technique based on Brillouin\n",
      "light scattering that is capable of measuring an intracellular longitudinal modulus with optical\n",
      "resolution. The 3D Brillouin maps we obtained of cells in 2D and 3D microenvironments revealed\n",
      "mechanical changes due to cytoskeletal modulation and cell-volume regulation.\n",
      "['Brillouin microscopy', 'optical resolution']\n",
      "Elapsed time: 6.97 seconds\n",
      "-------------------\n",
      "26457862\n",
      "Transparent intracortical microprobe array for simultaneous spatiotemporal optical stimulation and\n",
      "multichannel electrical recording.\n",
      "Optogenetics, the selective excitation or inhibition of neural circuits by light, has become a\n",
      "transformative approach for dissecting functional brain microcircuits, particularly in in vivo\n",
      "rodent models, owing to the expanding libraries of opsins and promoters. Yet there is a lack of\n",
      "versatile devices that can deliver spatiotemporally patterned light while performing simultaneous\n",
      "sensing to map the dynamics of perturbed neural populations at the network level. We have created\n",
      "optoelectronic actuator and sensor microarrays that can be used as monolithic intracortical\n",
      "implants, fabricated from an optically transparent, electrically highly conducting semiconductor ZnO\n",
      "crystal. The devices can perform simultaneous light delivery and electrical readout in precise\n",
      "spatial registry across the microprobe array. We applied the device technology in transgenic mice to\n",
      "study light-perturbed cortical microcircuit dynamics and their effects on behavior. The\n",
      "functionality of this device can be further expanded to optical imaging and patterned electrical\n",
      "microstimulation.\n",
      "['optical stimulation',\n",
      " 'electrical recording',\n",
      " 'optogenetics',\n",
      " 'spatiotemporal patterned light']\n",
      "Elapsed time: 10.69 seconds\n",
      "-------------------\n",
      "26480475\n",
      "Tissue matrix arrays for high-throughput screening and systems analysis of cell function.\n",
      "Cell and protein arrays have demonstrated remarkable utility in the high-throughput evaluation of\n",
      "biological responses; however, they lack the complexity of native tissue and organs. Here we spotted\n",
      "tissue extracellular matrix (ECM) particles as two-dimensional (2D) arrays or incorporated them with\n",
      "cells to generate three-dimensional (3D) cell-matrix microtissue arrays. We then investigated the\n",
      "responses of human stem, cancer and immune cells to tissue ECM arrays originating from 11 different\n",
      "tissues. We validated the 2D and 3D arrays as representative of the in vivo microenvironment by\n",
      "means of quantitative analysis of tissue-specific cellular responses, including matrix production,\n",
      "adhesion and proliferation, and morphological changes after culture. The biological outputs\n",
      "correlated with tissue proteomics, and network analysis identified several proteins linked to cell\n",
      "function. Our methodology enables broad screening of ECMs to connect tissue-specific composition\n",
      "with biological activity, providing a new resource for biomaterials research and further\n",
      "understanding of regeneration and disease mechanisms.\n",
      "['quantitative analysis',\n",
      " 'matrix production',\n",
      " 'adhesion',\n",
      " 'proliferation',\n",
      " 'morphological changes']\n",
      "Elapsed time: 11.07 seconds\n",
      "-------------------\n",
      "26501515\n",
      "Whole-animal functional and developmental imaging with isotropic spatial resolution.\n",
      "Imaging fast cellular dynamics across large specimens requires high resolution in all dimensions,\n",
      "high imaging speeds, good physical coverage and low photo-damage. To meet these requirements, we\n",
      "developed isotropic multiview (IsoView) light-sheet microscopy, which rapidly images large specimens\n",
      "via simultaneous light-sheet illumination and fluorescence detection along four orthogonal\n",
      "directions. Combining these four views by means of high-throughput multiview deconvolution yields\n",
      "images with high resolution in all three dimensions. We demonstrate whole-animal functional imaging\n",
      "of Drosophila larvae at a spatial resolution of 1.1-2.5 μm and temporal resolution of 2 Hz for\n",
      "several hours. We also present spatially isotropic whole-brain functional imaging in Danio rerio\n",
      "larvae and spatially isotropic multicolor imaging of fast cellular dynamics across gastrulating\n",
      "Drosophila embryos. Compared with conventional light-sheet microscopy, IsoView microscopy improves\n",
      "spatial resolution at least sevenfold and decreases resolution anisotropy at least threefold.\n",
      "Compared with existing high-resolution light-sheet techniques, IsoView microscopy effectively\n",
      "doubles the penetration depth and provides subsecond temporal resolution for specimens 400-fold\n",
      "larger than could previously be imaged.\n",
      "['light-sheet microscopy',\n",
      " 'multiview deconvolution',\n",
      " 'isotropic multiview (IsoView) light-sheet microscopy']\n",
      "Elapsed time: 13.13 seconds\n",
      "-------------------\n",
      "26513553\n",
      "Focused ion beams in biology.\n",
      "A quiet revolution is under way in technologies used for nanoscale cellular imaging. Focused ion\n",
      "beams, previously restricted to the materials sciences and semiconductor fields, are rapidly\n",
      "becoming powerful tools for ultrastructural imaging of biological samples. Cell and tissue\n",
      "architecture, as preserved in plastic-embedded resin or in plunge-frozen form, can be investigated\n",
      "in three dimensions by scanning electron microscopy imaging of freshly created surfaces that result\n",
      "from the progressive removal of material using a focused ion beam. The focused ion beam can also be\n",
      "used as a sculpting tool to create specific specimen shapes such as lamellae or needles that can be\n",
      "analyzed further by transmission electron microscopy or by methods that probe chemical composition.\n",
      "Here we provide an in-depth primer to the application of focused ion beams in biology, including a\n",
      "guide to the practical aspects of using the technology, as well as selected examples of its\n",
      "contribution to the generation of new insights into subcellular architecture and mechanisms\n",
      "underlying host-pathogen interactions.\n",
      "['scanning electron microscopy',\n",
      " 'transmission electron microscopy',\n",
      " 'focused ion beam']\n",
      "Elapsed time: 10.16 seconds\n",
      "-------------------\n",
      "26524239\n",
      "Fixed single-cell transcriptomic characterization of human radial glial diversity.\n",
      "The diverse progenitors that give rise to the human neocortex have been difficult to characterize\n",
      "because progenitors, particularly radial glia (RG), are rare and are defined by a combination of\n",
      "intracellular markers, position and morphology. To circumvent these problems, we developed Fixed and\n",
      "Recovered Intact Single-cell RNA (FRISCR), a method for profiling the transcriptomes of individual\n",
      "fixed, stained and sorted cells. Using FRISCR, we profiled primary human RG that constitute only 1%\n",
      "of the midgestation cortex and classified them as ventricular zone-enriched RG (vRG) that express\n",
      "ANXA1 and CRYAB, and outer subventricular zone-localized RG (oRG) that express HOPX. Our study\n",
      "identified vRG and oRG markers and molecular profiles, an essential step for understanding human\n",
      "neocortical progenitor development. FRISCR allows targeted single-cell profiling of any tissues that\n",
      "lack live-cell markers.\n",
      "['FRISCR', 'fixed single-cell transcriptomic characterization']\n",
      "Elapsed time: 11.40 seconds\n",
      "-------------------\n",
      "26524242\n",
      "Tissue cartography: compressing bio-image data by dimensional reduction.\n",
      "The high volumes of data produced by state-of-the-art optical microscopes encumber research. We\n",
      "developed a method that reduces data size and processing time by orders of magnitude while\n",
      "disentangling signal by taking advantage of the laminar structure of many biological specimens. Our\n",
      "Image Surface Analysis Environment automatically constructs an atlas of 2D images for arbitrarily\n",
      "shaped, dynamic and possibly multilayered surfaces of interest. Built-in correction for cartographic\n",
      "distortion ensures that no information on the surface is lost, making the method suitable for\n",
      "quantitative analysis. We applied our approach to 4D imaging of a range of samples, including a\n",
      "Drosophila melanogaster embryo and a Danio rerio beating heart.\n",
      "['atlas of 2D images', 'Image Surface Analysis Environment', '4D imaging']\n",
      "Elapsed time: 8.65 seconds\n",
      "-------------------\n",
      "26550774\n",
      "Multiscale photoacoustic tomography using reversibly switchable bacterial phytochrome as a near-\n",
      "infrared photochromic probe.\n",
      "Photoacoustic tomography (PAT) of genetically encoded probes allows for imaging of targeted\n",
      "biological processes deep in tissues with high spatial resolution; however, high background signals\n",
      "from blood can limit the achievable detection sensitivity. Here we describe a reversibly switchable\n",
      "nonfluorescent bacterial phytochrome for use in multiscale photoacoustic imaging, BphP1, with the\n",
      "most red-shifted absorption among genetically encoded probes. BphP1 binds a heme-derived biliverdin\n",
      "chromophore and is reversibly photoconvertible between red and near-infrared light-absorption\n",
      "states. We combined single-wavelength PAT with efficient BphP1 photoswitching, which enabled\n",
      "differential imaging with substantially decreased background signals, enhanced detection\n",
      "sensitivity, increased penetration depth and improved spatial resolution. We monitored tumor growth\n",
      "and metastasis with ∼ 100-μm resolution at depths approaching 10 mm using photoacoustic computed\n",
      "tomography, and we imaged individual cancer cells with a suboptical-diffraction resolution of ∼ 140\n",
      "nm using photoacoustic microscopy. This technology is promising for biomedical studies at several\n",
      "scales.\n",
      "['photoacoustic tomography (PAT)',\n",
      " 'photoacoustic microscopy',\n",
      " 'computed tomography']\n",
      "Elapsed time: 13.31 seconds\n",
      "-------------------\n",
      "26619013\n",
      "Concentric-flow electrokinetic injector enables serial crystallography of ribosome and photosystem\n",
      "II.\n",
      "We describe a concentric-flow electrokinetic injector for efficiently delivering microcrystals for\n",
      "serial femtosecond X-ray crystallography analysis that enables studies of challenging biological\n",
      "systems in their unadulterated mother liquor. We used the injector to analyze microcrystals of\n",
      "Geobacillus stearothermophilus thermolysin (2.2-Å structure), Thermosynechococcus elongatus\n",
      "photosystem II (<3-Å diffraction) and Thermus thermophilus small ribosomal subunit bound to the\n",
      "antibiotic paromomycin at ambient temperature (3.4-Å structure).\n",
      "['femtosecond X-ray crystallography',\n",
      " 'electrokinetic injector',\n",
      " 'microcrystals']\n",
      "Elapsed time: 8.84 seconds\n",
      "-------------------\n",
      "26641311\n",
      "Three-dimensional force microscopy of cells in biopolymer networks.\n",
      "We describe a technique for the quantitative measurement of cell-generated forces in highly\n",
      "nonlinear three-dimensional biopolymer networks that mimic the physiological situation of living\n",
      "cells. We computed forces of MDA-MB-231 breast carcinoma cells from the measured network\n",
      "deformations around the cells using a finite-element approach based on a constitutive equation that\n",
      "captures the complex mechanical properties of diverse biopolymers such as collagen gels, fibrin gels\n",
      "and Matrigel. Our measurements show that breast carcinoma cells cultured in collagen gels generated\n",
      "nearly constant forces regardless of the collagen concentration and matrix stiffness. Furthermore,\n",
      "time-lapse force measurements showed that these cells migrated in a gliding motion with alternating\n",
      "phases of high and low contractility, elongation, migratory speed and persistence.\n",
      "['force microscopy', 'finite-element approach', 'time-lapse force measurements']\n",
      "Elapsed time: 8.28 seconds\n",
      "-------------------\n",
      "26657556\n",
      "TRP channel mediated neuronal activation and ablation in freely behaving zebrafish.\n",
      "The zebrafish (Danio rerio) is a useful vertebrate model system in which to study neural circuits\n",
      "and behavior, but tools to modulate neurons in freely behaving animals are limited. As poikilotherms\n",
      "that live in water, zebrafish are amenable to thermal and pharmacological perturbations. We exploit\n",
      "these properties by using transient receptor potential (TRP) channels to activate or ablate specific\n",
      "neuronal populations using the chemical and thermal agonists of heterologously expressed TRPV1,\n",
      "TRPM8 and TRPA1.\n",
      "['TRP channel activation',\n",
      " 'TRP channel ablation',\n",
      " 'thermal agonists',\n",
      " 'chemical agonists',\n",
      " 'heterologously expressed TRPV1',\n",
      " 'heterologously expressed TRPM8',\n",
      " 'heterologously expressed TRPA1']\n",
      "Elapsed time: 8.69 seconds\n",
      "-------------------\n",
      "26657558\n",
      "Nanoscale optomechanical actuators for controlling mechanotransduction in living cells.\n",
      "To control receptor tension optically at the cell surface, we developed an approach involving\n",
      "optomechanical actuator nanoparticles that are controlled with near-infrared light. Illumination\n",
      "leads to particle collapse, delivering piconewton forces to specific cell surface receptors with\n",
      "high spatial and temporal resolution. We demonstrate optomechanical actuation by controlling\n",
      "integrin-based focal adhesion formation, cell protrusion and migration, and T cell receptor\n",
      "activation.\n",
      "['optomechanical actuators',\n",
      " 'near-infrared light',\n",
      " 'particle collapse',\n",
      " 'piconewton forces',\n",
      " 'cell surface receptors',\n",
      " 'integrin-based focal adhesion formation',\n",
      " 'cell protrusion and migration',\n",
      " 'T cell receptor activation']\n",
      "Elapsed time: 8.53 seconds\n",
      "-------------------\n",
      "26657559\n",
      "Inverted light-sheet microscope for imaging mouse pre-implantation development.\n",
      "Despite its importance for understanding human infertility and congenital diseases, early mammalian\n",
      "development has remained inaccessible to in toto imaging. We developed an inverted light-sheet\n",
      "microscope that enabled us to image mouse embryos from zygote to blastocyst, computationally track\n",
      "all cells and reconstruct a complete lineage tree of mouse pre-implantation development. We used\n",
      "this unique data set to show that the first cell fate specification occurs at the 16-cell stage.\n",
      "['inverted light-sheet microscope',\n",
      " 'computational tracking',\n",
      " 'complete lineage tree']\n",
      "Elapsed time: 7.29 seconds\n",
      "-------------------\n",
      "26716559\n",
      "How good can cryo-EM become?\n",
      "The suddenness with which single-particle cryo-electron microscopy (cryo-EM) has emerged as a method\n",
      "for determining high-resolution structures of biological macromolecules invites the questions, how\n",
      "much better can this technology get, and how fast is that likely to happen? Though we can rightly\n",
      "celebrate the maturation of cryo-EM as a high-resolution structure-determination tool, I believe\n",
      "there still are many developments to look forward to.\n",
      "['cryo-electron microscopy',\n",
      " 'single-particle cryo-EM',\n",
      " 'high-resolution structure determination']\n",
      "Elapsed time: 7.34 seconds\n",
      "-------------------\n",
      "26778924\n",
      "Whole-brain activity mapping onto a zebrafish brain atlas.\n",
      "In order to localize the neural circuits involved in generating behaviors, it is necessary to assign\n",
      "activity onto anatomical maps of the nervous system. Using brain registration across hundreds of\n",
      "larval zebrafish, we have built an expandable open-source atlas containing molecular labels and\n",
      "definitions of anatomical regions, the Z-Brain. Using this platform and immunohistochemical\n",
      "detection of phosphorylated extracellular signal–regulated kinase (ERK) as a readout of neural\n",
      "activity, we have developed a system to create and contextualize whole-brain maps of stimulus- and\n",
      "behavior-dependent neural activity. This mitogen-activated protein kinase (MAP)-mapping assay is\n",
      "technically simple, and data analysis is completely automated. Because MAP-mapping is performed on\n",
      "freely swimming fish, it is applicable to studies of nearly any stimulus or behavior. Here we\n",
      "demonstrate our high-throughput approach using pharmacological, visual and noxious stimuli, as well\n",
      "as hunting and feeding. The resultant maps outline hundreds of areas associated with behaviors.\n",
      "['immunohistochemical detection',\n",
      " 'phosphorylated extracellular signal–regulated kinase (ERK)',\n",
      " 'mitogen-activated protein kinase (MAP)-mapping assay']\n",
      "Elapsed time: 11.60 seconds\n",
      "-------------------\n",
      "26780092\n",
      "Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis.\n",
      "The transcriptional state of a cell reflects a variety of biological factors, from cell-type-\n",
      "specific features to transient processes such as the cell cycle, all of which may be of interest.\n",
      "However, identifying such aspects from noisy single-cell RNA-seq data remains challenging. We\n",
      "developed pathway and gene set overdispersion analysis (PAGODA) to resolve multiple, potentially\n",
      "overlapping aspects of transcriptional heterogeneity by testing gene sets for coordinated\n",
      "variability among measured cells.\n",
      "['PAGODA', 'single-cell RNA-seq data']\n",
      "Elapsed time: 7.26 seconds\n",
      "-------------------\n",
      "26808668\n",
      "Multi-target spectrally resolved fluorescence lifetime imaging microscopy.\n",
      "We introduce a pattern-matching technique for efficient identification of fluorophore ratios in\n",
      "complex multidimensional fluorescence signals using reference fluorescence decay and spectral\n",
      "signature patterns of individual fluorescent probes. Alternating pulsed laser excitation at three\n",
      "different wavelengths and time-resolved detection on 32 spectrally separated detection channels\n",
      "ensures efficient excitation of fluorophores and a maximum gain of fluorescence information. Using\n",
      "spectrally resolved fluorescence lifetime imaging microscopy (sFLIM), we were able to visualize up\n",
      "to nine different target molecules simultaneously in mouse C2C12 cells. By exploiting the\n",
      "sensitivity of fluorescence emission spectra and the lifetime of organic fluorophores on\n",
      "environmental factors, we carried out fluorescence imaging of three different target molecules in\n",
      "human U2OS cells with the same fluorophore. Our results demonstrate that sFLIM can be used for\n",
      "super-resolution multi-target imaging by stimulated emission depletion (STED).\n",
      "['fluorescence lifetime imaging microscopy',\n",
      " 'spectrally resolved fluorescence lifetime imaging microscopy',\n",
      " 'alternating pulsed laser excitation',\n",
      " 'time-resolved detection',\n",
      " 'spectral signature patterns',\n",
      " 'STED microscopy']\n",
      "Elapsed time: 12.02 seconds\n",
      "-------------------\n",
      "26808669\n",
      "A genetically targetable near-infrared photosensitizer.\n",
      "Upon illumination, photosensitizer molecules produce reactive oxygen species that can be used for\n",
      "functional manipulation of living cells, including protein inactivation, targeted-damage\n",
      "introduction and cellular ablation. Photosensitizers used to date have been either exogenous,\n",
      "resulting in delivery and removal challenges, or genetically encoded proteins that form or bind a\n",
      "native photosensitizing molecule, resulting in a constitutively active photosensitizer inside the\n",
      "cell. We describe a genetically encoded fluorogen-activating protein (FAP) that binds a heavy atom-\n",
      "substituted fluorogenic dye, forming an 'on-demand' activated photosensitizer that produces singlet\n",
      "oxygen and fluorescence when activated with near-infrared light. This targeted and activated\n",
      "photosensitizer (TAPs) approach enables protein inactivation, targeted cell killing and rapid\n",
      "targeted lineage ablation in living larval and adult zebrafish. The near-infrared excitation and\n",
      "emission of this FAP-TAPs provides a new spectral range for photosensitizer proteins that could be\n",
      "useful for imaging, manipulation and cellular ablation deep within living organisms.\n",
      "['fluorescence', 'singlet oxygen', 'near-infrared light']\n",
      "Elapsed time: 11.54 seconds\n",
      "-------------------\n",
      "26808670\n",
      "Highly multiplexed simultaneous detection of RNAs and proteins in single cells.\n",
      "To enable the detection of expression signatures specific to individual cells, we developed PLAYR\n",
      "(proximity ligation assay for RNA), a method for highly multiplexed transcript quantification by\n",
      "flow and mass cytometry that is compatible with standard antibody staining. When used with mass\n",
      "cytometry, PLAYR allowed for the simultaneous quantification of more than 40 different mRNAs and\n",
      "proteins. In primary cells, we quantified multiple transcripts, with the identity and functional\n",
      "state of each analyzed cell defined on the basis of the expression of a separate set of transcripts\n",
      "or proteins. By expanding high-throughput deep phenotyping of cells beyond protein epitopes to\n",
      "include RNA expression, PLAYR opens a new avenue for the characterization of cellular metabolism.\n",
      "['flow cytometry', 'mass cytometry', 'proximity ligation assay for RNA']\n",
      "Elapsed time: 8.77 seconds\n",
      "-------------------\n",
      "26855362\n",
      "Sensory and optogenetically driven single-vessel fMRI.\n",
      "Magnetic resonance imaging (MRI) sensitivity approaches vessel specificity. We developed a single-\n",
      "vessel functional MRI (fMRI) method to image the contribution of vascular components to blood\n",
      "oxygenation level-dependent (BOLD) and cerebral blood volume (CBV) fMRI signal. We mapped individual\n",
      "vessels penetrating the rat somatosensory cortex with 100-ms temporal resolution by MRI with sensory\n",
      "or optogenetic stimulation. The BOLD signal originated primarily from venules, and the CBV signal\n",
      "from arterioles. The single-vessel fMRI method and its combination with optogenetics provide a\n",
      "platform for mapping the hemodynamic signal through the neurovascular network with specificity at\n",
      "the level of individual arterioles and venules.\n",
      "['magnetic resonance imaging',\n",
      " 'functional MRI',\n",
      " 'sensory stimulation',\n",
      " 'optogenetic stimulation']\n",
      "Elapsed time: 9.03 seconds\n",
      "-------------------\n",
      "26878381\n",
      "Simultaneous fast measurement of circuit dynamics at multiple sites across the mammalian brain.\n",
      "Real-time activity measurements from multiple specific cell populations and projections are likely\n",
      "to be important for understanding the brain as a dynamical system. Here we developed frame-projected\n",
      "independent-fiber photometry (FIP), which we used to record fluorescence activity signals from many\n",
      "brain regions simultaneously in freely behaving mice. We explored the versatility of the FIP\n",
      "microscope by quantifying real-time activity relationships among many brain regions during social\n",
      "behavior, simultaneously recording activity along multiple axonal pathways during sensory\n",
      "experience, performing simultaneous two-color activity recording, and applying optical perturbation\n",
      "tuned to elicit dynamics that match naturally occurring patterns observed during behavior.\n",
      "['frame-projected independent-fiber photometry (FIP)',\n",
      " 'real-time activity measurements',\n",
      " 'simultaneous two-color activity recording',\n",
      " 'optical perturbation']\n",
      "Elapsed time: 8.80 seconds\n",
      "-------------------\n",
      "26878382\n",
      "Single-molecule imaging of non-equilibrium molecular ensembles on the millisecond timescale.\n",
      "Single-molecule fluorescence microscopy is uniquely suited for detecting transient molecular\n",
      "recognition events, yet achieving the time resolution and statistics needed to realize this\n",
      "potential has proven challenging. Here we present a single-molecule imaging and analysis platform\n",
      "using scientific complementary metal-oxide semiconductor (sCMOS) detectors that enables imaging of\n",
      "15,000 individual molecules simultaneously at millisecond rates. This system enabled the detection\n",
      "of previously obscured processes relevant to the fidelity mechanism in protein synthesis.\n",
      "['single-molecule fluorescence microscopy', 'sCMOS detectors']\n",
      "Elapsed time: 7.44 seconds\n",
      "-------------------\n",
      "26878383\n",
      "Apollo-NADP(+): a spectrally tunable family of genetically encoded sensors for NADP(+).\n",
      "NADPH-dependent antioxidant pathways have a critical role in scavenging hydrogen peroxide (H2O2)\n",
      "produced by oxidative phosphorylation. Inadequate scavenging results in H2O2 accumulation and can\n",
      "cause disease. To measure NADPH/NADP(+) redox states, we explored genetically encoded sensors based\n",
      "on steady-state fluorescence anisotropy due to FRET (fluorescence resonance energy transfer) between\n",
      "homologous fluorescent proteins (homoFRET); we refer to these sensors as Apollo sensors. We created\n",
      "an Apollo sensor for NADP(+) (Apollo-NADP(+)) that exploits NADP(+)-dependent homodimerization of\n",
      "enzymatically inactive glucose-6-phosphate dehydrogenase (G6PD). This sensor is reversible,\n",
      "responsive to glucose-stimulated metabolism and spectrally tunable for compatibility with many other\n",
      "sensors. We used Apollo-NADP(+) to study beta cells responding to oxidative stress and demonstrated\n",
      "that NADPH is significantly depleted before H2O2 accumulation by imaging a Cerulean-tagged version\n",
      "of Apollo-NADP(+) with the H2O2 sensor HyPer.\n",
      "['fluorescence resonance energy transfer',\n",
      " 'steady-state fluorescence anisotropy',\n",
      " 'spectral tunability']\n",
      "Elapsed time: 14.17 seconds\n",
      "-------------------\n",
      "26928761\n",
      "SuReSim: simulating localization microscopy experiments from ground truth models.\n",
      "Super-resolution fluorescence microscopy has become a widely used tool in many areas of research.\n",
      "However, designing and validating super-resolution experiments to address a research question in a\n",
      "technically feasible and scientifically rigorous manner remains a fundamental challenge. We\n",
      "developed SuReSim, a software tool that simulates localization data of arbitrary three-dimensional\n",
      "structures represented by ground truth models, allowing users to systematically explore how changing\n",
      "experimental parameters can affect potential imaging outcomes.\n",
      "['localization microscopy', 'super-resolution fluorescence microscopy']\n",
      "Elapsed time: 6.87 seconds\n",
      "-------------------\n",
      "26928762\n",
      "One library to make them all: streamlining the creation of yeast libraries via a SWAp-Tag strategy.\n",
      "The yeast Saccharomyces cerevisiae is ideal for systematic studies relying on collections of\n",
      "modified strains (libraries). Despite the significance of yeast libraries and the immense variety of\n",
      "available tags and regulatory elements, only a few such libraries exist, as their construction is\n",
      "extremely expensive and laborious. To overcome these limitations, we developed a SWAp-Tag (SWAT)\n",
      "method that enables one parental library to be modified easily and efficiently to give rise to an\n",
      "endless variety of libraries of choice. To showcase the versatility of the SWAT approach, we\n",
      "constructed and investigated a library of ∼1,800 strains carrying SWAT-GFP modules at the amino\n",
      "termini of endomembrane proteins and then used it to create two new libraries (mCherry and seamless\n",
      "GFP). Our work demonstrates how the SWAT method allows fast and effortless creation of yeast\n",
      "libraries, opening the door to new ways of systematically studying cell biology.\n",
      "['SWAp-Tag method',\n",
      " 'SWAT approach',\n",
      " 'GFP modules',\n",
      " 'endomembrane proteins',\n",
      " 'mCherry',\n",
      " 'seamless GFP']\n",
      "Elapsed time: 10.72 seconds\n",
      "-------------------\n",
      "26950744\n",
      "A saposin-lipoprotein nanoparticle system for membrane proteins.\n",
      "A limiting factor in membrane protein research is the ability to solubilize and stabilize such\n",
      "proteins. Detergents are used most often for solubilizing membrane proteins, but they are associated\n",
      "with protein instability and poor compatibility with structural and biophysical studies. Here we\n",
      "present a saposin-lipoprotein nanoparticle system, Salipro, which allows for the reconstitution of\n",
      "membrane proteins in a lipid environment that is stabilized by a scaffold of saposin proteins. We\n",
      "demonstrate the applicability of the method on two purified membrane protein complexes as well as by\n",
      "the direct solubilization and nanoparticle incorporation of a viral membrane protein complex from\n",
      "the virus membrane. Our approach facilitated high-resolution structural studies of the bacterial\n",
      "peptide transporter PeptTSo2 by single-particle cryo-electron microscopy (cryo-EM) and allowed us to\n",
      "stabilize the HIV envelope glycoprotein in a functional state.\n",
      "['cryo-electron microscopy (cryo-EM)',\n",
      " 'single-particle cryo-electron microscopy (cryo-EM)']\n",
      "Elapsed time: 10.81 seconds\n",
      "-------------------\n",
      "26950745\n",
      "High-density three-dimensional localization microscopy across large volumes.\n",
      "Extending three-dimensional (3D) single-molecule localization microscopy away from the coverslip and\n",
      "into thicker specimens will greatly broaden its biological utility. However, because of the\n",
      "limitations of both conventional imaging modalities and conventional labeling techniques, it is a\n",
      "challenge to localize molecules in three dimensions with high precision in such samples while\n",
      "simultaneously achieving the labeling densities required for high resolution of densely crowded\n",
      "structures. Here we combined lattice light-sheet microscopy with newly developed, freely diffusing,\n",
      "cell-permeable chemical probes with targeted affinity for DNA, intracellular membranes or the plasma\n",
      "membrane. We used this combination to perform high-localization precision, ultrahigh-labeling\n",
      "density, multicolor localization microscopy in samples up to 20 μm thick, including dividing cells\n",
      "and the neuromast organ of a zebrafish embryo. We also demonstrate super-resolution correlative\n",
      "imaging with protein-specific photoactivable fluorophores, providing a mutually compatible, single-\n",
      "platform alternative to correlative light-electron microscopy over large volumes.\n",
      "['lattice light-sheet microscopy',\n",
      " 'single-molecule localization microscopy',\n",
      " 'ultrahigh-labeling density localization microscopy',\n",
      " 'super-resolution correlative imaging',\n",
      " 'protein-specific photoactivable fluorophores']\n",
      "Elapsed time: 12.30 seconds\n",
      "-------------------\n",
      "27018577\n",
      "Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP).\n",
      "As RNA-binding proteins (RBPs) play essential roles in cellular physiology by interacting with\n",
      "target RNA molecules, binding site identification by UV crosslinking and immunoprecipitation (CLIP)\n",
      "of ribonucleoprotein complexes is critical to understanding RBP function. However, current CLIP\n",
      "protocols are technically demanding and yield low-complexity libraries with high experimental\n",
      "failure rates. We have developed an enhanced CLIP (eCLIP) protocol that decreases requisite\n",
      "amplification by ∼1,000-fold, decreasing discarded PCR duplicate reads by ∼60% while maintaining\n",
      "single-nucleotide binding resolution. By simplifying the generation of paired IgG and size-matched\n",
      "input controls, eCLIP improves specificity in the discovery of authentic binding sites. We generated\n",
      "102 eCLIP experiments for 73 diverse RBPs in HepG2 and K562 cells (available at\n",
      "https://www.encodeproject.org), demonstrating that eCLIP enables large-scale and robust profiling,\n",
      "with amplification and sample requirements similar to those of ChIP-seq. eCLIP enables integrative\n",
      "analysis of diverse RBPs to reveal factor-specific profiles, common artifacts for CLIP and RNA-\n",
      "centric perspectives on RBP activity.\n",
      "['CLIP', 'ChIP-seq', 'eCLIP']\n",
      "Elapsed time: 12.43 seconds\n",
      "-------------------\n",
      "27018580\n",
      "Quantitative super-resolution imaging with qPAINT.\n",
      "Counting molecules in complexes is challenging, even with super-resolution microscopy. Here, we use\n",
      "the programmable and specific binding of dye-labeled DNA probes to count integer numbers of targets.\n",
      "This method, called quantitative points accumulation in nanoscale topography (qPAINT), works\n",
      "independently of dye photophysics for robust counting with high precision and accuracy over a wide\n",
      "dynamic range. qPAINT was benchmarked on DNA nanostructures and demonstrated for cellular\n",
      "applications by quantifying proteins in situ and the number of single-molecule FISH probes bound to\n",
      "an mRNA target.\n",
      "['super-resolution microscopy', 'qPAINT', 'DNA probes']\n",
      "Elapsed time: 7.67 seconds\n",
      "-------------------\n",
      "27064647\n",
      "Expansion microscopy with conventional antibodies and fluorescent proteins.\n",
      "Expansion microscopy is a technique in which fluorophores on fixed specimens are linked to a\n",
      "swellable polymer that is physically expanded to enable super-resolution microscopy with ordinary\n",
      "microscopes. We have developed and characterized new methods for linking fluorophores to the polymer\n",
      "that now enable expansion microscopy with conventional fluorescently labeled antibodies and\n",
      "fluorescent proteins. Our methods simplify the procedure and expand the palette of compatible\n",
      "labels, allowing rapid dissemination of the technique.\n",
      "['expansion microscopy',\n",
      " 'super-resolution microscopy',\n",
      " 'fluorescently labeled antibodies',\n",
      " 'fluorescent proteins']\n",
      "Elapsed time: 7.58 seconds\n",
      "-------------------\n",
      "27088314\n",
      "siFLIM: single-image frequency-domain FLIM provides fast and photon-efficient lifetime data.\n",
      "We developed single-image fluorescence lifetime imaging microscopy (siFLIM), a method for acquiring\n",
      "quantitative lifetime images from a single exposure. siFLIM takes advantage of a new generation of\n",
      "dedicated cameras that simultaneously record two 180°-phase-shifted images, and it allows for video-\n",
      "rate lifetime imaging with minimal phototoxicity and bleaching. siFLIM is also inherently immune to\n",
      "artifacts stemming from rapid cellular movements and signal transients.\n",
      "['siFLIM',\n",
      " 'single-image fluorescence lifetime imaging microscopy',\n",
      " 'video-rate lifetime imaging',\n",
      " 'minimal phototoxicity and bleaching']\n",
      "Elapsed time: 7.99 seconds\n",
      "-------------------\n",
      "27110629\n",
      "The development of cryo-EM into a mainstream  structural biology technique.\n",
      "Single-particle cryo-electron microscopy (cryo-EM) has emerged over the last two decades as a\n",
      "technique capable of studying challenging systems that otherwise defy structural characterization.\n",
      "Recent technical advances have resulted in a 'quantum leap' in applicability, throughput and\n",
      "achievable resolution that has gained this technique worldwide attention. Here I discuss some of the\n",
      "major historical landmarks in the development of the cryo-EM field, ultimately leading to its\n",
      "present success.\n",
      "['cryo-electron microscopy', 'single-particle cryo-EM', 'quantum leap']\n",
      "Elapsed time: 7.30 seconds\n",
      "-------------------\n",
      "27111507\n",
      "Automated structure modeling of large protein assemblies using crosslinks as distance restraints.\n",
      "Crosslinking mass spectrometry is increasingly used for structural characterization of multisubunit\n",
      "protein complexes. Chemical crosslinking captures conformational heterogeneity, which typically\n",
      "results in conflicting crosslinks that cannot be satisfied in a single model, making detailed\n",
      "modeling a challenging task. Here we introduce an automated modeling method dedicated to large\n",
      "protein assemblies ('XL-MOD' software is available at http://aria.pasteur.fr/supplementary-\n",
      "data/x-links) that (i) uses a form of spatial restraints that realistically reflects the\n",
      "distribution of experimentally observed crosslinked distances; (ii) automatically deals with\n",
      "ambiguous and/or conflicting crosslinks and identifies alternative conformations within a Bayesian\n",
      "framework; and (iii) allows subunit structures to be flexible during conformational sampling. We\n",
      "demonstrate our method by testing it on known structures and available crosslinking data. We also\n",
      "crosslinked and modeled the 17-subunit yeast RNA polymerase III at atomic resolution; the resulting\n",
      "model agrees remarkably well with recently published cryoelectron microscopy structures and provides\n",
      "additional insights into the polymerase structure.\n",
      "['crosslinking mass spectrometry',\n",
      " 'chemical crosslinking',\n",
      " 'automated modeling method']\n",
      "Elapsed time: 11.59 seconds\n",
      "-------------------\n",
      "27123817\n",
      "Measuring cell-generated forces: a guide to the available tools.\n",
      "Forces generated by cells are critical regulators of cell adhesion, signaling, and function, and\n",
      "they are also essential drivers in the morphogenetic events of development. Over the past 20 years,\n",
      "several methods have been developed to measure these forces. However, despite recent substantial\n",
      "interest in understanding the contribution of these forces in biology, implementation and adoption\n",
      "of the developed methods by the broader biological community remain challenging because of the\n",
      "inherently multidisciplinary expertise required to conduct and interpret the measurements. In this\n",
      "review, we introduce the established methods and highlight the technical challenges associated with\n",
      "implementing each technique in a biological laboratory.\n",
      "['available tools',\n",
      " 'cell adhesion',\n",
      " 'cell signaling',\n",
      " 'cell function',\n",
      " 'morphogenetic events',\n",
      " 'development']\n",
      "Elapsed time: 7.82 seconds\n",
      "-------------------\n",
      "27135972\n",
      "Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs.\n",
      "Drug sensitivity and resistance are conventionally quantified by IC50 or Emax values, but these\n",
      "metrics are highly sensitive to the number of divisions taking place over the course of a response\n",
      "assay. The dependency of IC50 and Emax on division rate creates artefactual correlations between\n",
      "genotype and drug sensitivity, while obscuring valuable biological insights and interfering with\n",
      "biomarker discovery. We derive alternative small molecule drug-response metrics that are insensitive\n",
      "to division number. These are based on estimation of the magnitude of drug-induced growth rate\n",
      "inhibition (GR) using endpoint or time-course assays. We show that GR50 and GRmax are superior to\n",
      "conventional metrics for assessing the effects of small molecule drugs in dividing cells. Moreover,\n",
      "adopting GR metrics requires only modest changes in experimental protocols. We expect GR metrics to\n",
      "improve the study of cell signaling and growth using small molecules and biologics and to facilitate\n",
      "the discovery of drug-response biomarkers and the identification of drugs effective against specific\n",
      "patient-derived tumor cells.\n",
      "['endpoint assays', 'time-course assays']\n",
      "Elapsed time: 9.79 seconds\n",
      "-------------------\n",
      "27135974\n",
      "An unbiased metric of antiproliferative drug effect in vitro.\n",
      "In vitro cell proliferation assays are widely used in pharmacology, molecular biology, and drug\n",
      "discovery. Using theoretical modeling and experimentation, we show that current metrics of\n",
      "antiproliferative small molecule effect suffer from time-dependent bias, leading to inaccurate\n",
      "assessments of parameters such as drug potency and efficacy. We propose the drug-induced\n",
      "proliferation (DIP) rate, the slope of the line on a plot of cell population doublings versus time,\n",
      "as an alternative, time-independent metric.\n",
      "['theoretical modeling', 'experimentation', 'cell population doublings']\n",
      "Elapsed time: 7.03 seconds\n",
      "-------------------\n",
      "27159085\n",
      "A bacterial phytochrome-based optogenetic system controllable with near-infrared light.\n",
      "Light-mediated control of protein-protein interactions to regulate cellular pathways is an important\n",
      "application of optogenetics. Here, we report an optogenetic system based on the reversible light-\n",
      "induced binding between the bacterial phytochrome BphP1 and its natural partner PpsR2 from\n",
      "Rhodopseudomonas palustris bacteria. We extensively characterized the BphP1-PpsR2 interaction both\n",
      "in vitro and in mammalian cells and then used this interaction to translocate target proteins to\n",
      "specific cellular compartments, such as the plasma membrane and the nucleus. We showed light-\n",
      "inducible control of cell morphology that resulted in a substantial increase of the cell area. We\n",
      "demonstrated light-dependent gene expression with 40-fold contrast in cultured cells, 32-fold in\n",
      "subcutaneous mouse tissue, and 5.7-fold in deep tissues in mice. Characteristics of the BphP1-PpsR2\n",
      "optogenetic system include its sensitivity to 740- to 780-nm near-infrared light, its ability to\n",
      "utilize an endogenous biliverdin chromophore in eukaryotes (including mammals), and its spectral\n",
      "compatibility with blue-light-driven optogenetic systems.\n",
      "['near-infrared light', 'blue-light-driven optogenetic systems']\n",
      "Elapsed time: 12.39 seconds\n",
      "-------------------\n",
      "27183441\n",
      "Flyception: imaging brain activity in freely walking fruit flies.\n",
      "Genetically encoded calcium sensors have enabled monitoring of neural activity in vivo using optical\n",
      "imaging techniques. Linking neural activity to complex behavior remains challenging, however, as\n",
      "most imaging systems require tethering the animal, which can impact the animal's behavioral\n",
      "repertoire. Here, we report a method for monitoring the brain activity of untethered, freely walking\n",
      "Drosophila melanogaster during sensorially and socially evoked behaviors to facilitate the study of\n",
      "neural mechanisms that underlie naturalistic behaviors.\n",
      "['optical imaging techniques', 'genetically encoded calcium sensors', 'none']\n",
      "Elapsed time: 7.04 seconds\n",
      "-------------------\n",
      "27240257\n",
      "Quantitative assessment of fluorescent proteins.\n",
      "The advent of fluorescent proteins (FPs) for genetic labeling of molecules and cells has\n",
      "revolutionized fluorescence microscopy. Genetic manipulations have created a vast array of bright\n",
      "and stable FPs spanning blue to red spectral regions. Common to autofluorescent FPs is their tight\n",
      "β-barrel structure, which provides the rigidity and chemical environment needed for effectual\n",
      "fluorescence. Despite the common structure, each FP has unique properties. Thus, there is no single\n",
      "'best' FP for every circumstance, and each FP has advantages and disadvantages. To guide decisions\n",
      "about which FP is right for a given application, we have quantitatively characterized the\n",
      "brightness, photostability, pH stability and monomeric properties of more than 40 FPs to enable\n",
      "straightforward and direct comparison between them. We focus on popular and/or top-performing FPs in\n",
      "each spectral region.\n",
      "['brightness', 'photostability', 'pH stability', 'momeric properties']\n",
      "Elapsed time: 11.35 seconds\n",
      "-------------------\n",
      "27271196\n",
      "An E3-ligase-based method for ablating inhibitory synapses.\n",
      "Although neuronal activity can be modulated using a variety of techniques, there are currently few\n",
      "methods for controlling neuronal connectivity. We introduce a tool (GFE3) that mediates the fast,\n",
      "specific and reversible elimination of inhibitory synaptic inputs onto genetically determined\n",
      "neurons. GFE3 is a fusion between an E3 ligase, which mediates the ubiquitination and rapid\n",
      "degradation of proteins, and a recombinant, antibody-like protein (FingR) that binds to gephyrin.\n",
      "Expression of GFE3 leads to a strong and specific reduction of gephyrin in culture or in vivo and to\n",
      "a substantial decrease in phasic inhibition onto cells that express GFE3. By temporarily expressing\n",
      "GFE3 we showed that inhibitory synapses regrow following ablation. Thus, we have created a simple,\n",
      "reversible method for modulating inhibitory synaptic input onto genetically determined cells.\n",
      "['fluorescence microscopy',\n",
      " 'ubiquitination',\n",
      " 'recombinant protein',\n",
      " 'antibody-like protein',\n",
      " 'phasic inhibition']\n",
      "Elapsed time: 10.19 seconds\n",
      "-------------------\n",
      "27271198\n",
      "Dense transcript profiling in single cells by image correlation decoding.\n",
      "Sequential barcoded fluorescent in situ hybridization (seqFISH) allows large numbers of molecular\n",
      "species to be accurately detected in single cells, but multiplexing is limited by the density of\n",
      "barcoded objects. We present correlation FISH (corrFISH), a method to resolve dense temporal\n",
      "barcodes in sequential hybridization experiments. Using corrFISH, we quantified highly expressed\n",
      "ribosomal protein genes in single cultured cells and mouse thymus sections, revealing cell-type-\n",
      "specific gene expression.\n",
      "['seqFISH', 'corrFISH', 'fluorescent in situ hybridization']\n",
      "Elapsed time: 7.16 seconds\n",
      "-------------------\n",
      "27295310\n",
      "Varying label density allows artifact-free analysis of membrane-protein nanoclusters.\n",
      "We present a method to robustly discriminate clustered from randomly distributed molecules detected\n",
      "with techniques based on single-molecule localization microscopy, such as PALM and STORM. The\n",
      "approach is based on deliberate variation of labeling density, such as titration of fluorescent\n",
      "antibody, combined with quantitative cluster analysis, and it thereby circumvents the problem of\n",
      "cluster artifacts generated by overcounting of blinking fluorophores. The method was used to analyze\n",
      "nanocluster formation in resting and activated immune cells.\n",
      "['PALM', 'STORM', 'single-molecule localization microscopy']\n",
      "Elapsed time: 7.38 seconds\n",
      "-------------------\n",
      "27376770\n",
      "Nanoscale imaging of RNA with expansion microscopy.\n",
      "The ability to image RNA identity and location with nanoscale precision in intact tissues is of\n",
      "great interest for defining cell types and states in normal and pathological biological settings.\n",
      "Here, we present a strategy for expansion microscopy of RNA. We developed a small-molecule linker\n",
      "that enables RNA to be covalently attached to a swellable polyelectrolyte gel synthesized throughout\n",
      "a biological specimen. Then, postexpansion, fluorescent in situ hybridization (FISH) imaging of RNA\n",
      "can be performed with high yield and specificity as well as single-molecule precision in both\n",
      "cultured cells and intact brain tissue. Expansion FISH (ExFISH) separates RNAs and supports\n",
      "amplification of single-molecule signals (i.e., via hybridization chain reaction) as well as\n",
      "multiplexed RNA FISH readout. ExFISH thus enables super-resolution imaging of RNA structure and\n",
      "location with diffraction-limited microscopes in thick specimens, such as intact brain tissue and\n",
      "other tissues of importance to biology and medicine.\n",
      "['expansion microscopy',\n",
      " 'fluorescent in situ hybridization (FISH)',\n",
      " 'hybridization chain reaction']\n",
      "Elapsed time: 10.95 seconds\n",
      "-------------------\n",
      "27400419\n",
      "Reversible cryo-arrest for imaging molecules in living cells at high spatial resolution.\n",
      "The dynamics of molecules in living cells hampers precise imaging of molecular patterns by\n",
      "functional and super-resolution microscopy. We developed a method that circumvents lethal chemical\n",
      "fixation and allows on-stage cryo-arrest for consecutive imaging of molecular patterns within the\n",
      "same living, but arrested, cells. The reversibility of consecutive cryo-arrests was demonstrated by\n",
      "the high survival rate of different cell lines and by intact growth factor signaling that was not\n",
      "perturbed by stress response. Reversible cryo-arrest was applied to study the evolution of ligand-\n",
      "induced receptor tyrosine kinase activation at different scales. The nanoscale clustering of\n",
      "epidermal growth factor receptor (EGFR) in the plasma membrane was assessed by single-molecule\n",
      "localization microscopy, and endosomal microscale activity patterns of ephrin receptor A2 (EphA2)\n",
      "were assessed by fluorescence lifetime imaging microscopy. Reversible cryo-arrest allows the precise\n",
      "determination of molecular patterns while conserving the dynamic capabilities of living cells.\n",
      "['single-molecule localization microscopy',\n",
      " 'fluorescence lifetime imaging microscopy']\n",
      "Elapsed time: 10.56 seconds\n",
      "-------------------\n",
      "27427858\n",
      "LOVTRAP: an optogenetic system for photoinduced protein dissociation.\n",
      "LOVTRAP is an optogenetic approach for reversible light-induced protein dissociation using protein A\n",
      "fragments that bind to the LOV domain only in the dark, with tunable kinetics and a >150-fold change\n",
      "in the dissociation constant (Kd). By reversibly sequestering proteins at mitochondria, we precisely\n",
      "modulated the proteins' access to the cell edge, demonstrating a naturally occurring 3-mHz cell-edge\n",
      "oscillation driven by interactions of Vav2, Rac1, and PI3K proteins.\n",
      "['optogenetic', 'photoinduced protein dissociation']\n",
      "Elapsed time: 7.41 seconds\n",
      "-------------------\n",
      "27467726\n",
      "A practical guide to photoacoustic tomography in the life sciences.\n",
      "The life sciences can benefit greatly from imaging technologies that connect microscopic discoveries\n",
      "with macroscopic observations. One technology uniquely positioned to provide such benefits is\n",
      "photoacoustic tomography (PAT), a sensitive modality for imaging optical absorption contrast over a\n",
      "range of spatial scales at high speed. In PAT, endogenous contrast reveals a tissue's anatomical,\n",
      "functional, metabolic, and histologic properties, and exogenous contrast provides molecular and\n",
      "cellular specificity. The spatial scale of PAT covers organelles, cells, tissues, organs, and small\n",
      "animals. Consequently, PAT is complementary to other imaging modalities in contrast mechanism,\n",
      "penetration, spatial resolution, and temporal resolution. We review the fundamentals of PAT and\n",
      "provide practical guidelines for matching PAT systems with research needs. We also summarize the\n",
      "most promising biomedical applications of PAT, discuss related challenges, and envision PAT's\n",
      "potential to lead to further breakthroughs.\n",
      "['photoacoustic tomography', 'PAT']\n",
      "Elapsed time: 9.64 seconds\n",
      "-------------------\n",
      "27467727\n",
      "Contrast agents for molecular photoacoustic imaging.\n",
      "Photoacoustic imaging (PAI) is an emerging tool that bridges the traditional depth limits of\n",
      "ballistic optical imaging and the resolution limits of diffuse optical imaging. Using the acoustic\n",
      "waves generated in response to the absorption of pulsed laser light, it provides noninvasive images\n",
      "of absorbed optical energy density at depths of several centimeters with a resolution of ∼100 μm.\n",
      "This versatile and scalable imaging modality has now shown potential for molecular imaging, which\n",
      "enables visualization of biological processes with systemically introduced contrast agents.\n",
      "Understanding the relative merits of the vast range of contrast agents available, from small-\n",
      "molecule dyes to gold and carbon nanostructures to liposome encapsulations, is a considerable\n",
      "challenge. Here we critically review the physical, chemical and biochemical characteristics of the\n",
      "existing photoacoustic contrast agents, highlighting key applications and present challenges for\n",
      "molecular PAI.\n",
      "['photoacoustic imaging',\n",
      " 'molecular photoacoustic imaging',\n",
      " 'ballistic optical imaging',\n",
      " 'diffuse optical imaging']\n",
      "Elapsed time: 10.28 seconds\n",
      "-------------------\n",
      "27479327\n",
      "Revealing disease-associated pathways by network integration of untargeted metabolomics.\n",
      "Uncovering the molecular context of dysregulated metabolites is crucial to understand pathogenic\n",
      "pathways. However, their system-level analysis has been limited owing to challenges in global\n",
      "metabolite identification. Most metabolite features detected by untargeted metabolomics carried out\n",
      "by liquid-chromatography-mass spectrometry cannot be uniquely identified without additional, time-\n",
      "consuming experiments. We report a network-based approach, prize-collecting Steiner forest algorithm\n",
      "for integrative analysis of untargeted metabolomics (PIUMet), that infers molecular pathways and\n",
      "components via integrative analysis of metabolite features, without requiring their identification.\n",
      "We demonstrated PIUMet by analyzing changes in metabolism of sphingolipids, fatty acids and steroids\n",
      "in a Huntington's disease model. Additionally, PIUMet enabled us to elucidate putative identities of\n",
      "altered metabolite features in diseased cells, and infer experimentally undetected, disease-\n",
      "associated metabolites and dysregulated proteins. Finally, we established PIUMet's ability for\n",
      "integrative analysis of untargeted metabolomics data with proteomics data, demonstrating that this\n",
      "approach elicits disease-associated metabolites and proteins that cannot be inferred by individual\n",
      "analysis of these data.\n",
      "{'imaging_techniques': ['liquid-chromatography-mass spectrometry',\n",
      "                        'proteomics']}\n",
      "Elapsed time: 13.36 seconds\n",
      "-------------------\n",
      "27479328\n",
      "A far-red fluorescent protein evolved from a cyanobacterial phycobiliprotein.\n",
      "Far-red fluorescent proteins (FPs) are desirable for in vivo imaging because with these molecules\n",
      "less light is scattered, absorbed, or re-emitted by endogenous biomolecules compared with cyan,\n",
      "green, yellow, and orange FPs. We developed a new class of FP from an allophycocyanin α-subunit\n",
      "(APCα). Native APC requires a lyase to incorporate phycocyanobilin. The evolved FP, which we named\n",
      "small ultra-red FP (smURFP), covalently attaches a biliverdin (BV) chromophore without a lyase, and\n",
      "has 642/670-nm excitation-emission peaks, a large extinction coefficient (180,000 M(-1)cm(-1)) and\n",
      "quantum yield (18%), and photostability comparable to that of eGFP. smURFP has significantly greater\n",
      "BV incorporation rate and protein stability than the bacteriophytochrome (BPH) FPs. Moreover, BV\n",
      "supply is limited by membrane permeability, and smURFPs (but not BPH FPs) can incorporate a more\n",
      "membrane-permeant BV analog, making smURFP fluorescence comparable to that of FPs from jellyfish or\n",
      "coral. A far-red and near-infrared fluorescent cell cycle indicator was created with smURFP and a\n",
      "BPH FP.\n",
      "['fluorescence',\n",
      " 'in vivo imaging',\n",
      " 'far-red fluorescent proteins (FPs)',\n",
      " 'phycobiliprotein',\n",
      " 'lyase',\n",
      " 'biliverdin (BV)',\n",
      " 'chromophore',\n",
      " 'excitation-emission peaks',\n",
      " 'extinction coefficient',\n",
      " 'quantum yield',\n",
      " 'photostability',\n",
      " 'bacteriophytochrome (BPH) FPs',\n",
      " 'membrane permeability']\n",
      "Elapsed time: 15.76 seconds\n",
      "-------------------\n",
      "27479330\n",
      "Virtual microfluidics for digital quantification and single-cell sequencing.\n",
      "We have developed hydrogel-based virtual microfluidics as a simple and robust alternative to complex\n",
      "engineered microfluidic systems for the compartmentalization of nucleic acid amplification\n",
      "reactions. We applied in-gel digital multiple displacement amplification (dMDA) to purified DNA\n",
      "templates, cultured bacterial cells and human microbiome samples in the virtual microfluidics\n",
      "system, and demonstrated whole-genome sequencing of single-cell MDA products with excellent coverage\n",
      "uniformity and markedly reduced chimerism compared with products of liquid MDA reactions.\n",
      "['hydrogel-based virtual microfluidics',\n",
      " 'digital multiple displacement amplification (dMDA)',\n",
      " 'whole-genome sequencing']\n",
      "Elapsed time: 7.61 seconds\n",
      "-------------------\n",
      "27548807\n",
      "Shrinkage-mediated imaging of entire organs and organisms using uDISCO.\n",
      "Recent tissue-clearing approaches have become important alternatives to standard histology\n",
      "approaches. However, light scattering in thick tissues and the size restrictions on samples that can\n",
      "be imaged with standard light-sheet microscopy pose limitations for analyzing large samples such as\n",
      "an entire rodent body. We developed 'ultimate DISCO' (uDISCO) clearing to overcome these limitations\n",
      "in volumetric imaging. uDISCO preserves fluorescent proteins over months and renders intact organs\n",
      "and rodent bodies transparent while reducing their size up to 65%. We used uDISCO to image neuronal\n",
      "connections and vasculature from head to toe over 7 cm and to perform unbiased screening of\n",
      "transplanted stem cells within the entire body of adult mice. uDISCO is compatible with diverse\n",
      "labeling methods and archival human tissue, and it can readily be used in various biomedical\n",
      "applications to study organization of large organ systems throughout entire organisms.\n",
      "['uDISCO',\n",
      " 'standard histology approaches',\n",
      " 'light-sheet microscopy',\n",
      " 'fluorescent proteins']\n",
      "Elapsed time: 10.06 seconds\n",
      "-------------------\n",
      "27595403\n",
      "Augmenting CRISPR applications in Drosophila with tRNA-flanked sgRNAs.\n",
      "We present tRNA-based vectors for producing multiple clustered regularly interspaced short\n",
      "palindromic repeats (CRISPR) single guide RNAs (sgRNAs) from a single RNA polymerase II or III\n",
      "transcript in Drosophila. The system, which is based on liberation of sgRNAs by processing flanking\n",
      "tRNAs, permits highly efficient multiplexing of Cas9-based mutagenesis. We also demonstrate that the\n",
      "tRNA-sgRNA system markedly increases the efficacy of conditional gene disruption by Cas9 and can\n",
      "promote editing by the recently discovered RNA-guided endonuclease Cpf1.\n",
      "['RNA polymerase II', 'RNA polymerase III', 'tRNA-based vectors', 'sgRNAs']\n",
      "Elapsed time: 8.39 seconds\n",
      "-------------------\n",
      "27595405\n",
      "A multifunctional AAV-CRISPR-Cas9 and its host response.\n",
      "CRISPR-Cas9 delivery by adeno-associated virus (AAV) holds promise for gene therapy but faces\n",
      "critical barriers on account of its potential immunogenicity and limited payload capacity. Here, we\n",
      "demonstrate genome engineering in postnatal mice using AAV-split-Cas9, a multifunctional platform\n",
      "customizable for genome editing, transcriptional regulation, and other previously impracticable\n",
      "applications of AAV-CRISPR-Cas9. We identify crucial parameters that impact efficacy and clinical\n",
      "translation of our platform, including viral biodistribution, editing efficiencies in various\n",
      "organs, antigenicity, immunological reactions, and physiological outcomes. These results reveal that\n",
      "AAV-CRISPR-Cas9 evokes host responses with distinct cellular and molecular signatures, but unlike\n",
      "alternative delivery methods, does not induce extensive cellular damage in vivo. Our study provides\n",
      "a foundation for developing effective genome therapeutics.\n",
      "['AAV-CRISPR-Cas9',\n",
      " 'split-Cas9',\n",
      " 'biodistribution',\n",
      " 'editing efficiencies',\n",
      " 'antigenicity',\n",
      " 'immunological reactions',\n",
      " 'physiological outcomes']\n",
      "Elapsed time: 11.14 seconds\n",
      "-------------------\n",
      "27669326\n",
      "Enzyme-catalyzed expressed protein ligation.\n",
      "Expressed protein ligation is a valuable method for protein semisynthesis that involves the reaction\n",
      "of recombinant protein C-terminal thioesters with N-terminal cysteine (N-Cys)-containing peptides,\n",
      "but the requirement of a Cys residue at the ligation junction can limit the utility of this method.\n",
      "Here we employ subtiligase variants to efficiently ligate Cys-free peptides to protein thioesters.\n",
      "Using this method, we have more accurately determined the effect of C-terminal phosphorylation on\n",
      "the tumor suppressor protein PTEN.\n",
      "{'imaging_techniques': ['subtiligase variants', 'protein thioesters']}\n",
      "Elapsed time: 7.84 seconds\n",
      "-------------------\n"
     ]
    }
   ],
   "source": [
    "l2 = []\n",
    "t2 = []\n",
    "for i, row in df.iterrows():\n",
    "    if row.ABSTRACT == '':\n",
    "        continue\n",
    "    if row.ID_PAPER in lset:\n",
    "        continue\n",
    "    start_time = time.time()\n",
    "    print(row.ID_PAPER)\n",
    "    print(textwrap.fill(row.TITLE, 100))\n",
    "    print(textwrap.fill(row.ABSTRACT, 100))\n",
    "    p = prompt.format(article_text=(row.TITLE+'\\n'+row.ABSTRACT))\n",
    "    try:\n",
    "        output_text = llm(p)\n",
    "        output = output_parser.parse(output_text)\n",
    "        pprint(output, depth=2)\n",
    "        delta_t = time.time() - start_time\n",
    "        t.append(delta_t)\n",
    "        print(f'Elapsed time: {delta_t:.2f} seconds')\n",
    "        print('-------------------')\n",
    "        for ner in output:\n",
    "            l2.append((row.ID_PAPER, ner))\n",
    "    except Exception as e:\n",
    "        print(e)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3941\n",
      "1115\n"
     ]
    }
   ],
   "source": [
    "l.extend(l2)\n",
    "print(len(l))\n",
    "t.extend(t2)\n",
    "print(len(t))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2510\n"
     ]
    }
   ],
   "source": [
    "print(len(l2))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#| hide\n",
    "import nbdev; nbdev.nbdev_export()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Row(ID_PAPER='PPR418001', DOI='10.1101/2021.11.09.467827', YEAR='2021', ABSTRACT='<h4>ABSTRACT</h4> The close association between astrocytes and microglia causes great difficulties to distinguish their individual roles in innate immune responses in central nervous system. Current chemical-based methods to eliminate microglia in glial cell culture introduce various molecular and functional alterations to astrocytes. Here, we describe a novel two-step approach to achieve a complete elimination of microglia without affecting the biological properties of co-cultured astrocytes by temporal treatment of histone deacetylase inhibitor trichostatin A (TSA). We verify TSA as a potent inducer for microglial-specific cell death, which also causes comprehensive gene expression changes in astrocytes. However, withdrawal of TSA not only ensures no microglia repopulation, but also restores all the gene expression changes in terms of astrocyte functions, including neurotrophic factors, glutamate and potassium transporters, and reactive astrocyte subtypes. By contrast, withdrawal of PLX5622, the commonly used colony-stimulating factor 1 receptor inhibitor neither prevents microglia repopulation nor restores the gene expression changes mentioned above. Using this method, we are able to discriminate differential roles of microglia and astrocytes in the induced expression of antiviral and pro-inflammatory cytokines upon various pathological stimuli including the spike protein of SARS-CoV-2. This simple and efficient method can be customized for the understanding of microglia-astrocyte interaction and the development of epigenetic therapies that target over-activated microglia in neuroinflammation-related diseases.', TITLE='A novel histone deacetylase inhibitor-based approach to eliminate microglia and retain astrocyte properties in glial cell culture', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418002', DOI='10.1101/2021.11.09.467981', YEAR='2021', ABSTRACT='Although ACE2 is the primary receptor for SARS-CoV-2 infection, a systematic assessment of host factors that regulate binding to SARS-CoV-2 spike protein has not been described. Here we use whole genome CRISPR activation to identify host factors controlling cellular interactions with SARS-CoV-2. Our top hit was a TLR -related cell surface receptor called leucine-rich repeat-containing protein 15 ( LRRC15 ). LRRC15 expression was sufficient to promote SARS-CoV-2 Spike binding where they form a cell surface complex. LRRC15 mRNA is expressed in human collagen-producing lung myofibroblasts and LRRC15 protein is induced in severe COVID-19 infection where it can be found lining the airways. Mechanistically, LRRC15 does not itself support SARS-CoV-2 infection, but fibroblasts expressing LRRC15 can suppress both pseudotyped and authentic SARS-CoV-2 infection in trans . Moreover, LRRC15 expression in fibroblasts suppresses collagen production and promotes expression of IFIT, OAS, and MX-family antiviral factors. Overall, LRRC15 is a novel SARS-CoV-2 spike-binding receptor that can help control viral load and regulate antiviral and antifibrotic transcriptional programs in the context of COVID-19 infection.', TITLE='Fibroblast-expressed LRRC15 suppresses SARS-CoV-2 infection and controls antiviral and antifibrotic transcriptional programs', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418023', DOI='10.1101/2021.11.09.467862', YEAR='2021', ABSTRACT='<h4>ABSTRACT</h4> Mycobacterium tuberculosis (Mtb) and SARS-CoV-2 (CoV2) are the leading causes of death due to infectious disease. Although Mtb and CoV2 both cause serious and sometimes fatal respiratory infections, the effect of Mtb infection and its associated immune response on secondary infection with CoV2 is unknown. To address this question we applied two mouse models of COVID19, using mice which were chronically infected with Mtb. In both model systems, Mtb-infected mice were resistant to secondary CoV2 infection and its pathological consequences, and CoV2 infection did not affect Mtb burdens. Single cell RNA sequencing of coinfected and monoinfected lungs demonstrated the resistance of Mtb-infected mice is associated with expansion of T and B cell subsets upon viral challenge. Collectively, these data demonstrate that Mtb infection conditions the lung environment in a manner that is not conducive to CoV2 survival. <h4>AUTHOR SUMMARY</h4> Mycobacterium tuberculosis (Mtb) and SARS-CoV-2 (CoV2) are distinct organisms which both cause lung disease. We report the surprising observation that Mtb-infected mice are resistant to secondary infection with CoV2, with no impact on Mtb burden and resistance associating with lung T and B cell expansion.', TITLE='Mice infected with <i>Mycobacterium tuberculosis</i> are resistant to secondary infection with SARS-CoV-2', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418024', DOI='10.1101/2021.11.10.468025', YEAR='2021', ABSTRACT='The C30 Endopeptidase (3C-like protease; 3CL pro ) is essential for the life cycle of SARS-CoV-2 (severe acute respiratory syndrome-coronavirus-2) since it plays a pivotal role in viral replication and transcription and is hence a promising drug target. Molecules isolated from animals, insects, plants or microorganisms can serve as a scaffold for the design of novel biopharmaceutical products. Crotamine, a small cationic peptide from the venom of the rattlesnake Crotalus durissus terrificus has been the focus of many studies since it exhibits activities such as analgesic, in vitro antibacterial and hemolytic activities. The crotamine derivative L-peptides (L-CDP) that inhibit the 3CL protease in the low µM range were examined since they are susceptible to proteolytic degradation; we explored the utility of their D-enantiomers form. Comparative uptake inhibition analysis showed D-CDP as a promising prototype for a D-peptide-based drug. We also found that the D-peptides can impair SARS-CoV-2 replication in vivo , probably targeting the viral protease 3CL pro .', TITLE='Design of D-amino acids SARS-CoV-2 Main protease inhibitors using the cationic peptide from rattlesnake venom as a scaffold', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418038', DOI='10.1101/2021.11.08.467683', YEAR='2021', ABSTRACT='<h4>Background</h4> Tracking animal movement is important for understanding how animals interact with their (changing) environment, and crucial for predicting and explaining how animals are affected by anthropogenic effects. The Wadden Sea is a UNESCO World Heritage Site and a region of global importance for millions of small shorebirds. Due to climate change and anthropogenic activity, understanding and predicting movement and space-use in areas like the Wadden Sea is increasingly important. Monitoring and predicting animal movement, however, requires high-resolution tracking of many individuals. While high-resolution tracking has been made possible through GPS, trade-offs between tag weight and battery life limit its use to larger species. <h4>Methods</h4> Here, we introduce WATLAS (the Wadden Sea deployment of the ATLAS tracking system) capable of monitoring the movements of hundreds of (small) birds simultaneously in the Dutch Wadden Sea. WATLAS employs an array of receiver stations that can detect and localise small, low-cost tags at fine spatial (meters) and temporal resolution (seconds). From 2017-2021, we tracked red knots, sanderlings, bar-tailed godwits, and common terns. We use parts of these data to give four examples on its performance and how WATLAS can be used to study numerous aspects of animal behaviour, such as, space-use (both intra- and inter-specific), among-individual variation, and social networks. <h4>Results</h4> After describing the WATLAS system, we first illustrate space-use of red knots across the study area and how the tidal environment affects their movement. Secondly, we show large among-individual differences in distances travelled per day, and thirdly illustrate how high-throughput WATLAS data allows calculating a proximity-based social network. Finally, we demonstrate that using WATLAS to monitor multiple species can reveal differential space use. For example, despite sanderlings and red knots roosting together, they foraged in different areas of the mudflats. <h4>Conclusions</h4> The high-resolution tracking data collected by WATLAS offers many possibilities for research into the drivers of bird movement in the Wadden Sea. WATLAS could provide a tool for impact assessment, and thus aid nature conservation and management of the globally important Wadden Sea ecosystem.', TITLE='WATLAS: high throughput and real-time tracking of many small birds in the Dutch Wadden Sea', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418041', DOI='10.1101/2021.11.08.467694', YEAR='2021', ABSTRACT='The study of the brain’s static and dynamical activity is opening a valuable source of assistance for the clinical assessment of patients with disorders of consciousness. For example, glucose uptake and dysfunctional spread of naturalistic and synthetic stimuli has proven useful to characterize hampered consciousness. However, understanding of the mechanisms behind loss of consciousness following brain injury is still missing. Here, we study the propagation of endogenous and in-silico exogenous perturbations in patients with disorders of consciousness, based upon directed and causal interactions estimated from resting-state fMRI. We found that patients with disorders of consciousness suffer decreased capacity for neural propagation and responsiveness to events, and that this can be related to glucose metabolism as measured with [ 18 F]FDG-PET. In particular, we show that loss of consciousness is related to the malfunctioning of two neural circuits: the posterior cortical regions failing to convey information, in conjunction with reduced broadcasting of information from subcortical, temporal, parietal and frontal regions. These results seed light on the mechanisms behind disorders of consciousness, triangulating network function with basic measures of brain integrity and behavior. <h4>Highlights</h4> Propagation of neural events and network responses are disrupted in patients with DoC. Loss of consciousness is related to the malfunctioning of two neural circuits. Posterior cortical regions lack to integrate information in altered consciousness. Breakdown of information broadcasting of subcortical cortical areas in DoC. Loss of network responses in DoC patients is related to glucose metabolism.', TITLE='Posterior integration and thalamo-frontotemporal broadcasting are impaired in disorders of consciousness', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418047', DOI='10.1101/2021.11.08.467745', YEAR='2021', ABSTRACT='Theories of neural replay propose that it supports a range of functions, most prominently planning and memory consolidation. Here, we test the hypothesis that distinct signatures of replay in the same task are related to model-based decisionmaking (‘planning’) and memory preservation. We designed a reward learning task wherein participants utilized structure knowledge for model-based evaluation, while at the same time had to maintain knowledge of two independent and randomly alternating task environments. Using magnetoencephalography (MEG) and multivariate analysis, we first identified temporally compressed sequential reactivation, or replay, both prior to choice and following reward feedback. Before choice, prospective replay strength was enhanced for the current task-relevant environment when a model-based planning strategy was beneficial. Following reward receipt, and consistent with a memory preservation role, replay for the alternative distal task environment was enhanced as a function of decreasing recency of experience with that environment. Critically, these planning and memory preservation relationships were selective to pre-choice and post-feedback periods. Our results provide new support for key theoretical proposals regarding the functional role of replay and demonstrate that the relative strength of planning and memory-related signals are modulated by on-going computational and task demands. <h4>Significance statement</h4> The sequential neural reactivation of prior experience, known as replay, is considered to be an important mechanism for both future planning and preserving memories of the past. Whether, and how, replay supports both of these functions remains unknown. Here, in humans, we found that prior to a choice, rapid replay of potential future paths was enhanced when planning was more beneficial. By contrast, after choice feedback, when no future actions are imminent, we found evidence for a memory preservation signal evident in enhanced replay of paths that had been visited less in the recent past. The results demonstrate that distinct replay signatures, expressed at different times, relate to two dissociable cognitive functions.', TITLE='Distinct replay signatures for prospective decision-making and memory preservation', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418058', DOI='10.1101/2021.11.08.467670', YEAR='2021', ABSTRACT='Boundary extension (BE) is a classic memory illusion in which observers remember more of a scene than was presented. According to predictive processing accounts, BE reflects the integration of visual input and expectations of what is beyond a scene’s boundaries. According to normalization accounts, BE rather reflects one end of a normalization process towards a scene’s typically-experienced viewing distance, such that close-up views give BE but distant views give boundary contraction. Here across four experiments, we show that BE strongly depends on depth-of-field (DOF), as determined by the aperture settings on a camera. Photographs with naturalistic DOF led to larger BE than photographs with unnaturalistic DOF, even when showing distant views. We propose that BE reflects a predictive mechanism with adaptive value that is strongest for naturalistic views of scenes. The current findings indicate that DOF is an important variable to consider in the study of scene perception and memory. <h4>Statement of Relevance</h4> In daily life, we experience a rich and continuous visual world in spite of the capacity limits of the visual system. We may compensate for such limits with our memory, by filling-in the visual input with anticipatory representations of upcoming views. The boundary extension illusion (BE) provides a tool to investigate this phenomenon. For example, not all images equally lead to BE. In this set of studies, we show that memory extrapolation beyond scene boundaries is strongest for images resembling human visual experience, showing depth-of-field in the range of human vision. Based on these findings, we propose that predicting upcoming views is conditional to a scene being perceived as naturalistic. More generally, the strong reliance of a cognitive effect, such as BE, on naturalistic image properties indicates that it is imperative to use image sets that are ecologically-representative when studying the cognitive, computational, and neural mechanisms of scene processing.', TITLE='Predictive processing of scene layout depends on naturalistic depth of field', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418067', DOI='10.1101/2021.11.05.467537', YEAR='2021', ABSTRACT='The urgent approval of the use of the inactivated COVID-19 vaccine is essential to reduce the threat and burden of the epidemic on global public health, however, our current understanding of the host immune response to inactivated vaccine remains limited. Herein, we performed serum IgG antibody detection and transcriptomics analysis on 20 SARS-CoV-2 naïve individuals who received multiple doses of inactivated vaccine and 5 SARS-CoV-2 recovered individuals who received single dose of inactivated vaccine. Our research revealed the important role of many innate immune pathways after vaccination, identified a significant correlation with the third dose of booster vaccine and proteasome-related genes, and found that SARS-CoV-2 recovered individuals can produces a strong immune response to a single dose of inactivated vaccine. These results help us understand the reaction mechanism of the host’s molecular immune system to the inactivated vaccine, and provide a basis for the choice of vaccination strategy.', TITLE='Transcriptome analysis of SARS-CoV-2 naïve and recovered individuals vaccinated with inactivated vaccine', MESH_DESCRIPTORS='')\n",
      "Row(ID_PAPER='PPR418076', DOI='10.1101/2021.11.09.467911', YEAR='2021', ABSTRACT='SARS-CoV-2 has affected people worldwide as the causative agent of COVID-19. The virus is related to the highly lethal SARS-CoV responsible for the 2002-2003 SARS outbreak in Asia. Research is ongoing to understand why both viruses have different spreading capacities and mortality rates. Like other beta coronaviruses, RNA-RNA interactions occur between different parts of the viral genomic RNA, resulting in discontinuous transcription and production of various sub-genomic RNAs. These sub-genomic RNAs are then translated into other viral proteins. In this work, we performed a comparative analysis for novel long-range RNA-RNA interactions that may involve the Spike region. Comparing predictions between reference sequences of SARS-CoV-1 and SARS-CoV-2 revealed several predictions amongst which a thermodynamically stable long-range RNA-RNA interaction between (23660-23703 Spike ) and (28025-28060 ORF8 ) unique to SARS-CoV-2 was observed. Using data gathered worldwide, sequence variation patterns observed in the population support the in-silico RNA-RNA base-pairing predictions within these regions, suggesting further evidence for the interaction. The predicted interactions can potentially be related to the regulation of sub-genomic RNA production rates in SARS-CoV-2 and their subsequent accessibility to the host transcriptome.', TITLE='Evidence for a long-range RNA-RNA interaction between <i>ORF8</i> and <i>Spike</i> of SARS-CoV-2', MESH_DESCRIPTORS='')\n"
     ]
    }
   ],
   "source": [
    "from databricks import sql\n",
    "import os\n",
    "\n",
    "with sql.connect(server_hostname = \"czi-shared-infra-czi-sci-general-prod-databricks.cloud.databricks.com\",\n",
    "                 http_path       = \"/sql/1.0/warehouses/1c4df94f2f1a6305\",\n",
    "                 access_token    = os.getenv(\"DB_TOKEN\")) as connection:\n",
    "\n",
    "  with connection.cursor() as cursor:\n",
    "    cursor.execute(\"SELECT * FROM gburns.imaging_tech.papers WHERE LEN(ABSTRACT) > 0 LIMIT 10\")\n",
    "    result = cursor.fetchall()\n",
    "\n",
    "    for row in result:\n",
    "      print(row)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n",
      "DatabricksRetryPolicy is currently bypassed. The CommandType cannot be set.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>DOI</th>\n",
       "      <th>YEAR</th>\n",
       "      <th>ABSTRACT</th>\n",
       "      <th>TITLE</th>\n",
       "      <th>MESH_DESCRIPTORS</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PPR438151</td>\n",
       "      <td>10.1101/2022.01.02.474731</td>\n",
       "      <td>2022</td>\n",
       "      <td>Neuronal responses to similar stimuli change d...</td>\n",
       "      <td>Contribution of behavioural variability to rep...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PPR438262</td>\n",
       "      <td>10.1101/2022.01.01.474694</td>\n",
       "      <td>2022</td>\n",
       "      <td>Red blood cells (RBCs) express the nucleic aci...</td>\n",
       "      <td>Human red blood cells express the RNA sensor T...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PPR438779</td>\n",
       "      <td>10.1101/2021.12.30.474580</td>\n",
       "      <td>2022</td>\n",
       "      <td>Severe acute respiratory syndrome coronavirus ...</td>\n",
       "      <td>SARS-CoV-2 entry sites are present in all stru...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PPR438788</td>\n",
       "      <td>10.1101/2022.01.01.474713</td>\n",
       "      <td>2022</td>\n",
       "      <td>Thailand was the first country outside China t...</td>\n",
       "      <td>The SARS-CoV-2 infection in Thailand: analysis...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PPR438789</td>\n",
       "      <td>10.1101/2022.01.03.474773</td>\n",
       "      <td>2022</td>\n",
       "      <td>The SARS-CoV-2 Omicron variant is currently ca...</td>\n",
       "      <td>Reduced interferon antagonism but similar drug...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3939</th>\n",
       "      <td>PPR587527</td>\n",
       "      <td>10.1101/2022.12.19.519427</td>\n",
       "      <td>2022</td>\n",
       "      <td>Cohort studies increasingly collect biosamples...</td>\n",
       "      <td>Omada: Robust clustering of transcriptomes thr...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3940</th>\n",
       "      <td>PPR587989</td>\n",
       "      <td>10.1101/2022.12.21.520923</td>\n",
       "      <td>2022</td>\n",
       "      <td>The two clathrin isoforms, CHC17 and CHC22, ge...</td>\n",
       "      <td>CHC22 clathrin membrane recruitment uses SNX5 ...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3941</th>\n",
       "      <td>PPR588013</td>\n",
       "      <td>10.1101/2022.12.21.521417</td>\n",
       "      <td>2022</td>\n",
       "      <td>With the recent growth in spectral flow cytome...</td>\n",
       "      <td>Back to the Future- Unleashing your cytometer’...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3942</th>\n",
       "      <td>PPR588026</td>\n",
       "      <td>10.1101/2022.12.21.521384</td>\n",
       "      <td>2022</td>\n",
       "      <td>The shape of cells is the outcome of the balan...</td>\n",
       "      <td>Friction patterns guide actin network contraction</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3943</th>\n",
       "      <td>PPR588048</td>\n",
       "      <td>10.1101/2022.12.21.521283</td>\n",
       "      <td>2022</td>\n",
       "      <td>Inflammation is essential for responding to in...</td>\n",
       "      <td>Inflammatory Synovial Fibroblast Culture in 3D...</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3944 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       ID_PAPER                        DOI  YEAR  \\\n",
       "0     PPR438151  10.1101/2022.01.02.474731  2022   \n",
       "1     PPR438262  10.1101/2022.01.01.474694  2022   \n",
       "2     PPR438779  10.1101/2021.12.30.474580  2022   \n",
       "3     PPR438788  10.1101/2022.01.01.474713  2022   \n",
       "4     PPR438789  10.1101/2022.01.03.474773  2022   \n",
       "...         ...                        ...   ...   \n",
       "3939  PPR587527  10.1101/2022.12.19.519427  2022   \n",
       "3940  PPR587989  10.1101/2022.12.21.520923  2022   \n",
       "3941  PPR588013  10.1101/2022.12.21.521417  2022   \n",
       "3942  PPR588026  10.1101/2022.12.21.521384  2022   \n",
       "3943  PPR588048  10.1101/2022.12.21.521283  2022   \n",
       "\n",
       "                                               ABSTRACT  \\\n",
       "0     Neuronal responses to similar stimuli change d...   \n",
       "1     Red blood cells (RBCs) express the nucleic aci...   \n",
       "2     Severe acute respiratory syndrome coronavirus ...   \n",
       "3     Thailand was the first country outside China t...   \n",
       "4     The SARS-CoV-2 Omicron variant is currently ca...   \n",
       "...                                                 ...   \n",
       "3939  Cohort studies increasingly collect biosamples...   \n",
       "3940  The two clathrin isoforms, CHC17 and CHC22, ge...   \n",
       "3941  With the recent growth in spectral flow cytome...   \n",
       "3942  The shape of cells is the outcome of the balan...   \n",
       "3943  Inflammation is essential for responding to in...   \n",
       "\n",
       "                                                  TITLE MESH_DESCRIPTORS  \n",
       "0     Contribution of behavioural variability to rep...                   \n",
       "1     Human red blood cells express the RNA sensor T...                   \n",
       "2     SARS-CoV-2 entry sites are present in all stru...                   \n",
       "3     The SARS-CoV-2 infection in Thailand: analysis...                   \n",
       "4     Reduced interferon antagonism but similar drug...                   \n",
       "...                                                 ...              ...  \n",
       "3939  Omada: Robust clustering of transcriptomes thr...                   \n",
       "3940  CHC22 clathrin membrane recruitment uses SNX5 ...                   \n",
       "3941  Back to the Future- Unleashing your cytometer’...                   \n",
       "3942  Friction patterns guide actin network contraction                   \n",
       "3943  Inflammatory Synovial Fibroblast Culture in 3D...                   \n",
       "\n",
       "[3944 rows x 6 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "engine = create_engine(\n",
    "    'databricks://token:'+os.environ['DB_TOKEN']+ \n",
    "    '@czi-shared-infra-czi-sci-general-prod-databricks.cloud.databricks.com'+\n",
    "    '?http_path=/sql/1.0/warehouses/1c4df94f2f1a6305')\n",
    "\n",
    "sql = '''\n",
    "SELECT * FROM gburns.imaging_tech.papers \n",
    "WHERE LEN(ABSTRACT) > 0 and YEAR=2022\n",
    "'''\n",
    "with engine.connect() as con:\n",
    "    stmt = db.text(sql)\n",
    "    rs = con.execute(stmt)\n",
    "    df = pd.DataFrame(rs.fetchall(), columns=rs.keys())\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t2 = \"\"\"\n",
    "You are an expert biological scientist trained in imaging and microscopy. \n",
    "Your knowledge is deep and wide. \n",
    "You are terse and precise in your language.\n",
    "\n",
    "Your job is to read the title and abstract of a scientific paper in detail, understand what you read, and perform the task described below based on what you read. \n",
    "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
    "\n",
    "First, break down the text of the title and abstract into separate sentences.\n",
    "\n",
    "Second, identify which sentences describe \n",
    "(A) background information about the work, \n",
    "(B) the goals of the work being described, \n",
    "(C) the methods used to achieve the goals, \n",
    "(D) the results of the work, and \n",
    "(E) the conclusions of the work. \n",
    "\n",
    "Third, use only sentences you identify as goals, methods, and results to identify the main contribution of the paper. \n",
    "Record this contribution as a complete noun phrase shorter than 12 words or so as main_contribution in the output.\n",
    "\n",
    "Fourth, decide if that contribution of the paper is developing an imaging technique or method. Be careful about this decision and only answer 'true' \n",
    "if you are confident that the contribution of the paper states that a new imaging technique or method was developed. Possible examples of imaging techniques\n",
    "could be new types of microscope, new imaging techniques, hardware, optics, or sample preparation for imaging. \n",
    "\n",
    "If the main contribution is the development of a new imaging method, record a true value in a variable called is_imaging_method in the output. \n",
    "\n",
    "If the main contribution is not the development of a new imaging method (such as the development of a genetic test, or a finding about a biological system), record a false value in a variable called is_imaging_method in the output. \n",
    "\n",
    "Format your response as a JSON object with \"main_contribution\" and \"is_imaging_method\" as the keys. \n",
    "\n",
    "Do not include any other response other than the JSON object. Remember, stop generating text after you finish the JSON object. Do not provide additional explanations of your answer.\n",
    "\n",
    "Article\n",
    "title: '''{title}'''\n",
    "abstract: '''{abstract}'''\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t3 = \"\"\"\n",
    "Your job is to read the title and abstract of a scientific paper and perform the task described below. \n",
    "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
    "\n",
    "First, split the text of the title and abstract into separate sentences.\n",
    "\n",
    "Second, classify each sentence as describing \n",
    "- background information about the work, \n",
    "- the goals of the paper, \n",
    "- the methods used by the scientists, \n",
    "- the results of the work, and \n",
    "- the conclusions of the work. \n",
    "\n",
    "Third, using only the goal and method sentences, decide if the work is concerned with (A) developing new technology or methods or \n",
    "(B) investigating how biological systems work.  \n",
    "\n",
    "Be careful about this decision and only answer 'true' if you are confident that the main goal of the published work \n",
    "was to create or test a new technique or method.  \n",
    "\n",
    "If paper describes the development of a new method, technique or approach, record a true value in a variable called is_method_paper in the output. \n",
    "\n",
    "If the goal of the work understanding biological phenomena, record a false value in a variable called is_method_paper in the output.\n",
    "\n",
    "Format your response as a JSON object with \"is_method_paper\" as the key.  \n",
    "\n",
    "Briefly explain your reasoning for your decision in one or two sentences.\n",
    "\n",
    "Article\n",
    "title: '''{title}'''\n",
    "abstract: '''{abstract}'''\n",
    "\"\"\"\n",
    "p3 = PromptTemplate(\n",
    "    template=t3,\n",
    "    input_variables=[\"title\", \"abstract\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>DOI</th>\n",
       "      <th>ABSTRACT</th>\n",
       "      <th>TITLE</th>\n",
       "      <th>MESH_DESCRIPTORS</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>27749836</td>\n",
       "      <td>10.1038/nmeth.4033</td>\n",
       "      <td>Understanding how neural circuits process info...</td>\n",
       "      <td>Random-access scanning microscopy for 3D imagi...</td>\n",
       "      <td>Cerebellar Cortex | Pyramidal Cells | Visual C...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>27749837</td>\n",
       "      <td>10.1038/nmeth.4031</td>\n",
       "      <td>Spatial organization of the genome plays a cen...</td>\n",
       "      <td>ATAC-see reveals the accessible genome by tran...</td>\n",
       "      <td>Neutrophils | CD4-Positive T-Lymphocytes | Cel...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>27776112</td>\n",
       "      <td>10.1038/nmeth.4034</td>\n",
       "      <td>Small-molecule fluorophores are important tool...</td>\n",
       "      <td>Bright photoactivatable fluorophores for singl...</td>\n",
       "      <td>COS Cells | Cell Line, Tumor | Animals | Human...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>27798609</td>\n",
       "      <td>10.1038/nmeth.4046</td>\n",
       "      <td>We describe a red-shifted fluorescence resonan...</td>\n",
       "      <td>Simultaneous dual-color fluorescence lifetime ...</td>\n",
       "      <td>Hela Cells | Endoplasmic Reticulum | Animals |...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>27798610</td>\n",
       "      <td>10.1038/nmeth.4045</td>\n",
       "      <td>A robust method for simultaneous visualization...</td>\n",
       "      <td>Fluorescent indicators for simultaneous report...</td>\n",
       "      <td>Hela Cells | NIH 3T3 Cells | Chromatin | Anima...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1121</th>\n",
       "      <td>27479330</td>\n",
       "      <td>10.1038/nmeth.3955</td>\n",
       "      <td>We have developed hydrogel-based virtual micro...</td>\n",
       "      <td>Virtual microfluidics for digital quantificati...</td>\n",
       "      <td>Escherichia coli | Staphylococcus aureus | Hyd...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1122</th>\n",
       "      <td>27548807</td>\n",
       "      <td>10.1038/nmeth.3964</td>\n",
       "      <td>Recent tissue-clearing approaches have become ...</td>\n",
       "      <td>Shrinkage-mediated imaging of entire organs an...</td>\n",
       "      <td>Central Nervous System | Animals | Mice, Inbre...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1123</th>\n",
       "      <td>27595403</td>\n",
       "      <td>10.1038/nmeth.3972</td>\n",
       "      <td>We present tRNA-based vectors for producing mu...</td>\n",
       "      <td>Augmenting CRISPR applications in Drosophila w...</td>\n",
       "      <td>Animals | Animals, Genetically Modified | Dros...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1124</th>\n",
       "      <td>27595405</td>\n",
       "      <td>10.1038/nmeth.3993</td>\n",
       "      <td>CRISPR-Cas9 delivery by adeno-associated virus...</td>\n",
       "      <td>A multifunctional AAV-CRISPR-Cas9 and its host...</td>\n",
       "      <td>Animals | Mice, Inbred C57BL | Humans | Mice |...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1125</th>\n",
       "      <td>27669326</td>\n",
       "      <td>10.1038/nmeth.4004</td>\n",
       "      <td>Expressed protein ligation is a valuable metho...</td>\n",
       "      <td>Enzyme-catalyzed expressed protein ligation.</td>\n",
       "      <td>Cells, Cultured | Fibroblasts | Animals | Mice...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1126 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      ID_PAPER                 DOI  \\\n",
       "0     27749836  10.1038/nmeth.4033   \n",
       "1     27749837  10.1038/nmeth.4031   \n",
       "2     27776112  10.1038/nmeth.4034   \n",
       "3     27798609  10.1038/nmeth.4046   \n",
       "4     27798610  10.1038/nmeth.4045   \n",
       "...        ...                 ...   \n",
       "1121  27479330  10.1038/nmeth.3955   \n",
       "1122  27548807  10.1038/nmeth.3964   \n",
       "1123  27595403  10.1038/nmeth.3972   \n",
       "1124  27595405  10.1038/nmeth.3993   \n",
       "1125  27669326  10.1038/nmeth.4004   \n",
       "\n",
       "                                               ABSTRACT  \\\n",
       "0     Understanding how neural circuits process info...   \n",
       "1     Spatial organization of the genome plays a cen...   \n",
       "2     Small-molecule fluorophores are important tool...   \n",
       "3     We describe a red-shifted fluorescence resonan...   \n",
       "4     A robust method for simultaneous visualization...   \n",
       "...                                                 ...   \n",
       "1121  We have developed hydrogel-based virtual micro...   \n",
       "1122  Recent tissue-clearing approaches have become ...   \n",
       "1123  We present tRNA-based vectors for producing mu...   \n",
       "1124  CRISPR-Cas9 delivery by adeno-associated virus...   \n",
       "1125  Expressed protein ligation is a valuable metho...   \n",
       "\n",
       "                                                  TITLE  \\\n",
       "0     Random-access scanning microscopy for 3D imagi...   \n",
       "1     ATAC-see reveals the accessible genome by tran...   \n",
       "2     Bright photoactivatable fluorophores for singl...   \n",
       "3     Simultaneous dual-color fluorescence lifetime ...   \n",
       "4     Fluorescent indicators for simultaneous report...   \n",
       "...                                                 ...   \n",
       "1121  Virtual microfluidics for digital quantificati...   \n",
       "1122  Shrinkage-mediated imaging of entire organs an...   \n",
       "1123  Augmenting CRISPR applications in Drosophila w...   \n",
       "1124  A multifunctional AAV-CRISPR-Cas9 and its host...   \n",
       "1125       Enzyme-catalyzed expressed protein ligation.   \n",
       "\n",
       "                                       MESH_DESCRIPTORS  \n",
       "0     Cerebellar Cortex | Pyramidal Cells | Visual C...  \n",
       "1     Neutrophils | CD4-Positive T-Lymphocytes | Cel...  \n",
       "2     COS Cells | Cell Line, Tumor | Animals | Human...  \n",
       "3     Hela Cells | Endoplasmic Reticulum | Animals |...  \n",
       "4     Hela Cells | NIH 3T3 Cells | Chromatin | Anima...  \n",
       "...                                                 ...  \n",
       "1121  Escherichia coli | Staphylococcus aureus | Hyd...  \n",
       "1122  Central Nervous System | Animals | Mice, Inbre...  \n",
       "1123  Animals | Animals, Genetically Modified | Dros...  \n",
       "1124  Animals | Mice, Inbred C57BL | Humans | Mice |...  \n",
       "1125  Cells, Cultured | Fibroblasts | Animals | Mice...  \n",
       "\n",
       "[1126 rows x 5 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "LlamaCpp(cache=None, verbose=True, callbacks=<langchain.callbacks.manager.CallbackManager object>, tags=None, metadata=None, client=<llama_cpp.llama.Llama object>, model_path='/Users/gburns/Documents/LLms/llama-2-7b-chat.Q5_K_M.gguf', lora_base=None, lora_path=None, n_ctx=512, n_parts=-1, seed=-1, f16_kv=True, logits_all=False, vocab_only=False, use_mlock=False, n_threads=None, n_batch=4096, n_gpu_layers=40, suffix=None, max_tokens=256, temperature=0.8, top_p=0.95, logprobs=None, echo=False, stop=[], repeat_penalty=1.1, top_k=40, last_n_tokens_size=64, use_mmap=True, rope_freq_scale=1.0, rope_freq_base=10000.0, model_kwargs={}, streaming=True, grammar_path=None, grammar=None)"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "llm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "-----------------------------------------------------------------\n",
      "\n",
      "0\n",
      "PPR590719\n",
      "Lung lipid deposition in pneumonias of viral and non-viral aetiology\n",
      "Pneumonia is an acute respiratory disease of varying aetiology, which drew much attention during the\n",
      "COVID-19 pandemic. Among many thoroughly studied aspects of pneumonia, lipid metabolism has been\n",
      "addressed insufficiently. Here, we report on abnormal lipid metabolism of both COVID-19- and non-\n",
      "COVID-19-associated pneumonias in human lungs. Morphometric analysis revealed extracellular and\n",
      "intracellular lipid depositions, most notably within vessels adjacent to inflamed regions, where\n",
      "they apparently interfere with the blood flow. Lipids were visualized on Sudan III- and Oil Red\n",
      "O-stained cryosections and on OsO 4 -contrasted semi-thin and ultrathin sections. Chromato-mass\n",
      "spectrometry revealed that unsaturated fatty acid content was elevated at inflammation sites\n",
      "compared with the intact sites of the same lung. The genes involved in lipid metabolism were\n",
      "downregulated in pneumonia, as shown by qPCR and in silico RNAseq analysis. Thus, pneumonias are\n",
      "associated with marked lipid abnormalities, and therefore lipid metabolism can be considered a\n",
      "target for new therapeutic strategies.\n",
      "Please do this task. \n",
      "\n",
      "Thank you."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "llama_print_timings:        load time = 88410.89 ms\n",
      "llama_print_timings:      sample time =     8.68 ms /    12 runs   (    0.72 ms per token,  1382.01 tokens per second)\n",
      "llama_print_timings: prompt eval time = 88410.80 ms /   629 tokens (  140.56 ms per token,     7.11 tokens per second)\n",
      "llama_print_timings:        eval time =  1971.93 ms /    11 runs   (  179.27 ms per token,     5.58 tokens per second)\n",
      "llama_print_timings:       total time = 90409.99 ms\n",
      "ggml_metal_free: deallocating\n"
     ]
    },
    {
     "ename": "OutputParserException",
     "evalue": "Could not find json-formatted data in: Please do this task. \n\nThank you.",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mOutputParserException\u001b[0m                     Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[28], line 35\u001b[0m\n\u001b[1;32m     32\u001b[0m     df2 \u001b[39m=\u001b[39m pd\u001b[39m.\u001b[39mDataFrame(l)\n\u001b[1;32m     33\u001b[0m     \u001b[39mreturn\u001b[39;00m df2\n\u001b[0;32m---> 35\u001b[0m df_2022_2 \u001b[39m=\u001b[39m run_df(df_2022[\u001b[39m0\u001b[39;49m:\u001b[39m2\u001b[39;49m], llm, p3)\n\u001b[1;32m     36\u001b[0m \u001b[39m#df_nat_meth_0_100 = run_df(df_nat_meth[0:100], llm, p3)\u001b[39;00m\n",
      "Cell \u001b[0;32mIn[28], line 20\u001b[0m, in \u001b[0;36mrun_df\u001b[0;34m(df, llm, p)\u001b[0m\n\u001b[1;32m     18\u001b[0m t \u001b[39m=\u001b[39m time\u001b[39m.\u001b[39mtime()\n\u001b[1;32m     19\u001b[0m output \u001b[39m=\u001b[39m execute_llm(llm, p, r\u001b[39m.\u001b[39mID_PAPER, r\u001b[39m.\u001b[39mTITLE, r\u001b[39m.\u001b[39mABSTRACT)\n\u001b[0;32m---> 20\u001b[0m m \u001b[39m=\u001b[39m output_parser\u001b[39m.\u001b[39;49mparse(output)\n\u001b[1;32m     21\u001b[0m \u001b[39mif\u001b[39;00m m \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m: \n\u001b[1;32m     22\u001b[0m     m \u001b[39m=\u001b[39m {\n\u001b[1;32m     23\u001b[0m         \u001b[39m'\u001b[39m\u001b[39mmain_contribution\u001b[39m\u001b[39m'\u001b[39m: \u001b[39mNone\u001b[39;00m,\n\u001b[1;32m     24\u001b[0m         \u001b[39m'\u001b[39m\u001b[39mis_imaging_method\u001b[39m\u001b[39m'\u001b[39m: \u001b[39mFalse\u001b[39;00m,\n\u001b[1;32m     25\u001b[0m     } \n",
      "Cell \u001b[0;32mIn[11], line 27\u001b[0m, in \u001b[0;36mJsonEnclosedByTextOutputParser.parse\u001b[0;34m(self, text)\u001b[0m\n\u001b[1;32m     24\u001b[0m         \u001b[39mreturn\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\u001b[1;32m     25\u001b[0m         \u001b[39m#raise OutputParserException(f\"Invalid json output: {text1} derived from {text}\") from e\u001b[39;00m\n\u001b[1;32m     26\u001b[0m \u001b[39melse\u001b[39;00m: \n\u001b[0;32m---> 27\u001b[0m     \u001b[39mraise\u001b[39;00m OutputParserException(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mCould not find json-formatted data in: \u001b[39m\u001b[39m{\u001b[39;00mtext\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m)\n",
      "\u001b[0;31mOutputParserException\u001b[0m: Could not find json-formatted data in: Please do this task. \n\nThank you."
     ]
    }
   ],
   "source": [
    "import time\n",
    "\n",
    "def execute_llm(llm, p, i, t, a):\n",
    "    print(textwrap.fill(i, 100))\n",
    "    print(textwrap.fill(t, 100))\n",
    "    print(textwrap.fill(a, 100))\n",
    "    output = llm(p.format(title=t, abstract=a))\n",
    "    return output\n",
    "\n",
    "def run_df(df, llm, p):\n",
    "    l = []\n",
    "    for i, r in df.iterrows():\n",
    "        if r.ABSTRACT is None: \n",
    "            continue\n",
    "        print('\\n-----------------------------------------------------------------\\n')\n",
    "        print(i)\n",
    "        # measure time to execute\n",
    "        t = time.time()\n",
    "        output = execute_llm(llm, p, r.ID_PAPER, r.TITLE, r.ABSTRACT)\n",
    "        m = output_parser.parse(output)\n",
    "        if m is None: \n",
    "            m = {\n",
    "                'main_contribution': None,\n",
    "                'is_imaging_method': False,\n",
    "            } \n",
    "        m['llm_output'] = output\n",
    "        m['ID_PAPER'] = r.ID_PAPER\n",
    "        dt = time.time() - t    \n",
    "        m['time'] = dt\n",
    "        l.append(m)\n",
    "\n",
    "    df2 = pd.DataFrame(l)\n",
    "    return df2\n",
    "    \n",
    "df_2022_2 = run_df(df_2022[0:2], llm, p3)\n",
    "#df_nat_meth_0_100 = run_df(df_nat_meth[0:100], llm, p3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>DOI</th>\n",
       "      <th>YEAR</th>\n",
       "      <th>ABSTRACT</th>\n",
       "      <th>TITLE</th>\n",
       "      <th>MESH_DESCRIPTORS</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>34811556</td>\n",
       "      <td>10.1038/s41592-021-01316-y</td>\n",
       "      <td>2022</td>\n",
       "      <td>Tissues and organs are composed of distinct ce...</td>\n",
       "      <td>Spatial mapping of protein composition and tis...</td>\n",
       "      <td>Antibodies | Diagnostic Imaging | Cell Communi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>34824477</td>\n",
       "      <td>10.1038/s41592-021-01308-y</td>\n",
       "      <td>2022</td>\n",
       "      <td>Highly multiplexed tissue imaging makes detail...</td>\n",
       "      <td>MCMICRO: a scalable, modular image-processing ...</td>\n",
       "      <td>Humans | Neoplasms | Diagnostic Imaging | Imag...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>34887550</td>\n",
       "      <td>10.1038/s41592-021-01335-9</td>\n",
       "      <td>2022</td>\n",
       "      <td>Mako is a software tool that converts microbio...</td>\n",
       "      <td>Fast and flexible analysis of linked microbiom...</td>\n",
       "      <td>Animals | Computational Biology | Computer Gra...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>34887551</td>\n",
       "      <td>10.1038/s41592-021-01334-w</td>\n",
       "      <td>2022</td>\n",
       "      <td>Recent whole-brain mapping projects are collec...</td>\n",
       "      <td>Cross-modal coherent registration of whole mou...</td>\n",
       "      <td>Brain | Animals | Mice, Inbred C57BL | Imaging...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>34916672</td>\n",
       "      <td>10.1038/s41592-021-01341-x</td>\n",
       "      <td>2022</td>\n",
       "      <td>Self-labeling protein tags such as HaloTag are...</td>\n",
       "      <td>Engineered HaloTag variants for fluorescence l...</td>\n",
       "      <td>Cell Line | Humans | Rhodamines | Hydrolases |...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>36396787</td>\n",
       "      <td>10.1038/s41592-022-01669-y</td>\n",
       "      <td>2022</td>\n",
       "      <td>Fluorescent in-situ hybridization (FISH)-based...</td>\n",
       "      <td>RS-FISH: precise, interactive, fast, and scala...</td>\n",
       "      <td>Microscopy | In Situ Hybridization, Fluorescen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>36424441</td>\n",
       "      <td>10.1038/s41592-022-01673-2</td>\n",
       "      <td>2022</td>\n",
       "      <td>Tissue function depends on cellular organizati...</td>\n",
       "      <td>Image-seq: spatially resolved single-cell sequ...</td>\n",
       "      <td>Humans | Leukemia | Diagnostic Imaging | Cell ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>36443486</td>\n",
       "      <td>10.1038/s41592-022-01675-0</td>\n",
       "      <td>2022</td>\n",
       "      <td>Achieving state-of-the-art performance with de...</td>\n",
       "      <td>A large-scale neural network training framewor...</td>\n",
       "      <td>Motor Cortex | Somatosensory Cortex | Animals ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>36456784</td>\n",
       "      <td>10.1038/s41592-022-01684-z</td>\n",
       "      <td>2022</td>\n",
       "      <td>We present proximity sequencing (Prox-seq) for...</td>\n",
       "      <td>Quantification of extracellular proteins, prot...</td>\n",
       "      <td>Leukocytes, Mononuclear | CD8-Positive T-Lymph...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>36456786</td>\n",
       "      <td>10.1038/s41592-022-01681-2</td>\n",
       "      <td>2022</td>\n",
       "      <td>Reference anatomies of the brain ('templates')...</td>\n",
       "      <td>TemplateFlow: FAIR-sharing of multi-scale, mul...</td>\n",
       "      <td>Brain | Problem Solving | Databases, Factual |...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>140 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     ID_PAPER                         DOI  YEAR  \\\n",
       "0    34811556  10.1038/s41592-021-01316-y  2022   \n",
       "1    34824477  10.1038/s41592-021-01308-y  2022   \n",
       "2    34887550  10.1038/s41592-021-01335-9  2022   \n",
       "3    34887551  10.1038/s41592-021-01334-w  2022   \n",
       "4    34916672  10.1038/s41592-021-01341-x  2022   \n",
       "..        ...                         ...   ...   \n",
       "135  36396787  10.1038/s41592-022-01669-y  2022   \n",
       "136  36424441  10.1038/s41592-022-01673-2  2022   \n",
       "137  36443486  10.1038/s41592-022-01675-0  2022   \n",
       "138  36456784  10.1038/s41592-022-01684-z  2022   \n",
       "139  36456786  10.1038/s41592-022-01681-2  2022   \n",
       "\n",
       "                                              ABSTRACT  \\\n",
       "0    Tissues and organs are composed of distinct ce...   \n",
       "1    Highly multiplexed tissue imaging makes detail...   \n",
       "2    Mako is a software tool that converts microbio...   \n",
       "3    Recent whole-brain mapping projects are collec...   \n",
       "4    Self-labeling protein tags such as HaloTag are...   \n",
       "..                                                 ...   \n",
       "135  Fluorescent in-situ hybridization (FISH)-based...   \n",
       "136  Tissue function depends on cellular organizati...   \n",
       "137  Achieving state-of-the-art performance with de...   \n",
       "138  We present proximity sequencing (Prox-seq) for...   \n",
       "139  Reference anatomies of the brain ('templates')...   \n",
       "\n",
       "                                                 TITLE  \\\n",
       "0    Spatial mapping of protein composition and tis...   \n",
       "1    MCMICRO: a scalable, modular image-processing ...   \n",
       "2    Fast and flexible analysis of linked microbiom...   \n",
       "3    Cross-modal coherent registration of whole mou...   \n",
       "4    Engineered HaloTag variants for fluorescence l...   \n",
       "..                                                 ...   \n",
       "135  RS-FISH: precise, interactive, fast, and scala...   \n",
       "136  Image-seq: spatially resolved single-cell sequ...   \n",
       "137  A large-scale neural network training framewor...   \n",
       "138  Quantification of extracellular proteins, prot...   \n",
       "139  TemplateFlow: FAIR-sharing of multi-scale, mul...   \n",
       "\n",
       "                                      MESH_DESCRIPTORS  \n",
       "0    Antibodies | Diagnostic Imaging | Cell Communi...  \n",
       "1    Humans | Neoplasms | Diagnostic Imaging | Imag...  \n",
       "2    Animals | Computational Biology | Computer Gra...  \n",
       "3    Brain | Animals | Mice, Inbred C57BL | Imaging...  \n",
       "4    Cell Line | Humans | Rhodamines | Hydrolases |...  \n",
       "..                                                 ...  \n",
       "135  Microscopy | In Situ Hybridization, Fluorescen...  \n",
       "136  Humans | Leukemia | Diagnostic Imaging | Cell ...  \n",
       "137  Motor Cortex | Somatosensory Cortex | Animals ...  \n",
       "138  Leukocytes, Mononuclear | CD8-Positive T-Lymph...  \n",
       "139  Brain | Problem Solving | Databases, Factual |...  \n",
       "\n",
       "[140 rows x 6 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_nat_meth"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_nat_met_out = df1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>is_method_paper</th>\n",
       "      <th>llm_output</th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>time</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>34811556</td>\n",
       "      <td>66.178736</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title in...</td>\n",
       "      <td>34824477</td>\n",
       "      <td>54.061088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title su...</td>\n",
       "      <td>34887550</td>\n",
       "      <td>56.417907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title su...</td>\n",
       "      <td>34887551</td>\n",
       "      <td>67.477511</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>34916672</td>\n",
       "      <td>60.608709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title in...</td>\n",
       "      <td>36064772</td>\n",
       "      <td>69.074156</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe title a...</td>\n",
       "      <td>36064773</td>\n",
       "      <td>70.971109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>36064775</td>\n",
       "      <td>77.545475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>36068320</td>\n",
       "      <td>65.244119</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>36076037</td>\n",
       "      <td>71.304745</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    is_method_paper                                         llm_output  \\\n",
       "0              True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "1              True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title in...   \n",
       "2              True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title su...   \n",
       "3              True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title su...   \n",
       "4              True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "..              ...                                                ...   \n",
       "95             True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title in...   \n",
       "96            False   {\\n\"is_method_paper\": false\\n}\\n\\nThe title a...   \n",
       "97             True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "98             True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "99             True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "\n",
       "    ID_PAPER       time  \n",
       "0   34811556  66.178736  \n",
       "1   34824477  54.061088  \n",
       "2   34887550  56.417907  \n",
       "3   34887551  67.477511  \n",
       "4   34916672  60.608709  \n",
       "..       ...        ...  \n",
       "95  36064772  69.074156  \n",
       "96  36064773  70.971109  \n",
       "97  36064775  77.545475  \n",
       "98  36068320  65.244119  \n",
       "99  36076037  71.304745  \n",
       "\n",
       "[100 rows x 4 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_nat_meth_0_100"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>is_method_paper</th>\n",
       "      <th>llm_output</th>\n",
       "      <th>ID_PAPER</th>\n",
       "      <th>time</th>\n",
       "      <th>main_contribution</th>\n",
       "      <th>is_imaging_method</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nExplanation...</td>\n",
       "      <td>PPR512857</td>\n",
       "      <td>137.945491</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe article...</td>\n",
       "      <td>PPR512860</td>\n",
       "      <td>98.742057</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>PPR512861</td>\n",
       "      <td>145.949785</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe title a...</td>\n",
       "      <td>PPR512885</td>\n",
       "      <td>89.226944</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe title a...</td>\n",
       "      <td>PPR512913</td>\n",
       "      <td>118.418904</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>PPR517227</td>\n",
       "      <td>80.874245</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe article...</td>\n",
       "      <td>PPR517231</td>\n",
       "      <td>115.259636</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe title a...</td>\n",
       "      <td>PPR517235</td>\n",
       "      <td>145.999283</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>False</td>\n",
       "      <td>{\\n\"is_method_paper\": false\\n}\\n\\nThe title a...</td>\n",
       "      <td>PPR517237</td>\n",
       "      <td>94.295474</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>True</td>\n",
       "      <td>{\\n\"is_method_paper\": true\\n}\\n\\nThe title an...</td>\n",
       "      <td>PPR517244</td>\n",
       "      <td>74.263383</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   is_method_paper                                         llm_output  \\\n",
       "0            False   {\\n\"is_method_paper\": false\\n}\\n\\nExplanation...   \n",
       "1            False   {\\n\"is_method_paper\": false\\n}\\n\\nThe article...   \n",
       "2             True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "3            False   {\\n\"is_method_paper\": false\\n}\\n\\nThe title a...   \n",
       "4            False   {\\n\"is_method_paper\": false\\n}\\n\\nThe title a...   \n",
       "..             ...                                                ...   \n",
       "95            True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "96           False   {\\n\"is_method_paper\": false\\n}\\n\\nThe article...   \n",
       "97           False   {\\n\"is_method_paper\": false\\n}\\n\\nThe title a...   \n",
       "98           False   {\\n\"is_method_paper\": false\\n}\\n\\nThe title a...   \n",
       "99            True   {\\n\"is_method_paper\": true\\n}\\n\\nThe title an...   \n",
       "\n",
       "     ID_PAPER        time  main_contribution is_imaging_method  \n",
       "0   PPR512857  137.945491                NaN               NaN  \n",
       "1   PPR512860   98.742057                NaN               NaN  \n",
       "2   PPR512861  145.949785                NaN               NaN  \n",
       "3   PPR512885   89.226944                NaN               NaN  \n",
       "4   PPR512913  118.418904                NaN               NaN  \n",
       "..        ...         ...                ...               ...  \n",
       "95  PPR517227   80.874245                NaN               NaN  \n",
       "96  PPR517231  115.259636                NaN               NaN  \n",
       "97  PPR517235  145.999283                NaN               NaN  \n",
       "98  PPR517237   94.295474                NaN               NaN  \n",
       "99  PPR517244   74.263383                NaN               NaN  \n",
       "\n",
       "[100 rows x 6 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_biorxiv_0_100"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t3 = \"\"\"[INST]\n",
    "<<SYS>>\n",
    "Your job is to read the title and abstract of a scientific paper and perform the task described below. \n",
    "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
    "<</SYS>>\n",
    "\n",
    "First, split the text of the title and abstract into separate sentences.\n",
    "\n",
    "Second, classify each sentence as describing \n",
    "- background information about the work, \n",
    "- the goals of the paper, \n",
    "- the methods used by the scientists, \n",
    "- the results of the work, and \n",
    "- the conclusions of the work. \n",
    "\n",
    "Third, using only the goal and method sentences, decide if the work is concerned with (A) developing new technology or methods or \n",
    "(B) investigating how biological systems work.  \n",
    "\n",
    "Be careful about this decision and only answer 'true' if you are confident that the main goal of the published work \n",
    "was to create or test a new technique or method.  \n",
    "\n",
    "If paper describes the development of a new method, technique or approach, record a true value in a variable called is_method_paper in the output. \n",
    "\n",
    "If the goal of the work understanding biological phenomena, record a false value in a variable called is_method_paper in the output.\n",
    "\n",
    "Format your response as a JSON object with \"is_method_paper\" as the key.  \n",
    "\n",
    "Briefly explain your reasoning for your decision in one or two sentences.\n",
    "Article\n",
    "title: '''{title}'''\n",
    "abstract: '''{abstract}'''\n",
    "[INST]\n",
    "\"\"\"\n",
    "p3 = PromptTemplate(\n",
    "    template=t3,\n",
    "    input_variables=[\"title\", \"abstract\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "llama_model_loader: loaded meta data with 19 key-value pairs and 723 tensors from /Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf (version GGUF V2 (latest))\n",
      "llama_model_loader: - tensor    0:                token_embd.weight q5_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - tensor    1:           blk.0.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    2:            blk.0.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    3:            blk.0.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    4:              blk.0.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    5:            blk.0.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    6:              blk.0.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor    7:         blk.0.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    8:              blk.0.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    9:              blk.0.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   10:           blk.1.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   11:            blk.1.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   12:            blk.1.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   13:              blk.1.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   14:            blk.1.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   15:              blk.1.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   16:         blk.1.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   17:              blk.1.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   18:              blk.1.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   19:           blk.2.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   20:            blk.2.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   21:            blk.2.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   22:              blk.2.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   23:            blk.2.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   24:              blk.2.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   25:         blk.2.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   26:              blk.2.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   27:              blk.2.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   28:           blk.3.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   29:            blk.3.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   30:            blk.3.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   31:              blk.3.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   32:            blk.3.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   33:              blk.3.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   34:         blk.3.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   35:              blk.3.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   36:              blk.3.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   37:           blk.4.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   38:            blk.4.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   39:            blk.4.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   40:              blk.4.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   41:            blk.4.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   42:              blk.4.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   43:         blk.4.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   44:              blk.4.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   45:              blk.4.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   46:            blk.5.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   47:              blk.5.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   48:         blk.5.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   49:              blk.5.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   50:              blk.5.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   51:          blk.10.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   52:           blk.10.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   53:           blk.10.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   54:             blk.10.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   55:           blk.10.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   56:             blk.10.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   57:        blk.10.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   58:             blk.10.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   59:             blk.10.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   60:             blk.11.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   61:        blk.11.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   62:             blk.11.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   63:             blk.11.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   64:           blk.5.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   65:            blk.5.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   66:              blk.5.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   67:            blk.5.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   68:           blk.6.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   69:            blk.6.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   70:            blk.6.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   71:              blk.6.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   72:            blk.6.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   73:              blk.6.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   74:         blk.6.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   75:              blk.6.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   76:              blk.6.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   77:           blk.7.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   78:            blk.7.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   79:            blk.7.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   80:              blk.7.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   81:            blk.7.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   82:              blk.7.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   83:         blk.7.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   84:              blk.7.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   85:              blk.7.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   86:           blk.8.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   87:            blk.8.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   88:            blk.8.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   89:              blk.8.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   90:            blk.8.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   91:              blk.8.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   92:         blk.8.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   93:              blk.8.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   94:              blk.8.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   95:           blk.9.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   96:            blk.9.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   97:            blk.9.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   98:              blk.9.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   99:            blk.9.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  100:              blk.9.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  101:         blk.9.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  102:              blk.9.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  103:              blk.9.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  104:          blk.11.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  105:           blk.11.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  106:           blk.11.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  107:             blk.11.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  108:           blk.11.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  109:          blk.12.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  110:           blk.12.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  111:           blk.12.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  112:             blk.12.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  113:           blk.12.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  114:             blk.12.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  115:        blk.12.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  116:             blk.12.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  117:             blk.12.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  118:          blk.13.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  119:           blk.13.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  120:           blk.13.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  121:             blk.13.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  122:           blk.13.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  123:             blk.13.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  124:        blk.13.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  125:             blk.13.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  126:             blk.13.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  127:          blk.14.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  128:           blk.14.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  129:           blk.14.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  130:             blk.14.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  131:           blk.14.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  132:             blk.14.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  133:        blk.14.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  134:             blk.14.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  135:             blk.14.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  136:          blk.15.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  137:           blk.15.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  138:           blk.15.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  139:             blk.15.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  140:           blk.15.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  141:             blk.15.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  142:        blk.15.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  143:             blk.15.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  144:             blk.15.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  145:          blk.16.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  146:           blk.16.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  147:           blk.16.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  148:             blk.16.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  149:           blk.16.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  150:             blk.16.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  151:        blk.16.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  152:             blk.16.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  153:             blk.16.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  154:          blk.17.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  155:           blk.17.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  156:           blk.17.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  157:             blk.17.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  158:           blk.17.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  159:             blk.17.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  160:        blk.17.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  161:             blk.17.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  162:             blk.17.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  163:          blk.18.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  164:           blk.18.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  165:           blk.18.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  166:             blk.18.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  167:           blk.18.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  168:             blk.18.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  169:        blk.18.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  170:             blk.18.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  171:             blk.18.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  172:          blk.19.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  173:           blk.19.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  174:           blk.19.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  175:             blk.19.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  176:           blk.19.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  177:             blk.19.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  178:        blk.19.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  179:             blk.19.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  180:             blk.19.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  181:          blk.20.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  182:           blk.20.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  183:           blk.20.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  184:             blk.20.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  185:           blk.20.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  186:             blk.20.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  187:        blk.20.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  188:             blk.20.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  189:             blk.20.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  190:          blk.21.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  191:           blk.21.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  192:           blk.21.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  193:             blk.21.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  194:           blk.21.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  195:             blk.21.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  196:        blk.21.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  197:             blk.21.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  198:             blk.21.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  199:           blk.22.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  200:             blk.22.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  201:             blk.22.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  202:        blk.22.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  203:             blk.22.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  204:             blk.22.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  205:          blk.22.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  206:           blk.22.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  207:           blk.22.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  208:          blk.23.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  209:           blk.23.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  210:           blk.23.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  211:             blk.23.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  212:           blk.23.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  213:             blk.23.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  214:        blk.23.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  215:             blk.23.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  216:             blk.23.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  217:          blk.24.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  218:           blk.24.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  219:           blk.24.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  220:             blk.24.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  221:           blk.24.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  222:             blk.24.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  223:        blk.24.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  224:             blk.24.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  225:             blk.24.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  226:          blk.25.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  227:           blk.25.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  228:           blk.25.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  229:             blk.25.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  230:           blk.25.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  231:             blk.25.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  232:        blk.25.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  233:             blk.25.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  234:             blk.25.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  235:          blk.26.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  236:           blk.26.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  237:           blk.26.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  238:             blk.26.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  239:           blk.26.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  240:             blk.26.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  241:        blk.26.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  242:             blk.26.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  243:             blk.26.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  244:          blk.27.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  245:           blk.27.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  246:           blk.27.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  247:             blk.27.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  248:           blk.27.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  249:             blk.27.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  250:        blk.27.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  251:             blk.27.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  252:             blk.27.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  253:           blk.28.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  254:             blk.28.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  255:        blk.28.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  256:             blk.28.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  257:             blk.28.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  258:          blk.28.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  259:           blk.28.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  260:             blk.28.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  261:           blk.28.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  262:          blk.29.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  263:           blk.29.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  264:           blk.29.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  265:             blk.29.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  266:           blk.29.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  267:             blk.29.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  268:        blk.29.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  269:             blk.29.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  270:             blk.29.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  271:          blk.30.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  272:           blk.30.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  273:           blk.30.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  274:             blk.30.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  275:           blk.30.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  276:             blk.30.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  277:        blk.30.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  278:             blk.30.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  279:             blk.30.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  280:          blk.31.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  281:           blk.31.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  282:           blk.31.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  283:             blk.31.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  284:           blk.31.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  285:             blk.31.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  286:        blk.31.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  287:             blk.31.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  288:             blk.31.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  289:          blk.32.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  290:           blk.32.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  291:           blk.32.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  292:             blk.32.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  293:           blk.32.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  294:             blk.32.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  295:        blk.32.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  296:             blk.32.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  297:             blk.32.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  298:          blk.33.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  299:           blk.33.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  300:           blk.33.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  301:             blk.33.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  302:           blk.33.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  303:             blk.33.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  304:        blk.33.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  305:             blk.33.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  306:             blk.33.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  307:             blk.34.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  308:        blk.34.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  309:             blk.34.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  310:             blk.34.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  311:          blk.34.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  312:           blk.34.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  313:           blk.34.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  314:             blk.34.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  315:           blk.34.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  316:          blk.35.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  317:           blk.35.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  318:           blk.35.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  319:             blk.35.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  320:           blk.35.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  321:             blk.35.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  322:        blk.35.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  323:             blk.35.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  324:             blk.35.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  325:          blk.36.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  326:           blk.36.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  327:           blk.36.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  328:             blk.36.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  329:           blk.36.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  330:             blk.36.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  331:        blk.36.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  332:             blk.36.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  333:             blk.36.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  334:          blk.37.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  335:           blk.37.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  336:           blk.37.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  337:             blk.37.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  338:           blk.37.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  339:             blk.37.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  340:        blk.37.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  341:             blk.37.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  342:             blk.37.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  343:          blk.38.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  344:           blk.38.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  345:           blk.38.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  346:             blk.38.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  347:           blk.38.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  348:             blk.38.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  349:        blk.38.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  350:             blk.38.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  351:             blk.38.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  352:          blk.39.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  353:           blk.39.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  354:           blk.39.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  355:             blk.39.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  356:           blk.39.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  357:             blk.39.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  358:        blk.39.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  359:             blk.39.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  360:             blk.39.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  361:          blk.40.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  362:           blk.40.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  363:           blk.40.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  364:             blk.40.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  365:           blk.40.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  366:             blk.40.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  367:        blk.40.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  368:             blk.40.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  369:             blk.40.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  370:          blk.41.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  371:           blk.41.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  372:           blk.41.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  373:             blk.41.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  374:           blk.41.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  375:             blk.41.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  376:        blk.41.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  377:             blk.41.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  378:             blk.41.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  379:          blk.42.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  380:           blk.42.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  381:           blk.42.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  382:             blk.42.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  383:           blk.42.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  384:             blk.42.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  385:        blk.42.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  386:             blk.42.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  387:             blk.42.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  388:          blk.43.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  389:           blk.43.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  390:           blk.43.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  391:             blk.43.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  392:           blk.43.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  393:             blk.43.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  394:        blk.43.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  395:             blk.43.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  396:             blk.43.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  397:          blk.44.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  398:           blk.44.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  399:           blk.44.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  400:             blk.44.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  401:           blk.44.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  402:             blk.44.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  403:        blk.44.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  404:             blk.44.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  405:             blk.44.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  406:           blk.45.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  407:             blk.45.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  408:             blk.45.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  409:        blk.45.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  410:             blk.45.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  411:             blk.45.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  412:          blk.45.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  413:           blk.45.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  414:           blk.45.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  415:          blk.46.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  416:           blk.46.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  417:           blk.46.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  418:             blk.46.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  419:           blk.46.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  420:             blk.46.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  421:        blk.46.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  422:             blk.46.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  423:             blk.46.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  424:          blk.47.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  425:           blk.47.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  426:           blk.47.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  427:             blk.47.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  428:           blk.47.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  429:             blk.47.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  430:        blk.47.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  431:             blk.47.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  432:             blk.47.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  433:          blk.48.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  434:           blk.48.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  435:           blk.48.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  436:             blk.48.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  437:           blk.48.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  438:             blk.48.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  439:        blk.48.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  440:             blk.48.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  441:             blk.48.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  442:          blk.49.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  443:           blk.49.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  444:           blk.49.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  445:             blk.49.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  446:           blk.49.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  447:             blk.49.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  448:        blk.49.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  449:             blk.49.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  450:             blk.49.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  451:          blk.50.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  452:           blk.50.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  453:           blk.50.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  454:             blk.50.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  455:           blk.50.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  456:             blk.50.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  457:        blk.50.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  458:             blk.50.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  459:             blk.50.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  460:           blk.51.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  461:             blk.51.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  462:        blk.51.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  463:             blk.51.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  464:             blk.51.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  465:          blk.51.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  466:           blk.51.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  467:             blk.51.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  468:           blk.51.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  469:          blk.52.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  470:           blk.52.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  471:           blk.52.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  472:             blk.52.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  473:           blk.52.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  474:             blk.52.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  475:        blk.52.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  476:             blk.52.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  477:             blk.52.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  478:          blk.53.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  479:           blk.53.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  480:           blk.53.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  481:             blk.53.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  482:           blk.53.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  483:             blk.53.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  484:        blk.53.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  485:             blk.53.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  486:             blk.53.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  487:          blk.54.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  488:           blk.54.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  489:           blk.54.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  490:             blk.54.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  491:           blk.54.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  492:             blk.54.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  493:        blk.54.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  494:             blk.54.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  495:             blk.54.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  496:          blk.55.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  497:           blk.55.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  498:           blk.55.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  499:             blk.55.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  500:           blk.55.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  501:             blk.55.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  502:        blk.55.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  503:             blk.55.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  504:             blk.55.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  505:          blk.56.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  506:           blk.56.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  507:           blk.56.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  508:             blk.56.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  509:           blk.56.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  510:             blk.56.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  511:        blk.56.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  512:             blk.56.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  513:             blk.56.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  514:             blk.57.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  515:        blk.57.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  516:             blk.57.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  517:             blk.57.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  518:          blk.57.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  519:           blk.57.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  520:           blk.57.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  521:             blk.57.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  522:           blk.57.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  523:          blk.58.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  524:           blk.58.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  525:           blk.58.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  526:             blk.58.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  527:           blk.58.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  528:             blk.58.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  529:        blk.58.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  530:             blk.58.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  531:             blk.58.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  532:          blk.59.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  533:           blk.59.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  534:           blk.59.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  535:             blk.59.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  536:           blk.59.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  537:             blk.59.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  538:        blk.59.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  539:             blk.59.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  540:             blk.59.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  541:          blk.60.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  542:           blk.60.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  543:           blk.60.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  544:             blk.60.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  545:           blk.60.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  546:             blk.60.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  547:        blk.60.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  548:             blk.60.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  549:             blk.60.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  550:          blk.61.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  551:           blk.61.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  552:           blk.61.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  553:             blk.61.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  554:           blk.61.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  555:             blk.61.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  556:        blk.61.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  557:             blk.61.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  558:             blk.61.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  559:          blk.62.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  560:           blk.62.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  561:           blk.62.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  562:             blk.62.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  563:           blk.62.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  564:             blk.62.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  565:        blk.62.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  566:             blk.62.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  567:             blk.62.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  568:          blk.63.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  569:           blk.63.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  570:           blk.63.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  571:             blk.63.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  572:           blk.63.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  573:             blk.63.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  574:        blk.63.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  575:             blk.63.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  576:             blk.63.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  577:          blk.64.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  578:           blk.64.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  579:           blk.64.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  580:             blk.64.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  581:           blk.64.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  582:             blk.64.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  583:        blk.64.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  584:             blk.64.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  585:             blk.64.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  586:          blk.65.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  587:           blk.65.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  588:           blk.65.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  589:             blk.65.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  590:           blk.65.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  591:             blk.65.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  592:        blk.65.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  593:             blk.65.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  594:             blk.65.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  595:          blk.66.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  596:           blk.66.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  597:           blk.66.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  598:             blk.66.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  599:           blk.66.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  600:             blk.66.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  601:        blk.66.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  602:             blk.66.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  603:             blk.66.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  604:          blk.67.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  605:           blk.67.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  606:           blk.67.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  607:             blk.67.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  608:           blk.67.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  609:             blk.67.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  610:        blk.67.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  611:             blk.67.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  612:             blk.67.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  613:           blk.68.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  614:             blk.68.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  615:             blk.68.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  616:        blk.68.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  617:             blk.68.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  618:             blk.68.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  619:          blk.68.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  620:           blk.68.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  621:           blk.68.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  622:          blk.69.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  623:           blk.69.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  624:           blk.69.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  625:             blk.69.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  626:           blk.69.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  627:             blk.69.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  628:        blk.69.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  629:             blk.69.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  630:             blk.69.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  631:          blk.70.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  632:           blk.70.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  633:           blk.70.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  634:             blk.70.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  635:           blk.70.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  636:             blk.70.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  637:        blk.70.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  638:             blk.70.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  639:             blk.70.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  640:          blk.71.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  641:           blk.71.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  642:           blk.71.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  643:             blk.71.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  644:           blk.71.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  645:             blk.71.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  646:        blk.71.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  647:             blk.71.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  648:             blk.71.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  649:          blk.72.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  650:           blk.72.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  651:           blk.72.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  652:             blk.72.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  653:           blk.72.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  654:             blk.72.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  655:        blk.72.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  656:             blk.72.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  657:             blk.72.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  658:          blk.73.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  659:           blk.73.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  660:           blk.73.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  661:             blk.73.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  662:           blk.73.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  663:             blk.73.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  664:        blk.73.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  665:             blk.73.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  666:             blk.73.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  667:           blk.74.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  668:             blk.74.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  669:        blk.74.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  670:             blk.74.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  671:             blk.74.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  672:          blk.74.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  673:           blk.74.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  674:             blk.74.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  675:           blk.74.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  676:          blk.75.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  677:           blk.75.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  678:           blk.75.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  679:             blk.75.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  680:           blk.75.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  681:             blk.75.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  682:        blk.75.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  683:             blk.75.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  684:             blk.75.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  685:          blk.76.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  686:           blk.76.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  687:           blk.76.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  688:             blk.76.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  689:           blk.76.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  690:             blk.76.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  691:        blk.76.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  692:             blk.76.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  693:             blk.76.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  694:          blk.77.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  695:           blk.77.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  696:           blk.77.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  697:             blk.77.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  698:           blk.77.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  699:             blk.77.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  700:        blk.77.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  701:             blk.77.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  702:             blk.77.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  703:          blk.78.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  704:           blk.78.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  705:           blk.78.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  706:             blk.78.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  707:           blk.78.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  708:             blk.78.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  709:        blk.78.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  710:             blk.78.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  711:             blk.78.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  712:          blk.79.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  713:           blk.79.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  714:           blk.79.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  715:             blk.79.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  716:           blk.79.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  717:             blk.79.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  718:        blk.79.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  719:             blk.79.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  720:             blk.79.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  721:               output_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  722:                    output.weight q6_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - kv   0:                       general.architecture str     \n",
      "llama_model_loader: - kv   1:                               general.name str     \n",
      "llama_model_loader: - kv   2:                       llama.context_length u32     \n",
      "llama_model_loader: - kv   3:                     llama.embedding_length u32     \n",
      "llama_model_loader: - kv   4:                          llama.block_count u32     \n",
      "llama_model_loader: - kv   5:                  llama.feed_forward_length u32     \n",
      "llama_model_loader: - kv   6:                 llama.rope.dimension_count u32     \n",
      "llama_model_loader: - kv   7:                 llama.attention.head_count u32     \n",
      "llama_model_loader: - kv   8:              llama.attention.head_count_kv u32     \n",
      "llama_model_loader: - kv   9:     llama.attention.layer_norm_rms_epsilon f32     \n",
      "llama_model_loader: - kv  10:                          general.file_type u32     \n",
      "llama_model_loader: - kv  11:                       tokenizer.ggml.model str     \n",
      "llama_model_loader: - kv  12:                      tokenizer.ggml.tokens arr     \n",
      "llama_model_loader: - kv  13:                      tokenizer.ggml.scores arr     \n",
      "llama_model_loader: - kv  14:                  tokenizer.ggml.token_type arr     \n",
      "llama_model_loader: - kv  15:                tokenizer.ggml.bos_token_id u32     \n",
      "llama_model_loader: - kv  16:                tokenizer.ggml.eos_token_id u32     \n",
      "llama_model_loader: - kv  17:            tokenizer.ggml.unknown_token_id u32     \n",
      "llama_model_loader: - kv  18:               general.quantization_version u32     \n",
      "llama_model_loader: - type  f32:  161 tensors\n",
      "llama_model_loader: - type q5_K:  481 tensors\n",
      "llama_model_loader: - type q6_K:   81 tensors\n",
      "llm_load_print_meta: format         = GGUF V2 (latest)\n",
      "llm_load_print_meta: arch           = llama\n",
      "llm_load_print_meta: vocab type     = SPM\n",
      "llm_load_print_meta: n_vocab        = 32000\n",
      "llm_load_print_meta: n_merges       = 0\n",
      "llm_load_print_meta: n_ctx_train    = 4096\n",
      "llm_load_print_meta: n_ctx          = 4096\n",
      "llm_load_print_meta: n_embd         = 8192\n",
      "llm_load_print_meta: n_head         = 64\n",
      "llm_load_print_meta: n_head_kv      = 8\n",
      "llm_load_print_meta: n_layer        = 80\n",
      "llm_load_print_meta: n_rot          = 128\n",
      "llm_load_print_meta: n_gqa          = 8\n",
      "llm_load_print_meta: f_norm_eps     = 1.0e-05\n",
      "llm_load_print_meta: f_norm_rms_eps = 1.0e-05\n",
      "llm_load_print_meta: n_ff           = 28672\n",
      "llm_load_print_meta: freq_base      = 10000.0\n",
      "llm_load_print_meta: freq_scale     = 1\n",
      "llm_load_print_meta: model type     = 70B\n",
      "llm_load_print_meta: model ftype    = mostly Q5_K - Medium\n",
      "llm_load_print_meta: model size     = 68.98 B\n",
      "llm_load_print_meta: general.name   = LLaMA v2\n",
      "llm_load_print_meta: BOS token = 1 '<s>'\n",
      "llm_load_print_meta: EOS token = 2 '</s>'\n",
      "llm_load_print_meta: UNK token = 0 '<unk>'\n",
      "llm_load_print_meta: LF token  = 13 '<0x0A>'\n",
      "llm_load_tensors: ggml ctx size =    0.23 MB\n",
      "llm_load_tensors: mem required  = 46494.72 MB (+ 1280.00 MB per state)\n",
      "....................................................................................................\n",
      "llama_new_context_with_model: kv self size  = 1280.00 MB\n",
      "ggml_metal_init: allocating\n",
      "ggml_metal_init: found device: Apple M2 Max\n",
      "ggml_metal_init: picking default device: Apple M2 Max\n",
      "ggml_metal_init: loading '/Users/gburns/miniconda3/envs/alhazen/lib/python3.11/site-packages/llama_cpp/ggml-metal.metal'\n",
      "ggml_metal_init: loaded kernel_add                            0x11f195710 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_add_row                        0x11f19cba0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul                            0x11f19ad60 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_row                        0x11f19ec50 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_scale                          0x11f19eeb0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_silu                           0x11f19f110 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_relu                           0x11f19f370 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_gelu                           0x11a3b11a0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_soft_max                       0x11a3ce8e0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_soft_max_4                     0x11a3ceb40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_diag_mask_inf                  0x11a3ceda0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_diag_mask_inf_8                0x11a3cf000 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_f16                   0x11a3cf260 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_0                  0x11a3cf4c0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_1                  0x11a3cf720 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q8_0                  0x11a3cf980 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q2_K                  0x11a3cfbe0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q3_K                  0x11a3cfe40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_K                  0x11a3b74c0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q5_K                  0x11a3b7720 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q6_K                  0x11a3b7980 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_rms_norm                       0x11a3b7be0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_norm                           0x11a3b7e40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32                0x11a3b80a0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32_1row           0x11a3af020 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32_l4             0x11a3af280 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_0_f32               0x11a3af4e0 | th_max =  896 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_1_f32               0x11a3f4fc0 | th_max =  896 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q8_0_f32               0x11a3f5220 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q2_K_f32               0x11a3f5480 | th_max =  640 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q3_K_f32               0x11a3f56e0 | th_max =  576 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_K_f32               0x11a3f5940 | th_max =  576 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q5_K_f32               0x11a3f5ba0 | th_max =  640 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q6_K_f32               0x11a3f5e00 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_f16_f32                 0x11a3f6060 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_0_f32                0x11a3f62c0 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q8_0_f32                0x11a3f6520 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_1_f32                0x11a3f6780 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q2_K_f32                0x11a3f69e0 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q3_K_f32                0x11a3f6c40 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_K_f32                0x11a3f6ea0 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q5_K_f32                0x11a3f7100 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q6_K_f32                0x11a3f7360 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_rope                           0x11a3f75c0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_alibi_f32                      0x11a3f7820 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f32_f16                    0x11a3f7a80 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f32_f32                    0x11a3f7ce0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f16_f16                    0x11a3f7f40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: hasUnifiedMemory              = true\n",
      "ggml_metal_init: recommendedMaxWorkingSetSize  = 73728.00 MB\n",
      "ggml_metal_init: maxTransferRate               = built-in GPU\n",
      "llama_new_context_with_model: compute buffer total size =  565.47 MB\n",
      "llama_new_context_with_model: max tensor size =   205.08 MB\n",
      "ggml_metal_add_buffer: allocated 'data            ' buffer, size = 46495.22 MB, (94850.16 / 73728.00), warning: current allocated size is greater than the recommended max working set size\n",
      "ggml_metal_add_buffer: allocated 'eval            ' buffer, size =     1.48 MB, (94851.64 / 73728.00), warning: current allocated size is greater than the recommended max working set size\n",
      "ggml_metal_add_buffer: allocated 'kv              ' buffer, size =  1282.00 MB, (96133.64 / 73728.00), warning: current allocated size is greater than the recommended max working set size\n",
      "AVX = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 0 | NEON = 1 | ARM_FMA = 1 | F16C = 0 | FP16_VA = 1 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 0 | SSSE3 = 0 | VSX = 0 | \n",
      "ggml_metal_add_buffer: allocated 'alloc           ' buffer, size =   564.02 MB, (96697.66 / 73728.00), warning: current allocated size is greater than the recommended max working set size\n",
      "ggml_metal_free: deallocating\n"
     ]
    }
   ],
   "source": [
    "# Callbacks support token-wise streaming\n",
    "callback_manager = CallbackManager([StreamingStdOutCallbackHandler()])\n",
    "\n",
    "n_gpu_layers = 10  # Change this value based on your model and your GPU VRAM pool.\n",
    "n_batch = 1024  # Should be between 1 and n_ctx, consider the amount of VRAM in your GPU.\n",
    "\n",
    "# Make sure the model path is correct for your system!\n",
    "llm = LlamaCpp(\n",
    "    model_path=\"/Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf\",\n",
    "    n_ctx=4096,\n",
    "    n_gpu_layers=n_gpu_layers,\n",
    "    n_batch=n_batch,\n",
    "    callback_manager=callback_manager,\n",
    "    f16_kv=True,\n",
    "    verbose=True, # Verbose is required to pass to the callback manager\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "You are an expert biological scientist trained in imaging and microscopy. \n",
      "Your knowledge is deep and wide. You are terse and precise in your language.\n",
      "\n",
      "Your job is to read the title and abstract of a scientific paper in detail, understand what you read, and perform the task described below based on what you read. \n",
      "The text of the title and the text of the abstract will be delimited with triple backticks. \n",
      "\n",
      "First, break down the text of the title and abstract into separate sentences.\n",
      "\n",
      "Second, identify which sentences describe \n",
      "(A) background information about the work, \n",
      "(B) the goals of the work being described, \n",
      "(C) the methods used to achieve the goals, \n",
      "(D) the results of the work, and \n",
      "(E) the conclusions of the work. \n",
      "\n",
      "Third, use only sentences you identify as goals, methods, and results to identify the main contribution of the paper. \n",
      "Record this contribution as a complete noun phrase shorter than 12 words or so as main_contribution in the output.\n",
      "\n",
      "Fourth, decide if that contribution of the paper is developing an imaging technique or method. Be careful about this decision and only answer 'true' \n",
      "if you are confident that the contribution of the paper states that a new imaging technique or method was developed. Possible examples of imaging techniques\n",
      "could be new types of microscope, new imaging techniques, hardware, optics, or sample preparation for imaging. \n",
      "\n",
      "If the main contribution is the development of a new imaging method, record a true value in a variable called is_imaging_method in the output. \n",
      "\n",
      "If the main contribution is not the development of a new imaging method (such as the development of a genetic test, or a finding about a biological system), record a false value in a variable called is_imaging_method in the output. \n",
      "\n",
      "Format your response as a JSON object with \"main_contribution\" and \"is_imaging_method\" as the keys. \n",
      "\n",
      "Do not include any other response other than the JSON object. Remember, stop generating text after you finish the JSON object. Do not provide additional explanations of your answer.\n",
      "\n",
      "Article\n",
      "title: '''Multiplexing molecular tension sensors reveals piconewton force gradient across talin-1.'''\n",
      "abstract: '''Förster resonance energy transfer (FRET)-based tension sensor modules (TSMs) are available for investigating how distinct proteins bear mechanical forces in cells. Yet, forces in the single piconewton (pN) regime remain difficult to resolve, and tools for multiplexed tension sensing are lacking. Here, we report the generation and calibration of a genetically encoded, FRET-based biosensor called FL-TSM, which is characterized by a near-digital force response and increased sensitivity at 3-5 pN. In addition, we present a method allowing the simultaneous evaluation of coexpressed tension sensor constructs using two-color fluorescence lifetime microscopy. Finally, we introduce a procedure to calculate the fraction of mechanically engaged molecules within cells. Application of these techniques to new talin biosensors reveals an intramolecular tension gradient across talin-1 that is established upon integrin-mediated cell adhesion. The tension gradient is actomyosin- and vinculin-dependent and sensitive to the rigidity of the extracellular environment.'''\n",
      "\n",
      "{\n",
      "\"is_method_paper\": true\n",
      "}\n",
      "\n",
      "The paper's title and abstract indicate that the work is focused on developing a new method for measuring piconewton forces in cells, specifically the \"FL-TSM\" biosensor. The abstract mentions the calibration of this biosensor and its application in revealing an intramolecular tension gradient across talin-1. Therefore, the main goal of the published work was to create or test a new technique or method, making it a true value for the \"is_method_paper\" variable.\n",
      "\n",
      "\n",
      "{\n",
      "\"is_method_paper\": true\n",
      "}\n",
      "\n",
      "The paper's title and abstract indicate that the work is focused on developing a new method for measuring piconewton forces in cells, specifically the \"FL-TSM\" biosensor. The abstract mentions the calibration of this biosensor and its application in revealing an intramolecular tension gradient across talin-1. Therefore, the main goal of the published work was to create or test a new technique or method, making it a true value for the \"is_method_paper\" variable.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "llama_print_timings:        load time = 14193.17 ms\n",
      "llama_print_timings:      sample time =    89.03 ms /   123 runs   (    0.72 ms per token,  1381.60 tokens per second)\n",
      "llama_print_timings: prompt eval time = 14193.08 ms /   615 tokens (   23.08 ms per token,    43.33 tokens per second)\n",
      "llama_print_timings:        eval time = 20797.63 ms /   122 runs   (  170.47 ms per token,     5.87 tokens per second)\n",
      "llama_print_timings:       total time = 35276.71 ms\n"
     ]
    }
   ],
   "source": [
    "# check\n",
    "r = df[df.ID_PAPER==\"28945706\"]\n",
    "a = r.ABSTRACT.values[0]\n",
    "t = r.TITLE.values[0]\n",
    "prompt = p2.format(title=t, abstract=a)\n",
    "print(prompt)\n",
    "output = llm(p3.format(title=t, abstract=a))\n",
    "#m = output_parser.parse(output)\n",
    "\n",
    "print('\\n\\n\\n'+output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "python3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
