<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="description" content="Developing a simple tutorial to provide a walkthrough for users attempting to use Alhazen for the first time. This is based on analysis of the Cryo Electron Tomography literature and tools we have developed to analyze that data.">

<title>alhazen - CryoET Tutorial</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="../styles.css">
<meta property="og:title" content="alhazen - CryoET Tutorial">
<meta property="og:description" content="Developing a simple tutorial to provide a walkthrough for users attempting to use Alhazen for the first time. This is based on analysis of the Cryo Electron Tomography literature and tools we have developed to analyze that data.">
<meta property="og:site-name" content="alhazen">
<meta name="twitter:title" content="alhazen - CryoET Tutorial">
<meta name="twitter:description" content="Developing a simple tutorial to provide a walkthrough for users attempting to use Alhazen for the first time. This is based on analysis of the Cryo Electron Tomography literature and tools we have developed to analyze that data.">
<meta name="twitter:card" content="summary">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">alhazen</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../index.html#install-dependencies" rel="" target="">
 <span class="menu-text">Get Started</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../docnb1_use_cases.html" rel="" target="" aria-current="page">
 <span class="menu-text">Use Cases</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../docnb2_architecture.html" rel="" target="">
 <span class="menu-text">Platform Architecture</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-help" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Help</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-help">    
        <li>
    <a class="dropdown-item" href="https://github.com/chanzuckerberg/alhazen/issues" rel="" target=""><i class="bi bi-bug" role="img">
</i> 
 <span class="dropdown-text">Report an Issue</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/chanzuckerberg/alhazen" rel="" target=""><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../tutorials/index.html">Tutorial Notebooks</a></li><li class="breadcrumb-item"><a href="../tutorials/cryoet_tutorial.html">CryoET Tutorial</a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Home - Alhazen</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../docnb1_use_cases.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Use Cases</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../docnb2_architecture.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Platform Architecture</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Tech Elements</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../agents/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Agents</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../agents/agent.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Alhazen Agent / Chatbot.</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../tools/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/toolkit.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Alhazen Toolkit</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/basic_tools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Tools for Alhazen</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/metadata_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Methods Metadata Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/protocol_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Protocol Workflow Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/paperqa_emulation_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Paper QA Emulation Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/tiab_clasifier_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/tiab_mapping_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tools/tiab_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../utils/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Utilities</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../utils/ceifns_db.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Database for Scientific Knowledge</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../utils/search_engine_eutils.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Search Engine Tools</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../utils/web_robot.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Web Robots</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../utils/html_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">HTML Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../utils/jats_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">JATS Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../utils/pdf_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PDF Text Extractor Utility</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../tutorials/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tutorial Notebooks</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../tutorials/cryoet_tutorial.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">CryoET Tutorial</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true">
 <span class="menu-text">Example Notebooks</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../cookbook/single_doc_extraction/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Metadata Extraction</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/single_doc_extraction/cellxgene_datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CellXGene Datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/single_doc_extraction/cryoet.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CryoET</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/single_doc_extraction/ncats_natural_history_studies.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">NCATS Natural History Studies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/single_doc_extraction/rnaquarium.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">RNAquarium</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../cookbook/trend_analysis/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Trend Analysis</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/trend_analysis/capacity_building.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Capacity Building</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/trend_analysis/imaging_technology_evolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Image Tech Evolution</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/trend_analysis/pathogen_landscaping.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pathogen_Landscaping</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/trend_analysis/rare_disease_literature.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Rare Disease Literature.</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../cookbook/key_opinion_leaders/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Key Opinion Leaders</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-9" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-9" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/key_opinion_leaders/african_microscopy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">African Microscopy</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../cookbook/question_answering/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Question Answering</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-10" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-10" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../cookbook/question_answering/virtualcell_corepapers.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Virtual Cell Landscaping Analysis</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#introduction-to-cryoet" id="toc-introduction-to-cryoet" class="nav-link active" data-scroll-target="#introduction-to-cryoet">Introduction to CryoET</a></li>
  <li><a href="#basics" id="toc-basics" class="nav-link" data-scroll-target="#basics">Basics</a>
  <ul class="collapse">
  <li><a href="#python-imports" id="toc-python-imports" class="nav-link" data-scroll-target="#python-imports">Python Imports</a></li>
  <li><a href="#environment-variables" id="toc-environment-variables" class="nav-link" data-scroll-target="#environment-variables">Environment Variables</a></li>
  <li><a href="#setup-utils-agents-and-tools" id="toc-setup-utils-agents-and-tools" class="nav-link" data-scroll-target="#setup-utils-agents-and-tools">Setup utils, agents, and tools</a></li>
  </ul></li>
  <li><a href="#building-the-database" id="toc-building-the-database" class="nav-link" data-scroll-target="#building-the-database">Building the database</a>
  <ul class="collapse">
  <li><a href="#scripts-to-build-delete-the-database" id="toc-scripts-to-build-delete-the-database" class="nav-link" data-scroll-target="#scripts-to-build-delete-the-database">Scripts to Build / Delete the database</a></li>
  <li><a href="#build-ceifns-database-from-queries" id="toc-build-ceifns-database-from-queries" class="nav-link" data-scroll-target="#build-ceifns-database-from-queries">Build CEIFNS database from queries</a></li>
  </ul></li>
  <li><a href="#analyze-collections" id="toc-analyze-collections" class="nav-link" data-scroll-target="#analyze-collections">Analyze Collections</a>
  <ul class="collapse">
  <li><a href="#survey-run-classifications-over-papers" id="toc-survey-run-classifications-over-papers" class="nav-link" data-scroll-target="#survey-run-classifications-over-papers">Survey + Run Classifications over Papers</a></li>
  </ul></li>
  <li><a href="#run-metadata-extraction-chain-over-listed-papers" id="toc-run-metadata-extraction-chain-over-listed-papers" class="nav-link" data-scroll-target="#run-metadata-extraction-chain-over-listed-papers">Run MetaData Extraction Chain over listed papers</a></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/chanzuckerberg/alhazen/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">CryoET Tutorial</h1>
</div>

<div>
  <div class="description">
    Developing a simple tutorial to provide a walkthrough for users attempting to use Alhazen for the first time. This is based on analysis of the Cryo Electron Tomography literature and tools we have developed to analyze that data.
  </div>
</div>


<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
<section id="introduction-to-cryoet" class="level2">
<h2 class="anchored" data-anchor-id="introduction-to-cryoet">Introduction to CryoET</h2>
<p>Cryo-electron Tomography (CryoET) involves rapidly freezing biological samples in their natural state to preserve their three-dimensional structure without the need for staining or crystallization. This methodology allows researchers to visualize proteins and other biomolecules at near-atomic resolution.</p>
<p>This digital library is based on capturing all papers that mention the technique in their titles, abstracts, or methods sections and then analyzing the various methods used and their applications. Our focus is on supporting the work of the Chan Zuckerberg Imaging Institute, <a href="https://www.czimaginginstitute.org/">CZII</a> on developing <a href="https://cryoetdataportal.czscience.com/">the CryoET data portal</a>, an open source repository for CryoET-based data.</p>
</section>
<section id="basics" class="level2">
<h2 class="anchored" data-anchor-id="basics">Basics</h2>
<section id="python-imports" class="level3">
<h3 class="anchored" data-anchor-id="python-imports">Python Imports</h3>
<p>Setting python imports, environment variables, and other crucial set up parameters here.</p>
<div class="cell" data-execution_count="6">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.aliases <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.core <span class="im">import</span> lookup_chat_models</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.agent <span class="im">import</span> AlhazenAgent</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.schema_sqla <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.core <span class="im">import</span> lookup_chat_models</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.basic <span class="im">import</span> AddCollectionFromEPMCTool, DeleteCollectionTool</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.paperqa_emulation_tool <span class="im">import</span> PaperQAEmulationTool</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.metadata_extraction_tool <span class="im">import</span> <span class="op">*</span> </span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.protocol_extraction_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.tiab_classifier_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.tiab_extraction_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.tiab_mapping_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.toolkit <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.ceifns_db <span class="im">import</span> Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, backup_ceifns_database</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.searchEngineUtils <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.callbacks.tracers <span class="im">import</span> ConsoleCallbackHandler</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.docstore.document <span class="im">import</span> Document</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.text_splitter <span class="im">import</span> CharacterTextSplitter</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.vectorstores.pgvector <span class="im">import</span> PGVector</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_community.chat_models.ollama <span class="im">import</span> ChatOllama</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_google_vertexai <span class="im">import</span> ChatVertexAI</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_openai <span class="im">import</span> ChatOpenAI</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> bs4 <span class="im">import</span> BeautifulSoup,Tag,Comment,NavigableString</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> databricks <span class="im">import</span> sql</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> datetime <span class="im">import</span> datetime</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> importlib_resources <span class="im">import</span> files</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> json</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> re</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> requests</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy <span class="im">import</span> text, create_engine, exists, func, or_, and_, not_, desc, asc</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy.orm <span class="im">import</span> sessionmaker, aliased</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> time <span class="im">import</span> time,sleep</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> tqdm <span class="im">import</span> tqdm</span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.request <span class="im">import</span> urlopen</span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.parse <span class="im">import</span> quote_plus, quote, unquote</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.error <span class="im">import</span> URLError, HTTPError</span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> uuid</span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> yaml</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="environment-variables" class="level3">
<h3 class="anchored" data-anchor-id="environment-variables">Environment Variables</h3>
<p>You must set the following environmental variables for this code:</p>
<ul>
<li><code>LOCAL_FILE_PATH</code> - the location on disk where you save temporary files, downloaded models or other data.</li>
</ul>
<p>Note that this notebook will build and use a database specified as <code>cryoet_tutorial</code>, specified below</p>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.environ.get(<span class="st">'LOCAL_FILE_PATH'</span>) <span class="kw">is</span> <span class="va">None</span>: </span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>    <span class="cf">raise</span> <span class="pp">Exception</span>(<span class="st">'Where are you storing your local literature database?'</span>)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.path.exists(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]) <span class="kw">is</span> <span class="va">False</span>:</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>    os.makedirs(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>])    </span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>loc <span class="op">=</span> os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>db_name <span class="op">=</span> <span class="st">'cryoet_tutorial'</span></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Variable to prevent accidental deletion of the database or any records</span></span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>OK_TO_DELETE <span class="op">=</span> <span class="va">False</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="setup-utils-agents-and-tools" class="level3">
<h3 class="anchored" data-anchor-id="setup-utils-agents-and-tools">Setup utils, agents, and tools</h3>
<p>This cell sets up a database engine (<code>ldb</code>) and lists the available large-language models you can use.</p>
<div class="cell" data-execution_count="8">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>ldb <span class="op">=</span> Ceifns_LiteratureDb(loc<span class="op">=</span>loc, name<span class="op">=</span>db_name)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>llms_lookup <span class="op">=</span> lookup_chat_models()</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(llms_lookup.keys())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>dict_keys(['ollama_llama3', 'ollama_mixtral', 'gemini1.0', 'databricks_dbrx', 'databricks_mixtral', 'groq_mixtral', 'groq_llama3', 'gpt4_1106', 'gpt35'])</code></pre>
</div>
</div>
<p>This cell initiates an <a href="https://chanzuckerberg.github.io/agents/agent.html#alhazenagent"><code>AlhazenAgent</code></a> that you can use to run tools or execute commands over.</p>
<div class="cell" data-execution_count="9">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>llm <span class="op">=</span> llms_lookup.get(<span class="st">'groq_llama3'</span>)</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>cb <span class="op">=</span> AlhazenAgent(llm, llm, db_name<span class="op">=</span>db_name)</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'AGENT TOOLS'</span>)</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools():</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="st">'</span><span class="ch">\t</span><span class="st">'</span><span class="op">+</span><span class="bu">type</span>(t).<span class="va">__name__</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>AGENT TOOLS
    AddCollectionFromEPMCTool
    AddAuthorsToCollectionTool
    DescribeCollectionCompositionTool
    DeleteCollectionTool
    RetrieveFullTextTool
    RetrieveFullTextToolForACollection
    MetadataExtraction_EverythingEverywhere_Tool
    MetadataExtraction_MethodsSectionOnly_Tool
    SimpleExtractionWithRAGTool
    PaperQAEmulationTool
    ProcotolEntitiesExtractionTool
    CheckExpressionTool
    TitleAbstractClassifier_OneDocAtATime_Tool
    TitleAbstractDiscourseMappingTool
    TitleAbstractExtraction_OneDocAtATime_Tool</code></pre>
</div>
</div>
</section>
</section>
<section id="building-the-database" class="level2">
<h2 class="anchored" data-anchor-id="building-the-database">Building the database</h2>
<section id="scripts-to-build-delete-the-database" class="level3">
<h3 class="anchored" data-anchor-id="scripts-to-build-delete-the-database">Scripts to Build / Delete the database</h3>
<p>If you need to restore a deleted database from backup, use the following shell commands:</p>
<pre><code>$ createdb em_tech
$ psql -d em_tech -f /local/file/path/em_tech/backup&lt;date_time&gt;.sql</code></pre>
<p>This command will delete your existing database (but will also store a copy).</p>
<div class="cell" data-execution_count="5">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> OK_TO_DELETE:</span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>    drop_ceifns_database(db_name, backupFirst<span class="op">=</span><span class="va">True</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>This command will backup your current database</p>
<div class="cell" data-execution_count="6">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> OK_TO_DELETE:</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>    current_date_time <span class="op">=</span> datetime.now()</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>    formatted_date_time <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>current_date_time<span class="sc">:</span><span class="op">%</span>Y<span class="op">-%</span>m<span class="op">-%</span>d<span class="op">-%</span>H<span class="op">-%</span>M<span class="op">-%</span>S<span class="sc">}</span><span class="ss">'</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>    backup_path <span class="op">=</span> loc<span class="op">+</span><span class="st">'/'</span><span class="op">+</span>db_name<span class="op">+</span><span class="st">'/backup'</span><span class="op">+</span>formatted_date_time<span class="op">+</span><span class="st">'.sql'</span></span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>    backup_ceifns_database(db_name, backup_path)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>This command will create a new, fresh, empty copy of your database.</p>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>create_ceifns_database(db_name)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-error">
<pre><code>DuplicateDatabase: database "cryoet_tutorial" already exists</code></pre>
</div>
</div>
</section>
<section id="build-ceifns-database-from-queries" class="level3">
<h3 class="anchored" data-anchor-id="build-ceifns-database-from-queries">Build CEIFNS database from queries</h3>
<section id="add-a-collection-of-all-cryoet-papers-based-on-a-query" class="level4">
<h4 class="anchored" data-anchor-id="add-a-collection-of-all-cryoet-papers-based-on-a-query">Add a collection of all CryoET papers based on a query</h4>
<p>This runs a query on European PMC for terms + synonyms related to Cryo Electron Tomography</p>
<div class="cell" data-execution_count="18">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>cryoet_query <span class="op">=</span> <span class="st">'''</span></span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a><span class="st">("Cryoelectron Tomography" OR "Cryo Electron Tomography" OR "Cryo-Electron Tomography" OR</span></span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a><span class="st">    "Cryo-ET" OR "CryoET" OR "Cryoelectron Tomography" OR "cryo electron tomography" or </span></span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a><span class="st">    "cryo-electron tomography" OR "cryo-et" OR cryoet)</span></span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a><span class="st">'''</span></span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool.run(tool_input<span class="op">=</span>{<span class="st">'id'</span>: <span class="st">'1'</span>, </span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">'name'</span>: <span class="st">'CryoET Papers'</span>, </span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">'query'</span>: cryoet_query, </span>
<span id="cb12-10"><a href="#cb12-10" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">'full_text'</span>: <span class="va">False</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=
("Cryoelectron Tomography" OR "Cryo Electron Tomography" OR "Cryo-Electron Tomography" OR
    "Cryo-ET" OR "CryoET" OR "Cryoelectron Tomography" OR "cryo electron tomography" or 
    "cryo-electron tomography" OR "cryo-et" OR cryoet)
, 6016 European PMC PAPERS FOUND
 Returning 5911</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 7/7 [01:05&lt;00:00,  9.39s/it]
100%|██████████| 5911/5911 [00:50&lt;00:00, 116.38it/s]
/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py:293: LangChainPendingDeprecationWarning: Please use JSONB instead of JSON for metadata. This change will allow for more efficient querying that involves filtering based on metadata.Please note that filtering operators have been changed when using JSOB metadata to be prefixed with a $ sign to avoid name collisions with columns. If you're using an existing database, you will need to create adb migration for your metadata column to be JSONB and update your queries to use the new operators. 
  warn_deprecated(</code></pre>
</div>
<div class="cell-output cell-output-display" data-execution_count="18">
<pre><code>{'response': 'We added a collection to the database called `CryoET Papers` containing 0 papers from this query: `\n("Cryoelectron Tomography" OR "Cryo Electron Tomography" OR "Cryo-Electron Tomography" OR\n    "Cryo-ET" OR "CryoET" OR "Cryoelectron Tomography" OR "cryo electron tomography" or \n    "cryo-electron tomography" OR "cryo-et" OR cryoet)\n`.'}</code></pre>
</div>
</div>
</section>
<section id="adding-machine-learning-also-from-a-query" class="level4">
<h4 class="anchored" data-anchor-id="adding-machine-learning-also-from-a-query">Adding Machine Learning also from a query</h4>
<div class="cell" data-execution_count="19">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>ml_query <span class="op">=</span> <span class="st">'''</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a><span class="st">("Cryoelectron Tomography" OR "Cryo Electron Tomography" OR "Cryo-Electron Tomography" OR</span></span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a><span class="st">    "Cryo-ET" OR "CryoET" OR "Cryoelectron Tomography" OR "cryo electron tomography" or </span></span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a><span class="st">    "cryo-electron tomography" OR "cryo-et" OR cryoet ) AND </span></span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a><span class="st">("Machine Learning" OR "Artificial Intelligence" OR "Deep Learning" OR "Neural Networks")</span></span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a><span class="st">'''</span></span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool.run(tool_input<span class="op">=</span>{<span class="st">'id'</span>: <span class="st">'2'</span>, </span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">'name'</span>: <span class="st">'Machine Learning in CryoET'</span>, </span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">'query'</span>: ml_query, </span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">'full_text'</span>: <span class="va">False</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=
("Cryoelectron Tomography" OR "Cryo Electron Tomography" OR "Cryo-Electron Tomography" OR
    "Cryo-ET" OR "CryoET" OR "Cryoelectron Tomography" OR "cryo electron tomography" or 
    "cryo-electron tomography" OR "cryo-et" OR cryoet ) AND 
("Machine Learning" OR "Artificial Intelligence" OR "Deep Learning" OR "Neural Networks")
, 540 European PMC PAPERS FOUND
 Returning 517</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 1/1 [00:06&lt;00:00,  6.60s/it]
100%|██████████| 517/517 [00:02&lt;00:00, 228.36it/s]
/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py:293: LangChainPendingDeprecationWarning: Please use JSONB instead of JSON for metadata. This change will allow for more efficient querying that involves filtering based on metadata.Please note that filtering operators have been changed when using JSOB metadata to be prefixed with a $ sign to avoid name collisions with columns. If you're using an existing database, you will need to create adb migration for your metadata column to be JSONB and update your queries to use the new operators. 
  warn_deprecated(</code></pre>
</div>
<div class="cell-output cell-output-display" data-execution_count="19">
<pre><code>{'response': 'We added a collection to the database called `Machine Learning in CryoET` containing 0 papers from this query: `\n("Cryoelectron Tomography" OR "Cryo Electron Tomography" OR "Cryo-Electron Tomography" OR\n    "Cryo-ET" OR "CryoET" OR "Cryoelectron Tomography" OR "cryo electron tomography" or \n    "cryo-electron tomography" OR "cryo-et" OR cryoet ) AND \n("Machine Learning" OR "Artificial Intelligence" OR "Deep Learning" OR "Neural Networks")\n`.'}</code></pre>
</div>
</div>
</section>
<section id="creates-a-new-collection-of-randomly-sampled-papers-to-showcase-full-text-download-capability" class="level4">
<h4 class="anchored" data-anchor-id="creates-a-new-collection-of-randomly-sampled-papers-to-showcase-full-text-download-capability">Creates a new collection of randomly sampled papers to showcase full-text download capability</h4>
<div class="cell" data-execution_count="21">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>ldb.create_new_collection_from_sample(<span class="st">'3'</span>, <span class="st">'CryoET Papers Tests'</span>, <span class="st">'1'</span>, <span class="dv">20</span>, [<span class="st">'ScientificPrimaryResearchArticle'</span>, <span class="st">'ScientificPrimaryResearchPreprint'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="21">
<pre><code>ScientificKnowledgeCollection(creation_date=None,content=None,token_count=None,format=None,provenance=None,license=None,name=CryoET Papers Tests,id=3,type=skem:ScientificKnowledgeCollection,)</code></pre>
</div>
</div>
</section>
</section>
</section>
<section id="analyze-collections" class="level2">
<h2 class="anchored" data-anchor-id="analyze-collections">Analyze Collections</h2>
<section id="survey-run-classifications-over-papers" class="level3">
<h3 class="anchored" data-anchor-id="survey-run-classifications-over-papers">Survey + Run Classifications over Papers</h3>
<p>This invoke the following classification process on the paper (defined in the prompt definition in <code>./local_resources/prompts/tiab_prompts</code>):</p>
<ul>
<li>A - Structural descriptions of Viral Pathogens (such as HIV, Influenza, SARS-CoV-2, etc.)</li>
<li>B - Studies of mutated protein structures associated with disease (such as Alzheimer’s, Parkinson’s, etc.)</li>
<li>C - Structural studies of bacterial pathogens (such as E. coli, Salmonella, etc.)</li>
<li>D - Structural studies of plant cells</li>
<li>E - Structural studies of material science of non-biological samples</li>
<li>F - Structural studies of transporters or transport mechanisms within cells, studies involving the cytoskeleton or active transport processes.</li>
<li>G - Structural studies of synapses or other mechansism of releasing vesicles over the plasma membrane</li>
<li>H - Structural studies of any other organelle or structured component of a cell.</li>
<li>I - Studies of dynamic biological processes at a cellular level (such as cell division, cell migration, etc.)</li>
<li>J - Studies of dynamics of molecular interactions within a cell.<br>
</li>
<li>K - Development of new CryoET imaging methods (including grid preparation techniques, such as lift-out).</li>
<li>L - Development of new data analysis methods (including machine learning, segmentation, point-picking, object recognition, or reconstruction).</li>
</ul>
<div class="cell" data-execution_count="10">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>t <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, TitleAbstractClassifier_OneDocAtATime_Tool)][<span class="dv">0</span>]</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>t.run({<span class="st">'collection_id'</span>: <span class="st">'3'</span>, <span class="st">'classification_type'</span>:<span class="st">'cryoet_study_types'</span>, <span class="st">'repeat_run'</span>:<span class="va">True</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>  0%|          | 0/20 [00:00&lt;?, ?it/s] 45%|████▌     | 9/20 [02:41&lt;03:17, 17.99s/it]</code></pre>
</div>
<div class="cell-output cell-output-error">
<pre><code>KeyboardInterrupt: </code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="co"># USE WITH </span><span class="al">CAUTION</span><span class="co"> - this will delete all extracted metadata notes in the database</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a><span class="co"># clear all notes across papers listed in `dois` list</span></span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> OK_TO_DELETE:        </span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>    l <span class="op">=</span> []</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>    q <span class="op">=</span> ldb.session.query(N, SKE) <span class="op">\</span></span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>            .<span class="bu">filter</span>(N.<span class="bu">id</span> <span class="op">==</span> NIA.Note_id) <span class="op">\</span></span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>            .<span class="bu">filter</span>(NIA.is_about_id <span class="op">==</span> SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>            .<span class="bu">filter</span>(N.<span class="bu">type</span> <span class="op">==</span> <span class="st">'TiAbClassificationNote__cryoet_study_types'</span>) <span class="op">\</span></span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a>    output <span class="op">=</span> []        </span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="bu">len</span>(q.<span class="bu">all</span>()))</span>
<span id="cb25-12"><a href="#cb25-12" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> n, ske <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb25-13"><a href="#cb25-13" aria-hidden="true" tabindex="-1"></a>        ldb.delete_note(n.<span class="bu">id</span>)    </span>
<span id="cb25-14"><a href="#cb25-14" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="bu">len</span>(q.<span class="bu">all</span>()))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Runs a query over the notes extracted and saved to the database to show the zero-shot document classifications based on the titles + abstracts</p>
<div class="cell" data-execution_count="19">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(N, SKE) <span class="op">\</span></span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">id</span> <span class="op">==</span> NIA.Note_id) <span class="op">\</span></span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(NIA.is_about_id <span class="op">==</span> SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">type</span> <span class="op">==</span> <span class="st">'TiAbClassificationNote__cryoet_study_types'</span>) <span class="op">\</span></span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>        .order_by(SKE.<span class="bu">id</span>)</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>output <span class="op">=</span> []        </span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> n, ske <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a>        tup <span class="op">=</span> json.loads(n.content)</span>
<span id="cb26-11"><a href="#cb26-11" aria-hidden="true" tabindex="-1"></a>        tup[<span class="st">'prov'</span>] <span class="op">=</span> n.name</span>
<span id="cb26-12"><a href="#cb26-12" aria-hidden="true" tabindex="-1"></a>        tup[<span class="st">'doi'</span>] <span class="op">=</span> <span class="st">'http://doi.org/'</span><span class="op">+</span>re.sub(<span class="st">'doi:'</span>, <span class="st">''</span>, ske.<span class="bu">id</span>)</span>
<span id="cb26-13"><a href="#cb26-13" aria-hidden="true" tabindex="-1"></a>        tup[<span class="st">'year'</span>] <span class="op">=</span> ske.publication_date.year</span>
<span id="cb26-14"><a href="#cb26-14" aria-hidden="true" tabindex="-1"></a>        tup[<span class="st">'month'</span>] <span class="op">=</span> ske.publication_date.month</span>
<span id="cb26-15"><a href="#cb26-15" aria-hidden="true" tabindex="-1"></a>        tup[<span class="st">'ref'</span>] <span class="op">=</span> ske.content</span>
<span id="cb26-16"><a href="#cb26-16" aria-hidden="true" tabindex="-1"></a>        output.append(tup)</span>
<span id="cb26-17"><a href="#cb26-17" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(output).sort_values([<span class="st">'year'</span>, <span class="st">'month'</span>], ascending<span class="op">=</span>[<span class="va">False</span>, <span class="va">False</span>])</span>
<span id="cb26-18"><a href="#cb26-18" aria-hidden="true" tabindex="-1"></a>df.to_csv(loc<span class="op">+</span><span class="st">'/'</span><span class="op">+</span>db_name<span class="op">+</span><span class="st">'/cryoet_study_types.tsv'</span>, sep<span class="op">=</span><span class="st">'</span><span class="ch">\t</span><span class="st">'</span>)</span>
<span id="cb26-19"><a href="#cb26-19" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="19">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">cryoet_study_type_code</th>
<th data-quarto-table-cell-role="th">cryoet_study_type_name</th>
<th data-quarto-table-cell-role="th">explanation</th>
<th data-quarto-table-cell-role="th">prov</th>
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th">year</th>
<th data-quarto-table-cell-role="th">month</th>
<th data-quarto-table-cell-role="th">ref</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">24</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the structural and functi...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">25</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the structure and functio...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">26</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the structure and functio...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">27</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the structure and functio...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">28</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the structure and functio...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">29</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper primarily concerns the study of cili...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">30</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper is primarily focused on the structur...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2024.03.25.586532</td>
<td>2024</td>
<td>3</td>
<td>Saunders HAJ, van den Berg CM, Hoogebeen R, Sc...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">31</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper discusses the implementation of an e...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1107/s2059798324000986</td>
<td>2024</td>
<td>2</td>
<td>Horstmann A, Riggs S, Chaban Y, Clare DK, de F...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">32</td>
<td>L</td>
<td>Development of new data analysis methods</td>
<td>The paper discusses developing a processing pi...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1107/s2059798324000986</td>
<td>2024</td>
<td>2</td>
<td>Horstmann A, Riggs S, Chaban Y, Clare DK, de F...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">33</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper discusses the implementation of an e...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1107/s2059798324000986</td>
<td>2024</td>
<td>2</td>
<td>Horstmann A, Riggs S, Chaban Y, Clare DK, de F...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">34</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper discusses the implementation of an e...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1107/s2059798324000986</td>
<td>2024</td>
<td>2</td>
<td>Horstmann A, Riggs S, Chaban Y, Clare DK, de F...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">35</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper discusses the implementation of an e...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1107/s2059798324000986</td>
<td>2024</td>
<td>2</td>
<td>Horstmann A, Riggs S, Chaban Y, Clare DK, de F...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">22</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper presents a new program called Domain...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2023.11.28.569001</td>
<td>2023</td>
<td>11</td>
<td>Gao J, Tong M, Lee C, Gaertig J, Legal T, Bui ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">23</td>
<td>L</td>
<td>Development of new data analysis methods</td>
<td>The paper presents 'DomainFit', a program for ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1101/2023.11.28.569001</td>
<td>2023</td>
<td>11</td>
<td>Gao J, Tong M, Lee C, Gaertig J, Legal T, Bui ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">48</td>
<td>K</td>
<td>Development of new CryoET imaging methods (inc...</td>
<td>The paper is an erratum that updates the autho...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.3791/6515</td>
<td>2023</td>
<td>9</td>
<td>(2023) Erratum: Preparing Lamellae from Vitre...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">49</td>
<td>K</td>
<td>Development of new CryoET imaging methods (inc...</td>
<td>The paper is an erratum that corrects the auth...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.3791/6515</td>
<td>2023</td>
<td>9</td>
<td>(2023) Erratum: Preparing Lamellae from Vitre...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">50</td>
<td>K</td>
<td>Development of new CryoET imaging methods (inc...</td>
<td>The paper addresses the correction of the auth...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.3791/6515</td>
<td>2023</td>
<td>9</td>
<td>(2023) Erratum: Preparing Lamellae from Vitre...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">9</td>
<td>D</td>
<td>Structural studies of plant cells</td>
<td>The paper is primarily focused on studying the...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1038/s41598-022-26760-4</td>
<td>2023</td>
<td>1</td>
<td>Kondo K, Yoshimi R, Apdila ET, Wakabayashi KI,...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">10</td>
<td>D</td>
<td>Structural studies of plant cells</td>
<td>The paper focuses on the effects of increased ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1038/s41598-022-26760-4</td>
<td>2023</td>
<td>1</td>
<td>Kondo K, Yoshimi R, Apdila ET, Wakabayashi KI,...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">11</td>
<td>D</td>
<td>Structural studies of plant cells</td>
<td>The paper is focused on the lipid composition ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1038/s41598-022-26760-4</td>
<td>2023</td>
<td>1</td>
<td>Kondo K, Yoshimi R, Apdila ET, Wakabayashi KI,...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">12</td>
<td>J</td>
<td>Studies of dynamics of molecular interactions ...</td>
<td>The paper primarily focuses on developing a me...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1073/pnas.2210249119</td>
<td>2022</td>
<td>10</td>
<td>Vakser IA, Grudinin S, Jenkins NW, Kundrotas P...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">13</td>
<td>J</td>
<td>Studies of dynamics of molecular interactions ...</td>
<td>The paper discusses the development of a compu...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1073/pnas.2210249119</td>
<td>2022</td>
<td>10</td>
<td>Vakser IA, Grudinin S, Jenkins NW, Kundrotas P...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">14</td>
<td>J</td>
<td>Studies of dynamics of molecular interactions ...</td>
<td>The paper primarily focuses on developing a me...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1073/pnas.2210249119</td>
<td>2022</td>
<td>10</td>
<td>Vakser IA, Grudinin S, Jenkins NW, Kundrotas P...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">42</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper focuses on the mechanistic basis of ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1371/journal.pbio.3001601</td>
<td>2022</td>
<td>4</td>
<td>Simonetti B, Guo Q, Giménez-Andrés M, Chen KE,...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">43</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper discusses the SNX27-Retromer's role ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1371/journal.pbio.3001601</td>
<td>2022</td>
<td>4</td>
<td>Simonetti B, Guo Q, Giménez-Andrés M, Chen KE,...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">20</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The abstract mentions that cryoEM has had a ma...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1093/jmicro/dfab041</td>
<td>2022</td>
<td>2</td>
<td>Kühlbrandt W. (2022) Forty years in cryoEM of ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">21</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper discusses the impact of cryoEM on th...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1093/jmicro/dfab041</td>
<td>2022</td>
<td>2</td>
<td>Kühlbrandt W. (2022) Forty years in cryoEM of ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">17</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper focuses on a framework for unsupervi...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1093/bioinformatics/btab794</td>
<td>2022</td>
<td>1</td>
<td>Bandyopadhyay H, Deng Z, Ding L, Liu S, Uddin ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">18</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper presents a novel framework for subto...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1093/bioinformatics/btab794</td>
<td>2022</td>
<td>1</td>
<td>Bandyopadhyay H, Deng Z, Ding L, Liu S, Uddin ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">19</td>
<td>L</td>
<td>Development of new data analysis methods (incl...</td>
<td>The paper presents a new framework for cross-d...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1093/bioinformatics/btab794</td>
<td>2022</td>
<td>1</td>
<td>Bandyopadhyay H, Deng Z, Ding L, Liu S, Uddin ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>J</td>
<td>Studies of dynamics of molecular interactions ...</td>
<td>The paper describes the development and use of...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1002/advs.202103498</td>
<td>2021</td>
<td>12</td>
<td>Lin J, Li Y, Wang P, Wu M, Zhu F, Zhang Y, Hou...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td>I</td>
<td>Studies of dynamic biological processes at a c...</td>
<td>The paper primarily focuses on developing a na...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1002/advs.202103498</td>
<td>2021</td>
<td>12</td>
<td>Lin J, Li Y, Wang P, Wu M, Zhu F, Zhang Y, Hou...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td>I</td>
<td>Studies of dynamic biological processes at a c...</td>
<td>The paper discusses the development of a nanog...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1002/advs.202103498</td>
<td>2021</td>
<td>12</td>
<td>Lin J, Li Y, Wang P, Wu M, Zhu F, Zhang Y, Hou...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">7</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper focuses on determining the atomic-re...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1038/s41594-020-0489-2</td>
<td>2020</td>
<td>9</td>
<td>Lu M, Russell RW, Bryer AJ, Quinn CM, Hou G, Z...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">8</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper is primarily focused on determining ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1038/s41594-020-0489-2</td>
<td>2020</td>
<td>9</td>
<td>Lu M, Russell RW, Bryer AJ, Quinn CM, Hou G, Z...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">15</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper focuses on the interaction between A...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1091/mbc.e19-11-0658</td>
<td>2020</td>
<td>2</td>
<td>Mattera R, Williamson CD, Ren X, Bonifacino JS...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">16</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper is primarily concerned with the inte...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1091/mbc.e19-11-0658</td>
<td>2020</td>
<td>2</td>
<td>Mattera R, Williamson CD, Ren X, Bonifacino JS...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">38</td>
<td>C</td>
<td>Structural studies of bacterial pathogens (suc...</td>
<td>The paper is primarily concerned with the stru...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1128/jb.00592-19</td>
<td>2020</td>
<td>1</td>
<td>Zhu S, Nishikino T, Takekawa N, Terashima H, K...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">39</td>
<td>C</td>
<td>Structural studies of bacterial pathogens</td>
<td>The paper focuses on the in situ structure of ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1128/jb.00592-19</td>
<td>2020</td>
<td>1</td>
<td>Zhu S, Nishikino T, Takekawa N, Terashima H, K...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">3</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper primarily focuses on the structural ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1016/j.cell.2018.10.056</td>
<td>2018</td>
<td>11</td>
<td>Vahey MD, Fletcher DA. (2018) Low-Fidelity Ass...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">4</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens</td>
<td>The primary concern of the paper is the morpho...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1016/j.cell.2018.10.056</td>
<td>2018</td>
<td>11</td>
<td>Vahey MD, Fletcher DA. (2018) Low-Fidelity Ass...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">51</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper primarily focuses on the structural ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.7554/elife.34271</td>
<td>2018</td>
<td>3</td>
<td>Ma X, Lu M, Gorman J, Terry DS, Hong X, Zhou Z...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">52</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens</td>
<td>The paper focuses on the structural dynamics o...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.7554/elife.34271</td>
<td>2018</td>
<td>3</td>
<td>Ma X, Lu M, Gorman J, Terry DS, Hong X, Zhou Z...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">40</td>
<td>F</td>
<td>Structural studies of transporters or transpor...</td>
<td>The paper primarily concerns the mechanism by ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1128/jvi.00641-17</td>
<td>2017</td>
<td>7</td>
<td>Huffman JB, Daniel GR, Falck-Pedersen E, Huet ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">41</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper primarily focuses on the structural ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1128/jvi.00641-17</td>
<td>2017</td>
<td>7</td>
<td>Huffman JB, Daniel GR, Falck-Pedersen E, Huet ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">5</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper focuses on the HIV-1 CA capsid prote...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1021/jacs.6b08744</td>
<td>2016</td>
<td>10</td>
<td>Zhang H, Hou G, Lu M, Ahn J, Byeon IL, Langmea...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">6</td>
<td>A</td>
<td>Structural descriptions of Viral Pathogens (su...</td>
<td>The paper focuses on the structural dynamics o...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1021/jacs.6b08744</td>
<td>2016</td>
<td>10</td>
<td>Zhang H, Hou G, Lu M, Ahn J, Byeon IL, Langmea...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">36</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper is primarily about the biology of ci...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1111/tra.12166</td>
<td>2014</td>
<td>3</td>
<td>Sengupta P, Barr MM. (2014) New insights into ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">37</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the biology of cilia and ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1111/tra.12166</td>
<td>2014</td>
<td>3</td>
<td>Sengupta P, Barr MM. (2014) New insights into ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">44</td>
<td>C</td>
<td>Structural studies of bacterial pathogens (suc...</td>
<td>This paper is classified under 'C' because it ...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1371/journal.pone.0053368</td>
<td>2013</td>
<td>1</td>
<td>Kalirai SS, Bazylinski DA, Hitchcock AP. (2013...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">45</td>
<td>H</td>
<td>Structural studies of any other organelle or s...</td>
<td>The paper focuses on the magnetic orientations...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.1371/journal.pone.0053368</td>
<td>2013</td>
<td>1</td>
<td>Kalirai SS, Bazylinski DA, Hitchcock AP. (2013...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">46</td>
<td>E</td>
<td>Structural studies of material science of non-...</td>
<td>The paper focuses on the visualization of drug...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.2217/nnm.09.56</td>
<td>2009</td>
<td>10</td>
<td>Bhirde AA, Sousa AA, Patel V, Azari AA, Gutkin...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">47</td>
<td>E</td>
<td>Structural studies of material science of non-...</td>
<td>The paper is primarily about using scanning tr...</td>
<td>TitleAbstractClassifier_OneDocAtATime_Tool__cr...</td>
<td>http://doi.org/10.2217/nnm.09.56</td>
<td>2009</td>
<td>10</td>
<td>Bhirde AA, Sousa AA, Patel V, Azari AA, Gutkin...</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="cell" data-execution_count="22">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>df[<span class="st">'explanation'</span>][<span class="dv">44</span>]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="22">
<pre><code>"This paper is classified under 'C' because it focuses on the structural study of magnetosomes in a specific bacterium, Magnetovibrio blakemorei strain MV-1, and investigates their magnetic orientation and chain formation mechanisms."</code></pre>
</div>
</div>
</section>
</section>
<section id="run-metadata-extraction-chain-over-listed-papers" class="level2">
<h2 class="anchored" data-anchor-id="run-metadata-extraction-chain-over-listed-papers">Run MetaData Extraction Chain over listed papers</h2>
<p>Here, we run various versions of the metadata extraction tool to examine performance over the cryoet dataset.</p>
<section id="get-full-text-copies-of-all-the-papers-about-cryoet" class="level4">
<h4 class="anchored" data-anchor-id="get-full-text-copies-of-all-the-papers-about-cryoet">Get full text copies of all the papers about CryoET</h4>
<div class="cell" data-execution_count="5">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>cb.agent_executor.invoke({<span class="st">'input'</span>:<span class="st">'Get full text copies of all papers in the collection with id="3".'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>

&gt; Entering new AgentExecutor chain...
{
  "action": "retrieve_full_text_for_papers_in_collection",
  "action_input": {
    "collection_id": "3"
  }
}   10.1101/2024.03.25.586532 does not exist
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=10910546&amp;format=pdf
    10.1107/s2059798324000986 no full text as PDF file.
Message: The element with the reference 71c2a41b-f6b1-45e1-8328-9b8fc22e9c63 is stale; either its node document is not the active document, or it is no longer connected to the DOM; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Stacktrace:
RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8
WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5
StaleElementReferenceError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:725:5
getKnownElement@chrome://remote/content/marionette/json.sys.mjs:401:11
deserializeJSON@chrome://remote/content/marionette/json.sys.mjs:259:20
cloneObject/result&lt;@chrome://remote/content/marionette/json.sys.mjs:53:52
cloneObject@chrome://remote/content/marionette/json.sys.mjs:53:25
deserializeJSON@chrome://remote/content/marionette/json.sys.mjs:289:16
cloneObject@chrome://remote/content/marionette/json.sys.mjs:59:24
deserializeJSON@chrome://remote/content/marionette/json.sys.mjs:289:16
json.deserialize@chrome://remote/content/marionette/json.sys.mjs:293:10
receiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:72:30

https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1107/s2059798324000986[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1107/s2059798324000986.html
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=9816115&amp;format=pdf
    10.1038/s41598-022-26760-4 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1038/s41598-022-26760-4[doi]&amp;retmode=xml
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=9565162&amp;format=pdf
    10.1073/pnas.2210249119 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1073/pnas.2210249119[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1073/pnas.2210249119.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=9038204&amp;format=pdf
    10.1371/journal.pbio.3001601 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1371/journal.pbio.3001601[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1371/journal.pbio.3001601.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=8855526&amp;format=pdf
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1093/jmicro/dfab041[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1093/jmicro/dfab041.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=10060710&amp;format=pdf
    10.1093/bioinformatics/btab794 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1093/bioinformatics/btab794[doi]&amp;retmode=xml
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=8844581&amp;format=pdf
    10.1002/advs.202103498 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1002/advs.202103498[doi]&amp;retmode=xml
Type Error: this element does not have children or attributes
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1002/advs.202103498.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=7490828&amp;format=pdf
    10.1038/s41594-020-0489-2 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1038/s41594-020-0489-2[doi]&amp;retmode=xml
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=7185972&amp;format=pdf
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1091/mbc.e19-11-0658[doi]&amp;retmode=xml
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=6989802&amp;format=pdf
    10.1128/jb.00592-19 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1128/jb.00592-19[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1128/jb.00592-19.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=6476638&amp;format=pdf
    10.1016/j.cell.2018.10.056 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1016/j.cell.2018.10.056[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1016/j.cell.2018.10.056.html
    10.7554/elife.34271 does not exist
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.7554/elife.34271[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.7554/elife.34271.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=5512264&amp;format=pdf
    10.1128/jvi.00641-17 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1128/jvi.00641-17[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1128/jvi.00641-17.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=5380593&amp;format=pdf
    10.1021/jacs.6b08744 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1021/jacs.6b08744[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1021/jacs.6b08744.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=4028063&amp;format=pdf
    10.1111/tra.12166 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1111/tra.12166[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1111/tra.12166.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=3540082&amp;format=pdf
    10.1371/journal.pone.0053368 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.1371/journal.pone.0053368[doi]&amp;retmode=xml
Error loading HTML file: /Users/gully.burns/alhazen/cryoet_tutorial/ft/10.1371/journal.pone.0053368.html
https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=2789973&amp;format=pdf
    10.2217/nnm.09.56 no full text as PDF file.
https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?api_key=d086451c882fabace54d7b049b6fb8481908&amp;db=pmc&amp;term=10.2217/nnm.09.56[doi]&amp;retmode=xml
Huridocs `pdf_paragraphs_extraction` service not running. Please download it and run it from `https://github.com/GullyBurns/pdf_paragraphs_extraction`
{'response': 'I retrieved full text papers for the collection named `CryoET Papers Tests` (id=`3).'}


&gt; Finished chain.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  0%|          | 0/20 [00:00&lt;?, ?it/s]  5%|▌         | 1/20 [00:19&lt;06:03, 19.14s/it] 10%|█         | 2/20 [00:36&lt;05:23, 17.95s/it] 15%|█▌        | 3/20 [00:47&lt;04:12, 14.82s/it] 20%|██        | 4/20 [00:47&lt;02:25,  9.11s/it]rewritetex: likely error invoking catdvi (empty output)
rewritetex: likely error invoking catdvi (empty output)
rewritetex: likely error invoking catdvi (empty output)
 25%|██▌       | 5/20 [01:20&lt;04:25, 17.67s/it] 30%|███       | 6/20 [01:42&lt;04:29, 19.23s/it] 35%|███▌      | 7/20 [02:02&lt;04:12, 19.41s/it] 40%|████      | 8/20 [02:27&lt;04:12, 21.06s/it] 45%|████▌     | 9/20 [02:46&lt;03:44, 20.45s/it] 50%|█████     | 10/20 [03:06&lt;03:25, 20.52s/it] 55%|█████▌    | 11/20 [03:55&lt;04:20, 28.99s/it] 60%|██████    | 12/20 [04:16&lt;03:32, 26.62s/it] 65%|██████▌   | 13/20 [04:38&lt;02:55, 25.14s/it] 70%|███████   | 14/20 [04:52&lt;02:11, 22.00s/it] 75%|███████▌  | 15/20 [05:48&lt;02:40, 32.05s/it] 80%|████████  | 16/20 [06:20&lt;02:07, 31.99s/it] 85%|████████▌ | 17/20 [06:33&lt;01:19, 26.47s/it] 90%|█████████ | 18/20 [06:47&lt;00:45, 22.74s/it] 95%|█████████▌| 19/20 [07:04&lt;00:21, 21.03s/it]100%|██████████| 20/20 [07:20&lt;00:00, 22.02s/it]</code></pre>
</div>
<div class="cell-output cell-output-display" data-execution_count="5">
<pre><code>{'input': 'Get full text copies of all papers in the collection with id="3".',
 'output': {'response': 'I retrieved full text papers for the collection named `CryoET Papers Tests` (id=`3).'},
 'intermediate_steps': [(AgentAction(tool='retrieve_full_text_for_papers_in_collection', tool_input={'collection_id': '3'}, log='{\n  "action": "retrieve_full_text_for_papers_in_collection",\n  "action_input": {\n    "collection_id": "3"\n  }\n}'),
   {'response': 'I retrieved full text papers for the collection named `CryoET Papers Tests` (id=`3).'})]}</code></pre>
</div>
</div>
<p>Identify which papers are in the sampled collection through their dois.</p>
<div class="cell" data-execution_count="7">
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span><span class="st">'2'</span>)  </span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>dois <span class="op">=</span> [e.<span class="bu">id</span> <span class="cf">for</span> e <span class="kw">in</span> q.<span class="bu">all</span>()]</span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>dois</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="7">
<pre><code>['doi:10.1002/1873-3468.14067',
 'doi:10.1002/2211-5463.13206',
 'doi:10.1002/2211-5463.13473',
 'doi:10.1002/bies.202200060',
 'doi:10.1002/cnr2.1185',
 'doi:10.1002/jev2.12224',
 'doi:10.1002/mrd.23346',
 'doi:10.1002/pld3.271',
 'doi:10.1002/pmic.202000198',
 'doi:10.1002/pro.2504',
 'doi:10.1002/pro.3018',
 'doi:10.1002/pro.3513',
 'doi:10.1002/pro.3710',
 'doi:10.1002/pro.3805',
 'doi:10.1002/pro.3967',
 'doi:10.1002/pro.4038',
 'doi:10.1002/pro.4191',
 'doi:10.1002/pro.4472',
 'doi:10.1002/pro.4792',
 'doi:10.1002/smll.202301838',
 'doi:10.1007/82_2021_243',
 'doi:10.1007/978-3-030-57821-3_8',
 'doi:10.1007/s00418-023-02191-8',
 'doi:10.1007/s10278-018-0101-z',
 'doi:10.1007/s10974-019-09537-7',
 'doi:10.1007/s11274-024-03891-6',
 'doi:10.1007/s11426-022-1408-5',
 'doi:10.1007/s12154-009-0033-7',
 'doi:10.1007/s12551-022-01013-w',
 'doi:10.1007/s12551-023-01049-6',
 'doi:10.1007/s13238-021-00895-y',
 'doi:10.1007/s40484-019-0191-8',
 'doi:10.1007/s41048-017-0040-0',
 'doi:10.1007/s42242-022-00226-y',
 'doi:10.1016/bs.mcb.2023.01.003',
 'doi:10.1016/j.apsb.2023.08.010',
 'doi:10.1016/j.asoc.2023.110210',
 'doi:10.1016/j.bbadva.2024.100113',
 'doi:10.1016/j.biotechadv.2021.107753',
 'doi:10.1016/j.bpr.2021.100028',
 'doi:10.1016/j.ceb.2024.102356',
 'doi:10.1016/j.cell.2018.10.056',
 'doi:10.1016/j.cell.2021.12.015',
 'doi:10.1016/j.cell.2023.03.015',
 'doi:10.1016/j.celrep.2021.110052',
 'doi:10.1016/j.cels.2021.05.014',
 'doi:10.1016/j.chom.2023.03.004',
 'doi:10.1016/j.cmpb.2022.106871',
 'doi:10.1016/j.cmpb.2022.106990',
 'doi:10.1016/j.cmpb.2022.107017',
 'doi:10.1016/j.coviro.2021.03.005',
 'doi:10.1016/j.crmeth.2023.100557',
 'doi:10.1016/j.csbj.2020.05.002',
 'doi:10.1016/j.csbj.2020.11.052',
 'doi:10.1016/j.csbj.2021.04.019',
 'doi:10.1016/j.cub.2022.04.024',
 'doi:10.1016/j.genrep.2021.101122',
 'doi:10.1016/j.isci.2021.102959',
 'doi:10.1016/j.isci.2021.103402',
 'doi:10.1016/j.isci.2021.103458',
 'doi:10.1016/j.isci.2024.108976',
 'doi:10.1016/j.jbc.2021.100555',
 'doi:10.1016/j.jbc.2021.100557',
 'doi:10.1016/j.jbc.2021.100559',
 'doi:10.1016/j.jbc.2021.100560',
 'doi:10.1016/j.jbc.2021.100938',
 'doi:10.1016/j.jbc.2022.101727',
 'doi:10.1016/j.jbc.2023.105182',
 'doi:10.1016/j.jbc.2023.105608',
 'doi:10.1016/j.jmb.2021.167187',
 'doi:10.1016/j.jmb.2021.167351',
 'doi:10.1016/j.jmb.2022.167774',
 'doi:10.1016/j.jmb.2023.168051',
 'doi:10.1016/j.jmb.2023.168059',
 'doi:10.1016/j.jmb.2023.168068',
 'doi:10.1016/j.jneumeth.2011.03.013',
 'doi:10.1016/j.jpi.2023.100298',
 'doi:10.1016/j.jprot.2020.103777',
 'doi:10.1016/j.jsb.2008.04.006',
 'doi:10.1016/j.jsb.2011.05.011',
 'doi:10.1016/j.jsb.2017.02.007',
 'doi:10.1016/j.jsb.2018.01.012',
 'doi:10.1016/j.jsb.2021.107727',
 'doi:10.1016/j.jsb.2021.107804',
 'doi:10.1016/j.jsb.2021.107808',
 'doi:10.1016/j.jsb.2022.107872',
 'doi:10.1016/j.jsb.2022.107897',
 'doi:10.1016/j.jsb.2022.107906',
 'doi:10.1016/j.jsb.2023.107990',
 'doi:10.1016/j.mcpro.2023.100657',
 'doi:10.1016/j.mib.2017.09.014',
 'doi:10.1016/j.mib.2017.10.005',
 'doi:10.1016/j.molcel.2021.05.023',
 'doi:10.1016/j.molcel.2021.08.020',
 'doi:10.1016/j.molcel.2021.12.022',
 'doi:10.1016/j.molcel.2021.12.027',
 'doi:10.1016/j.molcel.2022.06.032',
 'doi:10.1016/j.nantod.2022.101665',
 'doi:10.1016/j.nbd.2020.105088',
 'doi:10.1016/j.neuint.2017.04.009',
 'doi:10.1016/j.sbi.2019.05.011',
 'doi:10.1016/j.sbi.2019.06.012',
 'doi:10.1016/j.sbi.2022.102488',
 'doi:10.1016/j.sbi.2022.102523',
 'doi:10.1016/j.sbi.2023.102565',
 'doi:10.1016/j.sbi.2024.102795',
 'doi:10.1016/j.str.2019.01.005',
 'doi:10.1016/j.str.2020.07.017',
 'doi:10.1016/j.str.2021.04.010',
 'doi:10.1016/j.str.2021.12.010',
 'doi:10.1016/j.str.2022.01.006',
 'doi:10.1016/j.str.2022.02.014',
 'doi:10.1016/j.str.2023.05.002',
 'doi:10.1016/j.str.2023.08.015',
 'doi:10.1016/j.str.2023.09.009',
 'doi:10.1016/j.tcsw.2024.100122',
 'doi:10.1016/j.tibs.2017.12.006',
 'doi:10.1016/j.tibs.2021.10.005',
 'doi:10.1016/j.tibs.2021.10.007',
 'doi:10.1016/j.tibs.2024.01.011',
 'doi:10.1016/j.ultramic.2023.113860',
 'doi:10.1016/j.ultramic.2023.113882',
 'doi:10.1016/j.virusres.2014.09.010',
 'doi:10.1016/j.xinn.2023.100539',
 'doi:10.1016/j.yjsbx.2019.100013',
 'doi:10.1016/j.yjsbx.2020.100041',
 'doi:10.1016/j.yjsbx.2022.100063',
 'doi:10.1016/j.yjsbx.2022.100065',
 'doi:10.1016/j.yjsbx.2022.100076',
 'doi:10.1016/j.yjsbx.2023.100094',
 'doi:10.1017/qrd.2020.15',
 'doi:10.1017/s1431927622005165',
 'doi:10.1017/s2633903x22000046',
 'doi:10.1017/s2633903x23000107',
 'doi:10.1017/s2633903x24000035',
 'doi:10.1017/s2633903x24000047',
 'doi:10.1021/acs.chemrev.1c00837',
 'doi:10.1021/acs.chemrev.1c00936',
 'doi:10.1021/acs.chemrev.1c00937',
 'doi:10.1021/acs.chemrev.2c00608',
 'doi:10.1021/acs.chemrev.3c00436',
 'doi:10.1021/acs.chemrev.8b00451',
 'doi:10.1021/acs.chemrev.8b00460',
 'doi:10.1021/acs.jproteome.0c00764',
 'doi:10.1021/acsnano.0c09563',
 'doi:10.1021/acsomega.3c00944',
 'doi:10.1021/jacsau.3c00712',
 'doi:10.1038/nmeth.4405',
 'doi:10.1038/nrm.2017.103',
 'doi:10.1038/s41392-022-00916-0',
 'doi:10.1038/s41392-023-01649-4',
 'doi:10.1038/s41421-021-00281-2',
 'doi:10.1038/s41421-023-00606-3',
 'doi:10.1038/s41467-020-18952-1',
 'doi:10.1038/s41467-021-21377-z',
 'doi:10.1038/s41467-021-23351-1',
 'doi:10.1038/s41467-021-26329-1',
 'doi:10.1038/s41467-021-26641-w',
 'doi:10.1038/s41467-021-26684-z',
 'doi:10.1038/s41467-021-27146-2',
 'doi:10.1038/s41467-022-30116-x',
 'doi:10.1038/s41467-022-31748-9',
 'doi:10.1038/s41467-022-33483-7',
 'doi:10.1038/s41467-022-33957-8',
 'doi:10.1038/s41467-022-34972-5',
 'doi:10.1038/s41467-023-36096-w',
 'doi:10.1038/s41467-023-36372-9',
 'doi:10.1038/s41467-023-37868-0',
 'doi:10.1038/s41467-023-38539-w',
 'doi:10.1038/s41467-023-39172-3',
 'doi:10.1038/s41467-023-42085-w',
 'doi:10.1038/s41467-023-44442-1',
 'doi:10.1038/s41467-024-45429-2',
 'doi:10.1038/s41467-024-46041-0',
 'doi:10.1038/s41564-023-01333-x',
 'doi:10.1038/s41565-023-01321-6',
 'doi:10.1038/s41570-018-0148',
 'doi:10.1038/s41576-023-00587-9',
 'doi:10.1038/s41586-019-1089-3',
 'doi:10.1038/s41586-022-04859-y',
 'doi:10.1038/s41586-023-05990-0',
 'doi:10.1038/s41586-023-06140-2',
 'doi:10.1038/s41592-018-0167-z',
 'doi:10.1038/s41592-019-0396-9',
 'doi:10.1038/s41592-019-0580-y',
 'doi:10.1038/s41592-019-0585-6',
 'doi:10.1038/s41592-019-0591-8',
 'doi:10.1038/s41592-021-01166-8',
 'doi:10.1038/s41592-021-01220-5',
 'doi:10.1038/s41592-021-01275-4',
 'doi:10.1038/s41592-022-01746-2',
 'doi:10.1038/s41592-023-01878-z',
 'doi:10.1038/s41592-023-01999-5',
 'doi:10.1038/s41592-023-02045-0',
 'doi:10.1038/s41592-023-02053-0',
 'doi:10.1038/s41592-023-02082-9',
 'doi:10.1038/s41592-024-02210-z',
 'doi:10.1038/s41594-020-00530-0',
 'doi:10.1038/s41594-021-00716-0',
 'doi:10.1038/s41594-022-00770-2',
 'doi:10.1038/s41594-022-00863-y',
 'doi:10.1038/s41597-023-02280-2',
 'doi:10.1038/s41598-020-60598-y',
 'doi:10.1038/s41598-020-66793-1',
 'doi:10.1038/s41598-021-86770-6',
 'doi:10.1038/s41598-023-30045-9',
 'doi:10.1038/s41598-023-37037-9',
 'doi:10.1038/s41598-023-38027-7',
 'doi:10.1038/s41598-023-39829-5',
 'doi:10.1038/s41929-023-00947-y',
 'doi:10.1038/s42003-021-02360-2',
 'doi:10.1038/s42003-022-03009-4',
 'doi:10.1038/s42003-023-04850-x',
 'doi:10.1038/s42003-023-04916-w',
 'doi:10.1038/s43586-022-00131-9',
 'doi:10.1038/s44318-023-00023-y',
 'doi:10.1038/s44320-024-00016-x',
 'doi:10.1039/d2dd00128d',
 'doi:10.1039/d2fd00022a',
 'doi:10.1039/d2fd00076h',
 'doi:10.1039/d2fd00081d',
 'doi:10.1039/d3ra06479d',
 'doi:10.1042/bcj20200715',
 'doi:10.1042/bcj20210375',
 'doi:10.1042/bst20210444',
 'doi:10.1042/bst20210674',
 'doi:10.1042/bst20210706',
 'doi:10.1042/bst20220194',
 'doi:10.1042/bst20220221',
 'doi:10.1042/bst20221246',
 'doi:10.1042/bst20230103',
 'doi:10.1042/ebc20220168',
 'doi:10.1042/etls20220090',
 'doi:10.1063/4.0000177',
 'doi:10.1063/5.0080480',
 'doi:10.1073/pnas.1920323117',
 'doi:10.1073/pnas.2000700117',
 'doi:10.1073/pnas.2104559118',
 'doi:10.1073/pnas.2105004118',
 'doi:10.1073/pnas.2108738118',
 'doi:10.1073/pnas.2111661119',
 'doi:10.1073/pnas.2117245119',
 'doi:10.1073/pnas.2119467119',
 'doi:10.1073/pnas.2210249119',
 'doi:10.1073/pnas.2213149120',
 'doi:10.1073/pnas.2218010119',
 'doi:10.1073/pnas.2221712120',
 'doi:10.1073/pnas.2309984121',
 'doi:10.1073/pnas.2310469121',
 'doi:10.1073/pnas.2313816120',
 'doi:10.1080/01677063.2016.1253693',
 'doi:10.1080/09537104.2020.1763939',
 'doi:10.1080/15476286.2023.2237719',
 'doi:10.1080/19491034.2020.1806661',
 'doi:10.1080/19491034.2023.2178201',
 'doi:10.1083/jcb.202104100',
 'doi:10.1083/jcb.202208062',
 'doi:10.1083/jcb.202309125',
 'doi:10.1085/jgp.201812227',
 'doi:10.1089/cmb.2021.0606',
 'doi:10.1091/mbc.e18-02-0087',
 'doi:10.1091/mbc.e18-05-0329',
 'doi:10.1093/bib/bbae113',
 'doi:10.1093/bioadv/vbad131',
 'doi:10.1093/bioinformatics/btaa671',
 'doi:10.1093/bioinformatics/btab123',
 'doi:10.1093/bioinformatics/btab511',
 'doi:10.1093/bioinformatics/btab794',
 'doi:10.1093/bioinformatics/btac052',
 'doi:10.1093/bioinformatics/btx653',
 'doi:10.1093/bioinformatics/btz323',
 'doi:10.1093/gigascience/giz077',
 'doi:10.1093/jmicro/dfab041',
 'doi:10.1093/jmicro/dfad024',
 'doi:10.1093/micmic/ozad067.483',
 'doi:10.1093/micmic/ozad083',
 'doi:10.1093/nar/gkac1062',
 'doi:10.1093/nar/gkac1077',
 'doi:10.1093/nar/gkac1208',
 'doi:10.1093/nar/gkad1019',
 'doi:10.1093/pnasnexus/pgac183',
 'doi:10.1098/rsob.200186',
 'doi:10.1098/rsob.220079',
 'doi:10.1098/rsos.221426',
 'doi:10.1099/mic.0.001165',
 'doi:10.1099/mic.0.001311',
 'doi:10.1099/mic.0.001328',
 'doi:10.1099/mic.0.001438',
 'doi:10.1101/2020.04.15.042747',
 'doi:10.1101/2020.11.05.370239',
 'doi:10.1101/2020.11.19.390187',
 'doi:10.1101/2020.11.25.398008',
 'doi:10.1101/2021.02.25.432265',
 'doi:10.1101/2021.05.16.444381',
 'doi:10.1101/2021.07.17.452128',
 'doi:10.1101/2021.09.01.458650',
 'doi:10.1101/2021.10.09.463779',
 'doi:10.1101/2021.10.26.465776',
 'doi:10.1101/2021.11.12.468428',
 'doi:10.1101/2021.12.08.471777',
 'doi:10.1101/2021.12.09.472010',
 'doi:10.1101/2022.01.12.476068',
 'doi:10.1101/2022.02.11.480155',
 'doi:10.1101/2022.03.01.480844',
 'doi:10.1101/2022.03.10.483789',
 'doi:10.1101/2022.03.26.485912',
 'doi:10.1101/2022.04.12.488077',
 'doi:10.1101/2022.06.23.497320',
 'doi:10.1101/2022.07.11.499538',
 'doi:10.1101/2022.08.01.502333',
 'doi:10.1101/2022.08.11.503617',
 'doi:10.1101/2022.08.16.502515',
 'doi:10.1101/2022.09.21.508877',
 'doi:10.1101/2022.10.04.510785',
 'doi:10.1101/2022.12.15.520632',
 'doi:10.1101/2023.01.30.526190',
 'doi:10.1101/2023.02.25.529934',
 'doi:10.1101/2023.03.06.531305',
 'doi:10.1101/2023.04.12.536539',
 'doi:10.1101/2023.04.28.538636',
 'doi:10.1101/2023.05.26.542411',
 'doi:10.1101/2023.05.31.542975',
 'doi:10.1101/2023.06.04.543601',
 'doi:10.1101/2023.07.12.548777',
 'doi:10.1101/2023.08.18.553799',
 'doi:10.1101/2023.08.21.554193',
 'doi:10.1101/2023.09.25.559261',
 'doi:10.1101/2023.10.03.560672',
 'doi:10.1101/2023.11.28.569001',
 'doi:10.1101/2023.12.14.571776',
 'doi:10.1101/2023.12.21.572824',
 'doi:10.1101/2024.01.05.574336',
 'doi:10.1101/2024.02.17.580557',
 'doi:10.1101/2024.02.26.582080',
 'doi:10.1101/2024.03.20.585905',
 'doi:10.1101/2024.03.25.586532',
 'doi:10.1101/2024.04.03.587742',
 'doi:10.1101/2024.04.12.589090',
 'doi:10.1101/838920',
 'doi:10.1107/s2052252521011088',
 'doi:10.1107/s2052252522009721',
 'doi:10.1107/s2052252523004943',
 'doi:10.1107/s2053230x23004934',
 'doi:10.1107/s2059798317016709',
 'doi:10.1107/s205979832101278x',
 'doi:10.1107/s2059798322001966',
 'doi:10.1107/s2059798323004497',
 'doi:10.1107/s2059798324001815',
 'doi:10.1109/cvpr42600.2020.00413',
 'doi:10.1109/cvpr52688.2022.01999',
 'doi:10.1109/iccv48922.2021.00283',
 'doi:10.1109/iccv48922.2021.00383',
 'doi:10.1109/icip42928.2021.9506205',
 'doi:10.1109/icip46576.2022.9898002',
 'doi:10.1109/memb.2007.364938',
 'doi:10.1109/msp.2021.3119273',
 'doi:10.1109/tcbb.2021.3065986',
 'doi:10.1109/tci.2022.3174801',
 'doi:10.1109/tmi.2021.3066318',
 'doi:10.1109/tvcg.2022.3186146',
 'doi:10.1111/bpa.12675',
 'doi:10.1111/cmi.13319',
 'doi:10.1111/febs.16568',
 'doi:10.1111/jcmm.17103',
 'doi:10.1111/mmi.13709',
 'doi:10.1111/pbi.13929',
 'doi:10.1111/pbi.14244',
 'doi:10.1117/1.jbo.28.9.096005',
 'doi:10.1126/sciadv.abc8262',
 'doi:10.1126/sciadv.abm6570',
 'doi:10.1126/sciadv.adj1640',
 'doi:10.1126/sciadv.adk1954',
 'doi:10.1126/sciadv.adl1126',
 'doi:10.1126/science.abm9326',
 'doi:10.1128/cmr.00014-14',
 'doi:10.1128/iai.00834-10',
 'doi:10.1128/mbio.00631-23',
 'doi:10.1128/mbio.00793-23',
 'doi:10.1128/mbio.01423-23',
 'doi:10.1128/msphere.00944-21',
 'doi:10.1128/spectrum.01290-22',
 'doi:10.1146/annurev-biochem-032620-110705',
 'doi:10.1146/annurev-biophys-091720-102019',
 'doi:10.1146/annurev-biophys-111622-091327',
 'doi:10.1146/annurev-biophys-111821-104732',
 'doi:10.1155/2011/723283',
 'doi:10.1177/10943420221128233',
 'doi:10.1177/25152564241231364',
 'doi:10.1186/s12859-019-2650-7',
 'doi:10.1186/s12859-019-3003-2',
 'doi:10.1186/s12859-020-03660-w',
 'doi:10.1186/s12859-020-03680-6',
 'doi:10.1186/s12859-022-04901-w',
 'doi:10.1186/s12905-023-02443-8',
 'doi:10.1186/s12915-023-01795-9',
 'doi:10.1186/s12919-020-00191-3',
 'doi:10.1242/jcs.259234',
 'doi:10.1242/jcs.260813',
 'doi:10.12688/f1000research.25097.1',
 'doi:10.12703/r-01-000004',
 'doi:10.12703/r/10-14',
 'doi:10.1371/journal.pbio.3001319',
 'doi:10.1371/journal.pbio.3001601',
 'doi:10.1371/journal.pbio.3002005',
 'doi:10.1371/journal.pcbi.1000955',
 'doi:10.1371/journal.pcbi.1006958',
 'doi:10.1371/journal.pcbi.1008227',
 'doi:10.1371/journal.pone.0027269',
 'doi:10.1523/jneurosci.0743-19.2019',
 'doi:10.15252/embj.2020105789',
 'doi:10.15252/embj.2022112440',
 'doi:10.15252/embj.2022113010',
 'doi:10.15252/embr.202255719',
 'doi:10.15252/embr.202356766',
 'doi:10.15252/msb.20188601',
 'doi:10.15252/msb.202110442',
 'doi:10.15252/msb.202311799',
 'doi:10.21203/rs.3.rs-1182359/v1',
 'doi:10.21203/rs.3.rs-1336182/v1',
 'doi:10.21203/rs.3.rs-134794/v1',
 'doi:10.21203/rs.3.rs-1493068/v1',
 'doi:10.21203/rs.3.rs-2245843/v1',
 'doi:10.21203/rs.3.rs-2500102/v1',
 'doi:10.2139/ssrn.3733984',
 'doi:10.2144/fsoa-2022-0048',
 'doi:10.22541/au.169261941.10272122/v1',
 'doi:10.2741/4794',
 'doi:10.3389/fbinf.2021.710119',
 'doi:10.3389/fcell.2020.588556',
 'doi:10.3389/fcell.2022.819534',
 'doi:10.3389/fcell.2022.842342',
 'doi:10.3389/fcell.2023.1052245',
 'doi:10.3389/fcell.2023.1245939',
 'doi:10.3389/fchem.2022.889203',
 'doi:10.3389/fchem.2023.1106495',
 'doi:10.3389/fchem.2023.1171240',
 'doi:10.3389/fchem.2023.1200409',
 'doi:10.3389/fcimb.2022.794264',
 'doi:10.3389/fcimb.2022.908241',
 'doi:10.3389/fcimb.2023.1135013',
 'doi:10.3389/fcimb.2023.1305902',
 'doi:10.3389/fgene.2022.845305',
 'doi:10.3389/fmicb.2021.692707',
 'doi:10.3389/fmicb.2022.911114',
 'doi:10.3389/fmolb.2020.613347',
 'doi:10.3389/fmolb.2021.658269',
 'doi:10.3389/fmolb.2021.660542',
 'doi:10.3389/fmolb.2021.804476',
 'doi:10.3389/fmolb.2021.830304',
 'doi:10.3389/fmolb.2022.1041373',
 'doi:10.3389/fmolb.2022.1041386',
 'doi:10.3389/fmolb.2022.761173',
 'doi:10.3389/fmolb.2022.906437',
 'doi:10.3389/fmolb.2022.931949',
 'doi:10.3389/fmolb.2022.934465',
 'doi:10.3389/fmolb.2022.960940',
 'doi:10.3389/fmolb.2023.1252529',
 'doi:10.3389/fmolb.2023.1296941',
 'doi:10.3389/fmolb.2024.1342179',
 'doi:10.3389/fncel.2023.1132241',
 'doi:10.3389/fnins.2020.00914',
 'doi:10.3389/fnins.2022.881534',
 'doi:10.3389/fnsyn.2022.1024073',
 'doi:10.3389/fphys.2022.760404',
 'doi:10.3389/fphys.2022.957484',
 'doi:10.3389/fpls.2022.967203',
 'doi:10.3390/biom10040629',
 'doi:10.3390/biom11111607',
 'doi:10.3390/biom12081022',
 'doi:10.3390/biom12101425',
 'doi:10.3390/biom13030450',
 'doi:10.3390/cimb45040240',
 'doi:10.3390/e24121813',
 'doi:10.3390/genes10070493',
 'doi:10.3390/ijms21082843',
 'doi:10.3390/ijms22031308',
 'doi:10.3390/ijms22126177',
 'doi:10.3390/ijms232415917',
 'doi:10.3390/ijms241713375',
 'doi:10.3390/ijms242316781',
 'doi:10.3390/ijms242417183',
 'doi:10.3390/ijms25010020',
 'doi:10.3390/ijms25052850',
 'doi:10.3390/life11080842',
 'doi:10.3390/life12081267',
 'doi:10.3390/mi14091674',
 'doi:10.3390/molecules24030637',
 'doi:10.3390/molecules25010082',
 'doi:10.3390/molecules25184262',
 'doi:10.3390/molecules26206224',
 'doi:10.3390/molecules28186538',
 'doi:10.3390/molecules28207176',
 'doi:10.3390/nano13121839',
 'doi:10.3390/pathogens10091108',
 'doi:10.3390/pathogens11030368',
 'doi:10.3390/s22145292',
 'doi:10.3390/v11050405',
 'doi:10.3390/v13040611',
 'doi:10.3390/v13122368',
 'doi:10.3402/jev.v1i0.18182',
 'doi:10.3402/jev.v4.27783',
 'doi:10.34133/2021/9892152',
 'doi:10.3724/abbs.2022115',
 'doi:10.3791/56162',
 'doi:10.3791/63923',
 'doi:10.3791/64435',
 'doi:10.52601/bpr.2023.230008',
 'doi:10.7554/elife.60049',
 'doi:10.7554/elife.63756',
 'doi:10.7554/elife.68946',
 'doi:10.7554/elife.78133',
 'doi:10.7554/elife.78385',
 'doi:10.7554/elife.78823',
 'doi:10.7554/elife.79272',
 'doi:10.7554/elife.80111',
 'doi:10.7554/elife.83623',
 'doi:10.7554/elife.88579']</code></pre>
</div>
</div>
<p>Iterate over those dois and extract 15 metadata variables based on the questions shown in <code>./local_resources/prompt_elements/metadata_extraction.yaml</code></p>
<div class="cell" data-execution_count="8">
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Get the metadata extraction tool</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>t2 <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, MetadataExtraction_MethodsSectionOnly_Tool)][<span class="dv">0</span>]</span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a dataframe to store previously extracted metadata</span></span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a><span class="co">#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:</span></span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame()</span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d <span class="kw">in</span> [d <span class="cf">for</span> d <span class="kw">in</span> dois]:</span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>    item_types <span class="op">=</span> <span class="bu">set</span>()</span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>    l <span class="op">=</span> t2.read_metadata_extraction_notes(d, <span class="st">'cryoet'</span>, <span class="st">'test'</span>)</span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, pd.DataFrame(l)]) </span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a>     </span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Iterate over papers to run the metadata extraction tool</span></span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a><span class="co">#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:</span></span>
<span id="cb35-14"><a href="#cb35-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d <span class="kw">in</span> [d <span class="cf">for</span> d <span class="kw">in</span> dois]:</span>
<span id="cb35-15"><a href="#cb35-15" aria-hidden="true" tabindex="-1"></a>    item_types <span class="op">=</span> <span class="bu">set</span>()</span>
<span id="cb35-16"><a href="#cb35-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-17"><a href="#cb35-17" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Skip if the doi is already in the database</span></span>
<span id="cb35-18"><a href="#cb35-18" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> <span class="bu">len</span>(df)<span class="op">&gt;</span><span class="dv">0</span> <span class="kw">and</span> d <span class="kw">in</span> df.doi.unique():</span>
<span id="cb35-19"><a href="#cb35-19" aria-hidden="true" tabindex="-1"></a>        <span class="cf">continue</span></span>
<span id="cb35-20"><a href="#cb35-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-21"><a href="#cb35-21" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Run the metadata extraction tool on the doi</span></span>
<span id="cb35-22"><a href="#cb35-22" aria-hidden="true" tabindex="-1"></a>    t2.run(tool_input<span class="op">=</span>{<span class="st">'paper_id'</span>: d, <span class="st">'extraction_type'</span>: <span class="st">'cryoet'</span>, <span class="st">'run_label'</span>: <span class="st">'test'</span>})</span>
<span id="cb35-23"><a href="#cb35-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-24"><a href="#cb35-24" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Add the results to the dataframe    </span></span>
<span id="cb35-25"><a href="#cb35-25" aria-hidden="true" tabindex="-1"></a>    l2 <span class="op">=</span> t2.read_metadata_extraction_notes(d, <span class="st">'cryoet'</span>, <span class="st">'test'</span>)</span>
<span id="cb35-26"><a href="#cb35-26" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, pd.DataFrame(l2)]) </span>
<span id="cb35-27"><a href="#cb35-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-28"><a href="#cb35-28" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Materials and Methods\n\nModeling Paradigm.\nOur approach was to dramatically speed up the sampling of the intermolecular energy landscape by skipping the low-probability (high-energy) states, focusing only on the set of high-probability (low-energy) states corresponding to the energy minima. The \"minima hopping\" paradigm has been widely used since the early days of molecular modeling for the sampling of the energy landscapes of biomolecules, such as conformational analysis of biopolymers (&lt;&lt;REF:Galaktionov-1988-33-556&gt;&gt;), rotamer libraries (&lt;&lt;REF:Kirys-2012-80-2089&gt;&gt;), and refinement of protein-protein interfaces (&lt;&lt;REF:Dauzhenka-2018-39-2012&gt;&gt;), providing extraordinary savings of computing time by avoiding travel in low-probability areas of the landscape. Markov state models have been used to study protein folding, dynamics (&lt;&lt;REF:Shukla-2015-48-414&gt;&gt;), and association (&lt;&lt;REF:He-2021-154-084101&gt;&gt;) by representing the energy landscape by a set of the energy minima and the probabilities of transition between them. In this study we use a similar idea, namely a Markov state MC approach to sampling transitions between low-energy states, to perform very long trajectory simulations of large systems of proteins at atomic resolution.\n\nMolecular Systems.\nSimulations were performed on three different sets of proteins. To determine the volume fraction of the system, for each protein, the volume was calculated by the 3V server (3vee.molmovdb.org) (&lt;&lt;REF:Voss-2010-38-W555-62&gt;&gt;)\nSet 1. Five arbitrarily selected globular proteins of average size to represent a \"typical\" crowded cellular environment (hereafter called \"5 mix\" set; Fig. 1A &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1).\nSet 2. Set 1 plus green fluorescent protein (\"GFP + 5 mix\" set; Fig. 1B &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1) for comparison with the experimental data on GFP diffusion.\nSet 3. Three small proteins (\"3 mix\" set; Fig. 1C &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1) from Feig and coworkers (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;) representing the nonmembrane part of that study: ubiquitin, G protein B subunit, and villin.\nFig. 1.\nMolecular systems used in the study. (A) Five arbitrarily selected globular proteins of average size to represent a typical crowded cellular environment (PDB: 1mat, 1g81, 3chy, 1jxb, and 1cm2). (B) GFP (1ema). (C) Three small proteins: ubiquitin (1ubq), G protein B subunit (1pga), and villin (1vii). Molecular images were obtained using PyMOL version 2.5 (Schrodinger).\n\nGeneration of the Initial State.\nFor the starting point of the simulation, the proteins were placed on a cubical grid of a preset size, with the step of the grid calculated according to the desired protein volume fraction. In this study, we used a 500 x 500 x 500-A3 grid (the linear dimension approximately half of that of the smallest cell--Mycoplasma genitalium) with periodic boundary conditions. Each protein had an equal share of copies (e.g., in the 5 mix set of the 5-proteins mixture, each protein had a 1 in 5 share of copies). The total number of protein copies and the step of the grid were calculated according to the preset protein volume fraction V. In this study, we used a range of volume fraction values, from V = 0.10 to close to physiological V = 0.30. SI Appendix, Table S2 shows the total number of molecular copies corresponding to each volume fraction.\nThe proteins were placed in a random order. They were randomly rotated and translated within half of the grid step interval. No collision check was applied at this stage since the collisions were eliminated at the start of the simulation. SI Appendix, Fig. S1 shows a fragment of the initial state of the system before the start of the simulation.\n\nSimulation Protocol.\nAn MC procedure was developed to simulate the cellular environment with proteins in rigid-body approximation, using an all-atom representation. The procedure by design is based on proteins transitioning between different protein-protein associations. Thus, our approach applies to crowded protein environments only, in which proteins encounter each other in close proximity and monomeric states (the absence of all protein-protein interactions, including transient) are uncommon. The energy landscape of the system is represented by our GRAMM FFT docking (&lt;&lt;REF:Katchalski-Katzir-1992-89-2195&gt;&gt;) scores/energies, based on the step function approximation of the Lennard-Jones potential (&lt;&lt;REF:Vakser-1996-9-37&gt;&gt;). In this representation, the docking poses (including the multiplicity of transient encounters) correspond to negative energy values, and the monomeric states (i.e., the barriers between the minima) have energy zero.\nThe position of each protein is described with the 3 x 3 rotation matrix and the translation vector relative to the origin of the coordinate system. Protein-protein docking poses are systematically precomputed for all of the rotations and translations of each protein, relative to all of the other proteins in the system by GRAMM docking, unscored and unrefined, at intermediate resolution, previously optimized for the docking of unbound proteins (&lt;&lt;REF:Ruvinsky-2008-95-2150&gt;&gt;) (grid step 3.5 A, repulsion 9.0, and rotation interval 10degrees). For proteins A and B, both docking combinations A-B (A is the ligand and B is the receptor) and B-A (B, ligand; A, receptor) are precalculated. Thus, for example, for the 5 mix set the number of precalculated docking outputs is 25 (5 x 5). If A is the moving molecule (ligand), its new putative energy is taken from the A-B docking (and vice versa).\nThe docking results are stored on 6D grids (3 translations and 3 rotations), accessed during the MC runs. The MC move is initiated by a random selection of a protein (\"ligand\") considered for a move to proteins (\"receptors\") within a certain neighborhood (described below) from the current position of the ligand. The receptor to move to is selected randomly among all of the neighborhood proteins. Our minima-hopping paradigm, based on the approximation of the Lennard-Jones potential (see above), assumes only the short-distance interactions between the immediate docking partners. The presence of the neighboring proteins not selected for this move is accounted for by the detailed balance condition in the Metropolis acceptance criterion (described below). Once the ligand and the receptor are selected, the move is chosen randomly among the precalculated 30,000 lowest energy-docking matches for that ligand-receptor pair.\nThe simulation step is completed when all of the proteins have attempted to move. Once the ligand moves, the energy (GRAMM docking score) of the new match is added to the energy of the ligand and the energy of the old match it detaches from is subtracted. Correspondingly, the energy of the new receptor adds the energy of that new match, and the energy from the old receptor (the one the ligand is detaching from) subtracts the energy of the detaching docking match.\nThe move is accepted or rejected based on the Metropolis acceptance criterion (detailed balance condition). Ligands (L) are allowed to move to the neighboring receptors (R) only (randomly selected among all neighboring proteins), defined as those within the distance between R and L geometric centers less than the sum of the R and L radii, plus 50 A, to accommodate binding to the first layer of receptors in the crowded environment. Collision check is performed for each attempted move according to the Calpha-Calpha minimal distance of 8 A. The moves resulting in collision are rejected. Fig. 2 &lt;&lt;FIG:fig02&gt;&gt; illustrates the general principle of the move set. Periodic boundary conditions are introduced. Temperature is a parameter to be adjusted for an adequate acceptance rate.\nFig. 2.\nThe simulation move set. Docking results are precalculated and stored on 6D grids, accessed during the MC runs. The move set includes a move of one protein (L, ligand) at a time to a putative docking match with another protein (R, receptor) in the vicinity of the ligand. The energies of the states are set according to the docking scores. The move is accepted or rejected based on the Metropolis criterion (detailed balance condition).\nThe detailed balance condition for the system was implemented. The probability Pij of move from step i to step j had to be the same as Pji from j to i. Accordingly, the Metropolis criterion was normalized (&lt;&lt;REF:Hastings-1970-57-97&gt;&gt;) a\n[1]\nAs noted above, in our system, the monomeric states have energy zero, and all of the minima have negative energy values. Our model assumes no additional barriers between states i and j. We also assume the same curvature of the potential wells of each state. Thus, in the Kramers' (or Arrhenius) rate equation, which for our system can be written a\n[2]\nThe observed parameters of the simulation (per simulation step) were potential energy E, the average energy of a molecule (the sum of all molecules' energies--GRAMM docking scores--divided by the number of molecules); the shift (the average length of a molecule's move per simulation step); the MSD (the average mean square deviation of a molecule's geometric center after unwrapping coordinates from the periodic boundary conditions); acceptance rate (percentage of accepted moves); and the aggregation number Nc (the average number of proteins in an aggregate/oligomer formed by docked proteins). To allow off-the-grid relaxation of the system, the reference position for MSD calculation was set at step 100. Diffusion rates Dt were calculated from the slope of MSD according to the Einstein relationship Dt = MSD(t)/6t, where t is the lag time.\n\nResults and Discussion\n\nTemperature.\nThe results of the simulation on the 5 mix set at the physiological volume fraction (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt;) and lower volume fractions (SI Appendix, Fig. S3) showed that at low temperatures, the system is frozen (little to no movement of the proteins). At high temperatures, the system is overheated (moves accepted regardless of the energy). The melting curves (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt; and SI Appendix, Fig. S3) had a clear inflection point at T = 100, consistently at all volume fractions, at which the system melts (breaks from the freeze) but is not overheated yet, and thus is likely most representative of the physiological conditions. The value of T corresponding to the melting phase transition reflects the docking energy landscape (mapped in GRAMM energy units), as follows from Eq. 1, namely the energy gap between a few deep minima (frozen system states) and multiple high-energy/transient states (melted system).\nFig. 3.\nSimulations of the 5 mix set at physiological volume fraction and a range of temperatures. The volume fraction V was set to close to physiological 0.3 value. The top panels show the energy E, shift, and MSD versus simulation steps. MSD was calculated as the average for 1mat proteins. The temperatures T = 1 to 10,000 are shown by different colors. The data on the plots were smoothed by a 100-step averaging sliding window. At low temperatures, the system is frozen (little or no movement of the proteins). At high temperatures, the system is overheated (moves accepted regardless of the energy). The melting curves (the bottom panels in log scale) have a clear inflection point at T = 100, indicating the optimal temperature at which the system melts (breaks from the freeze) but is not overheated yet.\nSimulation on the 3 mix set, which is a very different system from the 5 mix set (the 3 mix proteins are much smaller than the ones in the 5 mix), yielded virtually identical melting behavior, at all volume fractions, with the same optimal temperature T = 100 (SI Appendix, Fig. S4). This confirms the robustness of our approach and adds evidence to the validity of our approximation. Accordingly, for the rest of this study, we used T = 100 as the temperature of the systems.\n\nCalibration.\nWe calibrated the time units of the simulation protocol on the available data from MD simulation of villin at the physiological volume fraction in the nonmembrane system (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). Here, the diffusion coefficient Dt value was determined to be 3.5 A2/ns, which, according to the authors, is three times greater than in the experiment. Our simulation of the villin within the 3 mix protein set at the physiological volume fraction (SI Appendix, Fig. S5) allowed us to calibrate the time variable t of our system by matching the Dt values calculated as Dt = MSD/6t (see Materials and Methods) with the MD results, corrected by the above-mentioned factor of 3. Accordingly, one step of our simulation protocol was determined to be 20 ns.\n\nValidation and Quantitative Characterization of Protein Systems.\nThe simulation protocol was validated on a number of observable parameters, testing for consistency of the results and correspondence to experimental and modeling studies. Our minima hopping paradigm, which by design allows no intermediate states between the minima (the minima correspond to the protein bound to another protein), assumes close proximity of the minima to one another (i.e., a crowded environment). Thus, our approximation would not hold for dilute systems. However, it allows for an observation of quantitative characteristics at a range of volume fractions. In our study, this range was set from 0.1 to close to physiological 0.3.\n\nMelting temperature.\nAs described above, the melting temperature for very different protein systems--the 5 mix set of average size proteins and the 3 mix set of much smaller proteins--at the full range of volume fractions, from 0.1 to 0.3, is the same. This supports the validity of our approximation and its consistency across different concentrations and size scales of proteins.\n\nDiffusion rate in different systems.\nExperimental data on the diffusion of GFP in the cytoplasm of E. coli (&lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;) puts the GFP diffusion coefficient Dt in the 0.2 to 0.9 A2/ns range. We ran simulations of the GFP with the 5 mix protein set at a physiological volume fraction. The results (SI Appendix, Fig. S6) showed that the GFP diffusion rate was 0.3 A2/ns, in excellent agreement with the experiment. It provides another confirmation of the approach validity and consistency across very disparate systems of proteins.\n\nDiffusion rate dependence on concentration.\nSimulation in the 5 mix set at different volume fractions showed a pronounced slowdown of the diffusion Dt with the increase in the protein volume fraction V in accordance with long-established evidence (&lt;&lt;REF:von Bülow-2019-116-9843&gt;&gt;, &lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). The data (Fig. 4 &lt;&lt;FIG:fig04&gt;&gt;) are an excellent fit to the Cohen-Turnbull expression (&lt;&lt;REF:O’Leary-1987-52-137&gt;&gt;) Dt = D0\nexp [-gamma V/(1 - V)], where D0 is the dilute diffusion rate and gamma is a constant characterizing the slowdown of the diffusion with the increase in the volume fraction (D0 = 4.9 A2/ns and gamma = 7.7 in our simulation). The quantitative scope of this slowdown according to the ratio of the diffusion rates, for our range of volume fractions, is available from the MD simulation for villin as 5.4, from V = 0.1 to 0.3 (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). In our simulation of the 3 mix set, that slowdown for villin was 3.4, in good agreement with the MD data.\nFig. 4.\nThe slowdown of protein diffusion with the increase in protein volume fraction. The diffusion rate Dt was calculated for 1mat proteins in the 5 mix set. The solid line is the data fit by the Cohen-Turnbull expression (&lt;&lt;REF:O’Leary-1987-52-137&gt;&gt;) Dt = D0\nexp [-gamma V/(1 - V)], where D0 is the dilute diffusion rate, and gamma is a constant characterizing the slowdown of the diffusion with the increase of the volume fraction V (D0 = 4.9 A2/ns and gamma = 7.7 in our simulation).\n\nDiffusion rate dependence on size.\nIt is well established by experiment and simulation that larger proteins diffuse at a slower rate (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;, &lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;). Due to the complexity and heterogeneity of the systems, the quantitative estimates of the size versus diffusion correlation vary significantly. Our simulation of sets of small proteins versus those of much larger proteins (see above) showed that the smaller ones diffuse significantly faster. Diffusion of proteins in the same 5 mix set simulation showed clear size versus diffusion rate correlation, at all volume fractions (Fig. 5 &lt;&lt;FIG:fig05&gt;&gt;). A similar trend was observed in the simulation of the 3 mix set (SI Appendix, Fig. S7). The rate of the slowdown scales exponentially with the size of the protein defined by the number of residues N (SI Appendix, Table S3 includes the parameters).\nFig. 5.\nDiffusion rates versus size of proteins. Results obtained on the 5 mix set for the range of volume fractions. The vertical axis shows the slowdown of the diffusion rate relative to the fastest diffusion rate. The slowdown correlates with the size of the protein at all volume fractions.\nThe Einstein-Stokes equation for diffusion of spherical particles predicts that the diffusion rate is inversely proportional to the particle radius. Thus, the slowdown of the diffusion relative to the fastest diffusion rate (Dt\nmax/Dt) would have linear dependence on the radius. Our data (SI Appendix, Fig. S8), based on the protein size defined by the radius-related metric R = N1/3, show that this dependence is close to linear at lower volume fractions. However, the slowdown rate becomes more pronounced for larger proteins, deviating to exponential at closer to physiological concentrations (&lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;), possibly reflecting the complexity and heterogeneity of the dense protein solutions. Modifying the move set based on the moves acceptance probability (&lt;&lt;REF:Sanz-2010-132-194102&gt;&gt;), which we plan to use in a future study, may provide further insights into the diffusion dependence on protein size at higher volume fractions.\n\nAggregation.\nExperimental data on the aggregation of proteins (cluster formation) at close to physiological concentrations point to the aggregation number Nc (the average number of proteins in protein assemblies) for lysozyme Nc =~ 5 (&lt;&lt;REF:Stradner-2004-432-492&gt;&gt;), and monoclonal antibodies Nc = 4 to 6 (&lt;&lt;REF:Scherer-2010-114-12948&gt;&gt;). Our data obtained on the 5 mix set (Fig. 6A &lt;&lt;FIG:fig06&gt;&gt;), at the physiological volume fraction 0.3, yielded the aggregation number (cluster size) Nc = 3.9, in excellent agreement with these estimates. The results show that the aggregation number does not change much across the whole range of the volume fractions (Fig. 6A &lt;&lt;FIG:fig06&gt;&gt;). This explains the similarity of the energy values E per molecule at different volume fractions (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt; and SI Appendix, Fig. S3), since according to our move set, this energy is determined by the number of the protein's interfaces with other proteins.\nFig. 6.\nCluster formation. (A) The aggregation number Nc (the average size of protein clusters) across volume fractions V. (B) Distribution of cluster sizes at different volume fractions. The total number of proteins in the simulation box grows with the increase in the volume fraction (SI Appendix, Table S2). Thus, the absolute numbers of clusters at higher volume fractions are larger than those at the lower volume fractions.\nThe distribution of the cluster sizes (Fig. 6B &lt;&lt;FIG:fig06&gt;&gt;) is in qualitative agreement with the results of the MD simulation in the Nc = 1 to 10 range (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). On average, at each step of the simulation, a small percentage of proteins in our system (4% for V = 0.3 and 7% for V = 0.1) are monomers (proteins whose partners have moved away and who have not acquired another partner yet, according to our move set).\n\nResidence time.\nThe existing estimates of the proteins' residence time (the lifetime of a protein pair) vary dramatically among the studies. An experimental study of lysozyme protein solution determined that the protein clusters (complexes) have a lifetime longer than the time required to diffuse over a distance of a monomer diameter (&lt;&lt;REF:Liu-2011-115-7238&gt;&gt;). Such a distance would correspond to ~50 steps in our simulation protocol (1 mus). The MD simulation, however, predicted far shorter lifetimes, with most times &lt;20 ns (&lt;&lt;REF:von Bülow-2019-116-9843&gt;&gt;). In our simulation, at volume fractions comparable to the ones in the above studies, the protein residence time is ~570 ns. Thus, our results are between the above experimental and MD estimates.\n\nTrajectory Length.\nRunning the 5 mix protein set in a 500 x 500 x 500-A3 box (the smallest cell is ~1,000 A in linear dimension) for 10,000 steps (200 mus) at volume fraction 0.3 (Fig. 7 &lt;&lt;FIG:fig07&gt;&gt;) takes ~5 h on a 3.1-GHz Intel Core i7 processor (one core). The same calculation at volume fraction 0.1 takes ~30 min. That puts a 0.3- to 3-s simulation of such system in about 1 year 1 central processing unit-core time frame. Given the all-atom resolution of our approach, this is an extraordinarily long simulation trajectory, which provides an opportunity to explicitly recreate in silico the physiological mechanisms that now are beyond the reach of atomic-resolution simulations.\nFig. 7.\nThe simulation box. Protein volume fraction is the physiological 0.3. The image was obtained using PyMOL version 2.5 (Schrodinger).\n\nConclusions and Future Directions\nSpectacular achievements of the deep learning approaches to protein structure prediction open opportunities for protein docking to refocus from the unique lowest energy states to the enormous multitude of the transient protein interactions that dominate the crowded cellular environment. Taking account of the transient interactions makes it possible to propagate in time the results of static protein docking, thus taking advantage of the powerful and efficient methodologies accumulated in the protein docking field. It opens exciting opportunities in structural modeling of the protein interactions, allowing modeling of larger systems at longer timescales, based on the atomic resolution, which is integral to docking approaches.\nRapid progress in experiment and modeling is leading to the merger of molecular and cellular biology fields. Computational methodologies increasingly address modeling of the whole cell at the molecular level. Whole-cell modeling can provide better understanding of cellular mechanisms and increase our ability to modulate them. The overarching goal, however, is the intellectual challenge of modeling life in silico.\nProteins and their interactions are the key component of cellular processes. Techniques for modeling protein interactions include protein docking and molecular simulation. The latter approaches account for the dynamics of the interactions. However, they are relatively slow, if carried out at the all-atom resolution, or significantly coarse grained (e.g., one particle representing a protein). Protein docking algorithms (e.g., systematic docking by FFT) are far more efficient in sampling the spatial coordinates. However, they do not account for the kinetics of the association (i.e., do not involve the time coordinate). The approach put forward in this study bridges the two modeling techniques. The global intermolecular energy landscape of a large system of proteins was mapped by the pairwise FFT docking and sampled in space and time using MC simulations. The approach is capable of reaching unprecedented simulation timescales at all-atom resolution.\nThe simulation protocol was parametrized on existing MD data and validated on observations from experiments and MD simulations. The simulation performed consistently across very different systems of proteins at a broad range of concentrations. It recapitulated data on the previously observed protein diffusion rates and aggregation. The speed of calculation allows reaching second-long trajectories of protein systems that approach the size of the cells at atomic resolution.\nThe long timescale atomic resolution simulations will provide the tool to explore the dynamics of cellular processes in structural detail and address important biological questions based on the molecular mechanisms involving protein association, such as cell signaling pathways and cellular metabolism. These simulations can provide important insights into fundamental biological problems of the specificity of protein interactions, facilitate studies of multinetwork phenotypes, emergent behavior in cellular protein systems, and advance our ability to modulate interaction networks.\nThis proof-of concept study is obviously just the very beginning of an expansive task of incorporating other types of macromolecules, using more sophisticated force fields that include electrostatics and solvent effects, more accurately accounting for energy barriers, optimizing the move set based on the moves acceptance probability, introducing structural flexibility, adding membrane environment and other cellular components, multiscale modeling, and improving computational efficiency. Nonetheless, our study shows that approaches grounded in protein docking can produce unprecedented dynamic simulations of protein systems at the cellular scale.\n\nSupplementary Material\nSupplementary File\n\nData, Materials, and Software Availability\nAll of the study data are included in the article and/or supporting information.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Materials and Methods\n\nModeling Paradigm.\nOur approach was to dramatically speed up the sampling of the intermolecular energy landscape by skipping the low-probability (high-energy) states, focusing only on the set of high-probability (low-energy) states corresponding to the energy minima. The \"minima hopping\" paradigm has been widely used since the early days of molecular modeling for the sampling of the energy landscapes of biomolecules, such as conformational analysis of biopolymers (&lt;&lt;REF:Galaktionov-1988-33-556&gt;&gt;), rotamer libraries (&lt;&lt;REF:Kirys-2012-80-2089&gt;&gt;), and refinement of protein-protein interfaces (&lt;&lt;REF:Dauzhenka-2018-39-2012&gt;&gt;), providing extraordinary savings of computing time by avoiding travel in low-probability areas of the landscape. Markov state models have been used to study protein folding, dynamics (&lt;&lt;REF:Shukla-2015-48-414&gt;&gt;), and association (&lt;&lt;REF:He-2021-154-084101&gt;&gt;) by representing the energy landscape by a set of the energy minima and the probabilities of transition between them. In this study we use a similar idea, namely a Markov state MC approach to sampling transitions between low-energy states, to perform very long trajectory simulations of large systems of proteins at atomic resolution.\n\nMolecular Systems.\nSimulations were performed on three different sets of proteins. To determine the volume fraction of the system, for each protein, the volume was calculated by the 3V server (3vee.molmovdb.org) (&lt;&lt;REF:Voss-2010-38-W555-62&gt;&gt;)\nSet 1. Five arbitrarily selected globular proteins of average size to represent a \"typical\" crowded cellular environment (hereafter called \"5 mix\" set; Fig. 1A &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1).\nSet 2. Set 1 plus green fluorescent protein (\"GFP + 5 mix\" set; Fig. 1B &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1) for comparison with the experimental data on GFP diffusion.\nSet 3. Three small proteins (\"3 mix\" set; Fig. 1C &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1) from Feig and coworkers (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;) representing the nonmembrane part of that study: ubiquitin, G protein B subunit, and villin.\nFig. 1.\nMolecular systems used in the study. (A) Five arbitrarily selected globular proteins of average size to represent a typical crowded cellular environment (PDB: 1mat, 1g81, 3chy, 1jxb, and 1cm2). (B) GFP (1ema). (C) Three small proteins: ubiquitin (1ubq), G protein B subunit (1pga), and villin (1vii). Molecular images were obtained using PyMOL version 2.5 (Schrodinger).\n\nGeneration of the Initial State.\nFor the starting point of the simulation, the proteins were placed on a cubical grid of a preset size, with the step of the grid calculated according to the desired protein volume fraction. In this study, we used a 500 x 500 x 500-A3 grid (the linear dimension approximately half of that of the smallest cell--Mycoplasma genitalium) with periodic boundary conditions. Each protein had an equal share of copies (e.g., in the 5 mix set of the 5-proteins mixture, each protein had a 1 in 5 share of copies). The total number of protein copies and the step of the grid were calculated according to the preset protein volume fraction V. In this study, we used a range of volume fraction values, from V = 0.10 to close to physiological V = 0.30. SI Appendix, Table S2 shows the total number of molecular copies corresponding to each volume fraction.\nThe proteins were placed in a random order. They were randomly rotated and translated within half of the grid step interval. No collision check was applied at this stage since the collisions were eliminated at the start of the simulation. SI Appendix, Fig. S1 shows a fragment of the initial state of the system before the start of the simulation.\n\nSimulation Protocol.\nAn MC procedure was developed to simulate the cellular environment with proteins in rigid-body approximation, using an all-atom representation. The procedure by design is based on proteins transitioning between different protein-protein associations. Thus, our approach applies to crowded protein environments only, in which proteins encounter each other in close proximity and monomeric states (the absence of all protein-protein interactions, including transient) are uncommon. The energy landscape of the system is represented by our GRAMM FFT docking (&lt;&lt;REF:Katchalski-Katzir-1992-89-2195&gt;&gt;) scores/energies, based on the step function approximation of the Lennard-Jones potential (&lt;&lt;REF:Vakser-1996-9-37&gt;&gt;). In this representation, the docking poses (including the multiplicity of transient encounters) correspond to negative energy values, and the monomeric states (i.e., the barriers between the minima) have energy zero.\nThe position of each protein is described with the 3 x 3 rotation matrix and the translation vector relative to the origin of the coordinate system. Protein-protein docking poses are systematically precomputed for all of the rotations and translations of each protein, relative to all of the other proteins in the system by GRAMM docking, unscored and unrefined, at intermediate resolution, previously optimized for the docking of unbound proteins (&lt;&lt;REF:Ruvinsky-2008-95-2150&gt;&gt;) (grid step 3.5 A, repulsion 9.0, and rotation interval 10degrees). For proteins A and B, both docking combinations A-B (A is the ligand and B is the receptor) and B-A (B, ligand; A, receptor) are precalculated. Thus, for example, for the 5 mix set the number of precalculated docking outputs is 25 (5 x 5). If A is the moving molecule (ligand), its new putative energy is taken from the A-B docking (and vice versa).\nThe docking results are stored on 6D grids (3 translations and 3 rotations), accessed during the MC runs. The MC move is initiated by a random selection of a protein (\"ligand\") considered for a move to proteins (\"receptors\") within a certain neighborhood (described below) from the current position of the ligand. The receptor to move to is selected randomly among all of the neighborhood proteins. Our minima-hopping paradigm, based on the approximation of the Lennard-Jones potential (see above), assumes only the short-distance interactions between the immediate docking partners. The presence of the neighboring proteins not selected for this move is accounted for by the detailed balance condition in the Metropolis acceptance criterion (described below). Once the ligand and the receptor are selected, the move is chosen randomly among the precalculated 30,000 lowest energy-docking matches for that ligand-receptor pair.\nThe simulation step is completed when all of the proteins have attempted to move. Once the ligand moves, the energy (GRAMM docking score) of the new match is added to the energy of the ligand and the energy of the old match it detaches from is subtracted. Correspondingly, the energy of the new receptor adds the energy of that new match, and the energy from the old receptor (the one the ligand is detaching from) subtracts the energy of the detaching docking match.\nThe move is accepted or rejected based on the Metropolis acceptance criterion (detailed balance condition). Ligands (L) are allowed to move to the neighboring receptors (R) only (randomly selected among all neighboring proteins), defined as those within the distance between R and L geometric centers less than the sum of the R and L radii, plus 50 A, to accommodate binding to the first layer of receptors in the crowded environment. Collision check is performed for each attempted move according to the Calpha-Calpha minimal distance of 8 A. The moves resulting in collision are rejected. Fig. 2 &lt;&lt;FIG:fig02&gt;&gt; illustrates the general principle of the move set. Periodic boundary conditions are introduced. Temperature is a parameter to be adjusted for an adequate acceptance rate.\nFig. 2.\nThe simulation move set. Docking results are precalculated and stored on 6D grids, accessed during the MC runs. The move set includes a move of one protein (L, ligand) at a time to a putative docking match with another protein (R, receptor) in the vicinity of the ligand. The energies of the states are set according to the docking scores. The move is accepted or rejected based on the Metropolis criterion (detailed balance condition).\nThe detailed balance condition for the system was implemented. The probability Pij of move from step i to step j had to be the same as Pji from j to i. Accordingly, the Metropolis criterion was normalized (&lt;&lt;REF:Hastings-1970-57-97&gt;&gt;) a\n[1]\nAs noted above, in our system, the monomeric states have energy zero, and all of the minima have negative energy values. Our model assumes no additional barriers between states i and j. We also assume the same curvature of the potential wells of each state. Thus, in the Kramers' (or Arrhenius) rate equation, which for our system can be written a\n[2]\nThe observed parameters of the simulation (per simulation step) were potential energy E, the average energy of a molecule (the sum of all molecules' energies--GRAMM docking scores--divided by the number of molecules); the shift (the average length of a molecule's move per simulation step); the MSD (the average mean square deviation of a molecule's geometric center after unwrapping coordinates from the periodic boundary conditions); acceptance rate (percentage of accepted moves); and the aggregation number Nc (the average number of proteins in an aggregate/oligomer formed by docked proteins). To allow off-the-grid relaxation of the system, the reference position for MSD calculation was set at step 100. Diffusion rates Dt were calculated from the slope of MSD according to the Einstein relationship Dt = MSD(t)/6t, where t is the lag time.\n\nResults and Discussion\n\nTemperature.\nThe results of the simulation on the 5 mix set at the physiological volume fraction (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt;) and lower volume fractions (SI Appendix, Fig. S3) showed that at low temperatures, the system is frozen (little to no movement of the proteins). At high temperatures, the system is overheated (moves accepted regardless of the energy). The melting curves (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt; and SI Appendix, Fig. S3) had a clear inflection point at T = 100, consistently at all volume fractions, at which the system melts (breaks from the freeze) but is not overheated yet, and thus is likely most representative of the physiological conditions. The value of T corresponding to the melting phase transition reflects the docking energy landscape (mapped in GRAMM energy units), as follows from Eq. 1, namely the energy gap between a few deep minima (frozen system states) and multiple high-energy/transient states (melted system).\nFig. 3.\nSimulations of the 5 mix set at physiological volume fraction and a range of temperatures. The volume fraction V was set to close to physiological 0.3 value. The top panels show the energy E, shift, and MSD versus simulation steps. MSD was calculated as the average for 1mat proteins. The temperatures T = 1 to 10,000 are shown by different colors. The data on the plots were smoothed by a 100-step averaging sliding window. At low temperatures, the system is frozen (little or no movement of the proteins). At high temperatures, the system is overheated (moves accepted regardless of the energy). The melting curves (the bottom panels in log scale) have a clear inflection point at T = 100, indicating the optimal temperature at which the system melts (breaks from the freeze) but is not overheated yet.\nSimulation on the 3 mix set, which is a very different system from the 5 mix set (the 3 mix proteins are much smaller than the ones in the 5 mix), yielded virtually identical melting behavior, at all volume fractions, with the same optimal temperature T = 100 (SI Appendix, Fig. S4). This confirms the robustness of our approach and adds evidence to the validity of our approximation. Accordingly, for the rest of this study, we used T = 100 as the temperature of the systems.\n\nCalibration.\nWe calibrated the time units of the simulation protocol on the available data from MD simulation of villin at the physiological volume fraction in the nonmembrane system (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). Here, the diffusion coefficient Dt value was determined to be 3.5 A2/ns, which, according to the authors, is three times greater than in the experiment. Our simulation of the villin within the 3 mix protein set at the physiological volume fraction (SI Appendix, Fig. S5) allowed us to calibrate the time variable t of our system by matching the Dt values calculated as Dt = MSD/6t (see Materials and Methods) with the MD results, corrected by the above-mentioned factor of 3. Accordingly, one step of our simulation protocol was determined to be 20 ns.\n\nValidation and Quantitative Characterization of Protein Systems.\nThe simulation protocol was validated on a number of observable parameters, testing for consistency of the results and correspondence to experimental and modeling studies. Our minima hopping paradigm, which by design allows no intermediate states between the minima (the minima correspond to the protein bound to another protein), assumes close proximity of the minima to one another (i.e., a crowded environment). Thus, our approximation would not hold for dilute systems. However, it allows for an observation of quantitative characteristics at a range of volume fractions. In our study, this range was set from 0.1 to close to physiological 0.3.\n\nMelting temperature.\nAs described above, the melting temperature for very different protein systems--the 5 mix set of average size proteins and the 3 mix set of much smaller proteins--at the full range of volume fractions, from 0.1 to 0.3, is the same. This supports the validity of our approximation and its consistency across different concentrations and size scales of proteins.\n\nDiffusion rate in different systems.\nExperimental data on the diffusion of GFP in the cytoplasm of E. coli (&lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;) puts the GFP diffusion coefficient Dt in the 0.2 to 0.9 A2/ns range. We ran simulations of the GFP with the 5 mix protein set at a physiological volume fraction. The results (SI Appendix, Fig. S6) showed that the GFP diffusion rate was 0.3 A2/ns, in excellent agreement with the experiment. It provides another confirmation of the approach validity and consistency across very disparate systems of proteins.\n\nDiffusion rate dependence on concentration.\nSimulation in the 5 mix set at different volume fractions showed a pronounced slowdown of the diffusion Dt with the increase in the protein volume fraction V in accordance with long-established evidence (&lt;&lt;REF:von Bülow-2019-116-9843&gt;&gt;, &lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). The data (Fig. 4 &lt;&lt;FIG:fig04&gt;&gt;) are an excellent fit to the Cohen-Turnbull expression (&lt;&lt;REF:O’Leary-1987-52-137&gt;&gt;) Dt = D0\nexp [-gamma V/(1 - V)], where D0 is the dilute diffusion rate and gamma is a constant characterizing the slowdown of the diffusion with the increase in the volume fraction (D0 = 4.9 A2/ns and gamma = 7.7 in our simulation). The quantitative scope of this slowdown according to the ratio of the diffusion rates, for our range of volume fractions, is available from the MD simulation for villin as 5.4, from V = 0.1 to 0.3 (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). In our simulation of the 3 mix set, that slowdown for villin was 3.4, in good agreement with the MD data.\nFig. 4.\nThe slowdown of protein diffusion with the increase in protein volume fraction. The diffusion rate Dt was calculated for 1mat proteins in the 5 mix set. The solid line is the data fit by the Cohen-Turnbull expression (&lt;&lt;REF:O’Leary-1987-52-137&gt;&gt;) Dt = D0\nexp [-gamma V/(1 - V)], where D0 is the dilute diffusion rate, and gamma is a constant characterizing the slowdown of the diffusion with the increase of the volume fraction V (D0 = 4.9 A2/ns and gamma = 7.7 in our simulation).\n\nDiffusion rate dependence on size.\nIt is well established by experiment and simulation that larger proteins diffuse at a slower rate (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;, &lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;). Due to the complexity and heterogeneity of the systems, the quantitative estimates of the size versus diffusion correlation vary significantly. Our simulation of sets of small proteins versus those of much larger proteins (see above) showed that the smaller ones diffuse significantly faster. Diffusion of proteins in the same 5 mix set simulation showed clear size versus diffusion rate correlation, at all volume fractions (Fig. 5 &lt;&lt;FIG:fig05&gt;&gt;). A similar trend was observed in the simulation of the 3 mix set (SI Appendix, Fig. S7). The rate of the slowdown scales exponentially with the size of the protein defined by the number of residues N (SI Appendix, Table S3 includes the parameters).\nFig. 5.\nDiffusion rates versus size of proteins. Results obtained on the 5 mix set for the range of volume fractions. The vertical axis shows the slowdown of the diffusion rate relative to the fastest diffusion rate. The slowdown correlates with the size of the protein at all volume fractions.\nThe Einstein-Stokes equation for diffusion of spherical particles predicts that the diffusion rate is inversely proportional to the particle radius. Thus, the slowdown of the diffusion relative to the fastest diffusion rate (Dt\nmax/Dt) would have linear dependence on the radius. Our data (SI Appendix, Fig. S8), based on the protein size defined by the radius-related metric R = N1/3, show that this dependence is close to linear at lower volume fractions. However, the slowdown rate becomes more pronounced for larger proteins, deviating to exponential at closer to physiological concentrations (&lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;), possibly reflecting the complexity and heterogeneity of the dense protein solutions. Modifying the move set based on the moves acceptance probability (&lt;&lt;REF:Sanz-2010-132-194102&gt;&gt;), which we plan to use in a future study, may provide further insights into the diffusion dependence on protein size at higher volume fractions.\n\nAggregation.\nExperimental data on the aggregation of proteins (cluster formation) at close to physiological concentrations point to the aggregation number Nc (the average number of proteins in protein assemblies) for lysozyme Nc =~ 5 (&lt;&lt;REF:Stradner-2004-432-492&gt;&gt;), and monoclonal antibodies Nc = 4 to 6 (&lt;&lt;REF:Scherer-2010-114-12948&gt;&gt;). Our data obtained on the 5 mix set (Fig. 6A &lt;&lt;FIG:fig06&gt;&gt;), at the physiological volume fraction 0.3, yielded the aggregation number (cluster size) Nc = 3.9, in excellent agreement with these estimates. The results show that the aggregation number does not change much across the whole range of the volume fractions (Fig. 6A &lt;&lt;FIG:fig06&gt;&gt;). This explains the similarity of the energy values E per molecule at different volume fractions (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt; and SI Appendix, Fig. S3), since according to our move set, this energy is determined by the number of the protein's interfaces with other proteins.\nFig. 6.\nCluster formation. (A) The aggregation number Nc (the average size of protein clusters) across volume fractions V. (B) Distribution of cluster sizes at different volume fractions. The total number of proteins in the simulation box grows with the increase in the volume fraction (SI Appendix, Table S2). Thus, the absolute numbers of clusters at higher volume fractions are larger than those at the lower volume fractions.\nThe distribution of the cluster sizes (Fig. 6B &lt;&lt;FIG:fig06&gt;&gt;) is in qualitative agreement with the results of the MD simulation in the Nc = 1 to 10 range (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). On average, at each step of the simulation, a small percentage of proteins in our system (4% for V = 0.3 and 7% for V = 0.1) are monomers (proteins whose partners have moved away and who have not acquired another partner yet, according to our move set).\n\nResidence time.\nThe existing estimates of the proteins' residence time (the lifetime of a protein pair) vary dramatically among the studies. An experimental study of lysozyme protein solution determined that the protein clusters (complexes) have a lifetime longer than the time required to diffuse over a distance of a monomer diameter (&lt;&lt;REF:Liu-2011-115-7238&gt;&gt;). Such a distance would correspond to ~50 steps in our simulation protocol (1 mus). The MD simulation, however, predicted far shorter lifetimes, with most times &lt;20 ns (&lt;&lt;REF:von Bülow-2019-116-9843&gt;&gt;). In our simulation, at volume fractions comparable to the ones in the above studies, the protein residence time is ~570 ns. Thus, our results are between the above experimental and MD estimates.\n\nTrajectory Length.\nRunning the 5 mix protein set in a 500 x 500 x 500-A3 box (the smallest cell is ~1,000 A in linear dimension) for 10,000 steps (200 mus) at volume fraction 0.3 (Fig. 7 &lt;&lt;FIG:fig07&gt;&gt;) takes ~5 h on a 3.1-GHz Intel Core i7 processor (one core). The same calculation at volume fraction 0.1 takes ~30 min. That puts a 0.3- to 3-s simulation of such system in about 1 year 1 central processing unit-core time frame. Given the all-atom resolution of our approach, this is an extraordinarily long simulation trajectory, which provides an opportunity to explicitly recreate in silico the physiological mechanisms that now are beyond the reach of atomic-resolution simulations.\nFig. 7.\nThe simulation box. Protein volume fraction is the physiological 0.3. The image was obtained using PyMOL version 2.5 (Schrodinger).\n\nConclusions and Future Directions\nSpectacular achievements of the deep learning approaches to protein structure prediction open opportunities for protein docking to refocus from the unique lowest energy states to the enormous multitude of the transient protein interactions that dominate the crowded cellular environment. Taking account of the transient interactions makes it possible to propagate in time the results of static protein docking, thus taking advantage of the powerful and efficient methodologies accumulated in the protein docking field. It opens exciting opportunities in structural modeling of the protein interactions, allowing modeling of larger systems at longer timescales, based on the atomic resolution, which is integral to docking approaches.\nRapid progress in experiment and modeling is leading to the merger of molecular and cellular biology fields. Computational methodologies increasingly address modeling of the whole cell at the molecular level. Whole-cell modeling can provide better understanding of cellular mechanisms and increase our ability to modulate them. The overarching goal, however, is the intellectual challenge of modeling life in silico.\nProteins and their interactions are the key component of cellular processes. Techniques for modeling protein interactions include protein docking and molecular simulation. The latter approaches account for the dynamics of the interactions. However, they are relatively slow, if carried out at the all-atom resolution, or significantly coarse grained (e.g., one particle representing a protein). Protein docking algorithms (e.g., systematic docking by FFT) are far more efficient in sampling the spatial coordinates. However, they do not account for the kinetics of the association (i.e., do not involve the time coordinate). The approach put forward in this study bridges the two modeling techniques. The global intermolecular energy landscape of a large system of proteins was mapped by the pairwise FFT docking and sampled in space and time using MC simulations. The approach is capable of reaching unprecedented simulation timescales at all-atom resolution.\nThe simulation protocol was parametrized on existing MD data and validated on observations from experiments and MD simulations. The simulation performed consistently across very different systems of proteins at a broad range of concentrations. It recapitulated data on the previously observed protein diffusion rates and aggregation. The speed of calculation allows reaching second-long trajectories of protein systems that approach the size of the cells at atomic resolution.\nThe long timescale atomic resolution simulations will provide the tool to explore the dynamics of cellular processes in structural detail and address important biological questions based on the molecular mechanisms involving protein association, such as cell signaling pathways and cellular metabolism. These simulations can provide important insights into fundamental biological problems of the specificity of protein interactions, facilitate studies of multinetwork phenotypes, emergent behavior in cellular protein systems, and advance our ability to modulate interaction networks.\nThis proof-of concept study is obviously just the very beginning of an expansive task of incorporating other types of macromolecules, using more sophisticated force fields that include electrostatics and solvent effects, more accurately accounting for energy barriers, optimizing the move set based on the moves acceptance probability, introducing structural flexibility, adding membrane environment and other cellular components, multiscale modeling, and improving computational efficiency. Nonetheless, our study shows that approaches grounded in protein docking can produce unprecedented dynamic simulations of protein systems at the cellular scale.\n\nSupplementary Material\nSupplementary File\n\nData, Materials, and Software Availability\nAll of the study data are included in the article and/or supporting information.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:
[outputs]
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Materials and Methods\n\nModeling Paradigm.\nOur approach was to dramatically speed up the sampling of the intermolecular energy landscape by skipping the low-probability (high-energy) states, focusing only on the set of high-probability (low-energy) states corresponding to the energy minima. The \"minima hopping\" paradigm has been widely used since the early days of molecular modeling for the sampling of the energy landscapes of biomolecules, such as conformational analysis of biopolymers (&lt;&lt;REF:Galaktionov-1988-33-556&gt;&gt;), rotamer libraries (&lt;&lt;REF:Kirys-2012-80-2089&gt;&gt;), and refinement of protein-protein interfaces (&lt;&lt;REF:Dauzhenka-2018-39-2012&gt;&gt;), providing extraordinary savings of computing time by avoiding travel in low-probability areas of the landscape. Markov state models have been used to study protein folding, dynamics (&lt;&lt;REF:Shukla-2015-48-414&gt;&gt;), and association (&lt;&lt;REF:He-2021-154-084101&gt;&gt;) by representing the energy landscape by a set of the energy minima and the probabilities of transition between them. In this study we use a similar idea, namely a Markov state MC approach to sampling transitions between low-energy states, to perform very long trajectory simulations of large systems of proteins at atomic resolution.\n\nMolecular Systems.\nSimulations were performed on three different sets of proteins. To determine the volume fraction of the system, for each protein, the volume was calculated by the 3V server (3vee.molmovdb.org) (&lt;&lt;REF:Voss-2010-38-W555-62&gt;&gt;)\nSet 1. Five arbitrarily selected globular proteins of average size to represent a \"typical\" crowded cellular environment (hereafter called \"5 mix\" set; Fig. 1A &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1).\nSet 2. Set 1 plus green fluorescent protein (\"GFP + 5 mix\" set; Fig. 1B &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1) for comparison with the experimental data on GFP diffusion.\nSet 3. Three small proteins (\"3 mix\" set; Fig. 1C &lt;&lt;FIG:fig01&gt;&gt; and SI Appendix, Table S1) from Feig and coworkers (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;) representing the nonmembrane part of that study: ubiquitin, G protein B subunit, and villin.\nFig. 1.\nMolecular systems used in the study. (A) Five arbitrarily selected globular proteins of average size to represent a typical crowded cellular environment (PDB: 1mat, 1g81, 3chy, 1jxb, and 1cm2). (B) GFP (1ema). (C) Three small proteins: ubiquitin (1ubq), G protein B subunit (1pga), and villin (1vii). Molecular images were obtained using PyMOL version 2.5 (Schrodinger).\n\nGeneration of the Initial State.\nFor the starting point of the simulation, the proteins were placed on a cubical grid of a preset size, with the step of the grid calculated according to the desired protein volume fraction. In this study, we used a 500 x 500 x 500-A3 grid (the linear dimension approximately half of that of the smallest cell--Mycoplasma genitalium) with periodic boundary conditions. Each protein had an equal share of copies (e.g., in the 5 mix set of the 5-proteins mixture, each protein had a 1 in 5 share of copies). The total number of protein copies and the step of the grid were calculated according to the preset protein volume fraction V. In this study, we used a range of volume fraction values, from V = 0.10 to close to physiological V = 0.30. SI Appendix, Table S2 shows the total number of molecular copies corresponding to each volume fraction.\nThe proteins were placed in a random order. They were randomly rotated and translated within half of the grid step interval. No collision check was applied at this stage since the collisions were eliminated at the start of the simulation. SI Appendix, Fig. S1 shows a fragment of the initial state of the system before the start of the simulation.\n\nSimulation Protocol.\nAn MC procedure was developed to simulate the cellular environment with proteins in rigid-body approximation, using an all-atom representation. The procedure by design is based on proteins transitioning between different protein-protein associations. Thus, our approach applies to crowded protein environments only, in which proteins encounter each other in close proximity and monomeric states (the absence of all protein-protein interactions, including transient) are uncommon. The energy landscape of the system is represented by our GRAMM FFT docking (&lt;&lt;REF:Katchalski-Katzir-1992-89-2195&gt;&gt;) scores/energies, based on the step function approximation of the Lennard-Jones potential (&lt;&lt;REF:Vakser-1996-9-37&gt;&gt;). In this representation, the docking poses (including the multiplicity of transient encounters) correspond to negative energy values, and the monomeric states (i.e., the barriers between the minima) have energy zero.\nThe position of each protein is described with the 3 x 3 rotation matrix and the translation vector relative to the origin of the coordinate system. Protein-protein docking poses are systematically precomputed for all of the rotations and translations of each protein, relative to all of the other proteins in the system by GRAMM docking, unscored and unrefined, at intermediate resolution, previously optimized for the docking of unbound proteins (&lt;&lt;REF:Ruvinsky-2008-95-2150&gt;&gt;) (grid step 3.5 A, repulsion 9.0, and rotation interval 10degrees). For proteins A and B, both docking combinations A-B (A is the ligand and B is the receptor) and B-A (B, ligand; A, receptor) are precalculated. Thus, for example, for the 5 mix set the number of precalculated docking outputs is 25 (5 x 5). If A is the moving molecule (ligand), its new putative energy is taken from the A-B docking (and vice versa).\nThe docking results are stored on 6D grids (3 translations and 3 rotations), accessed during the MC runs. The MC move is initiated by a random selection of a protein (\"ligand\") considered for a move to proteins (\"receptors\") within a certain neighborhood (described below) from the current position of the ligand. The receptor to move to is selected randomly among all of the neighborhood proteins. Our minima-hopping paradigm, based on the approximation of the Lennard-Jones potential (see above), assumes only the short-distance interactions between the immediate docking partners. The presence of the neighboring proteins not selected for this move is accounted for by the detailed balance condition in the Metropolis acceptance criterion (described below). Once the ligand and the receptor are selected, the move is chosen randomly among the precalculated 30,000 lowest energy-docking matches for that ligand-receptor pair.\nThe simulation step is completed when all of the proteins have attempted to move. Once the ligand moves, the energy (GRAMM docking score) of the new match is added to the energy of the ligand and the energy of the old match it detaches from is subtracted. Correspondingly, the energy of the new receptor adds the energy of that new match, and the energy from the old receptor (the one the ligand is detaching from) subtracts the energy of the detaching docking match.\nThe move is accepted or rejected based on the Metropolis acceptance criterion (detailed balance condition). Ligands (L) are allowed to move to the neighboring receptors (R) only (randomly selected among all neighboring proteins), defined as those within the distance between R and L geometric centers less than the sum of the R and L radii, plus 50 A, to accommodate binding to the first layer of receptors in the crowded environment. Collision check is performed for each attempted move according to the Calpha-Calpha minimal distance of 8 A. The moves resulting in collision are rejected. Fig. 2 &lt;&lt;FIG:fig02&gt;&gt; illustrates the general principle of the move set. Periodic boundary conditions are introduced. Temperature is a parameter to be adjusted for an adequate acceptance rate.\nFig. 2.\nThe simulation move set. Docking results are precalculated and stored on 6D grids, accessed during the MC runs. The move set includes a move of one protein (L, ligand) at a time to a putative docking match with another protein (R, receptor) in the vicinity of the ligand. The energies of the states are set according to the docking scores. The move is accepted or rejected based on the Metropolis criterion (detailed balance condition).\nThe detailed balance condition for the system was implemented. The probability Pij of move from step i to step j had to be the same as Pji from j to i. Accordingly, the Metropolis criterion was normalized (&lt;&lt;REF:Hastings-1970-57-97&gt;&gt;) a\n[1]\nAs noted above, in our system, the monomeric states have energy zero, and all of the minima have negative energy values. Our model assumes no additional barriers between states i and j. We also assume the same curvature of the potential wells of each state. Thus, in the Kramers' (or Arrhenius) rate equation, which for our system can be written a\n[2]\nThe observed parameters of the simulation (per simulation step) were potential energy E, the average energy of a molecule (the sum of all molecules' energies--GRAMM docking scores--divided by the number of molecules); the shift (the average length of a molecule's move per simulation step); the MSD (the average mean square deviation of a molecule's geometric center after unwrapping coordinates from the periodic boundary conditions); acceptance rate (percentage of accepted moves); and the aggregation number Nc (the average number of proteins in an aggregate/oligomer formed by docked proteins). To allow off-the-grid relaxation of the system, the reference position for MSD calculation was set at step 100. Diffusion rates Dt were calculated from the slope of MSD according to the Einstein relationship Dt = MSD(t)/6t, where t is the lag time.\n\nResults and Discussion\n\nTemperature.\nThe results of the simulation on the 5 mix set at the physiological volume fraction (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt;) and lower volume fractions (SI Appendix, Fig. S3) showed that at low temperatures, the system is frozen (little to no movement of the proteins). At high temperatures, the system is overheated (moves accepted regardless of the energy). The melting curves (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt; and SI Appendix, Fig. S3) had a clear inflection point at T = 100, consistently at all volume fractions, at which the system melts (breaks from the freeze) but is not overheated yet, and thus is likely most representative of the physiological conditions. The value of T corresponding to the melting phase transition reflects the docking energy landscape (mapped in GRAMM energy units), as follows from Eq. 1, namely the energy gap between a few deep minima (frozen system states) and multiple high-energy/transient states (melted system).\nFig. 3.\nSimulations of the 5 mix set at physiological volume fraction and a range of temperatures. The volume fraction V was set to close to physiological 0.3 value. The top panels show the energy E, shift, and MSD versus simulation steps. MSD was calculated as the average for 1mat proteins. The temperatures T = 1 to 10,000 are shown by different colors. The data on the plots were smoothed by a 100-step averaging sliding window. At low temperatures, the system is frozen (little or no movement of the proteins). At high temperatures, the system is overheated (moves accepted regardless of the energy). The melting curves (the bottom panels in log scale) have a clear inflection point at T = 100, indicating the optimal temperature at which the system melts (breaks from the freeze) but is not overheated yet.\nSimulation on the 3 mix set, which is a very different system from the 5 mix set (the 3 mix proteins are much smaller than the ones in the 5 mix), yielded virtually identical melting behavior, at all volume fractions, with the same optimal temperature T = 100 (SI Appendix, Fig. S4). This confirms the robustness of our approach and adds evidence to the validity of our approximation. Accordingly, for the rest of this study, we used T = 100 as the temperature of the systems.\n\nCalibration.\nWe calibrated the time units of the simulation protocol on the available data from MD simulation of villin at the physiological volume fraction in the nonmembrane system (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). Here, the diffusion coefficient Dt value was determined to be 3.5 A2/ns, which, according to the authors, is three times greater than in the experiment. Our simulation of the villin within the 3 mix protein set at the physiological volume fraction (SI Appendix, Fig. S5) allowed us to calibrate the time variable t of our system by matching the Dt values calculated as Dt = MSD/6t (see Materials and Methods) with the MD results, corrected by the above-mentioned factor of 3. Accordingly, one step of our simulation protocol was determined to be 20 ns.\n\nValidation and Quantitative Characterization of Protein Systems.\nThe simulation protocol was validated on a number of observable parameters, testing for consistency of the results and correspondence to experimental and modeling studies. Our minima hopping paradigm, which by design allows no intermediate states between the minima (the minima correspond to the protein bound to another protein), assumes close proximity of the minima to one another (i.e., a crowded environment). Thus, our approximation would not hold for dilute systems. However, it allows for an observation of quantitative characteristics at a range of volume fractions. In our study, this range was set from 0.1 to close to physiological 0.3.\n\nMelting temperature.\nAs described above, the melting temperature for very different protein systems--the 5 mix set of average size proteins and the 3 mix set of much smaller proteins--at the full range of volume fractions, from 0.1 to 0.3, is the same. This supports the validity of our approximation and its consistency across different concentrations and size scales of proteins.\n\nDiffusion rate in different systems.\nExperimental data on the diffusion of GFP in the cytoplasm of E. coli (&lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;) puts the GFP diffusion coefficient Dt in the 0.2 to 0.9 A2/ns range. We ran simulations of the GFP with the 5 mix protein set at a physiological volume fraction. The results (SI Appendix, Fig. S6) showed that the GFP diffusion rate was 0.3 A2/ns, in excellent agreement with the experiment. It provides another confirmation of the approach validity and consistency across very disparate systems of proteins.\n\nDiffusion rate dependence on concentration.\nSimulation in the 5 mix set at different volume fractions showed a pronounced slowdown of the diffusion Dt with the increase in the protein volume fraction V in accordance with long-established evidence (&lt;&lt;REF:von Bülow-2019-116-9843&gt;&gt;, &lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). The data (Fig. 4 &lt;&lt;FIG:fig04&gt;&gt;) are an excellent fit to the Cohen-Turnbull expression (&lt;&lt;REF:O’Leary-1987-52-137&gt;&gt;) Dt = D0\nexp [-gamma V/(1 - V)], where D0 is the dilute diffusion rate and gamma is a constant characterizing the slowdown of the diffusion with the increase in the volume fraction (D0 = 4.9 A2/ns and gamma = 7.7 in our simulation). The quantitative scope of this slowdown according to the ratio of the diffusion rates, for our range of volume fractions, is available from the MD simulation for villin as 5.4, from V = 0.1 to 0.3 (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). In our simulation of the 3 mix set, that slowdown for villin was 3.4, in good agreement with the MD data.\nFig. 4.\nThe slowdown of protein diffusion with the increase in protein volume fraction. The diffusion rate Dt was calculated for 1mat proteins in the 5 mix set. The solid line is the data fit by the Cohen-Turnbull expression (&lt;&lt;REF:O’Leary-1987-52-137&gt;&gt;) Dt = D0\nexp [-gamma V/(1 - V)], where D0 is the dilute diffusion rate, and gamma is a constant characterizing the slowdown of the diffusion with the increase of the volume fraction V (D0 = 4.9 A2/ns and gamma = 7.7 in our simulation).\n\nDiffusion rate dependence on size.\nIt is well established by experiment and simulation that larger proteins diffuse at a slower rate (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;, &lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;). Due to the complexity and heterogeneity of the systems, the quantitative estimates of the size versus diffusion correlation vary significantly. Our simulation of sets of small proteins versus those of much larger proteins (see above) showed that the smaller ones diffuse significantly faster. Diffusion of proteins in the same 5 mix set simulation showed clear size versus diffusion rate correlation, at all volume fractions (Fig. 5 &lt;&lt;FIG:fig05&gt;&gt;). A similar trend was observed in the simulation of the 3 mix set (SI Appendix, Fig. S7). The rate of the slowdown scales exponentially with the size of the protein defined by the number of residues N (SI Appendix, Table S3 includes the parameters).\nFig. 5.\nDiffusion rates versus size of proteins. Results obtained on the 5 mix set for the range of volume fractions. The vertical axis shows the slowdown of the diffusion rate relative to the fastest diffusion rate. The slowdown correlates with the size of the protein at all volume fractions.\nThe Einstein-Stokes equation for diffusion of spherical particles predicts that the diffusion rate is inversely proportional to the particle radius. Thus, the slowdown of the diffusion relative to the fastest diffusion rate (Dt\nmax/Dt) would have linear dependence on the radius. Our data (SI Appendix, Fig. S8), based on the protein size defined by the radius-related metric R = N1/3, show that this dependence is close to linear at lower volume fractions. However, the slowdown rate becomes more pronounced for larger proteins, deviating to exponential at closer to physiological concentrations (&lt;&lt;REF:Nenninger-2010-192-4535&gt;&gt;), possibly reflecting the complexity and heterogeneity of the dense protein solutions. Modifying the move set based on the moves acceptance probability (&lt;&lt;REF:Sanz-2010-132-194102&gt;&gt;), which we plan to use in a future study, may provide further insights into the diffusion dependence on protein size at higher volume fractions.\n\nAggregation.\nExperimental data on the aggregation of proteins (cluster formation) at close to physiological concentrations point to the aggregation number Nc (the average number of proteins in protein assemblies) for lysozyme Nc =~ 5 (&lt;&lt;REF:Stradner-2004-432-492&gt;&gt;), and monoclonal antibodies Nc = 4 to 6 (&lt;&lt;REF:Scherer-2010-114-12948&gt;&gt;). Our data obtained on the 5 mix set (Fig. 6A &lt;&lt;FIG:fig06&gt;&gt;), at the physiological volume fraction 0.3, yielded the aggregation number (cluster size) Nc = 3.9, in excellent agreement with these estimates. The results show that the aggregation number does not change much across the whole range of the volume fractions (Fig. 6A &lt;&lt;FIG:fig06&gt;&gt;). This explains the similarity of the energy values E per molecule at different volume fractions (Fig. 3 &lt;&lt;FIG:fig03&gt;&gt; and SI Appendix, Fig. S3), since according to our move set, this energy is determined by the number of the protein's interfaces with other proteins.\nFig. 6.\nCluster formation. (A) The aggregation number Nc (the average size of protein clusters) across volume fractions V. (B) Distribution of cluster sizes at different volume fractions. The total number of proteins in the simulation box grows with the increase in the volume fraction (SI Appendix, Table S2). Thus, the absolute numbers of clusters at higher volume fractions are larger than those at the lower volume fractions.\nThe distribution of the cluster sizes (Fig. 6B &lt;&lt;FIG:fig06&gt;&gt;) is in qualitative agreement with the results of the MD simulation in the Nc = 1 to 10 range (&lt;&lt;REF:Nawrocki-2019-116-24562&gt;&gt;). On average, at each step of the simulation, a small percentage of proteins in our system (4% for V = 0.3 and 7% for V = 0.1) are monomers (proteins whose partners have moved away and who have not acquired another partner yet, according to our move set).\n\nResidence time.\nThe existing estimates of the proteins' residence time (the lifetime of a protein pair) vary dramatically among the studies. An experimental study of lysozyme protein solution determined that the protein clusters (complexes) have a lifetime longer than the time required to diffuse over a distance of a monomer diameter (&lt;&lt;REF:Liu-2011-115-7238&gt;&gt;). Such a distance would correspond to ~50 steps in our simulation protocol (1 mus). The MD simulation, however, predicted far shorter lifetimes, with most times &lt;20 ns (&lt;&lt;REF:von Bülow-2019-116-9843&gt;&gt;). In our simulation, at volume fractions comparable to the ones in the above studies, the protein residence time is ~570 ns. Thus, our results are between the above experimental and MD estimates.\n\nTrajectory Length.\nRunning the 5 mix protein set in a 500 x 500 x 500-A3 box (the smallest cell is ~1,000 A in linear dimension) for 10,000 steps (200 mus) at volume fraction 0.3 (Fig. 7 &lt;&lt;FIG:fig07&gt;&gt;) takes ~5 h on a 3.1-GHz Intel Core i7 processor (one core). The same calculation at volume fraction 0.1 takes ~30 min. That puts a 0.3- to 3-s simulation of such system in about 1 year 1 central processing unit-core time frame. Given the all-atom resolution of our approach, this is an extraordinarily long simulation trajectory, which provides an opportunity to explicitly recreate in silico the physiological mechanisms that now are beyond the reach of atomic-resolution simulations.\nFig. 7.\nThe simulation box. Protein volume fraction is the physiological 0.3. The image was obtained using PyMOL version 2.5 (Schrodinger).\n\nConclusions and Future Directions\nSpectacular achievements of the deep learning approaches to protein structure prediction open opportunities for protein docking to refocus from the unique lowest energy states to the enormous multitude of the transient protein interactions that dominate the crowded cellular environment. Taking account of the transient interactions makes it possible to propagate in time the results of static protein docking, thus taking advantage of the powerful and efficient methodologies accumulated in the protein docking field. It opens exciting opportunities in structural modeling of the protein interactions, allowing modeling of larger systems at longer timescales, based on the atomic resolution, which is integral to docking approaches.\nRapid progress in experiment and modeling is leading to the merger of molecular and cellular biology fields. Computational methodologies increasingly address modeling of the whole cell at the molecular level. Whole-cell modeling can provide better understanding of cellular mechanisms and increase our ability to modulate them. The overarching goal, however, is the intellectual challenge of modeling life in silico.\nProteins and their interactions are the key component of cellular processes. Techniques for modeling protein interactions include protein docking and molecular simulation. The latter approaches account for the dynamics of the interactions. However, they are relatively slow, if carried out at the all-atom resolution, or significantly coarse grained (e.g., one particle representing a protein). Protein docking algorithms (e.g., systematic docking by FFT) are far more efficient in sampling the spatial coordinates. However, they do not account for the kinetics of the association (i.e., do not involve the time coordinate). The approach put forward in this study bridges the two modeling techniques. The global intermolecular energy landscape of a large system of proteins was mapped by the pairwise FFT docking and sampled in space and time using MC simulations. The approach is capable of reaching unprecedented simulation timescales at all-atom resolution.\nThe simulation protocol was parametrized on existing MD data and validated on observations from experiments and MD simulations. The simulation performed consistently across very different systems of proteins at a broad range of concentrations. It recapitulated data on the previously observed protein diffusion rates and aggregation. The speed of calculation allows reaching second-long trajectories of protein systems that approach the size of the cells at atomic resolution.\nThe long timescale atomic resolution simulations will provide the tool to explore the dynamics of cellular processes in structural detail and address important biological questions based on the molecular mechanisms involving protein association, such as cell signaling pathways and cellular metabolism. These simulations can provide important insights into fundamental biological problems of the specificity of protein interactions, facilitate studies of multinetwork phenotypes, emergent behavior in cellular protein systems, and advance our ability to modulate interaction networks.\nThis proof-of concept study is obviously just the very beginning of an expansive task of incorporating other types of macromolecules, using more sophisticated force fields that include electrostatics and solvent effects, more accurately accounting for energy barriers, optimizing the move set based on the moves acceptance probability, introducing structural flexibility, adding membrane environment and other cellular components, multiscale modeling, and improving computational efficiency. Nonetheless, our study shows that approaches grounded in protein docking can produce unprecedented dynamic simulations of protein systems at the cellular scale.\n\nSupplementary Material\nSupplementary File\n\nData, Materials, and Software Availability\nAll of the study data are included in the article and/or supporting information.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state..\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [10.88s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"sample_type\": \"not present\",\n  \"sample_preparation_type\": \"Unknown\",\n  \"sample_preparation_buffer_ph\": \"not present\",\n  \"grid_vitrification_cryogen\": \"not present\",\n  \"sample_preparation_cryo_protectant\": \"not present\",\n  \"grid_model\": \"not present\",\n  \"grid_material\": \"not present\",\n  \"grid_mesh\": \"not present\",\n  \"grid_support_topology\": \"not present\",\n  \"grid_vit_ctemp\": \"not present\",\n  \"grid_vit_chumid\": \"not present\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"Unknown\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"sample_type\": \"not present\",\n  \"sample_preparation_type\": \"Unknown\",\n  \"sample_preparation_buffer_ph\": \"not present\",\n  \"grid_vitrification_cryogen\": \"not present\",\n  \"sample_preparation_cryo_protectant\": \"not present\",\n  \"grid_model\": \"not present\",\n  \"grid_material\": \"not present\",\n  \"grid_mesh\": \"not present\",\n  \"grid_support_topology\": \"not present\",\n  \"grid_vit_ctemp\": \"not present\",\n  \"grid_vit_chumid\": \"not present\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"Unknown\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 170,
      "prompt_tokens": 6814,
      "total_tokens": 6984
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_fb84c680ea"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "sample_type": "not present",
  "sample_preparation_type": "Unknown",
  "sample_preparation_buffer_ph": "not present",
  "grid_vitrification_cryogen": "not present",
  "sample_preparation_cryo_protectant": "not present",
  "grid_model": "not present",
  "grid_material": "not present",
  "grid_mesh": "not present",
  "grid_support_topology": "not present",
  "grid_vit_ctemp": "not present",
  "grid_vit_chumid": "not present",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "Unknown",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/end] [1:chain:RunnableSequence] [10.88s] Exiting Chain run with output:
{
  "sample_type": "not present",
  "sample_preparation_type": "Unknown",
  "sample_preparation_buffer_ph": "not present",
  "grid_vitrification_cryogen": "not present",
  "sample_preparation_cryo_protectant": "not present",
  "grid_model": "not present",
  "grid_material": "not present",
  "grid_mesh": "not present",
  "grid_support_topology": "not present",
  "grid_vit_ctemp": "not present",
  "grid_vit_chumid": "not present",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "Unknown",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nOver the past 40 years, electron cryo-microscopy (cryoEM) has advanced from a rarified, avant-garde technique pursued by a small number of dedicated laboratories, into one of the most powerful and sought-after methods in molecular cell biology. CryoEM can address a wider scope of biological questions than any other method in structural biology, on samples ranging from tens or hundreds of nanometers in electron cryo-tomography (cryoET) to truly atomic resolution in single-particle work. CryoEM is having a major impact on our understanding of membrane systems, in particular the large energy-converting complexes of mitochondria and chloroplasts, but also ion channels, membrane transport proteins, protein translocases and membrane-perforating toxins. I present this breathtaking development from the perspective of someone who witnessed it at first hand from its beginnings in the early 1980s to the present day. Please note that this short paper is intended as a personal account rather than a comprehensive overview, focusing on how our own projects have progressed and benefitted from cryoEM, without wishing to detract in any way from the outstanding contributions of numerous colleagues.\n\nEarly days\nWhen I completed my PhD with Nigel Unwin at the MRC-LMB in 1981, biological electron microscopy was mostly concerned with negatively stained samples, usually in uranyl acetate. Negative stain was able to produce useful projection views, and in the right hands even three-dimensional (3D) maps of objects such as ribosomes or viruses, at resolutions up to ~20 A, limited by the grain size of the stain. These images and maps showed the overall outer shape, but no internal details of the structures were visible. Looking inside the molecules to find out how they work was not possible. The most notable and by that time already famous exception was Richard Henderson's and Nigel Unwin's 7 A map of bacteriorhodopsin, the first high-resolution EM structure of any biological macromolecule and the first structure of a membrane protein to reveal trans-membrane alpha-helices [&lt;&lt;REF:Henderson-1975-257-28&gt;&gt;]. Richard and Nigel had collected data of unstained purple membrane, protected against dehydration in the high vacuum of the microscope column in the quasi-aqueous environment of an air-dried glucose solution. Purple membranes are near-perfect two-dimensional (2D) crystals of bacteriorhodopsin that form naturally in the cell membrane of some halophilic archaea. My PhD project was to apply the same approach to 2D crystals of cytoplasmic ribosomes that formed naturally on the surface of endoplasmic reticulum membranes of hibernating lizards. The resolution we achieved was only ~60 A, due to the poor inherent order of the ribosome arrays, even though they were lightly crosslinked with glutaraldehyde, and air-drying in glucose probably did not help. Still, since glucose and protein scatter electrons roughly to the same extent, the contrast in the micrographs came mainly from the denser, more strongly scattering ribosome components, and therefore, the resulting structure gave us a first low-resolution map of the ribosomal RNA [&lt;&lt;REF:Kühlbrandt-1982-156-431&gt;&gt;,&lt;&lt;REF:Kühlbrandt-1982-7-221&gt;&gt;]. The membrane was nowhere to be seen.\nData for both the bacteriorhodopsin and the ribosome maps had been collected at room temperature, because at the time the development of electron cryo-microscopes had barely begun. The pioneering groups were at the Lawrence Berkeley National Laboratory in the USA (Glaeser), EMBL Heidelberg (Dubochet), Osaka University (Fujiyoshi) and the Fritz-Haber-Institute in Berlin (Zemlin). Jacques Dubochet at EMBL [&lt;&lt;REF:Dubochet-1982-10-55&gt;&gt;] and Robert Glaeser in Berkeley [&lt;&lt;REF:Jaffe-1984-13-373&gt;&gt;] were experimenting with frozen-hydrated specimens. At the MRC-LMB in Cambridge, Richard Henderson was perfecting computer programs for processing high-resolution images of 2D crystals. In 1990, he and his colleagues published the first cryoEM structure, again bacteriorhodopsin, at near-atomic resolution [&lt;&lt;REF:Henderson-1990-213-899&gt;&gt;]. The 3.5 A map was the culmination of a 15-year effort, combining electron diffraction data that provided very accurate amplitudes of Fourier terms, with precise structure factor phases obtained by processing of high-resolution images. The majority of images had been recorded at liquid-helium temperature with a prototype 100-kV instrument, called Suleika, at the Fritz Haber Institute of the Max Planck Society in Berlin, in the research group of Fritz Zemlin [&lt;&lt;REF:Baldwin-1988-202-585&gt;&gt;].\nRecording a single image on photographic film with the Suleika required no fewer than 70 steps of manual operation, not counting film development, quality assessment by optical diffraction and digitization on a flat-bed microdensitometer. Somewhere in the middle of this bafflingly complex procedure, Fritz would call out 'Freuen!' (rejoice!) to those around him, who looked on in amazement. Rejoicing was certainly in order if nothing had gone wrong up to that point, which did happen occasionally. The Suleika had a helium-cooled superconducting objective lens, designed and built by Isolde Dietrich at the company Siemens in Munich [&lt;&lt;REF:Dietrich-1977-2-241&gt;&gt;]. The superconducting lens was extremely stable, which had its advantages, but the objective lens current could not be changed without warming it up, and therefore, each image had to be focused by adjusting the acceleration voltage. Many of the film images at high tilt angles for the 3.5 A map came from Ken Downing in Berkeley. They were recorded with a liquid-nitrogen-cooled 100-kV JEOL 100B, converted into the world's first electron cryo-microscope by Ken Taylor and Robert Glaeser [&lt;&lt;REF:Taylor-1974-186-1036&gt;&gt;,&lt;&lt;REF:Hayward-1980-5-3&gt;&gt;]. The Berkeley 100B was far ahead of its time in other respects as well: it could be operated in spotscan mode, which largely avoided beam-induced movement, and it was equipped with a field emission electron gun, which became standard in cryoEM only 15-20 years later.\nFor my postdoc, I was hoping to combine my two passions in science--membrane proteins and high-resolution cryoEM. I was therefore looking for a membrane protein amenable to structure determination by the methods that had been devised by Richard Henderson and Nigel Unwin for purple membrane. Fortunately, I found such a protein in the light-harvesting chlorophyll-protein complex from the photosynthetic membranes of chloroplasts, LHC-II for short. I was introduced to it by Ernst Wehrli at the ETH Zurich in Switzerland. Ernst had attended the first-ever EMBO course on EM and image processing (now known as the EMBO cryo-course, but at the time there was no cryo). The course was held at the MRC-LMB in 1978 by Linda Amos, Richard Henderson, Nigel Unwin, Aaron Klug, Tony Crowther and others, with first-year PhD students like me as instructors and luminaries such as Michael Rossmann, the world-famous protein crystallographer, as participants.\nLHC-II has a natural, fortuitous (but unphysiological) tendency to form crystalline arrays when the isolated, detergent-solubilized complex is precipitated from solution with monovalent or divalent cations. That much was known from freeze-fracture electron microscopy when I started my postdoc at the ETH in Ernst Wehrli's group, but it took more than two years to persuade the complex to form large, well-ordered, purple-membrane-like 2D crystals that would lend themselves to electron diffraction and high-resolution image processing [&lt;&lt;REF:Kühlbrandt-1983-96-1414&gt;&gt;]. It took another year to work out an initial structure of negatively stained LHC-II membrane crystals by electron crystallography [&lt;&lt;REF:Kühlbrandt-1984-307-478&gt;&gt;]. The map resolution was 16 A. None of this was cryoEM, but the stage was set.\n\nLHC-II at liquid-helium temperature\nSince there were no instruments for high-resolution cryoEM anywhere nearby, I took the 2D crystals to Berkeley, where Ken Downing and I were able to record images for a 3.7 A projection map of LHC-II with their JEOL 100B [&lt;&lt;REF:Kühlbrandt-1989-207-823&gt;&gt;]. The visit proved that the crystals had potential, but large-scale data collection with this ageing machine (bits of the mu metal magnetic shielding sometimes dropped off the ceiling during a recording session) was out of the question. The temperamental home-built cryo-stage did not go much below -120degreesC. Occasionally, the column vacuum would collapse for no apparent reason, and on the fluorescent screen, I could watch my precious sample freeze over with a thick layer of contaminating ice. Fixing the microscope and getting it ready for another cryo-transfer took Ken and his team most of the day. Because I was not able to help and there was nothing else to do, I had ample time to read and explore the beautiful UC Berkeley campus. I also took the 2D crystals to Berlin, but Suleika's long-winded and idiosyncratic image recording routine would not have made large-scale data collection easy either. In any case, the Berlin group seemed none too keen to accommodate a young whipper-snapper like me, who was not even a professor. A perfect solution offered itself when I visited Yoshinori Fujiyoshi, then at the Protein Engineering Research Laboratory (PERI) in Osaka. Yoshi had designed and built a top-entry, helium-cooled cold stage for his 400-kV JEOL electron microscope [&lt;&lt;REF:Fujiyoshi-1991-38-241&gt;&gt;]. The stage was particularly suitable for high-resolution cryoEM of 2D crystals, and Yoshi welcomed me with open arms. The lens was not superconducting, and therefore, it took only ~10 manual operations to record one film image, which made data collection feasible.\nI used my Heisenberg Fellowship from the DFG for repeated visits to Yoshi's laboratory. I took the developed negatives back to EMBL Heidelberg, where in the meantime I had taken up a research group leader position. Together with my first postdoc Da Neng Wang, now Professor of Structural Biology at New York University, we scanned and processed the films. Within a year or two, we had gathered a sufficient number of images for a 3.4 A structure of LHC-II [&lt;&lt;REF:Kühlbrandt-1994-367-614&gt;&gt;] (Fig. 1 &lt;&lt;FIG:F1&gt;&gt;). The complementary high-resolution electron diffraction patterns required no more than a standard electron microscope with a liquid-nitrogen-cooled Gatan cryo-transfer holder, and Da Neng and I collected them with the JEOL 2000 at EMBL [&lt;&lt;REF:Wang-1992-61-287&gt;&gt;]. The 3.4 A map of LHC-II was the second high-resolution structure determined by cryoEM (again a membrane protein) and proved that the method worked for specimens other than bacteriorhodopsin. The map afforded fascinating insights into how chlorophylls and carotenoids were held and oriented by the polypeptide scaffold in an optimal position for harvesting solar energy and photoprotection.\nFig. 1.\nStructure of plant light-harvesting complex LHC-II determined by electron crystallography of two-dimensional crystals at liquid-helium temperature. (a) Atomic model of LHC-II trimers on the 2D crystal lattice. Polypeptide (purple) with bound chlorophylls (green and cyan) and carotenoids (yellow). (b) Details of the 3.4 A map with fitted polypeptide (white), chlorophylls (green) and carotenoids (yellow). From [&lt;&lt;REF:Kühlbrandt-1994-367-614&gt;&gt;].\nThe LHC-II structure was the overture to a number of highly successful collaborations for which colleagues from other laboratories teamed up with Yoshi to collect data with his superb instrument. Tom Walz came with his 2D crystals of aquaporin he had grown with Andreas Engel in Basel (Switzerland) [&lt;&lt;REF:Walz-1997-387-624&gt;&gt;] and Inga Schmidt-Krey and Hans Hebert with theirs of microsomal glutathione transferase [&lt;&lt;REF:Hebert-1997-271-751&gt;&gt;]. An enduring partnership ensued with Nigel Unwin. Nigel recorded countless images of his tubular crystals of nicotinic acetyl-choline receptor at increasingly higher resolution with Yoshi's microscopes [&lt;&lt;REF:Miyazawa-1999-288-765&gt;&gt;,&lt;&lt;REF:Miyazawa-2003-423-949&gt;&gt;], culminating in a 4 A map of the receptor in the membrane that resolved sidechains [&lt;&lt;REF:Unwin-2005-346-967&gt;&gt;]. Yoshi was accumulating a growing number of helium-cooled 400-kV and later 300-kV instruments, each of them filling a fairly large room, which all moved with him from Nara to Kyoto and then to Nagoya. When Nigel was awarded the Jeantet Prize in recognition of his groundbreaking work on the acetyl-choline receptor, he bought an apartment in Kyoto for his frequent stays, and Yoshi kept a JEOL 4000 specially for him that the Kyoto team called Janet. When I was appointed as a director at the Max Planck Institute of Biophysics in Frankfurt in 1997, it was my greatest wish to set up my own Yoshi-style microscope, and I was fortunate in that the Max Planck Society afforded me the means to do so. Like Yoshi's latest model it was a 300-kV instrument with a liquid-helium-cooled top-entry stage, and it worked very reliably until 2012. Liquid helium was roughly twice as effective in protecting the specimen against radiation damage as liquid nitrogen [&lt;&lt;REF:Stark-1996-63-75&gt;&gt;,&lt;&lt;REF:Fujiyoshi-1998-35-25&gt;&gt;], but also several times more expensive, and supplies were dwindling. Fortunately, it was possible to compensate for the lower signal-to-noise ratio by collecting and processing a correspondingly larger number of images at liquid-nitrogen temperature. When the opportunity arose, we replaced the JEOL 3000 SFF with a similar model, the JEOL 3200 FSC, which had a side-entry stage that could be cooled with either liquid nitrogen or liquid helium. Liquid nitrogen was less demanding and less costly, and we used it for cooling this new instrument almost exclusively.\nYears later, we discovered another and--at least for me--completely unexpected disadvantage of liquid-helium cooling. It turned out that beam-induced movement of unsupported cryo-specimens in vitreous water is up to four times worse at liquid-helium than at liquid-nitrogen temperature [&lt;&lt;REF:Pfeil-Gardiner-2019-6-1099&gt;&gt;]. This seemed to matter less for 2D crystals, which were routinely deposited on a continuous carbon support (even so, at least 80% of the film negatives showed signs of severe beam-induced movement, which made them useless). However, it does matter greatly for single-particle cryoEM. In retrospect, this finding explained why we had never been able to record any good single-particle images with the helium-cooled JEOL 3000. The reasons for the excessive beam-induced movement of vitrified solutions at liquid-helium temperature are not understood and need to be investigated further, if the superior cryo-protection qualities of liquid helium are ever to be taken advantage of for single particles. The causes may be related to the poor conductivity and viscous fluidity of vitreous water at very low temperatures, where carbon or graphene films are both highly conductive and mechanically stable.\n\n2D and 3D crystals of membrane proteins\nThe liquid-helium-cooled top entry stage of the JEOL 3000 SFF was particularly appropriate for 2D crystals, but suitable specimens were hard to come by. No more than around 10 2D protein crystals are known to diffract electrons to better than 4 A: purple membrane; LHC-II; the bacterial porins OmpF and PhoE; aquaporins-0, 1 and 4; microsomal glutathione transferase and its relatives (for reviews, see [&lt;&lt;REF:Kühlbrandt-1992-25-1&gt;&gt;,&lt;&lt;REF:Abeyrathne-2012-?-277&gt;&gt;]); and zinc-induced sheets of tubulin [&lt;&lt;REF:Nogales-1998-391-199&gt;&gt;], the only non-membrane protein amongst them. All are naturally abundant and readily available from natural sources in milligram quantities and thus amenable to extensive 2D crystallization screens. In the late 1990s, it became routine to produce--at first mostly bacterial--membrane proteins in similar amounts by heterologous expression. Quite a few of these formed 2D arrays, mostly in the shape of wide, flattened tubular vesicles. In our JEOL 3000, some yielded useful projection maps at 4-7 A or 3D structures at 6-8 A resolution by electron crystallography, notably the sodium-proton antiporters NhaA [&lt;&lt;REF:Williams-1999-18-3558&gt;&gt;,&lt;&lt;REF:Williams-2000-403-112&gt;&gt;] and NhaP [&lt;&lt;REF:Paulino-2014-3-?&gt;&gt;,&lt;&lt;REF:Paulino-2014-3-?&gt;&gt;], the osmoregulatory transporter BetP [&lt;&lt;REF:Ziegler-2004-337-1137&gt;&gt;], the bacterial protein translocase SecYEG [&lt;&lt;REF:Breyton-2002-418-662&gt;&gt;] and the light-gated ion channel channelrhodopsin [&lt;&lt;REF:Müller-2011-414-86&gt;&gt;,&lt;&lt;REF:Müller-2015-427-341&gt;&gt;]. However, the 2D crystals remained small, and none of them were ever as good as LHC-II or purple membrane. At the same time, heterologously expressed membrane transporters invariably produced good 3D crystals, which meant that we [&lt;&lt;REF:Ressl-2009-458-47&gt;&gt;] or others [&lt;&lt;REF:van den Berg-2004-427-36&gt;&gt;,&lt;&lt;REF:Kato-2012-482-369&gt;&gt;] were able to solve their high-resolution structures by X-ray crystallography. With a few rare exceptions, the--mostly hydrophobic--interactions that hold 2D crystals together are weaker and less numerous than the--mostly polar--crystal contacts in a 3D protein lattice, and therefore, the 2D crystals remained small and tended to be poorly ordered. Whatever the reason, for anyone interested in understanding the structure and mechanisms of membrane proteins rather than optimizing the growth of 2D crystals, X-ray crystallography was clearly the way to go. So for the next 10 years, this was what we did, until, very recently, single-particle cryoEM caught up. Structures of even fairly small membrane proteins can now be determined without any crystals at all, at considerably higher resolution.\n\nCryo-tomography of mitochondrial membranes\nAs a method in structural biology, cryoEM has the great advantage of not being restricted to crystals or pure protein solutions. cryoET can examine objects from any biological source of any shape and--within limits--size, provided that they are thin enough to be considered as weak phase objects, so that multiple electron scattering is negligible. In principle, the same microscopes can be used equally well for the three forms of cryoEM--electron crystallography, single-particle analysis and cryoET--as long as the sample can be tilted to about +/-60degrees, which is a prerequisite for tomography and electron crystallography.\nOne of the first non-crystalline specimens we investigated by single-particle cryoEM was the mitochondrial respiratory supercomplex, also called the respirasome. The respirasome is a 1.7-MDa assembly of three mitochondrial respiratory chain complexes, complex I, III and IV, the latter two also known as cytochrome c reductase and cytochrome c oxidase. The supercomplex was first identified on blue-native gels of detergent-solubilized mitochondrial membranes by the Frankfurt membrane biochemist Hermann Schagger [&lt;&lt;REF:Schägger-2000-19-1777&gt;&gt;]. Initially, we extracted it from gels and examined it in negative stain [&lt;&lt;REF:Schäfer-2006-281-15370&gt;&gt;]. Later, we purified it for cryoEM by gradient centrifugation [&lt;&lt;REF:Althoff-2011-30-4652&gt;&gt;]. The resolution of ~20-25 A we achieved with film images was low by today's standards, but good enough to dock X-ray structures of the component complexes, which indicated where they interacted. In the meantime, with better preparation techniques but mainly thanks to better electron detectors and image processing programs (see below), several cryoEM structures of respiratory chain supercomplexes at higher resolution have been published by us [&lt;&lt;REF:Sousa-2016-5-?&gt;&gt;] and others [&lt;&lt;REF:Letts-2016-537-644&gt;&gt;,&lt;&lt;REF:Wu-2016-167-1598&gt;&gt;]. The precise function of these large assemblies and how they work in fine-tuning cellular metabolism and organismal fitness are not known and remain controversial.\nAt the time, there was a debate as to whether the respiratory chain supercomplexes actually existed in mitochondria or whether they were artefacts of detergent solubilization. It occurred to me that, if they existed, they should be visible in tomographic volumes of mitochondrial membranes. The membranes were easy enough to prepare, but we had never done cryo-tomography. Mike Strauss, who was then in the middle of his PhD and now runs the cryoEM facility at McGill University in Montreal, visited the laboratory of Grant Jensen at CalTech to learn the technique. He stayed for 2 weeks and on his return to Frankfurt established the procedure on our recently acquired FEI Polara electron microscope. Compared to 2D crystallography with the JEOL 3000 or single-particle cryoEM, cryo-tomography with the Polara turned out to be remarkably straightforward.\nWe collected tilt series of isolated inner mitochondrial membranes and generated 3D tomographic volumes, as Mike had learned at CalTech. But when we examined the volumes in the computer, the first thing that struck us was not the respirasome, but long rows of lollipop-shaped 10-nm particles that were clearly the F1 heads of the mitochondrial F1Fo ATPase [&lt;&lt;REF:Strauss-2008-27-1154&gt;&gt;]. We were intrigued that seemingly identical rows showed up in consecutive parallel slices through the tomographic volumes. At first, we suspected a problem with the program that had generated the tomograms, but when we discovered parallel rows of the same particles in perpendicular views, it dawned on us that they were rows of ATP synthase dimers. Dimeric forms of the mitochondrial ATP synthase had been known for some time from negative-stain single-particle EM [&lt;&lt;REF:Minauro-Sanmiguel-2005-102-12356&gt;&gt;,&lt;&lt;REF:Dudkina-2006-580-3427&gt;&gt;], but because they all looked different, I thought they were bound to be artefacts. Also, it was difficult to see how a V-shaped dimer of two such large assemblies could sit stably in a planar lipid bilayer.\n\nMitochondrial ATP synthase dimers\nOur tomographic volumes showed that the membranes were of course far from planar. The edges of the inner membrane cristae were tightly bent, and the dimer rows always sat in the regions with the smallest, ~17-nm curvature radius. The dimer rows actually induce high local membrane curvature and indeed cristae formation, as we now know and were able to demonstrate by reconstituting purified ATP synthase dimers into liposomes [&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;]. Tomographic volumes of the reconstituted proteoliposomes resembled those of mitochondrial inner membranes closely and the membrane curvature at the position of the dimer rows was the same [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;]. The rows run along the tightly bent cristae rims or ridges, but they are absent from the remaining regions of the inner membrane that are indeed roughly planar. Long rows of ATP synthase dimers are a defining feature of the mitochondrial inner membrane, and the high local curvature of the ridges is characteristic of metazoan and fungal mitochondria. Dimer rows are ubiquitous in mitochondria of all of the roughly 15 species we investigated, even though the dimers themselves vary greatly in terms of overall structure and subunit composition [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;] (Fig. 2 &lt;&lt;FIG:F2&gt;&gt;). ATP synthase dimers of metazoans and fungi differ significantly from those of ciliates such as Paramecium [&lt;&lt;REF:Mühleip-2016-113-8442&gt;&gt;], which in turn differ from those of trypanosomes and Euglena [&lt;&lt;REF:Mühleip-2017-114-992&gt;&gt;]. Dimers of unicellular green algae including Chlamydomonas and Polytomella are strikingly different from all the above [&lt;&lt;REF:Dudkina-2010-1797-272&gt;&gt;,&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;], and green-plant dimers are again different (unpublished). Yet, mitochondria of all these groups have dimer rows and their inner membranes form cristae, even though their appearance varies from lamellar (metazoans, fungi, green plants) to tubular (ciliates) to anywhere in between (Polytomella, trypanosomes). The dimer rows are often &gt;1 microm long [&lt;&lt;REF:Strauss-2008-27-1154&gt;&gt;] and conspicuous in tomographic volumes of all unfixed, flash-frozen mitochondria. The reason why they had escaped notice for so long is that they do not survive the preparation steps (chemical fixation, dehydration, plastic embedding and heavy-metal staining) for conventional room-temperature EM. They are difficult to see by any technique other than cryoET, although they have been spotted once by rapid-freeze deep-etch electron microscopy of Paramecium mitochondria [&lt;&lt;REF:Allen-1989-108-2233&gt;&gt;].\nFig. 2.\nCryo-ET of mitochondrial ATP synthase dimers. Subtomogram averages (above) and rendered tomographic volumes of inner mitochondrial membranes (below) from (a) Saccharomyces cerevisiae, (b) Polytomella sp., (c) Paramecium tetraurelia and (d) Euglena gracilis. From [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;].\nATP synthase dimers, dimer rows and inner membrane cristae seem to play a central part in cellular and organismal physiology, as do the mitochondria themselves. The tight curvature at the cristae ridges is a prerequisite for close lamellar stacking, e.g. in cardiac muscle [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;] (Fig. 3a &lt;&lt;FIG:F3&gt;&gt;). Cristae stacking increases the inner membrane surface area, and therefore, the potential for ATP production, by a factor of ~10 [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;], but this is most likely not the main reason why the dimers rows are so important. Yeast mutants lacking the dimer-specific ATP synthase subunits e and g that stabilize the dimers do not have dimer rows and do not develop cristae [&lt;&lt;REF:Davies-2012-109-13602&gt;&gt;]. The mutants are less fit than wild type, grow more slowly and, according to one study, their membrane potential is reduced [&lt;&lt;REF:Bornhövd-2006-281-13990&gt;&gt;]. Even more strikingly, Drosophila fruit flies in which subunits equivalent to e and g are knocked down are deficient in germline stem cell maturation [&lt;&lt;REF:Teixeira-2015-17-689&gt;&gt;]. Mitochondria of these knockdowns do not have cristae, which points to a profound role of the cristae in Drosophila development. At present, we cannot even guess what this role might be.\nFig. 3.\nCryo-ET of whole mitochondria from (a) mouse heart and (b) mouse liver. Upper panels: slices through tomographic volumes (scale bar, 250 nm). Lower panels: segmented 3D volumes of two typical mitochondria. Heart mitochondria have closely stacked, roughly parallel cristae (blue). Cristae of liver mitochondria are more loosely arranged and not stacked. Grey, inter-membrane space; light yellow, outer membrane. From [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;].\nAfter the dimer rows had sidetracked us for some time, we collected more tomograms of mitochondrial membranes to look for respiratory chain complexes and respirasomes. Indeed, they were there, in the flat membrane regions on either side of the ATP synthase rows. We detected the L-shaped density of the 1-MDa complex I next to ATP synthase dimers in four different species and were able to confirm its identity by antibody labelling [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;]. The resolution and quality of the tomograms, recorded in our Polara with a 2k charge-coupled device (CCD) camera and Tridiem filter, was not sufficient to tell reliably whether or not they contained complex III or IV as well, which would have meant that they were respirasomes. A few years later, after we had installed our first Titan Krios electron microscope with a Quantum energy filter and Gatan K2 detector, we re-examined isolated inner membranes of bovine, fungal (Yarrowia lipolytica) and plant (Asparagus officinalis) mitochondria by cryoET. The tomographic volumes were very much clearer and enabled us to generate sub-tomogram averages of respirasomes from the three species at about 25 A resolution [&lt;&lt;REF:Davies-2018-115-3024&gt;&gt;] (Fig. 4 &lt;&lt;FIG:F4&gt;&gt;). Like the dimer rows, the supercomplexes were ubiquitous. They looked similar in mammalian, yeast and plant mitochondria, which told us that interactions between the two major components, complexes I and III, were conserved.\nFig. 4.\nSubtomogram averages of respiratory chain supercomplexes in inner membranes of (a) bovine heart, (b) fungal (Y. lipolytica) and (c) plant mitochondria. The arrangement of complex I (blue) and complex III dimer (orange) is conserved, as indicated by the blue and red guidelines. Adapted from [&lt;&lt;REF:Davies-2018-115-3024&gt;&gt;].\n\nCryo-tomography of whole mitochondria\nHaving examined membrane protein complexes in mitochondrial membrane fragments, we were curious to see the dimer rows in intact organelles. We were especially interested in mitochondria of the filamentous fungus Podospora anserina, which has an exceptionally short lifespan of 20 days, and we were wondering how the mitochondria and dimer rows would change during ageing. To our surprise, we found that the inner membrane of mitochondria from aged Podospora underwent a profound transformation within a few days [&lt;&lt;REF:Daum-2013-110-15301&gt;&gt;]. The cristae disappeared, the dimer rows became increasingly disrupted, the dimers broke up into monomers and finally the mitochondrial matrix vesiculated. The double rows along the cristae ridges, or what was left of them in ageing cultures, were easily visible in small organelles [&lt;&lt;REF:Daum-2013-110-15301&gt;&gt;] (Fig. 5 &lt;&lt;FIG:F5&gt;&gt;), but in larger mitochondria, molecular detail was obscured by the dense mitochondrial matrix.\nFig. 5.\nThree-dimensional tomographic volume of a small mitochondrion from P. anserina. The inner membrane is light blue, the outer membrane is grey. Yellow spheres indicate the catalytic F1 heads of ATP synthase dimers arranged in rows on the cristae ridges. Scale bar, 100 nm (B. Daum and W. Kuhlbrandt, MPI of Biophysics, unpublished).\nIn collaboration with the MPI for the Biology of Ageing, we examined age-related changes in mitochondria of mouse and Drosopohila melanogaster, two popular, longer-lived model organisms. The differences were less drastic than in Podospora, but nonetheless clear [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;]. In mouse, changes in inner membrane morphology were strongly organ-dependent. Heart mitochondria were particularly resilient to ageing, such that mitochondria from young and old mice were indistinguishable and equally tightly packed with lamellar cristae (Fig. 3a &lt;&lt;FIG:F3&gt;&gt;). We found that liver mitochondria have fewer cristae, which are not stacked (Fig. 3b &lt;&lt;FIG:F3&gt;&gt;). In old mice, around 20% of liver mitochondria developed a central vacuole that was contiguous with the cristae. In kidney, a similar proportion of mitochondria lost their inner membrane morphology. These changes presumably go along with reduced respiratory activity, ATP production and organellar fitness with increasing age. The picture was similar for Drosophila.\nAs a postdoc in the group, Vicki Gold used cryoET to investigate protein import into mitochondria. She generated a pre-protein construct with a mitochondrial import signal sequence and conjugated it with fluorescent quantum dots by biotin/streptavidin biochemistry. When import-competent mitochondria were incubated with this construct, the quantum-dot-labelled pre-protein got stuck in the translocase of the outer mitochondrial membrane, known as the Translocase of the Outer Membrane (TOM) complex. Her approach ensured that only complexes active in protein translocation were labelled, and that she could keep track of them by fluorescence during cryoEM grid preparation. In the tomographic volumes, the dense quantum dots showed up beautifully on the mitochondrial outer membrane, highlighting the location of active import complexes [&lt;&lt;REF:Gold-2014-5-?&gt;&gt;]. By subtomogram averaging, she also managed to visualize the import receptor protein subunits protruding on the cytoplasmic side at low resolution. Apparently, the receptor subunits are flexible, which explains why they have so far eluded us in our single-particle cryoEM studies of the isolated TOM complex (see below).\n\nCryo-tomography of reconstituted membranes\nCryoET also works well for membrane assemblies reconstituted in vitro. Oli Daumke and Katja Faelber at the Max Delbruck Centre in Berlin were determining the X-ray structure of the mitochondrial membrane-remodelling GTPase Mgm1 and wanted to know how it interacts with the lipid bilayer. In collaboration with them, Lea Dietrich was able to assemble a lattice of helical Mgm1 filaments on pre-formed liposomes. The helical arrays shaped the liposomes into thin long tubes, and by image processing and sub-tomogram averaging, she generated a volume that indicated how the GTPase in the X-ray structure interacts with the membrane [&lt;&lt;REF:Faelber-2019-571-429&gt;&gt;]. To our surprise, Lea discovered similar but different helical arrays inside the membrane tubes after some of them had ruptured, so that the protein could diffuse to the inside. The inside Mgm1 lattice generated wider tubes, presumably resembling the situation at tubular cristae junctions, which are most likely stabilized by Mgm1. The topography of the wider tubes would resemble the inner surface of the mitochondrial inner membrane, where Mgm1 is thought to promote membrane fission or fusion. The structures of the inside and outside helical lattices suggested mechanisms by which GTP hydrolysis generates a power stroke that would constrict or dilate the membrane tube [&lt;&lt;REF:Faelber-2019-571-429&gt;&gt;].\n\nPore-forming proteins\nOther membrane systems that lend themselves well to cryoET are bacterial and other pore-forming proteins. Membrane-perforating bacterial toxins are produced, or can be expressed, as soluble monomers which insert into the target membranes to form large circular pores. In collaboration with Trinad Charkaborty at the University of Giessen (Germany), we expressed, crystallized and determined the X-ray structure of the membrane-perforating toxin listeriolysin (LLO) from Listeria monocytogenes [&lt;&lt;REF:Köster-2014-5-?&gt;&gt;]. Later, we turned to pneumolysin (PLY), the virulence factor of the human pathogen Streptococcus pneumoniae. Both toxins insert into cholesterol-containing membranes of animal cells, assembling into ring-like structures of 40 or more monomers [&lt;&lt;REF:van Pee-2016-16-7915&gt;&gt;]. We obtained the structure of the closed PLY prepore in the membrane by cryoET and subtomogram averaging, and that of the fully open, 42-membered ring of the isolated, detergent-solubilized pore complex by single-particle cryoEM [&lt;&lt;REF:van Pee-2017-6-?&gt;&gt;] (Fig. 6 &lt;&lt;FIG:F6&gt;&gt;). Comparison of the prepore and pore structures indicated how the four-domain monomer inserts into the lipid bilayer and then, upon conversion of one of its domains into 80 A beta hairpins, forms ~400 A beta barrels that perforate the bilayer and kill the cell. Molecular dynamics simulation in collaboration with Gerhard Hummer at the MPI of Biophysics revealed the detailed mechanism of pore formation and extrusion of the lipid plug to open the pore [&lt;&lt;REF:Vögele-2019-116-13352&gt;&gt;].\nFig. 6.\nSingle-particle cryoEM structure of the pneumolysin (PLY) toxin pore complex at 4.5 A resolution. The four fitted domains in the 2.4 A X-ray structure of the PLY monomer are shown in colour. One domain has rearranged into ~80 A beta hairpins (cyan) that form a 42-membered ring with an inner diameter of ~400 A perforating the membrane. From [&lt;&lt;REF:van Pee-2017-6-?&gt;&gt;].\nA very different way in which a protein can perforate a membrane, this time from the inside to allow a virus to escape from an infected cell, was brought to our attention by David Prangishvili at the Institut Pasteur in Paris, France. David was studying phages that infect certain thermophilic archaea, inducing them to produce large numbers of a small, 10-kDa membrane protein encoded in the viral genome. The gene product inserts spontaneously into the archaeal membrane, where it self-assembles into ~1000 A pyramids of 7-fold symmetry that pierce through the S-layer. The S-layer forms a tough protective lattice around archaea living in extreme environments and would otherwise trap the virus in the cell. Once the S-layer is punctured, the sides of the pyramids open like the petals of a flower to create ~200 nm holes, through which the new virions emerge [&lt;&lt;REF:Daum-2014-111-3829&gt;&gt;].\n\nChloroplast tomography\nEver since our LHC-II work in the 1990s, we had kept a close interest in chloroplasts, the characteristic membrane organelles of green plants and algae, and their photosynthetic membrane proteins, including the photosystem II reaction centre [&lt;&lt;REF:Rhee-1997-389-522&gt;&gt;,&lt;&lt;REF:Rhee-1998-396-283&gt;&gt;]. Chloroplasts are much larger than mitochondria, measuring around 5 microm across. They are too bulky and dense for direct tomographic imaging of plunge-frozen samples, the approach we had taken with whole mitochondria. At the beginning of his PhD, Bertram Daum (now a PI at the University of Exeter, UK) found out that chloroplasts ruptured by mechanical blotting on the cryoEM grid maintained the intricate molecular organization of their membrane networks at least for the few seconds it takes to flash-freeze them [&lt;&lt;REF:Daum-2010-22-1299&gt;&gt;,&lt;&lt;REF:Daum-2011-62-2393&gt;&gt;]. The protein-dense chloroplast stroma had leaked out, which improved the contrast and resolution of the tomograms. By cryoET of such samples, Bertram examined the arrangement and distribution of large membrane protein complexes in chloroplast thylakoids. We were especially interested in the chloroplast ATP synthase, to see whether it formed dimers and dimer rows as in mitochondria. The tomographic volumes of chloroplast grana stacks and unstacked stromal lamellae showed unambiguously that this was not the case. The chloroplast ATP synthase was homogenously distributed over the stroma thylakoids and grana end membranes, and all copies were monomers (Fig. 7 &lt;&lt;FIG:F7&gt;&gt;). In particular, there were no dimers or dimer rows around the tightly curved edges of the thylakoid grana, where one might have expected to find them. In retrospect, this is not surprising since the chloroplast ATP synthase lacks the subunits that are required for dimer formation [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;]. The same is true of the related bacterial ATP synthases, which are likewise all monomers.\nFig. 7.\nCryoET of chloroplast thylakoid membranes. F1 heads (yellow) of the chloroplast F1Fo ATP synthase are confined to stroma and grana end membranes. The chloroplast ATP synthase is monomeric and not found at the highly curved grana edges. Grana stacks (dark green); stroma membranes (transparent green). Scale bar, 100 nm. Courtesy of Bertram Daum, MPI of Biophysics.\nMore recently, when he joined the laboratory as a PhD student, Davide Floris adopted the same approach of specimen preparation to examine the molecular landscape of etioplasts, the precursor organelles of chloroplasts in the leaves of greening plants. The etioplast inner membrane is special because it forms the pro-lamellar body, a meshwork of tubular membranes arranged in the lattice of a lipidic cubic sponge phase. This had been known since the 1970s from conventional EM of plastic sections, but no one knew how or why the membranes were arranged in this peculiar way that appears to have no parallel in biology. By cryoET, Davide discovered that the membrane tubes were surrounded, and indeed formed, by helical arrays of large numbers of a small, membrane-associated protein. By sub-tomogram averaging, he obtained a map at 9 A resolution and was able to show that the protein is the 37-kDa light-dependent protochlorophyllide oxidoreductase (LPOR) [&lt;&lt;REF:Floris-2021-7-514&gt;&gt;]. LPOR is the most abundant enzyme in etioplasts. It converts the pale yellow protochlorophyllide into chlorophyllide, which in turn is converted into the deep-green chlorophyll by chlorophyll synthase. Both enzymatic reactions are key steps in the biogenesis of photosynthetic reaction centres and light-harvesting complexes of chloroplasts, including LHC-II that consists of 33% chlorophyll by weight.\n\nDirect electron detectors\nBefore 2012, only very few cryoEM structures reached a resolution of 3.5-4 A, where large sidechains become visible. All were of 2D crystals, mostly of membrane proteins (see above), or of large icosahedral viruses (see [&lt;&lt;REF:Grigorieff-2011-21-265&gt;&gt;]). For assemblies smaller than ~20 MDa that did not consist of large numbers of identical units related by strict crystallographic, helical or point-group symmetry, high resolution was out of the question. The reason was the poor signal-to-noise ratio of individual electron micrographs, which meant that smaller particles could not be aligned and averaged with sufficient accuracy to warrant high resolution. Until that point, photographic film was by far the best recording medium for cryoEM. Although tedious to use, it was very much better than the solid-state CCD electron detectors available at the time, which were quick and convenient, but their modulation transfer function (MTF), and hence, detective quantum efficiency (DQE) at high resolution was poor. Richard Henderson's incisive 1995 review [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;] had shown beyond any doubt that of the three forms of radiation which can in principle deliver images of individual biological macromolecules--X-rays, neutrons and electrons--only the latter held the promise of producing such images at 3 A resolution or better. Therefore, if cryoEM was ever to realize this promise and come close to the success of X-ray crystallography, electronic detectors with a signal-to-noise ratio and DQE at least as good as film would be mandatory.\nIn 2004, Richard set up a consortium of engineers and members of the cryoEM community to develop such a detector. Apart from himself and his colleagues Greg McMullan and Wasi Faruqi at the MRC-LMB, core members of the team were Renato Turchetta and Nicola Guerrini at the Rutherford Appleton Laboratories (RAL) outside Oxford, UK. Both were brilliant electronics engineers with expertise in detector chip design. Together, Renato and Nicola embarked on designing and building a solid-state device with all the desirable properties of an optimal electron detector: a large active area (at least 4000 x 4000 pixels), suitable pixel size (the consortium decided on 14 microm), a DQE as close as possible to 1 (where 1.0 would be that of a perfect detector), high sensitivity, rapid readout, low readout noise, high resistance to electron irradiation and reasonable production cost. These very demanding specifications were defined in preliminary consortium meetings. The new generation of detectors would be based on complementary metal oxide semiconductor technology that was already in widespread use in digital cameras and mobile phones, but needed to be adapted to the stringent requirements of high-resolution cryoEM. Although Richard could easily have shouldered the whole effort by himself, he was intent on putting the endeavour on a broader scientific base and invited me and Jurgen Plitzko of the MPI for Biochemistry in Martinsried (Germany) to join the consortium. The FEI company in Eindhoven (The Netherlands; acquired by Thermo Fisher in 2017) joined with a view of producing and marketing the detector in due course. The LMB, the two MPIs, FEI and RAL each contributed 20% to the considerable cost of getting this project off the ground. It was clear to us that if these new direct electron detectors could be made to work, everyone in the cryoEM world would want to have one. We now know they work far better than we could have imagined in our wildest dreams and everyone does want one. Indeed, most cryoEM groups have at least one already.\nInitially, test chips with different pixel topologies were designed and produced at RAL and evaluated by the LMB detector development team. Criteria included signal-to-noise ratio, radiation hardness and DQE. Results exceeded expectations [&lt;&lt;REF:McMullan-2009-109-1126&gt;&gt;,&lt;&lt;REF:Faruqi-2011-44-357&gt;&gt;]. The most promising pixel design was then selected for further development. Before the camera chip went into production, the consortium decided that the silicon wafer of the detector should be back-thinned to ~30-50 microm, which improved the signal-to-noise ratio and MTF substantially [&lt;&lt;REF:McMullan-2009-109-1144&gt;&gt;]. Eventually, FEI produced a back-thinned detector chip for a camera called the Falcon-II, and each consortium member obtained one prototype.\nOf course, our consortium was not the only initiative in the world to develop direct electron detectors. The US company Gatan Inc., who had long experience in producing CCD cameras and energy filters for cryoEM, came out with their own design more or less simultaneously, as did the smaller Direct Electron company in San Diego, CA. Gatan's direct electron detector camera was called the K2 Summit, and the one from Direct Electron was the DE-20. The sensor areas of all three cameras were similar (roughly 4000 by 4000 pixels), but the pixels of both the K2 Summit and the DE20 were very much smaller (5 and 6.4 microm). All three performed well (see [&lt;&lt;REF:McMullan-2014-147-156&gt;&gt;]), but in practical terms, the high readout speed of the K2 (400 frames per second) meant that up to 20 times more images could be recorded per hour. Primarily for this reason, the K2 became particularly popular with cryoEM labs. It was also the camera of choice for cryoET, not least because the Falcon-II (produced by FEI) was not on offer in combination with an energy filter (essential for tomography), and post-column energy filters were at the time only produced by FEI's competitor Gatan. On the other hand, the larger pixel of the Falcon-II meant that it had a better DQE at high spatial frequencies, which made it the best choice for high-resolution cryoEM of large single particles. Since then, both the Gatan K2 and the Falcon-II have been upgraded to the Gatan K3 and Falcon-III (and now the Falcon-IV), each with significantly improved performance, although the K3 has the same pixel as the K2. Even better, faster and more sensitive direct electron detectors are bound to be developed in the years to come.\n\nThe resolution revolution\nWe had our Falcon-II camera installed on our Polara electron microscope, and by October 2012, it performed to specification. With Greg McMullan's help, we recorded our first high-resolution images of the Frh complex a few months later. Frh is a rigid, soluble, dodecameric assembly with a mass of 1.2 MDa that was isolated and purified from methanogenic archaea by our colleagues Seigo Shima and Stella Vitt at the MPI of Terrestrial Microbiology in Marburg (Germany). We had just published a nice cryoEM map of Frh from images recorded on Kodak SO163 film [&lt;&lt;REF:Mills-2013-2-?&gt;&gt;]. The nominal resolution was 5.5 A, although beta-strands were fully resolved and the map therefore looked much better. With the Falcon-II, we were able to improve the nominal resolution to 3.4 A within a few weeks [&lt;&lt;REF:Allegretti-2014-3-?&gt;&gt;]. Since then, we have improved the map further to just better than 2 A with the Falcon-III and K3 cameras (Fig. 8 &lt;&lt;FIG:F8&gt;&gt;). The Frh map was one of the first single-particle cryoEM structures in this resolution range. The other three were the 20S proteasome from Thermoplasma thermophilum at 3.3 A [&lt;&lt;REF:Li-2013-10-584&gt;&gt;], the TRPV1 ion cannel from rats at 3.4 A [&lt;&lt;REF:Liao-2013-504-107&gt;&gt;] and the large subunit of the yeast mitochondrial ribosome at 3.2 A [&lt;&lt;REF:Amunts-2014-343-1485&gt;&gt;]. Two of them (Frh and the mitoribosome large subunit) were done with the Falcon-II, the other two with the K2 Summit. Together, these four structures marked the beginning of what has become known as the 'resolution revolution' in cryoEM [&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;], which is still gathering momentum.\nFig. 8.\n(a) CryoEM structure of Frh complex at 1.97 A resolution, with each of the 12 heterotrimers in a different colour. Small red densities on the protein surface are water molecules. (b, c) Map details with fitted polypeptide chain and metal ions (Fe, orange; Ni, green). Sulphur is yellow. The iron in (c) coordinates one carbon monoxide and two cyanide ligands (J. Vonck and D. J. Mills, unpublished).\nAlthough the resolution revolution was triggered by the direct electron detectors, it could not have progressed with the same force and speed without a set of powerful image processing programs that fortuitously became available around the same time. These programs enabled the cryoEM community to take full advantage of the superior image quality delivered by the new detectors. The high sensitivity and fast readout speed of the new cameras meant that the electron dose of ~20 e/A2 for a high-resolution micrograph could be spread over a series of movie frames. This made it possible to measure [&lt;&lt;REF:Brilot-2012-177-630&gt;&gt;] and correct for beam-induced movement [&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;] that had been a persistent and intractable problem with film images. With the new cameras, it became convenient and quick to determine and correct for the contrast transfer function at the stage of image recording, e.g. by the programs ctffind4 [&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;] or Gctf [&lt;&lt;REF:Zhang-2016-193-1&gt;&gt;]. Well-known and much used image program suites include EMAN2 [&lt;&lt;REF:Tang-2007-157-38&gt;&gt;], Frealign [&lt;&lt;REF:Grigorieff-2007-157-117&gt;&gt;] and CryoSparc [&lt;&lt;REF:Punjani-2017-14-290&gt;&gt;]. The most popular and perhaps most incisive program suite is RELION, developed by Sjors Scheres at the MRC-LMB [&lt;&lt;REF:Scheres-2012-415-406&gt;&gt;,&lt;&lt;REF:Zivanov-2018-7-?&gt;&gt;], on the basis of the Xmipp programs from Jose-Maria Carazo in Madrid [&lt;&lt;REF:Scheres-2008-3-977&gt;&gt;]. RELION applies rigorous Bayesian maximum-likelihood statistics and an intuitively obvious and generally accepted 'gold standard' resolution criterion based on an objective comparison of maps generated from two random halves of a data set. As the resolution revolution progresses, top software developers in protein crystallography and structural biology are joining the fray, devising powerful programs, e.g. phenix_resolve [&lt;&lt;REF:Terwilliger-2020-17-923&gt;&gt;], to improve high-resolution cryoEM maps further.\n\nSingle-particle cryoEM of membrane proteins\nEven though most single-chain membrane proteins are at present too small for high-resolution cryoEM, they often form dimers or higher oligomers, or associate with other subunits into complexes, which brings them into a suitable size range for single-particle analysis. When Arne Moller joined the department as a group leader in 2015, he focused on two such membrane proteins. His long-standing interest in ABC transporters came to fruition in the cryoEM structure of TmrAB, a 137-kDa heterodimeric ABC transporter [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;]. A bound camelid nanobody added ~15 kDa to the mass of the heterodimer, breaking its near-2-fold symmetry and helping alignment. The membrane environment of a lipid nanodisc stabilized the protein without restricting functionally important conformational changes. Arne's team was able to resolve no fewer than eight different conformations of the transporter under turnover conditions at resolutions between 2.8 and 4.2 A (Fig. 9 &lt;&lt;FIG:F9&gt;&gt;). Together, the different conformations provided detailed snapshots of the complete transport cycle, as an impressive demonstration of the post-revolutionary power of cryoEM. The other moderately sized membrane protein studied by Arne and his team was a ~210-kDa P-type ATPase from yeast that works as a lipid flippase [&lt;&lt;REF:Timcenko-2019-571-366&gt;&gt;]. The project was a collaboration with Poul Nissen in Aarhus (Denmark), who has a long and distinguished track record in protein crystallography of P-type ATPases. Arne and colleagues were able to resolve three different states, which together illustrated the molecular mechanism by which lipids are moved from one membrane leaflet to the other.\nFig. 9.\nCryoEM structures of the ABC transporter TmrAB under turnover conditions. (a) Narrow and wide inward-facing conformations. (b) In the ATP-bound state, the nucleotide-binding domains associate closely and the exporter is either in an outward-facing open or occluded conformation. (c) Outward-facing open and occluded conformations also dominate the vanadate-trapped state. (d) Asymmetric states during TmrAB turnover resemble an outward-facing occluded conformation with slightly separated nucleotide-binding domains. TmrA is blue, TmrB is yellow and the bound nanobody is light grey. From [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;].\n\nProtein translocases\nFor years, we had been fascinated by membrane protein complexes that translocate pre-proteins into or out of cells or into cell organelles. The first of these was the protein translocase SecYEG from E. coli. As a postdoc in the group, Ian Collinson had grown 2D crystals of the complex, which Cecile Breyton analysed by electron crystallography, yielding a ~8 A 3D map [&lt;&lt;REF:Breyton-2002-418-662&gt;&gt;,&lt;&lt;REF:Bostina-2005-352-1035&gt;&gt;]. The collaboration continued when Ian, by then Professor of Biochemistry at the University of Bristol (UK), was able to bind a pre-protein mimic to SecY. This gave rise to a 3D map at similar resolution, which indicated how the substrate unlocks the complex for protein export [&lt;&lt;REF:Hizlan-2012-1-21&gt;&gt;].\nAnother of our long-standing interests was the mitochondrial TOM complex. An initial collaboration with Nikolaus Pfanner at the University of Freiburg (Germany) resulted in the cryoEM structure of a heterotrimeric form from yeast mitochondria at ~18 A resolution. Data were recorded on film, first with an FEI Technai F20, and then with the Polara [&lt;&lt;REF:Model-2008-383-1049&gt;&gt;]. We revisited the project after the arrival of the direct detectors, this time together with Stephan Nussberger, who had been my first PhD student at EMBL and was now a professor at the University of Stuttgart (Germany). Stephan had tried for 10 years to crystallize the TOM complex from the filamentous fungus Neurospora crassa, but the crystals never diffracted to better than ~8 A. He had however fine-tuned the preparation of the complex from N. crassa mitochondria, which put us in a position to make rapid progress by cryoEM. As a PhD student, Thomas Bausewein collected single-particle images of the TOM complex in detergent or reconstituted into amphipols with our JEOL 3200 FSC at liquid-nitrogen temperature, which he processed to generate a map at 6.8 A resolution [&lt;&lt;REF:Bausewein-2017-170-693&gt;&gt;]. The map showed how the two Tom40 beta-barrels in the dimeric complex form a twin pore. The alpha-helical Tom20 preprotein receptor and the small Tom7, 6 and 5 subunits were assigned on the basis of the shape and size of their map density. Subunit assignment was helped by mass spectrometry indicating the size and sequence of subunits present in the complex. Since then, two higher-resolution cryoEM structures of the S. cerevisiae TOM complex have both confirmed our subunit assignments and orientations [&lt;&lt;REF:Araiso-2019-575-395&gt;&gt;,&lt;&lt;REF:Tucker-2019-26-1158&gt;&gt;].\n\nRespiratory chain complexes\nAn interesting side product of the TOM project was the cryoEM structure of the N. crassa cytochrome c oxidase, which co-purified accidentally with the TOM complex by Ni affinity chromatography. Its identity was revealed by the shape of the 5.5 A cryoEM map [&lt;&lt;REF:Bausewein-2019-6-773&gt;&gt;]. As far as we know, this is the first time that an unknown protein contaminant has been identified solely on the basis of its cryoEM structure, but it will almost certainly not be the last.\nIn collaboration with Manuela Pereira at the ITQB in Lisbon (Portugal), Joana Sousa worked out the 3.9 A cryoEM structure of a bacterial Alternative Complex III [&lt;&lt;REF:Sousa-2018-9-?&gt;&gt;]. More recently, Alex Hahn in my group and Schara Safarian in Hartmut Michel's department at our institute obtained a superb 2.7 A map of a bacterial cytochrome bd oxidase that resolved a number of water molecules in its two proton channels [&lt;&lt;REF:Safarian-2019-366-100&gt;&gt;]. Hartmut Michel, who pioneered X-ray crystallography of membrane proteins in the 1980s, is perhaps the most prominent recent convert in the growing number of protein crystallographers who now use cryoEM almost exclusively.\n\nComplex I\nFor ~20 years, we had collaborated closely with the group of Uli Brandt and Volker Zickermann at the Frankfurt University Medical School to study the structure of complex I from the aerobic yeast Yarrowia lipolytica by electron microscopy, initially in negative stain [&lt;&lt;REF:Zickermann-2003-278-29072&gt;&gt;]. Complex I from Y. lipolytica has an important advantage over the mammalian complex in that it is readily accessible to directed mutagenesis. Brandt and Zickermann had established a genetic system to modify the complex of this particular organism, since the more established bakers' yeast S. cerevisiae lacks complex I. This long-term investment is now paying off. The group had been quite successful in growing 3D crystals and determining the X-ray structure of this ~1-MDa membrane complex, first at 6.3 A [&lt;&lt;REF:Hunte-2010-329-448&gt;&gt;] and then at 3.6-3.9 A resolution [&lt;&lt;REF:Zickermann-2015-347-44&gt;&gt;]. They were thus reluctant to jettison crystallography and EM of negatively stained samples in favour of high-resolution cryoEM. Only when K.R. Vinothkumar (who had been my PhD student in Frankfurt and now runs the first Indian cryoEM facility in Bangalore) together with Judy Hirst in Cambridge published the structure of the bovine complex at 5 A resolution [&lt;&lt;REF:Vinothkumar-2014-515-80&gt;&gt;] did they see the point of the new method, but by that time we had almost missed the boat.\nOur first cryoEM structure of the Yarrowia complex [&lt;&lt;REF:D’Imprima-2016-1857-1935&gt;&gt;] was not at very high resolution (7.9 A), but did resolve the 8 FeS clusters in the peripheral arm and most of the roughly 80 trans-membrane helices in the membrane arm. It also revealed a new accessory subunit in the peripheral arm, which we identified as a sulfur transferase that links the respiratory chain to mitochondrial sulfur metabolism. The project took off when Kristian Parey joined Volker Zickermann and me as a postdoc. With the help of Deryck Mills, and taking advantage of our best microscopes and cameras, Kristian tirelessly collected large numbers of particle images of increasingly high quality. He generated 3D maps, first at 4.3 A [&lt;&lt;REF:Parey-2018-7-?&gt;&gt;], then at 3.2 A [&lt;&lt;REF:Parey-2019-5-?&gt;&gt;], and, most recently, at 2.1 A resolution [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;]. Janet Vonck built the atomic models. Of the recent high-resolution cryoEM structures of complex I [&lt;&lt;REF:Agip-2018-25-548&gt;&gt;,&lt;&lt;REF:Kampjut-2020-370-?&gt;&gt;], this latest map is in some ways the most astonishing in that it resolves ~1600 water molecules, of which roughly 100 were found in the complex interior. The inner water molecules (Fig. 10 &lt;&lt;FIG:F10&gt;&gt;) define a horizontal proton path that runs from the quinone reduction site at the junction between the peripheral and membrane arms, to the outlet of the proton pump at the far end of the membrane arm. Notwithstanding all this great progress, we still do not understand the mechanistic details of how the electron transfer reactions at the quinone reduction site are energetically coupled to proton transport by the three proton translocation modules in the membrane arm.\nFig. 10.\nStructure of mitochondrial complex I from Y. lipolytica at 2.1 A resolution. (a) Map coloured by subunit. Small red densities are water molecules. Map section without (b) and with (c) fitted atomic model. Small red spheres in (c) indicate a subset of the ~100 internal water molecules along the proton transfer path. From [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;].\nFollowing on from his work on the Polytomella ATP synthase dimer (see below), Niklas Klusch took it upon himself to isolate and determine the cryoEM structure of complex I from Polytomella mitochondria. We joined forces with Hans-Peter Braun at the University of Hannover (Germany), who had spent years in purifying mitochondrial complex I of the model plant Arabidopsis thaliana and in characterizing its many subunits by mass spectrometry. In a tour-de-force of data collection and image processing, Niklas was able to obtain the cryoEM structures of both the Polytomella and the Arabidopsis complex at ~2.9 A resolution in minimum time, the latter in two different conformations [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;]. Mitochondrial complex I from plants and green algae (including Polytomella) distinguishes itself from that of fungi and mammals by a large external domain of three carbonic anhydrase subunits (Fig. 11a,b &lt;&lt;FIG:F11&gt;&gt;). A metal ion (most likely zinc) resolved in the active site of one of the Arabidopsis subunits was absent in Polytomella. From this observation, we concluded that the carbonic anhydrase in the plant complex is active, whereas that of Polytomella is not.\nFig. 11.\nCryoEM structures of mitochondrial complex I from (a) Arabidopsis thaliana and (b) Polytomella sp., both at ~2.9 A resolution, coloured by subunit. The 14 core subunits are drawn in shades of blue; accessory subunits in shades of green. The three subunits of the carbonic anhydrase domain are yellow, light orange and orange; subunits of the bridge domain are pink, purple and red. Insets: gamma-carbonic anhydrase (gammaCA) and bridge domain. Unknown subunits in the membrane arm (unk) are grey. From [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;].\nAnother striking difference between complex I from plants and algae to that of fungi and mammals is a bridge of three accessory subunits, joining the peripheral and membrane arms. By structural homology, Niklas discovered that one of the bridge subunits was a ferredoxin (Fig. 11a,b &lt;&lt;FIG:F11&gt;&gt;). Again, a metal ion (in this case most likely Fe) in the Arabidopsis ferredoxin was missing in Polytomella, suggesting that the ferredoxin is active in Arabidopsis but not in Polytomella. We proposed that the ferredoxin plays a key role in redox-coupling the activity of the mitochondrial respiratory chain to photosynthesis [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;].\n\nChloroplast and mitochondrial ATP synthases\nOne of our main aims with the new direct electron detectors had been to obtain a high-resolution structure of a mitochondrial or chloroplast ATP synthase. The F1Fo ATPases consist of two rotors, both of which can stall in distinct but structurally similar states that are simultaneously present, e.g. in a crystallization drop. Partly for this reason and because of the long-term instability of the complex in detergent, decades of intense effort had failed to produce a high-resolution X-ray structure of any intact, functionally competent ATP synthase. Only structures of some key components, notably an F1 subcomplex [&lt;&lt;REF:Abrahams-1994-370-621&gt;&gt;] and the c-ring rotor [&lt;&lt;REF:Meier-2005-308-659&gt;&gt;], were available from X-ray crystallography--the latter preceded by a cryoEM structure from 2D crystals at intermediate resolution [&lt;&lt;REF:Vonck-2002-321-307&gt;&gt;]. Yet for an understanding of the fundamental mechanisms of ATP production in all forms of life, detailed structures of the whole, functional assembly were essential, ideally in several different states. CryoEM, which examines ATP synthase structures particle-by-particle within hours or minutes of isolation rather than in crystals of detergent-protein complexes that take weeks to grow, promised to overcome the problem.\nNiklas Klusch, at that point an MSc student, isolated the dimers from Polytomella, and Matteo Allegretti, who had worked on the Frh structure during the first part of his PhD, proceeded to collect data with the Falcon-II camera on our Polara. Together with Karen Davies, an experienced postdoc who up to that point had done mostly cryoET, they obtained a map at 6.2 A resolution, which revealed something entirely unexpected: a set of alpha-helices in the membrane-embedded Fo subcomplex that were &gt;80 A long and included a shallow angle of ~20degrees with the membrane plane [&lt;&lt;REF:Allegretti-2015-521-237&gt;&gt;]. The paper was rejected twice, because one reviewer was sure that our structure had to be wrong. Everybody knew, the reviewer maintained, that alpha-helices in membrane proteins were ~40 A long and ran perpendicular to the membrane, which had been common knowledge since the first bacteriorhodopsin structure [&lt;&lt;REF:Henderson-1975-257-28&gt;&gt;]. Eventually, we managed to convince the editor that our structure was not wrong. We found the long Fo helices again in the 7 A map of the yeast ATP synthase dimer. This time we had collected the data in our JEOL 3200 that was now equipped with a K2 Summit camera [&lt;&lt;REF:Hahn-2016-63-445&gt;&gt;]. Subsequently, the long, membrane-intrinsic helices were seen in a number of similar structures and are now known to be a hallmark of all rotary ATPases [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;,&lt;&lt;REF:Kühlbrandt-2016-41-106&gt;&gt;].\nOf course, the different but similar ATP synthase conformations are also simultaneously present on a cryoEM grid. Our first structures of mitochondrial ATP synthase dimers from Polytomella [&lt;&lt;REF:Allegretti-2015-521-237&gt;&gt;] and yeast [&lt;&lt;REF:Hahn-2016-63-445&gt;&gt;] appeared to be averages of several such states, and this explains why the resolution was limited to 6 or 7 A. Once we had learned to apply yet more powerful image processing programs and had collected sufficient numbers of images to ensure that the various conformations in the particle set were well enough populated, it became possible to tease them apart by image processing. The first to succeed in this endeavour in our laboratory was Alexander Hahn, as a postdoc in the independent group of Thomas Meier (now Professor of Structural Biology at Imperial College London). Alex and Thomas had isolated the chloroplast ATP synthase from kilograms of spinach bought at a local supermarket--incidentally by ammonium sulphate precipitation, an old, brute-force method which I would never have thought would yield structurally and functionally intact preparations of such an intricate and fragile assembly. Alex collected a large number of movies with the faster and better Falcon-III camera installed on our new FEI Titan Krios electron microscope. He generated a very beautiful map that extended to ~2.9 A resolution in the catalytic F1 assembly of the most populated of three rotary states [&lt;&lt;REF:Hahn-2018-360-?&gt;&gt;] (Fig. 12 &lt;&lt;FIG:F12&gt;&gt;). In the catalytic sites of the F1 head, bound nucleotides, Mg ions and even water molecules were resolved. A comparison of the three rotary states indicated that the long and thin peripheral stalk, which connects the F1 head to the Fo rotor, is flexible, whereas the gamma-subunit of the central stalk works as a rigid, non-flexible driveshaft. Without the benefit of high-resolution cryoEM, earlier biophysical work and computer simulations had come to the opposite conclusion, i.e. that the central stalk was flexible and the peripheral stalk was rigid [&lt;&lt;REF:Wächter-2011-108-3924&gt;&gt;].\nFig. 12.\nCryo-EM structure of the spinach chloroplast ATP synthase. Surface representation of the map density with subunits alpha, dark green; beta, light green; gamma, blue; delta, orange; epsilon, purple; a, light blue; b, violet; b', pink; 14-membered c-ring rotor c14, pale yellow; ATP, red. The Fo subcomplex (abb'c14) is embedded in the thylakoid membrane (grey), while the catalytic F1 subcomplex (alpha3beta3gammaepsilondelta) extends into the stroma. N-terminal helices of the alphaA, alphaC and alphaE subunits are indicated. The resolution of the F1 subcomplex is 2.9 A. From [&lt;&lt;REF:Hahn-2018-360-?&gt;&gt;].\nEncouraged by Alex Hahn's and Thomas Meier's chloroplast ATP synthase structure, Bonnie Murphy, a new postdoc, together with Niklas Klusch set out to tackle the Polytomella dimer. They progressed rapidly to a 4.1 A map that refined locally to 3.7 A in the most rigid parts of the peripheral stalk and Fo subcomplex. In this region, all sidechains were resolved [&lt;&lt;REF:Klusch-2017-6-?&gt;&gt;]. The map revealed two aqueous channels, one each extending from the middle of the membrane to the lumenal or matrix membrane surface. Each channel was lined by charged and polar residues of the a-subunit stator. In the hydrophobic core of the Fo subcomplex, the two half-channels were separated by a conserved arginine sidechain and approached one another to within ~6 A. The steep potential gradient at the channel end points would exert a strong force on the charge of the critical glutamates in the c-ring rotor, generating torque and resulting in net directional rotation [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;].\nIn the 3.7 A map, the static parts of the stator assembly were well-defined, but rotary states were not resolved. Data for the 3.7 A map had been collected with a K2 Summit detector and the in-column energy filter of the JEOL 3200 FSC. Collecting a much larger number of particle images (~730 000) with a more modern, fully automated microscope (Titan Krios G2) and a considerably better detector (Falcon-III), Niklas and Bonnie produced a map of the Polytomella ATP synthase dimer at an overall resolution of 2.7-2.8 A. In an image processing marathon, Bonnie was able to separate 13 well-defined rotary substates that together provided a detailed picture of the molecular motions that accompany c-ring rotation and ATP synthesis [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;] (Fig. 13 &lt;&lt;FIG:F13&gt;&gt;). We found that the F1 head rotates along with the central stalk and c-ring rotor for the first ~30degrees of each 120degrees primary rotary step to facilitate flexible coupling of the stoichiometrically mismatched F1 and Fo subcomplexes. Flexibility is mediated by the interdomain hinge of the OSCP subunit (known as delta in the chloroplast ATP synthase) on top of the F1 head. Since this subunit is highly conserved, as are all crucial elements of rotary catalysis, we can be confident that our mechanistic conclusions apply in principle to all rotary ATP synthases. With the high-resolution structures of the ATP synthase dimer and complex I from Polytomella and the known distribution of complex I and ATP synthase dimer rows in the membrane [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;], we can now draw a model of a crista that contains all the known molecular details and summarizes how they are arranged and work together in the mitochondrial inner membrane (Fig. 14 &lt;&lt;FIG:F14&gt;&gt;).\nFig. 13.\n(a) Cryo-EM map density of the 62-subunit 1.58 MDa Polytomella ATP synthase dimer. The monomer map on the left is shown in grey. The map on the right is coloured by subunit. The F1 head consists of three catalytic beta subunits (bright green) and three structural alpha subunits (dark green). The dashed white line indicates the section plane in (b). The long C-terminal extensions of beta wrap around the alpha subunits on the outside of the F1 head. The central stalk subunits gamma (dark blue), delta (cyan) and epsilon (light blue) connect F1 to the Fo motor in the membrane (grey). The c10 rotor ring (yellow) and subunit a (bright blue) are the main components of the Fo motor complex. The two-domain OSCP subunit (orange) links the three alpha subunits to the peripheral stalk subunits ASA 1-10. The local map resolution is 2.7 A for the peripheral stalk, Fo, and c-ring, and 3.0-3.2 A for the F1 head and central stalk. (b) Rotary substates. Sections through cryo-EM maps of the Polytomella ATP synthase F1 head at the level of nucleotide-binding sites (dashed white line in a). Top row: Six substates of primary state 1. Middle row: Four substates of primary state 2. Bottom row: Three substates of primary state 3. Bright green, beta-subunits; dark green, alpha-subunits; blue, gamma-subunit; red, ADP in catalytic sites; orange, structural ATP; grey, peripheral stalk. Red lines indicate the rotary positions of consecutive substates. From [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;].\nFig. 14.\nMolecular organization of mitochondrial inner membrane cristae. Cryo-EM structures of the F1Fo ATP synthase dimers [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;] and complex I [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;], both from Polytomella sp. in the membrane. Three-dimensional density maps at 2.7 A (ATP synthase) and 2.9 A resolution (complex I) are coloured by subunit. ATP synthase dimers assemble into long rows that impose a tight local curvature on the inner membrane, giving rise to the formation of cristae as mitochondrial microcompartments. Complex I and other respiratory chain complexes (not shown) occupy the flat membrane regions on either side of the dimer rows. Complex I generates most of the proton gradient across the inner membrane, pumping protons (dashed red arrows) into the cristae, which work as proton traps. Protons (red dots) flow through channels in the Fo subcomplex of the ATP synthase from the cristae lumen (~pH 7.2) to the mitochondrial matrix (pH 8), driving the rotor (yellow) in the membrane. The torque generated by the electrochemical gradient is transmitted via the central stalk (blue) to the catalytic F1 subcomplex (green), where ATP is produced from ADP and phosphate by rotary catalysis.\n\nOutlook\nIn the past 40 years, cryoEM of membrane proteins has come further than most of us would have predicted, and it has done so more rapidly than any of us could have hoped. Yet we are by no means at the end, but right in the middle, or perhaps even still at the beginning of this exhilarating and fascinating journey.\nFor single-particle cryoEM, a few technical challenges are left. Beam-induced movement, for many years the bane of cryoEM, may have been overcome once and for all by the new HexAuFoil grids developed by Chris Russo and colleagues [&lt;&lt;REF:Naydenova-2020-370-223&gt;&gt;], although they have not yet been widely tested. Alternative approaches, such as controlled devitrification of vitrified samples [&lt;&lt;REF:Wieferig-2021-8-186&gt;&gt;], may help in special circumstances. Protein adsorption and denaturation at the air-water interface is a fundamental and ubiquitous problem in single-particle cryoEM, which means that only a small percentage (typically ~5-20%) of a particle set contribute to a high-resolution structure. The remaining 80-95% are in some way damaged or distorted at the air-water interface. It may be possible to overcome these problems with hydrophilized graphene support films, which the complexes adhere to in random orientations in preference to the air-water interface [&lt;&lt;REF:D’Imprima-2019-8-?&gt;&gt;,&lt;&lt;REF:D’Imprima-2021-54-?&gt;&gt;].\nUltimately, the success of single-particle cryoEM depends on the quality of the protein sample that goes on the grid. There is no substitute for optimizing the conditions a protein is exposed to during isolation. Therefore, when all other technical and computational issues are settled, protein biochemistry is likely to remain as the hardest part of single-particle cryoEM, because each protein is different, details matter, and there are no shortcuts.\nWith the most recent detectors and a matching battery of image processing software, structures of virtually any macromolecular assembly above a certain size can now be tackled by single-particle cryoEM. The current lower limit with soluble proteins is ~60 kDa [&lt;&lt;REF:Herzik-2019-10-?&gt;&gt;]. The recent 3.3 A structure of the 55-kDa proton-coupled folate transporter in complex with a nanobody [&lt;&lt;REF:Parker-2021-595-130&gt;&gt;] suggests that the size limit for membrane proteins is similar, even though single-particle cryoEM of small membrane proteins is inherently more difficult. The detergent micelles or lipid nanodiscs that keep them in solution do not have a defined structure and reduce the contrast in the trans-membrane region, which makes the particles harder to align. The theoretical size limit has been estimated by Richard Henderson for perfect images recorded with 300-kV electrons (see Table 2 in [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;]) to be ~38 kDa. This limit is now almost within reach and likely to be approached asymptotically. With 100 -kV microscopes [&lt;&lt;REF:Naydenova-2019-6-1086&gt;&gt;] and detectors, slightly smaller molecules should become accessible, because the scattering cross section is higher and the ratio of elastically and inelastically scattered electrons more favourable at lower acceleration voltage (see Fig. 2 &lt;&lt;FIG:F2&gt;&gt; in [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;]).\nVery accurate high-resolution structures can be obtained by electron diffraction of microscopic 3D crystals, a technique known as MicroED. MicroED has been applied mostly to small or very small proteins or protein fragments, in particular to a number of fibre-forming amyloid proteins that are recalcitrant to other techniques [&lt;&lt;REF:Rodriguez-2017-46-79&gt;&gt;]. As such, it is a welcome addition to the cryoEM methods spectrum. It is however unlikely to become as popular as single-particle cryoEM or cryoET, because it still requires 3D protein crystals, which removes one of the key advantages of cryoEM over X-ray crystallography.\nAt present, structural biology is undergoing a profound transformation. After years of futile efforts, computational methods for solving the protein folding problem have become so powerful that the structure of virtually any single protein can now be instantly predicted with near-atomic accuracy from its genomic sequence. A new deep-learning algorithm called AlphaFold2 [&lt;&lt;REF:Jumper-2021-596-583&gt;&gt;,&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;] has been shown to produce correct protein structures even if there are no related templates in the protein data bank (pdb). We tried it out with a new membrane protein without known homologues in the pdb, and the result was shockingly similar to our unreleased experimental structure. In the short to medium term, AlphaFold2 will thus shift the emphasis and take some of the pain and hard work (but also fun!) out of structure determination by cryoEM and X-ray crystallography. However, it will not make the experimental methods redundant.\nA single structure is rarely enough to understand how a protein works. Most if not all proteins in the cell undergo conformational changes while performing their tasks, and cryoEM in particular is good at resolving different, simultaneously present functional states. Such conformational ensembles would be difficult to predict. More often than not, the function of a protein depends on interactions with other proteins or small molecules, and some of the interaction partners may not be known. Similarly, the structures of protein subunits that assemble into large macromolecular complexes may not be predicted reliably at present. Computational methods are unlikely to outperform cryoEM in determining the structures of large multi-subunit complexes, such as complex I or ATP synthase in the short term. It will be some time before AlphaFold can generate the correct structures of the 13 different rotary states of the Polytomella ATP synthase [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;], the 8 different conformations of an ABC transporter [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;] or even the two different states of plant complex I [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;] as parts of a molecular mechanism. There is however no principal reason why it should not work, and therefore, it is likely to happen sooner or later. In any event, single-particle cryoEM and crystallography will be critical for validating structure predictions.\nFor cryoET, the best times are still ahead. As more and more high-resolution structures of proteins and complexes are done or predicted, cryoEM labs will increasingly turn from crystallography and single-particle work to cryoET, which is not subject to the same constraints of sample purity and homogeneity. At the same time, it will become increasingly urgent to investigate and understand the context of macromolecular components in the cell. CryoET in particular stands to gain from protein structure prediction, because it generates lower-resolution volumes quite readily, to which the predicted structures can then be fitted. CryoEM instrumentation can in principle be used equally well for single-particle work and cryoET (although an acceleration voltage of 300 kV is preferable for tomography), which makes the transition easier. CryoET does have its own constraints, but in terms of the molecular systems and biomedical questions it can address, it is essentially unlimited. Every cell contains a universe of molecular landscapes that is best explored by cryoET.\nConstraints include sample thickness and tilt geometry. Most samples are too thick, but they can be thinned into thin lamellae by focused ion beam (FIB) milling at low temperature. Cryo-FIB milling is a demanding technique but it is practiced successfully in a growing number of cryoEM laboratories around the world [&lt;&lt;REF:Wagner-2020-15-2041&gt;&gt;,&lt;&lt;REF:Zachs-2020-9-?&gt;&gt;]. Constraints of tilt geometry can be overcome by FIB-milling cylindrical pencils instead of planar lamellae, and several laboratories are experimenting in this direction.\nA major unresolved challenge in cryoET is the lack of a universal cloneable label equivalent to green fluorescent protein (GFP) in light microscopy that is fused with a protein so it can be identified and studied in detail in its native context. Correlative light/electron microscopy alleviates the problem by identifying regions of interest in a larger volume, such as a cell, but within that region individual assemblies still have to be recognized by their shape and position, e.g. relative to a membrane. Because phosphorus scatters electrons more strongly than carbon, nitrogen or oxygen, phospholipid bilayers show up well in tomographic volumes, and therefore, cryoET works well for protein complexes in or at a membrane. CryoET of HIV capsids in vitrified aqueous buffer has achieved 3.4 A resolution [&lt;&lt;REF:Turonova-2017-199-187&gt;&gt;]. There is no fundamental reason why cryoET, just as single-particle cryoEM, should not go to true atomic resolution, given a sufficiently large number of tomographic volumes to average. In cellular tomography, the target complexes are usually surrounded by other macromolecules of similar density, which makes the targets more difficult to detect and average than single particles in dilute aqueous buffers. The ultimate limitation in both cases is the signal-to-noise ratio of the electron micrographs.\nThe signal-to-noise ratio of electron micrographs can be enhanced by a phase plate. It is fair to say that current phase plate designs, including the Volta phase plate [&lt;&lt;REF:Buijsse-2020-218-?&gt;&gt;] or our own electrostatic Boersch phase plate [&lt;&lt;REF:Walter-2015-153-22&gt;&gt;], have not kept their promise. Even though they can improve low-resolution image contrast, potential gains at high resolution are more than offset by charging problems and a lack of reproducibility associated with introducing an external device into the electron beam [&lt;&lt;REF:Buijsse-2020-218-?&gt;&gt;]. These problems may be overcome by a new elegant phase plate design that shifts the phase of the electron beam by the ponderomotive force of a strong laser [&lt;&lt;REF:Schwartz-2019-36-1016&gt;&gt;]. The laser phase plate enhances contrast in the entire resolution range and would be particularly useful for cryoET. However, its implementation is technically demanding, the electron microscope needs an extra lens and a tremendously powerful laser is necessary for shifting the phase of a 300 kV beam by 90degrees for optimum contrast. For single-particle cryoEM, recent improvements in electron microscope hardware, in particular cold field emitters, better image filters and direct electron detectors, have increased phase contrast to an extent that makes the available phase plates less of a help than a hindrance for high resolution.\nEven though the present generation of direct electron detectors is already superb, further improvements in their design and production are likely and will benefit both single-particle cryoEM and cryoET. At present, the DQE of the best available detectors (Falcon-IV) is 1.0 at low resolution and close to 0.4 at Nyquist frequency [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;]. The Nyquist frequency defines the maximum resolution possible with the given pixel size of the detector. For a perfect detector, the DQE would be 1.0 at Nyquist, but this may be hard to achieve. Still, any improvement will boost the signal-to-noise ratio at high resolution and translate into better maps from fewer particle images or tomograms.\nSo far, the high cost of cryoEM hardware does not seem to have taken away much of its popularity. The number of new cryoEM labs around the globe is still growing, and research councils and agencies are willing to support them, because of the expected gains in mechanistic insight for basic biomedical research, which will ultimately translate into benefits for human health. But this trend cannot go on forever. Cost scales roughly with acceleration voltage. It is possible to produce less expensive 100-kV electron microscopes that perform equally well, and in some ways better for single-particle cryoEM than 300-kV instruments [&lt;&lt;REF:Naydenova-2019-6-1086&gt;&gt;]. Bringing the cost of EM hardware down substantially will be essential if cryoEM is to be as widely accessible to biomedical research as protein crystallography has been in the past. An alternative would be to create more of the national cryoEM facilities such as eBIC at the Diamond synchrotron in the UK or the National Centers in the USA, to cumulate resources and provide cryoEM as a service. However, sample preparation for cryoEM is still very much a skill and a craft. Automation is in its infancy [&lt;&lt;REF:Budell-2021-?-?&gt;&gt;,&lt;&lt;REF:Weissenberger-2021-18-463&gt;&gt;] and reproducibly thin ice layers, a key factor for high resolution, are hard to achieve. Similarly, collecting top-quality high-resolution images requires close familiarity with the sample. In the long run, it may therefore be preferable not to separate researchers from their microscopes and cryoEM samples.\nWithin the past year, two cryoEM structures of a standard soluble test specimen, recombinant mouse apoferritin, have been reported at ~1.25 A, i.e. true atomic resolution [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;,&lt;&lt;REF:Yip-2020-587-157&gt;&gt;]. This proves that cryoEM is capable of delivering structures of equal quality as atomic-resolution X-ray crystallography, which not many of us would have imagined a few years ago. For most biological questions, a resolution ~2 A is probably more than adequate, but higher resolution is always nice, and it is hard to predict what it might reveal. Further improvements in electron microscopes, detectors and image processing software are bound to result in more and more atomic-resolution structures of interesting membrane proteins. Already with the present generation of cryoEM hardware and software, water molecules are resolved in the cytochrome bd oxidase [&lt;&lt;REF:Safarian-2019-366-100&gt;&gt;], complex I [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;,&lt;&lt;REF:Kampjut-2020-370-?&gt;&gt;] and the GABAA receptor [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;]. At very high resolution, cryoEM may be able to visualize ionization states of bound ions and protonation states of sidechains and water molecules, which would provide more detailed insights into molecular mechanisms. The future for studying membrane proteins by cryoEM gets brighter and more fascinating all the time!\n\nFunding\nMost of the work described in this review was funded by the Max Planck Society. Some of the earlier work at EMBL was funded by the European Molecular Biology Laboratory.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nOver the past 40 years, electron cryo-microscopy (cryoEM) has advanced from a rarified, avant-garde technique pursued by a small number of dedicated laboratories, into one of the most powerful and sought-after methods in molecular cell biology. CryoEM can address a wider scope of biological questions than any other method in structural biology, on samples ranging from tens or hundreds of nanometers in electron cryo-tomography (cryoET) to truly atomic resolution in single-particle work. CryoEM is having a major impact on our understanding of membrane systems, in particular the large energy-converting complexes of mitochondria and chloroplasts, but also ion channels, membrane transport proteins, protein translocases and membrane-perforating toxins. I present this breathtaking development from the perspective of someone who witnessed it at first hand from its beginnings in the early 1980s to the present day. Please note that this short paper is intended as a personal account rather than a comprehensive overview, focusing on how our own projects have progressed and benefitted from cryoEM, without wishing to detract in any way from the outstanding contributions of numerous colleagues.\n\nEarly days\nWhen I completed my PhD with Nigel Unwin at the MRC-LMB in 1981, biological electron microscopy was mostly concerned with negatively stained samples, usually in uranyl acetate. Negative stain was able to produce useful projection views, and in the right hands even three-dimensional (3D) maps of objects such as ribosomes or viruses, at resolutions up to ~20 A, limited by the grain size of the stain. These images and maps showed the overall outer shape, but no internal details of the structures were visible. Looking inside the molecules to find out how they work was not possible. The most notable and by that time already famous exception was Richard Henderson's and Nigel Unwin's 7 A map of bacteriorhodopsin, the first high-resolution EM structure of any biological macromolecule and the first structure of a membrane protein to reveal trans-membrane alpha-helices [&lt;&lt;REF:Henderson-1975-257-28&gt;&gt;]. Richard and Nigel had collected data of unstained purple membrane, protected against dehydration in the high vacuum of the microscope column in the quasi-aqueous environment of an air-dried glucose solution. Purple membranes are near-perfect two-dimensional (2D) crystals of bacteriorhodopsin that form naturally in the cell membrane of some halophilic archaea. My PhD project was to apply the same approach to 2D crystals of cytoplasmic ribosomes that formed naturally on the surface of endoplasmic reticulum membranes of hibernating lizards. The resolution we achieved was only ~60 A, due to the poor inherent order of the ribosome arrays, even though they were lightly crosslinked with glutaraldehyde, and air-drying in glucose probably did not help. Still, since glucose and protein scatter electrons roughly to the same extent, the contrast in the micrographs came mainly from the denser, more strongly scattering ribosome components, and therefore, the resulting structure gave us a first low-resolution map of the ribosomal RNA [&lt;&lt;REF:Kühlbrandt-1982-156-431&gt;&gt;,&lt;&lt;REF:Kühlbrandt-1982-7-221&gt;&gt;]. The membrane was nowhere to be seen.\nData for both the bacteriorhodopsin and the ribosome maps had been collected at room temperature, because at the time the development of electron cryo-microscopes had barely begun. The pioneering groups were at the Lawrence Berkeley National Laboratory in the USA (Glaeser), EMBL Heidelberg (Dubochet), Osaka University (Fujiyoshi) and the Fritz-Haber-Institute in Berlin (Zemlin). Jacques Dubochet at EMBL [&lt;&lt;REF:Dubochet-1982-10-55&gt;&gt;] and Robert Glaeser in Berkeley [&lt;&lt;REF:Jaffe-1984-13-373&gt;&gt;] were experimenting with frozen-hydrated specimens. At the MRC-LMB in Cambridge, Richard Henderson was perfecting computer programs for processing high-resolution images of 2D crystals. In 1990, he and his colleagues published the first cryoEM structure, again bacteriorhodopsin, at near-atomic resolution [&lt;&lt;REF:Henderson-1990-213-899&gt;&gt;]. The 3.5 A map was the culmination of a 15-year effort, combining electron diffraction data that provided very accurate amplitudes of Fourier terms, with precise structure factor phases obtained by processing of high-resolution images. The majority of images had been recorded at liquid-helium temperature with a prototype 100-kV instrument, called Suleika, at the Fritz Haber Institute of the Max Planck Society in Berlin, in the research group of Fritz Zemlin [&lt;&lt;REF:Baldwin-1988-202-585&gt;&gt;].\nRecording a single image on photographic film with the Suleika required no fewer than 70 steps of manual operation, not counting film development, quality assessment by optical diffraction and digitization on a flat-bed microdensitometer. Somewhere in the middle of this bafflingly complex procedure, Fritz would call out 'Freuen!' (rejoice!) to those around him, who looked on in amazement. Rejoicing was certainly in order if nothing had gone wrong up to that point, which did happen occasionally. The Suleika had a helium-cooled superconducting objective lens, designed and built by Isolde Dietrich at the company Siemens in Munich [&lt;&lt;REF:Dietrich-1977-2-241&gt;&gt;]. The superconducting lens was extremely stable, which had its advantages, but the objective lens current could not be changed without warming it up, and therefore, each image had to be focused by adjusting the acceleration voltage. Many of the film images at high tilt angles for the 3.5 A map came from Ken Downing in Berkeley. They were recorded with a liquid-nitrogen-cooled 100-kV JEOL 100B, converted into the world's first electron cryo-microscope by Ken Taylor and Robert Glaeser [&lt;&lt;REF:Taylor-1974-186-1036&gt;&gt;,&lt;&lt;REF:Hayward-1980-5-3&gt;&gt;]. The Berkeley 100B was far ahead of its time in other respects as well: it could be operated in spotscan mode, which largely avoided beam-induced movement, and it was equipped with a field emission electron gun, which became standard in cryoEM only 15-20 years later.\nFor my postdoc, I was hoping to combine my two passions in science--membrane proteins and high-resolution cryoEM. I was therefore looking for a membrane protein amenable to structure determination by the methods that had been devised by Richard Henderson and Nigel Unwin for purple membrane. Fortunately, I found such a protein in the light-harvesting chlorophyll-protein complex from the photosynthetic membranes of chloroplasts, LHC-II for short. I was introduced to it by Ernst Wehrli at the ETH Zurich in Switzerland. Ernst had attended the first-ever EMBO course on EM and image processing (now known as the EMBO cryo-course, but at the time there was no cryo). The course was held at the MRC-LMB in 1978 by Linda Amos, Richard Henderson, Nigel Unwin, Aaron Klug, Tony Crowther and others, with first-year PhD students like me as instructors and luminaries such as Michael Rossmann, the world-famous protein crystallographer, as participants.\nLHC-II has a natural, fortuitous (but unphysiological) tendency to form crystalline arrays when the isolated, detergent-solubilized complex is precipitated from solution with monovalent or divalent cations. That much was known from freeze-fracture electron microscopy when I started my postdoc at the ETH in Ernst Wehrli's group, but it took more than two years to persuade the complex to form large, well-ordered, purple-membrane-like 2D crystals that would lend themselves to electron diffraction and high-resolution image processing [&lt;&lt;REF:Kühlbrandt-1983-96-1414&gt;&gt;]. It took another year to work out an initial structure of negatively stained LHC-II membrane crystals by electron crystallography [&lt;&lt;REF:Kühlbrandt-1984-307-478&gt;&gt;]. The map resolution was 16 A. None of this was cryoEM, but the stage was set.\n\nLHC-II at liquid-helium temperature\nSince there were no instruments for high-resolution cryoEM anywhere nearby, I took the 2D crystals to Berkeley, where Ken Downing and I were able to record images for a 3.7 A projection map of LHC-II with their JEOL 100B [&lt;&lt;REF:Kühlbrandt-1989-207-823&gt;&gt;]. The visit proved that the crystals had potential, but large-scale data collection with this ageing machine (bits of the mu metal magnetic shielding sometimes dropped off the ceiling during a recording session) was out of the question. The temperamental home-built cryo-stage did not go much below -120degreesC. Occasionally, the column vacuum would collapse for no apparent reason, and on the fluorescent screen, I could watch my precious sample freeze over with a thick layer of contaminating ice. Fixing the microscope and getting it ready for another cryo-transfer took Ken and his team most of the day. Because I was not able to help and there was nothing else to do, I had ample time to read and explore the beautiful UC Berkeley campus. I also took the 2D crystals to Berlin, but Suleika's long-winded and idiosyncratic image recording routine would not have made large-scale data collection easy either. In any case, the Berlin group seemed none too keen to accommodate a young whipper-snapper like me, who was not even a professor. A perfect solution offered itself when I visited Yoshinori Fujiyoshi, then at the Protein Engineering Research Laboratory (PERI) in Osaka. Yoshi had designed and built a top-entry, helium-cooled cold stage for his 400-kV JEOL electron microscope [&lt;&lt;REF:Fujiyoshi-1991-38-241&gt;&gt;]. The stage was particularly suitable for high-resolution cryoEM of 2D crystals, and Yoshi welcomed me with open arms. The lens was not superconducting, and therefore, it took only ~10 manual operations to record one film image, which made data collection feasible.\nI used my Heisenberg Fellowship from the DFG for repeated visits to Yoshi's laboratory. I took the developed negatives back to EMBL Heidelberg, where in the meantime I had taken up a research group leader position. Together with my first postdoc Da Neng Wang, now Professor of Structural Biology at New York University, we scanned and processed the films. Within a year or two, we had gathered a sufficient number of images for a 3.4 A structure of LHC-II [&lt;&lt;REF:Kühlbrandt-1994-367-614&gt;&gt;] (Fig. 1 &lt;&lt;FIG:F1&gt;&gt;). The complementary high-resolution electron diffraction patterns required no more than a standard electron microscope with a liquid-nitrogen-cooled Gatan cryo-transfer holder, and Da Neng and I collected them with the JEOL 2000 at EMBL [&lt;&lt;REF:Wang-1992-61-287&gt;&gt;]. The 3.4 A map of LHC-II was the second high-resolution structure determined by cryoEM (again a membrane protein) and proved that the method worked for specimens other than bacteriorhodopsin. The map afforded fascinating insights into how chlorophylls and carotenoids were held and oriented by the polypeptide scaffold in an optimal position for harvesting solar energy and photoprotection.\nFig. 1.\nStructure of plant light-harvesting complex LHC-II determined by electron crystallography of two-dimensional crystals at liquid-helium temperature. (a) Atomic model of LHC-II trimers on the 2D crystal lattice. Polypeptide (purple) with bound chlorophylls (green and cyan) and carotenoids (yellow). (b) Details of the 3.4 A map with fitted polypeptide (white), chlorophylls (green) and carotenoids (yellow). From [&lt;&lt;REF:Kühlbrandt-1994-367-614&gt;&gt;].\nThe LHC-II structure was the overture to a number of highly successful collaborations for which colleagues from other laboratories teamed up with Yoshi to collect data with his superb instrument. Tom Walz came with his 2D crystals of aquaporin he had grown with Andreas Engel in Basel (Switzerland) [&lt;&lt;REF:Walz-1997-387-624&gt;&gt;] and Inga Schmidt-Krey and Hans Hebert with theirs of microsomal glutathione transferase [&lt;&lt;REF:Hebert-1997-271-751&gt;&gt;]. An enduring partnership ensued with Nigel Unwin. Nigel recorded countless images of his tubular crystals of nicotinic acetyl-choline receptor at increasingly higher resolution with Yoshi's microscopes [&lt;&lt;REF:Miyazawa-1999-288-765&gt;&gt;,&lt;&lt;REF:Miyazawa-2003-423-949&gt;&gt;], culminating in a 4 A map of the receptor in the membrane that resolved sidechains [&lt;&lt;REF:Unwin-2005-346-967&gt;&gt;]. Yoshi was accumulating a growing number of helium-cooled 400-kV and later 300-kV instruments, each of them filling a fairly large room, which all moved with him from Nara to Kyoto and then to Nagoya. When Nigel was awarded the Jeantet Prize in recognition of his groundbreaking work on the acetyl-choline receptor, he bought an apartment in Kyoto for his frequent stays, and Yoshi kept a JEOL 4000 specially for him that the Kyoto team called Janet. When I was appointed as a director at the Max Planck Institute of Biophysics in Frankfurt in 1997, it was my greatest wish to set up my own Yoshi-style microscope, and I was fortunate in that the Max Planck Society afforded me the means to do so. Like Yoshi's latest model it was a 300-kV instrument with a liquid-helium-cooled top-entry stage, and it worked very reliably until 2012. Liquid helium was roughly twice as effective in protecting the specimen against radiation damage as liquid nitrogen [&lt;&lt;REF:Stark-1996-63-75&gt;&gt;,&lt;&lt;REF:Fujiyoshi-1998-35-25&gt;&gt;], but also several times more expensive, and supplies were dwindling. Fortunately, it was possible to compensate for the lower signal-to-noise ratio by collecting and processing a correspondingly larger number of images at liquid-nitrogen temperature. When the opportunity arose, we replaced the JEOL 3000 SFF with a similar model, the JEOL 3200 FSC, which had a side-entry stage that could be cooled with either liquid nitrogen or liquid helium. Liquid nitrogen was less demanding and less costly, and we used it for cooling this new instrument almost exclusively.\nYears later, we discovered another and--at least for me--completely unexpected disadvantage of liquid-helium cooling. It turned out that beam-induced movement of unsupported cryo-specimens in vitreous water is up to four times worse at liquid-helium than at liquid-nitrogen temperature [&lt;&lt;REF:Pfeil-Gardiner-2019-6-1099&gt;&gt;]. This seemed to matter less for 2D crystals, which were routinely deposited on a continuous carbon support (even so, at least 80% of the film negatives showed signs of severe beam-induced movement, which made them useless). However, it does matter greatly for single-particle cryoEM. In retrospect, this finding explained why we had never been able to record any good single-particle images with the helium-cooled JEOL 3000. The reasons for the excessive beam-induced movement of vitrified solutions at liquid-helium temperature are not understood and need to be investigated further, if the superior cryo-protection qualities of liquid helium are ever to be taken advantage of for single particles. The causes may be related to the poor conductivity and viscous fluidity of vitreous water at very low temperatures, where carbon or graphene films are both highly conductive and mechanically stable.\n\n2D and 3D crystals of membrane proteins\nThe liquid-helium-cooled top entry stage of the JEOL 3000 SFF was particularly appropriate for 2D crystals, but suitable specimens were hard to come by. No more than around 10 2D protein crystals are known to diffract electrons to better than 4 A: purple membrane; LHC-II; the bacterial porins OmpF and PhoE; aquaporins-0, 1 and 4; microsomal glutathione transferase and its relatives (for reviews, see [&lt;&lt;REF:Kühlbrandt-1992-25-1&gt;&gt;,&lt;&lt;REF:Abeyrathne-2012-?-277&gt;&gt;]); and zinc-induced sheets of tubulin [&lt;&lt;REF:Nogales-1998-391-199&gt;&gt;], the only non-membrane protein amongst them. All are naturally abundant and readily available from natural sources in milligram quantities and thus amenable to extensive 2D crystallization screens. In the late 1990s, it became routine to produce--at first mostly bacterial--membrane proteins in similar amounts by heterologous expression. Quite a few of these formed 2D arrays, mostly in the shape of wide, flattened tubular vesicles. In our JEOL 3000, some yielded useful projection maps at 4-7 A or 3D structures at 6-8 A resolution by electron crystallography, notably the sodium-proton antiporters NhaA [&lt;&lt;REF:Williams-1999-18-3558&gt;&gt;,&lt;&lt;REF:Williams-2000-403-112&gt;&gt;] and NhaP [&lt;&lt;REF:Paulino-2014-3-?&gt;&gt;,&lt;&lt;REF:Paulino-2014-3-?&gt;&gt;], the osmoregulatory transporter BetP [&lt;&lt;REF:Ziegler-2004-337-1137&gt;&gt;], the bacterial protein translocase SecYEG [&lt;&lt;REF:Breyton-2002-418-662&gt;&gt;] and the light-gated ion channel channelrhodopsin [&lt;&lt;REF:Müller-2011-414-86&gt;&gt;,&lt;&lt;REF:Müller-2015-427-341&gt;&gt;]. However, the 2D crystals remained small, and none of them were ever as good as LHC-II or purple membrane. At the same time, heterologously expressed membrane transporters invariably produced good 3D crystals, which meant that we [&lt;&lt;REF:Ressl-2009-458-47&gt;&gt;] or others [&lt;&lt;REF:van den Berg-2004-427-36&gt;&gt;,&lt;&lt;REF:Kato-2012-482-369&gt;&gt;] were able to solve their high-resolution structures by X-ray crystallography. With a few rare exceptions, the--mostly hydrophobic--interactions that hold 2D crystals together are weaker and less numerous than the--mostly polar--crystal contacts in a 3D protein lattice, and therefore, the 2D crystals remained small and tended to be poorly ordered. Whatever the reason, for anyone interested in understanding the structure and mechanisms of membrane proteins rather than optimizing the growth of 2D crystals, X-ray crystallography was clearly the way to go. So for the next 10 years, this was what we did, until, very recently, single-particle cryoEM caught up. Structures of even fairly small membrane proteins can now be determined without any crystals at all, at considerably higher resolution.\n\nCryo-tomography of mitochondrial membranes\nAs a method in structural biology, cryoEM has the great advantage of not being restricted to crystals or pure protein solutions. cryoET can examine objects from any biological source of any shape and--within limits--size, provided that they are thin enough to be considered as weak phase objects, so that multiple electron scattering is negligible. In principle, the same microscopes can be used equally well for the three forms of cryoEM--electron crystallography, single-particle analysis and cryoET--as long as the sample can be tilted to about +/-60degrees, which is a prerequisite for tomography and electron crystallography.\nOne of the first non-crystalline specimens we investigated by single-particle cryoEM was the mitochondrial respiratory supercomplex, also called the respirasome. The respirasome is a 1.7-MDa assembly of three mitochondrial respiratory chain complexes, complex I, III and IV, the latter two also known as cytochrome c reductase and cytochrome c oxidase. The supercomplex was first identified on blue-native gels of detergent-solubilized mitochondrial membranes by the Frankfurt membrane biochemist Hermann Schagger [&lt;&lt;REF:Schägger-2000-19-1777&gt;&gt;]. Initially, we extracted it from gels and examined it in negative stain [&lt;&lt;REF:Schäfer-2006-281-15370&gt;&gt;]. Later, we purified it for cryoEM by gradient centrifugation [&lt;&lt;REF:Althoff-2011-30-4652&gt;&gt;]. The resolution of ~20-25 A we achieved with film images was low by today's standards, but good enough to dock X-ray structures of the component complexes, which indicated where they interacted. In the meantime, with better preparation techniques but mainly thanks to better electron detectors and image processing programs (see below), several cryoEM structures of respiratory chain supercomplexes at higher resolution have been published by us [&lt;&lt;REF:Sousa-2016-5-?&gt;&gt;] and others [&lt;&lt;REF:Letts-2016-537-644&gt;&gt;,&lt;&lt;REF:Wu-2016-167-1598&gt;&gt;]. The precise function of these large assemblies and how they work in fine-tuning cellular metabolism and organismal fitness are not known and remain controversial.\nAt the time, there was a debate as to whether the respiratory chain supercomplexes actually existed in mitochondria or whether they were artefacts of detergent solubilization. It occurred to me that, if they existed, they should be visible in tomographic volumes of mitochondrial membranes. The membranes were easy enough to prepare, but we had never done cryo-tomography. Mike Strauss, who was then in the middle of his PhD and now runs the cryoEM facility at McGill University in Montreal, visited the laboratory of Grant Jensen at CalTech to learn the technique. He stayed for 2 weeks and on his return to Frankfurt established the procedure on our recently acquired FEI Polara electron microscope. Compared to 2D crystallography with the JEOL 3000 or single-particle cryoEM, cryo-tomography with the Polara turned out to be remarkably straightforward.\nWe collected tilt series of isolated inner mitochondrial membranes and generated 3D tomographic volumes, as Mike had learned at CalTech. But when we examined the volumes in the computer, the first thing that struck us was not the respirasome, but long rows of lollipop-shaped 10-nm particles that were clearly the F1 heads of the mitochondrial F1Fo ATPase [&lt;&lt;REF:Strauss-2008-27-1154&gt;&gt;]. We were intrigued that seemingly identical rows showed up in consecutive parallel slices through the tomographic volumes. At first, we suspected a problem with the program that had generated the tomograms, but when we discovered parallel rows of the same particles in perpendicular views, it dawned on us that they were rows of ATP synthase dimers. Dimeric forms of the mitochondrial ATP synthase had been known for some time from negative-stain single-particle EM [&lt;&lt;REF:Minauro-Sanmiguel-2005-102-12356&gt;&gt;,&lt;&lt;REF:Dudkina-2006-580-3427&gt;&gt;], but because they all looked different, I thought they were bound to be artefacts. Also, it was difficult to see how a V-shaped dimer of two such large assemblies could sit stably in a planar lipid bilayer.\n\nMitochondrial ATP synthase dimers\nOur tomographic volumes showed that the membranes were of course far from planar. The edges of the inner membrane cristae were tightly bent, and the dimer rows always sat in the regions with the smallest, ~17-nm curvature radius. The dimer rows actually induce high local membrane curvature and indeed cristae formation, as we now know and were able to demonstrate by reconstituting purified ATP synthase dimers into liposomes [&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;]. Tomographic volumes of the reconstituted proteoliposomes resembled those of mitochondrial inner membranes closely and the membrane curvature at the position of the dimer rows was the same [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;]. The rows run along the tightly bent cristae rims or ridges, but they are absent from the remaining regions of the inner membrane that are indeed roughly planar. Long rows of ATP synthase dimers are a defining feature of the mitochondrial inner membrane, and the high local curvature of the ridges is characteristic of metazoan and fungal mitochondria. Dimer rows are ubiquitous in mitochondria of all of the roughly 15 species we investigated, even though the dimers themselves vary greatly in terms of overall structure and subunit composition [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;] (Fig. 2 &lt;&lt;FIG:F2&gt;&gt;). ATP synthase dimers of metazoans and fungi differ significantly from those of ciliates such as Paramecium [&lt;&lt;REF:Mühleip-2016-113-8442&gt;&gt;], which in turn differ from those of trypanosomes and Euglena [&lt;&lt;REF:Mühleip-2017-114-992&gt;&gt;]. Dimers of unicellular green algae including Chlamydomonas and Polytomella are strikingly different from all the above [&lt;&lt;REF:Dudkina-2010-1797-272&gt;&gt;,&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;], and green-plant dimers are again different (unpublished). Yet, mitochondria of all these groups have dimer rows and their inner membranes form cristae, even though their appearance varies from lamellar (metazoans, fungi, green plants) to tubular (ciliates) to anywhere in between (Polytomella, trypanosomes). The dimer rows are often &gt;1 microm long [&lt;&lt;REF:Strauss-2008-27-1154&gt;&gt;] and conspicuous in tomographic volumes of all unfixed, flash-frozen mitochondria. The reason why they had escaped notice for so long is that they do not survive the preparation steps (chemical fixation, dehydration, plastic embedding and heavy-metal staining) for conventional room-temperature EM. They are difficult to see by any technique other than cryoET, although they have been spotted once by rapid-freeze deep-etch electron microscopy of Paramecium mitochondria [&lt;&lt;REF:Allen-1989-108-2233&gt;&gt;].\nFig. 2.\nCryo-ET of mitochondrial ATP synthase dimers. Subtomogram averages (above) and rendered tomographic volumes of inner mitochondrial membranes (below) from (a) Saccharomyces cerevisiae, (b) Polytomella sp., (c) Paramecium tetraurelia and (d) Euglena gracilis. From [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;].\nATP synthase dimers, dimer rows and inner membrane cristae seem to play a central part in cellular and organismal physiology, as do the mitochondria themselves. The tight curvature at the cristae ridges is a prerequisite for close lamellar stacking, e.g. in cardiac muscle [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;] (Fig. 3a &lt;&lt;FIG:F3&gt;&gt;). Cristae stacking increases the inner membrane surface area, and therefore, the potential for ATP production, by a factor of ~10 [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;], but this is most likely not the main reason why the dimers rows are so important. Yeast mutants lacking the dimer-specific ATP synthase subunits e and g that stabilize the dimers do not have dimer rows and do not develop cristae [&lt;&lt;REF:Davies-2012-109-13602&gt;&gt;]. The mutants are less fit than wild type, grow more slowly and, according to one study, their membrane potential is reduced [&lt;&lt;REF:Bornhövd-2006-281-13990&gt;&gt;]. Even more strikingly, Drosophila fruit flies in which subunits equivalent to e and g are knocked down are deficient in germline stem cell maturation [&lt;&lt;REF:Teixeira-2015-17-689&gt;&gt;]. Mitochondria of these knockdowns do not have cristae, which points to a profound role of the cristae in Drosophila development. At present, we cannot even guess what this role might be.\nFig. 3.\nCryo-ET of whole mitochondria from (a) mouse heart and (b) mouse liver. Upper panels: slices through tomographic volumes (scale bar, 250 nm). Lower panels: segmented 3D volumes of two typical mitochondria. Heart mitochondria have closely stacked, roughly parallel cristae (blue). Cristae of liver mitochondria are more loosely arranged and not stacked. Grey, inter-membrane space; light yellow, outer membrane. From [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;].\nAfter the dimer rows had sidetracked us for some time, we collected more tomograms of mitochondrial membranes to look for respiratory chain complexes and respirasomes. Indeed, they were there, in the flat membrane regions on either side of the ATP synthase rows. We detected the L-shaped density of the 1-MDa complex I next to ATP synthase dimers in four different species and were able to confirm its identity by antibody labelling [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;]. The resolution and quality of the tomograms, recorded in our Polara with a 2k charge-coupled device (CCD) camera and Tridiem filter, was not sufficient to tell reliably whether or not they contained complex III or IV as well, which would have meant that they were respirasomes. A few years later, after we had installed our first Titan Krios electron microscope with a Quantum energy filter and Gatan K2 detector, we re-examined isolated inner membranes of bovine, fungal (Yarrowia lipolytica) and plant (Asparagus officinalis) mitochondria by cryoET. The tomographic volumes were very much clearer and enabled us to generate sub-tomogram averages of respirasomes from the three species at about 25 A resolution [&lt;&lt;REF:Davies-2018-115-3024&gt;&gt;] (Fig. 4 &lt;&lt;FIG:F4&gt;&gt;). Like the dimer rows, the supercomplexes were ubiquitous. They looked similar in mammalian, yeast and plant mitochondria, which told us that interactions between the two major components, complexes I and III, were conserved.\nFig. 4.\nSubtomogram averages of respiratory chain supercomplexes in inner membranes of (a) bovine heart, (b) fungal (Y. lipolytica) and (c) plant mitochondria. The arrangement of complex I (blue) and complex III dimer (orange) is conserved, as indicated by the blue and red guidelines. Adapted from [&lt;&lt;REF:Davies-2018-115-3024&gt;&gt;].\n\nCryo-tomography of whole mitochondria\nHaving examined membrane protein complexes in mitochondrial membrane fragments, we were curious to see the dimer rows in intact organelles. We were especially interested in mitochondria of the filamentous fungus Podospora anserina, which has an exceptionally short lifespan of 20 days, and we were wondering how the mitochondria and dimer rows would change during ageing. To our surprise, we found that the inner membrane of mitochondria from aged Podospora underwent a profound transformation within a few days [&lt;&lt;REF:Daum-2013-110-15301&gt;&gt;]. The cristae disappeared, the dimer rows became increasingly disrupted, the dimers broke up into monomers and finally the mitochondrial matrix vesiculated. The double rows along the cristae ridges, or what was left of them in ageing cultures, were easily visible in small organelles [&lt;&lt;REF:Daum-2013-110-15301&gt;&gt;] (Fig. 5 &lt;&lt;FIG:F5&gt;&gt;), but in larger mitochondria, molecular detail was obscured by the dense mitochondrial matrix.\nFig. 5.\nThree-dimensional tomographic volume of a small mitochondrion from P. anserina. The inner membrane is light blue, the outer membrane is grey. Yellow spheres indicate the catalytic F1 heads of ATP synthase dimers arranged in rows on the cristae ridges. Scale bar, 100 nm (B. Daum and W. Kuhlbrandt, MPI of Biophysics, unpublished).\nIn collaboration with the MPI for the Biology of Ageing, we examined age-related changes in mitochondria of mouse and Drosopohila melanogaster, two popular, longer-lived model organisms. The differences were less drastic than in Podospora, but nonetheless clear [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;]. In mouse, changes in inner membrane morphology were strongly organ-dependent. Heart mitochondria were particularly resilient to ageing, such that mitochondria from young and old mice were indistinguishable and equally tightly packed with lamellar cristae (Fig. 3a &lt;&lt;FIG:F3&gt;&gt;). We found that liver mitochondria have fewer cristae, which are not stacked (Fig. 3b &lt;&lt;FIG:F3&gt;&gt;). In old mice, around 20% of liver mitochondria developed a central vacuole that was contiguous with the cristae. In kidney, a similar proportion of mitochondria lost their inner membrane morphology. These changes presumably go along with reduced respiratory activity, ATP production and organellar fitness with increasing age. The picture was similar for Drosophila.\nAs a postdoc in the group, Vicki Gold used cryoET to investigate protein import into mitochondria. She generated a pre-protein construct with a mitochondrial import signal sequence and conjugated it with fluorescent quantum dots by biotin/streptavidin biochemistry. When import-competent mitochondria were incubated with this construct, the quantum-dot-labelled pre-protein got stuck in the translocase of the outer mitochondrial membrane, known as the Translocase of the Outer Membrane (TOM) complex. Her approach ensured that only complexes active in protein translocation were labelled, and that she could keep track of them by fluorescence during cryoEM grid preparation. In the tomographic volumes, the dense quantum dots showed up beautifully on the mitochondrial outer membrane, highlighting the location of active import complexes [&lt;&lt;REF:Gold-2014-5-?&gt;&gt;]. By subtomogram averaging, she also managed to visualize the import receptor protein subunits protruding on the cytoplasmic side at low resolution. Apparently, the receptor subunits are flexible, which explains why they have so far eluded us in our single-particle cryoEM studies of the isolated TOM complex (see below).\n\nCryo-tomography of reconstituted membranes\nCryoET also works well for membrane assemblies reconstituted in vitro. Oli Daumke and Katja Faelber at the Max Delbruck Centre in Berlin were determining the X-ray structure of the mitochondrial membrane-remodelling GTPase Mgm1 and wanted to know how it interacts with the lipid bilayer. In collaboration with them, Lea Dietrich was able to assemble a lattice of helical Mgm1 filaments on pre-formed liposomes. The helical arrays shaped the liposomes into thin long tubes, and by image processing and sub-tomogram averaging, she generated a volume that indicated how the GTPase in the X-ray structure interacts with the membrane [&lt;&lt;REF:Faelber-2019-571-429&gt;&gt;]. To our surprise, Lea discovered similar but different helical arrays inside the membrane tubes after some of them had ruptured, so that the protein could diffuse to the inside. The inside Mgm1 lattice generated wider tubes, presumably resembling the situation at tubular cristae junctions, which are most likely stabilized by Mgm1. The topography of the wider tubes would resemble the inner surface of the mitochondrial inner membrane, where Mgm1 is thought to promote membrane fission or fusion. The structures of the inside and outside helical lattices suggested mechanisms by which GTP hydrolysis generates a power stroke that would constrict or dilate the membrane tube [&lt;&lt;REF:Faelber-2019-571-429&gt;&gt;].\n\nPore-forming proteins\nOther membrane systems that lend themselves well to cryoET are bacterial and other pore-forming proteins. Membrane-perforating bacterial toxins are produced, or can be expressed, as soluble monomers which insert into the target membranes to form large circular pores. In collaboration with Trinad Charkaborty at the University of Giessen (Germany), we expressed, crystallized and determined the X-ray structure of the membrane-perforating toxin listeriolysin (LLO) from Listeria monocytogenes [&lt;&lt;REF:Köster-2014-5-?&gt;&gt;]. Later, we turned to pneumolysin (PLY), the virulence factor of the human pathogen Streptococcus pneumoniae. Both toxins insert into cholesterol-containing membranes of animal cells, assembling into ring-like structures of 40 or more monomers [&lt;&lt;REF:van Pee-2016-16-7915&gt;&gt;]. We obtained the structure of the closed PLY prepore in the membrane by cryoET and subtomogram averaging, and that of the fully open, 42-membered ring of the isolated, detergent-solubilized pore complex by single-particle cryoEM [&lt;&lt;REF:van Pee-2017-6-?&gt;&gt;] (Fig. 6 &lt;&lt;FIG:F6&gt;&gt;). Comparison of the prepore and pore structures indicated how the four-domain monomer inserts into the lipid bilayer and then, upon conversion of one of its domains into 80 A beta hairpins, forms ~400 A beta barrels that perforate the bilayer and kill the cell. Molecular dynamics simulation in collaboration with Gerhard Hummer at the MPI of Biophysics revealed the detailed mechanism of pore formation and extrusion of the lipid plug to open the pore [&lt;&lt;REF:Vögele-2019-116-13352&gt;&gt;].\nFig. 6.\nSingle-particle cryoEM structure of the pneumolysin (PLY) toxin pore complex at 4.5 A resolution. The four fitted domains in the 2.4 A X-ray structure of the PLY monomer are shown in colour. One domain has rearranged into ~80 A beta hairpins (cyan) that form a 42-membered ring with an inner diameter of ~400 A perforating the membrane. From [&lt;&lt;REF:van Pee-2017-6-?&gt;&gt;].\nA very different way in which a protein can perforate a membrane, this time from the inside to allow a virus to escape from an infected cell, was brought to our attention by David Prangishvili at the Institut Pasteur in Paris, France. David was studying phages that infect certain thermophilic archaea, inducing them to produce large numbers of a small, 10-kDa membrane protein encoded in the viral genome. The gene product inserts spontaneously into the archaeal membrane, where it self-assembles into ~1000 A pyramids of 7-fold symmetry that pierce through the S-layer. The S-layer forms a tough protective lattice around archaea living in extreme environments and would otherwise trap the virus in the cell. Once the S-layer is punctured, the sides of the pyramids open like the petals of a flower to create ~200 nm holes, through which the new virions emerge [&lt;&lt;REF:Daum-2014-111-3829&gt;&gt;].\n\nChloroplast tomography\nEver since our LHC-II work in the 1990s, we had kept a close interest in chloroplasts, the characteristic membrane organelles of green plants and algae, and their photosynthetic membrane proteins, including the photosystem II reaction centre [&lt;&lt;REF:Rhee-1997-389-522&gt;&gt;,&lt;&lt;REF:Rhee-1998-396-283&gt;&gt;]. Chloroplasts are much larger than mitochondria, measuring around 5 microm across. They are too bulky and dense for direct tomographic imaging of plunge-frozen samples, the approach we had taken with whole mitochondria. At the beginning of his PhD, Bertram Daum (now a PI at the University of Exeter, UK) found out that chloroplasts ruptured by mechanical blotting on the cryoEM grid maintained the intricate molecular organization of their membrane networks at least for the few seconds it takes to flash-freeze them [&lt;&lt;REF:Daum-2010-22-1299&gt;&gt;,&lt;&lt;REF:Daum-2011-62-2393&gt;&gt;]. The protein-dense chloroplast stroma had leaked out, which improved the contrast and resolution of the tomograms. By cryoET of such samples, Bertram examined the arrangement and distribution of large membrane protein complexes in chloroplast thylakoids. We were especially interested in the chloroplast ATP synthase, to see whether it formed dimers and dimer rows as in mitochondria. The tomographic volumes of chloroplast grana stacks and unstacked stromal lamellae showed unambiguously that this was not the case. The chloroplast ATP synthase was homogenously distributed over the stroma thylakoids and grana end membranes, and all copies were monomers (Fig. 7 &lt;&lt;FIG:F7&gt;&gt;). In particular, there were no dimers or dimer rows around the tightly curved edges of the thylakoid grana, where one might have expected to find them. In retrospect, this is not surprising since the chloroplast ATP synthase lacks the subunits that are required for dimer formation [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;]. The same is true of the related bacterial ATP synthases, which are likewise all monomers.\nFig. 7.\nCryoET of chloroplast thylakoid membranes. F1 heads (yellow) of the chloroplast F1Fo ATP synthase are confined to stroma and grana end membranes. The chloroplast ATP synthase is monomeric and not found at the highly curved grana edges. Grana stacks (dark green); stroma membranes (transparent green). Scale bar, 100 nm. Courtesy of Bertram Daum, MPI of Biophysics.\nMore recently, when he joined the laboratory as a PhD student, Davide Floris adopted the same approach of specimen preparation to examine the molecular landscape of etioplasts, the precursor organelles of chloroplasts in the leaves of greening plants. The etioplast inner membrane is special because it forms the pro-lamellar body, a meshwork of tubular membranes arranged in the lattice of a lipidic cubic sponge phase. This had been known since the 1970s from conventional EM of plastic sections, but no one knew how or why the membranes were arranged in this peculiar way that appears to have no parallel in biology. By cryoET, Davide discovered that the membrane tubes were surrounded, and indeed formed, by helical arrays of large numbers of a small, membrane-associated protein. By sub-tomogram averaging, he obtained a map at 9 A resolution and was able to show that the protein is the 37-kDa light-dependent protochlorophyllide oxidoreductase (LPOR) [&lt;&lt;REF:Floris-2021-7-514&gt;&gt;]. LPOR is the most abundant enzyme in etioplasts. It converts the pale yellow protochlorophyllide into chlorophyllide, which in turn is converted into the deep-green chlorophyll by chlorophyll synthase. Both enzymatic reactions are key steps in the biogenesis of photosynthetic reaction centres and light-harvesting complexes of chloroplasts, including LHC-II that consists of 33% chlorophyll by weight.\n\nDirect electron detectors\nBefore 2012, only very few cryoEM structures reached a resolution of 3.5-4 A, where large sidechains become visible. All were of 2D crystals, mostly of membrane proteins (see above), or of large icosahedral viruses (see [&lt;&lt;REF:Grigorieff-2011-21-265&gt;&gt;]). For assemblies smaller than ~20 MDa that did not consist of large numbers of identical units related by strict crystallographic, helical or point-group symmetry, high resolution was out of the question. The reason was the poor signal-to-noise ratio of individual electron micrographs, which meant that smaller particles could not be aligned and averaged with sufficient accuracy to warrant high resolution. Until that point, photographic film was by far the best recording medium for cryoEM. Although tedious to use, it was very much better than the solid-state CCD electron detectors available at the time, which were quick and convenient, but their modulation transfer function (MTF), and hence, detective quantum efficiency (DQE) at high resolution was poor. Richard Henderson's incisive 1995 review [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;] had shown beyond any doubt that of the three forms of radiation which can in principle deliver images of individual biological macromolecules--X-rays, neutrons and electrons--only the latter held the promise of producing such images at 3 A resolution or better. Therefore, if cryoEM was ever to realize this promise and come close to the success of X-ray crystallography, electronic detectors with a signal-to-noise ratio and DQE at least as good as film would be mandatory.\nIn 2004, Richard set up a consortium of engineers and members of the cryoEM community to develop such a detector. Apart from himself and his colleagues Greg McMullan and Wasi Faruqi at the MRC-LMB, core members of the team were Renato Turchetta and Nicola Guerrini at the Rutherford Appleton Laboratories (RAL) outside Oxford, UK. Both were brilliant electronics engineers with expertise in detector chip design. Together, Renato and Nicola embarked on designing and building a solid-state device with all the desirable properties of an optimal electron detector: a large active area (at least 4000 x 4000 pixels), suitable pixel size (the consortium decided on 14 microm), a DQE as close as possible to 1 (where 1.0 would be that of a perfect detector), high sensitivity, rapid readout, low readout noise, high resistance to electron irradiation and reasonable production cost. These very demanding specifications were defined in preliminary consortium meetings. The new generation of detectors would be based on complementary metal oxide semiconductor technology that was already in widespread use in digital cameras and mobile phones, but needed to be adapted to the stringent requirements of high-resolution cryoEM. Although Richard could easily have shouldered the whole effort by himself, he was intent on putting the endeavour on a broader scientific base and invited me and Jurgen Plitzko of the MPI for Biochemistry in Martinsried (Germany) to join the consortium. The FEI company in Eindhoven (The Netherlands; acquired by Thermo Fisher in 2017) joined with a view of producing and marketing the detector in due course. The LMB, the two MPIs, FEI and RAL each contributed 20% to the considerable cost of getting this project off the ground. It was clear to us that if these new direct electron detectors could be made to work, everyone in the cryoEM world would want to have one. We now know they work far better than we could have imagined in our wildest dreams and everyone does want one. Indeed, most cryoEM groups have at least one already.\nInitially, test chips with different pixel topologies were designed and produced at RAL and evaluated by the LMB detector development team. Criteria included signal-to-noise ratio, radiation hardness and DQE. Results exceeded expectations [&lt;&lt;REF:McMullan-2009-109-1126&gt;&gt;,&lt;&lt;REF:Faruqi-2011-44-357&gt;&gt;]. The most promising pixel design was then selected for further development. Before the camera chip went into production, the consortium decided that the silicon wafer of the detector should be back-thinned to ~30-50 microm, which improved the signal-to-noise ratio and MTF substantially [&lt;&lt;REF:McMullan-2009-109-1144&gt;&gt;]. Eventually, FEI produced a back-thinned detector chip for a camera called the Falcon-II, and each consortium member obtained one prototype.\nOf course, our consortium was not the only initiative in the world to develop direct electron detectors. The US company Gatan Inc., who had long experience in producing CCD cameras and energy filters for cryoEM, came out with their own design more or less simultaneously, as did the smaller Direct Electron company in San Diego, CA. Gatan's direct electron detector camera was called the K2 Summit, and the one from Direct Electron was the DE-20. The sensor areas of all three cameras were similar (roughly 4000 by 4000 pixels), but the pixels of both the K2 Summit and the DE20 were very much smaller (5 and 6.4 microm). All three performed well (see [&lt;&lt;REF:McMullan-2014-147-156&gt;&gt;]), but in practical terms, the high readout speed of the K2 (400 frames per second) meant that up to 20 times more images could be recorded per hour. Primarily for this reason, the K2 became particularly popular with cryoEM labs. It was also the camera of choice for cryoET, not least because the Falcon-II (produced by FEI) was not on offer in combination with an energy filter (essential for tomography), and post-column energy filters were at the time only produced by FEI's competitor Gatan. On the other hand, the larger pixel of the Falcon-II meant that it had a better DQE at high spatial frequencies, which made it the best choice for high-resolution cryoEM of large single particles. Since then, both the Gatan K2 and the Falcon-II have been upgraded to the Gatan K3 and Falcon-III (and now the Falcon-IV), each with significantly improved performance, although the K3 has the same pixel as the K2. Even better, faster and more sensitive direct electron detectors are bound to be developed in the years to come.\n\nThe resolution revolution\nWe had our Falcon-II camera installed on our Polara electron microscope, and by October 2012, it performed to specification. With Greg McMullan's help, we recorded our first high-resolution images of the Frh complex a few months later. Frh is a rigid, soluble, dodecameric assembly with a mass of 1.2 MDa that was isolated and purified from methanogenic archaea by our colleagues Seigo Shima and Stella Vitt at the MPI of Terrestrial Microbiology in Marburg (Germany). We had just published a nice cryoEM map of Frh from images recorded on Kodak SO163 film [&lt;&lt;REF:Mills-2013-2-?&gt;&gt;]. The nominal resolution was 5.5 A, although beta-strands were fully resolved and the map therefore looked much better. With the Falcon-II, we were able to improve the nominal resolution to 3.4 A within a few weeks [&lt;&lt;REF:Allegretti-2014-3-?&gt;&gt;]. Since then, we have improved the map further to just better than 2 A with the Falcon-III and K3 cameras (Fig. 8 &lt;&lt;FIG:F8&gt;&gt;). The Frh map was one of the first single-particle cryoEM structures in this resolution range. The other three were the 20S proteasome from Thermoplasma thermophilum at 3.3 A [&lt;&lt;REF:Li-2013-10-584&gt;&gt;], the TRPV1 ion cannel from rats at 3.4 A [&lt;&lt;REF:Liao-2013-504-107&gt;&gt;] and the large subunit of the yeast mitochondrial ribosome at 3.2 A [&lt;&lt;REF:Amunts-2014-343-1485&gt;&gt;]. Two of them (Frh and the mitoribosome large subunit) were done with the Falcon-II, the other two with the K2 Summit. Together, these four structures marked the beginning of what has become known as the 'resolution revolution' in cryoEM [&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;], which is still gathering momentum.\nFig. 8.\n(a) CryoEM structure of Frh complex at 1.97 A resolution, with each of the 12 heterotrimers in a different colour. Small red densities on the protein surface are water molecules. (b, c) Map details with fitted polypeptide chain and metal ions (Fe, orange; Ni, green). Sulphur is yellow. The iron in (c) coordinates one carbon monoxide and two cyanide ligands (J. Vonck and D. J. Mills, unpublished).\nAlthough the resolution revolution was triggered by the direct electron detectors, it could not have progressed with the same force and speed without a set of powerful image processing programs that fortuitously became available around the same time. These programs enabled the cryoEM community to take full advantage of the superior image quality delivered by the new detectors. The high sensitivity and fast readout speed of the new cameras meant that the electron dose of ~20 e/A2 for a high-resolution micrograph could be spread over a series of movie frames. This made it possible to measure [&lt;&lt;REF:Brilot-2012-177-630&gt;&gt;] and correct for beam-induced movement [&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;] that had been a persistent and intractable problem with film images. With the new cameras, it became convenient and quick to determine and correct for the contrast transfer function at the stage of image recording, e.g. by the programs ctffind4 [&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;] or Gctf [&lt;&lt;REF:Zhang-2016-193-1&gt;&gt;]. Well-known and much used image program suites include EMAN2 [&lt;&lt;REF:Tang-2007-157-38&gt;&gt;], Frealign [&lt;&lt;REF:Grigorieff-2007-157-117&gt;&gt;] and CryoSparc [&lt;&lt;REF:Punjani-2017-14-290&gt;&gt;]. The most popular and perhaps most incisive program suite is RELION, developed by Sjors Scheres at the MRC-LMB [&lt;&lt;REF:Scheres-2012-415-406&gt;&gt;,&lt;&lt;REF:Zivanov-2018-7-?&gt;&gt;], on the basis of the Xmipp programs from Jose-Maria Carazo in Madrid [&lt;&lt;REF:Scheres-2008-3-977&gt;&gt;]. RELION applies rigorous Bayesian maximum-likelihood statistics and an intuitively obvious and generally accepted 'gold standard' resolution criterion based on an objective comparison of maps generated from two random halves of a data set. As the resolution revolution progresses, top software developers in protein crystallography and structural biology are joining the fray, devising powerful programs, e.g. phenix_resolve [&lt;&lt;REF:Terwilliger-2020-17-923&gt;&gt;], to improve high-resolution cryoEM maps further.\n\nSingle-particle cryoEM of membrane proteins\nEven though most single-chain membrane proteins are at present too small for high-resolution cryoEM, they often form dimers or higher oligomers, or associate with other subunits into complexes, which brings them into a suitable size range for single-particle analysis. When Arne Moller joined the department as a group leader in 2015, he focused on two such membrane proteins. His long-standing interest in ABC transporters came to fruition in the cryoEM structure of TmrAB, a 137-kDa heterodimeric ABC transporter [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;]. A bound camelid nanobody added ~15 kDa to the mass of the heterodimer, breaking its near-2-fold symmetry and helping alignment. The membrane environment of a lipid nanodisc stabilized the protein without restricting functionally important conformational changes. Arne's team was able to resolve no fewer than eight different conformations of the transporter under turnover conditions at resolutions between 2.8 and 4.2 A (Fig. 9 &lt;&lt;FIG:F9&gt;&gt;). Together, the different conformations provided detailed snapshots of the complete transport cycle, as an impressive demonstration of the post-revolutionary power of cryoEM. The other moderately sized membrane protein studied by Arne and his team was a ~210-kDa P-type ATPase from yeast that works as a lipid flippase [&lt;&lt;REF:Timcenko-2019-571-366&gt;&gt;]. The project was a collaboration with Poul Nissen in Aarhus (Denmark), who has a long and distinguished track record in protein crystallography of P-type ATPases. Arne and colleagues were able to resolve three different states, which together illustrated the molecular mechanism by which lipids are moved from one membrane leaflet to the other.\nFig. 9.\nCryoEM structures of the ABC transporter TmrAB under turnover conditions. (a) Narrow and wide inward-facing conformations. (b) In the ATP-bound state, the nucleotide-binding domains associate closely and the exporter is either in an outward-facing open or occluded conformation. (c) Outward-facing open and occluded conformations also dominate the vanadate-trapped state. (d) Asymmetric states during TmrAB turnover resemble an outward-facing occluded conformation with slightly separated nucleotide-binding domains. TmrA is blue, TmrB is yellow and the bound nanobody is light grey. From [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;].\n\nProtein translocases\nFor years, we had been fascinated by membrane protein complexes that translocate pre-proteins into or out of cells or into cell organelles. The first of these was the protein translocase SecYEG from E. coli. As a postdoc in the group, Ian Collinson had grown 2D crystals of the complex, which Cecile Breyton analysed by electron crystallography, yielding a ~8 A 3D map [&lt;&lt;REF:Breyton-2002-418-662&gt;&gt;,&lt;&lt;REF:Bostina-2005-352-1035&gt;&gt;]. The collaboration continued when Ian, by then Professor of Biochemistry at the University of Bristol (UK), was able to bind a pre-protein mimic to SecY. This gave rise to a 3D map at similar resolution, which indicated how the substrate unlocks the complex for protein export [&lt;&lt;REF:Hizlan-2012-1-21&gt;&gt;].\nAnother of our long-standing interests was the mitochondrial TOM complex. An initial collaboration with Nikolaus Pfanner at the University of Freiburg (Germany) resulted in the cryoEM structure of a heterotrimeric form from yeast mitochondria at ~18 A resolution. Data were recorded on film, first with an FEI Technai F20, and then with the Polara [&lt;&lt;REF:Model-2008-383-1049&gt;&gt;]. We revisited the project after the arrival of the direct detectors, this time together with Stephan Nussberger, who had been my first PhD student at EMBL and was now a professor at the University of Stuttgart (Germany). Stephan had tried for 10 years to crystallize the TOM complex from the filamentous fungus Neurospora crassa, but the crystals never diffracted to better than ~8 A. He had however fine-tuned the preparation of the complex from N. crassa mitochondria, which put us in a position to make rapid progress by cryoEM. As a PhD student, Thomas Bausewein collected single-particle images of the TOM complex in detergent or reconstituted into amphipols with our JEOL 3200 FSC at liquid-nitrogen temperature, which he processed to generate a map at 6.8 A resolution [&lt;&lt;REF:Bausewein-2017-170-693&gt;&gt;]. The map showed how the two Tom40 beta-barrels in the dimeric complex form a twin pore. The alpha-helical Tom20 preprotein receptor and the small Tom7, 6 and 5 subunits were assigned on the basis of the shape and size of their map density. Subunit assignment was helped by mass spectrometry indicating the size and sequence of subunits present in the complex. Since then, two higher-resolution cryoEM structures of the S. cerevisiae TOM complex have both confirmed our subunit assignments and orientations [&lt;&lt;REF:Araiso-2019-575-395&gt;&gt;,&lt;&lt;REF:Tucker-2019-26-1158&gt;&gt;].\n\nRespiratory chain complexes\nAn interesting side product of the TOM project was the cryoEM structure of the N. crassa cytochrome c oxidase, which co-purified accidentally with the TOM complex by Ni affinity chromatography. Its identity was revealed by the shape of the 5.5 A cryoEM map [&lt;&lt;REF:Bausewein-2019-6-773&gt;&gt;]. As far as we know, this is the first time that an unknown protein contaminant has been identified solely on the basis of its cryoEM structure, but it will almost certainly not be the last.\nIn collaboration with Manuela Pereira at the ITQB in Lisbon (Portugal), Joana Sousa worked out the 3.9 A cryoEM structure of a bacterial Alternative Complex III [&lt;&lt;REF:Sousa-2018-9-?&gt;&gt;]. More recently, Alex Hahn in my group and Schara Safarian in Hartmut Michel's department at our institute obtained a superb 2.7 A map of a bacterial cytochrome bd oxidase that resolved a number of water molecules in its two proton channels [&lt;&lt;REF:Safarian-2019-366-100&gt;&gt;]. Hartmut Michel, who pioneered X-ray crystallography of membrane proteins in the 1980s, is perhaps the most prominent recent convert in the growing number of protein crystallographers who now use cryoEM almost exclusively.\n\nComplex I\nFor ~20 years, we had collaborated closely with the group of Uli Brandt and Volker Zickermann at the Frankfurt University Medical School to study the structure of complex I from the aerobic yeast Yarrowia lipolytica by electron microscopy, initially in negative stain [&lt;&lt;REF:Zickermann-2003-278-29072&gt;&gt;]. Complex I from Y. lipolytica has an important advantage over the mammalian complex in that it is readily accessible to directed mutagenesis. Brandt and Zickermann had established a genetic system to modify the complex of this particular organism, since the more established bakers' yeast S. cerevisiae lacks complex I. This long-term investment is now paying off. The group had been quite successful in growing 3D crystals and determining the X-ray structure of this ~1-MDa membrane complex, first at 6.3 A [&lt;&lt;REF:Hunte-2010-329-448&gt;&gt;] and then at 3.6-3.9 A resolution [&lt;&lt;REF:Zickermann-2015-347-44&gt;&gt;]. They were thus reluctant to jettison crystallography and EM of negatively stained samples in favour of high-resolution cryoEM. Only when K.R. Vinothkumar (who had been my PhD student in Frankfurt and now runs the first Indian cryoEM facility in Bangalore) together with Judy Hirst in Cambridge published the structure of the bovine complex at 5 A resolution [&lt;&lt;REF:Vinothkumar-2014-515-80&gt;&gt;] did they see the point of the new method, but by that time we had almost missed the boat.\nOur first cryoEM structure of the Yarrowia complex [&lt;&lt;REF:D’Imprima-2016-1857-1935&gt;&gt;] was not at very high resolution (7.9 A), but did resolve the 8 FeS clusters in the peripheral arm and most of the roughly 80 trans-membrane helices in the membrane arm. It also revealed a new accessory subunit in the peripheral arm, which we identified as a sulfur transferase that links the respiratory chain to mitochondrial sulfur metabolism. The project took off when Kristian Parey joined Volker Zickermann and me as a postdoc. With the help of Deryck Mills, and taking advantage of our best microscopes and cameras, Kristian tirelessly collected large numbers of particle images of increasingly high quality. He generated 3D maps, first at 4.3 A [&lt;&lt;REF:Parey-2018-7-?&gt;&gt;], then at 3.2 A [&lt;&lt;REF:Parey-2019-5-?&gt;&gt;], and, most recently, at 2.1 A resolution [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;]. Janet Vonck built the atomic models. Of the recent high-resolution cryoEM structures of complex I [&lt;&lt;REF:Agip-2018-25-548&gt;&gt;,&lt;&lt;REF:Kampjut-2020-370-?&gt;&gt;], this latest map is in some ways the most astonishing in that it resolves ~1600 water molecules, of which roughly 100 were found in the complex interior. The inner water molecules (Fig. 10 &lt;&lt;FIG:F10&gt;&gt;) define a horizontal proton path that runs from the quinone reduction site at the junction between the peripheral and membrane arms, to the outlet of the proton pump at the far end of the membrane arm. Notwithstanding all this great progress, we still do not understand the mechanistic details of how the electron transfer reactions at the quinone reduction site are energetically coupled to proton transport by the three proton translocation modules in the membrane arm.\nFig. 10.\nStructure of mitochondrial complex I from Y. lipolytica at 2.1 A resolution. (a) Map coloured by subunit. Small red densities are water molecules. Map section without (b) and with (c) fitted atomic model. Small red spheres in (c) indicate a subset of the ~100 internal water molecules along the proton transfer path. From [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;].\nFollowing on from his work on the Polytomella ATP synthase dimer (see below), Niklas Klusch took it upon himself to isolate and determine the cryoEM structure of complex I from Polytomella mitochondria. We joined forces with Hans-Peter Braun at the University of Hannover (Germany), who had spent years in purifying mitochondrial complex I of the model plant Arabidopsis thaliana and in characterizing its many subunits by mass spectrometry. In a tour-de-force of data collection and image processing, Niklas was able to obtain the cryoEM structures of both the Polytomella and the Arabidopsis complex at ~2.9 A resolution in minimum time, the latter in two different conformations [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;]. Mitochondrial complex I from plants and green algae (including Polytomella) distinguishes itself from that of fungi and mammals by a large external domain of three carbonic anhydrase subunits (Fig. 11a,b &lt;&lt;FIG:F11&gt;&gt;). A metal ion (most likely zinc) resolved in the active site of one of the Arabidopsis subunits was absent in Polytomella. From this observation, we concluded that the carbonic anhydrase in the plant complex is active, whereas that of Polytomella is not.\nFig. 11.\nCryoEM structures of mitochondrial complex I from (a) Arabidopsis thaliana and (b) Polytomella sp., both at ~2.9 A resolution, coloured by subunit. The 14 core subunits are drawn in shades of blue; accessory subunits in shades of green. The three subunits of the carbonic anhydrase domain are yellow, light orange and orange; subunits of the bridge domain are pink, purple and red. Insets: gamma-carbonic anhydrase (gammaCA) and bridge domain. Unknown subunits in the membrane arm (unk) are grey. From [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;].\nAnother striking difference between complex I from plants and algae to that of fungi and mammals is a bridge of three accessory subunits, joining the peripheral and membrane arms. By structural homology, Niklas discovered that one of the bridge subunits was a ferredoxin (Fig. 11a,b &lt;&lt;FIG:F11&gt;&gt;). Again, a metal ion (in this case most likely Fe) in the Arabidopsis ferredoxin was missing in Polytomella, suggesting that the ferredoxin is active in Arabidopsis but not in Polytomella. We proposed that the ferredoxin plays a key role in redox-coupling the activity of the mitochondrial respiratory chain to photosynthesis [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;].\n\nChloroplast and mitochondrial ATP synthases\nOne of our main aims with the new direct electron detectors had been to obtain a high-resolution structure of a mitochondrial or chloroplast ATP synthase. The F1Fo ATPases consist of two rotors, both of which can stall in distinct but structurally similar states that are simultaneously present, e.g. in a crystallization drop. Partly for this reason and because of the long-term instability of the complex in detergent, decades of intense effort had failed to produce a high-resolution X-ray structure of any intact, functionally competent ATP synthase. Only structures of some key components, notably an F1 subcomplex [&lt;&lt;REF:Abrahams-1994-370-621&gt;&gt;] and the c-ring rotor [&lt;&lt;REF:Meier-2005-308-659&gt;&gt;], were available from X-ray crystallography--the latter preceded by a cryoEM structure from 2D crystals at intermediate resolution [&lt;&lt;REF:Vonck-2002-321-307&gt;&gt;]. Yet for an understanding of the fundamental mechanisms of ATP production in all forms of life, detailed structures of the whole, functional assembly were essential, ideally in several different states. CryoEM, which examines ATP synthase structures particle-by-particle within hours or minutes of isolation rather than in crystals of detergent-protein complexes that take weeks to grow, promised to overcome the problem.\nNiklas Klusch, at that point an MSc student, isolated the dimers from Polytomella, and Matteo Allegretti, who had worked on the Frh structure during the first part of his PhD, proceeded to collect data with the Falcon-II camera on our Polara. Together with Karen Davies, an experienced postdoc who up to that point had done mostly cryoET, they obtained a map at 6.2 A resolution, which revealed something entirely unexpected: a set of alpha-helices in the membrane-embedded Fo subcomplex that were &gt;80 A long and included a shallow angle of ~20degrees with the membrane plane [&lt;&lt;REF:Allegretti-2015-521-237&gt;&gt;]. The paper was rejected twice, because one reviewer was sure that our structure had to be wrong. Everybody knew, the reviewer maintained, that alpha-helices in membrane proteins were ~40 A long and ran perpendicular to the membrane, which had been common knowledge since the first bacteriorhodopsin structure [&lt;&lt;REF:Henderson-1975-257-28&gt;&gt;]. Eventually, we managed to convince the editor that our structure was not wrong. We found the long Fo helices again in the 7 A map of the yeast ATP synthase dimer. This time we had collected the data in our JEOL 3200 that was now equipped with a K2 Summit camera [&lt;&lt;REF:Hahn-2016-63-445&gt;&gt;]. Subsequently, the long, membrane-intrinsic helices were seen in a number of similar structures and are now known to be a hallmark of all rotary ATPases [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;,&lt;&lt;REF:Kühlbrandt-2016-41-106&gt;&gt;].\nOf course, the different but similar ATP synthase conformations are also simultaneously present on a cryoEM grid. Our first structures of mitochondrial ATP synthase dimers from Polytomella [&lt;&lt;REF:Allegretti-2015-521-237&gt;&gt;] and yeast [&lt;&lt;REF:Hahn-2016-63-445&gt;&gt;] appeared to be averages of several such states, and this explains why the resolution was limited to 6 or 7 A. Once we had learned to apply yet more powerful image processing programs and had collected sufficient numbers of images to ensure that the various conformations in the particle set were well enough populated, it became possible to tease them apart by image processing. The first to succeed in this endeavour in our laboratory was Alexander Hahn, as a postdoc in the independent group of Thomas Meier (now Professor of Structural Biology at Imperial College London). Alex and Thomas had isolated the chloroplast ATP synthase from kilograms of spinach bought at a local supermarket--incidentally by ammonium sulphate precipitation, an old, brute-force method which I would never have thought would yield structurally and functionally intact preparations of such an intricate and fragile assembly. Alex collected a large number of movies with the faster and better Falcon-III camera installed on our new FEI Titan Krios electron microscope. He generated a very beautiful map that extended to ~2.9 A resolution in the catalytic F1 assembly of the most populated of three rotary states [&lt;&lt;REF:Hahn-2018-360-?&gt;&gt;] (Fig. 12 &lt;&lt;FIG:F12&gt;&gt;). In the catalytic sites of the F1 head, bound nucleotides, Mg ions and even water molecules were resolved. A comparison of the three rotary states indicated that the long and thin peripheral stalk, which connects the F1 head to the Fo rotor, is flexible, whereas the gamma-subunit of the central stalk works as a rigid, non-flexible driveshaft. Without the benefit of high-resolution cryoEM, earlier biophysical work and computer simulations had come to the opposite conclusion, i.e. that the central stalk was flexible and the peripheral stalk was rigid [&lt;&lt;REF:Wächter-2011-108-3924&gt;&gt;].\nFig. 12.\nCryo-EM structure of the spinach chloroplast ATP synthase. Surface representation of the map density with subunits alpha, dark green; beta, light green; gamma, blue; delta, orange; epsilon, purple; a, light blue; b, violet; b', pink; 14-membered c-ring rotor c14, pale yellow; ATP, red. The Fo subcomplex (abb'c14) is embedded in the thylakoid membrane (grey), while the catalytic F1 subcomplex (alpha3beta3gammaepsilondelta) extends into the stroma. N-terminal helices of the alphaA, alphaC and alphaE subunits are indicated. The resolution of the F1 subcomplex is 2.9 A. From [&lt;&lt;REF:Hahn-2018-360-?&gt;&gt;].\nEncouraged by Alex Hahn's and Thomas Meier's chloroplast ATP synthase structure, Bonnie Murphy, a new postdoc, together with Niklas Klusch set out to tackle the Polytomella dimer. They progressed rapidly to a 4.1 A map that refined locally to 3.7 A in the most rigid parts of the peripheral stalk and Fo subcomplex. In this region, all sidechains were resolved [&lt;&lt;REF:Klusch-2017-6-?&gt;&gt;]. The map revealed two aqueous channels, one each extending from the middle of the membrane to the lumenal or matrix membrane surface. Each channel was lined by charged and polar residues of the a-subunit stator. In the hydrophobic core of the Fo subcomplex, the two half-channels were separated by a conserved arginine sidechain and approached one another to within ~6 A. The steep potential gradient at the channel end points would exert a strong force on the charge of the critical glutamates in the c-ring rotor, generating torque and resulting in net directional rotation [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;].\nIn the 3.7 A map, the static parts of the stator assembly were well-defined, but rotary states were not resolved. Data for the 3.7 A map had been collected with a K2 Summit detector and the in-column energy filter of the JEOL 3200 FSC. Collecting a much larger number of particle images (~730 000) with a more modern, fully automated microscope (Titan Krios G2) and a considerably better detector (Falcon-III), Niklas and Bonnie produced a map of the Polytomella ATP synthase dimer at an overall resolution of 2.7-2.8 A. In an image processing marathon, Bonnie was able to separate 13 well-defined rotary substates that together provided a detailed picture of the molecular motions that accompany c-ring rotation and ATP synthesis [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;] (Fig. 13 &lt;&lt;FIG:F13&gt;&gt;). We found that the F1 head rotates along with the central stalk and c-ring rotor for the first ~30degrees of each 120degrees primary rotary step to facilitate flexible coupling of the stoichiometrically mismatched F1 and Fo subcomplexes. Flexibility is mediated by the interdomain hinge of the OSCP subunit (known as delta in the chloroplast ATP synthase) on top of the F1 head. Since this subunit is highly conserved, as are all crucial elements of rotary catalysis, we can be confident that our mechanistic conclusions apply in principle to all rotary ATP synthases. With the high-resolution structures of the ATP synthase dimer and complex I from Polytomella and the known distribution of complex I and ATP synthase dimer rows in the membrane [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;], we can now draw a model of a crista that contains all the known molecular details and summarizes how they are arranged and work together in the mitochondrial inner membrane (Fig. 14 &lt;&lt;FIG:F14&gt;&gt;).\nFig. 13.\n(a) Cryo-EM map density of the 62-subunit 1.58 MDa Polytomella ATP synthase dimer. The monomer map on the left is shown in grey. The map on the right is coloured by subunit. The F1 head consists of three catalytic beta subunits (bright green) and three structural alpha subunits (dark green). The dashed white line indicates the section plane in (b). The long C-terminal extensions of beta wrap around the alpha subunits on the outside of the F1 head. The central stalk subunits gamma (dark blue), delta (cyan) and epsilon (light blue) connect F1 to the Fo motor in the membrane (grey). The c10 rotor ring (yellow) and subunit a (bright blue) are the main components of the Fo motor complex. The two-domain OSCP subunit (orange) links the three alpha subunits to the peripheral stalk subunits ASA 1-10. The local map resolution is 2.7 A for the peripheral stalk, Fo, and c-ring, and 3.0-3.2 A for the F1 head and central stalk. (b) Rotary substates. Sections through cryo-EM maps of the Polytomella ATP synthase F1 head at the level of nucleotide-binding sites (dashed white line in a). Top row: Six substates of primary state 1. Middle row: Four substates of primary state 2. Bottom row: Three substates of primary state 3. Bright green, beta-subunits; dark green, alpha-subunits; blue, gamma-subunit; red, ADP in catalytic sites; orange, structural ATP; grey, peripheral stalk. Red lines indicate the rotary positions of consecutive substates. From [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;].\nFig. 14.\nMolecular organization of mitochondrial inner membrane cristae. Cryo-EM structures of the F1Fo ATP synthase dimers [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;] and complex I [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;], both from Polytomella sp. in the membrane. Three-dimensional density maps at 2.7 A (ATP synthase) and 2.9 A resolution (complex I) are coloured by subunit. ATP synthase dimers assemble into long rows that impose a tight local curvature on the inner membrane, giving rise to the formation of cristae as mitochondrial microcompartments. Complex I and other respiratory chain complexes (not shown) occupy the flat membrane regions on either side of the dimer rows. Complex I generates most of the proton gradient across the inner membrane, pumping protons (dashed red arrows) into the cristae, which work as proton traps. Protons (red dots) flow through channels in the Fo subcomplex of the ATP synthase from the cristae lumen (~pH 7.2) to the mitochondrial matrix (pH 8), driving the rotor (yellow) in the membrane. The torque generated by the electrochemical gradient is transmitted via the central stalk (blue) to the catalytic F1 subcomplex (green), where ATP is produced from ADP and phosphate by rotary catalysis.\n\nOutlook\nIn the past 40 years, cryoEM of membrane proteins has come further than most of us would have predicted, and it has done so more rapidly than any of us could have hoped. Yet we are by no means at the end, but right in the middle, or perhaps even still at the beginning of this exhilarating and fascinating journey.\nFor single-particle cryoEM, a few technical challenges are left. Beam-induced movement, for many years the bane of cryoEM, may have been overcome once and for all by the new HexAuFoil grids developed by Chris Russo and colleagues [&lt;&lt;REF:Naydenova-2020-370-223&gt;&gt;], although they have not yet been widely tested. Alternative approaches, such as controlled devitrification of vitrified samples [&lt;&lt;REF:Wieferig-2021-8-186&gt;&gt;], may help in special circumstances. Protein adsorption and denaturation at the air-water interface is a fundamental and ubiquitous problem in single-particle cryoEM, which means that only a small percentage (typically ~5-20%) of a particle set contribute to a high-resolution structure. The remaining 80-95% are in some way damaged or distorted at the air-water interface. It may be possible to overcome these problems with hydrophilized graphene support films, which the complexes adhere to in random orientations in preference to the air-water interface [&lt;&lt;REF:D’Imprima-2019-8-?&gt;&gt;,&lt;&lt;REF:D’Imprima-2021-54-?&gt;&gt;].\nUltimately, the success of single-particle cryoEM depends on the quality of the protein sample that goes on the grid. There is no substitute for optimizing the conditions a protein is exposed to during isolation. Therefore, when all other technical and computational issues are settled, protein biochemistry is likely to remain as the hardest part of single-particle cryoEM, because each protein is different, details matter, and there are no shortcuts.\nWith the most recent detectors and a matching battery of image processing software, structures of virtually any macromolecular assembly above a certain size can now be tackled by single-particle cryoEM. The current lower limit with soluble proteins is ~60 kDa [&lt;&lt;REF:Herzik-2019-10-?&gt;&gt;]. The recent 3.3 A structure of the 55-kDa proton-coupled folate transporter in complex with a nanobody [&lt;&lt;REF:Parker-2021-595-130&gt;&gt;] suggests that the size limit for membrane proteins is similar, even though single-particle cryoEM of small membrane proteins is inherently more difficult. The detergent micelles or lipid nanodiscs that keep them in solution do not have a defined structure and reduce the contrast in the trans-membrane region, which makes the particles harder to align. The theoretical size limit has been estimated by Richard Henderson for perfect images recorded with 300-kV electrons (see Table 2 in [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;]) to be ~38 kDa. This limit is now almost within reach and likely to be approached asymptotically. With 100 -kV microscopes [&lt;&lt;REF:Naydenova-2019-6-1086&gt;&gt;] and detectors, slightly smaller molecules should become accessible, because the scattering cross section is higher and the ratio of elastically and inelastically scattered electrons more favourable at lower acceleration voltage (see Fig. 2 &lt;&lt;FIG:F2&gt;&gt; in [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;]).\nVery accurate high-resolution structures can be obtained by electron diffraction of microscopic 3D crystals, a technique known as MicroED. MicroED has been applied mostly to small or very small proteins or protein fragments, in particular to a number of fibre-forming amyloid proteins that are recalcitrant to other techniques [&lt;&lt;REF:Rodriguez-2017-46-79&gt;&gt;]. As such, it is a welcome addition to the cryoEM methods spectrum. It is however unlikely to become as popular as single-particle cryoEM or cryoET, because it still requires 3D protein crystals, which removes one of the key advantages of cryoEM over X-ray crystallography.\nAt present, structural biology is undergoing a profound transformation. After years of futile efforts, computational methods for solving the protein folding problem have become so powerful that the structure of virtually any single protein can now be instantly predicted with near-atomic accuracy from its genomic sequence. A new deep-learning algorithm called AlphaFold2 [&lt;&lt;REF:Jumper-2021-596-583&gt;&gt;,&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;] has been shown to produce correct protein structures even if there are no related templates in the protein data bank (pdb). We tried it out with a new membrane protein without known homologues in the pdb, and the result was shockingly similar to our unreleased experimental structure. In the short to medium term, AlphaFold2 will thus shift the emphasis and take some of the pain and hard work (but also fun!) out of structure determination by cryoEM and X-ray crystallography. However, it will not make the experimental methods redundant.\nA single structure is rarely enough to understand how a protein works. Most if not all proteins in the cell undergo conformational changes while performing their tasks, and cryoEM in particular is good at resolving different, simultaneously present functional states. Such conformational ensembles would be difficult to predict. More often than not, the function of a protein depends on interactions with other proteins or small molecules, and some of the interaction partners may not be known. Similarly, the structures of protein subunits that assemble into large macromolecular complexes may not be predicted reliably at present. Computational methods are unlikely to outperform cryoEM in determining the structures of large multi-subunit complexes, such as complex I or ATP synthase in the short term. It will be some time before AlphaFold can generate the correct structures of the 13 different rotary states of the Polytomella ATP synthase [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;], the 8 different conformations of an ABC transporter [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;] or even the two different states of plant complex I [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;] as parts of a molecular mechanism. There is however no principal reason why it should not work, and therefore, it is likely to happen sooner or later. In any event, single-particle cryoEM and crystallography will be critical for validating structure predictions.\nFor cryoET, the best times are still ahead. As more and more high-resolution structures of proteins and complexes are done or predicted, cryoEM labs will increasingly turn from crystallography and single-particle work to cryoET, which is not subject to the same constraints of sample purity and homogeneity. At the same time, it will become increasingly urgent to investigate and understand the context of macromolecular components in the cell. CryoET in particular stands to gain from protein structure prediction, because it generates lower-resolution volumes quite readily, to which the predicted structures can then be fitted. CryoEM instrumentation can in principle be used equally well for single-particle work and cryoET (although an acceleration voltage of 300 kV is preferable for tomography), which makes the transition easier. CryoET does have its own constraints, but in terms of the molecular systems and biomedical questions it can address, it is essentially unlimited. Every cell contains a universe of molecular landscapes that is best explored by cryoET.\nConstraints include sample thickness and tilt geometry. Most samples are too thick, but they can be thinned into thin lamellae by focused ion beam (FIB) milling at low temperature. Cryo-FIB milling is a demanding technique but it is practiced successfully in a growing number of cryoEM laboratories around the world [&lt;&lt;REF:Wagner-2020-15-2041&gt;&gt;,&lt;&lt;REF:Zachs-2020-9-?&gt;&gt;]. Constraints of tilt geometry can be overcome by FIB-milling cylindrical pencils instead of planar lamellae, and several laboratories are experimenting in this direction.\nA major unresolved challenge in cryoET is the lack of a universal cloneable label equivalent to green fluorescent protein (GFP) in light microscopy that is fused with a protein so it can be identified and studied in detail in its native context. Correlative light/electron microscopy alleviates the problem by identifying regions of interest in a larger volume, such as a cell, but within that region individual assemblies still have to be recognized by their shape and position, e.g. relative to a membrane. Because phosphorus scatters electrons more strongly than carbon, nitrogen or oxygen, phospholipid bilayers show up well in tomographic volumes, and therefore, cryoET works well for protein complexes in or at a membrane. CryoET of HIV capsids in vitrified aqueous buffer has achieved 3.4 A resolution [&lt;&lt;REF:Turonova-2017-199-187&gt;&gt;]. There is no fundamental reason why cryoET, just as single-particle cryoEM, should not go to true atomic resolution, given a sufficiently large number of tomographic volumes to average. In cellular tomography, the target complexes are usually surrounded by other macromolecules of similar density, which makes the targets more difficult to detect and average than single particles in dilute aqueous buffers. The ultimate limitation in both cases is the signal-to-noise ratio of the electron micrographs.\nThe signal-to-noise ratio of electron micrographs can be enhanced by a phase plate. It is fair to say that current phase plate designs, including the Volta phase plate [&lt;&lt;REF:Buijsse-2020-218-?&gt;&gt;] or our own electrostatic Boersch phase plate [&lt;&lt;REF:Walter-2015-153-22&gt;&gt;], have not kept their promise. Even though they can improve low-resolution image contrast, potential gains at high resolution are more than offset by charging problems and a lack of reproducibility associated with introducing an external device into the electron beam [&lt;&lt;REF:Buijsse-2020-218-?&gt;&gt;]. These problems may be overcome by a new elegant phase plate design that shifts the phase of the electron beam by the ponderomotive force of a strong laser [&lt;&lt;REF:Schwartz-2019-36-1016&gt;&gt;]. The laser phase plate enhances contrast in the entire resolution range and would be particularly useful for cryoET. However, its implementation is technically demanding, the electron microscope needs an extra lens and a tremendously powerful laser is necessary for shifting the phase of a 300 kV beam by 90degrees for optimum contrast. For single-particle cryoEM, recent improvements in electron microscope hardware, in particular cold field emitters, better image filters and direct electron detectors, have increased phase contrast to an extent that makes the available phase plates less of a help than a hindrance for high resolution.\nEven though the present generation of direct electron detectors is already superb, further improvements in their design and production are likely and will benefit both single-particle cryoEM and cryoET. At present, the DQE of the best available detectors (Falcon-IV) is 1.0 at low resolution and close to 0.4 at Nyquist frequency [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;]. The Nyquist frequency defines the maximum resolution possible with the given pixel size of the detector. For a perfect detector, the DQE would be 1.0 at Nyquist, but this may be hard to achieve. Still, any improvement will boost the signal-to-noise ratio at high resolution and translate into better maps from fewer particle images or tomograms.\nSo far, the high cost of cryoEM hardware does not seem to have taken away much of its popularity. The number of new cryoEM labs around the globe is still growing, and research councils and agencies are willing to support them, because of the expected gains in mechanistic insight for basic biomedical research, which will ultimately translate into benefits for human health. But this trend cannot go on forever. Cost scales roughly with acceleration voltage. It is possible to produce less expensive 100-kV electron microscopes that perform equally well, and in some ways better for single-particle cryoEM than 300-kV instruments [&lt;&lt;REF:Naydenova-2019-6-1086&gt;&gt;]. Bringing the cost of EM hardware down substantially will be essential if cryoEM is to be as widely accessible to biomedical research as protein crystallography has been in the past. An alternative would be to create more of the national cryoEM facilities such as eBIC at the Diamond synchrotron in the UK or the National Centers in the USA, to cumulate resources and provide cryoEM as a service. However, sample preparation for cryoEM is still very much a skill and a craft. Automation is in its infancy [&lt;&lt;REF:Budell-2021-?-?&gt;&gt;,&lt;&lt;REF:Weissenberger-2021-18-463&gt;&gt;] and reproducibly thin ice layers, a key factor for high resolution, are hard to achieve. Similarly, collecting top-quality high-resolution images requires close familiarity with the sample. In the long run, it may therefore be preferable not to separate researchers from their microscopes and cryoEM samples.\nWithin the past year, two cryoEM structures of a standard soluble test specimen, recombinant mouse apoferritin, have been reported at ~1.25 A, i.e. true atomic resolution [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;,&lt;&lt;REF:Yip-2020-587-157&gt;&gt;]. This proves that cryoEM is capable of delivering structures of equal quality as atomic-resolution X-ray crystallography, which not many of us would have imagined a few years ago. For most biological questions, a resolution ~2 A is probably more than adequate, but higher resolution is always nice, and it is hard to predict what it might reveal. Further improvements in electron microscopes, detectors and image processing software are bound to result in more and more atomic-resolution structures of interesting membrane proteins. Already with the present generation of cryoEM hardware and software, water molecules are resolved in the cytochrome bd oxidase [&lt;&lt;REF:Safarian-2019-366-100&gt;&gt;], complex I [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;,&lt;&lt;REF:Kampjut-2020-370-?&gt;&gt;] and the GABAA receptor [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;]. At very high resolution, cryoEM may be able to visualize ionization states of bound ions and protonation states of sidechains and water molecules, which would provide more detailed insights into molecular mechanisms. The future for studying membrane proteins by cryoEM gets brighter and more fascinating all the time!\n\nFunding\nMost of the work described in this review was funded by the Max Planck Society. Some of the earlier work at EMBL was funded by the European Molecular Biology Laboratory.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
[outputs]
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nOver the past 40 years, electron cryo-microscopy (cryoEM) has advanced from a rarified, avant-garde technique pursued by a small number of dedicated laboratories, into one of the most powerful and sought-after methods in molecular cell biology. CryoEM can address a wider scope of biological questions than any other method in structural biology, on samples ranging from tens or hundreds of nanometers in electron cryo-tomography (cryoET) to truly atomic resolution in single-particle work. CryoEM is having a major impact on our understanding of membrane systems, in particular the large energy-converting complexes of mitochondria and chloroplasts, but also ion channels, membrane transport proteins, protein translocases and membrane-perforating toxins. I present this breathtaking development from the perspective of someone who witnessed it at first hand from its beginnings in the early 1980s to the present day. Please note that this short paper is intended as a personal account rather than a comprehensive overview, focusing on how our own projects have progressed and benefitted from cryoEM, without wishing to detract in any way from the outstanding contributions of numerous colleagues.\n\nEarly days\nWhen I completed my PhD with Nigel Unwin at the MRC-LMB in 1981, biological electron microscopy was mostly concerned with negatively stained samples, usually in uranyl acetate. Negative stain was able to produce useful projection views, and in the right hands even three-dimensional (3D) maps of objects such as ribosomes or viruses, at resolutions up to ~20 A, limited by the grain size of the stain. These images and maps showed the overall outer shape, but no internal details of the structures were visible. Looking inside the molecules to find out how they work was not possible. The most notable and by that time already famous exception was Richard Henderson's and Nigel Unwin's 7 A map of bacteriorhodopsin, the first high-resolution EM structure of any biological macromolecule and the first structure of a membrane protein to reveal trans-membrane alpha-helices [&lt;&lt;REF:Henderson-1975-257-28&gt;&gt;]. Richard and Nigel had collected data of unstained purple membrane, protected against dehydration in the high vacuum of the microscope column in the quasi-aqueous environment of an air-dried glucose solution. Purple membranes are near-perfect two-dimensional (2D) crystals of bacteriorhodopsin that form naturally in the cell membrane of some halophilic archaea. My PhD project was to apply the same approach to 2D crystals of cytoplasmic ribosomes that formed naturally on the surface of endoplasmic reticulum membranes of hibernating lizards. The resolution we achieved was only ~60 A, due to the poor inherent order of the ribosome arrays, even though they were lightly crosslinked with glutaraldehyde, and air-drying in glucose probably did not help. Still, since glucose and protein scatter electrons roughly to the same extent, the contrast in the micrographs came mainly from the denser, more strongly scattering ribosome components, and therefore, the resulting structure gave us a first low-resolution map of the ribosomal RNA [&lt;&lt;REF:Kühlbrandt-1982-156-431&gt;&gt;,&lt;&lt;REF:Kühlbrandt-1982-7-221&gt;&gt;]. The membrane was nowhere to be seen.\nData for both the bacteriorhodopsin and the ribosome maps had been collected at room temperature, because at the time the development of electron cryo-microscopes had barely begun. The pioneering groups were at the Lawrence Berkeley National Laboratory in the USA (Glaeser), EMBL Heidelberg (Dubochet), Osaka University (Fujiyoshi) and the Fritz-Haber-Institute in Berlin (Zemlin). Jacques Dubochet at EMBL [&lt;&lt;REF:Dubochet-1982-10-55&gt;&gt;] and Robert Glaeser in Berkeley [&lt;&lt;REF:Jaffe-1984-13-373&gt;&gt;] were experimenting with frozen-hydrated specimens. At the MRC-LMB in Cambridge, Richard Henderson was perfecting computer programs for processing high-resolution images of 2D crystals. In 1990, he and his colleagues published the first cryoEM structure, again bacteriorhodopsin, at near-atomic resolution [&lt;&lt;REF:Henderson-1990-213-899&gt;&gt;]. The 3.5 A map was the culmination of a 15-year effort, combining electron diffraction data that provided very accurate amplitudes of Fourier terms, with precise structure factor phases obtained by processing of high-resolution images. The majority of images had been recorded at liquid-helium temperature with a prototype 100-kV instrument, called Suleika, at the Fritz Haber Institute of the Max Planck Society in Berlin, in the research group of Fritz Zemlin [&lt;&lt;REF:Baldwin-1988-202-585&gt;&gt;].\nRecording a single image on photographic film with the Suleika required no fewer than 70 steps of manual operation, not counting film development, quality assessment by optical diffraction and digitization on a flat-bed microdensitometer. Somewhere in the middle of this bafflingly complex procedure, Fritz would call out 'Freuen!' (rejoice!) to those around him, who looked on in amazement. Rejoicing was certainly in order if nothing had gone wrong up to that point, which did happen occasionally. The Suleika had a helium-cooled superconducting objective lens, designed and built by Isolde Dietrich at the company Siemens in Munich [&lt;&lt;REF:Dietrich-1977-2-241&gt;&gt;]. The superconducting lens was extremely stable, which had its advantages, but the objective lens current could not be changed without warming it up, and therefore, each image had to be focused by adjusting the acceleration voltage. Many of the film images at high tilt angles for the 3.5 A map came from Ken Downing in Berkeley. They were recorded with a liquid-nitrogen-cooled 100-kV JEOL 100B, converted into the world's first electron cryo-microscope by Ken Taylor and Robert Glaeser [&lt;&lt;REF:Taylor-1974-186-1036&gt;&gt;,&lt;&lt;REF:Hayward-1980-5-3&gt;&gt;]. The Berkeley 100B was far ahead of its time in other respects as well: it could be operated in spotscan mode, which largely avoided beam-induced movement, and it was equipped with a field emission electron gun, which became standard in cryoEM only 15-20 years later.\nFor my postdoc, I was hoping to combine my two passions in science--membrane proteins and high-resolution cryoEM. I was therefore looking for a membrane protein amenable to structure determination by the methods that had been devised by Richard Henderson and Nigel Unwin for purple membrane. Fortunately, I found such a protein in the light-harvesting chlorophyll-protein complex from the photosynthetic membranes of chloroplasts, LHC-II for short. I was introduced to it by Ernst Wehrli at the ETH Zurich in Switzerland. Ernst had attended the first-ever EMBO course on EM and image processing (now known as the EMBO cryo-course, but at the time there was no cryo). The course was held at the MRC-LMB in 1978 by Linda Amos, Richard Henderson, Nigel Unwin, Aaron Klug, Tony Crowther and others, with first-year PhD students like me as instructors and luminaries such as Michael Rossmann, the world-famous protein crystallographer, as participants.\nLHC-II has a natural, fortuitous (but unphysiological) tendency to form crystalline arrays when the isolated, detergent-solubilized complex is precipitated from solution with monovalent or divalent cations. That much was known from freeze-fracture electron microscopy when I started my postdoc at the ETH in Ernst Wehrli's group, but it took more than two years to persuade the complex to form large, well-ordered, purple-membrane-like 2D crystals that would lend themselves to electron diffraction and high-resolution image processing [&lt;&lt;REF:Kühlbrandt-1983-96-1414&gt;&gt;]. It took another year to work out an initial structure of negatively stained LHC-II membrane crystals by electron crystallography [&lt;&lt;REF:Kühlbrandt-1984-307-478&gt;&gt;]. The map resolution was 16 A. None of this was cryoEM, but the stage was set.\n\nLHC-II at liquid-helium temperature\nSince there were no instruments for high-resolution cryoEM anywhere nearby, I took the 2D crystals to Berkeley, where Ken Downing and I were able to record images for a 3.7 A projection map of LHC-II with their JEOL 100B [&lt;&lt;REF:Kühlbrandt-1989-207-823&gt;&gt;]. The visit proved that the crystals had potential, but large-scale data collection with this ageing machine (bits of the mu metal magnetic shielding sometimes dropped off the ceiling during a recording session) was out of the question. The temperamental home-built cryo-stage did not go much below -120degreesC. Occasionally, the column vacuum would collapse for no apparent reason, and on the fluorescent screen, I could watch my precious sample freeze over with a thick layer of contaminating ice. Fixing the microscope and getting it ready for another cryo-transfer took Ken and his team most of the day. Because I was not able to help and there was nothing else to do, I had ample time to read and explore the beautiful UC Berkeley campus. I also took the 2D crystals to Berlin, but Suleika's long-winded and idiosyncratic image recording routine would not have made large-scale data collection easy either. In any case, the Berlin group seemed none too keen to accommodate a young whipper-snapper like me, who was not even a professor. A perfect solution offered itself when I visited Yoshinori Fujiyoshi, then at the Protein Engineering Research Laboratory (PERI) in Osaka. Yoshi had designed and built a top-entry, helium-cooled cold stage for his 400-kV JEOL electron microscope [&lt;&lt;REF:Fujiyoshi-1991-38-241&gt;&gt;]. The stage was particularly suitable for high-resolution cryoEM of 2D crystals, and Yoshi welcomed me with open arms. The lens was not superconducting, and therefore, it took only ~10 manual operations to record one film image, which made data collection feasible.\nI used my Heisenberg Fellowship from the DFG for repeated visits to Yoshi's laboratory. I took the developed negatives back to EMBL Heidelberg, where in the meantime I had taken up a research group leader position. Together with my first postdoc Da Neng Wang, now Professor of Structural Biology at New York University, we scanned and processed the films. Within a year or two, we had gathered a sufficient number of images for a 3.4 A structure of LHC-II [&lt;&lt;REF:Kühlbrandt-1994-367-614&gt;&gt;] (Fig. 1 &lt;&lt;FIG:F1&gt;&gt;). The complementary high-resolution electron diffraction patterns required no more than a standard electron microscope with a liquid-nitrogen-cooled Gatan cryo-transfer holder, and Da Neng and I collected them with the JEOL 2000 at EMBL [&lt;&lt;REF:Wang-1992-61-287&gt;&gt;]. The 3.4 A map of LHC-II was the second high-resolution structure determined by cryoEM (again a membrane protein) and proved that the method worked for specimens other than bacteriorhodopsin. The map afforded fascinating insights into how chlorophylls and carotenoids were held and oriented by the polypeptide scaffold in an optimal position for harvesting solar energy and photoprotection.\nFig. 1.\nStructure of plant light-harvesting complex LHC-II determined by electron crystallography of two-dimensional crystals at liquid-helium temperature. (a) Atomic model of LHC-II trimers on the 2D crystal lattice. Polypeptide (purple) with bound chlorophylls (green and cyan) and carotenoids (yellow). (b) Details of the 3.4 A map with fitted polypeptide (white), chlorophylls (green) and carotenoids (yellow). From [&lt;&lt;REF:Kühlbrandt-1994-367-614&gt;&gt;].\nThe LHC-II structure was the overture to a number of highly successful collaborations for which colleagues from other laboratories teamed up with Yoshi to collect data with his superb instrument. Tom Walz came with his 2D crystals of aquaporin he had grown with Andreas Engel in Basel (Switzerland) [&lt;&lt;REF:Walz-1997-387-624&gt;&gt;] and Inga Schmidt-Krey and Hans Hebert with theirs of microsomal glutathione transferase [&lt;&lt;REF:Hebert-1997-271-751&gt;&gt;]. An enduring partnership ensued with Nigel Unwin. Nigel recorded countless images of his tubular crystals of nicotinic acetyl-choline receptor at increasingly higher resolution with Yoshi's microscopes [&lt;&lt;REF:Miyazawa-1999-288-765&gt;&gt;,&lt;&lt;REF:Miyazawa-2003-423-949&gt;&gt;], culminating in a 4 A map of the receptor in the membrane that resolved sidechains [&lt;&lt;REF:Unwin-2005-346-967&gt;&gt;]. Yoshi was accumulating a growing number of helium-cooled 400-kV and later 300-kV instruments, each of them filling a fairly large room, which all moved with him from Nara to Kyoto and then to Nagoya. When Nigel was awarded the Jeantet Prize in recognition of his groundbreaking work on the acetyl-choline receptor, he bought an apartment in Kyoto for his frequent stays, and Yoshi kept a JEOL 4000 specially for him that the Kyoto team called Janet. When I was appointed as a director at the Max Planck Institute of Biophysics in Frankfurt in 1997, it was my greatest wish to set up my own Yoshi-style microscope, and I was fortunate in that the Max Planck Society afforded me the means to do so. Like Yoshi's latest model it was a 300-kV instrument with a liquid-helium-cooled top-entry stage, and it worked very reliably until 2012. Liquid helium was roughly twice as effective in protecting the specimen against radiation damage as liquid nitrogen [&lt;&lt;REF:Stark-1996-63-75&gt;&gt;,&lt;&lt;REF:Fujiyoshi-1998-35-25&gt;&gt;], but also several times more expensive, and supplies were dwindling. Fortunately, it was possible to compensate for the lower signal-to-noise ratio by collecting and processing a correspondingly larger number of images at liquid-nitrogen temperature. When the opportunity arose, we replaced the JEOL 3000 SFF with a similar model, the JEOL 3200 FSC, which had a side-entry stage that could be cooled with either liquid nitrogen or liquid helium. Liquid nitrogen was less demanding and less costly, and we used it for cooling this new instrument almost exclusively.\nYears later, we discovered another and--at least for me--completely unexpected disadvantage of liquid-helium cooling. It turned out that beam-induced movement of unsupported cryo-specimens in vitreous water is up to four times worse at liquid-helium than at liquid-nitrogen temperature [&lt;&lt;REF:Pfeil-Gardiner-2019-6-1099&gt;&gt;]. This seemed to matter less for 2D crystals, which were routinely deposited on a continuous carbon support (even so, at least 80% of the film negatives showed signs of severe beam-induced movement, which made them useless). However, it does matter greatly for single-particle cryoEM. In retrospect, this finding explained why we had never been able to record any good single-particle images with the helium-cooled JEOL 3000. The reasons for the excessive beam-induced movement of vitrified solutions at liquid-helium temperature are not understood and need to be investigated further, if the superior cryo-protection qualities of liquid helium are ever to be taken advantage of for single particles. The causes may be related to the poor conductivity and viscous fluidity of vitreous water at very low temperatures, where carbon or graphene films are both highly conductive and mechanically stable.\n\n2D and 3D crystals of membrane proteins\nThe liquid-helium-cooled top entry stage of the JEOL 3000 SFF was particularly appropriate for 2D crystals, but suitable specimens were hard to come by. No more than around 10 2D protein crystals are known to diffract electrons to better than 4 A: purple membrane; LHC-II; the bacterial porins OmpF and PhoE; aquaporins-0, 1 and 4; microsomal glutathione transferase and its relatives (for reviews, see [&lt;&lt;REF:Kühlbrandt-1992-25-1&gt;&gt;,&lt;&lt;REF:Abeyrathne-2012-?-277&gt;&gt;]); and zinc-induced sheets of tubulin [&lt;&lt;REF:Nogales-1998-391-199&gt;&gt;], the only non-membrane protein amongst them. All are naturally abundant and readily available from natural sources in milligram quantities and thus amenable to extensive 2D crystallization screens. In the late 1990s, it became routine to produce--at first mostly bacterial--membrane proteins in similar amounts by heterologous expression. Quite a few of these formed 2D arrays, mostly in the shape of wide, flattened tubular vesicles. In our JEOL 3000, some yielded useful projection maps at 4-7 A or 3D structures at 6-8 A resolution by electron crystallography, notably the sodium-proton antiporters NhaA [&lt;&lt;REF:Williams-1999-18-3558&gt;&gt;,&lt;&lt;REF:Williams-2000-403-112&gt;&gt;] and NhaP [&lt;&lt;REF:Paulino-2014-3-?&gt;&gt;,&lt;&lt;REF:Paulino-2014-3-?&gt;&gt;], the osmoregulatory transporter BetP [&lt;&lt;REF:Ziegler-2004-337-1137&gt;&gt;], the bacterial protein translocase SecYEG [&lt;&lt;REF:Breyton-2002-418-662&gt;&gt;] and the light-gated ion channel channelrhodopsin [&lt;&lt;REF:Müller-2011-414-86&gt;&gt;,&lt;&lt;REF:Müller-2015-427-341&gt;&gt;]. However, the 2D crystals remained small, and none of them were ever as good as LHC-II or purple membrane. At the same time, heterologously expressed membrane transporters invariably produced good 3D crystals, which meant that we [&lt;&lt;REF:Ressl-2009-458-47&gt;&gt;] or others [&lt;&lt;REF:van den Berg-2004-427-36&gt;&gt;,&lt;&lt;REF:Kato-2012-482-369&gt;&gt;] were able to solve their high-resolution structures by X-ray crystallography. With a few rare exceptions, the--mostly hydrophobic--interactions that hold 2D crystals together are weaker and less numerous than the--mostly polar--crystal contacts in a 3D protein lattice, and therefore, the 2D crystals remained small and tended to be poorly ordered. Whatever the reason, for anyone interested in understanding the structure and mechanisms of membrane proteins rather than optimizing the growth of 2D crystals, X-ray crystallography was clearly the way to go. So for the next 10 years, this was what we did, until, very recently, single-particle cryoEM caught up. Structures of even fairly small membrane proteins can now be determined without any crystals at all, at considerably higher resolution.\n\nCryo-tomography of mitochondrial membranes\nAs a method in structural biology, cryoEM has the great advantage of not being restricted to crystals or pure protein solutions. cryoET can examine objects from any biological source of any shape and--within limits--size, provided that they are thin enough to be considered as weak phase objects, so that multiple electron scattering is negligible. In principle, the same microscopes can be used equally well for the three forms of cryoEM--electron crystallography, single-particle analysis and cryoET--as long as the sample can be tilted to about +/-60degrees, which is a prerequisite for tomography and electron crystallography.\nOne of the first non-crystalline specimens we investigated by single-particle cryoEM was the mitochondrial respiratory supercomplex, also called the respirasome. The respirasome is a 1.7-MDa assembly of three mitochondrial respiratory chain complexes, complex I, III and IV, the latter two also known as cytochrome c reductase and cytochrome c oxidase. The supercomplex was first identified on blue-native gels of detergent-solubilized mitochondrial membranes by the Frankfurt membrane biochemist Hermann Schagger [&lt;&lt;REF:Schägger-2000-19-1777&gt;&gt;]. Initially, we extracted it from gels and examined it in negative stain [&lt;&lt;REF:Schäfer-2006-281-15370&gt;&gt;]. Later, we purified it for cryoEM by gradient centrifugation [&lt;&lt;REF:Althoff-2011-30-4652&gt;&gt;]. The resolution of ~20-25 A we achieved with film images was low by today's standards, but good enough to dock X-ray structures of the component complexes, which indicated where they interacted. In the meantime, with better preparation techniques but mainly thanks to better electron detectors and image processing programs (see below), several cryoEM structures of respiratory chain supercomplexes at higher resolution have been published by us [&lt;&lt;REF:Sousa-2016-5-?&gt;&gt;] and others [&lt;&lt;REF:Letts-2016-537-644&gt;&gt;,&lt;&lt;REF:Wu-2016-167-1598&gt;&gt;]. The precise function of these large assemblies and how they work in fine-tuning cellular metabolism and organismal fitness are not known and remain controversial.\nAt the time, there was a debate as to whether the respiratory chain supercomplexes actually existed in mitochondria or whether they were artefacts of detergent solubilization. It occurred to me that, if they existed, they should be visible in tomographic volumes of mitochondrial membranes. The membranes were easy enough to prepare, but we had never done cryo-tomography. Mike Strauss, who was then in the middle of his PhD and now runs the cryoEM facility at McGill University in Montreal, visited the laboratory of Grant Jensen at CalTech to learn the technique. He stayed for 2 weeks and on his return to Frankfurt established the procedure on our recently acquired FEI Polara electron microscope. Compared to 2D crystallography with the JEOL 3000 or single-particle cryoEM, cryo-tomography with the Polara turned out to be remarkably straightforward.\nWe collected tilt series of isolated inner mitochondrial membranes and generated 3D tomographic volumes, as Mike had learned at CalTech. But when we examined the volumes in the computer, the first thing that struck us was not the respirasome, but long rows of lollipop-shaped 10-nm particles that were clearly the F1 heads of the mitochondrial F1Fo ATPase [&lt;&lt;REF:Strauss-2008-27-1154&gt;&gt;]. We were intrigued that seemingly identical rows showed up in consecutive parallel slices through the tomographic volumes. At first, we suspected a problem with the program that had generated the tomograms, but when we discovered parallel rows of the same particles in perpendicular views, it dawned on us that they were rows of ATP synthase dimers. Dimeric forms of the mitochondrial ATP synthase had been known for some time from negative-stain single-particle EM [&lt;&lt;REF:Minauro-Sanmiguel-2005-102-12356&gt;&gt;,&lt;&lt;REF:Dudkina-2006-580-3427&gt;&gt;], but because they all looked different, I thought they were bound to be artefacts. Also, it was difficult to see how a V-shaped dimer of two such large assemblies could sit stably in a planar lipid bilayer.\n\nMitochondrial ATP synthase dimers\nOur tomographic volumes showed that the membranes were of course far from planar. The edges of the inner membrane cristae were tightly bent, and the dimer rows always sat in the regions with the smallest, ~17-nm curvature radius. The dimer rows actually induce high local membrane curvature and indeed cristae formation, as we now know and were able to demonstrate by reconstituting purified ATP synthase dimers into liposomes [&lt;&lt;REF:Blum-2019-116-4250&gt;&gt;]. Tomographic volumes of the reconstituted proteoliposomes resembled those of mitochondrial inner membranes closely and the membrane curvature at the position of the dimer rows was the same [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;]. The rows run along the tightly bent cristae rims or ridges, but they are absent from the remaining regions of the inner membrane that are indeed roughly planar. Long rows of ATP synthase dimers are a defining feature of the mitochondrial inner membrane, and the high local curvature of the ridges is characteristic of metazoan and fungal mitochondria. Dimer rows are ubiquitous in mitochondria of all of the roughly 15 species we investigated, even though the dimers themselves vary greatly in terms of overall structure and subunit composition [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;] (Fig. 2 &lt;&lt;FIG:F2&gt;&gt;). ATP synthase dimers of metazoans and fungi differ significantly from those of ciliates such as Paramecium [&lt;&lt;REF:Mühleip-2016-113-8442&gt;&gt;], which in turn differ from those of trypanosomes and Euglena [&lt;&lt;REF:Mühleip-2017-114-992&gt;&gt;]. Dimers of unicellular green algae including Chlamydomonas and Polytomella are strikingly different from all the above [&lt;&lt;REF:Dudkina-2010-1797-272&gt;&gt;,&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;], and green-plant dimers are again different (unpublished). Yet, mitochondria of all these groups have dimer rows and their inner membranes form cristae, even though their appearance varies from lamellar (metazoans, fungi, green plants) to tubular (ciliates) to anywhere in between (Polytomella, trypanosomes). The dimer rows are often &gt;1 microm long [&lt;&lt;REF:Strauss-2008-27-1154&gt;&gt;] and conspicuous in tomographic volumes of all unfixed, flash-frozen mitochondria. The reason why they had escaped notice for so long is that they do not survive the preparation steps (chemical fixation, dehydration, plastic embedding and heavy-metal staining) for conventional room-temperature EM. They are difficult to see by any technique other than cryoET, although they have been spotted once by rapid-freeze deep-etch electron microscopy of Paramecium mitochondria [&lt;&lt;REF:Allen-1989-108-2233&gt;&gt;].\nFig. 2.\nCryo-ET of mitochondrial ATP synthase dimers. Subtomogram averages (above) and rendered tomographic volumes of inner mitochondrial membranes (below) from (a) Saccharomyces cerevisiae, (b) Polytomella sp., (c) Paramecium tetraurelia and (d) Euglena gracilis. From [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;].\nATP synthase dimers, dimer rows and inner membrane cristae seem to play a central part in cellular and organismal physiology, as do the mitochondria themselves. The tight curvature at the cristae ridges is a prerequisite for close lamellar stacking, e.g. in cardiac muscle [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;] (Fig. 3a &lt;&lt;FIG:F3&gt;&gt;). Cristae stacking increases the inner membrane surface area, and therefore, the potential for ATP production, by a factor of ~10 [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;], but this is most likely not the main reason why the dimers rows are so important. Yeast mutants lacking the dimer-specific ATP synthase subunits e and g that stabilize the dimers do not have dimer rows and do not develop cristae [&lt;&lt;REF:Davies-2012-109-13602&gt;&gt;]. The mutants are less fit than wild type, grow more slowly and, according to one study, their membrane potential is reduced [&lt;&lt;REF:Bornhövd-2006-281-13990&gt;&gt;]. Even more strikingly, Drosophila fruit flies in which subunits equivalent to e and g are knocked down are deficient in germline stem cell maturation [&lt;&lt;REF:Teixeira-2015-17-689&gt;&gt;]. Mitochondria of these knockdowns do not have cristae, which points to a profound role of the cristae in Drosophila development. At present, we cannot even guess what this role might be.\nFig. 3.\nCryo-ET of whole mitochondria from (a) mouse heart and (b) mouse liver. Upper panels: slices through tomographic volumes (scale bar, 250 nm). Lower panels: segmented 3D volumes of two typical mitochondria. Heart mitochondria have closely stacked, roughly parallel cristae (blue). Cristae of liver mitochondria are more loosely arranged and not stacked. Grey, inter-membrane space; light yellow, outer membrane. From [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;].\nAfter the dimer rows had sidetracked us for some time, we collected more tomograms of mitochondrial membranes to look for respiratory chain complexes and respirasomes. Indeed, they were there, in the flat membrane regions on either side of the ATP synthase rows. We detected the L-shaped density of the 1-MDa complex I next to ATP synthase dimers in four different species and were able to confirm its identity by antibody labelling [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;]. The resolution and quality of the tomograms, recorded in our Polara with a 2k charge-coupled device (CCD) camera and Tridiem filter, was not sufficient to tell reliably whether or not they contained complex III or IV as well, which would have meant that they were respirasomes. A few years later, after we had installed our first Titan Krios electron microscope with a Quantum energy filter and Gatan K2 detector, we re-examined isolated inner membranes of bovine, fungal (Yarrowia lipolytica) and plant (Asparagus officinalis) mitochondria by cryoET. The tomographic volumes were very much clearer and enabled us to generate sub-tomogram averages of respirasomes from the three species at about 25 A resolution [&lt;&lt;REF:Davies-2018-115-3024&gt;&gt;] (Fig. 4 &lt;&lt;FIG:F4&gt;&gt;). Like the dimer rows, the supercomplexes were ubiquitous. They looked similar in mammalian, yeast and plant mitochondria, which told us that interactions between the two major components, complexes I and III, were conserved.\nFig. 4.\nSubtomogram averages of respiratory chain supercomplexes in inner membranes of (a) bovine heart, (b) fungal (Y. lipolytica) and (c) plant mitochondria. The arrangement of complex I (blue) and complex III dimer (orange) is conserved, as indicated by the blue and red guidelines. Adapted from [&lt;&lt;REF:Davies-2018-115-3024&gt;&gt;].\n\nCryo-tomography of whole mitochondria\nHaving examined membrane protein complexes in mitochondrial membrane fragments, we were curious to see the dimer rows in intact organelles. We were especially interested in mitochondria of the filamentous fungus Podospora anserina, which has an exceptionally short lifespan of 20 days, and we were wondering how the mitochondria and dimer rows would change during ageing. To our surprise, we found that the inner membrane of mitochondria from aged Podospora underwent a profound transformation within a few days [&lt;&lt;REF:Daum-2013-110-15301&gt;&gt;]. The cristae disappeared, the dimer rows became increasingly disrupted, the dimers broke up into monomers and finally the mitochondrial matrix vesiculated. The double rows along the cristae ridges, or what was left of them in ageing cultures, were easily visible in small organelles [&lt;&lt;REF:Daum-2013-110-15301&gt;&gt;] (Fig. 5 &lt;&lt;FIG:F5&gt;&gt;), but in larger mitochondria, molecular detail was obscured by the dense mitochondrial matrix.\nFig. 5.\nThree-dimensional tomographic volume of a small mitochondrion from P. anserina. The inner membrane is light blue, the outer membrane is grey. Yellow spheres indicate the catalytic F1 heads of ATP synthase dimers arranged in rows on the cristae ridges. Scale bar, 100 nm (B. Daum and W. Kuhlbrandt, MPI of Biophysics, unpublished).\nIn collaboration with the MPI for the Biology of Ageing, we examined age-related changes in mitochondria of mouse and Drosopohila melanogaster, two popular, longer-lived model organisms. The differences were less drastic than in Podospora, but nonetheless clear [&lt;&lt;REF:Brandt-2017-6-?&gt;&gt;]. In mouse, changes in inner membrane morphology were strongly organ-dependent. Heart mitochondria were particularly resilient to ageing, such that mitochondria from young and old mice were indistinguishable and equally tightly packed with lamellar cristae (Fig. 3a &lt;&lt;FIG:F3&gt;&gt;). We found that liver mitochondria have fewer cristae, which are not stacked (Fig. 3b &lt;&lt;FIG:F3&gt;&gt;). In old mice, around 20% of liver mitochondria developed a central vacuole that was contiguous with the cristae. In kidney, a similar proportion of mitochondria lost their inner membrane morphology. These changes presumably go along with reduced respiratory activity, ATP production and organellar fitness with increasing age. The picture was similar for Drosophila.\nAs a postdoc in the group, Vicki Gold used cryoET to investigate protein import into mitochondria. She generated a pre-protein construct with a mitochondrial import signal sequence and conjugated it with fluorescent quantum dots by biotin/streptavidin biochemistry. When import-competent mitochondria were incubated with this construct, the quantum-dot-labelled pre-protein got stuck in the translocase of the outer mitochondrial membrane, known as the Translocase of the Outer Membrane (TOM) complex. Her approach ensured that only complexes active in protein translocation were labelled, and that she could keep track of them by fluorescence during cryoEM grid preparation. In the tomographic volumes, the dense quantum dots showed up beautifully on the mitochondrial outer membrane, highlighting the location of active import complexes [&lt;&lt;REF:Gold-2014-5-?&gt;&gt;]. By subtomogram averaging, she also managed to visualize the import receptor protein subunits protruding on the cytoplasmic side at low resolution. Apparently, the receptor subunits are flexible, which explains why they have so far eluded us in our single-particle cryoEM studies of the isolated TOM complex (see below).\n\nCryo-tomography of reconstituted membranes\nCryoET also works well for membrane assemblies reconstituted in vitro. Oli Daumke and Katja Faelber at the Max Delbruck Centre in Berlin were determining the X-ray structure of the mitochondrial membrane-remodelling GTPase Mgm1 and wanted to know how it interacts with the lipid bilayer. In collaboration with them, Lea Dietrich was able to assemble a lattice of helical Mgm1 filaments on pre-formed liposomes. The helical arrays shaped the liposomes into thin long tubes, and by image processing and sub-tomogram averaging, she generated a volume that indicated how the GTPase in the X-ray structure interacts with the membrane [&lt;&lt;REF:Faelber-2019-571-429&gt;&gt;]. To our surprise, Lea discovered similar but different helical arrays inside the membrane tubes after some of them had ruptured, so that the protein could diffuse to the inside. The inside Mgm1 lattice generated wider tubes, presumably resembling the situation at tubular cristae junctions, which are most likely stabilized by Mgm1. The topography of the wider tubes would resemble the inner surface of the mitochondrial inner membrane, where Mgm1 is thought to promote membrane fission or fusion. The structures of the inside and outside helical lattices suggested mechanisms by which GTP hydrolysis generates a power stroke that would constrict or dilate the membrane tube [&lt;&lt;REF:Faelber-2019-571-429&gt;&gt;].\n\nPore-forming proteins\nOther membrane systems that lend themselves well to cryoET are bacterial and other pore-forming proteins. Membrane-perforating bacterial toxins are produced, or can be expressed, as soluble monomers which insert into the target membranes to form large circular pores. In collaboration with Trinad Charkaborty at the University of Giessen (Germany), we expressed, crystallized and determined the X-ray structure of the membrane-perforating toxin listeriolysin (LLO) from Listeria monocytogenes [&lt;&lt;REF:Köster-2014-5-?&gt;&gt;]. Later, we turned to pneumolysin (PLY), the virulence factor of the human pathogen Streptococcus pneumoniae. Both toxins insert into cholesterol-containing membranes of animal cells, assembling into ring-like structures of 40 or more monomers [&lt;&lt;REF:van Pee-2016-16-7915&gt;&gt;]. We obtained the structure of the closed PLY prepore in the membrane by cryoET and subtomogram averaging, and that of the fully open, 42-membered ring of the isolated, detergent-solubilized pore complex by single-particle cryoEM [&lt;&lt;REF:van Pee-2017-6-?&gt;&gt;] (Fig. 6 &lt;&lt;FIG:F6&gt;&gt;). Comparison of the prepore and pore structures indicated how the four-domain monomer inserts into the lipid bilayer and then, upon conversion of one of its domains into 80 A beta hairpins, forms ~400 A beta barrels that perforate the bilayer and kill the cell. Molecular dynamics simulation in collaboration with Gerhard Hummer at the MPI of Biophysics revealed the detailed mechanism of pore formation and extrusion of the lipid plug to open the pore [&lt;&lt;REF:Vögele-2019-116-13352&gt;&gt;].\nFig. 6.\nSingle-particle cryoEM structure of the pneumolysin (PLY) toxin pore complex at 4.5 A resolution. The four fitted domains in the 2.4 A X-ray structure of the PLY monomer are shown in colour. One domain has rearranged into ~80 A beta hairpins (cyan) that form a 42-membered ring with an inner diameter of ~400 A perforating the membrane. From [&lt;&lt;REF:van Pee-2017-6-?&gt;&gt;].\nA very different way in which a protein can perforate a membrane, this time from the inside to allow a virus to escape from an infected cell, was brought to our attention by David Prangishvili at the Institut Pasteur in Paris, France. David was studying phages that infect certain thermophilic archaea, inducing them to produce large numbers of a small, 10-kDa membrane protein encoded in the viral genome. The gene product inserts spontaneously into the archaeal membrane, where it self-assembles into ~1000 A pyramids of 7-fold symmetry that pierce through the S-layer. The S-layer forms a tough protective lattice around archaea living in extreme environments and would otherwise trap the virus in the cell. Once the S-layer is punctured, the sides of the pyramids open like the petals of a flower to create ~200 nm holes, through which the new virions emerge [&lt;&lt;REF:Daum-2014-111-3829&gt;&gt;].\n\nChloroplast tomography\nEver since our LHC-II work in the 1990s, we had kept a close interest in chloroplasts, the characteristic membrane organelles of green plants and algae, and their photosynthetic membrane proteins, including the photosystem II reaction centre [&lt;&lt;REF:Rhee-1997-389-522&gt;&gt;,&lt;&lt;REF:Rhee-1998-396-283&gt;&gt;]. Chloroplasts are much larger than mitochondria, measuring around 5 microm across. They are too bulky and dense for direct tomographic imaging of plunge-frozen samples, the approach we had taken with whole mitochondria. At the beginning of his PhD, Bertram Daum (now a PI at the University of Exeter, UK) found out that chloroplasts ruptured by mechanical blotting on the cryoEM grid maintained the intricate molecular organization of their membrane networks at least for the few seconds it takes to flash-freeze them [&lt;&lt;REF:Daum-2010-22-1299&gt;&gt;,&lt;&lt;REF:Daum-2011-62-2393&gt;&gt;]. The protein-dense chloroplast stroma had leaked out, which improved the contrast and resolution of the tomograms. By cryoET of such samples, Bertram examined the arrangement and distribution of large membrane protein complexes in chloroplast thylakoids. We were especially interested in the chloroplast ATP synthase, to see whether it formed dimers and dimer rows as in mitochondria. The tomographic volumes of chloroplast grana stacks and unstacked stromal lamellae showed unambiguously that this was not the case. The chloroplast ATP synthase was homogenously distributed over the stroma thylakoids and grana end membranes, and all copies were monomers (Fig. 7 &lt;&lt;FIG:F7&gt;&gt;). In particular, there were no dimers or dimer rows around the tightly curved edges of the thylakoid grana, where one might have expected to find them. In retrospect, this is not surprising since the chloroplast ATP synthase lacks the subunits that are required for dimer formation [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;]. The same is true of the related bacterial ATP synthases, which are likewise all monomers.\nFig. 7.\nCryoET of chloroplast thylakoid membranes. F1 heads (yellow) of the chloroplast F1Fo ATP synthase are confined to stroma and grana end membranes. The chloroplast ATP synthase is monomeric and not found at the highly curved grana edges. Grana stacks (dark green); stroma membranes (transparent green). Scale bar, 100 nm. Courtesy of Bertram Daum, MPI of Biophysics.\nMore recently, when he joined the laboratory as a PhD student, Davide Floris adopted the same approach of specimen preparation to examine the molecular landscape of etioplasts, the precursor organelles of chloroplasts in the leaves of greening plants. The etioplast inner membrane is special because it forms the pro-lamellar body, a meshwork of tubular membranes arranged in the lattice of a lipidic cubic sponge phase. This had been known since the 1970s from conventional EM of plastic sections, but no one knew how or why the membranes were arranged in this peculiar way that appears to have no parallel in biology. By cryoET, Davide discovered that the membrane tubes were surrounded, and indeed formed, by helical arrays of large numbers of a small, membrane-associated protein. By sub-tomogram averaging, he obtained a map at 9 A resolution and was able to show that the protein is the 37-kDa light-dependent protochlorophyllide oxidoreductase (LPOR) [&lt;&lt;REF:Floris-2021-7-514&gt;&gt;]. LPOR is the most abundant enzyme in etioplasts. It converts the pale yellow protochlorophyllide into chlorophyllide, which in turn is converted into the deep-green chlorophyll by chlorophyll synthase. Both enzymatic reactions are key steps in the biogenesis of photosynthetic reaction centres and light-harvesting complexes of chloroplasts, including LHC-II that consists of 33% chlorophyll by weight.\n\nDirect electron detectors\nBefore 2012, only very few cryoEM structures reached a resolution of 3.5-4 A, where large sidechains become visible. All were of 2D crystals, mostly of membrane proteins (see above), or of large icosahedral viruses (see [&lt;&lt;REF:Grigorieff-2011-21-265&gt;&gt;]). For assemblies smaller than ~20 MDa that did not consist of large numbers of identical units related by strict crystallographic, helical or point-group symmetry, high resolution was out of the question. The reason was the poor signal-to-noise ratio of individual electron micrographs, which meant that smaller particles could not be aligned and averaged with sufficient accuracy to warrant high resolution. Until that point, photographic film was by far the best recording medium for cryoEM. Although tedious to use, it was very much better than the solid-state CCD electron detectors available at the time, which were quick and convenient, but their modulation transfer function (MTF), and hence, detective quantum efficiency (DQE) at high resolution was poor. Richard Henderson's incisive 1995 review [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;] had shown beyond any doubt that of the three forms of radiation which can in principle deliver images of individual biological macromolecules--X-rays, neutrons and electrons--only the latter held the promise of producing such images at 3 A resolution or better. Therefore, if cryoEM was ever to realize this promise and come close to the success of X-ray crystallography, electronic detectors with a signal-to-noise ratio and DQE at least as good as film would be mandatory.\nIn 2004, Richard set up a consortium of engineers and members of the cryoEM community to develop such a detector. Apart from himself and his colleagues Greg McMullan and Wasi Faruqi at the MRC-LMB, core members of the team were Renato Turchetta and Nicola Guerrini at the Rutherford Appleton Laboratories (RAL) outside Oxford, UK. Both were brilliant electronics engineers with expertise in detector chip design. Together, Renato and Nicola embarked on designing and building a solid-state device with all the desirable properties of an optimal electron detector: a large active area (at least 4000 x 4000 pixels), suitable pixel size (the consortium decided on 14 microm), a DQE as close as possible to 1 (where 1.0 would be that of a perfect detector), high sensitivity, rapid readout, low readout noise, high resistance to electron irradiation and reasonable production cost. These very demanding specifications were defined in preliminary consortium meetings. The new generation of detectors would be based on complementary metal oxide semiconductor technology that was already in widespread use in digital cameras and mobile phones, but needed to be adapted to the stringent requirements of high-resolution cryoEM. Although Richard could easily have shouldered the whole effort by himself, he was intent on putting the endeavour on a broader scientific base and invited me and Jurgen Plitzko of the MPI for Biochemistry in Martinsried (Germany) to join the consortium. The FEI company in Eindhoven (The Netherlands; acquired by Thermo Fisher in 2017) joined with a view of producing and marketing the detector in due course. The LMB, the two MPIs, FEI and RAL each contributed 20% to the considerable cost of getting this project off the ground. It was clear to us that if these new direct electron detectors could be made to work, everyone in the cryoEM world would want to have one. We now know they work far better than we could have imagined in our wildest dreams and everyone does want one. Indeed, most cryoEM groups have at least one already.\nInitially, test chips with different pixel topologies were designed and produced at RAL and evaluated by the LMB detector development team. Criteria included signal-to-noise ratio, radiation hardness and DQE. Results exceeded expectations [&lt;&lt;REF:McMullan-2009-109-1126&gt;&gt;,&lt;&lt;REF:Faruqi-2011-44-357&gt;&gt;]. The most promising pixel design was then selected for further development. Before the camera chip went into production, the consortium decided that the silicon wafer of the detector should be back-thinned to ~30-50 microm, which improved the signal-to-noise ratio and MTF substantially [&lt;&lt;REF:McMullan-2009-109-1144&gt;&gt;]. Eventually, FEI produced a back-thinned detector chip for a camera called the Falcon-II, and each consortium member obtained one prototype.\nOf course, our consortium was not the only initiative in the world to develop direct electron detectors. The US company Gatan Inc., who had long experience in producing CCD cameras and energy filters for cryoEM, came out with their own design more or less simultaneously, as did the smaller Direct Electron company in San Diego, CA. Gatan's direct electron detector camera was called the K2 Summit, and the one from Direct Electron was the DE-20. The sensor areas of all three cameras were similar (roughly 4000 by 4000 pixels), but the pixels of both the K2 Summit and the DE20 were very much smaller (5 and 6.4 microm). All three performed well (see [&lt;&lt;REF:McMullan-2014-147-156&gt;&gt;]), but in practical terms, the high readout speed of the K2 (400 frames per second) meant that up to 20 times more images could be recorded per hour. Primarily for this reason, the K2 became particularly popular with cryoEM labs. It was also the camera of choice for cryoET, not least because the Falcon-II (produced by FEI) was not on offer in combination with an energy filter (essential for tomography), and post-column energy filters were at the time only produced by FEI's competitor Gatan. On the other hand, the larger pixel of the Falcon-II meant that it had a better DQE at high spatial frequencies, which made it the best choice for high-resolution cryoEM of large single particles. Since then, both the Gatan K2 and the Falcon-II have been upgraded to the Gatan K3 and Falcon-III (and now the Falcon-IV), each with significantly improved performance, although the K3 has the same pixel as the K2. Even better, faster and more sensitive direct electron detectors are bound to be developed in the years to come.\n\nThe resolution revolution\nWe had our Falcon-II camera installed on our Polara electron microscope, and by October 2012, it performed to specification. With Greg McMullan's help, we recorded our first high-resolution images of the Frh complex a few months later. Frh is a rigid, soluble, dodecameric assembly with a mass of 1.2 MDa that was isolated and purified from methanogenic archaea by our colleagues Seigo Shima and Stella Vitt at the MPI of Terrestrial Microbiology in Marburg (Germany). We had just published a nice cryoEM map of Frh from images recorded on Kodak SO163 film [&lt;&lt;REF:Mills-2013-2-?&gt;&gt;]. The nominal resolution was 5.5 A, although beta-strands were fully resolved and the map therefore looked much better. With the Falcon-II, we were able to improve the nominal resolution to 3.4 A within a few weeks [&lt;&lt;REF:Allegretti-2014-3-?&gt;&gt;]. Since then, we have improved the map further to just better than 2 A with the Falcon-III and K3 cameras (Fig. 8 &lt;&lt;FIG:F8&gt;&gt;). The Frh map was one of the first single-particle cryoEM structures in this resolution range. The other three were the 20S proteasome from Thermoplasma thermophilum at 3.3 A [&lt;&lt;REF:Li-2013-10-584&gt;&gt;], the TRPV1 ion cannel from rats at 3.4 A [&lt;&lt;REF:Liao-2013-504-107&gt;&gt;] and the large subunit of the yeast mitochondrial ribosome at 3.2 A [&lt;&lt;REF:Amunts-2014-343-1485&gt;&gt;]. Two of them (Frh and the mitoribosome large subunit) were done with the Falcon-II, the other two with the K2 Summit. Together, these four structures marked the beginning of what has become known as the 'resolution revolution' in cryoEM [&lt;&lt;REF:Kühlbrandt-2014-343-1443&gt;&gt;], which is still gathering momentum.\nFig. 8.\n(a) CryoEM structure of Frh complex at 1.97 A resolution, with each of the 12 heterotrimers in a different colour. Small red densities on the protein surface are water molecules. (b, c) Map details with fitted polypeptide chain and metal ions (Fe, orange; Ni, green). Sulphur is yellow. The iron in (c) coordinates one carbon monoxide and two cyanide ligands (J. Vonck and D. J. Mills, unpublished).\nAlthough the resolution revolution was triggered by the direct electron detectors, it could not have progressed with the same force and speed without a set of powerful image processing programs that fortuitously became available around the same time. These programs enabled the cryoEM community to take full advantage of the superior image quality delivered by the new detectors. The high sensitivity and fast readout speed of the new cameras meant that the electron dose of ~20 e/A2 for a high-resolution micrograph could be spread over a series of movie frames. This made it possible to measure [&lt;&lt;REF:Brilot-2012-177-630&gt;&gt;] and correct for beam-induced movement [&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;] that had been a persistent and intractable problem with film images. With the new cameras, it became convenient and quick to determine and correct for the contrast transfer function at the stage of image recording, e.g. by the programs ctffind4 [&lt;&lt;REF:Rohou-2015-192-216&gt;&gt;] or Gctf [&lt;&lt;REF:Zhang-2016-193-1&gt;&gt;]. Well-known and much used image program suites include EMAN2 [&lt;&lt;REF:Tang-2007-157-38&gt;&gt;], Frealign [&lt;&lt;REF:Grigorieff-2007-157-117&gt;&gt;] and CryoSparc [&lt;&lt;REF:Punjani-2017-14-290&gt;&gt;]. The most popular and perhaps most incisive program suite is RELION, developed by Sjors Scheres at the MRC-LMB [&lt;&lt;REF:Scheres-2012-415-406&gt;&gt;,&lt;&lt;REF:Zivanov-2018-7-?&gt;&gt;], on the basis of the Xmipp programs from Jose-Maria Carazo in Madrid [&lt;&lt;REF:Scheres-2008-3-977&gt;&gt;]. RELION applies rigorous Bayesian maximum-likelihood statistics and an intuitively obvious and generally accepted 'gold standard' resolution criterion based on an objective comparison of maps generated from two random halves of a data set. As the resolution revolution progresses, top software developers in protein crystallography and structural biology are joining the fray, devising powerful programs, e.g. phenix_resolve [&lt;&lt;REF:Terwilliger-2020-17-923&gt;&gt;], to improve high-resolution cryoEM maps further.\n\nSingle-particle cryoEM of membrane proteins\nEven though most single-chain membrane proteins are at present too small for high-resolution cryoEM, they often form dimers or higher oligomers, or associate with other subunits into complexes, which brings them into a suitable size range for single-particle analysis. When Arne Moller joined the department as a group leader in 2015, he focused on two such membrane proteins. His long-standing interest in ABC transporters came to fruition in the cryoEM structure of TmrAB, a 137-kDa heterodimeric ABC transporter [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;]. A bound camelid nanobody added ~15 kDa to the mass of the heterodimer, breaking its near-2-fold symmetry and helping alignment. The membrane environment of a lipid nanodisc stabilized the protein without restricting functionally important conformational changes. Arne's team was able to resolve no fewer than eight different conformations of the transporter under turnover conditions at resolutions between 2.8 and 4.2 A (Fig. 9 &lt;&lt;FIG:F9&gt;&gt;). Together, the different conformations provided detailed snapshots of the complete transport cycle, as an impressive demonstration of the post-revolutionary power of cryoEM. The other moderately sized membrane protein studied by Arne and his team was a ~210-kDa P-type ATPase from yeast that works as a lipid flippase [&lt;&lt;REF:Timcenko-2019-571-366&gt;&gt;]. The project was a collaboration with Poul Nissen in Aarhus (Denmark), who has a long and distinguished track record in protein crystallography of P-type ATPases. Arne and colleagues were able to resolve three different states, which together illustrated the molecular mechanism by which lipids are moved from one membrane leaflet to the other.\nFig. 9.\nCryoEM structures of the ABC transporter TmrAB under turnover conditions. (a) Narrow and wide inward-facing conformations. (b) In the ATP-bound state, the nucleotide-binding domains associate closely and the exporter is either in an outward-facing open or occluded conformation. (c) Outward-facing open and occluded conformations also dominate the vanadate-trapped state. (d) Asymmetric states during TmrAB turnover resemble an outward-facing occluded conformation with slightly separated nucleotide-binding domains. TmrA is blue, TmrB is yellow and the bound nanobody is light grey. From [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;].\n\nProtein translocases\nFor years, we had been fascinated by membrane protein complexes that translocate pre-proteins into or out of cells or into cell organelles. The first of these was the protein translocase SecYEG from E. coli. As a postdoc in the group, Ian Collinson had grown 2D crystals of the complex, which Cecile Breyton analysed by electron crystallography, yielding a ~8 A 3D map [&lt;&lt;REF:Breyton-2002-418-662&gt;&gt;,&lt;&lt;REF:Bostina-2005-352-1035&gt;&gt;]. The collaboration continued when Ian, by then Professor of Biochemistry at the University of Bristol (UK), was able to bind a pre-protein mimic to SecY. This gave rise to a 3D map at similar resolution, which indicated how the substrate unlocks the complex for protein export [&lt;&lt;REF:Hizlan-2012-1-21&gt;&gt;].\nAnother of our long-standing interests was the mitochondrial TOM complex. An initial collaboration with Nikolaus Pfanner at the University of Freiburg (Germany) resulted in the cryoEM structure of a heterotrimeric form from yeast mitochondria at ~18 A resolution. Data were recorded on film, first with an FEI Technai F20, and then with the Polara [&lt;&lt;REF:Model-2008-383-1049&gt;&gt;]. We revisited the project after the arrival of the direct detectors, this time together with Stephan Nussberger, who had been my first PhD student at EMBL and was now a professor at the University of Stuttgart (Germany). Stephan had tried for 10 years to crystallize the TOM complex from the filamentous fungus Neurospora crassa, but the crystals never diffracted to better than ~8 A. He had however fine-tuned the preparation of the complex from N. crassa mitochondria, which put us in a position to make rapid progress by cryoEM. As a PhD student, Thomas Bausewein collected single-particle images of the TOM complex in detergent or reconstituted into amphipols with our JEOL 3200 FSC at liquid-nitrogen temperature, which he processed to generate a map at 6.8 A resolution [&lt;&lt;REF:Bausewein-2017-170-693&gt;&gt;]. The map showed how the two Tom40 beta-barrels in the dimeric complex form a twin pore. The alpha-helical Tom20 preprotein receptor and the small Tom7, 6 and 5 subunits were assigned on the basis of the shape and size of their map density. Subunit assignment was helped by mass spectrometry indicating the size and sequence of subunits present in the complex. Since then, two higher-resolution cryoEM structures of the S. cerevisiae TOM complex have both confirmed our subunit assignments and orientations [&lt;&lt;REF:Araiso-2019-575-395&gt;&gt;,&lt;&lt;REF:Tucker-2019-26-1158&gt;&gt;].\n\nRespiratory chain complexes\nAn interesting side product of the TOM project was the cryoEM structure of the N. crassa cytochrome c oxidase, which co-purified accidentally with the TOM complex by Ni affinity chromatography. Its identity was revealed by the shape of the 5.5 A cryoEM map [&lt;&lt;REF:Bausewein-2019-6-773&gt;&gt;]. As far as we know, this is the first time that an unknown protein contaminant has been identified solely on the basis of its cryoEM structure, but it will almost certainly not be the last.\nIn collaboration with Manuela Pereira at the ITQB in Lisbon (Portugal), Joana Sousa worked out the 3.9 A cryoEM structure of a bacterial Alternative Complex III [&lt;&lt;REF:Sousa-2018-9-?&gt;&gt;]. More recently, Alex Hahn in my group and Schara Safarian in Hartmut Michel's department at our institute obtained a superb 2.7 A map of a bacterial cytochrome bd oxidase that resolved a number of water molecules in its two proton channels [&lt;&lt;REF:Safarian-2019-366-100&gt;&gt;]. Hartmut Michel, who pioneered X-ray crystallography of membrane proteins in the 1980s, is perhaps the most prominent recent convert in the growing number of protein crystallographers who now use cryoEM almost exclusively.\n\nComplex I\nFor ~20 years, we had collaborated closely with the group of Uli Brandt and Volker Zickermann at the Frankfurt University Medical School to study the structure of complex I from the aerobic yeast Yarrowia lipolytica by electron microscopy, initially in negative stain [&lt;&lt;REF:Zickermann-2003-278-29072&gt;&gt;]. Complex I from Y. lipolytica has an important advantage over the mammalian complex in that it is readily accessible to directed mutagenesis. Brandt and Zickermann had established a genetic system to modify the complex of this particular organism, since the more established bakers' yeast S. cerevisiae lacks complex I. This long-term investment is now paying off. The group had been quite successful in growing 3D crystals and determining the X-ray structure of this ~1-MDa membrane complex, first at 6.3 A [&lt;&lt;REF:Hunte-2010-329-448&gt;&gt;] and then at 3.6-3.9 A resolution [&lt;&lt;REF:Zickermann-2015-347-44&gt;&gt;]. They were thus reluctant to jettison crystallography and EM of negatively stained samples in favour of high-resolution cryoEM. Only when K.R. Vinothkumar (who had been my PhD student in Frankfurt and now runs the first Indian cryoEM facility in Bangalore) together with Judy Hirst in Cambridge published the structure of the bovine complex at 5 A resolution [&lt;&lt;REF:Vinothkumar-2014-515-80&gt;&gt;] did they see the point of the new method, but by that time we had almost missed the boat.\nOur first cryoEM structure of the Yarrowia complex [&lt;&lt;REF:D’Imprima-2016-1857-1935&gt;&gt;] was not at very high resolution (7.9 A), but did resolve the 8 FeS clusters in the peripheral arm and most of the roughly 80 trans-membrane helices in the membrane arm. It also revealed a new accessory subunit in the peripheral arm, which we identified as a sulfur transferase that links the respiratory chain to mitochondrial sulfur metabolism. The project took off when Kristian Parey joined Volker Zickermann and me as a postdoc. With the help of Deryck Mills, and taking advantage of our best microscopes and cameras, Kristian tirelessly collected large numbers of particle images of increasingly high quality. He generated 3D maps, first at 4.3 A [&lt;&lt;REF:Parey-2018-7-?&gt;&gt;], then at 3.2 A [&lt;&lt;REF:Parey-2019-5-?&gt;&gt;], and, most recently, at 2.1 A resolution [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;]. Janet Vonck built the atomic models. Of the recent high-resolution cryoEM structures of complex I [&lt;&lt;REF:Agip-2018-25-548&gt;&gt;,&lt;&lt;REF:Kampjut-2020-370-?&gt;&gt;], this latest map is in some ways the most astonishing in that it resolves ~1600 water molecules, of which roughly 100 were found in the complex interior. The inner water molecules (Fig. 10 &lt;&lt;FIG:F10&gt;&gt;) define a horizontal proton path that runs from the quinone reduction site at the junction between the peripheral and membrane arms, to the outlet of the proton pump at the far end of the membrane arm. Notwithstanding all this great progress, we still do not understand the mechanistic details of how the electron transfer reactions at the quinone reduction site are energetically coupled to proton transport by the three proton translocation modules in the membrane arm.\nFig. 10.\nStructure of mitochondrial complex I from Y. lipolytica at 2.1 A resolution. (a) Map coloured by subunit. Small red densities are water molecules. Map section without (b) and with (c) fitted atomic model. Small red spheres in (c) indicate a subset of the ~100 internal water molecules along the proton transfer path. From [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;].\nFollowing on from his work on the Polytomella ATP synthase dimer (see below), Niklas Klusch took it upon himself to isolate and determine the cryoEM structure of complex I from Polytomella mitochondria. We joined forces with Hans-Peter Braun at the University of Hannover (Germany), who had spent years in purifying mitochondrial complex I of the model plant Arabidopsis thaliana and in characterizing its many subunits by mass spectrometry. In a tour-de-force of data collection and image processing, Niklas was able to obtain the cryoEM structures of both the Polytomella and the Arabidopsis complex at ~2.9 A resolution in minimum time, the latter in two different conformations [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;]. Mitochondrial complex I from plants and green algae (including Polytomella) distinguishes itself from that of fungi and mammals by a large external domain of three carbonic anhydrase subunits (Fig. 11a,b &lt;&lt;FIG:F11&gt;&gt;). A metal ion (most likely zinc) resolved in the active site of one of the Arabidopsis subunits was absent in Polytomella. From this observation, we concluded that the carbonic anhydrase in the plant complex is active, whereas that of Polytomella is not.\nFig. 11.\nCryoEM structures of mitochondrial complex I from (a) Arabidopsis thaliana and (b) Polytomella sp., both at ~2.9 A resolution, coloured by subunit. The 14 core subunits are drawn in shades of blue; accessory subunits in shades of green. The three subunits of the carbonic anhydrase domain are yellow, light orange and orange; subunits of the bridge domain are pink, purple and red. Insets: gamma-carbonic anhydrase (gammaCA) and bridge domain. Unknown subunits in the membrane arm (unk) are grey. From [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;].\nAnother striking difference between complex I from plants and algae to that of fungi and mammals is a bridge of three accessory subunits, joining the peripheral and membrane arms. By structural homology, Niklas discovered that one of the bridge subunits was a ferredoxin (Fig. 11a,b &lt;&lt;FIG:F11&gt;&gt;). Again, a metal ion (in this case most likely Fe) in the Arabidopsis ferredoxin was missing in Polytomella, suggesting that the ferredoxin is active in Arabidopsis but not in Polytomella. We proposed that the ferredoxin plays a key role in redox-coupling the activity of the mitochondrial respiratory chain to photosynthesis [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;].\n\nChloroplast and mitochondrial ATP synthases\nOne of our main aims with the new direct electron detectors had been to obtain a high-resolution structure of a mitochondrial or chloroplast ATP synthase. The F1Fo ATPases consist of two rotors, both of which can stall in distinct but structurally similar states that are simultaneously present, e.g. in a crystallization drop. Partly for this reason and because of the long-term instability of the complex in detergent, decades of intense effort had failed to produce a high-resolution X-ray structure of any intact, functionally competent ATP synthase. Only structures of some key components, notably an F1 subcomplex [&lt;&lt;REF:Abrahams-1994-370-621&gt;&gt;] and the c-ring rotor [&lt;&lt;REF:Meier-2005-308-659&gt;&gt;], were available from X-ray crystallography--the latter preceded by a cryoEM structure from 2D crystals at intermediate resolution [&lt;&lt;REF:Vonck-2002-321-307&gt;&gt;]. Yet for an understanding of the fundamental mechanisms of ATP production in all forms of life, detailed structures of the whole, functional assembly were essential, ideally in several different states. CryoEM, which examines ATP synthase structures particle-by-particle within hours or minutes of isolation rather than in crystals of detergent-protein complexes that take weeks to grow, promised to overcome the problem.\nNiklas Klusch, at that point an MSc student, isolated the dimers from Polytomella, and Matteo Allegretti, who had worked on the Frh structure during the first part of his PhD, proceeded to collect data with the Falcon-II camera on our Polara. Together with Karen Davies, an experienced postdoc who up to that point had done mostly cryoET, they obtained a map at 6.2 A resolution, which revealed something entirely unexpected: a set of alpha-helices in the membrane-embedded Fo subcomplex that were &gt;80 A long and included a shallow angle of ~20degrees with the membrane plane [&lt;&lt;REF:Allegretti-2015-521-237&gt;&gt;]. The paper was rejected twice, because one reviewer was sure that our structure had to be wrong. Everybody knew, the reviewer maintained, that alpha-helices in membrane proteins were ~40 A long and ran perpendicular to the membrane, which had been common knowledge since the first bacteriorhodopsin structure [&lt;&lt;REF:Henderson-1975-257-28&gt;&gt;]. Eventually, we managed to convince the editor that our structure was not wrong. We found the long Fo helices again in the 7 A map of the yeast ATP synthase dimer. This time we had collected the data in our JEOL 3200 that was now equipped with a K2 Summit camera [&lt;&lt;REF:Hahn-2016-63-445&gt;&gt;]. Subsequently, the long, membrane-intrinsic helices were seen in a number of similar structures and are now known to be a hallmark of all rotary ATPases [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;,&lt;&lt;REF:Kühlbrandt-2016-41-106&gt;&gt;].\nOf course, the different but similar ATP synthase conformations are also simultaneously present on a cryoEM grid. Our first structures of mitochondrial ATP synthase dimers from Polytomella [&lt;&lt;REF:Allegretti-2015-521-237&gt;&gt;] and yeast [&lt;&lt;REF:Hahn-2016-63-445&gt;&gt;] appeared to be averages of several such states, and this explains why the resolution was limited to 6 or 7 A. Once we had learned to apply yet more powerful image processing programs and had collected sufficient numbers of images to ensure that the various conformations in the particle set were well enough populated, it became possible to tease them apart by image processing. The first to succeed in this endeavour in our laboratory was Alexander Hahn, as a postdoc in the independent group of Thomas Meier (now Professor of Structural Biology at Imperial College London). Alex and Thomas had isolated the chloroplast ATP synthase from kilograms of spinach bought at a local supermarket--incidentally by ammonium sulphate precipitation, an old, brute-force method which I would never have thought would yield structurally and functionally intact preparations of such an intricate and fragile assembly. Alex collected a large number of movies with the faster and better Falcon-III camera installed on our new FEI Titan Krios electron microscope. He generated a very beautiful map that extended to ~2.9 A resolution in the catalytic F1 assembly of the most populated of three rotary states [&lt;&lt;REF:Hahn-2018-360-?&gt;&gt;] (Fig. 12 &lt;&lt;FIG:F12&gt;&gt;). In the catalytic sites of the F1 head, bound nucleotides, Mg ions and even water molecules were resolved. A comparison of the three rotary states indicated that the long and thin peripheral stalk, which connects the F1 head to the Fo rotor, is flexible, whereas the gamma-subunit of the central stalk works as a rigid, non-flexible driveshaft. Without the benefit of high-resolution cryoEM, earlier biophysical work and computer simulations had come to the opposite conclusion, i.e. that the central stalk was flexible and the peripheral stalk was rigid [&lt;&lt;REF:Wächter-2011-108-3924&gt;&gt;].\nFig. 12.\nCryo-EM structure of the spinach chloroplast ATP synthase. Surface representation of the map density with subunits alpha, dark green; beta, light green; gamma, blue; delta, orange; epsilon, purple; a, light blue; b, violet; b', pink; 14-membered c-ring rotor c14, pale yellow; ATP, red. The Fo subcomplex (abb'c14) is embedded in the thylakoid membrane (grey), while the catalytic F1 subcomplex (alpha3beta3gammaepsilondelta) extends into the stroma. N-terminal helices of the alphaA, alphaC and alphaE subunits are indicated. The resolution of the F1 subcomplex is 2.9 A. From [&lt;&lt;REF:Hahn-2018-360-?&gt;&gt;].\nEncouraged by Alex Hahn's and Thomas Meier's chloroplast ATP synthase structure, Bonnie Murphy, a new postdoc, together with Niklas Klusch set out to tackle the Polytomella dimer. They progressed rapidly to a 4.1 A map that refined locally to 3.7 A in the most rigid parts of the peripheral stalk and Fo subcomplex. In this region, all sidechains were resolved [&lt;&lt;REF:Klusch-2017-6-?&gt;&gt;]. The map revealed two aqueous channels, one each extending from the middle of the membrane to the lumenal or matrix membrane surface. Each channel was lined by charged and polar residues of the a-subunit stator. In the hydrophobic core of the Fo subcomplex, the two half-channels were separated by a conserved arginine sidechain and approached one another to within ~6 A. The steep potential gradient at the channel end points would exert a strong force on the charge of the critical glutamates in the c-ring rotor, generating torque and resulting in net directional rotation [&lt;&lt;REF:Kühlbrandt-2019-88-515&gt;&gt;].\nIn the 3.7 A map, the static parts of the stator assembly were well-defined, but rotary states were not resolved. Data for the 3.7 A map had been collected with a K2 Summit detector and the in-column energy filter of the JEOL 3200 FSC. Collecting a much larger number of particle images (~730 000) with a more modern, fully automated microscope (Titan Krios G2) and a considerably better detector (Falcon-III), Niklas and Bonnie produced a map of the Polytomella ATP synthase dimer at an overall resolution of 2.7-2.8 A. In an image processing marathon, Bonnie was able to separate 13 well-defined rotary substates that together provided a detailed picture of the molecular motions that accompany c-ring rotation and ATP synthesis [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;] (Fig. 13 &lt;&lt;FIG:F13&gt;&gt;). We found that the F1 head rotates along with the central stalk and c-ring rotor for the first ~30degrees of each 120degrees primary rotary step to facilitate flexible coupling of the stoichiometrically mismatched F1 and Fo subcomplexes. Flexibility is mediated by the interdomain hinge of the OSCP subunit (known as delta in the chloroplast ATP synthase) on top of the F1 head. Since this subunit is highly conserved, as are all crucial elements of rotary catalysis, we can be confident that our mechanistic conclusions apply in principle to all rotary ATP synthases. With the high-resolution structures of the ATP synthase dimer and complex I from Polytomella and the known distribution of complex I and ATP synthase dimer rows in the membrane [&lt;&lt;REF:Davies-2011-108-14121&gt;&gt;], we can now draw a model of a crista that contains all the known molecular details and summarizes how they are arranged and work together in the mitochondrial inner membrane (Fig. 14 &lt;&lt;FIG:F14&gt;&gt;).\nFig. 13.\n(a) Cryo-EM map density of the 62-subunit 1.58 MDa Polytomella ATP synthase dimer. The monomer map on the left is shown in grey. The map on the right is coloured by subunit. The F1 head consists of three catalytic beta subunits (bright green) and three structural alpha subunits (dark green). The dashed white line indicates the section plane in (b). The long C-terminal extensions of beta wrap around the alpha subunits on the outside of the F1 head. The central stalk subunits gamma (dark blue), delta (cyan) and epsilon (light blue) connect F1 to the Fo motor in the membrane (grey). The c10 rotor ring (yellow) and subunit a (bright blue) are the main components of the Fo motor complex. The two-domain OSCP subunit (orange) links the three alpha subunits to the peripheral stalk subunits ASA 1-10. The local map resolution is 2.7 A for the peripheral stalk, Fo, and c-ring, and 3.0-3.2 A for the F1 head and central stalk. (b) Rotary substates. Sections through cryo-EM maps of the Polytomella ATP synthase F1 head at the level of nucleotide-binding sites (dashed white line in a). Top row: Six substates of primary state 1. Middle row: Four substates of primary state 2. Bottom row: Three substates of primary state 3. Bright green, beta-subunits; dark green, alpha-subunits; blue, gamma-subunit; red, ADP in catalytic sites; orange, structural ATP; grey, peripheral stalk. Red lines indicate the rotary positions of consecutive substates. From [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;].\nFig. 14.\nMolecular organization of mitochondrial inner membrane cristae. Cryo-EM structures of the F1Fo ATP synthase dimers [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;] and complex I [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;], both from Polytomella sp. in the membrane. Three-dimensional density maps at 2.7 A (ATP synthase) and 2.9 A resolution (complex I) are coloured by subunit. ATP synthase dimers assemble into long rows that impose a tight local curvature on the inner membrane, giving rise to the formation of cristae as mitochondrial microcompartments. Complex I and other respiratory chain complexes (not shown) occupy the flat membrane regions on either side of the dimer rows. Complex I generates most of the proton gradient across the inner membrane, pumping protons (dashed red arrows) into the cristae, which work as proton traps. Protons (red dots) flow through channels in the Fo subcomplex of the ATP synthase from the cristae lumen (~pH 7.2) to the mitochondrial matrix (pH 8), driving the rotor (yellow) in the membrane. The torque generated by the electrochemical gradient is transmitted via the central stalk (blue) to the catalytic F1 subcomplex (green), where ATP is produced from ADP and phosphate by rotary catalysis.\n\nOutlook\nIn the past 40 years, cryoEM of membrane proteins has come further than most of us would have predicted, and it has done so more rapidly than any of us could have hoped. Yet we are by no means at the end, but right in the middle, or perhaps even still at the beginning of this exhilarating and fascinating journey.\nFor single-particle cryoEM, a few technical challenges are left. Beam-induced movement, for many years the bane of cryoEM, may have been overcome once and for all by the new HexAuFoil grids developed by Chris Russo and colleagues [&lt;&lt;REF:Naydenova-2020-370-223&gt;&gt;], although they have not yet been widely tested. Alternative approaches, such as controlled devitrification of vitrified samples [&lt;&lt;REF:Wieferig-2021-8-186&gt;&gt;], may help in special circumstances. Protein adsorption and denaturation at the air-water interface is a fundamental and ubiquitous problem in single-particle cryoEM, which means that only a small percentage (typically ~5-20%) of a particle set contribute to a high-resolution structure. The remaining 80-95% are in some way damaged or distorted at the air-water interface. It may be possible to overcome these problems with hydrophilized graphene support films, which the complexes adhere to in random orientations in preference to the air-water interface [&lt;&lt;REF:D’Imprima-2019-8-?&gt;&gt;,&lt;&lt;REF:D’Imprima-2021-54-?&gt;&gt;].\nUltimately, the success of single-particle cryoEM depends on the quality of the protein sample that goes on the grid. There is no substitute for optimizing the conditions a protein is exposed to during isolation. Therefore, when all other technical and computational issues are settled, protein biochemistry is likely to remain as the hardest part of single-particle cryoEM, because each protein is different, details matter, and there are no shortcuts.\nWith the most recent detectors and a matching battery of image processing software, structures of virtually any macromolecular assembly above a certain size can now be tackled by single-particle cryoEM. The current lower limit with soluble proteins is ~60 kDa [&lt;&lt;REF:Herzik-2019-10-?&gt;&gt;]. The recent 3.3 A structure of the 55-kDa proton-coupled folate transporter in complex with a nanobody [&lt;&lt;REF:Parker-2021-595-130&gt;&gt;] suggests that the size limit for membrane proteins is similar, even though single-particle cryoEM of small membrane proteins is inherently more difficult. The detergent micelles or lipid nanodiscs that keep them in solution do not have a defined structure and reduce the contrast in the trans-membrane region, which makes the particles harder to align. The theoretical size limit has been estimated by Richard Henderson for perfect images recorded with 300-kV electrons (see Table 2 in [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;]) to be ~38 kDa. This limit is now almost within reach and likely to be approached asymptotically. With 100 -kV microscopes [&lt;&lt;REF:Naydenova-2019-6-1086&gt;&gt;] and detectors, slightly smaller molecules should become accessible, because the scattering cross section is higher and the ratio of elastically and inelastically scattered electrons more favourable at lower acceleration voltage (see Fig. 2 &lt;&lt;FIG:F2&gt;&gt; in [&lt;&lt;REF:Henderson-1995-28-171&gt;&gt;]).\nVery accurate high-resolution structures can be obtained by electron diffraction of microscopic 3D crystals, a technique known as MicroED. MicroED has been applied mostly to small or very small proteins or protein fragments, in particular to a number of fibre-forming amyloid proteins that are recalcitrant to other techniques [&lt;&lt;REF:Rodriguez-2017-46-79&gt;&gt;]. As such, it is a welcome addition to the cryoEM methods spectrum. It is however unlikely to become as popular as single-particle cryoEM or cryoET, because it still requires 3D protein crystals, which removes one of the key advantages of cryoEM over X-ray crystallography.\nAt present, structural biology is undergoing a profound transformation. After years of futile efforts, computational methods for solving the protein folding problem have become so powerful that the structure of virtually any single protein can now be instantly predicted with near-atomic accuracy from its genomic sequence. A new deep-learning algorithm called AlphaFold2 [&lt;&lt;REF:Jumper-2021-596-583&gt;&gt;,&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;] has been shown to produce correct protein structures even if there are no related templates in the protein data bank (pdb). We tried it out with a new membrane protein without known homologues in the pdb, and the result was shockingly similar to our unreleased experimental structure. In the short to medium term, AlphaFold2 will thus shift the emphasis and take some of the pain and hard work (but also fun!) out of structure determination by cryoEM and X-ray crystallography. However, it will not make the experimental methods redundant.\nA single structure is rarely enough to understand how a protein works. Most if not all proteins in the cell undergo conformational changes while performing their tasks, and cryoEM in particular is good at resolving different, simultaneously present functional states. Such conformational ensembles would be difficult to predict. More often than not, the function of a protein depends on interactions with other proteins or small molecules, and some of the interaction partners may not be known. Similarly, the structures of protein subunits that assemble into large macromolecular complexes may not be predicted reliably at present. Computational methods are unlikely to outperform cryoEM in determining the structures of large multi-subunit complexes, such as complex I or ATP synthase in the short term. It will be some time before AlphaFold can generate the correct structures of the 13 different rotary states of the Polytomella ATP synthase [&lt;&lt;REF:Murphy-2019-364-?&gt;&gt;], the 8 different conformations of an ABC transporter [&lt;&lt;REF:Hofmann-2019-571-580&gt;&gt;] or even the two different states of plant complex I [&lt;&lt;REF:Klusch-2021-33-2072&gt;&gt;] as parts of a molecular mechanism. There is however no principal reason why it should not work, and therefore, it is likely to happen sooner or later. In any event, single-particle cryoEM and crystallography will be critical for validating structure predictions.\nFor cryoET, the best times are still ahead. As more and more high-resolution structures of proteins and complexes are done or predicted, cryoEM labs will increasingly turn from crystallography and single-particle work to cryoET, which is not subject to the same constraints of sample purity and homogeneity. At the same time, it will become increasingly urgent to investigate and understand the context of macromolecular components in the cell. CryoET in particular stands to gain from protein structure prediction, because it generates lower-resolution volumes quite readily, to which the predicted structures can then be fitted. CryoEM instrumentation can in principle be used equally well for single-particle work and cryoET (although an acceleration voltage of 300 kV is preferable for tomography), which makes the transition easier. CryoET does have its own constraints, but in terms of the molecular systems and biomedical questions it can address, it is essentially unlimited. Every cell contains a universe of molecular landscapes that is best explored by cryoET.\nConstraints include sample thickness and tilt geometry. Most samples are too thick, but they can be thinned into thin lamellae by focused ion beam (FIB) milling at low temperature. Cryo-FIB milling is a demanding technique but it is practiced successfully in a growing number of cryoEM laboratories around the world [&lt;&lt;REF:Wagner-2020-15-2041&gt;&gt;,&lt;&lt;REF:Zachs-2020-9-?&gt;&gt;]. Constraints of tilt geometry can be overcome by FIB-milling cylindrical pencils instead of planar lamellae, and several laboratories are experimenting in this direction.\nA major unresolved challenge in cryoET is the lack of a universal cloneable label equivalent to green fluorescent protein (GFP) in light microscopy that is fused with a protein so it can be identified and studied in detail in its native context. Correlative light/electron microscopy alleviates the problem by identifying regions of interest in a larger volume, such as a cell, but within that region individual assemblies still have to be recognized by their shape and position, e.g. relative to a membrane. Because phosphorus scatters electrons more strongly than carbon, nitrogen or oxygen, phospholipid bilayers show up well in tomographic volumes, and therefore, cryoET works well for protein complexes in or at a membrane. CryoET of HIV capsids in vitrified aqueous buffer has achieved 3.4 A resolution [&lt;&lt;REF:Turonova-2017-199-187&gt;&gt;]. There is no fundamental reason why cryoET, just as single-particle cryoEM, should not go to true atomic resolution, given a sufficiently large number of tomographic volumes to average. In cellular tomography, the target complexes are usually surrounded by other macromolecules of similar density, which makes the targets more difficult to detect and average than single particles in dilute aqueous buffers. The ultimate limitation in both cases is the signal-to-noise ratio of the electron micrographs.\nThe signal-to-noise ratio of electron micrographs can be enhanced by a phase plate. It is fair to say that current phase plate designs, including the Volta phase plate [&lt;&lt;REF:Buijsse-2020-218-?&gt;&gt;] or our own electrostatic Boersch phase plate [&lt;&lt;REF:Walter-2015-153-22&gt;&gt;], have not kept their promise. Even though they can improve low-resolution image contrast, potential gains at high resolution are more than offset by charging problems and a lack of reproducibility associated with introducing an external device into the electron beam [&lt;&lt;REF:Buijsse-2020-218-?&gt;&gt;]. These problems may be overcome by a new elegant phase plate design that shifts the phase of the electron beam by the ponderomotive force of a strong laser [&lt;&lt;REF:Schwartz-2019-36-1016&gt;&gt;]. The laser phase plate enhances contrast in the entire resolution range and would be particularly useful for cryoET. However, its implementation is technically demanding, the electron microscope needs an extra lens and a tremendously powerful laser is necessary for shifting the phase of a 300 kV beam by 90degrees for optimum contrast. For single-particle cryoEM, recent improvements in electron microscope hardware, in particular cold field emitters, better image filters and direct electron detectors, have increased phase contrast to an extent that makes the available phase plates less of a help than a hindrance for high resolution.\nEven though the present generation of direct electron detectors is already superb, further improvements in their design and production are likely and will benefit both single-particle cryoEM and cryoET. At present, the DQE of the best available detectors (Falcon-IV) is 1.0 at low resolution and close to 0.4 at Nyquist frequency [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;]. The Nyquist frequency defines the maximum resolution possible with the given pixel size of the detector. For a perfect detector, the DQE would be 1.0 at Nyquist, but this may be hard to achieve. Still, any improvement will boost the signal-to-noise ratio at high resolution and translate into better maps from fewer particle images or tomograms.\nSo far, the high cost of cryoEM hardware does not seem to have taken away much of its popularity. The number of new cryoEM labs around the globe is still growing, and research councils and agencies are willing to support them, because of the expected gains in mechanistic insight for basic biomedical research, which will ultimately translate into benefits for human health. But this trend cannot go on forever. Cost scales roughly with acceleration voltage. It is possible to produce less expensive 100-kV electron microscopes that perform equally well, and in some ways better for single-particle cryoEM than 300-kV instruments [&lt;&lt;REF:Naydenova-2019-6-1086&gt;&gt;]. Bringing the cost of EM hardware down substantially will be essential if cryoEM is to be as widely accessible to biomedical research as protein crystallography has been in the past. An alternative would be to create more of the national cryoEM facilities such as eBIC at the Diamond synchrotron in the UK or the National Centers in the USA, to cumulate resources and provide cryoEM as a service. However, sample preparation for cryoEM is still very much a skill and a craft. Automation is in its infancy [&lt;&lt;REF:Budell-2021-?-?&gt;&gt;,&lt;&lt;REF:Weissenberger-2021-18-463&gt;&gt;] and reproducibly thin ice layers, a key factor for high resolution, are hard to achieve. Similarly, collecting top-quality high-resolution images requires close familiarity with the sample. In the long run, it may therefore be preferable not to separate researchers from their microscopes and cryoEM samples.\nWithin the past year, two cryoEM structures of a standard soluble test specimen, recombinant mouse apoferritin, have been reported at ~1.25 A, i.e. true atomic resolution [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;,&lt;&lt;REF:Yip-2020-587-157&gt;&gt;]. This proves that cryoEM is capable of delivering structures of equal quality as atomic-resolution X-ray crystallography, which not many of us would have imagined a few years ago. For most biological questions, a resolution ~2 A is probably more than adequate, but higher resolution is always nice, and it is hard to predict what it might reveal. Further improvements in electron microscopes, detectors and image processing software are bound to result in more and more atomic-resolution structures of interesting membrane proteins. Already with the present generation of cryoEM hardware and software, water molecules are resolved in the cytochrome bd oxidase [&lt;&lt;REF:Safarian-2019-366-100&gt;&gt;], complex I [&lt;&lt;REF:Parey-2021-7-?&gt;&gt;,&lt;&lt;REF:Kampjut-2020-370-?&gt;&gt;] and the GABAA receptor [&lt;&lt;REF:Nakane-2020-587-152&gt;&gt;]. At very high resolution, cryoEM may be able to visualize ionization states of bound ions and protonation states of sidechains and water molecules, which would provide more detailed insights into molecular mechanisms. The future for studying membrane proteins by cryoEM gets brighter and more fascinating all the time!\n\nFunding\nMost of the work described in this review was funded by the Max Planck Society. Some of the earlier work at EMBL was funded by the European Molecular Biology Laboratory.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state..\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [14.83s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"sample_type\": [\"micro-organism\", \"virus\", \"other\"],\n  \"sample_preparation_type\": \"Unknown\",\n  \"sample_preparation_buffer_ph\": \"Unknown\",\n  \"grid_vitrification_cryogen\": [\"ETHANE\", \"NITROGEN\"],\n  \"sample_preparation_cryo_protectant\": \"glucose\",\n  \"grid_model\": \"Unknown\",\n  \"grid_material\": \"GOLD\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"HOLEY\",\n  \"grid_vit_ctemp\": \"Unknown\",\n  \"grid_vit_chumid\": \"Unknown\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"Unknown\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"sample_type\": [\"micro-organism\", \"virus\", \"other\"],\n  \"sample_preparation_type\": \"Unknown\",\n  \"sample_preparation_buffer_ph\": \"Unknown\",\n  \"grid_vitrification_cryogen\": [\"ETHANE\", \"NITROGEN\"],\n  \"sample_preparation_cryo_protectant\": \"glucose\",\n  \"grid_model\": \"Unknown\",\n  \"grid_material\": \"GOLD\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"HOLEY\",\n  \"grid_vit_ctemp\": \"Unknown\",\n  \"grid_vit_chumid\": \"Unknown\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"Unknown\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 181,
      "prompt_tokens": 21684,
      "total_tokens": 21865
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_fb84c680ea"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "sample_type": [
    "micro-organism",
    "virus",
    "other"
  ],
  "sample_preparation_type": "Unknown",
  "sample_preparation_buffer_ph": "Unknown",
  "grid_vitrification_cryogen": [
    "ETHANE",
    "NITROGEN"
  ],
  "sample_preparation_cryo_protectant": "glucose",
  "grid_model": "Unknown",
  "grid_material": "GOLD",
  "grid_mesh": "Unknown",
  "grid_support_topology": "HOLEY",
  "grid_vit_ctemp": "Unknown",
  "grid_vit_chumid": "Unknown",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "Unknown",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/end] [1:chain:RunnableSequence] [14.84s] Exiting Chain run with output:
{
  "sample_type": [
    "micro-organism",
    "virus",
    "other"
  ],
  "sample_preparation_type": "Unknown",
  "sample_preparation_buffer_ph": "Unknown",
  "grid_vitrification_cryogen": [
    "ETHANE",
    "NITROGEN"
  ],
  "sample_preparation_cryo_protectant": "glucose",
  "grid_model": "Unknown",
  "grid_material": "GOLD",
  "grid_mesh": "Unknown",
  "grid_support_topology": "HOLEY",
  "grid_vit_ctemp": "Unknown",
  "grid_vit_chumid": "Unknown",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "Unknown",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nIn the last decade, cryo-electron microscopy (cryo-EM) has become a powerful technique to determine the structures of macromolecular complexes. With advances in cryo-EM image processing algorithms, large endogenous complexes have been solved at high resolutions such as the phycobilisomes in red algae &lt;&lt;REF:Ma-2020-579-146&gt;&gt;, the mitochondrial membrane bending supercomplex &lt;&lt;REF:Muhleip-2023-615-934&gt;&gt; and the doublet microtubules of the cilia &lt;&lt;REF:Ma-2019-179-909&gt;&gt;,&lt;&lt;REF:Ichikawa-2019-116-19930&gt;&gt;. Recently, in situ cryo-electron tomography (cryo-ET) from thin lamellae of cells prepared by a cryo-focused ion beam (cryo-FIB) instrument has emerged as a transformative technique, revolutionizing our understanding of cellular structures and molecular processes &lt;&lt;REF:Berger-2023-20-499&gt;&gt;. With high-throughput tilt series acquisition and improvement of subtomogram averaging software &lt;&lt;REF:Wan-2020-26-2516&gt;&gt;-&lt;&lt;REF:Zivanov-2022-11-e83724&gt;&gt;, subtomogram averaging of complexes can reach sub-nanometer resolution and in some cases better than 4 A resolution &lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;,&lt;&lt;REF:Xing-2023-381-70&gt;&gt;-&lt;&lt;REF:Tai-2023-9-116&gt;&gt;. Structures of protein complexes obtained in situ often contain unknown components, unlike in vitro reconstituted protein complexes. In addition, depending on the cellular context, space-and/or time-specific interactions with unknown proteins might be revealed by subtomogram averaging. Due to the lower resolution of these in situ structures, identification of proteins requires a different approach than traditional molecular modelling programs.\nFor high-resolution structures, there are many methods that can be used for the modelling and identification of unknown proteins including CryoID, DeeptracerID, FindMySequence, and modelAngelo &lt;&lt;REF:Jamali-2023-?-2023.2005.2016.541002&gt;&gt;-&lt;&lt;REF:Pfab-2021-118-e2017525118&gt;&gt;. These programs trace and model the backbone of the protein density. Then, the identity of the protein is predicted by comparing the side-chain densities against a database of protein sequences. This approach was successful in determining many proteins in the doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;,&lt;&lt;REF:Leung-2023-186-2880&gt;&gt;, radial spokes and central apparatus and the phycobilisomes &lt;&lt;REF:Jamali-2023-?-2023.2005.2016.541002&gt;&gt;. However, all these methods only work reliably at a resolution better than 4 A.\nIn most cases, the attainable resolution by subtomogram averaging is lower than 4 A. This is due to the low throughput of cryo-FIB milling and low abundance of the target protein complex. Furthermore, in situ structures can have problems related to orientation, sample thickness and signal-to-noise ratio. Therefore, it is a challenge to identify proteins from in situ subtomogram averages.\nIn the last few years, there has been a breakthrough in artificial intelligence (AI)-assisted protein structure prediction. Programs like AlphaFold &lt;&lt;REF:Jumper-2021-596-583&gt;&gt;, ColabFold &lt;&lt;REF:Mirdita-2021-?-2021.2008.2015.456425&gt;&gt; and RoseTTAfold &lt;&lt;REF:Baek-2021-373-871&gt;&gt; can produce accurate structure predictions. With the establishment of the AlphaFold Protein Structure Database (AlphaFold DB), everyone now has access to over 200 million predicted models. For certain organisms such as humans, mice, zebrafish and nematodes, AlphaFold2-predicted models of the entire proteome are available.\nWith the good accuracy of the AI-predicted models, it is possible to identify the fold signature of a protein at a resolution of less than 4 A by fitting the AI-predicted models into the density. Using complementary data such as in situ chemical cross-linking mass spectrometry, quantitative mass spectrometry and/or BioID, it is possible to identify proteins from the fit list. For example, microtubule inner proteins (MIPs) were identified from the in situ subtomogram average of human sperm doublet microtubules at ~6-7 A using Colores Situs program by fitting over 21,000 AlphaFold2-predicted protein models of the mouse proteome &lt;&lt;REF:Chen-2023-186-5041&gt;&gt;. In another study, 38 proteins were identified from the mouse sperm by manually fitting AlphaFold2-predicted models of proteins found in the mass spectrometry analysis of the same sample &lt;&lt;REF:Tai-2023-9-116&gt;&gt;.\nWhile those studies illustrate that it is possible to identify well-structured proteins from a map lower than 4 A resolution, the methods are not automated or are difficult to use. In addition, the criteria to evaluate and identify proteins from the fitting solutions are not clear from these studies. While the accuracy of the AI-predicted model is high for compact domains, the tertiary structure of the predicted model might not be correct, due in part to poor prediction of flexible regions. Therefore, protein identification is easier when domains are extracted from each AI-predicted model and used instead of the entire model.\nIn this work, we developed a program called DomainFit to identify domains of proteins from cryo-EM maps of lower resolution than 4 A. DomainFit uses popular programs such as Phenix &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt;, R &lt;&lt;REF:???-2021-?-?&gt;&gt; and UCSF ChimeraX (ChimeraX) &lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;, making it easy to install and accessible to people. In addition, DomainFit is flexible and has a clear statistical approach to evaluate the identity of proteins. Lastly, DomainFit uses ChimeraX, which allows easy visualization to aid in evaluating the fit. Our workflow will be useful for the upcoming wave of in situ structures as cryo-FIB and cryo-ET become more popular.\n\nResults\n\nAn automated pipeline for domain parsing and fitting into cryo-EM map\nOverall, the program is designed to first download a database of AI-predicted models of candidate proteins (i.e. full species proteome or a limited list based on mass spectrometry studies), then divide each model into different domains and fit each individual domain into a segmented cryo-EM density. Finally, the program generates a statistical evaluation of all the fitted domains (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). The identification of the correct domain can be based purely on statistical analysis but also on other complementary data such as surrounding densities, protein size, chemical cross-linking and quantitative mass spectrometry.\nFigure 1:\nWorkflow of DomainFit.\n(A) Overview of the program. (i) Download the predicted PDBs for all candidate proteins. (ii) Predict domains and split them into single PDB files. This step can be done through the script process_predicted_models.py or results from other programs such as DPAM. (iii) Fit each domain into the unknown density using ChimeraX nd generate the best fitting position of each domain and fitting scores. (iv) Interpret the fitting results based on p-value, correlation, and size of proteins.\nThe list of candidate proteins can be obtained from the mass spectrometry results of the target sample such as the proteome of isolated organelles or pull-down analysis. In case a targeted list is not available, it is possible to use AI-predicted models of the entire proteome of the organism, even though it will increase computational time significantly. In DomainFit, we provide a script getAlphaFoldPDBs.py to automatically download the AlphaFold2-predicted models from the AlphaFold DB using a list of protein UniProtIDs (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). The downside of downloading automatically from the AlphaFold DB is that for many organisms, high molecular weight proteins (&gt; 100kDa) are not yet available in the database. These missing proteins can be predicted using ColabFold, RoseTTAFold or a local instance of AlphaFold.\nMany ideas have been proposed to break PDB models into compact domains automatically &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt;,&lt;&lt;REF:Croll-2022-?-?&gt;&gt;,&lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt;. With AI-predicted models, it is possible to partition into domains based on the predicted alignment errors (PAE) of each model (Fig. 2A-D &lt;&lt;FIG:fig2&gt;&gt;). PAE is the expected positional error for residue x if the predicted and actual models are aligned at residue y &lt;&lt;REF:Jumper-2021-596-583&gt;&gt;. Using an image-based approach, it is possible to partition predicted models into domains based on their respective PAE files (Fig. 2B-D &lt;&lt;FIG:fig2&gt;&gt;) &lt;&lt;REF:Croll-2022-?-?&gt;&gt;. We used phenix.process_predicted_model program from the Phenix package &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt; for the partitioning of the PDB into domains because of its flexibility. phenix.process_predicted_model can function based on PAE but also based on the 3D arrangement of the proteins without PAE information. Therefore, we implemented a Python wrapper process_predicted_models.py to partition PDBs into domains in batch (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). Users can customize the options for parsing based on their needs.\nFigure 2.\nDomain partitioning and fitting evaluation in DomainFit.\n(A) An example of an AlphaFold2-predicted model of a multi-domain protein (RIB72B, UniprotID I7MCU1). (B) The PAE plot of the RIB72B. (C) Domain partitioning of RIB72B based on the PAE plot. (D) Domain partitioning of RIB72B corresponding to (C). (E) Fitting of a domain of RIB72B inside its corresponding segmented density at 4 A resolution. (F) Histogram of the z-transformed correlation score of all obtained fits of the correct domain of RIB72B. The top hit (red) is separated from the score distribution. (G) Histogram of the z-transformed correlation score of a wrong domain into the same density in (F). (H) The scatter plot to visualize the value of - log10 of Benjamini Hochberg adjusted p-value versus the normalize cross correlation coefficient of each domain fitting into the density of interest. The top right corner two points are well separate for the rest of the point cloud, indicating likely well-matched domains into the density.\nTo increase the flexibility of our program, process_predicted_models.py writes out the information of the predicted domains using the format employed by DPAM &lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt;. As a result, we can use domain information data from either phenix.process_predicted_model or existing domain data of certain organisms such as human, mouse, and zebrafish from DPAM &lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt; to generate the PDBs of individual domains. In the case of a small number of target proteins, we can even edit the domain information files manually to generate customized domain parsing.\nAt this step, we recommended imposing a filter on the minimum and maximum domain sizes for fitting. At the lower end, 40 amino acids is the minimum size that allows us to reduce the number of domains used without affecting the results since domains under 40 amino acids tend to be either wrongly partitioned or not compact. On the upper limit, we recommend using a bigger value like 1,000 amino acids as a fail-safe in the case that domain parsing does not work properly.\nTo prepare for the fitting of domains, a volume containing a compact density should be segmented from the cryo-EM map. The compact density means either globular or coiled-coil bundle. A long helical density does not work since its shape is not unique for identification of protein fold. Ideally, the segmented density should be equivalent to a domains of more than 50-300 amino acids in size. This serves two purposes: (i) to reduce computational time, and (ii) to avoid the wrongly predicted tertiary conformation caused by long flexible loop between different domains.\nOnce the segmented density is prepared, fit_domains_in_chimerax.py is used to fit all the individual domain models into the density to find the best fitting position of each domain (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;, Fig. 2E &lt;&lt;FIG:fig2&gt;&gt;). The script uses the ChimeraX fitmap command to fit each domain in the density from a fixed number of initial random translated and rotated search positions within the density defined as \"initial search placements\". Instead of exhaustive six dimensional search like Colores package in Situs, for global search, ChimeraX fitmap places the model randomly in rotation and translation within the map, and performs rigid-body local optimization from each initial placement. After the local optimization, ChimeraX will group the fitting solutions of based on similarity in orientation. Therefore, the number of fitting solutions tend to be smaller than the number of initial search positions due to clustering. These fitting solutions's correlation score are then z-transformed and then used for the calculation of the p-value, the likelihood that the top hit is correct and clearly significant compared to the rest of the fitting solutions (See Materials &amp; Methods, Fig. 2F &lt;&lt;FIG:fig2&gt;&gt;). Once the fitting is done for every domain in the density, we expect that top hits with the correct fold/domain are clearly distinguished from other fitting solutions by their z-transformed scores, whereas those with the incorrect domain are not so differentiated (Fig. 2G &lt;&lt;FIG:fig2&gt;&gt;). Visualizing the fitting statistics i.e. p-values and cross correlation coefficient of all domains allow a quick way to see how distinct is the top hits (Fig. 2H &lt;&lt;FIG:fig2&gt;&gt;).\nA comma-separated value (CSV) file containing the best fitting position of each domains is written at the end, sorted first by p-value and then by the correlation score for quick evaluation.\n\nDomainFit benchmarking for MIP identification\nTo assess the efficiency of DomainFit, we tested it on segmented densities from the single-particle cryo-EM map of the doublet microtubule from Tetrahymena thermophila (EMD-29685) at ~4 A resolution. More than 40 MIPs were identified and modelled using an AI-assisted modelling approach (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt;). Even though the resolution of the doublet microtubule cryo-EM map was high, modelling and identifying proteins was laborious and time-consuming. Our goal for this test was to see whether we could identify the MIPs using DomainFit independently.\nFigure 3:\nIdentification of MIPs in the T. thermophila doublet microtubule.\n(A) The doublet microtubule structure with MIPs (PDB: 8g2z). (B) Densities (red) in the cryo-EM map of the doublet microtubule (EMD-29685) for testing with DomainFit. (C) A histogram of domain size. Red bars indicate domains of less than 40 amino acids and are excluded from fitting. The x-axis is limited to a domain size of 800 amino acids. (D) DomainFit ranking by p-value and correlation about mean of the correct domain for 24 densities. (E) Example of the perfect match between the AlphaFold2-predicted model of PACRGB fitted within the corresponding density. (F) The domain of RIB22 is not partitioned as expected. However, the bigger domain was still fitted correctly into the density due to good agreement between the AlphaFold2-predicted model and the density. Due to incorrect parsing and the fact that the EF-hand domain is abundant in the cilium, the fit of the correct domain is ranked 9 but the p-value is still ranked 1st. (G) When the fold of the density is common (EF-hand domain RIB571-91 - Density 10), the top hits consist of domains with a similar fold. The top five hits are all domains from CFAP115, which have the same EF-hand fold as the correct domain RIB571-91. (H) The experimentally determined model of RIB57 (PDB: 8g2z) is shown inside its corresponding density. (I) There is a big discrepancy between the experimentally determined model (cyan) and AlphaFold2-predicted model (purple) of RIB571-91 (RMSD: 5.164 *), which leads to the poor fitting of RIB57.\nWe segmented 24 densities from 13 identified MIPs in the cryo-EM map corresponding to different domains of MIPs (Supplementary Table 1, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We established a database comprising 856 AlphaFold2-predicted models based on the mass spectrometry data of the WT cilia without membrane (994 detected proteins) &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. One hundred and thirty eight proteins were not available from the AlphaFold DB because of a lack of predictions due to their high molecular weight. For benchmarking purpose, we ignored them for this test since the 856 AlphaFold2-predicted models covered all the tested MIPs.\nUsing process_predicted_models.py and save_domains_from_info.py, we divided 856 AlphaFold2-predicted models into 2819 domains. We removed domains of less than 40 amino acids which left us with 1561 domains for fitting (Fig. 3C &lt;&lt;FIG:fig3&gt;&gt;). We then fitted all 1561 domains into each density filtered at 4 A using 200 initial search position placements. On average, each density search took ~30 minutes using 12 cores (20 threads) on an i7-12700K (12 core-CPU and 32GB RAM).\nOut of all the densities tested, we found 22 out of 24 correct domains are present in the top 5 hits ranked by descending p-value (Supplementary Table 1, Fig. 3D &lt;&lt;FIG:fig3&gt;&gt;, Fig. S1 &lt;&lt;FIG:figs1&gt;&gt;). Overall, we found the p-value to be a better predictor of correct identification and fitting than the normalized cross correlation coefficient, which is called correlation about mean in ChimeraX. Often, the domains were correctly found as the top hits with the highest correlation and best p-value such as PACRGB, an inner junction protein &lt;&lt;REF:Khalifa-2020-9-e52760&gt;&gt; (Fig. 3E &lt;&lt;FIG:fig3&gt;&gt;). Our results show that the p-value ranks consistently higher than the correlation score for the correct domains for the densities (Fig. 3D &lt;&lt;FIG:fig3&gt;&gt;). In the cases where domains have a common fold, multiple domains returned a p-value of 2.22*10-16 (limit of R program). Thus, we sorted the fitting solutions first based on p-values, and then by their normalized cross correlation coefficient. As a result, hereafter, top hits will refer to the top p-value ranked hits.\nIn some cases such as Density 6 (CFAP5213-317), by looking at the fit of the domains in the density, we can roughly estimate the size of the domain and establish a minimum domain size threshold, which allows quicker identification of the correct domain (Fig. S2A-B &lt;&lt;FIG:figs2&gt;&gt;).\nThere are quite common cases where the partitioned domain is bigger than the segmented density (Supplementary Table 1, Fig. S1 &lt;&lt;FIG:figs1&gt;&gt;, Density 6, 10-13, 16-17, 19-22, 24). This might affect the fitting of DomainFit. For example, our parameters for domain parsing tend to keep entire EF-hand domain pairs as a single domain, instead of two separate EF-hand domains. For RIB22, while the density represents only RIB221-91, RIB224-191 was partitioned as one domain (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt;). On the other hand, the correct fitting was found for RIB224-191 with a p-value and correlation value of rank 1st (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt;, Supplementary Table 1).\nInterestingly, for Density 10, corresponding to the EF-hand domain RIB571-91, DomainFit failed to find the correct domain (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt;). While all the top hits were the wrong protein, they all had the correct EF-hand fold (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt;). when we compared the experimentally determined model of RIB57 (Fig. 3H &lt;&lt;FIG:fig3&gt;&gt;) and the AlphaFold2-predicted model, there was a big difference between the two models from residues 6 to 91 (RMSD 5.164 A, Fig. 3I &lt;&lt;FIG:fig3&gt;&gt;). Therefore, the failure to find RIB571-91 density (Density 10) was due to both the difference in the AlphaFold2-predicted model and the unsuccessful parsing of RIB571-91 domain.\nWe conclude that DomainFit works well but attention needs to be paid to potential mis-identification of proteins due to their size and domain parsing. Also, running with a higher number of initial search placements is a safeguard against inaccuracy in domain parsing to accommodate more initial translated and rotated positions.\n\nPerformance of DomainFit depending on the number of initial placements and resolution\nOur test results showed that DomainFit is effective in identifying compact densities and the p-value is the most reliable indicator of the correct hit. For p-value to be estimated properly, a reasonable number of data points, i.e. fitting solutions are needed. Therefore, we wanted to explore the sensitivity of the p-value estimation to the number of initial search placements. We ran DomainFit for Density 1 100 times for each initial search placement value ranging from 1 to 300 and plotted the average p-value found. Our analysis shows that above 150 initial search placements, the resulting p-value is consistently found to be as high a value as it can be (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Thus, setting a search placement value of 150 should yield optimal results for the speed and accuracy of the fits in this setup. This setting is appropriate for a domain of 100-150 amino acids while larger domains may require a higher number of initial search placements. We also looked at the z-transform correlation score for the correct domain at different numbers of initial search placements (Fig. S3A-D &lt;&lt;FIG:figs3&gt;&gt;). The statistical calculation is certainly less reliable when the number of fitting points is small. For the correct domain, if the correlation score of the top hit is distinct from that of the rest, a good p-value can still be obtained.\nFigure 4:\nPerformance of DomainFit at different numbers of initial search placements and resolutions.\n(A) -log10 of the average p-value at increasing initial search placements for a correct and an unrelated domain. Standard deviations are shown in coloured regions. (B) Appearance changes of a density filtering at 4, 6, 8 and 10 A resolution. (C) p-value rank of the correct domains fitted to different densities of MIPs at 4, 6, 8, and 10 A resolution. (D) Number of fitting solutions versus number of initial search placements for maps at 4 A (blue), 6 A (red), 8 A (green) and 10 A (yellow) resolution. (E) A histogram of the z-transformed correlation score of a correct domain fitted to a density at 4 A resolution shows a clear separation between the top hit and the rest of the fitting solutions. (F) Histograms of z-transformed correlation score of the same domain fitted to the densities filtered at 6, 8 and 10 A resolutions. There are significantly fewer fitting solutions at lower resolutions and the separation of the top hit, and the rest is reduced.\nThe number of fitting solutions found does not depend only on the number of initial search placements but also on the resolution of the map. Next, we wanted to check the effect of map resolutions on the success of DomainFit. To test whether the program could successfully identify domains at lower resolutions, we filtered and ran the density maps of the MIPs at 4, 6, 8, and 10 * (Fig. 4B &lt;&lt;FIG:fig4&gt;&gt;). We found that the search consistently identified the correct domain within the top 10 results at 4 to 8 * resolution (Fig. 4C &lt;&lt;FIG:fig4&gt;&gt;). At lower resolutions, there are significantly fewer fitting solutions found even with the same number of initial search placements (Fig. 4D &lt;&lt;FIG:fig4&gt;&gt;). This is because of the smoothening of the map at lower resolutions, which does not allow many unique fitting solutions. Because of this, ChimeraX produces a small number of clusters of fitting solutions from a large number of initial search placements. This leads to less confidence in statistical analysis of the top hits. If we look at the z-transformed correlation score at 4, 6, 8 and 10 A of the correct domain (Fig. 4E, F &lt;&lt;FIG:fig4&gt;&gt;), we can see that at 6 A, the distribution of the fitting solution and the top hits still maintains its shape, allowing reliable p-value calculation. At 8 A, while the top hit is still well separated from the distribution, the rest of the solution does not look normally distributed. At 10 A, not only does the distribution of the correlation score is not normally distributed but the difference between the top hit and the rest gets smaller.\nAs a result, DomainFit seems to work best in the 4-to-6 A resolution range. At 8 A resolution, the number of initial search placements must be increased to help with statistical calculation. At lower than 8 A, DomainFit should only be used for finding types of folds that fit well in the density by visual inspection of the top hits but it should not be used for domain identification.\n\nNew MIP identified in the doublet microtubule\nWith our success in testing and validating the DomainFit program, we attempted to identify unknown proteins from cryo-EM maps with a resolution lower than 4 A. An unidentified MIP was previously reported in the subtomogram average of the T. thermophila doublet microtubule &lt;&lt;REF:Li-2022-5-e202101225&gt;&gt; (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt;). However, this density was not observed in the single particle cryo-EM map of the doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; (Fig. 5B &lt;&lt;FIG:fig5&gt;&gt;) and at too low resolution to be identified.\nFigure 5:\nNew proteins identified from the doublet microtubule of T. thermophila.\n(A) Overview of the unknown density (yellow) from the 48-nm repeating unit subtomogram average map of doublet microtubule (EMD-24376) &lt;&lt;REF:Li-2022-5-e202101225&gt;&gt;. (B) The same density is not visible in the same view of the single particle cryo-EM map of the 48-nm repeating unit of the doublet microtubule from the T. thermophila K40R map (EMD-29692) &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. (C) The unknown density exists in the doublet microtubule map after 3D classification. (D) Segmentation of the unknown density into different small unknown densities for DomainFit. (E) DomainFit top hits for each unknown density using an AlphaFold database of salt-treated ciliome. (F) Corrected model of protein domains after examining, showing BMIP1 (UniprotID I7MB72) is density 1 and CCDC81B (UniprotID I7M688) forms density 2-12. (G) Fit of AlphaFold2-predicted model BMIP112-168 into density 1. (H) Fit of AlphaFold2-predicted model CCDC81B38-218 into density 2.\nWe re-processed the cryo-EM dataset containing the 48-nm particles of T. thermophila K40R doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; with Cryosparc &lt;&lt;REF:Punjani-2017-14-290&gt;&gt; Using 3D classification and refinement, we obtained a 4.5 A resolution cryo-EM map with the same density as the subtomogram average (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;, Fig. S4A, B &lt;&lt;FIG:figs4&gt;&gt;).\nTo identify the proteins in the unknown density, we segmented the density into 12 smaller densities (Fig. 5D &lt;&lt;FIG:fig5&gt;&gt;). To improve the accuracy of the search, we used the ciliome of the salt-treated doublet microtubules &lt;&lt;REF:Ichikawa-2019-116-19930&gt;&gt;. The reason is that the cryo-EM map of salt-treated doublet microtubules retains most of the MIPs and the unknown density &lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;. The salt-treated ciliome of 166 proteins was divided into 334 domains of at least 40 amino acids. We ran DomainFit with these 334 domain models for the above 12 densities.\nInterestingly, the top hits for the 12 densities came from only three proteins (Fig. 5E &lt;&lt;FIG:fig5&gt;&gt;, Supplementary Table 2). Upon inspecting the results, we concluded that the 12 densities are composed of only two proteins I7MB72 (TTHERM_00525130) and I7M688 (TTHERM_00649260) because of the overall architecture (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;, Fig. S5A &lt;&lt;FIG:figs5&gt;&gt;) and the fit of unique domains in both proteins (Fig. 5G-H &lt;&lt;FIG:fig5&gt;&gt;).\nI7MB72 matches unknown density 1 with high confidence in p-value and correlation (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;), referring as BMIP1 from here onwards. Upon inspection of the map, density 1 was always present in the doublet microtubule (EMD-29692) but was not identified previously. As a result, we could model extra regions of the protein BMIP1 in the T. thermophila K40R map (EMD-29692) at 3.5 A resolution, confirming the identity of unknown density 1 and the success of DomainFit (Fig. 6A, B &lt;&lt;FIG:fig6&gt;&gt;). In addition, BMIP1 is abundant in both salt-treated and native doublet microtubules similar to the profiles of most MIPs. BMIP1 has a cross-link to CFAP45, which exists as two copies between protofilaments B8B9 and B7B8, as shown in studies that usedchemical cross-linking coupled with mass spectrometry of cilia &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;,&lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt; (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). While the cross-linked residues (Lysines 213 and 222) from BMIP1 are not modelled, they seem to be within the cross-link distance (&lt; 35 A) from lysines 403 and 283 of CFAP45.\nFigure 6:\nValidation of the newly identified MIPs.\nA. Model of BMIP1 (UniprotID I7MB72) using Coot. B. Side-chain densities of a helix from BMIP1. C. Intra-cross-links (yellow) within CCDC81B and inter-cross-links (orange) between CCDC81B and CFAP106, a known MIP protein &lt;&lt;REF:Khalifa-2020-9-e52760&gt;&gt;. D. Merged super resolution-structured illumination microscopy image of T. thermophila cells with CCDC81B-GFP shows that CCDC81B only localizes to the proximal region of the cilia. Rectangle indicates the zoom-in view in (E). Red: CCDC81B-GFP signal; Green: doublet microtubule labelled by polyG-Antibody; Blue: Hoechst dye labelling the nucleus. E. The zoom-in merged image (top) and GFP channel (bottom) showing localization of CCDC81B to the proximal region.\nDensities 2-12 (Fig. 5D &lt;&lt;FIG:fig5&gt;&gt;) belong to I7M688, a CCDC81-domain containing protein. We named I7M688 as CCDC81B from here onwards. The CCDC81 domain of CCDC81B is unambiguously fitted into unknown density 2 (Fig. 5H &lt;&lt;FIG:fig5&gt;&gt;). There are a few CCDC81-containing proteins in the cilia of T. thermophila: IJ34 (UniprotID I7M9T0), CCDC81A (UniprotID I7MLF6), CCDC81B (UniprotID I7M688), Q240Y1 and Q22HG4. IJ34 is a MIP near the inner junction &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; and CCDC81A was identified as a ciliary tip protein &lt;&lt;REF:Legal-2023-222-?&gt;&gt;. Q240Y1 and Q22HG4 are clear paralogs of CCDC81B with similar architecture but are washed away in salt-treated cilia &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;.\nMass spectrometry results show that CCDC81B is resistant to salt and is similar to a MIP profile &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. In situ cross-linking mass spectrometry shows that CCDC81B is crosslinked to tubulin alpha, CFAP106, FAP210, and interestingly RIB43A &lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt;. All these cross-links are satisfied relative to the CCDC81B position in the unknown density, except for RIB43A (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). Therefore, we are confident that CCDC81B corresponds to unknown densities 2-12.\nFor a quick comparison of DomainFit with complete AlphaFold2 model fitting, we also fitted 166 complete AlphaFold2-predicted models into the full segmented density (Fig. S4 &lt;&lt;FIG:figs4&gt;&gt;). The top hits were IJ34 and CCDC81B. Both proteins contain a CCDC81 domain (Fig. S5B &lt;&lt;FIG:figs5&gt;&gt;). BMIP1 only ranks 22 in p-value. The AlphaFold2-predicted model of CCDC81B has good tertiary structure prediction, which helps the complete model fitting. However, this shows the weakness of complete model fitting. First, complete model fitting seems to penalize smaller domains such as BMIP1. Second, doing complete model fitting does not allow to filter the sizes of the fitted domains, which is useful to eliminate false positive fits. Finally, with DomainFit, we can visualize the top hits from different densities (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt;), allowing a quicker identification of multidomain proteins without relying on the quality of the tertiary structure prediction. On the other hand, complete model fitting of 166 AlphaFold2-predicted models in the density filtered at 10 A resolution also found CCDC81B as the top hit with good fitting accuracy, thanks to good tertiary structure prediction (Fig. S5C, D &lt;&lt;FIG:figs5&gt;&gt;).\nTo further confirm that CCDC81B is a MIP found only in a subset of the doublet microtubule, we generated a T. thermophila strain with CCDC81B fused with GFP. Under superresolution structured illumination microscopy, the signal of CCDC81B was limited to the 1-1.5 um proximal region of the cilia (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). That explains the ratio of about substoichiometric fraction of the particles containing CCDC81B densities from the single-particle cryo-EM data. As a result, we can confirm that CCDC81B is a MIP that localizes to the proximal region of the cilia.\n\nDiscussion\nIn this paper, we demonstrated that it is possible to identify compact domains of proteins in maps at a resolution lower than 4 A using the DomainFit program. We benchmarked DomainFit using known structures and successfully identified CCDC81B and BMIP1 as two new proteins in the doublet microtubule of T. thermophila. Notably, we showed that CCDC81B only exists in the proximal region of the cilium using superresolution structured illumination microscopy of T. thermophila cells expressing CCDC81B fused to GFP.\nWhile DomainFit works, it is not as definite without complementary data as high-resolution cryo-EM data where side-chain information is available. We showed that the statistical analysis of the best-fitting domain works well. When the fold of the domain is unique, the identification works excellently. When the fold is not unique, DomainFit still finds the correct fold. Tools like Foldseek &lt;&lt;REF:van Kempen-2023-?-?&gt;&gt; can then be used to list all the proteins with the same fold available. With other complementary info such as chemical cross-linking mass spectrometry, BioID, and stoichiometry of proteins from quantitative mass spectrometry, it is possible to narrow down and identify the right proteins as demonstrated by this work and other integrated structural biology approach &lt;&lt;REF:Ghanaeian-2023-14-5741&gt;&gt;. There are certain cases in which the AlphaFold2-predicted model is not similar to the protein fold in the map, leading to the wrong identification. However, it does not happen often with compact domains.\nThe validation of DomainFit using known MIPs highlights a caveat of the workflow in the partitioning of domains. Breaking proteins into compact domains will never be perfect since over-partitioning will produce smaller domains and therefore increase false positive fits while under-partitioning produces bigger domains, requiring a high number of initial search positions, compromising the computational requirement. Perhaps, the development of domain partitioning with better customization in the future can improve the usability of DomainFit. Despite that caveat, our work shows that domain fitting is in general more accurate than complete model fitting since AlphaFold2 domain prediction is in general more accurate than AlphaFold2 tertiary structure prediction.\nAnother point to consider for a successful identification is the quality of the map. It seems that DomainFit works well between 4 and 6 A resolution where detailed secondary structural features can be visualized. It is recommended to use appropriate post-processing methods to improve the interpretability of the map such as DeepEmhancer &lt;&lt;REF:Sanchez-Garcia-2021-4-874&gt;&gt;, LocSpiral &lt;&lt;REF:Kaur-2021-12-1240&gt;&gt; or Density Modification &lt;&lt;REF:Terwilliger-2020-17-923&gt;&gt;.\nAt the lower resolution of 8-10 A, DomainFit can still work but requires complementary data for validation. For lower resolution, perhaps using domains with bigger partitions or even the entire AlphaFold2-predicted model might help with the fitting and identification if the tertiary structure is predicted correctly. With the flexibility of DomainFit, users can try both approaches and evaluate the fitting visually in the case of low resolutions using scripts provided by DomainFit.\nIn addition, since the program is rather quick, it is possible to use DomainFit to construct the structures of complexes with known compositions. A database of domains from all the known components can fit into different densities of the map. This allows a more unbiased way of building up the structure of the protein complex.\nIn conclusion, we presented here the DomainFit program, which allows the unbiased fitting of domains of proteins into the cryo-EM map. At resolutions better than 6 A, the program can be used reliably as a tool to identify compact proteins in the map.\n\nMaterials &amp; Methods\n\nDomain parsing\nFor domain parsing, we used the option split_model_by_compact_regions=True from phenix.process_predicted_model. In addition, we set the option maximum_domains for each protein so that the maximum_domains equal the protein size in amino acids divided by 100 amino acids.\n\np-value calculation\nThe p-value calculation for the fitting of each domain has been implemented in R &lt;&lt;REF:???-2021-?-?&gt;&gt; in the integrated modelling software Assembline &lt;&lt;REF:Rantos-2022-17-152&gt;&gt; and used for the identification of the nuclear pore complex Y-shaped scaffold &lt;&lt;REF:Bui-2013-155-1233&gt;&gt;.\nIn brief, for each domain, ChimeraX clusters the results into a number of fitting solutions with distinct rotational and translational parameters, and cross-correlation scores. As a result, many searches with different initial rotation and translation placement, resulting in similar final fitting orientation are clustered into one fitting solution. The correlation scores of all fitting solutions are transformed to z-scores using Fisher's z-transform and centered to yield an approximately normal distribution (Fig. 2F &lt;&lt;FIG:fig2&gt;&gt;).\nThen, the two-sided p-value and Benjamini-Hochberg adjusted p-value are calculated from the z-scores using the false discovery rate (fdr) package in R. The top hit for each domain which corresponds to the highest correlation score and lowest p-value is recorded in an aggregated list. Once this process is done for all the domains, the list is then sorted by p-value. When the correct domain is evaluated, the top hit's z-transformed score is clearly discriminated from the rest of the fitting solution while the difference in value is not as clear when a wrong domain is evaluated (Fig. 2F, G &lt;&lt;FIG:fig2&gt;&gt;). That translates to a lower p-value for the top hit of the correct domain.\nThe p-value is a more reliable indication of true positives than the correlation score since it is less sensitive to the size of the domain fitted in the density. Small domain models due to wrong partitioning always have high correlation scores because they fit perfectly in a small area of the density. For the correct domain, the correlation score difference between the top hit and the second hit tends to be larger except for domains with a symmetrical shape (e.g. WD40 beta-propeller). We also found that the Benjamini-Hochberg adjusted p-value functions similar to p-value.\n\nCryo-EM and image analysis\nThe single particle cryo-EM data of the 48-nm repeating unit of the doublet microtubule from T. thermophila was originally published in &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;.\nTo obtain the density with a resolution better than 12 A resolution of the subtomogram average, we constructed a focused classification mask at the region of the density based on the EMD-24376 subtomogram average map and performed 3D classification without alignment for four classes on the 48-nm particles cryo-EM dataset of T. thermophila K40R doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; using Cryosparc &lt;&lt;REF:Punjani-2017-14-290&gt;&gt; (Fig. S4A &lt;&lt;FIG:figs4&gt;&gt;). After classification, class 1 contains ~45,000 particles showing the same density as the subtomogram average (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;). This suggests that the density feature observed by subtomogram averaging is not uniformly located in the cilia. We further refined Class 1 in Cryosparc with focused local refinement using a refinement mask slightly larger than the classification mask, resulting in a cryo-EM map of 4.5 A resolution (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;, Fig. S4A, B &lt;&lt;FIG:figs4&gt;&gt;). The details in the density map suggest a 5-5.5 A resolution.\nThe final map was post-processed using DeepEMhancer &lt;&lt;REF:Sanchez-Garcia-2021-4-874&gt;&gt;.\nAll the visualization of maps and models were done in ChimeraX &lt;&lt;REF:Emsley-2010-66-486&gt;&gt;.\n\nCryo-EM density map segmentation\nTo segment the density of interest for DomainFit from the cryo-EM map, we manually placed markers onto the density of interest in ChimeraX. After that, we colored the density around the markers, and use volume splitbyzone function of ChimeraX to segment the density out and save.\n\nModelling\nFor modelling of BMIP1 (UniprotID I7MB72, TTHERM_00525130), we started with the AlphaFold2-predicted model of the globular domain of BMIP1 fitted in its density. We fixed the model manually in Coot &lt;&lt;REF:Afonine-2018-74-531&gt;&gt; and modelled some extra regions of the protein. The final models were then real-space refined in Phenix &lt;&lt;REF:Emsley-2010-66-486&gt;&gt;.\nFor the modelling of CCDC81B (UniprotID I7M688, TTHERM_00649260), best fitting positions of different domains of CCDC81B were found using DomainFit. We joined all the domains together using Coot &lt;&lt;REF:Afonine-2018-74-531&gt;&gt; and refined in Phenix &lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt;.\n\nCross-link mass spectrometry visualization\nCross-links to BMIP1 and CCDC81B were obtained from a chemical cross-linking mass spectrometry of T. thermophila cilia (Reported in Supplementary Table 1 in &lt;&lt;REF:Lagerwaard-2022-?-2022.2004.2021.489026&gt;&gt;). The crosslinks were visualized in ChimeraX using the bundle XMAS &lt;&lt;REF:Williams-1980-27-327&gt;&gt;.\n\nCell culture and gene editing\nAll Tetrahymena strains used in this study were grown in SPP media &lt;&lt;REF:Jiang-2019-15-e1008099&gt;&gt; in a shaker incubator at 30degreesC and 120 rpm.\nThe CCDC81B gene, TTHERM_00649260 gene was edited by homologous DNA recombination using a targeting plasmid carrying the neo4 selectable marker. The portions of TTHERM_00649260 required for gene targeting were amplified using primers 5F (5'- ctatagggcgaattggagctttgtgaaatagatggaagag-3') and 5R (5'- atcaagcttgccatccgcggacttgtgaatttttaaagagat-3') amplified a terminal portion of the coding region and primers 3F (5'- gcttatcgataccgtcgaccatcaattatttcaaagtattaa-3') and 3R (5'- agggaacaaaagctgggtacgcattatccaaaatatattctaa -3') amplified a portion of the 3' UTR and cloned into the pNeo24-GFP plasmid &lt;&lt;REF:Gaertig-2013-525-265&gt;&gt;. The resulting edited fragment TTHERM_00649260 was targeted to the native locus using biolistic bombardment of T. thermophila cells and paromomycin selection.\n\nImmunofluorescence\nFor immunofluorescence, T. thermophila cells were fixed and stained as described &lt;&lt;REF:Bre-1996-109-727&gt;&gt;. The primary antibodies used were the mouse monoclonal anti-polyglycylated tubulin AXO49 (diluted 1:200) &lt;&lt;REF:Bre-1996-109-727&gt;&gt; and polyclonal anti-GFP antibodies (Rockland, 1:800). The secondary antibodies used were goat-anti-mouse IgG-FITC and goat-anti-rabbit-Cy3 antibodies (Jackson Immunoresearch). SR-SIM imaging was conducted on an ELYRA S1 microscope equipped with a 63x NA 1.4 Oil Plan-Apochromat DIC objective. The optical slices were analyzed by Fiji/ImageJ (Z project tool).\n\nData Availability\nThe data generated in this study are available in the following databases: XXX (The proximal proteins from the 48-nm Tetra K40R doublet), EMD-YYY (The proximal density from the 48-nm Tetra K40R doublet). All data used but not produced in this study are available in the following databases: 8G2Z (PDB-model of the 48-nm Tetra WT doublet), EMD-29685 (48-nm Tetra WT doublet), EMD-29692 (48-nm Tetra K40R doublet).\n\nCode Availability\nThe code for DomainFit is available at https://github.com/builab/DomainFit\n\nAuthor Contributions\nConceptualisation: KHB. Methodology: MT, JGao, TL, KHB. Software: MT, JGao, KHB. Formal analysis: JGao, MT, TL, KHB. Investigation: JGao, MT, TL, CL, JG, KHB. Resources: JG, KHB. Supervision: TL, JG, KHB. Writing-original draft: TL, KHB. Writing-review &amp; editing: JGao, TL, JG, KHB.\n\nSupplementary Figures\nAll segmented densities and best-fitted domains found by DomainFit.\nBlue: correct domain; Orange: incorrect domain. The correct domains corresponding to Density 6, 10-13, 16-17, 19-22, 24) were partitioned bigger than the densities.\nApplication of domain size filtering for domain identification.\n(A) Top three hits for Density 6 fitting. While rank 1 and 2 domains fit well in Density 6, they are clearly not the right domain due to the unoccupied region. With the size of rank 2 domain (212 aa), it is possible to guess that domain occupied Density 6 should be at least 250 amino acids. (B) Plotting of the fitting statistics for all domains in Density 6 using a size filter of 250 amino acids. Purple indicates domain larger or equal 250 amino acids while light cyan indicates domains smaller than 250 amino acids and filtered out. With this size filtering, we can reduce false positive hits and identify the right domain easier. In this case, the correct domain (CFAP52) was ranked 3 in p-value before size filtering and ranked 1 in p-value after size filtering of 250 amino acids.\nStatistical analysis of fitting solutions.\n(A-D) The effect of 100 (A), 200 (B), 400 (C) and 800 (D) initial search placements on z-transformed correlation score for the correct domain. Even at a sampling of 100 initial search placements, the top hit (correct solution) still seems discriminated from the rest of the fits.\nAnalysis and reconstruction of the unknown density in the 48-nm doublet microtubule structure.\n(A) The cartoon indicates the view of the section of the B-tubule of the doublet microtubule. The classification mask is shown inside the 48-nm unit of the doublet microtubule (EMD-29692) showing the density inside is barely visible. Using Cryosparc 3D classification into four classes, Class 1 shows the density clearly inside the classification mask compared to classes 2-4. Local refinement of class 1 with the classification mask resulted in a 4.5 A resolution structure. After post-processing with DeepEmhancer, the density is segmented out of the map for protein identification by DomainFit. (B) The Fourier Shell Correlation curve of the map obtained in (A).\nComparison between domain fitting and complete model fitting.\n(A) The bump chart shows the top 5-hit proteins across 12 densities by p-value with a size filtering of 60 amino acids. Any domain ranked outside the top 4 hits is plotted at the same position in the plot (&gt; 4). There are only three proteins among the top hits (BMIP1, CCDC81B and Q23KF9). CCDC81B is consistently the top hit in many densities, helping us to rationalize the identity of the multidomain CCDC81B. (B) Top three hits for complete model fitting into the density and the right protein BMIP1 (ranked 22). Top 1 and 2 fit well to the density while I7MCW2 and BMIP1 fit poorly in the density. (C) Complete model fitting into the proximal density filtere at 10 A resolution. At this resolution, CCDC81B became the top hit. (D) Overlap of the best fit of CCDC81B into the proximal density at 4 and 10 A resolution shows that the fitting positions found are almost identical (RMSD = 1.309 A).\n\nSupplementary Table\nIdentification of MIPs in the cryo-EM map of the doublet microtubule using DomainFit.\nReason for identification of unknown densities",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nIn the last decade, cryo-electron microscopy (cryo-EM) has become a powerful technique to determine the structures of macromolecular complexes. With advances in cryo-EM image processing algorithms, large endogenous complexes have been solved at high resolutions such as the phycobilisomes in red algae &lt;&lt;REF:Ma-2020-579-146&gt;&gt;, the mitochondrial membrane bending supercomplex &lt;&lt;REF:Muhleip-2023-615-934&gt;&gt; and the doublet microtubules of the cilia &lt;&lt;REF:Ma-2019-179-909&gt;&gt;,&lt;&lt;REF:Ichikawa-2019-116-19930&gt;&gt;. Recently, in situ cryo-electron tomography (cryo-ET) from thin lamellae of cells prepared by a cryo-focused ion beam (cryo-FIB) instrument has emerged as a transformative technique, revolutionizing our understanding of cellular structures and molecular processes &lt;&lt;REF:Berger-2023-20-499&gt;&gt;. With high-throughput tilt series acquisition and improvement of subtomogram averaging software &lt;&lt;REF:Wan-2020-26-2516&gt;&gt;-&lt;&lt;REF:Zivanov-2022-11-e83724&gt;&gt;, subtomogram averaging of complexes can reach sub-nanometer resolution and in some cases better than 4 A resolution &lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;,&lt;&lt;REF:Xing-2023-381-70&gt;&gt;-&lt;&lt;REF:Tai-2023-9-116&gt;&gt;. Structures of protein complexes obtained in situ often contain unknown components, unlike in vitro reconstituted protein complexes. In addition, depending on the cellular context, space-and/or time-specific interactions with unknown proteins might be revealed by subtomogram averaging. Due to the lower resolution of these in situ structures, identification of proteins requires a different approach than traditional molecular modelling programs.\nFor high-resolution structures, there are many methods that can be used for the modelling and identification of unknown proteins including CryoID, DeeptracerID, FindMySequence, and modelAngelo &lt;&lt;REF:Jamali-2023-?-2023.2005.2016.541002&gt;&gt;-&lt;&lt;REF:Pfab-2021-118-e2017525118&gt;&gt;. These programs trace and model the backbone of the protein density. Then, the identity of the protein is predicted by comparing the side-chain densities against a database of protein sequences. This approach was successful in determining many proteins in the doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;,&lt;&lt;REF:Leung-2023-186-2880&gt;&gt;, radial spokes and central apparatus and the phycobilisomes &lt;&lt;REF:Jamali-2023-?-2023.2005.2016.541002&gt;&gt;. However, all these methods only work reliably at a resolution better than 4 A.\nIn most cases, the attainable resolution by subtomogram averaging is lower than 4 A. This is due to the low throughput of cryo-FIB milling and low abundance of the target protein complex. Furthermore, in situ structures can have problems related to orientation, sample thickness and signal-to-noise ratio. Therefore, it is a challenge to identify proteins from in situ subtomogram averages.\nIn the last few years, there has been a breakthrough in artificial intelligence (AI)-assisted protein structure prediction. Programs like AlphaFold &lt;&lt;REF:Jumper-2021-596-583&gt;&gt;, ColabFold &lt;&lt;REF:Mirdita-2021-?-2021.2008.2015.456425&gt;&gt; and RoseTTAfold &lt;&lt;REF:Baek-2021-373-871&gt;&gt; can produce accurate structure predictions. With the establishment of the AlphaFold Protein Structure Database (AlphaFold DB), everyone now has access to over 200 million predicted models. For certain organisms such as humans, mice, zebrafish and nematodes, AlphaFold2-predicted models of the entire proteome are available.\nWith the good accuracy of the AI-predicted models, it is possible to identify the fold signature of a protein at a resolution of less than 4 A by fitting the AI-predicted models into the density. Using complementary data such as in situ chemical cross-linking mass spectrometry, quantitative mass spectrometry and/or BioID, it is possible to identify proteins from the fit list. For example, microtubule inner proteins (MIPs) were identified from the in situ subtomogram average of human sperm doublet microtubules at ~6-7 A using Colores Situs program by fitting over 21,000 AlphaFold2-predicted protein models of the mouse proteome &lt;&lt;REF:Chen-2023-186-5041&gt;&gt;. In another study, 38 proteins were identified from the mouse sperm by manually fitting AlphaFold2-predicted models of proteins found in the mass spectrometry analysis of the same sample &lt;&lt;REF:Tai-2023-9-116&gt;&gt;.\nWhile those studies illustrate that it is possible to identify well-structured proteins from a map lower than 4 A resolution, the methods are not automated or are difficult to use. In addition, the criteria to evaluate and identify proteins from the fitting solutions are not clear from these studies. While the accuracy of the AI-predicted model is high for compact domains, the tertiary structure of the predicted model might not be correct, due in part to poor prediction of flexible regions. Therefore, protein identification is easier when domains are extracted from each AI-predicted model and used instead of the entire model.\nIn this work, we developed a program called DomainFit to identify domains of proteins from cryo-EM maps of lower resolution than 4 A. DomainFit uses popular programs such as Phenix &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt;, R &lt;&lt;REF:???-2021-?-?&gt;&gt; and UCSF ChimeraX (ChimeraX) &lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;, making it easy to install and accessible to people. In addition, DomainFit is flexible and has a clear statistical approach to evaluate the identity of proteins. Lastly, DomainFit uses ChimeraX, which allows easy visualization to aid in evaluating the fit. Our workflow will be useful for the upcoming wave of in situ structures as cryo-FIB and cryo-ET become more popular.\n\nResults\n\nAn automated pipeline for domain parsing and fitting into cryo-EM map\nOverall, the program is designed to first download a database of AI-predicted models of candidate proteins (i.e. full species proteome or a limited list based on mass spectrometry studies), then divide each model into different domains and fit each individual domain into a segmented cryo-EM density. Finally, the program generates a statistical evaluation of all the fitted domains (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). The identification of the correct domain can be based purely on statistical analysis but also on other complementary data such as surrounding densities, protein size, chemical cross-linking and quantitative mass spectrometry.\nFigure 1:\nWorkflow of DomainFit.\n(A) Overview of the program. (i) Download the predicted PDBs for all candidate proteins. (ii) Predict domains and split them into single PDB files. This step can be done through the script process_predicted_models.py or results from other programs such as DPAM. (iii) Fit each domain into the unknown density using ChimeraX nd generate the best fitting position of each domain and fitting scores. (iv) Interpret the fitting results based on p-value, correlation, and size of proteins.\nThe list of candidate proteins can be obtained from the mass spectrometry results of the target sample such as the proteome of isolated organelles or pull-down analysis. In case a targeted list is not available, it is possible to use AI-predicted models of the entire proteome of the organism, even though it will increase computational time significantly. In DomainFit, we provide a script getAlphaFoldPDBs.py to automatically download the AlphaFold2-predicted models from the AlphaFold DB using a list of protein UniProtIDs (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). The downside of downloading automatically from the AlphaFold DB is that for many organisms, high molecular weight proteins (&gt; 100kDa) are not yet available in the database. These missing proteins can be predicted using ColabFold, RoseTTAFold or a local instance of AlphaFold.\nMany ideas have been proposed to break PDB models into compact domains automatically &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt;,&lt;&lt;REF:Croll-2022-?-?&gt;&gt;,&lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt;. With AI-predicted models, it is possible to partition into domains based on the predicted alignment errors (PAE) of each model (Fig. 2A-D &lt;&lt;FIG:fig2&gt;&gt;). PAE is the expected positional error for residue x if the predicted and actual models are aligned at residue y &lt;&lt;REF:Jumper-2021-596-583&gt;&gt;. Using an image-based approach, it is possible to partition predicted models into domains based on their respective PAE files (Fig. 2B-D &lt;&lt;FIG:fig2&gt;&gt;) &lt;&lt;REF:Croll-2022-?-?&gt;&gt;. We used phenix.process_predicted_model program from the Phenix package &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt; for the partitioning of the PDB into domains because of its flexibility. phenix.process_predicted_model can function based on PAE but also based on the 3D arrangement of the proteins without PAE information. Therefore, we implemented a Python wrapper process_predicted_models.py to partition PDBs into domains in batch (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). Users can customize the options for parsing based on their needs.\nFigure 2.\nDomain partitioning and fitting evaluation in DomainFit.\n(A) An example of an AlphaFold2-predicted model of a multi-domain protein (RIB72B, UniprotID I7MCU1). (B) The PAE plot of the RIB72B. (C) Domain partitioning of RIB72B based on the PAE plot. (D) Domain partitioning of RIB72B corresponding to (C). (E) Fitting of a domain of RIB72B inside its corresponding segmented density at 4 A resolution. (F) Histogram of the z-transformed correlation score of all obtained fits of the correct domain of RIB72B. The top hit (red) is separated from the score distribution. (G) Histogram of the z-transformed correlation score of a wrong domain into the same density in (F). (H) The scatter plot to visualize the value of - log10 of Benjamini Hochberg adjusted p-value versus the normalize cross correlation coefficient of each domain fitting into the density of interest. The top right corner two points are well separate for the rest of the point cloud, indicating likely well-matched domains into the density.\nTo increase the flexibility of our program, process_predicted_models.py writes out the information of the predicted domains using the format employed by DPAM &lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt;. As a result, we can use domain information data from either phenix.process_predicted_model or existing domain data of certain organisms such as human, mouse, and zebrafish from DPAM &lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt; to generate the PDBs of individual domains. In the case of a small number of target proteins, we can even edit the domain information files manually to generate customized domain parsing.\nAt this step, we recommended imposing a filter on the minimum and maximum domain sizes for fitting. At the lower end, 40 amino acids is the minimum size that allows us to reduce the number of domains used without affecting the results since domains under 40 amino acids tend to be either wrongly partitioned or not compact. On the upper limit, we recommend using a bigger value like 1,000 amino acids as a fail-safe in the case that domain parsing does not work properly.\nTo prepare for the fitting of domains, a volume containing a compact density should be segmented from the cryo-EM map. The compact density means either globular or coiled-coil bundle. A long helical density does not work since its shape is not unique for identification of protein fold. Ideally, the segmented density should be equivalent to a domains of more than 50-300 amino acids in size. This serves two purposes: (i) to reduce computational time, and (ii) to avoid the wrongly predicted tertiary conformation caused by long flexible loop between different domains.\nOnce the segmented density is prepared, fit_domains_in_chimerax.py is used to fit all the individual domain models into the density to find the best fitting position of each domain (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;, Fig. 2E &lt;&lt;FIG:fig2&gt;&gt;). The script uses the ChimeraX fitmap command to fit each domain in the density from a fixed number of initial random translated and rotated search positions within the density defined as \"initial search placements\". Instead of exhaustive six dimensional search like Colores package in Situs, for global search, ChimeraX fitmap places the model randomly in rotation and translation within the map, and performs rigid-body local optimization from each initial placement. After the local optimization, ChimeraX will group the fitting solutions of based on similarity in orientation. Therefore, the number of fitting solutions tend to be smaller than the number of initial search positions due to clustering. These fitting solutions's correlation score are then z-transformed and then used for the calculation of the p-value, the likelihood that the top hit is correct and clearly significant compared to the rest of the fitting solutions (See Materials &amp; Methods, Fig. 2F &lt;&lt;FIG:fig2&gt;&gt;). Once the fitting is done for every domain in the density, we expect that top hits with the correct fold/domain are clearly distinguished from other fitting solutions by their z-transformed scores, whereas those with the incorrect domain are not so differentiated (Fig. 2G &lt;&lt;FIG:fig2&gt;&gt;). Visualizing the fitting statistics i.e. p-values and cross correlation coefficient of all domains allow a quick way to see how distinct is the top hits (Fig. 2H &lt;&lt;FIG:fig2&gt;&gt;).\nA comma-separated value (CSV) file containing the best fitting position of each domains is written at the end, sorted first by p-value and then by the correlation score for quick evaluation.\n\nDomainFit benchmarking for MIP identification\nTo assess the efficiency of DomainFit, we tested it on segmented densities from the single-particle cryo-EM map of the doublet microtubule from Tetrahymena thermophila (EMD-29685) at ~4 A resolution. More than 40 MIPs were identified and modelled using an AI-assisted modelling approach (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt;). Even though the resolution of the doublet microtubule cryo-EM map was high, modelling and identifying proteins was laborious and time-consuming. Our goal for this test was to see whether we could identify the MIPs using DomainFit independently.\nFigure 3:\nIdentification of MIPs in the T. thermophila doublet microtubule.\n(A) The doublet microtubule structure with MIPs (PDB: 8g2z). (B) Densities (red) in the cryo-EM map of the doublet microtubule (EMD-29685) for testing with DomainFit. (C) A histogram of domain size. Red bars indicate domains of less than 40 amino acids and are excluded from fitting. The x-axis is limited to a domain size of 800 amino acids. (D) DomainFit ranking by p-value and correlation about mean of the correct domain for 24 densities. (E) Example of the perfect match between the AlphaFold2-predicted model of PACRGB fitted within the corresponding density. (F) The domain of RIB22 is not partitioned as expected. However, the bigger domain was still fitted correctly into the density due to good agreement between the AlphaFold2-predicted model and the density. Due to incorrect parsing and the fact that the EF-hand domain is abundant in the cilium, the fit of the correct domain is ranked 9 but the p-value is still ranked 1st. (G) When the fold of the density is common (EF-hand domain RIB571-91 - Density 10), the top hits consist of domains with a similar fold. The top five hits are all domains from CFAP115, which have the same EF-hand fold as the correct domain RIB571-91. (H) The experimentally determined model of RIB57 (PDB: 8g2z) is shown inside its corresponding density. (I) There is a big discrepancy between the experimentally determined model (cyan) and AlphaFold2-predicted model (purple) of RIB571-91 (RMSD: 5.164 *), which leads to the poor fitting of RIB57.\nWe segmented 24 densities from 13 identified MIPs in the cryo-EM map corresponding to different domains of MIPs (Supplementary Table 1, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We established a database comprising 856 AlphaFold2-predicted models based on the mass spectrometry data of the WT cilia without membrane (994 detected proteins) &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. One hundred and thirty eight proteins were not available from the AlphaFold DB because of a lack of predictions due to their high molecular weight. For benchmarking purpose, we ignored them for this test since the 856 AlphaFold2-predicted models covered all the tested MIPs.\nUsing process_predicted_models.py and save_domains_from_info.py, we divided 856 AlphaFold2-predicted models into 2819 domains. We removed domains of less than 40 amino acids which left us with 1561 domains for fitting (Fig. 3C &lt;&lt;FIG:fig3&gt;&gt;). We then fitted all 1561 domains into each density filtered at 4 A using 200 initial search position placements. On average, each density search took ~30 minutes using 12 cores (20 threads) on an i7-12700K (12 core-CPU and 32GB RAM).\nOut of all the densities tested, we found 22 out of 24 correct domains are present in the top 5 hits ranked by descending p-value (Supplementary Table 1, Fig. 3D &lt;&lt;FIG:fig3&gt;&gt;, Fig. S1 &lt;&lt;FIG:figs1&gt;&gt;). Overall, we found the p-value to be a better predictor of correct identification and fitting than the normalized cross correlation coefficient, which is called correlation about mean in ChimeraX. Often, the domains were correctly found as the top hits with the highest correlation and best p-value such as PACRGB, an inner junction protein &lt;&lt;REF:Khalifa-2020-9-e52760&gt;&gt; (Fig. 3E &lt;&lt;FIG:fig3&gt;&gt;). Our results show that the p-value ranks consistently higher than the correlation score for the correct domains for the densities (Fig. 3D &lt;&lt;FIG:fig3&gt;&gt;). In the cases where domains have a common fold, multiple domains returned a p-value of 2.22*10-16 (limit of R program). Thus, we sorted the fitting solutions first based on p-values, and then by their normalized cross correlation coefficient. As a result, hereafter, top hits will refer to the top p-value ranked hits.\nIn some cases such as Density 6 (CFAP5213-317), by looking at the fit of the domains in the density, we can roughly estimate the size of the domain and establish a minimum domain size threshold, which allows quicker identification of the correct domain (Fig. S2A-B &lt;&lt;FIG:figs2&gt;&gt;).\nThere are quite common cases where the partitioned domain is bigger than the segmented density (Supplementary Table 1, Fig. S1 &lt;&lt;FIG:figs1&gt;&gt;, Density 6, 10-13, 16-17, 19-22, 24). This might affect the fitting of DomainFit. For example, our parameters for domain parsing tend to keep entire EF-hand domain pairs as a single domain, instead of two separate EF-hand domains. For RIB22, while the density represents only RIB221-91, RIB224-191 was partitioned as one domain (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt;). On the other hand, the correct fitting was found for RIB224-191 with a p-value and correlation value of rank 1st (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt;, Supplementary Table 1).\nInterestingly, for Density 10, corresponding to the EF-hand domain RIB571-91, DomainFit failed to find the correct domain (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt;). While all the top hits were the wrong protein, they all had the correct EF-hand fold (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt;). when we compared the experimentally determined model of RIB57 (Fig. 3H &lt;&lt;FIG:fig3&gt;&gt;) and the AlphaFold2-predicted model, there was a big difference between the two models from residues 6 to 91 (RMSD 5.164 A, Fig. 3I &lt;&lt;FIG:fig3&gt;&gt;). Therefore, the failure to find RIB571-91 density (Density 10) was due to both the difference in the AlphaFold2-predicted model and the unsuccessful parsing of RIB571-91 domain.\nWe conclude that DomainFit works well but attention needs to be paid to potential mis-identification of proteins due to their size and domain parsing. Also, running with a higher number of initial search placements is a safeguard against inaccuracy in domain parsing to accommodate more initial translated and rotated positions.\n\nPerformance of DomainFit depending on the number of initial placements and resolution\nOur test results showed that DomainFit is effective in identifying compact densities and the p-value is the most reliable indicator of the correct hit. For p-value to be estimated properly, a reasonable number of data points, i.e. fitting solutions are needed. Therefore, we wanted to explore the sensitivity of the p-value estimation to the number of initial search placements. We ran DomainFit for Density 1 100 times for each initial search placement value ranging from 1 to 300 and plotted the average p-value found. Our analysis shows that above 150 initial search placements, the resulting p-value is consistently found to be as high a value as it can be (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Thus, setting a search placement value of 150 should yield optimal results for the speed and accuracy of the fits in this setup. This setting is appropriate for a domain of 100-150 amino acids while larger domains may require a higher number of initial search placements. We also looked at the z-transform correlation score for the correct domain at different numbers of initial search placements (Fig. S3A-D &lt;&lt;FIG:figs3&gt;&gt;). The statistical calculation is certainly less reliable when the number of fitting points is small. For the correct domain, if the correlation score of the top hit is distinct from that of the rest, a good p-value can still be obtained.\nFigure 4:\nPerformance of DomainFit at different numbers of initial search placements and resolutions.\n(A) -log10 of the average p-value at increasing initial search placements for a correct and an unrelated domain. Standard deviations are shown in coloured regions. (B) Appearance changes of a density filtering at 4, 6, 8 and 10 A resolution. (C) p-value rank of the correct domains fitted to different densities of MIPs at 4, 6, 8, and 10 A resolution. (D) Number of fitting solutions versus number of initial search placements for maps at 4 A (blue), 6 A (red), 8 A (green) and 10 A (yellow) resolution. (E) A histogram of the z-transformed correlation score of a correct domain fitted to a density at 4 A resolution shows a clear separation between the top hit and the rest of the fitting solutions. (F) Histograms of z-transformed correlation score of the same domain fitted to the densities filtered at 6, 8 and 10 A resolutions. There are significantly fewer fitting solutions at lower resolutions and the separation of the top hit, and the rest is reduced.\nThe number of fitting solutions found does not depend only on the number of initial search placements but also on the resolution of the map. Next, we wanted to check the effect of map resolutions on the success of DomainFit. To test whether the program could successfully identify domains at lower resolutions, we filtered and ran the density maps of the MIPs at 4, 6, 8, and 10 * (Fig. 4B &lt;&lt;FIG:fig4&gt;&gt;). We found that the search consistently identified the correct domain within the top 10 results at 4 to 8 * resolution (Fig. 4C &lt;&lt;FIG:fig4&gt;&gt;). At lower resolutions, there are significantly fewer fitting solutions found even with the same number of initial search placements (Fig. 4D &lt;&lt;FIG:fig4&gt;&gt;). This is because of the smoothening of the map at lower resolutions, which does not allow many unique fitting solutions. Because of this, ChimeraX produces a small number of clusters of fitting solutions from a large number of initial search placements. This leads to less confidence in statistical analysis of the top hits. If we look at the z-transformed correlation score at 4, 6, 8 and 10 A of the correct domain (Fig. 4E, F &lt;&lt;FIG:fig4&gt;&gt;), we can see that at 6 A, the distribution of the fitting solution and the top hits still maintains its shape, allowing reliable p-value calculation. At 8 A, while the top hit is still well separated from the distribution, the rest of the solution does not look normally distributed. At 10 A, not only does the distribution of the correlation score is not normally distributed but the difference between the top hit and the rest gets smaller.\nAs a result, DomainFit seems to work best in the 4-to-6 A resolution range. At 8 A resolution, the number of initial search placements must be increased to help with statistical calculation. At lower than 8 A, DomainFit should only be used for finding types of folds that fit well in the density by visual inspection of the top hits but it should not be used for domain identification.\n\nNew MIP identified in the doublet microtubule\nWith our success in testing and validating the DomainFit program, we attempted to identify unknown proteins from cryo-EM maps with a resolution lower than 4 A. An unidentified MIP was previously reported in the subtomogram average of the T. thermophila doublet microtubule &lt;&lt;REF:Li-2022-5-e202101225&gt;&gt; (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt;). However, this density was not observed in the single particle cryo-EM map of the doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; (Fig. 5B &lt;&lt;FIG:fig5&gt;&gt;) and at too low resolution to be identified.\nFigure 5:\nNew proteins identified from the doublet microtubule of T. thermophila.\n(A) Overview of the unknown density (yellow) from the 48-nm repeating unit subtomogram average map of doublet microtubule (EMD-24376) &lt;&lt;REF:Li-2022-5-e202101225&gt;&gt;. (B) The same density is not visible in the same view of the single particle cryo-EM map of the 48-nm repeating unit of the doublet microtubule from the T. thermophila K40R map (EMD-29692) &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. (C) The unknown density exists in the doublet microtubule map after 3D classification. (D) Segmentation of the unknown density into different small unknown densities for DomainFit. (E) DomainFit top hits for each unknown density using an AlphaFold database of salt-treated ciliome. (F) Corrected model of protein domains after examining, showing BMIP1 (UniprotID I7MB72) is density 1 and CCDC81B (UniprotID I7M688) forms density 2-12. (G) Fit of AlphaFold2-predicted model BMIP112-168 into density 1. (H) Fit of AlphaFold2-predicted model CCDC81B38-218 into density 2.\nWe re-processed the cryo-EM dataset containing the 48-nm particles of T. thermophila K40R doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; with Cryosparc &lt;&lt;REF:Punjani-2017-14-290&gt;&gt; Using 3D classification and refinement, we obtained a 4.5 A resolution cryo-EM map with the same density as the subtomogram average (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;, Fig. S4A, B &lt;&lt;FIG:figs4&gt;&gt;).\nTo identify the proteins in the unknown density, we segmented the density into 12 smaller densities (Fig. 5D &lt;&lt;FIG:fig5&gt;&gt;). To improve the accuracy of the search, we used the ciliome of the salt-treated doublet microtubules &lt;&lt;REF:Ichikawa-2019-116-19930&gt;&gt;. The reason is that the cryo-EM map of salt-treated doublet microtubules retains most of the MIPs and the unknown density &lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;. The salt-treated ciliome of 166 proteins was divided into 334 domains of at least 40 amino acids. We ran DomainFit with these 334 domain models for the above 12 densities.\nInterestingly, the top hits for the 12 densities came from only three proteins (Fig. 5E &lt;&lt;FIG:fig5&gt;&gt;, Supplementary Table 2). Upon inspecting the results, we concluded that the 12 densities are composed of only two proteins I7MB72 (TTHERM_00525130) and I7M688 (TTHERM_00649260) because of the overall architecture (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;, Fig. S5A &lt;&lt;FIG:figs5&gt;&gt;) and the fit of unique domains in both proteins (Fig. 5G-H &lt;&lt;FIG:fig5&gt;&gt;).\nI7MB72 matches unknown density 1 with high confidence in p-value and correlation (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;), referring as BMIP1 from here onwards. Upon inspection of the map, density 1 was always present in the doublet microtubule (EMD-29692) but was not identified previously. As a result, we could model extra regions of the protein BMIP1 in the T. thermophila K40R map (EMD-29692) at 3.5 A resolution, confirming the identity of unknown density 1 and the success of DomainFit (Fig. 6A, B &lt;&lt;FIG:fig6&gt;&gt;). In addition, BMIP1 is abundant in both salt-treated and native doublet microtubules similar to the profiles of most MIPs. BMIP1 has a cross-link to CFAP45, which exists as two copies between protofilaments B8B9 and B7B8, as shown in studies that usedchemical cross-linking coupled with mass spectrometry of cilia &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;,&lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt; (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). While the cross-linked residues (Lysines 213 and 222) from BMIP1 are not modelled, they seem to be within the cross-link distance (&lt; 35 A) from lysines 403 and 283 of CFAP45.\nFigure 6:\nValidation of the newly identified MIPs.\nA. Model of BMIP1 (UniprotID I7MB72) using Coot. B. Side-chain densities of a helix from BMIP1. C. Intra-cross-links (yellow) within CCDC81B and inter-cross-links (orange) between CCDC81B and CFAP106, a known MIP protein &lt;&lt;REF:Khalifa-2020-9-e52760&gt;&gt;. D. Merged super resolution-structured illumination microscopy image of T. thermophila cells with CCDC81B-GFP shows that CCDC81B only localizes to the proximal region of the cilia. Rectangle indicates the zoom-in view in (E). Red: CCDC81B-GFP signal; Green: doublet microtubule labelled by polyG-Antibody; Blue: Hoechst dye labelling the nucleus. E. The zoom-in merged image (top) and GFP channel (bottom) showing localization of CCDC81B to the proximal region.\nDensities 2-12 (Fig. 5D &lt;&lt;FIG:fig5&gt;&gt;) belong to I7M688, a CCDC81-domain containing protein. We named I7M688 as CCDC81B from here onwards. The CCDC81 domain of CCDC81B is unambiguously fitted into unknown density 2 (Fig. 5H &lt;&lt;FIG:fig5&gt;&gt;). There are a few CCDC81-containing proteins in the cilia of T. thermophila: IJ34 (UniprotID I7M9T0), CCDC81A (UniprotID I7MLF6), CCDC81B (UniprotID I7M688), Q240Y1 and Q22HG4. IJ34 is a MIP near the inner junction &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; and CCDC81A was identified as a ciliary tip protein &lt;&lt;REF:Legal-2023-222-?&gt;&gt;. Q240Y1 and Q22HG4 are clear paralogs of CCDC81B with similar architecture but are washed away in salt-treated cilia &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;.\nMass spectrometry results show that CCDC81B is resistant to salt and is similar to a MIP profile &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. In situ cross-linking mass spectrometry shows that CCDC81B is crosslinked to tubulin alpha, CFAP106, FAP210, and interestingly RIB43A &lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt;. All these cross-links are satisfied relative to the CCDC81B position in the unknown density, except for RIB43A (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). Therefore, we are confident that CCDC81B corresponds to unknown densities 2-12.\nFor a quick comparison of DomainFit with complete AlphaFold2 model fitting, we also fitted 166 complete AlphaFold2-predicted models into the full segmented density (Fig. S4 &lt;&lt;FIG:figs4&gt;&gt;). The top hits were IJ34 and CCDC81B. Both proteins contain a CCDC81 domain (Fig. S5B &lt;&lt;FIG:figs5&gt;&gt;). BMIP1 only ranks 22 in p-value. The AlphaFold2-predicted model of CCDC81B has good tertiary structure prediction, which helps the complete model fitting. However, this shows the weakness of complete model fitting. First, complete model fitting seems to penalize smaller domains such as BMIP1. Second, doing complete model fitting does not allow to filter the sizes of the fitted domains, which is useful to eliminate false positive fits. Finally, with DomainFit, we can visualize the top hits from different densities (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt;), allowing a quicker identification of multidomain proteins without relying on the quality of the tertiary structure prediction. On the other hand, complete model fitting of 166 AlphaFold2-predicted models in the density filtered at 10 A resolution also found CCDC81B as the top hit with good fitting accuracy, thanks to good tertiary structure prediction (Fig. S5C, D &lt;&lt;FIG:figs5&gt;&gt;).\nTo further confirm that CCDC81B is a MIP found only in a subset of the doublet microtubule, we generated a T. thermophila strain with CCDC81B fused with GFP. Under superresolution structured illumination microscopy, the signal of CCDC81B was limited to the 1-1.5 um proximal region of the cilia (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). That explains the ratio of about substoichiometric fraction of the particles containing CCDC81B densities from the single-particle cryo-EM data. As a result, we can confirm that CCDC81B is a MIP that localizes to the proximal region of the cilia.\n\nDiscussion\nIn this paper, we demonstrated that it is possible to identify compact domains of proteins in maps at a resolution lower than 4 A using the DomainFit program. We benchmarked DomainFit using known structures and successfully identified CCDC81B and BMIP1 as two new proteins in the doublet microtubule of T. thermophila. Notably, we showed that CCDC81B only exists in the proximal region of the cilium using superresolution structured illumination microscopy of T. thermophila cells expressing CCDC81B fused to GFP.\nWhile DomainFit works, it is not as definite without complementary data as high-resolution cryo-EM data where side-chain information is available. We showed that the statistical analysis of the best-fitting domain works well. When the fold of the domain is unique, the identification works excellently. When the fold is not unique, DomainFit still finds the correct fold. Tools like Foldseek &lt;&lt;REF:van Kempen-2023-?-?&gt;&gt; can then be used to list all the proteins with the same fold available. With other complementary info such as chemical cross-linking mass spectrometry, BioID, and stoichiometry of proteins from quantitative mass spectrometry, it is possible to narrow down and identify the right proteins as demonstrated by this work and other integrated structural biology approach &lt;&lt;REF:Ghanaeian-2023-14-5741&gt;&gt;. There are certain cases in which the AlphaFold2-predicted model is not similar to the protein fold in the map, leading to the wrong identification. However, it does not happen often with compact domains.\nThe validation of DomainFit using known MIPs highlights a caveat of the workflow in the partitioning of domains. Breaking proteins into compact domains will never be perfect since over-partitioning will produce smaller domains and therefore increase false positive fits while under-partitioning produces bigger domains, requiring a high number of initial search positions, compromising the computational requirement. Perhaps, the development of domain partitioning with better customization in the future can improve the usability of DomainFit. Despite that caveat, our work shows that domain fitting is in general more accurate than complete model fitting since AlphaFold2 domain prediction is in general more accurate than AlphaFold2 tertiary structure prediction.\nAnother point to consider for a successful identification is the quality of the map. It seems that DomainFit works well between 4 and 6 A resolution where detailed secondary structural features can be visualized. It is recommended to use appropriate post-processing methods to improve the interpretability of the map such as DeepEmhancer &lt;&lt;REF:Sanchez-Garcia-2021-4-874&gt;&gt;, LocSpiral &lt;&lt;REF:Kaur-2021-12-1240&gt;&gt; or Density Modification &lt;&lt;REF:Terwilliger-2020-17-923&gt;&gt;.\nAt the lower resolution of 8-10 A, DomainFit can still work but requires complementary data for validation. For lower resolution, perhaps using domains with bigger partitions or even the entire AlphaFold2-predicted model might help with the fitting and identification if the tertiary structure is predicted correctly. With the flexibility of DomainFit, users can try both approaches and evaluate the fitting visually in the case of low resolutions using scripts provided by DomainFit.\nIn addition, since the program is rather quick, it is possible to use DomainFit to construct the structures of complexes with known compositions. A database of domains from all the known components can fit into different densities of the map. This allows a more unbiased way of building up the structure of the protein complex.\nIn conclusion, we presented here the DomainFit program, which allows the unbiased fitting of domains of proteins into the cryo-EM map. At resolutions better than 6 A, the program can be used reliably as a tool to identify compact proteins in the map.\n\nMaterials &amp; Methods\n\nDomain parsing\nFor domain parsing, we used the option split_model_by_compact_regions=True from phenix.process_predicted_model. In addition, we set the option maximum_domains for each protein so that the maximum_domains equal the protein size in amino acids divided by 100 amino acids.\n\np-value calculation\nThe p-value calculation for the fitting of each domain has been implemented in R &lt;&lt;REF:???-2021-?-?&gt;&gt; in the integrated modelling software Assembline &lt;&lt;REF:Rantos-2022-17-152&gt;&gt; and used for the identification of the nuclear pore complex Y-shaped scaffold &lt;&lt;REF:Bui-2013-155-1233&gt;&gt;.\nIn brief, for each domain, ChimeraX clusters the results into a number of fitting solutions with distinct rotational and translational parameters, and cross-correlation scores. As a result, many searches with different initial rotation and translation placement, resulting in similar final fitting orientation are clustered into one fitting solution. The correlation scores of all fitting solutions are transformed to z-scores using Fisher's z-transform and centered to yield an approximately normal distribution (Fig. 2F &lt;&lt;FIG:fig2&gt;&gt;).\nThen, the two-sided p-value and Benjamini-Hochberg adjusted p-value are calculated from the z-scores using the false discovery rate (fdr) package in R. The top hit for each domain which corresponds to the highest correlation score and lowest p-value is recorded in an aggregated list. Once this process is done for all the domains, the list is then sorted by p-value. When the correct domain is evaluated, the top hit's z-transformed score is clearly discriminated from the rest of the fitting solution while the difference in value is not as clear when a wrong domain is evaluated (Fig. 2F, G &lt;&lt;FIG:fig2&gt;&gt;). That translates to a lower p-value for the top hit of the correct domain.\nThe p-value is a more reliable indication of true positives than the correlation score since it is less sensitive to the size of the domain fitted in the density. Small domain models due to wrong partitioning always have high correlation scores because they fit perfectly in a small area of the density. For the correct domain, the correlation score difference between the top hit and the second hit tends to be larger except for domains with a symmetrical shape (e.g. WD40 beta-propeller). We also found that the Benjamini-Hochberg adjusted p-value functions similar to p-value.\n\nCryo-EM and image analysis\nThe single particle cryo-EM data of the 48-nm repeating unit of the doublet microtubule from T. thermophila was originally published in &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;.\nTo obtain the density with a resolution better than 12 A resolution of the subtomogram average, we constructed a focused classification mask at the region of the density based on the EMD-24376 subtomogram average map and performed 3D classification without alignment for four classes on the 48-nm particles cryo-EM dataset of T. thermophila K40R doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; using Cryosparc &lt;&lt;REF:Punjani-2017-14-290&gt;&gt; (Fig. S4A &lt;&lt;FIG:figs4&gt;&gt;). After classification, class 1 contains ~45,000 particles showing the same density as the subtomogram average (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;). This suggests that the density feature observed by subtomogram averaging is not uniformly located in the cilia. We further refined Class 1 in Cryosparc with focused local refinement using a refinement mask slightly larger than the classification mask, resulting in a cryo-EM map of 4.5 A resolution (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;, Fig. S4A, B &lt;&lt;FIG:figs4&gt;&gt;). The details in the density map suggest a 5-5.5 A resolution.\nThe final map was post-processed using DeepEMhancer &lt;&lt;REF:Sanchez-Garcia-2021-4-874&gt;&gt;.\nAll the visualization of maps and models were done in ChimeraX &lt;&lt;REF:Emsley-2010-66-486&gt;&gt;.\n\nCryo-EM density map segmentation\nTo segment the density of interest for DomainFit from the cryo-EM map, we manually placed markers onto the density of interest in ChimeraX. After that, we colored the density around the markers, and use volume splitbyzone function of ChimeraX to segment the density out and save.\n\nModelling\nFor modelling of BMIP1 (UniprotID I7MB72, TTHERM_00525130), we started with the AlphaFold2-predicted model of the globular domain of BMIP1 fitted in its density. We fixed the model manually in Coot &lt;&lt;REF:Afonine-2018-74-531&gt;&gt; and modelled some extra regions of the protein. The final models were then real-space refined in Phenix &lt;&lt;REF:Emsley-2010-66-486&gt;&gt;.\nFor the modelling of CCDC81B (UniprotID I7M688, TTHERM_00649260), best fitting positions of different domains of CCDC81B were found using DomainFit. We joined all the domains together using Coot &lt;&lt;REF:Afonine-2018-74-531&gt;&gt; and refined in Phenix &lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt;.\n\nCross-link mass spectrometry visualization\nCross-links to BMIP1 and CCDC81B were obtained from a chemical cross-linking mass spectrometry of T. thermophila cilia (Reported in Supplementary Table 1 in &lt;&lt;REF:Lagerwaard-2022-?-2022.2004.2021.489026&gt;&gt;). The crosslinks were visualized in ChimeraX using the bundle XMAS &lt;&lt;REF:Williams-1980-27-327&gt;&gt;.\n\nCell culture and gene editing\nAll Tetrahymena strains used in this study were grown in SPP media &lt;&lt;REF:Jiang-2019-15-e1008099&gt;&gt; in a shaker incubator at 30degreesC and 120 rpm.\nThe CCDC81B gene, TTHERM_00649260 gene was edited by homologous DNA recombination using a targeting plasmid carrying the neo4 selectable marker. The portions of TTHERM_00649260 required for gene targeting were amplified using primers 5F (5'- ctatagggcgaattggagctttgtgaaatagatggaagag-3') and 5R (5'- atcaagcttgccatccgcggacttgtgaatttttaaagagat-3') amplified a terminal portion of the coding region and primers 3F (5'- gcttatcgataccgtcgaccatcaattatttcaaagtattaa-3') and 3R (5'- agggaacaaaagctgggtacgcattatccaaaatatattctaa -3') amplified a portion of the 3' UTR and cloned into the pNeo24-GFP plasmid &lt;&lt;REF:Gaertig-2013-525-265&gt;&gt;. The resulting edited fragment TTHERM_00649260 was targeted to the native locus using biolistic bombardment of T. thermophila cells and paromomycin selection.\n\nImmunofluorescence\nFor immunofluorescence, T. thermophila cells were fixed and stained as described &lt;&lt;REF:Bre-1996-109-727&gt;&gt;. The primary antibodies used were the mouse monoclonal anti-polyglycylated tubulin AXO49 (diluted 1:200) &lt;&lt;REF:Bre-1996-109-727&gt;&gt; and polyclonal anti-GFP antibodies (Rockland, 1:800). The secondary antibodies used were goat-anti-mouse IgG-FITC and goat-anti-rabbit-Cy3 antibodies (Jackson Immunoresearch). SR-SIM imaging was conducted on an ELYRA S1 microscope equipped with a 63x NA 1.4 Oil Plan-Apochromat DIC objective. The optical slices were analyzed by Fiji/ImageJ (Z project tool).\n\nData Availability\nThe data generated in this study are available in the following databases: XXX (The proximal proteins from the 48-nm Tetra K40R doublet), EMD-YYY (The proximal density from the 48-nm Tetra K40R doublet). All data used but not produced in this study are available in the following databases: 8G2Z (PDB-model of the 48-nm Tetra WT doublet), EMD-29685 (48-nm Tetra WT doublet), EMD-29692 (48-nm Tetra K40R doublet).\n\nCode Availability\nThe code for DomainFit is available at https://github.com/builab/DomainFit\n\nAuthor Contributions\nConceptualisation: KHB. Methodology: MT, JGao, TL, KHB. Software: MT, JGao, KHB. Formal analysis: JGao, MT, TL, KHB. Investigation: JGao, MT, TL, CL, JG, KHB. Resources: JG, KHB. Supervision: TL, JG, KHB. Writing-original draft: TL, KHB. Writing-review &amp; editing: JGao, TL, JG, KHB.\n\nSupplementary Figures\nAll segmented densities and best-fitted domains found by DomainFit.\nBlue: correct domain; Orange: incorrect domain. The correct domains corresponding to Density 6, 10-13, 16-17, 19-22, 24) were partitioned bigger than the densities.\nApplication of domain size filtering for domain identification.\n(A) Top three hits for Density 6 fitting. While rank 1 and 2 domains fit well in Density 6, they are clearly not the right domain due to the unoccupied region. With the size of rank 2 domain (212 aa), it is possible to guess that domain occupied Density 6 should be at least 250 amino acids. (B) Plotting of the fitting statistics for all domains in Density 6 using a size filter of 250 amino acids. Purple indicates domain larger or equal 250 amino acids while light cyan indicates domains smaller than 250 amino acids and filtered out. With this size filtering, we can reduce false positive hits and identify the right domain easier. In this case, the correct domain (CFAP52) was ranked 3 in p-value before size filtering and ranked 1 in p-value after size filtering of 250 amino acids.\nStatistical analysis of fitting solutions.\n(A-D) The effect of 100 (A), 200 (B), 400 (C) and 800 (D) initial search placements on z-transformed correlation score for the correct domain. Even at a sampling of 100 initial search placements, the top hit (correct solution) still seems discriminated from the rest of the fits.\nAnalysis and reconstruction of the unknown density in the 48-nm doublet microtubule structure.\n(A) The cartoon indicates the view of the section of the B-tubule of the doublet microtubule. The classification mask is shown inside the 48-nm unit of the doublet microtubule (EMD-29692) showing the density inside is barely visible. Using Cryosparc 3D classification into four classes, Class 1 shows the density clearly inside the classification mask compared to classes 2-4. Local refinement of class 1 with the classification mask resulted in a 4.5 A resolution structure. After post-processing with DeepEmhancer, the density is segmented out of the map for protein identification by DomainFit. (B) The Fourier Shell Correlation curve of the map obtained in (A).\nComparison between domain fitting and complete model fitting.\n(A) The bump chart shows the top 5-hit proteins across 12 densities by p-value with a size filtering of 60 amino acids. Any domain ranked outside the top 4 hits is plotted at the same position in the plot (&gt; 4). There are only three proteins among the top hits (BMIP1, CCDC81B and Q23KF9). CCDC81B is consistently the top hit in many densities, helping us to rationalize the identity of the multidomain CCDC81B. (B) Top three hits for complete model fitting into the density and the right protein BMIP1 (ranked 22). Top 1 and 2 fit well to the density while I7MCW2 and BMIP1 fit poorly in the density. (C) Complete model fitting into the proximal density filtere at 10 A resolution. At this resolution, CCDC81B became the top hit. (D) Overlap of the best fit of CCDC81B into the proximal density at 4 and 10 A resolution shows that the fitting positions found are almost identical (RMSD = 1.309 A).\n\nSupplementary Table\nIdentification of MIPs in the cryo-EM map of the doublet microtubule using DomainFit.\nReason for identification of unknown densities",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:
[outputs]
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nIn the last decade, cryo-electron microscopy (cryo-EM) has become a powerful technique to determine the structures of macromolecular complexes. With advances in cryo-EM image processing algorithms, large endogenous complexes have been solved at high resolutions such as the phycobilisomes in red algae &lt;&lt;REF:Ma-2020-579-146&gt;&gt;, the mitochondrial membrane bending supercomplex &lt;&lt;REF:Muhleip-2023-615-934&gt;&gt; and the doublet microtubules of the cilia &lt;&lt;REF:Ma-2019-179-909&gt;&gt;,&lt;&lt;REF:Ichikawa-2019-116-19930&gt;&gt;. Recently, in situ cryo-electron tomography (cryo-ET) from thin lamellae of cells prepared by a cryo-focused ion beam (cryo-FIB) instrument has emerged as a transformative technique, revolutionizing our understanding of cellular structures and molecular processes &lt;&lt;REF:Berger-2023-20-499&gt;&gt;. With high-throughput tilt series acquisition and improvement of subtomogram averaging software &lt;&lt;REF:Wan-2020-26-2516&gt;&gt;-&lt;&lt;REF:Zivanov-2022-11-e83724&gt;&gt;, subtomogram averaging of complexes can reach sub-nanometer resolution and in some cases better than 4 A resolution &lt;&lt;REF:Tegunov-2021-18-186&gt;&gt;,&lt;&lt;REF:Xing-2023-381-70&gt;&gt;-&lt;&lt;REF:Tai-2023-9-116&gt;&gt;. Structures of protein complexes obtained in situ often contain unknown components, unlike in vitro reconstituted protein complexes. In addition, depending on the cellular context, space-and/or time-specific interactions with unknown proteins might be revealed by subtomogram averaging. Due to the lower resolution of these in situ structures, identification of proteins requires a different approach than traditional molecular modelling programs.\nFor high-resolution structures, there are many methods that can be used for the modelling and identification of unknown proteins including CryoID, DeeptracerID, FindMySequence, and modelAngelo &lt;&lt;REF:Jamali-2023-?-2023.2005.2016.541002&gt;&gt;-&lt;&lt;REF:Pfab-2021-118-e2017525118&gt;&gt;. These programs trace and model the backbone of the protein density. Then, the identity of the protein is predicted by comparing the side-chain densities against a database of protein sequences. This approach was successful in determining many proteins in the doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;,&lt;&lt;REF:Leung-2023-186-2880&gt;&gt;, radial spokes and central apparatus and the phycobilisomes &lt;&lt;REF:Jamali-2023-?-2023.2005.2016.541002&gt;&gt;. However, all these methods only work reliably at a resolution better than 4 A.\nIn most cases, the attainable resolution by subtomogram averaging is lower than 4 A. This is due to the low throughput of cryo-FIB milling and low abundance of the target protein complex. Furthermore, in situ structures can have problems related to orientation, sample thickness and signal-to-noise ratio. Therefore, it is a challenge to identify proteins from in situ subtomogram averages.\nIn the last few years, there has been a breakthrough in artificial intelligence (AI)-assisted protein structure prediction. Programs like AlphaFold &lt;&lt;REF:Jumper-2021-596-583&gt;&gt;, ColabFold &lt;&lt;REF:Mirdita-2021-?-2021.2008.2015.456425&gt;&gt; and RoseTTAfold &lt;&lt;REF:Baek-2021-373-871&gt;&gt; can produce accurate structure predictions. With the establishment of the AlphaFold Protein Structure Database (AlphaFold DB), everyone now has access to over 200 million predicted models. For certain organisms such as humans, mice, zebrafish and nematodes, AlphaFold2-predicted models of the entire proteome are available.\nWith the good accuracy of the AI-predicted models, it is possible to identify the fold signature of a protein at a resolution of less than 4 A by fitting the AI-predicted models into the density. Using complementary data such as in situ chemical cross-linking mass spectrometry, quantitative mass spectrometry and/or BioID, it is possible to identify proteins from the fit list. For example, microtubule inner proteins (MIPs) were identified from the in situ subtomogram average of human sperm doublet microtubules at ~6-7 A using Colores Situs program by fitting over 21,000 AlphaFold2-predicted protein models of the mouse proteome &lt;&lt;REF:Chen-2023-186-5041&gt;&gt;. In another study, 38 proteins were identified from the mouse sperm by manually fitting AlphaFold2-predicted models of proteins found in the mass spectrometry analysis of the same sample &lt;&lt;REF:Tai-2023-9-116&gt;&gt;.\nWhile those studies illustrate that it is possible to identify well-structured proteins from a map lower than 4 A resolution, the methods are not automated or are difficult to use. In addition, the criteria to evaluate and identify proteins from the fitting solutions are not clear from these studies. While the accuracy of the AI-predicted model is high for compact domains, the tertiary structure of the predicted model might not be correct, due in part to poor prediction of flexible regions. Therefore, protein identification is easier when domains are extracted from each AI-predicted model and used instead of the entire model.\nIn this work, we developed a program called DomainFit to identify domains of proteins from cryo-EM maps of lower resolution than 4 A. DomainFit uses popular programs such as Phenix &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt;, R &lt;&lt;REF:???-2021-?-?&gt;&gt; and UCSF ChimeraX (ChimeraX) &lt;&lt;REF:Pettersen-2021-30-70&gt;&gt;, making it easy to install and accessible to people. In addition, DomainFit is flexible and has a clear statistical approach to evaluate the identity of proteins. Lastly, DomainFit uses ChimeraX, which allows easy visualization to aid in evaluating the fit. Our workflow will be useful for the upcoming wave of in situ structures as cryo-FIB and cryo-ET become more popular.\n\nResults\n\nAn automated pipeline for domain parsing and fitting into cryo-EM map\nOverall, the program is designed to first download a database of AI-predicted models of candidate proteins (i.e. full species proteome or a limited list based on mass spectrometry studies), then divide each model into different domains and fit each individual domain into a segmented cryo-EM density. Finally, the program generates a statistical evaluation of all the fitted domains (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). The identification of the correct domain can be based purely on statistical analysis but also on other complementary data such as surrounding densities, protein size, chemical cross-linking and quantitative mass spectrometry.\nFigure 1:\nWorkflow of DomainFit.\n(A) Overview of the program. (i) Download the predicted PDBs for all candidate proteins. (ii) Predict domains and split them into single PDB files. This step can be done through the script process_predicted_models.py or results from other programs such as DPAM. (iii) Fit each domain into the unknown density using ChimeraX nd generate the best fitting position of each domain and fitting scores. (iv) Interpret the fitting results based on p-value, correlation, and size of proteins.\nThe list of candidate proteins can be obtained from the mass spectrometry results of the target sample such as the proteome of isolated organelles or pull-down analysis. In case a targeted list is not available, it is possible to use AI-predicted models of the entire proteome of the organism, even though it will increase computational time significantly. In DomainFit, we provide a script getAlphaFoldPDBs.py to automatically download the AlphaFold2-predicted models from the AlphaFold DB using a list of protein UniProtIDs (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). The downside of downloading automatically from the AlphaFold DB is that for many organisms, high molecular weight proteins (&gt; 100kDa) are not yet available in the database. These missing proteins can be predicted using ColabFold, RoseTTAFold or a local instance of AlphaFold.\nMany ideas have been proposed to break PDB models into compact domains automatically &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt;,&lt;&lt;REF:Croll-2022-?-?&gt;&gt;,&lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt;. With AI-predicted models, it is possible to partition into domains based on the predicted alignment errors (PAE) of each model (Fig. 2A-D &lt;&lt;FIG:fig2&gt;&gt;). PAE is the expected positional error for residue x if the predicted and actual models are aligned at residue y &lt;&lt;REF:Jumper-2021-596-583&gt;&gt;. Using an image-based approach, it is possible to partition predicted models into domains based on their respective PAE files (Fig. 2B-D &lt;&lt;FIG:fig2&gt;&gt;) &lt;&lt;REF:Croll-2022-?-?&gt;&gt;. We used phenix.process_predicted_model program from the Phenix package &lt;&lt;REF:Oeffner-2022-78-1303&gt;&gt; for the partitioning of the PDB into domains because of its flexibility. phenix.process_predicted_model can function based on PAE but also based on the 3D arrangement of the proteins without PAE information. Therefore, we implemented a Python wrapper process_predicted_models.py to partition PDBs into domains in batch (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). Users can customize the options for parsing based on their needs.\nFigure 2.\nDomain partitioning and fitting evaluation in DomainFit.\n(A) An example of an AlphaFold2-predicted model of a multi-domain protein (RIB72B, UniprotID I7MCU1). (B) The PAE plot of the RIB72B. (C) Domain partitioning of RIB72B based on the PAE plot. (D) Domain partitioning of RIB72B corresponding to (C). (E) Fitting of a domain of RIB72B inside its corresponding segmented density at 4 A resolution. (F) Histogram of the z-transformed correlation score of all obtained fits of the correct domain of RIB72B. The top hit (red) is separated from the score distribution. (G) Histogram of the z-transformed correlation score of a wrong domain into the same density in (F). (H) The scatter plot to visualize the value of - log10 of Benjamini Hochberg adjusted p-value versus the normalize cross correlation coefficient of each domain fitting into the density of interest. The top right corner two points are well separate for the rest of the point cloud, indicating likely well-matched domains into the density.\nTo increase the flexibility of our program, process_predicted_models.py writes out the information of the predicted domains using the format employed by DPAM &lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt;. As a result, we can use domain information data from either phenix.process_predicted_model or existing domain data of certain organisms such as human, mouse, and zebrafish from DPAM &lt;&lt;REF:Zhang-2023-32-ARTN e4548&gt;&gt; to generate the PDBs of individual domains. In the case of a small number of target proteins, we can even edit the domain information files manually to generate customized domain parsing.\nAt this step, we recommended imposing a filter on the minimum and maximum domain sizes for fitting. At the lower end, 40 amino acids is the minimum size that allows us to reduce the number of domains used without affecting the results since domains under 40 amino acids tend to be either wrongly partitioned or not compact. On the upper limit, we recommend using a bigger value like 1,000 amino acids as a fail-safe in the case that domain parsing does not work properly.\nTo prepare for the fitting of domains, a volume containing a compact density should be segmented from the cryo-EM map. The compact density means either globular or coiled-coil bundle. A long helical density does not work since its shape is not unique for identification of protein fold. Ideally, the segmented density should be equivalent to a domains of more than 50-300 amino acids in size. This serves two purposes: (i) to reduce computational time, and (ii) to avoid the wrongly predicted tertiary conformation caused by long flexible loop between different domains.\nOnce the segmented density is prepared, fit_domains_in_chimerax.py is used to fit all the individual domain models into the density to find the best fitting position of each domain (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;, Fig. 2E &lt;&lt;FIG:fig2&gt;&gt;). The script uses the ChimeraX fitmap command to fit each domain in the density from a fixed number of initial random translated and rotated search positions within the density defined as \"initial search placements\". Instead of exhaustive six dimensional search like Colores package in Situs, for global search, ChimeraX fitmap places the model randomly in rotation and translation within the map, and performs rigid-body local optimization from each initial placement. After the local optimization, ChimeraX will group the fitting solutions of based on similarity in orientation. Therefore, the number of fitting solutions tend to be smaller than the number of initial search positions due to clustering. These fitting solutions's correlation score are then z-transformed and then used for the calculation of the p-value, the likelihood that the top hit is correct and clearly significant compared to the rest of the fitting solutions (See Materials &amp; Methods, Fig. 2F &lt;&lt;FIG:fig2&gt;&gt;). Once the fitting is done for every domain in the density, we expect that top hits with the correct fold/domain are clearly distinguished from other fitting solutions by their z-transformed scores, whereas those with the incorrect domain are not so differentiated (Fig. 2G &lt;&lt;FIG:fig2&gt;&gt;). Visualizing the fitting statistics i.e. p-values and cross correlation coefficient of all domains allow a quick way to see how distinct is the top hits (Fig. 2H &lt;&lt;FIG:fig2&gt;&gt;).\nA comma-separated value (CSV) file containing the best fitting position of each domains is written at the end, sorted first by p-value and then by the correlation score for quick evaluation.\n\nDomainFit benchmarking for MIP identification\nTo assess the efficiency of DomainFit, we tested it on segmented densities from the single-particle cryo-EM map of the doublet microtubule from Tetrahymena thermophila (EMD-29685) at ~4 A resolution. More than 40 MIPs were identified and modelled using an AI-assisted modelling approach (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt;). Even though the resolution of the doublet microtubule cryo-EM map was high, modelling and identifying proteins was laborious and time-consuming. Our goal for this test was to see whether we could identify the MIPs using DomainFit independently.\nFigure 3:\nIdentification of MIPs in the T. thermophila doublet microtubule.\n(A) The doublet microtubule structure with MIPs (PDB: 8g2z). (B) Densities (red) in the cryo-EM map of the doublet microtubule (EMD-29685) for testing with DomainFit. (C) A histogram of domain size. Red bars indicate domains of less than 40 amino acids and are excluded from fitting. The x-axis is limited to a domain size of 800 amino acids. (D) DomainFit ranking by p-value and correlation about mean of the correct domain for 24 densities. (E) Example of the perfect match between the AlphaFold2-predicted model of PACRGB fitted within the corresponding density. (F) The domain of RIB22 is not partitioned as expected. However, the bigger domain was still fitted correctly into the density due to good agreement between the AlphaFold2-predicted model and the density. Due to incorrect parsing and the fact that the EF-hand domain is abundant in the cilium, the fit of the correct domain is ranked 9 but the p-value is still ranked 1st. (G) When the fold of the density is common (EF-hand domain RIB571-91 - Density 10), the top hits consist of domains with a similar fold. The top five hits are all domains from CFAP115, which have the same EF-hand fold as the correct domain RIB571-91. (H) The experimentally determined model of RIB57 (PDB: 8g2z) is shown inside its corresponding density. (I) There is a big discrepancy between the experimentally determined model (cyan) and AlphaFold2-predicted model (purple) of RIB571-91 (RMSD: 5.164 *), which leads to the poor fitting of RIB57.\nWe segmented 24 densities from 13 identified MIPs in the cryo-EM map corresponding to different domains of MIPs (Supplementary Table 1, Figure 2B &lt;&lt;FIG:fig2&gt;&gt;). We established a database comprising 856 AlphaFold2-predicted models based on the mass spectrometry data of the WT cilia without membrane (994 detected proteins) &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. One hundred and thirty eight proteins were not available from the AlphaFold DB because of a lack of predictions due to their high molecular weight. For benchmarking purpose, we ignored them for this test since the 856 AlphaFold2-predicted models covered all the tested MIPs.\nUsing process_predicted_models.py and save_domains_from_info.py, we divided 856 AlphaFold2-predicted models into 2819 domains. We removed domains of less than 40 amino acids which left us with 1561 domains for fitting (Fig. 3C &lt;&lt;FIG:fig3&gt;&gt;). We then fitted all 1561 domains into each density filtered at 4 A using 200 initial search position placements. On average, each density search took ~30 minutes using 12 cores (20 threads) on an i7-12700K (12 core-CPU and 32GB RAM).\nOut of all the densities tested, we found 22 out of 24 correct domains are present in the top 5 hits ranked by descending p-value (Supplementary Table 1, Fig. 3D &lt;&lt;FIG:fig3&gt;&gt;, Fig. S1 &lt;&lt;FIG:figs1&gt;&gt;). Overall, we found the p-value to be a better predictor of correct identification and fitting than the normalized cross correlation coefficient, which is called correlation about mean in ChimeraX. Often, the domains were correctly found as the top hits with the highest correlation and best p-value such as PACRGB, an inner junction protein &lt;&lt;REF:Khalifa-2020-9-e52760&gt;&gt; (Fig. 3E &lt;&lt;FIG:fig3&gt;&gt;). Our results show that the p-value ranks consistently higher than the correlation score for the correct domains for the densities (Fig. 3D &lt;&lt;FIG:fig3&gt;&gt;). In the cases where domains have a common fold, multiple domains returned a p-value of 2.22*10-16 (limit of R program). Thus, we sorted the fitting solutions first based on p-values, and then by their normalized cross correlation coefficient. As a result, hereafter, top hits will refer to the top p-value ranked hits.\nIn some cases such as Density 6 (CFAP5213-317), by looking at the fit of the domains in the density, we can roughly estimate the size of the domain and establish a minimum domain size threshold, which allows quicker identification of the correct domain (Fig. S2A-B &lt;&lt;FIG:figs2&gt;&gt;).\nThere are quite common cases where the partitioned domain is bigger than the segmented density (Supplementary Table 1, Fig. S1 &lt;&lt;FIG:figs1&gt;&gt;, Density 6, 10-13, 16-17, 19-22, 24). This might affect the fitting of DomainFit. For example, our parameters for domain parsing tend to keep entire EF-hand domain pairs as a single domain, instead of two separate EF-hand domains. For RIB22, while the density represents only RIB221-91, RIB224-191 was partitioned as one domain (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt;). On the other hand, the correct fitting was found for RIB224-191 with a p-value and correlation value of rank 1st (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt;, Supplementary Table 1).\nInterestingly, for Density 10, corresponding to the EF-hand domain RIB571-91, DomainFit failed to find the correct domain (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt;). While all the top hits were the wrong protein, they all had the correct EF-hand fold (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt;). when we compared the experimentally determined model of RIB57 (Fig. 3H &lt;&lt;FIG:fig3&gt;&gt;) and the AlphaFold2-predicted model, there was a big difference between the two models from residues 6 to 91 (RMSD 5.164 A, Fig. 3I &lt;&lt;FIG:fig3&gt;&gt;). Therefore, the failure to find RIB571-91 density (Density 10) was due to both the difference in the AlphaFold2-predicted model and the unsuccessful parsing of RIB571-91 domain.\nWe conclude that DomainFit works well but attention needs to be paid to potential mis-identification of proteins due to their size and domain parsing. Also, running with a higher number of initial search placements is a safeguard against inaccuracy in domain parsing to accommodate more initial translated and rotated positions.\n\nPerformance of DomainFit depending on the number of initial placements and resolution\nOur test results showed that DomainFit is effective in identifying compact densities and the p-value is the most reliable indicator of the correct hit. For p-value to be estimated properly, a reasonable number of data points, i.e. fitting solutions are needed. Therefore, we wanted to explore the sensitivity of the p-value estimation to the number of initial search placements. We ran DomainFit for Density 1 100 times for each initial search placement value ranging from 1 to 300 and plotted the average p-value found. Our analysis shows that above 150 initial search placements, the resulting p-value is consistently found to be as high a value as it can be (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Thus, setting a search placement value of 150 should yield optimal results for the speed and accuracy of the fits in this setup. This setting is appropriate for a domain of 100-150 amino acids while larger domains may require a higher number of initial search placements. We also looked at the z-transform correlation score for the correct domain at different numbers of initial search placements (Fig. S3A-D &lt;&lt;FIG:figs3&gt;&gt;). The statistical calculation is certainly less reliable when the number of fitting points is small. For the correct domain, if the correlation score of the top hit is distinct from that of the rest, a good p-value can still be obtained.\nFigure 4:\nPerformance of DomainFit at different numbers of initial search placements and resolutions.\n(A) -log10 of the average p-value at increasing initial search placements for a correct and an unrelated domain. Standard deviations are shown in coloured regions. (B) Appearance changes of a density filtering at 4, 6, 8 and 10 A resolution. (C) p-value rank of the correct domains fitted to different densities of MIPs at 4, 6, 8, and 10 A resolution. (D) Number of fitting solutions versus number of initial search placements for maps at 4 A (blue), 6 A (red), 8 A (green) and 10 A (yellow) resolution. (E) A histogram of the z-transformed correlation score of a correct domain fitted to a density at 4 A resolution shows a clear separation between the top hit and the rest of the fitting solutions. (F) Histograms of z-transformed correlation score of the same domain fitted to the densities filtered at 6, 8 and 10 A resolutions. There are significantly fewer fitting solutions at lower resolutions and the separation of the top hit, and the rest is reduced.\nThe number of fitting solutions found does not depend only on the number of initial search placements but also on the resolution of the map. Next, we wanted to check the effect of map resolutions on the success of DomainFit. To test whether the program could successfully identify domains at lower resolutions, we filtered and ran the density maps of the MIPs at 4, 6, 8, and 10 * (Fig. 4B &lt;&lt;FIG:fig4&gt;&gt;). We found that the search consistently identified the correct domain within the top 10 results at 4 to 8 * resolution (Fig. 4C &lt;&lt;FIG:fig4&gt;&gt;). At lower resolutions, there are significantly fewer fitting solutions found even with the same number of initial search placements (Fig. 4D &lt;&lt;FIG:fig4&gt;&gt;). This is because of the smoothening of the map at lower resolutions, which does not allow many unique fitting solutions. Because of this, ChimeraX produces a small number of clusters of fitting solutions from a large number of initial search placements. This leads to less confidence in statistical analysis of the top hits. If we look at the z-transformed correlation score at 4, 6, 8 and 10 A of the correct domain (Fig. 4E, F &lt;&lt;FIG:fig4&gt;&gt;), we can see that at 6 A, the distribution of the fitting solution and the top hits still maintains its shape, allowing reliable p-value calculation. At 8 A, while the top hit is still well separated from the distribution, the rest of the solution does not look normally distributed. At 10 A, not only does the distribution of the correlation score is not normally distributed but the difference between the top hit and the rest gets smaller.\nAs a result, DomainFit seems to work best in the 4-to-6 A resolution range. At 8 A resolution, the number of initial search placements must be increased to help with statistical calculation. At lower than 8 A, DomainFit should only be used for finding types of folds that fit well in the density by visual inspection of the top hits but it should not be used for domain identification.\n\nNew MIP identified in the doublet microtubule\nWith our success in testing and validating the DomainFit program, we attempted to identify unknown proteins from cryo-EM maps with a resolution lower than 4 A. An unidentified MIP was previously reported in the subtomogram average of the T. thermophila doublet microtubule &lt;&lt;REF:Li-2022-5-e202101225&gt;&gt; (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt;). However, this density was not observed in the single particle cryo-EM map of the doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; (Fig. 5B &lt;&lt;FIG:fig5&gt;&gt;) and at too low resolution to be identified.\nFigure 5:\nNew proteins identified from the doublet microtubule of T. thermophila.\n(A) Overview of the unknown density (yellow) from the 48-nm repeating unit subtomogram average map of doublet microtubule (EMD-24376) &lt;&lt;REF:Li-2022-5-e202101225&gt;&gt;. (B) The same density is not visible in the same view of the single particle cryo-EM map of the 48-nm repeating unit of the doublet microtubule from the T. thermophila K40R map (EMD-29692) &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. (C) The unknown density exists in the doublet microtubule map after 3D classification. (D) Segmentation of the unknown density into different small unknown densities for DomainFit. (E) DomainFit top hits for each unknown density using an AlphaFold database of salt-treated ciliome. (F) Corrected model of protein domains after examining, showing BMIP1 (UniprotID I7MB72) is density 1 and CCDC81B (UniprotID I7M688) forms density 2-12. (G) Fit of AlphaFold2-predicted model BMIP112-168 into density 1. (H) Fit of AlphaFold2-predicted model CCDC81B38-218 into density 2.\nWe re-processed the cryo-EM dataset containing the 48-nm particles of T. thermophila K40R doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; with Cryosparc &lt;&lt;REF:Punjani-2017-14-290&gt;&gt; Using 3D classification and refinement, we obtained a 4.5 A resolution cryo-EM map with the same density as the subtomogram average (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;, Fig. S4A, B &lt;&lt;FIG:figs4&gt;&gt;).\nTo identify the proteins in the unknown density, we segmented the density into 12 smaller densities (Fig. 5D &lt;&lt;FIG:fig5&gt;&gt;). To improve the accuracy of the search, we used the ciliome of the salt-treated doublet microtubules &lt;&lt;REF:Ichikawa-2019-116-19930&gt;&gt;. The reason is that the cryo-EM map of salt-treated doublet microtubules retains most of the MIPs and the unknown density &lt;&lt;REF:Ichikawa-2017-8-15035&gt;&gt;. The salt-treated ciliome of 166 proteins was divided into 334 domains of at least 40 amino acids. We ran DomainFit with these 334 domain models for the above 12 densities.\nInterestingly, the top hits for the 12 densities came from only three proteins (Fig. 5E &lt;&lt;FIG:fig5&gt;&gt;, Supplementary Table 2). Upon inspecting the results, we concluded that the 12 densities are composed of only two proteins I7MB72 (TTHERM_00525130) and I7M688 (TTHERM_00649260) because of the overall architecture (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;, Fig. S5A &lt;&lt;FIG:figs5&gt;&gt;) and the fit of unique domains in both proteins (Fig. 5G-H &lt;&lt;FIG:fig5&gt;&gt;).\nI7MB72 matches unknown density 1 with high confidence in p-value and correlation (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;), referring as BMIP1 from here onwards. Upon inspection of the map, density 1 was always present in the doublet microtubule (EMD-29692) but was not identified previously. As a result, we could model extra regions of the protein BMIP1 in the T. thermophila K40R map (EMD-29692) at 3.5 A resolution, confirming the identity of unknown density 1 and the success of DomainFit (Fig. 6A, B &lt;&lt;FIG:fig6&gt;&gt;). In addition, BMIP1 is abundant in both salt-treated and native doublet microtubules similar to the profiles of most MIPs. BMIP1 has a cross-link to CFAP45, which exists as two copies between protofilaments B8B9 and B7B8, as shown in studies that usedchemical cross-linking coupled with mass spectrometry of cilia &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;,&lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt; (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). While the cross-linked residues (Lysines 213 and 222) from BMIP1 are not modelled, they seem to be within the cross-link distance (&lt; 35 A) from lysines 403 and 283 of CFAP45.\nFigure 6:\nValidation of the newly identified MIPs.\nA. Model of BMIP1 (UniprotID I7MB72) using Coot. B. Side-chain densities of a helix from BMIP1. C. Intra-cross-links (yellow) within CCDC81B and inter-cross-links (orange) between CCDC81B and CFAP106, a known MIP protein &lt;&lt;REF:Khalifa-2020-9-e52760&gt;&gt;. D. Merged super resolution-structured illumination microscopy image of T. thermophila cells with CCDC81B-GFP shows that CCDC81B only localizes to the proximal region of the cilia. Rectangle indicates the zoom-in view in (E). Red: CCDC81B-GFP signal; Green: doublet microtubule labelled by polyG-Antibody; Blue: Hoechst dye labelling the nucleus. E. The zoom-in merged image (top) and GFP channel (bottom) showing localization of CCDC81B to the proximal region.\nDensities 2-12 (Fig. 5D &lt;&lt;FIG:fig5&gt;&gt;) belong to I7M688, a CCDC81-domain containing protein. We named I7M688 as CCDC81B from here onwards. The CCDC81 domain of CCDC81B is unambiguously fitted into unknown density 2 (Fig. 5H &lt;&lt;FIG:fig5&gt;&gt;). There are a few CCDC81-containing proteins in the cilia of T. thermophila: IJ34 (UniprotID I7M9T0), CCDC81A (UniprotID I7MLF6), CCDC81B (UniprotID I7M688), Q240Y1 and Q22HG4. IJ34 is a MIP near the inner junction &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; and CCDC81A was identified as a ciliary tip protein &lt;&lt;REF:Legal-2023-222-?&gt;&gt;. Q240Y1 and Q22HG4 are clear paralogs of CCDC81B with similar architecture but are washed away in salt-treated cilia &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;.\nMass spectrometry results show that CCDC81B is resistant to salt and is similar to a MIP profile &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;. In situ cross-linking mass spectrometry shows that CCDC81B is crosslinked to tubulin alpha, CFAP106, FAP210, and interestingly RIB43A &lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt;. All these cross-links are satisfied relative to the CCDC81B position in the unknown density, except for RIB43A (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). Therefore, we are confident that CCDC81B corresponds to unknown densities 2-12.\nFor a quick comparison of DomainFit with complete AlphaFold2 model fitting, we also fitted 166 complete AlphaFold2-predicted models into the full segmented density (Fig. S4 &lt;&lt;FIG:figs4&gt;&gt;). The top hits were IJ34 and CCDC81B. Both proteins contain a CCDC81 domain (Fig. S5B &lt;&lt;FIG:figs5&gt;&gt;). BMIP1 only ranks 22 in p-value. The AlphaFold2-predicted model of CCDC81B has good tertiary structure prediction, which helps the complete model fitting. However, this shows the weakness of complete model fitting. First, complete model fitting seems to penalize smaller domains such as BMIP1. Second, doing complete model fitting does not allow to filter the sizes of the fitted domains, which is useful to eliminate false positive fits. Finally, with DomainFit, we can visualize the top hits from different densities (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt;), allowing a quicker identification of multidomain proteins without relying on the quality of the tertiary structure prediction. On the other hand, complete model fitting of 166 AlphaFold2-predicted models in the density filtered at 10 A resolution also found CCDC81B as the top hit with good fitting accuracy, thanks to good tertiary structure prediction (Fig. S5C, D &lt;&lt;FIG:figs5&gt;&gt;).\nTo further confirm that CCDC81B is a MIP found only in a subset of the doublet microtubule, we generated a T. thermophila strain with CCDC81B fused with GFP. Under superresolution structured illumination microscopy, the signal of CCDC81B was limited to the 1-1.5 um proximal region of the cilia (Fig. 6C &lt;&lt;FIG:fig6&gt;&gt;). That explains the ratio of about substoichiometric fraction of the particles containing CCDC81B densities from the single-particle cryo-EM data. As a result, we can confirm that CCDC81B is a MIP that localizes to the proximal region of the cilia.\n\nDiscussion\nIn this paper, we demonstrated that it is possible to identify compact domains of proteins in maps at a resolution lower than 4 A using the DomainFit program. We benchmarked DomainFit using known structures and successfully identified CCDC81B and BMIP1 as two new proteins in the doublet microtubule of T. thermophila. Notably, we showed that CCDC81B only exists in the proximal region of the cilium using superresolution structured illumination microscopy of T. thermophila cells expressing CCDC81B fused to GFP.\nWhile DomainFit works, it is not as definite without complementary data as high-resolution cryo-EM data where side-chain information is available. We showed that the statistical analysis of the best-fitting domain works well. When the fold of the domain is unique, the identification works excellently. When the fold is not unique, DomainFit still finds the correct fold. Tools like Foldseek &lt;&lt;REF:van Kempen-2023-?-?&gt;&gt; can then be used to list all the proteins with the same fold available. With other complementary info such as chemical cross-linking mass spectrometry, BioID, and stoichiometry of proteins from quantitative mass spectrometry, it is possible to narrow down and identify the right proteins as demonstrated by this work and other integrated structural biology approach &lt;&lt;REF:Ghanaeian-2023-14-5741&gt;&gt;. There are certain cases in which the AlphaFold2-predicted model is not similar to the protein fold in the map, leading to the wrong identification. However, it does not happen often with compact domains.\nThe validation of DomainFit using known MIPs highlights a caveat of the workflow in the partitioning of domains. Breaking proteins into compact domains will never be perfect since over-partitioning will produce smaller domains and therefore increase false positive fits while under-partitioning produces bigger domains, requiring a high number of initial search positions, compromising the computational requirement. Perhaps, the development of domain partitioning with better customization in the future can improve the usability of DomainFit. Despite that caveat, our work shows that domain fitting is in general more accurate than complete model fitting since AlphaFold2 domain prediction is in general more accurate than AlphaFold2 tertiary structure prediction.\nAnother point to consider for a successful identification is the quality of the map. It seems that DomainFit works well between 4 and 6 A resolution where detailed secondary structural features can be visualized. It is recommended to use appropriate post-processing methods to improve the interpretability of the map such as DeepEmhancer &lt;&lt;REF:Sanchez-Garcia-2021-4-874&gt;&gt;, LocSpiral &lt;&lt;REF:Kaur-2021-12-1240&gt;&gt; or Density Modification &lt;&lt;REF:Terwilliger-2020-17-923&gt;&gt;.\nAt the lower resolution of 8-10 A, DomainFit can still work but requires complementary data for validation. For lower resolution, perhaps using domains with bigger partitions or even the entire AlphaFold2-predicted model might help with the fitting and identification if the tertiary structure is predicted correctly. With the flexibility of DomainFit, users can try both approaches and evaluate the fitting visually in the case of low resolutions using scripts provided by DomainFit.\nIn addition, since the program is rather quick, it is possible to use DomainFit to construct the structures of complexes with known compositions. A database of domains from all the known components can fit into different densities of the map. This allows a more unbiased way of building up the structure of the protein complex.\nIn conclusion, we presented here the DomainFit program, which allows the unbiased fitting of domains of proteins into the cryo-EM map. At resolutions better than 6 A, the program can be used reliably as a tool to identify compact proteins in the map.\n\nMaterials &amp; Methods\n\nDomain parsing\nFor domain parsing, we used the option split_model_by_compact_regions=True from phenix.process_predicted_model. In addition, we set the option maximum_domains for each protein so that the maximum_domains equal the protein size in amino acids divided by 100 amino acids.\n\np-value calculation\nThe p-value calculation for the fitting of each domain has been implemented in R &lt;&lt;REF:???-2021-?-?&gt;&gt; in the integrated modelling software Assembline &lt;&lt;REF:Rantos-2022-17-152&gt;&gt; and used for the identification of the nuclear pore complex Y-shaped scaffold &lt;&lt;REF:Bui-2013-155-1233&gt;&gt;.\nIn brief, for each domain, ChimeraX clusters the results into a number of fitting solutions with distinct rotational and translational parameters, and cross-correlation scores. As a result, many searches with different initial rotation and translation placement, resulting in similar final fitting orientation are clustered into one fitting solution. The correlation scores of all fitting solutions are transformed to z-scores using Fisher's z-transform and centered to yield an approximately normal distribution (Fig. 2F &lt;&lt;FIG:fig2&gt;&gt;).\nThen, the two-sided p-value and Benjamini-Hochberg adjusted p-value are calculated from the z-scores using the false discovery rate (fdr) package in R. The top hit for each domain which corresponds to the highest correlation score and lowest p-value is recorded in an aggregated list. Once this process is done for all the domains, the list is then sorted by p-value. When the correct domain is evaluated, the top hit's z-transformed score is clearly discriminated from the rest of the fitting solution while the difference in value is not as clear when a wrong domain is evaluated (Fig. 2F, G &lt;&lt;FIG:fig2&gt;&gt;). That translates to a lower p-value for the top hit of the correct domain.\nThe p-value is a more reliable indication of true positives than the correlation score since it is less sensitive to the size of the domain fitted in the density. Small domain models due to wrong partitioning always have high correlation scores because they fit perfectly in a small area of the density. For the correct domain, the correlation score difference between the top hit and the second hit tends to be larger except for domains with a symmetrical shape (e.g. WD40 beta-propeller). We also found that the Benjamini-Hochberg adjusted p-value functions similar to p-value.\n\nCryo-EM and image analysis\nThe single particle cryo-EM data of the 48-nm repeating unit of the doublet microtubule from T. thermophila was originally published in &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt;.\nTo obtain the density with a resolution better than 12 A resolution of the subtomogram average, we constructed a focused classification mask at the region of the density based on the EMD-24376 subtomogram average map and performed 3D classification without alignment for four classes on the 48-nm particles cryo-EM dataset of T. thermophila K40R doublet microtubule &lt;&lt;REF:Kubo-2023-14-2168&gt;&gt; using Cryosparc &lt;&lt;REF:Punjani-2017-14-290&gt;&gt; (Fig. S4A &lt;&lt;FIG:figs4&gt;&gt;). After classification, class 1 contains ~45,000 particles showing the same density as the subtomogram average (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;). This suggests that the density feature observed by subtomogram averaging is not uniformly located in the cilia. We further refined Class 1 in Cryosparc with focused local refinement using a refinement mask slightly larger than the classification mask, resulting in a cryo-EM map of 4.5 A resolution (Fig. 5C &lt;&lt;FIG:fig5&gt;&gt;, Fig. S4A, B &lt;&lt;FIG:figs4&gt;&gt;). The details in the density map suggest a 5-5.5 A resolution.\nThe final map was post-processed using DeepEMhancer &lt;&lt;REF:Sanchez-Garcia-2021-4-874&gt;&gt;.\nAll the visualization of maps and models were done in ChimeraX &lt;&lt;REF:Emsley-2010-66-486&gt;&gt;.\n\nCryo-EM density map segmentation\nTo segment the density of interest for DomainFit from the cryo-EM map, we manually placed markers onto the density of interest in ChimeraX. After that, we colored the density around the markers, and use volume splitbyzone function of ChimeraX to segment the density out and save.\n\nModelling\nFor modelling of BMIP1 (UniprotID I7MB72, TTHERM_00525130), we started with the AlphaFold2-predicted model of the globular domain of BMIP1 fitted in its density. We fixed the model manually in Coot &lt;&lt;REF:Afonine-2018-74-531&gt;&gt; and modelled some extra regions of the protein. The final models were then real-space refined in Phenix &lt;&lt;REF:Emsley-2010-66-486&gt;&gt;.\nFor the modelling of CCDC81B (UniprotID I7M688, TTHERM_00649260), best fitting positions of different domains of CCDC81B were found using DomainFit. We joined all the domains together using Coot &lt;&lt;REF:Afonine-2018-74-531&gt;&gt; and refined in Phenix &lt;&lt;REF:McCafferty-2023-?-2023.2007.2009.548259&gt;&gt;.\n\nCross-link mass spectrometry visualization\nCross-links to BMIP1 and CCDC81B were obtained from a chemical cross-linking mass spectrometry of T. thermophila cilia (Reported in Supplementary Table 1 in &lt;&lt;REF:Lagerwaard-2022-?-2022.2004.2021.489026&gt;&gt;). The crosslinks were visualized in ChimeraX using the bundle XMAS &lt;&lt;REF:Williams-1980-27-327&gt;&gt;.\n\nCell culture and gene editing\nAll Tetrahymena strains used in this study were grown in SPP media &lt;&lt;REF:Jiang-2019-15-e1008099&gt;&gt; in a shaker incubator at 30degreesC and 120 rpm.\nThe CCDC81B gene, TTHERM_00649260 gene was edited by homologous DNA recombination using a targeting plasmid carrying the neo4 selectable marker. The portions of TTHERM_00649260 required for gene targeting were amplified using primers 5F (5'- ctatagggcgaattggagctttgtgaaatagatggaagag-3') and 5R (5'- atcaagcttgccatccgcggacttgtgaatttttaaagagat-3') amplified a terminal portion of the coding region and primers 3F (5'- gcttatcgataccgtcgaccatcaattatttcaaagtattaa-3') and 3R (5'- agggaacaaaagctgggtacgcattatccaaaatatattctaa -3') amplified a portion of the 3' UTR and cloned into the pNeo24-GFP plasmid &lt;&lt;REF:Gaertig-2013-525-265&gt;&gt;. The resulting edited fragment TTHERM_00649260 was targeted to the native locus using biolistic bombardment of T. thermophila cells and paromomycin selection.\n\nImmunofluorescence\nFor immunofluorescence, T. thermophila cells were fixed and stained as described &lt;&lt;REF:Bre-1996-109-727&gt;&gt;. The primary antibodies used were the mouse monoclonal anti-polyglycylated tubulin AXO49 (diluted 1:200) &lt;&lt;REF:Bre-1996-109-727&gt;&gt; and polyclonal anti-GFP antibodies (Rockland, 1:800). The secondary antibodies used were goat-anti-mouse IgG-FITC and goat-anti-rabbit-Cy3 antibodies (Jackson Immunoresearch). SR-SIM imaging was conducted on an ELYRA S1 microscope equipped with a 63x NA 1.4 Oil Plan-Apochromat DIC objective. The optical slices were analyzed by Fiji/ImageJ (Z project tool).\n\nData Availability\nThe data generated in this study are available in the following databases: XXX (The proximal proteins from the 48-nm Tetra K40R doublet), EMD-YYY (The proximal density from the 48-nm Tetra K40R doublet). All data used but not produced in this study are available in the following databases: 8G2Z (PDB-model of the 48-nm Tetra WT doublet), EMD-29685 (48-nm Tetra WT doublet), EMD-29692 (48-nm Tetra K40R doublet).\n\nCode Availability\nThe code for DomainFit is available at https://github.com/builab/DomainFit\n\nAuthor Contributions\nConceptualisation: KHB. Methodology: MT, JGao, TL, KHB. Software: MT, JGao, KHB. Formal analysis: JGao, MT, TL, KHB. Investigation: JGao, MT, TL, CL, JG, KHB. Resources: JG, KHB. Supervision: TL, JG, KHB. Writing-original draft: TL, KHB. Writing-review &amp; editing: JGao, TL, JG, KHB.\n\nSupplementary Figures\nAll segmented densities and best-fitted domains found by DomainFit.\nBlue: correct domain; Orange: incorrect domain. The correct domains corresponding to Density 6, 10-13, 16-17, 19-22, 24) were partitioned bigger than the densities.\nApplication of domain size filtering for domain identification.\n(A) Top three hits for Density 6 fitting. While rank 1 and 2 domains fit well in Density 6, they are clearly not the right domain due to the unoccupied region. With the size of rank 2 domain (212 aa), it is possible to guess that domain occupied Density 6 should be at least 250 amino acids. (B) Plotting of the fitting statistics for all domains in Density 6 using a size filter of 250 amino acids. Purple indicates domain larger or equal 250 amino acids while light cyan indicates domains smaller than 250 amino acids and filtered out. With this size filtering, we can reduce false positive hits and identify the right domain easier. In this case, the correct domain (CFAP52) was ranked 3 in p-value before size filtering and ranked 1 in p-value after size filtering of 250 amino acids.\nStatistical analysis of fitting solutions.\n(A-D) The effect of 100 (A), 200 (B), 400 (C) and 800 (D) initial search placements on z-transformed correlation score for the correct domain. Even at a sampling of 100 initial search placements, the top hit (correct solution) still seems discriminated from the rest of the fits.\nAnalysis and reconstruction of the unknown density in the 48-nm doublet microtubule structure.\n(A) The cartoon indicates the view of the section of the B-tubule of the doublet microtubule. The classification mask is shown inside the 48-nm unit of the doublet microtubule (EMD-29692) showing the density inside is barely visible. Using Cryosparc 3D classification into four classes, Class 1 shows the density clearly inside the classification mask compared to classes 2-4. Local refinement of class 1 with the classification mask resulted in a 4.5 A resolution structure. After post-processing with DeepEmhancer, the density is segmented out of the map for protein identification by DomainFit. (B) The Fourier Shell Correlation curve of the map obtained in (A).\nComparison between domain fitting and complete model fitting.\n(A) The bump chart shows the top 5-hit proteins across 12 densities by p-value with a size filtering of 60 amino acids. Any domain ranked outside the top 4 hits is plotted at the same position in the plot (&gt; 4). There are only three proteins among the top hits (BMIP1, CCDC81B and Q23KF9). CCDC81B is consistently the top hit in many densities, helping us to rationalize the identity of the multidomain CCDC81B. (B) Top three hits for complete model fitting into the density and the right protein BMIP1 (ranked 22). Top 1 and 2 fit well to the density while I7MCW2 and BMIP1 fit poorly in the density. (C) Complete model fitting into the proximal density filtere at 10 A resolution. At this resolution, CCDC81B became the top hit. (D) Overlap of the best fit of CCDC81B into the proximal density at 4 and 10 A resolution shows that the fitting positions found are almost identical (RMSD = 1.309 A).\n\nSupplementary Table\nIdentification of MIPs in the cryo-EM map of the doublet microtubule using DomainFit.\nReason for identification of unknown densities'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state..\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [13.27s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"sample_type\": \"micro-organism\",\n  \"sample_preparation_type\": \"subtomogram averaging\",\n  \"sample_preparation_buffer_ph\": \"Unknown\",\n  \"grid_vitrification_cryogen\": [\"ETHANE\"],\n  \"sample_preparation_cryo_protectant\": \"None\",\n  \"grid_model\": \"Unknown\",\n  \"grid_material\": \"GOLD\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"HOLEY\",\n  \"grid_vit_ctemp\": \"Unknown\",\n  \"grid_vit_chumid\": \"Unknown\",\n  \"organism_name\": \"Tetrahymena thermophila\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"None\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"sample_type\": \"micro-organism\",\n  \"sample_preparation_type\": \"subtomogram averaging\",\n  \"sample_preparation_buffer_ph\": \"Unknown\",\n  \"grid_vitrification_cryogen\": [\"ETHANE\"],\n  \"sample_preparation_cryo_protectant\": \"None\",\n  \"grid_model\": \"Unknown\",\n  \"grid_material\": \"GOLD\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"HOLEY\",\n  \"grid_vit_ctemp\": \"Unknown\",\n  \"grid_vit_chumid\": \"Unknown\",\n  \"organism_name\": \"Tetrahymena thermophila\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"None\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 177,
      "prompt_tokens": 12686,
      "total_tokens": 12863
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_94f711dcf6"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [0ms] Exiting Parser run with output:
{
  "sample_type": "micro-organism",
  "sample_preparation_type": "subtomogram averaging",
  "sample_preparation_buffer_ph": "Unknown",
  "grid_vitrification_cryogen": [
    "ETHANE"
  ],
  "sample_preparation_cryo_protectant": "None",
  "grid_model": "Unknown",
  "grid_material": "GOLD",
  "grid_mesh": "Unknown",
  "grid_support_topology": "HOLEY",
  "grid_vit_ctemp": "Unknown",
  "grid_vit_chumid": "Unknown",
  "organism_name": "Tetrahymena thermophila",
  "tissue": "None",
  "cell_type": "None",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/end] [1:chain:RunnableSequence] [13.28s] Exiting Chain run with output:
{
  "sample_type": "micro-organism",
  "sample_preparation_type": "subtomogram averaging",
  "sample_preparation_buffer_ph": "Unknown",
  "grid_vitrification_cryogen": [
    "ETHANE"
  ],
  "sample_preparation_cryo_protectant": "None",
  "grid_model": "Unknown",
  "grid_material": "GOLD",
  "grid_mesh": "Unknown",
  "grid_support_topology": "HOLEY",
  "grid_vit_ctemp": "Unknown",
  "grid_vit_chumid": "Unknown",
  "organism_name": "Tetrahymena thermophila",
  "tissue": "None",
  "cell_type": "None",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nCilia are motile or sensory organelles present on the surface of many eukaryotic cells. Motile cilia generate movement for processes such as mucus flow in airways or swimming of unicellular green algae, while primary cilia function as signaling hubs with fundamental roles in organism development (&lt;&lt;REF:Klena-2022-38-103&gt;&gt;; &lt;&lt;REF:Mill-2023-24-421&gt;&gt;). Ciliary dysfunction is associated with numerous diseases, collectively called ciliopathies, which affect multiple organs and tissues (&lt;&lt;REF:Badano-2006-7-125&gt;&gt;; &lt;&lt;REF:Hildebrandt-2011-364-1533&gt;&gt;; &lt;&lt;REF:Klena-2022-38-103&gt;&gt;). The core of each cilium is formed by a microtubule (MT)-based structure, the axoneme, which is often disrupted in ciliopathies (&lt;&lt;REF:Deretic-2023-136-?&gt;&gt;). Diverse proteins and pathways responsible for axoneme formation have been identified through genetic screens and cell biology studies, yet the biochemical mechanisms governing the dynamics of axonemal MTs remain very poorly understood (&lt;&lt;REF:Keeling-2016-5-6&gt;&gt;).\nUnlike cytoplasmic MTs, axonemal plus ends grow very slowly likely impeding the formation of a stabilizing GTP cap, which is required for preventing MT disassembly (&lt;&lt;REF:Desai-1997-13-83&gt;&gt;) and efficient tubulin addition (&lt;&lt;REF:Wieczorek-2015-17-907&gt;&gt;). The absence of a stable GTP cap can be compensated by specific factors bound to MT ends. Excellent candidates for this function in cilia are the components of the recently identified ciliary tip interaction network or \"module\", CEP104, CSPP1, TOGARAM1, ARMC9 and CCDC66 (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Depletion of any of these proteins causes ciliary defects, and mutations in most of them lead to a primary ciliopathy called Joubert syndrome, characterized by brain malformations, breathing problems or more severe developmental disorders (&lt;&lt;REF:Akizu-2014-94-80&gt;&gt;; &lt;&lt;REF:Hildebrandt-2011-364-1533&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Shaheen-2014-94-73&gt;&gt;; &lt;&lt;REF:Srour-2015-97-744&gt;&gt;; &lt;&lt;REF:Tuz-2014-94-62&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;). The proteins comprising the ciliary tip module interact with MTs and with each other, and although they do not form a stoichiometric complex, they are all important for controlling ciliary length and the signaling pathways dependent on primary cilia (&lt;&lt;REF:Conkar-2017-130-1450&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Patzke-2006-209-199&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;). In addition to their roles in cilia, some of the tip module proteins also associate with centrioles, centrosomes, centriolar satellites and cytoplasmic MTs, where they participate in cilia-independent processes, such as cell division (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;; &lt;&lt;REF:Conkar-2019-9-14250&gt;&gt;; &lt;&lt;REF:Patzke-2006-209-199&gt;&gt;; &lt;&lt;REF:Ryniawec-2023-33-4202&gt;&gt;; &lt;&lt;REF:Zhu-2015-290-27053&gt;&gt;).\nCEP104 (also known as FAP256), TOGARAM1 (also known as Crescerin or CHE-12) and ARMC9 are conserved across eukaryotes, including Chlamydomonas, Tetrahymena and C. elegans, where they participate in the biogenesis of motile or sensory cilia (&lt;&lt;REF:Bacaj-2008-178-989&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;; &lt;&lt;REF:Satish Tammana-2013-126-5018&gt;&gt;). Both CEP104 and TOGARAM1 have evolutionary conserved TOG domains (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). TOGARAM1 contains four TOG domains, two of which can promote tubulin polymerization in light scattering experiments (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;), and the same is true for the single tubulin-binding TOG domain of CEP104 (&lt;&lt;REF:Al-Jassar-2017-25-146&gt;&gt;; &lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Loss of function mutations in TOGARAM1 and CEP104 in multiple organisms result in short cilia (&lt;&lt;REF:Bacaj-2008-178-989&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;; &lt;&lt;REF:Satish Tammana-2013-126-5018&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Similarly, loss of CSPP1, a protein that increases MT stability by binding inside the MT lumen and promoting pausing (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;; &lt;&lt;REF:Wang-2024-22-mjae007&gt;&gt;), results in short cilia (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;). ARMC9 on its own cannot bind to MTs but acts as a scaffold for other ciliary tip module components (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). ARMC9 and TOGARAM1 have been localized to motile cilia, where they were found to have opposite functions in regulating MT length (&lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;), though mutants of both proteins make mammalian primary cilia shorter (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Finally, CSPP1 and CEP104 interact with CCDC66, and co-depletion of CCDC66 with either CSPP1 or CEP104 results in further reduction of ciliary length compared to the removal of either CSPP1 or CEP104 alone (&lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;).\nFigure 1.\nCiliary tip module proteins have distinct effects on MT dynamics in vitro.\n(A) Schematic representation of ciliary tip module members and summary table highlighting individual protein effects on MT dynamics. Jelly-roll, JR; zinc finger, ZNF; MT binding, MTB; pause domain, PD; alpha-helical domain, H.\n(B-C) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488-labelled tubulin or 20 nM GFP-EB3.\n(D-F) Parameters of MT plus-end dynamics in the presence of 15 microM tubulin alone or with 20 nM EB3 in combination with indicated concentrations of proteins (from kymographs shown in B-C, I-P, and S1C). (D) Bars represent pooled data from three independent experiments (growth rate) or averaged means from three independent experiments (pause and block duration), total number of growth events, pauses or blocks: tubulin alone, n=356, 0; 10 nM CCDC66, n=411, 0; 10 nM CEP104, n=306, 0; 100 nM CEP104, n=0, 138; 10 nM CSPP1,n=422, 97; 10 nM TOGARAM1, n=347, 0; EB3 alone, n=938, 0; EB3 with 10 nM CCDC66, n=562, 0; EB3 with 10 nM CEP104, n=213, 101; EB3 with CSPP1, n=1715, 273; EB3 with TOGARAM1, n=861, 0. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test, all conditions were compared to their relevant control (either tubulin alone or tubulin with 20 nM EB3). (E-F) Bars represent averaged means from three independent experiments. Error bars represent s.e.m.. Movies were acquired for 10 minutes, therefore this is the maximum time for pause duration,\n(G-P) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488 or Rhodamine-labelled tubulin, or 20 nM GFP- or mCherry-EB3, and indicated concentrations and colors of ciliary tip module proteins. Orange arrowheads, blocked plus ends; orange arrows, CEP104 tracking minus ends; yellow arrowheads, pauses; blue arrowheads, rescues.\nTo uncover biochemical mechanisms underlying the function of these proteins and their combined regulation of ciliary MTs, we reconstituted in vitro their individual and collective effects on MT dynamics. Although the individual TOG domain of CEP104 was previously shown to promote MT polymerization (&lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;), we found that the full-length protein specifically inhibited MT plus-end elongation and shortening. This activity was potentiated by EB3, CSPP1 and CCDC66, which could all recruit CEP104 to MTs. This was surprising as EB3 and CSPP1 bind to the outer and inner surface of the MT wall, respectively (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;; &lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). TOGARAM1 functioned as a polymerase that converted the inhibition of MT polymerization imposed by CEP104 into slow growth. ARMC9 and CCDC66 behaved as scaffolds, enhancing the effects of their binding partners. The combination of all five ciliary tip module proteins resulted in very slow and processive MT plus-end elongation, an effect that required TOGARAM1 and became less robust when one of the other proteins was left out. MT growth rates in these conditions were in the same range as initial elongation rates of regenerating flagella of single-celled organisms (&lt;&lt;REF:Rosenbaum-1967-34-345&gt;&gt;; &lt;&lt;REF:Rosenbaum-1969-41-600&gt;&gt;; &lt;&lt;REF:Witman-1975-253-178&gt;&gt;). Cryo-electron tomography showed that together, ciliary tip proteins formed a champagne cork-like structure that protruded from MT plus ends and diminished protofilament flaring. Altogether, our findings demonstrate that, through a combination of opposing activities, ciliary tip module components can stabilize MT plus ends and drive their slow but persistent elongation.\n\nResults\n\nCiliary tip module proteins have distinct effects on MT dynamics in vitro\nTo characterize the effects of ciliary tip module proteins on MT polymerization, we N-terminally tagged them with mCherry or GFP and purified them from transiently transfected HEK293 cells (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;, S1A &lt;&lt;FIG:figs1&gt;&gt;). Mass spectrometry-based analysis demonstrated that the protein preparations contained only very minor contaminations with other known regulators of MT dynamics, such as CSPP1 (Fig. S1B &lt;&lt;FIG:figs1&gt;&gt;). In addition, the heat shock protein Hsp70, a protein that to our knowledge has no effect on MT dynamics, was present in the preparations of CCDC66 and CSPP1 (Fig. S1B &lt;&lt;FIG:figs1&gt;&gt;). We used these proteins to perform in vitro reconstitution assays with MTs grown from GMPCPP-stabilized seeds and observed their impact on MT dynamics by Total Internal Reflection Fluorescence (TIRF) microscopy (&lt;&lt;REF:Bieling-2007-450-1100&gt;&gt;; &lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Although axonemes are composed of MT doublets, single MTs are appropriate substrates to study the effects of ciliary tip proteins, because doublets become singlets at ciliary tips (&lt;&lt;REF:Fisch-2011-103-249&gt;&gt;; &lt;&lt;REF:Kiesel-2020-27-1115&gt;&gt;; &lt;&lt;REF:Legal-2023-222-e202301129&gt;&gt;). To detect MTs, we used either fluorescent tubulin or fluorescently tagged EB3, which increases MT growth rate and induces catastrophes (&lt;&lt;REF:Komarova-2009-184-691&gt;&gt;) (Fig. 1B-F &lt;&lt;FIG:fig1&gt;&gt;). Studying the effects of ciliary tip proteins in the presence of EBs is relevant, because EB1 and EB3 localize to axonemal MTs in both primary and motile cilia (&lt;&lt;REF:Kiesel-2020-27-1115&gt;&gt;; &lt;&lt;REF:Pedersen-2003-13-1969&gt;&gt;; &lt;&lt;REF:Schroder-2007-17-1134&gt;&gt;).\nFigure S1 (Related to Figure 1).\nCharacterization of ciliary tip module proteins in vitro.\n(A) Analysis of purified GFP-tagged ciliary tip module proteins by SDS-PAGE. Asterisks indicate the full-length protein bands. Protein concentrations were determined from a BSA standard (not shown).\n(B) Mass spectrometry analysis of purified GFP-tagged ciliary tip module proteins. For all module members except TOGARAM1 the human sequence was used, due to technical difficulties the mouse sequence of TOGARAM1 was used, it is 84% identical to human TOGARAM1.\n(C) Fields of view (top, scale bar 2 microm) and kymograph (bottom, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with HiLyte-488-tubulin and mCherry-CEP104.\n(D) Kymographs illustrating mobility of DmKHC(1-421) on CEP104-blocked MT labelled with HiLyte-488-tubulin and mCherry-CEP104 (top) and DmKHC (bottom) proving that the blocked end of the MT is the plus end. Scale bars 2 microm and 60 s for both kymographs.\nAmong the tested proteins, ARMC9 was the only one that displayed no MT binding even when present at 300 nM (Fig. 1G &lt;&lt;FIG:fig1&gt;&gt; and H &lt;&lt;FIG:fig1&gt;&gt;), in agreement with data in cells (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). CCDC66, which is known to interact with MTs (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;), bound along the MT lattice already at 10 nM and slightly decreased the growth rate, but did not affect the frequencies of catastrophes and rescues and did not induce pausing (Fig. 1D-F, I &lt;&lt;FIG:fig1&gt;&gt; and J &lt;&lt;FIG:fig1&gt;&gt;).\nCEP104 could not autonomously bind MTs at 10 nM in assays with tubulin alone (Fig. S1C &lt;&lt;FIG:figs1&gt;&gt;). However, at 100 nM it efficiently blocked MT growth specifically at the plus end (Fig. 1D-F &lt;&lt;FIG:fig1&gt;&gt; and K &lt;&lt;FIG:fig1&gt;&gt;), as confirmed by including in the assay a constitutively active fragment of the plus-end directed Kinesin-1 DmKHC(1-421) (Fig. S1D &lt;&lt;FIG:figs1&gt;&gt;). In the presence of its binding partner EB3 (&lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;), CEP104 could specifically inhibit MT plus-end polymerization already at 10 nM (Fig. 1D-F &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt;). CEP104 colocalized with EB3 at some growing plus and occasionally also minus ends, however, this did not alter growth rates at the plus end (Fig.1D &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt;). Once MT growth was arrested, no regrowth was detected for the remainder of the experiment (Fig 1E, K &lt;&lt;FIG:fig1&gt;&gt;, and L &lt;&lt;FIG:fig1&gt;&gt;).\nEpisodes of inhibited growth, albeit transient ones, were also observed with CSPP1, as we have shown previously (Fig. 1D-F, M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;) (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Both with and without EB3 CSPP1 preferentially binds at both plus and minus ends when they undergo growth perturbations (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;; &lt;&lt;REF:Wang-2024-22-mjae007&gt;&gt;), and therefore it displayed discreet sites of increased lattice accumulation (Fig. 1M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;). At the plus ends, CSPP1 accumulations decreased growth rate, induced pauses and facilitated transitions from pause to growth, whereas pausing was less obvious at the minus ends due to their overall slower dynamics (Fig. 1D-F, M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;).\nFinally, TOGARAM1 bound to MT plus ends, reduced their growth rate and induced transitions from shrinkage to growth (rescues), and these effects were not altered by the presence of EB3 (Fig. 1D-F, O &lt;&lt;FIG:fig1&gt;&gt; and P &lt;&lt;FIG:fig1&gt;&gt;). A decrease in MT growth rate was unexpected, as TOGARAM1 structurally resembles the MT polymerase ch-TOG/XMAP215 (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;), which strongly accelerates MT polymerization (&lt;&lt;REF:Brouhard-2008-132-79&gt;&gt;). Altogether, we show that all the full-length members of the ciliary tip module can be purified and used for in vitro assays, where they display opposing effects on MT dynamics.\n\nPlus-end blocking by CEP104 depends on the TOG domain and is potentiated by EB3, CCDC66 and CSPP1\nAmong the ciliary tip proteins analyzed above, the effect of CEP104 was the most unexpected as it was thought to be a MT polymerase (&lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;), and we therefore set out to dissect it in more detail. Single molecule counting experiments demonstrated that CEP104 forms a dimer, in agreement with the sedimentation profile of its first helical domain (Fig. 2A &lt;&lt;FIG:fig2&gt;&gt;) (&lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;). In the presence of EB3, ~5-6 CEP104 molecules (2-3 dimers) were sufficient to block MT plus-end growth (Fig. 2B &lt;&lt;FIG:fig2&gt;&gt;). In these conditions, Fluorescence Recovery after Photobleaching (FRAP) showed that CEP104 did not exchange at blocked MT plus ends (Fig. 2C &lt;&lt;FIG:fig2&gt;&gt;), whereas EB3, associated with the same ends, did partially turn over, albeit significantly slower than on growing MT plus ends (Fig. 2D &lt;&lt;FIG:fig2&gt;&gt; and E &lt;&lt;FIG:fig2&gt;&gt;). This could be explained by the direct interaction of EB3 and CEP104 on the MT through the SxIP motif of CEP104, SKIP (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt;) (&lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;). Mutation of this motif has previously been shown to abolish the interactions between the C-terminus of EB and its partners (&lt;&lt;REF:Honnappa-2009-138-366&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Indeed, CEP104 SKNN double mutation prevented EB3-dependent recruitment to MTs; therefore, even in the presence of EB3, 100 nM CEP104 SKNN was needed to block plus-end growth (Fig. 3A-D &lt;&lt;FIG:fig3&gt;&gt;, and S2A &lt;&lt;FIG:figs2&gt;&gt;).\nFigure S2 (Related to Figure 3).\nCharacterization of CEP104 constructs in vitro.\n(A) Analysis of purified GFP-tagged CEP104 constructs by SDS-PAGE. Asterisks show protein bands. Protein concentrations were determined from a BSA standard (not shown).\n(B) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with either HiLyte-488-tubulin or GFP-EB3 and indicated concentrations and colors of CEP104 constructs.\n(C) Co-immunoprecipitation of either CSPP1 (left) or CCDC66 (right) with indicated CEP104 constructs, both CSPP1 and CCDC66 interact with the jelly-roll domain of CEP104.\n(D) Analysis of purified GFP-tagged CEP104 DeltaJR construct by SDS-PAGE. Asterisk indicates the full-length protein band. Protein concentration was determined from a BSA standard (not shown).\nFigure 2.\nA few CEP104 molecules stably block MT plus ends\n(A) Fields of view (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules, GFP-EB3 dimers, and GFP-CEP104 dimers immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=29981; GFP-EB3, n=55378; GFP-CEP104, n=32335. Scale bar 2 microm.\n(B) Representative MT with CEP104-blocked plus end (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules and GFP-CEP104 intensity at blocked plus end immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=13925; GFP-CEP104, n=92. Scale bar 2 microm.\n(C-E) FRAP analysis of CEP104 (C) and EB3 (D) at blocked MT plus ends, or dynamic EB3 at growing plus ends (E). Arrowhead marks point of photobleaching in representative kymographs, scale bars 2 microm and 60 s (C), 2 microm and 10 s (D, E). Plots show averaged curves with exponential fit. Number of FRAP measurements: CEP104 n=17; stationary EB3 n=14; dynamic EB3 n=10. Error bars represent s.e.m..\nFigure 3.\nPlus-end blocking by CEP104 is potentiated by EB3, CCDC66 and CSPP1.\n(A) Schematic representation of different CEP104 constructs; jelly-roll, JR; alpha-helical domain, H; zinc finger, ZNF.\n(B) Percentage of time MT plus ends spent blocked in the presence of 20 nM EB3 in combination with CEP104 constructs (from kymographs shown in C-E, 1C, 1L). Bars represent averaged means from three independent experiments. Error bars represent s.e.m..\n(C-F) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488 or Rhodamine-labelled tubulin or, 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module constructs. Orange arrowheads, blocked plus ends; white arrowheads, CEP104 DeltaTOG at plus ends of seeds; white arrows, CEP104 tracking the minus end.\n(G) Parameters of MT plus-end dynamics in the presence of 15 microM tubulin alone or with 20 nM EB3 in combination with indicated concentrations of ciliary tip module proteins (from kymographs shown in F, H-L, S2B &lt;&lt;FIG:figs2&gt;&gt;). For dynamic state bars represent averaged means from three independent experiments. For depolymerization rate bars represent pooled data from three independent experiments: 2 nM CEP104, n=133; 2 nM CEP104 with 15 nM CCDC66, n=23; 10 nM CEP104 with 15 nM CCDC66, n=122; 2 nM CEP104 with 10 nM CSPP1, n=69; 10 nM CEP104 with 10 nM CSPP1, n=99; EB3 with 2 nM CEP104, n=1; EB3 with 2 nM CEP104 and 15 nM CCDC66, n=63; EB3 with 2 nM CEP104 and 10 nM CSPP1, n=116.\n(H-N) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte-488- or Rhodamine-labelled tubulin, or 20 nM GFP or mCherry-EB3 and indicated concentrations and colors of ciliary tip module constructs. Light orange arrowheads, blocked seeds; dark orange arrowhead, blocked lattice; blue arrowheads, slow depolymerization of plus ends.\n(O) Percentage of time of MT plus ends spent blocked at the seed in the presence of 20 nM EB3 in combination with CEP104 DeltaJR and other ciliary tip module proteins (from kymographs shown in M, N, P, S2B &lt;&lt;FIG:figs2&gt;&gt;). Bars represent averaged means from three independent experiments.\n(P) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- and 50 nM mCherry-CEP104 DeltaJR. Orange arrowheads, blocked plus ends.\n(Q) Schematic model showing CEP104 domain functions and interactions with other ciliary tip module proteins. Jelly-roll, JR; zinc finger, ZNF; alpha-helical domain, H.\nFor (B, G and O) error bars represent s.e.m. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\nWhereas the SxIP motif was not essential for plus-end blocking by CEP104, the single TOG domain of CEP104 was required to inhibit MT growth. CEP104 lacking the TOG domain (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt; and S2A &lt;&lt;FIG:figs2&gt;&gt;) could still be recruited to MTs by EB3, but no plus-end blocking was observed (Fig. 3B &lt;&lt;FIG:fig3&gt;&gt; and E &lt;&lt;FIG:fig3&gt;&gt;). In the presence of EB3, CEP104 DeltaTOG could bind to growing MT ends and accumulate at the border between the seed and the plus-end-grown lattice (Fig. 3E &lt;&lt;FIG:fig3&gt;&gt;), a localization for which we currently have no explanation.\nSince recent studies described functional and biochemical interactions between CEP104, CCDC66, CSPP1 and ARMC9 (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;), we next tested in vitro the combinations of these proteins. We found that in the presence of 15 nM CCDC66, already 2 nM CEP104 was sufficient to occasionally block MT plus end outgrowth from the seed (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt; and G &lt;&lt;FIG:fig3&gt;&gt;). 2 nM CEP104 on its own, or in the presence of EB3, or together with ARMC9, showed hardly any MT binding (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;). However, CEP104 driven growth inhibition at MT seeds became much more common at 10 nM CEP104 with 15 nM CCDC66, and in these conditions, CEP104 and CCDC66 could also strongly inhibit depolymerization of dynamic MTs by either pausing the plus ends or reducing their shrinkage rate from 36.91+/-1.34 microm/min to 0.12+/-0.02 microm/min (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt; and H &lt;&lt;FIG:fig3&gt;&gt;). Similar effects were observed when either 2 nM or 10 nM CEP104 was combined with 10 nM CSPP1: both growth inhibition at the seed and at the dynamic plus ends, as well as slow depolymerization were observed (Fig. 3G, I &lt;&lt;FIG:fig3&gt;&gt; and J &lt;&lt;FIG:fig3&gt;&gt;). These data indicate that CEP104 is not just a MT growth inhibitor or a depolymerase that can prevent tubulin addition to the plus ends, but also a specific MT stabilizing factor that can inhibit tubulin loss from MT plus ends lacking a GTP cap. The addition of EB3 had no strong impact on MT dynamics when combined with 2 nM CEP104 and 15 nM CCDC66 (Fig. 3F, G &lt;&lt;FIG:fig3&gt;&gt; and K &lt;&lt;FIG:fig3&gt;&gt;), but potentiated seed blocking when included with 2 nM CEP104 and 10 nM CSPP1, as most MTs were blocked at the seed in these conditions (Fig. 3G, I &lt;&lt;FIG:fig3&gt;&gt; and L &lt;&lt;FIG:fig3&gt;&gt;).\nBased on our own and previously published co-immunoprecipitation experiments, the interactions of CCDC66 and CSPP1 with CEP104 depended on its jelly-roll (JR) domain (Fig. S2C &lt;&lt;FIG:figs2&gt;&gt;) (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;). The deletion of this domain abolished the ability of both proteins to recruit CEP104 to MTs (Fig. 3M-O &lt;&lt;FIG:fig3&gt;&gt; and S2D &lt;&lt;FIG:figs2&gt;&gt;). However, CEP104 DeltaJR could still block seed outgrowth in the presence of EB3 (Fig. 3O &lt;&lt;FIG:fig3&gt;&gt; and P &lt;&lt;FIG:fig3&gt;&gt;), though a higher concentration of CEP104 DeltaJR compared to the full-length CEP104 was needed to achieve significant plus-end blocking (50 nM versus 10 nM, compare Fig. 1D &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt; to Fig. 3O, P &lt;&lt;FIG:fig3&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;), indicating that the JR domain might directly or indirectly contribute to MT blocking.\nTaken together, our results demonstrate that CEP104 can stably and specifically associate with MT plus ends and inhibit both tubulin addition and removal from these ends in a TOG domain-dependent manner. However, the affinity of CEP104 for MTs is rather low, and its activity is strongly enhanced by association with its binding partners EB3, CCDC66 and CSPP1 that recruit it to MTs. Since EB3 binds to the outer MT surface (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;) and the same might be true for CCDC66 as it can bundle MTs (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;), whereas CSPP1 associates with the MT lumen (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;), CEP104 might be positioned at protofilament ends to allow access to both surfaces of the MT wall (Fig. 3Q &lt;&lt;FIG:fig3&gt;&gt;).\n\nThe rescue activity of TOGARAM1 depends on TOG3 and TOG4 domains\nHaving characterized the activity of CEP104, we next turned to the other TOG domain component of the ciliary tip module, TOGARAM1, which contains two pairs of TOG domains (TOG1-TOG4) connected by a long linker (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Single molecule counting experiments showed that it is a monomer (Fig. 4B &lt;&lt;FIG:fig4&gt;&gt;), similar to other MT regulators with multiple TOG domains, XMAP215 (&lt;&lt;REF:Brouhard-2008-132-79&gt;&gt;) and CLASP2 (&lt;&lt;REF:Aher-2018-46-40&gt;&gt;). A previous study showed that isolated TOG2 and TOG4, but not TOG1 and TOG3 promoted tubulin polymerization in light scattering assays (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). To test how these domains contribute to the rescue activity of the full-length protein, we took advantage of previously generated TOGARAM1 mutants where the tubulin-binding surfaces of the individual TOG domains were disrupted by point substitutions within the intra-HEAT loop of HEAT repeat A (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;) (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Although there is no direct interaction between TOGARAM1 and EB3, we included EB3 in our in vitro reconstitution assays as EB3 increases catastrophe frequency and therefore facilitates observing rescues (Fig 1F &lt;&lt;FIG:fig1&gt;&gt;) (&lt;&lt;REF:Komarova-2009-184-691&gt;&gt;). We found that simultaneous mutation of TOG3 and TOG4 (TOGARAM1 123'4'), but not TOG1 and TOG2 (TOGARAM1 1'2'34) abolished the rescue activity, although both mutants could still bind to MTs at 10 nM concentration (Fig. 4A, C-E &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). This was different from previous observations in cells, where the double TOG3-TOG4 mutant (TOGARAM1 123'4') could not bind to MTs (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). TOG3 and TOG4 showed redundancy within the full-length protein because single mutations in each one of them did not abolish rescues (Fig. 4A, C, F, G &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). TOG1 and TOG2 could be deleted without impairing rescue activity (the construct L-TOG-34, Fig. 4A, C, H &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;), and the combination of these two domains (TOG-12) without the adjacent linker region did not bind to MTs even at 200 nM (Fig. 4A, I &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). TOG-34 did not visibly associate with MTs at 10 nM, but at 200 nM it bound to MTs and induced rescues (Fig. 4A, C, J, K &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). Since the construct containing TOG3 and TOG4 domains together with adjacent linker (L-TOG-34) was active in our assays already at 10 nM, in agreement with previous findings, the linker region of TOGARAM1 contributes to MT affinity (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). All tested constructs mildly reduced MT growth rate and TOG-34 constructs with or without the adjacent linker mildly suppressed catastrophes (Fig. 4C &lt;&lt;FIG:fig4&gt;&gt;). Our results demonstrate that, on its own, TOGARAM1 has relatively mild effects on MT dynamics, which rely in its two C-terminal TOG domains.\nFigure S3 (Related to Figure 4).\nCharacterization of TOGARAM1 constructs in vitro.\nAnalysis of purified GFP-tagged TOGARAM1 constructs by SDS-PAGE. Asterisks show protein bands. Protein concentrations were determined from a BSA standard (not shown).\nFigure 4.\nThe rescue activity of TOGARAM1 depends on the TOG3 and TOG4 domains.\n(A) Schematic representation of different TOGARAM1 constructs and summary table highlighting their effects on MT dynamics. Vertical lines indicate point mutations predicted to ablate tubulin-binding activity.\n(B) Fields of view (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules, EB3 dimers and TOGARAM1 molecules were immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=57865; EB3, n=73074; TOGARAM1, n=74306. Scale bar, 2 microm.\n(C) Parameters of MT plus-end dynamics in the presence of 20 nM EB3 in combination with indicated concentrations of TOGARAM1 constructs (from kymographs shown in D-H, K, 1C, 1P). For growth rates bars represent pooled data from three independent experiments, total number of growth events: EB3 alone, n=938; EB3 with TOGARAM1, n=861; EB3 with 1'2'34, n=350; EB3 with 123'4', n=337; EB3 with 123'4, n=560; EB3 with 1234', n=253; EB3 with L-TOG-34, n=131; EB3 with 200nM TOG-34, n=150. For transition frequencies bars represent average means from three independent experiments. Error bars represent s.e.m., ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(D-K) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of TOGARAM1 constructs. Blue arrowheads, rescues.\n\nARMC9 colocalizes with TOGARAM1 and CSPP1 and enhances their effects on MT dynamics\nWe next set out to characterize the function of ARMC9. Single molecule counting experiments showed that ARMC9 forms dimers dependent on its central helical region (Fig. S4A &lt;&lt;FIG:figs4&gt;&gt; and B &lt;&lt;FIG:figs4&gt;&gt;). As shown above, ARMC9 does not bind to MTs on its own nor in the presence of CEP104 and EB3 (Fig. 1G, H &lt;&lt;FIG:fig1&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;). Consistently, no direct interaction has been described for ARMC9 and CEP104, even though they were previously shown to weakly co-immunoprecipitate with each other (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;), an observation we were not able confirm (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt;). We could confirm by co-immunoprecipitation the interaction between ARMC9 and CCDC66, however when this combination was tested in reconstitution experiments we did not see any recruitment of ARMC9 to the MT lattice (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt; and D &lt;&lt;FIG:figs4&gt;&gt;). We also confirmed by co-immunoprecipitation the interaction between ARMC9 and TOGARAM1 (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt;) and refined the previous mapping of this interaction (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;) by showing that it requires the armadillo (ARM) repeats of ARMC9, but not its helical domain responsible for dimerization, and the tubulin-binding surface of the TOG2 domain of TOGARAM1 (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, S4A, E &lt;&lt;FIG:figs4&gt;&gt; and F &lt;&lt;FIG:figs4&gt;&gt;). We also tested the impact of different point mutations identified in Joubert syndrome patients on the interaction of ARMC9 and TOGARAM1 (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;), and found that all tested mutations in the TOG2 domain (all located in the first HEAT repeat) and one of the several known mutations in the ARM domain (G492R) perturb binding (Fig. S4A, G &lt;&lt;FIG:figs4&gt;&gt; and H &lt;&lt;FIG:figs4&gt;&gt;), supporting the functional importance of this interaction.\nFigure S4 (Related to Figure 5).\nMapping of ARMC9 and TOGARAM1 interaction.\n(A) Schematic representation of different ARMC9 constructs.\n(B) Histogram plots of fluorescent intensities of single GFP molecules, GFP-EB3 dimers, and full-length GFP-ARMC dimers (left) or GFP-ARMC9 DeltaH molecules immobilized in separate chambers of the same coverslips. Number of molecules analyzed, left, right: GFP, n=18415, 14914; GFP-EB3, n= 132826, 163959; GFP-ARMC9 FL, n=99873; GFP-ARMC9 DeltaH, 55479.\n(C) Co-immunoprecipitation of full-length ARMC9 with indicated proteins.\n(D) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins.\n(E-F) Co-immunoprecipitation experiments of either full-length TOGARAM1 with indicated ARMC9 constructs (E) or full-length ARMC9 with indicated TOGARAM1 point mutations\n(F). ARMC9 and TOGARAM1 interact through ARM domain and TOG2 domain, respectively.\n(G-H) Co-immunoprecipitation experiments of either full-length TOGARAM1 with indicated ARMC9 Joubert syndrome mutations (G) or full-length ARMC9 with indicated TOGARAM1 Joubert syndrome mutations (H).\nIn the in vitro assays, TOGARAM1 could recruit ARMC9 to MTs, and the colocalization between the two proteins was particularly prominent in particles that could stationarily bind to or diffuse on MTs (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt;). As expected from the co-immunoprecipitation experiments, this colocalization was not perturbed by the 123'4' mutant of TOGARAM1 (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, 5B &lt;&lt;FIG:fig5&gt;&gt;) but was abolished by the 1'2'34 mutant (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, 5C &lt;&lt;FIG:fig5&gt;&gt;). The brightness of ARMC9 and TOGARAM1 in MT-bound particles increased over time, indicating binding of additional molecules (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt; and B &lt;&lt;FIG:fig5&gt;&gt;). This binding was likely driven by ARMC9, as it was not observed for TOGARAM1 alone (Fig. 1P &lt;&lt;FIG:fig1&gt;&gt;). Presence of ARMC9 did not drastically alter MT growth rates but did cause a roughly two-fold increase in the rescue activity of TOGARAM1, which was still dependent on TOG3 and TOG4 (Fig. 5A, B &lt;&lt;FIG:fig5&gt;&gt;, and D &lt;&lt;FIG:fig5&gt;&gt;). In contrast to ARMC9, the addition of CCDC66 did not alter the effects of TOGARAM1 on MT dynamics, and no colocalization between CCDC66 and TOGARAM1 was observed (Fig. S4D &lt;&lt;FIG:figs4&gt;&gt;).\nFigure 5.\nARMC9 colocalizes with TOGARAM1 and CSPP1 and enhances their effects on MT dynamics.\n(A-C) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCH-EB3 and indicated concentrations and colors of TOGARAM1 constructs and ARMC9. Blue arrowheads, ARMC9 and TOGARAM1 colocalization.\n(D) Parameters of MT plus-end dynamics in the presence of 20 nM EB3 in combination with indicated concentrations of TOGARAM1 constructs and ARMC9 (from kymographs shown in A-B, 1C, 1P, 4E). For growth rates bars represent pooled data from three independent experiments, total number of growth events: EB3 alone, n=938; EB3 with TOGARAM1, n=861; EB3 with TOGARAM1 and ARMC9, n=342; EB3 with 123'4', n=337; EB3 with 123'4' and ARMC9, n=325. For transition frequencies bars represent averaged means from three independent experiments. Error bars represent s.e.m.. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(E) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP-EB3 and ciliary tip module proteins indicated. Yellow arrowheads, pauses.\n(F) Parameters of MT plus-end dynamics with 20 nM EB3 in combination with indicated concentrations of ciliary top module proteins (from kymographs shown in E, 1C, 1N). For growth rate and pause duration bars represent pooled data from three independent experiments, total number of growth events, pauses: EB3 alone, n=938, 0; EB3 with CSPP1, n=1715, 273; EB3 with CSPP1 and ARMC9, n=627, 315. For transition frequencies and dynamic state bar represent averaged means from three independent experiments. Error bars represent s.e.m., ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(G) Schematic model showing TOGARAM1, ARMC9, and CSPP1 interaction domains; MTB; pause domain, PD; alpha-helical domain, H.\nWe observed co-immunoprecipitation of ARMC9 and CSPP1 (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt; and B &lt;&lt;FIG:figs5&gt;&gt;), confirming previous observations (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). The interaction required the C-terminal part of CSPP1 and the linker regions surrounding the central helical domain of ARMC9, though not the helical domain itself (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt; and B &lt;&lt;FIG:figs5&gt;&gt;). In vitro reconstitution assays showed that CSPP1 could trigger ARMC9 accumulation on MTs in the vicinity of the growing MT ends (Fig. 5E &lt;&lt;FIG:fig5&gt;&gt;) and mildly increased the percentage of time CSPP1-induced MT pausing (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;).\nFigure S5 (Related to Figure 5).\nMapping of ARMC9 and CSPP1 interaction.\n(A) Co-immunoprecipitation experiment of full-length ARMC9 with indicated CSPP1 constructs illustrated in schematic below.\n(B) Co-immunoprecipitation experiment of full-length CSPP1 with indicated ARMC9 constructs illustrated in Figure S4A &lt;&lt;FIG:figs4&gt;&gt;.\n(C) Co-immunoprecipitation experiment of full-length CSPP1 with full-length TOGARAM1 shows no interaction.\n(D) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins.\nWe also tested whether CSPP1 and TOGARAM1 can bind to each other but detected no co-immunoprecipitation (Fig. S5C &lt;&lt;FIG:figs5&gt;&gt;), and CSPP1 displayed no colocalization with either TOGARAM1 or CCDC66 in the in vitro assays (Fig. S5D &lt;&lt;FIG:figs5&gt;&gt;). Altogether, our results indicate that CCDC66 does not colocalize with either TOGARAM1 or CSPP1 in reconstitution assays, whereas ARMC9 can bind to both of these proteins through two distinct domains and serve as a scaffold mildly enhancing their effects on MT dynamics (Fig. 5G &lt;&lt;FIG:fig5&gt;&gt;).\n\nCombined action of ciliary tip proteins drives slow processive MT growth\nTo complete the exploration of all pairwise combinations of the studied ciliary tip proteins, we examined the joint effects of CEP104 and TOGARAM1. The two proteins coprecipitated with each other in a manner dependent on the zinc finger domain of CEP104 and the linker region of TOGARAM1 (Fig. S6A &lt;&lt;FIG:figs6&gt;&gt; and B &lt;&lt;FIG:figs6&gt;&gt;). When they were combined on dynamic MTs in the presence of EB3, there were periods of pausing both at the plus end of the seeds and of the dynamic lattice (Fig 6A-C &lt;&lt;FIG:fig6&gt;&gt;). However, these pauses were transient and followed by growth events (Fig 6D &lt;&lt;FIG:fig6&gt;&gt; and E &lt;&lt;FIG:fig6&gt;&gt;), something we did not see with any other CEP104-ciliary tip protein combination. Furthermore, the two proteins together induced periods of very slow MT polymerization with the rates of 0.12+/-0.01 microm/min, ~20 times slower than 2.27+/-0.02 microm/min observed with TOGARAM1 alone (Fig. 6A, C &lt;&lt;FIG:fig6&gt;&gt; and D &lt;&lt;FIG:fig6&gt;&gt;). CEP104 and TOGARAM1 can thus interact through regions that do not engage their tubulin-binding domains and together impose slow MT polymerization (Fig. 6F &lt;&lt;FIG:fig6&gt;&gt;).\nFigure S6 (Related to Figure 6).\nMapping of CEP104 and TOGARAM1 interaction.\n(A-B) Co-immunoprecipitation of either wildtype CEP104 with indicated TOGARAM1 constructs (A) or wildtype TOGARAM1 with indicated CEP104 constructs (B), the two proteins interact between the linker of TOGARAM1 and the zinc finger of CEP104.\n(C) Kymographs illustrating mobility of DmKHC(1-421) on slow growing MT with all ciliary tip module proteins (top) and DmKHC (bottom) proving that the slow growing end of the MT is the plus end. Scale bars 2 microm and 60 s for both kymographs.\nFigure 6.\nCombined action of ciliary tip proteins drives slow processive MT growth.\n(A) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with GFP-EB3, mCherry-TOGARAM1, and GFP-CEP104. Orange arrowheads, blocked plus end; green arrowheads, slow MT growth.\n(B-E) Parameters of MT plus-end dynamics in the presence of EB3 in combination with indicated concentrations of ciliary tip module proteins (from kymographs shown in A, 1C, 1L, 1P). For pause &amp; block duration (B) and growth rates (D) data was pooled from three independent experiments, total number of growth events, pauses: EB3 alone, n=938, 0; EB3 with CEP104, n=213, 101; EB3 with TOGARAM1, n=861, 0; EB3 with CEP104 and TOGARAM1, n=373, 140. Bars represent mean + s.e.m.. For dynamic state (C) and transition frequencies (E) bars represent averaged means from three independent experiments. Error bars represent s.e.m.\n(F) Schematic model showing interactions between CEP104, TOGARAM1 and tubulin. Jelly-roll, JR; zinc finger, ZNF; alpha-helical domain, H.\n(G-H) Weighted growth rates (G) and dynamic state (H) for combinations of ciliary tip module proteins indicated (from kymographs I-N, and 1C). For growth rates (G) data was pooled from three independent experiments, total number of growth events: EB3 alone, n=938; entire ciliary tip module, n=70; no ARMC9, n=81; no CCDC66, n=83; no CEP104, n=41; no CSPP1, n=141; no TOGARAM1, n=0. For dynamic state (H) bars represent or averaged means from two independent experiments, error bars represent s.e.m..\n(I-N) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins indicated. Orange arrow, blocked plus end; green arrows, slow plus-end growth.\nNext, we combined all five ciliary tip module components in the presence of EB3. Strikingly, in these conditions, all MTs displayed slow and highly processive plus-end polymerization with occasional pausing, with an average rate of 0.19+/-0.04 microm/min, while minus ends could still undergo phases of growth and shrinkage (Fig. 6G-I &lt;&lt;FIG:fig6&gt;&gt; and S6C &lt;&lt;FIG:figs6&gt;&gt;). To test which proteins are essential for this dynamic state, we repeated the assays while systematically leaving out each of the ciliary tip proteins. Omitting TOGARAM1 abolished slow growth completely, as all MT plus ends were paused (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt; and J &lt;&lt;FIG:fig6&gt;&gt;). This was not surprising, because MT behavior in these conditions was dominated by the two pausing factors, CEP104 and CSPP1. This indicates that an important function of TOGARAM1 is to overcome MT growth inhibition imposed by CEP104 or CSPP1. Leaving out CEP104 had no major effect, except for a slight increase in pausing and very occasional periods of fast growth (Fig. 6G, H &lt;&lt;FIG:fig6&gt;&gt; and K &lt;&lt;FIG:fig6&gt;&gt;), whereas fast elongation became much more common in the absence of CSPP1 (Fig. 6G, H &lt;&lt;FIG:fig6&gt;&gt;, and L &lt;&lt;FIG:fig6&gt;&gt;). Omitting either ARMC9 or CCDC66 had no major effect except that no pausing was observed, and occasional episodes of fast growth were detected (Fig. 6G, H, M &lt;&lt;FIG:fig6&gt;&gt; and N &lt;&lt;FIG:fig6&gt;&gt;). No MT catastrophes or shrinkage were observed in any of these conditions (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt;). Altogether, ciliary tip module components stabilize MTs by preventing their depolymerization and promoting robust but very slow tubulin addition.\n\nCiliary tip proteins form cork-like densities at MT plus ends and reduce protofilament flaring\nTo better understand the localization of the ciliary tip module on MT ends, we turned to cryo-electron tomography (cryo-ET). We reconstructed 3D volumes containing MTs grown with either EB3 alone or in the presence of the entire ciliary tip module. To best match our in vitro reconstitution conditions, samples were incubated at 30 degreesC with GMPCPP-stabilized seeds and 15 microM tubulin for 10 minutes before being added to holey carbon-coated copper grids and vitrified. Based on our analysis of MT dynamics we presumed that they were mostly fast growing for EB3 controls and elongating slowly with a few paused ends for ciliary tip module samples (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt;). MTs grown in the presence of the ciliary tip module showed densities at their ends reminiscent of champagne corks, whereas MTs grown in the presence of only EB3 had no clear density at their ends (Fig. 7A-D &lt;&lt;FIG:fig7&gt;&gt;, S7A &lt;&lt;FIG:figs7&gt;&gt; and B &lt;&lt;FIG:figs7&gt;&gt;). Despite the density for the ciliary tip module appearing quite varied in structure, it was always situated in both the lumen and on the tips of the MT protofilaments (Fig. 7B, D &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;). We used 2D rotational averaging to resolve MT polarity based on chirality of the cross-section (&lt;&lt;REF:Bouchet-Marquis-2007-99-45&gt;&gt;), through which we determined that cork-like electron densities were specific for MT plus ends (Fig. 7C, D &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;). 36 out of 40 captured \"corks\" where confidently assigned to plus ends, the polarity of the other 4 MTs remains undetermined due to poor tilt-alignment of the tomograms. We performed semi-automated segmentation of the tomographic volumes, both for visualization and segregation of the MT protofilaments and ciliary tip module (Fig. 7E &lt;&lt;FIG:fig7&gt;&gt; and F &lt;&lt;FIG:fig7&gt;&gt;). It proved difficult to segregate all individual protofilaments from each other for further analysis due to the limited tilt range. However, the majority of protofilaments were individually resolved, and separation between the ciliary tip module and protofilaments was often clear. Extensive contacts on the luminal surface of tubulin and the exposed protofilament ends were observed, while the exterior surface of the MT remained largely unbound.\nFigure S7 (Related to Figure 7).\nCharacterization of MT plus ends by cryo-ET\n(A, B) Representative examples of MT plus ends grown in presence of EB3 alone (A) or ciliary tip module and EB3 (B). Per MT, the following is shown from left to right; a slice of the denoised tomogram containing the MT plus end, an 8 nm thick transverse cross-section accompanied by rotational averaging analysis to determine MT polarity, and the corresponding 3D model of manual protofilament tracing accompanied by its transverse cross-section. Scale bars 25 nm.\nFigure 7.\nCiliary tip proteins form cork-like densities at MT plus ends and reduce protofilament flaring\n(A, B) Slices (4.3 nm thick) of denoised tomograms showing MT plus ends grown in presence of EB3 alone (A) or ciliary tip module and EB3 (B). Scale bars 25 nm.\n(C, D) 8nm thick transverse cross-section of the corresponding MTs in panels A and B. The transverse cross-section (top) is accompanied by rotational average (bottom) to indicate MT polarity. Scale bars 25 nm.\n(E, F) 3D rendered segmentation volume of a free/dynamic MT plus end and ciliary tip module-plugged plus end. The MT is shown in grey and the ciliary tip module density in green.\n(G-J) 3D model of manually traced protofilament shapes (G, H), accompanied by their transverse cross-sections (I, J).\n(K) Distribution of protofilament length measured from the last segment within the MT wall until the tip of the protofilament. Total number of protofilaments analyzed; EB3, n=329; CTM, n=352. Lines represent median (solid) and quartiles (dotted). ****, p&lt;0.0001; Mann-Whitney test.\n(L) Protofilament standard deviation per MT. Total number of MTs analyzed: EB3, n=26; ciliary tip module, n=26. Bars represent mean, with error bars representing s.e.m., ****, p&lt;0.0001 (Mann-Whitney test).\n(M) Schematic representation of parameters that were obtained from manual tracing of MT plus-end protofilaments.\n(N) Average local protofilament curvature per MT. Total number of MTs analyzed: EB3, n=25; ciliary tip module, n=25. Bars represent mean, with error bars representing s.e.m., *, p&lt;0.05; Mann-Whitney test.\n(O) MT raggedness plotted as standard deviation of protofilament axial distance of last segment within the wall per MT. Total number of MTs analyzed: EB3, n=26; ciliary tip module, n=26. Bars represent mean, with error bars representing s.e.m., **, p&lt;0.01; Mann-Whitney test.\n(P) Schematic model of the ciliary tip module at a MT plus end.\nWe turned to manual tracing to analyze how the ciliary tip module may be altering MT tip architecture, and how this architecture deviates from more typical MT architecture associated with fast dynamics (Fig. 7G-J &lt;&lt;FIG:fig7&gt;&gt;, S7A &lt;&lt;FIG:figs7&gt;&gt; and B &lt;&lt;FIG:figs7&gt;&gt;). We first looked at how flared the MT protofilaments were by measuring both protofilament length (length of curved segment) and protofilament standard deviation per MT (standard deviation of the distance between first point that deviated from being completely straight to the end of the protofilament, see materials and methods) (Fig. 7K-M &lt;&lt;FIG:fig7&gt;&gt;). As expected, MT plus ends grown with only EB3 showed primarily flared protofilaments (&lt;&lt;REF:Gudimchuk-2020-11-3765&gt;&gt;; &lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;) (Fig. 7A, K, L &lt;&lt;FIG:fig7&gt;&gt; and S7A &lt;&lt;FIG:figs7&gt;&gt;), whereas plus ends \"corked\" with ciliary tip module proteins had a dramatic decrease in both protofilament length and standard deviation in protofilament length per MT (Fig. 7B, K, L &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;), as well as a mild decrease in average local curvature (Fig. 7N &lt;&lt;FIG:fig7&gt;&gt;). Finally, MT raggedness, defined as the standard deviation in the axial distance along the MT of the first point for each protofilament that deviated from being completely straight (Fig. 7M &lt;&lt;FIG:fig7&gt;&gt;), was modestly but significantly decreased in MT plus ends \"corked\" by ciliary tip module proteins (Fig. 7O &lt;&lt;FIG:fig7&gt;&gt;). Together our results indicate a potential mechanism where the ciliary tip module acts from both the lumen and tips of MTs to stabilize slow and processive elongation (Fig 7P &lt;&lt;FIG:fig7&gt;&gt;).\n\nDiscussion\nCiliary tip proteins play important roles in controlling the structure and therefore function of motile and primary cilia. Mutations in these proteins cause various ciliopathies often resulting from shorter cilia causing defects in cilia-dependent signaling pathways (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;). Axonemal MTs are stable and elongate very slowly without undergoing long depolymerization events. Here, we have reconstituted these behaviors in vitro with five components of the ciliary tip module. We found that collectively, these proteins robustly impose very slow and processive MT growth. Furthermore, elongation rates measured in our assays, 0.19+/-0.04 microm/min, fall within the range of those measured for the initial elongation of regenerating flagella in single-celled organisms such as Chlamydomonas reinhardtii (0.08-0.40 microm/min) (&lt;&lt;REF:Marshall-2005-16-270&gt;&gt;; &lt;&lt;REF:Rosenbaum-1967-34-345&gt;&gt;; &lt;&lt;REF:Rosenbaum-1969-41-600&gt;&gt;; &lt;&lt;REF:Witman-1975-253-178&gt;&gt;), though these rates are reduced when flagella reach their normal length. However, it remains to be determined if our measured growth rates are within the range of elongating primary cilia.\nSlow growth is an unusual state for dynamic MTs, because it is incompatible with the formation of a long stabilizing GTP cap, which is needed to avoid MT disassembly (&lt;&lt;REF:Desai-1997-13-83&gt;&gt;) and promote growth (&lt;&lt;REF:Wieczorek-2015-17-907&gt;&gt;). Factors controlling slow MT polymerization must thus inhibit tubulin detachment and promote tubulin addition at MT plus ends that have only a very short or no GTP cap. We found that several ciliary tip proteins can perform these functions. Two of them, TOGARAM1 and CEP104, were predicted to be MT polymerases because their individual TOG domains can bind tubulin and enhance tubulin polymerization in vitro\n(&lt;&lt;REF:Al-Jassar-2017-25-146&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). However, full-length TOGARAM1 and CEP104 do not accelerate tubulin polymerization at freely growing MT plus ends. On the contrary, TOGARAM1 somewhat slows down MT growth rate, and CEP104 potently blocks MT elongation. Still, these proteins are MT stabilizers rather than depolymerases as both can inhibit tubulin detachment from GDP-bound MT ends: TOGARAM1 induces rescues, whilst CEP104 dramatically slows down disassembly of MT plus ends. Moreover, TOGARAM1 can promote tubulin addition and thus serve as a polymerase at paused MT ends, and together CEP104 and TOGARAM1 can induce phases of very slow polymerization.\nThe processive slow MT growth reconstituted by either CEP104 and TOGARAM1 together, or the ciliary tip module as a whole, is unique to the plus ends of MTs. This is likely driven by the two TOG domain-containing module members, because specific orientation of the TOG domains can position tubulin dimers favorably for binding to protofilament tips at the plus end (&lt;&lt;REF:Ayaz-2012-337-857&gt;&gt;). Generally, the interaction of TOG domain proteins with MTs depends on positively charged intrinsically disordered regions (&lt;&lt;REF:Aher-2018-46-40&gt;&gt;; &lt;&lt;REF:Byrnes-2017-216-1641&gt;&gt;; &lt;&lt;REF:Widlund-2011-108-2741&gt;&gt;) and additional adaptors, such as EB1 and SLAIN2 (&lt;&lt;REF:Li-2011-193-973&gt;&gt;; &lt;&lt;REF:van der Vaart-2011-193-1083&gt;&gt;). This is also true for the ciliary TOG domain regulators. TOGARAM1, similar to XMAP215 (&lt;&lt;REF:Widlund-2011-108-2741&gt;&gt;), contains a linker region which promotes binding to MTs in vitro, and is further recruited to MTs by ARMC9. CEP104 depends on interactions with several binding partners, EB3, CSPP1 and CCDC66. EB3 is known to bind to the outer MT surface (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;), whereas CSPP1 is a luminal protein (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The binding mode of CCDC66 has not yet been clarified, but it is likely that some of its parts are present on MT surface as it can bundle MTs in vitro (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;). With binding partners on both sides of the MT wall, CEP104 is in a good position to span the MT tip. TOGARAM1 is likely positioned very close to protofilament ends, since it can catalyze tubulin addition. Therefore, ARMC9 is another good candidate for connecting across MT surfaces as it not only interacts and promotes TOGARAM1 activity but also the pausing behavior of luminal protein CSPP1. Collectively, this fits with the cryo-ET data, which show that the complex of ciliary tip proteins localizes only at the plus ends of MTs and is associated with the luminal side (likely CSPP1) and the tip of MTs (likely CEP104, TOGARAM1 and CCDC66), resembling a cork on a champagne bottle.\nThe interactions between ciliary tip proteins depend at least partially on non-overlapping domains: for example, CEP104 binds to tubulin through its TOG domain, to EB3 through the SxIP motif within its flexible C-terminal region, to TOGARAM1 through the zinc finger domain and to CCDC66 and CSPP1 through the jelly-roll domain (Fig 3Q &lt;&lt;FIG:fig3&gt;&gt;, 6F &lt;&lt;FIG:fig6&gt;&gt;). Similarly, TOGARAM1 interacts with tubulin through TOG3 and TOG4, with ARMC9 through TOG2 and with MTs and CEP104 through its linker region (Fig. 5G &lt;&lt;FIG:fig5&gt;&gt;). Additionally, Joubert syndrome-linked mutations in TOGARAM1 TOG2 and ARMC9 ARM domains disrupt their interaction, highlighting the functional importance for understanding how the ciliary tip module forms. For several reasons, we do not think the five ciliary tip proteins form a stoichiometric complex but rather a flexible interaction network, the function of which is likely to enhance the accumulation of these proteins at the distal ends of cilia. Firstly, a flexible interaction network could explain why we saw variation in size of \"corks\" in our cryo-ET data. Additionally, in vitro, we saw that ciliary tip module members act in a partially redundant fashion and therefore we cannot be sure that all ciliary tip proteins are present in every \"cork\". Leaving individual ciliary tip module proteins out of reconstitutions, whilst making slow growth less robust, had no major effect on MT dynamics. The one exception was TOGARAM1, which was essential for MT elongation; and that aligned well with the observation that overexpression of TOGARAM1 induces longer cilia (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Furthermore, in vertebrate cells, the loss of any of the ciliary tip module proteins results in the generally mild phenotype of shorter cilia, whereas simultaneous loss of several module components has more severe consequences (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Finally, a loose interaction mode is also in line with the fact that in other species, such as ciliates, these proteins bind to different sites (e.g., A- and B-tubules) and can have opposing functions (&lt;&lt;REF:Louka-2018-217-4298&gt;&gt;).\nIt appears counterintuitive that deletions or mutations in all vertebrate ciliary tip module proteins cause shorter cilia even though CEP104 and CSPP1 inhibit MT elongation (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). This could be explained by the interplay with additional factors not included in our assays. For example, CEP104 and CSPP1 might counteract MT depolymerase factors, such as kinesin-13, kinesin-18 KIF19A or kinesin-4 KIF7, which are known to act at ciliary tips (&lt;&lt;REF:Blaineau-2007-17-778&gt;&gt;; &lt;&lt;REF:He-2014-16-663&gt;&gt;; &lt;&lt;REF:Niwa-2012-23-1167&gt;&gt;). Alternatively, similar to our in vitro assays with TOGARAM1, CEP104 and CSPP1 might also cooperate with MT polymerization-promoting factors, such as XMAP215 or CLASP. This idea is supported by CEP104 promoting centriole elongation in flies (&lt;&lt;REF:Ryniawec-2023-33-4202&gt;&gt;), where it is established that Orbit/CLASP and kinesin-13 KLP10A act as positive and negative regulators of fly centriole length, respectively (&lt;&lt;REF:Shoda-2021-134-jcs251231&gt;&gt;).\nAn important question is the nature of structural changes at MT plus ends associated with slow polymerization. Cryo-ET analysis showed that rapidly growing MTs terminate with strongly flared protofilaments (&lt;&lt;REF:Gudimchuk-2020-11-3765&gt;&gt;; &lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;), whereas cork-like structures formed by ciliary tip proteins reduce protofilament flaring and at least partially cap the plus ends. Still, in these conditions, MT tips retain conformational variability, with some protofilaments more flared and some more blunt. It is possible that at each given time, the protofilaments that are blunt are occluded by ciliary tip proteins and therefore paused. Despite lacking a GTP cap, these protofilaments don't depolymerize because ciliary tip regulators keep them together and stabilize them from the luminal side. In this model, the more curved protofilaments are the ones undergoing tubulin addition. This would be consistent with the idea that TOG domains increase local tubulin concentration and deliver 'curved' tubulins to protofilament tips (&lt;&lt;REF:Ayaz-2014-3-e03069&gt;&gt;). Alternatively, ciliary tip proteins could be responsible for keeping tubulin dimers in flared protofilaments in a bent state, while new dimers are added onto the straighter protofilaments, which slowly forces the corks out of the plus ends as they elongate.\nFormation of MT plus-end-specific \"corks\" that impose blunt protofilament conformation is strikingly similar to the \"plugs\" observed in reconstitution experiments with the centriolar cap protein CP110 (&lt;&lt;REF:Ogunmolu-2021-?-2021.2012.2029.474442&gt;&gt;). MT stabilization from the luminal side combined with the reduction of protofilament curvature and occlusion of the longitudinal interface of beta-tubulin might thus be a common mechanism for generation of very stable and slowly growing MTs. The relevance of such structures is supported by the observations of plugs at the tips of ciliary MTs in cells (&lt;&lt;REF:Legal-2023-222-e202301129&gt;&gt;; &lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;). A mechanism involving the luminal MT surface and occlusion of the longitudinal interfaces of beta-tubulin is different from the regulation of much more dynamic cytoplasmic MT ends, which involves factors located on the outer MT surface (&lt;&lt;REF:Akhmanova-2015-16-711&gt;&gt;). Strikingly, the fundamental principles are the same: in both cases, plus-end regulation depends on the balance of growth-promoting and inhibiting activities that are brought together by a multivalent network of interacting proteins.\n\nMaterials and Methods\n\nDNA constructs, cell lines and cell culture\nAll proteins were cloned into modified pEGFP-C1 or pmCherry-C1 vectors with a StrepII tag and expressed in HEK293T cells (ATCC). For all module members except TOGARAM1 the human sequence was used, due to technical difficulties the mouse sequence of TOGARAM1 was used, it is 84% identical to human TOGARAM1. HEK293T cells were cultured in DMEM:F10 (1:1) for co-immunoprecipitation assays, or DMEM (Lonza) for protein purification, both were supplemented with 10% fetal calf serum (GE Healthcare Life Sciences) and 1% (v/v) penicillin/streptomycin. All cells were routinely checked for mycoplasma contamination using the MycoAlertTM Mycoplasma Detection Kit (Lonza).\n\nCo-immunoprecipitation assays\nHEK293T cells were transiently transfected with a mix consisting of polyethyleneimine (Polysciences) and constructs for the ciliary tip module proteins. Co-IP was performed with ChromoTek GFP-Trap(R) Magnetic particles M-270 (Proteintech) following manufacturer's instructions with company advised buffers. 24 hours post-transfection, cells were harvested in 1x PBS and lysed for 30 minutes on ice in 200 muL lysis buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.5% IGEPAL CA-630) supplemented with EDTA free protease inhibitor cocktail (Roche) and PhosSTOP(TM) phosphatase inhibitor cocktail (Roche). Lysate was cleared by 20 min centrifugation at 4 degreesC at 17,000 x g and diluted with 300 muL dilution buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA) supplemented with EDTA free protease inhibitor cocktail(Roche) and PhosSTOP(TM) phosphatase inhibitor cocktail (Roche) per sample. Diluted lysate was incubated rotating end-over-end for 45 minutes with wash buffer equilibrated beads. Beads were washed four times with 500 muL wash buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.05% IGEPAL CA-630) and protein was eluted in 80 muL 2x Laemmli sample buffer (0.125M tris HCl, 20% glycerol, 10% 2-mercaptoethanol, 0.02% Bromophenol blue, 0.2M DTT) and heated at 95 degreesC for 5 minutes. All steps prior to elution were performed at 4 degreesC and all reagents and samples were kept on ice.\nPrepared samples were run on 8% SDS-page gel and blotted onto Amersham Protran Premium 0.45 mum NC nitrocellulose membrane (Cytiva) by wet transfer at 37 constant volts overnight at 4 degreesC in transfer buffer (0.2M tris-HCl, 2M glycine, 10% MeOH and 0.01% SDS), or by semi-dry transfer for 1 hour at 0.15 constant Ampere in transfer buffer (0.2M tris-HCl, 2M glycine, 10% MeOH). Membranes were blocked in blocking buffer (2% BSA diluted in 1x PBS 0.05% Tween-20) for 1 hour and sequentially incubated with primary antibodies and secondary antibodies diluted in blocking buffer for 1 hour rotating at room temperature (primary and secondary antibodies) or overnight at 4 degreesC (primary antibodies only). Blots were thoroughly washed in between and after incubation with antibodies with wash buffer (1x PBS 0.05% Tween-20) and imaged with Odyssey(R) CLx Infrared Imaging System (LI-COR biosciences) at various exposure times. The following antibodies were used for western blotting: rabbit anti-RFP (1:2000, Rockland immunochemicals), mouse anti-GFP (1:2000, Sigma-Aldrich), donkey anti-mouse IgG (H + L) IRDye(R) 800CW (1:10000, LI-COR bioscience) and donkey anti-rabbit IgG (H + L) IRDye(R) 680CW (1:10000, LI-COR bioscience).\n\nProtein purification from HEK293T cells for in vitro reconstitution assays\nHEK293T cells were transiently transfected with a mix consisting of polyethyleneimine (Polysciences) and constructs for one of the ciliary tip module proteins. Protein purification was performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The cells were harvested 28 hours after transfection in lysis buffer (50 mM HEPES, 300 mM NaCl, 1 mM MgCl2, 1 mM DTT, 0.5% Triton X-100, pH 7.4) supplemented with protease inhibitors (Roche) and kept on ice for 15 minutes. After clearance of the debris by centrifugation, the supernatant was incubated with 20 microl StrepTactin beads (GE Healthcare) for 45 min. After several washing steps, five times with a 300 mM salt wash buffer (50 mM HEPES, 300 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4) and three times with a 150 mM salt wash buffer (50 mM HEPES, 150 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4), the protein was eluted in elution buffer (50 mM HEPES, 150 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4, 2.5 mM d-Desthiobiotin (Sigma-Aldrich)). Purified proteins were snap-frozen and stored at -80degreesC.\n\nMass spectrometry\nTo confirm each protein was purified, and the eluted protein did not contain any contaminants or known interactors that could affect MT dynamics, we performed mass spectrometry (MS) analysis. MS measurements were performed as described previously (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The purified protein sample was digested using S-TRAP microfilters (ProtiFi) according to the manufacturer's protocol. Digested peptides were eluted and dried in a vacuum centrifuge before liquid chromatography-mass spectrometry (LC-MS) analysis. The samples were analyzed by reversed-phase nLC-MS/MS using an Ultimate 3000 UHPLC coupled to an Orbitrap Exploris 480 mass spectrometer (Thermo Scientific). Digested peptides were separated using a 50 cm reversed-phase column packed in-house (Agilent Poroshell EC-C18, 2.7 microm, 50cm x 75 microm) and eluted from the column at a flow rate of 300 nL/min. MS data were acquired using a data-dependent acquisition (DDA) method with a MS1 resolution of 60,000 and mass range of 375-1600m/z. Fragmentation spectra were collected at a resolution of 15,000 using an HCD of 28, isolation window of 1.4m/z, and a fixed first mass of 120m/z. MS/MS fragment spectra were searched using Sequest HT in Proteome Discoverer (Thermo Scientific) against a human database (UniProt, year 2020) that was modified to contain the tagged protein sequence from each ciliary tip module protein and a common contaminants database. Tryptic miss cleavage tolerance was set to 2, fixed modifications were set to cysteine carbamidomethylation, and variable modifications included oxidized methionine and protein N-terminal acetylation. Peptides that matched the common contaminate database were filtered out from the results table.\n\nIn vitro reconstitution assays\n\nMT seed preparation\nDouble-cycled GMPCPP-stabilized MT seeds were used as templates for MT nucleation in vitro assays. GMPCPP-stabilized MT seeds were prepared as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). A tubulin mix consisting of 70% unlabeled porcine brain tubulin, 18% biotin-labeled porcine tubulin and 12% HiLyte-488-, rhodamine-, or HiLyte647-labeled porcine tubulin (all from Cytoskeleton) was incubated with 1 mM GMPCPP (Jena Biosciences) at 37degreesC for 30 minutes. Polymerized MTs were then pelleted using an Airfuge for 5 min at 199,000 x g and subsequently depolymerized on ice for 20 min. Next, 1 mM GMPCPP was added and MTs were let to polymerize again at 37degreesC for 30 minutes. Polymerized MTs were again pelleted (as above) and diluted tenfold in MRB80 buffer containing 10% glycerol prior to snap-freezing to store them at -80degreesC.\n\nIn vitro reconstitution assays\nIn vitro assays with dynamic MTs were performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Microscopic slides were prepared by adding two strips of double-sided tape to mount plasma-cleaned glass coverslips. The coverslips were functionalized by sequential incubation with 0.2 mg/mL PLL-PEG-biotin (Susos AG, Switzerland) and 1 mg/mL neutravidin (Invitrogen) in MRB80 buffer (80 mM piperazine-N, N[prime]-bis (2-ethane sulfonic acid), pH 6.8, supplemented with 4 mM MgCl2, and 1 mM EGTA). Then, GMPCPP-stabilized MT seeds were attached to the coverslips through biotin-neutravidin interactions. The coverslip was blocked with 1 mg/mL kappa-casein before the reaction mix was flushed in. The reaction mix consisted of different concentrations and combinations of fluorescently labeled purified proteins in MRB80 buffer supplemented with 15 microM porcine brain tubulin (100% dark porcine brain tubulin when 20 nM GFP-EB3 or mCherry-EB3 was added, or 97% dark porcine brain tubulin with 3% rhodamine- or HiLyte-488-labeled porcine tubulin), 0.1% methylcellulose, 1 mM GTP, 50 mM KCl, 0.2 mg/mL kappa-casein, and oxygen scavenger mix [50 mM glucose, 400 microg/mL glucose oxidase, 200 microg/mL catalase and 4 mM DTT]. This mix was spun down in an Airfuge for 5 min at 119,000 x g before addition to the flow chamber and the flow chamber was sealed with vacuum grease. Microtubules were imaged immediately at 30degreesC using a total internal reflection fluorescence (TIRF) microscope.\n\nIn vitro assays for cryo-ET\nSamples were prepared as above for reconstitution assays with slight modifications. Instead of flow chambers, all steps occur in a tube. Reaction mixes consisted of either just 20nM EB3 or all CTM components (20nM TOGARAM1, 20nM CEP104, 20nM CSPP1, 20nM CCDC66, 200nM ARMC9) with 20nM EB3 in MRB80 buffer supplemented with 15 microM porcine brain tubulin, 1 mM GTP, 0.2 mg/mL kappa-casein, and 15 microM DTT. After centrifugation of the reaction mix for dynamic MTs, GMPCPP-stabilized seeds and ProtA-Au5 fiducials (1:20) were added, and MTs were left to polymerize for 10-20 min at 30degreesC in an Eppendorf tube. Subsequently, the sample was gently resuspended and 4 microl was transferred to a glow-discharged holey carbon R2/1 copper grid (Quantifoil Micro Tools) suspended in the chamber of a Vitrobot (Thermo Fisher Scientific). The sample was incubated for 2 min inside the Vitrobot chamber, equilibrated at 30 degreesC and 95% relative humidity, to allow for potential MT repolymerization after sample transfer to the grid. Subsequently, grids were manually back-blotted for 3 s and plunge-frozen in liquid ethane. Vitrified grids were clipped into auto-grid cartridges and stored in liquid nitrogen until future use.\n\nSingle-molecule fluorescence intensity analysis\nProtein samples of GFP, GFP-EB3 and either GFP-CEP104, GFP-TOGARAM1 or GFP-ARMC9 were diluted in PBS and immobilized in adjacent flow chambers of the sample plasma cleaned glass coverslip as described in (&lt;&lt;REF:Sharma-2016-37-362&gt;&gt;). Flow chambers were then sealed with vacuum grease and immediately imaged using TIRF microscopy. 10-20 images (per condition) of previously unexposed coverslip areas were acquired with 200 ms exposure time.\n\nCEP104 molecule counting at blocked MT plus ends\nTo determine the number of CEP104 molecules at blocked MT plus ends we immobilized GFP in one chamber of the same plasma cleaned glass coverslip and in vitro reconstitution assay in the adjacent chamber (as described above). Chambers were sealed and immediately imaged using TIRF microscope. First images on unbleached GFP single molecules were acquired, then using the same illumination conditions, images of unexposed GFP-CEP104 bound MTs were acquired.\n\nFluorescence Recovery After Photobleaching\nFor FRAP experiments, in vitro reconstitutions were prepared as described above and imaging was conducted using TIRF microscopy. Either focused 488-laser (for GFP-EB3) or 561-laser (for mCherry-CEP104) was used to bleach specific regions of the MT lattice.\n\nMicroscopy\n\nTIRF microscopy\nIn vitro reconstitution assays imaged on a previously described (iLas2) TIRF microscope setup (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). We used the iLas3 system (Gataca Systems (France)) which is a dual laser illuminator for azimuthal spinning TIRF (or Hilo) illumination. This system was installed on Nikon Ti microscope (with the perfect focus system, Nikon), equipped with 489 nm 150 mW Vortran Stradus 488 laser (Vortran) and 100 mW 561 nm OBIS laser (Coherent), 49002 and 49008 Chroma filter sets. Additionally, a CCD camera CoolSNAP MYO (Teledyne Photometrics) was installed and the set up was controlled with MetaMorph 7.10.2.240 software (Molecular Device). To keep the in vitro samples at 30 degreesC, a stage top incubator model INUBG2E-ZILCS (Tokai Hit) was used. The final resolution using CCD camera was 0.045 mum/pixel. For all assays MT dynamics was measured at 200 ms exposure and 3 second intervals for 10 minutes. For EB3 FRAP experiments continuous imaging was used with 100 ms exposure. For kinesin experiments first a 10 minute movie was acquired at 200 ms exposure and 3 second intervals to measure MT dynamics, subsequently Dm-KHC(1-421)-SNAP-6xHis (gift from Kapitein lab, Addgene plasmid #196976) labelled with Alexa647-SNAP dye (NEB) was continuous imaged using 100 ms exposure.\n\nCryo-ET\nVitrified in vitro reconstituted MTs were imaged on a Talos Arctica transmission electron microscope (200 keV) (Thermo Fisher Scientific), equipped with a K2 summit direct electron detector (Gatan) and post-column energy filter aligned to the zero-loss peak with 20 keV slit width. Image acquisition was controlled by Serial-EM (&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;). MT ends were located on TEM overview images, acquired at 4,100x magnification (33.3 A/pixel). Tilt series were collected at 63,000x magnification (2.17 A/pixel) with a dose rate of ~5 e-/pixel/s and a total dosage of 100 e-/A2. Tilt images were recorded using a dose symmetric scheme (&lt;&lt;REF:Hagen-2017-197-191&gt;&gt;) over a tilt range of 54degrees to -54degrees or 60degrees to -60degrees with a tilt increment of 2degrees, at a defocus target of -3 mum.\n\nImage analysis\n\nAnalysis of MT plus-end dynamics in vitro\nAnalysis of MT plus-end dynamics was performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Movies of dynamic MTs were corrected for drift, and kymographs were generated using the plugin KymoResliceWide v.0.4 in ImageJ (https://github.com/ekatrukha/KymoResliceWide). MT tips were traced with lines, and measured lengths and angles were used to calculate the MT dynamics parameters such as growth rate and transition events. All events with growth rates faster than 0.02 microm/min and slower than 0.5 microm/min were categorized as slow growing, events faster than 0.5 microm/min were categorized as fast growing, events with shrinkage rates faster than 0.02 microm/min were categorized as shrinking. The events with slower growth rates or faster shrinkage rates than the before mentioned rates were categorized as pause events. Transition frequencies were calculated by dividing the sum of the transition events per experiment by the total time this event could have occurred. Weighted growth rate histograms were calculated by taking individual growth durations and dividing by total growth time.\n\nAnalysis of FRAP experiments\nData were normalized to 1 for image acquired immediately before the FRAP event and 0 for the image acquired immediately after the FRAP event using the following equation; Normalized intensity=(intensity t=x - intensity t=1)(intensity t=0 - intensity t=1). For EB3, non-linear fit curves were fitted to the post-bleach intensities and half-life calculated using GraphPad Prism.\n\nAnalysis of single molecule fluorescence intensities\nSingle molecule fluorescence spots of proteins immobilized directly on the cover glass were detected and fitted with 2D Gaussain function using custom written ImageJ plugin DoM-Utrect (https://github.com/ekatrukha/DoM_Utrecht). Fitted peak intensities were then used to create intensity histograms. CEP104-blocked plus ends were then manually annotated and fitted with 2D Gaussian function using the same plugin.\n\nCryo-ET 3D volume reconstruction and analysis\nTomogram reconstruction and denoising was performed using an in house Python script, as described previously (&lt;&lt;REF:Chaillet-2023-24-13375&gt;&gt;). The tilt series' frames were initially aligned and dose-weighted using MotionCor2 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;). Subsequent tilt series alignment and tomographic reconstruction for denoising, visualization and analysis were done through AreTomo 1.3.3. Final reconstructed tomographic volumes were created through weighted back-projection and binned with a factor of two. Cryo-CARE 0.2.2 was used for tomogram denoising (&lt;&lt;REF:Buchholz-2019-152-277&gt;&gt;). For this, even and odd tomographic reconstructions were generated through splitting of the movie frames for each individual tilt into even and odd summed frames with MotionCor2 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;). Alignment parameters were first calculated on the full tilt series and subsequently applied to the even and odd stacks. The cryo-CARE network was trained on 5 tomograms per dataset, and then applied to the entire set.\nFor visualization purposes, semi-automated segmentation was performed on denoised tomograms using the EMAN2 2.91 tomoseg module (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). For this, separate neural networks were trained for the two features of interest: \"MT\" and \"ciliary tip module\". Each feature was included as 'negative' training reference, together with background noise and ice contamination, for the neural network training of the other feature. Final visualization and 3D rendering was performed in ChimeraX.\nMT ends, both free and CTM-plugged, were only analyzed for protofilament shape in case they were plus ends. MT polarity was determined through 2D rotational averaging with applied symmetry, using the EMAN2 program e2proc2d. Manual 3D tracing of protofilament shapes at MT plus ends was performed, as previously described (&lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;; &lt;&lt;REF:McIntosh-2020-2101-191&gt;&gt;; &lt;&lt;REF:Ogunmolu-2021-?-2021.2012.2029.474442&gt;&gt;), in the IMOD 4.9.6 program 3dmod (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;). Manual tracing was performed on tomographic subvolumes, generated with the mtrotlong program of IMOD (4.11.20). Traces of individual protofilaments were saved as separate contours within one object per MT. To satisfy the three-point requirement of howflared, for blunt end protofilaments a third point was added one pixel upstream of the second point that indicated the last protofilament segment in the MT wall. Subsequently, protofilament coordinates were extracted using the howflared program of IMOD (4.11.20). MT raggedness and protofilament length parameters, as described in the results section, were calculated by howflared. Curvature analysis was performed on the howflared output with Matlab scripts available at https://github.com/ngudimchuk/Process-PFs.\n\nStatistical analysis\nStatistical analysis was performed using GraphPad Prism 9. Figure legends contain statistical details of each experiment, including the statistical tests used, the number of measurements and the number of experiments.\n\nCode availability\nThe python script for automated tomogram reconstruction and cryo-CARE denoising is available online at https://github.com/SBC-Utrecht/cryocare-from-movies. MATLAB scripts for analysis of protofilament tracings in IMOD are available online at https://github.com/ngudimchuk/Process-PFs.\n\nAuthor Contributions\nH.A.J.S. and C. M. vdB. performed experiments, analyzed data and wrote the paper, R.A.H. performed and analyzed cryo-electron tomography experiments, and wrote the paper, D.S. performed immunoprecipitation experiments, K.E.S. performed and analyzed mass spectrometry experiments, S.H. supervised and analyzed cryo-electron tomography experiments, R.R has generated essential reagents and acquired funding, A.A. coordinated the project, acquired funding and wrote the paper. All authors reviewed and edited the paper.\n\nCompeting financial interests\nThe authors declare no competing financial interests.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nCilia are motile or sensory organelles present on the surface of many eukaryotic cells. Motile cilia generate movement for processes such as mucus flow in airways or swimming of unicellular green algae, while primary cilia function as signaling hubs with fundamental roles in organism development (&lt;&lt;REF:Klena-2022-38-103&gt;&gt;; &lt;&lt;REF:Mill-2023-24-421&gt;&gt;). Ciliary dysfunction is associated with numerous diseases, collectively called ciliopathies, which affect multiple organs and tissues (&lt;&lt;REF:Badano-2006-7-125&gt;&gt;; &lt;&lt;REF:Hildebrandt-2011-364-1533&gt;&gt;; &lt;&lt;REF:Klena-2022-38-103&gt;&gt;). The core of each cilium is formed by a microtubule (MT)-based structure, the axoneme, which is often disrupted in ciliopathies (&lt;&lt;REF:Deretic-2023-136-?&gt;&gt;). Diverse proteins and pathways responsible for axoneme formation have been identified through genetic screens and cell biology studies, yet the biochemical mechanisms governing the dynamics of axonemal MTs remain very poorly understood (&lt;&lt;REF:Keeling-2016-5-6&gt;&gt;).\nUnlike cytoplasmic MTs, axonemal plus ends grow very slowly likely impeding the formation of a stabilizing GTP cap, which is required for preventing MT disassembly (&lt;&lt;REF:Desai-1997-13-83&gt;&gt;) and efficient tubulin addition (&lt;&lt;REF:Wieczorek-2015-17-907&gt;&gt;). The absence of a stable GTP cap can be compensated by specific factors bound to MT ends. Excellent candidates for this function in cilia are the components of the recently identified ciliary tip interaction network or \"module\", CEP104, CSPP1, TOGARAM1, ARMC9 and CCDC66 (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Depletion of any of these proteins causes ciliary defects, and mutations in most of them lead to a primary ciliopathy called Joubert syndrome, characterized by brain malformations, breathing problems or more severe developmental disorders (&lt;&lt;REF:Akizu-2014-94-80&gt;&gt;; &lt;&lt;REF:Hildebrandt-2011-364-1533&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Shaheen-2014-94-73&gt;&gt;; &lt;&lt;REF:Srour-2015-97-744&gt;&gt;; &lt;&lt;REF:Tuz-2014-94-62&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;). The proteins comprising the ciliary tip module interact with MTs and with each other, and although they do not form a stoichiometric complex, they are all important for controlling ciliary length and the signaling pathways dependent on primary cilia (&lt;&lt;REF:Conkar-2017-130-1450&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Patzke-2006-209-199&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;). In addition to their roles in cilia, some of the tip module proteins also associate with centrioles, centrosomes, centriolar satellites and cytoplasmic MTs, where they participate in cilia-independent processes, such as cell division (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;; &lt;&lt;REF:Conkar-2019-9-14250&gt;&gt;; &lt;&lt;REF:Patzke-2006-209-199&gt;&gt;; &lt;&lt;REF:Ryniawec-2023-33-4202&gt;&gt;; &lt;&lt;REF:Zhu-2015-290-27053&gt;&gt;).\nCEP104 (also known as FAP256), TOGARAM1 (also known as Crescerin or CHE-12) and ARMC9 are conserved across eukaryotes, including Chlamydomonas, Tetrahymena and C. elegans, where they participate in the biogenesis of motile or sensory cilia (&lt;&lt;REF:Bacaj-2008-178-989&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;; &lt;&lt;REF:Satish Tammana-2013-126-5018&gt;&gt;). Both CEP104 and TOGARAM1 have evolutionary conserved TOG domains (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). TOGARAM1 contains four TOG domains, two of which can promote tubulin polymerization in light scattering experiments (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;), and the same is true for the single tubulin-binding TOG domain of CEP104 (&lt;&lt;REF:Al-Jassar-2017-25-146&gt;&gt;; &lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Loss of function mutations in TOGARAM1 and CEP104 in multiple organisms result in short cilia (&lt;&lt;REF:Bacaj-2008-178-989&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;; &lt;&lt;REF:Satish Tammana-2013-126-5018&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Similarly, loss of CSPP1, a protein that increases MT stability by binding inside the MT lumen and promoting pausing (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;; &lt;&lt;REF:Wang-2024-22-mjae007&gt;&gt;), results in short cilia (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;). ARMC9 on its own cannot bind to MTs but acts as a scaffold for other ciliary tip module components (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). ARMC9 and TOGARAM1 have been localized to motile cilia, where they were found to have opposite functions in regulating MT length (&lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;), though mutants of both proteins make mammalian primary cilia shorter (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Finally, CSPP1 and CEP104 interact with CCDC66, and co-depletion of CCDC66 with either CSPP1 or CEP104 results in further reduction of ciliary length compared to the removal of either CSPP1 or CEP104 alone (&lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;).\nFigure 1.\nCiliary tip module proteins have distinct effects on MT dynamics in vitro.\n(A) Schematic representation of ciliary tip module members and summary table highlighting individual protein effects on MT dynamics. Jelly-roll, JR; zinc finger, ZNF; MT binding, MTB; pause domain, PD; alpha-helical domain, H.\n(B-C) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488-labelled tubulin or 20 nM GFP-EB3.\n(D-F) Parameters of MT plus-end dynamics in the presence of 15 microM tubulin alone or with 20 nM EB3 in combination with indicated concentrations of proteins (from kymographs shown in B-C, I-P, and S1C). (D) Bars represent pooled data from three independent experiments (growth rate) or averaged means from three independent experiments (pause and block duration), total number of growth events, pauses or blocks: tubulin alone, n=356, 0; 10 nM CCDC66, n=411, 0; 10 nM CEP104, n=306, 0; 100 nM CEP104, n=0, 138; 10 nM CSPP1,n=422, 97; 10 nM TOGARAM1, n=347, 0; EB3 alone, n=938, 0; EB3 with 10 nM CCDC66, n=562, 0; EB3 with 10 nM CEP104, n=213, 101; EB3 with CSPP1, n=1715, 273; EB3 with TOGARAM1, n=861, 0. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test, all conditions were compared to their relevant control (either tubulin alone or tubulin with 20 nM EB3). (E-F) Bars represent averaged means from three independent experiments. Error bars represent s.e.m.. Movies were acquired for 10 minutes, therefore this is the maximum time for pause duration,\n(G-P) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488 or Rhodamine-labelled tubulin, or 20 nM GFP- or mCherry-EB3, and indicated concentrations and colors of ciliary tip module proteins. Orange arrowheads, blocked plus ends; orange arrows, CEP104 tracking minus ends; yellow arrowheads, pauses; blue arrowheads, rescues.\nTo uncover biochemical mechanisms underlying the function of these proteins and their combined regulation of ciliary MTs, we reconstituted in vitro their individual and collective effects on MT dynamics. Although the individual TOG domain of CEP104 was previously shown to promote MT polymerization (&lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;), we found that the full-length protein specifically inhibited MT plus-end elongation and shortening. This activity was potentiated by EB3, CSPP1 and CCDC66, which could all recruit CEP104 to MTs. This was surprising as EB3 and CSPP1 bind to the outer and inner surface of the MT wall, respectively (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;; &lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). TOGARAM1 functioned as a polymerase that converted the inhibition of MT polymerization imposed by CEP104 into slow growth. ARMC9 and CCDC66 behaved as scaffolds, enhancing the effects of their binding partners. The combination of all five ciliary tip module proteins resulted in very slow and processive MT plus-end elongation, an effect that required TOGARAM1 and became less robust when one of the other proteins was left out. MT growth rates in these conditions were in the same range as initial elongation rates of regenerating flagella of single-celled organisms (&lt;&lt;REF:Rosenbaum-1967-34-345&gt;&gt;; &lt;&lt;REF:Rosenbaum-1969-41-600&gt;&gt;; &lt;&lt;REF:Witman-1975-253-178&gt;&gt;). Cryo-electron tomography showed that together, ciliary tip proteins formed a champagne cork-like structure that protruded from MT plus ends and diminished protofilament flaring. Altogether, our findings demonstrate that, through a combination of opposing activities, ciliary tip module components can stabilize MT plus ends and drive their slow but persistent elongation.\n\nResults\n\nCiliary tip module proteins have distinct effects on MT dynamics in vitro\nTo characterize the effects of ciliary tip module proteins on MT polymerization, we N-terminally tagged them with mCherry or GFP and purified them from transiently transfected HEK293 cells (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;, S1A &lt;&lt;FIG:figs1&gt;&gt;). Mass spectrometry-based analysis demonstrated that the protein preparations contained only very minor contaminations with other known regulators of MT dynamics, such as CSPP1 (Fig. S1B &lt;&lt;FIG:figs1&gt;&gt;). In addition, the heat shock protein Hsp70, a protein that to our knowledge has no effect on MT dynamics, was present in the preparations of CCDC66 and CSPP1 (Fig. S1B &lt;&lt;FIG:figs1&gt;&gt;). We used these proteins to perform in vitro reconstitution assays with MTs grown from GMPCPP-stabilized seeds and observed their impact on MT dynamics by Total Internal Reflection Fluorescence (TIRF) microscopy (&lt;&lt;REF:Bieling-2007-450-1100&gt;&gt;; &lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Although axonemes are composed of MT doublets, single MTs are appropriate substrates to study the effects of ciliary tip proteins, because doublets become singlets at ciliary tips (&lt;&lt;REF:Fisch-2011-103-249&gt;&gt;; &lt;&lt;REF:Kiesel-2020-27-1115&gt;&gt;; &lt;&lt;REF:Legal-2023-222-e202301129&gt;&gt;). To detect MTs, we used either fluorescent tubulin or fluorescently tagged EB3, which increases MT growth rate and induces catastrophes (&lt;&lt;REF:Komarova-2009-184-691&gt;&gt;) (Fig. 1B-F &lt;&lt;FIG:fig1&gt;&gt;). Studying the effects of ciliary tip proteins in the presence of EBs is relevant, because EB1 and EB3 localize to axonemal MTs in both primary and motile cilia (&lt;&lt;REF:Kiesel-2020-27-1115&gt;&gt;; &lt;&lt;REF:Pedersen-2003-13-1969&gt;&gt;; &lt;&lt;REF:Schroder-2007-17-1134&gt;&gt;).\nFigure S1 (Related to Figure 1).\nCharacterization of ciliary tip module proteins in vitro.\n(A) Analysis of purified GFP-tagged ciliary tip module proteins by SDS-PAGE. Asterisks indicate the full-length protein bands. Protein concentrations were determined from a BSA standard (not shown).\n(B) Mass spectrometry analysis of purified GFP-tagged ciliary tip module proteins. For all module members except TOGARAM1 the human sequence was used, due to technical difficulties the mouse sequence of TOGARAM1 was used, it is 84% identical to human TOGARAM1.\n(C) Fields of view (top, scale bar 2 microm) and kymograph (bottom, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with HiLyte-488-tubulin and mCherry-CEP104.\n(D) Kymographs illustrating mobility of DmKHC(1-421) on CEP104-blocked MT labelled with HiLyte-488-tubulin and mCherry-CEP104 (top) and DmKHC (bottom) proving that the blocked end of the MT is the plus end. Scale bars 2 microm and 60 s for both kymographs.\nAmong the tested proteins, ARMC9 was the only one that displayed no MT binding even when present at 300 nM (Fig. 1G &lt;&lt;FIG:fig1&gt;&gt; and H &lt;&lt;FIG:fig1&gt;&gt;), in agreement with data in cells (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). CCDC66, which is known to interact with MTs (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;), bound along the MT lattice already at 10 nM and slightly decreased the growth rate, but did not affect the frequencies of catastrophes and rescues and did not induce pausing (Fig. 1D-F, I &lt;&lt;FIG:fig1&gt;&gt; and J &lt;&lt;FIG:fig1&gt;&gt;).\nCEP104 could not autonomously bind MTs at 10 nM in assays with tubulin alone (Fig. S1C &lt;&lt;FIG:figs1&gt;&gt;). However, at 100 nM it efficiently blocked MT growth specifically at the plus end (Fig. 1D-F &lt;&lt;FIG:fig1&gt;&gt; and K &lt;&lt;FIG:fig1&gt;&gt;), as confirmed by including in the assay a constitutively active fragment of the plus-end directed Kinesin-1 DmKHC(1-421) (Fig. S1D &lt;&lt;FIG:figs1&gt;&gt;). In the presence of its binding partner EB3 (&lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;), CEP104 could specifically inhibit MT plus-end polymerization already at 10 nM (Fig. 1D-F &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt;). CEP104 colocalized with EB3 at some growing plus and occasionally also minus ends, however, this did not alter growth rates at the plus end (Fig.1D &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt;). Once MT growth was arrested, no regrowth was detected for the remainder of the experiment (Fig 1E, K &lt;&lt;FIG:fig1&gt;&gt;, and L &lt;&lt;FIG:fig1&gt;&gt;).\nEpisodes of inhibited growth, albeit transient ones, were also observed with CSPP1, as we have shown previously (Fig. 1D-F, M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;) (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Both with and without EB3 CSPP1 preferentially binds at both plus and minus ends when they undergo growth perturbations (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;; &lt;&lt;REF:Wang-2024-22-mjae007&gt;&gt;), and therefore it displayed discreet sites of increased lattice accumulation (Fig. 1M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;). At the plus ends, CSPP1 accumulations decreased growth rate, induced pauses and facilitated transitions from pause to growth, whereas pausing was less obvious at the minus ends due to their overall slower dynamics (Fig. 1D-F, M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;).\nFinally, TOGARAM1 bound to MT plus ends, reduced their growth rate and induced transitions from shrinkage to growth (rescues), and these effects were not altered by the presence of EB3 (Fig. 1D-F, O &lt;&lt;FIG:fig1&gt;&gt; and P &lt;&lt;FIG:fig1&gt;&gt;). A decrease in MT growth rate was unexpected, as TOGARAM1 structurally resembles the MT polymerase ch-TOG/XMAP215 (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;), which strongly accelerates MT polymerization (&lt;&lt;REF:Brouhard-2008-132-79&gt;&gt;). Altogether, we show that all the full-length members of the ciliary tip module can be purified and used for in vitro assays, where they display opposing effects on MT dynamics.\n\nPlus-end blocking by CEP104 depends on the TOG domain and is potentiated by EB3, CCDC66 and CSPP1\nAmong the ciliary tip proteins analyzed above, the effect of CEP104 was the most unexpected as it was thought to be a MT polymerase (&lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;), and we therefore set out to dissect it in more detail. Single molecule counting experiments demonstrated that CEP104 forms a dimer, in agreement with the sedimentation profile of its first helical domain (Fig. 2A &lt;&lt;FIG:fig2&gt;&gt;) (&lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;). In the presence of EB3, ~5-6 CEP104 molecules (2-3 dimers) were sufficient to block MT plus-end growth (Fig. 2B &lt;&lt;FIG:fig2&gt;&gt;). In these conditions, Fluorescence Recovery after Photobleaching (FRAP) showed that CEP104 did not exchange at blocked MT plus ends (Fig. 2C &lt;&lt;FIG:fig2&gt;&gt;), whereas EB3, associated with the same ends, did partially turn over, albeit significantly slower than on growing MT plus ends (Fig. 2D &lt;&lt;FIG:fig2&gt;&gt; and E &lt;&lt;FIG:fig2&gt;&gt;). This could be explained by the direct interaction of EB3 and CEP104 on the MT through the SxIP motif of CEP104, SKIP (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt;) (&lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;). Mutation of this motif has previously been shown to abolish the interactions between the C-terminus of EB and its partners (&lt;&lt;REF:Honnappa-2009-138-366&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Indeed, CEP104 SKNN double mutation prevented EB3-dependent recruitment to MTs; therefore, even in the presence of EB3, 100 nM CEP104 SKNN was needed to block plus-end growth (Fig. 3A-D &lt;&lt;FIG:fig3&gt;&gt;, and S2A &lt;&lt;FIG:figs2&gt;&gt;).\nFigure S2 (Related to Figure 3).\nCharacterization of CEP104 constructs in vitro.\n(A) Analysis of purified GFP-tagged CEP104 constructs by SDS-PAGE. Asterisks show protein bands. Protein concentrations were determined from a BSA standard (not shown).\n(B) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with either HiLyte-488-tubulin or GFP-EB3 and indicated concentrations and colors of CEP104 constructs.\n(C) Co-immunoprecipitation of either CSPP1 (left) or CCDC66 (right) with indicated CEP104 constructs, both CSPP1 and CCDC66 interact with the jelly-roll domain of CEP104.\n(D) Analysis of purified GFP-tagged CEP104 DeltaJR construct by SDS-PAGE. Asterisk indicates the full-length protein band. Protein concentration was determined from a BSA standard (not shown).\nFigure 2.\nA few CEP104 molecules stably block MT plus ends\n(A) Fields of view (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules, GFP-EB3 dimers, and GFP-CEP104 dimers immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=29981; GFP-EB3, n=55378; GFP-CEP104, n=32335. Scale bar 2 microm.\n(B) Representative MT with CEP104-blocked plus end (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules and GFP-CEP104 intensity at blocked plus end immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=13925; GFP-CEP104, n=92. Scale bar 2 microm.\n(C-E) FRAP analysis of CEP104 (C) and EB3 (D) at blocked MT plus ends, or dynamic EB3 at growing plus ends (E). Arrowhead marks point of photobleaching in representative kymographs, scale bars 2 microm and 60 s (C), 2 microm and 10 s (D, E). Plots show averaged curves with exponential fit. Number of FRAP measurements: CEP104 n=17; stationary EB3 n=14; dynamic EB3 n=10. Error bars represent s.e.m..\nFigure 3.\nPlus-end blocking by CEP104 is potentiated by EB3, CCDC66 and CSPP1.\n(A) Schematic representation of different CEP104 constructs; jelly-roll, JR; alpha-helical domain, H; zinc finger, ZNF.\n(B) Percentage of time MT plus ends spent blocked in the presence of 20 nM EB3 in combination with CEP104 constructs (from kymographs shown in C-E, 1C, 1L). Bars represent averaged means from three independent experiments. Error bars represent s.e.m..\n(C-F) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488 or Rhodamine-labelled tubulin or, 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module constructs. Orange arrowheads, blocked plus ends; white arrowheads, CEP104 DeltaTOG at plus ends of seeds; white arrows, CEP104 tracking the minus end.\n(G) Parameters of MT plus-end dynamics in the presence of 15 microM tubulin alone or with 20 nM EB3 in combination with indicated concentrations of ciliary tip module proteins (from kymographs shown in F, H-L, S2B &lt;&lt;FIG:figs2&gt;&gt;). For dynamic state bars represent averaged means from three independent experiments. For depolymerization rate bars represent pooled data from three independent experiments: 2 nM CEP104, n=133; 2 nM CEP104 with 15 nM CCDC66, n=23; 10 nM CEP104 with 15 nM CCDC66, n=122; 2 nM CEP104 with 10 nM CSPP1, n=69; 10 nM CEP104 with 10 nM CSPP1, n=99; EB3 with 2 nM CEP104, n=1; EB3 with 2 nM CEP104 and 15 nM CCDC66, n=63; EB3 with 2 nM CEP104 and 10 nM CSPP1, n=116.\n(H-N) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte-488- or Rhodamine-labelled tubulin, or 20 nM GFP or mCherry-EB3 and indicated concentrations and colors of ciliary tip module constructs. Light orange arrowheads, blocked seeds; dark orange arrowhead, blocked lattice; blue arrowheads, slow depolymerization of plus ends.\n(O) Percentage of time of MT plus ends spent blocked at the seed in the presence of 20 nM EB3 in combination with CEP104 DeltaJR and other ciliary tip module proteins (from kymographs shown in M, N, P, S2B &lt;&lt;FIG:figs2&gt;&gt;). Bars represent averaged means from three independent experiments.\n(P) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- and 50 nM mCherry-CEP104 DeltaJR. Orange arrowheads, blocked plus ends.\n(Q) Schematic model showing CEP104 domain functions and interactions with other ciliary tip module proteins. Jelly-roll, JR; zinc finger, ZNF; alpha-helical domain, H.\nFor (B, G and O) error bars represent s.e.m. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\nWhereas the SxIP motif was not essential for plus-end blocking by CEP104, the single TOG domain of CEP104 was required to inhibit MT growth. CEP104 lacking the TOG domain (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt; and S2A &lt;&lt;FIG:figs2&gt;&gt;) could still be recruited to MTs by EB3, but no plus-end blocking was observed (Fig. 3B &lt;&lt;FIG:fig3&gt;&gt; and E &lt;&lt;FIG:fig3&gt;&gt;). In the presence of EB3, CEP104 DeltaTOG could bind to growing MT ends and accumulate at the border between the seed and the plus-end-grown lattice (Fig. 3E &lt;&lt;FIG:fig3&gt;&gt;), a localization for which we currently have no explanation.\nSince recent studies described functional and biochemical interactions between CEP104, CCDC66, CSPP1 and ARMC9 (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;), we next tested in vitro the combinations of these proteins. We found that in the presence of 15 nM CCDC66, already 2 nM CEP104 was sufficient to occasionally block MT plus end outgrowth from the seed (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt; and G &lt;&lt;FIG:fig3&gt;&gt;). 2 nM CEP104 on its own, or in the presence of EB3, or together with ARMC9, showed hardly any MT binding (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;). However, CEP104 driven growth inhibition at MT seeds became much more common at 10 nM CEP104 with 15 nM CCDC66, and in these conditions, CEP104 and CCDC66 could also strongly inhibit depolymerization of dynamic MTs by either pausing the plus ends or reducing their shrinkage rate from 36.91+/-1.34 microm/min to 0.12+/-0.02 microm/min (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt; and H &lt;&lt;FIG:fig3&gt;&gt;). Similar effects were observed when either 2 nM or 10 nM CEP104 was combined with 10 nM CSPP1: both growth inhibition at the seed and at the dynamic plus ends, as well as slow depolymerization were observed (Fig. 3G, I &lt;&lt;FIG:fig3&gt;&gt; and J &lt;&lt;FIG:fig3&gt;&gt;). These data indicate that CEP104 is not just a MT growth inhibitor or a depolymerase that can prevent tubulin addition to the plus ends, but also a specific MT stabilizing factor that can inhibit tubulin loss from MT plus ends lacking a GTP cap. The addition of EB3 had no strong impact on MT dynamics when combined with 2 nM CEP104 and 15 nM CCDC66 (Fig. 3F, G &lt;&lt;FIG:fig3&gt;&gt; and K &lt;&lt;FIG:fig3&gt;&gt;), but potentiated seed blocking when included with 2 nM CEP104 and 10 nM CSPP1, as most MTs were blocked at the seed in these conditions (Fig. 3G, I &lt;&lt;FIG:fig3&gt;&gt; and L &lt;&lt;FIG:fig3&gt;&gt;).\nBased on our own and previously published co-immunoprecipitation experiments, the interactions of CCDC66 and CSPP1 with CEP104 depended on its jelly-roll (JR) domain (Fig. S2C &lt;&lt;FIG:figs2&gt;&gt;) (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;). The deletion of this domain abolished the ability of both proteins to recruit CEP104 to MTs (Fig. 3M-O &lt;&lt;FIG:fig3&gt;&gt; and S2D &lt;&lt;FIG:figs2&gt;&gt;). However, CEP104 DeltaJR could still block seed outgrowth in the presence of EB3 (Fig. 3O &lt;&lt;FIG:fig3&gt;&gt; and P &lt;&lt;FIG:fig3&gt;&gt;), though a higher concentration of CEP104 DeltaJR compared to the full-length CEP104 was needed to achieve significant plus-end blocking (50 nM versus 10 nM, compare Fig. 1D &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt; to Fig. 3O, P &lt;&lt;FIG:fig3&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;), indicating that the JR domain might directly or indirectly contribute to MT blocking.\nTaken together, our results demonstrate that CEP104 can stably and specifically associate with MT plus ends and inhibit both tubulin addition and removal from these ends in a TOG domain-dependent manner. However, the affinity of CEP104 for MTs is rather low, and its activity is strongly enhanced by association with its binding partners EB3, CCDC66 and CSPP1 that recruit it to MTs. Since EB3 binds to the outer MT surface (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;) and the same might be true for CCDC66 as it can bundle MTs (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;), whereas CSPP1 associates with the MT lumen (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;), CEP104 might be positioned at protofilament ends to allow access to both surfaces of the MT wall (Fig. 3Q &lt;&lt;FIG:fig3&gt;&gt;).\n\nThe rescue activity of TOGARAM1 depends on TOG3 and TOG4 domains\nHaving characterized the activity of CEP104, we next turned to the other TOG domain component of the ciliary tip module, TOGARAM1, which contains two pairs of TOG domains (TOG1-TOG4) connected by a long linker (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Single molecule counting experiments showed that it is a monomer (Fig. 4B &lt;&lt;FIG:fig4&gt;&gt;), similar to other MT regulators with multiple TOG domains, XMAP215 (&lt;&lt;REF:Brouhard-2008-132-79&gt;&gt;) and CLASP2 (&lt;&lt;REF:Aher-2018-46-40&gt;&gt;). A previous study showed that isolated TOG2 and TOG4, but not TOG1 and TOG3 promoted tubulin polymerization in light scattering assays (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). To test how these domains contribute to the rescue activity of the full-length protein, we took advantage of previously generated TOGARAM1 mutants where the tubulin-binding surfaces of the individual TOG domains were disrupted by point substitutions within the intra-HEAT loop of HEAT repeat A (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;) (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Although there is no direct interaction between TOGARAM1 and EB3, we included EB3 in our in vitro reconstitution assays as EB3 increases catastrophe frequency and therefore facilitates observing rescues (Fig 1F &lt;&lt;FIG:fig1&gt;&gt;) (&lt;&lt;REF:Komarova-2009-184-691&gt;&gt;). We found that simultaneous mutation of TOG3 and TOG4 (TOGARAM1 123'4'), but not TOG1 and TOG2 (TOGARAM1 1'2'34) abolished the rescue activity, although both mutants could still bind to MTs at 10 nM concentration (Fig. 4A, C-E &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). This was different from previous observations in cells, where the double TOG3-TOG4 mutant (TOGARAM1 123'4') could not bind to MTs (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). TOG3 and TOG4 showed redundancy within the full-length protein because single mutations in each one of them did not abolish rescues (Fig. 4A, C, F, G &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). TOG1 and TOG2 could be deleted without impairing rescue activity (the construct L-TOG-34, Fig. 4A, C, H &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;), and the combination of these two domains (TOG-12) without the adjacent linker region did not bind to MTs even at 200 nM (Fig. 4A, I &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). TOG-34 did not visibly associate with MTs at 10 nM, but at 200 nM it bound to MTs and induced rescues (Fig. 4A, C, J, K &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). Since the construct containing TOG3 and TOG4 domains together with adjacent linker (L-TOG-34) was active in our assays already at 10 nM, in agreement with previous findings, the linker region of TOGARAM1 contributes to MT affinity (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). All tested constructs mildly reduced MT growth rate and TOG-34 constructs with or without the adjacent linker mildly suppressed catastrophes (Fig. 4C &lt;&lt;FIG:fig4&gt;&gt;). Our results demonstrate that, on its own, TOGARAM1 has relatively mild effects on MT dynamics, which rely in its two C-terminal TOG domains.\nFigure S3 (Related to Figure 4).\nCharacterization of TOGARAM1 constructs in vitro.\nAnalysis of purified GFP-tagged TOGARAM1 constructs by SDS-PAGE. Asterisks show protein bands. Protein concentrations were determined from a BSA standard (not shown).\nFigure 4.\nThe rescue activity of TOGARAM1 depends on the TOG3 and TOG4 domains.\n(A) Schematic representation of different TOGARAM1 constructs and summary table highlighting their effects on MT dynamics. Vertical lines indicate point mutations predicted to ablate tubulin-binding activity.\n(B) Fields of view (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules, EB3 dimers and TOGARAM1 molecules were immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=57865; EB3, n=73074; TOGARAM1, n=74306. Scale bar, 2 microm.\n(C) Parameters of MT plus-end dynamics in the presence of 20 nM EB3 in combination with indicated concentrations of TOGARAM1 constructs (from kymographs shown in D-H, K, 1C, 1P). For growth rates bars represent pooled data from three independent experiments, total number of growth events: EB3 alone, n=938; EB3 with TOGARAM1, n=861; EB3 with 1'2'34, n=350; EB3 with 123'4', n=337; EB3 with 123'4, n=560; EB3 with 1234', n=253; EB3 with L-TOG-34, n=131; EB3 with 200nM TOG-34, n=150. For transition frequencies bars represent average means from three independent experiments. Error bars represent s.e.m., ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(D-K) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of TOGARAM1 constructs. Blue arrowheads, rescues.\n\nARMC9 colocalizes with TOGARAM1 and CSPP1 and enhances their effects on MT dynamics\nWe next set out to characterize the function of ARMC9. Single molecule counting experiments showed that ARMC9 forms dimers dependent on its central helical region (Fig. S4A &lt;&lt;FIG:figs4&gt;&gt; and B &lt;&lt;FIG:figs4&gt;&gt;). As shown above, ARMC9 does not bind to MTs on its own nor in the presence of CEP104 and EB3 (Fig. 1G, H &lt;&lt;FIG:fig1&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;). Consistently, no direct interaction has been described for ARMC9 and CEP104, even though they were previously shown to weakly co-immunoprecipitate with each other (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;), an observation we were not able confirm (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt;). We could confirm by co-immunoprecipitation the interaction between ARMC9 and CCDC66, however when this combination was tested in reconstitution experiments we did not see any recruitment of ARMC9 to the MT lattice (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt; and D &lt;&lt;FIG:figs4&gt;&gt;). We also confirmed by co-immunoprecipitation the interaction between ARMC9 and TOGARAM1 (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt;) and refined the previous mapping of this interaction (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;) by showing that it requires the armadillo (ARM) repeats of ARMC9, but not its helical domain responsible for dimerization, and the tubulin-binding surface of the TOG2 domain of TOGARAM1 (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, S4A, E &lt;&lt;FIG:figs4&gt;&gt; and F &lt;&lt;FIG:figs4&gt;&gt;). We also tested the impact of different point mutations identified in Joubert syndrome patients on the interaction of ARMC9 and TOGARAM1 (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;), and found that all tested mutations in the TOG2 domain (all located in the first HEAT repeat) and one of the several known mutations in the ARM domain (G492R) perturb binding (Fig. S4A, G &lt;&lt;FIG:figs4&gt;&gt; and H &lt;&lt;FIG:figs4&gt;&gt;), supporting the functional importance of this interaction.\nFigure S4 (Related to Figure 5).\nMapping of ARMC9 and TOGARAM1 interaction.\n(A) Schematic representation of different ARMC9 constructs.\n(B) Histogram plots of fluorescent intensities of single GFP molecules, GFP-EB3 dimers, and full-length GFP-ARMC dimers (left) or GFP-ARMC9 DeltaH molecules immobilized in separate chambers of the same coverslips. Number of molecules analyzed, left, right: GFP, n=18415, 14914; GFP-EB3, n= 132826, 163959; GFP-ARMC9 FL, n=99873; GFP-ARMC9 DeltaH, 55479.\n(C) Co-immunoprecipitation of full-length ARMC9 with indicated proteins.\n(D) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins.\n(E-F) Co-immunoprecipitation experiments of either full-length TOGARAM1 with indicated ARMC9 constructs (E) or full-length ARMC9 with indicated TOGARAM1 point mutations\n(F). ARMC9 and TOGARAM1 interact through ARM domain and TOG2 domain, respectively.\n(G-H) Co-immunoprecipitation experiments of either full-length TOGARAM1 with indicated ARMC9 Joubert syndrome mutations (G) or full-length ARMC9 with indicated TOGARAM1 Joubert syndrome mutations (H).\nIn the in vitro assays, TOGARAM1 could recruit ARMC9 to MTs, and the colocalization between the two proteins was particularly prominent in particles that could stationarily bind to or diffuse on MTs (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt;). As expected from the co-immunoprecipitation experiments, this colocalization was not perturbed by the 123'4' mutant of TOGARAM1 (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, 5B &lt;&lt;FIG:fig5&gt;&gt;) but was abolished by the 1'2'34 mutant (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, 5C &lt;&lt;FIG:fig5&gt;&gt;). The brightness of ARMC9 and TOGARAM1 in MT-bound particles increased over time, indicating binding of additional molecules (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt; and B &lt;&lt;FIG:fig5&gt;&gt;). This binding was likely driven by ARMC9, as it was not observed for TOGARAM1 alone (Fig. 1P &lt;&lt;FIG:fig1&gt;&gt;). Presence of ARMC9 did not drastically alter MT growth rates but did cause a roughly two-fold increase in the rescue activity of TOGARAM1, which was still dependent on TOG3 and TOG4 (Fig. 5A, B &lt;&lt;FIG:fig5&gt;&gt;, and D &lt;&lt;FIG:fig5&gt;&gt;). In contrast to ARMC9, the addition of CCDC66 did not alter the effects of TOGARAM1 on MT dynamics, and no colocalization between CCDC66 and TOGARAM1 was observed (Fig. S4D &lt;&lt;FIG:figs4&gt;&gt;).\nFigure 5.\nARMC9 colocalizes with TOGARAM1 and CSPP1 and enhances their effects on MT dynamics.\n(A-C) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCH-EB3 and indicated concentrations and colors of TOGARAM1 constructs and ARMC9. Blue arrowheads, ARMC9 and TOGARAM1 colocalization.\n(D) Parameters of MT plus-end dynamics in the presence of 20 nM EB3 in combination with indicated concentrations of TOGARAM1 constructs and ARMC9 (from kymographs shown in A-B, 1C, 1P, 4E). For growth rates bars represent pooled data from three independent experiments, total number of growth events: EB3 alone, n=938; EB3 with TOGARAM1, n=861; EB3 with TOGARAM1 and ARMC9, n=342; EB3 with 123'4', n=337; EB3 with 123'4' and ARMC9, n=325. For transition frequencies bars represent averaged means from three independent experiments. Error bars represent s.e.m.. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(E) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP-EB3 and ciliary tip module proteins indicated. Yellow arrowheads, pauses.\n(F) Parameters of MT plus-end dynamics with 20 nM EB3 in combination with indicated concentrations of ciliary top module proteins (from kymographs shown in E, 1C, 1N). For growth rate and pause duration bars represent pooled data from three independent experiments, total number of growth events, pauses: EB3 alone, n=938, 0; EB3 with CSPP1, n=1715, 273; EB3 with CSPP1 and ARMC9, n=627, 315. For transition frequencies and dynamic state bar represent averaged means from three independent experiments. Error bars represent s.e.m., ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(G) Schematic model showing TOGARAM1, ARMC9, and CSPP1 interaction domains; MTB; pause domain, PD; alpha-helical domain, H.\nWe observed co-immunoprecipitation of ARMC9 and CSPP1 (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt; and B &lt;&lt;FIG:figs5&gt;&gt;), confirming previous observations (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). The interaction required the C-terminal part of CSPP1 and the linker regions surrounding the central helical domain of ARMC9, though not the helical domain itself (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt; and B &lt;&lt;FIG:figs5&gt;&gt;). In vitro reconstitution assays showed that CSPP1 could trigger ARMC9 accumulation on MTs in the vicinity of the growing MT ends (Fig. 5E &lt;&lt;FIG:fig5&gt;&gt;) and mildly increased the percentage of time CSPP1-induced MT pausing (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;).\nFigure S5 (Related to Figure 5).\nMapping of ARMC9 and CSPP1 interaction.\n(A) Co-immunoprecipitation experiment of full-length ARMC9 with indicated CSPP1 constructs illustrated in schematic below.\n(B) Co-immunoprecipitation experiment of full-length CSPP1 with indicated ARMC9 constructs illustrated in Figure S4A &lt;&lt;FIG:figs4&gt;&gt;.\n(C) Co-immunoprecipitation experiment of full-length CSPP1 with full-length TOGARAM1 shows no interaction.\n(D) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins.\nWe also tested whether CSPP1 and TOGARAM1 can bind to each other but detected no co-immunoprecipitation (Fig. S5C &lt;&lt;FIG:figs5&gt;&gt;), and CSPP1 displayed no colocalization with either TOGARAM1 or CCDC66 in the in vitro assays (Fig. S5D &lt;&lt;FIG:figs5&gt;&gt;). Altogether, our results indicate that CCDC66 does not colocalize with either TOGARAM1 or CSPP1 in reconstitution assays, whereas ARMC9 can bind to both of these proteins through two distinct domains and serve as a scaffold mildly enhancing their effects on MT dynamics (Fig. 5G &lt;&lt;FIG:fig5&gt;&gt;).\n\nCombined action of ciliary tip proteins drives slow processive MT growth\nTo complete the exploration of all pairwise combinations of the studied ciliary tip proteins, we examined the joint effects of CEP104 and TOGARAM1. The two proteins coprecipitated with each other in a manner dependent on the zinc finger domain of CEP104 and the linker region of TOGARAM1 (Fig. S6A &lt;&lt;FIG:figs6&gt;&gt; and B &lt;&lt;FIG:figs6&gt;&gt;). When they were combined on dynamic MTs in the presence of EB3, there were periods of pausing both at the plus end of the seeds and of the dynamic lattice (Fig 6A-C &lt;&lt;FIG:fig6&gt;&gt;). However, these pauses were transient and followed by growth events (Fig 6D &lt;&lt;FIG:fig6&gt;&gt; and E &lt;&lt;FIG:fig6&gt;&gt;), something we did not see with any other CEP104-ciliary tip protein combination. Furthermore, the two proteins together induced periods of very slow MT polymerization with the rates of 0.12+/-0.01 microm/min, ~20 times slower than 2.27+/-0.02 microm/min observed with TOGARAM1 alone (Fig. 6A, C &lt;&lt;FIG:fig6&gt;&gt; and D &lt;&lt;FIG:fig6&gt;&gt;). CEP104 and TOGARAM1 can thus interact through regions that do not engage their tubulin-binding domains and together impose slow MT polymerization (Fig. 6F &lt;&lt;FIG:fig6&gt;&gt;).\nFigure S6 (Related to Figure 6).\nMapping of CEP104 and TOGARAM1 interaction.\n(A-B) Co-immunoprecipitation of either wildtype CEP104 with indicated TOGARAM1 constructs (A) or wildtype TOGARAM1 with indicated CEP104 constructs (B), the two proteins interact between the linker of TOGARAM1 and the zinc finger of CEP104.\n(C) Kymographs illustrating mobility of DmKHC(1-421) on slow growing MT with all ciliary tip module proteins (top) and DmKHC (bottom) proving that the slow growing end of the MT is the plus end. Scale bars 2 microm and 60 s for both kymographs.\nFigure 6.\nCombined action of ciliary tip proteins drives slow processive MT growth.\n(A) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with GFP-EB3, mCherry-TOGARAM1, and GFP-CEP104. Orange arrowheads, blocked plus end; green arrowheads, slow MT growth.\n(B-E) Parameters of MT plus-end dynamics in the presence of EB3 in combination with indicated concentrations of ciliary tip module proteins (from kymographs shown in A, 1C, 1L, 1P). For pause &amp; block duration (B) and growth rates (D) data was pooled from three independent experiments, total number of growth events, pauses: EB3 alone, n=938, 0; EB3 with CEP104, n=213, 101; EB3 with TOGARAM1, n=861, 0; EB3 with CEP104 and TOGARAM1, n=373, 140. Bars represent mean + s.e.m.. For dynamic state (C) and transition frequencies (E) bars represent averaged means from three independent experiments. Error bars represent s.e.m.\n(F) Schematic model showing interactions between CEP104, TOGARAM1 and tubulin. Jelly-roll, JR; zinc finger, ZNF; alpha-helical domain, H.\n(G-H) Weighted growth rates (G) and dynamic state (H) for combinations of ciliary tip module proteins indicated (from kymographs I-N, and 1C). For growth rates (G) data was pooled from three independent experiments, total number of growth events: EB3 alone, n=938; entire ciliary tip module, n=70; no ARMC9, n=81; no CCDC66, n=83; no CEP104, n=41; no CSPP1, n=141; no TOGARAM1, n=0. For dynamic state (H) bars represent or averaged means from two independent experiments, error bars represent s.e.m..\n(I-N) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins indicated. Orange arrow, blocked plus end; green arrows, slow plus-end growth.\nNext, we combined all five ciliary tip module components in the presence of EB3. Strikingly, in these conditions, all MTs displayed slow and highly processive plus-end polymerization with occasional pausing, with an average rate of 0.19+/-0.04 microm/min, while minus ends could still undergo phases of growth and shrinkage (Fig. 6G-I &lt;&lt;FIG:fig6&gt;&gt; and S6C &lt;&lt;FIG:figs6&gt;&gt;). To test which proteins are essential for this dynamic state, we repeated the assays while systematically leaving out each of the ciliary tip proteins. Omitting TOGARAM1 abolished slow growth completely, as all MT plus ends were paused (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt; and J &lt;&lt;FIG:fig6&gt;&gt;). This was not surprising, because MT behavior in these conditions was dominated by the two pausing factors, CEP104 and CSPP1. This indicates that an important function of TOGARAM1 is to overcome MT growth inhibition imposed by CEP104 or CSPP1. Leaving out CEP104 had no major effect, except for a slight increase in pausing and very occasional periods of fast growth (Fig. 6G, H &lt;&lt;FIG:fig6&gt;&gt; and K &lt;&lt;FIG:fig6&gt;&gt;), whereas fast elongation became much more common in the absence of CSPP1 (Fig. 6G, H &lt;&lt;FIG:fig6&gt;&gt;, and L &lt;&lt;FIG:fig6&gt;&gt;). Omitting either ARMC9 or CCDC66 had no major effect except that no pausing was observed, and occasional episodes of fast growth were detected (Fig. 6G, H, M &lt;&lt;FIG:fig6&gt;&gt; and N &lt;&lt;FIG:fig6&gt;&gt;). No MT catastrophes or shrinkage were observed in any of these conditions (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt;). Altogether, ciliary tip module components stabilize MTs by preventing their depolymerization and promoting robust but very slow tubulin addition.\n\nCiliary tip proteins form cork-like densities at MT plus ends and reduce protofilament flaring\nTo better understand the localization of the ciliary tip module on MT ends, we turned to cryo-electron tomography (cryo-ET). We reconstructed 3D volumes containing MTs grown with either EB3 alone or in the presence of the entire ciliary tip module. To best match our in vitro reconstitution conditions, samples were incubated at 30 degreesC with GMPCPP-stabilized seeds and 15 microM tubulin for 10 minutes before being added to holey carbon-coated copper grids and vitrified. Based on our analysis of MT dynamics we presumed that they were mostly fast growing for EB3 controls and elongating slowly with a few paused ends for ciliary tip module samples (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt;). MTs grown in the presence of the ciliary tip module showed densities at their ends reminiscent of champagne corks, whereas MTs grown in the presence of only EB3 had no clear density at their ends (Fig. 7A-D &lt;&lt;FIG:fig7&gt;&gt;, S7A &lt;&lt;FIG:figs7&gt;&gt; and B &lt;&lt;FIG:figs7&gt;&gt;). Despite the density for the ciliary tip module appearing quite varied in structure, it was always situated in both the lumen and on the tips of the MT protofilaments (Fig. 7B, D &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;). We used 2D rotational averaging to resolve MT polarity based on chirality of the cross-section (&lt;&lt;REF:Bouchet-Marquis-2007-99-45&gt;&gt;), through which we determined that cork-like electron densities were specific for MT plus ends (Fig. 7C, D &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;). 36 out of 40 captured \"corks\" where confidently assigned to plus ends, the polarity of the other 4 MTs remains undetermined due to poor tilt-alignment of the tomograms. We performed semi-automated segmentation of the tomographic volumes, both for visualization and segregation of the MT protofilaments and ciliary tip module (Fig. 7E &lt;&lt;FIG:fig7&gt;&gt; and F &lt;&lt;FIG:fig7&gt;&gt;). It proved difficult to segregate all individual protofilaments from each other for further analysis due to the limited tilt range. However, the majority of protofilaments were individually resolved, and separation between the ciliary tip module and protofilaments was often clear. Extensive contacts on the luminal surface of tubulin and the exposed protofilament ends were observed, while the exterior surface of the MT remained largely unbound.\nFigure S7 (Related to Figure 7).\nCharacterization of MT plus ends by cryo-ET\n(A, B) Representative examples of MT plus ends grown in presence of EB3 alone (A) or ciliary tip module and EB3 (B). Per MT, the following is shown from left to right; a slice of the denoised tomogram containing the MT plus end, an 8 nm thick transverse cross-section accompanied by rotational averaging analysis to determine MT polarity, and the corresponding 3D model of manual protofilament tracing accompanied by its transverse cross-section. Scale bars 25 nm.\nFigure 7.\nCiliary tip proteins form cork-like densities at MT plus ends and reduce protofilament flaring\n(A, B) Slices (4.3 nm thick) of denoised tomograms showing MT plus ends grown in presence of EB3 alone (A) or ciliary tip module and EB3 (B). Scale bars 25 nm.\n(C, D) 8nm thick transverse cross-section of the corresponding MTs in panels A and B. The transverse cross-section (top) is accompanied by rotational average (bottom) to indicate MT polarity. Scale bars 25 nm.\n(E, F) 3D rendered segmentation volume of a free/dynamic MT plus end and ciliary tip module-plugged plus end. The MT is shown in grey and the ciliary tip module density in green.\n(G-J) 3D model of manually traced protofilament shapes (G, H), accompanied by their transverse cross-sections (I, J).\n(K) Distribution of protofilament length measured from the last segment within the MT wall until the tip of the protofilament. Total number of protofilaments analyzed; EB3, n=329; CTM, n=352. Lines represent median (solid) and quartiles (dotted). ****, p&lt;0.0001; Mann-Whitney test.\n(L) Protofilament standard deviation per MT. Total number of MTs analyzed: EB3, n=26; ciliary tip module, n=26. Bars represent mean, with error bars representing s.e.m., ****, p&lt;0.0001 (Mann-Whitney test).\n(M) Schematic representation of parameters that were obtained from manual tracing of MT plus-end protofilaments.\n(N) Average local protofilament curvature per MT. Total number of MTs analyzed: EB3, n=25; ciliary tip module, n=25. Bars represent mean, with error bars representing s.e.m., *, p&lt;0.05; Mann-Whitney test.\n(O) MT raggedness plotted as standard deviation of protofilament axial distance of last segment within the wall per MT. Total number of MTs analyzed: EB3, n=26; ciliary tip module, n=26. Bars represent mean, with error bars representing s.e.m., **, p&lt;0.01; Mann-Whitney test.\n(P) Schematic model of the ciliary tip module at a MT plus end.\nWe turned to manual tracing to analyze how the ciliary tip module may be altering MT tip architecture, and how this architecture deviates from more typical MT architecture associated with fast dynamics (Fig. 7G-J &lt;&lt;FIG:fig7&gt;&gt;, S7A &lt;&lt;FIG:figs7&gt;&gt; and B &lt;&lt;FIG:figs7&gt;&gt;). We first looked at how flared the MT protofilaments were by measuring both protofilament length (length of curved segment) and protofilament standard deviation per MT (standard deviation of the distance between first point that deviated from being completely straight to the end of the protofilament, see materials and methods) (Fig. 7K-M &lt;&lt;FIG:fig7&gt;&gt;). As expected, MT plus ends grown with only EB3 showed primarily flared protofilaments (&lt;&lt;REF:Gudimchuk-2020-11-3765&gt;&gt;; &lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;) (Fig. 7A, K, L &lt;&lt;FIG:fig7&gt;&gt; and S7A &lt;&lt;FIG:figs7&gt;&gt;), whereas plus ends \"corked\" with ciliary tip module proteins had a dramatic decrease in both protofilament length and standard deviation in protofilament length per MT (Fig. 7B, K, L &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;), as well as a mild decrease in average local curvature (Fig. 7N &lt;&lt;FIG:fig7&gt;&gt;). Finally, MT raggedness, defined as the standard deviation in the axial distance along the MT of the first point for each protofilament that deviated from being completely straight (Fig. 7M &lt;&lt;FIG:fig7&gt;&gt;), was modestly but significantly decreased in MT plus ends \"corked\" by ciliary tip module proteins (Fig. 7O &lt;&lt;FIG:fig7&gt;&gt;). Together our results indicate a potential mechanism where the ciliary tip module acts from both the lumen and tips of MTs to stabilize slow and processive elongation (Fig 7P &lt;&lt;FIG:fig7&gt;&gt;).\n\nDiscussion\nCiliary tip proteins play important roles in controlling the structure and therefore function of motile and primary cilia. Mutations in these proteins cause various ciliopathies often resulting from shorter cilia causing defects in cilia-dependent signaling pathways (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;). Axonemal MTs are stable and elongate very slowly without undergoing long depolymerization events. Here, we have reconstituted these behaviors in vitro with five components of the ciliary tip module. We found that collectively, these proteins robustly impose very slow and processive MT growth. Furthermore, elongation rates measured in our assays, 0.19+/-0.04 microm/min, fall within the range of those measured for the initial elongation of regenerating flagella in single-celled organisms such as Chlamydomonas reinhardtii (0.08-0.40 microm/min) (&lt;&lt;REF:Marshall-2005-16-270&gt;&gt;; &lt;&lt;REF:Rosenbaum-1967-34-345&gt;&gt;; &lt;&lt;REF:Rosenbaum-1969-41-600&gt;&gt;; &lt;&lt;REF:Witman-1975-253-178&gt;&gt;), though these rates are reduced when flagella reach their normal length. However, it remains to be determined if our measured growth rates are within the range of elongating primary cilia.\nSlow growth is an unusual state for dynamic MTs, because it is incompatible with the formation of a long stabilizing GTP cap, which is needed to avoid MT disassembly (&lt;&lt;REF:Desai-1997-13-83&gt;&gt;) and promote growth (&lt;&lt;REF:Wieczorek-2015-17-907&gt;&gt;). Factors controlling slow MT polymerization must thus inhibit tubulin detachment and promote tubulin addition at MT plus ends that have only a very short or no GTP cap. We found that several ciliary tip proteins can perform these functions. Two of them, TOGARAM1 and CEP104, were predicted to be MT polymerases because their individual TOG domains can bind tubulin and enhance tubulin polymerization in vitro\n(&lt;&lt;REF:Al-Jassar-2017-25-146&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). However, full-length TOGARAM1 and CEP104 do not accelerate tubulin polymerization at freely growing MT plus ends. On the contrary, TOGARAM1 somewhat slows down MT growth rate, and CEP104 potently blocks MT elongation. Still, these proteins are MT stabilizers rather than depolymerases as both can inhibit tubulin detachment from GDP-bound MT ends: TOGARAM1 induces rescues, whilst CEP104 dramatically slows down disassembly of MT plus ends. Moreover, TOGARAM1 can promote tubulin addition and thus serve as a polymerase at paused MT ends, and together CEP104 and TOGARAM1 can induce phases of very slow polymerization.\nThe processive slow MT growth reconstituted by either CEP104 and TOGARAM1 together, or the ciliary tip module as a whole, is unique to the plus ends of MTs. This is likely driven by the two TOG domain-containing module members, because specific orientation of the TOG domains can position tubulin dimers favorably for binding to protofilament tips at the plus end (&lt;&lt;REF:Ayaz-2012-337-857&gt;&gt;). Generally, the interaction of TOG domain proteins with MTs depends on positively charged intrinsically disordered regions (&lt;&lt;REF:Aher-2018-46-40&gt;&gt;; &lt;&lt;REF:Byrnes-2017-216-1641&gt;&gt;; &lt;&lt;REF:Widlund-2011-108-2741&gt;&gt;) and additional adaptors, such as EB1 and SLAIN2 (&lt;&lt;REF:Li-2011-193-973&gt;&gt;; &lt;&lt;REF:van der Vaart-2011-193-1083&gt;&gt;). This is also true for the ciliary TOG domain regulators. TOGARAM1, similar to XMAP215 (&lt;&lt;REF:Widlund-2011-108-2741&gt;&gt;), contains a linker region which promotes binding to MTs in vitro, and is further recruited to MTs by ARMC9. CEP104 depends on interactions with several binding partners, EB3, CSPP1 and CCDC66. EB3 is known to bind to the outer MT surface (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;), whereas CSPP1 is a luminal protein (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The binding mode of CCDC66 has not yet been clarified, but it is likely that some of its parts are present on MT surface as it can bundle MTs in vitro (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;). With binding partners on both sides of the MT wall, CEP104 is in a good position to span the MT tip. TOGARAM1 is likely positioned very close to protofilament ends, since it can catalyze tubulin addition. Therefore, ARMC9 is another good candidate for connecting across MT surfaces as it not only interacts and promotes TOGARAM1 activity but also the pausing behavior of luminal protein CSPP1. Collectively, this fits with the cryo-ET data, which show that the complex of ciliary tip proteins localizes only at the plus ends of MTs and is associated with the luminal side (likely CSPP1) and the tip of MTs (likely CEP104, TOGARAM1 and CCDC66), resembling a cork on a champagne bottle.\nThe interactions between ciliary tip proteins depend at least partially on non-overlapping domains: for example, CEP104 binds to tubulin through its TOG domain, to EB3 through the SxIP motif within its flexible C-terminal region, to TOGARAM1 through the zinc finger domain and to CCDC66 and CSPP1 through the jelly-roll domain (Fig 3Q &lt;&lt;FIG:fig3&gt;&gt;, 6F &lt;&lt;FIG:fig6&gt;&gt;). Similarly, TOGARAM1 interacts with tubulin through TOG3 and TOG4, with ARMC9 through TOG2 and with MTs and CEP104 through its linker region (Fig. 5G &lt;&lt;FIG:fig5&gt;&gt;). Additionally, Joubert syndrome-linked mutations in TOGARAM1 TOG2 and ARMC9 ARM domains disrupt their interaction, highlighting the functional importance for understanding how the ciliary tip module forms. For several reasons, we do not think the five ciliary tip proteins form a stoichiometric complex but rather a flexible interaction network, the function of which is likely to enhance the accumulation of these proteins at the distal ends of cilia. Firstly, a flexible interaction network could explain why we saw variation in size of \"corks\" in our cryo-ET data. Additionally, in vitro, we saw that ciliary tip module members act in a partially redundant fashion and therefore we cannot be sure that all ciliary tip proteins are present in every \"cork\". Leaving individual ciliary tip module proteins out of reconstitutions, whilst making slow growth less robust, had no major effect on MT dynamics. The one exception was TOGARAM1, which was essential for MT elongation; and that aligned well with the observation that overexpression of TOGARAM1 induces longer cilia (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Furthermore, in vertebrate cells, the loss of any of the ciliary tip module proteins results in the generally mild phenotype of shorter cilia, whereas simultaneous loss of several module components has more severe consequences (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Finally, a loose interaction mode is also in line with the fact that in other species, such as ciliates, these proteins bind to different sites (e.g., A- and B-tubules) and can have opposing functions (&lt;&lt;REF:Louka-2018-217-4298&gt;&gt;).\nIt appears counterintuitive that deletions or mutations in all vertebrate ciliary tip module proteins cause shorter cilia even though CEP104 and CSPP1 inhibit MT elongation (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). This could be explained by the interplay with additional factors not included in our assays. For example, CEP104 and CSPP1 might counteract MT depolymerase factors, such as kinesin-13, kinesin-18 KIF19A or kinesin-4 KIF7, which are known to act at ciliary tips (&lt;&lt;REF:Blaineau-2007-17-778&gt;&gt;; &lt;&lt;REF:He-2014-16-663&gt;&gt;; &lt;&lt;REF:Niwa-2012-23-1167&gt;&gt;). Alternatively, similar to our in vitro assays with TOGARAM1, CEP104 and CSPP1 might also cooperate with MT polymerization-promoting factors, such as XMAP215 or CLASP. This idea is supported by CEP104 promoting centriole elongation in flies (&lt;&lt;REF:Ryniawec-2023-33-4202&gt;&gt;), where it is established that Orbit/CLASP and kinesin-13 KLP10A act as positive and negative regulators of fly centriole length, respectively (&lt;&lt;REF:Shoda-2021-134-jcs251231&gt;&gt;).\nAn important question is the nature of structural changes at MT plus ends associated with slow polymerization. Cryo-ET analysis showed that rapidly growing MTs terminate with strongly flared protofilaments (&lt;&lt;REF:Gudimchuk-2020-11-3765&gt;&gt;; &lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;), whereas cork-like structures formed by ciliary tip proteins reduce protofilament flaring and at least partially cap the plus ends. Still, in these conditions, MT tips retain conformational variability, with some protofilaments more flared and some more blunt. It is possible that at each given time, the protofilaments that are blunt are occluded by ciliary tip proteins and therefore paused. Despite lacking a GTP cap, these protofilaments don't depolymerize because ciliary tip regulators keep them together and stabilize them from the luminal side. In this model, the more curved protofilaments are the ones undergoing tubulin addition. This would be consistent with the idea that TOG domains increase local tubulin concentration and deliver 'curved' tubulins to protofilament tips (&lt;&lt;REF:Ayaz-2014-3-e03069&gt;&gt;). Alternatively, ciliary tip proteins could be responsible for keeping tubulin dimers in flared protofilaments in a bent state, while new dimers are added onto the straighter protofilaments, which slowly forces the corks out of the plus ends as they elongate.\nFormation of MT plus-end-specific \"corks\" that impose blunt protofilament conformation is strikingly similar to the \"plugs\" observed in reconstitution experiments with the centriolar cap protein CP110 (&lt;&lt;REF:Ogunmolu-2021-?-2021.2012.2029.474442&gt;&gt;). MT stabilization from the luminal side combined with the reduction of protofilament curvature and occlusion of the longitudinal interface of beta-tubulin might thus be a common mechanism for generation of very stable and slowly growing MTs. The relevance of such structures is supported by the observations of plugs at the tips of ciliary MTs in cells (&lt;&lt;REF:Legal-2023-222-e202301129&gt;&gt;; &lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;). A mechanism involving the luminal MT surface and occlusion of the longitudinal interfaces of beta-tubulin is different from the regulation of much more dynamic cytoplasmic MT ends, which involves factors located on the outer MT surface (&lt;&lt;REF:Akhmanova-2015-16-711&gt;&gt;). Strikingly, the fundamental principles are the same: in both cases, plus-end regulation depends on the balance of growth-promoting and inhibiting activities that are brought together by a multivalent network of interacting proteins.\n\nMaterials and Methods\n\nDNA constructs, cell lines and cell culture\nAll proteins were cloned into modified pEGFP-C1 or pmCherry-C1 vectors with a StrepII tag and expressed in HEK293T cells (ATCC). For all module members except TOGARAM1 the human sequence was used, due to technical difficulties the mouse sequence of TOGARAM1 was used, it is 84% identical to human TOGARAM1. HEK293T cells were cultured in DMEM:F10 (1:1) for co-immunoprecipitation assays, or DMEM (Lonza) for protein purification, both were supplemented with 10% fetal calf serum (GE Healthcare Life Sciences) and 1% (v/v) penicillin/streptomycin. All cells were routinely checked for mycoplasma contamination using the MycoAlertTM Mycoplasma Detection Kit (Lonza).\n\nCo-immunoprecipitation assays\nHEK293T cells were transiently transfected with a mix consisting of polyethyleneimine (Polysciences) and constructs for the ciliary tip module proteins. Co-IP was performed with ChromoTek GFP-Trap(R) Magnetic particles M-270 (Proteintech) following manufacturer's instructions with company advised buffers. 24 hours post-transfection, cells were harvested in 1x PBS and lysed for 30 minutes on ice in 200 muL lysis buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.5% IGEPAL CA-630) supplemented with EDTA free protease inhibitor cocktail (Roche) and PhosSTOP(TM) phosphatase inhibitor cocktail (Roche). Lysate was cleared by 20 min centrifugation at 4 degreesC at 17,000 x g and diluted with 300 muL dilution buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA) supplemented with EDTA free protease inhibitor cocktail(Roche) and PhosSTOP(TM) phosphatase inhibitor cocktail (Roche) per sample. Diluted lysate was incubated rotating end-over-end for 45 minutes with wash buffer equilibrated beads. Beads were washed four times with 500 muL wash buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.05% IGEPAL CA-630) and protein was eluted in 80 muL 2x Laemmli sample buffer (0.125M tris HCl, 20% glycerol, 10% 2-mercaptoethanol, 0.02% Bromophenol blue, 0.2M DTT) and heated at 95 degreesC for 5 minutes. All steps prior to elution were performed at 4 degreesC and all reagents and samples were kept on ice.\nPrepared samples were run on 8% SDS-page gel and blotted onto Amersham Protran Premium 0.45 mum NC nitrocellulose membrane (Cytiva) by wet transfer at 37 constant volts overnight at 4 degreesC in transfer buffer (0.2M tris-HCl, 2M glycine, 10% MeOH and 0.01% SDS), or by semi-dry transfer for 1 hour at 0.15 constant Ampere in transfer buffer (0.2M tris-HCl, 2M glycine, 10% MeOH). Membranes were blocked in blocking buffer (2% BSA diluted in 1x PBS 0.05% Tween-20) for 1 hour and sequentially incubated with primary antibodies and secondary antibodies diluted in blocking buffer for 1 hour rotating at room temperature (primary and secondary antibodies) or overnight at 4 degreesC (primary antibodies only). Blots were thoroughly washed in between and after incubation with antibodies with wash buffer (1x PBS 0.05% Tween-20) and imaged with Odyssey(R) CLx Infrared Imaging System (LI-COR biosciences) at various exposure times. The following antibodies were used for western blotting: rabbit anti-RFP (1:2000, Rockland immunochemicals), mouse anti-GFP (1:2000, Sigma-Aldrich), donkey anti-mouse IgG (H + L) IRDye(R) 800CW (1:10000, LI-COR bioscience) and donkey anti-rabbit IgG (H + L) IRDye(R) 680CW (1:10000, LI-COR bioscience).\n\nProtein purification from HEK293T cells for in vitro reconstitution assays\nHEK293T cells were transiently transfected with a mix consisting of polyethyleneimine (Polysciences) and constructs for one of the ciliary tip module proteins. Protein purification was performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The cells were harvested 28 hours after transfection in lysis buffer (50 mM HEPES, 300 mM NaCl, 1 mM MgCl2, 1 mM DTT, 0.5% Triton X-100, pH 7.4) supplemented with protease inhibitors (Roche) and kept on ice for 15 minutes. After clearance of the debris by centrifugation, the supernatant was incubated with 20 microl StrepTactin beads (GE Healthcare) for 45 min. After several washing steps, five times with a 300 mM salt wash buffer (50 mM HEPES, 300 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4) and three times with a 150 mM salt wash buffer (50 mM HEPES, 150 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4), the protein was eluted in elution buffer (50 mM HEPES, 150 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4, 2.5 mM d-Desthiobiotin (Sigma-Aldrich)). Purified proteins were snap-frozen and stored at -80degreesC.\n\nMass spectrometry\nTo confirm each protein was purified, and the eluted protein did not contain any contaminants or known interactors that could affect MT dynamics, we performed mass spectrometry (MS) analysis. MS measurements were performed as described previously (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The purified protein sample was digested using S-TRAP microfilters (ProtiFi) according to the manufacturer's protocol. Digested peptides were eluted and dried in a vacuum centrifuge before liquid chromatography-mass spectrometry (LC-MS) analysis. The samples were analyzed by reversed-phase nLC-MS/MS using an Ultimate 3000 UHPLC coupled to an Orbitrap Exploris 480 mass spectrometer (Thermo Scientific). Digested peptides were separated using a 50 cm reversed-phase column packed in-house (Agilent Poroshell EC-C18, 2.7 microm, 50cm x 75 microm) and eluted from the column at a flow rate of 300 nL/min. MS data were acquired using a data-dependent acquisition (DDA) method with a MS1 resolution of 60,000 and mass range of 375-1600m/z. Fragmentation spectra were collected at a resolution of 15,000 using an HCD of 28, isolation window of 1.4m/z, and a fixed first mass of 120m/z. MS/MS fragment spectra were searched using Sequest HT in Proteome Discoverer (Thermo Scientific) against a human database (UniProt, year 2020) that was modified to contain the tagged protein sequence from each ciliary tip module protein and a common contaminants database. Tryptic miss cleavage tolerance was set to 2, fixed modifications were set to cysteine carbamidomethylation, and variable modifications included oxidized methionine and protein N-terminal acetylation. Peptides that matched the common contaminate database were filtered out from the results table.\n\nIn vitro reconstitution assays\n\nMT seed preparation\nDouble-cycled GMPCPP-stabilized MT seeds were used as templates for MT nucleation in vitro assays. GMPCPP-stabilized MT seeds were prepared as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). A tubulin mix consisting of 70% unlabeled porcine brain tubulin, 18% biotin-labeled porcine tubulin and 12% HiLyte-488-, rhodamine-, or HiLyte647-labeled porcine tubulin (all from Cytoskeleton) was incubated with 1 mM GMPCPP (Jena Biosciences) at 37degreesC for 30 minutes. Polymerized MTs were then pelleted using an Airfuge for 5 min at 199,000 x g and subsequently depolymerized on ice for 20 min. Next, 1 mM GMPCPP was added and MTs were let to polymerize again at 37degreesC for 30 minutes. Polymerized MTs were again pelleted (as above) and diluted tenfold in MRB80 buffer containing 10% glycerol prior to snap-freezing to store them at -80degreesC.\n\nIn vitro reconstitution assays\nIn vitro assays with dynamic MTs were performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Microscopic slides were prepared by adding two strips of double-sided tape to mount plasma-cleaned glass coverslips. The coverslips were functionalized by sequential incubation with 0.2 mg/mL PLL-PEG-biotin (Susos AG, Switzerland) and 1 mg/mL neutravidin (Invitrogen) in MRB80 buffer (80 mM piperazine-N, N[prime]-bis (2-ethane sulfonic acid), pH 6.8, supplemented with 4 mM MgCl2, and 1 mM EGTA). Then, GMPCPP-stabilized MT seeds were attached to the coverslips through biotin-neutravidin interactions. The coverslip was blocked with 1 mg/mL kappa-casein before the reaction mix was flushed in. The reaction mix consisted of different concentrations and combinations of fluorescently labeled purified proteins in MRB80 buffer supplemented with 15 microM porcine brain tubulin (100% dark porcine brain tubulin when 20 nM GFP-EB3 or mCherry-EB3 was added, or 97% dark porcine brain tubulin with 3% rhodamine- or HiLyte-488-labeled porcine tubulin), 0.1% methylcellulose, 1 mM GTP, 50 mM KCl, 0.2 mg/mL kappa-casein, and oxygen scavenger mix [50 mM glucose, 400 microg/mL glucose oxidase, 200 microg/mL catalase and 4 mM DTT]. This mix was spun down in an Airfuge for 5 min at 119,000 x g before addition to the flow chamber and the flow chamber was sealed with vacuum grease. Microtubules were imaged immediately at 30degreesC using a total internal reflection fluorescence (TIRF) microscope.\n\nIn vitro assays for cryo-ET\nSamples were prepared as above for reconstitution assays with slight modifications. Instead of flow chambers, all steps occur in a tube. Reaction mixes consisted of either just 20nM EB3 or all CTM components (20nM TOGARAM1, 20nM CEP104, 20nM CSPP1, 20nM CCDC66, 200nM ARMC9) with 20nM EB3 in MRB80 buffer supplemented with 15 microM porcine brain tubulin, 1 mM GTP, 0.2 mg/mL kappa-casein, and 15 microM DTT. After centrifugation of the reaction mix for dynamic MTs, GMPCPP-stabilized seeds and ProtA-Au5 fiducials (1:20) were added, and MTs were left to polymerize for 10-20 min at 30degreesC in an Eppendorf tube. Subsequently, the sample was gently resuspended and 4 microl was transferred to a glow-discharged holey carbon R2/1 copper grid (Quantifoil Micro Tools) suspended in the chamber of a Vitrobot (Thermo Fisher Scientific). The sample was incubated for 2 min inside the Vitrobot chamber, equilibrated at 30 degreesC and 95% relative humidity, to allow for potential MT repolymerization after sample transfer to the grid. Subsequently, grids were manually back-blotted for 3 s and plunge-frozen in liquid ethane. Vitrified grids were clipped into auto-grid cartridges and stored in liquid nitrogen until future use.\n\nSingle-molecule fluorescence intensity analysis\nProtein samples of GFP, GFP-EB3 and either GFP-CEP104, GFP-TOGARAM1 or GFP-ARMC9 were diluted in PBS and immobilized in adjacent flow chambers of the sample plasma cleaned glass coverslip as described in (&lt;&lt;REF:Sharma-2016-37-362&gt;&gt;). Flow chambers were then sealed with vacuum grease and immediately imaged using TIRF microscopy. 10-20 images (per condition) of previously unexposed coverslip areas were acquired with 200 ms exposure time.\n\nCEP104 molecule counting at blocked MT plus ends\nTo determine the number of CEP104 molecules at blocked MT plus ends we immobilized GFP in one chamber of the same plasma cleaned glass coverslip and in vitro reconstitution assay in the adjacent chamber (as described above). Chambers were sealed and immediately imaged using TIRF microscope. First images on unbleached GFP single molecules were acquired, then using the same illumination conditions, images of unexposed GFP-CEP104 bound MTs were acquired.\n\nFluorescence Recovery After Photobleaching\nFor FRAP experiments, in vitro reconstitutions were prepared as described above and imaging was conducted using TIRF microscopy. Either focused 488-laser (for GFP-EB3) or 561-laser (for mCherry-CEP104) was used to bleach specific regions of the MT lattice.\n\nMicroscopy\n\nTIRF microscopy\nIn vitro reconstitution assays imaged on a previously described (iLas2) TIRF microscope setup (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). We used the iLas3 system (Gataca Systems (France)) which is a dual laser illuminator for azimuthal spinning TIRF (or Hilo) illumination. This system was installed on Nikon Ti microscope (with the perfect focus system, Nikon), equipped with 489 nm 150 mW Vortran Stradus 488 laser (Vortran) and 100 mW 561 nm OBIS laser (Coherent), 49002 and 49008 Chroma filter sets. Additionally, a CCD camera CoolSNAP MYO (Teledyne Photometrics) was installed and the set up was controlled with MetaMorph 7.10.2.240 software (Molecular Device). To keep the in vitro samples at 30 degreesC, a stage top incubator model INUBG2E-ZILCS (Tokai Hit) was used. The final resolution using CCD camera was 0.045 mum/pixel. For all assays MT dynamics was measured at 200 ms exposure and 3 second intervals for 10 minutes. For EB3 FRAP experiments continuous imaging was used with 100 ms exposure. For kinesin experiments first a 10 minute movie was acquired at 200 ms exposure and 3 second intervals to measure MT dynamics, subsequently Dm-KHC(1-421)-SNAP-6xHis (gift from Kapitein lab, Addgene plasmid #196976) labelled with Alexa647-SNAP dye (NEB) was continuous imaged using 100 ms exposure.\n\nCryo-ET\nVitrified in vitro reconstituted MTs were imaged on a Talos Arctica transmission electron microscope (200 keV) (Thermo Fisher Scientific), equipped with a K2 summit direct electron detector (Gatan) and post-column energy filter aligned to the zero-loss peak with 20 keV slit width. Image acquisition was controlled by Serial-EM (&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;). MT ends were located on TEM overview images, acquired at 4,100x magnification (33.3 A/pixel). Tilt series were collected at 63,000x magnification (2.17 A/pixel) with a dose rate of ~5 e-/pixel/s and a total dosage of 100 e-/A2. Tilt images were recorded using a dose symmetric scheme (&lt;&lt;REF:Hagen-2017-197-191&gt;&gt;) over a tilt range of 54degrees to -54degrees or 60degrees to -60degrees with a tilt increment of 2degrees, at a defocus target of -3 mum.\n\nImage analysis\n\nAnalysis of MT plus-end dynamics in vitro\nAnalysis of MT plus-end dynamics was performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Movies of dynamic MTs were corrected for drift, and kymographs were generated using the plugin KymoResliceWide v.0.4 in ImageJ (https://github.com/ekatrukha/KymoResliceWide). MT tips were traced with lines, and measured lengths and angles were used to calculate the MT dynamics parameters such as growth rate and transition events. All events with growth rates faster than 0.02 microm/min and slower than 0.5 microm/min were categorized as slow growing, events faster than 0.5 microm/min were categorized as fast growing, events with shrinkage rates faster than 0.02 microm/min were categorized as shrinking. The events with slower growth rates or faster shrinkage rates than the before mentioned rates were categorized as pause events. Transition frequencies were calculated by dividing the sum of the transition events per experiment by the total time this event could have occurred. Weighted growth rate histograms were calculated by taking individual growth durations and dividing by total growth time.\n\nAnalysis of FRAP experiments\nData were normalized to 1 for image acquired immediately before the FRAP event and 0 for the image acquired immediately after the FRAP event using the following equation; Normalized intensity=(intensity t=x - intensity t=1)(intensity t=0 - intensity t=1). For EB3, non-linear fit curves were fitted to the post-bleach intensities and half-life calculated using GraphPad Prism.\n\nAnalysis of single molecule fluorescence intensities\nSingle molecule fluorescence spots of proteins immobilized directly on the cover glass were detected and fitted with 2D Gaussain function using custom written ImageJ plugin DoM-Utrect (https://github.com/ekatrukha/DoM_Utrecht). Fitted peak intensities were then used to create intensity histograms. CEP104-blocked plus ends were then manually annotated and fitted with 2D Gaussian function using the same plugin.\n\nCryo-ET 3D volume reconstruction and analysis\nTomogram reconstruction and denoising was performed using an in house Python script, as described previously (&lt;&lt;REF:Chaillet-2023-24-13375&gt;&gt;). The tilt series' frames were initially aligned and dose-weighted using MotionCor2 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;). Subsequent tilt series alignment and tomographic reconstruction for denoising, visualization and analysis were done through AreTomo 1.3.3. Final reconstructed tomographic volumes were created through weighted back-projection and binned with a factor of two. Cryo-CARE 0.2.2 was used for tomogram denoising (&lt;&lt;REF:Buchholz-2019-152-277&gt;&gt;). For this, even and odd tomographic reconstructions were generated through splitting of the movie frames for each individual tilt into even and odd summed frames with MotionCor2 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;). Alignment parameters were first calculated on the full tilt series and subsequently applied to the even and odd stacks. The cryo-CARE network was trained on 5 tomograms per dataset, and then applied to the entire set.\nFor visualization purposes, semi-automated segmentation was performed on denoised tomograms using the EMAN2 2.91 tomoseg module (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). For this, separate neural networks were trained for the two features of interest: \"MT\" and \"ciliary tip module\". Each feature was included as 'negative' training reference, together with background noise and ice contamination, for the neural network training of the other feature. Final visualization and 3D rendering was performed in ChimeraX.\nMT ends, both free and CTM-plugged, were only analyzed for protofilament shape in case they were plus ends. MT polarity was determined through 2D rotational averaging with applied symmetry, using the EMAN2 program e2proc2d. Manual 3D tracing of protofilament shapes at MT plus ends was performed, as previously described (&lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;; &lt;&lt;REF:McIntosh-2020-2101-191&gt;&gt;; &lt;&lt;REF:Ogunmolu-2021-?-2021.2012.2029.474442&gt;&gt;), in the IMOD 4.9.6 program 3dmod (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;). Manual tracing was performed on tomographic subvolumes, generated with the mtrotlong program of IMOD (4.11.20). Traces of individual protofilaments were saved as separate contours within one object per MT. To satisfy the three-point requirement of howflared, for blunt end protofilaments a third point was added one pixel upstream of the second point that indicated the last protofilament segment in the MT wall. Subsequently, protofilament coordinates were extracted using the howflared program of IMOD (4.11.20). MT raggedness and protofilament length parameters, as described in the results section, were calculated by howflared. Curvature analysis was performed on the howflared output with Matlab scripts available at https://github.com/ngudimchuk/Process-PFs.\n\nStatistical analysis\nStatistical analysis was performed using GraphPad Prism 9. Figure legends contain statistical details of each experiment, including the statistical tests used, the number of measurements and the number of experiments.\n\nCode availability\nThe python script for automated tomogram reconstruction and cryo-CARE denoising is available online at https://github.com/SBC-Utrecht/cryocare-from-movies. MATLAB scripts for analysis of protofilament tracings in IMOD are available online at https://github.com/ngudimchuk/Process-PFs.\n\nAuthor Contributions\nH.A.J.S. and C. M. vdB. performed experiments, analyzed data and wrote the paper, R.A.H. performed and analyzed cryo-electron tomography experiments, and wrote the paper, D.S. performed immunoprecipitation experiments, K.E.S. performed and analyzed mass spectrometry experiments, S.H. supervised and analyzed cryo-electron tomography experiments, R.R has generated essential reagents and acquired funding, A.A. coordinated the project, acquired funding and wrote the paper. All authors reviewed and edited the paper.\n\nCompeting financial interests\nThe authors declare no competing financial interests.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
[outputs]
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nCilia are motile or sensory organelles present on the surface of many eukaryotic cells. Motile cilia generate movement for processes such as mucus flow in airways or swimming of unicellular green algae, while primary cilia function as signaling hubs with fundamental roles in organism development (&lt;&lt;REF:Klena-2022-38-103&gt;&gt;; &lt;&lt;REF:Mill-2023-24-421&gt;&gt;). Ciliary dysfunction is associated with numerous diseases, collectively called ciliopathies, which affect multiple organs and tissues (&lt;&lt;REF:Badano-2006-7-125&gt;&gt;; &lt;&lt;REF:Hildebrandt-2011-364-1533&gt;&gt;; &lt;&lt;REF:Klena-2022-38-103&gt;&gt;). The core of each cilium is formed by a microtubule (MT)-based structure, the axoneme, which is often disrupted in ciliopathies (&lt;&lt;REF:Deretic-2023-136-?&gt;&gt;). Diverse proteins and pathways responsible for axoneme formation have been identified through genetic screens and cell biology studies, yet the biochemical mechanisms governing the dynamics of axonemal MTs remain very poorly understood (&lt;&lt;REF:Keeling-2016-5-6&gt;&gt;).\nUnlike cytoplasmic MTs, axonemal plus ends grow very slowly likely impeding the formation of a stabilizing GTP cap, which is required for preventing MT disassembly (&lt;&lt;REF:Desai-1997-13-83&gt;&gt;) and efficient tubulin addition (&lt;&lt;REF:Wieczorek-2015-17-907&gt;&gt;). The absence of a stable GTP cap can be compensated by specific factors bound to MT ends. Excellent candidates for this function in cilia are the components of the recently identified ciliary tip interaction network or \"module\", CEP104, CSPP1, TOGARAM1, ARMC9 and CCDC66 (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Depletion of any of these proteins causes ciliary defects, and mutations in most of them lead to a primary ciliopathy called Joubert syndrome, characterized by brain malformations, breathing problems or more severe developmental disorders (&lt;&lt;REF:Akizu-2014-94-80&gt;&gt;; &lt;&lt;REF:Hildebrandt-2011-364-1533&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Shaheen-2014-94-73&gt;&gt;; &lt;&lt;REF:Srour-2015-97-744&gt;&gt;; &lt;&lt;REF:Tuz-2014-94-62&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;). The proteins comprising the ciliary tip module interact with MTs and with each other, and although they do not form a stoichiometric complex, they are all important for controlling ciliary length and the signaling pathways dependent on primary cilia (&lt;&lt;REF:Conkar-2017-130-1450&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Patzke-2006-209-199&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;). In addition to their roles in cilia, some of the tip module proteins also associate with centrioles, centrosomes, centriolar satellites and cytoplasmic MTs, where they participate in cilia-independent processes, such as cell division (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;; &lt;&lt;REF:Conkar-2019-9-14250&gt;&gt;; &lt;&lt;REF:Patzke-2006-209-199&gt;&gt;; &lt;&lt;REF:Ryniawec-2023-33-4202&gt;&gt;; &lt;&lt;REF:Zhu-2015-290-27053&gt;&gt;).\nCEP104 (also known as FAP256), TOGARAM1 (also known as Crescerin or CHE-12) and ARMC9 are conserved across eukaryotes, including Chlamydomonas, Tetrahymena and C. elegans, where they participate in the biogenesis of motile or sensory cilia (&lt;&lt;REF:Bacaj-2008-178-989&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;; &lt;&lt;REF:Satish Tammana-2013-126-5018&gt;&gt;). Both CEP104 and TOGARAM1 have evolutionary conserved TOG domains (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;). TOGARAM1 contains four TOG domains, two of which can promote tubulin polymerization in light scattering experiments (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;), and the same is true for the single tubulin-binding TOG domain of CEP104 (&lt;&lt;REF:Al-Jassar-2017-25-146&gt;&gt;; &lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Loss of function mutations in TOGARAM1 and CEP104 in multiple organisms result in short cilia (&lt;&lt;REF:Bacaj-2008-178-989&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;; &lt;&lt;REF:Satish Tammana-2013-126-5018&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Similarly, loss of CSPP1, a protein that increases MT stability by binding inside the MT lumen and promoting pausing (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;; &lt;&lt;REF:Wang-2024-22-mjae007&gt;&gt;), results in short cilia (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;). ARMC9 on its own cannot bind to MTs but acts as a scaffold for other ciliary tip module components (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). ARMC9 and TOGARAM1 have been localized to motile cilia, where they were found to have opposite functions in regulating MT length (&lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;), though mutants of both proteins make mammalian primary cilia shorter (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Finally, CSPP1 and CEP104 interact with CCDC66, and co-depletion of CCDC66 with either CSPP1 or CEP104 results in further reduction of ciliary length compared to the removal of either CSPP1 or CEP104 alone (&lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;).\nFigure 1.\nCiliary tip module proteins have distinct effects on MT dynamics in vitro.\n(A) Schematic representation of ciliary tip module members and summary table highlighting individual protein effects on MT dynamics. Jelly-roll, JR; zinc finger, ZNF; MT binding, MTB; pause domain, PD; alpha-helical domain, H.\n(B-C) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488-labelled tubulin or 20 nM GFP-EB3.\n(D-F) Parameters of MT plus-end dynamics in the presence of 15 microM tubulin alone or with 20 nM EB3 in combination with indicated concentrations of proteins (from kymographs shown in B-C, I-P, and S1C). (D) Bars represent pooled data from three independent experiments (growth rate) or averaged means from three independent experiments (pause and block duration), total number of growth events, pauses or blocks: tubulin alone, n=356, 0; 10 nM CCDC66, n=411, 0; 10 nM CEP104, n=306, 0; 100 nM CEP104, n=0, 138; 10 nM CSPP1,n=422, 97; 10 nM TOGARAM1, n=347, 0; EB3 alone, n=938, 0; EB3 with 10 nM CCDC66, n=562, 0; EB3 with 10 nM CEP104, n=213, 101; EB3 with CSPP1, n=1715, 273; EB3 with TOGARAM1, n=861, 0. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test, all conditions were compared to their relevant control (either tubulin alone or tubulin with 20 nM EB3). (E-F) Bars represent averaged means from three independent experiments. Error bars represent s.e.m.. Movies were acquired for 10 minutes, therefore this is the maximum time for pause duration,\n(G-P) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488 or Rhodamine-labelled tubulin, or 20 nM GFP- or mCherry-EB3, and indicated concentrations and colors of ciliary tip module proteins. Orange arrowheads, blocked plus ends; orange arrows, CEP104 tracking minus ends; yellow arrowheads, pauses; blue arrowheads, rescues.\nTo uncover biochemical mechanisms underlying the function of these proteins and their combined regulation of ciliary MTs, we reconstituted in vitro their individual and collective effects on MT dynamics. Although the individual TOG domain of CEP104 was previously shown to promote MT polymerization (&lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;), we found that the full-length protein specifically inhibited MT plus-end elongation and shortening. This activity was potentiated by EB3, CSPP1 and CCDC66, which could all recruit CEP104 to MTs. This was surprising as EB3 and CSPP1 bind to the outer and inner surface of the MT wall, respectively (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;; &lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). TOGARAM1 functioned as a polymerase that converted the inhibition of MT polymerization imposed by CEP104 into slow growth. ARMC9 and CCDC66 behaved as scaffolds, enhancing the effects of their binding partners. The combination of all five ciliary tip module proteins resulted in very slow and processive MT plus-end elongation, an effect that required TOGARAM1 and became less robust when one of the other proteins was left out. MT growth rates in these conditions were in the same range as initial elongation rates of regenerating flagella of single-celled organisms (&lt;&lt;REF:Rosenbaum-1967-34-345&gt;&gt;; &lt;&lt;REF:Rosenbaum-1969-41-600&gt;&gt;; &lt;&lt;REF:Witman-1975-253-178&gt;&gt;). Cryo-electron tomography showed that together, ciliary tip proteins formed a champagne cork-like structure that protruded from MT plus ends and diminished protofilament flaring. Altogether, our findings demonstrate that, through a combination of opposing activities, ciliary tip module components can stabilize MT plus ends and drive their slow but persistent elongation.\n\nResults\n\nCiliary tip module proteins have distinct effects on MT dynamics in vitro\nTo characterize the effects of ciliary tip module proteins on MT polymerization, we N-terminally tagged them with mCherry or GFP and purified them from transiently transfected HEK293 cells (Fig. 1A &lt;&lt;FIG:fig1&gt;&gt;, S1A &lt;&lt;FIG:figs1&gt;&gt;). Mass spectrometry-based analysis demonstrated that the protein preparations contained only very minor contaminations with other known regulators of MT dynamics, such as CSPP1 (Fig. S1B &lt;&lt;FIG:figs1&gt;&gt;). In addition, the heat shock protein Hsp70, a protein that to our knowledge has no effect on MT dynamics, was present in the preparations of CCDC66 and CSPP1 (Fig. S1B &lt;&lt;FIG:figs1&gt;&gt;). We used these proteins to perform in vitro reconstitution assays with MTs grown from GMPCPP-stabilized seeds and observed their impact on MT dynamics by Total Internal Reflection Fluorescence (TIRF) microscopy (&lt;&lt;REF:Bieling-2007-450-1100&gt;&gt;; &lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Although axonemes are composed of MT doublets, single MTs are appropriate substrates to study the effects of ciliary tip proteins, because doublets become singlets at ciliary tips (&lt;&lt;REF:Fisch-2011-103-249&gt;&gt;; &lt;&lt;REF:Kiesel-2020-27-1115&gt;&gt;; &lt;&lt;REF:Legal-2023-222-e202301129&gt;&gt;). To detect MTs, we used either fluorescent tubulin or fluorescently tagged EB3, which increases MT growth rate and induces catastrophes (&lt;&lt;REF:Komarova-2009-184-691&gt;&gt;) (Fig. 1B-F &lt;&lt;FIG:fig1&gt;&gt;). Studying the effects of ciliary tip proteins in the presence of EBs is relevant, because EB1 and EB3 localize to axonemal MTs in both primary and motile cilia (&lt;&lt;REF:Kiesel-2020-27-1115&gt;&gt;; &lt;&lt;REF:Pedersen-2003-13-1969&gt;&gt;; &lt;&lt;REF:Schroder-2007-17-1134&gt;&gt;).\nFigure S1 (Related to Figure 1).\nCharacterization of ciliary tip module proteins in vitro.\n(A) Analysis of purified GFP-tagged ciliary tip module proteins by SDS-PAGE. Asterisks indicate the full-length protein bands. Protein concentrations were determined from a BSA standard (not shown).\n(B) Mass spectrometry analysis of purified GFP-tagged ciliary tip module proteins. For all module members except TOGARAM1 the human sequence was used, due to technical difficulties the mouse sequence of TOGARAM1 was used, it is 84% identical to human TOGARAM1.\n(C) Fields of view (top, scale bar 2 microm) and kymograph (bottom, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with HiLyte-488-tubulin and mCherry-CEP104.\n(D) Kymographs illustrating mobility of DmKHC(1-421) on CEP104-blocked MT labelled with HiLyte-488-tubulin and mCherry-CEP104 (top) and DmKHC (bottom) proving that the blocked end of the MT is the plus end. Scale bars 2 microm and 60 s for both kymographs.\nAmong the tested proteins, ARMC9 was the only one that displayed no MT binding even when present at 300 nM (Fig. 1G &lt;&lt;FIG:fig1&gt;&gt; and H &lt;&lt;FIG:fig1&gt;&gt;), in agreement with data in cells (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). CCDC66, which is known to interact with MTs (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;), bound along the MT lattice already at 10 nM and slightly decreased the growth rate, but did not affect the frequencies of catastrophes and rescues and did not induce pausing (Fig. 1D-F, I &lt;&lt;FIG:fig1&gt;&gt; and J &lt;&lt;FIG:fig1&gt;&gt;).\nCEP104 could not autonomously bind MTs at 10 nM in assays with tubulin alone (Fig. S1C &lt;&lt;FIG:figs1&gt;&gt;). However, at 100 nM it efficiently blocked MT growth specifically at the plus end (Fig. 1D-F &lt;&lt;FIG:fig1&gt;&gt; and K &lt;&lt;FIG:fig1&gt;&gt;), as confirmed by including in the assay a constitutively active fragment of the plus-end directed Kinesin-1 DmKHC(1-421) (Fig. S1D &lt;&lt;FIG:figs1&gt;&gt;). In the presence of its binding partner EB3 (&lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;), CEP104 could specifically inhibit MT plus-end polymerization already at 10 nM (Fig. 1D-F &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt;). CEP104 colocalized with EB3 at some growing plus and occasionally also minus ends, however, this did not alter growth rates at the plus end (Fig.1D &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt;). Once MT growth was arrested, no regrowth was detected for the remainder of the experiment (Fig 1E, K &lt;&lt;FIG:fig1&gt;&gt;, and L &lt;&lt;FIG:fig1&gt;&gt;).\nEpisodes of inhibited growth, albeit transient ones, were also observed with CSPP1, as we have shown previously (Fig. 1D-F, M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;) (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Both with and without EB3 CSPP1 preferentially binds at both plus and minus ends when they undergo growth perturbations (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;; &lt;&lt;REF:Wang-2024-22-mjae007&gt;&gt;), and therefore it displayed discreet sites of increased lattice accumulation (Fig. 1M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;). At the plus ends, CSPP1 accumulations decreased growth rate, induced pauses and facilitated transitions from pause to growth, whereas pausing was less obvious at the minus ends due to their overall slower dynamics (Fig. 1D-F, M &lt;&lt;FIG:fig1&gt;&gt; and N &lt;&lt;FIG:fig1&gt;&gt;).\nFinally, TOGARAM1 bound to MT plus ends, reduced their growth rate and induced transitions from shrinkage to growth (rescues), and these effects were not altered by the presence of EB3 (Fig. 1D-F, O &lt;&lt;FIG:fig1&gt;&gt; and P &lt;&lt;FIG:fig1&gt;&gt;). A decrease in MT growth rate was unexpected, as TOGARAM1 structurally resembles the MT polymerase ch-TOG/XMAP215 (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;), which strongly accelerates MT polymerization (&lt;&lt;REF:Brouhard-2008-132-79&gt;&gt;). Altogether, we show that all the full-length members of the ciliary tip module can be purified and used for in vitro assays, where they display opposing effects on MT dynamics.\n\nPlus-end blocking by CEP104 depends on the TOG domain and is potentiated by EB3, CCDC66 and CSPP1\nAmong the ciliary tip proteins analyzed above, the effect of CEP104 was the most unexpected as it was thought to be a MT polymerase (&lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;), and we therefore set out to dissect it in more detail. Single molecule counting experiments demonstrated that CEP104 forms a dimer, in agreement with the sedimentation profile of its first helical domain (Fig. 2A &lt;&lt;FIG:fig2&gt;&gt;) (&lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;). In the presence of EB3, ~5-6 CEP104 molecules (2-3 dimers) were sufficient to block MT plus-end growth (Fig. 2B &lt;&lt;FIG:fig2&gt;&gt;). In these conditions, Fluorescence Recovery after Photobleaching (FRAP) showed that CEP104 did not exchange at blocked MT plus ends (Fig. 2C &lt;&lt;FIG:fig2&gt;&gt;), whereas EB3, associated with the same ends, did partially turn over, albeit significantly slower than on growing MT plus ends (Fig. 2D &lt;&lt;FIG:fig2&gt;&gt; and E &lt;&lt;FIG:fig2&gt;&gt;). This could be explained by the direct interaction of EB3 and CEP104 on the MT through the SxIP motif of CEP104, SKIP (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt;) (&lt;&lt;REF:Jiang-2012-22-1800&gt;&gt;). Mutation of this motif has previously been shown to abolish the interactions between the C-terminus of EB and its partners (&lt;&lt;REF:Honnappa-2009-138-366&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Indeed, CEP104 SKNN double mutation prevented EB3-dependent recruitment to MTs; therefore, even in the presence of EB3, 100 nM CEP104 SKNN was needed to block plus-end growth (Fig. 3A-D &lt;&lt;FIG:fig3&gt;&gt;, and S2A &lt;&lt;FIG:figs2&gt;&gt;).\nFigure S2 (Related to Figure 3).\nCharacterization of CEP104 constructs in vitro.\n(A) Analysis of purified GFP-tagged CEP104 constructs by SDS-PAGE. Asterisks show protein bands. Protein concentrations were determined from a BSA standard (not shown).\n(B) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with either HiLyte-488-tubulin or GFP-EB3 and indicated concentrations and colors of CEP104 constructs.\n(C) Co-immunoprecipitation of either CSPP1 (left) or CCDC66 (right) with indicated CEP104 constructs, both CSPP1 and CCDC66 interact with the jelly-roll domain of CEP104.\n(D) Analysis of purified GFP-tagged CEP104 DeltaJR construct by SDS-PAGE. Asterisk indicates the full-length protein band. Protein concentration was determined from a BSA standard (not shown).\nFigure 2.\nA few CEP104 molecules stably block MT plus ends\n(A) Fields of view (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules, GFP-EB3 dimers, and GFP-CEP104 dimers immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=29981; GFP-EB3, n=55378; GFP-CEP104, n=32335. Scale bar 2 microm.\n(B) Representative MT with CEP104-blocked plus end (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules and GFP-CEP104 intensity at blocked plus end immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=13925; GFP-CEP104, n=92. Scale bar 2 microm.\n(C-E) FRAP analysis of CEP104 (C) and EB3 (D) at blocked MT plus ends, or dynamic EB3 at growing plus ends (E). Arrowhead marks point of photobleaching in representative kymographs, scale bars 2 microm and 60 s (C), 2 microm and 10 s (D, E). Plots show averaged curves with exponential fit. Number of FRAP measurements: CEP104 n=17; stationary EB3 n=14; dynamic EB3 n=10. Error bars represent s.e.m..\nFigure 3.\nPlus-end blocking by CEP104 is potentiated by EB3, CCDC66 and CSPP1.\n(A) Schematic representation of different CEP104 constructs; jelly-roll, JR; alpha-helical domain, H; zinc finger, ZNF.\n(B) Percentage of time MT plus ends spent blocked in the presence of 20 nM EB3 in combination with CEP104 constructs (from kymographs shown in C-E, 1C, 1L). Bars represent averaged means from three independent experiments. Error bars represent s.e.m..\n(C-F) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte488 or Rhodamine-labelled tubulin or, 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module constructs. Orange arrowheads, blocked plus ends; white arrowheads, CEP104 DeltaTOG at plus ends of seeds; white arrows, CEP104 tracking the minus end.\n(G) Parameters of MT plus-end dynamics in the presence of 15 microM tubulin alone or with 20 nM EB3 in combination with indicated concentrations of ciliary tip module proteins (from kymographs shown in F, H-L, S2B &lt;&lt;FIG:figs2&gt;&gt;). For dynamic state bars represent averaged means from three independent experiments. For depolymerization rate bars represent pooled data from three independent experiments: 2 nM CEP104, n=133; 2 nM CEP104 with 15 nM CCDC66, n=23; 10 nM CEP104 with 15 nM CCDC66, n=122; 2 nM CEP104 with 10 nM CSPP1, n=69; 10 nM CEP104 with 10 nM CSPP1, n=99; EB3 with 2 nM CEP104, n=1; EB3 with 2 nM CEP104 and 15 nM CCDC66, n=63; EB3 with 2 nM CEP104 and 10 nM CSPP1, n=116.\n(H-N) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with either 15 microM tubulin supplemented with 3% HiLyte-488- or Rhodamine-labelled tubulin, or 20 nM GFP or mCherry-EB3 and indicated concentrations and colors of ciliary tip module constructs. Light orange arrowheads, blocked seeds; dark orange arrowhead, blocked lattice; blue arrowheads, slow depolymerization of plus ends.\n(O) Percentage of time of MT plus ends spent blocked at the seed in the presence of 20 nM EB3 in combination with CEP104 DeltaJR and other ciliary tip module proteins (from kymographs shown in M, N, P, S2B &lt;&lt;FIG:figs2&gt;&gt;). Bars represent averaged means from three independent experiments.\n(P) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- and 50 nM mCherry-CEP104 DeltaJR. Orange arrowheads, blocked plus ends.\n(Q) Schematic model showing CEP104 domain functions and interactions with other ciliary tip module proteins. Jelly-roll, JR; zinc finger, ZNF; alpha-helical domain, H.\nFor (B, G and O) error bars represent s.e.m. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\nWhereas the SxIP motif was not essential for plus-end blocking by CEP104, the single TOG domain of CEP104 was required to inhibit MT growth. CEP104 lacking the TOG domain (Fig. 3A &lt;&lt;FIG:fig3&gt;&gt; and S2A &lt;&lt;FIG:figs2&gt;&gt;) could still be recruited to MTs by EB3, but no plus-end blocking was observed (Fig. 3B &lt;&lt;FIG:fig3&gt;&gt; and E &lt;&lt;FIG:fig3&gt;&gt;). In the presence of EB3, CEP104 DeltaTOG could bind to growing MT ends and accumulate at the border between the seed and the plus-end-grown lattice (Fig. 3E &lt;&lt;FIG:fig3&gt;&gt;), a localization for which we currently have no explanation.\nSince recent studies described functional and biochemical interactions between CEP104, CCDC66, CSPP1 and ARMC9 (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;), we next tested in vitro the combinations of these proteins. We found that in the presence of 15 nM CCDC66, already 2 nM CEP104 was sufficient to occasionally block MT plus end outgrowth from the seed (Fig. 3F &lt;&lt;FIG:fig3&gt;&gt; and G &lt;&lt;FIG:fig3&gt;&gt;). 2 nM CEP104 on its own, or in the presence of EB3, or together with ARMC9, showed hardly any MT binding (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;). However, CEP104 driven growth inhibition at MT seeds became much more common at 10 nM CEP104 with 15 nM CCDC66, and in these conditions, CEP104 and CCDC66 could also strongly inhibit depolymerization of dynamic MTs by either pausing the plus ends or reducing their shrinkage rate from 36.91+/-1.34 microm/min to 0.12+/-0.02 microm/min (Fig. 3G &lt;&lt;FIG:fig3&gt;&gt; and H &lt;&lt;FIG:fig3&gt;&gt;). Similar effects were observed when either 2 nM or 10 nM CEP104 was combined with 10 nM CSPP1: both growth inhibition at the seed and at the dynamic plus ends, as well as slow depolymerization were observed (Fig. 3G, I &lt;&lt;FIG:fig3&gt;&gt; and J &lt;&lt;FIG:fig3&gt;&gt;). These data indicate that CEP104 is not just a MT growth inhibitor or a depolymerase that can prevent tubulin addition to the plus ends, but also a specific MT stabilizing factor that can inhibit tubulin loss from MT plus ends lacking a GTP cap. The addition of EB3 had no strong impact on MT dynamics when combined with 2 nM CEP104 and 15 nM CCDC66 (Fig. 3F, G &lt;&lt;FIG:fig3&gt;&gt; and K &lt;&lt;FIG:fig3&gt;&gt;), but potentiated seed blocking when included with 2 nM CEP104 and 10 nM CSPP1, as most MTs were blocked at the seed in these conditions (Fig. 3G, I &lt;&lt;FIG:fig3&gt;&gt; and L &lt;&lt;FIG:fig3&gt;&gt;).\nBased on our own and previously published co-immunoprecipitation experiments, the interactions of CCDC66 and CSPP1 with CEP104 depended on its jelly-roll (JR) domain (Fig. S2C &lt;&lt;FIG:figs2&gt;&gt;) (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;). The deletion of this domain abolished the ability of both proteins to recruit CEP104 to MTs (Fig. 3M-O &lt;&lt;FIG:fig3&gt;&gt; and S2D &lt;&lt;FIG:figs2&gt;&gt;). However, CEP104 DeltaJR could still block seed outgrowth in the presence of EB3 (Fig. 3O &lt;&lt;FIG:fig3&gt;&gt; and P &lt;&lt;FIG:fig3&gt;&gt;), though a higher concentration of CEP104 DeltaJR compared to the full-length CEP104 was needed to achieve significant plus-end blocking (50 nM versus 10 nM, compare Fig. 1D &lt;&lt;FIG:fig1&gt;&gt; and L &lt;&lt;FIG:fig1&gt;&gt; to Fig. 3O, P &lt;&lt;FIG:fig3&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;), indicating that the JR domain might directly or indirectly contribute to MT blocking.\nTaken together, our results demonstrate that CEP104 can stably and specifically associate with MT plus ends and inhibit both tubulin addition and removal from these ends in a TOG domain-dependent manner. However, the affinity of CEP104 for MTs is rather low, and its activity is strongly enhanced by association with its binding partners EB3, CCDC66 and CSPP1 that recruit it to MTs. Since EB3 binds to the outer MT surface (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;) and the same might be true for CCDC66 as it can bundle MTs (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;), whereas CSPP1 associates with the MT lumen (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;), CEP104 might be positioned at protofilament ends to allow access to both surfaces of the MT wall (Fig. 3Q &lt;&lt;FIG:fig3&gt;&gt;).\n\nThe rescue activity of TOGARAM1 depends on TOG3 and TOG4 domains\nHaving characterized the activity of CEP104, we next turned to the other TOG domain component of the ciliary tip module, TOGARAM1, which contains two pairs of TOG domains (TOG1-TOG4) connected by a long linker (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Single molecule counting experiments showed that it is a monomer (Fig. 4B &lt;&lt;FIG:fig4&gt;&gt;), similar to other MT regulators with multiple TOG domains, XMAP215 (&lt;&lt;REF:Brouhard-2008-132-79&gt;&gt;) and CLASP2 (&lt;&lt;REF:Aher-2018-46-40&gt;&gt;). A previous study showed that isolated TOG2 and TOG4, but not TOG1 and TOG3 promoted tubulin polymerization in light scattering assays (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). To test how these domains contribute to the rescue activity of the full-length protein, we took advantage of previously generated TOGARAM1 mutants where the tubulin-binding surfaces of the individual TOG domains were disrupted by point substitutions within the intra-HEAT loop of HEAT repeat A (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;) (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;). Although there is no direct interaction between TOGARAM1 and EB3, we included EB3 in our in vitro reconstitution assays as EB3 increases catastrophe frequency and therefore facilitates observing rescues (Fig 1F &lt;&lt;FIG:fig1&gt;&gt;) (&lt;&lt;REF:Komarova-2009-184-691&gt;&gt;). We found that simultaneous mutation of TOG3 and TOG4 (TOGARAM1 123'4'), but not TOG1 and TOG2 (TOGARAM1 1'2'34) abolished the rescue activity, although both mutants could still bind to MTs at 10 nM concentration (Fig. 4A, C-E &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). This was different from previous observations in cells, where the double TOG3-TOG4 mutant (TOGARAM1 123'4') could not bind to MTs (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). TOG3 and TOG4 showed redundancy within the full-length protein because single mutations in each one of them did not abolish rescues (Fig. 4A, C, F, G &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). TOG1 and TOG2 could be deleted without impairing rescue activity (the construct L-TOG-34, Fig. 4A, C, H &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;), and the combination of these two domains (TOG-12) without the adjacent linker region did not bind to MTs even at 200 nM (Fig. 4A, I &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). TOG-34 did not visibly associate with MTs at 10 nM, but at 200 nM it bound to MTs and induced rescues (Fig. 4A, C, J, K &lt;&lt;FIG:fig4&gt;&gt; and S3 &lt;&lt;FIG:figs3&gt;&gt;). Since the construct containing TOG3 and TOG4 domains together with adjacent linker (L-TOG-34) was active in our assays already at 10 nM, in agreement with previous findings, the linker region of TOGARAM1 contributes to MT affinity (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;). All tested constructs mildly reduced MT growth rate and TOG-34 constructs with or without the adjacent linker mildly suppressed catastrophes (Fig. 4C &lt;&lt;FIG:fig4&gt;&gt;). Our results demonstrate that, on its own, TOGARAM1 has relatively mild effects on MT dynamics, which rely in its two C-terminal TOG domains.\nFigure S3 (Related to Figure 4).\nCharacterization of TOGARAM1 constructs in vitro.\nAnalysis of purified GFP-tagged TOGARAM1 constructs by SDS-PAGE. Asterisks show protein bands. Protein concentrations were determined from a BSA standard (not shown).\nFigure 4.\nThe rescue activity of TOGARAM1 depends on the TOG3 and TOG4 domains.\n(A) Schematic representation of different TOGARAM1 constructs and summary table highlighting their effects on MT dynamics. Vertical lines indicate point mutations predicted to ablate tubulin-binding activity.\n(B) Fields of view (top) and histogram plot (bottom) of fluorescent intensities of single GFP molecules, EB3 dimers and TOGARAM1 molecules were immobilized in separate chambers of the same coverslip. Number of molecules analyzed: GFP, n=57865; EB3, n=73074; TOGARAM1, n=74306. Scale bar, 2 microm.\n(C) Parameters of MT plus-end dynamics in the presence of 20 nM EB3 in combination with indicated concentrations of TOGARAM1 constructs (from kymographs shown in D-H, K, 1C, 1P). For growth rates bars represent pooled data from three independent experiments, total number of growth events: EB3 alone, n=938; EB3 with TOGARAM1, n=861; EB3 with 1'2'34, n=350; EB3 with 123'4', n=337; EB3 with 123'4, n=560; EB3 with 1234', n=253; EB3 with L-TOG-34, n=131; EB3 with 200nM TOG-34, n=150. For transition frequencies bars represent average means from three independent experiments. Error bars represent s.e.m., ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(D-K) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of TOGARAM1 constructs. Blue arrowheads, rescues.\n\nARMC9 colocalizes with TOGARAM1 and CSPP1 and enhances their effects on MT dynamics\nWe next set out to characterize the function of ARMC9. Single molecule counting experiments showed that ARMC9 forms dimers dependent on its central helical region (Fig. S4A &lt;&lt;FIG:figs4&gt;&gt; and B &lt;&lt;FIG:figs4&gt;&gt;). As shown above, ARMC9 does not bind to MTs on its own nor in the presence of CEP104 and EB3 (Fig. 1G, H &lt;&lt;FIG:fig1&gt;&gt; and S2B &lt;&lt;FIG:figs2&gt;&gt;). Consistently, no direct interaction has been described for ARMC9 and CEP104, even though they were previously shown to weakly co-immunoprecipitate with each other (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;), an observation we were not able confirm (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt;). We could confirm by co-immunoprecipitation the interaction between ARMC9 and CCDC66, however when this combination was tested in reconstitution experiments we did not see any recruitment of ARMC9 to the MT lattice (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt; and D &lt;&lt;FIG:figs4&gt;&gt;). We also confirmed by co-immunoprecipitation the interaction between ARMC9 and TOGARAM1 (Fig. S4C &lt;&lt;FIG:figs4&gt;&gt;) and refined the previous mapping of this interaction (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;) by showing that it requires the armadillo (ARM) repeats of ARMC9, but not its helical domain responsible for dimerization, and the tubulin-binding surface of the TOG2 domain of TOGARAM1 (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, S4A, E &lt;&lt;FIG:figs4&gt;&gt; and F &lt;&lt;FIG:figs4&gt;&gt;). We also tested the impact of different point mutations identified in Joubert syndrome patients on the interaction of ARMC9 and TOGARAM1 (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Van De Weghe-2017-101-23&gt;&gt;), and found that all tested mutations in the TOG2 domain (all located in the first HEAT repeat) and one of the several known mutations in the ARM domain (G492R) perturb binding (Fig. S4A, G &lt;&lt;FIG:figs4&gt;&gt; and H &lt;&lt;FIG:figs4&gt;&gt;), supporting the functional importance of this interaction.\nFigure S4 (Related to Figure 5).\nMapping of ARMC9 and TOGARAM1 interaction.\n(A) Schematic representation of different ARMC9 constructs.\n(B) Histogram plots of fluorescent intensities of single GFP molecules, GFP-EB3 dimers, and full-length GFP-ARMC dimers (left) or GFP-ARMC9 DeltaH molecules immobilized in separate chambers of the same coverslips. Number of molecules analyzed, left, right: GFP, n=18415, 14914; GFP-EB3, n= 132826, 163959; GFP-ARMC9 FL, n=99873; GFP-ARMC9 DeltaH, 55479.\n(C) Co-immunoprecipitation of full-length ARMC9 with indicated proteins.\n(D) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins.\n(E-F) Co-immunoprecipitation experiments of either full-length TOGARAM1 with indicated ARMC9 constructs (E) or full-length ARMC9 with indicated TOGARAM1 point mutations\n(F). ARMC9 and TOGARAM1 interact through ARM domain and TOG2 domain, respectively.\n(G-H) Co-immunoprecipitation experiments of either full-length TOGARAM1 with indicated ARMC9 Joubert syndrome mutations (G) or full-length ARMC9 with indicated TOGARAM1 Joubert syndrome mutations (H).\nIn the in vitro assays, TOGARAM1 could recruit ARMC9 to MTs, and the colocalization between the two proteins was particularly prominent in particles that could stationarily bind to or diffuse on MTs (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt;). As expected from the co-immunoprecipitation experiments, this colocalization was not perturbed by the 123'4' mutant of TOGARAM1 (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, 5B &lt;&lt;FIG:fig5&gt;&gt;) but was abolished by the 1'2'34 mutant (Fig. 4A &lt;&lt;FIG:fig4&gt;&gt;, 5C &lt;&lt;FIG:fig5&gt;&gt;). The brightness of ARMC9 and TOGARAM1 in MT-bound particles increased over time, indicating binding of additional molecules (Fig. 5A &lt;&lt;FIG:fig5&gt;&gt; and B &lt;&lt;FIG:fig5&gt;&gt;). This binding was likely driven by ARMC9, as it was not observed for TOGARAM1 alone (Fig. 1P &lt;&lt;FIG:fig1&gt;&gt;). Presence of ARMC9 did not drastically alter MT growth rates but did cause a roughly two-fold increase in the rescue activity of TOGARAM1, which was still dependent on TOG3 and TOG4 (Fig. 5A, B &lt;&lt;FIG:fig5&gt;&gt;, and D &lt;&lt;FIG:fig5&gt;&gt;). In contrast to ARMC9, the addition of CCDC66 did not alter the effects of TOGARAM1 on MT dynamics, and no colocalization between CCDC66 and TOGARAM1 was observed (Fig. S4D &lt;&lt;FIG:figs4&gt;&gt;).\nFigure 5.\nARMC9 colocalizes with TOGARAM1 and CSPP1 and enhances their effects on MT dynamics.\n(A-C) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCH-EB3 and indicated concentrations and colors of TOGARAM1 constructs and ARMC9. Blue arrowheads, ARMC9 and TOGARAM1 colocalization.\n(D) Parameters of MT plus-end dynamics in the presence of 20 nM EB3 in combination with indicated concentrations of TOGARAM1 constructs and ARMC9 (from kymographs shown in A-B, 1C, 1P, 4E). For growth rates bars represent pooled data from three independent experiments, total number of growth events: EB3 alone, n=938; EB3 with TOGARAM1, n=861; EB3 with TOGARAM1 and ARMC9, n=342; EB3 with 123'4', n=337; EB3 with 123'4' and ARMC9, n=325. For transition frequencies bars represent averaged means from three independent experiments. Error bars represent s.e.m.. ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(E) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP-EB3 and ciliary tip module proteins indicated. Yellow arrowheads, pauses.\n(F) Parameters of MT plus-end dynamics with 20 nM EB3 in combination with indicated concentrations of ciliary top module proteins (from kymographs shown in E, 1C, 1N). For growth rate and pause duration bars represent pooled data from three independent experiments, total number of growth events, pauses: EB3 alone, n=938, 0; EB3 with CSPP1, n=1715, 273; EB3 with CSPP1 and ARMC9, n=627, 315. For transition frequencies and dynamic state bar represent averaged means from three independent experiments. Error bars represent s.e.m., ****, p&lt;0.0001; n.s., not significant; Kruskal-Wallis test followed by Dunns post-test.\n(G) Schematic model showing TOGARAM1, ARMC9, and CSPP1 interaction domains; MTB; pause domain, PD; alpha-helical domain, H.\nWe observed co-immunoprecipitation of ARMC9 and CSPP1 (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt; and B &lt;&lt;FIG:figs5&gt;&gt;), confirming previous observations (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). The interaction required the C-terminal part of CSPP1 and the linker regions surrounding the central helical domain of ARMC9, though not the helical domain itself (Fig. S5A &lt;&lt;FIG:figs5&gt;&gt; and B &lt;&lt;FIG:figs5&gt;&gt;). In vitro reconstitution assays showed that CSPP1 could trigger ARMC9 accumulation on MTs in the vicinity of the growing MT ends (Fig. 5E &lt;&lt;FIG:fig5&gt;&gt;) and mildly increased the percentage of time CSPP1-induced MT pausing (Fig. 5F &lt;&lt;FIG:fig5&gt;&gt;).\nFigure S5 (Related to Figure 5).\nMapping of ARMC9 and CSPP1 interaction.\n(A) Co-immunoprecipitation experiment of full-length ARMC9 with indicated CSPP1 constructs illustrated in schematic below.\n(B) Co-immunoprecipitation experiment of full-length CSPP1 with indicated ARMC9 constructs illustrated in Figure S4A &lt;&lt;FIG:figs4&gt;&gt;.\n(C) Co-immunoprecipitation experiment of full-length CSPP1 with full-length TOGARAM1 shows no interaction.\n(D) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins.\nWe also tested whether CSPP1 and TOGARAM1 can bind to each other but detected no co-immunoprecipitation (Fig. S5C &lt;&lt;FIG:figs5&gt;&gt;), and CSPP1 displayed no colocalization with either TOGARAM1 or CCDC66 in the in vitro assays (Fig. S5D &lt;&lt;FIG:figs5&gt;&gt;). Altogether, our results indicate that CCDC66 does not colocalize with either TOGARAM1 or CSPP1 in reconstitution assays, whereas ARMC9 can bind to both of these proteins through two distinct domains and serve as a scaffold mildly enhancing their effects on MT dynamics (Fig. 5G &lt;&lt;FIG:fig5&gt;&gt;).\n\nCombined action of ciliary tip proteins drives slow processive MT growth\nTo complete the exploration of all pairwise combinations of the studied ciliary tip proteins, we examined the joint effects of CEP104 and TOGARAM1. The two proteins coprecipitated with each other in a manner dependent on the zinc finger domain of CEP104 and the linker region of TOGARAM1 (Fig. S6A &lt;&lt;FIG:figs6&gt;&gt; and B &lt;&lt;FIG:figs6&gt;&gt;). When they were combined on dynamic MTs in the presence of EB3, there were periods of pausing both at the plus end of the seeds and of the dynamic lattice (Fig 6A-C &lt;&lt;FIG:fig6&gt;&gt;). However, these pauses were transient and followed by growth events (Fig 6D &lt;&lt;FIG:fig6&gt;&gt; and E &lt;&lt;FIG:fig6&gt;&gt;), something we did not see with any other CEP104-ciliary tip protein combination. Furthermore, the two proteins together induced periods of very slow MT polymerization with the rates of 0.12+/-0.01 microm/min, ~20 times slower than 2.27+/-0.02 microm/min observed with TOGARAM1 alone (Fig. 6A, C &lt;&lt;FIG:fig6&gt;&gt; and D &lt;&lt;FIG:fig6&gt;&gt;). CEP104 and TOGARAM1 can thus interact through regions that do not engage their tubulin-binding domains and together impose slow MT polymerization (Fig. 6F &lt;&lt;FIG:fig6&gt;&gt;).\nFigure S6 (Related to Figure 6).\nMapping of CEP104 and TOGARAM1 interaction.\n(A-B) Co-immunoprecipitation of either wildtype CEP104 with indicated TOGARAM1 constructs (A) or wildtype TOGARAM1 with indicated CEP104 constructs (B), the two proteins interact between the linker of TOGARAM1 and the zinc finger of CEP104.\n(C) Kymographs illustrating mobility of DmKHC(1-421) on slow growing MT with all ciliary tip module proteins (top) and DmKHC (bottom) proving that the slow growing end of the MT is the plus end. Scale bars 2 microm and 60 s for both kymographs.\nFigure 6.\nCombined action of ciliary tip proteins drives slow processive MT growth.\n(A) Fields of view (left, scale bar 2 microm) and kymograph (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seed with GFP-EB3, mCherry-TOGARAM1, and GFP-CEP104. Orange arrowheads, blocked plus end; green arrowheads, slow MT growth.\n(B-E) Parameters of MT plus-end dynamics in the presence of EB3 in combination with indicated concentrations of ciliary tip module proteins (from kymographs shown in A, 1C, 1L, 1P). For pause &amp; block duration (B) and growth rates (D) data was pooled from three independent experiments, total number of growth events, pauses: EB3 alone, n=938, 0; EB3 with CEP104, n=213, 101; EB3 with TOGARAM1, n=861, 0; EB3 with CEP104 and TOGARAM1, n=373, 140. Bars represent mean + s.e.m.. For dynamic state (C) and transition frequencies (E) bars represent averaged means from three independent experiments. Error bars represent s.e.m.\n(F) Schematic model showing interactions between CEP104, TOGARAM1 and tubulin. Jelly-roll, JR; zinc finger, ZNF; alpha-helical domain, H.\n(G-H) Weighted growth rates (G) and dynamic state (H) for combinations of ciliary tip module proteins indicated (from kymographs I-N, and 1C). For growth rates (G) data was pooled from three independent experiments, total number of growth events: EB3 alone, n=938; entire ciliary tip module, n=70; no ARMC9, n=81; no CCDC66, n=83; no CEP104, n=41; no CSPP1, n=141; no TOGARAM1, n=0. For dynamic state (H) bars represent or averaged means from two independent experiments, error bars represent s.e.m..\n(I-N) Fields of view (left, scale bar 2 microm) and kymographs (right, scale bars 2 microm and 60 s) illustrating MT dynamics from GMPCPP-stabilized seeds with 20 nM GFP- or mCherry-EB3 and indicated concentrations and colors of ciliary tip module proteins indicated. Orange arrow, blocked plus end; green arrows, slow plus-end growth.\nNext, we combined all five ciliary tip module components in the presence of EB3. Strikingly, in these conditions, all MTs displayed slow and highly processive plus-end polymerization with occasional pausing, with an average rate of 0.19+/-0.04 microm/min, while minus ends could still undergo phases of growth and shrinkage (Fig. 6G-I &lt;&lt;FIG:fig6&gt;&gt; and S6C &lt;&lt;FIG:figs6&gt;&gt;). To test which proteins are essential for this dynamic state, we repeated the assays while systematically leaving out each of the ciliary tip proteins. Omitting TOGARAM1 abolished slow growth completely, as all MT plus ends were paused (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt; and J &lt;&lt;FIG:fig6&gt;&gt;). This was not surprising, because MT behavior in these conditions was dominated by the two pausing factors, CEP104 and CSPP1. This indicates that an important function of TOGARAM1 is to overcome MT growth inhibition imposed by CEP104 or CSPP1. Leaving out CEP104 had no major effect, except for a slight increase in pausing and very occasional periods of fast growth (Fig. 6G, H &lt;&lt;FIG:fig6&gt;&gt; and K &lt;&lt;FIG:fig6&gt;&gt;), whereas fast elongation became much more common in the absence of CSPP1 (Fig. 6G, H &lt;&lt;FIG:fig6&gt;&gt;, and L &lt;&lt;FIG:fig6&gt;&gt;). Omitting either ARMC9 or CCDC66 had no major effect except that no pausing was observed, and occasional episodes of fast growth were detected (Fig. 6G, H, M &lt;&lt;FIG:fig6&gt;&gt; and N &lt;&lt;FIG:fig6&gt;&gt;). No MT catastrophes or shrinkage were observed in any of these conditions (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt;). Altogether, ciliary tip module components stabilize MTs by preventing their depolymerization and promoting robust but very slow tubulin addition.\n\nCiliary tip proteins form cork-like densities at MT plus ends and reduce protofilament flaring\nTo better understand the localization of the ciliary tip module on MT ends, we turned to cryo-electron tomography (cryo-ET). We reconstructed 3D volumes containing MTs grown with either EB3 alone or in the presence of the entire ciliary tip module. To best match our in vitro reconstitution conditions, samples were incubated at 30 degreesC with GMPCPP-stabilized seeds and 15 microM tubulin for 10 minutes before being added to holey carbon-coated copper grids and vitrified. Based on our analysis of MT dynamics we presumed that they were mostly fast growing for EB3 controls and elongating slowly with a few paused ends for ciliary tip module samples (Fig. 6H &lt;&lt;FIG:fig6&gt;&gt;). MTs grown in the presence of the ciliary tip module showed densities at their ends reminiscent of champagne corks, whereas MTs grown in the presence of only EB3 had no clear density at their ends (Fig. 7A-D &lt;&lt;FIG:fig7&gt;&gt;, S7A &lt;&lt;FIG:figs7&gt;&gt; and B &lt;&lt;FIG:figs7&gt;&gt;). Despite the density for the ciliary tip module appearing quite varied in structure, it was always situated in both the lumen and on the tips of the MT protofilaments (Fig. 7B, D &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;). We used 2D rotational averaging to resolve MT polarity based on chirality of the cross-section (&lt;&lt;REF:Bouchet-Marquis-2007-99-45&gt;&gt;), through which we determined that cork-like electron densities were specific for MT plus ends (Fig. 7C, D &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;). 36 out of 40 captured \"corks\" where confidently assigned to plus ends, the polarity of the other 4 MTs remains undetermined due to poor tilt-alignment of the tomograms. We performed semi-automated segmentation of the tomographic volumes, both for visualization and segregation of the MT protofilaments and ciliary tip module (Fig. 7E &lt;&lt;FIG:fig7&gt;&gt; and F &lt;&lt;FIG:fig7&gt;&gt;). It proved difficult to segregate all individual protofilaments from each other for further analysis due to the limited tilt range. However, the majority of protofilaments were individually resolved, and separation between the ciliary tip module and protofilaments was often clear. Extensive contacts on the luminal surface of tubulin and the exposed protofilament ends were observed, while the exterior surface of the MT remained largely unbound.\nFigure S7 (Related to Figure 7).\nCharacterization of MT plus ends by cryo-ET\n(A, B) Representative examples of MT plus ends grown in presence of EB3 alone (A) or ciliary tip module and EB3 (B). Per MT, the following is shown from left to right; a slice of the denoised tomogram containing the MT plus end, an 8 nm thick transverse cross-section accompanied by rotational averaging analysis to determine MT polarity, and the corresponding 3D model of manual protofilament tracing accompanied by its transverse cross-section. Scale bars 25 nm.\nFigure 7.\nCiliary tip proteins form cork-like densities at MT plus ends and reduce protofilament flaring\n(A, B) Slices (4.3 nm thick) of denoised tomograms showing MT plus ends grown in presence of EB3 alone (A) or ciliary tip module and EB3 (B). Scale bars 25 nm.\n(C, D) 8nm thick transverse cross-section of the corresponding MTs in panels A and B. The transverse cross-section (top) is accompanied by rotational average (bottom) to indicate MT polarity. Scale bars 25 nm.\n(E, F) 3D rendered segmentation volume of a free/dynamic MT plus end and ciliary tip module-plugged plus end. The MT is shown in grey and the ciliary tip module density in green.\n(G-J) 3D model of manually traced protofilament shapes (G, H), accompanied by their transverse cross-sections (I, J).\n(K) Distribution of protofilament length measured from the last segment within the MT wall until the tip of the protofilament. Total number of protofilaments analyzed; EB3, n=329; CTM, n=352. Lines represent median (solid) and quartiles (dotted). ****, p&lt;0.0001; Mann-Whitney test.\n(L) Protofilament standard deviation per MT. Total number of MTs analyzed: EB3, n=26; ciliary tip module, n=26. Bars represent mean, with error bars representing s.e.m., ****, p&lt;0.0001 (Mann-Whitney test).\n(M) Schematic representation of parameters that were obtained from manual tracing of MT plus-end protofilaments.\n(N) Average local protofilament curvature per MT. Total number of MTs analyzed: EB3, n=25; ciliary tip module, n=25. Bars represent mean, with error bars representing s.e.m., *, p&lt;0.05; Mann-Whitney test.\n(O) MT raggedness plotted as standard deviation of protofilament axial distance of last segment within the wall per MT. Total number of MTs analyzed: EB3, n=26; ciliary tip module, n=26. Bars represent mean, with error bars representing s.e.m., **, p&lt;0.01; Mann-Whitney test.\n(P) Schematic model of the ciliary tip module at a MT plus end.\nWe turned to manual tracing to analyze how the ciliary tip module may be altering MT tip architecture, and how this architecture deviates from more typical MT architecture associated with fast dynamics (Fig. 7G-J &lt;&lt;FIG:fig7&gt;&gt;, S7A &lt;&lt;FIG:figs7&gt;&gt; and B &lt;&lt;FIG:figs7&gt;&gt;). We first looked at how flared the MT protofilaments were by measuring both protofilament length (length of curved segment) and protofilament standard deviation per MT (standard deviation of the distance between first point that deviated from being completely straight to the end of the protofilament, see materials and methods) (Fig. 7K-M &lt;&lt;FIG:fig7&gt;&gt;). As expected, MT plus ends grown with only EB3 showed primarily flared protofilaments (&lt;&lt;REF:Gudimchuk-2020-11-3765&gt;&gt;; &lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;) (Fig. 7A, K, L &lt;&lt;FIG:fig7&gt;&gt; and S7A &lt;&lt;FIG:figs7&gt;&gt;), whereas plus ends \"corked\" with ciliary tip module proteins had a dramatic decrease in both protofilament length and standard deviation in protofilament length per MT (Fig. 7B, K, L &lt;&lt;FIG:fig7&gt;&gt; and S7B &lt;&lt;FIG:figs7&gt;&gt;), as well as a mild decrease in average local curvature (Fig. 7N &lt;&lt;FIG:fig7&gt;&gt;). Finally, MT raggedness, defined as the standard deviation in the axial distance along the MT of the first point for each protofilament that deviated from being completely straight (Fig. 7M &lt;&lt;FIG:fig7&gt;&gt;), was modestly but significantly decreased in MT plus ends \"corked\" by ciliary tip module proteins (Fig. 7O &lt;&lt;FIG:fig7&gt;&gt;). Together our results indicate a potential mechanism where the ciliary tip module acts from both the lumen and tips of MTs to stabilize slow and processive elongation (Fig 7P &lt;&lt;FIG:fig7&gt;&gt;).\n\nDiscussion\nCiliary tip proteins play important roles in controlling the structure and therefore function of motile and primary cilia. Mutations in these proteins cause various ciliopathies often resulting from shorter cilia causing defects in cilia-dependent signaling pathways (&lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Louka-2018-217-4298&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Perlaza-2022-33-ar12&gt;&gt;). Axonemal MTs are stable and elongate very slowly without undergoing long depolymerization events. Here, we have reconstituted these behaviors in vitro with five components of the ciliary tip module. We found that collectively, these proteins robustly impose very slow and processive MT growth. Furthermore, elongation rates measured in our assays, 0.19+/-0.04 microm/min, fall within the range of those measured for the initial elongation of regenerating flagella in single-celled organisms such as Chlamydomonas reinhardtii (0.08-0.40 microm/min) (&lt;&lt;REF:Marshall-2005-16-270&gt;&gt;; &lt;&lt;REF:Rosenbaum-1967-34-345&gt;&gt;; &lt;&lt;REF:Rosenbaum-1969-41-600&gt;&gt;; &lt;&lt;REF:Witman-1975-253-178&gt;&gt;), though these rates are reduced when flagella reach their normal length. However, it remains to be determined if our measured growth rates are within the range of elongating primary cilia.\nSlow growth is an unusual state for dynamic MTs, because it is incompatible with the formation of a long stabilizing GTP cap, which is needed to avoid MT disassembly (&lt;&lt;REF:Desai-1997-13-83&gt;&gt;) and promote growth (&lt;&lt;REF:Wieczorek-2015-17-907&gt;&gt;). Factors controlling slow MT polymerization must thus inhibit tubulin detachment and promote tubulin addition at MT plus ends that have only a very short or no GTP cap. We found that several ciliary tip proteins can perform these functions. Two of them, TOGARAM1 and CEP104, were predicted to be MT polymerases because their individual TOG domains can bind tubulin and enhance tubulin polymerization in vitro\n(&lt;&lt;REF:Al-Jassar-2017-25-146&gt;&gt;; &lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Rezabkova-2016-291-18496&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). However, full-length TOGARAM1 and CEP104 do not accelerate tubulin polymerization at freely growing MT plus ends. On the contrary, TOGARAM1 somewhat slows down MT growth rate, and CEP104 potently blocks MT elongation. Still, these proteins are MT stabilizers rather than depolymerases as both can inhibit tubulin detachment from GDP-bound MT ends: TOGARAM1 induces rescues, whilst CEP104 dramatically slows down disassembly of MT plus ends. Moreover, TOGARAM1 can promote tubulin addition and thus serve as a polymerase at paused MT ends, and together CEP104 and TOGARAM1 can induce phases of very slow polymerization.\nThe processive slow MT growth reconstituted by either CEP104 and TOGARAM1 together, or the ciliary tip module as a whole, is unique to the plus ends of MTs. This is likely driven by the two TOG domain-containing module members, because specific orientation of the TOG domains can position tubulin dimers favorably for binding to protofilament tips at the plus end (&lt;&lt;REF:Ayaz-2012-337-857&gt;&gt;). Generally, the interaction of TOG domain proteins with MTs depends on positively charged intrinsically disordered regions (&lt;&lt;REF:Aher-2018-46-40&gt;&gt;; &lt;&lt;REF:Byrnes-2017-216-1641&gt;&gt;; &lt;&lt;REF:Widlund-2011-108-2741&gt;&gt;) and additional adaptors, such as EB1 and SLAIN2 (&lt;&lt;REF:Li-2011-193-973&gt;&gt;; &lt;&lt;REF:van der Vaart-2011-193-1083&gt;&gt;). This is also true for the ciliary TOG domain regulators. TOGARAM1, similar to XMAP215 (&lt;&lt;REF:Widlund-2011-108-2741&gt;&gt;), contains a linker region which promotes binding to MTs in vitro, and is further recruited to MTs by ARMC9. CEP104 depends on interactions with several binding partners, EB3, CSPP1 and CCDC66. EB3 is known to bind to the outer MT surface (&lt;&lt;REF:Maurer-2012-149-371&gt;&gt;), whereas CSPP1 is a luminal protein (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The binding mode of CCDC66 has not yet been clarified, but it is likely that some of its parts are present on MT surface as it can bundle MTs in vitro (&lt;&lt;REF:Batman-2022-20-e3001708&gt;&gt;). With binding partners on both sides of the MT wall, CEP104 is in a good position to span the MT tip. TOGARAM1 is likely positioned very close to protofilament ends, since it can catalyze tubulin addition. Therefore, ARMC9 is another good candidate for connecting across MT surfaces as it not only interacts and promotes TOGARAM1 activity but also the pausing behavior of luminal protein CSPP1. Collectively, this fits with the cryo-ET data, which show that the complex of ciliary tip proteins localizes only at the plus ends of MTs and is associated with the luminal side (likely CSPP1) and the tip of MTs (likely CEP104, TOGARAM1 and CCDC66), resembling a cork on a champagne bottle.\nThe interactions between ciliary tip proteins depend at least partially on non-overlapping domains: for example, CEP104 binds to tubulin through its TOG domain, to EB3 through the SxIP motif within its flexible C-terminal region, to TOGARAM1 through the zinc finger domain and to CCDC66 and CSPP1 through the jelly-roll domain (Fig 3Q &lt;&lt;FIG:fig3&gt;&gt;, 6F &lt;&lt;FIG:fig6&gt;&gt;). Similarly, TOGARAM1 interacts with tubulin through TOG3 and TOG4, with ARMC9 through TOG2 and with MTs and CEP104 through its linker region (Fig. 5G &lt;&lt;FIG:fig5&gt;&gt;). Additionally, Joubert syndrome-linked mutations in TOGARAM1 TOG2 and ARMC9 ARM domains disrupt their interaction, highlighting the functional importance for understanding how the ciliary tip module forms. For several reasons, we do not think the five ciliary tip proteins form a stoichiometric complex but rather a flexible interaction network, the function of which is likely to enhance the accumulation of these proteins at the distal ends of cilia. Firstly, a flexible interaction network could explain why we saw variation in size of \"corks\" in our cryo-ET data. Additionally, in vitro, we saw that ciliary tip module members act in a partially redundant fashion and therefore we cannot be sure that all ciliary tip proteins are present in every \"cork\". Leaving individual ciliary tip module proteins out of reconstitutions, whilst making slow growth less robust, had no major effect on MT dynamics. The one exception was TOGARAM1, which was essential for MT elongation; and that aligned well with the observation that overexpression of TOGARAM1 induces longer cilia (&lt;&lt;REF:Latour-2020-130-4423&gt;&gt;). Furthermore, in vertebrate cells, the loss of any of the ciliary tip module proteins results in the generally mild phenotype of shorter cilia, whereas simultaneous loss of several module components has more severe consequences (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). Finally, a loose interaction mode is also in line with the fact that in other species, such as ciliates, these proteins bind to different sites (e.g., A- and B-tubules) and can have opposing functions (&lt;&lt;REF:Louka-2018-217-4298&gt;&gt;).\nIt appears counterintuitive that deletions or mutations in all vertebrate ciliary tip module proteins cause shorter cilia even though CEP104 and CSPP1 inhibit MT elongation (&lt;&lt;REF:Das-2015-26-4248&gt;&gt;; &lt;&lt;REF:Frikstad-2019-28-1907&gt;&gt;; &lt;&lt;REF:Latour-2020-130-4423&gt;&gt;; &lt;&lt;REF:Odabasi-2023-136-jcs260327&gt;&gt;; &lt;&lt;REF:Yamazoe-2020-295-14723&gt;&gt;). This could be explained by the interplay with additional factors not included in our assays. For example, CEP104 and CSPP1 might counteract MT depolymerase factors, such as kinesin-13, kinesin-18 KIF19A or kinesin-4 KIF7, which are known to act at ciliary tips (&lt;&lt;REF:Blaineau-2007-17-778&gt;&gt;; &lt;&lt;REF:He-2014-16-663&gt;&gt;; &lt;&lt;REF:Niwa-2012-23-1167&gt;&gt;). Alternatively, similar to our in vitro assays with TOGARAM1, CEP104 and CSPP1 might also cooperate with MT polymerization-promoting factors, such as XMAP215 or CLASP. This idea is supported by CEP104 promoting centriole elongation in flies (&lt;&lt;REF:Ryniawec-2023-33-4202&gt;&gt;), where it is established that Orbit/CLASP and kinesin-13 KLP10A act as positive and negative regulators of fly centriole length, respectively (&lt;&lt;REF:Shoda-2021-134-jcs251231&gt;&gt;).\nAn important question is the nature of structural changes at MT plus ends associated with slow polymerization. Cryo-ET analysis showed that rapidly growing MTs terminate with strongly flared protofilaments (&lt;&lt;REF:Gudimchuk-2020-11-3765&gt;&gt;; &lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;), whereas cork-like structures formed by ciliary tip proteins reduce protofilament flaring and at least partially cap the plus ends. Still, in these conditions, MT tips retain conformational variability, with some protofilaments more flared and some more blunt. It is possible that at each given time, the protofilaments that are blunt are occluded by ciliary tip proteins and therefore paused. Despite lacking a GTP cap, these protofilaments don't depolymerize because ciliary tip regulators keep them together and stabilize them from the luminal side. In this model, the more curved protofilaments are the ones undergoing tubulin addition. This would be consistent with the idea that TOG domains increase local tubulin concentration and deliver 'curved' tubulins to protofilament tips (&lt;&lt;REF:Ayaz-2014-3-e03069&gt;&gt;). Alternatively, ciliary tip proteins could be responsible for keeping tubulin dimers in flared protofilaments in a bent state, while new dimers are added onto the straighter protofilaments, which slowly forces the corks out of the plus ends as they elongate.\nFormation of MT plus-end-specific \"corks\" that impose blunt protofilament conformation is strikingly similar to the \"plugs\" observed in reconstitution experiments with the centriolar cap protein CP110 (&lt;&lt;REF:Ogunmolu-2021-?-2021.2012.2029.474442&gt;&gt;). MT stabilization from the luminal side combined with the reduction of protofilament curvature and occlusion of the longitudinal interface of beta-tubulin might thus be a common mechanism for generation of very stable and slowly growing MTs. The relevance of such structures is supported by the observations of plugs at the tips of ciliary MTs in cells (&lt;&lt;REF:Legal-2023-222-e202301129&gt;&gt;; &lt;&lt;REF:Leung-2021-40-e107410&gt;&gt;). A mechanism involving the luminal MT surface and occlusion of the longitudinal interfaces of beta-tubulin is different from the regulation of much more dynamic cytoplasmic MT ends, which involves factors located on the outer MT surface (&lt;&lt;REF:Akhmanova-2015-16-711&gt;&gt;). Strikingly, the fundamental principles are the same: in both cases, plus-end regulation depends on the balance of growth-promoting and inhibiting activities that are brought together by a multivalent network of interacting proteins.\n\nMaterials and Methods\n\nDNA constructs, cell lines and cell culture\nAll proteins were cloned into modified pEGFP-C1 or pmCherry-C1 vectors with a StrepII tag and expressed in HEK293T cells (ATCC). For all module members except TOGARAM1 the human sequence was used, due to technical difficulties the mouse sequence of TOGARAM1 was used, it is 84% identical to human TOGARAM1. HEK293T cells were cultured in DMEM:F10 (1:1) for co-immunoprecipitation assays, or DMEM (Lonza) for protein purification, both were supplemented with 10% fetal calf serum (GE Healthcare Life Sciences) and 1% (v/v) penicillin/streptomycin. All cells were routinely checked for mycoplasma contamination using the MycoAlertTM Mycoplasma Detection Kit (Lonza).\n\nCo-immunoprecipitation assays\nHEK293T cells were transiently transfected with a mix consisting of polyethyleneimine (Polysciences) and constructs for the ciliary tip module proteins. Co-IP was performed with ChromoTek GFP-Trap(R) Magnetic particles M-270 (Proteintech) following manufacturer's instructions with company advised buffers. 24 hours post-transfection, cells were harvested in 1x PBS and lysed for 30 minutes on ice in 200 muL lysis buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.5% IGEPAL CA-630) supplemented with EDTA free protease inhibitor cocktail (Roche) and PhosSTOP(TM) phosphatase inhibitor cocktail (Roche). Lysate was cleared by 20 min centrifugation at 4 degreesC at 17,000 x g and diluted with 300 muL dilution buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA) supplemented with EDTA free protease inhibitor cocktail(Roche) and PhosSTOP(TM) phosphatase inhibitor cocktail (Roche) per sample. Diluted lysate was incubated rotating end-over-end for 45 minutes with wash buffer equilibrated beads. Beads were washed four times with 500 muL wash buffer (10 mM Tris HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.05% IGEPAL CA-630) and protein was eluted in 80 muL 2x Laemmli sample buffer (0.125M tris HCl, 20% glycerol, 10% 2-mercaptoethanol, 0.02% Bromophenol blue, 0.2M DTT) and heated at 95 degreesC for 5 minutes. All steps prior to elution were performed at 4 degreesC and all reagents and samples were kept on ice.\nPrepared samples were run on 8% SDS-page gel and blotted onto Amersham Protran Premium 0.45 mum NC nitrocellulose membrane (Cytiva) by wet transfer at 37 constant volts overnight at 4 degreesC in transfer buffer (0.2M tris-HCl, 2M glycine, 10% MeOH and 0.01% SDS), or by semi-dry transfer for 1 hour at 0.15 constant Ampere in transfer buffer (0.2M tris-HCl, 2M glycine, 10% MeOH). Membranes were blocked in blocking buffer (2% BSA diluted in 1x PBS 0.05% Tween-20) for 1 hour and sequentially incubated with primary antibodies and secondary antibodies diluted in blocking buffer for 1 hour rotating at room temperature (primary and secondary antibodies) or overnight at 4 degreesC (primary antibodies only). Blots were thoroughly washed in between and after incubation with antibodies with wash buffer (1x PBS 0.05% Tween-20) and imaged with Odyssey(R) CLx Infrared Imaging System (LI-COR biosciences) at various exposure times. The following antibodies were used for western blotting: rabbit anti-RFP (1:2000, Rockland immunochemicals), mouse anti-GFP (1:2000, Sigma-Aldrich), donkey anti-mouse IgG (H + L) IRDye(R) 800CW (1:10000, LI-COR bioscience) and donkey anti-rabbit IgG (H + L) IRDye(R) 680CW (1:10000, LI-COR bioscience).\n\nProtein purification from HEK293T cells for in vitro reconstitution assays\nHEK293T cells were transiently transfected with a mix consisting of polyethyleneimine (Polysciences) and constructs for one of the ciliary tip module proteins. Protein purification was performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The cells were harvested 28 hours after transfection in lysis buffer (50 mM HEPES, 300 mM NaCl, 1 mM MgCl2, 1 mM DTT, 0.5% Triton X-100, pH 7.4) supplemented with protease inhibitors (Roche) and kept on ice for 15 minutes. After clearance of the debris by centrifugation, the supernatant was incubated with 20 microl StrepTactin beads (GE Healthcare) for 45 min. After several washing steps, five times with a 300 mM salt wash buffer (50 mM HEPES, 300 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4) and three times with a 150 mM salt wash buffer (50 mM HEPES, 150 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4), the protein was eluted in elution buffer (50 mM HEPES, 150 mM NaCl, 1 mM MgCl2, 1 mM EGTA, 1 mM DTT, 0.05% Triton X-100, pH 7.4, 2.5 mM d-Desthiobiotin (Sigma-Aldrich)). Purified proteins were snap-frozen and stored at -80degreesC.\n\nMass spectrometry\nTo confirm each protein was purified, and the eluted protein did not contain any contaminants or known interactors that could affect MT dynamics, we performed mass spectrometry (MS) analysis. MS measurements were performed as described previously (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). The purified protein sample was digested using S-TRAP microfilters (ProtiFi) according to the manufacturer's protocol. Digested peptides were eluted and dried in a vacuum centrifuge before liquid chromatography-mass spectrometry (LC-MS) analysis. The samples were analyzed by reversed-phase nLC-MS/MS using an Ultimate 3000 UHPLC coupled to an Orbitrap Exploris 480 mass spectrometer (Thermo Scientific). Digested peptides were separated using a 50 cm reversed-phase column packed in-house (Agilent Poroshell EC-C18, 2.7 microm, 50cm x 75 microm) and eluted from the column at a flow rate of 300 nL/min. MS data were acquired using a data-dependent acquisition (DDA) method with a MS1 resolution of 60,000 and mass range of 375-1600m/z. Fragmentation spectra were collected at a resolution of 15,000 using an HCD of 28, isolation window of 1.4m/z, and a fixed first mass of 120m/z. MS/MS fragment spectra were searched using Sequest HT in Proteome Discoverer (Thermo Scientific) against a human database (UniProt, year 2020) that was modified to contain the tagged protein sequence from each ciliary tip module protein and a common contaminants database. Tryptic miss cleavage tolerance was set to 2, fixed modifications were set to cysteine carbamidomethylation, and variable modifications included oxidized methionine and protein N-terminal acetylation. Peptides that matched the common contaminate database were filtered out from the results table.\n\nIn vitro reconstitution assays\n\nMT seed preparation\nDouble-cycled GMPCPP-stabilized MT seeds were used as templates for MT nucleation in vitro assays. GMPCPP-stabilized MT seeds were prepared as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). A tubulin mix consisting of 70% unlabeled porcine brain tubulin, 18% biotin-labeled porcine tubulin and 12% HiLyte-488-, rhodamine-, or HiLyte647-labeled porcine tubulin (all from Cytoskeleton) was incubated with 1 mM GMPCPP (Jena Biosciences) at 37degreesC for 30 minutes. Polymerized MTs were then pelleted using an Airfuge for 5 min at 199,000 x g and subsequently depolymerized on ice for 20 min. Next, 1 mM GMPCPP was added and MTs were let to polymerize again at 37degreesC for 30 minutes. Polymerized MTs were again pelleted (as above) and diluted tenfold in MRB80 buffer containing 10% glycerol prior to snap-freezing to store them at -80degreesC.\n\nIn vitro reconstitution assays\nIn vitro assays with dynamic MTs were performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Microscopic slides were prepared by adding two strips of double-sided tape to mount plasma-cleaned glass coverslips. The coverslips were functionalized by sequential incubation with 0.2 mg/mL PLL-PEG-biotin (Susos AG, Switzerland) and 1 mg/mL neutravidin (Invitrogen) in MRB80 buffer (80 mM piperazine-N, N[prime]-bis (2-ethane sulfonic acid), pH 6.8, supplemented with 4 mM MgCl2, and 1 mM EGTA). Then, GMPCPP-stabilized MT seeds were attached to the coverslips through biotin-neutravidin interactions. The coverslip was blocked with 1 mg/mL kappa-casein before the reaction mix was flushed in. The reaction mix consisted of different concentrations and combinations of fluorescently labeled purified proteins in MRB80 buffer supplemented with 15 microM porcine brain tubulin (100% dark porcine brain tubulin when 20 nM GFP-EB3 or mCherry-EB3 was added, or 97% dark porcine brain tubulin with 3% rhodamine- or HiLyte-488-labeled porcine tubulin), 0.1% methylcellulose, 1 mM GTP, 50 mM KCl, 0.2 mg/mL kappa-casein, and oxygen scavenger mix [50 mM glucose, 400 microg/mL glucose oxidase, 200 microg/mL catalase and 4 mM DTT]. This mix was spun down in an Airfuge for 5 min at 119,000 x g before addition to the flow chamber and the flow chamber was sealed with vacuum grease. Microtubules were imaged immediately at 30degreesC using a total internal reflection fluorescence (TIRF) microscope.\n\nIn vitro assays for cryo-ET\nSamples were prepared as above for reconstitution assays with slight modifications. Instead of flow chambers, all steps occur in a tube. Reaction mixes consisted of either just 20nM EB3 or all CTM components (20nM TOGARAM1, 20nM CEP104, 20nM CSPP1, 20nM CCDC66, 200nM ARMC9) with 20nM EB3 in MRB80 buffer supplemented with 15 microM porcine brain tubulin, 1 mM GTP, 0.2 mg/mL kappa-casein, and 15 microM DTT. After centrifugation of the reaction mix for dynamic MTs, GMPCPP-stabilized seeds and ProtA-Au5 fiducials (1:20) were added, and MTs were left to polymerize for 10-20 min at 30degreesC in an Eppendorf tube. Subsequently, the sample was gently resuspended and 4 microl was transferred to a glow-discharged holey carbon R2/1 copper grid (Quantifoil Micro Tools) suspended in the chamber of a Vitrobot (Thermo Fisher Scientific). The sample was incubated for 2 min inside the Vitrobot chamber, equilibrated at 30 degreesC and 95% relative humidity, to allow for potential MT repolymerization after sample transfer to the grid. Subsequently, grids were manually back-blotted for 3 s and plunge-frozen in liquid ethane. Vitrified grids were clipped into auto-grid cartridges and stored in liquid nitrogen until future use.\n\nSingle-molecule fluorescence intensity analysis\nProtein samples of GFP, GFP-EB3 and either GFP-CEP104, GFP-TOGARAM1 or GFP-ARMC9 were diluted in PBS and immobilized in adjacent flow chambers of the sample plasma cleaned glass coverslip as described in (&lt;&lt;REF:Sharma-2016-37-362&gt;&gt;). Flow chambers were then sealed with vacuum grease and immediately imaged using TIRF microscopy. 10-20 images (per condition) of previously unexposed coverslip areas were acquired with 200 ms exposure time.\n\nCEP104 molecule counting at blocked MT plus ends\nTo determine the number of CEP104 molecules at blocked MT plus ends we immobilized GFP in one chamber of the same plasma cleaned glass coverslip and in vitro reconstitution assay in the adjacent chamber (as described above). Chambers were sealed and immediately imaged using TIRF microscope. First images on unbleached GFP single molecules were acquired, then using the same illumination conditions, images of unexposed GFP-CEP104 bound MTs were acquired.\n\nFluorescence Recovery After Photobleaching\nFor FRAP experiments, in vitro reconstitutions were prepared as described above and imaging was conducted using TIRF microscopy. Either focused 488-laser (for GFP-EB3) or 561-laser (for mCherry-CEP104) was used to bleach specific regions of the MT lattice.\n\nMicroscopy\n\nTIRF microscopy\nIn vitro reconstitution assays imaged on a previously described (iLas2) TIRF microscope setup (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). We used the iLas3 system (Gataca Systems (France)) which is a dual laser illuminator for azimuthal spinning TIRF (or Hilo) illumination. This system was installed on Nikon Ti microscope (with the perfect focus system, Nikon), equipped with 489 nm 150 mW Vortran Stradus 488 laser (Vortran) and 100 mW 561 nm OBIS laser (Coherent), 49002 and 49008 Chroma filter sets. Additionally, a CCD camera CoolSNAP MYO (Teledyne Photometrics) was installed and the set up was controlled with MetaMorph 7.10.2.240 software (Molecular Device). To keep the in vitro samples at 30 degreesC, a stage top incubator model INUBG2E-ZILCS (Tokai Hit) was used. The final resolution using CCD camera was 0.045 mum/pixel. For all assays MT dynamics was measured at 200 ms exposure and 3 second intervals for 10 minutes. For EB3 FRAP experiments continuous imaging was used with 100 ms exposure. For kinesin experiments first a 10 minute movie was acquired at 200 ms exposure and 3 second intervals to measure MT dynamics, subsequently Dm-KHC(1-421)-SNAP-6xHis (gift from Kapitein lab, Addgene plasmid #196976) labelled with Alexa647-SNAP dye (NEB) was continuous imaged using 100 ms exposure.\n\nCryo-ET\nVitrified in vitro reconstituted MTs were imaged on a Talos Arctica transmission electron microscope (200 keV) (Thermo Fisher Scientific), equipped with a K2 summit direct electron detector (Gatan) and post-column energy filter aligned to the zero-loss peak with 20 keV slit width. Image acquisition was controlled by Serial-EM (&lt;&lt;REF:Mastronarde-2005-152-36&gt;&gt;). MT ends were located on TEM overview images, acquired at 4,100x magnification (33.3 A/pixel). Tilt series were collected at 63,000x magnification (2.17 A/pixel) with a dose rate of ~5 e-/pixel/s and a total dosage of 100 e-/A2. Tilt images were recorded using a dose symmetric scheme (&lt;&lt;REF:Hagen-2017-197-191&gt;&gt;) over a tilt range of 54degrees to -54degrees or 60degrees to -60degrees with a tilt increment of 2degrees, at a defocus target of -3 mum.\n\nImage analysis\n\nAnalysis of MT plus-end dynamics in vitro\nAnalysis of MT plus-end dynamics was performed as described before (&lt;&lt;REF:van den Berg-2023-222-e202208062&gt;&gt;). Movies of dynamic MTs were corrected for drift, and kymographs were generated using the plugin KymoResliceWide v.0.4 in ImageJ (https://github.com/ekatrukha/KymoResliceWide). MT tips were traced with lines, and measured lengths and angles were used to calculate the MT dynamics parameters such as growth rate and transition events. All events with growth rates faster than 0.02 microm/min and slower than 0.5 microm/min were categorized as slow growing, events faster than 0.5 microm/min were categorized as fast growing, events with shrinkage rates faster than 0.02 microm/min were categorized as shrinking. The events with slower growth rates or faster shrinkage rates than the before mentioned rates were categorized as pause events. Transition frequencies were calculated by dividing the sum of the transition events per experiment by the total time this event could have occurred. Weighted growth rate histograms were calculated by taking individual growth durations and dividing by total growth time.\n\nAnalysis of FRAP experiments\nData were normalized to 1 for image acquired immediately before the FRAP event and 0 for the image acquired immediately after the FRAP event using the following equation; Normalized intensity=(intensity t=x - intensity t=1)(intensity t=0 - intensity t=1). For EB3, non-linear fit curves were fitted to the post-bleach intensities and half-life calculated using GraphPad Prism.\n\nAnalysis of single molecule fluorescence intensities\nSingle molecule fluorescence spots of proteins immobilized directly on the cover glass were detected and fitted with 2D Gaussain function using custom written ImageJ plugin DoM-Utrect (https://github.com/ekatrukha/DoM_Utrecht). Fitted peak intensities were then used to create intensity histograms. CEP104-blocked plus ends were then manually annotated and fitted with 2D Gaussian function using the same plugin.\n\nCryo-ET 3D volume reconstruction and analysis\nTomogram reconstruction and denoising was performed using an in house Python script, as described previously (&lt;&lt;REF:Chaillet-2023-24-13375&gt;&gt;). The tilt series' frames were initially aligned and dose-weighted using MotionCor2 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;). Subsequent tilt series alignment and tomographic reconstruction for denoising, visualization and analysis were done through AreTomo 1.3.3. Final reconstructed tomographic volumes were created through weighted back-projection and binned with a factor of two. Cryo-CARE 0.2.2 was used for tomogram denoising (&lt;&lt;REF:Buchholz-2019-152-277&gt;&gt;). For this, even and odd tomographic reconstructions were generated through splitting of the movie frames for each individual tilt into even and odd summed frames with MotionCor2 (&lt;&lt;REF:Zheng-2017-14-331&gt;&gt;). Alignment parameters were first calculated on the full tilt series and subsequently applied to the even and odd stacks. The cryo-CARE network was trained on 5 tomograms per dataset, and then applied to the entire set.\nFor visualization purposes, semi-automated segmentation was performed on denoised tomograms using the EMAN2 2.91 tomoseg module (&lt;&lt;REF:Chen-2017-14-983&gt;&gt;). For this, separate neural networks were trained for the two features of interest: \"MT\" and \"ciliary tip module\". Each feature was included as 'negative' training reference, together with background noise and ice contamination, for the neural network training of the other feature. Final visualization and 3D rendering was performed in ChimeraX.\nMT ends, both free and CTM-plugged, were only analyzed for protofilament shape in case they were plus ends. MT polarity was determined through 2D rotational averaging with applied symmetry, using the EMAN2 program e2proc2d. Manual 3D tracing of protofilament shapes at MT plus ends was performed, as previously described (&lt;&lt;REF:McIntosh-2018-217-2691&gt;&gt;; &lt;&lt;REF:McIntosh-2020-2101-191&gt;&gt;; &lt;&lt;REF:Ogunmolu-2021-?-2021.2012.2029.474442&gt;&gt;), in the IMOD 4.9.6 program 3dmod (&lt;&lt;REF:Kremer-1996-116-71&gt;&gt;). Manual tracing was performed on tomographic subvolumes, generated with the mtrotlong program of IMOD (4.11.20). Traces of individual protofilaments were saved as separate contours within one object per MT. To satisfy the three-point requirement of howflared, for blunt end protofilaments a third point was added one pixel upstream of the second point that indicated the last protofilament segment in the MT wall. Subsequently, protofilament coordinates were extracted using the howflared program of IMOD (4.11.20). MT raggedness and protofilament length parameters, as described in the results section, were calculated by howflared. Curvature analysis was performed on the howflared output with Matlab scripts available at https://github.com/ngudimchuk/Process-PFs.\n\nStatistical analysis\nStatistical analysis was performed using GraphPad Prism 9. Figure legends contain statistical details of each experiment, including the statistical tests used, the number of measurements and the number of experiments.\n\nCode availability\nThe python script for automated tomogram reconstruction and cryo-CARE denoising is available online at https://github.com/SBC-Utrecht/cryocare-from-movies. MATLAB scripts for analysis of protofilament tracings in IMOD are available online at https://github.com/ngudimchuk/Process-PFs.\n\nAuthor Contributions\nH.A.J.S. and C. M. vdB. performed experiments, analyzed data and wrote the paper, R.A.H. performed and analyzed cryo-electron tomography experiments, and wrote the paper, D.S. performed immunoprecipitation experiments, K.E.S. performed and analyzed mass spectrometry experiments, S.H. supervised and analyzed cryo-electron tomography experiments, R.R has generated essential reagents and acquired funding, A.A. coordinated the project, acquired funding and wrote the paper. All authors reviewed and edited the paper.\n\nCompeting financial interests\nThe authors declare no competing financial interests.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state..\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [14.66s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"sample_type\": \"micro-organism\",\n  \"sample_preparation_type\": \"tomography\",\n  \"sample_preparation_buffer_ph\": \"6.8\",\n  \"grid_vitrification_cryogen\": [\"ETHANE\"],\n  \"sample_preparation_cryo_protectant\": \"None\",\n  \"grid_model\": \"Quantifoil R2/1\",\n  \"grid_material\": \"COPPER\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"HOLEY\",\n  \"grid_vit_ctemp\": \"30 degreesC\",\n  \"grid_vit_chumid\": \"95%\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"HEK293T\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"sample_type\": \"micro-organism\",\n  \"sample_preparation_type\": \"tomography\",\n  \"sample_preparation_buffer_ph\": \"6.8\",\n  \"grid_vitrification_cryogen\": [\"ETHANE\"],\n  \"sample_preparation_cryo_protectant\": \"None\",\n  \"grid_model\": \"Quantifoil R2/1\",\n  \"grid_material\": \"COPPER\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"HOLEY\",\n  \"grid_vit_ctemp\": \"30 degreesC\",\n  \"grid_vit_chumid\": \"95%\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"HEK293T\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 182,
      "prompt_tokens": 23037,
      "total_tokens": 23219
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_94f711dcf6"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [0ms] Exiting Parser run with output:
{
  "sample_type": "micro-organism",
  "sample_preparation_type": "tomography",
  "sample_preparation_buffer_ph": "6.8",
  "grid_vitrification_cryogen": [
    "ETHANE"
  ],
  "sample_preparation_cryo_protectant": "None",
  "grid_model": "Quantifoil R2/1",
  "grid_material": "COPPER",
  "grid_mesh": "Unknown",
  "grid_support_topology": "HOLEY",
  "grid_vit_ctemp": "30 degreesC",
  "grid_vit_chumid": "95%",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "HEK293T",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/end] [1:chain:RunnableSequence] [14.66s] Exiting Chain run with output:
{
  "sample_type": "micro-organism",
  "sample_preparation_type": "tomography",
  "sample_preparation_buffer_ph": "6.8",
  "grid_vitrification_cryogen": [
    "ETHANE"
  ],
  "sample_preparation_cryo_protectant": "None",
  "grid_model": "Quantifoil R2/1",
  "grid_material": "COPPER",
  "grid_mesh": "Unknown",
  "grid_support_topology": "HOLEY",
  "grid_vit_ctemp": "30 degreesC",
  "grid_vit_chumid": "95%",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "HEK293T",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nA central role of the endosomal-lysosomal network is to determine the sorting of integral proteins and their associated proteins and lipids, collectively termed \"cargos,\" between 2 fates, either degradation within lysosomes or retrieval from this fate for the promotion of recycling to a variety of organelles that include the cell surface [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;]. Efficient and selective endosomal-lysosomal cargo sorting is essential for the function of all eukaryotic cells with increasing evidence pointing to defects in endosomal retrieval and recycling being at the heart of a wide array of human diseases, most notably those associated with neurodegeneration [&lt;&lt;REF:Small-2020-12-?&gt;&gt;]. Establishing the molecular mechanisms and functional significance of endosomal cargo retrieval and recycling is therefore fundamental to understand eukaryotic cell biology.\nTwo general concepts are considered to account for the mechanism of endosomal cargo retrieval and recycling. Sequence-independent sorting posits that sorting can proceed without the need for any sequence information within the intracellular cytoplasmic domains of integral membrane cargos [&lt;&lt;REF:Maxfield-2004-5-121&gt;&gt;]. On the other hand, sequence-dependent sorting is controlled by specific signals in the cytoplasmic domain of cargos. These signals are typically short linear peptide motifs that direct the sorting process through coupling to ancient and evolutionary conserved coat complexes [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;,&lt;&lt;REF:McNally-2018-28-807&gt;&gt;,&lt;&lt;REF:Chen-2019-20-465&gt;&gt;]. In coupling sorting motif recognition with membrane remodeling, these coats coordinate the process of cargo retrieval with the formation of cargo-enriched transport carriers for onward recycling. In mammals, these coats include sorting nexin-3 (SNX3)-Retromer [&lt;&lt;REF:Strochlic-2007-177-115&gt;&gt;-&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;], SNX27-Retromer [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], SNX17-Retriever [&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;], the CCC (CCDC22/CCDC93/COMMD) [&lt;&lt;REF:Phillips-Krawczak-2015-26-91&gt;&gt;,&lt;&lt;REF:Bartuzi-2016-7-10961&gt;&gt;], and endosomal SNX-BAR sorting complex for promoting exit 1 (ESCPE-1) complexes [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;] and various accessory proteins including the actin polymerizing Wiskott-Aldrich syndrome protein and SCAR homologue (WASH) complex [&lt;&lt;REF:Gomez-2009-17-699&gt;&gt;,&lt;&lt;REF:Derivery-2009-17-712&gt;&gt;].\nSorting nexin-27 (SNX27) is an endosome-associated cargo adaptor composed of a central PX (phox homology) domain flanked by an amino-terminal PDZ (PSD95, disks large, and zona occludens) domain and a carboxyl-terminal FERM (4.1/ezrin/radixin/moesin) domain [&lt;&lt;REF:Kaiji-2003-8-434&gt;&gt;,&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;]. The PDZ domain recognizes a specific sorting motif, termed the PDZ-binding motif (PDZbm), present at the very carboxyl termini of over 400 cargo proteins including receptors, transporters, channels, enzymes, and adhesion molecules [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;-&lt;&lt;REF:McMillan-2021-10-e59432&gt;&gt;]. In recognizing these internalized cargos, SNX27 regulates their retrieval from lysosomal degradation and promotes their recycling back to the plasma membrane [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;].\nTo facilitate the retrieval process, the SNX27 PDZ domain associates with Retromer, a stable heterotrimer of VPS26 (A and B isoforms expressed in humans [&lt;&lt;REF:Kerr-2005-6-991&gt;&gt;]), VPS35 and VPS29, to form the SNX27-Retromer [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. Retromer itself assembles to form dimeric arches that may cluster SNX27 captured cargos [&lt;&lt;REF:Kendall-2020-28-393&gt;&gt;,&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;,&lt;&lt;REF:Kovtun-2018-561-561&gt;&gt;] and associates with the WASH complex to allow the localized Arp2/3-dependent polymerization of branched filamentous actin [&lt;&lt;REF:Harbour-2012-442-209&gt;&gt;,&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;]. In a poorly understood process, these assemblies and actin dynamics are proposed to generate a retrieval subdomain on the endosomal membrane that is enriched in cargo destined for recycling [&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;-&lt;&lt;REF:Lee-2016-7-10939&gt;&gt;]. It is from these subdomains that further membrane remodeling ensues to drive the biogenesis of cargo-enriched tubular profiles and tubulovesicular carriers that physically transport cargo back to the cell surface. Importantly, defects in SNX27-Retromer function are associated with the pathoetiology of human disease most notable neurological disorders and neurodegenerative disease [&lt;&lt;REF:Chandra-2021-9-642378&gt;&gt;].\nA membrane tubulating Bin/Amphiphysin/Rvs (BAR) domain-containing protein associated with SNX27-Retromer tubular profiles is sorting nexin-1 (SNX1) [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;]. SNX1, and its functional ortholog SNX2, is part of the ESCPE-1 complex [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;,&lt;&lt;REF:Carlton-2004-14-1791&gt;&gt;], a membrane tubulating complex that in vitro can reconstitute the formation of membrane tubules with diameters similar to those observed in cellulo [&lt;&lt;REF:van Weering-2012-31-4466&gt;&gt;]. To be functional, ESCPE-1 forms a stable heterodimer of either SNX1 or SNX2 associated with either of SNX5 or SNX6, 2 additional BAR domain-containing orthologs [&lt;&lt;REF:Wassmer-2007-120-45&gt;&gt;]. Importantly, the mechanistic basis of how SNX27-Retromer delivers cargo into forming ESCPE-1 tubular profiles and tubulovesicular carriers remains a major unanswered question.\nHere, we establish that a conserved surface in the FERM domain of SNX27 binds directly to acidic-Asp-Leu-Phe (aDLF) sequence motifs present in the disordered amino termini of the SNX1 and SNX2 subunits of ESCPE-1. We show that this interaction is required to handover PDZbm containing cargo proteins retrieved from lysosomal degradation by SNX27-Retromer into ESCPE-1 tubular profiles that promote cargo recycling for subsequent repopulation at the cell surface. Through evolutionary analysis of those protein-protein interactions that assemble the SNX27:Retromer:ESCPE-1 coat complex, we reveal how this sorting axis has evolved to accommodate the increasing complexities of endosomal sorting observed in animals.\n\nResults\n\nThe FERM domain of SNX27 promotes cargo recycling following SNX27-Retromer-mediated retrieval from lysosomal degradation\nPreviously, we have established that the interaction of Retromer with the PDZ domain of SNX27 is a crucial step in preventing the routing of internalized cargo into the lysosomal degradative pathway [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. To identify the region(s) of SNX27 required for promoting subsequent cargo recycling to the plasma membrane, we used CRISPR gene editing to generate SNX27 knockout HeLa cells (Fig 1A &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Compared to control cells, where endogenous GLUT1 was enriched at the cell surface, in the SNX27 knockout cells, GLUT1 displayed a steady-state enrichment with LAMP1-positive lysosomes, a phenotype that was also observed in a VPS35 knockout HeLa cell line (Fig 1B &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Importantly, reexpression of GFP-SNX27 in the SNX27 knockout cells rescued the retrieval of GLUT1 from the lysosomal distribution and promoted its repopulation at the cell surface (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). These data mirror those previously published using RNA interference (RNAi)-mediated SNX27 suppression [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;] and confirm the SNX27 knockout HeLa cell line as a suitable screening platform through which to dissect the features of SNX27 required for promoting GLUT1 endosome-to-plasma membrane recycling.\nFig 1\nThe FERM domain of SNX27 promotes cargo recycling following SNX27-Retromer-mediated lysosomal retrieval.\n(A) Western blot confirming the absence of the proteins of interest in VPS35 KO and SNX27 KO cells. (B) Representative images of parental Hela cells, VPS35 KO HeLa clonal line and SNX27 KO HeLa cells. The steady-state distribution of GLUT1 was analyzed by immunofluorescence staining of fixed cells with GLUT1 and the endosomal and lysosomal markers VPS35 and LAMP1. A total of 70 cells were analyzed in each condition for colocalization between GLUT1 and LAMP1 across n  =  4 independent experiments. The Pearson coefficient values of LAMP1 colocalization were compared to the values of parental HeLa using 1-way ANOVA and Dunnett test: P &lt; 0.0001 (HeLa vs VPS35 KO), &lt; 0.0001 (HeLa versus SNX27 KO). Pearson coefficient values of VPS35 colocalization were compared using unpaired 2-tailed t test: P &lt; 0.0001 (HeLa versus SNX27 KO). (C) Representative images of SNX27 KO HeLa cells transiently transfected with GFP-SNX27 WT, GFP-SNX27 Delta67-77 and GFP-SNX27 DeltaFERM. Moreover, 48 hours after transfection, cells were fixed and immunostained. The steady-state distribution of GLUT1 was analyzed by immunofluorescence staining of fixed cells with GLUT1 and the endosomal and lysosomal markers VPS35 and LAMP1. A total of 70 cells were analyzed in each condition for colocalization between GLUT1 and LAMP1, and GLUT1 and VPS35 across n  =  4 independent experiments. The Pearson coefficient values were compared to the values of SNX27 KO cells rescued with GFP-SNX27 WT using 1-way ANOVA and Dunnett test. LAMP1 colocalization values were P &lt; 0.0001 (+WT versus +Delta67-77), 0.3559 (+WT versus +DeltaFERM). VPS35 colocalization values were P &lt; 0.0001 (+WT versus Delta67-77), 0.001 (+WT versus +DeltaFERM). Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. Scale bars, 25 mum (micrographs) and 5 mum (magnified images). The data underlying the graphs shown in the figure can be found in S1 Data. KO, knockout; SNX27, sorting nexin-27; WT, wild-type.\nIn contrast to wild-type SNX27, expression of a SNX27 mutant lacking the loop region essential for Retromer association (GFP-SNX27Delta67-77) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;] failed to rescue retrieval from lysosomal colocalization and failed to promote GLUT1 cell surface recycling (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). These data further confirm that the association of SNX27 with Retromer is a crucial step in coupling sequence-dependent cargo recognition with cargo retrieval from the lysosomal degradative fate [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. Consistent with this, the expression of a SNX27 deletion mutant lacking the FERM domain (GFP-SNX27DeltaFERM), a mutant that retained the association of SNX27 with Retromer, retained the ability to retrieve GLUT1 from lysosomes, as defined by a decreased colocalization between GLUT1 and LAMP1 (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). However, the GFP-SNX27DeltaFERM mutant failed to promote GLUT1 recycling to the cell surface, rather GLUT1 was enriched in a Retromer-positive endosomal compartment (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). The overall levels of GLUT1 were not decreased in the KO and rescue conditions compared to parental cells (S1 Fig). Together, these data are consistent with a model in which the coupling of SNX27 to Retromer is necessary for retrieval from the lysosomal degradative fate, while the SNX27 FERM domain is required to promote the subsequent endosome-to-cell surface recycling of GLUT1.\n\nSNX1 and SNX2 couple ESCPE-1 to the FERM domain of SNX27\nESCPE-1 regulates tubular-based endosomal recycling of several cargos, many of which are also dependent on SNX27-Retromer [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;]. ESCPE-1 is a heterodimeric assembly of the functionally redundant SNX1 or SNX2 associated with the functionally redundant SNX5 or SNX6 (Fig 2A &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;) [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;], while SNX27-Retromer is an assembly of SNX27 with the VPS26A/B:VPS35:VPS29 heterotrimeric Retromer complex (Fig 2B &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. To define a possible connection between SNX27 and ESCPE-1, we first performed a series of GFP-nanotrap immunoisolation of SNX27 and various SNX27 mutants expressed in HEK-293T cells and probed for the presence of associated endogenous ESCPE-1. Robust association was observed between full-length GFP-SNX27 and all components of ESCPE-1 (Fig 2C-2E &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). This association was independent of the SNX27 PDZ domain as binding was retained in the PDZ deletion mutant GFP-SNX27DeltaPDZ and was not observed when just the PDZ domain was expressed in HEK-293T cells (Fig 2C and 2D &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Retromer association with SNX27 was not required for ESCPE-1 binding as GFP-SNX27Delta67-77 and GFP-SNX27(L67A/L77A), 2 mutants that fail to bind to Retromer [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], each retained association with ESCPE-1 (Fig 2E &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). However, ESCPE-1 association was lost upon deletion of the SNX27 FERM domain (SNX27DeltaFERM) or upon switching the FERM domain of SNX27 for the corresponding FERM domain from SNX17 (Fig 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Moreover, just the isolated FERM domain of SNX27, when expressed in HEK-293T cells, was able to associate with ESCPE-1 (Fig 2D &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Finally, in the reverse immunoisolations, GFP-tagged SNX1, SNX2, SNX5, or SNX6 revealed that only SNX1 and SNX2 displayed a pronounced association with endogenous SNX27 when expressed in HEK-293T cells (Fig 2F &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Moreover, this association did not require the presence of the SNX1 or SNX2 BAR domains, since deletion of these domains failed to affect binding to SNX27 (Fig 2G &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Together, these data are consistent with the FERM domain of SNX27 associating with the SNX1 and SNX2 components of ESCPE-1.\nFig 2\nThe FERM domain of SNX27 interacts with SNX1 and SNX2.\n(A) Schematics of ESCPE-1 assembly consisting of heterodimers of the SNX-BAR proteins SNX1 or SXN2 with SNX5 or SNX6. These SNXs have BAR and PX domains responsible of dimer formation and membrane targeting, respectively. SNX1 and SNX2 also have an extended amino-terminal unstructured extension (NT). (B) Schematics of the SNX27-Retromer assembly. Retromer is a stable heterotrimer of VPS35, VPS26, and VPS29. Retromer directly interacts with SNX27 through a beta-hairpin insertion within the PDZ domain (residues 67 to 79) that engages a cleft of the VPS26 protein between the amino-terminal and carboxyl-terminal subdomains. (C) Co-immunoprecipitation of GFP-tagged SNX27 FL, SNX27 DeltaPDZ, SNX27 DeltaFERM and a chimera consisting of the SNX27 gene where its FERM domain was swapped for that of SNX17. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensities of VPS35, VPS26A, VPS29, SNX1, and GFP were measured from n =  3 independent experiments using Odyssey software. The band intensities, normalized to GFP expression, are presented as the average fraction of the FL. (D) Co-immunoprecipitation of GFP-tagged SNX27 FL, isolated SNX27 PDZ, PX, and FERM domains expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (E) Co-immunoprecipitation of GFP-tagged SNX27 FL and the SNX27 mutants Delta67-77 and L67A-L74A expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (F) Co-immunoprecipitation of GFP-tagged SNX1, SNX2, SNX5, and SNX6 expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. (G) Co-immunoprecipitation of GFP-tagged SNX1, SNX2, and SNX5 lacking the BAR domain (DeltaBAR) expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; FL, full-length; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27.\n\nThe amino termini of SNX1 and SNX2 present tandem acidic-Asp-Leu-Phe motifs that associate with the SNX27 FERM domain\nCompared to SNX5 and SNX6, human SNX1 and SNX2 each contain amino-terminal extensions of 142 and 139 amino acids, respectively, which are predicted to be unstructured. Expression in HEK-293T cells of GFP-SNX1(1-279), a construct encoding for the amino-terminal extension plus the SNX1 PX domain, and GFP-SNX1(1-138), a construct encoding for just the amino-terminal extension of SNX1, established that the amino-terminal extension of SNX1 is sufficient for association with endogenous SNX27 (Fig 3A &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Likewise, the amino terminus of SNX2 interacted with SNX27, while those of SNX5 and SNX6 failed to do so (S2A Fig). To establish the direct nature of the association, we recombinantly expressed and purified full-length mouse SNX1 and the isolated FERM domain of human SNX27; mouse and human SNX1 are 94.2% identical. Isothermal titration calorimetry (ITC) established that SNX1 bound directly to the FERM domain of SNX27 with an affinity (Kd) of 41.0 muM (Fig 3B &lt;&lt;FIG:pbio.3001601.g003&gt;&gt; and S1 Table). A similar binding affinity (39.1 muM) was observed with a recombinant protein corresponding to the amino-terminal extension of mouse SNX1 (85.9% sequence identity with human SNX1) (Fig 3B &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). To map down the features of SNX1 and SNX2 required for SNX27 binding, we performed a deletion mutagenesis screen of the amino-terminal extension of human SNX1 (Fig 3C &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Walking upstream from a minimal binding region encoded by residues 1 to 138 established that 2 regions were required for maximal SNX27 binding. Deletion of residues 79 to 100 led to a pronounced reduction in binding and subsequent deletion of residues 41 to 54 abolished the interaction. Alignment of these regions of SNX1 revealed 2 similar motifs with DIF or DLF sequences flanked by upstream acidic residues. Comparison with the amino terminus of SNX2, which also binds SNX27 (S2A Fig), showed that it also has 2 similar sequences in its disordered amino-terminal domains, and, altogether, these define a conserved sequence D/E-x-x-D-I/L-F, here termed the aDLF motif (Fig 3D and 3E &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;).\nFig 3\nThe amino termini of SNX1 and SNX2 directly interact with SNX27 FERM domain through tandem aDLF motifs.\n(A) Co-immunoprecipitation of GFP-tagged SNX1 FL or truncation mutants corresponding to residues 1 to 279 and 1 to 138 expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The band intensity of SNX27 was measured from n =  3 independent experiments using Odyssey software and was normalized to GFP expression. The relative binding of SNX1 constructs was compared with the SNX1 FL using a 1-way ANOVA and Dunnett test. (B) Binding of mSNX1FL (orange) and mSNX11-139 (blue) with hSNX27FERM by ITC. (C) Co-immunoprecipitation of GFP-tagged SNX1 amino terminus (1 to 138) and a panel of constructs corresponding to serial truncations of SNX1 amino terminus. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensity of SNX27 and GFP was measured from n = 4 independent experiments using Odyssey software and was normalized to GFP expression. The relative binding of SNX1 truncation mutants was compared with the SNX1 amino terminus (1-138) using a 1-way ANOVA and Dunnett test. (D) Alignment of the SNX1 regions that are involved in SNX27 binding with the corresponding sequences in the SNX2 amino terminus. The conserved residues between SNX1 and SNX2 account for a consensus motif of acidic residues followed by DI/LF (aDLF motif). (E) Alphafold structural prediction for FL SNX1 [&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;]. The amino terminus of SNX1 is shown in its full extension and the residues involved in SNX27 binding are colored in red and green. (F) Co-immunoprecipitation of GFP-tagged SNX1 WT and a panel of amino acid swap mutants across the aDLF motifs of SNX1. The constructs were transiently transfected in HEK293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensities of SNX27, SNX6, and GFP were measured from n = 4 independent experiments using Odyssey software. The band intensity, normalized to GFP expression, is presented as the average fraction of SNX1 WT. The binding of SNX1 point mutants to SNX27 and SNX6 were compared with the SNX1 WT using a 1-way ANOVA and Dunnett test. (G) Binding of mSNX27FL and hSNX1 peptides by ITC. SNX135-51(D45K): green; SNX135-51: orange; SNX175-92: lilac; ND*: no binding detected. (H) Binding of mSNX27FL and hSNX2 peptides by ITC. SNX216-33(DLF/SSS): blue; SNX216-33: green; SNX262-82: lilac; ND*: no binding detected. The ITC graphs represent the integrated and normalized data fit with 1:1 ratio binding. The binding affinity (Kd) is given as mean of at least 2 independent experiments. Thermodynamic parameters for the binding are provided in S1 Table. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001. The data underlying the graphs shown in the figure can be found in S1 Data. aDLF, acidic-Asp-Leu-Phe; FL, full-length; ITC, isothermal titration calorimetry; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27; WT, wild-type.\nSite-directed mutagenesis targeting the first aDLF motif of SNX1 established that full-length GFP-SNX1(E42K), GFP-SNX1(E44K), GFP-SNX1(D45K), GFP-SNX1(I46K), and GFP-SNX1(F47K) all displayed reduced binding to SNX27 of around 50% (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). A similar reduction in binding was also observed when targeting the second aDLF motif of SNX1 as observed with the GFP-SNX1(E84K), GFP-SNX1(D87K), and GFP-SNX1(F89D) mutants (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). In a double GFP-SNX1(D45K,D87K) mutant that simultaneously targeted key residues in both aDLF motifs SNX27, binding was reduced to near undetectable levels (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Importantly, all mutants retained their ability to associate with SNX6, establishing that ESCPE-1 was still able to assemble. In addition, the mutant targeting both aDLF motifs in SNX2, GFP-SNX2(D26K,D72K), also reduced SNX27 binding to near undetectable levels (S2B Fig). Next, we used ITC to quantify the binding of aDLF peptides from SNX1 and SNX2 to full-length mouse SNX27. Peptides corresponding to residues 35 to 51 and 75 to 92 encompassing the first and second aDLF motifs of human SNX1, respectively, bound to SNX27 with respective affinities of 33.4 muM and 13.3 muM (Fig 3G &lt;&lt;FIG:pbio.3001601.g003&gt;&gt; and S1 Table). Establishing the redundant nature of the interaction with SNX27, we designed peptides corresponding to the first and second aDLF motifs of SNX2, namely residues 16 to 33 and 62 to 82. These also bound to full-length mouse SNX27 with affinities of 37.8 muM and 22.6 muM, respectively (Fig 3H &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Further defining the redundancy between SNX1 and SNX2, introduction of a charge swap mutation into the SNX1 35-51 peptide, SNX1(D45K), and a triple mutation in the SNX2 16-33 peptide, SNX2(DLF-SSS), resulted in mutant peptides that showed no detectable binding to SNX27 (Fig 3G and 3H &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Finally, a competitive ITC assay using full-length SNX27 preincubated with excess SNX1 75-92 peptide established that it competed for binding of SNX2 16-33 peptide, consistent with both sequences associating with the same surface within the SNX27 FERM domain (S2C Fig). Overall, these data establish that through tandem aDLF motifs present within their unstructured amino terminal extensions SNX1 and SNX2 directly associate with the FERM domain of SNX27.\n\nA basic surface in the SNX27 FERM domain is required for SNX1/SNX2 binding\nSNX27 and SNX17 are related endosome associated cargo adaptors [&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;,&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;]. While they display entirely distinct functional roles in sequence-dependent endosomal cargo sorting [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;], they share structurally similar PX and FERM domains [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;,&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;] (Fig 4A &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). A comparative immunoisolation of GFP-SNX17 and GFP-SNX27 from HEK-293T cells established that SNX17 does not bind to ESCPE-1 (Fig 4B &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;), consistent with the loss of ESCPE-1 binding to the SNX27 chimera containing the SNX17 FERM domain (Fig 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). To identify potential regions for aDLF motif binding to the SNX27 FERM domain, we initially performed a comparative structural analysis between the X-ray structure of the SNX17 FERM domain [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;] and a homology model of the SNX27 FERM domain [&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;]. In particular, we focused on the distribution and clustering of complementary basic amino acids. This revealed that the FERM domain of SNX27 possesses a basic surface defined in part by R437, R438, K495, K496, R498, and K501, residues that in SNX17 often corresponded to uncharged residues or negatively charged amino acids (Fig 4A and 4C &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Indeed, charge swap site-directed mutagenesis of these residues generated full-length GFP-SNX27 mutants that when expressed and immunoisolated from HEK-293T cells lacked the ability to associate with ESCPE-1; all mutants, however, retained the ability to associate with Retromer (Fig 4D &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). In direct ITC-quantified binding assays human SNX27(K495D), SNX27(R498D), and SNX27(K501D) all displayed no detectable binding to the human SNX1 75-92 peptide, while the SNX27(R496D) mutant displayed a reduce binding affinity of 56.4 muM when compared with the 12.7 muM observed for wild-type SNX27 (Fig 4E &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;).\nFig 4\nSNX1 and SNX2 amino termini bind a unique basic surface of the SNX27-FERM domain.\n(A) Sequence alignment of SNX27 FERM domain with the SNX17 FERM domain. The positively charged residues that constitute a basic surface unique to SNX27 are boxed and labeled in blue. (B) Co-immunoprecipitation of GFP-tagged SNX27 and SNX17. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (C) Homology model of the SNX27 FERM domain [&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;]. The residues that are unique to SNX27 FERM domain and those that may account for the interaction with ESCPE-1 are labeled. (D) Co-immunoprecipitation of GFP-tagged SNX27 WT and a panel of amino acid swap mutants across the FERM domain. The constructs were transiently transfected in HEK293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, and SNX6. The band intensities were measured from n = 3 independent experiments using Odyssey software. The band intensity, normalized to GFP expression, is presented as the average fraction of the WT. The binding of SNX27 point mutants to VPS35, SNX1, SNX2, SNX5, and SNX6 was compared with the SNX27 WT using a 2-way ANOVA and Dunnett test. (E) Binding of SNX175-92 peptide to hSNX27FL proteins by ITC (hSNX27 R498D: dark-blue; hSNX27 K501D: orange; hSNX27 K495D: lilac; hSNX27 K496D: sky-blue; hSNX27 WT: red-orange; ND*: no binding detected). The ITC graphs represent the integrated and normalized data fit with 1:1 ratio binding. The binding affinity (Kd) is given as mean of at least 2 independent experiments. Thermodynamic parameters for the binding are provided in S1 Table. (F) The left panel shows the AlphaFold2 generated model of the SNX27 FERM domain (pink ribbons) bound to the core aDLF sequences (yellow sticks) of SNX1 and SNX2 with identical binding orientations. For clarity, only one SNX27 model is shown with the 8 aDLF peptides overlaid (2 peptide models each of the 4 sequences from SNX1 and SNX2). The 3 subdomains of the SNX27 FERM domain F1, F2, and F3 are indicated. The middle panel shows a close-up view of the modeled SNX1 aDLF core sequence 44EDIFTGA50. The strictly conserved Phe side chain of the aDLF sequences is numbered F0 for reference, and other residues are numbered with respect to this. (G) Co-immunoprecipitation of GFP-tagged SNX27 and a panel of hydrophobic amino acid mutations predicted to mediate the SNX27 FERM-ESCPE1 binding. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps with the normalized enrichment of SNX27 association relative to expression of GFP-tagged protein quantified in the bar chart. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001. The data underlying the graphs shown in the figure can be found in S1 Data. aDLF, acidic-Asp-Leu-Phe; ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; ITC, isothermal titration calorimetry; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\nFinally, we modeled the binding of the aDLF sequences from SNX1 and SNX2 with SNX27 using the AlphaFold2 machine learning algorithm [&lt;&lt;REF:Jumper-2021-15-1&gt;&gt;,&lt;&lt;REF:Mirdita-2021-?-?&gt;&gt;]. In total, we generated 12 models, with 3 models each of SNX27 in the presence of the 4 separate aDLF peptides (Figs 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt; and S3A-S3C). Without any prior inputs apart from the SNX27 and peptide sequences, AlphaFold2 produced very high confidence models of the SNX27 FERM domain. Remarkably, in all 12 models, the aDLF peptides associated specifically with the SNX27 F3 subdomain, near the basic surface identified by mutagenesis (Fig 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Closer inspection showed that in 8 of the models the core D[L/I]F residues adopted essentially identical bound structures, with the hydrophobic [L/I]F side chains inserting into a hydrophobic pocket and interacting with SNX27 I467, F469, Y490, and V500 and the acidic Asp side chain surrounded by the complementary basic SNX27 residues R437, R438, K439, and R498 (Fig 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). To further validate the model, we therefore mutated I467, Y490, and V500 to Lys, all of which disrupted the interaction of SNX27 with ESCPE-1 (Fig 4G &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Together, these models are entirely consistent and strongly support the mutagenesis data for the SNX27-peptide interactions.\n\nSNX27 association with ESCPE-1 couples cargo retrieval with promotion of cell surface recycling\nTo define the functional significance of the association of SNX27 to ESCPE-1, we first transfected our SNX27 knockout HeLa cells line with constructs encoding for either GFP-tagged wild-type SNX27 or the ESCPE-1 binding defective mutant GFP-SNX27(R498D) (Fig 5A &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Analysis of their relative subcellular distributions established that they each associated with EEA1 and SNX6-positive endosomes (Fig 5B &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Previously, we established that by binding to the PDZbm present within the cytoplasmic facing tail of GLUT1, the SNX27-Retromer complex serves to retrieve internalized GLUT1 away from lysosomal mediated degradation ([&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;], and see Fig 1A &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). We therefore performed a series of rescue experiments to quantify the SNX27-dependent retrieval of internalized GLUT1 from lysosomal mediated degradation, using the SNX17-Retriever-mediated retrieval of alpha-5 integrin as a negative control (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;) [&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;]. In the SNX27 knockout HeLa cells, GLUT1 displayed an enhanced rate of degradation following 8 hours of treatment with cycloheximide, data that are consistent with the loss of SNX27 expression leading to a reduced ability for transporter retrieval from the lysosomal degradative fate (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). This phenotype was rescued by reexpression of wild-type GFP-tagged SNX27 (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Interestingly, expression of GFP-SNX27(R498D), a mutant that fails to associate with ESCPE-1 but retains binding to Retromer (Fig 4D and 4E &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;), also fully rescued the retrieval of GLUT1 from lysosomal degradation (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). The coupling of SNX27-Retromer to the ESCPE-1 complex is therefore not required for the retrieval of GLUT1 from lysosomal degradation.\nFig 5\nSNX27:ESCPE-1 interaction is required for endosome-to-plasma membrane recycling of SNX27 cargos.\n(A) Representative western blot showing similar expression levels between transiently expressed GFP-SNX27 and GFP-SNX27(R498D). (B) Representative images of SNX27 KO HeLa cells transiently transfected with GFP-SNX27 or GFP-SNX27(R498D). Moreover, 48 hours after transfection, cells were fixed and immunostained for the endosomal markers EEA1 and SNX6. Magnified views of the white boxes are shown boxed. Cell numbers analyzed for colocalization were 60 GFP-SNX27 and 60 GFP-SNX27 R498D cells across n  =  3 independent experiments. The Pearson coefficient values were compared using a 2-tailed unpaired t test; for EEA1 P = 0.2971, SNX6 P = 0.0506. (C) Degradation assay in parental HeLa cells or SNX27 KO HeLa cells that were transiently transfected with GFP-SNX27 or GFP-SNX27(R498D) mutant. Moreover, 48 hours after transfection, cells were incubated with 10 mug/ml cycloheximide and lysed at different time points as indicated. The band intensities of endogenous GLUT1 and ITGA5 were measured from n = 3 independent experiments using Odyssey software. The levels of GLUT1 and ITGA5 at 8 hours were compared with the corresponding level at 0-hour time point. Analysis was done using a 2-way ANOVA and Sidak test. GLUT1: P = 0.0735 (HeLa), 0.0068 (KO), 0.9997 (+GFP-SNX27), 0.9975 (+GFP-R498D). ITGA5: P = 0.1151 (HeLa), 0.9820 (KO), 0.9999 (+GFP-SNX27), and 0.6899 (+GFP-R498D). (D) Analysis of surface levels of GLUT1 and ITGA5 in parental HeLa cells or SNX27 KO HeLa cells that were transiently transfected with GFP-SNX27 or GFP-SNX27(R498D) mutant. Moreover, 48 hours after transfection, cells were subjected to surface biotinylation followed by streptavidin-based immunoisolation. The immunoisolates were blotted for GLUT1, and the band intensities were measured from n =  4 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. GLUT1: P = 0.0134 (KO versus HeLa), 0.9587 (+GFP-SNX27 versus HeLa), and 0.0163 (+GFP-SNX27 R498D versus HeLa). The immunoisolates were blotted for ITGA5, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. ITGA5: P = 0.5616 (KO versus HeLa), 0.0733 (+GFP-SNX27 versus HeLa), and 0.2354 (+GFP-SNX27 R498D versus HeLa). (E) Representative images of parental Hela cells, SNX27 KO HeLa cells transiently transfected with GFP-SNX27 or GFP-SNX27(R498D). Moreover, 48 hours after transfection, cells were fixed and immunostained for the cargo GLUT1, the endosomal marker VPS35, and the late endosome/lysosome marker LAMP1. A total of 60 cells were quantified per condition in n = 4. The Pearson coefficient values were compared to the values of Parental HeLa using 1-way ANOVA and Dunnett test. The values for LAMP1 colocalization were P &lt; 0.0001 (Parental versus KO), 0.0001 (Parental versus +WT), 0.1084 (Parental versus +R498D), &lt;0.0001 (+WT versus KO), and 0.0692 (+WT versus +R498D). The values for VPS35 colocalization were P &lt; 0.0001 (Parental versus KO), 0.0029 (Parental versus +WT), 0.0001 (Parental versus +R498D), &lt;0.0001 (+WT versus KO), and 0.6346 (+WT versus +R498D). Scale bars, 25 mum (micrographs) and 5 mum (magnified images). Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; KO, knockout; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\nNext, we turned to examining the promotion of GLUT1 recycling back to the cell surface. Here, we utilized restricted cell surface biotinylation to quantify the amount of GLUT1 at the cell surface [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. Compared with control HeLa cells, SNX27 knockout HeLa cells displayed a reduced cell surface abundance of GLUT1, a result of missorting of internalized GLUT1 for lysosomal degradation (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Again, this phenotype was rescued by expression of GFP-SNX27 (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. In contrast, expression of GFP-SNX27(R498D) failed to rescue the cell surface level of GLUT1 (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), consistent with the association of SNX27 with ESCPE-1 being required to promote the cell surface recycling of internalized GLUT1. Importantly, GFP-SNX27(R498D) exhibited an endosomal localization indistinguishable from wild-type SNX27 (Fig 5B &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), consistent with the inability to promote endosome-to-plasma membrane recycling of GLUT1 not arising from a loss of endosomal localization. Furthermore, confocal analysis of the steady-state distribution of GLUT1 in GFP-SNX27(R498D) rescue cells confirmed that GLUT1 was retrieved from the lysosomal degradative fate, as shown by reduced co-localization with the LAMP1-labeled lysosome (Fig 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). However, unlike the rescue with wild-type GFP-SNX27, where GLUT1 was enriched at the cell surface, the GLUT1 in the GFP-SNX27(R498D) rescue displayed an increased level of intracellular puncta and failed to repopulate the cell surface suggestive of the transporter being trapped in an endosomal compartment (Fig 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), the precise nature of which will require further study. Together, these data are consistent with SNX27 orchestrating cargo retrieval from the degradative fate through its PDZ domain-mediated association with cargo and Retromer, while the SNX27 FERM domain mediates association with ESCPE-1 to promote tubular-based plasma membrane recycling of the retrieved cargo.\n\nESCPE-1 association with SNX27 couples cargo retrieval with promotion of cell surface recycling\nTo complete the mechanistic dissection of SNX27 coupling to ESCPE-1, we used a previously characterized double SNX1 and SNX2 knockout HeLa cell line [&lt;&lt;REF:Simonetti-2017-216-3695&gt;&gt;] to generate lentiviral transduced stable lines expressing GFP-SNX1 or GFP-SNX1(D45K,D87K) (Fig 6A &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). When expressed at near to endogenous levels each chimera displayed an indistinguishable association with EEA1 and SNX6-labeled endosomes (Fig 6B &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Consistent with ESCPE-1 not being required for GLUT1 retrieval from lysosomal degradation, the rate of internalized GLUT1 degradation following 8 hours of treatment with cycloheximide was equivalent between parental HeLa cells, the SNX1/SNX2 knockout cells, and the GFP-SNX1 and GFP-SNX1(D45K,D87K) rescue lines (Fig 6C &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). In contrast, the quantification of restricted cell surface biotinylation established that compared to the parental HeLa cells, the SNX1/SNX2 double knockout HeLa cells displayed a reduced cell surface abundance of GLUT1 consistent with a defect in recycling of internalized GLUT1 (Fig 6D &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). While reexpression of GFP-SNX1 rescued the cell surface level of GLUT1, expression of the GFP-SNX1(D45K,D87K) mutant, defective in SNX27 association (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;), failed to rescue the cell surface recycling of GLUT1 (Fig 6D &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Finally, confocal analysis of the steady-state distribution of GLUT1 revealed that when compared to parental cells, the glucose transporter was enriched in Retromer decorated endosomes in the SNX1/SNX2 double knockout HeLa cells (Fig 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;), in agreement with GFP-SNX27DeltaFERM rescue (see Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Reexpression of GFP-SNX1 partially rescued the cell surface delivery of GLUT1 from this compartment, while expression of GFP-SNX1(D45K,D87K) showed an increase in colocalization of GLUT1 with Retromer decorated endosomes (Fig 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Taken together, these data provide an independent validation that the association of ESCPE-1 to the FERM domain of SNX27 is required to promote tubular-based endosomal exit and cell surface recycling of cargo retrieved from lysosomal degradation.\nFig 6\nESCPE-1 interaction with SNX27 is required for endosome-to-plasma membrane recycling of GLUT1.\n(A) Representative western blot showing similar expression levels between stably transfected GFP-SNX1 and GFP-SNX1(KK) in SNX1/2KO cells. SNX1 band intensities were measured from n = 3 and normalized to actin. Analysis was done using a 2-tailed unpaired t test. (B) Representative images of SNX1/2 KO HeLa cell line stably transfected with GFP-SNX1 or GFP-SNX1(KK). Cells were fixed and immunostained for the endosomal markers EEA1 and SNX6. Magnified views of the white boxes are shown boxed. Cell numbers analyzed for colocalization were 50 GFP-SNX1 and 50 GFP-SNX1(KK) cells across n  =  3 independent experiments. The Pearson coefficient values were compared using a 2-tailed unpaired t test; for EEA1 P  = 0.5971, SNX6 P = 0.6459. (C) Degradation assay in parental HeLa cells or SNX1/2 KO HeLa cells that were stably transfected with GFP-SNX1 or GFP-SNX1(KK) mutant. Cells were incubated with 10 mug/ml cycloheximide and lysed at different time points as indicated. The band intensities of endogenous GLUT1 and ITGA5 were measured from n = 3 independent experiments using Odyssey software. The levels of GLUT1 and ITGA5 at 8 hours were compared with the corresponding level at 0-hour time point. Analysis was done using a 2-way ANOVA and Sidak test. GLUT1: P = 0.6796 (HeLa), 0.9768 (KO), 0.8430 (+GFP-SNX1), and 0.2665 (+GFP-KK). ITGA5: P = 0.2247 (HeLa), 0.7892 (KO), 0.9196 (+GFP-SNX1), and 0.9930 (+GFP-KK). (D) Analysis of surface levels of GLUT1 and ITGA5 in parental HeLa cells or SNX1/2 KO HeLa cells that were stably transfected with GFP-SNX1 or GFP-SNX1(KK) mutant. Cells were subjected to surface biotinylation followed by streptavidin-based immunoisolation. The immunoisolates were blotted for GLUT1, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. GLUT1: P = &lt;0.0001 (KO versus HeLa), 0.0878 (+GFP-SNX1 versus HeLa), and &lt;0.0001 (+GFP-KK versus HeLa). The immunoisolates were blotted for ITGA5, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. ITGA5: P = 0.2592 (KO versus HeLa), 0.7185 (+GFP-SNX1 versus HeLa), and 0.0653 (+GFP-KK versus HeLa). (E) Representative images of parental Hela cells, SNX1/2 KO HeLa cells stably transfected with GFP-SNX1 or GFP-SNX1(KK). Cells were fixed and immunostained for the cargo GLUT1, the endosomal marker VPS35, and the late endosome/lysosome marker LAMP1. A total of 50 cells were quantified per condition in n = 3. The Pearson coefficient values were compared to the values of Parental HeLa using 1-way ANOVA and Dunnett test. The values for LAMP1 colocalization were P = 0.0002 (Parental versus KO), 0.7919 (Parental versus +WT), 0.2225 (Parental versus +KK), 0.0028 (+WT versus KO), and 0.6677 (+WT versus +KK). The values for VPS35 colocalization were &lt;0.0001 (Parental versus KO), 0.0020 (Parental versus WT), &lt;0.0001 (Parental versus +KK), &lt;0.0001 (+WT versus KO), and 0.2073 (+WT versus +KK). Scale bars, 25 mum (micrographs) and 5 mum (magnified images). Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; KO, knockout; SNX1, sorting nexin-1; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\n\nEvolutionary assembly of the SNX27:Retromer:ESCPE-1 sorting axis\nWe used phylogenetic analyses to investigate the evolution of the association of SNX27 with Retromer and with the ESCPE-1 complex. The inferred phylogeny of SNX27 suggests the gene was ancestrally present in filozoans (i.e., the group comprising filastereans, choanoflagellates, and metazoans) (S4A Fig). In SNX27, an exposed beta-hairpin in the SNX27 PDZ domain, encoded by residues 67 to 79 in human SNX27, engages a groove in the VPS26A (and VPS26B) subunit of Retromer [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;] (Fig 7A &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;). An equivalent sequence, including key residues Leu-67 and Leu-74 of the beta-hairpin, is present in SNX27 across all choanozoans (this is, the group comprising metazoans and choanoflagellates; Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4A). Key residues in VPS26A required for SNX27 binding include D44 and L154, equivalent residues in VPS26B being D42 and L152 [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. The inferred phylogeny of VPS26A and VPS26B revealed that VPS26B is ancestrally present across eukaryotes, with a duplication of VPS26B in vertebrates leading to the emergence of VPS26A (Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4B). Interestingly, within VPS26B, the D44 residue is present ancestrally in filozoans, while the L154 residue evolved in choanozoans (Fig 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;). Together, these analyses suggest that the SNX27-Retromer association first appeared in the last common ancestor of choanoflagellates and metazoans.\nFig 7\nMolecular evolution of the SNX27-Retromer and SNX27-ESCPE-1 interactions.\n(A) Alphafold2 structural prediction for FL human SNX27 [&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;]. The residues of the exposed beta-hairpin loop in the SNX27 PDZ domain (67 to 79) that account for the interaction with Retromer are highlighted in cyan. Those residues of the FERM domain that account for the interaction with ESCPE-1 (positively charged surface) are highlighted in pink. Our data support a model where the interaction between SNX27 and Retromer is required for the retrieval of PDZ cargos away from the degradative fate. Subsequently, the interaction between SNX27 and ESCPE-1 is required for the export and endosome-to-plasma membrane recycling of PDZ cargos. (B) A cartoon representation incorporating the results of multiple phylogenetic analyses, including VPS5/SNX1/SNX2, SNX27, and VPS26A/VPS26B (see S4A-S4C Fig for detailed phylogenies). The gene families that contain VPS5/SNX1/SNX2 and VPS26A/VPS26B are present across eukaryotes: SNX1 and SNX2 and VPS26A and VPS26B, arose by gene duplication in the common ancestor of vertebrates. SNX27 is found in metazoans, choanoflagellates and the filasterean Capsaspora, but not in more distantly related opisthokonts such as Ichthyosporea or fungi; it is therefore likely arose in the filazoan common ancestor. The residue D44 (VPS26B, Homo sapiens) in VPS26 is found across Filazoa, laying the foundations for the binding interaction between VPS26 and SNX27. L154 (VPS26B, H. sapiens) likely arose in the ancestral choanozoan along with the PDZ loop and key interacting residues (67 to 97 L67, L74, SNX27, H. sapiens), suggesting that the SNX27-Retromer interaction was already present in the common ancestor of metazoans and choanoflagellates. Key residues of the FERM domain: R437, R498, and K501 (SNX27, H. sapiens) are recognizable in the ancestral metazoan. The first aDLF motif (SNX1) was acquired in the common ancestor of cnidarians and bilaterians, along with the residues K495 and K496, which make up part of the FERM domain in SNX27, potentially suggesting that SNX27:ESCPE-1 binding has evolved by this time, if not earlier during metazoan evolution. Interestingly, the first aDLF motif does not appear to be present in the placazoan or ctenophoran sequences, (although the R437, R498, and K501 residues are present), which might reflect secondary loss of the motif in these groups. The second aDLF motif is only present in vertebrates.\nThe association between SNX27 and ESCPE-1 depends on the presence of binding motifs in SNX1 and SNX2 and the FERM domain of SNX27. SNX1 is present across eukaryotes; a gene duplication giving rise to SNX1 and SNX2 from a single ancestral protein occurred in the common ancestor of vertebrates (Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4C). Several motifs required in the FERM domain of H. sapiens SNX27 for binding to ESCPE-1 appear to have evolved in the common ancestor of metazoans or early during their radiation (Fig 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;); many of the key residues (R437, R498, and K501) were in place in the last metazoan common ancestor or prior to the divergence of cnidarians and bilaterians (K495 and K496). For SNX1, the first aDLF motif was likely already present in the last metazoan common ancestor. It is present in sequences from bilaterians and Porifera (sponges) but is absent in cnidarians, placazoans, and ctenophorans, which might represent secondary losses of the motif. As for the second aDLF motif, this is present in all vertebrates. This analysis therefore suggests that the SNX27:ESCPE-1 interaction evolved after the SNX27-Retromer link during early metazoan evolution.\n\nDiscussion\nHere, in combining biochemical, structural, and cellular analyses, we have established that aDLF motifs in the redundant SNX1 and SNX2 subunits of ESCPE-1 directly associate with the FERM domain of SNX27 within the SNX27-Retromer assembly. Studies of endosomal cargo sorting show that this association is necessary to promote the ESCPE-1-dependent tubulovesicular cell surface recycling of internalized transmembrane proteins. Our data suggest that ESCPE-1 mediated recycling occurs subsequently or in conjunction with SNX27-Retromer-dependent retrieval from lysosomal degradation. Building on previous studies [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], these data collectively identify the SNX27:Retromer:ESCPE-1 assembly as a dynamic endosomal coat complex that couples sequence-dependent cargo recognition with membrane remodeling to facilitate the formation of cargo-enriched transport carriers for cargo recycling. In addition, we have begun to describe the functional diversification of Retromer as it has evolved from its tight association with the SNX-BAR homologues Vps5 and Vps17, as originally identified in yeast [&lt;&lt;REF:Seaman-1998-142-665&gt;&gt;], to a more transient engagement of ESCPE-1 in metazoans and their relatives mediated at least in part through the SNX27 cargo adaptor.\nAt present, we propose the following model to describe the functional connectivity of the SNX27:Retromer:ESCPE-1 coat complex within the context of cargo retrieval and recycling. Newly endocytosed cargo containing a functional SNX27 PDZbm are recognized by SNX27, associated with the PI(3)P-enriched early endosome, and these interactions together can stabilize the endosomal residency of SNX27 [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;]. As the endosome matures, SNX27 binds to Retromer through direct association of VPS26A (or VPS26B) to the SNX27 PDZ domain. These interactions are mutually stabilizing, thereby locking down cargo association [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;,&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;]: This assembly is essential for cargo retrieval from the lysosomal degradative fate [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. Retromer dimerization and the formation of Retromer arches [&lt;&lt;REF:Kendall-2020-28-393&gt;&gt;,&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;,&lt;&lt;REF:Kovtun-2018-561-561&gt;&gt;] may aid SNX27 and cargo clustering, thereby forming the retrieval subdomain and initiating weak membrane curvature [&lt;&lt;REF:Simunovic-2015-25-780&gt;&gt;]. These events are supported through the Retromer-mediated recruitment of the WASH complex and the localized production of branched filamentous actin [&lt;&lt;REF:Harbour-2012-442-209&gt;&gt;,&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;]. An additional accessory protein, ANKRD50, may serve to stabilize the formation of the SNX27:Retromer:WASH assembly [&lt;&lt;REF:Kvainickas-2017-130-382&gt;&gt;]. To this emerging \"bud,\" ESCPE-1 is recruited through recognition of 3-phosphoinositides and membrane curvature [&lt;&lt;REF:Carlton-2004-14-1791&gt;&gt;,&lt;&lt;REF:Cozier-2002-277-4873048736&gt;&gt;,&lt;&lt;REF:Chandra-2019-10-1528&gt;&gt;] and the direct binding to SNX27 described in the present study. The timed increase in ESCPE-1 residency and localized concentration provides the handover mechanism to ensure that the captured and enriched cargo enter the forming tubular profile and the ensuing transport carrier. Continued localized WASH-mediated actin polymerization and the recruitment of motor proteins facilitates tubule maturation and fission [&lt;&lt;REF:Wassmer-2009-17-110&gt;&gt;-&lt;&lt;REF:Freeman-2014-127-2053&gt;&gt;] (Fig 8 &lt;&lt;FIG:pbio.3001601.g008&gt;&gt;). Additional cargo entry may be achieved through direct recognition by ESCPE-1 and yet to be described mechanisms [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;]. As with the assembly of other coat complexes [&lt;&lt;REF:Schmid-2007-448-883&gt;&gt;], association of the SNX27:Retromer:ESCPE-1 coat is driven by avidity through a series of low affinity interactions that evoke the consideration of checkpoints for monitoring the fidelity of pathway progression. It is important to state that this model is certainly an oversimplification and many key questions remain unanswered. Not least whether such a coat complex can be visualized through membrane associated cryo-electron tomography of reconstituted SNX27:Retromer:ESCPE-1 perhaps in the presence of ANKRD50.\nFig 8\nA 2-step model for the endosomal sorting of SNX27 cargos.\n(A) Schematics of the endosomal sorting of PDZ cargos. Cargos possessing a PDZbm that enter the endosomal system are recognized by SNX27 via its PDZ domain. (B) SNX27 binding to Retromer aids the retrieval of cargos from the lysosomal degradative path. (C) Once cargos have been retrieved, the SNX27 FERM engages the amino terminus of SNX1 and SNX2 mediating the inclusion of SNX27 cargos into ESCPE-1 tubulovesicular carriers. (D) This mechanism results in SNX27 cargos entering carriers that exit endosomes and are transported, directly and/or indirectly, to the cell surface. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; PDZbm, PDZ-binding motif; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27; WASH, Wiskott-Aldrich syndrome protein and SCAR homologue.\nThe importance of the amino-terminal extension of SNX1 and SNX2 has not previously been considered. While we and others [&lt;&lt;REF:Chandra-2022-83-100842&gt;&gt;-&lt;&lt;REF:Yong-2021-118-?&gt;&gt;] have independently established that they constitute primary sites of interaction with SNX27, it is tempting to speculate that they could orchestrate a broader set of protein-protein interactions. Interestingly, these unstructured amino termini may extend to over 30 nm in length and could allow the ESCPE-1 oligomeric assemblies to capture peripheral factors for the fine-tuned regulation of tubular maturation and scission. This would resemble the organization of the endocytic clathrin coat where several accessory proteins protrude from the core clathrin lattice into the cytoplasm to enable a regulated function of the endocytic coat during endocytic vesicle formation [&lt;&lt;REF:Skruzny-2020-16-e9009&gt;&gt;].\nOther than mediating the recruitment of accessory proteins, there is some evidence that these amino-terminal extensions could play a role in the assembly and membrane targeting of SNX-BAR dimers. In fact, besides SNX1 and SNX2, SNX4, SNX7, SNX8, and SNX30, all possess unstructured amino-terminal extensions of different length and of unknown function. The low-complexity region of the SNX8 homologue Mvp1 appears to stabilize soluble Mvp1 in a cytosolic tetrameric form where the PX and BAR domains are masked and unable to target membranes [&lt;&lt;REF:Sun-2020-11-1506&gt;&gt;]. The release of the amino terminus liberates functional dimers of Mvp1 facilitating their membrane association and function [&lt;&lt;REF:Sun-2020-11-1506&gt;&gt;]. It will be interesting to explore whether a similar mechanism also regulates ESCPE-1 targeting to the endosomal network in mammalian cells during endosomal recycling.\nAn important aspect of cargo sorting by SNX27 is regulation by posttranslational modification. Phosphorylation of residues within the core carboxyl-terminal PDZbm negatively impairs SNX27 association, but phosphorylation of residues preceding the minimal PDZbm can dramatically enhance affinity for the SNX27 PDZ domain [&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;]. A well-characterized example of cargo tail phosphorylation modulating SNX27 affinity and subsequent endosomal sorting is the beta2AR [&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;,&lt;&lt;REF:Cao-1999-410-286&gt;&gt;]. Furthermore, phosphorylation of SNX27 at Ser-51 alters the conformation of its PDZbm-binding pocket decreasing the affinity for cargo proteins, thereby inhibiting endosomal sorting [&lt;&lt;REF:Mao-2021-220-e202010048&gt;&gt;] and Ser-302 and Ser-304 phosphorylation in VPS26B can modulate SNX27 association [&lt;&lt;REF:Wang-2021-12-4826&gt;&gt;]. Similarly, ESCPE-1 function can be regulated by phosphorylation; in this case, Ser-226 of SNX5 undergoes phosphorylation, which decreases dimerization with SNX1 and SNX2 and reduces endosomal recycling [&lt;&lt;REF:Itai-2018-13-e0207205&gt;&gt;]. Therefore, a fascinating question regards whether the SNX27:ESCPE-1 interaction may be orchestrated by posttranslational modification. We have here shown that a series of acidic residues upstream of the DLF motif in the SNX1 and SNX2 amino termini contribute to the binding to SNX27 (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Interestingly, several phosphorylation sites in the aDLF sequences of SNX1 have been annotated, including Ser-39, Ser-41, and Ser-72, as well as Thr-72 [&lt;&lt;REF:Dephoure-2008-105-10762&gt;&gt;-&lt;&lt;REF:Bian-2014-96-253&gt;&gt;]. It is tempting to speculate that multiple phosphorylations of the ESCPE-1 amino termini may regulate its association with SNX27-Retromer and, in turn, dictate the timing and kinetics of the endosomal recycling route.\nFinally, a fascinating question relates to the molecular evolution of Retromer [&lt;&lt;REF:Koumandou-2011-124-1496&gt;&gt;]. In yeast, Retromer can form a stable pentameric assembly of 2 subcomplexes, the so called \"cargo-selective\" Vps26:Vps35p:Vps29p heterotrimer and the membrane remodeling Vps5p and Vps17p heterodimer [&lt;&lt;REF:Seaman-1998-142-665&gt;&gt;], the latter being considered analogous to ESCPE-1. Two functional processes, sequence-dependent cargo recognition and the biogenesis of tubulovesicular transport carriers, are therefore brought together in a single pentameric assembly [&lt;&lt;REF:Suzuki-2019-218-2876&gt;&gt;]. In metazoans, the VPS26:VPS35:VPS29 Retromer and ESCPE-1 appear to no longer form a stable pentameric assembly [&lt;&lt;REF:Norwood-2011-12-56&gt;&gt;]; rather, they have diverged into 2 separate complexes and in so doing have expanded their functional roles [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;,&lt;&lt;REF:Simonetti-2017-216-3695&gt;&gt;,&lt;&lt;REF:Norwood-2011-12-56&gt;&gt;,&lt;&lt;REF:Kvainickas-2017-216-3677&gt;&gt;]. Our phylogenetic analysis suggests that the SNX27:Retromer:ESCPE-1 assembly observed in metazoans evolved in a stepwise manner, with the SNX27-Retromer interaction evolving prior to the origin of metazoans, in the common ancestor of metazoans and choanoflagellates--the group of unicellular and colonical flagellates that are the closest living relatives of animals. The SNX27:ESCPE-1 interaction evolved later, during early metazoan evolution and likely prior to the divergence of cnidarians and bilaterians. Thus, within the SNX27:Retromer:ESCPE-1 pathway, the retrieval of cargo from the degradative fate appears to have evolved prior to the direct coupling to ESCPE-1 for promotion of tubulovesicular-based recycling to the cell surface. The divergence of the ancestral pentameric Retromer coat complex therefore appears to have occurred to accommodate the expanding complexity of the endomembrane system in metazoans [&lt;&lt;REF:Rout-2017-86-637&gt;&gt;]. The ability of metazoan Retromer to associate with SNX27 and the ability of ESCPE-1 to directly bind cargo through a helix-loop-helix region [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;], features not observed in yeast, have greatly expanded the capacity for sequence-dependent endosomal retrieval and recycling in metazoans. Indeed, it is tempting to speculate that the increased plasticity of the SNX27:Retromer:ESCPE-1 axis, and the existence of additional Retromer-independent pathways [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;], has allowed the metazoan endosomal network to accommodate the evolving diversity and number of cargos undergoing sequence-dependent sorting, leading to an expansion in the number of recycling itineraries that cargos can engage to interface with distinct membrane compartments. Dissecting how these sorting complexes and pathways are integrated and regulated, for example, by posttranslational modifications, to shape sequence-dependent cargo sorting within the metazoan endosomal network will be a major challenge moving forward.\nOverall, in defining the mechanistic basis of SNX27-Retromer association with ESCPE-1, we have provided new functional insight into the endosomal sorting of hundreds of internalized PDZbm-containing receptors, channels, transporter, enzymes, and adhesion molecules for repopulation of the cell surface [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. This increased molecular understanding provides new insight into the role of sequence-dependent endosome sorting in cellular processes ranging from the establishment of polarity and the uptake of nutrients through to synaptic function and cancer cell metastasis in both health and disease [&lt;&lt;REF:Chandra-2021-9-642378&gt;&gt;].\n\nMaterial and methods\n\nAntibodies\nAntibodies used in this study were mouse monoclonal antibodies GFP (clones 7.1 and 13.1; 11814460001; Roche, Germany) (1:2,000 for WB), LAMP1 (clone H4A3, AB2296838; DSHB, USA) (IF 1:200), SNX1 (clone 51/SNX1; 611482; BD, USA) (1:1,000 for WB, 1:200 for IF), SNX2 (clone 13/SNX2; 5345661; BD) (1:1,000 for WB, 1:200 for IF), SNX6 (clone d-5, 365965; Santa Cruz Biotechnology, USA) (1:1,000 for WB, 1:200 for IF), SNX27 (ab77799, Abcam, UK) (1:1,000 for WB, 1:200 for IF), beta-actin (A1978; Sigma-Aldrich, USA) (1:2,000 for WB), ITGA5 (610633, BD) (1:1,000 for WB), VPS29 (clone d-1, 398874; Santa Cruz Biotechnology) (1:200 for WB); rabbit monoclonal antibodies: EEA1 (C45B10; Cell Signaling Technology, USA) (IF 1:200), VPS35 (EPR11501(B); 157220; Abcam) (1:1,000 for WB), GLUT1 (ab115730, Abcam) (1:1,000 for WB, 1:200 for IF); rabbit polyclonal antibodies: GFP (GTX20290; GeneTex, USA) (WB 1:2,000), VPS26A (23892; Abcam) (1:1,000 for WB), VPS35 (97545; Abcam) (1:1,000 for WB, 1:200 for IF), VPS29 (98929; Abcam) (1:100 for WB); and goat polyclonal antibodies to VPS35 (10099; Abcam) (1:200 for IF).\n\nCell culture and transfection\nHeLa and HEK-293T cell lines were sourced from ATCC. Authentication was from the ATCC. Cells were grown in DMEM (Sigma-Aldrich) supplemented with 10% (v/v) FCS (Sigma-Aldrich) and penicillin/streptomycin (Gibco, USA) and grown under standard conditions. FuGENE HD (Promega, USA) was used for transient transfection of DNA according to the manufacturer's instructions. Cycloheximide was used to prevent up-regulation of protein synthesis. In this case, cycloheximide (C7698; Sigma-Aldrich) at 10 mug/ml was added to the cells 48 hours after FuGENE transfection for the indicated time points. For the generation of CRISPR-Cas9 SNX27 knockout HeLa cells, HeLa cells were transfected with pX330 plasmid coding for the gRNA against the gene of interest together with a puromycin resistance-expressing plasmid. Cells were then subjected to puromycin selection for 24 hours. The pooled population of cells was subjected to lysis and western blotting to confirm the knockout. The VPS35 knockout clonal cell line used in this study was characterized previously [&lt;&lt;REF:Kvainickas-2017-216-3677&gt;&gt;]. For GFP-based immunoprecipitations, HEK293T cells were transfected with GFP constructs using polyethylenimine (Sigma-Aldrich) and expression was allowed for 48 hours.\n\nImmunoprecipitation and quantitative western blot analysis\nFor western blotting, cells were lysed in PBS with 1% (v/v) Triton X-100 and protease inhibitor cocktail. The protein concentration was determined with a BCA assay kit (Thermo Fisher Scientific, USA), and equal amounts were resolved on NuPAGE 4% to 12% precast gels (Invitrogen, USA). Blotting was performed onto polyvinylidene fluoride membranes (Immobilon-FL; EMD Millipore, USA) followed by detection using the Odyssey infrared scanning system (LI-COR Biosciences, USA). For GFP-based immunoprecipitations, HEK-293T cells were lysed 48 hours after transfection in immunoprecipitation buffer (50 mM Tris-HCl, 0.5% (v/v) NP-40, and Roche protease inhibitor cocktail) and subjected to GFP trap (ChromoTek, Germany). Immunoblotting was performed using standard procedures. Detection was performed on an Odyssey infrared scanning system (LI-COR Biosciences) using fluorescently labeled secondary antibodies. In using the Odyssey, we routinely performed western blot analysis where a single blot is simultaneously probed with antibodies against 2 proteins of interest (distinct antibody species) followed by visualization with the corresponding secondary antibodies conjugated to distinct spectral dyes.\n\nBiotinylation of cell surface proteins\nFor surface biotinylation experiments, fresh Sulfo-NHS-SS Biotin (Thermo Fisher Scientific, #21217) was dissolved in ice-cold PBS at pH 7.8 at a final concentration of 0.2 mg/ml. Cells were washed twice in ice-cold PBS and placed on ice to slow down the endocytic pathway. Next, cells were incubated with the biotinylation reagent for 30 minutes at 4degreesC followed by incubation in TBS for 10 minutes to quench the unbound biotin. The cells were then lysed in lysis buffer and subjected to Streptavidin beads-based affinity isolation (GE Healthcare, USA).\n\nImmunofluorescence staining\nCells were fixed in 4% (v/v) PFA for 20 minutes and washed 3 times in PBS and permeabilized with 0.1% (v/v) Triton X-100. For LAMP1 staining, cells were instead permeabilized by submersion in liquid nitrogen for 20 seconds. Fixed cells were blocked in 1% (w/v) BSA and incubated in primary antibody and respective secondary antibody (Alexa Fluor; Thermo Fisher Scientific) in 1% (w/v) BSA.\n\nImage acquisition and image analysis\nMicroscopy images were collected with a confocal laser-scanning microscope (SP5 AOBS; Leica Microsystems, Germany) attached to an inverted epifluorescence microscope (DMI6000; Thermo Fisher Scientific). A 63x 1.4 NA oil immersion objective (Plan Apochromat BL; Leica Biosystems, Germany) and the standard SP5 system acquisition software and detector were used. Images were captured at room temperature as z stacks with photomultiplier tube detectors with a photocathode made of gallium-arsenide-phosphide (Leica Microsystems) for collecting light emission. Images were captured using Application Suite AF software (version 2.7.3.9723; Leica Microsystems) and then analyzed with the Volocity 6.3 software (PerkinElmer, USA).\n\nStatistics and reproducibility\nAll quantified western blot are the mean of at least 3 independent experiments. Statistical analyses were performed using Prism 7 (GraphPad Software, USA). Graphs represent means and SEM For all statistical tests, P &lt; 0.05 was considered significant and is indicated by asterisks.\n\nMolecular biology and cloning for recombinant protein production\nFor bacterial expression, full-length mouse SNX1 with an amino-terminal His-tag (mSNX1FL hereafter) was cloned into the pMW172Kan vector [&lt;&lt;REF:Collins-2008-9-366&gt;&gt;], and the truncated mSNX1 construct containing residues 1 to 139 was cloned into pGEX4T-2 vector for expression as a thrombin-cleavable amino-terminal GST fusion protein. DNA encoding full-length mouse SNX27 (mSNX27FL hereafter) was inserted into the pMCSG7 vector [&lt;&lt;REF:Eschenfeldt-2009-498-105&gt;&gt;] by the ligation-independent cloning (LIC) approach. Human SNX27 constructs, hSNX27FL, hSNX27FERM, and hSNX27FL single-site mutants R437D, K495D, K496D, R498D, and K501D were cloned into the pET-28a vector by GenScript and codon optimized for bacterial protein expression. An additional TEV cleavage site was inserted between the amino-terminal His-tag and hSNX27 coding sequences.\n\nProtein expression and purification\nThe DNA constructs of mSNX27 and mSNX1 were expressed using Escherichia coli BL21 CodonPlus (DE3) competent cells, while the hSNX27 constructs were expressed using E. coli BL21 (DE3) competent cells. The cells were grown in Ultra Yield flasks at 37degreesC, 180 rpm until the cell density reached OD600 approximately 0.8. Protein expression was then induced with 1 mM IPTG and cells incubated for 16 to 18 hours at 18degreesC. For SNX27, 0.5 mM IPTG was added for induction to get optimal protein expression yield. To collect cell pellets, the cells were harvested by 10-minute centrifugation (JLA 8.1 rotor) at 6,000 rpm, 4degreesC.\nFor SNX1 purification, the cell pellet was resuspended in lysis buffer containing 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-mercaptoethanol (beta-ME), 1 mM benzamidine, and 10 mug/mL DNase I dissolved in PBS containing 10% glycerol. For SNX27 proteins, the cell pellet was resuspended in lysis buffer containing 20 mM Tris-HCl pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM beta-ME, 1% Triton X-100, 1 mM benzamidine, and 10 mug/mL DNase I dissolved in PBS containing 10% glycerol. Cells were lysed by high-pressure homogenization (35 Kpsi). The homogenate was centrifuged for 30 minutes at 4degreesC using a JA 25.50 rotor with 19,000 rpm. The supernatant of His-tagged protein homogenates were loaded onto preequilibrated TALON metal affinity resin (Clontech, USA) or Ni-nitrilotriacetic acid (NTA) resin, while the supernatant of GST-fusion protein homogenates were loaded onto preequilibrated glutathione Sepharose beads (GE Healthcare) and incubated on an orbital rotator for an hour in cold room.\nAffinity matrices with bound fusion proteins were loaded into empty glass chromatography columns (Bio-Rad, USA) washed with 100 ml of washing buffer containing either 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-ME or 20 mM Tris-HCl pH 8.0, 200 mM NaCl, 5% glycerol, 1 mM beta-ME. For His-tagged protein purification, 10 mM and 300 mM imidazole was added to wash and elution buffers, respectively. To collect the GST-tagged mSNX1 constructs, 50 mM reduced glutathione in elution buffer was applied for elution. cOmplete EDTA-free protease inhibitor tablets were added to the eluted SNX1 samples to reduce proteolytic degradation of their disordered amino-terminal region.\nThe fractions containing eluted proteins were concentrated to approximately 5 ml by Amicon cc ultrafiltration devices (10 kDa or 30 kDa molecular mass cutoff) (Millipore) and centrifuged in a bench centrifuge for 10 minutes at maximum speed to remove any precipitated material before being subjected to size-exclusion chromatography (SEC). For SEC, the HiLoad 16/600 Superdex 200 (GE Healthcare) column was used with proteins eluted in SEC buffer containing 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-ME or 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5% glycerol, and 1 mM beta-ME. To check the purify and integrity of the purified fractions, sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) was applied using Bolt 12% Bis-Tris gel with 1x MES running buffer at 190V for 25 minutes. After the SDS-PAGE, gels were stained with Coomassie Brilliant Blue R250 solution for visualization. For hSNX27FERM, an additional anion exchange chromatography (IEX) was applied using the Mono Q anion exchange chromatography column preequilibrated with IEX buffer A: 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5% glycerol, 1 mM beta-ME. A linear salt gradient from 0% to 40% IEX buffer B (20 mM Tris-HCl pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM beta-ME) was conducted. Proteins usually eluted in buffer containing 200 mM NaCl, and fractions collected were analyzed by SDS-PAGE.\n\nIsothermal titration calorimetry\nHuman SNX1 and SNX2 peptide sequences were synthesized by Genscript (USA). To obtain the peptide stock solutions at neutral pH, 100 mM Tris-HCl, pH 10.6 was used to dissolve the peptide to a final concentration of 8 mM. The peptides were SNX135-51 EAGDSDTEGEDIFTGAA, SNX135-51(D45K) EAGDSDTEGEKIFTGAA, SNX175-92 NGIHEEQDQEPQDLFADA, SNX216-33 TDFEDLEDGEDLFTSTVS, SNX216-33(DLF/SSS) TDFEDLEDGESSSTSTVS, and SNX262-82 TEVVLDDDREDLFAEATEEVS.\nAll microcalorimetry experiments were conducted at 25degreesC using a PEAQ ITC (Malvern, UK) in 50 mM HEPES (pH 7.5), 200 mM NaCl, 5% glycerol, and 1 mM beta-ME (ITC buffer). All proteins were purified by SEC using the same buffer prior to ITC experiments to avoid buffer mismatches. To measure binding of SNX27 to SNX1 proteins, 800 muM of hSNX27FERM was titrated into 20 muM of GST-mSNX11-139 or His-SNX1FL. For the binding of SNX1 and SNX2 peptides to SNX27, peptides were diluted 10-fold to 800 muM in ITC buffer peptides and titrated into 20 muM SNX27FL proteins containing the same 1:10 ratio of peptide buffer (100 mM Tris-HCl, pH 10.6). The competitive ITC experiments were carried out by titrating 800 muM SNX175-92 peptides into 20 muM proteins with 2-fold molar excess SNX216-33 peptide preincubated with SNX27 on ice for an hour. For ITC experiment using SNX175-92 against hSNX27 wild-type and single-site mutants, 600 muM SNX175-92 peptide was titrated into 20 muM hSNX27 proteins.\nThe ITC experiments were conducted with 1 injection of 0.4 muL followed by a series of 12 injections of 3.2 muL each with 180-second intervals and a stirring speed of 850 rpm. The heat exchange of interactions was obtained through integrating the observed peaks and background correction by subtracting the heats of dilution. Data were analyzed with Malvern software package by fitting and normalized data to a single-site binding model, yielding the thermodynamic parameters Kd, DeltaH, DeltaG, and -TDeltaS for all binding experiments. The stoichiometry was refined initially, and if the value was close to 1, then N was set to exactly 1.0 for calculation. All experiments were performed at least twice to guarantee reproducibility of the data.\n\nAlphaFold predictions of peptide binding to SNX27\nTo generate predicted models of SNX1 and SNX2 amino-terminal peptides associating with the SNX27 FERM domain, we used the AlphaFold2 neural network [&lt;&lt;REF:Jumper-2021-15-1&gt;&gt;] implemented within the freely accessible ColabFold pipeline [&lt;&lt;REF:Mirdita-2021-?-?&gt;&gt;]. For each peptide-modeling experiment, the sequence of the human SNX27a FERM domain (residues 269 to 531; Q96L92) was modeled with the 4 separate amino-terminal aDLF sequences from SNX1 and SNX2. The specific peptides we modeled were residues 33 to 56 and 75 to 98 from human SNX1 (Q13596) and residues 14 to 37 and 60 to 83 from human SNX2 (O60749). ColabFold was executed using default settings where multiple sequence alignments were generated with MMseqs2 [&lt;&lt;REF:Mirdita-2019-35-2856&gt;&gt;], and structural relaxation of final peptide geometry was performed with Amber [&lt;&lt;REF:Hornak-2006-65-712&gt;&gt;] to generate 3 models per peptide. Sequence conservation was mapped onto the modeled SNX27 FERM domain structure with Consurf [&lt;&lt;REF:Ashkenazy-2016-44-W344&gt;&gt;]. Structural alignments and images were generated with Pymol (Schrodinger, USA).\n\nPhylogenetic analyses\nHomologous proteins were identified using NCBI protein-protein BLAST (with default parameters) against all non-redundant GenBank CDS Translations+PDB+SwissProt+PIR+PRF excluding environmental samples from WGS projects for a wider range of species (unaligned sequence data with accession numbers available in S4 Fig). For the ctenophore data, we used the BLAST (with default parameters) from the Mnemiopsis Genome Project Portal (MGP) portal [&lt;&lt;REF:Ryan-2013-342-1242592&gt;&gt;-&lt;&lt;REF:Moreland-2020-?-?&gt;&gt;]. Sequences were aligned using MAFFT v7.480 [&lt;&lt;REF:Katoh-2013-30-772&gt;&gt;], L-INS-I mode, and then used to infer initial exploratory trees to identify specific duplications (using IQtree 2.1.4 [&lt;&lt;REF:Minh-2020-37-1530&gt;&gt;] with the LG+F+G model and 1,000 ultrafast bootstraps). After orthologues of each copy of the gene were identified in a range of taxa, we then performed iterative tree inferences to confirm paralogues and to identify species-specific duplication events. Subsequent maximum likelihood phylogenetic analyses were performed using IQtree 2.1.4 with the best fitting model selected using the Bayesian information criterion (BIC), including models which account for site-rate heterogeneity (LG+C10 ...C60) [&lt;&lt;REF:Si Quang-2008-24-2317&gt;&gt;], empirical amino acid frequencies (+F), and accounting for across-site rate variation using either a Gamma distribution (+G) [&lt;&lt;REF:Yang-1994-39-306&gt;&gt;] or the free rates model (+R) [&lt;&lt;REF:Yang-1995-139-993&gt;&gt;,&lt;&lt;REF:Soubrier-2012-29-3345&gt;&gt;] with 10,000 ultrafast bootstrap replicates. The best fitting model according to BIC was LG+C30+F+G for the final SNX1 and SNX2 tree (see S4 Fig) and LG+C20+F+G for SNX27 and VPS26 (alignments and maximum likelihood trees in S4 Fig).\n\nSupporting information\nS1 Fig\nWestern blot showing the levels of GLUT1 in parental HeLa, VPS35 KO, and SNX27KO not transfected or transiently transfected with GFP-SNX27 WT, GFP-SNX27 Delta67-77, and GFP-SNX27 DeltaFERM.\nThe blot is representative of 3 independent assays. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. KO, knockout; SNX27, sorting nexin-27; WT, wild-type.\n(TIF)\nClick here for additional data file.\nS2 Fig\n(A) Co-immunoprecipitation of GFP-tagged extended amino-terminal regions (NT) of SNX1 and SNX2 alongside the GFP-tagged short amino-terminal regions of SNX5 and SNX6 expressed in HEK293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. (B) Co-immunoprecipitation of GFP-tagged SNX2 WT or SNX2 KK (D27K, D72K) expressed in HEK293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation and the immunoprecipitates were blotted for SNX27, SNX6, and GFP. The blot is representative of 4 independent GFP traps. (C) Competitive ITC assay of SNX216-33 and mSNX27FL preincubated with 2-fold molar excess of SNX175-92 peptide showing that binding is blocked by the competing SNX175-92 peptide. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. SNX1, sorting nexin-1; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\n(TIF)\nClick here for additional data file.\nS3 Fig\n(A) SNX27 surface colored according to sequence conservation. The peptides are predicted to bind to a highly conserved pocket in SNX27. (B, C) As described in methods, 12 models of SNX27 were generated in total, consisting of 3 models each of SNX27 in association with the 4 aDLF sequences from human SNX1 and SNX2. B shows the overlay of all 12 models in Calpha trace representation colored by the pLDDT score. The pLDDT is a per-residue confidence score between 0 (lowest confidence) and 100 (highest confidence). The majority of the SNX27 structures show very high pLDDT scores and overlay with very high precision. C shows only the best 8 models where the core aDLF sequences have identical binding conformations. aDLF, acidic-Asp-Leu-Phe; pLDDT, predicted Local Distance Difference Test; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27.\n(TIF)\nClick here for additional data file.\nS4 Fig\nMaximum likelihood phylogenies of (A) SNX27, (B) VPS26A/B and (C) VPS5/SNX1/SNX2. (A) SNX27 (green) with SNX17 (red) used as an outgroup with representatives across Filozoa. Amino acids aligned with Homo sapiens in positions 437, 495, 496, 498, and 501 are written next to the species name. (B) VPS26A, VPS26B, and VPS26C, with VPS26C in red and VPS26B in green; the duplication event occurring in the ancestral vertebrate is in blue. Interestingly, we also find another independent duplication of VPS26B in plants evidenced by Arabidopsis thaliana and Quercus suber in this tree. (C) The SNX1 sequences across choanozoans, with the duplication in vertebrates in blue. The number of aDLF motifs in the amino terminus is written next to the species name. Phylogenies were inferred under the best-fit model as determined by BIC. Branch support values are percentages based on 10,000 ultrafast bootstraps. aDLF, acidic-Asp-Leu-Phe; BIC, Bayesian information criterion; SNX1, sorting nexin-1; SNX27, sorting nexin-27.\n(TIF)\nClick here for additional data file.\nS1 Table\nSummary of ITC binding parameters.\nITC, isothermal titration calorimetry.\n(PDF)\nClick here for additional data file.\nS1 Raw Images\nUnprocessed images for blots shown in the paper.\n(PDF)\nClick here for additional data file.\nS1 Data\nNumerical data for Figs 1B, 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;, 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;, 3A, 3C, 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;, 4D, 4G &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;, 5A, 5B, 5C, 5D, 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;, 6A, 6B, 6C, 6D and 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt; and S1A and S2B.\n(XLSX)\nClick here for additional data file.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nA central role of the endosomal-lysosomal network is to determine the sorting of integral proteins and their associated proteins and lipids, collectively termed \"cargos,\" between 2 fates, either degradation within lysosomes or retrieval from this fate for the promotion of recycling to a variety of organelles that include the cell surface [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;]. Efficient and selective endosomal-lysosomal cargo sorting is essential for the function of all eukaryotic cells with increasing evidence pointing to defects in endosomal retrieval and recycling being at the heart of a wide array of human diseases, most notably those associated with neurodegeneration [&lt;&lt;REF:Small-2020-12-?&gt;&gt;]. Establishing the molecular mechanisms and functional significance of endosomal cargo retrieval and recycling is therefore fundamental to understand eukaryotic cell biology.\nTwo general concepts are considered to account for the mechanism of endosomal cargo retrieval and recycling. Sequence-independent sorting posits that sorting can proceed without the need for any sequence information within the intracellular cytoplasmic domains of integral membrane cargos [&lt;&lt;REF:Maxfield-2004-5-121&gt;&gt;]. On the other hand, sequence-dependent sorting is controlled by specific signals in the cytoplasmic domain of cargos. These signals are typically short linear peptide motifs that direct the sorting process through coupling to ancient and evolutionary conserved coat complexes [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;,&lt;&lt;REF:McNally-2018-28-807&gt;&gt;,&lt;&lt;REF:Chen-2019-20-465&gt;&gt;]. In coupling sorting motif recognition with membrane remodeling, these coats coordinate the process of cargo retrieval with the formation of cargo-enriched transport carriers for onward recycling. In mammals, these coats include sorting nexin-3 (SNX3)-Retromer [&lt;&lt;REF:Strochlic-2007-177-115&gt;&gt;-&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;], SNX27-Retromer [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], SNX17-Retriever [&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;], the CCC (CCDC22/CCDC93/COMMD) [&lt;&lt;REF:Phillips-Krawczak-2015-26-91&gt;&gt;,&lt;&lt;REF:Bartuzi-2016-7-10961&gt;&gt;], and endosomal SNX-BAR sorting complex for promoting exit 1 (ESCPE-1) complexes [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;] and various accessory proteins including the actin polymerizing Wiskott-Aldrich syndrome protein and SCAR homologue (WASH) complex [&lt;&lt;REF:Gomez-2009-17-699&gt;&gt;,&lt;&lt;REF:Derivery-2009-17-712&gt;&gt;].\nSorting nexin-27 (SNX27) is an endosome-associated cargo adaptor composed of a central PX (phox homology) domain flanked by an amino-terminal PDZ (PSD95, disks large, and zona occludens) domain and a carboxyl-terminal FERM (4.1/ezrin/radixin/moesin) domain [&lt;&lt;REF:Kaiji-2003-8-434&gt;&gt;,&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;]. The PDZ domain recognizes a specific sorting motif, termed the PDZ-binding motif (PDZbm), present at the very carboxyl termini of over 400 cargo proteins including receptors, transporters, channels, enzymes, and adhesion molecules [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;-&lt;&lt;REF:McMillan-2021-10-e59432&gt;&gt;]. In recognizing these internalized cargos, SNX27 regulates their retrieval from lysosomal degradation and promotes their recycling back to the plasma membrane [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;].\nTo facilitate the retrieval process, the SNX27 PDZ domain associates with Retromer, a stable heterotrimer of VPS26 (A and B isoforms expressed in humans [&lt;&lt;REF:Kerr-2005-6-991&gt;&gt;]), VPS35 and VPS29, to form the SNX27-Retromer [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. Retromer itself assembles to form dimeric arches that may cluster SNX27 captured cargos [&lt;&lt;REF:Kendall-2020-28-393&gt;&gt;,&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;,&lt;&lt;REF:Kovtun-2018-561-561&gt;&gt;] and associates with the WASH complex to allow the localized Arp2/3-dependent polymerization of branched filamentous actin [&lt;&lt;REF:Harbour-2012-442-209&gt;&gt;,&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;]. In a poorly understood process, these assemblies and actin dynamics are proposed to generate a retrieval subdomain on the endosomal membrane that is enriched in cargo destined for recycling [&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;-&lt;&lt;REF:Lee-2016-7-10939&gt;&gt;]. It is from these subdomains that further membrane remodeling ensues to drive the biogenesis of cargo-enriched tubular profiles and tubulovesicular carriers that physically transport cargo back to the cell surface. Importantly, defects in SNX27-Retromer function are associated with the pathoetiology of human disease most notable neurological disorders and neurodegenerative disease [&lt;&lt;REF:Chandra-2021-9-642378&gt;&gt;].\nA membrane tubulating Bin/Amphiphysin/Rvs (BAR) domain-containing protein associated with SNX27-Retromer tubular profiles is sorting nexin-1 (SNX1) [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;]. SNX1, and its functional ortholog SNX2, is part of the ESCPE-1 complex [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;,&lt;&lt;REF:Carlton-2004-14-1791&gt;&gt;], a membrane tubulating complex that in vitro can reconstitute the formation of membrane tubules with diameters similar to those observed in cellulo [&lt;&lt;REF:van Weering-2012-31-4466&gt;&gt;]. To be functional, ESCPE-1 forms a stable heterodimer of either SNX1 or SNX2 associated with either of SNX5 or SNX6, 2 additional BAR domain-containing orthologs [&lt;&lt;REF:Wassmer-2007-120-45&gt;&gt;]. Importantly, the mechanistic basis of how SNX27-Retromer delivers cargo into forming ESCPE-1 tubular profiles and tubulovesicular carriers remains a major unanswered question.\nHere, we establish that a conserved surface in the FERM domain of SNX27 binds directly to acidic-Asp-Leu-Phe (aDLF) sequence motifs present in the disordered amino termini of the SNX1 and SNX2 subunits of ESCPE-1. We show that this interaction is required to handover PDZbm containing cargo proteins retrieved from lysosomal degradation by SNX27-Retromer into ESCPE-1 tubular profiles that promote cargo recycling for subsequent repopulation at the cell surface. Through evolutionary analysis of those protein-protein interactions that assemble the SNX27:Retromer:ESCPE-1 coat complex, we reveal how this sorting axis has evolved to accommodate the increasing complexities of endosomal sorting observed in animals.\n\nResults\n\nThe FERM domain of SNX27 promotes cargo recycling following SNX27-Retromer-mediated retrieval from lysosomal degradation\nPreviously, we have established that the interaction of Retromer with the PDZ domain of SNX27 is a crucial step in preventing the routing of internalized cargo into the lysosomal degradative pathway [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. To identify the region(s) of SNX27 required for promoting subsequent cargo recycling to the plasma membrane, we used CRISPR gene editing to generate SNX27 knockout HeLa cells (Fig 1A &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Compared to control cells, where endogenous GLUT1 was enriched at the cell surface, in the SNX27 knockout cells, GLUT1 displayed a steady-state enrichment with LAMP1-positive lysosomes, a phenotype that was also observed in a VPS35 knockout HeLa cell line (Fig 1B &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Importantly, reexpression of GFP-SNX27 in the SNX27 knockout cells rescued the retrieval of GLUT1 from the lysosomal distribution and promoted its repopulation at the cell surface (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). These data mirror those previously published using RNA interference (RNAi)-mediated SNX27 suppression [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;] and confirm the SNX27 knockout HeLa cell line as a suitable screening platform through which to dissect the features of SNX27 required for promoting GLUT1 endosome-to-plasma membrane recycling.\nFig 1\nThe FERM domain of SNX27 promotes cargo recycling following SNX27-Retromer-mediated lysosomal retrieval.\n(A) Western blot confirming the absence of the proteins of interest in VPS35 KO and SNX27 KO cells. (B) Representative images of parental Hela cells, VPS35 KO HeLa clonal line and SNX27 KO HeLa cells. The steady-state distribution of GLUT1 was analyzed by immunofluorescence staining of fixed cells with GLUT1 and the endosomal and lysosomal markers VPS35 and LAMP1. A total of 70 cells were analyzed in each condition for colocalization between GLUT1 and LAMP1 across n  =  4 independent experiments. The Pearson coefficient values of LAMP1 colocalization were compared to the values of parental HeLa using 1-way ANOVA and Dunnett test: P &lt; 0.0001 (HeLa vs VPS35 KO), &lt; 0.0001 (HeLa versus SNX27 KO). Pearson coefficient values of VPS35 colocalization were compared using unpaired 2-tailed t test: P &lt; 0.0001 (HeLa versus SNX27 KO). (C) Representative images of SNX27 KO HeLa cells transiently transfected with GFP-SNX27 WT, GFP-SNX27 Delta67-77 and GFP-SNX27 DeltaFERM. Moreover, 48 hours after transfection, cells were fixed and immunostained. The steady-state distribution of GLUT1 was analyzed by immunofluorescence staining of fixed cells with GLUT1 and the endosomal and lysosomal markers VPS35 and LAMP1. A total of 70 cells were analyzed in each condition for colocalization between GLUT1 and LAMP1, and GLUT1 and VPS35 across n  =  4 independent experiments. The Pearson coefficient values were compared to the values of SNX27 KO cells rescued with GFP-SNX27 WT using 1-way ANOVA and Dunnett test. LAMP1 colocalization values were P &lt; 0.0001 (+WT versus +Delta67-77), 0.3559 (+WT versus +DeltaFERM). VPS35 colocalization values were P &lt; 0.0001 (+WT versus Delta67-77), 0.001 (+WT versus +DeltaFERM). Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. Scale bars, 25 mum (micrographs) and 5 mum (magnified images). The data underlying the graphs shown in the figure can be found in S1 Data. KO, knockout; SNX27, sorting nexin-27; WT, wild-type.\nIn contrast to wild-type SNX27, expression of a SNX27 mutant lacking the loop region essential for Retromer association (GFP-SNX27Delta67-77) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;] failed to rescue retrieval from lysosomal colocalization and failed to promote GLUT1 cell surface recycling (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). These data further confirm that the association of SNX27 with Retromer is a crucial step in coupling sequence-dependent cargo recognition with cargo retrieval from the lysosomal degradative fate [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. Consistent with this, the expression of a SNX27 deletion mutant lacking the FERM domain (GFP-SNX27DeltaFERM), a mutant that retained the association of SNX27 with Retromer, retained the ability to retrieve GLUT1 from lysosomes, as defined by a decreased colocalization between GLUT1 and LAMP1 (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). However, the GFP-SNX27DeltaFERM mutant failed to promote GLUT1 recycling to the cell surface, rather GLUT1 was enriched in a Retromer-positive endosomal compartment (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). The overall levels of GLUT1 were not decreased in the KO and rescue conditions compared to parental cells (S1 Fig). Together, these data are consistent with a model in which the coupling of SNX27 to Retromer is necessary for retrieval from the lysosomal degradative fate, while the SNX27 FERM domain is required to promote the subsequent endosome-to-cell surface recycling of GLUT1.\n\nSNX1 and SNX2 couple ESCPE-1 to the FERM domain of SNX27\nESCPE-1 regulates tubular-based endosomal recycling of several cargos, many of which are also dependent on SNX27-Retromer [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;]. ESCPE-1 is a heterodimeric assembly of the functionally redundant SNX1 or SNX2 associated with the functionally redundant SNX5 or SNX6 (Fig 2A &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;) [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;], while SNX27-Retromer is an assembly of SNX27 with the VPS26A/B:VPS35:VPS29 heterotrimeric Retromer complex (Fig 2B &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. To define a possible connection between SNX27 and ESCPE-1, we first performed a series of GFP-nanotrap immunoisolation of SNX27 and various SNX27 mutants expressed in HEK-293T cells and probed for the presence of associated endogenous ESCPE-1. Robust association was observed between full-length GFP-SNX27 and all components of ESCPE-1 (Fig 2C-2E &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). This association was independent of the SNX27 PDZ domain as binding was retained in the PDZ deletion mutant GFP-SNX27DeltaPDZ and was not observed when just the PDZ domain was expressed in HEK-293T cells (Fig 2C and 2D &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Retromer association with SNX27 was not required for ESCPE-1 binding as GFP-SNX27Delta67-77 and GFP-SNX27(L67A/L77A), 2 mutants that fail to bind to Retromer [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], each retained association with ESCPE-1 (Fig 2E &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). However, ESCPE-1 association was lost upon deletion of the SNX27 FERM domain (SNX27DeltaFERM) or upon switching the FERM domain of SNX27 for the corresponding FERM domain from SNX17 (Fig 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Moreover, just the isolated FERM domain of SNX27, when expressed in HEK-293T cells, was able to associate with ESCPE-1 (Fig 2D &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Finally, in the reverse immunoisolations, GFP-tagged SNX1, SNX2, SNX5, or SNX6 revealed that only SNX1 and SNX2 displayed a pronounced association with endogenous SNX27 when expressed in HEK-293T cells (Fig 2F &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Moreover, this association did not require the presence of the SNX1 or SNX2 BAR domains, since deletion of these domains failed to affect binding to SNX27 (Fig 2G &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Together, these data are consistent with the FERM domain of SNX27 associating with the SNX1 and SNX2 components of ESCPE-1.\nFig 2\nThe FERM domain of SNX27 interacts with SNX1 and SNX2.\n(A) Schematics of ESCPE-1 assembly consisting of heterodimers of the SNX-BAR proteins SNX1 or SXN2 with SNX5 or SNX6. These SNXs have BAR and PX domains responsible of dimer formation and membrane targeting, respectively. SNX1 and SNX2 also have an extended amino-terminal unstructured extension (NT). (B) Schematics of the SNX27-Retromer assembly. Retromer is a stable heterotrimer of VPS35, VPS26, and VPS29. Retromer directly interacts with SNX27 through a beta-hairpin insertion within the PDZ domain (residues 67 to 79) that engages a cleft of the VPS26 protein between the amino-terminal and carboxyl-terminal subdomains. (C) Co-immunoprecipitation of GFP-tagged SNX27 FL, SNX27 DeltaPDZ, SNX27 DeltaFERM and a chimera consisting of the SNX27 gene where its FERM domain was swapped for that of SNX17. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensities of VPS35, VPS26A, VPS29, SNX1, and GFP were measured from n =  3 independent experiments using Odyssey software. The band intensities, normalized to GFP expression, are presented as the average fraction of the FL. (D) Co-immunoprecipitation of GFP-tagged SNX27 FL, isolated SNX27 PDZ, PX, and FERM domains expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (E) Co-immunoprecipitation of GFP-tagged SNX27 FL and the SNX27 mutants Delta67-77 and L67A-L74A expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (F) Co-immunoprecipitation of GFP-tagged SNX1, SNX2, SNX5, and SNX6 expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. (G) Co-immunoprecipitation of GFP-tagged SNX1, SNX2, and SNX5 lacking the BAR domain (DeltaBAR) expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; FL, full-length; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27.\n\nThe amino termini of SNX1 and SNX2 present tandem acidic-Asp-Leu-Phe motifs that associate with the SNX27 FERM domain\nCompared to SNX5 and SNX6, human SNX1 and SNX2 each contain amino-terminal extensions of 142 and 139 amino acids, respectively, which are predicted to be unstructured. Expression in HEK-293T cells of GFP-SNX1(1-279), a construct encoding for the amino-terminal extension plus the SNX1 PX domain, and GFP-SNX1(1-138), a construct encoding for just the amino-terminal extension of SNX1, established that the amino-terminal extension of SNX1 is sufficient for association with endogenous SNX27 (Fig 3A &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Likewise, the amino terminus of SNX2 interacted with SNX27, while those of SNX5 and SNX6 failed to do so (S2A Fig). To establish the direct nature of the association, we recombinantly expressed and purified full-length mouse SNX1 and the isolated FERM domain of human SNX27; mouse and human SNX1 are 94.2% identical. Isothermal titration calorimetry (ITC) established that SNX1 bound directly to the FERM domain of SNX27 with an affinity (Kd) of 41.0 muM (Fig 3B &lt;&lt;FIG:pbio.3001601.g003&gt;&gt; and S1 Table). A similar binding affinity (39.1 muM) was observed with a recombinant protein corresponding to the amino-terminal extension of mouse SNX1 (85.9% sequence identity with human SNX1) (Fig 3B &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). To map down the features of SNX1 and SNX2 required for SNX27 binding, we performed a deletion mutagenesis screen of the amino-terminal extension of human SNX1 (Fig 3C &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Walking upstream from a minimal binding region encoded by residues 1 to 138 established that 2 regions were required for maximal SNX27 binding. Deletion of residues 79 to 100 led to a pronounced reduction in binding and subsequent deletion of residues 41 to 54 abolished the interaction. Alignment of these regions of SNX1 revealed 2 similar motifs with DIF or DLF sequences flanked by upstream acidic residues. Comparison with the amino terminus of SNX2, which also binds SNX27 (S2A Fig), showed that it also has 2 similar sequences in its disordered amino-terminal domains, and, altogether, these define a conserved sequence D/E-x-x-D-I/L-F, here termed the aDLF motif (Fig 3D and 3E &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;).\nFig 3\nThe amino termini of SNX1 and SNX2 directly interact with SNX27 FERM domain through tandem aDLF motifs.\n(A) Co-immunoprecipitation of GFP-tagged SNX1 FL or truncation mutants corresponding to residues 1 to 279 and 1 to 138 expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The band intensity of SNX27 was measured from n =  3 independent experiments using Odyssey software and was normalized to GFP expression. The relative binding of SNX1 constructs was compared with the SNX1 FL using a 1-way ANOVA and Dunnett test. (B) Binding of mSNX1FL (orange) and mSNX11-139 (blue) with hSNX27FERM by ITC. (C) Co-immunoprecipitation of GFP-tagged SNX1 amino terminus (1 to 138) and a panel of constructs corresponding to serial truncations of SNX1 amino terminus. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensity of SNX27 and GFP was measured from n = 4 independent experiments using Odyssey software and was normalized to GFP expression. The relative binding of SNX1 truncation mutants was compared with the SNX1 amino terminus (1-138) using a 1-way ANOVA and Dunnett test. (D) Alignment of the SNX1 regions that are involved in SNX27 binding with the corresponding sequences in the SNX2 amino terminus. The conserved residues between SNX1 and SNX2 account for a consensus motif of acidic residues followed by DI/LF (aDLF motif). (E) Alphafold structural prediction for FL SNX1 [&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;]. The amino terminus of SNX1 is shown in its full extension and the residues involved in SNX27 binding are colored in red and green. (F) Co-immunoprecipitation of GFP-tagged SNX1 WT and a panel of amino acid swap mutants across the aDLF motifs of SNX1. The constructs were transiently transfected in HEK293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensities of SNX27, SNX6, and GFP were measured from n = 4 independent experiments using Odyssey software. The band intensity, normalized to GFP expression, is presented as the average fraction of SNX1 WT. The binding of SNX1 point mutants to SNX27 and SNX6 were compared with the SNX1 WT using a 1-way ANOVA and Dunnett test. (G) Binding of mSNX27FL and hSNX1 peptides by ITC. SNX135-51(D45K): green; SNX135-51: orange; SNX175-92: lilac; ND*: no binding detected. (H) Binding of mSNX27FL and hSNX2 peptides by ITC. SNX216-33(DLF/SSS): blue; SNX216-33: green; SNX262-82: lilac; ND*: no binding detected. The ITC graphs represent the integrated and normalized data fit with 1:1 ratio binding. The binding affinity (Kd) is given as mean of at least 2 independent experiments. Thermodynamic parameters for the binding are provided in S1 Table. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001. The data underlying the graphs shown in the figure can be found in S1 Data. aDLF, acidic-Asp-Leu-Phe; FL, full-length; ITC, isothermal titration calorimetry; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27; WT, wild-type.\nSite-directed mutagenesis targeting the first aDLF motif of SNX1 established that full-length GFP-SNX1(E42K), GFP-SNX1(E44K), GFP-SNX1(D45K), GFP-SNX1(I46K), and GFP-SNX1(F47K) all displayed reduced binding to SNX27 of around 50% (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). A similar reduction in binding was also observed when targeting the second aDLF motif of SNX1 as observed with the GFP-SNX1(E84K), GFP-SNX1(D87K), and GFP-SNX1(F89D) mutants (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). In a double GFP-SNX1(D45K,D87K) mutant that simultaneously targeted key residues in both aDLF motifs SNX27, binding was reduced to near undetectable levels (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Importantly, all mutants retained their ability to associate with SNX6, establishing that ESCPE-1 was still able to assemble. In addition, the mutant targeting both aDLF motifs in SNX2, GFP-SNX2(D26K,D72K), also reduced SNX27 binding to near undetectable levels (S2B Fig). Next, we used ITC to quantify the binding of aDLF peptides from SNX1 and SNX2 to full-length mouse SNX27. Peptides corresponding to residues 35 to 51 and 75 to 92 encompassing the first and second aDLF motifs of human SNX1, respectively, bound to SNX27 with respective affinities of 33.4 muM and 13.3 muM (Fig 3G &lt;&lt;FIG:pbio.3001601.g003&gt;&gt; and S1 Table). Establishing the redundant nature of the interaction with SNX27, we designed peptides corresponding to the first and second aDLF motifs of SNX2, namely residues 16 to 33 and 62 to 82. These also bound to full-length mouse SNX27 with affinities of 37.8 muM and 22.6 muM, respectively (Fig 3H &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Further defining the redundancy between SNX1 and SNX2, introduction of a charge swap mutation into the SNX1 35-51 peptide, SNX1(D45K), and a triple mutation in the SNX2 16-33 peptide, SNX2(DLF-SSS), resulted in mutant peptides that showed no detectable binding to SNX27 (Fig 3G and 3H &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Finally, a competitive ITC assay using full-length SNX27 preincubated with excess SNX1 75-92 peptide established that it competed for binding of SNX2 16-33 peptide, consistent with both sequences associating with the same surface within the SNX27 FERM domain (S2C Fig). Overall, these data establish that through tandem aDLF motifs present within their unstructured amino terminal extensions SNX1 and SNX2 directly associate with the FERM domain of SNX27.\n\nA basic surface in the SNX27 FERM domain is required for SNX1/SNX2 binding\nSNX27 and SNX17 are related endosome associated cargo adaptors [&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;,&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;]. While they display entirely distinct functional roles in sequence-dependent endosomal cargo sorting [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;], they share structurally similar PX and FERM domains [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;,&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;] (Fig 4A &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). A comparative immunoisolation of GFP-SNX17 and GFP-SNX27 from HEK-293T cells established that SNX17 does not bind to ESCPE-1 (Fig 4B &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;), consistent with the loss of ESCPE-1 binding to the SNX27 chimera containing the SNX17 FERM domain (Fig 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). To identify potential regions for aDLF motif binding to the SNX27 FERM domain, we initially performed a comparative structural analysis between the X-ray structure of the SNX17 FERM domain [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;] and a homology model of the SNX27 FERM domain [&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;]. In particular, we focused on the distribution and clustering of complementary basic amino acids. This revealed that the FERM domain of SNX27 possesses a basic surface defined in part by R437, R438, K495, K496, R498, and K501, residues that in SNX17 often corresponded to uncharged residues or negatively charged amino acids (Fig 4A and 4C &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Indeed, charge swap site-directed mutagenesis of these residues generated full-length GFP-SNX27 mutants that when expressed and immunoisolated from HEK-293T cells lacked the ability to associate with ESCPE-1; all mutants, however, retained the ability to associate with Retromer (Fig 4D &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). In direct ITC-quantified binding assays human SNX27(K495D), SNX27(R498D), and SNX27(K501D) all displayed no detectable binding to the human SNX1 75-92 peptide, while the SNX27(R496D) mutant displayed a reduce binding affinity of 56.4 muM when compared with the 12.7 muM observed for wild-type SNX27 (Fig 4E &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;).\nFig 4\nSNX1 and SNX2 amino termini bind a unique basic surface of the SNX27-FERM domain.\n(A) Sequence alignment of SNX27 FERM domain with the SNX17 FERM domain. The positively charged residues that constitute a basic surface unique to SNX27 are boxed and labeled in blue. (B) Co-immunoprecipitation of GFP-tagged SNX27 and SNX17. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (C) Homology model of the SNX27 FERM domain [&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;]. The residues that are unique to SNX27 FERM domain and those that may account for the interaction with ESCPE-1 are labeled. (D) Co-immunoprecipitation of GFP-tagged SNX27 WT and a panel of amino acid swap mutants across the FERM domain. The constructs were transiently transfected in HEK293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, and SNX6. The band intensities were measured from n = 3 independent experiments using Odyssey software. The band intensity, normalized to GFP expression, is presented as the average fraction of the WT. The binding of SNX27 point mutants to VPS35, SNX1, SNX2, SNX5, and SNX6 was compared with the SNX27 WT using a 2-way ANOVA and Dunnett test. (E) Binding of SNX175-92 peptide to hSNX27FL proteins by ITC (hSNX27 R498D: dark-blue; hSNX27 K501D: orange; hSNX27 K495D: lilac; hSNX27 K496D: sky-blue; hSNX27 WT: red-orange; ND*: no binding detected). The ITC graphs represent the integrated and normalized data fit with 1:1 ratio binding. The binding affinity (Kd) is given as mean of at least 2 independent experiments. Thermodynamic parameters for the binding are provided in S1 Table. (F) The left panel shows the AlphaFold2 generated model of the SNX27 FERM domain (pink ribbons) bound to the core aDLF sequences (yellow sticks) of SNX1 and SNX2 with identical binding orientations. For clarity, only one SNX27 model is shown with the 8 aDLF peptides overlaid (2 peptide models each of the 4 sequences from SNX1 and SNX2). The 3 subdomains of the SNX27 FERM domain F1, F2, and F3 are indicated. The middle panel shows a close-up view of the modeled SNX1 aDLF core sequence 44EDIFTGA50. The strictly conserved Phe side chain of the aDLF sequences is numbered F0 for reference, and other residues are numbered with respect to this. (G) Co-immunoprecipitation of GFP-tagged SNX27 and a panel of hydrophobic amino acid mutations predicted to mediate the SNX27 FERM-ESCPE1 binding. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps with the normalized enrichment of SNX27 association relative to expression of GFP-tagged protein quantified in the bar chart. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001. The data underlying the graphs shown in the figure can be found in S1 Data. aDLF, acidic-Asp-Leu-Phe; ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; ITC, isothermal titration calorimetry; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\nFinally, we modeled the binding of the aDLF sequences from SNX1 and SNX2 with SNX27 using the AlphaFold2 machine learning algorithm [&lt;&lt;REF:Jumper-2021-15-1&gt;&gt;,&lt;&lt;REF:Mirdita-2021-?-?&gt;&gt;]. In total, we generated 12 models, with 3 models each of SNX27 in the presence of the 4 separate aDLF peptides (Figs 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt; and S3A-S3C). Without any prior inputs apart from the SNX27 and peptide sequences, AlphaFold2 produced very high confidence models of the SNX27 FERM domain. Remarkably, in all 12 models, the aDLF peptides associated specifically with the SNX27 F3 subdomain, near the basic surface identified by mutagenesis (Fig 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Closer inspection showed that in 8 of the models the core D[L/I]F residues adopted essentially identical bound structures, with the hydrophobic [L/I]F side chains inserting into a hydrophobic pocket and interacting with SNX27 I467, F469, Y490, and V500 and the acidic Asp side chain surrounded by the complementary basic SNX27 residues R437, R438, K439, and R498 (Fig 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). To further validate the model, we therefore mutated I467, Y490, and V500 to Lys, all of which disrupted the interaction of SNX27 with ESCPE-1 (Fig 4G &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Together, these models are entirely consistent and strongly support the mutagenesis data for the SNX27-peptide interactions.\n\nSNX27 association with ESCPE-1 couples cargo retrieval with promotion of cell surface recycling\nTo define the functional significance of the association of SNX27 to ESCPE-1, we first transfected our SNX27 knockout HeLa cells line with constructs encoding for either GFP-tagged wild-type SNX27 or the ESCPE-1 binding defective mutant GFP-SNX27(R498D) (Fig 5A &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Analysis of their relative subcellular distributions established that they each associated with EEA1 and SNX6-positive endosomes (Fig 5B &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Previously, we established that by binding to the PDZbm present within the cytoplasmic facing tail of GLUT1, the SNX27-Retromer complex serves to retrieve internalized GLUT1 away from lysosomal mediated degradation ([&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;], and see Fig 1A &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). We therefore performed a series of rescue experiments to quantify the SNX27-dependent retrieval of internalized GLUT1 from lysosomal mediated degradation, using the SNX17-Retriever-mediated retrieval of alpha-5 integrin as a negative control (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;) [&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;]. In the SNX27 knockout HeLa cells, GLUT1 displayed an enhanced rate of degradation following 8 hours of treatment with cycloheximide, data that are consistent with the loss of SNX27 expression leading to a reduced ability for transporter retrieval from the lysosomal degradative fate (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). This phenotype was rescued by reexpression of wild-type GFP-tagged SNX27 (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Interestingly, expression of GFP-SNX27(R498D), a mutant that fails to associate with ESCPE-1 but retains binding to Retromer (Fig 4D and 4E &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;), also fully rescued the retrieval of GLUT1 from lysosomal degradation (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). The coupling of SNX27-Retromer to the ESCPE-1 complex is therefore not required for the retrieval of GLUT1 from lysosomal degradation.\nFig 5\nSNX27:ESCPE-1 interaction is required for endosome-to-plasma membrane recycling of SNX27 cargos.\n(A) Representative western blot showing similar expression levels between transiently expressed GFP-SNX27 and GFP-SNX27(R498D). (B) Representative images of SNX27 KO HeLa cells transiently transfected with GFP-SNX27 or GFP-SNX27(R498D). Moreover, 48 hours after transfection, cells were fixed and immunostained for the endosomal markers EEA1 and SNX6. Magnified views of the white boxes are shown boxed. Cell numbers analyzed for colocalization were 60 GFP-SNX27 and 60 GFP-SNX27 R498D cells across n  =  3 independent experiments. The Pearson coefficient values were compared using a 2-tailed unpaired t test; for EEA1 P = 0.2971, SNX6 P = 0.0506. (C) Degradation assay in parental HeLa cells or SNX27 KO HeLa cells that were transiently transfected with GFP-SNX27 or GFP-SNX27(R498D) mutant. Moreover, 48 hours after transfection, cells were incubated with 10 mug/ml cycloheximide and lysed at different time points as indicated. The band intensities of endogenous GLUT1 and ITGA5 were measured from n = 3 independent experiments using Odyssey software. The levels of GLUT1 and ITGA5 at 8 hours were compared with the corresponding level at 0-hour time point. Analysis was done using a 2-way ANOVA and Sidak test. GLUT1: P = 0.0735 (HeLa), 0.0068 (KO), 0.9997 (+GFP-SNX27), 0.9975 (+GFP-R498D). ITGA5: P = 0.1151 (HeLa), 0.9820 (KO), 0.9999 (+GFP-SNX27), and 0.6899 (+GFP-R498D). (D) Analysis of surface levels of GLUT1 and ITGA5 in parental HeLa cells or SNX27 KO HeLa cells that were transiently transfected with GFP-SNX27 or GFP-SNX27(R498D) mutant. Moreover, 48 hours after transfection, cells were subjected to surface biotinylation followed by streptavidin-based immunoisolation. The immunoisolates were blotted for GLUT1, and the band intensities were measured from n =  4 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. GLUT1: P = 0.0134 (KO versus HeLa), 0.9587 (+GFP-SNX27 versus HeLa), and 0.0163 (+GFP-SNX27 R498D versus HeLa). The immunoisolates were blotted for ITGA5, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. ITGA5: P = 0.5616 (KO versus HeLa), 0.0733 (+GFP-SNX27 versus HeLa), and 0.2354 (+GFP-SNX27 R498D versus HeLa). (E) Representative images of parental Hela cells, SNX27 KO HeLa cells transiently transfected with GFP-SNX27 or GFP-SNX27(R498D). Moreover, 48 hours after transfection, cells were fixed and immunostained for the cargo GLUT1, the endosomal marker VPS35, and the late endosome/lysosome marker LAMP1. A total of 60 cells were quantified per condition in n = 4. The Pearson coefficient values were compared to the values of Parental HeLa using 1-way ANOVA and Dunnett test. The values for LAMP1 colocalization were P &lt; 0.0001 (Parental versus KO), 0.0001 (Parental versus +WT), 0.1084 (Parental versus +R498D), &lt;0.0001 (+WT versus KO), and 0.0692 (+WT versus +R498D). The values for VPS35 colocalization were P &lt; 0.0001 (Parental versus KO), 0.0029 (Parental versus +WT), 0.0001 (Parental versus +R498D), &lt;0.0001 (+WT versus KO), and 0.6346 (+WT versus +R498D). Scale bars, 25 mum (micrographs) and 5 mum (magnified images). Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; KO, knockout; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\nNext, we turned to examining the promotion of GLUT1 recycling back to the cell surface. Here, we utilized restricted cell surface biotinylation to quantify the amount of GLUT1 at the cell surface [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. Compared with control HeLa cells, SNX27 knockout HeLa cells displayed a reduced cell surface abundance of GLUT1, a result of missorting of internalized GLUT1 for lysosomal degradation (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Again, this phenotype was rescued by expression of GFP-SNX27 (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. In contrast, expression of GFP-SNX27(R498D) failed to rescue the cell surface level of GLUT1 (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), consistent with the association of SNX27 with ESCPE-1 being required to promote the cell surface recycling of internalized GLUT1. Importantly, GFP-SNX27(R498D) exhibited an endosomal localization indistinguishable from wild-type SNX27 (Fig 5B &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), consistent with the inability to promote endosome-to-plasma membrane recycling of GLUT1 not arising from a loss of endosomal localization. Furthermore, confocal analysis of the steady-state distribution of GLUT1 in GFP-SNX27(R498D) rescue cells confirmed that GLUT1 was retrieved from the lysosomal degradative fate, as shown by reduced co-localization with the LAMP1-labeled lysosome (Fig 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). However, unlike the rescue with wild-type GFP-SNX27, where GLUT1 was enriched at the cell surface, the GLUT1 in the GFP-SNX27(R498D) rescue displayed an increased level of intracellular puncta and failed to repopulate the cell surface suggestive of the transporter being trapped in an endosomal compartment (Fig 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), the precise nature of which will require further study. Together, these data are consistent with SNX27 orchestrating cargo retrieval from the degradative fate through its PDZ domain-mediated association with cargo and Retromer, while the SNX27 FERM domain mediates association with ESCPE-1 to promote tubular-based plasma membrane recycling of the retrieved cargo.\n\nESCPE-1 association with SNX27 couples cargo retrieval with promotion of cell surface recycling\nTo complete the mechanistic dissection of SNX27 coupling to ESCPE-1, we used a previously characterized double SNX1 and SNX2 knockout HeLa cell line [&lt;&lt;REF:Simonetti-2017-216-3695&gt;&gt;] to generate lentiviral transduced stable lines expressing GFP-SNX1 or GFP-SNX1(D45K,D87K) (Fig 6A &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). When expressed at near to endogenous levels each chimera displayed an indistinguishable association with EEA1 and SNX6-labeled endosomes (Fig 6B &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Consistent with ESCPE-1 not being required for GLUT1 retrieval from lysosomal degradation, the rate of internalized GLUT1 degradation following 8 hours of treatment with cycloheximide was equivalent between parental HeLa cells, the SNX1/SNX2 knockout cells, and the GFP-SNX1 and GFP-SNX1(D45K,D87K) rescue lines (Fig 6C &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). In contrast, the quantification of restricted cell surface biotinylation established that compared to the parental HeLa cells, the SNX1/SNX2 double knockout HeLa cells displayed a reduced cell surface abundance of GLUT1 consistent with a defect in recycling of internalized GLUT1 (Fig 6D &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). While reexpression of GFP-SNX1 rescued the cell surface level of GLUT1, expression of the GFP-SNX1(D45K,D87K) mutant, defective in SNX27 association (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;), failed to rescue the cell surface recycling of GLUT1 (Fig 6D &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Finally, confocal analysis of the steady-state distribution of GLUT1 revealed that when compared to parental cells, the glucose transporter was enriched in Retromer decorated endosomes in the SNX1/SNX2 double knockout HeLa cells (Fig 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;), in agreement with GFP-SNX27DeltaFERM rescue (see Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Reexpression of GFP-SNX1 partially rescued the cell surface delivery of GLUT1 from this compartment, while expression of GFP-SNX1(D45K,D87K) showed an increase in colocalization of GLUT1 with Retromer decorated endosomes (Fig 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Taken together, these data provide an independent validation that the association of ESCPE-1 to the FERM domain of SNX27 is required to promote tubular-based endosomal exit and cell surface recycling of cargo retrieved from lysosomal degradation.\nFig 6\nESCPE-1 interaction with SNX27 is required for endosome-to-plasma membrane recycling of GLUT1.\n(A) Representative western blot showing similar expression levels between stably transfected GFP-SNX1 and GFP-SNX1(KK) in SNX1/2KO cells. SNX1 band intensities were measured from n = 3 and normalized to actin. Analysis was done using a 2-tailed unpaired t test. (B) Representative images of SNX1/2 KO HeLa cell line stably transfected with GFP-SNX1 or GFP-SNX1(KK). Cells were fixed and immunostained for the endosomal markers EEA1 and SNX6. Magnified views of the white boxes are shown boxed. Cell numbers analyzed for colocalization were 50 GFP-SNX1 and 50 GFP-SNX1(KK) cells across n  =  3 independent experiments. The Pearson coefficient values were compared using a 2-tailed unpaired t test; for EEA1 P  = 0.5971, SNX6 P = 0.6459. (C) Degradation assay in parental HeLa cells or SNX1/2 KO HeLa cells that were stably transfected with GFP-SNX1 or GFP-SNX1(KK) mutant. Cells were incubated with 10 mug/ml cycloheximide and lysed at different time points as indicated. The band intensities of endogenous GLUT1 and ITGA5 were measured from n = 3 independent experiments using Odyssey software. The levels of GLUT1 and ITGA5 at 8 hours were compared with the corresponding level at 0-hour time point. Analysis was done using a 2-way ANOVA and Sidak test. GLUT1: P = 0.6796 (HeLa), 0.9768 (KO), 0.8430 (+GFP-SNX1), and 0.2665 (+GFP-KK). ITGA5: P = 0.2247 (HeLa), 0.7892 (KO), 0.9196 (+GFP-SNX1), and 0.9930 (+GFP-KK). (D) Analysis of surface levels of GLUT1 and ITGA5 in parental HeLa cells or SNX1/2 KO HeLa cells that were stably transfected with GFP-SNX1 or GFP-SNX1(KK) mutant. Cells were subjected to surface biotinylation followed by streptavidin-based immunoisolation. The immunoisolates were blotted for GLUT1, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. GLUT1: P = &lt;0.0001 (KO versus HeLa), 0.0878 (+GFP-SNX1 versus HeLa), and &lt;0.0001 (+GFP-KK versus HeLa). The immunoisolates were blotted for ITGA5, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. ITGA5: P = 0.2592 (KO versus HeLa), 0.7185 (+GFP-SNX1 versus HeLa), and 0.0653 (+GFP-KK versus HeLa). (E) Representative images of parental Hela cells, SNX1/2 KO HeLa cells stably transfected with GFP-SNX1 or GFP-SNX1(KK). Cells were fixed and immunostained for the cargo GLUT1, the endosomal marker VPS35, and the late endosome/lysosome marker LAMP1. A total of 50 cells were quantified per condition in n = 3. The Pearson coefficient values were compared to the values of Parental HeLa using 1-way ANOVA and Dunnett test. The values for LAMP1 colocalization were P = 0.0002 (Parental versus KO), 0.7919 (Parental versus +WT), 0.2225 (Parental versus +KK), 0.0028 (+WT versus KO), and 0.6677 (+WT versus +KK). The values for VPS35 colocalization were &lt;0.0001 (Parental versus KO), 0.0020 (Parental versus WT), &lt;0.0001 (Parental versus +KK), &lt;0.0001 (+WT versus KO), and 0.2073 (+WT versus +KK). Scale bars, 25 mum (micrographs) and 5 mum (magnified images). Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; KO, knockout; SNX1, sorting nexin-1; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\n\nEvolutionary assembly of the SNX27:Retromer:ESCPE-1 sorting axis\nWe used phylogenetic analyses to investigate the evolution of the association of SNX27 with Retromer and with the ESCPE-1 complex. The inferred phylogeny of SNX27 suggests the gene was ancestrally present in filozoans (i.e., the group comprising filastereans, choanoflagellates, and metazoans) (S4A Fig). In SNX27, an exposed beta-hairpin in the SNX27 PDZ domain, encoded by residues 67 to 79 in human SNX27, engages a groove in the VPS26A (and VPS26B) subunit of Retromer [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;] (Fig 7A &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;). An equivalent sequence, including key residues Leu-67 and Leu-74 of the beta-hairpin, is present in SNX27 across all choanozoans (this is, the group comprising metazoans and choanoflagellates; Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4A). Key residues in VPS26A required for SNX27 binding include D44 and L154, equivalent residues in VPS26B being D42 and L152 [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. The inferred phylogeny of VPS26A and VPS26B revealed that VPS26B is ancestrally present across eukaryotes, with a duplication of VPS26B in vertebrates leading to the emergence of VPS26A (Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4B). Interestingly, within VPS26B, the D44 residue is present ancestrally in filozoans, while the L154 residue evolved in choanozoans (Fig 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;). Together, these analyses suggest that the SNX27-Retromer association first appeared in the last common ancestor of choanoflagellates and metazoans.\nFig 7\nMolecular evolution of the SNX27-Retromer and SNX27-ESCPE-1 interactions.\n(A) Alphafold2 structural prediction for FL human SNX27 [&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;]. The residues of the exposed beta-hairpin loop in the SNX27 PDZ domain (67 to 79) that account for the interaction with Retromer are highlighted in cyan. Those residues of the FERM domain that account for the interaction with ESCPE-1 (positively charged surface) are highlighted in pink. Our data support a model where the interaction between SNX27 and Retromer is required for the retrieval of PDZ cargos away from the degradative fate. Subsequently, the interaction between SNX27 and ESCPE-1 is required for the export and endosome-to-plasma membrane recycling of PDZ cargos. (B) A cartoon representation incorporating the results of multiple phylogenetic analyses, including VPS5/SNX1/SNX2, SNX27, and VPS26A/VPS26B (see S4A-S4C Fig for detailed phylogenies). The gene families that contain VPS5/SNX1/SNX2 and VPS26A/VPS26B are present across eukaryotes: SNX1 and SNX2 and VPS26A and VPS26B, arose by gene duplication in the common ancestor of vertebrates. SNX27 is found in metazoans, choanoflagellates and the filasterean Capsaspora, but not in more distantly related opisthokonts such as Ichthyosporea or fungi; it is therefore likely arose in the filazoan common ancestor. The residue D44 (VPS26B, Homo sapiens) in VPS26 is found across Filazoa, laying the foundations for the binding interaction between VPS26 and SNX27. L154 (VPS26B, H. sapiens) likely arose in the ancestral choanozoan along with the PDZ loop and key interacting residues (67 to 97 L67, L74, SNX27, H. sapiens), suggesting that the SNX27-Retromer interaction was already present in the common ancestor of metazoans and choanoflagellates. Key residues of the FERM domain: R437, R498, and K501 (SNX27, H. sapiens) are recognizable in the ancestral metazoan. The first aDLF motif (SNX1) was acquired in the common ancestor of cnidarians and bilaterians, along with the residues K495 and K496, which make up part of the FERM domain in SNX27, potentially suggesting that SNX27:ESCPE-1 binding has evolved by this time, if not earlier during metazoan evolution. Interestingly, the first aDLF motif does not appear to be present in the placazoan or ctenophoran sequences, (although the R437, R498, and K501 residues are present), which might reflect secondary loss of the motif in these groups. The second aDLF motif is only present in vertebrates.\nThe association between SNX27 and ESCPE-1 depends on the presence of binding motifs in SNX1 and SNX2 and the FERM domain of SNX27. SNX1 is present across eukaryotes; a gene duplication giving rise to SNX1 and SNX2 from a single ancestral protein occurred in the common ancestor of vertebrates (Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4C). Several motifs required in the FERM domain of H. sapiens SNX27 for binding to ESCPE-1 appear to have evolved in the common ancestor of metazoans or early during their radiation (Fig 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;); many of the key residues (R437, R498, and K501) were in place in the last metazoan common ancestor or prior to the divergence of cnidarians and bilaterians (K495 and K496). For SNX1, the first aDLF motif was likely already present in the last metazoan common ancestor. It is present in sequences from bilaterians and Porifera (sponges) but is absent in cnidarians, placazoans, and ctenophorans, which might represent secondary losses of the motif. As for the second aDLF motif, this is present in all vertebrates. This analysis therefore suggests that the SNX27:ESCPE-1 interaction evolved after the SNX27-Retromer link during early metazoan evolution.\n\nDiscussion\nHere, in combining biochemical, structural, and cellular analyses, we have established that aDLF motifs in the redundant SNX1 and SNX2 subunits of ESCPE-1 directly associate with the FERM domain of SNX27 within the SNX27-Retromer assembly. Studies of endosomal cargo sorting show that this association is necessary to promote the ESCPE-1-dependent tubulovesicular cell surface recycling of internalized transmembrane proteins. Our data suggest that ESCPE-1 mediated recycling occurs subsequently or in conjunction with SNX27-Retromer-dependent retrieval from lysosomal degradation. Building on previous studies [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], these data collectively identify the SNX27:Retromer:ESCPE-1 assembly as a dynamic endosomal coat complex that couples sequence-dependent cargo recognition with membrane remodeling to facilitate the formation of cargo-enriched transport carriers for cargo recycling. In addition, we have begun to describe the functional diversification of Retromer as it has evolved from its tight association with the SNX-BAR homologues Vps5 and Vps17, as originally identified in yeast [&lt;&lt;REF:Seaman-1998-142-665&gt;&gt;], to a more transient engagement of ESCPE-1 in metazoans and their relatives mediated at least in part through the SNX27 cargo adaptor.\nAt present, we propose the following model to describe the functional connectivity of the SNX27:Retromer:ESCPE-1 coat complex within the context of cargo retrieval and recycling. Newly endocytosed cargo containing a functional SNX27 PDZbm are recognized by SNX27, associated with the PI(3)P-enriched early endosome, and these interactions together can stabilize the endosomal residency of SNX27 [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;]. As the endosome matures, SNX27 binds to Retromer through direct association of VPS26A (or VPS26B) to the SNX27 PDZ domain. These interactions are mutually stabilizing, thereby locking down cargo association [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;,&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;]: This assembly is essential for cargo retrieval from the lysosomal degradative fate [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. Retromer dimerization and the formation of Retromer arches [&lt;&lt;REF:Kendall-2020-28-393&gt;&gt;,&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;,&lt;&lt;REF:Kovtun-2018-561-561&gt;&gt;] may aid SNX27 and cargo clustering, thereby forming the retrieval subdomain and initiating weak membrane curvature [&lt;&lt;REF:Simunovic-2015-25-780&gt;&gt;]. These events are supported through the Retromer-mediated recruitment of the WASH complex and the localized production of branched filamentous actin [&lt;&lt;REF:Harbour-2012-442-209&gt;&gt;,&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;]. An additional accessory protein, ANKRD50, may serve to stabilize the formation of the SNX27:Retromer:WASH assembly [&lt;&lt;REF:Kvainickas-2017-130-382&gt;&gt;]. To this emerging \"bud,\" ESCPE-1 is recruited through recognition of 3-phosphoinositides and membrane curvature [&lt;&lt;REF:Carlton-2004-14-1791&gt;&gt;,&lt;&lt;REF:Cozier-2002-277-4873048736&gt;&gt;,&lt;&lt;REF:Chandra-2019-10-1528&gt;&gt;] and the direct binding to SNX27 described in the present study. The timed increase in ESCPE-1 residency and localized concentration provides the handover mechanism to ensure that the captured and enriched cargo enter the forming tubular profile and the ensuing transport carrier. Continued localized WASH-mediated actin polymerization and the recruitment of motor proteins facilitates tubule maturation and fission [&lt;&lt;REF:Wassmer-2009-17-110&gt;&gt;-&lt;&lt;REF:Freeman-2014-127-2053&gt;&gt;] (Fig 8 &lt;&lt;FIG:pbio.3001601.g008&gt;&gt;). Additional cargo entry may be achieved through direct recognition by ESCPE-1 and yet to be described mechanisms [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;]. As with the assembly of other coat complexes [&lt;&lt;REF:Schmid-2007-448-883&gt;&gt;], association of the SNX27:Retromer:ESCPE-1 coat is driven by avidity through a series of low affinity interactions that evoke the consideration of checkpoints for monitoring the fidelity of pathway progression. It is important to state that this model is certainly an oversimplification and many key questions remain unanswered. Not least whether such a coat complex can be visualized through membrane associated cryo-electron tomography of reconstituted SNX27:Retromer:ESCPE-1 perhaps in the presence of ANKRD50.\nFig 8\nA 2-step model for the endosomal sorting of SNX27 cargos.\n(A) Schematics of the endosomal sorting of PDZ cargos. Cargos possessing a PDZbm that enter the endosomal system are recognized by SNX27 via its PDZ domain. (B) SNX27 binding to Retromer aids the retrieval of cargos from the lysosomal degradative path. (C) Once cargos have been retrieved, the SNX27 FERM engages the amino terminus of SNX1 and SNX2 mediating the inclusion of SNX27 cargos into ESCPE-1 tubulovesicular carriers. (D) This mechanism results in SNX27 cargos entering carriers that exit endosomes and are transported, directly and/or indirectly, to the cell surface. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; PDZbm, PDZ-binding motif; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27; WASH, Wiskott-Aldrich syndrome protein and SCAR homologue.\nThe importance of the amino-terminal extension of SNX1 and SNX2 has not previously been considered. While we and others [&lt;&lt;REF:Chandra-2022-83-100842&gt;&gt;-&lt;&lt;REF:Yong-2021-118-?&gt;&gt;] have independently established that they constitute primary sites of interaction with SNX27, it is tempting to speculate that they could orchestrate a broader set of protein-protein interactions. Interestingly, these unstructured amino termini may extend to over 30 nm in length and could allow the ESCPE-1 oligomeric assemblies to capture peripheral factors for the fine-tuned regulation of tubular maturation and scission. This would resemble the organization of the endocytic clathrin coat where several accessory proteins protrude from the core clathrin lattice into the cytoplasm to enable a regulated function of the endocytic coat during endocytic vesicle formation [&lt;&lt;REF:Skruzny-2020-16-e9009&gt;&gt;].\nOther than mediating the recruitment of accessory proteins, there is some evidence that these amino-terminal extensions could play a role in the assembly and membrane targeting of SNX-BAR dimers. In fact, besides SNX1 and SNX2, SNX4, SNX7, SNX8, and SNX30, all possess unstructured amino-terminal extensions of different length and of unknown function. The low-complexity region of the SNX8 homologue Mvp1 appears to stabilize soluble Mvp1 in a cytosolic tetrameric form where the PX and BAR domains are masked and unable to target membranes [&lt;&lt;REF:Sun-2020-11-1506&gt;&gt;]. The release of the amino terminus liberates functional dimers of Mvp1 facilitating their membrane association and function [&lt;&lt;REF:Sun-2020-11-1506&gt;&gt;]. It will be interesting to explore whether a similar mechanism also regulates ESCPE-1 targeting to the endosomal network in mammalian cells during endosomal recycling.\nAn important aspect of cargo sorting by SNX27 is regulation by posttranslational modification. Phosphorylation of residues within the core carboxyl-terminal PDZbm negatively impairs SNX27 association, but phosphorylation of residues preceding the minimal PDZbm can dramatically enhance affinity for the SNX27 PDZ domain [&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;]. A well-characterized example of cargo tail phosphorylation modulating SNX27 affinity and subsequent endosomal sorting is the beta2AR [&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;,&lt;&lt;REF:Cao-1999-410-286&gt;&gt;]. Furthermore, phosphorylation of SNX27 at Ser-51 alters the conformation of its PDZbm-binding pocket decreasing the affinity for cargo proteins, thereby inhibiting endosomal sorting [&lt;&lt;REF:Mao-2021-220-e202010048&gt;&gt;] and Ser-302 and Ser-304 phosphorylation in VPS26B can modulate SNX27 association [&lt;&lt;REF:Wang-2021-12-4826&gt;&gt;]. Similarly, ESCPE-1 function can be regulated by phosphorylation; in this case, Ser-226 of SNX5 undergoes phosphorylation, which decreases dimerization with SNX1 and SNX2 and reduces endosomal recycling [&lt;&lt;REF:Itai-2018-13-e0207205&gt;&gt;]. Therefore, a fascinating question regards whether the SNX27:ESCPE-1 interaction may be orchestrated by posttranslational modification. We have here shown that a series of acidic residues upstream of the DLF motif in the SNX1 and SNX2 amino termini contribute to the binding to SNX27 (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Interestingly, several phosphorylation sites in the aDLF sequences of SNX1 have been annotated, including Ser-39, Ser-41, and Ser-72, as well as Thr-72 [&lt;&lt;REF:Dephoure-2008-105-10762&gt;&gt;-&lt;&lt;REF:Bian-2014-96-253&gt;&gt;]. It is tempting to speculate that multiple phosphorylations of the ESCPE-1 amino termini may regulate its association with SNX27-Retromer and, in turn, dictate the timing and kinetics of the endosomal recycling route.\nFinally, a fascinating question relates to the molecular evolution of Retromer [&lt;&lt;REF:Koumandou-2011-124-1496&gt;&gt;]. In yeast, Retromer can form a stable pentameric assembly of 2 subcomplexes, the so called \"cargo-selective\" Vps26:Vps35p:Vps29p heterotrimer and the membrane remodeling Vps5p and Vps17p heterodimer [&lt;&lt;REF:Seaman-1998-142-665&gt;&gt;], the latter being considered analogous to ESCPE-1. Two functional processes, sequence-dependent cargo recognition and the biogenesis of tubulovesicular transport carriers, are therefore brought together in a single pentameric assembly [&lt;&lt;REF:Suzuki-2019-218-2876&gt;&gt;]. In metazoans, the VPS26:VPS35:VPS29 Retromer and ESCPE-1 appear to no longer form a stable pentameric assembly [&lt;&lt;REF:Norwood-2011-12-56&gt;&gt;]; rather, they have diverged into 2 separate complexes and in so doing have expanded their functional roles [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;,&lt;&lt;REF:Simonetti-2017-216-3695&gt;&gt;,&lt;&lt;REF:Norwood-2011-12-56&gt;&gt;,&lt;&lt;REF:Kvainickas-2017-216-3677&gt;&gt;]. Our phylogenetic analysis suggests that the SNX27:Retromer:ESCPE-1 assembly observed in metazoans evolved in a stepwise manner, with the SNX27-Retromer interaction evolving prior to the origin of metazoans, in the common ancestor of metazoans and choanoflagellates--the group of unicellular and colonical flagellates that are the closest living relatives of animals. The SNX27:ESCPE-1 interaction evolved later, during early metazoan evolution and likely prior to the divergence of cnidarians and bilaterians. Thus, within the SNX27:Retromer:ESCPE-1 pathway, the retrieval of cargo from the degradative fate appears to have evolved prior to the direct coupling to ESCPE-1 for promotion of tubulovesicular-based recycling to the cell surface. The divergence of the ancestral pentameric Retromer coat complex therefore appears to have occurred to accommodate the expanding complexity of the endomembrane system in metazoans [&lt;&lt;REF:Rout-2017-86-637&gt;&gt;]. The ability of metazoan Retromer to associate with SNX27 and the ability of ESCPE-1 to directly bind cargo through a helix-loop-helix region [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;], features not observed in yeast, have greatly expanded the capacity for sequence-dependent endosomal retrieval and recycling in metazoans. Indeed, it is tempting to speculate that the increased plasticity of the SNX27:Retromer:ESCPE-1 axis, and the existence of additional Retromer-independent pathways [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;], has allowed the metazoan endosomal network to accommodate the evolving diversity and number of cargos undergoing sequence-dependent sorting, leading to an expansion in the number of recycling itineraries that cargos can engage to interface with distinct membrane compartments. Dissecting how these sorting complexes and pathways are integrated and regulated, for example, by posttranslational modifications, to shape sequence-dependent cargo sorting within the metazoan endosomal network will be a major challenge moving forward.\nOverall, in defining the mechanistic basis of SNX27-Retromer association with ESCPE-1, we have provided new functional insight into the endosomal sorting of hundreds of internalized PDZbm-containing receptors, channels, transporter, enzymes, and adhesion molecules for repopulation of the cell surface [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. This increased molecular understanding provides new insight into the role of sequence-dependent endosome sorting in cellular processes ranging from the establishment of polarity and the uptake of nutrients through to synaptic function and cancer cell metastasis in both health and disease [&lt;&lt;REF:Chandra-2021-9-642378&gt;&gt;].\n\nMaterial and methods\n\nAntibodies\nAntibodies used in this study were mouse monoclonal antibodies GFP (clones 7.1 and 13.1; 11814460001; Roche, Germany) (1:2,000 for WB), LAMP1 (clone H4A3, AB2296838; DSHB, USA) (IF 1:200), SNX1 (clone 51/SNX1; 611482; BD, USA) (1:1,000 for WB, 1:200 for IF), SNX2 (clone 13/SNX2; 5345661; BD) (1:1,000 for WB, 1:200 for IF), SNX6 (clone d-5, 365965; Santa Cruz Biotechnology, USA) (1:1,000 for WB, 1:200 for IF), SNX27 (ab77799, Abcam, UK) (1:1,000 for WB, 1:200 for IF), beta-actin (A1978; Sigma-Aldrich, USA) (1:2,000 for WB), ITGA5 (610633, BD) (1:1,000 for WB), VPS29 (clone d-1, 398874; Santa Cruz Biotechnology) (1:200 for WB); rabbit monoclonal antibodies: EEA1 (C45B10; Cell Signaling Technology, USA) (IF 1:200), VPS35 (EPR11501(B); 157220; Abcam) (1:1,000 for WB), GLUT1 (ab115730, Abcam) (1:1,000 for WB, 1:200 for IF); rabbit polyclonal antibodies: GFP (GTX20290; GeneTex, USA) (WB 1:2,000), VPS26A (23892; Abcam) (1:1,000 for WB), VPS35 (97545; Abcam) (1:1,000 for WB, 1:200 for IF), VPS29 (98929; Abcam) (1:100 for WB); and goat polyclonal antibodies to VPS35 (10099; Abcam) (1:200 for IF).\n\nCell culture and transfection\nHeLa and HEK-293T cell lines were sourced from ATCC. Authentication was from the ATCC. Cells were grown in DMEM (Sigma-Aldrich) supplemented with 10% (v/v) FCS (Sigma-Aldrich) and penicillin/streptomycin (Gibco, USA) and grown under standard conditions. FuGENE HD (Promega, USA) was used for transient transfection of DNA according to the manufacturer's instructions. Cycloheximide was used to prevent up-regulation of protein synthesis. In this case, cycloheximide (C7698; Sigma-Aldrich) at 10 mug/ml was added to the cells 48 hours after FuGENE transfection for the indicated time points. For the generation of CRISPR-Cas9 SNX27 knockout HeLa cells, HeLa cells were transfected with pX330 plasmid coding for the gRNA against the gene of interest together with a puromycin resistance-expressing plasmid. Cells were then subjected to puromycin selection for 24 hours. The pooled population of cells was subjected to lysis and western blotting to confirm the knockout. The VPS35 knockout clonal cell line used in this study was characterized previously [&lt;&lt;REF:Kvainickas-2017-216-3677&gt;&gt;]. For GFP-based immunoprecipitations, HEK293T cells were transfected with GFP constructs using polyethylenimine (Sigma-Aldrich) and expression was allowed for 48 hours.\n\nImmunoprecipitation and quantitative western blot analysis\nFor western blotting, cells were lysed in PBS with 1% (v/v) Triton X-100 and protease inhibitor cocktail. The protein concentration was determined with a BCA assay kit (Thermo Fisher Scientific, USA), and equal amounts were resolved on NuPAGE 4% to 12% precast gels (Invitrogen, USA). Blotting was performed onto polyvinylidene fluoride membranes (Immobilon-FL; EMD Millipore, USA) followed by detection using the Odyssey infrared scanning system (LI-COR Biosciences, USA). For GFP-based immunoprecipitations, HEK-293T cells were lysed 48 hours after transfection in immunoprecipitation buffer (50 mM Tris-HCl, 0.5% (v/v) NP-40, and Roche protease inhibitor cocktail) and subjected to GFP trap (ChromoTek, Germany). Immunoblotting was performed using standard procedures. Detection was performed on an Odyssey infrared scanning system (LI-COR Biosciences) using fluorescently labeled secondary antibodies. In using the Odyssey, we routinely performed western blot analysis where a single blot is simultaneously probed with antibodies against 2 proteins of interest (distinct antibody species) followed by visualization with the corresponding secondary antibodies conjugated to distinct spectral dyes.\n\nBiotinylation of cell surface proteins\nFor surface biotinylation experiments, fresh Sulfo-NHS-SS Biotin (Thermo Fisher Scientific, #21217) was dissolved in ice-cold PBS at pH 7.8 at a final concentration of 0.2 mg/ml. Cells were washed twice in ice-cold PBS and placed on ice to slow down the endocytic pathway. Next, cells were incubated with the biotinylation reagent for 30 minutes at 4degreesC followed by incubation in TBS for 10 minutes to quench the unbound biotin. The cells were then lysed in lysis buffer and subjected to Streptavidin beads-based affinity isolation (GE Healthcare, USA).\n\nImmunofluorescence staining\nCells were fixed in 4% (v/v) PFA for 20 minutes and washed 3 times in PBS and permeabilized with 0.1% (v/v) Triton X-100. For LAMP1 staining, cells were instead permeabilized by submersion in liquid nitrogen for 20 seconds. Fixed cells were blocked in 1% (w/v) BSA and incubated in primary antibody and respective secondary antibody (Alexa Fluor; Thermo Fisher Scientific) in 1% (w/v) BSA.\n\nImage acquisition and image analysis\nMicroscopy images were collected with a confocal laser-scanning microscope (SP5 AOBS; Leica Microsystems, Germany) attached to an inverted epifluorescence microscope (DMI6000; Thermo Fisher Scientific). A 63x 1.4 NA oil immersion objective (Plan Apochromat BL; Leica Biosystems, Germany) and the standard SP5 system acquisition software and detector were used. Images were captured at room temperature as z stacks with photomultiplier tube detectors with a photocathode made of gallium-arsenide-phosphide (Leica Microsystems) for collecting light emission. Images were captured using Application Suite AF software (version 2.7.3.9723; Leica Microsystems) and then analyzed with the Volocity 6.3 software (PerkinElmer, USA).\n\nStatistics and reproducibility\nAll quantified western blot are the mean of at least 3 independent experiments. Statistical analyses were performed using Prism 7 (GraphPad Software, USA). Graphs represent means and SEM For all statistical tests, P &lt; 0.05 was considered significant and is indicated by asterisks.\n\nMolecular biology and cloning for recombinant protein production\nFor bacterial expression, full-length mouse SNX1 with an amino-terminal His-tag (mSNX1FL hereafter) was cloned into the pMW172Kan vector [&lt;&lt;REF:Collins-2008-9-366&gt;&gt;], and the truncated mSNX1 construct containing residues 1 to 139 was cloned into pGEX4T-2 vector for expression as a thrombin-cleavable amino-terminal GST fusion protein. DNA encoding full-length mouse SNX27 (mSNX27FL hereafter) was inserted into the pMCSG7 vector [&lt;&lt;REF:Eschenfeldt-2009-498-105&gt;&gt;] by the ligation-independent cloning (LIC) approach. Human SNX27 constructs, hSNX27FL, hSNX27FERM, and hSNX27FL single-site mutants R437D, K495D, K496D, R498D, and K501D were cloned into the pET-28a vector by GenScript and codon optimized for bacterial protein expression. An additional TEV cleavage site was inserted between the amino-terminal His-tag and hSNX27 coding sequences.\n\nProtein expression and purification\nThe DNA constructs of mSNX27 and mSNX1 were expressed using Escherichia coli BL21 CodonPlus (DE3) competent cells, while the hSNX27 constructs were expressed using E. coli BL21 (DE3) competent cells. The cells were grown in Ultra Yield flasks at 37degreesC, 180 rpm until the cell density reached OD600 approximately 0.8. Protein expression was then induced with 1 mM IPTG and cells incubated for 16 to 18 hours at 18degreesC. For SNX27, 0.5 mM IPTG was added for induction to get optimal protein expression yield. To collect cell pellets, the cells were harvested by 10-minute centrifugation (JLA 8.1 rotor) at 6,000 rpm, 4degreesC.\nFor SNX1 purification, the cell pellet was resuspended in lysis buffer containing 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-mercaptoethanol (beta-ME), 1 mM benzamidine, and 10 mug/mL DNase I dissolved in PBS containing 10% glycerol. For SNX27 proteins, the cell pellet was resuspended in lysis buffer containing 20 mM Tris-HCl pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM beta-ME, 1% Triton X-100, 1 mM benzamidine, and 10 mug/mL DNase I dissolved in PBS containing 10% glycerol. Cells were lysed by high-pressure homogenization (35 Kpsi). The homogenate was centrifuged for 30 minutes at 4degreesC using a JA 25.50 rotor with 19,000 rpm. The supernatant of His-tagged protein homogenates were loaded onto preequilibrated TALON metal affinity resin (Clontech, USA) or Ni-nitrilotriacetic acid (NTA) resin, while the supernatant of GST-fusion protein homogenates were loaded onto preequilibrated glutathione Sepharose beads (GE Healthcare) and incubated on an orbital rotator for an hour in cold room.\nAffinity matrices with bound fusion proteins were loaded into empty glass chromatography columns (Bio-Rad, USA) washed with 100 ml of washing buffer containing either 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-ME or 20 mM Tris-HCl pH 8.0, 200 mM NaCl, 5% glycerol, 1 mM beta-ME. For His-tagged protein purification, 10 mM and 300 mM imidazole was added to wash and elution buffers, respectively. To collect the GST-tagged mSNX1 constructs, 50 mM reduced glutathione in elution buffer was applied for elution. cOmplete EDTA-free protease inhibitor tablets were added to the eluted SNX1 samples to reduce proteolytic degradation of their disordered amino-terminal region.\nThe fractions containing eluted proteins were concentrated to approximately 5 ml by Amicon cc ultrafiltration devices (10 kDa or 30 kDa molecular mass cutoff) (Millipore) and centrifuged in a bench centrifuge for 10 minutes at maximum speed to remove any precipitated material before being subjected to size-exclusion chromatography (SEC). For SEC, the HiLoad 16/600 Superdex 200 (GE Healthcare) column was used with proteins eluted in SEC buffer containing 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-ME or 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5% glycerol, and 1 mM beta-ME. To check the purify and integrity of the purified fractions, sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) was applied using Bolt 12% Bis-Tris gel with 1x MES running buffer at 190V for 25 minutes. After the SDS-PAGE, gels were stained with Coomassie Brilliant Blue R250 solution for visualization. For hSNX27FERM, an additional anion exchange chromatography (IEX) was applied using the Mono Q anion exchange chromatography column preequilibrated with IEX buffer A: 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5% glycerol, 1 mM beta-ME. A linear salt gradient from 0% to 40% IEX buffer B (20 mM Tris-HCl pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM beta-ME) was conducted. Proteins usually eluted in buffer containing 200 mM NaCl, and fractions collected were analyzed by SDS-PAGE.\n\nIsothermal titration calorimetry\nHuman SNX1 and SNX2 peptide sequences were synthesized by Genscript (USA). To obtain the peptide stock solutions at neutral pH, 100 mM Tris-HCl, pH 10.6 was used to dissolve the peptide to a final concentration of 8 mM. The peptides were SNX135-51 EAGDSDTEGEDIFTGAA, SNX135-51(D45K) EAGDSDTEGEKIFTGAA, SNX175-92 NGIHEEQDQEPQDLFADA, SNX216-33 TDFEDLEDGEDLFTSTVS, SNX216-33(DLF/SSS) TDFEDLEDGESSSTSTVS, and SNX262-82 TEVVLDDDREDLFAEATEEVS.\nAll microcalorimetry experiments were conducted at 25degreesC using a PEAQ ITC (Malvern, UK) in 50 mM HEPES (pH 7.5), 200 mM NaCl, 5% glycerol, and 1 mM beta-ME (ITC buffer). All proteins were purified by SEC using the same buffer prior to ITC experiments to avoid buffer mismatches. To measure binding of SNX27 to SNX1 proteins, 800 muM of hSNX27FERM was titrated into 20 muM of GST-mSNX11-139 or His-SNX1FL. For the binding of SNX1 and SNX2 peptides to SNX27, peptides were diluted 10-fold to 800 muM in ITC buffer peptides and titrated into 20 muM SNX27FL proteins containing the same 1:10 ratio of peptide buffer (100 mM Tris-HCl, pH 10.6). The competitive ITC experiments were carried out by titrating 800 muM SNX175-92 peptides into 20 muM proteins with 2-fold molar excess SNX216-33 peptide preincubated with SNX27 on ice for an hour. For ITC experiment using SNX175-92 against hSNX27 wild-type and single-site mutants, 600 muM SNX175-92 peptide was titrated into 20 muM hSNX27 proteins.\nThe ITC experiments were conducted with 1 injection of 0.4 muL followed by a series of 12 injections of 3.2 muL each with 180-second intervals and a stirring speed of 850 rpm. The heat exchange of interactions was obtained through integrating the observed peaks and background correction by subtracting the heats of dilution. Data were analyzed with Malvern software package by fitting and normalized data to a single-site binding model, yielding the thermodynamic parameters Kd, DeltaH, DeltaG, and -TDeltaS for all binding experiments. The stoichiometry was refined initially, and if the value was close to 1, then N was set to exactly 1.0 for calculation. All experiments were performed at least twice to guarantee reproducibility of the data.\n\nAlphaFold predictions of peptide binding to SNX27\nTo generate predicted models of SNX1 and SNX2 amino-terminal peptides associating with the SNX27 FERM domain, we used the AlphaFold2 neural network [&lt;&lt;REF:Jumper-2021-15-1&gt;&gt;] implemented within the freely accessible ColabFold pipeline [&lt;&lt;REF:Mirdita-2021-?-?&gt;&gt;]. For each peptide-modeling experiment, the sequence of the human SNX27a FERM domain (residues 269 to 531; Q96L92) was modeled with the 4 separate amino-terminal aDLF sequences from SNX1 and SNX2. The specific peptides we modeled were residues 33 to 56 and 75 to 98 from human SNX1 (Q13596) and residues 14 to 37 and 60 to 83 from human SNX2 (O60749). ColabFold was executed using default settings where multiple sequence alignments were generated with MMseqs2 [&lt;&lt;REF:Mirdita-2019-35-2856&gt;&gt;], and structural relaxation of final peptide geometry was performed with Amber [&lt;&lt;REF:Hornak-2006-65-712&gt;&gt;] to generate 3 models per peptide. Sequence conservation was mapped onto the modeled SNX27 FERM domain structure with Consurf [&lt;&lt;REF:Ashkenazy-2016-44-W344&gt;&gt;]. Structural alignments and images were generated with Pymol (Schrodinger, USA).\n\nPhylogenetic analyses\nHomologous proteins were identified using NCBI protein-protein BLAST (with default parameters) against all non-redundant GenBank CDS Translations+PDB+SwissProt+PIR+PRF excluding environmental samples from WGS projects for a wider range of species (unaligned sequence data with accession numbers available in S4 Fig). For the ctenophore data, we used the BLAST (with default parameters) from the Mnemiopsis Genome Project Portal (MGP) portal [&lt;&lt;REF:Ryan-2013-342-1242592&gt;&gt;-&lt;&lt;REF:Moreland-2020-?-?&gt;&gt;]. Sequences were aligned using MAFFT v7.480 [&lt;&lt;REF:Katoh-2013-30-772&gt;&gt;], L-INS-I mode, and then used to infer initial exploratory trees to identify specific duplications (using IQtree 2.1.4 [&lt;&lt;REF:Minh-2020-37-1530&gt;&gt;] with the LG+F+G model and 1,000 ultrafast bootstraps). After orthologues of each copy of the gene were identified in a range of taxa, we then performed iterative tree inferences to confirm paralogues and to identify species-specific duplication events. Subsequent maximum likelihood phylogenetic analyses were performed using IQtree 2.1.4 with the best fitting model selected using the Bayesian information criterion (BIC), including models which account for site-rate heterogeneity (LG+C10 ...C60) [&lt;&lt;REF:Si Quang-2008-24-2317&gt;&gt;], empirical amino acid frequencies (+F), and accounting for across-site rate variation using either a Gamma distribution (+G) [&lt;&lt;REF:Yang-1994-39-306&gt;&gt;] or the free rates model (+R) [&lt;&lt;REF:Yang-1995-139-993&gt;&gt;,&lt;&lt;REF:Soubrier-2012-29-3345&gt;&gt;] with 10,000 ultrafast bootstrap replicates. The best fitting model according to BIC was LG+C30+F+G for the final SNX1 and SNX2 tree (see S4 Fig) and LG+C20+F+G for SNX27 and VPS26 (alignments and maximum likelihood trees in S4 Fig).\n\nSupporting information\nS1 Fig\nWestern blot showing the levels of GLUT1 in parental HeLa, VPS35 KO, and SNX27KO not transfected or transiently transfected with GFP-SNX27 WT, GFP-SNX27 Delta67-77, and GFP-SNX27 DeltaFERM.\nThe blot is representative of 3 independent assays. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. KO, knockout; SNX27, sorting nexin-27; WT, wild-type.\n(TIF)\nClick here for additional data file.\nS2 Fig\n(A) Co-immunoprecipitation of GFP-tagged extended amino-terminal regions (NT) of SNX1 and SNX2 alongside the GFP-tagged short amino-terminal regions of SNX5 and SNX6 expressed in HEK293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. (B) Co-immunoprecipitation of GFP-tagged SNX2 WT or SNX2 KK (D27K, D72K) expressed in HEK293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation and the immunoprecipitates were blotted for SNX27, SNX6, and GFP. The blot is representative of 4 independent GFP traps. (C) Competitive ITC assay of SNX216-33 and mSNX27FL preincubated with 2-fold molar excess of SNX175-92 peptide showing that binding is blocked by the competing SNX175-92 peptide. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. SNX1, sorting nexin-1; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\n(TIF)\nClick here for additional data file.\nS3 Fig\n(A) SNX27 surface colored according to sequence conservation. The peptides are predicted to bind to a highly conserved pocket in SNX27. (B, C) As described in methods, 12 models of SNX27 were generated in total, consisting of 3 models each of SNX27 in association with the 4 aDLF sequences from human SNX1 and SNX2. B shows the overlay of all 12 models in Calpha trace representation colored by the pLDDT score. The pLDDT is a per-residue confidence score between 0 (lowest confidence) and 100 (highest confidence). The majority of the SNX27 structures show very high pLDDT scores and overlay with very high precision. C shows only the best 8 models where the core aDLF sequences have identical binding conformations. aDLF, acidic-Asp-Leu-Phe; pLDDT, predicted Local Distance Difference Test; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27.\n(TIF)\nClick here for additional data file.\nS4 Fig\nMaximum likelihood phylogenies of (A) SNX27, (B) VPS26A/B and (C) VPS5/SNX1/SNX2. (A) SNX27 (green) with SNX17 (red) used as an outgroup with representatives across Filozoa. Amino acids aligned with Homo sapiens in positions 437, 495, 496, 498, and 501 are written next to the species name. (B) VPS26A, VPS26B, and VPS26C, with VPS26C in red and VPS26B in green; the duplication event occurring in the ancestral vertebrate is in blue. Interestingly, we also find another independent duplication of VPS26B in plants evidenced by Arabidopsis thaliana and Quercus suber in this tree. (C) The SNX1 sequences across choanozoans, with the duplication in vertebrates in blue. The number of aDLF motifs in the amino terminus is written next to the species name. Phylogenies were inferred under the best-fit model as determined by BIC. Branch support values are percentages based on 10,000 ultrafast bootstraps. aDLF, acidic-Asp-Leu-Phe; BIC, Bayesian information criterion; SNX1, sorting nexin-1; SNX27, sorting nexin-27.\n(TIF)\nClick here for additional data file.\nS1 Table\nSummary of ITC binding parameters.\nITC, isothermal titration calorimetry.\n(PDF)\nClick here for additional data file.\nS1 Raw Images\nUnprocessed images for blots shown in the paper.\n(PDF)\nClick here for additional data file.\nS1 Data\nNumerical data for Figs 1B, 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;, 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;, 3A, 3C, 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;, 4D, 4G &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;, 5A, 5B, 5C, 5D, 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;, 6A, 6B, 6C, 6D and 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt; and S1A and S2B.\n(XLSX)\nClick here for additional data file.",
  "methodology": "Cryo-Electron Tomography (CryoET)",
  "method_goal": "Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state.",
  "questions_output_specification": "1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:
[outputs]
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nA central role of the endosomal-lysosomal network is to determine the sorting of integral proteins and their associated proteins and lipids, collectively termed \"cargos,\" between 2 fates, either degradation within lysosomes or retrieval from this fate for the promotion of recycling to a variety of organelles that include the cell surface [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;]. Efficient and selective endosomal-lysosomal cargo sorting is essential for the function of all eukaryotic cells with increasing evidence pointing to defects in endosomal retrieval and recycling being at the heart of a wide array of human diseases, most notably those associated with neurodegeneration [&lt;&lt;REF:Small-2020-12-?&gt;&gt;]. Establishing the molecular mechanisms and functional significance of endosomal cargo retrieval and recycling is therefore fundamental to understand eukaryotic cell biology.\nTwo general concepts are considered to account for the mechanism of endosomal cargo retrieval and recycling. Sequence-independent sorting posits that sorting can proceed without the need for any sequence information within the intracellular cytoplasmic domains of integral membrane cargos [&lt;&lt;REF:Maxfield-2004-5-121&gt;&gt;]. On the other hand, sequence-dependent sorting is controlled by specific signals in the cytoplasmic domain of cargos. These signals are typically short linear peptide motifs that direct the sorting process through coupling to ancient and evolutionary conserved coat complexes [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;,&lt;&lt;REF:McNally-2018-28-807&gt;&gt;,&lt;&lt;REF:Chen-2019-20-465&gt;&gt;]. In coupling sorting motif recognition with membrane remodeling, these coats coordinate the process of cargo retrieval with the formation of cargo-enriched transport carriers for onward recycling. In mammals, these coats include sorting nexin-3 (SNX3)-Retromer [&lt;&lt;REF:Strochlic-2007-177-115&gt;&gt;-&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;], SNX27-Retromer [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], SNX17-Retriever [&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;], the CCC (CCDC22/CCDC93/COMMD) [&lt;&lt;REF:Phillips-Krawczak-2015-26-91&gt;&gt;,&lt;&lt;REF:Bartuzi-2016-7-10961&gt;&gt;], and endosomal SNX-BAR sorting complex for promoting exit 1 (ESCPE-1) complexes [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;] and various accessory proteins including the actin polymerizing Wiskott-Aldrich syndrome protein and SCAR homologue (WASH) complex [&lt;&lt;REF:Gomez-2009-17-699&gt;&gt;,&lt;&lt;REF:Derivery-2009-17-712&gt;&gt;].\nSorting nexin-27 (SNX27) is an endosome-associated cargo adaptor composed of a central PX (phox homology) domain flanked by an amino-terminal PDZ (PSD95, disks large, and zona occludens) domain and a carboxyl-terminal FERM (4.1/ezrin/radixin/moesin) domain [&lt;&lt;REF:Kaiji-2003-8-434&gt;&gt;,&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;]. The PDZ domain recognizes a specific sorting motif, termed the PDZ-binding motif (PDZbm), present at the very carboxyl termini of over 400 cargo proteins including receptors, transporters, channels, enzymes, and adhesion molecules [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;-&lt;&lt;REF:McMillan-2021-10-e59432&gt;&gt;]. In recognizing these internalized cargos, SNX27 regulates their retrieval from lysosomal degradation and promotes their recycling back to the plasma membrane [&lt;&lt;REF:Lauffer-2010-190-565&gt;&gt;-&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;].\nTo facilitate the retrieval process, the SNX27 PDZ domain associates with Retromer, a stable heterotrimer of VPS26 (A and B isoforms expressed in humans [&lt;&lt;REF:Kerr-2005-6-991&gt;&gt;]), VPS35 and VPS29, to form the SNX27-Retromer [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. Retromer itself assembles to form dimeric arches that may cluster SNX27 captured cargos [&lt;&lt;REF:Kendall-2020-28-393&gt;&gt;,&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;,&lt;&lt;REF:Kovtun-2018-561-561&gt;&gt;] and associates with the WASH complex to allow the localized Arp2/3-dependent polymerization of branched filamentous actin [&lt;&lt;REF:Harbour-2012-442-209&gt;&gt;,&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;]. In a poorly understood process, these assemblies and actin dynamics are proposed to generate a retrieval subdomain on the endosomal membrane that is enriched in cargo destined for recycling [&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;-&lt;&lt;REF:Lee-2016-7-10939&gt;&gt;]. It is from these subdomains that further membrane remodeling ensues to drive the biogenesis of cargo-enriched tubular profiles and tubulovesicular carriers that physically transport cargo back to the cell surface. Importantly, defects in SNX27-Retromer function are associated with the pathoetiology of human disease most notable neurological disorders and neurodegenerative disease [&lt;&lt;REF:Chandra-2021-9-642378&gt;&gt;].\nA membrane tubulating Bin/Amphiphysin/Rvs (BAR) domain-containing protein associated with SNX27-Retromer tubular profiles is sorting nexin-1 (SNX1) [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;]. SNX1, and its functional ortholog SNX2, is part of the ESCPE-1 complex [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;,&lt;&lt;REF:Carlton-2004-14-1791&gt;&gt;], a membrane tubulating complex that in vitro can reconstitute the formation of membrane tubules with diameters similar to those observed in cellulo [&lt;&lt;REF:van Weering-2012-31-4466&gt;&gt;]. To be functional, ESCPE-1 forms a stable heterodimer of either SNX1 or SNX2 associated with either of SNX5 or SNX6, 2 additional BAR domain-containing orthologs [&lt;&lt;REF:Wassmer-2007-120-45&gt;&gt;]. Importantly, the mechanistic basis of how SNX27-Retromer delivers cargo into forming ESCPE-1 tubular profiles and tubulovesicular carriers remains a major unanswered question.\nHere, we establish that a conserved surface in the FERM domain of SNX27 binds directly to acidic-Asp-Leu-Phe (aDLF) sequence motifs present in the disordered amino termini of the SNX1 and SNX2 subunits of ESCPE-1. We show that this interaction is required to handover PDZbm containing cargo proteins retrieved from lysosomal degradation by SNX27-Retromer into ESCPE-1 tubular profiles that promote cargo recycling for subsequent repopulation at the cell surface. Through evolutionary analysis of those protein-protein interactions that assemble the SNX27:Retromer:ESCPE-1 coat complex, we reveal how this sorting axis has evolved to accommodate the increasing complexities of endosomal sorting observed in animals.\n\nResults\n\nThe FERM domain of SNX27 promotes cargo recycling following SNX27-Retromer-mediated retrieval from lysosomal degradation\nPreviously, we have established that the interaction of Retromer with the PDZ domain of SNX27 is a crucial step in preventing the routing of internalized cargo into the lysosomal degradative pathway [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. To identify the region(s) of SNX27 required for promoting subsequent cargo recycling to the plasma membrane, we used CRISPR gene editing to generate SNX27 knockout HeLa cells (Fig 1A &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Compared to control cells, where endogenous GLUT1 was enriched at the cell surface, in the SNX27 knockout cells, GLUT1 displayed a steady-state enrichment with LAMP1-positive lysosomes, a phenotype that was also observed in a VPS35 knockout HeLa cell line (Fig 1B &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Importantly, reexpression of GFP-SNX27 in the SNX27 knockout cells rescued the retrieval of GLUT1 from the lysosomal distribution and promoted its repopulation at the cell surface (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). These data mirror those previously published using RNA interference (RNAi)-mediated SNX27 suppression [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;] and confirm the SNX27 knockout HeLa cell line as a suitable screening platform through which to dissect the features of SNX27 required for promoting GLUT1 endosome-to-plasma membrane recycling.\nFig 1\nThe FERM domain of SNX27 promotes cargo recycling following SNX27-Retromer-mediated lysosomal retrieval.\n(A) Western blot confirming the absence of the proteins of interest in VPS35 KO and SNX27 KO cells. (B) Representative images of parental Hela cells, VPS35 KO HeLa clonal line and SNX27 KO HeLa cells. The steady-state distribution of GLUT1 was analyzed by immunofluorescence staining of fixed cells with GLUT1 and the endosomal and lysosomal markers VPS35 and LAMP1. A total of 70 cells were analyzed in each condition for colocalization between GLUT1 and LAMP1 across n  =  4 independent experiments. The Pearson coefficient values of LAMP1 colocalization were compared to the values of parental HeLa using 1-way ANOVA and Dunnett test: P &lt; 0.0001 (HeLa vs VPS35 KO), &lt; 0.0001 (HeLa versus SNX27 KO). Pearson coefficient values of VPS35 colocalization were compared using unpaired 2-tailed t test: P &lt; 0.0001 (HeLa versus SNX27 KO). (C) Representative images of SNX27 KO HeLa cells transiently transfected with GFP-SNX27 WT, GFP-SNX27 Delta67-77 and GFP-SNX27 DeltaFERM. Moreover, 48 hours after transfection, cells were fixed and immunostained. The steady-state distribution of GLUT1 was analyzed by immunofluorescence staining of fixed cells with GLUT1 and the endosomal and lysosomal markers VPS35 and LAMP1. A total of 70 cells were analyzed in each condition for colocalization between GLUT1 and LAMP1, and GLUT1 and VPS35 across n  =  4 independent experiments. The Pearson coefficient values were compared to the values of SNX27 KO cells rescued with GFP-SNX27 WT using 1-way ANOVA and Dunnett test. LAMP1 colocalization values were P &lt; 0.0001 (+WT versus +Delta67-77), 0.3559 (+WT versus +DeltaFERM). VPS35 colocalization values were P &lt; 0.0001 (+WT versus Delta67-77), 0.001 (+WT versus +DeltaFERM). Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. Scale bars, 25 mum (micrographs) and 5 mum (magnified images). The data underlying the graphs shown in the figure can be found in S1 Data. KO, knockout; SNX27, sorting nexin-27; WT, wild-type.\nIn contrast to wild-type SNX27, expression of a SNX27 mutant lacking the loop region essential for Retromer association (GFP-SNX27Delta67-77) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;] failed to rescue retrieval from lysosomal colocalization and failed to promote GLUT1 cell surface recycling (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). These data further confirm that the association of SNX27 with Retromer is a crucial step in coupling sequence-dependent cargo recognition with cargo retrieval from the lysosomal degradative fate [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. Consistent with this, the expression of a SNX27 deletion mutant lacking the FERM domain (GFP-SNX27DeltaFERM), a mutant that retained the association of SNX27 with Retromer, retained the ability to retrieve GLUT1 from lysosomes, as defined by a decreased colocalization between GLUT1 and LAMP1 (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). However, the GFP-SNX27DeltaFERM mutant failed to promote GLUT1 recycling to the cell surface, rather GLUT1 was enriched in a Retromer-positive endosomal compartment (Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). The overall levels of GLUT1 were not decreased in the KO and rescue conditions compared to parental cells (S1 Fig). Together, these data are consistent with a model in which the coupling of SNX27 to Retromer is necessary for retrieval from the lysosomal degradative fate, while the SNX27 FERM domain is required to promote the subsequent endosome-to-cell surface recycling of GLUT1.\n\nSNX1 and SNX2 couple ESCPE-1 to the FERM domain of SNX27\nESCPE-1 regulates tubular-based endosomal recycling of several cargos, many of which are also dependent on SNX27-Retromer [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;]. ESCPE-1 is a heterodimeric assembly of the functionally redundant SNX1 or SNX2 associated with the functionally redundant SNX5 or SNX6 (Fig 2A &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;) [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;], while SNX27-Retromer is an assembly of SNX27 with the VPS26A/B:VPS35:VPS29 heterotrimeric Retromer complex (Fig 2B &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. To define a possible connection between SNX27 and ESCPE-1, we first performed a series of GFP-nanotrap immunoisolation of SNX27 and various SNX27 mutants expressed in HEK-293T cells and probed for the presence of associated endogenous ESCPE-1. Robust association was observed between full-length GFP-SNX27 and all components of ESCPE-1 (Fig 2C-2E &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). This association was independent of the SNX27 PDZ domain as binding was retained in the PDZ deletion mutant GFP-SNX27DeltaPDZ and was not observed when just the PDZ domain was expressed in HEK-293T cells (Fig 2C and 2D &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Retromer association with SNX27 was not required for ESCPE-1 binding as GFP-SNX27Delta67-77 and GFP-SNX27(L67A/L77A), 2 mutants that fail to bind to Retromer [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], each retained association with ESCPE-1 (Fig 2E &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). However, ESCPE-1 association was lost upon deletion of the SNX27 FERM domain (SNX27DeltaFERM) or upon switching the FERM domain of SNX27 for the corresponding FERM domain from SNX17 (Fig 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Moreover, just the isolated FERM domain of SNX27, when expressed in HEK-293T cells, was able to associate with ESCPE-1 (Fig 2D &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Finally, in the reverse immunoisolations, GFP-tagged SNX1, SNX2, SNX5, or SNX6 revealed that only SNX1 and SNX2 displayed a pronounced association with endogenous SNX27 when expressed in HEK-293T cells (Fig 2F &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Moreover, this association did not require the presence of the SNX1 or SNX2 BAR domains, since deletion of these domains failed to affect binding to SNX27 (Fig 2G &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). Together, these data are consistent with the FERM domain of SNX27 associating with the SNX1 and SNX2 components of ESCPE-1.\nFig 2\nThe FERM domain of SNX27 interacts with SNX1 and SNX2.\n(A) Schematics of ESCPE-1 assembly consisting of heterodimers of the SNX-BAR proteins SNX1 or SXN2 with SNX5 or SNX6. These SNXs have BAR and PX domains responsible of dimer formation and membrane targeting, respectively. SNX1 and SNX2 also have an extended amino-terminal unstructured extension (NT). (B) Schematics of the SNX27-Retromer assembly. Retromer is a stable heterotrimer of VPS35, VPS26, and VPS29. Retromer directly interacts with SNX27 through a beta-hairpin insertion within the PDZ domain (residues 67 to 79) that engages a cleft of the VPS26 protein between the amino-terminal and carboxyl-terminal subdomains. (C) Co-immunoprecipitation of GFP-tagged SNX27 FL, SNX27 DeltaPDZ, SNX27 DeltaFERM and a chimera consisting of the SNX27 gene where its FERM domain was swapped for that of SNX17. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensities of VPS35, VPS26A, VPS29, SNX1, and GFP were measured from n =  3 independent experiments using Odyssey software. The band intensities, normalized to GFP expression, are presented as the average fraction of the FL. (D) Co-immunoprecipitation of GFP-tagged SNX27 FL, isolated SNX27 PDZ, PX, and FERM domains expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (E) Co-immunoprecipitation of GFP-tagged SNX27 FL and the SNX27 mutants Delta67-77 and L67A-L74A expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (F) Co-immunoprecipitation of GFP-tagged SNX1, SNX2, SNX5, and SNX6 expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. (G) Co-immunoprecipitation of GFP-tagged SNX1, SNX2, and SNX5 lacking the BAR domain (DeltaBAR) expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; FL, full-length; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27.\n\nThe amino termini of SNX1 and SNX2 present tandem acidic-Asp-Leu-Phe motifs that associate with the SNX27 FERM domain\nCompared to SNX5 and SNX6, human SNX1 and SNX2 each contain amino-terminal extensions of 142 and 139 amino acids, respectively, which are predicted to be unstructured. Expression in HEK-293T cells of GFP-SNX1(1-279), a construct encoding for the amino-terminal extension plus the SNX1 PX domain, and GFP-SNX1(1-138), a construct encoding for just the amino-terminal extension of SNX1, established that the amino-terminal extension of SNX1 is sufficient for association with endogenous SNX27 (Fig 3A &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Likewise, the amino terminus of SNX2 interacted with SNX27, while those of SNX5 and SNX6 failed to do so (S2A Fig). To establish the direct nature of the association, we recombinantly expressed and purified full-length mouse SNX1 and the isolated FERM domain of human SNX27; mouse and human SNX1 are 94.2% identical. Isothermal titration calorimetry (ITC) established that SNX1 bound directly to the FERM domain of SNX27 with an affinity (Kd) of 41.0 muM (Fig 3B &lt;&lt;FIG:pbio.3001601.g003&gt;&gt; and S1 Table). A similar binding affinity (39.1 muM) was observed with a recombinant protein corresponding to the amino-terminal extension of mouse SNX1 (85.9% sequence identity with human SNX1) (Fig 3B &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). To map down the features of SNX1 and SNX2 required for SNX27 binding, we performed a deletion mutagenesis screen of the amino-terminal extension of human SNX1 (Fig 3C &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Walking upstream from a minimal binding region encoded by residues 1 to 138 established that 2 regions were required for maximal SNX27 binding. Deletion of residues 79 to 100 led to a pronounced reduction in binding and subsequent deletion of residues 41 to 54 abolished the interaction. Alignment of these regions of SNX1 revealed 2 similar motifs with DIF or DLF sequences flanked by upstream acidic residues. Comparison with the amino terminus of SNX2, which also binds SNX27 (S2A Fig), showed that it also has 2 similar sequences in its disordered amino-terminal domains, and, altogether, these define a conserved sequence D/E-x-x-D-I/L-F, here termed the aDLF motif (Fig 3D and 3E &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;).\nFig 3\nThe amino termini of SNX1 and SNX2 directly interact with SNX27 FERM domain through tandem aDLF motifs.\n(A) Co-immunoprecipitation of GFP-tagged SNX1 FL or truncation mutants corresponding to residues 1 to 279 and 1 to 138 expressed in HEK-293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for SNX27 and GFP. The band intensity of SNX27 was measured from n =  3 independent experiments using Odyssey software and was normalized to GFP expression. The relative binding of SNX1 constructs was compared with the SNX1 FL using a 1-way ANOVA and Dunnett test. (B) Binding of mSNX1FL (orange) and mSNX11-139 (blue) with hSNX27FERM by ITC. (C) Co-immunoprecipitation of GFP-tagged SNX1 amino terminus (1 to 138) and a panel of constructs corresponding to serial truncations of SNX1 amino terminus. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensity of SNX27 and GFP was measured from n = 4 independent experiments using Odyssey software and was normalized to GFP expression. The relative binding of SNX1 truncation mutants was compared with the SNX1 amino terminus (1-138) using a 1-way ANOVA and Dunnett test. (D) Alignment of the SNX1 regions that are involved in SNX27 binding with the corresponding sequences in the SNX2 amino terminus. The conserved residues between SNX1 and SNX2 account for a consensus motif of acidic residues followed by DI/LF (aDLF motif). (E) Alphafold structural prediction for FL SNX1 [&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;]. The amino terminus of SNX1 is shown in its full extension and the residues involved in SNX27 binding are colored in red and green. (F) Co-immunoprecipitation of GFP-tagged SNX1 WT and a panel of amino acid swap mutants across the aDLF motifs of SNX1. The constructs were transiently transfected in HEK293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were analyzed by quantitative fluorescence-based western blotting. The band intensities of SNX27, SNX6, and GFP were measured from n = 4 independent experiments using Odyssey software. The band intensity, normalized to GFP expression, is presented as the average fraction of SNX1 WT. The binding of SNX1 point mutants to SNX27 and SNX6 were compared with the SNX1 WT using a 1-way ANOVA and Dunnett test. (G) Binding of mSNX27FL and hSNX1 peptides by ITC. SNX135-51(D45K): green; SNX135-51: orange; SNX175-92: lilac; ND*: no binding detected. (H) Binding of mSNX27FL and hSNX2 peptides by ITC. SNX216-33(DLF/SSS): blue; SNX216-33: green; SNX262-82: lilac; ND*: no binding detected. The ITC graphs represent the integrated and normalized data fit with 1:1 ratio binding. The binding affinity (Kd) is given as mean of at least 2 independent experiments. Thermodynamic parameters for the binding are provided in S1 Table. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001. The data underlying the graphs shown in the figure can be found in S1 Data. aDLF, acidic-Asp-Leu-Phe; FL, full-length; ITC, isothermal titration calorimetry; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27; WT, wild-type.\nSite-directed mutagenesis targeting the first aDLF motif of SNX1 established that full-length GFP-SNX1(E42K), GFP-SNX1(E44K), GFP-SNX1(D45K), GFP-SNX1(I46K), and GFP-SNX1(F47K) all displayed reduced binding to SNX27 of around 50% (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). A similar reduction in binding was also observed when targeting the second aDLF motif of SNX1 as observed with the GFP-SNX1(E84K), GFP-SNX1(D87K), and GFP-SNX1(F89D) mutants (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). In a double GFP-SNX1(D45K,D87K) mutant that simultaneously targeted key residues in both aDLF motifs SNX27, binding was reduced to near undetectable levels (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Importantly, all mutants retained their ability to associate with SNX6, establishing that ESCPE-1 was still able to assemble. In addition, the mutant targeting both aDLF motifs in SNX2, GFP-SNX2(D26K,D72K), also reduced SNX27 binding to near undetectable levels (S2B Fig). Next, we used ITC to quantify the binding of aDLF peptides from SNX1 and SNX2 to full-length mouse SNX27. Peptides corresponding to residues 35 to 51 and 75 to 92 encompassing the first and second aDLF motifs of human SNX1, respectively, bound to SNX27 with respective affinities of 33.4 muM and 13.3 muM (Fig 3G &lt;&lt;FIG:pbio.3001601.g003&gt;&gt; and S1 Table). Establishing the redundant nature of the interaction with SNX27, we designed peptides corresponding to the first and second aDLF motifs of SNX2, namely residues 16 to 33 and 62 to 82. These also bound to full-length mouse SNX27 with affinities of 37.8 muM and 22.6 muM, respectively (Fig 3H &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Further defining the redundancy between SNX1 and SNX2, introduction of a charge swap mutation into the SNX1 35-51 peptide, SNX1(D45K), and a triple mutation in the SNX2 16-33 peptide, SNX2(DLF-SSS), resulted in mutant peptides that showed no detectable binding to SNX27 (Fig 3G and 3H &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Finally, a competitive ITC assay using full-length SNX27 preincubated with excess SNX1 75-92 peptide established that it competed for binding of SNX2 16-33 peptide, consistent with both sequences associating with the same surface within the SNX27 FERM domain (S2C Fig). Overall, these data establish that through tandem aDLF motifs present within their unstructured amino terminal extensions SNX1 and SNX2 directly associate with the FERM domain of SNX27.\n\nA basic surface in the SNX27 FERM domain is required for SNX1/SNX2 binding\nSNX27 and SNX17 are related endosome associated cargo adaptors [&lt;&lt;REF:Ghai-2011-108-7763&gt;&gt;,&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;]. While they display entirely distinct functional roles in sequence-dependent endosomal cargo sorting [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;,&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;], they share structurally similar PX and FERM domains [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;,&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;] (Fig 4A &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). A comparative immunoisolation of GFP-SNX17 and GFP-SNX27 from HEK-293T cells established that SNX17 does not bind to ESCPE-1 (Fig 4B &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;), consistent with the loss of ESCPE-1 binding to the SNX27 chimera containing the SNX17 FERM domain (Fig 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;). To identify potential regions for aDLF motif binding to the SNX27 FERM domain, we initially performed a comparative structural analysis between the X-ray structure of the SNX17 FERM domain [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;] and a homology model of the SNX27 FERM domain [&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;]. In particular, we focused on the distribution and clustering of complementary basic amino acids. This revealed that the FERM domain of SNX27 possesses a basic surface defined in part by R437, R438, K495, K496, R498, and K501, residues that in SNX17 often corresponded to uncharged residues or negatively charged amino acids (Fig 4A and 4C &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Indeed, charge swap site-directed mutagenesis of these residues generated full-length GFP-SNX27 mutants that when expressed and immunoisolated from HEK-293T cells lacked the ability to associate with ESCPE-1; all mutants, however, retained the ability to associate with Retromer (Fig 4D &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). In direct ITC-quantified binding assays human SNX27(K495D), SNX27(R498D), and SNX27(K501D) all displayed no detectable binding to the human SNX1 75-92 peptide, while the SNX27(R496D) mutant displayed a reduce binding affinity of 56.4 muM when compared with the 12.7 muM observed for wild-type SNX27 (Fig 4E &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;).\nFig 4\nSNX1 and SNX2 amino termini bind a unique basic surface of the SNX27-FERM domain.\n(A) Sequence alignment of SNX27 FERM domain with the SNX17 FERM domain. The positively charged residues that constitute a basic surface unique to SNX27 are boxed and labeled in blue. (B) Co-immunoprecipitation of GFP-tagged SNX27 and SNX17. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps. (C) Homology model of the SNX27 FERM domain [&lt;&lt;REF:Ghai-2015-128-553&gt;&gt;]. The residues that are unique to SNX27 FERM domain and those that may account for the interaction with ESCPE-1 are labeled. (D) Co-immunoprecipitation of GFP-tagged SNX27 WT and a panel of amino acid swap mutants across the FERM domain. The constructs were transiently transfected in HEK293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX2, SNX5, and SNX6. The band intensities were measured from n = 3 independent experiments using Odyssey software. The band intensity, normalized to GFP expression, is presented as the average fraction of the WT. The binding of SNX27 point mutants to VPS35, SNX1, SNX2, SNX5, and SNX6 was compared with the SNX27 WT using a 2-way ANOVA and Dunnett test. (E) Binding of SNX175-92 peptide to hSNX27FL proteins by ITC (hSNX27 R498D: dark-blue; hSNX27 K501D: orange; hSNX27 K495D: lilac; hSNX27 K496D: sky-blue; hSNX27 WT: red-orange; ND*: no binding detected). The ITC graphs represent the integrated and normalized data fit with 1:1 ratio binding. The binding affinity (Kd) is given as mean of at least 2 independent experiments. Thermodynamic parameters for the binding are provided in S1 Table. (F) The left panel shows the AlphaFold2 generated model of the SNX27 FERM domain (pink ribbons) bound to the core aDLF sequences (yellow sticks) of SNX1 and SNX2 with identical binding orientations. For clarity, only one SNX27 model is shown with the 8 aDLF peptides overlaid (2 peptide models each of the 4 sequences from SNX1 and SNX2). The 3 subdomains of the SNX27 FERM domain F1, F2, and F3 are indicated. The middle panel shows a close-up view of the modeled SNX1 aDLF core sequence 44EDIFTGA50. The strictly conserved Phe side chain of the aDLF sequences is numbered F0 for reference, and other residues are numbered with respect to this. (G) Co-immunoprecipitation of GFP-tagged SNX27 and a panel of hydrophobic amino acid mutations predicted to mediate the SNX27 FERM-ESCPE1 binding. The constructs were transiently transfected in HEK-293T cells, the cell lysates were subjected to GFP trap-based immunoprecipitation, and the immunoprecipitates were blotted for VPS35, SNX1, SNX5, SNX6, and GFP. The blot is representative of 3 independent GFP traps with the normalized enrichment of SNX27 association relative to expression of GFP-tagged protein quantified in the bar chart. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001. The data underlying the graphs shown in the figure can be found in S1 Data. aDLF, acidic-Asp-Leu-Phe; ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; ITC, isothermal titration calorimetry; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\nFinally, we modeled the binding of the aDLF sequences from SNX1 and SNX2 with SNX27 using the AlphaFold2 machine learning algorithm [&lt;&lt;REF:Jumper-2021-15-1&gt;&gt;,&lt;&lt;REF:Mirdita-2021-?-?&gt;&gt;]. In total, we generated 12 models, with 3 models each of SNX27 in the presence of the 4 separate aDLF peptides (Figs 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt; and S3A-S3C). Without any prior inputs apart from the SNX27 and peptide sequences, AlphaFold2 produced very high confidence models of the SNX27 FERM domain. Remarkably, in all 12 models, the aDLF peptides associated specifically with the SNX27 F3 subdomain, near the basic surface identified by mutagenesis (Fig 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Closer inspection showed that in 8 of the models the core D[L/I]F residues adopted essentially identical bound structures, with the hydrophobic [L/I]F side chains inserting into a hydrophobic pocket and interacting with SNX27 I467, F469, Y490, and V500 and the acidic Asp side chain surrounded by the complementary basic SNX27 residues R437, R438, K439, and R498 (Fig 4F &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). To further validate the model, we therefore mutated I467, Y490, and V500 to Lys, all of which disrupted the interaction of SNX27 with ESCPE-1 (Fig 4G &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;). Together, these models are entirely consistent and strongly support the mutagenesis data for the SNX27-peptide interactions.\n\nSNX27 association with ESCPE-1 couples cargo retrieval with promotion of cell surface recycling\nTo define the functional significance of the association of SNX27 to ESCPE-1, we first transfected our SNX27 knockout HeLa cells line with constructs encoding for either GFP-tagged wild-type SNX27 or the ESCPE-1 binding defective mutant GFP-SNX27(R498D) (Fig 5A &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Analysis of their relative subcellular distributions established that they each associated with EEA1 and SNX6-positive endosomes (Fig 5B &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Previously, we established that by binding to the PDZbm present within the cytoplasmic facing tail of GLUT1, the SNX27-Retromer complex serves to retrieve internalized GLUT1 away from lysosomal mediated degradation ([&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;], and see Fig 1A &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). We therefore performed a series of rescue experiments to quantify the SNX27-dependent retrieval of internalized GLUT1 from lysosomal mediated degradation, using the SNX17-Retriever-mediated retrieval of alpha-5 integrin as a negative control (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;) [&lt;&lt;REF:McNally-2017-19-1214&gt;&gt;]. In the SNX27 knockout HeLa cells, GLUT1 displayed an enhanced rate of degradation following 8 hours of treatment with cycloheximide, data that are consistent with the loss of SNX27 expression leading to a reduced ability for transporter retrieval from the lysosomal degradative fate (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). This phenotype was rescued by reexpression of wild-type GFP-tagged SNX27 (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Interestingly, expression of GFP-SNX27(R498D), a mutant that fails to associate with ESCPE-1 but retains binding to Retromer (Fig 4D and 4E &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;), also fully rescued the retrieval of GLUT1 from lysosomal degradation (Fig 5C &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). The coupling of SNX27-Retromer to the ESCPE-1 complex is therefore not required for the retrieval of GLUT1 from lysosomal degradation.\nFig 5\nSNX27:ESCPE-1 interaction is required for endosome-to-plasma membrane recycling of SNX27 cargos.\n(A) Representative western blot showing similar expression levels between transiently expressed GFP-SNX27 and GFP-SNX27(R498D). (B) Representative images of SNX27 KO HeLa cells transiently transfected with GFP-SNX27 or GFP-SNX27(R498D). Moreover, 48 hours after transfection, cells were fixed and immunostained for the endosomal markers EEA1 and SNX6. Magnified views of the white boxes are shown boxed. Cell numbers analyzed for colocalization were 60 GFP-SNX27 and 60 GFP-SNX27 R498D cells across n  =  3 independent experiments. The Pearson coefficient values were compared using a 2-tailed unpaired t test; for EEA1 P = 0.2971, SNX6 P = 0.0506. (C) Degradation assay in parental HeLa cells or SNX27 KO HeLa cells that were transiently transfected with GFP-SNX27 or GFP-SNX27(R498D) mutant. Moreover, 48 hours after transfection, cells were incubated with 10 mug/ml cycloheximide and lysed at different time points as indicated. The band intensities of endogenous GLUT1 and ITGA5 were measured from n = 3 independent experiments using Odyssey software. The levels of GLUT1 and ITGA5 at 8 hours were compared with the corresponding level at 0-hour time point. Analysis was done using a 2-way ANOVA and Sidak test. GLUT1: P = 0.0735 (HeLa), 0.0068 (KO), 0.9997 (+GFP-SNX27), 0.9975 (+GFP-R498D). ITGA5: P = 0.1151 (HeLa), 0.9820 (KO), 0.9999 (+GFP-SNX27), and 0.6899 (+GFP-R498D). (D) Analysis of surface levels of GLUT1 and ITGA5 in parental HeLa cells or SNX27 KO HeLa cells that were transiently transfected with GFP-SNX27 or GFP-SNX27(R498D) mutant. Moreover, 48 hours after transfection, cells were subjected to surface biotinylation followed by streptavidin-based immunoisolation. The immunoisolates were blotted for GLUT1, and the band intensities were measured from n =  4 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. GLUT1: P = 0.0134 (KO versus HeLa), 0.9587 (+GFP-SNX27 versus HeLa), and 0.0163 (+GFP-SNX27 R498D versus HeLa). The immunoisolates were blotted for ITGA5, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. ITGA5: P = 0.5616 (KO versus HeLa), 0.0733 (+GFP-SNX27 versus HeLa), and 0.2354 (+GFP-SNX27 R498D versus HeLa). (E) Representative images of parental Hela cells, SNX27 KO HeLa cells transiently transfected with GFP-SNX27 or GFP-SNX27(R498D). Moreover, 48 hours after transfection, cells were fixed and immunostained for the cargo GLUT1, the endosomal marker VPS35, and the late endosome/lysosome marker LAMP1. A total of 60 cells were quantified per condition in n = 4. The Pearson coefficient values were compared to the values of Parental HeLa using 1-way ANOVA and Dunnett test. The values for LAMP1 colocalization were P &lt; 0.0001 (Parental versus KO), 0.0001 (Parental versus +WT), 0.1084 (Parental versus +R498D), &lt;0.0001 (+WT versus KO), and 0.0692 (+WT versus +R498D). The values for VPS35 colocalization were P &lt; 0.0001 (Parental versus KO), 0.0029 (Parental versus +WT), 0.0001 (Parental versus +R498D), &lt;0.0001 (+WT versus KO), and 0.6346 (+WT versus +R498D). Scale bars, 25 mum (micrographs) and 5 mum (magnified images). Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; KO, knockout; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\nNext, we turned to examining the promotion of GLUT1 recycling back to the cell surface. Here, we utilized restricted cell surface biotinylation to quantify the amount of GLUT1 at the cell surface [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. Compared with control HeLa cells, SNX27 knockout HeLa cells displayed a reduced cell surface abundance of GLUT1, a result of missorting of internalized GLUT1 for lysosomal degradation (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). Again, this phenotype was rescued by expression of GFP-SNX27 (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;) [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. In contrast, expression of GFP-SNX27(R498D) failed to rescue the cell surface level of GLUT1 (Fig 5D &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), consistent with the association of SNX27 with ESCPE-1 being required to promote the cell surface recycling of internalized GLUT1. Importantly, GFP-SNX27(R498D) exhibited an endosomal localization indistinguishable from wild-type SNX27 (Fig 5B &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), consistent with the inability to promote endosome-to-plasma membrane recycling of GLUT1 not arising from a loss of endosomal localization. Furthermore, confocal analysis of the steady-state distribution of GLUT1 in GFP-SNX27(R498D) rescue cells confirmed that GLUT1 was retrieved from the lysosomal degradative fate, as shown by reduced co-localization with the LAMP1-labeled lysosome (Fig 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;). However, unlike the rescue with wild-type GFP-SNX27, where GLUT1 was enriched at the cell surface, the GLUT1 in the GFP-SNX27(R498D) rescue displayed an increased level of intracellular puncta and failed to repopulate the cell surface suggestive of the transporter being trapped in an endosomal compartment (Fig 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;), the precise nature of which will require further study. Together, these data are consistent with SNX27 orchestrating cargo retrieval from the degradative fate through its PDZ domain-mediated association with cargo and Retromer, while the SNX27 FERM domain mediates association with ESCPE-1 to promote tubular-based plasma membrane recycling of the retrieved cargo.\n\nESCPE-1 association with SNX27 couples cargo retrieval with promotion of cell surface recycling\nTo complete the mechanistic dissection of SNX27 coupling to ESCPE-1, we used a previously characterized double SNX1 and SNX2 knockout HeLa cell line [&lt;&lt;REF:Simonetti-2017-216-3695&gt;&gt;] to generate lentiviral transduced stable lines expressing GFP-SNX1 or GFP-SNX1(D45K,D87K) (Fig 6A &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). When expressed at near to endogenous levels each chimera displayed an indistinguishable association with EEA1 and SNX6-labeled endosomes (Fig 6B &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Consistent with ESCPE-1 not being required for GLUT1 retrieval from lysosomal degradation, the rate of internalized GLUT1 degradation following 8 hours of treatment with cycloheximide was equivalent between parental HeLa cells, the SNX1/SNX2 knockout cells, and the GFP-SNX1 and GFP-SNX1(D45K,D87K) rescue lines (Fig 6C &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). In contrast, the quantification of restricted cell surface biotinylation established that compared to the parental HeLa cells, the SNX1/SNX2 double knockout HeLa cells displayed a reduced cell surface abundance of GLUT1 consistent with a defect in recycling of internalized GLUT1 (Fig 6D &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). While reexpression of GFP-SNX1 rescued the cell surface level of GLUT1, expression of the GFP-SNX1(D45K,D87K) mutant, defective in SNX27 association (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;), failed to rescue the cell surface recycling of GLUT1 (Fig 6D &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Finally, confocal analysis of the steady-state distribution of GLUT1 revealed that when compared to parental cells, the glucose transporter was enriched in Retromer decorated endosomes in the SNX1/SNX2 double knockout HeLa cells (Fig 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;), in agreement with GFP-SNX27DeltaFERM rescue (see Fig 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;). Reexpression of GFP-SNX1 partially rescued the cell surface delivery of GLUT1 from this compartment, while expression of GFP-SNX1(D45K,D87K) showed an increase in colocalization of GLUT1 with Retromer decorated endosomes (Fig 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt;). Taken together, these data provide an independent validation that the association of ESCPE-1 to the FERM domain of SNX27 is required to promote tubular-based endosomal exit and cell surface recycling of cargo retrieved from lysosomal degradation.\nFig 6\nESCPE-1 interaction with SNX27 is required for endosome-to-plasma membrane recycling of GLUT1.\n(A) Representative western blot showing similar expression levels between stably transfected GFP-SNX1 and GFP-SNX1(KK) in SNX1/2KO cells. SNX1 band intensities were measured from n = 3 and normalized to actin. Analysis was done using a 2-tailed unpaired t test. (B) Representative images of SNX1/2 KO HeLa cell line stably transfected with GFP-SNX1 or GFP-SNX1(KK). Cells were fixed and immunostained for the endosomal markers EEA1 and SNX6. Magnified views of the white boxes are shown boxed. Cell numbers analyzed for colocalization were 50 GFP-SNX1 and 50 GFP-SNX1(KK) cells across n  =  3 independent experiments. The Pearson coefficient values were compared using a 2-tailed unpaired t test; for EEA1 P  = 0.5971, SNX6 P = 0.6459. (C) Degradation assay in parental HeLa cells or SNX1/2 KO HeLa cells that were stably transfected with GFP-SNX1 or GFP-SNX1(KK) mutant. Cells were incubated with 10 mug/ml cycloheximide and lysed at different time points as indicated. The band intensities of endogenous GLUT1 and ITGA5 were measured from n = 3 independent experiments using Odyssey software. The levels of GLUT1 and ITGA5 at 8 hours were compared with the corresponding level at 0-hour time point. Analysis was done using a 2-way ANOVA and Sidak test. GLUT1: P = 0.6796 (HeLa), 0.9768 (KO), 0.8430 (+GFP-SNX1), and 0.2665 (+GFP-KK). ITGA5: P = 0.2247 (HeLa), 0.7892 (KO), 0.9196 (+GFP-SNX1), and 0.9930 (+GFP-KK). (D) Analysis of surface levels of GLUT1 and ITGA5 in parental HeLa cells or SNX1/2 KO HeLa cells that were stably transfected with GFP-SNX1 or GFP-SNX1(KK) mutant. Cells were subjected to surface biotinylation followed by streptavidin-based immunoisolation. The immunoisolates were blotted for GLUT1, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. GLUT1: P = &lt;0.0001 (KO versus HeLa), 0.0878 (+GFP-SNX1 versus HeLa), and &lt;0.0001 (+GFP-KK versus HeLa). The immunoisolates were blotted for ITGA5, and the band intensities were measured from n =  3 independent experiments using Odyssey software and compared to the levels in parental HeLa using 1-way ANOVA and Dunnett test. ITGA5: P = 0.2592 (KO versus HeLa), 0.7185 (+GFP-SNX1 versus HeLa), and 0.0653 (+GFP-KK versus HeLa). (E) Representative images of parental Hela cells, SNX1/2 KO HeLa cells stably transfected with GFP-SNX1 or GFP-SNX1(KK). Cells were fixed and immunostained for the cargo GLUT1, the endosomal marker VPS35, and the late endosome/lysosome marker LAMP1. A total of 50 cells were quantified per condition in n = 3. The Pearson coefficient values were compared to the values of Parental HeLa using 1-way ANOVA and Dunnett test. The values for LAMP1 colocalization were P = 0.0002 (Parental versus KO), 0.7919 (Parental versus +WT), 0.2225 (Parental versus +KK), 0.0028 (+WT versus KO), and 0.6677 (+WT versus +KK). The values for VPS35 colocalization were &lt;0.0001 (Parental versus KO), 0.0020 (Parental versus WT), &lt;0.0001 (Parental versus +KK), &lt;0.0001 (+WT versus KO), and 0.2073 (+WT versus +KK). Scale bars, 25 mum (micrographs) and 5 mum (magnified images). Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; KO, knockout; SNX1, sorting nexin-1; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\n\nEvolutionary assembly of the SNX27:Retromer:ESCPE-1 sorting axis\nWe used phylogenetic analyses to investigate the evolution of the association of SNX27 with Retromer and with the ESCPE-1 complex. The inferred phylogeny of SNX27 suggests the gene was ancestrally present in filozoans (i.e., the group comprising filastereans, choanoflagellates, and metazoans) (S4A Fig). In SNX27, an exposed beta-hairpin in the SNX27 PDZ domain, encoded by residues 67 to 79 in human SNX27, engages a groove in the VPS26A (and VPS26B) subunit of Retromer [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;] (Fig 7A &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;). An equivalent sequence, including key residues Leu-67 and Leu-74 of the beta-hairpin, is present in SNX27 across all choanozoans (this is, the group comprising metazoans and choanoflagellates; Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4A). Key residues in VPS26A required for SNX27 binding include D44 and L154, equivalent residues in VPS26B being D42 and L152 [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;]. The inferred phylogeny of VPS26A and VPS26B revealed that VPS26B is ancestrally present across eukaryotes, with a duplication of VPS26B in vertebrates leading to the emergence of VPS26A (Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4B). Interestingly, within VPS26B, the D44 residue is present ancestrally in filozoans, while the L154 residue evolved in choanozoans (Fig 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;). Together, these analyses suggest that the SNX27-Retromer association first appeared in the last common ancestor of choanoflagellates and metazoans.\nFig 7\nMolecular evolution of the SNX27-Retromer and SNX27-ESCPE-1 interactions.\n(A) Alphafold2 structural prediction for FL human SNX27 [&lt;&lt;REF:Tunyasuvunakool-2021-596-590&gt;&gt;]. The residues of the exposed beta-hairpin loop in the SNX27 PDZ domain (67 to 79) that account for the interaction with Retromer are highlighted in cyan. Those residues of the FERM domain that account for the interaction with ESCPE-1 (positively charged surface) are highlighted in pink. Our data support a model where the interaction between SNX27 and Retromer is required for the retrieval of PDZ cargos away from the degradative fate. Subsequently, the interaction between SNX27 and ESCPE-1 is required for the export and endosome-to-plasma membrane recycling of PDZ cargos. (B) A cartoon representation incorporating the results of multiple phylogenetic analyses, including VPS5/SNX1/SNX2, SNX27, and VPS26A/VPS26B (see S4A-S4C Fig for detailed phylogenies). The gene families that contain VPS5/SNX1/SNX2 and VPS26A/VPS26B are present across eukaryotes: SNX1 and SNX2 and VPS26A and VPS26B, arose by gene duplication in the common ancestor of vertebrates. SNX27 is found in metazoans, choanoflagellates and the filasterean Capsaspora, but not in more distantly related opisthokonts such as Ichthyosporea or fungi; it is therefore likely arose in the filazoan common ancestor. The residue D44 (VPS26B, Homo sapiens) in VPS26 is found across Filazoa, laying the foundations for the binding interaction between VPS26 and SNX27. L154 (VPS26B, H. sapiens) likely arose in the ancestral choanozoan along with the PDZ loop and key interacting residues (67 to 97 L67, L74, SNX27, H. sapiens), suggesting that the SNX27-Retromer interaction was already present in the common ancestor of metazoans and choanoflagellates. Key residues of the FERM domain: R437, R498, and K501 (SNX27, H. sapiens) are recognizable in the ancestral metazoan. The first aDLF motif (SNX1) was acquired in the common ancestor of cnidarians and bilaterians, along with the residues K495 and K496, which make up part of the FERM domain in SNX27, potentially suggesting that SNX27:ESCPE-1 binding has evolved by this time, if not earlier during metazoan evolution. Interestingly, the first aDLF motif does not appear to be present in the placazoan or ctenophoran sequences, (although the R437, R498, and K501 residues are present), which might reflect secondary loss of the motif in these groups. The second aDLF motif is only present in vertebrates.\nThe association between SNX27 and ESCPE-1 depends on the presence of binding motifs in SNX1 and SNX2 and the FERM domain of SNX27. SNX1 is present across eukaryotes; a gene duplication giving rise to SNX1 and SNX2 from a single ancestral protein occurred in the common ancestor of vertebrates (Figs 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt; and S4C). Several motifs required in the FERM domain of H. sapiens SNX27 for binding to ESCPE-1 appear to have evolved in the common ancestor of metazoans or early during their radiation (Fig 7B &lt;&lt;FIG:pbio.3001601.g007&gt;&gt;); many of the key residues (R437, R498, and K501) were in place in the last metazoan common ancestor or prior to the divergence of cnidarians and bilaterians (K495 and K496). For SNX1, the first aDLF motif was likely already present in the last metazoan common ancestor. It is present in sequences from bilaterians and Porifera (sponges) but is absent in cnidarians, placazoans, and ctenophorans, which might represent secondary losses of the motif. As for the second aDLF motif, this is present in all vertebrates. This analysis therefore suggests that the SNX27:ESCPE-1 interaction evolved after the SNX27-Retromer link during early metazoan evolution.\n\nDiscussion\nHere, in combining biochemical, structural, and cellular analyses, we have established that aDLF motifs in the redundant SNX1 and SNX2 subunits of ESCPE-1 directly associate with the FERM domain of SNX27 within the SNX27-Retromer assembly. Studies of endosomal cargo sorting show that this association is necessary to promote the ESCPE-1-dependent tubulovesicular cell surface recycling of internalized transmembrane proteins. Our data suggest that ESCPE-1 mediated recycling occurs subsequently or in conjunction with SNX27-Retromer-dependent retrieval from lysosomal degradation. Building on previous studies [&lt;&lt;REF:Temkin-2011-13-715&gt;&gt;-&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;], these data collectively identify the SNX27:Retromer:ESCPE-1 assembly as a dynamic endosomal coat complex that couples sequence-dependent cargo recognition with membrane remodeling to facilitate the formation of cargo-enriched transport carriers for cargo recycling. In addition, we have begun to describe the functional diversification of Retromer as it has evolved from its tight association with the SNX-BAR homologues Vps5 and Vps17, as originally identified in yeast [&lt;&lt;REF:Seaman-1998-142-665&gt;&gt;], to a more transient engagement of ESCPE-1 in metazoans and their relatives mediated at least in part through the SNX27 cargo adaptor.\nAt present, we propose the following model to describe the functional connectivity of the SNX27:Retromer:ESCPE-1 coat complex within the context of cargo retrieval and recycling. Newly endocytosed cargo containing a functional SNX27 PDZbm are recognized by SNX27, associated with the PI(3)P-enriched early endosome, and these interactions together can stabilize the endosomal residency of SNX27 [&lt;&lt;REF:Ghai-2013-110-E643&gt;&gt;]. As the endosome matures, SNX27 binds to Retromer through direct association of VPS26A (or VPS26B) to the SNX27 PDZ domain. These interactions are mutually stabilizing, thereby locking down cargo association [&lt;&lt;REF:Gallon-2014-111-E3604&gt;&gt;,&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;]: This assembly is essential for cargo retrieval from the lysosomal degradative fate [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. Retromer dimerization and the formation of Retromer arches [&lt;&lt;REF:Kendall-2020-28-393&gt;&gt;,&lt;&lt;REF:Leneva-2021-7-?&gt;&gt;,&lt;&lt;REF:Kovtun-2018-561-561&gt;&gt;] may aid SNX27 and cargo clustering, thereby forming the retrieval subdomain and initiating weak membrane curvature [&lt;&lt;REF:Simunovic-2015-25-780&gt;&gt;]. These events are supported through the Retromer-mediated recruitment of the WASH complex and the localized production of branched filamentous actin [&lt;&lt;REF:Harbour-2012-442-209&gt;&gt;,&lt;&lt;REF:Jia-2012-23-2352&gt;&gt;]. An additional accessory protein, ANKRD50, may serve to stabilize the formation of the SNX27:Retromer:WASH assembly [&lt;&lt;REF:Kvainickas-2017-130-382&gt;&gt;]. To this emerging \"bud,\" ESCPE-1 is recruited through recognition of 3-phosphoinositides and membrane curvature [&lt;&lt;REF:Carlton-2004-14-1791&gt;&gt;,&lt;&lt;REF:Cozier-2002-277-4873048736&gt;&gt;,&lt;&lt;REF:Chandra-2019-10-1528&gt;&gt;] and the direct binding to SNX27 described in the present study. The timed increase in ESCPE-1 residency and localized concentration provides the handover mechanism to ensure that the captured and enriched cargo enter the forming tubular profile and the ensuing transport carrier. Continued localized WASH-mediated actin polymerization and the recruitment of motor proteins facilitates tubule maturation and fission [&lt;&lt;REF:Wassmer-2009-17-110&gt;&gt;-&lt;&lt;REF:Freeman-2014-127-2053&gt;&gt;] (Fig 8 &lt;&lt;FIG:pbio.3001601.g008&gt;&gt;). Additional cargo entry may be achieved through direct recognition by ESCPE-1 and yet to be described mechanisms [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;]. As with the assembly of other coat complexes [&lt;&lt;REF:Schmid-2007-448-883&gt;&gt;], association of the SNX27:Retromer:ESCPE-1 coat is driven by avidity through a series of low affinity interactions that evoke the consideration of checkpoints for monitoring the fidelity of pathway progression. It is important to state that this model is certainly an oversimplification and many key questions remain unanswered. Not least whether such a coat complex can be visualized through membrane associated cryo-electron tomography of reconstituted SNX27:Retromer:ESCPE-1 perhaps in the presence of ANKRD50.\nFig 8\nA 2-step model for the endosomal sorting of SNX27 cargos.\n(A) Schematics of the endosomal sorting of PDZ cargos. Cargos possessing a PDZbm that enter the endosomal system are recognized by SNX27 via its PDZ domain. (B) SNX27 binding to Retromer aids the retrieval of cargos from the lysosomal degradative path. (C) Once cargos have been retrieved, the SNX27 FERM engages the amino terminus of SNX1 and SNX2 mediating the inclusion of SNX27 cargos into ESCPE-1 tubulovesicular carriers. (D) This mechanism results in SNX27 cargos entering carriers that exit endosomes and are transported, directly and/or indirectly, to the cell surface. ESCPE-1, endosomal SNX-BAR sorting complex for promoting exit 1; PDZbm, PDZ-binding motif; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27; WASH, Wiskott-Aldrich syndrome protein and SCAR homologue.\nThe importance of the amino-terminal extension of SNX1 and SNX2 has not previously been considered. While we and others [&lt;&lt;REF:Chandra-2022-83-100842&gt;&gt;-&lt;&lt;REF:Yong-2021-118-?&gt;&gt;] have independently established that they constitute primary sites of interaction with SNX27, it is tempting to speculate that they could orchestrate a broader set of protein-protein interactions. Interestingly, these unstructured amino termini may extend to over 30 nm in length and could allow the ESCPE-1 oligomeric assemblies to capture peripheral factors for the fine-tuned regulation of tubular maturation and scission. This would resemble the organization of the endocytic clathrin coat where several accessory proteins protrude from the core clathrin lattice into the cytoplasm to enable a regulated function of the endocytic coat during endocytic vesicle formation [&lt;&lt;REF:Skruzny-2020-16-e9009&gt;&gt;].\nOther than mediating the recruitment of accessory proteins, there is some evidence that these amino-terminal extensions could play a role in the assembly and membrane targeting of SNX-BAR dimers. In fact, besides SNX1 and SNX2, SNX4, SNX7, SNX8, and SNX30, all possess unstructured amino-terminal extensions of different length and of unknown function. The low-complexity region of the SNX8 homologue Mvp1 appears to stabilize soluble Mvp1 in a cytosolic tetrameric form where the PX and BAR domains are masked and unable to target membranes [&lt;&lt;REF:Sun-2020-11-1506&gt;&gt;]. The release of the amino terminus liberates functional dimers of Mvp1 facilitating their membrane association and function [&lt;&lt;REF:Sun-2020-11-1506&gt;&gt;]. It will be interesting to explore whether a similar mechanism also regulates ESCPE-1 targeting to the endosomal network in mammalian cells during endosomal recycling.\nAn important aspect of cargo sorting by SNX27 is regulation by posttranslational modification. Phosphorylation of residues within the core carboxyl-terminal PDZbm negatively impairs SNX27 association, but phosphorylation of residues preceding the minimal PDZbm can dramatically enhance affinity for the SNX27 PDZ domain [&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;]. A well-characterized example of cargo tail phosphorylation modulating SNX27 affinity and subsequent endosomal sorting is the beta2AR [&lt;&lt;REF:Clairfeuille-2016-23-921&gt;&gt;,&lt;&lt;REF:Cao-1999-410-286&gt;&gt;]. Furthermore, phosphorylation of SNX27 at Ser-51 alters the conformation of its PDZbm-binding pocket decreasing the affinity for cargo proteins, thereby inhibiting endosomal sorting [&lt;&lt;REF:Mao-2021-220-e202010048&gt;&gt;] and Ser-302 and Ser-304 phosphorylation in VPS26B can modulate SNX27 association [&lt;&lt;REF:Wang-2021-12-4826&gt;&gt;]. Similarly, ESCPE-1 function can be regulated by phosphorylation; in this case, Ser-226 of SNX5 undergoes phosphorylation, which decreases dimerization with SNX1 and SNX2 and reduces endosomal recycling [&lt;&lt;REF:Itai-2018-13-e0207205&gt;&gt;]. Therefore, a fascinating question regards whether the SNX27:ESCPE-1 interaction may be orchestrated by posttranslational modification. We have here shown that a series of acidic residues upstream of the DLF motif in the SNX1 and SNX2 amino termini contribute to the binding to SNX27 (Fig 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;). Interestingly, several phosphorylation sites in the aDLF sequences of SNX1 have been annotated, including Ser-39, Ser-41, and Ser-72, as well as Thr-72 [&lt;&lt;REF:Dephoure-2008-105-10762&gt;&gt;-&lt;&lt;REF:Bian-2014-96-253&gt;&gt;]. It is tempting to speculate that multiple phosphorylations of the ESCPE-1 amino termini may regulate its association with SNX27-Retromer and, in turn, dictate the timing and kinetics of the endosomal recycling route.\nFinally, a fascinating question relates to the molecular evolution of Retromer [&lt;&lt;REF:Koumandou-2011-124-1496&gt;&gt;]. In yeast, Retromer can form a stable pentameric assembly of 2 subcomplexes, the so called \"cargo-selective\" Vps26:Vps35p:Vps29p heterotrimer and the membrane remodeling Vps5p and Vps17p heterodimer [&lt;&lt;REF:Seaman-1998-142-665&gt;&gt;], the latter being considered analogous to ESCPE-1. Two functional processes, sequence-dependent cargo recognition and the biogenesis of tubulovesicular transport carriers, are therefore brought together in a single pentameric assembly [&lt;&lt;REF:Suzuki-2019-218-2876&gt;&gt;]. In metazoans, the VPS26:VPS35:VPS29 Retromer and ESCPE-1 appear to no longer form a stable pentameric assembly [&lt;&lt;REF:Norwood-2011-12-56&gt;&gt;]; rather, they have diverged into 2 separate complexes and in so doing have expanded their functional roles [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;,&lt;&lt;REF:Simonetti-2017-216-3695&gt;&gt;,&lt;&lt;REF:Norwood-2011-12-56&gt;&gt;,&lt;&lt;REF:Kvainickas-2017-216-3677&gt;&gt;]. Our phylogenetic analysis suggests that the SNX27:Retromer:ESCPE-1 assembly observed in metazoans evolved in a stepwise manner, with the SNX27-Retromer interaction evolving prior to the origin of metazoans, in the common ancestor of metazoans and choanoflagellates--the group of unicellular and colonical flagellates that are the closest living relatives of animals. The SNX27:ESCPE-1 interaction evolved later, during early metazoan evolution and likely prior to the divergence of cnidarians and bilaterians. Thus, within the SNX27:Retromer:ESCPE-1 pathway, the retrieval of cargo from the degradative fate appears to have evolved prior to the direct coupling to ESCPE-1 for promotion of tubulovesicular-based recycling to the cell surface. The divergence of the ancestral pentameric Retromer coat complex therefore appears to have occurred to accommodate the expanding complexity of the endomembrane system in metazoans [&lt;&lt;REF:Rout-2017-86-637&gt;&gt;]. The ability of metazoan Retromer to associate with SNX27 and the ability of ESCPE-1 to directly bind cargo through a helix-loop-helix region [&lt;&lt;REF:Simonetti-2019-21-1219&gt;&gt;,&lt;&lt;REF:Yong-2020-18-e3000631&gt;&gt;], features not observed in yeast, have greatly expanded the capacity for sequence-dependent endosomal retrieval and recycling in metazoans. Indeed, it is tempting to speculate that the increased plasticity of the SNX27:Retromer:ESCPE-1 axis, and the existence of additional Retromer-independent pathways [&lt;&lt;REF:Cullen-2018-19-679&gt;&gt;], has allowed the metazoan endosomal network to accommodate the evolving diversity and number of cargos undergoing sequence-dependent sorting, leading to an expansion in the number of recycling itineraries that cargos can engage to interface with distinct membrane compartments. Dissecting how these sorting complexes and pathways are integrated and regulated, for example, by posttranslational modifications, to shape sequence-dependent cargo sorting within the metazoan endosomal network will be a major challenge moving forward.\nOverall, in defining the mechanistic basis of SNX27-Retromer association with ESCPE-1, we have provided new functional insight into the endosomal sorting of hundreds of internalized PDZbm-containing receptors, channels, transporter, enzymes, and adhesion molecules for repopulation of the cell surface [&lt;&lt;REF:Steinberg-2013-15-461&gt;&gt;]. This increased molecular understanding provides new insight into the role of sequence-dependent endosome sorting in cellular processes ranging from the establishment of polarity and the uptake of nutrients through to synaptic function and cancer cell metastasis in both health and disease [&lt;&lt;REF:Chandra-2021-9-642378&gt;&gt;].\n\nMaterial and methods\n\nAntibodies\nAntibodies used in this study were mouse monoclonal antibodies GFP (clones 7.1 and 13.1; 11814460001; Roche, Germany) (1:2,000 for WB), LAMP1 (clone H4A3, AB2296838; DSHB, USA) (IF 1:200), SNX1 (clone 51/SNX1; 611482; BD, USA) (1:1,000 for WB, 1:200 for IF), SNX2 (clone 13/SNX2; 5345661; BD) (1:1,000 for WB, 1:200 for IF), SNX6 (clone d-5, 365965; Santa Cruz Biotechnology, USA) (1:1,000 for WB, 1:200 for IF), SNX27 (ab77799, Abcam, UK) (1:1,000 for WB, 1:200 for IF), beta-actin (A1978; Sigma-Aldrich, USA) (1:2,000 for WB), ITGA5 (610633, BD) (1:1,000 for WB), VPS29 (clone d-1, 398874; Santa Cruz Biotechnology) (1:200 for WB); rabbit monoclonal antibodies: EEA1 (C45B10; Cell Signaling Technology, USA) (IF 1:200), VPS35 (EPR11501(B); 157220; Abcam) (1:1,000 for WB), GLUT1 (ab115730, Abcam) (1:1,000 for WB, 1:200 for IF); rabbit polyclonal antibodies: GFP (GTX20290; GeneTex, USA) (WB 1:2,000), VPS26A (23892; Abcam) (1:1,000 for WB), VPS35 (97545; Abcam) (1:1,000 for WB, 1:200 for IF), VPS29 (98929; Abcam) (1:100 for WB); and goat polyclonal antibodies to VPS35 (10099; Abcam) (1:200 for IF).\n\nCell culture and transfection\nHeLa and HEK-293T cell lines were sourced from ATCC. Authentication was from the ATCC. Cells were grown in DMEM (Sigma-Aldrich) supplemented with 10% (v/v) FCS (Sigma-Aldrich) and penicillin/streptomycin (Gibco, USA) and grown under standard conditions. FuGENE HD (Promega, USA) was used for transient transfection of DNA according to the manufacturer's instructions. Cycloheximide was used to prevent up-regulation of protein synthesis. In this case, cycloheximide (C7698; Sigma-Aldrich) at 10 mug/ml was added to the cells 48 hours after FuGENE transfection for the indicated time points. For the generation of CRISPR-Cas9 SNX27 knockout HeLa cells, HeLa cells were transfected with pX330 plasmid coding for the gRNA against the gene of interest together with a puromycin resistance-expressing plasmid. Cells were then subjected to puromycin selection for 24 hours. The pooled population of cells was subjected to lysis and western blotting to confirm the knockout. The VPS35 knockout clonal cell line used in this study was characterized previously [&lt;&lt;REF:Kvainickas-2017-216-3677&gt;&gt;]. For GFP-based immunoprecipitations, HEK293T cells were transfected with GFP constructs using polyethylenimine (Sigma-Aldrich) and expression was allowed for 48 hours.\n\nImmunoprecipitation and quantitative western blot analysis\nFor western blotting, cells were lysed in PBS with 1% (v/v) Triton X-100 and protease inhibitor cocktail. The protein concentration was determined with a BCA assay kit (Thermo Fisher Scientific, USA), and equal amounts were resolved on NuPAGE 4% to 12% precast gels (Invitrogen, USA). Blotting was performed onto polyvinylidene fluoride membranes (Immobilon-FL; EMD Millipore, USA) followed by detection using the Odyssey infrared scanning system (LI-COR Biosciences, USA). For GFP-based immunoprecipitations, HEK-293T cells were lysed 48 hours after transfection in immunoprecipitation buffer (50 mM Tris-HCl, 0.5% (v/v) NP-40, and Roche protease inhibitor cocktail) and subjected to GFP trap (ChromoTek, Germany). Immunoblotting was performed using standard procedures. Detection was performed on an Odyssey infrared scanning system (LI-COR Biosciences) using fluorescently labeled secondary antibodies. In using the Odyssey, we routinely performed western blot analysis where a single blot is simultaneously probed with antibodies against 2 proteins of interest (distinct antibody species) followed by visualization with the corresponding secondary antibodies conjugated to distinct spectral dyes.\n\nBiotinylation of cell surface proteins\nFor surface biotinylation experiments, fresh Sulfo-NHS-SS Biotin (Thermo Fisher Scientific, #21217) was dissolved in ice-cold PBS at pH 7.8 at a final concentration of 0.2 mg/ml. Cells were washed twice in ice-cold PBS and placed on ice to slow down the endocytic pathway. Next, cells were incubated with the biotinylation reagent for 30 minutes at 4degreesC followed by incubation in TBS for 10 minutes to quench the unbound biotin. The cells were then lysed in lysis buffer and subjected to Streptavidin beads-based affinity isolation (GE Healthcare, USA).\n\nImmunofluorescence staining\nCells were fixed in 4% (v/v) PFA for 20 minutes and washed 3 times in PBS and permeabilized with 0.1% (v/v) Triton X-100. For LAMP1 staining, cells were instead permeabilized by submersion in liquid nitrogen for 20 seconds. Fixed cells were blocked in 1% (w/v) BSA and incubated in primary antibody and respective secondary antibody (Alexa Fluor; Thermo Fisher Scientific) in 1% (w/v) BSA.\n\nImage acquisition and image analysis\nMicroscopy images were collected with a confocal laser-scanning microscope (SP5 AOBS; Leica Microsystems, Germany) attached to an inverted epifluorescence microscope (DMI6000; Thermo Fisher Scientific). A 63x 1.4 NA oil immersion objective (Plan Apochromat BL; Leica Biosystems, Germany) and the standard SP5 system acquisition software and detector were used. Images were captured at room temperature as z stacks with photomultiplier tube detectors with a photocathode made of gallium-arsenide-phosphide (Leica Microsystems) for collecting light emission. Images were captured using Application Suite AF software (version 2.7.3.9723; Leica Microsystems) and then analyzed with the Volocity 6.3 software (PerkinElmer, USA).\n\nStatistics and reproducibility\nAll quantified western blot are the mean of at least 3 independent experiments. Statistical analyses were performed using Prism 7 (GraphPad Software, USA). Graphs represent means and SEM For all statistical tests, P &lt; 0.05 was considered significant and is indicated by asterisks.\n\nMolecular biology and cloning for recombinant protein production\nFor bacterial expression, full-length mouse SNX1 with an amino-terminal His-tag (mSNX1FL hereafter) was cloned into the pMW172Kan vector [&lt;&lt;REF:Collins-2008-9-366&gt;&gt;], and the truncated mSNX1 construct containing residues 1 to 139 was cloned into pGEX4T-2 vector for expression as a thrombin-cleavable amino-terminal GST fusion protein. DNA encoding full-length mouse SNX27 (mSNX27FL hereafter) was inserted into the pMCSG7 vector [&lt;&lt;REF:Eschenfeldt-2009-498-105&gt;&gt;] by the ligation-independent cloning (LIC) approach. Human SNX27 constructs, hSNX27FL, hSNX27FERM, and hSNX27FL single-site mutants R437D, K495D, K496D, R498D, and K501D were cloned into the pET-28a vector by GenScript and codon optimized for bacterial protein expression. An additional TEV cleavage site was inserted between the amino-terminal His-tag and hSNX27 coding sequences.\n\nProtein expression and purification\nThe DNA constructs of mSNX27 and mSNX1 were expressed using Escherichia coli BL21 CodonPlus (DE3) competent cells, while the hSNX27 constructs were expressed using E. coli BL21 (DE3) competent cells. The cells were grown in Ultra Yield flasks at 37degreesC, 180 rpm until the cell density reached OD600 approximately 0.8. Protein expression was then induced with 1 mM IPTG and cells incubated for 16 to 18 hours at 18degreesC. For SNX27, 0.5 mM IPTG was added for induction to get optimal protein expression yield. To collect cell pellets, the cells were harvested by 10-minute centrifugation (JLA 8.1 rotor) at 6,000 rpm, 4degreesC.\nFor SNX1 purification, the cell pellet was resuspended in lysis buffer containing 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-mercaptoethanol (beta-ME), 1 mM benzamidine, and 10 mug/mL DNase I dissolved in PBS containing 10% glycerol. For SNX27 proteins, the cell pellet was resuspended in lysis buffer containing 20 mM Tris-HCl pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM beta-ME, 1% Triton X-100, 1 mM benzamidine, and 10 mug/mL DNase I dissolved in PBS containing 10% glycerol. Cells were lysed by high-pressure homogenization (35 Kpsi). The homogenate was centrifuged for 30 minutes at 4degreesC using a JA 25.50 rotor with 19,000 rpm. The supernatant of His-tagged protein homogenates were loaded onto preequilibrated TALON metal affinity resin (Clontech, USA) or Ni-nitrilotriacetic acid (NTA) resin, while the supernatant of GST-fusion protein homogenates were loaded onto preequilibrated glutathione Sepharose beads (GE Healthcare) and incubated on an orbital rotator for an hour in cold room.\nAffinity matrices with bound fusion proteins were loaded into empty glass chromatography columns (Bio-Rad, USA) washed with 100 ml of washing buffer containing either 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-ME or 20 mM Tris-HCl pH 8.0, 200 mM NaCl, 5% glycerol, 1 mM beta-ME. For His-tagged protein purification, 10 mM and 300 mM imidazole was added to wash and elution buffers, respectively. To collect the GST-tagged mSNX1 constructs, 50 mM reduced glutathione in elution buffer was applied for elution. cOmplete EDTA-free protease inhibitor tablets were added to the eluted SNX1 samples to reduce proteolytic degradation of their disordered amino-terminal region.\nThe fractions containing eluted proteins were concentrated to approximately 5 ml by Amicon cc ultrafiltration devices (10 kDa or 30 kDa molecular mass cutoff) (Millipore) and centrifuged in a bench centrifuge for 10 minutes at maximum speed to remove any precipitated material before being subjected to size-exclusion chromatography (SEC). For SEC, the HiLoad 16/600 Superdex 200 (GE Healthcare) column was used with proteins eluted in SEC buffer containing 50 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM beta-ME or 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5% glycerol, and 1 mM beta-ME. To check the purify and integrity of the purified fractions, sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) was applied using Bolt 12% Bis-Tris gel with 1x MES running buffer at 190V for 25 minutes. After the SDS-PAGE, gels were stained with Coomassie Brilliant Blue R250 solution for visualization. For hSNX27FERM, an additional anion exchange chromatography (IEX) was applied using the Mono Q anion exchange chromatography column preequilibrated with IEX buffer A: 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5% glycerol, 1 mM beta-ME. A linear salt gradient from 0% to 40% IEX buffer B (20 mM Tris-HCl pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM beta-ME) was conducted. Proteins usually eluted in buffer containing 200 mM NaCl, and fractions collected were analyzed by SDS-PAGE.\n\nIsothermal titration calorimetry\nHuman SNX1 and SNX2 peptide sequences were synthesized by Genscript (USA). To obtain the peptide stock solutions at neutral pH, 100 mM Tris-HCl, pH 10.6 was used to dissolve the peptide to a final concentration of 8 mM. The peptides were SNX135-51 EAGDSDTEGEDIFTGAA, SNX135-51(D45K) EAGDSDTEGEKIFTGAA, SNX175-92 NGIHEEQDQEPQDLFADA, SNX216-33 TDFEDLEDGEDLFTSTVS, SNX216-33(DLF/SSS) TDFEDLEDGESSSTSTVS, and SNX262-82 TEVVLDDDREDLFAEATEEVS.\nAll microcalorimetry experiments were conducted at 25degreesC using a PEAQ ITC (Malvern, UK) in 50 mM HEPES (pH 7.5), 200 mM NaCl, 5% glycerol, and 1 mM beta-ME (ITC buffer). All proteins were purified by SEC using the same buffer prior to ITC experiments to avoid buffer mismatches. To measure binding of SNX27 to SNX1 proteins, 800 muM of hSNX27FERM was titrated into 20 muM of GST-mSNX11-139 or His-SNX1FL. For the binding of SNX1 and SNX2 peptides to SNX27, peptides were diluted 10-fold to 800 muM in ITC buffer peptides and titrated into 20 muM SNX27FL proteins containing the same 1:10 ratio of peptide buffer (100 mM Tris-HCl, pH 10.6). The competitive ITC experiments were carried out by titrating 800 muM SNX175-92 peptides into 20 muM proteins with 2-fold molar excess SNX216-33 peptide preincubated with SNX27 on ice for an hour. For ITC experiment using SNX175-92 against hSNX27 wild-type and single-site mutants, 600 muM SNX175-92 peptide was titrated into 20 muM hSNX27 proteins.\nThe ITC experiments were conducted with 1 injection of 0.4 muL followed by a series of 12 injections of 3.2 muL each with 180-second intervals and a stirring speed of 850 rpm. The heat exchange of interactions was obtained through integrating the observed peaks and background correction by subtracting the heats of dilution. Data were analyzed with Malvern software package by fitting and normalized data to a single-site binding model, yielding the thermodynamic parameters Kd, DeltaH, DeltaG, and -TDeltaS for all binding experiments. The stoichiometry was refined initially, and if the value was close to 1, then N was set to exactly 1.0 for calculation. All experiments were performed at least twice to guarantee reproducibility of the data.\n\nAlphaFold predictions of peptide binding to SNX27\nTo generate predicted models of SNX1 and SNX2 amino-terminal peptides associating with the SNX27 FERM domain, we used the AlphaFold2 neural network [&lt;&lt;REF:Jumper-2021-15-1&gt;&gt;] implemented within the freely accessible ColabFold pipeline [&lt;&lt;REF:Mirdita-2021-?-?&gt;&gt;]. For each peptide-modeling experiment, the sequence of the human SNX27a FERM domain (residues 269 to 531; Q96L92) was modeled with the 4 separate amino-terminal aDLF sequences from SNX1 and SNX2. The specific peptides we modeled were residues 33 to 56 and 75 to 98 from human SNX1 (Q13596) and residues 14 to 37 and 60 to 83 from human SNX2 (O60749). ColabFold was executed using default settings where multiple sequence alignments were generated with MMseqs2 [&lt;&lt;REF:Mirdita-2019-35-2856&gt;&gt;], and structural relaxation of final peptide geometry was performed with Amber [&lt;&lt;REF:Hornak-2006-65-712&gt;&gt;] to generate 3 models per peptide. Sequence conservation was mapped onto the modeled SNX27 FERM domain structure with Consurf [&lt;&lt;REF:Ashkenazy-2016-44-W344&gt;&gt;]. Structural alignments and images were generated with Pymol (Schrodinger, USA).\n\nPhylogenetic analyses\nHomologous proteins were identified using NCBI protein-protein BLAST (with default parameters) against all non-redundant GenBank CDS Translations+PDB+SwissProt+PIR+PRF excluding environmental samples from WGS projects for a wider range of species (unaligned sequence data with accession numbers available in S4 Fig). For the ctenophore data, we used the BLAST (with default parameters) from the Mnemiopsis Genome Project Portal (MGP) portal [&lt;&lt;REF:Ryan-2013-342-1242592&gt;&gt;-&lt;&lt;REF:Moreland-2020-?-?&gt;&gt;]. Sequences were aligned using MAFFT v7.480 [&lt;&lt;REF:Katoh-2013-30-772&gt;&gt;], L-INS-I mode, and then used to infer initial exploratory trees to identify specific duplications (using IQtree 2.1.4 [&lt;&lt;REF:Minh-2020-37-1530&gt;&gt;] with the LG+F+G model and 1,000 ultrafast bootstraps). After orthologues of each copy of the gene were identified in a range of taxa, we then performed iterative tree inferences to confirm paralogues and to identify species-specific duplication events. Subsequent maximum likelihood phylogenetic analyses were performed using IQtree 2.1.4 with the best fitting model selected using the Bayesian information criterion (BIC), including models which account for site-rate heterogeneity (LG+C10 ...C60) [&lt;&lt;REF:Si Quang-2008-24-2317&gt;&gt;], empirical amino acid frequencies (+F), and accounting for across-site rate variation using either a Gamma distribution (+G) [&lt;&lt;REF:Yang-1994-39-306&gt;&gt;] or the free rates model (+R) [&lt;&lt;REF:Yang-1995-139-993&gt;&gt;,&lt;&lt;REF:Soubrier-2012-29-3345&gt;&gt;] with 10,000 ultrafast bootstrap replicates. The best fitting model according to BIC was LG+C30+F+G for the final SNX1 and SNX2 tree (see S4 Fig) and LG+C20+F+G for SNX27 and VPS26 (alignments and maximum likelihood trees in S4 Fig).\n\nSupporting information\nS1 Fig\nWestern blot showing the levels of GLUT1 in parental HeLa, VPS35 KO, and SNX27KO not transfected or transiently transfected with GFP-SNX27 WT, GFP-SNX27 Delta67-77, and GFP-SNX27 DeltaFERM.\nThe blot is representative of 3 independent assays. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. KO, knockout; SNX27, sorting nexin-27; WT, wild-type.\n(TIF)\nClick here for additional data file.\nS2 Fig\n(A) Co-immunoprecipitation of GFP-tagged extended amino-terminal regions (NT) of SNX1 and SNX2 alongside the GFP-tagged short amino-terminal regions of SNX5 and SNX6 expressed in HEK293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation and the immunoprecipitates were blotted for SNX27 and GFP. The blot is representative of 3 independent GFP traps. (B) Co-immunoprecipitation of GFP-tagged SNX2 WT or SNX2 KK (D27K, D72K) expressed in HEK293T cells. The cell lysates were subjected to GFP trap-based immunoprecipitation and the immunoprecipitates were blotted for SNX27, SNX6, and GFP. The blot is representative of 4 independent GFP traps. (C) Competitive ITC assay of SNX216-33 and mSNX27FL preincubated with 2-fold molar excess of SNX175-92 peptide showing that binding is blocked by the competing SNX175-92 peptide. Molecular masses are given in kilodaltons. Bars, error bars, and symbols represent the mean, SEM, and individual data points, respectively. *P &lt;  0.05, **P &lt;  0.01, ***P &lt;  0.001, ****P &lt;  0.0001, ns = not significant. The data underlying the graphs shown in the figure can be found in S1 Data. SNX1, sorting nexin-1; SNX5, sorting nexin-5; SNX6, sorting nexin-6; SNX27, sorting nexin-27; WT, wild-type.\n(TIF)\nClick here for additional data file.\nS3 Fig\n(A) SNX27 surface colored according to sequence conservation. The peptides are predicted to bind to a highly conserved pocket in SNX27. (B, C) As described in methods, 12 models of SNX27 were generated in total, consisting of 3 models each of SNX27 in association with the 4 aDLF sequences from human SNX1 and SNX2. B shows the overlay of all 12 models in Calpha trace representation colored by the pLDDT score. The pLDDT is a per-residue confidence score between 0 (lowest confidence) and 100 (highest confidence). The majority of the SNX27 structures show very high pLDDT scores and overlay with very high precision. C shows only the best 8 models where the core aDLF sequences have identical binding conformations. aDLF, acidic-Asp-Leu-Phe; pLDDT, predicted Local Distance Difference Test; SNX1, sorting nexin-1; SNX2, sorting nexin-2; SNX27, sorting nexin-27.\n(TIF)\nClick here for additional data file.\nS4 Fig\nMaximum likelihood phylogenies of (A) SNX27, (B) VPS26A/B and (C) VPS5/SNX1/SNX2. (A) SNX27 (green) with SNX17 (red) used as an outgroup with representatives across Filozoa. Amino acids aligned with Homo sapiens in positions 437, 495, 496, 498, and 501 are written next to the species name. (B) VPS26A, VPS26B, and VPS26C, with VPS26C in red and VPS26B in green; the duplication event occurring in the ancestral vertebrate is in blue. Interestingly, we also find another independent duplication of VPS26B in plants evidenced by Arabidopsis thaliana and Quercus suber in this tree. (C) The SNX1 sequences across choanozoans, with the duplication in vertebrates in blue. The number of aDLF motifs in the amino terminus is written next to the species name. Phylogenies were inferred under the best-fit model as determined by BIC. Branch support values are percentages based on 10,000 ultrafast bootstraps. aDLF, acidic-Asp-Leu-Phe; BIC, Bayesian information criterion; SNX1, sorting nexin-1; SNX27, sorting nexin-27.\n(TIF)\nClick here for additional data file.\nS1 Table\nSummary of ITC binding parameters.\nITC, isothermal titration calorimetry.\n(PDF)\nClick here for additional data file.\nS1 Raw Images\nUnprocessed images for blots shown in the paper.\n(PDF)\nClick here for additional data file.\nS1 Data\nNumerical data for Figs 1B, 1C &lt;&lt;FIG:pbio.3001601.g001&gt;&gt;, 2C &lt;&lt;FIG:pbio.3001601.g002&gt;&gt;, 3A, 3C, 3F &lt;&lt;FIG:pbio.3001601.g003&gt;&gt;, 4D, 4G &lt;&lt;FIG:pbio.3001601.g004&gt;&gt;, 5A, 5B, 5C, 5D, 5E &lt;&lt;FIG:pbio.3001601.g005&gt;&gt;, 6A, 6B, 6C, 6D and 6E &lt;&lt;FIG:pbio.3001601.g006&gt;&gt; and S1A and S2B.\n(XLSX)\nClick here for additional data file.'''\nThe text is taken from a scientific paper that uses Cryo-Electron Tomography (CryoET) to Study the structure of a biological sample at near atomic resolution after rapid vitrification to preseve the sample in a near-native state..\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. List all different types of biological sample being imaged using CryoEM (i.e., any biological samples that were mounted on grids and subsequently imaged). Select values from the following list: cell, tissue, organoid, organ, whole organism, virus, micro-organism, other.\n Record this value in the 'sample_type' field of the output.\n2. What is the type sample/gird preparation being developed? Choose from one of the following types: 'subtomogram averaging', 'tomography', 'helical', or 'singleParticle'. If you are unsure, please specify 'Unknown'.\n Record this value in the 'sample_preparation_type' field of the output.\n3. What was the pH of the buffer used to store the sample being imaged? If you cannot find the pH used, respond 'Unknown'.\n Record this value in the 'sample_preparation_buffer_ph' field of the output.\n4. What combination of gases acted as the medium used to vitrify the sample? Select any combination of the following: ETHANE, PROPANE, NITROGEN, OXYGEN, ARGON. Write the answer as a list. \n Record this value in the 'grid_vitrification_cryogen' field of the output.\n5. What cryo-protectant was used to prepare the sample for vitrification? If no cryo-protectant was used, respond 'None'.\n Record this value in the 'sample_preparation_cryo_protectant' field of the output.\n6. What model was used to prepare the grid for imaging? These models are specified as different types of UltrAuFoil, C-flat, Quantifoil, Homemade and include a short string like 'R 2/2' to denote the dimensions of holes. Records might be 'C-Flat R 2/2' or 'UltrAuFoil R 3/2' Please include all details as written in the paper. If you cannot find the model, respond 'Unknown'. \n Record this value in the 'grid_model' field of the output.\n7. What metal or combination of metals was used to make the grid? Select from the following list of metals: COPPER, GOLD, RHODIUM, MOLYBDENUM, PALLADIUM. If a grid is made of more than one material, captialize the metals and separate them with a forward slash (e.g., 'COPPER/PALLADIUM'). The paper may use chemical symbols such as 'Cu' for copper and 'Au' for gold. \n Record this value in the 'grid_material' field of the output.\n8. What is the mesh number of the grid used to prepare the sample for imaging? If you cannot find the mesh number, respond 'Unknown'.\n Record this value in the 'grid_mesh' field of the output.\n9. What topology is used in the grid. Select from the following list: HOLEY, CONTINUOUS, HOLEY ARRAY, or LACEY. If the grid topology is not described, respond 'Unknown'.\n Record this value in the 'grid_support_topology' field of the output.\n10. What was the temperature of the grid during vitrification? Include the temperature scale used in the paper (e.g., Kelvin). If you cannot find the temperature, respond 'Unknown'.\n Record this value in the 'grid_vit_ctemp' field of the output.\n11. What was the humidity used in the grid chamber during vitrification? If you cannot find the humidity, respond 'Unknown'. Record this value in the 'grid_vit_chumid' field of the output.\n12. If the biological sample was taken from an organism, provide the scientific name of the organism. If the biological sample is not from an organism, respond 'None'.\n Record this value in the 'organism_name' field of the output.\n13. If the sample being imaged originates from specified tissue from a multicellular animal, provide the name of the tissue? If the sample originates from another source, respond 'None'.\n Record this value in the 'tissue' field of the output.\n14. If the biological sample described in the text is a cell or cell culture, what was the cell type, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_type' field of the output.\n15. If the biological sample described in the text is a cell or cell culture, what was the cell strain, if known? If not known, respond 'Unknown'.\n Record this value in the 'cell_strain' field of the output.\n16. If the preparation or imaging process involves named subcellular structures (ribosomes, cellular nuclei, cell membranes, specific vesicles, etc), please list them here. If cellular subcomponents are not mentioned, respond 'None'.\n Record this value in the 'cell_component' field of the output.\nGenerate only JSON formatted output with 16 fields:\nsample_type, sample_preparation_type, sample_preparation_buffer_ph, grid_vitrification_cryogen, sample_preparation_cryo_protectant, grid_model, grid_material, grid_mesh, grid_support_topology, grid_vit_ctemp, grid_vit_chumid, organism_name, tissue, cell_type, cell_strain, cell_component\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [9.79s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"sample_type\": \"not present\",\n  \"sample_preparation_type\": \"not present\",\n  \"sample_preparation_buffer_ph\": \"Unknown\",\n  \"grid_vitrification_cryogen\": \"not present\",\n  \"sample_preparation_cryo_protectant\": \"None\",\n  \"grid_model\": \"Unknown\",\n  \"grid_material\": \"not present\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"Unknown\",\n  \"grid_vit_ctemp\": \"Unknown\",\n  \"grid_vit_chumid\": \"Unknown\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"Unknown\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"sample_type\": \"not present\",\n  \"sample_preparation_type\": \"not present\",\n  \"sample_preparation_buffer_ph\": \"Unknown\",\n  \"grid_vitrification_cryogen\": \"not present\",\n  \"sample_preparation_cryo_protectant\": \"None\",\n  \"grid_model\": \"Unknown\",\n  \"grid_material\": \"not present\",\n  \"grid_mesh\": \"Unknown\",\n  \"grid_support_topology\": \"Unknown\",\n  \"grid_vit_ctemp\": \"Unknown\",\n  \"grid_vit_chumid\": \"Unknown\",\n  \"organism_name\": \"None\",\n  \"tissue\": \"None\",\n  \"cell_type\": \"Unknown\",\n  \"cell_strain\": \"Unknown\",\n  \"cell_component\": \"None\"\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 164,
      "prompt_tokens": 24604,
      "total_tokens": 24768
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_94f711dcf6"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [0ms] Exiting Parser run with output:
{
  "sample_type": "not present",
  "sample_preparation_type": "not present",
  "sample_preparation_buffer_ph": "Unknown",
  "grid_vitrification_cryogen": "not present",
  "sample_preparation_cryo_protectant": "None",
  "grid_model": "Unknown",
  "grid_material": "not present",
  "grid_mesh": "Unknown",
  "grid_support_topology": "Unknown",
  "grid_vit_ctemp": "Unknown",
  "grid_vit_chumid": "Unknown",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "Unknown",
  "cell_strain": "Unknown",
  "cell_component": "None"
}
[chain/end] [1:chain:RunnableSequence] [9.80s] Exiting Chain run with output:
{
  "sample_type": "not present",
  "sample_preparation_type": "not present",
  "sample_preparation_buffer_ph": "Unknown",
  "grid_vitrification_cryogen": "not present",
  "sample_preparation_cryo_protectant": "None",
  "grid_model": "Unknown",
  "grid_material": "not present",
  "grid_mesh": "Unknown",
  "grid_support_topology": "Unknown",
  "grid_vit_ctemp": "Unknown",
  "grid_vit_chumid": "Unknown",
  "organism_name": "None",
  "tissue": "None",
  "cell_type": "Unknown",
  "cell_strain": "Unknown",
  "cell_component": "None"
}</code></pre>
</div>
<div class="cell-output cell-output-display" data-execution_count="8">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">sample_type</th>
<th data-quarto-table-cell-role="th">sample_preparation_type</th>
<th data-quarto-table-cell-role="th">sample_preparation_buffer_ph</th>
<th data-quarto-table-cell-role="th">grid_vitrification_cryogen</th>
<th data-quarto-table-cell-role="th">sample_preparation_cryo_protectant</th>
<th data-quarto-table-cell-role="th">grid_model</th>
<th data-quarto-table-cell-role="th">grid_material</th>
<th data-quarto-table-cell-role="th">grid_mesh</th>
<th data-quarto-table-cell-role="th">grid_support_topology</th>
<th data-quarto-table-cell-role="th">grid_vit_ctemp</th>
<th data-quarto-table-cell-role="th">grid_vit_chumid</th>
<th data-quarto-table-cell-role="th">organism_name</th>
<th data-quarto-table-cell-role="th">tissue</th>
<th data-quarto-table-cell-role="th">cell_type</th>
<th data-quarto-table-cell-role="th">cell_strain</th>
<th data-quarto-table-cell-role="th">cell_component</th>
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th">extraction_type</th>
<th data-quarto-table-cell-role="th">run_label</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>not present</td>
<td>Unknown</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>doi:10.1073/pnas.2210249119</td>
<td>cryoet</td>
<td>test</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">0</td>
<td>[micro-organism, virus, other]</td>
<td>Unknown</td>
<td>Unknown</td>
<td>[ETHANE, NITROGEN]</td>
<td>glucose</td>
<td>Unknown</td>
<td>GOLD</td>
<td>Unknown</td>
<td>HOLEY</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>doi:10.1093/jmicro/dfab041</td>
<td>cryoet</td>
<td>test</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>micro-organism</td>
<td>subtomogram averaging</td>
<td>Unknown</td>
<td>[ETHANE]</td>
<td>None</td>
<td>Unknown</td>
<td>GOLD</td>
<td>Unknown</td>
<td>HOLEY</td>
<td>Unknown</td>
<td>Unknown</td>
<td>Tetrahymena thermophila</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>None</td>
<td>doi:10.1101/2023.11.28.569001</td>
<td>cryoet</td>
<td>test</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">0</td>
<td>micro-organism</td>
<td>tomography</td>
<td>6.8</td>
<td>[ETHANE]</td>
<td>None</td>
<td>Quantifoil R2/1</td>
<td>COPPER</td>
<td>Unknown</td>
<td>HOLEY</td>
<td>30 degreesC</td>
<td>95%</td>
<td>None</td>
<td>None</td>
<td>HEK293T</td>
<td>Unknown</td>
<td>None</td>
<td>doi:10.1101/2024.03.25.586532</td>
<td>cryoet</td>
<td>test</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>not present</td>
<td>not present</td>
<td>Unknown</td>
<td>not present</td>
<td>None</td>
<td>Unknown</td>
<td>not present</td>
<td>Unknown</td>
<td>Unknown</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>doi:10.1371/journal.pbio.3001601</td>
<td>cryoet</td>
<td>test</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="cell" data-execution_count="10">
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>ldb.create_zip_archive_of_full_text_files(<span class="st">'2'</span>, loc<span class="op">+</span><span class="st">'/'</span><span class="op">+</span>db_name<span class="op">+</span><span class="st">'/full_text_files.zip'</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="9">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a>q3 <span class="op">=</span> ldb.session.query(SKE.<span class="bu">id</span>, N.name, N.provenance, N.content) <span class="op">\</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">id</span> <span class="op">==</span> NIA.Note_id) <span class="op">\</span></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(NIA.is_about_id <span class="op">==</span> SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">type</span> <span class="op">==</span> <span class="st">'MetadataExtractionNote'</span>) </span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> row <span class="kw">in</span> q3.<span class="bu">all</span>():</span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a>    paper <span class="op">=</span> row[<span class="dv">0</span>]</span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a>    name <span class="op">=</span> row[<span class="dv">1</span>]</span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a><span class="co">#    provenance = json.loads(row[2])</span></span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a>    result <span class="op">=</span> json.loads(row[<span class="dv">3</span>])</span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>    kv <span class="op">=</span> {k:result[k] <span class="cf">for</span> k <span class="kw">in</span> result}</span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a>    kv[<span class="st">'DOI'</span>] <span class="op">=</span> paper</span>
<span id="cb38-13"><a href="#cb38-13" aria-hidden="true" tabindex="-1"></a>    kv[<span class="st">'run'</span>] <span class="op">=</span> name</span>
<span id="cb38-14"><a href="#cb38-14" aria-hidden="true" tabindex="-1"></a>    l.append(kv)</span>
<span id="cb38-15"><a href="#cb38-15" aria-hidden="true" tabindex="-1"></a><span class="co"># create a dataframe from the list of dictionaries with DOI as the index column</span></span>
<span id="cb38-16"><a href="#cb38-16" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> <span class="bu">len</span>(l)<span class="op">&gt;</span><span class="dv">0</span>:</span>
<span id="cb38-17"><a href="#cb38-17" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.DataFrame(l).set_index([<span class="st">'DOI'</span>, <span class="st">'run'</span>])</span>
<span id="cb38-18"><a href="#cb38-18" aria-hidden="true" tabindex="-1"></a><span class="cf">else</span>: </span>
<span id="cb38-19"><a href="#cb38-19" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.DataFrame()</span>
<span id="cb38-20"><a href="#cb38-20" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="9">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">sample_type</th>
<th data-quarto-table-cell-role="th">sample_preparation_type</th>
<th data-quarto-table-cell-role="th">sample_preparation_buffer_ph</th>
<th data-quarto-table-cell-role="th">grid_vitrification_cryogen</th>
<th data-quarto-table-cell-role="th">sample_preparation_cryo_protectant</th>
<th data-quarto-table-cell-role="th">grid_model</th>
<th data-quarto-table-cell-role="th">grid_material</th>
<th data-quarto-table-cell-role="th">grid_mesh</th>
<th data-quarto-table-cell-role="th">grid_support_topology</th>
<th data-quarto-table-cell-role="th">grid_vit_ctemp</th>
<th data-quarto-table-cell-role="th">grid_vit_chumid</th>
<th data-quarto-table-cell-role="th">organism_name</th>
<th data-quarto-table-cell-role="th">tissue</th>
<th data-quarto-table-cell-role="th">cell_type</th>
<th data-quarto-table-cell-role="th">cell_strain</th>
<th data-quarto-table-cell-role="th">cell_component</th>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">DOI</th>
<th data-quarto-table-cell-role="th">run</th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1073/pnas.2210249119</td>
<td data-quarto-table-cell-role="th">cryoet__doi:10.1073/pnas.2210249119__test</td>
<td>not present</td>
<td>Unknown</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>not present</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">doi:10.1093/jmicro/dfab041</td>
<td data-quarto-table-cell-role="th">cryoet__doi:10.1093/jmicro/dfab041__test</td>
<td>[micro-organism, virus, other]</td>
<td>Unknown</td>
<td>Unknown</td>
<td>[ETHANE, NITROGEN]</td>
<td>glucose</td>
<td>Unknown</td>
<td>GOLD</td>
<td>Unknown</td>
<td>HOLEY</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1101/2023.11.28.569001</td>
<td data-quarto-table-cell-role="th">cryoet__doi:10.1101/2023.11.28.569001__test</td>
<td>micro-organism</td>
<td>subtomogram averaging</td>
<td>Unknown</td>
<td>[ETHANE]</td>
<td>None</td>
<td>Unknown</td>
<td>GOLD</td>
<td>Unknown</td>
<td>HOLEY</td>
<td>Unknown</td>
<td>Unknown</td>
<td>Tetrahymena thermophila</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>None</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">doi:10.1101/2024.03.25.586532</td>
<td data-quarto-table-cell-role="th">cryoet__doi:10.1101/2024.03.25.586532__test</td>
<td>micro-organism</td>
<td>tomography</td>
<td>6.8</td>
<td>[ETHANE]</td>
<td>None</td>
<td>Quantifoil R2/1</td>
<td>COPPER</td>
<td>Unknown</td>
<td>HOLEY</td>
<td>30 degreesC</td>
<td>95%</td>
<td>None</td>
<td>None</td>
<td>HEK293T</td>
<td>Unknown</td>
<td>None</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1371/journal.pbio.3001601</td>
<td data-quarto-table-cell-role="th">cryoet__doi:10.1371/journal.pbio.3001601__test</td>
<td>not present</td>
<td>not present</td>
<td>Unknown</td>
<td>not present</td>
<td>None</td>
<td>Unknown</td>
<td>not present</td>
<td>Unknown</td>
<td>Unknown</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
<td>None</td>
<td>Unknown</td>
<td>Unknown</td>
<td>None</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="cell" data-execution_count="11">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="co"># USE WITH </span><span class="al">CAUTION</span><span class="co"> - this will delete all extracted metadata notes in the database</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a><span class="co"># clear all notes across papers listed in `dois` list</span></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> OK_TO_DELETE:</span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> row <span class="kw">in</span> q3.<span class="bu">all</span>():</span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>        d_id <span class="op">=</span> row[<span class="dv">0</span>]</span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>        e <span class="op">=</span> ldb.session.query(SKE).<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>d_id).first()</span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>        notes_to_delete <span class="op">=</span> []</span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>        <span class="cf">for</span> n <span class="kw">in</span> ldb.read_notes_about_x(e):</span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>            notes_to_delete.append(n.<span class="bu">id</span>)</span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>        <span class="cf">for</span> n <span class="kw">in</span> notes_to_delete:</span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>            ldb.delete_note(n)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>