<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.555">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="description" content="Methods to extract and model how imaging technology evolves.">

<title>alhazen - Image Tech Evolution</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script><script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>

<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="../../styles.css">
<meta property="og:title" content="alhazen - Image Tech Evolution">
<meta property="og:description" content="Methods to extract and model how imaging technology evolves.">
<meta property="og:site_name" content="alhazen">
<meta name="twitter:title" content="alhazen - Image Tech Evolution">
<meta name="twitter:description" content="Methods to extract and model how imaging technology evolves.">
<meta name="twitter:card" content="summary">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">alhazen</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html#install-dependencies"> 
<span class="menu-text">Get Started</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../../docnb1_use_cases.html" aria-current="page"> 
<span class="menu-text">Use Cases</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../docnb2_architecture.html"> 
<span class="menu-text">Platform Architecture</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-help" role="button" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Help</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-help">    
        <li>
    <a class="dropdown-item" href="https://github.com/chanzuckerberg/alhazen/issues"><i class="bi bi-bug" role="img">
</i> 
 <span class="dropdown-text">Report an Issue</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/chanzuckerberg/alhazen"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../cookbook/single_doc_extraction/index.html">Example Notebooks</a></li><li class="breadcrumb-item"><a href="../../cookbook/trend_analysis/index.html">Trend Analysis</a></li><li class="breadcrumb-item"><a href="../../cookbook/trend_analysis/imaging_technology_evolution.html">Image Tech Evolution</a></li></ol></nav>
        <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Home - Alhazen</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../docnb1_use_cases.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Use Cases</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../docnb2_architecture.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Platform Architecture</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Tech Elements</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../agents/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Agents</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../agents/agent.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Alhazen Agent / Chatbot.</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../tools/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/toolkit.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Alhazen Toolkit</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/basic_tools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Tools for Alhazen</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/metadata_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Methods Metadata Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/protocol_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Protocol Workflow Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/paperqa_emulation_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Paper QA Emulation Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_clasifier_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Title / Abstract Classifier Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_mapping_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Title / Abstract Mapping Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Title / Abstract Extraction Tool</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../utils/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Utilities</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/ceifns_db.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Database for Scientific Knowledge</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/search_engine_eutils.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Search Engine Tools</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/web_robot.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Web Robots</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/html_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">HTML Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/jats_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">JATS Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/pdf_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PDF Text Extractor Utility</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../tutorials/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tutorial Notebooks</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tutorials/cryoet_tutorial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CryoET Tutorial</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true">
 <span class="menu-text">Example Notebooks</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/single_doc_extraction/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Metadata Extraction</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/cellxgene_datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CellXGene Datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/cryoet.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CryoET</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/ncats_natural_history_studies.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">NCATS Natural History Studies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/rnaquarium.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">RNAquarium</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/trend_analysis/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Trend Analysis</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/capacity_building.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Capacity Building</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/imaging_technology_evolution.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">Image Tech Evolution</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/pathogen_landscaping.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pathogen_Landscaping</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/rare_disease_literature.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Rare Disease Literature.</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/key_opinion_leaders/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Key Opinion Leaders</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-9" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-9" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/key_opinion_leaders/african_microscopy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">African Microscopy</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/question_answering/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Question Answering</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-10" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-10" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/question_answering/virtualcell_corepapers.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Virtual Cell Landscaping Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/question_answering/cellbiology.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Cell Biology Database.</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#modeling-technological-evolution-and-innovation" id="toc-modeling-technological-evolution-and-innovation" class="nav-link active" data-scroll-target="#modeling-technological-evolution-and-innovation">Modeling Technological Evolution and Innovation</a>
  <ul class="collapse">
  <li><a href="#python-imports" id="toc-python-imports" class="nav-link" data-scroll-target="#python-imports">Python Imports</a></li>
  <li><a href="#environment-variables" id="toc-environment-variables" class="nav-link" data-scroll-target="#environment-variables">Environment Variables</a></li>
  <li><a href="#setup-utils-agents-and-tools" id="toc-setup-utils-agents-and-tools" class="nav-link" data-scroll-target="#setup-utils-agents-and-tools">Setup utils, agents, and tools</a></li>
  </ul></li>
  <li><a href="#building-the-database" id="toc-building-the-database" class="nav-link" data-scroll-target="#building-the-database">Building the database</a>
  <ul class="collapse">
  <li><a href="#scripts-to-build-delete-the-database" id="toc-scripts-to-build-delete-the-database" class="nav-link" data-scroll-target="#scripts-to-build-delete-the-database">Scripts to Build / Delete the database</a></li>
  <li><a href="#build-ceifns-database-from-queries" id="toc-build-ceifns-database-from-queries" class="nav-link" data-scroll-target="#build-ceifns-database-from-queries">Build CEIFNS database from queries</a></li>
  </ul></li>
  <li><a href="#analyze-collections" id="toc-analyze-collections" class="nav-link" data-scroll-target="#analyze-collections">Analyze Collections</a></li>
  <li><a href="#run-llm-over-each-paper-in-imaging-program-collection-to-determine-if-papers-are-methods-or-applications" id="toc-run-llm-over-each-paper-in-imaging-program-collection-to-determine-if-papers-are-methods-or-applications" class="nav-link" data-scroll-target="#run-llm-over-each-paper-in-imaging-program-collection-to-determine-if-papers-are-methods-or-applications">Run LLM over each paper in Imaging Program collection to determine if papers are methods or applications</a></li>
  <li><a href="#run-the-tool-for-high-level-subtypes" id="toc-run-the-tool-for-high-level-subtypes" class="nav-link" data-scroll-target="#run-the-tool-for-high-level-subtypes">Run the tool for high-level subtypes</a></li>
  </ul>
<div class="toc-actions"><ul><li><a href="https://github.com/chanzuckerberg/alhazen/issues/new" class="toc-action"><i class="bi bi-github"></i>Report an issue</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../cookbook/single_doc_extraction/index.html">Example Notebooks</a></li><li class="breadcrumb-item"><a href="../../cookbook/trend_analysis/index.html">Trend Analysis</a></li><li class="breadcrumb-item"><a href="../../cookbook/trend_analysis/imaging_technology_evolution.html">Image Tech Evolution</a></li></ol></nav>
<div class="quarto-title">
<h1 class="title">Image Tech Evolution</h1>
</div>

<div>
  <div class="description">
    Methods to extract and model how imaging technology evolves.
  </div>
</div>


<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
<section id="modeling-technological-evolution-and-innovation" class="level2">
<h2 class="anchored" data-anchor-id="modeling-technological-evolution-and-innovation">Modeling Technological Evolution and Innovation</h2>
<p>This notebook is concerned with building a digital library of publications derived from four subdisciplines of biomedical imaging:</p>
<ol type="1">
<li>Cryo-Electron Tomography</li>
<li>Volume Electron Microscopy</li>
<li>Hiercharchy Phase Contrast Tomography</li>
<li>Photoacoustic Imaging</li>
</ol>
<section id="python-imports" class="level3">
<h3 class="anchored" data-anchor-id="python-imports">Python Imports</h3>
<p>Setting python imports, environment variables, and other crucial set up parameters here.</p>
<div id="cell-4" class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.core <span class="im">import</span> get_langchain_chatmodel, MODEL_TYPE</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.agent <span class="im">import</span> AlhazenAgent</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.schema_sqla <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.basic <span class="im">import</span> AddCollectionFromEPMCTool, DeleteCollectionTool</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.paperqa_emulation_tool <span class="im">import</span> PaperQAEmulationTool</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.metadata_extraction_tool <span class="im">import</span> <span class="op">*</span> </span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.protocol_extraction_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.tiab_classifier_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.toolkit <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.ceifns_db <span class="im">import</span> Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, restore_ceifns_database</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.searchEngineUtils <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.callbacks.tracers <span class="im">import</span> ConsoleCallbackHandler</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.docstore.document <span class="im">import</span> Document</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.text_splitter <span class="im">import</span> CharacterTextSplitter</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.vectorstores.pgvector <span class="im">import</span> PGVector</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_community.chat_models.ollama <span class="im">import</span> ChatOllama</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_google_vertexai <span class="im">import</span> ChatVertexAI</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_openai <span class="im">import</span> ChatOpenAI</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> bs4 <span class="im">import</span> BeautifulSoup,Tag,Comment,NavigableString</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> databricks <span class="im">import</span> sql</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> datetime <span class="im">import</span> datetime</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> importlib_resources <span class="im">import</span> files</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> re</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> requests</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy <span class="im">import</span> create_engine, exists, func, or_, and_, not_, desc, asc</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy.orm <span class="im">import</span> sessionmaker, aliased</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> time <span class="im">import</span> time,sleep</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> tqdm <span class="im">import</span> tqdm</span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.request <span class="im">import</span> urlopen</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.parse <span class="im">import</span> quote_plus, quote, unquote</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.error <span class="im">import</span> URLError, HTTPError</span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> yaml</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.searchEngineUtils <span class="im">import</span> load_paper_from_openalex, read_references_from_openalex </span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pyalex <span class="im">import</span> config, Works, Work</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a>config.email <span class="op">=</span> <span class="st">"gully.burns@chanzuckerberg.com"</span></span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> local_resources.queries.imaging_tech <span class="im">as</span> imaging_tech</span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.queryTranslator <span class="im">import</span> QueryTranslator, QueryType</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div id="cell-5" class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Using Aliases like this massively simplifies the use of SQLAlchemy</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>IR <span class="op">=</span> aliased(InformationResource)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>SKC <span class="op">=</span> aliased(ScientificKnowledgeCollection)</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>SKC_HM <span class="op">=</span> aliased(ScientificKnowledgeCollectionHasMembers)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>SKE <span class="op">=</span> aliased(ScientificKnowledgeExpression)</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>SKE_XREF <span class="op">=</span> aliased(ScientificKnowledgeExpressionXref)</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>SKE_IRI <span class="op">=</span> aliased(ScientificKnowledgeExpressionIri)</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>SKE_HR <span class="op">=</span> aliased(ScientificKnowledgeExpressionHasRepresentation)</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>SKE_MO <span class="op">=</span> aliased(ScientificKnowledgeExpressionMemberOf)</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>SKI <span class="op">=</span> aliased(ScientificKnowledgeItem)</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>SKI_HP <span class="op">=</span> aliased(ScientificKnowledgeItemHasPart)</span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a>SKF <span class="op">=</span> aliased(ScientificKnowledgeFragment)</span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a>N <span class="op">=</span> aliased(Note)</span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a>NIA <span class="op">=</span> aliased(NoteIsAbout)</span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>SKC_HN <span class="op">=</span> aliased(ScientificKnowledgeCollectionHasNotes)</span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a>SKE_HN <span class="op">=</span> aliased(ScientificKnowledgeExpressionHasNotes)</span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a>SKI_HN <span class="op">=</span> aliased(ScientificKnowledgeItemHasNotes)</span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a>SKF_HN <span class="op">=</span> aliased(ScientificKnowledgeFragmentHasNotes)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="environment-variables" class="level3">
<h3 class="anchored" data-anchor-id="environment-variables">Environment Variables</h3>
<p>Remember to set environmental variables for this code:</p>
<ul>
<li><code>ALHAZEN_DB_NAME</code> - the name of the PostGresQL database you are storing information into</li>
<li><code>LOCAL_FILE_PATH</code> - the location on disk where you save temporary files, downloaded models or other data.</li>
</ul>
<div id="cell-7" class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.environ.get(<span class="st">'LOCAL_FILE_PATH'</span>) <span class="kw">is</span> <span class="va">None</span>: </span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>    <span class="cf">raise</span> <span class="pp">Exception</span>(<span class="st">'Where are you storing your local literature database?'</span>)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.path.exists(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]) <span class="kw">is</span> <span class="va">False</span>:</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>    os.makedirs(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>])    </span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>loc <span class="op">=</span> os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>db_name <span class="op">=</span> <span class="st">'imaging_tech_innovation'</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="setup-utils-agents-and-tools" class="level3">
<h3 class="anchored" data-anchor-id="setup-utils-agents-and-tools">Setup utils, agents, and tools</h3>
<div id="cell-9" class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>ldb <span class="op">=</span> Ceifns_LiteratureDb(loc<span class="op">=</span>loc, name<span class="op">=</span>db_name)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>llm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'mixtral:instruct'</span>) </span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>llm2 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-4-1106-preview'</span>) </span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>llm3 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-3.5-turbo'</span>) </span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a><span class="co">#llm3 = ChatVertexAI(model_name="gemini-pro", convert_system_message_to_human=True)</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>cb <span class="op">=</span> AlhazenAgent(llm2, llm2)</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'AGENT TOOLS'</span>)</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools():</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="st">'</span><span class="ch">\t</span><span class="st">'</span><span class="op">+</span><span class="bu">type</span>(t).<span class="va">__name__</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>AGENT TOOLS
    AddCollectionFromEPMCTool
    AddAuthorsToCollectionTool
    DescribeCollectionCompositionTool
    DeleteCollectionTool
    RetrieveFullTextTool
    RetrieveFullTextToolForACollection
    MetadataExtraction_EverythingEverywhere_Tool
    SimpleExtractionWithRAGTool
    PaperQAEmulationTool
    ProcotolEntitiesExtractionTool
    CheckExpressionTool
    TitleAbstractClassifier_OneDocAtATime_Tool</code></pre>
</div>
</div>
</section>
</section>
<section id="building-the-database" class="level2">
<h2 class="anchored" data-anchor-id="building-the-database">Building the database</h2>
<section id="scripts-to-build-delete-the-database" class="level3">
<h3 class="anchored" data-anchor-id="scripts-to-build-delete-the-database">Scripts to Build / Delete the database</h3>
<p>If you need to restore a deleted database from backup, use the following shell commands:</p>
<pre><code>$ createdb em_tech
$ psql -d em_tech -f /local/file/path/em_tech/backup&lt;date_time&gt;.sql</code></pre>
<div id="cell-12" class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>drop_ceifns_database(os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
    - Avoid using `tokenizers` before the fork if possible
    - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
Database has been backed up to /users/gully.burns/alhazen/imaging_tech_innovation/backup2024-03-04-23-34-21.sql
Database has been dropped successfully !!</code></pre>
</div>
</div>
<div id="cell-13" class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>create_ceifns_database(os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 310/310 [00:00&lt;00:00, 3467.45it/s]</code></pre>
</div>
</div>
</section>
<section id="build-ceifns-database-from-queries" class="level3">
<h3 class="anchored" data-anchor-id="build-ceifns-database-from-queries">Build CEIFNS database from queries</h3>
<section id="run-queries-on-european-pmc-based-on-innovation-categories" class="level4">
<h4 class="anchored" data-anchor-id="run-queries-on-european-pmc-based-on-innovation-categories">Run queries on European PMC based on innovation categories</h4>
<p>Here we build general corpora across the categories of interest.</p>
<ul>
<li>Hierarchical phase-contrast tomography</li>
<li>Cryo-Electron Tomography</li>
<li>Volume Electron Microscopy</li>
<li>Photoacoustic imaging</li>
</ul>
<div id="cell-16" class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>cols_to_include <span class="op">=</span> [<span class="st">'ID'</span>, <span class="st">'CORPUS_NAME'</span>, <span class="st">'QUERY'</span>]</span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.read_csv(files(imaging_tech).joinpath(<span class="st">'imaging_tech.tsv'</span>), sep<span class="op">=</span><span class="st">'</span><span class="ch">\t</span><span class="st">'</span>, )</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> df.drop(columns<span class="op">=</span>[c <span class="cf">for</span> c <span class="kw">in</span> df.columns <span class="cf">if</span> c <span class="kw">not</span> <span class="kw">in</span> cols_to_include])</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>


<table class="dataframe caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">ID</th>
<th data-quarto-table-cell-role="th">CORPUS_NAME</th>
<th data-quarto-table-cell-role="th">QUERY</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>1</td>
<td>Hierarchical phase-contrast tomography</td>
<td>Hierarchical phase-contrast tomography | HIP-C...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td>2</td>
<td>Cryo-Electron Tomography</td>
<td>Cryoelectron Tomography | Cryo Electron Tomogr...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td>3</td>
<td>Volume Electron Microscopy</td>
<td>Volume Electron Microscopy | Volume EM | (seri...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">3</td>
<td>4</td>
<td>Photoacoustic imaging</td>
<td>Photoacoustic imaging | Photoacoustic microscopy</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div id="cell-17" class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>qt <span class="op">=</span> QueryTranslator(df.sort_values(<span class="st">'ID'</span>), <span class="st">'ID'</span>, <span class="st">'QUERY'</span>, <span class="st">'CORPUS_NAME'</span>)</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>(corpus_ids, epmc_queries) <span class="op">=</span> qt.generate_queries(QueryType.epmc, sections<span class="op">=</span>[<span class="st">'TITLE_ABS'</span>, <span class="st">'METHODS'</span>])</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>corpus_names <span class="op">=</span> df[<span class="st">'CORPUS_NAME'</span>]</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (<span class="bu">id</span>, name, query) <span class="kw">in</span> <span class="bu">zip</span>(corpus_ids, corpus_names, epmc_queries):</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>    addEMPCCollection_tool.run(tool_input<span class="op">=</span>{<span class="st">'id'</span>: <span class="bu">id</span>, <span class="st">'name'</span>:name, <span class="st">'query'</span>:query, <span class="st">'full_text'</span>:<span class="va">False</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 4/4 [00:00&lt;00:00, 7533.55it/s]
100%|██████████| 4/4 [00:00&lt;00:00, 3442.19it/s]
100%|██████████| 1/1 [00:03&lt;00:00,  3.54s/it]
100%|██████████| 135/135 [00:00&lt;00:00, 507.06it/s]
100%|██████████| 3/3 [00:55&lt;00:00, 18.45s/it]
100%|██████████| 2558/2558 [00:06&lt;00:00, 375.25it/s]
100%|██████████| 7/7 [02:28&lt;00:00, 21.18s/it]
100%|██████████| 6820/6820 [00:41&lt;00:00, 164.39it/s]
100%|██████████| 5/5 [00:59&lt;00:00, 11.84s/it]
100%|██████████| 4478/4478 [00:17&lt;00:00, 257.83it/s]</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=((TITLE_ABS:"Hierarchical phase-contrast tomography" OR METHODS:"Hierarchical phase-contrast tomography") OR (TITLE_ABS:"HIP-CT" OR METHODS:"HIP-CT") OR (TITLE_ABS:"Hierarchical phase contrast tomography" OR METHODS:"Hierarchical phase contrast tomography")), 143 European PMC PAPERS FOUND
 Returning 135
https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=((TITLE_ABS:"Cryoelectron Tomography" OR METHODS:"Cryoelectron Tomography") OR (TITLE_ABS:"Cryo Electron Tomography" OR METHODS:"Cryo Electron Tomography") OR (TITLE_ABS:"Cryo-Electron Tomography" OR METHODS:"Cryo-Electron Tomography") OR (TITLE_ABS:"Cryo-ET" OR METHODS:"Cryo-ET") OR (TITLE_ABS:"CryoET" OR METHODS:"CryoET")), 2581 European PMC PAPERS FOUND
 Returning 2558
https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=((TITLE_ABS:"Volume Electron Microscopy" OR METHODS:"Volume Electron Microscopy") OR (TITLE_ABS:"Volume EM" OR METHODS:"Volume EM") OR (TITLE_ABS:"multibeam SEM" OR METHODS:"multibeam SEM") OR (TITLE_ABS:"FAST-SEM" OR METHODS:"FAST-SEM") OR ((TITLE_ABS:"serial section" OR METHODS:"serial section") AND ((TITLE_ABS:"electron microscopy" OR METHODS:"electron microscopy") OR (TITLE_ABS:"EM" OR METHODS:"EM") OR (TITLE_ABS:"transmission electron microscopy" OR METHODS:"transmission electron microscopy") OR (TITLE_ABS:"TEM" OR METHODS:"TEM") OR (TITLE_ABS:"scanning electron microscopy" OR METHODS:"scanning electron microscopy") OR (TITLE_ABS:"SEM" OR METHODS:"SEM") OR (TITLE_ABS:"electron tomography" OR METHODS:"electron tomography"))) OR ((TITLE_ABS:"serial block-face" OR METHODS:"serial block-face") AND ((TITLE_ABS:"scanning electron microscopy" OR METHODS:"scanning electron microscopy") OR (TITLE_ABS:"SEM" OR METHODS:"SEM"))) OR ((TITLE_ABS:"focused ion beam" OR METHODS:"focused ion beam") AND ((TITLE_ABS:"scanning electron microscopy" OR METHODS:"scanning electron microscopy") OR (TITLE_ABS:"SEM" OR METHODS:"SEM"))) OR ((TITLE_ABS:"automated serial" OR METHODS:"automated serial") AND ((TITLE_ABS:"transmission electron microscopy" OR METHODS:"transmission electron microscopy") OR (TITLE_ABS:"TEM" OR METHODS:"TEM"))) OR ((TITLE_ABS:"massively parallel imaging" OR METHODS:"massively parallel imaging") AND ((TITLE_ABS:"scanning electron microscopy" OR METHODS:"scanning electron microscopy") OR (TITLE_ABS:"SEM" OR METHODS:"SEM")))), 6891 European PMC PAPERS FOUND
 Returning 6820
https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=((TITLE_ABS:"Photoacoustic imaging" OR METHODS:"Photoacoustic imaging") OR (TITLE_ABS:"Photoacoustic microscopy" OR METHODS:"Photoacoustic microscopy")), 4600 European PMC PAPERS FOUND
 Returning 4478</code></pre>
</div>
</div>
</section>
<section id="run-queries-on-known-lists-of-papers-from-czi-grantees-on-the-four-imaging-innovation-categories" class="level4">
<h4 class="anchored" data-anchor-id="run-queries-on-known-lists-of-papers-from-czi-grantees-on-the-four-imaging-innovation-categories">Run queries on known lists of papers from CZI grantees on the four imaging innovation categories</h4>
<p>Here we seach pre-developed lists of papers from CZI grantee’s work, indexed in a local file: <code>./local_resources/queries/imaging_tech/grantee_dois.json</code></p>
<div id="cell-19" class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="cf">with</span> <span class="bu">open</span>(files(imaging_tech).joinpath(<span class="st">'grantee_dois.json'</span>), <span class="st">'r'</span>) <span class="im">as</span> f:</span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>    dict_lists <span class="op">=</span> json.load(f)</span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> i, k <span class="kw">in</span> <span class="bu">enumerate</span>(dict_lists.keys()):</span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a>    query <span class="op">=</span> <span class="st">' OR '</span>.join([<span class="st">'doi:"'</span><span class="op">+</span>d_id<span class="op">+</span><span class="st">'"'</span> <span class="cf">for</span> d_id <span class="kw">in</span> dict_lists[k] ])</span>
<span id="cb15-7"><a href="#cb15-7" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="st">'</span><span class="sc">%s</span><span class="st">: Searching for </span><span class="sc">%d</span><span class="st">'</span><span class="op">%</span>(k, <span class="bu">len</span>(dict_lists[k])))</span>
<span id="cb15-8"><a href="#cb15-8" aria-hidden="true" tabindex="-1"></a>    addEMPCCollection_tool.run(tool_input<span class="op">=</span>{<span class="st">'id'</span>: <span class="bu">str</span>(<span class="dv">5</span><span class="op">+</span>i), <span class="st">'name'</span>: k <span class="op">+</span> <span class="st">' (grantees)'</span>, <span class="st">'query'</span>:query})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Cryo-Electron Tomography: Searching for 23
Volume Electron Microscopy: Searching for 12
Hierarchical phase-contrast tomography: Searching for 14
Photoacoustic imaging: Searching for 26</code></pre>
</div>
</div>
</section>
</section>
</section>
<section id="analyze-collections" class="level2">
<h2 class="anchored" data-anchor-id="analyze-collections">Analyze Collections</h2>
<div id="cell-21" class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKC.<span class="bu">id</span>, SKC.name, SKE.<span class="bu">id</span>, SKI.<span class="bu">type</span>) <span class="op">\</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>SKE_HR.ScientificKnowledgeExpression_id) <span class="op">\</span></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) </span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(q.<span class="bu">all</span>(), columns<span class="op">=</span>[<span class="st">'id'</span>, <span class="st">'collection name'</span>, <span class="st">'doi'</span>, <span class="st">'item type'</span>])    </span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>df.pivot_table(index<span class="op">=</span>[<span class="st">'id'</span>, <span class="st">'collection name'</span>], columns<span class="op">=</span><span class="st">'item type'</span>, values<span class="op">=</span><span class="st">'doi'</span>, aggfunc<span class="op">=</span><span class="kw">lambda</span> x: <span class="bu">len</span>(x.unique()))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>


<table class="dataframe caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">item type</th>
<th data-quarto-table-cell-role="th">CitationRecord</th>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">id</th>
<th data-quarto-table-cell-role="th">collection name</th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td data-quarto-table-cell-role="th">Imaging Program</td>
<td>752</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td data-quarto-table-cell-role="th">Hierarchical phase-contrast tomography</td>
<td>135</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td data-quarto-table-cell-role="th">Cryo-Electron Tomography</td>
<td>2556</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">3</td>
<td data-quarto-table-cell-role="th">Volume Electron Microscopy</td>
<td>6817</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">4</td>
<td data-quarto-table-cell-role="th">Photoacoustic imaging</td>
<td>4477</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">5</td>
<td data-quarto-table-cell-role="th">Cryo-Electron Tomography (grantees)</td>
<td>20</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">6</td>
<td data-quarto-table-cell-role="th">Volume Electron Microscopy (grantees)</td>
<td>11</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">7</td>
<td data-quarto-table-cell-role="th">Hierarchical phase-contrast tomography (grantees)</td>
<td>12</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">8</td>
<td data-quarto-table-cell-role="th">Photoacoustic imaging (grantees)</td>
<td>23</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div id="cell-22" class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a>cb.agent_executor.invoke({<span class="st">'input'</span>:<span class="st">'Drop collection with id="0"'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>

&gt; Entering new AgentExecutor chain...
{
    "action": "delete_collection",
    "action_input": {
        "collection_id": "0"
    }
}{'response': 'Successfully deleted a collection with collection_id:`0`.'}


&gt; Finished chain.</code></pre>
</div>
<div class="cell-output cell-output-display">
<pre><code>{'input': 'Drop collection with id="0"',
 'output': {'response': 'Successfully deleted a collection with collection_id:`0`.'},
 'intermediate_steps': [(AgentAction(tool='delete_collection', tool_input={'collection_id': '0'}, log='{\n    "action": "delete_collection",\n    "action_input": {\n        "collection_id": "0"\n    }\n}'),
   {'response': 'Successfully deleted a collection with collection_id:`0`.'})]}</code></pre>
</div>
</div>
<div id="cell-23" class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="cf">with</span> <span class="bu">open</span>(files(imaging_tech).joinpath(<span class="st">'dois.txt'</span>), <span class="st">'r'</span>) <span class="im">as</span> f:</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>    dois <span class="op">=</span> f.readlines()</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>dois <span class="op">=</span> [d.strip() <span class="cf">for</span> d <span class="kw">in</span> dois]</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="bu">len</span>(dois))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>806</code></pre>
</div>
</div>
<div id="cell-24" class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>ldb.add_collection_from_dois_using_openalex(<span class="st">'0'</span>, <span class="st">'Imaging Program'</span>, dois, commit_this<span class="op">=</span><span class="va">True</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 806/806 [04:04&lt;00:00,  3.30it/s]</code></pre>
</div>
</div>
<div id="cell-25" class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Run local analysis on program data</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>step <span class="op">=</span> <span class="dv">40</span></span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> start_i <span class="kw">in</span> <span class="bu">range</span>(<span class="dv">0</span>, <span class="bu">len</span>(dois), step):</span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>    query <span class="op">=</span> <span class="st">' OR '</span>.join([<span class="st">'doi:</span><span class="ch">\"</span><span class="st">'</span><span class="op">+</span>dois[i].lower()<span class="op">+</span><span class="st">'</span><span class="ch">\"</span><span class="st">'</span> <span class="cf">for</span> i <span class="kw">in</span> <span class="bu">range</span>(start_i, start_i<span class="op">+</span>step) <span class="cf">if</span> i <span class="op">&lt;</span> <span class="bu">len</span>(dois)])</span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>    addEMPCCollection_tool.run({<span class="st">'id'</span>: <span class="st">'0'</span>, <span class="st">'name'</span>:<span class="st">'Imaging Program'</span>, <span class="st">'query'</span>:query, <span class="st">'full_text'</span>:<span class="va">True</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div id="cell-26" class="cell">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>missing_dois <span class="op">=</span> []</span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> doi <span class="kw">in</span> dois:</span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>    c <span class="op">=</span> ldb.session.query(func.count(SKE.<span class="bu">id</span>)).<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span><span class="st">'doi:'</span><span class="op">+</span>doi.lower()).first()</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> c[<span class="dv">0</span>] <span class="op">==</span> <span class="dv">0</span>:</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>        missing_dois.append(doi)</span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'Missing </span><span class="sc">%d</span><span class="st"> DOIs'</span><span class="op">%</span>(<span class="bu">len</span>(missing_dois)))</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(missing_dois)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Missing 34 DOIs
['10.3389%2Ffmed.2022.849677', '10.48550/arXiv.2210.04033', '10.3389%2Ffnins.2023.1135494', '10.48550/arXiv.2308.00870', '10.48550/arXiv.2307.14572', '10.48550/arXiv.2107.09145', '10.1038/s41598-021-94852-4', '10.1172%2Fjci.insight.142945', '10.1016/j.biopha.2022', '10.6084/m9.figshare.12758957.v1', '10.6084/m9.figshare.12299915.v2', '10.1126/sciadv.aaz2598.', '10.5281/ZENODO.3901011', '10.1152/ajpendo.00501.2018.', '10.48550/arXiv.2008.00807', '10.1136/ jitc-2022-006133', '10.1101/2020.05.27/119750', '10.1038/s41592-021- 01156-w', '0.1021/acschembio.0c00988', '10.1126/sciimmunol.abm693', '10.1038/s41598-021-85036-w', '10.1016%2Fj.ijcha.2020.100672', '10.1017/S2633903X2300003X[Opens in a new window]', '10.1016%2Fj.pacs.2021.100276', '10.1117%2F1.JBO.29.S1.S11521', '10.22443/rms.mmc2023.274', '10.5281/zenodo.10200758', '10.48550/arXiv.2306.15898', '10.48550/arXiv.2311.13417', '10.5281/zenodo.10451511', '10.5281/zenodo.10685021', '10.5281/zenodo.10057023', '10.5281/zenodo.10591803', '10.5281/zenodo.10591588']</code></pre>
</div>
</div>
<div id="cell-27" class="cell">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>step <span class="op">=</span> <span class="dv">40</span></span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> start_i <span class="kw">in</span> <span class="bu">range</span>(<span class="dv">0</span>, <span class="bu">len</span>(missing_dois), step):</span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>    query <span class="op">=</span> <span class="st">' OR '</span>.join([<span class="st">'doi:</span><span class="ch">\"</span><span class="st">'</span><span class="op">+</span>missing_dois[i].lower()<span class="op">+</span><span class="st">'</span><span class="ch">\"</span><span class="st">'</span> <span class="cf">for</span> i <span class="kw">in</span> <span class="bu">range</span>(start_i, start_i<span class="op">+</span>step) <span class="cf">if</span> i <span class="op">&lt;</span> <span class="bu">len</span>(missing_dois)])</span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>    addEMPCCollection_tool.run({<span class="st">'id'</span>: <span class="st">'0'</span>, <span class="st">'name'</span>:<span class="st">'Imaging Program'</span>, <span class="st">'query'</span>:query, <span class="st">'full_text'</span>:<span class="va">True</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&amp;pageSize=1000&amp;synonym=TRUE&amp;resultType=core&amp;query=doi:"10.3389%2ffmed.2022.849677" OR doi:"10.48550/arxiv.2210.04033" OR doi:"10.3389%2ffnins.2023.1135494" OR doi:"10.48550/arxiv.2308.00870" OR doi:"10.48550/arxiv.2307.14572" OR doi:"10.48550/arxiv.2107.09145" OR doi:"10.1038/s41598-021-94852-4" OR doi:"10.1172%2fjci.insight.142945" OR doi:"10.1016/j.biopha.2022" OR doi:"10.6084/m9.figshare.12758957.v1" OR doi:"10.6084/m9.figshare.12299915.v2" OR doi:"10.1126/sciadv.aaz2598." OR doi:"10.5281/zenodo.3901011" OR doi:"10.1152/ajpendo.00501.2018." OR doi:"10.48550/arxiv.2008.00807" OR doi:"10.1136/ jitc-2022-006133" OR doi:"10.1101/2020.05.27/119750" OR doi:"10.1038/s41592-021- 01156-w" OR doi:"0.1021/acschembio.0c00988" OR doi:"10.1126/sciimmunol.abm693" OR doi:"10.1038/s41598-021-85036-w" OR doi:"10.1016%2fj.ijcha.2020.100672" OR doi:"10.1017/s2633903x2300003x[opens in a new window]" OR doi:"10.1016%2fj.pacs.2021.100276" OR doi:"10.1117%2f1.jbo.29.s1.s11521" OR doi:"10.22443/rms.mmc2023.274" OR doi:"10.5281/zenodo.10200758" OR doi:"10.48550/arxiv.2306.15898" OR doi:"10.48550/arxiv.2311.13417" OR doi:"10.5281/zenodo.10451511" OR doi:"10.5281/zenodo.10685021" OR doi:"10.5281/zenodo.10057023" OR doi:"10.5281/zenodo.10591803" OR doi:"10.5281/zenodo.10591588", 6 European PMC PAPERS FOUND
 Returning 6</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 1/1 [00:00&lt;00:00,  1.01it/s]
100%|██████████| 6/6 [00:00&lt;00:00, 168.60it/s]</code></pre>
</div>
</div>
</section>
<section id="run-llm-over-each-paper-in-imaging-program-collection-to-determine-if-papers-are-methods-or-applications" class="level2">
<h2 class="anchored" data-anchor-id="run-llm-over-each-paper-in-imaging-program-collection-to-determine-if-papers-are-methods-or-applications">Run LLM over each paper in Imaging Program collection to determine if papers are methods or applications</h2>
<div id="cell-29" class="cell">
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a>t <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, TitleAbstractClassifier_OneDocAtATime_Tool)][<span class="dv">0</span>]</span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>t.run({<span class="st">'collection_id'</span>: <span class="st">'0'</span>, <span class="st">'classification_type'</span>:<span class="st">'is_method_paper'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div id="cell-30" class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>collection_id  <span class="op">=</span> <span class="st">'0'</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>classification_type <span class="op">=</span> <span class="st">'is_methods_paper'</span></span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKE, N) <span class="op">\</span></span>
<span id="cb32-4"><a href="#cb32-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb32-5"><a href="#cb32-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb32-6"><a href="#cb32-6" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>SKE_HR.ScientificKnowledgeExpression_id) <span class="op">\</span></span>
<span id="cb32-7"><a href="#cb32-7" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb32-8"><a href="#cb32-8" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.ScientificKnowledgeExpression_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb32-9"><a href="#cb32-9" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.has_notes_id<span class="op">==</span>N.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb32-10"><a href="#cb32-10" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb32-11"><a href="#cb32-11" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>collection_id) <span class="op">\</span></span>
<span id="cb32-12"><a href="#cb32-12" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">type</span><span class="op">==</span><span class="st">'TiAbClassificationNote__'</span><span class="op">+</span>classification_type) </span>
<span id="cb32-13"><a href="#cb32-13" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb32-14"><a href="#cb32-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> e,n <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb32-15"><a href="#cb32-15" aria-hidden="true" tabindex="-1"></a>    c <span class="op">=</span> json.loads(n.content)</span>
<span id="cb32-16"><a href="#cb32-16" aria-hidden="true" tabindex="-1"></a>    doi_link <span class="op">=</span> <span class="st">'https://doi.org/'</span><span class="op">+</span>e.<span class="bu">id</span>[<span class="dv">4</span>:]</span>
<span id="cb32-17"><a href="#cb32-17" aria-hidden="true" tabindex="-1"></a>    l.append((doi_link, e.<span class="bu">type</span>, e.content, c.get(<span class="st">'is_method_paper'</span>), c.get(<span class="st">'explanation'</span>)))</span>
<span id="cb32-18"><a href="#cb32-18" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(l, columns<span class="op">=</span>[<span class="st">'doi'</span>, <span class="st">'paper_type'</span>, <span class="st">'citation'</span>, <span class="st">'is_methods_paper'</span>, <span class="st">'explanation'</span>]) </span>
<span id="cb32-19"><a href="#cb32-19" aria-hidden="true" tabindex="-1"></a><span class="co">#df.to_csv(loc+db_name+'/imaging_cohort_methods.tsv', sep='\t', index=False)  </span></span>
<span id="cb32-20"><a href="#cb32-20" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>


<table class="dataframe caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th">paper_type</th>
<th data-quarto-table-cell-role="th">citation</th>
<th data-quarto-table-cell-role="th">is_methods_paper</th>
<th data-quarto-table-cell-role="th">explanation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>https://doi.org/10.1073/pnas.2301852120</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Lucas BA, Grigorieff N. (2023) Quantification ...</td>
<td>True</td>
<td>The main goal of the paper is to develop and t...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td>https://doi.org/10.1016/j.ultramic.2023.113730</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Axelrod JJ, Petrov PN, Zhang JT, Remis J, Buij...</td>
<td>True</td>
<td>The main goal of the paper is to identify and ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td>https://doi.org/10.1021/acs.jpcb.2c08995</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Sartor AM, Dahlberg PD, Perez D, Moerner WE. (...</td>
<td>True</td>
<td>The main goal of the paper is to characterize ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">3</td>
<td>https://doi.org/10.1101/2023.02.12.528160</td>
<td>ScientificPrimaryResearchPreprint</td>
<td>Axelrod JJ, Petrov PN, Zhang JT, Remis J, Buij...</td>
<td>True</td>
<td>The paper is concerned with developing new tec...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">4</td>
<td>https://doi.org/10.1016/j.jsb.2023.107941</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Du DX, Simjanoska M, Fitzpatrick AWP. (2023) F...</td>
<td>True</td>
<td>The main goal of the paper is to combine two e...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">763</td>
<td>https://doi.org/10.1016/b978-0-12-420138-5.000...</td>
<td>ScientificReviewArticle</td>
<td>Lambert TJ, Waters JC. (2014) Assessing camera...</td>
<td>True</td>
<td>The main goal of the paper is to assess and me...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">764</td>
<td>https://doi.org/10.1016/b978-0-12-420138-5.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Petrak LJ, Waters JC. (2014) A practical guide...</td>
<td>True</td>
<td>The main goal of the paper is to provide a pra...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">765</td>
<td>https://doi.org/10.1016/b978-0-12-420138-5.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Waters JC, Wittmann T. (2014) Concepts in quan...</td>
<td>True</td>
<td>The main goal of the paper is to discuss conce...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">766</td>
<td>https://doi.org/10.1016/b978-0-12-407761-4.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Waters JC. (2013) Live-cell fluorescence imaging.</td>
<td>True</td>
<td>The main goal of the paper is to optimize live...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">767</td>
<td>https://doi.org/10.1016/b978-0-12-407761-4.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Salmon ED, Shaw SL, Waters JC, Waterman-Storer...</td>
<td>True</td>
<td>The main goal of the paper is to describe the ...</td>
</tr>
</tbody>
</table>

<p>768 rows × 5 columns</p>
</div>
</div>
</div>
</section>
<section id="run-the-tool-for-high-level-subtypes" class="level1">
<h1>Run the tool for high-level subtypes</h1>
<div id="cell-32" class="cell">
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>t <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, TitleAbstractClassifier_OneDocAtATime_Tool)][<span class="dv">0</span>]</span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>t.run({<span class="st">'collection_id'</span>: <span class="st">'0'</span>, <span class="st">'classification_type'</span>:<span class="st">'top_level_imaging_categories'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>100%|██████████| 752/752 [23:07&lt;00:00,  1.84s/it]</code></pre>
</div>
<div class="cell-output cell-output-display">
<pre><code>{'response': "completed document classification of type 'top_level_imaging_categories' for collection 0.",
 'data': [{'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The abstract mentions the use of MRI (Magnetic Resonance Imaging) and PET (Positron Emission Tomography) scans as part of their diagnostic methods, implying that MRI is a primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.3389/fnins.2021.768646'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The paper's title and abstract mention the use of cryo-electron tomography (CET) as the imaging technique for determining the structure of proteins in cells, which is central to the work being presented."},
   'paper_id': 'doi:10.1007/978-3-031-19803-8_38'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper primarily uses light sheet fluorescence microscopy (LSFM), which is a type of fluorescence microscopy, indicated by the title 'Quantifiable Intravital Light Sheet Microscopy.'"},
   'paper_id': 'doi:10.1007/978-1-0716-2051-9_11'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract mentions a focus on medical image analysis algorithms, which suggests computational methods for analyzing image data rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1007/978-3-031-16961-8'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The abstract mentions the use of electron microscopy, specifically in the context of analyzing mitochondrial dynamics in macrophages.'},
   'paper_id': 'doi:10.1007/978-1-0716-2449-4_18'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The context of the work specifically mentions the use of 'cryo-electron microscopy' (Cryo-EM) and 'cryo-electron tomography' (Cryo-ET), which are the primary imaging technologies discussed in the paper."},
   'paper_id': 'doi:10.1109/wacv51458.2022.00332'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper's title mentions 'The Bronchial Circulation in COVID-19 Pneumonia', which suggests a study of the blood vessels in the lungs. MRI is a common imaging technology used to visualize soft tissues and blood vessels without ionizing radiation, which makes it suitable for such studies. However, without an abstract, this categorization is only a best guess based on the information provided in the title."},
   'paper_id': 'doi:10.1164/rccm.202103-0594im'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper discusses the development of a custom peptide that enhances the ability of an AAV vector to cross the blood-brain barrier, which is a molecular probe strategy rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1016/j.biomaterials.2021.121340'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper does not explicitly mention a specific imaging technology, but it discusses the targeting and killing of bacteria within host cells, which implies the use of some form of molecular probes or related technology to observe or measure the effects of the peptides on the bacteria.'},
   'paper_id': 'doi:10.1007/s00018-021-04041-z'},
  {'classification': {'imaging_technology_code': 'B',
    'imaging_technology_name': 'Photoacoustic imaging',
    'explanation': "The title of the paper directly mentions 'photoacoustic microscopy' which is a known subtype of photoacoustic imaging. The context of the work focuses on enhancing the capabilities of this specific imaging technology."},
   'paper_id': 'doi:10.1101/2021.12.25.474038'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the imaging technology used, but it describes a computational analysis system that is integral to understanding the formation of endoggresomes, which suggests that computational methods are central to the study.'},
   'paper_id': 'doi:10.1126/sciadv.abg3693'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of widefield microscopy and laser scanning confocal microscopy, both of which are types of fluorescence microscopy techniques.'},
   'paper_id': 'doi:10.1007/s00125-021-05619-9'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The paper's title and abstract mention the use of cryo-electron microscopy (cryo-EM) as the primary imaging technology for studying the structure of CcsBA."},
   'paper_id': 'doi:10.1038/s41589-021-00935-y'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper primarily discusses a computational method (FourierDist) for cell segmentation in images, without specifying any particular imaging hardware technology used to generate the images.'},
   'paper_id': 'doi:10.1101/2021.12.17.472408'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper primarily discusses the implantation of novel wireless cardiac bioelectronic devices, which is a form of hardware technology not explicitly listed in the other categories.'},
   'paper_id': 'doi:10.21203/rs.3.pex-1684/v1'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract does not provide specific details about the imaging technology used. However, the study of actin dynamics and cellular responses at the microscopic level often involves fluorescence microscopy techniques. Since no specific advanced fluorescence microscopy technique like super-resolution is mentioned, 'Other fluorescence microscopy' is the most likely category."},
   'paper_id': 'doi:10.1016/j.devcel.2021.11.024'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper uses polarization-sensitive optical coherence tomography (PS-OCT), which is an optical imaging technique and does not fit into the specific categories listed but is a type of optical hardware for generating image data.'},
   'paper_id': 'doi:10.1364/boe.443066'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The abstract specifically mentions 'electron cryo-tomography (cryo-ET)' as the method for determining structures of macromolecular complexes within cells, indicating that this is the primary imaging technology being discussed."},
   'paper_id': 'doi:10.1016/j.jsb.2021.107827'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper describes using a modified optic microscope coupled with homemade digital image processing and data analysis software, which suggests a customized optical imaging technology that does not fit into the specific categories listed.'},
   'paper_id': 'doi:10.24254/cnib.21.5'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper describes the Micro-Meta App, which is a software tool for collecting microscopy metadata, indicating a focus on computational methodology rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41592-021-01315-z'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The title of the paper suggests a focus on data formats and repositories, which implies a primary concern with the computational aspects of image data management and analysis rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41592-021-01113-7'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The title mentions 'light microscopy' which generally refers to fluorescence microscopy and other related techniques. The specific subtype of imaging technology is not given, but 'light microscopy' commonly falls under the category of 'Other fluorescence microscopy'."},
   'paper_id': 'doi:10.1038/s41592-021-01162-y'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide any specific information about the imaging technology used. However, the title suggests the paper is focused on metadata for biological images, which is more related to data management and computational analysis rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41592-021-01166-8'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The paper discusses 'MDEmic', which is a metadata annotation tool, indicating a focus on computational methodology to manage image data, rather than a specific imaging technology."},
   'paper_id': 'doi:10.1038/s41592-021-01288-z'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The title suggests that the paper is focused on a software tool for generating comprehensive microscopy methods text, which implies it is about computational methodology for handling metadata in microscopy rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41592-021-01290-5'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The title of the paper references 'light microscopy,' which is a broad category but often includes various types of fluorescence microscopy. Given the absence of a specific subtype of light microscopy in the title, it is reasonable to infer that the paper deals with standards for metadata in fluorescence microscopy, which is a common application within light microscopy."},
   'paper_id': 'doi:10.1038/s41592-021-01327-9'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The abstract does not specify the imaging technology used but mentions 'microscopy' as part of their analysis methods. Since there is no specific imaging technique detailed that corresponds to the provided categories, the classification falls under 'Other imaging hardware technology'."},
   'paper_id': 'doi:10.3390/cancers13235998'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses a file format for bioimaging data, which is related to computational analysis rather than a specific type of imaging technology.'},
   'paper_id': 'doi:10.1038/s41592-021-01326-w'},
  {'classification': {'imaging_technology_code': 'A',
    'imaging_technology_name': 'Hierarchical phase-contrast tomography',
    'explanation': "The paper's title and abstract mention the use of 'hierarchical phase-contrast tomography' as the imaging technique employed for the study."},
   'paper_id': 'doi:10.1101/2021.11.20.469361'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'highly multiplexed single-cell imaging of 48 core cell cycle proteins,' which suggests the use of fluorescence microscopy techniques to visualize multiple proteins within single cells."},
   'paper_id': 'doi:10.1016/j.cels.2021.10.007'},
  {'classification': {'imaging_technology_code': 'C',
    'imaging_technology_name': 'Near infrared imaging',
    'explanation': "The abstract mentions the use of 'millikelvin resolution thermography' and 'delivery of optical stimulation', which are indicative of technologies related to infrared imaging. Since there is no explicit mention of other imaging technologies from the list, Near infrared imaging seems to be the primary technology used."},
   'paper_id': 'doi:10.1038/s41467-021-27003-2'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper mentions the use of '3D fluorescent data of spindles and chromatin,' which indicates the primary imaging technology is fluorescence microscopy. The specific type of fluorescence microscopy isn't detailed, suggesting it falls under the 'Other fluorescence microscopy' category."},
   'paper_id': 'doi:10.1083/jcb.202106170'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper focuses on a survey about the needs and experiences of the imaging community related to bioimage analysis, which implies a focus on computational methodologies for analyzing image data rather than on any specific imaging hardware technology.'},
   'paper_id': 'doi:10.1017/s2633903x21000039'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'While the abstract does not specify an imaging technology, the mention of ROS (reactive oxygen species) suggests the use of molecular probes that detect oxidative stress, which is common in studies of mitochondria and mitophagy.'},
   'paper_id': 'doi:10.1016/j.redox.2021.102186'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper mentions the use of three-dimensional brain shape analysis approaches, which is consistent with the capabilities of MRI, a common technique for detailed brain structure imaging.'},
   'paper_id': 'doi:10.1016/j.biopsych.2021.11.002'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The paper discusses the development of a fiber-based optical coherence microscopy system, which is an optical imaging hardware technology that does not fit into the specific categories listed, hence it is classified under 'Other imaging hardware technology'."},
   'paper_id': 'doi:10.1364/boe.442081'},
  {'classification': {'imaging_technology_code': 'A',
    'imaging_technology_name': 'Hierarchical phase-contrast tomography',
    'explanation': "The paper's title and abstract clearly state that the technology used is hierarchical phase-contrast tomography (HiP-CT), which is a form of X-ray phase contrast tomography."},
   'paper_id': 'doi:10.1038/s41592-021-01317-x'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper primarily discusses light-sheet microscopy, which is a type of fluorescence microscopy, specifically mentioned as 'light-sheet fluorescence microscopy' in the list of technologies."},
   'paper_id': 'doi:10.3389/fcell.2021.739079'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses CellProfiler Analyst 3.0, which is a software package for data exploration and machine learning in image analysis, indicating that the focus is on computational methodologies to analyze image data rather than on a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1093/bioinformatics/btab634'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technology; it focuses on biochemical methods such as quantification of chemokine levels, in vitro stimulation, and migration assays. Therefore, since no imaging technology is specified, we cannot accurately assign one. The closest classification would be computational analysis for the analysis of the biochemical data, although this is not explicitly stated.'},
   'paper_id': 'doi:10.1002/art.41882'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'Imaging of membrane bending dynamics and the CME pit ultrastructure,' which suggests the use of a microscopy technique. While the specific type of microscopy is not stated, fluorescence microscopy is commonly used to image cellular processes like CME, and the abstract does not provide sufficient information to ascertain the use of a super-resolution technique, which would be indicated by terms such as STED, SIM, etc."},
   'paper_id': 'doi:10.1016/j.celrep.2021.110008'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'Cell Painting assays,' which are a type of fluorescence microscopy used to profile cell morphology, indicating that fluorescence microscopy is the primary imaging technology used in the paper."},
   'paper_id': 'doi:10.1101/2021.10.21.465335'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of 'high-resolution light and electron microscopy' which suggests the use of advanced microscopy techniques. Given the context of the abstract and the lack of specific mention of 'super-resolution' technologies, 'Other fluorescence microscopy' is the most likely category, as it is a broad category that includes various forms of advanced light microscopy commonly used in biological research."},
   'paper_id': 'doi:10.7554/elife.67292'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': 'The paper discusses the Mean-Shift Super Resolution (MSSR), which is a computational method to extend spatial resolution in fluorescence images beyond the diffraction limit, indicating the paper is primarily about super-resolution imaging technology.'},
   'paper_id': 'doi:10.1101/2021.10.17.464398'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': 'The paper discusses a method for image denoising and detection of proteins imaged by cryo-EM, which is a form of Cryo-electron tomography.'},
   'paper_id': 'doi:10.1109/aipr52630.2021.9762098'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': 'The abstract explicitly mentions the use of cryoelectron microscopy (cryo-EM) to characterize the antibodies in complex with the spike protein, which is a form of Cryo-electron tomography.'},
   'paper_id': 'doi:10.1016/j.celrep.2021.109881'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper mentions the use of longitudinal MRI to reveal significant gray matter volume reductions, indicating that Magnetic Resonance Imaging is the primary imaging technology used.'},
   'paper_id': 'doi:10.1523/jneurosci.0378-21.2021'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the use of a novel glycoprotein IIb/IIIa-receptor radiotracer, 18F-GP1, which is a type of molecular imaging probe used in PET-CT imaging to detect thrombus formation in coronary arteries.'},
   'paper_id': 'doi:10.1093/eurheartj/ehab724.0261'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The title of the paper suggests the use of imaging technology to visualize thrombosis in patients recovering from COVID-19 and pulmonary embolism. Among the listed technologies, MRI is commonly used for imaging thrombosis and pulmonary embolism in clinical settings.'},
   'paper_id': 'doi:10.1164/rccm.202011-4182im'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The paper's title includes 'Cryo-TEM' and the abstract mentions 'frozen-hydrated specimens (cryo-EM)', which indicates that the primary technology used is cryo-electron microscopy, a form of cryo-electron tomography."},
   'paper_id': 'doi:10.1107/s2052252521008538'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper describes the use of total-body positron emission tomography (PET) data analysis for understanding complex bone metabolic interactions in vivo, which involves computational analysis of imaging data.'},
   'paper_id': 'doi:10.3389/fmed.2021.740615'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper's title and abstract mention the use of magnetic resonance imaging (MRI) data for analyzing brain structure in Friedreich Ataxia, indicating that MRI is the primary imaging technology used in this study."},
   'paper_id': 'doi:10.1002/ana.26200'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract discusses improvements in an image analysis program called CellProfiler, which is used for processing microscopy images, indicating that the paper is about computational analysis of image data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1186/s12859-021-04344-9'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper describes the use of in vivo confocal microscopy, which is a type of fluorescence microscopy, to visualize labeled cells in zebrafish.'},
   'paper_id': 'doi:10.1242/bio.058734'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'image-based assay' without specifying the type, but since fluorescence microscopy is a common method for cellular morphology assessment, it is the most likely technology used."},
   'paper_id': 'doi:10.1101/2021.09.08.459417'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses light sources used in fluorescence microscopy, comparing traditional arc-based sources to LEDs, which indicates that the primary technology being discussed is fluorescence microscopy.'},
   'paper_id': 'doi:10.1002/cpz1.243'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract mentions light sheet microscopy, which is a form of fluorescence microscopy, indicating that the paper primarily uses this technology.'},
   'paper_id': 'doi:10.1093/bioinformatics/btab106'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': "The work mentions 'high-resolution, live-cell imaging' which is indicative of super-resolution microscopy, a technique that allows imaging beyond the diffraction limit of light and is commonly used in live-cell imaging applications."},
   'paper_id': 'doi:10.1016/j.celrep.2021.109662'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper primarily uses in vivo magnetic resonance (MR) imaging to monitor brain injury post-exposure, which is indicated by the use of the term 'MR' in the methods."},
   'paper_id': 'doi:10.1016/j.neuro.2021.08.011'},
  {'classification': {'imaging_technology_code': 'E',
    'imaging_technology_name': 'Nonlinear microscopy',
    'explanation': 'The paper discusses the effects of pulsed-laser excitation used in nonlinear optical microscopy, specifically mentioning two-photon microscopy and stimulated Raman scattering microscopy, which are subtypes of nonlinear microscopy.'},
   'paper_id': 'doi:10.1101/2021.08.27.457929'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of lattice light sheet microscopy (LLSM), which is a type of light-sheet fluorescence microscopy, for imaging cells at high speed and 3D resolution.'},
   'paper_id': 'doi:10.1186/s12859-021-04324-z'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses a survey about community experiences and needs in bioimage analysis, which indicates a focus on computational methodologies for analyzing image data rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1101/2021.08.16.456498'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the use of positron emission tomography (PET), which is a type of molecular imaging that uses probes to detect specific biological processes such as inflammation, fibrosis, and angiogenesis.'},
   'paper_id': 'doi:10.3389/fcvm.2021.719031'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses lattice light sheet microscopy, which is a type of light-sheet fluorescence microscopy.'},
   'paper_id': 'doi:10.1038/s41467-021-25187-1'},
  {'classification': {'imaging_technology_code': 'I',
    'imaging_technology_name': 'Volume electron microscopy',
    'explanation': 'The abstract mentions the registration of an expression atlas to a whole-body electron microscopy volume, indicating that volume electron microscopy is the primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.1016/j.cell.2021.07.017'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily discusses live fluorescence imaging, which is a type of fluorescence microscopy, to visualize macropinosome features in live cells.'},
   'paper_id': 'doi:10.3791/62870'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The paper discusses the use of various low-tech and cutting-edge methods to address shortages in medical devices and equipment, including 3D-printed face shields and other devices. Although not explicitly stated, the context suggests the use of imaging hardware technologies that are not categorized under the specific imaging modalities listed, hence 'Other imaging hardware technology' is the most suitable classification."},
   'paper_id': 'doi:10.3389/fmedt.2021.702526'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper focuses on modelling PET data and evaluating simplified imaging protocols for quantification, which indicates a primary focus on computational analysis of image data rather than the development or use of a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1007/s00259-021-05482-1'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper focuses on protocols for Fluorescence Lifetime Imaging Microscopy (FLIM), a technique within the broader category of fluorescence microscopy, which is indicated by the mention of FLIM data processing in the abstract.'},
   'paper_id': 'doi:10.1002/cpz1.204'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': "The paper describes the use of 'three-color single-molecule imaging' with 'nanometer precision' to track protein conformation, which suggests the use of super-resolution techniques to achieve such high spatial resolution beyond the diffraction limit of standard fluorescence microscopy."},
   'paper_id': 'doi:10.1073/pnas.2101391118'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': "The title of the paper 'Deep Learning-Based Point-Scanning Super-Resolution Microscopy' clearly indicates that the primary imaging technology discussed is super-resolution microscopy."},
   'paper_id': 'doi:10.1017/s1431927621008977'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses a protocol for evaluating deformable registration algorithms, which is a computational method used to assess the quality of image registration. There is no specific imaging hardware technology described, hence the focus is on computational analysis of image data.'},
   'paper_id': 'doi:10.1109/tmi.2021.3070842'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The paper discusses bioelectronic devices that interface with biological tissues, with a focus on mechanical, electrical, and optical properties, rather than a specific imaging technology. The closest category is 'Other imaging hardware technology' since the devices might include hardware for interfacing and possibly imaging, but no specific imaging technology is clearly identified as the primary focus."},
   'paper_id': 'doi:10.1038/s41563-021-01051-x'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper primarily uses photographs of 96-well broth microdilution plates to measure MICs, and the processing of these images is done by a crowd of volunteers, indicating the use of computational analysis methodologies to analyze image data.'},
   'paper_id': 'doi:10.1101/2021.07.20.453060'},
  {'classification': {'imaging_technology_code': 'N',
    'imaging_technology_name': 'Computational analysis systems',
    'explanation': "The abstract does not provide explicit details on imaging technology; however, the focus on 'sequential and switchable patterning' and 'spatiotemporal control' suggests a strong computational component to analyze and manipulate patterns, hence the classification under computational analysis systems."},
   'paper_id': 'doi:10.1021/acsami.1c09850'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract discusses a computational approach to build 3D models from microscopy images, without specifying a particular imaging technology. It emphasizes feature segmentation and model construction, suggesting the primary focus is on computational analysis of image data.'},
   'paper_id': 'doi:10.1016/j.bpr.2021.100003'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': "The abstract mentions the use of 'three-dimensional (3D) super-resolution microscopy' which indicates that super-resolution imaging technology is primarily used in this paper."},
   'paper_id': 'doi:10.1101/2021.07.16.452693'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "Since the abstract is missing, there's no information available to make a decision. However, based on the title, the paper seems to be focused on data and specimen provenance standards rather than a specific imaging technology, which suggests a computational or informatics approach to managing and analyzing life science data."},
   'paper_id': 'doi:10.5281/zenodo.5093125'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of 'total internal reflection fluorescence analysis', which is a technique used in fluorescence microscopy."},
   'paper_id': 'doi:10.1101/2021.07.12.451665'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract mentions the use of γH2AX staining, which is a method used in fluorescence microscopy to detect DNA damage. This suggests that fluorescence microscopy is the primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.1021/acschembio.1c00322'},
  {'classification': {'imaging_technology_code': 'E',
    'imaging_technology_name': 'Nonlinear microscopy',
    'explanation': 'The paper primarily uses label-free second harmonic generation (SHG) light microscopy, which is a type of nonlinear microscopy, to quantify fibrillar collagen deposition.'},
   'paper_id': 'doi:10.1183/13993003.00221-2020'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper discusses a semiconducting nanowire wrapped by a superconducting shell, which suggests the use of quantum or electronic properties for imaging or measurement, rather than a specific imaging technique from the provided list.'},
   'paper_id': 'doi:10.1126/science.abf1513'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses improvements to CellProfiler, an open source image analysis program, indicating that the primary focus is computational analysis of image data rather than any specific imaging hardware technology.'},
   'paper_id': 'doi:10.1101/2021.06.30.450416'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technology. The focus is on the discovery and optimization of antimicrobial peptides and their mechanisms of action, which suggests the primary methodology is biochemical rather than imaging-based. Therefore, the paper likely relies on computational analysis to understand interactions and effects, rather than on a specific imaging technology.'},
   'paper_id': 'doi:10.1101/2021.06.27.450110'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses a software tool for improving the reporting of microscopy methods, which implies a computational approach to managing and analyzing microscopy data, rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1101/2021.06.23.449674'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': "The abstract mentions the use of 'multiplexed fluorescence imaging' and 'PRobe-based Imaging for Sequential Multiplexing (PRISM)' which involves the use of probes for imaging, indicating the primary technology is related to molecular imaging probes."},
   'paper_id': 'doi:10.1038/s42003-021-02276-x'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not mention a specific imaging technology used for visualizing the fibrosis or dysfunction. Therefore, we can infer that the primary focus is on the computational analysis of biochemical markers and physiological parameters rather than on a specific imaging technology.'},
   'paper_id': 'doi:10.1172/jci.insight.142945'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract discusses a computational method for 3D instance segmentation and tracking using voxel embedding in biomedical video analysis, rather than detailing a specific imaging hardware technology.'},
   'paper_id': 'doi:10.48550/arxiv.2106.11480'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': 'The abstract mentions the use of proteomic mass spectrometry data for training their model, indicating that mass spectrometry is the primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.1101/2021.06.19.449132'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The abstract mentions the use of cryo-focused-ion-beam scanning electron microscopy, which falls under the category of electron microscopy technologies.'},
   'paper_id': 'doi:10.1016/j.jcis.2021.06.093'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses the use of supervised and weakly supervised deep learning for imaging cytometry, which implies a computational methodology for analyzing image data rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1038/s41596-021-00549-7'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': 'Although the abstract does not explicitly mention the imaging technology used, cryo-electron tomography is a common technique for studying subcellular structures such as nuclear pore complexes at high resolution.'},
   'paper_id': 'doi:10.1016/j.devcel.2021.05.015'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper describes the use of 'phasor-based hyperspectral snapshot microscopy' which leverages 'light sheet microscopy', a type of fluorescence microscopy, indicating that it falls under the category of other fluorescence microscopy technologies."},
   'paper_id': 'doi:10.1038/s42003-021-02266-z'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': "The title mentions 'Structured Illumination Microscope' which is a type of super-resolution microscopy technique (specifically mentioned as 'SIM' in the list). This indicates that the primary imaging technology used in the paper is super-resolution."},
   'paper_id': 'doi:10.5281/zenodo.4891883'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify a particular imaging technology used in the paper. Since the main focus is on genetic mutagenesis and behavioural phenotypes, it is likely that computational analysis plays a role in interpreting these results, although the exact nature is not explicitly stated.'},
   'paper_id': 'doi:10.3390/ijms22115948'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper primarily discusses the use of dynamic 18F-FDG PET/CT and CT perfusion, which are specific imaging technologies that do not fit into the more defined categories listed but are well-established forms of medical imaging hardware.'},
   'paper_id': 'doi:10.2478/raon-2021-0024'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'light microscopy experiments,' which generally refers to a broad range of optical microscopy techniques, including various fluorescence microscopy methods. There is no specific mention of super-resolution or nonlinear microscopy, hence 'Other fluorescence microscopy' is the most suitable category."},
   'paper_id': 'doi:10.1101/2021.05.31.446382'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper focuses on a computational approach to instance segmentation and tracking in microscopy images, which does not specify the use of a particular imaging hardware technology but rather an annotation-free synthetic instance segmentation and tracking method using adversarial simulations.'},
   'paper_id': 'doi:10.1016/j.compbiomed.2021.104501'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper uses fluorescent sterols (e.g., DHE and BCh) and fluorescent protein markers, which are indicative of fluorescence microscopy techniques.'},
   'paper_id': 'doi:10.3389/fpls.2021.616631'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract mentions the use of integrated proteomics, transcriptomics, and imaging to demonstrate INPP4B localization to late endosomes, which implies the use of fluorescence microscopy, a common technique for visualizing subcellular localization. Although the exact type of fluorescence microscopy is not specified, it is a reasonable assumption given the context.'},
   'paper_id': 'doi:10.1038/s41467-021-23241-6'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The paper discusses instance segmentation of mitochondria in electron microscopy data, which implies that electron microscopy is the primary imaging technology used.'},
   'paper_id': 'doi:10.1101/2021.05.24.444785'},
  {'classification': {'imaging_technology_code': 'B',
    'imaging_technology_name': 'Photoacoustic imaging',
    'explanation': 'The paper is focused on photoacoustic tomography, which is a type of photoacoustic imaging, as indicated by both the title and the abstract.'},
   'paper_id': 'doi:10.1016/j.pacs.2021.100276'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses the use of a kinase translocation reporter for live-imaging of Erk-signalling dynamics in zebrafish, which is a technique associated with fluorescence microscopy.'},
   'paper_id': 'doi:10.7554/elife.62196'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper uses ACDAN fluorescence and spectral phasor analysis, which are indicative of fluorescence microscopy techniques, to study macromolecular crowding in zebrafish lenses.'},
   'paper_id': 'doi:10.1101/2021.05.15.442187'},
  {'classification': {'imaging_technology_code': 'I',
    'imaging_technology_name': 'Volume electron microscopy',
    'explanation': "The abstract mentions the use of 'serial blockface scanning electron microscopy,' which is a type of volume electron microscopy used for imaging cells at nanometre resolution."},
   'paper_id': 'doi:10.1111/tra.12789'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of a 'pipeline for automated embryo imaging on confocal microscopes.' Confocal microscopy is a fluorescence imaging technique, which falls under the category 'Other fluorescence microscopy'."},
   'paper_id': 'doi:10.1038/s41598-021-89676-5'},
  {'classification': {'imaging_technology_code': 'D',
    'imaging_technology_name': 'Ultrasound',
    'explanation': 'The paper primarily uses ultrasound-stimulated phase-change contrast agents (US-PCCA) as a method to improve drug penetration, indicating that ultrasound technology is a central component of their research.'},
   'paper_id': 'doi:10.1016/j.bioflm.2021.100049'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper discusses the evaluation of a reconstruction algorithm specifically for brain PET-MR (positron emission tomography-magnetic resonance) studies, indicating that the primary imaging technology used is magnetic resonance imaging in combination with PET.'},
   'paper_id': 'doi:10.1186/s40658-021-00386-3'},
  {'classification': {'imaging_technology_code': 'C',
    'imaging_technology_name': 'Near infrared imaging',
    'explanation': 'The paper discusses using polymethine dyes for fast and multiplexed SWIR imaging, which falls under the category of Near Infrared Imaging.'},
   'paper_id': 'doi:10.1021/jacs.0c11599'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses the phasor approach to fluorescence lifetime imaging, which is a method of analyzing fluorescence signals, indicating that the technology falls under the category of fluorescence microscopy.'},
   'paper_id': 'doi:10.1146/annurev-biophys-062920-063631'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper discusses the synthesis of radiolabeled compounds (cis- and trans-4-[18F]fluoro-l-proline) used as imaging agents in PET scans, a type of molecular imaging probe.'},
   'paper_id': 'doi:10.1021/acs.joc.1c00755'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper discusses metadata standards for light microscopy, which includes fluorescence microscopy as a subset. The abstract does not specify a focus on super-resolution or other specialized forms of microscopy, hence 'Other fluorescence microscopy' is the most appropriate category."},
   'paper_id': 'doi:10.1101/2021.04.25.441198'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses performance metrics for image analysis, which pertains to computational methodologies rather than specific imaging hardware or techniques.'},
   'paper_id': 'doi:10.48550/arxiv.2104.05642'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper mentions the use of 'multi-shell diffusion MRI data' indicating that magnetic resonance imaging (MRI) is the primary imaging technology used in the study."},
   'paper_id': 'doi:10.1007/s11682-021-00467-y'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper focuses on the computational aspects of cell segmentation and tracking using cosine embedding and a mean-shift algorithm, which is a method of computational analysis applied to image data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1016/j.media.2021.102048'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses open-source deep-learning tools for image segmentation, which is a computational methodology to analyze image data, rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1091/mbc.e20-10-0660'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses a workflow using raw microscopy images as input for analyzing CFTR function, which suggests the use of fluorescence microscopy, a common technique for visualizing live cell functions.'},
   'paper_id': 'doi:10.1093/bioinformatics/btaa1073'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': 'The abstract mentions the use of single-particle cryo-electron microscopy to reveal the DNA-binding domain structure, indicating that cryo-electron tomography is the primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.1038/s41467-021-22253-6'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses Expansion Microscopy (ExM), which is a technique that physically magnifies biological specimens and is typically used alongside fluorescence microscopy to enable nanoscale-resolution imaging with conventional microscopes.'},
   'paper_id': 'doi:10.1038/s41565-021-00875-7'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract mentions a 'high-throughput microscopy-based screen' which implies the use of microscopy along with computational analysis to screen a large number of drugs. However, the specific type of microscopy isn't mentioned, so it falls under the broad category of computational analysis used to analyze image data from the screening."},
   'paper_id': 'doi:10.1183/23120541.lsc-2021.33'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'quantitative live-cell imaging,' a method typically associated with fluorescence microscopy, used to study the dynamics and interactions of proteins within living cells."},
   'paper_id': 'doi:10.1016/j.cub.2021.02.028'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The abstract does not provide specific details about the imaging technology used, and none of the sentences specifically mention or imply the use of a particular imaging technology from the provided list. The decision is based on the absence of information that fits into the specific categories listed.'},
   'paper_id': 'doi:10.3389/fmed.2021.606678'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions using the fluorescent lipid probe LAURDAN, which is indicative of a fluorescence microscopy technique, likely not falling under super-resolution or nonlinear categories.'},
   'paper_id': 'doi:10.3389/fmolb.2021.630625'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper primarily discusses the use of Optical Coherence Tomography (OCT), which is an optical imaging hardware technology that does not fit into the specific categories listed but is well-established and distinct from those mentioned.'},
   'paper_id': 'doi:10.1117/12.2578489'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide explicit information about the imaging technology used. Since the paper discusses mitochondrial function and redox reactions, it is likely that the study involved some computational analysis of biochemical data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1002/hep4.1691'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract does not explicitly mention the imaging technology used, but the investigation of bloodborne microparticles and the mention of annexin-5 binding, which is a method for detecting phosphatidylserine on cell surfaces, suggest the use of fluorescence microscopy, possibly involving fluorescent tagging of proteins or other structures.'},
   'paper_id': 'doi:10.1084/jem.20200551'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper deals with the quantitation of fluorescent labeling intensity within segmented nuclei in 2D fluorescent still images, indicating the use of fluorescence microscopy.'},
   'paper_id': 'doi:10.1101/2021.03.01.433386'},
  {'classification': {'imaging_technology_code': 'E',
    'imaging_technology_name': 'Nonlinear microscopy',
    'explanation': 'The paper discusses pump-probe microscopy, which is a nonlinear imaging technique that involves the use of high repetition rate lasers, characteristic of nonlinear microscopy methods like two-photon excitation fluorescence.'},
   'paper_id': 'doi:10.1364/ol.417905'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The title mentions 'software for customized and reproducible microscope control', which suggests the paper is about computational software for controlling microscopes, rather than a specific imaging technology."},
   'paper_id': 'doi:10.1038/s41592-021-01087-6'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract does not specify the imaging technology used, but it discusses the study of gradients and migration that likely involve computational analysis of image data. Since there is no clear mention of specific imaging hardware or technology, the category of 'Other computational analysis' seems most appropriate."},
   'paper_id': 'doi:10.3389/fimmu.2021.630002'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract mentions the use of high resolution light-sheet microscopy, which is a type of fluorescence microscopy, to study the development of neuropil in C. elegans.'},
   'paper_id': 'doi:10.1038/s41586-020-03169-5'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technology. Instead, it discusses the need for increased investigation into the anatomy and physiology of the trigeminal system, which suggests a focus on computational analysis rather than a specific imaging modality.'},
   'paper_id': 'doi:10.1111/joa.13413'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract mentions the use of RNA-Seq analysis, which is a computational analysis method to analyze gene expression data. No specific imaging technology is indicated in the context provided.'},
   'paper_id': 'doi:10.1038/s41385-021-00378-7'},
  {'classification': {'imaging_technology_code': 'C',
    'imaging_technology_name': 'Near infrared imaging',
    'explanation': "The title and abstract mention 'near-infrared luminescent materials,' which aligns directly with near infrared imaging technology."},
   'paper_id': 'doi:10.1016/j.isci.2021.102156'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper describes using a photoactivatable fluorescent protein and fluorescence-activated cell sorting (FACS), which are techniques associated with fluorescence microscopy.'},
   'paper_id': 'doi:10.1083/jcb.202008158'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify a particular imaging technology being used, but it discusses the role of altered mitochondrial mobility which suggests a focus on computational analysis of existing data rather than a specific imaging technique.'},
   'paper_id': 'doi:10.3389/fcell.2021.624823'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract discusses the use of LAURDAN, a fluorescent probe, in combination with microscopy techniques such as lifetime imaging and spectral imaging, which are characteristic of fluorescence microscopy.'},
   'paper_id': 'doi:10.1021/acs.accounts.0c00687'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The paper discusses workflows and protocols for immuno-correlative light and electron microscopy (iCLEM) and specifically mentions the use of scanning EM (SEM), scanning TEM (STEM), and transmission EM (TEM), which are all forms of electron microscopy.'},
   'paper_id': 'doi:10.1038/s41598-020-79637-9'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': 'The paper mentions the use of Airyscan confocal microscopy, which is a type of super-resolution microscopy technique that enhances resolution beyond the diffraction limit of light.'},
   'paper_id': 'doi:10.1371/journal.pgen.1009277'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of kindlin-3-fluorescent protein (FP) fusion proteins and EGFP-kindlin-3, indicating the use of fluorescence microscopy techniques.'},
   'paper_id': 'doi:10.1182/blood.2019003446'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper describes using '12-color multiplexed fluorescence imaging' which falls under the category of fluorescence microscopy but is not specified as super-resolution or another subtype, so it is classified as other fluorescence microscopy."},
   'paper_id': 'doi:10.1523/eneuro.0286-20.2020'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract does not specify the imaging technology used, but given the study of myelination and axon-glial configurations in the brain, fluorescence microscopy techniques are commonly used for such investigations. None of the other categories are directly implied by the context provided.'},
   'paper_id': 'doi:10.1016/j.celrep.2020.108641'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper's primary focus is on Fluorescence Lifetime Imaging Microscopy (FLIM), which is a technique within the broader category of fluorescence microscopy."},
   'paper_id': 'doi:10.1371/journal.pone.0238327'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper describes a microscopy-based assay which implies the use of optical microscopy, and the mention of 'high-content' suggests the use of fluorescence microscopy to generate high-content data from samples."},
   'paper_id': 'doi:10.1002/bies.202000257'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide specific details on the imaging technology used, but it describes a ubiquitin-mediated mechanism and the regulation of a receptor by an E3 ubiquitin ligase, which suggests a biochemical or molecular biology focus, potentially involving computational analysis of data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.7554/elife.63452'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses lattice light sheet microscopy, which is a type of fluorescence microscopy, to investigate the structural dynamics of macropinosome formation.'},
   'paper_id': 'doi:10.1101/2020.12.01.390195'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not discuss any specific imaging technology being used in the study. Instead, it focuses on the pharmacological effects of PTC299 on SARS-CoV-2 replication and cytokine production. Since no imaging technology from the provided list is mentioned or implied, the closest category would be computational analysis for the study of drug effects, although this is not explicitly stated.'},
   'paper_id': 'doi:10.1016/j.virusres.2020.198246'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper focuses on ImageJ, which is an open-source image analysis software platform, indicating that the primary technology used is computational analysis for image data.'},
   'paper_id': 'doi:10.1002/pro.3993'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily uses 18F-fluoride, which is a radiotracer, as a non-invasive imaging biomarker. Radiotracers are a type of molecular imaging probe used to visualize biological processes.'},
   'paper_id': 'doi:10.1038/s41598-020-77391-6'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The abstract mentions the use of 3D electron microscopy reconstructions, which falls under the category of Other electron microscopy technologies such as Transmission EM or Scanning EM.'},
   'paper_id': 'doi:10.1016/j.chom.2020.11.003'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper uses tetramethylrhodamine methyl ester (TMRM), which is a fluorescent dye used to measure mitochondrial membrane potential, indicating that fluorescence microscopy is the primary imaging technology used.'},
   'paper_id': 'doi:10.1096/fj.202001693r'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The title of the paper indicates the use of PET/CT, which stands for Positron Emission Tomography/Computed Tomography, a type of imaging technology often grouped under the broader category of MRI as it involves tomographic imaging and is used in a similar context.'},
   'paper_id': 'doi:10.1016/j.ijcha.2020.100672'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify an imaging technology used in the research; therefore, the focus seems to be on computational analysis of genetic data rather than on a specific imaging technology.'},
   'paper_id': 'doi:10.1016/j.plantsci.2020.110750'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper discusses a denoising method for diffusion-weighted magnetic resonance imaging (dMRI), which is a type of MRI, indicating that magnetic resonance imaging is the primary imaging technology used.'},
   'paper_id': 'doi:10.1016/j.neuroimage.2020.117539'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': 'The abstract mentions the use of mass spectrometry to identify a novel HDAC6-regulated tau acetylation site, which suggests that this technology is used for analyzing protein modifications in the study.'},
   'paper_id': 'doi:10.1038/s41467-020-19317-4'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper primarily uses 18F-GP1 positron-emission tomography-computed tomography (PET-CT), which is a type of molecular imaging that falls under the broader category of Magnetic Resonance Imaging (MRI).'},
   'paper_id': 'doi:10.1093/ehjci/ehaa946.0290'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The title of the paper does not suggest the use of any specific imaging technology. Instead, it implies the paper is about management or operational rules for women principal investigators during a pandemic, which is likely to involve computational analysis in a broader sense rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1371/journal.pcbi.1008370'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract does not explicitly mention any imaging technology used in the study. However, since the study involves the analysis of cell proliferation, viability, migration, and the expression of proteins, it is likely that some form of computational analysis was involved to process and analyze data from these experiments. Given the lack of specific information, 'Other computational analysis' is the most appropriate category."},
   'paper_id': 'doi:10.3390/ijms21217845'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': 'The abstract does not mention any specific imaging technology being used; it focuses on biochemical assays, such as measuring inhibition constants and minimum inhibitory concentrations. However, mass-spectrometry is a common technique in studying enzyme interactions and drug discovery, which could be involved in determining the potency of herbicides against the enzyme from Candida auris.'},
   'paper_id': 'doi:10.1021/acsinfecdis.0c00229'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions a 'novel fluorescence-based micro-level matching approach', which indicates the use of fluorescence microscopy technology in their research."},
   'paper_id': 'doi:10.48550/arxiv.2010.11438'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper discusses improvements in the characterization of optical properties using high numerical aperture optical coherence tomography (OCT), which is an optical imaging hardware technology not specifically listed in the other categories.'},
   'paper_id': 'doi:10.1117/1.nph.7.4.045005'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper discusses the development and use of cell penetrating peptides (CPPs) as a molecular scaffold for delivering cargo into cells, which implies the use of molecular probes. No specific imaging technology is mentioned in the context provided.'},
   'paper_id': 'doi:10.1039/d0cb00099j'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the use of imaging technology, but it mentions the generation of zebrafish models and the screening of compounds, implying that the study primarily uses computational and genetic methods rather than imaging technology.'},
   'paper_id': 'doi:10.1080/15548627.2020.1833500'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify any imaging technology used, instead it mentions the use of an automated robotics pipeline and a mutation library, which implies a computational analysis of genetic data rather than the use of a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41586-020-2816-5'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper uses 'quantitative, multidimensional imaging of mammary cell ensembles from GCaMP6 transgenic mice,' which suggests the use of a fluorescence-based imaging technique, as GCaMP6 is a genetically encoded calcium indicator used in fluorescence microscopy."},
   'paper_id': 'doi:10.1073/pnas.2016905117'},
  {'classification': {'imaging_technology_code': 'N',
    'imaging_technology_name': 'Computational analysis systems',
    'explanation': 'The paper focuses on a novel algorithm for optimizing the removal of speckle noise from OCT images, which falls under computational methods tightly integrated with the sensing system.'},
   'paper_id': 'doi:10.1101/2020.10.07.329227'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The abstract does not specify the imaging technology used; however, the study of cadherin function in tissue cohesion during neurulation often involves the use of molecular imaging probes or stains to visualize proteins and cellular structures, making this category the most likely.'},
   'paper_id': 'doi:10.7554/elife.58626'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide explicit information about the imaging technology used. However, it discusses the regulation of actin polymerization/depolymerization dynamics and cell migration, which suggests that computational analysis methods might be involved in analyzing these dynamic processes. Without specific mention of imaging technologies, this categorization is speculative.'},
   'paper_id': 'doi:10.1126/sciadv.abb7854'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper discusses the development and translation of novel brain radiotracers, which are molecular imaging probes, for use in PET and SPECT imaging technologies.'},
   'paper_id': 'doi:10.3389/fnins.2020.00871'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The provided text does not contain enough information to conclusively determine the specific imaging technology used. However, since it is an editorial article for the Medical Physics and Imaging section, it is likely to discuss various technologies rather than focus on one specific type. Therefore, without explicit context, 'Other computational analysis' is chosen as a placeholder."},
   'paper_id': 'doi:10.3389/fphy.2020.00344'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper primarily uses force volume atomic force microscopy (AFM), which is a type of imaging technology that does not fit into the specific categories listed but is an established imaging hardware technology used to map mechanical properties at the microscale.'},
   'paper_id': 'doi:10.1038/s41598-020-72564-9'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The abstract does not provide specific details on the imaging technology used, but it mentions employing 'a range of biophysical methodologies with model membranes and bioassays,' which suggests that the study may have used specialized imaging hardware not described by the other categories."},
   'paper_id': 'doi:10.1016/j.bbamem.2020.183480'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The abstract mentions the use of in vivo micro-computed tomography (micro-CT), which is a type of imaging hardware technology. It does not fit precisely into the more specific categories listed, so 'Other imaging hardware technology' is the most appropriate classification."},
   'paper_id': 'doi:10.1016/j.ejphar.2020.173538'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': "The paper discusses 'nucleosome affinity proteomics', which is a technique that commonly relies on mass spectrometry to analyze protein interactions."},
   'paper_id': 'doi:10.1093/nar/gkaa544'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the development and use of a novel TSPO radiotracer, which is a type of molecular imaging probe, for PET scans to detect inflammation after myocardial infarction.'},
   'paper_id': 'doi:10.2967/jnumed.120.243600'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the use of any imaging technology, thus classification cannot be determined based on the provided context sentences.'},
   'paper_id': 'doi:10.1002/sctm.20-0143'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract mentions the use of enhanced green fluorescent protein (EGFP) for measuring diffusion in live cells, indicating the use of fluorescence microscopy techniques.'},
   'paper_id': 'doi:10.35459/tbp.2019.000111'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': 'The paper discusses the nanoclustering of cell surface receptors and signaling components, which suggests the use of imaging technology capable of visualizing phenomena at the nanoscale. Super-resolution microscopy fits this requirement as it is designed to resolve structures beyond the diffraction limit of light, allowing for nanoscale visualization.'},
   'paper_id': 'doi:10.1038/s41467-020-17602-w'},
  {'classification': {'imaging_technology_code': 'N',
    'imaging_technology_name': 'Computational analysis systems',
    'explanation': 'The paper primarily discusses the use of genome-wide association analysis and computational methods to identify genetic factors influencing the auditory brainstem response, which is a computational analysis rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1007/s10162-020-00762-3'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The paper's title suggests that it is focused on a measurement technique for facemask efficacy, which likely involves some form of imaging hardware to visualize expelled droplets during speech. However, due to the lack of specific information in the abstract, it is not possible to confidently categorize it into one of the more specific imaging technologies listed, so it is classified as 'Other imaging hardware technology'."},
   'paper_id': 'doi:10.7924/r4ww7dx6q'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper's title mentions 'light-sheet imaging,' which is a type of fluorescence microscopy, and the abstract discusses lattice light sheet microscopy, which is a specific type of light-sheet fluorescence microscopy."},
   'paper_id': 'doi:10.1002/jlb.1ce0320-082'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper uses single-cell RNA sequencing, flow cytometry, and lineage tracing, which are computational methodologies for analyzing biological data rather than specific imaging technologies.'},
   'paper_id': 'doi:10.1161/circulationaha.119.042863'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract mentions the use of 'molecular docking' to study interactions, which suggests a computational analysis method to analyze molecular structures, but does not explicitly mention a specific imaging technology."},
   'paper_id': 'doi:10.1021/acs.molpharmaceut.0c00417'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper primarily discusses the use of diffusion MRI (dMRI), which is a magnetic resonance imaging technique that measures the diffusion of water molecules to generate contrast in MR images.'},
   'paper_id': 'doi:10.1038/s42003-020-1093-z'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The abstract does not specify the type of imaging technology used, but it does refer to 'expert imaging scientists' and implies the use of shared instruments, which suggests a focus on imaging hardware technology that does not fit a specific category listed."},
   'paper_id': 'doi:10.1016/j.tcb.2020.06.001'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses improving the contrast and resolution of optical microscopy images through computational means such as deconvolution and computational fusion, without specifying a new optical hardware technology.'},
   'paper_id': 'doi:10.1038/s41587-020-0560-x'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper describes a wireless, bioresorbable sensor for monitoring internal body temperatures, which implies the development of a novel form of non-imaging sensor hardware technology rather than a specific imaging technique.'},
   'paper_id': 'doi:10.1002/adhm.202000942'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper discusses advancements in diffusion-weighted MRI (dMRI), which is a technique within the field of magnetic resonance imaging.'},
   'paper_id': 'doi:10.1002/jmri.27247'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper primarily uses positron emission tomography-computed tomography (PET/CT) with 18F-fluorodeoxyglucose, which is a molecular imaging technology but when mentioned in combination with PET/CT, it falls under the broader category of imaging modalities used in conjunction with MRI, commonly known as 'Molecular imaging'."},
   'paper_id': 'doi:10.1038/s41467-020-16878-2'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract does not explicitly mention the imaging technology used; however, the study of actin dynamics and the immunological synapse typically relies on fluorescence microscopy techniques to visualize these processes in live cells.'},
   'paper_id': 'doi:10.1101/2020.06.13.150045'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': 'The paper primarily discusses the preparation of cellular specimens for cryo-electron microscopy, which is included in the cryo-electron tomography category.'},
   'paper_id': 'doi:10.1101/2020.06.12.147678'},
  {'classification': {'imaging_technology_code': 'E',
    'imaging_technology_name': 'Nonlinear microscopy',
    'explanation': 'The paper uses two-photon phosphorescence lifetime microscopy, which is a type of nonlinear microscopy, to investigate the microvascular oxygenation and blood flow responses.'},
   'paper_id': 'doi:10.1177/0271678x20928011'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': 'The abstract does not explicitly mention an imaging technology; however, it discusses metabolic characteristics and spare respiratory capacity (SRC), which are typically studied using mass spectrometry to analyze metabolic profiles. Therefore, it is likely that mass spectrometry is the primary technology used in this paper.'},
   'paper_id': 'doi:10.1038/s41467-020-16633-7'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': 'The paper focuses on a technique called SuperSTORM, which is a form of super-resolution microscopy, specifically an advanced form of STORM (stochastic optical reconstruction microscopy). This places it clearly in the super-resolution category.'},
   'paper_id': 'doi:10.1016/j.xpro.2019.100012'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper describes the development of a novel radiotracer targeting Factor XIIIa, which is a type of molecular imaging probe used to identify and quantify thrombosis in vivo.'},
   'paper_id': 'doi:10.1093/ehjci/jez207'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'Although the abstract does not explicitly mention the imaging technology, the context of work suggests the use of some form of fluorescence microscopy to visualize microtubule dynamics, actomyosin contractility, and protrusion retraction in migrating cells.'},
   'paper_id': 'doi:10.1083/jcb.201907154'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technology. The focus is on the physiological effects of drugs on renal disease in a model of salt-sensitive hypertension, and the work likely involves computational analysis of data rather than imaging.'},
   'paper_id': 'doi:10.1152/ajprenal.00125.2020'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not directly mention the use of any specific imaging technology. The focus is on cellular and molecular interactions and pathways, suggesting that the primary methodology may involve computational analysis of data rather than direct imaging.'},
   'paper_id': 'doi:10.1016/j.jtcvs.2020.03.166'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': 'The abstract mentions the use of tandem MS (mass spectrometry) to determine the amino acid sequences of the cyclotides, indicating that mass spectrometry is the primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.1074/jbc.ra120.012627'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify a particular imaging technology used for the research, but it discusses the analysis of cell locomotion and actin cytoskeleton dynamics, which likely involves computational analysis of image data.'},
   'paper_id': 'doi:10.1038/s41586-020-2283-z'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention any imaging technology being used for the research. The focus is on the biochemical interaction and regulation mechanisms of enzymes involved in polyubiquitination. Therefore, it is likely that computational analysis tools are used to understand these interactions rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41594-020-0424-6'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper discusses the cellular uptake and cytosolic delivery of a cyclic cystine knot scaffold, which is a type of molecular probe used to target intracellular proteins. There is no mention of specific imaging technologies, suggesting the focus is on the molecular probe itself.'},
   'paper_id': 'doi:10.1021/acschembio.0c00297'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'quantitative confocal microscopy', which falls under the category of fluorescence microscopy. This suggests that fluorescence microscopy is the primary imaging technology used in the study."},
   'paper_id': 'doi:10.1164/rccm.201908-1513oc'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide specific details about imaging technologies used in the study. Therefore, based on the available information and the absence of explicit mention of imaging technology, it is not possible to confidently assign a category from the provided list.'},
   'paper_id': 'doi:10.1016/j.jid.2020.03.965'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract does not specify the imaging technology used directly, but the study of lamellipodia, adherens junctions, and actin regulators typically involves fluorescence microscopy techniques, which allows for visualization of specific proteins and cellular structures in live or fixed cells.'},
   'paper_id': 'doi:10.1101/2020.04.20.052118'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of Fluoro-Jade C (FJC) staining for evaluating neuropathology, which is a fluorescence staining technique used in microscopy to label degenerating neurons.'},
   'paper_id': 'doi:10.1007/s00204-020-02747-w'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses Total Internal Reflection Fluorescence (TIRF) microscopy, which is a type of fluorescence microscopy, to study the dynamics of cellular structures.'},
   'paper_id': 'doi:10.1074/jbc.ra119.011903'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract mentions the use of a model to determine how well gobies matched their backgrounds in terms of luminance contrast to a potential predator, which suggests the use of computational methods to analyze visual data rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1111/1365-2656.13226'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract does not provide explicit information about the imaging technology used. However, given the context of the work with zebrafish embryonic tissue explants and the study of cellular organization, fluorescence microscopy is commonly used in such biological studies for visualizing different cell types and developmental stages.'},
   'paper_id': 'doi:10.7554/elife.55190'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper primarily uses positron emission tomography (PET) for assessing myocardial fibrosis activity, but PET is not listed as an option. However, within the context section, the authors mention the use of magnetic resonance imaging (MRI) as a method for validating PET imaging results, which suggests that MRI is a significant component of the study's imaging methodology."},
   'paper_id': 'doi:10.1136/heartjnl-2020-scf.26'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper introduces a new imaging method called crossed-beam pump-probe microscopy, which is an optical hardware technique that does not fit into the specific categories listed but is related to microscopy and imaging hardware technology.'},
   'paper_id': 'doi:10.1364/oe.389004'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper primarily compares different sampling schemes and reconstruction methods of high angular resolution diffusion MRI (dMRI), which is a variant of magnetic resonance imaging.'},
   'paper_id': 'doi:10.1016/j.neuroimage.2020.116704'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper focuses on the use of various fluorescent proteins as molecular probes to visualize Cryptococcus neoformans, which is a form of molecular imaging.'},
   'paper_id': 'doi:10.1016/j.fgb.2020.103365'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper discusses improvements in reconstruction of diffusion weighted 3D multishot echo planar imaging (EPI), which is a technique used in MRI to image the diffusion process of water molecules in biological tissues.'},
   'paper_id': 'doi:10.1002/mrm.28231'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses optimizing conditions for live-cell fluorescence imaging, which is a subset of fluorescence microscopy technologies.'},
   'paper_id': 'doi:10.1242/jcs.242834'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide enough information to determine the specific imaging technology used; therefore, I cannot confidently assign a category from the provided list.'},
   'paper_id': 'doi:10.1177/1535370220903012'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract does not provide specific details about imaging technology, but it mentions 'transcriptomic analyses,' which implies a computational analysis of gene expression data rather than a physical imaging technology."},
   'paper_id': 'doi:10.3389/fphys.2020.00088'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper discusses manganese-enhanced MRI (MEMRI) and T1 mapping MRI for assessing myocardial viability, indicating that the primary imaging technology used is magnetic resonance imaging (MRI).'},
   'paper_id': 'doi:10.1038/s41598-020-58716-x'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper does not explicitly mention the use of imaging technology. However, it discusses the inhibition of vascular calcification, which often involves assessing calcified tissues, potentially using molecular imaging probes or stains to visualize the effects. Since no direct imaging method is referenced, this categorization is the closest fit based on the available information.'},
   'paper_id': 'doi:10.1038/s41467-019-14091-4'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': 'The abstract mentions the use of superresolution microscopy, which is a technique classified under super-resolution imaging technologies.'},
   'paper_id': 'doi:10.1172/jci124000'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': "The abstract mentions a 'pulse-chase labeling' technique, which is a method involving the use of molecular probes or labels to track the incorporation and turnover of proteins within cells. This suggests the use of molecular imaging techniques."},
   'paper_id': 'doi:10.1038/s41467-019-14276-x'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper discusses wireless, battery-free photometry systems for calcium dynamics monitoring in neurons, which implies the use of fluorescence to record signals. This is indicative of some form of fluorescence microscopy, but not a specific advanced type such as super-resolution or multiphoton, thus falling under the broader category of 'Other fluorescence microscopy'."},
   'paper_id': 'doi:10.1073/pnas.1920073117'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses an automated ImageJ macro for quantifying dorsal cell surface protrusions from 3D microscope images, which implies a computational methodology for image data analysis rather than a specific imaging technology.'},
   'paper_id': 'doi:10.21769/bioprotoc.3494'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not mention any specific imaging technology used for the primary research within the paper. The focus is on the biological function and impact of p66ShcA on breast cancer metastasis, which suggests that the paper likely relies on computational analysis of data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1186/s13058-020-1245-6'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper describes the development of multimodal probes, immunoactive gold nanostars (IGNs), which are used to detect immunomarkers. This falls under the category of molecular imaging probes as it involves manipulating biomedical samples to generate a visual signal.'},
   'paper_id': 'doi:10.1021/acsnano.9b07326'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper discusses the development of a miniaturized wireless pacemaker for small animal models, which implies the use of specialized hardware technology for implantation and operation, but does not specify the use of any particular imaging technology from the provided list.'},
   'paper_id': 'doi:10.1038/s41467-019-13637-w'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Positron emission tomography',
    'explanation': 'The title of the article specifies PET imaging, which stands for positron emission tomography, indicating that this is the primary imaging technology discussed in the paper.'},
   'paper_id': 'doi:10.1093/cvr/cvz162'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses AAV vectors encoding enhanced green fluorescent protein (EGFP) for brain transduction studies, which implies the use of fluorescence microscopy to visualize the EGFP.'},
   'paper_id': 'doi:10.1089/humc.2019.030'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the development and use of a new molecular probe, CAPRYDAA, for imaging membrane dynamics, which fits into the category of molecular imaging probes.'},
   'paper_id': 'doi:10.1039/c9tb01738k'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of translational fusion proteins with a GFP reporter expressed in Nicotiana benthamiana leaves, which indicates the use of fluorescence microscopy to determine the localization of the proteins.'},
   'paper_id': 'doi:10.3390/ijms20235855'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The abstract mentions the development of MR agents and the detection of enzyme activity in a mouse model using MR imaging, which indicates that the primary imaging technology used in the paper is Magnetic Resonance Imaging (MRI).'},
   'paper_id': 'doi:10.1002/anie.201909933'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The proximity ligation assays mentioned in the abstract suggest the use of fluorescence microscopy techniques to investigate molecular interactions, which is consistent with category G.'},
   'paper_id': 'doi:10.1016/j.bcp.2019.113728'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "While the abstract does not explicitly state the use of MRI, the use of microCT analysis suggests that the paper primarily uses imaging technology related to tomography. Since 'microCT' or 'X-ray computed tomography' is not listed as an option, MRI is the closest relevant category from the list that involves detailed imaging and analysis of internal structures."},
   'paper_id': 'doi:10.1155/2019/6956391'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The study mentions the use of '123I-PE2I single-photon emission computerized tomography scans targeting the dopamine transporter,' which is indicative of a form of nuclear imaging related to MRI, rather than the exact technologies listed."},
   'paper_id': 'doi:10.1093/brain/awz296'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'light microscopy', which is a broad category that typically includes various types of fluorescence microscopy techniques. As there is no specific mention of super-resolution or nonlinear microscopy, it is most appropriate to classify it under 'Other fluorescence microscopy'."},
   'paper_id': 'doi:10.48550/arxiv.1910.11370'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the use of any imaging technology. The focus is on the genetic and biochemical interactions between HIV-1 and human TRIM5α, which suggests computational analysis of genetic data or biochemical assays rather than imaging.'},
   'paper_id': 'doi:10.1038/s41564-019-0592-5'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper focuses on computational methods for segmenting cell nuclei in two-dimensional light microscopy images without human interaction, which falls under computational analysis of image data.'},
   'paper_id': 'doi:10.1038/s41592-019-0612-7'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper does not explicitly mention the imaging technology used, but the study of actin allocation in microvilli suggests the possible use of molecular probes or stains to visualize F-actin and G-actin within the cells, which is common in cellular and molecular biology research.'},
   'paper_id': 'doi:10.1016/j.cub.2019.08.051'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'Although the abstract does not specify the imaging technology used, the study of nucleoporin condensates and NPC assembly in Drosophila oogenesis likely involves visualizing fluorescently tagged proteins, which is a common application of fluorescence microscopy.'},
   'paper_id': 'doi:10.1016/j.cell.2019.09.022'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The paper discusses standardization in preclinical PET/CT imaging, which is a hardware technology used for noninvasive imaging but does not fit into the specific categories listed. PET (Positron Emission Tomography) and CT (Computed Tomography) are both well-established imaging technologies in their own right, and the paper's focus is on improving their quantitative accuracy and reproducibility."},
   'paper_id': 'doi:10.2967/jnumed.119.231308'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The paper's title mentions the use of 'cryo-EM' (cryo-Electron Microscopy), which is a form of Cryo-electron tomography, to determine the structure of the Macrophage-expressed gene 1."},
   'paper_id': 'doi:10.1038/s41467-019-12279-2'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technologies from the provided list. The focus appears to be on molecular and cellular processes rather than imaging, suggesting that computational analysis methods are likely used to analyze biochemical data.'},
   'paper_id': 'doi:10.1126/sciadv.aax1978'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper mentions the use of 'fluorescently labeled cell membranes' for imaging the Caenorhabditis elegans embryo, which indicates the use of fluorescence microscopy technology."},
   'paper_id': 'doi:10.1109/access.2019.2940161'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify a particular imaging technology but mentions employing biochemical and biophysical methodologies for the analysis. This suggests that the paper may involve computational analysis of biochemical data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1021/acschembio.9b00593'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of 'temporal image correlation microscopy,' which is a fluorescence microscopy technique, to study the dynamics of adhesions and protein binding."},
   'paper_id': 'doi:10.1038/s41598-019-47722-3'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the use of a molecular imaging probe, [18F]PBR111, in conjunction with positron emission tomography (PET) to monitor neuroinflammation.'},
   'paper_id': 'doi:10.1093/toxsci/kfz096'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'live cell imaging' which suggests the use of a form of microscopy suitable for visualizing live cells, likely involving fluorescence techniques to observe dynamic processes such as microvillar motility."},
   'paper_id': 'doi:10.1016/j.devcel.2019.07.008'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': "The title of the paper, 'Electron Ptychography of Single Biological Macromolecules', suggests that the primary technology used is a type of electron microscopy, specifically involving ptychography."},
   'paper_id': 'doi:10.1017/s1431927619001090'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses the use of the fluorescence lifetime of NADH and the phasor approach to study cellular metabolism, which falls under the category of fluorescence microscopy, specifically fluorescence lifetime imaging microscopy (FLIM).'},
   'paper_id': 'doi:10.1002/jbio.201900156'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention imaging technology used for the primary findings. However, it references the use of RNA-sequencing and siRNA approaches, which are computational and molecular biology techniques, respectively. Since the focus is on the analysis of gene expression data and the manipulation of RNA sequences, it fits better under computational analysis rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1161/circresaha.119.314876'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The abstract mentions the use of immunohistochemical detection, which typically involves the use of antibodies and/or stains to visualize specific proteins or cell types, indicating the use of molecular imaging probes.'},
   'paper_id': 'doi:10.1016/j.bbr.2019.112080'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': "The abstract mentions the 'in-depth analysis of the mitochondrial proteome,' which suggests the use of mass spectrometry for protein analysis and identification."},
   'paper_id': 'doi:10.1016/j.ekir.2019.05.133'},
  {'classification': {'imaging_technology_code': 'E',
    'imaging_technology_name': 'Nonlinear microscopy',
    'explanation': "The paper primarily discusses a two-photon fluorescence microscopy technique, which falls under the category of Nonlinear microscopy, as indicated by terms like 'two-photon fluorescence', 'SHG', and 'THG' (Second and Third Harmonic Generation)."},
   'paper_id': 'doi:10.3390/mps2020053'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the use of any imaging technology. The focus is on dietary intervention effects on muscle function in Drosophila, and there is no explicit mention of imaging technologies used to observe these effects.'},
   'paper_id': 'doi:10.1038/s41467-019-10563-9'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The abstract does not specify the imaging technology used, but given the study's cellular and molecular focus, it likely involves some form of microscopy not detailed in the provided text. Since none of the specific microscopy technologies listed are mentioned, 'Other imaging hardware technology' is the most appropriate classification by exclusion."},
   'paper_id': 'doi:10.1371/journal.pgen.1008228'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper introduces a discussion forum for scientific image software, focusing on digital image analysis, which implies a computational methodology to analyze image data.'},
   'paper_id': 'doi:10.1371/journal.pbio.3000340'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper utilizes Dual-view Inverted Selective Plane Illumination Microscopy (diSPIM), which is a type of light-sheet fluorescence microscopy, fitting under the 'Other fluorescence microscopy' category."},
   'paper_id': 'doi:10.3791/59533'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'quantitative, multidimensional imaging of mammary cell ensembles' and 'imaging activity across scales', which suggests the use of a fluorescence microscopy technique capable of capturing dynamic processes in cells, potentially involving Ca2+ indicator dyes."},
   'paper_id': 'doi:10.1101/657510'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the use of [18F]AB5186, a novel radiotracer for PET imaging, which falls under the category of molecular imaging probes used to generate visual signals for detecting inflammation in disease.'},
   'paper_id': 'doi:10.1371/journal.pone.0217515'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper uses an engineered EGFP reporter, which is a green fluorescent protein commonly used in fluorescence microscopy, suggesting that fluorescence microscopy is the primary imaging technology employed.'},
   'paper_id': 'doi:10.1016/j.scr.2019.101470'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of 'fluorescently-labeled ChTxβ' for tracking anterograde transport in optic nerves, which suggests the use of fluorescence microscopy techniques."},
   'paper_id': 'doi:10.1523/jneurosci.1760-18.2019'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technology. The focus is on the metabolic characteristics of cells, which suggests that the study may involve computational analysis of data rather than imaging technologies.'},
   'paper_id': 'doi:10.4049/jimmunol.202.supp.60.7'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of fluorescence lifetime imaging microscopy (FLIM), which is a technique under the category of other fluorescence microscopy.'},
   'paper_id': 'doi:10.1194/jlr.m091967'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper is focused on Positron Emission Tomography (PET), which is a type of imaging technology related to magnetic resonance imaging, used for observing metabolic processes in the body.'},
   'paper_id': 'doi:10.1016/j.jchromb.2019.04.026'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper describes the use of positron emission tomography (PET), which is a type of molecular imaging closely related to MRI, often categorized under the broader umbrella of MRI technology due to its use in clinical neuroimaging.'},
   'paper_id': 'doi:10.1038/s41398-019-0449-y'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The context of the work mentions the use of postmortem magnetic resonance imaging of the lumbar spine to measure spinal angulation and cross-sectional area, indicating MRI is the primary imaging technology used in the paper.'},
   'paper_id': 'doi:10.1159/000496201'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of a specific imaging technology, but it discusses the ability of leukocytes to navigate and choose paths, which suggests the involvement of computational analysis to understand and model their migration behavior. Since no specific hardware technology is mentioned, it is most likely that the paper involves computational analysis of existing imaging data.'},
   'paper_id': 'doi:10.1038/s41586-019-1087-5'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract discusses designing rigorous light microscopy experiments, which suggests that the primary technology used is related to light microscopy. Among the options, 'Other fluorescence microscopy' is a broad category that includes various light microscopy techniques that may not be specified here but are related to fluorescence."},
   'paper_id': 'doi:10.1083/jcb.201812109'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of time-lapse microscopy, which is a technique typically associated with fluorescence microscopy for live-cell imaging.'},
   'paper_id': 'doi:10.15252/msb.20188604'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract does not provide specific details on the imaging technology used; however, it mentions the use of 'transposon-directed insertion site sequencing' and 'reporter systems' as methods to investigate bacterial responses, which suggests a focus on genetic and computational analysis rather than a specific imaging technology."},
   'paper_id': 'doi:10.1073/pnas.1820870116'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper utilizes a comparative genomics approach, which implies computational analysis of genomic data to identify enhancers. There is no mention of specific imaging technology being used.'},
   'paper_id': 'doi:10.1007/s00427-019-00629-5'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper primarily uses 18F-fluoride positron emission tomography (PET) combined with computed tomography (CT), which falls under the broader category of magnetic resonance imaging technologies.'},
   'paper_id': 'doi:10.1016/j.jacc.2018.12.056'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide specific details on the imaging technology used; therefore, it is not possible to confidently assign it to any of the imaging categories listed.'},
   'paper_id': 'doi:10.1038/s41598-019-39867-y'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses the light sources used in fluorescence microscopy, indicating that it is related to fluorescence microscopy technologies.'},
   'paper_id': 'doi:10.1007/s00418-019-01776-6'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the use of any imaging technology directly. The study seems to be focused on molecular interactions and their effects on autophagy, which suggests that the primary technology could be related to computational analysis of data rather than a specific imaging modality.'},
   'paper_id': 'doi:10.1016/j.cell.2019.01.030'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide explicit details about the imaging or analytical technologies used. However, it discusses the regulation of DNA metabolism and proliferation, implying that the study likely involves computational analysis of genetic or cellular data rather than a specific imaging technology.'},
   'paper_id': 'doi:10.1038/s41598-019-39078-5'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper discusses a novel nitroalkene-α-tocopherol analogue (NATOH) that is designed to be incorporated into lipoproteins and taken up by atheroma lesions. This suggests the use of a molecular probe for targeting and visualizing the lesions, but the abstract does not specify imaging technology, so this classification is based on the broader context of the work involving molecular probes.'},
   'paper_id': 'doi:10.1111/bph.14561'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of 'lattice light sheet imaging' and 'fluorescently-tagged Rab8a', which are indicative of fluorescence microscopy techniques. Lattice light sheet microscopy is a type of light-sheet fluorescence microscopy."},
   'paper_id': 'doi:10.1098/rstb.2018.0151'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper describes the use of time-lapse microscopy in combination with image analysis, which falls under computational methodology to analyze image data.'},
   'paper_id': 'doi:10.21769/bioprotoc.3137'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper primarily discusses the use of 18F-fluoride as a ligand for PET/CT imaging to identify microcalcifications in atherosclerotic plaques, which classifies it as a molecular imaging probe.'},
   'paper_id': 'doi:10.1161/circimaging.118.007835'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract provided does not contain any information about the content of the paper, but the title indicates it is a collection of general abstracts from a congress. Therefore, it is likely that a range of imaging technologies were discussed, including various computational analysis methods to analyze image data.'},
   'paper_id': 'doi:10.1007/s11307-018-01305-2'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The abstract mentions the use of 99mTc-duramycin for whole-body imaging, which is a radiopharmaceutical imaging probe used to identify dead and dying cells, fitting the category of molecular imaging probes.'},
   'paper_id': 'doi:10.1158/1078-0432.ccr-18-1846'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The abstract does not explicitly mention the imaging technology used, but the study of microvilli morphology, packing, and brush border structure in tissues typically involves electron microscopy, particularly TEM or SEM, for high-resolution imaging at the cellular and subcellular level.'},
   'paper_id': 'doi:10.1091/mbc.e18-09-0558'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of a specific imaging technology. The focus is on signaling pathways, protein interactions, and cellular processes. The study likely relies on computational analysis of data rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1074/jbc.ra118.002779'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper primarily uses Optical Coherence Microscopy (OCM), which is an optical imaging technique that does not fit into the more specific categories listed but is a type of imaging hardware technology.'},
   'paper_id': 'doi:10.1007/s00429-018-1777-z'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract does not explicitly mention a specific imaging technology but discusses a 'detailed quantitative live-cell imaging analysis', which suggests the use of computational methods to analyze image data rather than a specific imaging hardware technology."},
   'paper_id': 'doi:10.1091/mbc.e18-04-0242'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses fluorescence resonance energy transfer (FRET), which is a technique within the domain of fluorescence microscopy, to study protein-protein interactions.'},
   'paper_id': 'doi:10.1002/pro.3482'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract mentions the use of an endogenous fluorescent reporter for OCT4, which implies the use of fluorescence microscopy to track single-cell OCT4 levels.'},
   'paper_id': 'doi:10.15252/msb.20178140'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the imaging technology used, but it mentions the measurement of autophagic flux, which implies a computational analysis of data rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1111/nep.13251'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses lattice light sheet microscopy (LLSM), which is a type of advanced fluorescence microscopy, to image live macrophages.'},
   'paper_id': 'doi:10.1083/jcb.201804137'},
  {'classification': {'imaging_technology_code': 'H',
    'imaging_technology_name': 'Cryo-electron tomography',
    'explanation': "The abstract mentions the use of 'the 3.9\u2009Å resolution cryo EM structure' to study C9 in a polymeric assembly, indicating that cryo-electron microscopy is the primary imaging technology used in this paper."},
   'paper_id': 'doi:10.1038/s41467-018-05717-0'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The abstract does not specify any particular imaging technology but generally refers to the sophistication of scientific equipment and instrumentation in core facilities, which suggests a broad array of technologies could be included.'},
   'paper_id': 'doi:10.1101/cshperspect.a032805'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The abstract does not provide explicit details about the imaging technology used. However, the study of neutrophil extracellular traps (NETs), cell death, and the mechanisms involving gasdermin D typically involves molecular imaging techniques such as fluorescence microscopy, which may use probes or stains to visualize these biological processes.'},
   'paper_id': 'doi:10.1126/sciimmunol.aar6676'},
  {'classification': {'imaging_technology_code': '',
    'imaging_technology_name': '',
    'explanation': 'The provided abstract text is empty, hence there is no information available to make an informed decision regarding the imaging technology used in the paper.'},
   'paper_id': 'doi:10.1038/s41592-018-0077-0'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract mentions the development of an interactive web interface (the Cell Cycle Browser), which is based on computational models for simulating and predicting cell cycle dynamics, indicating the primary technology is computational analysis rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1016/j.cels.2018.06.004'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper mentions the use of 'Serial CT scanning' to assess the spines of mice, which is a technique related to Magnetic resonance imaging."},
   'paper_id': 'doi:10.1016/j.envres.2018.06.049'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of an imaging technology; instead, it discusses the development of a mathematical model for studying the dynamics of small GTPases Rac and Rho, which falls under computational analysis.'},
   'paper_id': 'doi:10.1371/journal.pcbi.1006303'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The paper discusses CellProfiler 3.0, which is software for image analysis, indicating the technology is computational rather than a specific imaging hardware technology.'},
   'paper_id': 'doi:10.1371/journal.pbio.2005970'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The paper likely uses electron microscopy to determine the size of the Helicobacter pylori OMVs and to analyze their protein content, as this is a common technique for studying small biological structures like vesicles. The abstract does not explicitly mention the imaging technology used, but electron microscopy is a standard tool for such analyses.'},
   'paper_id': 'doi:10.3389/fimmu.2018.01466'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper's title 'Characterisation of Dorsal Ruffles in Activated Macrophages' suggests the use of some form of microscopy to visualize small structures within cells. Since 'Dorsal Ruffles' are cellular features, they are likely studied using fluorescence microscopy, a common method for such cellular and subcellular characterizations. The abstract text was not provided, so the decision is based solely on the title and common practices in cell biology."},
   'paper_id': 'doi:10.14264/uql.2018.383'},
  {'classification': {'imaging_technology_code': 'J',
    'imaging_technology_name': 'Other electron microscopy',
    'explanation': 'The abstract mentions the characterization of nemaline bodies in the zebrafish model using electron-dense imaging, which implies the use of electron microscopy technology.'},
   'paper_id': 'doi:10.1186/s40478-018-0546-9'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'in vivo precision morphometry together with analysis of cytoskeletal dynamics in vitro,' which suggests the use of advanced microscopy techniques. Fluorescence microscopy is a common technique for such studies, especially when investigating cytoskeletal dynamics."},
   'paper_id': 'doi:10.1038/s41590-018-0109-z'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The title of the paper references '[18F]-FDG PET/CT imaging,' which stands for positron emission tomography/computed tomography imaging using the radiotracer fluorodeoxyglucose (FDG). PET/CT is a molecular imaging technique, which is closely related to MRI in the broader category of imaging technologies, especially in the context of functional and metabolic imaging."},
   'paper_id': 'doi:10.1530/endoabs.56.oc3.3'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The paper mentions the use of 'high-content screening' which is a technique often associated with fluorescence microscopy, especially when studying dynamic processes such as cellular protrusions in chemotaxis."},
   'paper_id': 'doi:10.1038/s41467-018-03571-8'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': "The abstract mentions 'Proteomic analyses of EEV fractions', which implies the use of mass spectrometry to analyze protein content in the extracellular vesicles."},
   'paper_id': 'doi:10.1083/jcb.201612051'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': "The abstract does not provide explicit details about imaging technology; however, it mentions the targeting of PorB to mitochondria, which often involves using molecular probes or stains to visualize such processes, thus 'Molecular imaging probes, stains, antibodies' seems to be the most likely category."},
   'paper_id': 'doi:10.1371/journal.ppat.1006945'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The abstract mentions cell tracking using MRI, which stands for Magnetic Resonance Imaging, indicating that this is the imaging technology primarily used in the paper.'},
   'paper_id': 'doi:10.1016/j.ymthe.2018.03.017'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not provide specific information about imaging technology being the primary focus. Instead, it discusses genetic engineering of cells, which suggests a focus on molecular biology techniques rather than a specific imaging technology. The mention of CRISPR/Cas9 system for genome editing implies computational analysis for gene editing rather than imaging technology.'},
   'paper_id': 'doi:10.1002/eji.201747358'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': "The paper primarily uses fluorine-18-sodium fluoride (18F-NaF) positron emission tomography (PET) and computed tomography (CT), which are forms of molecular imaging and not exclusively MRI, but since the paper mentions a trial related to MRI for predicting AAA rupture or surgery (MA3RS Trial), it could be inferred that MRI might also be a contextual technology in the study. However, the primary focus on PET-CT in the abstract suggests that 'O' for Molecular imaging probes, stains, antibodies would be a more accurate classification."},
   'paper_id': 'doi:10.1016/j.jacc.2017.11.053'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper uses live-cell lattice light-sheet microscopy, which is a type of fluorescence microscopy, to examine the mitochondrial network.'},
   'paper_id': 'doi:10.1126/science.aao6047'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions 'Image analysis of immunofluorescence colocalization' which suggests the use of fluorescence microscopy, a technique commonly used to visualize the localization of proteins within cells."},
   'paper_id': 'doi:10.1016/j.ajpath.2017.12.004'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily uses polarized total internal reflection fluorescence microscopy (pol-TIRF), which is a type of fluorescence microscopy.'},
   'paper_id': 'doi:10.1038/s41467-018-02818-8'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses the use of FUCCI (fluorescent ubiquitination-based cell cycle indicator) reporters, which are used in fluorescence microscopy to study cell cycle phase transitions.'},
   'paper_id': 'doi:10.1080/15384101.2018.1547001'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specifically mention the use of any imaging technology; instead, it focuses on biochemical and pharmacological analyses. The absence of explicit imaging technologies suggests the paper primarily relies on computational analysis of data collected through these methods.'},
   'paper_id': 'doi:10.1194/jlr.m080374'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper mentions the use of GFP (Green Fluorescent Protein) and fluorescence polarization microscopy, which are techniques associated with fluorescence microscopy.'},
   'paper_id': 'doi:10.1038/s41467-017-01848-y'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper primarily utilizes magnetic resonance imaging (MRI) to monitor brain lesions, as mentioned in the abstract.'},
   'paper_id': 'doi:10.1016/j.neuro.2017.11.009'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not specify the use of any particular imaging technology, therefore a classification cannot be confidently made from the provided context.'},
   'paper_id': 'doi:10.1007/s12079-017-0415-5'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': "The paper describes the use of polarization sensitive optical coherence tomography (PSOCT), which is an optical imaging method not specifically listed in the provided options, thus falling under 'Other imaging hardware technology'."},
   'paper_id': 'doi:10.1016/j.neuroimage.2017.10.012'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper uses immunohistochemical detection for monoaminergic neurons, α-synuclein, astrocytes, and microglia, which involves molecular imaging probes, stains, or antibodies to visualize specific proteins or cell types.'},
   'paper_id': 'doi:10.1016/j.brs.2017.08.008'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions the use of 'live cell imaging,' which is a process often associated with fluorescence microscopy techniques. While the specific type of fluorescence microscopy is not stated, it falls under the broad category of 'Other fluorescence microscopy'."},
   'paper_id': 'doi:10.1007/s00018-017-2625-7'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper discusses phototoxicity in the context of live fluorescence microscopy, which suggests that the primary imaging technology used is fluorescence microscopy.'},
   'paper_id': 'doi:10.1002/bies.201700003'},
  {'classification': {'imaging_technology_code': 'K',
    'imaging_technology_name': 'Magnetic resonance imaging',
    'explanation': 'The paper discusses the estimation of white matter fiber parameters from diffusion MRI data, which indicates that the primary imaging technology used is magnetic resonance imaging.'},
   'paper_id': 'doi:10.1016/j.neuroimage.2017.06.052'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': "The abstract does not provide specific details about the imaging technology used. However, it mentions signaling pathways and receptor interactions, which suggests that computational analysis methods may have been applied to study these complex biological processes. Since there is no explicit mention of imaging technology, 'Other computational analysis' seems like the most appropriate category, albeit with a low confidence due to the lack of specific information."},
   'paper_id': 'doi:10.1089/ars.2016.6966'},
  {'classification': {'imaging_technology_code': 'P',
    'imaging_technology_name': 'Other computational analysis',
    'explanation': 'The abstract does not explicitly mention the use of any imaging technology. The focus is on signaling pathways and molecular mechanisms, which suggests that computational analysis methods may have been used to analyze the data. However, without specific mention of imaging technology, this categorization remains speculative.'},
   'paper_id': 'doi:10.1074/jbc.m116.752469'},
  {'classification': {'imaging_technology_code': 'O',
    'imaging_technology_name': 'Molecular imaging probes, stains, antibodies',
    'explanation': 'The paper seems to focus on the molecular mechanisms involved in meiosis and does not explicitly mention a primary imaging technology. Given the molecular nature of the study, it is likely that some form of molecular imaging involving probes, stains, or antibodies was used to visualize the synaptonemal complex, although this is not directly stated in the provided text.'},
   'paper_id': 'doi:10.7554/elife.23437'},
  {'classification': {'imaging_technology_code': 'F',
    'imaging_technology_name': 'Super-resolution',
    'explanation': 'The paper discusses practical challenges and considerations in the application of superresolution microscopy, as indicated by the title and background information within the abstract.'},
   'paper_id': 'doi:10.1083/jcb.201610011'},
  {'classification': {'imaging_technology_code': 'E',
    'imaging_technology_name': 'Nonlinear microscopy',
    'explanation': "The paper mentions the use of 'second harmonic generation microscopy,' which is a type of nonlinear microscopy, to reveal hepatic fibrosis."},
   'paper_id': 'doi:10.1371/journal.pone.0163342'},
  {'classification': {'imaging_technology_code': 'L',
    'imaging_technology_name': 'Mass-Spectrometry',
    'explanation': "The abstract does not explicitly mention the use of a specific imaging technology. However, the study's focus on ATP/ADP turnover and mitochondrial function suggests that mass spectrometry might be used to measure metabolites, as it is a common technique in metabolic studies. Without more specific information, this is an educated guess based on the context of the work described."},
   'paper_id': 'doi:10.1074/jbc.m116.734814'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The paper discusses the performance characteristics of charge-coupled device (CCD) and scientific complementary metal oxide semiconductor (sCMOS) cameras, which are digital detectors used in various microscopy applications. This indicates that the paper is primarily concerned with imaging hardware technology.'},
   'paper_id': 'doi:10.1016/b978-0-12-420138-5.00003-3'},
  {'classification': {'imaging_technology_code': 'M',
    'imaging_technology_name': 'Other imaging hardware technology',
    'explanation': 'The title and abstract mention microscope care and maintenance without specific reference to a type of microscopy, suggesting the paper is about general optical hardware used for generating image data.'},
   'paper_id': 'doi:10.1016/b978-0-12-420138-5.00004-5'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The abstract mentions quantitative fluorescence microscopy, which fits within the 'Other fluorescence microscopy' category, as it is a broad term that encompasses various fluorescence microscopy techniques not specified elsewhere."},
   'paper_id': 'doi:10.1016/b978-0-12-420138-5.00001-x'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The abstract discusses live-cell imaging with fluorescent protein fusions, which is a technique primarily associated with fluorescence microscopy.'},
   'paper_id': 'doi:10.1016/b978-0-12-407761-4.00006-3'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': 'The paper primarily discusses a digital imaging system that includes an epifluorescent microscope, indicating the use of fluorescence microscopy as the core imaging technology.'},
   'paper_id': 'doi:10.1016/b978-0-12-407761-4.00009-9'},
  {'classification': {'imaging_technology_code': 'G',
    'imaging_technology_name': 'Other fluorescence microscopy',
    'explanation': "The title of the paper mentions 'fluorescence imaging of living cells' and references 'CCD cameras', which are commonly used in various types of fluorescence microscopy."},
   'paper_id': 'doi:10.1101/pdb.top113'}],
 'run_name': "TitleAbstractClassifier_OneDocAtATime_Tool__top_level_imaging_categories__0__ChatOpenAI__client=&lt;openai.resources.chat.completions.Completions object&gt; async_client=&lt;openai.resources.chat.completions.AsyncCompletions object&gt; model_name='gpt-4-1106-preview' openai_api_key=SecretStr('**********') openai_proxy=''"}</code></pre>
</div>
</div>
<div id="cell-33" class="cell">
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a>pd.set_option(<span class="st">'display.max_colwidth'</span>, <span class="va">None</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div id="cell-34" class="cell">
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>collection_id  <span class="op">=</span> <span class="st">'0'</span></span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>classification_type <span class="op">=</span> <span class="st">'top_level_imaging_categories'</span></span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKE, N) <span class="op">\</span></span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>SKE_HR.ScientificKnowledgeExpression_id) <span class="op">\</span></span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.ScientificKnowledgeExpression_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.has_notes_id<span class="op">==</span>N.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>collection_id) <span class="op">\</span></span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">type</span><span class="op">==</span><span class="st">'TiAbClassificationNote__'</span><span class="op">+</span>classification_type) </span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> e,n <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a>    c <span class="op">=</span> json.loads(n.content)</span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a>    doi_link <span class="op">=</span> <span class="st">'https://doi.org/'</span><span class="op">+</span>e.<span class="bu">id</span>[<span class="dv">4</span>:]</span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a>    l.append((doi_link, e.<span class="bu">type</span>, e.content, n.content))</span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(l, columns<span class="op">=</span>[<span class="st">'doi'</span>, <span class="st">'paper_type'</span>, <span class="st">'citation'</span>, <span class="st">'json'</span>]) </span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a><span class="co">#df.to_csv(loc+db_name+'/imaging_cohort_methods.tsv', sep='\t', index=False)  </span></span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>


<table class="dataframe caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th">paper_type</th>
<th data-quarto-table-cell-role="th">citation</th>
<th data-quarto-table-cell-role="th">json</th>
</tr>
</thead>
<tbody>
</tbody>
</table>

</div>
</div>
</div>
<div id="cell-35" class="cell">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a>collection_id  <span class="op">=</span> <span class="st">'0'</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a>classification_type <span class="op">=</span> <span class="st">'top_level_imaging_categories'</span></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKE, N) <span class="op">\</span></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>SKE_HR.ScientificKnowledgeExpression_id) <span class="op">\</span></span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.ScientificKnowledgeExpression_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.has_notes_id<span class="op">==</span>N.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>collection_id) <span class="op">\</span></span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">type</span><span class="op">==</span><span class="st">'TiAbClassificationNote__'</span><span class="op">+</span>classification_type) </span>
<span id="cb38-13"><a href="#cb38-13" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb38-14"><a href="#cb38-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> e,n <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb38-15"><a href="#cb38-15" aria-hidden="true" tabindex="-1"></a>    c <span class="op">=</span> json.loads(n.content)</span>
<span id="cb38-16"><a href="#cb38-16" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> <span class="bu">isinstance</span>(c, <span class="bu">list</span>):</span>
<span id="cb38-17"><a href="#cb38-17" aria-hidden="true" tabindex="-1"></a>        c <span class="op">=</span> c[<span class="dv">0</span>]</span>
<span id="cb38-18"><a href="#cb38-18" aria-hidden="true" tabindex="-1"></a>    doi_link <span class="op">=</span> <span class="st">'https://doi.org/'</span><span class="op">+</span>e.<span class="bu">id</span>[<span class="dv">4</span>:]</span>
<span id="cb38-19"><a href="#cb38-19" aria-hidden="true" tabindex="-1"></a>    l.append((doi_link, e.<span class="bu">type</span>, e.content, c.get(<span class="st">'imaging_technology_code'</span>), c.get(<span class="st">'imaging_technology_name'</span>),c.get(<span class="st">'explanation'</span>) ))</span>
<span id="cb38-20"><a href="#cb38-20" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(l, columns<span class="op">=</span>[<span class="st">'doi'</span>, <span class="st">'paper_type'</span>, <span class="st">'citation'</span>, <span class="st">'code'</span>, <span class="st">'name'</span>, <span class="st">'explanation'</span>]) </span>
<span id="cb38-21"><a href="#cb38-21" aria-hidden="true" tabindex="-1"></a>df.to_csv(loc<span class="op">+</span>db_name<span class="op">+</span><span class="st">'/imaging_cohort_categories.tsv'</span>, sep<span class="op">=</span><span class="st">'</span><span class="ch">\t</span><span class="st">'</span>, index<span class="op">=</span><span class="va">False</span>)  </span>
<span id="cb38-22"><a href="#cb38-22" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>


<table class="dataframe caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th">paper_type</th>
<th data-quarto-table-cell-role="th">citation</th>
<th data-quarto-table-cell-role="th">code</th>
<th data-quarto-table-cell-role="th">name</th>
<th data-quarto-table-cell-role="th">explanation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>https://doi.org/10.1101/2023.10.25.23297129</td>
<td>ScientificPrimaryResearchPreprint</td>
<td>Dofash LN, Miles LB, Saito Y, Rivas E, Calcino...</td>
<td>P</td>
<td>Other computational analysis</td>
<td>The paper primarily discusses genetic analysis...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td>https://doi.org/10.1016/j.ceb.2023.102271</td>
<td>ScientificReviewArticle</td>
<td>Pylvänäinen JW, Gómez-de-Mariscal E, Henriques...</td>
<td>G</td>
<td>Other fluorescence microscopy</td>
<td>The abstract mentions the combination of live ...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td>https://doi.org/10.1002/advs.202303381</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Chang S, Yang J, Novoseltseva A, Abdelhakeem A...</td>
<td>E</td>
<td>Nonlinear microscopy</td>
<td>The paper primarily uses two-photon microscopy...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">3</td>
<td>https://doi.org/10.1038/s41592-023-02045-0</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Liu HF, Zhou Y, Huang Q, Piland J, Jin W, Mand...</td>
<td>H</td>
<td>Cryo-electron tomography</td>
<td>The title and abstract clearly mention 'single...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">4</td>
<td>https://doi.org/10.1101/2023.10.26.564098</td>
<td>ScientificPrimaryResearchPreprint</td>
<td>Woldeyes RA, Nishiga M, Vander Roest AS, Engel...</td>
<td>H</td>
<td>Cryo-electron tomography</td>
<td>The paper primarily uses cryo-electron tomogra...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">819</td>
<td>https://doi.org/10.1016/b978-0-12-420138-5.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Petrak LJ, Waters JC. (2014) A practical guide...</td>
<td>M</td>
<td>Other imaging hardware technology</td>
<td>The title and abstract mention microscope care...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">820</td>
<td>https://doi.org/10.1016/b978-0-12-420138-5.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Waters JC, Wittmann T. (2014) Concepts in quan...</td>
<td>G</td>
<td>Other fluorescence microscopy</td>
<td>The abstract mentions quantitative fluorescenc...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">821</td>
<td>https://doi.org/10.1016/b978-0-12-407761-4.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Waters JC. (2013) Live-cell fluorescence imaging.</td>
<td>G</td>
<td>Other fluorescence microscopy</td>
<td>The abstract discusses live-cell imaging with ...</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">822</td>
<td>https://doi.org/10.1016/b978-0-12-407761-4.000...</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Salmon ED, Shaw SL, Waters JC, Waterman-Storer...</td>
<td>G</td>
<td>Other fluorescence microscopy</td>
<td>The paper primarily discusses a digital imagin...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">823</td>
<td>https://doi.org/10.1101/pdb.top113</td>
<td>ScientificPrimaryResearchArticle</td>
<td>Salmon WC, Waters JC. (2011) CCD cameras for f...</td>
<td>G</td>
<td>Other fluorescence microscopy</td>
<td>The title of the paper mentions 'fluorescence ...</td>
</tr>
</tbody>
</table>

<p>824 rows × 6 columns</p>
</div>
</div>
</div>
<div id="cell-36" class="cell">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>collection_id  <span class="op">=</span> <span class="st">'0'</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>classification_type <span class="op">=</span> <span class="st">'top_level_imaging_categories'</span></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> ldb.session.query(SKE, N) <span class="op">\</span></span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>SKC_HM.ScientificKnowledgeCollection_id) <span class="op">\</span></span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC_HM.has_members_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>SKE_HR.ScientificKnowledgeExpression_id) <span class="op">\</span></span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.ScientificKnowledgeExpression_id<span class="op">==</span>SKE.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HN.has_notes_id<span class="op">==</span>N.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKE_HR.has_representation_id<span class="op">==</span>SKI.<span class="bu">id</span>) <span class="op">\</span></span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span>collection_id) <span class="op">\</span></span>
<span id="cb39-12"><a href="#cb39-12" aria-hidden="true" tabindex="-1"></a>        .<span class="bu">filter</span>(N.<span class="bu">type</span><span class="op">==</span><span class="st">'TiAbClassificationNote__'</span><span class="op">+</span>classification_type) </span>
<span id="cb39-13"><a href="#cb39-13" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb39-14"><a href="#cb39-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> e,n <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb39-15"><a href="#cb39-15" aria-hidden="true" tabindex="-1"></a>    ldb.delete_note(n.<span class="bu">id</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/chanzuckerberg\.github\.io\/alhazen\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/chanzuckerberg/alhazen/issues/new" class="toc-action"><i class="bi bi-github"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>