<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="description" content="Using LLMs to extract information from RNA studies in Zebrafish">

<title>alhazen - RNAquarium</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="../../styles.css">
<meta property="og:title" content="alhazen - RNAquarium">
<meta property="og:description" content="Using LLMs to extract information from RNA studies in Zebrafish">
<meta property="og:image" content="https://chanzuckerberg.github.io/alhazen/cookbook/single_doc_extraction/RNAquarium_files/figure-html/cell-24-output-2.png">
<meta property="og:site-name" content="alhazen">
<meta property="og:image:height" content="785">
<meta property="og:image:width" content="792">
<meta name="twitter:title" content="alhazen - RNAquarium">
<meta name="twitter:description" content="Using LLMs to extract information from RNA studies in Zebrafish">
<meta name="twitter:image" content="https://chanzuckerberg.github.io/alhazen/cookbook/single_doc_extraction/RNAquarium_files/figure-html/cell-24-output-2.png">
<meta name="twitter:image-height" content="785">
<meta name="twitter:image-width" content="792">
<meta name="twitter:card" content="summary_large_image">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">alhazen</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html#install-dependencies" rel="" target="">
 <span class="menu-text">Get Started</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../../docnb1_use_cases.html" rel="" target="" aria-current="page">
 <span class="menu-text">Use Cases</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../docnb2_architecture.html" rel="" target="">
 <span class="menu-text">Platform Architecture</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-help" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Help</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-help">    
        <li>
    <a class="dropdown-item" href="https://github.com/chanzuckerberg/alhazen/issues" rel="" target=""><i class="bi bi-bug" role="img">
</i> 
 <span class="dropdown-text">Report an Issue</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/chanzuckerberg/alhazen" rel="" target=""><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../cookbook/single_doc_extraction/index.html">Example Notebooks</a></li><li class="breadcrumb-item"><a href="../../cookbook/single_doc_extraction/index.html">Metadata Extraction</a></li><li class="breadcrumb-item"><a href="../../cookbook/single_doc_extraction/rnaquarium.html">RNAquarium</a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Home - Alhazen</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../docnb1_use_cases.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Use Cases</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../docnb2_architecture.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Platform Architecture</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Tech Elements</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../agents/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Agents</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../agents/agent.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Alhazen Agent / Chatbot.</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../tools/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/toolkit.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Alhazen Toolkit</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/basic_tools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Basic Tools for Alhazen</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/metadata_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Methods Metadata Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/protocol_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Protocol Workflow Extraction Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/paperqa_emulation_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Paper QA Emulation Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_clasifier_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_clasifier_tool copy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_mapping_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../tools/tiab_extraction_tool.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Document Classifier Tool</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../utils/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Utilities</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/ceifns_db.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Database for Scientific Knowledge</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/search_engine_eutils.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Search Engine Tools</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/web_robot.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Web Robots</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/html_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">HTML Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/jats_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">JATS Text Extractor Utility</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../utils/pdf_text_extractor.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PDF Text Extractor Utility</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" aria-expanded="true">
 <span class="menu-text">Example Notebooks</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/single_doc_extraction/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Metadata Extraction</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/cellxgene_datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CellXGene Datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/cryoet.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CryoET</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/ncats_natural_history_studies.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">NCATS Natural History Studies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/single_doc_extraction/rnaquarium.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">RNAquarium</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/trend_analysis/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Trend Analysis</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/capacity_building.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Capacity Building</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/imaging_technology_evolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Image Tech Evolution</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/pathogen_landscaping.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pathogen_Landscaping</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/trend_analysis/rare_disease_literature.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Rare Disease Literature.</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/key_opinion_leaders/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Key Opinion Leaders</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/key_opinion_leaders/african_microscopy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">African Microscopy</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../cookbook/question_answering/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Question Answering</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-9" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-9" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../cookbook/question_answering/virtualcell_corepapers.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Virtual Cell Landscaping Analysis</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#basics" id="toc-basics" class="nav-link active" data-scroll-target="#basics">Basics</a>
  <ul class="collapse">
  <li><a href="#python-imports" id="toc-python-imports" class="nav-link" data-scroll-target="#python-imports">Python Imports</a></li>
  <li><a href="#environment-variables" id="toc-environment-variables" class="nav-link" data-scroll-target="#environment-variables">Environment Variables</a></li>
  <li><a href="#setup-utils-agents-and-tools" id="toc-setup-utils-agents-and-tools" class="nav-link" data-scroll-target="#setup-utils-agents-and-tools">Setup utils, agents, and tools</a></li>
  </ul></li>
  <li><a href="#building-the-database" id="toc-building-the-database" class="nav-link" data-scroll-target="#building-the-database">Building the database</a>
  <ul class="collapse">
  <li><a href="#scripts-to-build-delete-the-database" id="toc-scripts-to-build-delete-the-database" class="nav-link" data-scroll-target="#scripts-to-build-delete-the-database">Scripts to Build / Delete the database</a></li>
  <li><a href="#build-ceifns-database-from-900-dois-in-database" id="toc-build-ceifns-database-from-900-dois-in-database" class="nav-link" data-scroll-target="#build-ceifns-database-from-900-dois-in-database">Build CEIFNS database from 900 dois in database</a></li>
  </ul></li>
  <li><a href="#analyze-collections" id="toc-analyze-collections" class="nav-link" data-scroll-target="#analyze-collections">Analyze Collections</a></li>
  <li><a href="#tests-checks" id="toc-tests-checks" class="nav-link" data-scroll-target="#tests-checks">Tests + Checks</a>
  <ul class="collapse">
  <li><a href="#agent-tool-selection-execution-interpretation" id="toc-agent-tool-selection-execution-interpretation" class="nav-link" data-scroll-target="#agent-tool-selection-execution-interpretation">Agent tool selection + execution + interpretation</a></li>
  </ul></li>
  <li><a href="#run-metadata-extraction-chain-over-listed-papers" id="toc-run-metadata-extraction-chain-over-listed-papers" class="nav-link" data-scroll-target="#run-metadata-extraction-chain-over-listed-papers">Run MetaData Extraction Chain over listed papers</a></li>
  <li><a href="#protocol-modeling-extraction" id="toc-protocol-modeling-extraction" class="nav-link" data-scroll-target="#protocol-modeling-extraction">Protocol Modeling + Extraction</a></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/chanzuckerberg/alhazen/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">RNAquarium</h1>
</div>

<div>
  <div class="description">
    Using LLMs to extract information from RNA studies in Zebrafish
  </div>
</div>


<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
<section id="basics" class="level2">
<h2 class="anchored" data-anchor-id="basics">Basics</h2>
<section id="python-imports" class="level3">
<h3 class="anchored" data-anchor-id="python-imports">Python Imports</h3>
<p>Setting python imports, environment variables, and other crucial set up parameters here.</p>
<div class="cell" data-execution_count="45">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.aliases <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.core <span class="im">import</span> get_langchain_chatmodel, MODEL_TYPE</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.agent <span class="im">import</span> AlhazenAgent</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.schema_sqla <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.core <span class="im">import</span> get_langchain_chatmodel, MODEL_TYPE</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.basic <span class="im">import</span> AddCollectionFromEPMCTool, DeleteCollectionTool</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.paperqa_emulation_tool <span class="im">import</span> PaperQAEmulationTool</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.metadata_extraction_tool <span class="im">import</span> <span class="op">*</span> </span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.tools.protocol_extraction_tool <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.toolkit <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.jats_text_extractor <span class="im">import</span> NxmlDoc</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.ceifns_db <span class="im">import</span> Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, list_databases</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.searchEngineUtils <span class="im">import</span> <span class="op">*</span></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.callbacks.tracers <span class="im">import</span> ConsoleCallbackHandler</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.docstore.document <span class="im">import</span> Document</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.text_splitter <span class="im">import</span> CharacterTextSplitter</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain.vectorstores.pgvector <span class="im">import</span> PGVector</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_community.chat_models.ollama <span class="im">import</span> ChatOllama</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_google_vertexai <span class="im">import</span> ChatVertexAI</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> langchain_openai <span class="im">import</span> ChatOpenAI</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> bs4 <span class="im">import</span> BeautifulSoup,Tag,Comment,NavigableString</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> databricks <span class="im">import</span> sql</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> datetime <span class="im">import</span> datetime</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> importlib_resources <span class="im">import</span> files</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> re</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> requests</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy <span class="im">import</span> create_engine, text, exists, func, or_, and_, not_, desc, asc</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> sqlalchemy.orm <span class="im">import</span> sessionmaker, aliased</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> time <span class="im">import</span> time,sleep</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> tqdm <span class="im">import</span> tqdm</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.request <span class="im">import</span> urlopen</span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.parse <span class="im">import</span> quote_plus, quote, unquote</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> urllib.error <span class="im">import</span> URLError, HTTPError</span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> yaml</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="environment-variables" class="level3">
<h3 class="anchored" data-anchor-id="environment-variables">Environment Variables</h3>
<p>Remember to set environmental variables for this code:</p>
<ul>
<li><code>ALHAZEN_DB_NAME</code> - the name of the PostGresQL database you are storing information into</li>
<li><code>LOCAL_FILE_PATH</code> - the location on disk where you save temporary files, downloaded models or other data.</li>
</ul>
<div class="cell" data-execution_count="2">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>] <span class="op">=</span> <span class="st">'rnaquarium'</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>os.environ[<span class="st">'LOCAL_FILE_PATH'</span>] <span class="op">=</span> <span class="st">'/users/gully.burns/alhazen/'</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="3">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.path.exists(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]) <span class="kw">is</span> <span class="va">False</span>:</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>    os.makedirs(os.environ[<span class="st">'LOCAL_FILE_PATH'</span>])</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.environ.get(<span class="st">'ALHAZEN_DB_NAME'</span>) <span class="kw">is</span> <span class="va">None</span>: </span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>    <span class="cf">raise</span> <span class="pp">Exception</span>(<span class="st">'Which database do you want to use for this application?'</span>)</span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>db_name <span class="op">=</span> os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>]</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> os.environ.get(<span class="st">'LOCAL_FILE_PATH'</span>) <span class="kw">is</span> <span class="va">None</span>: </span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>    <span class="cf">raise</span> <span class="pp">Exception</span>(<span class="st">'Where are you storing your local literature database?'</span>)</span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>loc <span class="op">=</span> os.environ[<span class="st">'LOCAL_FILE_PATH'</span>]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="setup-utils-agents-and-tools" class="level3">
<h3 class="anchored" data-anchor-id="setup-utils-agents-and-tools">Setup utils, agents, and tools</h3>
<div class="cell" data-execution_count="4">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>ldb <span class="op">=</span> Ceifns_LiteratureDb(loc<span class="op">=</span>loc, name<span class="op">=</span>db_name)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>llm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'mixtral:instruct'</span>) </span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>llm2 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-4-1106-preview'</span>) </span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>llm2 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-4-1106-preview'</span>) </span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a><span class="co">#llm3 = ChatVertexAI(model_name="gemini-pro", convert_system_message_to_human=True)</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>cb <span class="op">=</span> AlhazenAgent(llm2, llm2)</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'AGENT TOOLS'</span>)</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools():</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="st">'</span><span class="ch">\t</span><span class="st">'</span><span class="op">+</span><span class="bu">type</span>(t).<span class="va">__name__</span>)</span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>test_tk <span class="op">=</span> MetadataExtractionToolkit(db<span class="op">=</span>ldb, llm<span class="op">=</span>llm2)</span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'</span><span class="ch">\n</span><span class="st">TESTING TOOLS'</span>)</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> t <span class="kw">in</span> test_tk.get_tools():</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>    <span class="bu">print</span>(<span class="st">'</span><span class="ch">\t</span><span class="st">'</span><span class="op">+</span><span class="bu">type</span>(t).<span class="va">__name__</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>AGENT TOOLS
    AddCollectionFromEPMCTool
    AddAuthorsToCollectionTool
    DescribeCollectionCompositionTool
    DeleteCollectionTool
    RetrieveFullTextTool
    RetrieveFullTextToolForACollection
    MetadataExtraction_EverythingEverywhere_Tool
    SimpleExtractionWithRAGTool
    PaperQAEmulationTool
    ProcotolEntitiesExtractionTool
    CheckExpressionTool
    TitleAbstractClassifier_OneDocAtATime_Tool

TESTING TOOLS
    MetadataExtraction_EverythingEverywhere_Tool
    MetadataExtraction_MethodsSectionOnly_Tool
    MetadataExtraction_RAGOnSections_Tool
    SimpleExtractionWithRAGTool</code></pre>
</div>
</div>
</section>
</section>
<section id="building-the-database" class="level2">
<h2 class="anchored" data-anchor-id="building-the-database">Building the database</h2>
<section id="scripts-to-build-delete-the-database" class="level3">
<h3 class="anchored" data-anchor-id="scripts-to-build-delete-the-database">Scripts to Build / Delete the database</h3>
<p>If you need to restore a deleted database from backup, use the following shell commands:</p>
<pre><code>$ createdb em_tech
$ psql -d em_tech -f /local/file/path/em_tech/backup&lt;date_time&gt;.sql</code></pre>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>drop_ceifns_database(os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>create_ceifns_database(os.environ[<span class="st">'ALHAZEN_DB_NAME'</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="build-ceifns-database-from-900-dois-in-database" class="level3">
<h3 class="anchored" data-anchor-id="build-ceifns-database-from-900-dois-in-database">Build CEIFNS database from 900 dois in database</h3>
<p>Load data from the spreadsheet</p>
<div class="cell" data-execution_count="8">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> local_resources.data_files.rnaquarium <span class="im">as</span> rnaquarium</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.queryTranslator <span class="im">import</span> QueryTranslator, QueryType</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.read_csv(files(rnaquarium).joinpath(<span class="st">'RNAquarium_paper_list.tsv'</span>), sep<span class="op">=</span><span class="st">'</span><span class="ch">\t</span><span class="st">'</span>)</span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>dois <span class="op">=</span> df[<span class="st">'DOI'</span>].to_list()</span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="8">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">Key</th>
<th data-quarto-table-cell-role="th">Item Type</th>
<th data-quarto-table-cell-role="th">Publication Year</th>
<th data-quarto-table-cell-role="th">Author</th>
<th data-quarto-table-cell-role="th">Title</th>
<th data-quarto-table-cell-role="th">Publication Title</th>
<th data-quarto-table-cell-role="th">ISBN</th>
<th data-quarto-table-cell-role="th">ISSN</th>
<th data-quarto-table-cell-role="th">DOI</th>
<th data-quarto-table-cell-role="th">Url</th>
<th data-quarto-table-cell-role="th">...</th>
<th data-quarto-table-cell-role="th">Unnamed: 163</th>
<th data-quarto-table-cell-role="th">Unnamed: 164</th>
<th data-quarto-table-cell-role="th">Unnamed: 165</th>
<th data-quarto-table-cell-role="th">Unnamed: 166</th>
<th data-quarto-table-cell-role="th">Unnamed: 167</th>
<th data-quarto-table-cell-role="th">Unnamed: 168</th>
<th data-quarto-table-cell-role="th">Unnamed: 169</th>
<th data-quarto-table-cell-role="th">Unnamed: 170</th>
<th data-quarto-table-cell-role="th">Unnamed: 171</th>
<th data-quarto-table-cell-role="th">Unnamed: 172</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td>2FYDLCGI</td>
<td>journalArticle</td>
<td>2014</td>
<td>Han, Peidong; Zhou, Xiao-Hai; Chang, Nannan; X...</td>
<td>Hydrogen peroxide primes heart regeneration wi...</td>
<td>Cell Research</td>
<td>NaN</td>
<td>1748-7838</td>
<td>10.1038/cr.2014.108</td>
<td>NaN</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">1</td>
<td>8RZKQRRU</td>
<td>journalArticle</td>
<td>2014</td>
<td>Marín-Juez, Rubén; Jong-Raadsen, Susanne; Yang...</td>
<td>Hyperinsulinemia induces insulin resistance an...</td>
<td>The Journal of Endocrinology</td>
<td>NaN</td>
<td>1479-6805</td>
<td>10.1530/JOE-14-0178</td>
<td>NaN</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">2</td>
<td>CPWU52VI</td>
<td>journalArticle</td>
<td>2014</td>
<td>Benard, Erica L.; Roobol, Stefan J.; Spaink, H...</td>
<td>Phagocytosis of mycobacteria by zebrafish macr...</td>
<td>Developmental and Comparative Immunology</td>
<td>NaN</td>
<td>1879-0089</td>
<td>10.1016/j.dci.2014.07.022</td>
<td>NaN</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">3</td>
<td>E2XPRJPW</td>
<td>journalArticle</td>
<td>2014</td>
<td>Howarth, Deanna L.; Lindtner, Claudia; Vacaru,...</td>
<td>Activating transcription factor 6 is necessary...</td>
<td>PLoS genetics</td>
<td>NaN</td>
<td>1553-7404</td>
<td>10.1371/journal.pgen.1004335</td>
<td>NaN</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">4</td>
<td>RYI6EAFJ</td>
<td>journalArticle</td>
<td>2014</td>
<td>Desvignes, Thomas; Beam, Michael J.; Batzel, P...</td>
<td>Expanding the annotation of zebrafish microRNA...</td>
<td>Gene</td>
<td>NaN</td>
<td>1879-0038</td>
<td>10.1016/j.gene.2014.05.036</td>
<td>NaN</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">906</td>
<td>MNQQTVH8</td>
<td>journalArticle</td>
<td>2022</td>
<td>Zhang, Zhicong; Ji, Fengyu; Jiang, Shouwen; Wu...</td>
<td>Scale Development-Related Genes Identified by ...</td>
<td>Fishes</td>
<td>NaN</td>
<td>2410-3888</td>
<td>10.3390/fishes7020064</td>
<td>https://www.mdpi.com/2410-3888/7/2/64</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">907</td>
<td>E9E2BJC8</td>
<td>journalArticle</td>
<td>2022</td>
<td>Hu, Ming-Liang; Wu, Bao-Sheng; Xu, Wen-Jie; Zh...</td>
<td>Gene expression responses in zebrafish to shor...</td>
<td>Zoological Research</td>
<td>NaN</td>
<td>2095-8137</td>
<td>10.24272/j.issn.2095-8137.2021.352</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">908</td>
<td>ME3REM8F</td>
<td>journalArticle</td>
<td>2022</td>
<td>Herrera-Rivero, Marisol; Gandhi, Shrey; Witten...</td>
<td>Cardiac chamber-specific genetic alterations s...</td>
<td>Genomics</td>
<td>NaN</td>
<td>8887543</td>
<td>10.1016/j.ygeno.2022.110320</td>
<td>https://linkinghub.elsevier.com/retrieve/pii/S...</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">909</td>
<td>ICZFTJFQ</td>
<td>journalArticle</td>
<td>2021</td>
<td>Gandhi, Shrey; Witten, Anika; De Majo, Federic...</td>
<td>Evolutionarily conserved transcriptional lands...</td>
<td>Genomics</td>
<td>NaN</td>
<td>8887543</td>
<td>10.1016/j.ygeno.2021.09.002</td>
<td>https://linkinghub.elsevier.com/retrieve/pii/S...</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">910</td>
<td>TF38GRRU</td>
<td>journalArticle</td>
<td>2020</td>
<td>Spreafico, Marco; Mangano, Eleonora; Mazzola, ...</td>
<td>The Genome-Wide Impact of Nipblb Loss-of-Funct...</td>
<td>International Journal of Molecular Sciences</td>
<td>NaN</td>
<td>1422-0067</td>
<td>10.3390/ijms21249719</td>
<td>https://www.mdpi.com/1422-0067/21/24/9719</td>
<td>...</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
</tr>
</tbody>
</table>

<p>911 rows × 173 columns</p>
</div>
</div>
</div>
<p>Run this cell to execute paged queries (length 40) over the European PMC for each of the DOIs mentioned in the spreadsheet loaded above.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>addEMPCCollection_tool <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> cb.tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, AddCollectionFromEPMCTool)][<span class="dv">0</span>]</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>step <span class="op">=</span> <span class="dv">40</span></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> start_i <span class="kw">in</span> <span class="bu">range</span>(<span class="dv">0</span>, <span class="bu">len</span>(dois), step):</span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>    query <span class="op">=</span> <span class="st">' OR '</span>.join([<span class="st">'doi:</span><span class="ch">\"</span><span class="st">'</span><span class="op">+</span>dois[i]<span class="op">+</span><span class="st">'</span><span class="ch">\"</span><span class="st">'</span> <span class="cf">for</span> i <span class="kw">in</span> <span class="bu">range</span>(start_i, start_i<span class="op">+</span>step) <span class="cf">if</span> i <span class="op">&lt;</span> <span class="bu">len</span>(dois)])</span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>    addEMPCCollection_tool.run({<span class="st">'id'</span>: <span class="st">'0'</span>, <span class="st">'name'</span>:<span class="st">'RNAquarium Papers'</span>, <span class="st">'query'</span>:query, <span class="st">'full_text'</span>:<span class="va">True</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Run this cell to check how many papers from the list are loaded in our database.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Compare contents of database to the list of dois</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>missing_list <span class="op">=</span> []</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>titles <span class="op">=</span> []</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> doi <span class="kw">in</span> dois:</span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>    row <span class="op">=</span> df[df[<span class="st">'DOI'</span>]<span class="op">==</span>doi]</span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>    doi_in_db <span class="op">=</span> ldb.session.query(SKE).<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span><span class="st">'doi:'</span><span class="op">+</span>doi.lower()).<span class="bu">all</span>()</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> <span class="bu">len</span>(doi_in_db) <span class="op">==</span> <span class="dv">0</span>:</span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>        <span class="bu">print</span>(<span class="st">'DOI: '</span><span class="op">+</span>doi)</span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>        <span class="bu">print</span>(<span class="st">'</span><span class="ch">\t</span><span class="sc">%s</span><span class="st"> (</span><span class="sc">%d</span><span class="st">) </span><span class="sc">%s</span><span class="st"> </span><span class="sc">%s</span><span class="st">'</span><span class="op">%</span>(row[<span class="st">'Author'</span>].iloc[<span class="dv">0</span>],row[<span class="st">'Publication Year'</span>].iloc[<span class="dv">0</span>],row[<span class="st">'Title'</span>].iloc[<span class="dv">0</span>],row[<span class="st">'Journal Abbreviation'</span>].iloc[<span class="dv">0</span>]))</span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>        missing_list.append(doi)</span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>        titles.append(row[<span class="st">'Title'</span>].iloc[<span class="dv">0</span>])</span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="st">'</span><span class="sc">%d</span><span class="st"> Missing DOIs'</span><span class="op">%</span>(<span class="bu">len</span>(missing_list)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>0 Missing DOIs</code></pre>
</div>
</div>
<p>Use OpenAlex as filler to add papers that were missed on EPMC</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> alhazen.utils.searchEngineUtils <span class="im">import</span> load_paper_from_openalex, read_references_from_openalex </span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pyalex <span class="im">import</span> config, Works, Work</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a>config.email <span class="op">=</span> <span class="st">"gully.burns@chanzuckerberg.com"</span></span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> requests</span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os</span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a>ldb.session.rollback()</span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a>corpus <span class="op">=</span> ldb.session.query(SKC).<span class="bu">filter</span>(SKC.<span class="bu">id</span><span class="op">==</span><span class="st">'0'</span>).first()</span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a>count <span class="op">=</span> <span class="dv">0</span></span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a><span class="bu">print</span>(<span class="bu">len</span>(corpus.has_members))</span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a>papers_to_index <span class="op">=</span> []</span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> i, doi <span class="kw">in</span> <span class="bu">enumerate</span>(missing_list):</span>
<span id="cb13-14"><a href="#cb13-14" aria-hidden="true" tabindex="-1"></a>    p <span class="op">=</span> load_paper_from_openalex(doi)</span>
<span id="cb13-15"><a href="#cb13-15" aria-hidden="true" tabindex="-1"></a>    ldb.session.add(p)</span>
<span id="cb13-16"><a href="#cb13-16" aria-hidden="true" tabindex="-1"></a>    corpus.has_members.append(p)</span>
<span id="cb13-17"><a href="#cb13-17" aria-hidden="true" tabindex="-1"></a>    p.member_of.append(corpus)</span>
<span id="cb13-18"><a href="#cb13-18" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> item <span class="kw">in</span> p.has_representation:</span>
<span id="cb13-19"><a href="#cb13-19" aria-hidden="true" tabindex="-1"></a>        <span class="cf">for</span> f <span class="kw">in</span> item.has_part:</span>
<span id="cb13-20"><a href="#cb13-20" aria-hidden="true" tabindex="-1"></a>            <span class="co">#f.content = '\n'.join(self.sent_detector.tokenize(f.content))</span></span>
<span id="cb13-21"><a href="#cb13-21" aria-hidden="true" tabindex="-1"></a>            f.part_of <span class="op">=</span> item.<span class="bu">id</span></span>
<span id="cb13-22"><a href="#cb13-22" aria-hidden="true" tabindex="-1"></a>            ldb.session.add(f)</span>
<span id="cb13-23"><a href="#cb13-23" aria-hidden="true" tabindex="-1"></a>        item.represented_by <span class="op">=</span> p.<span class="bu">id</span></span>
<span id="cb13-24"><a href="#cb13-24" aria-hidden="true" tabindex="-1"></a>        ldb.session.add(item)</span>
<span id="cb13-25"><a href="#cb13-25" aria-hidden="true" tabindex="-1"></a>    papers_to_index.append(p)</span>
<span id="cb13-26"><a href="#cb13-26" aria-hidden="true" tabindex="-1"></a>    ldb.session.flush()</span>
<span id="cb13-27"><a href="#cb13-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-28"><a href="#cb13-28" aria-hidden="true" tabindex="-1"></a>ldb.embed_expression_list(papers_to_index)</span>
<span id="cb13-29"><a href="#cb13-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-30"><a href="#cb13-30" aria-hidden="true" tabindex="-1"></a>ldb.session.commit()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="get-full-text-copies-of-all-the-papers-about-cryoet" class="level4">
<h4 class="anchored" data-anchor-id="get-full-text-copies-of-all-the-papers-about-cryoet">Get full text copies of all the papers about CryoET</h4>
<p>This invokes the agent directly to make it easy to run the retrieval tool.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>cb.db.session.rollback()</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>cb.agent_executor.invoke({<span class="st">'input'</span>:<span class="st">'Retrieve full text for the collection with id="0".'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
</section>
<section id="analyze-collections" class="level2">
<h2 class="anchored" data-anchor-id="analyze-collections">Analyze Collections</h2>
<p>Build a basic report of the composition over all collections in the database (listed by types of items).</p>
<div class="cell" data-execution_count="5">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>cb.db.report_collection_composition()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="5">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">item type</th>
<th data-quarto-table-cell-role="th">CitationRecord</th>
<th data-quarto-table-cell-role="th">HTMLFullText</th>
<th data-quarto-table-cell-role="th">JATSFullText</th>
<th data-quarto-table-cell-role="th">PDFFullText</th>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">id</th>
<th data-quarto-table-cell-role="th">collection name</th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">0</td>
<td data-quarto-table-cell-role="th">RNAquarium Papers</td>
<td>945</td>
<td>1</td>
<td>610</td>
<td>666</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="cell" data-execution_count="5">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>cb.db.report_non_full_text_for_collection(<span class="dv">0</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="5">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th">item type</th>
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th">CitationRecord</th>
<th data-quarto-table-cell-role="th">HTMLFullText</th>
<th data-quarto-table-cell-role="th">JATSFullText</th>
<th data-quarto-table-cell-role="th">PDFFullText</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">6</td>
<td>doi:10.1002/dvdy.24188</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">7</td>
<td>doi:10.1002/dvdy.24600</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">13</td>
<td>doi:10.1002/glia.24075</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">26</td>
<td>doi:10.1007/s00424-017-2009-8</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">31</td>
<td>doi:10.1007/s11427-020-1878-8</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
<td>...</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">736</td>
<td>doi:10.1261/rna.029090.111</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">792</td>
<td>doi:10.15252/embj.2022112590</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">801</td>
<td>doi:10.1534/genetics.119.302365</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">822</td>
<td>doi:10.3389/fcell.2021.699796</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">840</td>
<td>doi:10.3389/fimmu.2019.02720</td>
<td>1.0</td>
<td>0.0</td>
<td>0.0</td>
<td>0.0</td>
</tr>
</tbody>
</table>

<p>68 rows × 5 columns</p>
</div>
</div>
</div>
</section>
<section id="tests-checks" class="level2">
<h2 class="anchored" data-anchor-id="tests-checks">Tests + Checks</h2>
<section id="agent-tool-selection-execution-interpretation" class="level3">
<h3 class="anchored" data-anchor-id="agent-tool-selection-execution-interpretation">Agent tool selection + execution + interpretation</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># use this cell to test the agent's </span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>cb.agent_executor.invoke({<span class="st">'input'</span>:<span class="st">'Hi who are you and what can you do?'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="run-metadata-extraction-chain-over-listed-papers" class="level2">
<h2 class="anchored" data-anchor-id="run-metadata-extraction-chain-over-listed-papers">Run MetaData Extraction Chain over listed papers</h2>
<p>Here, we run various versions of the metadata extraction tool to examine performance over the cryoet dataset.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> local_resources.data_files.cryoet_portal_metadata <span class="im">as</span> cryoet_portal_metadata</span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="bu">str</span>(files(cryoet_portal_metadata).joinpath(<span class="st">'temp'</span>))[<span class="dv">0</span>:<span class="op">-</span><span class="dv">4</span>]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="40">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> local_resources.data_files.rnaquarium <span class="im">as</span> rnaquarium</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Get the metadata extraction tool</span></span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>t2 <span class="op">=</span> [t <span class="cf">for</span> t <span class="kw">in</span> test_tk.get_tools() <span class="cf">if</span> <span class="bu">isinstance</span>(t, MetadataExtraction_EverythingEverywhere_Tool)][<span class="dv">0</span>]</span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Hack to get the path to the metadata directory as a string</span></span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>metadata_dir <span class="op">=</span> <span class="bu">str</span>(files(rnaquarium).joinpath(<span class="st">'temp'</span>))[<span class="dv">0</span>:<span class="op">-</span><span class="dv">4</span>]</span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Compile the answers from the metadata directory</span></span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a><span class="co">#t2.compile_answers('cryoet', metadata_dir)</span></span>
<span id="cb19-11"><a href="#cb19-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-12"><a href="#cb19-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a dataframe to store previously extracted metadata</span></span>
<span id="cb19-13"><a href="#cb19-13" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame()</span>
<span id="cb19-14"><a href="#cb19-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d_id <span class="kw">in</span> dois:</span>
<span id="cb19-15"><a href="#cb19-15" aria-hidden="true" tabindex="-1"></a>    item_types <span class="op">=</span> <span class="bu">set</span>()</span>
<span id="cb19-16"><a href="#cb19-16" aria-hidden="true" tabindex="-1"></a>    <span class="co">#d_id = 'doi:'+d</span></span>
<span id="cb19-17"><a href="#cb19-17" aria-hidden="true" tabindex="-1"></a>    df2 <span class="op">=</span> pd.DataFrame(t2.read_metadata_extraction_notes(d_id, <span class="st">'rnaquarium'</span>)) </span>
<span id="cb19-18"><a href="#cb19-18" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, df2]) </span>
<span id="cb19-19"><a href="#cb19-19" aria-hidden="true" tabindex="-1"></a>     </span>
<span id="cb19-20"><a href="#cb19-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Iterate over papers to run the metadata extraction tool</span></span>
<span id="cb19-21"><a href="#cb19-21" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d_id <span class="kw">in</span> dois[<span class="dv">0</span>:<span class="dv">10</span>]:</span>
<span id="cb19-22"><a href="#cb19-22" aria-hidden="true" tabindex="-1"></a>    item_types <span class="op">=</span> <span class="bu">set</span>()</span>
<span id="cb19-23"><a href="#cb19-23" aria-hidden="true" tabindex="-1"></a>    <span class="co">#d_id = 'doi:'+d</span></span>
<span id="cb19-24"><a href="#cb19-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-25"><a href="#cb19-25" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Skip if the doi is already in the database</span></span>
<span id="cb19-26"><a href="#cb19-26" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> <span class="bu">len</span>(df)<span class="op">&gt;</span><span class="dv">0</span> <span class="kw">and</span> d_id <span class="kw">in</span> df.doi.unique():</span>
<span id="cb19-27"><a href="#cb19-27" aria-hidden="true" tabindex="-1"></a>        <span class="cf">continue</span></span>
<span id="cb19-28"><a href="#cb19-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-29"><a href="#cb19-29" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Run the metadata extraction tool on the doi</span></span>
<span id="cb19-30"><a href="#cb19-30" aria-hidden="true" tabindex="-1"></a>    t2.run(tool_input<span class="op">=</span>{<span class="st">'paper_id'</span>: d_id, <span class="st">'extraction_type'</span>: <span class="st">'rnaquarium'</span>})</span>
<span id="cb19-31"><a href="#cb19-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-32"><a href="#cb19-32" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Add the results to the dataframe</span></span>
<span id="cb19-33"><a href="#cb19-33" aria-hidden="true" tabindex="-1"></a>    df2 <span class="op">=</span> pd.DataFrame(t2.read_metadata_extraction_notes(d_id, <span class="st">'rnaquarium'</span>)) </span>
<span id="cb19-34"><a href="#cb19-34" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, df2])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "ORIGINAL ARTICLE\n© 2014 IBCB, SIBS, CAS    All rights reserved 1001-0602/14  $ 32.00 www.nature.com/cr\n\n\nHydrogen peroxide primes heart regeneration with a derepression mechanism\nPeidong Han 1, * , Xiao-Hai Zhou 1, * , Nannan Chang 1, * , Cheng-Lu Xiao 1 , Shouyu Yan 1 , He Ren 2, 3 , Xin-Zhuang Yang 1 , Mei-Ling Zhang 1 , Qing Wu 1 , Boyang Tang 1, 2 , Ju-Peng Diao 1 , Xiaojun Zhu 1 ,\nChuanmao Zhang 2, 3 , Chuan-Yun Li 1 , Heping Cheng 1, 2, 4 , Jing-Wei Xiong 1\n1 Institute of Molecular Medicine, Beijing Key Laboratory of Cardiometabolic Molecular Medicine and State Key Laboratory of Natural and Biomimetic Drugs, 2 State Key Laboratory of Biomembrane and Membrane Biotechnology, 3 College of Life Sciences, 4 Peking-Tsinghua Center for Life Sciences, Peking University, Beijing 100871, China\nWhile the adult human heart has very limited regenerative potential, the adult zebrafish heart can fully regenerate after 20% ventricular resection. Although previous reports suggest that developmental signaling pathways such as FGF and PDGF are reused in adult heart regeneration, the underlying intracellular mechanisms remain largely un - known. Here we show that H 2 O 2 acts as a novel epicardial and myocardial signal to prime the heart for regeneration in adult zebrafish. Live imaging of intact hearts revealed highly localized H 2 O 2 (~30 µM) production in the epicardi - um and adjacent compact myocardium at the resection site. Decreasing H 2 O 2 formation with the Duox inhibitors di - phenyleneiodonium (DPI) or apocynin, or scavenging H 2 O 2 by catalase overexpression markedly impaired cardiac re - generation while exogenous H 2 O 2 rescued the inhibitory effects of DPI on cardiac regeneration, indicating that H 2 O 2 is an essential and sufficient signal in this process. Mechanistically, elevated H 2 O 2 destabilized the redox-sensitive phosphatase Dusp6 and hence increased the phosphorylation of Erk1/2. The Dusp6 inhibitor BCI achieved similar pro-regenerative effects while transgenic overexpression of dusp6 impaired cardiac regeneration. H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration through two relatively independent pathways. We conclude that H 2 O 2 potentially generated from Duox/Nox2 promotes heart regeneration in zebrafish by unleashing MAP kinase signaling through a derepression mechanism involving Dusp6. Keywords : heart regeneration; zebrafish; Duox; H 2 O 2 ; Dusp6 Cell Research (2014) 24 :1091-1107. doi:10.1038/cr.2014.108; published online 15 August 2014\n*These three authors contributed equally to this work. Correspondence: Jing-Wei Xiong a , Heping Cheng b a, b Tel/Fax: 86-10-62766239 a E-mail: jingwei_xiong@pku.edu.cn b E-mail: chengp@pku.edu.cn\nReceived 11 February 2014; revised 6 May 2014; accepted 9 June 2014; published online 15 August 2014\n\n\nIntroduction\nThe ability to reverse heart failure, which accounts for &gt; 10% annual mortality in humans, is the most challeng - ing task in cardiac regenerative medicine. Many efforts are devoted to cardiac stem cell generation and tissue engineering, although making cardiac stem cells or tissue scaffolds functionally integrated into their host cardiac tissues remains difficult in clinical settings [1, 2]. A num - ber of studies have shown that the adult mammalian heart\nhas limited regenerative capacity, although not sufficient to fully restore heart function after substantial myocar - dial injury [2]. On the other hand, the adult zebrafish and newt hearts have full capacity of cardiac regeneration af - ter ventricular resection or cryoinjury, making zebrafish and newt as powerful animal models for studying heart regeneration in particular and organ regeneration in gen- eral [3-6]. The neonatal mouse heart in the first week of their postnatal life can fully regenerate after ventricular resection, suggesting that mammals gradually lose their regenerative potential during postnatal development [10]. It remains mysterious how a species gains or loses the regenerative potential during evolution.\nZebrafish has become a well-recognized model sys - tem for studying organ regeneration due to incredible regenerative potential in all of its adult organs tested and the availability of transgenesis, genetics and genomic\nresources [7]. The dedifferentiation model for the cellu - lar sources of cardiac repair/regeneration was supported by the fact that cardiac muscles mainly regenerate from pre-existing cardiomyocytes after injury in zebrafish [8, 9] and mammals [10-12]. The stem cell/progenitor cell model for cardiac regeneration in mammals was also recognized by the existence of Sca1 + and c-Kit + cardiac stem cells in mice [13, 14], while the transdifferentiation or lineage reprogramming model was proposed partly based on the ability of cardiac fibroblast transdifferenti - ation into myocytes upon induction by reprogramming factors or microRNA in mice [15-17] and reprogram - ming of embryonic atrial into ventricular myocytes after injury in zebrafish [18]. In spite of great efforts, the mo - lecular mechanisms underlying heart regeneration remain incompletely understood. Previous reports support that developmental signaling pathways such as FGF, PDGF, retinoid acid, TGFβ and integrin are essential for adult zebrafish heart regeneration [19-23]. Ventricular resec - tion induces expression of fgfr2/4 in the epicardium and their ligand fgf17b in the myocardium, and conditional blocking of the FGF signaling by overexpression of dominant-negative FGFR1 substantially increases fibrin/ collagen deposits and diminishes myocardial regenera - tion, supporting a notion that the FGF signaling promotes myocardial regeneration through epicardial-myocardial interaction [21]. A similar strategy revealed that the PDGF signaling is also required for zebrafish heart re - generation [20]. The exact molecular mechanisms down - stream of the FGF/PDGF signaling during adult heart regeneration are currently unknown.\nDuox is a member of NADPH-oxidase (Nox) and related dual oxidase (Duox) family, which has seven members, Nox1, Nox2, Nox3, Nox4, Nox5, Duox1 and Duox2 in humans, but only five (Nox1, Nox2, Nox4, Nox5 and Duox) identified in zebrafish [24]. Function - ally, the Duox or Nox enzyme complex in the plasma membrane catalyzes single-electron transfers from the electron donor NADPH to molecular oxygen, forming superoxide anions, which are quickly converted into hy - drogen peroxide (H 2 O 2 ) by spontaneous dismutation or catalytic reactions via Cu 2+ Zn 2+ -superoxide dismutase. Recently, it has been shown that Duox-generated H 2 O 2 in the tail-fin epithelium, which responds within ~3 min and peaks around 20 min after wounding, is critical for the rapid recruitment of leukocytes to initiate inflam - mation in zebrafish larvae [25, 26]. Sustained H 2 O 2 pro- duction would shift the intracellular redox homeostasis from a more reduced to a more oxidized state, affecting numerous redox-sensitive signaling pathways. Atax - ia-telangiectasia mutated (ATM), a PI3K-like serine/ threonine protein kinase preferentially activated by DNA\ndouble-strand breaks, serves as a sensor of oxidative stress (H 2 O 2 ). Oxidative stress (H 2 O 2 ) elicits the ATM-ho - modimer-Chk2 pathway to induce cell growth arrest or apoptosis [27]. On the other hand, H 2 O 2 acts through a redox sensor Lyn to mediate leukocyte wound attraction [25, 26]. Here we report that, after ventricular resection, H 2 O 2 levels are locally elevated in the epicardium and adjacent myocardium to promote heart regeneration by a derepression mechanism, whereby H 2 O 2 destabilizes the redox-sensitive phosphatase Dusp6 to enhance growth factor signaling.\n\n\nResults\nDuox and Nox2 are spatio-temporally associated with heart regeneration In a genome-wide search for genes critical for cardiac regeneration by mRNA deep sequencing (Supplemen - tary information, Figures S1 and S2), we found that the expression levels of a group of genes related to immune system diseases were upregulated, including p22 phox , p47 phox and p67 phox of the Nox2 complex (Supplementary information, Tables S1-S3; GEO repository accession number GSE50203). Combining with other public mi - croarray databases, we identified 37 genes in the reactive oxygen species (ROS) system that were differentially expressed in the injured zebrafish heart at 7 days post amputation (dpa) (Supplementary information, Table S4), when injury-induced inflammation started to fade and active myocardial regeneration began [3, 28, 29]. RNA in situ hybridization was then applied at multiple time points from 3 to 30 dpa for validation. We found that duox was particularly interesting because of the spatial and temporal patterns of its expression during regeneration. RNA in situ hybridization with two inde - pendent duox probes revealed that duox expression was weak at sham, peaked and was highly localized beneath the surface of the amputation at around 3, 7 and 14 dpa, and returned to the basal level at 30 dpa when regener - ation was nearly complete (Supplementary information, Figures S3A-S3E and S5A-S5E). Each of the two sense duox probes had no detectable signals (Supplementary information, Figures S4 and S5F), indicating the speci - ficity of the two duox probes. RNA in situ hybridization on Tg( tcf21 :nucEGFP) transgenic hearts revealed that duox expression largely overlapped with the EGFP signal (Figure 1A-1C and Supplementaryinformation, Figure S6), which primarily labels the epicardium [30], sug - gesting that duox was enriched in the epicardium. Thus, both the time course and epicardial localization of duox expression correlated with those of active myocardial re - generation after injury.\nSystematic examination revealed that the expression of nox2 , but none of the other nox genes, was moderately induced at 3, 7 and 14 dpa in a pattern similar to that of duox (Supplementary information, Figure S7). RT-PCR confirmed expression of duox and nox2 in 14 dpa hearts (Supplementary information, Figure S3F). These results indicate that the expression of duox and nox2 , with duox being dominant, is spatially and temporally associated with myocardial regeneration after ventricular resection.\nH 2 O 2 signal is essential for heart regeneration To visualize the spatiotemporal dynamics of H 2 O 2 in the intact heart during regeneration, we used Tol2-based transgenesis and zebrafish myosin light chain 7 ( myl7 ) promoter to generate transgenic zebrafish with cardi - ac-specific expression of Hyper (Figure 1D), a fluores - cent protein-based H 2 O 2 sensor [25, 31]. High-resolution confocal imaging was performed in intact adult hearts excised from Tg( myl7 :Hyper) transgenic animals, and H 2 O 2 levels were directly measured as the ratio between the fluorescent signals dually excited at 488 and 405 nm ( R = F 488 / F 405 ). By ex vivo titration on the Hyper-ex - pressing hearts freshly excised from uninjured animals, we obtained a calibration curve (Figure 1K and Supple - mentary information, Figure S9A-S9G) similar to that obtained in vitro [31]. Compared to sham controls (Figure 1E), local H 2 O 2 concentration was clearly elevated in injured hearts at 3, 7 and 14 dpa (Figure 1F-1G and 1I), and returned to the basal level after nearly full regenera - tion at 30 dpa (Figure 1H and 1I). In contrast to the rapid (on the scale of minutes) inflammatory H 2 O 2 response in the tail-fin epithelium of zebrafish larvae [25, 26], there was a slow onset of ventricular resection-induced H 2 O 2 production, as no detectable increase in the Hyper fluorescence ratio occurred within ~1 h after heart injury (data not shown). As seen on confocal optical sections (Figure 1E-1H) and from line plots of the spatial profiles (Figure 1J), the H 2 O 2 gradients spanned ~20 µm depth from the epicardium to the myocardium, reaching a peak concentration of ~30 µM (Figure 1K and Supplementa - ry information, Figure S9A-S9G), which is within the range of concentrations for H 2 O 2 to act as an intracellular messenger [32]. In an independent measurement, we stained 14 dpa Tg( myl7 :Hyper) transgenic hearts with Redox sensor cc-1 and visualized the H 2 O 2 signal in both the epicardium that lacked Hyper expression (Fig - ure 1O) and adjacent compact myocardium (Figure 1L- 1O). Therefore, direct visualization of H 2 O 2 with either a genetically encoded probe or a small-molecule indicator revealed that H 2 O 2 was elevated primarily in the epicar - dium and the adjacent myocardium at the resection site, highly correlated with myocardial regeneration.\nTo investigate whether there is a causal relationship between elevated Duox/Nox2 and H 2 O 2 production and cardiac regeneration, we applied diphenyleneiodoni - um (DPI) (10 µM) or apocynin (100 µM), widely used inhibitors of Duox/Nox enzymes [33] and examined subsequent changes in H 2 O 2 production and cardiac re- generation. We found that Hyper fluorescent signals were decreased by 50% within ~ 40 min after soaking 14 dpa Tg( myl7 :Hyper) hearts in DPI-containing medium (Fig - ure 2A-2B), while the Hyper signals remained stable in control medium containing DMSO only (Figure 2B and Supplementary information, Figure S8A), suggesting that high concentration of H 2 O 2 likely resulted from elevated duox/nox2 expression but was not produced from mito - chondria. Functionality of ex vivo cultured hearts was validated by their rhythmic contractions under 1 Hz elec - trical field stimulation after 1 h culture (Supplementary information, Figure S8B-S8C). Based on the previous re - port that myocardial cells enter mitotic phases in injured areas of the heart upon ventricular resection in zebrafish or myocardial infarction in mice [3, 34], we quantified injury-induced myocyte proliferation by counting BrdU + / Mef2C + double-positive myocytes during heart regener - ation. After ventricular resection, zebrafish were allowed to recover for 7 days and BrdU was then co-injected with either DPI or apocynin into the abdominal cavity once daily until the hearts were collected for examination. Compared with sham-operated controls, we found that BrdU + /Mef2C + myocyte proliferation was induced in in - jured hearts at 14 dpa (Figure 2C, 2C’ and 2E), but their numbers were significantly reduced by treatment with either DPI (Figure 2D, 2D’ and 2E) or apocynin (Figure 2E and data not shown). In addition, extensive cardiac fibrosis (Figure 2G) and reduced Myl7 + myocardial re- generation were evident in injured hearts with DPI (Figure 2I) or apocynin treatment (data not shown), compared with DMSO-treated hearts (Figure 2F and 2H). Although injection of H 2 O 2 did not induce myocyte proliferation in sham-operated animals (Supplementaryinformation, Fig - ure S10), it rescued the inhibitory effects of DPI on the numbers of BrdU + /Mef2C + myocytes and Gata4 + cells (Supplementary information, Figure S11). Importantly, DPI-treated hearts failed to regenerate even at 60 dpa (Supplementary information, Figure S12), suggesting a permanent arrest rather than a delay of regenerative pro - cess. In addition, exogenous H 2 O 2 partially rescued the inhibitory effect of DPI on embryonic heart regeneration (Supplementary information, Figure S13A and S13B) af - ter a large number of cardiomyocytes were damaged by incubating Tg( myl7 :CFP-NTR) transgenic embryos with metronidazole (Mtz), as Mtz can be converted into a tox - ic DNA crosslinking agent by nitroreductase (NTR) as\npreviously described [35]. We further validated the pene - tration of H 2 O 2 by imaging the 72 hours post fertilization (hpf) embryonic Tg( myl7 :Hyper) hearts and found that Hyper signals were increased upon elevated external H 2 O 2 concentration (Supplementary information, Figure\nS14). These results strongly support the hypothesis that H 2 O 2 is required and sufficient for cardiac regeneration. If H 2 O 2 is causally linked to cardiac regeneration as suggested by its inhibition upon treatment with DPI or apocynin, limiting H 2 O 2 signaling by augmenting an -\nFigure 1 Epicardial Duox-generated H 2 O 2 associated with myocardial regeneration. (A - C) After imaging of Tg( tcf21 :nucEG- FP), we performed RNA in situ hybridization with duox probes on 10- µ m cryosections of Tg( tcf21 :nucEGFP) hearts at sham, 7 and 14 dpa. Note the induced expression of duox and its colocalization with epicardial reporter Tg( tcf21 :nucEGFP) in in- jured areas (arrowheads) at 7 and 14 dpa (B , C) while little staining at sham (A) . Scale bar for A - C , 50 µ m. (D) Cardiac-spe- cific expression of Hyper in a zebrafish embryo. Hyper was restricted in the Tg( myl7 :Hyper) transgenic heart at 48 hpf. (E - H) Ex vivo Hyper heart images from sham control (E) , 7 dpa (F) , 14 dpa (G) and 30 dpa (H) . Spatially resolved H 2 O 2 image, in- dexed by the ratio between the F 488 and F 405 images of Hyper (left panels), is presented in pseudocolor. (I) Ratiometric Hyper signals ( F 488 / F 405 ) averaged over the regenerative zone of injured heart during the first month of regeneration after resection.\nn = 3. (J) Representative transmural spatial profiles of the Hyper signal at 14 dpa (red line) and sham (green line) hearts. (K) Ex vivo calibration of Hyper F 488 / F 405 ratio as a function of ambient H 2 O 2 concentration. n = 3. Arrow denotes the average F 488 / F 405 ratio seen in regenerative zones at 14 dpa. (L - O) Redox signal showing greater epicardial and myocardial oxidization during regeneration. A 14 dpa Tg( myl7 :hyper) heart was labeled with Redox sensor cc-1. Images show Hyper signals at 488 (L) and 405 nm (M) , and Redox sensor cc-1 signals at 555 nm excitation (N) , and the merging of L and N (O) . Note that the Re- dox sensor cc-1 signal (arrowheads), indicative of intracellular oxidization, was most conspicuous in epicardial cells lacking Hyper expression, and also overlapped with the Hyper signal (arrows) in adjacent myocardium. Scale bars, 100 µ m.\nFigure 2 H 2 O 2 signaling is required for heart regeneration. (A - B) Inhibiting Duox/Nox NADPH oxidases by DPI decreased H 2 O 2 generation. Time-lapse confocal images (A) and statistics (B) of the Hyper ratio in Tg( myl7 :Hyper) hearts prior to and after application of DPI (10 µ M). DMSO (0.1%) was used as control. n = 3-5. (C - E) Treatment with DPI or apocynin impaired cardiac myocyte regeneration. Proliferating myocytes were identified by double-staining with anti-Mef2C (red) and anti-BrdU (green) (C’ , D’ , arrows ) . Note that there were fewer double-positive cells (yellow) in DPI-treated heart (D , D’) than in DMSO control heart (C , C’) . Quantitative results with sham, DMSO, DPI or apocynin treatment are shown in E . n = 5 to 7. See Mate- rials and Methods for details of treatment. (F - I) Accumulated fibrin/collagens (white arrowheads in G , Masson’s staining) and compromised myocardial regeneration (white arrowhead in I , in situ hybridization with myl7 probes) after DPI treatment, com- pared with DMSO control (F , H) . (J - M) Cardiac-specific overexpression of catalase retarded heart regeneration. Tg( myl7 :- Catalase-DsRed) heart displayed larger amounts of fibrin/collagens (white arrowheads in K , Masson’s staining) and compro- mised myocardial regeneration (white arrowheads in M , in situ hybridization with myl7 probes), as compared with respective non-transgenic sibling hearts at 30 dpa (J , L) . n = 5-8. Scale bars, 100 µ m.\ntioxidant capacity of the heart should decrease cardiac regeneration. To directly test this hypothesis, we gener - ated Tg( myl7 :catalase-DsRed) transgenic zebrafish with cardiac-specific expression of catalase. The efficacy of catalase-DsRed was validated by its ability to scavenge H 2 O 2 [36] (Supplementary information, Figure S13C- S13E). Similar to the Duox/Nox inhibition, there were fewer Myl7 + regenerated myocytes (Figure 2M), and extensive cardiac fibrosis (Figure 2K) at 30 dpa in cat - alase-transgenic hearts, compared to non-transgenic sibling hearts (Figure 2L and 2J). However, we were not able to generate ubiquitously heat-inducible Tg( hsp70 :- catalase) transgenic lines, due to lethality of these trans - genic embryos from leaky expression of catalase without heat shock. Moreover, cardiac-specific overexpression of catalase-DsRed suppressed embryonic heart regener - ation in Mtz-induced heart injury of Tg( myl7 :CFP-NTR) transgenic embryos (Supplementary information, Figure S13F-S13H). Previous studies suggest that myocardial dedifferentiation is a critical step for heart regeneration\nin zebrafish [8, 9]. Interestingly, inhibiting H 2 O 2 signaling by either DPI treatment or catalase overexpression had no effects on myocardial sarcomere disassembly, an indi - cator of myocardial dedifferentiation, in injured hearts at 14 dpa (Supplementary information, Figure S15). These results suggest that the high concentration of H 2 O 2 , likely caused by elevated duox/nox2 expression, is obligatory for cardiac regeneration after injury without affecting in - jury-induced myocardial dedifferentiation.\nThe Duox-H 2 O 2 -Dusp6 pathway primes myocardial re- generation by elevating Erk1/2 activity via a derepres- sion mechanism Because developmental signaling pathways of FGF and PDGF are reused during adult zebrafish heart re - generation [20, 21], we investigated whether H 2 O 2 mod- ulated the common MAP kinase pathway downstream of the growth factors. We found that phosphorylation of Erk1/2 (pErk) was increased in injured hearts at 3, 7 and 14 dpa (Figure 3C and Supplementary information,\nFigure 3 H 2 O 2 activates Erk1/2 for myocardial regeneration. (A - B) Phosphorylation of Erk1/2 (pErk1/2) in epicardial cells (A , A’) in 14 dpa heart, as detected by anti-pErk1/2 antibody. Decrease of pErk1/2-positive cells by DPI treatment (B , B’) . Scale bar for A and B , 100 µ m. (C) Western blot showing greater pErk1/2 in injured hearts at 3, 7 and 14 dpa, and its blockade by DPI treatment at 14 dpa. (D-F) Gata4 + myocardial cells in injured areas in control Tg( gata4 :EGFP) heart (DMSO treatment, D , D’ , D’’ ) and their disappearance after inhibition of Duox (DPI treatment, E , E’ , E” ), or Mek1/2 (U0126 treatment, F , F’ , F” ). Car- diomyocytes were co-labeled with anti-EGFP and anti-MF20 antibodies and nuclei were labeled by DAPI. Scale bar for D - F , 100 µ m. (G) Statistics of Gata4 + cardiomyocytes at 14 dpa.\nFigure S16), with the pErk1/2 + cells primarily located in non-myocardium such as the epicardium (Figure 3A and 3A’), evidenced by overlapping with epicardial Raldh2, a molecular marker for both the epicardium and endocar - dium (Supplementary information, Figure S18A-S18C). The total erk1 and erk2 mRNA, as well as total Erk pro - teins were slightly increased during heart regeneration (Figure 3C and Supplementary information, Figure S17). Using Tg( gata4 :EGFP) transgenic reporter zebrafish to track newly regenerated myocardium in injured hearts [8], we found that decreasing pErk1/2 with U0126 (a Mek1/2 inhibitor) interfered with the regeneration of myocardial cells (Figure 3F, 3F’ and 3G), as evidenced by reduced EGFP-positive cells compared with DMSO controls (Figure 3D, 3D’ and 3G). More importantly, the formation of pErk1/2 + cells (Figure 3B, 3B’ and data not shown) and the generation of GATA4-EGFP + myocardial cells (Figure 3E, 3E’, 3G and data not shown) were sen - sitive to inhibition of Duox/Nox2 by treatment of either DPI or apocynin. Unfortunately, strong autofluorescence in Tg( myl7 :catalase-DsRed) prevented us from perform - ing any fluorescence-based immunostaining. These re - sults showed that epicardial Erk1/2- and H 2 O 2 -dependent signals are interlinked in promoting regeneration in the adult zebrafish heart.\nOf the potential redox-sensitive targets of H 2 O 2 , the dual-specificity MAPK phosphatase 6 (Dusp6) is dis - tinct in that it is stimulated by pErk1/2, but reciprocally dephosphorylates and deactivates pErk1/2, forming a negative feedback that limits Erk1/2 activity. Previous reports have shown that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, and that oxidized Dusp6 is prone to degradation, leading to aberrant Erk1/2 activation [37, 38]. Dusp6-knockout mice show greater proliferation of myocytes [39], but the function of Dusp6 in cardiac regeneration is unknown. We thus formulated a hypothesis with two seemingly paradoxical predictions. First, Erk1/2 activation would elevate Dusp6 expression during regeneration, activating a negative feedback (i.e., a repression). However, if H 2 O 2 can oxidize and thereby destabilize Dusp6, ventricular resection-induced H 2 O 2 production would unleash the pro-regenerative pErk1/2 signal (i.e., a derepression).\nTo test the above predictions, we examined Dusp6 expression at both the mRNA and protein levels. Upon injury, dusp6 mRNA level was elevated, peaking at 7 dpa (Figure 4A and Supplementary information, Figure S19), and Dusp6 protein levels were increased (Figure 4B) and Dusp6 was particularly localized in the epicar - dium at 7 and 14 dpa (Figure 4D-4E and Supplementary information, Figures S18D-S18F and S20B-S20D) and decreased at 30 dpa (Figure 4F and Supplementary in -\nformation, Figure S20G). The most intriguing finding is that in injured hearts at 14 dpa after DPI or apocynin treatment, Dusp6 protein, but not dusp6 mRNA, was dramatically further increased in both the epicardium and myocardium with clear nuclear enrichment (Figure 4A, 4B, 4G and 4H), unmasking an inhibitory effect of H 2 O 2 on Dusp6 protein stability in the epicardium and myocardium undergoing active regeneration. We also confirmed this finding on Tg( dusp6 :EGFP), where EGFP expression was previously shown to recapitulate the endogenous dusp6 mRNA expression pattern during embryogenesis [40]. Epicardial EGFP expression was evident in injured areas of adult hearts at 14 dpa (Figure 4I-4J). Furthermore, pErk1/2 was largely co-localized with Dusp6 (Supplementary information, Figure S21C) and inhibiting pErk1/2 by U0126 reduced Dusp6 expres - sion (Supplementary information, Figure S21A-S21D), suggesting a close interlink between them. These data substantiate the idea that Dusp6 mRNA and protein are initially induced as a negative feedback for pErk1/2, but subsequently H 2 O 2 , likely produced by Duox/Nox2, de - stabilizes Dusp6 to impart a derepression mechanism by suppressing this negative feedback in the epicardium.\nTo further evaluate the role of the Duox-H 2 O 2 -Dusp6 derepression signaling pathway in heart regeneration, we used the small molecule BCI, recently developed in a transgenic zebrafish chemical screen [40], which directly inhibits the phosphatase activity of Dusp6. We found that BCI counteracted the inhibitory effect of DPI or apocynin on the generation of the double-positive BrdU + /Mef2C + myocytes, but had no effects on DPI- or apocynin-enhanced cardiac fibrosis at 14 dpa (Figure 4K, 4M, 4N, 4O and Supplementary information, Figure S24A-S24C). Moreover, BCI also reversed the effects of DPI or apocynin on MF20 staining (which indicates myocardial regeneration), and cardiac fibrosis at 30 dpa (Supplementary information, Figure S24D-S24I). These results indicate that BCI mimics the endogenous H 2 O 2 signal when the latter is abrogated. However, in hearts untreated with DPI or apocynin, BCI did not alter the generation of BrdU + /Mef2C + myocytes at 14 dpa (Fig - ure 4K, 4L and 4O), consistent with the idea that BCI is redundant in the presence of an intact endogenous H 2 O 2 signal. Like exogenous H 2 O 2 (Supplementary informa - tion, Figure S10), BCI had no effects on myocardial proliferation in sham-operated hearts (Supplementary information, Figure S23), which is consistent with the relatively low levels of Dusp6 proteins in sham hearts (Figure 4C and Supplementary information, Figure S20A). To examine whether H 2 O 2 is involved in coronary vessel regeneration, we used the Tg( flk1 :EGFP) zebrafish to visualize coronary vessels in the adult heart. We found\nMef2C + proliferating myocytes. Scale bars, 100 µ m.\nthat DPI treatment reduced the amount of Flk1 + vessels, which was rescued by co-injection of BCI (Supplementa - ry information, Figure S22), suggesting an additional role of H 2 O 2 in coronary vessel repair upon ventricular resec - tion. Collectively, interventions at the levels of Duox/\nNox2, H 2 O 2 and Dusp6 corroborated the hypothesis that the Duox-H 2 O 2 -Dusp6 pathway plays an essential role in priming myocardial and blood vessel regeneration, by elevating Erk1/2 activity via a derepression mechanism.\nTo directly test whether zebrafish Dusp6 (zDusp6) is\nsubjected to H 2 O 2 -dependent ubiquitination, we mea - sured the ubiquitinylated zDusp6 (Ub-zDusp6) protein levels with or without H 2 O 2 stimulation in HeLa cells and zebrafish embryos, given that adult zebrafish heart is hardly accessible for these molecular and biochemical methods. We found that exogenous H 2 O 2 treatment de- creased, in a dose-dependent manner, the levels of Myc- tagged zDusp6 proteins in HeLa cells overexpressing zDusp6-Myc (Figure 5A and 5B). Similarly, endogenous zDusp6 underwent degradation in zebrafish embryos after H 2 O 2 treatment (Figure 5C and 5D). Furthermore, Ub-zDusp6 was hardly detectable, but was detected after treatment with 1 mM H 2 O 2 in HeLa cells overexpressing zDusp6-Myc and Flag-tagged ubiquitin (Ub-Flag) (Figure 5E), suggesting the existence of unstable Ub-zDusp6 that is dependent on H 2 O 2 . Proteasome inhibitor MG132 in - creased the amount of Ub-zDusp6 in the absence or pres - ence of H 2 O 2 , supporting the notion that H 2 O 2 treatment enhances the ubiquitination of zDusp6. On the other hand, we failed to detect Ub-zDusp6 in the presence of H 2 O 2 and MG132 in zebrafish embryos overexpressing zDusp6-Myc and Ub-Flag (data not shown), suggesting that Ub-zDusp6 is highly unstable in zebrafish embryos.\nTherefore, H 2 O 2 regulates the ubiquitination and protea - some degradation of zebrafish Dusp6.\nH 2 O 2 promotes heart regeneration independent of im- mune cell recruitment Previous studies have shown that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail-fin epithelium of zebrafish larvae [25, 26]. To differentiate potentially dual function of H 2 O 2 in recruiting inflamma - tory cells and priming heart regeneration, we examined temporal and spatial distribution of immune cells during heart regeneration by labeling macrophages and neu - trophils with either anti-lymphocyte cytosolic protein 1 (Lcp1) or Tg( coro1a :EGFP) [41]. We found that Lcp1 + leukocytes appeared around 3 dpa, peaked at 7 and 14 dpa, decreased at 19 dpa and almost disappeared at 24 dpa (Figure 6A-6F), and the recruitment occurred in a H 2 O 2 -dependent manner (Figure 6G and 6I). Quantifi - cation of Lcp1 + leukocytes during heart regeneration in Figure 6 was included in Supplementary information, Figure S25. Similar dynamics of leukocyte recruitment was obtained by labeling leukocytes with an independent Tg( coro1a :EGFP) transgenic reporter (Supplementary\nFigure 5 H 2 O 2 induces ubiquitination and proteasome degradation of zebrafish Dusp6. (A - B) Exogenous H 2 O 2 decreased the expression level of Myc-tagged zDusp6 in a dose-dependent manner in HeLa cells as evidenced by western blotting (A) . The statistics of A is shown in B . β -actin was used for normalizing protein loading. (C - D) Exogenous H 2 O 2 decreased the ex- pression level of endogenous zDusp6 in a dose-dependent manner in zebrafish embryos at 24 hpf (C) , and the statistics of C is shown in D . β -actin was used for normalizing protein loading. (E) Ub-zDusp6 (arrowhead) was detected after H 2 O 2 and/ or  MG132  treatments  in  HeLa  cells  overexpressing  Myc-tagged  zDusp6  and  Flag-tagged  ubiquitin  (Ub-Flag).  Myc-tagged Dusp6 proteins were detected for all groups except the group without zDusp6-Myc overexpression.\ninformation, Figure S26A-S26F). Furthermore, DPI treatment either from 7 to 14 dpa (when immune cells peaked, DPI timely inhibited leukocyte recruitment) or from 14 to 30 dpa (when immune cells gradually disap - peared, DPI should not affect normal leukocyte recruit - ment) led to increased cardiac fibrosis and compromised myocardial regeneration (Figure 6L and 6N), suggesting that both the early- (7-14 dpa) and late-phase H 2 O 2 pro- duction (14-30 dpa) are required for cardiac regeneration. Importantly, co-injection of BCI with DPI or apocynin largely restored myocardial regeneration (Figure 6M and 6O), but failed to rescue immune cell recruitment (Figure 6H, 6J and Supplementary information, Figure S26G-S26H), indicating that Dusp6 inhibition by BCI suffices to initiate regeneration independently of immune cell recruitment. Taken together, these data suggest that while H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration, the two pathways are apparently separable and relatively independent of each other.\nDusp6 is a repressor for zebrafish heart regeneration To directly examine the role of dusp6 in heart regen- eration, we established Tg( hsp70 :dusp6-His) transgenic zebrafish, where overexpression of Dusp6 in both epicar - dium and myocardium was conditionally driven by heat shock [21]. Transgenic expression of dusp6 was con - firmed to decrease pErk in both embryos and adult hearts (Figure 7A). Compared with non-transgenic siblings (Figure 7B, 7F, 7H, 7J and 7L), injured transgenic hearts displayed fewer Mef2C + /BrdU + proliferating myocytes (Figure 7C-7D), Gata4 + myocytes (Figure 7E and 7G) as well as pERK + cells (Figure 7I) at the injury site, leading to increased cardiac fibrosis (Figure 7K) and diminished cardiac regeneration (Figure 7M). Together with the BCI data (Figures 4 and 6), these results establish a repressive role of dusp6 in heart regeneration, acting through de- phosphorylation of pErk in the epicardium.\n\n\nDiscussion\nIn summary, we have presented three major inter - related findings. First, upon injury, H 2 O 2 is produced, likely by Duox/Nox2, up to 30 µM in the epicardium and adjacent myocardium during the entire course of active myocardial regeneration. Second, H 2 O 2 is required for promoting cardiac myocyte proliferation and suppress - ing fibrosis. Third, the pro-regeneration effect of H 2 O 2 is mediated through destabilizing the redox-sensitive and ubiquitin-conjugated phosphatase Dusp6, which provides a derepression of the common MAP kinase signaling. Furthermore, we have identified BCI as the first small\nmolecule for promoting cardiac regeneration when H 2 O 2 generation was defective. In spite of previous findings that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail- fin epithelium of zebrafish larvae [25, 26], our data show that the H 2 O 2 -guided inflammatory response occurs around 3 dpa in adult zebrafish hearts (Figure 6 and Sup - plementary information, Figure S26). This time course, when compared with that of Duox/Nox2 induction and H 2 O 2 production, supports the idea that it likely involves de novo synthesis of duox transcripts and possibly pro - teins for the production of H 2 O 2 and subsequent recruit - ment of leukocytes in heart regeneration. In contrast, because the injured larvae tail fin is directly exposed to the environment, it needs immediate immune defense after injury and therefore only takes 3 min to recruit neu - trophils to the injured site, through activation of Duox without transcription and translation in the tail fin epithe - lium [25, 26]. In addition to its function in recruiting in - flammatory leukocytes, our data suggest that H 2 O 2 is also required for promoting adult heart regeneration, which is consistent with previous reports on H 2 O 2 function in promoting injury-induced peripheral sensory axon regen - eration in zebrafish larvae [42] and tail regeneration in Xenopus tadpole [43]. H 2 O 2 acts through a redox sensor Lyn to mediate injury-induced leukocyte wound attrac - tion in zebrafish larvae tail fin [25, 26], but direct targets of H 2 O 2 in zebrafish sensory neurons or Xenopus tadpole tail epithelium are not known [42, 43]. Our data show that H 2 O 2 not only destabilizes Dusp6 to elevate growth factor signaling in promoting adult heart regeneration, but also acts through an unknown, independent mech - anism to recruit inflammatory leukocytes during heart regeneration. Future studies need to examine whether H 2 O 2 acts through Lyn or other redox sensors to induce inflammatory leukocyte recruitment during adult heart regeneration.\nGrowth factor signaling pathways such as FGF and PDGF pathways are essential for adult heart regenera - tion [20, 21]. In contrast to previous reports on a neg - ative role of p38 MAPK in mammalian and zebrafish heart regeneration [44,45], our data support that H 2 O 2 in the epicardium (Figure 1) is essential for elevating phosphorylation of Erk1/2 MAPK through repression of Dusp6 during zebrafish heart regeneration (Figures 3-5 and 7). Our data are aligned well with the previous re - ports that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, leading to ubiquitination and subsequent proteasome degradation of Dusp6 and thus aberrant Erk1/2 activation [37, 38] (Figure 5). The elevated pErk1/2 in the epicardium might then activate soluble factors at transcriptional and/or post-transcrip -\ntional levels, which in turn bind to their receptors in the myocardium and promote myocardial proliferation and regeneration. However, we failed to detect the changes in Dusp6 and pErk protein levels in the heart by west - ern blotting (data not shown). It is likely that the subtle changes in Dusp6 and pErk protein levels in the epicardi - um of injured hearts might not be well detected by west -\nFigure 6 H 2 O 2 has dual function in recruiting Lcp1 + leukocytes and promoting heart regeneration. (A - F) Immunostaining of Lcp1 showing that leukocytes (macrophages and neutrophils) were undetectable at sham (A) , appeared at 3 dpa (B) , peaked between 7 and 14 dpa (C , D) and gradually disappeared from 19 to 24 dpa (E , F) . Either apocynin (Apo) (G) or DPI (I) inhib- ited leukocyte recruitment, which could not be rescued by BCI (H , J) at 14 dpa. Myocytes were labeled by anti-MF20, and nuclei were labeled by DAPI. (K - M) All AFOG staining analyses were performed at 30 dpa and DPI and/or BCI treatment was applied as indicated. Myosin was visualized as orange, fibrin as red and collagen as bright blue. AFOG staining showed that DPI treatment from 7 to 14 dpa increased cardiac fibrosis (L) , which was rescued by BCI (M) , compared with DMSO control (K) at 30 dpa. (N - O) AFOG staining showed that DPI treatment from 14 to 30 dpa increased cardiac fibrosis (N) , which was rescued by BCI (O) . Scale bars, 100 µ m.\nern blotting. On the other hand, cardiac-specific overex - pression of catalase, the H 2 O 2 -specific scavenger, also led to regenerative defects in the heart (Figure 2J-2M). New transgenic lines, such as Tg( tcf21 :catalase) without autofluorescence or heat-inducible Tg( hsp70 :catalase) without leaky transgene expression are needed for deter - mining the levels of Dusp6 and pErk by immunostaining\nFigure 7 The repressive effects of Dusp6 overexpression on heart regeneration through dephosphorylation of pERK. (A) Top: a scheme showing that in Tg( hsp70 :dusp6-His), His-tagged Dusp6 is driven by the zebrafish hsp70 promoter. Transgenesis was mediated by Tol2 transponase. Bottom: heat shock-induced expression of His-tagged Dusp6 proteins diminished pErk in transgenic embryos (Tg) at 72 hpf or transgenic adult hearts (Tg), compared with their sibling embryos or adult hearts. β -actin was used to normalize protein loading. (B - D) Mef2C + /BrdU + proliferating myocytes (arrowheads) were decreased in dusp6 transgenic hearts (C) compared with those in non-transgenic siblings (B) , with the statistics shown in D . (E - G) Representative images showing that Tg( gata4 :EGFP) myocytes (arrowheads), which colocalized with cardiac marker MF20, were diminished in dusp6 transgenic hearts (G) compared with their siblings (F) , with the statistics shown in E . (H - I) Reduction of pErk + cells (arrowheads), not overlapping with cardiac marker MF20, in transgenic hearts (I) , compared with non-transgenic siblings (H) . (J - M) Transgenic overexpression of Dusp6 led to cardiac fibrosis (assayed by AFOG staining, K ) and compromised cardiac regeneration (assayed by MF20 staining, M ), compared with their siblings (J , L) . Nuclei were co-stained with DAPI. Scale bars, 100 µ m. after scavenging H 2 O 2 by catalase in the future. Based on the data from others and ours, we propose that the working model for cardiac regenerative signaling comprises two main branches: the forward pro-regenera - tive MAPK signaling triggered by FGF and other growth factors, as recently elucidated [20, 21]; and the derepres - sion mechanism through the Duox/Nox2-H 2 O 2 -Dusp6 pathway that converges on Erk1/2 MAPK signaling primarily in the epicardium, which then activate puta - tive soluble factors to promote myocardial regeneration\n(Figure 8). Such epicardial soluble factors might include Wnt1 and IGF2 that were previously shown to play an essential function in the epicardium/cardiac fibroblasts and myocardium [46-49]. During heart regeneration, the growth factor-MAPK signaling (pErk) is gradually en - hanced in injured hearts from 3 to 14 dpa with the peak around 14 dpa, which induces dusp6 expression (positive induction) that in turn dephosphorylates pErk (negative feedback) (Figures 3 and 4). Simultaneously, H 2 O 2 in- creases and destabilizes Dusp6 with the peak from 7 to\nFigure 8 H 2 O 2 promotes heart regeneration through a derepression mechanism in zebrafish. Upon ventricular resection, FGF and other growth factor receptors are activated and resultant phosphorylation of Mek1/2 (pMek1/2) and Erk1/2 (pErk1/2) in- duces expression of dusp6 , which in turn, exerts a negative feedback on pErk1/2 in non-myocardial cells (primarily epicardial cells). Concurrently, injury elicits a time-dependent, enduring localized duox and, to a minor extent, nox2 expression that sup- ports intense H 2 O 2 formation primarily in the epicardium close to the injury site. High-concentration intracellular H 2 O 2 oxidizes and thereby destabilizes the Dusp6 phosphatase, diminishing its inhibition of pErk1/2. Thus, the newly identified Duox/Nox2- H 2 O 2 -Dusp6 signaling serves as a derepression mechanism and, when activated, unleashes the pro-regenerative pErk1/2 signaling likely through generation of soluble factors that would eventually act to enhance myocardial regeneration. DPI and apocynin (Apo), inhibitors of Duox/Nox2; catalase, catalytic scavenger of H 2 O 2 ; BCI, inhibitor of Dusp6; U0126, inhibitor of Mek1/2.\n14 dpa, which results in decreased Dusp6 expresssion and thus increased pErk levels at 14 dpa when myocar - dial regeneration actively takes place. In addition, we have shown that H 2 O 2 /Dusp6 is also involved in coro - nary vessel regeneration upon ventricular resection. This is consistent with previous studies that FGF and PDGF signaling pathways are essential for coronary vessel formation during heart regeneration [20, 21]. Therefore, this work provides the first intracellular H 2 O 2 -Dusp6- pErk mechanisms downstream of growth factor signaling in adult myocardial and coronary vessel regeneration. As there are highly conserved mammalian homologs for all components of the Duox/Nox2-H 2 O 2 -Dusp6-Erk1/2 axis, this derepression mechanism for cardiac regenera -\ntion, as demonstrated here in zebrafish, may also operate in mammals, and if so, future efforts on activating the mammalian counterpart may offer opportunities for ther - apeutic myocardial regeneration in heart failure.\n\n\nMaterials and Methods\n\n\nZebrafish lines\nZebrafish were raised and handled in accordance with the Peking University Institutional Animal Care and Use Commit - tee accredited by the AAALAC International. Wild-type TL, Tg( myl7 :HyPer) pku327 , ( myl7 :catalase-DsRed), Tg( dusp6 :EGFP) and Tg( gata4 :EGFP) transgenic zebrafish lines at 6-12 months of age were used for heart surgery. Several Tg( myl7 :HyPer) pku327 , Tg( myl7 :catalase-DsRed) pku343 , Tg( myl7 :CFP-NTR) pku360 and\nTg( hsp70 :dusp6-His) pku344 transgenic zebrafish lines were gen - erated in TL background by using Tol2-based transgenesis [31, 50]. Tg( tcf21 :nucEGFP) was previously reported [30]. Tg( coro- 1a :EGFP) was provided by Dr Zilong Wen [41], Tg( dusp6 :EGFP) transgenic zebrafish were obtained from Dr Michael Tsang [40] and Tg( gata4 :EGFP) transgenic zebrafish were kindly provided by Dr Todd Evans [51]. The zebrafish myl7 promoter was provided by Dr C Geoff Burns [52]; pHyPer-Cyto cDNA clone was purchased from Evrogen (Cat No. FP941); zebrafish catalase cDNA (CDS) was isolated from 2-day embryos according to catalase sequence (NM_130912); rat Dusp6 cDNA was obtained from Dr Zhi-Xin Wang [53]; and CFP-NTR was provided by Dr Ling-Fei Luo [35]. These expression constructs were cloned into the Tol2 backbone [50]. Each of the transgenic lines made in our laboratory was char - acterized and/or confirmed by more than two independent found - ers.\nMtz-induced embryonic heart injury in Tg(myl7:CFP-NTR) embryos A large number of cardiomyocytes were damaged by incubat - ing Tg( myl7 :CFP-NTR) transgenic embryos with Mtz essentially as previously described [35]. Briefly, transgenic embryos at 70 hpf were incubated with 2 mM Mtz, and Mtz was then washed off and DPI and/or H 2 O 2 were added at 72 hpf. In this condition, about 80% control transgenic hearts restored normal morphology and function at 124 hpf.\n\n\nAdult zebrafish heart resection\nA ventricular resection procedure was carried out essentially as described by Poss and colleagues [3]. Briefly, adult zebrafish were anesthetized with tricaine and placed with ventral side up into the groove of a sponge. The pericardial sac was exposed by removing surface scales and a small piece of skin with iridectomy scissors. Opening the pericardial sac was aided by tweezers, and the apex of the ventricle was gently pulled up and removed with Vannas scissors. A small piece of wet tissue was placed on the top of the ventricular wound. The heart stopped bleeding very quickly (within 10-15 s) due to the formation of blood clots. The fish was placed back into a water tank, where water was puffed over the gills with a plastic pipette until the fish was breathing and swimming regu - larly. The surface opening sealed automatically within a few days.\nLibrary preparation for a strand-specific poly(A)-positive RNA-seq Sham and 7 dpa hearts were freshly excised and rinsed with pre-cooled PBS. The outflow tract and atrium were removed im - mediately. The total RNA of each group was extracted from about 25 hearts by using QIAGEN RNeasy Microarray Tissue Mini Kit (Cat No. 73304). Poly(A)-positive RNA was purified from 5 µg of total RNA (RNA integrity number ≥ 7.5) with the Dynabeads mRNA purification kit (Invitrogen), following the manufacturer’s instructions. A strand-specific RNA-seq library was then prepared according to the deoxy-UTP approach as reported previously [54]. Amplified materials were loaded onto a flow cell and sequencing was carried out on the Illumina HiSeq2000 platform by running 90 cycles (paired-end design) according to the manufacturer’s instruc - tions.\nRNA-Seq and microarray data analysis and mining Paired-end mRNA sequence tags were mapped on the Zebrafish\nGenome (Zv9) by Tophat (v1.4.1). Multiple alignment reads were discarded. The RPKM scores for each gene were calculated as pre - viously described [55]. A series of Perl (v5.12.2) and R (v2.13.1) scripts were implemented to evaluate the quality of the RNA-seq experiments. Overall, high-quality RNA-seq data were obtained with high base quality, unbiased read distribution on transcripts, correct strand information, little DNA contamination and low mu - tation rate across the reads (Supplementary information, Figure S1). Eight hundred and forty-five upregulated genes between sham and 7 dpa hearts were identified using Cuffdiff (1.3.0) (Supple - mentary information, Figure S2), on the basis of three criteria: (1) P (un-corrected) &lt; 0.05; (2) fold change &gt; 2; and (3) RPKM &gt; 0.2 in at least one sample. Protein sequences of upregulated genes were then subjected to KOBAS (v2.0) to identify enriched biological pathways, with the P -value calculated using a hyper - geometric distribution. FDR correction was performed to adjust for multiple testing and pathways with FDR-corrected P -value &lt; 0.05 were considered statistically significantly enriched. The RNA-seq data entitled “Transcriptome analysis of heart regenera - tion in adult zebrafish” were deposited to the GEO repository with accession number GSE50203 (GSM1215619 for sham heart and GSM1215618 for 7 dpa heart).\nA list of previous known ROS genes were manually selected and evaluated for their relative expression levels between sham and 7 dpa hearts, based on a publicly available microarray of ze - brafish heart regeneration [29] and the above RNA-seq data on ze - brafish heart regeneration. We found that 23 of them were changed between sham and 7 dpa hearts in the microarray data that were expressed as log (7dpa_log2FC_Belmonte), and 32 of them were changed between sham and 7 dpa hearts in our RNA-seq data that were expressed as log (7dpa_logFC_RNAseq) in Supplementary information, Table S4. We wrote in-house R scripts to analyze the microarray data, using package Limma from Bioconductor [56].\n\n\nEx vivo intact heart imaging\nAdult Tg( myl7 :Hyper) hearts were removed and rinsed in mod - ified Tyrode’s solution (in mM: 150 NaCl, 5.4 KCl, 1.5 MgSO 4 , 0.4 NaH 2 PO 4 , 2 CaCl 2 , 10 glucose, 10 HEPES, pH 7.4). The atrium was removed to diminish spontaneous beatings. Hearts were viable for up to 1 h in such conditions, manifested by responses to 1 Hz electric field stimulation. Viability of the hearts was confirmed by their rhythmic contractions upon stimulation. A high speed camera was used to capture the contractions of the hearts and then kymo - graph was generated from the movies. Hearts were imaged under the 20× water objective lens of a Zeiss LSM700 upright confocal microscope with 405- and 488-nm lasers for dual excitation of Hyper, and 505-565 nm emission lights were collected with band- pass filter. For calculating Hyper 488/405 ratio, F 488 and F 405 im- ages were smoothed, background subtracted and divided. For the Hyper ratiometric calibration curve, Hyper transgenic hearts were treated and imaged ex vivo under designated concentrations (5, 10, 20, 50, 100 and 200 µM) of H 2 O 2 for 2 min. For time-lapse record - ings with 10 µM DPI or DMSO treatment, Hyper transgenic hearts were imaged at 30-s intervals for 3 min in normal Tyrode’s solu - tion before application of DPI or DMSO; the total recording time was 45 min. To detect redox signals with chemical indicators, 30 µM of Redox sensor cc-1 (R-14060; Invitrogen) was injected into the thoracic cavity of 14 dpa Tg( myl7 :Hyper) or wild-type zebraf - ish 30 min before heart isolation. Redox sensor cc-1 was excited at 555 nm. Image processing and data analysis were performed using\nthe IDL program (Research Systems).\n\n\nBrdU incorporation and small molecule treatment\nAfter ventricular resection, zebrafish were allowed to recover for 7 days and 50 µl of 2.5 mg/ml BrdU (B5002; Sigma) was then co-injected with 10 µM DPI (D2926; Sigma), 100 µM apocynin (W508454; Sigma), 10 µM BCI (B4313; Sigma), or 10 µM U0126 (U120; Sigma) into the thoracic cavity once daily until the hearts were collected. In pilot experiments, BCI was kindly provided by Dr Michael Tsang of the University of Pittsburg [40].\nWestern blotting and immunoprecipitation Zebrafish tissues were treated in cell lysis buffer (P0013; Bey - otime) in the presence of a protease inhibitor cocktail (P8340; Sig - ma) and PhosSTOP phosphatase inhibitor cocktail (04906845001; Roche). Proteins were quantified with a BCA kit (PA115; Tiangen Biotech), resolved on 12% SDS-PAGE gels, and transferred onto PVDF membranes (162-0177; Bio-Rad). Membranes were in - cubated with the appropriate primary antibody followed by the incubation with the corresponding HRP-conjugated secondary antibody, and proteins were detected with ECL western blotting substrates (W1001; Promega).\nHeLa cells were grown in DMEM high glucose medium (Ther - mo HyClone, SH30022.01B) containing 10% FBS (Thermo Hy- Clone, SV30087.02). The zDusp6 expression construct pcDNA4- zDusp6-Myc and the Flag-tagged ubiquitin expression construct pcDNA-Ub-Flag [57] were cotransfected into HeLa cells using Lipofectamine 2000 (Invitrogen, 11668). Cells were collected and lysed using RIPA buffer (Applygen, C1053) with 0.1 mM PMSF, 5 mM EDTA and 1 mM Complete TM protease inhibitors (Roche, 04693132001) in the presence or absence of 10 mM MG132 (Sig - ma, M7449). Cell lysates were then incubated with rabbit anti-Myc mAb (Cell Signaling, 2278) at 4 °C overnight. After incubation, 40 µl of Protein A+G beads (Abmart, A10001) was added to each sample and the mixture was rotated at 4 °C for 6 h. After centrif - ugation, the beads were washed five times with RIPA buffer and then boiled for 10 min with 30 µl of 2× sample buffer. Mouse an - ti-Flag antibody (Sigma, F1804) and mouse anti-Myc (HRP) mAb (Cowin Bioscience, CW0300) were used for immunoblotting.\nIn situ hybridization, immunostaining and histological methods Adult hearts were removed and fixed in 4% paraformaldehyde at room temperature for 2 h, and then went through an ethanol series for dehydration followed by paraffin embedding and sec - tioning (4 µm). Two duox cDNA clones were isolated from an embryonic zebrafish cDNA library made in Xiong’s lab, and the primer sequences are shown as duox 1F and 1R, or duox 2F and 2R in Supplementary information, Table S5. Duox clone 1 and duox clone 2 cDNAs were used to make two independent duox probe 1 (Supplementary information, Figure S3) and duox probe 2 (Supplementary information, Figure S5). Dusp6 cDNA was kindly provided by Dr Michael Tsang [40] and tcf21 cDNA by Dr Fabrizio Serluca. In situ hybridization on paraffin sections was performed using digoxygenin-labeled probes. Primary antibodies were anti-BrdU (B8434; Sigma), anti-Mef2c (sc-313; Santa Cruz), anti-tErk (4695; Cell Signaling), anti-pErk (9101; Cell Signaling), anti-tErk (MA5-15134; Pierce), anti-GFP (A-11122; Invitrogen), anti-GAPDH (sc-166545; Santa Cruz), anti-MF20 (Developmental\nStudies Hybridoma Bank), anti-Dusp6 (WH0001848M1; Sigma), anti-Lcp1 (124420; Genetex), anti-β-actin (4967; Cell Signal - ing) and anti-Raldh2 (P30011; Abmart). Secondary antibodies used for immunostaining were Alexa Fluor 488 goat anti-mouse IgG (A21121; Invitrogen), Alexa Fluor 488 goat anti-rabbit IgG (A11034; Invitrogen), Alexa Fluor 555 goat anti-mouse IgG (A21424; Invitrogen) and Alexa Fluor 555 goat anti-rabbit IgG (A21428; Invitrogen). Secondary antibodies for western blotting were HRP goat anti-rabbit IgG (sc-2004; Santa Cruz) and HRP goat anti-mouse IgG (sc-2005; Santa Cruz).\n\n\nMasson’s and AFOG staining\nFor AFOG staining, paraffin sections were incubated in Bouin’s solution (HT10132; Sigma) preheated at 56 °C for 2.5 h, incubated at room temperature for 1 h, briefly washed in tap water, incubated in 1% phosphomolybdic acid (P4869; Sigma) for 5 min, rinsed with distilled water and stained with AFOG staining solution (3 g of acid fuchsin (F8129; Sigma), 2 g of orange G (O3756, Sigma) and 1 g of anilin blue (AB0083; BBI) that were dissolved in 200 ml of acidified distilled water, pH 1.1) for 10 min. Sections were rinsed with distilled water, dehydrated with ethanol gradients and mounted. Myosin was visualized as orange, fibrin as red and colla - gen as bright blue.\nMasson’s staining was performed on paraffin sections using BASO Masson’s Trichrome Stain Kit (BA-4079A; Baso Diagnos - tics Inc). Myosin was visualized as red and collagen as bright blue.\nTransmission electron microscopy DMSO- and DPI-treated wild-type adult hearts, as well as Tg( myl7 :catalase-DsRed) transgenic adult hearts after injury at 14 dpa were fixed with glutaraldehyde at a final concentration of 2.5% (v/v) in 0.1 M sodium cacodylate buffer. Samples were rinsed and post-fixed with 1.0% OsO4. Next, the samples were embedded in the Spur resin and sectioned. The sections were stained with uranyl acetate and lead citrate and observed under a transmission electron microscope JEOL 1010 (JEOL Ltd., Japan).\n\n\nRT-PCR analysis\nTotal RNA from sham and injured hearts (15-20 hearts per sample) was extracted and purified using RNeasy@ mini kit (74106; Qiagen), and RT-PCR was performed with OneStep RT- PCR kit (210212; Qiagen) using 40 cycles on 50 ng of total RNA per reaction. Semi-quantitative RT-PCR of ERK1 and ERK2 was performed using 25 PCR cycles. Primer sequences were shown in Supplementary information, Table S5.\nStatistical analysis Statistical analysis was performed on relevant data using Stu - dent’s two-tailed t -test and the GraphPad Prism software package or SPSS18.0. Data were reported as mean ± SEM. For Supple - mentary information, Figure S12B and S12F, statistic significance between the two groups was determined with χ 2 -test. P &lt; 0.05 was considered statistically significant.\n\n\nAcknowledgments\nWe thank Drs RP Xiao, Y Rao, JR Peng, C Wei, K Ouyang, YM Wang, ZL Huang and JC Luo for critical comments on the manuscript, Dr IC Bruce for language editing, Dr Yingchun Hu\nfor help on transmission electromicroscopy, and the Pathology Core of Institute of Molecular Medicine for histology. We also thank Drs J Wang and KD Poss for their kind help in performing in situ hybridization with duox probes on Tg( tcf21 :nucEGFP) ventricles and Drs M Tsang, ZL Wen, T Evans, ZX Wang, LF Luo, GC Burns, J Chen, CM Cao and F Serluca for generous gifts of protocols, plasmid clones and/or transgenic zebrafish lines. This work was supported by the National Basic Research Program of China (2012CB944501, 2010CB529503 and 2013CB531200) and the National Natural Science Foundation of China (31271549, 31130067, 31000644 and 31221002).\n\n\nCell Research (2014) 24:1091-1107.\n\n\nPeidong Han et al . 1093 npg\n\n\nPeidong Han et al . 1095 npg\n\n\nPeidong Han et al . 1097 npg\n\n\nPeidong Han et al . 1099 npg\n\n\nPeidong Han et al . 1101 npg\n\n\nPeidong Han et al . 1103 npg\n\n\nCell Research | Vol 24 No 9 | September 2014\n\n\nPeidong Han et al . 1105 npg\n\n\nPeidong Han et al . 1107 npg\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "ORIGINAL ARTICLE\n© 2014 IBCB, SIBS, CAS    All rights reserved 1001-0602/14  $ 32.00 www.nature.com/cr\n\n\nHydrogen peroxide primes heart regeneration with a derepression mechanism\nPeidong Han 1, * , Xiao-Hai Zhou 1, * , Nannan Chang 1, * , Cheng-Lu Xiao 1 , Shouyu Yan 1 , He Ren 2, 3 , Xin-Zhuang Yang 1 , Mei-Ling Zhang 1 , Qing Wu 1 , Boyang Tang 1, 2 , Ju-Peng Diao 1 , Xiaojun Zhu 1 ,\nChuanmao Zhang 2, 3 , Chuan-Yun Li 1 , Heping Cheng 1, 2, 4 , Jing-Wei Xiong 1\n1 Institute of Molecular Medicine, Beijing Key Laboratory of Cardiometabolic Molecular Medicine and State Key Laboratory of Natural and Biomimetic Drugs, 2 State Key Laboratory of Biomembrane and Membrane Biotechnology, 3 College of Life Sciences, 4 Peking-Tsinghua Center for Life Sciences, Peking University, Beijing 100871, China\nWhile the adult human heart has very limited regenerative potential, the adult zebrafish heart can fully regenerate after 20% ventricular resection. Although previous reports suggest that developmental signaling pathways such as FGF and PDGF are reused in adult heart regeneration, the underlying intracellular mechanisms remain largely un - known. Here we show that H 2 O 2 acts as a novel epicardial and myocardial signal to prime the heart for regeneration in adult zebrafish. Live imaging of intact hearts revealed highly localized H 2 O 2 (~30 µM) production in the epicardi - um and adjacent compact myocardium at the resection site. Decreasing H 2 O 2 formation with the Duox inhibitors di - phenyleneiodonium (DPI) or apocynin, or scavenging H 2 O 2 by catalase overexpression markedly impaired cardiac re - generation while exogenous H 2 O 2 rescued the inhibitory effects of DPI on cardiac regeneration, indicating that H 2 O 2 is an essential and sufficient signal in this process. Mechanistically, elevated H 2 O 2 destabilized the redox-sensitive phosphatase Dusp6 and hence increased the phosphorylation of Erk1/2. The Dusp6 inhibitor BCI achieved similar pro-regenerative effects while transgenic overexpression of dusp6 impaired cardiac regeneration. H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration through two relatively independent pathways. We conclude that H 2 O 2 potentially generated from Duox/Nox2 promotes heart regeneration in zebrafish by unleashing MAP kinase signaling through a derepression mechanism involving Dusp6. Keywords : heart regeneration; zebrafish; Duox; H 2 O 2 ; Dusp6 Cell Research (2014) 24 :1091-1107. doi:10.1038/cr.2014.108; published online 15 August 2014\n*These three authors contributed equally to this work. Correspondence: Jing-Wei Xiong a , Heping Cheng b a, b Tel/Fax: 86-10-62766239 a E-mail: jingwei_xiong@pku.edu.cn b E-mail: chengp@pku.edu.cn\nReceived 11 February 2014; revised 6 May 2014; accepted 9 June 2014; published online 15 August 2014\n\n\nIntroduction\nThe ability to reverse heart failure, which accounts for &gt; 10% annual mortality in humans, is the most challeng - ing task in cardiac regenerative medicine. Many efforts are devoted to cardiac stem cell generation and tissue engineering, although making cardiac stem cells or tissue scaffolds functionally integrated into their host cardiac tissues remains difficult in clinical settings [1, 2]. A num - ber of studies have shown that the adult mammalian heart\nhas limited regenerative capacity, although not sufficient to fully restore heart function after substantial myocar - dial injury [2]. On the other hand, the adult zebrafish and newt hearts have full capacity of cardiac regeneration af - ter ventricular resection or cryoinjury, making zebrafish and newt as powerful animal models for studying heart regeneration in particular and organ regeneration in gen- eral [3-6]. The neonatal mouse heart in the first week of their postnatal life can fully regenerate after ventricular resection, suggesting that mammals gradually lose their regenerative potential during postnatal development [10]. It remains mysterious how a species gains or loses the regenerative potential during evolution.\nZebrafish has become a well-recognized model sys - tem for studying organ regeneration due to incredible regenerative potential in all of its adult organs tested and the availability of transgenesis, genetics and genomic\nresources [7]. The dedifferentiation model for the cellu - lar sources of cardiac repair/regeneration was supported by the fact that cardiac muscles mainly regenerate from pre-existing cardiomyocytes after injury in zebrafish [8, 9] and mammals [10-12]. The stem cell/progenitor cell model for cardiac regeneration in mammals was also recognized by the existence of Sca1 + and c-Kit + cardiac stem cells in mice [13, 14], while the transdifferentiation or lineage reprogramming model was proposed partly based on the ability of cardiac fibroblast transdifferenti - ation into myocytes upon induction by reprogramming factors or microRNA in mice [15-17] and reprogram - ming of embryonic atrial into ventricular myocytes after injury in zebrafish [18]. In spite of great efforts, the mo - lecular mechanisms underlying heart regeneration remain incompletely understood. Previous reports support that developmental signaling pathways such as FGF, PDGF, retinoid acid, TGFβ and integrin are essential for adult zebrafish heart regeneration [19-23]. Ventricular resec - tion induces expression of fgfr2/4 in the epicardium and their ligand fgf17b in the myocardium, and conditional blocking of the FGF signaling by overexpression of dominant-negative FGFR1 substantially increases fibrin/ collagen deposits and diminishes myocardial regenera - tion, supporting a notion that the FGF signaling promotes myocardial regeneration through epicardial-myocardial interaction [21]. A similar strategy revealed that the PDGF signaling is also required for zebrafish heart re - generation [20]. The exact molecular mechanisms down - stream of the FGF/PDGF signaling during adult heart regeneration are currently unknown.\nDuox is a member of NADPH-oxidase (Nox) and related dual oxidase (Duox) family, which has seven members, Nox1, Nox2, Nox3, Nox4, Nox5, Duox1 and Duox2 in humans, but only five (Nox1, Nox2, Nox4, Nox5 and Duox) identified in zebrafish [24]. Function - ally, the Duox or Nox enzyme complex in the plasma membrane catalyzes single-electron transfers from the electron donor NADPH to molecular oxygen, forming superoxide anions, which are quickly converted into hy - drogen peroxide (H 2 O 2 ) by spontaneous dismutation or catalytic reactions via Cu 2+ Zn 2+ -superoxide dismutase. Recently, it has been shown that Duox-generated H 2 O 2 in the tail-fin epithelium, which responds within ~3 min and peaks around 20 min after wounding, is critical for the rapid recruitment of leukocytes to initiate inflam - mation in zebrafish larvae [25, 26]. Sustained H 2 O 2 pro- duction would shift the intracellular redox homeostasis from a more reduced to a more oxidized state, affecting numerous redox-sensitive signaling pathways. Atax - ia-telangiectasia mutated (ATM), a PI3K-like serine/ threonine protein kinase preferentially activated by DNA\ndouble-strand breaks, serves as a sensor of oxidative stress (H 2 O 2 ). Oxidative stress (H 2 O 2 ) elicits the ATM-ho - modimer-Chk2 pathway to induce cell growth arrest or apoptosis [27]. On the other hand, H 2 O 2 acts through a redox sensor Lyn to mediate leukocyte wound attraction [25, 26]. Here we report that, after ventricular resection, H 2 O 2 levels are locally elevated in the epicardium and adjacent myocardium to promote heart regeneration by a derepression mechanism, whereby H 2 O 2 destabilizes the redox-sensitive phosphatase Dusp6 to enhance growth factor signaling.\n\n\nResults\nDuox and Nox2 are spatio-temporally associated with heart regeneration In a genome-wide search for genes critical for cardiac regeneration by mRNA deep sequencing (Supplemen - tary information, Figures S1 and S2), we found that the expression levels of a group of genes related to immune system diseases were upregulated, including p22 phox , p47 phox and p67 phox of the Nox2 complex (Supplementary information, Tables S1-S3; GEO repository accession number GSE50203). Combining with other public mi - croarray databases, we identified 37 genes in the reactive oxygen species (ROS) system that were differentially expressed in the injured zebrafish heart at 7 days post amputation (dpa) (Supplementary information, Table S4), when injury-induced inflammation started to fade and active myocardial regeneration began [3, 28, 29]. RNA in situ hybridization was then applied at multiple time points from 3 to 30 dpa for validation. We found that duox was particularly interesting because of the spatial and temporal patterns of its expression during regeneration. RNA in situ hybridization with two inde - pendent duox probes revealed that duox expression was weak at sham, peaked and was highly localized beneath the surface of the amputation at around 3, 7 and 14 dpa, and returned to the basal level at 30 dpa when regener - ation was nearly complete (Supplementary information, Figures S3A-S3E and S5A-S5E). Each of the two sense duox probes had no detectable signals (Supplementary information, Figures S4 and S5F), indicating the speci - ficity of the two duox probes. RNA in situ hybridization on Tg( tcf21 :nucEGFP) transgenic hearts revealed that duox expression largely overlapped with the EGFP signal (Figure 1A-1C and Supplementaryinformation, Figure S6), which primarily labels the epicardium [30], sug - gesting that duox was enriched in the epicardium. Thus, both the time course and epicardial localization of duox expression correlated with those of active myocardial re - generation after injury.\nSystematic examination revealed that the expression of nox2 , but none of the other nox genes, was moderately induced at 3, 7 and 14 dpa in a pattern similar to that of duox (Supplementary information, Figure S7). RT-PCR confirmed expression of duox and nox2 in 14 dpa hearts (Supplementary information, Figure S3F). These results indicate that the expression of duox and nox2 , with duox being dominant, is spatially and temporally associated with myocardial regeneration after ventricular resection.\nH 2 O 2 signal is essential for heart regeneration To visualize the spatiotemporal dynamics of H 2 O 2 in the intact heart during regeneration, we used Tol2-based transgenesis and zebrafish myosin light chain 7 ( myl7 ) promoter to generate transgenic zebrafish with cardi - ac-specific expression of Hyper (Figure 1D), a fluores - cent protein-based H 2 O 2 sensor [25, 31]. High-resolution confocal imaging was performed in intact adult hearts excised from Tg( myl7 :Hyper) transgenic animals, and H 2 O 2 levels were directly measured as the ratio between the fluorescent signals dually excited at 488 and 405 nm ( R = F 488 / F 405 ). By ex vivo titration on the Hyper-ex - pressing hearts freshly excised from uninjured animals, we obtained a calibration curve (Figure 1K and Supple - mentary information, Figure S9A-S9G) similar to that obtained in vitro [31]. Compared to sham controls (Figure 1E), local H 2 O 2 concentration was clearly elevated in injured hearts at 3, 7 and 14 dpa (Figure 1F-1G and 1I), and returned to the basal level after nearly full regenera - tion at 30 dpa (Figure 1H and 1I). In contrast to the rapid (on the scale of minutes) inflammatory H 2 O 2 response in the tail-fin epithelium of zebrafish larvae [25, 26], there was a slow onset of ventricular resection-induced H 2 O 2 production, as no detectable increase in the Hyper fluorescence ratio occurred within ~1 h after heart injury (data not shown). As seen on confocal optical sections (Figure 1E-1H) and from line plots of the spatial profiles (Figure 1J), the H 2 O 2 gradients spanned ~20 µm depth from the epicardium to the myocardium, reaching a peak concentration of ~30 µM (Figure 1K and Supplementa - ry information, Figure S9A-S9G), which is within the range of concentrations for H 2 O 2 to act as an intracellular messenger [32]. In an independent measurement, we stained 14 dpa Tg( myl7 :Hyper) transgenic hearts with Redox sensor cc-1 and visualized the H 2 O 2 signal in both the epicardium that lacked Hyper expression (Fig - ure 1O) and adjacent compact myocardium (Figure 1L- 1O). Therefore, direct visualization of H 2 O 2 with either a genetically encoded probe or a small-molecule indicator revealed that H 2 O 2 was elevated primarily in the epicar - dium and the adjacent myocardium at the resection site, highly correlated with myocardial regeneration.\nTo investigate whether there is a causal relationship between elevated Duox/Nox2 and H 2 O 2 production and cardiac regeneration, we applied diphenyleneiodoni - um (DPI) (10 µM) or apocynin (100 µM), widely used inhibitors of Duox/Nox enzymes [33] and examined subsequent changes in H 2 O 2 production and cardiac re- generation. We found that Hyper fluorescent signals were decreased by 50% within ~ 40 min after soaking 14 dpa Tg( myl7 :Hyper) hearts in DPI-containing medium (Fig - ure 2A-2B), while the Hyper signals remained stable in control medium containing DMSO only (Figure 2B and Supplementary information, Figure S8A), suggesting that high concentration of H 2 O 2 likely resulted from elevated duox/nox2 expression but was not produced from mito - chondria. Functionality of ex vivo cultured hearts was validated by their rhythmic contractions under 1 Hz elec - trical field stimulation after 1 h culture (Supplementary information, Figure S8B-S8C). Based on the previous re - port that myocardial cells enter mitotic phases in injured areas of the heart upon ventricular resection in zebrafish or myocardial infarction in mice [3, 34], we quantified injury-induced myocyte proliferation by counting BrdU + / Mef2C + double-positive myocytes during heart regener - ation. After ventricular resection, zebrafish were allowed to recover for 7 days and BrdU was then co-injected with either DPI or apocynin into the abdominal cavity once daily until the hearts were collected for examination. Compared with sham-operated controls, we found that BrdU + /Mef2C + myocyte proliferation was induced in in - jured hearts at 14 dpa (Figure 2C, 2C’ and 2E), but their numbers were significantly reduced by treatment with either DPI (Figure 2D, 2D’ and 2E) or apocynin (Figure 2E and data not shown). In addition, extensive cardiac fibrosis (Figure 2G) and reduced Myl7 + myocardial re- generation were evident in injured hearts with DPI (Figure 2I) or apocynin treatment (data not shown), compared with DMSO-treated hearts (Figure 2F and 2H). Although injection of H 2 O 2 did not induce myocyte proliferation in sham-operated animals (Supplementaryinformation, Fig - ure S10), it rescued the inhibitory effects of DPI on the numbers of BrdU + /Mef2C + myocytes and Gata4 + cells (Supplementary information, Figure S11). Importantly, DPI-treated hearts failed to regenerate even at 60 dpa (Supplementary information, Figure S12), suggesting a permanent arrest rather than a delay of regenerative pro - cess. In addition, exogenous H 2 O 2 partially rescued the inhibitory effect of DPI on embryonic heart regeneration (Supplementary information, Figure S13A and S13B) af - ter a large number of cardiomyocytes were damaged by incubating Tg( myl7 :CFP-NTR) transgenic embryos with metronidazole (Mtz), as Mtz can be converted into a tox - ic DNA crosslinking agent by nitroreductase (NTR) as\npreviously described [35]. We further validated the pene - tration of H 2 O 2 by imaging the 72 hours post fertilization (hpf) embryonic Tg( myl7 :Hyper) hearts and found that Hyper signals were increased upon elevated external H 2 O 2 concentration (Supplementary information, Figure\nS14). These results strongly support the hypothesis that H 2 O 2 is required and sufficient for cardiac regeneration. If H 2 O 2 is causally linked to cardiac regeneration as suggested by its inhibition upon treatment with DPI or apocynin, limiting H 2 O 2 signaling by augmenting an -\nFigure 1 Epicardial Duox-generated H 2 O 2 associated with myocardial regeneration. (A - C) After imaging of Tg( tcf21 :nucEG- FP), we performed RNA in situ hybridization with duox probes on 10- µ m cryosections of Tg( tcf21 :nucEGFP) hearts at sham, 7 and 14 dpa. Note the induced expression of duox and its colocalization with epicardial reporter Tg( tcf21 :nucEGFP) in in- jured areas (arrowheads) at 7 and 14 dpa (B , C) while little staining at sham (A) . Scale bar for A - C , 50 µ m. (D) Cardiac-spe- cific expression of Hyper in a zebrafish embryo. Hyper was restricted in the Tg( myl7 :Hyper) transgenic heart at 48 hpf. (E - H) Ex vivo Hyper heart images from sham control (E) , 7 dpa (F) , 14 dpa (G) and 30 dpa (H) . Spatially resolved H 2 O 2 image, in- dexed by the ratio between the F 488 and F 405 images of Hyper (left panels), is presented in pseudocolor. (I) Ratiometric Hyper signals ( F 488 / F 405 ) averaged over the regenerative zone of injured heart during the first month of regeneration after resection.\nn = 3. (J) Representative transmural spatial profiles of the Hyper signal at 14 dpa (red line) and sham (green line) hearts. (K) Ex vivo calibration of Hyper F 488 / F 405 ratio as a function of ambient H 2 O 2 concentration. n = 3. Arrow denotes the average F 488 / F 405 ratio seen in regenerative zones at 14 dpa. (L - O) Redox signal showing greater epicardial and myocardial oxidization during regeneration. A 14 dpa Tg( myl7 :hyper) heart was labeled with Redox sensor cc-1. Images show Hyper signals at 488 (L) and 405 nm (M) , and Redox sensor cc-1 signals at 555 nm excitation (N) , and the merging of L and N (O) . Note that the Re- dox sensor cc-1 signal (arrowheads), indicative of intracellular oxidization, was most conspicuous in epicardial cells lacking Hyper expression, and also overlapped with the Hyper signal (arrows) in adjacent myocardium. Scale bars, 100 µ m.\nFigure 2 H 2 O 2 signaling is required for heart regeneration. (A - B) Inhibiting Duox/Nox NADPH oxidases by DPI decreased H 2 O 2 generation. Time-lapse confocal images (A) and statistics (B) of the Hyper ratio in Tg( myl7 :Hyper) hearts prior to and after application of DPI (10 µ M). DMSO (0.1%) was used as control. n = 3-5. (C - E) Treatment with DPI or apocynin impaired cardiac myocyte regeneration. Proliferating myocytes were identified by double-staining with anti-Mef2C (red) and anti-BrdU (green) (C’ , D’ , arrows ) . Note that there were fewer double-positive cells (yellow) in DPI-treated heart (D , D’) than in DMSO control heart (C , C’) . Quantitative results with sham, DMSO, DPI or apocynin treatment are shown in E . n = 5 to 7. See Mate- rials and Methods for details of treatment. (F - I) Accumulated fibrin/collagens (white arrowheads in G , Masson’s staining) and compromised myocardial regeneration (white arrowhead in I , in situ hybridization with myl7 probes) after DPI treatment, com- pared with DMSO control (F , H) . (J - M) Cardiac-specific overexpression of catalase retarded heart regeneration. Tg( myl7 :- Catalase-DsRed) heart displayed larger amounts of fibrin/collagens (white arrowheads in K , Masson’s staining) and compro- mised myocardial regeneration (white arrowheads in M , in situ hybridization with myl7 probes), as compared with respective non-transgenic sibling hearts at 30 dpa (J , L) . n = 5-8. Scale bars, 100 µ m.\ntioxidant capacity of the heart should decrease cardiac regeneration. To directly test this hypothesis, we gener - ated Tg( myl7 :catalase-DsRed) transgenic zebrafish with cardiac-specific expression of catalase. The efficacy of catalase-DsRed was validated by its ability to scavenge H 2 O 2 [36] (Supplementary information, Figure S13C- S13E). Similar to the Duox/Nox inhibition, there were fewer Myl7 + regenerated myocytes (Figure 2M), and extensive cardiac fibrosis (Figure 2K) at 30 dpa in cat - alase-transgenic hearts, compared to non-transgenic sibling hearts (Figure 2L and 2J). However, we were not able to generate ubiquitously heat-inducible Tg( hsp70 :- catalase) transgenic lines, due to lethality of these trans - genic embryos from leaky expression of catalase without heat shock. Moreover, cardiac-specific overexpression of catalase-DsRed suppressed embryonic heart regener - ation in Mtz-induced heart injury of Tg( myl7 :CFP-NTR) transgenic embryos (Supplementary information, Figure S13F-S13H). Previous studies suggest that myocardial dedifferentiation is a critical step for heart regeneration\nin zebrafish [8, 9]. Interestingly, inhibiting H 2 O 2 signaling by either DPI treatment or catalase overexpression had no effects on myocardial sarcomere disassembly, an indi - cator of myocardial dedifferentiation, in injured hearts at 14 dpa (Supplementary information, Figure S15). These results suggest that the high concentration of H 2 O 2 , likely caused by elevated duox/nox2 expression, is obligatory for cardiac regeneration after injury without affecting in - jury-induced myocardial dedifferentiation.\nThe Duox-H 2 O 2 -Dusp6 pathway primes myocardial re- generation by elevating Erk1/2 activity via a derepres- sion mechanism Because developmental signaling pathways of FGF and PDGF are reused during adult zebrafish heart re - generation [20, 21], we investigated whether H 2 O 2 mod- ulated the common MAP kinase pathway downstream of the growth factors. We found that phosphorylation of Erk1/2 (pErk) was increased in injured hearts at 3, 7 and 14 dpa (Figure 3C and Supplementary information,\nFigure 3 H 2 O 2 activates Erk1/2 for myocardial regeneration. (A - B) Phosphorylation of Erk1/2 (pErk1/2) in epicardial cells (A , A’) in 14 dpa heart, as detected by anti-pErk1/2 antibody. Decrease of pErk1/2-positive cells by DPI treatment (B , B’) . Scale bar for A and B , 100 µ m. (C) Western blot showing greater pErk1/2 in injured hearts at 3, 7 and 14 dpa, and its blockade by DPI treatment at 14 dpa. (D-F) Gata4 + myocardial cells in injured areas in control Tg( gata4 :EGFP) heart (DMSO treatment, D , D’ , D’’ ) and their disappearance after inhibition of Duox (DPI treatment, E , E’ , E” ), or Mek1/2 (U0126 treatment, F , F’ , F” ). Car- diomyocytes were co-labeled with anti-EGFP and anti-MF20 antibodies and nuclei were labeled by DAPI. Scale bar for D - F , 100 µ m. (G) Statistics of Gata4 + cardiomyocytes at 14 dpa.\nFigure S16), with the pErk1/2 + cells primarily located in non-myocardium such as the epicardium (Figure 3A and 3A’), evidenced by overlapping with epicardial Raldh2, a molecular marker for both the epicardium and endocar - dium (Supplementary information, Figure S18A-S18C). The total erk1 and erk2 mRNA, as well as total Erk pro - teins were slightly increased during heart regeneration (Figure 3C and Supplementary information, Figure S17). Using Tg( gata4 :EGFP) transgenic reporter zebrafish to track newly regenerated myocardium in injured hearts [8], we found that decreasing pErk1/2 with U0126 (a Mek1/2 inhibitor) interfered with the regeneration of myocardial cells (Figure 3F, 3F’ and 3G), as evidenced by reduced EGFP-positive cells compared with DMSO controls (Figure 3D, 3D’ and 3G). More importantly, the formation of pErk1/2 + cells (Figure 3B, 3B’ and data not shown) and the generation of GATA4-EGFP + myocardial cells (Figure 3E, 3E’, 3G and data not shown) were sen - sitive to inhibition of Duox/Nox2 by treatment of either DPI or apocynin. Unfortunately, strong autofluorescence in Tg( myl7 :catalase-DsRed) prevented us from perform - ing any fluorescence-based immunostaining. These re - sults showed that epicardial Erk1/2- and H 2 O 2 -dependent signals are interlinked in promoting regeneration in the adult zebrafish heart.\nOf the potential redox-sensitive targets of H 2 O 2 , the dual-specificity MAPK phosphatase 6 (Dusp6) is dis - tinct in that it is stimulated by pErk1/2, but reciprocally dephosphorylates and deactivates pErk1/2, forming a negative feedback that limits Erk1/2 activity. Previous reports have shown that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, and that oxidized Dusp6 is prone to degradation, leading to aberrant Erk1/2 activation [37, 38]. Dusp6-knockout mice show greater proliferation of myocytes [39], but the function of Dusp6 in cardiac regeneration is unknown. We thus formulated a hypothesis with two seemingly paradoxical predictions. First, Erk1/2 activation would elevate Dusp6 expression during regeneration, activating a negative feedback (i.e., a repression). However, if H 2 O 2 can oxidize and thereby destabilize Dusp6, ventricular resection-induced H 2 O 2 production would unleash the pro-regenerative pErk1/2 signal (i.e., a derepression).\nTo test the above predictions, we examined Dusp6 expression at both the mRNA and protein levels. Upon injury, dusp6 mRNA level was elevated, peaking at 7 dpa (Figure 4A and Supplementary information, Figure S19), and Dusp6 protein levels were increased (Figure 4B) and Dusp6 was particularly localized in the epicar - dium at 7 and 14 dpa (Figure 4D-4E and Supplementary information, Figures S18D-S18F and S20B-S20D) and decreased at 30 dpa (Figure 4F and Supplementary in -\nformation, Figure S20G). The most intriguing finding is that in injured hearts at 14 dpa after DPI or apocynin treatment, Dusp6 protein, but not dusp6 mRNA, was dramatically further increased in both the epicardium and myocardium with clear nuclear enrichment (Figure 4A, 4B, 4G and 4H), unmasking an inhibitory effect of H 2 O 2 on Dusp6 protein stability in the epicardium and myocardium undergoing active regeneration. We also confirmed this finding on Tg( dusp6 :EGFP), where EGFP expression was previously shown to recapitulate the endogenous dusp6 mRNA expression pattern during embryogenesis [40]. Epicardial EGFP expression was evident in injured areas of adult hearts at 14 dpa (Figure 4I-4J). Furthermore, pErk1/2 was largely co-localized with Dusp6 (Supplementary information, Figure S21C) and inhibiting pErk1/2 by U0126 reduced Dusp6 expres - sion (Supplementary information, Figure S21A-S21D), suggesting a close interlink between them. These data substantiate the idea that Dusp6 mRNA and protein are initially induced as a negative feedback for pErk1/2, but subsequently H 2 O 2 , likely produced by Duox/Nox2, de - stabilizes Dusp6 to impart a derepression mechanism by suppressing this negative feedback in the epicardium.\nTo further evaluate the role of the Duox-H 2 O 2 -Dusp6 derepression signaling pathway in heart regeneration, we used the small molecule BCI, recently developed in a transgenic zebrafish chemical screen [40], which directly inhibits the phosphatase activity of Dusp6. We found that BCI counteracted the inhibitory effect of DPI or apocynin on the generation of the double-positive BrdU + /Mef2C + myocytes, but had no effects on DPI- or apocynin-enhanced cardiac fibrosis at 14 dpa (Figure 4K, 4M, 4N, 4O and Supplementary information, Figure S24A-S24C). Moreover, BCI also reversed the effects of DPI or apocynin on MF20 staining (which indicates myocardial regeneration), and cardiac fibrosis at 30 dpa (Supplementary information, Figure S24D-S24I). These results indicate that BCI mimics the endogenous H 2 O 2 signal when the latter is abrogated. However, in hearts untreated with DPI or apocynin, BCI did not alter the generation of BrdU + /Mef2C + myocytes at 14 dpa (Fig - ure 4K, 4L and 4O), consistent with the idea that BCI is redundant in the presence of an intact endogenous H 2 O 2 signal. Like exogenous H 2 O 2 (Supplementary informa - tion, Figure S10), BCI had no effects on myocardial proliferation in sham-operated hearts (Supplementary information, Figure S23), which is consistent with the relatively low levels of Dusp6 proteins in sham hearts (Figure 4C and Supplementary information, Figure S20A). To examine whether H 2 O 2 is involved in coronary vessel regeneration, we used the Tg( flk1 :EGFP) zebrafish to visualize coronary vessels in the adult heart. We found\nMef2C + proliferating myocytes. Scale bars, 100 µ m.\nthat DPI treatment reduced the amount of Flk1 + vessels, which was rescued by co-injection of BCI (Supplementa - ry information, Figure S22), suggesting an additional role of H 2 O 2 in coronary vessel repair upon ventricular resec - tion. Collectively, interventions at the levels of Duox/\nNox2, H 2 O 2 and Dusp6 corroborated the hypothesis that the Duox-H 2 O 2 -Dusp6 pathway plays an essential role in priming myocardial and blood vessel regeneration, by elevating Erk1/2 activity via a derepression mechanism.\nTo directly test whether zebrafish Dusp6 (zDusp6) is\nsubjected to H 2 O 2 -dependent ubiquitination, we mea - sured the ubiquitinylated zDusp6 (Ub-zDusp6) protein levels with or without H 2 O 2 stimulation in HeLa cells and zebrafish embryos, given that adult zebrafish heart is hardly accessible for these molecular and biochemical methods. We found that exogenous H 2 O 2 treatment de- creased, in a dose-dependent manner, the levels of Myc- tagged zDusp6 proteins in HeLa cells overexpressing zDusp6-Myc (Figure 5A and 5B). Similarly, endogenous zDusp6 underwent degradation in zebrafish embryos after H 2 O 2 treatment (Figure 5C and 5D). Furthermore, Ub-zDusp6 was hardly detectable, but was detected after treatment with 1 mM H 2 O 2 in HeLa cells overexpressing zDusp6-Myc and Flag-tagged ubiquitin (Ub-Flag) (Figure 5E), suggesting the existence of unstable Ub-zDusp6 that is dependent on H 2 O 2 . Proteasome inhibitor MG132 in - creased the amount of Ub-zDusp6 in the absence or pres - ence of H 2 O 2 , supporting the notion that H 2 O 2 treatment enhances the ubiquitination of zDusp6. On the other hand, we failed to detect Ub-zDusp6 in the presence of H 2 O 2 and MG132 in zebrafish embryos overexpressing zDusp6-Myc and Ub-Flag (data not shown), suggesting that Ub-zDusp6 is highly unstable in zebrafish embryos.\nTherefore, H 2 O 2 regulates the ubiquitination and protea - some degradation of zebrafish Dusp6.\nH 2 O 2 promotes heart regeneration independent of im- mune cell recruitment Previous studies have shown that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail-fin epithelium of zebrafish larvae [25, 26]. To differentiate potentially dual function of H 2 O 2 in recruiting inflamma - tory cells and priming heart regeneration, we examined temporal and spatial distribution of immune cells during heart regeneration by labeling macrophages and neu - trophils with either anti-lymphocyte cytosolic protein 1 (Lcp1) or Tg( coro1a :EGFP) [41]. We found that Lcp1 + leukocytes appeared around 3 dpa, peaked at 7 and 14 dpa, decreased at 19 dpa and almost disappeared at 24 dpa (Figure 6A-6F), and the recruitment occurred in a H 2 O 2 -dependent manner (Figure 6G and 6I). Quantifi - cation of Lcp1 + leukocytes during heart regeneration in Figure 6 was included in Supplementary information, Figure S25. Similar dynamics of leukocyte recruitment was obtained by labeling leukocytes with an independent Tg( coro1a :EGFP) transgenic reporter (Supplementary\nFigure 5 H 2 O 2 induces ubiquitination and proteasome degradation of zebrafish Dusp6. (A - B) Exogenous H 2 O 2 decreased the expression level of Myc-tagged zDusp6 in a dose-dependent manner in HeLa cells as evidenced by western blotting (A) . The statistics of A is shown in B . β -actin was used for normalizing protein loading. (C - D) Exogenous H 2 O 2 decreased the ex- pression level of endogenous zDusp6 in a dose-dependent manner in zebrafish embryos at 24 hpf (C) , and the statistics of C is shown in D . β -actin was used for normalizing protein loading. (E) Ub-zDusp6 (arrowhead) was detected after H 2 O 2 and/ or  MG132  treatments  in  HeLa  cells  overexpressing  Myc-tagged  zDusp6  and  Flag-tagged  ubiquitin  (Ub-Flag).  Myc-tagged Dusp6 proteins were detected for all groups except the group without zDusp6-Myc overexpression.\ninformation, Figure S26A-S26F). Furthermore, DPI treatment either from 7 to 14 dpa (when immune cells peaked, DPI timely inhibited leukocyte recruitment) or from 14 to 30 dpa (when immune cells gradually disap - peared, DPI should not affect normal leukocyte recruit - ment) led to increased cardiac fibrosis and compromised myocardial regeneration (Figure 6L and 6N), suggesting that both the early- (7-14 dpa) and late-phase H 2 O 2 pro- duction (14-30 dpa) are required for cardiac regeneration. Importantly, co-injection of BCI with DPI or apocynin largely restored myocardial regeneration (Figure 6M and 6O), but failed to rescue immune cell recruitment (Figure 6H, 6J and Supplementary information, Figure S26G-S26H), indicating that Dusp6 inhibition by BCI suffices to initiate regeneration independently of immune cell recruitment. Taken together, these data suggest that while H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration, the two pathways are apparently separable and relatively independent of each other.\nDusp6 is a repressor for zebrafish heart regeneration To directly examine the role of dusp6 in heart regen- eration, we established Tg( hsp70 :dusp6-His) transgenic zebrafish, where overexpression of Dusp6 in both epicar - dium and myocardium was conditionally driven by heat shock [21]. Transgenic expression of dusp6 was con - firmed to decrease pErk in both embryos and adult hearts (Figure 7A). Compared with non-transgenic siblings (Figure 7B, 7F, 7H, 7J and 7L), injured transgenic hearts displayed fewer Mef2C + /BrdU + proliferating myocytes (Figure 7C-7D), Gata4 + myocytes (Figure 7E and 7G) as well as pERK + cells (Figure 7I) at the injury site, leading to increased cardiac fibrosis (Figure 7K) and diminished cardiac regeneration (Figure 7M). Together with the BCI data (Figures 4 and 6), these results establish a repressive role of dusp6 in heart regeneration, acting through de- phosphorylation of pErk in the epicardium.\n\n\nDiscussion\nIn summary, we have presented three major inter - related findings. First, upon injury, H 2 O 2 is produced, likely by Duox/Nox2, up to 30 µM in the epicardium and adjacent myocardium during the entire course of active myocardial regeneration. Second, H 2 O 2 is required for promoting cardiac myocyte proliferation and suppress - ing fibrosis. Third, the pro-regeneration effect of H 2 O 2 is mediated through destabilizing the redox-sensitive and ubiquitin-conjugated phosphatase Dusp6, which provides a derepression of the common MAP kinase signaling. Furthermore, we have identified BCI as the first small\nmolecule for promoting cardiac regeneration when H 2 O 2 generation was defective. In spite of previous findings that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail- fin epithelium of zebrafish larvae [25, 26], our data show that the H 2 O 2 -guided inflammatory response occurs around 3 dpa in adult zebrafish hearts (Figure 6 and Sup - plementary information, Figure S26). This time course, when compared with that of Duox/Nox2 induction and H 2 O 2 production, supports the idea that it likely involves de novo synthesis of duox transcripts and possibly pro - teins for the production of H 2 O 2 and subsequent recruit - ment of leukocytes in heart regeneration. In contrast, because the injured larvae tail fin is directly exposed to the environment, it needs immediate immune defense after injury and therefore only takes 3 min to recruit neu - trophils to the injured site, through activation of Duox without transcription and translation in the tail fin epithe - lium [25, 26]. In addition to its function in recruiting in - flammatory leukocytes, our data suggest that H 2 O 2 is also required for promoting adult heart regeneration, which is consistent with previous reports on H 2 O 2 function in promoting injury-induced peripheral sensory axon regen - eration in zebrafish larvae [42] and tail regeneration in Xenopus tadpole [43]. H 2 O 2 acts through a redox sensor Lyn to mediate injury-induced leukocyte wound attrac - tion in zebrafish larvae tail fin [25, 26], but direct targets of H 2 O 2 in zebrafish sensory neurons or Xenopus tadpole tail epithelium are not known [42, 43]. Our data show that H 2 O 2 not only destabilizes Dusp6 to elevate growth factor signaling in promoting adult heart regeneration, but also acts through an unknown, independent mech - anism to recruit inflammatory leukocytes during heart regeneration. Future studies need to examine whether H 2 O 2 acts through Lyn or other redox sensors to induce inflammatory leukocyte recruitment during adult heart regeneration.\nGrowth factor signaling pathways such as FGF and PDGF pathways are essential for adult heart regenera - tion [20, 21]. In contrast to previous reports on a neg - ative role of p38 MAPK in mammalian and zebrafish heart regeneration [44,45], our data support that H 2 O 2 in the epicardium (Figure 1) is essential for elevating phosphorylation of Erk1/2 MAPK through repression of Dusp6 during zebrafish heart regeneration (Figures 3-5 and 7). Our data are aligned well with the previous re - ports that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, leading to ubiquitination and subsequent proteasome degradation of Dusp6 and thus aberrant Erk1/2 activation [37, 38] (Figure 5). The elevated pErk1/2 in the epicardium might then activate soluble factors at transcriptional and/or post-transcrip -\ntional levels, which in turn bind to their receptors in the myocardium and promote myocardial proliferation and regeneration. However, we failed to detect the changes in Dusp6 and pErk protein levels in the heart by west - ern blotting (data not shown). It is likely that the subtle changes in Dusp6 and pErk protein levels in the epicardi - um of injured hearts might not be well detected by west -\nFigure 6 H 2 O 2 has dual function in recruiting Lcp1 + leukocytes and promoting heart regeneration. (A - F) Immunostaining of Lcp1 showing that leukocytes (macrophages and neutrophils) were undetectable at sham (A) , appeared at 3 dpa (B) , peaked between 7 and 14 dpa (C , D) and gradually disappeared from 19 to 24 dpa (E , F) . Either apocynin (Apo) (G) or DPI (I) inhib- ited leukocyte recruitment, which could not be rescued by BCI (H , J) at 14 dpa. Myocytes were labeled by anti-MF20, and nuclei were labeled by DAPI. (K - M) All AFOG staining analyses were performed at 30 dpa and DPI and/or BCI treatment was applied as indicated. Myosin was visualized as orange, fibrin as red and collagen as bright blue. AFOG staining showed that DPI treatment from 7 to 14 dpa increased cardiac fibrosis (L) , which was rescued by BCI (M) , compared with DMSO control (K) at 30 dpa. (N - O) AFOG staining showed that DPI treatment from 14 to 30 dpa increased cardiac fibrosis (N) , which was rescued by BCI (O) . Scale bars, 100 µ m.\nern blotting. On the other hand, cardiac-specific overex - pression of catalase, the H 2 O 2 -specific scavenger, also led to regenerative defects in the heart (Figure 2J-2M). New transgenic lines, such as Tg( tcf21 :catalase) without autofluorescence or heat-inducible Tg( hsp70 :catalase) without leaky transgene expression are needed for deter - mining the levels of Dusp6 and pErk by immunostaining\nFigure 7 The repressive effects of Dusp6 overexpression on heart regeneration through dephosphorylation of pERK. (A) Top: a scheme showing that in Tg( hsp70 :dusp6-His), His-tagged Dusp6 is driven by the zebrafish hsp70 promoter. Transgenesis was mediated by Tol2 transponase. Bottom: heat shock-induced expression of His-tagged Dusp6 proteins diminished pErk in transgenic embryos (Tg) at 72 hpf or transgenic adult hearts (Tg), compared with their sibling embryos or adult hearts. β -actin was used to normalize protein loading. (B - D) Mef2C + /BrdU + proliferating myocytes (arrowheads) were decreased in dusp6 transgenic hearts (C) compared with those in non-transgenic siblings (B) , with the statistics shown in D . (E - G) Representative images showing that Tg( gata4 :EGFP) myocytes (arrowheads), which colocalized with cardiac marker MF20, were diminished in dusp6 transgenic hearts (G) compared with their siblings (F) , with the statistics shown in E . (H - I) Reduction of pErk + cells (arrowheads), not overlapping with cardiac marker MF20, in transgenic hearts (I) , compared with non-transgenic siblings (H) . (J - M) Transgenic overexpression of Dusp6 led to cardiac fibrosis (assayed by AFOG staining, K ) and compromised cardiac regeneration (assayed by MF20 staining, M ), compared with their siblings (J , L) . Nuclei were co-stained with DAPI. Scale bars, 100 µ m. after scavenging H 2 O 2 by catalase in the future. Based on the data from others and ours, we propose that the working model for cardiac regenerative signaling comprises two main branches: the forward pro-regenera - tive MAPK signaling triggered by FGF and other growth factors, as recently elucidated [20, 21]; and the derepres - sion mechanism through the Duox/Nox2-H 2 O 2 -Dusp6 pathway that converges on Erk1/2 MAPK signaling primarily in the epicardium, which then activate puta - tive soluble factors to promote myocardial regeneration\n(Figure 8). Such epicardial soluble factors might include Wnt1 and IGF2 that were previously shown to play an essential function in the epicardium/cardiac fibroblasts and myocardium [46-49]. During heart regeneration, the growth factor-MAPK signaling (pErk) is gradually en - hanced in injured hearts from 3 to 14 dpa with the peak around 14 dpa, which induces dusp6 expression (positive induction) that in turn dephosphorylates pErk (negative feedback) (Figures 3 and 4). Simultaneously, H 2 O 2 in- creases and destabilizes Dusp6 with the peak from 7 to\nFigure 8 H 2 O 2 promotes heart regeneration through a derepression mechanism in zebrafish. Upon ventricular resection, FGF and other growth factor receptors are activated and resultant phosphorylation of Mek1/2 (pMek1/2) and Erk1/2 (pErk1/2) in- duces expression of dusp6 , which in turn, exerts a negative feedback on pErk1/2 in non-myocardial cells (primarily epicardial cells). Concurrently, injury elicits a time-dependent, enduring localized duox and, to a minor extent, nox2 expression that sup- ports intense H 2 O 2 formation primarily in the epicardium close to the injury site. High-concentration intracellular H 2 O 2 oxidizes and thereby destabilizes the Dusp6 phosphatase, diminishing its inhibition of pErk1/2. Thus, the newly identified Duox/Nox2- H 2 O 2 -Dusp6 signaling serves as a derepression mechanism and, when activated, unleashes the pro-regenerative pErk1/2 signaling likely through generation of soluble factors that would eventually act to enhance myocardial regeneration. DPI and apocynin (Apo), inhibitors of Duox/Nox2; catalase, catalytic scavenger of H 2 O 2 ; BCI, inhibitor of Dusp6; U0126, inhibitor of Mek1/2.\n14 dpa, which results in decreased Dusp6 expresssion and thus increased pErk levels at 14 dpa when myocar - dial regeneration actively takes place. In addition, we have shown that H 2 O 2 /Dusp6 is also involved in coro - nary vessel regeneration upon ventricular resection. This is consistent with previous studies that FGF and PDGF signaling pathways are essential for coronary vessel formation during heart regeneration [20, 21]. Therefore, this work provides the first intracellular H 2 O 2 -Dusp6- pErk mechanisms downstream of growth factor signaling in adult myocardial and coronary vessel regeneration. As there are highly conserved mammalian homologs for all components of the Duox/Nox2-H 2 O 2 -Dusp6-Erk1/2 axis, this derepression mechanism for cardiac regenera -\ntion, as demonstrated here in zebrafish, may also operate in mammals, and if so, future efforts on activating the mammalian counterpart may offer opportunities for ther - apeutic myocardial regeneration in heart failure.\n\n\nMaterials and Methods\n\n\nZebrafish lines\nZebrafish were raised and handled in accordance with the Peking University Institutional Animal Care and Use Commit - tee accredited by the AAALAC International. Wild-type TL, Tg( myl7 :HyPer) pku327 , ( myl7 :catalase-DsRed), Tg( dusp6 :EGFP) and Tg( gata4 :EGFP) transgenic zebrafish lines at 6-12 months of age were used for heart surgery. Several Tg( myl7 :HyPer) pku327 , Tg( myl7 :catalase-DsRed) pku343 , Tg( myl7 :CFP-NTR) pku360 and\nTg( hsp70 :dusp6-His) pku344 transgenic zebrafish lines were gen - erated in TL background by using Tol2-based transgenesis [31, 50]. Tg( tcf21 :nucEGFP) was previously reported [30]. Tg( coro- 1a :EGFP) was provided by Dr Zilong Wen [41], Tg( dusp6 :EGFP) transgenic zebrafish were obtained from Dr Michael Tsang [40] and Tg( gata4 :EGFP) transgenic zebrafish were kindly provided by Dr Todd Evans [51]. The zebrafish myl7 promoter was provided by Dr C Geoff Burns [52]; pHyPer-Cyto cDNA clone was purchased from Evrogen (Cat No. FP941); zebrafish catalase cDNA (CDS) was isolated from 2-day embryos according to catalase sequence (NM_130912); rat Dusp6 cDNA was obtained from Dr Zhi-Xin Wang [53]; and CFP-NTR was provided by Dr Ling-Fei Luo [35]. These expression constructs were cloned into the Tol2 backbone [50]. Each of the transgenic lines made in our laboratory was char - acterized and/or confirmed by more than two independent found - ers.\nMtz-induced embryonic heart injury in Tg(myl7:CFP-NTR) embryos A large number of cardiomyocytes were damaged by incubat - ing Tg( myl7 :CFP-NTR) transgenic embryos with Mtz essentially as previously described [35]. Briefly, transgenic embryos at 70 hpf were incubated with 2 mM Mtz, and Mtz was then washed off and DPI and/or H 2 O 2 were added at 72 hpf. In this condition, about 80% control transgenic hearts restored normal morphology and function at 124 hpf.\n\n\nAdult zebrafish heart resection\nA ventricular resection procedure was carried out essentially as described by Poss and colleagues [3]. Briefly, adult zebrafish were anesthetized with tricaine and placed with ventral side up into the groove of a sponge. The pericardial sac was exposed by removing surface scales and a small piece of skin with iridectomy scissors. Opening the pericardial sac was aided by tweezers, and the apex of the ventricle was gently pulled up and removed with Vannas scissors. A small piece of wet tissue was placed on the top of the ventricular wound. The heart stopped bleeding very quickly (within 10-15 s) due to the formation of blood clots. The fish was placed back into a water tank, where water was puffed over the gills with a plastic pipette until the fish was breathing and swimming regu - larly. The surface opening sealed automatically within a few days.\nLibrary preparation for a strand-specific poly(A)-positive RNA-seq Sham and 7 dpa hearts were freshly excised and rinsed with pre-cooled PBS. The outflow tract and atrium were removed im - mediately. The total RNA of each group was extracted from about 25 hearts by using QIAGEN RNeasy Microarray Tissue Mini Kit (Cat No. 73304). Poly(A)-positive RNA was purified from 5 µg of total RNA (RNA integrity number ≥ 7.5) with the Dynabeads mRNA purification kit (Invitrogen), following the manufacturer’s instructions. A strand-specific RNA-seq library was then prepared according to the deoxy-UTP approach as reported previously [54]. Amplified materials were loaded onto a flow cell and sequencing was carried out on the Illumina HiSeq2000 platform by running 90 cycles (paired-end design) according to the manufacturer’s instruc - tions.\nRNA-Seq and microarray data analysis and mining Paired-end mRNA sequence tags were mapped on the Zebrafish\nGenome (Zv9) by Tophat (v1.4.1). Multiple alignment reads were discarded. The RPKM scores for each gene were calculated as pre - viously described [55]. A series of Perl (v5.12.2) and R (v2.13.1) scripts were implemented to evaluate the quality of the RNA-seq experiments. Overall, high-quality RNA-seq data were obtained with high base quality, unbiased read distribution on transcripts, correct strand information, little DNA contamination and low mu - tation rate across the reads (Supplementary information, Figure S1). Eight hundred and forty-five upregulated genes between sham and 7 dpa hearts were identified using Cuffdiff (1.3.0) (Supple - mentary information, Figure S2), on the basis of three criteria: (1) P (un-corrected) &lt; 0.05; (2) fold change &gt; 2; and (3) RPKM &gt; 0.2 in at least one sample. Protein sequences of upregulated genes were then subjected to KOBAS (v2.0) to identify enriched biological pathways, with the P -value calculated using a hyper - geometric distribution. FDR correction was performed to adjust for multiple testing and pathways with FDR-corrected P -value &lt; 0.05 were considered statistically significantly enriched. The RNA-seq data entitled “Transcriptome analysis of heart regenera - tion in adult zebrafish” were deposited to the GEO repository with accession number GSE50203 (GSM1215619 for sham heart and GSM1215618 for 7 dpa heart).\nA list of previous known ROS genes were manually selected and evaluated for their relative expression levels between sham and 7 dpa hearts, based on a publicly available microarray of ze - brafish heart regeneration [29] and the above RNA-seq data on ze - brafish heart regeneration. We found that 23 of them were changed between sham and 7 dpa hearts in the microarray data that were expressed as log (7dpa_log2FC_Belmonte), and 32 of them were changed between sham and 7 dpa hearts in our RNA-seq data that were expressed as log (7dpa_logFC_RNAseq) in Supplementary information, Table S4. We wrote in-house R scripts to analyze the microarray data, using package Limma from Bioconductor [56].\n\n\nEx vivo intact heart imaging\nAdult Tg( myl7 :Hyper) hearts were removed and rinsed in mod - ified Tyrode’s solution (in mM: 150 NaCl, 5.4 KCl, 1.5 MgSO 4 , 0.4 NaH 2 PO 4 , 2 CaCl 2 , 10 glucose, 10 HEPES, pH 7.4). The atrium was removed to diminish spontaneous beatings. Hearts were viable for up to 1 h in such conditions, manifested by responses to 1 Hz electric field stimulation. Viability of the hearts was confirmed by their rhythmic contractions upon stimulation. A high speed camera was used to capture the contractions of the hearts and then kymo - graph was generated from the movies. Hearts were imaged under the 20× water objective lens of a Zeiss LSM700 upright confocal microscope with 405- and 488-nm lasers for dual excitation of Hyper, and 505-565 nm emission lights were collected with band- pass filter. For calculating Hyper 488/405 ratio, F 488 and F 405 im- ages were smoothed, background subtracted and divided. For the Hyper ratiometric calibration curve, Hyper transgenic hearts were treated and imaged ex vivo under designated concentrations (5, 10, 20, 50, 100 and 200 µM) of H 2 O 2 for 2 min. For time-lapse record - ings with 10 µM DPI or DMSO treatment, Hyper transgenic hearts were imaged at 30-s intervals for 3 min in normal Tyrode’s solu - tion before application of DPI or DMSO; the total recording time was 45 min. To detect redox signals with chemical indicators, 30 µM of Redox sensor cc-1 (R-14060; Invitrogen) was injected into the thoracic cavity of 14 dpa Tg( myl7 :Hyper) or wild-type zebraf - ish 30 min before heart isolation. Redox sensor cc-1 was excited at 555 nm. Image processing and data analysis were performed using\nthe IDL program (Research Systems).\n\n\nBrdU incorporation and small molecule treatment\nAfter ventricular resection, zebrafish were allowed to recover for 7 days and 50 µl of 2.5 mg/ml BrdU (B5002; Sigma) was then co-injected with 10 µM DPI (D2926; Sigma), 100 µM apocynin (W508454; Sigma), 10 µM BCI (B4313; Sigma), or 10 µM U0126 (U120; Sigma) into the thoracic cavity once daily until the hearts were collected. In pilot experiments, BCI was kindly provided by Dr Michael Tsang of the University of Pittsburg [40].\nWestern blotting and immunoprecipitation Zebrafish tissues were treated in cell lysis buffer (P0013; Bey - otime) in the presence of a protease inhibitor cocktail (P8340; Sig - ma) and PhosSTOP phosphatase inhibitor cocktail (04906845001; Roche). Proteins were quantified with a BCA kit (PA115; Tiangen Biotech), resolved on 12% SDS-PAGE gels, and transferred onto PVDF membranes (162-0177; Bio-Rad). Membranes were in - cubated with the appropriate primary antibody followed by the incubation with the corresponding HRP-conjugated secondary antibody, and proteins were detected with ECL western blotting substrates (W1001; Promega).\nHeLa cells were grown in DMEM high glucose medium (Ther - mo HyClone, SH30022.01B) containing 10% FBS (Thermo Hy- Clone, SV30087.02). The zDusp6 expression construct pcDNA4- zDusp6-Myc and the Flag-tagged ubiquitin expression construct pcDNA-Ub-Flag [57] were cotransfected into HeLa cells using Lipofectamine 2000 (Invitrogen, 11668). Cells were collected and lysed using RIPA buffer (Applygen, C1053) with 0.1 mM PMSF, 5 mM EDTA and 1 mM Complete TM protease inhibitors (Roche, 04693132001) in the presence or absence of 10 mM MG132 (Sig - ma, M7449). Cell lysates were then incubated with rabbit anti-Myc mAb (Cell Signaling, 2278) at 4 °C overnight. After incubation, 40 µl of Protein A+G beads (Abmart, A10001) was added to each sample and the mixture was rotated at 4 °C for 6 h. After centrif - ugation, the beads were washed five times with RIPA buffer and then boiled for 10 min with 30 µl of 2× sample buffer. Mouse an - ti-Flag antibody (Sigma, F1804) and mouse anti-Myc (HRP) mAb (Cowin Bioscience, CW0300) were used for immunoblotting.\nIn situ hybridization, immunostaining and histological methods Adult hearts were removed and fixed in 4% paraformaldehyde at room temperature for 2 h, and then went through an ethanol series for dehydration followed by paraffin embedding and sec - tioning (4 µm). Two duox cDNA clones were isolated from an embryonic zebrafish cDNA library made in Xiong’s lab, and the primer sequences are shown as duox 1F and 1R, or duox 2F and 2R in Supplementary information, Table S5. Duox clone 1 and duox clone 2 cDNAs were used to make two independent duox probe 1 (Supplementary information, Figure S3) and duox probe 2 (Supplementary information, Figure S5). Dusp6 cDNA was kindly provided by Dr Michael Tsang [40] and tcf21 cDNA by Dr Fabrizio Serluca. In situ hybridization on paraffin sections was performed using digoxygenin-labeled probes. Primary antibodies were anti-BrdU (B8434; Sigma), anti-Mef2c (sc-313; Santa Cruz), anti-tErk (4695; Cell Signaling), anti-pErk (9101; Cell Signaling), anti-tErk (MA5-15134; Pierce), anti-GFP (A-11122; Invitrogen), anti-GAPDH (sc-166545; Santa Cruz), anti-MF20 (Developmental\nStudies Hybridoma Bank), anti-Dusp6 (WH0001848M1; Sigma), anti-Lcp1 (124420; Genetex), anti-β-actin (4967; Cell Signal - ing) and anti-Raldh2 (P30011; Abmart). Secondary antibodies used for immunostaining were Alexa Fluor 488 goat anti-mouse IgG (A21121; Invitrogen), Alexa Fluor 488 goat anti-rabbit IgG (A11034; Invitrogen), Alexa Fluor 555 goat anti-mouse IgG (A21424; Invitrogen) and Alexa Fluor 555 goat anti-rabbit IgG (A21428; Invitrogen). Secondary antibodies for western blotting were HRP goat anti-rabbit IgG (sc-2004; Santa Cruz) and HRP goat anti-mouse IgG (sc-2005; Santa Cruz).\n\n\nMasson’s and AFOG staining\nFor AFOG staining, paraffin sections were incubated in Bouin’s solution (HT10132; Sigma) preheated at 56 °C for 2.5 h, incubated at room temperature for 1 h, briefly washed in tap water, incubated in 1% phosphomolybdic acid (P4869; Sigma) for 5 min, rinsed with distilled water and stained with AFOG staining solution (3 g of acid fuchsin (F8129; Sigma), 2 g of orange G (O3756, Sigma) and 1 g of anilin blue (AB0083; BBI) that were dissolved in 200 ml of acidified distilled water, pH 1.1) for 10 min. Sections were rinsed with distilled water, dehydrated with ethanol gradients and mounted. Myosin was visualized as orange, fibrin as red and colla - gen as bright blue.\nMasson’s staining was performed on paraffin sections using BASO Masson’s Trichrome Stain Kit (BA-4079A; Baso Diagnos - tics Inc). Myosin was visualized as red and collagen as bright blue.\nTransmission electron microscopy DMSO- and DPI-treated wild-type adult hearts, as well as Tg( myl7 :catalase-DsRed) transgenic adult hearts after injury at 14 dpa were fixed with glutaraldehyde at a final concentration of 2.5% (v/v) in 0.1 M sodium cacodylate buffer. Samples were rinsed and post-fixed with 1.0% OsO4. Next, the samples were embedded in the Spur resin and sectioned. The sections were stained with uranyl acetate and lead citrate and observed under a transmission electron microscope JEOL 1010 (JEOL Ltd., Japan).\n\n\nRT-PCR analysis\nTotal RNA from sham and injured hearts (15-20 hearts per sample) was extracted and purified using RNeasy@ mini kit (74106; Qiagen), and RT-PCR was performed with OneStep RT- PCR kit (210212; Qiagen) using 40 cycles on 50 ng of total RNA per reaction. Semi-quantitative RT-PCR of ERK1 and ERK2 was performed using 25 PCR cycles. Primer sequences were shown in Supplementary information, Table S5.\nStatistical analysis Statistical analysis was performed on relevant data using Stu - dent’s two-tailed t -test and the GraphPad Prism software package or SPSS18.0. Data were reported as mean ± SEM. For Supple - mentary information, Figure S12B and S12F, statistic significance between the two groups was determined with χ 2 -test. P &lt; 0.05 was considered statistically significant.\n\n\nAcknowledgments\nWe thank Drs RP Xiao, Y Rao, JR Peng, C Wei, K Ouyang, YM Wang, ZL Huang and JC Luo for critical comments on the manuscript, Dr IC Bruce for language editing, Dr Yingchun Hu\nfor help on transmission electromicroscopy, and the Pathology Core of Institute of Molecular Medicine for histology. We also thank Drs J Wang and KD Poss for their kind help in performing in situ hybridization with duox probes on Tg( tcf21 :nucEGFP) ventricles and Drs M Tsang, ZL Wen, T Evans, ZX Wang, LF Luo, GC Burns, J Chen, CM Cao and F Serluca for generous gifts of protocols, plasmid clones and/or transgenic zebrafish lines. This work was supported by the National Basic Research Program of China (2012CB944501, 2010CB529503 and 2013CB531200) and the National Natural Science Foundation of China (31271549, 31130067, 31000644 and 31221002).\n\n\nCell Research (2014) 24:1091-1107.\n\n\nPeidong Han et al . 1093 npg\n\n\nPeidong Han et al . 1095 npg\n\n\nPeidong Han et al . 1097 npg\n\n\nPeidong Han et al . 1099 npg\n\n\nPeidong Han et al . 1101 npg\n\n\nPeidong Han et al . 1103 npg\n\n\nCell Research | Vol 24 No 9 | September 2014\n\n\nPeidong Han et al . 1105 npg\n\n\nPeidong Han et al . 1107 npg\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''ORIGINAL ARTICLE\n© 2014 IBCB, SIBS, CAS    All rights reserved 1001-0602/14  $ 32.00 www.nature.com/cr\n\n\nHydrogen peroxide primes heart regeneration with a derepression mechanism\nPeidong Han 1, * , Xiao-Hai Zhou 1, * , Nannan Chang 1, * , Cheng-Lu Xiao 1 , Shouyu Yan 1 , He Ren 2, 3 , Xin-Zhuang Yang 1 , Mei-Ling Zhang 1 , Qing Wu 1 , Boyang Tang 1, 2 , Ju-Peng Diao 1 , Xiaojun Zhu 1 ,\nChuanmao Zhang 2, 3 , Chuan-Yun Li 1 , Heping Cheng 1, 2, 4 , Jing-Wei Xiong 1\n1 Institute of Molecular Medicine, Beijing Key Laboratory of Cardiometabolic Molecular Medicine and State Key Laboratory of Natural and Biomimetic Drugs, 2 State Key Laboratory of Biomembrane and Membrane Biotechnology, 3 College of Life Sciences, 4 Peking-Tsinghua Center for Life Sciences, Peking University, Beijing 100871, China\nWhile the adult human heart has very limited regenerative potential, the adult zebrafish heart can fully regenerate after 20% ventricular resection. Although previous reports suggest that developmental signaling pathways such as FGF and PDGF are reused in adult heart regeneration, the underlying intracellular mechanisms remain largely un - known. Here we show that H 2 O 2 acts as a novel epicardial and myocardial signal to prime the heart for regeneration in adult zebrafish. Live imaging of intact hearts revealed highly localized H 2 O 2 (~30 µM) production in the epicardi - um and adjacent compact myocardium at the resection site. Decreasing H 2 O 2 formation with the Duox inhibitors di - phenyleneiodonium (DPI) or apocynin, or scavenging H 2 O 2 by catalase overexpression markedly impaired cardiac re - generation while exogenous H 2 O 2 rescued the inhibitory effects of DPI on cardiac regeneration, indicating that H 2 O 2 is an essential and sufficient signal in this process. Mechanistically, elevated H 2 O 2 destabilized the redox-sensitive phosphatase Dusp6 and hence increased the phosphorylation of Erk1/2. The Dusp6 inhibitor BCI achieved similar pro-regenerative effects while transgenic overexpression of dusp6 impaired cardiac regeneration. H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration through two relatively independent pathways. We conclude that H 2 O 2 potentially generated from Duox/Nox2 promotes heart regeneration in zebrafish by unleashing MAP kinase signaling through a derepression mechanism involving Dusp6. Keywords : heart regeneration; zebrafish; Duox; H 2 O 2 ; Dusp6 Cell Research (2014) 24 :1091-1107. doi:10.1038/cr.2014.108; published online 15 August 2014\n*These three authors contributed equally to this work. Correspondence: Jing-Wei Xiong a , Heping Cheng b a, b Tel/Fax: 86-10-62766239 a E-mail: jingwei_xiong@pku.edu.cn b E-mail: chengp@pku.edu.cn\nReceived 11 February 2014; revised 6 May 2014; accepted 9 June 2014; published online 15 August 2014\n\n\nIntroduction\nThe ability to reverse heart failure, which accounts for &gt; 10% annual mortality in humans, is the most challeng - ing task in cardiac regenerative medicine. Many efforts are devoted to cardiac stem cell generation and tissue engineering, although making cardiac stem cells or tissue scaffolds functionally integrated into their host cardiac tissues remains difficult in clinical settings [1, 2]. A num - ber of studies have shown that the adult mammalian heart\nhas limited regenerative capacity, although not sufficient to fully restore heart function after substantial myocar - dial injury [2]. On the other hand, the adult zebrafish and newt hearts have full capacity of cardiac regeneration af - ter ventricular resection or cryoinjury, making zebrafish and newt as powerful animal models for studying heart regeneration in particular and organ regeneration in gen- eral [3-6]. The neonatal mouse heart in the first week of their postnatal life can fully regenerate after ventricular resection, suggesting that mammals gradually lose their regenerative potential during postnatal development [10]. It remains mysterious how a species gains or loses the regenerative potential during evolution.\nZebrafish has become a well-recognized model sys - tem for studying organ regeneration due to incredible regenerative potential in all of its adult organs tested and the availability of transgenesis, genetics and genomic\nresources [7]. The dedifferentiation model for the cellu - lar sources of cardiac repair/regeneration was supported by the fact that cardiac muscles mainly regenerate from pre-existing cardiomyocytes after injury in zebrafish [8, 9] and mammals [10-12]. The stem cell/progenitor cell model for cardiac regeneration in mammals was also recognized by the existence of Sca1 + and c-Kit + cardiac stem cells in mice [13, 14], while the transdifferentiation or lineage reprogramming model was proposed partly based on the ability of cardiac fibroblast transdifferenti - ation into myocytes upon induction by reprogramming factors or microRNA in mice [15-17] and reprogram - ming of embryonic atrial into ventricular myocytes after injury in zebrafish [18]. In spite of great efforts, the mo - lecular mechanisms underlying heart regeneration remain incompletely understood. Previous reports support that developmental signaling pathways such as FGF, PDGF, retinoid acid, TGFβ and integrin are essential for adult zebrafish heart regeneration [19-23]. Ventricular resec - tion induces expression of fgfr2/4 in the epicardium and their ligand fgf17b in the myocardium, and conditional blocking of the FGF signaling by overexpression of dominant-negative FGFR1 substantially increases fibrin/ collagen deposits and diminishes myocardial regenera - tion, supporting a notion that the FGF signaling promotes myocardial regeneration through epicardial-myocardial interaction [21]. A similar strategy revealed that the PDGF signaling is also required for zebrafish heart re - generation [20]. The exact molecular mechanisms down - stream of the FGF/PDGF signaling during adult heart regeneration are currently unknown.\nDuox is a member of NADPH-oxidase (Nox) and related dual oxidase (Duox) family, which has seven members, Nox1, Nox2, Nox3, Nox4, Nox5, Duox1 and Duox2 in humans, but only five (Nox1, Nox2, Nox4, Nox5 and Duox) identified in zebrafish [24]. Function - ally, the Duox or Nox enzyme complex in the plasma membrane catalyzes single-electron transfers from the electron donor NADPH to molecular oxygen, forming superoxide anions, which are quickly converted into hy - drogen peroxide (H 2 O 2 ) by spontaneous dismutation or catalytic reactions via Cu 2+ Zn 2+ -superoxide dismutase. Recently, it has been shown that Duox-generated H 2 O 2 in the tail-fin epithelium, which responds within ~3 min and peaks around 20 min after wounding, is critical for the rapid recruitment of leukocytes to initiate inflam - mation in zebrafish larvae [25, 26]. Sustained H 2 O 2 pro- duction would shift the intracellular redox homeostasis from a more reduced to a more oxidized state, affecting numerous redox-sensitive signaling pathways. Atax - ia-telangiectasia mutated (ATM), a PI3K-like serine/ threonine protein kinase preferentially activated by DNA\ndouble-strand breaks, serves as a sensor of oxidative stress (H 2 O 2 ). Oxidative stress (H 2 O 2 ) elicits the ATM-ho - modimer-Chk2 pathway to induce cell growth arrest or apoptosis [27]. On the other hand, H 2 O 2 acts through a redox sensor Lyn to mediate leukocyte wound attraction [25, 26]. Here we report that, after ventricular resection, H 2 O 2 levels are locally elevated in the epicardium and adjacent myocardium to promote heart regeneration by a derepression mechanism, whereby H 2 O 2 destabilizes the redox-sensitive phosphatase Dusp6 to enhance growth factor signaling.\n\n\nResults\nDuox and Nox2 are spatio-temporally associated with heart regeneration In a genome-wide search for genes critical for cardiac regeneration by mRNA deep sequencing (Supplemen - tary information, Figures S1 and S2), we found that the expression levels of a group of genes related to immune system diseases were upregulated, including p22 phox , p47 phox and p67 phox of the Nox2 complex (Supplementary information, Tables S1-S3; GEO repository accession number GSE50203). Combining with other public mi - croarray databases, we identified 37 genes in the reactive oxygen species (ROS) system that were differentially expressed in the injured zebrafish heart at 7 days post amputation (dpa) (Supplementary information, Table S4), when injury-induced inflammation started to fade and active myocardial regeneration began [3, 28, 29]. RNA in situ hybridization was then applied at multiple time points from 3 to 30 dpa for validation. We found that duox was particularly interesting because of the spatial and temporal patterns of its expression during regeneration. RNA in situ hybridization with two inde - pendent duox probes revealed that duox expression was weak at sham, peaked and was highly localized beneath the surface of the amputation at around 3, 7 and 14 dpa, and returned to the basal level at 30 dpa when regener - ation was nearly complete (Supplementary information, Figures S3A-S3E and S5A-S5E). Each of the two sense duox probes had no detectable signals (Supplementary information, Figures S4 and S5F), indicating the speci - ficity of the two duox probes. RNA in situ hybridization on Tg( tcf21 :nucEGFP) transgenic hearts revealed that duox expression largely overlapped with the EGFP signal (Figure 1A-1C and Supplementaryinformation, Figure S6), which primarily labels the epicardium [30], sug - gesting that duox was enriched in the epicardium. Thus, both the time course and epicardial localization of duox expression correlated with those of active myocardial re - generation after injury.\nSystematic examination revealed that the expression of nox2 , but none of the other nox genes, was moderately induced at 3, 7 and 14 dpa in a pattern similar to that of duox (Supplementary information, Figure S7). RT-PCR confirmed expression of duox and nox2 in 14 dpa hearts (Supplementary information, Figure S3F). These results indicate that the expression of duox and nox2 , with duox being dominant, is spatially and temporally associated with myocardial regeneration after ventricular resection.\nH 2 O 2 signal is essential for heart regeneration To visualize the spatiotemporal dynamics of H 2 O 2 in the intact heart during regeneration, we used Tol2-based transgenesis and zebrafish myosin light chain 7 ( myl7 ) promoter to generate transgenic zebrafish with cardi - ac-specific expression of Hyper (Figure 1D), a fluores - cent protein-based H 2 O 2 sensor [25, 31]. High-resolution confocal imaging was performed in intact adult hearts excised from Tg( myl7 :Hyper) transgenic animals, and H 2 O 2 levels were directly measured as the ratio between the fluorescent signals dually excited at 488 and 405 nm ( R = F 488 / F 405 ). By ex vivo titration on the Hyper-ex - pressing hearts freshly excised from uninjured animals, we obtained a calibration curve (Figure 1K and Supple - mentary information, Figure S9A-S9G) similar to that obtained in vitro [31]. Compared to sham controls (Figure 1E), local H 2 O 2 concentration was clearly elevated in injured hearts at 3, 7 and 14 dpa (Figure 1F-1G and 1I), and returned to the basal level after nearly full regenera - tion at 30 dpa (Figure 1H and 1I). In contrast to the rapid (on the scale of minutes) inflammatory H 2 O 2 response in the tail-fin epithelium of zebrafish larvae [25, 26], there was a slow onset of ventricular resection-induced H 2 O 2 production, as no detectable increase in the Hyper fluorescence ratio occurred within ~1 h after heart injury (data not shown). As seen on confocal optical sections (Figure 1E-1H) and from line plots of the spatial profiles (Figure 1J), the H 2 O 2 gradients spanned ~20 µm depth from the epicardium to the myocardium, reaching a peak concentration of ~30 µM (Figure 1K and Supplementa - ry information, Figure S9A-S9G), which is within the range of concentrations for H 2 O 2 to act as an intracellular messenger [32]. In an independent measurement, we stained 14 dpa Tg( myl7 :Hyper) transgenic hearts with Redox sensor cc-1 and visualized the H 2 O 2 signal in both the epicardium that lacked Hyper expression (Fig - ure 1O) and adjacent compact myocardium (Figure 1L- 1O). Therefore, direct visualization of H 2 O 2 with either a genetically encoded probe or a small-molecule indicator revealed that H 2 O 2 was elevated primarily in the epicar - dium and the adjacent myocardium at the resection site, highly correlated with myocardial regeneration.\nTo investigate whether there is a causal relationship between elevated Duox/Nox2 and H 2 O 2 production and cardiac regeneration, we applied diphenyleneiodoni - um (DPI) (10 µM) or apocynin (100 µM), widely used inhibitors of Duox/Nox enzymes [33] and examined subsequent changes in H 2 O 2 production and cardiac re- generation. We found that Hyper fluorescent signals were decreased by 50% within ~ 40 min after soaking 14 dpa Tg( myl7 :Hyper) hearts in DPI-containing medium (Fig - ure 2A-2B), while the Hyper signals remained stable in control medium containing DMSO only (Figure 2B and Supplementary information, Figure S8A), suggesting that high concentration of H 2 O 2 likely resulted from elevated duox/nox2 expression but was not produced from mito - chondria. Functionality of ex vivo cultured hearts was validated by their rhythmic contractions under 1 Hz elec - trical field stimulation after 1 h culture (Supplementary information, Figure S8B-S8C). Based on the previous re - port that myocardial cells enter mitotic phases in injured areas of the heart upon ventricular resection in zebrafish or myocardial infarction in mice [3, 34], we quantified injury-induced myocyte proliferation by counting BrdU + / Mef2C + double-positive myocytes during heart regener - ation. After ventricular resection, zebrafish were allowed to recover for 7 days and BrdU was then co-injected with either DPI or apocynin into the abdominal cavity once daily until the hearts were collected for examination. Compared with sham-operated controls, we found that BrdU + /Mef2C + myocyte proliferation was induced in in - jured hearts at 14 dpa (Figure 2C, 2C’ and 2E), but their numbers were significantly reduced by treatment with either DPI (Figure 2D, 2D’ and 2E) or apocynin (Figure 2E and data not shown). In addition, extensive cardiac fibrosis (Figure 2G) and reduced Myl7 + myocardial re- generation were evident in injured hearts with DPI (Figure 2I) or apocynin treatment (data not shown), compared with DMSO-treated hearts (Figure 2F and 2H). Although injection of H 2 O 2 did not induce myocyte proliferation in sham-operated animals (Supplementaryinformation, Fig - ure S10), it rescued the inhibitory effects of DPI on the numbers of BrdU + /Mef2C + myocytes and Gata4 + cells (Supplementary information, Figure S11). Importantly, DPI-treated hearts failed to regenerate even at 60 dpa (Supplementary information, Figure S12), suggesting a permanent arrest rather than a delay of regenerative pro - cess. In addition, exogenous H 2 O 2 partially rescued the inhibitory effect of DPI on embryonic heart regeneration (Supplementary information, Figure S13A and S13B) af - ter a large number of cardiomyocytes were damaged by incubating Tg( myl7 :CFP-NTR) transgenic embryos with metronidazole (Mtz), as Mtz can be converted into a tox - ic DNA crosslinking agent by nitroreductase (NTR) as\npreviously described [35]. We further validated the pene - tration of H 2 O 2 by imaging the 72 hours post fertilization (hpf) embryonic Tg( myl7 :Hyper) hearts and found that Hyper signals were increased upon elevated external H 2 O 2 concentration (Supplementary information, Figure\nS14). These results strongly support the hypothesis that H 2 O 2 is required and sufficient for cardiac regeneration. If H 2 O 2 is causally linked to cardiac regeneration as suggested by its inhibition upon treatment with DPI or apocynin, limiting H 2 O 2 signaling by augmenting an -\nFigure 1 Epicardial Duox-generated H 2 O 2 associated with myocardial regeneration. (A - C) After imaging of Tg( tcf21 :nucEG- FP), we performed RNA in situ hybridization with duox probes on 10- µ m cryosections of Tg( tcf21 :nucEGFP) hearts at sham, 7 and 14 dpa. Note the induced expression of duox and its colocalization with epicardial reporter Tg( tcf21 :nucEGFP) in in- jured areas (arrowheads) at 7 and 14 dpa (B , C) while little staining at sham (A) . Scale bar for A - C , 50 µ m. (D) Cardiac-spe- cific expression of Hyper in a zebrafish embryo. Hyper was restricted in the Tg( myl7 :Hyper) transgenic heart at 48 hpf. (E - H) Ex vivo Hyper heart images from sham control (E) , 7 dpa (F) , 14 dpa (G) and 30 dpa (H) . Spatially resolved H 2 O 2 image, in- dexed by the ratio between the F 488 and F 405 images of Hyper (left panels), is presented in pseudocolor. (I) Ratiometric Hyper signals ( F 488 / F 405 ) averaged over the regenerative zone of injured heart during the first month of regeneration after resection.\nn = 3. (J) Representative transmural spatial profiles of the Hyper signal at 14 dpa (red line) and sham (green line) hearts. (K) Ex vivo calibration of Hyper F 488 / F 405 ratio as a function of ambient H 2 O 2 concentration. n = 3. Arrow denotes the average F 488 / F 405 ratio seen in regenerative zones at 14 dpa. (L - O) Redox signal showing greater epicardial and myocardial oxidization during regeneration. A 14 dpa Tg( myl7 :hyper) heart was labeled with Redox sensor cc-1. Images show Hyper signals at 488 (L) and 405 nm (M) , and Redox sensor cc-1 signals at 555 nm excitation (N) , and the merging of L and N (O) . Note that the Re- dox sensor cc-1 signal (arrowheads), indicative of intracellular oxidization, was most conspicuous in epicardial cells lacking Hyper expression, and also overlapped with the Hyper signal (arrows) in adjacent myocardium. Scale bars, 100 µ m.\nFigure 2 H 2 O 2 signaling is required for heart regeneration. (A - B) Inhibiting Duox/Nox NADPH oxidases by DPI decreased H 2 O 2 generation. Time-lapse confocal images (A) and statistics (B) of the Hyper ratio in Tg( myl7 :Hyper) hearts prior to and after application of DPI (10 µ M). DMSO (0.1%) was used as control. n = 3-5. (C - E) Treatment with DPI or apocynin impaired cardiac myocyte regeneration. Proliferating myocytes were identified by double-staining with anti-Mef2C (red) and anti-BrdU (green) (C’ , D’ , arrows ) . Note that there were fewer double-positive cells (yellow) in DPI-treated heart (D , D’) than in DMSO control heart (C , C’) . Quantitative results with sham, DMSO, DPI or apocynin treatment are shown in E . n = 5 to 7. See Mate- rials and Methods for details of treatment. (F - I) Accumulated fibrin/collagens (white arrowheads in G , Masson’s staining) and compromised myocardial regeneration (white arrowhead in I , in situ hybridization with myl7 probes) after DPI treatment, com- pared with DMSO control (F , H) . (J - M) Cardiac-specific overexpression of catalase retarded heart regeneration. Tg( myl7 :- Catalase-DsRed) heart displayed larger amounts of fibrin/collagens (white arrowheads in K , Masson’s staining) and compro- mised myocardial regeneration (white arrowheads in M , in situ hybridization with myl7 probes), as compared with respective non-transgenic sibling hearts at 30 dpa (J , L) . n = 5-8. Scale bars, 100 µ m.\ntioxidant capacity of the heart should decrease cardiac regeneration. To directly test this hypothesis, we gener - ated Tg( myl7 :catalase-DsRed) transgenic zebrafish with cardiac-specific expression of catalase. The efficacy of catalase-DsRed was validated by its ability to scavenge H 2 O 2 [36] (Supplementary information, Figure S13C- S13E). Similar to the Duox/Nox inhibition, there were fewer Myl7 + regenerated myocytes (Figure 2M), and extensive cardiac fibrosis (Figure 2K) at 30 dpa in cat - alase-transgenic hearts, compared to non-transgenic sibling hearts (Figure 2L and 2J). However, we were not able to generate ubiquitously heat-inducible Tg( hsp70 :- catalase) transgenic lines, due to lethality of these trans - genic embryos from leaky expression of catalase without heat shock. Moreover, cardiac-specific overexpression of catalase-DsRed suppressed embryonic heart regener - ation in Mtz-induced heart injury of Tg( myl7 :CFP-NTR) transgenic embryos (Supplementary information, Figure S13F-S13H). Previous studies suggest that myocardial dedifferentiation is a critical step for heart regeneration\nin zebrafish [8, 9]. Interestingly, inhibiting H 2 O 2 signaling by either DPI treatment or catalase overexpression had no effects on myocardial sarcomere disassembly, an indi - cator of myocardial dedifferentiation, in injured hearts at 14 dpa (Supplementary information, Figure S15). These results suggest that the high concentration of H 2 O 2 , likely caused by elevated duox/nox2 expression, is obligatory for cardiac regeneration after injury without affecting in - jury-induced myocardial dedifferentiation.\nThe Duox-H 2 O 2 -Dusp6 pathway primes myocardial re- generation by elevating Erk1/2 activity via a derepres- sion mechanism Because developmental signaling pathways of FGF and PDGF are reused during adult zebrafish heart re - generation [20, 21], we investigated whether H 2 O 2 mod- ulated the common MAP kinase pathway downstream of the growth factors. We found that phosphorylation of Erk1/2 (pErk) was increased in injured hearts at 3, 7 and 14 dpa (Figure 3C and Supplementary information,\nFigure 3 H 2 O 2 activates Erk1/2 for myocardial regeneration. (A - B) Phosphorylation of Erk1/2 (pErk1/2) in epicardial cells (A , A’) in 14 dpa heart, as detected by anti-pErk1/2 antibody. Decrease of pErk1/2-positive cells by DPI treatment (B , B’) . Scale bar for A and B , 100 µ m. (C) Western blot showing greater pErk1/2 in injured hearts at 3, 7 and 14 dpa, and its blockade by DPI treatment at 14 dpa. (D-F) Gata4 + myocardial cells in injured areas in control Tg( gata4 :EGFP) heart (DMSO treatment, D , D’ , D’’ ) and their disappearance after inhibition of Duox (DPI treatment, E , E’ , E” ), or Mek1/2 (U0126 treatment, F , F’ , F” ). Car- diomyocytes were co-labeled with anti-EGFP and anti-MF20 antibodies and nuclei were labeled by DAPI. Scale bar for D - F , 100 µ m. (G) Statistics of Gata4 + cardiomyocytes at 14 dpa.\nFigure S16), with the pErk1/2 + cells primarily located in non-myocardium such as the epicardium (Figure 3A and 3A’), evidenced by overlapping with epicardial Raldh2, a molecular marker for both the epicardium and endocar - dium (Supplementary information, Figure S18A-S18C). The total erk1 and erk2 mRNA, as well as total Erk pro - teins were slightly increased during heart regeneration (Figure 3C and Supplementary information, Figure S17). Using Tg( gata4 :EGFP) transgenic reporter zebrafish to track newly regenerated myocardium in injured hearts [8], we found that decreasing pErk1/2 with U0126 (a Mek1/2 inhibitor) interfered with the regeneration of myocardial cells (Figure 3F, 3F’ and 3G), as evidenced by reduced EGFP-positive cells compared with DMSO controls (Figure 3D, 3D’ and 3G). More importantly, the formation of pErk1/2 + cells (Figure 3B, 3B’ and data not shown) and the generation of GATA4-EGFP + myocardial cells (Figure 3E, 3E’, 3G and data not shown) were sen - sitive to inhibition of Duox/Nox2 by treatment of either DPI or apocynin. Unfortunately, strong autofluorescence in Tg( myl7 :catalase-DsRed) prevented us from perform - ing any fluorescence-based immunostaining. These re - sults showed that epicardial Erk1/2- and H 2 O 2 -dependent signals are interlinked in promoting regeneration in the adult zebrafish heart.\nOf the potential redox-sensitive targets of H 2 O 2 , the dual-specificity MAPK phosphatase 6 (Dusp6) is dis - tinct in that it is stimulated by pErk1/2, but reciprocally dephosphorylates and deactivates pErk1/2, forming a negative feedback that limits Erk1/2 activity. Previous reports have shown that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, and that oxidized Dusp6 is prone to degradation, leading to aberrant Erk1/2 activation [37, 38]. Dusp6-knockout mice show greater proliferation of myocytes [39], but the function of Dusp6 in cardiac regeneration is unknown. We thus formulated a hypothesis with two seemingly paradoxical predictions. First, Erk1/2 activation would elevate Dusp6 expression during regeneration, activating a negative feedback (i.e., a repression). However, if H 2 O 2 can oxidize and thereby destabilize Dusp6, ventricular resection-induced H 2 O 2 production would unleash the pro-regenerative pErk1/2 signal (i.e., a derepression).\nTo test the above predictions, we examined Dusp6 expression at both the mRNA and protein levels. Upon injury, dusp6 mRNA level was elevated, peaking at 7 dpa (Figure 4A and Supplementary information, Figure S19), and Dusp6 protein levels were increased (Figure 4B) and Dusp6 was particularly localized in the epicar - dium at 7 and 14 dpa (Figure 4D-4E and Supplementary information, Figures S18D-S18F and S20B-S20D) and decreased at 30 dpa (Figure 4F and Supplementary in -\nformation, Figure S20G). The most intriguing finding is that in injured hearts at 14 dpa after DPI or apocynin treatment, Dusp6 protein, but not dusp6 mRNA, was dramatically further increased in both the epicardium and myocardium with clear nuclear enrichment (Figure 4A, 4B, 4G and 4H), unmasking an inhibitory effect of H 2 O 2 on Dusp6 protein stability in the epicardium and myocardium undergoing active regeneration. We also confirmed this finding on Tg( dusp6 :EGFP), where EGFP expression was previously shown to recapitulate the endogenous dusp6 mRNA expression pattern during embryogenesis [40]. Epicardial EGFP expression was evident in injured areas of adult hearts at 14 dpa (Figure 4I-4J). Furthermore, pErk1/2 was largely co-localized with Dusp6 (Supplementary information, Figure S21C) and inhibiting pErk1/2 by U0126 reduced Dusp6 expres - sion (Supplementary information, Figure S21A-S21D), suggesting a close interlink between them. These data substantiate the idea that Dusp6 mRNA and protein are initially induced as a negative feedback for pErk1/2, but subsequently H 2 O 2 , likely produced by Duox/Nox2, de - stabilizes Dusp6 to impart a derepression mechanism by suppressing this negative feedback in the epicardium.\nTo further evaluate the role of the Duox-H 2 O 2 -Dusp6 derepression signaling pathway in heart regeneration, we used the small molecule BCI, recently developed in a transgenic zebrafish chemical screen [40], which directly inhibits the phosphatase activity of Dusp6. We found that BCI counteracted the inhibitory effect of DPI or apocynin on the generation of the double-positive BrdU + /Mef2C + myocytes, but had no effects on DPI- or apocynin-enhanced cardiac fibrosis at 14 dpa (Figure 4K, 4M, 4N, 4O and Supplementary information, Figure S24A-S24C). Moreover, BCI also reversed the effects of DPI or apocynin on MF20 staining (which indicates myocardial regeneration), and cardiac fibrosis at 30 dpa (Supplementary information, Figure S24D-S24I). These results indicate that BCI mimics the endogenous H 2 O 2 signal when the latter is abrogated. However, in hearts untreated with DPI or apocynin, BCI did not alter the generation of BrdU + /Mef2C + myocytes at 14 dpa (Fig - ure 4K, 4L and 4O), consistent with the idea that BCI is redundant in the presence of an intact endogenous H 2 O 2 signal. Like exogenous H 2 O 2 (Supplementary informa - tion, Figure S10), BCI had no effects on myocardial proliferation in sham-operated hearts (Supplementary information, Figure S23), which is consistent with the relatively low levels of Dusp6 proteins in sham hearts (Figure 4C and Supplementary information, Figure S20A). To examine whether H 2 O 2 is involved in coronary vessel regeneration, we used the Tg( flk1 :EGFP) zebrafish to visualize coronary vessels in the adult heart. We found\nMef2C + proliferating myocytes. Scale bars, 100 µ m.\nthat DPI treatment reduced the amount of Flk1 + vessels, which was rescued by co-injection of BCI (Supplementa - ry information, Figure S22), suggesting an additional role of H 2 O 2 in coronary vessel repair upon ventricular resec - tion. Collectively, interventions at the levels of Duox/\nNox2, H 2 O 2 and Dusp6 corroborated the hypothesis that the Duox-H 2 O 2 -Dusp6 pathway plays an essential role in priming myocardial and blood vessel regeneration, by elevating Erk1/2 activity via a derepression mechanism.\nTo directly test whether zebrafish Dusp6 (zDusp6) is\nsubjected to H 2 O 2 -dependent ubiquitination, we mea - sured the ubiquitinylated zDusp6 (Ub-zDusp6) protein levels with or without H 2 O 2 stimulation in HeLa cells and zebrafish embryos, given that adult zebrafish heart is hardly accessible for these molecular and biochemical methods. We found that exogenous H 2 O 2 treatment de- creased, in a dose-dependent manner, the levels of Myc- tagged zDusp6 proteins in HeLa cells overexpressing zDusp6-Myc (Figure 5A and 5B). Similarly, endogenous zDusp6 underwent degradation in zebrafish embryos after H 2 O 2 treatment (Figure 5C and 5D). Furthermore, Ub-zDusp6 was hardly detectable, but was detected after treatment with 1 mM H 2 O 2 in HeLa cells overexpressing zDusp6-Myc and Flag-tagged ubiquitin (Ub-Flag) (Figure 5E), suggesting the existence of unstable Ub-zDusp6 that is dependent on H 2 O 2 . Proteasome inhibitor MG132 in - creased the amount of Ub-zDusp6 in the absence or pres - ence of H 2 O 2 , supporting the notion that H 2 O 2 treatment enhances the ubiquitination of zDusp6. On the other hand, we failed to detect Ub-zDusp6 in the presence of H 2 O 2 and MG132 in zebrafish embryos overexpressing zDusp6-Myc and Ub-Flag (data not shown), suggesting that Ub-zDusp6 is highly unstable in zebrafish embryos.\nTherefore, H 2 O 2 regulates the ubiquitination and protea - some degradation of zebrafish Dusp6.\nH 2 O 2 promotes heart regeneration independent of im- mune cell recruitment Previous studies have shown that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail-fin epithelium of zebrafish larvae [25, 26]. To differentiate potentially dual function of H 2 O 2 in recruiting inflamma - tory cells and priming heart regeneration, we examined temporal and spatial distribution of immune cells during heart regeneration by labeling macrophages and neu - trophils with either anti-lymphocyte cytosolic protein 1 (Lcp1) or Tg( coro1a :EGFP) [41]. We found that Lcp1 + leukocytes appeared around 3 dpa, peaked at 7 and 14 dpa, decreased at 19 dpa and almost disappeared at 24 dpa (Figure 6A-6F), and the recruitment occurred in a H 2 O 2 -dependent manner (Figure 6G and 6I). Quantifi - cation of Lcp1 + leukocytes during heart regeneration in Figure 6 was included in Supplementary information, Figure S25. Similar dynamics of leukocyte recruitment was obtained by labeling leukocytes with an independent Tg( coro1a :EGFP) transgenic reporter (Supplementary\nFigure 5 H 2 O 2 induces ubiquitination and proteasome degradation of zebrafish Dusp6. (A - B) Exogenous H 2 O 2 decreased the expression level of Myc-tagged zDusp6 in a dose-dependent manner in HeLa cells as evidenced by western blotting (A) . The statistics of A is shown in B . β -actin was used for normalizing protein loading. (C - D) Exogenous H 2 O 2 decreased the ex- pression level of endogenous zDusp6 in a dose-dependent manner in zebrafish embryos at 24 hpf (C) , and the statistics of C is shown in D . β -actin was used for normalizing protein loading. (E) Ub-zDusp6 (arrowhead) was detected after H 2 O 2 and/ or  MG132  treatments  in  HeLa  cells  overexpressing  Myc-tagged  zDusp6  and  Flag-tagged  ubiquitin  (Ub-Flag).  Myc-tagged Dusp6 proteins were detected for all groups except the group without zDusp6-Myc overexpression.\ninformation, Figure S26A-S26F). Furthermore, DPI treatment either from 7 to 14 dpa (when immune cells peaked, DPI timely inhibited leukocyte recruitment) or from 14 to 30 dpa (when immune cells gradually disap - peared, DPI should not affect normal leukocyte recruit - ment) led to increased cardiac fibrosis and compromised myocardial regeneration (Figure 6L and 6N), suggesting that both the early- (7-14 dpa) and late-phase H 2 O 2 pro- duction (14-30 dpa) are required for cardiac regeneration. Importantly, co-injection of BCI with DPI or apocynin largely restored myocardial regeneration (Figure 6M and 6O), but failed to rescue immune cell recruitment (Figure 6H, 6J and Supplementary information, Figure S26G-S26H), indicating that Dusp6 inhibition by BCI suffices to initiate regeneration independently of immune cell recruitment. Taken together, these data suggest that while H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration, the two pathways are apparently separable and relatively independent of each other.\nDusp6 is a repressor for zebrafish heart regeneration To directly examine the role of dusp6 in heart regen- eration, we established Tg( hsp70 :dusp6-His) transgenic zebrafish, where overexpression of Dusp6 in both epicar - dium and myocardium was conditionally driven by heat shock [21]. Transgenic expression of dusp6 was con - firmed to decrease pErk in both embryos and adult hearts (Figure 7A). Compared with non-transgenic siblings (Figure 7B, 7F, 7H, 7J and 7L), injured transgenic hearts displayed fewer Mef2C + /BrdU + proliferating myocytes (Figure 7C-7D), Gata4 + myocytes (Figure 7E and 7G) as well as pERK + cells (Figure 7I) at the injury site, leading to increased cardiac fibrosis (Figure 7K) and diminished cardiac regeneration (Figure 7M). Together with the BCI data (Figures 4 and 6), these results establish a repressive role of dusp6 in heart regeneration, acting through de- phosphorylation of pErk in the epicardium.\n\n\nDiscussion\nIn summary, we have presented three major inter - related findings. First, upon injury, H 2 O 2 is produced, likely by Duox/Nox2, up to 30 µM in the epicardium and adjacent myocardium during the entire course of active myocardial regeneration. Second, H 2 O 2 is required for promoting cardiac myocyte proliferation and suppress - ing fibrosis. Third, the pro-regeneration effect of H 2 O 2 is mediated through destabilizing the redox-sensitive and ubiquitin-conjugated phosphatase Dusp6, which provides a derepression of the common MAP kinase signaling. Furthermore, we have identified BCI as the first small\nmolecule for promoting cardiac regeneration when H 2 O 2 generation was defective. In spite of previous findings that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail- fin epithelium of zebrafish larvae [25, 26], our data show that the H 2 O 2 -guided inflammatory response occurs around 3 dpa in adult zebrafish hearts (Figure 6 and Sup - plementary information, Figure S26). This time course, when compared with that of Duox/Nox2 induction and H 2 O 2 production, supports the idea that it likely involves de novo synthesis of duox transcripts and possibly pro - teins for the production of H 2 O 2 and subsequent recruit - ment of leukocytes in heart regeneration. In contrast, because the injured larvae tail fin is directly exposed to the environment, it needs immediate immune defense after injury and therefore only takes 3 min to recruit neu - trophils to the injured site, through activation of Duox without transcription and translation in the tail fin epithe - lium [25, 26]. In addition to its function in recruiting in - flammatory leukocytes, our data suggest that H 2 O 2 is also required for promoting adult heart regeneration, which is consistent with previous reports on H 2 O 2 function in promoting injury-induced peripheral sensory axon regen - eration in zebrafish larvae [42] and tail regeneration in Xenopus tadpole [43]. H 2 O 2 acts through a redox sensor Lyn to mediate injury-induced leukocyte wound attrac - tion in zebrafish larvae tail fin [25, 26], but direct targets of H 2 O 2 in zebrafish sensory neurons or Xenopus tadpole tail epithelium are not known [42, 43]. Our data show that H 2 O 2 not only destabilizes Dusp6 to elevate growth factor signaling in promoting adult heart regeneration, but also acts through an unknown, independent mech - anism to recruit inflammatory leukocytes during heart regeneration. Future studies need to examine whether H 2 O 2 acts through Lyn or other redox sensors to induce inflammatory leukocyte recruitment during adult heart regeneration.\nGrowth factor signaling pathways such as FGF and PDGF pathways are essential for adult heart regenera - tion [20, 21]. In contrast to previous reports on a neg - ative role of p38 MAPK in mammalian and zebrafish heart regeneration [44,45], our data support that H 2 O 2 in the epicardium (Figure 1) is essential for elevating phosphorylation of Erk1/2 MAPK through repression of Dusp6 during zebrafish heart regeneration (Figures 3-5 and 7). Our data are aligned well with the previous re - ports that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, leading to ubiquitination and subsequent proteasome degradation of Dusp6 and thus aberrant Erk1/2 activation [37, 38] (Figure 5). The elevated pErk1/2 in the epicardium might then activate soluble factors at transcriptional and/or post-transcrip -\ntional levels, which in turn bind to their receptors in the myocardium and promote myocardial proliferation and regeneration. However, we failed to detect the changes in Dusp6 and pErk protein levels in the heart by west - ern blotting (data not shown). It is likely that the subtle changes in Dusp6 and pErk protein levels in the epicardi - um of injured hearts might not be well detected by west -\nFigure 6 H 2 O 2 has dual function in recruiting Lcp1 + leukocytes and promoting heart regeneration. (A - F) Immunostaining of Lcp1 showing that leukocytes (macrophages and neutrophils) were undetectable at sham (A) , appeared at 3 dpa (B) , peaked between 7 and 14 dpa (C , D) and gradually disappeared from 19 to 24 dpa (E , F) . Either apocynin (Apo) (G) or DPI (I) inhib- ited leukocyte recruitment, which could not be rescued by BCI (H , J) at 14 dpa. Myocytes were labeled by anti-MF20, and nuclei were labeled by DAPI. (K - M) All AFOG staining analyses were performed at 30 dpa and DPI and/or BCI treatment was applied as indicated. Myosin was visualized as orange, fibrin as red and collagen as bright blue. AFOG staining showed that DPI treatment from 7 to 14 dpa increased cardiac fibrosis (L) , which was rescued by BCI (M) , compared with DMSO control (K) at 30 dpa. (N - O) AFOG staining showed that DPI treatment from 14 to 30 dpa increased cardiac fibrosis (N) , which was rescued by BCI (O) . Scale bars, 100 µ m.\nern blotting. On the other hand, cardiac-specific overex - pression of catalase, the H 2 O 2 -specific scavenger, also led to regenerative defects in the heart (Figure 2J-2M). New transgenic lines, such as Tg( tcf21 :catalase) without autofluorescence or heat-inducible Tg( hsp70 :catalase) without leaky transgene expression are needed for deter - mining the levels of Dusp6 and pErk by immunostaining\nFigure 7 The repressive effects of Dusp6 overexpression on heart regeneration through dephosphorylation of pERK. (A) Top: a scheme showing that in Tg( hsp70 :dusp6-His), His-tagged Dusp6 is driven by the zebrafish hsp70 promoter. Transgenesis was mediated by Tol2 transponase. Bottom: heat shock-induced expression of His-tagged Dusp6 proteins diminished pErk in transgenic embryos (Tg) at 72 hpf or transgenic adult hearts (Tg), compared with their sibling embryos or adult hearts. β -actin was used to normalize protein loading. (B - D) Mef2C + /BrdU + proliferating myocytes (arrowheads) were decreased in dusp6 transgenic hearts (C) compared with those in non-transgenic siblings (B) , with the statistics shown in D . (E - G) Representative images showing that Tg( gata4 :EGFP) myocytes (arrowheads), which colocalized with cardiac marker MF20, were diminished in dusp6 transgenic hearts (G) compared with their siblings (F) , with the statistics shown in E . (H - I) Reduction of pErk + cells (arrowheads), not overlapping with cardiac marker MF20, in transgenic hearts (I) , compared with non-transgenic siblings (H) . (J - M) Transgenic overexpression of Dusp6 led to cardiac fibrosis (assayed by AFOG staining, K ) and compromised cardiac regeneration (assayed by MF20 staining, M ), compared with their siblings (J , L) . Nuclei were co-stained with DAPI. Scale bars, 100 µ m. after scavenging H 2 O 2 by catalase in the future. Based on the data from others and ours, we propose that the working model for cardiac regenerative signaling comprises two main branches: the forward pro-regenera - tive MAPK signaling triggered by FGF and other growth factors, as recently elucidated [20, 21]; and the derepres - sion mechanism through the Duox/Nox2-H 2 O 2 -Dusp6 pathway that converges on Erk1/2 MAPK signaling primarily in the epicardium, which then activate puta - tive soluble factors to promote myocardial regeneration\n(Figure 8). Such epicardial soluble factors might include Wnt1 and IGF2 that were previously shown to play an essential function in the epicardium/cardiac fibroblasts and myocardium [46-49]. During heart regeneration, the growth factor-MAPK signaling (pErk) is gradually en - hanced in injured hearts from 3 to 14 dpa with the peak around 14 dpa, which induces dusp6 expression (positive induction) that in turn dephosphorylates pErk (negative feedback) (Figures 3 and 4). Simultaneously, H 2 O 2 in- creases and destabilizes Dusp6 with the peak from 7 to\nFigure 8 H 2 O 2 promotes heart regeneration through a derepression mechanism in zebrafish. Upon ventricular resection, FGF and other growth factor receptors are activated and resultant phosphorylation of Mek1/2 (pMek1/2) and Erk1/2 (pErk1/2) in- duces expression of dusp6 , which in turn, exerts a negative feedback on pErk1/2 in non-myocardial cells (primarily epicardial cells). Concurrently, injury elicits a time-dependent, enduring localized duox and, to a minor extent, nox2 expression that sup- ports intense H 2 O 2 formation primarily in the epicardium close to the injury site. High-concentration intracellular H 2 O 2 oxidizes and thereby destabilizes the Dusp6 phosphatase, diminishing its inhibition of pErk1/2. Thus, the newly identified Duox/Nox2- H 2 O 2 -Dusp6 signaling serves as a derepression mechanism and, when activated, unleashes the pro-regenerative pErk1/2 signaling likely through generation of soluble factors that would eventually act to enhance myocardial regeneration. DPI and apocynin (Apo), inhibitors of Duox/Nox2; catalase, catalytic scavenger of H 2 O 2 ; BCI, inhibitor of Dusp6; U0126, inhibitor of Mek1/2.\n14 dpa, which results in decreased Dusp6 expresssion and thus increased pErk levels at 14 dpa when myocar - dial regeneration actively takes place. In addition, we have shown that H 2 O 2 /Dusp6 is also involved in coro - nary vessel regeneration upon ventricular resection. This is consistent with previous studies that FGF and PDGF signaling pathways are essential for coronary vessel formation during heart regeneration [20, 21]. Therefore, this work provides the first intracellular H 2 O 2 -Dusp6- pErk mechanisms downstream of growth factor signaling in adult myocardial and coronary vessel regeneration. As there are highly conserved mammalian homologs for all components of the Duox/Nox2-H 2 O 2 -Dusp6-Erk1/2 axis, this derepression mechanism for cardiac regenera -\ntion, as demonstrated here in zebrafish, may also operate in mammals, and if so, future efforts on activating the mammalian counterpart may offer opportunities for ther - apeutic myocardial regeneration in heart failure.\n\n\nMaterials and Methods\n\n\nZebrafish lines\nZebrafish were raised and handled in accordance with the Peking University Institutional Animal Care and Use Commit - tee accredited by the AAALAC International. Wild-type TL, Tg( myl7 :HyPer) pku327 , ( myl7 :catalase-DsRed), Tg( dusp6 :EGFP) and Tg( gata4 :EGFP) transgenic zebrafish lines at 6-12 months of age were used for heart surgery. Several Tg( myl7 :HyPer) pku327 , Tg( myl7 :catalase-DsRed) pku343 , Tg( myl7 :CFP-NTR) pku360 and\nTg( hsp70 :dusp6-His) pku344 transgenic zebrafish lines were gen - erated in TL background by using Tol2-based transgenesis [31, 50]. Tg( tcf21 :nucEGFP) was previously reported [30]. Tg( coro- 1a :EGFP) was provided by Dr Zilong Wen [41], Tg( dusp6 :EGFP) transgenic zebrafish were obtained from Dr Michael Tsang [40] and Tg( gata4 :EGFP) transgenic zebrafish were kindly provided by Dr Todd Evans [51]. The zebrafish myl7 promoter was provided by Dr C Geoff Burns [52]; pHyPer-Cyto cDNA clone was purchased from Evrogen (Cat No. FP941); zebrafish catalase cDNA (CDS) was isolated from 2-day embryos according to catalase sequence (NM_130912); rat Dusp6 cDNA was obtained from Dr Zhi-Xin Wang [53]; and CFP-NTR was provided by Dr Ling-Fei Luo [35]. These expression constructs were cloned into the Tol2 backbone [50]. Each of the transgenic lines made in our laboratory was char - acterized and/or confirmed by more than two independent found - ers.\nMtz-induced embryonic heart injury in Tg(myl7:CFP-NTR) embryos A large number of cardiomyocytes were damaged by incubat - ing Tg( myl7 :CFP-NTR) transgenic embryos with Mtz essentially as previously described [35]. Briefly, transgenic embryos at 70 hpf were incubated with 2 mM Mtz, and Mtz was then washed off and DPI and/or H 2 O 2 were added at 72 hpf. In this condition, about 80% control transgenic hearts restored normal morphology and function at 124 hpf.\n\n\nAdult zebrafish heart resection\nA ventricular resection procedure was carried out essentially as described by Poss and colleagues [3]. Briefly, adult zebrafish were anesthetized with tricaine and placed with ventral side up into the groove of a sponge. The pericardial sac was exposed by removing surface scales and a small piece of skin with iridectomy scissors. Opening the pericardial sac was aided by tweezers, and the apex of the ventricle was gently pulled up and removed with Vannas scissors. A small piece of wet tissue was placed on the top of the ventricular wound. The heart stopped bleeding very quickly (within 10-15 s) due to the formation of blood clots. The fish was placed back into a water tank, where water was puffed over the gills with a plastic pipette until the fish was breathing and swimming regu - larly. The surface opening sealed automatically within a few days.\nLibrary preparation for a strand-specific poly(A)-positive RNA-seq Sham and 7 dpa hearts were freshly excised and rinsed with pre-cooled PBS. The outflow tract and atrium were removed im - mediately. The total RNA of each group was extracted from about 25 hearts by using QIAGEN RNeasy Microarray Tissue Mini Kit (Cat No. 73304). Poly(A)-positive RNA was purified from 5 µg of total RNA (RNA integrity number ≥ 7.5) with the Dynabeads mRNA purification kit (Invitrogen), following the manufacturer’s instructions. A strand-specific RNA-seq library was then prepared according to the deoxy-UTP approach as reported previously [54]. Amplified materials were loaded onto a flow cell and sequencing was carried out on the Illumina HiSeq2000 platform by running 90 cycles (paired-end design) according to the manufacturer’s instruc - tions.\nRNA-Seq and microarray data analysis and mining Paired-end mRNA sequence tags were mapped on the Zebrafish\nGenome (Zv9) by Tophat (v1.4.1). Multiple alignment reads were discarded. The RPKM scores for each gene were calculated as pre - viously described [55]. A series of Perl (v5.12.2) and R (v2.13.1) scripts were implemented to evaluate the quality of the RNA-seq experiments. Overall, high-quality RNA-seq data were obtained with high base quality, unbiased read distribution on transcripts, correct strand information, little DNA contamination and low mu - tation rate across the reads (Supplementary information, Figure S1). Eight hundred and forty-five upregulated genes between sham and 7 dpa hearts were identified using Cuffdiff (1.3.0) (Supple - mentary information, Figure S2), on the basis of three criteria: (1) P (un-corrected) &lt; 0.05; (2) fold change &gt; 2; and (3) RPKM &gt; 0.2 in at least one sample. Protein sequences of upregulated genes were then subjected to KOBAS (v2.0) to identify enriched biological pathways, with the P -value calculated using a hyper - geometric distribution. FDR correction was performed to adjust for multiple testing and pathways with FDR-corrected P -value &lt; 0.05 were considered statistically significantly enriched. The RNA-seq data entitled “Transcriptome analysis of heart regenera - tion in adult zebrafish” were deposited to the GEO repository with accession number GSE50203 (GSM1215619 for sham heart and GSM1215618 for 7 dpa heart).\nA list of previous known ROS genes were manually selected and evaluated for their relative expression levels between sham and 7 dpa hearts, based on a publicly available microarray of ze - brafish heart regeneration [29] and the above RNA-seq data on ze - brafish heart regeneration. We found that 23 of them were changed between sham and 7 dpa hearts in the microarray data that were expressed as log (7dpa_log2FC_Belmonte), and 32 of them were changed between sham and 7 dpa hearts in our RNA-seq data that were expressed as log (7dpa_logFC_RNAseq) in Supplementary information, Table S4. We wrote in-house R scripts to analyze the microarray data, using package Limma from Bioconductor [56].\n\n\nEx vivo intact heart imaging\nAdult Tg( myl7 :Hyper) hearts were removed and rinsed in mod - ified Tyrode’s solution (in mM: 150 NaCl, 5.4 KCl, 1.5 MgSO 4 , 0.4 NaH 2 PO 4 , 2 CaCl 2 , 10 glucose, 10 HEPES, pH 7.4). The atrium was removed to diminish spontaneous beatings. Hearts were viable for up to 1 h in such conditions, manifested by responses to 1 Hz electric field stimulation. Viability of the hearts was confirmed by their rhythmic contractions upon stimulation. A high speed camera was used to capture the contractions of the hearts and then kymo - graph was generated from the movies. Hearts were imaged under the 20× water objective lens of a Zeiss LSM700 upright confocal microscope with 405- and 488-nm lasers for dual excitation of Hyper, and 505-565 nm emission lights were collected with band- pass filter. For calculating Hyper 488/405 ratio, F 488 and F 405 im- ages were smoothed, background subtracted and divided. For the Hyper ratiometric calibration curve, Hyper transgenic hearts were treated and imaged ex vivo under designated concentrations (5, 10, 20, 50, 100 and 200 µM) of H 2 O 2 for 2 min. For time-lapse record - ings with 10 µM DPI or DMSO treatment, Hyper transgenic hearts were imaged at 30-s intervals for 3 min in normal Tyrode’s solu - tion before application of DPI or DMSO; the total recording time was 45 min. To detect redox signals with chemical indicators, 30 µM of Redox sensor cc-1 (R-14060; Invitrogen) was injected into the thoracic cavity of 14 dpa Tg( myl7 :Hyper) or wild-type zebraf - ish 30 min before heart isolation. Redox sensor cc-1 was excited at 555 nm. Image processing and data analysis were performed using\nthe IDL program (Research Systems).\n\n\nBrdU incorporation and small molecule treatment\nAfter ventricular resection, zebrafish were allowed to recover for 7 days and 50 µl of 2.5 mg/ml BrdU (B5002; Sigma) was then co-injected with 10 µM DPI (D2926; Sigma), 100 µM apocynin (W508454; Sigma), 10 µM BCI (B4313; Sigma), or 10 µM U0126 (U120; Sigma) into the thoracic cavity once daily until the hearts were collected. In pilot experiments, BCI was kindly provided by Dr Michael Tsang of the University of Pittsburg [40].\nWestern blotting and immunoprecipitation Zebrafish tissues were treated in cell lysis buffer (P0013; Bey - otime) in the presence of a protease inhibitor cocktail (P8340; Sig - ma) and PhosSTOP phosphatase inhibitor cocktail (04906845001; Roche). Proteins were quantified with a BCA kit (PA115; Tiangen Biotech), resolved on 12% SDS-PAGE gels, and transferred onto PVDF membranes (162-0177; Bio-Rad). Membranes were in - cubated with the appropriate primary antibody followed by the incubation with the corresponding HRP-conjugated secondary antibody, and proteins were detected with ECL western blotting substrates (W1001; Promega).\nHeLa cells were grown in DMEM high glucose medium (Ther - mo HyClone, SH30022.01B) containing 10% FBS (Thermo Hy- Clone, SV30087.02). The zDusp6 expression construct pcDNA4- zDusp6-Myc and the Flag-tagged ubiquitin expression construct pcDNA-Ub-Flag [57] were cotransfected into HeLa cells using Lipofectamine 2000 (Invitrogen, 11668). Cells were collected and lysed using RIPA buffer (Applygen, C1053) with 0.1 mM PMSF, 5 mM EDTA and 1 mM Complete TM protease inhibitors (Roche, 04693132001) in the presence or absence of 10 mM MG132 (Sig - ma, M7449). Cell lysates were then incubated with rabbit anti-Myc mAb (Cell Signaling, 2278) at 4 °C overnight. After incubation, 40 µl of Protein A+G beads (Abmart, A10001) was added to each sample and the mixture was rotated at 4 °C for 6 h. After centrif - ugation, the beads were washed five times with RIPA buffer and then boiled for 10 min with 30 µl of 2× sample buffer. Mouse an - ti-Flag antibody (Sigma, F1804) and mouse anti-Myc (HRP) mAb (Cowin Bioscience, CW0300) were used for immunoblotting.\nIn situ hybridization, immunostaining and histological methods Adult hearts were removed and fixed in 4% paraformaldehyde at room temperature for 2 h, and then went through an ethanol series for dehydration followed by paraffin embedding and sec - tioning (4 µm). Two duox cDNA clones were isolated from an embryonic zebrafish cDNA library made in Xiong’s lab, and the primer sequences are shown as duox 1F and 1R, or duox 2F and 2R in Supplementary information, Table S5. Duox clone 1 and duox clone 2 cDNAs were used to make two independent duox probe 1 (Supplementary information, Figure S3) and duox probe 2 (Supplementary information, Figure S5). Dusp6 cDNA was kindly provided by Dr Michael Tsang [40] and tcf21 cDNA by Dr Fabrizio Serluca. In situ hybridization on paraffin sections was performed using digoxygenin-labeled probes. Primary antibodies were anti-BrdU (B8434; Sigma), anti-Mef2c (sc-313; Santa Cruz), anti-tErk (4695; Cell Signaling), anti-pErk (9101; Cell Signaling), anti-tErk (MA5-15134; Pierce), anti-GFP (A-11122; Invitrogen), anti-GAPDH (sc-166545; Santa Cruz), anti-MF20 (Developmental\nStudies Hybridoma Bank), anti-Dusp6 (WH0001848M1; Sigma), anti-Lcp1 (124420; Genetex), anti-β-actin (4967; Cell Signal - ing) and anti-Raldh2 (P30011; Abmart). Secondary antibodies used for immunostaining were Alexa Fluor 488 goat anti-mouse IgG (A21121; Invitrogen), Alexa Fluor 488 goat anti-rabbit IgG (A11034; Invitrogen), Alexa Fluor 555 goat anti-mouse IgG (A21424; Invitrogen) and Alexa Fluor 555 goat anti-rabbit IgG (A21428; Invitrogen). Secondary antibodies for western blotting were HRP goat anti-rabbit IgG (sc-2004; Santa Cruz) and HRP goat anti-mouse IgG (sc-2005; Santa Cruz).\n\n\nMasson’s and AFOG staining\nFor AFOG staining, paraffin sections were incubated in Bouin’s solution (HT10132; Sigma) preheated at 56 °C for 2.5 h, incubated at room temperature for 1 h, briefly washed in tap water, incubated in 1% phosphomolybdic acid (P4869; Sigma) for 5 min, rinsed with distilled water and stained with AFOG staining solution (3 g of acid fuchsin (F8129; Sigma), 2 g of orange G (O3756, Sigma) and 1 g of anilin blue (AB0083; BBI) that were dissolved in 200 ml of acidified distilled water, pH 1.1) for 10 min. Sections were rinsed with distilled water, dehydrated with ethanol gradients and mounted. Myosin was visualized as orange, fibrin as red and colla - gen as bright blue.\nMasson’s staining was performed on paraffin sections using BASO Masson’s Trichrome Stain Kit (BA-4079A; Baso Diagnos - tics Inc). Myosin was visualized as red and collagen as bright blue.\nTransmission electron microscopy DMSO- and DPI-treated wild-type adult hearts, as well as Tg( myl7 :catalase-DsRed) transgenic adult hearts after injury at 14 dpa were fixed with glutaraldehyde at a final concentration of 2.5% (v/v) in 0.1 M sodium cacodylate buffer. Samples were rinsed and post-fixed with 1.0% OsO4. Next, the samples were embedded in the Spur resin and sectioned. The sections were stained with uranyl acetate and lead citrate and observed under a transmission electron microscope JEOL 1010 (JEOL Ltd., Japan).\n\n\nRT-PCR analysis\nTotal RNA from sham and injured hearts (15-20 hearts per sample) was extracted and purified using RNeasy@ mini kit (74106; Qiagen), and RT-PCR was performed with OneStep RT- PCR kit (210212; Qiagen) using 40 cycles on 50 ng of total RNA per reaction. Semi-quantitative RT-PCR of ERK1 and ERK2 was performed using 25 PCR cycles. Primer sequences were shown in Supplementary information, Table S5.\nStatistical analysis Statistical analysis was performed on relevant data using Stu - dent’s two-tailed t -test and the GraphPad Prism software package or SPSS18.0. Data were reported as mean ± SEM. For Supple - mentary information, Figure S12B and S12F, statistic significance between the two groups was determined with χ 2 -test. P &lt; 0.05 was considered statistically significant.\n\n\nAcknowledgments\nWe thank Drs RP Xiao, Y Rao, JR Peng, C Wei, K Ouyang, YM Wang, ZL Huang and JC Luo for critical comments on the manuscript, Dr IC Bruce for language editing, Dr Yingchun Hu\nfor help on transmission electromicroscopy, and the Pathology Core of Institute of Molecular Medicine for histology. We also thank Drs J Wang and KD Poss for their kind help in performing in situ hybridization with duox probes on Tg( tcf21 :nucEGFP) ventricles and Drs M Tsang, ZL Wen, T Evans, ZX Wang, LF Luo, GC Burns, J Chen, CM Cao and F Serluca for generous gifts of protocols, plasmid clones and/or transgenic zebrafish lines. This work was supported by the National Basic Research Program of China (2012CB944501, 2010CB529503 and 2013CB531200) and the National Natural Science Foundation of China (31271549, 31130067, 31000644 and 31221002).\n\n\nCell Research (2014) 24:1091-1107.\n\n\nPeidong Han et al . 1093 npg\n\n\nPeidong Han et al . 1095 npg\n\n\nPeidong Han et al . 1097 npg\n\n\nPeidong Han et al . 1099 npg\n\n\nPeidong Han et al . 1101 npg\n\n\nPeidong Han et al . 1103 npg\n\n\nCell Research | Vol 24 No 9 | September 2014\n\n\nPeidong Han et al . 1105 npg\n\n\nPeidong Han et al . 1107 npg\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''ORIGINAL ARTICLE\n© 2014 IBCB, SIBS, CAS    All rights reserved 1001-0602/14  $ 32.00 www.nature.com/cr\n\n\nHydrogen peroxide primes heart regeneration with a derepression mechanism\nPeidong Han 1, * , Xiao-Hai Zhou 1, * , Nannan Chang 1, * , Cheng-Lu Xiao 1 , Shouyu Yan 1 , He Ren 2, 3 , Xin-Zhuang Yang 1 , Mei-Ling Zhang 1 , Qing Wu 1 , Boyang Tang 1, 2 , Ju-Peng Diao 1 , Xiaojun Zhu 1 ,\nChuanmao Zhang 2, 3 , Chuan-Yun Li 1 , Heping Cheng 1, 2, 4 , Jing-Wei Xiong 1\n1 Institute of Molecular Medicine, Beijing Key Laboratory of Cardiometabolic Molecular Medicine and State Key Laboratory of Natural and Biomimetic Drugs, 2 State Key Laboratory of Biomembrane and Membrane Biotechnology, 3 College of Life Sciences, 4 Peking-Tsinghua Center for Life Sciences, Peking University, Beijing 100871, China\nWhile the adult human heart has very limited regenerative potential, the adult zebrafish heart can fully regenerate after 20% ventricular resection. Although previous reports suggest that developmental signaling pathways such as FGF and PDGF are reused in adult heart regeneration, the underlying intracellular mechanisms remain largely un - known. Here we show that H 2 O 2 acts as a novel epicardial and myocardial signal to prime the heart for regeneration in adult zebrafish. Live imaging of intact hearts revealed highly localized H 2 O 2 (~30 µM) production in the epicardi - um and adjacent compact myocardium at the resection site. Decreasing H 2 O 2 formation with the Duox inhibitors di - phenyleneiodonium (DPI) or apocynin, or scavenging H 2 O 2 by catalase overexpression markedly impaired cardiac re - generation while exogenous H 2 O 2 rescued the inhibitory effects of DPI on cardiac regeneration, indicating that H 2 O 2 is an essential and sufficient signal in this process. Mechanistically, elevated H 2 O 2 destabilized the redox-sensitive phosphatase Dusp6 and hence increased the phosphorylation of Erk1/2. The Dusp6 inhibitor BCI achieved similar pro-regenerative effects while transgenic overexpression of dusp6 impaired cardiac regeneration. H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration through two relatively independent pathways. We conclude that H 2 O 2 potentially generated from Duox/Nox2 promotes heart regeneration in zebrafish by unleashing MAP kinase signaling through a derepression mechanism involving Dusp6. Keywords : heart regeneration; zebrafish; Duox; H 2 O 2 ; Dusp6 Cell Research (2014) 24 :1091-1107. doi:10.1038/cr.2014.108; published online 15 August 2014\n*These three authors contributed equally to this work. Correspondence: Jing-Wei Xiong a , Heping Cheng b a, b Tel/Fax: 86-10-62766239 a E-mail: jingwei_xiong@pku.edu.cn b E-mail: chengp@pku.edu.cn\nReceived 11 February 2014; revised 6 May 2014; accepted 9 June 2014; published online 15 August 2014\n\n\nIntroduction\nThe ability to reverse heart failure, which accounts for &gt; 10% annual mortality in humans, is the most challeng - ing task in cardiac regenerative medicine. Many efforts are devoted to cardiac stem cell generation and tissue engineering, although making cardiac stem cells or tissue scaffolds functionally integrated into their host cardiac tissues remains difficult in clinical settings [1, 2]. A num - ber of studies have shown that the adult mammalian heart\nhas limited regenerative capacity, although not sufficient to fully restore heart function after substantial myocar - dial injury [2]. On the other hand, the adult zebrafish and newt hearts have full capacity of cardiac regeneration af - ter ventricular resection or cryoinjury, making zebrafish and newt as powerful animal models for studying heart regeneration in particular and organ regeneration in gen- eral [3-6]. The neonatal mouse heart in the first week of their postnatal life can fully regenerate after ventricular resection, suggesting that mammals gradually lose their regenerative potential during postnatal development [10]. It remains mysterious how a species gains or loses the regenerative potential during evolution.\nZebrafish has become a well-recognized model sys - tem for studying organ regeneration due to incredible regenerative potential in all of its adult organs tested and the availability of transgenesis, genetics and genomic\nresources [7]. The dedifferentiation model for the cellu - lar sources of cardiac repair/regeneration was supported by the fact that cardiac muscles mainly regenerate from pre-existing cardiomyocytes after injury in zebrafish [8, 9] and mammals [10-12]. The stem cell/progenitor cell model for cardiac regeneration in mammals was also recognized by the existence of Sca1 + and c-Kit + cardiac stem cells in mice [13, 14], while the transdifferentiation or lineage reprogramming model was proposed partly based on the ability of cardiac fibroblast transdifferenti - ation into myocytes upon induction by reprogramming factors or microRNA in mice [15-17] and reprogram - ming of embryonic atrial into ventricular myocytes after injury in zebrafish [18]. In spite of great efforts, the mo - lecular mechanisms underlying heart regeneration remain incompletely understood. Previous reports support that developmental signaling pathways such as FGF, PDGF, retinoid acid, TGFβ and integrin are essential for adult zebrafish heart regeneration [19-23]. Ventricular resec - tion induces expression of fgfr2/4 in the epicardium and their ligand fgf17b in the myocardium, and conditional blocking of the FGF signaling by overexpression of dominant-negative FGFR1 substantially increases fibrin/ collagen deposits and diminishes myocardial regenera - tion, supporting a notion that the FGF signaling promotes myocardial regeneration through epicardial-myocardial interaction [21]. A similar strategy revealed that the PDGF signaling is also required for zebrafish heart re - generation [20]. The exact molecular mechanisms down - stream of the FGF/PDGF signaling during adult heart regeneration are currently unknown.\nDuox is a member of NADPH-oxidase (Nox) and related dual oxidase (Duox) family, which has seven members, Nox1, Nox2, Nox3, Nox4, Nox5, Duox1 and Duox2 in humans, but only five (Nox1, Nox2, Nox4, Nox5 and Duox) identified in zebrafish [24]. Function - ally, the Duox or Nox enzyme complex in the plasma membrane catalyzes single-electron transfers from the electron donor NADPH to molecular oxygen, forming superoxide anions, which are quickly converted into hy - drogen peroxide (H 2 O 2 ) by spontaneous dismutation or catalytic reactions via Cu 2+ Zn 2+ -superoxide dismutase. Recently, it has been shown that Duox-generated H 2 O 2 in the tail-fin epithelium, which responds within ~3 min and peaks around 20 min after wounding, is critical for the rapid recruitment of leukocytes to initiate inflam - mation in zebrafish larvae [25, 26]. Sustained H 2 O 2 pro- duction would shift the intracellular redox homeostasis from a more reduced to a more oxidized state, affecting numerous redox-sensitive signaling pathways. Atax - ia-telangiectasia mutated (ATM), a PI3K-like serine/ threonine protein kinase preferentially activated by DNA\ndouble-strand breaks, serves as a sensor of oxidative stress (H 2 O 2 ). Oxidative stress (H 2 O 2 ) elicits the ATM-ho - modimer-Chk2 pathway to induce cell growth arrest or apoptosis [27]. On the other hand, H 2 O 2 acts through a redox sensor Lyn to mediate leukocyte wound attraction [25, 26]. Here we report that, after ventricular resection, H 2 O 2 levels are locally elevated in the epicardium and adjacent myocardium to promote heart regeneration by a derepression mechanism, whereby H 2 O 2 destabilizes the redox-sensitive phosphatase Dusp6 to enhance growth factor signaling.\n\n\nResults\nDuox and Nox2 are spatio-temporally associated with heart regeneration In a genome-wide search for genes critical for cardiac regeneration by mRNA deep sequencing (Supplemen - tary information, Figures S1 and S2), we found that the expression levels of a group of genes related to immune system diseases were upregulated, including p22 phox , p47 phox and p67 phox of the Nox2 complex (Supplementary information, Tables S1-S3; GEO repository accession number GSE50203). Combining with other public mi - croarray databases, we identified 37 genes in the reactive oxygen species (ROS) system that were differentially expressed in the injured zebrafish heart at 7 days post amputation (dpa) (Supplementary information, Table S4), when injury-induced inflammation started to fade and active myocardial regeneration began [3, 28, 29]. RNA in situ hybridization was then applied at multiple time points from 3 to 30 dpa for validation. We found that duox was particularly interesting because of the spatial and temporal patterns of its expression during regeneration. RNA in situ hybridization with two inde - pendent duox probes revealed that duox expression was weak at sham, peaked and was highly localized beneath the surface of the amputation at around 3, 7 and 14 dpa, and returned to the basal level at 30 dpa when regener - ation was nearly complete (Supplementary information, Figures S3A-S3E and S5A-S5E). Each of the two sense duox probes had no detectable signals (Supplementary information, Figures S4 and S5F), indicating the speci - ficity of the two duox probes. RNA in situ hybridization on Tg( tcf21 :nucEGFP) transgenic hearts revealed that duox expression largely overlapped with the EGFP signal (Figure 1A-1C and Supplementaryinformation, Figure S6), which primarily labels the epicardium [30], sug - gesting that duox was enriched in the epicardium. Thus, both the time course and epicardial localization of duox expression correlated with those of active myocardial re - generation after injury.\nSystematic examination revealed that the expression of nox2 , but none of the other nox genes, was moderately induced at 3, 7 and 14 dpa in a pattern similar to that of duox (Supplementary information, Figure S7). RT-PCR confirmed expression of duox and nox2 in 14 dpa hearts (Supplementary information, Figure S3F). These results indicate that the expression of duox and nox2 , with duox being dominant, is spatially and temporally associated with myocardial regeneration after ventricular resection.\nH 2 O 2 signal is essential for heart regeneration To visualize the spatiotemporal dynamics of H 2 O 2 in the intact heart during regeneration, we used Tol2-based transgenesis and zebrafish myosin light chain 7 ( myl7 ) promoter to generate transgenic zebrafish with cardi - ac-specific expression of Hyper (Figure 1D), a fluores - cent protein-based H 2 O 2 sensor [25, 31]. High-resolution confocal imaging was performed in intact adult hearts excised from Tg( myl7 :Hyper) transgenic animals, and H 2 O 2 levels were directly measured as the ratio between the fluorescent signals dually excited at 488 and 405 nm ( R = F 488 / F 405 ). By ex vivo titration on the Hyper-ex - pressing hearts freshly excised from uninjured animals, we obtained a calibration curve (Figure 1K and Supple - mentary information, Figure S9A-S9G) similar to that obtained in vitro [31]. Compared to sham controls (Figure 1E), local H 2 O 2 concentration was clearly elevated in injured hearts at 3, 7 and 14 dpa (Figure 1F-1G and 1I), and returned to the basal level after nearly full regenera - tion at 30 dpa (Figure 1H and 1I). In contrast to the rapid (on the scale of minutes) inflammatory H 2 O 2 response in the tail-fin epithelium of zebrafish larvae [25, 26], there was a slow onset of ventricular resection-induced H 2 O 2 production, as no detectable increase in the Hyper fluorescence ratio occurred within ~1 h after heart injury (data not shown). As seen on confocal optical sections (Figure 1E-1H) and from line plots of the spatial profiles (Figure 1J), the H 2 O 2 gradients spanned ~20 µm depth from the epicardium to the myocardium, reaching a peak concentration of ~30 µM (Figure 1K and Supplementa - ry information, Figure S9A-S9G), which is within the range of concentrations for H 2 O 2 to act as an intracellular messenger [32]. In an independent measurement, we stained 14 dpa Tg( myl7 :Hyper) transgenic hearts with Redox sensor cc-1 and visualized the H 2 O 2 signal in both the epicardium that lacked Hyper expression (Fig - ure 1O) and adjacent compact myocardium (Figure 1L- 1O). Therefore, direct visualization of H 2 O 2 with either a genetically encoded probe or a small-molecule indicator revealed that H 2 O 2 was elevated primarily in the epicar - dium and the adjacent myocardium at the resection site, highly correlated with myocardial regeneration.\nTo investigate whether there is a causal relationship between elevated Duox/Nox2 and H 2 O 2 production and cardiac regeneration, we applied diphenyleneiodoni - um (DPI) (10 µM) or apocynin (100 µM), widely used inhibitors of Duox/Nox enzymes [33] and examined subsequent changes in H 2 O 2 production and cardiac re- generation. We found that Hyper fluorescent signals were decreased by 50% within ~ 40 min after soaking 14 dpa Tg( myl7 :Hyper) hearts in DPI-containing medium (Fig - ure 2A-2B), while the Hyper signals remained stable in control medium containing DMSO only (Figure 2B and Supplementary information, Figure S8A), suggesting that high concentration of H 2 O 2 likely resulted from elevated duox/nox2 expression but was not produced from mito - chondria. Functionality of ex vivo cultured hearts was validated by their rhythmic contractions under 1 Hz elec - trical field stimulation after 1 h culture (Supplementary information, Figure S8B-S8C). Based on the previous re - port that myocardial cells enter mitotic phases in injured areas of the heart upon ventricular resection in zebrafish or myocardial infarction in mice [3, 34], we quantified injury-induced myocyte proliferation by counting BrdU + / Mef2C + double-positive myocytes during heart regener - ation. After ventricular resection, zebrafish were allowed to recover for 7 days and BrdU was then co-injected with either DPI or apocynin into the abdominal cavity once daily until the hearts were collected for examination. Compared with sham-operated controls, we found that BrdU + /Mef2C + myocyte proliferation was induced in in - jured hearts at 14 dpa (Figure 2C, 2C’ and 2E), but their numbers were significantly reduced by treatment with either DPI (Figure 2D, 2D’ and 2E) or apocynin (Figure 2E and data not shown). In addition, extensive cardiac fibrosis (Figure 2G) and reduced Myl7 + myocardial re- generation were evident in injured hearts with DPI (Figure 2I) or apocynin treatment (data not shown), compared with DMSO-treated hearts (Figure 2F and 2H). Although injection of H 2 O 2 did not induce myocyte proliferation in sham-operated animals (Supplementaryinformation, Fig - ure S10), it rescued the inhibitory effects of DPI on the numbers of BrdU + /Mef2C + myocytes and Gata4 + cells (Supplementary information, Figure S11). Importantly, DPI-treated hearts failed to regenerate even at 60 dpa (Supplementary information, Figure S12), suggesting a permanent arrest rather than a delay of regenerative pro - cess. In addition, exogenous H 2 O 2 partially rescued the inhibitory effect of DPI on embryonic heart regeneration (Supplementary information, Figure S13A and S13B) af - ter a large number of cardiomyocytes were damaged by incubating Tg( myl7 :CFP-NTR) transgenic embryos with metronidazole (Mtz), as Mtz can be converted into a tox - ic DNA crosslinking agent by nitroreductase (NTR) as\npreviously described [35]. We further validated the pene - tration of H 2 O 2 by imaging the 72 hours post fertilization (hpf) embryonic Tg( myl7 :Hyper) hearts and found that Hyper signals were increased upon elevated external H 2 O 2 concentration (Supplementary information, Figure\nS14). These results strongly support the hypothesis that H 2 O 2 is required and sufficient for cardiac regeneration. If H 2 O 2 is causally linked to cardiac regeneration as suggested by its inhibition upon treatment with DPI or apocynin, limiting H 2 O 2 signaling by augmenting an -\nFigure 1 Epicardial Duox-generated H 2 O 2 associated with myocardial regeneration. (A - C) After imaging of Tg( tcf21 :nucEG- FP), we performed RNA in situ hybridization with duox probes on 10- µ m cryosections of Tg( tcf21 :nucEGFP) hearts at sham, 7 and 14 dpa. Note the induced expression of duox and its colocalization with epicardial reporter Tg( tcf21 :nucEGFP) in in- jured areas (arrowheads) at 7 and 14 dpa (B , C) while little staining at sham (A) . Scale bar for A - C , 50 µ m. (D) Cardiac-spe- cific expression of Hyper in a zebrafish embryo. Hyper was restricted in the Tg( myl7 :Hyper) transgenic heart at 48 hpf. (E - H) Ex vivo Hyper heart images from sham control (E) , 7 dpa (F) , 14 dpa (G) and 30 dpa (H) . Spatially resolved H 2 O 2 image, in- dexed by the ratio between the F 488 and F 405 images of Hyper (left panels), is presented in pseudocolor. (I) Ratiometric Hyper signals ( F 488 / F 405 ) averaged over the regenerative zone of injured heart during the first month of regeneration after resection.\nn = 3. (J) Representative transmural spatial profiles of the Hyper signal at 14 dpa (red line) and sham (green line) hearts. (K) Ex vivo calibration of Hyper F 488 / F 405 ratio as a function of ambient H 2 O 2 concentration. n = 3. Arrow denotes the average F 488 / F 405 ratio seen in regenerative zones at 14 dpa. (L - O) Redox signal showing greater epicardial and myocardial oxidization during regeneration. A 14 dpa Tg( myl7 :hyper) heart was labeled with Redox sensor cc-1. Images show Hyper signals at 488 (L) and 405 nm (M) , and Redox sensor cc-1 signals at 555 nm excitation (N) , and the merging of L and N (O) . Note that the Re- dox sensor cc-1 signal (arrowheads), indicative of intracellular oxidization, was most conspicuous in epicardial cells lacking Hyper expression, and also overlapped with the Hyper signal (arrows) in adjacent myocardium. Scale bars, 100 µ m.\nFigure 2 H 2 O 2 signaling is required for heart regeneration. (A - B) Inhibiting Duox/Nox NADPH oxidases by DPI decreased H 2 O 2 generation. Time-lapse confocal images (A) and statistics (B) of the Hyper ratio in Tg( myl7 :Hyper) hearts prior to and after application of DPI (10 µ M). DMSO (0.1%) was used as control. n = 3-5. (C - E) Treatment with DPI or apocynin impaired cardiac myocyte regeneration. Proliferating myocytes were identified by double-staining with anti-Mef2C (red) and anti-BrdU (green) (C’ , D’ , arrows ) . Note that there were fewer double-positive cells (yellow) in DPI-treated heart (D , D’) than in DMSO control heart (C , C’) . Quantitative results with sham, DMSO, DPI or apocynin treatment are shown in E . n = 5 to 7. See Mate- rials and Methods for details of treatment. (F - I) Accumulated fibrin/collagens (white arrowheads in G , Masson’s staining) and compromised myocardial regeneration (white arrowhead in I , in situ hybridization with myl7 probes) after DPI treatment, com- pared with DMSO control (F , H) . (J - M) Cardiac-specific overexpression of catalase retarded heart regeneration. Tg( myl7 :- Catalase-DsRed) heart displayed larger amounts of fibrin/collagens (white arrowheads in K , Masson’s staining) and compro- mised myocardial regeneration (white arrowheads in M , in situ hybridization with myl7 probes), as compared with respective non-transgenic sibling hearts at 30 dpa (J , L) . n = 5-8. Scale bars, 100 µ m.\ntioxidant capacity of the heart should decrease cardiac regeneration. To directly test this hypothesis, we gener - ated Tg( myl7 :catalase-DsRed) transgenic zebrafish with cardiac-specific expression of catalase. The efficacy of catalase-DsRed was validated by its ability to scavenge H 2 O 2 [36] (Supplementary information, Figure S13C- S13E). Similar to the Duox/Nox inhibition, there were fewer Myl7 + regenerated myocytes (Figure 2M), and extensive cardiac fibrosis (Figure 2K) at 30 dpa in cat - alase-transgenic hearts, compared to non-transgenic sibling hearts (Figure 2L and 2J). However, we were not able to generate ubiquitously heat-inducible Tg( hsp70 :- catalase) transgenic lines, due to lethality of these trans - genic embryos from leaky expression of catalase without heat shock. Moreover, cardiac-specific overexpression of catalase-DsRed suppressed embryonic heart regener - ation in Mtz-induced heart injury of Tg( myl7 :CFP-NTR) transgenic embryos (Supplementary information, Figure S13F-S13H). Previous studies suggest that myocardial dedifferentiation is a critical step for heart regeneration\nin zebrafish [8, 9]. Interestingly, inhibiting H 2 O 2 signaling by either DPI treatment or catalase overexpression had no effects on myocardial sarcomere disassembly, an indi - cator of myocardial dedifferentiation, in injured hearts at 14 dpa (Supplementary information, Figure S15). These results suggest that the high concentration of H 2 O 2 , likely caused by elevated duox/nox2 expression, is obligatory for cardiac regeneration after injury without affecting in - jury-induced myocardial dedifferentiation.\nThe Duox-H 2 O 2 -Dusp6 pathway primes myocardial re- generation by elevating Erk1/2 activity via a derepres- sion mechanism Because developmental signaling pathways of FGF and PDGF are reused during adult zebrafish heart re - generation [20, 21], we investigated whether H 2 O 2 mod- ulated the common MAP kinase pathway downstream of the growth factors. We found that phosphorylation of Erk1/2 (pErk) was increased in injured hearts at 3, 7 and 14 dpa (Figure 3C and Supplementary information,\nFigure 3 H 2 O 2 activates Erk1/2 for myocardial regeneration. (A - B) Phosphorylation of Erk1/2 (pErk1/2) in epicardial cells (A , A’) in 14 dpa heart, as detected by anti-pErk1/2 antibody. Decrease of pErk1/2-positive cells by DPI treatment (B , B’) . Scale bar for A and B , 100 µ m. (C) Western blot showing greater pErk1/2 in injured hearts at 3, 7 and 14 dpa, and its blockade by DPI treatment at 14 dpa. (D-F) Gata4 + myocardial cells in injured areas in control Tg( gata4 :EGFP) heart (DMSO treatment, D , D’ , D’’ ) and their disappearance after inhibition of Duox (DPI treatment, E , E’ , E” ), or Mek1/2 (U0126 treatment, F , F’ , F” ). Car- diomyocytes were co-labeled with anti-EGFP and anti-MF20 antibodies and nuclei were labeled by DAPI. Scale bar for D - F , 100 µ m. (G) Statistics of Gata4 + cardiomyocytes at 14 dpa.\nFigure S16), with the pErk1/2 + cells primarily located in non-myocardium such as the epicardium (Figure 3A and 3A’), evidenced by overlapping with epicardial Raldh2, a molecular marker for both the epicardium and endocar - dium (Supplementary information, Figure S18A-S18C). The total erk1 and erk2 mRNA, as well as total Erk pro - teins were slightly increased during heart regeneration (Figure 3C and Supplementary information, Figure S17). Using Tg( gata4 :EGFP) transgenic reporter zebrafish to track newly regenerated myocardium in injured hearts [8], we found that decreasing pErk1/2 with U0126 (a Mek1/2 inhibitor) interfered with the regeneration of myocardial cells (Figure 3F, 3F’ and 3G), as evidenced by reduced EGFP-positive cells compared with DMSO controls (Figure 3D, 3D’ and 3G). More importantly, the formation of pErk1/2 + cells (Figure 3B, 3B’ and data not shown) and the generation of GATA4-EGFP + myocardial cells (Figure 3E, 3E’, 3G and data not shown) were sen - sitive to inhibition of Duox/Nox2 by treatment of either DPI or apocynin. Unfortunately, strong autofluorescence in Tg( myl7 :catalase-DsRed) prevented us from perform - ing any fluorescence-based immunostaining. These re - sults showed that epicardial Erk1/2- and H 2 O 2 -dependent signals are interlinked in promoting regeneration in the adult zebrafish heart.\nOf the potential redox-sensitive targets of H 2 O 2 , the dual-specificity MAPK phosphatase 6 (Dusp6) is dis - tinct in that it is stimulated by pErk1/2, but reciprocally dephosphorylates and deactivates pErk1/2, forming a negative feedback that limits Erk1/2 activity. Previous reports have shown that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, and that oxidized Dusp6 is prone to degradation, leading to aberrant Erk1/2 activation [37, 38]. Dusp6-knockout mice show greater proliferation of myocytes [39], but the function of Dusp6 in cardiac regeneration is unknown. We thus formulated a hypothesis with two seemingly paradoxical predictions. First, Erk1/2 activation would elevate Dusp6 expression during regeneration, activating a negative feedback (i.e., a repression). However, if H 2 O 2 can oxidize and thereby destabilize Dusp6, ventricular resection-induced H 2 O 2 production would unleash the pro-regenerative pErk1/2 signal (i.e., a derepression).\nTo test the above predictions, we examined Dusp6 expression at both the mRNA and protein levels. Upon injury, dusp6 mRNA level was elevated, peaking at 7 dpa (Figure 4A and Supplementary information, Figure S19), and Dusp6 protein levels were increased (Figure 4B) and Dusp6 was particularly localized in the epicar - dium at 7 and 14 dpa (Figure 4D-4E and Supplementary information, Figures S18D-S18F and S20B-S20D) and decreased at 30 dpa (Figure 4F and Supplementary in -\nformation, Figure S20G). The most intriguing finding is that in injured hearts at 14 dpa after DPI or apocynin treatment, Dusp6 protein, but not dusp6 mRNA, was dramatically further increased in both the epicardium and myocardium with clear nuclear enrichment (Figure 4A, 4B, 4G and 4H), unmasking an inhibitory effect of H 2 O 2 on Dusp6 protein stability in the epicardium and myocardium undergoing active regeneration. We also confirmed this finding on Tg( dusp6 :EGFP), where EGFP expression was previously shown to recapitulate the endogenous dusp6 mRNA expression pattern during embryogenesis [40]. Epicardial EGFP expression was evident in injured areas of adult hearts at 14 dpa (Figure 4I-4J). Furthermore, pErk1/2 was largely co-localized with Dusp6 (Supplementary information, Figure S21C) and inhibiting pErk1/2 by U0126 reduced Dusp6 expres - sion (Supplementary information, Figure S21A-S21D), suggesting a close interlink between them. These data substantiate the idea that Dusp6 mRNA and protein are initially induced as a negative feedback for pErk1/2, but subsequently H 2 O 2 , likely produced by Duox/Nox2, de - stabilizes Dusp6 to impart a derepression mechanism by suppressing this negative feedback in the epicardium.\nTo further evaluate the role of the Duox-H 2 O 2 -Dusp6 derepression signaling pathway in heart regeneration, we used the small molecule BCI, recently developed in a transgenic zebrafish chemical screen [40], which directly inhibits the phosphatase activity of Dusp6. We found that BCI counteracted the inhibitory effect of DPI or apocynin on the generation of the double-positive BrdU + /Mef2C + myocytes, but had no effects on DPI- or apocynin-enhanced cardiac fibrosis at 14 dpa (Figure 4K, 4M, 4N, 4O and Supplementary information, Figure S24A-S24C). Moreover, BCI also reversed the effects of DPI or apocynin on MF20 staining (which indicates myocardial regeneration), and cardiac fibrosis at 30 dpa (Supplementary information, Figure S24D-S24I). These results indicate that BCI mimics the endogenous H 2 O 2 signal when the latter is abrogated. However, in hearts untreated with DPI or apocynin, BCI did not alter the generation of BrdU + /Mef2C + myocytes at 14 dpa (Fig - ure 4K, 4L and 4O), consistent with the idea that BCI is redundant in the presence of an intact endogenous H 2 O 2 signal. Like exogenous H 2 O 2 (Supplementary informa - tion, Figure S10), BCI had no effects on myocardial proliferation in sham-operated hearts (Supplementary information, Figure S23), which is consistent with the relatively low levels of Dusp6 proteins in sham hearts (Figure 4C and Supplementary information, Figure S20A). To examine whether H 2 O 2 is involved in coronary vessel regeneration, we used the Tg( flk1 :EGFP) zebrafish to visualize coronary vessels in the adult heart. We found\nMef2C + proliferating myocytes. Scale bars, 100 µ m.\nthat DPI treatment reduced the amount of Flk1 + vessels, which was rescued by co-injection of BCI (Supplementa - ry information, Figure S22), suggesting an additional role of H 2 O 2 in coronary vessel repair upon ventricular resec - tion. Collectively, interventions at the levels of Duox/\nNox2, H 2 O 2 and Dusp6 corroborated the hypothesis that the Duox-H 2 O 2 -Dusp6 pathway plays an essential role in priming myocardial and blood vessel regeneration, by elevating Erk1/2 activity via a derepression mechanism.\nTo directly test whether zebrafish Dusp6 (zDusp6) is\nsubjected to H 2 O 2 -dependent ubiquitination, we mea - sured the ubiquitinylated zDusp6 (Ub-zDusp6) protein levels with or without H 2 O 2 stimulation in HeLa cells and zebrafish embryos, given that adult zebrafish heart is hardly accessible for these molecular and biochemical methods. We found that exogenous H 2 O 2 treatment de- creased, in a dose-dependent manner, the levels of Myc- tagged zDusp6 proteins in HeLa cells overexpressing zDusp6-Myc (Figure 5A and 5B). Similarly, endogenous zDusp6 underwent degradation in zebrafish embryos after H 2 O 2 treatment (Figure 5C and 5D). Furthermore, Ub-zDusp6 was hardly detectable, but was detected after treatment with 1 mM H 2 O 2 in HeLa cells overexpressing zDusp6-Myc and Flag-tagged ubiquitin (Ub-Flag) (Figure 5E), suggesting the existence of unstable Ub-zDusp6 that is dependent on H 2 O 2 . Proteasome inhibitor MG132 in - creased the amount of Ub-zDusp6 in the absence or pres - ence of H 2 O 2 , supporting the notion that H 2 O 2 treatment enhances the ubiquitination of zDusp6. On the other hand, we failed to detect Ub-zDusp6 in the presence of H 2 O 2 and MG132 in zebrafish embryos overexpressing zDusp6-Myc and Ub-Flag (data not shown), suggesting that Ub-zDusp6 is highly unstable in zebrafish embryos.\nTherefore, H 2 O 2 regulates the ubiquitination and protea - some degradation of zebrafish Dusp6.\nH 2 O 2 promotes heart regeneration independent of im- mune cell recruitment Previous studies have shown that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail-fin epithelium of zebrafish larvae [25, 26]. To differentiate potentially dual function of H 2 O 2 in recruiting inflamma - tory cells and priming heart regeneration, we examined temporal and spatial distribution of immune cells during heart regeneration by labeling macrophages and neu - trophils with either anti-lymphocyte cytosolic protein 1 (Lcp1) or Tg( coro1a :EGFP) [41]. We found that Lcp1 + leukocytes appeared around 3 dpa, peaked at 7 and 14 dpa, decreased at 19 dpa and almost disappeared at 24 dpa (Figure 6A-6F), and the recruitment occurred in a H 2 O 2 -dependent manner (Figure 6G and 6I). Quantifi - cation of Lcp1 + leukocytes during heart regeneration in Figure 6 was included in Supplementary information, Figure S25. Similar dynamics of leukocyte recruitment was obtained by labeling leukocytes with an independent Tg( coro1a :EGFP) transgenic reporter (Supplementary\nFigure 5 H 2 O 2 induces ubiquitination and proteasome degradation of zebrafish Dusp6. (A - B) Exogenous H 2 O 2 decreased the expression level of Myc-tagged zDusp6 in a dose-dependent manner in HeLa cells as evidenced by western blotting (A) . The statistics of A is shown in B . β -actin was used for normalizing protein loading. (C - D) Exogenous H 2 O 2 decreased the ex- pression level of endogenous zDusp6 in a dose-dependent manner in zebrafish embryos at 24 hpf (C) , and the statistics of C is shown in D . β -actin was used for normalizing protein loading. (E) Ub-zDusp6 (arrowhead) was detected after H 2 O 2 and/ or  MG132  treatments  in  HeLa  cells  overexpressing  Myc-tagged  zDusp6  and  Flag-tagged  ubiquitin  (Ub-Flag).  Myc-tagged Dusp6 proteins were detected for all groups except the group without zDusp6-Myc overexpression.\ninformation, Figure S26A-S26F). Furthermore, DPI treatment either from 7 to 14 dpa (when immune cells peaked, DPI timely inhibited leukocyte recruitment) or from 14 to 30 dpa (when immune cells gradually disap - peared, DPI should not affect normal leukocyte recruit - ment) led to increased cardiac fibrosis and compromised myocardial regeneration (Figure 6L and 6N), suggesting that both the early- (7-14 dpa) and late-phase H 2 O 2 pro- duction (14-30 dpa) are required for cardiac regeneration. Importantly, co-injection of BCI with DPI or apocynin largely restored myocardial regeneration (Figure 6M and 6O), but failed to rescue immune cell recruitment (Figure 6H, 6J and Supplementary information, Figure S26G-S26H), indicating that Dusp6 inhibition by BCI suffices to initiate regeneration independently of immune cell recruitment. Taken together, these data suggest that while H 2 O 2 plays a dual role in recruiting immune cells and promoting heart regeneration, the two pathways are apparently separable and relatively independent of each other.\nDusp6 is a repressor for zebrafish heart regeneration To directly examine the role of dusp6 in heart regen- eration, we established Tg( hsp70 :dusp6-His) transgenic zebrafish, where overexpression of Dusp6 in both epicar - dium and myocardium was conditionally driven by heat shock [21]. Transgenic expression of dusp6 was con - firmed to decrease pErk in both embryos and adult hearts (Figure 7A). Compared with non-transgenic siblings (Figure 7B, 7F, 7H, 7J and 7L), injured transgenic hearts displayed fewer Mef2C + /BrdU + proliferating myocytes (Figure 7C-7D), Gata4 + myocytes (Figure 7E and 7G) as well as pERK + cells (Figure 7I) at the injury site, leading to increased cardiac fibrosis (Figure 7K) and diminished cardiac regeneration (Figure 7M). Together with the BCI data (Figures 4 and 6), these results establish a repressive role of dusp6 in heart regeneration, acting through de- phosphorylation of pErk in the epicardium.\n\n\nDiscussion\nIn summary, we have presented three major inter - related findings. First, upon injury, H 2 O 2 is produced, likely by Duox/Nox2, up to 30 µM in the epicardium and adjacent myocardium during the entire course of active myocardial regeneration. Second, H 2 O 2 is required for promoting cardiac myocyte proliferation and suppress - ing fibrosis. Third, the pro-regeneration effect of H 2 O 2 is mediated through destabilizing the redox-sensitive and ubiquitin-conjugated phosphatase Dusp6, which provides a derepression of the common MAP kinase signaling. Furthermore, we have identified BCI as the first small\nmolecule for promoting cardiac regeneration when H 2 O 2 generation was defective. In spite of previous findings that H 2 O 2 signal is re - quired for the rapid inflammatory response in the tail- fin epithelium of zebrafish larvae [25, 26], our data show that the H 2 O 2 -guided inflammatory response occurs around 3 dpa in adult zebrafish hearts (Figure 6 and Sup - plementary information, Figure S26). This time course, when compared with that of Duox/Nox2 induction and H 2 O 2 production, supports the idea that it likely involves de novo synthesis of duox transcripts and possibly pro - teins for the production of H 2 O 2 and subsequent recruit - ment of leukocytes in heart regeneration. In contrast, because the injured larvae tail fin is directly exposed to the environment, it needs immediate immune defense after injury and therefore only takes 3 min to recruit neu - trophils to the injured site, through activation of Duox without transcription and translation in the tail fin epithe - lium [25, 26]. In addition to its function in recruiting in - flammatory leukocytes, our data suggest that H 2 O 2 is also required for promoting adult heart regeneration, which is consistent with previous reports on H 2 O 2 function in promoting injury-induced peripheral sensory axon regen - eration in zebrafish larvae [42] and tail regeneration in Xenopus tadpole [43]. H 2 O 2 acts through a redox sensor Lyn to mediate injury-induced leukocyte wound attrac - tion in zebrafish larvae tail fin [25, 26], but direct targets of H 2 O 2 in zebrafish sensory neurons or Xenopus tadpole tail epithelium are not known [42, 43]. Our data show that H 2 O 2 not only destabilizes Dusp6 to elevate growth factor signaling in promoting adult heart regeneration, but also acts through an unknown, independent mech - anism to recruit inflammatory leukocytes during heart regeneration. Future studies need to examine whether H 2 O 2 acts through Lyn or other redox sensors to induce inflammatory leukocyte recruitment during adult heart regeneration.\nGrowth factor signaling pathways such as FGF and PDGF pathways are essential for adult heart regenera - tion [20, 21]. In contrast to previous reports on a neg - ative role of p38 MAPK in mammalian and zebrafish heart regeneration [44,45], our data support that H 2 O 2 in the epicardium (Figure 1) is essential for elevating phosphorylation of Erk1/2 MAPK through repression of Dusp6 during zebrafish heart regeneration (Figures 3-5 and 7). Our data are aligned well with the previous re - ports that H 2 O 2 directly oxidizes Dusp6 in TNFα-induced cell death and ovarian cancers, leading to ubiquitination and subsequent proteasome degradation of Dusp6 and thus aberrant Erk1/2 activation [37, 38] (Figure 5). The elevated pErk1/2 in the epicardium might then activate soluble factors at transcriptional and/or post-transcrip -\ntional levels, which in turn bind to their receptors in the myocardium and promote myocardial proliferation and regeneration. However, we failed to detect the changes in Dusp6 and pErk protein levels in the heart by west - ern blotting (data not shown). It is likely that the subtle changes in Dusp6 and pErk protein levels in the epicardi - um of injured hearts might not be well detected by west -\nFigure 6 H 2 O 2 has dual function in recruiting Lcp1 + leukocytes and promoting heart regeneration. (A - F) Immunostaining of Lcp1 showing that leukocytes (macrophages and neutrophils) were undetectable at sham (A) , appeared at 3 dpa (B) , peaked between 7 and 14 dpa (C , D) and gradually disappeared from 19 to 24 dpa (E , F) . Either apocynin (Apo) (G) or DPI (I) inhib- ited leukocyte recruitment, which could not be rescued by BCI (H , J) at 14 dpa. Myocytes were labeled by anti-MF20, and nuclei were labeled by DAPI. (K - M) All AFOG staining analyses were performed at 30 dpa and DPI and/or BCI treatment was applied as indicated. Myosin was visualized as orange, fibrin as red and collagen as bright blue. AFOG staining showed that DPI treatment from 7 to 14 dpa increased cardiac fibrosis (L) , which was rescued by BCI (M) , compared with DMSO control (K) at 30 dpa. (N - O) AFOG staining showed that DPI treatment from 14 to 30 dpa increased cardiac fibrosis (N) , which was rescued by BCI (O) . Scale bars, 100 µ m.\nern blotting. On the other hand, cardiac-specific overex - pression of catalase, the H 2 O 2 -specific scavenger, also led to regenerative defects in the heart (Figure 2J-2M). New transgenic lines, such as Tg( tcf21 :catalase) without autofluorescence or heat-inducible Tg( hsp70 :catalase) without leaky transgene expression are needed for deter - mining the levels of Dusp6 and pErk by immunostaining\nFigure 7 The repressive effects of Dusp6 overexpression on heart regeneration through dephosphorylation of pERK. (A) Top: a scheme showing that in Tg( hsp70 :dusp6-His), His-tagged Dusp6 is driven by the zebrafish hsp70 promoter. Transgenesis was mediated by Tol2 transponase. Bottom: heat shock-induced expression of His-tagged Dusp6 proteins diminished pErk in transgenic embryos (Tg) at 72 hpf or transgenic adult hearts (Tg), compared with their sibling embryos or adult hearts. β -actin was used to normalize protein loading. (B - D) Mef2C + /BrdU + proliferating myocytes (arrowheads) were decreased in dusp6 transgenic hearts (C) compared with those in non-transgenic siblings (B) , with the statistics shown in D . (E - G) Representative images showing that Tg( gata4 :EGFP) myocytes (arrowheads), which colocalized with cardiac marker MF20, were diminished in dusp6 transgenic hearts (G) compared with their siblings (F) , with the statistics shown in E . (H - I) Reduction of pErk + cells (arrowheads), not overlapping with cardiac marker MF20, in transgenic hearts (I) , compared with non-transgenic siblings (H) . (J - M) Transgenic overexpression of Dusp6 led to cardiac fibrosis (assayed by AFOG staining, K ) and compromised cardiac regeneration (assayed by MF20 staining, M ), compared with their siblings (J , L) . Nuclei were co-stained with DAPI. Scale bars, 100 µ m. after scavenging H 2 O 2 by catalase in the future. Based on the data from others and ours, we propose that the working model for cardiac regenerative signaling comprises two main branches: the forward pro-regenera - tive MAPK signaling triggered by FGF and other growth factors, as recently elucidated [20, 21]; and the derepres - sion mechanism through the Duox/Nox2-H 2 O 2 -Dusp6 pathway that converges on Erk1/2 MAPK signaling primarily in the epicardium, which then activate puta - tive soluble factors to promote myocardial regeneration\n(Figure 8). Such epicardial soluble factors might include Wnt1 and IGF2 that were previously shown to play an essential function in the epicardium/cardiac fibroblasts and myocardium [46-49]. During heart regeneration, the growth factor-MAPK signaling (pErk) is gradually en - hanced in injured hearts from 3 to 14 dpa with the peak around 14 dpa, which induces dusp6 expression (positive induction) that in turn dephosphorylates pErk (negative feedback) (Figures 3 and 4). Simultaneously, H 2 O 2 in- creases and destabilizes Dusp6 with the peak from 7 to\nFigure 8 H 2 O 2 promotes heart regeneration through a derepression mechanism in zebrafish. Upon ventricular resection, FGF and other growth factor receptors are activated and resultant phosphorylation of Mek1/2 (pMek1/2) and Erk1/2 (pErk1/2) in- duces expression of dusp6 , which in turn, exerts a negative feedback on pErk1/2 in non-myocardial cells (primarily epicardial cells). Concurrently, injury elicits a time-dependent, enduring localized duox and, to a minor extent, nox2 expression that sup- ports intense H 2 O 2 formation primarily in the epicardium close to the injury site. High-concentration intracellular H 2 O 2 oxidizes and thereby destabilizes the Dusp6 phosphatase, diminishing its inhibition of pErk1/2. Thus, the newly identified Duox/Nox2- H 2 O 2 -Dusp6 signaling serves as a derepression mechanism and, when activated, unleashes the pro-regenerative pErk1/2 signaling likely through generation of soluble factors that would eventually act to enhance myocardial regeneration. DPI and apocynin (Apo), inhibitors of Duox/Nox2; catalase, catalytic scavenger of H 2 O 2 ; BCI, inhibitor of Dusp6; U0126, inhibitor of Mek1/2.\n14 dpa, which results in decreased Dusp6 expresssion and thus increased pErk levels at 14 dpa when myocar - dial regeneration actively takes place. In addition, we have shown that H 2 O 2 /Dusp6 is also involved in coro - nary vessel regeneration upon ventricular resection. This is consistent with previous studies that FGF and PDGF signaling pathways are essential for coronary vessel formation during heart regeneration [20, 21]. Therefore, this work provides the first intracellular H 2 O 2 -Dusp6- pErk mechanisms downstream of growth factor signaling in adult myocardial and coronary vessel regeneration. As there are highly conserved mammalian homologs for all components of the Duox/Nox2-H 2 O 2 -Dusp6-Erk1/2 axis, this derepression mechanism for cardiac regenera -\ntion, as demonstrated here in zebrafish, may also operate in mammals, and if so, future efforts on activating the mammalian counterpart may offer opportunities for ther - apeutic myocardial regeneration in heart failure.\n\n\nMaterials and Methods\n\n\nZebrafish lines\nZebrafish were raised and handled in accordance with the Peking University Institutional Animal Care and Use Commit - tee accredited by the AAALAC International. Wild-type TL, Tg( myl7 :HyPer) pku327 , ( myl7 :catalase-DsRed), Tg( dusp6 :EGFP) and Tg( gata4 :EGFP) transgenic zebrafish lines at 6-12 months of age were used for heart surgery. Several Tg( myl7 :HyPer) pku327 , Tg( myl7 :catalase-DsRed) pku343 , Tg( myl7 :CFP-NTR) pku360 and\nTg( hsp70 :dusp6-His) pku344 transgenic zebrafish lines were gen - erated in TL background by using Tol2-based transgenesis [31, 50]. Tg( tcf21 :nucEGFP) was previously reported [30]. Tg( coro- 1a :EGFP) was provided by Dr Zilong Wen [41], Tg( dusp6 :EGFP) transgenic zebrafish were obtained from Dr Michael Tsang [40] and Tg( gata4 :EGFP) transgenic zebrafish were kindly provided by Dr Todd Evans [51]. The zebrafish myl7 promoter was provided by Dr C Geoff Burns [52]; pHyPer-Cyto cDNA clone was purchased from Evrogen (Cat No. FP941); zebrafish catalase cDNA (CDS) was isolated from 2-day embryos according to catalase sequence (NM_130912); rat Dusp6 cDNA was obtained from Dr Zhi-Xin Wang [53]; and CFP-NTR was provided by Dr Ling-Fei Luo [35]. These expression constructs were cloned into the Tol2 backbone [50]. Each of the transgenic lines made in our laboratory was char - acterized and/or confirmed by more than two independent found - ers.\nMtz-induced embryonic heart injury in Tg(myl7:CFP-NTR) embryos A large number of cardiomyocytes were damaged by incubat - ing Tg( myl7 :CFP-NTR) transgenic embryos with Mtz essentially as previously described [35]. Briefly, transgenic embryos at 70 hpf were incubated with 2 mM Mtz, and Mtz was then washed off and DPI and/or H 2 O 2 were added at 72 hpf. In this condition, about 80% control transgenic hearts restored normal morphology and function at 124 hpf.\n\n\nAdult zebrafish heart resection\nA ventricular resection procedure was carried out essentially as described by Poss and colleagues [3]. Briefly, adult zebrafish were anesthetized with tricaine and placed with ventral side up into the groove of a sponge. The pericardial sac was exposed by removing surface scales and a small piece of skin with iridectomy scissors. Opening the pericardial sac was aided by tweezers, and the apex of the ventricle was gently pulled up and removed with Vannas scissors. A small piece of wet tissue was placed on the top of the ventricular wound. The heart stopped bleeding very quickly (within 10-15 s) due to the formation of blood clots. The fish was placed back into a water tank, where water was puffed over the gills with a plastic pipette until the fish was breathing and swimming regu - larly. The surface opening sealed automatically within a few days.\nLibrary preparation for a strand-specific poly(A)-positive RNA-seq Sham and 7 dpa hearts were freshly excised and rinsed with pre-cooled PBS. The outflow tract and atrium were removed im - mediately. The total RNA of each group was extracted from about 25 hearts by using QIAGEN RNeasy Microarray Tissue Mini Kit (Cat No. 73304). Poly(A)-positive RNA was purified from 5 µg of total RNA (RNA integrity number ≥ 7.5) with the Dynabeads mRNA purification kit (Invitrogen), following the manufacturer’s instructions. A strand-specific RNA-seq library was then prepared according to the deoxy-UTP approach as reported previously [54]. Amplified materials were loaded onto a flow cell and sequencing was carried out on the Illumina HiSeq2000 platform by running 90 cycles (paired-end design) according to the manufacturer’s instruc - tions.\nRNA-Seq and microarray data analysis and mining Paired-end mRNA sequence tags were mapped on the Zebrafish\nGenome (Zv9) by Tophat (v1.4.1). Multiple alignment reads were discarded. The RPKM scores for each gene were calculated as pre - viously described [55]. A series of Perl (v5.12.2) and R (v2.13.1) scripts were implemented to evaluate the quality of the RNA-seq experiments. Overall, high-quality RNA-seq data were obtained with high base quality, unbiased read distribution on transcripts, correct strand information, little DNA contamination and low mu - tation rate across the reads (Supplementary information, Figure S1). Eight hundred and forty-five upregulated genes between sham and 7 dpa hearts were identified using Cuffdiff (1.3.0) (Supple - mentary information, Figure S2), on the basis of three criteria: (1) P (un-corrected) &lt; 0.05; (2) fold change &gt; 2; and (3) RPKM &gt; 0.2 in at least one sample. Protein sequences of upregulated genes were then subjected to KOBAS (v2.0) to identify enriched biological pathways, with the P -value calculated using a hyper - geometric distribution. FDR correction was performed to adjust for multiple testing and pathways with FDR-corrected P -value &lt; 0.05 were considered statistically significantly enriched. The RNA-seq data entitled “Transcriptome analysis of heart regenera - tion in adult zebrafish” were deposited to the GEO repository with accession number GSE50203 (GSM1215619 for sham heart and GSM1215618 for 7 dpa heart).\nA list of previous known ROS genes were manually selected and evaluated for their relative expression levels between sham and 7 dpa hearts, based on a publicly available microarray of ze - brafish heart regeneration [29] and the above RNA-seq data on ze - brafish heart regeneration. We found that 23 of them were changed between sham and 7 dpa hearts in the microarray data that were expressed as log (7dpa_log2FC_Belmonte), and 32 of them were changed between sham and 7 dpa hearts in our RNA-seq data that were expressed as log (7dpa_logFC_RNAseq) in Supplementary information, Table S4. We wrote in-house R scripts to analyze the microarray data, using package Limma from Bioconductor [56].\n\n\nEx vivo intact heart imaging\nAdult Tg( myl7 :Hyper) hearts were removed and rinsed in mod - ified Tyrode’s solution (in mM: 150 NaCl, 5.4 KCl, 1.5 MgSO 4 , 0.4 NaH 2 PO 4 , 2 CaCl 2 , 10 glucose, 10 HEPES, pH 7.4). The atrium was removed to diminish spontaneous beatings. Hearts were viable for up to 1 h in such conditions, manifested by responses to 1 Hz electric field stimulation. Viability of the hearts was confirmed by their rhythmic contractions upon stimulation. A high speed camera was used to capture the contractions of the hearts and then kymo - graph was generated from the movies. Hearts were imaged under the 20× water objective lens of a Zeiss LSM700 upright confocal microscope with 405- and 488-nm lasers for dual excitation of Hyper, and 505-565 nm emission lights were collected with band- pass filter. For calculating Hyper 488/405 ratio, F 488 and F 405 im- ages were smoothed, background subtracted and divided. For the Hyper ratiometric calibration curve, Hyper transgenic hearts were treated and imaged ex vivo under designated concentrations (5, 10, 20, 50, 100 and 200 µM) of H 2 O 2 for 2 min. For time-lapse record - ings with 10 µM DPI or DMSO treatment, Hyper transgenic hearts were imaged at 30-s intervals for 3 min in normal Tyrode’s solu - tion before application of DPI or DMSO; the total recording time was 45 min. To detect redox signals with chemical indicators, 30 µM of Redox sensor cc-1 (R-14060; Invitrogen) was injected into the thoracic cavity of 14 dpa Tg( myl7 :Hyper) or wild-type zebraf - ish 30 min before heart isolation. Redox sensor cc-1 was excited at 555 nm. Image processing and data analysis were performed using\nthe IDL program (Research Systems).\n\n\nBrdU incorporation and small molecule treatment\nAfter ventricular resection, zebrafish were allowed to recover for 7 days and 50 µl of 2.5 mg/ml BrdU (B5002; Sigma) was then co-injected with 10 µM DPI (D2926; Sigma), 100 µM apocynin (W508454; Sigma), 10 µM BCI (B4313; Sigma), or 10 µM U0126 (U120; Sigma) into the thoracic cavity once daily until the hearts were collected. In pilot experiments, BCI was kindly provided by Dr Michael Tsang of the University of Pittsburg [40].\nWestern blotting and immunoprecipitation Zebrafish tissues were treated in cell lysis buffer (P0013; Bey - otime) in the presence of a protease inhibitor cocktail (P8340; Sig - ma) and PhosSTOP phosphatase inhibitor cocktail (04906845001; Roche). Proteins were quantified with a BCA kit (PA115; Tiangen Biotech), resolved on 12% SDS-PAGE gels, and transferred onto PVDF membranes (162-0177; Bio-Rad). Membranes were in - cubated with the appropriate primary antibody followed by the incubation with the corresponding HRP-conjugated secondary antibody, and proteins were detected with ECL western blotting substrates (W1001; Promega).\nHeLa cells were grown in DMEM high glucose medium (Ther - mo HyClone, SH30022.01B) containing 10% FBS (Thermo Hy- Clone, SV30087.02). The zDusp6 expression construct pcDNA4- zDusp6-Myc and the Flag-tagged ubiquitin expression construct pcDNA-Ub-Flag [57] were cotransfected into HeLa cells using Lipofectamine 2000 (Invitrogen, 11668). Cells were collected and lysed using RIPA buffer (Applygen, C1053) with 0.1 mM PMSF, 5 mM EDTA and 1 mM Complete TM protease inhibitors (Roche, 04693132001) in the presence or absence of 10 mM MG132 (Sig - ma, M7449). Cell lysates were then incubated with rabbit anti-Myc mAb (Cell Signaling, 2278) at 4 °C overnight. After incubation, 40 µl of Protein A+G beads (Abmart, A10001) was added to each sample and the mixture was rotated at 4 °C for 6 h. After centrif - ugation, the beads were washed five times with RIPA buffer and then boiled for 10 min with 30 µl of 2× sample buffer. Mouse an - ti-Flag antibody (Sigma, F1804) and mouse anti-Myc (HRP) mAb (Cowin Bioscience, CW0300) were used for immunoblotting.\nIn situ hybridization, immunostaining and histological methods Adult hearts were removed and fixed in 4% paraformaldehyde at room temperature for 2 h, and then went through an ethanol series for dehydration followed by paraffin embedding and sec - tioning (4 µm). Two duox cDNA clones were isolated from an embryonic zebrafish cDNA library made in Xiong’s lab, and the primer sequences are shown as duox 1F and 1R, or duox 2F and 2R in Supplementary information, Table S5. Duox clone 1 and duox clone 2 cDNAs were used to make two independent duox probe 1 (Supplementary information, Figure S3) and duox probe 2 (Supplementary information, Figure S5). Dusp6 cDNA was kindly provided by Dr Michael Tsang [40] and tcf21 cDNA by Dr Fabrizio Serluca. In situ hybridization on paraffin sections was performed using digoxygenin-labeled probes. Primary antibodies were anti-BrdU (B8434; Sigma), anti-Mef2c (sc-313; Santa Cruz), anti-tErk (4695; Cell Signaling), anti-pErk (9101; Cell Signaling), anti-tErk (MA5-15134; Pierce), anti-GFP (A-11122; Invitrogen), anti-GAPDH (sc-166545; Santa Cruz), anti-MF20 (Developmental\nStudies Hybridoma Bank), anti-Dusp6 (WH0001848M1; Sigma), anti-Lcp1 (124420; Genetex), anti-β-actin (4967; Cell Signal - ing) and anti-Raldh2 (P30011; Abmart). Secondary antibodies used for immunostaining were Alexa Fluor 488 goat anti-mouse IgG (A21121; Invitrogen), Alexa Fluor 488 goat anti-rabbit IgG (A11034; Invitrogen), Alexa Fluor 555 goat anti-mouse IgG (A21424; Invitrogen) and Alexa Fluor 555 goat anti-rabbit IgG (A21428; Invitrogen). Secondary antibodies for western blotting were HRP goat anti-rabbit IgG (sc-2004; Santa Cruz) and HRP goat anti-mouse IgG (sc-2005; Santa Cruz).\n\n\nMasson’s and AFOG staining\nFor AFOG staining, paraffin sections were incubated in Bouin’s solution (HT10132; Sigma) preheated at 56 °C for 2.5 h, incubated at room temperature for 1 h, briefly washed in tap water, incubated in 1% phosphomolybdic acid (P4869; Sigma) for 5 min, rinsed with distilled water and stained with AFOG staining solution (3 g of acid fuchsin (F8129; Sigma), 2 g of orange G (O3756, Sigma) and 1 g of anilin blue (AB0083; BBI) that were dissolved in 200 ml of acidified distilled water, pH 1.1) for 10 min. Sections were rinsed with distilled water, dehydrated with ethanol gradients and mounted. Myosin was visualized as orange, fibrin as red and colla - gen as bright blue.\nMasson’s staining was performed on paraffin sections using BASO Masson’s Trichrome Stain Kit (BA-4079A; Baso Diagnos - tics Inc). Myosin was visualized as red and collagen as bright blue.\nTransmission electron microscopy DMSO- and DPI-treated wild-type adult hearts, as well as Tg( myl7 :catalase-DsRed) transgenic adult hearts after injury at 14 dpa were fixed with glutaraldehyde at a final concentration of 2.5% (v/v) in 0.1 M sodium cacodylate buffer. Samples were rinsed and post-fixed with 1.0% OsO4. Next, the samples were embedded in the Spur resin and sectioned. The sections were stained with uranyl acetate and lead citrate and observed under a transmission electron microscope JEOL 1010 (JEOL Ltd., Japan).\n\n\nRT-PCR analysis\nTotal RNA from sham and injured hearts (15-20 hearts per sample) was extracted and purified using RNeasy@ mini kit (74106; Qiagen), and RT-PCR was performed with OneStep RT- PCR kit (210212; Qiagen) using 40 cycles on 50 ng of total RNA per reaction. Semi-quantitative RT-PCR of ERK1 and ERK2 was performed using 25 PCR cycles. Primer sequences were shown in Supplementary information, Table S5.\nStatistical analysis Statistical analysis was performed on relevant data using Stu - dent’s two-tailed t -test and the GraphPad Prism software package or SPSS18.0. Data were reported as mean ± SEM. For Supple - mentary information, Figure S12B and S12F, statistic significance between the two groups was determined with χ 2 -test. P &lt; 0.05 was considered statistically significant.\n\n\nAcknowledgments\nWe thank Drs RP Xiao, Y Rao, JR Peng, C Wei, K Ouyang, YM Wang, ZL Huang and JC Luo for critical comments on the manuscript, Dr IC Bruce for language editing, Dr Yingchun Hu\nfor help on transmission electromicroscopy, and the Pathology Core of Institute of Molecular Medicine for histology. We also thank Drs J Wang and KD Poss for their kind help in performing in situ hybridization with duox probes on Tg( tcf21 :nucEGFP) ventricles and Drs M Tsang, ZL Wen, T Evans, ZX Wang, LF Luo, GC Burns, J Chen, CM Cao and F Serluca for generous gifts of protocols, plasmid clones and/or transgenic zebrafish lines. This work was supported by the National Basic Research Program of China (2012CB944501, 2010CB529503 and 2013CB531200) and the National Natural Science Foundation of China (31271549, 31130067, 31000644 and 31221002).\n\n\nCell Research (2014) 24:1091-1107.\n\n\nPeidong Han et al . 1093 npg\n\n\nPeidong Han et al . 1095 npg\n\n\nPeidong Han et al . 1097 npg\n\n\nPeidong Han et al . 1099 npg\n\n\nPeidong Han et al . 1101 npg\n\n\nPeidong Han et al . 1103 npg\n\n\nCell Research | Vol 24 No 9 | September 2014\n\n\nPeidong Han et al . 1105 npg\n\n\nPeidong Han et al . 1107 npg\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [7.09s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"heart\",\n  \"tissue_name_structured\": \"heart\",\n  \"developmental_stage_text\": \"adult\",\n  \"developmental_stage_coarse\": \"adult\",\n  \"developmental_stage_fine\": \"adult\",\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"heart\",\n  \"tissue_name_structured\": \"heart\",\n  \"developmental_stage_text\": \"adult\",\n  \"developmental_stage_coarse\": \"adult\",\n  \"developmental_stage_fine\": \"adult\",\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 79,
      "prompt_tokens": 16426,
      "total_tokens": 16505
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_123d5a9f90"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "heart",
  "tissue_name_structured": "heart",
  "developmental_stage_text": "adult",
  "developmental_stage_coarse": "adult",
  "developmental_stage_fine": "adult",
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [7.10s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "heart",
  "tissue_name_structured": "heart",
  "developmental_stage_text": "adult",
  "developmental_stage_coarse": "adult",
  "developmental_stage_fine": "adult",
  "is_scRNA_data": false
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nThe unfolded protein response (UPR) acts in most cells to maintain homeostasis within the protein secretory pathway during physiological conditions. During stress, the UPR becomes further induced to mitigate the accumulation of misfolded or unfolded proteins in the endoplasmic reticulum (ER). If UPR activation cannot relieve the excess protein load, the ER becomes dilated and dysfunctional, and all branches of the UPR remain chronically activated in a condition referred to as ER stress. Many studies have implicated ER stress in a range of pathologies, and there is a clear association between UPR activation and metabolic diseases such as fatty liver disease (FLD) &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;-&lt;&lt;REF:Rinella-2011-54-1600&gt;&gt;. However, it is not known whether factors that control the UPR can also directly impact lipid metabolism and it remains unclear how UPR activation causes FLD.\nFLD is the most common hepatic pathology worldwide &lt;&lt;REF:???-?-?-?&gt;&gt;, and alcohol abuse is a leading cause. Even a single episode of binge drinking causes lipid accumulation in hepatocytes (steatosis) in over 90% of drinkers &lt;&lt;REF:???-?-?-?&gt;&gt;. While acute steatosis can resolve, chronic steatosis can render hepatocytes susceptible to damage and is a prerequisite step in developing more severe liver disease, including steatohepatitis and cirrhosis. Most FLD etiologies are accompanied by impairment of protein secretion by hepatocytes that result in serum protein deficiencies, which are most apparent in chronic alcoholics and contribute to the systemic complications of alcoholic liver disease (ALD). These defects are reflected in many reports demonstrating that alcohol induces some UPR sensors and targets in the liver of mice &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;, &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, rats &lt;&lt;REF:Shinohara-2010-51-796&gt;&gt;, micropigs &lt;&lt;REF:Esfandiari-2005-289-G54&gt;&gt;, and zebrafish &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, deleting ATF4 or CHOP, two key UPR effectors, reduces ethanol-induced liver injury &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, indicating that UPR activation can contribute to ALD. However, it is not clear whether these genes, or other UPR effectors, contribute to steatosis.\nThere are three main UPR sensors that function through shared and independent mechanisms to maintain ER function by enhancing protein processing and folding in the ER, and by promoting degradation of terminally misfolded secretory proteins. Activating transcription factor 6 (ATF6) and inositol-requiring enzyme 1-alpha (IRE1alpha or ERN1) pathways generate the active transcription factors nATF6 and XBP1, respectively, that induce hundreds of UPR target genes &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. PRKR-like endoplasmic reticulum kinase (PERK or EIF2AK3) phosphorylates EIF2A to repress translation &lt;&lt;REF:Harding-2000-5-897&gt;&gt; and to promote production of ATF4 &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;, which induces a subset of target genes &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;. Each of these primary sensors has been evaluated for its contribution to FLD caused by a robust ER stressor &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt;. ATF6 knockout mice fail to resolve steatosis caused by acute stress due to tunicamycin injection &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Yamamoto-2010-21-2975&gt;&gt;, suggesting that loss of ATF6 promotes fatty liver. Our previous work using zebrafish confirmed the finding that steatosis caused by acute stress is augmented by atf6 loss, and also demonstrates that steatosis caused by chronic stress is reduced when Atf6 is depleted &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it appears that atf6 loss can alternatively enhance or reduce FLD, depending on the nature and duration of the stress.\nThe relationship between metabolic disease and UPR activation is under intensive investigation, yet several important questions remain unanswered. First, is UPR activation a cause or consequence of FLD? While some studies indicate that lipotoxicity and fatty acid accumulation can induce the UPR &lt;&lt;REF:Wei-2006-291-E275&gt;&gt;, &lt;&lt;REF:Wang-2006-147-943&gt;&gt;, there is incontrovertible evidence that robust UPR activation is sufficient to induce steatosis &lt;&lt;REF:Lee-2012-211-29&gt;&gt;, &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;. These data are incorporated into a current model proposing that robust UPR activation can cause steatosis and, if the lipid burden is not resolved, this can further augment cellular stress and contribute to chronic UPR induction. The second question is which, if any, of the main UPR factors directly cause fatty liver? All studies to date that have addressed this question, including our work in zebrafish &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, utilize loss of function approaches whereby a key UPR gene is deleted and the effect on stress-induced steatosis is evaluated. However, these studies are complicated by the extensive crosstalk between UPR effectors, and by the ability of cells to adapt to changes in UPR capacity. Third, what is the mechanism by which UPR activation causes fatty liver? Activation of lipogenic genes by the sterol response element binding protein (SREBP) transcription factors has been implicated as an essential pathway in both alcoholic &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt; and non-alcoholic &lt;&lt;REF:Higuchi-2008-38-1122&gt;&gt;-&lt;&lt;REF:Zhang-2012-212-229&gt;&gt; FLD. Since SREBPs and ATF6 are activated by a shared proteolytic mechanism &lt;&lt;REF:Ye-2000-6-1355&gt;&gt;, it is possible that these two proteins are activated simultaneously in response to stress. However, the role of SREBPs in FLD caused by UPR activation has not been addressed.\nHere, we use genetic and genomic approaches in a well-established zebrafish model of ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;-&lt;&lt;REF:Liu-2013-6-1271&gt;&gt; to demonstrate that blocking Atf6 prevents alcohol-induced steatosis and that overexpression of the nuclear, active form of Atf6 (nAtf6) causes steatosis. We found that nAtf6 overexpression increases expression of genes in the lipogenic pathway and alters lipid flux to promote triglyceride synthesis by a mechanism that does not involve Srebps. Instead, we demonstrate an epistatic interaction between atf6 and fasn in driving alcoholic steatosis. These findings demonstrate the first clear and causative link between a central UPR sensor and FLD.\n\nResults\n\nAtf6 is required for alcoholic steatosis\nWe previously reported that transcription of multiple UPR effector genes and translation of the major ER chaperone, Bip, are induced in zebrafish livers following alcohol exposure &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. We have also demonstrated that xbp1 splicing, a direct measure of Ire1a activation, is only detectable within the first few hours of exposure to ethanol &lt;&lt;REF:???-?-?-?&gt;&gt; yet UPR target gene induction persists long afterwards &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, suggesting that the Ire1a pathway is not entirely responsible for the UPR in ALD. Our previous finding that Eif2a is phosphorylated in the liver of zebrafish with ALD &lt;&lt;REF:???-?-?-?&gt;&gt; suggests that Perk is activated by alcohol, but since Eif2a can also be phosphorylated by other kinases &lt;&lt;REF:Dever-1992-68-585&gt;&gt;, this has yet to be resolved. The lack of reliable antibodies that recognize the active form of Atf6, Perk and Ire1a in zebrafish hampered direct measurement of the activation status of each pathway (not shown). However, quantitative real-time PCR (qPCR) analysis of liver samples from 5 day post fertilization (dpf) larvae treated with alcohol demonstrated that atf6 mRNA was significantly induced in the liver (Figure 1A &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;) prior to the onset of steatosis (see Figure 1B &lt;&lt;FIG:pgen-1004335-g001&gt;&gt; and &lt;&lt;REF:???-?-?-?&gt;&gt;). Since Atf6 has been shown to act in a positive feedback loop to induce its own expression &lt;&lt;REF:Namba-2007-355-543&gt;&gt;, these data suggest that the increase in atf6 expression could reflect Atf6 activation in response to alcohol.\nFigure 1\nAtf6 is required for alcoholic steatosis.\n\nA: qPCR analysis of UPR sensors atf6, perk, ire1a, and atf4 in the livers of control (EtOH-) and 350 mM ethanol-treated (EtOH+) larvae. dCt values were calculated by normalization to rpp0. Fold changes were calculated based on median dCt values. Statistics: paired t-test. *, p&lt;0.05. B: Expression of atf6 in the livers of 350 mM ethanol treated larvae from 0-32 hours. Fold changes were determined by normalizing to control (0 mM EtOH) dCt values. Statistics: one-sample t-test. *, p&lt;0.05. C: Images of oil red O stained whole larvae injected with two different morpholinos targeting atf6 (ATG-targeting, atf6 ATG, and splice-blocking, atf6 SPL) and exposed to either 0 or 350 mM ethanol. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring oil red O stained larvae at 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored.\nTo test whether Atf6 was required for alcoholic steatosis, we injected a morpholino (MO) targeting either the translation initiation site (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; or the boundary between coding exon 1 and intron 1 of atf6 (atf6MO-SPL) (Table S1 and Figure S1A), which blocked splicing (Figure S1B) and introduced an early stop codon (Figure S1C). Morphants and uninjected controls were treated with 350 mM ethanol for 32 hours, stained with the neutral lipid dye, oil red O and scored for the presence or absence of steatosis (Figure 1C and S &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;2A). Consistent with previous findings &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;, the percent of larvae with steatosis (i.e. steatosis incidence) was significantly higher in uninjected larvae exposed to 350 mM ethanol for 32 hours (13% in untreated larvae vs. 53% in ethanol treated larvae; p&lt;0.05). atf6 morphants had reduced steatosis incidence (p&lt;0.05, Figure 1C-D &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;), but still displayed other gross morphological abnormalities caused by ethanol including increased liver circularity, an indication of hepatomegaly (Figure S2B-C), a common feature of liver disease. Thus, Atf6 is required for alcoholic steatosis.\n\nnAtf6 overexpression in hepatocytes induces robust UPR activation\nTo determine whether Atf6 is sufficient to cause FLD, we created a transgenic zebrafish line expressing the predicted nuclear, active fragment of zebrafish Atf6 (amino acids 1-366; Figure S3A) fused to mCherry under the hepatocyte-specific promoter fabp10, along with a cassette that expressed GFP in cardiomyocytes as a marker of transgenesis ((Tg(fabp10:nAtf6-cherry; cmlc2:GFP) hereafter called nAtf6 TG). There is high sequence identity between human and zebrafish Atf6 proteins in the transmembrane, leucine zipper and basic (bZIP) domains and the protease recognition sites are highly conserved (Figure S3A). To confirm its nuclear localization, the predicted nAtf6 fragment of the zebrafish protein was fused to GFP and transfected into HepG2 cells (Figure S3B).\nTransgenics were selected based on GFP expression in the heart, appeared grossly normal throughout development (Figure S3C), and developed into viable, fertile adults. nAtf6-mCherry was not detectable using low-resolution fluorescence microscopy in larvae from all four of the transgenic lines we generated. However, we confirmed transgene expression by PCR and blotting for mCherry (Figures S3D-E). atf6 mRNA expression in the liver was ~4 fold higher in nAtf6 TG larvae compared to wildtype (WT) larvae (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4) at 5 dpf, and was persistently elevated in older fish (Figure S4D and not shown). We detected mCherry mRNA in 5 dpf nAtf6 TG larvae by PCR, and Western blotting for mCherry detected the transgene in the liver of adult fish, albeit at much lower levels than achieved in transgenic fish expressing nuclear-localized mCherry without fusion to nAtf6 (i.e. Tg(fabp10:nls-mCherry); &lt;&lt;REF:???-?-?-?&gt;&gt;\nFigures S3D-E).\nFigure 2\nnAtf6 overexpression is sufficient to drive steatosis.\n\nA: Heatmap of upregulated UPR effector genes in nAtf6 transgenic larvae. The log values based on mRNA-Seq analysis and the median fold change in 3-6 liver samples assessed by qPCR at 5 dpf and 14 dpf. B: Most Atf6 targets are upregulated in the liver in response to ethanol. Genes identified as part of the UPR are significantly induced in the liver of nAtf6 TG and ethanol treated larvae (log value &gt;= 0.2; see Table S2). C: Oil red O stained WT and nAtf6 transgenic larvae. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring of whole mount oil red O stained larvae at 4, 5, and 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored. E: Hepatic triglyceride levels in extracts from pooled livers from WT and nAtf6 transgenic larvae at 5 dpf and 14 dpf, and from single adult livers (~9 months). Statistics: unpaired t-test. *, p&lt;0.05. Median fold changes are noted.\nWe confirmed that nAtf6 overexpression did not induce the other UPR branches by assessing xbp1 splicing (Figure S4A) and Eif2a phosphorylation (Figure S4B) in the liver of nAtf6 TG 5 dpf larvae. We found no difference in these markers in unstressed nAtf6 TG and WT larvae, however, xbp1 splicing was lower in transgenics than in WTs after exposure to stress (i.e. 1 microg/ml tunicamycin for 48 hours; Figure S4A). This suggests that nAtf6 overexpression adapts hepatocytes to withstand a robust, pharmacologically-induced ER stress.\nWe next examined the expression of UPR target genes in the liver of 5 dpf transgenic and control larvae. We found that bip (also called hspa5), a transcriptional target of Atf6, was elevated at the protein (Figure S4C) and mRNA levels (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D) in nAtf6 TG larvae. mRNA-Seq analysis comparing control larvae (Tg(fabp10:nls-mCherry); n = 2 pools of ~40-50 livers each; see &lt;&lt;REF:???-?-?-?&gt;&gt; and GEO dataset GSE52605), ethanol-treated (n = 2 pools of ~40-50 livers each), and nAtf6 TG larvae (n = 1 pool of ~40-50 livers) revealed 33 UPR target genes to be highly and significantly upregulated in nAtf6 TG livers (Tables S2-S4 and Figure 2A &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;; GEO dataset GSE56498). These genes were identified as ATF6 target genes in other studies &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; or by having the UPR or protein folding as a primary GO descriptor, and included ER chaperones (bip/hspa5, dnajc3, and grp94/hsp90b1), quality control effectors (canx, calret, calrl, and calrl2), protein disulfide isomerases (pdia3, pdia4) and ERAD components (edem1, derl1 and derl3). A subset of these genes was confirmed by qPCR at 5 dpf and at 14 dpf (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D-E). atf4 and derl1 were the only genes detected as significantly upregulated by qPCR but not by mRNA-Seq.\nAnalysis of the mRNA-Seq data from ethanol-treated larvae revealed a striking overlap between the \"UPR-ome\" induced by ethanol and by nAtf6 overexpression: of the 49 UPR target genes induced by ethanol, 30 were also induced in the liver of nAtf6 TG larvae (Figure 2B &lt;&lt;FIG:pgen-1004335-g002&gt;&gt; and Table S2). It is possible that the remaining 19 genes unique to the ethanol-induced transcriptome are attributed to the transient, early increase in xbp1 splicing &lt;&lt;REF:???-?-?-?&gt;&gt; or to other transcription factors that regulate their expression. Collectively, these data indicate that Atf6 is a major regulator of the UPR in ALD.\n\nAtf6 overexpression causes steatosis\nWe next asked whether nAtf6 was sufficient to cause FLD (Figure 2C-E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Induction of the fabp10:nAtf6-cherry transgene occurs at ~2.5 dpf &lt;&lt;REF:Her-2003-538-125&gt;&gt; and by 4 dpf, 44% of nAtf6 TG larvae developed steatosis which increased to 69% and 76% by 5 and 5.5 dpf, respectively (Figure 2C-D and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;5A) and persisted until at least 14 dpf (Figure S5A), but there were no marked histological abnormalities in the nAtf6 TG liver at 5 dpf (Figure S5B). Hepatic triglyceride (TAG) levels were nearly doubled at 5 dpf, and six times higher in 14 dpf nAtf6 TG larvae (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), clutch-to-clutch variability non-withstanding. However, TAG levels were the same in WT and nAtf6 TG adults (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), despite persistent expression of the transgene (Figure S3E and not shown). We speculate that, over time, animals adapt to nAtf6 overexpression or that other metabolic parameters affect hepatic lipid accumulation in adult fish. Together, our data demonstrate that Atf6 is necessary for alcoholic steatosis and sufficient to cause steatosis in the absence of any other stress.\n\nnAtf6 overexpression alters expression of genes involved in lipogenesis and lipid export\nSteatosis is caused by elevated TAGs in hepatocytes resulting from increased lipid synthesis or uptake, or from decreased lipid utilization or export. We therefore analyzed pathways relevant to these processes in the mRNA-Seq dataset from nAtf6 TG livers to determine if any were dysregulated. TAGs are generated from linking free fatty acids (FFAs) to a glycerol backbone, which is generated by conversion of dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P) by G3P dehydrogenase (GPD1) (see schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). We found genes that participate in TAG synthesis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;) or export (Figure S6) were dysregulated in nAtf6 TG livers at 5 dpf, albeit not to the same degree as observed for the UPR target genes (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). DHAP is generated via glycolysis or by glyceroneogenesis, a pathway that has significant overlap with gluconeogenesis (see &lt;&lt;REF:Nye-2008-283-27565&gt;&gt; and schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). mRNA-Seq analysis demonstrated that several genes involved in glycolysis were downregulated in nAtf6 TG livers including aldob, glut2/slc2a2, gck, pgk1, and pklr (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Genes promoting glyceroneogenesis, particularly pck1 and got1, which generates oxaloacetate for conversion to phosphoenoylpyruvate by pck1, were upregulated, as was gpd1b, which drives glycerol formation. This suggests that nAtf6 overexpression increases factors regulating glyceroneogenesis preferentially over those that promote glycolysis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). In addition, ech1 and tecrb, two genes responsible for fatty acid elongation in the mitochondria and ER, respectively, were induced in nAtf6 TG livers by mRNA-Seq (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Some of these genes were confirmed by qPCR analysis of nAtf6 TG livers, with the median fold change from qPCR on 3 to 6 clutches of livers from 5 and 14 dpf larvae depicted in Figure S7.\nFigure 3\nGlyceroneogenesis and fatty acid elongation pathways are dysregulated by nAtf6 overexpression.\nSchematic of the glycolysis, gluconeogenesis and glyceroneogenesis pathway (A, adapted from WikiPathways) and heatmap of associated genes (B). Upregulated genes are colored in shades of orange, downregulated genes are colored in shades of green, genes that are unchanged are white and those genes that did not appear in mRNA-Seq analyses are colored gray. Log values and median fold changes from mRNA-Seq analysis and qPCR, respectively, are noted.\nTAGs are assembled in the hepatocyte ER and secreted in association with lipoproteins that are required for transport in plasma. Several genes that regulate lipoprotein assembly and export, including apom, apoeb, apoea, cetp and zgc:162608 and zgc:194131, two apolipoprotein-like genes, were downregulated in nAtf6 TG livers (Figure S7 and Table S3). pdia2, which interacts with MTP to transport lipids out of hepatocytes &lt;&lt;REF:Wetterau-1997-1345-136&gt;&gt;, was also downregulated. These data suggest that hepatic lipid transport may be impaired by nAtf6 overexpression. While these expression data do not capture the complex metabolic and post-translational regulation of the proteins encoded by these genes, they do suggest enhanced lipogenesis or suppressed lipoprotein export as potential mechanisms of steatosis in nAtf6 transgenics.\n\n\nDe novo lipogenesis is enhanced by nAtf6 overexpression\nAcetate is a precursor to acetyl-CoA, the major building block of fatty acid and, therefore, TAG synthesis. To determine if nAtf6 overexpression increased de novo lipogenesis, we measured the incorporation of 14C-acetate into lipids in nAtf6 TG larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) and in HepG2 cells overexpressing nATF6 (Figure 4B-D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) compared to their respective controls. Larvae were incubated with 14C-acetate from 3-5 dpf and we measured the amount of radiolabel present in the lipid fraction compared to that present in the lysate of the whole larvae. In an average of 5 individual pools of larvae, more 14C radiolabel was detected in the lipid fraction of nAtf6 TG larvae than in wildtype larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;).\nFigure 4\n\nDe novo lipogenesis is enhanced in vitro and in vivo by nAtf6 overexpression.\n\nA:\n14C-acetate is preferentially incorporated into lipids in 5 dpf nAtf6 TG larvae compared to WT. The percent of 14C in lipid fraction was divided by 14C measured in the unextracted lysate from whole fish. The median fold change is noted; n =  number of samples of pooled larvae. B: Oil red O staining of HepG2 cells transfected with GFP or nATF6. Bar  = 10 microm. C: Quantification of lipid droplet number and area (square pixels). Statistics: unpaired t-test. *, p&lt;0.05; n indicates the number of cells quantified over 20 fields from 2 independent experiments. D: Incorporation of 14C-acetate into lipids by HepG2 cells transfected with GFP or nATF6 normalized to protein. The median fold change is noted; n =  number of separate batches of cells analyzed in 2 independent experiments.\nSince the liver in 5 dpf zebrafish larvae is too small to analyze incorporation in this organ exclusively, analysis of the whole larvae could not determine whether the label was preferentially incorporated into hepatocytes of nAtf6 TG larvae. We thus used human hepatoma (HepG2) cells transfected with human nATF6 to address this question. nATF6 overexpression causes accumulation of oil red O stained cytoplasmic droplets (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) that are both greater in number and larger in size than in cells transfected with GFP (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). These findings were confirmed in 293T cells (Figure S8) and correlate with our findings in nAtf6 TG larvae (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Thus, nAtf6 overexpression causes lipid accumulation across cell types and species. Moreover, in eleven independent cell samples from two separate experiments, HepG2 cells overexpressing nATF6 had slightly more 14C radiolabel in the lipid fraction compared to GFP-transfected cells (Figure 4D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). While we found that the incorporation tended to be higher in the nATF6 expressing cells, the average increase was moderate, at best. We attribute this to the transfection efficiency of HepG2 cells combined with experimental conditions used to label lipids, which likely act to select against nATF6 expressing cells.\n\nSrebp activation does not contribute to Atf6-mediated steatosis\nFatty acid synthase (fasn) mediates multiple steps in fatty acid synthesis that culminate in generation of palmitate from acetyl-CoA, a building block for TAG biosynthesis. fasn transcription in hepatocytes is primarily regulated by Srebp1c &lt;&lt;REF:Shimano-1997-100-2115&gt;&gt;-&lt;&lt;REF:Shimano-1996-98-1575&gt;&gt;, and we previously demonstrated that fasn transcripts and other Srebp target genes are induced in zebrafish with ALD &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, blocking Srebp activation by injecting a morpholino targeting the Srebp activating protein, Scap, reduced the incidence of alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Since Atf6 and Srebps are activated by similar mechanisms - they are both retained as inactive precursors in the ER and processed by the same Golgi-resident enzymes &lt;&lt;REF:Ye-2000-6-1355&gt;&gt; - Srebp mediated lipogenesis has been proposed as a mechanism by which UPR activation causes FLD &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;. We tested this using a genetic approach. First, we asked whether blocking Atf6 affected the induction of lipogenic gene expression in the liver of ethanol treated larvae. Both fasn (Figure 5A &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;), and acc1 (acaca;\nFigure S9A) were upregulated by ~2 fold in the liver of ethanol treated larvae, and this was blocked by atf6 MO injection (Figures 5A and S &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;9A). However, induction of other Srebp target genes was not blocked by Atf6 depletion, including srebp1 (Figure S9A) and the Srebp2 targets hmgcra, hmgcs1 and mvk (Figure S9B). Since the entire Srebp target gene set was not uniformly affected by atf6 loss, we conclude that Atf6 does not interact with Srebps. Instead, Atf6 may affect expression of specific lipogenic genes (i.e. fasn and acc1) by a mechanism distinct from Srebp1. Interestingly, we found mild induction of Srebp2 target genes in untreated atf6 morphants (Figure S9B), supporting conclusions based on experiments in mammalian cultured cells where ATF6 was found to directly suppress SREBP2 activity &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;.\nFigure 5\nFasn functions downstream of Atf6 to cause steatosis independent of Srebp activation.\n\nA: qPCR analysis of fasn expression in atf6 morphants and uninjected larvae treated with EtOH for 32 hours. B: qPCR analysis of fasn expression in nAtf6 TG larvae and HepG2 cells transfected with nATF6. Statistics: unpaired t-test. *, p&lt;0.05. C: Quantification of whole mount oil red O staining of nAtf6 TG larvae injected with scap MO. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored, respectively. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. D: Quantification of oil red O staining in nAtf6 TG larvae injected with fasn MO. E: Quantification of oil red O staining in atf6/fasn double morphants treated with 350 mM EtOH for 24 hours. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05.\nNext, we asked whether nAtf6 was sufficient to induce fasn and acc1 by assessing their expression in livers of nAtf6 TG larvae. Both fasn (p&lt;0.05; Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;) and acc1 (Figure S9C) were increased by ~2 fold at 5 dpf, and fasn expression was also higher at 14 dpf in nAtf6 TG larvae and in HepG2 cells overexpressing nATF6 (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). However, other Srebp1 and Srebp2 genes were not significantly induced by nAtf6 overexpression in nAtf6 TG larvae (Figure S9C-D) or in HepG2 cells overexpressing human nATF6 (not shown), indicating that nAtf6 does not activate the entire Srebp target gene program. We then tested whether nAtf6 functioned upstream of Srebps by assessing whether scap morpholino injection suppressed steatosis in nAtf6 TG larvae. While scap morphants are resistant to ethanol-induced steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt; we found no difference in steatosis incidence in nAtf6 transgenics (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Finally, to test whether atf6 and scap interacted to cause alcoholic steatosis we injected low concentrations of morpholinos targeting both genes and found that they did not synergize to suppress alcoholic steatosis (Figure S10). We thus conclude that lipogenic gene induction and steatosis caused by nAtf6 overexpression does not require Srebps.\n\nSteatosis in nAtf6 TG larvae requires fasn\n\nThe functional relevance of the finding that nAtf6 overexpression increases lipogenesis was tested by assessing the effect of morpholino-mediated fasn (Table S1) knockdown on steatosis incidence in nAtf6 TG larvae. Injection of high concentrations of fasn MO induced severe morphological defects, so we optimized the fasn MO concentration to have minimal toxicity. This had no effect on steatosis incidence in control larvae, but reduced steatosis incidence in nAtf6 TG larvae from 74% to 42% (Figure 5C &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Furthermore, we found that atf6 and fasn interacted epistatically to cause alcoholic steatosis by co-injecting concentrations of fasn and atf6-SPL MOs that did not have any effect on alcoholic steatosis on their own but, together, significantly reduced steatosis incidence in response to ethanol (Figure 5D &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Thus, Atf6 and Fasn function in the same genetic pathway. Taken together, our data suggest a model by which Atf6 causes steatosis, in part, by inducing fasn and TAG synthesis by a mechanism that is independent of Srebps (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;).\nFigure 6\nWorking model by which Atf6 functions as a positive regulator of alcoholic steatosis.\nGenetic tools used in this study are illustrated in red.\n\nDiscussion\nThere is a wealth of recent literature showing that the UPR is activated in FLD and that this aspect of the disease is conserved across species &lt;&lt;REF:Puri-2008-134-568&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:Kaplowitz-2006-21-S7&gt;&gt;-&lt;&lt;REF:Ye-2010-59-6&gt;&gt;. However, it has not been shown that any single UPR factor can directly cause this disease and the mechanism by which UPR activation causes lipid accumulation is unclear. Here, we use zebrafish genetics and biochemical analyses to show that Atf6 is both necessary and sufficient for steatosis by showing that: (i) Atf6 is required for alcoholic steatosis, (ii) activation of Atf6 is sufficient to cause steatosis, (iii) Atf6 activation induces expression of genes involved in glyceroneogenesis and fatty acid elongation and causes de novo lipogenesis, and (iv) Atf6 epistatically interacts with fatty acid synthase (fasn), a key enzyme involved in TAG biosynthesis, to cause FLD. This is among the first data to directly link a main UPR sensor and lipid metabolic pathways. \nA number of studies have demonstrated that loss of one of the key UPR sensors - PERK &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, IRE1alpha &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt; or ATF6 &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; - enhances steatosis caused by acute stress. Additionally, we previously demonstrated that Atf6 depletion suppresses steatosis caused by chronic stress &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it is clear that UPR activation is a conserved, common feature of FLD, but whether it functions to promote or prevent steatosis appears to depend on the nature and duration of the FLD-causing stress. Importantly, these loss of function studies have described a requirement for the UPR in FLD caused by tunicamycin injection, but not in the context of a stress that mirrors human conditions, such as obesity or alcohol abuse. Here, we show that Atf6 is induced in ALD, that its activation precedes steatosis, and that knocking down Atf6 reduces alcoholic steatosis. Thus, Atf6 is required for ALD. Moreover, since overexpression of nAtf6 is sufficient to cause FLD in the absence of any other stress, we conclude that this branch of the UPR is a pathophysiological mechanism of FLD.\nAtf6 and Xbp1 are the primary transcription factors that both independently and cooperatively regulate expression of hundreds of UPR target genes. We previously found significant upregulation of many UPR target genes in zebrafish with ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt; and hypothesized that since xbp1 splicing in the liver was only an early, transient response to alcohol &lt;&lt;REF:???-?-?-?&gt;&gt;, other transcription factors must participate in this robust UPR. Transcriptome analysis of the livers from nAtf6 TG and ethanol treated larvae enabled us to identify a set of target genes that are likely to be directly activated by Atf6. Many of these genes were also reported to be upregulated in mice overexpressing inducible ATF6 in the heart &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; and they were not induced by tunicamycin treatment of Atf6-/- MEFs &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. Genes occupying the intersection of these different datasets are likely bona fide Atf6 targets and the transgenic larvae we generated provides a system to test this directly. In contrast, many genes that were downregulated when ATF6 was overexpressed in the mouse heart were unchanged in nAtf6 transgenic zebrafish livers and, conversely, genes such as canx, derl1, and atf4 were upregulated in the nAtf6 TG zebrafish liver samples but not in the mouse model. The variations between the two datasets may be due to inherent differences in the models or approaches used to detect changes in gene expression, or may reflect the ability of Atf6 to differentially regulate target genes depending on cell type: hepatocytes possess a significantly higher secretory capacity than cardiomyocytes, and thus the Atf6 transcriptome in hepatocytes may be more extensive in order to maintain ER homeostasis. Finally, comparative transcriptome analysis between nAtf6 TG and ethanol treated larvae identified significant overlap between the UPR target genes in these two datasets, indicating that Atf6 is the main transcriptional driver of the ethanol-induced \"UPR-ome\".\nWhat is the mechanism by which the UPR causes FLD? Our data showing that nAtf6 overexpression induces the expression of some lipogenic enzymes and increases lipid synthesis, and that blocking fatty acid synthesis reduces FLD, fits a model (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;) whereby increased lipid production is, in part, the mechanism of steatosis in FLD caused by UPR activation. However, despite our finding that Atf6 depletion suppresses fasn expression and nAtf6 overexpression induces fasn, we have no evidence that fasn is a direct transcriptional target of Atf6. Indeed, the level of fasn induction is far less than the established Atf6 target, bip, and there are no canonical UPREs or ERSEs &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt; in the fasn promoter (not shown), thus other pathways are also likely at play. The mechanism by which nAtf6 induces fasn and acc1 require further investigation, and the tools we describe here will facilitate such studies.\nThe SREBP transcription factors are well-characterized regulators of hepatic lipogenesis; SREBP1c functions by increasing the expression of the full panel of genes required for TAG biogenesis &lt;&lt;REF:Horton-2003-100-12027&gt;&gt;. One possibility, suggested by the finding that the UPR and SREBPs are activated in parallel in some systems &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;, is that nAtf6 causes Srebp activation, leading to increased expression of fasn and other lipogenic genes and promoting lipogenesis. Our data argues against this: (i) the full panel of Srebp targets are not induced by nAtf6 overexpression in zebrafish, (ii) Atf6 and Srebps did not epistatically interact to modify alcoholic steatosis and (iii) Srebp activation is not required for steatosis in nAtf6 TG larvae. While it does appear that Srebp activation is required for alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt;, the current study shows that it is not the only important pathway that regulates hepatic lipid metabolism, as steatosis in nAtf6 TG larvae is independent of Srebps.\nWhile Srebps are not required for the effects of Atf6 on steatosis, there does appear to be some interaction between Atf6 and Srebp2, consistent with in vitro data showing that ATF6 suppresses SREBP2 &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;. We found that Atf6 causes upregulation of Srebp2 target genes, and while this modest increase in Srebp2 target genes in atf6 morphants did not appear to have a functional impact on cholesterol biogenesis, since atf6 morphants do not develop steatosis but are protected from it, it does suggest that Atf6 may suppress Srebp2 activity.\nWe used mRNA-Seq analyses to identify potential pathways that are dysregulated in response to Atf6 overexpression, although it is clear that the complex post-translational regulation of these pathways are not captured by gene expression studies. Our data suggested that lipoprotein export may be impaired by Atf6 overexpression, but we did not find that the flux of 14C-acetate into the extracellular lipids was impaired in cells overexpressing nATF6. While there are some technical caveats to these biochemical studies, we speculate that decreased lipoprotein export is not the only mechanism by which nAtf6 overexpression causes steatosis. Future studies to optimize cell conditions to ensure maximal cell viability and sustained high levels of nAtf6 expression will be required to fully address this.\nWhile steatosis is clearly a first step on the continuum to more severe liver disease, we also propose a different possibility: that lipid accumulation is a sign of an active and adaptive stress response. Protein folding in the ER is metabolically demanding, and when secretory cargo increases or the ER becomes full of unfolded proteins, increased lipid accumulation may enhance the ATP supply and thereby sustain the increased protein folding demands. Thus, the UPR may serve to enhance the protein folding capacity of the ER function in two ways: by activating genes required for protein folding, quality and export, and by promoting metabolic flux to TAGs as an energy source to withstand the challenge presented by a high level of secretory cargo. In this scenario, acute steatosis would not be a pathological response but, instead, would provide a protective role to sustain hepatocyte function during stress. However, if the stress response persists and steatosis becomes chronic, lipotoxicity could potentiate liver injury. Based on these data, it is tempting to speculate that enhancing protein folding in the ER by chemical chaperones, which have been used in both mouse models and humans to alleviate UPR activity, attenuate fatty liver disease and increase insulin sensitivity &lt;&lt;REF:Ozcan-2006-313-1137&gt;&gt;, &lt;&lt;REF:Kars-2010-59-1899&gt;&gt;, &lt;&lt;REF:Xiao-2011-60-918&gt;&gt;, could reducr the demand for energy and thus reduce the need to accumulate lipid. Whether Atf6 functions both as sensor of unfolded proteins and of metabolic demand remains to be elucidated.\n\nMaterials and Methods\n\nAnimal husbandry, treatments and transgenics\nAdult wildtype (WT, TAB14 and AB), Tg(fabp10:dsRed) &lt;&lt;REF:Dong-2007-39-397&gt;&gt; and Tg(fabp10:nls-mCherry)\n&lt;&lt;REF:???-?-?-?&gt;&gt; zebrafish were maintained according to standard conditions. Larvae were exposed to 350 mM ethanol (Pharmco-AAPER, Brookfield, CT) in fish water starting at 96-98 hours post fertilization (hpf) for up to 32 hours as described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;. All zebrafish protocols were approved by Mount Sinai's Institutional Animal Care and Use Committee.\nMorpholinos targeting the translation initiation ATG of atf6 (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, an atf6 intron-exon boundary (atf6MO-SPL, Figure S1), a scap intron-exon boundary &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, and a fasn intron-exon boundary were ordered from GeneTools (Philomath, OR). Approximately 1-5 pmol were injected into 1-2 cell stage embryos. Morpholino sequences and amount injected per embryo (ng) are listed in Table S1.\nThe Tg(fabp10:nAtf6-cherry; cmlc2:GFP) transgenic line was created by injecting a vector containing 2813 bp of the fabp10 promoter &lt;&lt;REF:Dong-2007-39-397&gt;&gt; upstream of the predicted nuclear fragment of zebrafish Atf6 (amino acids 1-366) as identified via DNA and protein alignments with human ATF6 (NCBI Reference Sequence: NP_031374.2). A cassette driving GFP expression in cardiomyocytes was included for rapid screening of transgenics (cmlc2:GFP). The transgene was flanked by Tol2 sites and the vector was injected into fertilized eggs along with transposase mRNA. Larvae were selected for cmlc2:GFP expression and raised to adulthood, outcrossed to TAB14 adults and 4 germline founders were identified.\n\nPlasmids\nThe nATF6-pcDNA-DEST47 plasmid was created using the Invitrogen Gateway System in which human nATF6 (amino acids 1-380) was amplified from a construct containing the full ORF (pEGFP-hATF6, from Dr. Aguirre-Ghiso) and ligated into pcDNA-DEST47. The nAtf6-GFP/pCI-Neo plasmid was generated by amplifying zebrafish nAtf6-GFP from a Tol2-generated plasmid (fabp10:nAtf6-GFP) with primers containing restriction sites for EcoRI and SmaI. The resulting PCR product was ligated into pCI-Neo via T4 ligase. pEGFP-C1 (Clontech) was used as a positive control for transfection.\n\nCell culture and transfection\n293T and HepG2 cells were grown in 100 mm dishes (BD Biosciences) with DMEM (Corning Cellgro, Manassas, VA) containing 10% fetal bovine serum (Invitrogen) and penicillin/streptomycin (Cellgro), and housed in a humidified incubator at 37 C with 5% CO2. Cells were passaged at 80-90% confluency either into 100 mm dishes or 6 well culture plates (Corning). Each well was transfected with 1-2 microg plasmid using Xtreme GENE 9 (Roche) or Lipofectamine &lt;&lt;REF:???-?-?-?&gt;&gt;  (Invitrogen) for 24 hours. Following transfection, cells were collected for RNA extraction using TRIzol (Invitrogen), used for oil red O staining, or labeled with 14C-acetate as described below.\n\n\n14C-acetate radiolabeling\nHepG2 cells were washed with PBS 24-28 hours after transfection and kept in serum free high glucose DMEM (Corning Cellgro, Manassas, VA) containing penicillin/streptomycin (Cellgro), 100 nM insulin (Lilly USA, Indianapolis, IN) and 0.5 microCi 14C-acetate (PerkinElmer, Waltham, MA) for 17 hours. Media was harvested and spun to remove dead cells, while cells were scraped, washed twice and lysed in cold PBS. Protein concentration in the cell lysate was measured with a BCA protein quantification kit (Thermo Scientific, Waltham, MA). Fractions of the media and cell lysate portion were used for lipid extraction as described below.\nWT and nATF6 5 dpf larvae were labeled with 1-5 microCi 14C-acetate for 48 hours. Larvae were washed twice and then sonicated in cold PBS. Protein concentration in the larval lysate was measured with a BCA protein quantification kit (Thermo Scientific). Fractions of the larval lysate were used for lipid extraction as described below.\n\nLipid extractions\nLipids from media, cells and 5 dpf larvae were extracted according to Bligh and Dyer &lt;&lt;REF:Bligh-1959-37-911&gt;&gt; with the following modifications. Briefly, methanol and chloroform were added to 400 microl media or 50 microl of 100 microl total cell lysate in a ratio of 2:1. Samples were vortexed and chloroform and 0.45% NaCl were added at a ratio of 1:1, vortexed again and phases were separated by centrifugation. The aqueous phase was re-extracted with one part chloroform and combined with the lipid from the first second extraction, washed with methanol and 0.45% NaCl at a 1:1 ratio, centrifuged and the lipid phase was recovered and dried under a stream of N2 gas. Lipids were reconstituted in a 2:1 mixture of chloroform and methanol (Thermo Scientific, Waltham, MA), dissolved in Ultima Gold and counted using a 2460 MicroBeta2 LumiJET (PerkinElmer, Waltham, MA) liquid scintillation counter. CCPM were normalized to protein concentration to account for variations in cell number.\n\nQuantitative real-time PCR (qPCR)\nTotal RNA isolated using TRIzol (Invitrogen) from pools of 15-20 dissected livers was reverse-transcribed using qScript cDNA SuperMix (Quanta Biosciences, Gaithersburg, MD). qPCR using A Light Cycler 480 (Roche) with PerfeCTa SYBRGreen FastMix (Quanta Biosciences) was used as previously described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;. Values for target gene were normalized to reference gene rpp0, and dCts calculated using the comparative threshold method (dCt  = 2-(Ct, gene - Ct, rpp0)). Primer sequences are listed in Table S1.\n\nTriglyceride measurements\nApproximately 20 livers from control and nAtf6 transgenic larvae were dissected and lysed in 0.5% Triton-X 100 and heated at 65degreesC for 5 minutes to inactivate hepatic lipases. Triglycerides were measured using the Infinity Triglyceride Liquid Stable Reagent (Thermo Fisher Scientific, Waltham, MA) following manufacturer's instructions, and were normalized to total protein concentration as determined by Bradford Assay (Bio-Rad, Hercules, CA).\n\nOil red O staining and quantification\nLarvae were fixed in 4% paraformaldehyde (PFA; Electron Microscopy Sciences, Hatfield, PA) overnight at 4degreesC, stained with oil red O, and scored for steatosis as previously described &lt;&lt;REF:Howarth-2013-10-199&gt;&gt;. Cryosections were stained by immersing slides in increasing concentrations of propylene glycol (85%, 100%) for 10 minutes followed by an overnight incubation in oil red O (0.5% in propylene glycol, Polysciences, Warrington, PA). Excess oil red O was removed the next day by sequential washes in 100% and 85% propylene glycol for 5 minutes. Nuclei were counterstained with hematoxylin.\nHepG2 and 293T cells were stained as previously described &lt;&lt;REF:Wu-2010-402-116&gt;&gt; and counterstained with hematoxylin. Oil red O droplet area and number were calculated using ImageJ. In brief, images were split into the red, green, and blue channels and the green channel was further processed for quantification as described, as oil red O has an excitation at 510 nm &lt;&lt;REF:Narasimhan-2011-7-e1001377&gt;&gt;. Following this, the background was subtracted from each image to ensure only counting of oil red O droplets. Droplets equal or greater than five square pixels were counted and area quantified. The values were copied into Microsoft Excel and the average droplet area and number of droplets per nucleus were counted for each field. Twenty independent fields from nATF6 and GFP-transfected HepG2 and 293T cells were imaged at 60x magnification.\n\nRNA sequencing (mRNA-Seq)\nTotal RNA was isolated using TRIzol (Invitrogen) from pools of ~40-50 livers dissected on 5 dpf. Two clutches of Tg(fabp10:nls-mCherry) larvae that were either untreated (control) or treated with 350 mM ethanol for 24 hours and one clutch of nAtf6 TG larvae were used. polyA-tailed mRNA was selected using oligo-dT beads and then fragmented. cDNAs were created using random-hexamers and ligated with bar-coded adaptors compatible with HiSeq &lt;&lt;REF:???-?-?-?&gt;&gt; . Single-end, 100 bp reads were sequenced at the Genomics Core of the Icahn School of Medicine at Mount Sinai. Custom-built software was used to map the reads to the zebrafish genome (Zv9/DanRer7) and estimate the coverage of each gene. Briefly, the reads were split into three 32 bp parts after trimming 2 bp at each end and mapped to the genome using a suffix-array based approach. The median of coverage across the transcript was used as an estimate of gene expression. The expression values were quantile normalized and log-ratios were calculated by comparing nAtf6 TGs to the average of the controls. Each ethanol treated sample was compared to its paired control (untreated siblings). Unique Gene Ontology terms (GO terms) were assigned to each gene by ranking the GO terms by relevance to the biology of the response, and using annotations from the human orthologues if the zebrafish annotations were lacking.\nThe distribution of expression values were plotted to identify the peak in the distribution, which is the level of noise in the system. The values were regularized by adding the noise to each gene's expression level before the log-ratios were calculated. This ensures that genes with low expression do not contribute to list of genes with large fold-changes. An absolute natural log ratio of 0.2 was used as the cutoff (known, non-responding genes are all below this threshold). The list of genes that show changes above this cutoff were analyzed for pathway enrichment using GO terms annotated as described above.\nThis data is available via the Gene Expression Omnibus (GEO) at accession number GSE56498. The controls for this dataset, untreated 5 dpf Tg(fabp10:nls-mCherry), also serve as controls in GEO dataset GSE52605 &lt;&lt;REF:???-?-?-?&gt;&gt; and were included in both sets for ease of comparison between genotypes and treatments.\n\nImage processing and graphics\nImages were cropped and minimally processed using Adobe Photoshop CS4 (Adobe Systems, San Jose, CA). Graphs were plotted using Prism 5.0c (GraphPad Software Inc., La Jolla, CA). Heat maps for mRNA-Seq data were generated using GENE-E (Broad Institute, Cambridge, MA). The global maximum and minimum for each gene set was set to orange and green, respectively, and zero was set to white. Metabolic pathway schematics were adapted from WikiPathways (URL: www.wikipathways.org).\n\nStatistical analyses\nStatistical tests were performed using GraphPad QuickCalcs (GraphPad Software, URL: http://www.graphpad.com/quickcalcs/index.cfm). For oil red O staining of whole larvae, chi-square with Fisher's Exact test were performed. For qPCR, radiolabeling, and oil red O staining of HepG2 and 293T cells we performed unpaired and one-sample t-tests as appropriate.\n\nSupplementary methods\nMethods for histological analysis, cryosectioning and staining, and liver circularity analysis are listed in Text S1.\n\nSupporting Information\nFigure S1\nInjection of an atf6 splice-blocking morpholino induces mis-splicing of atf6 mRNA. A: Schematic of boundary between intron 1 and exon 1 of zebrafish atf6. Exon sequence is shown in black and intron sequence is shown in blue, with the morpholino shown in red. The start codon is highlighted in yellow. B: Gel electrophoresis of atf6 morphants at 2 dpf showing additional bands compared to uninjected controls. C: Sequencing results from atf6 morphants displaying inclusion of intronic sequence (highlighted in blue). The predicted protein translation is shown, and early stop codons are noted as red asterisks.\n(TIF)\nClick here for additional data file.\nFigure S2\nKnockdown of atf6 does not confer resistance to morphological defects from alcohol exposure. A: Oil red O staining of cryosections from uninjected and atf6 morphant larvae treated with 0 or 350 mM ethanol for 32 hours. B: Live images of uninjected and atf6 morphant Tg(fabp10:dsRed) larvae treated with 0 or 350 mM ethanol for 32 hours. C: Quantification of liver circularity, a measure of hepatomegaly, following ethanol treatment in uninjected and atf6 morphant larvae. Livers were traced and circularity quantified in ImageJ.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nTg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae express the nAtf6-cherry transgene despite a lack of visual fluorescence. A: Schematic of human ATF6 transactivation (TAD), DNA-binding (bZIP) and transmembrane (TM) domains, cleavage sites for S1P (MBTPS1) and S2P (MBTPS2), and protein alignment with zebrafish Atf6. Numbers in the diagram correspond to amino acid positions in human ATF6. Numbers at the cleavage sites in the protein alignment correspond to amino acid positions in zebrafish Atf6. B: Confirmation of nuclear localization of zebrafish nAtf6 by transfection of nAtf6-GFP/pCI-Neo into HepG2 cells. C: Live image of Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larva at 5 dpf. D: Agarose gel electrophoresis showing expression of Cherry mRNA in nAtf6 TG but not WT larvae. rpp0 was used to ensure PCR efficacy. E: Western blot visualization of Cherry protein in adult nAtf6 transgenic zebrafish liver tissue. Tg(fabp10:nls-cherry) adult liver protein was used as a positive control. Actin was used as a loading control.\n(TIF)\nClick here for additional data file.\nFigure S4\nOverexpression of nAtf6 induces robust transcription of UPR target genes but does not activate Ire1a or Perk. A: Agarose gel electrophoresis visualization of xbp1 splicing in WT and nAtf6 TG larvae in the presence and absence of tunicamycin (Tm). The percentage of xbp1 splicing was quantified by measuring band intensity in ImageJ. B: Western blot visualization of Eif2a phosphorylation in livers of 5 dpf WT and nAtf6 TG larvae. Tunicamycin (1 microg/mL, 3-5 dpf) treated WT larvae were used as a positive control for immunoblotting. Fold was calculated over 2-3 separate experiments as noted. C: Western blot visualization of Bip protein in livers of 5 dpf WT and nAtf6 TG larvae. Fold change (below blot) was calculated from 3 separate experiments. D and E: Quantitative real-time PCR of multiple UPR effectors in the livers of WT and nAtf6 TG larvae at 5 (D) and 14 (E) dpf. Statistics: unpaired t-test. *, p&lt;0.05.\n(TIF)\nClick here for additional data file.\nFigure S5\nLipid accumulation is evident in nAtf6 TG larvae in the absence of other hepatic injury. A: Staining of cryosections with oil red O showing significant steatosis in nAtf6 TG larvae at 5 and 14 dpf. B: Hematoxylin and eosin staining of nAtf6 TG and control larvae at 5 dpf. No other alterations in hepatic pathology were observed. The number of larvae used is indicated.\n(TIF)\nClick here for additional data file.\nFigure S6\nmRNA-Seq reveals potential disruptions in lipid transport by nAtf6 overexpression. Heatmap of genes associated with lipid binding and transport (A) and basic schematic of lipid synthesis and lipoprotein circulation (B, adapted from WikiPathways). Upregulated genes are colored in shades of orange, and downregulated genes are colored in shades of green. Genes that are unchanged are colored white. Genes that did not appear in mRNA-Seq analyses are colored gray in the schematic.\n(TIF)\nClick here for additional data file.\nFigure S7\nnAtf6 overexpression induces transcription of genes involved in glyceroneogenesis. Quantitative, real-time PCR analysis of gene expression from WT and nAtf6 TG larvae at 5 dpf (A) and 14 dpf (B). Statistics: unpaired t-test. *, p&lt;0.05. For (A), \"n\" corresponds to the number of clutches used and approximately 8-12 livers were dissected for RNA extraction. For (B), individual livers were dissected for analysis.\n(TIF)\nClick here for additional data file.\nFigure S8\nOverexpression of nATF6 in 293T cells drives lipid accumulation. A: Staining of GFP and nATF6 transfected cells with oil red O. B: Quantification of lipid droplet number and area (square pixels). Droplet number and area were calculated using ImageJ. Statistics: unpaired t-test. *, p&lt;0.05. The number of cells used for quantification is noted.\n(TIF)\nClick here for additional data file.\nFigure S9\nSrebp1 and Srebp2 target genes are not dysregulated by nAtf6 overexpression. A and B: qPCR analysis of Srebp1 (A) and Srebp2 (B) target genes in atf6 morphants treated with 350 mM EtOH for 32 hours. Fold changes were calculated based on median dCt values. C and D: qPCR analysis of Srebp1 (C) and Srebp2 (D) target genes in Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae at 5 dpf. Fold changes were calculated based on median dCt values, and \"n\" corresponds to the number of clutches used.\n(TIF)\nClick here for additional data file.\nFigure S10\nAtf6 and Srebps do not epistically interact to prevent alcoholic steatosis. Steatosis incidence in larvae injected with atf6, scap, or atf6 + scap MO. \"Total n\" corresponds to the number of larvae scored. \"Clutch n\" corresponds to the number of clutches analyzed. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05 and corresponds to groups significantly different from uninjected controls. b, p&lt;0.05 and corresponds to ethanol-treated groups significantly different from uninjected, ethanol treated larvae.\n(TIF)\nClick here for additional data file.\nTable S1\nMorpholino and primer sequences. A: Morpholino sequences. Amount of morpholino injected is based on the assumption that 4 nl of MO is injected per embryo. B: Primer sequences used for conventional and quantitative, real-time PCR.\n(DOCX)\nClick here for additional data file.\nTable S2\nUPR effectors upregulated by nAtf6 and by ethanol. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Log values &gt;= 0.2 are highlighted in red.\n(XLSX)\nClick here for additional data file.\nTable S3\nUpregulated and downregulated genes in the liver of nAtf6 TG 5 dpf larvae. A: Upregulated genes in the liver of nAtf6 TG 5 dpf zebrafish larvae. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. UPR target genes are highlighted in pink. GO terms were assigned as described in the Materials and Methods. B: Downregulated genes in the liver of nAtf6 TG 5 dpf larvae. Genes regulating lipoprotein secretion are highlighted in orange.\n(XLSX)\nClick here for additional data file.\nTable S4\nRaw data from mRNA-Seq. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Genes are organized from highest to lowest log value for nAtf6 TG larvae.\n(XLS)\nClick here for additional data file.\nText S1\nSupplementary methods.\n(DOC)\nClick here for additional data file.",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nThe unfolded protein response (UPR) acts in most cells to maintain homeostasis within the protein secretory pathway during physiological conditions. During stress, the UPR becomes further induced to mitigate the accumulation of misfolded or unfolded proteins in the endoplasmic reticulum (ER). If UPR activation cannot relieve the excess protein load, the ER becomes dilated and dysfunctional, and all branches of the UPR remain chronically activated in a condition referred to as ER stress. Many studies have implicated ER stress in a range of pathologies, and there is a clear association between UPR activation and metabolic diseases such as fatty liver disease (FLD) &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;-&lt;&lt;REF:Rinella-2011-54-1600&gt;&gt;. However, it is not known whether factors that control the UPR can also directly impact lipid metabolism and it remains unclear how UPR activation causes FLD.\nFLD is the most common hepatic pathology worldwide &lt;&lt;REF:???-?-?-?&gt;&gt;, and alcohol abuse is a leading cause. Even a single episode of binge drinking causes lipid accumulation in hepatocytes (steatosis) in over 90% of drinkers &lt;&lt;REF:???-?-?-?&gt;&gt;. While acute steatosis can resolve, chronic steatosis can render hepatocytes susceptible to damage and is a prerequisite step in developing more severe liver disease, including steatohepatitis and cirrhosis. Most FLD etiologies are accompanied by impairment of protein secretion by hepatocytes that result in serum protein deficiencies, which are most apparent in chronic alcoholics and contribute to the systemic complications of alcoholic liver disease (ALD). These defects are reflected in many reports demonstrating that alcohol induces some UPR sensors and targets in the liver of mice &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;, &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, rats &lt;&lt;REF:Shinohara-2010-51-796&gt;&gt;, micropigs &lt;&lt;REF:Esfandiari-2005-289-G54&gt;&gt;, and zebrafish &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, deleting ATF4 or CHOP, two key UPR effectors, reduces ethanol-induced liver injury &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, indicating that UPR activation can contribute to ALD. However, it is not clear whether these genes, or other UPR effectors, contribute to steatosis.\nThere are three main UPR sensors that function through shared and independent mechanisms to maintain ER function by enhancing protein processing and folding in the ER, and by promoting degradation of terminally misfolded secretory proteins. Activating transcription factor 6 (ATF6) and inositol-requiring enzyme 1-alpha (IRE1alpha or ERN1) pathways generate the active transcription factors nATF6 and XBP1, respectively, that induce hundreds of UPR target genes &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. PRKR-like endoplasmic reticulum kinase (PERK or EIF2AK3) phosphorylates EIF2A to repress translation &lt;&lt;REF:Harding-2000-5-897&gt;&gt; and to promote production of ATF4 &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;, which induces a subset of target genes &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;. Each of these primary sensors has been evaluated for its contribution to FLD caused by a robust ER stressor &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt;. ATF6 knockout mice fail to resolve steatosis caused by acute stress due to tunicamycin injection &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Yamamoto-2010-21-2975&gt;&gt;, suggesting that loss of ATF6 promotes fatty liver. Our previous work using zebrafish confirmed the finding that steatosis caused by acute stress is augmented by atf6 loss, and also demonstrates that steatosis caused by chronic stress is reduced when Atf6 is depleted &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it appears that atf6 loss can alternatively enhance or reduce FLD, depending on the nature and duration of the stress.\nThe relationship between metabolic disease and UPR activation is under intensive investigation, yet several important questions remain unanswered. First, is UPR activation a cause or consequence of FLD? While some studies indicate that lipotoxicity and fatty acid accumulation can induce the UPR &lt;&lt;REF:Wei-2006-291-E275&gt;&gt;, &lt;&lt;REF:Wang-2006-147-943&gt;&gt;, there is incontrovertible evidence that robust UPR activation is sufficient to induce steatosis &lt;&lt;REF:Lee-2012-211-29&gt;&gt;, &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;. These data are incorporated into a current model proposing that robust UPR activation can cause steatosis and, if the lipid burden is not resolved, this can further augment cellular stress and contribute to chronic UPR induction. The second question is which, if any, of the main UPR factors directly cause fatty liver? All studies to date that have addressed this question, including our work in zebrafish &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, utilize loss of function approaches whereby a key UPR gene is deleted and the effect on stress-induced steatosis is evaluated. However, these studies are complicated by the extensive crosstalk between UPR effectors, and by the ability of cells to adapt to changes in UPR capacity. Third, what is the mechanism by which UPR activation causes fatty liver? Activation of lipogenic genes by the sterol response element binding protein (SREBP) transcription factors has been implicated as an essential pathway in both alcoholic &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt; and non-alcoholic &lt;&lt;REF:Higuchi-2008-38-1122&gt;&gt;-&lt;&lt;REF:Zhang-2012-212-229&gt;&gt; FLD. Since SREBPs and ATF6 are activated by a shared proteolytic mechanism &lt;&lt;REF:Ye-2000-6-1355&gt;&gt;, it is possible that these two proteins are activated simultaneously in response to stress. However, the role of SREBPs in FLD caused by UPR activation has not been addressed.\nHere, we use genetic and genomic approaches in a well-established zebrafish model of ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;-&lt;&lt;REF:Liu-2013-6-1271&gt;&gt; to demonstrate that blocking Atf6 prevents alcohol-induced steatosis and that overexpression of the nuclear, active form of Atf6 (nAtf6) causes steatosis. We found that nAtf6 overexpression increases expression of genes in the lipogenic pathway and alters lipid flux to promote triglyceride synthesis by a mechanism that does not involve Srebps. Instead, we demonstrate an epistatic interaction between atf6 and fasn in driving alcoholic steatosis. These findings demonstrate the first clear and causative link between a central UPR sensor and FLD.\n\nResults\n\nAtf6 is required for alcoholic steatosis\nWe previously reported that transcription of multiple UPR effector genes and translation of the major ER chaperone, Bip, are induced in zebrafish livers following alcohol exposure &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. We have also demonstrated that xbp1 splicing, a direct measure of Ire1a activation, is only detectable within the first few hours of exposure to ethanol &lt;&lt;REF:???-?-?-?&gt;&gt; yet UPR target gene induction persists long afterwards &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, suggesting that the Ire1a pathway is not entirely responsible for the UPR in ALD. Our previous finding that Eif2a is phosphorylated in the liver of zebrafish with ALD &lt;&lt;REF:???-?-?-?&gt;&gt; suggests that Perk is activated by alcohol, but since Eif2a can also be phosphorylated by other kinases &lt;&lt;REF:Dever-1992-68-585&gt;&gt;, this has yet to be resolved. The lack of reliable antibodies that recognize the active form of Atf6, Perk and Ire1a in zebrafish hampered direct measurement of the activation status of each pathway (not shown). However, quantitative real-time PCR (qPCR) analysis of liver samples from 5 day post fertilization (dpf) larvae treated with alcohol demonstrated that atf6 mRNA was significantly induced in the liver (Figure 1A &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;) prior to the onset of steatosis (see Figure 1B &lt;&lt;FIG:pgen-1004335-g001&gt;&gt; and &lt;&lt;REF:???-?-?-?&gt;&gt;). Since Atf6 has been shown to act in a positive feedback loop to induce its own expression &lt;&lt;REF:Namba-2007-355-543&gt;&gt;, these data suggest that the increase in atf6 expression could reflect Atf6 activation in response to alcohol.\nFigure 1\nAtf6 is required for alcoholic steatosis.\n\nA: qPCR analysis of UPR sensors atf6, perk, ire1a, and atf4 in the livers of control (EtOH-) and 350 mM ethanol-treated (EtOH+) larvae. dCt values were calculated by normalization to rpp0. Fold changes were calculated based on median dCt values. Statistics: paired t-test. *, p&lt;0.05. B: Expression of atf6 in the livers of 350 mM ethanol treated larvae from 0-32 hours. Fold changes were determined by normalizing to control (0 mM EtOH) dCt values. Statistics: one-sample t-test. *, p&lt;0.05. C: Images of oil red O stained whole larvae injected with two different morpholinos targeting atf6 (ATG-targeting, atf6 ATG, and splice-blocking, atf6 SPL) and exposed to either 0 or 350 mM ethanol. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring oil red O stained larvae at 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored.\nTo test whether Atf6 was required for alcoholic steatosis, we injected a morpholino (MO) targeting either the translation initiation site (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; or the boundary between coding exon 1 and intron 1 of atf6 (atf6MO-SPL) (Table S1 and Figure S1A), which blocked splicing (Figure S1B) and introduced an early stop codon (Figure S1C). Morphants and uninjected controls were treated with 350 mM ethanol for 32 hours, stained with the neutral lipid dye, oil red O and scored for the presence or absence of steatosis (Figure 1C and S &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;2A). Consistent with previous findings &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;, the percent of larvae with steatosis (i.e. steatosis incidence) was significantly higher in uninjected larvae exposed to 350 mM ethanol for 32 hours (13% in untreated larvae vs. 53% in ethanol treated larvae; p&lt;0.05). atf6 morphants had reduced steatosis incidence (p&lt;0.05, Figure 1C-D &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;), but still displayed other gross morphological abnormalities caused by ethanol including increased liver circularity, an indication of hepatomegaly (Figure S2B-C), a common feature of liver disease. Thus, Atf6 is required for alcoholic steatosis.\n\nnAtf6 overexpression in hepatocytes induces robust UPR activation\nTo determine whether Atf6 is sufficient to cause FLD, we created a transgenic zebrafish line expressing the predicted nuclear, active fragment of zebrafish Atf6 (amino acids 1-366; Figure S3A) fused to mCherry under the hepatocyte-specific promoter fabp10, along with a cassette that expressed GFP in cardiomyocytes as a marker of transgenesis ((Tg(fabp10:nAtf6-cherry; cmlc2:GFP) hereafter called nAtf6 TG). There is high sequence identity between human and zebrafish Atf6 proteins in the transmembrane, leucine zipper and basic (bZIP) domains and the protease recognition sites are highly conserved (Figure S3A). To confirm its nuclear localization, the predicted nAtf6 fragment of the zebrafish protein was fused to GFP and transfected into HepG2 cells (Figure S3B).\nTransgenics were selected based on GFP expression in the heart, appeared grossly normal throughout development (Figure S3C), and developed into viable, fertile adults. nAtf6-mCherry was not detectable using low-resolution fluorescence microscopy in larvae from all four of the transgenic lines we generated. However, we confirmed transgene expression by PCR and blotting for mCherry (Figures S3D-E). atf6 mRNA expression in the liver was ~4 fold higher in nAtf6 TG larvae compared to wildtype (WT) larvae (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4) at 5 dpf, and was persistently elevated in older fish (Figure S4D and not shown). We detected mCherry mRNA in 5 dpf nAtf6 TG larvae by PCR, and Western blotting for mCherry detected the transgene in the liver of adult fish, albeit at much lower levels than achieved in transgenic fish expressing nuclear-localized mCherry without fusion to nAtf6 (i.e. Tg(fabp10:nls-mCherry); &lt;&lt;REF:???-?-?-?&gt;&gt;\nFigures S3D-E).\nFigure 2\nnAtf6 overexpression is sufficient to drive steatosis.\n\nA: Heatmap of upregulated UPR effector genes in nAtf6 transgenic larvae. The log values based on mRNA-Seq analysis and the median fold change in 3-6 liver samples assessed by qPCR at 5 dpf and 14 dpf. B: Most Atf6 targets are upregulated in the liver in response to ethanol. Genes identified as part of the UPR are significantly induced in the liver of nAtf6 TG and ethanol treated larvae (log value &gt;= 0.2; see Table S2). C: Oil red O stained WT and nAtf6 transgenic larvae. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring of whole mount oil red O stained larvae at 4, 5, and 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored. E: Hepatic triglyceride levels in extracts from pooled livers from WT and nAtf6 transgenic larvae at 5 dpf and 14 dpf, and from single adult livers (~9 months). Statistics: unpaired t-test. *, p&lt;0.05. Median fold changes are noted.\nWe confirmed that nAtf6 overexpression did not induce the other UPR branches by assessing xbp1 splicing (Figure S4A) and Eif2a phosphorylation (Figure S4B) in the liver of nAtf6 TG 5 dpf larvae. We found no difference in these markers in unstressed nAtf6 TG and WT larvae, however, xbp1 splicing was lower in transgenics than in WTs after exposure to stress (i.e. 1 microg/ml tunicamycin for 48 hours; Figure S4A). This suggests that nAtf6 overexpression adapts hepatocytes to withstand a robust, pharmacologically-induced ER stress.\nWe next examined the expression of UPR target genes in the liver of 5 dpf transgenic and control larvae. We found that bip (also called hspa5), a transcriptional target of Atf6, was elevated at the protein (Figure S4C) and mRNA levels (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D) in nAtf6 TG larvae. mRNA-Seq analysis comparing control larvae (Tg(fabp10:nls-mCherry); n = 2 pools of ~40-50 livers each; see &lt;&lt;REF:???-?-?-?&gt;&gt; and GEO dataset GSE52605), ethanol-treated (n = 2 pools of ~40-50 livers each), and nAtf6 TG larvae (n = 1 pool of ~40-50 livers) revealed 33 UPR target genes to be highly and significantly upregulated in nAtf6 TG livers (Tables S2-S4 and Figure 2A &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;; GEO dataset GSE56498). These genes were identified as ATF6 target genes in other studies &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; or by having the UPR or protein folding as a primary GO descriptor, and included ER chaperones (bip/hspa5, dnajc3, and grp94/hsp90b1), quality control effectors (canx, calret, calrl, and calrl2), protein disulfide isomerases (pdia3, pdia4) and ERAD components (edem1, derl1 and derl3). A subset of these genes was confirmed by qPCR at 5 dpf and at 14 dpf (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D-E). atf4 and derl1 were the only genes detected as significantly upregulated by qPCR but not by mRNA-Seq.\nAnalysis of the mRNA-Seq data from ethanol-treated larvae revealed a striking overlap between the \"UPR-ome\" induced by ethanol and by nAtf6 overexpression: of the 49 UPR target genes induced by ethanol, 30 were also induced in the liver of nAtf6 TG larvae (Figure 2B &lt;&lt;FIG:pgen-1004335-g002&gt;&gt; and Table S2). It is possible that the remaining 19 genes unique to the ethanol-induced transcriptome are attributed to the transient, early increase in xbp1 splicing &lt;&lt;REF:???-?-?-?&gt;&gt; or to other transcription factors that regulate their expression. Collectively, these data indicate that Atf6 is a major regulator of the UPR in ALD.\n\nAtf6 overexpression causes steatosis\nWe next asked whether nAtf6 was sufficient to cause FLD (Figure 2C-E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Induction of the fabp10:nAtf6-cherry transgene occurs at ~2.5 dpf &lt;&lt;REF:Her-2003-538-125&gt;&gt; and by 4 dpf, 44% of nAtf6 TG larvae developed steatosis which increased to 69% and 76% by 5 and 5.5 dpf, respectively (Figure 2C-D and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;5A) and persisted until at least 14 dpf (Figure S5A), but there were no marked histological abnormalities in the nAtf6 TG liver at 5 dpf (Figure S5B). Hepatic triglyceride (TAG) levels were nearly doubled at 5 dpf, and six times higher in 14 dpf nAtf6 TG larvae (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), clutch-to-clutch variability non-withstanding. However, TAG levels were the same in WT and nAtf6 TG adults (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), despite persistent expression of the transgene (Figure S3E and not shown). We speculate that, over time, animals adapt to nAtf6 overexpression or that other metabolic parameters affect hepatic lipid accumulation in adult fish. Together, our data demonstrate that Atf6 is necessary for alcoholic steatosis and sufficient to cause steatosis in the absence of any other stress.\n\nnAtf6 overexpression alters expression of genes involved in lipogenesis and lipid export\nSteatosis is caused by elevated TAGs in hepatocytes resulting from increased lipid synthesis or uptake, or from decreased lipid utilization or export. We therefore analyzed pathways relevant to these processes in the mRNA-Seq dataset from nAtf6 TG livers to determine if any were dysregulated. TAGs are generated from linking free fatty acids (FFAs) to a glycerol backbone, which is generated by conversion of dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P) by G3P dehydrogenase (GPD1) (see schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). We found genes that participate in TAG synthesis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;) or export (Figure S6) were dysregulated in nAtf6 TG livers at 5 dpf, albeit not to the same degree as observed for the UPR target genes (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). DHAP is generated via glycolysis or by glyceroneogenesis, a pathway that has significant overlap with gluconeogenesis (see &lt;&lt;REF:Nye-2008-283-27565&gt;&gt; and schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). mRNA-Seq analysis demonstrated that several genes involved in glycolysis were downregulated in nAtf6 TG livers including aldob, glut2/slc2a2, gck, pgk1, and pklr (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Genes promoting glyceroneogenesis, particularly pck1 and got1, which generates oxaloacetate for conversion to phosphoenoylpyruvate by pck1, were upregulated, as was gpd1b, which drives glycerol formation. This suggests that nAtf6 overexpression increases factors regulating glyceroneogenesis preferentially over those that promote glycolysis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). In addition, ech1 and tecrb, two genes responsible for fatty acid elongation in the mitochondria and ER, respectively, were induced in nAtf6 TG livers by mRNA-Seq (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Some of these genes were confirmed by qPCR analysis of nAtf6 TG livers, with the median fold change from qPCR on 3 to 6 clutches of livers from 5 and 14 dpf larvae depicted in Figure S7.\nFigure 3\nGlyceroneogenesis and fatty acid elongation pathways are dysregulated by nAtf6 overexpression.\nSchematic of the glycolysis, gluconeogenesis and glyceroneogenesis pathway (A, adapted from WikiPathways) and heatmap of associated genes (B). Upregulated genes are colored in shades of orange, downregulated genes are colored in shades of green, genes that are unchanged are white and those genes that did not appear in mRNA-Seq analyses are colored gray. Log values and median fold changes from mRNA-Seq analysis and qPCR, respectively, are noted.\nTAGs are assembled in the hepatocyte ER and secreted in association with lipoproteins that are required for transport in plasma. Several genes that regulate lipoprotein assembly and export, including apom, apoeb, apoea, cetp and zgc:162608 and zgc:194131, two apolipoprotein-like genes, were downregulated in nAtf6 TG livers (Figure S7 and Table S3). pdia2, which interacts with MTP to transport lipids out of hepatocytes &lt;&lt;REF:Wetterau-1997-1345-136&gt;&gt;, was also downregulated. These data suggest that hepatic lipid transport may be impaired by nAtf6 overexpression. While these expression data do not capture the complex metabolic and post-translational regulation of the proteins encoded by these genes, they do suggest enhanced lipogenesis or suppressed lipoprotein export as potential mechanisms of steatosis in nAtf6 transgenics.\n\n\nDe novo lipogenesis is enhanced by nAtf6 overexpression\nAcetate is a precursor to acetyl-CoA, the major building block of fatty acid and, therefore, TAG synthesis. To determine if nAtf6 overexpression increased de novo lipogenesis, we measured the incorporation of 14C-acetate into lipids in nAtf6 TG larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) and in HepG2 cells overexpressing nATF6 (Figure 4B-D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) compared to their respective controls. Larvae were incubated with 14C-acetate from 3-5 dpf and we measured the amount of radiolabel present in the lipid fraction compared to that present in the lysate of the whole larvae. In an average of 5 individual pools of larvae, more 14C radiolabel was detected in the lipid fraction of nAtf6 TG larvae than in wildtype larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;).\nFigure 4\n\nDe novo lipogenesis is enhanced in vitro and in vivo by nAtf6 overexpression.\n\nA:\n14C-acetate is preferentially incorporated into lipids in 5 dpf nAtf6 TG larvae compared to WT. The percent of 14C in lipid fraction was divided by 14C measured in the unextracted lysate from whole fish. The median fold change is noted; n =  number of samples of pooled larvae. B: Oil red O staining of HepG2 cells transfected with GFP or nATF6. Bar  = 10 microm. C: Quantification of lipid droplet number and area (square pixels). Statistics: unpaired t-test. *, p&lt;0.05; n indicates the number of cells quantified over 20 fields from 2 independent experiments. D: Incorporation of 14C-acetate into lipids by HepG2 cells transfected with GFP or nATF6 normalized to protein. The median fold change is noted; n =  number of separate batches of cells analyzed in 2 independent experiments.\nSince the liver in 5 dpf zebrafish larvae is too small to analyze incorporation in this organ exclusively, analysis of the whole larvae could not determine whether the label was preferentially incorporated into hepatocytes of nAtf6 TG larvae. We thus used human hepatoma (HepG2) cells transfected with human nATF6 to address this question. nATF6 overexpression causes accumulation of oil red O stained cytoplasmic droplets (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) that are both greater in number and larger in size than in cells transfected with GFP (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). These findings were confirmed in 293T cells (Figure S8) and correlate with our findings in nAtf6 TG larvae (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Thus, nAtf6 overexpression causes lipid accumulation across cell types and species. Moreover, in eleven independent cell samples from two separate experiments, HepG2 cells overexpressing nATF6 had slightly more 14C radiolabel in the lipid fraction compared to GFP-transfected cells (Figure 4D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). While we found that the incorporation tended to be higher in the nATF6 expressing cells, the average increase was moderate, at best. We attribute this to the transfection efficiency of HepG2 cells combined with experimental conditions used to label lipids, which likely act to select against nATF6 expressing cells.\n\nSrebp activation does not contribute to Atf6-mediated steatosis\nFatty acid synthase (fasn) mediates multiple steps in fatty acid synthesis that culminate in generation of palmitate from acetyl-CoA, a building block for TAG biosynthesis. fasn transcription in hepatocytes is primarily regulated by Srebp1c &lt;&lt;REF:Shimano-1997-100-2115&gt;&gt;-&lt;&lt;REF:Shimano-1996-98-1575&gt;&gt;, and we previously demonstrated that fasn transcripts and other Srebp target genes are induced in zebrafish with ALD &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, blocking Srebp activation by injecting a morpholino targeting the Srebp activating protein, Scap, reduced the incidence of alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Since Atf6 and Srebps are activated by similar mechanisms - they are both retained as inactive precursors in the ER and processed by the same Golgi-resident enzymes &lt;&lt;REF:Ye-2000-6-1355&gt;&gt; - Srebp mediated lipogenesis has been proposed as a mechanism by which UPR activation causes FLD &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;. We tested this using a genetic approach. First, we asked whether blocking Atf6 affected the induction of lipogenic gene expression in the liver of ethanol treated larvae. Both fasn (Figure 5A &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;), and acc1 (acaca;\nFigure S9A) were upregulated by ~2 fold in the liver of ethanol treated larvae, and this was blocked by atf6 MO injection (Figures 5A and S &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;9A). However, induction of other Srebp target genes was not blocked by Atf6 depletion, including srebp1 (Figure S9A) and the Srebp2 targets hmgcra, hmgcs1 and mvk (Figure S9B). Since the entire Srebp target gene set was not uniformly affected by atf6 loss, we conclude that Atf6 does not interact with Srebps. Instead, Atf6 may affect expression of specific lipogenic genes (i.e. fasn and acc1) by a mechanism distinct from Srebp1. Interestingly, we found mild induction of Srebp2 target genes in untreated atf6 morphants (Figure S9B), supporting conclusions based on experiments in mammalian cultured cells where ATF6 was found to directly suppress SREBP2 activity &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;.\nFigure 5\nFasn functions downstream of Atf6 to cause steatosis independent of Srebp activation.\n\nA: qPCR analysis of fasn expression in atf6 morphants and uninjected larvae treated with EtOH for 32 hours. B: qPCR analysis of fasn expression in nAtf6 TG larvae and HepG2 cells transfected with nATF6. Statistics: unpaired t-test. *, p&lt;0.05. C: Quantification of whole mount oil red O staining of nAtf6 TG larvae injected with scap MO. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored, respectively. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. D: Quantification of oil red O staining in nAtf6 TG larvae injected with fasn MO. E: Quantification of oil red O staining in atf6/fasn double morphants treated with 350 mM EtOH for 24 hours. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05.\nNext, we asked whether nAtf6 was sufficient to induce fasn and acc1 by assessing their expression in livers of nAtf6 TG larvae. Both fasn (p&lt;0.05; Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;) and acc1 (Figure S9C) were increased by ~2 fold at 5 dpf, and fasn expression was also higher at 14 dpf in nAtf6 TG larvae and in HepG2 cells overexpressing nATF6 (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). However, other Srebp1 and Srebp2 genes were not significantly induced by nAtf6 overexpression in nAtf6 TG larvae (Figure S9C-D) or in HepG2 cells overexpressing human nATF6 (not shown), indicating that nAtf6 does not activate the entire Srebp target gene program. We then tested whether nAtf6 functioned upstream of Srebps by assessing whether scap morpholino injection suppressed steatosis in nAtf6 TG larvae. While scap morphants are resistant to ethanol-induced steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt; we found no difference in steatosis incidence in nAtf6 transgenics (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Finally, to test whether atf6 and scap interacted to cause alcoholic steatosis we injected low concentrations of morpholinos targeting both genes and found that they did not synergize to suppress alcoholic steatosis (Figure S10). We thus conclude that lipogenic gene induction and steatosis caused by nAtf6 overexpression does not require Srebps.\n\nSteatosis in nAtf6 TG larvae requires fasn\n\nThe functional relevance of the finding that nAtf6 overexpression increases lipogenesis was tested by assessing the effect of morpholino-mediated fasn (Table S1) knockdown on steatosis incidence in nAtf6 TG larvae. Injection of high concentrations of fasn MO induced severe morphological defects, so we optimized the fasn MO concentration to have minimal toxicity. This had no effect on steatosis incidence in control larvae, but reduced steatosis incidence in nAtf6 TG larvae from 74% to 42% (Figure 5C &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Furthermore, we found that atf6 and fasn interacted epistatically to cause alcoholic steatosis by co-injecting concentrations of fasn and atf6-SPL MOs that did not have any effect on alcoholic steatosis on their own but, together, significantly reduced steatosis incidence in response to ethanol (Figure 5D &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Thus, Atf6 and Fasn function in the same genetic pathway. Taken together, our data suggest a model by which Atf6 causes steatosis, in part, by inducing fasn and TAG synthesis by a mechanism that is independent of Srebps (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;).\nFigure 6\nWorking model by which Atf6 functions as a positive regulator of alcoholic steatosis.\nGenetic tools used in this study are illustrated in red.\n\nDiscussion\nThere is a wealth of recent literature showing that the UPR is activated in FLD and that this aspect of the disease is conserved across species &lt;&lt;REF:Puri-2008-134-568&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:Kaplowitz-2006-21-S7&gt;&gt;-&lt;&lt;REF:Ye-2010-59-6&gt;&gt;. However, it has not been shown that any single UPR factor can directly cause this disease and the mechanism by which UPR activation causes lipid accumulation is unclear. Here, we use zebrafish genetics and biochemical analyses to show that Atf6 is both necessary and sufficient for steatosis by showing that: (i) Atf6 is required for alcoholic steatosis, (ii) activation of Atf6 is sufficient to cause steatosis, (iii) Atf6 activation induces expression of genes involved in glyceroneogenesis and fatty acid elongation and causes de novo lipogenesis, and (iv) Atf6 epistatically interacts with fatty acid synthase (fasn), a key enzyme involved in TAG biosynthesis, to cause FLD. This is among the first data to directly link a main UPR sensor and lipid metabolic pathways. \nA number of studies have demonstrated that loss of one of the key UPR sensors - PERK &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, IRE1alpha &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt; or ATF6 &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; - enhances steatosis caused by acute stress. Additionally, we previously demonstrated that Atf6 depletion suppresses steatosis caused by chronic stress &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it is clear that UPR activation is a conserved, common feature of FLD, but whether it functions to promote or prevent steatosis appears to depend on the nature and duration of the FLD-causing stress. Importantly, these loss of function studies have described a requirement for the UPR in FLD caused by tunicamycin injection, but not in the context of a stress that mirrors human conditions, such as obesity or alcohol abuse. Here, we show that Atf6 is induced in ALD, that its activation precedes steatosis, and that knocking down Atf6 reduces alcoholic steatosis. Thus, Atf6 is required for ALD. Moreover, since overexpression of nAtf6 is sufficient to cause FLD in the absence of any other stress, we conclude that this branch of the UPR is a pathophysiological mechanism of FLD.\nAtf6 and Xbp1 are the primary transcription factors that both independently and cooperatively regulate expression of hundreds of UPR target genes. We previously found significant upregulation of many UPR target genes in zebrafish with ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt; and hypothesized that since xbp1 splicing in the liver was only an early, transient response to alcohol &lt;&lt;REF:???-?-?-?&gt;&gt;, other transcription factors must participate in this robust UPR. Transcriptome analysis of the livers from nAtf6 TG and ethanol treated larvae enabled us to identify a set of target genes that are likely to be directly activated by Atf6. Many of these genes were also reported to be upregulated in mice overexpressing inducible ATF6 in the heart &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; and they were not induced by tunicamycin treatment of Atf6-/- MEFs &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. Genes occupying the intersection of these different datasets are likely bona fide Atf6 targets and the transgenic larvae we generated provides a system to test this directly. In contrast, many genes that were downregulated when ATF6 was overexpressed in the mouse heart were unchanged in nAtf6 transgenic zebrafish livers and, conversely, genes such as canx, derl1, and atf4 were upregulated in the nAtf6 TG zebrafish liver samples but not in the mouse model. The variations between the two datasets may be due to inherent differences in the models or approaches used to detect changes in gene expression, or may reflect the ability of Atf6 to differentially regulate target genes depending on cell type: hepatocytes possess a significantly higher secretory capacity than cardiomyocytes, and thus the Atf6 transcriptome in hepatocytes may be more extensive in order to maintain ER homeostasis. Finally, comparative transcriptome analysis between nAtf6 TG and ethanol treated larvae identified significant overlap between the UPR target genes in these two datasets, indicating that Atf6 is the main transcriptional driver of the ethanol-induced \"UPR-ome\".\nWhat is the mechanism by which the UPR causes FLD? Our data showing that nAtf6 overexpression induces the expression of some lipogenic enzymes and increases lipid synthesis, and that blocking fatty acid synthesis reduces FLD, fits a model (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;) whereby increased lipid production is, in part, the mechanism of steatosis in FLD caused by UPR activation. However, despite our finding that Atf6 depletion suppresses fasn expression and nAtf6 overexpression induces fasn, we have no evidence that fasn is a direct transcriptional target of Atf6. Indeed, the level of fasn induction is far less than the established Atf6 target, bip, and there are no canonical UPREs or ERSEs &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt; in the fasn promoter (not shown), thus other pathways are also likely at play. The mechanism by which nAtf6 induces fasn and acc1 require further investigation, and the tools we describe here will facilitate such studies.\nThe SREBP transcription factors are well-characterized regulators of hepatic lipogenesis; SREBP1c functions by increasing the expression of the full panel of genes required for TAG biogenesis &lt;&lt;REF:Horton-2003-100-12027&gt;&gt;. One possibility, suggested by the finding that the UPR and SREBPs are activated in parallel in some systems &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;, is that nAtf6 causes Srebp activation, leading to increased expression of fasn and other lipogenic genes and promoting lipogenesis. Our data argues against this: (i) the full panel of Srebp targets are not induced by nAtf6 overexpression in zebrafish, (ii) Atf6 and Srebps did not epistatically interact to modify alcoholic steatosis and (iii) Srebp activation is not required for steatosis in nAtf6 TG larvae. While it does appear that Srebp activation is required for alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt;, the current study shows that it is not the only important pathway that regulates hepatic lipid metabolism, as steatosis in nAtf6 TG larvae is independent of Srebps.\nWhile Srebps are not required for the effects of Atf6 on steatosis, there does appear to be some interaction between Atf6 and Srebp2, consistent with in vitro data showing that ATF6 suppresses SREBP2 &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;. We found that Atf6 causes upregulation of Srebp2 target genes, and while this modest increase in Srebp2 target genes in atf6 morphants did not appear to have a functional impact on cholesterol biogenesis, since atf6 morphants do not develop steatosis but are protected from it, it does suggest that Atf6 may suppress Srebp2 activity.\nWe used mRNA-Seq analyses to identify potential pathways that are dysregulated in response to Atf6 overexpression, although it is clear that the complex post-translational regulation of these pathways are not captured by gene expression studies. Our data suggested that lipoprotein export may be impaired by Atf6 overexpression, but we did not find that the flux of 14C-acetate into the extracellular lipids was impaired in cells overexpressing nATF6. While there are some technical caveats to these biochemical studies, we speculate that decreased lipoprotein export is not the only mechanism by which nAtf6 overexpression causes steatosis. Future studies to optimize cell conditions to ensure maximal cell viability and sustained high levels of nAtf6 expression will be required to fully address this.\nWhile steatosis is clearly a first step on the continuum to more severe liver disease, we also propose a different possibility: that lipid accumulation is a sign of an active and adaptive stress response. Protein folding in the ER is metabolically demanding, and when secretory cargo increases or the ER becomes full of unfolded proteins, increased lipid accumulation may enhance the ATP supply and thereby sustain the increased protein folding demands. Thus, the UPR may serve to enhance the protein folding capacity of the ER function in two ways: by activating genes required for protein folding, quality and export, and by promoting metabolic flux to TAGs as an energy source to withstand the challenge presented by a high level of secretory cargo. In this scenario, acute steatosis would not be a pathological response but, instead, would provide a protective role to sustain hepatocyte function during stress. However, if the stress response persists and steatosis becomes chronic, lipotoxicity could potentiate liver injury. Based on these data, it is tempting to speculate that enhancing protein folding in the ER by chemical chaperones, which have been used in both mouse models and humans to alleviate UPR activity, attenuate fatty liver disease and increase insulin sensitivity &lt;&lt;REF:Ozcan-2006-313-1137&gt;&gt;, &lt;&lt;REF:Kars-2010-59-1899&gt;&gt;, &lt;&lt;REF:Xiao-2011-60-918&gt;&gt;, could reducr the demand for energy and thus reduce the need to accumulate lipid. Whether Atf6 functions both as sensor of unfolded proteins and of metabolic demand remains to be elucidated.\n\nMaterials and Methods\n\nAnimal husbandry, treatments and transgenics\nAdult wildtype (WT, TAB14 and AB), Tg(fabp10:dsRed) &lt;&lt;REF:Dong-2007-39-397&gt;&gt; and Tg(fabp10:nls-mCherry)\n&lt;&lt;REF:???-?-?-?&gt;&gt; zebrafish were maintained according to standard conditions. Larvae were exposed to 350 mM ethanol (Pharmco-AAPER, Brookfield, CT) in fish water starting at 96-98 hours post fertilization (hpf) for up to 32 hours as described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;. All zebrafish protocols were approved by Mount Sinai's Institutional Animal Care and Use Committee.\nMorpholinos targeting the translation initiation ATG of atf6 (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, an atf6 intron-exon boundary (atf6MO-SPL, Figure S1), a scap intron-exon boundary &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, and a fasn intron-exon boundary were ordered from GeneTools (Philomath, OR). Approximately 1-5 pmol were injected into 1-2 cell stage embryos. Morpholino sequences and amount injected per embryo (ng) are listed in Table S1.\nThe Tg(fabp10:nAtf6-cherry; cmlc2:GFP) transgenic line was created by injecting a vector containing 2813 bp of the fabp10 promoter &lt;&lt;REF:Dong-2007-39-397&gt;&gt; upstream of the predicted nuclear fragment of zebrafish Atf6 (amino acids 1-366) as identified via DNA and protein alignments with human ATF6 (NCBI Reference Sequence: NP_031374.2). A cassette driving GFP expression in cardiomyocytes was included for rapid screening of transgenics (cmlc2:GFP). The transgene was flanked by Tol2 sites and the vector was injected into fertilized eggs along with transposase mRNA. Larvae were selected for cmlc2:GFP expression and raised to adulthood, outcrossed to TAB14 adults and 4 germline founders were identified.\n\nPlasmids\nThe nATF6-pcDNA-DEST47 plasmid was created using the Invitrogen Gateway System in which human nATF6 (amino acids 1-380) was amplified from a construct containing the full ORF (pEGFP-hATF6, from Dr. Aguirre-Ghiso) and ligated into pcDNA-DEST47. The nAtf6-GFP/pCI-Neo plasmid was generated by amplifying zebrafish nAtf6-GFP from a Tol2-generated plasmid (fabp10:nAtf6-GFP) with primers containing restriction sites for EcoRI and SmaI. The resulting PCR product was ligated into pCI-Neo via T4 ligase. pEGFP-C1 (Clontech) was used as a positive control for transfection.\n\nCell culture and transfection\n293T and HepG2 cells were grown in 100 mm dishes (BD Biosciences) with DMEM (Corning Cellgro, Manassas, VA) containing 10% fetal bovine serum (Invitrogen) and penicillin/streptomycin (Cellgro), and housed in a humidified incubator at 37 C with 5% CO2. Cells were passaged at 80-90% confluency either into 100 mm dishes or 6 well culture plates (Corning). Each well was transfected with 1-2 microg plasmid using Xtreme GENE 9 (Roche) or Lipofectamine &lt;&lt;REF:???-?-?-?&gt;&gt;  (Invitrogen) for 24 hours. Following transfection, cells were collected for RNA extraction using TRIzol (Invitrogen), used for oil red O staining, or labeled with 14C-acetate as described below.\n\n\n14C-acetate radiolabeling\nHepG2 cells were washed with PBS 24-28 hours after transfection and kept in serum free high glucose DMEM (Corning Cellgro, Manassas, VA) containing penicillin/streptomycin (Cellgro), 100 nM insulin (Lilly USA, Indianapolis, IN) and 0.5 microCi 14C-acetate (PerkinElmer, Waltham, MA) for 17 hours. Media was harvested and spun to remove dead cells, while cells were scraped, washed twice and lysed in cold PBS. Protein concentration in the cell lysate was measured with a BCA protein quantification kit (Thermo Scientific, Waltham, MA). Fractions of the media and cell lysate portion were used for lipid extraction as described below.\nWT and nATF6 5 dpf larvae were labeled with 1-5 microCi 14C-acetate for 48 hours. Larvae were washed twice and then sonicated in cold PBS. Protein concentration in the larval lysate was measured with a BCA protein quantification kit (Thermo Scientific). Fractions of the larval lysate were used for lipid extraction as described below.\n\nLipid extractions\nLipids from media, cells and 5 dpf larvae were extracted according to Bligh and Dyer &lt;&lt;REF:Bligh-1959-37-911&gt;&gt; with the following modifications. Briefly, methanol and chloroform were added to 400 microl media or 50 microl of 100 microl total cell lysate in a ratio of 2:1. Samples were vortexed and chloroform and 0.45% NaCl were added at a ratio of 1:1, vortexed again and phases were separated by centrifugation. The aqueous phase was re-extracted with one part chloroform and combined with the lipid from the first second extraction, washed with methanol and 0.45% NaCl at a 1:1 ratio, centrifuged and the lipid phase was recovered and dried under a stream of N2 gas. Lipids were reconstituted in a 2:1 mixture of chloroform and methanol (Thermo Scientific, Waltham, MA), dissolved in Ultima Gold and counted using a 2460 MicroBeta2 LumiJET (PerkinElmer, Waltham, MA) liquid scintillation counter. CCPM were normalized to protein concentration to account for variations in cell number.\n\nQuantitative real-time PCR (qPCR)\nTotal RNA isolated using TRIzol (Invitrogen) from pools of 15-20 dissected livers was reverse-transcribed using qScript cDNA SuperMix (Quanta Biosciences, Gaithersburg, MD). qPCR using A Light Cycler 480 (Roche) with PerfeCTa SYBRGreen FastMix (Quanta Biosciences) was used as previously described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;. Values for target gene were normalized to reference gene rpp0, and dCts calculated using the comparative threshold method (dCt  = 2-(Ct, gene - Ct, rpp0)). Primer sequences are listed in Table S1.\n\nTriglyceride measurements\nApproximately 20 livers from control and nAtf6 transgenic larvae were dissected and lysed in 0.5% Triton-X 100 and heated at 65degreesC for 5 minutes to inactivate hepatic lipases. Triglycerides were measured using the Infinity Triglyceride Liquid Stable Reagent (Thermo Fisher Scientific, Waltham, MA) following manufacturer's instructions, and were normalized to total protein concentration as determined by Bradford Assay (Bio-Rad, Hercules, CA).\n\nOil red O staining and quantification\nLarvae were fixed in 4% paraformaldehyde (PFA; Electron Microscopy Sciences, Hatfield, PA) overnight at 4degreesC, stained with oil red O, and scored for steatosis as previously described &lt;&lt;REF:Howarth-2013-10-199&gt;&gt;. Cryosections were stained by immersing slides in increasing concentrations of propylene glycol (85%, 100%) for 10 minutes followed by an overnight incubation in oil red O (0.5% in propylene glycol, Polysciences, Warrington, PA). Excess oil red O was removed the next day by sequential washes in 100% and 85% propylene glycol for 5 minutes. Nuclei were counterstained with hematoxylin.\nHepG2 and 293T cells were stained as previously described &lt;&lt;REF:Wu-2010-402-116&gt;&gt; and counterstained with hematoxylin. Oil red O droplet area and number were calculated using ImageJ. In brief, images were split into the red, green, and blue channels and the green channel was further processed for quantification as described, as oil red O has an excitation at 510 nm &lt;&lt;REF:Narasimhan-2011-7-e1001377&gt;&gt;. Following this, the background was subtracted from each image to ensure only counting of oil red O droplets. Droplets equal or greater than five square pixels were counted and area quantified. The values were copied into Microsoft Excel and the average droplet area and number of droplets per nucleus were counted for each field. Twenty independent fields from nATF6 and GFP-transfected HepG2 and 293T cells were imaged at 60x magnification.\n\nRNA sequencing (mRNA-Seq)\nTotal RNA was isolated using TRIzol (Invitrogen) from pools of ~40-50 livers dissected on 5 dpf. Two clutches of Tg(fabp10:nls-mCherry) larvae that were either untreated (control) or treated with 350 mM ethanol for 24 hours and one clutch of nAtf6 TG larvae were used. polyA-tailed mRNA was selected using oligo-dT beads and then fragmented. cDNAs were created using random-hexamers and ligated with bar-coded adaptors compatible with HiSeq &lt;&lt;REF:???-?-?-?&gt;&gt; . Single-end, 100 bp reads were sequenced at the Genomics Core of the Icahn School of Medicine at Mount Sinai. Custom-built software was used to map the reads to the zebrafish genome (Zv9/DanRer7) and estimate the coverage of each gene. Briefly, the reads were split into three 32 bp parts after trimming 2 bp at each end and mapped to the genome using a suffix-array based approach. The median of coverage across the transcript was used as an estimate of gene expression. The expression values were quantile normalized and log-ratios were calculated by comparing nAtf6 TGs to the average of the controls. Each ethanol treated sample was compared to its paired control (untreated siblings). Unique Gene Ontology terms (GO terms) were assigned to each gene by ranking the GO terms by relevance to the biology of the response, and using annotations from the human orthologues if the zebrafish annotations were lacking.\nThe distribution of expression values were plotted to identify the peak in the distribution, which is the level of noise in the system. The values were regularized by adding the noise to each gene's expression level before the log-ratios were calculated. This ensures that genes with low expression do not contribute to list of genes with large fold-changes. An absolute natural log ratio of 0.2 was used as the cutoff (known, non-responding genes are all below this threshold). The list of genes that show changes above this cutoff were analyzed for pathway enrichment using GO terms annotated as described above.\nThis data is available via the Gene Expression Omnibus (GEO) at accession number GSE56498. The controls for this dataset, untreated 5 dpf Tg(fabp10:nls-mCherry), also serve as controls in GEO dataset GSE52605 &lt;&lt;REF:???-?-?-?&gt;&gt; and were included in both sets for ease of comparison between genotypes and treatments.\n\nImage processing and graphics\nImages were cropped and minimally processed using Adobe Photoshop CS4 (Adobe Systems, San Jose, CA). Graphs were plotted using Prism 5.0c (GraphPad Software Inc., La Jolla, CA). Heat maps for mRNA-Seq data were generated using GENE-E (Broad Institute, Cambridge, MA). The global maximum and minimum for each gene set was set to orange and green, respectively, and zero was set to white. Metabolic pathway schematics were adapted from WikiPathways (URL: www.wikipathways.org).\n\nStatistical analyses\nStatistical tests were performed using GraphPad QuickCalcs (GraphPad Software, URL: http://www.graphpad.com/quickcalcs/index.cfm). For oil red O staining of whole larvae, chi-square with Fisher's Exact test were performed. For qPCR, radiolabeling, and oil red O staining of HepG2 and 293T cells we performed unpaired and one-sample t-tests as appropriate.\n\nSupplementary methods\nMethods for histological analysis, cryosectioning and staining, and liver circularity analysis are listed in Text S1.\n\nSupporting Information\nFigure S1\nInjection of an atf6 splice-blocking morpholino induces mis-splicing of atf6 mRNA. A: Schematic of boundary between intron 1 and exon 1 of zebrafish atf6. Exon sequence is shown in black and intron sequence is shown in blue, with the morpholino shown in red. The start codon is highlighted in yellow. B: Gel electrophoresis of atf6 morphants at 2 dpf showing additional bands compared to uninjected controls. C: Sequencing results from atf6 morphants displaying inclusion of intronic sequence (highlighted in blue). The predicted protein translation is shown, and early stop codons are noted as red asterisks.\n(TIF)\nClick here for additional data file.\nFigure S2\nKnockdown of atf6 does not confer resistance to morphological defects from alcohol exposure. A: Oil red O staining of cryosections from uninjected and atf6 morphant larvae treated with 0 or 350 mM ethanol for 32 hours. B: Live images of uninjected and atf6 morphant Tg(fabp10:dsRed) larvae treated with 0 or 350 mM ethanol for 32 hours. C: Quantification of liver circularity, a measure of hepatomegaly, following ethanol treatment in uninjected and atf6 morphant larvae. Livers were traced and circularity quantified in ImageJ.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nTg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae express the nAtf6-cherry transgene despite a lack of visual fluorescence. A: Schematic of human ATF6 transactivation (TAD), DNA-binding (bZIP) and transmembrane (TM) domains, cleavage sites for S1P (MBTPS1) and S2P (MBTPS2), and protein alignment with zebrafish Atf6. Numbers in the diagram correspond to amino acid positions in human ATF6. Numbers at the cleavage sites in the protein alignment correspond to amino acid positions in zebrafish Atf6. B: Confirmation of nuclear localization of zebrafish nAtf6 by transfection of nAtf6-GFP/pCI-Neo into HepG2 cells. C: Live image of Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larva at 5 dpf. D: Agarose gel electrophoresis showing expression of Cherry mRNA in nAtf6 TG but not WT larvae. rpp0 was used to ensure PCR efficacy. E: Western blot visualization of Cherry protein in adult nAtf6 transgenic zebrafish liver tissue. Tg(fabp10:nls-cherry) adult liver protein was used as a positive control. Actin was used as a loading control.\n(TIF)\nClick here for additional data file.\nFigure S4\nOverexpression of nAtf6 induces robust transcription of UPR target genes but does not activate Ire1a or Perk. A: Agarose gel electrophoresis visualization of xbp1 splicing in WT and nAtf6 TG larvae in the presence and absence of tunicamycin (Tm). The percentage of xbp1 splicing was quantified by measuring band intensity in ImageJ. B: Western blot visualization of Eif2a phosphorylation in livers of 5 dpf WT and nAtf6 TG larvae. Tunicamycin (1 microg/mL, 3-5 dpf) treated WT larvae were used as a positive control for immunoblotting. Fold was calculated over 2-3 separate experiments as noted. C: Western blot visualization of Bip protein in livers of 5 dpf WT and nAtf6 TG larvae. Fold change (below blot) was calculated from 3 separate experiments. D and E: Quantitative real-time PCR of multiple UPR effectors in the livers of WT and nAtf6 TG larvae at 5 (D) and 14 (E) dpf. Statistics: unpaired t-test. *, p&lt;0.05.\n(TIF)\nClick here for additional data file.\nFigure S5\nLipid accumulation is evident in nAtf6 TG larvae in the absence of other hepatic injury. A: Staining of cryosections with oil red O showing significant steatosis in nAtf6 TG larvae at 5 and 14 dpf. B: Hematoxylin and eosin staining of nAtf6 TG and control larvae at 5 dpf. No other alterations in hepatic pathology were observed. The number of larvae used is indicated.\n(TIF)\nClick here for additional data file.\nFigure S6\nmRNA-Seq reveals potential disruptions in lipid transport by nAtf6 overexpression. Heatmap of genes associated with lipid binding and transport (A) and basic schematic of lipid synthesis and lipoprotein circulation (B, adapted from WikiPathways). Upregulated genes are colored in shades of orange, and downregulated genes are colored in shades of green. Genes that are unchanged are colored white. Genes that did not appear in mRNA-Seq analyses are colored gray in the schematic.\n(TIF)\nClick here for additional data file.\nFigure S7\nnAtf6 overexpression induces transcription of genes involved in glyceroneogenesis. Quantitative, real-time PCR analysis of gene expression from WT and nAtf6 TG larvae at 5 dpf (A) and 14 dpf (B). Statistics: unpaired t-test. *, p&lt;0.05. For (A), \"n\" corresponds to the number of clutches used and approximately 8-12 livers were dissected for RNA extraction. For (B), individual livers were dissected for analysis.\n(TIF)\nClick here for additional data file.\nFigure S8\nOverexpression of nATF6 in 293T cells drives lipid accumulation. A: Staining of GFP and nATF6 transfected cells with oil red O. B: Quantification of lipid droplet number and area (square pixels). Droplet number and area were calculated using ImageJ. Statistics: unpaired t-test. *, p&lt;0.05. The number of cells used for quantification is noted.\n(TIF)\nClick here for additional data file.\nFigure S9\nSrebp1 and Srebp2 target genes are not dysregulated by nAtf6 overexpression. A and B: qPCR analysis of Srebp1 (A) and Srebp2 (B) target genes in atf6 morphants treated with 350 mM EtOH for 32 hours. Fold changes were calculated based on median dCt values. C and D: qPCR analysis of Srebp1 (C) and Srebp2 (D) target genes in Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae at 5 dpf. Fold changes were calculated based on median dCt values, and \"n\" corresponds to the number of clutches used.\n(TIF)\nClick here for additional data file.\nFigure S10\nAtf6 and Srebps do not epistically interact to prevent alcoholic steatosis. Steatosis incidence in larvae injected with atf6, scap, or atf6 + scap MO. \"Total n\" corresponds to the number of larvae scored. \"Clutch n\" corresponds to the number of clutches analyzed. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05 and corresponds to groups significantly different from uninjected controls. b, p&lt;0.05 and corresponds to ethanol-treated groups significantly different from uninjected, ethanol treated larvae.\n(TIF)\nClick here for additional data file.\nTable S1\nMorpholino and primer sequences. A: Morpholino sequences. Amount of morpholino injected is based on the assumption that 4 nl of MO is injected per embryo. B: Primer sequences used for conventional and quantitative, real-time PCR.\n(DOCX)\nClick here for additional data file.\nTable S2\nUPR effectors upregulated by nAtf6 and by ethanol. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Log values &gt;= 0.2 are highlighted in red.\n(XLSX)\nClick here for additional data file.\nTable S3\nUpregulated and downregulated genes in the liver of nAtf6 TG 5 dpf larvae. A: Upregulated genes in the liver of nAtf6 TG 5 dpf zebrafish larvae. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. UPR target genes are highlighted in pink. GO terms were assigned as described in the Materials and Methods. B: Downregulated genes in the liver of nAtf6 TG 5 dpf larvae. Genes regulating lipoprotein secretion are highlighted in orange.\n(XLSX)\nClick here for additional data file.\nTable S4\nRaw data from mRNA-Seq. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Genes are organized from highest to lowest log value for nAtf6 TG larvae.\n(XLS)\nClick here for additional data file.\nText S1\nSupplementary methods.\n(DOC)\nClick here for additional data file.",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nThe unfolded protein response (UPR) acts in most cells to maintain homeostasis within the protein secretory pathway during physiological conditions. During stress, the UPR becomes further induced to mitigate the accumulation of misfolded or unfolded proteins in the endoplasmic reticulum (ER). If UPR activation cannot relieve the excess protein load, the ER becomes dilated and dysfunctional, and all branches of the UPR remain chronically activated in a condition referred to as ER stress. Many studies have implicated ER stress in a range of pathologies, and there is a clear association between UPR activation and metabolic diseases such as fatty liver disease (FLD) &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;-&lt;&lt;REF:Rinella-2011-54-1600&gt;&gt;. However, it is not known whether factors that control the UPR can also directly impact lipid metabolism and it remains unclear how UPR activation causes FLD.\nFLD is the most common hepatic pathology worldwide &lt;&lt;REF:???-?-?-?&gt;&gt;, and alcohol abuse is a leading cause. Even a single episode of binge drinking causes lipid accumulation in hepatocytes (steatosis) in over 90% of drinkers &lt;&lt;REF:???-?-?-?&gt;&gt;. While acute steatosis can resolve, chronic steatosis can render hepatocytes susceptible to damage and is a prerequisite step in developing more severe liver disease, including steatohepatitis and cirrhosis. Most FLD etiologies are accompanied by impairment of protein secretion by hepatocytes that result in serum protein deficiencies, which are most apparent in chronic alcoholics and contribute to the systemic complications of alcoholic liver disease (ALD). These defects are reflected in many reports demonstrating that alcohol induces some UPR sensors and targets in the liver of mice &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;, &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, rats &lt;&lt;REF:Shinohara-2010-51-796&gt;&gt;, micropigs &lt;&lt;REF:Esfandiari-2005-289-G54&gt;&gt;, and zebrafish &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, deleting ATF4 or CHOP, two key UPR effectors, reduces ethanol-induced liver injury &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, indicating that UPR activation can contribute to ALD. However, it is not clear whether these genes, or other UPR effectors, contribute to steatosis.\nThere are three main UPR sensors that function through shared and independent mechanisms to maintain ER function by enhancing protein processing and folding in the ER, and by promoting degradation of terminally misfolded secretory proteins. Activating transcription factor 6 (ATF6) and inositol-requiring enzyme 1-alpha (IRE1alpha or ERN1) pathways generate the active transcription factors nATF6 and XBP1, respectively, that induce hundreds of UPR target genes &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. PRKR-like endoplasmic reticulum kinase (PERK or EIF2AK3) phosphorylates EIF2A to repress translation &lt;&lt;REF:Harding-2000-5-897&gt;&gt; and to promote production of ATF4 &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;, which induces a subset of target genes &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;. Each of these primary sensors has been evaluated for its contribution to FLD caused by a robust ER stressor &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt;. ATF6 knockout mice fail to resolve steatosis caused by acute stress due to tunicamycin injection &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Yamamoto-2010-21-2975&gt;&gt;, suggesting that loss of ATF6 promotes fatty liver. Our previous work using zebrafish confirmed the finding that steatosis caused by acute stress is augmented by atf6 loss, and also demonstrates that steatosis caused by chronic stress is reduced when Atf6 is depleted &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it appears that atf6 loss can alternatively enhance or reduce FLD, depending on the nature and duration of the stress.\nThe relationship between metabolic disease and UPR activation is under intensive investigation, yet several important questions remain unanswered. First, is UPR activation a cause or consequence of FLD? While some studies indicate that lipotoxicity and fatty acid accumulation can induce the UPR &lt;&lt;REF:Wei-2006-291-E275&gt;&gt;, &lt;&lt;REF:Wang-2006-147-943&gt;&gt;, there is incontrovertible evidence that robust UPR activation is sufficient to induce steatosis &lt;&lt;REF:Lee-2012-211-29&gt;&gt;, &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;. These data are incorporated into a current model proposing that robust UPR activation can cause steatosis and, if the lipid burden is not resolved, this can further augment cellular stress and contribute to chronic UPR induction. The second question is which, if any, of the main UPR factors directly cause fatty liver? All studies to date that have addressed this question, including our work in zebrafish &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, utilize loss of function approaches whereby a key UPR gene is deleted and the effect on stress-induced steatosis is evaluated. However, these studies are complicated by the extensive crosstalk between UPR effectors, and by the ability of cells to adapt to changes in UPR capacity. Third, what is the mechanism by which UPR activation causes fatty liver? Activation of lipogenic genes by the sterol response element binding protein (SREBP) transcription factors has been implicated as an essential pathway in both alcoholic &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt; and non-alcoholic &lt;&lt;REF:Higuchi-2008-38-1122&gt;&gt;-&lt;&lt;REF:Zhang-2012-212-229&gt;&gt; FLD. Since SREBPs and ATF6 are activated by a shared proteolytic mechanism &lt;&lt;REF:Ye-2000-6-1355&gt;&gt;, it is possible that these two proteins are activated simultaneously in response to stress. However, the role of SREBPs in FLD caused by UPR activation has not been addressed.\nHere, we use genetic and genomic approaches in a well-established zebrafish model of ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;-&lt;&lt;REF:Liu-2013-6-1271&gt;&gt; to demonstrate that blocking Atf6 prevents alcohol-induced steatosis and that overexpression of the nuclear, active form of Atf6 (nAtf6) causes steatosis. We found that nAtf6 overexpression increases expression of genes in the lipogenic pathway and alters lipid flux to promote triglyceride synthesis by a mechanism that does not involve Srebps. Instead, we demonstrate an epistatic interaction between atf6 and fasn in driving alcoholic steatosis. These findings demonstrate the first clear and causative link between a central UPR sensor and FLD.\n\nResults\n\nAtf6 is required for alcoholic steatosis\nWe previously reported that transcription of multiple UPR effector genes and translation of the major ER chaperone, Bip, are induced in zebrafish livers following alcohol exposure &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. We have also demonstrated that xbp1 splicing, a direct measure of Ire1a activation, is only detectable within the first few hours of exposure to ethanol &lt;&lt;REF:???-?-?-?&gt;&gt; yet UPR target gene induction persists long afterwards &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, suggesting that the Ire1a pathway is not entirely responsible for the UPR in ALD. Our previous finding that Eif2a is phosphorylated in the liver of zebrafish with ALD &lt;&lt;REF:???-?-?-?&gt;&gt; suggests that Perk is activated by alcohol, but since Eif2a can also be phosphorylated by other kinases &lt;&lt;REF:Dever-1992-68-585&gt;&gt;, this has yet to be resolved. The lack of reliable antibodies that recognize the active form of Atf6, Perk and Ire1a in zebrafish hampered direct measurement of the activation status of each pathway (not shown). However, quantitative real-time PCR (qPCR) analysis of liver samples from 5 day post fertilization (dpf) larvae treated with alcohol demonstrated that atf6 mRNA was significantly induced in the liver (Figure 1A &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;) prior to the onset of steatosis (see Figure 1B &lt;&lt;FIG:pgen-1004335-g001&gt;&gt; and &lt;&lt;REF:???-?-?-?&gt;&gt;). Since Atf6 has been shown to act in a positive feedback loop to induce its own expression &lt;&lt;REF:Namba-2007-355-543&gt;&gt;, these data suggest that the increase in atf6 expression could reflect Atf6 activation in response to alcohol.\nFigure 1\nAtf6 is required for alcoholic steatosis.\n\nA: qPCR analysis of UPR sensors atf6, perk, ire1a, and atf4 in the livers of control (EtOH-) and 350 mM ethanol-treated (EtOH+) larvae. dCt values were calculated by normalization to rpp0. Fold changes were calculated based on median dCt values. Statistics: paired t-test. *, p&lt;0.05. B: Expression of atf6 in the livers of 350 mM ethanol treated larvae from 0-32 hours. Fold changes were determined by normalizing to control (0 mM EtOH) dCt values. Statistics: one-sample t-test. *, p&lt;0.05. C: Images of oil red O stained whole larvae injected with two different morpholinos targeting atf6 (ATG-targeting, atf6 ATG, and splice-blocking, atf6 SPL) and exposed to either 0 or 350 mM ethanol. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring oil red O stained larvae at 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored.\nTo test whether Atf6 was required for alcoholic steatosis, we injected a morpholino (MO) targeting either the translation initiation site (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; or the boundary between coding exon 1 and intron 1 of atf6 (atf6MO-SPL) (Table S1 and Figure S1A), which blocked splicing (Figure S1B) and introduced an early stop codon (Figure S1C). Morphants and uninjected controls were treated with 350 mM ethanol for 32 hours, stained with the neutral lipid dye, oil red O and scored for the presence or absence of steatosis (Figure 1C and S &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;2A). Consistent with previous findings &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;, the percent of larvae with steatosis (i.e. steatosis incidence) was significantly higher in uninjected larvae exposed to 350 mM ethanol for 32 hours (13% in untreated larvae vs. 53% in ethanol treated larvae; p&lt;0.05). atf6 morphants had reduced steatosis incidence (p&lt;0.05, Figure 1C-D &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;), but still displayed other gross morphological abnormalities caused by ethanol including increased liver circularity, an indication of hepatomegaly (Figure S2B-C), a common feature of liver disease. Thus, Atf6 is required for alcoholic steatosis.\n\nnAtf6 overexpression in hepatocytes induces robust UPR activation\nTo determine whether Atf6 is sufficient to cause FLD, we created a transgenic zebrafish line expressing the predicted nuclear, active fragment of zebrafish Atf6 (amino acids 1-366; Figure S3A) fused to mCherry under the hepatocyte-specific promoter fabp10, along with a cassette that expressed GFP in cardiomyocytes as a marker of transgenesis ((Tg(fabp10:nAtf6-cherry; cmlc2:GFP) hereafter called nAtf6 TG). There is high sequence identity between human and zebrafish Atf6 proteins in the transmembrane, leucine zipper and basic (bZIP) domains and the protease recognition sites are highly conserved (Figure S3A). To confirm its nuclear localization, the predicted nAtf6 fragment of the zebrafish protein was fused to GFP and transfected into HepG2 cells (Figure S3B).\nTransgenics were selected based on GFP expression in the heart, appeared grossly normal throughout development (Figure S3C), and developed into viable, fertile adults. nAtf6-mCherry was not detectable using low-resolution fluorescence microscopy in larvae from all four of the transgenic lines we generated. However, we confirmed transgene expression by PCR and blotting for mCherry (Figures S3D-E). atf6 mRNA expression in the liver was ~4 fold higher in nAtf6 TG larvae compared to wildtype (WT) larvae (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4) at 5 dpf, and was persistently elevated in older fish (Figure S4D and not shown). We detected mCherry mRNA in 5 dpf nAtf6 TG larvae by PCR, and Western blotting for mCherry detected the transgene in the liver of adult fish, albeit at much lower levels than achieved in transgenic fish expressing nuclear-localized mCherry without fusion to nAtf6 (i.e. Tg(fabp10:nls-mCherry); &lt;&lt;REF:???-?-?-?&gt;&gt;\nFigures S3D-E).\nFigure 2\nnAtf6 overexpression is sufficient to drive steatosis.\n\nA: Heatmap of upregulated UPR effector genes in nAtf6 transgenic larvae. The log values based on mRNA-Seq analysis and the median fold change in 3-6 liver samples assessed by qPCR at 5 dpf and 14 dpf. B: Most Atf6 targets are upregulated in the liver in response to ethanol. Genes identified as part of the UPR are significantly induced in the liver of nAtf6 TG and ethanol treated larvae (log value &gt;= 0.2; see Table S2). C: Oil red O stained WT and nAtf6 transgenic larvae. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring of whole mount oil red O stained larvae at 4, 5, and 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored. E: Hepatic triglyceride levels in extracts from pooled livers from WT and nAtf6 transgenic larvae at 5 dpf and 14 dpf, and from single adult livers (~9 months). Statistics: unpaired t-test. *, p&lt;0.05. Median fold changes are noted.\nWe confirmed that nAtf6 overexpression did not induce the other UPR branches by assessing xbp1 splicing (Figure S4A) and Eif2a phosphorylation (Figure S4B) in the liver of nAtf6 TG 5 dpf larvae. We found no difference in these markers in unstressed nAtf6 TG and WT larvae, however, xbp1 splicing was lower in transgenics than in WTs after exposure to stress (i.e. 1 microg/ml tunicamycin for 48 hours; Figure S4A). This suggests that nAtf6 overexpression adapts hepatocytes to withstand a robust, pharmacologically-induced ER stress.\nWe next examined the expression of UPR target genes in the liver of 5 dpf transgenic and control larvae. We found that bip (also called hspa5), a transcriptional target of Atf6, was elevated at the protein (Figure S4C) and mRNA levels (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D) in nAtf6 TG larvae. mRNA-Seq analysis comparing control larvae (Tg(fabp10:nls-mCherry); n = 2 pools of ~40-50 livers each; see &lt;&lt;REF:???-?-?-?&gt;&gt; and GEO dataset GSE52605), ethanol-treated (n = 2 pools of ~40-50 livers each), and nAtf6 TG larvae (n = 1 pool of ~40-50 livers) revealed 33 UPR target genes to be highly and significantly upregulated in nAtf6 TG livers (Tables S2-S4 and Figure 2A &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;; GEO dataset GSE56498). These genes were identified as ATF6 target genes in other studies &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; or by having the UPR or protein folding as a primary GO descriptor, and included ER chaperones (bip/hspa5, dnajc3, and grp94/hsp90b1), quality control effectors (canx, calret, calrl, and calrl2), protein disulfide isomerases (pdia3, pdia4) and ERAD components (edem1, derl1 and derl3). A subset of these genes was confirmed by qPCR at 5 dpf and at 14 dpf (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D-E). atf4 and derl1 were the only genes detected as significantly upregulated by qPCR but not by mRNA-Seq.\nAnalysis of the mRNA-Seq data from ethanol-treated larvae revealed a striking overlap between the \"UPR-ome\" induced by ethanol and by nAtf6 overexpression: of the 49 UPR target genes induced by ethanol, 30 were also induced in the liver of nAtf6 TG larvae (Figure 2B &lt;&lt;FIG:pgen-1004335-g002&gt;&gt; and Table S2). It is possible that the remaining 19 genes unique to the ethanol-induced transcriptome are attributed to the transient, early increase in xbp1 splicing &lt;&lt;REF:???-?-?-?&gt;&gt; or to other transcription factors that regulate their expression. Collectively, these data indicate that Atf6 is a major regulator of the UPR in ALD.\n\nAtf6 overexpression causes steatosis\nWe next asked whether nAtf6 was sufficient to cause FLD (Figure 2C-E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Induction of the fabp10:nAtf6-cherry transgene occurs at ~2.5 dpf &lt;&lt;REF:Her-2003-538-125&gt;&gt; and by 4 dpf, 44% of nAtf6 TG larvae developed steatosis which increased to 69% and 76% by 5 and 5.5 dpf, respectively (Figure 2C-D and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;5A) and persisted until at least 14 dpf (Figure S5A), but there were no marked histological abnormalities in the nAtf6 TG liver at 5 dpf (Figure S5B). Hepatic triglyceride (TAG) levels were nearly doubled at 5 dpf, and six times higher in 14 dpf nAtf6 TG larvae (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), clutch-to-clutch variability non-withstanding. However, TAG levels were the same in WT and nAtf6 TG adults (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), despite persistent expression of the transgene (Figure S3E and not shown). We speculate that, over time, animals adapt to nAtf6 overexpression or that other metabolic parameters affect hepatic lipid accumulation in adult fish. Together, our data demonstrate that Atf6 is necessary for alcoholic steatosis and sufficient to cause steatosis in the absence of any other stress.\n\nnAtf6 overexpression alters expression of genes involved in lipogenesis and lipid export\nSteatosis is caused by elevated TAGs in hepatocytes resulting from increased lipid synthesis or uptake, or from decreased lipid utilization or export. We therefore analyzed pathways relevant to these processes in the mRNA-Seq dataset from nAtf6 TG livers to determine if any were dysregulated. TAGs are generated from linking free fatty acids (FFAs) to a glycerol backbone, which is generated by conversion of dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P) by G3P dehydrogenase (GPD1) (see schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). We found genes that participate in TAG synthesis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;) or export (Figure S6) were dysregulated in nAtf6 TG livers at 5 dpf, albeit not to the same degree as observed for the UPR target genes (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). DHAP is generated via glycolysis or by glyceroneogenesis, a pathway that has significant overlap with gluconeogenesis (see &lt;&lt;REF:Nye-2008-283-27565&gt;&gt; and schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). mRNA-Seq analysis demonstrated that several genes involved in glycolysis were downregulated in nAtf6 TG livers including aldob, glut2/slc2a2, gck, pgk1, and pklr (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Genes promoting glyceroneogenesis, particularly pck1 and got1, which generates oxaloacetate for conversion to phosphoenoylpyruvate by pck1, were upregulated, as was gpd1b, which drives glycerol formation. This suggests that nAtf6 overexpression increases factors regulating glyceroneogenesis preferentially over those that promote glycolysis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). In addition, ech1 and tecrb, two genes responsible for fatty acid elongation in the mitochondria and ER, respectively, were induced in nAtf6 TG livers by mRNA-Seq (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Some of these genes were confirmed by qPCR analysis of nAtf6 TG livers, with the median fold change from qPCR on 3 to 6 clutches of livers from 5 and 14 dpf larvae depicted in Figure S7.\nFigure 3\nGlyceroneogenesis and fatty acid elongation pathways are dysregulated by nAtf6 overexpression.\nSchematic of the glycolysis, gluconeogenesis and glyceroneogenesis pathway (A, adapted from WikiPathways) and heatmap of associated genes (B). Upregulated genes are colored in shades of orange, downregulated genes are colored in shades of green, genes that are unchanged are white and those genes that did not appear in mRNA-Seq analyses are colored gray. Log values and median fold changes from mRNA-Seq analysis and qPCR, respectively, are noted.\nTAGs are assembled in the hepatocyte ER and secreted in association with lipoproteins that are required for transport in plasma. Several genes that regulate lipoprotein assembly and export, including apom, apoeb, apoea, cetp and zgc:162608 and zgc:194131, two apolipoprotein-like genes, were downregulated in nAtf6 TG livers (Figure S7 and Table S3). pdia2, which interacts with MTP to transport lipids out of hepatocytes &lt;&lt;REF:Wetterau-1997-1345-136&gt;&gt;, was also downregulated. These data suggest that hepatic lipid transport may be impaired by nAtf6 overexpression. While these expression data do not capture the complex metabolic and post-translational regulation of the proteins encoded by these genes, they do suggest enhanced lipogenesis or suppressed lipoprotein export as potential mechanisms of steatosis in nAtf6 transgenics.\n\n\nDe novo lipogenesis is enhanced by nAtf6 overexpression\nAcetate is a precursor to acetyl-CoA, the major building block of fatty acid and, therefore, TAG synthesis. To determine if nAtf6 overexpression increased de novo lipogenesis, we measured the incorporation of 14C-acetate into lipids in nAtf6 TG larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) and in HepG2 cells overexpressing nATF6 (Figure 4B-D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) compared to their respective controls. Larvae were incubated with 14C-acetate from 3-5 dpf and we measured the amount of radiolabel present in the lipid fraction compared to that present in the lysate of the whole larvae. In an average of 5 individual pools of larvae, more 14C radiolabel was detected in the lipid fraction of nAtf6 TG larvae than in wildtype larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;).\nFigure 4\n\nDe novo lipogenesis is enhanced in vitro and in vivo by nAtf6 overexpression.\n\nA:\n14C-acetate is preferentially incorporated into lipids in 5 dpf nAtf6 TG larvae compared to WT. The percent of 14C in lipid fraction was divided by 14C measured in the unextracted lysate from whole fish. The median fold change is noted; n =  number of samples of pooled larvae. B: Oil red O staining of HepG2 cells transfected with GFP or nATF6. Bar  = 10 microm. C: Quantification of lipid droplet number and area (square pixels). Statistics: unpaired t-test. *, p&lt;0.05; n indicates the number of cells quantified over 20 fields from 2 independent experiments. D: Incorporation of 14C-acetate into lipids by HepG2 cells transfected with GFP or nATF6 normalized to protein. The median fold change is noted; n =  number of separate batches of cells analyzed in 2 independent experiments.\nSince the liver in 5 dpf zebrafish larvae is too small to analyze incorporation in this organ exclusively, analysis of the whole larvae could not determine whether the label was preferentially incorporated into hepatocytes of nAtf6 TG larvae. We thus used human hepatoma (HepG2) cells transfected with human nATF6 to address this question. nATF6 overexpression causes accumulation of oil red O stained cytoplasmic droplets (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) that are both greater in number and larger in size than in cells transfected with GFP (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). These findings were confirmed in 293T cells (Figure S8) and correlate with our findings in nAtf6 TG larvae (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Thus, nAtf6 overexpression causes lipid accumulation across cell types and species. Moreover, in eleven independent cell samples from two separate experiments, HepG2 cells overexpressing nATF6 had slightly more 14C radiolabel in the lipid fraction compared to GFP-transfected cells (Figure 4D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). While we found that the incorporation tended to be higher in the nATF6 expressing cells, the average increase was moderate, at best. We attribute this to the transfection efficiency of HepG2 cells combined with experimental conditions used to label lipids, which likely act to select against nATF6 expressing cells.\n\nSrebp activation does not contribute to Atf6-mediated steatosis\nFatty acid synthase (fasn) mediates multiple steps in fatty acid synthesis that culminate in generation of palmitate from acetyl-CoA, a building block for TAG biosynthesis. fasn transcription in hepatocytes is primarily regulated by Srebp1c &lt;&lt;REF:Shimano-1997-100-2115&gt;&gt;-&lt;&lt;REF:Shimano-1996-98-1575&gt;&gt;, and we previously demonstrated that fasn transcripts and other Srebp target genes are induced in zebrafish with ALD &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, blocking Srebp activation by injecting a morpholino targeting the Srebp activating protein, Scap, reduced the incidence of alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Since Atf6 and Srebps are activated by similar mechanisms - they are both retained as inactive precursors in the ER and processed by the same Golgi-resident enzymes &lt;&lt;REF:Ye-2000-6-1355&gt;&gt; - Srebp mediated lipogenesis has been proposed as a mechanism by which UPR activation causes FLD &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;. We tested this using a genetic approach. First, we asked whether blocking Atf6 affected the induction of lipogenic gene expression in the liver of ethanol treated larvae. Both fasn (Figure 5A &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;), and acc1 (acaca;\nFigure S9A) were upregulated by ~2 fold in the liver of ethanol treated larvae, and this was blocked by atf6 MO injection (Figures 5A and S &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;9A). However, induction of other Srebp target genes was not blocked by Atf6 depletion, including srebp1 (Figure S9A) and the Srebp2 targets hmgcra, hmgcs1 and mvk (Figure S9B). Since the entire Srebp target gene set was not uniformly affected by atf6 loss, we conclude that Atf6 does not interact with Srebps. Instead, Atf6 may affect expression of specific lipogenic genes (i.e. fasn and acc1) by a mechanism distinct from Srebp1. Interestingly, we found mild induction of Srebp2 target genes in untreated atf6 morphants (Figure S9B), supporting conclusions based on experiments in mammalian cultured cells where ATF6 was found to directly suppress SREBP2 activity &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;.\nFigure 5\nFasn functions downstream of Atf6 to cause steatosis independent of Srebp activation.\n\nA: qPCR analysis of fasn expression in atf6 morphants and uninjected larvae treated with EtOH for 32 hours. B: qPCR analysis of fasn expression in nAtf6 TG larvae and HepG2 cells transfected with nATF6. Statistics: unpaired t-test. *, p&lt;0.05. C: Quantification of whole mount oil red O staining of nAtf6 TG larvae injected with scap MO. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored, respectively. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. D: Quantification of oil red O staining in nAtf6 TG larvae injected with fasn MO. E: Quantification of oil red O staining in atf6/fasn double morphants treated with 350 mM EtOH for 24 hours. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05.\nNext, we asked whether nAtf6 was sufficient to induce fasn and acc1 by assessing their expression in livers of nAtf6 TG larvae. Both fasn (p&lt;0.05; Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;) and acc1 (Figure S9C) were increased by ~2 fold at 5 dpf, and fasn expression was also higher at 14 dpf in nAtf6 TG larvae and in HepG2 cells overexpressing nATF6 (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). However, other Srebp1 and Srebp2 genes were not significantly induced by nAtf6 overexpression in nAtf6 TG larvae (Figure S9C-D) or in HepG2 cells overexpressing human nATF6 (not shown), indicating that nAtf6 does not activate the entire Srebp target gene program. We then tested whether nAtf6 functioned upstream of Srebps by assessing whether scap morpholino injection suppressed steatosis in nAtf6 TG larvae. While scap morphants are resistant to ethanol-induced steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt; we found no difference in steatosis incidence in nAtf6 transgenics (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Finally, to test whether atf6 and scap interacted to cause alcoholic steatosis we injected low concentrations of morpholinos targeting both genes and found that they did not synergize to suppress alcoholic steatosis (Figure S10). We thus conclude that lipogenic gene induction and steatosis caused by nAtf6 overexpression does not require Srebps.\n\nSteatosis in nAtf6 TG larvae requires fasn\n\nThe functional relevance of the finding that nAtf6 overexpression increases lipogenesis was tested by assessing the effect of morpholino-mediated fasn (Table S1) knockdown on steatosis incidence in nAtf6 TG larvae. Injection of high concentrations of fasn MO induced severe morphological defects, so we optimized the fasn MO concentration to have minimal toxicity. This had no effect on steatosis incidence in control larvae, but reduced steatosis incidence in nAtf6 TG larvae from 74% to 42% (Figure 5C &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Furthermore, we found that atf6 and fasn interacted epistatically to cause alcoholic steatosis by co-injecting concentrations of fasn and atf6-SPL MOs that did not have any effect on alcoholic steatosis on their own but, together, significantly reduced steatosis incidence in response to ethanol (Figure 5D &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Thus, Atf6 and Fasn function in the same genetic pathway. Taken together, our data suggest a model by which Atf6 causes steatosis, in part, by inducing fasn and TAG synthesis by a mechanism that is independent of Srebps (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;).\nFigure 6\nWorking model by which Atf6 functions as a positive regulator of alcoholic steatosis.\nGenetic tools used in this study are illustrated in red.\n\nDiscussion\nThere is a wealth of recent literature showing that the UPR is activated in FLD and that this aspect of the disease is conserved across species &lt;&lt;REF:Puri-2008-134-568&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:Kaplowitz-2006-21-S7&gt;&gt;-&lt;&lt;REF:Ye-2010-59-6&gt;&gt;. However, it has not been shown that any single UPR factor can directly cause this disease and the mechanism by which UPR activation causes lipid accumulation is unclear. Here, we use zebrafish genetics and biochemical analyses to show that Atf6 is both necessary and sufficient for steatosis by showing that: (i) Atf6 is required for alcoholic steatosis, (ii) activation of Atf6 is sufficient to cause steatosis, (iii) Atf6 activation induces expression of genes involved in glyceroneogenesis and fatty acid elongation and causes de novo lipogenesis, and (iv) Atf6 epistatically interacts with fatty acid synthase (fasn), a key enzyme involved in TAG biosynthesis, to cause FLD. This is among the first data to directly link a main UPR sensor and lipid metabolic pathways. \nA number of studies have demonstrated that loss of one of the key UPR sensors - PERK &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, IRE1alpha &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt; or ATF6 &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; - enhances steatosis caused by acute stress. Additionally, we previously demonstrated that Atf6 depletion suppresses steatosis caused by chronic stress &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it is clear that UPR activation is a conserved, common feature of FLD, but whether it functions to promote or prevent steatosis appears to depend on the nature and duration of the FLD-causing stress. Importantly, these loss of function studies have described a requirement for the UPR in FLD caused by tunicamycin injection, but not in the context of a stress that mirrors human conditions, such as obesity or alcohol abuse. Here, we show that Atf6 is induced in ALD, that its activation precedes steatosis, and that knocking down Atf6 reduces alcoholic steatosis. Thus, Atf6 is required for ALD. Moreover, since overexpression of nAtf6 is sufficient to cause FLD in the absence of any other stress, we conclude that this branch of the UPR is a pathophysiological mechanism of FLD.\nAtf6 and Xbp1 are the primary transcription factors that both independently and cooperatively regulate expression of hundreds of UPR target genes. We previously found significant upregulation of many UPR target genes in zebrafish with ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt; and hypothesized that since xbp1 splicing in the liver was only an early, transient response to alcohol &lt;&lt;REF:???-?-?-?&gt;&gt;, other transcription factors must participate in this robust UPR. Transcriptome analysis of the livers from nAtf6 TG and ethanol treated larvae enabled us to identify a set of target genes that are likely to be directly activated by Atf6. Many of these genes were also reported to be upregulated in mice overexpressing inducible ATF6 in the heart &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; and they were not induced by tunicamycin treatment of Atf6-/- MEFs &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. Genes occupying the intersection of these different datasets are likely bona fide Atf6 targets and the transgenic larvae we generated provides a system to test this directly. In contrast, many genes that were downregulated when ATF6 was overexpressed in the mouse heart were unchanged in nAtf6 transgenic zebrafish livers and, conversely, genes such as canx, derl1, and atf4 were upregulated in the nAtf6 TG zebrafish liver samples but not in the mouse model. The variations between the two datasets may be due to inherent differences in the models or approaches used to detect changes in gene expression, or may reflect the ability of Atf6 to differentially regulate target genes depending on cell type: hepatocytes possess a significantly higher secretory capacity than cardiomyocytes, and thus the Atf6 transcriptome in hepatocytes may be more extensive in order to maintain ER homeostasis. Finally, comparative transcriptome analysis between nAtf6 TG and ethanol treated larvae identified significant overlap between the UPR target genes in these two datasets, indicating that Atf6 is the main transcriptional driver of the ethanol-induced \"UPR-ome\".\nWhat is the mechanism by which the UPR causes FLD? Our data showing that nAtf6 overexpression induces the expression of some lipogenic enzymes and increases lipid synthesis, and that blocking fatty acid synthesis reduces FLD, fits a model (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;) whereby increased lipid production is, in part, the mechanism of steatosis in FLD caused by UPR activation. However, despite our finding that Atf6 depletion suppresses fasn expression and nAtf6 overexpression induces fasn, we have no evidence that fasn is a direct transcriptional target of Atf6. Indeed, the level of fasn induction is far less than the established Atf6 target, bip, and there are no canonical UPREs or ERSEs &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt; in the fasn promoter (not shown), thus other pathways are also likely at play. The mechanism by which nAtf6 induces fasn and acc1 require further investigation, and the tools we describe here will facilitate such studies.\nThe SREBP transcription factors are well-characterized regulators of hepatic lipogenesis; SREBP1c functions by increasing the expression of the full panel of genes required for TAG biogenesis &lt;&lt;REF:Horton-2003-100-12027&gt;&gt;. One possibility, suggested by the finding that the UPR and SREBPs are activated in parallel in some systems &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;, is that nAtf6 causes Srebp activation, leading to increased expression of fasn and other lipogenic genes and promoting lipogenesis. Our data argues against this: (i) the full panel of Srebp targets are not induced by nAtf6 overexpression in zebrafish, (ii) Atf6 and Srebps did not epistatically interact to modify alcoholic steatosis and (iii) Srebp activation is not required for steatosis in nAtf6 TG larvae. While it does appear that Srebp activation is required for alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt;, the current study shows that it is not the only important pathway that regulates hepatic lipid metabolism, as steatosis in nAtf6 TG larvae is independent of Srebps.\nWhile Srebps are not required for the effects of Atf6 on steatosis, there does appear to be some interaction between Atf6 and Srebp2, consistent with in vitro data showing that ATF6 suppresses SREBP2 &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;. We found that Atf6 causes upregulation of Srebp2 target genes, and while this modest increase in Srebp2 target genes in atf6 morphants did not appear to have a functional impact on cholesterol biogenesis, since atf6 morphants do not develop steatosis but are protected from it, it does suggest that Atf6 may suppress Srebp2 activity.\nWe used mRNA-Seq analyses to identify potential pathways that are dysregulated in response to Atf6 overexpression, although it is clear that the complex post-translational regulation of these pathways are not captured by gene expression studies. Our data suggested that lipoprotein export may be impaired by Atf6 overexpression, but we did not find that the flux of 14C-acetate into the extracellular lipids was impaired in cells overexpressing nATF6. While there are some technical caveats to these biochemical studies, we speculate that decreased lipoprotein export is not the only mechanism by which nAtf6 overexpression causes steatosis. Future studies to optimize cell conditions to ensure maximal cell viability and sustained high levels of nAtf6 expression will be required to fully address this.\nWhile steatosis is clearly a first step on the continuum to more severe liver disease, we also propose a different possibility: that lipid accumulation is a sign of an active and adaptive stress response. Protein folding in the ER is metabolically demanding, and when secretory cargo increases or the ER becomes full of unfolded proteins, increased lipid accumulation may enhance the ATP supply and thereby sustain the increased protein folding demands. Thus, the UPR may serve to enhance the protein folding capacity of the ER function in two ways: by activating genes required for protein folding, quality and export, and by promoting metabolic flux to TAGs as an energy source to withstand the challenge presented by a high level of secretory cargo. In this scenario, acute steatosis would not be a pathological response but, instead, would provide a protective role to sustain hepatocyte function during stress. However, if the stress response persists and steatosis becomes chronic, lipotoxicity could potentiate liver injury. Based on these data, it is tempting to speculate that enhancing protein folding in the ER by chemical chaperones, which have been used in both mouse models and humans to alleviate UPR activity, attenuate fatty liver disease and increase insulin sensitivity &lt;&lt;REF:Ozcan-2006-313-1137&gt;&gt;, &lt;&lt;REF:Kars-2010-59-1899&gt;&gt;, &lt;&lt;REF:Xiao-2011-60-918&gt;&gt;, could reducr the demand for energy and thus reduce the need to accumulate lipid. Whether Atf6 functions both as sensor of unfolded proteins and of metabolic demand remains to be elucidated.\n\nMaterials and Methods\n\nAnimal husbandry, treatments and transgenics\nAdult wildtype (WT, TAB14 and AB), Tg(fabp10:dsRed) &lt;&lt;REF:Dong-2007-39-397&gt;&gt; and Tg(fabp10:nls-mCherry)\n&lt;&lt;REF:???-?-?-?&gt;&gt; zebrafish were maintained according to standard conditions. Larvae were exposed to 350 mM ethanol (Pharmco-AAPER, Brookfield, CT) in fish water starting at 96-98 hours post fertilization (hpf) for up to 32 hours as described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;. All zebrafish protocols were approved by Mount Sinai's Institutional Animal Care and Use Committee.\nMorpholinos targeting the translation initiation ATG of atf6 (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, an atf6 intron-exon boundary (atf6MO-SPL, Figure S1), a scap intron-exon boundary &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, and a fasn intron-exon boundary were ordered from GeneTools (Philomath, OR). Approximately 1-5 pmol were injected into 1-2 cell stage embryos. Morpholino sequences and amount injected per embryo (ng) are listed in Table S1.\nThe Tg(fabp10:nAtf6-cherry; cmlc2:GFP) transgenic line was created by injecting a vector containing 2813 bp of the fabp10 promoter &lt;&lt;REF:Dong-2007-39-397&gt;&gt; upstream of the predicted nuclear fragment of zebrafish Atf6 (amino acids 1-366) as identified via DNA and protein alignments with human ATF6 (NCBI Reference Sequence: NP_031374.2). A cassette driving GFP expression in cardiomyocytes was included for rapid screening of transgenics (cmlc2:GFP). The transgene was flanked by Tol2 sites and the vector was injected into fertilized eggs along with transposase mRNA. Larvae were selected for cmlc2:GFP expression and raised to adulthood, outcrossed to TAB14 adults and 4 germline founders were identified.\n\nPlasmids\nThe nATF6-pcDNA-DEST47 plasmid was created using the Invitrogen Gateway System in which human nATF6 (amino acids 1-380) was amplified from a construct containing the full ORF (pEGFP-hATF6, from Dr. Aguirre-Ghiso) and ligated into pcDNA-DEST47. The nAtf6-GFP/pCI-Neo plasmid was generated by amplifying zebrafish nAtf6-GFP from a Tol2-generated plasmid (fabp10:nAtf6-GFP) with primers containing restriction sites for EcoRI and SmaI. The resulting PCR product was ligated into pCI-Neo via T4 ligase. pEGFP-C1 (Clontech) was used as a positive control for transfection.\n\nCell culture and transfection\n293T and HepG2 cells were grown in 100 mm dishes (BD Biosciences) with DMEM (Corning Cellgro, Manassas, VA) containing 10% fetal bovine serum (Invitrogen) and penicillin/streptomycin (Cellgro), and housed in a humidified incubator at 37 C with 5% CO2. Cells were passaged at 80-90% confluency either into 100 mm dishes or 6 well culture plates (Corning). Each well was transfected with 1-2 microg plasmid using Xtreme GENE 9 (Roche) or Lipofectamine &lt;&lt;REF:???-?-?-?&gt;&gt;  (Invitrogen) for 24 hours. Following transfection, cells were collected for RNA extraction using TRIzol (Invitrogen), used for oil red O staining, or labeled with 14C-acetate as described below.\n\n\n14C-acetate radiolabeling\nHepG2 cells were washed with PBS 24-28 hours after transfection and kept in serum free high glucose DMEM (Corning Cellgro, Manassas, VA) containing penicillin/streptomycin (Cellgro), 100 nM insulin (Lilly USA, Indianapolis, IN) and 0.5 microCi 14C-acetate (PerkinElmer, Waltham, MA) for 17 hours. Media was harvested and spun to remove dead cells, while cells were scraped, washed twice and lysed in cold PBS. Protein concentration in the cell lysate was measured with a BCA protein quantification kit (Thermo Scientific, Waltham, MA). Fractions of the media and cell lysate portion were used for lipid extraction as described below.\nWT and nATF6 5 dpf larvae were labeled with 1-5 microCi 14C-acetate for 48 hours. Larvae were washed twice and then sonicated in cold PBS. Protein concentration in the larval lysate was measured with a BCA protein quantification kit (Thermo Scientific). Fractions of the larval lysate were used for lipid extraction as described below.\n\nLipid extractions\nLipids from media, cells and 5 dpf larvae were extracted according to Bligh and Dyer &lt;&lt;REF:Bligh-1959-37-911&gt;&gt; with the following modifications. Briefly, methanol and chloroform were added to 400 microl media or 50 microl of 100 microl total cell lysate in a ratio of 2:1. Samples were vortexed and chloroform and 0.45% NaCl were added at a ratio of 1:1, vortexed again and phases were separated by centrifugation. The aqueous phase was re-extracted with one part chloroform and combined with the lipid from the first second extraction, washed with methanol and 0.45% NaCl at a 1:1 ratio, centrifuged and the lipid phase was recovered and dried under a stream of N2 gas. Lipids were reconstituted in a 2:1 mixture of chloroform and methanol (Thermo Scientific, Waltham, MA), dissolved in Ultima Gold and counted using a 2460 MicroBeta2 LumiJET (PerkinElmer, Waltham, MA) liquid scintillation counter. CCPM were normalized to protein concentration to account for variations in cell number.\n\nQuantitative real-time PCR (qPCR)\nTotal RNA isolated using TRIzol (Invitrogen) from pools of 15-20 dissected livers was reverse-transcribed using qScript cDNA SuperMix (Quanta Biosciences, Gaithersburg, MD). qPCR using A Light Cycler 480 (Roche) with PerfeCTa SYBRGreen FastMix (Quanta Biosciences) was used as previously described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;. Values for target gene were normalized to reference gene rpp0, and dCts calculated using the comparative threshold method (dCt  = 2-(Ct, gene - Ct, rpp0)). Primer sequences are listed in Table S1.\n\nTriglyceride measurements\nApproximately 20 livers from control and nAtf6 transgenic larvae were dissected and lysed in 0.5% Triton-X 100 and heated at 65degreesC for 5 minutes to inactivate hepatic lipases. Triglycerides were measured using the Infinity Triglyceride Liquid Stable Reagent (Thermo Fisher Scientific, Waltham, MA) following manufacturer's instructions, and were normalized to total protein concentration as determined by Bradford Assay (Bio-Rad, Hercules, CA).\n\nOil red O staining and quantification\nLarvae were fixed in 4% paraformaldehyde (PFA; Electron Microscopy Sciences, Hatfield, PA) overnight at 4degreesC, stained with oil red O, and scored for steatosis as previously described &lt;&lt;REF:Howarth-2013-10-199&gt;&gt;. Cryosections were stained by immersing slides in increasing concentrations of propylene glycol (85%, 100%) for 10 minutes followed by an overnight incubation in oil red O (0.5% in propylene glycol, Polysciences, Warrington, PA). Excess oil red O was removed the next day by sequential washes in 100% and 85% propylene glycol for 5 minutes. Nuclei were counterstained with hematoxylin.\nHepG2 and 293T cells were stained as previously described &lt;&lt;REF:Wu-2010-402-116&gt;&gt; and counterstained with hematoxylin. Oil red O droplet area and number were calculated using ImageJ. In brief, images were split into the red, green, and blue channels and the green channel was further processed for quantification as described, as oil red O has an excitation at 510 nm &lt;&lt;REF:Narasimhan-2011-7-e1001377&gt;&gt;. Following this, the background was subtracted from each image to ensure only counting of oil red O droplets. Droplets equal or greater than five square pixels were counted and area quantified. The values were copied into Microsoft Excel and the average droplet area and number of droplets per nucleus were counted for each field. Twenty independent fields from nATF6 and GFP-transfected HepG2 and 293T cells were imaged at 60x magnification.\n\nRNA sequencing (mRNA-Seq)\nTotal RNA was isolated using TRIzol (Invitrogen) from pools of ~40-50 livers dissected on 5 dpf. Two clutches of Tg(fabp10:nls-mCherry) larvae that were either untreated (control) or treated with 350 mM ethanol for 24 hours and one clutch of nAtf6 TG larvae were used. polyA-tailed mRNA was selected using oligo-dT beads and then fragmented. cDNAs were created using random-hexamers and ligated with bar-coded adaptors compatible with HiSeq &lt;&lt;REF:???-?-?-?&gt;&gt; . Single-end, 100 bp reads were sequenced at the Genomics Core of the Icahn School of Medicine at Mount Sinai. Custom-built software was used to map the reads to the zebrafish genome (Zv9/DanRer7) and estimate the coverage of each gene. Briefly, the reads were split into three 32 bp parts after trimming 2 bp at each end and mapped to the genome using a suffix-array based approach. The median of coverage across the transcript was used as an estimate of gene expression. The expression values were quantile normalized and log-ratios were calculated by comparing nAtf6 TGs to the average of the controls. Each ethanol treated sample was compared to its paired control (untreated siblings). Unique Gene Ontology terms (GO terms) were assigned to each gene by ranking the GO terms by relevance to the biology of the response, and using annotations from the human orthologues if the zebrafish annotations were lacking.\nThe distribution of expression values were plotted to identify the peak in the distribution, which is the level of noise in the system. The values were regularized by adding the noise to each gene's expression level before the log-ratios were calculated. This ensures that genes with low expression do not contribute to list of genes with large fold-changes. An absolute natural log ratio of 0.2 was used as the cutoff (known, non-responding genes are all below this threshold). The list of genes that show changes above this cutoff were analyzed for pathway enrichment using GO terms annotated as described above.\nThis data is available via the Gene Expression Omnibus (GEO) at accession number GSE56498. The controls for this dataset, untreated 5 dpf Tg(fabp10:nls-mCherry), also serve as controls in GEO dataset GSE52605 &lt;&lt;REF:???-?-?-?&gt;&gt; and were included in both sets for ease of comparison between genotypes and treatments.\n\nImage processing and graphics\nImages were cropped and minimally processed using Adobe Photoshop CS4 (Adobe Systems, San Jose, CA). Graphs were plotted using Prism 5.0c (GraphPad Software Inc., La Jolla, CA). Heat maps for mRNA-Seq data were generated using GENE-E (Broad Institute, Cambridge, MA). The global maximum and minimum for each gene set was set to orange and green, respectively, and zero was set to white. Metabolic pathway schematics were adapted from WikiPathways (URL: www.wikipathways.org).\n\nStatistical analyses\nStatistical tests were performed using GraphPad QuickCalcs (GraphPad Software, URL: http://www.graphpad.com/quickcalcs/index.cfm). For oil red O staining of whole larvae, chi-square with Fisher's Exact test were performed. For qPCR, radiolabeling, and oil red O staining of HepG2 and 293T cells we performed unpaired and one-sample t-tests as appropriate.\n\nSupplementary methods\nMethods for histological analysis, cryosectioning and staining, and liver circularity analysis are listed in Text S1.\n\nSupporting Information\nFigure S1\nInjection of an atf6 splice-blocking morpholino induces mis-splicing of atf6 mRNA. A: Schematic of boundary between intron 1 and exon 1 of zebrafish atf6. Exon sequence is shown in black and intron sequence is shown in blue, with the morpholino shown in red. The start codon is highlighted in yellow. B: Gel electrophoresis of atf6 morphants at 2 dpf showing additional bands compared to uninjected controls. C: Sequencing results from atf6 morphants displaying inclusion of intronic sequence (highlighted in blue). The predicted protein translation is shown, and early stop codons are noted as red asterisks.\n(TIF)\nClick here for additional data file.\nFigure S2\nKnockdown of atf6 does not confer resistance to morphological defects from alcohol exposure. A: Oil red O staining of cryosections from uninjected and atf6 morphant larvae treated with 0 or 350 mM ethanol for 32 hours. B: Live images of uninjected and atf6 morphant Tg(fabp10:dsRed) larvae treated with 0 or 350 mM ethanol for 32 hours. C: Quantification of liver circularity, a measure of hepatomegaly, following ethanol treatment in uninjected and atf6 morphant larvae. Livers were traced and circularity quantified in ImageJ.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nTg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae express the nAtf6-cherry transgene despite a lack of visual fluorescence. A: Schematic of human ATF6 transactivation (TAD), DNA-binding (bZIP) and transmembrane (TM) domains, cleavage sites for S1P (MBTPS1) and S2P (MBTPS2), and protein alignment with zebrafish Atf6. Numbers in the diagram correspond to amino acid positions in human ATF6. Numbers at the cleavage sites in the protein alignment correspond to amino acid positions in zebrafish Atf6. B: Confirmation of nuclear localization of zebrafish nAtf6 by transfection of nAtf6-GFP/pCI-Neo into HepG2 cells. C: Live image of Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larva at 5 dpf. D: Agarose gel electrophoresis showing expression of Cherry mRNA in nAtf6 TG but not WT larvae. rpp0 was used to ensure PCR efficacy. E: Western blot visualization of Cherry protein in adult nAtf6 transgenic zebrafish liver tissue. Tg(fabp10:nls-cherry) adult liver protein was used as a positive control. Actin was used as a loading control.\n(TIF)\nClick here for additional data file.\nFigure S4\nOverexpression of nAtf6 induces robust transcription of UPR target genes but does not activate Ire1a or Perk. A: Agarose gel electrophoresis visualization of xbp1 splicing in WT and nAtf6 TG larvae in the presence and absence of tunicamycin (Tm). The percentage of xbp1 splicing was quantified by measuring band intensity in ImageJ. B: Western blot visualization of Eif2a phosphorylation in livers of 5 dpf WT and nAtf6 TG larvae. Tunicamycin (1 microg/mL, 3-5 dpf) treated WT larvae were used as a positive control for immunoblotting. Fold was calculated over 2-3 separate experiments as noted. C: Western blot visualization of Bip protein in livers of 5 dpf WT and nAtf6 TG larvae. Fold change (below blot) was calculated from 3 separate experiments. D and E: Quantitative real-time PCR of multiple UPR effectors in the livers of WT and nAtf6 TG larvae at 5 (D) and 14 (E) dpf. Statistics: unpaired t-test. *, p&lt;0.05.\n(TIF)\nClick here for additional data file.\nFigure S5\nLipid accumulation is evident in nAtf6 TG larvae in the absence of other hepatic injury. A: Staining of cryosections with oil red O showing significant steatosis in nAtf6 TG larvae at 5 and 14 dpf. B: Hematoxylin and eosin staining of nAtf6 TG and control larvae at 5 dpf. No other alterations in hepatic pathology were observed. The number of larvae used is indicated.\n(TIF)\nClick here for additional data file.\nFigure S6\nmRNA-Seq reveals potential disruptions in lipid transport by nAtf6 overexpression. Heatmap of genes associated with lipid binding and transport (A) and basic schematic of lipid synthesis and lipoprotein circulation (B, adapted from WikiPathways). Upregulated genes are colored in shades of orange, and downregulated genes are colored in shades of green. Genes that are unchanged are colored white. Genes that did not appear in mRNA-Seq analyses are colored gray in the schematic.\n(TIF)\nClick here for additional data file.\nFigure S7\nnAtf6 overexpression induces transcription of genes involved in glyceroneogenesis. Quantitative, real-time PCR analysis of gene expression from WT and nAtf6 TG larvae at 5 dpf (A) and 14 dpf (B). Statistics: unpaired t-test. *, p&lt;0.05. For (A), \"n\" corresponds to the number of clutches used and approximately 8-12 livers were dissected for RNA extraction. For (B), individual livers were dissected for analysis.\n(TIF)\nClick here for additional data file.\nFigure S8\nOverexpression of nATF6 in 293T cells drives lipid accumulation. A: Staining of GFP and nATF6 transfected cells with oil red O. B: Quantification of lipid droplet number and area (square pixels). Droplet number and area were calculated using ImageJ. Statistics: unpaired t-test. *, p&lt;0.05. The number of cells used for quantification is noted.\n(TIF)\nClick here for additional data file.\nFigure S9\nSrebp1 and Srebp2 target genes are not dysregulated by nAtf6 overexpression. A and B: qPCR analysis of Srebp1 (A) and Srebp2 (B) target genes in atf6 morphants treated with 350 mM EtOH for 32 hours. Fold changes were calculated based on median dCt values. C and D: qPCR analysis of Srebp1 (C) and Srebp2 (D) target genes in Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae at 5 dpf. Fold changes were calculated based on median dCt values, and \"n\" corresponds to the number of clutches used.\n(TIF)\nClick here for additional data file.\nFigure S10\nAtf6 and Srebps do not epistically interact to prevent alcoholic steatosis. Steatosis incidence in larvae injected with atf6, scap, or atf6 + scap MO. \"Total n\" corresponds to the number of larvae scored. \"Clutch n\" corresponds to the number of clutches analyzed. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05 and corresponds to groups significantly different from uninjected controls. b, p&lt;0.05 and corresponds to ethanol-treated groups significantly different from uninjected, ethanol treated larvae.\n(TIF)\nClick here for additional data file.\nTable S1\nMorpholino and primer sequences. A: Morpholino sequences. Amount of morpholino injected is based on the assumption that 4 nl of MO is injected per embryo. B: Primer sequences used for conventional and quantitative, real-time PCR.\n(DOCX)\nClick here for additional data file.\nTable S2\nUPR effectors upregulated by nAtf6 and by ethanol. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Log values &gt;= 0.2 are highlighted in red.\n(XLSX)\nClick here for additional data file.\nTable S3\nUpregulated and downregulated genes in the liver of nAtf6 TG 5 dpf larvae. A: Upregulated genes in the liver of nAtf6 TG 5 dpf zebrafish larvae. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. UPR target genes are highlighted in pink. GO terms were assigned as described in the Materials and Methods. B: Downregulated genes in the liver of nAtf6 TG 5 dpf larvae. Genes regulating lipoprotein secretion are highlighted in orange.\n(XLSX)\nClick here for additional data file.\nTable S4\nRaw data from mRNA-Seq. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Genes are organized from highest to lowest log value for nAtf6 TG larvae.\n(XLS)\nClick here for additional data file.\nText S1\nSupplementary methods.\n(DOC)\nClick here for additional data file.'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nThe unfolded protein response (UPR) acts in most cells to maintain homeostasis within the protein secretory pathway during physiological conditions. During stress, the UPR becomes further induced to mitigate the accumulation of misfolded or unfolded proteins in the endoplasmic reticulum (ER). If UPR activation cannot relieve the excess protein load, the ER becomes dilated and dysfunctional, and all branches of the UPR remain chronically activated in a condition referred to as ER stress. Many studies have implicated ER stress in a range of pathologies, and there is a clear association between UPR activation and metabolic diseases such as fatty liver disease (FLD) &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;-&lt;&lt;REF:Rinella-2011-54-1600&gt;&gt;. However, it is not known whether factors that control the UPR can also directly impact lipid metabolism and it remains unclear how UPR activation causes FLD.\nFLD is the most common hepatic pathology worldwide &lt;&lt;REF:???-?-?-?&gt;&gt;, and alcohol abuse is a leading cause. Even a single episode of binge drinking causes lipid accumulation in hepatocytes (steatosis) in over 90% of drinkers &lt;&lt;REF:???-?-?-?&gt;&gt;. While acute steatosis can resolve, chronic steatosis can render hepatocytes susceptible to damage and is a prerequisite step in developing more severe liver disease, including steatohepatitis and cirrhosis. Most FLD etiologies are accompanied by impairment of protein secretion by hepatocytes that result in serum protein deficiencies, which are most apparent in chronic alcoholics and contribute to the systemic complications of alcoholic liver disease (ALD). These defects are reflected in many reports demonstrating that alcohol induces some UPR sensors and targets in the liver of mice &lt;&lt;REF:Ji-2003-124-1488&gt;&gt;, &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, rats &lt;&lt;REF:Shinohara-2010-51-796&gt;&gt;, micropigs &lt;&lt;REF:Esfandiari-2005-289-G54&gt;&gt;, and zebrafish &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, deleting ATF4 or CHOP, two key UPR effectors, reduces ethanol-induced liver injury &lt;&lt;REF:Ji-2005-29-1496&gt;&gt;, &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, indicating that UPR activation can contribute to ALD. However, it is not clear whether these genes, or other UPR effectors, contribute to steatosis.\nThere are three main UPR sensors that function through shared and independent mechanisms to maintain ER function by enhancing protein processing and folding in the ER, and by promoting degradation of terminally misfolded secretory proteins. Activating transcription factor 6 (ATF6) and inositol-requiring enzyme 1-alpha (IRE1alpha or ERN1) pathways generate the active transcription factors nATF6 and XBP1, respectively, that induce hundreds of UPR target genes &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. PRKR-like endoplasmic reticulum kinase (PERK or EIF2AK3) phosphorylates EIF2A to repress translation &lt;&lt;REF:Harding-2000-5-897&gt;&gt; and to promote production of ATF4 &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;, which induces a subset of target genes &lt;&lt;REF:Magne-2011-54-729&gt;&gt;, &lt;&lt;REF:Harding-2000-6-1099&gt;&gt;. Each of these primary sensors has been evaluated for its contribution to FLD caused by a robust ER stressor &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt;. ATF6 knockout mice fail to resolve steatosis caused by acute stress due to tunicamycin injection &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Yamamoto-2010-21-2975&gt;&gt;, suggesting that loss of ATF6 promotes fatty liver. Our previous work using zebrafish confirmed the finding that steatosis caused by acute stress is augmented by atf6 loss, and also demonstrates that steatosis caused by chronic stress is reduced when Atf6 is depleted &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it appears that atf6 loss can alternatively enhance or reduce FLD, depending on the nature and duration of the stress.\nThe relationship between metabolic disease and UPR activation is under intensive investigation, yet several important questions remain unanswered. First, is UPR activation a cause or consequence of FLD? While some studies indicate that lipotoxicity and fatty acid accumulation can induce the UPR &lt;&lt;REF:Wei-2006-291-E275&gt;&gt;, &lt;&lt;REF:Wang-2006-147-943&gt;&gt;, there is incontrovertible evidence that robust UPR activation is sufficient to induce steatosis &lt;&lt;REF:Lee-2012-211-29&gt;&gt;, &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;. These data are incorporated into a current model proposing that robust UPR activation can cause steatosis and, if the lipid burden is not resolved, this can further augment cellular stress and contribute to chronic UPR induction. The second question is which, if any, of the main UPR factors directly cause fatty liver? All studies to date that have addressed this question, including our work in zebrafish &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, utilize loss of function approaches whereby a key UPR gene is deleted and the effect on stress-induced steatosis is evaluated. However, these studies are complicated by the extensive crosstalk between UPR effectors, and by the ability of cells to adapt to changes in UPR capacity. Third, what is the mechanism by which UPR activation causes fatty liver? Activation of lipogenic genes by the sterol response element binding protein (SREBP) transcription factors has been implicated as an essential pathway in both alcoholic &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt; and non-alcoholic &lt;&lt;REF:Higuchi-2008-38-1122&gt;&gt;-&lt;&lt;REF:Zhang-2012-212-229&gt;&gt; FLD. Since SREBPs and ATF6 are activated by a shared proteolytic mechanism &lt;&lt;REF:Ye-2000-6-1355&gt;&gt;, it is possible that these two proteins are activated simultaneously in response to stress. However, the role of SREBPs in FLD caused by UPR activation has not been addressed.\nHere, we use genetic and genomic approaches in a well-established zebrafish model of ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;-&lt;&lt;REF:Liu-2013-6-1271&gt;&gt; to demonstrate that blocking Atf6 prevents alcohol-induced steatosis and that overexpression of the nuclear, active form of Atf6 (nAtf6) causes steatosis. We found that nAtf6 overexpression increases expression of genes in the lipogenic pathway and alters lipid flux to promote triglyceride synthesis by a mechanism that does not involve Srebps. Instead, we demonstrate an epistatic interaction between atf6 and fasn in driving alcoholic steatosis. These findings demonstrate the first clear and causative link between a central UPR sensor and FLD.\n\nResults\n\nAtf6 is required for alcoholic steatosis\nWe previously reported that transcription of multiple UPR effector genes and translation of the major ER chaperone, Bip, are induced in zebrafish livers following alcohol exposure &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;. We have also demonstrated that xbp1 splicing, a direct measure of Ire1a activation, is only detectable within the first few hours of exposure to ethanol &lt;&lt;REF:???-?-?-?&gt;&gt; yet UPR target gene induction persists long afterwards &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, suggesting that the Ire1a pathway is not entirely responsible for the UPR in ALD. Our previous finding that Eif2a is phosphorylated in the liver of zebrafish with ALD &lt;&lt;REF:???-?-?-?&gt;&gt; suggests that Perk is activated by alcohol, but since Eif2a can also be phosphorylated by other kinases &lt;&lt;REF:Dever-1992-68-585&gt;&gt;, this has yet to be resolved. The lack of reliable antibodies that recognize the active form of Atf6, Perk and Ire1a in zebrafish hampered direct measurement of the activation status of each pathway (not shown). However, quantitative real-time PCR (qPCR) analysis of liver samples from 5 day post fertilization (dpf) larvae treated with alcohol demonstrated that atf6 mRNA was significantly induced in the liver (Figure 1A &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;) prior to the onset of steatosis (see Figure 1B &lt;&lt;FIG:pgen-1004335-g001&gt;&gt; and &lt;&lt;REF:???-?-?-?&gt;&gt;). Since Atf6 has been shown to act in a positive feedback loop to induce its own expression &lt;&lt;REF:Namba-2007-355-543&gt;&gt;, these data suggest that the increase in atf6 expression could reflect Atf6 activation in response to alcohol.\nFigure 1\nAtf6 is required for alcoholic steatosis.\n\nA: qPCR analysis of UPR sensors atf6, perk, ire1a, and atf4 in the livers of control (EtOH-) and 350 mM ethanol-treated (EtOH+) larvae. dCt values were calculated by normalization to rpp0. Fold changes were calculated based on median dCt values. Statistics: paired t-test. *, p&lt;0.05. B: Expression of atf6 in the livers of 350 mM ethanol treated larvae from 0-32 hours. Fold changes were determined by normalizing to control (0 mM EtOH) dCt values. Statistics: one-sample t-test. *, p&lt;0.05. C: Images of oil red O stained whole larvae injected with two different morpholinos targeting atf6 (ATG-targeting, atf6 ATG, and splice-blocking, atf6 SPL) and exposed to either 0 or 350 mM ethanol. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring oil red O stained larvae at 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored.\nTo test whether Atf6 was required for alcoholic steatosis, we injected a morpholino (MO) targeting either the translation initiation site (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; or the boundary between coding exon 1 and intron 1 of atf6 (atf6MO-SPL) (Table S1 and Figure S1A), which blocked splicing (Figure S1B) and introduced an early stop codon (Figure S1C). Morphants and uninjected controls were treated with 350 mM ethanol for 32 hours, stained with the neutral lipid dye, oil red O and scored for the presence or absence of steatosis (Figure 1C and S &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;2A). Consistent with previous findings &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;-&lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:North-2010-107-17315&gt;&gt;, the percent of larvae with steatosis (i.e. steatosis incidence) was significantly higher in uninjected larvae exposed to 350 mM ethanol for 32 hours (13% in untreated larvae vs. 53% in ethanol treated larvae; p&lt;0.05). atf6 morphants had reduced steatosis incidence (p&lt;0.05, Figure 1C-D &lt;&lt;FIG:pgen-1004335-g001&gt;&gt;), but still displayed other gross morphological abnormalities caused by ethanol including increased liver circularity, an indication of hepatomegaly (Figure S2B-C), a common feature of liver disease. Thus, Atf6 is required for alcoholic steatosis.\n\nnAtf6 overexpression in hepatocytes induces robust UPR activation\nTo determine whether Atf6 is sufficient to cause FLD, we created a transgenic zebrafish line expressing the predicted nuclear, active fragment of zebrafish Atf6 (amino acids 1-366; Figure S3A) fused to mCherry under the hepatocyte-specific promoter fabp10, along with a cassette that expressed GFP in cardiomyocytes as a marker of transgenesis ((Tg(fabp10:nAtf6-cherry; cmlc2:GFP) hereafter called nAtf6 TG). There is high sequence identity between human and zebrafish Atf6 proteins in the transmembrane, leucine zipper and basic (bZIP) domains and the protease recognition sites are highly conserved (Figure S3A). To confirm its nuclear localization, the predicted nAtf6 fragment of the zebrafish protein was fused to GFP and transfected into HepG2 cells (Figure S3B).\nTransgenics were selected based on GFP expression in the heart, appeared grossly normal throughout development (Figure S3C), and developed into viable, fertile adults. nAtf6-mCherry was not detectable using low-resolution fluorescence microscopy in larvae from all four of the transgenic lines we generated. However, we confirmed transgene expression by PCR and blotting for mCherry (Figures S3D-E). atf6 mRNA expression in the liver was ~4 fold higher in nAtf6 TG larvae compared to wildtype (WT) larvae (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4) at 5 dpf, and was persistently elevated in older fish (Figure S4D and not shown). We detected mCherry mRNA in 5 dpf nAtf6 TG larvae by PCR, and Western blotting for mCherry detected the transgene in the liver of adult fish, albeit at much lower levels than achieved in transgenic fish expressing nuclear-localized mCherry without fusion to nAtf6 (i.e. Tg(fabp10:nls-mCherry); &lt;&lt;REF:???-?-?-?&gt;&gt;\nFigures S3D-E).\nFigure 2\nnAtf6 overexpression is sufficient to drive steatosis.\n\nA: Heatmap of upregulated UPR effector genes in nAtf6 transgenic larvae. The log values based on mRNA-Seq analysis and the median fold change in 3-6 liver samples assessed by qPCR at 5 dpf and 14 dpf. B: Most Atf6 targets are upregulated in the liver in response to ethanol. Genes identified as part of the UPR are significantly induced in the liver of nAtf6 TG and ethanol treated larvae (log value &gt;= 0.2; see Table S2). C: Oil red O stained WT and nAtf6 transgenic larvae. Livers are circled in the enlarged boxes. D: Steatosis incidence based on scoring of whole mount oil red O stained larvae at 4, 5, and 5.5 dpf. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored. E: Hepatic triglyceride levels in extracts from pooled livers from WT and nAtf6 transgenic larvae at 5 dpf and 14 dpf, and from single adult livers (~9 months). Statistics: unpaired t-test. *, p&lt;0.05. Median fold changes are noted.\nWe confirmed that nAtf6 overexpression did not induce the other UPR branches by assessing xbp1 splicing (Figure S4A) and Eif2a phosphorylation (Figure S4B) in the liver of nAtf6 TG 5 dpf larvae. We found no difference in these markers in unstressed nAtf6 TG and WT larvae, however, xbp1 splicing was lower in transgenics than in WTs after exposure to stress (i.e. 1 microg/ml tunicamycin for 48 hours; Figure S4A). This suggests that nAtf6 overexpression adapts hepatocytes to withstand a robust, pharmacologically-induced ER stress.\nWe next examined the expression of UPR target genes in the liver of 5 dpf transgenic and control larvae. We found that bip (also called hspa5), a transcriptional target of Atf6, was elevated at the protein (Figure S4C) and mRNA levels (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D) in nAtf6 TG larvae. mRNA-Seq analysis comparing control larvae (Tg(fabp10:nls-mCherry); n = 2 pools of ~40-50 livers each; see &lt;&lt;REF:???-?-?-?&gt;&gt; and GEO dataset GSE52605), ethanol-treated (n = 2 pools of ~40-50 livers each), and nAtf6 TG larvae (n = 1 pool of ~40-50 livers) revealed 33 UPR target genes to be highly and significantly upregulated in nAtf6 TG livers (Tables S2-S4 and Figure 2A &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;; GEO dataset GSE56498). These genes were identified as ATF6 target genes in other studies &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; or by having the UPR or protein folding as a primary GO descriptor, and included ER chaperones (bip/hspa5, dnajc3, and grp94/hsp90b1), quality control effectors (canx, calret, calrl, and calrl2), protein disulfide isomerases (pdia3, pdia4) and ERAD components (edem1, derl1 and derl3). A subset of these genes was confirmed by qPCR at 5 dpf and at 14 dpf (Figure 2A and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;4D-E). atf4 and derl1 were the only genes detected as significantly upregulated by qPCR but not by mRNA-Seq.\nAnalysis of the mRNA-Seq data from ethanol-treated larvae revealed a striking overlap between the \"UPR-ome\" induced by ethanol and by nAtf6 overexpression: of the 49 UPR target genes induced by ethanol, 30 were also induced in the liver of nAtf6 TG larvae (Figure 2B &lt;&lt;FIG:pgen-1004335-g002&gt;&gt; and Table S2). It is possible that the remaining 19 genes unique to the ethanol-induced transcriptome are attributed to the transient, early increase in xbp1 splicing &lt;&lt;REF:???-?-?-?&gt;&gt; or to other transcription factors that regulate their expression. Collectively, these data indicate that Atf6 is a major regulator of the UPR in ALD.\n\nAtf6 overexpression causes steatosis\nWe next asked whether nAtf6 was sufficient to cause FLD (Figure 2C-E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Induction of the fabp10:nAtf6-cherry transgene occurs at ~2.5 dpf &lt;&lt;REF:Her-2003-538-125&gt;&gt; and by 4 dpf, 44% of nAtf6 TG larvae developed steatosis which increased to 69% and 76% by 5 and 5.5 dpf, respectively (Figure 2C-D and S &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;5A) and persisted until at least 14 dpf (Figure S5A), but there were no marked histological abnormalities in the nAtf6 TG liver at 5 dpf (Figure S5B). Hepatic triglyceride (TAG) levels were nearly doubled at 5 dpf, and six times higher in 14 dpf nAtf6 TG larvae (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), clutch-to-clutch variability non-withstanding. However, TAG levels were the same in WT and nAtf6 TG adults (Figure 2E &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;), despite persistent expression of the transgene (Figure S3E and not shown). We speculate that, over time, animals adapt to nAtf6 overexpression or that other metabolic parameters affect hepatic lipid accumulation in adult fish. Together, our data demonstrate that Atf6 is necessary for alcoholic steatosis and sufficient to cause steatosis in the absence of any other stress.\n\nnAtf6 overexpression alters expression of genes involved in lipogenesis and lipid export\nSteatosis is caused by elevated TAGs in hepatocytes resulting from increased lipid synthesis or uptake, or from decreased lipid utilization or export. We therefore analyzed pathways relevant to these processes in the mRNA-Seq dataset from nAtf6 TG livers to determine if any were dysregulated. TAGs are generated from linking free fatty acids (FFAs) to a glycerol backbone, which is generated by conversion of dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P) by G3P dehydrogenase (GPD1) (see schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). We found genes that participate in TAG synthesis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;) or export (Figure S6) were dysregulated in nAtf6 TG livers at 5 dpf, albeit not to the same degree as observed for the UPR target genes (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). DHAP is generated via glycolysis or by glyceroneogenesis, a pathway that has significant overlap with gluconeogenesis (see &lt;&lt;REF:Nye-2008-283-27565&gt;&gt; and schematic in Figure 3A &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). mRNA-Seq analysis demonstrated that several genes involved in glycolysis were downregulated in nAtf6 TG livers including aldob, glut2/slc2a2, gck, pgk1, and pklr (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Genes promoting glyceroneogenesis, particularly pck1 and got1, which generates oxaloacetate for conversion to phosphoenoylpyruvate by pck1, were upregulated, as was gpd1b, which drives glycerol formation. This suggests that nAtf6 overexpression increases factors regulating glyceroneogenesis preferentially over those that promote glycolysis (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). In addition, ech1 and tecrb, two genes responsible for fatty acid elongation in the mitochondria and ER, respectively, were induced in nAtf6 TG livers by mRNA-Seq (Figure 3A-B &lt;&lt;FIG:pgen-1004335-g003&gt;&gt;). Some of these genes were confirmed by qPCR analysis of nAtf6 TG livers, with the median fold change from qPCR on 3 to 6 clutches of livers from 5 and 14 dpf larvae depicted in Figure S7.\nFigure 3\nGlyceroneogenesis and fatty acid elongation pathways are dysregulated by nAtf6 overexpression.\nSchematic of the glycolysis, gluconeogenesis and glyceroneogenesis pathway (A, adapted from WikiPathways) and heatmap of associated genes (B). Upregulated genes are colored in shades of orange, downregulated genes are colored in shades of green, genes that are unchanged are white and those genes that did not appear in mRNA-Seq analyses are colored gray. Log values and median fold changes from mRNA-Seq analysis and qPCR, respectively, are noted.\nTAGs are assembled in the hepatocyte ER and secreted in association with lipoproteins that are required for transport in plasma. Several genes that regulate lipoprotein assembly and export, including apom, apoeb, apoea, cetp and zgc:162608 and zgc:194131, two apolipoprotein-like genes, were downregulated in nAtf6 TG livers (Figure S7 and Table S3). pdia2, which interacts with MTP to transport lipids out of hepatocytes &lt;&lt;REF:Wetterau-1997-1345-136&gt;&gt;, was also downregulated. These data suggest that hepatic lipid transport may be impaired by nAtf6 overexpression. While these expression data do not capture the complex metabolic and post-translational regulation of the proteins encoded by these genes, they do suggest enhanced lipogenesis or suppressed lipoprotein export as potential mechanisms of steatosis in nAtf6 transgenics.\n\n\nDe novo lipogenesis is enhanced by nAtf6 overexpression\nAcetate is a precursor to acetyl-CoA, the major building block of fatty acid and, therefore, TAG synthesis. To determine if nAtf6 overexpression increased de novo lipogenesis, we measured the incorporation of 14C-acetate into lipids in nAtf6 TG larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) and in HepG2 cells overexpressing nATF6 (Figure 4B-D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) compared to their respective controls. Larvae were incubated with 14C-acetate from 3-5 dpf and we measured the amount of radiolabel present in the lipid fraction compared to that present in the lysate of the whole larvae. In an average of 5 individual pools of larvae, more 14C radiolabel was detected in the lipid fraction of nAtf6 TG larvae than in wildtype larvae (Figure 4A &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;).\nFigure 4\n\nDe novo lipogenesis is enhanced in vitro and in vivo by nAtf6 overexpression.\n\nA:\n14C-acetate is preferentially incorporated into lipids in 5 dpf nAtf6 TG larvae compared to WT. The percent of 14C in lipid fraction was divided by 14C measured in the unextracted lysate from whole fish. The median fold change is noted; n =  number of samples of pooled larvae. B: Oil red O staining of HepG2 cells transfected with GFP or nATF6. Bar  = 10 microm. C: Quantification of lipid droplet number and area (square pixels). Statistics: unpaired t-test. *, p&lt;0.05; n indicates the number of cells quantified over 20 fields from 2 independent experiments. D: Incorporation of 14C-acetate into lipids by HepG2 cells transfected with GFP or nATF6 normalized to protein. The median fold change is noted; n =  number of separate batches of cells analyzed in 2 independent experiments.\nSince the liver in 5 dpf zebrafish larvae is too small to analyze incorporation in this organ exclusively, analysis of the whole larvae could not determine whether the label was preferentially incorporated into hepatocytes of nAtf6 TG larvae. We thus used human hepatoma (HepG2) cells transfected with human nATF6 to address this question. nATF6 overexpression causes accumulation of oil red O stained cytoplasmic droplets (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;) that are both greater in number and larger in size than in cells transfected with GFP (Figure 4B &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). These findings were confirmed in 293T cells (Figure S8) and correlate with our findings in nAtf6 TG larvae (Figure 2 &lt;&lt;FIG:pgen-1004335-g002&gt;&gt;). Thus, nAtf6 overexpression causes lipid accumulation across cell types and species. Moreover, in eleven independent cell samples from two separate experiments, HepG2 cells overexpressing nATF6 had slightly more 14C radiolabel in the lipid fraction compared to GFP-transfected cells (Figure 4D &lt;&lt;FIG:pgen-1004335-g004&gt;&gt;). While we found that the incorporation tended to be higher in the nATF6 expressing cells, the average increase was moderate, at best. We attribute this to the transfection efficiency of HepG2 cells combined with experimental conditions used to label lipids, which likely act to select against nATF6 expressing cells.\n\nSrebp activation does not contribute to Atf6-mediated steatosis\nFatty acid synthase (fasn) mediates multiple steps in fatty acid synthesis that culminate in generation of palmitate from acetyl-CoA, a building block for TAG biosynthesis. fasn transcription in hepatocytes is primarily regulated by Srebp1c &lt;&lt;REF:Shimano-1997-100-2115&gt;&gt;-&lt;&lt;REF:Shimano-1996-98-1575&gt;&gt;, and we previously demonstrated that fasn transcripts and other Srebp target genes are induced in zebrafish with ALD &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Moreover, blocking Srebp activation by injecting a morpholino targeting the Srebp activating protein, Scap, reduced the incidence of alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt;. Since Atf6 and Srebps are activated by similar mechanisms - they are both retained as inactive precursors in the ER and processed by the same Golgi-resident enzymes &lt;&lt;REF:Ye-2000-6-1355&gt;&gt; - Srebp mediated lipogenesis has been proposed as a mechanism by which UPR activation causes FLD &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;. We tested this using a genetic approach. First, we asked whether blocking Atf6 affected the induction of lipogenic gene expression in the liver of ethanol treated larvae. Both fasn (Figure 5A &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;), and acc1 (acaca;\nFigure S9A) were upregulated by ~2 fold in the liver of ethanol treated larvae, and this was blocked by atf6 MO injection (Figures 5A and S &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;9A). However, induction of other Srebp target genes was not blocked by Atf6 depletion, including srebp1 (Figure S9A) and the Srebp2 targets hmgcra, hmgcs1 and mvk (Figure S9B). Since the entire Srebp target gene set was not uniformly affected by atf6 loss, we conclude that Atf6 does not interact with Srebps. Instead, Atf6 may affect expression of specific lipogenic genes (i.e. fasn and acc1) by a mechanism distinct from Srebp1. Interestingly, we found mild induction of Srebp2 target genes in untreated atf6 morphants (Figure S9B), supporting conclusions based on experiments in mammalian cultured cells where ATF6 was found to directly suppress SREBP2 activity &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;.\nFigure 5\nFasn functions downstream of Atf6 to cause steatosis independent of Srebp activation.\n\nA: qPCR analysis of fasn expression in atf6 morphants and uninjected larvae treated with EtOH for 32 hours. B: qPCR analysis of fasn expression in nAtf6 TG larvae and HepG2 cells transfected with nATF6. Statistics: unpaired t-test. *, p&lt;0.05. C: Quantification of whole mount oil red O staining of nAtf6 TG larvae injected with scap MO. \"Total n\" and \"clutch n\" corresponds to the number of larvae and number of clutches scored, respectively. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05. D: Quantification of oil red O staining in nAtf6 TG larvae injected with fasn MO. E: Quantification of oil red O staining in atf6/fasn double morphants treated with 350 mM EtOH for 24 hours. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05.\nNext, we asked whether nAtf6 was sufficient to induce fasn and acc1 by assessing their expression in livers of nAtf6 TG larvae. Both fasn (p&lt;0.05; Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;) and acc1 (Figure S9C) were increased by ~2 fold at 5 dpf, and fasn expression was also higher at 14 dpf in nAtf6 TG larvae and in HepG2 cells overexpressing nATF6 (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). However, other Srebp1 and Srebp2 genes were not significantly induced by nAtf6 overexpression in nAtf6 TG larvae (Figure S9C-D) or in HepG2 cells overexpressing human nATF6 (not shown), indicating that nAtf6 does not activate the entire Srebp target gene program. We then tested whether nAtf6 functioned upstream of Srebps by assessing whether scap morpholino injection suppressed steatosis in nAtf6 TG larvae. While scap morphants are resistant to ethanol-induced steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt; we found no difference in steatosis incidence in nAtf6 transgenics (Figure 5B &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Finally, to test whether atf6 and scap interacted to cause alcoholic steatosis we injected low concentrations of morpholinos targeting both genes and found that they did not synergize to suppress alcoholic steatosis (Figure S10). We thus conclude that lipogenic gene induction and steatosis caused by nAtf6 overexpression does not require Srebps.\n\nSteatosis in nAtf6 TG larvae requires fasn\n\nThe functional relevance of the finding that nAtf6 overexpression increases lipogenesis was tested by assessing the effect of morpholino-mediated fasn (Table S1) knockdown on steatosis incidence in nAtf6 TG larvae. Injection of high concentrations of fasn MO induced severe morphological defects, so we optimized the fasn MO concentration to have minimal toxicity. This had no effect on steatosis incidence in control larvae, but reduced steatosis incidence in nAtf6 TG larvae from 74% to 42% (Figure 5C &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Furthermore, we found that atf6 and fasn interacted epistatically to cause alcoholic steatosis by co-injecting concentrations of fasn and atf6-SPL MOs that did not have any effect on alcoholic steatosis on their own but, together, significantly reduced steatosis incidence in response to ethanol (Figure 5D &lt;&lt;FIG:pgen-1004335-g005&gt;&gt;). Thus, Atf6 and Fasn function in the same genetic pathway. Taken together, our data suggest a model by which Atf6 causes steatosis, in part, by inducing fasn and TAG synthesis by a mechanism that is independent of Srebps (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;).\nFigure 6\nWorking model by which Atf6 functions as a positive regulator of alcoholic steatosis.\nGenetic tools used in this study are illustrated in red.\n\nDiscussion\nThere is a wealth of recent literature showing that the UPR is activated in FLD and that this aspect of the disease is conserved across species &lt;&lt;REF:Puri-2008-134-568&gt;&gt;, &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, &lt;&lt;REF:Kaplowitz-2006-21-S7&gt;&gt;-&lt;&lt;REF:Ye-2010-59-6&gt;&gt;. However, it has not been shown that any single UPR factor can directly cause this disease and the mechanism by which UPR activation causes lipid accumulation is unclear. Here, we use zebrafish genetics and biochemical analyses to show that Atf6 is both necessary and sufficient for steatosis by showing that: (i) Atf6 is required for alcoholic steatosis, (ii) activation of Atf6 is sufficient to cause steatosis, (iii) Atf6 activation induces expression of genes involved in glyceroneogenesis and fatty acid elongation and causes de novo lipogenesis, and (iv) Atf6 epistatically interacts with fatty acid synthase (fasn), a key enzyme involved in TAG biosynthesis, to cause FLD. This is among the first data to directly link a main UPR sensor and lipid metabolic pathways. \nA number of studies have demonstrated that loss of one of the key UPR sensors - PERK &lt;&lt;REF:Teske-2011-22-4390&gt;&gt;, IRE1alpha &lt;&lt;REF:Zhang-2011-30-1357&gt;&gt; or ATF6 &lt;&lt;REF:Wu-2007-13-351&gt;&gt;, &lt;&lt;REF:Rutkowski-2008-15-829&gt;&gt;, &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt; - enhances steatosis caused by acute stress. Additionally, we previously demonstrated that Atf6 depletion suppresses steatosis caused by chronic stress &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;. Thus, it is clear that UPR activation is a conserved, common feature of FLD, but whether it functions to promote or prevent steatosis appears to depend on the nature and duration of the FLD-causing stress. Importantly, these loss of function studies have described a requirement for the UPR in FLD caused by tunicamycin injection, but not in the context of a stress that mirrors human conditions, such as obesity or alcohol abuse. Here, we show that Atf6 is induced in ALD, that its activation precedes steatosis, and that knocking down Atf6 reduces alcoholic steatosis. Thus, Atf6 is required for ALD. Moreover, since overexpression of nAtf6 is sufficient to cause FLD in the absence of any other stress, we conclude that this branch of the UPR is a pathophysiological mechanism of FLD.\nAtf6 and Xbp1 are the primary transcription factors that both independently and cooperatively regulate expression of hundreds of UPR target genes. We previously found significant upregulation of many UPR target genes in zebrafish with ALD &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:???-?-?-?&gt;&gt; and hypothesized that since xbp1 splicing in the liver was only an early, transient response to alcohol &lt;&lt;REF:???-?-?-?&gt;&gt;, other transcription factors must participate in this robust UPR. Transcriptome analysis of the livers from nAtf6 TG and ethanol treated larvae enabled us to identify a set of target genes that are likely to be directly activated by Atf6. Many of these genes were also reported to be upregulated in mice overexpressing inducible ATF6 in the heart &lt;&lt;REF:Belmont-2008-283-14012&gt;&gt; and they were not induced by tunicamycin treatment of Atf6-/- MEFs &lt;&lt;REF:Wu-2007-13-351&gt;&gt;. Genes occupying the intersection of these different datasets are likely bona fide Atf6 targets and the transgenic larvae we generated provides a system to test this directly. In contrast, many genes that were downregulated when ATF6 was overexpressed in the mouse heart were unchanged in nAtf6 transgenic zebrafish livers and, conversely, genes such as canx, derl1, and atf4 were upregulated in the nAtf6 TG zebrafish liver samples but not in the mouse model. The variations between the two datasets may be due to inherent differences in the models or approaches used to detect changes in gene expression, or may reflect the ability of Atf6 to differentially regulate target genes depending on cell type: hepatocytes possess a significantly higher secretory capacity than cardiomyocytes, and thus the Atf6 transcriptome in hepatocytes may be more extensive in order to maintain ER homeostasis. Finally, comparative transcriptome analysis between nAtf6 TG and ethanol treated larvae identified significant overlap between the UPR target genes in these two datasets, indicating that Atf6 is the main transcriptional driver of the ethanol-induced \"UPR-ome\".\nWhat is the mechanism by which the UPR causes FLD? Our data showing that nAtf6 overexpression induces the expression of some lipogenic enzymes and increases lipid synthesis, and that blocking fatty acid synthesis reduces FLD, fits a model (Figure 6 &lt;&lt;FIG:pgen-1004335-g006&gt;&gt;) whereby increased lipid production is, in part, the mechanism of steatosis in FLD caused by UPR activation. However, despite our finding that Atf6 depletion suppresses fasn expression and nAtf6 overexpression induces fasn, we have no evidence that fasn is a direct transcriptional target of Atf6. Indeed, the level of fasn induction is far less than the established Atf6 target, bip, and there are no canonical UPREs or ERSEs &lt;&lt;REF:Yamamoto-2004-136-343&gt;&gt; in the fasn promoter (not shown), thus other pathways are also likely at play. The mechanism by which nAtf6 induces fasn and acc1 require further investigation, and the tools we describe here will facilitate such studies.\nThe SREBP transcription factors are well-characterized regulators of hepatic lipogenesis; SREBP1c functions by increasing the expression of the full panel of genes required for TAG biogenesis &lt;&lt;REF:Horton-2003-100-12027&gt;&gt;. One possibility, suggested by the finding that the UPR and SREBPs are activated in parallel in some systems &lt;&lt;REF:Colgan-2007-39-1843&gt;&gt;, &lt;&lt;REF:Kammoun-2009-119-1201&gt;&gt;, &lt;&lt;REF:Zhang-2012-212-229&gt;&gt;, is that nAtf6 causes Srebp activation, leading to increased expression of fasn and other lipogenic genes and promoting lipogenesis. Our data argues against this: (i) the full panel of Srebp targets are not induced by nAtf6 overexpression in zebrafish, (ii) Atf6 and Srebps did not epistatically interact to modify alcoholic steatosis and (iii) Srebp activation is not required for steatosis in nAtf6 TG larvae. While it does appear that Srebp activation is required for alcoholic steatosis &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, &lt;&lt;REF:You-2004-34-39&gt;&gt;, &lt;&lt;REF:You-2002-277-29342&gt;&gt;, the current study shows that it is not the only important pathway that regulates hepatic lipid metabolism, as steatosis in nAtf6 TG larvae is independent of Srebps.\nWhile Srebps are not required for the effects of Atf6 on steatosis, there does appear to be some interaction between Atf6 and Srebp2, consistent with in vitro data showing that ATF6 suppresses SREBP2 &lt;&lt;REF:Zeng-2004-23-950&gt;&gt;. We found that Atf6 causes upregulation of Srebp2 target genes, and while this modest increase in Srebp2 target genes in atf6 morphants did not appear to have a functional impact on cholesterol biogenesis, since atf6 morphants do not develop steatosis but are protected from it, it does suggest that Atf6 may suppress Srebp2 activity.\nWe used mRNA-Seq analyses to identify potential pathways that are dysregulated in response to Atf6 overexpression, although it is clear that the complex post-translational regulation of these pathways are not captured by gene expression studies. Our data suggested that lipoprotein export may be impaired by Atf6 overexpression, but we did not find that the flux of 14C-acetate into the extracellular lipids was impaired in cells overexpressing nATF6. While there are some technical caveats to these biochemical studies, we speculate that decreased lipoprotein export is not the only mechanism by which nAtf6 overexpression causes steatosis. Future studies to optimize cell conditions to ensure maximal cell viability and sustained high levels of nAtf6 expression will be required to fully address this.\nWhile steatosis is clearly a first step on the continuum to more severe liver disease, we also propose a different possibility: that lipid accumulation is a sign of an active and adaptive stress response. Protein folding in the ER is metabolically demanding, and when secretory cargo increases or the ER becomes full of unfolded proteins, increased lipid accumulation may enhance the ATP supply and thereby sustain the increased protein folding demands. Thus, the UPR may serve to enhance the protein folding capacity of the ER function in two ways: by activating genes required for protein folding, quality and export, and by promoting metabolic flux to TAGs as an energy source to withstand the challenge presented by a high level of secretory cargo. In this scenario, acute steatosis would not be a pathological response but, instead, would provide a protective role to sustain hepatocyte function during stress. However, if the stress response persists and steatosis becomes chronic, lipotoxicity could potentiate liver injury. Based on these data, it is tempting to speculate that enhancing protein folding in the ER by chemical chaperones, which have been used in both mouse models and humans to alleviate UPR activity, attenuate fatty liver disease and increase insulin sensitivity &lt;&lt;REF:Ozcan-2006-313-1137&gt;&gt;, &lt;&lt;REF:Kars-2010-59-1899&gt;&gt;, &lt;&lt;REF:Xiao-2011-60-918&gt;&gt;, could reducr the demand for energy and thus reduce the need to accumulate lipid. Whether Atf6 functions both as sensor of unfolded proteins and of metabolic demand remains to be elucidated.\n\nMaterials and Methods\n\nAnimal husbandry, treatments and transgenics\nAdult wildtype (WT, TAB14 and AB), Tg(fabp10:dsRed) &lt;&lt;REF:Dong-2007-39-397&gt;&gt; and Tg(fabp10:nls-mCherry)\n&lt;&lt;REF:???-?-?-?&gt;&gt; zebrafish were maintained according to standard conditions. Larvae were exposed to 350 mM ethanol (Pharmco-AAPER, Brookfield, CT) in fish water starting at 96-98 hours post fertilization (hpf) for up to 32 hours as described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;, &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;. All zebrafish protocols were approved by Mount Sinai's Institutional Animal Care and Use Committee.\nMorpholinos targeting the translation initiation ATG of atf6 (atf6MO-ATG) &lt;&lt;REF:Cinaroglu-2011-54-495&gt;&gt;, an atf6 intron-exon boundary (atf6MO-SPL, Figure S1), a scap intron-exon boundary &lt;&lt;REF:Passeri-2009-49-443&gt;&gt;, and a fasn intron-exon boundary were ordered from GeneTools (Philomath, OR). Approximately 1-5 pmol were injected into 1-2 cell stage embryos. Morpholino sequences and amount injected per embryo (ng) are listed in Table S1.\nThe Tg(fabp10:nAtf6-cherry; cmlc2:GFP) transgenic line was created by injecting a vector containing 2813 bp of the fabp10 promoter &lt;&lt;REF:Dong-2007-39-397&gt;&gt; upstream of the predicted nuclear fragment of zebrafish Atf6 (amino acids 1-366) as identified via DNA and protein alignments with human ATF6 (NCBI Reference Sequence: NP_031374.2). A cassette driving GFP expression in cardiomyocytes was included for rapid screening of transgenics (cmlc2:GFP). The transgene was flanked by Tol2 sites and the vector was injected into fertilized eggs along with transposase mRNA. Larvae were selected for cmlc2:GFP expression and raised to adulthood, outcrossed to TAB14 adults and 4 germline founders were identified.\n\nPlasmids\nThe nATF6-pcDNA-DEST47 plasmid was created using the Invitrogen Gateway System in which human nATF6 (amino acids 1-380) was amplified from a construct containing the full ORF (pEGFP-hATF6, from Dr. Aguirre-Ghiso) and ligated into pcDNA-DEST47. The nAtf6-GFP/pCI-Neo plasmid was generated by amplifying zebrafish nAtf6-GFP from a Tol2-generated plasmid (fabp10:nAtf6-GFP) with primers containing restriction sites for EcoRI and SmaI. The resulting PCR product was ligated into pCI-Neo via T4 ligase. pEGFP-C1 (Clontech) was used as a positive control for transfection.\n\nCell culture and transfection\n293T and HepG2 cells were grown in 100 mm dishes (BD Biosciences) with DMEM (Corning Cellgro, Manassas, VA) containing 10% fetal bovine serum (Invitrogen) and penicillin/streptomycin (Cellgro), and housed in a humidified incubator at 37 C with 5% CO2. Cells were passaged at 80-90% confluency either into 100 mm dishes or 6 well culture plates (Corning). Each well was transfected with 1-2 microg plasmid using Xtreme GENE 9 (Roche) or Lipofectamine &lt;&lt;REF:???-?-?-?&gt;&gt;  (Invitrogen) for 24 hours. Following transfection, cells were collected for RNA extraction using TRIzol (Invitrogen), used for oil red O staining, or labeled with 14C-acetate as described below.\n\n\n14C-acetate radiolabeling\nHepG2 cells were washed with PBS 24-28 hours after transfection and kept in serum free high glucose DMEM (Corning Cellgro, Manassas, VA) containing penicillin/streptomycin (Cellgro), 100 nM insulin (Lilly USA, Indianapolis, IN) and 0.5 microCi 14C-acetate (PerkinElmer, Waltham, MA) for 17 hours. Media was harvested and spun to remove dead cells, while cells were scraped, washed twice and lysed in cold PBS. Protein concentration in the cell lysate was measured with a BCA protein quantification kit (Thermo Scientific, Waltham, MA). Fractions of the media and cell lysate portion were used for lipid extraction as described below.\nWT and nATF6 5 dpf larvae were labeled with 1-5 microCi 14C-acetate for 48 hours. Larvae were washed twice and then sonicated in cold PBS. Protein concentration in the larval lysate was measured with a BCA protein quantification kit (Thermo Scientific). Fractions of the larval lysate were used for lipid extraction as described below.\n\nLipid extractions\nLipids from media, cells and 5 dpf larvae were extracted according to Bligh and Dyer &lt;&lt;REF:Bligh-1959-37-911&gt;&gt; with the following modifications. Briefly, methanol and chloroform were added to 400 microl media or 50 microl of 100 microl total cell lysate in a ratio of 2:1. Samples were vortexed and chloroform and 0.45% NaCl were added at a ratio of 1:1, vortexed again and phases were separated by centrifugation. The aqueous phase was re-extracted with one part chloroform and combined with the lipid from the first second extraction, washed with methanol and 0.45% NaCl at a 1:1 ratio, centrifuged and the lipid phase was recovered and dried under a stream of N2 gas. Lipids were reconstituted in a 2:1 mixture of chloroform and methanol (Thermo Scientific, Waltham, MA), dissolved in Ultima Gold and counted using a 2460 MicroBeta2 LumiJET (PerkinElmer, Waltham, MA) liquid scintillation counter. CCPM were normalized to protein concentration to account for variations in cell number.\n\nQuantitative real-time PCR (qPCR)\nTotal RNA isolated using TRIzol (Invitrogen) from pools of 15-20 dissected livers was reverse-transcribed using qScript cDNA SuperMix (Quanta Biosciences, Gaithersburg, MD). qPCR using A Light Cycler 480 (Roche) with PerfeCTa SYBRGreen FastMix (Quanta Biosciences) was used as previously described &lt;&lt;REF:Howarth-2012-36-14&gt;&gt;. Values for target gene were normalized to reference gene rpp0, and dCts calculated using the comparative threshold method (dCt  = 2-(Ct, gene - Ct, rpp0)). Primer sequences are listed in Table S1.\n\nTriglyceride measurements\nApproximately 20 livers from control and nAtf6 transgenic larvae were dissected and lysed in 0.5% Triton-X 100 and heated at 65degreesC for 5 minutes to inactivate hepatic lipases. Triglycerides were measured using the Infinity Triglyceride Liquid Stable Reagent (Thermo Fisher Scientific, Waltham, MA) following manufacturer's instructions, and were normalized to total protein concentration as determined by Bradford Assay (Bio-Rad, Hercules, CA).\n\nOil red O staining and quantification\nLarvae were fixed in 4% paraformaldehyde (PFA; Electron Microscopy Sciences, Hatfield, PA) overnight at 4degreesC, stained with oil red O, and scored for steatosis as previously described &lt;&lt;REF:Howarth-2013-10-199&gt;&gt;. Cryosections were stained by immersing slides in increasing concentrations of propylene glycol (85%, 100%) for 10 minutes followed by an overnight incubation in oil red O (0.5% in propylene glycol, Polysciences, Warrington, PA). Excess oil red O was removed the next day by sequential washes in 100% and 85% propylene glycol for 5 minutes. Nuclei were counterstained with hematoxylin.\nHepG2 and 293T cells were stained as previously described &lt;&lt;REF:Wu-2010-402-116&gt;&gt; and counterstained with hematoxylin. Oil red O droplet area and number were calculated using ImageJ. In brief, images were split into the red, green, and blue channels and the green channel was further processed for quantification as described, as oil red O has an excitation at 510 nm &lt;&lt;REF:Narasimhan-2011-7-e1001377&gt;&gt;. Following this, the background was subtracted from each image to ensure only counting of oil red O droplets. Droplets equal or greater than five square pixels were counted and area quantified. The values were copied into Microsoft Excel and the average droplet area and number of droplets per nucleus were counted for each field. Twenty independent fields from nATF6 and GFP-transfected HepG2 and 293T cells were imaged at 60x magnification.\n\nRNA sequencing (mRNA-Seq)\nTotal RNA was isolated using TRIzol (Invitrogen) from pools of ~40-50 livers dissected on 5 dpf. Two clutches of Tg(fabp10:nls-mCherry) larvae that were either untreated (control) or treated with 350 mM ethanol for 24 hours and one clutch of nAtf6 TG larvae were used. polyA-tailed mRNA was selected using oligo-dT beads and then fragmented. cDNAs were created using random-hexamers and ligated with bar-coded adaptors compatible with HiSeq &lt;&lt;REF:???-?-?-?&gt;&gt; . Single-end, 100 bp reads were sequenced at the Genomics Core of the Icahn School of Medicine at Mount Sinai. Custom-built software was used to map the reads to the zebrafish genome (Zv9/DanRer7) and estimate the coverage of each gene. Briefly, the reads were split into three 32 bp parts after trimming 2 bp at each end and mapped to the genome using a suffix-array based approach. The median of coverage across the transcript was used as an estimate of gene expression. The expression values were quantile normalized and log-ratios were calculated by comparing nAtf6 TGs to the average of the controls. Each ethanol treated sample was compared to its paired control (untreated siblings). Unique Gene Ontology terms (GO terms) were assigned to each gene by ranking the GO terms by relevance to the biology of the response, and using annotations from the human orthologues if the zebrafish annotations were lacking.\nThe distribution of expression values were plotted to identify the peak in the distribution, which is the level of noise in the system. The values were regularized by adding the noise to each gene's expression level before the log-ratios were calculated. This ensures that genes with low expression do not contribute to list of genes with large fold-changes. An absolute natural log ratio of 0.2 was used as the cutoff (known, non-responding genes are all below this threshold). The list of genes that show changes above this cutoff were analyzed for pathway enrichment using GO terms annotated as described above.\nThis data is available via the Gene Expression Omnibus (GEO) at accession number GSE56498. The controls for this dataset, untreated 5 dpf Tg(fabp10:nls-mCherry), also serve as controls in GEO dataset GSE52605 &lt;&lt;REF:???-?-?-?&gt;&gt; and were included in both sets for ease of comparison between genotypes and treatments.\n\nImage processing and graphics\nImages were cropped and minimally processed using Adobe Photoshop CS4 (Adobe Systems, San Jose, CA). Graphs were plotted using Prism 5.0c (GraphPad Software Inc., La Jolla, CA). Heat maps for mRNA-Seq data were generated using GENE-E (Broad Institute, Cambridge, MA). The global maximum and minimum for each gene set was set to orange and green, respectively, and zero was set to white. Metabolic pathway schematics were adapted from WikiPathways (URL: www.wikipathways.org).\n\nStatistical analyses\nStatistical tests were performed using GraphPad QuickCalcs (GraphPad Software, URL: http://www.graphpad.com/quickcalcs/index.cfm). For oil red O staining of whole larvae, chi-square with Fisher's Exact test were performed. For qPCR, radiolabeling, and oil red O staining of HepG2 and 293T cells we performed unpaired and one-sample t-tests as appropriate.\n\nSupplementary methods\nMethods for histological analysis, cryosectioning and staining, and liver circularity analysis are listed in Text S1.\n\nSupporting Information\nFigure S1\nInjection of an atf6 splice-blocking morpholino induces mis-splicing of atf6 mRNA. A: Schematic of boundary between intron 1 and exon 1 of zebrafish atf6. Exon sequence is shown in black and intron sequence is shown in blue, with the morpholino shown in red. The start codon is highlighted in yellow. B: Gel electrophoresis of atf6 morphants at 2 dpf showing additional bands compared to uninjected controls. C: Sequencing results from atf6 morphants displaying inclusion of intronic sequence (highlighted in blue). The predicted protein translation is shown, and early stop codons are noted as red asterisks.\n(TIF)\nClick here for additional data file.\nFigure S2\nKnockdown of atf6 does not confer resistance to morphological defects from alcohol exposure. A: Oil red O staining of cryosections from uninjected and atf6 morphant larvae treated with 0 or 350 mM ethanol for 32 hours. B: Live images of uninjected and atf6 morphant Tg(fabp10:dsRed) larvae treated with 0 or 350 mM ethanol for 32 hours. C: Quantification of liver circularity, a measure of hepatomegaly, following ethanol treatment in uninjected and atf6 morphant larvae. Livers were traced and circularity quantified in ImageJ.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nTg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae express the nAtf6-cherry transgene despite a lack of visual fluorescence. A: Schematic of human ATF6 transactivation (TAD), DNA-binding (bZIP) and transmembrane (TM) domains, cleavage sites for S1P (MBTPS1) and S2P (MBTPS2), and protein alignment with zebrafish Atf6. Numbers in the diagram correspond to amino acid positions in human ATF6. Numbers at the cleavage sites in the protein alignment correspond to amino acid positions in zebrafish Atf6. B: Confirmation of nuclear localization of zebrafish nAtf6 by transfection of nAtf6-GFP/pCI-Neo into HepG2 cells. C: Live image of Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larva at 5 dpf. D: Agarose gel electrophoresis showing expression of Cherry mRNA in nAtf6 TG but not WT larvae. rpp0 was used to ensure PCR efficacy. E: Western blot visualization of Cherry protein in adult nAtf6 transgenic zebrafish liver tissue. Tg(fabp10:nls-cherry) adult liver protein was used as a positive control. Actin was used as a loading control.\n(TIF)\nClick here for additional data file.\nFigure S4\nOverexpression of nAtf6 induces robust transcription of UPR target genes but does not activate Ire1a or Perk. A: Agarose gel electrophoresis visualization of xbp1 splicing in WT and nAtf6 TG larvae in the presence and absence of tunicamycin (Tm). The percentage of xbp1 splicing was quantified by measuring band intensity in ImageJ. B: Western blot visualization of Eif2a phosphorylation in livers of 5 dpf WT and nAtf6 TG larvae. Tunicamycin (1 microg/mL, 3-5 dpf) treated WT larvae were used as a positive control for immunoblotting. Fold was calculated over 2-3 separate experiments as noted. C: Western blot visualization of Bip protein in livers of 5 dpf WT and nAtf6 TG larvae. Fold change (below blot) was calculated from 3 separate experiments. D and E: Quantitative real-time PCR of multiple UPR effectors in the livers of WT and nAtf6 TG larvae at 5 (D) and 14 (E) dpf. Statistics: unpaired t-test. *, p&lt;0.05.\n(TIF)\nClick here for additional data file.\nFigure S5\nLipid accumulation is evident in nAtf6 TG larvae in the absence of other hepatic injury. A: Staining of cryosections with oil red O showing significant steatosis in nAtf6 TG larvae at 5 and 14 dpf. B: Hematoxylin and eosin staining of nAtf6 TG and control larvae at 5 dpf. No other alterations in hepatic pathology were observed. The number of larvae used is indicated.\n(TIF)\nClick here for additional data file.\nFigure S6\nmRNA-Seq reveals potential disruptions in lipid transport by nAtf6 overexpression. Heatmap of genes associated with lipid binding and transport (A) and basic schematic of lipid synthesis and lipoprotein circulation (B, adapted from WikiPathways). Upregulated genes are colored in shades of orange, and downregulated genes are colored in shades of green. Genes that are unchanged are colored white. Genes that did not appear in mRNA-Seq analyses are colored gray in the schematic.\n(TIF)\nClick here for additional data file.\nFigure S7\nnAtf6 overexpression induces transcription of genes involved in glyceroneogenesis. Quantitative, real-time PCR analysis of gene expression from WT and nAtf6 TG larvae at 5 dpf (A) and 14 dpf (B). Statistics: unpaired t-test. *, p&lt;0.05. For (A), \"n\" corresponds to the number of clutches used and approximately 8-12 livers were dissected for RNA extraction. For (B), individual livers were dissected for analysis.\n(TIF)\nClick here for additional data file.\nFigure S8\nOverexpression of nATF6 in 293T cells drives lipid accumulation. A: Staining of GFP and nATF6 transfected cells with oil red O. B: Quantification of lipid droplet number and area (square pixels). Droplet number and area were calculated using ImageJ. Statistics: unpaired t-test. *, p&lt;0.05. The number of cells used for quantification is noted.\n(TIF)\nClick here for additional data file.\nFigure S9\nSrebp1 and Srebp2 target genes are not dysregulated by nAtf6 overexpression. A and B: qPCR analysis of Srebp1 (A) and Srebp2 (B) target genes in atf6 morphants treated with 350 mM EtOH for 32 hours. Fold changes were calculated based on median dCt values. C and D: qPCR analysis of Srebp1 (C) and Srebp2 (D) target genes in Tg(fabp10:nAtf6-cherry; cmlc2:GFP) larvae at 5 dpf. Fold changes were calculated based on median dCt values, and \"n\" corresponds to the number of clutches used.\n(TIF)\nClick here for additional data file.\nFigure S10\nAtf6 and Srebps do not epistically interact to prevent alcoholic steatosis. Steatosis incidence in larvae injected with atf6, scap, or atf6 + scap MO. \"Total n\" corresponds to the number of larvae scored. \"Clutch n\" corresponds to the number of clutches analyzed. Statistics: chi-square with Fisher's Exact Test. *, p&lt;0.05 and corresponds to groups significantly different from uninjected controls. b, p&lt;0.05 and corresponds to ethanol-treated groups significantly different from uninjected, ethanol treated larvae.\n(TIF)\nClick here for additional data file.\nTable S1\nMorpholino and primer sequences. A: Morpholino sequences. Amount of morpholino injected is based on the assumption that 4 nl of MO is injected per embryo. B: Primer sequences used for conventional and quantitative, real-time PCR.\n(DOCX)\nClick here for additional data file.\nTable S2\nUPR effectors upregulated by nAtf6 and by ethanol. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Log values &gt;= 0.2 are highlighted in red.\n(XLSX)\nClick here for additional data file.\nTable S3\nUpregulated and downregulated genes in the liver of nAtf6 TG 5 dpf larvae. A: Upregulated genes in the liver of nAtf6 TG 5 dpf zebrafish larvae. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. UPR target genes are highlighted in pink. GO terms were assigned as described in the Materials and Methods. B: Downregulated genes in the liver of nAtf6 TG 5 dpf larvae. Genes regulating lipoprotein secretion are highlighted in orange.\n(XLSX)\nClick here for additional data file.\nTable S4\nRaw data from mRNA-Seq. Livers from 2 clutches of Tg(fabp10:nls-mCherry) expressing larvae or from 1 clutch of nAtf6 TG larvae were dissected and pooled for RNA extraction and library preparation. Genes are organized from highest to lowest log value for nAtf6 TG larvae.\n(XLS)\nClick here for additional data file.\nText S1\nSupplementary methods.\n(DOC)\nClick here for additional data file.'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [5.52s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"liver\",\n  \"tissue_name_structured\": \"liver\",\n  \"developmental_stage_text\": \"larvae\",\n  \"developmental_stage_coarse\": [\"larval\"],\n  \"developmental_stage_fine\": [\"larval\"],\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"liver\",\n  \"tissue_name_structured\": \"liver\",\n  \"developmental_stage_text\": \"larvae\",\n  \"developmental_stage_coarse\": [\"larval\"],\n  \"developmental_stage_fine\": [\"larval\"],\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 84,
      "prompt_tokens": 15800,
      "total_tokens": 15884
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_d986a8d1ba"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "liver",
  "tissue_name_structured": "liver",
  "developmental_stage_text": "larvae",
  "developmental_stage_coarse": [
    "larval"
  ],
  "developmental_stage_fine": [
    "larval"
  ],
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [5.53s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "liver",
  "tissue_name_structured": "liver",
  "developmental_stage_text": "larvae",
  "developmental_stage_coarse": [
    "larval"
  ],
  "developmental_stage_fine": [
    "larval"
  ],
  "is_scRNA_data": false
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Expanding the annotation of zebrafish microRNAs based on\nsmallRNA sequencing\nThomas Desvignes a,* , Michael J. Beam a , Peter Batzel a , Jason Sydes a,b , and John H.\nPostlethwait a\na Institute of Neuroscience, University of Oregon, Eugene, OR 97403, USA\nb Institute of Ecology and Evolution, University of Oregon, Eugene, OR 97403, USA\n\n\nAbstract\nMicroRNAs (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes. Because of their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation and are increasingly relevant to human health and disease.\nAlthough the zebrafish Danio rerio is a major model for studies of development, genetics,\nphysiology, evolution, and human biology, the annotation of zebrafish miR-producing genes remains limited. In the present work, we report deep sequencing data of zebrafish smallRNAs from brain, heart, testis, and ovary. Results provide evidence for the expression of 56 un-annotated mir genes and 248 un-annotated mature strands, increasing the number of zebrafish mir genes over those already deposited in miRBase by 16% and the number of mature sequences by 63%. We also describe the existence of three pairs of mirror-mir genes and two mirtron genes, genetic features previously undescribed in non-mammalian vertebrates. This report provides information that substantially increases our knowledge of the zebrafish miRNome and will benefit the entire miR community.\n\n\nKeywords\nmiRNA; zebrafish; teleost; smallRNA-seq; mirror-mir; mirtron\n\n\nIntroduction\nMicroRNA (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes (Ambros, 2004; Kosik, 2010). MicroRNAs are transcribed from mir genes and primary miR transcripts are processed to approximately 22 nucleotide single strand mature forms that function as repressors of\n* Corresponding author Work Phone Number: (541)346-4495.\nGene. Author manuscript; available in PMC 2015 August 10.\nPublished in final edited form as:\nGene. 2014 August 10; 546(2): 386–389. doi:10.1016/j.gene.2014.05.036.\n\n\nNIH-PA Author Manuscript\ntranscript translation when bound to the 3’UTR of protein-coding transcripts in association with the RISC (Davis and Hata, 2009; Kim et al., 2009). Due to their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation (Ivey and Srivastava, 2010). the zebrafish Danio rerio is a major model for studies of development, genetics, physiology, evolution, and human health and disease, and many miR-related discoveries have come from zebrafish investigations (Giraldez et al., 2005; Wienholds et al., 2005; Giraldez et al., 2006; Flynt et al., 2007; He et al., 2009; Mishima et al., 2009; He et al., 2011a, 2011b). For instance, mir140 was shown to regulate zebrafish palatal development by down-regulating the translation of Pdgfra protein in neural crest derived cells (Eberhart et al., 2008).\nExperiments stimulated by that discovery showed that a single nucleotide polymorphism in the human MIR140 gene leads to improper mature MIR140 production and contributes to nonsyndromic cleft palate susceptibility in patients (Li et al., 2010). Another example of successful transfer of discoveries from zebrafish to human is the discovery of the crucial role of mir451 in zebrafish erythropoiesis (Pase et al., 2009), which was later confirmed in mouse (Rasmussen et al., 2010; Patrick et al., 2010) and recently in human (Kim et al.,\n2013), thereby highlighting the therapeutic potential of miR inhibitors. Despite the high impact of zebrafish research in many aspects of development and physiology, the annotation of zebrafish miR-producing genes remains limited.\nThis limitation in our knowledge is a broad problem because the annotation of the zebrafish genome, the most complete among fish (Howe et al., 2013), often drives annotation in all teleost fish, which constitute half of all vertebrate species (Nelson, 2006). Moreover, a largely incomplete characterization of the miR repertoire compromises studies on miRNA evolution, their emergence and retention in vertebrates, especially in the rayfin fish lineage following the teleost genome duplication. Furthermore, uncovering and transferring evolutionarily conserved miR functions from fish models to human health depends on recognizing true fish orthologs of human miRs. Thus, the entire miR community would benefit substantially by a better understanding of the zebrafish miRNome.\nIn the present report, we provide extensive miRNA sequencing data from several zebrafish tissues demonstrating the expression of new mir genes and the expression of previously un- annotated mature strands. Further, we describe for the first time in non-mammalian vertebrates, the expression of mirror-mirs (Scott et al., 2012; Tyler et al., 2008) and mirtrons (Chung et al., 2011; Ladewig et al., 2012; Okamura et al., 2007).\n\n\nMaterials and methods\nSmall RNAs were extracted using the Norgen microRNA purification kit from four different organs (brain, heart, testis and ovary). Animals were actively reproductive adult “AB” strain zebrafish (Danio rerio) obtained from the University of Oregon fish facility. From two males, we sampled brain, heart, and testes and made six libraries, three for each individual; from two females, we sampled ovaries and made two libraries, one for each individual. Eight tissue-specific sequencing libraries were prepared and barcoded using the\nBiooScientific NEXTflex smallRNA Sequencing Kit which uses a 3’ adenylated adapter that ligates onto miRs and other small RNAs with a 3’ hydroxyl group, following the\n\n\nNIH-PA Author Manuscript\nmanufacturer’s instructions. Libraries were subsequently sequenced by Illumina HiSeq2500 at the University of Oregon Genomics Core Facility and raw single-end 50nt long reads were deposited in the NCBI Short Reads Archive under the accession number SRP039502. All animal work was performed according to the University of Oregon IACUC approved protocol (#09–1BRRA).\nBioinformatic processing involved removing 3’ adapter sequences and filtering reads based on quality; reads with any base Q&lt;30 were removed using the FASTX-Toolkit ( http:// hannonlab.cshl.edu/fastx_toolkit/ ). A custom pipeline was created to remove reads outside of the targeted size range (&lt;15 and &gt;28), count the number of occurrences of each read, and remove reads with low counts (&lt;30 summed across all eight libraries). The script then aligns reads to the Zv9 version of the zebrafish reference genome using blastn (Altschul et al.,\n1990) allowing up to five mismatches, one gap, and alignments at up to 20 locations in the genome; finally, the script groups reads together based on their genomic locations. Aligned groups were annotated against mature and hairpin sequences already annotated in miRBase (release 20)(Kozomara and Griffiths-Jones, 2013) as well as zebrafish noncoding RNA sequences annotated by Ensembl (release 74)(Flicek et al., 2012).\nPutative new mirs were predicted based on read presence in our experimental dataset and secondary structures were computed on the RNAfold web server (Zuker and Stiegler, 1981) using default parameters except for changing the calculation of minimum free energy from 37°C to 28°C, the temperature at which zebrafish are reared (Westerfield, 2000). A tentative name was assigned to newly recognized sequences based on identity with deposited sequences in miRBase (Kozomara and Griffiths-Jones, 2013) for zebrafish or other species.\n\n\nResults/discussion\nThe sequencing of small RNAs with 3’ hydroxyls from several tissues of adult zebrafish yielded a total of 59.5 million reads that passed all filtering criteria, among which 47.5 million (79.8%) could be directly annotated from mature zebrafish entries in miRBase due to sequence identity. Among remaining reads, sequences mapping perfectly to the zebrafish reference genome at fewer than 20 locations were further studied for putative new mir genes and/or for mature strand annotation.\nThe annotation pipeline identified 56 mir genes that do not appear in miRBase, although 43 have of them have been bioinformatically predicted as “Novel miRNA” genes in Ensembl (Flicek et al., 2012); thus, our work increases the number of annotated zebrafish mir genes by 16% compared to the most recent miRBase (Release20) (Kozomara and Griffiths-Jones, 2013) (Table 1). Most of these new mir genes (54/56) are clear homologs to genes belonging to known mir families or described in related species and could thus be easily added to their corresponding group.\nTwo of the miR-encoding genes we identified in our experimental data, tentatively named dre-mir-733-2 and dre-mir-733b, are paralogs of dre-mir-733 (MI0004778) and originate from vaultRNAs, and a third miR, tentatively named dre-mir-735b, a paralog of dre-mir-735 (MI0004781), actually originates from a Y_RNA gene. Human orthologs (hsa-mir-886 and\n\n\nNIH-PA Author Manuscript\nhsa-mir-1979 respectively) of these zebrafish genes were withdrawn from miRBase after their biogenesis was understood (Stadler et al., 2009; Meiri et al., 2010). The role of vaultRNA and Y_RNA short fragments in translation repression is, however, not fully understood and they may act in a “mir-like” manner (Meiri et al., 2010; Persson et al., 2009; Nicolas et al., 2012), as do some snoRNA-derived or lncRNA-derived miRs (Yang and Lai, 2011). If the vaultRNA and Y_RNA derived short sequences do act like miRs, then production of short RNA fragments from vaultRNA and Y_RNA would be a new non- canonical way of generating miRs (Verhagen and Pruijn, 2011).\nOur zebrafish smallRNA-seq dataset contained two completely new mir genes that showed no similarity with sequences yet deposited in miRBase for any species. These two new mirs, along with the 54 mir genes in our experimental dataset that are predicted in Ensembl but not referenced in miRBase, are described in the Supplemental File, which gives a tentative name, genomic localizations, predicted hairpin sequences, corresponding secondary structures, minimum free energy calculated at 28°C, the localization of mature sequences on the hairpin sequence, and secondary structure.\nThe breadth and depth of our sequencing dataset also allowed us to provide annotation for a total of 248 mature strands, 158 of them corresponding to previously annotated mir genes and 90 from the set of 56 new mir genes not present yet in miRBase and described here above, increasing the total number of annotated zebrafish mature sequences from 391 to 639 (a 63% increase) (Table 1). The Supplemental File gives the sequence, and genomic location of each newly identified miR.\nAmong the newly annotated structures are miRs with several interesting features. The zebrafish smallRNA-seq dataset clearly contained three pairs of mirror-mirs (Table 1), which are defined as a pair of mir genes originating from overlapping genomic regions but on opposite strands (Tyler et al., 2008; Scott et al., 2012). For example, the expression of mir3120, mirror-mir of mir214, has been described in mammals but in no other species (Scott et al., 2012; Desvignes et al., 2014). Here we show that miR3120, a mature fragment originating from the mirror-mir of mir214, is expressed in adult zebrafish. We also found the pair mir7547 and mir7553, which had been previously identified as independent mirs in the channel catfish Ictalurus punctatus (MI0024671 and MI0024685 respectively)(Xu et al.,\n2013), but had not been described as a mirror-mir pair, probably due to the lack of a reference genome. The newly annotated zebrafish mir7552b and its mirror gene, mir7552bOS, had also not previously been annotated in zebrafish, even though a homolog of mir7552b is annotated in the channel catfish (MI0024684)(Xu et al., 2013). This finding is of particular interest given that, to our knowledge, mirror-mirs have been described so far only in Drosophila and mammals (Tyler et al., 2008; Scott et al., 2012). Evidence given in this report demonstrating the expression of mirror-mirs in zebrafish suggests that mirror- mirs are likely a type of gene featured in all vertebrates.\nFinally, we also report the presence and expression of a pair of 3’-tailed mirtron paralogs located in gria3 ohnologs (gria3a, ENSDARG00000032737; and gria3b,\nENSDARG00000037498) (Table 1). These mir genes are co-orthologous mirtrons to the predicted human mirtron gene AL356213.1 (ENSG00000265082), which is located in\n\n\nNIH-PA Author Manuscript\nGRIA3 (ENSG00000125675); they lie at orthologous positions in both zebrafish ohnologs; and they show high similarities with tetrapod mir2985, which is deposited in miRBase in zebra finch Taeniopygia guttata, platypus Ornithorhynchus anatinus and rat Rattus norvegicus as intronic MiRs of the Gria2 genes (Kozomara and Griffiths-Jones, 2013).\nSimilar to mirror-mirs, to our knowledge, mirtrons have only been shown in Drosophila, the nematode Caenorhabditis elegans and mammals (Okamura et al., 2007; Chung et al., 2011; Ladewig et al., 2012). This report is thus the first example of mirtrons in a non-mammalian vertebrate.\n\n\nConclusion\nThe present report analyzing smallRNA deep sequencing data expands the annotation of microRNA genes and mature strands in the laboratory model zebrafish Danio rerio, whose mir gene repertoire remains understudied and poorly characterized in comparison to other species, such as Drosophila, worms, mouse, or human, despite its relevance for microRNA research. Increasing the annottion of zebrafish miRs will contribute to a better understanding of the evolution of mir genes in vertebrates and within the rayfin fish lineage, will help annotate mir genes in other teleost species, and will increase confidence in the annotation of each gene thanks to the description of both mature strands, considered as a criterion for mir gene annotation confidence (Kozomara and Griffiths-Jones, 2013).\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nAuthors would like to thank the University of Oregon fish facility crew, especially Trevor Enright, for help in fish husbandry, and members of the Postlethwait Lab and of the Institute of Neuroscience of the University of Oregon for helpful discussions. This work was funded by NIH grants U01 DE020076, R01 OD011116 awarded to JHP and 5R240D011199 to W. Cresko and JHP.\n\n\nReferences\nAltschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J. Mol. Biol. 1990; 215:403–410. [PubMed: 2231712]\nAmbros V. The functions of animal microRNAs. Nature. 2004; 431:350–355. [PubMed: 15372042] Chung W-J, Agius P, Westholm JO, Chen M, Okamura K, Robine N, Leslie CS, Lai EC.\nComputational and experimental identification of mirtrons in Drosophila melanogaster and Caenorhabditis elegans. Genome Res. 2011; 21:286–300. [PubMed: 21177960]\nDavis BN, Hata A. Regulation of MicroRNA Biogenesis: A miRiad of mechanisms. Cell Commun. Signal. 2009; 7:18. [PubMed: 19664273]\nDesvignes T, Contreras A, Postlethwait JH. Evolution of the miR199–214 cluster and vertebrate skeletal development. RNA Biol. 2014; 11:0–1.\nEberhart JK, He X, Swartz ME, Yan Y-L, Song H, Boling TC, Kunerth AK, Walker MB, Kimmel CB, Postlethwait JH. MicroRNA Mirn140 modulates Pdgf signaling during palatogenesis. Nat. Genet. 2008; 40:290–298. [PubMed: 18264099]\n\n\nNIH-PA Author Manuscript\nM, Sheppard D, Sobral D, Taylor K, Thormann A, Trevanion S, White S, Wilder SP, Aken BL, Birney E, Cunningham F, Dunham I, Harrow J, Herrero J, Hubbard TJP, Johnson N, Kinsella R, Parker A, Spudich G, Yates A, Zadissa A, Searle SMJ. Ensembl 2013. Nucleic Acids Res. 2012; 41:D48–D55. [PubMed: 23203987]\nFlynt AS, Li N, Thatcher EJ, Solnica-Krezel L, Patton JG. Zebrafish miR-214 modulates Hedgehog signaling to specify muscle cell fate. Nat. Genet. 2007; 39:259–263. [PubMed: 17220889]\nGiraldez AJ, Cinalli RM, Glasner ME, Enright AJ, Thomson JM, Baskerville S, Hammond SM, Bartel DP, Schier AF. MicroRNAs Regulate Brain Morphogenesis in Zebrafish. Science. 2005; 308:833– 838. [PubMed: 15774722]\nGiraldez AJ, Mishima Y, Rihel J, Grocock RJ, Dongen SV, Inoue K, Enright AJ, Schier AF. Zebrafish MiR-430 Promotes Deadenylation and Clearance of Maternal mRNAs. Science. 2006; 312:75–79. [PubMed: 16484454]\nHe X, Eberhart JK, Postlethwait JH. MicroRNAs and micromanaging the skeleton in disease, development and evolution. J. Cell. Mol. Med. 2009; 13:606–618. [PubMed: 19220576]\nHe X, Yan Y-L, DeLaurier A, Postlethwait JH. Observation of miRNA gene expression in zebrafish embryos by in situ hybridization to microRNA primary transcripts. Zebrafish. 2011a; 8:1–8. [PubMed: 21288128]\nHe X, Yan Y-L, Eberhart JK, Herpin A, Wagner TU, Schartl M, Postlethwait JH. miR-196 regulates axial patterning and pectoral appendage initiation. Dev. Biol. 2011b; 357:463–477. [PubMed: 21787766]\nHowe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, Collins JE, Humphray S, McLaren K, Matthews L, McLaren S, Sealy I, Caccamo M, Churcher C, Scott C, Barrett JC, Koch R, Rauch G-J, White S, Chow W, Kilian B, Quintais LT, Guerra-Assuncao JA, Zhou Y, Gu Y, Yen J, Vogel J-H, Eyre T, Redmond S, Banerjee R, Chi J, Fu B, Langley E, Maguire SF, Laird GK, Lloyd D, Kenyon E, Donaldson S, Sehra H, Almeida-King J, Loveland J, Trevanion S, Jones M, Quail M, Willey D, Hunt A, Burton J, Sims S, McLay K, Plumb B, Davis J, Clee C, Oliver K, Clark R, Riddle C, Eliott D, Threadgold G, Harden G, Ware D, Mortimer B, Kerry G, Heath P, Phillimore B, Tracey A, Corby N, Dunn M, Johnson C, Wood J, Clark S, Pelan S, Griffiths G, Smith M, Glithero R, Howden P, Barker N, Stevens C, Harley J, Holt K, Panagiotidis G, Lovell J, Beasley H, Henderson C, Gordon D, Auger K, Wright D, Collins J, Raisen C, Dyer L, Leung K, Robertson L, Ambridge K, Leongamornlert D, McGuire S, Gilderthorp R, Griffiths C, Manthravadi D, Nichol S, Barker G, Whitehead S, Kay M, Brown J, Murnane C, Gray E, Humphries M, Sycamore N, Barker D, Saunders D, Wallis J, Babbage A, Hammond S, Mashreghi-Mohammadi M, Barr L, Martin S, Wray P, Ellington A, Matthews N, Ellwood M, Woodmansey R, Clark G, Cooper J, Tromans A, Grafham D, Skuce C, Pandian R, Andrews R, Harrison E, Kimberley A, Garnett J, Fosker N, Hall R, Garner P, Kelly D, Bird C, Palmer S, Gehring I, Berger A, Dooley CM, Ersan-Urun Z, Eser C, Geiger H, Geisler M, Karotki L, Kirn A, Konantz J, Konantz M, Oberlander M, Rudolph-Geiger S, Teucke M, Osoegawa K, Zhu B, Rapp A, Widaa S, Langford C, Yang F, Carter NP, Harrow J, Ning Z, Herrero J, Searle SMJ, Enright A, Geisler R, Plasterk RHA, Lee C, Westerfield M, de Jong PJ, Zon LI, Postlethwait JH, Nusslein- Volhard C, Hubbard TJP, Crollius HR, Rogers J, Stemple DL. The zebrafish reference genome sequence and its relationship to the human genome. Nature. 2013 advance online publication.\nIvey KN, Srivastava D. MicroRNAs as Regulators of Differentiation and Cell Fate Decisions. Cell Stem Cell. 2010; 7:36–41. [PubMed: 20621048]\nKim M, Tan YS, Cheng W-C, Civin CI. MiR-144 and MiR-451 Regulate Human Erythropoiesis By Targeting RAB14. Blood. 2013; 122:942–942.\nKim VN, Han J, Siomi MC. Biogenesis of small RNAs in animals. Nat. Rev. Mol. Cell Biol. 2009; 10:126–139. [PubMed: 19165215]\nKosik KS. MicroRNAs and Cellular Phenotypy. Cell. 2010; 143:21–26. [PubMed: 20887887]\nKozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic Acids Res. 2013; 42:D68–D73. [PubMed: 24275495]\nLadewig E, Okamura K, Flynt AS, Westholm JO, Lai EC. Discovery of hundreds of mirtrons in mouse and human small RNA data. Genome Res. 2012; 22:1634–1645. [PubMed: 22955976]\n\n\nNIH-PA Author Manuscript\nLi L, Meng T, Jia Z, Zhu G, Shi B. Single nucleotide polymorphism associated with nonsyndromic cleft palate influences the processing of miR-140. Am. J. Med. Genet. A. 2010; 152A:856–862. [PubMed: 20358594]\nMeiri E, Levy A, Benjamin H, Ben-David M, Cohen L, Dov A, Dromi N, Elyakim E, Yerushalmi N, Zion O, Lithwick-Yanai G, Sitbon E. Discovery of microRNAs and other small RNAs in solid tumors. Nucleic Acids Res. 2010; 38:6234–6246. [PubMed: 20483914]\nMishima Y, Abreu-Goodger C, Staton AA, Stahlhut C, Shou C, Cheng C, Gerstein M, Enright AJ, Giraldez AJ. Zebrafish miR-1 and miR-133 shape muscle gene expression and regulate sarcomeric actin organization. Genes Dev. 2009; 23:619–632. [PubMed: 19240126]\nNelson, JS. Fishes of the World. John Wiley &amp; Sons; 2006.\nNicolas FE, Hall AE, Csorba T, Turnbull C, Dalmay T. Biogenesis of Y RNA-derived small RNAs is independent of the microRNA pathway. FEBS Lett. 2012; 586:1226–1230. [PubMed: 22575660]\nOkamura K, Hagen JW, Duan H, Tyler DM, Lai EC. The Mirtron Pathway Generates microRNA- Class Regulatory RNAs in Drosophila. Cell. 2007; 130:89–100. [PubMed: 17599402]\nPase L, Layton JE, Kloosterman WP, Carradice D, Waterhouse PM, Lieschke GJ. miR-451 regulates zebrafish erythroid maturation in vivo via its target gata2. Blood. 2009; 113:1794–1804. [PubMed: 18849488]\nPatrick DM, Zhang CC, Tao Y, Yao H, Qi X, Schwartz RJ, Huang LJ-S, Olson EN. Defective erythroid differentiation in miR-451 mutant mice mediated by 14-3-3 ζ . Genes Dev. 2010; 24:1614–1619. [PubMed: 20679397]\nPersson H, Kvist A, Vallon-Christersson J, Medstrand P, Borg Å, Rovira C. The non-coding RNA of the multidrug resistance-linked vault particle encodes multiple regulatory small RNAs. Nat. Cell Biol. 2009; 11:1268–1271. [PubMed: 19749744]\nRasmussen KD, Simmini S, Abreu-Goodger C, Bartonicek N, Giacomo MD, Bilbao-Cortes D, Horos R, Lindern MV, Enright AJ, O’Carroll D. The miR-144/451 locus is required for erythroid homeostasis. J. Exp. Med. 2010; 207:1351–1358. [PubMed: 20513743]\nScott H, Howarth J, Lee YB, Wong L-F, Bantounas I, Phylactou L, Verkade P, Uney JB. MiR-3120 Is a Mirror MicroRNA That Targets Heat Shock Cognate Protein 70 and Auxilin Messenger RNAs and Regulates Clathrin Vesicle Uncoating. J. Biol. Chem. 2012; 287:14726–14733. [PubMed: 22393045]\nStadler PF, Chen JJ-L, Hackermüller J, Hoffmann S, Horn F, Khaitovich P, Kretzschmar AK, Mosig A, Prohaska SJ, Qi X, Schutt K, Ullmann K. Evolution of vault RNAs. Mol. Biol. Evol. 2009; 26:1975–1991. [PubMed: 19491402]\nTyler DM, Okamura K, Chung W-J, Hagen JW, Berezikov E, Hannon GJ, Lai EC. Functionally distinct regulatory RNAs generated by bidirectional transcription and processing of microRNA loci. Genes Dev. 2008; 22:26–36. [PubMed: 18172163]\nVerhagen APM, Pruijn GJM. Are the Ro RNP-associated Y RNAs concealing microRNAs? Y RNA- derived miRNAs may be involved in autoimmunity. BioEssays. 2011; 33:674–682. [PubMed: 21735459]\nWesterfield, M. A guide for the laboratory use of zebrafish (Danio rerio). 4th ed.. Eugene: Univ. of Oregon Press; 2000. The zebrafish book.\nWienholds E, Kloosterman WP, Miska E, Alvarez-Saavedra E, Berezikov E, Bruijn E, de, Horvitz HR, Kauppinen S, Plasterk RHA. MicroRNA Expression in Zebrafish Embryonic Development. Science. 2005; 309:310–311. [PubMed: 15919954]\nXu Z, Chen J, Li X, Ge J, Pan J, Xu X. Identification and Characterization of MicroRNAs in Channel Catfish (Ictalurus punctatus) by Using Solexa Sequencing Technology. PLoS ONE. 2013; 8:e54174. [PubMed: 23342099]\nYang J-S, Lai EC. Alternative miRNA Biogenesis Pathways and the Interpretation of Core miRNA Pathway Mutants. Mol. Cell. 2011; 43:892–903. [PubMed: 21925378]\nZuker M, Stiegler P. Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information. Nucleic Acids Res. 1981; 9:133–148. [PubMed: 6163133]\n\n\nAbbreviations list\nmiR microRNA\nsmallRNA-seq small RNA sequencing\n\n\nHighlights\n\n\nAnnotation of 56 new miRNA genes in zebrafish\nAnnotation of 248 new mature miRNA strands in zebrafish\nFirst description of mirror-miR pairs outside of Drosophila and mammals\nFirst description of 3’-tailed mirtrons outside of Drosophila, worms and mammals\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 2 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 4 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 6\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 7\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nUTR untranslated region RISC RNA-induced silencing complex Desvignes et al. Page 8 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 9\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 10 Table 1\n\n\nZebrafish miRBase Release 20 Present study % increase Total mir genes 346 56 16 402 mature sequences 391 248 63 639 mirror-mir pairs 0 3 N/A 3 mirtrons 0 2 N/A 2\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Expanding the annotation of zebrafish microRNAs based on\nsmallRNA sequencing\nThomas Desvignes a,* , Michael J. Beam a , Peter Batzel a , Jason Sydes a,b , and John H.\nPostlethwait a\na Institute of Neuroscience, University of Oregon, Eugene, OR 97403, USA\nb Institute of Ecology and Evolution, University of Oregon, Eugene, OR 97403, USA\n\n\nAbstract\nMicroRNAs (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes. Because of their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation and are increasingly relevant to human health and disease.\nAlthough the zebrafish Danio rerio is a major model for studies of development, genetics,\nphysiology, evolution, and human biology, the annotation of zebrafish miR-producing genes remains limited. In the present work, we report deep sequencing data of zebrafish smallRNAs from brain, heart, testis, and ovary. Results provide evidence for the expression of 56 un-annotated mir genes and 248 un-annotated mature strands, increasing the number of zebrafish mir genes over those already deposited in miRBase by 16% and the number of mature sequences by 63%. We also describe the existence of three pairs of mirror-mir genes and two mirtron genes, genetic features previously undescribed in non-mammalian vertebrates. This report provides information that substantially increases our knowledge of the zebrafish miRNome and will benefit the entire miR community.\n\n\nKeywords\nmiRNA; zebrafish; teleost; smallRNA-seq; mirror-mir; mirtron\n\n\nIntroduction\nMicroRNA (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes (Ambros, 2004; Kosik, 2010). MicroRNAs are transcribed from mir genes and primary miR transcripts are processed to approximately 22 nucleotide single strand mature forms that function as repressors of\n* Corresponding author Work Phone Number: (541)346-4495.\nGene. Author manuscript; available in PMC 2015 August 10.\nPublished in final edited form as:\nGene. 2014 August 10; 546(2): 386–389. doi:10.1016/j.gene.2014.05.036.\n\n\nNIH-PA Author Manuscript\ntranscript translation when bound to the 3’UTR of protein-coding transcripts in association with the RISC (Davis and Hata, 2009; Kim et al., 2009). Due to their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation (Ivey and Srivastava, 2010). the zebrafish Danio rerio is a major model for studies of development, genetics, physiology, evolution, and human health and disease, and many miR-related discoveries have come from zebrafish investigations (Giraldez et al., 2005; Wienholds et al., 2005; Giraldez et al., 2006; Flynt et al., 2007; He et al., 2009; Mishima et al., 2009; He et al., 2011a, 2011b). For instance, mir140 was shown to regulate zebrafish palatal development by down-regulating the translation of Pdgfra protein in neural crest derived cells (Eberhart et al., 2008).\nExperiments stimulated by that discovery showed that a single nucleotide polymorphism in the human MIR140 gene leads to improper mature MIR140 production and contributes to nonsyndromic cleft palate susceptibility in patients (Li et al., 2010). Another example of successful transfer of discoveries from zebrafish to human is the discovery of the crucial role of mir451 in zebrafish erythropoiesis (Pase et al., 2009), which was later confirmed in mouse (Rasmussen et al., 2010; Patrick et al., 2010) and recently in human (Kim et al.,\n2013), thereby highlighting the therapeutic potential of miR inhibitors. Despite the high impact of zebrafish research in many aspects of development and physiology, the annotation of zebrafish miR-producing genes remains limited.\nThis limitation in our knowledge is a broad problem because the annotation of the zebrafish genome, the most complete among fish (Howe et al., 2013), often drives annotation in all teleost fish, which constitute half of all vertebrate species (Nelson, 2006). Moreover, a largely incomplete characterization of the miR repertoire compromises studies on miRNA evolution, their emergence and retention in vertebrates, especially in the rayfin fish lineage following the teleost genome duplication. Furthermore, uncovering and transferring evolutionarily conserved miR functions from fish models to human health depends on recognizing true fish orthologs of human miRs. Thus, the entire miR community would benefit substantially by a better understanding of the zebrafish miRNome.\nIn the present report, we provide extensive miRNA sequencing data from several zebrafish tissues demonstrating the expression of new mir genes and the expression of previously un- annotated mature strands. Further, we describe for the first time in non-mammalian vertebrates, the expression of mirror-mirs (Scott et al., 2012; Tyler et al., 2008) and mirtrons (Chung et al., 2011; Ladewig et al., 2012; Okamura et al., 2007).\n\n\nMaterials and methods\nSmall RNAs were extracted using the Norgen microRNA purification kit from four different organs (brain, heart, testis and ovary). Animals were actively reproductive adult “AB” strain zebrafish (Danio rerio) obtained from the University of Oregon fish facility. From two males, we sampled brain, heart, and testes and made six libraries, three for each individual; from two females, we sampled ovaries and made two libraries, one for each individual. Eight tissue-specific sequencing libraries were prepared and barcoded using the\nBiooScientific NEXTflex smallRNA Sequencing Kit which uses a 3’ adenylated adapter that ligates onto miRs and other small RNAs with a 3’ hydroxyl group, following the\n\n\nNIH-PA Author Manuscript\nmanufacturer’s instructions. Libraries were subsequently sequenced by Illumina HiSeq2500 at the University of Oregon Genomics Core Facility and raw single-end 50nt long reads were deposited in the NCBI Short Reads Archive under the accession number SRP039502. All animal work was performed according to the University of Oregon IACUC approved protocol (#09–1BRRA).\nBioinformatic processing involved removing 3’ adapter sequences and filtering reads based on quality; reads with any base Q&lt;30 were removed using the FASTX-Toolkit ( http:// hannonlab.cshl.edu/fastx_toolkit/ ). A custom pipeline was created to remove reads outside of the targeted size range (&lt;15 and &gt;28), count the number of occurrences of each read, and remove reads with low counts (&lt;30 summed across all eight libraries). The script then aligns reads to the Zv9 version of the zebrafish reference genome using blastn (Altschul et al.,\n1990) allowing up to five mismatches, one gap, and alignments at up to 20 locations in the genome; finally, the script groups reads together based on their genomic locations. Aligned groups were annotated against mature and hairpin sequences already annotated in miRBase (release 20)(Kozomara and Griffiths-Jones, 2013) as well as zebrafish noncoding RNA sequences annotated by Ensembl (release 74)(Flicek et al., 2012).\nPutative new mirs were predicted based on read presence in our experimental dataset and secondary structures were computed on the RNAfold web server (Zuker and Stiegler, 1981) using default parameters except for changing the calculation of minimum free energy from 37°C to 28°C, the temperature at which zebrafish are reared (Westerfield, 2000). A tentative name was assigned to newly recognized sequences based on identity with deposited sequences in miRBase (Kozomara and Griffiths-Jones, 2013) for zebrafish or other species.\n\n\nResults/discussion\nThe sequencing of small RNAs with 3’ hydroxyls from several tissues of adult zebrafish yielded a total of 59.5 million reads that passed all filtering criteria, among which 47.5 million (79.8%) could be directly annotated from mature zebrafish entries in miRBase due to sequence identity. Among remaining reads, sequences mapping perfectly to the zebrafish reference genome at fewer than 20 locations were further studied for putative new mir genes and/or for mature strand annotation.\nThe annotation pipeline identified 56 mir genes that do not appear in miRBase, although 43 have of them have been bioinformatically predicted as “Novel miRNA” genes in Ensembl (Flicek et al., 2012); thus, our work increases the number of annotated zebrafish mir genes by 16% compared to the most recent miRBase (Release20) (Kozomara and Griffiths-Jones, 2013) (Table 1). Most of these new mir genes (54/56) are clear homologs to genes belonging to known mir families or described in related species and could thus be easily added to their corresponding group.\nTwo of the miR-encoding genes we identified in our experimental data, tentatively named dre-mir-733-2 and dre-mir-733b, are paralogs of dre-mir-733 (MI0004778) and originate from vaultRNAs, and a third miR, tentatively named dre-mir-735b, a paralog of dre-mir-735 (MI0004781), actually originates from a Y_RNA gene. Human orthologs (hsa-mir-886 and\n\n\nNIH-PA Author Manuscript\nhsa-mir-1979 respectively) of these zebrafish genes were withdrawn from miRBase after their biogenesis was understood (Stadler et al., 2009; Meiri et al., 2010). The role of vaultRNA and Y_RNA short fragments in translation repression is, however, not fully understood and they may act in a “mir-like” manner (Meiri et al., 2010; Persson et al., 2009; Nicolas et al., 2012), as do some snoRNA-derived or lncRNA-derived miRs (Yang and Lai, 2011). If the vaultRNA and Y_RNA derived short sequences do act like miRs, then production of short RNA fragments from vaultRNA and Y_RNA would be a new non- canonical way of generating miRs (Verhagen and Pruijn, 2011).\nOur zebrafish smallRNA-seq dataset contained two completely new mir genes that showed no similarity with sequences yet deposited in miRBase for any species. These two new mirs, along with the 54 mir genes in our experimental dataset that are predicted in Ensembl but not referenced in miRBase, are described in the Supplemental File, which gives a tentative name, genomic localizations, predicted hairpin sequences, corresponding secondary structures, minimum free energy calculated at 28°C, the localization of mature sequences on the hairpin sequence, and secondary structure.\nThe breadth and depth of our sequencing dataset also allowed us to provide annotation for a total of 248 mature strands, 158 of them corresponding to previously annotated mir genes and 90 from the set of 56 new mir genes not present yet in miRBase and described here above, increasing the total number of annotated zebrafish mature sequences from 391 to 639 (a 63% increase) (Table 1). The Supplemental File gives the sequence, and genomic location of each newly identified miR.\nAmong the newly annotated structures are miRs with several interesting features. The zebrafish smallRNA-seq dataset clearly contained three pairs of mirror-mirs (Table 1), which are defined as a pair of mir genes originating from overlapping genomic regions but on opposite strands (Tyler et al., 2008; Scott et al., 2012). For example, the expression of mir3120, mirror-mir of mir214, has been described in mammals but in no other species (Scott et al., 2012; Desvignes et al., 2014). Here we show that miR3120, a mature fragment originating from the mirror-mir of mir214, is expressed in adult zebrafish. We also found the pair mir7547 and mir7553, which had been previously identified as independent mirs in the channel catfish Ictalurus punctatus (MI0024671 and MI0024685 respectively)(Xu et al.,\n2013), but had not been described as a mirror-mir pair, probably due to the lack of a reference genome. The newly annotated zebrafish mir7552b and its mirror gene, mir7552bOS, had also not previously been annotated in zebrafish, even though a homolog of mir7552b is annotated in the channel catfish (MI0024684)(Xu et al., 2013). This finding is of particular interest given that, to our knowledge, mirror-mirs have been described so far only in Drosophila and mammals (Tyler et al., 2008; Scott et al., 2012). Evidence given in this report demonstrating the expression of mirror-mirs in zebrafish suggests that mirror- mirs are likely a type of gene featured in all vertebrates.\nFinally, we also report the presence and expression of a pair of 3’-tailed mirtron paralogs located in gria3 ohnologs (gria3a, ENSDARG00000032737; and gria3b,\nENSDARG00000037498) (Table 1). These mir genes are co-orthologous mirtrons to the predicted human mirtron gene AL356213.1 (ENSG00000265082), which is located in\n\n\nNIH-PA Author Manuscript\nGRIA3 (ENSG00000125675); they lie at orthologous positions in both zebrafish ohnologs; and they show high similarities with tetrapod mir2985, which is deposited in miRBase in zebra finch Taeniopygia guttata, platypus Ornithorhynchus anatinus and rat Rattus norvegicus as intronic MiRs of the Gria2 genes (Kozomara and Griffiths-Jones, 2013).\nSimilar to mirror-mirs, to our knowledge, mirtrons have only been shown in Drosophila, the nematode Caenorhabditis elegans and mammals (Okamura et al., 2007; Chung et al., 2011; Ladewig et al., 2012). This report is thus the first example of mirtrons in a non-mammalian vertebrate.\n\n\nConclusion\nThe present report analyzing smallRNA deep sequencing data expands the annotation of microRNA genes and mature strands in the laboratory model zebrafish Danio rerio, whose mir gene repertoire remains understudied and poorly characterized in comparison to other species, such as Drosophila, worms, mouse, or human, despite its relevance for microRNA research. Increasing the annottion of zebrafish miRs will contribute to a better understanding of the evolution of mir genes in vertebrates and within the rayfin fish lineage, will help annotate mir genes in other teleost species, and will increase confidence in the annotation of each gene thanks to the description of both mature strands, considered as a criterion for mir gene annotation confidence (Kozomara and Griffiths-Jones, 2013).\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nAuthors would like to thank the University of Oregon fish facility crew, especially Trevor Enright, for help in fish husbandry, and members of the Postlethwait Lab and of the Institute of Neuroscience of the University of Oregon for helpful discussions. This work was funded by NIH grants U01 DE020076, R01 OD011116 awarded to JHP and 5R240D011199 to W. Cresko and JHP.\n\n\nReferences\nAltschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J. Mol. Biol. 1990; 215:403–410. [PubMed: 2231712]\nAmbros V. The functions of animal microRNAs. Nature. 2004; 431:350–355. [PubMed: 15372042] Chung W-J, Agius P, Westholm JO, Chen M, Okamura K, Robine N, Leslie CS, Lai EC.\nComputational and experimental identification of mirtrons in Drosophila melanogaster and Caenorhabditis elegans. Genome Res. 2011; 21:286–300. [PubMed: 21177960]\nDavis BN, Hata A. Regulation of MicroRNA Biogenesis: A miRiad of mechanisms. Cell Commun. Signal. 2009; 7:18. [PubMed: 19664273]\nDesvignes T, Contreras A, Postlethwait JH. Evolution of the miR199–214 cluster and vertebrate skeletal development. RNA Biol. 2014; 11:0–1.\nEberhart JK, He X, Swartz ME, Yan Y-L, Song H, Boling TC, Kunerth AK, Walker MB, Kimmel CB, Postlethwait JH. MicroRNA Mirn140 modulates Pdgf signaling during palatogenesis. Nat. Genet. 2008; 40:290–298. [PubMed: 18264099]\n\n\nNIH-PA Author Manuscript\nM, Sheppard D, Sobral D, Taylor K, Thormann A, Trevanion S, White S, Wilder SP, Aken BL, Birney E, Cunningham F, Dunham I, Harrow J, Herrero J, Hubbard TJP, Johnson N, Kinsella R, Parker A, Spudich G, Yates A, Zadissa A, Searle SMJ. Ensembl 2013. Nucleic Acids Res. 2012; 41:D48–D55. [PubMed: 23203987]\nFlynt AS, Li N, Thatcher EJ, Solnica-Krezel L, Patton JG. Zebrafish miR-214 modulates Hedgehog signaling to specify muscle cell fate. Nat. Genet. 2007; 39:259–263. [PubMed: 17220889]\nGiraldez AJ, Cinalli RM, Glasner ME, Enright AJ, Thomson JM, Baskerville S, Hammond SM, Bartel DP, Schier AF. MicroRNAs Regulate Brain Morphogenesis in Zebrafish. Science. 2005; 308:833– 838. [PubMed: 15774722]\nGiraldez AJ, Mishima Y, Rihel J, Grocock RJ, Dongen SV, Inoue K, Enright AJ, Schier AF. Zebrafish MiR-430 Promotes Deadenylation and Clearance of Maternal mRNAs. Science. 2006; 312:75–79. [PubMed: 16484454]\nHe X, Eberhart JK, Postlethwait JH. MicroRNAs and micromanaging the skeleton in disease, development and evolution. J. Cell. Mol. Med. 2009; 13:606–618. [PubMed: 19220576]\nHe X, Yan Y-L, DeLaurier A, Postlethwait JH. Observation of miRNA gene expression in zebrafish embryos by in situ hybridization to microRNA primary transcripts. Zebrafish. 2011a; 8:1–8. [PubMed: 21288128]\nHe X, Yan Y-L, Eberhart JK, Herpin A, Wagner TU, Schartl M, Postlethwait JH. miR-196 regulates axial patterning and pectoral appendage initiation. Dev. Biol. 2011b; 357:463–477. [PubMed: 21787766]\nHowe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, Collins JE, Humphray S, McLaren K, Matthews L, McLaren S, Sealy I, Caccamo M, Churcher C, Scott C, Barrett JC, Koch R, Rauch G-J, White S, Chow W, Kilian B, Quintais LT, Guerra-Assuncao JA, Zhou Y, Gu Y, Yen J, Vogel J-H, Eyre T, Redmond S, Banerjee R, Chi J, Fu B, Langley E, Maguire SF, Laird GK, Lloyd D, Kenyon E, Donaldson S, Sehra H, Almeida-King J, Loveland J, Trevanion S, Jones M, Quail M, Willey D, Hunt A, Burton J, Sims S, McLay K, Plumb B, Davis J, Clee C, Oliver K, Clark R, Riddle C, Eliott D, Threadgold G, Harden G, Ware D, Mortimer B, Kerry G, Heath P, Phillimore B, Tracey A, Corby N, Dunn M, Johnson C, Wood J, Clark S, Pelan S, Griffiths G, Smith M, Glithero R, Howden P, Barker N, Stevens C, Harley J, Holt K, Panagiotidis G, Lovell J, Beasley H, Henderson C, Gordon D, Auger K, Wright D, Collins J, Raisen C, Dyer L, Leung K, Robertson L, Ambridge K, Leongamornlert D, McGuire S, Gilderthorp R, Griffiths C, Manthravadi D, Nichol S, Barker G, Whitehead S, Kay M, Brown J, Murnane C, Gray E, Humphries M, Sycamore N, Barker D, Saunders D, Wallis J, Babbage A, Hammond S, Mashreghi-Mohammadi M, Barr L, Martin S, Wray P, Ellington A, Matthews N, Ellwood M, Woodmansey R, Clark G, Cooper J, Tromans A, Grafham D, Skuce C, Pandian R, Andrews R, Harrison E, Kimberley A, Garnett J, Fosker N, Hall R, Garner P, Kelly D, Bird C, Palmer S, Gehring I, Berger A, Dooley CM, Ersan-Urun Z, Eser C, Geiger H, Geisler M, Karotki L, Kirn A, Konantz J, Konantz M, Oberlander M, Rudolph-Geiger S, Teucke M, Osoegawa K, Zhu B, Rapp A, Widaa S, Langford C, Yang F, Carter NP, Harrow J, Ning Z, Herrero J, Searle SMJ, Enright A, Geisler R, Plasterk RHA, Lee C, Westerfield M, de Jong PJ, Zon LI, Postlethwait JH, Nusslein- Volhard C, Hubbard TJP, Crollius HR, Rogers J, Stemple DL. The zebrafish reference genome sequence and its relationship to the human genome. Nature. 2013 advance online publication.\nIvey KN, Srivastava D. MicroRNAs as Regulators of Differentiation and Cell Fate Decisions. Cell Stem Cell. 2010; 7:36–41. [PubMed: 20621048]\nKim M, Tan YS, Cheng W-C, Civin CI. MiR-144 and MiR-451 Regulate Human Erythropoiesis By Targeting RAB14. Blood. 2013; 122:942–942.\nKim VN, Han J, Siomi MC. Biogenesis of small RNAs in animals. Nat. Rev. Mol. Cell Biol. 2009; 10:126–139. [PubMed: 19165215]\nKosik KS. MicroRNAs and Cellular Phenotypy. Cell. 2010; 143:21–26. [PubMed: 20887887]\nKozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic Acids Res. 2013; 42:D68–D73. [PubMed: 24275495]\nLadewig E, Okamura K, Flynt AS, Westholm JO, Lai EC. Discovery of hundreds of mirtrons in mouse and human small RNA data. Genome Res. 2012; 22:1634–1645. [PubMed: 22955976]\n\n\nNIH-PA Author Manuscript\nLi L, Meng T, Jia Z, Zhu G, Shi B. Single nucleotide polymorphism associated with nonsyndromic cleft palate influences the processing of miR-140. Am. J. Med. Genet. A. 2010; 152A:856–862. [PubMed: 20358594]\nMeiri E, Levy A, Benjamin H, Ben-David M, Cohen L, Dov A, Dromi N, Elyakim E, Yerushalmi N, Zion O, Lithwick-Yanai G, Sitbon E. Discovery of microRNAs and other small RNAs in solid tumors. Nucleic Acids Res. 2010; 38:6234–6246. [PubMed: 20483914]\nMishima Y, Abreu-Goodger C, Staton AA, Stahlhut C, Shou C, Cheng C, Gerstein M, Enright AJ, Giraldez AJ. Zebrafish miR-1 and miR-133 shape muscle gene expression and regulate sarcomeric actin organization. Genes Dev. 2009; 23:619–632. [PubMed: 19240126]\nNelson, JS. Fishes of the World. John Wiley &amp; Sons; 2006.\nNicolas FE, Hall AE, Csorba T, Turnbull C, Dalmay T. Biogenesis of Y RNA-derived small RNAs is independent of the microRNA pathway. FEBS Lett. 2012; 586:1226–1230. [PubMed: 22575660]\nOkamura K, Hagen JW, Duan H, Tyler DM, Lai EC. The Mirtron Pathway Generates microRNA- Class Regulatory RNAs in Drosophila. Cell. 2007; 130:89–100. [PubMed: 17599402]\nPase L, Layton JE, Kloosterman WP, Carradice D, Waterhouse PM, Lieschke GJ. miR-451 regulates zebrafish erythroid maturation in vivo via its target gata2. Blood. 2009; 113:1794–1804. [PubMed: 18849488]\nPatrick DM, Zhang CC, Tao Y, Yao H, Qi X, Schwartz RJ, Huang LJ-S, Olson EN. Defective erythroid differentiation in miR-451 mutant mice mediated by 14-3-3 ζ . Genes Dev. 2010; 24:1614–1619. [PubMed: 20679397]\nPersson H, Kvist A, Vallon-Christersson J, Medstrand P, Borg Å, Rovira C. The non-coding RNA of the multidrug resistance-linked vault particle encodes multiple regulatory small RNAs. Nat. Cell Biol. 2009; 11:1268–1271. [PubMed: 19749744]\nRasmussen KD, Simmini S, Abreu-Goodger C, Bartonicek N, Giacomo MD, Bilbao-Cortes D, Horos R, Lindern MV, Enright AJ, O’Carroll D. The miR-144/451 locus is required for erythroid homeostasis. J. Exp. Med. 2010; 207:1351–1358. [PubMed: 20513743]\nScott H, Howarth J, Lee YB, Wong L-F, Bantounas I, Phylactou L, Verkade P, Uney JB. MiR-3120 Is a Mirror MicroRNA That Targets Heat Shock Cognate Protein 70 and Auxilin Messenger RNAs and Regulates Clathrin Vesicle Uncoating. J. Biol. Chem. 2012; 287:14726–14733. [PubMed: 22393045]\nStadler PF, Chen JJ-L, Hackermüller J, Hoffmann S, Horn F, Khaitovich P, Kretzschmar AK, Mosig A, Prohaska SJ, Qi X, Schutt K, Ullmann K. Evolution of vault RNAs. Mol. Biol. Evol. 2009; 26:1975–1991. [PubMed: 19491402]\nTyler DM, Okamura K, Chung W-J, Hagen JW, Berezikov E, Hannon GJ, Lai EC. Functionally distinct regulatory RNAs generated by bidirectional transcription and processing of microRNA loci. Genes Dev. 2008; 22:26–36. [PubMed: 18172163]\nVerhagen APM, Pruijn GJM. Are the Ro RNP-associated Y RNAs concealing microRNAs? Y RNA- derived miRNAs may be involved in autoimmunity. BioEssays. 2011; 33:674–682. [PubMed: 21735459]\nWesterfield, M. A guide for the laboratory use of zebrafish (Danio rerio). 4th ed.. Eugene: Univ. of Oregon Press; 2000. The zebrafish book.\nWienholds E, Kloosterman WP, Miska E, Alvarez-Saavedra E, Berezikov E, Bruijn E, de, Horvitz HR, Kauppinen S, Plasterk RHA. MicroRNA Expression in Zebrafish Embryonic Development. Science. 2005; 309:310–311. [PubMed: 15919954]\nXu Z, Chen J, Li X, Ge J, Pan J, Xu X. Identification and Characterization of MicroRNAs in Channel Catfish (Ictalurus punctatus) by Using Solexa Sequencing Technology. PLoS ONE. 2013; 8:e54174. [PubMed: 23342099]\nYang J-S, Lai EC. Alternative miRNA Biogenesis Pathways and the Interpretation of Core miRNA Pathway Mutants. Mol. Cell. 2011; 43:892–903. [PubMed: 21925378]\nZuker M, Stiegler P. Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information. Nucleic Acids Res. 1981; 9:133–148. [PubMed: 6163133]\n\n\nAbbreviations list\nmiR microRNA\nsmallRNA-seq small RNA sequencing\n\n\nHighlights\n\n\nAnnotation of 56 new miRNA genes in zebrafish\nAnnotation of 248 new mature miRNA strands in zebrafish\nFirst description of mirror-miR pairs outside of Drosophila and mammals\nFirst description of 3’-tailed mirtrons outside of Drosophila, worms and mammals\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 2 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 4 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 6\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 7\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nUTR untranslated region RISC RNA-induced silencing complex Desvignes et al. Page 8 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 9\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 10 Table 1\n\n\nZebrafish miRBase Release 20 Present study % increase Total mir genes 346 56 16 402 mature sequences 391 248 63 639 mirror-mir pairs 0 3 N/A 3 mirtrons 0 2 N/A 2\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Expanding the annotation of zebrafish microRNAs based on\nsmallRNA sequencing\nThomas Desvignes a,* , Michael J. Beam a , Peter Batzel a , Jason Sydes a,b , and John H.\nPostlethwait a\na Institute of Neuroscience, University of Oregon, Eugene, OR 97403, USA\nb Institute of Ecology and Evolution, University of Oregon, Eugene, OR 97403, USA\n\n\nAbstract\nMicroRNAs (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes. Because of their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation and are increasingly relevant to human health and disease.\nAlthough the zebrafish Danio rerio is a major model for studies of development, genetics,\nphysiology, evolution, and human biology, the annotation of zebrafish miR-producing genes remains limited. In the present work, we report deep sequencing data of zebrafish smallRNAs from brain, heart, testis, and ovary. Results provide evidence for the expression of 56 un-annotated mir genes and 248 un-annotated mature strands, increasing the number of zebrafish mir genes over those already deposited in miRBase by 16% and the number of mature sequences by 63%. We also describe the existence of three pairs of mirror-mir genes and two mirtron genes, genetic features previously undescribed in non-mammalian vertebrates. This report provides information that substantially increases our knowledge of the zebrafish miRNome and will benefit the entire miR community.\n\n\nKeywords\nmiRNA; zebrafish; teleost; smallRNA-seq; mirror-mir; mirtron\n\n\nIntroduction\nMicroRNA (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes (Ambros, 2004; Kosik, 2010). MicroRNAs are transcribed from mir genes and primary miR transcripts are processed to approximately 22 nucleotide single strand mature forms that function as repressors of\n* Corresponding author Work Phone Number: (541)346-4495.\nGene. Author manuscript; available in PMC 2015 August 10.\nPublished in final edited form as:\nGene. 2014 August 10; 546(2): 386–389. doi:10.1016/j.gene.2014.05.036.\n\n\nNIH-PA Author Manuscript\ntranscript translation when bound to the 3’UTR of protein-coding transcripts in association with the RISC (Davis and Hata, 2009; Kim et al., 2009). Due to their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation (Ivey and Srivastava, 2010). the zebrafish Danio rerio is a major model for studies of development, genetics, physiology, evolution, and human health and disease, and many miR-related discoveries have come from zebrafish investigations (Giraldez et al., 2005; Wienholds et al., 2005; Giraldez et al., 2006; Flynt et al., 2007; He et al., 2009; Mishima et al., 2009; He et al., 2011a, 2011b). For instance, mir140 was shown to regulate zebrafish palatal development by down-regulating the translation of Pdgfra protein in neural crest derived cells (Eberhart et al., 2008).\nExperiments stimulated by that discovery showed that a single nucleotide polymorphism in the human MIR140 gene leads to improper mature MIR140 production and contributes to nonsyndromic cleft palate susceptibility in patients (Li et al., 2010). Another example of successful transfer of discoveries from zebrafish to human is the discovery of the crucial role of mir451 in zebrafish erythropoiesis (Pase et al., 2009), which was later confirmed in mouse (Rasmussen et al., 2010; Patrick et al., 2010) and recently in human (Kim et al.,\n2013), thereby highlighting the therapeutic potential of miR inhibitors. Despite the high impact of zebrafish research in many aspects of development and physiology, the annotation of zebrafish miR-producing genes remains limited.\nThis limitation in our knowledge is a broad problem because the annotation of the zebrafish genome, the most complete among fish (Howe et al., 2013), often drives annotation in all teleost fish, which constitute half of all vertebrate species (Nelson, 2006). Moreover, a largely incomplete characterization of the miR repertoire compromises studies on miRNA evolution, their emergence and retention in vertebrates, especially in the rayfin fish lineage following the teleost genome duplication. Furthermore, uncovering and transferring evolutionarily conserved miR functions from fish models to human health depends on recognizing true fish orthologs of human miRs. Thus, the entire miR community would benefit substantially by a better understanding of the zebrafish miRNome.\nIn the present report, we provide extensive miRNA sequencing data from several zebrafish tissues demonstrating the expression of new mir genes and the expression of previously un- annotated mature strands. Further, we describe for the first time in non-mammalian vertebrates, the expression of mirror-mirs (Scott et al., 2012; Tyler et al., 2008) and mirtrons (Chung et al., 2011; Ladewig et al., 2012; Okamura et al., 2007).\n\n\nMaterials and methods\nSmall RNAs were extracted using the Norgen microRNA purification kit from four different organs (brain, heart, testis and ovary). Animals were actively reproductive adult “AB” strain zebrafish (Danio rerio) obtained from the University of Oregon fish facility. From two males, we sampled brain, heart, and testes and made six libraries, three for each individual; from two females, we sampled ovaries and made two libraries, one for each individual. Eight tissue-specific sequencing libraries were prepared and barcoded using the\nBiooScientific NEXTflex smallRNA Sequencing Kit which uses a 3’ adenylated adapter that ligates onto miRs and other small RNAs with a 3’ hydroxyl group, following the\n\n\nNIH-PA Author Manuscript\nmanufacturer’s instructions. Libraries were subsequently sequenced by Illumina HiSeq2500 at the University of Oregon Genomics Core Facility and raw single-end 50nt long reads were deposited in the NCBI Short Reads Archive under the accession number SRP039502. All animal work was performed according to the University of Oregon IACUC approved protocol (#09–1BRRA).\nBioinformatic processing involved removing 3’ adapter sequences and filtering reads based on quality; reads with any base Q&lt;30 were removed using the FASTX-Toolkit ( http:// hannonlab.cshl.edu/fastx_toolkit/ ). A custom pipeline was created to remove reads outside of the targeted size range (&lt;15 and &gt;28), count the number of occurrences of each read, and remove reads with low counts (&lt;30 summed across all eight libraries). The script then aligns reads to the Zv9 version of the zebrafish reference genome using blastn (Altschul et al.,\n1990) allowing up to five mismatches, one gap, and alignments at up to 20 locations in the genome; finally, the script groups reads together based on their genomic locations. Aligned groups were annotated against mature and hairpin sequences already annotated in miRBase (release 20)(Kozomara and Griffiths-Jones, 2013) as well as zebrafish noncoding RNA sequences annotated by Ensembl (release 74)(Flicek et al., 2012).\nPutative new mirs were predicted based on read presence in our experimental dataset and secondary structures were computed on the RNAfold web server (Zuker and Stiegler, 1981) using default parameters except for changing the calculation of minimum free energy from 37°C to 28°C, the temperature at which zebrafish are reared (Westerfield, 2000). A tentative name was assigned to newly recognized sequences based on identity with deposited sequences in miRBase (Kozomara and Griffiths-Jones, 2013) for zebrafish or other species.\n\n\nResults/discussion\nThe sequencing of small RNAs with 3’ hydroxyls from several tissues of adult zebrafish yielded a total of 59.5 million reads that passed all filtering criteria, among which 47.5 million (79.8%) could be directly annotated from mature zebrafish entries in miRBase due to sequence identity. Among remaining reads, sequences mapping perfectly to the zebrafish reference genome at fewer than 20 locations were further studied for putative new mir genes and/or for mature strand annotation.\nThe annotation pipeline identified 56 mir genes that do not appear in miRBase, although 43 have of them have been bioinformatically predicted as “Novel miRNA” genes in Ensembl (Flicek et al., 2012); thus, our work increases the number of annotated zebrafish mir genes by 16% compared to the most recent miRBase (Release20) (Kozomara and Griffiths-Jones, 2013) (Table 1). Most of these new mir genes (54/56) are clear homologs to genes belonging to known mir families or described in related species and could thus be easily added to their corresponding group.\nTwo of the miR-encoding genes we identified in our experimental data, tentatively named dre-mir-733-2 and dre-mir-733b, are paralogs of dre-mir-733 (MI0004778) and originate from vaultRNAs, and a third miR, tentatively named dre-mir-735b, a paralog of dre-mir-735 (MI0004781), actually originates from a Y_RNA gene. Human orthologs (hsa-mir-886 and\n\n\nNIH-PA Author Manuscript\nhsa-mir-1979 respectively) of these zebrafish genes were withdrawn from miRBase after their biogenesis was understood (Stadler et al., 2009; Meiri et al., 2010). The role of vaultRNA and Y_RNA short fragments in translation repression is, however, not fully understood and they may act in a “mir-like” manner (Meiri et al., 2010; Persson et al., 2009; Nicolas et al., 2012), as do some snoRNA-derived or lncRNA-derived miRs (Yang and Lai, 2011). If the vaultRNA and Y_RNA derived short sequences do act like miRs, then production of short RNA fragments from vaultRNA and Y_RNA would be a new non- canonical way of generating miRs (Verhagen and Pruijn, 2011).\nOur zebrafish smallRNA-seq dataset contained two completely new mir genes that showed no similarity with sequences yet deposited in miRBase for any species. These two new mirs, along with the 54 mir genes in our experimental dataset that are predicted in Ensembl but not referenced in miRBase, are described in the Supplemental File, which gives a tentative name, genomic localizations, predicted hairpin sequences, corresponding secondary structures, minimum free energy calculated at 28°C, the localization of mature sequences on the hairpin sequence, and secondary structure.\nThe breadth and depth of our sequencing dataset also allowed us to provide annotation for a total of 248 mature strands, 158 of them corresponding to previously annotated mir genes and 90 from the set of 56 new mir genes not present yet in miRBase and described here above, increasing the total number of annotated zebrafish mature sequences from 391 to 639 (a 63% increase) (Table 1). The Supplemental File gives the sequence, and genomic location of each newly identified miR.\nAmong the newly annotated structures are miRs with several interesting features. The zebrafish smallRNA-seq dataset clearly contained three pairs of mirror-mirs (Table 1), which are defined as a pair of mir genes originating from overlapping genomic regions but on opposite strands (Tyler et al., 2008; Scott et al., 2012). For example, the expression of mir3120, mirror-mir of mir214, has been described in mammals but in no other species (Scott et al., 2012; Desvignes et al., 2014). Here we show that miR3120, a mature fragment originating from the mirror-mir of mir214, is expressed in adult zebrafish. We also found the pair mir7547 and mir7553, which had been previously identified as independent mirs in the channel catfish Ictalurus punctatus (MI0024671 and MI0024685 respectively)(Xu et al.,\n2013), but had not been described as a mirror-mir pair, probably due to the lack of a reference genome. The newly annotated zebrafish mir7552b and its mirror gene, mir7552bOS, had also not previously been annotated in zebrafish, even though a homolog of mir7552b is annotated in the channel catfish (MI0024684)(Xu et al., 2013). This finding is of particular interest given that, to our knowledge, mirror-mirs have been described so far only in Drosophila and mammals (Tyler et al., 2008; Scott et al., 2012). Evidence given in this report demonstrating the expression of mirror-mirs in zebrafish suggests that mirror- mirs are likely a type of gene featured in all vertebrates.\nFinally, we also report the presence and expression of a pair of 3’-tailed mirtron paralogs located in gria3 ohnologs (gria3a, ENSDARG00000032737; and gria3b,\nENSDARG00000037498) (Table 1). These mir genes are co-orthologous mirtrons to the predicted human mirtron gene AL356213.1 (ENSG00000265082), which is located in\n\n\nNIH-PA Author Manuscript\nGRIA3 (ENSG00000125675); they lie at orthologous positions in both zebrafish ohnologs; and they show high similarities with tetrapod mir2985, which is deposited in miRBase in zebra finch Taeniopygia guttata, platypus Ornithorhynchus anatinus and rat Rattus norvegicus as intronic MiRs of the Gria2 genes (Kozomara and Griffiths-Jones, 2013).\nSimilar to mirror-mirs, to our knowledge, mirtrons have only been shown in Drosophila, the nematode Caenorhabditis elegans and mammals (Okamura et al., 2007; Chung et al., 2011; Ladewig et al., 2012). This report is thus the first example of mirtrons in a non-mammalian vertebrate.\n\n\nConclusion\nThe present report analyzing smallRNA deep sequencing data expands the annotation of microRNA genes and mature strands in the laboratory model zebrafish Danio rerio, whose mir gene repertoire remains understudied and poorly characterized in comparison to other species, such as Drosophila, worms, mouse, or human, despite its relevance for microRNA research. Increasing the annottion of zebrafish miRs will contribute to a better understanding of the evolution of mir genes in vertebrates and within the rayfin fish lineage, will help annotate mir genes in other teleost species, and will increase confidence in the annotation of each gene thanks to the description of both mature strands, considered as a criterion for mir gene annotation confidence (Kozomara and Griffiths-Jones, 2013).\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nAuthors would like to thank the University of Oregon fish facility crew, especially Trevor Enright, for help in fish husbandry, and members of the Postlethwait Lab and of the Institute of Neuroscience of the University of Oregon for helpful discussions. This work was funded by NIH grants U01 DE020076, R01 OD011116 awarded to JHP and 5R240D011199 to W. Cresko and JHP.\n\n\nReferences\nAltschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J. Mol. Biol. 1990; 215:403–410. [PubMed: 2231712]\nAmbros V. The functions of animal microRNAs. Nature. 2004; 431:350–355. [PubMed: 15372042] Chung W-J, Agius P, Westholm JO, Chen M, Okamura K, Robine N, Leslie CS, Lai EC.\nComputational and experimental identification of mirtrons in Drosophila melanogaster and Caenorhabditis elegans. Genome Res. 2011; 21:286–300. [PubMed: 21177960]\nDavis BN, Hata A. Regulation of MicroRNA Biogenesis: A miRiad of mechanisms. Cell Commun. Signal. 2009; 7:18. [PubMed: 19664273]\nDesvignes T, Contreras A, Postlethwait JH. Evolution of the miR199–214 cluster and vertebrate skeletal development. RNA Biol. 2014; 11:0–1.\nEberhart JK, He X, Swartz ME, Yan Y-L, Song H, Boling TC, Kunerth AK, Walker MB, Kimmel CB, Postlethwait JH. MicroRNA Mirn140 modulates Pdgf signaling during palatogenesis. Nat. Genet. 2008; 40:290–298. [PubMed: 18264099]\n\n\nNIH-PA Author Manuscript\nM, Sheppard D, Sobral D, Taylor K, Thormann A, Trevanion S, White S, Wilder SP, Aken BL, Birney E, Cunningham F, Dunham I, Harrow J, Herrero J, Hubbard TJP, Johnson N, Kinsella R, Parker A, Spudich G, Yates A, Zadissa A, Searle SMJ. Ensembl 2013. Nucleic Acids Res. 2012; 41:D48–D55. [PubMed: 23203987]\nFlynt AS, Li N, Thatcher EJ, Solnica-Krezel L, Patton JG. Zebrafish miR-214 modulates Hedgehog signaling to specify muscle cell fate. Nat. Genet. 2007; 39:259–263. [PubMed: 17220889]\nGiraldez AJ, Cinalli RM, Glasner ME, Enright AJ, Thomson JM, Baskerville S, Hammond SM, Bartel DP, Schier AF. MicroRNAs Regulate Brain Morphogenesis in Zebrafish. Science. 2005; 308:833– 838. [PubMed: 15774722]\nGiraldez AJ, Mishima Y, Rihel J, Grocock RJ, Dongen SV, Inoue K, Enright AJ, Schier AF. Zebrafish MiR-430 Promotes Deadenylation and Clearance of Maternal mRNAs. Science. 2006; 312:75–79. [PubMed: 16484454]\nHe X, Eberhart JK, Postlethwait JH. MicroRNAs and micromanaging the skeleton in disease, development and evolution. J. Cell. Mol. Med. 2009; 13:606–618. [PubMed: 19220576]\nHe X, Yan Y-L, DeLaurier A, Postlethwait JH. Observation of miRNA gene expression in zebrafish embryos by in situ hybridization to microRNA primary transcripts. Zebrafish. 2011a; 8:1–8. [PubMed: 21288128]\nHe X, Yan Y-L, Eberhart JK, Herpin A, Wagner TU, Schartl M, Postlethwait JH. miR-196 regulates axial patterning and pectoral appendage initiation. Dev. Biol. 2011b; 357:463–477. [PubMed: 21787766]\nHowe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, Collins JE, Humphray S, McLaren K, Matthews L, McLaren S, Sealy I, Caccamo M, Churcher C, Scott C, Barrett JC, Koch R, Rauch G-J, White S, Chow W, Kilian B, Quintais LT, Guerra-Assuncao JA, Zhou Y, Gu Y, Yen J, Vogel J-H, Eyre T, Redmond S, Banerjee R, Chi J, Fu B, Langley E, Maguire SF, Laird GK, Lloyd D, Kenyon E, Donaldson S, Sehra H, Almeida-King J, Loveland J, Trevanion S, Jones M, Quail M, Willey D, Hunt A, Burton J, Sims S, McLay K, Plumb B, Davis J, Clee C, Oliver K, Clark R, Riddle C, Eliott D, Threadgold G, Harden G, Ware D, Mortimer B, Kerry G, Heath P, Phillimore B, Tracey A, Corby N, Dunn M, Johnson C, Wood J, Clark S, Pelan S, Griffiths G, Smith M, Glithero R, Howden P, Barker N, Stevens C, Harley J, Holt K, Panagiotidis G, Lovell J, Beasley H, Henderson C, Gordon D, Auger K, Wright D, Collins J, Raisen C, Dyer L, Leung K, Robertson L, Ambridge K, Leongamornlert D, McGuire S, Gilderthorp R, Griffiths C, Manthravadi D, Nichol S, Barker G, Whitehead S, Kay M, Brown J, Murnane C, Gray E, Humphries M, Sycamore N, Barker D, Saunders D, Wallis J, Babbage A, Hammond S, Mashreghi-Mohammadi M, Barr L, Martin S, Wray P, Ellington A, Matthews N, Ellwood M, Woodmansey R, Clark G, Cooper J, Tromans A, Grafham D, Skuce C, Pandian R, Andrews R, Harrison E, Kimberley A, Garnett J, Fosker N, Hall R, Garner P, Kelly D, Bird C, Palmer S, Gehring I, Berger A, Dooley CM, Ersan-Urun Z, Eser C, Geiger H, Geisler M, Karotki L, Kirn A, Konantz J, Konantz M, Oberlander M, Rudolph-Geiger S, Teucke M, Osoegawa K, Zhu B, Rapp A, Widaa S, Langford C, Yang F, Carter NP, Harrow J, Ning Z, Herrero J, Searle SMJ, Enright A, Geisler R, Plasterk RHA, Lee C, Westerfield M, de Jong PJ, Zon LI, Postlethwait JH, Nusslein- Volhard C, Hubbard TJP, Crollius HR, Rogers J, Stemple DL. The zebrafish reference genome sequence and its relationship to the human genome. Nature. 2013 advance online publication.\nIvey KN, Srivastava D. MicroRNAs as Regulators of Differentiation and Cell Fate Decisions. Cell Stem Cell. 2010; 7:36–41. [PubMed: 20621048]\nKim M, Tan YS, Cheng W-C, Civin CI. MiR-144 and MiR-451 Regulate Human Erythropoiesis By Targeting RAB14. Blood. 2013; 122:942–942.\nKim VN, Han J, Siomi MC. Biogenesis of small RNAs in animals. Nat. Rev. Mol. Cell Biol. 2009; 10:126–139. [PubMed: 19165215]\nKosik KS. MicroRNAs and Cellular Phenotypy. Cell. 2010; 143:21–26. [PubMed: 20887887]\nKozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic Acids Res. 2013; 42:D68–D73. [PubMed: 24275495]\nLadewig E, Okamura K, Flynt AS, Westholm JO, Lai EC. Discovery of hundreds of mirtrons in mouse and human small RNA data. Genome Res. 2012; 22:1634–1645. [PubMed: 22955976]\n\n\nNIH-PA Author Manuscript\nLi L, Meng T, Jia Z, Zhu G, Shi B. Single nucleotide polymorphism associated with nonsyndromic cleft palate influences the processing of miR-140. Am. J. Med. Genet. A. 2010; 152A:856–862. [PubMed: 20358594]\nMeiri E, Levy A, Benjamin H, Ben-David M, Cohen L, Dov A, Dromi N, Elyakim E, Yerushalmi N, Zion O, Lithwick-Yanai G, Sitbon E. Discovery of microRNAs and other small RNAs in solid tumors. Nucleic Acids Res. 2010; 38:6234–6246. [PubMed: 20483914]\nMishima Y, Abreu-Goodger C, Staton AA, Stahlhut C, Shou C, Cheng C, Gerstein M, Enright AJ, Giraldez AJ. Zebrafish miR-1 and miR-133 shape muscle gene expression and regulate sarcomeric actin organization. Genes Dev. 2009; 23:619–632. [PubMed: 19240126]\nNelson, JS. Fishes of the World. John Wiley &amp; Sons; 2006.\nNicolas FE, Hall AE, Csorba T, Turnbull C, Dalmay T. Biogenesis of Y RNA-derived small RNAs is independent of the microRNA pathway. FEBS Lett. 2012; 586:1226–1230. [PubMed: 22575660]\nOkamura K, Hagen JW, Duan H, Tyler DM, Lai EC. The Mirtron Pathway Generates microRNA- Class Regulatory RNAs in Drosophila. Cell. 2007; 130:89–100. [PubMed: 17599402]\nPase L, Layton JE, Kloosterman WP, Carradice D, Waterhouse PM, Lieschke GJ. miR-451 regulates zebrafish erythroid maturation in vivo via its target gata2. Blood. 2009; 113:1794–1804. [PubMed: 18849488]\nPatrick DM, Zhang CC, Tao Y, Yao H, Qi X, Schwartz RJ, Huang LJ-S, Olson EN. Defective erythroid differentiation in miR-451 mutant mice mediated by 14-3-3 ζ . Genes Dev. 2010; 24:1614–1619. [PubMed: 20679397]\nPersson H, Kvist A, Vallon-Christersson J, Medstrand P, Borg Å, Rovira C. The non-coding RNA of the multidrug resistance-linked vault particle encodes multiple regulatory small RNAs. Nat. Cell Biol. 2009; 11:1268–1271. [PubMed: 19749744]\nRasmussen KD, Simmini S, Abreu-Goodger C, Bartonicek N, Giacomo MD, Bilbao-Cortes D, Horos R, Lindern MV, Enright AJ, O’Carroll D. The miR-144/451 locus is required for erythroid homeostasis. J. Exp. Med. 2010; 207:1351–1358. [PubMed: 20513743]\nScott H, Howarth J, Lee YB, Wong L-F, Bantounas I, Phylactou L, Verkade P, Uney JB. MiR-3120 Is a Mirror MicroRNA That Targets Heat Shock Cognate Protein 70 and Auxilin Messenger RNAs and Regulates Clathrin Vesicle Uncoating. J. Biol. Chem. 2012; 287:14726–14733. [PubMed: 22393045]\nStadler PF, Chen JJ-L, Hackermüller J, Hoffmann S, Horn F, Khaitovich P, Kretzschmar AK, Mosig A, Prohaska SJ, Qi X, Schutt K, Ullmann K. Evolution of vault RNAs. Mol. Biol. Evol. 2009; 26:1975–1991. [PubMed: 19491402]\nTyler DM, Okamura K, Chung W-J, Hagen JW, Berezikov E, Hannon GJ, Lai EC. Functionally distinct regulatory RNAs generated by bidirectional transcription and processing of microRNA loci. Genes Dev. 2008; 22:26–36. [PubMed: 18172163]\nVerhagen APM, Pruijn GJM. Are the Ro RNP-associated Y RNAs concealing microRNAs? Y RNA- derived miRNAs may be involved in autoimmunity. BioEssays. 2011; 33:674–682. [PubMed: 21735459]\nWesterfield, M. A guide for the laboratory use of zebrafish (Danio rerio). 4th ed.. Eugene: Univ. of Oregon Press; 2000. The zebrafish book.\nWienholds E, Kloosterman WP, Miska E, Alvarez-Saavedra E, Berezikov E, Bruijn E, de, Horvitz HR, Kauppinen S, Plasterk RHA. MicroRNA Expression in Zebrafish Embryonic Development. Science. 2005; 309:310–311. [PubMed: 15919954]\nXu Z, Chen J, Li X, Ge J, Pan J, Xu X. Identification and Characterization of MicroRNAs in Channel Catfish (Ictalurus punctatus) by Using Solexa Sequencing Technology. PLoS ONE. 2013; 8:e54174. [PubMed: 23342099]\nYang J-S, Lai EC. Alternative miRNA Biogenesis Pathways and the Interpretation of Core miRNA Pathway Mutants. Mol. Cell. 2011; 43:892–903. [PubMed: 21925378]\nZuker M, Stiegler P. Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information. Nucleic Acids Res. 1981; 9:133–148. [PubMed: 6163133]\n\n\nAbbreviations list\nmiR microRNA\nsmallRNA-seq small RNA sequencing\n\n\nHighlights\n\n\nAnnotation of 56 new miRNA genes in zebrafish\nAnnotation of 248 new mature miRNA strands in zebrafish\nFirst description of mirror-miR pairs outside of Drosophila and mammals\nFirst description of 3’-tailed mirtrons outside of Drosophila, worms and mammals\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 2 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 4 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 6\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 7\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nUTR untranslated region RISC RNA-induced silencing complex Desvignes et al. Page 8 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 9\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 10 Table 1\n\n\nZebrafish miRBase Release 20 Present study % increase Total mir genes 346 56 16 402 mature sequences 391 248 63 639 mirror-mir pairs 0 3 N/A 3 mirtrons 0 2 N/A 2\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Expanding the annotation of zebrafish microRNAs based on\nsmallRNA sequencing\nThomas Desvignes a,* , Michael J. Beam a , Peter Batzel a , Jason Sydes a,b , and John H.\nPostlethwait a\na Institute of Neuroscience, University of Oregon, Eugene, OR 97403, USA\nb Institute of Ecology and Evolution, University of Oregon, Eugene, OR 97403, USA\n\n\nAbstract\nMicroRNAs (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes. Because of their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation and are increasingly relevant to human health and disease.\nAlthough the zebrafish Danio rerio is a major model for studies of development, genetics,\nphysiology, evolution, and human biology, the annotation of zebrafish miR-producing genes remains limited. In the present work, we report deep sequencing data of zebrafish smallRNAs from brain, heart, testis, and ovary. Results provide evidence for the expression of 56 un-annotated mir genes and 248 un-annotated mature strands, increasing the number of zebrafish mir genes over those already deposited in miRBase by 16% and the number of mature sequences by 63%. We also describe the existence of three pairs of mirror-mir genes and two mirtron genes, genetic features previously undescribed in non-mammalian vertebrates. This report provides information that substantially increases our knowledge of the zebrafish miRNome and will benefit the entire miR community.\n\n\nKeywords\nmiRNA; zebrafish; teleost; smallRNA-seq; mirror-mir; mirtron\n\n\nIntroduction\nMicroRNA (miRs) are short non-coding RNAs that fine-tune the regulation of gene expression to coordinate a wide range of biological processes (Ambros, 2004; Kosik, 2010). MicroRNAs are transcribed from mir genes and primary miR transcripts are processed to approximately 22 nucleotide single strand mature forms that function as repressors of\n* Corresponding author Work Phone Number: (541)346-4495.\nGene. Author manuscript; available in PMC 2015 August 10.\nPublished in final edited form as:\nGene. 2014 August 10; 546(2): 386–389. doi:10.1016/j.gene.2014.05.036.\n\n\nNIH-PA Author Manuscript\ntranscript translation when bound to the 3’UTR of protein-coding transcripts in association with the RISC (Davis and Hata, 2009; Kim et al., 2009). Due to their role in the regulation of gene expression, miRs are essential players in development by acting on cell fate determination and progression towards cell differentiation (Ivey and Srivastava, 2010). the zebrafish Danio rerio is a major model for studies of development, genetics, physiology, evolution, and human health and disease, and many miR-related discoveries have come from zebrafish investigations (Giraldez et al., 2005; Wienholds et al., 2005; Giraldez et al., 2006; Flynt et al., 2007; He et al., 2009; Mishima et al., 2009; He et al., 2011a, 2011b). For instance, mir140 was shown to regulate zebrafish palatal development by down-regulating the translation of Pdgfra protein in neural crest derived cells (Eberhart et al., 2008).\nExperiments stimulated by that discovery showed that a single nucleotide polymorphism in the human MIR140 gene leads to improper mature MIR140 production and contributes to nonsyndromic cleft palate susceptibility in patients (Li et al., 2010). Another example of successful transfer of discoveries from zebrafish to human is the discovery of the crucial role of mir451 in zebrafish erythropoiesis (Pase et al., 2009), which was later confirmed in mouse (Rasmussen et al., 2010; Patrick et al., 2010) and recently in human (Kim et al.,\n2013), thereby highlighting the therapeutic potential of miR inhibitors. Despite the high impact of zebrafish research in many aspects of development and physiology, the annotation of zebrafish miR-producing genes remains limited.\nThis limitation in our knowledge is a broad problem because the annotation of the zebrafish genome, the most complete among fish (Howe et al., 2013), often drives annotation in all teleost fish, which constitute half of all vertebrate species (Nelson, 2006). Moreover, a largely incomplete characterization of the miR repertoire compromises studies on miRNA evolution, their emergence and retention in vertebrates, especially in the rayfin fish lineage following the teleost genome duplication. Furthermore, uncovering and transferring evolutionarily conserved miR functions from fish models to human health depends on recognizing true fish orthologs of human miRs. Thus, the entire miR community would benefit substantially by a better understanding of the zebrafish miRNome.\nIn the present report, we provide extensive miRNA sequencing data from several zebrafish tissues demonstrating the expression of new mir genes and the expression of previously un- annotated mature strands. Further, we describe for the first time in non-mammalian vertebrates, the expression of mirror-mirs (Scott et al., 2012; Tyler et al., 2008) and mirtrons (Chung et al., 2011; Ladewig et al., 2012; Okamura et al., 2007).\n\n\nMaterials and methods\nSmall RNAs were extracted using the Norgen microRNA purification kit from four different organs (brain, heart, testis and ovary). Animals were actively reproductive adult “AB” strain zebrafish (Danio rerio) obtained from the University of Oregon fish facility. From two males, we sampled brain, heart, and testes and made six libraries, three for each individual; from two females, we sampled ovaries and made two libraries, one for each individual. Eight tissue-specific sequencing libraries were prepared and barcoded using the\nBiooScientific NEXTflex smallRNA Sequencing Kit which uses a 3’ adenylated adapter that ligates onto miRs and other small RNAs with a 3’ hydroxyl group, following the\n\n\nNIH-PA Author Manuscript\nmanufacturer’s instructions. Libraries were subsequently sequenced by Illumina HiSeq2500 at the University of Oregon Genomics Core Facility and raw single-end 50nt long reads were deposited in the NCBI Short Reads Archive under the accession number SRP039502. All animal work was performed according to the University of Oregon IACUC approved protocol (#09–1BRRA).\nBioinformatic processing involved removing 3’ adapter sequences and filtering reads based on quality; reads with any base Q&lt;30 were removed using the FASTX-Toolkit ( http:// hannonlab.cshl.edu/fastx_toolkit/ ). A custom pipeline was created to remove reads outside of the targeted size range (&lt;15 and &gt;28), count the number of occurrences of each read, and remove reads with low counts (&lt;30 summed across all eight libraries). The script then aligns reads to the Zv9 version of the zebrafish reference genome using blastn (Altschul et al.,\n1990) allowing up to five mismatches, one gap, and alignments at up to 20 locations in the genome; finally, the script groups reads together based on their genomic locations. Aligned groups were annotated against mature and hairpin sequences already annotated in miRBase (release 20)(Kozomara and Griffiths-Jones, 2013) as well as zebrafish noncoding RNA sequences annotated by Ensembl (release 74)(Flicek et al., 2012).\nPutative new mirs were predicted based on read presence in our experimental dataset and secondary structures were computed on the RNAfold web server (Zuker and Stiegler, 1981) using default parameters except for changing the calculation of minimum free energy from 37°C to 28°C, the temperature at which zebrafish are reared (Westerfield, 2000). A tentative name was assigned to newly recognized sequences based on identity with deposited sequences in miRBase (Kozomara and Griffiths-Jones, 2013) for zebrafish or other species.\n\n\nResults/discussion\nThe sequencing of small RNAs with 3’ hydroxyls from several tissues of adult zebrafish yielded a total of 59.5 million reads that passed all filtering criteria, among which 47.5 million (79.8%) could be directly annotated from mature zebrafish entries in miRBase due to sequence identity. Among remaining reads, sequences mapping perfectly to the zebrafish reference genome at fewer than 20 locations were further studied for putative new mir genes and/or for mature strand annotation.\nThe annotation pipeline identified 56 mir genes that do not appear in miRBase, although 43 have of them have been bioinformatically predicted as “Novel miRNA” genes in Ensembl (Flicek et al., 2012); thus, our work increases the number of annotated zebrafish mir genes by 16% compared to the most recent miRBase (Release20) (Kozomara and Griffiths-Jones, 2013) (Table 1). Most of these new mir genes (54/56) are clear homologs to genes belonging to known mir families or described in related species and could thus be easily added to their corresponding group.\nTwo of the miR-encoding genes we identified in our experimental data, tentatively named dre-mir-733-2 and dre-mir-733b, are paralogs of dre-mir-733 (MI0004778) and originate from vaultRNAs, and a third miR, tentatively named dre-mir-735b, a paralog of dre-mir-735 (MI0004781), actually originates from a Y_RNA gene. Human orthologs (hsa-mir-886 and\n\n\nNIH-PA Author Manuscript\nhsa-mir-1979 respectively) of these zebrafish genes were withdrawn from miRBase after their biogenesis was understood (Stadler et al., 2009; Meiri et al., 2010). The role of vaultRNA and Y_RNA short fragments in translation repression is, however, not fully understood and they may act in a “mir-like” manner (Meiri et al., 2010; Persson et al., 2009; Nicolas et al., 2012), as do some snoRNA-derived or lncRNA-derived miRs (Yang and Lai, 2011). If the vaultRNA and Y_RNA derived short sequences do act like miRs, then production of short RNA fragments from vaultRNA and Y_RNA would be a new non- canonical way of generating miRs (Verhagen and Pruijn, 2011).\nOur zebrafish smallRNA-seq dataset contained two completely new mir genes that showed no similarity with sequences yet deposited in miRBase for any species. These two new mirs, along with the 54 mir genes in our experimental dataset that are predicted in Ensembl but not referenced in miRBase, are described in the Supplemental File, which gives a tentative name, genomic localizations, predicted hairpin sequences, corresponding secondary structures, minimum free energy calculated at 28°C, the localization of mature sequences on the hairpin sequence, and secondary structure.\nThe breadth and depth of our sequencing dataset also allowed us to provide annotation for a total of 248 mature strands, 158 of them corresponding to previously annotated mir genes and 90 from the set of 56 new mir genes not present yet in miRBase and described here above, increasing the total number of annotated zebrafish mature sequences from 391 to 639 (a 63% increase) (Table 1). The Supplemental File gives the sequence, and genomic location of each newly identified miR.\nAmong the newly annotated structures are miRs with several interesting features. The zebrafish smallRNA-seq dataset clearly contained three pairs of mirror-mirs (Table 1), which are defined as a pair of mir genes originating from overlapping genomic regions but on opposite strands (Tyler et al., 2008; Scott et al., 2012). For example, the expression of mir3120, mirror-mir of mir214, has been described in mammals but in no other species (Scott et al., 2012; Desvignes et al., 2014). Here we show that miR3120, a mature fragment originating from the mirror-mir of mir214, is expressed in adult zebrafish. We also found the pair mir7547 and mir7553, which had been previously identified as independent mirs in the channel catfish Ictalurus punctatus (MI0024671 and MI0024685 respectively)(Xu et al.,\n2013), but had not been described as a mirror-mir pair, probably due to the lack of a reference genome. The newly annotated zebrafish mir7552b and its mirror gene, mir7552bOS, had also not previously been annotated in zebrafish, even though a homolog of mir7552b is annotated in the channel catfish (MI0024684)(Xu et al., 2013). This finding is of particular interest given that, to our knowledge, mirror-mirs have been described so far only in Drosophila and mammals (Tyler et al., 2008; Scott et al., 2012). Evidence given in this report demonstrating the expression of mirror-mirs in zebrafish suggests that mirror- mirs are likely a type of gene featured in all vertebrates.\nFinally, we also report the presence and expression of a pair of 3’-tailed mirtron paralogs located in gria3 ohnologs (gria3a, ENSDARG00000032737; and gria3b,\nENSDARG00000037498) (Table 1). These mir genes are co-orthologous mirtrons to the predicted human mirtron gene AL356213.1 (ENSG00000265082), which is located in\n\n\nNIH-PA Author Manuscript\nGRIA3 (ENSG00000125675); they lie at orthologous positions in both zebrafish ohnologs; and they show high similarities with tetrapod mir2985, which is deposited in miRBase in zebra finch Taeniopygia guttata, platypus Ornithorhynchus anatinus and rat Rattus norvegicus as intronic MiRs of the Gria2 genes (Kozomara and Griffiths-Jones, 2013).\nSimilar to mirror-mirs, to our knowledge, mirtrons have only been shown in Drosophila, the nematode Caenorhabditis elegans and mammals (Okamura et al., 2007; Chung et al., 2011; Ladewig et al., 2012). This report is thus the first example of mirtrons in a non-mammalian vertebrate.\n\n\nConclusion\nThe present report analyzing smallRNA deep sequencing data expands the annotation of microRNA genes and mature strands in the laboratory model zebrafish Danio rerio, whose mir gene repertoire remains understudied and poorly characterized in comparison to other species, such as Drosophila, worms, mouse, or human, despite its relevance for microRNA research. Increasing the annottion of zebrafish miRs will contribute to a better understanding of the evolution of mir genes in vertebrates and within the rayfin fish lineage, will help annotate mir genes in other teleost species, and will increase confidence in the annotation of each gene thanks to the description of both mature strands, considered as a criterion for mir gene annotation confidence (Kozomara and Griffiths-Jones, 2013).\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nAuthors would like to thank the University of Oregon fish facility crew, especially Trevor Enright, for help in fish husbandry, and members of the Postlethwait Lab and of the Institute of Neuroscience of the University of Oregon for helpful discussions. This work was funded by NIH grants U01 DE020076, R01 OD011116 awarded to JHP and 5R240D011199 to W. Cresko and JHP.\n\n\nReferences\nAltschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J. Mol. Biol. 1990; 215:403–410. [PubMed: 2231712]\nAmbros V. The functions of animal microRNAs. Nature. 2004; 431:350–355. [PubMed: 15372042] Chung W-J, Agius P, Westholm JO, Chen M, Okamura K, Robine N, Leslie CS, Lai EC.\nComputational and experimental identification of mirtrons in Drosophila melanogaster and Caenorhabditis elegans. Genome Res. 2011; 21:286–300. [PubMed: 21177960]\nDavis BN, Hata A. Regulation of MicroRNA Biogenesis: A miRiad of mechanisms. Cell Commun. Signal. 2009; 7:18. [PubMed: 19664273]\nDesvignes T, Contreras A, Postlethwait JH. Evolution of the miR199–214 cluster and vertebrate skeletal development. RNA Biol. 2014; 11:0–1.\nEberhart JK, He X, Swartz ME, Yan Y-L, Song H, Boling TC, Kunerth AK, Walker MB, Kimmel CB, Postlethwait JH. MicroRNA Mirn140 modulates Pdgf signaling during palatogenesis. Nat. Genet. 2008; 40:290–298. [PubMed: 18264099]\n\n\nNIH-PA Author Manuscript\nM, Sheppard D, Sobral D, Taylor K, Thormann A, Trevanion S, White S, Wilder SP, Aken BL, Birney E, Cunningham F, Dunham I, Harrow J, Herrero J, Hubbard TJP, Johnson N, Kinsella R, Parker A, Spudich G, Yates A, Zadissa A, Searle SMJ. Ensembl 2013. Nucleic Acids Res. 2012; 41:D48–D55. [PubMed: 23203987]\nFlynt AS, Li N, Thatcher EJ, Solnica-Krezel L, Patton JG. Zebrafish miR-214 modulates Hedgehog signaling to specify muscle cell fate. Nat. Genet. 2007; 39:259–263. [PubMed: 17220889]\nGiraldez AJ, Cinalli RM, Glasner ME, Enright AJ, Thomson JM, Baskerville S, Hammond SM, Bartel DP, Schier AF. MicroRNAs Regulate Brain Morphogenesis in Zebrafish. Science. 2005; 308:833– 838. [PubMed: 15774722]\nGiraldez AJ, Mishima Y, Rihel J, Grocock RJ, Dongen SV, Inoue K, Enright AJ, Schier AF. Zebrafish MiR-430 Promotes Deadenylation and Clearance of Maternal mRNAs. Science. 2006; 312:75–79. [PubMed: 16484454]\nHe X, Eberhart JK, Postlethwait JH. MicroRNAs and micromanaging the skeleton in disease, development and evolution. J. Cell. Mol. Med. 2009; 13:606–618. [PubMed: 19220576]\nHe X, Yan Y-L, DeLaurier A, Postlethwait JH. Observation of miRNA gene expression in zebrafish embryos by in situ hybridization to microRNA primary transcripts. Zebrafish. 2011a; 8:1–8. [PubMed: 21288128]\nHe X, Yan Y-L, Eberhart JK, Herpin A, Wagner TU, Schartl M, Postlethwait JH. miR-196 regulates axial patterning and pectoral appendage initiation. Dev. Biol. 2011b; 357:463–477. [PubMed: 21787766]\nHowe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, Collins JE, Humphray S, McLaren K, Matthews L, McLaren S, Sealy I, Caccamo M, Churcher C, Scott C, Barrett JC, Koch R, Rauch G-J, White S, Chow W, Kilian B, Quintais LT, Guerra-Assuncao JA, Zhou Y, Gu Y, Yen J, Vogel J-H, Eyre T, Redmond S, Banerjee R, Chi J, Fu B, Langley E, Maguire SF, Laird GK, Lloyd D, Kenyon E, Donaldson S, Sehra H, Almeida-King J, Loveland J, Trevanion S, Jones M, Quail M, Willey D, Hunt A, Burton J, Sims S, McLay K, Plumb B, Davis J, Clee C, Oliver K, Clark R, Riddle C, Eliott D, Threadgold G, Harden G, Ware D, Mortimer B, Kerry G, Heath P, Phillimore B, Tracey A, Corby N, Dunn M, Johnson C, Wood J, Clark S, Pelan S, Griffiths G, Smith M, Glithero R, Howden P, Barker N, Stevens C, Harley J, Holt K, Panagiotidis G, Lovell J, Beasley H, Henderson C, Gordon D, Auger K, Wright D, Collins J, Raisen C, Dyer L, Leung K, Robertson L, Ambridge K, Leongamornlert D, McGuire S, Gilderthorp R, Griffiths C, Manthravadi D, Nichol S, Barker G, Whitehead S, Kay M, Brown J, Murnane C, Gray E, Humphries M, Sycamore N, Barker D, Saunders D, Wallis J, Babbage A, Hammond S, Mashreghi-Mohammadi M, Barr L, Martin S, Wray P, Ellington A, Matthews N, Ellwood M, Woodmansey R, Clark G, Cooper J, Tromans A, Grafham D, Skuce C, Pandian R, Andrews R, Harrison E, Kimberley A, Garnett J, Fosker N, Hall R, Garner P, Kelly D, Bird C, Palmer S, Gehring I, Berger A, Dooley CM, Ersan-Urun Z, Eser C, Geiger H, Geisler M, Karotki L, Kirn A, Konantz J, Konantz M, Oberlander M, Rudolph-Geiger S, Teucke M, Osoegawa K, Zhu B, Rapp A, Widaa S, Langford C, Yang F, Carter NP, Harrow J, Ning Z, Herrero J, Searle SMJ, Enright A, Geisler R, Plasterk RHA, Lee C, Westerfield M, de Jong PJ, Zon LI, Postlethwait JH, Nusslein- Volhard C, Hubbard TJP, Crollius HR, Rogers J, Stemple DL. The zebrafish reference genome sequence and its relationship to the human genome. Nature. 2013 advance online publication.\nIvey KN, Srivastava D. MicroRNAs as Regulators of Differentiation and Cell Fate Decisions. Cell Stem Cell. 2010; 7:36–41. [PubMed: 20621048]\nKim M, Tan YS, Cheng W-C, Civin CI. MiR-144 and MiR-451 Regulate Human Erythropoiesis By Targeting RAB14. Blood. 2013; 122:942–942.\nKim VN, Han J, Siomi MC. Biogenesis of small RNAs in animals. Nat. Rev. Mol. Cell Biol. 2009; 10:126–139. [PubMed: 19165215]\nKosik KS. MicroRNAs and Cellular Phenotypy. Cell. 2010; 143:21–26. [PubMed: 20887887]\nKozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic Acids Res. 2013; 42:D68–D73. [PubMed: 24275495]\nLadewig E, Okamura K, Flynt AS, Westholm JO, Lai EC. Discovery of hundreds of mirtrons in mouse and human small RNA data. Genome Res. 2012; 22:1634–1645. [PubMed: 22955976]\n\n\nNIH-PA Author Manuscript\nLi L, Meng T, Jia Z, Zhu G, Shi B. Single nucleotide polymorphism associated with nonsyndromic cleft palate influences the processing of miR-140. Am. J. Med. Genet. A. 2010; 152A:856–862. [PubMed: 20358594]\nMeiri E, Levy A, Benjamin H, Ben-David M, Cohen L, Dov A, Dromi N, Elyakim E, Yerushalmi N, Zion O, Lithwick-Yanai G, Sitbon E. Discovery of microRNAs and other small RNAs in solid tumors. Nucleic Acids Res. 2010; 38:6234–6246. [PubMed: 20483914]\nMishima Y, Abreu-Goodger C, Staton AA, Stahlhut C, Shou C, Cheng C, Gerstein M, Enright AJ, Giraldez AJ. Zebrafish miR-1 and miR-133 shape muscle gene expression and regulate sarcomeric actin organization. Genes Dev. 2009; 23:619–632. [PubMed: 19240126]\nNelson, JS. Fishes of the World. John Wiley &amp; Sons; 2006.\nNicolas FE, Hall AE, Csorba T, Turnbull C, Dalmay T. Biogenesis of Y RNA-derived small RNAs is independent of the microRNA pathway. FEBS Lett. 2012; 586:1226–1230. [PubMed: 22575660]\nOkamura K, Hagen JW, Duan H, Tyler DM, Lai EC. The Mirtron Pathway Generates microRNA- Class Regulatory RNAs in Drosophila. Cell. 2007; 130:89–100. [PubMed: 17599402]\nPase L, Layton JE, Kloosterman WP, Carradice D, Waterhouse PM, Lieschke GJ. miR-451 regulates zebrafish erythroid maturation in vivo via its target gata2. Blood. 2009; 113:1794–1804. [PubMed: 18849488]\nPatrick DM, Zhang CC, Tao Y, Yao H, Qi X, Schwartz RJ, Huang LJ-S, Olson EN. Defective erythroid differentiation in miR-451 mutant mice mediated by 14-3-3 ζ . Genes Dev. 2010; 24:1614–1619. [PubMed: 20679397]\nPersson H, Kvist A, Vallon-Christersson J, Medstrand P, Borg Å, Rovira C. The non-coding RNA of the multidrug resistance-linked vault particle encodes multiple regulatory small RNAs. Nat. Cell Biol. 2009; 11:1268–1271. [PubMed: 19749744]\nRasmussen KD, Simmini S, Abreu-Goodger C, Bartonicek N, Giacomo MD, Bilbao-Cortes D, Horos R, Lindern MV, Enright AJ, O’Carroll D. The miR-144/451 locus is required for erythroid homeostasis. J. Exp. Med. 2010; 207:1351–1358. [PubMed: 20513743]\nScott H, Howarth J, Lee YB, Wong L-F, Bantounas I, Phylactou L, Verkade P, Uney JB. MiR-3120 Is a Mirror MicroRNA That Targets Heat Shock Cognate Protein 70 and Auxilin Messenger RNAs and Regulates Clathrin Vesicle Uncoating. J. Biol. Chem. 2012; 287:14726–14733. [PubMed: 22393045]\nStadler PF, Chen JJ-L, Hackermüller J, Hoffmann S, Horn F, Khaitovich P, Kretzschmar AK, Mosig A, Prohaska SJ, Qi X, Schutt K, Ullmann K. Evolution of vault RNAs. Mol. Biol. Evol. 2009; 26:1975–1991. [PubMed: 19491402]\nTyler DM, Okamura K, Chung W-J, Hagen JW, Berezikov E, Hannon GJ, Lai EC. Functionally distinct regulatory RNAs generated by bidirectional transcription and processing of microRNA loci. Genes Dev. 2008; 22:26–36. [PubMed: 18172163]\nVerhagen APM, Pruijn GJM. Are the Ro RNP-associated Y RNAs concealing microRNAs? Y RNA- derived miRNAs may be involved in autoimmunity. BioEssays. 2011; 33:674–682. [PubMed: 21735459]\nWesterfield, M. A guide for the laboratory use of zebrafish (Danio rerio). 4th ed.. Eugene: Univ. of Oregon Press; 2000. The zebrafish book.\nWienholds E, Kloosterman WP, Miska E, Alvarez-Saavedra E, Berezikov E, Bruijn E, de, Horvitz HR, Kauppinen S, Plasterk RHA. MicroRNA Expression in Zebrafish Embryonic Development. Science. 2005; 309:310–311. [PubMed: 15919954]\nXu Z, Chen J, Li X, Ge J, Pan J, Xu X. Identification and Characterization of MicroRNAs in Channel Catfish (Ictalurus punctatus) by Using Solexa Sequencing Technology. PLoS ONE. 2013; 8:e54174. [PubMed: 23342099]\nYang J-S, Lai EC. Alternative miRNA Biogenesis Pathways and the Interpretation of Core miRNA Pathway Mutants. Mol. Cell. 2011; 43:892–903. [PubMed: 21925378]\nZuker M, Stiegler P. Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information. Nucleic Acids Res. 1981; 9:133–148. [PubMed: 6163133]\n\n\nAbbreviations list\nmiR microRNA\nsmallRNA-seq small RNA sequencing\n\n\nHighlights\n\n\nAnnotation of 56 new miRNA genes in zebrafish\nAnnotation of 248 new mature miRNA strands in zebrafish\nFirst description of mirror-miR pairs outside of Drosophila and mammals\nFirst description of 3’-tailed mirtrons outside of Drosophila, worms and mammals\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 2 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 4 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 6\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 7\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nUTR untranslated region RISC RNA-induced silencing complex Desvignes et al. Page 8 Gene. Author manuscript; available in PMC 2015 August 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 9\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nDesvignes et al. Page 10 Table 1\n\n\nZebrafish miRBase Release 20 Present study % increase Total mir genes 346 56 16 402 mature sequences 391 248 63 639 mirror-mir pairs 0 3 N/A 3 mirtrons 0 2 N/A 2\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [8.80s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"brain, heart, testis, and ovary\",\n  \"tissue_name_structured\": [\"brain\", \"heart\", \"testis\", \"ovary\"],\n  \"developmental_stage_text\": \"adult\",\n  \"developmental_stage_coarse\": \"adult\",\n  \"developmental_stage_fine\": \"adult\",\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"brain, heart, testis, and ovary\",\n  \"tissue_name_structured\": [\"brain\", \"heart\", \"testis\", \"ovary\"],\n  \"developmental_stage_text\": \"adult\",\n  \"developmental_stage_coarse\": \"adult\",\n  \"developmental_stage_fine\": \"adult\",\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 99,
      "prompt_tokens": 8142,
      "total_tokens": 8241
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_e467c31c3d"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "brain, heart, testis, and ovary",
  "tissue_name_structured": [
    "brain",
    "heart",
    "testis",
    "ovary"
  ],
  "developmental_stage_text": "adult",
  "developmental_stage_coarse": "adult",
  "developmental_stage_fine": "adult",
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [8.81s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "brain, heart, testis, and ovary",
  "tissue_name_structured": [
    "brain",
    "heart",
    "testis",
    "ovary"
  ],
  "developmental_stage_text": "adult",
  "developmental_stage_coarse": "adult",
  "developmental_stage_fine": "adult",
  "is_scRNA_data": false
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Gene-expression analysis of hair cell regeneration in the zebrafish lateral line\nLinjia Jiang a , Andres Romero-Carvajal a,b , Jeff S. Haug a , Christopher W. Seidel a , and Tatjana Piotrowski a,1\na Stowers Institute for Medical Research, Kansas City, MO 64110; and b Department of Neurobiology and Anatomy, University of Utah Medical School, Salt Lake City, UT 84132\nEdited* by Yuh Nung Jan, Howard Hughes Medical Institute, University of California, San Francisco, CA, and approved March 4, 2014 (received for review February 16, 2014)\nDeafness caused by the terminal loss of inner ear hair cells is one of the most common sensory diseases. However, nonmammalian animals (e.g., birds, amphibians, and fish) regenerate damaged hair cells. To understand better the reasons underpinning such disparities in regeneration among vertebrates, we set out to define at high resolution the changes in gene expression associ- ated with the regeneration of hair cells in the zebrafish lateral line. We performed RNA-Seq analyses on regenerating support cells purified by FACS. The resulting expression data were subjected to pathway enrichment analyses, and the differentially expressed genes were validated in vivo via whole-mount in situ hybrid- izations. We discovered that cell cycle regulators are expressed hours before the activation of Wnt/ β -catenin signaling following hair cell death. We propose that Wnt/ β -catenin signaling is not involved in regulating the onset of proliferation but governs pro- liferation at later stages of regeneration. In addition, and in marked contrast to mammals, our data clearly indicate that the Notch pathway is significantly down-regulated shortly after in- jury, thus uncovering a key difference between the zebrafish and mammalian responses to hair cell injury. Taken together, our findings lay the foundation for identifying differences in sig- naling pathway regulation that could be exploited as potential therapeutic targets to promote either sensory epithelium or hair cell regeneration in mammals.\nsignaling pathway analysis | RNA sequencing | neuromast | Jak/Stat3 | cdkn1b\nH earing disorders are the most common sensory disabilities in humans and can be either inherited or acquired. De- generation and death of sensory hair cells in the inner ear is the underlying cause of age-related, noise-, antibiotic-, or chemo- therapy drug-induced hearing loss ( www.nidcd.nih.gov ). Such loss is permanent, because mammalian inner ear hair cells do not regenerate. The inability of mammals to restore sensory hair cells after injury is intriguing given that ( i ) mammalian vestibular support cells exhibit low levels of proliferation after injury; ( ii ) cochlear support cells of newborn mice are able to proliferate subsequent to hair cell death in cell culture (1); and ( iii ) other vertebrates such as birds, amphibians, and fish replace their sensory hair cells continuously and, importantly, are capable of regenerating hair cells after damage (2 – 10).\nWork in birds has shed light on the mechanisms regulating sensory hair cell regeneration in vertebrates (2, 4, 11 – 18). For instance, during sensory hair cell regeneration in the chick, the majority of hair cells arise by proliferation of neighboring sup- port cells (19). Given that mammalian support cells do not re- spond to hair cell death by proliferating, molecular studies in chick have focused on identifying the mitogenic signals as well as the target genes that induce re-entering of the cell cycle in support cells (11, 20 – 22). For example, the transcription factors Atoh1 and Sox2 and the Wnt/ β -catenin and Notch pathways have been shown to play crucial roles (23), and gene-expression analyses using microarrays have identified hundreds of genes involved in chick hair cell regeneration (18, 24).\nThe Wnt/ β -catenin pathway is used repeatedly during hair cell development, and recent studies have emphasized the importance of Wnt/ β -catenin signaling in modulating the proliferation of sup- port cells. For example, activated β -catenin causes proliferation in dissociated chicken utricular epithelia and ectopic expression of hair cell differentiation in the chicken basilar papilla (17, 25). Likewise, during mouse cochlear development Wnt/ β -catenin sig- naling activates proliferation of support cells and is required sub- sequently for their differentiation into hair cells (3, 26 – 29). Perturbing the signaling pathways known to be activated during chick inner ear regeneration therefore may provide an opportunity to modulate regenerative capacities in the mam- malian inner ear. For instance, it has been shown recently that down-regulation of Notch signaling in adult mice after injury induces the transdifferentiation of support cells into hair cells and can partially restore hearing (30). However, this manipula- tion causes the loss of support cells, and the implications of this loss for long-term hearing restoration are unknown.\nSignificant progress has been made in the study of sensory hair regeneration. However, in part because of the complexity of spatial and temporal gene interactions and signaling dynamics, our knowl- edge still is too incomplete to enable us to promote the generation of a functional epithelium after injury in mammals. The zebrafish lateral line provides an experimentally accessible system to define the complex signaling events triggered by injury and regeneration, because these cells can be killed acutely and then regenerate rap- idly (31). These characteristics allow the detection of changes in gene expression at defined time points following hair cell death.\nSignificance\nDeafness is caused largely by the death of sensory hair cells in the inner ear. In contrast to nonmammalian vertebrates, hu- man hair cells do not regenerate. Understanding the mecha- nisms that regulate hair cell regeneration in zebrafish may shed light on the factors that prevent hair cell regeneration in mammals. RNA-Seq analysis of regenerating sensory organs uncovered dynamic changes in the expression of signaling pathways during zebrafish hair cell regeneration. Unexpect- edly, the Wnt/ β -catenin, Notch, and Fgf pathways are down- regulated following hair cell death, whereas the Jak1/Stat3 pathway and cell cycle are activated. We propose that mim- icking the zebrafish activation status of a combination of path- ways at the correct time points in mammals may improve the chances of triggering regeneration of functional hair cells.\nAuthor contributions: L.J. and T.P. designed research; L.J., A.R.-C., and J.S.H. performed research; L.J., J.S.H., C.W.S., and T.P. analyzed data; and L.J. and T.P. wrote the paper.\nThe authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nData deposition: The data reported in this paper have been deposited in the Gene Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE56176 ). 1 To whom correspondence should be addressed. E-mail: pio@stowers.org.\nZebrafish and the majority of all other aquatic vertebrates possess hair cells not only in their vestibular system in the ear but also in their skin. These hair cells are part of the sensory lateral line system. The lateral line system senses water motion and initiates the appropriate behavioral response for capturing prey, avoiding predators, and schooling. The morphology and function of sensory hair cells are evolutionarily conserved from fish to mammals (32, 33). Lateral line hair cells are located in the center of a mechanosensory organ known as the neuromast and are surrounded by inner support cells and an outer ring of mantle cells. Importantly, despite the unusual location of the hair cells on the trunk, lateral line and ear hair cells develop and differ- entiate by similar developmental mechanisms. For example, mutations in genes causing deafness in humans also disrupt hair cell function in the zebrafish lateral line and vestibular system (34). Because of its genetic tractability and the accessibility of sensory organs to experimental manipulation and visualization, the zebrafish recently has been recognized as an excellent model for discovering and functionally characterizing genes crucial for hair cell development and regeneration (16, 19, 35, 36).\nIn zebrafish hair cell regeneration occurs via the proliferation and differentiation of inner support cells (5, 6, 9, 37, 38), but detailed lineage analyses have not yet been performed, and the stem cell population has not been identified. Mantle cells are good stem cell candidates, because they migrate to form neu- romasts on regenerating tail tips and also form postembryonic neuromasts by a process called “ stitching. ” However, after chemical ablation of hair cells by antibiotic treatment, mantle cells proliferate much less frequently than inner support cells (9). Possibly, mantle cells replenish inner support cells that then differentiate into hair cells. Mantle cells have not been shown to differentiate directly into hair cells (37, 38). Therefore, both mantle and inner support cells seem to be involved in reestablishing proper neuromast morphology after hair cell death.\nTo identify the activation status of signaling pathways during the first few hours of regeneration, we performed RNA-Seq gene expression analyses of zebrafish mantle and inner support cells isolated by FACS. We discovered that the Notch pathway is transiently down-regulated immediately after hair cell death; to our knowledge, this down-regulation has not been previously reported in other studies of regenerating species. Our studies also demonstrate that Wnt/ β -catenin signaling is not among the earliest pathways to respond to hair cell death but instead is up- regulated relatively late during the regenerative response. These results suggest that Wnt/ β -catenin signaling does not trigger proliferation by down-regulating cyclin-dependent kinase inhib- itors immediately after hair cell death but rather regulates pro- liferation at later stages of regeneration. Our findings have defined the temporal regulation of key signaling pathways asso- ciated with the early stages of hair cell regeneration and suggest that the successful initiation of mammalian hair cell regeneration, either in vivo or in vitro, might require precise temporal manip- ulation of these pathways.\n\n\nResults\nIsolation of Mantle and Inner Support Cells by FACS. To identify changes in gene expression in mantle and inner support cells after hair cell death, we isolated GFP + cells from regenerating and control Tg ( sqET20 ) transgenic zebrafish via FACS (Fig. 1 A and B ) (39). Tg ( sqET20 ) larvae express GFP strongly in mantle cells and to a lesser degree in inner support cells (Fig. 1 B ). Hair cells are GFP − and therefore are excluded from the GFP + population. We isolated GFP − and GFP + cells from Tg ( sqET20 ) control larvae at 5 days postfertilization (dpf) and from neo- mycin-treated larvae at 1, 3, and 5 h after neomycin treatment. Larvae were dissociated into a single-cell suspension in a trypsin solution and stained with 7-aminoactinomycin D (7-AAD) and Hoechst 33342 to exclude dead cells and cell debris (Fig. 1 C ). Live GFP + cells (7-AAD − , Hoechst + , GFP + cells) and control GFP − cells (7-AAD − , Hoechst + , GFP − cells) were sorted by FACS (Fig. 1 D ). The sorting yielded highly GFP + -enriched cell populations, as demonstrated by a postsort reanalysis of the collected cells (Fig. 1 E and F ). GFP + -sorted cells showed 94.4 ± 1.3% enrichment ( n = 7). To confirm that GFP + cells are lateral line cells, we performed immunostaining with an antibody against the support cell marker Sox2 that labels inner support cells as well as some mantle cells (Fig. 1 G and Fig. S1 ) (40). Counts of Sox2 + FACS-purified cells revealed that 95% of GFP + cells but only 6.6% of GFP − cells were positive for Sox2 (Fig. 1 H ). Ad- ditionally, we performed RT-PCR for genes highly expressed in neuromast cell types and with limited expression in other tissues. RT-PCR for gfp , eya1 , klf4b , and cldnb showed that these genes are expressed at significantly higher levels in GFP + than in GFP − cells ( Fig. S2 A – D ). In contrast, genes expressed in tissues outside the lateral line system, such as rhodopsin ( rho , eye), vil- lin1 ( vil1 , gut), and myod1 (muscle), are expressed at much lower levels in GFP + cells ( Fig. S2 E – G ). In Tg ( sqET20 ) larvae, mantle B A G K Sox2    cells (%) 0 20 40 60 80 100 H lateral line genes neomycin induced changes J Tg(sqET20) mantle cell inner support cell GFP GFP GFP 1h Sox2 GFP DAPI GFP GFP GFP control neo control neo control 30min 1h 3h 5h 30min 1h 3h 5h GFP 3h GFP 5h GFP 1h GFP 3h GFP 5h GFP changes in mantle and support cells C D 97.8% FSC FSC 0 1000 2000 3000 4000 FSC 100 101 2 103 10 100 101 10 2 103 104 Hoechst GFP GFP 94.3% live cells 10 4 E F GFP GFP 100 101 2 103 10 10 4 100 101 2 103 10 10 4 0 1000 2000 3000 4000 0 1000 2000 3000 4000 100 101 2 103 10 10 4 I 7-AAD GFP GFP GFP GFP\nFig. 1. Purification of mantle and inner support cells from Tg ( sqET20 ). ( A ) In 5-dpf Tg ( sqET20 ) larvae GFP is expressed in neuromasts (arrows) and in the interneuromast cells that connect adjacent neuromasts. ( B ) Magnification of a neuromast shows that mantle cells are strongly labeled by GFP, but inner support cells are weakly labeled. ( C and D ) Dissociated cells of Tg ( sqET20 ) larvae at 5 dpf were FACS sorted using a two-gate sorting strategy. Live cells were gated by 7-AAD − and Hoechst + ( C ) and subsequently were gated by forward scatter (FSC) and GFP intensity to isolate GFP − and GFP + cells ( D ). ( E and F ) A postsorting experiment shows that GFP − cells are 97.8% pure ( E ) and GFP + cells are 94.3% pure ( F ). ( G ) Sorted GFP − and GFP + cells were stained with Sox2 antibody and DAPI. ( Upper ) A GFP − cell is Sox2 − . ( Lower ) A GFP + cell expresses Sox2. (Scale bars: 5 μ m.) ( H ) Quantification of Sox2 + cells in each population as mean ± SD; * P &lt; 0.01 by t test. ( I ) Regeneration- induced mantle and inner support cell genes were identified at each time point by comparing GFP + neomycin-treated cells with GFP + untreated samples. ( J ) Neomycin-induced genes were identified by comparing GFP − neomycin-treated cells at each time point with GFP − untreated samples. Neomycin-induced genes subsequently were subtracted from the gene lists obtained with GFP + cells to select for regeneration-specific changes in gene expression in inner support and mantle cells. ( K ) Comparisons between GFP + and GFP − cells of untreated larvae generated a list of genes enriched in lateral line cells.\ncells express GFP most strongly, but inner support cells respond more robustly to hair cell death by proliferating (5, 6). Because Tg ( sqET20 ) larvae show weaker GFP expression in inner support cells, we were concerned that our cell sorts would exclude inner support cells. However, RT-PCR with inner support cell markers such as klf4b showed that inner support cells were included in the GFP + cell populations ( Fig. S2 C ). Combined, these studies dem- onstrated that our FACS protocol yielded a highly enriched pool of lateral line cells containing mantle and inner support cells but not hair cells.\nTo investigate the temporal activation or repression of in- dividual genes and signaling pathways in GFP + mantle and inner support cells during the first few hours of regeneration, we ex- amined GFP + and GFP − cells at 1, 3, and 5 h after a 30-min treatment with neomycin (Fig. 1 I ). Sorts for each time point were performed in triplicate and subjected to RNA-Seq analysis. Bacte- ricidal antibiotics cause oxidative stress in mammalian cells, leading us to speculate that sublethal concentrations of neomycin may cause changes in gene expression unrelated to hair cell death (41). To control for this nonspecific effect, we examined gene expression in GFP − cells in the presence or absence of neomycin (Fig. 1 J ). Al- though this population includes all larval nonlateral cells, it allows us to control for a nonspecific neomycin effect when comparing GFP + cells from treated and control larvae.\nAnalysis of the RNA-Seq Results. Sequencing libraries were pre- pared from mRNAs of collected cells and were quality checked on a Bioanalyzer. We performed 50-bp single-end RNA-Seq using the Illumina platform. To test the enrichment of lateral line genes further, we examined the fragments per kilobase of transcript per million mapped reads (FPKM) values for known lateral line genes and for genes expressed in other tissues ( Fig. S3 ). The results confirmed the RT-PCR analysis described above. Pan lateral line, as well as inner support cell, markers show much higher FPKM values in FACS-sorted GFP + cells than in GFP − cells. As suggested by the RT-PCR results, the FPKM values for the inner support cell markers pea3 and klf4b are significantly higher in the GFP + than in the GFP − cells, confirming that inner support cells were included in our FACS sorts. This enrichment of lateral line genes is supported by the information in Dataset S1 , in which neuromast-specific genes were identified by comparing the expression profiles of untreated GFP + and GFP − cells. In addition to known lateral line genes, the resulting lists of differentially expressed genes provide a valuable resource of as yet uncharacterized genes that potentially could play important roles in hair cell development and/or re- generation (Fig. 1 K and Dataset S1 ). The number of genes enriched in GFP + cells relative to GFP − controls at an adjusted p-value ≤ 0.05 is 1,670 ( SI Materials and Methods and Dataset S1 ). This dataset also contains many of the genes reported in the dataset of Steiner et al. (42), who identified mantle cell-specific genes using a different transgenic line (see below).\nGene Identification for Each Time Point. To identify genes from transcripts specifically enriched or depleted in GFP + mantle and inner support cells after hair cell death, we created several comparisons between the RNA-Seq datasets. Ratios of gene expression were created between the neomycin-treated GFP + cells at 1, 3, and 5 h and the nontreated GFP + cells at 1 h to identify genes responding to hair cell death (Fig. 1 I ). To control for a nonspecific neomycin response, a similar set of ratios was created between the neomycin-treated GFP − cells at each time point and the untreated GFP − cells at 1 h (Fig. 1 J ). The neomycin- induced genes were subtracted from all gene lists generated with GFP + cells. We then used ratios and P values between datasets to select genes of interest at any given time point ( SI Materials and Methods and Dataset S2 ). Genes identified by these criteria are marked with a numeric flag with positive numbers indicating up-\nregulated and negative numbers indicating down-regulated genes. The numeric value indicates the time point at which a gene is up- or down-regulated ( Dataset S2 , flagged column). A principal component analysis of the biological replicates of GFP − and GFP + cell populations at the three different time points demonstrates that GFP + cells are very different from GFP − cells. In addition, cell sorts performed in triplicate for each time point are highly re- producible ( Fig. S4 ). To define a set of the top 100 up- and down-regulated genes to use as candidates for validation, we ranked 193 up-regulated and 200 down-regulated significant (flagged) genes from the 1-h dataset as a function of the ratio and general abundance ( SI Materials and Methods and Table S1 ). We validated the RNA- Seq results by performing in situ hybridizations with 28 up-regulated and 21 down-regulated genes selected from the top 100 gene list in larvae 1 h after neomycin treatment ( Table S2 ). All 28 up- regulated genes are expressed in the lateral line, and 20 of these genes show up-regulation by in situ hybridization after neomycin treatment. Of the 21 down-regulated genes, 19 are expressed in the lateral line, and 12 genes are detectably down-regulated by in situ hybridization (Fig. 2 and Table S2 ). These experiments de- monstrated that the FACS sorting followed by RNA-Seq analysis produced high-quality results that enable us to study hair cell regeneration in zebrafish in detail.\nPathway Analysis at 1, 3, and 5 h After Hair Cell Death. The Wnt/ β -catenin, Notch, Jak/Stat, Bmp, and Fgf pathways are important for hair cell development as well as for the regeneration of hair cells in fish and chick epithelia (15, 23, 43). Our understanding of which pathways act upstream and how these pathways affect each other is still very limited. We performed a temporal ex- pression analysis at 1, 3, and 5 h after hair cell death to de- termine when particular pathways are activated or inhibited; this temporal information provides clues into how they might interact with each other. We also aimed to identify other signaling path- ways and record any dynamic changes in their behavior that could indicate how they integrate with the Notch and Wnt/ β -catenin signaling pathways. To identify enriched pathways in our datasets, we used the signaling pathway impact analysis (SPIA) package in\nccdc48\nR (44). This software tool not only takes into account how many genes belong to a pathway but also focuses on the importance of a gene by analyzing the perturbation of the pathway if the gene expression changes. Therefore, even pathways that are repre- sented by only a small number of genes can be classified as en- riched if these genes are important regulators of the pathway. For the SPIA analysis we included all flagged genes (flagged column) listed in Dataset S2 . For each time point the up- and down-regulated genes were combined and analyzed against a panel of pathways from zebrafish defined in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (45). The 1-h postneomycin treatment time point contained 509 genes from the National Center for Bio- technology Information Entrez database. The 3-h postneomycin treatment time point contained 401 Entrez genes, and the 5-h postneomycin treatment time point contained 418 genes.\nThe results of the SPIA analysis are shown in Table S3 . Au- tomated pathway annotation is powerful but subject to caveats, because genes can be annotated to function in several pathways, some of which might not be active in the lateral line. In addition, in the KEGG database, the Wnt pathway encompasses the ca- nonical Wnt/ β -catenin, the PCP, and the Wnt- regulated calcium pathways, potentially providing erroneous status assignment of the Wnt/ β -catenin pathway. We therefore tested the SPIA ana- lysis by in situ hybridization experiments with pathway members at various time points of regeneration (see below and Figs. 3 – 5). For improved visualization we generated heat maps of log2 ratio values of KEGG pathway members. We show only the genes that were flagged in our analysis and reported to be expressed in the lateral line by the zebrafish information network ( www.zfin.org ) or prior reports (46, 47).\nThe Wnt/ β -Catenin Pathway Is Not Activated During Early Stages of Lateral Line Hair Cell Regeneration. The analysis of hair cell re- generation in a Tg ( sqEt20;sqEt4 ) double-transgenic larva allows us to draw comparisons between gene expression and cellular changes at particular time points (Fig. 3 A ) (39). At 1 h after neomycin treatment almost all hair cells have been eliminated. Two hair cells remain that were too immature to be killed by neomycin. Proliferation of inner support cells begins within the first 2 h of regeneration (37). At 5 h two newly differentiating hair cells start to express GFP (Fig. 3 A ; arrows), and by 12 h after neomycin treatment six hair cells are present.\nWnt/ β -catenin signaling is required for proliferation in de- veloping and regenerating neuromasts (26, 48). Unexpectedly, the SPIA pathway analysis suggested that the Wnt/ β -catenin pathway is inactive at 1 h after hair cell death ( Table S3 ). To investigate this finding in more detail, we performed in situ hy- bridization experiments with Wnt/ β -catenin pathway genes in untreated and treated larvae at various time points after hair cell death. Our analysis showed that Wnt/ β -catenin signaling is not active in 5-dpf untreated control neuromasts. The ligand wnt10a is not expressed in 5-dpf control neuromasts, and we failed to detect any expression of dgfp in the Tg ( Tcf/Lef-miniP:dGFP ) Wnt/ β -catenin reporter line (Fig. 3 B ) (49). Even though Wnt/ β -catenin signaling is inactive, the β -catenin cotranscription fac- tor tcf7l2 ( tcf4 ) is expressed. Likewise, the Wnt receptors fzd7b and 8a are present, although whether they act as receptors for canonical or noncanonical Wnt signaling in neuromasts is un- known (Fig. 3 B ) (50, 51). At 1 h after neomycin treatment tcf7l2 ( tcf4 ), frz7b , and fzd8a are drastically down-regulated, and, along with wnt10a , their tcf4 wnt10a dgfp control 1h 3h 5h 10h control 1h 5h 12h 18h fzd8a fzd7b lef1 fzd7b fzd8a wif1 dkk1b tcf7l1a tcf7l2 fzd9b ctbp2 axin2 lef1 wnt8a csnk1a1 jun sfrp1a wnt10a cul1a siah1 gro2 sox17 tbl1x fbxw11b gsk3b csnk1a1 1h 3h 5h 1h 3h 5h fzd6 dkk2 control 1h 3h 5h 10h 12h 18h 24h GFP GFP Wnt 2 0 -2 A B C Fig. 3. The Wnt/ β -catenin pathway is inactive during the early stages of hair cell regeneration. ( A ) Still images of a Tg ( sqET20;sqET4 ) larval neuromast in the process of neomycin-induced regeneration. All hair cells, except for two immature hair cells, were killed by neomycin by 1 h after neomycin treatment. Two newly formed hair cells (arrows) start to express GFP at 5 h, and other pairs of hair cells (arrows) appear at 12 and 18 h. ( B ) In situ hybridization of Wnt/ β -catenin pathway genes in 5-dpf untreated (control) larvae and in larvae at different time points after neomycin treatment. Expression of wnt10a is in- creased at 3 and 5 h. Expression of tcf7l2 ( tcf4 ), fzd7b , and fzd8a is largely decreased at 1 h after neomycin. The Wnt/ β -catenin reporter line Tg ( Tcf/Lef-miniP: dGFP ) demonstrates activation starting at 12 h as shown by dgfp expression, but lef1 is not induced in regenerating neuromasts. ( C ) Heat map of selected Wnt/ β -catenin pathway genes based on RNA-Seq results. Log2 ratios of GFP − and GFP + cells at 1, 3, and 5 h after neomycin treatment are color coded (red: up- regulation, blue: down-regulation). Genes that are modulated in GFP − cells are neomycin-induced genes.\nexpression in the Wnt/ β -catenin Tg ( Tcf/Lef-miniP:dGFP ) reporter line remains low or undetectable. At 3 h after neomycin treatment wnt10a , tcf7l2 ( tcf4 ) and fzd7b , and fzd8a are up-regulated, leading to Wnt/ β -catenin pathway activation at 12 h as demonstrated by dgfp expression in the Tg ( Tcf/Lef-miniP:dGFP ) reporter line (Fig. 3 B ). The activation status of other Wnt/ β -catenin pathway genes at the different time points is shown in Fig. 3 C . Genes that change expression in the GFP − cells represent neomycin-induced genes. Our analyses reveal that Wnt/ β -catenin signaling is a late- responding pathway and does not play a role in the initiation of regeneration.\nThe Notch Pathway Is Inhibited 1 h After Neomycin Treatment. The Notch pathway is of particular interest because experimental down-regulation of Notch induces transdifferentiation of mam- malian support cells into hair cells (30, 52 – 57). In contrast to previous reports in zebrafish and chicken (9, 13), our SPIA pathway analysis suggested that the Notch pathway is inhibited immediately after hair cell death because of simultaneous up- regulation of numb (Fig. 4 A and B ). Numb inhibits Notch ac- tivity by regulating postendocytic sorting events and degradation of the Notch receptor and thereby regulates neural stem cell homeostasis (58, 59). However, because Numb is a multifunc- tional membrane-associated protein with additional functions\nindependent of the regulation of Notch signaling, we tested if the Notch pathway is indeed down-regulated by in situ hybridization of Notch pathway members. Our analysis of the Notch target genes her4.1 and notch3 confirms that Notch signaling is attenuated immediately 1 h after neomycin treatment and is accompanied\nFig. 5. The Jak1/Stat3 pathway is activated during the early stages of hair cell regeneration. ( A ) In situ hybridization shows that stat3 is increased at 1 and 3 h after neomycin treatment. ( B ) bmp5 is up-regulated at 1 h after neomycin treatment in situ. ( C ) Table depicting the status of signaling path- ways at different time points after hair cell death. All comparisons (no change, inhibition, or activation) were evaluated relative to untreated control samples. The Wnt, Notch, and Fgf pathways are inhibited immediately after neomycin treatment, whereas the cell cycle and Jak1/Stat3 pathways are activated during the early stages of regeneration. The Wnt and Notch pathways are reactivated at 12 h and 5 h after neomycin treatment, respectively.\nby down-regulation of sox2 (Fig. 4 A and B ) and up-regulation of atoh1a (Fig. 4 A and B ). The atoh1a target deltaD ( dld ) is up- regulated between 3 – 5 h after neomycin treatment, reflecting the appearance of hair cell precursors (Fig. 4 A and B ) (60). Notch signaling must be reinitiated for proper fate specification into hair and support cells and proper organ size control (9, 60). Accordingly, between 3 and 5 h after neomycin treatment (Fig. 4 A and B ), we observed renewed activation of the Notch target genes her4.1 , notch3 , and sox2 that likely is caused by the up- regulation of dld (60).\nWe postulate that down-regulation of Notch signaling is a characteristic event during hair cell regeneration in zebrafish. In contrast to zebrafish, the Notch target genes are decreased only modestly, and Notch signaling is still active, in hair cell-depleted neonatal mouse cochleae and 3- to 4-wk-old mouse utricles (55, 56, 61). Therefore we hypothesize that the failure of transient Notch down-regulation in mammals could be one of the reasons that mammalian hair cells do not regenerate spontaneously after injury.\nFgf Signaling Is Down-Regulated Immediately After Hair Cell Death. The Fgf pathway initiates dld and subsequently Notch signaling in the mouse cochlea, chicken inner ear, and zebrafish lateral line neuromasts (23, 60). Both pathways are active in 5-dpf un- treated neuromasts (Fig. 4 A and C and Fig. S5 D ). Identically to Notch pathway members, fgf3 and its transcriptional target pea3 are turned off 1 h after neomycin treatment. However, the Notch target her4.1 is reactivated at 3 h, whereas pea3 is not transcribed until 5 h (Fig. 4 A and C ). Therefore, it is possible that Notch signaling does not depend on Fgf signaling during regeneration; however, this hypothesis needs to be tested. The rapid down- regulation of Fgf signaling is in accordance with findings in chicken sensory epithelia, in which Fgfr3 is highly expressed in support cells but is down-regulated rapidly after hair cell death (62). Additionally, in the chicken down-regulation of Fgf sig- naling is required for regeneration to occur, because the addition of Fgf inhibits support cell division after damage (63).\nThe Cell Cycle Exit Genes cdkn1b , cdkn1c , and rbl2 Are Down- Regulated at 1 h After Hair Cell Death. The failure to down-regu- late cell cycle exit regulators is one of the reasons mammalian support cells do not proliferate in response to hair cell death (10, 12, 64, 65). Determining the earliest changes in the expression of cell cycle regulators and mapping correlating changes in signaling pathway activations will aid in identifying candidate pathways in- volved in cell cycle reentry. We detected the earliest changes in cell cycle genes at 1 h after neomycin treatment. The cyclin- dependent kinase (cdk) inhibitors cdkn1b ( p27/kip1 ) and cdkn1c ( p57/kip2 ), as well as the pocket domain-containing rbl2 , are down-regulated 1 h after neomycin treatment and are not up- regulated again until 5 – 10 h after neomycin treatment (Fig. 4 D and E ). Time-lapse analyses revealed that support cells enter the cell cycle 1 – 5 h after neomycin treatment (37), likely as a con- sequence of the down-regulation of the cdk inhibitors at 1 h after neomycin treatment. Because we did not observe any signs of Wnt/ β -catenin signaling activation, we conclude that Wnt/ β -catenin signaling is not required for the reentry of support cells into the cell cycle immediately after hair cell death. In contrast, down- regulation of Notch and Fgf signaling correlates with the down- regulation of cdk inhibitors. It is possible that active Notch or Fgf signaling induces cdk inhibitors, causing cells to exit the cell cycle or become quiescent (66). However, manipulation of Notch signal- ing does not affect the cell cycle in chick or murine inner ears (13, 30), suggesting that signaling pathways other than Wnt/ β -catenin and Notch are involved in triggering or inhibiting proliferation.\nThe Jak1/Stat3 Pathway Is Up-Regulated Immediately After Neomycin Treatment. The Jak1/Stat3 pathway regulates a plethora of cel- lular processes, such as cell proliferation, cell differentiation, and\ncell survival (67, 68). The Jak1/Stat3 pathway is up-regulated during zebrafish inner ear regeneration and is required for proliferation of cardiomyocytes during zebrafish heart regeneration (36, 69). Like- wise, in zebrafish lateral line regeneration stat3 is up-regulated immediately, 1 h after neomycin treatment, as evidenced by in situ hybridization (Fig. 5 A and Fig. S5 A ). Other Jak1/Stat3 pathway members that are highly up-regulated are jak1 , interleukin 6 signal transducer ( il6st ), the stat inhibitor and target gene suppressor of cy- tokine signaling ( socs3b ), and the ligand interleukin 11b ( il11b ) (Fig. 2 A and Fig. S5 A ). In zebrafish heart regeneration, the proliferation of cardiomyocytes is regulated by the Jak1/Stat3 downstream target relaxin 3a ( rln3a ) . rln3a also is highly up-regulated in GFP + cells 3 h after neomycin treatment ( Fig. S5 A ) (69). Possibly, Jak1/Stat3 sig- naling regulates proliferation in support cells in the first few hours after hair cell death, whereas Wnt/ β -catenin signaling is required for proliferation during later stages of regeneration.\nAnother signaling pathway that is modulated during zebrafish hair cell regeneration and that might interact with the pathways described above is the Bmp pathway. Bmp signaling influences hair cell differentiation during chicken and mouse inner ear development (70 – 73). During zebrafish hair cell regeneration, bmp5 is up-regulated between 1 and 5 h after hair cell death (Fig. 5 B and Fig. S5 B ), whereas the BMP inhibitor noggin 3 ( nog3 ) is not down-regulated until 5 h after hair cell death ( Fig. S5 B ). Because ligand expression by itself is not a reliable indicator for signaling pathway activation, future studies will need to de- termine how the activation status of Bmp signaling is affected by bmp5 and nog3 .\nOther pathways that respond to zebrafish lateral line hair cell death are the Mapk ( Fig. S5 C and D ), TNF- α , ( Fig. S5 E ), insulin ( Fig. S5 F ), and apoptosis signaling ( Fig. S5 G ) pathways and nitric oxide (NO) and reactive oxygen species (ROS) ( Fig. S5 H ), some of which increase proliferation in undamaged chicken ves- tibular support cells (74). For example, we observed up-regulation of nox1 , noxo1a , and noxa1 ( Fig. S5 H and Dataset S2 ). NO is a radical that regulates a wide variety of physiological processes and is important during liver regeneration (75).\nWe summarized the activation status of some of the major signaling pathways based on the in situ hybridization patterns of pathway members in Fig. 5 C . The Wnt/ β -catenin pathway is not active in homeostatic or newly regenerating neuromasts, and the Notch and Fgf pathways are down-regulated during the first few hours of regeneration. The Wnt/ β -catenin and Notch pathways seem to play a role only during later stages of regeneration. The cell cycle is activated immediately, possibly via cytokine stimu- lation of the Jak1/Stat3 pathway.\nIndependent Microarray Studies Support the Differential Expression of Genes Not Associated with Key Signaling Pathways. An in- dependent expression analysis performed by Steiner et al. and reported in this issue (42), used a transgenic line with an alkaline phosphatase enhancer [ Tg ( -4.7alp:mCherry )] instead of the Tg ( sqET20 ) line we used to isolate mantle cells by FACS anal- ysis. We compared our respective datasets to determine how similar the data are. We first analyzed the overlap of genes enriched in untreated mantle/lateral line cells in both studies. The obtained results are a clear indicator of how similar the sorted cell populations in the two studies are. We chose the top 1,000 up- regulated genes from our Dataset S1 by p-value and then ex- amined how many of these genes also are among the top 1,000 genes identified by Steiner et al. (dataset S1 in ref. 42). If the datasets were independent and unrelated, we would expect 53 genes to overlap by chance. We identified 201 common genes with a p-value of 3.9e-107, demonstrating that our two datasets are significantly related. All six mantle cell-specific genes iden- tified by Steiner et al. (42) also show enrichment in our untreated GFP + cell population ( Table S4 A , blue rows). Ten of the 26 top up- and down-regulated genes that we validated by in situ hybridization\nafter hair cell loss ( Table S2 ) and for which we found a match in the Steiner dataset also show significant changes in the Steiner microarray dataset ( Table S4 B , blue rows) (42). Several of the genes that were not present in the Steiner dataset are expressed more highly by in situ hybridization in inner support cells than in mantle cells, reflecting the nature of the transgenic lines used and the type of cells purified. Nine of the 14 genes that Steiner et al. identified by microarray analysis, by RT-PCR, or by in situ hybridization (42) also showed the same trend in expression changes in our RNA-Seq analysis ( Table S4 C , blue rows).\nThe overlap between a subset of our data and that of Steiner et al. (42), combined with the identification and corresponding validation of genes in our study, strongly indicates that the two lines of investigation are complementary, with most of the tar- gets identified by Steiner et al. (42) found in the mantle cells and the majority of genes identified in our study expressed primarily in the inner support cells. As such, our respective studies both validate each other and help resolve the cell-type specificity of the revealed differential gene expressions.\n\n\nDiscussion\nSeveral studies have aimed to induce mammalian hair cell re- generation by inactivating or activating candidate genes and pathways. Knockdown of the cell cycle exit regulators Cdkn1b ( p27/Kip1 ), p19/Ink4d , and pRb led to the initiation of cell cycle reentry and the differentiation of hair cells in injured epithelia in mice (20, 21, 64, 76, 77). Unfortunately, these ectopic hair cells cannot be maintained and eventually apoptose. Because direct manipulation of cell cycle regulators has not been successful in reconstituting a fully functional sensory epithelium in mammals, it is important to identify the mitogens that activate cell cycle entry in regenerating organisms. It is plausible that manipu- lations of upstream signaling events are a more promising ave- nue to regenerate mammalian hair cells.\nMore recently, in mice the down-regulation of Notch signaling caused transdifferentiation of support cells into functional hair cells without inducing proliferation (30). However, the number of regenerated hair cells was modest, and it is unclear if the accompanied depletion of support cells affects the function of the partially regenerated sensory epithelium in the long term. Although these efforts have not resulted in the regeneration of a fully functional, morphologically normal epithelium, they clearly demonstrate that the mammalian cells of the inner ear can be manipulated and coaxed away from their terminally differenti- ated state.\nKnowledge gained from studying the development of hair cells has demonstrated that the underlying molecular mechanisms are complex, with a multitude of interacting signaling pathways and feedback loops that ensure that an unspecified progenitor cell transitions to a prosensory cell that ultimately differentiates into a functional sensory cell. We also have learned that signaling pathways are activated and inhibited repeatedly as hair cell progenitors are specified and differentiate, further complicating their experimental manipulation. Hence, a global up- or down- regulation of a particular pathway is unlikely to lead to fully functional sensory epithelia.\nWe hypothesize that the regeneration of a complete sensory epithelium will require the manipulation of several pathways in the correct order. However, the model systems currently used to study the regeneration of sensory epithelia present unique ob- stacles. For example, in chicken, antibiotic-induced hair cell death occurs over a 24-h period, and the proliferation of support cells and differentiation of hair cells occurs 1 – 4 wk after injury (19). This long time frame makes it difficult to determine the precise time point when pathways are turned on and off, because support cells will be in different stages of regeneration. Likewise, a previous expression study of the regenerating zebrafish ear was\nperformed 24 h after the larvae were exposed to noise that does not cause death simultaneously in all hair cells (36). Further adding to the complexity, genes are used repeatedly during hair cell development and can turn from an activator into a repressor (23). We have selected the zebrafish lateral line tissue to eluci- date the gene-expression patterns triggered by hair cell death. The zebrafish lateral line is particularly well suited for these analyses, because all hair cells can be killed acutely within 30 min, and regeneration occurs within 24 h (Fig. 3 A ), allowing us to determine the earliest signals and responses in support cells after hair cell death.\nWnt/ β -Catenin Signaling Is Not Detected Early in Lateral Line Hair Cell Regeneration. Our analyses focused on pathways involved in hair cell development as they might be reused during regeneration (15, 23, 43, 78, 79). For example, Wnt/ β -catenin signaling is crucial for the development of many organs across the animal kingdom (80). During sensory development, Wnt/ β -catenin sig- naling governs the proliferation of unspecified progenitor cells, renders them competent to acquire a neural cell fate, and sub- sequently is involved in the differentiation of these postmitotic prosensory cells (3, 23, 81). Wnt/ β -catenin signaling also is in- volved in regeneration. Wnt/ β -catenin signaling controls stem cell maintenance, quiescence, and proliferation during mam- malian epithelial hair cell, zebrafish fin, mammalian intestine, hydra head, and planarian regeneration (82 – 86). Importantly, Wnt/ β -catenin also is crucial for regulating proliferation in de- veloping and regenerating zebrafish neuromasts (26, 48).\nIn the mouse cochlea, ectopic Wnt/ β -catenin signaling pro- motes proliferation of Lgr5 + support cells via the down-regula- tion of Cdkn1c (p27/Kip1); however, this effect diminishes as the animals mature (3, 20, 28, 81, 87). Because Wnt/ β -catenin sig- naling is down-regulated in the mammalian cochlea after birth, it was hypothesized that the absence of Wnt/ β -catenin signaling is responsible for the inability of support cells to proliferate in response to hair cell death (3, 28, 29, 65). Therefore, recent studies have focused on up-regulating the Wnt/ β -catenin path- way in uninjured cochleae with the expectation of coaxing mammalian support cells into proliferating and producing hair cells (3, 17, 28, 29, 81). These experiments indeed yielded ad- ditional hair cells, but the epithelium was disorganized.\nSurprisingly, our data demonstrate that in zebrafish, as in adult mice cochleae, Wnt/ β -catenin signaling is inactive in ho- meostatic neuromasts and is up-regulated only at 12 h after neomycin treatment. Therefore Wnt/ β -catenin signaling is un- likely to play a role in the initial triggering of the regenerative response in zebrafish, even though it is sufficient and necessary to induce proliferation during later stages of regeneration, for example by inhibiting Cdkn1b (p27/Kip1) and regulating dyclinD1 (3, 27, 87).\nNotch Signaling Is Down-Regulated Early in Lateral Line Hair Cell Regeneration. A growing number of studies have explored how the manipulation of Notch signaling could aid in regenerating mammalian hair cells. It is well established that the inhibition of Notch signaling causes an increase in the number of hair cells in uninjured and injured chicken, guinea pig, and mouse utricles and cochleae and in injured zebrafish ears and lateral line neu- romasts (52, 53, 55 – 57, 88). The presence of Notch is detrimental to hair cell differentiation, because it blocks the proneural gene atoh1a and thereby maintains support cells (9, 13, 65, 89, 90). The ability of support cells to transdifferentiate into hair cells after down-regulation of Notch signaling was exploited recently to restore a modest amount of hair cells in the injured adult mouse cochlea that led to some promising functional recovery of hearing (30). This experimental down-regulation of Notch sig- naling recapitulates our observations during zebrafish lateral line hair cell regeneration. In zebrafish, Notch signaling is transiently\ndown-regulated 1 h after hair cell death, correlating with the attenuation of sox2 and up-regulation of atoh1a . The interactions between Notch signaling, sox2 , and atoh1a have not yet been tested in the zebrafish lateral line; however, because of their behavior during regeneration, it is very likely that their regula- tion is similar to that occuring during mouse, Xenopus , and zebrafish central nervous system or sensory organ development (23, 52, 61, 91 – 93). During murine ear development Sox2 ex- pression depends on Notch signaling and is required for the acquisition of neural potential in the prosensory epithelium (52). Subsequently, for neurogenesis to proceed, Sox2 is shut off via the accumulation of proneural genes, such as Atoh (93 – 97). In contrast, Sox2 is maintained in Notch + support cells, where it inhibits hair cell differentiation by inhibiting the proneural basic helix-loop-helix gene Atoh1a (91).\nIt is plausible that the down-regulation of Notch signaling in neuromasts is required during the earliest stages of hair cell regeneration to down-regulate sox2 in support cells, allowing these cells to acquire a proneural character via the up-regulation of atoh1a . Because Notch signaling needs to be reinitiated for fate specification of hair and support cells and control of organ size (9, 60), the Notch target genes her4.1 , notch3 , and sox2 are again up- regulated between 3 and 5 h after neomycin treatment (Fig. 4 A and B ). These genes likely are activated by the up-regulation of dld (60). Therefore, we hypothesize that the orchestration of genes downstream of Notch signaling appears to be the same during zebrafish hair cell development and regeneration.\nIt has not been appreciated previously that the down-regula- tion of Notch signaling could be a hallmark of spontaneously regenerating sensory epithelia, because it was reported that the Notch signaling pathway is up-regulated in response to hair cell death in chicken, guinea pigs, and zebrafish (9, 88, 98). However, these expression analyses were performed at later time points during the regeneration process, when Notch signaling is re- quired once again for hair cell versus support cell specification to occur. We also observed that zebrafish Notch target genes are reexpressed at 3 h after hair cell death (Fig. 4 A ). Because the zebrafish lateral line system allows us to kill hair cells very acutely, causing a relatively synchronized response in support cells, we were able to detect the immediate, transient down-regulation of Notch signaling after hair cell death. In the injured mammalian cochlea or utricle such drastic down-regulation of Notch target genes, such as Sox2 and several Hes and Hey genes, does not occur (55, 56); the absence of such down-regulation could be one of the reasons mammalian hair cells do not regenerate.\nThe Regulation of Notch Signaling in Combination with Mitogenic Signals May Offer Novel Therapeutic Strategies to Promote Hair Cell Regeneration in Mammals. Our data suggest that identifying the molecules that down-regulate Notch signaling in regenerat- ing species will be important in efforts to understand why mammals fail to regenerate injured hair cells. Our RNA-Seq analyses identified the transmembrane protein Numb as a po- tential candidate for inhibiting Notch signaling. Future studies should focus on the sufficiency of Numb to down-regulate Notch signaling in zebrafish and understanding Numb ’ s temporal ex- pression patterns during homeostasis and after injury in other regenerating species, as well as in mice. Numb could open a new therapeutic avenue for manipulating Notch signaling in mammals and facilitating successful regeneration of hair cells in mammals.\nEven though down-regulation of Notch signaling induced differentiation of new hair cells in mammals, these hair cells developed via transdifferentiation of support cells into hair cells, leading to the depletion of support cells (30). The down-regu- lation of Notch was not sufficient to induce support cell pro- liferation as observed in spontaneously regenerating species. Zebrafish regenerate hair cells mainly or exclusively by pro- liferation of support cells, and regenerating chicken hair cells\narise via transdifferentiation, as well as proliferation (19). In zebrafish we observe that the down-regulation of Notch signaling at 1 h after neomycin treatment correlates with the down-regu- lation of cyclin-dependent kinase inhibitors cdkn1b (p27/kip1) and cdkn1c ( p57/kip2 ) (Fig. 4 A and D ). This result suggests that Notch could be an activator of cdkn1b ( p27/kip1 ) that keeps cells out of the cell cycle and therefore that the down-regulation of Notch signaling could be a crucial event for the onset of re- generation. Indeed in the murine inner ear, the Notch target Sox2 regulates Cdkn1b (p27/Kip1), similar to activity described in the epidermis (99, 100) but in contrast to the activity in the intestinal crypt and inner ear, where Notch inhibits Cdkn1b (p27/ Kip1) (101, 102). In zebrafish the attenuation of Notch signaling has no effect on proliferation in uninjured sensory organs and has only a proproliferative effect on support cells after injury (9). In chicken cochleae the proliferation rate is not increased after injury, nor does inhibition of Notch cause cell cycle reentry in mice cochleae (13, 19, 30). Based on these findings it was con- cluded that Notch signaling is not required to initiate pro- liferation during regeneration (13). Therefore, it is possible that Cdk inhibitors are regulated by as yet unidentified, Notch- independent signals. Possibly, Fgf signaling inhibits proliferation, as demonstrated in the chicken inner ear (63).\nBecause complete regeneration of sensory epithelia in mam- mals requires coaxing support cells into hair cell differentiation as well as stimulating the support cells to re-enter the cell cycle and self-renew, manipulation of Notch signaling is not sufficient and needs to be married with manipulations of mitogenic signals. Furthermore, similar to our observations during zebrafish hair cell regeneration, Notch signaling must be reinitiated for the inhibition of proneural genes and specification of support cells (12, 23, 91). A temporally restricted Notch down-regulation or a combination of Notch down-regulation with subsequent ectopic expression of support cell inducers, such as Sox2, could be fruitful.\nOur efforts presented here have characterized the pattern of temporal activation of signaling pathways during zebrafish sensory hair cell regeneration. These results will aid in the design and in- terpretation of functional studies aimed at elucidating the gene-reg- ulatory network underlying zebrafish hair cell regeneration and can guide experiments to trigger hair cell regeneration in mammals.\n\n\nThis article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1402898111/-/DCSupplemental . www.pnas.org/cgi/doi/10.1073/pnas.1402898111 PNAS | Published online March 27, 2014 | E1383 – E1392 DEVELOPMENTA L BIOLOGY PNAS PLUS SEE CORRECTION FOR THIS ARTICLE\n\n\ncontrol 1h ptp4a2 lima1 junba abcf2a eif4a1a control 1h il11b nlrc3 hk2 (glulc) g3bp1 rapgef3 txn clic2 1h lad1 znf462 dfna5 gpr37b zgc:110353 slc35c2 tcea3 A B control zgc:152741 Fig. 2. Validation by in situ hybridization of a selection of 14 genes up- regulated ( A ) and seven genes down-regulated ( B ) at 1 h after neomycin treatment. ( Upper Rows ) Untreated larvae (control) at 5 dpf. ( Lower Rows ) Treated larvae.\n\n\nPNAS PLUS\n\n\nrbl2\n\n\nher4.1 notch3 1h 3h 5h 10h cdkn1c cdkn1b 1h 3h 5h 10h C A numb sox2 atoh1a dld pea3 fgf3 1h 3h 5h 10h D her4.1 atoh1b her6 numbl dlc gro2 ctbp2 notch3 cir1 jag2 aph1b ncor2 dld dla atoh1a numb cdk6 prkdc rbl2 mad2l2 cdkn1c cdkn1b ccna1 gadd45aa gadd45ba gadd45ab cdk7 plk1 ywhag1 pcna ccne1 anapc1 ywhai abl1 stag2 gadd45g mcm6l wee1 1h 3h 5h 1h 3h 5h 1h 3h 5h 1h 3h 5h B E control control control Notch cell cycle notch1b jag1a jag1b rbpjb sox2 GFP GFP GFP GFP 2 0 -2 2 0 -2 Fig. 4. Inhibition of Notch and Fgf pathways is accompanied with cell cycle reentry immediately after hair cell ablation. ( A , C , and D ) In situ hybridization of Notch pathway, Fgf pathway, and cell cycle genes in 5-dpf untreated and neomycin-treated larvae. ( A ) The Notch pathway inhibitor numb is increased at 1 h after neomycin. The Notch target genes her4.1 , notch3 , and sox2 are decreased at 1 h and up-regulated at 3 h after neomycin treatment. a toh1a is up- regulated starting at 1 h after neomycin, and its target dld is increased at 5 and 10 h. ( C ) fgf3 and its target pea3 are down-regulated at 1 and 3 h after neomycin treatment. ( D ) Cell cycle inhibitory genes cdkn1b , cdkn1c , and rbl2 are decreased at 1 and 3 h after neomycin compared with control. ( B and E ) Heat maps of selected Notch pathway genes and cell cycle genes.\n\n\nbmp5 control 1h 3h 5h 10h A stat3 1h 3h 5h 10h control B C Wnt Notch Fgf cell cycle Stat3 1h 3h 5h 10h 12h no change inhibition activation not tested\n\n\nPNAS PLUS\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nE1384 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1385 DEVELOPMENTA L\n\n\nE1386 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1387 DEVELOPMENTA L\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1389 DEVELOPMENTA L\n\n\nE1390 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\n1. White PM, Doetzlhofer A, Lee YS, Groves AK, Segil N (2006) Mammalian cochlear supporting cells can divide and trans-differentiate into hair cells. Nature 441(7096): 984 – 987.\n\n\n2. Corwin JT, Oberholtzer JC (1997) Fish n ’ chicks: Model recipes for hair-cell regeneration? Neuron 19(5):951 – 954.\n\n\n5. Williams JA, Holder N (2000) Cell turnover in neuromasts of zebrafish larvae. Hear Res 143(1-2):171 – 181.\n\n\n8. López-Schier H, Hudspeth AJ (2006) A two-step mechanism underlies the planar polarization of regenerating sensory hair cells. Proc Natl Acad Sci USA 103(49): 18615 – 18620.\n\n\n9. Ma EY, Rubel EW, Raible DW (2008) Notch signaling regulates the extent of hair cell regeneration in the zebrafish lateral line. J Neurosci 28(9):2261 – 2273. 10. Groves AK (2010) The challenge of hair cell regeneration. Exp Biol Med (Maywood) 235(4):434 – 446.\n\n\n27. Wada H, et al. (2013) Wnt/Dkk negative feedback regulates sensory organ size in zebrafish. Curr Biol 23(16):1559 – 1565.\n\n\n28. Shi F, Hu L, Edge AS (2013) Generation of hair cells in neonatal mice by β -catenin overexpression in Lgr5-positive cochlear progenitors. Proc Natl Acad Sci USA 110(34): 13851 – 13856.\n\n\n33. Whitfield TT (2002) Zebrafish as a model for hearing and deafness. J Neurobiol 53(2): 157 – 171.\n\n\n56. Korrapati S, Roux I, Glowatzki E, Doetzlhofer A (2013) Notch signaling limits sup- porting cell plasticity in the hair cell-damaged early postnatal murine cochlea. PLoS ONE 8(8):e73276.\n\n\n57. Lin V, et al. (2011) Inhibition of Notch activity promotes nonmitotic regeneration of hair cells in the adult mouse utricles. J Neurosci 31(43):15329 – 15339.\n\n\n58. McGill MA, Dho SE, Weinmaster G, McGlade CJ (2009) Numb regulates post-endo- cytic trafficking and degradation of Notch1. J Biol Chem 284(39):26427 – 26438.\n\n\n60. Matsuda M, Chitnis AB (2010) Atoh1a expression must be restricted by Notch sig- naling for effective morphogenesis of the posterior lateral line primordium in ze- brafish. Development 137(20):3477 – 3487.\n\n\n63. Oesterle EC, Bhave SA, Coltrera MD (2000) Basic fibroblast growth factor inhibits cell proliferation in cultured avian inner ear sensory epithelia. J Comp Neurol 424(2): 307 – 326.\n\n\n69. Fang Y, et al. (2013) Translational profiling of cardiomyocytes identifies an early Jak1/Stat3 injury response required for zebrafish heart regeneration. Proc Natl Acad Sci USA 110(33):13416 – 13421.\n\n\n74. Oesterle EC, Tsue TT, Rubel EW (1997) Induction of cell proliferation in avian inner ear sensory epithelia by insulin-like growth factor-I and insulin. J Comp Neurol 380(2): 262 – 274.\n\n\n75. Carnovale CE, Ronco MT (2012) Role of nitric oxide in liver regeneration. Ann Hepatol 11(5):636 – 647.\n\n\n76. Sage C, et al. (2005) Proliferation of functional hair cells in vivo in the absence of the retinoblastoma protein. Science 307(5712):1114 – 1118.\n\n\n77. Sage C, et al. (2006) Essential role of retinoblastoma protein in mammalian hair cell development and hearing. Proc Natl Acad Sci USA 103(19):7345 – 7350.\n\n\n80. van Amerongen R, Nusse R (2009) Towards an integrated view of Wnt signaling in development. Development 136(19):3205 – 3214.\n\n\n81. Shi F, Kempfle JS, Edge AS (2012) Wnt-responsive Lgr5-expressing stem cells are hair cell progenitors in the cochlea. J Neurosci 32(28):9639 – 9648.\n\n\n82. Galliot B, Chera S (2010) The Hydra model: Disclosing an apoptosis-driven generator of Wnt-based regeneration. Trends Cell Biol 20(9):514 – 523.\n\n\n83. Fleming HE, et al. (2008) Wnt signaling in the niche enforces hematopoietic stem cell quiescence and is necessary to preserve self-renewal in vivo. Cell Stem Cell 2(3): 274 – 283. 84. Li L, Bhatia R (2011) Stem cell quiescence. Clin Cancer Res 17(15):4936 – 4941.\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Gene-expression analysis of hair cell regeneration in the zebrafish lateral line\nLinjia Jiang a , Andres Romero-Carvajal a,b , Jeff S. Haug a , Christopher W. Seidel a , and Tatjana Piotrowski a,1\na Stowers Institute for Medical Research, Kansas City, MO 64110; and b Department of Neurobiology and Anatomy, University of Utah Medical School, Salt Lake City, UT 84132\nEdited* by Yuh Nung Jan, Howard Hughes Medical Institute, University of California, San Francisco, CA, and approved March 4, 2014 (received for review February 16, 2014)\nDeafness caused by the terminal loss of inner ear hair cells is one of the most common sensory diseases. However, nonmammalian animals (e.g., birds, amphibians, and fish) regenerate damaged hair cells. To understand better the reasons underpinning such disparities in regeneration among vertebrates, we set out to define at high resolution the changes in gene expression associ- ated with the regeneration of hair cells in the zebrafish lateral line. We performed RNA-Seq analyses on regenerating support cells purified by FACS. The resulting expression data were subjected to pathway enrichment analyses, and the differentially expressed genes were validated in vivo via whole-mount in situ hybrid- izations. We discovered that cell cycle regulators are expressed hours before the activation of Wnt/ β -catenin signaling following hair cell death. We propose that Wnt/ β -catenin signaling is not involved in regulating the onset of proliferation but governs pro- liferation at later stages of regeneration. In addition, and in marked contrast to mammals, our data clearly indicate that the Notch pathway is significantly down-regulated shortly after in- jury, thus uncovering a key difference between the zebrafish and mammalian responses to hair cell injury. Taken together, our findings lay the foundation for identifying differences in sig- naling pathway regulation that could be exploited as potential therapeutic targets to promote either sensory epithelium or hair cell regeneration in mammals.\nsignaling pathway analysis | RNA sequencing | neuromast | Jak/Stat3 | cdkn1b\nH earing disorders are the most common sensory disabilities in humans and can be either inherited or acquired. De- generation and death of sensory hair cells in the inner ear is the underlying cause of age-related, noise-, antibiotic-, or chemo- therapy drug-induced hearing loss ( www.nidcd.nih.gov ). Such loss is permanent, because mammalian inner ear hair cells do not regenerate. The inability of mammals to restore sensory hair cells after injury is intriguing given that ( i ) mammalian vestibular support cells exhibit low levels of proliferation after injury; ( ii ) cochlear support cells of newborn mice are able to proliferate subsequent to hair cell death in cell culture (1); and ( iii ) other vertebrates such as birds, amphibians, and fish replace their sensory hair cells continuously and, importantly, are capable of regenerating hair cells after damage (2 – 10).\nWork in birds has shed light on the mechanisms regulating sensory hair cell regeneration in vertebrates (2, 4, 11 – 18). For instance, during sensory hair cell regeneration in the chick, the majority of hair cells arise by proliferation of neighboring sup- port cells (19). Given that mammalian support cells do not re- spond to hair cell death by proliferating, molecular studies in chick have focused on identifying the mitogenic signals as well as the target genes that induce re-entering of the cell cycle in support cells (11, 20 – 22). For example, the transcription factors Atoh1 and Sox2 and the Wnt/ β -catenin and Notch pathways have been shown to play crucial roles (23), and gene-expression analyses using microarrays have identified hundreds of genes involved in chick hair cell regeneration (18, 24).\nThe Wnt/ β -catenin pathway is used repeatedly during hair cell development, and recent studies have emphasized the importance of Wnt/ β -catenin signaling in modulating the proliferation of sup- port cells. For example, activated β -catenin causes proliferation in dissociated chicken utricular epithelia and ectopic expression of hair cell differentiation in the chicken basilar papilla (17, 25). Likewise, during mouse cochlear development Wnt/ β -catenin sig- naling activates proliferation of support cells and is required sub- sequently for their differentiation into hair cells (3, 26 – 29). Perturbing the signaling pathways known to be activated during chick inner ear regeneration therefore may provide an opportunity to modulate regenerative capacities in the mam- malian inner ear. For instance, it has been shown recently that down-regulation of Notch signaling in adult mice after injury induces the transdifferentiation of support cells into hair cells and can partially restore hearing (30). However, this manipula- tion causes the loss of support cells, and the implications of this loss for long-term hearing restoration are unknown.\nSignificant progress has been made in the study of sensory hair regeneration. However, in part because of the complexity of spatial and temporal gene interactions and signaling dynamics, our knowl- edge still is too incomplete to enable us to promote the generation of a functional epithelium after injury in mammals. The zebrafish lateral line provides an experimentally accessible system to define the complex signaling events triggered by injury and regeneration, because these cells can be killed acutely and then regenerate rap- idly (31). These characteristics allow the detection of changes in gene expression at defined time points following hair cell death.\nSignificance\nDeafness is caused largely by the death of sensory hair cells in the inner ear. In contrast to nonmammalian vertebrates, hu- man hair cells do not regenerate. Understanding the mecha- nisms that regulate hair cell regeneration in zebrafish may shed light on the factors that prevent hair cell regeneration in mammals. RNA-Seq analysis of regenerating sensory organs uncovered dynamic changes in the expression of signaling pathways during zebrafish hair cell regeneration. Unexpect- edly, the Wnt/ β -catenin, Notch, and Fgf pathways are down- regulated following hair cell death, whereas the Jak1/Stat3 pathway and cell cycle are activated. We propose that mim- icking the zebrafish activation status of a combination of path- ways at the correct time points in mammals may improve the chances of triggering regeneration of functional hair cells.\nAuthor contributions: L.J. and T.P. designed research; L.J., A.R.-C., and J.S.H. performed research; L.J., J.S.H., C.W.S., and T.P. analyzed data; and L.J. and T.P. wrote the paper.\nThe authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nData deposition: The data reported in this paper have been deposited in the Gene Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE56176 ). 1 To whom correspondence should be addressed. E-mail: pio@stowers.org.\nZebrafish and the majority of all other aquatic vertebrates possess hair cells not only in their vestibular system in the ear but also in their skin. These hair cells are part of the sensory lateral line system. The lateral line system senses water motion and initiates the appropriate behavioral response for capturing prey, avoiding predators, and schooling. The morphology and function of sensory hair cells are evolutionarily conserved from fish to mammals (32, 33). Lateral line hair cells are located in the center of a mechanosensory organ known as the neuromast and are surrounded by inner support cells and an outer ring of mantle cells. Importantly, despite the unusual location of the hair cells on the trunk, lateral line and ear hair cells develop and differ- entiate by similar developmental mechanisms. For example, mutations in genes causing deafness in humans also disrupt hair cell function in the zebrafish lateral line and vestibular system (34). Because of its genetic tractability and the accessibility of sensory organs to experimental manipulation and visualization, the zebrafish recently has been recognized as an excellent model for discovering and functionally characterizing genes crucial for hair cell development and regeneration (16, 19, 35, 36).\nIn zebrafish hair cell regeneration occurs via the proliferation and differentiation of inner support cells (5, 6, 9, 37, 38), but detailed lineage analyses have not yet been performed, and the stem cell population has not been identified. Mantle cells are good stem cell candidates, because they migrate to form neu- romasts on regenerating tail tips and also form postembryonic neuromasts by a process called “ stitching. ” However, after chemical ablation of hair cells by antibiotic treatment, mantle cells proliferate much less frequently than inner support cells (9). Possibly, mantle cells replenish inner support cells that then differentiate into hair cells. Mantle cells have not been shown to differentiate directly into hair cells (37, 38). Therefore, both mantle and inner support cells seem to be involved in reestablishing proper neuromast morphology after hair cell death.\nTo identify the activation status of signaling pathways during the first few hours of regeneration, we performed RNA-Seq gene expression analyses of zebrafish mantle and inner support cells isolated by FACS. We discovered that the Notch pathway is transiently down-regulated immediately after hair cell death; to our knowledge, this down-regulation has not been previously reported in other studies of regenerating species. Our studies also demonstrate that Wnt/ β -catenin signaling is not among the earliest pathways to respond to hair cell death but instead is up- regulated relatively late during the regenerative response. These results suggest that Wnt/ β -catenin signaling does not trigger proliferation by down-regulating cyclin-dependent kinase inhib- itors immediately after hair cell death but rather regulates pro- liferation at later stages of regeneration. Our findings have defined the temporal regulation of key signaling pathways asso- ciated with the early stages of hair cell regeneration and suggest that the successful initiation of mammalian hair cell regeneration, either in vivo or in vitro, might require precise temporal manip- ulation of these pathways.\n\n\nResults\nIsolation of Mantle and Inner Support Cells by FACS. To identify changes in gene expression in mantle and inner support cells after hair cell death, we isolated GFP + cells from regenerating and control Tg ( sqET20 ) transgenic zebrafish via FACS (Fig. 1 A and B ) (39). Tg ( sqET20 ) larvae express GFP strongly in mantle cells and to a lesser degree in inner support cells (Fig. 1 B ). Hair cells are GFP − and therefore are excluded from the GFP + population. We isolated GFP − and GFP + cells from Tg ( sqET20 ) control larvae at 5 days postfertilization (dpf) and from neo- mycin-treated larvae at 1, 3, and 5 h after neomycin treatment. Larvae were dissociated into a single-cell suspension in a trypsin solution and stained with 7-aminoactinomycin D (7-AAD) and Hoechst 33342 to exclude dead cells and cell debris (Fig. 1 C ). Live GFP + cells (7-AAD − , Hoechst + , GFP + cells) and control GFP − cells (7-AAD − , Hoechst + , GFP − cells) were sorted by FACS (Fig. 1 D ). The sorting yielded highly GFP + -enriched cell populations, as demonstrated by a postsort reanalysis of the collected cells (Fig. 1 E and F ). GFP + -sorted cells showed 94.4 ± 1.3% enrichment ( n = 7). To confirm that GFP + cells are lateral line cells, we performed immunostaining with an antibody against the support cell marker Sox2 that labels inner support cells as well as some mantle cells (Fig. 1 G and Fig. S1 ) (40). Counts of Sox2 + FACS-purified cells revealed that 95% of GFP + cells but only 6.6% of GFP − cells were positive for Sox2 (Fig. 1 H ). Ad- ditionally, we performed RT-PCR for genes highly expressed in neuromast cell types and with limited expression in other tissues. RT-PCR for gfp , eya1 , klf4b , and cldnb showed that these genes are expressed at significantly higher levels in GFP + than in GFP − cells ( Fig. S2 A – D ). In contrast, genes expressed in tissues outside the lateral line system, such as rhodopsin ( rho , eye), vil- lin1 ( vil1 , gut), and myod1 (muscle), are expressed at much lower levels in GFP + cells ( Fig. S2 E – G ). In Tg ( sqET20 ) larvae, mantle B A G K Sox2    cells (%) 0 20 40 60 80 100 H lateral line genes neomycin induced changes J Tg(sqET20) mantle cell inner support cell GFP GFP GFP 1h Sox2 GFP DAPI GFP GFP GFP control neo control neo control 30min 1h 3h 5h 30min 1h 3h 5h GFP 3h GFP 5h GFP 1h GFP 3h GFP 5h GFP changes in mantle and support cells C D 97.8% FSC FSC 0 1000 2000 3000 4000 FSC 100 101 2 103 10 100 101 10 2 103 104 Hoechst GFP GFP 94.3% live cells 10 4 E F GFP GFP 100 101 2 103 10 10 4 100 101 2 103 10 10 4 0 1000 2000 3000 4000 0 1000 2000 3000 4000 100 101 2 103 10 10 4 I 7-AAD GFP GFP GFP GFP\nFig. 1. Purification of mantle and inner support cells from Tg ( sqET20 ). ( A ) In 5-dpf Tg ( sqET20 ) larvae GFP is expressed in neuromasts (arrows) and in the interneuromast cells that connect adjacent neuromasts. ( B ) Magnification of a neuromast shows that mantle cells are strongly labeled by GFP, but inner support cells are weakly labeled. ( C and D ) Dissociated cells of Tg ( sqET20 ) larvae at 5 dpf were FACS sorted using a two-gate sorting strategy. Live cells were gated by 7-AAD − and Hoechst + ( C ) and subsequently were gated by forward scatter (FSC) and GFP intensity to isolate GFP − and GFP + cells ( D ). ( E and F ) A postsorting experiment shows that GFP − cells are 97.8% pure ( E ) and GFP + cells are 94.3% pure ( F ). ( G ) Sorted GFP − and GFP + cells were stained with Sox2 antibody and DAPI. ( Upper ) A GFP − cell is Sox2 − . ( Lower ) A GFP + cell expresses Sox2. (Scale bars: 5 μ m.) ( H ) Quantification of Sox2 + cells in each population as mean ± SD; * P &lt; 0.01 by t test. ( I ) Regeneration- induced mantle and inner support cell genes were identified at each time point by comparing GFP + neomycin-treated cells with GFP + untreated samples. ( J ) Neomycin-induced genes were identified by comparing GFP − neomycin-treated cells at each time point with GFP − untreated samples. Neomycin-induced genes subsequently were subtracted from the gene lists obtained with GFP + cells to select for regeneration-specific changes in gene expression in inner support and mantle cells. ( K ) Comparisons between GFP + and GFP − cells of untreated larvae generated a list of genes enriched in lateral line cells.\ncells express GFP most strongly, but inner support cells respond more robustly to hair cell death by proliferating (5, 6). Because Tg ( sqET20 ) larvae show weaker GFP expression in inner support cells, we were concerned that our cell sorts would exclude inner support cells. However, RT-PCR with inner support cell markers such as klf4b showed that inner support cells were included in the GFP + cell populations ( Fig. S2 C ). Combined, these studies dem- onstrated that our FACS protocol yielded a highly enriched pool of lateral line cells containing mantle and inner support cells but not hair cells.\nTo investigate the temporal activation or repression of in- dividual genes and signaling pathways in GFP + mantle and inner support cells during the first few hours of regeneration, we ex- amined GFP + and GFP − cells at 1, 3, and 5 h after a 30-min treatment with neomycin (Fig. 1 I ). Sorts for each time point were performed in triplicate and subjected to RNA-Seq analysis. Bacte- ricidal antibiotics cause oxidative stress in mammalian cells, leading us to speculate that sublethal concentrations of neomycin may cause changes in gene expression unrelated to hair cell death (41). To control for this nonspecific effect, we examined gene expression in GFP − cells in the presence or absence of neomycin (Fig. 1 J ). Al- though this population includes all larval nonlateral cells, it allows us to control for a nonspecific neomycin effect when comparing GFP + cells from treated and control larvae.\nAnalysis of the RNA-Seq Results. Sequencing libraries were pre- pared from mRNAs of collected cells and were quality checked on a Bioanalyzer. We performed 50-bp single-end RNA-Seq using the Illumina platform. To test the enrichment of lateral line genes further, we examined the fragments per kilobase of transcript per million mapped reads (FPKM) values for known lateral line genes and for genes expressed in other tissues ( Fig. S3 ). The results confirmed the RT-PCR analysis described above. Pan lateral line, as well as inner support cell, markers show much higher FPKM values in FACS-sorted GFP + cells than in GFP − cells. As suggested by the RT-PCR results, the FPKM values for the inner support cell markers pea3 and klf4b are significantly higher in the GFP + than in the GFP − cells, confirming that inner support cells were included in our FACS sorts. This enrichment of lateral line genes is supported by the information in Dataset S1 , in which neuromast-specific genes were identified by comparing the expression profiles of untreated GFP + and GFP − cells. In addition to known lateral line genes, the resulting lists of differentially expressed genes provide a valuable resource of as yet uncharacterized genes that potentially could play important roles in hair cell development and/or re- generation (Fig. 1 K and Dataset S1 ). The number of genes enriched in GFP + cells relative to GFP − controls at an adjusted p-value ≤ 0.05 is 1,670 ( SI Materials and Methods and Dataset S1 ). This dataset also contains many of the genes reported in the dataset of Steiner et al. (42), who identified mantle cell-specific genes using a different transgenic line (see below).\nGene Identification for Each Time Point. To identify genes from transcripts specifically enriched or depleted in GFP + mantle and inner support cells after hair cell death, we created several comparisons between the RNA-Seq datasets. Ratios of gene expression were created between the neomycin-treated GFP + cells at 1, 3, and 5 h and the nontreated GFP + cells at 1 h to identify genes responding to hair cell death (Fig. 1 I ). To control for a nonspecific neomycin response, a similar set of ratios was created between the neomycin-treated GFP − cells at each time point and the untreated GFP − cells at 1 h (Fig. 1 J ). The neomycin- induced genes were subtracted from all gene lists generated with GFP + cells. We then used ratios and P values between datasets to select genes of interest at any given time point ( SI Materials and Methods and Dataset S2 ). Genes identified by these criteria are marked with a numeric flag with positive numbers indicating up-\nregulated and negative numbers indicating down-regulated genes. The numeric value indicates the time point at which a gene is up- or down-regulated ( Dataset S2 , flagged column). A principal component analysis of the biological replicates of GFP − and GFP + cell populations at the three different time points demonstrates that GFP + cells are very different from GFP − cells. In addition, cell sorts performed in triplicate for each time point are highly re- producible ( Fig. S4 ). To define a set of the top 100 up- and down-regulated genes to use as candidates for validation, we ranked 193 up-regulated and 200 down-regulated significant (flagged) genes from the 1-h dataset as a function of the ratio and general abundance ( SI Materials and Methods and Table S1 ). We validated the RNA- Seq results by performing in situ hybridizations with 28 up-regulated and 21 down-regulated genes selected from the top 100 gene list in larvae 1 h after neomycin treatment ( Table S2 ). All 28 up- regulated genes are expressed in the lateral line, and 20 of these genes show up-regulation by in situ hybridization after neomycin treatment. Of the 21 down-regulated genes, 19 are expressed in the lateral line, and 12 genes are detectably down-regulated by in situ hybridization (Fig. 2 and Table S2 ). These experiments de- monstrated that the FACS sorting followed by RNA-Seq analysis produced high-quality results that enable us to study hair cell regeneration in zebrafish in detail.\nPathway Analysis at 1, 3, and 5 h After Hair Cell Death. The Wnt/ β -catenin, Notch, Jak/Stat, Bmp, and Fgf pathways are important for hair cell development as well as for the regeneration of hair cells in fish and chick epithelia (15, 23, 43). Our understanding of which pathways act upstream and how these pathways affect each other is still very limited. We performed a temporal ex- pression analysis at 1, 3, and 5 h after hair cell death to de- termine when particular pathways are activated or inhibited; this temporal information provides clues into how they might interact with each other. We also aimed to identify other signaling path- ways and record any dynamic changes in their behavior that could indicate how they integrate with the Notch and Wnt/ β -catenin signaling pathways. To identify enriched pathways in our datasets, we used the signaling pathway impact analysis (SPIA) package in\nccdc48\nR (44). This software tool not only takes into account how many genes belong to a pathway but also focuses on the importance of a gene by analyzing the perturbation of the pathway if the gene expression changes. Therefore, even pathways that are repre- sented by only a small number of genes can be classified as en- riched if these genes are important regulators of the pathway. For the SPIA analysis we included all flagged genes (flagged column) listed in Dataset S2 . For each time point the up- and down-regulated genes were combined and analyzed against a panel of pathways from zebrafish defined in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (45). The 1-h postneomycin treatment time point contained 509 genes from the National Center for Bio- technology Information Entrez database. The 3-h postneomycin treatment time point contained 401 Entrez genes, and the 5-h postneomycin treatment time point contained 418 genes.\nThe results of the SPIA analysis are shown in Table S3 . Au- tomated pathway annotation is powerful but subject to caveats, because genes can be annotated to function in several pathways, some of which might not be active in the lateral line. In addition, in the KEGG database, the Wnt pathway encompasses the ca- nonical Wnt/ β -catenin, the PCP, and the Wnt- regulated calcium pathways, potentially providing erroneous status assignment of the Wnt/ β -catenin pathway. We therefore tested the SPIA ana- lysis by in situ hybridization experiments with pathway members at various time points of regeneration (see below and Figs. 3 – 5). For improved visualization we generated heat maps of log2 ratio values of KEGG pathway members. We show only the genes that were flagged in our analysis and reported to be expressed in the lateral line by the zebrafish information network ( www.zfin.org ) or prior reports (46, 47).\nThe Wnt/ β -Catenin Pathway Is Not Activated During Early Stages of Lateral Line Hair Cell Regeneration. The analysis of hair cell re- generation in a Tg ( sqEt20;sqEt4 ) double-transgenic larva allows us to draw comparisons between gene expression and cellular changes at particular time points (Fig. 3 A ) (39). At 1 h after neomycin treatment almost all hair cells have been eliminated. Two hair cells remain that were too immature to be killed by neomycin. Proliferation of inner support cells begins within the first 2 h of regeneration (37). At 5 h two newly differentiating hair cells start to express GFP (Fig. 3 A ; arrows), and by 12 h after neomycin treatment six hair cells are present.\nWnt/ β -catenin signaling is required for proliferation in de- veloping and regenerating neuromasts (26, 48). Unexpectedly, the SPIA pathway analysis suggested that the Wnt/ β -catenin pathway is inactive at 1 h after hair cell death ( Table S3 ). To investigate this finding in more detail, we performed in situ hy- bridization experiments with Wnt/ β -catenin pathway genes in untreated and treated larvae at various time points after hair cell death. Our analysis showed that Wnt/ β -catenin signaling is not active in 5-dpf untreated control neuromasts. The ligand wnt10a is not expressed in 5-dpf control neuromasts, and we failed to detect any expression of dgfp in the Tg ( Tcf/Lef-miniP:dGFP ) Wnt/ β -catenin reporter line (Fig. 3 B ) (49). Even though Wnt/ β -catenin signaling is inactive, the β -catenin cotranscription fac- tor tcf7l2 ( tcf4 ) is expressed. Likewise, the Wnt receptors fzd7b and 8a are present, although whether they act as receptors for canonical or noncanonical Wnt signaling in neuromasts is un- known (Fig. 3 B ) (50, 51). At 1 h after neomycin treatment tcf7l2 ( tcf4 ), frz7b , and fzd8a are drastically down-regulated, and, along with wnt10a , their tcf4 wnt10a dgfp control 1h 3h 5h 10h control 1h 5h 12h 18h fzd8a fzd7b lef1 fzd7b fzd8a wif1 dkk1b tcf7l1a tcf7l2 fzd9b ctbp2 axin2 lef1 wnt8a csnk1a1 jun sfrp1a wnt10a cul1a siah1 gro2 sox17 tbl1x fbxw11b gsk3b csnk1a1 1h 3h 5h 1h 3h 5h fzd6 dkk2 control 1h 3h 5h 10h 12h 18h 24h GFP GFP Wnt 2 0 -2 A B C Fig. 3. The Wnt/ β -catenin pathway is inactive during the early stages of hair cell regeneration. ( A ) Still images of a Tg ( sqET20;sqET4 ) larval neuromast in the process of neomycin-induced regeneration. All hair cells, except for two immature hair cells, were killed by neomycin by 1 h after neomycin treatment. Two newly formed hair cells (arrows) start to express GFP at 5 h, and other pairs of hair cells (arrows) appear at 12 and 18 h. ( B ) In situ hybridization of Wnt/ β -catenin pathway genes in 5-dpf untreated (control) larvae and in larvae at different time points after neomycin treatment. Expression of wnt10a is in- creased at 3 and 5 h. Expression of tcf7l2 ( tcf4 ), fzd7b , and fzd8a is largely decreased at 1 h after neomycin. The Wnt/ β -catenin reporter line Tg ( Tcf/Lef-miniP: dGFP ) demonstrates activation starting at 12 h as shown by dgfp expression, but lef1 is not induced in regenerating neuromasts. ( C ) Heat map of selected Wnt/ β -catenin pathway genes based on RNA-Seq results. Log2 ratios of GFP − and GFP + cells at 1, 3, and 5 h after neomycin treatment are color coded (red: up- regulation, blue: down-regulation). Genes that are modulated in GFP − cells are neomycin-induced genes.\nexpression in the Wnt/ β -catenin Tg ( Tcf/Lef-miniP:dGFP ) reporter line remains low or undetectable. At 3 h after neomycin treatment wnt10a , tcf7l2 ( tcf4 ) and fzd7b , and fzd8a are up-regulated, leading to Wnt/ β -catenin pathway activation at 12 h as demonstrated by dgfp expression in the Tg ( Tcf/Lef-miniP:dGFP ) reporter line (Fig. 3 B ). The activation status of other Wnt/ β -catenin pathway genes at the different time points is shown in Fig. 3 C . Genes that change expression in the GFP − cells represent neomycin-induced genes. Our analyses reveal that Wnt/ β -catenin signaling is a late- responding pathway and does not play a role in the initiation of regeneration.\nThe Notch Pathway Is Inhibited 1 h After Neomycin Treatment. The Notch pathway is of particular interest because experimental down-regulation of Notch induces transdifferentiation of mam- malian support cells into hair cells (30, 52 – 57). In contrast to previous reports in zebrafish and chicken (9, 13), our SPIA pathway analysis suggested that the Notch pathway is inhibited immediately after hair cell death because of simultaneous up- regulation of numb (Fig. 4 A and B ). Numb inhibits Notch ac- tivity by regulating postendocytic sorting events and degradation of the Notch receptor and thereby regulates neural stem cell homeostasis (58, 59). However, because Numb is a multifunc- tional membrane-associated protein with additional functions\nindependent of the regulation of Notch signaling, we tested if the Notch pathway is indeed down-regulated by in situ hybridization of Notch pathway members. Our analysis of the Notch target genes her4.1 and notch3 confirms that Notch signaling is attenuated immediately 1 h after neomycin treatment and is accompanied\nFig. 5. The Jak1/Stat3 pathway is activated during the early stages of hair cell regeneration. ( A ) In situ hybridization shows that stat3 is increased at 1 and 3 h after neomycin treatment. ( B ) bmp5 is up-regulated at 1 h after neomycin treatment in situ. ( C ) Table depicting the status of signaling path- ways at different time points after hair cell death. All comparisons (no change, inhibition, or activation) were evaluated relative to untreated control samples. The Wnt, Notch, and Fgf pathways are inhibited immediately after neomycin treatment, whereas the cell cycle and Jak1/Stat3 pathways are activated during the early stages of regeneration. The Wnt and Notch pathways are reactivated at 12 h and 5 h after neomycin treatment, respectively.\nby down-regulation of sox2 (Fig. 4 A and B ) and up-regulation of atoh1a (Fig. 4 A and B ). The atoh1a target deltaD ( dld ) is up- regulated between 3 – 5 h after neomycin treatment, reflecting the appearance of hair cell precursors (Fig. 4 A and B ) (60). Notch signaling must be reinitiated for proper fate specification into hair and support cells and proper organ size control (9, 60). Accordingly, between 3 and 5 h after neomycin treatment (Fig. 4 A and B ), we observed renewed activation of the Notch target genes her4.1 , notch3 , and sox2 that likely is caused by the up- regulation of dld (60).\nWe postulate that down-regulation of Notch signaling is a characteristic event during hair cell regeneration in zebrafish. In contrast to zebrafish, the Notch target genes are decreased only modestly, and Notch signaling is still active, in hair cell-depleted neonatal mouse cochleae and 3- to 4-wk-old mouse utricles (55, 56, 61). Therefore we hypothesize that the failure of transient Notch down-regulation in mammals could be one of the reasons that mammalian hair cells do not regenerate spontaneously after injury.\nFgf Signaling Is Down-Regulated Immediately After Hair Cell Death. The Fgf pathway initiates dld and subsequently Notch signaling in the mouse cochlea, chicken inner ear, and zebrafish lateral line neuromasts (23, 60). Both pathways are active in 5-dpf un- treated neuromasts (Fig. 4 A and C and Fig. S5 D ). Identically to Notch pathway members, fgf3 and its transcriptional target pea3 are turned off 1 h after neomycin treatment. However, the Notch target her4.1 is reactivated at 3 h, whereas pea3 is not transcribed until 5 h (Fig. 4 A and C ). Therefore, it is possible that Notch signaling does not depend on Fgf signaling during regeneration; however, this hypothesis needs to be tested. The rapid down- regulation of Fgf signaling is in accordance with findings in chicken sensory epithelia, in which Fgfr3 is highly expressed in support cells but is down-regulated rapidly after hair cell death (62). Additionally, in the chicken down-regulation of Fgf sig- naling is required for regeneration to occur, because the addition of Fgf inhibits support cell division after damage (63).\nThe Cell Cycle Exit Genes cdkn1b , cdkn1c , and rbl2 Are Down- Regulated at 1 h After Hair Cell Death. The failure to down-regu- late cell cycle exit regulators is one of the reasons mammalian support cells do not proliferate in response to hair cell death (10, 12, 64, 65). Determining the earliest changes in the expression of cell cycle regulators and mapping correlating changes in signaling pathway activations will aid in identifying candidate pathways in- volved in cell cycle reentry. We detected the earliest changes in cell cycle genes at 1 h after neomycin treatment. The cyclin- dependent kinase (cdk) inhibitors cdkn1b ( p27/kip1 ) and cdkn1c ( p57/kip2 ), as well as the pocket domain-containing rbl2 , are down-regulated 1 h after neomycin treatment and are not up- regulated again until 5 – 10 h after neomycin treatment (Fig. 4 D and E ). Time-lapse analyses revealed that support cells enter the cell cycle 1 – 5 h after neomycin treatment (37), likely as a con- sequence of the down-regulation of the cdk inhibitors at 1 h after neomycin treatment. Because we did not observe any signs of Wnt/ β -catenin signaling activation, we conclude that Wnt/ β -catenin signaling is not required for the reentry of support cells into the cell cycle immediately after hair cell death. In contrast, down- regulation of Notch and Fgf signaling correlates with the down- regulation of cdk inhibitors. It is possible that active Notch or Fgf signaling induces cdk inhibitors, causing cells to exit the cell cycle or become quiescent (66). However, manipulation of Notch signal- ing does not affect the cell cycle in chick or murine inner ears (13, 30), suggesting that signaling pathways other than Wnt/ β -catenin and Notch are involved in triggering or inhibiting proliferation.\nThe Jak1/Stat3 Pathway Is Up-Regulated Immediately After Neomycin Treatment. The Jak1/Stat3 pathway regulates a plethora of cel- lular processes, such as cell proliferation, cell differentiation, and\ncell survival (67, 68). The Jak1/Stat3 pathway is up-regulated during zebrafish inner ear regeneration and is required for proliferation of cardiomyocytes during zebrafish heart regeneration (36, 69). Like- wise, in zebrafish lateral line regeneration stat3 is up-regulated immediately, 1 h after neomycin treatment, as evidenced by in situ hybridization (Fig. 5 A and Fig. S5 A ). Other Jak1/Stat3 pathway members that are highly up-regulated are jak1 , interleukin 6 signal transducer ( il6st ), the stat inhibitor and target gene suppressor of cy- tokine signaling ( socs3b ), and the ligand interleukin 11b ( il11b ) (Fig. 2 A and Fig. S5 A ). In zebrafish heart regeneration, the proliferation of cardiomyocytes is regulated by the Jak1/Stat3 downstream target relaxin 3a ( rln3a ) . rln3a also is highly up-regulated in GFP + cells 3 h after neomycin treatment ( Fig. S5 A ) (69). Possibly, Jak1/Stat3 sig- naling regulates proliferation in support cells in the first few hours after hair cell death, whereas Wnt/ β -catenin signaling is required for proliferation during later stages of regeneration.\nAnother signaling pathway that is modulated during zebrafish hair cell regeneration and that might interact with the pathways described above is the Bmp pathway. Bmp signaling influences hair cell differentiation during chicken and mouse inner ear development (70 – 73). During zebrafish hair cell regeneration, bmp5 is up-regulated between 1 and 5 h after hair cell death (Fig. 5 B and Fig. S5 B ), whereas the BMP inhibitor noggin 3 ( nog3 ) is not down-regulated until 5 h after hair cell death ( Fig. S5 B ). Because ligand expression by itself is not a reliable indicator for signaling pathway activation, future studies will need to de- termine how the activation status of Bmp signaling is affected by bmp5 and nog3 .\nOther pathways that respond to zebrafish lateral line hair cell death are the Mapk ( Fig. S5 C and D ), TNF- α , ( Fig. S5 E ), insulin ( Fig. S5 F ), and apoptosis signaling ( Fig. S5 G ) pathways and nitric oxide (NO) and reactive oxygen species (ROS) ( Fig. S5 H ), some of which increase proliferation in undamaged chicken ves- tibular support cells (74). For example, we observed up-regulation of nox1 , noxo1a , and noxa1 ( Fig. S5 H and Dataset S2 ). NO is a radical that regulates a wide variety of physiological processes and is important during liver regeneration (75).\nWe summarized the activation status of some of the major signaling pathways based on the in situ hybridization patterns of pathway members in Fig. 5 C . The Wnt/ β -catenin pathway is not active in homeostatic or newly regenerating neuromasts, and the Notch and Fgf pathways are down-regulated during the first few hours of regeneration. The Wnt/ β -catenin and Notch pathways seem to play a role only during later stages of regeneration. The cell cycle is activated immediately, possibly via cytokine stimu- lation of the Jak1/Stat3 pathway.\nIndependent Microarray Studies Support the Differential Expression of Genes Not Associated with Key Signaling Pathways. An in- dependent expression analysis performed by Steiner et al. and reported in this issue (42), used a transgenic line with an alkaline phosphatase enhancer [ Tg ( -4.7alp:mCherry )] instead of the Tg ( sqET20 ) line we used to isolate mantle cells by FACS anal- ysis. We compared our respective datasets to determine how similar the data are. We first analyzed the overlap of genes enriched in untreated mantle/lateral line cells in both studies. The obtained results are a clear indicator of how similar the sorted cell populations in the two studies are. We chose the top 1,000 up- regulated genes from our Dataset S1 by p-value and then ex- amined how many of these genes also are among the top 1,000 genes identified by Steiner et al. (dataset S1 in ref. 42). If the datasets were independent and unrelated, we would expect 53 genes to overlap by chance. We identified 201 common genes with a p-value of 3.9e-107, demonstrating that our two datasets are significantly related. All six mantle cell-specific genes iden- tified by Steiner et al. (42) also show enrichment in our untreated GFP + cell population ( Table S4 A , blue rows). Ten of the 26 top up- and down-regulated genes that we validated by in situ hybridization\nafter hair cell loss ( Table S2 ) and for which we found a match in the Steiner dataset also show significant changes in the Steiner microarray dataset ( Table S4 B , blue rows) (42). Several of the genes that were not present in the Steiner dataset are expressed more highly by in situ hybridization in inner support cells than in mantle cells, reflecting the nature of the transgenic lines used and the type of cells purified. Nine of the 14 genes that Steiner et al. identified by microarray analysis, by RT-PCR, or by in situ hybridization (42) also showed the same trend in expression changes in our RNA-Seq analysis ( Table S4 C , blue rows).\nThe overlap between a subset of our data and that of Steiner et al. (42), combined with the identification and corresponding validation of genes in our study, strongly indicates that the two lines of investigation are complementary, with most of the tar- gets identified by Steiner et al. (42) found in the mantle cells and the majority of genes identified in our study expressed primarily in the inner support cells. As such, our respective studies both validate each other and help resolve the cell-type specificity of the revealed differential gene expressions.\n\n\nDiscussion\nSeveral studies have aimed to induce mammalian hair cell re- generation by inactivating or activating candidate genes and pathways. Knockdown of the cell cycle exit regulators Cdkn1b ( p27/Kip1 ), p19/Ink4d , and pRb led to the initiation of cell cycle reentry and the differentiation of hair cells in injured epithelia in mice (20, 21, 64, 76, 77). Unfortunately, these ectopic hair cells cannot be maintained and eventually apoptose. Because direct manipulation of cell cycle regulators has not been successful in reconstituting a fully functional sensory epithelium in mammals, it is important to identify the mitogens that activate cell cycle entry in regenerating organisms. It is plausible that manipu- lations of upstream signaling events are a more promising ave- nue to regenerate mammalian hair cells.\nMore recently, in mice the down-regulation of Notch signaling caused transdifferentiation of support cells into functional hair cells without inducing proliferation (30). However, the number of regenerated hair cells was modest, and it is unclear if the accompanied depletion of support cells affects the function of the partially regenerated sensory epithelium in the long term. Although these efforts have not resulted in the regeneration of a fully functional, morphologically normal epithelium, they clearly demonstrate that the mammalian cells of the inner ear can be manipulated and coaxed away from their terminally differenti- ated state.\nKnowledge gained from studying the development of hair cells has demonstrated that the underlying molecular mechanisms are complex, with a multitude of interacting signaling pathways and feedback loops that ensure that an unspecified progenitor cell transitions to a prosensory cell that ultimately differentiates into a functional sensory cell. We also have learned that signaling pathways are activated and inhibited repeatedly as hair cell progenitors are specified and differentiate, further complicating their experimental manipulation. Hence, a global up- or down- regulation of a particular pathway is unlikely to lead to fully functional sensory epithelia.\nWe hypothesize that the regeneration of a complete sensory epithelium will require the manipulation of several pathways in the correct order. However, the model systems currently used to study the regeneration of sensory epithelia present unique ob- stacles. For example, in chicken, antibiotic-induced hair cell death occurs over a 24-h period, and the proliferation of support cells and differentiation of hair cells occurs 1 – 4 wk after injury (19). This long time frame makes it difficult to determine the precise time point when pathways are turned on and off, because support cells will be in different stages of regeneration. Likewise, a previous expression study of the regenerating zebrafish ear was\nperformed 24 h after the larvae were exposed to noise that does not cause death simultaneously in all hair cells (36). Further adding to the complexity, genes are used repeatedly during hair cell development and can turn from an activator into a repressor (23). We have selected the zebrafish lateral line tissue to eluci- date the gene-expression patterns triggered by hair cell death. The zebrafish lateral line is particularly well suited for these analyses, because all hair cells can be killed acutely within 30 min, and regeneration occurs within 24 h (Fig. 3 A ), allowing us to determine the earliest signals and responses in support cells after hair cell death.\nWnt/ β -Catenin Signaling Is Not Detected Early in Lateral Line Hair Cell Regeneration. Our analyses focused on pathways involved in hair cell development as they might be reused during regeneration (15, 23, 43, 78, 79). For example, Wnt/ β -catenin signaling is crucial for the development of many organs across the animal kingdom (80). During sensory development, Wnt/ β -catenin sig- naling governs the proliferation of unspecified progenitor cells, renders them competent to acquire a neural cell fate, and sub- sequently is involved in the differentiation of these postmitotic prosensory cells (3, 23, 81). Wnt/ β -catenin signaling also is in- volved in regeneration. Wnt/ β -catenin signaling controls stem cell maintenance, quiescence, and proliferation during mam- malian epithelial hair cell, zebrafish fin, mammalian intestine, hydra head, and planarian regeneration (82 – 86). Importantly, Wnt/ β -catenin also is crucial for regulating proliferation in de- veloping and regenerating zebrafish neuromasts (26, 48).\nIn the mouse cochlea, ectopic Wnt/ β -catenin signaling pro- motes proliferation of Lgr5 + support cells via the down-regula- tion of Cdkn1c (p27/Kip1); however, this effect diminishes as the animals mature (3, 20, 28, 81, 87). Because Wnt/ β -catenin sig- naling is down-regulated in the mammalian cochlea after birth, it was hypothesized that the absence of Wnt/ β -catenin signaling is responsible for the inability of support cells to proliferate in response to hair cell death (3, 28, 29, 65). Therefore, recent studies have focused on up-regulating the Wnt/ β -catenin path- way in uninjured cochleae with the expectation of coaxing mammalian support cells into proliferating and producing hair cells (3, 17, 28, 29, 81). These experiments indeed yielded ad- ditional hair cells, but the epithelium was disorganized.\nSurprisingly, our data demonstrate that in zebrafish, as in adult mice cochleae, Wnt/ β -catenin signaling is inactive in ho- meostatic neuromasts and is up-regulated only at 12 h after neomycin treatment. Therefore Wnt/ β -catenin signaling is un- likely to play a role in the initial triggering of the regenerative response in zebrafish, even though it is sufficient and necessary to induce proliferation during later stages of regeneration, for example by inhibiting Cdkn1b (p27/Kip1) and regulating dyclinD1 (3, 27, 87).\nNotch Signaling Is Down-Regulated Early in Lateral Line Hair Cell Regeneration. A growing number of studies have explored how the manipulation of Notch signaling could aid in regenerating mammalian hair cells. It is well established that the inhibition of Notch signaling causes an increase in the number of hair cells in uninjured and injured chicken, guinea pig, and mouse utricles and cochleae and in injured zebrafish ears and lateral line neu- romasts (52, 53, 55 – 57, 88). The presence of Notch is detrimental to hair cell differentiation, because it blocks the proneural gene atoh1a and thereby maintains support cells (9, 13, 65, 89, 90). The ability of support cells to transdifferentiate into hair cells after down-regulation of Notch signaling was exploited recently to restore a modest amount of hair cells in the injured adult mouse cochlea that led to some promising functional recovery of hearing (30). This experimental down-regulation of Notch sig- naling recapitulates our observations during zebrafish lateral line hair cell regeneration. In zebrafish, Notch signaling is transiently\ndown-regulated 1 h after hair cell death, correlating with the attenuation of sox2 and up-regulation of atoh1a . The interactions between Notch signaling, sox2 , and atoh1a have not yet been tested in the zebrafish lateral line; however, because of their behavior during regeneration, it is very likely that their regula- tion is similar to that occuring during mouse, Xenopus , and zebrafish central nervous system or sensory organ development (23, 52, 61, 91 – 93). During murine ear development Sox2 ex- pression depends on Notch signaling and is required for the acquisition of neural potential in the prosensory epithelium (52). Subsequently, for neurogenesis to proceed, Sox2 is shut off via the accumulation of proneural genes, such as Atoh (93 – 97). In contrast, Sox2 is maintained in Notch + support cells, where it inhibits hair cell differentiation by inhibiting the proneural basic helix-loop-helix gene Atoh1a (91).\nIt is plausible that the down-regulation of Notch signaling in neuromasts is required during the earliest stages of hair cell regeneration to down-regulate sox2 in support cells, allowing these cells to acquire a proneural character via the up-regulation of atoh1a . Because Notch signaling needs to be reinitiated for fate specification of hair and support cells and control of organ size (9, 60), the Notch target genes her4.1 , notch3 , and sox2 are again up- regulated between 3 and 5 h after neomycin treatment (Fig. 4 A and B ). These genes likely are activated by the up-regulation of dld (60). Therefore, we hypothesize that the orchestration of genes downstream of Notch signaling appears to be the same during zebrafish hair cell development and regeneration.\nIt has not been appreciated previously that the down-regula- tion of Notch signaling could be a hallmark of spontaneously regenerating sensory epithelia, because it was reported that the Notch signaling pathway is up-regulated in response to hair cell death in chicken, guinea pigs, and zebrafish (9, 88, 98). However, these expression analyses were performed at later time points during the regeneration process, when Notch signaling is re- quired once again for hair cell versus support cell specification to occur. We also observed that zebrafish Notch target genes are reexpressed at 3 h after hair cell death (Fig. 4 A ). Because the zebrafish lateral line system allows us to kill hair cells very acutely, causing a relatively synchronized response in support cells, we were able to detect the immediate, transient down-regulation of Notch signaling after hair cell death. In the injured mammalian cochlea or utricle such drastic down-regulation of Notch target genes, such as Sox2 and several Hes and Hey genes, does not occur (55, 56); the absence of such down-regulation could be one of the reasons mammalian hair cells do not regenerate.\nThe Regulation of Notch Signaling in Combination with Mitogenic Signals May Offer Novel Therapeutic Strategies to Promote Hair Cell Regeneration in Mammals. Our data suggest that identifying the molecules that down-regulate Notch signaling in regenerat- ing species will be important in efforts to understand why mammals fail to regenerate injured hair cells. Our RNA-Seq analyses identified the transmembrane protein Numb as a po- tential candidate for inhibiting Notch signaling. Future studies should focus on the sufficiency of Numb to down-regulate Notch signaling in zebrafish and understanding Numb ’ s temporal ex- pression patterns during homeostasis and after injury in other regenerating species, as well as in mice. Numb could open a new therapeutic avenue for manipulating Notch signaling in mammals and facilitating successful regeneration of hair cells in mammals.\nEven though down-regulation of Notch signaling induced differentiation of new hair cells in mammals, these hair cells developed via transdifferentiation of support cells into hair cells, leading to the depletion of support cells (30). The down-regu- lation of Notch was not sufficient to induce support cell pro- liferation as observed in spontaneously regenerating species. Zebrafish regenerate hair cells mainly or exclusively by pro- liferation of support cells, and regenerating chicken hair cells\narise via transdifferentiation, as well as proliferation (19). In zebrafish we observe that the down-regulation of Notch signaling at 1 h after neomycin treatment correlates with the down-regu- lation of cyclin-dependent kinase inhibitors cdkn1b (p27/kip1) and cdkn1c ( p57/kip2 ) (Fig. 4 A and D ). This result suggests that Notch could be an activator of cdkn1b ( p27/kip1 ) that keeps cells out of the cell cycle and therefore that the down-regulation of Notch signaling could be a crucial event for the onset of re- generation. Indeed in the murine inner ear, the Notch target Sox2 regulates Cdkn1b (p27/Kip1), similar to activity described in the epidermis (99, 100) but in contrast to the activity in the intestinal crypt and inner ear, where Notch inhibits Cdkn1b (p27/ Kip1) (101, 102). In zebrafish the attenuation of Notch signaling has no effect on proliferation in uninjured sensory organs and has only a proproliferative effect on support cells after injury (9). In chicken cochleae the proliferation rate is not increased after injury, nor does inhibition of Notch cause cell cycle reentry in mice cochleae (13, 19, 30). Based on these findings it was con- cluded that Notch signaling is not required to initiate pro- liferation during regeneration (13). Therefore, it is possible that Cdk inhibitors are regulated by as yet unidentified, Notch- independent signals. Possibly, Fgf signaling inhibits proliferation, as demonstrated in the chicken inner ear (63).\nBecause complete regeneration of sensory epithelia in mam- mals requires coaxing support cells into hair cell differentiation as well as stimulating the support cells to re-enter the cell cycle and self-renew, manipulation of Notch signaling is not sufficient and needs to be married with manipulations of mitogenic signals. Furthermore, similar to our observations during zebrafish hair cell regeneration, Notch signaling must be reinitiated for the inhibition of proneural genes and specification of support cells (12, 23, 91). A temporally restricted Notch down-regulation or a combination of Notch down-regulation with subsequent ectopic expression of support cell inducers, such as Sox2, could be fruitful.\nOur efforts presented here have characterized the pattern of temporal activation of signaling pathways during zebrafish sensory hair cell regeneration. These results will aid in the design and in- terpretation of functional studies aimed at elucidating the gene-reg- ulatory network underlying zebrafish hair cell regeneration and can guide experiments to trigger hair cell regeneration in mammals.\n\n\nThis article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1402898111/-/DCSupplemental . www.pnas.org/cgi/doi/10.1073/pnas.1402898111 PNAS | Published online March 27, 2014 | E1383 – E1392 DEVELOPMENTA L BIOLOGY PNAS PLUS SEE CORRECTION FOR THIS ARTICLE\n\n\ncontrol 1h ptp4a2 lima1 junba abcf2a eif4a1a control 1h il11b nlrc3 hk2 (glulc) g3bp1 rapgef3 txn clic2 1h lad1 znf462 dfna5 gpr37b zgc:110353 slc35c2 tcea3 A B control zgc:152741 Fig. 2. Validation by in situ hybridization of a selection of 14 genes up- regulated ( A ) and seven genes down-regulated ( B ) at 1 h after neomycin treatment. ( Upper Rows ) Untreated larvae (control) at 5 dpf. ( Lower Rows ) Treated larvae.\n\n\nPNAS PLUS\n\n\nrbl2\n\n\nher4.1 notch3 1h 3h 5h 10h cdkn1c cdkn1b 1h 3h 5h 10h C A numb sox2 atoh1a dld pea3 fgf3 1h 3h 5h 10h D her4.1 atoh1b her6 numbl dlc gro2 ctbp2 notch3 cir1 jag2 aph1b ncor2 dld dla atoh1a numb cdk6 prkdc rbl2 mad2l2 cdkn1c cdkn1b ccna1 gadd45aa gadd45ba gadd45ab cdk7 plk1 ywhag1 pcna ccne1 anapc1 ywhai abl1 stag2 gadd45g mcm6l wee1 1h 3h 5h 1h 3h 5h 1h 3h 5h 1h 3h 5h B E control control control Notch cell cycle notch1b jag1a jag1b rbpjb sox2 GFP GFP GFP GFP 2 0 -2 2 0 -2 Fig. 4. Inhibition of Notch and Fgf pathways is accompanied with cell cycle reentry immediately after hair cell ablation. ( A , C , and D ) In situ hybridization of Notch pathway, Fgf pathway, and cell cycle genes in 5-dpf untreated and neomycin-treated larvae. ( A ) The Notch pathway inhibitor numb is increased at 1 h after neomycin. The Notch target genes her4.1 , notch3 , and sox2 are decreased at 1 h and up-regulated at 3 h after neomycin treatment. a toh1a is up- regulated starting at 1 h after neomycin, and its target dld is increased at 5 and 10 h. ( C ) fgf3 and its target pea3 are down-regulated at 1 and 3 h after neomycin treatment. ( D ) Cell cycle inhibitory genes cdkn1b , cdkn1c , and rbl2 are decreased at 1 and 3 h after neomycin compared with control. ( B and E ) Heat maps of selected Notch pathway genes and cell cycle genes.\n\n\nbmp5 control 1h 3h 5h 10h A stat3 1h 3h 5h 10h control B C Wnt Notch Fgf cell cycle Stat3 1h 3h 5h 10h 12h no change inhibition activation not tested\n\n\nPNAS PLUS\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nE1384 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1385 DEVELOPMENTA L\n\n\nE1386 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1387 DEVELOPMENTA L\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1389 DEVELOPMENTA L\n\n\nE1390 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\n1. White PM, Doetzlhofer A, Lee YS, Groves AK, Segil N (2006) Mammalian cochlear supporting cells can divide and trans-differentiate into hair cells. Nature 441(7096): 984 – 987.\n\n\n2. Corwin JT, Oberholtzer JC (1997) Fish n ’ chicks: Model recipes for hair-cell regeneration? Neuron 19(5):951 – 954.\n\n\n5. Williams JA, Holder N (2000) Cell turnover in neuromasts of zebrafish larvae. Hear Res 143(1-2):171 – 181.\n\n\n8. López-Schier H, Hudspeth AJ (2006) A two-step mechanism underlies the planar polarization of regenerating sensory hair cells. Proc Natl Acad Sci USA 103(49): 18615 – 18620.\n\n\n9. Ma EY, Rubel EW, Raible DW (2008) Notch signaling regulates the extent of hair cell regeneration in the zebrafish lateral line. J Neurosci 28(9):2261 – 2273. 10. Groves AK (2010) The challenge of hair cell regeneration. Exp Biol Med (Maywood) 235(4):434 – 446.\n\n\n27. Wada H, et al. (2013) Wnt/Dkk negative feedback regulates sensory organ size in zebrafish. Curr Biol 23(16):1559 – 1565.\n\n\n28. Shi F, Hu L, Edge AS (2013) Generation of hair cells in neonatal mice by β -catenin overexpression in Lgr5-positive cochlear progenitors. Proc Natl Acad Sci USA 110(34): 13851 – 13856.\n\n\n33. Whitfield TT (2002) Zebrafish as a model for hearing and deafness. J Neurobiol 53(2): 157 – 171.\n\n\n56. Korrapati S, Roux I, Glowatzki E, Doetzlhofer A (2013) Notch signaling limits sup- porting cell plasticity in the hair cell-damaged early postnatal murine cochlea. PLoS ONE 8(8):e73276.\n\n\n57. Lin V, et al. (2011) Inhibition of Notch activity promotes nonmitotic regeneration of hair cells in the adult mouse utricles. J Neurosci 31(43):15329 – 15339.\n\n\n58. McGill MA, Dho SE, Weinmaster G, McGlade CJ (2009) Numb regulates post-endo- cytic trafficking and degradation of Notch1. J Biol Chem 284(39):26427 – 26438.\n\n\n60. Matsuda M, Chitnis AB (2010) Atoh1a expression must be restricted by Notch sig- naling for effective morphogenesis of the posterior lateral line primordium in ze- brafish. Development 137(20):3477 – 3487.\n\n\n63. Oesterle EC, Bhave SA, Coltrera MD (2000) Basic fibroblast growth factor inhibits cell proliferation in cultured avian inner ear sensory epithelia. J Comp Neurol 424(2): 307 – 326.\n\n\n69. Fang Y, et al. (2013) Translational profiling of cardiomyocytes identifies an early Jak1/Stat3 injury response required for zebrafish heart regeneration. Proc Natl Acad Sci USA 110(33):13416 – 13421.\n\n\n74. Oesterle EC, Tsue TT, Rubel EW (1997) Induction of cell proliferation in avian inner ear sensory epithelia by insulin-like growth factor-I and insulin. J Comp Neurol 380(2): 262 – 274.\n\n\n75. Carnovale CE, Ronco MT (2012) Role of nitric oxide in liver regeneration. Ann Hepatol 11(5):636 – 647.\n\n\n76. Sage C, et al. (2005) Proliferation of functional hair cells in vivo in the absence of the retinoblastoma protein. Science 307(5712):1114 – 1118.\n\n\n77. Sage C, et al. (2006) Essential role of retinoblastoma protein in mammalian hair cell development and hearing. Proc Natl Acad Sci USA 103(19):7345 – 7350.\n\n\n80. van Amerongen R, Nusse R (2009) Towards an integrated view of Wnt signaling in development. Development 136(19):3205 – 3214.\n\n\n81. Shi F, Kempfle JS, Edge AS (2012) Wnt-responsive Lgr5-expressing stem cells are hair cell progenitors in the cochlea. J Neurosci 32(28):9639 – 9648.\n\n\n82. Galliot B, Chera S (2010) The Hydra model: Disclosing an apoptosis-driven generator of Wnt-based regeneration. Trends Cell Biol 20(9):514 – 523.\n\n\n83. Fleming HE, et al. (2008) Wnt signaling in the niche enforces hematopoietic stem cell quiescence and is necessary to preserve self-renewal in vivo. Cell Stem Cell 2(3): 274 – 283. 84. Li L, Bhatia R (2011) Stem cell quiescence. Clin Cancer Res 17(15):4936 – 4941.\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Gene-expression analysis of hair cell regeneration in the zebrafish lateral line\nLinjia Jiang a , Andres Romero-Carvajal a,b , Jeff S. Haug a , Christopher W. Seidel a , and Tatjana Piotrowski a,1\na Stowers Institute for Medical Research, Kansas City, MO 64110; and b Department of Neurobiology and Anatomy, University of Utah Medical School, Salt Lake City, UT 84132\nEdited* by Yuh Nung Jan, Howard Hughes Medical Institute, University of California, San Francisco, CA, and approved March 4, 2014 (received for review February 16, 2014)\nDeafness caused by the terminal loss of inner ear hair cells is one of the most common sensory diseases. However, nonmammalian animals (e.g., birds, amphibians, and fish) regenerate damaged hair cells. To understand better the reasons underpinning such disparities in regeneration among vertebrates, we set out to define at high resolution the changes in gene expression associ- ated with the regeneration of hair cells in the zebrafish lateral line. We performed RNA-Seq analyses on regenerating support cells purified by FACS. The resulting expression data were subjected to pathway enrichment analyses, and the differentially expressed genes were validated in vivo via whole-mount in situ hybrid- izations. We discovered that cell cycle regulators are expressed hours before the activation of Wnt/ β -catenin signaling following hair cell death. We propose that Wnt/ β -catenin signaling is not involved in regulating the onset of proliferation but governs pro- liferation at later stages of regeneration. In addition, and in marked contrast to mammals, our data clearly indicate that the Notch pathway is significantly down-regulated shortly after in- jury, thus uncovering a key difference between the zebrafish and mammalian responses to hair cell injury. Taken together, our findings lay the foundation for identifying differences in sig- naling pathway regulation that could be exploited as potential therapeutic targets to promote either sensory epithelium or hair cell regeneration in mammals.\nsignaling pathway analysis | RNA sequencing | neuromast | Jak/Stat3 | cdkn1b\nH earing disorders are the most common sensory disabilities in humans and can be either inherited or acquired. De- generation and death of sensory hair cells in the inner ear is the underlying cause of age-related, noise-, antibiotic-, or chemo- therapy drug-induced hearing loss ( www.nidcd.nih.gov ). Such loss is permanent, because mammalian inner ear hair cells do not regenerate. The inability of mammals to restore sensory hair cells after injury is intriguing given that ( i ) mammalian vestibular support cells exhibit low levels of proliferation after injury; ( ii ) cochlear support cells of newborn mice are able to proliferate subsequent to hair cell death in cell culture (1); and ( iii ) other vertebrates such as birds, amphibians, and fish replace their sensory hair cells continuously and, importantly, are capable of regenerating hair cells after damage (2 – 10).\nWork in birds has shed light on the mechanisms regulating sensory hair cell regeneration in vertebrates (2, 4, 11 – 18). For instance, during sensory hair cell regeneration in the chick, the majority of hair cells arise by proliferation of neighboring sup- port cells (19). Given that mammalian support cells do not re- spond to hair cell death by proliferating, molecular studies in chick have focused on identifying the mitogenic signals as well as the target genes that induce re-entering of the cell cycle in support cells (11, 20 – 22). For example, the transcription factors Atoh1 and Sox2 and the Wnt/ β -catenin and Notch pathways have been shown to play crucial roles (23), and gene-expression analyses using microarrays have identified hundreds of genes involved in chick hair cell regeneration (18, 24).\nThe Wnt/ β -catenin pathway is used repeatedly during hair cell development, and recent studies have emphasized the importance of Wnt/ β -catenin signaling in modulating the proliferation of sup- port cells. For example, activated β -catenin causes proliferation in dissociated chicken utricular epithelia and ectopic expression of hair cell differentiation in the chicken basilar papilla (17, 25). Likewise, during mouse cochlear development Wnt/ β -catenin sig- naling activates proliferation of support cells and is required sub- sequently for their differentiation into hair cells (3, 26 – 29). Perturbing the signaling pathways known to be activated during chick inner ear regeneration therefore may provide an opportunity to modulate regenerative capacities in the mam- malian inner ear. For instance, it has been shown recently that down-regulation of Notch signaling in adult mice after injury induces the transdifferentiation of support cells into hair cells and can partially restore hearing (30). However, this manipula- tion causes the loss of support cells, and the implications of this loss for long-term hearing restoration are unknown.\nSignificant progress has been made in the study of sensory hair regeneration. However, in part because of the complexity of spatial and temporal gene interactions and signaling dynamics, our knowl- edge still is too incomplete to enable us to promote the generation of a functional epithelium after injury in mammals. The zebrafish lateral line provides an experimentally accessible system to define the complex signaling events triggered by injury and regeneration, because these cells can be killed acutely and then regenerate rap- idly (31). These characteristics allow the detection of changes in gene expression at defined time points following hair cell death.\nSignificance\nDeafness is caused largely by the death of sensory hair cells in the inner ear. In contrast to nonmammalian vertebrates, hu- man hair cells do not regenerate. Understanding the mecha- nisms that regulate hair cell regeneration in zebrafish may shed light on the factors that prevent hair cell regeneration in mammals. RNA-Seq analysis of regenerating sensory organs uncovered dynamic changes in the expression of signaling pathways during zebrafish hair cell regeneration. Unexpect- edly, the Wnt/ β -catenin, Notch, and Fgf pathways are down- regulated following hair cell death, whereas the Jak1/Stat3 pathway and cell cycle are activated. We propose that mim- icking the zebrafish activation status of a combination of path- ways at the correct time points in mammals may improve the chances of triggering regeneration of functional hair cells.\nAuthor contributions: L.J. and T.P. designed research; L.J., A.R.-C., and J.S.H. performed research; L.J., J.S.H., C.W.S., and T.P. analyzed data; and L.J. and T.P. wrote the paper.\nThe authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nData deposition: The data reported in this paper have been deposited in the Gene Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE56176 ). 1 To whom correspondence should be addressed. E-mail: pio@stowers.org.\nZebrafish and the majority of all other aquatic vertebrates possess hair cells not only in their vestibular system in the ear but also in their skin. These hair cells are part of the sensory lateral line system. The lateral line system senses water motion and initiates the appropriate behavioral response for capturing prey, avoiding predators, and schooling. The morphology and function of sensory hair cells are evolutionarily conserved from fish to mammals (32, 33). Lateral line hair cells are located in the center of a mechanosensory organ known as the neuromast and are surrounded by inner support cells and an outer ring of mantle cells. Importantly, despite the unusual location of the hair cells on the trunk, lateral line and ear hair cells develop and differ- entiate by similar developmental mechanisms. For example, mutations in genes causing deafness in humans also disrupt hair cell function in the zebrafish lateral line and vestibular system (34). Because of its genetic tractability and the accessibility of sensory organs to experimental manipulation and visualization, the zebrafish recently has been recognized as an excellent model for discovering and functionally characterizing genes crucial for hair cell development and regeneration (16, 19, 35, 36).\nIn zebrafish hair cell regeneration occurs via the proliferation and differentiation of inner support cells (5, 6, 9, 37, 38), but detailed lineage analyses have not yet been performed, and the stem cell population has not been identified. Mantle cells are good stem cell candidates, because they migrate to form neu- romasts on regenerating tail tips and also form postembryonic neuromasts by a process called “ stitching. ” However, after chemical ablation of hair cells by antibiotic treatment, mantle cells proliferate much less frequently than inner support cells (9). Possibly, mantle cells replenish inner support cells that then differentiate into hair cells. Mantle cells have not been shown to differentiate directly into hair cells (37, 38). Therefore, both mantle and inner support cells seem to be involved in reestablishing proper neuromast morphology after hair cell death.\nTo identify the activation status of signaling pathways during the first few hours of regeneration, we performed RNA-Seq gene expression analyses of zebrafish mantle and inner support cells isolated by FACS. We discovered that the Notch pathway is transiently down-regulated immediately after hair cell death; to our knowledge, this down-regulation has not been previously reported in other studies of regenerating species. Our studies also demonstrate that Wnt/ β -catenin signaling is not among the earliest pathways to respond to hair cell death but instead is up- regulated relatively late during the regenerative response. These results suggest that Wnt/ β -catenin signaling does not trigger proliferation by down-regulating cyclin-dependent kinase inhib- itors immediately after hair cell death but rather regulates pro- liferation at later stages of regeneration. Our findings have defined the temporal regulation of key signaling pathways asso- ciated with the early stages of hair cell regeneration and suggest that the successful initiation of mammalian hair cell regeneration, either in vivo or in vitro, might require precise temporal manip- ulation of these pathways.\n\n\nResults\nIsolation of Mantle and Inner Support Cells by FACS. To identify changes in gene expression in mantle and inner support cells after hair cell death, we isolated GFP + cells from regenerating and control Tg ( sqET20 ) transgenic zebrafish via FACS (Fig. 1 A and B ) (39). Tg ( sqET20 ) larvae express GFP strongly in mantle cells and to a lesser degree in inner support cells (Fig. 1 B ). Hair cells are GFP − and therefore are excluded from the GFP + population. We isolated GFP − and GFP + cells from Tg ( sqET20 ) control larvae at 5 days postfertilization (dpf) and from neo- mycin-treated larvae at 1, 3, and 5 h after neomycin treatment. Larvae were dissociated into a single-cell suspension in a trypsin solution and stained with 7-aminoactinomycin D (7-AAD) and Hoechst 33342 to exclude dead cells and cell debris (Fig. 1 C ). Live GFP + cells (7-AAD − , Hoechst + , GFP + cells) and control GFP − cells (7-AAD − , Hoechst + , GFP − cells) were sorted by FACS (Fig. 1 D ). The sorting yielded highly GFP + -enriched cell populations, as demonstrated by a postsort reanalysis of the collected cells (Fig. 1 E and F ). GFP + -sorted cells showed 94.4 ± 1.3% enrichment ( n = 7). To confirm that GFP + cells are lateral line cells, we performed immunostaining with an antibody against the support cell marker Sox2 that labels inner support cells as well as some mantle cells (Fig. 1 G and Fig. S1 ) (40). Counts of Sox2 + FACS-purified cells revealed that 95% of GFP + cells but only 6.6% of GFP − cells were positive for Sox2 (Fig. 1 H ). Ad- ditionally, we performed RT-PCR for genes highly expressed in neuromast cell types and with limited expression in other tissues. RT-PCR for gfp , eya1 , klf4b , and cldnb showed that these genes are expressed at significantly higher levels in GFP + than in GFP − cells ( Fig. S2 A – D ). In contrast, genes expressed in tissues outside the lateral line system, such as rhodopsin ( rho , eye), vil- lin1 ( vil1 , gut), and myod1 (muscle), are expressed at much lower levels in GFP + cells ( Fig. S2 E – G ). In Tg ( sqET20 ) larvae, mantle B A G K Sox2    cells (%) 0 20 40 60 80 100 H lateral line genes neomycin induced changes J Tg(sqET20) mantle cell inner support cell GFP GFP GFP 1h Sox2 GFP DAPI GFP GFP GFP control neo control neo control 30min 1h 3h 5h 30min 1h 3h 5h GFP 3h GFP 5h GFP 1h GFP 3h GFP 5h GFP changes in mantle and support cells C D 97.8% FSC FSC 0 1000 2000 3000 4000 FSC 100 101 2 103 10 100 101 10 2 103 104 Hoechst GFP GFP 94.3% live cells 10 4 E F GFP GFP 100 101 2 103 10 10 4 100 101 2 103 10 10 4 0 1000 2000 3000 4000 0 1000 2000 3000 4000 100 101 2 103 10 10 4 I 7-AAD GFP GFP GFP GFP\nFig. 1. Purification of mantle and inner support cells from Tg ( sqET20 ). ( A ) In 5-dpf Tg ( sqET20 ) larvae GFP is expressed in neuromasts (arrows) and in the interneuromast cells that connect adjacent neuromasts. ( B ) Magnification of a neuromast shows that mantle cells are strongly labeled by GFP, but inner support cells are weakly labeled. ( C and D ) Dissociated cells of Tg ( sqET20 ) larvae at 5 dpf were FACS sorted using a two-gate sorting strategy. Live cells were gated by 7-AAD − and Hoechst + ( C ) and subsequently were gated by forward scatter (FSC) and GFP intensity to isolate GFP − and GFP + cells ( D ). ( E and F ) A postsorting experiment shows that GFP − cells are 97.8% pure ( E ) and GFP + cells are 94.3% pure ( F ). ( G ) Sorted GFP − and GFP + cells were stained with Sox2 antibody and DAPI. ( Upper ) A GFP − cell is Sox2 − . ( Lower ) A GFP + cell expresses Sox2. (Scale bars: 5 μ m.) ( H ) Quantification of Sox2 + cells in each population as mean ± SD; * P &lt; 0.01 by t test. ( I ) Regeneration- induced mantle and inner support cell genes were identified at each time point by comparing GFP + neomycin-treated cells with GFP + untreated samples. ( J ) Neomycin-induced genes were identified by comparing GFP − neomycin-treated cells at each time point with GFP − untreated samples. Neomycin-induced genes subsequently were subtracted from the gene lists obtained with GFP + cells to select for regeneration-specific changes in gene expression in inner support and mantle cells. ( K ) Comparisons between GFP + and GFP − cells of untreated larvae generated a list of genes enriched in lateral line cells.\ncells express GFP most strongly, but inner support cells respond more robustly to hair cell death by proliferating (5, 6). Because Tg ( sqET20 ) larvae show weaker GFP expression in inner support cells, we were concerned that our cell sorts would exclude inner support cells. However, RT-PCR with inner support cell markers such as klf4b showed that inner support cells were included in the GFP + cell populations ( Fig. S2 C ). Combined, these studies dem- onstrated that our FACS protocol yielded a highly enriched pool of lateral line cells containing mantle and inner support cells but not hair cells.\nTo investigate the temporal activation or repression of in- dividual genes and signaling pathways in GFP + mantle and inner support cells during the first few hours of regeneration, we ex- amined GFP + and GFP − cells at 1, 3, and 5 h after a 30-min treatment with neomycin (Fig. 1 I ). Sorts for each time point were performed in triplicate and subjected to RNA-Seq analysis. Bacte- ricidal antibiotics cause oxidative stress in mammalian cells, leading us to speculate that sublethal concentrations of neomycin may cause changes in gene expression unrelated to hair cell death (41). To control for this nonspecific effect, we examined gene expression in GFP − cells in the presence or absence of neomycin (Fig. 1 J ). Al- though this population includes all larval nonlateral cells, it allows us to control for a nonspecific neomycin effect when comparing GFP + cells from treated and control larvae.\nAnalysis of the RNA-Seq Results. Sequencing libraries were pre- pared from mRNAs of collected cells and were quality checked on a Bioanalyzer. We performed 50-bp single-end RNA-Seq using the Illumina platform. To test the enrichment of lateral line genes further, we examined the fragments per kilobase of transcript per million mapped reads (FPKM) values for known lateral line genes and for genes expressed in other tissues ( Fig. S3 ). The results confirmed the RT-PCR analysis described above. Pan lateral line, as well as inner support cell, markers show much higher FPKM values in FACS-sorted GFP + cells than in GFP − cells. As suggested by the RT-PCR results, the FPKM values for the inner support cell markers pea3 and klf4b are significantly higher in the GFP + than in the GFP − cells, confirming that inner support cells were included in our FACS sorts. This enrichment of lateral line genes is supported by the information in Dataset S1 , in which neuromast-specific genes were identified by comparing the expression profiles of untreated GFP + and GFP − cells. In addition to known lateral line genes, the resulting lists of differentially expressed genes provide a valuable resource of as yet uncharacterized genes that potentially could play important roles in hair cell development and/or re- generation (Fig. 1 K and Dataset S1 ). The number of genes enriched in GFP + cells relative to GFP − controls at an adjusted p-value ≤ 0.05 is 1,670 ( SI Materials and Methods and Dataset S1 ). This dataset also contains many of the genes reported in the dataset of Steiner et al. (42), who identified mantle cell-specific genes using a different transgenic line (see below).\nGene Identification for Each Time Point. To identify genes from transcripts specifically enriched or depleted in GFP + mantle and inner support cells after hair cell death, we created several comparisons between the RNA-Seq datasets. Ratios of gene expression were created between the neomycin-treated GFP + cells at 1, 3, and 5 h and the nontreated GFP + cells at 1 h to identify genes responding to hair cell death (Fig. 1 I ). To control for a nonspecific neomycin response, a similar set of ratios was created between the neomycin-treated GFP − cells at each time point and the untreated GFP − cells at 1 h (Fig. 1 J ). The neomycin- induced genes were subtracted from all gene lists generated with GFP + cells. We then used ratios and P values between datasets to select genes of interest at any given time point ( SI Materials and Methods and Dataset S2 ). Genes identified by these criteria are marked with a numeric flag with positive numbers indicating up-\nregulated and negative numbers indicating down-regulated genes. The numeric value indicates the time point at which a gene is up- or down-regulated ( Dataset S2 , flagged column). A principal component analysis of the biological replicates of GFP − and GFP + cell populations at the three different time points demonstrates that GFP + cells are very different from GFP − cells. In addition, cell sorts performed in triplicate for each time point are highly re- producible ( Fig. S4 ). To define a set of the top 100 up- and down-regulated genes to use as candidates for validation, we ranked 193 up-regulated and 200 down-regulated significant (flagged) genes from the 1-h dataset as a function of the ratio and general abundance ( SI Materials and Methods and Table S1 ). We validated the RNA- Seq results by performing in situ hybridizations with 28 up-regulated and 21 down-regulated genes selected from the top 100 gene list in larvae 1 h after neomycin treatment ( Table S2 ). All 28 up- regulated genes are expressed in the lateral line, and 20 of these genes show up-regulation by in situ hybridization after neomycin treatment. Of the 21 down-regulated genes, 19 are expressed in the lateral line, and 12 genes are detectably down-regulated by in situ hybridization (Fig. 2 and Table S2 ). These experiments de- monstrated that the FACS sorting followed by RNA-Seq analysis produced high-quality results that enable us to study hair cell regeneration in zebrafish in detail.\nPathway Analysis at 1, 3, and 5 h After Hair Cell Death. The Wnt/ β -catenin, Notch, Jak/Stat, Bmp, and Fgf pathways are important for hair cell development as well as for the regeneration of hair cells in fish and chick epithelia (15, 23, 43). Our understanding of which pathways act upstream and how these pathways affect each other is still very limited. We performed a temporal ex- pression analysis at 1, 3, and 5 h after hair cell death to de- termine when particular pathways are activated or inhibited; this temporal information provides clues into how they might interact with each other. We also aimed to identify other signaling path- ways and record any dynamic changes in their behavior that could indicate how they integrate with the Notch and Wnt/ β -catenin signaling pathways. To identify enriched pathways in our datasets, we used the signaling pathway impact analysis (SPIA) package in\nccdc48\nR (44). This software tool not only takes into account how many genes belong to a pathway but also focuses on the importance of a gene by analyzing the perturbation of the pathway if the gene expression changes. Therefore, even pathways that are repre- sented by only a small number of genes can be classified as en- riched if these genes are important regulators of the pathway. For the SPIA analysis we included all flagged genes (flagged column) listed in Dataset S2 . For each time point the up- and down-regulated genes were combined and analyzed against a panel of pathways from zebrafish defined in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (45). The 1-h postneomycin treatment time point contained 509 genes from the National Center for Bio- technology Information Entrez database. The 3-h postneomycin treatment time point contained 401 Entrez genes, and the 5-h postneomycin treatment time point contained 418 genes.\nThe results of the SPIA analysis are shown in Table S3 . Au- tomated pathway annotation is powerful but subject to caveats, because genes can be annotated to function in several pathways, some of which might not be active in the lateral line. In addition, in the KEGG database, the Wnt pathway encompasses the ca- nonical Wnt/ β -catenin, the PCP, and the Wnt- regulated calcium pathways, potentially providing erroneous status assignment of the Wnt/ β -catenin pathway. We therefore tested the SPIA ana- lysis by in situ hybridization experiments with pathway members at various time points of regeneration (see below and Figs. 3 – 5). For improved visualization we generated heat maps of log2 ratio values of KEGG pathway members. We show only the genes that were flagged in our analysis and reported to be expressed in the lateral line by the zebrafish information network ( www.zfin.org ) or prior reports (46, 47).\nThe Wnt/ β -Catenin Pathway Is Not Activated During Early Stages of Lateral Line Hair Cell Regeneration. The analysis of hair cell re- generation in a Tg ( sqEt20;sqEt4 ) double-transgenic larva allows us to draw comparisons between gene expression and cellular changes at particular time points (Fig. 3 A ) (39). At 1 h after neomycin treatment almost all hair cells have been eliminated. Two hair cells remain that were too immature to be killed by neomycin. Proliferation of inner support cells begins within the first 2 h of regeneration (37). At 5 h two newly differentiating hair cells start to express GFP (Fig. 3 A ; arrows), and by 12 h after neomycin treatment six hair cells are present.\nWnt/ β -catenin signaling is required for proliferation in de- veloping and regenerating neuromasts (26, 48). Unexpectedly, the SPIA pathway analysis suggested that the Wnt/ β -catenin pathway is inactive at 1 h after hair cell death ( Table S3 ). To investigate this finding in more detail, we performed in situ hy- bridization experiments with Wnt/ β -catenin pathway genes in untreated and treated larvae at various time points after hair cell death. Our analysis showed that Wnt/ β -catenin signaling is not active in 5-dpf untreated control neuromasts. The ligand wnt10a is not expressed in 5-dpf control neuromasts, and we failed to detect any expression of dgfp in the Tg ( Tcf/Lef-miniP:dGFP ) Wnt/ β -catenin reporter line (Fig. 3 B ) (49). Even though Wnt/ β -catenin signaling is inactive, the β -catenin cotranscription fac- tor tcf7l2 ( tcf4 ) is expressed. Likewise, the Wnt receptors fzd7b and 8a are present, although whether they act as receptors for canonical or noncanonical Wnt signaling in neuromasts is un- known (Fig. 3 B ) (50, 51). At 1 h after neomycin treatment tcf7l2 ( tcf4 ), frz7b , and fzd8a are drastically down-regulated, and, along with wnt10a , their tcf4 wnt10a dgfp control 1h 3h 5h 10h control 1h 5h 12h 18h fzd8a fzd7b lef1 fzd7b fzd8a wif1 dkk1b tcf7l1a tcf7l2 fzd9b ctbp2 axin2 lef1 wnt8a csnk1a1 jun sfrp1a wnt10a cul1a siah1 gro2 sox17 tbl1x fbxw11b gsk3b csnk1a1 1h 3h 5h 1h 3h 5h fzd6 dkk2 control 1h 3h 5h 10h 12h 18h 24h GFP GFP Wnt 2 0 -2 A B C Fig. 3. The Wnt/ β -catenin pathway is inactive during the early stages of hair cell regeneration. ( A ) Still images of a Tg ( sqET20;sqET4 ) larval neuromast in the process of neomycin-induced regeneration. All hair cells, except for two immature hair cells, were killed by neomycin by 1 h after neomycin treatment. Two newly formed hair cells (arrows) start to express GFP at 5 h, and other pairs of hair cells (arrows) appear at 12 and 18 h. ( B ) In situ hybridization of Wnt/ β -catenin pathway genes in 5-dpf untreated (control) larvae and in larvae at different time points after neomycin treatment. Expression of wnt10a is in- creased at 3 and 5 h. Expression of tcf7l2 ( tcf4 ), fzd7b , and fzd8a is largely decreased at 1 h after neomycin. The Wnt/ β -catenin reporter line Tg ( Tcf/Lef-miniP: dGFP ) demonstrates activation starting at 12 h as shown by dgfp expression, but lef1 is not induced in regenerating neuromasts. ( C ) Heat map of selected Wnt/ β -catenin pathway genes based on RNA-Seq results. Log2 ratios of GFP − and GFP + cells at 1, 3, and 5 h after neomycin treatment are color coded (red: up- regulation, blue: down-regulation). Genes that are modulated in GFP − cells are neomycin-induced genes.\nexpression in the Wnt/ β -catenin Tg ( Tcf/Lef-miniP:dGFP ) reporter line remains low or undetectable. At 3 h after neomycin treatment wnt10a , tcf7l2 ( tcf4 ) and fzd7b , and fzd8a are up-regulated, leading to Wnt/ β -catenin pathway activation at 12 h as demonstrated by dgfp expression in the Tg ( Tcf/Lef-miniP:dGFP ) reporter line (Fig. 3 B ). The activation status of other Wnt/ β -catenin pathway genes at the different time points is shown in Fig. 3 C . Genes that change expression in the GFP − cells represent neomycin-induced genes. Our analyses reveal that Wnt/ β -catenin signaling is a late- responding pathway and does not play a role in the initiation of regeneration.\nThe Notch Pathway Is Inhibited 1 h After Neomycin Treatment. The Notch pathway is of particular interest because experimental down-regulation of Notch induces transdifferentiation of mam- malian support cells into hair cells (30, 52 – 57). In contrast to previous reports in zebrafish and chicken (9, 13), our SPIA pathway analysis suggested that the Notch pathway is inhibited immediately after hair cell death because of simultaneous up- regulation of numb (Fig. 4 A and B ). Numb inhibits Notch ac- tivity by regulating postendocytic sorting events and degradation of the Notch receptor and thereby regulates neural stem cell homeostasis (58, 59). However, because Numb is a multifunc- tional membrane-associated protein with additional functions\nindependent of the regulation of Notch signaling, we tested if the Notch pathway is indeed down-regulated by in situ hybridization of Notch pathway members. Our analysis of the Notch target genes her4.1 and notch3 confirms that Notch signaling is attenuated immediately 1 h after neomycin treatment and is accompanied\nFig. 5. The Jak1/Stat3 pathway is activated during the early stages of hair cell regeneration. ( A ) In situ hybridization shows that stat3 is increased at 1 and 3 h after neomycin treatment. ( B ) bmp5 is up-regulated at 1 h after neomycin treatment in situ. ( C ) Table depicting the status of signaling path- ways at different time points after hair cell death. All comparisons (no change, inhibition, or activation) were evaluated relative to untreated control samples. The Wnt, Notch, and Fgf pathways are inhibited immediately after neomycin treatment, whereas the cell cycle and Jak1/Stat3 pathways are activated during the early stages of regeneration. The Wnt and Notch pathways are reactivated at 12 h and 5 h after neomycin treatment, respectively.\nby down-regulation of sox2 (Fig. 4 A and B ) and up-regulation of atoh1a (Fig. 4 A and B ). The atoh1a target deltaD ( dld ) is up- regulated between 3 – 5 h after neomycin treatment, reflecting the appearance of hair cell precursors (Fig. 4 A and B ) (60). Notch signaling must be reinitiated for proper fate specification into hair and support cells and proper organ size control (9, 60). Accordingly, between 3 and 5 h after neomycin treatment (Fig. 4 A and B ), we observed renewed activation of the Notch target genes her4.1 , notch3 , and sox2 that likely is caused by the up- regulation of dld (60).\nWe postulate that down-regulation of Notch signaling is a characteristic event during hair cell regeneration in zebrafish. In contrast to zebrafish, the Notch target genes are decreased only modestly, and Notch signaling is still active, in hair cell-depleted neonatal mouse cochleae and 3- to 4-wk-old mouse utricles (55, 56, 61). Therefore we hypothesize that the failure of transient Notch down-regulation in mammals could be one of the reasons that mammalian hair cells do not regenerate spontaneously after injury.\nFgf Signaling Is Down-Regulated Immediately After Hair Cell Death. The Fgf pathway initiates dld and subsequently Notch signaling in the mouse cochlea, chicken inner ear, and zebrafish lateral line neuromasts (23, 60). Both pathways are active in 5-dpf un- treated neuromasts (Fig. 4 A and C and Fig. S5 D ). Identically to Notch pathway members, fgf3 and its transcriptional target pea3 are turned off 1 h after neomycin treatment. However, the Notch target her4.1 is reactivated at 3 h, whereas pea3 is not transcribed until 5 h (Fig. 4 A and C ). Therefore, it is possible that Notch signaling does not depend on Fgf signaling during regeneration; however, this hypothesis needs to be tested. The rapid down- regulation of Fgf signaling is in accordance with findings in chicken sensory epithelia, in which Fgfr3 is highly expressed in support cells but is down-regulated rapidly after hair cell death (62). Additionally, in the chicken down-regulation of Fgf sig- naling is required for regeneration to occur, because the addition of Fgf inhibits support cell division after damage (63).\nThe Cell Cycle Exit Genes cdkn1b , cdkn1c , and rbl2 Are Down- Regulated at 1 h After Hair Cell Death. The failure to down-regu- late cell cycle exit regulators is one of the reasons mammalian support cells do not proliferate in response to hair cell death (10, 12, 64, 65). Determining the earliest changes in the expression of cell cycle regulators and mapping correlating changes in signaling pathway activations will aid in identifying candidate pathways in- volved in cell cycle reentry. We detected the earliest changes in cell cycle genes at 1 h after neomycin treatment. The cyclin- dependent kinase (cdk) inhibitors cdkn1b ( p27/kip1 ) and cdkn1c ( p57/kip2 ), as well as the pocket domain-containing rbl2 , are down-regulated 1 h after neomycin treatment and are not up- regulated again until 5 – 10 h after neomycin treatment (Fig. 4 D and E ). Time-lapse analyses revealed that support cells enter the cell cycle 1 – 5 h after neomycin treatment (37), likely as a con- sequence of the down-regulation of the cdk inhibitors at 1 h after neomycin treatment. Because we did not observe any signs of Wnt/ β -catenin signaling activation, we conclude that Wnt/ β -catenin signaling is not required for the reentry of support cells into the cell cycle immediately after hair cell death. In contrast, down- regulation of Notch and Fgf signaling correlates with the down- regulation of cdk inhibitors. It is possible that active Notch or Fgf signaling induces cdk inhibitors, causing cells to exit the cell cycle or become quiescent (66). However, manipulation of Notch signal- ing does not affect the cell cycle in chick or murine inner ears (13, 30), suggesting that signaling pathways other than Wnt/ β -catenin and Notch are involved in triggering or inhibiting proliferation.\nThe Jak1/Stat3 Pathway Is Up-Regulated Immediately After Neomycin Treatment. The Jak1/Stat3 pathway regulates a plethora of cel- lular processes, such as cell proliferation, cell differentiation, and\ncell survival (67, 68). The Jak1/Stat3 pathway is up-regulated during zebrafish inner ear regeneration and is required for proliferation of cardiomyocytes during zebrafish heart regeneration (36, 69). Like- wise, in zebrafish lateral line regeneration stat3 is up-regulated immediately, 1 h after neomycin treatment, as evidenced by in situ hybridization (Fig. 5 A and Fig. S5 A ). Other Jak1/Stat3 pathway members that are highly up-regulated are jak1 , interleukin 6 signal transducer ( il6st ), the stat inhibitor and target gene suppressor of cy- tokine signaling ( socs3b ), and the ligand interleukin 11b ( il11b ) (Fig. 2 A and Fig. S5 A ). In zebrafish heart regeneration, the proliferation of cardiomyocytes is regulated by the Jak1/Stat3 downstream target relaxin 3a ( rln3a ) . rln3a also is highly up-regulated in GFP + cells 3 h after neomycin treatment ( Fig. S5 A ) (69). Possibly, Jak1/Stat3 sig- naling regulates proliferation in support cells in the first few hours after hair cell death, whereas Wnt/ β -catenin signaling is required for proliferation during later stages of regeneration.\nAnother signaling pathway that is modulated during zebrafish hair cell regeneration and that might interact with the pathways described above is the Bmp pathway. Bmp signaling influences hair cell differentiation during chicken and mouse inner ear development (70 – 73). During zebrafish hair cell regeneration, bmp5 is up-regulated between 1 and 5 h after hair cell death (Fig. 5 B and Fig. S5 B ), whereas the BMP inhibitor noggin 3 ( nog3 ) is not down-regulated until 5 h after hair cell death ( Fig. S5 B ). Because ligand expression by itself is not a reliable indicator for signaling pathway activation, future studies will need to de- termine how the activation status of Bmp signaling is affected by bmp5 and nog3 .\nOther pathways that respond to zebrafish lateral line hair cell death are the Mapk ( Fig. S5 C and D ), TNF- α , ( Fig. S5 E ), insulin ( Fig. S5 F ), and apoptosis signaling ( Fig. S5 G ) pathways and nitric oxide (NO) and reactive oxygen species (ROS) ( Fig. S5 H ), some of which increase proliferation in undamaged chicken ves- tibular support cells (74). For example, we observed up-regulation of nox1 , noxo1a , and noxa1 ( Fig. S5 H and Dataset S2 ). NO is a radical that regulates a wide variety of physiological processes and is important during liver regeneration (75).\nWe summarized the activation status of some of the major signaling pathways based on the in situ hybridization patterns of pathway members in Fig. 5 C . The Wnt/ β -catenin pathway is not active in homeostatic or newly regenerating neuromasts, and the Notch and Fgf pathways are down-regulated during the first few hours of regeneration. The Wnt/ β -catenin and Notch pathways seem to play a role only during later stages of regeneration. The cell cycle is activated immediately, possibly via cytokine stimu- lation of the Jak1/Stat3 pathway.\nIndependent Microarray Studies Support the Differential Expression of Genes Not Associated with Key Signaling Pathways. An in- dependent expression analysis performed by Steiner et al. and reported in this issue (42), used a transgenic line with an alkaline phosphatase enhancer [ Tg ( -4.7alp:mCherry )] instead of the Tg ( sqET20 ) line we used to isolate mantle cells by FACS anal- ysis. We compared our respective datasets to determine how similar the data are. We first analyzed the overlap of genes enriched in untreated mantle/lateral line cells in both studies. The obtained results are a clear indicator of how similar the sorted cell populations in the two studies are. We chose the top 1,000 up- regulated genes from our Dataset S1 by p-value and then ex- amined how many of these genes also are among the top 1,000 genes identified by Steiner et al. (dataset S1 in ref. 42). If the datasets were independent and unrelated, we would expect 53 genes to overlap by chance. We identified 201 common genes with a p-value of 3.9e-107, demonstrating that our two datasets are significantly related. All six mantle cell-specific genes iden- tified by Steiner et al. (42) also show enrichment in our untreated GFP + cell population ( Table S4 A , blue rows). Ten of the 26 top up- and down-regulated genes that we validated by in situ hybridization\nafter hair cell loss ( Table S2 ) and for which we found a match in the Steiner dataset also show significant changes in the Steiner microarray dataset ( Table S4 B , blue rows) (42). Several of the genes that were not present in the Steiner dataset are expressed more highly by in situ hybridization in inner support cells than in mantle cells, reflecting the nature of the transgenic lines used and the type of cells purified. Nine of the 14 genes that Steiner et al. identified by microarray analysis, by RT-PCR, or by in situ hybridization (42) also showed the same trend in expression changes in our RNA-Seq analysis ( Table S4 C , blue rows).\nThe overlap between a subset of our data and that of Steiner et al. (42), combined with the identification and corresponding validation of genes in our study, strongly indicates that the two lines of investigation are complementary, with most of the tar- gets identified by Steiner et al. (42) found in the mantle cells and the majority of genes identified in our study expressed primarily in the inner support cells. As such, our respective studies both validate each other and help resolve the cell-type specificity of the revealed differential gene expressions.\n\n\nDiscussion\nSeveral studies have aimed to induce mammalian hair cell re- generation by inactivating or activating candidate genes and pathways. Knockdown of the cell cycle exit regulators Cdkn1b ( p27/Kip1 ), p19/Ink4d , and pRb led to the initiation of cell cycle reentry and the differentiation of hair cells in injured epithelia in mice (20, 21, 64, 76, 77). Unfortunately, these ectopic hair cells cannot be maintained and eventually apoptose. Because direct manipulation of cell cycle regulators has not been successful in reconstituting a fully functional sensory epithelium in mammals, it is important to identify the mitogens that activate cell cycle entry in regenerating organisms. It is plausible that manipu- lations of upstream signaling events are a more promising ave- nue to regenerate mammalian hair cells.\nMore recently, in mice the down-regulation of Notch signaling caused transdifferentiation of support cells into functional hair cells without inducing proliferation (30). However, the number of regenerated hair cells was modest, and it is unclear if the accompanied depletion of support cells affects the function of the partially regenerated sensory epithelium in the long term. Although these efforts have not resulted in the regeneration of a fully functional, morphologically normal epithelium, they clearly demonstrate that the mammalian cells of the inner ear can be manipulated and coaxed away from their terminally differenti- ated state.\nKnowledge gained from studying the development of hair cells has demonstrated that the underlying molecular mechanisms are complex, with a multitude of interacting signaling pathways and feedback loops that ensure that an unspecified progenitor cell transitions to a prosensory cell that ultimately differentiates into a functional sensory cell. We also have learned that signaling pathways are activated and inhibited repeatedly as hair cell progenitors are specified and differentiate, further complicating their experimental manipulation. Hence, a global up- or down- regulation of a particular pathway is unlikely to lead to fully functional sensory epithelia.\nWe hypothesize that the regeneration of a complete sensory epithelium will require the manipulation of several pathways in the correct order. However, the model systems currently used to study the regeneration of sensory epithelia present unique ob- stacles. For example, in chicken, antibiotic-induced hair cell death occurs over a 24-h period, and the proliferation of support cells and differentiation of hair cells occurs 1 – 4 wk after injury (19). This long time frame makes it difficult to determine the precise time point when pathways are turned on and off, because support cells will be in different stages of regeneration. Likewise, a previous expression study of the regenerating zebrafish ear was\nperformed 24 h after the larvae were exposed to noise that does not cause death simultaneously in all hair cells (36). Further adding to the complexity, genes are used repeatedly during hair cell development and can turn from an activator into a repressor (23). We have selected the zebrafish lateral line tissue to eluci- date the gene-expression patterns triggered by hair cell death. The zebrafish lateral line is particularly well suited for these analyses, because all hair cells can be killed acutely within 30 min, and regeneration occurs within 24 h (Fig. 3 A ), allowing us to determine the earliest signals and responses in support cells after hair cell death.\nWnt/ β -Catenin Signaling Is Not Detected Early in Lateral Line Hair Cell Regeneration. Our analyses focused on pathways involved in hair cell development as they might be reused during regeneration (15, 23, 43, 78, 79). For example, Wnt/ β -catenin signaling is crucial for the development of many organs across the animal kingdom (80). During sensory development, Wnt/ β -catenin sig- naling governs the proliferation of unspecified progenitor cells, renders them competent to acquire a neural cell fate, and sub- sequently is involved in the differentiation of these postmitotic prosensory cells (3, 23, 81). Wnt/ β -catenin signaling also is in- volved in regeneration. Wnt/ β -catenin signaling controls stem cell maintenance, quiescence, and proliferation during mam- malian epithelial hair cell, zebrafish fin, mammalian intestine, hydra head, and planarian regeneration (82 – 86). Importantly, Wnt/ β -catenin also is crucial for regulating proliferation in de- veloping and regenerating zebrafish neuromasts (26, 48).\nIn the mouse cochlea, ectopic Wnt/ β -catenin signaling pro- motes proliferation of Lgr5 + support cells via the down-regula- tion of Cdkn1c (p27/Kip1); however, this effect diminishes as the animals mature (3, 20, 28, 81, 87). Because Wnt/ β -catenin sig- naling is down-regulated in the mammalian cochlea after birth, it was hypothesized that the absence of Wnt/ β -catenin signaling is responsible for the inability of support cells to proliferate in response to hair cell death (3, 28, 29, 65). Therefore, recent studies have focused on up-regulating the Wnt/ β -catenin path- way in uninjured cochleae with the expectation of coaxing mammalian support cells into proliferating and producing hair cells (3, 17, 28, 29, 81). These experiments indeed yielded ad- ditional hair cells, but the epithelium was disorganized.\nSurprisingly, our data demonstrate that in zebrafish, as in adult mice cochleae, Wnt/ β -catenin signaling is inactive in ho- meostatic neuromasts and is up-regulated only at 12 h after neomycin treatment. Therefore Wnt/ β -catenin signaling is un- likely to play a role in the initial triggering of the regenerative response in zebrafish, even though it is sufficient and necessary to induce proliferation during later stages of regeneration, for example by inhibiting Cdkn1b (p27/Kip1) and regulating dyclinD1 (3, 27, 87).\nNotch Signaling Is Down-Regulated Early in Lateral Line Hair Cell Regeneration. A growing number of studies have explored how the manipulation of Notch signaling could aid in regenerating mammalian hair cells. It is well established that the inhibition of Notch signaling causes an increase in the number of hair cells in uninjured and injured chicken, guinea pig, and mouse utricles and cochleae and in injured zebrafish ears and lateral line neu- romasts (52, 53, 55 – 57, 88). The presence of Notch is detrimental to hair cell differentiation, because it blocks the proneural gene atoh1a and thereby maintains support cells (9, 13, 65, 89, 90). The ability of support cells to transdifferentiate into hair cells after down-regulation of Notch signaling was exploited recently to restore a modest amount of hair cells in the injured adult mouse cochlea that led to some promising functional recovery of hearing (30). This experimental down-regulation of Notch sig- naling recapitulates our observations during zebrafish lateral line hair cell regeneration. In zebrafish, Notch signaling is transiently\ndown-regulated 1 h after hair cell death, correlating with the attenuation of sox2 and up-regulation of atoh1a . The interactions between Notch signaling, sox2 , and atoh1a have not yet been tested in the zebrafish lateral line; however, because of their behavior during regeneration, it is very likely that their regula- tion is similar to that occuring during mouse, Xenopus , and zebrafish central nervous system or sensory organ development (23, 52, 61, 91 – 93). During murine ear development Sox2 ex- pression depends on Notch signaling and is required for the acquisition of neural potential in the prosensory epithelium (52). Subsequently, for neurogenesis to proceed, Sox2 is shut off via the accumulation of proneural genes, such as Atoh (93 – 97). In contrast, Sox2 is maintained in Notch + support cells, where it inhibits hair cell differentiation by inhibiting the proneural basic helix-loop-helix gene Atoh1a (91).\nIt is plausible that the down-regulation of Notch signaling in neuromasts is required during the earliest stages of hair cell regeneration to down-regulate sox2 in support cells, allowing these cells to acquire a proneural character via the up-regulation of atoh1a . Because Notch signaling needs to be reinitiated for fate specification of hair and support cells and control of organ size (9, 60), the Notch target genes her4.1 , notch3 , and sox2 are again up- regulated between 3 and 5 h after neomycin treatment (Fig. 4 A and B ). These genes likely are activated by the up-regulation of dld (60). Therefore, we hypothesize that the orchestration of genes downstream of Notch signaling appears to be the same during zebrafish hair cell development and regeneration.\nIt has not been appreciated previously that the down-regula- tion of Notch signaling could be a hallmark of spontaneously regenerating sensory epithelia, because it was reported that the Notch signaling pathway is up-regulated in response to hair cell death in chicken, guinea pigs, and zebrafish (9, 88, 98). However, these expression analyses were performed at later time points during the regeneration process, when Notch signaling is re- quired once again for hair cell versus support cell specification to occur. We also observed that zebrafish Notch target genes are reexpressed at 3 h after hair cell death (Fig. 4 A ). Because the zebrafish lateral line system allows us to kill hair cells very acutely, causing a relatively synchronized response in support cells, we were able to detect the immediate, transient down-regulation of Notch signaling after hair cell death. In the injured mammalian cochlea or utricle such drastic down-regulation of Notch target genes, such as Sox2 and several Hes and Hey genes, does not occur (55, 56); the absence of such down-regulation could be one of the reasons mammalian hair cells do not regenerate.\nThe Regulation of Notch Signaling in Combination with Mitogenic Signals May Offer Novel Therapeutic Strategies to Promote Hair Cell Regeneration in Mammals. Our data suggest that identifying the molecules that down-regulate Notch signaling in regenerat- ing species will be important in efforts to understand why mammals fail to regenerate injured hair cells. Our RNA-Seq analyses identified the transmembrane protein Numb as a po- tential candidate for inhibiting Notch signaling. Future studies should focus on the sufficiency of Numb to down-regulate Notch signaling in zebrafish and understanding Numb ’ s temporal ex- pression patterns during homeostasis and after injury in other regenerating species, as well as in mice. Numb could open a new therapeutic avenue for manipulating Notch signaling in mammals and facilitating successful regeneration of hair cells in mammals.\nEven though down-regulation of Notch signaling induced differentiation of new hair cells in mammals, these hair cells developed via transdifferentiation of support cells into hair cells, leading to the depletion of support cells (30). The down-regu- lation of Notch was not sufficient to induce support cell pro- liferation as observed in spontaneously regenerating species. Zebrafish regenerate hair cells mainly or exclusively by pro- liferation of support cells, and regenerating chicken hair cells\narise via transdifferentiation, as well as proliferation (19). In zebrafish we observe that the down-regulation of Notch signaling at 1 h after neomycin treatment correlates with the down-regu- lation of cyclin-dependent kinase inhibitors cdkn1b (p27/kip1) and cdkn1c ( p57/kip2 ) (Fig. 4 A and D ). This result suggests that Notch could be an activator of cdkn1b ( p27/kip1 ) that keeps cells out of the cell cycle and therefore that the down-regulation of Notch signaling could be a crucial event for the onset of re- generation. Indeed in the murine inner ear, the Notch target Sox2 regulates Cdkn1b (p27/Kip1), similar to activity described in the epidermis (99, 100) but in contrast to the activity in the intestinal crypt and inner ear, where Notch inhibits Cdkn1b (p27/ Kip1) (101, 102). In zebrafish the attenuation of Notch signaling has no effect on proliferation in uninjured sensory organs and has only a proproliferative effect on support cells after injury (9). In chicken cochleae the proliferation rate is not increased after injury, nor does inhibition of Notch cause cell cycle reentry in mice cochleae (13, 19, 30). Based on these findings it was con- cluded that Notch signaling is not required to initiate pro- liferation during regeneration (13). Therefore, it is possible that Cdk inhibitors are regulated by as yet unidentified, Notch- independent signals. Possibly, Fgf signaling inhibits proliferation, as demonstrated in the chicken inner ear (63).\nBecause complete regeneration of sensory epithelia in mam- mals requires coaxing support cells into hair cell differentiation as well as stimulating the support cells to re-enter the cell cycle and self-renew, manipulation of Notch signaling is not sufficient and needs to be married with manipulations of mitogenic signals. Furthermore, similar to our observations during zebrafish hair cell regeneration, Notch signaling must be reinitiated for the inhibition of proneural genes and specification of support cells (12, 23, 91). A temporally restricted Notch down-regulation or a combination of Notch down-regulation with subsequent ectopic expression of support cell inducers, such as Sox2, could be fruitful.\nOur efforts presented here have characterized the pattern of temporal activation of signaling pathways during zebrafish sensory hair cell regeneration. These results will aid in the design and in- terpretation of functional studies aimed at elucidating the gene-reg- ulatory network underlying zebrafish hair cell regeneration and can guide experiments to trigger hair cell regeneration in mammals.\n\n\nThis article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1402898111/-/DCSupplemental . www.pnas.org/cgi/doi/10.1073/pnas.1402898111 PNAS | Published online March 27, 2014 | E1383 – E1392 DEVELOPMENTA L BIOLOGY PNAS PLUS SEE CORRECTION FOR THIS ARTICLE\n\n\ncontrol 1h ptp4a2 lima1 junba abcf2a eif4a1a control 1h il11b nlrc3 hk2 (glulc) g3bp1 rapgef3 txn clic2 1h lad1 znf462 dfna5 gpr37b zgc:110353 slc35c2 tcea3 A B control zgc:152741 Fig. 2. Validation by in situ hybridization of a selection of 14 genes up- regulated ( A ) and seven genes down-regulated ( B ) at 1 h after neomycin treatment. ( Upper Rows ) Untreated larvae (control) at 5 dpf. ( Lower Rows ) Treated larvae.\n\n\nPNAS PLUS\n\n\nrbl2\n\n\nher4.1 notch3 1h 3h 5h 10h cdkn1c cdkn1b 1h 3h 5h 10h C A numb sox2 atoh1a dld pea3 fgf3 1h 3h 5h 10h D her4.1 atoh1b her6 numbl dlc gro2 ctbp2 notch3 cir1 jag2 aph1b ncor2 dld dla atoh1a numb cdk6 prkdc rbl2 mad2l2 cdkn1c cdkn1b ccna1 gadd45aa gadd45ba gadd45ab cdk7 plk1 ywhag1 pcna ccne1 anapc1 ywhai abl1 stag2 gadd45g mcm6l wee1 1h 3h 5h 1h 3h 5h 1h 3h 5h 1h 3h 5h B E control control control Notch cell cycle notch1b jag1a jag1b rbpjb sox2 GFP GFP GFP GFP 2 0 -2 2 0 -2 Fig. 4. Inhibition of Notch and Fgf pathways is accompanied with cell cycle reentry immediately after hair cell ablation. ( A , C , and D ) In situ hybridization of Notch pathway, Fgf pathway, and cell cycle genes in 5-dpf untreated and neomycin-treated larvae. ( A ) The Notch pathway inhibitor numb is increased at 1 h after neomycin. The Notch target genes her4.1 , notch3 , and sox2 are decreased at 1 h and up-regulated at 3 h after neomycin treatment. a toh1a is up- regulated starting at 1 h after neomycin, and its target dld is increased at 5 and 10 h. ( C ) fgf3 and its target pea3 are down-regulated at 1 and 3 h after neomycin treatment. ( D ) Cell cycle inhibitory genes cdkn1b , cdkn1c , and rbl2 are decreased at 1 and 3 h after neomycin compared with control. ( B and E ) Heat maps of selected Notch pathway genes and cell cycle genes.\n\n\nbmp5 control 1h 3h 5h 10h A stat3 1h 3h 5h 10h control B C Wnt Notch Fgf cell cycle Stat3 1h 3h 5h 10h 12h no change inhibition activation not tested\n\n\nPNAS PLUS\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nE1384 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1385 DEVELOPMENTA L\n\n\nE1386 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1387 DEVELOPMENTA L\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1389 DEVELOPMENTA L\n\n\nE1390 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\n1. White PM, Doetzlhofer A, Lee YS, Groves AK, Segil N (2006) Mammalian cochlear supporting cells can divide and trans-differentiate into hair cells. Nature 441(7096): 984 – 987.\n\n\n2. Corwin JT, Oberholtzer JC (1997) Fish n ’ chicks: Model recipes for hair-cell regeneration? Neuron 19(5):951 – 954.\n\n\n5. Williams JA, Holder N (2000) Cell turnover in neuromasts of zebrafish larvae. Hear Res 143(1-2):171 – 181.\n\n\n8. López-Schier H, Hudspeth AJ (2006) A two-step mechanism underlies the planar polarization of regenerating sensory hair cells. Proc Natl Acad Sci USA 103(49): 18615 – 18620.\n\n\n9. Ma EY, Rubel EW, Raible DW (2008) Notch signaling regulates the extent of hair cell regeneration in the zebrafish lateral line. J Neurosci 28(9):2261 – 2273. 10. Groves AK (2010) The challenge of hair cell regeneration. Exp Biol Med (Maywood) 235(4):434 – 446.\n\n\n27. Wada H, et al. (2013) Wnt/Dkk negative feedback regulates sensory organ size in zebrafish. Curr Biol 23(16):1559 – 1565.\n\n\n28. Shi F, Hu L, Edge AS (2013) Generation of hair cells in neonatal mice by β -catenin overexpression in Lgr5-positive cochlear progenitors. Proc Natl Acad Sci USA 110(34): 13851 – 13856.\n\n\n33. Whitfield TT (2002) Zebrafish as a model for hearing and deafness. J Neurobiol 53(2): 157 – 171.\n\n\n56. Korrapati S, Roux I, Glowatzki E, Doetzlhofer A (2013) Notch signaling limits sup- porting cell plasticity in the hair cell-damaged early postnatal murine cochlea. PLoS ONE 8(8):e73276.\n\n\n57. Lin V, et al. (2011) Inhibition of Notch activity promotes nonmitotic regeneration of hair cells in the adult mouse utricles. J Neurosci 31(43):15329 – 15339.\n\n\n58. McGill MA, Dho SE, Weinmaster G, McGlade CJ (2009) Numb regulates post-endo- cytic trafficking and degradation of Notch1. J Biol Chem 284(39):26427 – 26438.\n\n\n60. Matsuda M, Chitnis AB (2010) Atoh1a expression must be restricted by Notch sig- naling for effective morphogenesis of the posterior lateral line primordium in ze- brafish. Development 137(20):3477 – 3487.\n\n\n63. Oesterle EC, Bhave SA, Coltrera MD (2000) Basic fibroblast growth factor inhibits cell proliferation in cultured avian inner ear sensory epithelia. J Comp Neurol 424(2): 307 – 326.\n\n\n69. Fang Y, et al. (2013) Translational profiling of cardiomyocytes identifies an early Jak1/Stat3 injury response required for zebrafish heart regeneration. Proc Natl Acad Sci USA 110(33):13416 – 13421.\n\n\n74. Oesterle EC, Tsue TT, Rubel EW (1997) Induction of cell proliferation in avian inner ear sensory epithelia by insulin-like growth factor-I and insulin. J Comp Neurol 380(2): 262 – 274.\n\n\n75. Carnovale CE, Ronco MT (2012) Role of nitric oxide in liver regeneration. Ann Hepatol 11(5):636 – 647.\n\n\n76. Sage C, et al. (2005) Proliferation of functional hair cells in vivo in the absence of the retinoblastoma protein. Science 307(5712):1114 – 1118.\n\n\n77. Sage C, et al. (2006) Essential role of retinoblastoma protein in mammalian hair cell development and hearing. Proc Natl Acad Sci USA 103(19):7345 – 7350.\n\n\n80. van Amerongen R, Nusse R (2009) Towards an integrated view of Wnt signaling in development. Development 136(19):3205 – 3214.\n\n\n81. Shi F, Kempfle JS, Edge AS (2012) Wnt-responsive Lgr5-expressing stem cells are hair cell progenitors in the cochlea. J Neurosci 32(28):9639 – 9648.\n\n\n82. Galliot B, Chera S (2010) The Hydra model: Disclosing an apoptosis-driven generator of Wnt-based regeneration. Trends Cell Biol 20(9):514 – 523.\n\n\n83. Fleming HE, et al. (2008) Wnt signaling in the niche enforces hematopoietic stem cell quiescence and is necessary to preserve self-renewal in vivo. Cell Stem Cell 2(3): 274 – 283. 84. Li L, Bhatia R (2011) Stem cell quiescence. Clin Cancer Res 17(15):4936 – 4941.\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Gene-expression analysis of hair cell regeneration in the zebrafish lateral line\nLinjia Jiang a , Andres Romero-Carvajal a,b , Jeff S. Haug a , Christopher W. Seidel a , and Tatjana Piotrowski a,1\na Stowers Institute for Medical Research, Kansas City, MO 64110; and b Department of Neurobiology and Anatomy, University of Utah Medical School, Salt Lake City, UT 84132\nEdited* by Yuh Nung Jan, Howard Hughes Medical Institute, University of California, San Francisco, CA, and approved March 4, 2014 (received for review February 16, 2014)\nDeafness caused by the terminal loss of inner ear hair cells is one of the most common sensory diseases. However, nonmammalian animals (e.g., birds, amphibians, and fish) regenerate damaged hair cells. To understand better the reasons underpinning such disparities in regeneration among vertebrates, we set out to define at high resolution the changes in gene expression associ- ated with the regeneration of hair cells in the zebrafish lateral line. We performed RNA-Seq analyses on regenerating support cells purified by FACS. The resulting expression data were subjected to pathway enrichment analyses, and the differentially expressed genes were validated in vivo via whole-mount in situ hybrid- izations. We discovered that cell cycle regulators are expressed hours before the activation of Wnt/ β -catenin signaling following hair cell death. We propose that Wnt/ β -catenin signaling is not involved in regulating the onset of proliferation but governs pro- liferation at later stages of regeneration. In addition, and in marked contrast to mammals, our data clearly indicate that the Notch pathway is significantly down-regulated shortly after in- jury, thus uncovering a key difference between the zebrafish and mammalian responses to hair cell injury. Taken together, our findings lay the foundation for identifying differences in sig- naling pathway regulation that could be exploited as potential therapeutic targets to promote either sensory epithelium or hair cell regeneration in mammals.\nsignaling pathway analysis | RNA sequencing | neuromast | Jak/Stat3 | cdkn1b\nH earing disorders are the most common sensory disabilities in humans and can be either inherited or acquired. De- generation and death of sensory hair cells in the inner ear is the underlying cause of age-related, noise-, antibiotic-, or chemo- therapy drug-induced hearing loss ( www.nidcd.nih.gov ). Such loss is permanent, because mammalian inner ear hair cells do not regenerate. The inability of mammals to restore sensory hair cells after injury is intriguing given that ( i ) mammalian vestibular support cells exhibit low levels of proliferation after injury; ( ii ) cochlear support cells of newborn mice are able to proliferate subsequent to hair cell death in cell culture (1); and ( iii ) other vertebrates such as birds, amphibians, and fish replace their sensory hair cells continuously and, importantly, are capable of regenerating hair cells after damage (2 – 10).\nWork in birds has shed light on the mechanisms regulating sensory hair cell regeneration in vertebrates (2, 4, 11 – 18). For instance, during sensory hair cell regeneration in the chick, the majority of hair cells arise by proliferation of neighboring sup- port cells (19). Given that mammalian support cells do not re- spond to hair cell death by proliferating, molecular studies in chick have focused on identifying the mitogenic signals as well as the target genes that induce re-entering of the cell cycle in support cells (11, 20 – 22). For example, the transcription factors Atoh1 and Sox2 and the Wnt/ β -catenin and Notch pathways have been shown to play crucial roles (23), and gene-expression analyses using microarrays have identified hundreds of genes involved in chick hair cell regeneration (18, 24).\nThe Wnt/ β -catenin pathway is used repeatedly during hair cell development, and recent studies have emphasized the importance of Wnt/ β -catenin signaling in modulating the proliferation of sup- port cells. For example, activated β -catenin causes proliferation in dissociated chicken utricular epithelia and ectopic expression of hair cell differentiation in the chicken basilar papilla (17, 25). Likewise, during mouse cochlear development Wnt/ β -catenin sig- naling activates proliferation of support cells and is required sub- sequently for their differentiation into hair cells (3, 26 – 29). Perturbing the signaling pathways known to be activated during chick inner ear regeneration therefore may provide an opportunity to modulate regenerative capacities in the mam- malian inner ear. For instance, it has been shown recently that down-regulation of Notch signaling in adult mice after injury induces the transdifferentiation of support cells into hair cells and can partially restore hearing (30). However, this manipula- tion causes the loss of support cells, and the implications of this loss for long-term hearing restoration are unknown.\nSignificant progress has been made in the study of sensory hair regeneration. However, in part because of the complexity of spatial and temporal gene interactions and signaling dynamics, our knowl- edge still is too incomplete to enable us to promote the generation of a functional epithelium after injury in mammals. The zebrafish lateral line provides an experimentally accessible system to define the complex signaling events triggered by injury and regeneration, because these cells can be killed acutely and then regenerate rap- idly (31). These characteristics allow the detection of changes in gene expression at defined time points following hair cell death.\nSignificance\nDeafness is caused largely by the death of sensory hair cells in the inner ear. In contrast to nonmammalian vertebrates, hu- man hair cells do not regenerate. Understanding the mecha- nisms that regulate hair cell regeneration in zebrafish may shed light on the factors that prevent hair cell regeneration in mammals. RNA-Seq analysis of regenerating sensory organs uncovered dynamic changes in the expression of signaling pathways during zebrafish hair cell regeneration. Unexpect- edly, the Wnt/ β -catenin, Notch, and Fgf pathways are down- regulated following hair cell death, whereas the Jak1/Stat3 pathway and cell cycle are activated. We propose that mim- icking the zebrafish activation status of a combination of path- ways at the correct time points in mammals may improve the chances of triggering regeneration of functional hair cells.\nAuthor contributions: L.J. and T.P. designed research; L.J., A.R.-C., and J.S.H. performed research; L.J., J.S.H., C.W.S., and T.P. analyzed data; and L.J. and T.P. wrote the paper.\nThe authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nData deposition: The data reported in this paper have been deposited in the Gene Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE56176 ). 1 To whom correspondence should be addressed. E-mail: pio@stowers.org.\nZebrafish and the majority of all other aquatic vertebrates possess hair cells not only in their vestibular system in the ear but also in their skin. These hair cells are part of the sensory lateral line system. The lateral line system senses water motion and initiates the appropriate behavioral response for capturing prey, avoiding predators, and schooling. The morphology and function of sensory hair cells are evolutionarily conserved from fish to mammals (32, 33). Lateral line hair cells are located in the center of a mechanosensory organ known as the neuromast and are surrounded by inner support cells and an outer ring of mantle cells. Importantly, despite the unusual location of the hair cells on the trunk, lateral line and ear hair cells develop and differ- entiate by similar developmental mechanisms. For example, mutations in genes causing deafness in humans also disrupt hair cell function in the zebrafish lateral line and vestibular system (34). Because of its genetic tractability and the accessibility of sensory organs to experimental manipulation and visualization, the zebrafish recently has been recognized as an excellent model for discovering and functionally characterizing genes crucial for hair cell development and regeneration (16, 19, 35, 36).\nIn zebrafish hair cell regeneration occurs via the proliferation and differentiation of inner support cells (5, 6, 9, 37, 38), but detailed lineage analyses have not yet been performed, and the stem cell population has not been identified. Mantle cells are good stem cell candidates, because they migrate to form neu- romasts on regenerating tail tips and also form postembryonic neuromasts by a process called “ stitching. ” However, after chemical ablation of hair cells by antibiotic treatment, mantle cells proliferate much less frequently than inner support cells (9). Possibly, mantle cells replenish inner support cells that then differentiate into hair cells. Mantle cells have not been shown to differentiate directly into hair cells (37, 38). Therefore, both mantle and inner support cells seem to be involved in reestablishing proper neuromast morphology after hair cell death.\nTo identify the activation status of signaling pathways during the first few hours of regeneration, we performed RNA-Seq gene expression analyses of zebrafish mantle and inner support cells isolated by FACS. We discovered that the Notch pathway is transiently down-regulated immediately after hair cell death; to our knowledge, this down-regulation has not been previously reported in other studies of regenerating species. Our studies also demonstrate that Wnt/ β -catenin signaling is not among the earliest pathways to respond to hair cell death but instead is up- regulated relatively late during the regenerative response. These results suggest that Wnt/ β -catenin signaling does not trigger proliferation by down-regulating cyclin-dependent kinase inhib- itors immediately after hair cell death but rather regulates pro- liferation at later stages of regeneration. Our findings have defined the temporal regulation of key signaling pathways asso- ciated with the early stages of hair cell regeneration and suggest that the successful initiation of mammalian hair cell regeneration, either in vivo or in vitro, might require precise temporal manip- ulation of these pathways.\n\n\nResults\nIsolation of Mantle and Inner Support Cells by FACS. To identify changes in gene expression in mantle and inner support cells after hair cell death, we isolated GFP + cells from regenerating and control Tg ( sqET20 ) transgenic zebrafish via FACS (Fig. 1 A and B ) (39). Tg ( sqET20 ) larvae express GFP strongly in mantle cells and to a lesser degree in inner support cells (Fig. 1 B ). Hair cells are GFP − and therefore are excluded from the GFP + population. We isolated GFP − and GFP + cells from Tg ( sqET20 ) control larvae at 5 days postfertilization (dpf) and from neo- mycin-treated larvae at 1, 3, and 5 h after neomycin treatment. Larvae were dissociated into a single-cell suspension in a trypsin solution and stained with 7-aminoactinomycin D (7-AAD) and Hoechst 33342 to exclude dead cells and cell debris (Fig. 1 C ). Live GFP + cells (7-AAD − , Hoechst + , GFP + cells) and control GFP − cells (7-AAD − , Hoechst + , GFP − cells) were sorted by FACS (Fig. 1 D ). The sorting yielded highly GFP + -enriched cell populations, as demonstrated by a postsort reanalysis of the collected cells (Fig. 1 E and F ). GFP + -sorted cells showed 94.4 ± 1.3% enrichment ( n = 7). To confirm that GFP + cells are lateral line cells, we performed immunostaining with an antibody against the support cell marker Sox2 that labels inner support cells as well as some mantle cells (Fig. 1 G and Fig. S1 ) (40). Counts of Sox2 + FACS-purified cells revealed that 95% of GFP + cells but only 6.6% of GFP − cells were positive for Sox2 (Fig. 1 H ). Ad- ditionally, we performed RT-PCR for genes highly expressed in neuromast cell types and with limited expression in other tissues. RT-PCR for gfp , eya1 , klf4b , and cldnb showed that these genes are expressed at significantly higher levels in GFP + than in GFP − cells ( Fig. S2 A – D ). In contrast, genes expressed in tissues outside the lateral line system, such as rhodopsin ( rho , eye), vil- lin1 ( vil1 , gut), and myod1 (muscle), are expressed at much lower levels in GFP + cells ( Fig. S2 E – G ). In Tg ( sqET20 ) larvae, mantle B A G K Sox2    cells (%) 0 20 40 60 80 100 H lateral line genes neomycin induced changes J Tg(sqET20) mantle cell inner support cell GFP GFP GFP 1h Sox2 GFP DAPI GFP GFP GFP control neo control neo control 30min 1h 3h 5h 30min 1h 3h 5h GFP 3h GFP 5h GFP 1h GFP 3h GFP 5h GFP changes in mantle and support cells C D 97.8% FSC FSC 0 1000 2000 3000 4000 FSC 100 101 2 103 10 100 101 10 2 103 104 Hoechst GFP GFP 94.3% live cells 10 4 E F GFP GFP 100 101 2 103 10 10 4 100 101 2 103 10 10 4 0 1000 2000 3000 4000 0 1000 2000 3000 4000 100 101 2 103 10 10 4 I 7-AAD GFP GFP GFP GFP\nFig. 1. Purification of mantle and inner support cells from Tg ( sqET20 ). ( A ) In 5-dpf Tg ( sqET20 ) larvae GFP is expressed in neuromasts (arrows) and in the interneuromast cells that connect adjacent neuromasts. ( B ) Magnification of a neuromast shows that mantle cells are strongly labeled by GFP, but inner support cells are weakly labeled. ( C and D ) Dissociated cells of Tg ( sqET20 ) larvae at 5 dpf were FACS sorted using a two-gate sorting strategy. Live cells were gated by 7-AAD − and Hoechst + ( C ) and subsequently were gated by forward scatter (FSC) and GFP intensity to isolate GFP − and GFP + cells ( D ). ( E and F ) A postsorting experiment shows that GFP − cells are 97.8% pure ( E ) and GFP + cells are 94.3% pure ( F ). ( G ) Sorted GFP − and GFP + cells were stained with Sox2 antibody and DAPI. ( Upper ) A GFP − cell is Sox2 − . ( Lower ) A GFP + cell expresses Sox2. (Scale bars: 5 μ m.) ( H ) Quantification of Sox2 + cells in each population as mean ± SD; * P &lt; 0.01 by t test. ( I ) Regeneration- induced mantle and inner support cell genes were identified at each time point by comparing GFP + neomycin-treated cells with GFP + untreated samples. ( J ) Neomycin-induced genes were identified by comparing GFP − neomycin-treated cells at each time point with GFP − untreated samples. Neomycin-induced genes subsequently were subtracted from the gene lists obtained with GFP + cells to select for regeneration-specific changes in gene expression in inner support and mantle cells. ( K ) Comparisons between GFP + and GFP − cells of untreated larvae generated a list of genes enriched in lateral line cells.\ncells express GFP most strongly, but inner support cells respond more robustly to hair cell death by proliferating (5, 6). Because Tg ( sqET20 ) larvae show weaker GFP expression in inner support cells, we were concerned that our cell sorts would exclude inner support cells. However, RT-PCR with inner support cell markers such as klf4b showed that inner support cells were included in the GFP + cell populations ( Fig. S2 C ). Combined, these studies dem- onstrated that our FACS protocol yielded a highly enriched pool of lateral line cells containing mantle and inner support cells but not hair cells.\nTo investigate the temporal activation or repression of in- dividual genes and signaling pathways in GFP + mantle and inner support cells during the first few hours of regeneration, we ex- amined GFP + and GFP − cells at 1, 3, and 5 h after a 30-min treatment with neomycin (Fig. 1 I ). Sorts for each time point were performed in triplicate and subjected to RNA-Seq analysis. Bacte- ricidal antibiotics cause oxidative stress in mammalian cells, leading us to speculate that sublethal concentrations of neomycin may cause changes in gene expression unrelated to hair cell death (41). To control for this nonspecific effect, we examined gene expression in GFP − cells in the presence or absence of neomycin (Fig. 1 J ). Al- though this population includes all larval nonlateral cells, it allows us to control for a nonspecific neomycin effect when comparing GFP + cells from treated and control larvae.\nAnalysis of the RNA-Seq Results. Sequencing libraries were pre- pared from mRNAs of collected cells and were quality checked on a Bioanalyzer. We performed 50-bp single-end RNA-Seq using the Illumina platform. To test the enrichment of lateral line genes further, we examined the fragments per kilobase of transcript per million mapped reads (FPKM) values for known lateral line genes and for genes expressed in other tissues ( Fig. S3 ). The results confirmed the RT-PCR analysis described above. Pan lateral line, as well as inner support cell, markers show much higher FPKM values in FACS-sorted GFP + cells than in GFP − cells. As suggested by the RT-PCR results, the FPKM values for the inner support cell markers pea3 and klf4b are significantly higher in the GFP + than in the GFP − cells, confirming that inner support cells were included in our FACS sorts. This enrichment of lateral line genes is supported by the information in Dataset S1 , in which neuromast-specific genes were identified by comparing the expression profiles of untreated GFP + and GFP − cells. In addition to known lateral line genes, the resulting lists of differentially expressed genes provide a valuable resource of as yet uncharacterized genes that potentially could play important roles in hair cell development and/or re- generation (Fig. 1 K and Dataset S1 ). The number of genes enriched in GFP + cells relative to GFP − controls at an adjusted p-value ≤ 0.05 is 1,670 ( SI Materials and Methods and Dataset S1 ). This dataset also contains many of the genes reported in the dataset of Steiner et al. (42), who identified mantle cell-specific genes using a different transgenic line (see below).\nGene Identification for Each Time Point. To identify genes from transcripts specifically enriched or depleted in GFP + mantle and inner support cells after hair cell death, we created several comparisons between the RNA-Seq datasets. Ratios of gene expression were created between the neomycin-treated GFP + cells at 1, 3, and 5 h and the nontreated GFP + cells at 1 h to identify genes responding to hair cell death (Fig. 1 I ). To control for a nonspecific neomycin response, a similar set of ratios was created between the neomycin-treated GFP − cells at each time point and the untreated GFP − cells at 1 h (Fig. 1 J ). The neomycin- induced genes were subtracted from all gene lists generated with GFP + cells. We then used ratios and P values between datasets to select genes of interest at any given time point ( SI Materials and Methods and Dataset S2 ). Genes identified by these criteria are marked with a numeric flag with positive numbers indicating up-\nregulated and negative numbers indicating down-regulated genes. The numeric value indicates the time point at which a gene is up- or down-regulated ( Dataset S2 , flagged column). A principal component analysis of the biological replicates of GFP − and GFP + cell populations at the three different time points demonstrates that GFP + cells are very different from GFP − cells. In addition, cell sorts performed in triplicate for each time point are highly re- producible ( Fig. S4 ). To define a set of the top 100 up- and down-regulated genes to use as candidates for validation, we ranked 193 up-regulated and 200 down-regulated significant (flagged) genes from the 1-h dataset as a function of the ratio and general abundance ( SI Materials and Methods and Table S1 ). We validated the RNA- Seq results by performing in situ hybridizations with 28 up-regulated and 21 down-regulated genes selected from the top 100 gene list in larvae 1 h after neomycin treatment ( Table S2 ). All 28 up- regulated genes are expressed in the lateral line, and 20 of these genes show up-regulation by in situ hybridization after neomycin treatment. Of the 21 down-regulated genes, 19 are expressed in the lateral line, and 12 genes are detectably down-regulated by in situ hybridization (Fig. 2 and Table S2 ). These experiments de- monstrated that the FACS sorting followed by RNA-Seq analysis produced high-quality results that enable us to study hair cell regeneration in zebrafish in detail.\nPathway Analysis at 1, 3, and 5 h After Hair Cell Death. The Wnt/ β -catenin, Notch, Jak/Stat, Bmp, and Fgf pathways are important for hair cell development as well as for the regeneration of hair cells in fish and chick epithelia (15, 23, 43). Our understanding of which pathways act upstream and how these pathways affect each other is still very limited. We performed a temporal ex- pression analysis at 1, 3, and 5 h after hair cell death to de- termine when particular pathways are activated or inhibited; this temporal information provides clues into how they might interact with each other. We also aimed to identify other signaling path- ways and record any dynamic changes in their behavior that could indicate how they integrate with the Notch and Wnt/ β -catenin signaling pathways. To identify enriched pathways in our datasets, we used the signaling pathway impact analysis (SPIA) package in\nccdc48\nR (44). This software tool not only takes into account how many genes belong to a pathway but also focuses on the importance of a gene by analyzing the perturbation of the pathway if the gene expression changes. Therefore, even pathways that are repre- sented by only a small number of genes can be classified as en- riched if these genes are important regulators of the pathway. For the SPIA analysis we included all flagged genes (flagged column) listed in Dataset S2 . For each time point the up- and down-regulated genes were combined and analyzed against a panel of pathways from zebrafish defined in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (45). The 1-h postneomycin treatment time point contained 509 genes from the National Center for Bio- technology Information Entrez database. The 3-h postneomycin treatment time point contained 401 Entrez genes, and the 5-h postneomycin treatment time point contained 418 genes.\nThe results of the SPIA analysis are shown in Table S3 . Au- tomated pathway annotation is powerful but subject to caveats, because genes can be annotated to function in several pathways, some of which might not be active in the lateral line. In addition, in the KEGG database, the Wnt pathway encompasses the ca- nonical Wnt/ β -catenin, the PCP, and the Wnt- regulated calcium pathways, potentially providing erroneous status assignment of the Wnt/ β -catenin pathway. We therefore tested the SPIA ana- lysis by in situ hybridization experiments with pathway members at various time points of regeneration (see below and Figs. 3 – 5). For improved visualization we generated heat maps of log2 ratio values of KEGG pathway members. We show only the genes that were flagged in our analysis and reported to be expressed in the lateral line by the zebrafish information network ( www.zfin.org ) or prior reports (46, 47).\nThe Wnt/ β -Catenin Pathway Is Not Activated During Early Stages of Lateral Line Hair Cell Regeneration. The analysis of hair cell re- generation in a Tg ( sqEt20;sqEt4 ) double-transgenic larva allows us to draw comparisons between gene expression and cellular changes at particular time points (Fig. 3 A ) (39). At 1 h after neomycin treatment almost all hair cells have been eliminated. Two hair cells remain that were too immature to be killed by neomycin. Proliferation of inner support cells begins within the first 2 h of regeneration (37). At 5 h two newly differentiating hair cells start to express GFP (Fig. 3 A ; arrows), and by 12 h after neomycin treatment six hair cells are present.\nWnt/ β -catenin signaling is required for proliferation in de- veloping and regenerating neuromasts (26, 48). Unexpectedly, the SPIA pathway analysis suggested that the Wnt/ β -catenin pathway is inactive at 1 h after hair cell death ( Table S3 ). To investigate this finding in more detail, we performed in situ hy- bridization experiments with Wnt/ β -catenin pathway genes in untreated and treated larvae at various time points after hair cell death. Our analysis showed that Wnt/ β -catenin signaling is not active in 5-dpf untreated control neuromasts. The ligand wnt10a is not expressed in 5-dpf control neuromasts, and we failed to detect any expression of dgfp in the Tg ( Tcf/Lef-miniP:dGFP ) Wnt/ β -catenin reporter line (Fig. 3 B ) (49). Even though Wnt/ β -catenin signaling is inactive, the β -catenin cotranscription fac- tor tcf7l2 ( tcf4 ) is expressed. Likewise, the Wnt receptors fzd7b and 8a are present, although whether they act as receptors for canonical or noncanonical Wnt signaling in neuromasts is un- known (Fig. 3 B ) (50, 51). At 1 h after neomycin treatment tcf7l2 ( tcf4 ), frz7b , and fzd8a are drastically down-regulated, and, along with wnt10a , their tcf4 wnt10a dgfp control 1h 3h 5h 10h control 1h 5h 12h 18h fzd8a fzd7b lef1 fzd7b fzd8a wif1 dkk1b tcf7l1a tcf7l2 fzd9b ctbp2 axin2 lef1 wnt8a csnk1a1 jun sfrp1a wnt10a cul1a siah1 gro2 sox17 tbl1x fbxw11b gsk3b csnk1a1 1h 3h 5h 1h 3h 5h fzd6 dkk2 control 1h 3h 5h 10h 12h 18h 24h GFP GFP Wnt 2 0 -2 A B C Fig. 3. The Wnt/ β -catenin pathway is inactive during the early stages of hair cell regeneration. ( A ) Still images of a Tg ( sqET20;sqET4 ) larval neuromast in the process of neomycin-induced regeneration. All hair cells, except for two immature hair cells, were killed by neomycin by 1 h after neomycin treatment. Two newly formed hair cells (arrows) start to express GFP at 5 h, and other pairs of hair cells (arrows) appear at 12 and 18 h. ( B ) In situ hybridization of Wnt/ β -catenin pathway genes in 5-dpf untreated (control) larvae and in larvae at different time points after neomycin treatment. Expression of wnt10a is in- creased at 3 and 5 h. Expression of tcf7l2 ( tcf4 ), fzd7b , and fzd8a is largely decreased at 1 h after neomycin. The Wnt/ β -catenin reporter line Tg ( Tcf/Lef-miniP: dGFP ) demonstrates activation starting at 12 h as shown by dgfp expression, but lef1 is not induced in regenerating neuromasts. ( C ) Heat map of selected Wnt/ β -catenin pathway genes based on RNA-Seq results. Log2 ratios of GFP − and GFP + cells at 1, 3, and 5 h after neomycin treatment are color coded (red: up- regulation, blue: down-regulation). Genes that are modulated in GFP − cells are neomycin-induced genes.\nexpression in the Wnt/ β -catenin Tg ( Tcf/Lef-miniP:dGFP ) reporter line remains low or undetectable. At 3 h after neomycin treatment wnt10a , tcf7l2 ( tcf4 ) and fzd7b , and fzd8a are up-regulated, leading to Wnt/ β -catenin pathway activation at 12 h as demonstrated by dgfp expression in the Tg ( Tcf/Lef-miniP:dGFP ) reporter line (Fig. 3 B ). The activation status of other Wnt/ β -catenin pathway genes at the different time points is shown in Fig. 3 C . Genes that change expression in the GFP − cells represent neomycin-induced genes. Our analyses reveal that Wnt/ β -catenin signaling is a late- responding pathway and does not play a role in the initiation of regeneration.\nThe Notch Pathway Is Inhibited 1 h After Neomycin Treatment. The Notch pathway is of particular interest because experimental down-regulation of Notch induces transdifferentiation of mam- malian support cells into hair cells (30, 52 – 57). In contrast to previous reports in zebrafish and chicken (9, 13), our SPIA pathway analysis suggested that the Notch pathway is inhibited immediately after hair cell death because of simultaneous up- regulation of numb (Fig. 4 A and B ). Numb inhibits Notch ac- tivity by regulating postendocytic sorting events and degradation of the Notch receptor and thereby regulates neural stem cell homeostasis (58, 59). However, because Numb is a multifunc- tional membrane-associated protein with additional functions\nindependent of the regulation of Notch signaling, we tested if the Notch pathway is indeed down-regulated by in situ hybridization of Notch pathway members. Our analysis of the Notch target genes her4.1 and notch3 confirms that Notch signaling is attenuated immediately 1 h after neomycin treatment and is accompanied\nFig. 5. The Jak1/Stat3 pathway is activated during the early stages of hair cell regeneration. ( A ) In situ hybridization shows that stat3 is increased at 1 and 3 h after neomycin treatment. ( B ) bmp5 is up-regulated at 1 h after neomycin treatment in situ. ( C ) Table depicting the status of signaling path- ways at different time points after hair cell death. All comparisons (no change, inhibition, or activation) were evaluated relative to untreated control samples. The Wnt, Notch, and Fgf pathways are inhibited immediately after neomycin treatment, whereas the cell cycle and Jak1/Stat3 pathways are activated during the early stages of regeneration. The Wnt and Notch pathways are reactivated at 12 h and 5 h after neomycin treatment, respectively.\nby down-regulation of sox2 (Fig. 4 A and B ) and up-regulation of atoh1a (Fig. 4 A and B ). The atoh1a target deltaD ( dld ) is up- regulated between 3 – 5 h after neomycin treatment, reflecting the appearance of hair cell precursors (Fig. 4 A and B ) (60). Notch signaling must be reinitiated for proper fate specification into hair and support cells and proper organ size control (9, 60). Accordingly, between 3 and 5 h after neomycin treatment (Fig. 4 A and B ), we observed renewed activation of the Notch target genes her4.1 , notch3 , and sox2 that likely is caused by the up- regulation of dld (60).\nWe postulate that down-regulation of Notch signaling is a characteristic event during hair cell regeneration in zebrafish. In contrast to zebrafish, the Notch target genes are decreased only modestly, and Notch signaling is still active, in hair cell-depleted neonatal mouse cochleae and 3- to 4-wk-old mouse utricles (55, 56, 61). Therefore we hypothesize that the failure of transient Notch down-regulation in mammals could be one of the reasons that mammalian hair cells do not regenerate spontaneously after injury.\nFgf Signaling Is Down-Regulated Immediately After Hair Cell Death. The Fgf pathway initiates dld and subsequently Notch signaling in the mouse cochlea, chicken inner ear, and zebrafish lateral line neuromasts (23, 60). Both pathways are active in 5-dpf un- treated neuromasts (Fig. 4 A and C and Fig. S5 D ). Identically to Notch pathway members, fgf3 and its transcriptional target pea3 are turned off 1 h after neomycin treatment. However, the Notch target her4.1 is reactivated at 3 h, whereas pea3 is not transcribed until 5 h (Fig. 4 A and C ). Therefore, it is possible that Notch signaling does not depend on Fgf signaling during regeneration; however, this hypothesis needs to be tested. The rapid down- regulation of Fgf signaling is in accordance with findings in chicken sensory epithelia, in which Fgfr3 is highly expressed in support cells but is down-regulated rapidly after hair cell death (62). Additionally, in the chicken down-regulation of Fgf sig- naling is required for regeneration to occur, because the addition of Fgf inhibits support cell division after damage (63).\nThe Cell Cycle Exit Genes cdkn1b , cdkn1c , and rbl2 Are Down- Regulated at 1 h After Hair Cell Death. The failure to down-regu- late cell cycle exit regulators is one of the reasons mammalian support cells do not proliferate in response to hair cell death (10, 12, 64, 65). Determining the earliest changes in the expression of cell cycle regulators and mapping correlating changes in signaling pathway activations will aid in identifying candidate pathways in- volved in cell cycle reentry. We detected the earliest changes in cell cycle genes at 1 h after neomycin treatment. The cyclin- dependent kinase (cdk) inhibitors cdkn1b ( p27/kip1 ) and cdkn1c ( p57/kip2 ), as well as the pocket domain-containing rbl2 , are down-regulated 1 h after neomycin treatment and are not up- regulated again until 5 – 10 h after neomycin treatment (Fig. 4 D and E ). Time-lapse analyses revealed that support cells enter the cell cycle 1 – 5 h after neomycin treatment (37), likely as a con- sequence of the down-regulation of the cdk inhibitors at 1 h after neomycin treatment. Because we did not observe any signs of Wnt/ β -catenin signaling activation, we conclude that Wnt/ β -catenin signaling is not required for the reentry of support cells into the cell cycle immediately after hair cell death. In contrast, down- regulation of Notch and Fgf signaling correlates with the down- regulation of cdk inhibitors. It is possible that active Notch or Fgf signaling induces cdk inhibitors, causing cells to exit the cell cycle or become quiescent (66). However, manipulation of Notch signal- ing does not affect the cell cycle in chick or murine inner ears (13, 30), suggesting that signaling pathways other than Wnt/ β -catenin and Notch are involved in triggering or inhibiting proliferation.\nThe Jak1/Stat3 Pathway Is Up-Regulated Immediately After Neomycin Treatment. The Jak1/Stat3 pathway regulates a plethora of cel- lular processes, such as cell proliferation, cell differentiation, and\ncell survival (67, 68). The Jak1/Stat3 pathway is up-regulated during zebrafish inner ear regeneration and is required for proliferation of cardiomyocytes during zebrafish heart regeneration (36, 69). Like- wise, in zebrafish lateral line regeneration stat3 is up-regulated immediately, 1 h after neomycin treatment, as evidenced by in situ hybridization (Fig. 5 A and Fig. S5 A ). Other Jak1/Stat3 pathway members that are highly up-regulated are jak1 , interleukin 6 signal transducer ( il6st ), the stat inhibitor and target gene suppressor of cy- tokine signaling ( socs3b ), and the ligand interleukin 11b ( il11b ) (Fig. 2 A and Fig. S5 A ). In zebrafish heart regeneration, the proliferation of cardiomyocytes is regulated by the Jak1/Stat3 downstream target relaxin 3a ( rln3a ) . rln3a also is highly up-regulated in GFP + cells 3 h after neomycin treatment ( Fig. S5 A ) (69). Possibly, Jak1/Stat3 sig- naling regulates proliferation in support cells in the first few hours after hair cell death, whereas Wnt/ β -catenin signaling is required for proliferation during later stages of regeneration.\nAnother signaling pathway that is modulated during zebrafish hair cell regeneration and that might interact with the pathways described above is the Bmp pathway. Bmp signaling influences hair cell differentiation during chicken and mouse inner ear development (70 – 73). During zebrafish hair cell regeneration, bmp5 is up-regulated between 1 and 5 h after hair cell death (Fig. 5 B and Fig. S5 B ), whereas the BMP inhibitor noggin 3 ( nog3 ) is not down-regulated until 5 h after hair cell death ( Fig. S5 B ). Because ligand expression by itself is not a reliable indicator for signaling pathway activation, future studies will need to de- termine how the activation status of Bmp signaling is affected by bmp5 and nog3 .\nOther pathways that respond to zebrafish lateral line hair cell death are the Mapk ( Fig. S5 C and D ), TNF- α , ( Fig. S5 E ), insulin ( Fig. S5 F ), and apoptosis signaling ( Fig. S5 G ) pathways and nitric oxide (NO) and reactive oxygen species (ROS) ( Fig. S5 H ), some of which increase proliferation in undamaged chicken ves- tibular support cells (74). For example, we observed up-regulation of nox1 , noxo1a , and noxa1 ( Fig. S5 H and Dataset S2 ). NO is a radical that regulates a wide variety of physiological processes and is important during liver regeneration (75).\nWe summarized the activation status of some of the major signaling pathways based on the in situ hybridization patterns of pathway members in Fig. 5 C . The Wnt/ β -catenin pathway is not active in homeostatic or newly regenerating neuromasts, and the Notch and Fgf pathways are down-regulated during the first few hours of regeneration. The Wnt/ β -catenin and Notch pathways seem to play a role only during later stages of regeneration. The cell cycle is activated immediately, possibly via cytokine stimu- lation of the Jak1/Stat3 pathway.\nIndependent Microarray Studies Support the Differential Expression of Genes Not Associated with Key Signaling Pathways. An in- dependent expression analysis performed by Steiner et al. and reported in this issue (42), used a transgenic line with an alkaline phosphatase enhancer [ Tg ( -4.7alp:mCherry )] instead of the Tg ( sqET20 ) line we used to isolate mantle cells by FACS anal- ysis. We compared our respective datasets to determine how similar the data are. We first analyzed the overlap of genes enriched in untreated mantle/lateral line cells in both studies. The obtained results are a clear indicator of how similar the sorted cell populations in the two studies are. We chose the top 1,000 up- regulated genes from our Dataset S1 by p-value and then ex- amined how many of these genes also are among the top 1,000 genes identified by Steiner et al. (dataset S1 in ref. 42). If the datasets were independent and unrelated, we would expect 53 genes to overlap by chance. We identified 201 common genes with a p-value of 3.9e-107, demonstrating that our two datasets are significantly related. All six mantle cell-specific genes iden- tified by Steiner et al. (42) also show enrichment in our untreated GFP + cell population ( Table S4 A , blue rows). Ten of the 26 top up- and down-regulated genes that we validated by in situ hybridization\nafter hair cell loss ( Table S2 ) and for which we found a match in the Steiner dataset also show significant changes in the Steiner microarray dataset ( Table S4 B , blue rows) (42). Several of the genes that were not present in the Steiner dataset are expressed more highly by in situ hybridization in inner support cells than in mantle cells, reflecting the nature of the transgenic lines used and the type of cells purified. Nine of the 14 genes that Steiner et al. identified by microarray analysis, by RT-PCR, or by in situ hybridization (42) also showed the same trend in expression changes in our RNA-Seq analysis ( Table S4 C , blue rows).\nThe overlap between a subset of our data and that of Steiner et al. (42), combined with the identification and corresponding validation of genes in our study, strongly indicates that the two lines of investigation are complementary, with most of the tar- gets identified by Steiner et al. (42) found in the mantle cells and the majority of genes identified in our study expressed primarily in the inner support cells. As such, our respective studies both validate each other and help resolve the cell-type specificity of the revealed differential gene expressions.\n\n\nDiscussion\nSeveral studies have aimed to induce mammalian hair cell re- generation by inactivating or activating candidate genes and pathways. Knockdown of the cell cycle exit regulators Cdkn1b ( p27/Kip1 ), p19/Ink4d , and pRb led to the initiation of cell cycle reentry and the differentiation of hair cells in injured epithelia in mice (20, 21, 64, 76, 77). Unfortunately, these ectopic hair cells cannot be maintained and eventually apoptose. Because direct manipulation of cell cycle regulators has not been successful in reconstituting a fully functional sensory epithelium in mammals, it is important to identify the mitogens that activate cell cycle entry in regenerating organisms. It is plausible that manipu- lations of upstream signaling events are a more promising ave- nue to regenerate mammalian hair cells.\nMore recently, in mice the down-regulation of Notch signaling caused transdifferentiation of support cells into functional hair cells without inducing proliferation (30). However, the number of regenerated hair cells was modest, and it is unclear if the accompanied depletion of support cells affects the function of the partially regenerated sensory epithelium in the long term. Although these efforts have not resulted in the regeneration of a fully functional, morphologically normal epithelium, they clearly demonstrate that the mammalian cells of the inner ear can be manipulated and coaxed away from their terminally differenti- ated state.\nKnowledge gained from studying the development of hair cells has demonstrated that the underlying molecular mechanisms are complex, with a multitude of interacting signaling pathways and feedback loops that ensure that an unspecified progenitor cell transitions to a prosensory cell that ultimately differentiates into a functional sensory cell. We also have learned that signaling pathways are activated and inhibited repeatedly as hair cell progenitors are specified and differentiate, further complicating their experimental manipulation. Hence, a global up- or down- regulation of a particular pathway is unlikely to lead to fully functional sensory epithelia.\nWe hypothesize that the regeneration of a complete sensory epithelium will require the manipulation of several pathways in the correct order. However, the model systems currently used to study the regeneration of sensory epithelia present unique ob- stacles. For example, in chicken, antibiotic-induced hair cell death occurs over a 24-h period, and the proliferation of support cells and differentiation of hair cells occurs 1 – 4 wk after injury (19). This long time frame makes it difficult to determine the precise time point when pathways are turned on and off, because support cells will be in different stages of regeneration. Likewise, a previous expression study of the regenerating zebrafish ear was\nperformed 24 h after the larvae were exposed to noise that does not cause death simultaneously in all hair cells (36). Further adding to the complexity, genes are used repeatedly during hair cell development and can turn from an activator into a repressor (23). We have selected the zebrafish lateral line tissue to eluci- date the gene-expression patterns triggered by hair cell death. The zebrafish lateral line is particularly well suited for these analyses, because all hair cells can be killed acutely within 30 min, and regeneration occurs within 24 h (Fig. 3 A ), allowing us to determine the earliest signals and responses in support cells after hair cell death.\nWnt/ β -Catenin Signaling Is Not Detected Early in Lateral Line Hair Cell Regeneration. Our analyses focused on pathways involved in hair cell development as they might be reused during regeneration (15, 23, 43, 78, 79). For example, Wnt/ β -catenin signaling is crucial for the development of many organs across the animal kingdom (80). During sensory development, Wnt/ β -catenin sig- naling governs the proliferation of unspecified progenitor cells, renders them competent to acquire a neural cell fate, and sub- sequently is involved in the differentiation of these postmitotic prosensory cells (3, 23, 81). Wnt/ β -catenin signaling also is in- volved in regeneration. Wnt/ β -catenin signaling controls stem cell maintenance, quiescence, and proliferation during mam- malian epithelial hair cell, zebrafish fin, mammalian intestine, hydra head, and planarian regeneration (82 – 86). Importantly, Wnt/ β -catenin also is crucial for regulating proliferation in de- veloping and regenerating zebrafish neuromasts (26, 48).\nIn the mouse cochlea, ectopic Wnt/ β -catenin signaling pro- motes proliferation of Lgr5 + support cells via the down-regula- tion of Cdkn1c (p27/Kip1); however, this effect diminishes as the animals mature (3, 20, 28, 81, 87). Because Wnt/ β -catenin sig- naling is down-regulated in the mammalian cochlea after birth, it was hypothesized that the absence of Wnt/ β -catenin signaling is responsible for the inability of support cells to proliferate in response to hair cell death (3, 28, 29, 65). Therefore, recent studies have focused on up-regulating the Wnt/ β -catenin path- way in uninjured cochleae with the expectation of coaxing mammalian support cells into proliferating and producing hair cells (3, 17, 28, 29, 81). These experiments indeed yielded ad- ditional hair cells, but the epithelium was disorganized.\nSurprisingly, our data demonstrate that in zebrafish, as in adult mice cochleae, Wnt/ β -catenin signaling is inactive in ho- meostatic neuromasts and is up-regulated only at 12 h after neomycin treatment. Therefore Wnt/ β -catenin signaling is un- likely to play a role in the initial triggering of the regenerative response in zebrafish, even though it is sufficient and necessary to induce proliferation during later stages of regeneration, for example by inhibiting Cdkn1b (p27/Kip1) and regulating dyclinD1 (3, 27, 87).\nNotch Signaling Is Down-Regulated Early in Lateral Line Hair Cell Regeneration. A growing number of studies have explored how the manipulation of Notch signaling could aid in regenerating mammalian hair cells. It is well established that the inhibition of Notch signaling causes an increase in the number of hair cells in uninjured and injured chicken, guinea pig, and mouse utricles and cochleae and in injured zebrafish ears and lateral line neu- romasts (52, 53, 55 – 57, 88). The presence of Notch is detrimental to hair cell differentiation, because it blocks the proneural gene atoh1a and thereby maintains support cells (9, 13, 65, 89, 90). The ability of support cells to transdifferentiate into hair cells after down-regulation of Notch signaling was exploited recently to restore a modest amount of hair cells in the injured adult mouse cochlea that led to some promising functional recovery of hearing (30). This experimental down-regulation of Notch sig- naling recapitulates our observations during zebrafish lateral line hair cell regeneration. In zebrafish, Notch signaling is transiently\ndown-regulated 1 h after hair cell death, correlating with the attenuation of sox2 and up-regulation of atoh1a . The interactions between Notch signaling, sox2 , and atoh1a have not yet been tested in the zebrafish lateral line; however, because of their behavior during regeneration, it is very likely that their regula- tion is similar to that occuring during mouse, Xenopus , and zebrafish central nervous system or sensory organ development (23, 52, 61, 91 – 93). During murine ear development Sox2 ex- pression depends on Notch signaling and is required for the acquisition of neural potential in the prosensory epithelium (52). Subsequently, for neurogenesis to proceed, Sox2 is shut off via the accumulation of proneural genes, such as Atoh (93 – 97). In contrast, Sox2 is maintained in Notch + support cells, where it inhibits hair cell differentiation by inhibiting the proneural basic helix-loop-helix gene Atoh1a (91).\nIt is plausible that the down-regulation of Notch signaling in neuromasts is required during the earliest stages of hair cell regeneration to down-regulate sox2 in support cells, allowing these cells to acquire a proneural character via the up-regulation of atoh1a . Because Notch signaling needs to be reinitiated for fate specification of hair and support cells and control of organ size (9, 60), the Notch target genes her4.1 , notch3 , and sox2 are again up- regulated between 3 and 5 h after neomycin treatment (Fig. 4 A and B ). These genes likely are activated by the up-regulation of dld (60). Therefore, we hypothesize that the orchestration of genes downstream of Notch signaling appears to be the same during zebrafish hair cell development and regeneration.\nIt has not been appreciated previously that the down-regula- tion of Notch signaling could be a hallmark of spontaneously regenerating sensory epithelia, because it was reported that the Notch signaling pathway is up-regulated in response to hair cell death in chicken, guinea pigs, and zebrafish (9, 88, 98). However, these expression analyses were performed at later time points during the regeneration process, when Notch signaling is re- quired once again for hair cell versus support cell specification to occur. We also observed that zebrafish Notch target genes are reexpressed at 3 h after hair cell death (Fig. 4 A ). Because the zebrafish lateral line system allows us to kill hair cells very acutely, causing a relatively synchronized response in support cells, we were able to detect the immediate, transient down-regulation of Notch signaling after hair cell death. In the injured mammalian cochlea or utricle such drastic down-regulation of Notch target genes, such as Sox2 and several Hes and Hey genes, does not occur (55, 56); the absence of such down-regulation could be one of the reasons mammalian hair cells do not regenerate.\nThe Regulation of Notch Signaling in Combination with Mitogenic Signals May Offer Novel Therapeutic Strategies to Promote Hair Cell Regeneration in Mammals. Our data suggest that identifying the molecules that down-regulate Notch signaling in regenerat- ing species will be important in efforts to understand why mammals fail to regenerate injured hair cells. Our RNA-Seq analyses identified the transmembrane protein Numb as a po- tential candidate for inhibiting Notch signaling. Future studies should focus on the sufficiency of Numb to down-regulate Notch signaling in zebrafish and understanding Numb ’ s temporal ex- pression patterns during homeostasis and after injury in other regenerating species, as well as in mice. Numb could open a new therapeutic avenue for manipulating Notch signaling in mammals and facilitating successful regeneration of hair cells in mammals.\nEven though down-regulation of Notch signaling induced differentiation of new hair cells in mammals, these hair cells developed via transdifferentiation of support cells into hair cells, leading to the depletion of support cells (30). The down-regu- lation of Notch was not sufficient to induce support cell pro- liferation as observed in spontaneously regenerating species. Zebrafish regenerate hair cells mainly or exclusively by pro- liferation of support cells, and regenerating chicken hair cells\narise via transdifferentiation, as well as proliferation (19). In zebrafish we observe that the down-regulation of Notch signaling at 1 h after neomycin treatment correlates with the down-regu- lation of cyclin-dependent kinase inhibitors cdkn1b (p27/kip1) and cdkn1c ( p57/kip2 ) (Fig. 4 A and D ). This result suggests that Notch could be an activator of cdkn1b ( p27/kip1 ) that keeps cells out of the cell cycle and therefore that the down-regulation of Notch signaling could be a crucial event for the onset of re- generation. Indeed in the murine inner ear, the Notch target Sox2 regulates Cdkn1b (p27/Kip1), similar to activity described in the epidermis (99, 100) but in contrast to the activity in the intestinal crypt and inner ear, where Notch inhibits Cdkn1b (p27/ Kip1) (101, 102). In zebrafish the attenuation of Notch signaling has no effect on proliferation in uninjured sensory organs and has only a proproliferative effect on support cells after injury (9). In chicken cochleae the proliferation rate is not increased after injury, nor does inhibition of Notch cause cell cycle reentry in mice cochleae (13, 19, 30). Based on these findings it was con- cluded that Notch signaling is not required to initiate pro- liferation during regeneration (13). Therefore, it is possible that Cdk inhibitors are regulated by as yet unidentified, Notch- independent signals. Possibly, Fgf signaling inhibits proliferation, as demonstrated in the chicken inner ear (63).\nBecause complete regeneration of sensory epithelia in mam- mals requires coaxing support cells into hair cell differentiation as well as stimulating the support cells to re-enter the cell cycle and self-renew, manipulation of Notch signaling is not sufficient and needs to be married with manipulations of mitogenic signals. Furthermore, similar to our observations during zebrafish hair cell regeneration, Notch signaling must be reinitiated for the inhibition of proneural genes and specification of support cells (12, 23, 91). A temporally restricted Notch down-regulation or a combination of Notch down-regulation with subsequent ectopic expression of support cell inducers, such as Sox2, could be fruitful.\nOur efforts presented here have characterized the pattern of temporal activation of signaling pathways during zebrafish sensory hair cell regeneration. These results will aid in the design and in- terpretation of functional studies aimed at elucidating the gene-reg- ulatory network underlying zebrafish hair cell regeneration and can guide experiments to trigger hair cell regeneration in mammals.\n\n\nThis article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1402898111/-/DCSupplemental . www.pnas.org/cgi/doi/10.1073/pnas.1402898111 PNAS | Published online March 27, 2014 | E1383 – E1392 DEVELOPMENTA L BIOLOGY PNAS PLUS SEE CORRECTION FOR THIS ARTICLE\n\n\ncontrol 1h ptp4a2 lima1 junba abcf2a eif4a1a control 1h il11b nlrc3 hk2 (glulc) g3bp1 rapgef3 txn clic2 1h lad1 znf462 dfna5 gpr37b zgc:110353 slc35c2 tcea3 A B control zgc:152741 Fig. 2. Validation by in situ hybridization of a selection of 14 genes up- regulated ( A ) and seven genes down-regulated ( B ) at 1 h after neomycin treatment. ( Upper Rows ) Untreated larvae (control) at 5 dpf. ( Lower Rows ) Treated larvae.\n\n\nPNAS PLUS\n\n\nrbl2\n\n\nher4.1 notch3 1h 3h 5h 10h cdkn1c cdkn1b 1h 3h 5h 10h C A numb sox2 atoh1a dld pea3 fgf3 1h 3h 5h 10h D her4.1 atoh1b her6 numbl dlc gro2 ctbp2 notch3 cir1 jag2 aph1b ncor2 dld dla atoh1a numb cdk6 prkdc rbl2 mad2l2 cdkn1c cdkn1b ccna1 gadd45aa gadd45ba gadd45ab cdk7 plk1 ywhag1 pcna ccne1 anapc1 ywhai abl1 stag2 gadd45g mcm6l wee1 1h 3h 5h 1h 3h 5h 1h 3h 5h 1h 3h 5h B E control control control Notch cell cycle notch1b jag1a jag1b rbpjb sox2 GFP GFP GFP GFP 2 0 -2 2 0 -2 Fig. 4. Inhibition of Notch and Fgf pathways is accompanied with cell cycle reentry immediately after hair cell ablation. ( A , C , and D ) In situ hybridization of Notch pathway, Fgf pathway, and cell cycle genes in 5-dpf untreated and neomycin-treated larvae. ( A ) The Notch pathway inhibitor numb is increased at 1 h after neomycin. The Notch target genes her4.1 , notch3 , and sox2 are decreased at 1 h and up-regulated at 3 h after neomycin treatment. a toh1a is up- regulated starting at 1 h after neomycin, and its target dld is increased at 5 and 10 h. ( C ) fgf3 and its target pea3 are down-regulated at 1 and 3 h after neomycin treatment. ( D ) Cell cycle inhibitory genes cdkn1b , cdkn1c , and rbl2 are decreased at 1 and 3 h after neomycin compared with control. ( B and E ) Heat maps of selected Notch pathway genes and cell cycle genes.\n\n\nbmp5 control 1h 3h 5h 10h A stat3 1h 3h 5h 10h control B C Wnt Notch Fgf cell cycle Stat3 1h 3h 5h 10h 12h no change inhibition activation not tested\n\n\nPNAS PLUS\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nPNAS PLUS\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nE1384 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1385 DEVELOPMENTA L\n\n\nE1386 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1387 DEVELOPMENTA L\n\n\nJiang et al. PNAS | Published online March 27, 2014 | E1389 DEVELOPMENTA L\n\n\nE1390 | www.pnas.org/cgi/doi/10.1073/pnas.1402898111\n\n\n1. White PM, Doetzlhofer A, Lee YS, Groves AK, Segil N (2006) Mammalian cochlear supporting cells can divide and trans-differentiate into hair cells. Nature 441(7096): 984 – 987.\n\n\n2. Corwin JT, Oberholtzer JC (1997) Fish n ’ chicks: Model recipes for hair-cell regeneration? Neuron 19(5):951 – 954.\n\n\n5. Williams JA, Holder N (2000) Cell turnover in neuromasts of zebrafish larvae. Hear Res 143(1-2):171 – 181.\n\n\n8. López-Schier H, Hudspeth AJ (2006) A two-step mechanism underlies the planar polarization of regenerating sensory hair cells. Proc Natl Acad Sci USA 103(49): 18615 – 18620.\n\n\n9. Ma EY, Rubel EW, Raible DW (2008) Notch signaling regulates the extent of hair cell regeneration in the zebrafish lateral line. J Neurosci 28(9):2261 – 2273. 10. Groves AK (2010) The challenge of hair cell regeneration. Exp Biol Med (Maywood) 235(4):434 – 446.\n\n\n27. Wada H, et al. (2013) Wnt/Dkk negative feedback regulates sensory organ size in zebrafish. Curr Biol 23(16):1559 – 1565.\n\n\n28. Shi F, Hu L, Edge AS (2013) Generation of hair cells in neonatal mice by β -catenin overexpression in Lgr5-positive cochlear progenitors. Proc Natl Acad Sci USA 110(34): 13851 – 13856.\n\n\n33. Whitfield TT (2002) Zebrafish as a model for hearing and deafness. J Neurobiol 53(2): 157 – 171.\n\n\n56. Korrapati S, Roux I, Glowatzki E, Doetzlhofer A (2013) Notch signaling limits sup- porting cell plasticity in the hair cell-damaged early postnatal murine cochlea. PLoS ONE 8(8):e73276.\n\n\n57. Lin V, et al. (2011) Inhibition of Notch activity promotes nonmitotic regeneration of hair cells in the adult mouse utricles. J Neurosci 31(43):15329 – 15339.\n\n\n58. McGill MA, Dho SE, Weinmaster G, McGlade CJ (2009) Numb regulates post-endo- cytic trafficking and degradation of Notch1. J Biol Chem 284(39):26427 – 26438.\n\n\n60. Matsuda M, Chitnis AB (2010) Atoh1a expression must be restricted by Notch sig- naling for effective morphogenesis of the posterior lateral line primordium in ze- brafish. Development 137(20):3477 – 3487.\n\n\n63. Oesterle EC, Bhave SA, Coltrera MD (2000) Basic fibroblast growth factor inhibits cell proliferation in cultured avian inner ear sensory epithelia. J Comp Neurol 424(2): 307 – 326.\n\n\n69. Fang Y, et al. (2013) Translational profiling of cardiomyocytes identifies an early Jak1/Stat3 injury response required for zebrafish heart regeneration. Proc Natl Acad Sci USA 110(33):13416 – 13421.\n\n\n74. Oesterle EC, Tsue TT, Rubel EW (1997) Induction of cell proliferation in avian inner ear sensory epithelia by insulin-like growth factor-I and insulin. J Comp Neurol 380(2): 262 – 274.\n\n\n75. Carnovale CE, Ronco MT (2012) Role of nitric oxide in liver regeneration. Ann Hepatol 11(5):636 – 647.\n\n\n76. Sage C, et al. (2005) Proliferation of functional hair cells in vivo in the absence of the retinoblastoma protein. Science 307(5712):1114 – 1118.\n\n\n77. Sage C, et al. (2006) Essential role of retinoblastoma protein in mammalian hair cell development and hearing. Proc Natl Acad Sci USA 103(19):7345 – 7350.\n\n\n80. van Amerongen R, Nusse R (2009) Towards an integrated view of Wnt signaling in development. Development 136(19):3205 – 3214.\n\n\n81. Shi F, Kempfle JS, Edge AS (2012) Wnt-responsive Lgr5-expressing stem cells are hair cell progenitors in the cochlea. J Neurosci 32(28):9639 – 9648.\n\n\n82. Galliot B, Chera S (2010) The Hydra model: Disclosing an apoptosis-driven generator of Wnt-based regeneration. Trends Cell Biol 20(9):514 – 523.\n\n\n83. Fleming HE, et al. (2008) Wnt signaling in the niche enforces hematopoietic stem cell quiescence and is necessary to preserve self-renewal in vivo. Cell Stem Cell 2(3): 274 – 283. 84. Li L, Bhatia R (2011) Stem cell quiescence. Clin Cancer Res 17(15):4936 – 4941.\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [6.87s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"zebrafish lateral line\",\n  \"tissue_name_structured\": \"zebrafish neuromast hair cells\",\n  \"developmental_stage_text\": \"5 days post-fertilization (dpf)\",\n  \"developmental_stage_coarse\": \"larval\",\n  \"developmental_stage_fine\": \"larval\",\n  \"is_scRNA_data\": true\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"zebrafish lateral line\",\n  \"tissue_name_structured\": \"zebrafish neuromast hair cells\",\n  \"developmental_stage_text\": \"5 days post-fertilization (dpf)\",\n  \"developmental_stage_coarse\": \"larval\",\n  \"developmental_stage_fine\": \"larval\",\n  \"is_scRNA_data\": true\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 100,
      "prompt_tokens": 15407,
      "total_tokens": 15507
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_d986a8d1ba"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "zebrafish lateral line",
  "tissue_name_structured": "zebrafish neuromast hair cells",
  "developmental_stage_text": "5 days post-fertilization (dpf)",
  "developmental_stage_coarse": "larval",
  "developmental_stage_fine": "larval",
  "is_scRNA_data": true
}
[chain/end] [1:chain:RunnableSequence] [6.88s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "zebrafish lateral line",
  "tissue_name_structured": "zebrafish neuromast hair cells",
  "developmental_stage_text": "5 days post-fertilization (dpf)",
  "developmental_stage_coarse": "larval",
  "developmental_stage_fine": "larval",
  "is_scRNA_data": true
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Introduction\nHuman hepatocellular carcinoma (HCC) is known to be a very heterogeneous disease, especially at intermediate and advanced stages &lt;&lt;REF:Llovet-1999-29-62&gt;&gt;. Due to the diverse and complex etiologies contributing to HCC incidence, different genetic mutations or altered molecular pathways could be responsible for hepatocarcinogenesis. So far, several carcinogenic pathways have been identified to be involved in the development and progression of HCC, including the VEGFR, EGFR, and mTOR pathways &lt;&lt;REF:Siegel-2010-52-360&gt;&gt;. In effort to decipher the role of different oncogenic pathways, a number of transgenic mouse models have been established &lt;&lt;REF:Leenders-2008-14-6915&gt;&gt;, &lt;&lt;REF:Lee-2004-36-1306&gt;&gt; and comparative transcriptomic analyses have been used to identify the best transgenic mouse models for human HCCs &lt;&lt;REF:Lee-2004-36-1306&gt;&gt;.\nThe zebrafish has been increasingly recognized as a valuable experimental model for human diseases, particularly for cancers &lt;&lt;REF:Liu-2011-6-71&gt;&gt; including liver cancers &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Liu-2012-56-2268&gt;&gt;. It has been shown that the zebrafish tumors had striking similarities with human cancer histologically &lt;&lt;REF:Amatruda-2002-1-229&gt;&gt;, &lt;&lt;REF:Spitsbergen-2000-28-705&gt;&gt;. Transcriptomic and epigenetic analyses have also indicated conserved features of carcinogen-induced zebrafish HCC with human HCC &lt;&lt;REF:Mirbahai-2011-12-3&gt;&gt;-&lt;&lt;REF:Lam-2006-24-73&gt;&gt;.\nPreviously we have generated several liver tumor models by transgenic expression of three different oncogenes (kras, xmrk or Myc) specifically in the zebrafish liver and these transgenic zebrafish usually produce liver tumors with variable degrees of severity from hepatocellular adenoma (HCA) to HCC &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;. The three oncogenes we used in the zebrafish have all been shown to be involved in hepatocarcinogenesis. KRAS is mutated in ~7% of liver cancers in human &lt;&lt;REF:Karnoub-2008-9-517&gt;&gt; but Ras signaling is ubiquitously activated in HCC &lt;&lt;REF:Calvisi-2006-130-1117&gt;&gt;. Xmrk is a naturally occurring variant of the EGFR in fish of the genus Xiphophorus (platyfish and swordtails) with constitutive autophosphorylation and activation of downstream signals &lt;&lt;REF:Gomez-2001-276-3333&gt;&gt;. Activation of EGFR signal is correlated with poor prognosis of HCC patients &lt;&lt;REF:Foster-2007-14-752&gt;&gt;. MYC is commonly amplified in many cancers including HCC and higher expression level of MYC is associated with more advanced status of HCC &lt;&lt;REF:Gan-1993-41-1185&gt;&gt;. We have shown that overexpression of kras and xmrk in the zebrafish liver could induce HCC &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;, while overexpression of Myc induced mostly HCA &lt;&lt;REF:???-?-?-?&gt;&gt;.\nComparative transcriptomic analyses of animal models and human clinical samples provide a powerful tool for identification of conserved molecular pathways and biomarker genes for diagnosis and therapy &lt;&lt;REF:Lam-2006-5-573&gt;&gt;, &lt;&lt;REF:Sweet-Cordero-2005-37-48&gt;&gt;. Our existing oncogene transgenic zebrafish models have well defined up-regulation of driver oncogene and this may provide a valuable tool to identify the molecular driving forces in human carcinogenesis by comparative transcriptomic analyses. Thus, in this study, we employed RNA sequencing technology to compare the transcriptomic profiles of the three oncogene-induced liver tumors in transgenic zebrafish. By comparative analyses with human liver transcriptomes from cirrhotic livers to very advanced HCC, we found that they all showed strong molecular correlation with advanced or very advanced human HCCs. Nevertheless, there are quite distinct deregulated biological pathways based on deregulated genes in the three oncogene transgenic models. Interestingly, each zebrafish liver tumor model correlated with a subset of human HCCs and each subset has some distinct molecular features. We showed that the transgenic zebrafish models with well-defined driver-gene activity should be valuable for classification of human HCCs and for understanding the molecular mechanisms behind each HCC subtype.\n\nMaterials and Methods\n\nTreatment and Induction of Liver Cancer in the Three Zebrafish Transgenic Models\nZebrafish were maintained following the approved protocol by Institutional Animal Care and Use Committee of National University of Singapore (Protocol 079/07). The generation of xmrk and Myc transgenic zebrafish was previously described and they were termed as TO(xmrk)\n&lt;&lt;REF:Li-2011-56-419&gt;&gt; and TO(Myc)\n&lt;&lt;REF:???-?-?-?&gt;&gt; respectively in the previous publications. The two transgenic lines were constructed by using a tetracycline-inducible transgenic system and the oncogene expression were induced by doxycycline. The krasV12 transgenic line used in the present study was newly generated by using a mifepristone-inducible system &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt; in combination with a Cre-loxP system (unpublished). For the xmrk and Myc transgenic lines, transgenic fish and their non-transgenic siblings were treated with 60 microg/ml doxycycline (Sigma, USA) starting from 3.5 mpf (month post fertilization) for 6 weeks. All xmrk fish developed HCC and all Myc fish developed HCA. In total, for each transgenic line, one tumor sample (transgenic fish treated with doxycycline) and three control samples (non-transgenic siblings similarly treated with doxycycline, transgenic siblings without doxycycline treatment, and non-transgenic siblings without doxycycline treatment) were collected for RNA sequencing. In all cases, liver samples used for RNA sequencing were pooled from four to five male fish. For the krasV12 transgenic fish, one-month-old transgenic fish were treated with 1 microM mifepristone (Sigma, USA) for 36 hours to induce Cre-mediated recombination for activation of krasV12 transgene expression in the liver. The krasV12 activated transgenic fish were then allowed to grow for six months to develop HCC and HCC samples were then collected for RNA sequencing. Three liver tumors from induced transgenic fish and three normal livers from uninduced transgenic fish were pooled separately as tumor and control samples. All samples used were from male fish and two sets of biological replicates were used.\n\nIdentification of Signature Gene Lists in Each Zebrafish Liver Cancer Model\nTotal RNA was extracted using TRIzol Reagent (Invitrogen, USA) and treated with DNase I to remove genomic DNA contamination. 3' RNA-SAGE (serial analysis of gene expression) sequencing was performed on ABI SOLiD platform by Mission Biotech (Taiwan) according to manufacturer's protocol and 10-23 million reads were generated from each sampler (Table S1). Briefly, mRNA was purified using Dynabeads Oligo(dT) EcoP (Invitrogen) and subjected to cDNA synthesis. Resultant cDNA was digested by NlaIII and EcoP15I to result in a 27 nucleotides cDNA tag between the two sequencing adapters. The tags were mapped to the NCBI RefSeq (Reference Sequence) mRNA database for zebrafish with a criterion of allowing maximum 2 nucleotide mismatches. All RNA-Seq data were submitted to Gene Expression Omnibus database with the following access numbers: GSE53342 for xmrk and Myc data and GSE53630 for kras data. Tag counts for each transcript were normalized to TPM (transcripts per million) to facilitate comparison among different samples. The differentially regulated genes in the Myc- and xmrk- induced liver cancers were identified using one sample t-test as previously described &lt;&lt;REF:Zheng-2011-6-e24019&gt;&gt;, and the differentially expressed genes in the kras-induced liver cancer was identified using two-tailed Student's t-test.\nTo facilitate functional implications of zebrafish transcriptome, all zebrafish genes were mapped to annotated human genes in order to use existing online software developed in human genes. Thus, human homology mapping of zebrafish Unigene clusters were retrieved from the Genome Institute of Singapore Zebrafish Annotation Database (http://123.136.65.67/). For Unigene clusters mapped by more than one transcript entries, the highest TPM was used to represent the expression level of the Unigene cluster. Some zebrafish Unigene clusters were mapped to more than one human Unigene clusters, which usually came from the same gene family. To remove redundancy and avoid causing bias in functional analyses, only the first human Unigene cluster in the list was selected to represent the zebrafish Unigene clusters. The lists of significantly up-regulated zebrafish genes that were mapped with human homologs and used for comparative analyese with human HCC data are shown in Table S2.\n\nGene Set Enrichment Analysis (GSEA)\nGSEA was used to establish the relatedness between the transgenic zebrafish models and human liver cancers &lt;&lt;REF:Subramanian-2005-102-15545&gt;&gt;. GSEA is a computational method that determines whether a priori defined set of genes shows statistically significant, concordant differences between two biological samples; it calculates an enrichment score using a running-sum statistic through a ranked list of gene expression data set. Human homologs of the significantly up-regulated genes from the zebrafish tumor tissues were used as cancer signatures for each transgenic zebrafish model for transcriptomic comparison with human HCC data. Each chosen phenotype of human HCCs (either one specific HCC stage or one particular HCC sample) was compared to the rest of the samples in the same dataset. All genes in the chosen phenotype were ranked by t-test to determine expression differences among different HCC stages or different HCC samples. The enrichment score of the pre-defined transgenic zebrafish cancer signature was calculated using a running-sum statistic through the ranked genes. The statistical significance of the enrichment score was estimated by using an empirical phenotype-based permutation test procedure. An FDR (false discovery rate) value was provided by introducing adjustment of multiple hypothesis testing. Human liver cancer transcriptome data were retrieved from Gene Expression Omnibus (GEO) database. The human dataset including different stages of HCCs used in the comparison was GSE6764 &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The ten human HCC datasets used for examining the representation of zebrafish liver cancer gene signatures are summarized in Table S3. Annotation information was retrieved from SOURCE (http://smd.stanford.edu/cgi-bin/source/sourceSearch). For multiple probes which can be mapped to one Unigene cluster, the maximum signal intensity was selected to represent the expression level of the Unigene cluster.\n\nGSEA Pre-ranked Analysis\nGSEA pre-ranked option was used to analyze the deregulated pathways in each transgenic zebrafish model and subgroups of human HCCs. Briefly, the entire transcriptome was ranked by logarithm transformed p-value (base 10). The up-regulated genes were given positive values, and the down-regulated genes negative values. The curated canonical pathways from the MSigDB (Molecular Signature Database) were used. The number of permutation used was 1000.\n\nRT-qPCR Validation\nTotal RNA were reverse-transcribed using the SuperScript II cDNA Synthesis Kit (Invitrogen). RT-qPCR was performed with same sets of cDNAs used for SAGE sequencing using the LightCycler 480 SYBR Green I Master system (Roche). Reactions were conducted in triplicate for each cDNA sample and primer sequences are shown in the Table S4. Gene expression levels in each control or transgenic liver sample were normalized with the level of beta-actin mRNA as the internal control. The log2 fold changes between tumor and control samples were calculated using the CT method according to the formula: log2 fold changes = -DeltaDeltaCT = -[(CT gene of interest-CT beta-actin) transgenic sample-(CT gene of interest-CT beta-actin) control sample]. Two-tailed heteroscedastic t test was performed using normalized CT values (CT gene - CT beta-actin) and changes with p&lt;0.05 are considered to be significant.\n\nResults\n\nIdentification of Differentially Expressed Genes in the Three Transgenic Zebrafish Liver Cancer Models\nThe three oncogene transgenic lines (xmrk, Myc and krasV12) were induced to develop liver tumors (Figure S1) and these tumor samples were subjected to RNA-SAGE sequencing. By a selection criteria of fold change&gt;1.5, p value&lt;0.05 and TPM&gt;10 (in either control or tumor samples), differentially expressed genes were selected from the three tumor sets. There were 2,892, 797 and 494 genes up-regulated and 169, 902 and 676 genes down-regulated in the xmrk-, kras- and Myc- induced zebrafish liver cancer, respectively (Figure 1A,B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;). Deregulated genes from the three transgenic models showed relatively small overlaps, indicating that the three oncogenes regulated quite distinct sets of genes. This is consistent with the report that morphologically uniform cancer type is frequently classified into different subgroups based on their distinct gene expression patterns &lt;&lt;REF:Lee-2005-26-1013&gt;&gt;. Interestingly, there were 21 up-regulated and 16 down-regulated genes commonly found in all the three tumor models (Figure 1A, B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table 1).\nFigure 1\nDeregulated genes and pathways in xmrk, kras and Myc transgenic liver tumors.\n(A) Venn diagram of up-regulated genes in the three zebrafish transgenic HCC models. (B) Venn diagram of down-regulated genes in the three zebrafish transgenic HCC models. (C) Deregulated pathways in the three oncogene transgenic liver tumors. Up- and down-regulated pathways in each zebrafish transgenic HCC model were analyzed by GSEA pre-ranked analysis. Red and green indicate up-and down-regulation respectively and the color code based on FDR is shown on the left. Pathways correlating with grey cells were not detected with any change. The up-regulated pathways were assigned into seven cancer hallmark categories (excluding enabling replicative immortality) according to Hanahan and Weinberg &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt; and down-regulated pathways was classified based on different aspects of the liver metabolism (see Table S1 for details).\nTable 1\nCommonly deregulated genes in the three transgenic zebrafish liver cancer models.\n\nDistinct Pathways Regulated in the Three Zebrafish Liver Tumor Models\nPathway analysis using GSEA showed that the three transgenic liver cancer models have different pathways deregulated (Figure 1C &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table S5). It has been widely accepted that there are eight cancer hallmarks for multistep tumorigenesis and the complexities of neoplasms &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt;, &lt;&lt;REF:Hanahan-2000-100-57&gt;&gt;. We found that the GSEA-identified pathways fallen into at least seven cancer hallmarks (except for enabling replicative immortality). Xmrk mainly up-regulated pathways involved in evading growth suppressors and avoiding immune destruction, which included activating cell cycle, promoting RNA transcription, up-regulating proteasome and altering immune properties. Kras provided the tumor cells with the ability of self-sustaining proliferative signals by up-regulating EGFR, Raf-MEK-ERK, PI3K-AKT-mTOR and GSK3 signaling pathways. Specifically, it also altered the focal adhesive characters of tumor cells which could activate invasion and metastasis. Myc mainly up-regulated translation and proteolysis to assist tumor cells to evade growth suppressors, and it also up-regulated VEGF pathway, thus potentially inducing angiogenesis. While there was no single pathway significantly up-regulated in all three tumor models, there were some pathways up-regulated in two models, such as proteasome in the xmrk and Myc models, and IGF1 pathway, mTOR pathway, tRNA biosynthesis and focal adhesion in the xmrk and kras models. In contrast, pathways in reprogramming of energy metabolism were generally down-regulated in all three models though the down-regulation in the Myc model was less apparent than the other two models. Meanwhile, many other pathways involved in normal liver function such as blood factors, amino acid metabolism, detoxification and xenobiotic metabolism, fatty acid metabolism, and bile synthesis were uniformly down-regulated in all three tumor models. However, one exception was hormone regulation that was apparently up-regulated in the kras tumors.\n\nCorrelation of the Three Gene Signatures of Zebrafish Liver Tumors with Different Stages of Human HCCs\nThe up-regulated genes, 2,892, 797 and 494 from xmrk, kras and Myc tumors respectively, were used as the signature genes for each model. These up-regulated genes were converted to 1,362, 490, and 146 human Unigenes respectively in order to compare with available transcriptomic data from human studies. We then compared the three signature gene sets with a set of human transcriptomic data (GSE6764) from different stages of human liver conditions: cirrhotic liver, low grade dysplastic nodules (LGDN), high grade dysplastic nodules (HGDN), and very early, early, advanced, and very advanced HCC (veHCC, eHCC, aHCC and vaHCC), in which the pathological HCC stages have been defined by tumor size, differentiation status and metastasis level &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The three signature gene sets were all up-regulated as the disease progresses, but they started to be up-regulated at different stages of tumorigenesis (Figure 2 &lt;&lt;FIG:pone-0091179-g002&gt;&gt;). The xmrk signature showed positive correlation with HCCs starting from eHCC, and it was significantly correlated with aHCC and vaHCC. The kras signature positively correlated with aHCC and vaHCC, and it was only significantly correlated with vaHCC. The Myc signature showed significantly positive correlation with HCCs starting from eHCC, similar with our previous result using an independent set of RNA-seq data from the same transgenic line &lt;&lt;REF:???-?-?-?&gt;&gt;. Interestingly, the common 21 up-regulated genes in all three tumor models also showed up-regulation even from the very early stage of HCC, indicating these genes are correlated with the entire neoplastic process.\nFigure 2\nCorrelation of the zebrafish liver tumor signatures with different stages of human hepatocarcinogenesis.\nThe up-regulated gene signatures from the three transgenic tumors (xmrk, kras and Myc) and 21 commonly up-regulated genes (Common) were used for GSEA and NESs (normalized enrichment scores (y-axis) are plotted against different stages of human HCCs (x-axis). NES is listed in the table and asterisk indicate statistical significance: FDR&lt;0.25. Abbreviations: CL, cirrhotic livers; LGDN, low grade dysplastic nodules; HGDN, high grade dysplastic nodules; veHCC, very early HCC; eHCC, early HCC; aHCC, advanced HCC; vaHCC, very advanced HCC.\n\nRepresentation of the Three Gene Signatures of Zebrafish Liver Tumors in Human HCC Samples\nGiven the distinctive pathways altered in the three zebrafish liver tumor models and the high level of heterogeneity in human HCC patients, we sought to examine the degree of representation of the three zebrafish liver cancer models in human HCC samples. We examined ten sets of published clinical HCC microarray data, each of which included at least 80 clinical samples. The ten datasets contain a total of 1,272 samples covering different ethnic groups and risk factors (Table S2). We found that the xmrk, kras and Myc gene signatures were significantly correlated with 30.8%, 24.8% and 25.6% of all clinical samples, respectively. 47.2% of the human HCC samples had significant correlation with at least one of the three zebrafish gene signatures (Table 2, Figure S2). In different human HCC datasets, the percentage ranges from 22.0% to 60.4%. Moreover, some of the clinical samples were correlated with two or even three zebrafish gene signatures. Co-correlation of two gene signatures, namely xmrk/kras, kras/Myc and kras/Myc, accounted for 17.5%, 13.5% and 12.4% of the human HCC samples. 9.3% of the human samples showed co-correlation of all the three gene signatures. Thus, it appears that the three transgenic zebrafish liver tumor models represent molecular mechanisms of hepatocarcinogenesis in almost half of the human HCC cases and the other half of human HCC may be due to different molecular mechanisms.\nTable 2\nDegree of representation of the three transgenic zebrafish liver cancer signatures in human HCCs.\n\nPercentages indicate the percentages of human HCC samples which showed significant positive correlation (FDR&lt;0.25) with the zebrafish liver cancer signatures.\n\n*Total: The total percentage of human HCCs which showed significant correlation with any one or more of the zebrafish signatures.\nWe further demonstrated that the subsets of human HCCs which showed significantly positive correlation with the same gene signature also shared similar up-regulated pathways (Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, Table S6). For this analysis, each of the human HCC sets was separated into two subgroups: those showing significant correlation with one of the zebrafish signatures, and the rest. Differentially expressed genes between the two groups were identified by two-tailed t-test, and pathway analyses were performed by GSEA pre-ranked analysis. The pathways were subjected to hierarchical clustering by logarithm-transformed FDR values using MeV &lt;&lt;REF:Saeed-2003-34-374&gt;&gt;, &lt;&lt;REF:Saeed-2006-411-134&gt;&gt;. As shown in Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, the pathways differentially regulated in human HCCs which showed significant correlation with one of the zebrafish signatures had distinct patterns. The pathway cluster A is consisted of pathways up-regulated in all the three subgroups, including proteasome, tRNA biosynthesis and oxidative phosphorylation. Ribosome, transcription and translation, and cell cycle were also highly up-regulated, suggesting that the human HCC samples significantly correlated with any of the zebrafish signatures were probably more proliferative than those not significantly correlated. The pathway cluster B contains pathways down-regulated in most of the subgroups associated with xmrk but up-regulated in the subgroups associated with kras and Myc. Interestingly, theses pathways were consistent with highly down-regulated pathways in the xmrk-induced zebrafish HCC, including energy metabolism, amino acid metabolism, fatty acid metabolism, bile acid biosynthesis, complement pathway, biosynthesis of steroid, and N-glycan biosynthesis. The pathway cluster C is more up-regulated in the kras-associated HCC subgroups, including glutamate metabolism and glycolysis. It has been reported that Kras could increase the conversion of glucose to glutamate, and this was essential for Kras-mediated tumorigenicity &lt;&lt;REF:Weinberg-2010-107-8788&gt;&gt;. The pathway cluster D was generally up-regulated in the xmrk- and kras-correlated human HCCs, but showed a disparate pattern in the Myc-correlated human HCCs. This cluster contained many kinase pathways which were deregulated in the xmrk- and kras-induced zebrafish liver cancer, but not significantly changed in the Myc-induced zebrafish liver cancer. The pathway cluster E was quite heterogeneous. The pathway cluster F was well separated from all the others and it contained pathways which were mostly down-regulated in all the human HCC subgroups associated with zebrafish signatures, including hematopoietic cell lineage, cytokine pathway, calcium signaling, and GPCR pathway. Since most of these down-regulated pathways are involved in inflammatory response, it is likely that the subgroups of human HCCs not captured by the three zebrafish models have more severe inflammatory status.\nFigure 3\nHierarchical clustering of pathways differentially expressed in the subgroups of human HCCs significantly correlated with the three zebrafish tumor signatures.\nThe color bars represent the logarithm-transformed FDR values. Up-regulated pathways are given positive values (red) and down-regulated pathways negative values (green). Pathways correlating with grey cells are not detected in the pathway analyses. Pathways with either FDR = 1 or not detected in more than five out of the 30 combinations are pre-excluded from the analysis.\n\nThe Commonly Up- and Down-regulated Genes in Zebrafish Liver Tumors are also Consistently Up- and Down-regulated in Human HCCs\nWhile the three gene signatures represent up-regulation of distinct pathways and correlate with different subgroups of human HCCs, we sought to investigate whether the 21 commonly up-regulated and 16 commonly down-regulated genes in all three transgenic models would be similarly regulated in human HCCs. Among the ten human HCC datasets we used, we were able to examine two datasets which included both HCCs and their corresponding non-tumor tissues (GSE14520 and GSE25097). However, only GSE14520 could be compared with the zebrafish data as most of the human genes homologous to the common zebrafish genes could be identified from the microarray platform used: 19 of the 21 up-regulated genes (except FOXA3 and MRPS9) and 12 of the 16 down-regulated genes (except for cyp2ad2, slco1d1, TDH and MIOX). GSE14520 dataset contains 229 primary HCCs and the corresponding paired non-tumor hepatic tissues &lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Roessler-2010-70-10202&gt;&gt;. The xmrk, kras and Myc signatures were significantly correlated with 30.5%, 20.0%, and 20.5% of the HCC samples, and in total 47.6% of the HCC samples showed significant up-regulation of at least one of the zebrafish liver cancer signatures (Table 2). Among the 19 up-regulated genes, 16 of them were up-regulated in more than half of the HCC patients, and 9 of them were up-regulated in more than 75% of the patients (Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). STMN1 was up-regulated in 96.9% of the human HCCs from the dataset, which is the most ubiquitously up-regulated. STT3A and SRP14 were up-regulated in 93.0% of the human HCCs. Among the 12 down-regulated genes, 11 of them were down-regulated in more than half of the HCC patients. FBP1 was down-regulated in 96.9% of the examined human HCCs. FBP1 is a gluconeogenesis regulatory enzyme and it functions to antagonize glycolysis in gastric cancer &lt;&lt;REF:Liu-2010-29-442&gt;&gt;. FBP1 is down-regulated in majority of human HCCs by methylation &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;. Restoration of FBP1 expression in human HCC cell lines significantly inhibited cell growth, suggesting that it might function as a tumor suppressor &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;.\nFigure 4\n\nIn silico validation of the 21 commonly up-regulated and 16 commonly down-regulated genes in human HCC dataset.\n19 of the up-regulated genes and 12 of the down-regulated genes were identified in the microarray platform. The red color indicates up-regulation, and the green color indicates down-regulation.\nFinally, the expression of these 19 up- and 16 down-regulated genes was also validated by RT-qPCR in all three different types of tumors. As shown in Figure S3, the majority of gene in the majority of tests (~90%) confirmed consistent trend of changes in the tumor samples.\n\nDiscussion\nIt is well known that human HCCs are highly heterogeneous; thus, cross-species comparative studies at the transcriptomic level should be valuable to identify conserved and critical pathways in carcinogenesis in vertebrate species. Here we first determined deregulated pathways from each of the oncogene transgenic zebrafish model. Although the three transgenic tumor models had quite distinct deregulated biological pathways, they all correlated to advanced or very advanced HCCs by comparison with gene signatures from human HCCs. Furthermore, we also found that each of the zebrafish model represent a subset of human HCCs. Since our oncogene transgenic lines have well-defined driving pathways in carcinogenesis, the information from the transgenic zebrafish should be valuable for understanding the main molecular mechanisms of each HCC subgroup, which is imperative for developing more effective therapeutics specific for each subgroup. Interestingly, our three oncogene transgenic zebrafish models significantly represent only less than half of the human HCCs and there is a need to develop more and different oncogene transgenic animal models for covering more human HCCs for further understanding of distinct molecular mechanisms in hepatocarcinogenesis, with focus on inflammatory pathways.\nIn the present study, we also identified a list of commonly deregulated genes in liver tumors induced by different oncogenic signals was identified and their expressional changes in human HCC samples were validated in silico (Table 1, Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). These genes could be served as potential therapeutic targets since they were independent of individual oncogenic pathways. The up-regulated genes includes those in protein translation and processing (eif5a2, abce1, rrp9, srp14, itm1, srprb), pro-apoptosis (rhot1a, c20orf24, fam162a) anti-apoptosis (tp53), cell cycle regulation (stmn1a, cdkrap3), purine synthesis (gart), rRNA processing (noc4l), G protein-coupled receptors signaling (reep2), n-glycan biosynthesis (mgat4b), stress response (cirbp), ubiquitination (ubap2), peroxisome (hmgcra), mitochondrial function (mrps9) and transcription (foxa3). Some of them have been implicated in hepatocarcinogenesis or identified as therapeutic targets. For example, hmgcra, a top up-regulated gene in all three tumor models, encodes the rate-limiting enzyme for cholesterol synthesis. Inhibition of Hmgcr could block tumor cell growth and metastasis &lt;&lt;REF:Cao-2011-71-2286&gt;&gt;, but clinical trials with Hmgcr inhibitor (pravastatin) have shown discrepant results &lt;&lt;REF:Lersch-2004-51-1099&gt;&gt;-&lt;&lt;REF:Kawata-2001-84-886&gt;&gt;, which may be attributed to the genetic heterogeneity of HCCs. The fact that hmgcra was highly up-regulated in all of the three transgenic zebrafish liver tumor models and these tumor models represent about half of human HCCs may suggest that it should be a therapeutic target in a broad, though not all, range of HCCs. Another top up-regulated gene, mgat4b, is one of the important enzymes in the biosynthetic pathway of N-glycans. N-glycan is up-regulated in human HCC &lt;&lt;REF:Goldman-2009-15-1808&gt;&gt; and is associated with drug resistance &lt;&lt;REF:Kudo-2007-6-32&gt;&gt;. Another gene which was highly up-regulated is stmn1a. The human homolog of stmn1a, STATHMIN1, is over-expressed and is associated with polyploidy, metastasis, early recurrence, and poor prognosis in hepatocarcinogenesis &lt;&lt;REF:Hsieh-2010-49-476&gt;&gt;-&lt;&lt;REF:Yuan-2006-209-549&gt;&gt;. It has also been identified as a major molecular target of an anticancer drug &lt;&lt;REF:Wang-2009-9-242&gt;&gt;.\nMost of the 16 genes commonly down-regulated (Table 1) are apparently involved in metabolism for normal liver function, including fatty acid metabolism (cyp2ad2, hsdl2, ech1), intracellular fatty acid and lipid transport (scp2, slc27a2, apobl), androgen metabolism (hsd17b3), glucose metabolism (fbp1b), mitochondria function (tdh, ak4), integrin complex (itgb1b.2), antioxidation (gpx4a, sod1), inositol catabolism (miox), bile acid metabolism (slco1d1) and neuron cell adhesion (nrxn1b). Several of them may have a direct connection with hepatocarcinogenesis. For example, Sod1 deficient mice showed extensive cellular oxidative damage and majority of them developed HCC &lt;&lt;REF:Elchuri-2005-24-367&gt;&gt;. Moreover, SOD1 has also been markedly down-regulated in human HCCs induced from different etiological factors &lt;&lt;REF:Li-2008-38-85&gt;&gt;. Gpx4 has also been reported to be associated with multiple types of cancers, including breast cancer &lt;&lt;REF:Mavaddat-2009-18-255&gt;&gt;, colorectal cancer &lt;&lt;REF:Meplan-2010-31-1074&gt;&gt;, and aggressive prostate cancer &lt;&lt;REF:Abe-2011-107-126&gt;&gt;; however, no study has presented its correlation with HCC. Both Sod1 and Gpx4 are important components of the cellular antioxidant mechanisms. The down-regulation of these two genes in the three transgenic zebrafish liver cancer models suggested that oxidative damage is a common and conserved part of hepatocarcinogenesis.\n\nSupporting Information\nFigure S1\n\nInduction of liver tumors in the three zebrafish transgenic liver cancer models. (A-D) Gross morphology of treated transgenic fish and the non-transgenic siblings. The treated non-transgenic siblings have normal liver size and gross morphology (A). The livers in the treated transgenic fish were obviously enlarged (B-D) compared to the treated non-transgenic siblings (A). (E-H) Histological examination of the treated transgenic fish and non-transgenic siblings. Treated xmrk fish developed HCC (F), treated Myc fish developed HCA (G), and treated kras fish developed heterogeneous HCC (H). The liver was circled out in white dotted lines. Scale bars: 25 mm for A-D, 50 microm for E-H.\n(TIF)\nClick here for additional data file.\nFigure S2\n\nCorrelation of the three transgenic zebrafish liver cancer signatures with human HCC samples. The heat maps showed the positive- and negative-correlation of the three transgenic zebrafish liver cancer signatures with 9 sets of human HCC samples, including GSE364 (A), GSE1898 (B), GSE10141 (C), GSE9843 (D), GSE19977 (E), GSE10186 (F), GSE20017 (G), GSE25097 (H) and GSE5975 (I). The color was determined by normalized enrichment score (NES) of the GSEA analysis. The red color indicates up-regulation or positive correlation, and the green color indicates down-regulation or negative correlation.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nRT-qPCR validation of commonly up- and down-regulated genes in three transgenic zebrafish liver tumors. (A) Up-regulated genes. (B) Down-regulated genes. Gene names are indicated at the top and transgenic lines are indicaed on the left. RT-qPCR data are presented with red bars in comparison with corresponding RNA-Seq data represented by blue bars. Y-axes indicate fold changes on Log2 scale. Standard error bars are included for the RT-qPCR data and asterisks indicate statistically significance (P&lt;0.05).\n(TIF)\nClick here for additional data file.\nTable S1\n\nSummary of RNA-SAGE data.\n\n(DOCX)\nClick here for additional data file.\nTable S2\n\nSignificantary up-regulated zebrafish genes with mapped human homologs.\n\n(XLSX)\nClick here for additional data file.\nTable S3\n\nSummary of human HCC datasets used in the present Study.\n\n(DOCX)\nClick here for additional data file.\nTable S4\n\nSequences of PCR primers used for RT-qPCR validation of commonly up- and down-regulated genes in zebrafish liver tumors.\n\n(DOCX)\nClick here for additional data file.\nTable S5\n\nDetails of pathways deregulated in the three transgenic zebrafish liver cancer models as classified into the seven cancer hallmarks and different aspects of the liver metabolisms.\n\n(DOCX)\nClick here for additional data file.\nTable S6\n\nDetails of pathways differentially expressed in the subgroups of human HCCs which showed significantly correlation with the three zebrafish signatures.\n\n(DOCX)\nClick here for additional data file.",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Introduction\nHuman hepatocellular carcinoma (HCC) is known to be a very heterogeneous disease, especially at intermediate and advanced stages &lt;&lt;REF:Llovet-1999-29-62&gt;&gt;. Due to the diverse and complex etiologies contributing to HCC incidence, different genetic mutations or altered molecular pathways could be responsible for hepatocarcinogenesis. So far, several carcinogenic pathways have been identified to be involved in the development and progression of HCC, including the VEGFR, EGFR, and mTOR pathways &lt;&lt;REF:Siegel-2010-52-360&gt;&gt;. In effort to decipher the role of different oncogenic pathways, a number of transgenic mouse models have been established &lt;&lt;REF:Leenders-2008-14-6915&gt;&gt;, &lt;&lt;REF:Lee-2004-36-1306&gt;&gt; and comparative transcriptomic analyses have been used to identify the best transgenic mouse models for human HCCs &lt;&lt;REF:Lee-2004-36-1306&gt;&gt;.\nThe zebrafish has been increasingly recognized as a valuable experimental model for human diseases, particularly for cancers &lt;&lt;REF:Liu-2011-6-71&gt;&gt; including liver cancers &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Liu-2012-56-2268&gt;&gt;. It has been shown that the zebrafish tumors had striking similarities with human cancer histologically &lt;&lt;REF:Amatruda-2002-1-229&gt;&gt;, &lt;&lt;REF:Spitsbergen-2000-28-705&gt;&gt;. Transcriptomic and epigenetic analyses have also indicated conserved features of carcinogen-induced zebrafish HCC with human HCC &lt;&lt;REF:Mirbahai-2011-12-3&gt;&gt;-&lt;&lt;REF:Lam-2006-24-73&gt;&gt;.\nPreviously we have generated several liver tumor models by transgenic expression of three different oncogenes (kras, xmrk or Myc) specifically in the zebrafish liver and these transgenic zebrafish usually produce liver tumors with variable degrees of severity from hepatocellular adenoma (HCA) to HCC &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;. The three oncogenes we used in the zebrafish have all been shown to be involved in hepatocarcinogenesis. KRAS is mutated in ~7% of liver cancers in human &lt;&lt;REF:Karnoub-2008-9-517&gt;&gt; but Ras signaling is ubiquitously activated in HCC &lt;&lt;REF:Calvisi-2006-130-1117&gt;&gt;. Xmrk is a naturally occurring variant of the EGFR in fish of the genus Xiphophorus (platyfish and swordtails) with constitutive autophosphorylation and activation of downstream signals &lt;&lt;REF:Gomez-2001-276-3333&gt;&gt;. Activation of EGFR signal is correlated with poor prognosis of HCC patients &lt;&lt;REF:Foster-2007-14-752&gt;&gt;. MYC is commonly amplified in many cancers including HCC and higher expression level of MYC is associated with more advanced status of HCC &lt;&lt;REF:Gan-1993-41-1185&gt;&gt;. We have shown that overexpression of kras and xmrk in the zebrafish liver could induce HCC &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;, while overexpression of Myc induced mostly HCA &lt;&lt;REF:???-?-?-?&gt;&gt;.\nComparative transcriptomic analyses of animal models and human clinical samples provide a powerful tool for identification of conserved molecular pathways and biomarker genes for diagnosis and therapy &lt;&lt;REF:Lam-2006-5-573&gt;&gt;, &lt;&lt;REF:Sweet-Cordero-2005-37-48&gt;&gt;. Our existing oncogene transgenic zebrafish models have well defined up-regulation of driver oncogene and this may provide a valuable tool to identify the molecular driving forces in human carcinogenesis by comparative transcriptomic analyses. Thus, in this study, we employed RNA sequencing technology to compare the transcriptomic profiles of the three oncogene-induced liver tumors in transgenic zebrafish. By comparative analyses with human liver transcriptomes from cirrhotic livers to very advanced HCC, we found that they all showed strong molecular correlation with advanced or very advanced human HCCs. Nevertheless, there are quite distinct deregulated biological pathways based on deregulated genes in the three oncogene transgenic models. Interestingly, each zebrafish liver tumor model correlated with a subset of human HCCs and each subset has some distinct molecular features. We showed that the transgenic zebrafish models with well-defined driver-gene activity should be valuable for classification of human HCCs and for understanding the molecular mechanisms behind each HCC subtype.\n\nMaterials and Methods\n\nTreatment and Induction of Liver Cancer in the Three Zebrafish Transgenic Models\nZebrafish were maintained following the approved protocol by Institutional Animal Care and Use Committee of National University of Singapore (Protocol 079/07). The generation of xmrk and Myc transgenic zebrafish was previously described and they were termed as TO(xmrk)\n&lt;&lt;REF:Li-2011-56-419&gt;&gt; and TO(Myc)\n&lt;&lt;REF:???-?-?-?&gt;&gt; respectively in the previous publications. The two transgenic lines were constructed by using a tetracycline-inducible transgenic system and the oncogene expression were induced by doxycycline. The krasV12 transgenic line used in the present study was newly generated by using a mifepristone-inducible system &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt; in combination with a Cre-loxP system (unpublished). For the xmrk and Myc transgenic lines, transgenic fish and their non-transgenic siblings were treated with 60 microg/ml doxycycline (Sigma, USA) starting from 3.5 mpf (month post fertilization) for 6 weeks. All xmrk fish developed HCC and all Myc fish developed HCA. In total, for each transgenic line, one tumor sample (transgenic fish treated with doxycycline) and three control samples (non-transgenic siblings similarly treated with doxycycline, transgenic siblings without doxycycline treatment, and non-transgenic siblings without doxycycline treatment) were collected for RNA sequencing. In all cases, liver samples used for RNA sequencing were pooled from four to five male fish. For the krasV12 transgenic fish, one-month-old transgenic fish were treated with 1 microM mifepristone (Sigma, USA) for 36 hours to induce Cre-mediated recombination for activation of krasV12 transgene expression in the liver. The krasV12 activated transgenic fish were then allowed to grow for six months to develop HCC and HCC samples were then collected for RNA sequencing. Three liver tumors from induced transgenic fish and three normal livers from uninduced transgenic fish were pooled separately as tumor and control samples. All samples used were from male fish and two sets of biological replicates were used.\n\nIdentification of Signature Gene Lists in Each Zebrafish Liver Cancer Model\nTotal RNA was extracted using TRIzol Reagent (Invitrogen, USA) and treated with DNase I to remove genomic DNA contamination. 3' RNA-SAGE (serial analysis of gene expression) sequencing was performed on ABI SOLiD platform by Mission Biotech (Taiwan) according to manufacturer's protocol and 10-23 million reads were generated from each sampler (Table S1). Briefly, mRNA was purified using Dynabeads Oligo(dT) EcoP (Invitrogen) and subjected to cDNA synthesis. Resultant cDNA was digested by NlaIII and EcoP15I to result in a 27 nucleotides cDNA tag between the two sequencing adapters. The tags were mapped to the NCBI RefSeq (Reference Sequence) mRNA database for zebrafish with a criterion of allowing maximum 2 nucleotide mismatches. All RNA-Seq data were submitted to Gene Expression Omnibus database with the following access numbers: GSE53342 for xmrk and Myc data and GSE53630 for kras data. Tag counts for each transcript were normalized to TPM (transcripts per million) to facilitate comparison among different samples. The differentially regulated genes in the Myc- and xmrk- induced liver cancers were identified using one sample t-test as previously described &lt;&lt;REF:Zheng-2011-6-e24019&gt;&gt;, and the differentially expressed genes in the kras-induced liver cancer was identified using two-tailed Student's t-test.\nTo facilitate functional implications of zebrafish transcriptome, all zebrafish genes were mapped to annotated human genes in order to use existing online software developed in human genes. Thus, human homology mapping of zebrafish Unigene clusters were retrieved from the Genome Institute of Singapore Zebrafish Annotation Database (http://123.136.65.67/). For Unigene clusters mapped by more than one transcript entries, the highest TPM was used to represent the expression level of the Unigene cluster. Some zebrafish Unigene clusters were mapped to more than one human Unigene clusters, which usually came from the same gene family. To remove redundancy and avoid causing bias in functional analyses, only the first human Unigene cluster in the list was selected to represent the zebrafish Unigene clusters. The lists of significantly up-regulated zebrafish genes that were mapped with human homologs and used for comparative analyese with human HCC data are shown in Table S2.\n\nGene Set Enrichment Analysis (GSEA)\nGSEA was used to establish the relatedness between the transgenic zebrafish models and human liver cancers &lt;&lt;REF:Subramanian-2005-102-15545&gt;&gt;. GSEA is a computational method that determines whether a priori defined set of genes shows statistically significant, concordant differences between two biological samples; it calculates an enrichment score using a running-sum statistic through a ranked list of gene expression data set. Human homologs of the significantly up-regulated genes from the zebrafish tumor tissues were used as cancer signatures for each transgenic zebrafish model for transcriptomic comparison with human HCC data. Each chosen phenotype of human HCCs (either one specific HCC stage or one particular HCC sample) was compared to the rest of the samples in the same dataset. All genes in the chosen phenotype were ranked by t-test to determine expression differences among different HCC stages or different HCC samples. The enrichment score of the pre-defined transgenic zebrafish cancer signature was calculated using a running-sum statistic through the ranked genes. The statistical significance of the enrichment score was estimated by using an empirical phenotype-based permutation test procedure. An FDR (false discovery rate) value was provided by introducing adjustment of multiple hypothesis testing. Human liver cancer transcriptome data were retrieved from Gene Expression Omnibus (GEO) database. The human dataset including different stages of HCCs used in the comparison was GSE6764 &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The ten human HCC datasets used for examining the representation of zebrafish liver cancer gene signatures are summarized in Table S3. Annotation information was retrieved from SOURCE (http://smd.stanford.edu/cgi-bin/source/sourceSearch). For multiple probes which can be mapped to one Unigene cluster, the maximum signal intensity was selected to represent the expression level of the Unigene cluster.\n\nGSEA Pre-ranked Analysis\nGSEA pre-ranked option was used to analyze the deregulated pathways in each transgenic zebrafish model and subgroups of human HCCs. Briefly, the entire transcriptome was ranked by logarithm transformed p-value (base 10). The up-regulated genes were given positive values, and the down-regulated genes negative values. The curated canonical pathways from the MSigDB (Molecular Signature Database) were used. The number of permutation used was 1000.\n\nRT-qPCR Validation\nTotal RNA were reverse-transcribed using the SuperScript II cDNA Synthesis Kit (Invitrogen). RT-qPCR was performed with same sets of cDNAs used for SAGE sequencing using the LightCycler 480 SYBR Green I Master system (Roche). Reactions were conducted in triplicate for each cDNA sample and primer sequences are shown in the Table S4. Gene expression levels in each control or transgenic liver sample were normalized with the level of beta-actin mRNA as the internal control. The log2 fold changes between tumor and control samples were calculated using the CT method according to the formula: log2 fold changes = -DeltaDeltaCT = -[(CT gene of interest-CT beta-actin) transgenic sample-(CT gene of interest-CT beta-actin) control sample]. Two-tailed heteroscedastic t test was performed using normalized CT values (CT gene - CT beta-actin) and changes with p&lt;0.05 are considered to be significant.\n\nResults\n\nIdentification of Differentially Expressed Genes in the Three Transgenic Zebrafish Liver Cancer Models\nThe three oncogene transgenic lines (xmrk, Myc and krasV12) were induced to develop liver tumors (Figure S1) and these tumor samples were subjected to RNA-SAGE sequencing. By a selection criteria of fold change&gt;1.5, p value&lt;0.05 and TPM&gt;10 (in either control or tumor samples), differentially expressed genes were selected from the three tumor sets. There were 2,892, 797 and 494 genes up-regulated and 169, 902 and 676 genes down-regulated in the xmrk-, kras- and Myc- induced zebrafish liver cancer, respectively (Figure 1A,B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;). Deregulated genes from the three transgenic models showed relatively small overlaps, indicating that the three oncogenes regulated quite distinct sets of genes. This is consistent with the report that morphologically uniform cancer type is frequently classified into different subgroups based on their distinct gene expression patterns &lt;&lt;REF:Lee-2005-26-1013&gt;&gt;. Interestingly, there were 21 up-regulated and 16 down-regulated genes commonly found in all the three tumor models (Figure 1A, B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table 1).\nFigure 1\nDeregulated genes and pathways in xmrk, kras and Myc transgenic liver tumors.\n(A) Venn diagram of up-regulated genes in the three zebrafish transgenic HCC models. (B) Venn diagram of down-regulated genes in the three zebrafish transgenic HCC models. (C) Deregulated pathways in the three oncogene transgenic liver tumors. Up- and down-regulated pathways in each zebrafish transgenic HCC model were analyzed by GSEA pre-ranked analysis. Red and green indicate up-and down-regulation respectively and the color code based on FDR is shown on the left. Pathways correlating with grey cells were not detected with any change. The up-regulated pathways were assigned into seven cancer hallmark categories (excluding enabling replicative immortality) according to Hanahan and Weinberg &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt; and down-regulated pathways was classified based on different aspects of the liver metabolism (see Table S1 for details).\nTable 1\nCommonly deregulated genes in the three transgenic zebrafish liver cancer models.\n\nDistinct Pathways Regulated in the Three Zebrafish Liver Tumor Models\nPathway analysis using GSEA showed that the three transgenic liver cancer models have different pathways deregulated (Figure 1C &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table S5). It has been widely accepted that there are eight cancer hallmarks for multistep tumorigenesis and the complexities of neoplasms &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt;, &lt;&lt;REF:Hanahan-2000-100-57&gt;&gt;. We found that the GSEA-identified pathways fallen into at least seven cancer hallmarks (except for enabling replicative immortality). Xmrk mainly up-regulated pathways involved in evading growth suppressors and avoiding immune destruction, which included activating cell cycle, promoting RNA transcription, up-regulating proteasome and altering immune properties. Kras provided the tumor cells with the ability of self-sustaining proliferative signals by up-regulating EGFR, Raf-MEK-ERK, PI3K-AKT-mTOR and GSK3 signaling pathways. Specifically, it also altered the focal adhesive characters of tumor cells which could activate invasion and metastasis. Myc mainly up-regulated translation and proteolysis to assist tumor cells to evade growth suppressors, and it also up-regulated VEGF pathway, thus potentially inducing angiogenesis. While there was no single pathway significantly up-regulated in all three tumor models, there were some pathways up-regulated in two models, such as proteasome in the xmrk and Myc models, and IGF1 pathway, mTOR pathway, tRNA biosynthesis and focal adhesion in the xmrk and kras models. In contrast, pathways in reprogramming of energy metabolism were generally down-regulated in all three models though the down-regulation in the Myc model was less apparent than the other two models. Meanwhile, many other pathways involved in normal liver function such as blood factors, amino acid metabolism, detoxification and xenobiotic metabolism, fatty acid metabolism, and bile synthesis were uniformly down-regulated in all three tumor models. However, one exception was hormone regulation that was apparently up-regulated in the kras tumors.\n\nCorrelation of the Three Gene Signatures of Zebrafish Liver Tumors with Different Stages of Human HCCs\nThe up-regulated genes, 2,892, 797 and 494 from xmrk, kras and Myc tumors respectively, were used as the signature genes for each model. These up-regulated genes were converted to 1,362, 490, and 146 human Unigenes respectively in order to compare with available transcriptomic data from human studies. We then compared the three signature gene sets with a set of human transcriptomic data (GSE6764) from different stages of human liver conditions: cirrhotic liver, low grade dysplastic nodules (LGDN), high grade dysplastic nodules (HGDN), and very early, early, advanced, and very advanced HCC (veHCC, eHCC, aHCC and vaHCC), in which the pathological HCC stages have been defined by tumor size, differentiation status and metastasis level &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The three signature gene sets were all up-regulated as the disease progresses, but they started to be up-regulated at different stages of tumorigenesis (Figure 2 &lt;&lt;FIG:pone-0091179-g002&gt;&gt;). The xmrk signature showed positive correlation with HCCs starting from eHCC, and it was significantly correlated with aHCC and vaHCC. The kras signature positively correlated with aHCC and vaHCC, and it was only significantly correlated with vaHCC. The Myc signature showed significantly positive correlation with HCCs starting from eHCC, similar with our previous result using an independent set of RNA-seq data from the same transgenic line &lt;&lt;REF:???-?-?-?&gt;&gt;. Interestingly, the common 21 up-regulated genes in all three tumor models also showed up-regulation even from the very early stage of HCC, indicating these genes are correlated with the entire neoplastic process.\nFigure 2\nCorrelation of the zebrafish liver tumor signatures with different stages of human hepatocarcinogenesis.\nThe up-regulated gene signatures from the three transgenic tumors (xmrk, kras and Myc) and 21 commonly up-regulated genes (Common) were used for GSEA and NESs (normalized enrichment scores (y-axis) are plotted against different stages of human HCCs (x-axis). NES is listed in the table and asterisk indicate statistical significance: FDR&lt;0.25. Abbreviations: CL, cirrhotic livers; LGDN, low grade dysplastic nodules; HGDN, high grade dysplastic nodules; veHCC, very early HCC; eHCC, early HCC; aHCC, advanced HCC; vaHCC, very advanced HCC.\n\nRepresentation of the Three Gene Signatures of Zebrafish Liver Tumors in Human HCC Samples\nGiven the distinctive pathways altered in the three zebrafish liver tumor models and the high level of heterogeneity in human HCC patients, we sought to examine the degree of representation of the three zebrafish liver cancer models in human HCC samples. We examined ten sets of published clinical HCC microarray data, each of which included at least 80 clinical samples. The ten datasets contain a total of 1,272 samples covering different ethnic groups and risk factors (Table S2). We found that the xmrk, kras and Myc gene signatures were significantly correlated with 30.8%, 24.8% and 25.6% of all clinical samples, respectively. 47.2% of the human HCC samples had significant correlation with at least one of the three zebrafish gene signatures (Table 2, Figure S2). In different human HCC datasets, the percentage ranges from 22.0% to 60.4%. Moreover, some of the clinical samples were correlated with two or even three zebrafish gene signatures. Co-correlation of two gene signatures, namely xmrk/kras, kras/Myc and kras/Myc, accounted for 17.5%, 13.5% and 12.4% of the human HCC samples. 9.3% of the human samples showed co-correlation of all the three gene signatures. Thus, it appears that the three transgenic zebrafish liver tumor models represent molecular mechanisms of hepatocarcinogenesis in almost half of the human HCC cases and the other half of human HCC may be due to different molecular mechanisms.\nTable 2\nDegree of representation of the three transgenic zebrafish liver cancer signatures in human HCCs.\n\nPercentages indicate the percentages of human HCC samples which showed significant positive correlation (FDR&lt;0.25) with the zebrafish liver cancer signatures.\n\n*Total: The total percentage of human HCCs which showed significant correlation with any one or more of the zebrafish signatures.\nWe further demonstrated that the subsets of human HCCs which showed significantly positive correlation with the same gene signature also shared similar up-regulated pathways (Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, Table S6). For this analysis, each of the human HCC sets was separated into two subgroups: those showing significant correlation with one of the zebrafish signatures, and the rest. Differentially expressed genes between the two groups were identified by two-tailed t-test, and pathway analyses were performed by GSEA pre-ranked analysis. The pathways were subjected to hierarchical clustering by logarithm-transformed FDR values using MeV &lt;&lt;REF:Saeed-2003-34-374&gt;&gt;, &lt;&lt;REF:Saeed-2006-411-134&gt;&gt;. As shown in Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, the pathways differentially regulated in human HCCs which showed significant correlation with one of the zebrafish signatures had distinct patterns. The pathway cluster A is consisted of pathways up-regulated in all the three subgroups, including proteasome, tRNA biosynthesis and oxidative phosphorylation. Ribosome, transcription and translation, and cell cycle were also highly up-regulated, suggesting that the human HCC samples significantly correlated with any of the zebrafish signatures were probably more proliferative than those not significantly correlated. The pathway cluster B contains pathways down-regulated in most of the subgroups associated with xmrk but up-regulated in the subgroups associated with kras and Myc. Interestingly, theses pathways were consistent with highly down-regulated pathways in the xmrk-induced zebrafish HCC, including energy metabolism, amino acid metabolism, fatty acid metabolism, bile acid biosynthesis, complement pathway, biosynthesis of steroid, and N-glycan biosynthesis. The pathway cluster C is more up-regulated in the kras-associated HCC subgroups, including glutamate metabolism and glycolysis. It has been reported that Kras could increase the conversion of glucose to glutamate, and this was essential for Kras-mediated tumorigenicity &lt;&lt;REF:Weinberg-2010-107-8788&gt;&gt;. The pathway cluster D was generally up-regulated in the xmrk- and kras-correlated human HCCs, but showed a disparate pattern in the Myc-correlated human HCCs. This cluster contained many kinase pathways which were deregulated in the xmrk- and kras-induced zebrafish liver cancer, but not significantly changed in the Myc-induced zebrafish liver cancer. The pathway cluster E was quite heterogeneous. The pathway cluster F was well separated from all the others and it contained pathways which were mostly down-regulated in all the human HCC subgroups associated with zebrafish signatures, including hematopoietic cell lineage, cytokine pathway, calcium signaling, and GPCR pathway. Since most of these down-regulated pathways are involved in inflammatory response, it is likely that the subgroups of human HCCs not captured by the three zebrafish models have more severe inflammatory status.\nFigure 3\nHierarchical clustering of pathways differentially expressed in the subgroups of human HCCs significantly correlated with the three zebrafish tumor signatures.\nThe color bars represent the logarithm-transformed FDR values. Up-regulated pathways are given positive values (red) and down-regulated pathways negative values (green). Pathways correlating with grey cells are not detected in the pathway analyses. Pathways with either FDR = 1 or not detected in more than five out of the 30 combinations are pre-excluded from the analysis.\n\nThe Commonly Up- and Down-regulated Genes in Zebrafish Liver Tumors are also Consistently Up- and Down-regulated in Human HCCs\nWhile the three gene signatures represent up-regulation of distinct pathways and correlate with different subgroups of human HCCs, we sought to investigate whether the 21 commonly up-regulated and 16 commonly down-regulated genes in all three transgenic models would be similarly regulated in human HCCs. Among the ten human HCC datasets we used, we were able to examine two datasets which included both HCCs and their corresponding non-tumor tissues (GSE14520 and GSE25097). However, only GSE14520 could be compared with the zebrafish data as most of the human genes homologous to the common zebrafish genes could be identified from the microarray platform used: 19 of the 21 up-regulated genes (except FOXA3 and MRPS9) and 12 of the 16 down-regulated genes (except for cyp2ad2, slco1d1, TDH and MIOX). GSE14520 dataset contains 229 primary HCCs and the corresponding paired non-tumor hepatic tissues &lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Roessler-2010-70-10202&gt;&gt;. The xmrk, kras and Myc signatures were significantly correlated with 30.5%, 20.0%, and 20.5% of the HCC samples, and in total 47.6% of the HCC samples showed significant up-regulation of at least one of the zebrafish liver cancer signatures (Table 2). Among the 19 up-regulated genes, 16 of them were up-regulated in more than half of the HCC patients, and 9 of them were up-regulated in more than 75% of the patients (Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). STMN1 was up-regulated in 96.9% of the human HCCs from the dataset, which is the most ubiquitously up-regulated. STT3A and SRP14 were up-regulated in 93.0% of the human HCCs. Among the 12 down-regulated genes, 11 of them were down-regulated in more than half of the HCC patients. FBP1 was down-regulated in 96.9% of the examined human HCCs. FBP1 is a gluconeogenesis regulatory enzyme and it functions to antagonize glycolysis in gastric cancer &lt;&lt;REF:Liu-2010-29-442&gt;&gt;. FBP1 is down-regulated in majority of human HCCs by methylation &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;. Restoration of FBP1 expression in human HCC cell lines significantly inhibited cell growth, suggesting that it might function as a tumor suppressor &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;.\nFigure 4\n\nIn silico validation of the 21 commonly up-regulated and 16 commonly down-regulated genes in human HCC dataset.\n19 of the up-regulated genes and 12 of the down-regulated genes were identified in the microarray platform. The red color indicates up-regulation, and the green color indicates down-regulation.\nFinally, the expression of these 19 up- and 16 down-regulated genes was also validated by RT-qPCR in all three different types of tumors. As shown in Figure S3, the majority of gene in the majority of tests (~90%) confirmed consistent trend of changes in the tumor samples.\n\nDiscussion\nIt is well known that human HCCs are highly heterogeneous; thus, cross-species comparative studies at the transcriptomic level should be valuable to identify conserved and critical pathways in carcinogenesis in vertebrate species. Here we first determined deregulated pathways from each of the oncogene transgenic zebrafish model. Although the three transgenic tumor models had quite distinct deregulated biological pathways, they all correlated to advanced or very advanced HCCs by comparison with gene signatures from human HCCs. Furthermore, we also found that each of the zebrafish model represent a subset of human HCCs. Since our oncogene transgenic lines have well-defined driving pathways in carcinogenesis, the information from the transgenic zebrafish should be valuable for understanding the main molecular mechanisms of each HCC subgroup, which is imperative for developing more effective therapeutics specific for each subgroup. Interestingly, our three oncogene transgenic zebrafish models significantly represent only less than half of the human HCCs and there is a need to develop more and different oncogene transgenic animal models for covering more human HCCs for further understanding of distinct molecular mechanisms in hepatocarcinogenesis, with focus on inflammatory pathways.\nIn the present study, we also identified a list of commonly deregulated genes in liver tumors induced by different oncogenic signals was identified and their expressional changes in human HCC samples were validated in silico (Table 1, Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). These genes could be served as potential therapeutic targets since they were independent of individual oncogenic pathways. The up-regulated genes includes those in protein translation and processing (eif5a2, abce1, rrp9, srp14, itm1, srprb), pro-apoptosis (rhot1a, c20orf24, fam162a) anti-apoptosis (tp53), cell cycle regulation (stmn1a, cdkrap3), purine synthesis (gart), rRNA processing (noc4l), G protein-coupled receptors signaling (reep2), n-glycan biosynthesis (mgat4b), stress response (cirbp), ubiquitination (ubap2), peroxisome (hmgcra), mitochondrial function (mrps9) and transcription (foxa3). Some of them have been implicated in hepatocarcinogenesis or identified as therapeutic targets. For example, hmgcra, a top up-regulated gene in all three tumor models, encodes the rate-limiting enzyme for cholesterol synthesis. Inhibition of Hmgcr could block tumor cell growth and metastasis &lt;&lt;REF:Cao-2011-71-2286&gt;&gt;, but clinical trials with Hmgcr inhibitor (pravastatin) have shown discrepant results &lt;&lt;REF:Lersch-2004-51-1099&gt;&gt;-&lt;&lt;REF:Kawata-2001-84-886&gt;&gt;, which may be attributed to the genetic heterogeneity of HCCs. The fact that hmgcra was highly up-regulated in all of the three transgenic zebrafish liver tumor models and these tumor models represent about half of human HCCs may suggest that it should be a therapeutic target in a broad, though not all, range of HCCs. Another top up-regulated gene, mgat4b, is one of the important enzymes in the biosynthetic pathway of N-glycans. N-glycan is up-regulated in human HCC &lt;&lt;REF:Goldman-2009-15-1808&gt;&gt; and is associated with drug resistance &lt;&lt;REF:Kudo-2007-6-32&gt;&gt;. Another gene which was highly up-regulated is stmn1a. The human homolog of stmn1a, STATHMIN1, is over-expressed and is associated with polyploidy, metastasis, early recurrence, and poor prognosis in hepatocarcinogenesis &lt;&lt;REF:Hsieh-2010-49-476&gt;&gt;-&lt;&lt;REF:Yuan-2006-209-549&gt;&gt;. It has also been identified as a major molecular target of an anticancer drug &lt;&lt;REF:Wang-2009-9-242&gt;&gt;.\nMost of the 16 genes commonly down-regulated (Table 1) are apparently involved in metabolism for normal liver function, including fatty acid metabolism (cyp2ad2, hsdl2, ech1), intracellular fatty acid and lipid transport (scp2, slc27a2, apobl), androgen metabolism (hsd17b3), glucose metabolism (fbp1b), mitochondria function (tdh, ak4), integrin complex (itgb1b.2), antioxidation (gpx4a, sod1), inositol catabolism (miox), bile acid metabolism (slco1d1) and neuron cell adhesion (nrxn1b). Several of them may have a direct connection with hepatocarcinogenesis. For example, Sod1 deficient mice showed extensive cellular oxidative damage and majority of them developed HCC &lt;&lt;REF:Elchuri-2005-24-367&gt;&gt;. Moreover, SOD1 has also been markedly down-regulated in human HCCs induced from different etiological factors &lt;&lt;REF:Li-2008-38-85&gt;&gt;. Gpx4 has also been reported to be associated with multiple types of cancers, including breast cancer &lt;&lt;REF:Mavaddat-2009-18-255&gt;&gt;, colorectal cancer &lt;&lt;REF:Meplan-2010-31-1074&gt;&gt;, and aggressive prostate cancer &lt;&lt;REF:Abe-2011-107-126&gt;&gt;; however, no study has presented its correlation with HCC. Both Sod1 and Gpx4 are important components of the cellular antioxidant mechanisms. The down-regulation of these two genes in the three transgenic zebrafish liver cancer models suggested that oxidative damage is a common and conserved part of hepatocarcinogenesis.\n\nSupporting Information\nFigure S1\n\nInduction of liver tumors in the three zebrafish transgenic liver cancer models. (A-D) Gross morphology of treated transgenic fish and the non-transgenic siblings. The treated non-transgenic siblings have normal liver size and gross morphology (A). The livers in the treated transgenic fish were obviously enlarged (B-D) compared to the treated non-transgenic siblings (A). (E-H) Histological examination of the treated transgenic fish and non-transgenic siblings. Treated xmrk fish developed HCC (F), treated Myc fish developed HCA (G), and treated kras fish developed heterogeneous HCC (H). The liver was circled out in white dotted lines. Scale bars: 25 mm for A-D, 50 microm for E-H.\n(TIF)\nClick here for additional data file.\nFigure S2\n\nCorrelation of the three transgenic zebrafish liver cancer signatures with human HCC samples. The heat maps showed the positive- and negative-correlation of the three transgenic zebrafish liver cancer signatures with 9 sets of human HCC samples, including GSE364 (A), GSE1898 (B), GSE10141 (C), GSE9843 (D), GSE19977 (E), GSE10186 (F), GSE20017 (G), GSE25097 (H) and GSE5975 (I). The color was determined by normalized enrichment score (NES) of the GSEA analysis. The red color indicates up-regulation or positive correlation, and the green color indicates down-regulation or negative correlation.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nRT-qPCR validation of commonly up- and down-regulated genes in three transgenic zebrafish liver tumors. (A) Up-regulated genes. (B) Down-regulated genes. Gene names are indicated at the top and transgenic lines are indicaed on the left. RT-qPCR data are presented with red bars in comparison with corresponding RNA-Seq data represented by blue bars. Y-axes indicate fold changes on Log2 scale. Standard error bars are included for the RT-qPCR data and asterisks indicate statistically significance (P&lt;0.05).\n(TIF)\nClick here for additional data file.\nTable S1\n\nSummary of RNA-SAGE data.\n\n(DOCX)\nClick here for additional data file.\nTable S2\n\nSignificantary up-regulated zebrafish genes with mapped human homologs.\n\n(XLSX)\nClick here for additional data file.\nTable S3\n\nSummary of human HCC datasets used in the present Study.\n\n(DOCX)\nClick here for additional data file.\nTable S4\n\nSequences of PCR primers used for RT-qPCR validation of commonly up- and down-regulated genes in zebrafish liver tumors.\n\n(DOCX)\nClick here for additional data file.\nTable S5\n\nDetails of pathways deregulated in the three transgenic zebrafish liver cancer models as classified into the seven cancer hallmarks and different aspects of the liver metabolisms.\n\n(DOCX)\nClick here for additional data file.\nTable S6\n\nDetails of pathways differentially expressed in the subgroups of human HCCs which showed significantly correlation with the three zebrafish signatures.\n\n(DOCX)\nClick here for additional data file.",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nHuman hepatocellular carcinoma (HCC) is known to be a very heterogeneous disease, especially at intermediate and advanced stages &lt;&lt;REF:Llovet-1999-29-62&gt;&gt;. Due to the diverse and complex etiologies contributing to HCC incidence, different genetic mutations or altered molecular pathways could be responsible for hepatocarcinogenesis. So far, several carcinogenic pathways have been identified to be involved in the development and progression of HCC, including the VEGFR, EGFR, and mTOR pathways &lt;&lt;REF:Siegel-2010-52-360&gt;&gt;. In effort to decipher the role of different oncogenic pathways, a number of transgenic mouse models have been established &lt;&lt;REF:Leenders-2008-14-6915&gt;&gt;, &lt;&lt;REF:Lee-2004-36-1306&gt;&gt; and comparative transcriptomic analyses have been used to identify the best transgenic mouse models for human HCCs &lt;&lt;REF:Lee-2004-36-1306&gt;&gt;.\nThe zebrafish has been increasingly recognized as a valuable experimental model for human diseases, particularly for cancers &lt;&lt;REF:Liu-2011-6-71&gt;&gt; including liver cancers &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Liu-2012-56-2268&gt;&gt;. It has been shown that the zebrafish tumors had striking similarities with human cancer histologically &lt;&lt;REF:Amatruda-2002-1-229&gt;&gt;, &lt;&lt;REF:Spitsbergen-2000-28-705&gt;&gt;. Transcriptomic and epigenetic analyses have also indicated conserved features of carcinogen-induced zebrafish HCC with human HCC &lt;&lt;REF:Mirbahai-2011-12-3&gt;&gt;-&lt;&lt;REF:Lam-2006-24-73&gt;&gt;.\nPreviously we have generated several liver tumor models by transgenic expression of three different oncogenes (kras, xmrk or Myc) specifically in the zebrafish liver and these transgenic zebrafish usually produce liver tumors with variable degrees of severity from hepatocellular adenoma (HCA) to HCC &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;. The three oncogenes we used in the zebrafish have all been shown to be involved in hepatocarcinogenesis. KRAS is mutated in ~7% of liver cancers in human &lt;&lt;REF:Karnoub-2008-9-517&gt;&gt; but Ras signaling is ubiquitously activated in HCC &lt;&lt;REF:Calvisi-2006-130-1117&gt;&gt;. Xmrk is a naturally occurring variant of the EGFR in fish of the genus Xiphophorus (platyfish and swordtails) with constitutive autophosphorylation and activation of downstream signals &lt;&lt;REF:Gomez-2001-276-3333&gt;&gt;. Activation of EGFR signal is correlated with poor prognosis of HCC patients &lt;&lt;REF:Foster-2007-14-752&gt;&gt;. MYC is commonly amplified in many cancers including HCC and higher expression level of MYC is associated with more advanced status of HCC &lt;&lt;REF:Gan-1993-41-1185&gt;&gt;. We have shown that overexpression of kras and xmrk in the zebrafish liver could induce HCC &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;, while overexpression of Myc induced mostly HCA &lt;&lt;REF:???-?-?-?&gt;&gt;.\nComparative transcriptomic analyses of animal models and human clinical samples provide a powerful tool for identification of conserved molecular pathways and biomarker genes for diagnosis and therapy &lt;&lt;REF:Lam-2006-5-573&gt;&gt;, &lt;&lt;REF:Sweet-Cordero-2005-37-48&gt;&gt;. Our existing oncogene transgenic zebrafish models have well defined up-regulation of driver oncogene and this may provide a valuable tool to identify the molecular driving forces in human carcinogenesis by comparative transcriptomic analyses. Thus, in this study, we employed RNA sequencing technology to compare the transcriptomic profiles of the three oncogene-induced liver tumors in transgenic zebrafish. By comparative analyses with human liver transcriptomes from cirrhotic livers to very advanced HCC, we found that they all showed strong molecular correlation with advanced or very advanced human HCCs. Nevertheless, there are quite distinct deregulated biological pathways based on deregulated genes in the three oncogene transgenic models. Interestingly, each zebrafish liver tumor model correlated with a subset of human HCCs and each subset has some distinct molecular features. We showed that the transgenic zebrafish models with well-defined driver-gene activity should be valuable for classification of human HCCs and for understanding the molecular mechanisms behind each HCC subtype.\n\nMaterials and Methods\n\nTreatment and Induction of Liver Cancer in the Three Zebrafish Transgenic Models\nZebrafish were maintained following the approved protocol by Institutional Animal Care and Use Committee of National University of Singapore (Protocol 079/07). The generation of xmrk and Myc transgenic zebrafish was previously described and they were termed as TO(xmrk)\n&lt;&lt;REF:Li-2011-56-419&gt;&gt; and TO(Myc)\n&lt;&lt;REF:???-?-?-?&gt;&gt; respectively in the previous publications. The two transgenic lines were constructed by using a tetracycline-inducible transgenic system and the oncogene expression were induced by doxycycline. The krasV12 transgenic line used in the present study was newly generated by using a mifepristone-inducible system &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt; in combination with a Cre-loxP system (unpublished). For the xmrk and Myc transgenic lines, transgenic fish and their non-transgenic siblings were treated with 60 microg/ml doxycycline (Sigma, USA) starting from 3.5 mpf (month post fertilization) for 6 weeks. All xmrk fish developed HCC and all Myc fish developed HCA. In total, for each transgenic line, one tumor sample (transgenic fish treated with doxycycline) and three control samples (non-transgenic siblings similarly treated with doxycycline, transgenic siblings without doxycycline treatment, and non-transgenic siblings without doxycycline treatment) were collected for RNA sequencing. In all cases, liver samples used for RNA sequencing were pooled from four to five male fish. For the krasV12 transgenic fish, one-month-old transgenic fish were treated with 1 microM mifepristone (Sigma, USA) for 36 hours to induce Cre-mediated recombination for activation of krasV12 transgene expression in the liver. The krasV12 activated transgenic fish were then allowed to grow for six months to develop HCC and HCC samples were then collected for RNA sequencing. Three liver tumors from induced transgenic fish and three normal livers from uninduced transgenic fish were pooled separately as tumor and control samples. All samples used were from male fish and two sets of biological replicates were used.\n\nIdentification of Signature Gene Lists in Each Zebrafish Liver Cancer Model\nTotal RNA was extracted using TRIzol Reagent (Invitrogen, USA) and treated with DNase I to remove genomic DNA contamination. 3' RNA-SAGE (serial analysis of gene expression) sequencing was performed on ABI SOLiD platform by Mission Biotech (Taiwan) according to manufacturer's protocol and 10-23 million reads were generated from each sampler (Table S1). Briefly, mRNA was purified using Dynabeads Oligo(dT) EcoP (Invitrogen) and subjected to cDNA synthesis. Resultant cDNA was digested by NlaIII and EcoP15I to result in a 27 nucleotides cDNA tag between the two sequencing adapters. The tags were mapped to the NCBI RefSeq (Reference Sequence) mRNA database for zebrafish with a criterion of allowing maximum 2 nucleotide mismatches. All RNA-Seq data were submitted to Gene Expression Omnibus database with the following access numbers: GSE53342 for xmrk and Myc data and GSE53630 for kras data. Tag counts for each transcript were normalized to TPM (transcripts per million) to facilitate comparison among different samples. The differentially regulated genes in the Myc- and xmrk- induced liver cancers were identified using one sample t-test as previously described &lt;&lt;REF:Zheng-2011-6-e24019&gt;&gt;, and the differentially expressed genes in the kras-induced liver cancer was identified using two-tailed Student's t-test.\nTo facilitate functional implications of zebrafish transcriptome, all zebrafish genes were mapped to annotated human genes in order to use existing online software developed in human genes. Thus, human homology mapping of zebrafish Unigene clusters were retrieved from the Genome Institute of Singapore Zebrafish Annotation Database (http://123.136.65.67/). For Unigene clusters mapped by more than one transcript entries, the highest TPM was used to represent the expression level of the Unigene cluster. Some zebrafish Unigene clusters were mapped to more than one human Unigene clusters, which usually came from the same gene family. To remove redundancy and avoid causing bias in functional analyses, only the first human Unigene cluster in the list was selected to represent the zebrafish Unigene clusters. The lists of significantly up-regulated zebrafish genes that were mapped with human homologs and used for comparative analyese with human HCC data are shown in Table S2.\n\nGene Set Enrichment Analysis (GSEA)\nGSEA was used to establish the relatedness between the transgenic zebrafish models and human liver cancers &lt;&lt;REF:Subramanian-2005-102-15545&gt;&gt;. GSEA is a computational method that determines whether a priori defined set of genes shows statistically significant, concordant differences between two biological samples; it calculates an enrichment score using a running-sum statistic through a ranked list of gene expression data set. Human homologs of the significantly up-regulated genes from the zebrafish tumor tissues were used as cancer signatures for each transgenic zebrafish model for transcriptomic comparison with human HCC data. Each chosen phenotype of human HCCs (either one specific HCC stage or one particular HCC sample) was compared to the rest of the samples in the same dataset. All genes in the chosen phenotype were ranked by t-test to determine expression differences among different HCC stages or different HCC samples. The enrichment score of the pre-defined transgenic zebrafish cancer signature was calculated using a running-sum statistic through the ranked genes. The statistical significance of the enrichment score was estimated by using an empirical phenotype-based permutation test procedure. An FDR (false discovery rate) value was provided by introducing adjustment of multiple hypothesis testing. Human liver cancer transcriptome data were retrieved from Gene Expression Omnibus (GEO) database. The human dataset including different stages of HCCs used in the comparison was GSE6764 &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The ten human HCC datasets used for examining the representation of zebrafish liver cancer gene signatures are summarized in Table S3. Annotation information was retrieved from SOURCE (http://smd.stanford.edu/cgi-bin/source/sourceSearch). For multiple probes which can be mapped to one Unigene cluster, the maximum signal intensity was selected to represent the expression level of the Unigene cluster.\n\nGSEA Pre-ranked Analysis\nGSEA pre-ranked option was used to analyze the deregulated pathways in each transgenic zebrafish model and subgroups of human HCCs. Briefly, the entire transcriptome was ranked by logarithm transformed p-value (base 10). The up-regulated genes were given positive values, and the down-regulated genes negative values. The curated canonical pathways from the MSigDB (Molecular Signature Database) were used. The number of permutation used was 1000.\n\nRT-qPCR Validation\nTotal RNA were reverse-transcribed using the SuperScript II cDNA Synthesis Kit (Invitrogen). RT-qPCR was performed with same sets of cDNAs used for SAGE sequencing using the LightCycler 480 SYBR Green I Master system (Roche). Reactions were conducted in triplicate for each cDNA sample and primer sequences are shown in the Table S4. Gene expression levels in each control or transgenic liver sample were normalized with the level of beta-actin mRNA as the internal control. The log2 fold changes between tumor and control samples were calculated using the CT method according to the formula: log2 fold changes = -DeltaDeltaCT = -[(CT gene of interest-CT beta-actin) transgenic sample-(CT gene of interest-CT beta-actin) control sample]. Two-tailed heteroscedastic t test was performed using normalized CT values (CT gene - CT beta-actin) and changes with p&lt;0.05 are considered to be significant.\n\nResults\n\nIdentification of Differentially Expressed Genes in the Three Transgenic Zebrafish Liver Cancer Models\nThe three oncogene transgenic lines (xmrk, Myc and krasV12) were induced to develop liver tumors (Figure S1) and these tumor samples were subjected to RNA-SAGE sequencing. By a selection criteria of fold change&gt;1.5, p value&lt;0.05 and TPM&gt;10 (in either control or tumor samples), differentially expressed genes were selected from the three tumor sets. There were 2,892, 797 and 494 genes up-regulated and 169, 902 and 676 genes down-regulated in the xmrk-, kras- and Myc- induced zebrafish liver cancer, respectively (Figure 1A,B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;). Deregulated genes from the three transgenic models showed relatively small overlaps, indicating that the three oncogenes regulated quite distinct sets of genes. This is consistent with the report that morphologically uniform cancer type is frequently classified into different subgroups based on their distinct gene expression patterns &lt;&lt;REF:Lee-2005-26-1013&gt;&gt;. Interestingly, there were 21 up-regulated and 16 down-regulated genes commonly found in all the three tumor models (Figure 1A, B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table 1).\nFigure 1\nDeregulated genes and pathways in xmrk, kras and Myc transgenic liver tumors.\n(A) Venn diagram of up-regulated genes in the three zebrafish transgenic HCC models. (B) Venn diagram of down-regulated genes in the three zebrafish transgenic HCC models. (C) Deregulated pathways in the three oncogene transgenic liver tumors. Up- and down-regulated pathways in each zebrafish transgenic HCC model were analyzed by GSEA pre-ranked analysis. Red and green indicate up-and down-regulation respectively and the color code based on FDR is shown on the left. Pathways correlating with grey cells were not detected with any change. The up-regulated pathways were assigned into seven cancer hallmark categories (excluding enabling replicative immortality) according to Hanahan and Weinberg &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt; and down-regulated pathways was classified based on different aspects of the liver metabolism (see Table S1 for details).\nTable 1\nCommonly deregulated genes in the three transgenic zebrafish liver cancer models.\n\nDistinct Pathways Regulated in the Three Zebrafish Liver Tumor Models\nPathway analysis using GSEA showed that the three transgenic liver cancer models have different pathways deregulated (Figure 1C &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table S5). It has been widely accepted that there are eight cancer hallmarks for multistep tumorigenesis and the complexities of neoplasms &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt;, &lt;&lt;REF:Hanahan-2000-100-57&gt;&gt;. We found that the GSEA-identified pathways fallen into at least seven cancer hallmarks (except for enabling replicative immortality). Xmrk mainly up-regulated pathways involved in evading growth suppressors and avoiding immune destruction, which included activating cell cycle, promoting RNA transcription, up-regulating proteasome and altering immune properties. Kras provided the tumor cells with the ability of self-sustaining proliferative signals by up-regulating EGFR, Raf-MEK-ERK, PI3K-AKT-mTOR and GSK3 signaling pathways. Specifically, it also altered the focal adhesive characters of tumor cells which could activate invasion and metastasis. Myc mainly up-regulated translation and proteolysis to assist tumor cells to evade growth suppressors, and it also up-regulated VEGF pathway, thus potentially inducing angiogenesis. While there was no single pathway significantly up-regulated in all three tumor models, there were some pathways up-regulated in two models, such as proteasome in the xmrk and Myc models, and IGF1 pathway, mTOR pathway, tRNA biosynthesis and focal adhesion in the xmrk and kras models. In contrast, pathways in reprogramming of energy metabolism were generally down-regulated in all three models though the down-regulation in the Myc model was less apparent than the other two models. Meanwhile, many other pathways involved in normal liver function such as blood factors, amino acid metabolism, detoxification and xenobiotic metabolism, fatty acid metabolism, and bile synthesis were uniformly down-regulated in all three tumor models. However, one exception was hormone regulation that was apparently up-regulated in the kras tumors.\n\nCorrelation of the Three Gene Signatures of Zebrafish Liver Tumors with Different Stages of Human HCCs\nThe up-regulated genes, 2,892, 797 and 494 from xmrk, kras and Myc tumors respectively, were used as the signature genes for each model. These up-regulated genes were converted to 1,362, 490, and 146 human Unigenes respectively in order to compare with available transcriptomic data from human studies. We then compared the three signature gene sets with a set of human transcriptomic data (GSE6764) from different stages of human liver conditions: cirrhotic liver, low grade dysplastic nodules (LGDN), high grade dysplastic nodules (HGDN), and very early, early, advanced, and very advanced HCC (veHCC, eHCC, aHCC and vaHCC), in which the pathological HCC stages have been defined by tumor size, differentiation status and metastasis level &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The three signature gene sets were all up-regulated as the disease progresses, but they started to be up-regulated at different stages of tumorigenesis (Figure 2 &lt;&lt;FIG:pone-0091179-g002&gt;&gt;). The xmrk signature showed positive correlation with HCCs starting from eHCC, and it was significantly correlated with aHCC and vaHCC. The kras signature positively correlated with aHCC and vaHCC, and it was only significantly correlated with vaHCC. The Myc signature showed significantly positive correlation with HCCs starting from eHCC, similar with our previous result using an independent set of RNA-seq data from the same transgenic line &lt;&lt;REF:???-?-?-?&gt;&gt;. Interestingly, the common 21 up-regulated genes in all three tumor models also showed up-regulation even from the very early stage of HCC, indicating these genes are correlated with the entire neoplastic process.\nFigure 2\nCorrelation of the zebrafish liver tumor signatures with different stages of human hepatocarcinogenesis.\nThe up-regulated gene signatures from the three transgenic tumors (xmrk, kras and Myc) and 21 commonly up-regulated genes (Common) were used for GSEA and NESs (normalized enrichment scores (y-axis) are plotted against different stages of human HCCs (x-axis). NES is listed in the table and asterisk indicate statistical significance: FDR&lt;0.25. Abbreviations: CL, cirrhotic livers; LGDN, low grade dysplastic nodules; HGDN, high grade dysplastic nodules; veHCC, very early HCC; eHCC, early HCC; aHCC, advanced HCC; vaHCC, very advanced HCC.\n\nRepresentation of the Three Gene Signatures of Zebrafish Liver Tumors in Human HCC Samples\nGiven the distinctive pathways altered in the three zebrafish liver tumor models and the high level of heterogeneity in human HCC patients, we sought to examine the degree of representation of the three zebrafish liver cancer models in human HCC samples. We examined ten sets of published clinical HCC microarray data, each of which included at least 80 clinical samples. The ten datasets contain a total of 1,272 samples covering different ethnic groups and risk factors (Table S2). We found that the xmrk, kras and Myc gene signatures were significantly correlated with 30.8%, 24.8% and 25.6% of all clinical samples, respectively. 47.2% of the human HCC samples had significant correlation with at least one of the three zebrafish gene signatures (Table 2, Figure S2). In different human HCC datasets, the percentage ranges from 22.0% to 60.4%. Moreover, some of the clinical samples were correlated with two or even three zebrafish gene signatures. Co-correlation of two gene signatures, namely xmrk/kras, kras/Myc and kras/Myc, accounted for 17.5%, 13.5% and 12.4% of the human HCC samples. 9.3% of the human samples showed co-correlation of all the three gene signatures. Thus, it appears that the three transgenic zebrafish liver tumor models represent molecular mechanisms of hepatocarcinogenesis in almost half of the human HCC cases and the other half of human HCC may be due to different molecular mechanisms.\nTable 2\nDegree of representation of the three transgenic zebrafish liver cancer signatures in human HCCs.\n\nPercentages indicate the percentages of human HCC samples which showed significant positive correlation (FDR&lt;0.25) with the zebrafish liver cancer signatures.\n\n*Total: The total percentage of human HCCs which showed significant correlation with any one or more of the zebrafish signatures.\nWe further demonstrated that the subsets of human HCCs which showed significantly positive correlation with the same gene signature also shared similar up-regulated pathways (Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, Table S6). For this analysis, each of the human HCC sets was separated into two subgroups: those showing significant correlation with one of the zebrafish signatures, and the rest. Differentially expressed genes between the two groups were identified by two-tailed t-test, and pathway analyses were performed by GSEA pre-ranked analysis. The pathways were subjected to hierarchical clustering by logarithm-transformed FDR values using MeV &lt;&lt;REF:Saeed-2003-34-374&gt;&gt;, &lt;&lt;REF:Saeed-2006-411-134&gt;&gt;. As shown in Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, the pathways differentially regulated in human HCCs which showed significant correlation with one of the zebrafish signatures had distinct patterns. The pathway cluster A is consisted of pathways up-regulated in all the three subgroups, including proteasome, tRNA biosynthesis and oxidative phosphorylation. Ribosome, transcription and translation, and cell cycle were also highly up-regulated, suggesting that the human HCC samples significantly correlated with any of the zebrafish signatures were probably more proliferative than those not significantly correlated. The pathway cluster B contains pathways down-regulated in most of the subgroups associated with xmrk but up-regulated in the subgroups associated with kras and Myc. Interestingly, theses pathways were consistent with highly down-regulated pathways in the xmrk-induced zebrafish HCC, including energy metabolism, amino acid metabolism, fatty acid metabolism, bile acid biosynthesis, complement pathway, biosynthesis of steroid, and N-glycan biosynthesis. The pathway cluster C is more up-regulated in the kras-associated HCC subgroups, including glutamate metabolism and glycolysis. It has been reported that Kras could increase the conversion of glucose to glutamate, and this was essential for Kras-mediated tumorigenicity &lt;&lt;REF:Weinberg-2010-107-8788&gt;&gt;. The pathway cluster D was generally up-regulated in the xmrk- and kras-correlated human HCCs, but showed a disparate pattern in the Myc-correlated human HCCs. This cluster contained many kinase pathways which were deregulated in the xmrk- and kras-induced zebrafish liver cancer, but not significantly changed in the Myc-induced zebrafish liver cancer. The pathway cluster E was quite heterogeneous. The pathway cluster F was well separated from all the others and it contained pathways which were mostly down-regulated in all the human HCC subgroups associated with zebrafish signatures, including hematopoietic cell lineage, cytokine pathway, calcium signaling, and GPCR pathway. Since most of these down-regulated pathways are involved in inflammatory response, it is likely that the subgroups of human HCCs not captured by the three zebrafish models have more severe inflammatory status.\nFigure 3\nHierarchical clustering of pathways differentially expressed in the subgroups of human HCCs significantly correlated with the three zebrafish tumor signatures.\nThe color bars represent the logarithm-transformed FDR values. Up-regulated pathways are given positive values (red) and down-regulated pathways negative values (green). Pathways correlating with grey cells are not detected in the pathway analyses. Pathways with either FDR = 1 or not detected in more than five out of the 30 combinations are pre-excluded from the analysis.\n\nThe Commonly Up- and Down-regulated Genes in Zebrafish Liver Tumors are also Consistently Up- and Down-regulated in Human HCCs\nWhile the three gene signatures represent up-regulation of distinct pathways and correlate with different subgroups of human HCCs, we sought to investigate whether the 21 commonly up-regulated and 16 commonly down-regulated genes in all three transgenic models would be similarly regulated in human HCCs. Among the ten human HCC datasets we used, we were able to examine two datasets which included both HCCs and their corresponding non-tumor tissues (GSE14520 and GSE25097). However, only GSE14520 could be compared with the zebrafish data as most of the human genes homologous to the common zebrafish genes could be identified from the microarray platform used: 19 of the 21 up-regulated genes (except FOXA3 and MRPS9) and 12 of the 16 down-regulated genes (except for cyp2ad2, slco1d1, TDH and MIOX). GSE14520 dataset contains 229 primary HCCs and the corresponding paired non-tumor hepatic tissues &lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Roessler-2010-70-10202&gt;&gt;. The xmrk, kras and Myc signatures were significantly correlated with 30.5%, 20.0%, and 20.5% of the HCC samples, and in total 47.6% of the HCC samples showed significant up-regulation of at least one of the zebrafish liver cancer signatures (Table 2). Among the 19 up-regulated genes, 16 of them were up-regulated in more than half of the HCC patients, and 9 of them were up-regulated in more than 75% of the patients (Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). STMN1 was up-regulated in 96.9% of the human HCCs from the dataset, which is the most ubiquitously up-regulated. STT3A and SRP14 were up-regulated in 93.0% of the human HCCs. Among the 12 down-regulated genes, 11 of them were down-regulated in more than half of the HCC patients. FBP1 was down-regulated in 96.9% of the examined human HCCs. FBP1 is a gluconeogenesis regulatory enzyme and it functions to antagonize glycolysis in gastric cancer &lt;&lt;REF:Liu-2010-29-442&gt;&gt;. FBP1 is down-regulated in majority of human HCCs by methylation &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;. Restoration of FBP1 expression in human HCC cell lines significantly inhibited cell growth, suggesting that it might function as a tumor suppressor &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;.\nFigure 4\n\nIn silico validation of the 21 commonly up-regulated and 16 commonly down-regulated genes in human HCC dataset.\n19 of the up-regulated genes and 12 of the down-regulated genes were identified in the microarray platform. The red color indicates up-regulation, and the green color indicates down-regulation.\nFinally, the expression of these 19 up- and 16 down-regulated genes was also validated by RT-qPCR in all three different types of tumors. As shown in Figure S3, the majority of gene in the majority of tests (~90%) confirmed consistent trend of changes in the tumor samples.\n\nDiscussion\nIt is well known that human HCCs are highly heterogeneous; thus, cross-species comparative studies at the transcriptomic level should be valuable to identify conserved and critical pathways in carcinogenesis in vertebrate species. Here we first determined deregulated pathways from each of the oncogene transgenic zebrafish model. Although the three transgenic tumor models had quite distinct deregulated biological pathways, they all correlated to advanced or very advanced HCCs by comparison with gene signatures from human HCCs. Furthermore, we also found that each of the zebrafish model represent a subset of human HCCs. Since our oncogene transgenic lines have well-defined driving pathways in carcinogenesis, the information from the transgenic zebrafish should be valuable for understanding the main molecular mechanisms of each HCC subgroup, which is imperative for developing more effective therapeutics specific for each subgroup. Interestingly, our three oncogene transgenic zebrafish models significantly represent only less than half of the human HCCs and there is a need to develop more and different oncogene transgenic animal models for covering more human HCCs for further understanding of distinct molecular mechanisms in hepatocarcinogenesis, with focus on inflammatory pathways.\nIn the present study, we also identified a list of commonly deregulated genes in liver tumors induced by different oncogenic signals was identified and their expressional changes in human HCC samples were validated in silico (Table 1, Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). These genes could be served as potential therapeutic targets since they were independent of individual oncogenic pathways. The up-regulated genes includes those in protein translation and processing (eif5a2, abce1, rrp9, srp14, itm1, srprb), pro-apoptosis (rhot1a, c20orf24, fam162a) anti-apoptosis (tp53), cell cycle regulation (stmn1a, cdkrap3), purine synthesis (gart), rRNA processing (noc4l), G protein-coupled receptors signaling (reep2), n-glycan biosynthesis (mgat4b), stress response (cirbp), ubiquitination (ubap2), peroxisome (hmgcra), mitochondrial function (mrps9) and transcription (foxa3). Some of them have been implicated in hepatocarcinogenesis or identified as therapeutic targets. For example, hmgcra, a top up-regulated gene in all three tumor models, encodes the rate-limiting enzyme for cholesterol synthesis. Inhibition of Hmgcr could block tumor cell growth and metastasis &lt;&lt;REF:Cao-2011-71-2286&gt;&gt;, but clinical trials with Hmgcr inhibitor (pravastatin) have shown discrepant results &lt;&lt;REF:Lersch-2004-51-1099&gt;&gt;-&lt;&lt;REF:Kawata-2001-84-886&gt;&gt;, which may be attributed to the genetic heterogeneity of HCCs. The fact that hmgcra was highly up-regulated in all of the three transgenic zebrafish liver tumor models and these tumor models represent about half of human HCCs may suggest that it should be a therapeutic target in a broad, though not all, range of HCCs. Another top up-regulated gene, mgat4b, is one of the important enzymes in the biosynthetic pathway of N-glycans. N-glycan is up-regulated in human HCC &lt;&lt;REF:Goldman-2009-15-1808&gt;&gt; and is associated with drug resistance &lt;&lt;REF:Kudo-2007-6-32&gt;&gt;. Another gene which was highly up-regulated is stmn1a. The human homolog of stmn1a, STATHMIN1, is over-expressed and is associated with polyploidy, metastasis, early recurrence, and poor prognosis in hepatocarcinogenesis &lt;&lt;REF:Hsieh-2010-49-476&gt;&gt;-&lt;&lt;REF:Yuan-2006-209-549&gt;&gt;. It has also been identified as a major molecular target of an anticancer drug &lt;&lt;REF:Wang-2009-9-242&gt;&gt;.\nMost of the 16 genes commonly down-regulated (Table 1) are apparently involved in metabolism for normal liver function, including fatty acid metabolism (cyp2ad2, hsdl2, ech1), intracellular fatty acid and lipid transport (scp2, slc27a2, apobl), androgen metabolism (hsd17b3), glucose metabolism (fbp1b), mitochondria function (tdh, ak4), integrin complex (itgb1b.2), antioxidation (gpx4a, sod1), inositol catabolism (miox), bile acid metabolism (slco1d1) and neuron cell adhesion (nrxn1b). Several of them may have a direct connection with hepatocarcinogenesis. For example, Sod1 deficient mice showed extensive cellular oxidative damage and majority of them developed HCC &lt;&lt;REF:Elchuri-2005-24-367&gt;&gt;. Moreover, SOD1 has also been markedly down-regulated in human HCCs induced from different etiological factors &lt;&lt;REF:Li-2008-38-85&gt;&gt;. Gpx4 has also been reported to be associated with multiple types of cancers, including breast cancer &lt;&lt;REF:Mavaddat-2009-18-255&gt;&gt;, colorectal cancer &lt;&lt;REF:Meplan-2010-31-1074&gt;&gt;, and aggressive prostate cancer &lt;&lt;REF:Abe-2011-107-126&gt;&gt;; however, no study has presented its correlation with HCC. Both Sod1 and Gpx4 are important components of the cellular antioxidant mechanisms. The down-regulation of these two genes in the three transgenic zebrafish liver cancer models suggested that oxidative damage is a common and conserved part of hepatocarcinogenesis.\n\nSupporting Information\nFigure S1\n\nInduction of liver tumors in the three zebrafish transgenic liver cancer models. (A-D) Gross morphology of treated transgenic fish and the non-transgenic siblings. The treated non-transgenic siblings have normal liver size and gross morphology (A). The livers in the treated transgenic fish were obviously enlarged (B-D) compared to the treated non-transgenic siblings (A). (E-H) Histological examination of the treated transgenic fish and non-transgenic siblings. Treated xmrk fish developed HCC (F), treated Myc fish developed HCA (G), and treated kras fish developed heterogeneous HCC (H). The liver was circled out in white dotted lines. Scale bars: 25 mm for A-D, 50 microm for E-H.\n(TIF)\nClick here for additional data file.\nFigure S2\n\nCorrelation of the three transgenic zebrafish liver cancer signatures with human HCC samples. The heat maps showed the positive- and negative-correlation of the three transgenic zebrafish liver cancer signatures with 9 sets of human HCC samples, including GSE364 (A), GSE1898 (B), GSE10141 (C), GSE9843 (D), GSE19977 (E), GSE10186 (F), GSE20017 (G), GSE25097 (H) and GSE5975 (I). The color was determined by normalized enrichment score (NES) of the GSEA analysis. The red color indicates up-regulation or positive correlation, and the green color indicates down-regulation or negative correlation.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nRT-qPCR validation of commonly up- and down-regulated genes in three transgenic zebrafish liver tumors. (A) Up-regulated genes. (B) Down-regulated genes. Gene names are indicated at the top and transgenic lines are indicaed on the left. RT-qPCR data are presented with red bars in comparison with corresponding RNA-Seq data represented by blue bars. Y-axes indicate fold changes on Log2 scale. Standard error bars are included for the RT-qPCR data and asterisks indicate statistically significance (P&lt;0.05).\n(TIF)\nClick here for additional data file.\nTable S1\n\nSummary of RNA-SAGE data.\n\n(DOCX)\nClick here for additional data file.\nTable S2\n\nSignificantary up-regulated zebrafish genes with mapped human homologs.\n\n(XLSX)\nClick here for additional data file.\nTable S3\n\nSummary of human HCC datasets used in the present Study.\n\n(DOCX)\nClick here for additional data file.\nTable S4\n\nSequences of PCR primers used for RT-qPCR validation of commonly up- and down-regulated genes in zebrafish liver tumors.\n\n(DOCX)\nClick here for additional data file.\nTable S5\n\nDetails of pathways deregulated in the three transgenic zebrafish liver cancer models as classified into the seven cancer hallmarks and different aspects of the liver metabolisms.\n\n(DOCX)\nClick here for additional data file.\nTable S6\n\nDetails of pathways differentially expressed in the subgroups of human HCCs which showed significantly correlation with the three zebrafish signatures.\n\n(DOCX)\nClick here for additional data file.'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Introduction\nHuman hepatocellular carcinoma (HCC) is known to be a very heterogeneous disease, especially at intermediate and advanced stages &lt;&lt;REF:Llovet-1999-29-62&gt;&gt;. Due to the diverse and complex etiologies contributing to HCC incidence, different genetic mutations or altered molecular pathways could be responsible for hepatocarcinogenesis. So far, several carcinogenic pathways have been identified to be involved in the development and progression of HCC, including the VEGFR, EGFR, and mTOR pathways &lt;&lt;REF:Siegel-2010-52-360&gt;&gt;. In effort to decipher the role of different oncogenic pathways, a number of transgenic mouse models have been established &lt;&lt;REF:Leenders-2008-14-6915&gt;&gt;, &lt;&lt;REF:Lee-2004-36-1306&gt;&gt; and comparative transcriptomic analyses have been used to identify the best transgenic mouse models for human HCCs &lt;&lt;REF:Lee-2004-36-1306&gt;&gt;.\nThe zebrafish has been increasingly recognized as a valuable experimental model for human diseases, particularly for cancers &lt;&lt;REF:Liu-2011-6-71&gt;&gt; including liver cancers &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Liu-2012-56-2268&gt;&gt;. It has been shown that the zebrafish tumors had striking similarities with human cancer histologically &lt;&lt;REF:Amatruda-2002-1-229&gt;&gt;, &lt;&lt;REF:Spitsbergen-2000-28-705&gt;&gt;. Transcriptomic and epigenetic analyses have also indicated conserved features of carcinogen-induced zebrafish HCC with human HCC &lt;&lt;REF:Mirbahai-2011-12-3&gt;&gt;-&lt;&lt;REF:Lam-2006-24-73&gt;&gt;.\nPreviously we have generated several liver tumor models by transgenic expression of three different oncogenes (kras, xmrk or Myc) specifically in the zebrafish liver and these transgenic zebrafish usually produce liver tumors with variable degrees of severity from hepatocellular adenoma (HCA) to HCC &lt;&lt;REF:???-?-?-?&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;. The three oncogenes we used in the zebrafish have all been shown to be involved in hepatocarcinogenesis. KRAS is mutated in ~7% of liver cancers in human &lt;&lt;REF:Karnoub-2008-9-517&gt;&gt; but Ras signaling is ubiquitously activated in HCC &lt;&lt;REF:Calvisi-2006-130-1117&gt;&gt;. Xmrk is a naturally occurring variant of the EGFR in fish of the genus Xiphophorus (platyfish and swordtails) with constitutive autophosphorylation and activation of downstream signals &lt;&lt;REF:Gomez-2001-276-3333&gt;&gt;. Activation of EGFR signal is correlated with poor prognosis of HCC patients &lt;&lt;REF:Foster-2007-14-752&gt;&gt;. MYC is commonly amplified in many cancers including HCC and higher expression level of MYC is associated with more advanced status of HCC &lt;&lt;REF:Gan-1993-41-1185&gt;&gt;. We have shown that overexpression of kras and xmrk in the zebrafish liver could induce HCC &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt;-&lt;&lt;REF:Li-2011-56-419&gt;&gt;, while overexpression of Myc induced mostly HCA &lt;&lt;REF:???-?-?-?&gt;&gt;.\nComparative transcriptomic analyses of animal models and human clinical samples provide a powerful tool for identification of conserved molecular pathways and biomarker genes for diagnosis and therapy &lt;&lt;REF:Lam-2006-5-573&gt;&gt;, &lt;&lt;REF:Sweet-Cordero-2005-37-48&gt;&gt;. Our existing oncogene transgenic zebrafish models have well defined up-regulation of driver oncogene and this may provide a valuable tool to identify the molecular driving forces in human carcinogenesis by comparative transcriptomic analyses. Thus, in this study, we employed RNA sequencing technology to compare the transcriptomic profiles of the three oncogene-induced liver tumors in transgenic zebrafish. By comparative analyses with human liver transcriptomes from cirrhotic livers to very advanced HCC, we found that they all showed strong molecular correlation with advanced or very advanced human HCCs. Nevertheless, there are quite distinct deregulated biological pathways based on deregulated genes in the three oncogene transgenic models. Interestingly, each zebrafish liver tumor model correlated with a subset of human HCCs and each subset has some distinct molecular features. We showed that the transgenic zebrafish models with well-defined driver-gene activity should be valuable for classification of human HCCs and for understanding the molecular mechanisms behind each HCC subtype.\n\nMaterials and Methods\n\nTreatment and Induction of Liver Cancer in the Three Zebrafish Transgenic Models\nZebrafish were maintained following the approved protocol by Institutional Animal Care and Use Committee of National University of Singapore (Protocol 079/07). The generation of xmrk and Myc transgenic zebrafish was previously described and they were termed as TO(xmrk)\n&lt;&lt;REF:Li-2011-56-419&gt;&gt; and TO(Myc)\n&lt;&lt;REF:???-?-?-?&gt;&gt; respectively in the previous publications. The two transgenic lines were constructed by using a tetracycline-inducible transgenic system and the oncogene expression were induced by doxycycline. The krasV12 transgenic line used in the present study was newly generated by using a mifepristone-inducible system &lt;&lt;REF:Nguyen-2012-5-63&gt;&gt; in combination with a Cre-loxP system (unpublished). For the xmrk and Myc transgenic lines, transgenic fish and their non-transgenic siblings were treated with 60 microg/ml doxycycline (Sigma, USA) starting from 3.5 mpf (month post fertilization) for 6 weeks. All xmrk fish developed HCC and all Myc fish developed HCA. In total, for each transgenic line, one tumor sample (transgenic fish treated with doxycycline) and three control samples (non-transgenic siblings similarly treated with doxycycline, transgenic siblings without doxycycline treatment, and non-transgenic siblings without doxycycline treatment) were collected for RNA sequencing. In all cases, liver samples used for RNA sequencing were pooled from four to five male fish. For the krasV12 transgenic fish, one-month-old transgenic fish were treated with 1 microM mifepristone (Sigma, USA) for 36 hours to induce Cre-mediated recombination for activation of krasV12 transgene expression in the liver. The krasV12 activated transgenic fish were then allowed to grow for six months to develop HCC and HCC samples were then collected for RNA sequencing. Three liver tumors from induced transgenic fish and three normal livers from uninduced transgenic fish were pooled separately as tumor and control samples. All samples used were from male fish and two sets of biological replicates were used.\n\nIdentification of Signature Gene Lists in Each Zebrafish Liver Cancer Model\nTotal RNA was extracted using TRIzol Reagent (Invitrogen, USA) and treated with DNase I to remove genomic DNA contamination. 3' RNA-SAGE (serial analysis of gene expression) sequencing was performed on ABI SOLiD platform by Mission Biotech (Taiwan) according to manufacturer's protocol and 10-23 million reads were generated from each sampler (Table S1). Briefly, mRNA was purified using Dynabeads Oligo(dT) EcoP (Invitrogen) and subjected to cDNA synthesis. Resultant cDNA was digested by NlaIII and EcoP15I to result in a 27 nucleotides cDNA tag between the two sequencing adapters. The tags were mapped to the NCBI RefSeq (Reference Sequence) mRNA database for zebrafish with a criterion of allowing maximum 2 nucleotide mismatches. All RNA-Seq data were submitted to Gene Expression Omnibus database with the following access numbers: GSE53342 for xmrk and Myc data and GSE53630 for kras data. Tag counts for each transcript were normalized to TPM (transcripts per million) to facilitate comparison among different samples. The differentially regulated genes in the Myc- and xmrk- induced liver cancers were identified using one sample t-test as previously described &lt;&lt;REF:Zheng-2011-6-e24019&gt;&gt;, and the differentially expressed genes in the kras-induced liver cancer was identified using two-tailed Student's t-test.\nTo facilitate functional implications of zebrafish transcriptome, all zebrafish genes were mapped to annotated human genes in order to use existing online software developed in human genes. Thus, human homology mapping of zebrafish Unigene clusters were retrieved from the Genome Institute of Singapore Zebrafish Annotation Database (http://123.136.65.67/). For Unigene clusters mapped by more than one transcript entries, the highest TPM was used to represent the expression level of the Unigene cluster. Some zebrafish Unigene clusters were mapped to more than one human Unigene clusters, which usually came from the same gene family. To remove redundancy and avoid causing bias in functional analyses, only the first human Unigene cluster in the list was selected to represent the zebrafish Unigene clusters. The lists of significantly up-regulated zebrafish genes that were mapped with human homologs and used for comparative analyese with human HCC data are shown in Table S2.\n\nGene Set Enrichment Analysis (GSEA)\nGSEA was used to establish the relatedness between the transgenic zebrafish models and human liver cancers &lt;&lt;REF:Subramanian-2005-102-15545&gt;&gt;. GSEA is a computational method that determines whether a priori defined set of genes shows statistically significant, concordant differences between two biological samples; it calculates an enrichment score using a running-sum statistic through a ranked list of gene expression data set. Human homologs of the significantly up-regulated genes from the zebrafish tumor tissues were used as cancer signatures for each transgenic zebrafish model for transcriptomic comparison with human HCC data. Each chosen phenotype of human HCCs (either one specific HCC stage or one particular HCC sample) was compared to the rest of the samples in the same dataset. All genes in the chosen phenotype were ranked by t-test to determine expression differences among different HCC stages or different HCC samples. The enrichment score of the pre-defined transgenic zebrafish cancer signature was calculated using a running-sum statistic through the ranked genes. The statistical significance of the enrichment score was estimated by using an empirical phenotype-based permutation test procedure. An FDR (false discovery rate) value was provided by introducing adjustment of multiple hypothesis testing. Human liver cancer transcriptome data were retrieved from Gene Expression Omnibus (GEO) database. The human dataset including different stages of HCCs used in the comparison was GSE6764 &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The ten human HCC datasets used for examining the representation of zebrafish liver cancer gene signatures are summarized in Table S3. Annotation information was retrieved from SOURCE (http://smd.stanford.edu/cgi-bin/source/sourceSearch). For multiple probes which can be mapped to one Unigene cluster, the maximum signal intensity was selected to represent the expression level of the Unigene cluster.\n\nGSEA Pre-ranked Analysis\nGSEA pre-ranked option was used to analyze the deregulated pathways in each transgenic zebrafish model and subgroups of human HCCs. Briefly, the entire transcriptome was ranked by logarithm transformed p-value (base 10). The up-regulated genes were given positive values, and the down-regulated genes negative values. The curated canonical pathways from the MSigDB (Molecular Signature Database) were used. The number of permutation used was 1000.\n\nRT-qPCR Validation\nTotal RNA were reverse-transcribed using the SuperScript II cDNA Synthesis Kit (Invitrogen). RT-qPCR was performed with same sets of cDNAs used for SAGE sequencing using the LightCycler 480 SYBR Green I Master system (Roche). Reactions were conducted in triplicate for each cDNA sample and primer sequences are shown in the Table S4. Gene expression levels in each control or transgenic liver sample were normalized with the level of beta-actin mRNA as the internal control. The log2 fold changes between tumor and control samples were calculated using the CT method according to the formula: log2 fold changes = -DeltaDeltaCT = -[(CT gene of interest-CT beta-actin) transgenic sample-(CT gene of interest-CT beta-actin) control sample]. Two-tailed heteroscedastic t test was performed using normalized CT values (CT gene - CT beta-actin) and changes with p&lt;0.05 are considered to be significant.\n\nResults\n\nIdentification of Differentially Expressed Genes in the Three Transgenic Zebrafish Liver Cancer Models\nThe three oncogene transgenic lines (xmrk, Myc and krasV12) were induced to develop liver tumors (Figure S1) and these tumor samples were subjected to RNA-SAGE sequencing. By a selection criteria of fold change&gt;1.5, p value&lt;0.05 and TPM&gt;10 (in either control or tumor samples), differentially expressed genes were selected from the three tumor sets. There were 2,892, 797 and 494 genes up-regulated and 169, 902 and 676 genes down-regulated in the xmrk-, kras- and Myc- induced zebrafish liver cancer, respectively (Figure 1A,B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;). Deregulated genes from the three transgenic models showed relatively small overlaps, indicating that the three oncogenes regulated quite distinct sets of genes. This is consistent with the report that morphologically uniform cancer type is frequently classified into different subgroups based on their distinct gene expression patterns &lt;&lt;REF:Lee-2005-26-1013&gt;&gt;. Interestingly, there were 21 up-regulated and 16 down-regulated genes commonly found in all the three tumor models (Figure 1A, B &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table 1).\nFigure 1\nDeregulated genes and pathways in xmrk, kras and Myc transgenic liver tumors.\n(A) Venn diagram of up-regulated genes in the three zebrafish transgenic HCC models. (B) Venn diagram of down-regulated genes in the three zebrafish transgenic HCC models. (C) Deregulated pathways in the three oncogene transgenic liver tumors. Up- and down-regulated pathways in each zebrafish transgenic HCC model were analyzed by GSEA pre-ranked analysis. Red and green indicate up-and down-regulation respectively and the color code based on FDR is shown on the left. Pathways correlating with grey cells were not detected with any change. The up-regulated pathways were assigned into seven cancer hallmark categories (excluding enabling replicative immortality) according to Hanahan and Weinberg &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt; and down-regulated pathways was classified based on different aspects of the liver metabolism (see Table S1 for details).\nTable 1\nCommonly deregulated genes in the three transgenic zebrafish liver cancer models.\n\nDistinct Pathways Regulated in the Three Zebrafish Liver Tumor Models\nPathway analysis using GSEA showed that the three transgenic liver cancer models have different pathways deregulated (Figure 1C &lt;&lt;FIG:pone-0091179-g001&gt;&gt;, Table S5). It has been widely accepted that there are eight cancer hallmarks for multistep tumorigenesis and the complexities of neoplasms &lt;&lt;REF:Hanahan-2011-144-646&gt;&gt;, &lt;&lt;REF:Hanahan-2000-100-57&gt;&gt;. We found that the GSEA-identified pathways fallen into at least seven cancer hallmarks (except for enabling replicative immortality). Xmrk mainly up-regulated pathways involved in evading growth suppressors and avoiding immune destruction, which included activating cell cycle, promoting RNA transcription, up-regulating proteasome and altering immune properties. Kras provided the tumor cells with the ability of self-sustaining proliferative signals by up-regulating EGFR, Raf-MEK-ERK, PI3K-AKT-mTOR and GSK3 signaling pathways. Specifically, it also altered the focal adhesive characters of tumor cells which could activate invasion and metastasis. Myc mainly up-regulated translation and proteolysis to assist tumor cells to evade growth suppressors, and it also up-regulated VEGF pathway, thus potentially inducing angiogenesis. While there was no single pathway significantly up-regulated in all three tumor models, there were some pathways up-regulated in two models, such as proteasome in the xmrk and Myc models, and IGF1 pathway, mTOR pathway, tRNA biosynthesis and focal adhesion in the xmrk and kras models. In contrast, pathways in reprogramming of energy metabolism were generally down-regulated in all three models though the down-regulation in the Myc model was less apparent than the other two models. Meanwhile, many other pathways involved in normal liver function such as blood factors, amino acid metabolism, detoxification and xenobiotic metabolism, fatty acid metabolism, and bile synthesis were uniformly down-regulated in all three tumor models. However, one exception was hormone regulation that was apparently up-regulated in the kras tumors.\n\nCorrelation of the Three Gene Signatures of Zebrafish Liver Tumors with Different Stages of Human HCCs\nThe up-regulated genes, 2,892, 797 and 494 from xmrk, kras and Myc tumors respectively, were used as the signature genes for each model. These up-regulated genes were converted to 1,362, 490, and 146 human Unigenes respectively in order to compare with available transcriptomic data from human studies. We then compared the three signature gene sets with a set of human transcriptomic data (GSE6764) from different stages of human liver conditions: cirrhotic liver, low grade dysplastic nodules (LGDN), high grade dysplastic nodules (HGDN), and very early, early, advanced, and very advanced HCC (veHCC, eHCC, aHCC and vaHCC), in which the pathological HCC stages have been defined by tumor size, differentiation status and metastasis level &lt;&lt;REF:Wurmbach-2007-45-938&gt;&gt;. The three signature gene sets were all up-regulated as the disease progresses, but they started to be up-regulated at different stages of tumorigenesis (Figure 2 &lt;&lt;FIG:pone-0091179-g002&gt;&gt;). The xmrk signature showed positive correlation with HCCs starting from eHCC, and it was significantly correlated with aHCC and vaHCC. The kras signature positively correlated with aHCC and vaHCC, and it was only significantly correlated with vaHCC. The Myc signature showed significantly positive correlation with HCCs starting from eHCC, similar with our previous result using an independent set of RNA-seq data from the same transgenic line &lt;&lt;REF:???-?-?-?&gt;&gt;. Interestingly, the common 21 up-regulated genes in all three tumor models also showed up-regulation even from the very early stage of HCC, indicating these genes are correlated with the entire neoplastic process.\nFigure 2\nCorrelation of the zebrafish liver tumor signatures with different stages of human hepatocarcinogenesis.\nThe up-regulated gene signatures from the three transgenic tumors (xmrk, kras and Myc) and 21 commonly up-regulated genes (Common) were used for GSEA and NESs (normalized enrichment scores (y-axis) are plotted against different stages of human HCCs (x-axis). NES is listed in the table and asterisk indicate statistical significance: FDR&lt;0.25. Abbreviations: CL, cirrhotic livers; LGDN, low grade dysplastic nodules; HGDN, high grade dysplastic nodules; veHCC, very early HCC; eHCC, early HCC; aHCC, advanced HCC; vaHCC, very advanced HCC.\n\nRepresentation of the Three Gene Signatures of Zebrafish Liver Tumors in Human HCC Samples\nGiven the distinctive pathways altered in the three zebrafish liver tumor models and the high level of heterogeneity in human HCC patients, we sought to examine the degree of representation of the three zebrafish liver cancer models in human HCC samples. We examined ten sets of published clinical HCC microarray data, each of which included at least 80 clinical samples. The ten datasets contain a total of 1,272 samples covering different ethnic groups and risk factors (Table S2). We found that the xmrk, kras and Myc gene signatures were significantly correlated with 30.8%, 24.8% and 25.6% of all clinical samples, respectively. 47.2% of the human HCC samples had significant correlation with at least one of the three zebrafish gene signatures (Table 2, Figure S2). In different human HCC datasets, the percentage ranges from 22.0% to 60.4%. Moreover, some of the clinical samples were correlated with two or even three zebrafish gene signatures. Co-correlation of two gene signatures, namely xmrk/kras, kras/Myc and kras/Myc, accounted for 17.5%, 13.5% and 12.4% of the human HCC samples. 9.3% of the human samples showed co-correlation of all the three gene signatures. Thus, it appears that the three transgenic zebrafish liver tumor models represent molecular mechanisms of hepatocarcinogenesis in almost half of the human HCC cases and the other half of human HCC may be due to different molecular mechanisms.\nTable 2\nDegree of representation of the three transgenic zebrafish liver cancer signatures in human HCCs.\n\nPercentages indicate the percentages of human HCC samples which showed significant positive correlation (FDR&lt;0.25) with the zebrafish liver cancer signatures.\n\n*Total: The total percentage of human HCCs which showed significant correlation with any one or more of the zebrafish signatures.\nWe further demonstrated that the subsets of human HCCs which showed significantly positive correlation with the same gene signature also shared similar up-regulated pathways (Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, Table S6). For this analysis, each of the human HCC sets was separated into two subgroups: those showing significant correlation with one of the zebrafish signatures, and the rest. Differentially expressed genes between the two groups were identified by two-tailed t-test, and pathway analyses were performed by GSEA pre-ranked analysis. The pathways were subjected to hierarchical clustering by logarithm-transformed FDR values using MeV &lt;&lt;REF:Saeed-2003-34-374&gt;&gt;, &lt;&lt;REF:Saeed-2006-411-134&gt;&gt;. As shown in Figure 3 &lt;&lt;FIG:pone-0091179-g003&gt;&gt;, the pathways differentially regulated in human HCCs which showed significant correlation with one of the zebrafish signatures had distinct patterns. The pathway cluster A is consisted of pathways up-regulated in all the three subgroups, including proteasome, tRNA biosynthesis and oxidative phosphorylation. Ribosome, transcription and translation, and cell cycle were also highly up-regulated, suggesting that the human HCC samples significantly correlated with any of the zebrafish signatures were probably more proliferative than those not significantly correlated. The pathway cluster B contains pathways down-regulated in most of the subgroups associated with xmrk but up-regulated in the subgroups associated with kras and Myc. Interestingly, theses pathways were consistent with highly down-regulated pathways in the xmrk-induced zebrafish HCC, including energy metabolism, amino acid metabolism, fatty acid metabolism, bile acid biosynthesis, complement pathway, biosynthesis of steroid, and N-glycan biosynthesis. The pathway cluster C is more up-regulated in the kras-associated HCC subgroups, including glutamate metabolism and glycolysis. It has been reported that Kras could increase the conversion of glucose to glutamate, and this was essential for Kras-mediated tumorigenicity &lt;&lt;REF:Weinberg-2010-107-8788&gt;&gt;. The pathway cluster D was generally up-regulated in the xmrk- and kras-correlated human HCCs, but showed a disparate pattern in the Myc-correlated human HCCs. This cluster contained many kinase pathways which were deregulated in the xmrk- and kras-induced zebrafish liver cancer, but not significantly changed in the Myc-induced zebrafish liver cancer. The pathway cluster E was quite heterogeneous. The pathway cluster F was well separated from all the others and it contained pathways which were mostly down-regulated in all the human HCC subgroups associated with zebrafish signatures, including hematopoietic cell lineage, cytokine pathway, calcium signaling, and GPCR pathway. Since most of these down-regulated pathways are involved in inflammatory response, it is likely that the subgroups of human HCCs not captured by the three zebrafish models have more severe inflammatory status.\nFigure 3\nHierarchical clustering of pathways differentially expressed in the subgroups of human HCCs significantly correlated with the three zebrafish tumor signatures.\nThe color bars represent the logarithm-transformed FDR values. Up-regulated pathways are given positive values (red) and down-regulated pathways negative values (green). Pathways correlating with grey cells are not detected in the pathway analyses. Pathways with either FDR = 1 or not detected in more than five out of the 30 combinations are pre-excluded from the analysis.\n\nThe Commonly Up- and Down-regulated Genes in Zebrafish Liver Tumors are also Consistently Up- and Down-regulated in Human HCCs\nWhile the three gene signatures represent up-regulation of distinct pathways and correlate with different subgroups of human HCCs, we sought to investigate whether the 21 commonly up-regulated and 16 commonly down-regulated genes in all three transgenic models would be similarly regulated in human HCCs. Among the ten human HCC datasets we used, we were able to examine two datasets which included both HCCs and their corresponding non-tumor tissues (GSE14520 and GSE25097). However, only GSE14520 could be compared with the zebrafish data as most of the human genes homologous to the common zebrafish genes could be identified from the microarray platform used: 19 of the 21 up-regulated genes (except FOXA3 and MRPS9) and 12 of the 16 down-regulated genes (except for cyp2ad2, slco1d1, TDH and MIOX). GSE14520 dataset contains 229 primary HCCs and the corresponding paired non-tumor hepatic tissues &lt;&lt;REF:???-?-?-?&gt;&gt;, &lt;&lt;REF:Roessler-2010-70-10202&gt;&gt;. The xmrk, kras and Myc signatures were significantly correlated with 30.5%, 20.0%, and 20.5% of the HCC samples, and in total 47.6% of the HCC samples showed significant up-regulation of at least one of the zebrafish liver cancer signatures (Table 2). Among the 19 up-regulated genes, 16 of them were up-regulated in more than half of the HCC patients, and 9 of them were up-regulated in more than 75% of the patients (Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). STMN1 was up-regulated in 96.9% of the human HCCs from the dataset, which is the most ubiquitously up-regulated. STT3A and SRP14 were up-regulated in 93.0% of the human HCCs. Among the 12 down-regulated genes, 11 of them were down-regulated in more than half of the HCC patients. FBP1 was down-regulated in 96.9% of the examined human HCCs. FBP1 is a gluconeogenesis regulatory enzyme and it functions to antagonize glycolysis in gastric cancer &lt;&lt;REF:Liu-2010-29-442&gt;&gt;. FBP1 is down-regulated in majority of human HCCs by methylation &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;. Restoration of FBP1 expression in human HCC cell lines significantly inhibited cell growth, suggesting that it might function as a tumor suppressor &lt;&lt;REF:Chen-2011-6-e25564&gt;&gt;.\nFigure 4\n\nIn silico validation of the 21 commonly up-regulated and 16 commonly down-regulated genes in human HCC dataset.\n19 of the up-regulated genes and 12 of the down-regulated genes were identified in the microarray platform. The red color indicates up-regulation, and the green color indicates down-regulation.\nFinally, the expression of these 19 up- and 16 down-regulated genes was also validated by RT-qPCR in all three different types of tumors. As shown in Figure S3, the majority of gene in the majority of tests (~90%) confirmed consistent trend of changes in the tumor samples.\n\nDiscussion\nIt is well known that human HCCs are highly heterogeneous; thus, cross-species comparative studies at the transcriptomic level should be valuable to identify conserved and critical pathways in carcinogenesis in vertebrate species. Here we first determined deregulated pathways from each of the oncogene transgenic zebrafish model. Although the three transgenic tumor models had quite distinct deregulated biological pathways, they all correlated to advanced or very advanced HCCs by comparison with gene signatures from human HCCs. Furthermore, we also found that each of the zebrafish model represent a subset of human HCCs. Since our oncogene transgenic lines have well-defined driving pathways in carcinogenesis, the information from the transgenic zebrafish should be valuable for understanding the main molecular mechanisms of each HCC subgroup, which is imperative for developing more effective therapeutics specific for each subgroup. Interestingly, our three oncogene transgenic zebrafish models significantly represent only less than half of the human HCCs and there is a need to develop more and different oncogene transgenic animal models for covering more human HCCs for further understanding of distinct molecular mechanisms in hepatocarcinogenesis, with focus on inflammatory pathways.\nIn the present study, we also identified a list of commonly deregulated genes in liver tumors induced by different oncogenic signals was identified and their expressional changes in human HCC samples were validated in silico (Table 1, Figure 4 &lt;&lt;FIG:pone-0091179-g004&gt;&gt;). These genes could be served as potential therapeutic targets since they were independent of individual oncogenic pathways. The up-regulated genes includes those in protein translation and processing (eif5a2, abce1, rrp9, srp14, itm1, srprb), pro-apoptosis (rhot1a, c20orf24, fam162a) anti-apoptosis (tp53), cell cycle regulation (stmn1a, cdkrap3), purine synthesis (gart), rRNA processing (noc4l), G protein-coupled receptors signaling (reep2), n-glycan biosynthesis (mgat4b), stress response (cirbp), ubiquitination (ubap2), peroxisome (hmgcra), mitochondrial function (mrps9) and transcription (foxa3). Some of them have been implicated in hepatocarcinogenesis or identified as therapeutic targets. For example, hmgcra, a top up-regulated gene in all three tumor models, encodes the rate-limiting enzyme for cholesterol synthesis. Inhibition of Hmgcr could block tumor cell growth and metastasis &lt;&lt;REF:Cao-2011-71-2286&gt;&gt;, but clinical trials with Hmgcr inhibitor (pravastatin) have shown discrepant results &lt;&lt;REF:Lersch-2004-51-1099&gt;&gt;-&lt;&lt;REF:Kawata-2001-84-886&gt;&gt;, which may be attributed to the genetic heterogeneity of HCCs. The fact that hmgcra was highly up-regulated in all of the three transgenic zebrafish liver tumor models and these tumor models represent about half of human HCCs may suggest that it should be a therapeutic target in a broad, though not all, range of HCCs. Another top up-regulated gene, mgat4b, is one of the important enzymes in the biosynthetic pathway of N-glycans. N-glycan is up-regulated in human HCC &lt;&lt;REF:Goldman-2009-15-1808&gt;&gt; and is associated with drug resistance &lt;&lt;REF:Kudo-2007-6-32&gt;&gt;. Another gene which was highly up-regulated is stmn1a. The human homolog of stmn1a, STATHMIN1, is over-expressed and is associated with polyploidy, metastasis, early recurrence, and poor prognosis in hepatocarcinogenesis &lt;&lt;REF:Hsieh-2010-49-476&gt;&gt;-&lt;&lt;REF:Yuan-2006-209-549&gt;&gt;. It has also been identified as a major molecular target of an anticancer drug &lt;&lt;REF:Wang-2009-9-242&gt;&gt;.\nMost of the 16 genes commonly down-regulated (Table 1) are apparently involved in metabolism for normal liver function, including fatty acid metabolism (cyp2ad2, hsdl2, ech1), intracellular fatty acid and lipid transport (scp2, slc27a2, apobl), androgen metabolism (hsd17b3), glucose metabolism (fbp1b), mitochondria function (tdh, ak4), integrin complex (itgb1b.2), antioxidation (gpx4a, sod1), inositol catabolism (miox), bile acid metabolism (slco1d1) and neuron cell adhesion (nrxn1b). Several of them may have a direct connection with hepatocarcinogenesis. For example, Sod1 deficient mice showed extensive cellular oxidative damage and majority of them developed HCC &lt;&lt;REF:Elchuri-2005-24-367&gt;&gt;. Moreover, SOD1 has also been markedly down-regulated in human HCCs induced from different etiological factors &lt;&lt;REF:Li-2008-38-85&gt;&gt;. Gpx4 has also been reported to be associated with multiple types of cancers, including breast cancer &lt;&lt;REF:Mavaddat-2009-18-255&gt;&gt;, colorectal cancer &lt;&lt;REF:Meplan-2010-31-1074&gt;&gt;, and aggressive prostate cancer &lt;&lt;REF:Abe-2011-107-126&gt;&gt;; however, no study has presented its correlation with HCC. Both Sod1 and Gpx4 are important components of the cellular antioxidant mechanisms. The down-regulation of these two genes in the three transgenic zebrafish liver cancer models suggested that oxidative damage is a common and conserved part of hepatocarcinogenesis.\n\nSupporting Information\nFigure S1\n\nInduction of liver tumors in the three zebrafish transgenic liver cancer models. (A-D) Gross morphology of treated transgenic fish and the non-transgenic siblings. The treated non-transgenic siblings have normal liver size and gross morphology (A). The livers in the treated transgenic fish were obviously enlarged (B-D) compared to the treated non-transgenic siblings (A). (E-H) Histological examination of the treated transgenic fish and non-transgenic siblings. Treated xmrk fish developed HCC (F), treated Myc fish developed HCA (G), and treated kras fish developed heterogeneous HCC (H). The liver was circled out in white dotted lines. Scale bars: 25 mm for A-D, 50 microm for E-H.\n(TIF)\nClick here for additional data file.\nFigure S2\n\nCorrelation of the three transgenic zebrafish liver cancer signatures with human HCC samples. The heat maps showed the positive- and negative-correlation of the three transgenic zebrafish liver cancer signatures with 9 sets of human HCC samples, including GSE364 (A), GSE1898 (B), GSE10141 (C), GSE9843 (D), GSE19977 (E), GSE10186 (F), GSE20017 (G), GSE25097 (H) and GSE5975 (I). The color was determined by normalized enrichment score (NES) of the GSEA analysis. The red color indicates up-regulation or positive correlation, and the green color indicates down-regulation or negative correlation.\n(TIF)\nClick here for additional data file.\nFigure S3\n\nRT-qPCR validation of commonly up- and down-regulated genes in three transgenic zebrafish liver tumors. (A) Up-regulated genes. (B) Down-regulated genes. Gene names are indicated at the top and transgenic lines are indicaed on the left. RT-qPCR data are presented with red bars in comparison with corresponding RNA-Seq data represented by blue bars. Y-axes indicate fold changes on Log2 scale. Standard error bars are included for the RT-qPCR data and asterisks indicate statistically significance (P&lt;0.05).\n(TIF)\nClick here for additional data file.\nTable S1\n\nSummary of RNA-SAGE data.\n\n(DOCX)\nClick here for additional data file.\nTable S2\n\nSignificantary up-regulated zebrafish genes with mapped human homologs.\n\n(XLSX)\nClick here for additional data file.\nTable S3\n\nSummary of human HCC datasets used in the present Study.\n\n(DOCX)\nClick here for additional data file.\nTable S4\n\nSequences of PCR primers used for RT-qPCR validation of commonly up- and down-regulated genes in zebrafish liver tumors.\n\n(DOCX)\nClick here for additional data file.\nTable S5\n\nDetails of pathways deregulated in the three transgenic zebrafish liver cancer models as classified into the seven cancer hallmarks and different aspects of the liver metabolisms.\n\n(DOCX)\nClick here for additional data file.\nTable S6\n\nDetails of pathways differentially expressed in the subgroups of human HCCs which showed significantly correlation with the three zebrafish signatures.\n\n(DOCX)\nClick here for additional data file.'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [5.44s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"liver\",\n  \"tissue_name_structured\": \"liver\",\n  \"developmental_stage_text\": \"3.5 months post fertilization\",\n  \"developmental_stage_coarse\": \"juvenile\",\n  \"developmental_stage_fine\": \"juvenile\",\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"liver\",\n  \"tissue_name_structured\": \"liver\",\n  \"developmental_stage_text\": \"3.5 months post fertilization\",\n  \"developmental_stage_coarse\": \"juvenile\",\n  \"developmental_stage_fine\": \"juvenile\",\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 89,
      "prompt_tokens": 9117,
      "total_tokens": 9206
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_d986a8d1ba"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "liver",
  "tissue_name_structured": "liver",
  "developmental_stage_text": "3.5 months post fertilization",
  "developmental_stage_coarse": "juvenile",
  "developmental_stage_fine": "juvenile",
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [5.44s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "liver",
  "tissue_name_structured": "liver",
  "developmental_stage_text": "3.5 months post fertilization",
  "developmental_stage_coarse": "juvenile",
  "developmental_stage_fine": "juvenile",
  "is_scRNA_data": false
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Methods\n\nCAGE tags mapping and CTSS calling\nSequenced CAGE tags (27 bp) from Nepal et al.&lt;&lt;REF:Nepal-2013-23-1938&gt;&gt; were mapped to a reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. An additional G nucleotide, which is often attached to the 5' end of the tag by the template-free activity of the reverse transcriptase in the cDNA preparation step of CAGE protocol&lt;&lt;REF:Kodzius-2006-3-211&gt;&gt;, was removed in cases where it did not map to the genome. All unique 5' ends of tags were considered as CAGE tag-defined transcriptional start sites (CTSSs) and the number of tags supporting each CTSS was counted. Raw tag count was normalized to a referent power-law distribution based on total 106 tags and alpha=-1.25 as described in Balwierz et al.&lt;&lt;REF:Balwierz-2009-10-R79&gt;&gt; resulting in normalized tags per million (tpm). All analyses were done in R statistical computing environment&lt;&lt;REF:???-2013-?-1&gt;&gt; (http://www.R-project.org/) using Bioconductor&lt;&lt;REF:Gentleman-2004-5-R80&gt;&gt; (http://www.bioconductor.org/) software packages and custom scripts.\n\nCTSS clustering into TCs and promoter regions\nCTSSs supported by at least 1 tpm in at least one of the 12 developmental stages were clustered at two levels. First, tag clusters (TCs) were created for each stage individually using simple distance-based approach with a maximum allowed distance of 20bp between two neighbouring CTSSs. Next, for each TC we calculated a cumulative distribution of CAGE signal and determined the positions of 10th and 90th percentile to obtain more robust boundaries of a TC. TCs across all developmental stages within 100bp of each other were aggregated into a single promoter region. Only promoter regions supported by at least 5 tpm in at least one developmental stage were used in further analyses.\n\nExpression profiling\nExpression profiling was done at two levels: individual CTSSs and entire promoter regions. To minimize the noise from weakly supported CTSSs, we selected only CTSSs with at least 5tpm in at least one developmental stage. Normalized tpm values across 12 developmental stages for each CTSS (or promoter region) were divided by their standard deviation to obtain scaled expression measures. Self-organizing map&lt;&lt;REF:Toronen-1999-451-142&gt;&gt; (SOM) unsupervised learning algorithm was applied to distribute CTSSs (or promoter regions) across 5 x 5 = 25 expression profiles.\n\nDinucleotide patterns analysis\nTo visualize dinucleotide composition patterns of sequences flanking TSS we first created an occurrence matrix (n x m; where n = number of sequences and m = length of sequences) for each individual dinucleotide, by placing 1 if the given dinucleotide is present at given position or 0 if it is not. Values in the matrix were then smoothened: at each position in the matrix the weighted average dinucleotide occurrence was calculated by taking into account surrounding positions. Weights of the surrounding positions were assigned by centring a 2D Gaussian kernel with bandwidth = 3 (in both dimensions) at the central position. Matrix of smoothened values (densities) was visualized using different shades of blue in a map-like representation. Extended Data Figure 10 illustrates how the calculation and visualization was done.\n\nTATA-box motif analysis\nTATA-box position weight matrix (PWM) was obtained from JASPAR database&lt;&lt;REF:Portales-Casamar-2009-38-D105&gt;&gt; (http://jaspar.genereg.net/) and used to scan the region -35 bp to -22 bp upstream of TSS (expected position for a TATA-box according to Ponjavic et al.&lt;&lt;REF:Ponjavic-2006-7-R78&gt;&gt;). For each promoter sequence a maximal detected match (%) to TATA-box PWM was reported for maternal and zygotic dominant TSS separately. Distribution of obtained values across all promoters was visualized by histograms. In addition, the frequency of the top 10 most abundant pentamers found in the scanned sequences was shown. De novo motif discovery was performed on a set of 14 bp long sequences spanning the region from -35 bp to -22 bp upstream of TSS using MEME&lt;&lt;REF:Bailey-1994-2-28&gt;&gt; (http://meme.sdsc.edu/) with default parameters. Only motifs with E-value &lt;= 0.01 were selected as significant.\n\nsf3a2 promoter reporter constructs\nRegion spanning 500 bp upstream and 200 bp downstream of the dominant zygotic TSS in the sf3a2 promoter was chosen for validation of TSS selection grammar. sf3a2 promoter carries both maternal and zygotic promoter determinants and exhibits TSS shifting. The selected sequence, which ends within the first intron of sf3a2, was fused to a sequence containing 3' end of the zebrafish txnipa first intron and splice acceptor fused to a mCherry reporter (Extended Data Fig. 6a). Genomic DNA of AB* zebrafish strain was used for PCR amplification using Advantage HD DNA Polymerase Mix (Clontech). The amplified fragments were cloned into pDB896 vector (kindly provided by D. Balciunas, Temple University, Philadelphia, PA and subsequently modified by replacing the gamma-crystallin:gfp with mCherry) using In-Fusion(R) PCR Cloning System (Clontech) following the manufacturer's instructions. The expression cassette is flanked by Tol2 transposon arms for Tol2 transgenesis&lt;&lt;REF:Abe-2011-104-23&gt;&gt;. A polymorphic nucleotide (G-&gt;T) was identified in the promoter sequence at chr2:58,656,711 (Ensembl, Zebrafish Assembly Zv9/danRer7). The wild-type promoter reporter vector was used as a template for in site-directed mutagenesis PCR to introduce the mutation in the W-boxes as indicated in Fig. 3d &lt;&lt;FIG:F3&gt;&gt;. The resulting mutated PCR fragment was cloned in the same reporter vector using the In-Fusion(R) PCR Cloning System. Sequences of all primers are provided in Extended Data Figure 6b.\n\nMicroinjection and transgene expression analysis\n1.5-2 nl of injection solution containing 20 ng/mul reporter plasmid DNA and 15 ng/mul Tol2 transposase mRNA, supplemented with 0.1% Phenol red (injection marker), was injected into zebrafish eggs within 10-15 min after fertilization. In the automated imaging and expression analysis experiments ecfp mRNA was added (30 ng/mul). The mCherry reporter activity was measured at prim 20 stage, in both wild-type and mutated sf3a2 promoter construct injected embryos by automated imaging as described&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. Embryo images were analyzed with Zebrafish Miner software&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. The level of reporter expression was measured as pixel intensity value and normalized to the intensity of the ECFP signal (injection control) and averaged for all embryos in the experiment. In addition a percentage of expressing embryos was calculated from the total number of ECFP positive embryos with ECFP signal equal or above the detection threshold of the Zebrafish Miner software. Embryo images were wrapped onto reference embryo shape and overlaid by summing pixel intensity values.\n\n5' RACE\nThe 5' RACE was performed with FirstChoice(R) RLM-RACE Kit (Life Technologies) following the manufacturer's protocol. Total RNA was isolated at prim6 stage from ~100 phenotypically normal looking and reporter gene expressing zebrafish embryos injected with either wild type or mutated sf3a2 promoter reporter construct, using TRIZOL (Life Technologies) following the manufacturer's instructions. The PCR products of the expected size from the nested (inner) PCR reaction were purified from agarose gel and sequenced to identify the TSS. To demonstrate that the generated 5' RACE products are specific to the 5' ends of de-capped RNA, a \"minus TAP\" (Tobacco Acid Pyrophosphatase) treated sample was carried through adapter ligation, reverse transcription and PCR. Sequences of primers used in 5' RACE are provided in Extended Data Figure 6f.\n\nTransgenic zebrafish lines\nTransgenic zebrafish lines with the sf3a2 promoter (wild type and mutated) reporter constructs were generated by microinjection of the corresponding construct into zebrafish zygotes as described above. The reporter positive (mCherry) embryos were grown to adulthood and germline-transmitting female individuals were identified by crossing to wild type zebrafish and selecting for presence of reporter expressing offspring. Transgene expression and TSS usage was analysed in F1 embryos. Experiments were carried out under licence by the Home Office Licence Number 40/3681 and PPL 40/3131.\n\nQuantification of the reporter mRNA levels by qPCR\nRNA from reporter expressing embryos at high/sphere stage was isolated using GeneElute Total RNA extraction kit (Sigma-Aldrich), following the manufacturer's instructions. qPCR was performed using the SYBR Green detection method on 7900HT Fast Real-Time PCR System (Applied Biosystems). Two primer pairs were used for both the mCherry reporter and the endogenous sf3a2 gene (normalisation control). Technical triplicates were run for each primer pair. The Ct values were determined by the SDSv2.4 software (Applied Biosystems), using manual threshold of 0.2 and automatic base line. Expression levels of the transgene were calculated relative to the endogenous sf3a2 in the same sample, using the average Ct values across technical triplicates and both primer pairs. The sequence of the primers used in qPCR is provided in Extended Data Figure 7c.\n\nSL-CAGE\nWe have introduced a novel method for quantitative high-resolution detection of TSSs and their usage within a targeted promoter, called Single Locus deep CAGE. The method combines generation of 5' complete cDNAs transcribed from capped mRNAs as described in the CAGE protocol&lt;&lt;REF:Kane-1993-119-447&gt;&gt; with the amplification of targeted cDNAs using gene-specific primers and subsequent high throughput paired-end sequencing of the single locus (typically single promoter region) based library. Supplementary Table 2 describes main steps of the protocol and provides sequences of all primers used in different steps. Paired-end sequenced reads (34 bp + 35 bp) were mapped to either spliced sequence of sf3a2:mCherry transgene or endogenous sf3a2 and TSS usage was reconstructed as described above for CAGE tags.\n\nChromatin immunoprecipitation (ChIP)\nChIP experiments were carried out using the ChIP-IT Express Enzymatic kit (Active Motif) in line with the manufacturer's instructions. Chromatin was prepared using ~5000 and ~3000 embryos for 512 cell and oblong stage, respectively. Embryos were dechorionated enzymatically using pronase and fixed in 1.85% formaldehyde in Hanks Media for 20 min at room temperature. The embryos were washed once with PBS and the fixation was stopped by incubating in 1x Glycine for 10 min at room temperature followed by 3 washes with ice-cold PBS. Embryos were resuspended in 1 ml ice-cold lysis buffer, incubated on ice for 20 min, transferred to a pre-cooled dounce homogenizer and dounced by 10 strokes. Nuclei were collected by centrifugation, resuspended in 200 mul digestion buffer and incubated at 37degreesC for 5 min. Chromatin was sheared by adding 10 mul of enzymatic shearing cocktail working stock (200 U/ml) and incubating for 10 min at 37degreesC. Shearing efficiency was checked by gel electrophoresis according to manufacturer's instructions. The reaction was stopped by adding 5 mul ice-cold 0.5M EDTA and incubating on ice for 10 min and sheared chromatin was cleared by centrifugation. For ChIP reactions 70 mul of sheared chromatin were mixed with 25 mul Protein G magnetic beads, 20 mul ChIP buffer 1, 1mul protein inhibitor cocktail, 4 mug of anti-H3K4me3 (Abcam ab8580) or anti-H2A.Z (Abcam ab4174) antibody or an equivalent volume of water (no antibody control) respectively, and water to a final volume of 200mul. ChIP was performed in duplicates for each stage. ChIP reactions were then incubated overnight at 4degreesC while rotating. Magnetic beads were washed and incubated in elution buffer. After addition of reverse crosslinking buffer samples were decrosslinked for 4h at 65degreesC. Samples were Proteinase K and RNase A treated and purified using pheonol chloroform extraction.\n\nTBP knock-down\nOne-cell stage embryos were injected with either beta-actin:yfp&lt;&lt;REF:Higashijima-1997-192-289&gt;&gt; (1.7nl, 42pg/nl) or 1.3ntl:yfp&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; (1.7nl, 68pg/nl) constructs. The injected embryos were then split into four groups. One was kept as non-injected control and the other three groups were further injected with one of the two tbp-targeting morpholinos (1.7nl, 2.5mM) or with a mismatch morpholino described in Ferg et al.&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. All embryos were kept in E3 medium at 28.5degreesC until non-injected group reached 30% epiboly stage (4.7 hpf) and were then analyzed under fluorescence stereoscope (Nikon SMZ1500). Arrest of epiboly movements, loss of beta-actin:yfp (TBP dependent) and retention of ntl:yfp (TBP independent) reporter activities&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; were used as marker for assessing and sorting TBP morphants for ChIP analysis. Approximately 1500 non-injected, 1500 mismatch morpholino injected embryos, 1200 tbp mo1 morphants and 1000 tbp mo2 morphants were used for ChIP as described above. We used a previously published set of genes downregulated in zebrafish TBP morphants&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. Genes with log fold change &lt;=-1.5 were selected as TBP-dependent and were aligned with respect to dominant TSS of the nearest promoter detected by CAGE for H3K4me3 ChIP-seq signal visualisation.\n\nDeep sequencing of chromatin DNA\nChIP-seq was performed as described before&lt;&lt;REF:Soler-2011-53-151&gt;&gt;. In brief, 10 ng of ChIP DNA was end-repaired, ligated to single read adaptors, size selected and amplified for 18 cycles according to Illumina's ChIP-seq protocol. Cluster generation was performed according to the Illumina Cluster Reagents preparation protocol (http://www.illumina.com/). Samples were sequenced for 36 bp or 56 bp (tbp morphants and controls) on the HiSeq 2000 system.\n\nChIP-seq data analysis\nSequenced reads were mapped to the reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. Coverage was calculated for plus and minus strands separately using unextended reads and taking max. 20 reads mapping to the exactly same position. Minus strand coverage was subtracted from plus strand coverage to obtain subtracted coverage, which was used for visualisation and nucleosome midpoint estimation. Significantly enriched regions (peaks) were detected using MACS&lt;&lt;REF:Zhang-2008-9-R137&gt;&gt; (http://liulab.dfci.harvard.edu/MACS/) with default parameters. Midpoints of nucleosomes within significantly enriched regions (FDR&lt;=0.01) were estimated from subtracted coverage and the nearest CAGE signal was used to determine strand specificity and relative position of the first downstream nucleosome.\n\nSupplementary Material\nSupplementary Information Contents List\nSupplementary Table 1\nSupplementary Table 2\nExtended Data Legend\nExtended Data Figure 1\nExtended Data Figure 2\nExtended Data Figure 3\nExtended Data Figure 4\nExtended Data Figure 5\nExtended Data Figure 6\nExtended Data Figure 7\nExtended Data Figure 8\nExtended Data Figure 9\nExtended Data Figure 10",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Methods\n\nCAGE tags mapping and CTSS calling\nSequenced CAGE tags (27 bp) from Nepal et al.&lt;&lt;REF:Nepal-2013-23-1938&gt;&gt; were mapped to a reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. An additional G nucleotide, which is often attached to the 5' end of the tag by the template-free activity of the reverse transcriptase in the cDNA preparation step of CAGE protocol&lt;&lt;REF:Kodzius-2006-3-211&gt;&gt;, was removed in cases where it did not map to the genome. All unique 5' ends of tags were considered as CAGE tag-defined transcriptional start sites (CTSSs) and the number of tags supporting each CTSS was counted. Raw tag count was normalized to a referent power-law distribution based on total 106 tags and alpha=-1.25 as described in Balwierz et al.&lt;&lt;REF:Balwierz-2009-10-R79&gt;&gt; resulting in normalized tags per million (tpm). All analyses were done in R statistical computing environment&lt;&lt;REF:???-2013-?-1&gt;&gt; (http://www.R-project.org/) using Bioconductor&lt;&lt;REF:Gentleman-2004-5-R80&gt;&gt; (http://www.bioconductor.org/) software packages and custom scripts.\n\nCTSS clustering into TCs and promoter regions\nCTSSs supported by at least 1 tpm in at least one of the 12 developmental stages were clustered at two levels. First, tag clusters (TCs) were created for each stage individually using simple distance-based approach with a maximum allowed distance of 20bp between two neighbouring CTSSs. Next, for each TC we calculated a cumulative distribution of CAGE signal and determined the positions of 10th and 90th percentile to obtain more robust boundaries of a TC. TCs across all developmental stages within 100bp of each other were aggregated into a single promoter region. Only promoter regions supported by at least 5 tpm in at least one developmental stage were used in further analyses.\n\nExpression profiling\nExpression profiling was done at two levels: individual CTSSs and entire promoter regions. To minimize the noise from weakly supported CTSSs, we selected only CTSSs with at least 5tpm in at least one developmental stage. Normalized tpm values across 12 developmental stages for each CTSS (or promoter region) were divided by their standard deviation to obtain scaled expression measures. Self-organizing map&lt;&lt;REF:Toronen-1999-451-142&gt;&gt; (SOM) unsupervised learning algorithm was applied to distribute CTSSs (or promoter regions) across 5 x 5 = 25 expression profiles.\n\nDinucleotide patterns analysis\nTo visualize dinucleotide composition patterns of sequences flanking TSS we first created an occurrence matrix (n x m; where n = number of sequences and m = length of sequences) for each individual dinucleotide, by placing 1 if the given dinucleotide is present at given position or 0 if it is not. Values in the matrix were then smoothened: at each position in the matrix the weighted average dinucleotide occurrence was calculated by taking into account surrounding positions. Weights of the surrounding positions were assigned by centring a 2D Gaussian kernel with bandwidth = 3 (in both dimensions) at the central position. Matrix of smoothened values (densities) was visualized using different shades of blue in a map-like representation. Extended Data Figure 10 illustrates how the calculation and visualization was done.\n\nTATA-box motif analysis\nTATA-box position weight matrix (PWM) was obtained from JASPAR database&lt;&lt;REF:Portales-Casamar-2009-38-D105&gt;&gt; (http://jaspar.genereg.net/) and used to scan the region -35 bp to -22 bp upstream of TSS (expected position for a TATA-box according to Ponjavic et al.&lt;&lt;REF:Ponjavic-2006-7-R78&gt;&gt;). For each promoter sequence a maximal detected match (%) to TATA-box PWM was reported for maternal and zygotic dominant TSS separately. Distribution of obtained values across all promoters was visualized by histograms. In addition, the frequency of the top 10 most abundant pentamers found in the scanned sequences was shown. De novo motif discovery was performed on a set of 14 bp long sequences spanning the region from -35 bp to -22 bp upstream of TSS using MEME&lt;&lt;REF:Bailey-1994-2-28&gt;&gt; (http://meme.sdsc.edu/) with default parameters. Only motifs with E-value &lt;= 0.01 were selected as significant.\n\nsf3a2 promoter reporter constructs\nRegion spanning 500 bp upstream and 200 bp downstream of the dominant zygotic TSS in the sf3a2 promoter was chosen for validation of TSS selection grammar. sf3a2 promoter carries both maternal and zygotic promoter determinants and exhibits TSS shifting. The selected sequence, which ends within the first intron of sf3a2, was fused to a sequence containing 3' end of the zebrafish txnipa first intron and splice acceptor fused to a mCherry reporter (Extended Data Fig. 6a). Genomic DNA of AB* zebrafish strain was used for PCR amplification using Advantage HD DNA Polymerase Mix (Clontech). The amplified fragments were cloned into pDB896 vector (kindly provided by D. Balciunas, Temple University, Philadelphia, PA and subsequently modified by replacing the gamma-crystallin:gfp with mCherry) using In-Fusion(R) PCR Cloning System (Clontech) following the manufacturer's instructions. The expression cassette is flanked by Tol2 transposon arms for Tol2 transgenesis&lt;&lt;REF:Abe-2011-104-23&gt;&gt;. A polymorphic nucleotide (G-&gt;T) was identified in the promoter sequence at chr2:58,656,711 (Ensembl, Zebrafish Assembly Zv9/danRer7). The wild-type promoter reporter vector was used as a template for in site-directed mutagenesis PCR to introduce the mutation in the W-boxes as indicated in Fig. 3d &lt;&lt;FIG:F3&gt;&gt;. The resulting mutated PCR fragment was cloned in the same reporter vector using the In-Fusion(R) PCR Cloning System. Sequences of all primers are provided in Extended Data Figure 6b.\n\nMicroinjection and transgene expression analysis\n1.5-2 nl of injection solution containing 20 ng/mul reporter plasmid DNA and 15 ng/mul Tol2 transposase mRNA, supplemented with 0.1% Phenol red (injection marker), was injected into zebrafish eggs within 10-15 min after fertilization. In the automated imaging and expression analysis experiments ecfp mRNA was added (30 ng/mul). The mCherry reporter activity was measured at prim 20 stage, in both wild-type and mutated sf3a2 promoter construct injected embryos by automated imaging as described&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. Embryo images were analyzed with Zebrafish Miner software&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. The level of reporter expression was measured as pixel intensity value and normalized to the intensity of the ECFP signal (injection control) and averaged for all embryos in the experiment. In addition a percentage of expressing embryos was calculated from the total number of ECFP positive embryos with ECFP signal equal or above the detection threshold of the Zebrafish Miner software. Embryo images were wrapped onto reference embryo shape and overlaid by summing pixel intensity values.\n\n5' RACE\nThe 5' RACE was performed with FirstChoice(R) RLM-RACE Kit (Life Technologies) following the manufacturer's protocol. Total RNA was isolated at prim6 stage from ~100 phenotypically normal looking and reporter gene expressing zebrafish embryos injected with either wild type or mutated sf3a2 promoter reporter construct, using TRIZOL (Life Technologies) following the manufacturer's instructions. The PCR products of the expected size from the nested (inner) PCR reaction were purified from agarose gel and sequenced to identify the TSS. To demonstrate that the generated 5' RACE products are specific to the 5' ends of de-capped RNA, a \"minus TAP\" (Tobacco Acid Pyrophosphatase) treated sample was carried through adapter ligation, reverse transcription and PCR. Sequences of primers used in 5' RACE are provided in Extended Data Figure 6f.\n\nTransgenic zebrafish lines\nTransgenic zebrafish lines with the sf3a2 promoter (wild type and mutated) reporter constructs were generated by microinjection of the corresponding construct into zebrafish zygotes as described above. The reporter positive (mCherry) embryos were grown to adulthood and germline-transmitting female individuals were identified by crossing to wild type zebrafish and selecting for presence of reporter expressing offspring. Transgene expression and TSS usage was analysed in F1 embryos. Experiments were carried out under licence by the Home Office Licence Number 40/3681 and PPL 40/3131.\n\nQuantification of the reporter mRNA levels by qPCR\nRNA from reporter expressing embryos at high/sphere stage was isolated using GeneElute Total RNA extraction kit (Sigma-Aldrich), following the manufacturer's instructions. qPCR was performed using the SYBR Green detection method on 7900HT Fast Real-Time PCR System (Applied Biosystems). Two primer pairs were used for both the mCherry reporter and the endogenous sf3a2 gene (normalisation control). Technical triplicates were run for each primer pair. The Ct values were determined by the SDSv2.4 software (Applied Biosystems), using manual threshold of 0.2 and automatic base line. Expression levels of the transgene were calculated relative to the endogenous sf3a2 in the same sample, using the average Ct values across technical triplicates and both primer pairs. The sequence of the primers used in qPCR is provided in Extended Data Figure 7c.\n\nSL-CAGE\nWe have introduced a novel method for quantitative high-resolution detection of TSSs and their usage within a targeted promoter, called Single Locus deep CAGE. The method combines generation of 5' complete cDNAs transcribed from capped mRNAs as described in the CAGE protocol&lt;&lt;REF:Kane-1993-119-447&gt;&gt; with the amplification of targeted cDNAs using gene-specific primers and subsequent high throughput paired-end sequencing of the single locus (typically single promoter region) based library. Supplementary Table 2 describes main steps of the protocol and provides sequences of all primers used in different steps. Paired-end sequenced reads (34 bp + 35 bp) were mapped to either spliced sequence of sf3a2:mCherry transgene or endogenous sf3a2 and TSS usage was reconstructed as described above for CAGE tags.\n\nChromatin immunoprecipitation (ChIP)\nChIP experiments were carried out using the ChIP-IT Express Enzymatic kit (Active Motif) in line with the manufacturer's instructions. Chromatin was prepared using ~5000 and ~3000 embryos for 512 cell and oblong stage, respectively. Embryos were dechorionated enzymatically using pronase and fixed in 1.85% formaldehyde in Hanks Media for 20 min at room temperature. The embryos were washed once with PBS and the fixation was stopped by incubating in 1x Glycine for 10 min at room temperature followed by 3 washes with ice-cold PBS. Embryos were resuspended in 1 ml ice-cold lysis buffer, incubated on ice for 20 min, transferred to a pre-cooled dounce homogenizer and dounced by 10 strokes. Nuclei were collected by centrifugation, resuspended in 200 mul digestion buffer and incubated at 37degreesC for 5 min. Chromatin was sheared by adding 10 mul of enzymatic shearing cocktail working stock (200 U/ml) and incubating for 10 min at 37degreesC. Shearing efficiency was checked by gel electrophoresis according to manufacturer's instructions. The reaction was stopped by adding 5 mul ice-cold 0.5M EDTA and incubating on ice for 10 min and sheared chromatin was cleared by centrifugation. For ChIP reactions 70 mul of sheared chromatin were mixed with 25 mul Protein G magnetic beads, 20 mul ChIP buffer 1, 1mul protein inhibitor cocktail, 4 mug of anti-H3K4me3 (Abcam ab8580) or anti-H2A.Z (Abcam ab4174) antibody or an equivalent volume of water (no antibody control) respectively, and water to a final volume of 200mul. ChIP was performed in duplicates for each stage. ChIP reactions were then incubated overnight at 4degreesC while rotating. Magnetic beads were washed and incubated in elution buffer. After addition of reverse crosslinking buffer samples were decrosslinked for 4h at 65degreesC. Samples were Proteinase K and RNase A treated and purified using pheonol chloroform extraction.\n\nTBP knock-down\nOne-cell stage embryos were injected with either beta-actin:yfp&lt;&lt;REF:Higashijima-1997-192-289&gt;&gt; (1.7nl, 42pg/nl) or 1.3ntl:yfp&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; (1.7nl, 68pg/nl) constructs. The injected embryos were then split into four groups. One was kept as non-injected control and the other three groups were further injected with one of the two tbp-targeting morpholinos (1.7nl, 2.5mM) or with a mismatch morpholino described in Ferg et al.&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. All embryos were kept in E3 medium at 28.5degreesC until non-injected group reached 30% epiboly stage (4.7 hpf) and were then analyzed under fluorescence stereoscope (Nikon SMZ1500). Arrest of epiboly movements, loss of beta-actin:yfp (TBP dependent) and retention of ntl:yfp (TBP independent) reporter activities&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; were used as marker for assessing and sorting TBP morphants for ChIP analysis. Approximately 1500 non-injected, 1500 mismatch morpholino injected embryos, 1200 tbp mo1 morphants and 1000 tbp mo2 morphants were used for ChIP as described above. We used a previously published set of genes downregulated in zebrafish TBP morphants&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. Genes with log fold change &lt;=-1.5 were selected as TBP-dependent and were aligned with respect to dominant TSS of the nearest promoter detected by CAGE for H3K4me3 ChIP-seq signal visualisation.\n\nDeep sequencing of chromatin DNA\nChIP-seq was performed as described before&lt;&lt;REF:Soler-2011-53-151&gt;&gt;. In brief, 10 ng of ChIP DNA was end-repaired, ligated to single read adaptors, size selected and amplified for 18 cycles according to Illumina's ChIP-seq protocol. Cluster generation was performed according to the Illumina Cluster Reagents preparation protocol (http://www.illumina.com/). Samples were sequenced for 36 bp or 56 bp (tbp morphants and controls) on the HiSeq 2000 system.\n\nChIP-seq data analysis\nSequenced reads were mapped to the reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. Coverage was calculated for plus and minus strands separately using unextended reads and taking max. 20 reads mapping to the exactly same position. Minus strand coverage was subtracted from plus strand coverage to obtain subtracted coverage, which was used for visualisation and nucleosome midpoint estimation. Significantly enriched regions (peaks) were detected using MACS&lt;&lt;REF:Zhang-2008-9-R137&gt;&gt; (http://liulab.dfci.harvard.edu/MACS/) with default parameters. Midpoints of nucleosomes within significantly enriched regions (FDR&lt;=0.01) were estimated from subtracted coverage and the nearest CAGE signal was used to determine strand specificity and relative position of the first downstream nucleosome.\n\nSupplementary Material\nSupplementary Information Contents List\nSupplementary Table 1\nSupplementary Table 2\nExtended Data Legend\nExtended Data Figure 1\nExtended Data Figure 2\nExtended Data Figure 3\nExtended Data Figure 4\nExtended Data Figure 5\nExtended Data Figure 6\nExtended Data Figure 7\nExtended Data Figure 8\nExtended Data Figure 9\nExtended Data Figure 10",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Methods\n\nCAGE tags mapping and CTSS calling\nSequenced CAGE tags (27 bp) from Nepal et al.&lt;&lt;REF:Nepal-2013-23-1938&gt;&gt; were mapped to a reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. An additional G nucleotide, which is often attached to the 5' end of the tag by the template-free activity of the reverse transcriptase in the cDNA preparation step of CAGE protocol&lt;&lt;REF:Kodzius-2006-3-211&gt;&gt;, was removed in cases where it did not map to the genome. All unique 5' ends of tags were considered as CAGE tag-defined transcriptional start sites (CTSSs) and the number of tags supporting each CTSS was counted. Raw tag count was normalized to a referent power-law distribution based on total 106 tags and alpha=-1.25 as described in Balwierz et al.&lt;&lt;REF:Balwierz-2009-10-R79&gt;&gt; resulting in normalized tags per million (tpm). All analyses were done in R statistical computing environment&lt;&lt;REF:???-2013-?-1&gt;&gt; (http://www.R-project.org/) using Bioconductor&lt;&lt;REF:Gentleman-2004-5-R80&gt;&gt; (http://www.bioconductor.org/) software packages and custom scripts.\n\nCTSS clustering into TCs and promoter regions\nCTSSs supported by at least 1 tpm in at least one of the 12 developmental stages were clustered at two levels. First, tag clusters (TCs) were created for each stage individually using simple distance-based approach with a maximum allowed distance of 20bp between two neighbouring CTSSs. Next, for each TC we calculated a cumulative distribution of CAGE signal and determined the positions of 10th and 90th percentile to obtain more robust boundaries of a TC. TCs across all developmental stages within 100bp of each other were aggregated into a single promoter region. Only promoter regions supported by at least 5 tpm in at least one developmental stage were used in further analyses.\n\nExpression profiling\nExpression profiling was done at two levels: individual CTSSs and entire promoter regions. To minimize the noise from weakly supported CTSSs, we selected only CTSSs with at least 5tpm in at least one developmental stage. Normalized tpm values across 12 developmental stages for each CTSS (or promoter region) were divided by their standard deviation to obtain scaled expression measures. Self-organizing map&lt;&lt;REF:Toronen-1999-451-142&gt;&gt; (SOM) unsupervised learning algorithm was applied to distribute CTSSs (or promoter regions) across 5 x 5 = 25 expression profiles.\n\nDinucleotide patterns analysis\nTo visualize dinucleotide composition patterns of sequences flanking TSS we first created an occurrence matrix (n x m; where n = number of sequences and m = length of sequences) for each individual dinucleotide, by placing 1 if the given dinucleotide is present at given position or 0 if it is not. Values in the matrix were then smoothened: at each position in the matrix the weighted average dinucleotide occurrence was calculated by taking into account surrounding positions. Weights of the surrounding positions were assigned by centring a 2D Gaussian kernel with bandwidth = 3 (in both dimensions) at the central position. Matrix of smoothened values (densities) was visualized using different shades of blue in a map-like representation. Extended Data Figure 10 illustrates how the calculation and visualization was done.\n\nTATA-box motif analysis\nTATA-box position weight matrix (PWM) was obtained from JASPAR database&lt;&lt;REF:Portales-Casamar-2009-38-D105&gt;&gt; (http://jaspar.genereg.net/) and used to scan the region -35 bp to -22 bp upstream of TSS (expected position for a TATA-box according to Ponjavic et al.&lt;&lt;REF:Ponjavic-2006-7-R78&gt;&gt;). For each promoter sequence a maximal detected match (%) to TATA-box PWM was reported for maternal and zygotic dominant TSS separately. Distribution of obtained values across all promoters was visualized by histograms. In addition, the frequency of the top 10 most abundant pentamers found in the scanned sequences was shown. De novo motif discovery was performed on a set of 14 bp long sequences spanning the region from -35 bp to -22 bp upstream of TSS using MEME&lt;&lt;REF:Bailey-1994-2-28&gt;&gt; (http://meme.sdsc.edu/) with default parameters. Only motifs with E-value &lt;= 0.01 were selected as significant.\n\nsf3a2 promoter reporter constructs\nRegion spanning 500 bp upstream and 200 bp downstream of the dominant zygotic TSS in the sf3a2 promoter was chosen for validation of TSS selection grammar. sf3a2 promoter carries both maternal and zygotic promoter determinants and exhibits TSS shifting. The selected sequence, which ends within the first intron of sf3a2, was fused to a sequence containing 3' end of the zebrafish txnipa first intron and splice acceptor fused to a mCherry reporter (Extended Data Fig. 6a). Genomic DNA of AB* zebrafish strain was used for PCR amplification using Advantage HD DNA Polymerase Mix (Clontech). The amplified fragments were cloned into pDB896 vector (kindly provided by D. Balciunas, Temple University, Philadelphia, PA and subsequently modified by replacing the gamma-crystallin:gfp with mCherry) using In-Fusion(R) PCR Cloning System (Clontech) following the manufacturer's instructions. The expression cassette is flanked by Tol2 transposon arms for Tol2 transgenesis&lt;&lt;REF:Abe-2011-104-23&gt;&gt;. A polymorphic nucleotide (G-&gt;T) was identified in the promoter sequence at chr2:58,656,711 (Ensembl, Zebrafish Assembly Zv9/danRer7). The wild-type promoter reporter vector was used as a template for in site-directed mutagenesis PCR to introduce the mutation in the W-boxes as indicated in Fig. 3d &lt;&lt;FIG:F3&gt;&gt;. The resulting mutated PCR fragment was cloned in the same reporter vector using the In-Fusion(R) PCR Cloning System. Sequences of all primers are provided in Extended Data Figure 6b.\n\nMicroinjection and transgene expression analysis\n1.5-2 nl of injection solution containing 20 ng/mul reporter plasmid DNA and 15 ng/mul Tol2 transposase mRNA, supplemented with 0.1% Phenol red (injection marker), was injected into zebrafish eggs within 10-15 min after fertilization. In the automated imaging and expression analysis experiments ecfp mRNA was added (30 ng/mul). The mCherry reporter activity was measured at prim 20 stage, in both wild-type and mutated sf3a2 promoter construct injected embryos by automated imaging as described&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. Embryo images were analyzed with Zebrafish Miner software&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. The level of reporter expression was measured as pixel intensity value and normalized to the intensity of the ECFP signal (injection control) and averaged for all embryos in the experiment. In addition a percentage of expressing embryos was calculated from the total number of ECFP positive embryos with ECFP signal equal or above the detection threshold of the Zebrafish Miner software. Embryo images were wrapped onto reference embryo shape and overlaid by summing pixel intensity values.\n\n5' RACE\nThe 5' RACE was performed with FirstChoice(R) RLM-RACE Kit (Life Technologies) following the manufacturer's protocol. Total RNA was isolated at prim6 stage from ~100 phenotypically normal looking and reporter gene expressing zebrafish embryos injected with either wild type or mutated sf3a2 promoter reporter construct, using TRIZOL (Life Technologies) following the manufacturer's instructions. The PCR products of the expected size from the nested (inner) PCR reaction were purified from agarose gel and sequenced to identify the TSS. To demonstrate that the generated 5' RACE products are specific to the 5' ends of de-capped RNA, a \"minus TAP\" (Tobacco Acid Pyrophosphatase) treated sample was carried through adapter ligation, reverse transcription and PCR. Sequences of primers used in 5' RACE are provided in Extended Data Figure 6f.\n\nTransgenic zebrafish lines\nTransgenic zebrafish lines with the sf3a2 promoter (wild type and mutated) reporter constructs were generated by microinjection of the corresponding construct into zebrafish zygotes as described above. The reporter positive (mCherry) embryos were grown to adulthood and germline-transmitting female individuals were identified by crossing to wild type zebrafish and selecting for presence of reporter expressing offspring. Transgene expression and TSS usage was analysed in F1 embryos. Experiments were carried out under licence by the Home Office Licence Number 40/3681 and PPL 40/3131.\n\nQuantification of the reporter mRNA levels by qPCR\nRNA from reporter expressing embryos at high/sphere stage was isolated using GeneElute Total RNA extraction kit (Sigma-Aldrich), following the manufacturer's instructions. qPCR was performed using the SYBR Green detection method on 7900HT Fast Real-Time PCR System (Applied Biosystems). Two primer pairs were used for both the mCherry reporter and the endogenous sf3a2 gene (normalisation control). Technical triplicates were run for each primer pair. The Ct values were determined by the SDSv2.4 software (Applied Biosystems), using manual threshold of 0.2 and automatic base line. Expression levels of the transgene were calculated relative to the endogenous sf3a2 in the same sample, using the average Ct values across technical triplicates and both primer pairs. The sequence of the primers used in qPCR is provided in Extended Data Figure 7c.\n\nSL-CAGE\nWe have introduced a novel method for quantitative high-resolution detection of TSSs and their usage within a targeted promoter, called Single Locus deep CAGE. The method combines generation of 5' complete cDNAs transcribed from capped mRNAs as described in the CAGE protocol&lt;&lt;REF:Kane-1993-119-447&gt;&gt; with the amplification of targeted cDNAs using gene-specific primers and subsequent high throughput paired-end sequencing of the single locus (typically single promoter region) based library. Supplementary Table 2 describes main steps of the protocol and provides sequences of all primers used in different steps. Paired-end sequenced reads (34 bp + 35 bp) were mapped to either spliced sequence of sf3a2:mCherry transgene or endogenous sf3a2 and TSS usage was reconstructed as described above for CAGE tags.\n\nChromatin immunoprecipitation (ChIP)\nChIP experiments were carried out using the ChIP-IT Express Enzymatic kit (Active Motif) in line with the manufacturer's instructions. Chromatin was prepared using ~5000 and ~3000 embryos for 512 cell and oblong stage, respectively. Embryos were dechorionated enzymatically using pronase and fixed in 1.85% formaldehyde in Hanks Media for 20 min at room temperature. The embryos were washed once with PBS and the fixation was stopped by incubating in 1x Glycine for 10 min at room temperature followed by 3 washes with ice-cold PBS. Embryos were resuspended in 1 ml ice-cold lysis buffer, incubated on ice for 20 min, transferred to a pre-cooled dounce homogenizer and dounced by 10 strokes. Nuclei were collected by centrifugation, resuspended in 200 mul digestion buffer and incubated at 37degreesC for 5 min. Chromatin was sheared by adding 10 mul of enzymatic shearing cocktail working stock (200 U/ml) and incubating for 10 min at 37degreesC. Shearing efficiency was checked by gel electrophoresis according to manufacturer's instructions. The reaction was stopped by adding 5 mul ice-cold 0.5M EDTA and incubating on ice for 10 min and sheared chromatin was cleared by centrifugation. For ChIP reactions 70 mul of sheared chromatin were mixed with 25 mul Protein G magnetic beads, 20 mul ChIP buffer 1, 1mul protein inhibitor cocktail, 4 mug of anti-H3K4me3 (Abcam ab8580) or anti-H2A.Z (Abcam ab4174) antibody or an equivalent volume of water (no antibody control) respectively, and water to a final volume of 200mul. ChIP was performed in duplicates for each stage. ChIP reactions were then incubated overnight at 4degreesC while rotating. Magnetic beads were washed and incubated in elution buffer. After addition of reverse crosslinking buffer samples were decrosslinked for 4h at 65degreesC. Samples were Proteinase K and RNase A treated and purified using pheonol chloroform extraction.\n\nTBP knock-down\nOne-cell stage embryos were injected with either beta-actin:yfp&lt;&lt;REF:Higashijima-1997-192-289&gt;&gt; (1.7nl, 42pg/nl) or 1.3ntl:yfp&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; (1.7nl, 68pg/nl) constructs. The injected embryos were then split into four groups. One was kept as non-injected control and the other three groups were further injected with one of the two tbp-targeting morpholinos (1.7nl, 2.5mM) or with a mismatch morpholino described in Ferg et al.&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. All embryos were kept in E3 medium at 28.5degreesC until non-injected group reached 30% epiboly stage (4.7 hpf) and were then analyzed under fluorescence stereoscope (Nikon SMZ1500). Arrest of epiboly movements, loss of beta-actin:yfp (TBP dependent) and retention of ntl:yfp (TBP independent) reporter activities&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; were used as marker for assessing and sorting TBP morphants for ChIP analysis. Approximately 1500 non-injected, 1500 mismatch morpholino injected embryos, 1200 tbp mo1 morphants and 1000 tbp mo2 morphants were used for ChIP as described above. We used a previously published set of genes downregulated in zebrafish TBP morphants&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. Genes with log fold change &lt;=-1.5 were selected as TBP-dependent and were aligned with respect to dominant TSS of the nearest promoter detected by CAGE for H3K4me3 ChIP-seq signal visualisation.\n\nDeep sequencing of chromatin DNA\nChIP-seq was performed as described before&lt;&lt;REF:Soler-2011-53-151&gt;&gt;. In brief, 10 ng of ChIP DNA was end-repaired, ligated to single read adaptors, size selected and amplified for 18 cycles according to Illumina's ChIP-seq protocol. Cluster generation was performed according to the Illumina Cluster Reagents preparation protocol (http://www.illumina.com/). Samples were sequenced for 36 bp or 56 bp (tbp morphants and controls) on the HiSeq 2000 system.\n\nChIP-seq data analysis\nSequenced reads were mapped to the reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. Coverage was calculated for plus and minus strands separately using unextended reads and taking max. 20 reads mapping to the exactly same position. Minus strand coverage was subtracted from plus strand coverage to obtain subtracted coverage, which was used for visualisation and nucleosome midpoint estimation. Significantly enriched regions (peaks) were detected using MACS&lt;&lt;REF:Zhang-2008-9-R137&gt;&gt; (http://liulab.dfci.harvard.edu/MACS/) with default parameters. Midpoints of nucleosomes within significantly enriched regions (FDR&lt;=0.01) were estimated from subtracted coverage and the nearest CAGE signal was used to determine strand specificity and relative position of the first downstream nucleosome.\n\nSupplementary Material\nSupplementary Information Contents List\nSupplementary Table 1\nSupplementary Table 2\nExtended Data Legend\nExtended Data Figure 1\nExtended Data Figure 2\nExtended Data Figure 3\nExtended Data Figure 4\nExtended Data Figure 5\nExtended Data Figure 6\nExtended Data Figure 7\nExtended Data Figure 8\nExtended Data Figure 9\nExtended Data Figure 10'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Methods\n\nCAGE tags mapping and CTSS calling\nSequenced CAGE tags (27 bp) from Nepal et al.&lt;&lt;REF:Nepal-2013-23-1938&gt;&gt; were mapped to a reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. An additional G nucleotide, which is often attached to the 5' end of the tag by the template-free activity of the reverse transcriptase in the cDNA preparation step of CAGE protocol&lt;&lt;REF:Kodzius-2006-3-211&gt;&gt;, was removed in cases where it did not map to the genome. All unique 5' ends of tags were considered as CAGE tag-defined transcriptional start sites (CTSSs) and the number of tags supporting each CTSS was counted. Raw tag count was normalized to a referent power-law distribution based on total 106 tags and alpha=-1.25 as described in Balwierz et al.&lt;&lt;REF:Balwierz-2009-10-R79&gt;&gt; resulting in normalized tags per million (tpm). All analyses were done in R statistical computing environment&lt;&lt;REF:???-2013-?-1&gt;&gt; (http://www.R-project.org/) using Bioconductor&lt;&lt;REF:Gentleman-2004-5-R80&gt;&gt; (http://www.bioconductor.org/) software packages and custom scripts.\n\nCTSS clustering into TCs and promoter regions\nCTSSs supported by at least 1 tpm in at least one of the 12 developmental stages were clustered at two levels. First, tag clusters (TCs) were created for each stage individually using simple distance-based approach with a maximum allowed distance of 20bp between two neighbouring CTSSs. Next, for each TC we calculated a cumulative distribution of CAGE signal and determined the positions of 10th and 90th percentile to obtain more robust boundaries of a TC. TCs across all developmental stages within 100bp of each other were aggregated into a single promoter region. Only promoter regions supported by at least 5 tpm in at least one developmental stage were used in further analyses.\n\nExpression profiling\nExpression profiling was done at two levels: individual CTSSs and entire promoter regions. To minimize the noise from weakly supported CTSSs, we selected only CTSSs with at least 5tpm in at least one developmental stage. Normalized tpm values across 12 developmental stages for each CTSS (or promoter region) were divided by their standard deviation to obtain scaled expression measures. Self-organizing map&lt;&lt;REF:Toronen-1999-451-142&gt;&gt; (SOM) unsupervised learning algorithm was applied to distribute CTSSs (or promoter regions) across 5 x 5 = 25 expression profiles.\n\nDinucleotide patterns analysis\nTo visualize dinucleotide composition patterns of sequences flanking TSS we first created an occurrence matrix (n x m; where n = number of sequences and m = length of sequences) for each individual dinucleotide, by placing 1 if the given dinucleotide is present at given position or 0 if it is not. Values in the matrix were then smoothened: at each position in the matrix the weighted average dinucleotide occurrence was calculated by taking into account surrounding positions. Weights of the surrounding positions were assigned by centring a 2D Gaussian kernel with bandwidth = 3 (in both dimensions) at the central position. Matrix of smoothened values (densities) was visualized using different shades of blue in a map-like representation. Extended Data Figure 10 illustrates how the calculation and visualization was done.\n\nTATA-box motif analysis\nTATA-box position weight matrix (PWM) was obtained from JASPAR database&lt;&lt;REF:Portales-Casamar-2009-38-D105&gt;&gt; (http://jaspar.genereg.net/) and used to scan the region -35 bp to -22 bp upstream of TSS (expected position for a TATA-box according to Ponjavic et al.&lt;&lt;REF:Ponjavic-2006-7-R78&gt;&gt;). For each promoter sequence a maximal detected match (%) to TATA-box PWM was reported for maternal and zygotic dominant TSS separately. Distribution of obtained values across all promoters was visualized by histograms. In addition, the frequency of the top 10 most abundant pentamers found in the scanned sequences was shown. De novo motif discovery was performed on a set of 14 bp long sequences spanning the region from -35 bp to -22 bp upstream of TSS using MEME&lt;&lt;REF:Bailey-1994-2-28&gt;&gt; (http://meme.sdsc.edu/) with default parameters. Only motifs with E-value &lt;= 0.01 were selected as significant.\n\nsf3a2 promoter reporter constructs\nRegion spanning 500 bp upstream and 200 bp downstream of the dominant zygotic TSS in the sf3a2 promoter was chosen for validation of TSS selection grammar. sf3a2 promoter carries both maternal and zygotic promoter determinants and exhibits TSS shifting. The selected sequence, which ends within the first intron of sf3a2, was fused to a sequence containing 3' end of the zebrafish txnipa first intron and splice acceptor fused to a mCherry reporter (Extended Data Fig. 6a). Genomic DNA of AB* zebrafish strain was used for PCR amplification using Advantage HD DNA Polymerase Mix (Clontech). The amplified fragments were cloned into pDB896 vector (kindly provided by D. Balciunas, Temple University, Philadelphia, PA and subsequently modified by replacing the gamma-crystallin:gfp with mCherry) using In-Fusion(R) PCR Cloning System (Clontech) following the manufacturer's instructions. The expression cassette is flanked by Tol2 transposon arms for Tol2 transgenesis&lt;&lt;REF:Abe-2011-104-23&gt;&gt;. A polymorphic nucleotide (G-&gt;T) was identified in the promoter sequence at chr2:58,656,711 (Ensembl, Zebrafish Assembly Zv9/danRer7). The wild-type promoter reporter vector was used as a template for in site-directed mutagenesis PCR to introduce the mutation in the W-boxes as indicated in Fig. 3d &lt;&lt;FIG:F3&gt;&gt;. The resulting mutated PCR fragment was cloned in the same reporter vector using the In-Fusion(R) PCR Cloning System. Sequences of all primers are provided in Extended Data Figure 6b.\n\nMicroinjection and transgene expression analysis\n1.5-2 nl of injection solution containing 20 ng/mul reporter plasmid DNA and 15 ng/mul Tol2 transposase mRNA, supplemented with 0.1% Phenol red (injection marker), was injected into zebrafish eggs within 10-15 min after fertilization. In the automated imaging and expression analysis experiments ecfp mRNA was added (30 ng/mul). The mCherry reporter activity was measured at prim 20 stage, in both wild-type and mutated sf3a2 promoter construct injected embryos by automated imaging as described&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. Embryo images were analyzed with Zebrafish Miner software&lt;&lt;REF:Gehrig-2009-6-911&gt;&gt;. The level of reporter expression was measured as pixel intensity value and normalized to the intensity of the ECFP signal (injection control) and averaged for all embryos in the experiment. In addition a percentage of expressing embryos was calculated from the total number of ECFP positive embryos with ECFP signal equal or above the detection threshold of the Zebrafish Miner software. Embryo images were wrapped onto reference embryo shape and overlaid by summing pixel intensity values.\n\n5' RACE\nThe 5' RACE was performed with FirstChoice(R) RLM-RACE Kit (Life Technologies) following the manufacturer's protocol. Total RNA was isolated at prim6 stage from ~100 phenotypically normal looking and reporter gene expressing zebrafish embryos injected with either wild type or mutated sf3a2 promoter reporter construct, using TRIZOL (Life Technologies) following the manufacturer's instructions. The PCR products of the expected size from the nested (inner) PCR reaction were purified from agarose gel and sequenced to identify the TSS. To demonstrate that the generated 5' RACE products are specific to the 5' ends of de-capped RNA, a \"minus TAP\" (Tobacco Acid Pyrophosphatase) treated sample was carried through adapter ligation, reverse transcription and PCR. Sequences of primers used in 5' RACE are provided in Extended Data Figure 6f.\n\nTransgenic zebrafish lines\nTransgenic zebrafish lines with the sf3a2 promoter (wild type and mutated) reporter constructs were generated by microinjection of the corresponding construct into zebrafish zygotes as described above. The reporter positive (mCherry) embryos were grown to adulthood and germline-transmitting female individuals were identified by crossing to wild type zebrafish and selecting for presence of reporter expressing offspring. Transgene expression and TSS usage was analysed in F1 embryos. Experiments were carried out under licence by the Home Office Licence Number 40/3681 and PPL 40/3131.\n\nQuantification of the reporter mRNA levels by qPCR\nRNA from reporter expressing embryos at high/sphere stage was isolated using GeneElute Total RNA extraction kit (Sigma-Aldrich), following the manufacturer's instructions. qPCR was performed using the SYBR Green detection method on 7900HT Fast Real-Time PCR System (Applied Biosystems). Two primer pairs were used for both the mCherry reporter and the endogenous sf3a2 gene (normalisation control). Technical triplicates were run for each primer pair. The Ct values were determined by the SDSv2.4 software (Applied Biosystems), using manual threshold of 0.2 and automatic base line. Expression levels of the transgene were calculated relative to the endogenous sf3a2 in the same sample, using the average Ct values across technical triplicates and both primer pairs. The sequence of the primers used in qPCR is provided in Extended Data Figure 7c.\n\nSL-CAGE\nWe have introduced a novel method for quantitative high-resolution detection of TSSs and their usage within a targeted promoter, called Single Locus deep CAGE. The method combines generation of 5' complete cDNAs transcribed from capped mRNAs as described in the CAGE protocol&lt;&lt;REF:Kane-1993-119-447&gt;&gt; with the amplification of targeted cDNAs using gene-specific primers and subsequent high throughput paired-end sequencing of the single locus (typically single promoter region) based library. Supplementary Table 2 describes main steps of the protocol and provides sequences of all primers used in different steps. Paired-end sequenced reads (34 bp + 35 bp) were mapped to either spliced sequence of sf3a2:mCherry transgene or endogenous sf3a2 and TSS usage was reconstructed as described above for CAGE tags.\n\nChromatin immunoprecipitation (ChIP)\nChIP experiments were carried out using the ChIP-IT Express Enzymatic kit (Active Motif) in line with the manufacturer's instructions. Chromatin was prepared using ~5000 and ~3000 embryos for 512 cell and oblong stage, respectively. Embryos were dechorionated enzymatically using pronase and fixed in 1.85% formaldehyde in Hanks Media for 20 min at room temperature. The embryos were washed once with PBS and the fixation was stopped by incubating in 1x Glycine for 10 min at room temperature followed by 3 washes with ice-cold PBS. Embryos were resuspended in 1 ml ice-cold lysis buffer, incubated on ice for 20 min, transferred to a pre-cooled dounce homogenizer and dounced by 10 strokes. Nuclei were collected by centrifugation, resuspended in 200 mul digestion buffer and incubated at 37degreesC for 5 min. Chromatin was sheared by adding 10 mul of enzymatic shearing cocktail working stock (200 U/ml) and incubating for 10 min at 37degreesC. Shearing efficiency was checked by gel electrophoresis according to manufacturer's instructions. The reaction was stopped by adding 5 mul ice-cold 0.5M EDTA and incubating on ice for 10 min and sheared chromatin was cleared by centrifugation. For ChIP reactions 70 mul of sheared chromatin were mixed with 25 mul Protein G magnetic beads, 20 mul ChIP buffer 1, 1mul protein inhibitor cocktail, 4 mug of anti-H3K4me3 (Abcam ab8580) or anti-H2A.Z (Abcam ab4174) antibody or an equivalent volume of water (no antibody control) respectively, and water to a final volume of 200mul. ChIP was performed in duplicates for each stage. ChIP reactions were then incubated overnight at 4degreesC while rotating. Magnetic beads were washed and incubated in elution buffer. After addition of reverse crosslinking buffer samples were decrosslinked for 4h at 65degreesC. Samples were Proteinase K and RNase A treated and purified using pheonol chloroform extraction.\n\nTBP knock-down\nOne-cell stage embryos were injected with either beta-actin:yfp&lt;&lt;REF:Higashijima-1997-192-289&gt;&gt; (1.7nl, 42pg/nl) or 1.3ntl:yfp&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; (1.7nl, 68pg/nl) constructs. The injected embryos were then split into four groups. One was kept as non-injected control and the other three groups were further injected with one of the two tbp-targeting morpholinos (1.7nl, 2.5mM) or with a mismatch morpholino described in Ferg et al.&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. All embryos were kept in E3 medium at 28.5degreesC until non-injected group reached 30% epiboly stage (4.7 hpf) and were then analyzed under fluorescence stereoscope (Nikon SMZ1500). Arrest of epiboly movements, loss of beta-actin:yfp (TBP dependent) and retention of ntl:yfp (TBP independent) reporter activities&lt;&lt;REF:Ferg-2008-?-?&gt;&gt; were used as marker for assessing and sorting TBP morphants for ChIP analysis. Approximately 1500 non-injected, 1500 mismatch morpholino injected embryos, 1200 tbp mo1 morphants and 1000 tbp mo2 morphants were used for ChIP as described above. We used a previously published set of genes downregulated in zebrafish TBP morphants&lt;&lt;REF:Ferg-2007-26-3945&gt;&gt;. Genes with log fold change &lt;=-1.5 were selected as TBP-dependent and were aligned with respect to dominant TSS of the nearest promoter detected by CAGE for H3K4me3 ChIP-seq signal visualisation.\n\nDeep sequencing of chromatin DNA\nChIP-seq was performed as described before&lt;&lt;REF:Soler-2011-53-151&gt;&gt;. In brief, 10 ng of ChIP DNA was end-repaired, ligated to single read adaptors, size selected and amplified for 18 cycles according to Illumina's ChIP-seq protocol. Cluster generation was performed according to the Illumina Cluster Reagents preparation protocol (http://www.illumina.com/). Samples were sequenced for 36 bp or 56 bp (tbp morphants and controls) on the HiSeq 2000 system.\n\nChIP-seq data analysis\nSequenced reads were mapped to the reference zebrafish genome (Zv9/danRer7 assembly) using Bowtie&lt;&lt;REF:Langmead-2009-10-R25&gt;&gt; with default parameters allowing up to 2 mismatches and keeping only uniquely mapped reads. Coverage was calculated for plus and minus strands separately using unextended reads and taking max. 20 reads mapping to the exactly same position. Minus strand coverage was subtracted from plus strand coverage to obtain subtracted coverage, which was used for visualisation and nucleosome midpoint estimation. Significantly enriched regions (peaks) were detected using MACS&lt;&lt;REF:Zhang-2008-9-R137&gt;&gt; (http://liulab.dfci.harvard.edu/MACS/) with default parameters. Midpoints of nucleosomes within significantly enriched regions (FDR&lt;=0.01) were estimated from subtracted coverage and the nearest CAGE signal was used to determine strand specificity and relative position of the first downstream nucleosome.\n\nSupplementary Material\nSupplementary Information Contents List\nSupplementary Table 1\nSupplementary Table 2\nExtended Data Legend\nExtended Data Figure 1\nExtended Data Figure 2\nExtended Data Figure 3\nExtended Data Figure 4\nExtended Data Figure 5\nExtended Data Figure 6\nExtended Data Figure 7\nExtended Data Figure 8\nExtended Data Figure 9\nExtended Data Figure 10'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [5.99s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"not present\",\n  \"tissue_name_structured\": \"full embryo\",\n  \"developmental_stage_text\": \"12 developmental stages\",\n  \"developmental_stage_coarse\": [\n    \"embryo\"\n  ],\n  \"developmental_stage_fine\": [\n    \"embryo\"\n  ],\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"not present\",\n  \"tissue_name_structured\": \"full embryo\",\n  \"developmental_stage_text\": \"12 developmental stages\",\n  \"developmental_stage_coarse\": [\n    \"embryo\"\n  ],\n  \"developmental_stage_fine\": [\n    \"embryo\"\n  ],\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 93,
      "prompt_tokens": 4542,
      "total_tokens": 4635
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_0d1affc7a6"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "not present",
  "tissue_name_structured": "full embryo",
  "developmental_stage_text": "12 developmental stages",
  "developmental_stage_coarse": [
    "embryo"
  ],
  "developmental_stage_fine": [
    "embryo"
  ],
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [5.99s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "not present",
  "tissue_name_structured": "full embryo",
  "developmental_stage_text": "12 developmental stages",
  "developmental_stage_coarse": [
    "embryo"
  ],
  "developmental_stage_fine": [
    "embryo"
  ],
  "is_scRNA_data": false
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "Minor class splicing shapes the zebrafish transcriptome during development\nSebastian Markmiller a,b,1 , Nicole Cloonan c,2,3 , Rea M. Lardelli a,2,4 , Karen Doggett d,e , Maria-Cristina Keightley f , Yeliz Boglev a , Andrew J. Trotter a , Annie Y. Ng a,5 , Simon J. Wilkins c , Heather Verkade a,g , Elke A. Ober g,6 , Holly A. Field g , Sean M. Grimmond c , Graham J. Lieschke f , Didier Y. R. Stainier g,7 , and Joan K. Heath a,b,d,e,8\na Ludwig Institute for Cancer Research, Melbourne-Parkville Branch, Parkville, VIC 3050, Australia; b Department of Surgery, Royal Melbourne Hospital, University of Melbourne, Parkville, VIC 3050, Australia; c Queensland Centre for Medical Genomics, Institute for Molecular Bioscience, University of Queensland, St. Lucia, QLD 4072, Australia; d Walter and Eliza Hall Institute of Medical Research, Parkville, VIC 3052, Australia; e Department of Medical Biology, University of Melbourne, Parkville, VIC 3052, Australia; f Australian Regenerative Medicine Institute, Monash University, Clayton, VIC 3800, Australia; and g Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94158\nEdited* by Joan A. Steitz, Howard Hughes Medical Institute, New Haven, CT, and approved January 16, 2014 (received for review March 31, 2013)\nMinor class or U12-type splicing is a highly conserved process required to remove a minute fraction of introns from human pre- mRNAs. Defects in this splicing pathway have recently been linked to human disease, including a severe developmental disorder encom- passing brain and skeletal abnormalities known as Taybi-Linder syndrome or microcephalic osteodysplastic primordial dwarfism 1, and a hereditary intestinal polyposis condition, Peutz-Jeghers syn- drome. Although a key mechanism for regulating gene expression, the impact of impaired U12-type splicing on the transcriptome is unknown. Here, we describe a unique zebrafish mutant, caliban ( clbn ), with arrested development of the digestive organs caused by an ethylnitrosourea-induced recessive lethal point mutation in the rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ] gene. rnpc3 encodes the zebrafish ortholog of human RNPC3, also known as the U11/U12 di-snRNP 65-kDa protein, a unique compo- nent of the U12-type spliceosome. The biochemical impact of the mutation in clbn is the formation of aberrant U11- and U12-con- taining small nuclear ribonucleoproteins that impair the efficiency of U12-type splicing. Using RNA sequencing and microarrays, we show that multiple genes involved in various steps of mRNA pro- cessing, including transcription, splicing, and nuclear export are disrupted in clbn , either through intron retention or differential gene expression. Thus, clbn provides a useful and specific model of aberrant U12-type splicing in vivo. Analysis of its transcriptome reveals efficient mRNA processing as a critical process for the growth and proliferation of cells during vertebrate development.\nS plicing, the excision of introns from pre-mRNA, is an es- sential step in gene expression and a major source of complexity in the transcriptome (1). The process is catalyzed by highly dynamic complexes of small nuclear ribonucleoproteins (snRNPs) called spliceosomes (2). Not widely appreciated is the coexistence of two types of introns in most eukaryotic genomes. The vast majority, the major class or U2-type introns, are marked by GT and AG at their 5 ′ and 3 ′ ends, respectively. Minor class or U12-type introns, of which there are ∼ 700 in the human genome, were initially recog- nized by the presence of AT and AC in these positions, prompting their original name of AT-AC introns (3). However, we now know that most of these introns contain the same GT-AG termini found in U2-type introns (4, 5) and are instead distinguished from them by two highly conserved motifs: one adjacent to the 5 ′ splice site (ss) and one corresponding to the branch point sequence (BPS), close to the 3 ′ ss (6, 7). These introns also lack the 3 ′ polypyrimidine tract characteristic of U2-type introns. Minor class introns are excised by U12-type spliceosomes, which are analogous in function and similar in composition to U2-type spliceosomes; each comprise five small nuclear RNAs (snRNAs) and hun- dreds of associated proteins (6). Although the U5 snRNA is shared between the two complexes, the U12-type spliceosome contains four unique snRNAs (U11, U12, U4atac, and U6atac) (7 – 9) and at least seven unique proteins (10). Since its discovery two decades ago, the existence of this highly conserved, second\nsplicing pathway has been an intriguing aspect of gene ex- pression. Interestingly, U12-type introns are nonrandomly dis- tributed across the genome (11) and their removal is thought to be rate-limiting in the generation of mature mRNAs (12 – 14). Aberrant splicing is the basis of many human diseases and up to one-third of disease-causing mutations may lead to splicing defects (15). Although the majority of these defects occur in U2-type introns and splicing factors, perturbations in U12-type splicing have also been described. Most recently, mutations in the U12-type U4atac snRNA were found to underlie the human devel- opmental disorder Taybi-Linder syndrome (TALS)/microcephalic\n\n\nSignificance\nThe accurate removal of introns by pre-mRNA splicing is a crit- ical step in proper gene expression. Most eukaryotic genomes, from plant to human, contain a tiny subset of “ minor class ” introns with unique sequence elements that require their own splicing machinery. The significance of this second splicing pathway has intrigued RNA biologists for two decades, but its biological relevance was recently underscored when defects in the process were firmly linked to human disease. Here, we use a novel zebrafish mutant with defective minor class splicing to investigate how this pathway shapes the transcriptome during vertebrate development. We link its pleiotropic phenotype to widespread changes in gene expression that disrupt essential cellular pathways, including mRNA processing.\nAuthor contributions: S.M., R.M.L., and J.K.H. designed research; S.M., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., and H.V. performed research; N.C., H.V., E.A.O., H.A.F., S.M.G., and D.Y.R.S. contributed new reagents/analytic tools; S.M., N.C., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., G.J.L., and J.K.H. analyzed data; and S.M. and J.K.H. wrote the paper. The authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nFreely available online through the PNAS open access option.\nData deposition: The data reported in this paper have been deposited in the Gene Ex- pression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE53935 ). 1 Present address: Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla CA 92093.\n2 N.C. and R.M.L. contributed equally to this work. 3 Present address: QIMR Berghofer Medical Research Institute, Genomic Biology Labora- tory, Herston, QLD 4006, Australia.\n4 Present address: Division of Biology, University of California, San Diego, La Jolla, CA 92093.\n5 Present address: Institute of Medical Biology, Agency for Science, Technology and Re- search (A-STAR), Singapore 138648.\n6 Present address: The Danish Stem Cell Centre, University of Copenhagen, 2200 Copenhagen, Denmark.\n7 Present address: Department of Developmental Genetics, Max Planck Institute for Heart and Lung Research, D-61231 Bad Nauheim, Germany.\n8 To whom correspondance should be addressed. E-mail: joan.heath@wehi.edu.au. This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1305536111/-/DCSupplemental .\nosteodysplastic primordial dwarfism 1 (MOPD1) (16, 17). Other examples of diseases that can arise because of impaired splicing of U12-type introns include spinal muscular atrophy (18, 19) and Peutz-Jeghers syndrome (PJS), an intestinal polyposis syndrome that predisposes to cancer because of mutations in the tumor suppressor gene (TSG) LKB1 ( liver kinase B1 ) (20). The observation that impaired U12-type splicing can affect TSG expression has in- teresting implications for other TSGs containing U12-type introns, such as PTEN ( phosphatase and tensin homolog deleted on chromosome 10 ), where even a modest loss in PTEN function predisposes mice to cancer (21).\nTo specifically study the role of U12-type introns in gene ex- pression and development, we exploited a zebrafish mutant, caliban ( clbn ) , which carries a point mutation in rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ], encoding the zebrafish ortholog of human RNPC3, one of seven proteins unique to the U12-type spliceosome (22). We found aberrant minor class spliceosomal snRNPs in clbn extracts and used microarray and RNA sequencing (RNAseq) to show that impairment of U12-type splicing impacts broadly on gene expression, disrupting multiple genes involved in mRNA processing, transcription, splicing, and nuclear export. As a result, clbn mutants develop a pleiotropic phenotype with early lethality.\nResults The Intestine, Liver, and Pancreas Are Abnormal in clbn . clbn s846 was identified in a transgene-assisted ethylnitrosourea mutagenesis screen for zebrafish mutants with defective endodermal organ development (23). Abnormalities in the gross morphology of clbn larvae are first detectable at 108 h postfertilization (hpf) and mutants die between 7 and 10 d postfertilization (dpf). The head, eye, and lens are smaller, the swim bladder fails to inflate, and yolk absorption is delayed. At 120 hpf, the WT intestinal epi- thelium exhibits a columnar morphology and starts to elaborate folds; in contrast, the intestinal epithelium in clbn remains thin and unfolded (Fig. 1 A – C ). At 120 hpf, imaging of the digestive organs on the Tg(gutGFP) s854 (23) and Tg(fabp10:dsRed, ela3l: GFP) gz12 (24) backgrounds shows that the pancreas and liver are markedly smaller in clbn than in WT (Fig. 1 D and E ). clbn s846 Harbors a Mutation in rnpc3 . The clbn s846 locus was mapped to a region on chromosome 24 containing four genes, col11A1 , rnpc3 , prmt6 , and ntng1 ( Fig. S1 A ). RT-PCR amplification of rnpc3 from clbn s846 RNA generated aberrant rnpc3 isoforms (Fig. 2 A ), which were found by sequencing to correspond to the use of a de novo 3 ′ ss or retention of one or both of introns 13 and 14 (Fig. 2 B ). Sequencing of WT and clbn s846 genomic DNA re- vealed a T → A variation, creating a novel 3 ′ ss (AG) 10 nucleotides upstream of the canonical 3 ′ ss of intron 13 (Fig. 2 C ). The shift to the novel 3 ′ ss in clbn s846 is complete, as sequencing of the individual aberrant transcripts gave no evidence of a correctly spliced exon 13 – 14 junction in any of the products. All of the aberrant transcripts in clbn s846 contain premature stop codons after no more than six missense codons, making them likely targets for nonsense mediated decay ( Fig. S1 B ). We confirmed these findings using an in vivo minigene assay in which a WT or mutated minigene was injected into clutches containing WT and clbn s846 embryos. The T → A transition in the mutated ( clbn ) mini- gene led to a complete shift to the novel splice junction in both WT and clbn s846 embryos, as shown by a slightly shifted band in the agarose gel ( Fig. S1 C , lanes 1 and 3) and confirmed by sequencing. We provided firm evidence that rnpc3 is the mutated gene in clbn using rescue assays and a complementation test. Injection of one-cell stage clbn embryos with in vitro synthesized RNA encoding WT zebrafish Rnpc3 rescued the clbn phenotype (Fig. 2 D – G ). To perform a complementation test, we purchased a commercially available zebrafish line harboring a retroviral in- sertion in intron 1 of rnpc3 ( clbn ZM ) (Fig. 2 C ). We found that maternal deposits of correctly spliced, full-length rnpc3 mRNA in clbn ZM mutant embryos are largely depleted by 24 hpf ( Fig. S1 D ). clbn s846 and clbn ZM mutant larvae are morphologically\nindistinguishable at 120 hpf ( Fig. S1 F and G ). Upon crossing clbn s846 and clbn ZM heterozygotes, one-quarter of the offspring ex- hibit a phenotype indistinguishable from that of clbn s846 and clbn ZM mutants ( Fig. S1 H ). Taken together, these data show that rnpc3 is the mutated gene in clbn and suggest that both alleles are functionally null. We used whole mount in situ hybridization to show widespread rnpc 3 mRNA expression in WT embryos until 24 hpf ( Fig. S2 A and B ). Thereafter, the expression of rnpc3 becomes restricted to proliferating tissues, including the eye, lens, and digestive organs ( Fig. S2 C – F ). These tissues, which are abnormal in clbn , show reduced expression of rnpc3 in mutant larvae at 120 hpf ( Fig. S2 H ).\nclbn Contains Incompletely Spliced Transcripts. The zebrafish rnpc3 gene encodes a 505-aa nuclear protein with two RNA-recognition motifs (RRM) (25) and the same domain structure as its human ortholog (Fig. 3 A ). Whereas the excision of U2-type introns is initiated by the binding of individual U1 and U2 monosnRNPs, U12-type introns in human cells are recognized and bound by a preformed U11/U12 di-snRNP (26). This di-snRNP comprises U11 and U12 snRNAs, RNPC3/65K and two other proteins unique to the U12-type spliceosome, the U11/U12 59-kDa and 48-kDa proteins. Together, these components form a “ molecular bridge ” between the 5 ′ ss and BPS of U12-type introns (27, 28) (Fig. 3 B ). We used RT-PCR to assess the consequences of rnpc3 deficiency on the processing of mRNAs harboring U2- and U12-type introns in clbn and found that the U12-type introns in sms, mapk3 , and mapk12 were retained in clbn larvae (Fig. 3 C ), whereas the excision of the U2-type intron in mapk12 was unaffected.\nclbn Contains Aberrant Minor Spliceosomal snRNPs. To determine a biochemical basis for the observed specific defect in U12-type splicing, we sought to compare the size and conformation of zebrafish U11- and U12-containing snRNPs in WT and clbn larvae using glycerol gradient sedimentation followed by Northern blot analysis for U11 and U12 snRNAs. Extracts of WT larvae gave rise to U11 peaks in fractions 15 and 16 (Fig. 3 D , Upper ), whereas the U11 peak in clbn extracts was found two fractions closer to the top of the gradient, corresponding to a lighter U11 species (Fig. 3 D, Lower , and Fig. S3 A and B ). In contrast, glycerol gradient analyses of U12 snRNA sedimentation in clbn showed a dramatic shift of the predominant U12 signal toward a heavier particle than in WT (Fig. 3 E ). We further investigated these findings using native gel electrophoresis followed by Northern\nblotting for U11 and U12 snRNAs (Fig. 3 F ). Again, loss of Rnpc3 was found to disrupt the predominant U11 snRNPs and cause the accumulation of a slower-migrating U12-containing particle. This unexpected finding suggests a function for Rnpc3 beyond its known role in the U11/U12 di-snRNP, potentially at later stages of the splicing cycle. U12-containing complexes that might be stalled at any point after spliceosome activation would be expected to contain U6atac or U5 snRNAs (2). To test this theory we used native gel electrophoresis followed by Northern blotting for U12, U5, and U6atac snRNAs (Fig. 3 G ). To reduce potential nonspecific interactions and facilitate characterization of the stable complex accumulating in clbn embryos, heparin was added to the extracts before separation. Once again, we observed a strong signal corresponding to a heavier U12 snRNP complex\nFig. 2. The clbn phenotype is caused by a mutation in rnpc3 . ( A ) RT-PCR of full-length rnpc3 coding sequence amplifies a predominant 1.6-kb product in WT larvae (lane 4) and four aberrantly spliced transcripts in clbn s846 (lane 5). A retroviral insertion in clbn ZM abolishes generation of full-length rnpc3 mRNA (lane 6). Lanes 1 – 3 show – RT controls.\n( Lower ) Elongation factor alpha (elfa) control. Dashed lines link the cDNA products in lanes 4 and 5 to the corresponding transcripts, shown schemati- cally in B . The dark blue region denotes retention of 10 intronic nucleotides through use of a de novo 3 ′ ss. ( C ) Two noncomplementing clbn alleles. An as- terisk over T marks the position of the T → A varia- tion, 12 nucleotides upstream of exon 14 of rnpc3 , which causes the clbn s846 phenotype. The conserved 5 ′ ss and BPS in intron 14 (U12-type) of rnpc3 are underlined and in bold. clbn ZM carries a retroviral insertion in the first intron of rnpc3 . ( D – F ) Brightfield microscopy images of left lateral views of 5-dpf larvae derived from a clbn s846 heterozygous incross that were injected at the one- to two-cell stage with 380 pg of either WT rnpc3 mRNA or a mutant mRNA retaining introns 13 and 14, corresponding to the most abundant form of rnpc3 mRNA in clbn s846 (see B ). ( D ) No significant overexpression phenotype in WT larvae injected with WT rnpc3 mRNA. ( E ) Rescue of the mutant phenotype in genotyped clbn s846 homozygous mutant larvae injected with WT rnpc3 mRNA. ( F ) No rescue of the clbn mutant phenotype injected with mutant rnpc3 mRNA. ( G ) Noninjected control larvae from the same clutch show a near-perfect Mendelian ratio of WT to clbn larvae, whereas 96% of larvae injected with WT rnpc3 RNA appear WT at 5 dpf ( P &lt; 0.0001, Fisher ’ s exact test). There was no difference between uninjected and mutant rnpc3 -injected groups ( P = 0.9065). a, anus; e, eye; ib, intestinal bulb; sb, swim bladder; y, yolk. The brightfield images in D , E , and F were taken at the same magnification.\nFig. 3. Zebrafish Rnpc3 functions in the U12-type spliceosome. ( A ) Domain structure of zebrafish Rnpc3 and its human ortholog RNPC3, or U11/U12 di-snRNP 65-kDa protein ( www.uniprot.org , acces- sion no. Q96LT9). Red arrow in the C-terminal RRM denotes the approximate position of the premature stop codons in all clbn s846 isoforms. ( B ) Schematic of the human U11/U12 di-snRNP, with RNPC3 (65 K) bridging U12 snRNA and the 59 K protein (28). BPS, branch point sequence; “ A ” represents the branch point adenosine. ( C ) RT-PCR analysis of three dif- ferent transcripts ( sms , mapk3 , mapk12 ) shows retention of U12-type introns (lane 4, Top three panels) in clbn s846 (5 dpf), but not of a U2-type intron ( Bottom ). Lanes 1 and 2 show – RT controls. ( D and E ) Glycerol gradient/Northern analysis re- vealing differential sedimentation of U11 ( D ) and U12 ( E ) snRNA-containing snRNPs in WT and clbn ZM extracts. Direction of sedimentation is left to right.\nLane 1 contains total RNA. Full-length U11 or U12 signals were quantified using ImageQuant and ex- pressed as a percentage of the fraction with the highest intensity. U12 snRNA showed persistent specific degradation (asterisks) without affecting its migration profile. ( F ) Northern analysis of WT and clbn s846 extracts resolved on 4% (80:1) native poly-\nacrylamide gels and probed for U11 and U12 snRNAs. The predominant U11 snRNPs are disrupted in clbn compared with WT and the U12-containing particles are heavier and migrate more slowly (black arrowhead). ( G ) Northern analysis of WT and clbn s846 extracts resolved on native gels and probed for U5, U6atac, and U12 snRNAs shows retarded bands in clbn (red asterisks) compared with WT (blue asterisks). The same larval lysate was used in F and G . Data are representative of a total of 10 ( D and E ) and 5 ( F and G ) biological replicates.\nin clbn larvae from both alleles ( Fig. S3 C ), which appeared to comigrate with signals for both U5 and U6atac.\nTranscriptome Analysis Reveals a Specific U12-Type Splicing Defect in clbn . To determine the global impact of the clbn mutation on gene expression and U2- and U12-type splicing, we applied RNAseq to 108-hpf WT and clbn s846 mutants ( Table S1 ). RNAseq tags were aligned to the annotated zebrafish (Zv9) genome assembly ( Fig. S4 A ), which contains 606 manually curated U12-type introns ( SI Methods and Dataset S1 ). To normalize for the more common occurrence of nonunique sequences — such as repetitive elements — in introns compared with exons, we generated a zebra- fish uniqueome (29) and calculated all expression levels as reads per unique kilobase per million reads (RPuKM). Plotting ex- pression levels of all introns that satisfy minimum expression requirements (RPuKM &gt; 0.05) in both WT and clbn reveals a clear bias toward retention of U12-type introns in clbn (Fig. 4 A ). We found that a significantly higher fraction of U12-type introns (397 of 606 = 65.5%) show some degree of intron retention [intron retention coefficients (IRC) &gt; 0.05; defined in Fig. 4 B ] in WT larvae, compared with 49.6% of U2-type introns. This finding is consistent with U12-type introns being processed more slowly than their U2-type counterparts (12, 13). Remarkably, an addi- tional 148 of 606 (24.6%) of U12-type introns are detectable only in clbn (total = 90.1%), highlighting the specificity of the splicing defect. The defect in U12-type splicing in clbn is statistically sig- nificant (Fig. 4 C ); however, IRCs vary markedly between tran- scripts (Fig. 4 D ) and do not appear to be influenced by expression levels ( Fig. S4 B ), U12 intron length ( Fig. S4 C ), and AT-AC vs. GT-AG subtype ( Fig. S4 D ).\nActivation of Cryptic Splicing Occurs Rarely in clbn . A Peutz-Jeghers syndrome family has been described in which a mutation in the 5 ′ ss of the U12-type intron in LKB1 (20) leads to activation of cryptic splice sites and LKB1 haploinsufficiency. Activation of\ncryptic splice sites also occurs in human cells upon knockdown of the U11/U12 48-kDa protein with RNAi (30). To determine whether cryptic splice sites are activated in clbn , we sequenced the spliced transcripts obtained by RT-PCR for sms , mapk3 , and mapk12 mRNAs (Fig. 2 C ) and found only the annotated exon – exon junctions. We then used RT-PCR to amplify three more U12-type introns with low IRCs, namely fkbp3 , snrpe , and ppp2r2c . We found very low levels of U12-type intron retention in snrpe and ppp2r2c in clbn and no evidence of cryptic splicing for all three genes ( Fig. S4 E ). Furthermore, we also found no evidence of activation of cryptic splice sites in clbn when we aligned RNAseq reads against a custom-built library of exon junctions within 100 bp of the annotated splice junctions of all manually curated zebrafish U12-type introns ( Fig. S4 F and Dataset S2 ). Collectively these data show that impaired U12-type splicing in clbn is rarely circumvented by activation of cryptic splicing.\nRnpc3 Deficiency Impacts Broadly on Gene Expression. We combined datasets from microarray analysis and RNAseq to identify high- confidence differentially expressed genes (DEGs) between WT and clbn s846 larvae ( Datasets S3 – S5 ). Fold-changes for 203 DEGs (113 down, 90 up) were significantly correlated for the two methods ( Fig. S5 A ). As expected, rnpc3 mRNA levels were reduced, con- sistent with predicted nonsense mediated decay of rnpc3 transcripts carrying the clbn s846 mutation. Of the down-regulated genes, many exhibit tissue-specific expression in tissues that develop ab- normally in clbn , including the liver, pancreas, intestine, and lens ( Dataset S3 ). Consistent with a prior study in Drosophila (31), a substantial overrepresentation of U12-type intron-containing genes was observed among the DEGs, with a higher proportion of U12-type intron-containing genes in the down-regulated DEGs (35%; 40 of 113) than in the up-regulated set (11%; 10 of 90). Even so, about 90% of U12-type intron-containing genes were not differentially expressed ( Fig. S5 B ) and most of the DEGs do not contain U12-type introns; hence, Rnpc3 deficiency impacts broadly on gene expression and is not limited to U12-type intron- containing genes.\nExpression of mRNA Quality Control and Cell Cycle Genes Is Altered in clbn . U12-type introns are enriched in “ information processing ” genes encoding functions in DNA replication, repair, transcrip- tion, mRNA processing, and translation (11). In clbn , many such genes, particularly those encoding components of mRNA quality control pathways, such as exosome factors, are either differen- tially expressed ( dcp2 , edc4 , exosc1 , exosc5 , exosc9 ) or exhibit varying degrees of intron retention ( edc4 , exosc1 , exosc2 , exosc5 ) (Fig. 5, Fig. S5 C , and Table S2 ). Although the primary defect caused by the lack of Rnpc3 is in the processing of U12-type introns, the resulting widespread disruption to the expression of genes regulating RNA processing is likely to have a secondary impact on the expression of U2-type intron-containing genes, explaining why they feature so strongly in the list of DEGs. clbn mutants at 108 hpf also exhibit elevated expression of cell- cycle repressor genes, including prohibitin , cyclin G1 , e2f5 , and Δ 113p53 ( Fig. S5 D – F ). Δ 113p53 is a Tp53 target gene encoding an N-terminally truncated isoform of Tp53 thought to antagonize its proapoptotic activity (32).\n\n\nDiscussion\nU12-type splicing was a surprise discovery two decades ago, and its biological significance has remained something of an enigma. Here, we describe a unique vertebrate model system with a mu- tation in a core component that is specific to the U12-type spliceosome, and exploit it to gain new insights into the role of U12-type splicing in the regulation of gene expression during development and disease.\nRnpc3 Deficiency Leads to a Pleiotropic Phenotype in Developing Zebrafish. The loss of Rnpc3, either because of an intronic point mutation or a retroviral insertion into the rnpc3 locus, leads to a severe and pleiotropic phenotype in developing zebrafish\nFig. 4. Intron retention in clbn is specific to U12-type introns and highly variable. ( A ) Scatterplot of normalized intron expression derived by RNAseq for reference (Refseq) introns (gray dots) and U12-type introns (red dots). U12-type introns lie almost entirely above the midline representing equal expression, indicating enhanced intron retention in clbn . Intron expression is given in RPuKM. ( B ) To quantitate U12-type intron retention in individual transcripts, we defined an intron retention coefficient, which calculates in- tron retention relative to transcript level. ( C ) Comparison of the median log 2 of the IRCs for Refseq introns and U12-type introns between WT and clbn demonstrates that the clbn splicing defect is specific to U12-type introns. See also Fig. S4 . Statistical significance was determined by Student t test. ( D ) University of California at Santa Cruz genome browser views of three genes with high, intermediate, and low IRCs. ccdc43 shows a high IRC of 0.97, dcp2 an intermediate IRC of 0.41, and hsbp1 shows a low IRC of 0.05.\nlarvae, culminating in death of all larvae by 10 dpf. The most severely affected tissues are those that are highly proliferative between 48 and 96 hpf, including the digestive organs, eye, and lens. Our observation that Rnpc3 deficiency impacts broadly on gene expression, extending well beyond the small subset of genes containing U12-type introns, makes it difficult to pinpoint pre- cisely the molecular basis for the clbn phenotype. As yet, we do not know whether the abnormalities arise directly from pertur- bations in the expression of a small number of U12-type intron- containing genes, such as the dysregulated cell-cycle repressor genes we identified, or are the result of complex secondary effects emanating from a larger network of genes whose expression is affected by impaired U12-type splicing.\nComparison of the clbn Phenotype with MOPD. The severe human developmental disorder known as TALS/MOPD1 (16, 17, 33, 34) is caused by impaired U12-type splicing. Here, the affected infants inherit two recessive point mutations in U4atac snRNA and suffer extreme intrauterine growth retardation, severe mi- crocephaly, central nervous system abnormalities, dysmorphic facial features, skin and skeletal abnormalities, and early death (34). Although an impact on the growth of proliferating tissues is a common link in the zebrafish and human conditions, most MOPD1 features are not conspicuous in clbn . One possible ex- planation for this lack of concordance is that many of the severe defects observed in MOPD1 patients, such as microcephaly, most likely arise early in development. In zebrafish, adverse events early in embryogenesis are frequently circumvented by the availability of maternally derived WT mRNAs. In the particular case of clbn , we have shown that rnpc3 mRNA is maternally deposited, and it is reasonable to expect that this will also be the case for at least a proportion of the genes dependent on efficient minor class splicing, as well as their downstream effectors. If this hypothesis is correct, maternal-zygotic mutants of our clbn alleles would be expected to exhibit a more extensive phenotype that more closely mimics MOPD1.\nRnpc3 Deficiency Disrupts U11/U12 di-snRNP Assembly and Splicing. We examined the distribution of minor class snRNPs in rnpc3 - deficient zebrafish larvae to provide a molecular understanding\nof the specific U12-type splicing defects observed. Although the dynamic nature of snRNP interactions throughout the splicing cycle is relatively well characterized in mammalian cells, very little is known about this process in zebrafish. Previous studies have shown that the sedimentation profiles of zebrafish snRNPs in glycerol gradients differ in some aspects to those obtained from human cells (35, 36). Similarly, our glycerol gradient and native gel electrophoresis experiments reveal signals for U11 and U12 snRNPs that do not clearly correspond to a distribution into U11 and U12 mono-snRNPs and a U11/U12 di-snRNP, as de- scribed in human cells. Notwithstanding that, our data clearly demonstrate a specific role for Rnpc3 in maintaining the in- tegrity and proper distribution of U11 and U12 snRNPs. This result contrasts with our analysis of another zebrafish splicing mutant ( cph ), where a mutation in the shared core splicing factor, prpf8 , disrupts U2-type U4/U6/U5 tri-snRNP formation with negligible impact on U11/U12 snRNPs (35). In this study, the strong accumulation in clbn of a heavy, potentially stalled, complex containing U12, U5, and U6atac snRNAs, which does not seem to be a stable intermediate in WT, suggests a hitherto unsuspected role for Rnpc3 in spliceosome disassembly or recycling. Further studies are required to confirm this.\nRegulation of U12-Type Splicing. The results of our transcriptome analysis show that approximately two-thirds of U12-type intron- containing mRNAs exhibit some degree of U12-type intron retention even in developing WT zebrafish, much higher than the fraction of U2-type introns retained. At first sight, U12-type splicing appears to be a relatively inefficient process that may limit the full expression of U12-type intron-containing genes. However, recent findings suggest that regulation of U12-type splicing may be more complicated than previously thought. For example, p38MAPK-mediated up-regulation of the otherwise rate-limiting levels of U6atac in HeLa cells appears to con- stitute a sophisticated regulatory mechanism to facilitate the rapid mobilization of critical U12-type intron-containing transcripts in response to growth stimuli (14). Because many of these tran- scripts encode genes with functions in mRNA production, such an amplification of U12-type splicing activity is likely to generate a robust transcriptome-wide response.\nU12-Type Splicing and Disease. Recent breakthroughs that firmly link defects in U12-type splicing to human disease (16, 17, 19, 20) provide a strong impetus to explore its potential involvement in other pathological conditions. Interestingly, we have noted that U12-type introns are found in a number of cancer genes, including oncogenes, such as BRAF , C-RAF , and several mem- bers of the MAPK family. Extrapolating from our results in clbn , where the survival and integrity of rapidly proliferating cells is dependent on U12-type splicing, it is tantalizing to speculate that the efficiency of minor class splicing could influence the growth of tumors, particularly those that are dependent on the RAS- MAPK pathway (37, 38). In this regard, it is intriguing that p38MAPK has recently emerged as a key regulator of U12-type splicing activity (16), suggesting a potential regulatory mecha- nism to achieve efficient levels of U12-type splicing in tumors that are reliant on the MAPK pathway. Future investigations will determine whether efficient U12-type splicing is an im- portant contributor to the posttranscriptional RNA processing mechanisms governing gene expression in cancer.\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n3064 | www.pnas.org/cgi/doi/10.1073/pnas.1305536111\n\n\nMarkmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3065\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n1. Licatalosi DD, Darnell RB (2010) RNA processing and its regulation: Global insights into biological networks. Nat Rev Genet 11(1):75 – 87.\n\n\n2. Wahl MC, Will CL, Lührmann R (2009) The spliceosome: Design principles of a dynamic RNP machine. Cell 136(4):701 – 718.\n\n\n3. Jackson IJ (1991) A reappraisal of non-consensus mRNA splice sites. Nucleic Acids Res 19(14):3795 – 3798.\n\n\n6. Jurica MS, Moore MJ (2003) Pre-mRNA splicing: Awash in a sea of proteins. Mol Cell 12(1):5 – 14.\n\n\n20. Hastings ML, et al. (2005) An LKB1 AT-AC intron mutation causes Peutz-Jeghers syndrome via splicing at noncanonical cryptic splice sites. Nat Struct Mol Biol 12(1): 54 – 59.\n\n\n23. Ober EA, Verkade H, Field HA, Stainier DY (2006) Mesodermal Wnt2b signalling positively regulates liver specification. Nature 442(7103):688 – 691.\n\n\n41. Taylor J, Schenck I, Blankenberg D, Nekrutenko A (2007) Using galaxy to perform large-scale interactive data analyses. Curr Protoc Bioinformatics Chapter 10:5. Markmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3067\n\n\nGENETICS\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "Minor class splicing shapes the zebrafish transcriptome during development\nSebastian Markmiller a,b,1 , Nicole Cloonan c,2,3 , Rea M. Lardelli a,2,4 , Karen Doggett d,e , Maria-Cristina Keightley f , Yeliz Boglev a , Andrew J. Trotter a , Annie Y. Ng a,5 , Simon J. Wilkins c , Heather Verkade a,g , Elke A. Ober g,6 , Holly A. Field g , Sean M. Grimmond c , Graham J. Lieschke f , Didier Y. R. Stainier g,7 , and Joan K. Heath a,b,d,e,8\na Ludwig Institute for Cancer Research, Melbourne-Parkville Branch, Parkville, VIC 3050, Australia; b Department of Surgery, Royal Melbourne Hospital, University of Melbourne, Parkville, VIC 3050, Australia; c Queensland Centre for Medical Genomics, Institute for Molecular Bioscience, University of Queensland, St. Lucia, QLD 4072, Australia; d Walter and Eliza Hall Institute of Medical Research, Parkville, VIC 3052, Australia; e Department of Medical Biology, University of Melbourne, Parkville, VIC 3052, Australia; f Australian Regenerative Medicine Institute, Monash University, Clayton, VIC 3800, Australia; and g Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94158\nEdited* by Joan A. Steitz, Howard Hughes Medical Institute, New Haven, CT, and approved January 16, 2014 (received for review March 31, 2013)\nMinor class or U12-type splicing is a highly conserved process required to remove a minute fraction of introns from human pre- mRNAs. Defects in this splicing pathway have recently been linked to human disease, including a severe developmental disorder encom- passing brain and skeletal abnormalities known as Taybi-Linder syndrome or microcephalic osteodysplastic primordial dwarfism 1, and a hereditary intestinal polyposis condition, Peutz-Jeghers syn- drome. Although a key mechanism for regulating gene expression, the impact of impaired U12-type splicing on the transcriptome is unknown. Here, we describe a unique zebrafish mutant, caliban ( clbn ), with arrested development of the digestive organs caused by an ethylnitrosourea-induced recessive lethal point mutation in the rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ] gene. rnpc3 encodes the zebrafish ortholog of human RNPC3, also known as the U11/U12 di-snRNP 65-kDa protein, a unique compo- nent of the U12-type spliceosome. The biochemical impact of the mutation in clbn is the formation of aberrant U11- and U12-con- taining small nuclear ribonucleoproteins that impair the efficiency of U12-type splicing. Using RNA sequencing and microarrays, we show that multiple genes involved in various steps of mRNA pro- cessing, including transcription, splicing, and nuclear export are disrupted in clbn , either through intron retention or differential gene expression. Thus, clbn provides a useful and specific model of aberrant U12-type splicing in vivo. Analysis of its transcriptome reveals efficient mRNA processing as a critical process for the growth and proliferation of cells during vertebrate development.\nS plicing, the excision of introns from pre-mRNA, is an es- sential step in gene expression and a major source of complexity in the transcriptome (1). The process is catalyzed by highly dynamic complexes of small nuclear ribonucleoproteins (snRNPs) called spliceosomes (2). Not widely appreciated is the coexistence of two types of introns in most eukaryotic genomes. The vast majority, the major class or U2-type introns, are marked by GT and AG at their 5 ′ and 3 ′ ends, respectively. Minor class or U12-type introns, of which there are ∼ 700 in the human genome, were initially recog- nized by the presence of AT and AC in these positions, prompting their original name of AT-AC introns (3). However, we now know that most of these introns contain the same GT-AG termini found in U2-type introns (4, 5) and are instead distinguished from them by two highly conserved motifs: one adjacent to the 5 ′ splice site (ss) and one corresponding to the branch point sequence (BPS), close to the 3 ′ ss (6, 7). These introns also lack the 3 ′ polypyrimidine tract characteristic of U2-type introns. Minor class introns are excised by U12-type spliceosomes, which are analogous in function and similar in composition to U2-type spliceosomes; each comprise five small nuclear RNAs (snRNAs) and hun- dreds of associated proteins (6). Although the U5 snRNA is shared between the two complexes, the U12-type spliceosome contains four unique snRNAs (U11, U12, U4atac, and U6atac) (7 – 9) and at least seven unique proteins (10). Since its discovery two decades ago, the existence of this highly conserved, second\nsplicing pathway has been an intriguing aspect of gene ex- pression. Interestingly, U12-type introns are nonrandomly dis- tributed across the genome (11) and their removal is thought to be rate-limiting in the generation of mature mRNAs (12 – 14). Aberrant splicing is the basis of many human diseases and up to one-third of disease-causing mutations may lead to splicing defects (15). Although the majority of these defects occur in U2-type introns and splicing factors, perturbations in U12-type splicing have also been described. Most recently, mutations in the U12-type U4atac snRNA were found to underlie the human devel- opmental disorder Taybi-Linder syndrome (TALS)/microcephalic\n\n\nSignificance\nThe accurate removal of introns by pre-mRNA splicing is a crit- ical step in proper gene expression. Most eukaryotic genomes, from plant to human, contain a tiny subset of “ minor class ” introns with unique sequence elements that require their own splicing machinery. The significance of this second splicing pathway has intrigued RNA biologists for two decades, but its biological relevance was recently underscored when defects in the process were firmly linked to human disease. Here, we use a novel zebrafish mutant with defective minor class splicing to investigate how this pathway shapes the transcriptome during vertebrate development. We link its pleiotropic phenotype to widespread changes in gene expression that disrupt essential cellular pathways, including mRNA processing.\nAuthor contributions: S.M., R.M.L., and J.K.H. designed research; S.M., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., and H.V. performed research; N.C., H.V., E.A.O., H.A.F., S.M.G., and D.Y.R.S. contributed new reagents/analytic tools; S.M., N.C., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., G.J.L., and J.K.H. analyzed data; and S.M. and J.K.H. wrote the paper. The authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nFreely available online through the PNAS open access option.\nData deposition: The data reported in this paper have been deposited in the Gene Ex- pression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE53935 ). 1 Present address: Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla CA 92093.\n2 N.C. and R.M.L. contributed equally to this work. 3 Present address: QIMR Berghofer Medical Research Institute, Genomic Biology Labora- tory, Herston, QLD 4006, Australia.\n4 Present address: Division of Biology, University of California, San Diego, La Jolla, CA 92093.\n5 Present address: Institute of Medical Biology, Agency for Science, Technology and Re- search (A-STAR), Singapore 138648.\n6 Present address: The Danish Stem Cell Centre, University of Copenhagen, 2200 Copenhagen, Denmark.\n7 Present address: Department of Developmental Genetics, Max Planck Institute for Heart and Lung Research, D-61231 Bad Nauheim, Germany.\n8 To whom correspondance should be addressed. E-mail: joan.heath@wehi.edu.au. This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1305536111/-/DCSupplemental .\nosteodysplastic primordial dwarfism 1 (MOPD1) (16, 17). Other examples of diseases that can arise because of impaired splicing of U12-type introns include spinal muscular atrophy (18, 19) and Peutz-Jeghers syndrome (PJS), an intestinal polyposis syndrome that predisposes to cancer because of mutations in the tumor suppressor gene (TSG) LKB1 ( liver kinase B1 ) (20). The observation that impaired U12-type splicing can affect TSG expression has in- teresting implications for other TSGs containing U12-type introns, such as PTEN ( phosphatase and tensin homolog deleted on chromosome 10 ), where even a modest loss in PTEN function predisposes mice to cancer (21).\nTo specifically study the role of U12-type introns in gene ex- pression and development, we exploited a zebrafish mutant, caliban ( clbn ) , which carries a point mutation in rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ], encoding the zebrafish ortholog of human RNPC3, one of seven proteins unique to the U12-type spliceosome (22). We found aberrant minor class spliceosomal snRNPs in clbn extracts and used microarray and RNA sequencing (RNAseq) to show that impairment of U12-type splicing impacts broadly on gene expression, disrupting multiple genes involved in mRNA processing, transcription, splicing, and nuclear export. As a result, clbn mutants develop a pleiotropic phenotype with early lethality.\nResults The Intestine, Liver, and Pancreas Are Abnormal in clbn . clbn s846 was identified in a transgene-assisted ethylnitrosourea mutagenesis screen for zebrafish mutants with defective endodermal organ development (23). Abnormalities in the gross morphology of clbn larvae are first detectable at 108 h postfertilization (hpf) and mutants die between 7 and 10 d postfertilization (dpf). The head, eye, and lens are smaller, the swim bladder fails to inflate, and yolk absorption is delayed. At 120 hpf, the WT intestinal epi- thelium exhibits a columnar morphology and starts to elaborate folds; in contrast, the intestinal epithelium in clbn remains thin and unfolded (Fig. 1 A – C ). At 120 hpf, imaging of the digestive organs on the Tg(gutGFP) s854 (23) and Tg(fabp10:dsRed, ela3l: GFP) gz12 (24) backgrounds shows that the pancreas and liver are markedly smaller in clbn than in WT (Fig. 1 D and E ). clbn s846 Harbors a Mutation in rnpc3 . The clbn s846 locus was mapped to a region on chromosome 24 containing four genes, col11A1 , rnpc3 , prmt6 , and ntng1 ( Fig. S1 A ). RT-PCR amplification of rnpc3 from clbn s846 RNA generated aberrant rnpc3 isoforms (Fig. 2 A ), which were found by sequencing to correspond to the use of a de novo 3 ′ ss or retention of one or both of introns 13 and 14 (Fig. 2 B ). Sequencing of WT and clbn s846 genomic DNA re- vealed a T → A variation, creating a novel 3 ′ ss (AG) 10 nucleotides upstream of the canonical 3 ′ ss of intron 13 (Fig. 2 C ). The shift to the novel 3 ′ ss in clbn s846 is complete, as sequencing of the individual aberrant transcripts gave no evidence of a correctly spliced exon 13 – 14 junction in any of the products. All of the aberrant transcripts in clbn s846 contain premature stop codons after no more than six missense codons, making them likely targets for nonsense mediated decay ( Fig. S1 B ). We confirmed these findings using an in vivo minigene assay in which a WT or mutated minigene was injected into clutches containing WT and clbn s846 embryos. The T → A transition in the mutated ( clbn ) mini- gene led to a complete shift to the novel splice junction in both WT and clbn s846 embryos, as shown by a slightly shifted band in the agarose gel ( Fig. S1 C , lanes 1 and 3) and confirmed by sequencing. We provided firm evidence that rnpc3 is the mutated gene in clbn using rescue assays and a complementation test. Injection of one-cell stage clbn embryos with in vitro synthesized RNA encoding WT zebrafish Rnpc3 rescued the clbn phenotype (Fig. 2 D – G ). To perform a complementation test, we purchased a commercially available zebrafish line harboring a retroviral in- sertion in intron 1 of rnpc3 ( clbn ZM ) (Fig. 2 C ). We found that maternal deposits of correctly spliced, full-length rnpc3 mRNA in clbn ZM mutant embryos are largely depleted by 24 hpf ( Fig. S1 D ). clbn s846 and clbn ZM mutant larvae are morphologically\nindistinguishable at 120 hpf ( Fig. S1 F and G ). Upon crossing clbn s846 and clbn ZM heterozygotes, one-quarter of the offspring ex- hibit a phenotype indistinguishable from that of clbn s846 and clbn ZM mutants ( Fig. S1 H ). Taken together, these data show that rnpc3 is the mutated gene in clbn and suggest that both alleles are functionally null. We used whole mount in situ hybridization to show widespread rnpc 3 mRNA expression in WT embryos until 24 hpf ( Fig. S2 A and B ). Thereafter, the expression of rnpc3 becomes restricted to proliferating tissues, including the eye, lens, and digestive organs ( Fig. S2 C – F ). These tissues, which are abnormal in clbn , show reduced expression of rnpc3 in mutant larvae at 120 hpf ( Fig. S2 H ).\nclbn Contains Incompletely Spliced Transcripts. The zebrafish rnpc3 gene encodes a 505-aa nuclear protein with two RNA-recognition motifs (RRM) (25) and the same domain structure as its human ortholog (Fig. 3 A ). Whereas the excision of U2-type introns is initiated by the binding of individual U1 and U2 monosnRNPs, U12-type introns in human cells are recognized and bound by a preformed U11/U12 di-snRNP (26). This di-snRNP comprises U11 and U12 snRNAs, RNPC3/65K and two other proteins unique to the U12-type spliceosome, the U11/U12 59-kDa and 48-kDa proteins. Together, these components form a “ molecular bridge ” between the 5 ′ ss and BPS of U12-type introns (27, 28) (Fig. 3 B ). We used RT-PCR to assess the consequences of rnpc3 deficiency on the processing of mRNAs harboring U2- and U12-type introns in clbn and found that the U12-type introns in sms, mapk3 , and mapk12 were retained in clbn larvae (Fig. 3 C ), whereas the excision of the U2-type intron in mapk12 was unaffected.\nclbn Contains Aberrant Minor Spliceosomal snRNPs. To determine a biochemical basis for the observed specific defect in U12-type splicing, we sought to compare the size and conformation of zebrafish U11- and U12-containing snRNPs in WT and clbn larvae using glycerol gradient sedimentation followed by Northern blot analysis for U11 and U12 snRNAs. Extracts of WT larvae gave rise to U11 peaks in fractions 15 and 16 (Fig. 3 D , Upper ), whereas the U11 peak in clbn extracts was found two fractions closer to the top of the gradient, corresponding to a lighter U11 species (Fig. 3 D, Lower , and Fig. S3 A and B ). In contrast, glycerol gradient analyses of U12 snRNA sedimentation in clbn showed a dramatic shift of the predominant U12 signal toward a heavier particle than in WT (Fig. 3 E ). We further investigated these findings using native gel electrophoresis followed by Northern\nblotting for U11 and U12 snRNAs (Fig. 3 F ). Again, loss of Rnpc3 was found to disrupt the predominant U11 snRNPs and cause the accumulation of a slower-migrating U12-containing particle. This unexpected finding suggests a function for Rnpc3 beyond its known role in the U11/U12 di-snRNP, potentially at later stages of the splicing cycle. U12-containing complexes that might be stalled at any point after spliceosome activation would be expected to contain U6atac or U5 snRNAs (2). To test this theory we used native gel electrophoresis followed by Northern blotting for U12, U5, and U6atac snRNAs (Fig. 3 G ). To reduce potential nonspecific interactions and facilitate characterization of the stable complex accumulating in clbn embryos, heparin was added to the extracts before separation. Once again, we observed a strong signal corresponding to a heavier U12 snRNP complex\nFig. 2. The clbn phenotype is caused by a mutation in rnpc3 . ( A ) RT-PCR of full-length rnpc3 coding sequence amplifies a predominant 1.6-kb product in WT larvae (lane 4) and four aberrantly spliced transcripts in clbn s846 (lane 5). A retroviral insertion in clbn ZM abolishes generation of full-length rnpc3 mRNA (lane 6). Lanes 1 – 3 show – RT controls.\n( Lower ) Elongation factor alpha (elfa) control. Dashed lines link the cDNA products in lanes 4 and 5 to the corresponding transcripts, shown schemati- cally in B . The dark blue region denotes retention of 10 intronic nucleotides through use of a de novo 3 ′ ss. ( C ) Two noncomplementing clbn alleles. An as- terisk over T marks the position of the T → A varia- tion, 12 nucleotides upstream of exon 14 of rnpc3 , which causes the clbn s846 phenotype. The conserved 5 ′ ss and BPS in intron 14 (U12-type) of rnpc3 are underlined and in bold. clbn ZM carries a retroviral insertion in the first intron of rnpc3 . ( D – F ) Brightfield microscopy images of left lateral views of 5-dpf larvae derived from a clbn s846 heterozygous incross that were injected at the one- to two-cell stage with 380 pg of either WT rnpc3 mRNA or a mutant mRNA retaining introns 13 and 14, corresponding to the most abundant form of rnpc3 mRNA in clbn s846 (see B ). ( D ) No significant overexpression phenotype in WT larvae injected with WT rnpc3 mRNA. ( E ) Rescue of the mutant phenotype in genotyped clbn s846 homozygous mutant larvae injected with WT rnpc3 mRNA. ( F ) No rescue of the clbn mutant phenotype injected with mutant rnpc3 mRNA. ( G ) Noninjected control larvae from the same clutch show a near-perfect Mendelian ratio of WT to clbn larvae, whereas 96% of larvae injected with WT rnpc3 RNA appear WT at 5 dpf ( P &lt; 0.0001, Fisher ’ s exact test). There was no difference between uninjected and mutant rnpc3 -injected groups ( P = 0.9065). a, anus; e, eye; ib, intestinal bulb; sb, swim bladder; y, yolk. The brightfield images in D , E , and F were taken at the same magnification.\nFig. 3. Zebrafish Rnpc3 functions in the U12-type spliceosome. ( A ) Domain structure of zebrafish Rnpc3 and its human ortholog RNPC3, or U11/U12 di-snRNP 65-kDa protein ( www.uniprot.org , acces- sion no. Q96LT9). Red arrow in the C-terminal RRM denotes the approximate position of the premature stop codons in all clbn s846 isoforms. ( B ) Schematic of the human U11/U12 di-snRNP, with RNPC3 (65 K) bridging U12 snRNA and the 59 K protein (28). BPS, branch point sequence; “ A ” represents the branch point adenosine. ( C ) RT-PCR analysis of three dif- ferent transcripts ( sms , mapk3 , mapk12 ) shows retention of U12-type introns (lane 4, Top three panels) in clbn s846 (5 dpf), but not of a U2-type intron ( Bottom ). Lanes 1 and 2 show – RT controls. ( D and E ) Glycerol gradient/Northern analysis re- vealing differential sedimentation of U11 ( D ) and U12 ( E ) snRNA-containing snRNPs in WT and clbn ZM extracts. Direction of sedimentation is left to right.\nLane 1 contains total RNA. Full-length U11 or U12 signals were quantified using ImageQuant and ex- pressed as a percentage of the fraction with the highest intensity. U12 snRNA showed persistent specific degradation (asterisks) without affecting its migration profile. ( F ) Northern analysis of WT and clbn s846 extracts resolved on 4% (80:1) native poly-\nacrylamide gels and probed for U11 and U12 snRNAs. The predominant U11 snRNPs are disrupted in clbn compared with WT and the U12-containing particles are heavier and migrate more slowly (black arrowhead). ( G ) Northern analysis of WT and clbn s846 extracts resolved on native gels and probed for U5, U6atac, and U12 snRNAs shows retarded bands in clbn (red asterisks) compared with WT (blue asterisks). The same larval lysate was used in F and G . Data are representative of a total of 10 ( D and E ) and 5 ( F and G ) biological replicates.\nin clbn larvae from both alleles ( Fig. S3 C ), which appeared to comigrate with signals for both U5 and U6atac.\nTranscriptome Analysis Reveals a Specific U12-Type Splicing Defect in clbn . To determine the global impact of the clbn mutation on gene expression and U2- and U12-type splicing, we applied RNAseq to 108-hpf WT and clbn s846 mutants ( Table S1 ). RNAseq tags were aligned to the annotated zebrafish (Zv9) genome assembly ( Fig. S4 A ), which contains 606 manually curated U12-type introns ( SI Methods and Dataset S1 ). To normalize for the more common occurrence of nonunique sequences — such as repetitive elements — in introns compared with exons, we generated a zebra- fish uniqueome (29) and calculated all expression levels as reads per unique kilobase per million reads (RPuKM). Plotting ex- pression levels of all introns that satisfy minimum expression requirements (RPuKM &gt; 0.05) in both WT and clbn reveals a clear bias toward retention of U12-type introns in clbn (Fig. 4 A ). We found that a significantly higher fraction of U12-type introns (397 of 606 = 65.5%) show some degree of intron retention [intron retention coefficients (IRC) &gt; 0.05; defined in Fig. 4 B ] in WT larvae, compared with 49.6% of U2-type introns. This finding is consistent with U12-type introns being processed more slowly than their U2-type counterparts (12, 13). Remarkably, an addi- tional 148 of 606 (24.6%) of U12-type introns are detectable only in clbn (total = 90.1%), highlighting the specificity of the splicing defect. The defect in U12-type splicing in clbn is statistically sig- nificant (Fig. 4 C ); however, IRCs vary markedly between tran- scripts (Fig. 4 D ) and do not appear to be influenced by expression levels ( Fig. S4 B ), U12 intron length ( Fig. S4 C ), and AT-AC vs. GT-AG subtype ( Fig. S4 D ).\nActivation of Cryptic Splicing Occurs Rarely in clbn . A Peutz-Jeghers syndrome family has been described in which a mutation in the 5 ′ ss of the U12-type intron in LKB1 (20) leads to activation of cryptic splice sites and LKB1 haploinsufficiency. Activation of\ncryptic splice sites also occurs in human cells upon knockdown of the U11/U12 48-kDa protein with RNAi (30). To determine whether cryptic splice sites are activated in clbn , we sequenced the spliced transcripts obtained by RT-PCR for sms , mapk3 , and mapk12 mRNAs (Fig. 2 C ) and found only the annotated exon – exon junctions. We then used RT-PCR to amplify three more U12-type introns with low IRCs, namely fkbp3 , snrpe , and ppp2r2c . We found very low levels of U12-type intron retention in snrpe and ppp2r2c in clbn and no evidence of cryptic splicing for all three genes ( Fig. S4 E ). Furthermore, we also found no evidence of activation of cryptic splice sites in clbn when we aligned RNAseq reads against a custom-built library of exon junctions within 100 bp of the annotated splice junctions of all manually curated zebrafish U12-type introns ( Fig. S4 F and Dataset S2 ). Collectively these data show that impaired U12-type splicing in clbn is rarely circumvented by activation of cryptic splicing.\nRnpc3 Deficiency Impacts Broadly on Gene Expression. We combined datasets from microarray analysis and RNAseq to identify high- confidence differentially expressed genes (DEGs) between WT and clbn s846 larvae ( Datasets S3 – S5 ). Fold-changes for 203 DEGs (113 down, 90 up) were significantly correlated for the two methods ( Fig. S5 A ). As expected, rnpc3 mRNA levels were reduced, con- sistent with predicted nonsense mediated decay of rnpc3 transcripts carrying the clbn s846 mutation. Of the down-regulated genes, many exhibit tissue-specific expression in tissues that develop ab- normally in clbn , including the liver, pancreas, intestine, and lens ( Dataset S3 ). Consistent with a prior study in Drosophila (31), a substantial overrepresentation of U12-type intron-containing genes was observed among the DEGs, with a higher proportion of U12-type intron-containing genes in the down-regulated DEGs (35%; 40 of 113) than in the up-regulated set (11%; 10 of 90). Even so, about 90% of U12-type intron-containing genes were not differentially expressed ( Fig. S5 B ) and most of the DEGs do not contain U12-type introns; hence, Rnpc3 deficiency impacts broadly on gene expression and is not limited to U12-type intron- containing genes.\nExpression of mRNA Quality Control and Cell Cycle Genes Is Altered in clbn . U12-type introns are enriched in “ information processing ” genes encoding functions in DNA replication, repair, transcrip- tion, mRNA processing, and translation (11). In clbn , many such genes, particularly those encoding components of mRNA quality control pathways, such as exosome factors, are either differen- tially expressed ( dcp2 , edc4 , exosc1 , exosc5 , exosc9 ) or exhibit varying degrees of intron retention ( edc4 , exosc1 , exosc2 , exosc5 ) (Fig. 5, Fig. S5 C , and Table S2 ). Although the primary defect caused by the lack of Rnpc3 is in the processing of U12-type introns, the resulting widespread disruption to the expression of genes regulating RNA processing is likely to have a secondary impact on the expression of U2-type intron-containing genes, explaining why they feature so strongly in the list of DEGs. clbn mutants at 108 hpf also exhibit elevated expression of cell- cycle repressor genes, including prohibitin , cyclin G1 , e2f5 , and Δ 113p53 ( Fig. S5 D – F ). Δ 113p53 is a Tp53 target gene encoding an N-terminally truncated isoform of Tp53 thought to antagonize its proapoptotic activity (32).\n\n\nDiscussion\nU12-type splicing was a surprise discovery two decades ago, and its biological significance has remained something of an enigma. Here, we describe a unique vertebrate model system with a mu- tation in a core component that is specific to the U12-type spliceosome, and exploit it to gain new insights into the role of U12-type splicing in the regulation of gene expression during development and disease.\nRnpc3 Deficiency Leads to a Pleiotropic Phenotype in Developing Zebrafish. The loss of Rnpc3, either because of an intronic point mutation or a retroviral insertion into the rnpc3 locus, leads to a severe and pleiotropic phenotype in developing zebrafish\nFig. 4. Intron retention in clbn is specific to U12-type introns and highly variable. ( A ) Scatterplot of normalized intron expression derived by RNAseq for reference (Refseq) introns (gray dots) and U12-type introns (red dots). U12-type introns lie almost entirely above the midline representing equal expression, indicating enhanced intron retention in clbn . Intron expression is given in RPuKM. ( B ) To quantitate U12-type intron retention in individual transcripts, we defined an intron retention coefficient, which calculates in- tron retention relative to transcript level. ( C ) Comparison of the median log 2 of the IRCs for Refseq introns and U12-type introns between WT and clbn demonstrates that the clbn splicing defect is specific to U12-type introns. See also Fig. S4 . Statistical significance was determined by Student t test. ( D ) University of California at Santa Cruz genome browser views of three genes with high, intermediate, and low IRCs. ccdc43 shows a high IRC of 0.97, dcp2 an intermediate IRC of 0.41, and hsbp1 shows a low IRC of 0.05.\nlarvae, culminating in death of all larvae by 10 dpf. The most severely affected tissues are those that are highly proliferative between 48 and 96 hpf, including the digestive organs, eye, and lens. Our observation that Rnpc3 deficiency impacts broadly on gene expression, extending well beyond the small subset of genes containing U12-type introns, makes it difficult to pinpoint pre- cisely the molecular basis for the clbn phenotype. As yet, we do not know whether the abnormalities arise directly from pertur- bations in the expression of a small number of U12-type intron- containing genes, such as the dysregulated cell-cycle repressor genes we identified, or are the result of complex secondary effects emanating from a larger network of genes whose expression is affected by impaired U12-type splicing.\nComparison of the clbn Phenotype with MOPD. The severe human developmental disorder known as TALS/MOPD1 (16, 17, 33, 34) is caused by impaired U12-type splicing. Here, the affected infants inherit two recessive point mutations in U4atac snRNA and suffer extreme intrauterine growth retardation, severe mi- crocephaly, central nervous system abnormalities, dysmorphic facial features, skin and skeletal abnormalities, and early death (34). Although an impact on the growth of proliferating tissues is a common link in the zebrafish and human conditions, most MOPD1 features are not conspicuous in clbn . One possible ex- planation for this lack of concordance is that many of the severe defects observed in MOPD1 patients, such as microcephaly, most likely arise early in development. In zebrafish, adverse events early in embryogenesis are frequently circumvented by the availability of maternally derived WT mRNAs. In the particular case of clbn , we have shown that rnpc3 mRNA is maternally deposited, and it is reasonable to expect that this will also be the case for at least a proportion of the genes dependent on efficient minor class splicing, as well as their downstream effectors. If this hypothesis is correct, maternal-zygotic mutants of our clbn alleles would be expected to exhibit a more extensive phenotype that more closely mimics MOPD1.\nRnpc3 Deficiency Disrupts U11/U12 di-snRNP Assembly and Splicing. We examined the distribution of minor class snRNPs in rnpc3 - deficient zebrafish larvae to provide a molecular understanding\nof the specific U12-type splicing defects observed. Although the dynamic nature of snRNP interactions throughout the splicing cycle is relatively well characterized in mammalian cells, very little is known about this process in zebrafish. Previous studies have shown that the sedimentation profiles of zebrafish snRNPs in glycerol gradients differ in some aspects to those obtained from human cells (35, 36). Similarly, our glycerol gradient and native gel electrophoresis experiments reveal signals for U11 and U12 snRNPs that do not clearly correspond to a distribution into U11 and U12 mono-snRNPs and a U11/U12 di-snRNP, as de- scribed in human cells. Notwithstanding that, our data clearly demonstrate a specific role for Rnpc3 in maintaining the in- tegrity and proper distribution of U11 and U12 snRNPs. This result contrasts with our analysis of another zebrafish splicing mutant ( cph ), where a mutation in the shared core splicing factor, prpf8 , disrupts U2-type U4/U6/U5 tri-snRNP formation with negligible impact on U11/U12 snRNPs (35). In this study, the strong accumulation in clbn of a heavy, potentially stalled, complex containing U12, U5, and U6atac snRNAs, which does not seem to be a stable intermediate in WT, suggests a hitherto unsuspected role for Rnpc3 in spliceosome disassembly or recycling. Further studies are required to confirm this.\nRegulation of U12-Type Splicing. The results of our transcriptome analysis show that approximately two-thirds of U12-type intron- containing mRNAs exhibit some degree of U12-type intron retention even in developing WT zebrafish, much higher than the fraction of U2-type introns retained. At first sight, U12-type splicing appears to be a relatively inefficient process that may limit the full expression of U12-type intron-containing genes. However, recent findings suggest that regulation of U12-type splicing may be more complicated than previously thought. For example, p38MAPK-mediated up-regulation of the otherwise rate-limiting levels of U6atac in HeLa cells appears to con- stitute a sophisticated regulatory mechanism to facilitate the rapid mobilization of critical U12-type intron-containing transcripts in response to growth stimuli (14). Because many of these tran- scripts encode genes with functions in mRNA production, such an amplification of U12-type splicing activity is likely to generate a robust transcriptome-wide response.\nU12-Type Splicing and Disease. Recent breakthroughs that firmly link defects in U12-type splicing to human disease (16, 17, 19, 20) provide a strong impetus to explore its potential involvement in other pathological conditions. Interestingly, we have noted that U12-type introns are found in a number of cancer genes, including oncogenes, such as BRAF , C-RAF , and several mem- bers of the MAPK family. Extrapolating from our results in clbn , where the survival and integrity of rapidly proliferating cells is dependent on U12-type splicing, it is tantalizing to speculate that the efficiency of minor class splicing could influence the growth of tumors, particularly those that are dependent on the RAS- MAPK pathway (37, 38). In this regard, it is intriguing that p38MAPK has recently emerged as a key regulator of U12-type splicing activity (16), suggesting a potential regulatory mecha- nism to achieve efficient levels of U12-type splicing in tumors that are reliant on the MAPK pathway. Future investigations will determine whether efficient U12-type splicing is an im- portant contributor to the posttranscriptional RNA processing mechanisms governing gene expression in cancer.\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n3064 | www.pnas.org/cgi/doi/10.1073/pnas.1305536111\n\n\nMarkmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3065\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n1. Licatalosi DD, Darnell RB (2010) RNA processing and its regulation: Global insights into biological networks. Nat Rev Genet 11(1):75 – 87.\n\n\n2. Wahl MC, Will CL, Lührmann R (2009) The spliceosome: Design principles of a dynamic RNP machine. Cell 136(4):701 – 718.\n\n\n3. Jackson IJ (1991) A reappraisal of non-consensus mRNA splice sites. Nucleic Acids Res 19(14):3795 – 3798.\n\n\n6. Jurica MS, Moore MJ (2003) Pre-mRNA splicing: Awash in a sea of proteins. Mol Cell 12(1):5 – 14.\n\n\n20. Hastings ML, et al. (2005) An LKB1 AT-AC intron mutation causes Peutz-Jeghers syndrome via splicing at noncanonical cryptic splice sites. Nat Struct Mol Biol 12(1): 54 – 59.\n\n\n23. Ober EA, Verkade H, Field HA, Stainier DY (2006) Mesodermal Wnt2b signalling positively regulates liver specification. Nature 442(7103):688 – 691.\n\n\n41. Taylor J, Schenck I, Blankenberg D, Nekrutenko A (2007) Using galaxy to perform large-scale interactive data analyses. Curr Protoc Bioinformatics Chapter 10:5. Markmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3067\n\n\nGENETICS\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Minor class splicing shapes the zebrafish transcriptome during development\nSebastian Markmiller a,b,1 , Nicole Cloonan c,2,3 , Rea M. Lardelli a,2,4 , Karen Doggett d,e , Maria-Cristina Keightley f , Yeliz Boglev a , Andrew J. Trotter a , Annie Y. Ng a,5 , Simon J. Wilkins c , Heather Verkade a,g , Elke A. Ober g,6 , Holly A. Field g , Sean M. Grimmond c , Graham J. Lieschke f , Didier Y. R. Stainier g,7 , and Joan K. Heath a,b,d,e,8\na Ludwig Institute for Cancer Research, Melbourne-Parkville Branch, Parkville, VIC 3050, Australia; b Department of Surgery, Royal Melbourne Hospital, University of Melbourne, Parkville, VIC 3050, Australia; c Queensland Centre for Medical Genomics, Institute for Molecular Bioscience, University of Queensland, St. Lucia, QLD 4072, Australia; d Walter and Eliza Hall Institute of Medical Research, Parkville, VIC 3052, Australia; e Department of Medical Biology, University of Melbourne, Parkville, VIC 3052, Australia; f Australian Regenerative Medicine Institute, Monash University, Clayton, VIC 3800, Australia; and g Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94158\nEdited* by Joan A. Steitz, Howard Hughes Medical Institute, New Haven, CT, and approved January 16, 2014 (received for review March 31, 2013)\nMinor class or U12-type splicing is a highly conserved process required to remove a minute fraction of introns from human pre- mRNAs. Defects in this splicing pathway have recently been linked to human disease, including a severe developmental disorder encom- passing brain and skeletal abnormalities known as Taybi-Linder syndrome or microcephalic osteodysplastic primordial dwarfism 1, and a hereditary intestinal polyposis condition, Peutz-Jeghers syn- drome. Although a key mechanism for regulating gene expression, the impact of impaired U12-type splicing on the transcriptome is unknown. Here, we describe a unique zebrafish mutant, caliban ( clbn ), with arrested development of the digestive organs caused by an ethylnitrosourea-induced recessive lethal point mutation in the rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ] gene. rnpc3 encodes the zebrafish ortholog of human RNPC3, also known as the U11/U12 di-snRNP 65-kDa protein, a unique compo- nent of the U12-type spliceosome. The biochemical impact of the mutation in clbn is the formation of aberrant U11- and U12-con- taining small nuclear ribonucleoproteins that impair the efficiency of U12-type splicing. Using RNA sequencing and microarrays, we show that multiple genes involved in various steps of mRNA pro- cessing, including transcription, splicing, and nuclear export are disrupted in clbn , either through intron retention or differential gene expression. Thus, clbn provides a useful and specific model of aberrant U12-type splicing in vivo. Analysis of its transcriptome reveals efficient mRNA processing as a critical process for the growth and proliferation of cells during vertebrate development.\nS plicing, the excision of introns from pre-mRNA, is an es- sential step in gene expression and a major source of complexity in the transcriptome (1). The process is catalyzed by highly dynamic complexes of small nuclear ribonucleoproteins (snRNPs) called spliceosomes (2). Not widely appreciated is the coexistence of two types of introns in most eukaryotic genomes. The vast majority, the major class or U2-type introns, are marked by GT and AG at their 5 ′ and 3 ′ ends, respectively. Minor class or U12-type introns, of which there are ∼ 700 in the human genome, were initially recog- nized by the presence of AT and AC in these positions, prompting their original name of AT-AC introns (3). However, we now know that most of these introns contain the same GT-AG termini found in U2-type introns (4, 5) and are instead distinguished from them by two highly conserved motifs: one adjacent to the 5 ′ splice site (ss) and one corresponding to the branch point sequence (BPS), close to the 3 ′ ss (6, 7). These introns also lack the 3 ′ polypyrimidine tract characteristic of U2-type introns. Minor class introns are excised by U12-type spliceosomes, which are analogous in function and similar in composition to U2-type spliceosomes; each comprise five small nuclear RNAs (snRNAs) and hun- dreds of associated proteins (6). Although the U5 snRNA is shared between the two complexes, the U12-type spliceosome contains four unique snRNAs (U11, U12, U4atac, and U6atac) (7 – 9) and at least seven unique proteins (10). Since its discovery two decades ago, the existence of this highly conserved, second\nsplicing pathway has been an intriguing aspect of gene ex- pression. Interestingly, U12-type introns are nonrandomly dis- tributed across the genome (11) and their removal is thought to be rate-limiting in the generation of mature mRNAs (12 – 14). Aberrant splicing is the basis of many human diseases and up to one-third of disease-causing mutations may lead to splicing defects (15). Although the majority of these defects occur in U2-type introns and splicing factors, perturbations in U12-type splicing have also been described. Most recently, mutations in the U12-type U4atac snRNA were found to underlie the human devel- opmental disorder Taybi-Linder syndrome (TALS)/microcephalic\n\n\nSignificance\nThe accurate removal of introns by pre-mRNA splicing is a crit- ical step in proper gene expression. Most eukaryotic genomes, from plant to human, contain a tiny subset of “ minor class ” introns with unique sequence elements that require their own splicing machinery. The significance of this second splicing pathway has intrigued RNA biologists for two decades, but its biological relevance was recently underscored when defects in the process were firmly linked to human disease. Here, we use a novel zebrafish mutant with defective minor class splicing to investigate how this pathway shapes the transcriptome during vertebrate development. We link its pleiotropic phenotype to widespread changes in gene expression that disrupt essential cellular pathways, including mRNA processing.\nAuthor contributions: S.M., R.M.L., and J.K.H. designed research; S.M., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., and H.V. performed research; N.C., H.V., E.A.O., H.A.F., S.M.G., and D.Y.R.S. contributed new reagents/analytic tools; S.M., N.C., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., G.J.L., and J.K.H. analyzed data; and S.M. and J.K.H. wrote the paper. The authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nFreely available online through the PNAS open access option.\nData deposition: The data reported in this paper have been deposited in the Gene Ex- pression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE53935 ). 1 Present address: Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla CA 92093.\n2 N.C. and R.M.L. contributed equally to this work. 3 Present address: QIMR Berghofer Medical Research Institute, Genomic Biology Labora- tory, Herston, QLD 4006, Australia.\n4 Present address: Division of Biology, University of California, San Diego, La Jolla, CA 92093.\n5 Present address: Institute of Medical Biology, Agency for Science, Technology and Re- search (A-STAR), Singapore 138648.\n6 Present address: The Danish Stem Cell Centre, University of Copenhagen, 2200 Copenhagen, Denmark.\n7 Present address: Department of Developmental Genetics, Max Planck Institute for Heart and Lung Research, D-61231 Bad Nauheim, Germany.\n8 To whom correspondance should be addressed. E-mail: joan.heath@wehi.edu.au. This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1305536111/-/DCSupplemental .\nosteodysplastic primordial dwarfism 1 (MOPD1) (16, 17). Other examples of diseases that can arise because of impaired splicing of U12-type introns include spinal muscular atrophy (18, 19) and Peutz-Jeghers syndrome (PJS), an intestinal polyposis syndrome that predisposes to cancer because of mutations in the tumor suppressor gene (TSG) LKB1 ( liver kinase B1 ) (20). The observation that impaired U12-type splicing can affect TSG expression has in- teresting implications for other TSGs containing U12-type introns, such as PTEN ( phosphatase and tensin homolog deleted on chromosome 10 ), where even a modest loss in PTEN function predisposes mice to cancer (21).\nTo specifically study the role of U12-type introns in gene ex- pression and development, we exploited a zebrafish mutant, caliban ( clbn ) , which carries a point mutation in rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ], encoding the zebrafish ortholog of human RNPC3, one of seven proteins unique to the U12-type spliceosome (22). We found aberrant minor class spliceosomal snRNPs in clbn extracts and used microarray and RNA sequencing (RNAseq) to show that impairment of U12-type splicing impacts broadly on gene expression, disrupting multiple genes involved in mRNA processing, transcription, splicing, and nuclear export. As a result, clbn mutants develop a pleiotropic phenotype with early lethality.\nResults The Intestine, Liver, and Pancreas Are Abnormal in clbn . clbn s846 was identified in a transgene-assisted ethylnitrosourea mutagenesis screen for zebrafish mutants with defective endodermal organ development (23). Abnormalities in the gross morphology of clbn larvae are first detectable at 108 h postfertilization (hpf) and mutants die between 7 and 10 d postfertilization (dpf). The head, eye, and lens are smaller, the swim bladder fails to inflate, and yolk absorption is delayed. At 120 hpf, the WT intestinal epi- thelium exhibits a columnar morphology and starts to elaborate folds; in contrast, the intestinal epithelium in clbn remains thin and unfolded (Fig. 1 A – C ). At 120 hpf, imaging of the digestive organs on the Tg(gutGFP) s854 (23) and Tg(fabp10:dsRed, ela3l: GFP) gz12 (24) backgrounds shows that the pancreas and liver are markedly smaller in clbn than in WT (Fig. 1 D and E ). clbn s846 Harbors a Mutation in rnpc3 . The clbn s846 locus was mapped to a region on chromosome 24 containing four genes, col11A1 , rnpc3 , prmt6 , and ntng1 ( Fig. S1 A ). RT-PCR amplification of rnpc3 from clbn s846 RNA generated aberrant rnpc3 isoforms (Fig. 2 A ), which were found by sequencing to correspond to the use of a de novo 3 ′ ss or retention of one or both of introns 13 and 14 (Fig. 2 B ). Sequencing of WT and clbn s846 genomic DNA re- vealed a T → A variation, creating a novel 3 ′ ss (AG) 10 nucleotides upstream of the canonical 3 ′ ss of intron 13 (Fig. 2 C ). The shift to the novel 3 ′ ss in clbn s846 is complete, as sequencing of the individual aberrant transcripts gave no evidence of a correctly spliced exon 13 – 14 junction in any of the products. All of the aberrant transcripts in clbn s846 contain premature stop codons after no more than six missense codons, making them likely targets for nonsense mediated decay ( Fig. S1 B ). We confirmed these findings using an in vivo minigene assay in which a WT or mutated minigene was injected into clutches containing WT and clbn s846 embryos. The T → A transition in the mutated ( clbn ) mini- gene led to a complete shift to the novel splice junction in both WT and clbn s846 embryos, as shown by a slightly shifted band in the agarose gel ( Fig. S1 C , lanes 1 and 3) and confirmed by sequencing. We provided firm evidence that rnpc3 is the mutated gene in clbn using rescue assays and a complementation test. Injection of one-cell stage clbn embryos with in vitro synthesized RNA encoding WT zebrafish Rnpc3 rescued the clbn phenotype (Fig. 2 D – G ). To perform a complementation test, we purchased a commercially available zebrafish line harboring a retroviral in- sertion in intron 1 of rnpc3 ( clbn ZM ) (Fig. 2 C ). We found that maternal deposits of correctly spliced, full-length rnpc3 mRNA in clbn ZM mutant embryos are largely depleted by 24 hpf ( Fig. S1 D ). clbn s846 and clbn ZM mutant larvae are morphologically\nindistinguishable at 120 hpf ( Fig. S1 F and G ). Upon crossing clbn s846 and clbn ZM heterozygotes, one-quarter of the offspring ex- hibit a phenotype indistinguishable from that of clbn s846 and clbn ZM mutants ( Fig. S1 H ). Taken together, these data show that rnpc3 is the mutated gene in clbn and suggest that both alleles are functionally null. We used whole mount in situ hybridization to show widespread rnpc 3 mRNA expression in WT embryos until 24 hpf ( Fig. S2 A and B ). Thereafter, the expression of rnpc3 becomes restricted to proliferating tissues, including the eye, lens, and digestive organs ( Fig. S2 C – F ). These tissues, which are abnormal in clbn , show reduced expression of rnpc3 in mutant larvae at 120 hpf ( Fig. S2 H ).\nclbn Contains Incompletely Spliced Transcripts. The zebrafish rnpc3 gene encodes a 505-aa nuclear protein with two RNA-recognition motifs (RRM) (25) and the same domain structure as its human ortholog (Fig. 3 A ). Whereas the excision of U2-type introns is initiated by the binding of individual U1 and U2 monosnRNPs, U12-type introns in human cells are recognized and bound by a preformed U11/U12 di-snRNP (26). This di-snRNP comprises U11 and U12 snRNAs, RNPC3/65K and two other proteins unique to the U12-type spliceosome, the U11/U12 59-kDa and 48-kDa proteins. Together, these components form a “ molecular bridge ” between the 5 ′ ss and BPS of U12-type introns (27, 28) (Fig. 3 B ). We used RT-PCR to assess the consequences of rnpc3 deficiency on the processing of mRNAs harboring U2- and U12-type introns in clbn and found that the U12-type introns in sms, mapk3 , and mapk12 were retained in clbn larvae (Fig. 3 C ), whereas the excision of the U2-type intron in mapk12 was unaffected.\nclbn Contains Aberrant Minor Spliceosomal snRNPs. To determine a biochemical basis for the observed specific defect in U12-type splicing, we sought to compare the size and conformation of zebrafish U11- and U12-containing snRNPs in WT and clbn larvae using glycerol gradient sedimentation followed by Northern blot analysis for U11 and U12 snRNAs. Extracts of WT larvae gave rise to U11 peaks in fractions 15 and 16 (Fig. 3 D , Upper ), whereas the U11 peak in clbn extracts was found two fractions closer to the top of the gradient, corresponding to a lighter U11 species (Fig. 3 D, Lower , and Fig. S3 A and B ). In contrast, glycerol gradient analyses of U12 snRNA sedimentation in clbn showed a dramatic shift of the predominant U12 signal toward a heavier particle than in WT (Fig. 3 E ). We further investigated these findings using native gel electrophoresis followed by Northern\nblotting for U11 and U12 snRNAs (Fig. 3 F ). Again, loss of Rnpc3 was found to disrupt the predominant U11 snRNPs and cause the accumulation of a slower-migrating U12-containing particle. This unexpected finding suggests a function for Rnpc3 beyond its known role in the U11/U12 di-snRNP, potentially at later stages of the splicing cycle. U12-containing complexes that might be stalled at any point after spliceosome activation would be expected to contain U6atac or U5 snRNAs (2). To test this theory we used native gel electrophoresis followed by Northern blotting for U12, U5, and U6atac snRNAs (Fig. 3 G ). To reduce potential nonspecific interactions and facilitate characterization of the stable complex accumulating in clbn embryos, heparin was added to the extracts before separation. Once again, we observed a strong signal corresponding to a heavier U12 snRNP complex\nFig. 2. The clbn phenotype is caused by a mutation in rnpc3 . ( A ) RT-PCR of full-length rnpc3 coding sequence amplifies a predominant 1.6-kb product in WT larvae (lane 4) and four aberrantly spliced transcripts in clbn s846 (lane 5). A retroviral insertion in clbn ZM abolishes generation of full-length rnpc3 mRNA (lane 6). Lanes 1 – 3 show – RT controls.\n( Lower ) Elongation factor alpha (elfa) control. Dashed lines link the cDNA products in lanes 4 and 5 to the corresponding transcripts, shown schemati- cally in B . The dark blue region denotes retention of 10 intronic nucleotides through use of a de novo 3 ′ ss. ( C ) Two noncomplementing clbn alleles. An as- terisk over T marks the position of the T → A varia- tion, 12 nucleotides upstream of exon 14 of rnpc3 , which causes the clbn s846 phenotype. The conserved 5 ′ ss and BPS in intron 14 (U12-type) of rnpc3 are underlined and in bold. clbn ZM carries a retroviral insertion in the first intron of rnpc3 . ( D – F ) Brightfield microscopy images of left lateral views of 5-dpf larvae derived from a clbn s846 heterozygous incross that were injected at the one- to two-cell stage with 380 pg of either WT rnpc3 mRNA or a mutant mRNA retaining introns 13 and 14, corresponding to the most abundant form of rnpc3 mRNA in clbn s846 (see B ). ( D ) No significant overexpression phenotype in WT larvae injected with WT rnpc3 mRNA. ( E ) Rescue of the mutant phenotype in genotyped clbn s846 homozygous mutant larvae injected with WT rnpc3 mRNA. ( F ) No rescue of the clbn mutant phenotype injected with mutant rnpc3 mRNA. ( G ) Noninjected control larvae from the same clutch show a near-perfect Mendelian ratio of WT to clbn larvae, whereas 96% of larvae injected with WT rnpc3 RNA appear WT at 5 dpf ( P &lt; 0.0001, Fisher ’ s exact test). There was no difference between uninjected and mutant rnpc3 -injected groups ( P = 0.9065). a, anus; e, eye; ib, intestinal bulb; sb, swim bladder; y, yolk. The brightfield images in D , E , and F were taken at the same magnification.\nFig. 3. Zebrafish Rnpc3 functions in the U12-type spliceosome. ( A ) Domain structure of zebrafish Rnpc3 and its human ortholog RNPC3, or U11/U12 di-snRNP 65-kDa protein ( www.uniprot.org , acces- sion no. Q96LT9). Red arrow in the C-terminal RRM denotes the approximate position of the premature stop codons in all clbn s846 isoforms. ( B ) Schematic of the human U11/U12 di-snRNP, with RNPC3 (65 K) bridging U12 snRNA and the 59 K protein (28). BPS, branch point sequence; “ A ” represents the branch point adenosine. ( C ) RT-PCR analysis of three dif- ferent transcripts ( sms , mapk3 , mapk12 ) shows retention of U12-type introns (lane 4, Top three panels) in clbn s846 (5 dpf), but not of a U2-type intron ( Bottom ). Lanes 1 and 2 show – RT controls. ( D and E ) Glycerol gradient/Northern analysis re- vealing differential sedimentation of U11 ( D ) and U12 ( E ) snRNA-containing snRNPs in WT and clbn ZM extracts. Direction of sedimentation is left to right.\nLane 1 contains total RNA. Full-length U11 or U12 signals were quantified using ImageQuant and ex- pressed as a percentage of the fraction with the highest intensity. U12 snRNA showed persistent specific degradation (asterisks) without affecting its migration profile. ( F ) Northern analysis of WT and clbn s846 extracts resolved on 4% (80:1) native poly-\nacrylamide gels and probed for U11 and U12 snRNAs. The predominant U11 snRNPs are disrupted in clbn compared with WT and the U12-containing particles are heavier and migrate more slowly (black arrowhead). ( G ) Northern analysis of WT and clbn s846 extracts resolved on native gels and probed for U5, U6atac, and U12 snRNAs shows retarded bands in clbn (red asterisks) compared with WT (blue asterisks). The same larval lysate was used in F and G . Data are representative of a total of 10 ( D and E ) and 5 ( F and G ) biological replicates.\nin clbn larvae from both alleles ( Fig. S3 C ), which appeared to comigrate with signals for both U5 and U6atac.\nTranscriptome Analysis Reveals a Specific U12-Type Splicing Defect in clbn . To determine the global impact of the clbn mutation on gene expression and U2- and U12-type splicing, we applied RNAseq to 108-hpf WT and clbn s846 mutants ( Table S1 ). RNAseq tags were aligned to the annotated zebrafish (Zv9) genome assembly ( Fig. S4 A ), which contains 606 manually curated U12-type introns ( SI Methods and Dataset S1 ). To normalize for the more common occurrence of nonunique sequences — such as repetitive elements — in introns compared with exons, we generated a zebra- fish uniqueome (29) and calculated all expression levels as reads per unique kilobase per million reads (RPuKM). Plotting ex- pression levels of all introns that satisfy minimum expression requirements (RPuKM &gt; 0.05) in both WT and clbn reveals a clear bias toward retention of U12-type introns in clbn (Fig. 4 A ). We found that a significantly higher fraction of U12-type introns (397 of 606 = 65.5%) show some degree of intron retention [intron retention coefficients (IRC) &gt; 0.05; defined in Fig. 4 B ] in WT larvae, compared with 49.6% of U2-type introns. This finding is consistent with U12-type introns being processed more slowly than their U2-type counterparts (12, 13). Remarkably, an addi- tional 148 of 606 (24.6%) of U12-type introns are detectable only in clbn (total = 90.1%), highlighting the specificity of the splicing defect. The defect in U12-type splicing in clbn is statistically sig- nificant (Fig. 4 C ); however, IRCs vary markedly between tran- scripts (Fig. 4 D ) and do not appear to be influenced by expression levels ( Fig. S4 B ), U12 intron length ( Fig. S4 C ), and AT-AC vs. GT-AG subtype ( Fig. S4 D ).\nActivation of Cryptic Splicing Occurs Rarely in clbn . A Peutz-Jeghers syndrome family has been described in which a mutation in the 5 ′ ss of the U12-type intron in LKB1 (20) leads to activation of cryptic splice sites and LKB1 haploinsufficiency. Activation of\ncryptic splice sites also occurs in human cells upon knockdown of the U11/U12 48-kDa protein with RNAi (30). To determine whether cryptic splice sites are activated in clbn , we sequenced the spliced transcripts obtained by RT-PCR for sms , mapk3 , and mapk12 mRNAs (Fig. 2 C ) and found only the annotated exon – exon junctions. We then used RT-PCR to amplify three more U12-type introns with low IRCs, namely fkbp3 , snrpe , and ppp2r2c . We found very low levels of U12-type intron retention in snrpe and ppp2r2c in clbn and no evidence of cryptic splicing for all three genes ( Fig. S4 E ). Furthermore, we also found no evidence of activation of cryptic splice sites in clbn when we aligned RNAseq reads against a custom-built library of exon junctions within 100 bp of the annotated splice junctions of all manually curated zebrafish U12-type introns ( Fig. S4 F and Dataset S2 ). Collectively these data show that impaired U12-type splicing in clbn is rarely circumvented by activation of cryptic splicing.\nRnpc3 Deficiency Impacts Broadly on Gene Expression. We combined datasets from microarray analysis and RNAseq to identify high- confidence differentially expressed genes (DEGs) between WT and clbn s846 larvae ( Datasets S3 – S5 ). Fold-changes for 203 DEGs (113 down, 90 up) were significantly correlated for the two methods ( Fig. S5 A ). As expected, rnpc3 mRNA levels were reduced, con- sistent with predicted nonsense mediated decay of rnpc3 transcripts carrying the clbn s846 mutation. Of the down-regulated genes, many exhibit tissue-specific expression in tissues that develop ab- normally in clbn , including the liver, pancreas, intestine, and lens ( Dataset S3 ). Consistent with a prior study in Drosophila (31), a substantial overrepresentation of U12-type intron-containing genes was observed among the DEGs, with a higher proportion of U12-type intron-containing genes in the down-regulated DEGs (35%; 40 of 113) than in the up-regulated set (11%; 10 of 90). Even so, about 90% of U12-type intron-containing genes were not differentially expressed ( Fig. S5 B ) and most of the DEGs do not contain U12-type introns; hence, Rnpc3 deficiency impacts broadly on gene expression and is not limited to U12-type intron- containing genes.\nExpression of mRNA Quality Control and Cell Cycle Genes Is Altered in clbn . U12-type introns are enriched in “ information processing ” genes encoding functions in DNA replication, repair, transcrip- tion, mRNA processing, and translation (11). In clbn , many such genes, particularly those encoding components of mRNA quality control pathways, such as exosome factors, are either differen- tially expressed ( dcp2 , edc4 , exosc1 , exosc5 , exosc9 ) or exhibit varying degrees of intron retention ( edc4 , exosc1 , exosc2 , exosc5 ) (Fig. 5, Fig. S5 C , and Table S2 ). Although the primary defect caused by the lack of Rnpc3 is in the processing of U12-type introns, the resulting widespread disruption to the expression of genes regulating RNA processing is likely to have a secondary impact on the expression of U2-type intron-containing genes, explaining why they feature so strongly in the list of DEGs. clbn mutants at 108 hpf also exhibit elevated expression of cell- cycle repressor genes, including prohibitin , cyclin G1 , e2f5 , and Δ 113p53 ( Fig. S5 D – F ). Δ 113p53 is a Tp53 target gene encoding an N-terminally truncated isoform of Tp53 thought to antagonize its proapoptotic activity (32).\n\n\nDiscussion\nU12-type splicing was a surprise discovery two decades ago, and its biological significance has remained something of an enigma. Here, we describe a unique vertebrate model system with a mu- tation in a core component that is specific to the U12-type spliceosome, and exploit it to gain new insights into the role of U12-type splicing in the regulation of gene expression during development and disease.\nRnpc3 Deficiency Leads to a Pleiotropic Phenotype in Developing Zebrafish. The loss of Rnpc3, either because of an intronic point mutation or a retroviral insertion into the rnpc3 locus, leads to a severe and pleiotropic phenotype in developing zebrafish\nFig. 4. Intron retention in clbn is specific to U12-type introns and highly variable. ( A ) Scatterplot of normalized intron expression derived by RNAseq for reference (Refseq) introns (gray dots) and U12-type introns (red dots). U12-type introns lie almost entirely above the midline representing equal expression, indicating enhanced intron retention in clbn . Intron expression is given in RPuKM. ( B ) To quantitate U12-type intron retention in individual transcripts, we defined an intron retention coefficient, which calculates in- tron retention relative to transcript level. ( C ) Comparison of the median log 2 of the IRCs for Refseq introns and U12-type introns between WT and clbn demonstrates that the clbn splicing defect is specific to U12-type introns. See also Fig. S4 . Statistical significance was determined by Student t test. ( D ) University of California at Santa Cruz genome browser views of three genes with high, intermediate, and low IRCs. ccdc43 shows a high IRC of 0.97, dcp2 an intermediate IRC of 0.41, and hsbp1 shows a low IRC of 0.05.\nlarvae, culminating in death of all larvae by 10 dpf. The most severely affected tissues are those that are highly proliferative between 48 and 96 hpf, including the digestive organs, eye, and lens. Our observation that Rnpc3 deficiency impacts broadly on gene expression, extending well beyond the small subset of genes containing U12-type introns, makes it difficult to pinpoint pre- cisely the molecular basis for the clbn phenotype. As yet, we do not know whether the abnormalities arise directly from pertur- bations in the expression of a small number of U12-type intron- containing genes, such as the dysregulated cell-cycle repressor genes we identified, or are the result of complex secondary effects emanating from a larger network of genes whose expression is affected by impaired U12-type splicing.\nComparison of the clbn Phenotype with MOPD. The severe human developmental disorder known as TALS/MOPD1 (16, 17, 33, 34) is caused by impaired U12-type splicing. Here, the affected infants inherit two recessive point mutations in U4atac snRNA and suffer extreme intrauterine growth retardation, severe mi- crocephaly, central nervous system abnormalities, dysmorphic facial features, skin and skeletal abnormalities, and early death (34). Although an impact on the growth of proliferating tissues is a common link in the zebrafish and human conditions, most MOPD1 features are not conspicuous in clbn . One possible ex- planation for this lack of concordance is that many of the severe defects observed in MOPD1 patients, such as microcephaly, most likely arise early in development. In zebrafish, adverse events early in embryogenesis are frequently circumvented by the availability of maternally derived WT mRNAs. In the particular case of clbn , we have shown that rnpc3 mRNA is maternally deposited, and it is reasonable to expect that this will also be the case for at least a proportion of the genes dependent on efficient minor class splicing, as well as their downstream effectors. If this hypothesis is correct, maternal-zygotic mutants of our clbn alleles would be expected to exhibit a more extensive phenotype that more closely mimics MOPD1.\nRnpc3 Deficiency Disrupts U11/U12 di-snRNP Assembly and Splicing. We examined the distribution of minor class snRNPs in rnpc3 - deficient zebrafish larvae to provide a molecular understanding\nof the specific U12-type splicing defects observed. Although the dynamic nature of snRNP interactions throughout the splicing cycle is relatively well characterized in mammalian cells, very little is known about this process in zebrafish. Previous studies have shown that the sedimentation profiles of zebrafish snRNPs in glycerol gradients differ in some aspects to those obtained from human cells (35, 36). Similarly, our glycerol gradient and native gel electrophoresis experiments reveal signals for U11 and U12 snRNPs that do not clearly correspond to a distribution into U11 and U12 mono-snRNPs and a U11/U12 di-snRNP, as de- scribed in human cells. Notwithstanding that, our data clearly demonstrate a specific role for Rnpc3 in maintaining the in- tegrity and proper distribution of U11 and U12 snRNPs. This result contrasts with our analysis of another zebrafish splicing mutant ( cph ), where a mutation in the shared core splicing factor, prpf8 , disrupts U2-type U4/U6/U5 tri-snRNP formation with negligible impact on U11/U12 snRNPs (35). In this study, the strong accumulation in clbn of a heavy, potentially stalled, complex containing U12, U5, and U6atac snRNAs, which does not seem to be a stable intermediate in WT, suggests a hitherto unsuspected role for Rnpc3 in spliceosome disassembly or recycling. Further studies are required to confirm this.\nRegulation of U12-Type Splicing. The results of our transcriptome analysis show that approximately two-thirds of U12-type intron- containing mRNAs exhibit some degree of U12-type intron retention even in developing WT zebrafish, much higher than the fraction of U2-type introns retained. At first sight, U12-type splicing appears to be a relatively inefficient process that may limit the full expression of U12-type intron-containing genes. However, recent findings suggest that regulation of U12-type splicing may be more complicated than previously thought. For example, p38MAPK-mediated up-regulation of the otherwise rate-limiting levels of U6atac in HeLa cells appears to con- stitute a sophisticated regulatory mechanism to facilitate the rapid mobilization of critical U12-type intron-containing transcripts in response to growth stimuli (14). Because many of these tran- scripts encode genes with functions in mRNA production, such an amplification of U12-type splicing activity is likely to generate a robust transcriptome-wide response.\nU12-Type Splicing and Disease. Recent breakthroughs that firmly link defects in U12-type splicing to human disease (16, 17, 19, 20) provide a strong impetus to explore its potential involvement in other pathological conditions. Interestingly, we have noted that U12-type introns are found in a number of cancer genes, including oncogenes, such as BRAF , C-RAF , and several mem- bers of the MAPK family. Extrapolating from our results in clbn , where the survival and integrity of rapidly proliferating cells is dependent on U12-type splicing, it is tantalizing to speculate that the efficiency of minor class splicing could influence the growth of tumors, particularly those that are dependent on the RAS- MAPK pathway (37, 38). In this regard, it is intriguing that p38MAPK has recently emerged as a key regulator of U12-type splicing activity (16), suggesting a potential regulatory mecha- nism to achieve efficient levels of U12-type splicing in tumors that are reliant on the MAPK pathway. Future investigations will determine whether efficient U12-type splicing is an im- portant contributor to the posttranscriptional RNA processing mechanisms governing gene expression in cancer.\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n3064 | www.pnas.org/cgi/doi/10.1073/pnas.1305536111\n\n\nMarkmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3065\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n1. Licatalosi DD, Darnell RB (2010) RNA processing and its regulation: Global insights into biological networks. Nat Rev Genet 11(1):75 – 87.\n\n\n2. Wahl MC, Will CL, Lührmann R (2009) The spliceosome: Design principles of a dynamic RNP machine. Cell 136(4):701 – 718.\n\n\n3. Jackson IJ (1991) A reappraisal of non-consensus mRNA splice sites. Nucleic Acids Res 19(14):3795 – 3798.\n\n\n6. Jurica MS, Moore MJ (2003) Pre-mRNA splicing: Awash in a sea of proteins. Mol Cell 12(1):5 – 14.\n\n\n20. Hastings ML, et al. (2005) An LKB1 AT-AC intron mutation causes Peutz-Jeghers syndrome via splicing at noncanonical cryptic splice sites. Nat Struct Mol Biol 12(1): 54 – 59.\n\n\n23. Ober EA, Verkade H, Field HA, Stainier DY (2006) Mesodermal Wnt2b signalling positively regulates liver specification. Nature 442(7103):688 – 691.\n\n\n41. Taylor J, Schenck I, Blankenberg D, Nekrutenko A (2007) Using galaxy to perform large-scale interactive data analyses. Curr Protoc Bioinformatics Chapter 10:5. Markmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3067\n\n\nGENETICS\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''Minor class splicing shapes the zebrafish transcriptome during development\nSebastian Markmiller a,b,1 , Nicole Cloonan c,2,3 , Rea M. Lardelli a,2,4 , Karen Doggett d,e , Maria-Cristina Keightley f , Yeliz Boglev a , Andrew J. Trotter a , Annie Y. Ng a,5 , Simon J. Wilkins c , Heather Verkade a,g , Elke A. Ober g,6 , Holly A. Field g , Sean M. Grimmond c , Graham J. Lieschke f , Didier Y. R. Stainier g,7 , and Joan K. Heath a,b,d,e,8\na Ludwig Institute for Cancer Research, Melbourne-Parkville Branch, Parkville, VIC 3050, Australia; b Department of Surgery, Royal Melbourne Hospital, University of Melbourne, Parkville, VIC 3050, Australia; c Queensland Centre for Medical Genomics, Institute for Molecular Bioscience, University of Queensland, St. Lucia, QLD 4072, Australia; d Walter and Eliza Hall Institute of Medical Research, Parkville, VIC 3052, Australia; e Department of Medical Biology, University of Melbourne, Parkville, VIC 3052, Australia; f Australian Regenerative Medicine Institute, Monash University, Clayton, VIC 3800, Australia; and g Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94158\nEdited* by Joan A. Steitz, Howard Hughes Medical Institute, New Haven, CT, and approved January 16, 2014 (received for review March 31, 2013)\nMinor class or U12-type splicing is a highly conserved process required to remove a minute fraction of introns from human pre- mRNAs. Defects in this splicing pathway have recently been linked to human disease, including a severe developmental disorder encom- passing brain and skeletal abnormalities known as Taybi-Linder syndrome or microcephalic osteodysplastic primordial dwarfism 1, and a hereditary intestinal polyposis condition, Peutz-Jeghers syn- drome. Although a key mechanism for regulating gene expression, the impact of impaired U12-type splicing on the transcriptome is unknown. Here, we describe a unique zebrafish mutant, caliban ( clbn ), with arrested development of the digestive organs caused by an ethylnitrosourea-induced recessive lethal point mutation in the rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ] gene. rnpc3 encodes the zebrafish ortholog of human RNPC3, also known as the U11/U12 di-snRNP 65-kDa protein, a unique compo- nent of the U12-type spliceosome. The biochemical impact of the mutation in clbn is the formation of aberrant U11- and U12-con- taining small nuclear ribonucleoproteins that impair the efficiency of U12-type splicing. Using RNA sequencing and microarrays, we show that multiple genes involved in various steps of mRNA pro- cessing, including transcription, splicing, and nuclear export are disrupted in clbn , either through intron retention or differential gene expression. Thus, clbn provides a useful and specific model of aberrant U12-type splicing in vivo. Analysis of its transcriptome reveals efficient mRNA processing as a critical process for the growth and proliferation of cells during vertebrate development.\nS plicing, the excision of introns from pre-mRNA, is an es- sential step in gene expression and a major source of complexity in the transcriptome (1). The process is catalyzed by highly dynamic complexes of small nuclear ribonucleoproteins (snRNPs) called spliceosomes (2). Not widely appreciated is the coexistence of two types of introns in most eukaryotic genomes. The vast majority, the major class or U2-type introns, are marked by GT and AG at their 5 ′ and 3 ′ ends, respectively. Minor class or U12-type introns, of which there are ∼ 700 in the human genome, were initially recog- nized by the presence of AT and AC in these positions, prompting their original name of AT-AC introns (3). However, we now know that most of these introns contain the same GT-AG termini found in U2-type introns (4, 5) and are instead distinguished from them by two highly conserved motifs: one adjacent to the 5 ′ splice site (ss) and one corresponding to the branch point sequence (BPS), close to the 3 ′ ss (6, 7). These introns also lack the 3 ′ polypyrimidine tract characteristic of U2-type introns. Minor class introns are excised by U12-type spliceosomes, which are analogous in function and similar in composition to U2-type spliceosomes; each comprise five small nuclear RNAs (snRNAs) and hun- dreds of associated proteins (6). Although the U5 snRNA is shared between the two complexes, the U12-type spliceosome contains four unique snRNAs (U11, U12, U4atac, and U6atac) (7 – 9) and at least seven unique proteins (10). Since its discovery two decades ago, the existence of this highly conserved, second\nsplicing pathway has been an intriguing aspect of gene ex- pression. Interestingly, U12-type introns are nonrandomly dis- tributed across the genome (11) and their removal is thought to be rate-limiting in the generation of mature mRNAs (12 – 14). Aberrant splicing is the basis of many human diseases and up to one-third of disease-causing mutations may lead to splicing defects (15). Although the majority of these defects occur in U2-type introns and splicing factors, perturbations in U12-type splicing have also been described. Most recently, mutations in the U12-type U4atac snRNA were found to underlie the human devel- opmental disorder Taybi-Linder syndrome (TALS)/microcephalic\n\n\nSignificance\nThe accurate removal of introns by pre-mRNA splicing is a crit- ical step in proper gene expression. Most eukaryotic genomes, from plant to human, contain a tiny subset of “ minor class ” introns with unique sequence elements that require their own splicing machinery. The significance of this second splicing pathway has intrigued RNA biologists for two decades, but its biological relevance was recently underscored when defects in the process were firmly linked to human disease. Here, we use a novel zebrafish mutant with defective minor class splicing to investigate how this pathway shapes the transcriptome during vertebrate development. We link its pleiotropic phenotype to widespread changes in gene expression that disrupt essential cellular pathways, including mRNA processing.\nAuthor contributions: S.M., R.M.L., and J.K.H. designed research; S.M., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., and H.V. performed research; N.C., H.V., E.A.O., H.A.F., S.M.G., and D.Y.R.S. contributed new reagents/analytic tools; S.M., N.C., R.M.L., K.D., M.-C.K., Y.B., A.J.T., A.Y.N., S.J.W., G.J.L., and J.K.H. analyzed data; and S.M. and J.K.H. wrote the paper. The authors declare no conflict of interest.\n*This Direct Submission article had a prearranged editor.\nFreely available online through the PNAS open access option.\nData deposition: The data reported in this paper have been deposited in the Gene Ex- pression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE53935 ). 1 Present address: Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla CA 92093.\n2 N.C. and R.M.L. contributed equally to this work. 3 Present address: QIMR Berghofer Medical Research Institute, Genomic Biology Labora- tory, Herston, QLD 4006, Australia.\n4 Present address: Division of Biology, University of California, San Diego, La Jolla, CA 92093.\n5 Present address: Institute of Medical Biology, Agency for Science, Technology and Re- search (A-STAR), Singapore 138648.\n6 Present address: The Danish Stem Cell Centre, University of Copenhagen, 2200 Copenhagen, Denmark.\n7 Present address: Department of Developmental Genetics, Max Planck Institute for Heart and Lung Research, D-61231 Bad Nauheim, Germany.\n8 To whom correspondance should be addressed. E-mail: joan.heath@wehi.edu.au. This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. 1073/pnas.1305536111/-/DCSupplemental .\nosteodysplastic primordial dwarfism 1 (MOPD1) (16, 17). Other examples of diseases that can arise because of impaired splicing of U12-type introns include spinal muscular atrophy (18, 19) and Peutz-Jeghers syndrome (PJS), an intestinal polyposis syndrome that predisposes to cancer because of mutations in the tumor suppressor gene (TSG) LKB1 ( liver kinase B1 ) (20). The observation that impaired U12-type splicing can affect TSG expression has in- teresting implications for other TSGs containing U12-type introns, such as PTEN ( phosphatase and tensin homolog deleted on chromosome 10 ), where even a modest loss in PTEN function predisposes mice to cancer (21).\nTo specifically study the role of U12-type introns in gene ex- pression and development, we exploited a zebrafish mutant, caliban ( clbn ) , which carries a point mutation in rnpc3 [ RNA-binding region (RNP1, RRM) containing 3 ], encoding the zebrafish ortholog of human RNPC3, one of seven proteins unique to the U12-type spliceosome (22). We found aberrant minor class spliceosomal snRNPs in clbn extracts and used microarray and RNA sequencing (RNAseq) to show that impairment of U12-type splicing impacts broadly on gene expression, disrupting multiple genes involved in mRNA processing, transcription, splicing, and nuclear export. As a result, clbn mutants develop a pleiotropic phenotype with early lethality.\nResults The Intestine, Liver, and Pancreas Are Abnormal in clbn . clbn s846 was identified in a transgene-assisted ethylnitrosourea mutagenesis screen for zebrafish mutants with defective endodermal organ development (23). Abnormalities in the gross morphology of clbn larvae are first detectable at 108 h postfertilization (hpf) and mutants die between 7 and 10 d postfertilization (dpf). The head, eye, and lens are smaller, the swim bladder fails to inflate, and yolk absorption is delayed. At 120 hpf, the WT intestinal epi- thelium exhibits a columnar morphology and starts to elaborate folds; in contrast, the intestinal epithelium in clbn remains thin and unfolded (Fig. 1 A – C ). At 120 hpf, imaging of the digestive organs on the Tg(gutGFP) s854 (23) and Tg(fabp10:dsRed, ela3l: GFP) gz12 (24) backgrounds shows that the pancreas and liver are markedly smaller in clbn than in WT (Fig. 1 D and E ). clbn s846 Harbors a Mutation in rnpc3 . The clbn s846 locus was mapped to a region on chromosome 24 containing four genes, col11A1 , rnpc3 , prmt6 , and ntng1 ( Fig. S1 A ). RT-PCR amplification of rnpc3 from clbn s846 RNA generated aberrant rnpc3 isoforms (Fig. 2 A ), which were found by sequencing to correspond to the use of a de novo 3 ′ ss or retention of one or both of introns 13 and 14 (Fig. 2 B ). Sequencing of WT and clbn s846 genomic DNA re- vealed a T → A variation, creating a novel 3 ′ ss (AG) 10 nucleotides upstream of the canonical 3 ′ ss of intron 13 (Fig. 2 C ). The shift to the novel 3 ′ ss in clbn s846 is complete, as sequencing of the individual aberrant transcripts gave no evidence of a correctly spliced exon 13 – 14 junction in any of the products. All of the aberrant transcripts in clbn s846 contain premature stop codons after no more than six missense codons, making them likely targets for nonsense mediated decay ( Fig. S1 B ). We confirmed these findings using an in vivo minigene assay in which a WT or mutated minigene was injected into clutches containing WT and clbn s846 embryos. The T → A transition in the mutated ( clbn ) mini- gene led to a complete shift to the novel splice junction in both WT and clbn s846 embryos, as shown by a slightly shifted band in the agarose gel ( Fig. S1 C , lanes 1 and 3) and confirmed by sequencing. We provided firm evidence that rnpc3 is the mutated gene in clbn using rescue assays and a complementation test. Injection of one-cell stage clbn embryos with in vitro synthesized RNA encoding WT zebrafish Rnpc3 rescued the clbn phenotype (Fig. 2 D – G ). To perform a complementation test, we purchased a commercially available zebrafish line harboring a retroviral in- sertion in intron 1 of rnpc3 ( clbn ZM ) (Fig. 2 C ). We found that maternal deposits of correctly spliced, full-length rnpc3 mRNA in clbn ZM mutant embryos are largely depleted by 24 hpf ( Fig. S1 D ). clbn s846 and clbn ZM mutant larvae are morphologically\nindistinguishable at 120 hpf ( Fig. S1 F and G ). Upon crossing clbn s846 and clbn ZM heterozygotes, one-quarter of the offspring ex- hibit a phenotype indistinguishable from that of clbn s846 and clbn ZM mutants ( Fig. S1 H ). Taken together, these data show that rnpc3 is the mutated gene in clbn and suggest that both alleles are functionally null. We used whole mount in situ hybridization to show widespread rnpc 3 mRNA expression in WT embryos until 24 hpf ( Fig. S2 A and B ). Thereafter, the expression of rnpc3 becomes restricted to proliferating tissues, including the eye, lens, and digestive organs ( Fig. S2 C – F ). These tissues, which are abnormal in clbn , show reduced expression of rnpc3 in mutant larvae at 120 hpf ( Fig. S2 H ).\nclbn Contains Incompletely Spliced Transcripts. The zebrafish rnpc3 gene encodes a 505-aa nuclear protein with two RNA-recognition motifs (RRM) (25) and the same domain structure as its human ortholog (Fig. 3 A ). Whereas the excision of U2-type introns is initiated by the binding of individual U1 and U2 monosnRNPs, U12-type introns in human cells are recognized and bound by a preformed U11/U12 di-snRNP (26). This di-snRNP comprises U11 and U12 snRNAs, RNPC3/65K and two other proteins unique to the U12-type spliceosome, the U11/U12 59-kDa and 48-kDa proteins. Together, these components form a “ molecular bridge ” between the 5 ′ ss and BPS of U12-type introns (27, 28) (Fig. 3 B ). We used RT-PCR to assess the consequences of rnpc3 deficiency on the processing of mRNAs harboring U2- and U12-type introns in clbn and found that the U12-type introns in sms, mapk3 , and mapk12 were retained in clbn larvae (Fig. 3 C ), whereas the excision of the U2-type intron in mapk12 was unaffected.\nclbn Contains Aberrant Minor Spliceosomal snRNPs. To determine a biochemical basis for the observed specific defect in U12-type splicing, we sought to compare the size and conformation of zebrafish U11- and U12-containing snRNPs in WT and clbn larvae using glycerol gradient sedimentation followed by Northern blot analysis for U11 and U12 snRNAs. Extracts of WT larvae gave rise to U11 peaks in fractions 15 and 16 (Fig. 3 D , Upper ), whereas the U11 peak in clbn extracts was found two fractions closer to the top of the gradient, corresponding to a lighter U11 species (Fig. 3 D, Lower , and Fig. S3 A and B ). In contrast, glycerol gradient analyses of U12 snRNA sedimentation in clbn showed a dramatic shift of the predominant U12 signal toward a heavier particle than in WT (Fig. 3 E ). We further investigated these findings using native gel electrophoresis followed by Northern\nblotting for U11 and U12 snRNAs (Fig. 3 F ). Again, loss of Rnpc3 was found to disrupt the predominant U11 snRNPs and cause the accumulation of a slower-migrating U12-containing particle. This unexpected finding suggests a function for Rnpc3 beyond its known role in the U11/U12 di-snRNP, potentially at later stages of the splicing cycle. U12-containing complexes that might be stalled at any point after spliceosome activation would be expected to contain U6atac or U5 snRNAs (2). To test this theory we used native gel electrophoresis followed by Northern blotting for U12, U5, and U6atac snRNAs (Fig. 3 G ). To reduce potential nonspecific interactions and facilitate characterization of the stable complex accumulating in clbn embryos, heparin was added to the extracts before separation. Once again, we observed a strong signal corresponding to a heavier U12 snRNP complex\nFig. 2. The clbn phenotype is caused by a mutation in rnpc3 . ( A ) RT-PCR of full-length rnpc3 coding sequence amplifies a predominant 1.6-kb product in WT larvae (lane 4) and four aberrantly spliced transcripts in clbn s846 (lane 5). A retroviral insertion in clbn ZM abolishes generation of full-length rnpc3 mRNA (lane 6). Lanes 1 – 3 show – RT controls.\n( Lower ) Elongation factor alpha (elfa) control. Dashed lines link the cDNA products in lanes 4 and 5 to the corresponding transcripts, shown schemati- cally in B . The dark blue region denotes retention of 10 intronic nucleotides through use of a de novo 3 ′ ss. ( C ) Two noncomplementing clbn alleles. An as- terisk over T marks the position of the T → A varia- tion, 12 nucleotides upstream of exon 14 of rnpc3 , which causes the clbn s846 phenotype. The conserved 5 ′ ss and BPS in intron 14 (U12-type) of rnpc3 are underlined and in bold. clbn ZM carries a retroviral insertion in the first intron of rnpc3 . ( D – F ) Brightfield microscopy images of left lateral views of 5-dpf larvae derived from a clbn s846 heterozygous incross that were injected at the one- to two-cell stage with 380 pg of either WT rnpc3 mRNA or a mutant mRNA retaining introns 13 and 14, corresponding to the most abundant form of rnpc3 mRNA in clbn s846 (see B ). ( D ) No significant overexpression phenotype in WT larvae injected with WT rnpc3 mRNA. ( E ) Rescue of the mutant phenotype in genotyped clbn s846 homozygous mutant larvae injected with WT rnpc3 mRNA. ( F ) No rescue of the clbn mutant phenotype injected with mutant rnpc3 mRNA. ( G ) Noninjected control larvae from the same clutch show a near-perfect Mendelian ratio of WT to clbn larvae, whereas 96% of larvae injected with WT rnpc3 RNA appear WT at 5 dpf ( P &lt; 0.0001, Fisher ’ s exact test). There was no difference between uninjected and mutant rnpc3 -injected groups ( P = 0.9065). a, anus; e, eye; ib, intestinal bulb; sb, swim bladder; y, yolk. The brightfield images in D , E , and F were taken at the same magnification.\nFig. 3. Zebrafish Rnpc3 functions in the U12-type spliceosome. ( A ) Domain structure of zebrafish Rnpc3 and its human ortholog RNPC3, or U11/U12 di-snRNP 65-kDa protein ( www.uniprot.org , acces- sion no. Q96LT9). Red arrow in the C-terminal RRM denotes the approximate position of the premature stop codons in all clbn s846 isoforms. ( B ) Schematic of the human U11/U12 di-snRNP, with RNPC3 (65 K) bridging U12 snRNA and the 59 K protein (28). BPS, branch point sequence; “ A ” represents the branch point adenosine. ( C ) RT-PCR analysis of three dif- ferent transcripts ( sms , mapk3 , mapk12 ) shows retention of U12-type introns (lane 4, Top three panels) in clbn s846 (5 dpf), but not of a U2-type intron ( Bottom ). Lanes 1 and 2 show – RT controls. ( D and E ) Glycerol gradient/Northern analysis re- vealing differential sedimentation of U11 ( D ) and U12 ( E ) snRNA-containing snRNPs in WT and clbn ZM extracts. Direction of sedimentation is left to right.\nLane 1 contains total RNA. Full-length U11 or U12 signals were quantified using ImageQuant and ex- pressed as a percentage of the fraction with the highest intensity. U12 snRNA showed persistent specific degradation (asterisks) without affecting its migration profile. ( F ) Northern analysis of WT and clbn s846 extracts resolved on 4% (80:1) native poly-\nacrylamide gels and probed for U11 and U12 snRNAs. The predominant U11 snRNPs are disrupted in clbn compared with WT and the U12-containing particles are heavier and migrate more slowly (black arrowhead). ( G ) Northern analysis of WT and clbn s846 extracts resolved on native gels and probed for U5, U6atac, and U12 snRNAs shows retarded bands in clbn (red asterisks) compared with WT (blue asterisks). The same larval lysate was used in F and G . Data are representative of a total of 10 ( D and E ) and 5 ( F and G ) biological replicates.\nin clbn larvae from both alleles ( Fig. S3 C ), which appeared to comigrate with signals for both U5 and U6atac.\nTranscriptome Analysis Reveals a Specific U12-Type Splicing Defect in clbn . To determine the global impact of the clbn mutation on gene expression and U2- and U12-type splicing, we applied RNAseq to 108-hpf WT and clbn s846 mutants ( Table S1 ). RNAseq tags were aligned to the annotated zebrafish (Zv9) genome assembly ( Fig. S4 A ), which contains 606 manually curated U12-type introns ( SI Methods and Dataset S1 ). To normalize for the more common occurrence of nonunique sequences — such as repetitive elements — in introns compared with exons, we generated a zebra- fish uniqueome (29) and calculated all expression levels as reads per unique kilobase per million reads (RPuKM). Plotting ex- pression levels of all introns that satisfy minimum expression requirements (RPuKM &gt; 0.05) in both WT and clbn reveals a clear bias toward retention of U12-type introns in clbn (Fig. 4 A ). We found that a significantly higher fraction of U12-type introns (397 of 606 = 65.5%) show some degree of intron retention [intron retention coefficients (IRC) &gt; 0.05; defined in Fig. 4 B ] in WT larvae, compared with 49.6% of U2-type introns. This finding is consistent with U12-type introns being processed more slowly than their U2-type counterparts (12, 13). Remarkably, an addi- tional 148 of 606 (24.6%) of U12-type introns are detectable only in clbn (total = 90.1%), highlighting the specificity of the splicing defect. The defect in U12-type splicing in clbn is statistically sig- nificant (Fig. 4 C ); however, IRCs vary markedly between tran- scripts (Fig. 4 D ) and do not appear to be influenced by expression levels ( Fig. S4 B ), U12 intron length ( Fig. S4 C ), and AT-AC vs. GT-AG subtype ( Fig. S4 D ).\nActivation of Cryptic Splicing Occurs Rarely in clbn . A Peutz-Jeghers syndrome family has been described in which a mutation in the 5 ′ ss of the U12-type intron in LKB1 (20) leads to activation of cryptic splice sites and LKB1 haploinsufficiency. Activation of\ncryptic splice sites also occurs in human cells upon knockdown of the U11/U12 48-kDa protein with RNAi (30). To determine whether cryptic splice sites are activated in clbn , we sequenced the spliced transcripts obtained by RT-PCR for sms , mapk3 , and mapk12 mRNAs (Fig. 2 C ) and found only the annotated exon – exon junctions. We then used RT-PCR to amplify three more U12-type introns with low IRCs, namely fkbp3 , snrpe , and ppp2r2c . We found very low levels of U12-type intron retention in snrpe and ppp2r2c in clbn and no evidence of cryptic splicing for all three genes ( Fig. S4 E ). Furthermore, we also found no evidence of activation of cryptic splice sites in clbn when we aligned RNAseq reads against a custom-built library of exon junctions within 100 bp of the annotated splice junctions of all manually curated zebrafish U12-type introns ( Fig. S4 F and Dataset S2 ). Collectively these data show that impaired U12-type splicing in clbn is rarely circumvented by activation of cryptic splicing.\nRnpc3 Deficiency Impacts Broadly on Gene Expression. We combined datasets from microarray analysis and RNAseq to identify high- confidence differentially expressed genes (DEGs) between WT and clbn s846 larvae ( Datasets S3 – S5 ). Fold-changes for 203 DEGs (113 down, 90 up) were significantly correlated for the two methods ( Fig. S5 A ). As expected, rnpc3 mRNA levels were reduced, con- sistent with predicted nonsense mediated decay of rnpc3 transcripts carrying the clbn s846 mutation. Of the down-regulated genes, many exhibit tissue-specific expression in tissues that develop ab- normally in clbn , including the liver, pancreas, intestine, and lens ( Dataset S3 ). Consistent with a prior study in Drosophila (31), a substantial overrepresentation of U12-type intron-containing genes was observed among the DEGs, with a higher proportion of U12-type intron-containing genes in the down-regulated DEGs (35%; 40 of 113) than in the up-regulated set (11%; 10 of 90). Even so, about 90% of U12-type intron-containing genes were not differentially expressed ( Fig. S5 B ) and most of the DEGs do not contain U12-type introns; hence, Rnpc3 deficiency impacts broadly on gene expression and is not limited to U12-type intron- containing genes.\nExpression of mRNA Quality Control and Cell Cycle Genes Is Altered in clbn . U12-type introns are enriched in “ information processing ” genes encoding functions in DNA replication, repair, transcrip- tion, mRNA processing, and translation (11). In clbn , many such genes, particularly those encoding components of mRNA quality control pathways, such as exosome factors, are either differen- tially expressed ( dcp2 , edc4 , exosc1 , exosc5 , exosc9 ) or exhibit varying degrees of intron retention ( edc4 , exosc1 , exosc2 , exosc5 ) (Fig. 5, Fig. S5 C , and Table S2 ). Although the primary defect caused by the lack of Rnpc3 is in the processing of U12-type introns, the resulting widespread disruption to the expression of genes regulating RNA processing is likely to have a secondary impact on the expression of U2-type intron-containing genes, explaining why they feature so strongly in the list of DEGs. clbn mutants at 108 hpf also exhibit elevated expression of cell- cycle repressor genes, including prohibitin , cyclin G1 , e2f5 , and Δ 113p53 ( Fig. S5 D – F ). Δ 113p53 is a Tp53 target gene encoding an N-terminally truncated isoform of Tp53 thought to antagonize its proapoptotic activity (32).\n\n\nDiscussion\nU12-type splicing was a surprise discovery two decades ago, and its biological significance has remained something of an enigma. Here, we describe a unique vertebrate model system with a mu- tation in a core component that is specific to the U12-type spliceosome, and exploit it to gain new insights into the role of U12-type splicing in the regulation of gene expression during development and disease.\nRnpc3 Deficiency Leads to a Pleiotropic Phenotype in Developing Zebrafish. The loss of Rnpc3, either because of an intronic point mutation or a retroviral insertion into the rnpc3 locus, leads to a severe and pleiotropic phenotype in developing zebrafish\nFig. 4. Intron retention in clbn is specific to U12-type introns and highly variable. ( A ) Scatterplot of normalized intron expression derived by RNAseq for reference (Refseq) introns (gray dots) and U12-type introns (red dots). U12-type introns lie almost entirely above the midline representing equal expression, indicating enhanced intron retention in clbn . Intron expression is given in RPuKM. ( B ) To quantitate U12-type intron retention in individual transcripts, we defined an intron retention coefficient, which calculates in- tron retention relative to transcript level. ( C ) Comparison of the median log 2 of the IRCs for Refseq introns and U12-type introns between WT and clbn demonstrates that the clbn splicing defect is specific to U12-type introns. See also Fig. S4 . Statistical significance was determined by Student t test. ( D ) University of California at Santa Cruz genome browser views of three genes with high, intermediate, and low IRCs. ccdc43 shows a high IRC of 0.97, dcp2 an intermediate IRC of 0.41, and hsbp1 shows a low IRC of 0.05.\nlarvae, culminating in death of all larvae by 10 dpf. The most severely affected tissues are those that are highly proliferative between 48 and 96 hpf, including the digestive organs, eye, and lens. Our observation that Rnpc3 deficiency impacts broadly on gene expression, extending well beyond the small subset of genes containing U12-type introns, makes it difficult to pinpoint pre- cisely the molecular basis for the clbn phenotype. As yet, we do not know whether the abnormalities arise directly from pertur- bations in the expression of a small number of U12-type intron- containing genes, such as the dysregulated cell-cycle repressor genes we identified, or are the result of complex secondary effects emanating from a larger network of genes whose expression is affected by impaired U12-type splicing.\nComparison of the clbn Phenotype with MOPD. The severe human developmental disorder known as TALS/MOPD1 (16, 17, 33, 34) is caused by impaired U12-type splicing. Here, the affected infants inherit two recessive point mutations in U4atac snRNA and suffer extreme intrauterine growth retardation, severe mi- crocephaly, central nervous system abnormalities, dysmorphic facial features, skin and skeletal abnormalities, and early death (34). Although an impact on the growth of proliferating tissues is a common link in the zebrafish and human conditions, most MOPD1 features are not conspicuous in clbn . One possible ex- planation for this lack of concordance is that many of the severe defects observed in MOPD1 patients, such as microcephaly, most likely arise early in development. In zebrafish, adverse events early in embryogenesis are frequently circumvented by the availability of maternally derived WT mRNAs. In the particular case of clbn , we have shown that rnpc3 mRNA is maternally deposited, and it is reasonable to expect that this will also be the case for at least a proportion of the genes dependent on efficient minor class splicing, as well as their downstream effectors. If this hypothesis is correct, maternal-zygotic mutants of our clbn alleles would be expected to exhibit a more extensive phenotype that more closely mimics MOPD1.\nRnpc3 Deficiency Disrupts U11/U12 di-snRNP Assembly and Splicing. We examined the distribution of minor class snRNPs in rnpc3 - deficient zebrafish larvae to provide a molecular understanding\nof the specific U12-type splicing defects observed. Although the dynamic nature of snRNP interactions throughout the splicing cycle is relatively well characterized in mammalian cells, very little is known about this process in zebrafish. Previous studies have shown that the sedimentation profiles of zebrafish snRNPs in glycerol gradients differ in some aspects to those obtained from human cells (35, 36). Similarly, our glycerol gradient and native gel electrophoresis experiments reveal signals for U11 and U12 snRNPs that do not clearly correspond to a distribution into U11 and U12 mono-snRNPs and a U11/U12 di-snRNP, as de- scribed in human cells. Notwithstanding that, our data clearly demonstrate a specific role for Rnpc3 in maintaining the in- tegrity and proper distribution of U11 and U12 snRNPs. This result contrasts with our analysis of another zebrafish splicing mutant ( cph ), where a mutation in the shared core splicing factor, prpf8 , disrupts U2-type U4/U6/U5 tri-snRNP formation with negligible impact on U11/U12 snRNPs (35). In this study, the strong accumulation in clbn of a heavy, potentially stalled, complex containing U12, U5, and U6atac snRNAs, which does not seem to be a stable intermediate in WT, suggests a hitherto unsuspected role for Rnpc3 in spliceosome disassembly or recycling. Further studies are required to confirm this.\nRegulation of U12-Type Splicing. The results of our transcriptome analysis show that approximately two-thirds of U12-type intron- containing mRNAs exhibit some degree of U12-type intron retention even in developing WT zebrafish, much higher than the fraction of U2-type introns retained. At first sight, U12-type splicing appears to be a relatively inefficient process that may limit the full expression of U12-type intron-containing genes. However, recent findings suggest that regulation of U12-type splicing may be more complicated than previously thought. For example, p38MAPK-mediated up-regulation of the otherwise rate-limiting levels of U6atac in HeLa cells appears to con- stitute a sophisticated regulatory mechanism to facilitate the rapid mobilization of critical U12-type intron-containing transcripts in response to growth stimuli (14). Because many of these tran- scripts encode genes with functions in mRNA production, such an amplification of U12-type splicing activity is likely to generate a robust transcriptome-wide response.\nU12-Type Splicing and Disease. Recent breakthroughs that firmly link defects in U12-type splicing to human disease (16, 17, 19, 20) provide a strong impetus to explore its potential involvement in other pathological conditions. Interestingly, we have noted that U12-type introns are found in a number of cancer genes, including oncogenes, such as BRAF , C-RAF , and several mem- bers of the MAPK family. Extrapolating from our results in clbn , where the survival and integrity of rapidly proliferating cells is dependent on U12-type splicing, it is tantalizing to speculate that the efficiency of minor class splicing could influence the growth of tumors, particularly those that are dependent on the RAS- MAPK pathway (37, 38). In this regard, it is intriguing that p38MAPK has recently emerged as a key regulator of U12-type splicing activity (16), suggesting a potential regulatory mecha- nism to achieve efficient levels of U12-type splicing in tumors that are reliant on the MAPK pathway. Future investigations will determine whether efficient U12-type splicing is an im- portant contributor to the posttranscriptional RNA processing mechanisms governing gene expression in cancer.\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n3064 | www.pnas.org/cgi/doi/10.1073/pnas.1305536111\n\n\nMarkmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3065\n\n\nGENETICS Downloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\nDownloaded from https://www.pnas.org by Gully Burns on March 16, 2024 from IP address 73.231.74.21.\n\n\n1. Licatalosi DD, Darnell RB (2010) RNA processing and its regulation: Global insights into biological networks. Nat Rev Genet 11(1):75 – 87.\n\n\n2. Wahl MC, Will CL, Lührmann R (2009) The spliceosome: Design principles of a dynamic RNP machine. Cell 136(4):701 – 718.\n\n\n3. Jackson IJ (1991) A reappraisal of non-consensus mRNA splice sites. Nucleic Acids Res 19(14):3795 – 3798.\n\n\n6. Jurica MS, Moore MJ (2003) Pre-mRNA splicing: Awash in a sea of proteins. Mol Cell 12(1):5 – 14.\n\n\n20. Hastings ML, et al. (2005) An LKB1 AT-AC intron mutation causes Peutz-Jeghers syndrome via splicing at noncanonical cryptic splice sites. Nat Struct Mol Biol 12(1): 54 – 59.\n\n\n23. Ober EA, Verkade H, Field HA, Stainier DY (2006) Mesodermal Wnt2b signalling positively regulates liver specification. Nature 442(7103):688 – 691.\n\n\n41. Taylor J, Schenck I, Blankenberg D, Nekrutenko A (2007) Using galaxy to perform large-scale interactive data analyses. Curr Protoc Bioinformatics Chapter 10:5. Markmiller et al. PNAS | February 25, 2014 | vol. 111 | no. 8 | 3067\n\n\nGENETICS\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [9.18s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"digestive organs, eye, lens, liver, pancreas, intestinal epithelium\",\n  \"tissue_name_structured\": [\"eye\", \"liver\", \"pancreas\", \"gut\", \"Other\"],\n  \"developmental_stage_text\": \"larvae\",\n  \"developmental_stage_coarse\": [\"larval\"],\n  \"developmental_stage_fine\": [\"larval\"],\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"digestive organs, eye, lens, liver, pancreas, intestinal epithelium\",\n  \"tissue_name_structured\": [\"eye\", \"liver\", \"pancreas\", \"gut\", \"Other\"],\n  \"developmental_stage_text\": \"larvae\",\n  \"developmental_stage_coarse\": [\"larval\"],\n  \"developmental_stage_fine\": [\"larval\"],\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 114,
      "prompt_tokens": 9561,
      "total_tokens": 9675
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_e467c31c3d"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [0ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "digestive organs, eye, lens, liver, pancreas, intestinal epithelium",
  "tissue_name_structured": [
    "eye",
    "liver",
    "pancreas",
    "gut",
    "Other"
  ],
  "developmental_stage_text": "larvae",
  "developmental_stage_coarse": [
    "larval"
  ],
  "developmental_stage_fine": [
    "larval"
  ],
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [9.19s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "digestive organs, eye, lens, liver, pancreas, intestinal epithelium",
  "tissue_name_structured": [
    "eye",
    "liver",
    "pancreas",
    "gut",
    "Other"
  ],
  "developmental_stage_text": "larvae",
  "developmental_stage_coarse": [
    "larval"
  ],
  "developmental_stage_fine": [
    "larval"
  ],
  "is_scRNA_data": false
}
[chain/start] [1:chain:RunnableSequence] Entering Chain run with input:
{
  "section_text": "UHRF1 overexpression drives DNA hypomethylation and\nhepatocellular carcinoma\nRaksha Mudbhary a,b,c , Yujin Hoshida a,d,l , Yelena Chernyavskaya a,b,l , Vinitha Jacob a,b,c , Augusto Villanueva f , M. Isabel Fiel e , Xintong Chen a,d , Kensuke Kojima a,d , Swan Thung e ,\nRoderick T. Bronson g , Anja Lachenmayer d , Kate Revill d , Clara Alsinet f , Ravi Sachidanandam h , Anal Desai i , Sucharita SenBanerjee i , Chinweike Ukomadu i , Josep M. Llovet a,d,f,j , and Kirsten C. Sadler a,b,c,k\na Department of Medicine/Division of Liver Diseases, Icahn School of Medicine at Mount Sinai New York, NY 10029\nb Department of Developmental and Regenerative Biology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nc Graduate School of Biological Sciences, Icahn School of Medicine at Mount Sinai New York, NY 10029\nd Liver Cancer Program/Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai New York, NY 10029\ne Department of Pathology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nf HCC Translational Research Laboratory, IDIBAPS, CIBEREHD, Hospital Clinic, University of Barcelona, Catalonia, Spain\ng Rodent Histopathology Core Dana Farber/Harvard Cancer Center, Harvard Medical School Boston, MA 02115\nh Department of Genetics and Genomics, Icahn School of Medicine at Mount Sinai New York, NY 10029\nI Division of Gastroenterology, Department of Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115\nj Institució Catalana de Recerca i Estudis Avançats Lluís Companys, 23, Barcelona – Spain\n\n\nSUMMARY\nUHRF1 is an essential regulator of DNA methylation that is highly expressed in many cancers. Here, we use transgenic zebrafish, cultured cells and human tumors to demonstrate that UHRF1 is an oncogene. UHRF1 overexpression in zebrafish hepatocytes destabilizes and delocalizes DNMT1, causes DNA hypomethylation and Tp53-mediated senescence. Hepatocellular carcinoma\n© 2014 Elsevier Inc. All rights reserved.\nk Corresponding author: Kirsten.edepli@mssm.edu, 1 Gustave L. Levy Place, Box 1020, Icahn School of Medicine at Mount Sinai, New York, NY 10029, phone: 212-241-7152, fax: 212-860-9279. lthese authors contributed equally to this work.\n\n\nAccession Number\nData from RNAseq was deposited with accession number GSE52605.\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\nPublished in final edited form as:\nCancer Cell. 2014 February 10; 25(2): 196–209. doi:10.1016/j.ccr.2014.01.003.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n(HCC) emerges when senescence is bypassed. tp53 mutation both alleviates senescence and accelerates tumor onset. Human HCCs recapitulate this paradigm, as UHRF1 overexpression defines a subclass of aggressive HCCs characterized by genomic instability, TP53 mutation and abrogation of the TP53-mediated senescence program. We propose that UHRF1 overexpression is a mechanism underlying DNA hypomethylation in cancer cells and that senescence is a primary means of restricting tumorigenesis due to epigenetic disruption.\n\n\nINTRODUCTION\nThe expression of genes that encode readers and writers of the epigenetic code are widely deregulated across cancer types (You and Jones, 2012). This contributes to the massive gene expression changes and remodeling of the epigenetic landscape, a characteristic of many types of cancer. In particular, loss of global DNA methylation is a hallmark of cancer cells.\nDNA hypomethylation contributes to oncogenesis through multiple mechanisms, including chromosomal instability (Eden et al., 2003; Karpf and Matsui, 2005), derepression of imprinted genes (Berdasco and Esteller, 2010; Jirtle, 2004; Li et al., 1993), retrotransposon activation (Gaudet et al., 2004; Howard et al., 2008; Jackson-Grusby et al., 2001; Sharif et al., 2007) and aberrant gene expression, including induction of oncogenes (Cheah et al., 1984). Many studies have documented that expression of the core factors required for maintenance DNA methylation – i.e. DNA methyltransferase 1 (DNMT1) and ubiquitin-like with PHD and RING finger domains 1 (UHRF1) (Babbio et al., 2012; Jin et al., 2009; Unoki et al., 2010; Wang et al., 2012) – are significantly altered across cancer types. However, whether changes in the expression of these key factors are sufficient to alter the cancer cell methylome and drive carcinogenesis is unknown. Moreover, the mechanism by which DNA methylation is lost in cancer cells is poorly understood.\nThe cellular response to DNA hypomethylation varies by cell type, physiological context and degree of hypomethylation. In some cells, DNA hypomethylation induces tumor suppressive mechanisms, including apoptosis (Anderson et al., 2009; Biniszkiewicz et al., 2002; Chen et al., 2007; Jackson-Grusby et al., 2001) or senescence (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), whereas in other cells it blocks differentiation of progenitor cells (Rai et al., 2010) or causes cancer (Gaudet et al., 2003; Yamada et al., 2005). In part, the cellular response to loss of DNA methylation is dictated by the genomic region affected: hypomethylation of gene regulatory regions, such as promoters, can de-repress gene expression but hypomethylation of repetitive elements can reduce heterochromatin formation, promote recombination and genomic instability.\nUHRF1 plays an essential role in DNA methylation by both recognizing hemimethylated DNA generated during DNA replication and then recruiting DNMT1 to ensure faithful maintenance of DNA methylation patterns in daughter cells (Arita et al., 2008; Avvakumov et al., 2008; Bostick et al., 2007; Hashimoto et al., 2008; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Consequently, UHRF1 depletion results in global DNA hypomethylation (Bostick et al., 2007; Feng et al., 2010; Sharif et al., 2007; Tittle et al., 2011). Conversely, UHRF1 may also limit DNA methylation by targeting DNMT1 for ubiquitin-mediated degradation (Du et al., 2010; Qin et al., 2011) or by delocalizing DNMT1 (Sharif et al., 2007). How UHRF1 overexpression impacts the methylome is not known.\nWe previously reported that uhrf1 mutation in zebrafish blocks liver outgrowth in embryos and regeneration in adults (Sadler et al., 2007) and that depleting UHRF1 from cancer cells induces apoptosis (Tien et al., 2011). Here, we tested the hypothesis that UHRF1\n\n\nNIH-PA Author Manuscript\noverexpression would alter global DNA methylation, and promote hepatocellular carcinoma (HCC).\n\n\nRESULTS\n\n\nHigh UHRF1 expression causes DNA hypomethylation\nUHRF1 is required for DNA methylation, as it recruites DNMT1 to hemimethylated DNA during DNA replication (Bostick et al., 2007; Feng et al., 2010; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Paradoxically, UHRF1 also serves as an ubiquitin ligase that targets DNMT1 for degradation (Du et al., 2010; Qin et al., 2011). To determine how UHRF1 overexpression impacts DNA methylation, we generated transgenic zebrafish expressing human UHRF1 fused to GFP (hsa.UHRF-GFP) under the hepatocyte specific fabp10 promoter (Tg(fabp10:hsa.UHRF1-GFP) (Chu et al., 2012)).\nHuman and zebrafish UHRF1 are 66% identical and the ability of human UHRF1 to rescue zebrafish embryos depleted of uhrf1 (VJ and KCS, unpublished and (Chu et al., 2012)) indicates they are functional orthologs. Expression is first detected in hepatocytes on 3 dpf and 5 dpf, nuclear UHRF1-GFP is easily detected at variable levels in hepatocytes (Figure S1A and (Chu et al., 2012)). We isolated an allelic series of Tg(fabp10:hsa.UHRF1-GFP) transgenics expressing a range of UHRF1 levels (Figure S1A–B; hereafter referred to UHRF1-GFP High, Medium and Low).\nWe probed genomic DNA isolated from the liver of 5 dpf larvae from each line with an antibody specific for 5-methyl cytosine (5MeC) to assess DNA methylation and found significant hypomethylation only in the liver of UHRF1-GFP High larvae (38% of controls; p=0.007; Figure 1A and Figure S1C). 5MeC immunofluorescence showed DNA methylation as uniformly distributed in the nuclei of hepatocytes from control larvae that express nuclear-localized fluorescent mCherry under the fabp10 promoter (Tg(fabp10:nls-mCherry); abbreviated to nls-mCherry) but only dim staining in hepatocytes of UHRF1-GFP High larvae (Figure 1B). Since Equivalent levels of 5MeC were detected in the liverless carcasses (Figure 1A) or fins (Figure 1B), we conclude that DNA hypomethylation was specific to the liver.\nWe hypothesized that UHRF1-induced DNA hypomethylation could be caused by mislocalization or destabilization of Dnmt1. Immunofluorescence revealed uniform Dnmt1 distribution in the nucleoplasm of nls-mCherry hepatocytes (Figure 1C) but in UHRF1-GFP high hepatocytes, it was concentrated in nuclear foci that contained UHRF1-GFP (Figure 1C). Interestingly, the range of UHRF1-GFP expression within cells of the same liver revealed that in cells with high GFP, Dnmt1 was dim and punctate, but levels and distribution pattern was similar to control hepatocytes in those cells with low or no GFP (arrows, Figure 1C). Thus, high expression of UHRF1 co-localizes with Dnmt1 and redistributes it to intra-nuclear structures reminiscent of senescence associated heterochromatin foci (Di Micco et al., 2011).\nEndogenous Dnmt1 in the liver was below the levels detectable by immunoblotting (not shown), so we used a transgenic line expressing UHRF1-GFP under the inducible hsp70l promoter (Chu et al., 2012) to quantitatively assess the impact of UHRF1 overexpression on Dnmt1 stability at a developmental time when we could easily detect Dnmt1 levels by immunoblotting. We optimize a heat shock protocol that maximized UHRF1-GFP expression in Tg(hsp70l:UHRF1-GFP) larvae (Figure 1D) and found that UHRF1-GFP overexpression reduced Dnmt1 levels by 27% compared to non-heat-shocked transgenics (p=0.03; Figure 1E). Treatment with a non-toxic dose of the proteasome inhibitor, MG132, prevented the decrease in Dnmt1 induced by UHRF1 overexpression (p=0.01; Figure 1E).\n\n\nNIH-PA Author Manuscript\nNeither heat-shock nor MG132 significantly affected Dnmt1 protein levels in non-transgenic controls (Figure 1E). Thus, both Dnmt1 delocalization and destabilization could account for DNA hypomethylation caused by UHRF1 overexpression.\n\n\nDNA hypomethylation caused by UHRF1 overexpression reduces liver size\nUhrf1 in zebrafish is required for hepatic outgrowth and liver regeneration (Sadler et al., 2007). To determine if UHRF1 overexpression in hepatocytes affected hepatic outgrowth, we assessed liver size all transgenic lines (Figures 2A–C and S2A). Low and Medium expressing lines had no gross changes in liver size (Figure S2A), but 74% of UHRF1-GFP High larvae on 5 dpf had small livers (Figure 2C), with the median area of the left liver lobe reduced by 35% compared to controls (p&lt;0.001; Figures 2D, S2A), without reduction of total fish size (Figure S2B).\nLarvae with small livers appeared sick (see 10 dpf larvae in Figure 2A), and only 20% of UHRF1-GFP High fish survived to 20 dpf (Figure S2C and Figure 2E). Moreover, UHRF1- GFP High larvae with the “microliver” phenotype on 5 dpf had significantly higher mortality by 10 dpf (67%) than those that started with a normal sized liver (7%; Figure 2E). Thus, high UHRF1 expression in hepatocytes causes DNA hypomethylation, microliver and larval death.\nWe next investigated the relationship between DNA hypomethylation and the microliver phenotype by asking whether further reducing DNA methylation could enhance this phenotype or restoring Dnmt1 could suppress it. We previously reported that exposing embryos to 50 μ M of the Dnmt1 inhibitor, 5-azacytidine (5-Aza) from 0–5 dpf caused a profoundly small liver and DNA hypomethylation (Mudbhary and Sadler, 2011). By restricting the 5-Aza exposure time to 2.5–5 dpf, we reduced DNA methylation in the liver of control larvae by 40% (Figure S1C) which induced a moderately small liver in 16% of larvae (p=0.0035; Figure 2F). This same treatment of UHRF1-GFP High larvae significantly increased the percent with small livers (p&lt;0.0001; Figure 2F). While this could be attributed to DNA damage caused by 5-Aza, our finding that injecting UHRF1-GFP High embryos with mRNA encoding Dnmt1 modestly suppressed the percent of fish with a small liver (Figure 2G) and increased the average size of the left liver lobe by 10–15% (Figure S2D) suggests that hypomethylation, at least in part, contributes to the small liver phenotype of UHRF1-GFP High larvae.\n\n\nUHRF1 overexpression triggers Tp53-mediated senescence\nDNA hypomethylation can induce apoptosis, but we found no TUNEL positive cells on 5 dpf of UHRF1-GFP High livers (Figure S3A). However, senescence associated β - galactosidase (SA- β -gal) staining was detected throughout the liver of most UHRF1-GFP High 5 dpf larvae (n=93; Figure 3A–B), but not other transgenic lines (Figure S3B). Additionally, the DNA in control hepatocytes was evenly distributed throughout the uniformly sized nuclei compared to the large nuclei where DNA resembled senescence associated heterochromatic foci (inset, Figure 3C). Strikingly, the largest hepatocyte nuclei also had the brightest GFP (Figure 3C), suggesting the effect of UHRF1 overexpression cell autonomously affected nuclear morphology.\nSenescent cells do not divide, and we found significantly less BrdU incorporation in the liver of 5 dpf UHRF1-GFP High larvae compared to controls (p&lt;0.0001; Figure 3D). Interestingly, in nls-mCherry larvae, most BrdU incorporation was detected in hepatocytes that express nls-mCherry, but the only BrdU positive cells in UHRF1-GFP high larvae were negative for GFP (see inset in Figure 3D). RNAseq revealed some pro-proliferative genes (ccnd1 and myc) downregulated (Figure 3E), lending further support to the conclusion that\nsenescence is the primary response to high UHRF overexpression in hepatocytes during hepatic outgrowth.\nTp53 is a key mediator of senescence caused by DNA damage and oncogenic stress (Di Micco et al., 2011; McDuff and Turner, 2011; Ventura et al., 2007; Xue et al., 2007). RNAseq (Figure 3E) and qPCR analysis (Figure 3F) show that tp53 and its target genes, especially cdkn1a, are significantly induced in the liver of 5 dpf UHRF1-GFP High larvae, but then return to baseline by 20 dpf (Figure 3F). 5-Aza treatment of primary mouse hepatocytes induced Tp53 expression (Figure 3G), similar to the effects of 5-Aza treatment or Dnmt1 depletion in other cell types (Jackson-Grusby et al., 2001; Karpf et al., 2001) and in zebrafish (VJ and KCS, not shown). However, since no significant alteration in methylation of the tp53 promoter was detected in these models (not shown), we hypothesize that DNA methylation does not directly regulate Tp53 expression. Instead, DNA hypomethylation may induce tp53 by an indirect mechanism, such as increased DNA damage or genomic instability.\nA direct role for Tp53 in the phenotype of UHRF1-GFP High larvae was demonstrated by removing one copy of tp53. This reduced the incidence and intensity of SA- β -gal staining in the liver (Figure 3H), increased liver size (Figure 3I–J) and reduced mortality (Figure 3K). We thus propose a model whereby high UHRF1 causes DNA hypomethylation, induces Tp53-mediated senescence and prevents expansion of the hepatic bud, resulting in hepatic insufficiency and larval death.\n\n\nUHRF1 overexpression induces liver cancer in zebrafish\nTo determine if UHRF1 overexpression was sufficient to cause HCC, 281 control and UHRF1-GFP transgenics were collected between 5–300 dpf, serial sectioned and analyzed for atypical cells, dysplastic foci and HCC using histological criteria devised by two expert pathologists (RTB and MIF) based on disrupted tissue architecture, cell size, shape, nuclear structure and presence of mitotic figures (Figure S4A–B). Evidence of increased hepatocyte proliferation was detected in all UHRF1 overexpressing lines on 20 and 40 dpf (Figure S4C), but this was insufficient to cause HCC, as UHRF1-GFP Low fish were tumor free at all time points (Table 1). In contrast, UHRF1-GFP High fish developed atypical hepatocytes as early as 5 dpf, with an 8% incidence of dysplastic foci and 46% incidence of HCC by 15 dpf. On 20 dpf, 76% of fish had HCC (Figure 4A–B, Table 1). UHRF1-GFP Medium fish also developed atypical hepatocytes and dysplastic foci at young ages, and one large HCC was detected in a 60 dpf fish (Table 1). In a classical transformation assay using NIH-3T3, we found that UHRF1 cooperated with RAS to promote growth on soft agar (Figure 4C). These conclusively demonstrate that UHRF1 is an oncogene.\nWe found that UHRF1-GFP High larvae older than 8 dpf had a lower incidence and intensity of hepatic SA- β -gal staining (Figure 4D). Interestingly, in many of these fish, intense staining was distributed in a punctate pattern. By 20 dpf, 75% of fish had either punctate or no staining (Figure 4D), which is a striking correlation with the 70% incidence of HCC at this time point. This was mirrored by increased proliferation of liver cells, detected by increased BrdU incorporation in UHRF1-GFP High livers (22% vs. 4% in controls) at 11 dpf (p&lt;0.001; Figure 4E), and higher PCNA staining 20 and 40 dpf in all lines (Figure S4C). Loss of senescence was not attributed to reestablishment of DNA methylation or transgene silencing, as reduced 5MeC staining persisted in tumor cells (Figure S4D) and transgene expression was detectable in all 20 dpf fish (Figures S4E–F), albeit reduced from levels detected in 5 dpf livers.\nWe asked whether Tp53 epistatically interacted with UHRF1 overexpression to contribute to HCC by removing one copy of tp53 (Berghmans et al., 2005). The liver appeared normal in\n\n\nNIH-PA Author Manuscript\n13 tp53 +/+ fish without transgene expression (not shown), but in UHRF1-GFP High fish, tumor incidence on 15 dpf increased from 50% in WT to 87% in tp53 +/− fish (Figure 4B). Interestingly, in a single Tg(fabp10:UHRF1-GFP HIGH );p53 +/− , we found a tumor with immature cells resembling a cholangiocarcinoma. Thus, tp53 functions to suppress tumor formation and may alter the spectrum of tumors caused by UHRF1 overexpression.\n\n\nUHRF1 is upregulated in human HCC\nWe next investigated the relevance of UHRF1 expression in human HCC. We assessed UHRF1 expression by qPCR in 16 normal liver samples and in 2 cohorts of patients with dysplastic nodules or HCC: the 1 st cohort of 58 patients had hepatitis C infection (Figure 5A; (Wurmbach et al., 2007)) and the 2 nd cohort of 69 patients had HBV, alcohol and other etiologies (Figure S5A; (Villanueva et al., 2008)). Additional publically available transcriptome data sets from three other cohorts of HCCs (Figure S5B) and from lung, gastric, colorectal and breast cancer (Figure S5C) were also analyzed. All showed elevated UHRF1 in tumors, with expression elevated &gt;2-fold compared to controls in 17/18 dysplastic foci and 104/109 HCCs (Figures 5A and S5A). An average of 20- and 46-fold overexpression of UHRF1 was detected in advanced and very advanced HCCs. UHRF1 protein was barely detectable in 5 normal liver samples but highly expressed in 38/52 (73%) of the tumors analyzed for UHRF1 mRNA in Figure 5A (p&lt;0.003; Figure 5B). Thus, UHRF1 is overexpressed in HCCs of diverse etiologies as well as in other tumor types, and high UHRF1 expression is significantly associated with the most advanced tumors.\nTargeting UHRF1 in liver cancer cells (HepG2) using siRNA induced PARP cleavage (Figure 5C) and other markers of apoptosis (not shown), similar to results obtained using another HCC cell line (Hep3B; not shown) and colon cancer cells (Tien et al., 2011). Thus, blocking UHRF1 cancers such as HCC with high UHRF1 expression could be an effective means to induce tumor cell death.\nOf the 109 HCCs analyzed for UHRF1 expression by qPCR, 71 had multiple clinical and genomics parameters available (Chiang et al., 2008). These were rank-ordered based on UHRF1 expression determined by qPCR (Figure 6A). Tumors with expression above and equal to or below the median log2-fold change value of 3.64 were designated as “high” (n=35) and “low” (n=36), respectively (Figure 6A). High UHRF1 expressing tumors were associated with signs of poor clinical outcome: 80% had microvascular invasion (Figure 5D) and significantly higher alpha-feto protein (AFP) levels (Figure 5E), although AFP levels alone did not cooperate with UHRF1 to predict survival (not shown). Importantly, high UHRF1 expression significantly correlated with early (&lt;2 years; Figure 5F), but not late (Figure 5G) tumor recurrence and was inversely correlated with survival (Figure 5H). This suggests that high UHRF1 expression predicted recurrence of the primary tumor, causing decreased survival (Villanueva et al., 2011). Molecular signatures of aggressive HCC tumors compiled from several previous studies (Table S1) were concordantly and significantly enriched in high UHRF1 expressing tumors (Figure 6A). Moreover, high UHRF1 expressing tumors were distinguished by induction of pathways that drive the cell cycle, DNA replication and repair (Figure S6A, Tables S2 and S3). UHRF1 overexpression also significantly correlated with advanced stage prostate cancer, but not lung or colon (Figure S5D–G).\nHigh UHRF1 expression delineates a subclass of HCCs that have downregulated TP53- mediated senescence\nOur zebrafish studies demonstrated that bypass of Tp53-induced senescence is required for UHRF1 to act as an oncogene. Our analysis of human HCCs indicates that a similar paradigm occurs in these samples. First, inactivating mutations in TP53 significantly\n\n\nNIH-PA Author Manuscript\ncorrelated with high UHRF1 expression (Figure 6A). Second, many of the core enriched genes high UHRF1 expressing tumors are regulated by TP53 in senescent fibroblasts (Table S4 and (Tang et al., 2007), and high UHRF1 expressing tumors have downregulated the gene expression signature derived from TP53-induced senescence in fibroblasts (Table S5, Figure S6B; (Tang et al., 2007)) or in hepatic stellate cells (Table S6, Figure S6C; (Lujambio et al., 2013)). Fourth, significant correlation between UHRF1 overexpression, TP53 mutation and genome integrity in human HCCs indicates that these pathways act together: high UHRF1 expression (Figure 6B–C) and TP53 mutation (Figure 6D) are independently correlated with chromosomal loss but tumors that have both features display even more chromosomal loss (Figure 6D). Finally, UHRF1 expression was significantly higher in tumors with TP53 mutation (Figure 6E). UHRF1 overexpression did not correlate with copy number variation at the UHRF1 locus (Figure S6D) suggesting that a different mechanism drives UHRF1 overexpression.\nGenome wide DNA hypomethylation is found in most HCCs (Calvisi et al., 2007) and this rendered it difficult to correlate methylome changes with UHRF1 expression. However, DNMT1 expression was directly correlated with UHRF1 expression in HCC samples (Figure 6F). This may be a consequence of the high proliferation rate in these tumors or an induction of the methylation machinery to compensate for hypomethylation. Together, these data indicate that high UHRF1 expression in HCC defines a subset of aggressive tumors that have inactivated the TP53 induced senescence program, suggesting that in humans, as observed in zebrafish, TP53 acts as a tumor suppressor to restrict the oncogenic potential of UHRF1.\n\n\nDISCUSSION\nWe show that UHRF1 overexpression is sufficient to cause two oncogene-associated phenotypes: senescence and cancer. This defines UHRF1 as an epigenetic regulator that can cause cancer when overexpressed and is thus an oncogene. We hypothesize that UHRF1 overexpression is one mechanism by which cancer genomes can become hypomethylated, either via UHRF1-mediated Dnmt1 ubiquitination and degredation (Du et al., 2010; Qin et al., 2011) or by redistribution and/or sequestration of Dnmt1 away from DNA. Excess UHRF1 might also sequester the DNMT1 deubiquitinating enzyme, USP7 (Qin et al., 2011) to further promote DNMT1 ubiquitination and degradation.\nThe precise mechanism by which DNA hypomethylation causes cancer remains elusive. Both apoptosis and senescence serve to limit the propagation of cells with aberrant DNA methylation (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), however, once epigenetically altered cells escape these tumor suppressive mechanisms, they likely accumulate genetic lesions which predispose to cancer. Indeed, chromosomal instability and mitotic catastrophe occur following DNMT1 depletion (Chen et al., 2007; Karpf and Matsui, 2005; Weber and Schubeler, 2007) and transposons, which are heavily methylated in normal cells, could become activated and cause genomic instability upon UHRF1-induced DNA hypomethylation. Additionally, hypomethylated DNA is more likely to assume an open chromatin conformation, which may promote oncogene expression, although this possibility has not been fully evaluated. While the oncogenic role of UHRF1 could also be mediated by the impact of UHRF1 on other epigenetic marks or on DNA replication (Taylor et al., 2013), our data combined with findings from others (Eden et al., 2003; Gaudet et al., 2003) suggests that DNA hypomethylation is a likely mechanism driving UHRF1-mediated transformation.\nOur working model (Figure 7) proposes that UHRF1 overexpression causes DNA hypomethylation by reducing Dnmt1 levels and its access to DNA. Tp53 is induced in\n\n\nNIH-PA Author Manuscript\nresponse to either genomic instability or some unidentified direct mechanism, and causes hepatocyte senescence, which prevents hepatic expansion and results in larval death from hepatic insufficiency. We propose that Tp53 inactivation and senescence bypass allows for the unhindered proliferation and malignant transformation. Identifying the mechanism by which this tumor suppressive mechanism is overcome remains a central, unanswered question in cancer biology.\nStudies in a mouse liver cancer model show that Tp53 reactivation causes senescence, and these cells are then cleared by the immune system (Xue et al., 2007) and the liver is then repopulated with senescence-resistant tumor forming cells. Our finding that senescence decreases and BrdU incorporation increases in UHRF1-GFP high livers over time suggests a similar process at play. Moreover, BrdU incorporation in primarily GFP negative cells suggests the expansion of either immune cells or immature hepatic progenitors in response to senescent hepatocytes. The finding of a cholangiocarcinoma in a UHRF1-GFP High/ p53 +/− fish may indicate that a bipotential progenitor cell could be the tumor forming cell in this model. Additionally, our data indicate that there is a threshold effect of UHRF1 expression, in which the highest expressing cells undergo senescence and neighboring hepatocytes expressing UHRF1-GFP at levels below those detectable via microscopy undergo unhindered expansion.\nHCC is the third cause of cancer-related deaths globally (Llovet et al., 2003), yet curative therapies are limited, with Sorafanib as the only systemic therapy available for advanced cases (Llovet et al., 2008). Thus, there is an urgent and unmet need for novel therapies. HCC, like other cancers, is characterized by global DNA hypomethylation (Calvisi et al., 2007; Pogribny and Rusyn, 2012; Tischoff and Tannapfe, 2008), and high UHRF1 expression could be the cause. Our finding that UHRF1 depletion in HCC and other types of cancer cells causes apoptosis (Tien et al., 2011) presents UHRF1 as an attractive target for cancer therapy, whereby blocking UHRF1 could trigger massive epigenetic changes incompatible with cell survival or could reset the cancer cell methylome.\n\n\nEXPERIMENTAL PROCEDURES\n\n\nZebrafish maintenance and generation of transgenics\nZebrafish were maintained on a 14:10 hour light:dark cycle at 28°C. mRNA encoding zebrafish Dnmt1 (Rai et al., 2006) or mannose phosphate isomerase (Mpi; (Chu et al., 2013) as a control was injected into embryos just after fertilization.\nTg(fabp10:nls-mCherry) fish expressing nls-mCherry exclusively in hepatocytes were generated using Gateway cloning (Invitrogen) to produce vectors with tol2 transposon sites (Kwan et al., 2007). Tg(hsp70l:hsa.UHRF1-GFP) and Tg(fabp10:hsa.UHRF1-GFP) were described in (Chu et al., 2012). The high, medium and low expressing alleles are listed in ZFIN.org with superscripts mss1a, mss1b and mss1c, respectively. Transgenics were outcrossed to Tab14 (WT) or tp53 −/− fish (Berghmans et al., 2005).\nTg(hsp70l:UHRF1-GFP) embryos were heat-shocked at 37°C for 1 hour at 24 and 27 hours post fertilization (hpf). At 28 hpf, embryos were sorted visually for GFP expression and incubated with either 10 μ M MG132 or DMSO and collected for immunoblotting at 34 hpf. 5-Aza (50 μ M) was added to larvae from 2.5–5 dpf. The Mount Sinai Institutional Animal Care and Use Committee approved all protocols. Nomenclature guidelines for the species under discussion were followed, and when no species is specified, human nomenclature was used.\n\n\nNIH-PA Author Manuscript\n\n\nGene expression analysis\nRNA was isolated from a pool of at least 10 livers from 5 dpf fish and from 1–5 livers from 20 dpf fish using the RNeasy mini-kit (Qiagen). cDNA was prepared by polyA priming using qScript SuperMix (Quanta). qRT-PCR analysis was performed in the Light Cycler 480 (Roche) using gene-specific primers (see supplemental materials) and PerfeCTa SYBRGreen FastMix (Quanta). C t values from triplicate reactions were averaged and 2 -Ct(target) /2 -Ct(reference) was used to calculate expression, with rpp0 and cyclophilin A used as references genes zebrafish and mouse samples, respectively.\nRNAseq analysis was carried out on RNA from pools of 50 livers dissected from 2 clutches of 5 dpf UHRF1-GFP High and nls-mCherry larvae, described in Supplemental Material.\n\n\nHistology\nFish younger than 20 dpf were fixed overnight at 4°C in 4% paraformaldehyde (PFA) and older fish were fixed for 2–5 days at room temperature in Bouins fixative. 4 μ M serial sections of paraffin embedded fish were stained with hematoxylin and eosin as described (Imrie and Sadler, 2010) and imaged on an Olympus BX41 Clinical Microscope equipped with a Nikon DS-Ri1 digital camera. Histological criteria used for scoring tumors are described in Figure S4A–B.\nImmunoblotting\nLysates prepared from 15 embryos dissolved in 150 μ l protein lysis buffer were homogenized by sonication. 1 embryo equivalent was loaded per lane of an 8 or 12% SDS gel, transferred to nitrocellulose and blotted. HepG2 cell lysates were prepared as described (Tien et al., 2011).\n\n\nAntibodies\nAntibodies recognizing DNMT1 (1:1000 for blotting, 1:10 for immunofluorescence, Santa Cruz), UHRF1 (immunoblotting: 1:1000; BD Biosciences. Immunohistochemistry: 1:50; ab57083; Abcam), Tubulin (1:5000 DSHB), p89 PARP and total PARP (1:1000; Cell Signaling) and b-actin (1:2000; Sigma), 5MeC (1:500; Eurogentec), BrdU (1:200; BD Bioscences) and anti-rabbit or mouse conjugated to Alexa 555 or Alexa 488 (1:100; Invitrogen) were diluted in 10% FBS or 2% BSA in 1% Triton in PBS (PBST).\nZebrafish staining\nLarvae fixed in 4% PFA were washed in PBST and stained using the Senescence β - Galactosidase Staining Kit (Cell Signaling) or stained with CY3-streptavadin (CY3-SA; 1:300; Sigma) as described (Sadler et al., 2005). The left liver lobe area was measured using ImageJ.\nImmunofluorescence was carried out on whole fish or on livers dissected from fixed larvae. BrdU was added to larvae water (10 mM) for 4–6 hours followed by immediate fixation, dehydration in methanol, rehydration to PBST and permeablization with 10ug/ml Proteinase K. DNA was denatured in 1 or 2N HCl, renatured and blocked in 10% FBS or 1% BSA in PBST prior to immunofluorescence. DNA was stained with Hoechst 33342 (Sigma). A Leica SP5 DM confocal microscope was used for imaging.\n\n\nSlot blots\nGenomic DNA was denatured in 0.4M NaOH at 95°C for 10 minutes, neutralized in an equal volume of cold 2M ammonium acetate and 100 ng DNA was blotted in duplicate onto nitrocellulose membrane using a slot blot apparatus and washed in 2x SSC and vacuum\n\n\nNIH-PA Author Manuscript\nbaked at 80°C for 2 hours. Half was stained with 0.2% methylene blue in 0.3 M NaOAc and the other with anti-5MeC followed by HRP-conjugated anti-mouse (1:2000) and visualized by Chemilumiescence (Roche). Image J was used to quantify 5MeC and methylene blue intensity and 5MeC levels were determined by normalizing to total DNA.\n\n\nCell culture\nPrimary mouse hepatocytes were isolated and plated in triplicate at 50% confluency then treated with 5 μ m 5-Aza or DMSO for 24 hours and RNA was isolated.\nNIH-3T3 cells were transfected in triplicate with pCDNA3.1 lacking an insert (control) or containing human RAS, UHRF1 or both and were re-transfected 24 hours later. 48 hours after the second transfection, 700 mg /ml Neomycin was added for 7 days. 10,000 cells per condition were plated on 0.3% soft agar layered on top of 0.6% soft agar. Media was changed every 3 rd day for 2 weeks and plates were stained with 0.005% crystal violet and colony number was counted.\nHepG2 cells cultured in DMEM supplemented with 10% (v/v) FBS and 5% (w/v) penicillin- streptomycin were transfected twice, 24 hours apart, with 20 nM siRNA targeting firefly luciferase (GL2; Dhamarcon) or UHRF1 targeting si-A and si-B using RNAiMAX (Invitrogen) as described (Tien et al., 2011).\n\n\nHuman tissue samples\nPathologically staged human tumors, dysplastic foci and normal liver samples were obtained from the HCC Genomic Consortium (Mount Sinai Hospital, NY; Instituto Nazionale dei Tumori, Milan and Hospital Clinic, Barcelona). The study was approved by the IRB of each institution and informed consent was obtained from all participants. TaqMan Probes were used to analyze UHRF1 expression by qPCR as described (Villanueva et al., 2008). Samples were grouped into a training set (9 normal liver, 18 low- and high-grade dysplastic nodules, and 40 pathologically staged HCCs; (Llovet et al., 2006; Wurmbach et al., 2007)) and a validation set (7 normal liver and 69 HCC; (Villanueva et al., 2008)). Integrative genomic and clinical data analysis was performed on 71 of the hepatitis C-related, surgically treated HCC patients (NCBI Gene Expression Omnibus, accession numbers GSE9829 and GSE44970). Liver pathologist, ST, scored UHRF1 staining of 52 paraffin embedded tumors from the training set.\n\n\nBioinformatics and statistical analysis\n71 patients with HCC were grouped based on the median expression level of UHRF1 in the tumors determined by qPCR (UHRF1-high; n=35 or low; n=36). Presence of aggressive human HCC gene signatures (Table S1) were evaluated in the transcriptome dataset using nearest template prediction method (Hoshida, 2010) implemented by the GenePattern genomic analysis toolkit ( www.broadinstitute.org/genepattern ) based on prediction confidence p&lt;0.05. The senescence-related TP53 target genes were obtained from Molecular Signature Database (MSigDB, www.broadinstitute.org/msigdb , TANG_SENESCENCE_TP53_TARGETS_UP and _DN; (Tang et al., 2007)) and from NCBI Gene Expression Omnibus (GSE39469; (Lujambio et al., 2013)). Mouse genes from normalized microarray data were converted to human orthologs based on a mapping table ( www.informatics.jax.org ). Differentially expressed genes between senescent and proliferative cells were identified by using Bayesian T-test implemented in Cyber-T software (molgen51.biol.rug.nl/cybert) at the significance threshold of Posterior Probability of Differential Expression &gt;0.998 after excluding less variable genes with coefficient of variation ≤ 0.1 across the samples. Expression pattern of each signature was assessed by nearest template prediction algorithm. Significance of induction or suppression of each\n\n\nNIH-PA Author Manuscript\nsignature was quantitatively measured by nominal p-value of cosine distance. Pearson correlation test determined the significance between induction/suppression of gene signatures with UHRF1 mRNA expression level as measured by qPCR. Statistical difference between groups was assessed by Wilcoxon rank-sum test and Fisher’s exact test for continuous and categorical variables, respectively. Bonferroni correction for multiple hypothesis testing was applied when appropriate.\nSurvival analysis was performed by using Kaplan-Meier estimator and log-rank test. All analyses were performed using R statistical package ( www.r-project.org ) and GenePattern genomic analysis tool kit ( www.broadinstitute.org/genepattern ). Gene expression levels and liver size were compared using Student’s T or Mann Whitney U tests\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nSupport was provided by The Breast Cancer Alliance and the March of Dimes (to KCS), a Pilot Project from the DFCI NCI Cancer Center (5P30CA006516-45; to CU), the NIH (5R01DK080789-02 to CU and KCS,\n1R01DK099558 to YH, 1R01DK076986 to JML, F30DK094503 to VJ and T32CA078207-14 for YC), the European Commission 7th Framework Programme FP7-Health 2010 (Heptromic #259744 to YH and JML), The Samuel Waxman Cancer Research Foundation, The Spanish National Health Institute (SAF-2010-16055) and the Asociación Española Contra el Cáncer (JML). Liz Loughlin, Brandon Kent, Meghan Walsh, Alex Mir, Laia Cabellos, Helena Cornellà Vives and Sara Toffanin provided expert technical assistance. We are grateful to Sam Sidi for tp53−/− fish, stimulating discussions and critical reading of the manuscript.\n\n\nReferences\nAnderson RM, Bosch JA, Goll MG, Hesselson D, Dong PD, Shin D, Chi NC, Shin CH, Schlegel A, Halpern M, Stainier DY. Loss of Dnmt1 catalytic activity reveals multiple roles for DNA methylation during pancreas development and regeneration. Dev Biol. 2009\nArita K, Ariyoshi M, Tochio H, Nakamura Y, Shirakawa M. Recognition of hemi-methylated DNA by the SRA protein UHRF1 by a base-flipping mechanism. Nature. 2008; 455:818–821. [PubMed: 18772891]\nAvvakumov GV, Walker JR, Xue S, Li Y, Duan S, Bronner C, Arrowsmith CH, Dhe-Paganon S. Structural basis for recognition of hemi-methylated DNA by the SRA domain of human UHRF1. Nature. 2008; 455:822–825. [PubMed: 18772889]\nBabbio F, Pistore C, Curti L, Castiglioni I, Kunderfranco P, Brino L, Oudet P, Seiler R, Thalman GN, Roggero E, et al. The SRA protein UHRF1 promotes epigenetic crosstalks and is involved in prostate cancer progression. Oncogene. 2012\nBerdasco M, Esteller M. Aberrant epigenetic landscape in cancer: how cellular identity goes awry. Dev Cell. 2010; 19:698–711. [PubMed: 21074720]\nBerghmans S, Murphey RD, Wienholds E, Neuberg D, Kutok JL, Fletcher CD, Morris JP, Liu TX, Schulte-Merker S, Kanki JP, et al. tp53 mutant zebrafish develop malignant peripheral nerve sheath tumors. Proc Natl Acad Sci U S A. 2005; 102:407–412. [PubMed: 15630097]\nBiniszkiewicz D, Gribnau J, Ramsahoye B, Gaudet F, Eggan K, Humpherys D, Mastrangelo MA, Jun Z, Walter J, Jaenisch R. Dnmt1 overexpression causes genomic hypermethylation, loss of imprinting, and embryonic lethality. Mol Cell Biol. 2002; 22:2124–2135. [PubMed: 11884600]\nBostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE. UHRF1 plays a role in maintaining DNA methylation in mammalian cells. Science. 2007; 317:1760–1764. [PubMed: 17673620]\nCalvisi DF, Ladu S, Gorden A, Farina M, Lee JS, Conner EA, Schroeder I, Factor VM, Thorgeirsson SS. Mechanistic and prognostic significance of aberrant methylation in the molecular pathogenesis of human hepatocellular carcinoma. J Clin Invest. 2007; 117:2713–2722. [PubMed: 17717605]\n\n\nNIH-PA Author Manuscript\nCheah MS, Wallace CD, Hoffman RM. Hypomethylation of DNA in human cancer cells: a site- specific change in the c-myc oncogene. J Natl Cancer Inst. 1984; 73:1057–1065. [PubMed: 6092764]\nChen T, Hevi S, Gay F, Tsujimoto N, He T, Zhang B, Ueda Y, Li E. Complete inactivation of DNMT1 leads to mitotic catastrophe in human cancer cells. Nat Genet. 2007; 39:391–396. [PubMed: 17322882]\nChiang DY, Villanueva A, Hoshida Y, Peix J, Newell P, Minguez B, LeBlanc AC, Donovan DJ, Thung SN, Sole M, et al. Focal gains of VEGFA and molecular classification of hepatocellular carcinoma. Cancer Res. 2008; 68:6779–6788. [PubMed: 18701503]\nChu J, Loughlin EA, Gaur NA, Senbanerjee S, Jacob V, Monson C, Kent B, Oranu A, Ding Y, Ukomadu C, Sadler KC. UHRF1 phosphorylation by cyclin A2/cyclin-dependent kinase 2 is required for zebrafish embryogenesis. Mol Biol Cell. 2012; 23:59–70. [PubMed: 22072796]\nChu J, Mir A, Gao N, Rosa S, Monson C, Sharma V, Steet R, Freeze HH, Lehrman MA, Sadler KC. A zebrafish model of congenital disorders of glycosylation with phosphomannose isomerase deficiency reveals an early opportunity for corrective mannose supplementation. Dis Model Mech. 2013; 6:95–105. [PubMed: 22899857]\nDecottignies A, d’Adda di Fagagna F. Epigenetic alterations associated with cellular senescence: a barrier against tumorigenesis or a red carpet for cancer? Semin Cancer Biol. 2011; 21:360–366. [PubMed: 21946622]\nDi Micco R, Sulli G, Dobreva M, Liontos M, Botrugno OA, Gargiulo G, dal Zuffo R, Matti V, d’Ario G, Montani E, et al. Interplay between oncogene-induced DNA damage response and heterochromatin in senescence and cancer. Nat Cell Biol. 2011; 13:292–302. [PubMed: 21336312]\nDu Z, Song J, Wang Y, Zhao Y, Guda K, Yang S, Kao HY, Xu Y, Willis J, Markowitz SD, et al. DNMT1 stability is regulated by proteins coordinating deubiquitination and acetylation-driven ubiquitination. Sci Signal. 2010; 3:ra80. [PubMed: 21045206]\nEden A, Gaudet F, Waghmare A, Jaenisch R. Chromosomal instability and tumors promoted by DNA hypomethylation. Science. 2003; 300:455. [PubMed: 12702868]\nFairweather DS, Fox M, Margison GP. The in vitro lifespan of MRC-5 cells is shortened by 5- azacytidine-induced demethylation. Exp Cell Res. 1987; 168:153–159. [PubMed: 2430819]\nFeng S, Cokus SJ, Zhang X, Chen PY, Bostick M, Goll MG, Hetzel J, Jain J, Strauss SH, Halpern ME, et al. Conservation and divergence of methylation patterning in plants and animals. Proc Natl Acad Sci U S A. 2010; 107:8689–8694. [PubMed: 20395551]\nGaudet F, Hodgson JG, Eden A, Jackson-Grusby L, Dausman J, Gray JW, Leonhardt H, Jaenisch R. Induction of tumors in mice by genomic hypomethylation. Science. 2003; 300:489–492. [PubMed: 12702876]\nGaudet F, Rideout WM 3rd, Meissner A, Dausman J, Leonhardt H, Jaenisch R. Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. Mol Cell Biol. 2004; 24:1640–1648. [PubMed: 14749379]\nHashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X. The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. Nature. 2008; 455:826–829. [PubMed: 18772888]\nHoshida Y. Nearest template prediction: a single-sample-based flexible class prediction with confidence assessment. PLoS One. 2010; 5:e15543. [PubMed: 21124904]\nHoward G, Eiges R, Gaudet F, Jaenisch R, Eden A. Activation and transposition of endogenous retroviral elements in hypomethylation induced tumors in mice. Oncogene. 2008; 27:404–408. [PubMed: 17621273]\nImrie D, Sadler KC. White adipose tissue development in zebrafish is regulated by both developmental time and fish size. Dev Dyn. 2010; 239:3013–3023. [PubMed: 20925116]\nJackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G, Dausman J, Lee P, Wilson C, Lander E, Jaenisch R. Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. Nat Genet. 2001; 27:31–39. [PubMed: 11137995]\nJin W, Chen L, Chen Y, Xu SG, Di GH, Yin WJ, Wu J, Shao ZM. UHRF1 is associated with epigenetic silencing of BRCA1 in sporadic breast cancer. Breast Cancer Res Treat. 2009\nJirtle RL. IGF2 loss of imprinting: a potential heritable risk factor for colorectal cancer. Gastroenterology. 2004; 126:1190–1193. [PubMed: 15057757]\n\n\nNIH-PA Author Manuscript\nKarpf AR, Matsui S. Genetic disruption of cytosine DNA methyltransferase enzymes induces chromosomal instability in human cancer cells. Cancer Res. 2005; 65:8635–8639. [PubMed: 16204030]\nKarpf AR, Moore BC, Ririe TO, Jones DA. Activation of the p53 DNA damage response pathway after inhibition of DNA methyltransferase by 5-aza-2 ′ -deoxycytidine. Mol Pharmacol. 2001; 59:751–757. [PubMed: 11259619]\nKwan KM, Fujimoto E, Grabher C, Mangum BD, Hardy ME, Campbell DS, Parant JM, Yost HJ, Kanki JP, Chien CB. The Tol2kit: a multisite gateway-based construction kit for Tol2 transposon transgenesis constructs. Dev Dyn. 2007; 236:3088–3099. [PubMed: 17937395]\nLi E, Beard C, Jaenisch R. Role for DNA methylation in genomic imprinting. Nature. 1993; 366:362– 365. [PubMed: 8247133]\nLiu X, Gao Q, Li P, Zhao Q, Zhang J, Li J, Koseki H, Wong J. UHRF1 targets DNMT1 for DNA methylation through cooperative binding of hemi-methylated DNA and methylated H3K9. Nat Commun. 2013; 4:1563. [PubMed: 23463006]\nLlovet JM, Burroughs A, Bruix J. Hepatocellular carcinoma. Lancet. 2003; 362:1907–1917. [PubMed: 14667750]\nLlovet JM, Chen Y, Wurmbach E, Roayaie S, Fiel MI, Schwartz M, Thung SN, Khitrov G, Zhang W, Villanueva A, et al. A molecular signature to discriminate dysplastic nodules from early hepatocellular carcinoma in HCV cirrhosis. Gastroenterology. 2006; 131:1758–1767. [PubMed: 17087938]\nLlovet JM, Ricci S, Mazzaferro V, Hilgard P, Gane E, Blanc JF, de Oliveira AC, Santoro A, Raoul JL, Forner A, et al. Sorafenib in advanced hepatocellular carcinoma. N Engl J Med. 2008; 359:378– 390. [PubMed: 18650514]\nLujambio A, Akkari L, Simon J, Grace D, Tschaharganeh DF, Bolden JE, Zhao Z, Thapar V, Joyce JA, Krizhanovsky V, Lowe SW. Non-cell-autonomous tumor suppression by p53. Cell. 2013; 153:449–460. [PubMed: 23562644]\nMcDuff FK, Turner SD. Jailbreak: oncogene-induced senescence and its evasion. Cell Signal. 2011; 23:6–13. [PubMed: 20633638]\nMudbhary R, Sadler KC. Epigenetics, development, and cancer: Zebrafish make their mark. Birth Defects Res C Embryo Today. 2011; 93:194–203. [PubMed: 21671358]\nNishiyama A, Yamaguchi L, Sharif J, Johmura Y, Kawamura T, Nakanishi K, Shimamura S, Arita K, Kodama T, Ishikawa F, et al. Uhrf1-dependent H3K23 ubiquitylation couples maintenance DNA methylation and replication. Nature. 2013; 502:249–253. [PubMed: 24013172]\nPogribny IP, Rusyn I. Role of epigenetic aberrations in the development and progression of human hepatocellular carcinoma. Cancer Lett. 2012\nQin W, Leonhardt H, Spada F. Usp7 and Uhrf1 control ubiquitination and stability of the maintenance DNA methyltransferase Dnmt1. J Cell Biochem. 2011; 112:439–444. [PubMed: 21268065]\nRai K, Nadauld LD, Chidester S, Manos EJ, James SR, Karpf AR, Cairns BR, Jones DA. Zebra fish Dnmt1 and Suv39h1 regulate organ-specific terminal differentiation during development. Mol Cell Biol. 2006; 26:7077–7085. [PubMed: 16980612]\nRai K, Sarkar S, Broadbent TJ, Voas M, Grossmann KF, Nadauld LD, Dehghanizadeh S, Hagos FT, Li Y, Toth RK, et al. DNA demethylase activity maintains intestinal cells in an undifferentiated state following loss of APC. Cell. 2010; 142:930–942. [PubMed: 20850014]\nSadler KC, Amsterdam A, Soroka C, Boyer J, Hopkins N. A genetic screen in zebrafish identifies the mutants vps18, nf2 and foie gras as models of liver disease. Development. 2005; 132:3561–3572. [PubMed: 16000385]\nSadler KC, Krahn KN, Gaur NA, Ukomadu C. Liver growth in the embryo and during liver regeneration in zebrafish requires the cell cycle regulator, uhrf1. Proc Natl Acad Sci U S A. 2007; 104:1570–1575. [PubMed: 17242348]\nSharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA, Shinga J, Mizutani-Koseki Y, Toyoda T, Okamura K, et al. The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. Nature. 2007; 450:908–912. [PubMed: 17994007]\nTang X, Milyavsky M, Goldfinger N, Rotter V. Amyloid-beta precursor-like protein APLP1 is a novel p53 transcriptional target gene that augments neuroblastoma cell death upon genotoxic stress. Oncogene. 2007; 26:7302–7312. [PubMed: 17533371]\nTaylor EM, Bonsu NM, Price RJ, Lindsay HD. Depletion of Uhrf1 inhibits chromosomal DNA replication in Xenopus egg extracts. Nucleic Acids Res. 2013\nTien AL, Senbanerjee S, Kulkarni A, Mudbhary R, Goudreau B, Ganesan S, Sadler KC, Ukomadu C. UHRF1 depletion causes a G2/M arrest, activation of DNA damage response and apoptosis. Biochem J. 2011; 435:175–185. [PubMed: 21214517]\nTischoff I, Tannapfe A. DNA methylation in hepatocellular carcinoma. World J Gastroenterol. 2008; 14:1741–1748. [PubMed: 18350605]\nTittle RK, Sze R, Ng A, Nuckels RJ, Swartz ME, Anderson RM, Bosch J, Stainier DY, Eberhart JK, Gross JM. Uhrf1 and Dnmt1 are required for development and maintenance of the zebrafish lens. Dev Biol. 2011; 350:50–63. [PubMed: 21126517]\nUnoki M, Daigo Y, Koinuma J, Tsuchiya E, Hamamoto R, Nakamura Y. UHRF1 is a novel diagnostic marker of lung cancer. Br J Cancer. 2010; 103:217–222. [PubMed: 20517312]\nVentura A, Kirsch DG, McLaughlin ME, Tuveson DA, Grimm J, Lintault L, Newman J, Reczek EE, Weissleder R, Jacks T. Restoration of p53 function leads to tumour regression in vivo. Nature. 2007; 445:661–665. [PubMed: 17251932]\nVillanueva A, Chiang DY, Newell P, Peix J, Thung S, Alsinet C, Tovar V, Roayaie S, Minguez B, Sole M, et al. Pivotal role of mTOR signaling in hepatocellular carcinoma. Gastroenterology. 2008; 135:1972–1983. 1983 e1971–1911. [PubMed: 18929564]\nVillanueva A, Hoshida Y, Battiston C, Tovar V, Sia D, Alsinet C, Cornella H, Liberzon A, Kobayashi M, Kumada H, et al. Combining clinical, pathology, and gene expression data to predict recurrence of hepatocellular carcinoma. Gastroenterology. 2011; 140:1501–1512.e1502. [PubMed: 21320499]\nWang F, Yang YZ, Shi CZ, Zhang P, Moyer MP, Zhang HZ, Zou Y, Qin HL. UHRF1 Promotes Cell Growth and Metastasis Through Repression of p16(ink4a) in Colorectal Cancer. Ann Surg Oncol. 2012\nWeber M, Schubeler D. Genomic patterns of DNA methylation: targets and function of an epigenetic mark. Curr Opin Cell Biol. 2007; 19:273–280. [PubMed: 17466503]\nWurmbach E, Chen YB, Khitrov G, Zhang W, Roayaie S, Schwartz M, Fiel I, Thung S, Mazzaferro V, Bruix J, et al. Genome-wide molecular profiles of HCV-induced dysplasia and hepatocellular carcinoma. Hepatology. 2007; 45:938–947. [PubMed: 17393520]\nXue W, Zender L, Miething C, Dickins RA, Hernando E, Krizhanovsky V, Cordon-Cardo C, Lowe SW. Senescence and tumour clearance is triggered by p53 restoration in murine liver carcinomas. Nature. 2007; 445:656–660. [PubMed: 17251933]\nYamada Y, Jackson-Grusby L, Linhart H, Meissner A, Eden A, Lin H, Jaenisch R. Opposing effects of DNA hypomethylation on intestinal and liver carcinogenesis. Proc Natl Acad Sci U S A. 2005; 102:13580–13585. [PubMed: 16174748]\nYou JS, Jones PA. Cancer genetics and epigenetics: two sides of the same coin? Cancer Cell. 2012; 22:9–20. [PubMed: 22789535]\n\n\nHighlights\n• UHRF1 overexpression induces DNA hypomethylation\n• Liver specific overexpression of UHRF1 in zebrafish causes senescence and HCC\n• Human HCCs with high UHRF1 are aggressive and inactivate TP53-mediated senescence\nNIH-PA Author Manuscript\n\n\nSIGNIFICANCE\nGlobal DNA hypomethylation occurs in most types of cancer and can induce genomic instability and widespread changes in gene expression. UHRF1 is a key regulator of DNA methylation, and here we show that UHRF1 overexpression causes HCC in zebrafish without any other genetic alteration, demonstrating that it is an oncogene. High UHRF1 expression causes DNA hypomethylation and tp53-mediated senescence, which serves to restrict transformation of UHRF1 overexpressing cells. High UHRF1 expression in human HCC correlates with a poor prognosis, genomic instability, TP53 mutation and repression of the TP53-mediated senescence program. We conclude that UHRF1 is an oncogene that promotes widespread DNA hypomethylation, an epigenetic hallmark of cancer cells, and that UHRF1 overexpression drives tumorigenesis when senescence is bypassed.\nNIH-PA Author Manuscript\nFigure 1. High UHRF1 expression causes global DNA hypomethylation (A) 5MeC levels and total DNA stained with methylene blue were measured in 5 dpf control and UHRF1-GFP High livers (n=4) and liver-less carcasses (n=3). The ratio of 5MeC to total DNA was averaged and normalized to controls. Student’s T-test was used to determine p values. (B) Confocal stacks of livers (top) and fins (bottom) from 5 dpf nls-mCherry and UHRF1-GFP High larvae stained with anti-5MeC. Since a hepatocyte specific promoter was used for transgenesis, there was no transgene expression in the fin. (C) Dnmt1 is uniform in the hepatocyte nucleus of 4 dpf nls-mCherry larvae but is found in GFP-containing punctae in UHRF1-GFP High hepatocytes. Arrows point to cells that do not express GFP and have Dnmt1 distribution pattern similar to controls. (D) Tg(hsp70I:UHRF1-EGFP) and non- transgenic controls were heat shocked at 37° C for 1 hour at 24 and 27 hpf, treated with 10 μ M MG132 or DMSO at 28 hpf and collected at 34 hpf for immunoblotting. (E) Dnmt1 levels normalized to tubulin were averaged from 6 experiments. Student’s T-test was used to determine p values; n.s.: not significant; error bars represent SD. See also Figure S1.\n\n\nNIH-PA Author Manuscript\nFigure 2. UHRF1-induced hypomethylation reduces liver size (A) Individual larvae were imaged daily from 3–10 dpf. (B) 5 dpf UHRF1-GFP High larvae display a range of liver sizes scored as “normal” or “small”‘ (C) 3 clutches were scored according to criteria in B; n= number of larvae. Fisher’s exact test was used to determine p value. (D) The area of the left liver lobe was measured in 5 dpf fish from 2 clutches. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. Student’s T-test was used to determine p value. (E) UHRF1-GFP High larvae were sorted by liver size on 5 dpf and tracked daily for survival to 20 dpf. Data are pooled from three clutches. (F) UHRF1-GFP High and control larvae were treated with 50 μ M 5-Aza from 2.5–5 dpf and scored for liver size in 6 clutches. Fisher’s exact test was used to determine p values. (G) UHRF1-GFP High embryos were injected with mRNA encoding dnmt1 or Mpi before 1 hpf. The percent of fish with a normal liver size was scored at 5 dpf in 6 clutches. See also Figure S2.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\nFigure 3. UHRF1 overexpression in hepatocytes induces Tp53-mediated senescence (A) Intense senescence associated β -galactosidase (SA- β -gal) staining was detected in the liver (outlined) of 5 dpf UHRF1-GFP High larvae compared to light or no staining in controls. (B) 5 dpf fish from 5 clutches were scored for hepatic SA- β -gal staining. *** indicates p&lt;0.0001 by Fisher’s exact test. (C) Nuclear size was measured in hepatocytes of a single control or UHRF1-GFP High 5 dpf liver and cells were stratified according to GFP expression. Inset shows confocal stack of the DNA organized into foci. ** and *** indicates p&lt;0.001 or 0.0001, respectively compared to nuclear size in nls-mCherry larvae. (D) BrdU positive cells and the total number of transgene-expressing hepatocytes in nls-mCherry and UHRF1-GFP High larvae (Bottom) 5 dpf larvae. A Fisher’s exact test was used to calculate\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\np value. In nls-mCherry larvae, most BrdU positive cells also express the transgene whereas the BrdU positive cells in UHRF1-GFP High livers did not express GFP (white arrows in magnified regions which are marked by the white box). (E) Heatmap of log2 values from RNAseq shows cell cycle regulators are down and Tp53 target genes (marked by *) are up in UHRF1-GFP High 5 dpf livers. (F) tp53 and cdkn1a mRNA expression were induced on 5 dpf and down regulated on 20 dpf in UHRF1-GFP High livers. * indicates p=0.05; *** indicates p=0.001 calculated by 1 sample Student’s T-test. Error bars represent SD. (G) 5- Aza induces Tp53 expression in primary mouse hepatocytes. Student’s T-test was used to determine p value with SD indicated by the error bars across 3 replicates. tp53 +/− in UHRF1-GFP High larvae significantly reduced SA- β -gal staining in the liver (2 clutches) (H), and increased the percent of larvae with normal liver size (I), the area of the left liver lobe (J), and survival at 5 dpf (K). p values were calculated with a Fisher’s test with Freeman-Halton extension (H), Fisher’s exact test (I), and Student’s T-test. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. (J). See also Figure S3.\nNIH-PA Author Manuscript\nFigure 4. UHRF1 is an oncogene\n(A) Atypical cells, dysplastic foci (outlined) and HCC are apparent in H&amp;E stained UHRF1- GFP High livers. BD: bile duct, MF: mitotic figure. (B) Incidence of normal and atypical hepatocytes, dysplastic foci and cancer in the liver of UHRF1-GFP High fish on WT or tp53 +/− background. (C) NIH-3T3 cell growth in soft agar is enhanced when UHRF1 overexpression is combined with RAS (n=3). p value was calculated by Student’s T-test and error bars represent the SD. (D) Hepatic SA- β -gal staining patterns in UHRF1-GFP High larvae changes as fish age. Images of 8 dpf larvae illustrate the SA- β -gal staining patterns that were scored in the time course shown in the graph. A significant increase in the number of UHRF1-GFP High fish with intense or punctate SA- β -gal compared to controls at all time points (p&lt;0.01 by Fisher’s exact test) except at 4 dpf; not significant=n.s. (E) BrdU incorporation in the liver on 11 dpf is 5 times higher in UHRF1-GFP High fish than in\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\ncontrols. Total # of cells counted is indicated with n= # of clutches assessed. Fisher’s exact test was used to calculate p value. See also Figure S4.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 5. UHRF1 mRNA and protein are overexpressed in HCC (A) UHRF1 detected by qRT-PCR in 18 pre-neoplastic lesions and 40 HCCs from HCV infected patients compared to expression in 9 normal livers. Horizontal line indicates median. (B) Immunohistochemistry for UHRF1 protein (brown) was evaluated in 52 of the same HCCs examined in A plus 5 normal liver samples. Fisher’s exact test was used to calculate p value. 71 of the HCV-associated HCCs analyzed by qPCR were grouped into high (n=35) and low (n=36) UHRF1 expressing tumors based on the median log2-fold change of 3.64. (C) HepG2 cells transfected with control siRNA (GL2) or two different siRNAs targeting UHRF1 described in (Tien et al., 2011) were blotted for UHRF1, cleaved and total PARP (arrow indicates full length, * indicates cleaved protein). (D) Vascular invasion (33 high and 34 low tumors, 4 missing values), (E) serum AFP (29 UHRF1-high and 29 low tumors, 13 missing values), (F) early (&lt;2 years) and (G) late (&gt;2 years; 32 UHRF1-high and 35 low tumors, 4 missing values) tumor recurrence and (H) overall survival after surgery (32 high and 35 low tumors, 4 missing values) were stratified according to UHRF1-expression. Continuous and categorical variables were assessed by Wilcoxon rank-sum test and Fisher’s exact test, respectively. Clinical outcome difference was evaluated by log-rank test. In box and whisker plots, boxes represent the 75th and 25th\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\npercentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S5.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 6. High UHRF1 expression defines a subclass of tumors with inactivated tp53, repression of senescence and chromosomal instability\n(A) The 71 human HCC tumors analyzed in Figure 5C–G were rank-ordered according to UHRF1 expression by qPCR and classified as high (&lt;median, red; n=35) or low (≥median, blue; n=36). The presences of aggressive human HCC gene signatures from published studies and of TP53 inactivating mutations are indicated by red and black boxes, respectively. TP53-mediated senescence gene signatures (Lujambio et al., 2013; Tang et al., 2007) are displayed as a range from repressed (blue) to activated (red). (B) Genome-wide profile of DNA copy number variation was obtained from GEO geneset GSE9829. (C) Proportion of genes with DNA copy number loss and gain in tumors according to UHRF1 expression. (D) Proportion of genes with DNA copy number loss according to UHRF1 expression and TP53 mutation status. (E) UHRF1 expression is significantly higher in tumors with TP53 mutations. (F) DNMT1 expression by microarray analysis is significantly correlated with UHRF1 expression assessed by qPCR in HCCs. In box and whisker plots,\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nboxes represent the 75th and 25th percentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S6 and Tables S1-S6.\nNIH-PA Author Manuscript\nFigure 7. Model of the relationship between UHRF1 overexpression, DNA hypomethylation, Tp53 mediated senescence, cancer and survival\nFactors investigated in this study are in solid black boxes with black lines indicating the correlations demonstrated in this work and gray lines indicating relationships that are speculative. Senescence reduces liver size and function and reduces larval survival while cancer occurs when senescence is bypassed and also reduces survival.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 2\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 4\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 6\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 7\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 8\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 9\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 10\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 11\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 12\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 13\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 14\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\n• Our studies in zebrafish and human tumors identify UHRF1 as an oncogene in HCC Mudbhary et al. Page 15\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 16\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 17\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 18\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 19\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 20\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 21\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 22\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 23\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 24\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 25\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 26\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 27\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 28\n\n\nTable 1\n\n\nHCC onset and incidence in Tg(fabp10:nls-mCherry) and Tg(fabp10:UHRF1-GFP) zebrafish. transgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI nls-mCherry Normal 100% 100% 100% 100% 100% 100% 100% 100% 100% 100% 65 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 8 8 8 11 nd nd 5 5 5 5 5 5 65 UHRF1-GFP Low Normal 100% 100% 100% 100% 100% 100% 100% 100% 60 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 7 9 10 12 nd nd 4 nd nd 6 6 6 60 UHRF1-GFP Medium Normal 90% (9) 100% (11) 50% (3) 28% (4) 66% (4) 80% (4) 50% (4) 50% (5) 81% (13) 57 66%\n\n\nAtypical cells 10% (1) 0% 50% (3) 57% (8) 33% (2) 20% (1) 50% (4) 20% (2) 44% (7) 28 32%\n\n\nDysplastic foci 0% 0% 0% 7% (1) 0% 0% 25% (2) 30% (3) 0% 6 7%\n\n\nTumor 0% 0% 0% 0% 0% 0% 0% 0% 6% (1) 1 1% n = 10 11 6 14 6 5 8 10 16 86 UHRF1-GFP High Normal 0% 0% 7% (1) 0% 0% 1 2% Atypical cells 100% (13) 100% (7) 46% (6) 17% (3) 75% (3) 36 61% Dysplastic foci 0% 0% 8% (1) 12% (2) 25% (1) 4 7% Tumor 0% 0% 46% (6) 76% (13) 0% 18 30% n = 13 7 13 17 4 54 UHRF1-GFP High/ p53 +/− Normal 0% 0 0% Atypical cells 13% (2) 2 13% Dysplastic foci 0% 0 0%\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 29\n\n\ntransgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI\n\n\nTumor 88% (14) 14 88% n = 16 16\n\n\nThe incidence (percent) and absolute number of fish (parenthesis) with cancer-relevant histological phenotypes were scored in a total of 281 fish from 4 transgenic lines. The total n for each time point is indicated. nd: not done. Some fish were diagnosed with more than one lesion. CI: Cumulative incidence.\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/start] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] Entering Prompt run with input:
{
  "section_text": "UHRF1 overexpression drives DNA hypomethylation and\nhepatocellular carcinoma\nRaksha Mudbhary a,b,c , Yujin Hoshida a,d,l , Yelena Chernyavskaya a,b,l , Vinitha Jacob a,b,c , Augusto Villanueva f , M. Isabel Fiel e , Xintong Chen a,d , Kensuke Kojima a,d , Swan Thung e ,\nRoderick T. Bronson g , Anja Lachenmayer d , Kate Revill d , Clara Alsinet f , Ravi Sachidanandam h , Anal Desai i , Sucharita SenBanerjee i , Chinweike Ukomadu i , Josep M. Llovet a,d,f,j , and Kirsten C. Sadler a,b,c,k\na Department of Medicine/Division of Liver Diseases, Icahn School of Medicine at Mount Sinai New York, NY 10029\nb Department of Developmental and Regenerative Biology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nc Graduate School of Biological Sciences, Icahn School of Medicine at Mount Sinai New York, NY 10029\nd Liver Cancer Program/Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai New York, NY 10029\ne Department of Pathology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nf HCC Translational Research Laboratory, IDIBAPS, CIBEREHD, Hospital Clinic, University of Barcelona, Catalonia, Spain\ng Rodent Histopathology Core Dana Farber/Harvard Cancer Center, Harvard Medical School Boston, MA 02115\nh Department of Genetics and Genomics, Icahn School of Medicine at Mount Sinai New York, NY 10029\nI Division of Gastroenterology, Department of Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115\nj Institució Catalana de Recerca i Estudis Avançats Lluís Companys, 23, Barcelona – Spain\n\n\nSUMMARY\nUHRF1 is an essential regulator of DNA methylation that is highly expressed in many cancers. Here, we use transgenic zebrafish, cultured cells and human tumors to demonstrate that UHRF1 is an oncogene. UHRF1 overexpression in zebrafish hepatocytes destabilizes and delocalizes DNMT1, causes DNA hypomethylation and Tp53-mediated senescence. Hepatocellular carcinoma\n© 2014 Elsevier Inc. All rights reserved.\nk Corresponding author: Kirsten.edepli@mssm.edu, 1 Gustave L. Levy Place, Box 1020, Icahn School of Medicine at Mount Sinai, New York, NY 10029, phone: 212-241-7152, fax: 212-860-9279. lthese authors contributed equally to this work.\n\n\nAccession Number\nData from RNAseq was deposited with accession number GSE52605.\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\nPublished in final edited form as:\nCancer Cell. 2014 February 10; 25(2): 196–209. doi:10.1016/j.ccr.2014.01.003.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n(HCC) emerges when senescence is bypassed. tp53 mutation both alleviates senescence and accelerates tumor onset. Human HCCs recapitulate this paradigm, as UHRF1 overexpression defines a subclass of aggressive HCCs characterized by genomic instability, TP53 mutation and abrogation of the TP53-mediated senescence program. We propose that UHRF1 overexpression is a mechanism underlying DNA hypomethylation in cancer cells and that senescence is a primary means of restricting tumorigenesis due to epigenetic disruption.\n\n\nINTRODUCTION\nThe expression of genes that encode readers and writers of the epigenetic code are widely deregulated across cancer types (You and Jones, 2012). This contributes to the massive gene expression changes and remodeling of the epigenetic landscape, a characteristic of many types of cancer. In particular, loss of global DNA methylation is a hallmark of cancer cells.\nDNA hypomethylation contributes to oncogenesis through multiple mechanisms, including chromosomal instability (Eden et al., 2003; Karpf and Matsui, 2005), derepression of imprinted genes (Berdasco and Esteller, 2010; Jirtle, 2004; Li et al., 1993), retrotransposon activation (Gaudet et al., 2004; Howard et al., 2008; Jackson-Grusby et al., 2001; Sharif et al., 2007) and aberrant gene expression, including induction of oncogenes (Cheah et al., 1984). Many studies have documented that expression of the core factors required for maintenance DNA methylation – i.e. DNA methyltransferase 1 (DNMT1) and ubiquitin-like with PHD and RING finger domains 1 (UHRF1) (Babbio et al., 2012; Jin et al., 2009; Unoki et al., 2010; Wang et al., 2012) – are significantly altered across cancer types. However, whether changes in the expression of these key factors are sufficient to alter the cancer cell methylome and drive carcinogenesis is unknown. Moreover, the mechanism by which DNA methylation is lost in cancer cells is poorly understood.\nThe cellular response to DNA hypomethylation varies by cell type, physiological context and degree of hypomethylation. In some cells, DNA hypomethylation induces tumor suppressive mechanisms, including apoptosis (Anderson et al., 2009; Biniszkiewicz et al., 2002; Chen et al., 2007; Jackson-Grusby et al., 2001) or senescence (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), whereas in other cells it blocks differentiation of progenitor cells (Rai et al., 2010) or causes cancer (Gaudet et al., 2003; Yamada et al., 2005). In part, the cellular response to loss of DNA methylation is dictated by the genomic region affected: hypomethylation of gene regulatory regions, such as promoters, can de-repress gene expression but hypomethylation of repetitive elements can reduce heterochromatin formation, promote recombination and genomic instability.\nUHRF1 plays an essential role in DNA methylation by both recognizing hemimethylated DNA generated during DNA replication and then recruiting DNMT1 to ensure faithful maintenance of DNA methylation patterns in daughter cells (Arita et al., 2008; Avvakumov et al., 2008; Bostick et al., 2007; Hashimoto et al., 2008; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Consequently, UHRF1 depletion results in global DNA hypomethylation (Bostick et al., 2007; Feng et al., 2010; Sharif et al., 2007; Tittle et al., 2011). Conversely, UHRF1 may also limit DNA methylation by targeting DNMT1 for ubiquitin-mediated degradation (Du et al., 2010; Qin et al., 2011) or by delocalizing DNMT1 (Sharif et al., 2007). How UHRF1 overexpression impacts the methylome is not known.\nWe previously reported that uhrf1 mutation in zebrafish blocks liver outgrowth in embryos and regeneration in adults (Sadler et al., 2007) and that depleting UHRF1 from cancer cells induces apoptosis (Tien et al., 2011). Here, we tested the hypothesis that UHRF1\n\n\nNIH-PA Author Manuscript\noverexpression would alter global DNA methylation, and promote hepatocellular carcinoma (HCC).\n\n\nRESULTS\n\n\nHigh UHRF1 expression causes DNA hypomethylation\nUHRF1 is required for DNA methylation, as it recruites DNMT1 to hemimethylated DNA during DNA replication (Bostick et al., 2007; Feng et al., 2010; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Paradoxically, UHRF1 also serves as an ubiquitin ligase that targets DNMT1 for degradation (Du et al., 2010; Qin et al., 2011). To determine how UHRF1 overexpression impacts DNA methylation, we generated transgenic zebrafish expressing human UHRF1 fused to GFP (hsa.UHRF-GFP) under the hepatocyte specific fabp10 promoter (Tg(fabp10:hsa.UHRF1-GFP) (Chu et al., 2012)).\nHuman and zebrafish UHRF1 are 66% identical and the ability of human UHRF1 to rescue zebrafish embryos depleted of uhrf1 (VJ and KCS, unpublished and (Chu et al., 2012)) indicates they are functional orthologs. Expression is first detected in hepatocytes on 3 dpf and 5 dpf, nuclear UHRF1-GFP is easily detected at variable levels in hepatocytes (Figure S1A and (Chu et al., 2012)). We isolated an allelic series of Tg(fabp10:hsa.UHRF1-GFP) transgenics expressing a range of UHRF1 levels (Figure S1A–B; hereafter referred to UHRF1-GFP High, Medium and Low).\nWe probed genomic DNA isolated from the liver of 5 dpf larvae from each line with an antibody specific for 5-methyl cytosine (5MeC) to assess DNA methylation and found significant hypomethylation only in the liver of UHRF1-GFP High larvae (38% of controls; p=0.007; Figure 1A and Figure S1C). 5MeC immunofluorescence showed DNA methylation as uniformly distributed in the nuclei of hepatocytes from control larvae that express nuclear-localized fluorescent mCherry under the fabp10 promoter (Tg(fabp10:nls-mCherry); abbreviated to nls-mCherry) but only dim staining in hepatocytes of UHRF1-GFP High larvae (Figure 1B). Since Equivalent levels of 5MeC were detected in the liverless carcasses (Figure 1A) or fins (Figure 1B), we conclude that DNA hypomethylation was specific to the liver.\nWe hypothesized that UHRF1-induced DNA hypomethylation could be caused by mislocalization or destabilization of Dnmt1. Immunofluorescence revealed uniform Dnmt1 distribution in the nucleoplasm of nls-mCherry hepatocytes (Figure 1C) but in UHRF1-GFP high hepatocytes, it was concentrated in nuclear foci that contained UHRF1-GFP (Figure 1C). Interestingly, the range of UHRF1-GFP expression within cells of the same liver revealed that in cells with high GFP, Dnmt1 was dim and punctate, but levels and distribution pattern was similar to control hepatocytes in those cells with low or no GFP (arrows, Figure 1C). Thus, high expression of UHRF1 co-localizes with Dnmt1 and redistributes it to intra-nuclear structures reminiscent of senescence associated heterochromatin foci (Di Micco et al., 2011).\nEndogenous Dnmt1 in the liver was below the levels detectable by immunoblotting (not shown), so we used a transgenic line expressing UHRF1-GFP under the inducible hsp70l promoter (Chu et al., 2012) to quantitatively assess the impact of UHRF1 overexpression on Dnmt1 stability at a developmental time when we could easily detect Dnmt1 levels by immunoblotting. We optimize a heat shock protocol that maximized UHRF1-GFP expression in Tg(hsp70l:UHRF1-GFP) larvae (Figure 1D) and found that UHRF1-GFP overexpression reduced Dnmt1 levels by 27% compared to non-heat-shocked transgenics (p=0.03; Figure 1E). Treatment with a non-toxic dose of the proteasome inhibitor, MG132, prevented the decrease in Dnmt1 induced by UHRF1 overexpression (p=0.01; Figure 1E).\n\n\nNIH-PA Author Manuscript\nNeither heat-shock nor MG132 significantly affected Dnmt1 protein levels in non-transgenic controls (Figure 1E). Thus, both Dnmt1 delocalization and destabilization could account for DNA hypomethylation caused by UHRF1 overexpression.\n\n\nDNA hypomethylation caused by UHRF1 overexpression reduces liver size\nUhrf1 in zebrafish is required for hepatic outgrowth and liver regeneration (Sadler et al., 2007). To determine if UHRF1 overexpression in hepatocytes affected hepatic outgrowth, we assessed liver size all transgenic lines (Figures 2A–C and S2A). Low and Medium expressing lines had no gross changes in liver size (Figure S2A), but 74% of UHRF1-GFP High larvae on 5 dpf had small livers (Figure 2C), with the median area of the left liver lobe reduced by 35% compared to controls (p&lt;0.001; Figures 2D, S2A), without reduction of total fish size (Figure S2B).\nLarvae with small livers appeared sick (see 10 dpf larvae in Figure 2A), and only 20% of UHRF1-GFP High fish survived to 20 dpf (Figure S2C and Figure 2E). Moreover, UHRF1- GFP High larvae with the “microliver” phenotype on 5 dpf had significantly higher mortality by 10 dpf (67%) than those that started with a normal sized liver (7%; Figure 2E). Thus, high UHRF1 expression in hepatocytes causes DNA hypomethylation, microliver and larval death.\nWe next investigated the relationship between DNA hypomethylation and the microliver phenotype by asking whether further reducing DNA methylation could enhance this phenotype or restoring Dnmt1 could suppress it. We previously reported that exposing embryos to 50 μ M of the Dnmt1 inhibitor, 5-azacytidine (5-Aza) from 0–5 dpf caused a profoundly small liver and DNA hypomethylation (Mudbhary and Sadler, 2011). By restricting the 5-Aza exposure time to 2.5–5 dpf, we reduced DNA methylation in the liver of control larvae by 40% (Figure S1C) which induced a moderately small liver in 16% of larvae (p=0.0035; Figure 2F). This same treatment of UHRF1-GFP High larvae significantly increased the percent with small livers (p&lt;0.0001; Figure 2F). While this could be attributed to DNA damage caused by 5-Aza, our finding that injecting UHRF1-GFP High embryos with mRNA encoding Dnmt1 modestly suppressed the percent of fish with a small liver (Figure 2G) and increased the average size of the left liver lobe by 10–15% (Figure S2D) suggests that hypomethylation, at least in part, contributes to the small liver phenotype of UHRF1-GFP High larvae.\n\n\nUHRF1 overexpression triggers Tp53-mediated senescence\nDNA hypomethylation can induce apoptosis, but we found no TUNEL positive cells on 5 dpf of UHRF1-GFP High livers (Figure S3A). However, senescence associated β - galactosidase (SA- β -gal) staining was detected throughout the liver of most UHRF1-GFP High 5 dpf larvae (n=93; Figure 3A–B), but not other transgenic lines (Figure S3B). Additionally, the DNA in control hepatocytes was evenly distributed throughout the uniformly sized nuclei compared to the large nuclei where DNA resembled senescence associated heterochromatic foci (inset, Figure 3C). Strikingly, the largest hepatocyte nuclei also had the brightest GFP (Figure 3C), suggesting the effect of UHRF1 overexpression cell autonomously affected nuclear morphology.\nSenescent cells do not divide, and we found significantly less BrdU incorporation in the liver of 5 dpf UHRF1-GFP High larvae compared to controls (p&lt;0.0001; Figure 3D). Interestingly, in nls-mCherry larvae, most BrdU incorporation was detected in hepatocytes that express nls-mCherry, but the only BrdU positive cells in UHRF1-GFP high larvae were negative for GFP (see inset in Figure 3D). RNAseq revealed some pro-proliferative genes (ccnd1 and myc) downregulated (Figure 3E), lending further support to the conclusion that\nsenescence is the primary response to high UHRF overexpression in hepatocytes during hepatic outgrowth.\nTp53 is a key mediator of senescence caused by DNA damage and oncogenic stress (Di Micco et al., 2011; McDuff and Turner, 2011; Ventura et al., 2007; Xue et al., 2007). RNAseq (Figure 3E) and qPCR analysis (Figure 3F) show that tp53 and its target genes, especially cdkn1a, are significantly induced in the liver of 5 dpf UHRF1-GFP High larvae, but then return to baseline by 20 dpf (Figure 3F). 5-Aza treatment of primary mouse hepatocytes induced Tp53 expression (Figure 3G), similar to the effects of 5-Aza treatment or Dnmt1 depletion in other cell types (Jackson-Grusby et al., 2001; Karpf et al., 2001) and in zebrafish (VJ and KCS, not shown). However, since no significant alteration in methylation of the tp53 promoter was detected in these models (not shown), we hypothesize that DNA methylation does not directly regulate Tp53 expression. Instead, DNA hypomethylation may induce tp53 by an indirect mechanism, such as increased DNA damage or genomic instability.\nA direct role for Tp53 in the phenotype of UHRF1-GFP High larvae was demonstrated by removing one copy of tp53. This reduced the incidence and intensity of SA- β -gal staining in the liver (Figure 3H), increased liver size (Figure 3I–J) and reduced mortality (Figure 3K). We thus propose a model whereby high UHRF1 causes DNA hypomethylation, induces Tp53-mediated senescence and prevents expansion of the hepatic bud, resulting in hepatic insufficiency and larval death.\n\n\nUHRF1 overexpression induces liver cancer in zebrafish\nTo determine if UHRF1 overexpression was sufficient to cause HCC, 281 control and UHRF1-GFP transgenics were collected between 5–300 dpf, serial sectioned and analyzed for atypical cells, dysplastic foci and HCC using histological criteria devised by two expert pathologists (RTB and MIF) based on disrupted tissue architecture, cell size, shape, nuclear structure and presence of mitotic figures (Figure S4A–B). Evidence of increased hepatocyte proliferation was detected in all UHRF1 overexpressing lines on 20 and 40 dpf (Figure S4C), but this was insufficient to cause HCC, as UHRF1-GFP Low fish were tumor free at all time points (Table 1). In contrast, UHRF1-GFP High fish developed atypical hepatocytes as early as 5 dpf, with an 8% incidence of dysplastic foci and 46% incidence of HCC by 15 dpf. On 20 dpf, 76% of fish had HCC (Figure 4A–B, Table 1). UHRF1-GFP Medium fish also developed atypical hepatocytes and dysplastic foci at young ages, and one large HCC was detected in a 60 dpf fish (Table 1). In a classical transformation assay using NIH-3T3, we found that UHRF1 cooperated with RAS to promote growth on soft agar (Figure 4C). These conclusively demonstrate that UHRF1 is an oncogene.\nWe found that UHRF1-GFP High larvae older than 8 dpf had a lower incidence and intensity of hepatic SA- β -gal staining (Figure 4D). Interestingly, in many of these fish, intense staining was distributed in a punctate pattern. By 20 dpf, 75% of fish had either punctate or no staining (Figure 4D), which is a striking correlation with the 70% incidence of HCC at this time point. This was mirrored by increased proliferation of liver cells, detected by increased BrdU incorporation in UHRF1-GFP High livers (22% vs. 4% in controls) at 11 dpf (p&lt;0.001; Figure 4E), and higher PCNA staining 20 and 40 dpf in all lines (Figure S4C). Loss of senescence was not attributed to reestablishment of DNA methylation or transgene silencing, as reduced 5MeC staining persisted in tumor cells (Figure S4D) and transgene expression was detectable in all 20 dpf fish (Figures S4E–F), albeit reduced from levels detected in 5 dpf livers.\nWe asked whether Tp53 epistatically interacted with UHRF1 overexpression to contribute to HCC by removing one copy of tp53 (Berghmans et al., 2005). The liver appeared normal in\n\n\nNIH-PA Author Manuscript\n13 tp53 +/+ fish without transgene expression (not shown), but in UHRF1-GFP High fish, tumor incidence on 15 dpf increased from 50% in WT to 87% in tp53 +/− fish (Figure 4B). Interestingly, in a single Tg(fabp10:UHRF1-GFP HIGH );p53 +/− , we found a tumor with immature cells resembling a cholangiocarcinoma. Thus, tp53 functions to suppress tumor formation and may alter the spectrum of tumors caused by UHRF1 overexpression.\n\n\nUHRF1 is upregulated in human HCC\nWe next investigated the relevance of UHRF1 expression in human HCC. We assessed UHRF1 expression by qPCR in 16 normal liver samples and in 2 cohorts of patients with dysplastic nodules or HCC: the 1 st cohort of 58 patients had hepatitis C infection (Figure 5A; (Wurmbach et al., 2007)) and the 2 nd cohort of 69 patients had HBV, alcohol and other etiologies (Figure S5A; (Villanueva et al., 2008)). Additional publically available transcriptome data sets from three other cohorts of HCCs (Figure S5B) and from lung, gastric, colorectal and breast cancer (Figure S5C) were also analyzed. All showed elevated UHRF1 in tumors, with expression elevated &gt;2-fold compared to controls in 17/18 dysplastic foci and 104/109 HCCs (Figures 5A and S5A). An average of 20- and 46-fold overexpression of UHRF1 was detected in advanced and very advanced HCCs. UHRF1 protein was barely detectable in 5 normal liver samples but highly expressed in 38/52 (73%) of the tumors analyzed for UHRF1 mRNA in Figure 5A (p&lt;0.003; Figure 5B). Thus, UHRF1 is overexpressed in HCCs of diverse etiologies as well as in other tumor types, and high UHRF1 expression is significantly associated with the most advanced tumors.\nTargeting UHRF1 in liver cancer cells (HepG2) using siRNA induced PARP cleavage (Figure 5C) and other markers of apoptosis (not shown), similar to results obtained using another HCC cell line (Hep3B; not shown) and colon cancer cells (Tien et al., 2011). Thus, blocking UHRF1 cancers such as HCC with high UHRF1 expression could be an effective means to induce tumor cell death.\nOf the 109 HCCs analyzed for UHRF1 expression by qPCR, 71 had multiple clinical and genomics parameters available (Chiang et al., 2008). These were rank-ordered based on UHRF1 expression determined by qPCR (Figure 6A). Tumors with expression above and equal to or below the median log2-fold change value of 3.64 were designated as “high” (n=35) and “low” (n=36), respectively (Figure 6A). High UHRF1 expressing tumors were associated with signs of poor clinical outcome: 80% had microvascular invasion (Figure 5D) and significantly higher alpha-feto protein (AFP) levels (Figure 5E), although AFP levels alone did not cooperate with UHRF1 to predict survival (not shown). Importantly, high UHRF1 expression significantly correlated with early (&lt;2 years; Figure 5F), but not late (Figure 5G) tumor recurrence and was inversely correlated with survival (Figure 5H). This suggests that high UHRF1 expression predicted recurrence of the primary tumor, causing decreased survival (Villanueva et al., 2011). Molecular signatures of aggressive HCC tumors compiled from several previous studies (Table S1) were concordantly and significantly enriched in high UHRF1 expressing tumors (Figure 6A). Moreover, high UHRF1 expressing tumors were distinguished by induction of pathways that drive the cell cycle, DNA replication and repair (Figure S6A, Tables S2 and S3). UHRF1 overexpression also significantly correlated with advanced stage prostate cancer, but not lung or colon (Figure S5D–G).\nHigh UHRF1 expression delineates a subclass of HCCs that have downregulated TP53- mediated senescence\nOur zebrafish studies demonstrated that bypass of Tp53-induced senescence is required for UHRF1 to act as an oncogene. Our analysis of human HCCs indicates that a similar paradigm occurs in these samples. First, inactivating mutations in TP53 significantly\n\n\nNIH-PA Author Manuscript\ncorrelated with high UHRF1 expression (Figure 6A). Second, many of the core enriched genes high UHRF1 expressing tumors are regulated by TP53 in senescent fibroblasts (Table S4 and (Tang et al., 2007), and high UHRF1 expressing tumors have downregulated the gene expression signature derived from TP53-induced senescence in fibroblasts (Table S5, Figure S6B; (Tang et al., 2007)) or in hepatic stellate cells (Table S6, Figure S6C; (Lujambio et al., 2013)). Fourth, significant correlation between UHRF1 overexpression, TP53 mutation and genome integrity in human HCCs indicates that these pathways act together: high UHRF1 expression (Figure 6B–C) and TP53 mutation (Figure 6D) are independently correlated with chromosomal loss but tumors that have both features display even more chromosomal loss (Figure 6D). Finally, UHRF1 expression was significantly higher in tumors with TP53 mutation (Figure 6E). UHRF1 overexpression did not correlate with copy number variation at the UHRF1 locus (Figure S6D) suggesting that a different mechanism drives UHRF1 overexpression.\nGenome wide DNA hypomethylation is found in most HCCs (Calvisi et al., 2007) and this rendered it difficult to correlate methylome changes with UHRF1 expression. However, DNMT1 expression was directly correlated with UHRF1 expression in HCC samples (Figure 6F). This may be a consequence of the high proliferation rate in these tumors or an induction of the methylation machinery to compensate for hypomethylation. Together, these data indicate that high UHRF1 expression in HCC defines a subset of aggressive tumors that have inactivated the TP53 induced senescence program, suggesting that in humans, as observed in zebrafish, TP53 acts as a tumor suppressor to restrict the oncogenic potential of UHRF1.\n\n\nDISCUSSION\nWe show that UHRF1 overexpression is sufficient to cause two oncogene-associated phenotypes: senescence and cancer. This defines UHRF1 as an epigenetic regulator that can cause cancer when overexpressed and is thus an oncogene. We hypothesize that UHRF1 overexpression is one mechanism by which cancer genomes can become hypomethylated, either via UHRF1-mediated Dnmt1 ubiquitination and degredation (Du et al., 2010; Qin et al., 2011) or by redistribution and/or sequestration of Dnmt1 away from DNA. Excess UHRF1 might also sequester the DNMT1 deubiquitinating enzyme, USP7 (Qin et al., 2011) to further promote DNMT1 ubiquitination and degradation.\nThe precise mechanism by which DNA hypomethylation causes cancer remains elusive. Both apoptosis and senescence serve to limit the propagation of cells with aberrant DNA methylation (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), however, once epigenetically altered cells escape these tumor suppressive mechanisms, they likely accumulate genetic lesions which predispose to cancer. Indeed, chromosomal instability and mitotic catastrophe occur following DNMT1 depletion (Chen et al., 2007; Karpf and Matsui, 2005; Weber and Schubeler, 2007) and transposons, which are heavily methylated in normal cells, could become activated and cause genomic instability upon UHRF1-induced DNA hypomethylation. Additionally, hypomethylated DNA is more likely to assume an open chromatin conformation, which may promote oncogene expression, although this possibility has not been fully evaluated. While the oncogenic role of UHRF1 could also be mediated by the impact of UHRF1 on other epigenetic marks or on DNA replication (Taylor et al., 2013), our data combined with findings from others (Eden et al., 2003; Gaudet et al., 2003) suggests that DNA hypomethylation is a likely mechanism driving UHRF1-mediated transformation.\nOur working model (Figure 7) proposes that UHRF1 overexpression causes DNA hypomethylation by reducing Dnmt1 levels and its access to DNA. Tp53 is induced in\n\n\nNIH-PA Author Manuscript\nresponse to either genomic instability or some unidentified direct mechanism, and causes hepatocyte senescence, which prevents hepatic expansion and results in larval death from hepatic insufficiency. We propose that Tp53 inactivation and senescence bypass allows for the unhindered proliferation and malignant transformation. Identifying the mechanism by which this tumor suppressive mechanism is overcome remains a central, unanswered question in cancer biology.\nStudies in a mouse liver cancer model show that Tp53 reactivation causes senescence, and these cells are then cleared by the immune system (Xue et al., 2007) and the liver is then repopulated with senescence-resistant tumor forming cells. Our finding that senescence decreases and BrdU incorporation increases in UHRF1-GFP high livers over time suggests a similar process at play. Moreover, BrdU incorporation in primarily GFP negative cells suggests the expansion of either immune cells or immature hepatic progenitors in response to senescent hepatocytes. The finding of a cholangiocarcinoma in a UHRF1-GFP High/ p53 +/− fish may indicate that a bipotential progenitor cell could be the tumor forming cell in this model. Additionally, our data indicate that there is a threshold effect of UHRF1 expression, in which the highest expressing cells undergo senescence and neighboring hepatocytes expressing UHRF1-GFP at levels below those detectable via microscopy undergo unhindered expansion.\nHCC is the third cause of cancer-related deaths globally (Llovet et al., 2003), yet curative therapies are limited, with Sorafanib as the only systemic therapy available for advanced cases (Llovet et al., 2008). Thus, there is an urgent and unmet need for novel therapies. HCC, like other cancers, is characterized by global DNA hypomethylation (Calvisi et al., 2007; Pogribny and Rusyn, 2012; Tischoff and Tannapfe, 2008), and high UHRF1 expression could be the cause. Our finding that UHRF1 depletion in HCC and other types of cancer cells causes apoptosis (Tien et al., 2011) presents UHRF1 as an attractive target for cancer therapy, whereby blocking UHRF1 could trigger massive epigenetic changes incompatible with cell survival or could reset the cancer cell methylome.\n\n\nEXPERIMENTAL PROCEDURES\n\n\nZebrafish maintenance and generation of transgenics\nZebrafish were maintained on a 14:10 hour light:dark cycle at 28°C. mRNA encoding zebrafish Dnmt1 (Rai et al., 2006) or mannose phosphate isomerase (Mpi; (Chu et al., 2013) as a control was injected into embryos just after fertilization.\nTg(fabp10:nls-mCherry) fish expressing nls-mCherry exclusively in hepatocytes were generated using Gateway cloning (Invitrogen) to produce vectors with tol2 transposon sites (Kwan et al., 2007). Tg(hsp70l:hsa.UHRF1-GFP) and Tg(fabp10:hsa.UHRF1-GFP) were described in (Chu et al., 2012). The high, medium and low expressing alleles are listed in ZFIN.org with superscripts mss1a, mss1b and mss1c, respectively. Transgenics were outcrossed to Tab14 (WT) or tp53 −/− fish (Berghmans et al., 2005).\nTg(hsp70l:UHRF1-GFP) embryos were heat-shocked at 37°C for 1 hour at 24 and 27 hours post fertilization (hpf). At 28 hpf, embryos were sorted visually for GFP expression and incubated with either 10 μ M MG132 or DMSO and collected for immunoblotting at 34 hpf. 5-Aza (50 μ M) was added to larvae from 2.5–5 dpf. The Mount Sinai Institutional Animal Care and Use Committee approved all protocols. Nomenclature guidelines for the species under discussion were followed, and when no species is specified, human nomenclature was used.\n\n\nNIH-PA Author Manuscript\n\n\nGene expression analysis\nRNA was isolated from a pool of at least 10 livers from 5 dpf fish and from 1–5 livers from 20 dpf fish using the RNeasy mini-kit (Qiagen). cDNA was prepared by polyA priming using qScript SuperMix (Quanta). qRT-PCR analysis was performed in the Light Cycler 480 (Roche) using gene-specific primers (see supplemental materials) and PerfeCTa SYBRGreen FastMix (Quanta). C t values from triplicate reactions were averaged and 2 -Ct(target) /2 -Ct(reference) was used to calculate expression, with rpp0 and cyclophilin A used as references genes zebrafish and mouse samples, respectively.\nRNAseq analysis was carried out on RNA from pools of 50 livers dissected from 2 clutches of 5 dpf UHRF1-GFP High and nls-mCherry larvae, described in Supplemental Material.\n\n\nHistology\nFish younger than 20 dpf were fixed overnight at 4°C in 4% paraformaldehyde (PFA) and older fish were fixed for 2–5 days at room temperature in Bouins fixative. 4 μ M serial sections of paraffin embedded fish were stained with hematoxylin and eosin as described (Imrie and Sadler, 2010) and imaged on an Olympus BX41 Clinical Microscope equipped with a Nikon DS-Ri1 digital camera. Histological criteria used for scoring tumors are described in Figure S4A–B.\nImmunoblotting\nLysates prepared from 15 embryos dissolved in 150 μ l protein lysis buffer were homogenized by sonication. 1 embryo equivalent was loaded per lane of an 8 or 12% SDS gel, transferred to nitrocellulose and blotted. HepG2 cell lysates were prepared as described (Tien et al., 2011).\n\n\nAntibodies\nAntibodies recognizing DNMT1 (1:1000 for blotting, 1:10 for immunofluorescence, Santa Cruz), UHRF1 (immunoblotting: 1:1000; BD Biosciences. Immunohistochemistry: 1:50; ab57083; Abcam), Tubulin (1:5000 DSHB), p89 PARP and total PARP (1:1000; Cell Signaling) and b-actin (1:2000; Sigma), 5MeC (1:500; Eurogentec), BrdU (1:200; BD Bioscences) and anti-rabbit or mouse conjugated to Alexa 555 or Alexa 488 (1:100; Invitrogen) were diluted in 10% FBS or 2% BSA in 1% Triton in PBS (PBST).\nZebrafish staining\nLarvae fixed in 4% PFA were washed in PBST and stained using the Senescence β - Galactosidase Staining Kit (Cell Signaling) or stained with CY3-streptavadin (CY3-SA; 1:300; Sigma) as described (Sadler et al., 2005). The left liver lobe area was measured using ImageJ.\nImmunofluorescence was carried out on whole fish or on livers dissected from fixed larvae. BrdU was added to larvae water (10 mM) for 4–6 hours followed by immediate fixation, dehydration in methanol, rehydration to PBST and permeablization with 10ug/ml Proteinase K. DNA was denatured in 1 or 2N HCl, renatured and blocked in 10% FBS or 1% BSA in PBST prior to immunofluorescence. DNA was stained with Hoechst 33342 (Sigma). A Leica SP5 DM confocal microscope was used for imaging.\n\n\nSlot blots\nGenomic DNA was denatured in 0.4M NaOH at 95°C for 10 minutes, neutralized in an equal volume of cold 2M ammonium acetate and 100 ng DNA was blotted in duplicate onto nitrocellulose membrane using a slot blot apparatus and washed in 2x SSC and vacuum\n\n\nNIH-PA Author Manuscript\nbaked at 80°C for 2 hours. Half was stained with 0.2% methylene blue in 0.3 M NaOAc and the other with anti-5MeC followed by HRP-conjugated anti-mouse (1:2000) and visualized by Chemilumiescence (Roche). Image J was used to quantify 5MeC and methylene blue intensity and 5MeC levels were determined by normalizing to total DNA.\n\n\nCell culture\nPrimary mouse hepatocytes were isolated and plated in triplicate at 50% confluency then treated with 5 μ m 5-Aza or DMSO for 24 hours and RNA was isolated.\nNIH-3T3 cells were transfected in triplicate with pCDNA3.1 lacking an insert (control) or containing human RAS, UHRF1 or both and were re-transfected 24 hours later. 48 hours after the second transfection, 700 mg /ml Neomycin was added for 7 days. 10,000 cells per condition were plated on 0.3% soft agar layered on top of 0.6% soft agar. Media was changed every 3 rd day for 2 weeks and plates were stained with 0.005% crystal violet and colony number was counted.\nHepG2 cells cultured in DMEM supplemented with 10% (v/v) FBS and 5% (w/v) penicillin- streptomycin were transfected twice, 24 hours apart, with 20 nM siRNA targeting firefly luciferase (GL2; Dhamarcon) or UHRF1 targeting si-A and si-B using RNAiMAX (Invitrogen) as described (Tien et al., 2011).\n\n\nHuman tissue samples\nPathologically staged human tumors, dysplastic foci and normal liver samples were obtained from the HCC Genomic Consortium (Mount Sinai Hospital, NY; Instituto Nazionale dei Tumori, Milan and Hospital Clinic, Barcelona). The study was approved by the IRB of each institution and informed consent was obtained from all participants. TaqMan Probes were used to analyze UHRF1 expression by qPCR as described (Villanueva et al., 2008). Samples were grouped into a training set (9 normal liver, 18 low- and high-grade dysplastic nodules, and 40 pathologically staged HCCs; (Llovet et al., 2006; Wurmbach et al., 2007)) and a validation set (7 normal liver and 69 HCC; (Villanueva et al., 2008)). Integrative genomic and clinical data analysis was performed on 71 of the hepatitis C-related, surgically treated HCC patients (NCBI Gene Expression Omnibus, accession numbers GSE9829 and GSE44970). Liver pathologist, ST, scored UHRF1 staining of 52 paraffin embedded tumors from the training set.\n\n\nBioinformatics and statistical analysis\n71 patients with HCC were grouped based on the median expression level of UHRF1 in the tumors determined by qPCR (UHRF1-high; n=35 or low; n=36). Presence of aggressive human HCC gene signatures (Table S1) were evaluated in the transcriptome dataset using nearest template prediction method (Hoshida, 2010) implemented by the GenePattern genomic analysis toolkit ( www.broadinstitute.org/genepattern ) based on prediction confidence p&lt;0.05. The senescence-related TP53 target genes were obtained from Molecular Signature Database (MSigDB, www.broadinstitute.org/msigdb , TANG_SENESCENCE_TP53_TARGETS_UP and _DN; (Tang et al., 2007)) and from NCBI Gene Expression Omnibus (GSE39469; (Lujambio et al., 2013)). Mouse genes from normalized microarray data were converted to human orthologs based on a mapping table ( www.informatics.jax.org ). Differentially expressed genes between senescent and proliferative cells were identified by using Bayesian T-test implemented in Cyber-T software (molgen51.biol.rug.nl/cybert) at the significance threshold of Posterior Probability of Differential Expression &gt;0.998 after excluding less variable genes with coefficient of variation ≤ 0.1 across the samples. Expression pattern of each signature was assessed by nearest template prediction algorithm. Significance of induction or suppression of each\n\n\nNIH-PA Author Manuscript\nsignature was quantitatively measured by nominal p-value of cosine distance. Pearson correlation test determined the significance between induction/suppression of gene signatures with UHRF1 mRNA expression level as measured by qPCR. Statistical difference between groups was assessed by Wilcoxon rank-sum test and Fisher’s exact test for continuous and categorical variables, respectively. Bonferroni correction for multiple hypothesis testing was applied when appropriate.\nSurvival analysis was performed by using Kaplan-Meier estimator and log-rank test. All analyses were performed using R statistical package ( www.r-project.org ) and GenePattern genomic analysis tool kit ( www.broadinstitute.org/genepattern ). Gene expression levels and liver size were compared using Student’s T or Mann Whitney U tests\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nSupport was provided by The Breast Cancer Alliance and the March of Dimes (to KCS), a Pilot Project from the DFCI NCI Cancer Center (5P30CA006516-45; to CU), the NIH (5R01DK080789-02 to CU and KCS,\n1R01DK099558 to YH, 1R01DK076986 to JML, F30DK094503 to VJ and T32CA078207-14 for YC), the European Commission 7th Framework Programme FP7-Health 2010 (Heptromic #259744 to YH and JML), The Samuel Waxman Cancer Research Foundation, The Spanish National Health Institute (SAF-2010-16055) and the Asociación Española Contra el Cáncer (JML). Liz Loughlin, Brandon Kent, Meghan Walsh, Alex Mir, Laia Cabellos, Helena Cornellà Vives and Sara Toffanin provided expert technical assistance. We are grateful to Sam Sidi for tp53−/− fish, stimulating discussions and critical reading of the manuscript.\n\n\nReferences\nAnderson RM, Bosch JA, Goll MG, Hesselson D, Dong PD, Shin D, Chi NC, Shin CH, Schlegel A, Halpern M, Stainier DY. Loss of Dnmt1 catalytic activity reveals multiple roles for DNA methylation during pancreas development and regeneration. Dev Biol. 2009\nArita K, Ariyoshi M, Tochio H, Nakamura Y, Shirakawa M. Recognition of hemi-methylated DNA by the SRA protein UHRF1 by a base-flipping mechanism. Nature. 2008; 455:818–821. [PubMed: 18772891]\nAvvakumov GV, Walker JR, Xue S, Li Y, Duan S, Bronner C, Arrowsmith CH, Dhe-Paganon S. Structural basis for recognition of hemi-methylated DNA by the SRA domain of human UHRF1. Nature. 2008; 455:822–825. [PubMed: 18772889]\nBabbio F, Pistore C, Curti L, Castiglioni I, Kunderfranco P, Brino L, Oudet P, Seiler R, Thalman GN, Roggero E, et al. The SRA protein UHRF1 promotes epigenetic crosstalks and is involved in prostate cancer progression. Oncogene. 2012\nBerdasco M, Esteller M. Aberrant epigenetic landscape in cancer: how cellular identity goes awry. Dev Cell. 2010; 19:698–711. [PubMed: 21074720]\nBerghmans S, Murphey RD, Wienholds E, Neuberg D, Kutok JL, Fletcher CD, Morris JP, Liu TX, Schulte-Merker S, Kanki JP, et al. tp53 mutant zebrafish develop malignant peripheral nerve sheath tumors. Proc Natl Acad Sci U S A. 2005; 102:407–412. [PubMed: 15630097]\nBiniszkiewicz D, Gribnau J, Ramsahoye B, Gaudet F, Eggan K, Humpherys D, Mastrangelo MA, Jun Z, Walter J, Jaenisch R. Dnmt1 overexpression causes genomic hypermethylation, loss of imprinting, and embryonic lethality. Mol Cell Biol. 2002; 22:2124–2135. [PubMed: 11884600]\nBostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE. UHRF1 plays a role in maintaining DNA methylation in mammalian cells. Science. 2007; 317:1760–1764. [PubMed: 17673620]\nCalvisi DF, Ladu S, Gorden A, Farina M, Lee JS, Conner EA, Schroeder I, Factor VM, Thorgeirsson SS. Mechanistic and prognostic significance of aberrant methylation in the molecular pathogenesis of human hepatocellular carcinoma. J Clin Invest. 2007; 117:2713–2722. [PubMed: 17717605]\n\n\nNIH-PA Author Manuscript\nCheah MS, Wallace CD, Hoffman RM. Hypomethylation of DNA in human cancer cells: a site- specific change in the c-myc oncogene. J Natl Cancer Inst. 1984; 73:1057–1065. [PubMed: 6092764]\nChen T, Hevi S, Gay F, Tsujimoto N, He T, Zhang B, Ueda Y, Li E. Complete inactivation of DNMT1 leads to mitotic catastrophe in human cancer cells. Nat Genet. 2007; 39:391–396. [PubMed: 17322882]\nChiang DY, Villanueva A, Hoshida Y, Peix J, Newell P, Minguez B, LeBlanc AC, Donovan DJ, Thung SN, Sole M, et al. Focal gains of VEGFA and molecular classification of hepatocellular carcinoma. Cancer Res. 2008; 68:6779–6788. [PubMed: 18701503]\nChu J, Loughlin EA, Gaur NA, Senbanerjee S, Jacob V, Monson C, Kent B, Oranu A, Ding Y, Ukomadu C, Sadler KC. UHRF1 phosphorylation by cyclin A2/cyclin-dependent kinase 2 is required for zebrafish embryogenesis. Mol Biol Cell. 2012; 23:59–70. [PubMed: 22072796]\nChu J, Mir A, Gao N, Rosa S, Monson C, Sharma V, Steet R, Freeze HH, Lehrman MA, Sadler KC. A zebrafish model of congenital disorders of glycosylation with phosphomannose isomerase deficiency reveals an early opportunity for corrective mannose supplementation. Dis Model Mech. 2013; 6:95–105. [PubMed: 22899857]\nDecottignies A, d’Adda di Fagagna F. Epigenetic alterations associated with cellular senescence: a barrier against tumorigenesis or a red carpet for cancer? Semin Cancer Biol. 2011; 21:360–366. [PubMed: 21946622]\nDi Micco R, Sulli G, Dobreva M, Liontos M, Botrugno OA, Gargiulo G, dal Zuffo R, Matti V, d’Ario G, Montani E, et al. Interplay between oncogene-induced DNA damage response and heterochromatin in senescence and cancer. Nat Cell Biol. 2011; 13:292–302. [PubMed: 21336312]\nDu Z, Song J, Wang Y, Zhao Y, Guda K, Yang S, Kao HY, Xu Y, Willis J, Markowitz SD, et al. DNMT1 stability is regulated by proteins coordinating deubiquitination and acetylation-driven ubiquitination. Sci Signal. 2010; 3:ra80. [PubMed: 21045206]\nEden A, Gaudet F, Waghmare A, Jaenisch R. Chromosomal instability and tumors promoted by DNA hypomethylation. Science. 2003; 300:455. [PubMed: 12702868]\nFairweather DS, Fox M, Margison GP. The in vitro lifespan of MRC-5 cells is shortened by 5- azacytidine-induced demethylation. Exp Cell Res. 1987; 168:153–159. [PubMed: 2430819]\nFeng S, Cokus SJ, Zhang X, Chen PY, Bostick M, Goll MG, Hetzel J, Jain J, Strauss SH, Halpern ME, et al. Conservation and divergence of methylation patterning in plants and animals. Proc Natl Acad Sci U S A. 2010; 107:8689–8694. [PubMed: 20395551]\nGaudet F, Hodgson JG, Eden A, Jackson-Grusby L, Dausman J, Gray JW, Leonhardt H, Jaenisch R. Induction of tumors in mice by genomic hypomethylation. Science. 2003; 300:489–492. [PubMed: 12702876]\nGaudet F, Rideout WM 3rd, Meissner A, Dausman J, Leonhardt H, Jaenisch R. Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. Mol Cell Biol. 2004; 24:1640–1648. [PubMed: 14749379]\nHashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X. The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. Nature. 2008; 455:826–829. [PubMed: 18772888]\nHoshida Y. Nearest template prediction: a single-sample-based flexible class prediction with confidence assessment. PLoS One. 2010; 5:e15543. [PubMed: 21124904]\nHoward G, Eiges R, Gaudet F, Jaenisch R, Eden A. Activation and transposition of endogenous retroviral elements in hypomethylation induced tumors in mice. Oncogene. 2008; 27:404–408. [PubMed: 17621273]\nImrie D, Sadler KC. White adipose tissue development in zebrafish is regulated by both developmental time and fish size. Dev Dyn. 2010; 239:3013–3023. [PubMed: 20925116]\nJackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G, Dausman J, Lee P, Wilson C, Lander E, Jaenisch R. Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. Nat Genet. 2001; 27:31–39. [PubMed: 11137995]\nJin W, Chen L, Chen Y, Xu SG, Di GH, Yin WJ, Wu J, Shao ZM. UHRF1 is associated with epigenetic silencing of BRCA1 in sporadic breast cancer. Breast Cancer Res Treat. 2009\nJirtle RL. IGF2 loss of imprinting: a potential heritable risk factor for colorectal cancer. Gastroenterology. 2004; 126:1190–1193. [PubMed: 15057757]\n\n\nNIH-PA Author Manuscript\nKarpf AR, Matsui S. Genetic disruption of cytosine DNA methyltransferase enzymes induces chromosomal instability in human cancer cells. Cancer Res. 2005; 65:8635–8639. [PubMed: 16204030]\nKarpf AR, Moore BC, Ririe TO, Jones DA. Activation of the p53 DNA damage response pathway after inhibition of DNA methyltransferase by 5-aza-2 ′ -deoxycytidine. Mol Pharmacol. 2001; 59:751–757. [PubMed: 11259619]\nKwan KM, Fujimoto E, Grabher C, Mangum BD, Hardy ME, Campbell DS, Parant JM, Yost HJ, Kanki JP, Chien CB. The Tol2kit: a multisite gateway-based construction kit for Tol2 transposon transgenesis constructs. Dev Dyn. 2007; 236:3088–3099. [PubMed: 17937395]\nLi E, Beard C, Jaenisch R. Role for DNA methylation in genomic imprinting. Nature. 1993; 366:362– 365. [PubMed: 8247133]\nLiu X, Gao Q, Li P, Zhao Q, Zhang J, Li J, Koseki H, Wong J. UHRF1 targets DNMT1 for DNA methylation through cooperative binding of hemi-methylated DNA and methylated H3K9. Nat Commun. 2013; 4:1563. [PubMed: 23463006]\nLlovet JM, Burroughs A, Bruix J. Hepatocellular carcinoma. Lancet. 2003; 362:1907–1917. [PubMed: 14667750]\nLlovet JM, Chen Y, Wurmbach E, Roayaie S, Fiel MI, Schwartz M, Thung SN, Khitrov G, Zhang W, Villanueva A, et al. A molecular signature to discriminate dysplastic nodules from early hepatocellular carcinoma in HCV cirrhosis. Gastroenterology. 2006; 131:1758–1767. [PubMed: 17087938]\nLlovet JM, Ricci S, Mazzaferro V, Hilgard P, Gane E, Blanc JF, de Oliveira AC, Santoro A, Raoul JL, Forner A, et al. Sorafenib in advanced hepatocellular carcinoma. N Engl J Med. 2008; 359:378– 390. [PubMed: 18650514]\nLujambio A, Akkari L, Simon J, Grace D, Tschaharganeh DF, Bolden JE, Zhao Z, Thapar V, Joyce JA, Krizhanovsky V, Lowe SW. Non-cell-autonomous tumor suppression by p53. Cell. 2013; 153:449–460. [PubMed: 23562644]\nMcDuff FK, Turner SD. Jailbreak: oncogene-induced senescence and its evasion. Cell Signal. 2011; 23:6–13. [PubMed: 20633638]\nMudbhary R, Sadler KC. Epigenetics, development, and cancer: Zebrafish make their mark. Birth Defects Res C Embryo Today. 2011; 93:194–203. [PubMed: 21671358]\nNishiyama A, Yamaguchi L, Sharif J, Johmura Y, Kawamura T, Nakanishi K, Shimamura S, Arita K, Kodama T, Ishikawa F, et al. Uhrf1-dependent H3K23 ubiquitylation couples maintenance DNA methylation and replication. Nature. 2013; 502:249–253. [PubMed: 24013172]\nPogribny IP, Rusyn I. Role of epigenetic aberrations in the development and progression of human hepatocellular carcinoma. Cancer Lett. 2012\nQin W, Leonhardt H, Spada F. Usp7 and Uhrf1 control ubiquitination and stability of the maintenance DNA methyltransferase Dnmt1. J Cell Biochem. 2011; 112:439–444. [PubMed: 21268065]\nRai K, Nadauld LD, Chidester S, Manos EJ, James SR, Karpf AR, Cairns BR, Jones DA. Zebra fish Dnmt1 and Suv39h1 regulate organ-specific terminal differentiation during development. Mol Cell Biol. 2006; 26:7077–7085. [PubMed: 16980612]\nRai K, Sarkar S, Broadbent TJ, Voas M, Grossmann KF, Nadauld LD, Dehghanizadeh S, Hagos FT, Li Y, Toth RK, et al. DNA demethylase activity maintains intestinal cells in an undifferentiated state following loss of APC. Cell. 2010; 142:930–942. [PubMed: 20850014]\nSadler KC, Amsterdam A, Soroka C, Boyer J, Hopkins N. A genetic screen in zebrafish identifies the mutants vps18, nf2 and foie gras as models of liver disease. Development. 2005; 132:3561–3572. [PubMed: 16000385]\nSadler KC, Krahn KN, Gaur NA, Ukomadu C. Liver growth in the embryo and during liver regeneration in zebrafish requires the cell cycle regulator, uhrf1. Proc Natl Acad Sci U S A. 2007; 104:1570–1575. [PubMed: 17242348]\nSharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA, Shinga J, Mizutani-Koseki Y, Toyoda T, Okamura K, et al. The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. Nature. 2007; 450:908–912. [PubMed: 17994007]\nTang X, Milyavsky M, Goldfinger N, Rotter V. Amyloid-beta precursor-like protein APLP1 is a novel p53 transcriptional target gene that augments neuroblastoma cell death upon genotoxic stress. Oncogene. 2007; 26:7302–7312. [PubMed: 17533371]\nTaylor EM, Bonsu NM, Price RJ, Lindsay HD. Depletion of Uhrf1 inhibits chromosomal DNA replication in Xenopus egg extracts. Nucleic Acids Res. 2013\nTien AL, Senbanerjee S, Kulkarni A, Mudbhary R, Goudreau B, Ganesan S, Sadler KC, Ukomadu C. UHRF1 depletion causes a G2/M arrest, activation of DNA damage response and apoptosis. Biochem J. 2011; 435:175–185. [PubMed: 21214517]\nTischoff I, Tannapfe A. DNA methylation in hepatocellular carcinoma. World J Gastroenterol. 2008; 14:1741–1748. [PubMed: 18350605]\nTittle RK, Sze R, Ng A, Nuckels RJ, Swartz ME, Anderson RM, Bosch J, Stainier DY, Eberhart JK, Gross JM. Uhrf1 and Dnmt1 are required for development and maintenance of the zebrafish lens. Dev Biol. 2011; 350:50–63. [PubMed: 21126517]\nUnoki M, Daigo Y, Koinuma J, Tsuchiya E, Hamamoto R, Nakamura Y. UHRF1 is a novel diagnostic marker of lung cancer. Br J Cancer. 2010; 103:217–222. [PubMed: 20517312]\nVentura A, Kirsch DG, McLaughlin ME, Tuveson DA, Grimm J, Lintault L, Newman J, Reczek EE, Weissleder R, Jacks T. Restoration of p53 function leads to tumour regression in vivo. Nature. 2007; 445:661–665. [PubMed: 17251932]\nVillanueva A, Chiang DY, Newell P, Peix J, Thung S, Alsinet C, Tovar V, Roayaie S, Minguez B, Sole M, et al. Pivotal role of mTOR signaling in hepatocellular carcinoma. Gastroenterology. 2008; 135:1972–1983. 1983 e1971–1911. [PubMed: 18929564]\nVillanueva A, Hoshida Y, Battiston C, Tovar V, Sia D, Alsinet C, Cornella H, Liberzon A, Kobayashi M, Kumada H, et al. Combining clinical, pathology, and gene expression data to predict recurrence of hepatocellular carcinoma. Gastroenterology. 2011; 140:1501–1512.e1502. [PubMed: 21320499]\nWang F, Yang YZ, Shi CZ, Zhang P, Moyer MP, Zhang HZ, Zou Y, Qin HL. UHRF1 Promotes Cell Growth and Metastasis Through Repression of p16(ink4a) in Colorectal Cancer. Ann Surg Oncol. 2012\nWeber M, Schubeler D. Genomic patterns of DNA methylation: targets and function of an epigenetic mark. Curr Opin Cell Biol. 2007; 19:273–280. [PubMed: 17466503]\nWurmbach E, Chen YB, Khitrov G, Zhang W, Roayaie S, Schwartz M, Fiel I, Thung S, Mazzaferro V, Bruix J, et al. Genome-wide molecular profiles of HCV-induced dysplasia and hepatocellular carcinoma. Hepatology. 2007; 45:938–947. [PubMed: 17393520]\nXue W, Zender L, Miething C, Dickins RA, Hernando E, Krizhanovsky V, Cordon-Cardo C, Lowe SW. Senescence and tumour clearance is triggered by p53 restoration in murine liver carcinomas. Nature. 2007; 445:656–660. [PubMed: 17251933]\nYamada Y, Jackson-Grusby L, Linhart H, Meissner A, Eden A, Lin H, Jaenisch R. Opposing effects of DNA hypomethylation on intestinal and liver carcinogenesis. Proc Natl Acad Sci U S A. 2005; 102:13580–13585. [PubMed: 16174748]\nYou JS, Jones PA. Cancer genetics and epigenetics: two sides of the same coin? Cancer Cell. 2012; 22:9–20. [PubMed: 22789535]\n\n\nHighlights\n• UHRF1 overexpression induces DNA hypomethylation\n• Liver specific overexpression of UHRF1 in zebrafish causes senescence and HCC\n• Human HCCs with high UHRF1 are aggressive and inactivate TP53-mediated senescence\nNIH-PA Author Manuscript\n\n\nSIGNIFICANCE\nGlobal DNA hypomethylation occurs in most types of cancer and can induce genomic instability and widespread changes in gene expression. UHRF1 is a key regulator of DNA methylation, and here we show that UHRF1 overexpression causes HCC in zebrafish without any other genetic alteration, demonstrating that it is an oncogene. High UHRF1 expression causes DNA hypomethylation and tp53-mediated senescence, which serves to restrict transformation of UHRF1 overexpressing cells. High UHRF1 expression in human HCC correlates with a poor prognosis, genomic instability, TP53 mutation and repression of the TP53-mediated senescence program. We conclude that UHRF1 is an oncogene that promotes widespread DNA hypomethylation, an epigenetic hallmark of cancer cells, and that UHRF1 overexpression drives tumorigenesis when senescence is bypassed.\nNIH-PA Author Manuscript\nFigure 1. High UHRF1 expression causes global DNA hypomethylation (A) 5MeC levels and total DNA stained with methylene blue were measured in 5 dpf control and UHRF1-GFP High livers (n=4) and liver-less carcasses (n=3). The ratio of 5MeC to total DNA was averaged and normalized to controls. Student’s T-test was used to determine p values. (B) Confocal stacks of livers (top) and fins (bottom) from 5 dpf nls-mCherry and UHRF1-GFP High larvae stained with anti-5MeC. Since a hepatocyte specific promoter was used for transgenesis, there was no transgene expression in the fin. (C) Dnmt1 is uniform in the hepatocyte nucleus of 4 dpf nls-mCherry larvae but is found in GFP-containing punctae in UHRF1-GFP High hepatocytes. Arrows point to cells that do not express GFP and have Dnmt1 distribution pattern similar to controls. (D) Tg(hsp70I:UHRF1-EGFP) and non- transgenic controls were heat shocked at 37° C for 1 hour at 24 and 27 hpf, treated with 10 μ M MG132 or DMSO at 28 hpf and collected at 34 hpf for immunoblotting. (E) Dnmt1 levels normalized to tubulin were averaged from 6 experiments. Student’s T-test was used to determine p values; n.s.: not significant; error bars represent SD. See also Figure S1.\n\n\nNIH-PA Author Manuscript\nFigure 2. UHRF1-induced hypomethylation reduces liver size (A) Individual larvae were imaged daily from 3–10 dpf. (B) 5 dpf UHRF1-GFP High larvae display a range of liver sizes scored as “normal” or “small”‘ (C) 3 clutches were scored according to criteria in B; n= number of larvae. Fisher’s exact test was used to determine p value. (D) The area of the left liver lobe was measured in 5 dpf fish from 2 clutches. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. Student’s T-test was used to determine p value. (E) UHRF1-GFP High larvae were sorted by liver size on 5 dpf and tracked daily for survival to 20 dpf. Data are pooled from three clutches. (F) UHRF1-GFP High and control larvae were treated with 50 μ M 5-Aza from 2.5–5 dpf and scored for liver size in 6 clutches. Fisher’s exact test was used to determine p values. (G) UHRF1-GFP High embryos were injected with mRNA encoding dnmt1 or Mpi before 1 hpf. The percent of fish with a normal liver size was scored at 5 dpf in 6 clutches. See also Figure S2.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\nFigure 3. UHRF1 overexpression in hepatocytes induces Tp53-mediated senescence (A) Intense senescence associated β -galactosidase (SA- β -gal) staining was detected in the liver (outlined) of 5 dpf UHRF1-GFP High larvae compared to light or no staining in controls. (B) 5 dpf fish from 5 clutches were scored for hepatic SA- β -gal staining. *** indicates p&lt;0.0001 by Fisher’s exact test. (C) Nuclear size was measured in hepatocytes of a single control or UHRF1-GFP High 5 dpf liver and cells were stratified according to GFP expression. Inset shows confocal stack of the DNA organized into foci. ** and *** indicates p&lt;0.001 or 0.0001, respectively compared to nuclear size in nls-mCherry larvae. (D) BrdU positive cells and the total number of transgene-expressing hepatocytes in nls-mCherry and UHRF1-GFP High larvae (Bottom) 5 dpf larvae. A Fisher’s exact test was used to calculate\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\np value. In nls-mCherry larvae, most BrdU positive cells also express the transgene whereas the BrdU positive cells in UHRF1-GFP High livers did not express GFP (white arrows in magnified regions which are marked by the white box). (E) Heatmap of log2 values from RNAseq shows cell cycle regulators are down and Tp53 target genes (marked by *) are up in UHRF1-GFP High 5 dpf livers. (F) tp53 and cdkn1a mRNA expression were induced on 5 dpf and down regulated on 20 dpf in UHRF1-GFP High livers. * indicates p=0.05; *** indicates p=0.001 calculated by 1 sample Student’s T-test. Error bars represent SD. (G) 5- Aza induces Tp53 expression in primary mouse hepatocytes. Student’s T-test was used to determine p value with SD indicated by the error bars across 3 replicates. tp53 +/− in UHRF1-GFP High larvae significantly reduced SA- β -gal staining in the liver (2 clutches) (H), and increased the percent of larvae with normal liver size (I), the area of the left liver lobe (J), and survival at 5 dpf (K). p values were calculated with a Fisher’s test with Freeman-Halton extension (H), Fisher’s exact test (I), and Student’s T-test. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. (J). See also Figure S3.\nNIH-PA Author Manuscript\nFigure 4. UHRF1 is an oncogene\n(A) Atypical cells, dysplastic foci (outlined) and HCC are apparent in H&amp;E stained UHRF1- GFP High livers. BD: bile duct, MF: mitotic figure. (B) Incidence of normal and atypical hepatocytes, dysplastic foci and cancer in the liver of UHRF1-GFP High fish on WT or tp53 +/− background. (C) NIH-3T3 cell growth in soft agar is enhanced when UHRF1 overexpression is combined with RAS (n=3). p value was calculated by Student’s T-test and error bars represent the SD. (D) Hepatic SA- β -gal staining patterns in UHRF1-GFP High larvae changes as fish age. Images of 8 dpf larvae illustrate the SA- β -gal staining patterns that were scored in the time course shown in the graph. A significant increase in the number of UHRF1-GFP High fish with intense or punctate SA- β -gal compared to controls at all time points (p&lt;0.01 by Fisher’s exact test) except at 4 dpf; not significant=n.s. (E) BrdU incorporation in the liver on 11 dpf is 5 times higher in UHRF1-GFP High fish than in\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\ncontrols. Total # of cells counted is indicated with n= # of clutches assessed. Fisher’s exact test was used to calculate p value. See also Figure S4.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 5. UHRF1 mRNA and protein are overexpressed in HCC (A) UHRF1 detected by qRT-PCR in 18 pre-neoplastic lesions and 40 HCCs from HCV infected patients compared to expression in 9 normal livers. Horizontal line indicates median. (B) Immunohistochemistry for UHRF1 protein (brown) was evaluated in 52 of the same HCCs examined in A plus 5 normal liver samples. Fisher’s exact test was used to calculate p value. 71 of the HCV-associated HCCs analyzed by qPCR were grouped into high (n=35) and low (n=36) UHRF1 expressing tumors based on the median log2-fold change of 3.64. (C) HepG2 cells transfected with control siRNA (GL2) or two different siRNAs targeting UHRF1 described in (Tien et al., 2011) were blotted for UHRF1, cleaved and total PARP (arrow indicates full length, * indicates cleaved protein). (D) Vascular invasion (33 high and 34 low tumors, 4 missing values), (E) serum AFP (29 UHRF1-high and 29 low tumors, 13 missing values), (F) early (&lt;2 years) and (G) late (&gt;2 years; 32 UHRF1-high and 35 low tumors, 4 missing values) tumor recurrence and (H) overall survival after surgery (32 high and 35 low tumors, 4 missing values) were stratified according to UHRF1-expression. Continuous and categorical variables were assessed by Wilcoxon rank-sum test and Fisher’s exact test, respectively. Clinical outcome difference was evaluated by log-rank test. In box and whisker plots, boxes represent the 75th and 25th\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\npercentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S5.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 6. High UHRF1 expression defines a subclass of tumors with inactivated tp53, repression of senescence and chromosomal instability\n(A) The 71 human HCC tumors analyzed in Figure 5C–G were rank-ordered according to UHRF1 expression by qPCR and classified as high (&lt;median, red; n=35) or low (≥median, blue; n=36). The presences of aggressive human HCC gene signatures from published studies and of TP53 inactivating mutations are indicated by red and black boxes, respectively. TP53-mediated senescence gene signatures (Lujambio et al., 2013; Tang et al., 2007) are displayed as a range from repressed (blue) to activated (red). (B) Genome-wide profile of DNA copy number variation was obtained from GEO geneset GSE9829. (C) Proportion of genes with DNA copy number loss and gain in tumors according to UHRF1 expression. (D) Proportion of genes with DNA copy number loss according to UHRF1 expression and TP53 mutation status. (E) UHRF1 expression is significantly higher in tumors with TP53 mutations. (F) DNMT1 expression by microarray analysis is significantly correlated with UHRF1 expression assessed by qPCR in HCCs. In box and whisker plots,\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nboxes represent the 75th and 25th percentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S6 and Tables S1-S6.\nNIH-PA Author Manuscript\nFigure 7. Model of the relationship between UHRF1 overexpression, DNA hypomethylation, Tp53 mediated senescence, cancer and survival\nFactors investigated in this study are in solid black boxes with black lines indicating the correlations demonstrated in this work and gray lines indicating relationships that are speculative. Senescence reduces liver size and function and reduces larval survival while cancer occurs when senescence is bypassed and also reduces survival.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 2\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 4\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 6\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 7\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 8\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 9\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 10\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 11\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 12\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 13\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 14\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\n• Our studies in zebrafish and human tumors identify UHRF1 as an oncogene in HCC Mudbhary et al. Page 15\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 16\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 17\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 18\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 19\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 20\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 21\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 22\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 23\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 24\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 25\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 26\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 27\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 28\n\n\nTable 1\n\n\nHCC onset and incidence in Tg(fabp10:nls-mCherry) and Tg(fabp10:UHRF1-GFP) zebrafish. transgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI nls-mCherry Normal 100% 100% 100% 100% 100% 100% 100% 100% 100% 100% 65 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 8 8 8 11 nd nd 5 5 5 5 5 5 65 UHRF1-GFP Low Normal 100% 100% 100% 100% 100% 100% 100% 100% 60 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 7 9 10 12 nd nd 4 nd nd 6 6 6 60 UHRF1-GFP Medium Normal 90% (9) 100% (11) 50% (3) 28% (4) 66% (4) 80% (4) 50% (4) 50% (5) 81% (13) 57 66%\n\n\nAtypical cells 10% (1) 0% 50% (3) 57% (8) 33% (2) 20% (1) 50% (4) 20% (2) 44% (7) 28 32%\n\n\nDysplastic foci 0% 0% 0% 7% (1) 0% 0% 25% (2) 30% (3) 0% 6 7%\n\n\nTumor 0% 0% 0% 0% 0% 0% 0% 0% 6% (1) 1 1% n = 10 11 6 14 6 5 8 10 16 86 UHRF1-GFP High Normal 0% 0% 7% (1) 0% 0% 1 2% Atypical cells 100% (13) 100% (7) 46% (6) 17% (3) 75% (3) 36 61% Dysplastic foci 0% 0% 8% (1) 12% (2) 25% (1) 4 7% Tumor 0% 0% 46% (6) 76% (13) 0% 18 30% n = 13 7 13 17 4 54 UHRF1-GFP High/ p53 +/− Normal 0% 0 0% Atypical cells 13% (2) 2 13% Dysplastic foci 0% 0 0%\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 29\n\n\ntransgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI\n\n\nTumor 88% (14) 14 88% n = 16 16\n\n\nThe incidence (percent) and absolute number of fish (parenthesis) with cancer-relevant histological phenotypes were scored in a total of 281 fish from 4 transgenic lines. The total n for each time point is indicated. nd: not done. Some fish were diagnosed with more than one lesion. CI: Cumulative incidence.\n",
  "methodology": "RNAseq in Zebrafish",
  "method_goal": "Studies of RNA expression in Zebrafish in a variety of experimental conditions",
  "questions_output_specification": "1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data"
}
[chain/end] [1:chain:RunnableSequence &gt; 2:prompt:ChatPromptTemplate] [1ms] Exiting Prompt run with output:
{
  "lc": 1,
  "type": "constructor",
  "id": [
    "langchain",
    "prompts",
    "chat",
    "ChatPromptValue"
  ],
  "kwargs": {
    "messages": [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "SystemMessage"
        ],
        "kwargs": {
          "content": "You are an expert biological scientist, skilled at reading scientific papers.",
          "additional_kwargs": {}
        }
      },
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain",
          "schema",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''UHRF1 overexpression drives DNA hypomethylation and\nhepatocellular carcinoma\nRaksha Mudbhary a,b,c , Yujin Hoshida a,d,l , Yelena Chernyavskaya a,b,l , Vinitha Jacob a,b,c , Augusto Villanueva f , M. Isabel Fiel e , Xintong Chen a,d , Kensuke Kojima a,d , Swan Thung e ,\nRoderick T. Bronson g , Anja Lachenmayer d , Kate Revill d , Clara Alsinet f , Ravi Sachidanandam h , Anal Desai i , Sucharita SenBanerjee i , Chinweike Ukomadu i , Josep M. Llovet a,d,f,j , and Kirsten C. Sadler a,b,c,k\na Department of Medicine/Division of Liver Diseases, Icahn School of Medicine at Mount Sinai New York, NY 10029\nb Department of Developmental and Regenerative Biology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nc Graduate School of Biological Sciences, Icahn School of Medicine at Mount Sinai New York, NY 10029\nd Liver Cancer Program/Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai New York, NY 10029\ne Department of Pathology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nf HCC Translational Research Laboratory, IDIBAPS, CIBEREHD, Hospital Clinic, University of Barcelona, Catalonia, Spain\ng Rodent Histopathology Core Dana Farber/Harvard Cancer Center, Harvard Medical School Boston, MA 02115\nh Department of Genetics and Genomics, Icahn School of Medicine at Mount Sinai New York, NY 10029\nI Division of Gastroenterology, Department of Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115\nj Institució Catalana de Recerca i Estudis Avançats Lluís Companys, 23, Barcelona – Spain\n\n\nSUMMARY\nUHRF1 is an essential regulator of DNA methylation that is highly expressed in many cancers. Here, we use transgenic zebrafish, cultured cells and human tumors to demonstrate that UHRF1 is an oncogene. UHRF1 overexpression in zebrafish hepatocytes destabilizes and delocalizes DNMT1, causes DNA hypomethylation and Tp53-mediated senescence. Hepatocellular carcinoma\n© 2014 Elsevier Inc. All rights reserved.\nk Corresponding author: Kirsten.edepli@mssm.edu, 1 Gustave L. Levy Place, Box 1020, Icahn School of Medicine at Mount Sinai, New York, NY 10029, phone: 212-241-7152, fax: 212-860-9279. lthese authors contributed equally to this work.\n\n\nAccession Number\nData from RNAseq was deposited with accession number GSE52605.\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\nPublished in final edited form as:\nCancer Cell. 2014 February 10; 25(2): 196–209. doi:10.1016/j.ccr.2014.01.003.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n(HCC) emerges when senescence is bypassed. tp53 mutation both alleviates senescence and accelerates tumor onset. Human HCCs recapitulate this paradigm, as UHRF1 overexpression defines a subclass of aggressive HCCs characterized by genomic instability, TP53 mutation and abrogation of the TP53-mediated senescence program. We propose that UHRF1 overexpression is a mechanism underlying DNA hypomethylation in cancer cells and that senescence is a primary means of restricting tumorigenesis due to epigenetic disruption.\n\n\nINTRODUCTION\nThe expression of genes that encode readers and writers of the epigenetic code are widely deregulated across cancer types (You and Jones, 2012). This contributes to the massive gene expression changes and remodeling of the epigenetic landscape, a characteristic of many types of cancer. In particular, loss of global DNA methylation is a hallmark of cancer cells.\nDNA hypomethylation contributes to oncogenesis through multiple mechanisms, including chromosomal instability (Eden et al., 2003; Karpf and Matsui, 2005), derepression of imprinted genes (Berdasco and Esteller, 2010; Jirtle, 2004; Li et al., 1993), retrotransposon activation (Gaudet et al., 2004; Howard et al., 2008; Jackson-Grusby et al., 2001; Sharif et al., 2007) and aberrant gene expression, including induction of oncogenes (Cheah et al., 1984). Many studies have documented that expression of the core factors required for maintenance DNA methylation – i.e. DNA methyltransferase 1 (DNMT1) and ubiquitin-like with PHD and RING finger domains 1 (UHRF1) (Babbio et al., 2012; Jin et al., 2009; Unoki et al., 2010; Wang et al., 2012) – are significantly altered across cancer types. However, whether changes in the expression of these key factors are sufficient to alter the cancer cell methylome and drive carcinogenesis is unknown. Moreover, the mechanism by which DNA methylation is lost in cancer cells is poorly understood.\nThe cellular response to DNA hypomethylation varies by cell type, physiological context and degree of hypomethylation. In some cells, DNA hypomethylation induces tumor suppressive mechanisms, including apoptosis (Anderson et al., 2009; Biniszkiewicz et al., 2002; Chen et al., 2007; Jackson-Grusby et al., 2001) or senescence (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), whereas in other cells it blocks differentiation of progenitor cells (Rai et al., 2010) or causes cancer (Gaudet et al., 2003; Yamada et al., 2005). In part, the cellular response to loss of DNA methylation is dictated by the genomic region affected: hypomethylation of gene regulatory regions, such as promoters, can de-repress gene expression but hypomethylation of repetitive elements can reduce heterochromatin formation, promote recombination and genomic instability.\nUHRF1 plays an essential role in DNA methylation by both recognizing hemimethylated DNA generated during DNA replication and then recruiting DNMT1 to ensure faithful maintenance of DNA methylation patterns in daughter cells (Arita et al., 2008; Avvakumov et al., 2008; Bostick et al., 2007; Hashimoto et al., 2008; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Consequently, UHRF1 depletion results in global DNA hypomethylation (Bostick et al., 2007; Feng et al., 2010; Sharif et al., 2007; Tittle et al., 2011). Conversely, UHRF1 may also limit DNA methylation by targeting DNMT1 for ubiquitin-mediated degradation (Du et al., 2010; Qin et al., 2011) or by delocalizing DNMT1 (Sharif et al., 2007). How UHRF1 overexpression impacts the methylome is not known.\nWe previously reported that uhrf1 mutation in zebrafish blocks liver outgrowth in embryos and regeneration in adults (Sadler et al., 2007) and that depleting UHRF1 from cancer cells induces apoptosis (Tien et al., 2011). Here, we tested the hypothesis that UHRF1\n\n\nNIH-PA Author Manuscript\noverexpression would alter global DNA methylation, and promote hepatocellular carcinoma (HCC).\n\n\nRESULTS\n\n\nHigh UHRF1 expression causes DNA hypomethylation\nUHRF1 is required for DNA methylation, as it recruites DNMT1 to hemimethylated DNA during DNA replication (Bostick et al., 2007; Feng et al., 2010; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Paradoxically, UHRF1 also serves as an ubiquitin ligase that targets DNMT1 for degradation (Du et al., 2010; Qin et al., 2011). To determine how UHRF1 overexpression impacts DNA methylation, we generated transgenic zebrafish expressing human UHRF1 fused to GFP (hsa.UHRF-GFP) under the hepatocyte specific fabp10 promoter (Tg(fabp10:hsa.UHRF1-GFP) (Chu et al., 2012)).\nHuman and zebrafish UHRF1 are 66% identical and the ability of human UHRF1 to rescue zebrafish embryos depleted of uhrf1 (VJ and KCS, unpublished and (Chu et al., 2012)) indicates they are functional orthologs. Expression is first detected in hepatocytes on 3 dpf and 5 dpf, nuclear UHRF1-GFP is easily detected at variable levels in hepatocytes (Figure S1A and (Chu et al., 2012)). We isolated an allelic series of Tg(fabp10:hsa.UHRF1-GFP) transgenics expressing a range of UHRF1 levels (Figure S1A–B; hereafter referred to UHRF1-GFP High, Medium and Low).\nWe probed genomic DNA isolated from the liver of 5 dpf larvae from each line with an antibody specific for 5-methyl cytosine (5MeC) to assess DNA methylation and found significant hypomethylation only in the liver of UHRF1-GFP High larvae (38% of controls; p=0.007; Figure 1A and Figure S1C). 5MeC immunofluorescence showed DNA methylation as uniformly distributed in the nuclei of hepatocytes from control larvae that express nuclear-localized fluorescent mCherry under the fabp10 promoter (Tg(fabp10:nls-mCherry); abbreviated to nls-mCherry) but only dim staining in hepatocytes of UHRF1-GFP High larvae (Figure 1B). Since Equivalent levels of 5MeC were detected in the liverless carcasses (Figure 1A) or fins (Figure 1B), we conclude that DNA hypomethylation was specific to the liver.\nWe hypothesized that UHRF1-induced DNA hypomethylation could be caused by mislocalization or destabilization of Dnmt1. Immunofluorescence revealed uniform Dnmt1 distribution in the nucleoplasm of nls-mCherry hepatocytes (Figure 1C) but in UHRF1-GFP high hepatocytes, it was concentrated in nuclear foci that contained UHRF1-GFP (Figure 1C). Interestingly, the range of UHRF1-GFP expression within cells of the same liver revealed that in cells with high GFP, Dnmt1 was dim and punctate, but levels and distribution pattern was similar to control hepatocytes in those cells with low or no GFP (arrows, Figure 1C). Thus, high expression of UHRF1 co-localizes with Dnmt1 and redistributes it to intra-nuclear structures reminiscent of senescence associated heterochromatin foci (Di Micco et al., 2011).\nEndogenous Dnmt1 in the liver was below the levels detectable by immunoblotting (not shown), so we used a transgenic line expressing UHRF1-GFP under the inducible hsp70l promoter (Chu et al., 2012) to quantitatively assess the impact of UHRF1 overexpression on Dnmt1 stability at a developmental time when we could easily detect Dnmt1 levels by immunoblotting. We optimize a heat shock protocol that maximized UHRF1-GFP expression in Tg(hsp70l:UHRF1-GFP) larvae (Figure 1D) and found that UHRF1-GFP overexpression reduced Dnmt1 levels by 27% compared to non-heat-shocked transgenics (p=0.03; Figure 1E). Treatment with a non-toxic dose of the proteasome inhibitor, MG132, prevented the decrease in Dnmt1 induced by UHRF1 overexpression (p=0.01; Figure 1E).\n\n\nNIH-PA Author Manuscript\nNeither heat-shock nor MG132 significantly affected Dnmt1 protein levels in non-transgenic controls (Figure 1E). Thus, both Dnmt1 delocalization and destabilization could account for DNA hypomethylation caused by UHRF1 overexpression.\n\n\nDNA hypomethylation caused by UHRF1 overexpression reduces liver size\nUhrf1 in zebrafish is required for hepatic outgrowth and liver regeneration (Sadler et al., 2007). To determine if UHRF1 overexpression in hepatocytes affected hepatic outgrowth, we assessed liver size all transgenic lines (Figures 2A–C and S2A). Low and Medium expressing lines had no gross changes in liver size (Figure S2A), but 74% of UHRF1-GFP High larvae on 5 dpf had small livers (Figure 2C), with the median area of the left liver lobe reduced by 35% compared to controls (p&lt;0.001; Figures 2D, S2A), without reduction of total fish size (Figure S2B).\nLarvae with small livers appeared sick (see 10 dpf larvae in Figure 2A), and only 20% of UHRF1-GFP High fish survived to 20 dpf (Figure S2C and Figure 2E). Moreover, UHRF1- GFP High larvae with the “microliver” phenotype on 5 dpf had significantly higher mortality by 10 dpf (67%) than those that started with a normal sized liver (7%; Figure 2E). Thus, high UHRF1 expression in hepatocytes causes DNA hypomethylation, microliver and larval death.\nWe next investigated the relationship between DNA hypomethylation and the microliver phenotype by asking whether further reducing DNA methylation could enhance this phenotype or restoring Dnmt1 could suppress it. We previously reported that exposing embryos to 50 μ M of the Dnmt1 inhibitor, 5-azacytidine (5-Aza) from 0–5 dpf caused a profoundly small liver and DNA hypomethylation (Mudbhary and Sadler, 2011). By restricting the 5-Aza exposure time to 2.5–5 dpf, we reduced DNA methylation in the liver of control larvae by 40% (Figure S1C) which induced a moderately small liver in 16% of larvae (p=0.0035; Figure 2F). This same treatment of UHRF1-GFP High larvae significantly increased the percent with small livers (p&lt;0.0001; Figure 2F). While this could be attributed to DNA damage caused by 5-Aza, our finding that injecting UHRF1-GFP High embryos with mRNA encoding Dnmt1 modestly suppressed the percent of fish with a small liver (Figure 2G) and increased the average size of the left liver lobe by 10–15% (Figure S2D) suggests that hypomethylation, at least in part, contributes to the small liver phenotype of UHRF1-GFP High larvae.\n\n\nUHRF1 overexpression triggers Tp53-mediated senescence\nDNA hypomethylation can induce apoptosis, but we found no TUNEL positive cells on 5 dpf of UHRF1-GFP High livers (Figure S3A). However, senescence associated β - galactosidase (SA- β -gal) staining was detected throughout the liver of most UHRF1-GFP High 5 dpf larvae (n=93; Figure 3A–B), but not other transgenic lines (Figure S3B). Additionally, the DNA in control hepatocytes was evenly distributed throughout the uniformly sized nuclei compared to the large nuclei where DNA resembled senescence associated heterochromatic foci (inset, Figure 3C). Strikingly, the largest hepatocyte nuclei also had the brightest GFP (Figure 3C), suggesting the effect of UHRF1 overexpression cell autonomously affected nuclear morphology.\nSenescent cells do not divide, and we found significantly less BrdU incorporation in the liver of 5 dpf UHRF1-GFP High larvae compared to controls (p&lt;0.0001; Figure 3D). Interestingly, in nls-mCherry larvae, most BrdU incorporation was detected in hepatocytes that express nls-mCherry, but the only BrdU positive cells in UHRF1-GFP high larvae were negative for GFP (see inset in Figure 3D). RNAseq revealed some pro-proliferative genes (ccnd1 and myc) downregulated (Figure 3E), lending further support to the conclusion that\nsenescence is the primary response to high UHRF overexpression in hepatocytes during hepatic outgrowth.\nTp53 is a key mediator of senescence caused by DNA damage and oncogenic stress (Di Micco et al., 2011; McDuff and Turner, 2011; Ventura et al., 2007; Xue et al., 2007). RNAseq (Figure 3E) and qPCR analysis (Figure 3F) show that tp53 and its target genes, especially cdkn1a, are significantly induced in the liver of 5 dpf UHRF1-GFP High larvae, but then return to baseline by 20 dpf (Figure 3F). 5-Aza treatment of primary mouse hepatocytes induced Tp53 expression (Figure 3G), similar to the effects of 5-Aza treatment or Dnmt1 depletion in other cell types (Jackson-Grusby et al., 2001; Karpf et al., 2001) and in zebrafish (VJ and KCS, not shown). However, since no significant alteration in methylation of the tp53 promoter was detected in these models (not shown), we hypothesize that DNA methylation does not directly regulate Tp53 expression. Instead, DNA hypomethylation may induce tp53 by an indirect mechanism, such as increased DNA damage or genomic instability.\nA direct role for Tp53 in the phenotype of UHRF1-GFP High larvae was demonstrated by removing one copy of tp53. This reduced the incidence and intensity of SA- β -gal staining in the liver (Figure 3H), increased liver size (Figure 3I–J) and reduced mortality (Figure 3K). We thus propose a model whereby high UHRF1 causes DNA hypomethylation, induces Tp53-mediated senescence and prevents expansion of the hepatic bud, resulting in hepatic insufficiency and larval death.\n\n\nUHRF1 overexpression induces liver cancer in zebrafish\nTo determine if UHRF1 overexpression was sufficient to cause HCC, 281 control and UHRF1-GFP transgenics were collected between 5–300 dpf, serial sectioned and analyzed for atypical cells, dysplastic foci and HCC using histological criteria devised by two expert pathologists (RTB and MIF) based on disrupted tissue architecture, cell size, shape, nuclear structure and presence of mitotic figures (Figure S4A–B). Evidence of increased hepatocyte proliferation was detected in all UHRF1 overexpressing lines on 20 and 40 dpf (Figure S4C), but this was insufficient to cause HCC, as UHRF1-GFP Low fish were tumor free at all time points (Table 1). In contrast, UHRF1-GFP High fish developed atypical hepatocytes as early as 5 dpf, with an 8% incidence of dysplastic foci and 46% incidence of HCC by 15 dpf. On 20 dpf, 76% of fish had HCC (Figure 4A–B, Table 1). UHRF1-GFP Medium fish also developed atypical hepatocytes and dysplastic foci at young ages, and one large HCC was detected in a 60 dpf fish (Table 1). In a classical transformation assay using NIH-3T3, we found that UHRF1 cooperated with RAS to promote growth on soft agar (Figure 4C). These conclusively demonstrate that UHRF1 is an oncogene.\nWe found that UHRF1-GFP High larvae older than 8 dpf had a lower incidence and intensity of hepatic SA- β -gal staining (Figure 4D). Interestingly, in many of these fish, intense staining was distributed in a punctate pattern. By 20 dpf, 75% of fish had either punctate or no staining (Figure 4D), which is a striking correlation with the 70% incidence of HCC at this time point. This was mirrored by increased proliferation of liver cells, detected by increased BrdU incorporation in UHRF1-GFP High livers (22% vs. 4% in controls) at 11 dpf (p&lt;0.001; Figure 4E), and higher PCNA staining 20 and 40 dpf in all lines (Figure S4C). Loss of senescence was not attributed to reestablishment of DNA methylation or transgene silencing, as reduced 5MeC staining persisted in tumor cells (Figure S4D) and transgene expression was detectable in all 20 dpf fish (Figures S4E–F), albeit reduced from levels detected in 5 dpf livers.\nWe asked whether Tp53 epistatically interacted with UHRF1 overexpression to contribute to HCC by removing one copy of tp53 (Berghmans et al., 2005). The liver appeared normal in\n\n\nNIH-PA Author Manuscript\n13 tp53 +/+ fish without transgene expression (not shown), but in UHRF1-GFP High fish, tumor incidence on 15 dpf increased from 50% in WT to 87% in tp53 +/− fish (Figure 4B). Interestingly, in a single Tg(fabp10:UHRF1-GFP HIGH );p53 +/− , we found a tumor with immature cells resembling a cholangiocarcinoma. Thus, tp53 functions to suppress tumor formation and may alter the spectrum of tumors caused by UHRF1 overexpression.\n\n\nUHRF1 is upregulated in human HCC\nWe next investigated the relevance of UHRF1 expression in human HCC. We assessed UHRF1 expression by qPCR in 16 normal liver samples and in 2 cohorts of patients with dysplastic nodules or HCC: the 1 st cohort of 58 patients had hepatitis C infection (Figure 5A; (Wurmbach et al., 2007)) and the 2 nd cohort of 69 patients had HBV, alcohol and other etiologies (Figure S5A; (Villanueva et al., 2008)). Additional publically available transcriptome data sets from three other cohorts of HCCs (Figure S5B) and from lung, gastric, colorectal and breast cancer (Figure S5C) were also analyzed. All showed elevated UHRF1 in tumors, with expression elevated &gt;2-fold compared to controls in 17/18 dysplastic foci and 104/109 HCCs (Figures 5A and S5A). An average of 20- and 46-fold overexpression of UHRF1 was detected in advanced and very advanced HCCs. UHRF1 protein was barely detectable in 5 normal liver samples but highly expressed in 38/52 (73%) of the tumors analyzed for UHRF1 mRNA in Figure 5A (p&lt;0.003; Figure 5B). Thus, UHRF1 is overexpressed in HCCs of diverse etiologies as well as in other tumor types, and high UHRF1 expression is significantly associated with the most advanced tumors.\nTargeting UHRF1 in liver cancer cells (HepG2) using siRNA induced PARP cleavage (Figure 5C) and other markers of apoptosis (not shown), similar to results obtained using another HCC cell line (Hep3B; not shown) and colon cancer cells (Tien et al., 2011). Thus, blocking UHRF1 cancers such as HCC with high UHRF1 expression could be an effective means to induce tumor cell death.\nOf the 109 HCCs analyzed for UHRF1 expression by qPCR, 71 had multiple clinical and genomics parameters available (Chiang et al., 2008). These were rank-ordered based on UHRF1 expression determined by qPCR (Figure 6A). Tumors with expression above and equal to or below the median log2-fold change value of 3.64 were designated as “high” (n=35) and “low” (n=36), respectively (Figure 6A). High UHRF1 expressing tumors were associated with signs of poor clinical outcome: 80% had microvascular invasion (Figure 5D) and significantly higher alpha-feto protein (AFP) levels (Figure 5E), although AFP levels alone did not cooperate with UHRF1 to predict survival (not shown). Importantly, high UHRF1 expression significantly correlated with early (&lt;2 years; Figure 5F), but not late (Figure 5G) tumor recurrence and was inversely correlated with survival (Figure 5H). This suggests that high UHRF1 expression predicted recurrence of the primary tumor, causing decreased survival (Villanueva et al., 2011). Molecular signatures of aggressive HCC tumors compiled from several previous studies (Table S1) were concordantly and significantly enriched in high UHRF1 expressing tumors (Figure 6A). Moreover, high UHRF1 expressing tumors were distinguished by induction of pathways that drive the cell cycle, DNA replication and repair (Figure S6A, Tables S2 and S3). UHRF1 overexpression also significantly correlated with advanced stage prostate cancer, but not lung or colon (Figure S5D–G).\nHigh UHRF1 expression delineates a subclass of HCCs that have downregulated TP53- mediated senescence\nOur zebrafish studies demonstrated that bypass of Tp53-induced senescence is required for UHRF1 to act as an oncogene. Our analysis of human HCCs indicates that a similar paradigm occurs in these samples. First, inactivating mutations in TP53 significantly\n\n\nNIH-PA Author Manuscript\ncorrelated with high UHRF1 expression (Figure 6A). Second, many of the core enriched genes high UHRF1 expressing tumors are regulated by TP53 in senescent fibroblasts (Table S4 and (Tang et al., 2007), and high UHRF1 expressing tumors have downregulated the gene expression signature derived from TP53-induced senescence in fibroblasts (Table S5, Figure S6B; (Tang et al., 2007)) or in hepatic stellate cells (Table S6, Figure S6C; (Lujambio et al., 2013)). Fourth, significant correlation between UHRF1 overexpression, TP53 mutation and genome integrity in human HCCs indicates that these pathways act together: high UHRF1 expression (Figure 6B–C) and TP53 mutation (Figure 6D) are independently correlated with chromosomal loss but tumors that have both features display even more chromosomal loss (Figure 6D). Finally, UHRF1 expression was significantly higher in tumors with TP53 mutation (Figure 6E). UHRF1 overexpression did not correlate with copy number variation at the UHRF1 locus (Figure S6D) suggesting that a different mechanism drives UHRF1 overexpression.\nGenome wide DNA hypomethylation is found in most HCCs (Calvisi et al., 2007) and this rendered it difficult to correlate methylome changes with UHRF1 expression. However, DNMT1 expression was directly correlated with UHRF1 expression in HCC samples (Figure 6F). This may be a consequence of the high proliferation rate in these tumors or an induction of the methylation machinery to compensate for hypomethylation. Together, these data indicate that high UHRF1 expression in HCC defines a subset of aggressive tumors that have inactivated the TP53 induced senescence program, suggesting that in humans, as observed in zebrafish, TP53 acts as a tumor suppressor to restrict the oncogenic potential of UHRF1.\n\n\nDISCUSSION\nWe show that UHRF1 overexpression is sufficient to cause two oncogene-associated phenotypes: senescence and cancer. This defines UHRF1 as an epigenetic regulator that can cause cancer when overexpressed and is thus an oncogene. We hypothesize that UHRF1 overexpression is one mechanism by which cancer genomes can become hypomethylated, either via UHRF1-mediated Dnmt1 ubiquitination and degredation (Du et al., 2010; Qin et al., 2011) or by redistribution and/or sequestration of Dnmt1 away from DNA. Excess UHRF1 might also sequester the DNMT1 deubiquitinating enzyme, USP7 (Qin et al., 2011) to further promote DNMT1 ubiquitination and degradation.\nThe precise mechanism by which DNA hypomethylation causes cancer remains elusive. Both apoptosis and senescence serve to limit the propagation of cells with aberrant DNA methylation (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), however, once epigenetically altered cells escape these tumor suppressive mechanisms, they likely accumulate genetic lesions which predispose to cancer. Indeed, chromosomal instability and mitotic catastrophe occur following DNMT1 depletion (Chen et al., 2007; Karpf and Matsui, 2005; Weber and Schubeler, 2007) and transposons, which are heavily methylated in normal cells, could become activated and cause genomic instability upon UHRF1-induced DNA hypomethylation. Additionally, hypomethylated DNA is more likely to assume an open chromatin conformation, which may promote oncogene expression, although this possibility has not been fully evaluated. While the oncogenic role of UHRF1 could also be mediated by the impact of UHRF1 on other epigenetic marks or on DNA replication (Taylor et al., 2013), our data combined with findings from others (Eden et al., 2003; Gaudet et al., 2003) suggests that DNA hypomethylation is a likely mechanism driving UHRF1-mediated transformation.\nOur working model (Figure 7) proposes that UHRF1 overexpression causes DNA hypomethylation by reducing Dnmt1 levels and its access to DNA. Tp53 is induced in\n\n\nNIH-PA Author Manuscript\nresponse to either genomic instability or some unidentified direct mechanism, and causes hepatocyte senescence, which prevents hepatic expansion and results in larval death from hepatic insufficiency. We propose that Tp53 inactivation and senescence bypass allows for the unhindered proliferation and malignant transformation. Identifying the mechanism by which this tumor suppressive mechanism is overcome remains a central, unanswered question in cancer biology.\nStudies in a mouse liver cancer model show that Tp53 reactivation causes senescence, and these cells are then cleared by the immune system (Xue et al., 2007) and the liver is then repopulated with senescence-resistant tumor forming cells. Our finding that senescence decreases and BrdU incorporation increases in UHRF1-GFP high livers over time suggests a similar process at play. Moreover, BrdU incorporation in primarily GFP negative cells suggests the expansion of either immune cells or immature hepatic progenitors in response to senescent hepatocytes. The finding of a cholangiocarcinoma in a UHRF1-GFP High/ p53 +/− fish may indicate that a bipotential progenitor cell could be the tumor forming cell in this model. Additionally, our data indicate that there is a threshold effect of UHRF1 expression, in which the highest expressing cells undergo senescence and neighboring hepatocytes expressing UHRF1-GFP at levels below those detectable via microscopy undergo unhindered expansion.\nHCC is the third cause of cancer-related deaths globally (Llovet et al., 2003), yet curative therapies are limited, with Sorafanib as the only systemic therapy available for advanced cases (Llovet et al., 2008). Thus, there is an urgent and unmet need for novel therapies. HCC, like other cancers, is characterized by global DNA hypomethylation (Calvisi et al., 2007; Pogribny and Rusyn, 2012; Tischoff and Tannapfe, 2008), and high UHRF1 expression could be the cause. Our finding that UHRF1 depletion in HCC and other types of cancer cells causes apoptosis (Tien et al., 2011) presents UHRF1 as an attractive target for cancer therapy, whereby blocking UHRF1 could trigger massive epigenetic changes incompatible with cell survival or could reset the cancer cell methylome.\n\n\nEXPERIMENTAL PROCEDURES\n\n\nZebrafish maintenance and generation of transgenics\nZebrafish were maintained on a 14:10 hour light:dark cycle at 28°C. mRNA encoding zebrafish Dnmt1 (Rai et al., 2006) or mannose phosphate isomerase (Mpi; (Chu et al., 2013) as a control was injected into embryos just after fertilization.\nTg(fabp10:nls-mCherry) fish expressing nls-mCherry exclusively in hepatocytes were generated using Gateway cloning (Invitrogen) to produce vectors with tol2 transposon sites (Kwan et al., 2007). Tg(hsp70l:hsa.UHRF1-GFP) and Tg(fabp10:hsa.UHRF1-GFP) were described in (Chu et al., 2012). The high, medium and low expressing alleles are listed in ZFIN.org with superscripts mss1a, mss1b and mss1c, respectively. Transgenics were outcrossed to Tab14 (WT) or tp53 −/− fish (Berghmans et al., 2005).\nTg(hsp70l:UHRF1-GFP) embryos were heat-shocked at 37°C for 1 hour at 24 and 27 hours post fertilization (hpf). At 28 hpf, embryos were sorted visually for GFP expression and incubated with either 10 μ M MG132 or DMSO and collected for immunoblotting at 34 hpf. 5-Aza (50 μ M) was added to larvae from 2.5–5 dpf. The Mount Sinai Institutional Animal Care and Use Committee approved all protocols. Nomenclature guidelines for the species under discussion were followed, and when no species is specified, human nomenclature was used.\n\n\nNIH-PA Author Manuscript\n\n\nGene expression analysis\nRNA was isolated from a pool of at least 10 livers from 5 dpf fish and from 1–5 livers from 20 dpf fish using the RNeasy mini-kit (Qiagen). cDNA was prepared by polyA priming using qScript SuperMix (Quanta). qRT-PCR analysis was performed in the Light Cycler 480 (Roche) using gene-specific primers (see supplemental materials) and PerfeCTa SYBRGreen FastMix (Quanta). C t values from triplicate reactions were averaged and 2 -Ct(target) /2 -Ct(reference) was used to calculate expression, with rpp0 and cyclophilin A used as references genes zebrafish and mouse samples, respectively.\nRNAseq analysis was carried out on RNA from pools of 50 livers dissected from 2 clutches of 5 dpf UHRF1-GFP High and nls-mCherry larvae, described in Supplemental Material.\n\n\nHistology\nFish younger than 20 dpf were fixed overnight at 4°C in 4% paraformaldehyde (PFA) and older fish were fixed for 2–5 days at room temperature in Bouins fixative. 4 μ M serial sections of paraffin embedded fish were stained with hematoxylin and eosin as described (Imrie and Sadler, 2010) and imaged on an Olympus BX41 Clinical Microscope equipped with a Nikon DS-Ri1 digital camera. Histological criteria used for scoring tumors are described in Figure S4A–B.\nImmunoblotting\nLysates prepared from 15 embryos dissolved in 150 μ l protein lysis buffer were homogenized by sonication. 1 embryo equivalent was loaded per lane of an 8 or 12% SDS gel, transferred to nitrocellulose and blotted. HepG2 cell lysates were prepared as described (Tien et al., 2011).\n\n\nAntibodies\nAntibodies recognizing DNMT1 (1:1000 for blotting, 1:10 for immunofluorescence, Santa Cruz), UHRF1 (immunoblotting: 1:1000; BD Biosciences. Immunohistochemistry: 1:50; ab57083; Abcam), Tubulin (1:5000 DSHB), p89 PARP and total PARP (1:1000; Cell Signaling) and b-actin (1:2000; Sigma), 5MeC (1:500; Eurogentec), BrdU (1:200; BD Bioscences) and anti-rabbit or mouse conjugated to Alexa 555 or Alexa 488 (1:100; Invitrogen) were diluted in 10% FBS or 2% BSA in 1% Triton in PBS (PBST).\nZebrafish staining\nLarvae fixed in 4% PFA were washed in PBST and stained using the Senescence β - Galactosidase Staining Kit (Cell Signaling) or stained with CY3-streptavadin (CY3-SA; 1:300; Sigma) as described (Sadler et al., 2005). The left liver lobe area was measured using ImageJ.\nImmunofluorescence was carried out on whole fish or on livers dissected from fixed larvae. BrdU was added to larvae water (10 mM) for 4–6 hours followed by immediate fixation, dehydration in methanol, rehydration to PBST and permeablization with 10ug/ml Proteinase K. DNA was denatured in 1 or 2N HCl, renatured and blocked in 10% FBS or 1% BSA in PBST prior to immunofluorescence. DNA was stained with Hoechst 33342 (Sigma). A Leica SP5 DM confocal microscope was used for imaging.\n\n\nSlot blots\nGenomic DNA was denatured in 0.4M NaOH at 95°C for 10 minutes, neutralized in an equal volume of cold 2M ammonium acetate and 100 ng DNA was blotted in duplicate onto nitrocellulose membrane using a slot blot apparatus and washed in 2x SSC and vacuum\n\n\nNIH-PA Author Manuscript\nbaked at 80°C for 2 hours. Half was stained with 0.2% methylene blue in 0.3 M NaOAc and the other with anti-5MeC followed by HRP-conjugated anti-mouse (1:2000) and visualized by Chemilumiescence (Roche). Image J was used to quantify 5MeC and methylene blue intensity and 5MeC levels were determined by normalizing to total DNA.\n\n\nCell culture\nPrimary mouse hepatocytes were isolated and plated in triplicate at 50% confluency then treated with 5 μ m 5-Aza or DMSO for 24 hours and RNA was isolated.\nNIH-3T3 cells were transfected in triplicate with pCDNA3.1 lacking an insert (control) or containing human RAS, UHRF1 or both and were re-transfected 24 hours later. 48 hours after the second transfection, 700 mg /ml Neomycin was added for 7 days. 10,000 cells per condition were plated on 0.3% soft agar layered on top of 0.6% soft agar. Media was changed every 3 rd day for 2 weeks and plates were stained with 0.005% crystal violet and colony number was counted.\nHepG2 cells cultured in DMEM supplemented with 10% (v/v) FBS and 5% (w/v) penicillin- streptomycin were transfected twice, 24 hours apart, with 20 nM siRNA targeting firefly luciferase (GL2; Dhamarcon) or UHRF1 targeting si-A and si-B using RNAiMAX (Invitrogen) as described (Tien et al., 2011).\n\n\nHuman tissue samples\nPathologically staged human tumors, dysplastic foci and normal liver samples were obtained from the HCC Genomic Consortium (Mount Sinai Hospital, NY; Instituto Nazionale dei Tumori, Milan and Hospital Clinic, Barcelona). The study was approved by the IRB of each institution and informed consent was obtained from all participants. TaqMan Probes were used to analyze UHRF1 expression by qPCR as described (Villanueva et al., 2008). Samples were grouped into a training set (9 normal liver, 18 low- and high-grade dysplastic nodules, and 40 pathologically staged HCCs; (Llovet et al., 2006; Wurmbach et al., 2007)) and a validation set (7 normal liver and 69 HCC; (Villanueva et al., 2008)). Integrative genomic and clinical data analysis was performed on 71 of the hepatitis C-related, surgically treated HCC patients (NCBI Gene Expression Omnibus, accession numbers GSE9829 and GSE44970). Liver pathologist, ST, scored UHRF1 staining of 52 paraffin embedded tumors from the training set.\n\n\nBioinformatics and statistical analysis\n71 patients with HCC were grouped based on the median expression level of UHRF1 in the tumors determined by qPCR (UHRF1-high; n=35 or low; n=36). Presence of aggressive human HCC gene signatures (Table S1) were evaluated in the transcriptome dataset using nearest template prediction method (Hoshida, 2010) implemented by the GenePattern genomic analysis toolkit ( www.broadinstitute.org/genepattern ) based on prediction confidence p&lt;0.05. The senescence-related TP53 target genes were obtained from Molecular Signature Database (MSigDB, www.broadinstitute.org/msigdb , TANG_SENESCENCE_TP53_TARGETS_UP and _DN; (Tang et al., 2007)) and from NCBI Gene Expression Omnibus (GSE39469; (Lujambio et al., 2013)). Mouse genes from normalized microarray data were converted to human orthologs based on a mapping table ( www.informatics.jax.org ). Differentially expressed genes between senescent and proliferative cells were identified by using Bayesian T-test implemented in Cyber-T software (molgen51.biol.rug.nl/cybert) at the significance threshold of Posterior Probability of Differential Expression &gt;0.998 after excluding less variable genes with coefficient of variation ≤ 0.1 across the samples. Expression pattern of each signature was assessed by nearest template prediction algorithm. Significance of induction or suppression of each\n\n\nNIH-PA Author Manuscript\nsignature was quantitatively measured by nominal p-value of cosine distance. Pearson correlation test determined the significance between induction/suppression of gene signatures with UHRF1 mRNA expression level as measured by qPCR. Statistical difference between groups was assessed by Wilcoxon rank-sum test and Fisher’s exact test for continuous and categorical variables, respectively. Bonferroni correction for multiple hypothesis testing was applied when appropriate.\nSurvival analysis was performed by using Kaplan-Meier estimator and log-rank test. All analyses were performed using R statistical package ( www.r-project.org ) and GenePattern genomic analysis tool kit ( www.broadinstitute.org/genepattern ). Gene expression levels and liver size were compared using Student’s T or Mann Whitney U tests\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nSupport was provided by The Breast Cancer Alliance and the March of Dimes (to KCS), a Pilot Project from the DFCI NCI Cancer Center (5P30CA006516-45; to CU), the NIH (5R01DK080789-02 to CU and KCS,\n1R01DK099558 to YH, 1R01DK076986 to JML, F30DK094503 to VJ and T32CA078207-14 for YC), the European Commission 7th Framework Programme FP7-Health 2010 (Heptromic #259744 to YH and JML), The Samuel Waxman Cancer Research Foundation, The Spanish National Health Institute (SAF-2010-16055) and the Asociación Española Contra el Cáncer (JML). Liz Loughlin, Brandon Kent, Meghan Walsh, Alex Mir, Laia Cabellos, Helena Cornellà Vives and Sara Toffanin provided expert technical assistance. We are grateful to Sam Sidi for tp53−/− fish, stimulating discussions and critical reading of the manuscript.\n\n\nReferences\nAnderson RM, Bosch JA, Goll MG, Hesselson D, Dong PD, Shin D, Chi NC, Shin CH, Schlegel A, Halpern M, Stainier DY. Loss of Dnmt1 catalytic activity reveals multiple roles for DNA methylation during pancreas development and regeneration. Dev Biol. 2009\nArita K, Ariyoshi M, Tochio H, Nakamura Y, Shirakawa M. Recognition of hemi-methylated DNA by the SRA protein UHRF1 by a base-flipping mechanism. Nature. 2008; 455:818–821. [PubMed: 18772891]\nAvvakumov GV, Walker JR, Xue S, Li Y, Duan S, Bronner C, Arrowsmith CH, Dhe-Paganon S. Structural basis for recognition of hemi-methylated DNA by the SRA domain of human UHRF1. Nature. 2008; 455:822–825. [PubMed: 18772889]\nBabbio F, Pistore C, Curti L, Castiglioni I, Kunderfranco P, Brino L, Oudet P, Seiler R, Thalman GN, Roggero E, et al. The SRA protein UHRF1 promotes epigenetic crosstalks and is involved in prostate cancer progression. Oncogene. 2012\nBerdasco M, Esteller M. Aberrant epigenetic landscape in cancer: how cellular identity goes awry. Dev Cell. 2010; 19:698–711. [PubMed: 21074720]\nBerghmans S, Murphey RD, Wienholds E, Neuberg D, Kutok JL, Fletcher CD, Morris JP, Liu TX, Schulte-Merker S, Kanki JP, et al. tp53 mutant zebrafish develop malignant peripheral nerve sheath tumors. Proc Natl Acad Sci U S A. 2005; 102:407–412. [PubMed: 15630097]\nBiniszkiewicz D, Gribnau J, Ramsahoye B, Gaudet F, Eggan K, Humpherys D, Mastrangelo MA, Jun Z, Walter J, Jaenisch R. Dnmt1 overexpression causes genomic hypermethylation, loss of imprinting, and embryonic lethality. Mol Cell Biol. 2002; 22:2124–2135. [PubMed: 11884600]\nBostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE. UHRF1 plays a role in maintaining DNA methylation in mammalian cells. Science. 2007; 317:1760–1764. [PubMed: 17673620]\nCalvisi DF, Ladu S, Gorden A, Farina M, Lee JS, Conner EA, Schroeder I, Factor VM, Thorgeirsson SS. Mechanistic and prognostic significance of aberrant methylation in the molecular pathogenesis of human hepatocellular carcinoma. J Clin Invest. 2007; 117:2713–2722. [PubMed: 17717605]\n\n\nNIH-PA Author Manuscript\nCheah MS, Wallace CD, Hoffman RM. Hypomethylation of DNA in human cancer cells: a site- specific change in the c-myc oncogene. J Natl Cancer Inst. 1984; 73:1057–1065. [PubMed: 6092764]\nChen T, Hevi S, Gay F, Tsujimoto N, He T, Zhang B, Ueda Y, Li E. Complete inactivation of DNMT1 leads to mitotic catastrophe in human cancer cells. Nat Genet. 2007; 39:391–396. [PubMed: 17322882]\nChiang DY, Villanueva A, Hoshida Y, Peix J, Newell P, Minguez B, LeBlanc AC, Donovan DJ, Thung SN, Sole M, et al. Focal gains of VEGFA and molecular classification of hepatocellular carcinoma. Cancer Res. 2008; 68:6779–6788. [PubMed: 18701503]\nChu J, Loughlin EA, Gaur NA, Senbanerjee S, Jacob V, Monson C, Kent B, Oranu A, Ding Y, Ukomadu C, Sadler KC. UHRF1 phosphorylation by cyclin A2/cyclin-dependent kinase 2 is required for zebrafish embryogenesis. Mol Biol Cell. 2012; 23:59–70. [PubMed: 22072796]\nChu J, Mir A, Gao N, Rosa S, Monson C, Sharma V, Steet R, Freeze HH, Lehrman MA, Sadler KC. A zebrafish model of congenital disorders of glycosylation with phosphomannose isomerase deficiency reveals an early opportunity for corrective mannose supplementation. Dis Model Mech. 2013; 6:95–105. [PubMed: 22899857]\nDecottignies A, d’Adda di Fagagna F. Epigenetic alterations associated with cellular senescence: a barrier against tumorigenesis or a red carpet for cancer? Semin Cancer Biol. 2011; 21:360–366. [PubMed: 21946622]\nDi Micco R, Sulli G, Dobreva M, Liontos M, Botrugno OA, Gargiulo G, dal Zuffo R, Matti V, d’Ario G, Montani E, et al. Interplay between oncogene-induced DNA damage response and heterochromatin in senescence and cancer. Nat Cell Biol. 2011; 13:292–302. [PubMed: 21336312]\nDu Z, Song J, Wang Y, Zhao Y, Guda K, Yang S, Kao HY, Xu Y, Willis J, Markowitz SD, et al. DNMT1 stability is regulated by proteins coordinating deubiquitination and acetylation-driven ubiquitination. Sci Signal. 2010; 3:ra80. [PubMed: 21045206]\nEden A, Gaudet F, Waghmare A, Jaenisch R. Chromosomal instability and tumors promoted by DNA hypomethylation. Science. 2003; 300:455. [PubMed: 12702868]\nFairweather DS, Fox M, Margison GP. The in vitro lifespan of MRC-5 cells is shortened by 5- azacytidine-induced demethylation. Exp Cell Res. 1987; 168:153–159. [PubMed: 2430819]\nFeng S, Cokus SJ, Zhang X, Chen PY, Bostick M, Goll MG, Hetzel J, Jain J, Strauss SH, Halpern ME, et al. Conservation and divergence of methylation patterning in plants and animals. Proc Natl Acad Sci U S A. 2010; 107:8689–8694. [PubMed: 20395551]\nGaudet F, Hodgson JG, Eden A, Jackson-Grusby L, Dausman J, Gray JW, Leonhardt H, Jaenisch R. Induction of tumors in mice by genomic hypomethylation. Science. 2003; 300:489–492. [PubMed: 12702876]\nGaudet F, Rideout WM 3rd, Meissner A, Dausman J, Leonhardt H, Jaenisch R. Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. Mol Cell Biol. 2004; 24:1640–1648. [PubMed: 14749379]\nHashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X. The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. Nature. 2008; 455:826–829. [PubMed: 18772888]\nHoshida Y. Nearest template prediction: a single-sample-based flexible class prediction with confidence assessment. PLoS One. 2010; 5:e15543. [PubMed: 21124904]\nHoward G, Eiges R, Gaudet F, Jaenisch R, Eden A. Activation and transposition of endogenous retroviral elements in hypomethylation induced tumors in mice. Oncogene. 2008; 27:404–408. [PubMed: 17621273]\nImrie D, Sadler KC. White adipose tissue development in zebrafish is regulated by both developmental time and fish size. Dev Dyn. 2010; 239:3013–3023. [PubMed: 20925116]\nJackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G, Dausman J, Lee P, Wilson C, Lander E, Jaenisch R. Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. Nat Genet. 2001; 27:31–39. [PubMed: 11137995]\nJin W, Chen L, Chen Y, Xu SG, Di GH, Yin WJ, Wu J, Shao ZM. UHRF1 is associated with epigenetic silencing of BRCA1 in sporadic breast cancer. Breast Cancer Res Treat. 2009\nJirtle RL. IGF2 loss of imprinting: a potential heritable risk factor for colorectal cancer. Gastroenterology. 2004; 126:1190–1193. [PubMed: 15057757]\n\n\nNIH-PA Author Manuscript\nKarpf AR, Matsui S. Genetic disruption of cytosine DNA methyltransferase enzymes induces chromosomal instability in human cancer cells. Cancer Res. 2005; 65:8635–8639. [PubMed: 16204030]\nKarpf AR, Moore BC, Ririe TO, Jones DA. Activation of the p53 DNA damage response pathway after inhibition of DNA methyltransferase by 5-aza-2 ′ -deoxycytidine. Mol Pharmacol. 2001; 59:751–757. [PubMed: 11259619]\nKwan KM, Fujimoto E, Grabher C, Mangum BD, Hardy ME, Campbell DS, Parant JM, Yost HJ, Kanki JP, Chien CB. The Tol2kit: a multisite gateway-based construction kit for Tol2 transposon transgenesis constructs. Dev Dyn. 2007; 236:3088–3099. [PubMed: 17937395]\nLi E, Beard C, Jaenisch R. Role for DNA methylation in genomic imprinting. Nature. 1993; 366:362– 365. [PubMed: 8247133]\nLiu X, Gao Q, Li P, Zhao Q, Zhang J, Li J, Koseki H, Wong J. UHRF1 targets DNMT1 for DNA methylation through cooperative binding of hemi-methylated DNA and methylated H3K9. Nat Commun. 2013; 4:1563. [PubMed: 23463006]\nLlovet JM, Burroughs A, Bruix J. Hepatocellular carcinoma. Lancet. 2003; 362:1907–1917. [PubMed: 14667750]\nLlovet JM, Chen Y, Wurmbach E, Roayaie S, Fiel MI, Schwartz M, Thung SN, Khitrov G, Zhang W, Villanueva A, et al. A molecular signature to discriminate dysplastic nodules from early hepatocellular carcinoma in HCV cirrhosis. Gastroenterology. 2006; 131:1758–1767. [PubMed: 17087938]\nLlovet JM, Ricci S, Mazzaferro V, Hilgard P, Gane E, Blanc JF, de Oliveira AC, Santoro A, Raoul JL, Forner A, et al. Sorafenib in advanced hepatocellular carcinoma. N Engl J Med. 2008; 359:378– 390. [PubMed: 18650514]\nLujambio A, Akkari L, Simon J, Grace D, Tschaharganeh DF, Bolden JE, Zhao Z, Thapar V, Joyce JA, Krizhanovsky V, Lowe SW. Non-cell-autonomous tumor suppression by p53. Cell. 2013; 153:449–460. [PubMed: 23562644]\nMcDuff FK, Turner SD. Jailbreak: oncogene-induced senescence and its evasion. Cell Signal. 2011; 23:6–13. [PubMed: 20633638]\nMudbhary R, Sadler KC. Epigenetics, development, and cancer: Zebrafish make their mark. Birth Defects Res C Embryo Today. 2011; 93:194–203. [PubMed: 21671358]\nNishiyama A, Yamaguchi L, Sharif J, Johmura Y, Kawamura T, Nakanishi K, Shimamura S, Arita K, Kodama T, Ishikawa F, et al. Uhrf1-dependent H3K23 ubiquitylation couples maintenance DNA methylation and replication. Nature. 2013; 502:249–253. [PubMed: 24013172]\nPogribny IP, Rusyn I. Role of epigenetic aberrations in the development and progression of human hepatocellular carcinoma. Cancer Lett. 2012\nQin W, Leonhardt H, Spada F. Usp7 and Uhrf1 control ubiquitination and stability of the maintenance DNA methyltransferase Dnmt1. J Cell Biochem. 2011; 112:439–444. [PubMed: 21268065]\nRai K, Nadauld LD, Chidester S, Manos EJ, James SR, Karpf AR, Cairns BR, Jones DA. Zebra fish Dnmt1 and Suv39h1 regulate organ-specific terminal differentiation during development. Mol Cell Biol. 2006; 26:7077–7085. [PubMed: 16980612]\nRai K, Sarkar S, Broadbent TJ, Voas M, Grossmann KF, Nadauld LD, Dehghanizadeh S, Hagos FT, Li Y, Toth RK, et al. DNA demethylase activity maintains intestinal cells in an undifferentiated state following loss of APC. Cell. 2010; 142:930–942. [PubMed: 20850014]\nSadler KC, Amsterdam A, Soroka C, Boyer J, Hopkins N. A genetic screen in zebrafish identifies the mutants vps18, nf2 and foie gras as models of liver disease. Development. 2005; 132:3561–3572. [PubMed: 16000385]\nSadler KC, Krahn KN, Gaur NA, Ukomadu C. Liver growth in the embryo and during liver regeneration in zebrafish requires the cell cycle regulator, uhrf1. Proc Natl Acad Sci U S A. 2007; 104:1570–1575. [PubMed: 17242348]\nSharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA, Shinga J, Mizutani-Koseki Y, Toyoda T, Okamura K, et al. The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. Nature. 2007; 450:908–912. [PubMed: 17994007]\nTang X, Milyavsky M, Goldfinger N, Rotter V. Amyloid-beta precursor-like protein APLP1 is a novel p53 transcriptional target gene that augments neuroblastoma cell death upon genotoxic stress. Oncogene. 2007; 26:7302–7312. [PubMed: 17533371]\nTaylor EM, Bonsu NM, Price RJ, Lindsay HD. Depletion of Uhrf1 inhibits chromosomal DNA replication in Xenopus egg extracts. Nucleic Acids Res. 2013\nTien AL, Senbanerjee S, Kulkarni A, Mudbhary R, Goudreau B, Ganesan S, Sadler KC, Ukomadu C. UHRF1 depletion causes a G2/M arrest, activation of DNA damage response and apoptosis. Biochem J. 2011; 435:175–185. [PubMed: 21214517]\nTischoff I, Tannapfe A. DNA methylation in hepatocellular carcinoma. World J Gastroenterol. 2008; 14:1741–1748. [PubMed: 18350605]\nTittle RK, Sze R, Ng A, Nuckels RJ, Swartz ME, Anderson RM, Bosch J, Stainier DY, Eberhart JK, Gross JM. Uhrf1 and Dnmt1 are required for development and maintenance of the zebrafish lens. Dev Biol. 2011; 350:50–63. [PubMed: 21126517]\nUnoki M, Daigo Y, Koinuma J, Tsuchiya E, Hamamoto R, Nakamura Y. UHRF1 is a novel diagnostic marker of lung cancer. Br J Cancer. 2010; 103:217–222. [PubMed: 20517312]\nVentura A, Kirsch DG, McLaughlin ME, Tuveson DA, Grimm J, Lintault L, Newman J, Reczek EE, Weissleder R, Jacks T. Restoration of p53 function leads to tumour regression in vivo. Nature. 2007; 445:661–665. [PubMed: 17251932]\nVillanueva A, Chiang DY, Newell P, Peix J, Thung S, Alsinet C, Tovar V, Roayaie S, Minguez B, Sole M, et al. Pivotal role of mTOR signaling in hepatocellular carcinoma. Gastroenterology. 2008; 135:1972–1983. 1983 e1971–1911. [PubMed: 18929564]\nVillanueva A, Hoshida Y, Battiston C, Tovar V, Sia D, Alsinet C, Cornella H, Liberzon A, Kobayashi M, Kumada H, et al. Combining clinical, pathology, and gene expression data to predict recurrence of hepatocellular carcinoma. Gastroenterology. 2011; 140:1501–1512.e1502. [PubMed: 21320499]\nWang F, Yang YZ, Shi CZ, Zhang P, Moyer MP, Zhang HZ, Zou Y, Qin HL. UHRF1 Promotes Cell Growth and Metastasis Through Repression of p16(ink4a) in Colorectal Cancer. Ann Surg Oncol. 2012\nWeber M, Schubeler D. Genomic patterns of DNA methylation: targets and function of an epigenetic mark. Curr Opin Cell Biol. 2007; 19:273–280. [PubMed: 17466503]\nWurmbach E, Chen YB, Khitrov G, Zhang W, Roayaie S, Schwartz M, Fiel I, Thung S, Mazzaferro V, Bruix J, et al. Genome-wide molecular profiles of HCV-induced dysplasia and hepatocellular carcinoma. Hepatology. 2007; 45:938–947. [PubMed: 17393520]\nXue W, Zender L, Miething C, Dickins RA, Hernando E, Krizhanovsky V, Cordon-Cardo C, Lowe SW. Senescence and tumour clearance is triggered by p53 restoration in murine liver carcinomas. Nature. 2007; 445:656–660. [PubMed: 17251933]\nYamada Y, Jackson-Grusby L, Linhart H, Meissner A, Eden A, Lin H, Jaenisch R. Opposing effects of DNA hypomethylation on intestinal and liver carcinogenesis. Proc Natl Acad Sci U S A. 2005; 102:13580–13585. [PubMed: 16174748]\nYou JS, Jones PA. Cancer genetics and epigenetics: two sides of the same coin? Cancer Cell. 2012; 22:9–20. [PubMed: 22789535]\n\n\nHighlights\n• UHRF1 overexpression induces DNA hypomethylation\n• Liver specific overexpression of UHRF1 in zebrafish causes senescence and HCC\n• Human HCCs with high UHRF1 are aggressive and inactivate TP53-mediated senescence\nNIH-PA Author Manuscript\n\n\nSIGNIFICANCE\nGlobal DNA hypomethylation occurs in most types of cancer and can induce genomic instability and widespread changes in gene expression. UHRF1 is a key regulator of DNA methylation, and here we show that UHRF1 overexpression causes HCC in zebrafish without any other genetic alteration, demonstrating that it is an oncogene. High UHRF1 expression causes DNA hypomethylation and tp53-mediated senescence, which serves to restrict transformation of UHRF1 overexpressing cells. High UHRF1 expression in human HCC correlates with a poor prognosis, genomic instability, TP53 mutation and repression of the TP53-mediated senescence program. We conclude that UHRF1 is an oncogene that promotes widespread DNA hypomethylation, an epigenetic hallmark of cancer cells, and that UHRF1 overexpression drives tumorigenesis when senescence is bypassed.\nNIH-PA Author Manuscript\nFigure 1. High UHRF1 expression causes global DNA hypomethylation (A) 5MeC levels and total DNA stained with methylene blue were measured in 5 dpf control and UHRF1-GFP High livers (n=4) and liver-less carcasses (n=3). The ratio of 5MeC to total DNA was averaged and normalized to controls. Student’s T-test was used to determine p values. (B) Confocal stacks of livers (top) and fins (bottom) from 5 dpf nls-mCherry and UHRF1-GFP High larvae stained with anti-5MeC. Since a hepatocyte specific promoter was used for transgenesis, there was no transgene expression in the fin. (C) Dnmt1 is uniform in the hepatocyte nucleus of 4 dpf nls-mCherry larvae but is found in GFP-containing punctae in UHRF1-GFP High hepatocytes. Arrows point to cells that do not express GFP and have Dnmt1 distribution pattern similar to controls. (D) Tg(hsp70I:UHRF1-EGFP) and non- transgenic controls were heat shocked at 37° C for 1 hour at 24 and 27 hpf, treated with 10 μ M MG132 or DMSO at 28 hpf and collected at 34 hpf for immunoblotting. (E) Dnmt1 levels normalized to tubulin were averaged from 6 experiments. Student’s T-test was used to determine p values; n.s.: not significant; error bars represent SD. See also Figure S1.\n\n\nNIH-PA Author Manuscript\nFigure 2. UHRF1-induced hypomethylation reduces liver size (A) Individual larvae were imaged daily from 3–10 dpf. (B) 5 dpf UHRF1-GFP High larvae display a range of liver sizes scored as “normal” or “small”‘ (C) 3 clutches were scored according to criteria in B; n= number of larvae. Fisher’s exact test was used to determine p value. (D) The area of the left liver lobe was measured in 5 dpf fish from 2 clutches. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. Student’s T-test was used to determine p value. (E) UHRF1-GFP High larvae were sorted by liver size on 5 dpf and tracked daily for survival to 20 dpf. Data are pooled from three clutches. (F) UHRF1-GFP High and control larvae were treated with 50 μ M 5-Aza from 2.5–5 dpf and scored for liver size in 6 clutches. Fisher’s exact test was used to determine p values. (G) UHRF1-GFP High embryos were injected with mRNA encoding dnmt1 or Mpi before 1 hpf. The percent of fish with a normal liver size was scored at 5 dpf in 6 clutches. See also Figure S2.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\nFigure 3. UHRF1 overexpression in hepatocytes induces Tp53-mediated senescence (A) Intense senescence associated β -galactosidase (SA- β -gal) staining was detected in the liver (outlined) of 5 dpf UHRF1-GFP High larvae compared to light or no staining in controls. (B) 5 dpf fish from 5 clutches were scored for hepatic SA- β -gal staining. *** indicates p&lt;0.0001 by Fisher’s exact test. (C) Nuclear size was measured in hepatocytes of a single control or UHRF1-GFP High 5 dpf liver and cells were stratified according to GFP expression. Inset shows confocal stack of the DNA organized into foci. ** and *** indicates p&lt;0.001 or 0.0001, respectively compared to nuclear size in nls-mCherry larvae. (D) BrdU positive cells and the total number of transgene-expressing hepatocytes in nls-mCherry and UHRF1-GFP High larvae (Bottom) 5 dpf larvae. A Fisher’s exact test was used to calculate\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\np value. In nls-mCherry larvae, most BrdU positive cells also express the transgene whereas the BrdU positive cells in UHRF1-GFP High livers did not express GFP (white arrows in magnified regions which are marked by the white box). (E) Heatmap of log2 values from RNAseq shows cell cycle regulators are down and Tp53 target genes (marked by *) are up in UHRF1-GFP High 5 dpf livers. (F) tp53 and cdkn1a mRNA expression were induced on 5 dpf and down regulated on 20 dpf in UHRF1-GFP High livers. * indicates p=0.05; *** indicates p=0.001 calculated by 1 sample Student’s T-test. Error bars represent SD. (G) 5- Aza induces Tp53 expression in primary mouse hepatocytes. Student’s T-test was used to determine p value with SD indicated by the error bars across 3 replicates. tp53 +/− in UHRF1-GFP High larvae significantly reduced SA- β -gal staining in the liver (2 clutches) (H), and increased the percent of larvae with normal liver size (I), the area of the left liver lobe (J), and survival at 5 dpf (K). p values were calculated with a Fisher’s test with Freeman-Halton extension (H), Fisher’s exact test (I), and Student’s T-test. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. (J). See also Figure S3.\nNIH-PA Author Manuscript\nFigure 4. UHRF1 is an oncogene\n(A) Atypical cells, dysplastic foci (outlined) and HCC are apparent in H&amp;E stained UHRF1- GFP High livers. BD: bile duct, MF: mitotic figure. (B) Incidence of normal and atypical hepatocytes, dysplastic foci and cancer in the liver of UHRF1-GFP High fish on WT or tp53 +/− background. (C) NIH-3T3 cell growth in soft agar is enhanced when UHRF1 overexpression is combined with RAS (n=3). p value was calculated by Student’s T-test and error bars represent the SD. (D) Hepatic SA- β -gal staining patterns in UHRF1-GFP High larvae changes as fish age. Images of 8 dpf larvae illustrate the SA- β -gal staining patterns that were scored in the time course shown in the graph. A significant increase in the number of UHRF1-GFP High fish with intense or punctate SA- β -gal compared to controls at all time points (p&lt;0.01 by Fisher’s exact test) except at 4 dpf; not significant=n.s. (E) BrdU incorporation in the liver on 11 dpf is 5 times higher in UHRF1-GFP High fish than in\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\ncontrols. Total # of cells counted is indicated with n= # of clutches assessed. Fisher’s exact test was used to calculate p value. See also Figure S4.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 5. UHRF1 mRNA and protein are overexpressed in HCC (A) UHRF1 detected by qRT-PCR in 18 pre-neoplastic lesions and 40 HCCs from HCV infected patients compared to expression in 9 normal livers. Horizontal line indicates median. (B) Immunohistochemistry for UHRF1 protein (brown) was evaluated in 52 of the same HCCs examined in A plus 5 normal liver samples. Fisher’s exact test was used to calculate p value. 71 of the HCV-associated HCCs analyzed by qPCR were grouped into high (n=35) and low (n=36) UHRF1 expressing tumors based on the median log2-fold change of 3.64. (C) HepG2 cells transfected with control siRNA (GL2) or two different siRNAs targeting UHRF1 described in (Tien et al., 2011) were blotted for UHRF1, cleaved and total PARP (arrow indicates full length, * indicates cleaved protein). (D) Vascular invasion (33 high and 34 low tumors, 4 missing values), (E) serum AFP (29 UHRF1-high and 29 low tumors, 13 missing values), (F) early (&lt;2 years) and (G) late (&gt;2 years; 32 UHRF1-high and 35 low tumors, 4 missing values) tumor recurrence and (H) overall survival after surgery (32 high and 35 low tumors, 4 missing values) were stratified according to UHRF1-expression. Continuous and categorical variables were assessed by Wilcoxon rank-sum test and Fisher’s exact test, respectively. Clinical outcome difference was evaluated by log-rank test. In box and whisker plots, boxes represent the 75th and 25th\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\npercentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S5.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 6. High UHRF1 expression defines a subclass of tumors with inactivated tp53, repression of senescence and chromosomal instability\n(A) The 71 human HCC tumors analyzed in Figure 5C–G were rank-ordered according to UHRF1 expression by qPCR and classified as high (&lt;median, red; n=35) or low (≥median, blue; n=36). The presences of aggressive human HCC gene signatures from published studies and of TP53 inactivating mutations are indicated by red and black boxes, respectively. TP53-mediated senescence gene signatures (Lujambio et al., 2013; Tang et al., 2007) are displayed as a range from repressed (blue) to activated (red). (B) Genome-wide profile of DNA copy number variation was obtained from GEO geneset GSE9829. (C) Proportion of genes with DNA copy number loss and gain in tumors according to UHRF1 expression. (D) Proportion of genes with DNA copy number loss according to UHRF1 expression and TP53 mutation status. (E) UHRF1 expression is significantly higher in tumors with TP53 mutations. (F) DNMT1 expression by microarray analysis is significantly correlated with UHRF1 expression assessed by qPCR in HCCs. In box and whisker plots,\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nboxes represent the 75th and 25th percentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S6 and Tables S1-S6.\nNIH-PA Author Manuscript\nFigure 7. Model of the relationship between UHRF1 overexpression, DNA hypomethylation, Tp53 mediated senescence, cancer and survival\nFactors investigated in this study are in solid black boxes with black lines indicating the correlations demonstrated in this work and gray lines indicating relationships that are speculative. Senescence reduces liver size and function and reduces larval survival while cancer occurs when senescence is bypassed and also reduces survival.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 2\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 4\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 6\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 7\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 8\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 9\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 10\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 11\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 12\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 13\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 14\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\n• Our studies in zebrafish and human tumors identify UHRF1 as an oncogene in HCC Mudbhary et al. Page 15\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 16\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 17\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 18\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 19\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 20\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 21\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 22\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 23\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 24\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 25\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 26\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 27\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 28\n\n\nTable 1\n\n\nHCC onset and incidence in Tg(fabp10:nls-mCherry) and Tg(fabp10:UHRF1-GFP) zebrafish. transgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI nls-mCherry Normal 100% 100% 100% 100% 100% 100% 100% 100% 100% 100% 65 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 8 8 8 11 nd nd 5 5 5 5 5 5 65 UHRF1-GFP Low Normal 100% 100% 100% 100% 100% 100% 100% 100% 60 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 7 9 10 12 nd nd 4 nd nd 6 6 6 60 UHRF1-GFP Medium Normal 90% (9) 100% (11) 50% (3) 28% (4) 66% (4) 80% (4) 50% (4) 50% (5) 81% (13) 57 66%\n\n\nAtypical cells 10% (1) 0% 50% (3) 57% (8) 33% (2) 20% (1) 50% (4) 20% (2) 44% (7) 28 32%\n\n\nDysplastic foci 0% 0% 0% 7% (1) 0% 0% 25% (2) 30% (3) 0% 6 7%\n\n\nTumor 0% 0% 0% 0% 0% 0% 0% 0% 6% (1) 1 1% n = 10 11 6 14 6 5 8 10 16 86 UHRF1-GFP High Normal 0% 0% 7% (1) 0% 0% 1 2% Atypical cells 100% (13) 100% (7) 46% (6) 17% (3) 75% (3) 36 61% Dysplastic foci 0% 0% 8% (1) 12% (2) 25% (1) 4 7% Tumor 0% 0% 46% (6) 76% (13) 0% 18 30% n = 13 7 13 17 4 54 UHRF1-GFP High/ p53 +/− Normal 0% 0 0% Atypical cells 13% (2) 2 13% Dysplastic foci 0% 0 0%\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 29\n\n\ntransgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI\n\n\nTumor 88% (14) 14 88% n = 16 16\n\n\nThe incidence (percent) and absolute number of fish (parenthesis) with cancer-relevant histological phenotypes were scored in a total of 281 fish from 4 transgenic lines. The total n for each time point is indicated. nd: not done. Some fish were diagnosed with more than one lesion. CI: Cumulative incidence.\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object.",
          "additional_kwargs": {}
        }
      }
    ]
  }
}
[llm/start] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] Entering LLM run with input:
{
  "prompts": [
    "System: You are an expert biological scientist, skilled at reading scientific papers.\nHuman: Read the text of a section from the methods section of a research paper shown below (delimited with triple backticks).\nSection Text:- '''UHRF1 overexpression drives DNA hypomethylation and\nhepatocellular carcinoma\nRaksha Mudbhary a,b,c , Yujin Hoshida a,d,l , Yelena Chernyavskaya a,b,l , Vinitha Jacob a,b,c , Augusto Villanueva f , M. Isabel Fiel e , Xintong Chen a,d , Kensuke Kojima a,d , Swan Thung e ,\nRoderick T. Bronson g , Anja Lachenmayer d , Kate Revill d , Clara Alsinet f , Ravi Sachidanandam h , Anal Desai i , Sucharita SenBanerjee i , Chinweike Ukomadu i , Josep M. Llovet a,d,f,j , and Kirsten C. Sadler a,b,c,k\na Department of Medicine/Division of Liver Diseases, Icahn School of Medicine at Mount Sinai New York, NY 10029\nb Department of Developmental and Regenerative Biology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nc Graduate School of Biological Sciences, Icahn School of Medicine at Mount Sinai New York, NY 10029\nd Liver Cancer Program/Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai New York, NY 10029\ne Department of Pathology, Icahn School of Medicine at Mount Sinai New York, NY 10029\nf HCC Translational Research Laboratory, IDIBAPS, CIBEREHD, Hospital Clinic, University of Barcelona, Catalonia, Spain\ng Rodent Histopathology Core Dana Farber/Harvard Cancer Center, Harvard Medical School Boston, MA 02115\nh Department of Genetics and Genomics, Icahn School of Medicine at Mount Sinai New York, NY 10029\nI Division of Gastroenterology, Department of Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115\nj Institució Catalana de Recerca i Estudis Avançats Lluís Companys, 23, Barcelona – Spain\n\n\nSUMMARY\nUHRF1 is an essential regulator of DNA methylation that is highly expressed in many cancers. Here, we use transgenic zebrafish, cultured cells and human tumors to demonstrate that UHRF1 is an oncogene. UHRF1 overexpression in zebrafish hepatocytes destabilizes and delocalizes DNMT1, causes DNA hypomethylation and Tp53-mediated senescence. Hepatocellular carcinoma\n© 2014 Elsevier Inc. All rights reserved.\nk Corresponding author: Kirsten.edepli@mssm.edu, 1 Gustave L. Levy Place, Box 1020, Icahn School of Medicine at Mount Sinai, New York, NY 10029, phone: 212-241-7152, fax: 212-860-9279. lthese authors contributed equally to this work.\n\n\nAccession Number\nData from RNAseq was deposited with accession number GSE52605.\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\nPublished in final edited form as:\nCancer Cell. 2014 February 10; 25(2): 196–209. doi:10.1016/j.ccr.2014.01.003.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n(HCC) emerges when senescence is bypassed. tp53 mutation both alleviates senescence and accelerates tumor onset. Human HCCs recapitulate this paradigm, as UHRF1 overexpression defines a subclass of aggressive HCCs characterized by genomic instability, TP53 mutation and abrogation of the TP53-mediated senescence program. We propose that UHRF1 overexpression is a mechanism underlying DNA hypomethylation in cancer cells and that senescence is a primary means of restricting tumorigenesis due to epigenetic disruption.\n\n\nINTRODUCTION\nThe expression of genes that encode readers and writers of the epigenetic code are widely deregulated across cancer types (You and Jones, 2012). This contributes to the massive gene expression changes and remodeling of the epigenetic landscape, a characteristic of many types of cancer. In particular, loss of global DNA methylation is a hallmark of cancer cells.\nDNA hypomethylation contributes to oncogenesis through multiple mechanisms, including chromosomal instability (Eden et al., 2003; Karpf and Matsui, 2005), derepression of imprinted genes (Berdasco and Esteller, 2010; Jirtle, 2004; Li et al., 1993), retrotransposon activation (Gaudet et al., 2004; Howard et al., 2008; Jackson-Grusby et al., 2001; Sharif et al., 2007) and aberrant gene expression, including induction of oncogenes (Cheah et al., 1984). Many studies have documented that expression of the core factors required for maintenance DNA methylation – i.e. DNA methyltransferase 1 (DNMT1) and ubiquitin-like with PHD and RING finger domains 1 (UHRF1) (Babbio et al., 2012; Jin et al., 2009; Unoki et al., 2010; Wang et al., 2012) – are significantly altered across cancer types. However, whether changes in the expression of these key factors are sufficient to alter the cancer cell methylome and drive carcinogenesis is unknown. Moreover, the mechanism by which DNA methylation is lost in cancer cells is poorly understood.\nThe cellular response to DNA hypomethylation varies by cell type, physiological context and degree of hypomethylation. In some cells, DNA hypomethylation induces tumor suppressive mechanisms, including apoptosis (Anderson et al., 2009; Biniszkiewicz et al., 2002; Chen et al., 2007; Jackson-Grusby et al., 2001) or senescence (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), whereas in other cells it blocks differentiation of progenitor cells (Rai et al., 2010) or causes cancer (Gaudet et al., 2003; Yamada et al., 2005). In part, the cellular response to loss of DNA methylation is dictated by the genomic region affected: hypomethylation of gene regulatory regions, such as promoters, can de-repress gene expression but hypomethylation of repetitive elements can reduce heterochromatin formation, promote recombination and genomic instability.\nUHRF1 plays an essential role in DNA methylation by both recognizing hemimethylated DNA generated during DNA replication and then recruiting DNMT1 to ensure faithful maintenance of DNA methylation patterns in daughter cells (Arita et al., 2008; Avvakumov et al., 2008; Bostick et al., 2007; Hashimoto et al., 2008; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Consequently, UHRF1 depletion results in global DNA hypomethylation (Bostick et al., 2007; Feng et al., 2010; Sharif et al., 2007; Tittle et al., 2011). Conversely, UHRF1 may also limit DNA methylation by targeting DNMT1 for ubiquitin-mediated degradation (Du et al., 2010; Qin et al., 2011) or by delocalizing DNMT1 (Sharif et al., 2007). How UHRF1 overexpression impacts the methylome is not known.\nWe previously reported that uhrf1 mutation in zebrafish blocks liver outgrowth in embryos and regeneration in adults (Sadler et al., 2007) and that depleting UHRF1 from cancer cells induces apoptosis (Tien et al., 2011). Here, we tested the hypothesis that UHRF1\n\n\nNIH-PA Author Manuscript\noverexpression would alter global DNA methylation, and promote hepatocellular carcinoma (HCC).\n\n\nRESULTS\n\n\nHigh UHRF1 expression causes DNA hypomethylation\nUHRF1 is required for DNA methylation, as it recruites DNMT1 to hemimethylated DNA during DNA replication (Bostick et al., 2007; Feng et al., 2010; Liu et al., 2013; Nishiyama et al., 2013; Sharif et al., 2007). Paradoxically, UHRF1 also serves as an ubiquitin ligase that targets DNMT1 for degradation (Du et al., 2010; Qin et al., 2011). To determine how UHRF1 overexpression impacts DNA methylation, we generated transgenic zebrafish expressing human UHRF1 fused to GFP (hsa.UHRF-GFP) under the hepatocyte specific fabp10 promoter (Tg(fabp10:hsa.UHRF1-GFP) (Chu et al., 2012)).\nHuman and zebrafish UHRF1 are 66% identical and the ability of human UHRF1 to rescue zebrafish embryos depleted of uhrf1 (VJ and KCS, unpublished and (Chu et al., 2012)) indicates they are functional orthologs. Expression is first detected in hepatocytes on 3 dpf and 5 dpf, nuclear UHRF1-GFP is easily detected at variable levels in hepatocytes (Figure S1A and (Chu et al., 2012)). We isolated an allelic series of Tg(fabp10:hsa.UHRF1-GFP) transgenics expressing a range of UHRF1 levels (Figure S1A–B; hereafter referred to UHRF1-GFP High, Medium and Low).\nWe probed genomic DNA isolated from the liver of 5 dpf larvae from each line with an antibody specific for 5-methyl cytosine (5MeC) to assess DNA methylation and found significant hypomethylation only in the liver of UHRF1-GFP High larvae (38% of controls; p=0.007; Figure 1A and Figure S1C). 5MeC immunofluorescence showed DNA methylation as uniformly distributed in the nuclei of hepatocytes from control larvae that express nuclear-localized fluorescent mCherry under the fabp10 promoter (Tg(fabp10:nls-mCherry); abbreviated to nls-mCherry) but only dim staining in hepatocytes of UHRF1-GFP High larvae (Figure 1B). Since Equivalent levels of 5MeC were detected in the liverless carcasses (Figure 1A) or fins (Figure 1B), we conclude that DNA hypomethylation was specific to the liver.\nWe hypothesized that UHRF1-induced DNA hypomethylation could be caused by mislocalization or destabilization of Dnmt1. Immunofluorescence revealed uniform Dnmt1 distribution in the nucleoplasm of nls-mCherry hepatocytes (Figure 1C) but in UHRF1-GFP high hepatocytes, it was concentrated in nuclear foci that contained UHRF1-GFP (Figure 1C). Interestingly, the range of UHRF1-GFP expression within cells of the same liver revealed that in cells with high GFP, Dnmt1 was dim and punctate, but levels and distribution pattern was similar to control hepatocytes in those cells with low or no GFP (arrows, Figure 1C). Thus, high expression of UHRF1 co-localizes with Dnmt1 and redistributes it to intra-nuclear structures reminiscent of senescence associated heterochromatin foci (Di Micco et al., 2011).\nEndogenous Dnmt1 in the liver was below the levels detectable by immunoblotting (not shown), so we used a transgenic line expressing UHRF1-GFP under the inducible hsp70l promoter (Chu et al., 2012) to quantitatively assess the impact of UHRF1 overexpression on Dnmt1 stability at a developmental time when we could easily detect Dnmt1 levels by immunoblotting. We optimize a heat shock protocol that maximized UHRF1-GFP expression in Tg(hsp70l:UHRF1-GFP) larvae (Figure 1D) and found that UHRF1-GFP overexpression reduced Dnmt1 levels by 27% compared to non-heat-shocked transgenics (p=0.03; Figure 1E). Treatment with a non-toxic dose of the proteasome inhibitor, MG132, prevented the decrease in Dnmt1 induced by UHRF1 overexpression (p=0.01; Figure 1E).\n\n\nNIH-PA Author Manuscript\nNeither heat-shock nor MG132 significantly affected Dnmt1 protein levels in non-transgenic controls (Figure 1E). Thus, both Dnmt1 delocalization and destabilization could account for DNA hypomethylation caused by UHRF1 overexpression.\n\n\nDNA hypomethylation caused by UHRF1 overexpression reduces liver size\nUhrf1 in zebrafish is required for hepatic outgrowth and liver regeneration (Sadler et al., 2007). To determine if UHRF1 overexpression in hepatocytes affected hepatic outgrowth, we assessed liver size all transgenic lines (Figures 2A–C and S2A). Low and Medium expressing lines had no gross changes in liver size (Figure S2A), but 74% of UHRF1-GFP High larvae on 5 dpf had small livers (Figure 2C), with the median area of the left liver lobe reduced by 35% compared to controls (p&lt;0.001; Figures 2D, S2A), without reduction of total fish size (Figure S2B).\nLarvae with small livers appeared sick (see 10 dpf larvae in Figure 2A), and only 20% of UHRF1-GFP High fish survived to 20 dpf (Figure S2C and Figure 2E). Moreover, UHRF1- GFP High larvae with the “microliver” phenotype on 5 dpf had significantly higher mortality by 10 dpf (67%) than those that started with a normal sized liver (7%; Figure 2E). Thus, high UHRF1 expression in hepatocytes causes DNA hypomethylation, microliver and larval death.\nWe next investigated the relationship between DNA hypomethylation and the microliver phenotype by asking whether further reducing DNA methylation could enhance this phenotype or restoring Dnmt1 could suppress it. We previously reported that exposing embryos to 50 μ M of the Dnmt1 inhibitor, 5-azacytidine (5-Aza) from 0–5 dpf caused a profoundly small liver and DNA hypomethylation (Mudbhary and Sadler, 2011). By restricting the 5-Aza exposure time to 2.5–5 dpf, we reduced DNA methylation in the liver of control larvae by 40% (Figure S1C) which induced a moderately small liver in 16% of larvae (p=0.0035; Figure 2F). This same treatment of UHRF1-GFP High larvae significantly increased the percent with small livers (p&lt;0.0001; Figure 2F). While this could be attributed to DNA damage caused by 5-Aza, our finding that injecting UHRF1-GFP High embryos with mRNA encoding Dnmt1 modestly suppressed the percent of fish with a small liver (Figure 2G) and increased the average size of the left liver lobe by 10–15% (Figure S2D) suggests that hypomethylation, at least in part, contributes to the small liver phenotype of UHRF1-GFP High larvae.\n\n\nUHRF1 overexpression triggers Tp53-mediated senescence\nDNA hypomethylation can induce apoptosis, but we found no TUNEL positive cells on 5 dpf of UHRF1-GFP High livers (Figure S3A). However, senescence associated β - galactosidase (SA- β -gal) staining was detected throughout the liver of most UHRF1-GFP High 5 dpf larvae (n=93; Figure 3A–B), but not other transgenic lines (Figure S3B). Additionally, the DNA in control hepatocytes was evenly distributed throughout the uniformly sized nuclei compared to the large nuclei where DNA resembled senescence associated heterochromatic foci (inset, Figure 3C). Strikingly, the largest hepatocyte nuclei also had the brightest GFP (Figure 3C), suggesting the effect of UHRF1 overexpression cell autonomously affected nuclear morphology.\nSenescent cells do not divide, and we found significantly less BrdU incorporation in the liver of 5 dpf UHRF1-GFP High larvae compared to controls (p&lt;0.0001; Figure 3D). Interestingly, in nls-mCherry larvae, most BrdU incorporation was detected in hepatocytes that express nls-mCherry, but the only BrdU positive cells in UHRF1-GFP high larvae were negative for GFP (see inset in Figure 3D). RNAseq revealed some pro-proliferative genes (ccnd1 and myc) downregulated (Figure 3E), lending further support to the conclusion that\nsenescence is the primary response to high UHRF overexpression in hepatocytes during hepatic outgrowth.\nTp53 is a key mediator of senescence caused by DNA damage and oncogenic stress (Di Micco et al., 2011; McDuff and Turner, 2011; Ventura et al., 2007; Xue et al., 2007). RNAseq (Figure 3E) and qPCR analysis (Figure 3F) show that tp53 and its target genes, especially cdkn1a, are significantly induced in the liver of 5 dpf UHRF1-GFP High larvae, but then return to baseline by 20 dpf (Figure 3F). 5-Aza treatment of primary mouse hepatocytes induced Tp53 expression (Figure 3G), similar to the effects of 5-Aza treatment or Dnmt1 depletion in other cell types (Jackson-Grusby et al., 2001; Karpf et al., 2001) and in zebrafish (VJ and KCS, not shown). However, since no significant alteration in methylation of the tp53 promoter was detected in these models (not shown), we hypothesize that DNA methylation does not directly regulate Tp53 expression. Instead, DNA hypomethylation may induce tp53 by an indirect mechanism, such as increased DNA damage or genomic instability.\nA direct role for Tp53 in the phenotype of UHRF1-GFP High larvae was demonstrated by removing one copy of tp53. This reduced the incidence and intensity of SA- β -gal staining in the liver (Figure 3H), increased liver size (Figure 3I–J) and reduced mortality (Figure 3K). We thus propose a model whereby high UHRF1 causes DNA hypomethylation, induces Tp53-mediated senescence and prevents expansion of the hepatic bud, resulting in hepatic insufficiency and larval death.\n\n\nUHRF1 overexpression induces liver cancer in zebrafish\nTo determine if UHRF1 overexpression was sufficient to cause HCC, 281 control and UHRF1-GFP transgenics were collected between 5–300 dpf, serial sectioned and analyzed for atypical cells, dysplastic foci and HCC using histological criteria devised by two expert pathologists (RTB and MIF) based on disrupted tissue architecture, cell size, shape, nuclear structure and presence of mitotic figures (Figure S4A–B). Evidence of increased hepatocyte proliferation was detected in all UHRF1 overexpressing lines on 20 and 40 dpf (Figure S4C), but this was insufficient to cause HCC, as UHRF1-GFP Low fish were tumor free at all time points (Table 1). In contrast, UHRF1-GFP High fish developed atypical hepatocytes as early as 5 dpf, with an 8% incidence of dysplastic foci and 46% incidence of HCC by 15 dpf. On 20 dpf, 76% of fish had HCC (Figure 4A–B, Table 1). UHRF1-GFP Medium fish also developed atypical hepatocytes and dysplastic foci at young ages, and one large HCC was detected in a 60 dpf fish (Table 1). In a classical transformation assay using NIH-3T3, we found that UHRF1 cooperated with RAS to promote growth on soft agar (Figure 4C). These conclusively demonstrate that UHRF1 is an oncogene.\nWe found that UHRF1-GFP High larvae older than 8 dpf had a lower incidence and intensity of hepatic SA- β -gal staining (Figure 4D). Interestingly, in many of these fish, intense staining was distributed in a punctate pattern. By 20 dpf, 75% of fish had either punctate or no staining (Figure 4D), which is a striking correlation with the 70% incidence of HCC at this time point. This was mirrored by increased proliferation of liver cells, detected by increased BrdU incorporation in UHRF1-GFP High livers (22% vs. 4% in controls) at 11 dpf (p&lt;0.001; Figure 4E), and higher PCNA staining 20 and 40 dpf in all lines (Figure S4C). Loss of senescence was not attributed to reestablishment of DNA methylation or transgene silencing, as reduced 5MeC staining persisted in tumor cells (Figure S4D) and transgene expression was detectable in all 20 dpf fish (Figures S4E–F), albeit reduced from levels detected in 5 dpf livers.\nWe asked whether Tp53 epistatically interacted with UHRF1 overexpression to contribute to HCC by removing one copy of tp53 (Berghmans et al., 2005). The liver appeared normal in\n\n\nNIH-PA Author Manuscript\n13 tp53 +/+ fish without transgene expression (not shown), but in UHRF1-GFP High fish, tumor incidence on 15 dpf increased from 50% in WT to 87% in tp53 +/− fish (Figure 4B). Interestingly, in a single Tg(fabp10:UHRF1-GFP HIGH );p53 +/− , we found a tumor with immature cells resembling a cholangiocarcinoma. Thus, tp53 functions to suppress tumor formation and may alter the spectrum of tumors caused by UHRF1 overexpression.\n\n\nUHRF1 is upregulated in human HCC\nWe next investigated the relevance of UHRF1 expression in human HCC. We assessed UHRF1 expression by qPCR in 16 normal liver samples and in 2 cohorts of patients with dysplastic nodules or HCC: the 1 st cohort of 58 patients had hepatitis C infection (Figure 5A; (Wurmbach et al., 2007)) and the 2 nd cohort of 69 patients had HBV, alcohol and other etiologies (Figure S5A; (Villanueva et al., 2008)). Additional publically available transcriptome data sets from three other cohorts of HCCs (Figure S5B) and from lung, gastric, colorectal and breast cancer (Figure S5C) were also analyzed. All showed elevated UHRF1 in tumors, with expression elevated &gt;2-fold compared to controls in 17/18 dysplastic foci and 104/109 HCCs (Figures 5A and S5A). An average of 20- and 46-fold overexpression of UHRF1 was detected in advanced and very advanced HCCs. UHRF1 protein was barely detectable in 5 normal liver samples but highly expressed in 38/52 (73%) of the tumors analyzed for UHRF1 mRNA in Figure 5A (p&lt;0.003; Figure 5B). Thus, UHRF1 is overexpressed in HCCs of diverse etiologies as well as in other tumor types, and high UHRF1 expression is significantly associated with the most advanced tumors.\nTargeting UHRF1 in liver cancer cells (HepG2) using siRNA induced PARP cleavage (Figure 5C) and other markers of apoptosis (not shown), similar to results obtained using another HCC cell line (Hep3B; not shown) and colon cancer cells (Tien et al., 2011). Thus, blocking UHRF1 cancers such as HCC with high UHRF1 expression could be an effective means to induce tumor cell death.\nOf the 109 HCCs analyzed for UHRF1 expression by qPCR, 71 had multiple clinical and genomics parameters available (Chiang et al., 2008). These were rank-ordered based on UHRF1 expression determined by qPCR (Figure 6A). Tumors with expression above and equal to or below the median log2-fold change value of 3.64 were designated as “high” (n=35) and “low” (n=36), respectively (Figure 6A). High UHRF1 expressing tumors were associated with signs of poor clinical outcome: 80% had microvascular invasion (Figure 5D) and significantly higher alpha-feto protein (AFP) levels (Figure 5E), although AFP levels alone did not cooperate with UHRF1 to predict survival (not shown). Importantly, high UHRF1 expression significantly correlated with early (&lt;2 years; Figure 5F), but not late (Figure 5G) tumor recurrence and was inversely correlated with survival (Figure 5H). This suggests that high UHRF1 expression predicted recurrence of the primary tumor, causing decreased survival (Villanueva et al., 2011). Molecular signatures of aggressive HCC tumors compiled from several previous studies (Table S1) were concordantly and significantly enriched in high UHRF1 expressing tumors (Figure 6A). Moreover, high UHRF1 expressing tumors were distinguished by induction of pathways that drive the cell cycle, DNA replication and repair (Figure S6A, Tables S2 and S3). UHRF1 overexpression also significantly correlated with advanced stage prostate cancer, but not lung or colon (Figure S5D–G).\nHigh UHRF1 expression delineates a subclass of HCCs that have downregulated TP53- mediated senescence\nOur zebrafish studies demonstrated that bypass of Tp53-induced senescence is required for UHRF1 to act as an oncogene. Our analysis of human HCCs indicates that a similar paradigm occurs in these samples. First, inactivating mutations in TP53 significantly\n\n\nNIH-PA Author Manuscript\ncorrelated with high UHRF1 expression (Figure 6A). Second, many of the core enriched genes high UHRF1 expressing tumors are regulated by TP53 in senescent fibroblasts (Table S4 and (Tang et al., 2007), and high UHRF1 expressing tumors have downregulated the gene expression signature derived from TP53-induced senescence in fibroblasts (Table S5, Figure S6B; (Tang et al., 2007)) or in hepatic stellate cells (Table S6, Figure S6C; (Lujambio et al., 2013)). Fourth, significant correlation between UHRF1 overexpression, TP53 mutation and genome integrity in human HCCs indicates that these pathways act together: high UHRF1 expression (Figure 6B–C) and TP53 mutation (Figure 6D) are independently correlated with chromosomal loss but tumors that have both features display even more chromosomal loss (Figure 6D). Finally, UHRF1 expression was significantly higher in tumors with TP53 mutation (Figure 6E). UHRF1 overexpression did not correlate with copy number variation at the UHRF1 locus (Figure S6D) suggesting that a different mechanism drives UHRF1 overexpression.\nGenome wide DNA hypomethylation is found in most HCCs (Calvisi et al., 2007) and this rendered it difficult to correlate methylome changes with UHRF1 expression. However, DNMT1 expression was directly correlated with UHRF1 expression in HCC samples (Figure 6F). This may be a consequence of the high proliferation rate in these tumors or an induction of the methylation machinery to compensate for hypomethylation. Together, these data indicate that high UHRF1 expression in HCC defines a subset of aggressive tumors that have inactivated the TP53 induced senescence program, suggesting that in humans, as observed in zebrafish, TP53 acts as a tumor suppressor to restrict the oncogenic potential of UHRF1.\n\n\nDISCUSSION\nWe show that UHRF1 overexpression is sufficient to cause two oncogene-associated phenotypes: senescence and cancer. This defines UHRF1 as an epigenetic regulator that can cause cancer when overexpressed and is thus an oncogene. We hypothesize that UHRF1 overexpression is one mechanism by which cancer genomes can become hypomethylated, either via UHRF1-mediated Dnmt1 ubiquitination and degredation (Du et al., 2010; Qin et al., 2011) or by redistribution and/or sequestration of Dnmt1 away from DNA. Excess UHRF1 might also sequester the DNMT1 deubiquitinating enzyme, USP7 (Qin et al., 2011) to further promote DNMT1 ubiquitination and degradation.\nThe precise mechanism by which DNA hypomethylation causes cancer remains elusive. Both apoptosis and senescence serve to limit the propagation of cells with aberrant DNA methylation (Decottignies and d’Adda di Fagagna, 2011; Fairweather et al., 1987), however, once epigenetically altered cells escape these tumor suppressive mechanisms, they likely accumulate genetic lesions which predispose to cancer. Indeed, chromosomal instability and mitotic catastrophe occur following DNMT1 depletion (Chen et al., 2007; Karpf and Matsui, 2005; Weber and Schubeler, 2007) and transposons, which are heavily methylated in normal cells, could become activated and cause genomic instability upon UHRF1-induced DNA hypomethylation. Additionally, hypomethylated DNA is more likely to assume an open chromatin conformation, which may promote oncogene expression, although this possibility has not been fully evaluated. While the oncogenic role of UHRF1 could also be mediated by the impact of UHRF1 on other epigenetic marks or on DNA replication (Taylor et al., 2013), our data combined with findings from others (Eden et al., 2003; Gaudet et al., 2003) suggests that DNA hypomethylation is a likely mechanism driving UHRF1-mediated transformation.\nOur working model (Figure 7) proposes that UHRF1 overexpression causes DNA hypomethylation by reducing Dnmt1 levels and its access to DNA. Tp53 is induced in\n\n\nNIH-PA Author Manuscript\nresponse to either genomic instability or some unidentified direct mechanism, and causes hepatocyte senescence, which prevents hepatic expansion and results in larval death from hepatic insufficiency. We propose that Tp53 inactivation and senescence bypass allows for the unhindered proliferation and malignant transformation. Identifying the mechanism by which this tumor suppressive mechanism is overcome remains a central, unanswered question in cancer biology.\nStudies in a mouse liver cancer model show that Tp53 reactivation causes senescence, and these cells are then cleared by the immune system (Xue et al., 2007) and the liver is then repopulated with senescence-resistant tumor forming cells. Our finding that senescence decreases and BrdU incorporation increases in UHRF1-GFP high livers over time suggests a similar process at play. Moreover, BrdU incorporation in primarily GFP negative cells suggests the expansion of either immune cells or immature hepatic progenitors in response to senescent hepatocytes. The finding of a cholangiocarcinoma in a UHRF1-GFP High/ p53 +/− fish may indicate that a bipotential progenitor cell could be the tumor forming cell in this model. Additionally, our data indicate that there is a threshold effect of UHRF1 expression, in which the highest expressing cells undergo senescence and neighboring hepatocytes expressing UHRF1-GFP at levels below those detectable via microscopy undergo unhindered expansion.\nHCC is the third cause of cancer-related deaths globally (Llovet et al., 2003), yet curative therapies are limited, with Sorafanib as the only systemic therapy available for advanced cases (Llovet et al., 2008). Thus, there is an urgent and unmet need for novel therapies. HCC, like other cancers, is characterized by global DNA hypomethylation (Calvisi et al., 2007; Pogribny and Rusyn, 2012; Tischoff and Tannapfe, 2008), and high UHRF1 expression could be the cause. Our finding that UHRF1 depletion in HCC and other types of cancer cells causes apoptosis (Tien et al., 2011) presents UHRF1 as an attractive target for cancer therapy, whereby blocking UHRF1 could trigger massive epigenetic changes incompatible with cell survival or could reset the cancer cell methylome.\n\n\nEXPERIMENTAL PROCEDURES\n\n\nZebrafish maintenance and generation of transgenics\nZebrafish were maintained on a 14:10 hour light:dark cycle at 28°C. mRNA encoding zebrafish Dnmt1 (Rai et al., 2006) or mannose phosphate isomerase (Mpi; (Chu et al., 2013) as a control was injected into embryos just after fertilization.\nTg(fabp10:nls-mCherry) fish expressing nls-mCherry exclusively in hepatocytes were generated using Gateway cloning (Invitrogen) to produce vectors with tol2 transposon sites (Kwan et al., 2007). Tg(hsp70l:hsa.UHRF1-GFP) and Tg(fabp10:hsa.UHRF1-GFP) were described in (Chu et al., 2012). The high, medium and low expressing alleles are listed in ZFIN.org with superscripts mss1a, mss1b and mss1c, respectively. Transgenics were outcrossed to Tab14 (WT) or tp53 −/− fish (Berghmans et al., 2005).\nTg(hsp70l:UHRF1-GFP) embryos were heat-shocked at 37°C for 1 hour at 24 and 27 hours post fertilization (hpf). At 28 hpf, embryos were sorted visually for GFP expression and incubated with either 10 μ M MG132 or DMSO and collected for immunoblotting at 34 hpf. 5-Aza (50 μ M) was added to larvae from 2.5–5 dpf. The Mount Sinai Institutional Animal Care and Use Committee approved all protocols. Nomenclature guidelines for the species under discussion were followed, and when no species is specified, human nomenclature was used.\n\n\nNIH-PA Author Manuscript\n\n\nGene expression analysis\nRNA was isolated from a pool of at least 10 livers from 5 dpf fish and from 1–5 livers from 20 dpf fish using the RNeasy mini-kit (Qiagen). cDNA was prepared by polyA priming using qScript SuperMix (Quanta). qRT-PCR analysis was performed in the Light Cycler 480 (Roche) using gene-specific primers (see supplemental materials) and PerfeCTa SYBRGreen FastMix (Quanta). C t values from triplicate reactions were averaged and 2 -Ct(target) /2 -Ct(reference) was used to calculate expression, with rpp0 and cyclophilin A used as references genes zebrafish and mouse samples, respectively.\nRNAseq analysis was carried out on RNA from pools of 50 livers dissected from 2 clutches of 5 dpf UHRF1-GFP High and nls-mCherry larvae, described in Supplemental Material.\n\n\nHistology\nFish younger than 20 dpf were fixed overnight at 4°C in 4% paraformaldehyde (PFA) and older fish were fixed for 2–5 days at room temperature in Bouins fixative. 4 μ M serial sections of paraffin embedded fish were stained with hematoxylin and eosin as described (Imrie and Sadler, 2010) and imaged on an Olympus BX41 Clinical Microscope equipped with a Nikon DS-Ri1 digital camera. Histological criteria used for scoring tumors are described in Figure S4A–B.\nImmunoblotting\nLysates prepared from 15 embryos dissolved in 150 μ l protein lysis buffer were homogenized by sonication. 1 embryo equivalent was loaded per lane of an 8 or 12% SDS gel, transferred to nitrocellulose and blotted. HepG2 cell lysates were prepared as described (Tien et al., 2011).\n\n\nAntibodies\nAntibodies recognizing DNMT1 (1:1000 for blotting, 1:10 for immunofluorescence, Santa Cruz), UHRF1 (immunoblotting: 1:1000; BD Biosciences. Immunohistochemistry: 1:50; ab57083; Abcam), Tubulin (1:5000 DSHB), p89 PARP and total PARP (1:1000; Cell Signaling) and b-actin (1:2000; Sigma), 5MeC (1:500; Eurogentec), BrdU (1:200; BD Bioscences) and anti-rabbit or mouse conjugated to Alexa 555 or Alexa 488 (1:100; Invitrogen) were diluted in 10% FBS or 2% BSA in 1% Triton in PBS (PBST).\nZebrafish staining\nLarvae fixed in 4% PFA were washed in PBST and stained using the Senescence β - Galactosidase Staining Kit (Cell Signaling) or stained with CY3-streptavadin (CY3-SA; 1:300; Sigma) as described (Sadler et al., 2005). The left liver lobe area was measured using ImageJ.\nImmunofluorescence was carried out on whole fish or on livers dissected from fixed larvae. BrdU was added to larvae water (10 mM) for 4–6 hours followed by immediate fixation, dehydration in methanol, rehydration to PBST and permeablization with 10ug/ml Proteinase K. DNA was denatured in 1 or 2N HCl, renatured and blocked in 10% FBS or 1% BSA in PBST prior to immunofluorescence. DNA was stained with Hoechst 33342 (Sigma). A Leica SP5 DM confocal microscope was used for imaging.\n\n\nSlot blots\nGenomic DNA was denatured in 0.4M NaOH at 95°C for 10 minutes, neutralized in an equal volume of cold 2M ammonium acetate and 100 ng DNA was blotted in duplicate onto nitrocellulose membrane using a slot blot apparatus and washed in 2x SSC and vacuum\n\n\nNIH-PA Author Manuscript\nbaked at 80°C for 2 hours. Half was stained with 0.2% methylene blue in 0.3 M NaOAc and the other with anti-5MeC followed by HRP-conjugated anti-mouse (1:2000) and visualized by Chemilumiescence (Roche). Image J was used to quantify 5MeC and methylene blue intensity and 5MeC levels were determined by normalizing to total DNA.\n\n\nCell culture\nPrimary mouse hepatocytes were isolated and plated in triplicate at 50% confluency then treated with 5 μ m 5-Aza or DMSO for 24 hours and RNA was isolated.\nNIH-3T3 cells were transfected in triplicate with pCDNA3.1 lacking an insert (control) or containing human RAS, UHRF1 or both and were re-transfected 24 hours later. 48 hours after the second transfection, 700 mg /ml Neomycin was added for 7 days. 10,000 cells per condition were plated on 0.3% soft agar layered on top of 0.6% soft agar. Media was changed every 3 rd day for 2 weeks and plates were stained with 0.005% crystal violet and colony number was counted.\nHepG2 cells cultured in DMEM supplemented with 10% (v/v) FBS and 5% (w/v) penicillin- streptomycin were transfected twice, 24 hours apart, with 20 nM siRNA targeting firefly luciferase (GL2; Dhamarcon) or UHRF1 targeting si-A and si-B using RNAiMAX (Invitrogen) as described (Tien et al., 2011).\n\n\nHuman tissue samples\nPathologically staged human tumors, dysplastic foci and normal liver samples were obtained from the HCC Genomic Consortium (Mount Sinai Hospital, NY; Instituto Nazionale dei Tumori, Milan and Hospital Clinic, Barcelona). The study was approved by the IRB of each institution and informed consent was obtained from all participants. TaqMan Probes were used to analyze UHRF1 expression by qPCR as described (Villanueva et al., 2008). Samples were grouped into a training set (9 normal liver, 18 low- and high-grade dysplastic nodules, and 40 pathologically staged HCCs; (Llovet et al., 2006; Wurmbach et al., 2007)) and a validation set (7 normal liver and 69 HCC; (Villanueva et al., 2008)). Integrative genomic and clinical data analysis was performed on 71 of the hepatitis C-related, surgically treated HCC patients (NCBI Gene Expression Omnibus, accession numbers GSE9829 and GSE44970). Liver pathologist, ST, scored UHRF1 staining of 52 paraffin embedded tumors from the training set.\n\n\nBioinformatics and statistical analysis\n71 patients with HCC were grouped based on the median expression level of UHRF1 in the tumors determined by qPCR (UHRF1-high; n=35 or low; n=36). Presence of aggressive human HCC gene signatures (Table S1) were evaluated in the transcriptome dataset using nearest template prediction method (Hoshida, 2010) implemented by the GenePattern genomic analysis toolkit ( www.broadinstitute.org/genepattern ) based on prediction confidence p&lt;0.05. The senescence-related TP53 target genes were obtained from Molecular Signature Database (MSigDB, www.broadinstitute.org/msigdb , TANG_SENESCENCE_TP53_TARGETS_UP and _DN; (Tang et al., 2007)) and from NCBI Gene Expression Omnibus (GSE39469; (Lujambio et al., 2013)). Mouse genes from normalized microarray data were converted to human orthologs based on a mapping table ( www.informatics.jax.org ). Differentially expressed genes between senescent and proliferative cells were identified by using Bayesian T-test implemented in Cyber-T software (molgen51.biol.rug.nl/cybert) at the significance threshold of Posterior Probability of Differential Expression &gt;0.998 after excluding less variable genes with coefficient of variation ≤ 0.1 across the samples. Expression pattern of each signature was assessed by nearest template prediction algorithm. Significance of induction or suppression of each\n\n\nNIH-PA Author Manuscript\nsignature was quantitatively measured by nominal p-value of cosine distance. Pearson correlation test determined the significance between induction/suppression of gene signatures with UHRF1 mRNA expression level as measured by qPCR. Statistical difference between groups was assessed by Wilcoxon rank-sum test and Fisher’s exact test for continuous and categorical variables, respectively. Bonferroni correction for multiple hypothesis testing was applied when appropriate.\nSurvival analysis was performed by using Kaplan-Meier estimator and log-rank test. All analyses were performed using R statistical package ( www.r-project.org ) and GenePattern genomic analysis tool kit ( www.broadinstitute.org/genepattern ). Gene expression levels and liver size were compared using Student’s T or Mann Whitney U tests\n\n\nSupplementary Material\nRefer to Web version on PubMed Central for supplementary material.\n\n\nAcknowledgments\nSupport was provided by The Breast Cancer Alliance and the March of Dimes (to KCS), a Pilot Project from the DFCI NCI Cancer Center (5P30CA006516-45; to CU), the NIH (5R01DK080789-02 to CU and KCS,\n1R01DK099558 to YH, 1R01DK076986 to JML, F30DK094503 to VJ and T32CA078207-14 for YC), the European Commission 7th Framework Programme FP7-Health 2010 (Heptromic #259744 to YH and JML), The Samuel Waxman Cancer Research Foundation, The Spanish National Health Institute (SAF-2010-16055) and the Asociación Española Contra el Cáncer (JML). Liz Loughlin, Brandon Kent, Meghan Walsh, Alex Mir, Laia Cabellos, Helena Cornellà Vives and Sara Toffanin provided expert technical assistance. We are grateful to Sam Sidi for tp53−/− fish, stimulating discussions and critical reading of the manuscript.\n\n\nReferences\nAnderson RM, Bosch JA, Goll MG, Hesselson D, Dong PD, Shin D, Chi NC, Shin CH, Schlegel A, Halpern M, Stainier DY. Loss of Dnmt1 catalytic activity reveals multiple roles for DNA methylation during pancreas development and regeneration. Dev Biol. 2009\nArita K, Ariyoshi M, Tochio H, Nakamura Y, Shirakawa M. Recognition of hemi-methylated DNA by the SRA protein UHRF1 by a base-flipping mechanism. Nature. 2008; 455:818–821. [PubMed: 18772891]\nAvvakumov GV, Walker JR, Xue S, Li Y, Duan S, Bronner C, Arrowsmith CH, Dhe-Paganon S. Structural basis for recognition of hemi-methylated DNA by the SRA domain of human UHRF1. Nature. 2008; 455:822–825. [PubMed: 18772889]\nBabbio F, Pistore C, Curti L, Castiglioni I, Kunderfranco P, Brino L, Oudet P, Seiler R, Thalman GN, Roggero E, et al. The SRA protein UHRF1 promotes epigenetic crosstalks and is involved in prostate cancer progression. Oncogene. 2012\nBerdasco M, Esteller M. Aberrant epigenetic landscape in cancer: how cellular identity goes awry. Dev Cell. 2010; 19:698–711. [PubMed: 21074720]\nBerghmans S, Murphey RD, Wienholds E, Neuberg D, Kutok JL, Fletcher CD, Morris JP, Liu TX, Schulte-Merker S, Kanki JP, et al. tp53 mutant zebrafish develop malignant peripheral nerve sheath tumors. Proc Natl Acad Sci U S A. 2005; 102:407–412. [PubMed: 15630097]\nBiniszkiewicz D, Gribnau J, Ramsahoye B, Gaudet F, Eggan K, Humpherys D, Mastrangelo MA, Jun Z, Walter J, Jaenisch R. Dnmt1 overexpression causes genomic hypermethylation, loss of imprinting, and embryonic lethality. Mol Cell Biol. 2002; 22:2124–2135. [PubMed: 11884600]\nBostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE. UHRF1 plays a role in maintaining DNA methylation in mammalian cells. Science. 2007; 317:1760–1764. [PubMed: 17673620]\nCalvisi DF, Ladu S, Gorden A, Farina M, Lee JS, Conner EA, Schroeder I, Factor VM, Thorgeirsson SS. Mechanistic and prognostic significance of aberrant methylation in the molecular pathogenesis of human hepatocellular carcinoma. J Clin Invest. 2007; 117:2713–2722. [PubMed: 17717605]\n\n\nNIH-PA Author Manuscript\nCheah MS, Wallace CD, Hoffman RM. Hypomethylation of DNA in human cancer cells: a site- specific change in the c-myc oncogene. J Natl Cancer Inst. 1984; 73:1057–1065. [PubMed: 6092764]\nChen T, Hevi S, Gay F, Tsujimoto N, He T, Zhang B, Ueda Y, Li E. Complete inactivation of DNMT1 leads to mitotic catastrophe in human cancer cells. Nat Genet. 2007; 39:391–396. [PubMed: 17322882]\nChiang DY, Villanueva A, Hoshida Y, Peix J, Newell P, Minguez B, LeBlanc AC, Donovan DJ, Thung SN, Sole M, et al. Focal gains of VEGFA and molecular classification of hepatocellular carcinoma. Cancer Res. 2008; 68:6779–6788. [PubMed: 18701503]\nChu J, Loughlin EA, Gaur NA, Senbanerjee S, Jacob V, Monson C, Kent B, Oranu A, Ding Y, Ukomadu C, Sadler KC. UHRF1 phosphorylation by cyclin A2/cyclin-dependent kinase 2 is required for zebrafish embryogenesis. Mol Biol Cell. 2012; 23:59–70. [PubMed: 22072796]\nChu J, Mir A, Gao N, Rosa S, Monson C, Sharma V, Steet R, Freeze HH, Lehrman MA, Sadler KC. A zebrafish model of congenital disorders of glycosylation with phosphomannose isomerase deficiency reveals an early opportunity for corrective mannose supplementation. Dis Model Mech. 2013; 6:95–105. [PubMed: 22899857]\nDecottignies A, d’Adda di Fagagna F. Epigenetic alterations associated with cellular senescence: a barrier against tumorigenesis or a red carpet for cancer? Semin Cancer Biol. 2011; 21:360–366. [PubMed: 21946622]\nDi Micco R, Sulli G, Dobreva M, Liontos M, Botrugno OA, Gargiulo G, dal Zuffo R, Matti V, d’Ario G, Montani E, et al. Interplay between oncogene-induced DNA damage response and heterochromatin in senescence and cancer. Nat Cell Biol. 2011; 13:292–302. [PubMed: 21336312]\nDu Z, Song J, Wang Y, Zhao Y, Guda K, Yang S, Kao HY, Xu Y, Willis J, Markowitz SD, et al. DNMT1 stability is regulated by proteins coordinating deubiquitination and acetylation-driven ubiquitination. Sci Signal. 2010; 3:ra80. [PubMed: 21045206]\nEden A, Gaudet F, Waghmare A, Jaenisch R. Chromosomal instability and tumors promoted by DNA hypomethylation. Science. 2003; 300:455. [PubMed: 12702868]\nFairweather DS, Fox M, Margison GP. The in vitro lifespan of MRC-5 cells is shortened by 5- azacytidine-induced demethylation. Exp Cell Res. 1987; 168:153–159. [PubMed: 2430819]\nFeng S, Cokus SJ, Zhang X, Chen PY, Bostick M, Goll MG, Hetzel J, Jain J, Strauss SH, Halpern ME, et al. Conservation and divergence of methylation patterning in plants and animals. Proc Natl Acad Sci U S A. 2010; 107:8689–8694. [PubMed: 20395551]\nGaudet F, Hodgson JG, Eden A, Jackson-Grusby L, Dausman J, Gray JW, Leonhardt H, Jaenisch R. Induction of tumors in mice by genomic hypomethylation. Science. 2003; 300:489–492. [PubMed: 12702876]\nGaudet F, Rideout WM 3rd, Meissner A, Dausman J, Leonhardt H, Jaenisch R. Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. Mol Cell Biol. 2004; 24:1640–1648. [PubMed: 14749379]\nHashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X. The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. Nature. 2008; 455:826–829. [PubMed: 18772888]\nHoshida Y. Nearest template prediction: a single-sample-based flexible class prediction with confidence assessment. PLoS One. 2010; 5:e15543. [PubMed: 21124904]\nHoward G, Eiges R, Gaudet F, Jaenisch R, Eden A. Activation and transposition of endogenous retroviral elements in hypomethylation induced tumors in mice. Oncogene. 2008; 27:404–408. [PubMed: 17621273]\nImrie D, Sadler KC. White adipose tissue development in zebrafish is regulated by both developmental time and fish size. Dev Dyn. 2010; 239:3013–3023. [PubMed: 20925116]\nJackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G, Dausman J, Lee P, Wilson C, Lander E, Jaenisch R. Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. Nat Genet. 2001; 27:31–39. [PubMed: 11137995]\nJin W, Chen L, Chen Y, Xu SG, Di GH, Yin WJ, Wu J, Shao ZM. UHRF1 is associated with epigenetic silencing of BRCA1 in sporadic breast cancer. Breast Cancer Res Treat. 2009\nJirtle RL. IGF2 loss of imprinting: a potential heritable risk factor for colorectal cancer. Gastroenterology. 2004; 126:1190–1193. [PubMed: 15057757]\n\n\nNIH-PA Author Manuscript\nKarpf AR, Matsui S. Genetic disruption of cytosine DNA methyltransferase enzymes induces chromosomal instability in human cancer cells. Cancer Res. 2005; 65:8635–8639. [PubMed: 16204030]\nKarpf AR, Moore BC, Ririe TO, Jones DA. Activation of the p53 DNA damage response pathway after inhibition of DNA methyltransferase by 5-aza-2 ′ -deoxycytidine. Mol Pharmacol. 2001; 59:751–757. [PubMed: 11259619]\nKwan KM, Fujimoto E, Grabher C, Mangum BD, Hardy ME, Campbell DS, Parant JM, Yost HJ, Kanki JP, Chien CB. The Tol2kit: a multisite gateway-based construction kit for Tol2 transposon transgenesis constructs. Dev Dyn. 2007; 236:3088–3099. [PubMed: 17937395]\nLi E, Beard C, Jaenisch R. Role for DNA methylation in genomic imprinting. Nature. 1993; 366:362– 365. [PubMed: 8247133]\nLiu X, Gao Q, Li P, Zhao Q, Zhang J, Li J, Koseki H, Wong J. UHRF1 targets DNMT1 for DNA methylation through cooperative binding of hemi-methylated DNA and methylated H3K9. Nat Commun. 2013; 4:1563. [PubMed: 23463006]\nLlovet JM, Burroughs A, Bruix J. Hepatocellular carcinoma. Lancet. 2003; 362:1907–1917. [PubMed: 14667750]\nLlovet JM, Chen Y, Wurmbach E, Roayaie S, Fiel MI, Schwartz M, Thung SN, Khitrov G, Zhang W, Villanueva A, et al. A molecular signature to discriminate dysplastic nodules from early hepatocellular carcinoma in HCV cirrhosis. Gastroenterology. 2006; 131:1758–1767. [PubMed: 17087938]\nLlovet JM, Ricci S, Mazzaferro V, Hilgard P, Gane E, Blanc JF, de Oliveira AC, Santoro A, Raoul JL, Forner A, et al. Sorafenib in advanced hepatocellular carcinoma. N Engl J Med. 2008; 359:378– 390. [PubMed: 18650514]\nLujambio A, Akkari L, Simon J, Grace D, Tschaharganeh DF, Bolden JE, Zhao Z, Thapar V, Joyce JA, Krizhanovsky V, Lowe SW. Non-cell-autonomous tumor suppression by p53. Cell. 2013; 153:449–460. [PubMed: 23562644]\nMcDuff FK, Turner SD. Jailbreak: oncogene-induced senescence and its evasion. Cell Signal. 2011; 23:6–13. [PubMed: 20633638]\nMudbhary R, Sadler KC. Epigenetics, development, and cancer: Zebrafish make their mark. Birth Defects Res C Embryo Today. 2011; 93:194–203. [PubMed: 21671358]\nNishiyama A, Yamaguchi L, Sharif J, Johmura Y, Kawamura T, Nakanishi K, Shimamura S, Arita K, Kodama T, Ishikawa F, et al. Uhrf1-dependent H3K23 ubiquitylation couples maintenance DNA methylation and replication. Nature. 2013; 502:249–253. [PubMed: 24013172]\nPogribny IP, Rusyn I. Role of epigenetic aberrations in the development and progression of human hepatocellular carcinoma. Cancer Lett. 2012\nQin W, Leonhardt H, Spada F. Usp7 and Uhrf1 control ubiquitination and stability of the maintenance DNA methyltransferase Dnmt1. J Cell Biochem. 2011; 112:439–444. [PubMed: 21268065]\nRai K, Nadauld LD, Chidester S, Manos EJ, James SR, Karpf AR, Cairns BR, Jones DA. Zebra fish Dnmt1 and Suv39h1 regulate organ-specific terminal differentiation during development. Mol Cell Biol. 2006; 26:7077–7085. [PubMed: 16980612]\nRai K, Sarkar S, Broadbent TJ, Voas M, Grossmann KF, Nadauld LD, Dehghanizadeh S, Hagos FT, Li Y, Toth RK, et al. DNA demethylase activity maintains intestinal cells in an undifferentiated state following loss of APC. Cell. 2010; 142:930–942. [PubMed: 20850014]\nSadler KC, Amsterdam A, Soroka C, Boyer J, Hopkins N. A genetic screen in zebrafish identifies the mutants vps18, nf2 and foie gras as models of liver disease. Development. 2005; 132:3561–3572. [PubMed: 16000385]\nSadler KC, Krahn KN, Gaur NA, Ukomadu C. Liver growth in the embryo and during liver regeneration in zebrafish requires the cell cycle regulator, uhrf1. Proc Natl Acad Sci U S A. 2007; 104:1570–1575. [PubMed: 17242348]\nSharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA, Shinga J, Mizutani-Koseki Y, Toyoda T, Okamura K, et al. The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. Nature. 2007; 450:908–912. [PubMed: 17994007]\nTang X, Milyavsky M, Goldfinger N, Rotter V. Amyloid-beta precursor-like protein APLP1 is a novel p53 transcriptional target gene that augments neuroblastoma cell death upon genotoxic stress. Oncogene. 2007; 26:7302–7312. [PubMed: 17533371]\nTaylor EM, Bonsu NM, Price RJ, Lindsay HD. Depletion of Uhrf1 inhibits chromosomal DNA replication in Xenopus egg extracts. Nucleic Acids Res. 2013\nTien AL, Senbanerjee S, Kulkarni A, Mudbhary R, Goudreau B, Ganesan S, Sadler KC, Ukomadu C. UHRF1 depletion causes a G2/M arrest, activation of DNA damage response and apoptosis. Biochem J. 2011; 435:175–185. [PubMed: 21214517]\nTischoff I, Tannapfe A. DNA methylation in hepatocellular carcinoma. World J Gastroenterol. 2008; 14:1741–1748. [PubMed: 18350605]\nTittle RK, Sze R, Ng A, Nuckels RJ, Swartz ME, Anderson RM, Bosch J, Stainier DY, Eberhart JK, Gross JM. Uhrf1 and Dnmt1 are required for development and maintenance of the zebrafish lens. Dev Biol. 2011; 350:50–63. [PubMed: 21126517]\nUnoki M, Daigo Y, Koinuma J, Tsuchiya E, Hamamoto R, Nakamura Y. UHRF1 is a novel diagnostic marker of lung cancer. Br J Cancer. 2010; 103:217–222. [PubMed: 20517312]\nVentura A, Kirsch DG, McLaughlin ME, Tuveson DA, Grimm J, Lintault L, Newman J, Reczek EE, Weissleder R, Jacks T. Restoration of p53 function leads to tumour regression in vivo. Nature. 2007; 445:661–665. [PubMed: 17251932]\nVillanueva A, Chiang DY, Newell P, Peix J, Thung S, Alsinet C, Tovar V, Roayaie S, Minguez B, Sole M, et al. Pivotal role of mTOR signaling in hepatocellular carcinoma. Gastroenterology. 2008; 135:1972–1983. 1983 e1971–1911. [PubMed: 18929564]\nVillanueva A, Hoshida Y, Battiston C, Tovar V, Sia D, Alsinet C, Cornella H, Liberzon A, Kobayashi M, Kumada H, et al. Combining clinical, pathology, and gene expression data to predict recurrence of hepatocellular carcinoma. Gastroenterology. 2011; 140:1501–1512.e1502. [PubMed: 21320499]\nWang F, Yang YZ, Shi CZ, Zhang P, Moyer MP, Zhang HZ, Zou Y, Qin HL. UHRF1 Promotes Cell Growth and Metastasis Through Repression of p16(ink4a) in Colorectal Cancer. Ann Surg Oncol. 2012\nWeber M, Schubeler D. Genomic patterns of DNA methylation: targets and function of an epigenetic mark. Curr Opin Cell Biol. 2007; 19:273–280. [PubMed: 17466503]\nWurmbach E, Chen YB, Khitrov G, Zhang W, Roayaie S, Schwartz M, Fiel I, Thung S, Mazzaferro V, Bruix J, et al. Genome-wide molecular profiles of HCV-induced dysplasia and hepatocellular carcinoma. Hepatology. 2007; 45:938–947. [PubMed: 17393520]\nXue W, Zender L, Miething C, Dickins RA, Hernando E, Krizhanovsky V, Cordon-Cardo C, Lowe SW. Senescence and tumour clearance is triggered by p53 restoration in murine liver carcinomas. Nature. 2007; 445:656–660. [PubMed: 17251933]\nYamada Y, Jackson-Grusby L, Linhart H, Meissner A, Eden A, Lin H, Jaenisch R. Opposing effects of DNA hypomethylation on intestinal and liver carcinogenesis. Proc Natl Acad Sci U S A. 2005; 102:13580–13585. [PubMed: 16174748]\nYou JS, Jones PA. Cancer genetics and epigenetics: two sides of the same coin? Cancer Cell. 2012; 22:9–20. [PubMed: 22789535]\n\n\nHighlights\n• UHRF1 overexpression induces DNA hypomethylation\n• Liver specific overexpression of UHRF1 in zebrafish causes senescence and HCC\n• Human HCCs with high UHRF1 are aggressive and inactivate TP53-mediated senescence\nNIH-PA Author Manuscript\n\n\nSIGNIFICANCE\nGlobal DNA hypomethylation occurs in most types of cancer and can induce genomic instability and widespread changes in gene expression. UHRF1 is a key regulator of DNA methylation, and here we show that UHRF1 overexpression causes HCC in zebrafish without any other genetic alteration, demonstrating that it is an oncogene. High UHRF1 expression causes DNA hypomethylation and tp53-mediated senescence, which serves to restrict transformation of UHRF1 overexpressing cells. High UHRF1 expression in human HCC correlates with a poor prognosis, genomic instability, TP53 mutation and repression of the TP53-mediated senescence program. We conclude that UHRF1 is an oncogene that promotes widespread DNA hypomethylation, an epigenetic hallmark of cancer cells, and that UHRF1 overexpression drives tumorigenesis when senescence is bypassed.\nNIH-PA Author Manuscript\nFigure 1. High UHRF1 expression causes global DNA hypomethylation (A) 5MeC levels and total DNA stained with methylene blue were measured in 5 dpf control and UHRF1-GFP High livers (n=4) and liver-less carcasses (n=3). The ratio of 5MeC to total DNA was averaged and normalized to controls. Student’s T-test was used to determine p values. (B) Confocal stacks of livers (top) and fins (bottom) from 5 dpf nls-mCherry and UHRF1-GFP High larvae stained with anti-5MeC. Since a hepatocyte specific promoter was used for transgenesis, there was no transgene expression in the fin. (C) Dnmt1 is uniform in the hepatocyte nucleus of 4 dpf nls-mCherry larvae but is found in GFP-containing punctae in UHRF1-GFP High hepatocytes. Arrows point to cells that do not express GFP and have Dnmt1 distribution pattern similar to controls. (D) Tg(hsp70I:UHRF1-EGFP) and non- transgenic controls were heat shocked at 37° C for 1 hour at 24 and 27 hpf, treated with 10 μ M MG132 or DMSO at 28 hpf and collected at 34 hpf for immunoblotting. (E) Dnmt1 levels normalized to tubulin were averaged from 6 experiments. Student’s T-test was used to determine p values; n.s.: not significant; error bars represent SD. See also Figure S1.\n\n\nNIH-PA Author Manuscript\nFigure 2. UHRF1-induced hypomethylation reduces liver size (A) Individual larvae were imaged daily from 3–10 dpf. (B) 5 dpf UHRF1-GFP High larvae display a range of liver sizes scored as “normal” or “small”‘ (C) 3 clutches were scored according to criteria in B; n= number of larvae. Fisher’s exact test was used to determine p value. (D) The area of the left liver lobe was measured in 5 dpf fish from 2 clutches. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. Student’s T-test was used to determine p value. (E) UHRF1-GFP High larvae were sorted by liver size on 5 dpf and tracked daily for survival to 20 dpf. Data are pooled from three clutches. (F) UHRF1-GFP High and control larvae were treated with 50 μ M 5-Aza from 2.5–5 dpf and scored for liver size in 6 clutches. Fisher’s exact test was used to determine p values. (G) UHRF1-GFP High embryos were injected with mRNA encoding dnmt1 or Mpi before 1 hpf. The percent of fish with a normal liver size was scored at 5 dpf in 6 clutches. See also Figure S2.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\nFigure 3. UHRF1 overexpression in hepatocytes induces Tp53-mediated senescence (A) Intense senescence associated β -galactosidase (SA- β -gal) staining was detected in the liver (outlined) of 5 dpf UHRF1-GFP High larvae compared to light or no staining in controls. (B) 5 dpf fish from 5 clutches were scored for hepatic SA- β -gal staining. *** indicates p&lt;0.0001 by Fisher’s exact test. (C) Nuclear size was measured in hepatocytes of a single control or UHRF1-GFP High 5 dpf liver and cells were stratified according to GFP expression. Inset shows confocal stack of the DNA organized into foci. ** and *** indicates p&lt;0.001 or 0.0001, respectively compared to nuclear size in nls-mCherry larvae. (D) BrdU positive cells and the total number of transgene-expressing hepatocytes in nls-mCherry and UHRF1-GFP High larvae (Bottom) 5 dpf larvae. A Fisher’s exact test was used to calculate\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\np value. In nls-mCherry larvae, most BrdU positive cells also express the transgene whereas the BrdU positive cells in UHRF1-GFP High livers did not express GFP (white arrows in magnified regions which are marked by the white box). (E) Heatmap of log2 values from RNAseq shows cell cycle regulators are down and Tp53 target genes (marked by *) are up in UHRF1-GFP High 5 dpf livers. (F) tp53 and cdkn1a mRNA expression were induced on 5 dpf and down regulated on 20 dpf in UHRF1-GFP High livers. * indicates p=0.05; *** indicates p=0.001 calculated by 1 sample Student’s T-test. Error bars represent SD. (G) 5- Aza induces Tp53 expression in primary mouse hepatocytes. Student’s T-test was used to determine p value with SD indicated by the error bars across 3 replicates. tp53 +/− in UHRF1-GFP High larvae significantly reduced SA- β -gal staining in the liver (2 clutches) (H), and increased the percent of larvae with normal liver size (I), the area of the left liver lobe (J), and survival at 5 dpf (K). p values were calculated with a Fisher’s test with Freeman-Halton extension (H), Fisher’s exact test (I), and Student’s T-test. Boxes represent 75 th and 25 th percentile, horizontal line is the median and whiskers mark lowest and highest values. (J). See also Figure S3.\nNIH-PA Author Manuscript\nFigure 4. UHRF1 is an oncogene\n(A) Atypical cells, dysplastic foci (outlined) and HCC are apparent in H&amp;E stained UHRF1- GFP High livers. BD: bile duct, MF: mitotic figure. (B) Incidence of normal and atypical hepatocytes, dysplastic foci and cancer in the liver of UHRF1-GFP High fish on WT or tp53 +/− background. (C) NIH-3T3 cell growth in soft agar is enhanced when UHRF1 overexpression is combined with RAS (n=3). p value was calculated by Student’s T-test and error bars represent the SD. (D) Hepatic SA- β -gal staining patterns in UHRF1-GFP High larvae changes as fish age. Images of 8 dpf larvae illustrate the SA- β -gal staining patterns that were scored in the time course shown in the graph. A significant increase in the number of UHRF1-GFP High fish with intense or punctate SA- β -gal compared to controls at all time points (p&lt;0.01 by Fisher’s exact test) except at 4 dpf; not significant=n.s. (E) BrdU incorporation in the liver on 11 dpf is 5 times higher in UHRF1-GFP High fish than in\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\ncontrols. Total # of cells counted is indicated with n= # of clutches assessed. Fisher’s exact test was used to calculate p value. See also Figure S4.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 5. UHRF1 mRNA and protein are overexpressed in HCC (A) UHRF1 detected by qRT-PCR in 18 pre-neoplastic lesions and 40 HCCs from HCV infected patients compared to expression in 9 normal livers. Horizontal line indicates median. (B) Immunohistochemistry for UHRF1 protein (brown) was evaluated in 52 of the same HCCs examined in A plus 5 normal liver samples. Fisher’s exact test was used to calculate p value. 71 of the HCV-associated HCCs analyzed by qPCR were grouped into high (n=35) and low (n=36) UHRF1 expressing tumors based on the median log2-fold change of 3.64. (C) HepG2 cells transfected with control siRNA (GL2) or two different siRNAs targeting UHRF1 described in (Tien et al., 2011) were blotted for UHRF1, cleaved and total PARP (arrow indicates full length, * indicates cleaved protein). (D) Vascular invasion (33 high and 34 low tumors, 4 missing values), (E) serum AFP (29 UHRF1-high and 29 low tumors, 13 missing values), (F) early (&lt;2 years) and (G) late (&gt;2 years; 32 UHRF1-high and 35 low tumors, 4 missing values) tumor recurrence and (H) overall survival after surgery (32 high and 35 low tumors, 4 missing values) were stratified according to UHRF1-expression. Continuous and categorical variables were assessed by Wilcoxon rank-sum test and Fisher’s exact test, respectively. Clinical outcome difference was evaluated by log-rank test. In box and whisker plots, boxes represent the 75th and 25th\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\npercentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S5.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nFigure 6. High UHRF1 expression defines a subclass of tumors with inactivated tp53, repression of senescence and chromosomal instability\n(A) The 71 human HCC tumors analyzed in Figure 5C–G were rank-ordered according to UHRF1 expression by qPCR and classified as high (&lt;median, red; n=35) or low (≥median, blue; n=36). The presences of aggressive human HCC gene signatures from published studies and of TP53 inactivating mutations are indicated by red and black boxes, respectively. TP53-mediated senescence gene signatures (Lujambio et al., 2013; Tang et al., 2007) are displayed as a range from repressed (blue) to activated (red). (B) Genome-wide profile of DNA copy number variation was obtained from GEO geneset GSE9829. (C) Proportion of genes with DNA copy number loss and gain in tumors according to UHRF1 expression. (D) Proportion of genes with DNA copy number loss according to UHRF1 expression and TP53 mutation status. (E) UHRF1 expression is significantly higher in tumors with TP53 mutations. (F) DNMT1 expression by microarray analysis is significantly correlated with UHRF1 expression assessed by qPCR in HCCs. In box and whisker plots,\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\nboxes represent the 75th and 25th percentiles, the whiskers represent the most extreme data points within interquartile range x 1.5, and the horizontal bar represents the median. See also Figure S6 and Tables S1-S6.\nNIH-PA Author Manuscript\nFigure 7. Model of the relationship between UHRF1 overexpression, DNA hypomethylation, Tp53 mediated senescence, cancer and survival\nFactors investigated in this study are in solid black boxes with black lines indicating the correlations demonstrated in this work and gray lines indicating relationships that are speculative. Senescence reduces liver size and function and reduces larval survival while cancer occurs when senescence is bypassed and also reduces survival.\n\n\nNIH-PA Author Manuscript\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 2\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 3\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 4\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 5\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 6\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 7\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 8\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 9\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 10\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 11\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 12\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 13\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 14\n\n\nCancer Cell. Author manuscript; available in PMC 2015 February 10.\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\n• Our studies in zebrafish and human tumors identify UHRF1 as an oncogene in HCC Mudbhary et al. Page 15\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 16\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 17\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 18\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 19\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 20\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 21\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 22\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 23\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 24\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 25\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 26\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 27\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 28\n\n\nTable 1\n\n\nHCC onset and incidence in Tg(fabp10:nls-mCherry) and Tg(fabp10:UHRF1-GFP) zebrafish. transgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI nls-mCherry Normal 100% 100% 100% 100% 100% 100% 100% 100% 100% 100% 65 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 8 8 8 11 nd nd 5 5 5 5 5 5 65 UHRF1-GFP Low Normal 100% 100% 100% 100% 100% 100% 100% 100% 60 100%\n\n\nAtypical cells 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Dysplastic foci 0% 0% 0% 0% 0% 0% 0% 0% 0 0% Tumor 0% 0% 0% 0% 0% 0% 0% 0% 0 0% n = 7 9 10 12 nd nd 4 nd nd 6 6 6 60 UHRF1-GFP Medium Normal 90% (9) 100% (11) 50% (3) 28% (4) 66% (4) 80% (4) 50% (4) 50% (5) 81% (13) 57 66%\n\n\nAtypical cells 10% (1) 0% 50% (3) 57% (8) 33% (2) 20% (1) 50% (4) 20% (2) 44% (7) 28 32%\n\n\nDysplastic foci 0% 0% 0% 7% (1) 0% 0% 25% (2) 30% (3) 0% 6 7%\n\n\nTumor 0% 0% 0% 0% 0% 0% 0% 0% 6% (1) 1 1% n = 10 11 6 14 6 5 8 10 16 86 UHRF1-GFP High Normal 0% 0% 7% (1) 0% 0% 1 2% Atypical cells 100% (13) 100% (7) 46% (6) 17% (3) 75% (3) 36 61% Dysplastic foci 0% 0% 8% (1) 12% (2) 25% (1) 4 7% Tumor 0% 0% 46% (6) 76% (13) 0% 18 30% n = 13 7 13 17 4 54 UHRF1-GFP High/ p53 +/− Normal 0% 0 0% Atypical cells 13% (2) 2 13% Dysplastic foci 0% 0 0%\n\n\nNIH-PA Author Manuscript\n\n\nNIH-PA Author Manuscript\n\n\nMudbhary et al. Page 29\n\n\ntransgene dpf: 5 10 15 20 25 30 40 50 60 90 180 300 Total N = 281 CI\n\n\nTumor 88% (14) 14 88% n = 16 16\n\n\nThe incidence (percent) and absolute number of fish (parenthesis) with cancer-relevant histological phenotypes were scored in a total of 281 fish from 4 transgenic lines. The total n for each time point is indicated. nd: not done. Some fish were diagnosed with more than one lesion. CI: Cumulative incidence.\n'''\nThe text is taken from a scientific paper that uses RNAseq in Zebrafish to Studies of RNA expression in Zebrafish in a variety of experimental conditions.\nExtract metadata from the section text based on the following list of questions. Record answers as the specified fields of the output. Do not extract any other metadata.\n1. What organism was studied?\n Record this value in the 'organism_name' field of the output.\n2. What tissue was studied?\n Record this value in the 'tissue_name_text' field of the output.\n3. What tissue was studied?  Provide a list of answers drawn from the following tissues:  'aorta', 'blood', 'brain', 'caudal fin', 'cell line', 'endothelial', 'eye', 'fat',  'full embryo', 'germline', 'gill', 'gonad', 'gut', 'head', 'heart', 'hypothalamus',  'kidney marrow', 'kidney', 'liver', 'melanoma', 'muscle', 'neural crest', 'olfactory bulb',  'pancreas', 'retina', 'skin', 'spinal cord', 'spleen', 'tail', 'telencephalon', 'testis',  'thymocytes', 'thymus', 'trunk', 'tumor', or 'zebrafish neuromast hair cells'.  If the tissue is not listed, respond 'Other'.  If no tissue is mentioned, respond 'Unknown'.\n Record this value in the 'tissue_name_structured' field of the output.\n4. What developmental stage were the fish in?  If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_text' field of the output.\n5. What developmental stage were the fish in? Select all applicable categories from the following list:  'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_coarse' field of the output.\n6. What developmental stage were the fish in? Select all applicable categories from the following list: 'zygote', 'cleavage', 'blastula', 'gastrula', 'segmentation', 'pharyngula', 'hatching', 'larval', 'juvenile',  'adult'embryo', 'larval', 'juvenile', 'adult'.   If fish are not used as an experimental subject, respond 'Not Applicable'.  If fish are used, but the developmental stage is not mentioned, respond 'Unknown'.\n Record this value in the 'developmental_stage_fine' field of the output.\n7. Is single cell RNA / gene expression data used in this paper? To answer this question, pay attention to the papers' methods and results.  If methods of single cell isolation, RNA extraction, and sequencing are described it is likely that single cell RNA data was used. Similarly, if methods such as scRNAseq, STRT-seq, MARS-seq or any derivates are used, it is likely that single cell RNA data was used. Provide a True/False answer to this question. If you are unsure, respond 'Unknown'. Record this value in the 'is_scRNA_data' field of the output.\nGenerate only JSON formatted output with 7 fields:\norganism_name, tissue_name_text, tissue_name_structured, developmental_stage_text, developmental_stage_coarse, developmental_stage_fine, is_scRNA_data\nDo not provide additional explanation or context for the answer.\nIf you cannot answer a listed question, record a value of 'not present' in appropriate fields of the output. Do not include any other response other than a JSON object."
  ]
}
[llm/end] [1:chain:RunnableSequence &gt; 3:llm:ChatOpenAI] [8.80s] Exiting LLM run with output:
{
  "generations": [
    [
      {
        "text": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"liver\",\n  \"tissue_name_structured\": \"liver\",\n  \"developmental_stage_text\": \"embryos and larvae\",\n  \"developmental_stage_coarse\": [\"embryo\", \"larval\"],\n  \"developmental_stage_fine\": [\"embryo\", \"larval\"],\n  \"is_scRNA_data\": false\n}\n```",
        "generation_info": {
          "finish_reason": "stop",
          "logprobs": null
        },
        "type": "ChatGeneration",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain",
            "schema",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "```json\n{\n  \"organism_name\": \"zebrafish\",\n  \"tissue_name_text\": \"liver\",\n  \"tissue_name_structured\": \"liver\",\n  \"developmental_stage_text\": \"embryos and larvae\",\n  \"developmental_stage_coarse\": [\"embryo\", \"larval\"],\n  \"developmental_stage_fine\": [\"embryo\", \"larval\"],\n  \"is_scRNA_data\": false\n}\n```",
            "additional_kwargs": {}
          }
        }
      }
    ]
  ],
  "llm_output": {
    "token_usage": {
      "completion_tokens": 95,
      "prompt_tokens": 20111,
      "total_tokens": 20206
    },
    "model_name": "gpt-4-1106-preview",
    "system_fingerprint": "fp_123d5a9f90"
  },
  "run": null
}
[chain/start] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:
[inputs]
[chain/end] [1:chain:RunnableSequence &gt; 4:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "liver",
  "tissue_name_structured": "liver",
  "developmental_stage_text": "embryos and larvae",
  "developmental_stage_coarse": [
    "embryo",
    "larval"
  ],
  "developmental_stage_fine": [
    "embryo",
    "larval"
  ],
  "is_scRNA_data": false
}
[chain/end] [1:chain:RunnableSequence] [8.81s] Exiting Chain run with output:
{
  "organism_name": "zebrafish",
  "tissue_name_text": "liver",
  "tissue_name_structured": "liver",
  "developmental_stage_text": "embryos and larvae",
  "developmental_stage_coarse": [
    "embryo",
    "larval"
  ],
  "developmental_stage_fine": [
    "embryo",
    "larval"
  ],
  "is_scRNA_data": false
}</code></pre>
</div>
</div>
<div class="cell" data-execution_count="41">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>q <span class="op">=</span> cb.db.session.query(N) <span class="op">\</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>    .<span class="bu">filter</span>(N.<span class="bu">id</span> <span class="op">==</span> NIA.Note_id) <span class="op">\</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>    .<span class="bu">filter</span>(N.<span class="bu">type</span> <span class="op">==</span> <span class="st">'MetadataExtractionNote'</span>) <span class="op">\</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>    .<span class="bu">filter</span>(N.name.like(<span class="st">'rnaquarium_%'</span>)) </span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>l <span class="op">=</span> []</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> n <span class="kw">in</span> q.<span class="bu">all</span>():</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>    tup <span class="op">=</span> json.loads(n.content)</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>    t, doi, label <span class="op">=</span> n.name.split(<span class="st">'__'</span>)</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>    tup[<span class="st">'doi'</span>] <span class="op">=</span> <span class="st">'doi:'</span><span class="op">+</span>doi</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>    tup[<span class="st">'extraction_type'</span>] <span class="op">=</span> t</span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>    tup[<span class="st">'run_label'</span>] <span class="op">=</span> label</span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>    l.append(tup)</span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>report_df <span class="op">=</span> pd.DataFrame(l).set_index(<span class="st">'doi'</span>)</span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>report_df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="41">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">organism_name</th>
<th data-quarto-table-cell-role="th">tissue_name_text</th>
<th data-quarto-table-cell-role="th">tissue_name_structured</th>
<th data-quarto-table-cell-role="th">developmental_stage_text</th>
<th data-quarto-table-cell-role="th">developmental_stage_coarse</th>
<th data-quarto-table-cell-role="th">developmental_stage_fine</th>
<th data-quarto-table-cell-role="th">is_scRNA_data</th>
<th data-quarto-table-cell-role="th">extraction_type</th>
<th data-quarto-table-cell-role="th">run_label</th>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">doi</th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1038/cr.2014.108</td>
<td>zebrafish</td>
<td>heart</td>
<td>heart</td>
<td>adult</td>
<td>adult</td>
<td>adult</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">doi:10.1371/journal.pgen.1004335</td>
<td>zebrafish</td>
<td>liver</td>
<td>liver</td>
<td>larvae</td>
<td>[larval]</td>
<td>[larval]</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1016/j.gene.2014.05.036</td>
<td>zebrafish</td>
<td>brain, heart, testis, and ovary</td>
<td>[brain, heart, testis, ovary]</td>
<td>adult</td>
<td>adult</td>
<td>adult</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">doi:10.1073/pnas.1402898111</td>
<td>zebrafish</td>
<td>zebrafish lateral line</td>
<td>zebrafish neuromast hair cells</td>
<td>5 days post-fertilization (dpf)</td>
<td>larval</td>
<td>larval</td>
<td>True</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1371/journal.pone.0091179</td>
<td>zebrafish</td>
<td>liver</td>
<td>liver</td>
<td>3.5 months post fertilization</td>
<td>juvenile</td>
<td>juvenile</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">doi:10.1038/nature12974</td>
<td>zebrafish</td>
<td>not present</td>
<td>full embryo</td>
<td>12 developmental stages</td>
<td>[embryo]</td>
<td>[embryo]</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">doi:10.1073/pnas.1305536111</td>
<td>zebrafish</td>
<td>digestive organs, eye, lens, liver, pancreas, ...</td>
<td>[eye, liver, pancreas, gut, Other]</td>
<td>larvae</td>
<td>[larval]</td>
<td>[larval]</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">doi:10.1016/j.ccr.2014.01.003</td>
<td>zebrafish</td>
<td>liver</td>
<td>liver</td>
<td>embryos and larvae</td>
<td>[embryo, larval]</td>
<td>[embryo, larval]</td>
<td>False</td>
<td>rnaquarium</td>
<td>test</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="cell" data-execution_count="42">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>report_df.to_csv(loc<span class="op">+</span><span class="st">'/rnaquarium_metadata_extraction_report.tsv'</span>, sep<span class="op">=</span><span class="st">'</span><span class="ch">\t</span><span class="st">'</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="16">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a dataframe to store previously extracted metadata</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame()</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d_id <span class="kw">in</span> dois:</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a>    df2 <span class="op">=</span> pd.DataFrame(t2.read_metadata_extraction_notes(d_id, <span class="st">'rnaquarium'</span>)) </span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a>    df <span class="op">=</span> pd.concat([df, df2]) </span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-error">
<pre><code>KeyboardInterrupt: </code></pre>
</div>
</div>
<div class="cell" data-execution_count="37">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>ldb.session.rollback()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="38">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="co"># USE WITH </span><span class="al">CAUTION</span><span class="co"> - this will delete all extracted metadata notes in the database</span></span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a><span class="co"># clear all notes across papers listed in `dois` list</span></span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> d <span class="kw">in</span> <span class="bu">list</span>(<span class="bu">set</span>(dois[<span class="dv">0</span>:<span class="dv">10</span>])):</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>    d_id <span class="op">=</span> <span class="st">'doi:'</span><span class="op">+</span>d</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>    e <span class="op">=</span> ldb.session.query(SKE).<span class="bu">filter</span>(SKE.<span class="bu">id</span><span class="op">==</span>d_id).first()</span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>    notes_to_delete <span class="op">=</span> []</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> e <span class="kw">is</span> <span class="va">None</span>:</span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>        <span class="cf">continue</span></span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> n <span class="kw">in</span> ldb.read_notes_about_x(e):</span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a>        notes_to_delete.append(n.<span class="bu">id</span>)</span>
<span id="cb26-11"><a href="#cb26-11" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> n <span class="kw">in</span> notes_to_delete:</span>
<span id="cb26-12"><a href="#cb26-12" aria-hidden="true" tabindex="-1"></a>        ldb.delete_note(n)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="protocol-modeling-extraction" class="level2">
<h2 class="anchored" data-anchor-id="protocol-modeling-extraction">Protocol Modeling + Extraction</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>ldb <span class="op">=</span> Ceifns_LiteratureDb(loc<span class="op">=</span>loc, name<span class="op">=</span>db_name)</span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>slm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'stablelm-zephyr'</span>) </span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>llm <span class="op">=</span> ChatOllama(model<span class="op">=</span><span class="st">'mixtral:instruct'</span>) </span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>llm2 <span class="op">=</span> ChatOpenAI(model<span class="op">=</span><span class="st">'gpt-4-1106-preview'</span>) </span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>d <span class="op">=</span> (<span class="st">"This tool attempts to draw a protocol design from the description of a scientific paper."</span>)</span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>t <span class="op">=</span> ProcotolExtractionTool(db<span class="op">=</span>ldb, llm<span class="op">=</span>llm2, description<span class="op">=</span>d)</span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>t.run(tool_input<span class="op">=</span>{<span class="st">'paper_id'</span>: <span class="st">'doi:10.1101/2022.04.12.488077'</span>, <span class="st">'extraction_type'</span>: <span class="st">'cryoet'</span>})</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution_count="72">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pymde</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> torch</span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>ldb.session.rollback()</span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>rag_embeddings_list <span class="op">=</span> [json.loads(e[<span class="dv">0</span>]) <span class="cf">for</span> e <span class="kw">in</span> ldb.session.execute(text(<span class="st">"""</span></span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a><span class="st">                        SELECT DISTINCT emb.embedding </span></span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a><span class="st">                         FROM langchain_pg_embedding as emb, </span></span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a><span class="st">                            "ScientificKnowledgeExpression" as ske,</span></span>
<span id="cb28-9"><a href="#cb28-9" aria-hidden="true" tabindex="-1"></a><span class="st">                            "ScientificKnowledgeCollection_has_members" as skc_hm</span></span>
<span id="cb28-10"><a href="#cb28-10" aria-hidden="true" tabindex="-1"></a><span class="st">                         WHERE cmetadata-&gt;&gt;'i_type' = 'CitationRecord' AND</span></span>
<span id="cb28-11"><a href="#cb28-11" aria-hidden="true" tabindex="-1"></a><span class="st">                            cmetadata-&gt;&gt;'e_id' = ske.id AND </span></span>
<span id="cb28-12"><a href="#cb28-12" aria-hidden="true" tabindex="-1"></a><span class="st">                            ske.id = skc_hm.has_members_id AND</span></span>
<span id="cb28-13"><a href="#cb28-13" aria-hidden="true" tabindex="-1"></a><span class="st">                            skc_hm."ScientificKnowledgeCollection_id"='0';</span></span>
<span id="cb28-14"><a href="#cb28-14" aria-hidden="true" tabindex="-1"></a><span class="st">                        """</span>)).fetchall()]</span>
<span id="cb28-15"><a href="#cb28-15" aria-hidden="true" tabindex="-1"></a>rag_embeddings_tensor <span class="op">=</span> torch.FloatTensor(rag_embeddings_list)</span>
<span id="cb28-16"><a href="#cb28-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-17"><a href="#cb28-17" aria-hidden="true" tabindex="-1"></a>proj_embeddings <span class="op">=</span> pymde.preserve_neighbors(rag_embeddings_tensor, constraint<span class="op">=</span>pymde.Standardized()).embed()</span>
<span id="cb28-18"><a href="#cb28-18" aria-hidden="true" tabindex="-1"></a>pymde.plot(proj_embeddings)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="72">
<pre><code>&lt;Axes: &gt;</code></pre>
</div>
<div class="cell-output cell-output-display">
<p><img src="RNAquarium_files/figure-html/cell-24-output-2.png" class="img-fluid"></p>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>