{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Full Text Reading \n",
    "\n",
    "> Basic Experiments with Pipelines to read full text papers as part of CUSO sprint on corpus reviews about transcriptomics and machine learning.    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "\n",
    "import os\n",
    "import pandas as pd\n",
    "from pprint import pprint\n",
    "import textwrap\n",
    "\n",
    "import sqlalchemy as db\n",
    "from sqlalchemy.engine import create_engine\n",
    "import json\n",
    "\n",
    "import os\n",
    "import pandas as pd\n",
    "from pprint import pprint\n",
    "import textwrap\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# NXML Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.biorxiv.org/content/early/2022/04/13/2022.04.12.488077.source.xml\n",
      "200\n",
      "application/xml\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'<article article-type=\"article\" specific-use=\"production\" xml:lang=\"en\" xmlns:hw=\"org.highwire.hpp\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" xmlns:ref=\"http://schema.highwire.org/Reference\" xmlns:hwp=\"http://schema.highwire.org/Journal\" xmlns:l=\"http://schema.highwire.org/Linking\" xmlns:r=\"http://schema.highwire.org/Revision\" xmlns:x=\"http://www.w3.org/1999/xhtml\" xmlns:app=\"http://www.w3.org/2007/app\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:nlm=\"http://schema.highwire.org/NLM/Journal\" xmlns:a=\"http://www.w3.org/2005/Atom\" xmlns:c=\"http://schema.highwire.org/Compound\" xmlns:hpp=\"http://schema.highwire.org/Publishing\"><front><journal-meta><journal-id journal-id-type=\"hwp\">biorxiv</journal-id><journal-id journal-id-type=\"publisher-id\">BIORXIV</journal-id><journal-title>bioRxiv</journal-title><abbrev-journal-title abbrev-type=\"publisher\">bioRxiv</abbrev-journal-title><publisher><publisher-name>Cold Spring Harbor Laboratory</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type=\"doi\">10.1101/2022.04.12.488077</article-id><article-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1</article-id><article-id pub-id-type=\"other\" hwp:sub-type=\"pisa-master\">biorxiv;2022.04.12.488077</article-id><article-id pub-id-type=\"other\" hwp:sub-type=\"slug\">2022.04.12.488077</article-id><article-id pub-id-type=\"other\" hwp:sub-type=\"atom-slug\">2022.04.12.488077</article-id><article-id pub-id-type=\"other\" hwp:sub-type=\"tag\">2022.04.12.488077</article-id><article-version>1.1</article-version><article-categories><subj-group subj-group-type=\"author-type\"><subject>Regular Article</subject></subj-group><subj-group subj-group-type=\"heading\"><subject>New Results</subject></subj-group><subj-group subj-group-type=\"hwp-journal-coll\" hwp:journal-coll-id=\"Bioinformatics\" hwp:journal=\"biorxiv\"><subject>Bioinformatics</subject></subj-group></article-categories><title-group><article-title hwp:id=\"article-title-1\">Convolutional networks for supervised mining of molecular patterns within cellular context</article-title></title-group><author-notes hwp:id=\"author-notes-1\"><corresp id=\"cor1\" hwp:id=\"corresp-1\" hwp:rev-id=\"xref-corresp-1-1 xref-corresp-1-2\"><label>+</label> correspondence to: <email hwp:id=\"email-1\">julia.mahamid@embl.de</email>, <email hwp:id=\"email-2\">judith.zaugg@embl.de</email></corresp><fn id=\"n1\" fn-type=\"equal\" hwp:id=\"fn-1\" hwp:rev-id=\"xref-fn-1-1 xref-fn-1-2\"><label>*</label><p hwp:id=\"p-1\">These authors contributed equally to this work.</p></fn></author-notes><contrib-group hwp:id=\"contrib-group-1\"><contrib contrib-type=\"author\" hwp:id=\"contrib-1\"><name name-style=\"western\" hwp:sortable=\"de Teresa Irene\"><surname>de Teresa</surname><given-names>Irene</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-1\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a2\" hwp:id=\"xref-aff-2-1\" hwp:rel-id=\"aff-2\">2</xref><xref ref-type=\"fn\" rid=\"n1\" hwp:id=\"xref-fn-1-1\" hwp:rel-id=\"fn-1\">*</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-2\"><name name-style=\"western\" hwp:sortable=\"Goetz Sara K.\"><surname>Goetz</surname><given-names>Sara K.</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-2\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a3\" hwp:id=\"xref-aff-3-1\" hwp:rel-id=\"aff-3\">3</xref><xref ref-type=\"fn\" rid=\"n1\" hwp:id=\"xref-fn-1-2\" hwp:rel-id=\"fn-1\">*</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-3\"><name name-style=\"western\" hwp:sortable=\"Mattausch Alexander\"><surname>Mattausch</surname><given-names>Alexander</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-3\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a4\" hwp:id=\"xref-aff-4-1\" hwp:rel-id=\"aff-4\">4</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-4\"><name name-style=\"western\" hwp:sortable=\"Stojanovska Frosina\"><surname>Stojanovska</surname><given-names>Frosina</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-4\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a3\" hwp:id=\"xref-aff-3-2\" hwp:rel-id=\"aff-3\">3</xref><xref ref-type=\"aff\" rid=\"a8\" hwp:id=\"xref-aff-8-1\" hwp:rel-id=\"aff-8\">8</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-5\"><name name-style=\"western\" hwp:sortable=\"Zimmerli Christian E.\"><surname>Zimmerli</surname><given-names>Christian E.</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-5\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a5\" hwp:id=\"xref-aff-5-1\" hwp:rel-id=\"aff-5\">5</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-6\"><name name-style=\"western\" hwp:sortable=\"Toro-Nahuelpan Mauricio\"><surname>Toro-Nahuelpan</surname><given-names>Mauricio</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-6\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a6\" hwp:id=\"xref-aff-6-1\" hwp:rel-id=\"aff-6\">6</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-7\"><name name-style=\"western\" hwp:sortable=\"Cheng Dorothy W.C.\"><surname>Cheng</surname><given-names>Dorothy W.C.</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-7\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a3\" hwp:id=\"xref-aff-3-3\" hwp:rel-id=\"aff-3\">3</xref><xref ref-type=\"aff\" rid=\"a7\" hwp:id=\"xref-aff-7-1\" hwp:rel-id=\"aff-7\">7</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-8\"><name name-style=\"western\" hwp:sortable=\"Tollervey Fergus\"><surname>Tollervey</surname><given-names>Fergus</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-8\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a3\" hwp:id=\"xref-aff-3-4\" hwp:rel-id=\"aff-3\">3</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-9\"><name name-style=\"western\" hwp:sortable=\"Pape Constantin\"><surname>Pape</surname><given-names>Constantin</given-names></name><xref ref-type=\"aff\" rid=\"a7\" hwp:id=\"xref-aff-7-2\" hwp:rel-id=\"aff-7\">7</xref><xref ref-type=\"aff\" rid=\"a9\" hwp:id=\"xref-aff-9-1\" hwp:rel-id=\"aff-9\">9</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-10\"><name name-style=\"western\" hwp:sortable=\"Beck Martin\"><surname>Beck</surname><given-names>Martin</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-9\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a5\" hwp:id=\"xref-aff-5-2\" hwp:rel-id=\"aff-5\">5</xref></contrib><contrib contrib-type=\"author\" hwp:id=\"contrib-11\"><contrib-id contrib-id-type=\"orcid\" authenticated=\"false\">http://orcid.org/0000-0003-1334-6388</contrib-id><name name-style=\"western\" hwp:sortable=\"Kreshuk Anna\"><surname>Kreshuk</surname><given-names>Anna</given-names></name><xref ref-type=\"aff\" rid=\"a7\" hwp:id=\"xref-aff-7-3\" hwp:rel-id=\"aff-7\">7</xref><object-id pub-id-type=\"other\" hwp:sub-type=\"orcid\" xlink:href=\"http://orcid.org/0000-0003-1334-6388\"/></contrib><contrib contrib-type=\"author\" corresp=\"yes\" hwp:id=\"contrib-12\"><contrib-id contrib-id-type=\"orcid\" authenticated=\"false\">http://orcid.org/0000-0001-6968-041X</contrib-id><name name-style=\"western\" hwp:sortable=\"Mahamid Julia\"><surname>Mahamid</surname><given-names>Julia</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-10\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a7\" hwp:id=\"xref-aff-7-4\" hwp:rel-id=\"aff-7\">7</xref><xref ref-type=\"corresp\" rid=\"cor1\" hwp:id=\"xref-corresp-1-1\" hwp:rel-id=\"corresp-1\">+</xref><object-id pub-id-type=\"other\" hwp:sub-type=\"orcid\" xlink:href=\"http://orcid.org/0000-0001-6968-041X\"/></contrib><contrib contrib-type=\"author\" corresp=\"yes\" hwp:id=\"contrib-13\"><contrib-id contrib-id-type=\"orcid\" authenticated=\"false\">http://orcid.org/0000-0001-8324-4040</contrib-id><name name-style=\"western\" hwp:sortable=\"Zaugg Judith\"><surname>Zaugg</surname><given-names>Judith</given-names></name><xref ref-type=\"aff\" rid=\"a1\" hwp:id=\"xref-aff-1-11\" hwp:rel-id=\"aff-1\">1</xref><xref ref-type=\"aff\" rid=\"a8\" hwp:id=\"xref-aff-8-2\" hwp:rel-id=\"aff-8\">8</xref><xref ref-type=\"corresp\" rid=\"cor1\" hwp:id=\"xref-corresp-1-2\" hwp:rel-id=\"corresp-1\">+</xref><object-id pub-id-type=\"other\" hwp:sub-type=\"orcid\" xlink:href=\"http://orcid.org/0000-0001-8324-4040\"/></contrib><aff id=\"a1\" hwp:id=\"aff-1\" hwp:rev-id=\"xref-aff-1-1 xref-aff-1-2 xref-aff-1-3 xref-aff-1-4 xref-aff-1-5 xref-aff-1-6 xref-aff-1-7 xref-aff-1-8 xref-aff-1-9 xref-aff-1-10 xref-aff-1-11\"><label>1</label><institution hwp:id=\"institution-1\">Structural and Computational Biology Unit, European Molecular Biology Laboratory</institution>, Heidelberg, <country>Germany</country>.</aff><aff id=\"a2\" hwp:id=\"aff-2\" hwp:rev-id=\"xref-aff-2-1\"><label>2</label><institution hwp:id=\"institution-2\">Computer Science and Artificial Intelligence Lab, ENGIE Lab Crigen</institution>, Stains, <country>France</country>.</aff><aff id=\"a3\" hwp:id=\"aff-3\" hwp:rev-id=\"xref-aff-3-1 xref-aff-3-2 xref-aff-3-3 xref-aff-3-4\"><label>3</label><institution hwp:id=\"institution-3\">Collaboration for Joint PhD Degree between EMBL and Heidelberg University, Faculty of Biosciences</institution>, Heidelberg, <country>Germany</country>.</aff><aff id=\"a4\" hwp:id=\"aff-4\" hwp:rev-id=\"xref-aff-4-1\"><label>4</label><institution hwp:id=\"institution-4\">Institute of Pharmacy and Molecular Biotechnology, Heidelberg University</institution>, Heidelberg, <country>Germany</country>.</aff><aff id=\"a5\" hwp:id=\"aff-5\" hwp:rev-id=\"xref-aff-5-1 xref-aff-5-2\"><label>5</label><institution hwp:id=\"institution-5\">Max Planck Institute of Biophysics, Department of Molecular Sociology</institution>, Frankfurt, <country>Germany</country>.</aff><aff id=\"a6\" hwp:id=\"aff-6\" hwp:rev-id=\"xref-aff-6-1\"><label>6</label><institution hwp:id=\"institution-6\">Santiago GmbH &amp; Co. KG</institution>, Willich, <country>Germany</country>.</aff><aff id=\"a7\" hwp:id=\"aff-7\" hwp:rev-id=\"xref-aff-7-1 xref-aff-7-2 xref-aff-7-3 xref-aff-7-4\"><label>7</label><institution hwp:id=\"institution-7\">Cell Biology and Biophysics Unit, European Molecular Biology Laboratory</institution>, Heidelberg, <country>Germany</country>.</aff><aff id=\"a8\" hwp:id=\"aff-8\" hwp:rev-id=\"xref-aff-8-1 xref-aff-8-2\"><label>8</label><institution hwp:id=\"institution-8\">Genome Biology Unit, European Molecular Biology Laboratory</institution>, Heidelberg, <country>Germany</country>.</aff><aff id=\"a9\" hwp:id=\"aff-9\" hwp:rev-id=\"xref-aff-9-1\"><label>9</label><institution hwp:id=\"institution-9\">Institute for Computer Science, Universität Göttingen</institution></aff></contrib-group><pub-date pub-type=\"epub-original\" date-type=\"pub\" publication-format=\"electronic\" hwp:start=\"2022\"><year>2022</year></pub-date><pub-date pub-type=\"hwp-created\" hwp:start=\"2022-04-13T11:21:14-07:00\">\\n    <day>13</day><month>4</month><year>2022</year>\\n  </pub-date><pub-date pub-type=\"hwp-received\" hwp:start=\"2022-04-13T11:21:14-07:00\">\\n    <day>13</day><month>4</month><year>2022</year>\\n  </pub-date><pub-date pub-type=\"epub\" hwp:start=\"2022-04-13T11:25:29-07:00\">\\n    <day>13</day><month>4</month><year>2022</year>\\n  </pub-date><pub-date pub-type=\"epub-version\" hwp:start=\"2022-04-13T11:25:29-07:00\">\\n    <day>13</day><month>4</month><year>2022</year>\\n  </pub-date><elocation-id>2022.04.12.488077</elocation-id><history hwp:id=\"history-1\">\\n<date date-type=\"received\" hwp:start=\"2022-04-12\"><day>12</day><month>4</month><year>2022</year></date>\\n<date date-type=\"rev-recd\" hwp:start=\"2022-04-12\"><day>12</day><month>4</month><year>2022</year></date>\\n<date date-type=\"accepted\" hwp:start=\"2022-04-13\"><day>13</day><month>4</month><year>2022</year></date>\\n</history><permissions><copyright-statement hwp:id=\"copyright-statement-1\">© 2022, Posted by Cold Spring Harbor Laboratory</copyright-statement><copyright-year>2022</copyright-year><license license-type=\"creative-commons\" xlink:href=\"http://creativecommons.org/licenses/by-nc-nd/4.0/\" hwp:id=\"license-1\"><p hwp:id=\"p-2\">This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at <ext-link l:rel=\"related\" l:ref-type=\"uri\" l:ref=\"http://creativecommons.org/licenses/by-nc-nd/4.0/\" ext-link-type=\"uri\" xlink:href=\"http://creativecommons.org/licenses/by-nc-nd/4.0/\" hwp:id=\"ext-link-1\">http://creativecommons.org/licenses/by-nc-nd/4.0/</ext-link></p></license></permissions><self-uri xlink:href=\"488077.pdf\" content-type=\"pdf\" xlink:role=\"full-text\"/><self-uri l:ref=\"forthcoming:yes\" c:role=\"http://schema.highwire.org/variant/abstract\" xlink:role=\"abstract\" content-type=\"xhtml+xml\" hwp:variant=\"yes\"/><self-uri l:ref=\"forthcoming:yes\" c:role=\"http://schema.highwire.org/variant/external-links\" xlink:role=\"external-links\" content-type=\"xhtml+xml\" hwp:variant=\"yes\"/><self-uri l:ref=\"forthcoming:yes\" c:role=\"http://schema.highwire.org/variant/full-text\" xlink:href=\"file:/content/biorxiv/vol0/issue2022/pdf/2022.04.12.488077v1.pdf\" hwp:variant=\"yes\" content-type=\"pdf\" xlink:role=\"full-text\"/><self-uri l:ref=\"forthcoming:yes\" c:role=\"http://schema.highwire.org/variant/full-text\" xlink:role=\"full-text\" content-type=\"xhtml+xml\" hwp:variant=\"yes\"/><self-uri l:ref=\"forthcoming:yes\" c:role=\"http://schema.highwire.org/variant/source\" xlink:role=\"source\" content-type=\"xml\" xlink:show=\"none\" hwp:variant=\"yes\"/><self-uri l:ref=\"forthcoming:yes\" c:role=\"http://schema.highwire.org/variant/original\" xlink:role=\"original\" content-type=\"xml\" xlink:show=\"none\" hwp:variant=\"yes\" xlink:href=\"488077.xml\"/><self-uri content-type=\"abstract\" xlink:href=\"file:/content/biorxiv/vol0/issue2022/abstracts/2022.04.12.488077v1/2022.04.12.488077v1.htslp\"/><self-uri content-type=\"fulltext\" xlink:href=\"file:/content/biorxiv/vol0/issue2022/fulltext/2022.04.12.488077v1/2022.04.12.488077v1.htslp\"/><abstract hwp:id=\"abstract-1\"><title hwp:id=\"title-1\">Abstract</title><p hwp:id=\"p-3\">Cryo-electron tomograms capture a wealth of structural information on the molecular constituents of cells and tissues. We present DeePiCt (Deep Picker in Context), an open-source deep-learning framework for supervised structure segmentation and macromolecular complex localization in cellular cryo-electron tomography. To train and benchmark DeePiCt on experimental data, we comprehensively annotated 20 tomograms of <italic toggle=\"yes\">Schizosaccharomyces pombe</italic> for ribosomes, fatty acid synthases, membranes, nuclear pore complexes, organelles and cytosol. By comparing our method to state-of-the-art approaches on this dataset, we show its unique ability to identify low-abundance and low-density complexes. We use DeePiCt to study compositionally-distinct subpopulations of cellular ribosomes, with emphasis on their contextual association with mitochondria and the endoplasmic reticulum. Finally, by applying pre-trained networks to a HeLa cell dataset, we demonstrate that DeePiCt achieves high-quality predictions in unseen datasets from different biological species in a matter of minutes. The comprehensively annotated experimental data and pre-trained networks are provided for immediate exploitation by the community.</p></abstract><counts><page-count count=\"66\"/></counts><custom-meta-wrap><custom-meta hwp:id=\"custom-meta-1\"><meta-name>special-property</meta-name><meta-value>contains-inline-supplementary-material</meta-value></custom-meta></custom-meta-wrap></article-meta><notes hwp:id=\"notes-1\"><notes notes-type=\"competing-interest-statement\" hwp:id=\"notes-2\"><title hwp:id=\"title-2\">Competing Interest Statement</title><p hwp:id=\"p-4\">The authors have declared no competing interest.</p></notes><fn-group content-type=\"external-links\" hwp:id=\"fn-group-1\"><fn fn-type=\"dataset\" hwp:id=\"fn-2\"><p hwp:id=\"p-5\">\\n<ext-link l:rel=\"related\" l:ref-type=\"uri\" l:ref=\"https://github.com/ZauggGroup/DeePiCt\" ext-link-type=\"uri\" xlink:href=\"https://github.com/ZauggGroup/DeePiCt\" hwp:id=\"ext-link-2\">https://github.com/ZauggGroup/DeePiCt</ext-link>\\n</p></fn></fn-group></notes></front><body><sec id=\"s1\" hwp:id=\"sec-1\"><title hwp:id=\"title-3\">Introduction</title><p hwp:id=\"p-6\">Cryo-Electron Tomography (cryo-ET) produces three-dimensional (3D) snapshots of cellular landscapes at molecular resolution, making it possible to investigate the structural and functional states of macromolecular complexes in their native environment, and to unveil how different macromolecular populations interact with other cellular structures<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-1\" hwp:rel-id=\"ref-1\">1</xref>–<xref ref-type=\"bibr\" rid=\"c8\" hwp:id=\"xref-ref-8-1\" hwp:rel-id=\"ref-8\">8</xref></sup>. With improved instrumentation, sample preparation protocols, and automation, high-quality in-cell cryo-ET data are rapidly being generated, opening the possibility to conduct high-throughput studies<sup><xref ref-type=\"bibr\" rid=\"c9\" hwp:id=\"xref-ref-9-1\" hwp:rel-id=\"ref-9\">9</xref>–<xref ref-type=\"bibr\" rid=\"c12\" hwp:id=\"xref-ref-12-1\" hwp:rel-id=\"ref-12\">12</xref></sup>. However, due to the complex and crowded nature of the intracellular milieu, together with limitations arising from cryo-ET image acquisition (low signal-to-noise ratio (SNR) and incomplete angular sampling) data mining of 3D cryo-ET volumes remains a major bottleneck<sup><xref ref-type=\"bibr\" rid=\"c13\" hwp:id=\"xref-ref-13-1\" hwp:rel-id=\"ref-13\">13</xref>, <xref ref-type=\"bibr\" rid=\"c14\" hwp:id=\"xref-ref-14-1\" hwp:rel-id=\"ref-14\">14</xref></sup>. Data mining in terms of reliable identification of a relatively homogenous set of macromolecular complexes constitutes a fundamental prerequisite for structural analysis<sup><xref ref-type=\"bibr\" rid=\"c13\" hwp:id=\"xref-ref-13-2\" hwp:rel-id=\"ref-13\">13</xref>–<xref ref-type=\"bibr\" rid=\"c15\" hwp:id=\"xref-ref-15-1\" hwp:rel-id=\"ref-15\">15</xref></sup>.</p><p hwp:id=\"p-7\">A range of available semi-automated methods for segmentation of cellular structures and localization of macromolecular complexes (from here on, particles) in cryo-ET datasets are broadly classified as template-based and template-free approaches<sup><xref ref-type=\"bibr\" rid=\"c16\" hwp:id=\"xref-ref-16-1\" hwp:rel-id=\"ref-16\">16</xref></sup>. Traditional Template Matching (TM)<sup><xref ref-type=\"bibr\" rid=\"c17\" hwp:id=\"xref-ref-17-1\" hwp:rel-id=\"ref-17\">17</xref></sup> is a commonly applied computational approach and is based on a point-wise numerical computation of a similarity coefficient (cross-correlation) to a known template of the complex in question. TM is accurate in the localization of ribosomes and cytoskeletal filaments, but fails at identifying smaller or less dense particles<sup><xref ref-type=\"bibr\" rid=\"c18\" hwp:id=\"xref-ref-18-1\" hwp:rel-id=\"ref-18\">18</xref></sup>, and is computationally intensive. On the other hand, current template-free methods implementing classical image processing approaches are purposely designed for and thus limited to specific molecular shapes, or to special configurations in which particles are associated with large cellular structures such as membranes or microtubules<sup><xref ref-type=\"bibr\" rid=\"c16\" hwp:id=\"xref-ref-16-2\" hwp:rel-id=\"ref-16\">16</xref>, <xref ref-type=\"bibr\" rid=\"c19\" hwp:id=\"xref-ref-19-1\" hwp:rel-id=\"ref-19\">19</xref>, <xref ref-type=\"bibr\" rid=\"c20\" hwp:id=\"xref-ref-20-1\" hwp:rel-id=\"ref-20\">20</xref></sup>. These methods together with TM require manual inspection and are therefore laborious and time-consuming. The advent of deep learning methods, and particularly convolutional neural networks (CNNs)<sup><xref ref-type=\"bibr\" rid=\"c21\" hwp:id=\"xref-ref-21-1\" hwp:rel-id=\"ref-21\">21</xref>–<xref ref-type=\"bibr\" rid=\"c23\" hwp:id=\"xref-ref-23-1\" hwp:rel-id=\"ref-23\">23</xref></sup>, has enabled developing more generally applicable and automated approaches to accelerate the tasks of segmentation and localization in cryo-ET. The first of such methods was a two-dimensional (2D) CNN for semantic segmentation of ribosomes and large structures, such as cellular organelles or membranes<sup><xref ref-type=\"bibr\" rid=\"c24\" hwp:id=\"xref-ref-24-1\" hwp:rel-id=\"ref-24\">24</xref></sup>. However, its 2D nature makes it less suitable for particle localization, where probing the 3D structure becomes beneficial. More recently, DeepFinder, a fully supervised method based on the U-Net architecture<sup><xref ref-type=\"bibr\" rid=\"c25\" hwp:id=\"xref-ref-25-1\" hwp:rel-id=\"ref-25\">25</xref></sup> for multi-class semantic segmentation, has positioned itself as the state-of-the-art in automated particle localization in both simulated and real cryo-ET datasets<sup><xref ref-type=\"bibr\" rid=\"c26\" hwp:id=\"xref-ref-26-1\" hwp:rel-id=\"ref-26\">26</xref>–<xref ref-type=\"bibr\" rid=\"c29\" hwp:id=\"xref-ref-29-1\" hwp:rel-id=\"ref-29\">29</xref></sup>. Remaining limitations include the identification of less prevalent particles, as well as the interpretation of the obtained predictions within their cellular context. Furthermore, the field lacks publicly available non-synthetic expert-annotated cryo-ET data to train new models and to benchmark the available methods.</p><p hwp:id=\"p-8\">Here we present DeePiCt (Deep Picker in Context), our open-source software which synergizes supervised convolutional networks for segmentation of cellular compartments (organelles or cytosol) and structures (membranes or cytoskeletal filaments), and localization of particles. We generated a set of comprehensively annotated tomograms of wild-type <italic toggle=\"yes\">S. pombe</italic> for training and benchmarking our method, which we make publicly available to overcome the critical limitations arising from the absence of publicly available annotated experimental datasets. From here on, we refer to the term <italic toggle=\"yes\">network</italic> as the deep learning algorithm itself and <italic toggle=\"yes\">model</italic> as the algorithm once already trained. We provide DeePiCt models, trained on this dataset which show high data-mining performance and can be readily applied across species and datasets.</p></sec><sec id=\"s2\" hwp:id=\"sec-2\"><title hwp:id=\"title-4\">Results</title><sec id=\"s2a\" hwp:id=\"sec-3\"><title hwp:id=\"title-5\">DeePiCt: A deep-learning approach for automated compartment segmentation and particle localization</title><p hwp:id=\"p-9\">DeePiCt is based on deep learning technologies. It combines a 2D CNN for segmentation of cellular compartments and a 3D CNN for particle localization, membrane, and cytoskeletal filament annotations (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-1\" hwp:rel-id=\"F1\">Fig. 1</xref></bold>). This synergy allows for more precise particle picking and interpretation of their cellular context. The 2D and 3D CNN are adapted from the original U-Net architecture<sup><xref ref-type=\"bibr\" rid=\"c25\" hwp:id=\"xref-ref-25-2\" hwp:rel-id=\"ref-25\">25</xref></sup> (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-2\" hwp:rel-id=\"F1\">Fig. 1a</xref>, Supplementary Note 1</bold>). Since cellular compartments are easily recognizable in 2D image data, the compartment segmentation network operates in 2D, which has the advantage of requiring little training data. The 2D CNN employs a fixed depth (D) of 5 (4 max-pooling layers) and 16 initial filters (IF) (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-3\" hwp:rel-id=\"F1\">Fig. 1a</xref></bold>). Our implementation of the 3D CNN allows multi-label learning (<bold>Supplementary Note 1</bold>) and adjustable architectural parameters D, number of IF, a batch normalization layer (BN), and the dropout parameter in the encoder and decoder paths (ED and DD, respectively) (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-4\" hwp:rel-id=\"F1\">Fig. 1a</xref></bold>). These parameters can be set according to the amount and quality of training data, and the shape complexity, size and abundance of the particle of interest as suggested in <bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-1\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>. In general, larger particles benefit from larger values of D to increase the network’s receptive field, and particles with a low-density print (low SNR with respect to the surrounding context) require larger IF values. The remaining optional layers, BN and dropout, are well known techniques in computer vision to avoid overfitting<sup><xref ref-type=\"bibr\" rid=\"c30\" hwp:id=\"xref-ref-30-1\" hwp:rel-id=\"ref-30\">30</xref>, <xref ref-type=\"bibr\" rid=\"c31\" hwp:id=\"xref-ref-31-1\" hwp:rel-id=\"ref-31\">31</xref></sup>.</p><fig id=\"fig1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F1\" hwp:rev-id=\"xref-fig-1-1 xref-fig-1-2 xref-fig-1-3 xref-fig-1-4 xref-fig-1-5 xref-fig-1-6 xref-fig-1-7 xref-fig-1-8 xref-fig-1-9 xref-fig-1-10 xref-fig-1-11 xref-fig-1-12 xref-fig-1-13 xref-fig-1-14 xref-fig-1-15 xref-fig-1-16 xref-fig-1-17 xref-fig-1-18 xref-fig-1-19\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIG1</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F1</object-id><object-id pub-id-type=\"publisher-id\">fig1</object-id><label>Figure 1</label><caption hwp:id=\"caption-1\"><title hwp:id=\"title-6\">DeePiCt 2D and 3D CNN architecture implemented in an automated workflow combining compartment segmentations and particle localizations in cryo-ET data.</title><p hwp:id=\"p-10\"><bold>a</bold>. The CNN U-Net architecture: the 2D network performs all tensor operations on the two-dimensional spatial coordinates, with D=5 and IF=16; the 3D network performs tensor operations on the three spatial dimensions, and the architectural hyperparameters in red can be set by the user. <bold>b</bold>. The DeePiCt pipeline is used to train and predict various structural features in cryo-electron tomograms. <bold>b.1</bold> The DeePiCt pipeline consists of two independent CNNs: a 2D network for compartment segmentation and a 3D network for particle localization. <bold>b.2</bold> Trained networks are applied to input tomograms which can be pre-processed with a spectrum-matching filter to improve image contrast. The example 2D tomographic slice visualizes the cytoplasm with the endoplasmic reticulum (ER), vacuoles (V), nucleus and extracellular space (ECS). <bold>b.3</bold> DeePict raw predictions for cytosol, membranes, ribosomes and fatty acid synthase (FAS) are post-processed by thresholding, cluster size and centroid fitting. <bold>b.4</bold> The outputs of the two networks can be combined to include the cellular context by intersecting particle predictions with cytosol masks (top), selecting particles (NPC, green) in contact with specific membranes (nuclear envelope, middle), and to identify particles (e.g. ribosomes, orange) associated with specific organelles (e.g. ER).</p></caption><graphic xlink:href=\"488077v1_fig1\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-1\"/></fig><p hwp:id=\"p-11\">For training, each network requires tomograms and corresponding 3D binary masks, which represent the segmentation of the structures of interest, e.g. organelle segmentations for the 2D U-Net and spheres representing particles for the 3D U-Net (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-5\" hwp:rel-id=\"F1\">Fig. 1b</xref>.1)</bold>. The raw input tomograms are optionally preprocessed using an amplitude spectrum equalization filter (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-6\" hwp:rel-id=\"F1\">Fig. 1b</xref>.<xref ref-type=\"fig\" rid=\"figs2\" hwp:id=\"xref-fig-17-1\" hwp:rel-id=\"F17\">2</xref>, <xref ref-type=\"fig\" rid=\"figE1\" hwp:id=\"xref-fig-6-1\" hwp:rel-id=\"F6\">Extended Data Fig. 1</xref></bold>) to enhance image contrast by matching the amplitude spectrum of the input tomogram to a preselected target amplitude spectrum distribution (<bold><xref ref-type=\"fig\" rid=\"figE1\" hwp:id=\"xref-fig-6-2\" hwp:rel-id=\"F6\">Extended Data Fig. 1</xref></bold>). Both CNNs use the Adam optimizer algorithm and the Dice Loss function for training (<bold>Supplementary Note 1</bold>). In the 2D network, tiles are randomly flipped and rotated in 90-degree increments during training to improve generalization. For the 3D CNN, we implemented a data augmentation strategy by applying a number of optional random transformations to the input images (<bold>Supplementary Note 1</bold>). In our experience, more than 700 annotated instances are required for particle learning in the 3D CNNs, while about 5 tomograms are sufficient for membrane segmentation. In the case of 2D CNNs, 6 fully segmented tomograms are enough for training networks identifying either a cytosol class or a single class including all cellular organelles.</p><fig id=\"figE1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F6\" hwp:rev-id=\"xref-fig-6-1 xref-fig-6-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE1</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F6</object-id><object-id pub-id-type=\"publisher-id\">figE1</object-id><label>Extended Data Fig. 1</label><caption hwp:id=\"caption-6\"><title hwp:id=\"title-52\">Pre-processing of input data by spectrum equalization.</title><p hwp:id=\"p-86\"><bold>a</bold>. Input DEF tomogram with low signal-to-noise ratio. <bold>b</bold>. Target VPP tomogram with high signal-to-noise ratio. <bold>c</bold>. Output showing the tomogram in <bold>a</bold> after spectrum matching. <bold>d</bold>. Amplitude spectra of target, input and output tomograms, as well as per-frequency (1/Å) scaling factors of the rotational equalization kernel (y-axis cropped).</p></caption><graphic xlink:href=\"488077v1_figE1\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-6\"/></fig><p hwp:id=\"p-12\">For predicting, the trained networks receive as input unseen tomograms pre-processed in the same way as the training data, and output 3D probability maps that are subsequently post-processed (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-7\" hwp:rel-id=\"F1\">Fig. 1b</xref>.3</bold>). Automated post-processing for the 2D network includes the combination of predicted 2D slices into a 3D volume, application of a 1D Gaussian filter along the z-axis for smoothing the predictions across the z slices, and thresholding (user-definable, default=0.75) (<bold><xref ref-type=\"fig\" rid=\"figE2\" hwp:id=\"xref-fig-7-1\" hwp:rel-id=\"F7\">Extended Data Fig. 2a-c</xref></bold>). The 3D CNN post-processing entails the generation of a predicted segmentation. To that end, the 3D CNN output is thresholded by a probability value set by the user and then clustered. The clustered output can be integrated with contextual information from a binary map representing a tomographic region (e.g. the cytosol segmentation from the 2D CNN). The mode of integration can be chosen among three different options: <italic toggle=\"yes\">intersection</italic>, <italic toggle=\"yes\">contact</italic>, or <italic toggle=\"yes\">colocalization</italic>, depending on the users’ specific application (<bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-8\" hwp:rel-id=\"F1\">Fig. 1b</xref>.4</bold>, <bold>Supplementary Note 1, <xref ref-type=\"fig\" rid=\"figs1\" hwp:id=\"xref-fig-16-1\" hwp:rel-id=\"F16\">Supplementary Fig. 1</xref></bold>). In the case of particle localization, the clusters’ centroids are used to obtain a list of the particle coordinates; their orientations and structural features can then be obtained by subtomogram analysis in external softwares (e. g. Warp<sup><xref ref-type=\"bibr\" rid=\"c32\" hwp:id=\"xref-ref-32-1\" hwp:rel-id=\"ref-32\">32</xref></sup>, M<sup><xref ref-type=\"bibr\" rid=\"c33\" hwp:id=\"xref-ref-33-1\" hwp:rel-id=\"ref-33\">33</xref></sup>, RELION<sup><xref ref-type=\"bibr\" rid=\"c34\" hwp:id=\"xref-ref-34-1\" hwp:rel-id=\"ref-34\">34</xref></sup>, Dynamo<sup><xref ref-type=\"bibr\" rid=\"c35\" hwp:id=\"xref-ref-35-1\" hwp:rel-id=\"ref-35\">35</xref></sup>, PyTom<sup><xref ref-type=\"bibr\" rid=\"c36\" hwp:id=\"xref-ref-36-1\" hwp:rel-id=\"ref-36\">36</xref></sup> or EMAN2<sup><xref ref-type=\"bibr\" rid=\"c37\" hwp:id=\"xref-ref-37-1\" hwp:rel-id=\"ref-37\">37</xref></sup>). For more details on the method, we refer to <bold>Supplementary Note 1</bold> and our github repository (Code and data availability).</p><fig id=\"figE2\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F7\" hwp:rev-id=\"xref-fig-7-1 xref-fig-7-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE2</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F7</object-id><object-id pub-id-type=\"publisher-id\">figE2</object-id><label>Extended Data Fig. 2</label><caption hwp:id=\"caption-7\"><title hwp:id=\"title-53\">2D CNN organelle and cytosol segmentations.</title><p hwp:id=\"p-87\">Isosurface view of organelle ground truth annotations (<bold>a</bold>), model output prior to (<bold>b</bold>) and after 3D post-processing (<bold>c</bold>). <bold>d-f</bold>. Organelle (<bold>d</bold>) and cytosol (<bold>e</bold>) predictions of the tomogram in <bold><xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-17\" hwp:rel-id=\"F1\">Fig. 1</xref> b</bold>. The 2D CNN’s segmentations (red) are overlaid with the manually created ground truth (blue), with overlapping regions (i.e. true positives) in white. Ground truth annotations in the top and bottom left corners in <bold>e</bold> (yellow arrowheads) are artifacts resulting from cross-slice interpolation in the manual annotation process. <bold>f</bold>. 3D post-processing improves performance of the 2D CNN as exemplified in (<bold>a-c</bold>).</p></caption><graphic xlink:href=\"488077v1_figE2\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-7\"/></fig></sec><sec id=\"s2b\" hwp:id=\"sec-4\"><title hwp:id=\"title-7\">Generation of comprehensive ground-truth annotations in <italic toggle=\"yes\">S. pombe</italic></title><p hwp:id=\"p-13\">The ultimate requirement for developing and benchmarking any deep neural network is the existence of a ground truth (gt) non-synthetic dataset. Since this was not available at the outset of this study, we first created a comprehensive annotation of cellular features in wild-type <italic toggle=\"yes\">S. pombe</italic> representing diverse structures, sizes and abundances.</p><p hwp:id=\"p-14\">In cryo-ET data, the complete annotation of a given cellular structure or particle species is a challenging process, even for large compartments or complexes with a clear structural signature like the ribosome. To achieve the most comprehensive annotation possible, we devised an iterative workflow to mine patterns in three different datasets (<bold>Online Methods</bold>). In a set of 10 tomograms acquired by combining defocus and Volta potential phase plate (VPP<sup><xref ref-type=\"bibr\" rid=\"c38\" hwp:id=\"xref-ref-38-1\" hwp:rel-id=\"ref-38\">38</xref>, <xref ref-type=\"bibr\" rid=\"c39\" hwp:id=\"xref-ref-39-1\" hwp:rel-id=\"ref-39\">39</xref></sup>) and a second set of 10 tomograms using defocus only (DEF), we annotated ribosomes (RIBO), fatty acid synthases (FAS), membranes, organelles and cytoplasm (<bold><xref ref-type=\"fig\" rid=\"fig2\" hwp:id=\"xref-fig-2-1\" hwp:rel-id=\"F2\">Fig. 2</xref>, Extended Data Tables 1-3</bold>). For the nuclear pore complex (NPC), given its structural flexibility, low abundance (on average 3 per tomogram), and large size, a third dataset denoted by DEF* was used<sup><xref ref-type=\"bibr\" rid=\"c40\" hwp:id=\"xref-ref-40-1\" hwp:rel-id=\"ref-40\">40</xref></sup>. It consists of 127 tomograms featuring a total of 2,830 NPC subunits (∼354 NPCs) to ensure enough training data and is composed of two subsets: one of higher and one of lower quality classified based on lamella thickness and tilt-series alignment error. A detailed description of the ground truth construction is available in the <bold>Online Methods</bold>.</p><fig id=\"fig2\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F2\" hwp:rev-id=\"xref-fig-2-1 xref-fig-2-2 xref-fig-2-3 xref-fig-2-4 xref-fig-2-5\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIG2</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F2</object-id><object-id pub-id-type=\"publisher-id\">fig2</object-id><label>Figure 2</label><caption hwp:id=\"caption-2\"><title hwp:id=\"title-8\">Iterative comprehensive annotation of ground truth for non-synthetic data.</title><p hwp:id=\"p-15\">The three columns (left to right) show: the annotation process for RIBO (<bold>a, d, g</bold>), FAS (<bold>b, e, h</bold>), membranes (<bold>c, f</bold>). <bold>a-f</bold> Show the cumulative predictions of 3 rounds of DeePiCt for RIBO and one round for FAS as cross-section overlaid on a tomographic slice (<bold>a-c</bold>) and in 3D view (<bold>d-f</bold>). Particles are classified as true positives recovered from the initial annotation (tp recovered; yellow for RIBO; green for FAS), new true positives provided by DeePiCt predictions (tp new; blue), and the corresponding false positives (fp; red). <bold>g, h</bold>. 3D views of the resulting ground truth, including false negatives (fn) distinguishing unrecovered fn from the initial annotations (pink) and final round of manual picking (salmon). <bold>i</bold>. Combined ground truth annotation of RIBO, FAS, membranes, and NPC (green cylindrical mask, manual annotation). <bold>j-k</bold>. Relative contributions of the DeePiCt rounds for RIBO (<bold>j</bold>) and FAS (<bold>k</bold>) identification are plotted across 10 VPP and 10 DEF <italic toggle=\"yes\">S. pombe</italic> tomograms: contribution of step 1 (yellow: TM and manual annotation for RIBO; green: manual annotation for FAS; salmon: initial annotations not detected by DeePiCt), step 2 (blue); and step 3 (pink bars). For FAS the initial annotation was not based on TM, and instead was purely manual. <bold>l</bold>, <bold>m</bold>. Subtomogram averages of all RIBO, exclusively DeePiCt-detected (tp new; blue) and all FAS particles in VPP and DEF ground truth, respectively.</p></caption><graphic xlink:href=\"488077v1_fig2\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-2\"/></fig><p hwp:id=\"p-16\">The resulting ground truth RIBO annotation in VPP consisted of 25,311 particles. An initial TM annotation contributed 61.6% (1,559 particles per tomogram (ppt), on average) of the final ground truth, whereas iterative rounds of DeePiCt on the incomplete ground truth contributed to a total of additional 21.36% (541 ppt on average) and final manual annotations to 17.05% (431 ppt on average) (<bold><xref ref-type=\"fig\" rid=\"fig2\" hwp:id=\"xref-fig-2-2\" hwp:rel-id=\"F2\">Fig. 2j</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE3\" hwp:id=\"xref-fig-8-1\" hwp:rel-id=\"F8\">Extended Data Fig. 3a,e</xref>, Extended Data Table 2</bold>).</p><fig id=\"figE3\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F8\" hwp:rev-id=\"xref-fig-8-1 xref-fig-8-2 xref-fig-8-3 xref-fig-8-4 xref-fig-8-5 xref-fig-8-6\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE3</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F8</object-id><object-id pub-id-type=\"publisher-id\">figE3</object-id><label>Extended Data Fig. 3</label><caption hwp:id=\"caption-8\"><title hwp:id=\"title-54\">Details of the construction of ground truth for RIBO and FAS in VPP and DEF tomograms.</title><p hwp:id=\"p-88\"><bold>a</bold>-<bold>b</bold>. Contributions of the steps applied during the construction of RIBO ground truth in the VPP (<bold>a</bold>) and DEF (<bold>b</bold>) datasets. Upper subplots show the absolute number of particles per tomogram (ppt): true positives from the initial annotations that were recovered in step 2 (yellow), newly identified true positives in step 2 (blue), particles that were in the initial annotation but that were not recovered in step 2 (false negatives from initial annotation; pink), unidentified particles from any of the two first steps, which were manually picked in step 3 (fuchsia), and total false negatives (red) in step 2. Lower subplots show the associated relative contribution, as a percentage of the ground truth per tomogram (ground truth panel) or as percentage of the predictions per tomogram in step 2 (prediction panel). <bold>c</bold>-<bold>d</bold>. Equivalent plots for FAS ground truth construction, where the only difference is that the recovered true positives originate from an initial manual annotation (green), as opposed to RIBO where we used TM additionally. <bold>e-f</bold>. Summary of the relative contributions in the RIBO (<bold>e</bold>) and FAS (<bold>f</bold>) ground truth across the 10 VPP and 10 DEF tomograms.</p></caption><graphic xlink:href=\"488077v1_figE3\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-8\"/></fig><p hwp:id=\"p-17\">The final ground truth annotation of FAS in the 10 VPP datasets amounts to 731 particles, from which 58.96% (43 ppt on average) constituted the initial annotations from manual picking, as TM failed. The combined predictions of DeePiCt on the incomplete ground truth contributed to 21.88% (16 ppt on average). For FAS we performed only one round of DeePiCt using the initial manual annotations, since we observed a high false positive rate in subsequent rounds, which is likely a consequence of the natural lower abundance of FAS in the tomograms and its low imprint density. The final manual picking added a significant fraction of 19% (14 ppt on average) (<bold><xref ref-type=\"fig\" rid=\"figE3\" hwp:id=\"xref-fig-8-2\" hwp:rel-id=\"F8\">Extended Data Fig. 3c,e</xref></bold>, <bold>Extended Data Table 3</bold>).</p><p hwp:id=\"p-18\">The final ground truth RIBO annotations on the DEF dataset sum a total of 25,901 particles across the 10 tomograms, from which only 19.23% (498 ppt on average) were included in the initial TM annotation (step 1), 19.39% (502 ppt on average) were a contribution of DeePiCt on incomplete ground truth, and 61.36% (1,590 ppt on average) were manually identified (<bold><xref ref-type=\"fig\" rid=\"fig2\" hwp:id=\"xref-fig-2-3\" hwp:rel-id=\"F2\">Fig. 2k-m</xref>, <xref ref-type=\"fig\" rid=\"figE3\" hwp:id=\"xref-fig-8-3\" hwp:rel-id=\"F8\">Extended Data Fig. 3b,e</xref></bold>), highlighting the value of our carefully curated annotations. In the case of FAS, the ground truth annotations across the 10 tomograms amount to 366 particles, from which 48.63% (18 ppt on average) came from the initial manual annotations (as TM failed), 36.88% (14 ppt on average) from DeePiCt trained on incomplete ground truth, and 14.48% (5 ppt on average) from the final manual picking (<bold><xref ref-type=\"fig\" rid=\"fig2\" hwp:id=\"xref-fig-2-4\" hwp:rel-id=\"F2\">Fig. 2k-m</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE3\" hwp:id=\"xref-fig-8-4\" hwp:rel-id=\"F8\">Extended Data Fig. 3d,f</xref></bold>). Overall, a final round of manual picking for RIBO ground truth annotation was notably more crucial in DEF compared to VPP.</p><p hwp:id=\"p-19\">Total numbers of RIBO in both acquisition types are comparable, while for FAS fewer particles were detected in DEF likely due to the lower SNR despite the application of an equalization filter (<bold><xref ref-type=\"fig\" rid=\"figE3\" hwp:id=\"xref-fig-8-5\" hwp:rel-id=\"F8\">Extended Data Fig. 3</xref>, Extended Data Table 2-3, Supplementary Note 2</bold>). FAS annotations were similarly distributed among the ground truth construction steps in the two acquisition conditions (<bold><xref ref-type=\"fig\" rid=\"figE3\" hwp:id=\"xref-fig-8-6\" hwp:rel-id=\"F8\">Extended Data Fig. 3</xref></bold>). Comparison to previous proteomics studies<sup><xref ref-type=\"bibr\" rid=\"c41\" hwp:id=\"xref-ref-41-1\" hwp:rel-id=\"ref-41\">41</xref>, <xref ref-type=\"bibr\" rid=\"c42\" hwp:id=\"xref-ref-42-1\" hwp:rel-id=\"ref-42\">42</xref></sup> shows that our annotations of fully assembled FAS in cryo-electron tomograms is in agreement with levels of FAS-<italic toggle=\"yes\">α</italic> and FAS-<italic toggle=\"yes\">β</italic> quantified by means of mass spectrometry, while the abundance of cytosolic ribosomal proteins detected by mass spectrometry are underestimated (<bold><xref ref-type=\"fig\" rid=\"figE4\" hwp:id=\"xref-fig-9-1\" hwp:rel-id=\"F9\">Extended Data Fig. 4</xref></bold>).</p><fig id=\"figE4\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F9\" hwp:rev-id=\"xref-fig-9-1\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE4</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F9</object-id><object-id pub-id-type=\"publisher-id\">figE4</object-id><label>Extended Data Fig. 4</label><caption hwp:id=\"caption-9\"><title hwp:id=\"title-55\"><italic toggle=\"yes\">S. pombe</italic> ribosome and FAS complex abundances in cryo-electron tomograms in comparison to proteomics analysis.</title><p hwp:id=\"p-89\">In this study (cryo-ET, left), fully assembled ribosomes (<bold>a</bold>) and FAS (<bold>b</bold>) were iteratively annotated in 20 ground truth cryo-electron tomograms and cytosolic concentrations calculated using the cytosol segmentations. Copy numbers per cell were calculated for an <italic toggle=\"yes\">S. pombe</italic> cell with the assumption of 30 % cytosolic volume<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-7\" hwp:rel-id=\"ref-1\">1</xref></sup> of a total of 150 µm<sup><xref ref-type=\"bibr\" rid=\"c3\" hwp:id=\"xref-ref-3-2\" hwp:rel-id=\"ref-3\">3</xref></sup> average cell volume<sup><xref ref-type=\"bibr\" rid=\"c2\" hwp:id=\"xref-ref-2-3\" hwp:rel-id=\"ref-2\">2</xref>, <xref ref-type=\"bibr\" rid=\"c3\" hwp:id=\"xref-ref-3-3\" hwp:rel-id=\"ref-3\">3</xref></sup>. <bold>a</bold>. With cryo-ET, an average of 671,303土 96,708 ribosomes/cell were annotated. Ribosome counts from individual tomograms are represented as individual points (red). Proteomics data of individual ribosomal proteins were derived from Marguerat et al.<sup><xref ref-type=\"bibr\" rid=\"c4\" hwp:id=\"xref-ref-4-1\" hwp:rel-id=\"ref-4\">4</xref></sup> (yellow) and Carpy et al.<sup><xref ref-type=\"bibr\" rid=\"c5\" hwp:id=\"xref-ref-5-1\" hwp:rel-id=\"ref-5\">5</xref></sup> (purple) and resulted in average of 97,795 土 99,298 and 343,511 土 244,226 ribosomes/cell, respectively. <bold>b</bold>. Each measurement displayed in the plot corresponds to 6 times fully assembled FAS counts per tomogram (the complex is constituted by six alpha and beta subunits). With cryo-ET, an average of 106,282 土 86,247 FAS subunits/cell were observed. Proteomics data of individual FAS proteins (subunit alpha in purple, beta in yellow) were derived from Marguerat et al.<sup><xref ref-type=\"bibr\" rid=\"c4\" hwp:id=\"xref-ref-4-2\" hwp:rel-id=\"ref-4\">4</xref></sup> and Carpy et al.<sup><xref ref-type=\"bibr\" rid=\"c5\" hwp:id=\"xref-ref-5-2\" hwp:rel-id=\"ref-5\">5</xref></sup> and resulted in average of 67,035土 12,600 and 202,263 土 47,086 FAS subunits/cell, respectively.</p></caption><graphic xlink:href=\"488077v1_figE4\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-9\"/></fig><p hwp:id=\"p-20\">To assess the quality of the RIBO and FAS particle annotations in the VPP and DEF datasets, we processed the tomograms in Warp and performed structural 3D classification and refinement of the subtomograms in RELION (<bold>Extended Data Tables 4-5, Supplementary Note. 2, <xref ref-type=\"fig\" rid=\"figs2\" hwp:id=\"xref-fig-17-2\" hwp:rel-id=\"F17\">Supplementary Fig. 2</xref>-<xref ref-type=\"fig\" rid=\"figs4\" hwp:id=\"xref-fig-19-1\" hwp:rel-id=\"F19\">4</xref></bold>). Subtomogram averages of RIBO particles detected only by DeePiCt (’tp new’), represent 80S ribosomes, consistent with maps of all cytosolic RIBO particles for both VPP and DEF datasets (<bold><xref ref-type=\"fig\" rid=\"fig2\" hwp:id=\"xref-fig-2-5\" hwp:rel-id=\"F2\">Fig. 2l-m</xref></bold>, blue and gray, respectively). 3D classifications performed in both data types for FAS demonstrated that the 3D CNN together with manual annotations were capable of recovering these challenging shell-like structures independent of the data acquisition type (<bold>Supplementary Note 2, <xref ref-type=\"fig\" rid=\"figs3\" hwp:id=\"xref-fig-18-1\" hwp:rel-id=\"F18\">Supplementary Fig. 3</xref></bold>).</p><p hwp:id=\"p-21\">In conclusion, using DeePiCt, we provide comprehensive annotations for large macromolecular complexes and cellular structures in <italic toggle=\"yes\">S. pombe</italic> cryo-electron tomograms acquired with VPP and DEF, which set the ground for benchmarking the method’s performance and for future developments in particle detection approaches.</p></sec><sec id=\"s2c\" hwp:id=\"sec-5\"><title hwp:id=\"title-9\">Performance analysis and hyperparameter tuning of the DeePiCt workflow</title><sec id=\"s2c1\" hwp:id=\"sec-6\"><title hwp:id=\"title-10\">Performance analysis of 2D CNN in VPP</title><p hwp:id=\"p-22\">For the 2D CNN, basic hyperparameter tuning had close to no effect and a fixed architecture produced sufficiently good segmentations. Hence, no extensive hyper-parameter tuning was performed. The 2D CNN’s performance was evaluated on 10 manually annotated ground-truth VPP tomograms in a 5-fold cross-validation (CV). For each CV fold, two tomograms were split off from the training data to evaluate model performance during training. Two binary segmentation tasks were evaluated: segmentation of all organelles, <italic toggle=\"yes\">i.e.</italic> all membrane-enclosed organelles and the nucleoplasm, and segmentation of the cytosol. The results show that the 2D CNN achieves high areas under the precision-recall curve (AUPRC, <bold>Supplementary Note 1</bold>), with a median AUPRC of 0.98 for cytosol and 0.92 for organelles (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-1\" hwp:rel-id=\"F3\">Fig. 3a</xref></bold>, <bold>Extended Data</bold> <xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-9\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref>).</p><fig id=\"fig3\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F3\" hwp:rev-id=\"xref-fig-3-1 xref-fig-3-2 xref-fig-3-3 xref-fig-3-4 xref-fig-3-5 xref-fig-3-6 xref-fig-3-7 xref-fig-3-8 xref-fig-3-9 xref-fig-3-10\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIG3</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F3</object-id><object-id pub-id-type=\"publisher-id\">fig3</object-id><label>Figure 3</label><caption hwp:id=\"caption-3\"><title hwp:id=\"title-11\">DeePiCt performance, cross-domain generalization and comparison with other methods.</title><p hwp:id=\"p-23\"><bold>a</bold>. 2D CNN performance results for organelle and cytosol segmentation under a 5-fold CV scheme, when training and testing datasets are both in the same domain (VPP). The median AUPRC scores are indicated (black dotted lines). <bold>b</bold>. Performance of DeePiCt for the same-domain setting. The RIBO localization, FAS localization, and membrane segmentation tasks are shown (left to right), under a 3-fold CV scheme. In each case, the corresponding architectures of the 3D CNN were optimized by hyperparameter tuning (<bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-2\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE6\" hwp:id=\"xref-fig-11-1\" hwp:rel-id=\"F11\">Extended Data Fig. 6</xref></bold>). The median F1 score for RIBOS and FAS, and a median voxel-F1 for membranes are indicated (black dotted lines). <bold>c</bold>. Same-domain NPC segmentation results using a 3-fold CV scheme. From left to right: performance in all DEF* tomograms, high quality DEF* tomograms, and lower quality DEF* tomograms. <bold>d</bold>-<bold>e</bold>. DeepFinder (<bold>d</bold>) and TM (<bold>e</bold>) particle localization results for RIBO and FAS localization. The median F1 values are indicated in each case (black dashed line), along with corresponding values of DeePiCt’s performance for comparison (red dashed line). <bold>d</bold>. DeepFinder results are based on the same 3-fold CV scheme as in <bold>b</bold>, by training a multiclass DeepFinder network that simultaneously segments FAS and RIBO. <bold>e</bold>. TM results are shown for the VPP tomograms. <bold>f</bold>-<bold>g</bold>. Results of the cross-domain generalization for the 2D CNN (<bold>f</bold>) and DeePiCt (<bold>g</bold>) (training on 10 VPP tomograms, testing on 10 DEF tomograms). Dashed lines and their numerical labels indicate median performance values either of the corresponding plot (black), or the corresponding experiment performed in the same domain with DeePiCt (red), or in the same domain with the 2D CNN (green).</p></caption><graphic xlink:href=\"488077v1_fig3\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-3\"/></fig><fig id=\"figE5\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F10\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE5</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F10</object-id><object-id pub-id-type=\"publisher-id\">figE5</object-id><label>Extended Data Fig. 5</label><caption hwp:id=\"caption-10\"><title hwp:id=\"title-56\">Effects of spectrum equalization on cross-validation and performance evaluation across domains of the 2D CNN.</title><p hwp:id=\"p-90\"><bold>a-f</bold>. Performance was evaluated for 2D networks in three different scenarios (training on VPP in cross-validation, cross-domain generalization from VPP to DEF, training on VPP and DEF in cross validation), for either organelles or cytoplasm, without (-EQ) and with (+EQ) spectrum equalization. Areas under the precision-recall curves (AUPRCs) were computed by randomly picking 1000 voxels from each tomogram in test set (for the two cross-validation scenarios) or from the DEF tomograms (for the cross-domain tasks), while restricting evaluation to z-slices with any positive label. AUPRCs were computed after 3D post-processing. The baselines are defined as the fraction of positive labels within those z-slices and averaged across test tomograms. <bold>g-h</bold>. Precision, recall and F1-score of organelle (<bold>g</bold>) and cytosol (<bold>h</bold>) segmentation depending on probability threshold used. Scores were computed on the cross-validation results for the spectrum-equalized VPP tomograms.</p></caption><graphic xlink:href=\"488077v1_figE5\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-10\"/></fig><fig id=\"figE6\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F11\" hwp:rev-id=\"xref-fig-11-1 xref-fig-11-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE6</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F11</object-id><object-id pub-id-type=\"publisher-id\">figE6</object-id><label>Extended Data Fig. 6</label><caption hwp:id=\"caption-11\"><title hwp:id=\"title-57\">Hyperparameter tuning and performance analysis of 3D CNN for RIBO, FAS, membranes and NPCs.</title><p hwp:id=\"p-91\"><bold>a</bold>-<bold>c</bold>. CV analysis for the performance of RIBO, FAS and membranes, respectively. In these experiments we follow an incremental search: from left to right, the starting subplot corresponds to the default configuration of the 3D CNN, D=2, BN=0, and (ED, DD)=(0,0), except for variable IF. The best value of IF (denoted by IF*) is fixed in the subsequent subplot to the right, where D is varied over 1, 2, and 3. The next subplot to the right compares the effect of not applying and applying the batch normalization layer (BN=0 and BN=1, respectively), under the best IF=IF* and D=D* from previous subplots. Finally, the right most subplot varies the encoder- and decoder-dropout parameters (ED, DD), with fixed best IF=IF*, D=D*, and BN=BN* from previous subplots. <bold>d</bold>-<bold>f</bold>. Results of hyperparameter exploration for NPC; in all DEF* tomograms (left), in the subset of high-quality DEF* tomograms (middle), and in the subset of lower quality DEF* tomograms (right). <bold>d</bold>. Shows the variations in IF, when D=2, BN=0, and (ED, DD) = (0,0); <bold>e</bold>. Variations in IF, when D=3, BN=0, and (ED, DD) = (0,0); <bold>f</bold>. Variations in IF, when D=3, BN=1, and (ED, DD) = (0,0). The summary of best hyperparameters’ combination is provided in <bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-8\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>.</p></caption><graphic xlink:href=\"488077v1_figE6\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-11\"/></fig><p hwp:id=\"p-24\">The 2D CNN showed an average false positive rate (FPR) of 0.054 for cytosol and 0.024 for organelle segmentation, with the post-processing step providing marginal improvements (reducing FPR to 0.049 for cytosol and 0.021 for organelles, <bold><xref ref-type=\"fig\" rid=\"figE2\" hwp:id=\"xref-fig-7-2\" hwp:rel-id=\"F7\">Extended Data Fig. 2</xref></bold>). In addition to the two segmentation tasks described here, the 2D CNN has the potential to segment individual organelle types when provided with sufficient amounts of training data (<bold>Extended Data Table 1</bold>). Operating in 2D rather than 3D results in only little memory requirement (&lt;1GB at the default batch size) and short training time (∼15 minutes to train on ten tomograms using a Nvidia V100 GPU; <bold>Online Methods</bold>).</p></sec><sec id=\"s2c2\" hwp:id=\"sec-7\"><title hwp:id=\"title-12\">Hyperparameter tuning for the 3D CNN</title><p hwp:id=\"p-25\">The 3D CNN is built on a flexible architecture for which we performed hyperparameter tuning. We characterized the effect of varying the adjustable hyperparameters (<bold>Online Methods</bold>) on the DeePiCt workflow using a CV approach for RIBO, FAS and membranes in the VPP dataset, and for NPC in the DEF* dataset.</p><p hwp:id=\"p-26\">In each case we combined the 3D CNN segmentation of the target structure with an appropriate <italic toggle=\"yes\">region mask</italic> (<bold><xref ref-type=\"fig\" rid=\"figs1\" hwp:id=\"xref-fig-16-2\" hwp:rel-id=\"F16\">Supplementary Fig. 1</xref></bold>, <bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-3\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>) and determined the 3D CNN hyperparameter combination that optimizes the task of localization or segmentation (<bold><xref ref-type=\"fig\" rid=\"figE6\" hwp:id=\"xref-fig-11-2\" hwp:rel-id=\"F11\">Extended Data Fig. 6</xref></bold>, <bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-4\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>). Our results show that the best hyperparameter selection for the 3D CNN is structure-dependent, and not only related to the size of the complex or to the receptive field of the network (a function of the depth parameter D), but also to the particle’s density, symmetry and abundance, among others. Therefore, the architecture flexibility of our implementation of the 3D CNN is essential for the accurate segmentation of a diverse set of biological structures. In particular, BN shortens the learning speed (and notably improves the performance results in the case of NPCs and FAS). Dropout layers and a number of data augmentation strategies (Gaussian noise, salt-and-pepper noise, rotations, elastic transformations, see <bold>Supplementary Note 1</bold>) did not improve performance.</p></sec><sec id=\"s2c3\" hwp:id=\"sec-8\"><title hwp:id=\"title-13\">Performance of DeePiCt in the same-domain setting</title><p hwp:id=\"p-27\">Once the 3D CNN architectures which optimized the tasks of particle localization (for FAS and RIBO) and structure segmentation (membranes and NPC) were determined, we analyzed the performance of the DeePiCt workflow in the same-domain setting (<italic toggle=\"yes\">i.e.</italic> training and testing in the same domain). The 3-fold CV tests in VPP (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-2\" hwp:rel-id=\"F3\">Fig. 3b</xref>, <xref ref-type=\"fig\" rid=\"figE7\" hwp:id=\"xref-fig-12-1\" hwp:rel-id=\"F12\">Extended Data Fig. 7a-c</xref></bold>), showed that it achieved a performance F1 score between 0.68-0.80 for RIBO (median F1 of 0.79), 0.21-0.64 for FAS (median F1 of 0.46) and 0.58-0.90 for membranes (median voxel-F1 of 0.71). The results in DEF* for NPC segmentation (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-3\" hwp:rel-id=\"F3\">Fig. 3c</xref></bold>) show that, even if the network is trained on the whole DEF* dataset (23% of which is high quality data, <bold>Online Methods</bold>), the performance of the method is clearly different in the high-versus lower-quality DEF* tomograms: with a median voxel-F1 of 0.47 in high-quality data, while a median voxel-F1 of 0.19 in lower-quality DEF* tomograms.</p><fig id=\"figE7\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F12\" hwp:rev-id=\"xref-fig-12-1 xref-fig-12-2 xref-fig-12-3\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE7</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F12</object-id><object-id pub-id-type=\"publisher-id\">figE7</object-id><label>Extended Data Fig. 7</label><caption hwp:id=\"caption-12\"><title hwp:id=\"title-58\">Effects of spectrum equalization and cytosol masking on DeePiCt performance.</title><p hwp:id=\"p-92\"><bold>a-c</bold> Performance of the DeePiCt pipeline within the VPP domain for RIBO, FAS and membranes, without (-EQ) and with (+EQ) spectrum equalization pre-processing. The F1 score distribution shows that although for RIBO and membranes the spectrum equalization does not have a clear positive effect, for FAS it brings better results with a shift in median F1 score from 0.25 to 0.46. <bold>d-f</bold>. The same networks were used to test performance of the DeePiCt pipeline across the VPP and DEF domains (training on VPP and testing in DEF) for RIBO, FAS and membranes without (-EQ) and with (+EQ) spectrum equalization pre-processing. The F1 score distribution shows that while for ribosomes the spectrum equalization does not bring benefits, for FAS and membrane prediction across domains, its use brings a positive median F1 shift from 0.03 to 0.15 and from 0.16 to 0.41, respectively. <bold>g-h</bold>. Effects of employing a segmentation of cytosol as a <italic toggle=\"yes\">region mask</italic> (+ mask) or not (-mask) during DeePiCt’s post-processing, for the localization of RIBO and FAS. The plots show the results under a 3-fold CV in VPP data, where the cytosol segmentation was obtained by a 2D CNN. <bold>g</bold>. RIBO localization does not show a significant improvement in F1 score when the cytosol masking is used. <bold>h</bold>. FAS results show a stronger difference in F1 score, with a shift of median F1 from 0.10 to 0.46.</p></caption><graphic xlink:href=\"488077v1_figE7\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-12\"/></fig></sec></sec><sec id=\"s2d\" hwp:id=\"sec-9\"><title hwp:id=\"title-14\">Comparison of DeePiCt to state-of-the-art particle localization tools</title><p hwp:id=\"p-28\">To benchmark DeePiCt in the particle localization task, we compared it to TM and DeepFinder in the tasks of RIBO and FAS in the VPP dataset (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-4\" hwp:rel-id=\"F3\">Fig. 3d,e</xref></bold>). Following the suggestions by the authors<sup><xref ref-type=\"bibr\" rid=\"c28\" hwp:id=\"xref-ref-28-1\" hwp:rel-id=\"ref-28\">28</xref></sup>, DeepFinder was trained in a multi-class fashion, where both particles were learnt simultaneously by a single network. For the training, we used the same 3-fold CV splitting of the VPP datasets as for DeePiCt. In RIBO localization, DeepFinder achieved a median F1 of 0.83, comparable to DeePiCt. For FAS localization, DeepFinder achieved a median F1 of 0.11 (significantly below DeePiCt, t-test p-value of 0.007; <bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-5\" hwp:rel-id=\"F3\">Fig. 3d</xref></bold>). It is worth mentioning that single-class DeepFinder networks tested for FAS localization completely failed by not localizing any particles. DeePiCt needs ∼17h for training vs ∼3h for DeepFinder, while predicting and post processing is equally fast for both (∼500 clusters/minute) (<bold>Online Methods</bold>). Furthermore, DeePiCt’s trained networks (models) for all structures mentioned in this work are open source and publicly available (Code and data availability).</p><p hwp:id=\"p-29\">Finally, the analysis of TM performance was done with the raw output with respect to the top 2000 peaks in the case of RIBO and to the top 1000 peaks in the case of FAS. TM also takes several hours (s. parameters in Online Methods) and our results confirm that TM has lower performance than both deep-learning methods for the localization of RIBO and FAS, completely failing at the latter (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-6\" hwp:rel-id=\"F3\">Fig. 3e</xref></bold>).</p></sec><sec id=\"s2e\" hwp:id=\"sec-10\"><title hwp:id=\"title-15\">Domain generalization of DeePiCt across image acquisition conditions</title><p hwp:id=\"p-30\">The DEF data is characterized by a lower SNR and lower contrast in the image, as compared to VPP (<bold>Supplementary Note 2</bold>). This makes the task of particle localization and structure segmentation more challenging, but has the advantage that the resulting subtomogram averages generally provide better resolutions<sup><xref ref-type=\"bibr\" rid=\"c43\" hwp:id=\"xref-ref-43-1\" hwp:rel-id=\"ref-43\">43</xref></sup>. The comprehensive ground truth construction in DEF enabled us to study the generalization potential of the DeePiCt workflow across image acquisition conditions, from VPP to DEF data. To that end, we predicted cytosol, organelles, membranes, FAS and RIBO in 10 DEF tomograms, with networks trained in the yeast VPP data (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-7\" hwp:rel-id=\"F3\">Fig. 3f-g</xref></bold>).</p><p hwp:id=\"p-31\">Compared to segmentation of VPP data, the 2D CNN shows slightly poorer performance on DEF data for cytosol (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-8\" hwp:rel-id=\"F3\">Fig. 3a,f</xref></bold>). By employing amplitude spectrum equalization for the 2D CNN, the domain generalization capabilities greatly improved for both cytosol (from initial median AUPRC=0.82 to median AUPRC=0.95) and organelles (from AUPRC=0.42 to 0.74) (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-9\" hwp:rel-id=\"F3\">Fig. 3f</xref></bold>, <bold>Extended Data</bold> <xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-10\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref>). For DeePiCt, the performance scores in this domain generalization setting also showed a decay with respect to the same domain setting: with a median F1 of 0.59 for RIBO, a median F1 of 0.15 for FAS, and a median voxel-F1 of 0.41 for membranes (<bold><xref ref-type=\"fig\" rid=\"fig3\" hwp:id=\"xref-fig-3-10\" hwp:rel-id=\"F3\">Fig. 3g</xref></bold>). Importantly, as in the case of the 2D CNN, the spectrum equalization filter in the preprocessing step improved the domain generalization results (<bold><xref ref-type=\"fig\" rid=\"figE7\" hwp:id=\"xref-fig-12-2\" hwp:rel-id=\"F12\">Extended Data Fig. 7d-f</xref></bold>). Furthermore, we confirmed that using the cytosol predictions as a <italic toggle=\"yes\">region mask</italic> to clean particle annotations also improved the performances (<bold><xref ref-type=\"fig\" rid=\"figE7\" hwp:id=\"xref-fig-12-3\" hwp:rel-id=\"F12\">Extended Data Fig. 7g-h</xref></bold>). An example of DeePict segmentation outcome in the cross-domain setting is shown in <bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-1\" hwp:rel-id=\"F4\">Fig. 4a-b</xref></bold> and visually resembles the ground truth annotation (<bold><xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-1\" hwp:rel-id=\"F13\">Extended Data Fig. 8a</xref></bold>). The DeePiCt predictions and the ground truth annotations enabled us to analyze the DEF tomograms in the same way as we had done for the VPP data, by downstream 3D structural classification and refinement (<xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-11\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref>).</p><fig id=\"figE8\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F13\" hwp:rev-id=\"xref-fig-13-1 xref-fig-13-2 xref-fig-13-3 xref-fig-13-4 xref-fig-13-5 xref-fig-13-6 xref-fig-13-7\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE8</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F13</object-id><object-id pub-id-type=\"publisher-id\">figE8</object-id><label>Extended Data Fig. 8</label><caption hwp:id=\"caption-13\"><title hwp:id=\"title-59\">Ground truth annotations and 3D refinement of ribosomes and FAS in DEF tomograms.</title><p hwp:id=\"p-93\"><bold>a</bold>. Complete annotation of ribosomes (yellow, orange, green, dark and light blue; sub-populations described in (<bold>f</bold>) and in Extended Data Figure 9), FAS (pink), membranes (purple) and organelles (gray) in the tomogram depicted in <xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-15\" hwp:rel-id=\"F4\">Fig. 4</xref>. Ribosome subclasses in the context of specific organelles were identified by a maximal distance of 25 nm to mitochondria (mito-bound, green) or ER (ER-bound, orange). Focused classification on densities close to the head of the 40S small ribosomal subunit or in proximity to the exit tunnel recovered ribosome subsets with head (dark blue) and exit tunnel density (cyan). <bold>b-e</bold>. FAS subtomogram averages (pink) fit of the <italic toggle=\"yes\">S. cerevisiae</italic> X-ray structure (<bold>b</bold>, cyan) and the single particle cryo-EM density of <italic toggle=\"yes\">P. pastoris</italic> (<bold>c</bold>, cyan) including the PPT domains. An extra density cannot be assigned. The cross section views close to the alpha-wheel reveal three densities fitting three and one ACPs of <italic toggle=\"yes\">S. cerevisiae</italic> and <italic toggle=\"yes\">P. pastoris</italic>, respectively (white asterisks, bottom). <bold>d</bold>. Overlay with FAS from VPP ground truth (gt) tomograms (gray) shows overall similarity, but the ACPs could not be resolved (white asterisks, bottom). <bold>e</bold>. FAS predicted by DeePiCt (gray) matches the DEF gt average with three ACP densities close to the alpha-wheel. <bold>f-h</bold>. Subtomogram average of all ribosomes (yellow) matches the DeePiCt-predicted (g, gray) and VPP-derived (h, gray) ribosome densities. <bold>i-k</bold>. Well-aligned ribosomes (yellow) detected by hierarchical 3D classification in RELION and refined in M (<xref ref-type=\"fig\" rid=\"figs3\" hwp:id=\"xref-fig-18-2\" hwp:rel-id=\"F18\">Supplementary Figure 3</xref>). <bold>j-k</bold>. Slicing through the average depicted in (<bold>i</bold>) at different axes reveals the PTC with a P-site tRNA and the L1 stalk facing the E-site.</p></caption><graphic xlink:href=\"488077v1_figE8\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-13\"/></fig><fig id=\"fig4\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F4\" hwp:rev-id=\"xref-fig-4-1 xref-fig-4-2 xref-fig-4-3 xref-fig-4-4 xref-fig-4-5 xref-fig-4-6 xref-fig-4-7 xref-fig-4-8 xref-fig-4-9 xref-fig-4-10 xref-fig-4-11 xref-fig-4-12 xref-fig-4-13 xref-fig-4-14 xref-fig-4-15\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIG4</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F4</object-id><object-id pub-id-type=\"publisher-id\">fig4</object-id><label>Figure 4</label><caption hwp:id=\"caption-4\"><title hwp:id=\"title-16\">DeePiCt enables exploration of macromolecular complexes in their cellular context.</title><p hwp:id=\"p-32\"><bold>a</bold>. 2D slice of a DEF <italic toggle=\"yes\">S. pombe</italic> cytoplasm tomogram. Mitochondrion (M), vesicle (V), the ER and the cell wall (CW) are detectable in the raw (bottom left) and with improved contrast after amplitude spectrum equalization (top right). <bold>b</bold>. DeePiCt predictions generated with models trained in VPP data. Organelles (gray), membranes (purple), FAS (pink, <bold>c</bold>, <bold>d</bold>), ribosomes (yellow, <bold>e</bold>-<bold>h</bold>) and subsets classified in RELION (head density, dark blue, <bold>i;</bold> exit tunnel density, bright blue, <bold>j)</bold>, and within 25 nm to mitochondria (mito-bound, green, <bold>m</bold>) and the ER (ER-bound, orange, <bold>n</bold>). <bold>c</bold>. FAS subtomogram average (pink) fits the <italic toggle=\"yes\">S. cerevisiae</italic> structure (cyan) including PPT domains. An extra density cannot be assigned. <bold>d</bold>. Cross-section of <bold>c</bold> close to the alpha-wheel with three densities fitting ACPs (white asterisks). <bold>e</bold>. Subtomogram average of all ribosomes from 10 DEF tomograms. <bold>f</bold>. Well-aligned ribosome subset detected by hierarchical 3D classification in RELION and refined in M. <bold>g, h</bold>. Slices through (<bold>f</bold>) reveal the PTC with a P-site tRNA and the L1 stalk facing the E-site. <bold>i</bold>. Ribosome subclass with additional densities (white arrowheads) close to the head of the small ribosomal subunit which fits eEF3 (red), and close to the ribosomal exit tunnel. <bold>j</bold>. Ribosomes classified for a density below the ribosomal exit tunnel (white arrowhead). The <italic toggle=\"yes\">S. cerevisiae</italic> ribosome with ES27L in a particular configuration (left, purple) connects to the additional exit-tunnel density which fits Arx1 bound to the 60S pre-ribosome (middle and right, purple). <bold>k</bold>, <bold>l</bold>. Different z slices of the tomogram in <bold>a</bold> (white dashed boxes) show ribosomes bound to the ER (<bold>k</bold>) and a mitochondrion (<bold>l</bold>). <bold>m</bold>. ER-bound ribosomes from 7 tomograms show a connection of the peptide exit tunnel of the large subunit to the membrane density. <bold>n</bold>. Mitochondria-bound ribosomes from 3 tomograms show a linker connecting the large subunit, at a site close to the small subunit, to the membrane density. Overlay of the ribosomes in <bold>m</bold> and <bold>n</bold> shows different interfaces with the respective organelle membranes.</p></caption><graphic xlink:href=\"488077v1_fig4\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-4\"/></fig><fig id=\"fig5\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F5\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIG5</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F5</object-id><object-id pub-id-type=\"publisher-id\">fig5</object-id><label>Figure 5</label><caption hwp:id=\"caption-5\"><title hwp:id=\"title-17\">DeePiCt’s generalization across species.</title><p hwp:id=\"p-33\">A dataset depicting a HeLa cell nuclear periphery1 is segmented by applying four independently trained DeePiCt networks. The results show the segmentation of actin filaments (red) trained on RPE-1 and MEF 3T3 tomograms, microtubules (MTs, cyan) trained on <italic toggle=\"yes\">C. elegans</italic> tomograms, and cytosolic ribosomes (yellow) and membranes (purple) trained on <italic toggle=\"yes\">S. pombe</italic> tomograms.</p></caption><graphic xlink:href=\"488077v1_fig5\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-5\"/></fig></sec><sec id=\"s2f\" hwp:id=\"sec-11\"><title hwp:id=\"title-18\">DeePiCt predictions result in high quality subtomogram averages</title><p hwp:id=\"p-34\">In the case of FAS in the DEF dataset, fewer particles were predicted than in the ground truth (<bold>Extended Data Tables 5, 6</bold>). Nevertheless, the subtomogram average of the detected particles revealed the two half domes of the barrel-shaped type I FAS with applied D3 symmetry, consistent with the ground truth average and published structures of this fungal 2.6-megadalton heterododecameric (α6β6) complex from other yeast species (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-2\" hwp:rel-id=\"F4\">Fig. 4c-d</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-2\" hwp:rel-id=\"F13\">Extended Data Fig. 8b-e</xref></bold>). The phosphopantetheine transferase (PPT) domains required for activation of FAS could be observed along the equatorial plane of the complex. In addition, three equatorial densities were resolved that could not be assigned based on published structures. Three densities inside each half dome connected to the central α wheel and in close proximity to the ketosynthase (KS) fit with the acyl carrier protein (ACP) of <italic toggle=\"yes\">Saccharomyces cerevisiae</italic><sup><xref ref-type=\"bibr\" rid=\"c44\" hwp:id=\"xref-ref-44-1\" hwp:rel-id=\"ref-44\">44</xref></sup> (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-3\" hwp:rel-id=\"F4\">Fig. 4d</xref></bold>, PDB: 2uv8) and <italic toggle=\"yes\">Pichia pastoris</italic><sup><xref ref-type=\"bibr\" rid=\"c45\" hwp:id=\"xref-ref-45-1\" hwp:rel-id=\"ref-45\">45</xref></sup> (<bold><xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-3\" hwp:rel-id=\"F13\">Extended Data Fig. 8c</xref></bold>, EMD-12139). The ACP shuttles the growing acyl-chain between the different catalytic sites, and its localization has been suggested in connection with the activity of the whole multi-enzyme complex<sup><xref ref-type=\"bibr\" rid=\"c44\" hwp:id=\"xref-ref-44-2\" hwp:rel-id=\"ref-44\">44</xref>, <xref ref-type=\"bibr\" rid=\"c46\" hwp:id=\"xref-ref-46-1\" hwp:rel-id=\"ref-46\">46</xref>–<xref ref-type=\"bibr\" rid=\"c49\" hwp:id=\"xref-ref-49-1\" hwp:rel-id=\"ref-49\">49</xref></sup>. Here, <italic toggle=\"yes\">S. pombe</italic> FAS complexes within FIB-lamellae prepared from exponentially growing cells revealed a specific ACP site, which had not been observed by cryo-ET and thus in its native state before (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-4\" hwp:rel-id=\"F4\">Fig. 4d</xref>, <xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-4\" hwp:rel-id=\"F13\">Extended Data Fig. 8</xref>, Supplementary Note 2</bold>).</p><p hwp:id=\"p-35\">For ribosomes, the particle averages and numbers purely derived from DeePiCt predictions are comparable to the ground truth annotations in DEF (<bold>Extended Data Tables 5, 6</bold>; <bold>Supplementary Note 2, <xref ref-type=\"fig\" rid=\"figs4\" hwp:id=\"xref-fig-19-2\" hwp:rel-id=\"F19\">Supplementary Fig. 4</xref>, <xref ref-type=\"fig\" rid=\"figs5\" hwp:id=\"xref-fig-20-1\" hwp:rel-id=\"F20\">5</xref></bold>) complementing the performance analysis results described above. 3D refinements of these lamella-derived ribosomal particles from both ground truth and DeePiCt resulted in subtomogram averages with the nominal resolutions of 11 Å and 15 Å after multi-particle refinement in M<sup><xref ref-type=\"bibr\" rid=\"c33\" hwp:id=\"xref-ref-33-2\" hwp:rel-id=\"ref-33\">33</xref></sup>, respectively (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-5\" hwp:rel-id=\"F4\">Fig. 4e</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-5\" hwp:rel-id=\"F13\">Extended Data Fig. 8f</xref>, Extended Data Tables 5, 6</bold>). Hierarchical 3D classification revealed in both datasets a well-aligned class that was further refined in M to subnanometer resolutions of 9.3 Å and 9.4 Å for ground truth and DeePiCt predictions, respectively (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-6\" hwp:rel-id=\"F4\">Fig. 4f-h</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-6\" hwp:rel-id=\"F13\">Extended Data Fig. 8i-k</xref>, Extended Data Tables 5, 6, Supplementary Note 2, <xref ref-type=\"fig\" rid=\"figs4\" hwp:id=\"xref-fig-19-3\" hwp:rel-id=\"F19\">Supplementary Fig. 4</xref>, <xref ref-type=\"fig\" rid=\"figs5\" hwp:id=\"xref-fig-20-2\" hwp:rel-id=\"F20\">5</xref></bold>). This allowed the identification of tRNA occupying the P-site of the peptidyl transferase center (PTC) as well as the L1 stalk facing the E-site (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-7\" hwp:rel-id=\"F4\">Fig. 4g-h</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE8\" hwp:id=\"xref-fig-13-7\" hwp:rel-id=\"F13\">Extended Data Fig. 8j-k</xref></bold>).</p><fig id=\"figE9\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F14\" hwp:rev-id=\"xref-fig-14-1 xref-fig-14-2 xref-fig-14-3 xref-fig-14-4 xref-fig-14-5 xref-fig-14-6\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE9</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F14</object-id><object-id pub-id-type=\"publisher-id\">figE9</object-id><label>Extended Data Fig. 9</label><caption hwp:id=\"caption-14\"><title hwp:id=\"title-60\">Comparison of ribosome subtomogram averages in DEF ground truth (gt).</title><p hwp:id=\"p-94\"><bold>a-e.</bold> Subtomogram average of ribosomes (blue) with electron density in proximity to the head of the 40S small subunit (white arrowhead) which fits the ribosome-bound eEF3<sup><xref ref-type=\"bibr\" rid=\"c6\" hwp:id=\"xref-ref-6-1\" hwp:rel-id=\"ref-6\">6</xref></sup> (CC 0.8972, red). <bold>b</bold>. Cross section of (<bold>a</bold>) showing the PTC with P- and E-site tRNAs and the L1 stalk facing inwards. <bold>c</bold>. Zoom into the additional head density fitting eEF3. <bold>d-e</bold>. Overlay with the same ribosome class from DeePiCt predictions in DEF (<bold>d</bold>, gray) and from VPP gt data (<bold>e</bold>, gray). <bold>f</bold>. Ribosome subset (cyan) with electron density in proximity to the exit tunnel (white arrowhead). <bold>g.</bold> Overlay with the S. cerevisiae ribosome<sup><xref ref-type=\"bibr\" rid=\"c7\" hwp:id=\"xref-ref-7-1\" hwp:rel-id=\"ref-7\">7</xref></sup> (CC 0.9651, purple) and a specific conformation of the expansion segment ES27L<sup><xref ref-type=\"bibr\" rid=\"c8\" hwp:id=\"xref-ref-8-2\" hwp:rel-id=\"ref-8\">8</xref></sup> (CC 0.7806, purple). <bold>h</bold>. Zoom into the exit tunnel area in (<bold>g</bold>) highlights the additional density connected to ES27L that colocalizes with for example the human Ebp1<sup><xref ref-type=\"bibr\" rid=\"c9\" hwp:id=\"xref-ref-9-2\" hwp:rel-id=\"ref-9\">9</xref></sup> (gray, lower panel, CC 0.8495, EMD- 1068). <bold>i-j</bold>. Overlay with the same ribosome class from DeePiCt predictions in DEF (<bold>i</bold>, gray) and from VPP gt data (<bold>j</bold>, gray). <bold>k-n</bold>. Overlay of ribosomes within 25 nm distance to the ER (orange) with ER-bound ribosomes from DeePiCt predictions in DEF (<bold>k</bold>, gray), from VPP gt data (<bold>l</bold>, gray), with the published density of <italic toggle=\"yes\">S. cerevisiae</italic> ribosomes derived from extracted ER<sup><xref ref-type=\"bibr\" rid=\"c10\" hwp:id=\"xref-ref-10-1\" hwp:rel-id=\"ref-10\">10</xref></sup> (<bold>k</bold>, gray, CC 0.8972, EMD-3764), and the ER-bound HeLa ribosome<sup><xref ref-type=\"bibr\" rid=\"c11\" hwp:id=\"xref-ref-11-1\" hwp:rel-id=\"ref-11\">11</xref></sup> (<bold>n</bold>, gray, CC 0.9105, EMD-8056). All densities show the same interface between ribosome and ER membrane. <bold>o-p</bold>. Overlay of ribosomes within 25 nm distance to mitochondria (green) with mito-bound ribosomes from DeePiCt predictions in DEF (<bold>o</bold>, gray), and VPP gt data (<bold>p</bold>, gray). All densities show the same interface between ribosome and mitochondria membrane. <bold>q-r</bold>. Comparisons between ER-bound (orange) and mitochondria-bound (mito- bound, green) subtomogram averages in DEF gt (<bold>q</bold>) and VPP gt (<bold>r</bold>) data reveal an angular offset of the membrane interfaces of 35° and 33°, respectively.</p></caption><graphic xlink:href=\"488077v1_figE9\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-14\"/></fig><fig id=\"figE10\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F15\" hwp:rev-id=\"xref-fig-15-1 xref-fig-15-2 xref-fig-15-3 xref-fig-15-4 xref-fig-15-5\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGE10</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F15</object-id><object-id pub-id-type=\"publisher-id\">figE10</object-id><label>Extended Data Fig. 10</label><caption hwp:id=\"caption-15\"><title hwp:id=\"title-61\">Domain generalization of different cellular structures and species.</title><p hwp:id=\"p-95\"><bold>a</bold>. Performance results of five DeePiCt networks trained on a variety of cell types for the segmentation of different cellular structures used for predictions in a HeLa cell dataset<sup><xref ref-type=\"bibr\" rid=\"c11\" hwp:id=\"xref-ref-11-2\" hwp:rel-id=\"ref-11\">11</xref></sup>. Left: plot of performance scores when employing the spectrum equalization (+EQ) filter in both the training and on the HeLa cell data. The results correspond to segmentations of actin filaments and microtubules (MT) (trained on 4 RPE1 tomograms; pink), RIBO (trained on 10 VPP <italic toggle=\"yes\">S. pombe</italic> tomograms; yellow-green), membranes (trained on 10 <italic toggle=\"yes\">S. pombe</italic> VPP tomograms; aquamarine), MT (trained on 11 <italic toggle=\"yes\">C. elegans</italic> tomograms; blue) and actin filaments (trained on 2 RPE1 and 3 MEF 3T3 tomograms with actin filaments at different orientations; light purple). Right: performance for RIBO and membrane without employing the equalization filter (-EQ) in the HeLa cell dataset. <bold>b</bold>. RPE1 training data examples (left and middle, tomograms on top, ground truth annotations at the bottom) and predictions (right) of the actin filament and MT models in the HeLa cell dataset. The training examples show 2 of the RPE1 tomograms<sup><xref ref-type=\"bibr\" rid=\"c12\" hwp:id=\"xref-ref-12-2\" hwp:rel-id=\"ref-12\">12</xref></sup> used for this model. The performance of actin and microtubule predictions in the HeLa cell dataset is poor (voxel-F1=0.09 for actin and 0.26 for MT; <bold>a</bold> left) despite the high-quality training data (left). This is likely due to the high unidirectional orientation of the filaments in the training data (bundles in stress fibers).</p></caption><graphic xlink:href=\"488077v1_figE10\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-15\"/></fig><fig id=\"figs1\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F16\" hwp:rev-id=\"xref-fig-16-1 xref-fig-16-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS1</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F16</object-id><object-id pub-id-type=\"publisher-id\">figs1</object-id><label>Supplementary Fig. 1</label><caption hwp:id=\"caption-19\"><title hwp:id=\"title-78\">3D CNN post-processing steps.</title><p hwp:id=\"p-142\">The clustered 3D CNN output (yellow) and a <italic toggle=\"yes\">region mask</italic> (white) are combined according to the three contact modes. Cluster centroids are represented by the red dots.</p></caption><graphic xlink:href=\"488077v1_figs1\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-23\"/></fig><fig id=\"figs2\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F17\" hwp:rev-id=\"xref-fig-17-1 xref-fig-17-2 xref-fig-17-3 xref-fig-17-4\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS2</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F17</object-id><object-id pub-id-type=\"publisher-id\">figs2</object-id><label>Supplementary Fig. 2</label><caption hwp:id=\"caption-20\"><title hwp:id=\"title-79\">Hierarchical 3D classification of cytosolic ribosomes in VPP ground truth.</title><p hwp:id=\"p-143\">2D slices through 3D class averages of iteration 25 are displayed. The highlighted class (cyan box) shows the 60S large ribosomal subunit class.</p></caption><graphic xlink:href=\"488077v1_figs2\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-24\"/></fig><fig id=\"figs3\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F18\" hwp:rev-id=\"xref-fig-18-1 xref-fig-18-2 xref-fig-18-3\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS3</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F18</object-id><object-id pub-id-type=\"publisher-id\">figs3</object-id><label>Supplementary Fig. 3</label><caption hwp:id=\"caption-21\"><title hwp:id=\"title-80\">3D classification of VPP and DEF ground truth, and DeePiCt prediction in DEF dataset for FAS.</title><p hwp:id=\"p-144\"><bold>a-c.</bold> 2D slices through 3D-refined subtomogram average of FAS using D3 symmetry recovered from VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. 3D classifications separated in all 3 datasets into two classes of which one is better structurally defined than the other. 3D refinements for DEF were performed in M.</p></caption><graphic xlink:href=\"488077v1_figs3\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-25\"/></fig><fig id=\"figs4\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F19\" hwp:rev-id=\"xref-fig-19-1 xref-fig-19-2 xref-fig-19-3 xref-fig-19-4\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS4</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F19</object-id><object-id pub-id-type=\"publisher-id\">figs4</object-id><label>Supplementary Fig. 4</label><caption hwp:id=\"caption-22\"><title hwp:id=\"title-81\">Hierarchical 3D classification of cytosolic ribosomes in DEF ground truth starting from M-refined alignments.</title><p hwp:id=\"p-145\">2D slices through 3D class averages of iteration 25 are displayed. The highlighted class (yellow box) was further refined in M, resulting in a well aligned class with a resolution of 9.3 Å.</p></caption><graphic xlink:href=\"488077v1_figs4\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-26\"/></fig><fig id=\"figs5\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F20\" hwp:rev-id=\"xref-fig-20-1 xref-fig-20-2 xref-fig-20-3\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS5</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F20</object-id><object-id pub-id-type=\"publisher-id\">figs5</object-id><label>Supplementary Fig. 5</label><caption hwp:id=\"caption-23\"><title hwp:id=\"title-82\">Hierarchical 3D classification of cytosolic ribosomes from DeePiCt predictions in DEF data starting from M-refined alignments.</title><p hwp:id=\"p-146\">2D slices through 3D class averages of iteration 25 are displayed. The highlighted class (yellow box) was further refined in M resulting in a well-aligned class with a resolution of 9.4 Å. The right side of the classification representing the vast majority of ribosomes does not converge into a well defined map.</p></caption><graphic xlink:href=\"488077v1_figs5\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-27\"/></fig></sec><sec id=\"s2g\" hwp:id=\"sec-12\"><title hwp:id=\"title-19\">DeePiCt-predicted ribosomes reveal functional sub-populations</title><p hwp:id=\"p-36\">The large number of particles that DeePiCt localized in DEF in a high throughput manner allows for examining subpopulations of macromolecular complexes that are functionally distinct by downstream subtomogram analysis. Further classification of all DeePiCt-predicted ribosomes focused on the head of the 40S small subunit revealed a subset with additional densities close to the head and at the exit tunnel (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-8\" hwp:rel-id=\"F4\">Fig. 4i</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figs6\" hwp:id=\"xref-fig-21-1\" hwp:rel-id=\"F21\">Supplementary Fig. 6</xref></bold>). This class was also detected in VPP and in DEF ground truth datasets (<bold>Extended Data Tables 4-6</bold>). In the DEF ground truth dataset, densities for P- and E-site tRNAs and the L1 stalk facing inwards are resolved (<bold><xref ref-type=\"fig\" rid=\"figE9\" hwp:id=\"xref-fig-14-1\" hwp:rel-id=\"F14\">Extended Data Fig. 9a-e</xref></bold>). The ribosome-bound ATPase eEF3 from <italic toggle=\"yes\">S. cerevisiae</italic><sup><xref ref-type=\"bibr\" rid=\"c50\" hwp:id=\"xref-ref-50-1\" hwp:rel-id=\"ref-50\">50</xref></sup> fitted well into the additional head density (CC 0.8972, EMD-12062, <xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-12\" hwp:rel-id=\"F1\"><bold>Fig. 5i</bold></xref>, <bold><xref ref-type=\"fig\" rid=\"figE9\" hwp:id=\"xref-fig-14-2\" hwp:rel-id=\"F14\">Extended Data Fig. 9 a-c</xref></bold>). During translation, this eukaryotic elongation factor facilitates binding of a new tRNA to the A-site of the ribosome via the ternary aminoacyl-tRNA–eEF1A–GTP complex<sup><xref ref-type=\"bibr\" rid=\"c50\" hwp:id=\"xref-ref-50-2\" hwp:rel-id=\"ref-50\">50</xref>, <xref ref-type=\"bibr\" rid=\"c51\" hwp:id=\"xref-ref-51-1\" hwp:rel-id=\"ref-51\">51</xref></sup>. In addition, unassigned density at the ribosomal exit tunnel is observable for this class. Focused classification of all DeePiCt-predicted ribosomes at this location revealed a subtomogram average fitting the <italic toggle=\"yes\">S. cerevisiae</italic> ribosome<sup><xref ref-type=\"bibr\" rid=\"c52\" hwp:id=\"xref-ref-52-1\" hwp:rel-id=\"ref-52\">52</xref></sup> (CC 0.9657, EMD-1667) with the rRNA expansion segment ES27L in a particular configuration<sup><xref ref-type=\"bibr\" rid=\"c53\" hwp:id=\"xref-ref-53-1\" hwp:rel-id=\"ref-53\">53</xref></sup> (CC 0.7938, PDB-3izd) connecting to an additional density close to the ribosomal exit tunnel (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-9\" hwp:rel-id=\"F4\">Fig. 4j</xref>, <xref ref-type=\"fig\" rid=\"figs7\" hwp:id=\"xref-fig-22-1\" hwp:rel-id=\"F22\">Supplementary Fig. 7</xref></bold>). ES27L plays a role in translation fidelity and recruits enzymes to the ribosomal exit tunnel, such as the methionine aminopeptidase (MetAP) that co-translationally processes the nascent peptide chain<sup><xref ref-type=\"bibr\" rid=\"c54\" hwp:id=\"xref-ref-54-1\" hwp:rel-id=\"ref-54\">54</xref>, <xref ref-type=\"bibr\" rid=\"c55\" hwp:id=\"xref-ref-55-1\" hwp:rel-id=\"ref-55\">55</xref></sup>. The nuclear export factor Arx1, which is released during cytosolic ribosomal 60S maturation in <italic toggle=\"yes\">S. cerevisiae</italic><sup><xref ref-type=\"bibr\" rid=\"c56\" hwp:id=\"xref-ref-56-1\" hwp:rel-id=\"ref-56\">56</xref></sup>, and its human homologue Ebp1, a translation regulator<sup><xref ref-type=\"bibr\" rid=\"c57\" hwp:id=\"xref-ref-57-1\" hwp:rel-id=\"ref-57\">57</xref>, <xref ref-type=\"bibr\" rid=\"c58\" hwp:id=\"xref-ref-58-1\" hwp:rel-id=\"ref-58\">58</xref></sup>, similarly bind at locations of the observed extra density (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-10\" hwp:rel-id=\"F4\">Fig. 4j</xref>, <xref ref-type=\"fig\" rid=\"figE9\" hwp:id=\"xref-fig-14-3\" hwp:rel-id=\"F14\">Extended Data Fig. 9h</xref></bold>). These binding factors recruit the flexible rRNA scaffold ES27L and cover the ribosomal exit tunnel with their MetAP-like folds. This structural class was also detected in DEF gt and VPP gt datasets (<bold><xref ref-type=\"fig\" rid=\"figE9\" hwp:id=\"xref-fig-14-4\" hwp:rel-id=\"F14\">Extended Data Fig. 9f-j</xref>, Extended Data Tables 4-5</bold>).</p><fig id=\"figs6\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F21\" hwp:rev-id=\"xref-fig-21-1 xref-fig-21-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS6</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F21</object-id><object-id pub-id-type=\"publisher-id\">figs6</object-id><label>Supplementary Fig. 6</label><caption hwp:id=\"caption-24\"><title hwp:id=\"title-83\">3D classifications and refinements of cryo-EM densities in proximity to the head of the 40S small ribosomal subunit in VPP gt, DEF gt, and DeePiCt predictions in DEF datasets.</title><p hwp:id=\"p-147\"><bold>a-c.</bold> 2D slices through 3D-refined subtomogram average of all cytosolic ribosomes from VPP ground truth (gt), DEF gt or DeePiCt predictions on DEF datasets. Focused 3D classifications (blue sphere mask displayed in a) separated in all 3 datasets one class (dark blue) with an additional density close to the head of the 40S small ribosomal subunit. 3D refinements for DEF were performed in M.</p></caption><graphic xlink:href=\"488077v1_figs6\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-28\"/></fig><fig id=\"figs7\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F22\" hwp:rev-id=\"xref-fig-22-1 xref-fig-22-2 xref-fig-22-3 xref-fig-22-4\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS7</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F22</object-id><object-id pub-id-type=\"publisher-id\">figs7</object-id><label>Supplementary Fig. 7</label><caption hwp:id=\"caption-25\"><title hwp:id=\"title-84\">3D classifications and refinements of ribosomes at the exit tunnel in VPP gt, DEF gt, and DeePiCt predictions in DEF datasets.</title><p hwp:id=\"p-148\"><bold>a-c.</bold> 2D slices through 3D-refined subtomogram average of all ribosomes from VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. Focused 3D classifications (cyan sphere mask displayed in a) separated in all 3 datasets one class (cyan) with an additional exit tunnel density. 3D refinements for DEF were performed in M.</p></caption><graphic xlink:href=\"488077v1_figs7\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-29\"/></fig><p hwp:id=\"p-37\">Thus, DeePiCt predictions combined with structural analysis revealed functional subpopulations of ribosomes. Mapping them back into the tomograms revealed their 3D spatial distributions in relation to other macromolecules such as FAS and organelles in the cellular context (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-11\" hwp:rel-id=\"F4\">Fig. 4b</xref></bold>).</p></sec><sec id=\"s2h\" hwp:id=\"sec-13\"><title hwp:id=\"title-20\">DeePiCt’s combination of organelle segmentation and high-throughput particle localization enables structural insights into ribosome-mitochondria association</title><p hwp:id=\"p-38\">One advantage of DeePiCt over state-of-the-art tools is that it allows studying particle populations within their cellular context, by recovering particles in close proximity to specific organelles selected from predictions by the 2D CNN. We recovered cytosolic ribosomes within 25 nm distance to predicted endoplasmic reticulum (ER) and mitochondria from 7 and 3 DEF tomograms, respectively. Using subtomogram analyses for both subsets, two classes could be separated, one with and one without a membrane density (<bold><xref ref-type=\"fig\" rid=\"figs8\" hwp:id=\"xref-fig-23-1\" hwp:rel-id=\"F23\">Supplementary Fig. 8</xref>-<xref ref-type=\"fig\" rid=\"figs9\" hwp:id=\"xref-fig-24-1\" hwp:rel-id=\"F24\">9</xref></bold>). In the first class, ribosomes associated with the membrane in a specific orientation, whereas they were randomly oriented in the second class. The latter class likely arises from the highly crowded nature of the <italic toggle=\"yes\">S. pombe</italic> cytoplasm (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-12\" hwp:rel-id=\"F4\">Fig. 4k-l</xref></bold>), which in the case of DeePiCt predictions in DEF data, is also affected by imperfect organelle segmentations (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-13\" hwp:rel-id=\"F4\">Fig. 4b</xref></bold>). ER-bound ribosomes faced the membrane with the ribosomal exit tunnel in agreement with published structures from other species such as <italic toggle=\"yes\">S. cerevisiae</italic><sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-2\" hwp:rel-id=\"ref-1\">1</xref>, <xref ref-type=\"bibr\" rid=\"c52\" hwp:id=\"xref-ref-52-2\" hwp:rel-id=\"ref-52\">52</xref>, <xref ref-type=\"bibr\" rid=\"c59\" hwp:id=\"xref-ref-59-1\" hwp:rel-id=\"ref-59\">59</xref></sup> (<bold><xref ref-type=\"fig\" rid=\"figE9\" hwp:id=\"xref-fig-14-5\" hwp:rel-id=\"F14\">Extended Data Fig. 9 k-n</xref>, Extended Data Table 6</bold>). Mitochondria-bound ribosomes, which posed a particular challenge to structural analysis in previous studies<sup><xref ref-type=\"bibr\" rid=\"c59\" hwp:id=\"xref-ref-59-2\" hwp:rel-id=\"ref-59\">59</xref>, <xref ref-type=\"bibr\" rid=\"c60\" hwp:id=\"xref-ref-60-1\" hwp:rel-id=\"ref-60\">60</xref></sup>, were found to interact with the membrane at an angular offset of around 35° in comparison to the ER-bound ribosomes (<bold><xref ref-type=\"fig\" rid=\"fig4\" hwp:id=\"xref-fig-4-14\" hwp:rel-id=\"F4\">Fig. 4m-n</xref></bold>, <bold><xref ref-type=\"fig\" rid=\"figE9\" hwp:id=\"xref-fig-14-6\" hwp:rel-id=\"F14\">Extended Data Fig. 9o-r</xref>, Extended Data Table 6</bold>). Interestingly, a density connecting the ribosome to the mitochondrial membrane was detected at the bottom of the large subunit in close proximity to the small subunit, but not to the ribosomal exit tunnel. It possibly represents the ribosome nascent chain complex (NAC) in contact with the mitochondrion receptor OM14<sup><xref ref-type=\"bibr\" rid=\"c61\" hwp:id=\"xref-ref-61-1\" hwp:rel-id=\"ref-61\">61</xref>–<xref ref-type=\"bibr\" rid=\"c63\" hwp:id=\"xref-ref-63-1\" hwp:rel-id=\"ref-63\">63</xref></sup> which has yet to be structurally described.</p><fig id=\"figs8\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F23\" hwp:rev-id=\"xref-fig-23-1 xref-fig-23-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS8</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F23</object-id><object-id pub-id-type=\"publisher-id\">figs8</object-id><label>Supplementary Fig. 8</label><caption hwp:id=\"caption-26\"><title hwp:id=\"title-85\">3D classifications and refinements of ER-bound ribosomes in VPP gt, DEF gt, and DeePiCt predictions in DEF datasets.</title><p hwp:id=\"p-149\"><bold>a-c.</bold> 2D slices through 3D-refined subtomogram average of ribosomes in 25 nm distance to the ER recovered from VPP ground truth (gt), DEF gt or DeePiCt predictions on DEF datasets. Focused 3D classifications (cyan sphere mask displayed in <xref ref-type=\"fig\" rid=\"figs7\" hwp:id=\"xref-fig-22-3\" hwp:rel-id=\"F22\">Supplementary Fig. 7a</xref>) separated in all 3 datasets one class (orange) with membrane density. 3D refinements for DEF were performed in M.</p></caption><graphic xlink:href=\"488077v1_figs8\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-30\"/></fig><fig id=\"figs9\" position=\"float\" orientation=\"portrait\" fig-type=\"figure\" hwp:id=\"F24\" hwp:rev-id=\"xref-fig-24-1 xref-fig-24-2 xref-fig-24-3\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/FIGS9</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">F24</object-id><object-id pub-id-type=\"publisher-id\">figs9</object-id><label>Supplementary Fig. 9</label><caption hwp:id=\"caption-27\"><title hwp:id=\"title-86\">3D classifications and refinements of mitochondria-bound ribosomes in VPP gt, DEF gt, and DeePiCt predictions in DEF datasets.</title><p hwp:id=\"p-150\"><bold>a-c.</bold> 2D slices through 3D-refined subtomogram average of ribosomes in 25 nm distance to the mitochondria recovered from VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. Focused 3D classifications (cyan sphere mask displayed in <xref ref-type=\"fig\" rid=\"figs7\" hwp:id=\"xref-fig-22-4\" hwp:rel-id=\"F22\">Supplementary Fig. 7a</xref>) separated in all 3 datasets one class (green) with membrane density. 3D refinements for DEF were performed in M.</p></caption><graphic xlink:href=\"488077v1_figs9\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-31\"/></fig><p hwp:id=\"p-39\">In summary, ribosomes close to mitochondria and ER exhibit different interfaces with the respective membranes, facilitating specific protein nascent chain membrane insertion or transfer into the organelle<sup><xref ref-type=\"bibr\" rid=\"c2\" hwp:id=\"xref-ref-2-1\" hwp:rel-id=\"ref-2\">2</xref>, <xref ref-type=\"bibr\" rid=\"c52\" hwp:id=\"xref-ref-52-3\" hwp:rel-id=\"ref-52\">52</xref>, <xref ref-type=\"bibr\" rid=\"c62\" hwp:id=\"xref-ref-62-1\" hwp:rel-id=\"ref-62\">62</xref></sup>. These results highlight the power of DeePiCt for high-throughput particle localization and cryo-electron tomogram segmentation to rapidly gain new biological insights.</p></sec><sec id=\"s2i\" hwp:id=\"sec-14\"><title hwp:id=\"title-21\">Trained networks can be readily applied to other species</title><p hwp:id=\"p-40\">As a demonstration of the domain generalization potential of our workflow across species, we predicted segmentations of ribosomes, membranes and cytoskeleton structures (actin filaments and microtubules) in a published VPP HeLa cell dataset<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-3\" hwp:rel-id=\"ref-1\">1</xref></sup> (<xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-13\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref>). The application of the DeePiCt pipeline employed a manually generated mask using Amira<sup><xref ref-type=\"bibr\" rid=\"c64\" hwp:id=\"xref-ref-64-1\" hwp:rel-id=\"ref-64\">64</xref></sup> of the cytoplasmic volume, excluding the nucleus. For the evaluation of our results, we computed the voxel-F1 score in comparison to the publicly available annotations (EMD-11992<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-4\" hwp:rel-id=\"ref-1\">1</xref></sup>). For ribosomes and membranes, we used the ribosome- and membrane-networks trained on the 10 VPP-tomograms from <italic toggle=\"yes\">S. pombe</italic> described in previous sections. This resulted in a voxel-F1 of 0.55 for ribosomes and a voxel-F1 of 0.18 for membranes (<bold><xref ref-type=\"fig\" rid=\"figE10\" hwp:id=\"xref-fig-15-1\" hwp:rel-id=\"F15\">Extended Data Fig. 10a</xref></bold>). However, the available expert membrane segmentation for the dataset covered only ER membranes, which makes the voxel-F1 value less meaningful. Visual inspection of the predicted membrane segmentation revealed a good fit (<xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-14\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref>).</p><p hwp:id=\"p-41\">Additionally we generated models for microtubules prediction, initially training a network for simultaneous actin and microtubule segmentations in 4 tomograms of cryo-FIB milled Human retinal pigment epithelial-1 (RPE-1) cells<sup><xref ref-type=\"bibr\" rid=\"c65\" hwp:id=\"xref-ref-65-1\" hwp:rel-id=\"ref-65\">65</xref></sup>. Due to the high preferential orientation of the cytoskeletal filaments in this data exhibiting stress fibers, the performance for microtubules’ segmentation was low resulting in a voxel-F1 score of 0.26 (<bold><xref ref-type=\"fig\" rid=\"figE10\" hwp:id=\"xref-fig-15-2\" hwp:rel-id=\"F15\">Extended Data Fig. 10a</xref></bold>). We therefore trained a second 3D CNN for microtubule segmentation on 11 tomograms from lamella of cryo-FIB-milled, dissociated <italic toggle=\"yes\">C. elegans</italic> cells, containing 890 microtubules in a wide range of orientations. This new microtubule segmentation network achieved a voxel-F1 score of 0.83 (<bold><xref ref-type=\"fig\" rid=\"figE10\" hwp:id=\"xref-fig-15-3\" hwp:rel-id=\"F15\">Extended Data Fig. 10a</xref></bold>), exemplifying the importance of a training set with a high number of structures, as well as orientations to mitigate the effect of the missing wedge on the prediction performance.</p><p hwp:id=\"p-42\">Finally, we trained a dedicated network for predicting actin-filaments on 5 manually curated tomograms, 2 from RPE-1 cells and 3 from Mouse embryonic fibroblasts (MEF) 3T3 cells, that contained approximately 3,740 actin filaments (<bold>Online Methods</bold>). The actin filaments network shows a low voxel-F1 score of 0.10 in the HeLa cell tomogram (<xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-15\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref>, <bold><xref ref-type=\"fig\" rid=\"figE10\" hwp:id=\"xref-fig-15-4\" hwp:rel-id=\"F15\">Extended Data Fig. 10a</xref></bold>), which is likely caused by the finer structure of actin in comparison to microtubules, the minimal set of orientations sampled in the small amount of training data, and to the fact that most training filaments are arranged in bundles, probably causing the CNN to learn this superstructure rather than individual filaments (<bold><xref ref-type=\"fig\" rid=\"figE10\" hwp:id=\"xref-fig-15-5\" hwp:rel-id=\"F15\">Extended Data Fig. 10b</xref></bold>).</p><p hwp:id=\"p-43\">Overall, these results show that the prediction of ribosomes and microtubules, representing large macromolecular complexes, is especially well-preserved across species and that the availability of diverse and well-annotated training datasets are crucial for good performance of DeePiCt. The results shown in <xref rid=\"fig1\" ref-type=\"fig\" hwp:id=\"xref-fig-1-16\" hwp:rel-id=\"F1\"><bold>Fig. 5</bold></xref> highlight the use of trained networks from DeePiCt to segment novel datasets spanning different species.</p></sec></sec><sec id=\"s3\" hwp:id=\"sec-15\"><title hwp:id=\"title-22\">Discussion</title><p hwp:id=\"p-44\">Our DeePiCt workflow facilitates accurate and fast localization of diverse structures in cryo-ET of intact cells. The demonstrated high performance and the flexibility of the 3D CNN architecture offer the user a reliable tool for pattern recognition. This enabled us to detect particle species, such as FAS, that are low in abundance and have a less dense structural signature in comparison to ribosomes. The integration of structure segmentation (predicted by the 3D CNN) with the contextual information excludes false positives in the 3D localization of particles and structures segmentation task, and harnesses the cellular context to carry out spatial studies focused on regions of biological interest. This enabled us to investigate the 3D spatial distribution of macromolecules such as ribosomes in proximity to specific organelles (<italic toggle=\"yes\">e.g.</italic> mitochondria) and to obtain structural insights with functional implications.</p><p hwp:id=\"p-45\">Here, we provide the first publicly available experimental cryo-ET dataset of 20 <italic toggle=\"yes\">S. pombe</italic> tomograms under two microscopy acquisition settings (VPP and DEF), together with a high-quality comprehensive annotation of RIBO and FAS, membranes, organelle and cytosol segmentations. This constitutes the first gold-standard dataset in the field that is large enough for unbiased model training, which will enable benchmarking of current methods, and spur the development of future computational tools for unbiased data mining in cryo-ET data.</p><p hwp:id=\"p-46\">Subtomogram averaging of the annotated particles from either ground truth or DeePiCt predictions resulted in the first density maps of the <italic toggle=\"yes\">S. pombe</italic> ribosome and fatty acid synthase. We observed differences in the analysis of subtomograms derived from either VPP and DEF tomograms, despite both being derived from wild-type <italic toggle=\"yes\">S. pombe</italic> cryo-FIB lamellae (<bold>Supplementary Note 2</bold>). Due to the preservation of high frequency features in DEF data, a subset of well-aligned ribosomes could be identified and refined to subnanometer resolution densities from cryo-FIB lamellae. The obtained maps provide detailed structural insights; for example, the P-site tRNA occupying the PTC, which together with the elongation factor eEF3 detected close to the head of the 40S subunit in a subset of ribosomes, suggest active translation. Further classifications close to the ribosomal exit tunnel revealed ES27L in a specific configuration providing a platform for other factors to interact with the exit tunnel and a potentially protruding nascent chain. By utilizing the contextual information of the DeePiCt predictions, differences in relative membrane orientation of mitochondria- and ER-bound <italic toggle=\"yes\">S. pombe</italic> ribosomes were detected. The observed connecting density between ribosomes and mitochondria likely highlights the not yet structurally characterized ribosome-bound NAC in contact with OM-14<sup><xref ref-type=\"bibr\" rid=\"c61\" hwp:id=\"xref-ref-61-2\" hwp:rel-id=\"ref-61\">61</xref>–<xref ref-type=\"bibr\" rid=\"c63\" hwp:id=\"xref-ref-63-2\" hwp:rel-id=\"ref-63\">63</xref></sup> facilitating protein import into mitochondria<sup><xref ref-type=\"bibr\" rid=\"c60\" hwp:id=\"xref-ref-60-2\" hwp:rel-id=\"ref-60\">60</xref>, <xref ref-type=\"bibr\" rid=\"c61\" hwp:id=\"xref-ref-61-3\" hwp:rel-id=\"ref-61\">61</xref></sup>.</p><p hwp:id=\"p-47\">The analysis of DeePiCt’s performance confirmed that data quality is an important factor in its predictive ability, as demonstrated in the case of NPC predictions in the DEF* dataset, which we classified into high and lower quality subsets based on lamella thickness and tilt-series alignment error. As the NPC has a high degree of structural flexibility on the subunit and diameter level inside cells<sup><xref ref-type=\"bibr\" rid=\"c40\" hwp:id=\"xref-ref-40-2\" hwp:rel-id=\"ref-40\">40</xref></sup> it is a more challenging target than for example ribosomes and thus emphasizes the importance of higher quality data to achieve sufficient performance. The introduction of a pre-processing equalizing filter is also an important element of the pipeline that improves both the learning process during the training of 3D segmentation networks for particles with less dense print than the ribosome (<italic toggle=\"yes\">e.g.</italic> FAS), and especially the generalization power across domains, including different microscopy acquisition conditions. Thus, amplitude spectrum equalization allows localizing macromolecules across different data acquisition types. In the case of the 2D network, although pre-processing did not improve model performance on the same-domain inference, it led to cross-domain improvements when training on VPP data and inferring on DEF data, or when training on both data types combined to segment organelles and cytosol. More elaborate tasks, such as prediction of individual organelle types, will likely require more training data or training of a dedicated 3D CNN with a tailored network architecture.</p><p hwp:id=\"p-48\">Our workflow allows easy adaptation to the segmentations of other structures, as demonstrated by the application of cytoskeleton segmentation networks. The networks show high-quality performance for microtubules in the HeLa cell dataset after training on data with broad orientation sampling of the filaments. Actin predictions revealed a low F1 score and therefore likely require more training data sampling different orientations of the structural features to improve performance. Altogether, the application of multiple segmentation networks to the HeLa cell dataset revealed that DeePiCt trained on datasets from different microscopes, species, and conditions lead to reasonably good results when the quality of data is high. Although more in-depth analysis is needed to study the specific limitations for the applicability of DeePiCt networks on other datasets, the results presented here constitute the first step towards conducting large-scale quantitative analyses for structural biology using cryo-ET on cells from different laboratories and open source datasets. In this sense, the generated ground truth annotations in this study are a starting point to provide the community with a resource to improve and further develop cryo-ET object segmentation and detection tools, which will ultimately enable further exploration of particles in their cellular context. Together with the trained networks and their demonstrated applicability to other species, as well as the flexibility of the DeePiCt workflow, the software harbors great potential for quantitative cryo-ET studies in the future.</p></sec><sec sec-type=\"supplementary-material\" hwp:id=\"sec-16\"><title hwp:id=\"title-23\">Supporting information</title><supplementary-material position=\"float\" orientation=\"portrait\" hwp:id=\"DC1\"><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">DC1</object-id><label>Supplementary Data Tables</label><media xlink:href=\"supplements/488077_file04.xlsx\" position=\"float\" orientation=\"portrait\" hwp:id=\"media-1\"/></supplementary-material></sec></body><back><ack hwp:id=\"ack-1\"><title hwp:id=\"title-24\">Acknowledgments</title><p hwp:id=\"p-49\">We thank Jenia Zagoriy and Wim Hagen for technical support, EMBL IT and especially Thomas Hoffman for computational support. I.D.T. and M.T.N were supported by a fellowship from the EMBL Interdisciplinary Postdoctoral Program (EI3POD) under Marie Skłodowska-Curie Actions COFUND. J.M. acknowledges funding from the EMBL and the European Research Council starting grant (3DCellPhase-760067).</p></ack><sec id=\"s4\" hwp:id=\"sec-17\"><title hwp:id=\"title-25\">Author contributions</title><p hwp:id=\"p-50\">I.D.T., J.M. and J.Z. conceived the study. I.D.T. implemented the 3D CNN code, co-produced ground truth annotations, carried out numerical experiments to evaluate performance. S.K.G. prepared <italic toggle=\"yes\">S. pombe</italic> cryo-FIB lamellae and acquired cryo-electron tomograms, co-produced ground truth annotations, carried out subtomogram averaging and structural analysis. A.M. implemented the 2D CNN and spectrum equalization filter, produced ground truth segmentations and conducted performance analysis of the 2D CNN. F.S. carried out training and prediction with 3D CNN networks, implemented code and created Colab Notebooks for prediction. C.Z. generated data and provided NPC annotations under the supervision of M.B. M.T.N. generated RPE1 and MEF 3T3 data and together with D.W.C.C. produced ground truth and trained networks for actin segmentation. F.T. generated data, ground truth and trained networks for microtubule segmentation. C.P. and A.K. provided valuable input for the design and generation of the CNNs. I.D.T., S.K.G., A.M., J.M. and J.Z. wrote the manuscript with input from all authors.</p></sec><sec id=\"s5\" hwp:id=\"sec-18\"><title hwp:id=\"title-26\">Code and data availability</title><p hwp:id=\"p-51\">The DeePiCt code is assembled as two Snakemake pipelines (2D CNN and 3D CNN); the 3D CNN is implemented in the Pytorch framework while the 2D CNN is implemented in Keras, both in Python 3. The code, trained models, and link to the Google Colab notebook are available in the github repository <ext-link l:rel=\"related\" l:ref-type=\"uri\" l:ref=\"https://github.com/ZauggGroup/DeePiCt\" ext-link-type=\"uri\" xlink:href=\"https://github.com/ZauggGroup/DeePiCt\" hwp:id=\"ext-link-3\">https://github.com/ZauggGroup/DeePiCt</ext-link>.</p><p hwp:id=\"p-52\">Raw tilt series, tomograms, ground truth coordinates and segmentations are available via EMPIAR accession codes EMPIAR-10988 (<italic toggle=\"yes\">S. pombe</italic>) and EMPIAR-10989 (RPE1). Subtomogram averages for <italic toggle=\"yes\">S. pombe</italic> VPP and DEF ground truth annotations are available on EMDB (VPP gt: EMD-14404, EMD-14405, EMD-14406, EMD-14408, EMD-14409, EMD-14410, EMD-14411; DEF gt: EMD-14412, EMD-14413, EMD-14415, EMD-14417, EMD-14418, EMD-14419, EMD-14420; DEF DeePiCt predicted: EMD-14422, EMD-14423, EMD-14424, EMD-14425, EMD-14426).</p><p hwp:id=\"p-53\">Structural comparisons were performed with <italic toggle=\"yes\">S. cerevisiae</italic> FAS (PDB: 2uv8<sup><xref ref-type=\"bibr\" rid=\"c44\" hwp:id=\"xref-ref-44-3\" hwp:rel-id=\"ref-44\">44</xref></sup>), <italic toggle=\"yes\">P. pastoris</italic> FAS (EMD-12139<sup><xref ref-type=\"bibr\" rid=\"c45\" hwp:id=\"xref-ref-45-2\" hwp:rel-id=\"ref-45\">45</xref></sup>), eEF3 from <italic toggle=\"yes\">S. cerevisiae</italic> (EMD-12062<sup><xref ref-type=\"bibr\" rid=\"c50\" hwp:id=\"xref-ref-50-3\" hwp:rel-id=\"ref-50\">50</xref></sup>), the <italic toggle=\"yes\">S. cerevisiae</italic> ribosome (EMD-1667<sup><xref ref-type=\"bibr\" rid=\"c52\" hwp:id=\"xref-ref-52-4\" hwp:rel-id=\"ref-52\">52</xref></sup>) with the rRNA expansion segment ES27L in a particular configuration (PDB-3izd<sup><xref ref-type=\"bibr\" rid=\"c53\" hwp:id=\"xref-ref-53-2\" hwp:rel-id=\"ref-53\">53</xref></sup>), the nuclear export factor Arx1 bound to the 60S large ribosomal subunit <italic toggle=\"yes\">S. cerevisiae</italic> (EMD-2169<sup><xref ref-type=\"bibr\" rid=\"c56\" hwp:id=\"xref-ref-56-2\" hwp:rel-id=\"ref-56\">56</xref></sup>), the human Ebp1 (EMD-1068<sup><xref ref-type=\"bibr\" rid=\"c57\" hwp:id=\"xref-ref-57-2\" hwp:rel-id=\"ref-57\">57</xref></sup>), <italic toggle=\"yes\">S. cerevisiae</italic> ribosomes derived from extracted ER (EMD-3764<sup><xref ref-type=\"bibr\" rid=\"c59\" hwp:id=\"xref-ref-59-3\" hwp:rel-id=\"ref-59\">59</xref></sup>), and the ER-bound HeLa ribosomes (EMD-8056<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-5\" hwp:rel-id=\"ref-1\">1</xref></sup>). The large subunit (LSU, 60S) of a published <italic toggle=\"yes\">S. cerevisiae</italic> 80S ribosome map (EMD-3228<sup><xref ref-type=\"bibr\" rid=\"c66\" hwp:id=\"xref-ref-66-1\" hwp:rel-id=\"ref-66\">66</xref></sup>) and the <italic toggle=\"yes\">S. cerevisiae</italic> FAS map (EMD-1623<sup><xref ref-type=\"bibr\" rid=\"c46\" hwp:id=\"xref-ref-46-2\" hwp:rel-id=\"ref-46\">46</xref></sup>) were used as references for TM in <italic toggle=\"yes\">S. pombe</italic> tomograms. The HeLa cell dataset is available via accession code EMD-11992<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-6\" hwp:rel-id=\"ref-1\">1</xref></sup>.</p></sec><sec id=\"s6\" hwp:id=\"sec-19\"><title hwp:id=\"title-27\">Competing interests</title><p hwp:id=\"p-54\">The authors declare no competing interests.</p></sec><sec id=\"s7\" hwp:id=\"sec-20\"><title hwp:id=\"title-28\">Additional information</title><p hwp:id=\"p-55\">Supplementary Information is available for this paper.</p></sec><sec id=\"s8\" hwp:id=\"sec-21\"><title hwp:id=\"title-29\">Online Methods</title><sec id=\"s8a\" hwp:id=\"sec-22\"><title hwp:id=\"title-30\">Yeast cell culture</title><p hwp:id=\"p-56\"><italic toggle=\"yes\">S. pombe</italic> K972 <italic toggle=\"yes\">Sp h-</italic> wt haploid was kindly provided by C. Haering, originally from P. Nurse. Cells were recovered from frozen stock by streaking on YE5S agar plates (YES Broth, Formedium, 20 g agarose/L) and incubated at 30 °C for 1-3 days. Colonies were re-streaked on fresh YES agar plates and incubated 1-3 days at 30 °C. Single colonies were inoculated in 5 mL of YES medium (YES Broth, Formedium, PCM0302, FM0618/8573) and grown at 30 °C, 170 rpm overnight (NCU-Shaker mini, Benchmark). On the next day, cultures were grown to their log phase at OD600 of 0.5 - 0.6 and diluted beforehand in YES if necessary.</p></sec><sec id=\"s8b\" hwp:id=\"sec-23\"><title hwp:id=\"title-31\">Vitrification of yeast cells</title><p hwp:id=\"p-57\">A Leica EM GP (Leica Microsystems) was utilized to vitrify yeast cells at liquid nitrogen temperature. Yeast cells were either diluted to OD600 of 0.2-0.4 in YES medium or, following a prior wash step, in PBS containing 5 % or 10 % BSA as cryoprotectant. In the chamber, 4 µl were directly applied to TEM grids (Quantifoil R1/2, Cu 200 mesh, holey carbon or SiO2 film), which were glow discharged on both sides for 45 s (Pelco Easy glow) immediately before usage. Blotting from the back side of the support was performed for 1-2 s at 22 °C and 99% humidity. Cells were plunge-frozen in liquid ethane cooled by liquid nitrogen and transferred into grid boxes until further usage.</p></sec><sec id=\"s8c\" hwp:id=\"sec-24\"><title hwp:id=\"title-32\">Cryo-FIB of <italic toggle=\"yes\">S. pombe</italic></title><p hwp:id=\"p-58\">TEM grids containing vitrified yeast cells were clipped into an autogrid with a cut-out enabling cryo-focused ion beam (cryo-FIB) milling at shallow angles<sup><xref ref-type=\"bibr\" rid=\"c67\" hwp:id=\"xref-ref-67-1\" hwp:rel-id=\"ref-67\">67</xref></sup>. Mounted on a 45° pre-tilt shuttle, grids were transferred into an Aquilos Dual beam microscope (ThermoFisher Scientific). Prior to cryo-FIB milling at liquid nitrogen temperature, cells were sputter-coated with platinum for 10-15 s (1 kV, 10 mA, 10 Pa). Subsequently, a layer of organometallic platinum (reservoir at 28°C) was applied by opening the gas injection system (GIS) for 8 sec at a stage Z position of 3-4 mm below the coincidence point. In three independent sessions, three grids with 5 lamellae each were prepared at a milling angle of 15°. Agglomerations of several cells were micromachined in three steps of rough milling to a thickness of 5 µm at 1 nA ion beam current, 3 µm at 0.5 nA and 1 µm at 0.1 nA. Milling progress was visually monitored between each milling step with the scanning electron microscope beam (SEM, 10 kV, 50 pA). Fine milling was performed at 50 pA to a target thickness of 200 nm. To render the lamellae conductive for TEM imaging, they were sputtered with platinum for 5 s (1 kV, 10 mA, 10 Pa) and transferred into cryo boxes.</p></sec><sec id=\"s8d\" hwp:id=\"sec-25\"><title hwp:id=\"title-33\">Cryo-electron tomography of <italic toggle=\"yes\">S. pombe</italic></title><p hwp:id=\"p-59\">Autogrids containing lamellae were loaded into a Titan Krios (Thermo Fisher Scientific) such that the axis of the pre-tilt introduced by FIB milling was aligned perpendicular to the tilt axis of the microscope<sup><xref ref-type=\"bibr\" rid=\"c68\" hwp:id=\"xref-ref-68-1\" hwp:rel-id=\"ref-68\">68</xref></sup>. Cryo-electron tomography acquisition parameters are summarized in <bold>Extended Data Tables 4-5</bold>. Tomograms were acquired at 300 kV on a K2 Summit direct detection camera (Gatan) operating in dose fractionation mode and utilizing a Quantum post-column energy filter operated at zero-loss (Gatan). Magnification of 42,000 (EFTEM) with a calibrated pixel size of 3.37 Å was used for the ground truth VPP and DEF datasets and 3.45 Å for NPC data. Up to 14 tilt series were collected on a single lamella in low dose mode using SerialEM<sup><xref ref-type=\"bibr\" rid=\"c69\" hwp:id=\"xref-ref-69-1\" hwp:rel-id=\"ref-69\">69</xref></sup>. Starting from the lamella pre-tilt, images were acquired at 2.0 - 4.0 µm underfocus, in 2° increments within a range of + 58° to - 40° using a dose-symmetric tilt scheme<sup><xref ref-type=\"bibr\" rid=\"c70\" hwp:id=\"xref-ref-70-1\" hwp:rel-id=\"ref-70\">70</xref></sup>. The total dose was up to 120 e-/Å<sup><xref ref-type=\"bibr\" rid=\"c2\" hwp:id=\"xref-ref-2-2\" hwp:rel-id=\"ref-2\">2</xref></sup> with a constant electron dose per tilt image. For the ground truth data, a set of 10 tilt-series were either collected with a 70 µm objective aperture or a Volta potential phase plate (VPP, Thermo Fisher Scientific<sup><xref ref-type=\"bibr\" rid=\"c39\" hwp:id=\"xref-ref-39-2\" hwp:rel-id=\"ref-39\">39</xref></sup>) with prior conditioning for 5 min. NPC data was collected as described in previous work<sup><xref ref-type=\"bibr\" rid=\"c40\" hwp:id=\"xref-ref-40-3\" hwp:rel-id=\"ref-40\">40</xref>, <xref ref-type=\"bibr\" rid=\"c71\" hwp:id=\"xref-ref-71-1\" hwp:rel-id=\"ref-71\">71</xref></sup>, but at 1.5 - 4.5 µm underfocus, with 3° increments and an effective tilt range of + 50° to - 50°.</p></sec><sec id=\"s8e\" hwp:id=\"sec-26\"><title hwp:id=\"title-34\">Tomogram reconstruction</title><p hwp:id=\"p-60\">Tilt movie frames were aligned using a SerialEM plugin. Tilt series were filtered according to the accumulated electron dose by Fourier cropping using the mtffilter function in etomo (IMOD/BETA4.10.12<sup><xref ref-type=\"bibr\" rid=\"c73\" hwp:id=\"xref-ref-73-1\" hwp:rel-id=\"ref-73\">73</xref></sup>), and sorted by tilt angle using a python script. Four times binned tilt images (movie sums) were aligned in etomo (IMOD/4.9.4) software package<sup><xref ref-type=\"bibr\" rid=\"c73\" hwp:id=\"xref-ref-73-2\" hwp:rel-id=\"ref-73\">73</xref></sup> using patch tracking (residual error 0.291 - 0.569 px) and tomograms were reconstructed via weighted back projection. Tomogram thicknesses measured in 3dmod ranged between 80 - 310 nm.</p></sec><sec id=\"s8f\" hwp:id=\"sec-27\"><title hwp:id=\"title-35\">Ground truth annotation for organelles, cytoplasm, and membranes in VPP and DEF</title><p hwp:id=\"p-61\">Cellular compartment segmentation was performed for both the VPP and DEF datasets. The annotations include 10 different organelle classes (mitochondria, vesicle, tube, ER, nuclear envelope, nucleus, vacuole, lipid droplet, Golgi apparatus, vesicular body, see <bold>Extended Data Table 1</bold>) and cytosol annotations, each identified through a unique numerical label to allow for selection of specific subsets of compartments.. Segmentations of the 10 VPP tomograms were performed manually in Amira<sup><xref ref-type=\"bibr\" rid=\"c64\" hwp:id=\"xref-ref-64-2\" hwp:rel-id=\"ref-64\">64</xref></sup>. These annotations were used to train a 2D CNN and to predict the initial segmentations in 10 DEF tomograms, which were first pre-processed with the spectrum equalization filter. These predictions were then manually corrected to obtain high-quality ground truth segmentations in DEF (<bold>Extended Data Table 1</bold>).</p><p hwp:id=\"p-62\">Membrane annotations were performed on the 10 VPP tomograms and 5 DEF tomograms. Initially, 5 VPP tomograms were annotated using Amira to manually segment every 2 to 3 slices, which were subsequently interpolated by means of Amira’s interpolation tool. These 5 VPP tomograms were then used to train a membrane segmentation 3D CNN, whose predictions on the remaining 5 VPP tomograms as well as in 5 DEF tomograms, were manually corrected in Amira.</p></sec><sec id=\"s8g\" hwp:id=\"sec-28\"><title hwp:id=\"title-36\">Ground truth particle annotation in VPP</title><p hwp:id=\"p-63\">RIBO and FAS were localized in 4-times binned tomograms (13.48 Å pixel size) in an iterative workflow. TM (RIBO) or manual annotations (FAS) (step 1, <bold>Extended Data Tables 2-3</bold>) were used for training 3D CNNs (step 2). CNN predictions were masked with a segmentation of the cytosol. Step 2 was repeated several times and each training round trained on the combined predictions of step 1 and the preceding round. Each round in step 2 consisted of three simultaneously trained networks sharing the same set of default hyperparameters (<bold><xref ref-type=\"table\" rid=\"tbls2\" hwp:id=\"xref-table-wrap-2-1\" hwp:rel-id=\"T2\">Supplementary Tables 2</xref>-<xref ref-type=\"table\" rid=\"tbls3\" hwp:id=\"xref-table-wrap-3-1\" hwp:rel-id=\"T3\">3</xref></bold>), except for the number of IF=4, 8, and 32, to provide a cumulative prediction less overfitted to the incomplete training data. The cumulative total predictions from steps 1 and 2 were then complemented by a round of visual inspection to eliminate false positives and manual picking to recover remaining false negatives (step 3, <bold>Extended Data Tables 2-3</bold>). Finally, the particle lists were cleaned for duplicates by applying elliptic distance constraints to the coordinates (<bold>Supplementary Note 1</bold>, <bold>Extended Data Tables 2-3</bold>).</p><p hwp:id=\"p-64\">In particular, for RIBO in the VPP dataset, the initial annotation (step 1) was performed as a combination of two sub-steps: step 1a employed TM performed with the pyTOM toolbox<sup><xref ref-type=\"bibr\" rid=\"c36\" hwp:id=\"xref-ref-36-2\" hwp:rel-id=\"ref-36\">36</xref></sup>. In detail, a 3D cross-correlation search over 1,944 Euler angle combinations was conducted using the large subunit (LSU, 60S) of a published <italic toggle=\"yes\">S. cerevisiae</italic> 80S ribosome map (EMD-3228<sup><xref ref-type=\"bibr\" rid=\"c66\" hwp:id=\"xref-ref-66-2\" hwp:rel-id=\"ref-66\">66</xref></sup>) scaled to the corresponding pixel size, and a spherical mask with a diameter of 337 Å applied. The obtained coordinates corresponding to the top 2000-3000 highest cross correlation scores were manually revised in Gaussian filtered tomograms (sigma = 3) with tom_chooser (pyTOM toolbox<sup><xref ref-type=\"bibr\" rid=\"c36\" hwp:id=\"xref-ref-36-3\" hwp:rel-id=\"ref-36\">36</xref></sup>). This was followed by step 1b, which consisted of a non-exhaustive complementary manual annotation of particles missed by TM (<bold>Extended Data Tables 2-3</bold>). These initial ribosome coordinates served as input for step 2: three training rounds of CNNs were successively applied, as a global visual inspection confirmed that the false positive rate was low, while the fraction of true positives recovered was substantial. The resulting CNN predictions were manually revised in tom_chooser as described above. In step 3, remaining particles that were not detected by either TM or CNN were manually localized in either spectrum-matched or Gaussian filtered tomograms (sigma = 3) for up to 3 rounds in EMAN2 spt2_boxer<sup><xref ref-type=\"bibr\" rid=\"c74\" hwp:id=\"xref-ref-74-1\" hwp:rel-id=\"ref-74\">74</xref></sup>.</p><p hwp:id=\"p-65\">FAS in the VPP dataset was localized in step 1 manually in EMAN2, as TM with a published <italic toggle=\"yes\">S. cerevisiae</italic> FAS<sup><xref ref-type=\"bibr\" rid=\"c46\" hwp:id=\"xref-ref-46-3\" hwp:rel-id=\"ref-46\">46</xref></sup> map (EMD-1623) as reference failed (<bold>Extended Data Tables 3</bold>). In step 2 we performed only one round of training with these initial annotations and the resulting predictions were revised in EMAN2. Additional FAS particles were localized manually.</p></sec><sec id=\"s8h\" hwp:id=\"sec-29\"><title hwp:id=\"title-37\">Ground truth particle annotation in DEF</title><p hwp:id=\"p-66\">In the DEF dataset, the annotation procedure was the same as the one described for the VPP dataset, except that: 1) for RIBO the initial annotations relied on manually cleaned TM results and for FAS on incomplete manual picking, and 2) in step 2 of the RIBO ground truth construction, the first of the 3 rounds of CNN predictions used the DEF training data (from the TM results), while for the remaining 2 rounds the predictions were obtained using the models from the first and second rounds of RIBO VPP ground truth construction (trained only on VPP data, <bold><xref ref-type=\"table\" rid=\"tbls2\" hwp:id=\"xref-table-wrap-2-2\" hwp:rel-id=\"T2\">Supplementary Table 2</xref></bold>). The visual inspection in step 2 was performed over the combined predictions of both the previously mentioned VPP networks and a round of DEF networks (IF=4, 8, 32) based on DEF TM results. After the final manual picking for missing particles (not present in either the initial TM results or the manually cleaned network predictions), we achieved comprehensive RIBO and FAS annotations (<bold>Extended Data Tables 2-3</bold>).</p></sec><sec id=\"s8i\" hwp:id=\"sec-30\"><title hwp:id=\"title-38\">Comparison of cryo-ET-derived particle numbers with proteomics</title><p hwp:id=\"p-67\">Copy numbers of ribosomes and FAS per cell were calculated with the ground truth annotations for an <italic toggle=\"yes\">S. pombe</italic> cell with the assumption of 30 % cytosolic volume<sup><xref ref-type=\"bibr\" rid=\"c75\" hwp:id=\"xref-ref-75-1\" hwp:rel-id=\"ref-75\">75</xref></sup> of a total of 150 µm<sup><xref ref-type=\"bibr\" rid=\"c3\" hwp:id=\"xref-ref-3-1\" hwp:rel-id=\"ref-3\">3</xref></sup> average cell volume<sup><xref ref-type=\"bibr\" rid=\"c76\" hwp:id=\"xref-ref-76-1\" hwp:rel-id=\"ref-76\">76</xref>, <xref ref-type=\"bibr\" rid=\"c77\" hwp:id=\"xref-ref-77-1\" hwp:rel-id=\"ref-77\">77</xref></sup>. It was also considered in the case of FAS that one fully assembled FAS complex is constituted by each six alpha and beta subunits.</p></sec><sec id=\"s8j\" hwp:id=\"sec-31\"><title hwp:id=\"title-39\">NPC manual localization</title><p hwp:id=\"p-68\">NPCs were manually localized as described previously<sup><xref ref-type=\"bibr\" rid=\"c40\" hwp:id=\"xref-ref-40-4\" hwp:rel-id=\"ref-40\">40</xref></sup>. In brief, coordinates and initial orientations of 354 NPCs were manually determined in 127 4-times binned, SIRT-like filtered<sup><xref ref-type=\"bibr\" rid=\"c73\" hwp:id=\"xref-ref-73-3\" hwp:rel-id=\"ref-73\">73</xref></sup> defocus-only tomograms, as described earlier<sup><xref ref-type=\"bibr\" rid=\"c71\" hwp:id=\"xref-ref-71-2\" hwp:rel-id=\"ref-71\">71</xref></sup>. Two points located at opposing inner and outer nuclear membrane (INM-ONM) fusion points were manually picked for each NPC using IMOD. A third point was then chosen located at the INM-ONM fusion ring such that all three points define the horizontal plane of the NPC particle. The mean coordinates between the first two points were then used to assign the center of the NPC. The initial orientation of the NPCs was defined based on the normal vector to the defined NPC plane. A fourth point was chosen in the cytoplasmic side of the horizontal plane to assign the correct initial orientation and prevent false upside-down assignment of initial orientations. The annotated NPC data was divided based on quality criteria: 38 tomograms of quality 1 have a thickness below 300 nm, thus display optimal image contrast, and a tilt-series alignment residual error below 0.7 px. Quality 0 was assigned to 89 tomograms with 300 - 395 nm thickness and a residual error of 0.7 - 5.0 for tilt series alignments.</p></sec><sec id=\"s8k\" hwp:id=\"sec-32\"><title hwp:id=\"title-40\">Voxel-level representation of ground truth</title><p hwp:id=\"p-69\">As the final product of our ground truth generation process, we obtain voxel-based masks for membranes, cellular organelles, cytoplasm, and particles (RIBO, FAS, NPC). For RIBO and FAS, the lists of coordinates obtained were used to paste spherical masks (with radii of 10.78 nm for RIBO and 13.48 nm for FAS) centered on each of these points using a python script (Code and data availability). For the NPCs in DEF*, a subunit mask obtained by prior 3D averaging in novaSTA<sup><xref ref-type=\"bibr\" rid=\"c78\" hwp:id=\"xref-ref-78-1\" hwp:rel-id=\"ref-78\">78</xref></sup>, was pasted at each of the 8 subunit locations. The RIBO, FAS, membrane, organelle and cytosol masks in DEF and VPP along with the list of coordinates and respective tomograms are used in the subsequent training and performance analysis and are available on EMPIAR (Code and data availability).</p></sec><sec id=\"s8l\" hwp:id=\"sec-33\"><title hwp:id=\"title-41\">Cytoskeletal filaments segmentation</title><p hwp:id=\"p-70\">Microtubule and actin networks were trained on tomograms of Human retinal pigment epithelial-1 (RPE-1) cells previously described<sup><xref ref-type=\"bibr\" rid=\"c65\" hwp:id=\"xref-ref-65-2\" hwp:rel-id=\"ref-65\">65</xref></sup>. Segmentations of the individual cytoskeletal elements performed using the filament tracing function in Amira<sup><xref ref-type=\"bibr\" rid=\"c79\" hwp:id=\"xref-ref-79-1\" hwp:rel-id=\"ref-79\">79</xref>, <xref ref-type=\"bibr\" rid=\"c80\" hwp:id=\"xref-ref-80-1\" hwp:rel-id=\"ref-80\">80</xref></sup>, followed by manual curation. Trained networks were applied to a tomogram depicting the nuclear periphery of a HeLa cell and compared against the corresponding segmentations (EMD-11992).</p></sec><sec id=\"s8m\" hwp:id=\"sec-34\"><title hwp:id=\"title-42\">Subtomogram analysis for ribosomes and FAS</title><p hwp:id=\"p-71\">Contrast transfer function (CTF) estimations, generation of 3D CTF models and subtomogram reconstructions were performed in Warp<sup><xref ref-type=\"bibr\" rid=\"c32\" hwp:id=\"xref-ref-32-2\" hwp:rel-id=\"ref-32\">32</xref></sup>. CTFs were first estimated in the sums of raw tilt movies and subsequently in the tilt series taking the tilt angles into account. Subtomograms containing ribosomes and their respective CTF models were reconstructed in volumes of 140³ pixels with a pixel size of 3.3702 Å and a particle diameter of 350 Å. Initial alignments of the subtomograms were performed in RELION version 3.0.7<sup><xref ref-type=\"bibr\" rid=\"c34\" hwp:id=\"xref-ref-34-2\" hwp:rel-id=\"ref-34\">34</xref></sup> in 25 iterations of 3D classification, with the <italic toggle=\"yes\">S. cerevisiae</italic> 80S ribosome (EMD-3228, low-pass filtered to 60 Å) as reference to generate an initial single class average. 3D refinements were performed with the resulting average as a reference. In the case of DEF data, this RELION-refined average was further refined in M<sup><xref ref-type=\"bibr\" rid=\"c33\" hwp:id=\"xref-ref-33-3\" hwp:rel-id=\"ref-33\">33</xref></sup> to optimize particle poses, images and volumes warping to model non-linear deformations. Particles were re-extracted and hierarchical 3D classifications (25 iterations each) were performed in RELION. In the case of VPP data, hierarchical 3D classifications were performed directly after 3D refinements in RELION. Focused classifications were performed with binary masks indicated in the respective figures. Masks were generated from spheres produced in Matlab (MathWorks) in a cubic volume of 50 pixels per side and a radius of 20 pixels, scaled to a pixel size of 3.3702 Å and converted into binary masks resulting in a radius of around 130 Å using RELION, and placed relative to the area of interest using Chimera<sup><xref ref-type=\"bibr\" rid=\"c81\" hwp:id=\"xref-ref-81-1\" hwp:rel-id=\"ref-81\">81</xref></sup>.</p><p hwp:id=\"p-72\">FAS particles and CTF models were reconstructed in cubic volumes of 160 pixels per side with a pixel size of 3.3702 Å and a particle diameter of 400 Å. Initial alignments were performed in RELION in 25 iterations of 3D classification into a single class, and followingly refined with the 3D refinement option and applying D3 symmetry using the <italic toggle=\"yes\">S. cerevisiae</italic> FAS map (EMD-1623<sup><xref ref-type=\"bibr\" rid=\"c46\" hwp:id=\"xref-ref-46-4\" hwp:rel-id=\"ref-46\">46</xref></sup>, low-pass filtered to 60 Å) as reference. Hierarchical 3D classifications (25 iterations each) were performed either after 3D refinements in RELION (VPP data) or after subtomogram re-extraction using the ribosome-optimized image and volume models in M (described above).</p><p hwp:id=\"p-73\">Final subtomogram averages of each particle class were obtained by 3D refinement and post-processing, including filtering to their respective resolutions determined by Fourier Shell Correlation (FSC) of two independently refined half maps at a cut-off of 0.143. Details of particle numbers and resolutions for each subtomogram average are summarized in <bold>Extended Data Tables 4-6</bold>. Visualization and calculation of cross-correlations (CC) between different maps and models, was performed with the UCSF ChimeraX package<sup><xref ref-type=\"bibr\" rid=\"c82\" hwp:id=\"xref-ref-82-1\" hwp:rel-id=\"ref-82\">82</xref></sup>.</p></sec><sec id=\"s8n\" hwp:id=\"sec-35\"><title hwp:id=\"title-43\">CNN Pre- and post-processing</title><sec id=\"s8n1\" hwp:id=\"sec-36\"><title hwp:id=\"title-44\">Preprocessing</title><p hwp:id=\"p-74\">Tomograms are normalized to obtain uniform mean of 0 and variance of 1 in the frequency domain, prior to training. Additionally, the spectrum equalization filter is applied by matching the amplitude spectrum of the tomogram that is to be transformed to the target spectrum of one manually selected high-contrast VPP tomogram (Tomogram TS_001, <bold>Extended Data</bold> Fig. 1b). Extraction of spectra amplitudes is done using fast Fourier transform (FFT) followed by radially averaging amplitudes across the frequency domain. If the target tomogram’s Nyquist frequency is lower than the input tomogram’s, the target spectrum is padded with zeros to match the input spectrum’s size. Next, an equalization vector is created by dividing entry-wise the target spectrum by the respective input spectrum. The resulting equalization vector is converted into a rotational kernel and multiplied by the input tomograms in the frequency domain in combination with a sigmoidal-shaped low-pass filter to eliminate high frequency noise. After back transformation, the tomogram exhibits a similar contrast to the target tomogram (<bold>Extended Data</bold> Fig. 1). Tomogram intensities are additionally normalized to obtain uniform mean and variance. For the 2D CNN, tomograms and training segmentations are processed slice-wise into 2D tiles with a fixed size of 288x288 pixels (256x256 pixels and 16 pixels padding on each side). For the 3D CNN, tomograms are by default split into cubic patches of 64x64x64 voxels, and 12 voxels overlap in each dimension.</p></sec><sec id=\"s8n2\" hwp:id=\"sec-37\"><title hwp:id=\"title-45\">Post-processing (2D CNN)</title><p hwp:id=\"p-75\">The post-processing steps for the 2D network are focused on assembling the per-slice prediction into a 3D segmentation. To that aim, predicted tile segmentations are cropped on each side by 48 pixels to reduce artifacts around the edges, followed by reassembly into 3D stacks, with remaining overlapping areas being averaged. A 1D Gaussian filter is successively applied along the z axis in order to reduce single-slice false positives (<bold>Extended Data</bold> Fig. 2).</p></sec><sec id=\"s8n3\" hwp:id=\"sec-38\"><title hwp:id=\"title-46\">Post-processing (3D CNN)</title><p hwp:id=\"p-76\">After reassembly of the individual 64x64x64 voxel patches into the probability map resulting from the 3D CNN, the post processing step includes the clustering of the thresholded map (usually at threshold value of 0.5). According to user preferences, only clusters that lie within or close to a given mask consisting of an organelle or cytosol segmentation outputted by the 2D network, and whose size lies within a given range of numbers of voxels, are considered for the final prediction map (<bold>Supplementary Note 1</bold>). This ‘cleaning’ procedure allows adding <italic toggle=\"yes\">a priori</italic> information on the size of the particles in question. Additionally, for macromolecular complex localization, the user has the option to export a list of coordinates of cluster-centroids, which represent the particles’ locations prediction.</p></sec><sec id=\"s8n4\" hwp:id=\"sec-39\"><title hwp:id=\"title-47\">Precision-Recall and Performance Analysis</title><sec id=\"s8n4a\" hwp:id=\"sec-40\"><title hwp:id=\"title-48\">Cross-validation approach</title><p hwp:id=\"p-77\">The analysis of performance for RIBO and FAS localization, as well as membrane segmentation, were done using a 3-fold CV scheme, where 8 out of the 10 VPP tomograms were used for training, and the remaining 2 VPP tomograms were used for testing. In the test for domain generalization, the same three networks (trained on the 8 VPP tomograms) were applied to the 10 DEF tomograms.</p><p hwp:id=\"p-78\">The analysis of performance in the NPC segmentation task was done in the dedicated DEF* of 127 tomograms. Importantly, in order to apply the DeePiCt pipeline, we employed a predicted segmentation of the nuclear envelope as the <italic toggle=\"yes\">region mask</italic> in the <italic toggle=\"yes\">contact</italic> mode during post-processing of the NPC prediction (<bold>Supplementary Note 1</bold>). The NPC prediction was studied under a 3-fold CV scheme, where each split consisted of roughly 42 tomograms. The nuclear envelope prediction was achieved through an independently trained 3D CNN on 18 tomograms belonging to the DEF* dataset for which a manual annotation of the nuclear envelope was available, and which were uniformly split across the 3 CV subsets.</p><p hwp:id=\"p-79\">For organelle and cytosol segmentation evaluation, model performance was evaluated on the voxel-level for the post-processed 2D CNN predictions produced by each CV fold’s individually trained model, using 5000 voxels picked randomly from each tomogram of the CV fold’s respective test set. Precision and recall (<bold>Supplementary Note 1</bold>) were computed at threshold values varying from 0 to 1 on the picked voxels to compute the area under the precision-recall curves (AURPC) for each tomogram.</p></sec></sec><sec id=\"s8n5\" hwp:id=\"sec-41\"><title hwp:id=\"title-49\">Evaluation metrics</title><p hwp:id=\"p-80\">In the case of particles (RIBO and FAS), the localization evaluation was made directly from the predicted lists containing their centroids’ voxel coordinates. In the case of DeePict, the lists of particles’ coordinates are sorted according to a decreasing score (cluster size) value. For both RIBO and FAS, we defined a true positive when a predicted coordinate matched a ground-truth particle coordinate within a tolerance radius of 130 Å (<bold>Supplementary Note 1</bold>). The F1 score, which is calculated as the harmonic mean of the precision and the recall, provides a measure of the quality of the predictions. We chose this measure for easier comparison to previous studies for particle localization methods in the field of cryo-ET<sup><xref ref-type=\"bibr\" rid=\"c26\" hwp:id=\"xref-ref-26-2\" hwp:rel-id=\"ref-26\">26</xref>, <xref ref-type=\"bibr\" rid=\"c27\" hwp:id=\"xref-ref-27-1\" hwp:rel-id=\"ref-27\">27</xref>, <xref ref-type=\"bibr\" rid=\"c29\" hwp:id=\"xref-ref-29-2\" hwp:rel-id=\"ref-29\">29</xref>, <xref ref-type=\"bibr\" rid=\"c83\" hwp:id=\"xref-ref-83-1\" hwp:rel-id=\"ref-83\">83</xref></sup>. Other metrics such as the area under the precision recall curve (AUPRC) generated by increasing length of particles’ list were disregarded as the ordering of the list (a function of cluster size) does not necessarily reflect the method’s confidence on positive classification.</p><p hwp:id=\"p-81\">The evaluation of the segmentation task is of a totally different nature, as the shape of the object should be taken into account voxel-wise. For the 2D CNN predictions of organelles and cytosol, we used the AUPRC score with varying threshold over the network’s output map (<bold>Supplementary Note 1</bold>). For Deepict’s predictions of NPC and membrane segmentation, we instead used the voxel-based F1 score (<bold>Supplementary Note 1</bold>), to maintain hegemony with DeePiCt particle prediction evaluations.</p></sec><sec id=\"s8n6\" hwp:id=\"sec-42\"><title hwp:id=\"title-50\">Hyperparameter tuning of DeePiCt and performance assessment</title><p hwp:id=\"p-82\">The previously described cross-validation schemes and evaluation metrics were not only used for the final assessment of methods’ performance (Fig. 3), but to study the effect of the different hyperparameters that could be set by the user in the case of the 3D CNN (IF, D, ED, DD, and BN). The experiments’ starting point was always the default hyperparameter combination <italic toggle=\"yes\">D</italic> = 2, <italic toggle=\"yes\">IF</italic> = 4, <italic toggle=\"yes\">BN</italic> = 0, <italic toggle=\"yes\">ED</italic> = 0, <italic toggle=\"yes\">DD</italic> = 0 (<bold>Supplementary Note 1</bold>). In all cases, we tested first for different values of <italic toggle=\"yes\">D</italic>, while keeping the rest fixed. Then, we fixed <italic toggle=\"yes\">D=D*,</italic> where <italic toggle=\"yes\">D*</italic> denotes the value for which best performance was achieved and repeated the same process for <italic toggle=\"yes\">IF</italic>. We continued with each of the remaining parameters, <italic toggle=\"yes\">BN</italic>, <italic toggle=\"yes\">ED</italic> and <italic toggle=\"yes\">DD</italic>. The results are reported in <bold>Extended Data</bold> Fig. 6. They allowed us to define the best hyperparameter combination for each of the structures studied, which are summarized in <bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-5\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>.</p><p hwp:id=\"p-83\">In each case we combined the 3D CNN segmentation of the target structure with an appropriate <italic toggle=\"yes\">region mask</italic> (<bold>Supplementary Note 1</bold>, <bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-6\" hwp:rel-id=\"T1\">Supplementary Table 1</xref>)</bold>: in the RIBO and FAS localization task, we used a cytosol prediction from the 2D CNN in <italic toggle=\"yes\">intersection</italic> mode to eliminate false positives in organelles and the extracellular space; for membrane segmentation we used again the predicted cytosol from the 2D CNN, this time in <italic toggle=\"yes\">contact</italic> mode to eliminate clusters that are in the extracellular space or within organelles; for NPCs we used a 3D CNN prediction of the nuclear envelope in <italic toggle=\"yes\">contact</italic> mode to eliminate false NPC segmentations far from the nuclear boundary. For RIBO, FAS and membrane, both 3D CNN (for target structure prediction) and 2D CNN (for <italic toggle=\"yes\">region mask</italic> prediction) in DeePiCt employed a 3-fold CV in the VPP dataset. In each fold, 8 tomograms were used for training and model selection while the remaining 2 tomograms were used for testing (<bold>Extended Data</bold> Fig. 6a<bold>-c</bold>). The NPC predictions, which we treat as segmentation tasks, were evaluated in the DEF* independent data selected for that purpose (<bold>Supplementary Note 1</bold> and <bold>Extended Data</bold> Fig. 6d<bold>-f</bold>). The <italic toggle=\"yes\">region mask</italic> of predicted nuclear envelope segmentation was generated by a fixed 3D CNN trained on 18 of the 127 DEF* tomograms. These 18 tomograms were evenly distributed among the 3 splits of the DEF* dataset used for the corresponding 3-fold CV scheme employed for the hyperparameter tuning analysis of NPC segmentation by DeePiCt (<bold>Extended Data</bold> Fig. 6d<bold>-f</bold>). We used a 3-fold CV where the total 127 tomograms were split into 3 random subsets with roughly the same number of tomograms each. Each fold consists of two such sets as training data and the remaining one for testing. Throughout our experiments, high-quality DEF* tomograms (defined based on lamella thickness and tomographic alignment residual error as described above) consistently showed higher performance for the NPC segmentation task than the low-quality DEF* tomograms, regardless of architectural variations (<bold>Extended Data</bold> Fig. 6d<bold>-f</bold>). This hints at the importance of high data-quality in segmentation and localization tasks.</p><p hwp:id=\"p-84\">For the evaluation of the particle localization task, we compared the lists of coordinates of the particles in our ground truth data with the predicted coordinate lists. For both RIBO and FAS, we considered a true positive when coordinates coincide within a tolerance radius of 10 voxels (corresponding to approximately 130 Å). In this way, we computed the recall (the proportion of ground truth particles that were recovered by the method) and the precision (the proportion of predicted particles that were true positives) and reported the F1 score, which is the harmonic mean of precision and recall (<bold>Supplementary Note 1, <xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-7\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>). For the structure segmentation task (such as the membrane or the NPC segmentation evaluation), we compared the ground truth masks with the DeePiCt predicted post-processed segmentation by calculating their voxel-wise precision and recall, and reporting the corresponding voxel-based F1 (voxel-F1) score, also known as Sørensen-Dice coefficient (<bold>Supplementary Note 1</bold>).</p><p hwp:id=\"p-85\">All experiments for the 3D CNN of the DeePiCt pipeline and DeepFinder were performed using NVIDIA 2080 Ti GPU, Cuda 10.0, Python 3 and Pytorch 1.3.1. For the 2D CNN, training was conducted using an NVIDIA 2080 Ti GPU and an NVIDIA V100S GPU used for performance evaluation, using CUDA 10.0, Python 3 and Keras 2.3.1 with a tensorflow 2.0.0 backend. Detailed lists of parameters used for the 2D and 3D CNN are available alongside DeePiCt’s source code (Code and data availability).</p></sec></sec></sec><ref-list hwp:id=\"ref-list-1\"><title hwp:id=\"title-51\">Bibliography</title><ref id=\"c1\" hwp:id=\"ref-1\" hwp:rev-id=\"xref-ref-1-1 xref-ref-1-2 xref-ref-1-3 xref-ref-1-4 xref-ref-1-5 xref-ref-1-6 xref-ref-1-7 xref-ref-1-8\"><label>1.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.1\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-1\"><string-name name-style=\"western\" hwp:sortable=\"Mahamid J.\"><surname>Mahamid</surname>, <given-names>J.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-2\">Visualizing the molecular sociology at the HeLa cell nuclear periphery</article-title>. <source hwp:id=\"source-1\">Science</source> <volume>351</volume>, <fpage>969</fpage>–<lpage>972</lpage> (<year>2016</year>).</citation></ref><ref id=\"c2\" hwp:id=\"ref-2\" hwp:rev-id=\"xref-ref-2-1 xref-ref-2-2 xref-ref-2-3 xref-ref-2-4\"><label>2.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.2\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-2\"><string-name name-style=\"western\" hwp:sortable=\"Pfeffer S.\"><surname>Pfeffer</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-3\">Dissecting the molecular organization of the translocon-associated protein complex</article-title>. <source hwp:id=\"source-2\">Nat. Commun</source>. <volume>8</volume>, <issue>14516</issue> (<year>2017</year>).</citation></ref><ref id=\"c3\" hwp:id=\"ref-3\" hwp:rev-id=\"xref-ref-3-1 xref-ref-3-2 xref-ref-3-3 xref-ref-3-4\"><label>3.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.3\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-3\"><string-name name-style=\"western\" hwp:sortable=\"Albert S.\"><surname>Albert</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-4\">Proteasomes tether to two distinct sites at the nuclear pore complex</article-title>. <source hwp:id=\"source-3\">Proc Natl Acad Sci USA</source> <volume>114</volume>, <fpage>13726</fpage>–<lpage>13731</lpage> (<year>2017</year>).</citation></ref><ref id=\"c4\" hwp:id=\"ref-4\" hwp:rev-id=\"xref-ref-4-1 xref-ref-4-2 xref-ref-4-3\"><label>4.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.4\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-4\"><string-name name-style=\"western\" hwp:sortable=\"Bäuerlein F. J. B.\"><surname>Bäuerlein</surname>, <given-names>F. J. B.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-5\">In situ architecture and cellular interactions of polyq inclusions</article-title>. <source hwp:id=\"source-4\">Cell</source> <volume>171</volume>, <fpage>179</fpage>–<lpage>187</lpage>.e10 (<year>2017</year>).</citation></ref><ref id=\"c5\" hwp:id=\"ref-5\" hwp:rev-id=\"xref-ref-5-1 xref-ref-5-2 xref-ref-5-3\"><label>5.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.5\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-5\"><string-name name-style=\"western\" hwp:sortable=\"Guo Q.\"><surname>Guo</surname>, <given-names>Q.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-6\">In Situ Structure of Neuronal C9orf72 Poly-GA Aggregates Reveals Proteasome Recruitment</article-title>. <source hwp:id=\"source-5\">Cell</source> <volume>172</volume>, <fpage>696</fpage>–<lpage>705</lpage>.e12 (<year>2018</year>).</citation></ref><ref id=\"c6\" hwp:id=\"ref-6\" hwp:rev-id=\"xref-ref-6-1 xref-ref-6-2\"><label>6.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.6\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-6\"><string-name name-style=\"western\" hwp:sortable=\"Albert S.\"><surname>Albert</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-7\">Direct visualization of degradation microcompartments at the ER membrane</article-title>. <source hwp:id=\"source-6\">Proc Natl Acad Sci USA</source> <volume>117</volume>, <fpage>1069</fpage>–<lpage>1080</lpage> (<year>2020</year>).</citation></ref><ref id=\"c7\" hwp:id=\"ref-7\" hwp:rev-id=\"xref-ref-7-1 xref-ref-7-2\"><label>7.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.7\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-7\"><string-name name-style=\"western\" hwp:sortable=\"Wilfling F.\"><surname>Wilfling</surname>, <given-names>F.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-8\">A Selective Autophagy Pathway for Phase-Separated Endocytic Protein Deposits</article-title>. <source hwp:id=\"source-7\">Mol. Cell</source> <volume>80</volume>, <fpage>764</fpage>–<lpage>778</lpage>.e7 (<year>2020</year>).</citation></ref><ref id=\"c8\" hwp:id=\"ref-8\" hwp:rev-id=\"xref-ref-8-1 xref-ref-8-2\"><label>8.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.8\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-8\"><string-name name-style=\"western\" hwp:sortable=\"Watanabe R.\"><surname>Watanabe</surname>, <given-names>R.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-9\">The In Situ Structure of Parkinson’s Disease-Linked LRRK2</article-title>. <source hwp:id=\"source-8\">Cell</source> <volume>182</volume>, <fpage>1508</fpage>–<lpage>1518</lpage>.e16 (<year>2020</year>).</citation></ref><ref id=\"c9\" hwp:id=\"ref-9\" hwp:rev-id=\"xref-ref-9-1 xref-ref-9-2 xref-ref-9-3 xref-ref-9-4 xref-ref-9-5\"><label>9.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.9\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-9\"><string-name name-style=\"western\" hwp:sortable=\"Böhning J.\"><surname>Böhning</surname>, <given-names>J.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Bharat T. A. M\"><surname>Bharat</surname>, <given-names>T. A. M</given-names></string-name>. <article-title hwp:id=\"article-title-10\">Towards high-throughput in situ structural biology using electron cryotomography</article-title>. <source hwp:id=\"source-9\">Prog. Biophys. Mol. Biol</source>. <volume>160</volume>, <fpage>97</fpage>–<lpage>103</lpage> (<year>2021</year>).</citation></ref><ref id=\"c10\" hwp:id=\"ref-10\" hwp:rev-id=\"xref-ref-10-1 xref-ref-10-2\"><label>10.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.10\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-10\"><string-name name-style=\"western\" hwp:sortable=\"Weis F.\"><surname>Weis</surname>, <given-names>F.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Hagen W. J. H\"><surname>Hagen</surname>, <given-names>W. J. H</given-names></string-name>. <article-title hwp:id=\"article-title-11\">Combining high throughput and high quality for cryo-electron microscopy data collection</article-title>. <source hwp:id=\"source-10\">Acta Crystallogr. D Struct. Biol</source>. <volume>76</volume>, <fpage>724</fpage>–<lpage>728</lpage> (<year>2020</year>).</citation></ref><ref id=\"c11\" hwp:id=\"ref-11\" hwp:rev-id=\"xref-ref-11-1 xref-ref-11-2 xref-ref-11-3\"><label>11.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.11\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-11\"><string-name name-style=\"western\" hwp:sortable=\"Klumpe S.\"><surname>Klumpe</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-12\">A modular platform for automated cryo-FIB workflows</article-title>. <source hwp:id=\"source-11\">eLife</source> <volume>10</volume>, (<year>2021</year>).</citation></ref><ref id=\"c12\" hwp:id=\"ref-12\" hwp:rev-id=\"xref-ref-12-1 xref-ref-12-2\"><label>12.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.12\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-12\"><string-name name-style=\"western\" hwp:sortable=\"Zachs T.\"><surname>Zachs</surname>, <given-names>T.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-13\">Fully automated, sequential focused ion beam milling for cryo-electron tomography</article-title>. <source hwp:id=\"source-12\">eLife</source> <volume>9</volume>, (<year>2020</year>).</citation></ref><ref id=\"c13\" hwp:id=\"ref-13\" hwp:rev-id=\"xref-ref-13-1 xref-ref-13-2\"><label>13.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.13\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-13\"><string-name name-style=\"western\" hwp:sortable=\"Pyle E.\"><surname>Pyle</surname>, <given-names>E.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Zanetti G\"><surname>Zanetti</surname>, <given-names>G</given-names></string-name>. <article-title hwp:id=\"article-title-14\">Current data processing strategies for cryo-electron tomography and subtomogram averaging</article-title>. <source hwp:id=\"source-13\">Biochem. J</source>. <volume>478</volume>, <fpage>1827</fpage>–<lpage>1845</lpage> (<year>2021</year>).</citation></ref><ref id=\"c14\" hwp:id=\"ref-14\" hwp:rev-id=\"xref-ref-14-1\"><label>14.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.14\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-14\"><string-name name-style=\"western\" hwp:sortable=\"Lučič V.\"><surname>Lučič</surname>, <given-names>V.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Rigort A.\"><surname>Rigort</surname>, <given-names>A.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Baumeister W\"><surname>Baumeister</surname>, <given-names>W</given-names></string-name>. <article-title hwp:id=\"article-title-15\">Cryo-electron tomography: the challenge of doing structural biology in situ</article-title>. <source hwp:id=\"source-14\">J. Cell Biol</source>. <volume>202</volume>, <fpage>407</fpage>–<lpage>419</lpage> (<year>2013</year>).</citation></ref><ref id=\"c15\" hwp:id=\"ref-15\" hwp:rev-id=\"xref-ref-15-1\"><label>15.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.15\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-15\"><string-name name-style=\"western\" hwp:sortable=\"Wan W.\"><surname>Wan</surname>, <given-names>W.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Briggs J. A. G.\"><surname>Briggs</surname>, <given-names>J. A. G.</given-names></string-name> <article-title hwp:id=\"article-title-16\">Cryo-Electron Tomography and Subtomogram Averaging</article-title>. <source hwp:id=\"source-15\">Meth. Enzymol.</source> <volume>579</volume>, <fpage>329</fpage>–<lpage>367</lpage> (<year>2016</year>).</citation></ref><ref id=\"c16\" hwp:id=\"ref-16\" hwp:rev-id=\"xref-ref-16-1 xref-ref-16-2\"><label>16.</label><citation publication-type=\"book\" citation-type=\"book\" ref:id=\"2022.04.12.488077v1.16\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-16\"><string-name name-style=\"western\" hwp:sortable=\"Wu X.\"><surname>Wu</surname>, <given-names>X.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Zeng X.\"><surname>Zeng</surname>, <given-names>X.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Zhu Z.\"><surname>Zhu</surname>, <given-names>Z.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Gao X.\"><surname>Gao</surname>, <given-names>X.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Xu M\"><surname>Xu</surname>, <given-names>M</given-names></string-name>. <chapter-title>Template-Based and Template-Free Approaches in Cellular Cryo-Electron Tomography Structural Pattern Mining</chapter-title>. in <source hwp:id=\"source-16\">Computational Biology</source> (ed. <string-name name-style=\"western\" hwp:sortable=\"Husi H.\"><surname>Husi</surname>, <given-names>H.</given-names></string-name>) (<publisher-name>Codon Publications</publisher-name>, <year>2019</year>). doi:<pub-id pub-id-type=\"doi\">10.15586/computationalbiology.2019.ch11</pub-id>.</citation></ref><ref id=\"c17\" hwp:id=\"ref-17\" hwp:rev-id=\"xref-ref-17-1\"><label>17.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.17\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-17\"><string-name name-style=\"western\" hwp:sortable=\"Bohm J.\"><surname>Bohm</surname>, <given-names>J.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-17\">Toward detecting and identifying macromolecules in a cellular context: template matching applied to electron tomograms</article-title>. <source hwp:id=\"source-17\">Proc Natl Acad Sci USA</source> <volume>97</volume>, <fpage>14245</fpage>–<lpage>14250</lpage> (<year>2000</year>).</citation></ref><ref id=\"c18\" hwp:id=\"ref-18\" hwp:rev-id=\"xref-ref-18-1\"><label>18.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.18\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-18\"><string-name name-style=\"western\" hwp:sortable=\"Bäuerlein F. J. B.\"><surname>Bäuerlein</surname>, <given-names>F. J. B.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Baumeister W\"><surname>Baumeister</surname>, <given-names>W</given-names></string-name>. <article-title hwp:id=\"article-title-18\">Towards visual proteomics at high resolution</article-title>. <source hwp:id=\"source-18\">J. Mol. Biol</source>. <volume>433</volume>, <issue>167187</issue> (<year>2021</year>).</citation></ref><ref id=\"c19\" hwp:id=\"ref-19\" hwp:rev-id=\"xref-ref-19-1\"><label>19.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.19\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-19\"><string-name name-style=\"western\" hwp:sortable=\"Xu M.\"><surname>Xu</surname>, <given-names>M.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Beck M.\"><surname>Beck</surname>, <given-names>M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Alber F\"><surname>Alber</surname>, <given-names>F</given-names></string-name>. <article-title hwp:id=\"article-title-19\">Template-free detection of macromolecular complexes in cryo electron tomograms</article-title>. <source hwp:id=\"source-19\">Bioinformatics</source> <volume>27</volume>, <fpage>i69</fpage>–<lpage>76</lpage> (<year>2011</year>).</citation></ref><ref id=\"c20\" hwp:id=\"ref-20\" hwp:rev-id=\"xref-ref-20-1\"><label>20.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.20\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-20\"><string-name name-style=\"western\" hwp:sortable=\"Martinez-Sanchez A.\"><surname>Martinez-Sanchez</surname>, <given-names>A.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-20\">Template-free detection and classification of membrane-bound complexes in cryo-electron tomograms</article-title>. <source hwp:id=\"source-20\">Nat. Methods</source> <volume>17</volume>, <fpage>209</fpage>–<lpage>216</lpage> (<year>2020</year>).</citation></ref><ref id=\"c21\" hwp:id=\"ref-21\" hwp:rev-id=\"xref-ref-21-1\"><label>21.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.21\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-21\"><string-name name-style=\"western\" hwp:sortable=\"Krizhevsky A.\"><surname>Krizhevsky</surname>, <given-names>A.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Sutskever I.\"><surname>Sutskever</surname>, <given-names>I.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Hinton G. E\"><surname>Hinton</surname>, <given-names>G. E</given-names></string-name>. <article-title hwp:id=\"article-title-21\">ImageNet classification with deep convolutional neural networks</article-title>. <source hwp:id=\"source-21\">Commun. ACM</source> <volume>60</volume>, <fpage>84</fpage>–<lpage>90</lpage> (<year>2012</year>).</citation></ref><ref id=\"c22\" hwp:id=\"ref-22\"><label>22.</label><citation publication-type=\"confproc\" citation-type=\"confproc\" ref:id=\"2022.04.12.488077v1.22\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-22\"><string-name name-style=\"western\" hwp:sortable=\"LeCun Y.\"><surname>LeCun</surname>, <given-names>Y.</given-names></string-name> <article-title hwp:id=\"article-title-22\">Deep learning &amp; convolutional networks</article-title>. in <conf-name>2015 IEEE Hot Chips 27 Symposium (HCS) 1–95 (IEEE</conf-name>, <year>2015</year>). doi:<pub-id pub-id-type=\"doi\">10.1109/HOTCHIPS.2015.7477328</pub-id>.</citation></ref><ref id=\"c23\" hwp:id=\"ref-23\" hwp:rev-id=\"xref-ref-23-1\"><label>23.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.23\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-23\"><string-name name-style=\"western\" hwp:sortable=\"LeCun Y.\"><surname>LeCun</surname>, <given-names>Y.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Bengio Y.\"><surname>Bengio</surname>, <given-names>Y.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Hinton G.\"><surname>Hinton</surname>, <given-names>G.</given-names></string-name> <article-title hwp:id=\"article-title-23\">Deep learning</article-title>. <source hwp:id=\"source-22\">Nature</source> <volume>521</volume>, <fpage>436</fpage>–<lpage>444</lpage> (<year>2015</year>).</citation></ref><ref id=\"c24\" hwp:id=\"ref-24\" hwp:rev-id=\"xref-ref-24-1\"><label>24.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.24\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-24\"><string-name name-style=\"western\" hwp:sortable=\"Chen M.\"><surname>Chen</surname>, <given-names>M.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-24\">Convolutional neural networks for automated annotation of cellular cryo-electron tomograms</article-title>. <source hwp:id=\"source-23\">Nat. Methods</source> <volume>14</volume>, <fpage>983</fpage>–<lpage>985</lpage> (<year>2017</year>).</citation></ref><ref id=\"c25\" hwp:id=\"ref-25\" hwp:rev-id=\"xref-ref-25-1 xref-ref-25-2\"><label>25.</label><citation publication-type=\"book\" citation-type=\"book\" ref:id=\"2022.04.12.488077v1.25\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-25\"><string-name name-style=\"western\" hwp:sortable=\"Ronneberger O.\"><surname>Ronneberger</surname>, <given-names>O.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Fischer P.\"><surname>Fischer</surname>, <given-names>P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Brox T.\"><surname>Brox</surname>, <given-names>T.</given-names></string-name> <source hwp:id=\"source-24\">U-Net: Convolutional Networks for Biomedical Image Segmentation. in Medical Image Computing and Computer-Assisted Intervention (MICCAI)</source> (eds. <string-name name-style=\"western\" hwp:sortable=\"Navab N.\"><surname>Navab</surname>, <given-names>N.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Hornegger J.\"><surname>Hornegger</surname>, <given-names>J.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Wells W. M.\"><surname>Wells</surname>, <given-names>W. M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Frangi A. F.\"><surname>Frangi</surname>, <given-names>A. F.</given-names></string-name>) vol. <volume>9351</volume> <fpage>234</fpage>–<lpage>241</lpage> (<publisher-name>Springer International Publishing</publisher-name>, <year>2015</year>).</citation></ref><ref id=\"c26\" hwp:id=\"ref-26\" hwp:rev-id=\"xref-ref-26-1 xref-ref-26-2\"><label>26.</label><citation publication-type=\"other\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.26\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-26\"><string-name name-style=\"western\" hwp:sortable=\"Gubbins I.\"><surname>Gubbins</surname>, <given-names>I.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-25\">Classification in Cryo-Electron Tomograms</article-title>. <ext-link l:rel=\"related\" l:ref-type=\"uri\" l:ref=\"https://diglib.eg.org/handle/10.2312/3dor20191061\" ext-link-type=\"uri\" xlink:href=\"https://diglib.eg.org/handle/10.2312/3dor20191061\" hwp:id=\"ext-link-4\">https://diglib.eg.org/handle/10.2312/3dor20191061</ext-link>.</citation></ref><ref id=\"c27\" hwp:id=\"ref-27\" hwp:rev-id=\"xref-ref-27-1\"><label>27.</label><citation publication-type=\"other\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.27\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-27\"><string-name name-style=\"western\" hwp:sortable=\"Moebel E.\"><surname>Moebel</surname>, <given-names>E.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Martinez A.\"><surname>Martinez</surname>, <given-names>A.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Larivière D.\"><surname>Larivière</surname>, <given-names>D.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Ortiz J.\"><surname>Ortiz</surname>, <given-names>J.</given-names></string-name> <article-title hwp:id=\"article-title-26\">3D ConvNet improves macromolecule localization in 3D cellular cryo-electron tomograms</article-title>. (<year>2018</year>).</citation></ref><ref id=\"c28\" hwp:id=\"ref-28\" hwp:rev-id=\"xref-ref-28-1\"><label>28.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.28\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-28\"><string-name name-style=\"western\" hwp:sortable=\"Moebel E.\"><surname>Moebel</surname>, <given-names>E.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-27\">Deep learning improves macromolecule identification in 3D cellular cryo-electron tomograms</article-title>. <source hwp:id=\"source-25\">Nat. Methods</source> <volume>18</volume>, <fpage>1386</fpage>–<lpage>1394</lpage> (<year>2021</year>).</citation></ref><ref id=\"c29\" hwp:id=\"ref-29\" hwp:rev-id=\"xref-ref-29-1 xref-ref-29-2\"><label>29.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.29\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-29\"><string-name name-style=\"western\" hwp:sortable=\"Gubins I.\"><surname>Gubins</surname>, <given-names>I.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-28\">SHREC 2021: Classification in Cryo-electron Tomograms</article-title>. <source hwp:id=\"source-26\">The Eurographics Association</source> (<year>2021</year>) doi:<pub-id pub-id-type=\"doi\">10.2312/3dor.20211307</pub-id>.</citation></ref><ref id=\"c30\" hwp:id=\"ref-30\" hwp:rev-id=\"xref-ref-30-1\"><label>30.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.30\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-30\"><string-name name-style=\"western\" hwp:sortable=\"Srivastava N.\"><surname>Srivastava</surname>, <given-names>N.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Hinton G.\"><surname>Hinton</surname>, <given-names>G.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Krizhevsky A.\"><surname>Krizhevsky</surname>, <given-names>A.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Sutskever I.\"><surname>Sutskever</surname>, <given-names>I.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Salakhutdinov R\"><surname>Salakhutdinov</surname>, <given-names>R</given-names></string-name>. <article-title hwp:id=\"article-title-29\">Dropout: A Simple Way to Prevent Neural Networks from Overfitting</article-title>. <source hwp:id=\"source-27\">J Mach Learn Res</source> <volume>15</volume>, (<year>2014</year>).</citation></ref><ref id=\"c31\" hwp:id=\"ref-31\" hwp:rev-id=\"xref-ref-31-1\"><label>31.</label><citation publication-type=\"book\" citation-type=\"book\" ref:id=\"2022.04.12.488077v1.31\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-31\"><string-name name-style=\"western\" hwp:sortable=\". Ioffe S.\"><surname>. Ioffe</surname>, <given-names>S.</given-names></string-name> <source hwp:id=\"source-28\">Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift</source>. in vol. <volume>37</volume> (<publisher-name>Association for Computing Machinery</publisher-name>, <year>2015)</year>.</citation></ref><ref id=\"c32\" hwp:id=\"ref-32\" hwp:rev-id=\"xref-ref-32-1 xref-ref-32-2\"><label>32.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.32\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-32\"><string-name name-style=\"western\" hwp:sortable=\"Tegunov D.\"><surname>Tegunov</surname>, <given-names>D.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Cramer P\"><surname>Cramer</surname>, <given-names>P</given-names></string-name>. <article-title hwp:id=\"article-title-30\">Real-time cryo-electron microscopy data preprocessing with Warp</article-title>. <source hwp:id=\"source-29\">Nat. Methods</source> <volume>16</volume>, <fpage>1146</fpage>–<lpage>1152</lpage> (<year>2019</year>).</citation></ref><ref id=\"c33\" hwp:id=\"ref-33\" hwp:rev-id=\"xref-ref-33-1 xref-ref-33-2 xref-ref-33-3\"><label>33.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.33\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-33\"><string-name name-style=\"western\" hwp:sortable=\"Tegunov D.\"><surname>Tegunov</surname>, <given-names>D.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Xue L.\"><surname>Xue</surname>, <given-names>L.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Dienemann C.\"><surname>Dienemann</surname>, <given-names>C.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Cramer P.\"><surname>Cramer</surname>, <given-names>P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Mahamid J\"><surname>Mahamid</surname>, <given-names>J</given-names></string-name>. <article-title hwp:id=\"article-title-31\">Multi-particle cryo-EM refinement with M visualizes ribosome-antibiotic complex at 3.5 Å in cells</article-title>. <source hwp:id=\"source-30\">Nat. Methods</source> <volume>18</volume>, <fpage>186</fpage>–<lpage>193</lpage> (<year>2021</year>).</citation></ref><ref id=\"c34\" hwp:id=\"ref-34\" hwp:rev-id=\"xref-ref-34-1 xref-ref-34-2\"><label>34.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.34\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-34\"><string-name name-style=\"western\" hwp:sortable=\"Zivanov J.\"><surname>Zivanov</surname>, <given-names>J.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-32\">New tools for automated high-resolution cryo-EM structure determination in RELION-3</article-title>. <source hwp:id=\"source-31\">eLife</source> <volume>7</volume>, (<year>2018</year>).</citation></ref><ref id=\"c35\" hwp:id=\"ref-35\" hwp:rev-id=\"xref-ref-35-1\"><label>35.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.35\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-35\"><string-name name-style=\"western\" hwp:sortable=\"Castaño-Díez D\"><surname>Castaño-Díez</surname>, <given-names>D</given-names></string-name>. <article-title hwp:id=\"article-title-33\">The Dynamo package for tomography and subtomogram averaging: components for MATLAB, GPU computing and EC2 Amazon Web Services</article-title>. <source hwp:id=\"source-32\">Acta Crystallogr. D Struct. Biol.</source> <volume>73</volume>, <fpage>478</fpage>–<lpage>487</lpage> (<year>2017</year>).</citation></ref><ref id=\"c36\" hwp:id=\"ref-36\" hwp:rev-id=\"xref-ref-36-1 xref-ref-36-2 xref-ref-36-3\"><label>36.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.36\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-36\"><string-name name-style=\"western\" hwp:sortable=\"Hrabe T.\"><surname>Hrabe</surname>, <given-names>T.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-34\">PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis</article-title>. <source hwp:id=\"source-33\">J. Struct. Biol</source>. <volume>178</volume>, <fpage>177</fpage>–<lpage>188</lpage> (<year>2012</year>).</citation></ref><ref id=\"c37\" hwp:id=\"ref-37\" hwp:rev-id=\"xref-ref-37-1\"><label>37.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.37\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-37\"><string-name name-style=\"western\" hwp:sortable=\". Chen M.\"><surname>. Chen</surname>, <given-names>M.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-35\">A complete data processing workflow for cryo-ET and subtomogram averaging</article-title>. <source hwp:id=\"source-34\">Nat. Methods</source> <volume>16</volume>, <fpage>1161</fpage>–<lpage>1168</lpage> (<year>2019</year>).</citation></ref><ref id=\"c38\" hwp:id=\"ref-38\" hwp:rev-id=\"xref-ref-38-1\"><label>38.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.38\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-38\"><string-name name-style=\"western\" hwp:sortable=\"Danev R.\"><surname>Danev</surname>, <given-names>R.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Buijsse B.\"><surname>Buijsse</surname>, <given-names>B.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Khoshouei M.\"><surname>Khoshouei</surname>, <given-names>M.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Plitzko J. M.\"><surname>Plitzko</surname>, <given-names>J. M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Baumeister W\"><surname>Baumeister</surname>, <given-names>W</given-names></string-name>. <article-title hwp:id=\"article-title-36\">Volta potential phase plate for in-focus phase contrast transmission electron microscopy</article-title>. <source hwp:id=\"source-35\">Proc Natl Acad Sci USA</source> <volume>111</volume>, <fpage>15635</fpage>– <lpage>15640</lpage> (<year>2014</year>).</citation></ref><ref id=\"c39\" hwp:id=\"ref-39\" hwp:rev-id=\"xref-ref-39-1 xref-ref-39-2\"><label>39.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.39\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-39\"><string-name name-style=\"western\" hwp:sortable=\"Danev R.\"><surname>Danev</surname>, <given-names>R.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Tegunov D.\"><surname>Tegunov</surname>, <given-names>D.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Baumeister W\"><surname>Baumeister</surname>, <given-names>W</given-names></string-name>. <article-title hwp:id=\"article-title-37\">Using the Volta phase plate with defocus for cryo-EM single particle analysis</article-title>. <source hwp:id=\"source-36\">eLife</source> <volume>6</volume>, (<year>2017</year>).</citation></ref><ref id=\"c40\" hwp:id=\"ref-40\" hwp:rev-id=\"xref-ref-40-1 xref-ref-40-2 xref-ref-40-3 xref-ref-40-4\"><label>40.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.40\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-40\"><string-name name-style=\"western\" hwp:sortable=\"Zimmerli C. E.\"><surname>Zimmerli</surname>, <given-names>C. E.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-38\">Nuclear pores dilate and constrict in cellulo</article-title>. <source hwp:id=\"source-37\">Science</source> <volume>374</volume>, <fpage>eabd9776</fpage> (<year>2021</year>).</citation></ref><ref id=\"c41\" hwp:id=\"ref-41\" hwp:rev-id=\"xref-ref-41-1\"><label>41.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.41\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-41\"><string-name name-style=\"western\" hwp:sortable=\"Carpy A.\"><surname>Carpy</surname>, <given-names>A.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-39\">Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast)</article-title>. <source hwp:id=\"source-38\">Mol. Cell. Proteomics</source> <volume>13</volume>, <fpage>1925</fpage>–<lpage>1936</lpage> (<year>2014</year>).</citation></ref><ref id=\"c42\" hwp:id=\"ref-42\" hwp:rev-id=\"xref-ref-42-1\"><label>42.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.42\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-42\"><string-name name-style=\"western\" hwp:sortable=\"Marguerat S.\"><surname>Marguerat</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-40\">Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells</article-title>. <source hwp:id=\"source-39\">Cell</source> <volume>151</volume>, <fpage>671</fpage>–<lpage>683</lpage> (<year>2012</year>).</citation></ref><ref id=\"c43\" hwp:id=\"ref-43\" hwp:rev-id=\"xref-ref-43-1\"><label>43.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.43\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-43\"><string-name name-style=\"western\" hwp:sortable=\"Turoňová B.\"><surname>Turoňová</surname>, <given-names>B.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-41\">Benchmarking tomographic acquisition schemes for high-resolution structural biology</article-title>. <source hwp:id=\"source-40\">Nat. Commun</source>. <volume>11</volume>, <issue>876</issue> (<year>2020</year>).</citation></ref><ref id=\"c44\" hwp:id=\"ref-44\" hwp:rev-id=\"xref-ref-44-1 xref-ref-44-2 xref-ref-44-3\"><label>44.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.44\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-44\"><string-name name-style=\"western\" hwp:sortable=\"Leibundgut M.\"><surname>Leibundgut</surname>, <given-names>M.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Jenni S.\"><surname>Jenni</surname>, <given-names>S.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Frick C.\"><surname>Frick</surname>, <given-names>C.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Ban N\"><surname>Ban</surname>, <given-names>N</given-names></string-name>. <article-title hwp:id=\"article-title-42\">Structural basis for substrate delivery by acyl carrier protein in the yeast fatty acid synthase</article-title>. <source hwp:id=\"source-41\">Science</source> <volume>316</volume>, <fpage>288</fpage>–<lpage>290</lpage> (<year>2007</year>).</citation></ref><ref id=\"c45\" hwp:id=\"ref-45\" hwp:rev-id=\"xref-ref-45-1 xref-ref-45-2\"><label>45.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.45\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-45\"><string-name name-style=\"western\" hwp:sortable=\"Snowden J. S.\"><surname>Snowden</surname>, <given-names>J. S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-43\">Structural insight into Pichia pastoris fatty acid synthase</article-title>. <source hwp:id=\"source-42\">Sci. Rep</source>. <volume>11</volume>, <fpage>9773</fpage> (<year>2021</year>).</citation></ref><ref id=\"c46\" hwp:id=\"ref-46\" hwp:rev-id=\"xref-ref-46-1 xref-ref-46-2 xref-ref-46-3 xref-ref-46-4\"><label>46.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.46\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-46\"><string-name name-style=\"western\" hwp:sortable=\"Gipson P.\"><surname>Gipson</surname>, <given-names>P.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-44\">Direct structural insight into the substrate-shuttling mechanism of yeast fatty acid synthase by electron cryomicroscopy</article-title>. <source hwp:id=\"source-43\">Proc Natl Acad Sci USA</source> <volume>107</volume>, <fpage>9164</fpage>–<lpage>9169</lpage> (<year>2010</year>).</citation></ref><ref id=\"c47\" hwp:id=\"ref-47\"><label>47.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.47\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-47\"><string-name name-style=\"western\" hwp:sortable=\"Singh K.\"><surname>Singh</surname>, <given-names>K.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-45\">Discovery of a regulatory subunit of the yeast fatty acid synthase</article-title>. <source hwp:id=\"source-44\">Cell</source> <volume>180</volume>, <fpage>1130</fpage>–<lpage>1143</lpage>.e20 (<year>2020</year>).</citation></ref><ref id=\"c48\" hwp:id=\"ref-48\"><label>48.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.48\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-48\"><string-name name-style=\"western\" hwp:sortable=\"Kastritis P. L.\"><surname>Kastritis</surname>, <given-names>P. L.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-46\">Capturing protein communities by structural proteomics in a thermophilic eukaryote</article-title>. <source hwp:id=\"source-45\">Mol. Syst. Biol</source>. <volume>13</volume>, <issue>936</issue> (<year>2017</year>).</citation></ref><ref id=\"c49\" hwp:id=\"ref-49\" hwp:rev-id=\"xref-ref-49-1\"><label>49.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.49\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-49\"><string-name name-style=\"western\" hwp:sortable=\". Jenni S.\"><surname>. Jenni</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-47\">Structure of fungal fatty acid synthase and implications for iterative substrate shuttling</article-title>. <source hwp:id=\"source-46\">Science</source> <volume>316</volume>, <fpage>254</fpage>–<lpage>261</lpage> (<year>2007</year>).</citation></ref><ref id=\"c50\" hwp:id=\"ref-50\" hwp:rev-id=\"xref-ref-50-1 xref-ref-50-2 xref-ref-50-3\"><label>50.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.50\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-50\"><string-name name-style=\"western\" hwp:sortable=\"Ranjan N.\"><surname>Ranjan</surname>, <given-names>N.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-48\">Yeast translation elongation factor eEF3 promotes late stages of tRNA translocation</article-title>. <source hwp:id=\"source-47\">EMBO J</source>. <volume>40</volume>, <issue>e106449</issue> (<year>2021</year>).</citation></ref><ref id=\"c51\" hwp:id=\"ref-51\" hwp:rev-id=\"xref-ref-51-1\"><label>51.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.51\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-51\"><string-name name-style=\"western\" hwp:sortable=\"Andersen C. B. F.\"><surname>Andersen</surname>, <given-names>C. B. F.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-49\">Structure of eEF3 and the mechanism of transfer RNA release from the E-site</article-title>. <source hwp:id=\"source-48\">Nature</source> <volume>443</volume>, <fpage>663</fpage>–<lpage>668</lpage> (<year>2006</year>).</citation></ref><ref id=\"c52\" hwp:id=\"ref-52\" hwp:rev-id=\"xref-ref-52-1 xref-ref-52-2 xref-ref-52-3 xref-ref-52-4\"><label>52.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.52\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-52\"><string-name name-style=\"western\" hwp:sortable=\"Becker T.\"><surname>Becker</surname>, <given-names>T.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-50\">Structure of monomeric yeast and mammalian Sec61 complexes interacting with the translating ribosome</article-title>. <source hwp:id=\"source-49\">Science</source> <volume>326</volume>, <fpage>1369</fpage>–<lpage>1373</lpage> (<year>2009</year>).</citation></ref><ref id=\"c53\" hwp:id=\"ref-53\" hwp:rev-id=\"xref-ref-53-1 xref-ref-53-2\"><label>53.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.53\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-53\"><string-name name-style=\"western\" hwp:sortable=\"Armache J.-P.\"><surname>Armache</surname>, <given-names>J.-P.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-51\">Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome at 5.5-A resolution</article-title>. <source hwp:id=\"source-50\">Proc Natl Acad Sci USA</source> <volume>107</volume>, <fpage>19748</fpage>–<lpage>19753</lpage> (<year>2010</year>).</citation></ref><ref id=\"c54\" hwp:id=\"ref-54\" hwp:rev-id=\"xref-ref-54-1\"><label>54.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.54\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-54\"><string-name name-style=\"western\" hwp:sortable=\"Shankar V.\"><surname>Shankar</surname>, <given-names>V.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-52\">rRNA expansion segment 27Lb modulates the factor recruitment capacity of the yeast ribosome and shapes the proteome</article-title>. <source hwp:id=\"source-51\">Nucleic Acids Res</source>. <volume>48</volume>, <fpage>3244</fpage>–<lpage>3256</lpage> (<year>2020</year>).</citation></ref><ref id=\"c55\" hwp:id=\"ref-55\" hwp:rev-id=\"xref-ref-55-1\"><label>55.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.55\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-55\"><string-name name-style=\"western\" hwp:sortable=\"Fujii K.\"><surname>Fujii</surname>, <given-names>K.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Susanto T. T.\"><surname>Susanto</surname>, <given-names>T. T.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Saurabh S.\"><surname>Saurabh</surname>, <given-names>S.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Barna M\"><surname>Barna</surname>, <given-names>M</given-names></string-name>. <article-title hwp:id=\"article-title-53\">Decoding the function of expansion segments in ribosomes</article-title>. <source hwp:id=\"source-52\">Mol. Cell</source> <volume>72</volume>, <fpage>1013</fpage>–<lpage>1020</lpage>.e6 (<year>2018</year>).</citation></ref><ref id=\"c56\" hwp:id=\"ref-56\" hwp:rev-id=\"xref-ref-56-1 xref-ref-56-2\"><label>56.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.56\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-56\"><string-name name-style=\"western\" hwp:sortable=\"Greber B. J.\"><surname>Greber</surname>, <given-names>B. J.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-54\">Insertion of the Biogenesis Factor Rei1 Probes the Ribosomal Tunnel during 60S Maturation</article-title>. <source hwp:id=\"source-53\">Cell</source> <volume>164</volume>, <fpage>91</fpage>–<lpage>102</lpage> (<year>2016</year>).</citation></ref><ref id=\"c57\" hwp:id=\"ref-57\" hwp:rev-id=\"xref-ref-57-1 xref-ref-57-2\"><label>57.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.57\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-57\"><string-name name-style=\"western\" hwp:sortable=\"Wild K.\"><surname>Wild</surname>, <given-names>K.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-55\">MetAP-like Ebp1 occupies the human ribosomal tunnel exit and recruits flexible rRNA expansion segments</article-title>. <source hwp:id=\"source-54\">Nat. Commun</source>. <volume>11</volume>, <issue>776</issue> (<year>2020</year>).</citation></ref><ref id=\"c58\" hwp:id=\"ref-58\" hwp:rev-id=\"xref-ref-58-1\"><label>58.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.58\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-58\"><string-name name-style=\"western\" hwp:sortable=\"Kowalinski E.\"><surname>Kowalinski</surname>, <given-names>E.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-56\">The crystal structure of Ebp1 reveals a methionine aminopeptidase fold as binding platform for multiple interactions</article-title>. <source hwp:id=\"source-55\">FEBS Lett</source>. <volume>581</volume>, <fpage>4450</fpage>–<lpage>4454</lpage> (<year>2007</year>).</citation></ref><ref id=\"c59\" hwp:id=\"ref-59\" hwp:rev-id=\"xref-ref-59-1 xref-ref-59-2 xref-ref-59-3\"><label>59.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.59\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-59\"><string-name name-style=\"western\" hwp:sortable=\"Gold V. A.\"><surname>Gold</surname>, <given-names>V. A.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Chroscicki P.\"><surname>Chroscicki</surname>, <given-names>P.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Bragoszewski P.\"><surname>Bragoszewski</surname>, <given-names>P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Chacinska A\"><surname>Chacinska</surname>, <given-names>A</given-names></string-name>. <article-title hwp:id=\"article-title-57\">Visualization of cytosolic ribosomes on the surface of mitochondria by electron cryo-tomography</article-title>. <source hwp:id=\"source-56\">EMBO Rep</source>. <volume>18</volume>, <fpage>1786</fpage>–<lpage>1800</lpage> (<year>2017</year>).</citation></ref><ref id=\"c60\" hwp:id=\"ref-60\" hwp:rev-id=\"xref-ref-60-1 xref-ref-60-2\"><label>60.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.60\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-60\"><string-name name-style=\"western\" hwp:sortable=\"Avendaño-Monsalve M. C.\"><surname>Avendaño-Monsalve</surname>, <given-names>M. C.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Ponce-Rojas J. C.\"><surname>Ponce-Rojas</surname>, <given-names>J. C.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Funes S\"><surname>Funes</surname>, <given-names>S</given-names></string-name>. <article-title hwp:id=\"article-title-58\">From cytosol to mitochondria: the beginning of a protein journey</article-title>. <source hwp:id=\"source-57\">Biol. Chem</source>. <volume>401</volume>, <fpage>645</fpage>–<lpage>661</lpage> (<year>2020</year>).</citation></ref><ref id=\"c61\" hwp:id=\"ref-61\" hwp:rev-id=\"xref-ref-61-1 xref-ref-61-2 xref-ref-61-3\"><label>61.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.61\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-61\"><string-name name-style=\"western\" hwp:sortable=\"Tucker K.\"><surname>Tucker</surname>, <given-names>K.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Park E\"><surname>Park</surname>, <given-names>E</given-names></string-name>. <article-title hwp:id=\"article-title-59\">Cryo-EM structure of the mitochondrial protein-import channel TOM complex at near-atomic resolution</article-title>. <source hwp:id=\"source-58\">Nat. Struct. Mol. Biol</source>. <volume>26</volume>, <fpage>1158</fpage>–<lpage>1166</lpage> (<year>2019</year>).</citation></ref><ref id=\"c62\" hwp:id=\"ref-62\" hwp:rev-id=\"xref-ref-62-1\"><label>62.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.62\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-62\"><string-name name-style=\"western\" hwp:sortable=\"Lesnik C.\"><surname>Lesnik</surname>, <given-names>C.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Cohen Y.\"><surname>Cohen</surname>, <given-names>Y.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Atir-Lande A.\"><surname>Atir-Lande</surname>, <given-names>A.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Schuldiner M.\"><surname>Schuldiner</surname>, <given-names>M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Arava Y\"><surname>Arava</surname>, <given-names>Y</given-names></string-name>. <article-title hwp:id=\"article-title-60\">OM14 is a mitochondrial receptor for cytosolic ribosomes that supports co-translational import into mitochondria</article-title>. <source hwp:id=\"source-59\">Nat. Commun</source>. <volume>5</volume>, <fpage>5711</fpage> (<year>2014</year>).</citation></ref><ref id=\"c63\" hwp:id=\"ref-63\" hwp:rev-id=\"xref-ref-63-1 xref-ref-63-2\"><label>63.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.63\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-63\"><string-name name-style=\"western\" hwp:sortable=\"George R.\"><surname>George</surname>, <given-names>R.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Walsh P.\"><surname>Walsh</surname>, <given-names>P.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Beddoe T.\"><surname>Beddoe</surname>, <given-names>T.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Lithgow T\"><surname>Lithgow</surname>, <given-names>T</given-names></string-name>. <article-title hwp:id=\"article-title-61\">The nascent polypeptide-associated complex (NAC) promotes interaction of ribosomes with the mitochondrial surface in vivo</article-title>. <source hwp:id=\"source-60\">FEBS Lett</source>. <volume>516</volume>, <fpage>213</fpage>–<lpage>216</lpage> (<year>2002</year>).</citation></ref><ref id=\"c64\" hwp:id=\"ref-64\" hwp:rev-id=\"xref-ref-64-1 xref-ref-64-2\"><label>64.</label><citation publication-type=\"book\" citation-type=\"book\" ref:id=\"2022.04.12.488077v1.64\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-64\"><string-name name-style=\"western\" hwp:sortable=\"Stalling D.\"><surname>Stalling</surname>, <given-names>D.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Westerhoff M.\"><surname>Westerhoff</surname>, <given-names>M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Hege H.-C.\"><surname>Hege</surname>, <given-names>H.-C.</given-names></string-name> <source hwp:id=\"source-61\">amira: A Highly Interactive System for Visual Data Analysis. in Visualization Handbook</source> <fpage>749</fpage>–<lpage>767</lpage> (<publisher-name>Elsevier</publisher-name>, <year>2005</year>). doi:<pub-id pub-id-type=\"doi\">10.1016/B978-012387582-2/50040-X</pub-id>.</citation></ref><ref id=\"c65\" hwp:id=\"ref-65\" hwp:rev-id=\"xref-ref-65-1 xref-ref-65-2\"><label>65.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.65\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-65\"><string-name name-style=\"western\" hwp:sortable=\"Vignaud T.\"><surname>Vignaud</surname>, <given-names>T.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-62\">Stress fibres are embedded in a contractile cortical network</article-title>. <source hwp:id=\"source-62\">Nat. Mater</source>. <volume>20</volume>, <fpage>410</fpage>– <lpage>420</lpage> (<year>2021</year>).</citation></ref><ref id=\"c66\" hwp:id=\"ref-66\" hwp:rev-id=\"xref-ref-66-1 xref-ref-66-2\"><label>66.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.66\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-66\"><string-name name-style=\"western\" hwp:sortable=\"Bharat T. A. M.\"><surname>Bharat</surname>, <given-names>T. A. M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Scheres S. H. W\"><surname>Scheres</surname>, <given-names>S. H. W</given-names></string-name>. <article-title hwp:id=\"article-title-63\">Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION</article-title>. <source hwp:id=\"source-63\">Nat. Protoc</source>. <volume>11</volume>, <fpage>2054</fpage>–<lpage>2065</lpage> (<year>2016</year>).</citation></ref><ref id=\"c67\" hwp:id=\"ref-67\" hwp:rev-id=\"xref-ref-67-1\"><label>67.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.67\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-67\"><string-name name-style=\"western\" hwp:sortable=\"Wagner F. R.\"><surname>Wagner</surname>, <given-names>F. R.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-64\">Preparing samples from whole cells using focused-ion-beam milling for cryo-electron tomography</article-title>. <source hwp:id=\"source-64\">Nat. Protoc</source>. <volume>15</volume>, <fpage>2041</fpage>–<lpage>2070</lpage> (<year>2020</year>).</citation></ref><ref id=\"c68\" hwp:id=\"ref-68\" hwp:rev-id=\"xref-ref-68-1\"><label>68.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.68\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-68\"><string-name name-style=\"western\" hwp:sortable=\"Schaffer M.\"><surname>Schaffer</surname>, <given-names>M.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-65\">Optimized cryo-focused ion beam sample preparation aimed at in situ structural studies of membrane proteins</article-title>. <source hwp:id=\"source-65\">J. Struct. Biol</source>. <volume>197</volume>, <fpage>73</fpage>–<lpage>82</lpage> (<year>2017</year>).</citation></ref><ref id=\"c69\" hwp:id=\"ref-69\" hwp:rev-id=\"xref-ref-69-1\"><label>69.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.69\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-69\"><string-name name-style=\"western\" hwp:sortable=\"Mastronarde D. N\"><surname>Mastronarde</surname>, <given-names>D. N</given-names></string-name>. <article-title hwp:id=\"article-title-66\">Advanced data acquisition from electron microscopes with serialem</article-title>. <source hwp:id=\"source-66\">Microsc. Microanal</source>. <volume>24</volume>, <fpage>864</fpage>–<lpage>865</lpage> (<year>2018</year>).</citation></ref><ref id=\"c70\" hwp:id=\"ref-70\" hwp:rev-id=\"xref-ref-70-1\"><label>70.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.70\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-70\"><string-name name-style=\"western\" hwp:sortable=\"Hagen W. J. H.\"><surname>Hagen</surname>, <given-names>W. J. H.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Wan W.\"><surname>Wan</surname>, <given-names>W.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Briggs J. A. G\"><surname>Briggs</surname>, <given-names>J. A. G</given-names></string-name>. <article-title hwp:id=\"article-title-67\">Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging</article-title>. <source hwp:id=\"source-67\">J. Struct. Biol</source>. <volume>197</volume>, <fpage>191</fpage>–<lpage>198</lpage> (<year>2017</year>).</citation></ref><ref id=\"c71\" hwp:id=\"ref-71\" hwp:rev-id=\"xref-ref-71-1 xref-ref-71-2\"><label>71.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.71\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-71\"><string-name name-style=\"western\" hwp:sortable=\"Allegretti M.\"><surname>Allegretti</surname>, <given-names>M.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-68\">In-cell architecture of the nuclear pore and snapshots of its turnover</article-title>. <source hwp:id=\"source-68\">Nature</source> <volume>586</volume>, <fpage>796</fpage>–<lpage>800</lpage> (<year>2020</year>).</citation></ref><ref id=\"c72\" hwp:id=\"ref-72\"><label>72.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.72\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-72\"><string-name name-style=\"western\" hwp:sortable=\"Mastronarde D. N.\"><surname>Mastronarde</surname>, <given-names>D. N.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Held S. R\"><surname>Held</surname>, <given-names>S. R</given-names></string-name>. <article-title hwp:id=\"article-title-69\">Automated tilt series alignment and tomographic reconstruction in IMOD</article-title>. <source hwp:id=\"source-69\">J. Struct. Biol</source>. <volume>197</volume>, <fpage>102</fpage>–<lpage>113</lpage> (<year>2017</year>).</citation></ref><ref id=\"c73\" hwp:id=\"ref-73\" hwp:rev-id=\"xref-ref-73-1 xref-ref-73-2 xref-ref-73-3\"><label>73.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.73\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-73\"><string-name name-style=\"western\" hwp:sortable=\"Tang G.\"><surname>Tang</surname>, <given-names>G.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-70\">EMAN2: an extensible image processing suite for electron microscopy</article-title>. <source hwp:id=\"source-70\">J. Struct. Biol</source>. <volume>157</volume>, <fpage>38</fpage>–<lpage>46</lpage> (<year>2007</year>).</citation></ref><ref id=\"c74\" hwp:id=\"ref-74\" hwp:rev-id=\"xref-ref-74-1\"><label>74.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.74\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-74\"><string-name name-style=\"western\" hwp:sortable=\"Wu J.-Q.\"><surname>Wu</surname>, <given-names>J.-Q.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Pollard T. D\"><surname>Pollard</surname>, <given-names>T. D</given-names></string-name>. <article-title hwp:id=\"article-title-71\">Counting cytokinesis proteins globally and locally in fission yeast</article-title>. <source hwp:id=\"source-71\">Science</source> <volume>310</volume>, <fpage>310</fpage>–<lpage>314</lpage> (<year>2005</year>).</citation></ref><ref id=\"c75\" hwp:id=\"ref-75\" hwp:rev-id=\"xref-ref-75-1\"><label>75.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.75\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-75\"><string-name name-style=\"western\" hwp:sortable=\"Mitchison J. M\"><surname>Mitchison</surname>, <given-names>J. M</given-names></string-name>. <article-title hwp:id=\"article-title-72\">The growth of single cells</article-title>. <source hwp:id=\"source-72\">Exp. Cell Res</source>. <volume>13</volume>, <fpage>244</fpage>–<lpage>262</lpage> (<year>1957</year>).</citation></ref><ref id=\"c76\" hwp:id=\"ref-76\" hwp:rev-id=\"xref-ref-76-1\"><label>76.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.76\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-76\"><string-name name-style=\"western\" hwp:sortable=\"Nurse P\"><surname>Nurse</surname>, <given-names>P</given-names></string-name>. <article-title hwp:id=\"article-title-73\">Genetic control of cell size at cell division in yeast</article-title>. <source hwp:id=\"source-73\">Nature</source> <volume>256</volume>, <fpage>547</fpage>–<lpage>551</lpage> (<year>1975</year>).</citation></ref><ref id=\"c77\" hwp:id=\"ref-77\" hwp:rev-id=\"xref-ref-77-1\"><label>77.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.77\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-77\"><string-name name-style=\"western\" hwp:sortable=\"Turonova\"><surname>Turonova</surname></string-name>. <article-title hwp:id=\"article-title-74\">turonova/novaSTA: novaSTA</article-title>. <source hwp:id=\"source-74\">Zenodo</source> (<year>2020</year>) doi:<pub-id pub-id-type=\"doi\">10.5281/zenodo.3973623</pub-id>.</citation></ref><ref id=\"c78\" hwp:id=\"ref-78\" hwp:rev-id=\"xref-ref-78-1\"><label>78.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.78\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-78\"><string-name name-style=\"western\" hwp:sortable=\"Rigort A.\"><surname>Rigort</surname>, <given-names>A.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-75\">Automated segmentation of electron tomograms for a quantitative description of actin filament networks</article-title>. <source hwp:id=\"source-75\">J. Struct. Biol</source>. <volume>177</volume>, <fpage>135</fpage>–<lpage>144</lpage> (<year>2012</year>).</citation></ref><ref id=\"c79\" hwp:id=\"ref-79\" hwp:rev-id=\"xref-ref-79-1\"><label>79.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.79\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-79\"><string-name name-style=\"western\" hwp:sortable=\"Redemann S.\"><surname>Redemann</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-76\">The segmentation of microtubules in electron tomograms using Amira</article-title>. <source hwp:id=\"source-76\">Methods Mol. Biol</source>. <volume>1136</volume>, <fpage>261</fpage>–<lpage>278</lpage> (<year>2014</year>).</citation></ref><ref id=\"c80\" hwp:id=\"ref-80\" hwp:rev-id=\"xref-ref-80-1\"><label>80.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.80\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-80\"><string-name name-style=\"western\" hwp:sortable=\"Pettersen E. F.\"><surname>Pettersen</surname>, <given-names>E. F.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-77\">UCSF Chimera—a visualization system for exploratory research and analysis</article-title>. <source hwp:id=\"source-77\">J. Comput. Chem</source>. <volume>25</volume>, <fpage>1605</fpage>–<lpage>1612</lpage> (<year>2004</year>).</citation></ref><ref id=\"c81\" hwp:id=\"ref-81\" hwp:rev-id=\"xref-ref-81-1\"><label>81.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.81\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-81\"><string-name name-style=\"western\" hwp:sortable=\"Goddard T. D.\"><surname>Goddard</surname>, <given-names>T. D.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-78\">UCSF ChimeraX: Meeting modern challenges in visualization and analysis</article-title>.<source hwp:id=\"source-78\">Protein Sci</source>. <volume>27</volume>, <fpage>14</fpage>–<lpage>25</lpage> (<year>2018</year>).</citation></ref><ref id=\"c82\" hwp:id=\"ref-82\" hwp:rev-id=\"xref-ref-82-1\"><label>82.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.82\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-82\"><string-name name-style=\"western\" hwp:sortable=\"Moebel E.\"><surname>Moebel</surname>, <given-names>E.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-79\">Deep Learning Improves Macromolecules Localization and Identification in 3D Cellular Cryo-Electron Tomograms</article-title>. <source hwp:id=\"source-79\">BioRxiv</source> (<year>2020</year>) doi:<pub-id pub-id-type=\"doi\">10.1101/2020.04.15.042747</pub-id>.</citation></ref></ref-list><ref-list hwp:id=\"ref-list-2\"><title hwp:id=\"title-62\">Bibliography</title><ref id=\"c83\" hwp:id=\"ref-83\" hwp:rev-id=\"xref-ref-83-1\"><label>1.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.83\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-83\"><string-name name-style=\"western\" hwp:sortable=\"Wu J.-Q.\"><surname>Wu</surname>, <given-names>J.-Q.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Pollard T. D\"><surname>Pollard</surname>, <given-names>T. D</given-names></string-name>. <article-title hwp:id=\"article-title-80\">Counting cytokinesis proteins globally and locally in fission yeast</article-title>. <source hwp:id=\"source-80\">Science</source> <volume>310</volume>, <fpage>310</fpage>–<lpage>314</lpage> (<year>2005</year>).</citation></ref><ref id=\"c84\" hwp:id=\"ref-84\"><label>2.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.84\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-84\"><string-name name-style=\"western\" hwp:sortable=\"Mitchison J. M\"><surname>Mitchison</surname>, <given-names>J. M</given-names></string-name>. <article-title hwp:id=\"article-title-81\">The growth of single cells</article-title>. <source hwp:id=\"source-81\">Exp. Cell Res</source>. <volume>13</volume>, <fpage>244</fpage>–<lpage>262</lpage> (<year>1957</year>).</citation></ref><ref id=\"c85\" hwp:id=\"ref-85\"><label>3.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.85\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-85\"><string-name name-style=\"western\" hwp:sortable=\"Nurse P\"><surname>Nurse</surname>, <given-names>P</given-names></string-name>. <article-title hwp:id=\"article-title-82\">Genetic control of cell size at cell division in yeast</article-title>. <source hwp:id=\"source-82\">Nature</source> <volume>256</volume>, <fpage>547</fpage>–<lpage>551</lpage> (<year>1975</year>).</citation></ref><ref id=\"c86\" hwp:id=\"ref-86\"><label>4.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.86\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-86\"><string-name name-style=\"western\" hwp:sortable=\"Marguerat S.\"><surname>Marguerat</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-83\">Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells</article-title>. <source hwp:id=\"source-83\">Cell</source> <volume>151</volume>, <fpage>671</fpage>–<lpage>683</lpage> (<year>2012</year>).</citation></ref><ref id=\"c87\" hwp:id=\"ref-87\"><label>5.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.87\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-87\"><string-name name-style=\"western\" hwp:sortable=\"Carpy A.\"><surname>Carpy</surname>, <given-names>A.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-84\">Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast)</article-title>. <source hwp:id=\"source-84\">Mol. Cell. Proteomics</source> <volume>13</volume>, <fpage>1925</fpage>–<lpage>1936</lpage> (<year>2014</year>).</citation></ref><ref id=\"c88\" hwp:id=\"ref-88\"><label>6.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.88\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-88\"><string-name name-style=\"western\" hwp:sortable=\"Ranjan N.\"><surname>Ranjan</surname>, <given-names>N.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-85\">Yeast translation elongation factor eEF3 promotes late stages of tRNA translocation</article-title>. <source hwp:id=\"source-85\">EMBO J.</source> <volume>40</volume>, <fpage>e106449</fpage> (<year>2021</year>).</citation></ref><ref id=\"c89\" hwp:id=\"ref-89\"><label>7.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.89\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-89\"><string-name name-style=\"western\" hwp:sortable=\"Becker T.\"><surname>Becker</surname>, <given-names>T.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-86\">Structure of monomeric yeast and mammalian Sec61 complexes interacting with the translating ribosome</article-title>. <source hwp:id=\"source-86\">Science</source> <volume>326</volume>, <fpage>1369</fpage>–<lpage>1373</lpage> (<year>2009</year>).</citation></ref><ref id=\"c90\" hwp:id=\"ref-90\"><label>8.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.90\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-90\"><string-name name-style=\"western\" hwp:sortable=\"Armache J.-P.\"><surname>Armache</surname>, <given-names>J.-P.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-87\">Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome at 5.5- A resolution</article-title>. <source hwp:id=\"source-87\">Proc Natl Acad Sci USA</source> <volume>107</volume>, <fpage>19748</fpage>–<lpage>19753</lpage> (<year>2010</year>).</citation></ref><ref id=\"c91\" hwp:id=\"ref-91\"><label>9.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.91\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-91\"><string-name name-style=\"western\" hwp:sortable=\"Wild K.\"><surname>Wild</surname>, <given-names>K.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-88\">MetAP-like Ebp1 occupies the human ribosomal tunnel exit and recruits flexible rRNA expansion segments</article-title>. <source hwp:id=\"source-88\">Nat. Commun</source>. <volume>11</volume>, <issue>776</issue> (<year>2020</year>).</citation></ref><ref id=\"c92\" hwp:id=\"ref-92\"><label>10.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.92\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-92\"><string-name name-style=\"western\" hwp:sortable=\"Gold V. A.\"><surname>Gold</surname>, <given-names>V. A.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Chroscicki P.\"><surname>Chroscicki</surname>, <given-names>P.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Bragoszewski P.\"><surname>Bragoszewski</surname>, <given-names>P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Chacinska A\"><surname>Chacinska</surname>, <given-names>A</given-names></string-name>. <article-title hwp:id=\"article-title-89\">Visualization of cytosolic ribosomes on the surface of mitochondria by electron cryo-tomography</article-title>. <source hwp:id=\"source-89\">EMBO Rep</source>. <volume>18</volume>, <fpage>1786</fpage>–<lpage>1800</lpage> (<year>2017</year>).</citation></ref><ref id=\"c93\" hwp:id=\"ref-93\"><label>11.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.93\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-93\"><string-name name-style=\"western\" hwp:sortable=\"Mahamid J.\"><surname>Mahamid</surname>, <given-names>J.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-90\">Visualizing the molecular sociology at the HeLa cell nuclear periphery</article-title>. <source hwp:id=\"source-90\">Science</source> <volume>351</volume>, <fpage>969</fpage>–<lpage>972</lpage> (<year>2016</year>).</citation></ref><ref id=\"c94\" hwp:id=\"ref-94\"><label>12.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.94\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-94\"><string-name name-style=\"western\" hwp:sortable=\"Vignaud T.\"><surname>Vignaud</surname>, <given-names>T.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-91\">Stress fibres are embedded in a contractile cortical network</article-title>. <source hwp:id=\"source-91\">Nat. Mater</source>. <volume>20</volume>, <fpage>410</fpage>–<lpage>420</lpage> (<year>2021</year>).</citation></ref></ref-list><sec id=\"s9\" hwp:id=\"sec-44\"><title hwp:id=\"title-63\">Supplementary Information Guide</title><sec id=\"s9a\" hwp:id=\"sec-45\"><title hwp:id=\"title-64\">Supplementary Note 1</title><sec id=\"s9a1\" hwp:id=\"sec-46\"><title hwp:id=\"title-65\">Architectural description of 2D and 3D CNN, post processing steps and evaluation metrics</title><p hwp:id=\"p-96\">The general architecture of the 3D and 2D CNN is based also on the U-Net architecture<sup><xref ref-type=\"bibr\" rid=\"c1\" hwp:id=\"xref-ref-1-8\" hwp:rel-id=\"ref-1\">1</xref></sup> and consists of two consecutive symmetric paths: an encoder and a decoder path connected by skip-connections and a transitional base-layer (<xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-18\" hwp:rel-id=\"F1\">Fig. 1a</xref>), where:\\n<list list-type=\"alpha-lower\" hwp:id=\"list-1\"><list-item hwp:id=\"list-item-1\"><p hwp:id=\"p-97\">The encoder path is a sequence of alternated convolutional blocks and max-pooling layers. In turn, the convolutional blocks are a series of convolutions (kernel size <italic toggle=\"yes\">k</italic> = 3, stride <italic toggle=\"yes\">s</italic> = 1 and zero padding <italic toggle=\"yes\">p</italic> = 2) followed by a rectified linear unit (ReLU). In the case of the 3D CNN, optional batch normalization (BN) and dropout layers can be alternated (<xref ref-type=\"fig\" rid=\"fig1\" hwp:id=\"xref-fig-1-19\" hwp:rel-id=\"F1\">Fig. 1a</xref>). The max-pooling layer has fixed window size <italic toggle=\"yes\">w</italic> = 2 and stride <italic toggle=\"yes\">s</italic> = 2.</p><p hwp:id=\"p-98\">In our implementations, the initial convolutional block has a variable number of initial filters (IF) in its first convolutional layer, which must be set by the user. In all cases, subsequent convolutional blocks duplicate the number of feature maps from the previous level. The number of downsampling layers plus 1, is referred to as the network’s depth (D), which is also a hyperparameter to be set by the user. It determines the size of the receptive field (<italic toggle=\"yes\">RF</italic>) of the network (<italic toggle=\"yes\">RF</italic> = 2 ∗ <italic toggle=\"yes\">D</italic>). For the 2D CNN, the depth is a fixed value <italic toggle=\"yes\">D</italic> = 5.</p></list-item><list-item hwp:id=\"list-item-2\"><p hwp:id=\"p-99\">The base of the CNNs is the transitional convolutional layer between the encoder and decoder paths.</p></list-item><list-item hwp:id=\"list-item-3\"><p hwp:id=\"p-100\">The decoder path is symmetric with respect to the encoder path, and is a sequence of alternated convolutional and upsampling (via transposed convolution) layers<sup><xref ref-type=\"bibr\" rid=\"c2\" hwp:id=\"xref-ref-2-4\" hwp:rel-id=\"ref-2\">2</xref></sup>, with kernel size <italic toggle=\"yes\">k</italic> = 3, stride <italic toggle=\"yes\">s</italic> = 1 and padding <italic toggle=\"yes\">p</italic> = 1. For the 3D CNN, analogous optional layers can be alternated between convolutional and upsampling blocks (BN and dropout). The encoder and decoder paths’ associated parameters are denoted throughout the text as ED and DD, respectively.</p><p hwp:id=\"p-101\">Essential to the U-Net architecture are the <italic toggle=\"yes\">skip-connections</italic>, which consist of concatenating the feature maps from the encoder to the decoder path at every level. The last activation layer is in our case a sigmoid function, compatible with our choice of the Dice Loss function<sup><xref ref-type=\"bibr\" rid=\"c3\" hwp:id=\"xref-ref-3-4\" hwp:rel-id=\"ref-3\">3</xref></sup>. By definition, this is a multi- label approach, where single voxels may belong to multiple semantic classes, and which frees the user from defining additional weighting parameters to compensate for class imbalance necessary for other loss function choices such as categorical cross-entropy. In our experience, the use of Dice Loss provided better qualitative results in all cases.</p><p hwp:id=\"p-102\">All models were trained using Adam Optimiser<sup><xref ref-type=\"bibr\" rid=\"c4\" hwp:id=\"xref-ref-4-3\" hwp:rel-id=\"ref-4\">4</xref></sup>. Application of these networks to unseen tomograms requires scaling of their pixel size to match (at least approximately) that of the training data.</p></list-item></list></p></sec></sec><sec id=\"s9b\" hwp:id=\"sec-47\"><title hwp:id=\"title-66\">Data Augmentation</title><p hwp:id=\"p-103\">For the 2D CNN, tiles are randomly flipped and rotated in 90-degree increments during training to improve generalization.</p><p hwp:id=\"p-104\">Although it did not provide improved results in our hands (not shown), data augmentation for training the 3D CNN is available and optional. The number of times that each of the original training examples is augmented is given by a parameter <italic toggle=\"yes\">n<sub>DA</sub></italic> that can be set by the user. Each augmentation includes 4 different types of random volumetric transformations (applied to the original image and the labels, except in the case of noise), that we describe in what follows. Let A in <italic toggle=\"yes\">R</italic><sup>2×2×2</sup> be a single channel cubic image of side-length <italic toggle=\"yes\">n</italic> &gt; 0. Then:\\n<list list-type=\"order\" hwp:id=\"list-2\"><list-item hwp:id=\"list-item-4\"><p hwp:id=\"p-105\"><bold>Additive Gaussian noise.</bold> Given a predefined modulation factor ε &gt; 0, the Additive Gaussian noise transformation <italic toggle=\"yes\">G</italic><sub>ε</sub> is defined by <italic toggle=\"yes\">G</italic><sub>ε</sub>(<italic toggle=\"yes\">A</italic>) = <italic toggle=\"yes\">A</italic> + ε<italic toggle=\"yes\">B</italic>, where <italic toggle=\"yes\">B</italic><sub>ijk</sub> ∼ <italic toggle=\"yes\">N</italic>(0, 1) are independent and identically distributed random variables (i.i.d.r.v.), and <italic toggle=\"yes\">N</italic>(<italic toggle=\"yes\">μ</italic>, <italic toggle=\"yes\">σ</italic><sup>2</sup>) denotes a normally distributed Gaussian random variable with mean <italic toggle=\"yes\">μ</italic> and variance <italic toggle=\"yes\">σ</italic><sup>2</sup>.</p></list-item><list-item hwp:id=\"list-item-5\"><p hwp:id=\"p-106\"><bold>Salt-and-pepper noise</bold>. Given user-defined modulation factor <italic toggle=\"yes\">a</italic> &gt; 0 and a probability rate 1 &gt; <italic toggle=\"yes\">p</italic> &gt; 0, the salt-and-pepper noise transformation is defined by <italic toggle=\"yes\">S<sub>p,L</sub></italic>(<italic toggle=\"yes\">A</italic>) = <italic toggle=\"yes\">A</italic> + <italic toggle=\"yes\">a</italic> ∗ <italic toggle=\"yes\">B</italic>, where <italic toggle=\"yes\">B</italic><sub>ijk</sub> ∼ <italic toggle=\"yes\">U</italic>(<italic toggle=\"yes\">p</italic>, [−1, 1]) are i.i.d.r.v., and <italic toggle=\"yes\">U</italic>(<italic toggle=\"yes\">p</italic>, <italic toggle=\"yes\">I</italic>) denotes a uniformly distributed random variable in a real interval <italic toggle=\"yes\">I</italic>.</p></list-item><list-item hwp:id=\"list-item-6\"><p hwp:id=\"p-107\"><bold>Elastic transformation.</bold> Given coordinate entries of A that are located at a <italic toggle=\"yes\">coarse</italic> regular grid of thickness <italic toggle=\"yes\">n</italic><sub>O</sub> &gt; 1 are displaced by a random vector of i.i.d.r.v. uniform random variables. To achieve a smooth displacement vector field for all the points of the image <italic toggle=\"yes\">A</italic> (i.e. on the <italic toggle=\"yes\">fine</italic> grid), we generate the elastically transformed image as the interpolated vector field (via polynomial interpolation of order <italic toggle=\"yes\">k</italic> &gt; 0, defined by the user).</p></list-item><list-item hwp:id=\"list-item-7\"><p hwp:id=\"p-108\"><bold>Random rotation</bold>. Given a probability of rotation <italic toggle=\"yes\">p</italic> ∈ (0, 1) and angle range <italic toggle=\"yes\">α</italic> ∈ (0, 180), the image A is rotated with a probability <italic toggle=\"yes\">p</italic> with and angle in [−<italic toggle=\"yes\">α</italic>, <italic toggle=\"yes\">α</italic>], with respect to the <italic toggle=\"yes\">z</italic> −axis.</p></list-item></list></p><sec id=\"s9b1\" hwp:id=\"sec-48\"><title hwp:id=\"title-67\">3D CNN default hyperparameters</title><p hwp:id=\"p-109\">A default, starting point set of hyperparameters values for the 3D CNN for this study: <italic toggle=\"yes\">D</italic> = 2, <italic toggle=\"yes\">IF</italic> = 4, <italic toggle=\"yes\">BN</italic> = 0, <italic toggle=\"yes\">ED</italic> = 0, <italic toggle=\"yes\">DD</italic> = 0, and <italic toggle=\"yes\">n</italic><sub>01</sub> = 0. We used them to later, through testing variations on them, study the optimal combinations of hyperparameters in the different segmentation and localization tasks (<bold><xref ref-type=\"table\" rid=\"tbls1\" hwp:id=\"xref-table-wrap-1-9\" hwp:rel-id=\"T1\">Supplementary Table 1</xref></bold>).</p></sec><sec id=\"s9b2\" hwp:id=\"sec-49\"><title hwp:id=\"title-68\">2D CNN post-processing</title><p hwp:id=\"p-110\">Post-processing of the 2D CNN includes the reassembly of predicted tile segmentations, by cropping 48 px on each side to reduce artifacts around the edges. Areas of sets of tiles still overlapping are averaged after cropping. In order to remove inter-slide discontinuities, a Gaussian filter with parameter σ &gt; 0 (default σ = 5) is then applied along the z axis.</p></sec><sec id=\"s9b3\" hwp:id=\"sec-50\"><title hwp:id=\"title-69\">3D CNN post-processing</title><list list-type=\"order\" hwp:id=\"list-3\"><list-item hwp:id=\"list-item-8\"><p hwp:id=\"p-111\">Thresholding of the final probability map outputted by the last layer of the neural network (default threshold is the unbiased value 0.5, but is set by the user in the pipeline’s configuration file).</p></list-item><list-item hwp:id=\"list-item-9\"><p hwp:id=\"p-112\">Clustering of the resulting thresholded map via the <italic toggle=\"yes\">connected component labeling label</italic> algorithm, using the function <italic toggle=\"yes\">morphology.label</italic> of the <italic toggle=\"yes\">scikit-image</italic> python library<sup><xref ref-type=\"bibr\" rid=\"c5\" hwp:id=\"xref-ref-5-3\" hwp:rel-id=\"ref-5\">5</xref></sup>.</p></list-item><list-item hwp:id=\"list-item-10\"><p hwp:id=\"p-113\">Integration with other segmentation maps, <italic toggle=\"yes\">e.g.</italic> with the output of the 2D CNN. Importantly, a subsequent selection of clusters is made with respect to a <italic toggle=\"yes\">region_mask</italic> provided by the user, which consists of an auxiliary binary image defining a tomographic region of interest (Fig 1b). We then allow 3 types of options (<italic toggle=\"yes\">intersection</italic>, <italic toggle=\"yes\">contact</italic>, or <italic toggle=\"yes\">colocalization</italic>) for employing the <italic toggle=\"yes\">region_mask</italic> through the corresponding <italic toggle=\"yes\">contact_mode</italic> parameter, as follows:</p><p hwp:id=\"p-114\"><list list-type=\"alpha-lower\" hwp:id=\"list-4\"><list-item hwp:id=\"list-item-11\"><p hwp:id=\"p-115\"><italic toggle=\"yes\">contact_mode: intersection</italic></p><p hwp:id=\"p-116\">The region_mask is used to directly mask the output of point 2, by considering only the overlapping region, <italic toggle=\"yes\">e.g.</italic> including particles that are within an organelle’s mask.</p></list-item><list-item hwp:id=\"list-item-12\"><p hwp:id=\"p-117\"><italic toggle=\"yes\">contact_mode: contact</italic></p><p hwp:id=\"p-118\">Only clusters that have voxels in common with the region of interest are kept, <italic toggle=\"yes\">e.g</italic>. selecting NPC on the nuclear envelope.</p></list-item><list-item hwp:id=\"list-item-13\"><p hwp:id=\"p-119\"><italic toggle=\"yes\">contact_mode: colocalization</italic></p></list-item></list>\\n</p><p hwp:id=\"p-120\">Only clusters whose centroid is located at a given distance (<italic toggle=\"yes\">colocalization_radius</italic>) from the region of interest are kept, <italic toggle=\"yes\">e.g.</italic> particles within a distance to an organelle.</p><p hwp:id=\"p-121\">At this step, only clusters within the size range [<italic toggle=\"yes\">m</italic>, <italic toggle=\"yes\">M</italic>] are kept, where 0 ≤ <italic toggle=\"yes\">m</italic> &lt; <italic toggle=\"yes\">M</italic> ≤ ∞ are defined by the user in the configuration file.</p></list-item><list-item hwp:id=\"list-item-14\"><p hwp:id=\"p-122\">If the <italic toggle=\"yes\">calculate_motl</italic> parameter is set to <italic toggle=\"yes\">True</italic>, a list of the clusters’ centroids resulting from previous steps is saved as a 4-columns comma separated values (csv) file, where the first column is the score (cluster size), and the three remaining columns indicate the (<italic toggle=\"yes\">x</italic>, <italic toggle=\"yes\">y</italic>, <italic toggle=\"yes\">z</italic>) centroid voxel coordinates. The list is sorted according to decreasing-score.</p></list-item></list></sec><sec id=\"s9b4\" hwp:id=\"sec-51\"><title hwp:id=\"title-70\">Segmentation evaluation</title><p hwp:id=\"p-123\">Given a 3D cellular segmentation prediction <italic toggle=\"yes\">I</italic><sub><italic toggle=\"yes\">pred</italic></sub> (resulting either from the 2D or the 3D CNN), its quality can be voxel-wise evaluated, by comparison to a ground truth binary image <italic toggle=\"yes\">I<sub>GT</sub></italic>, according to different metrics:\\n<list list-type=\"order\" hwp:id=\"list-5\"><list-item hwp:id=\"list-item-15\"><p hwp:id=\"p-124\">Voxel-based precision (P) is defined by:\\n<disp-formula hwp:id=\"disp-formula-1\">\\n<alternatives hwp:id=\"alternatives-1\"><graphic xlink:href=\"488077v1_ueqn1.gif\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-16\"/></alternatives>\\n</disp-formula>\\nwhere the bars |indicate the number of entries different from 0. Thus, P is the proportion of voxels in the predicted segmentation that <italic toggle=\"yes\">truly</italic> belong to the wanted structure.</p></list-item><list-item hwp:id=\"list-item-16\"><p hwp:id=\"p-125\">Voxel-based recall (R) is defined by:\\n<disp-formula hwp:id=\"disp-formula-2\">\\n<alternatives hwp:id=\"alternatives-2\"><graphic xlink:href=\"488077v1_ueqn2.gif\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-17\"/></alternatives>\\n</disp-formula>\\nthus indicating the proportion of voxels of the ground truth segmentation that were correctly recovered by the prediction.</p></list-item><list-item hwp:id=\"list-item-17\"><p hwp:id=\"p-126\">Area under the precision-recall curve (AUPRC). Since <italic toggle=\"yes\">I</italic><sub><italic toggle=\"yes\">pred</italic></sub> is dependent on the threshold value 0 ≤ <italic toggle=\"yes\">thr</italic> ≤ 1 applied to the probability map, both precision and recall depend in ii as well, <italic toggle=\"yes\">P</italic> = <italic toggle=\"yes\">P</italic><sub><italic toggle=\"yes\">thr</italic></sub> and <italic toggle=\"yes\">R</italic> = <italic toggle=\"yes\">R</italic><sub><italic toggle=\"yes\">thr</italic></sub>. The area under the precision-recall curve (AUPRC) is the area under the parametrized curve (<italic toggle=\"yes\">P</italic><sub><italic toggle=\"yes\">thr</italic></sub>, <italic toggle=\"yes\">R</italic><sub><italic toggle=\"yes\">thr</italic></sub>) is a widely used tool to summarize the power of the method.</p></list-item><list-item hwp:id=\"list-item-18\"><p hwp:id=\"p-127\">Voxel-based F1-score (also known as Sørensen-Dice coefficient). A common metric to calculate the performance of the method by combining the P and R values is simply through the F1 score, defined by their harmonic mean:\\n<disp-formula hwp:id=\"disp-formula-3\">\\n<alternatives hwp:id=\"alternatives-3\"><graphic xlink:href=\"488077v1_ueqn3.gif\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-18\"/></alternatives>\\n</disp-formula></p><p hwp:id=\"p-128\">Notice that here no account of the dependence with respect to the <italic toggle=\"yes\">thr</italic> value is used.</p></list-item></list></p></sec><sec id=\"s9b5\" hwp:id=\"sec-52\"><title hwp:id=\"title-71\">Particle detection evaluation</title><p hwp:id=\"p-129\">In the case of particles as discrete objects that are small enough to be defined by their coordinates, the evaluation of the object detection task requires the following definitions:\\n<list list-type=\"order\" hwp:id=\"list-6\"><list-item hwp:id=\"list-item-19\"><p hwp:id=\"p-130\">Given a sorted list of predicted particles’ centroids, one by one (in that order) they are either classified as false positives (fp) or true positives (tp).</p></list-item><list-item hwp:id=\"list-item-20\"><p hwp:id=\"p-131\">A predicted particle coordinate is considered a true positive if and only if given a pre-defined tolerance radius r (<italic toggle=\"yes\">r</italic> = 10 vox for both RIBO and FAS), there exists a ground-truth coordinate not previously matched to any other predicted particle.</p></list-item><list-item hwp:id=\"list-item-21\"><p hwp:id=\"p-132\">Then the coordinate-based precision (p), recall (r), F1 score and area under the precision-recall curve (AUPRC) are defined by: <inline-formula hwp:id=\"inline-formula-1\"><alternatives hwp:id=\"alternatives-4\"><inline-graphic xlink:href=\"488077v1_inline1.gif\" hwp:id=\"inline-graphic-1\"/></alternatives></inline-formula>, and AUPRC is the area under the curve defined by the (<italic toggle=\"yes\">p, r</italic>) points when parametrized with respect to the cluster number (from 0 to all).</p></list-item></list></p></sec><sec id=\"s9b6\" hwp:id=\"sec-53\"><title hwp:id=\"title-72\">Merging several lists for ground truth construction</title><p hwp:id=\"p-133\">To account for possible duplicates when merging lists of particle coordinates from different methods (e.g. manually-, TM- and CNN-originated lists), we integrate them by restricting the distance between centroid coordinates, <italic toggle=\"yes\">i.e.</italic> with the constraint <italic toggle=\"yes\">d</italic>(<italic toggle=\"yes\">p</italic>, <italic toggle=\"yes\">p</italic>) &gt; 1, where the elliptic distance <italic toggle=\"yes\">d</italic>(<italic toggle=\"yes\">p</italic>, <italic toggle=\"yes\">p</italic>) is defined by:\\n<disp-formula hwp:id=\"disp-formula-4\">\\n<alternatives hwp:id=\"alternatives-5\"><graphic xlink:href=\"488077v1_ueqn4.gif\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-19\"/></alternatives>\\n</disp-formula>\\nfor <italic toggle=\"yes\">a</italic> = <italic toggle=\"yes\">b</italic> = 9, <italic toggle=\"yes\">c</italic> = 15, and where <italic toggle=\"yes\">p</italic> = (<italic toggle=\"yes\">p</italic><sub>p</sub>, <italic toggle=\"yes\">p</italic><sub>E</sub>, <italic toggle=\"yes\">p</italic><sub>z</sub>) and <italic toggle=\"yes\">p</italic> = (<italic toggle=\"yes\">p</italic><sub>p</sub>, <italic toggle=\"yes\">p</italic><sub>E</sub>, <italic toggle=\"yes\">p</italic><sub>z</sub>).</p><p hwp:id=\"p-134\">The choice of the elliptic distance coefficients takes on account both the effect of elongation along the z-axis in the cryo-ET images and the size of the particle in voxels of the tomogram (1 vox = 13.48 Å).</p></sec><sec id=\"s9b7\" hwp:id=\"sec-54\"><title hwp:id=\"title-73\">Supplementary Tables</title><table-wrap id=\"tbls1\" orientation=\"portrait\" position=\"float\" hwp:id=\"T1\" hwp:rev-id=\"xref-table-wrap-1-1 xref-table-wrap-1-2 xref-table-wrap-1-3 xref-table-wrap-1-4 xref-table-wrap-1-5 xref-table-wrap-1-6 xref-table-wrap-1-7 xref-table-wrap-1-8 xref-table-wrap-1-9\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/TBLS1</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">T1</object-id><object-id pub-id-type=\"publisher-id\">tbls1</object-id><label>Supplementary Table 1:</label><caption hwp:id=\"caption-16\"><title hwp:id=\"title-74\">Selected hyperparameters of DeePiCt for varying cellular structures.</title><p hwp:id=\"p-135\">Performances were calculated by cross-validation (CV) within the same domain, or in the case of microtubules and actin networks, in evaluations for generalization across cellular species with respect to expert annotations in the HeLa cell tomogram (\"). DeePiCt networks were applied to the test datasets with the following <italic toggle=\"yes\">region mask</italic> and <italic toggle=\"yes\">contact mode</italic> combinations: cytosol and intersection for ribosome and FAS; cytosol and contact for membranes; nuclear envelope and contact for NPCs; cytosolic volume and intersection for microtubules and actin filaments.</p></caption><graphic xlink:href=\"488077v1_tbls1\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-20\"/></table-wrap><table-wrap id=\"tbls2\" orientation=\"portrait\" position=\"float\" hwp:id=\"T2\" hwp:rev-id=\"xref-table-wrap-2-1 xref-table-wrap-2-2\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/TBLS2</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">T2</object-id><object-id pub-id-type=\"publisher-id\">tbls2</object-id><label>Supplementary Table 2:</label><caption hwp:id=\"caption-17\"><title hwp:id=\"title-75\">Specifications of three 3D-CNN training and prediction rounds performed for ribosome ground truth construction.</title><p hwp:id=\"p-136\">Each of the rounds in step 2 of the RIBO ground truth construction consisted of training 3 different 3D CNNs with default parameters except the IF number (<bold>Supplementary Note 1</bold>). The first training was performed on manually curated TM results of the corresponding dataset (VPP or DEF). For VPP, in the second round, the 3D CNN were trained on aggregated TM results and first round predictions; whereas the third CNN round was trained on all previous aggregated results. For DEF, the second and third round of 3D CNNs corresponded to the VPP 3D CNNs of first and second rounds, respectively.</p></caption><graphic xlink:href=\"488077v1_tbls2\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-21\"/></table-wrap><table-wrap id=\"tbls3\" orientation=\"portrait\" position=\"float\" hwp:id=\"T3\" hwp:rev-id=\"xref-table-wrap-3-1\"><object-id pub-id-type=\"other\" hwp:sub-type=\"pisa\">biorxiv;2022.04.12.488077v1/TBLS3</object-id><object-id pub-id-type=\"other\" hwp:sub-type=\"slug\">T3</object-id><object-id pub-id-type=\"publisher-id\">tbls3</object-id><label>Supplementary Table 3:</label><caption hwp:id=\"caption-18\"><title hwp:id=\"title-76\">Specifications of 3D-CNN training and prediction rounds performed for FAS ground truth construction.</title><p hwp:id=\"p-137\">Step 2 in FAS ground truth construction consisted of a single round of 3D CNN training, performed separately per dataset type (VPP and DEF). The round consisted of 3 networks with default parameters (<bold>Supplementary Note 1</bold>) except for the IF number, set to IF=4, 8, and 32. The networks were trained on an initial incomplete manual picking.</p></caption><graphic xlink:href=\"488077v1_tbls3\" position=\"float\" orientation=\"portrait\" hwp:id=\"graphic-22\"/></table-wrap></sec></sec><sec id=\"s9c\" hwp:id=\"sec-55\"><label>Supplementary Note 2</label><title hwp:id=\"title-77\">Considerations for structural analysis following CNN predictions under different imaging conditions</title><p hwp:id=\"p-138\">Cryo-ET datasets of <italic toggle=\"yes\">S. pombe</italic> utilized in this study were derived from cryo-FIB lamellae of wild-type, native, yeast cells. 20 tomograms used for comprehensive annotations were acquired with a dose- symmetric tilt scheme<sup><xref ref-type=\"bibr\" rid=\"c6\" hwp:id=\"xref-ref-6-2\" hwp:rel-id=\"ref-6\">6</xref></sup> employing similar acquisition parameters (<bold>Extended Data Tables 4-6, Online Methods</bold>), with the only difference being imaging with traditional defocus-only (DEF) or, in addition, the usage of a VPP. All RIBO and FAS particles annotated in the two datasets (DEF and VPP) were subjected to subtomogram analysis including CTF correction utilizing CTF fitting and model creation in Warp, and 3D refinements, hierarchical, and focused 3D classifications in RELION. During this structural analysis we made several observations which will be discussed in the following.</p><p hwp:id=\"p-139\">As described previously<sup><xref ref-type=\"bibr\" rid=\"c7\" hwp:id=\"xref-ref-7-2\" hwp:rel-id=\"ref-7\">7</xref>–<xref ref-type=\"bibr\" rid=\"c9\" hwp:id=\"xref-ref-9-3\" hwp:rel-id=\"ref-9\">9</xref></sup>, acquiring tomography data with a VPP emphasizes low frequencies. Pronounced low frequencies lead to improved image contrast which can facilitate data mining. With our method, more particles were detected in VPP than in DEF ground truth annotations of FAS complexes, which represent more challenging targets for data mining due to their low abundance and hollow structural signature. The increased SNR in VPP was also advantageous in hierarchical classifications of ribosomes (<bold><xref ref-type=\"fig\" rid=\"figs2\" hwp:id=\"xref-fig-17-3\" hwp:rel-id=\"F17\">Supplementary Fig. 2</xref></bold>), while no difference was apparent for FAS likely due to the low particle numbers (<bold><xref ref-type=\"fig\" rid=\"figs3\" hwp:id=\"xref-fig-18-3\" hwp:rel-id=\"F18\">Supplementary Fig. 3</xref></bold>). For VPP, all ribosomes clustered in defined averages which confirmed the high performance of DeePiCt and enabled the classification of a subset of 60S large subunits (<bold><xref ref-type=\"fig\" rid=\"figs2\" hwp:id=\"xref-fig-17-4\" hwp:rel-id=\"F17\">Supplementary Fig. 2</xref></bold>). In comparison, most DEF ribosomes from ground truth annotations or DeePiCt predictions clustered in poorly defined classes and no 60S large subunits could be separated even when classification results were improved by optimized particle poses (coordinates and orientations) after multi-particle refinement in M<sup><xref ref-type=\"bibr\" rid=\"c10\" hwp:id=\"xref-ref-10-2\" hwp:rel-id=\"ref-10\">10</xref></sup> <bold>(<xref ref-type=\"fig\" rid=\"figs4\" hwp:id=\"xref-fig-19-4\" hwp:rel-id=\"F19\">Supplementary Fig. 4</xref>, <xref ref-type=\"fig\" rid=\"figs5\" hwp:id=\"xref-fig-20-3\" hwp:rel-id=\"F20\">5</xref></bold>). Also, in classifications focused on an area close to the head of the ribosomal small subunit and at the peptide exit tunnel the additional head density was already apparent in 2D slices of the respective classes in the VPP data (<bold><xref ref-type=\"fig\" rid=\"figs6\" hwp:id=\"xref-fig-21-2\" hwp:rel-id=\"F21\">Supplementary Fig. 6</xref> and <xref ref-type=\"fig\" rid=\"figs7\" hwp:id=\"xref-fig-22-2\" hwp:rel-id=\"F22\">7</xref></bold>, respectively). Similarly, ribosomes close to ER and mitochondria that clustered into classes with adjacent membrane densities are more pronounced in VPP (<bold><xref ref-type=\"fig\" rid=\"figs8\" hwp:id=\"xref-fig-23-2\" hwp:rel-id=\"F23\">Supplementary Fig. 8</xref> and <xref ref-type=\"fig\" rid=\"figs9\" hwp:id=\"xref-fig-24-2\" hwp:rel-id=\"F24\">9</xref></bold>, respectively).</p><p hwp:id=\"p-140\">However, fine structural details that provide insights into functional configurations of molecular complexes are lost or cannot be recovered in the final reconstructions of VPP tomography data<sup><xref ref-type=\"bibr\" rid=\"c9\" hwp:id=\"xref-ref-9-4\" hwp:rel-id=\"ref-9\">9</xref></sup>. This has been suggested to be caused by inaccurate weighing<sup><xref ref-type=\"bibr\" rid=\"c9\" hwp:id=\"xref-ref-9-5\" hwp:rel-id=\"ref-9\">9</xref></sup> and a signal loss at high spatial frequencies<sup><xref ref-type=\"bibr\" rid=\"c11\" hwp:id=\"xref-ref-11-3\" hwp:rel-id=\"ref-11\">11</xref></sup>. The preservation of high-resolution information in DEF in comparison to VPP becomes especially apparent when looking at fine structural details in the subtomogram averages. Densities fitting ACPs of FAS (<bold>Extended data Fig. 8 d-e</bold>), P-site tRNAs (<bold>Extended data Fig. 8 j-k</bold>) inside the 80S ribosome and densities connecting ribosomes to an adjacent mitochondrion membrane (<bold><xref ref-type=\"fig\" rid=\"figs9\" hwp:id=\"xref-fig-24-3\" hwp:rel-id=\"F24\">Supplementary Fig. 9</xref></bold>) could only be resolved in DEF data.</p><p hwp:id=\"p-141\">We conclude that the usage of a VPP in tomography acquisition facilitated data mining including the demonstrated domain generalization of applying DeePiCt models trained in VPP to DEF, and improved 3D classifications. This makes it a valuable tool that ultimately provides benefits at the exploratory phase of structural analysis, helping the design of experiments and analysis pipelines that can eventually be performed on DEF data from which higher resolution reconstructions can be obtained.</p></sec></sec><ref-list hwp:id=\"ref-list-3\"><title hwp:id=\"title-87\">Bibliography</title><ref id=\"c95\" hwp:id=\"ref-95\"><label>1.</label><citation publication-type=\"book\" citation-type=\"book\" ref:id=\"2022.04.12.488077v1.95\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-95\"><string-name name-style=\"western\" hwp:sortable=\"Ronneberger O.\"><surname>Ronneberger</surname>, <given-names>O.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Fischer P.\"><surname>Fischer</surname>, <given-names>P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Brox T.\"><surname>Brox</surname>, <given-names>T.</given-names></string-name> <chapter-title>U-Net: Convolutional Networks for Biomedical Image Segmentation</chapter-title>. in <source hwp:id=\"source-92\">Medical Image Computing and Computer-Assisted Intervention (MICCAI)</source> (eds. <string-name name-style=\"western\" hwp:sortable=\"Navab N.\"><surname>Navab</surname>, <given-names>N.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Hornegger J.\"><surname>Hornegger</surname>, <given-names>J.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Wells W. M.\"><surname>Wells</surname>, <given-names>W. M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Frangi A. F.\"><surname>Frangi</surname>, <given-names>A. F.</given-names></string-name>) vol. <volume>9351</volume> <fpage>234</fpage>–<lpage>241</lpage> (<publisher-name>Springer International Publishing</publisher-name>, <year>2015</year>).</citation></ref><ref id=\"c96\" hwp:id=\"ref-96\"><label>2.</label><citation publication-type=\"confproc\" citation-type=\"confproc\" ref:id=\"2022.04.12.488077v1.96\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-96\"><string-name name-style=\"western\" hwp:sortable=\"Zeiler M. D.\"><surname>Zeiler</surname>, <given-names>M. D.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Krishnan D.\"><surname>Krishnan</surname>, <given-names>D.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Taylor G. W.\"><surname>Taylor</surname>, <given-names>G. W.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Fergus R.\"><surname>Fergus</surname>, <given-names>R.</given-names></string-name> <article-title hwp:id=\"article-title-92\">Deconvolutional networks</article-title>. in <conf-name>2010 IEEE Computer Society Conference on Computer Vision and Pattern Recognition</conf-name> <fpage>2528</fpage>–<lpage>2535</lpage> (IEEE, <year>2010</year>). doi:<pub-id pub-id-type=\"doi\">10.1109/CVPR.2010.5539957</pub-id>.</citation></ref><ref id=\"c97\" hwp:id=\"ref-97\"><label>3.</label><citation publication-type=\"other\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.97\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-97\"><string-name name-style=\"western\" hwp:sortable=\"Milletari F.\"><surname>Milletari</surname>, <given-names>F.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Navab N.\"><surname>Navab</surname>, <given-names>N.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Ahmadi S.-A.\"><surname>Ahmadi</surname>, <given-names>S.-A.</given-names></string-name> <article-title hwp:id=\"article-title-93\">V-Net: Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation</article-title>. in <conf-name>2016 Fourth International Conference on 3D Vision (3DV) 565–571 (IEEE, 2016)</conf-name>. doi:<pub-id pub-id-type=\"doi\">10.1109/3DV.2016.79</pub-id>.</citation></ref><ref id=\"c98\" hwp:id=\"ref-98\"><label>4.</label><citation publication-type=\"other\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.98\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-98\"><string-name name-style=\"western\" hwp:sortable=\"Kingma D. P.\"><surname>Kingma</surname>, <given-names>D. P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Ba J. Adam\"><surname>Ba</surname>, <given-names>J. Adam</given-names></string-name>: <article-title hwp:id=\"article-title-94\">A Method for Stochastic Optimization</article-title>. <ext-link l:rel=\"related\" l:ref-type=\"uri\" l:ref=\"https://arxiv.org/abs/1412.6980\" ext-link-type=\"uri\" xlink:href=\"https://arxiv.org/abs/1412.6980\" hwp:id=\"ext-link-5\">https://arxiv.org/abs/1412.6980</ext-link> (<year>2017</year>).</citation></ref><ref id=\"c99\" hwp:id=\"ref-99\"><label>5.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.99\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-99\"><string-name name-style=\"western\" hwp:sortable=\"van der Walt S.\"><surname>van der Walt</surname>, <given-names>S.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-95\">scikit-image: image processing in Python</article-title>. <source hwp:id=\"source-93\">PeerJ</source> <volume>2</volume>, <issue>e453</issue> (<year>2014</year>).</citation></ref><ref id=\"c100\" hwp:id=\"ref-100\"><label>6.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.100\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-100\"><string-name name-style=\"western\" hwp:sortable=\"Hagen W. J. H.\"><surname>Hagen</surname>, <given-names>W. J. H.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Wan W.\"><surname>Wan</surname>, <given-names>W.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Briggs J. A. G\"><surname>Briggs</surname>, <given-names>J. A. G</given-names></string-name>. <article-title hwp:id=\"article-title-96\">Implementation of a cryo-electron tomography tilt- scheme optimized for high resolution subtomogram averaging</article-title>. <source hwp:id=\"source-94\">J. Struct. Biol</source>. <volume>197</volume>, <fpage>191</fpage>–<lpage>198</lpage> (<year>2017</year>).</citation></ref><ref id=\"c101\" hwp:id=\"ref-101\"><label>7.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.101\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-101\"><string-name name-style=\"western\" hwp:sortable=\"Danev R.\"><surname>Danev</surname>, <given-names>R.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Buijsse B.\"><surname>Buijsse</surname>, <given-names>B.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Khoshouei M.\"><surname>Khoshouei</surname>, <given-names>M.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Plitzko J. M.\"><surname>Plitzko</surname>, <given-names>J. M.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Baumeister W\"><surname>Baumeister</surname>, <given-names>W</given-names></string-name>. <article-title hwp:id=\"article-title-97\">Volta potential phase plate for in-focus phase contrast transmission electron microscopy</article-title>. <source hwp:id=\"source-95\">Proc Natl Acad Sci USA</source> <volume>111</volume>, <fpage>15635</fpage>–<lpage>15640</lpage> (<year>2014</year>).</citation></ref><ref id=\"c102\" hwp:id=\"ref-102\"><label>8.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.102\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-102\"><string-name name-style=\"western\" hwp:sortable=\"Danev R.\"><surname>Danev</surname>, <given-names>R.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Tegunov D.\"><surname>Tegunov</surname>, <given-names>D.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Baumeister W\"><surname>Baumeister</surname>, <given-names>W</given-names></string-name>. <article-title hwp:id=\"article-title-98\">Using the Volta phase plate with defocus for cryo-EM single particle analysis</article-title>. <source hwp:id=\"source-96\">eLife</source> <volume>6</volume>, (<year>2017</year>).</citation></ref><ref id=\"c103\" hwp:id=\"ref-103\"><label>9.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.103\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-103\"><string-name name-style=\"western\" hwp:sortable=\"Turoňová B.\"><surname>Turoňová</surname>, <given-names>B.</given-names></string-name> <etal>et al.</etal> <article-title hwp:id=\"article-title-99\">Benchmarking tomographic acquisition schemes for high-resolution structural biology</article-title>. <source hwp:id=\"source-97\">Nat. Commun</source>. <volume>11</volume>, <issue>876</issue> (<year>2020</year>).</citation></ref><ref id=\"c104\" hwp:id=\"ref-104\"><label>10.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.104\" ref:linkable=\"yes\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-104\"><string-name name-style=\"western\" hwp:sortable=\"Tegunov D.\"><surname>Tegunov</surname>, <given-names>D.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Xue L.\"><surname>Xue</surname>, <given-names>L.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Dienemann C.\"><surname>Dienemann</surname>, <given-names>C.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Cramer P.\"><surname>Cramer</surname>, <given-names>P.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Mahamid J\"><surname>Mahamid</surname>, <given-names>J</given-names></string-name>. <article-title hwp:id=\"article-title-100\">Multi-particle cryo-EM refinement with M visualizes ribosome-antibiotic complex at 3.5 Å in cells</article-title>. <source hwp:id=\"source-98\">Nat. Methods</source> <volume>18</volume>, <fpage>186</fpage>–<lpage>193</lpage> (<year>2021</year>).</citation></ref><ref id=\"c105\" hwp:id=\"ref-105\"><label>11.</label><citation publication-type=\"journal\" citation-type=\"journal\" ref:id=\"2022.04.12.488077v1.105\" ref:linkable=\"no\" ref:use-reference-as-is=\"yes\" hwp:id=\"citation-105\"><string-name name-style=\"western\" hwp:sortable=\"Buijsse B.\"><surname>Buijsse</surname>, <given-names>B.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Trompenaars P.\"><surname>Trompenaars</surname>, <given-names>P.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Altin V.\"><surname>Altin</surname>, <given-names>V.</given-names></string-name>, <string-name name-style=\"western\" hwp:sortable=\"Danev R.\"><surname>Danev</surname>, <given-names>R.</given-names></string-name> &amp; <string-name name-style=\"western\" hwp:sortable=\"Glaeser R. M\"><surname>Glaeser</surname>, <given-names>R. M</given-names></string-name>. <article-title hwp:id=\"article-title-101\">Spectral DQE of the Volta phase plate</article-title>. <source hwp:id=\"source-99\">Ultramicroscopy</source> <volume>218</volume>, <issue>113079</issue> (<year>2020</year>).</citation></ref></ref-list></back></article>\\n'"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "engine = create_engine(\n",
    "    'databricks://token:'+os.environ['DB_TOKEN']+ \n",
    "    '@czi-shared-infra-czi-sci-general-prod-databricks.cloud.databricks.com'+\n",
    "    '?http_path=/sql/1.0/warehouses/1c4df94f2f1a6305')\n",
    "\n",
    "doi = '10.1101/2022.04.12.488077'\n",
    "\n",
    "sql1 = '''\n",
    "SELECT DISTINCT p.id, p.doi, p.pubYear, p.title, p.abstract, ft.text\n",
    "FROM scipubstore.ingestion.papers as p  \n",
    "    LEFT JOIN scipubstore.europe_pmc.articles as ft on (ft.pmc_id=p.pmcid) \n",
    "WHERE p.doi = '{}' \n",
    "'''.format(doi)\n",
    "\n",
    "sql = '''\n",
    "SELECT DISTINCT full_text_url\n",
    "FROM scipubstore.ingestion.papers as p  \n",
    "WHERE p.doi = '{}' \n",
    "'''.format(doi)\n",
    "\n",
    "with engine.connect() as con:\n",
    "    stmt = db.text(sql)\n",
    "    rs = con.execute(stmt)\n",
    "    df = pd.DataFrame(rs.fetchall(), columns=rs.keys())\n",
    "df\n",
    "\n",
    "#metadata = db.MetaData()\n",
    "#metadata.reflect(bind=engine)\n",
    "#tbls = metadata.tables\n",
    "#print(json.dumps([str(t) for t in metadata.tables], indent=4))\n",
    "\n",
    "#logs = Table(\"corpus\", MetaData(bind=engine), autoload=True)\n",
    "with engine.connect() as con:\n",
    "    stmt = db.text(sql)\n",
    "    rs = con.execute(stmt)\n",
    "    df = pd.DataFrame(rs.fetchall(), columns=rs.keys())\n",
    "\n",
    "url = df['full_text_url'][0]\n",
    "#print(select([func.count(\"*\")], from_obj=logs).scalar())\n",
    "print(url)\n",
    "\n",
    "import requests\n",
    "r = requests.get(url)\n",
    "print(r.status_code)\n",
    "print(r.headers['content-type'])\n",
    "#print(r.encoding)\n",
    "#print(r.json())\n",
    "\n",
    "xml = r.text\n",
    "xml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Translate XML to stardard format \n",
    "from alhazen.utils.nxml_text_extractor import NxmlDoc\n",
    "doc = NxmlDoc(doi, xml)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>PARAGRAPH_ID</th>\n",
       "      <th>TAG</th>\n",
       "      <th>TAG_TREE</th>\n",
       "      <th>OFFSET</th>\n",
       "      <th>LENGTH</th>\n",
       "      <th>FIG_REF</th>\n",
       "      <th>PLAIN_TEXT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>0</td>\n",
       "      <td>article-title</td>\n",
       "      <td></td>\n",
       "      <td>228</td>\n",
       "      <td>90</td>\n",
       "      <td></td>\n",
       "      <td>Convolutional networks for supervised mining o...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>1</td>\n",
       "      <td>abstract</td>\n",
       "      <td></td>\n",
       "      <td>1885</td>\n",
       "      <td>1216</td>\n",
       "      <td></td>\n",
       "      <td>AbstractCryo-electron tomograms capture a weal...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>2</td>\n",
       "      <td>title</td>\n",
       "      <td>Introduction</td>\n",
       "      <td>3270</td>\n",
       "      <td>12</td>\n",
       "      <td></td>\n",
       "      <td>Introduction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>3</td>\n",
       "      <td>p</td>\n",
       "      <td>Introduction</td>\n",
       "      <td>3282</td>\n",
       "      <td>1006</td>\n",
       "      <td></td>\n",
       "      <td>Cryo-Electron Tomography (cryo-ET) produces th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>4</td>\n",
       "      <td>p</td>\n",
       "      <td>Introduction</td>\n",
       "      <td>4288</td>\n",
       "      <td>2166</td>\n",
       "      <td></td>\n",
       "      <td>A range of available semi-automated methods fo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>124</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>61710</td>\n",
       "      <td>1434</td>\n",
       "      <td></td>\n",
       "      <td>Subtomogram averaging of the annotated particl...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>125</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>63144</td>\n",
       "      <td>1503</td>\n",
       "      <td></td>\n",
       "      <td>The analysis of DeePiCt's performance confirme...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>126</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>64647</td>\n",
       "      <td>1598</td>\n",
       "      <td></td>\n",
       "      <td>Our workflow allows easy adaptation to the seg...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>127</td>\n",
       "      <td>title</td>\n",
       "      <td>supplementary-material</td>\n",
       "      <td>66245</td>\n",
       "      <td>22</td>\n",
       "      <td></td>\n",
       "      <td>Supporting information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>128</td>\n",
       "      <td>label</td>\n",
       "      <td>supplementary-material</td>\n",
       "      <td>66270</td>\n",
       "      <td>25</td>\n",
       "      <td></td>\n",
       "      <td>Supplementary Data Tables</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>129 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          PMID  PARAGRAPH_ID            TAG  \\\n",
       "0    10.1101/2022.04.12.488077             0  article-title   \n",
       "1    10.1101/2022.04.12.488077             1       abstract   \n",
       "2    10.1101/2022.04.12.488077             2          title   \n",
       "3    10.1101/2022.04.12.488077             3              p   \n",
       "4    10.1101/2022.04.12.488077             4              p   \n",
       "..                         ...           ...            ...   \n",
       "124  10.1101/2022.04.12.488077           124              p   \n",
       "125  10.1101/2022.04.12.488077           125              p   \n",
       "126  10.1101/2022.04.12.488077           126              p   \n",
       "127  10.1101/2022.04.12.488077           127          title   \n",
       "128  10.1101/2022.04.12.488077           128          label   \n",
       "\n",
       "                   TAG_TREE  OFFSET  LENGTH FIG_REF  \\\n",
       "0                               228      90           \n",
       "1                              1885    1216           \n",
       "2              Introduction    3270      12           \n",
       "3              Introduction    3282    1006           \n",
       "4              Introduction    4288    2166           \n",
       "..                      ...     ...     ...     ...   \n",
       "124              Discussion   61710    1434           \n",
       "125              Discussion   63144    1503           \n",
       "126              Discussion   64647    1598           \n",
       "127  supplementary-material   66245      22           \n",
       "128  supplementary-material   66270      25           \n",
       "\n",
       "                                            PLAIN_TEXT  \n",
       "0    Convolutional networks for supervised mining o...  \n",
       "1    AbstractCryo-electron tomograms capture a weal...  \n",
       "2                                         Introduction  \n",
       "3    Cryo-Electron Tomography (cryo-ET) produces th...  \n",
       "4    A range of available semi-automated methods fo...  \n",
       "..                                                 ...  \n",
       "124  Subtomogram averaging of the annotated particl...  \n",
       "125  The analysis of DeePiCt's performance confirme...  \n",
       "126  Our workflow allows easy adaptation to the seg...  \n",
       "127                             Supporting information  \n",
       "128                          Supplementary Data Tables  \n",
       "\n",
       "[129 rows x 8 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = doc.build_document_dataframe()\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>PARAGRAPH_ID</th>\n",
       "      <th>TAG</th>\n",
       "      <th>TAG_TREE</th>\n",
       "      <th>OFFSET</th>\n",
       "      <th>LENGTH</th>\n",
       "      <th>FIG_REF</th>\n",
       "      <th>PLAIN_TEXT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>8</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>7384</td>\n",
       "      <td>1500</td>\n",
       "      <td></td>\n",
       "      <td>DeePiCt is based on deep learning technologies...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>19</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>13713</td>\n",
       "      <td>600</td>\n",
       "      <td>figE2</td>\n",
       "      <td>Isosurface view of organelle ground truth anno...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>20</td>\n",
       "      <td>title</td>\n",
       "      <td>Results</td>\n",
       "      <td>14313</td>\n",
       "      <td>64</td>\n",
       "      <td></td>\n",
       "      <td>Generation of comprehensive ground-truth annot...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>21</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>14377</td>\n",
       "      <td>343</td>\n",
       "      <td></td>\n",
       "      <td>The ultimate requirement for developing and be...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>22</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>14720</td>\n",
       "      <td>1169</td>\n",
       "      <td></td>\n",
       "      <td>In cryo-ET data, the complete annotation of a ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>24</td>\n",
       "      <td>title</td>\n",
       "      <td>Results</td>\n",
       "      <td>15935</td>\n",
       "      <td>74</td>\n",
       "      <td>fig2</td>\n",
       "      <td>Iterative comprehensive annotation of ground t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>25</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>16009</td>\n",
       "      <td>1401</td>\n",
       "      <td>fig2</td>\n",
       "      <td>The three columns (left to right) show: the an...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>26</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>17410</td>\n",
       "      <td>450</td>\n",
       "      <td></td>\n",
       "      <td>The resulting ground truth RIBO annotation in ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>29</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>18006</td>\n",
       "      <td>1162</td>\n",
       "      <td>figE3</td>\n",
       "      <td>a-b. Contributions of the steps applied during...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>30</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>19168</td>\n",
       "      <td>705</td>\n",
       "      <td></td>\n",
       "      <td>The final ground truth annotation of FAS in th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>31</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>19873</td>\n",
       "      <td>947</td>\n",
       "      <td></td>\n",
       "      <td>The final ground truth RIBO annotations on the...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>32</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>20820</td>\n",
       "      <td>752</td>\n",
       "      <td></td>\n",
       "      <td>Total numbers of RIBO in both acquisition type...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>36</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>22971</td>\n",
       "      <td>787</td>\n",
       "      <td></td>\n",
       "      <td>To assess the quality of the RIBO and FAS part...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>37</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>23758</td>\n",
       "      <td>313</td>\n",
       "      <td></td>\n",
       "      <td>In conclusion, using DeePiCt, we provide compr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>64</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>36155</td>\n",
       "      <td>1356</td>\n",
       "      <td></td>\n",
       "      <td>Compared to segmentation of VPP data, the 2D C...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>66</td>\n",
       "      <td>title</td>\n",
       "      <td>Results</td>\n",
       "      <td>37572</td>\n",
       "      <td>81</td>\n",
       "      <td>figE8</td>\n",
       "      <td>Ground truth annotations and 3D refinement of ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>67</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>37653</td>\n",
       "      <td>1669</td>\n",
       "      <td>figE8</td>\n",
       "      <td>a. Complete annotation of ribosomes (yellow, o...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>76</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>43455</td>\n",
       "      <td>1073</td>\n",
       "      <td></td>\n",
       "      <td>For ribosomes, the particle averages and numbe...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>82</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>46692</td>\n",
       "      <td>1418</td>\n",
       "      <td>figE10</td>\n",
       "      <td>a. Performance results of five DeePiCt network...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>117</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>57147</td>\n",
       "      <td>1016</td>\n",
       "      <td></td>\n",
       "      <td>As a demonstration of the domain generalizatio...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>123</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>61175</td>\n",
       "      <td>535</td>\n",
       "      <td></td>\n",
       "      <td>Here, we provide the first publicly available ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>126</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>64647</td>\n",
       "      <td>1598</td>\n",
       "      <td></td>\n",
       "      <td>Our workflow allows easy adaptation to the seg...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                          PMID  PARAGRAPH_ID    TAG    TAG_TREE  OFFSET  \\\n",
       "8    10.1101/2022.04.12.488077             8      p     Results    7384   \n",
       "19   10.1101/2022.04.12.488077            19      p     Results   13713   \n",
       "20   10.1101/2022.04.12.488077            20  title     Results   14313   \n",
       "21   10.1101/2022.04.12.488077            21      p     Results   14377   \n",
       "22   10.1101/2022.04.12.488077            22      p     Results   14720   \n",
       "24   10.1101/2022.04.12.488077            24  title     Results   15935   \n",
       "25   10.1101/2022.04.12.488077            25      p     Results   16009   \n",
       "26   10.1101/2022.04.12.488077            26      p     Results   17410   \n",
       "29   10.1101/2022.04.12.488077            29      p     Results   18006   \n",
       "30   10.1101/2022.04.12.488077            30      p     Results   19168   \n",
       "31   10.1101/2022.04.12.488077            31      p     Results   19873   \n",
       "32   10.1101/2022.04.12.488077            32      p     Results   20820   \n",
       "36   10.1101/2022.04.12.488077            36      p     Results   22971   \n",
       "37   10.1101/2022.04.12.488077            37      p     Results   23758   \n",
       "64   10.1101/2022.04.12.488077            64      p     Results   36155   \n",
       "66   10.1101/2022.04.12.488077            66  title     Results   37572   \n",
       "67   10.1101/2022.04.12.488077            67      p     Results   37653   \n",
       "76   10.1101/2022.04.12.488077            76      p     Results   43455   \n",
       "82   10.1101/2022.04.12.488077            82      p     Results   46692   \n",
       "117  10.1101/2022.04.12.488077           117      p     Results   57147   \n",
       "123  10.1101/2022.04.12.488077           123      p  Discussion   61175   \n",
       "126  10.1101/2022.04.12.488077           126      p  Discussion   64647   \n",
       "\n",
       "     LENGTH FIG_REF                                         PLAIN_TEXT  \n",
       "8      1500          DeePiCt is based on deep learning technologies...  \n",
       "19      600   figE2  Isosurface view of organelle ground truth anno...  \n",
       "20       64          Generation of comprehensive ground-truth annot...  \n",
       "21      343          The ultimate requirement for developing and be...  \n",
       "22     1169          In cryo-ET data, the complete annotation of a ...  \n",
       "24       74    fig2  Iterative comprehensive annotation of ground t...  \n",
       "25     1401    fig2  The three columns (left to right) show: the an...  \n",
       "26      450          The resulting ground truth RIBO annotation in ...  \n",
       "29     1162   figE3  a-b. Contributions of the steps applied during...  \n",
       "30      705          The final ground truth annotation of FAS in th...  \n",
       "31      947          The final ground truth RIBO annotations on the...  \n",
       "32      752          Total numbers of RIBO in both acquisition type...  \n",
       "36      787          To assess the quality of the RIBO and FAS part...  \n",
       "37      313          In conclusion, using DeePiCt, we provide compr...  \n",
       "64     1356          Compared to segmentation of VPP data, the 2D C...  \n",
       "66       81   figE8  Ground truth annotations and 3D refinement of ...  \n",
       "67     1669   figE8  a. Complete annotation of ribosomes (yellow, o...  \n",
       "76     1073          For ribosomes, the particle averages and numbe...  \n",
       "82     1418  figE10  a. Performance results of five DeePiCt network...  \n",
       "117    1016          As a demonstration of the domain generalizatio...  \n",
       "123     535          Here, we provide the first publicly available ...  \n",
       "126    1598          Our workflow allows easy adaptation to the seg...  "
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[df.PLAIN_TEXT.str.contains('annotation')]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Python-based PDF Text Extraction "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# papers about foundational large systems biology ML models  \n",
    "d = '/Users/gburns/Library/CloudStorage/GoogleDrive-gully.burns@chanzuckerberg.com/Shared drives/' \\\n",
    "    + 'Science/SciTech Strategy/5_Learning Sprint/4_Challenges Use Cases Outcomes/reading/foundational_models/scRNAseq models/'\n",
    "f = 'yang-2022-4-852-scBERT.pdf'\n",
    "\n",
    "d2 = '/Users/gburns/Library/CloudStorage/GoogleDrive-gully.burns@chanzuckerberg.com/Shared drives/' \\\n",
    "    + 'Science/SciTech Strategy/5_Learning Sprint/4_Challenges Use Cases Outcomes/reading//background_applications/'\n",
    "f2 = 'yang-2023-8-115-alphafold2apps.pdf'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['rangan-2023-10.1101-2023.08.18.553799.pdf', 'ke-2020-10.1101-2020.06.27.174979.pdf', 'barad-2022-10.1101-2022.01.23.477440.pdf', 'xue-2022-610-205.pdf', 'chen-2022-10.1101-2022.09.28.510016.pdf', 'de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'de_teresa-tueba-2023-20-284.pdf', 'chen-2023-30-360.pdf', 'tegunov-2020-10.1101-2020.06.05136341.pdf', 'tegunov-2021-18-186.pdf', 'barad-2023-222-e202204093.pdf', 'ke-2020-588-498.pdf', 'hoek-2022-377-543.pdf']\n"
     ]
    }
   ],
   "source": [
    "# papers about CryoET paper  \n",
    "import os\n",
    "d = '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/'\n",
    "\n",
    "# list papers in directory\n",
    "papers = [f for f in os.listdir(d) if f.endswith('.pdf')]\n",
    "print(papers)\n",
    "p = 'de_teresa-2022-10.1101-2022.04.12.488077.pdf'\n",
    "p2 = 'de_teresa-tueba-2023-20-284.pdf'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(page_content='Convolutional networks for supervised mining of molecular patterns within \\ncellular context \\n \\n', metadata={'x0': 72.0, 'x1': 543.6431884765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 471.6431884765625, 'height': 58.50422668457031, 'nlines': 3, 'fs': 19.50140889485677, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Irene de Teresa1,2*, Sara K. Goetz1,3*, Alexander Mattausch1,4, Frosina Stojanovska1,3,8, Christian E. \\nZimmerli1,5, Mauricio Toro-Nahuelpan1,6, Dorothy W.C. Cheng1,3,7, Fergus Tollervey1,3, Constantin Pape7,9, \\nMartin Beck1,5, Anna Kreshuk7, Julia Mahamid1,7+ and Judith Zaugg1,8+. \\n \\n', metadata={'x0': 72.0, 'x1': 542.76025390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76025390625, 'height': 56.59019470214844, 'nlines': 4, 'fs': 14.14754867553711, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Structural and Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, \\n', metadata={'x0': 90.0, 'x1': 542.759765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.759765625, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Germany. \\n', metadata={'x0': 108.0, 'x1': 154.4399871826172, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 46.43998718261719, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. Current address: Computer Science and Artificial Intelligence Lab, ENGIE Lab Crigen, Stains, \\n', metadata={'x0': 90.0, 'x1': 542.764404296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.764404296875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='France. \\n', metadata={'x0': 108.0, 'x1': 143.39999389648438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 35.399993896484375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Collaboration for Joint PhD Degree between EMBL and Heidelberg University, Faculty of \\n', metadata={'x0': 90.0, 'x1': 542.7587890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.7587890625, 'height': 12.382278442382812, 'nlines': 1, 'fs': 12.382278442382812, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Biosciences, Heidelberg, Germany. \\n', metadata={'x0': 108.0, 'x1': 266.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 158.760009765625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. Institute of Pharmacy and Molecular Biotechnology, Heidelberg University, Heidelberg, Germany. \\n5. Current address: Max Planck Institute of Biophysics, Department of Molecular Sociology, \\n', metadata={'x0': 90.0, 'x1': 542.7619018554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.76190185546875, 'height': 26.782318115234375, 'nlines': 2, 'fs': 13.391159057617188, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Frankfurt, Germany. \\n', metadata={'x0': 108.0, 'x1': 201.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 93.47999572753906, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. Current address: Santiago GmbH & Co. KG, Willich, Germany. \\n7. Cell Biology and Biophysics Unit, European Molecular Biology Laboratory, Heidelberg, Germany. \\n8. Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany. \\n9. Institute for Computer Science, Universität Göttingen \\n', metadata={'x0': 90.0, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.760009765625, 'height': 55.822296142578125, 'nlines': 4, 'fs': 13.955574035644531, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\n* These authors contributed equally to this work. \\n+ correspondence to: julia.mahamid@embl.de, judith.zaugg@embl.de  \\n \\n \\nAbstract. \\n', metadata={'x0': 72.0, 'x1': 386.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 314.52001953125, 'height': 89.42849731445312, 'nlines': 6, 'fs': 14.904749552408854, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cryo-electron tomograms capture a wealth of structural information on the molecular constituents of cells \\nand tissues. We present DeePiCt (Deep Picker in Context), an open-source deep-learning framework for \\nsupervised structure segmentation and macromolecular complex localization in cellular cryo-electron \\ntomography. To train and benchmark DeePiCt on experimental data, we comprehensively annotated 20 \\ntomograms of Schizosaccharomyces pombe for ribosomes, fatty acid synthases, membranes, nuclear pore \\ncomplexes, organelles and cytosol. By comparing our method to state-of-the-art approaches on this dataset, \\nwe show its unique ability to identify low-abundance and low-density complexes. We use DeePiCt to study \\ncompositionally-distinct subpopulations of cellular ribosomes, with emphasis on their contextual \\nassociation with mitochondria and the endoplasmic reticulum. Finally, by applying pre-trained networks to \\na HeLa cell dataset, we demonstrate that DeePiCt achieves high-quality predictions in unseen datasets from \\ndifferent biological species in a matter of minutes. The comprehensively annotated experimental data and \\npre-trained networks are provided for immediate exploitation by the community.  \\n \\n', metadata={'x0': 72.0, 'x1': 542.7642822265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7642822265625, 'height': 186.9459228515625, 'nlines': 13, 'fs': 14.380455603966347, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Main. \\n', metadata={'x0': 72.0, 'x1': 111.29999542236328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 39.29999542236328, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Introduction \\nCryo-Electron Tomography (cryo-ET) produces three-dimensional (3D) snapshots of cellular landscapes at \\nmolecular resolution, making it possible to investigate the structural and functional states of \\nmacromolecular complexes in their native environment, and to unveil how different macromolecular \\npopulations interact with other cellular structures1–8. With improved instrumentation, sample preparation \\nprotocols, and automation, high-quality in-cell cryo-ET data are rapidly being generated, opening the \\npossibility to conduct high-throughput studies9–12. However, due to the complex and crowded nature of the \\nintracellular milieu, together with limitations arising from cryo-ET image acquisition (low signal-to-noise \\nratio (SNR) and incomplete angular sampling) data mining of 3D cryo-ET volumes remains a major \\nbottleneck13,14. Data mining in terms of reliable identification of a relatively homogenous set of \\nmacromolecular complexes constitutes a fundamental prerequisite for structural analysis15,13,14. \\n \\nA range of available semi-automated methods for segmentation of cellular structures and localization of \\nmacromolecular complexes (from here on, particles) in cryo-ET datasets are broadly classified as template-\\nbased and template-free approaches16. Traditional Template Matching (TM)17 is a commonly applied \\ncomputational approach and is based on a point-wise numerical computation of a similarity coefficient \\n(cross-correlation) to a known template of the complex in question. TM is accurate in the localization of \\nribosomes and cytoskeletal filaments, but fails at identifying smaller or less dense particles18, and is \\ncomputationally intensive. On the other hand, current template-free methods implementing classical image \\nprocessing approaches are purposely designed for and thus limited to specific molecular shapes, or to \\nspecial configurations in which particles are associated with large cellular structures such as membranes or \\nmicrotubules16,19,20. These methods together with TM require manual inspection and are therefore laborious \\nand time-consuming. The advent of deep learning methods, and particularly convolutional neural networks \\n(CNNs)21–23, has enabled developing more generally applicable and automated approaches to accelerate the \\ntasks of segmentation and localization in cryo-ET. The first of such methods was a two-dimensional (2D) \\nCNN for semantic segmentation of ribosomes and large structures, such as cellular organelles or \\nmembranes24. However, its 2D nature makes it less suitable for particle localization, where probing the 3D \\nstructure becomes beneficial. More recently, DeepFinder, a fully supervised method based on the U-Net \\narchitecture25 for multi-class semantic segmentation, has positioned itself as the state-of-the-art in \\nautomated particle localization in both simulated and real cryo-ET datasets26–29. Remaining limitations \\ninclude the identification of less prevalent particles, as well as the interpretation of the obtained predictions \\nwithin their cellular context. Furthermore, the field lacks publicly available non-synthetic expert-annotated \\ncryo-ET data to train new models and to benchmark the available methods.  \\n \\nHere we present DeePiCt (Deep Picker in Context), our open-source software which synergizes supervised \\nconvolutional networks for segmentation of cellular compartments (organelles or cytosol) and structures \\n(membranes or cytoskeletal filaments), and localization of particles. We generated a set of comprehensively \\nannotated tomograms of wild-type S. pombe for training and benchmarking our method, which we make \\npublicly available to overcome the critical limitations arising from the absence of publicly available \\nannotated experimental datasets. From here on, we refer to the term network as the deep learning algorithm \\nitself and model as the algorithm once already trained. We provide DeePiCt models, trained on this dataset \\nwhich show high data-mining performance and can be readily applied across species and datasets.  \\n', metadata={'x0': 72.0, 'x1': 542.7666015625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7666015625, 'height': 614.1817016601562, 'nlines': 42, 'fs': 14.623373849051339, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Results \\n \\nDeePiCt: A deep-learning approach for automated compartment segmentation and particle \\nlocalization  \\nDeePiCt is based on deep learning technologies. It combines a 2D CNN for segmentation of cellular \\ncompartments and a 3D CNN for particle localization, membrane, and cytoskeletal filament annotations \\n(Fig. 1). This synergy allows for more precise particle picking and interpretation of their cellular context. \\nThe 2D and 3D CNN are adapted from the original U-Net architecture25 (Fig. 1a, Supplementary Note 1). \\nSince cellular compartments are easily recognizable in 2D image data, the compartment segmentation \\nnetwork operates in 2D, which has the advantage of requiring little training data. The 2D CNN employs a \\nfixed depth (D) of 5 (4 max-pooling layers) and 16 initial filters (IF) (Fig. 1a). Our implementation of the \\n3D CNN allows multi-label learning (Supplementary Note 1) and adjustable architectural parameters D, \\nnumber of IF, a batch normalization layer (BN), and the dropout parameter in the encoder and decoder \\npaths (ED and DD, respectively) (Fig. 1a). These parameters can be set according to the amount and quality \\nof training data, and the shape complexity, size and abundance of the particle of interest as suggested in \\nSupplementary Table 1. In general, larger particles benefit from larger values of D to increase the \\nnetwork’s receptive field, and particles with a low-density print (low SNR with respect to the surrounding \\ncontext) require larger IF values. The remaining optional layers, BN and dropout, are well known \\ntechniques in computer vision to avoid overfitting30,31. \\n \\nFor training, each network requires tomograms and corresponding 3D binary masks, which represent the \\nsegmentation of the structures of interest, e.g. organelle segmentations for the 2D U-Net and spheres \\nrepresenting particles for the 3D U-Net (Fig. 1b.1). The raw input tomograms are optionally preprocessed \\nusing an amplitude spectrum equalization filter (Fig. 1b.2, Extended Data Fig. 1) to enhance image \\ncontrast by matching the amplitude spectrum of the input tomogram to a preselected target amplitude \\nspectrum distribution (Extended Data Fig. 1). Both CNNs use the Adam optimizer algorithm and the Dice \\nLoss function for training (Supplementary Note 1). In the 2D network, tiles are randomly flipped and \\nrotated in 90-degree increments during training to improve generalization. For the 3D CNN, we \\nimplemented a data augmentation strategy by applying a number of optional random transformations to the \\ninput images (Supplementary Note 1). In our experience, more than 700 annotated instances are required \\nfor particle learning in the 3D CNNs, while about 5 tomograms are sufficient for membrane segmentation. \\nIn the case of 2D CNNs, 6 fully segmented tomograms are enough for training networks identifying either \\na cytosol class or a single class including all cellular organelles. \\n \\nFor predicting, the trained networks receive as input unseen tomograms pre-processed in the same way as \\nthe training data, and output 3D probability maps that are subsequently post-processed (Fig. 1b.3). \\nAutomated post-processing for the 2D network includes the combination of predicted 2D slices into a 3D \\nvolume, application of a 1D Gaussian filter along the z-axis for smoothing the predictions across the z \\nslices, and thresholding (user-definable, default=0.75) (Extended Data Fig. 2a-c). The 3D CNN post-\\nprocessing entails the generation of a predicted segmentation. To that end, the 3D CNN output is \\nthresholded by a probability value set by the user and then clustered. The clustered output can be integrated \\nwith contextual information from a binary map representing a tomographic region (e.g. the cytosol \\nsegmentation from the 2D CNN). The mode of integration can be chosen among three different options: \\nintersection, contact, or colocalization, depending on the users’ specific application (Fig. 1b.4, \\n', metadata={'x0': 72.0, 'x1': 543.0001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 471.0001220703125, 'height': 645.6217041015625, 'nlines': 44, 'fs': 14.673220547762783, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Note 1, Supplementary Fig. 1). In the case of particle localization, the clusters’ centroids \\nare used to obtain a list of the particle coordinates; their orientations and structural features can then be \\nobtained by subtomogram analysis in external softwares (e. g. Warp32, M33, RELION34, Dynamo35, \\nPyTom36 or EMAN237). For more details on the method, we refer to Supplementary Note 1 and our github \\nrepository (Code and data availability). \\n \\nGeneration of comprehensive ground-truth annotations in S. pombe \\n', metadata={'x0': 72.0, 'x1': 542.7622680664062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76226806640625, 'height': 100.75360107421875, 'nlines': 7, 'fs': 14.39337158203125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The ultimate requirement for developing and benchmarking any deep neural network is the existence of a \\nground truth (gt) non-synthetic dataset. Since this was not available at the outset of this study, we first \\ncreated a comprehensive annotation of cellular features in wild-type S. pombe representing diverse \\nstructures, sizes and abundances. \\n \\nIn cryo-ET data, the complete annotation of a given cellular structure or particle species is a challenging \\nprocess, even for large compartments or complexes with a clear structural signature like the ribosome. To \\nachieve the most comprehensive annotation possible, we devised an iterative workflow to mine patterns in \\nthree different datasets (Online Methods). In a set of 10 tomograms acquired by combining defocus and \\nVolta potential phase plate (VPP38,39) and a second set of 10 tomograms using defocus only (DEF), we \\nannotated ribosomes (RIBO), fatty acid synthases (FAS), membranes, organelles and cytoplasm  (Fig. 2, \\nExtended Data Tables 1-3). For the nuclear pore complex (NPC), given its structural flexibility, low \\nabundance (on average 3 per tomogram), and large size, a third dataset denoted by DEF* was used40. It \\nconsists of 127 tomograms featuring a total of 2,830 NPC subunits (~354 NPCs) to ensure enough training \\ndata and is composed of two subsets: one of higher and one of lower quality classified based on lamella \\nthickness and tilt-series alignment error. A detailed description of the ground truth construction is available \\nin the Online Methods.  \\n \\nThe resulting ground truth RIBO annotation in VPP consisted of 25,311 particles. An initial TM annotation \\ncontributed 61.6% (1,559 particles per tomogram (ppt), on average) of the final ground truth, whereas \\niterative rounds of DeePiCt on the incomplete ground truth contributed to a total of additional 21.36% (541 \\nppt on average) and final manual annotations to 17.05% (431 ppt on average) (Fig. 2j, Extended Data Fig. \\n3a,e, Extended Data Table 2). \\n \\nThe final ground truth annotation of FAS in the 10 VPP datasets amounts to 731 particles, from which \\n58.96% (43 ppt on average) constituted the initial annotations from manual picking, as TM failed. The \\ncombined predictions of DeePiCt on the incomplete ground truth contributed to 21.88% (16 ppt on average). \\nFor FAS we performed only one round of DeePiCt using the initial manual annotations, since we observed \\na high false positive rate in subsequent rounds, which is likely a consequence of the natural lower abundance \\nof FAS in the tomograms and its low imprint density. The final manual picking added a significant fraction \\nof 19% (14 ppt on average) (Extended Data Fig. 3c,e, Extended Data Table 3). \\n \\nThe final ground truth RIBO annotations on the DEF dataset sum a total of 25,901 particles across the 10 \\ntomograms, from which only 19.23% (498 ppt on average) were included in the initial TM annotation (step \\n1), 19.39% (502 ppt on average) were a contribution of DeePiCt on incomplete ground truth, and 61.36% \\n(1,590 ppt on average) were manually identified (Fig. 2k-m, Extended Data Fig. 3b,e), highlighting the \\nvalue of our carefully curated annotations. In the case of FAS, the ground truth annotations across the 10 \\n', metadata={'x0': 72.0, 'x1': 542.7744140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7744140625, 'height': 535.9059448242188, 'nlines': 37, 'fs': 14.483944454708615, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"tomograms amount to 366 particles, from which 48.63% (18 ppt on average) came from the initial manual \\nannotations (as TM failed), 36.88% (14 ppt on average) from DeePiCt trained on incomplete ground truth, \\nand 14.48% (5 ppt on average) from the final manual picking (Fig. 2k-m, Extended data Fig. 3d,f). \\nOverall, a final round of manual picking for RIBO ground truth annotation was notably more crucial in \\nDEF compared to VPP.  \\n \\nTotal numbers of RIBO in both acquisition types are comparable, while for FAS fewer particles were \\ndetected in DEF likely due to the lower SNR despite the application of an equalization filter (Extended \\nData Fig. 3, Extended Data Table 2-3, Supplementary Note 2). FAS annotations were similarly \\ndistributed among the ground truth construction steps in the two acquisition conditions (Extended Data \\nFig. 3). Comparison to previous proteomics studies41,42 shows that our annotations of fully assembled FAS \\nin cryo-electron tomograms is in agreement with levels of FAS-𝛼 and FAS-𝛽 quantified by means of mass \\nspectrometry, while the abundance of cytosolic ribosomal proteins detected by mass spectrometry are \\nunderestimated (Extended Data Fig. 4).  \\n \\nTo assess the quality of the RIBO and FAS particle annotations in the VPP and DEF datasets, we processed \\nthe tomograms in Warp and performed structural 3D classification and refinement of the subtomograms in \\nRELION (Extended Data Tables 4-5, Supplementary Note. 2, Supplementary Fig. 2-4). Subtomogram \\naverages of RIBO particles detected only by DeePiCt ('tp new'), represent 80S ribosomes, consistent with \\nmaps of all cytosolic RIBO particles for both VPP and DEF datasets (Fig. 2l-m, blue and gray, \\nrespectively). 3D classifications performed in both data types for FAS demonstrated that the 3D CNN \\ntogether with manual annotations were capable of recovering these challenging shell-like structures \\nindependent of the data acquisition type (Supplementary Note 2, Supplementary Fig. 3).  \\n \\nIn conclusion, using DeePiCt, we provide comprehensive annotations for large macromolecular complexes \\nand cellular structures in S. pombe cryo-electron tomograms acquired with VPP and DEF, which set the \\nground for benchmarking the method’s performance and for future developments in particle detection \\napproaches. \\n \\nPerformance analysis and hyperparameter tuning of the DeePiCt workflow \\n \\nPerformance analysis of 2D CNN in VPP \\nFor the 2D CNN, basic hyperparameter tuning had close to no effect and a fixed architecture produced \\nsufficiently good segmentations. Hence, no extensive hyper-parameter tuning was performed. The 2D \\nCNN's performance was evaluated on 10 manually annotated ground-truth VPP tomograms in a 5-fold \\ncross-validation (CV). For each CV fold, two tomograms were split off from the training data to evaluate \\nmodel performance during training. Two binary segmentation tasks were evaluated: segmentation of all \\norganelles, i.e. all membrane-enclosed organelles and the nucleoplasm, and segmentation of the cytosol. \\nThe results show that the 2D CNN achieves high areas under the precision-recall curve (AUPRC, \\nSupplementary Note 1), with a median AUPRC of 0.98 for cytosol and 0.92 for organelles (Fig. 3a, \\nExtended Data Fig. 5). \\n \\nThe 2D CNN showed an average false positive rate (FPR) of 0.054 for cytosol and 0.024 for organelle \\nsegmentation, with the post-processing step providing marginal improvements (reducing FPR to 0.049 for \\n\", metadata={'x0': 72.0, 'x1': 542.7688598632812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76885986328125, 'height': 639.345947265625, 'nlines': 44, 'fs': 14.530589710582387, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cytosol and 0.021 for organelles, Extended Data Fig. 2). In addition to the two segmentation tasks \\ndescribed here, the 2D CNN has the potential to segment individual organelle types when provided with \\nsufficient amounts of training data (Extended Data Table 1). Operating in 2D rather than 3D results in \\nonly little memory requirement (<1GB at the default batch size) and short training time (~15 minutes to \\ntrain on ten tomograms using a Nvidia V100 GPU; Online Methods). \\n \\nHyperparameter tuning for the 3D CNN \\nThe 3D CNN is built on a flexible architecture for which we performed hyperparameter tuning. We \\ncharacterized the effect of varying the adjustable hyperparameters (Online Methods) on the DeePiCt \\nworkflow using a CV approach for RIBO, FAS and membranes in the VPP dataset, and for NPC in the \\nDEF* dataset.  \\n \\nIn each case we combined the 3D CNN segmentation of the target structure with an appropriate region \\nmask (Supplementary Fig. 1, Supplementary Table 1) and determined the 3D CNN hyperparameter \\ncombination that optimizes the task of localization or segmentation (Extended Data Fig. 6, \\nSupplementary Table 1). Our results show that the best hyperparameter selection for the 3D CNN is \\nstructure-dependent, and not only related to the size of the complex or to the receptive field of the network \\n(a function of the depth parameter D), but also to the particle’s density, symmetry and abundance, among \\nothers. Therefore, the architecture flexibility of our implementation of the 3D CNN is essential for the \\naccurate segmentation of a diverse set of biological structures. In particular, BN shortens the learning speed \\n(and notably improves the performance results in the case of NPCs and FAS). Dropout layers and a number \\nof data augmentation strategies (Gaussian noise, salt-and-pepper noise, rotations, elastic transformations, \\nsee Supplementary Note 1) did not improve performance. \\n \\nPerformance of DeePiCt in the same-domain setting \\nOnce the 3D CNN architectures which optimized the tasks of particle localization (for FAS and RIBO) and \\nstructure segmentation (membranes and NPC) were determined, we analyzed the performance of the \\nDeePiCt workflow in the same-domain setting (i.e. training and testing in the same domain). The 3-fold \\nCV tests in VPP (Fig. 3b, Extended Data Fig. 7a-c), showed that it achieved a performance F1 score \\nbetween 0.68-0.80 for RIBO (median F1 of 0.79), 0.21-0.64 for FAS (median F1 of 0.46) and 0.58-0.90 for \\nmembranes (median voxel-F1 of 0.71). The results in DEF* for NPC segmentation (Fig. 3c) show that, \\neven if the network is trained on the whole DEF* dataset (23% of which is high quality data, Online \\nMethods), the performance of the method is clearly different in the high- versus lower-quality DEF* \\ntomograms: with a median voxel-F1 of 0.47 in high-quality data, while a median voxel-F1 of 0.19 in lower-\\nquality DEF* tomograms. \\n \\nComparison of DeePiCt to state-of-the-art particle localization tools \\nTo benchmark DeePiCt in the particle localization task, we compared it to TM and DeepFinder in the tasks \\nof RIBO and FAS in the VPP dataset (Fig. 3d,e). Following the suggestions by the authors28, DeepFinder \\nwas trained in a multi-class fashion, where both particles were learnt simultaneously by a single network. \\nFor the training, we used the same 3-fold CV splitting of the VPP datasets as for DeePiCt. In RIBO \\nlocalization, DeepFinder achieved a median F1 of 0.83, comparable to DeePiCt. For FAS localization, \\nDeepFinder achieved a median F1 of 0.11 (significantly below DeePiCt, t-test p-value of 0.007; Fig. 3d). \\nIt is worth mentioning that single-class DeepFinder networks tested for FAS localization completely failed \\n', metadata={'x0': 72.0, 'x1': 542.7683715820312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76837158203125, 'height': 639.10595703125, 'nlines': 44, 'fs': 14.525135387073863, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='by not localizing any particles. DeePiCt needs ~17h for training vs ~3h for DeepFinder, while predicting \\nand post processing is equally fast for both (~500 clusters/minute) (Online Methods). Furthermore, \\nDeePiCt’s trained networks (models) for all structures mentioned in this work are open source and publicly \\navailable (Code and data availability).  \\n \\nFinally, the analysis of TM performance was done with the raw output with respect to the top 2000 peaks \\nin the case of RIBO and to the top 1000 peaks in the case of FAS. TM also takes several hours (s. parameters \\nin Online Methods) and our results confirm that TM has lower performance than both deep-learning \\nmethods for the localization of RIBO and FAS, completely failing at the latter (Fig. 3e).  \\n \\nDomain generalization of DeePiCt across image acquisition conditions \\nThe DEF data is characterized by a lower SNR and lower contrast in the image, as compared to VPP \\n(Supplementary Note 2). This makes the task of particle localization and structure segmentation more \\nchallenging, but has the advantage that the resulting subtomogram averages generally provide better \\nresolutions43. The comprehensive ground truth construction in DEF enabled us to study the generalization \\npotential of the DeePiCt workflow across image acquisition conditions, from VPP to DEF data. To that \\nend, we predicted cytosol, organelles, membranes, FAS and RIBO in 10 DEF tomograms, with networks \\ntrained in the yeast VPP data (Fig. 3f-g).  \\n \\nCompared to segmentation of VPP data, the 2D CNN shows slightly poorer performance on DEF data for \\ncytosol (Fig. 3a,f). By employing amplitude spectrum equalization for the 2D CNN, the domain \\ngeneralization capabilities greatly improved for both cytosol (from initial median AUPRC=0.82 to median \\nAUPRC=0.95) and organelles (from AUPRC=0.42 to 0.74) (Fig. 3f, Extended Data Fig. 5). For DeePiCt, \\nthe performance scores in this domain generalization setting also showed a decay with respect to the same \\ndomain setting: with a median F1 of 0.59 for RIBO, a median F1 of 0.15 for FAS, and a median voxel-F1 \\nof 0.41 for membranes (Fig. 3g). Importantly, as in the case of the 2D CNN, the spectrum equalization filter \\nin the preprocessing step improved the domain generalization results (Extended Data Fig. 7d-f). \\nFurthermore, we confirmed that using the cytosol predictions as a region mask to clean particle annotations \\nalso improved the performances (Extended Data Fig. 7g-h). An example of DeePict segmentation outcome \\nin the cross-domain setting is shown in Fig. 4a-b and visually resembles the ground truth annotation \\n(Extended Data Fig. 8a). The DeePiCt predictions and the ground truth annotations enabled us to analyze \\nthe DEF tomograms in the same way as we had done for the VPP data, by downstream 3D structural \\nclassification and refinement (Fig. 5).  \\n \\nDeePiCt predictions result in high quality subtomogram averages  \\nIn the case of FAS in the DEF dataset, fewer particles were predicted than in the ground truth (Extended \\nData Tables 5, 6). Nevertheless, the subtomogram average of the detected particles revealed the two half \\ndomes of the barrel-shaped type I FAS with applied D3 symmetry, consistent with the ground truth average \\nand published structures of this fungal 2.6-megadalton heterododecameric (α6β6) complex from other yeast \\nspecies (Fig. 4c-d, Extended Data Fig. 8b-e). The phosphopantetheine transferase (PPT) domains required \\nfor activation of FAS could be observed along the equatorial plane of the complex. In addition, three \\nequatorial densities were resolved that could not be assigned based on published structures. Three densities \\ninside each half dome connected to the central α wheel and in close proximity to the ketosynthase (KS) fit \\nwith the acyl carrier protein (ACP) of Saccharomyces cerevisiae44 (Fig. 4d, PDB: 2uv8) and Pichia \\n', metadata={'x0': 72.0, 'x1': 542.7681274414062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76812744140625, 'height': 640.3059692382812, 'nlines': 44, 'fs': 14.55240839177912, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='pastoris45 (Extended Data Fig. 8c, EMD-12139). The ACP shuttles the growing acyl-chain between the \\ndifferent catalytic sites, and its localization has been suggested in connection with the activity of the whole \\nmulti-enzyme complex44,46–49. Here, S. pombe FAS complexes within FIB-lamellae prepared from \\nexponentially growing cells revealed a specific ACP site, which had not been observed by cryo-ET and \\nthus in its native state before (Fig. 4d, Extended Data Fig. 8, Supplementary Note 2). \\n \\nFor ribosomes, the particle averages and numbers purely derived from DeePiCt predictions are comparable \\nto the ground truth annotations in DEF (Extended Data Tables 5, 6; Supplementary Note 2, \\nSupplementary Fig. 4, 5) complementing the performance analysis results described above. 3D \\nrefinements of these lamella-derived ribosomal particles from both ground truth and DeePiCt resulted in \\nsubtomogram averages with the nominal resolutions of 11 Å and 15 Å after multi-particle refinement in \\nM33, respectively (Fig. 4e, Extended Data Fig. 8f, Extended Data Tables 5, 6). Hierarchical 3D \\nclassification revealed in both datasets a well-aligned class that was further refined in M to subnanometer \\nresolutions of 9.3 Å and 9.4 Å for ground truth and DeePiCt predictions, respectively (Fig. 4f-h, Extended \\nData Fig. 8i-k, Extended Data Tables 5, 6, Supplementary Note 2, Supplementary Fig. 4, 5). This \\nallowed the identification of tRNA occupying the P-site of the peptidyl transferase center (PTC) as well as \\nthe L1 stalk facing the E-site (Fig. 4g-h, Extended Data Fig. 8j-k).  \\n \\nDeePiCt-predicted ribosomes reveal functional sub-populations \\nThe large number of particles that DeePiCt localized in DEF in a high throughput manner allows for \\nexamining subpopulations of macromolecular complexes that are functionally distinct by downstream \\nsubtomogram analysis. Further classification of all DeePiCt-predicted ribosomes focused on the head of \\nthe 40S small subunit revealed a subset with additional densities close to the head and at the exit tunnel \\n(Fig. 4i, Supplementary Fig. 6). This class was also detected in VPP and in DEF ground truth datasets \\n(Extended Data Tables 4-6). In the DEF ground truth dataset, densities for P- and E-site tRNAs and the \\nL1 stalk facing inwards are resolved (Extended Data Fig. 9a-e). The ribosome-bound ATPase eEF3 from \\nS. cerevisiae50 fitted well into the additional head density (CC 0.8972, EMD-12062, Fig. 5i, Extended Data \\nFig. 9 a-c). During translation, this eukaryotic elongation factor facilitates binding of a new tRNA to the \\nA-site of the ribosome via the ternary aminoacyl-tRNA–eEF1A–GTP complex50,51. In addition, unassigned \\ndensity at the ribosomal exit tunnel is observable for this class. Focused classification of all DeePiCt-\\npredicted ribosomes at this location revealed a subtomogram average fitting the S. cerevisiae ribosome52 \\n(CC 0.9657, EMD-1667) with the rRNA expansion segment ES27L in a particular configuration53 (CC \\n0.7938, PDB-3izd) connecting to an additional density close to the ribosomal exit tunnel (Fig. 4j, \\nSupplementary Fig. 7). ES27L plays a role in translation fidelity and recruits enzymes to the ribosomal \\nexit tunnel, such as the methionine aminopeptidase (MetAP) that co-translationally processes the nascent \\npeptide chain54,55. The nuclear export factor Arx1, which is released during cytosolic ribosomal 60S \\nmaturation in S. cerevisiae56, and its human homologue Ebp1, a translation regulator57,58, similarly bind at \\nlocations of the observed extra density (Fig. 4j, Extended Data Fig. 9h). These binding factors recruit the \\nflexible rRNA scaffold ES27L and cover the ribosomal exit tunnel with their MetAP-like folds. This \\nstructural class was also detected in DEF gt and VPP gt datasets (Extended Data Fig. 9f-j, Extended Data \\nTables 4-5).  \\n \\nThus, DeePiCt predictions combined with structural analysis revealed functional subpopulations of \\nribosomes. Mapping them back into the tomograms revealed their 3D spatial distributions in relation to \\n', metadata={'x0': 72.0, 'x1': 542.766845703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.766845703125, 'height': 638.5902099609375, 'nlines': 44, 'fs': 14.51341386274858, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"other macromolecules such as FAS and organelles in the cellular context (Fig. 4b). \\n \\nDeePiCt's combination of organelle segmentation and high-throughput particle localization \\nenables structural insights into ribosome-mitochondria association  \\nOne advantage of DeePiCt over state-of-the-art tools is that it allows studying particle populations within \\ntheir cellular context, by recovering particles in close proximity to specific organelles selected from \\npredictions by the 2D CNN. We recovered cytosolic ribosomes within 25 nm distance to predicted \\nendoplasmic reticulum (ER) and mitochondria from 7 and 3 DEF tomograms, respectively. Using \\nsubtomogram analyses for both subsets, two classes could be separated, one with and one without a \\nmembrane density (Supplementary Fig. 8-9). In the first class, ribosomes associated with the membrane \\nin a specific orientation, whereas they were randomly oriented in the second class. The latter class likely \\narises from the highly crowded nature of the S. pombe cytoplasm (Fig. 4k-l), which in the case of DeePiCt \\npredictions in DEF data, is also affected by imperfect organelle segmentations (Fig. 4b). ER-bound \\nribosomes faced the membrane with the ribosomal exit tunnel in agreement with published structures from \\nother species such as S. cerevisiae1,52,59 (Extended Data Fig. 9 k-n, Extended Data Table 6). \\nMitochondria-bound ribosomes, which posed a particular challenge to structural analysis in previous \\nstudies59,60, were found to interact with the membrane at an angular offset of around 35° in comparison to \\nthe ER-bound ribosomes (Fig. 4m-n, Extended Data Fig. 9o-r, Extended Data Table 6). Interestingly, a \\ndensity connecting the ribosome to the mitochondrial membrane was detected at the bottom of the large \\nsubunit in close proximity to the small subunit, but not to the ribosomal exit tunnel. It possibly represents \\nthe ribosome nascent chain complex (NAC) in contact with the mitochondrion receptor OM1461–63 which \\nhas yet to be structurally described.  \\n \\nIn summary, ribosomes close to mitochondria and ER exhibit different interfaces with the respective \\nmembranes, facilitating specific protein nascent chain membrane insertion or transfer into the \\norganelle2,52,62. These results highlight the power of DeePiCt for high-throughput particle localization and \\ncryo-electron tomogram segmentation to rapidly gain new biological insights. \\n \\nTrained networks can be readily applied to other species \\nAs a demonstration of the domain generalization potential of our workflow across species, we predicted \\nsegmentations of ribosomes, membranes and cytoskeleton structures (actin filaments and microtubules) in \\na published VPP HeLa cell dataset1 (Fig. 5). The application of the DeePiCt pipeline employed a manually \\ngenerated mask using Amira64 of the cytoplasmic volume, excluding the nucleus. For the evaluation of our \\nresults, we computed the voxel-F1 score in comparison to the publicly available annotations (EMD-\\n119921). For ribosomes and membranes, we used the ribosome- and membrane-networks trained on the 10 \\nVPP-tomograms from S. pombe described in previous sections. This resulted in a voxel-F1 of 0.55 for \\nribosomes and a voxel-F1 of 0.18 for membranes (Extended Data Fig. 10a). However, the available expert \\nmembrane segmentation for the dataset covered only ER membranes, which makes the voxel-F1 value less \\nmeaningful. Visual inspection of the predicted membrane segmentation revealed a good fit (Fig. 5). \\n \\nAdditionally we generated models for microtubules prediction, initially training a network for simultaneous \\nactin and microtubule segmentations in 4 tomograms of cryo-FIB milled Human retinal pigment epithelial-\\n1 (RPE-1) cells65. Due to the high preferential orientation of the cytoskeletal filaments in this data exhibiting \\nstress fibers, the performance for microtubules’ segmentation was low resulting in a voxel-F1 score of 0.26 \\n\", metadata={'x0': 72.0, 'x1': 542.9998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.9998779296875, 'height': 640.3059692382812, 'nlines': 44, 'fs': 14.55240839177912, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(Extended Data Fig. 10a). We therefore trained a second 3D CNN for microtubule segmentation on 11 \\ntomograms from lamella of cryo-FIB-milled, dissociated C. elegans cells, containing 890 microtubules in \\na wide range of orientations. This new microtubule segmentation network achieved a voxel-F1 score of \\n0.83 (Extended Data Fig. 10a), exemplifying the importance of a training set with a high number of \\nstructures, as well as orientations to mitigate the effect of the missing wedge on the prediction performance.  \\n \\nFinally, we trained a dedicated network for predicting actin-filaments on 5 manually curated tomograms, 2 \\nfrom RPE-1 cells and 3 from Mouse embryonic fibroblasts (MEF) 3T3 cells, that contained approximately \\n3,740 actin filaments (Online Methods). The actin filaments network shows a low voxel-F1 score of 0.10 \\nin the HeLa cell tomogram (Fig. 5, Extended Data Fig. 10a), which is likely caused by the finer structure \\nof actin in comparison to microtubules, the minimal set of orientations sampled in the small amount of \\ntraining data, and to the fact that most training filaments are arranged in bundles, probably causing the CNN \\nto learn this superstructure rather than individual filaments (Extended Data Fig. 10b). \\n \\nOverall, these results show that the prediction of ribosomes and microtubules, representing large \\nmacromolecular complexes, is especially well-preserved across species and that the availability of diverse \\nand well-annotated training datasets are crucial for good performance of DeePiCt. The results shown in \\nFig. 5 highlight the use of trained networks from DeePiCt to segment novel datasets spanning different \\nspecies.  \\n \\nDiscussion \\n', metadata={'x0': 72.0, 'x1': 545.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 473.6400146484375, 'height': 307.5884704589844, 'nlines': 21, 'fs': 14.647070021856399, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Our DeePiCt workflow facilitates accurate and fast localization of diverse structures in cryo-ET of intact \\ncells. The demonstrated high performance and the flexibility of the 3D CNN architecture offer the user a \\nreliable tool for pattern recognition. This enabled us to detect particle species, such as FAS, that are low in \\nabundance and have a less dense structural signature in comparison to ribosomes. The integration of \\nstructure segmentation (predicted by the 3D CNN) with the contextual information excludes false positives \\nin the 3D localization of particles and structures segmentation task, and harnesses the cellular context to \\ncarry out spatial studies focused on regions of biological interest. This enabled us to investigate the 3D \\nspatial distribution of macromolecules such as ribosomes in proximity to specific organelles (e.g. \\nmitochondria) and to obtain structural insights with functional implications. \\n \\nHere, we provide the first publicly available experimental cryo-ET dataset of 20 S. pombe tomograms under \\ntwo microscopy acquisition settings (VPP and DEF), together with a high-quality comprehensive \\nannotation of RIBO and FAS, membranes, organelle and cytosol segmentations. This constitutes the first \\ngold-standard dataset in the field that is large enough for unbiased model training, which will enable \\nbenchmarking of current methods, and spur the development of future computational tools for unbiased \\ndata mining in cryo-ET data.  \\n \\nSubtomogram averaging of the annotated particles from either ground truth or DeePiCt predictions resulted \\nin the first density maps of the S. pombe ribosome and fatty acid synthase. We observed differences in the \\nanalysis of subtomograms derived from either VPP and DEF tomograms, despite both being derived from \\nwild-type S. pombe cryo-FIB lamellae (Supplementary Note 2). Due to the preservation of high frequency \\nfeatures in DEF data, a subset of well-aligned ribosomes could be identified and refined to subnanometer \\n', metadata={'x0': 72.0, 'x1': 542.7723388671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7723388671875, 'height': 317.5059509277344, 'nlines': 22, 'fs': 14.43208867853338, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"resolution densities from cryo-FIB lamellae. The obtained maps provide detailed structural insights; for \\nexample, the P-site tRNA occupying the PTC, which together with the elongation factor eEF3 detected \\nclose to the head of the 40S subunit in a subset of ribosomes, suggest active translation. Further \\nclassifications close to the ribosomal exit tunnel revealed ES27L in a specific configuration providing a \\nplatform for other factors to interact with the exit tunnel and a potentially protruding nascent chain. By \\nutilizing the contextual information of the DeePiCt predictions, differences in relative membrane \\norientation of mitochondria- and ER-bound S. pombe ribosomes were detected. The observed connecting \\ndensity between ribosomes and mitochondria likely highlights the not yet structurally characterized \\nribosome-bound NAC in contact with OM-1461–63  facilitating protein import into mitochondria60,61.  \\n \\nThe analysis of DeePiCt's performance confirmed that data quality is an important factor in its predictive \\nability, as demonstrated in the case of NPC predictions in the DEF* dataset, which we classified into high \\nand lower quality subsets based on lamella thickness and tilt-series alignment error. As the NPC has a high \\ndegree of structural flexibility on the subunit and diameter level inside cells40 it is a more challenging target \\nthan for example ribosomes and thus emphasizes the importance of higher quality data to achieve sufficient \\nperformance. The introduction of a pre-processing equalizing filter is also an important element of the \\npipeline that improves both the learning process during the training of 3D segmentation networks for \\nparticles with less dense print than the ribosome (e.g. FAS), and especially the generalization power across \\ndomains, including different microscopy acquisition conditions. Thus, amplitude spectrum equalization \\nallows localizing macromolecules across different data acquisition types. In the case of the 2D network, \\nalthough pre-processing did not improve model performance on the same-domain inference, it led to cross-\\ndomain improvements when training on VPP data and inferring on DEF data, or when training on both data \\ntypes combined to segment organelles and cytosol. More elaborate tasks, such as prediction of individual \\norganelle types, will likely require more training data or training of a dedicated 3D CNN with a tailored \\nnetwork architecture. \\n \\nOur workflow allows easy adaptation to the segmentations of other structures, as demonstrated by the \\napplication of cytoskeleton segmentation networks. The networks show high-quality performance for \\nmicrotubules in the HeLa cell dataset after training on data with broad orientation sampling of the filaments. \\nActin predictions revealed a low F1 score and therefore likely require more training data sampling different \\norientations of the structural features to improve performance. Altogether, the application of multiple \\nsegmentation networks to the HeLa cell dataset revealed that DeePiCt trained on datasets from different \\nmicroscopes, species, and conditions lead to reasonably good results when the quality of data is high. \\nAlthough more in-depth analysis is needed to study the specific limitations for the applicability of DeePiCt \\nnetworks on other datasets, the results presented here constitute the first step towards conducting large-\\nscale quantitative analyses for structural biology using cryo-ET on cells from different laboratories and \\nopen source datasets. In this sense, the generated ground truth annotations in this study are a starting point \\nto provide the community with a resource to improve and further develop cryo-ET object segmentation and \\ndetection tools, which will ultimately enable further exploration of particles in their cellular context. \\nTogether with the trained networks and their demonstrated applicability to other species, as well as the \\nflexibility of the DeePiCt workflow, the software harbors great potential for quantitative cryo-ET studies \\nin the future. \\n \\n\", metadata={'x0': 72.0, 'x1': 542.77099609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77099609375, 'height': 623.2659301757812, 'nlines': 43, 'fs': 14.494556515715843, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Acknowledgments \\n', metadata={'x0': 72.0, 'x1': 188.33999633789062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 116.33999633789062, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='We thank Jenia Zagoriy and Wim Hagen for technical support, EMBL IT and especially Thomas Hoffman \\nfor computational support. I.D.T. and M.T.N were supported by a fellowship from the EMBL \\nInterdisciplinary Postdoctoral Program (EI3POD) under Marie Skłodowska-Curie Actions COFUND. J.M. \\nacknowledges funding from the EMBL and the European Research Council starting grant (3DCellPhase-\\n760067). \\n', metadata={'x0': 72.0, 'x1': 542.7700805664062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77008056640625, 'height': 70.30595397949219, 'nlines': 5, 'fs': 14.061190795898437, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Author contributions \\n', metadata={'x0': 72.0, 'x1': 201.05999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 129.05999755859375, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='I.D.T., J.M. and J.Z. conceived the study. I.D.T. implemented the 3D CNN code, co-produced ground truth \\nannotations, carried out numerical experiments to evaluate performance. S.K.G. prepared S. pombe cryo-\\nFIB lamellae and acquired cryo-electron tomograms, co-produced ground truth annotations, carried out \\nsubtomogram averaging and structural analysis. A.M. implemented the 2D CNN and spectrum equalization \\nfilter, produced ground truth segmentations and conducted performance analysis of the 2D CNN. F.S. \\ncarried out training and prediction with 3D CNN networks, implemented code and created Colab Notebooks \\nfor prediction. C.Z. generated data and provided NPC annotations under the supervision of M.B. M.T.N. \\ngenerated RPE1 and MEF 3T3 data and together with D.W.C.C. produced ground truth and trained \\nnetworks for actin segmentation. F.T. generated data, ground truth and trained networks for microtubule \\nsegmentation. C.P. and A.K. provided valuable input for the design and generation of the CNNs. I.D.T., \\nS.K.G., A.M., J.M. and J.Z. wrote the manuscript with input from all authors. \\n', metadata={'x0': 72.0, 'x1': 542.771240234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.771240234375, 'height': 157.66595458984375, 'nlines': 11, 'fs': 14.333268599076705, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Code and data availability \\n', metadata={'x0': 72.0, 'x1': 233.70001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 161.70001220703125, 'height': 20.9967041015625, 'nlines': 1, 'fs': 20.9967041015625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The DeePiCt code is assembled as two Snakemake pipelines (2D CNN and 3D CNN); the 3D CNN is \\nimplemented in the Pytorch framework while the 2D CNN is implemented in Keras, both in Python 3. The \\ncode, trained models, and link to the Google Colab notebook are available in the github repository \\nhttps://github.com/ZauggGroup/DeePiCt.   \\n', metadata={'x0': 72.0, 'x1': 542.7699584960938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76995849609375, 'height': 55.90594482421875, 'nlines': 4, 'fs': 13.976486206054688, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Raw tilt series, tomograms, ground truth coordinates and segmentations are available via EMPIAR \\naccession codes EMPIAR-10988 (S. pombe) and EMPIAR-10989 (RPE1). Subtomogram averages for S. \\npombe VPP and DEF ground truth annotations are available on EMDB (VPP gt: EMD-14404, EMD-14405, \\nEMD-14406, EMD-14408, EMD-14409, EMD-14410, EMD-14411; DEF gt: EMD-14412, EMD-14413, \\nEMD-14415, EMD-14417, EMD-14418, EMD-14419, EMD-14420; DEF DeePiCt predicted: EMD-\\n14422, EMD-14423, EMD-14424, EMD-14425, EMD-14426).  \\n', metadata={'x0': 72.0, 'x1': 542.7652587890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7652587890625, 'height': 84.9459228515625, 'nlines': 6, 'fs': 14.15765380859375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Structural comparisons were performed with S. cerevisiae FAS (PDB: 2uv844), P. pastoris FAS (EMD-\\n1213945), eEF3 from S. cerevisiae (EMD-1206250), the S. cerevisiae ribosome (EMD-166752) with the \\nrRNA expansion segment ES27L in a particular configuration (PDB-3izd53), the nuclear export factor Arx1 \\nbound to the 60S large ribosomal subunit S. cerevisiae (EMD-216956), the human Ebp1 (EMD-106857), S. \\ncerevisiae ribosomes derived from extracted ER (EMD-376459), and the ER-bound HeLa ribosomes (EMD-\\n80561). The large subunit (LSU, 60S) of a published S. cerevisiae 80S ribosome map (EMD-322866) and \\nthe S. cerevisiae FAS map (EMD-162346) were used as references for TM in S. pombe tomograms. The \\nHeLa cell dataset is available via accession code EMD-119921. \\n', metadata={'x0': 72.0, 'x1': 542.76025390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76025390625, 'height': 115.15020751953125, 'nlines': 8, 'fs': 14.393775939941406, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Competing interests \\n', metadata={'x0': 72.0, 'x1': 196.25999450683594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 124.25999450683594, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The authors declare no competing interests. \\n', metadata={'x0': 72.0, 'x1': 266.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 194.52001953125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Additional information \\n', metadata={'x0': 72.0, 'x1': 213.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 141.47999572753906, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Information is available for this paper. \\n', metadata={'x0': 72.0, 'x1': 313.32000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 241.32000732421875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figures \\n', metadata={'x0': 72.0, 'x1': 119.39999389648438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 47.399993896484375, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 1 | DeePiCt 2D and 3D CNN architecture implemented in an automated workflow combining \\ncompartment segmentations and particle localizations in cryo-ET data. a. The CNN U-Net architecture: the 2D \\nnetwork performs all tensor operations on the two-dimensional spatial coordinates, with D=5 and IF=16; the 3D \\nnetwork performs tensor operations on the three spatial dimensions, and the architectural hyperparameters in red can \\nbe set by the user. b. The DeePiCt pipeline is used to train and predict various structural features in cryo-electron \\ntomograms. b.1 The DeePiCt pipeline consists of two independent CNNs: a 2D network for compartment \\nsegmentation and a 3D network for particle localization. b.2 Trained networks are applied to input tomograms which \\ncan be pre-processed with a spectrum-matching filter to improve image contrast. The example 2D tomographic slice \\nvisualizes the cytoplasm with the endoplasmic reticulum (ER), vacuoles (V), nucleus and extracellular space (ECS). \\nb.3 DeePict raw predictions for cytosol, membranes, ribosomes and fatty acid synthase (FAS) are post-processed by \\nthresholding, cluster size and centroid fitting. b.4 The outputs of the two networks can be combined to include the \\ncellular context by intersecting particle predictions with cytosol masks (top), selecting particles (NPC, green) in \\ncontact with specific membranes (nuclear envelope, middle), and to identify particles (e.g. ribosomes, orange) \\nassociated with specific organelles (e.g. ER). \\n \\n \\n \\n \\n \\n  \\n', metadata={'x0': 72.0, 'x1': 542.527587890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.527587890625, 'height': 183.00283813476562, 'nlines': 20, 'fs': 9.15014190673828, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 2 | Iterative comprehensive annotation of ground truth for non-synthetic data. The three columns (left to \\nright) show: the annotation process for RIBO (a, d, g), FAS (b, e, h), membranes (c, f). a-f Show the cumulative \\npredictions of 3 rounds of DeePiCt for RIBO and one round for FAS as cross-section overlaid on a tomographic slice \\n(a-c) and in 3D view (d-f). Particles are classified as true positives recovered from the initial annotation (tp recovered; \\nyellow for RIBO; green for FAS), new true positives provided by DeePiCt predictions (tp new; blue), and the \\ncorresponding false positives (fp; red). g, h. 3D views of the resulting ground truth, including false negatives (fn) \\ndistinguishing unrecovered fn from the initial annotations (pink) and final round of manual picking (salmon). i. \\nCombined ground truth annotation of RIBO, FAS, membranes, and NPC (green cylindrical mask, manual annotation). \\nj-k. Relative contributions of the DeePiCt rounds for RIBO (j) and FAS (k) identification are plotted across 10 VPP \\nand 10 DEF S. pombe tomograms: contribution of step 1 (yellow: TM and manual annotation for RIBO; green: manual \\nannotation for FAS; salmon: initial annotations not detected by DeePiCt), step 2 (blue); and step 3 (pink bars). For \\nFAS the initial annotation was not based on TM, and instead was purely manual. l, m. Subtomogram averages of all \\nRIBO, exclusively DeePiCt-detected (tp new; blue) and all FAS particles in VPP and DEF ground truth, respectively. \\n \\n', metadata={'x0': 72.0, 'x1': 542.5265502929688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.52655029296875, 'height': 184.17047119140625, 'nlines': 14, 'fs': 13.155033656529017, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 3 | DeePiCt performance, cross-domain generalization and comparison with other methods. a. 2D CNN \\nperformance results for organelle and cytosol segmentation under a 5-fold CV scheme, when training and testing \\ndatasets are both in the same domain (VPP). The median AUPRC scores are indicated (black dotted lines). b. \\nPerformance of DeePiCt for the same-domain setting. The RIBO localization, FAS localization, and membrane \\nsegmentation tasks are shown (left to right), under a 3-fold CV scheme. In each case, the corresponding architectures \\nof the 3D CNN were optimized by hyperparameter tuning (Supplementary Table 1, Extended Data Fig. 6). The \\nmedian F1 score for RIBOS and FAS, and a median voxel-F1 for membranes are indicated (black dotted lines). c. \\nSame-domain NPC segmentation results using a 3-fold CV scheme. From left to right: performance in all DEF* \\ntomograms, high quality DEF* tomograms, and lower quality DEF* tomograms. d-e. DeepFinder (d) and TM (e) \\nparticle localization results for RIBO and FAS localization. The median F1 values are indicated in each case (black \\ndashed line), along with corresponding values of DeePiCt’s performance for comparison (red dashed line). d. \\nDeepFinder results are based on the same 3-fold CV scheme as in b, by training a multiclass DeepFinder network that \\nsimultaneously segments FAS and RIBO. e. TM results are shown for the VPP tomograms. f-g. Results of the cross-\\ndomain generalization for the 2D CNN (f) and DeePiCt (g) (training on 10 VPP tomograms, testing on 10 DEF \\ntomograms). Dashed lines and their numerical labels indicate median performance values either of the corresponding \\nplot (black), or the corresponding experiment performed in the same domain with DeePiCt (red), or in the same domain \\nwith the 2D CNN (green).  \\n \\n', metadata={'x0': 72.0, 'x1': 542.5325317382812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.53253173828125, 'height': 237.21044921875, 'nlines': 18, 'fs': 13.178358289930555, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='19 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='20 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 4 | DeePiCt enables exploration of macromolecular complexes in their cellular context. a. 2D slice of a \\nDEF S. pombe cytoplasm tomogram. Mitochondrion (M), vesicle (V), the ER and the cell wall (CW) are detectable in \\nthe raw (bottom left) and with improved contrast after amplitude spectrum equalization (top right). b. DeePiCt \\npredictions generated with models trained in VPP data. Organelles (gray), membranes (purple), FAS (pink, c, d), \\nribosomes (yellow, e-h) and subsets classified in RELION (head density, dark blue, i; exit tunnel density, bright blue, \\nj), and within 25 nm to mitochondria (mito-bound, green, m) and the ER (ER-bound, orange, n). c. FAS subtomogram \\naverage (pink) fits the S. cerevisiae structure (cyan) including PPT domains. An extra density cannot be assigned. d. \\nCross-section of c close to the alpha-wheel with three densities fitting ACPs (white asterisks). e. Subtomogram average \\nof all ribosomes from 10 DEF tomograms. f. Well-aligned ribosome subset detected by hierarchical 3D classification \\nin RELION and refined in M. g, h. Slices through (f) reveal the PTC with a P-site tRNA and the L1 stalk facing the \\nE-site. i. Ribosome subclass with additional densities (white arrowheads) close to the head of the small ribosomal \\nsubunit which fits eEF3 (red), and close to the ribosomal exit tunnel. j. Ribosomes classified for a density below the \\nribosomal exit tunnel (white arrowhead). The S. cerevisiae ribosome with ES27L in a particular configuration (left, \\npurple) connects to the additional exit-tunnel density which fits Arx1 bound to the 60S pre-ribosome (middle and \\nright, purple). k, l. Different z slices of the tomogram in a (white dashed boxes) show ribosomes bound to the ER (k) \\nand a mitochondrion (l). m. ER-bound ribosomes from 7 tomograms show a connection of the peptide exit tunnel of \\nthe large subunit to the membrane density. n. Mitochondria-bound ribosomes from 3 tomograms show a linker \\nconnecting the large subunit, at a site close to the small subunit, to the membrane density. Overlay of the ribosomes \\nin m and n shows different interfaces with the respective organelle membranes.  \\n \\n', metadata={'x0': 72.0, 'x1': 542.5247802734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.5247802734375, 'height': 249.45050048828125, 'nlines': 20, 'fs': 12.472525024414063, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='21 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 5 | DeePiCt’s generalization across species. A dataset depicting a HeLa cell nuclear periphery1 is segmented \\nby applying four independently trained DeePiCt networks. The results show the segmentation of actin filaments (red) \\ntrained on RPE-1 and MEF 3T3 tomograms, microtubules (MTs, cyan) trained on C. elegans tomograms, and cytosolic \\nribosomes (yellow) and membranes (purple) trained on S. pombe tomograms.  \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 542.521728515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.521728515625, 'height': 83.20498657226562, 'nlines': 7, 'fs': 11.886426653180804, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='22 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Online Methods \\n', metadata={'x0': 72.0, 'x1': 172.74000549316406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 100.74000549316406, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Yeast cell culture \\nS. pombe K972 Sp h- wt haploid was kindly provided by C. Haering, originally from P. Nurse. Cells were \\nrecovered from frozen stock by streaking on YE5S agar plates (YES Broth, Formedium, 20 g agarose/L) \\nand incubated at 30 °C for 1-3 days. Colonies were re-streaked on fresh YES agar plates and incubated 1-\\n3 days at 30 °C. Single colonies were inoculated in 5 mL of YES medium (YES Broth, Formedium, \\nPCM0302, FM0618/8573) and grown at 30 °C, 170 rpm overnight (NCU-Shaker mini, Benchmark). On \\nthe next day, cultures were grown to their log phase at OD600 of 0.5 - 0.6 and diluted beforehand in YES if \\nnecessary.  \\n  \\nVitrification of yeast cells \\nA Leica EM GP (Leica Microsystems) was utilized to vitrify yeast cells at liquid nitrogen temperature. \\nYeast cells were either diluted to OD600 of 0.2-0.4 in YES medium or, following a prior wash step, in PBS \\ncontaining 5 % or 10 % BSA as cryoprotectant. In the chamber, 4 µl were directly applied to TEM grids \\n(Quantifoil R1/2, Cu 200 mesh, holey carbon or SiO2 film), which were glow discharged on both sides for \\n45 s (Pelco Easy glow) immediately before usage. Blotting from the back side of the support was performed \\nfor 1-2 s at 22 °C and 99% humidity. Cells were plunge-frozen in liquid ethane cooled by liquid nitrogen \\nand transferred into grid boxes until further usage. \\n  \\nCryo-FIB of S. pombe \\nTEM grids containing vitrified yeast cells were clipped into an autogrid with a cut-out enabling cryo-\\nfocused ion beam (cryo-FIB) milling at shallow angles67. Mounted on a 45° pre-tilt shuttle, grids were \\ntransferred into an Aquilos Dual beam microscope (ThermoFisher Scientific). Prior to cryo-FIB milling at \\nliquid nitrogen temperature, cells were sputter-coated with platinum for 10-15 s (1 kV, 10 mA, 10 Pa). \\nSubsequently, a layer of organometallic platinum (reservoir at 28°C) was applied by opening the gas \\ninjection system (GIS) for 8 sec at a stage Z position of 3-4 mm below the coincidence point. In three \\nindependent sessions, three grids with 5 lamellae each were prepared at a milling angle of 15°. \\nAgglomerations of several cells were micromachined in three steps of rough milling to a thickness of 5 µm \\nat 1 nA ion beam current, 3 µm at 0.5 nA and 1 µm at 0.1 nA. Milling progress was visually monitored \\nbetween each milling step with the scanning electron microscope beam (SEM, 10 kV, 50 pA). Fine milling \\nwas performed at 50 pA to a target thickness of 200 nm. To render the lamellae conductive for TEM \\nimaging, they were sputtered with platinum for 5 s (1 kV, 10 mA, 10 Pa) and transferred into cryo boxes. \\n  \\nCryo-electron tomography of S. pombe \\nAutogrids containing lamellae were loaded into a Titan Krios (Thermo Fisher Scientific) such that the axis \\nof the pre-tilt introduced by FIB milling was aligned perpendicular to the tilt axis of the microscope68. Cryo-\\nelectron tomography acquisition parameters are summarized in Extended Data Tables 4-5. Tomograms \\nwere acquired at 300 kV on a K2 Summit direct detection camera (Gatan) operating in dose fractionation \\nmode and utilizing a Quantum post-column energy filter operated at zero-loss (Gatan). Magnification of \\n42,000 (EFTEM) with a calibrated pixel size of 3.37 Å was used for the ground truth VPP and DEF datasets \\nand 3.45 Å for NPC data. Up to 14 tilt series were collected on a single lamella in low dose mode using \\nSerialEM69. Starting from the lamella pre-tilt, images were acquired at 2.0 - 4.0 µm underfocus, in 2° \\nincrements within a range of + 58° to - 40° using a dose-symmetric tilt scheme70. The total dose was up to \\n', metadata={'x0': 72.0, 'x1': 542.7710571289062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77105712890625, 'height': 612.7059936523438, 'nlines': 42, 'fs': 14.588237944103422, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='23 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='120 e-/Å2 with a constant electron dose per tilt image. For the ground truth data, a set of 10 tilt-series were \\neither collected with a 70 µm objective aperture or a Volta potential phase plate (VPP, Thermo Fisher \\nScientific39) with prior conditioning for 5 min. NPC data was collected as described in previous work71,40, \\nbut at 1.5 - 4.5 µm underfocus, with 3° increments and an effective tilt range of + 50° to - 50°.  \\n \\nTomogram reconstruction \\nTilt movie frames were aligned using a SerialEM plugin. Tilt series were filtered according to the \\naccumulated \\nelectron \\ndose \\nby \\nFourier \\ncropping \\nusing \\nthe \\nmtffilter \\nfunction \\nin \\netomo \\n', metadata={'x0': 72.0, 'x1': 542.770263671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.770263671875, 'height': 114.91020202636719, 'nlines': 19, 'fs': 6.0479053698088, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(IMOD/BETA4.10.1273), and sorted by tilt angle using a python script. Four times binned tilt images (movie \\nsums) were aligned in etomo (IMOD/4.9.4) software package73 using patch tracking (residual error 0.291 - \\n0.569 px) and tomograms were reconstructed via weighted back projection. Tomogram thicknesses \\nmeasured in 3dmod ranged between 80 - 310 nm. \\n', metadata={'x0': 72.0, 'x1': 542.7698364257812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76983642578125, 'height': 56.83024597167969, 'nlines': 4, 'fs': 14.207561492919922, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ground truth annotation for organelles, cytoplasm, and membranes in VPP and DEF \\n', metadata={'x0': 72.0, 'x1': 475.32000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 403.32000732421875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular compartment segmentation was performed for both the VPP and DEF datasets. The annotations \\ninclude 10 different organelle classes (mitochondria, vesicle, tube, ER, nuclear envelope, nucleus, vacuole, \\nlipid droplet, Golgi apparatus, vesicular body, see Extended Data Table 1) and cytosol annotations, each \\nidentified through a unique numerical label to allow for selection of specific subsets of compartments.. \\nSegmentations of the 10 VPP tomograms were performed manually in Amira64. These annotations were \\nused to train a 2D CNN and to predict the initial segmentations in 10 DEF tomograms, which were first \\npre-processed with the spectrum equalization filter. These predictions were then manually corrected to \\nobtain high-quality ground truth segmentations in DEF (Extended Data Table 1).  \\n', metadata={'x0': 72.0, 'x1': 542.7678833007812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76788330078125, 'height': 114.2259521484375, 'nlines': 8, 'fs': 14.278244018554688, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"Membrane annotations were performed on the 10 VPP tomograms and 5 DEF tomograms. Initially, 5 VPP \\ntomograms were annotated using Amira to manually segment every 2 to 3 slices, which were subsequently \\ninterpolated by means of Amira's interpolation tool. These 5 VPP tomograms were then used to train a \\nmembrane segmentation 3D CNN, whose predictions on the remaining 5 VPP tomograms as well as in 5 \\nDEF tomograms, were manually corrected in Amira.  \\n\", metadata={'x0': 72.0, 'x1': 542.7700805664062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77008056640625, 'height': 70.54595947265625, 'nlines': 5, 'fs': 14.10919189453125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ground truth particle annotation in VPP \\n', metadata={'x0': 72.0, 'x1': 267.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 195.48001098632812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='RIBO and FAS were localized in 4-times binned tomograms (13.48 Å pixel size) in an iterative workflow. \\nTM (RIBO) or manual annotations (FAS) (step 1, Extended Data Tables 2-3) were used for training 3D \\nCNNs (step 2). CNN predictions were masked with a segmentation of the cytosol. Step 2 was repeated \\nseveral times and each training round trained on the combined predictions of step 1 and the preceding round. \\nEach round in step 2 consisted of three simultaneously trained networks sharing the same set of default \\nhyperparameters (Supplementary Tables 2-3), except for the number of IF=4, 8, and 32, to provide a \\ncumulative prediction less overfitted to the incomplete training data. The cumulative total predictions from \\nsteps 1 and 2 were then complemented by a round of visual inspection to eliminate false positives and \\nmanual picking to recover remaining false negatives (step 3, Extended Data Tables 2-3). Finally, the \\nparticle lists were cleaned for duplicates by applying elliptic distance constraints to the coordinates \\n(Supplementary Note 1, Extended Data Tables 2-3). \\n \\nIn particular, for RIBO in the VPP dataset, the initial annotation (step 1) was performed as a combination \\nof two sub-steps: step 1a employed TM performed with the pyTOM toolbox36. In detail, a 3D cross-\\n', metadata={'x0': 72.0, 'x1': 542.7723388671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7723388671875, 'height': 201.34600830078125, 'nlines': 14, 'fs': 14.381857735770089, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='24 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='correlation search over 1,944 Euler angle combinations was conducted using the large subunit (LSU, 60S) \\nof a published S. cerevisiae 80S ribosome map (EMD-322866) scaled to the corresponding pixel size, and a \\nspherical mask with a diameter of 337 Å applied. The obtained coordinates corresponding to the top 2000-\\n3000 highest cross correlation scores were manually revised in Gaussian filtered tomograms (sigma = 3) \\nwith tom_chooser (pyTOM toolbox36). This was followed by step 1b, which consisted of a non-exhaustive \\ncomplementary manual annotation of particles missed by TM (Extended Data Tables 2-3). These initial \\nribosome coordinates served as input for step 2: three training rounds of CNNs were successively applied, \\nas a global visual inspection confirmed that the false positive rate was low, while the fraction of true \\npositives recovered was substantial. The resulting CNN predictions were manually revised in tom_chooser \\nas described above. In step 3, remaining particles that were not detected by either TM or CNN were \\nmanually localized in either spectrum-matched or Gaussian filtered tomograms (sigma = 3) for up to 3 \\nrounds in EMAN2 spt2_boxer74. \\n  \\nFAS in the VPP dataset was localized in step 1 manually in EMAN2, as TM with a published S. cerevisiae \\nFAS46 map (EMD-1623) as reference failed (Extended Data Tables 3). In step 2 we performed only one \\nround of training with these initial annotations and the resulting predictions were revised in EMAN2. \\nAdditional FAS particles were localized manually.  \\n', metadata={'x0': 72.0, 'x1': 542.7679443359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7679443359375, 'height': 245.02593994140625, 'nlines': 17, 'fs': 14.413290584788603, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ground truth particle annotation in DEF \\n', metadata={'x0': 72.0, 'x1': 268.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 196.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='In the DEF dataset, the annotation procedure was the same as the one described for the VPP dataset, except \\nthat: 1) for RIBO the initial annotations relied on manually cleaned TM results and for FAS on incomplete \\nmanual picking, and 2) in step 2 of the RIBO ground truth construction, the first of the 3 rounds of CNN \\npredictions used the DEF training data (from the TM results), while for the remaining 2 rounds the \\npredictions were obtained using the models from the first and second rounds of RIBO VPP ground truth \\nconstruction (trained only on VPP data, Supplementary Table 2). The visual inspection in step 2 was \\nperformed over the combined predictions of both the previously mentioned VPP networks and a round of \\nDEF networks (IF=4, 8, 32) based on DEF TM results. After the final manual picking for missing particles \\n(not present in either the initial TM results or the manually cleaned network predictions), we achieved \\ncomprehensive RIBO and FAS annotations (Extended Data Tables 2-3).  \\n \\nComparison of cryo-ET-derived particle numbers with proteomics \\nCopy numbers of ribosomes and FAS per cell were calculated with the ground truth annotations for an S. \\npombe cell with the assumption of 30 % cytosolic volume75 of a total of 150 µm3 average cell volume76,77. \\nIt was also considered in the case of FAS that one fully assembled FAS complex is constituted by each six \\nalpha and beta subunits. \\n \\nNPC manual localization \\nNPCs were manually localized as described previously40. In brief, coordinates and initial orientations of \\n354 NPCs were manually determined in 127 4-times binned, SIRT-like filtered73 defocus-only tomograms, \\nas described earlier71. Two points located at opposing inner and outer nuclear membrane (INM-ONM) \\nfusion points were manually picked for each NPC using IMOD. A third point was then chosen located at \\nthe INM-ONM fusion ring such that all three points define the horizontal plane of the NPC particle. The \\nmean coordinates between the first two points were then used to assign the center of the NPC. The initial \\norientation of the NPCs was defined based on the normal vector to the defined NPC plane. A fourth point \\n', metadata={'x0': 72.0, 'x1': 542.7698974609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7698974609375, 'height': 361.4259948730469, 'nlines': 25, 'fs': 14.457039794921876, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='25 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='was chosen in the cytoplasmic side of the horizontal plane to assign the correct initial orientation and \\nprevent false upside-down assignment of initial orientations. The annotated NPC data was divided based \\non quality criteria: 38 tomograms of quality 1 have a thickness below 300 nm, thus display optimal image \\ncontrast, and a tilt-series alignment residual error below 0.7 px. Quality 0 was assigned to 89 tomograms \\nwith 300 - 395 nm thickness and a residual error of 0.7 - 5.0 for tilt series alignments. \\n \\nVoxel-level representation of ground truth \\nAs the final product of our ground truth generation process, we obtain voxel-based masks for membranes, \\ncellular organelles, cytoplasm, and particles (RIBO, FAS, NPC). For RIBO and FAS, the lists of coordinates \\nobtained were used to paste spherical masks (with radii of 10.78 nm for RIBO and 13.48 nm for FAS) \\ncentered on each of these points using a python script (Code and data availability). For the NPCs in DEF*, \\na subunit mask obtained by prior 3D averaging in novaSTA78, was pasted at each of the 8 subunit locations. \\nThe RIBO, FAS, membrane, organelle and cytosol masks in DEF and VPP along with the list of coordinates \\nand respective tomograms are used in the subsequent training and performance analysis and are available \\non EMPIAR (Code and data availability).  \\n \\nCytoskeletal filaments segmentation \\nMicrotubule and actin networks were trained on tomograms of Human retinal pigment epithelial-1 (RPE-\\n1) cells previously described65. Segmentations of the individual cytoskeletal elements performed using the \\nfilament tracing function in Amira79,80, followed by manual curation. Trained networks were applied to a \\ntomogram depicting the nuclear periphery of a HeLa cell and compared against the corresponding \\nsegmentations (EMD-11992). \\n', metadata={'x0': 72.0, 'x1': 542.771240234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.771240234375, 'height': 317.7459411621094, 'nlines': 22, 'fs': 14.442997325550426, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Subtomogram analysis for ribosomes and FAS \\nContrast transfer function (CTF) estimations, generation of 3D CTF models and subtomogram \\nreconstructions were performed in Warp32. CTFs were first estimated in the sums of raw tilt movies and \\nsubsequently in the tilt series taking the tilt angles into account. Subtomograms containing ribosomes and \\ntheir respective CTF models were reconstructed in volumes of 140³ pixels with a pixel size of 3.3702 Å \\nand a particle diameter of 350 Å. Initial alignments of the subtomograms were performed in RELION \\nversion 3.0.734 in 25 iterations of 3D classification, with the S. cerevisiae 80S ribosome (EMD-3228, low-\\npass filtered to 60 Å) as reference to generate an initial single class average. 3D refinements were performed \\nwith the resulting average as a reference. In the case of DEF data, this RELION-refined average was further \\nrefined in M33 to optimize particle poses, images and volumes warping to model non-linear deformations. \\nParticles were re-extracted and hierarchical 3D classifications (25 iterations each) were performed in \\nRELION. In the case of VPP data, hierarchical 3D classifications were performed directly after 3D \\nrefinements in RELION. Focused classifications were performed with binary masks indicated in the \\nrespective figures. Masks were generated from spheres produced in Matlab (MathWorks) in a cubic volume \\nof 50 pixels per side and a radius of 20 pixels, scaled to a pixel size of 3.3702 Å and converted into binary \\nmasks resulting in a radius of around 130 Å using RELION, and placed relative to the area of interest using \\nChimera81. \\n \\nFAS particles and CTF models were reconstructed in cubic volumes of 160 pixels per side with a pixel size \\nof 3.3702 Å and a particle diameter of 400 Å. Initial alignments were performed in RELION in 25 iterations \\nof 3D classification into a single class, and followingly refined with the 3D refinement option and applying \\n', metadata={'x0': 72.0, 'x1': 542.7735595703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7735595703125, 'height': 303.345947265625, 'nlines': 21, 'fs': 14.445045107886905, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='26 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='D3 symmetry using the S. cerevisiae FAS map (EMD-162346, low-pass filtered to 60 Å) as reference. \\nHierarchical 3D classifications (25 iterations each) were performed either after 3D refinements in RELION \\n(VPP data) or after subtomogram re-extraction using the ribosome-optimized image and volume models in \\nM (described above). \\n \\nFinal subtomogram averages of each particle class were obtained by 3D refinement and post-processing, \\nincluding filtering to their respective resolutions determined by Fourier Shell Correlation (FSC) of two \\nindependently refined half maps at a cut-off of 0.143. Details of particle numbers and resolutions for each \\nsubtomogram average are summarized in Extended Data Tables 4-6. Visualization and calculation of \\ncross-correlations (CC) between different maps and models, was performed with the UCSF ChimeraX \\npackage82.  \\n \\nCNN Pre- and post-processing \\nPreprocessing \\nTomograms are normalized to obtain uniform mean of 0 and variance of 1 in the frequency domain, prior \\nto training. Additionally, the spectrum equalization filter is applied by matching the amplitude spectrum of \\nthe tomogram that is to be transformed to the target spectrum of one manually selected high-contrast VPP \\ntomogram (Tomogram TS_001, Extended Data Fig. 1b). Extraction of spectra amplitudes is done using \\nfast Fourier transform (FFT) followed by radially averaging amplitudes across the frequency domain. If the \\ntarget tomogram’s Nyquist frequency is lower than the input tomogram’s, the target spectrum is padded \\nwith zeros to match the input spectrum’s size. Next, an equalization vector is created by dividing entry-\\nwise the target spectrum by the respective input spectrum. The resulting equalization vector is converted \\ninto a rotational kernel and multiplied by the input tomograms in the frequency domain in combination with \\na sigmoidal-shaped low-pass filter to eliminate high frequency noise. After back transformation, the \\ntomogram exhibits a similar contrast to the target tomogram (Extended Data Fig. 1). Tomogram intensities \\nare additionally normalized to obtain uniform mean and variance. For the 2D CNN, tomograms and training \\nsegmentations are processed slice-wise into 2D tiles with a fixed size of 288x288 pixels (256x256 pixels \\nand 16 pixels padding on each side). For the 3D CNN, tomograms are by default split into cubic patches of \\n64x64x64 voxels, and 12 voxels overlap in each dimension.  \\n \\nPost-processing (2D CNN) \\nThe post-processing steps for the 2D network are focused on assembling the per-slice prediction into a 3D \\nsegmentation. To that aim, predicted tile segmentations are cropped on each side by 48 pixels to reduce \\nartifacts around the edges, followed by reassembly into 3D stacks, with remaining overlapping areas being \\naveraged. A 1D Gaussian filter is successively applied along the z axis in order to reduce single-slice false \\npositives (Extended Data Fig. 2). \\n \\nPost-processing (3D CNN) \\nAfter reassembly of the individual 64x64x64 voxel patches into the probability map resulting from the 3D \\nCNN, the post processing step includes the clustering of the thresholded map (usually at threshold value of \\n0.5). According to user preferences, only clusters that lie within or close to a given mask consisting of an \\norganelle or cytosol segmentation outputted by the 2D network, and whose size lies within a given range \\nof numbers of voxels, are considered for the final prediction map (Supplementary Note 1). This ‘cleaning’ \\nprocedure allows adding a priori information on the size of the particles in question. Additionally, for \\n', metadata={'x0': 72.0, 'x1': 542.76904296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76904296875, 'height': 638.5902099609375, 'nlines': 44, 'fs': 14.51341386274858, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='27 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='macromolecular complex localization, the user has the option to export a list of coordinates of cluster-\\ncentroids, which represent the particles’ locations prediction. \\n \\nPrecision-Recall and Performance Analysis \\nCross-validation approach \\nThe analysis of performance for RIBO and FAS localization, as well as membrane segmentation, were done \\nusing a 3-fold CV scheme, where 8 out of the 10 VPP tomograms were used for training, and the remaining \\n2 VPP tomograms were used for testing. In the test for domain generalization, the same three networks \\n(trained on the 8 VPP tomograms) were applied to the 10 DEF tomograms. \\n \\nThe analysis of performance in the NPC segmentation task was done in the dedicated DEF* of 127 \\ntomograms. Importantly, in order to apply the DeePiCt pipeline, we employed a predicted segmentation of \\nthe nuclear envelope as the region mask in the contact mode during post-processing of the NPC prediction \\n(Supplementary Note 1). The NPC prediction was studied under a 3-fold CV scheme, where each split \\nconsisted of roughly 42 tomograms. The nuclear envelope prediction was achieved through an \\nindependently trained 3D CNN on 18 tomograms belonging to the DEF* dataset for which a manual \\nannotation of the nuclear envelope was available, and which were uniformly split across the 3 CV subsets.  \\n \\nFor organelle and cytosol segmentation evaluation, model performance was evaluated on the voxel-level \\nfor the post-processed 2D CNN predictions produced by each CV fold’s individually trained model, using \\n5000 voxels picked randomly from each tomogram of the CV fold’s respective test set. Precision and recall \\n(Supplementary Note 1) were computed at threshold values varying from 0 to 1 on the picked voxels to \\ncompute the area under the precision-recall curves (AURPC) for each tomogram.  \\n \\nEvaluation metrics \\nIn the case of particles (RIBO and FAS), the localization evaluation was made directly from the predicted \\nlists containing their centroids’ voxel coordinates. In the case of DeePict, the lists of particles’ coordinates \\nare sorted according to a decreasing score (cluster size) value. For both RIBO and FAS, we defined a true \\npositive when a predicted coordinate matched a ground-truth particle coordinate within a tolerance radius \\nof 130 Å (Supplementary Note 1). The F1 score, which is calculated as the harmonic mean of the precision \\nand the recall, provides a measure of the quality of the predictions. We chose this measure for easier \\ncomparison to previous studies for particle localization methods in the field of cryo-ET26,27,29,83. Other \\nmetrics such as the area under the precision recall curve (AUPRC) generated by increasing length of \\nparticles’ list were disregarded as the ordering of the list (a function of cluster size) does not necessarily \\nreflect the method’s confidence on positive classification. \\n \\nThe evaluation of the segmentation task is of a totally different nature, as the shape of the object should be \\ntaken into account voxel-wise. For the 2D CNN predictions of organelles and cytosol, we used the AUPRC \\nscore with varying threshold over the network’s output map (Supplementary Note 1). For Deepict’s \\npredictions of NPC and membrane segmentation, we instead used the voxel-based F1 score \\n(Supplementary Note 1), to maintain hegemony with DeePiCt particle prediction evaluations. \\nHyperparameter tuning of DeePiCt and performance assessment \\nThe previously described cross-validation schemes and evaluation metrics were not only used for the final \\nassessment of methods’ performance (Fig. 3), but to study the effect of the different hyperparameters that \\n', metadata={'x0': 72.0, 'x1': 545.4000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 473.4000244140625, 'height': 637.6659545898438, 'nlines': 44, 'fs': 14.492408058860086, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='28 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='could be set by the user in the case of the 3D CNN (IF, D, ED, DD, and BN). The experiments’ starting \\npoint was always the default hyperparameter combination 𝐷 = 2, 𝐼𝐹 = 4, 𝐵𝑁 = 0, 𝐸𝐷 = 0, 𝐷𝐷 = 0 \\n(Supplementary Note 1). In all cases, we tested first for different values of D, while keeping the rest fixed. \\nThen, we fixed D=D*, where D* denotes the value for which best performance was achieved and repeated \\nthe same process for IF. We continued with each of the remaining parameters, BN, ED and DD. The results \\nare reported in Extended Data Fig. 6. They allowed us to define the best hyperparameter combination for \\neach of the structures studied, which are summarized in Supplementary Table 1. \\n \\nIn each case we combined the 3D CNN segmentation of the target structure with an appropriate region \\nmask (Supplementary Note 1, Supplementary Table 1): in the RIBO and FAS localization task, we used \\na cytosol prediction from the 2D CNN in intersection mode to eliminate false positives in organelles and \\nthe extracellular space; for membrane segmentation we used again the predicted cytosol from the 2D CNN, \\nthis time in contact mode to eliminate clusters that are in the extracellular space or within organelles; for \\nNPCs we used a 3D CNN prediction of the nuclear envelope in contact mode to eliminate false NPC \\nsegmentations far from the nuclear boundary. For RIBO, FAS and membrane, both 3D CNN (for target \\nstructure prediction) and 2D CNN (for region mask prediction) in DeePiCt employed a 3-fold CV in the \\nVPP dataset. In each fold, 8 tomograms were used for training and model selection while the remaining 2 \\ntomograms were used for testing (Extended Data Fig. 6a-c). The NPC predictions, which we treat as \\nsegmentation tasks, were evaluated in the DEF* independent data selected for that purpose \\n(Supplementary Note 1 and Extended Data Fig. 6d-f). The region mask of predicted nuclear envelope \\nsegmentation was generated by a fixed 3D CNN trained on 18 of the 127 DEF* tomograms. These 18 \\ntomograms were evenly distributed among the 3 splits of the DEF* dataset used for the corresponding 3-\\nfold CV scheme employed for the hyperparameter tuning analysis of NPC segmentation by DeePiCt \\n(Extended Data Fig. 6d-f). We used a 3-fold CV where the total 127 tomograms were split into 3 random \\nsubsets with roughly the same number of tomograms each. Each fold consists of two such sets as training \\ndata and the remaining one for testing. Throughout our experiments, high-quality DEF* tomograms \\n(defined based on lamella thickness and tomographic alignment residual error as described above) \\nconsistently showed higher performance for the NPC segmentation task than the low-quality DEF* \\ntomograms, regardless of architectural variations (Extended Data Fig. 6d-f). This hints at the importance \\nof high data-quality in segmentation and localization tasks. \\n \\nFor the evaluation of the particle localization task, we compared the lists of coordinates of the particles in \\nour ground truth data with the predicted coordinate lists. For both RIBO and FAS, we considered a true \\npositive when coordinates coincide within a tolerance radius of 10 voxels (corresponding to approximately \\n130 Å). In this way, we computed the recall (the proportion of ground truth particles that were recovered \\nby the method) and the precision (the proportion of predicted particles that were true positives) and reported \\nthe F1 score, which is the harmonic mean of precision and recall (Supplementary Note 1, Supplementary \\nTable 1). For the structure segmentation task (such as the membrane or the NPC segmentation evaluation), \\nwe compared the ground truth masks with the DeePiCt predicted post-processed segmentation by \\ncalculating their voxel-wise precision and recall, and reporting the corresponding voxel-based F1 (voxel-\\nF1) score, also known as Sørensen-Dice coefficient (Supplementary Note 1).  \\n \\nAll experiments for the 3D CNN of the DeePiCt pipeline and DeepFinder were performed using NVIDIA \\n2080 Ti GPU, Cuda 10.0, Python 3 and Pytorch 1.3.1. For the 2D CNN, training was conducted using an \\n', metadata={'x0': 72.0, 'x1': 542.7711181640625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7711181640625, 'height': 637.9059448242188, 'nlines': 44, 'fs': 14.497862382368607, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='29 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='NVIDIA 2080 Ti GPU and an NVIDIA V100S GPU used for performance evaluation, using CUDA 10.0, \\nPython 3 and Keras 2.3.1 with a tensorflow 2.0.0 backend. Detailed lists of parameters used for the 2D and \\n3D CNN are available alongside DeePiCt’s source code (Code and data availability). \\n \\n \\n', metadata={'x0': 72.0, 'x1': 542.7667236328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7667236328125, 'height': 55.90592956542969, 'nlines': 5, 'fs': 11.181185913085937, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='30 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. \\nMahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science \\n', metadata={'x0': 72.0, 'x1': 529.5599975585938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 457.55999755859375, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='351, 969–972 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 191.87998962402344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 98.03999328613281, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. \\nPfeffer, S. et al. Dissecting the molecular organization of the translocon-associated protein complex. \\n', metadata={'x0': 72.0, 'x1': 538.9215698242188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.92156982421875, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Nat. Commun. 8, 14516 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 236.51998901367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.67999267578125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. \\nAlbert, S. et al. Proteasomes tether to two distinct sites at the nuclear pore complex. Proc Natl Acad \\n', metadata={'x0': 72.0, 'x1': 536.760986328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 464.760986328125, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Sci USA 114, 13726–13731 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 253.0800018310547, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 159.24000549316406, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. \\nBäuerlein, F. J. B. et al. In situ architecture and cellular interactions of polyq inclusions. Cell 171, \\n', metadata={'x0': 72.0, 'x1': 527.6397705078125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 455.6397705078125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='179-187.e10 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 186.59999084472656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 92.75999450683594, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='5. \\nGuo, Q. et al. In Situ Structure of Neuronal C9orf72 Poly-GA Aggregates Reveals Proteasome \\n', metadata={'x0': 72.0, 'x1': 513.7218017578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.7218017578125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Recruitment. Cell 172, 696-705.e12 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 289.55999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 195.72000122070312, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. \\nAlbert, S. et al. Direct visualization of degradation microcompartments at the ER membrane. Proc \\n', metadata={'x0': 72.0, 'x1': 530.0406494140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.0406494140625, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Natl Acad Sci USA 117, 1069–1080 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 289.08001708984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 195.24002075195312, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='7. \\nWilfling, F. et al. A Selective Autophagy Pathway for Phase-Separated Endocytic Protein Deposits. \\n', metadata={'x0': 72.0, 'x1': 536.040771484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 464.040771484375, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Mol. Cell 80, 764-778.e7 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 242.0399932861328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 148.1999969482422, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='8. \\nWatanabe, R. et al. The In Situ Structure of Parkinson’s Disease-Linked LRRK2. Cell 182, 1508-\\n', metadata={'x0': 72.0, 'x1': 523.7564086914062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.75640869140625, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1518.e16 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 171.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 78.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='9. \\nBöhning, J. & Bharat, T. A. M. Towards high-throughput in situ structural biology using electron \\n', metadata={'x0': 72.0, 'x1': 525.005126953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.005126953125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cryotomography. Prog. Biophys. Mol. Biol. 160, 97–103 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 380.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 286.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='10. Weis, F. & Hagen, W. J. H. Combining high throughput and high quality for cryo-electron \\n', metadata={'x0': 72.0, 'x1': 494.76055908203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 422.76055908203125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='microscopy data collection. Acta Crystallogr. D Struct. Biol. 76, 724–728 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 455.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 361.8000183105469, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='11. Klumpe, S. et al. A modular platform for automated cryo-FIB workflows. eLife 10, (2021). \\n', metadata={'x0': 72.0, 'x1': 497.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 425.6400146484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='12. Zachs, T. et al. Fully automated, sequential focused ion beam milling for cryo-electron tomography. \\n', metadata={'x0': 72.0, 'x1': 537.720703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 465.720703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='eLife 9, (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 164.51998901367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 70.67999267578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='13. Pyle, E. & Zanetti, G. Current data processing strategies for cryo-electron tomography and \\n', metadata={'x0': 72.0, 'x1': 495.48193359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 423.48193359375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='subtomogram averaging. Biochem. J. 478, 1827–1845 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 369.2400207519531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 275.4000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='31 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='14. Lučič, V., Rigort, A. & Baumeister, W. Cryo-electron tomography: the challenge of doing structural \\n', metadata={'x0': 72.0, 'x1': 538.2041625976562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.20416259765625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='biology in situ. J. Cell Biol. 202, 407–419 (2013). \\n', metadata={'x0': 93.83999633789062, 'x1': 316.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 222.36001586914062, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='15. Wan, W. & Briggs, J. A. G. Cryo-Electron Tomography and Subtomogram Averaging. Meth. \\n', metadata={'x0': 72.0, 'x1': 507.9606018066406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 435.9606018066406, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Enzymol. 579, 329–367 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 235.3199920654297, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 141.47999572753906, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='16. Wu, X., Zeng, X., Zhu, Z., Gao, X. & Xu, M. Template-Based and Template-Free Approaches in \\n', metadata={'x0': 72.0, 'x1': 524.28173828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.28173828125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular Cryo-Electron Tomography Structural Pattern Mining. in Computational Biology (ed. Husi, \\n', metadata={'x0': 93.83999633789062, 'x1': 537.9610595703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.1210632324219, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='H.) (Codon Publications, 2019). doi:10.15586/computationalbiology.2019.ch11. \\n', metadata={'x0': 93.83999633789062, 'x1': 449.8800048828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 356.0400085449219, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='17. Bohm, J. et al. Toward detecting and identifying macromolecules in a cellular context: template \\n', metadata={'x0': 72.0, 'x1': 518.7681274414062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 446.76812744140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='matching applied to electron tomograms. Proc Natl Acad Sci USA 97, 14245–14250 (2000). \\n', metadata={'x0': 93.83999633789062, 'x1': 501.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 408.1200256347656, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='18. Bäuerlein, F. J. B. & Baumeister, W. Towards visual proteomics at high resolution. J. Mol. Biol. \\n', metadata={'x0': 72.0, 'x1': 520.9199829101562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.91998291015625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='433, 167187 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 186.36000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 92.52000427246094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='19. Xu, M., Beck, M. & Alber, F. Template-free detection of macromolecular complexes in cryo \\n', metadata={'x0': 72.0, 'x1': 505.3250732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.3250732421875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='electron tomograms. Bioinformatics 27, i69-76 (2011). \\n', metadata={'x0': 93.83999633789062, 'x1': 337.8000183105469, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 243.96002197265625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='20. Martinez-Sanchez, A. et al. Template-free detection and classification of membrane-bound \\n', metadata={'x0': 72.0, 'x1': 497.39996337890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 425.39996337890625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='complexes in cryo-electron tomograms. Nat. Methods 17, 209–216 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 425.8800048828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 332.0400085449219, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='21. Krizhevsky, A., Sutskever, I. & Hinton, G. E. ImageNet classification with deep convolutional \\n', metadata={'x0': 72.0, 'x1': 513.4891967773438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.48919677734375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='neural networks. Commun. ACM 60, 84–90 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 322.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 228.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='22. LeCun, Y. Deep learning & convolutional networks. in 2015 IEEE Hot Chips 27 Symposium (HCS) \\n', metadata={'x0': 72.0, 'x1': 535.5599975585938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 463.55999755859375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1–95 (IEEE, 2015). doi:10.1109/HOTCHIPS.2015.7477328. \\n', metadata={'x0': 93.83999633789062, 'x1': 363.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 270.1200256347656, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='23. LeCun, Y., Bengio, Y. & Hinton, G. Deep learning. Nature 521, 436–444 (2015). \\n', metadata={'x0': 72.0, 'x1': 455.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 383.6400146484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='24. Chen, M. et al. Convolutional neural networks for automated annotation of cellular cryo-electron \\n', metadata={'x0': 72.0, 'x1': 523.8001708984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.8001708984375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomograms. Nat. Methods 14, 983–985 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 303.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 209.6400146484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='25. Ronneberger, O., Fischer, P. & Brox, T. U-Net: Convolutional Networks for Biomedical Image \\n', metadata={'x0': 72.0, 'x1': 516.6036987304688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.60369873046875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Navab, N., Hornegger, J., Wells, W. M. & Frangi, A. F.) vol. 9351 234–241 (Springer International \\n', metadata={'x0': 93.83999633789062, 'x1': 534.6023559570312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 440.7623596191406, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Publishing, 2015). \\n', metadata={'x0': 93.83999633789062, 'x1': 177.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 83.63999938964844, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='32 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='26. Gubbins, I. et al. Classification in Cryo-Electron Tomograms. \\n', metadata={'x0': 72.0, 'x1': 369.4820861816406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 297.4820861816406, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='https://diglib.eg.org/handle/10.2312/3dor20191061. \\n', metadata={'x0': 93.83999633789062, 'x1': 324.6000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 230.760009765625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='27. Moebel, E., Martinez, A., Larivière, D. & Ortiz, J. 3D ConvNet improves macromolecule \\n', metadata={'x0': 72.0, 'x1': 490.6821594238281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.6821594238281, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='localization in 3D cellular cryo-electron tomograms. (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 362.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.44000244140625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='28. Moebel, E. et al. Deep learning improves macromolecule identification in 3D cellular cryo-electron \\n', metadata={'x0': 72.0, 'x1': 534.840576171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 462.840576171875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomograms. Nat. Methods 18, 1386–1394 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 314.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 220.68002319335938, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='29. Gubins, I. et al. SHREC 2021: Classification in Cryo-electron Tomograms. The Eurographics \\n', metadata={'x0': 72.0, 'x1': 510.6011657714844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.6011657714844, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Association (2021) doi:10.2312/3dor.20211307. \\n', metadata={'x0': 93.83999633789062, 'x1': 308.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 214.44000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='30. Srivastava, N., Hinton, G., Krizhevsky, A., Sutskever, I. & Salakhutdinov, R. Dropout: A Simple \\n', metadata={'x0': 72.0, 'x1': 524.2863159179688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.28631591796875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Way to Prevent Neural Networks from Overfitting. J Mach Learn Res 15, (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 456.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 362.2799987792969, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='31. Ioffe, S. Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate \\n', metadata={'x0': 72.0, 'x1': 538.2088623046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.2088623046875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Shift. in vol. 37 (Association for Computing Machinery, 2015). \\n', metadata={'x0': 93.83999633789062, 'x1': 375.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 281.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='32. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Nat. \\n', metadata={'x0': 72.0, 'x1': 533.3994140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.3994140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Methods 16, 1146–1152 (2019). \\n', metadata={'x0': 93.83999633789062, 'x1': 237.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 144.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='33. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement \\n', metadata={'x0': 72.0, 'x1': 540.8409423828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 468.8409423828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='with M visualizes ribosome-antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 533.1600341796875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 439.3200378417969, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='34. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in \\n', metadata={'x0': 72.0, 'x1': 511.0824890136719, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 439.0824890136719, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='RELION-3. eLife 7, (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 219.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 125.63999938964844, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='35. Castaño-Díez, D. The Dynamo package for tomography and subtomogram averaging: components \\n', metadata={'x0': 72.0, 'x1': 530.2884521484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.2884521484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='for MATLAB, GPU computing and EC2 Amazon Web Services. Acta Crystallogr. D Struct. Biol. \\n', metadata={'x0': 93.83999633789062, 'x1': 528.8400268554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 435.0000305175781, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='73, 478–487 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 186.36000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 92.52000427246094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='36. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron \\n', metadata={'x0': 72.0, 'x1': 536.5205688476562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 464.52056884765625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomograms and subtomogram analysis. J. Struct. Biol. 178, 177–188 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 432.8399963378906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 339.0, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='37. Chen, M. et al. A complete data processing workflow for cryo-ET and subtomogram averaging. Nat. \\n', metadata={'x0': 72.0, 'x1': 539.880615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 467.880615234375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Methods 16, 1161–1168 (2019). \\n', metadata={'x0': 93.83999633789062, 'x1': 237.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 144.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='38. Danev, R., Buijsse, B., Khoshouei, M., Plitzko, J. M. & Baumeister, W. Volta potential phase plate \\n', metadata={'x0': 72.0, 'x1': 533.4031372070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.40313720703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='33 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='for in-focus phase contrast transmission electron microscopy. Proc Natl Acad Sci USA 111, 15635–\\n', metadata={'x0': 93.83999633789062, 'x1': 532.3200073242188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.4800109863281, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='15640 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 159.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 65.16000366210938, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='39. Danev, R., Tegunov, D. & Baumeister, W. Using the Volta phase plate with defocus for cryo-EM \\n', metadata={'x0': 72.0, 'x1': 526.2044067382812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20440673828125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='40. Zimmerli, C. E. et al. Nuclear pores dilate and constrict in cellulo. Science 374, eabd9776 (2021). \\n', metadata={'x0': 72.0, 'x1': 526.4400024414062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.44000244140625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='41. Carpy, A. et al. Absolute proteome and phosphoproteome dynamics during the cell cycle of \\n', metadata={'x0': 72.0, 'x1': 500.2835998535156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 428.2835998535156, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='42. Marguerat, S. et al. Quantitative analysis of fission yeast transcriptomes and proteomes in \\n', metadata={'x0': 72.0, 'x1': 492.12591552734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 420.12591552734375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='proliferating and quiescent cells. Cell 151, 671–683 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 359.4000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.5600280761719, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='43. Turoňová, B. et al. Benchmarking tomographic acquisition schemes for high-resolution structural \\n', metadata={'x0': 72.0, 'x1': 526.4415283203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.4415283203125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='biology. Nat. Commun. 11, 876 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 270.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 176.27999877929688, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='44. Leibundgut, M., Jenni, S., Frick, C. & Ban, N. Structural basis for substrate delivery by acyl carrier \\n', metadata={'x0': 72.0, 'x1': 533.8854370117188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.88543701171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='protein in the yeast fatty acid synthase. Science 316, 288–290 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 402.3600158691406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 308.52001953125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='45. Snowden, J. S. et al. Structural insight into Pichia pastoris fatty acid synthase. Sci. Rep. 11, 9773 \\n', metadata={'x0': 72.0, 'x1': 522.6004638671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.6004638671875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='46. Gipson, P. et al. Direct structural insight into the substrate-shuttling mechanism of yeast fatty acid \\n', metadata={'x0': 72.0, 'x1': 528.8413696289062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 456.84136962890625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='synthase by electron cryomicroscopy. Proc Natl Acad Sci USA 107, 9164–9169 (2010). \\n', metadata={'x0': 93.83999633789062, 'x1': 482.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 388.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='47. Singh, K. et al. Discovery of a regulatory subunit of the yeast fatty acid synthase. Cell 180, 1130-\\n', metadata={'x0': 72.0, 'x1': 523.5164184570312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.51641845703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1143.e20 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 171.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 78.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='48. Kastritis, P. L. et al. Capturing protein communities by structural proteomics in a thermophilic \\n', metadata={'x0': 72.0, 'x1': 513.0034790039062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.00347900390625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='eukaryote. Mol. Syst. Biol. 13, 936 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 283.55999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 189.72000122070312, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='49. Jenni, S. et al. Structure of fungal fatty acid synthase and implications for iterative substrate \\n', metadata={'x0': 72.0, 'x1': 501.48443603515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 429.48443603515625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='shuttling. Science 316, 254–261 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 272.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 178.44000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='50. Ranjan, N. et al. Yeast translation elongation factor eEF3 promotes late stages of tRNA \\n', metadata={'x0': 72.0, 'x1': 482.0445861816406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 410.0445861816406, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='translocation. EMBO J. 40, e106449 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 291.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 197.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='51. Andersen, C. B. F. et al. Structure of eEF3 and the mechanism of transfer RNA release from the E-\\n', metadata={'x0': 72.0, 'x1': 530.7163696289062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.71636962890625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='34 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='site. Nature 443, 663–668 (2006). \\n', metadata={'x0': 93.83999633789062, 'x1': 245.87998962402344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 152.0399932861328, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='52. Becker, T. et al. Structure of monomeric yeast and mammalian Sec61 complexes interacting with the \\n', metadata={'x0': 72.0, 'x1': 541.56298828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 469.56298828125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='53. Armache, J.-P. et al. Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome \\n', metadata={'x0': 72.0, 'x1': 532.6869506835938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 460.68695068359375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='at 5.5-A resolution. Proc Natl Acad Sci USA 107, 19748–19753 (2010). \\n', metadata={'x0': 93.83999633789062, 'x1': 412.44000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 318.6000061035156, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='54. Shankar, V. et al. rRNA expansion segment 27Lb modulates the factor recruitment capacity of the \\n', metadata={'x0': 72.0, 'x1': 529.5655517578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 457.5655517578125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='yeast ribosome and shapes the proteome. Nucleic Acids Res. 48, 3244–3256 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 465.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 371.6400146484375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='55. Fujii, K., Susanto, T. T., Saurabh, S. & Barna, M. Decoding the function of expansion segments in \\n', metadata={'x0': 72.0, 'x1': 531.0042114257812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 459.00421142578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='ribosomes. Mol. Cell 72, 1013-1020.e6 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 303.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 209.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='56. Greber, B. J. et al. Insertion of the Biogenesis Factor Rei1 Probes the Ribosomal Tunnel during 60S \\n', metadata={'x0': 72.0, 'x1': 537.2453002929688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 465.24530029296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Maturation. Cell 164, 91–102 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 261.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 168.12002563476562, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='57. Wild, K. et al. MetAP-like Ebp1 occupies the human ribosomal tunnel exit and recruits flexible \\n', metadata={'x0': 72.0, 'x1': 517.323486328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 445.323486328125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='rRNA expansion segments. Nat. Commun. 11, 776 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 354.6000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 260.760009765625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='58. Kowalinski, E. et al. The crystal structure of Ebp1 reveals a methionine aminopeptidase fold as \\n', metadata={'x0': 72.0, 'x1': 516.1268920898438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.12689208984375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='binding platform for multiple interactions. FEBS Lett. 581, 4450–4454 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 442.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 348.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='59. Gold, V. A., Chroscicki, P., Bragoszewski, P. & Chacinska, A. Visualization of cytosolic ribosomes \\n', metadata={'x0': 72.0, 'x1': 537.2483520507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 465.24835205078125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='on the surface of mitochondria by electron cryo-tomography. EMBO Rep. 18, 1786–1800 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 524.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 430.9200134277344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='60. Avendaño-Monsalve, M. C., Ponce-Rojas, J. C. & Funes, S. From cytosol to mitochondria: the \\n', metadata={'x0': 72.0, 'x1': 513.9610595703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.9610595703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='beginning of a protein journey. Biol. Chem. 401, 645–661 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 386.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 292.44000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='61. Tucker, K. & Park, E. Cryo-EM structure of the mitochondrial protein-import channel TOM \\n', metadata={'x0': 72.0, 'x1': 502.9219970703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 430.9219970703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='complex at near-atomic resolution. Nat. Struct. Mol. Biol. 26, 1158–1166 (2019). \\n', metadata={'x0': 93.83999633789062, 'x1': 453.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 359.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='62. Lesnik, C., Cohen, Y., Atir-Lande, A., Schuldiner, M. & Arava, Y. OM14 is a mitochondrial \\n', metadata={'x0': 72.0, 'x1': 505.087158203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.087158203125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='receptor for cytosolic ribosomes that supports co-translational import into mitochondria. Nat. \\n', metadata={'x0': 93.83999633789062, 'x1': 506.5205383300781, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 412.6805419921875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Commun. 5, 5711 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 209.63999938964844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 115.80000305175781, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='63. George, R., Walsh, P., Beddoe, T. & Lithgow, T. The nascent polypeptide-associated complex \\n', metadata={'x0': 72.0, 'x1': 512.760986328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 440.760986328125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(NAC) promotes interaction of ribosomes with the mitochondrial surface in vivo. FEBS Lett. 516, \\n', metadata={'x0': 93.83999633789062, 'x1': 527.4002685546875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.5602722167969, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='35 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='213–216 (2002). \\n', metadata={'x0': 93.83999633789062, 'x1': 169.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='64. Stalling, D., Westerhoff, M. & Hege, H.-C. amira: A Highly Interactive System for Visual Data \\n', metadata={'x0': 72.0, 'x1': 517.5670776367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 445.56707763671875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Analysis. in Visualization Handbook 749–767 (Elsevier, 2005). doi:10.1016/B978-012387582-\\n', metadata={'x0': 93.83999633789062, 'x1': 512.7164306640625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.8764343261719, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2/50040-X. \\n', metadata={'x0': 93.83999633789062, 'x1': 147.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 53.160003662109375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='65. Vignaud, T. et al. Stress fibres are embedded in a contractile cortical network. Nat. Mater. 20, 410–\\n', metadata={'x0': 72.0, 'x1': 533.0400390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.0400390625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='420 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 147.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 54.1199951171875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='66. Bharat, T. A. M. & Scheres, S. H. W. Resolving macromolecular structures from electron cryo-\\n', metadata={'x0': 72.0, 'x1': 514.3964233398438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 442.39642333984375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 524.280029296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 430.4400329589844, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='67. Wagner, F. R. et al. Preparing samples from whole cells using focused-ion-beam milling for cryo-\\n', metadata={'x0': 72.0, 'x1': 525.9163818359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9163818359375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='electron tomography. Nat. Protoc. 15, 2041–2070 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 351.2400207519531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 257.4000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='68. Schaffer, M. et al. Optimized cryo-focused ion beam sample preparation aimed at in situ structural \\n', metadata={'x0': 72.0, 'x1': 530.5262451171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.5262451171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='studies of membrane proteins. J. Struct. Biol. 197, 73–82 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 382.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 288.3600158691406, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='69. Mastronarde, D. N. Advanced data acquisition from electron microscopes with serialem. Microsc. \\n', metadata={'x0': 72.0, 'x1': 527.8800048828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 455.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Microanal. 24, 864–865 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 238.4399871826172, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 144.59999084472656, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='70. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-\\n', metadata={'x0': 72.0, 'x1': 520.1563720703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.1563720703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 537.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 443.1600036621094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='71. Allegretti, M. et al. In-cell architecture of the nuclear pore and snapshots of its turnover. Nature 586, \\n', metadata={'x0': 72.0, 'x1': 540.3598022460938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 468.35980224609375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='796–800 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 169.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='73. Mastronarde, D. N. & Held, S. R. Automated tilt series alignment and tomographic reconstruction in \\n', metadata={'x0': 72.0, 'x1': 539.6473999023438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 467.64739990234375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IMOD. J. Struct. Biol. 197, 102–113 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 292.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 198.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='74. Tang, G. et al. EMAN2: an extensible image processing suite for electron microscopy. J. Struct. \\n', metadata={'x0': 72.0, 'x1': 520.200927734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.200927734375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Biol. 157, 38–46 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 204.83999633789062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 111.0, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='75. Wu, J.-Q. & Pollard, T. D. Counting cytokinesis proteins globally and locally in fission yeast. \\n', metadata={'x0': 72.0, 'x1': 509.4070129394531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 437.4070129394531, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Science 310, 310–314 (2005). \\n', metadata={'x0': 93.83999633789062, 'x1': 228.36000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 134.52000427246094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='76. Mitchison, J. M. The growth of single cells. Exp. Cell Res. 13, 244–262 (1957). \\n', metadata={'x0': 72.0, 'x1': 447.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 375.0, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='77. Nurse, P. Genetic control of cell size at cell division in yeast. Nature 256, 547–551 (1975). \\n', metadata={'x0': 72.0, 'x1': 496.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 424.6800231933594, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='36 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='78. Turonova. turonova/novaSTA: novaSTA. Zenodo (2020) doi:10.5281/zenodo.3973623. \\n', metadata={'x0': 72.0, 'x1': 482.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 410.0400085449219, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='79. Rigort, A. et al. Automated segmentation of electron tomograms for a quantitative description of \\n', metadata={'x0': 72.0, 'x1': 522.36474609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.36474609375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='actin filament networks. J. Struct. Biol. 177, 135–144 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 367.08001708984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 273.2400207519531, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='80. Redemann, S. et al. The segmentation of microtubules in electron tomograms using Amira. Methods \\n', metadata={'x0': 72.0, 'x1': 538.6806640625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.6806640625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Mol. Biol. 1136, 261–278 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 244.4399871826172, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 150.59999084472656, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='81. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. \\n', metadata={'x0': 72.0, 'x1': 535.3240966796875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 463.3240966796875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='J. Comput. Chem. 25, 1605–1612 (2004). \\n', metadata={'x0': 93.83999633789062, 'x1': 279.2400207519531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 185.4000244140625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='82. Goddard, T. D. et al. UCSF ChimeraX: Meeting modern challenges in visualization and analysis. \\n', metadata={'x0': 72.0, 'x1': 525.0051879882812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.00518798828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Protein Sci. 27, 14–25 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 230.0399932861328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 136.1999969482422, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='83. Moebel, E. et al. Deep Learning Improves Macromolecules Localization and Identification in 3D \\n', metadata={'x0': 72.0, 'x1': 524.5264892578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.5264892578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular Cryo-Electron Tomograms. BioRxiv (2020) doi:10.1101/2020.04.15.042747. \\n', metadata={'x0': 93.83999633789062, 'x1': 472.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 378.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 1 | Pre-processing of input data by spectrum equalization. a. Input DEF tomogram with \\nlow signal-to-noise ratio. b. Target VPP tomogram with high signal-to-noise ratio. c. Output showing the \\ntomogram in a after spectrum matching. d. Amplitude spectra of target, input and output tomograms, as well as \\nper-frequency (1/Å) scaling factors of the rotational equalization kernel (y-axis cropped). \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9611206054688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96112060546875, 'height': 63.269989013671875, 'nlines': 6, 'fs': 10.544998168945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='  \\n', metadata={'x0': 72.0, 'x1': 78.41999816894531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 6.4199981689453125, 'height': 17.807342529296875, 'nlines': 1, 'fs': 17.807342529296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 2 | 2D CNN organelle and cytosol segmentations. Isosurface view of organelle ground \\ntruth annotations (a), model output prior to (b) and after 3D post-processing (c). d-f. Organelle (d) and cytosol \\n(e) predictions of the tomogram in Fig. 1 b. The 2D CNN’s segmentations (red) are overlaid with the manually \\ncreated ground truth (blue), with overlapping regions (i.e. true positives) in white. Ground truth annotations in the \\ntop and bottom left corners in e (yellow arrowheads) are artifacts resulting from cross-slice interpolation in the \\nmanual annotation process. f. 3D post-processing improves performance of the 2D CNN as exemplified in (a-c). \\n', metadata={'x0': 72.0, 'x1': 525.960205078125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.960205078125, 'height': 81.86553955078125, 'nlines': 7, 'fs': 11.695077078683036, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 3 | Details of the construction of ground truth for RIBO and FAS in VPP and DEF \\ntomograms. a-b. Contributions of the steps applied during the construction of RIBO ground truth in the VPP (a) \\nand DEF (b) datasets. Upper subplots show the absolute number of particles per tomogram (ppt): true positives \\nfrom the initial annotations that were recovered in step 2 (yellow), newly identified true positives in step 2 (blue), \\nparticles that were in the initial annotation but that were not recovered in step 2 (false negatives from initial \\nannotation; pink), unidentified particles from any of the two first steps, which were manually picked in step 3 \\n(fuchsia), and total false negatives (red) in step 2. Lower subplots show the associated relative contribution, as a \\npercentage of the ground truth per tomogram (ground truth panel) or as percentage of the predictions per tomogram \\nin step 2 (prediction panel). c-d. Equivalent plots for FAS ground truth construction, where the only difference is \\nthat the recovered true positives originate from an initial manual annotation (green), as opposed to RIBO where \\nwe used TM additionally. e-f. Summary of the relative contributions in the RIBO (e) and FAS (f) ground truth \\nacross the 10 VPP and 10 DEF tomograms. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9671020507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96710205078125, 'height': 162.81048583984375, 'nlines': 14, 'fs': 11.629320417131696, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 4 | S. pombe ribosome and FAS complex abundances in cryo-electron tomograms in \\ncomparison to proteomics analysis. In this study (cryo-ET, left), fully assembled ribosomes (a) and FAS (b) \\nwere iteratively annotated in 20 ground truth cryo-electron tomograms and cytosolic concentrations calculated \\nusing the cytosol segmentations. Copy numbers per cell were calculated for an S. pombe cell with the assumption \\nof 30 % cytosolic volume1 of a total of 150 µm3 average cell volume2,3. a. With cryo-ET, an average of 671,303 \\n土 96,708 ribosomes/cell were annotated. Ribosome counts from individual tomograms are represented as \\nindividual points (red). Proteomics data of individual ribosomal proteins were derived from Marguerat et al.4 \\n(yellow) and Carpy et al.5 (purple) and resulted in average of 97,795 土 99,298 and 343,511 土 244,226 \\nribosomes/cell, respectively. b. Each measurement displayed in the plot corresponds to 6 times fully assembled \\nFAS counts per tomogram (the complex is constituted by six alpha and beta subunits). With cryo-ET, an average \\nof 106,282 土 86,247 FAS subunits/cell were observed. Proteomics data of individual FAS proteins (subunit alpha \\nin purple, beta in yellow) were derived from Marguerat et al.4 and Carpy et al.5 and resulted in average of 67,035 \\n土 12,600 and 202,263 土 47,086 FAS subunits/cell, respectively. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9640502929688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96405029296875, 'height': 204.4659423828125, 'nlines': 17, 'fs': 12.027408375459558, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 5 | Effects of spectrum equalization on cross-validation and performance evaluation \\nacross domains of the 2D CNN. a-f. Performance was evaluated for 2D networks in three different scenarios \\n(training on VPP in cross-validation, cross-domain generalization from VPP to DEF, training on VPP and DEF in \\ncross validation), for either organelles or cytoplasm, without (-EQ) and with (+EQ) spectrum equalization. Areas \\nunder the precision-recall curves (AUPRCs) were computed by randomly picking 1000 voxels from each \\ntomogram in test set (for the two cross-validation scenarios) or from the DEF tomograms (for the cross-domain \\ntasks), while restricting evaluation to z-slices with any positive label. AUPRCs were computed after 3D post-\\nprocessing. The baselines are defined as the fraction of positive labels within those z-slices and averaged across \\ntest tomograms. g-h. Precision, recall and F1-score of organelle (g) and cytosol (h) segmentation depending on \\nprobability threshold used. Scores were computed on the cross-validation results for the spectrum-equalized VPP \\ntomograms. \\n', metadata={'x0': 72.0, 'x1': 525.9631958007812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96319580078125, 'height': 138.73834228515625, 'nlines': 12, 'fs': 11.561528523763021, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 6 | Hyperparameter tuning and performance analysis of 3D CNN for RIBO, FAS, \\nmembranes and NPCs. a-c. CV analysis for the performance of RIBO, FAS and membranes, respectively. In \\nthese experiments we follow an incremental search: from left to right, the starting subplot corresponds to the \\ndefault configuration of the 3D CNN, D=2, BN=0, and (ED, DD)=(0,0), except for variable IF. The best value of \\nIF (denoted by IF*) is fixed in the subsequent subplot to the right, where D is varied over 1, 2, and 3. The next \\nsubplot to the right compares the effect of not applying and applying the batch normalization layer (BN=0 and \\nBN=1, respectively), under the best IF=IF* and D=D* from previous subplots. Finally, the right most subplot \\nvaries the encoder- and decoder-dropout parameters (ED, DD), with fixed best IF=IF*, D=D*, and BN=BN* from \\nprevious subplots. d-f. Results of hyperparameter exploration for NPC; in all DEF* tomograms (left), in the subset \\nof high-quality DEF* tomograms (middle), and in the subset of lower quality DEF* tomograms (right). d. Shows \\nthe variations in IF, when D=2, BN=0, and (ED, DD) = (0,0); e. Variations in IF, when D=3, BN=0, and (ED, \\nDD) = (0,0); f. Variations in IF, when D=3, BN=1, and (ED, DD) = (0,0). The summary of best hyperparameters’ \\ncombination is provided in Supplementary Table 1. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9760131835938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.97601318359375, 'height': 160.84197998046875, 'nlines': 15, 'fs': 10.722798665364584, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 7 | Effects of spectrum equalization and cytosol masking on DeePiCt performance. a-c \\nPerformance of the DeePiCt pipeline within the VPP domain for RIBO, FAS and membranes, without (-EQ) and \\nwith (+EQ) spectrum equalization pre-processing. The F1 score distribution shows that although for RIBO and \\nmembranes the spectrum equalization does not have a clear positive effect, for FAS it brings better results with a \\nshift in median F1 score from 0.25 to 0.46. d-f. The same networks were used to test performance of the DeePiCt \\npipeline across the VPP and DEF domains (training on VPP and testing in DEF) for RIBO, FAS and membranes \\nwithout (-EQ) and with (+EQ) spectrum equalization pre-processing. The F1 score distribution shows that while \\nfor ribosomes the spectrum equalization does not bring benefits, for FAS and membrane prediction across \\ndomains, its use brings a positive median F1 shift from 0.03 to 0.15 and from 0.16 to 0.41, respectively. g-h. \\nEffects of employing a segmentation of cytosol as a region mask (+ mask) or not (- mask) during DeePiCt’s post-\\nprocessing, for the localization of RIBO and FAS. The plots show the results under a 3-fold CV in VPP data, \\nwhere the cytosol segmentation was obtained by a 2D CNN. g. RIBO localization does not show a significant \\nimprovement in F1 score when the cytosol masking is used. h. FAS results show a stronger difference in F1 score, \\nwith a shift of median F1 from 0.10 to 0.46. \\n', metadata={'x0': 72.0, 'x1': 525.9652099609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9652099609375, 'height': 173.29833984375, 'nlines': 15, 'fs': 11.55322265625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 8 | Ground truth annotations and 3D refinement of ribosomes and FAS in DEF \\ntomograms. a. Complete annotation of ribosomes (yellow, orange, green, dark and light blue; sub-populations \\ndescribed in (f) and in Extended Data Figure 9), FAS (pink), membranes (purple) and organelles (gray) in the \\ntomogram depicted in Fig. 4. Ribosome subclasses in the context of specific organelles were identified by a \\nmaximal distance of 25 nm to mitochondria (mito-bound, green) or ER (ER-bound, orange). Focused classification \\non densities close to the head of the 40S small ribosomal subunit or in proximity to the exit tunnel recovered \\nribosome subsets with head (dark blue) and exit tunnel density (cyan). b-e. FAS subtomogram averages (pink) fit \\nof the S. cerevisiae X-ray structure (b, cyan) and the single particle cryo-EM density of P. pastoris (c, cyan) \\nincluding the PPT domains. An extra density cannot be assigned. The cross section views close to the alpha-wheel \\nreveal three densities fitting three and one ACPs of S. cerevisiae and P. pastoris, respectively (white asterisks, \\nbottom). d. Overlay with FAS from VPP ground truth (gt) tomograms (gray) shows overall similarity, but the \\nACPs could not be resolved (white asterisks, bottom). e. FAS predicted by DeePiCt (gray) matches the DEF gt \\naverage with three ACP densities close to the alpha-wheel. f-h. Subtomogram average of all ribosomes (yellow) \\nmatches the DeePiCt- predicted (g, gray) and VPP-derived (h, gray) ribosome densities. i-k. Well-aligned \\nribosomes (yellow) detected by hierarchical 3D classification in RELION and refined in M (Supplementary Figure \\n3). j-k. Slicing through the average depicted in (i) at different axes reveals the PTC with a P-site tRNA and the \\nL1 stalk facing the E-site. \\n', metadata={'x0': 72.0, 'x1': 525.963134765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.963134765625, 'height': 206.73049926757812, 'nlines': 18, 'fs': 11.485027737087673, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 9 | Comparison of ribosome subtomogram averages in DEF ground truth (gt). a-e. \\nSubtomogram average of ribosomes (blue) with electron density in proximity to the head of the 40S small subunit \\n(white arrowhead) which fits the ribosome-bound eEF36 (CC 0.8972, red). b. Cross section of (a) showing the \\nPTC with P- and E-site tRNAs and the L1 stalk facing inwards. c. Zoom into the additional head density fitting \\neEF3. d-e. Overlay with the same ribosome class from DeePiCt predictions in DEF (d, gray) and from VPP gt \\ndata (e, gray). f. Ribosome subset (cyan) with electron density in proximity to the exit tunnel (white arrowhead). \\ng. Overlay with the S. cerevisiae ribosome7 (CC 0.9651, purple) and a specific conformation of the expansion \\nsegment ES27L8 (CC 0.7806, purple). h. Zoom into the exit tunnel area in (g) highlights the additional density \\nconnected to ES27L that colocalizes with for example the human Ebp19 (gray, lower panel, CC 0.8495, EMD-\\n1068). i-j. Overlay with the same ribosome class from DeePiCt predictions in DEF (i, gray) and from VPP gt data \\n(j, gray). k-n. Overlay of ribosomes within 25 nm distance to the ER (orange) with ER-bound ribosomes from \\nDeePiCt predictions in DEF (k, gray), from VPP gt data (l, gray), with the published density of S. cerevisiae \\nribosomes derived from extracted ER10 (k, gray, CC 0.8972, EMD-3764), and the ER-bound HeLa ribosome11 (n, \\ngray, CC 0.9105, EMD-8056). All densities show the same interface between ribosome and ER membrane. o-p. \\nOverlay of ribosomes within 25 nm distance to mitochondria (green) with mito-bound ribosomes from DeePiCt \\npredictions in DEF (o, gray), and VPP gt data (p, gray). All densities show the same interface between ribosome \\nand mitochondria membrane. q-r. Comparisons between ER-bound (orange) and mitochondria-bound (mito-\\nbound, green) subtomogram averages in DEF gt (q) and VPP gt (r) data reveal an angular offset of the membrane \\ninterfaces of 35° and 33°, respectively. \\n', metadata={'x0': 72.0, 'x1': 525.9635620117188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96356201171875, 'height': 218.0428466796875, 'nlines': 19, 'fs': 11.475939298930921, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 10 | Domain generalization of different cellular structures and species. a. Performance \\nresults of five DeePiCt networks trained on a variety of cell types for the segmentation of different cellular \\nstructures used for predictions in a HeLa cell dataset11. Left: plot of performance scores when employing the \\nspectrum equalization (+EQ) filter in both the training and on the HeLa cell data. The results correspond to \\nsegmentations of actin filaments and microtubules (MT) (trained on 4 RPE1 tomograms; pink), RIBO (trained on \\n10 VPP S. pombe tomograms; yellow-green), membranes (trained on 10 S. pombe VPP tomograms; aquamarine), \\nMT (trained on 11 C. elegans tomograms; blue) and actin filaments (trained on 2 RPE1 and 3 MEF 3T3 tomograms \\nwith actin filaments at different orientations; light purple). Right: performance for RIBO and membrane without \\nemploying the equalization filter (-EQ) in the HeLa cell dataset. b. RPE1 training data examples (left and middle, \\ntomograms on top, ground truth annotations at the bottom) and predictions (right) of the actin filament and MT \\nmodels in the HeLa cell dataset. The training examples show 2 of the RPE1 tomograms12 used for this model. The \\nperformance of actin and microtubule predictions in the HeLa cell dataset is poor (voxel-F1=0.09 for actin and \\n0.26 for MT; a left) despite the high-quality training data (left). This is likely due to the high unidirectional \\norientation of the filaments in the training data (bundles in stress fibers).  \\n', metadata={'x0': 72.0, 'x1': 525.9666748046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9666748046875, 'height': 172.20281982421875, 'nlines': 15, 'fs': 11.48018798828125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. \\nWu, J.-Q. & Pollard, T. D. Counting cytokinesis proteins globally and locally in fission yeast. Science 310, \\n', metadata={'x0': 72.0, 'x1': 522.8399658203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.8399658203125, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='310–314 (2005). \\n', metadata={'x0': 91.91999816894531, 'x1': 161.16000366210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 69.24000549316406, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. \\nMitchison, J. M. The growth of single cells. Exp. Cell Res. 13, 244–262 (1957). \\n', metadata={'x0': 72.0, 'x1': 412.91998291015625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 340.91998291015625, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. \\nNurse, P. Genetic control of cell size at cell division in yeast. Nature 256, 547–551 (1975). \\n', metadata={'x0': 72.0, 'x1': 458.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 386.2799987792969, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. \\nMarguerat, S. et al. Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and \\n', metadata={'x0': 72.0, 'x1': 523.3216552734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.3216552734375, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='quiescent cells. Cell 151, 671–683 (2012). \\n', metadata={'x0': 91.91999816894531, 'x1': 263.8799743652344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 171.95997619628906, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='5. \\nCarpy, A. et al. Absolute proteome and phosphoproteome dynamics during the cell cycle of \\n', metadata={'x0': 72.0, 'x1': 461.64208984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 389.64208984375, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. \\nRanjan, N. et al. Yeast translation elongation factor eEF3 promotes late stages of tRNA translocation. \\n', metadata={'x0': 72.0, 'x1': 501.7206726074219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 429.7206726074219, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='EMBO J. 40, e106449 (2021). \\n', metadata={'x0': 91.91999816894531, 'x1': 215.16000366210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 123.24000549316406, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='7. \\nBecker, T. et al. Structure of monomeric yeast and mammalian Sec61 complexes interacting with the \\n', metadata={'x0': 72.0, 'x1': 499.0824890136719, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 427.0824890136719, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='8. \\nArmache, J.-P. et al. Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome at 5.5-\\n', metadata={'x0': 72.0, 'x1': 516.7166748046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.7166748046875, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='A resolution. Proc Natl Acad Sci USA 107, 19748–19753 (2010). \\n', metadata={'x0': 91.91999816894531, 'x1': 356.03997802734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 264.11997985839844, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='9. \\nWild, K. et al. MetAP-like Ebp1 occupies the human ribosomal tunnel exit and recruits flexible rRNA \\n', metadata={'x0': 72.0, 'x1': 503.8804016113281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 431.8804016113281, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='expansion segments. Nat. Commun. 11, 776 (2020). \\n', metadata={'x0': 91.91999816894531, 'x1': 301.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 209.87998962402344, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='10. Gold, V. A., Chroscicki, P., Bragoszewski, P. & Chacinska, A. Visualization of cytosolic ribosomes on the \\n', metadata={'x0': 72.0, 'x1': 522.1256103515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.1256103515625, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='surface of mitochondria by electron cryo-tomography. EMBO Rep. 18, 1786–1800 (2017). \\n', metadata={'x0': 91.91999816894531, 'x1': 456.5999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 364.6799774169922, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='11. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science 351, \\n', metadata={'x0': 72.0, 'x1': 508.2000427246094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2000427246094, 'height': 11.162841796875, 'nlines': 1, 'fs': 11.162841796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='969–972 (2016). \\n', metadata={'x0': 91.91999816894531, 'x1': 161.16000366210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 69.24000549316406, 'height': 11.162841796875, 'nlines': 1, 'fs': 11.162841796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='12. Vignaud, T. et al. Stress fibres are embedded in a contractile cortical network. Nat. Mater. 20, 410–420 \\n', metadata={'x0': 72.0, 'x1': 508.44012451171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.44012451171875, 'height': 11.162841796875, 'nlines': 1, 'fs': 11.162841796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Information Guide \\n', metadata={'x0': 118.55999755859375, 'x1': 483.1200256347656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 364.5600280761719, 'height': 28.704376220703125, 'nlines': 1, 'fs': 28.704376220703125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Figures.docx \\n10 Figures supporting main text. \\n', metadata={'x0': 72.0, 'x1': 218.0399932861328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 146.0399932861328, 'height': 41.26591491699219, 'nlines': 3, 'fs': 13.755304972330729, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Material.docx \\nContents:  \\n', metadata={'x0': 72.0, 'x1': 215.1599884033203, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 143.1599884033203, 'height': 26.865890502929688, 'nlines': 2, 'fs': 13.432945251464844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Supplementary Note 1: Architectural description of 2D and 3D CNNs, post processing steps \\n', metadata={'x0': 90.0, 'x1': 516.8466186523438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 426.84661865234375, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='and evaluation metrics. Includes Supplementary Figure 1. \\n', metadata={'x0': 108.0, 'x1': 364.9200134277344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 256.9200134277344, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Supplementary Tables:  \\n', metadata={'x0': 90.0, 'x1': 215.1599884033203, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 125.15998840332031, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='○ Supplementary Table 1: Selected hyperparameters of DeePiCt CNNs for different \\n', metadata={'x0': 126.0, 'x1': 506.76580810546875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 380.76580810546875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cellular features. \\n', metadata={'x0': 144.0, 'x1': 219.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='○ Supplementary Table 2: Specifications of 3D-CNN training and prediction rounds \\n', metadata={'x0': 126.0, 'x1': 508.2021484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.2021484375, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='performed for FAS ground truth construction. \\n', metadata={'x0': 144.0, 'x1': 348.3600158691406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 204.36001586914062, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='○ Supplementary Table 3: Hyperparameter tuning experiment settings. \\n', metadata={'x0': 126.0, 'x1': 448.9200134277344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 322.9200134277344, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Supplementary Note 2: Considerations for structural analysis following CNN predictions \\n', metadata={'x0': 90.0, 'x1': 502.924072265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 412.924072265625, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='under different image acquisition conditions. Includes Supplementary Figures 2-9 detailing \\nsubtomogram hierarchical classification results. \\n', metadata={'x0': 108.0, 'x1': 512.2811889648438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 404.28118896484375, 'height': 26.865936279296875, 'nlines': 2, 'fs': 13.432968139648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Tables.xlsx \\nContents: Excel file with the following 6 tables as separate sheets: \\n', metadata={'x0': 72.0, 'x1': 366.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 294.1199951171875, 'height': 26.625946044921875, 'nlines': 2, 'fs': 13.312973022460938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Table 1: Organelle content per tomogram in DEF and VPP datasets. \\n● Table 2: RIBO gt construction. \\n● Table 3: FAS gt construction. \\n● Table 4: VPP gt data acquisition, refinement statistics. \\n● Table 5: DEF gt data acquisition, refinement statistics. \\n● Table 6: Refinement statistics for subtomograms from DeePiCt in DEF data. \\n', metadata={'x0': 90.0, 'x1': 447.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 357.7200012207031, 'height': 85.10226440429688, 'nlines': 6, 'fs': 14.183710734049479, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Note 1  \\n', metadata={'x0': 230.16000366210938, 'x1': 372.6600036621094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.5, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Architectural description of 2D and 3D CNN, post processing steps and \\n', metadata={'x0': 82.55999755859375, 'x1': 516.5902099609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 434.03021240234375, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='evaluation metrics \\n', metadata={'x0': 242.39999389648438, 'x1': 356.5799865722656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 114.17999267578125, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The general architecture of the 3D and 2D CNN is based also on the U-Net architecture1 and consists \\nof two consecutive symmetric paths: an encoder and a decoder path connected by skip-connections and \\na transitional base-layer (Fig. 1a), where:  \\n \\n(a) The encoder path is a sequence of alternated convolutional blocks and max-pooling layers. In turn, \\nthe convolutional blocks are a series of convolutions (kernel size 𝑘\" = \"3, stride 𝑠 = 1 and zero padding \\n𝑝\" = \"2) followed by a rectified linear unit (ReLU). In the case of the 3D CNN, optional batch \\nnormalization (BN) and dropout layers can be alternated (Fig. 1a). The max-pooling layer has fixed \\nwindow size 𝑤 = 2 and stride 𝑠 = 2. \\nIn our implementations, the initial convolutional block has a variable number of initial filters (IF) in its \\nfirst convolutional layer, which must be set by the user. In all cases, subsequent convolutional blocks \\nduplicate the number of feature maps from the previous level. The number of downsampling layers plus \\n1, is referred to as the network’s depth (D), which is also a hyperparameter to be set by the user. It \\ndetermines the size of the receptive field (𝑅𝐹) of the network (𝑅𝐹\" = \"2 ∗ 𝐷). For the 2D CNN, the \\ndepth is a fixed value 𝐷 = 5. \\n \\n(b) The base of the CNNs is the transitional convolutional layer between the encoder and decoder paths.  \\n \\n(c) The decoder path is symmetric with respect to the encoder path, and is a sequence of alternated \\nconvolutional and upsampling (via transposed convolution) layers2, with kernel size 𝑘 = 3, stride 𝑠 =\\n1 and padding 𝑝 = 1. For the 3D CNN, analogous optional layers can be alternated between \\nconvolutional and upsampling blocks (BN and dropout). The encoder and decoder paths’ associated \\nparameters are denoted throughout the text as ED and DD, respectively. \\nEssential to the U-Net architecture are the skip-connections, which consist of concatenating the feature \\nmaps from the encoder to the decoder path at every level. The last activation layer is in our case a \\nsigmoid function, compatible with our choice of the Dice Loss function3. By definition, this is a multi-\\nlabel approach, where single voxels may belong to multiple semantic classes, and which frees the user \\nfrom defining additional weighting parameters to compensate for class imbalance necessary for other \\nloss function choices such as categorical cross-entropy. In our experience, the use of Dice Loss provided \\nbetter qualitative results in all cases. \\nAll models were trained using Adam Optimiser4. Application of these networks to unseen tomograms \\nrequires scaling of their pixel size to match (at least approximately) that of the training data. \\n \\nData Augmentation \\nFor the 2D CNN, tiles are randomly flipped and rotated in 90-degree increments during training to \\nimprove generalization.  \\n \\nAlthough it did not provide improved results in our hands (not shown), data augmentation for training \\nthe 3D CNN is available and optional. The number of times that each of the original training examples \\nis augmented is given by a parameter 𝑛01 that can be set by the user. Each augmentation includes 4 \\n', metadata={'x0': 72.0, 'x1': 529.0800170898438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 457.08001708984375, 'height': 585.4017333984375, 'nlines': 40, 'fs': 14.635043334960937, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='different types of random volumetric transformations (applied to the original image and the labels, \\nexcept in the case of noise), that we describe in what follows. Let A in 𝑅2×2×2 be a single channel \\ncubic image of side-length 𝑛 > 0. Then:  \\n \\n1. Additive Gaussian noise. Given a predefined modulation factor 𝜖\" > \"0 , the Additive Gaussian \\nnoise transformation 𝐺8 is defined by 𝐺8(𝐴)\"= \"𝐴\" + \"𝜖𝐵, where 𝐵>?@ ~ 𝑁(0, 1) are independent and \\nidentically distributed random variables (i.i.d.r.v.), and 𝑁(𝜇, 𝜎E) denotes a normally distributed \\nGaussian random variable with mean 𝜇 and variance 𝜎E. \\n \\n2. Salt-and-pepper noise. Given user-defined modulation factor 𝑎 > 0 and a probability rate 1 > 𝑝\" >\\n\"0 , the salt-and-pepper noise transformation is defined by 𝑆H,I(𝐴)\"= \"𝐴\" + \"𝑎 ∗ 𝐵, where 𝐵>?@ ~ \\n𝑈(𝑝, [−1, 1]) are i.i.d.r.v., and 𝑈(𝑝, 𝐼) denotes a uniformly distributed random variable in a real interval \\n𝐼. \\n \\n3. Elastic transformation. Given coordinate entries of A that are located at a coarse regular grid of \\nthickness 𝑛O > 1 are displaced by a random vector of i.i.d.r.v. uniform random variables. To achieve a \\nsmooth displacement vector field for all the points of the image 𝐴 (i.e. on the fine grid), we generate the \\nelastically transformed image as the interpolated vector field (via polynomial interpolation of order 𝑘 >\\n0, defined by the user). \\n \\n4. Random rotation. Given a probability of rotation 𝑝 ∈ (0, 1) and angle range 𝛼 ∈ (0, 180), the image \\nA is rotated with a probability 𝑝 with and angle in [−𝛼, 𝛼], with respect to the 𝑧 −axis.  \\n \\n3D CNN default hyperparameters \\nA default, starting point set of hyperparameters values for the 3D CNN for this study: 𝐷 = 2, 𝐼𝐹 = 4, \\n𝐵𝑁 = 0, 𝐸𝐷 = 0, 𝐷𝐷 = 0, and 𝑛01 = 0. We used them to later, through testing variations on them, \\nstudy the optimal combinations of hyperparameters in the different segmentation and localization tasks \\n(Supplementary Table 1). \\n \\n2D CNN post-processing  \\nPost-processing of the 2D CNN includes the reassembly of predicted tile segmentations, by cropping \\n48 px on each side to reduce artifacts around the edges. Areas of sets of tiles still overlapping are \\naveraged after cropping. In order to remove inter-slide discontinuities, a Gaussian filter with parameter \\n𝜎 > 0 (default 𝜎 = 5) is then applied along the z axis.  \\n \\n3D CNN post-processing  \\n', metadata={'x0': 72.0, 'x1': 526.2054443359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.2054443359375, 'height': 533.7135620117188, 'nlines': 36, 'fs': 14.825376722547743, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Thresholding of the final probability map outputted by the last layer of the neural network \\n', metadata={'x0': 90.0, 'x1': 526.2054443359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2054443359375, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(default threshold is the unbiased value 0.5, but is set by the user in the pipeline’s configuration \\nfile). \\n', metadata={'x0': 108.0, 'x1': 526.2009887695312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.20098876953125, 'height': 26.865966796875, 'nlines': 2, 'fs': 13.4329833984375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. Clustering of the resulting thresholded map via the connected component labeling label \\n', metadata={'x0': 90.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.20001220703125, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='algorithm, using the function morphology.label of the scikit-image python library5.  \\n', metadata={'x0': 108.0, 'x1': 477.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 369.96002197265625, 'height': 13.15020751953125, 'nlines': 1, 'fs': 13.15020751953125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Integration with other segmentation maps, e.g. with the output of the 2D CNN. Importantly, a \\n', metadata={'x0': 90.0, 'x1': 526.2005004882812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.20050048828125, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='subsequent selection of clusters is made with respect to a region_mask provided by the user, \\nwhich consists of an auxiliary binary image defining a tomographic region of interest (Fig 1b). \\nWe then allow 3 types of options (intersection, contact, or colocalization) for employing the \\nregion_mask through the corresponding contact_mode parameter, as follows: \\n', metadata={'x0': 108.0, 'x1': 526.2008666992188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.20086669921875, 'height': 55.906005859375, 'nlines': 4, 'fs': 13.97650146484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='a. contact_mode: \\nintersection \\n', metadata={'x0': 126.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 400.20001220703125, 'height': 12.38226318359375, 'nlines': 2, 'fs': 6.191131591796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The region_mask is used to directly mask the output of point 2, by considering only the \\noverlapping region, e.g. including particles that are within an organelle’s mask. \\n', metadata={'x0': 144.0, 'x1': 526.2008666992188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.20086669921875, 'height': 26.625900268554688, 'nlines': 2, 'fs': 13.312950134277344, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='b. contact_mode: \\ncontact \\n', metadata={'x0': 126.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 400.20001220703125, 'height': 12.38226318359375, 'nlines': 2, 'fs': 6.191131591796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Only clusters that have voxels in common with the region of interest are kept, e.g. \\nselecting NPC on the nuclear envelope. \\n', metadata={'x0': 144.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.20001220703125, 'height': 26.865890502929688, 'nlines': 2, 'fs': 13.432945251464844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='c. contact_mode: colocalization  \\n', metadata={'x0': 126.0, 'x1': 279.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 153.96002197265625, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Only clusters whose centroid is located at a given distance (colocalization_radius) \\nfrom the region of interest are kept, e.g. particles within a distance to an organelle. \\n', metadata={'x0': 144.0, 'x1': 526.1995239257812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.19952392578125, 'height': 26.865890502929688, 'nlines': 2, 'fs': 13.432945251464844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='At this step, only clusters within the size range [𝑚, 𝑀] are kept, where 0 ≤ 𝑚\" < \"𝑀\" ≤ \"∞ are \\ndefined by the user in the configuration file. \\n \\n', metadata={'x0': 108.0, 'x1': 526.2000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2000732421875, 'height': 42.158233642578125, 'nlines': 3, 'fs': 14.052744547526041, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Fig. 1 | 3D CNN post-processing steps. The clustered 3D CNN output (yellow) and a \\nregion mask (white) are combined according to the three contact modes. Cluster centroids are represented \\nby the red dots.  \\n \\n', metadata={'x0': 108.0, 'x1': 525.962646484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 417.962646484375, 'height': 51.9305419921875, 'nlines': 4, 'fs': 12.982635498046875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. If the calculate_motl parameter is set to True, a list of the clusters’ centroids resulting from \\n', metadata={'x0': 90.0, 'x1': 526.2010498046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2010498046875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='previous steps is saved as a 4-columns comma separated values (csv) file, where the first \\ncolumn is the score (cluster size), and the three remaining columns indicate the (𝑥, 𝑦, 𝑧) centroid \\nvoxel coordinates. The list is sorted according to decreasing-score. \\n', metadata={'x0': 108.0, 'x1': 526.2003784179688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.20037841796875, 'height': 41.50592041015625, 'nlines': 3, 'fs': 13.835306803385416, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSegmentation evaluation \\nGiven a 3D cellular segmentation prediction 𝐼H]^_ (resulting either from the 2D or the 3D CNN), its \\nquality can be voxel-wise evaluated, by comparison to a ground truth binary image 𝐼`a, according to \\ndifferent metrics: \\n  \\n', metadata={'x0': 72.0, 'x1': 526.20263671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20263671875, 'height': 88.30596923828125, 'nlines': 6, 'fs': 14.717661539713541, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Voxel-based precision (P) is defined by: \\n', metadata={'x0': 90.0, 'x1': 288.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 198.1199951171875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑃\" =\\n', metadata={'x0': 283.44000244140625, 'x1': 304.16766357421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 20.7276611328125, 'height': 12.942893981933594, 'nlines': 1, 'fs': 12.942893981933594, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='|cdefg\"| , \\n', metadata={'x0': 312.9599914550781, 'x1': 350.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 37.800018310546875, 'height': 18.477554321289062, 'nlines': 1, 'fs': 18.477554321289062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='  \\nwhere the bars || indicate the number of entries different from 0. Thus, P is the proportion of \\nvoxels in the predicted segmentation that truly belong to the wanted structure. \\n', metadata={'x0': 108.0, 'x1': 526.1981201171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.1981201171875, 'height': 41.74595642089844, 'nlines': 3, 'fs': 13.915318806966146, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. Voxel-based recall (R) is defined by: \\n', metadata={'x0': 90.0, 'x1': 273.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 183.48001098632812, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑅\" =\\n', metadata={'x0': 283.20001220703125, 'x1': 304.16766357421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 20.9676513671875, 'height': 12.942886352539062, 'nlines': 1, 'fs': 12.942886352539062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='\"|chi|\\n, \\n', metadata={'x0': 316.55999755859375, 'x1': 350.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 34.20001220703125, 'height': 18.477554321289062, 'nlines': 2, 'fs': 9.238777160644531, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='thus indicating the proportion of voxels of the ground truth segmentation that were correctly \\nrecovered by the prediction. \\n', metadata={'x0': 108.0, 'x1': 516.6055297851562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 408.60552978515625, 'height': 26.865951538085938, 'nlines': 2, 'fs': 13.432975769042969, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Area \\nunder \\nthe \\nprecision-recall \\ncurve \\n(AUPRC).  \\n', metadata={'x0': 90.0, 'x1': 528.8400268554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.84002685546875, 'height': 12.38226318359375, 'nlines': 6, 'fs': 2.0637105305989585, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Since 𝐼H]^_ is dependent on the threshold value 0 ≤ 𝑡ℎ𝑟 ≤ \"1 applied to the probability map, \\nboth \\nprecision \\nand \\nrecall \\ndepend \\nin \\nii \\nas \\nwell, \\n𝑃 = 𝑃no] \\nand \\n𝑅 = 𝑅no].  \\n', metadata={'x0': 108.0, 'x1': 528.8400268554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 420.84002685546875, 'height': 30.489715576171875, 'nlines': 13, 'fs': 2.345362736628606, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The area under the precision-recall curve (AUPRC) is the area under the parametrized curve \\n(𝑃no], 𝑅no]) is a widely used tool to summarize the power of the method. \\n', metadata={'x0': 108.0, 'x1': 526.2052001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2052001953125, 'height': 28.63751220703125, 'nlines': 2, 'fs': 14.318756103515625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. Voxel-based \\nF1-score \\n(also \\nknown \\nas \\nSørensen-Dice \\ncoefficient). \\n', metadata={'x0': 90.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.20001220703125, 'height': 12.382293701171875, 'nlines': 7, 'fs': 1.7688991001674108, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='A common metric to calculate the performance of the method by combining the P and R values \\nis \\nsimply \\nthrough \\nthe \\nF1 \\nscore, \\ndefined \\nby \\ntheir \\nharmonic \\nmean: \\n', metadata={'x0': 108.0, 'x1': 526.2020263671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2020263671875, 'height': 26.865966796875, 'nlines': 12, 'fs': 2.23883056640625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝐹p \"=\"\\n', metadata={'x0': 288.0, 'x1': 316.1111755371094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 28.111175537109375, 'height': 15.102874755859375, 'nlines': 1, 'fs': 15.102874755859375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='E∗q∗r\\n', metadata={'x0': 317.5199890136719, 'x1': 340.4852294921875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 22.965240478515625, 'height': 9.285125732421875, 'nlines': 1, 'fs': 9.285125732421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='q\"sr . \\n', metadata={'x0': 319.9200134277344, 'x1': 346.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 26.279998779296875, 'height': 17.745941162109375, 'nlines': 1, 'fs': 17.745941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Notice that here no account of the dependence with respect to the 𝑡ℎ𝑟 value is used. \\n \\n', metadata={'x0': 108.0, 'x1': 478.9200134277344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 370.9200134277344, 'height': 27.518218994140625, 'nlines': 2, 'fs': 13.759109497070312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Particle detection evaluation \\nIn the case of particles as discrete objects that are small enough to be defined by their coordinates, the \\nevaluation of the object detection task requires the following definitions: \\n', metadata={'x0': 72.0, 'x1': 526.2066040039062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20660400390625, 'height': 42.601409912109375, 'nlines': 3, 'fs': 14.200469970703125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Given a sorted list of predicted particles’ centroids, one by one (in that order) they are either \\n', metadata={'x0': 90.0, 'x1': 526.1964721679688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.19647216796875, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='classified as false positives (fp) or true positives (tp). \\n', metadata={'x0': 108.0, 'x1': 343.8000183105469, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 235.80001831054688, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. A predicted particle coordinate is considered a true positive if and only if given a pre-defined \\n', metadata={'x0': 90.0, 'x1': 526.2000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2000732421875, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tolerance radius r (𝑟 = 10 vox for both RIBO and FAS), there exists a ground-truth coordinate \\nnot previously matched to any other predicted particle. \\n', metadata={'x0': 108.0, 'x1': 526.2012939453125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2012939453125, 'height': 27.518218994140625, 'nlines': 2, 'fs': 13.759109497070312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Then the coordinate-based precision (p), recall (r), F1 score and area under the precision-recall \\n', metadata={'x0': 90.0, 'x1': 526.221923828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.221923828125, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='curve (AUPRC) are defined by: \\n', metadata={'x0': 108.0, 'x1': 250.67999267578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.67999267578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑝\" =\\n', metadata={'x0': 108.0, 'x1': 128.00765991210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 20.007659912109375, 'height': 12.94293212890625, 'nlines': 1, 'fs': 12.94293212890625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='nH\"s\"tH, 𝑟\" =\"\\n', metadata={'x0': 131.0399932861328, 'x1': 187.71116638183594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 56.671173095703125, 'height': 17.76971435546875, 'nlines': 1, 'fs': 17.76971435546875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='nH\"s\"t2, 𝐹p \"=\\n', metadata={'x0': 189.36000061035156, 'x1': 247.52764892578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 58.16764831542969, 'height': 17.76971435546875, 'nlines': 1, 'fs': 17.76971435546875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='E∗H∗]\\n', metadata={'x0': 250.55999755859375, 'x1': 272.2720031738281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 21.712005615234375, 'height': 9.28515625, 'nlines': 1, 'fs': 9.28515625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='H\"s\"] \", and AUPRC is the area under the curve defined by the \\n', metadata={'x0': 252.0, 'x1': 526.2023315429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 274.20233154296875, 'height': 18.462890625, 'nlines': 1, 'fs': 18.462890625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(𝑝, 𝑟) points when parametrized with respect to the cluster number (from 0 to all). \\n \\nMerging several lists for ground truth construction \\nTo account for possible duplicates when merging lists of particle coordinates from different methods \\n(e.g. manually-, TM- and CNN-originated lists), we integrate them by restricting the distance between \\ncentroid coordinates, i.e. with the constraint 𝑑(𝑝, 𝑞) > 1, where the elliptic distance 𝑑(𝑝, 𝑞) is defined \\nby: \\n \\n', metadata={'x0': 72.0, 'x1': 526.2056884765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.2056884765625, 'height': 116.3182373046875, 'nlines': 8, 'fs': 14.539779663085938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑑(𝑝, 𝑞)E \"=\\n', metadata={'x0': 154.55999755859375, 'x1': 205.7676544189453, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 51.20765686035156, 'height': 17.02288818359375, 'nlines': 1, 'fs': 17.02288818359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='wx\"(𝑝p\"−\"𝑞p)E +\"\\n', metadata={'x0': 208.8000030517578, 'x1': 290.1911926269531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 81.39118957519531, 'height': 21.8497314453125, 'nlines': 1, 'fs': 21.8497314453125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='yx (𝑝E\"−\"𝑞E)E \"+\"\\n', metadata={'x0': 291.3599853515625, 'x1': 372.9911804199219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 81.63119506835938, 'height': 21.8497314453125, 'nlines': 1, 'fs': 21.8497314453125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ox (𝑝z\"−\"𝑞z)E,  \\n', metadata={'x0': 374.1600036621094, 'x1': 446.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 72.1199951171875, 'height': 18.8851318359375, 'nlines': 1, 'fs': 18.8851318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nfor 𝑎 = 𝑏 = 9, 𝑐 = 15 , and where 𝑝 = (𝑝p, 𝑝E,𝑝z) and 𝑞 = (𝑞p, 𝑞E, 𝑞z). \\nThe choice of the elliptic distance coefficients takes on account both the effect of elongation along the \\nz-axis in the cryo-ET images and the size of the particle in voxels of the tomogram (1 vox = 13.48 Å).  \\n \\n \\n', metadata={'x0': 72.0, 'x1': 526.6799926757812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.67999267578125, 'height': 70.78594970703125, 'nlines': 6, 'fs': 11.797658284505209, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Tables \\n', metadata={'x0': 230.39999389648438, 'x1': 368.82000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 138.42001342773438, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Table 1: Selected hyperparameters of DeePiCt for varying cellular structures. Performances \\nwere calculated by cross-validation (CV) within the same domain, or in the case of microtubules and actin \\nnetworks, in evaluations for generalization across cellular species with respect to expert annotations in the HeLa \\n', metadata={'x0': 72.0, 'x1': 525.96533203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96533203125, 'height': 51.858306884765625, 'nlines': 4, 'fs': 12.964576721191406, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cell tomogram (!\"!). DeePiCt networks were applied to the test datasets with the following region mask and contact \\n', metadata={'x0': 72.0, 'x1': 525.9603881835938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96038818359375, 'height': 11.506546020507812, 'nlines': 1, 'fs': 11.506546020507812, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='mode combinations: cytosol and intersection for ribosome and FAS; cytosol and contact for membranes; nuclear \\nenvelope and contact for NPCs; cytosolic volume and intersection for microtubules and actin filaments.  \\n \\n', metadata={'x0': 72.0, 'x1': 525.9619750976562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96197509765625, 'height': 37.562835693359375, 'nlines': 3, 'fs': 12.520945231119791, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular \\nfeature \\nMethod \\n', metadata={'x0': 75.5999984741211, 'x1': 176.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 101.1600112915039, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Depth \\n(D) \\n', metadata={'x0': 188.8800048828125, 'x1': 217.5520477294922, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 28.672042846679688, 'height': 24.362823486328125, 'nlines': 2, 'fs': 12.181411743164062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Initial \\nfilters \\n(IF) \\nBN \\n', metadata={'x0': 229.44000244140625, 'x1': 293.1600036621094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 63.720001220703125, 'height': 37.562835693359375, 'nlines': 4, 'fs': 9.390708923339844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Median \\nPerformance \\n', metadata={'x0': 308.1600036621094, 'x1': 366.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 57.959991455078125, 'height': 24.362823486328125, 'nlines': 2, 'fs': 12.181411743164062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training \\ndataset \\nTesting dataset \\n', metadata={'x0': 383.0400085449219, 'x1': 518.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 135.72000122070312, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cytoplasm \\n2D CNN \\n5 \\n16 \\n1 AUPRC=0.97 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.635101318359375, 'nlines': 7, 'fs': 1.6621573311941964, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Organelles \\n2D CNN \\n5 \\n16 \\n1 AUPRC=0.92 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.395111083984375, 'nlines': 7, 'fs': 1.6278730119977678, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ribosomes \\nDeePiCt \\n2 \\n4 \\n1 F1=0.79 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.162811279296875, 'nlines': 7, 'fs': 1.5946873256138392, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='FAS \\nDeePiCt \\n2 \\n16 \\n1 F1=0.46 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.162811279296875, 'nlines': 7, 'fs': 1.5946873256138392, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Membranes \\nDeePiCt \\n2 \\n4 \\n1 voxel-F1=0.79 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.162811279296875, 'nlines': 7, 'fs': 1.5946873256138392, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='NPCs \\nDeePiCt \\n3 \\n16 \\n1 \\n', metadata={'x0': 75.5999984741211, 'x1': 306.8399963378906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 231.23999786376953, 'height': 11.162841796875, 'nlines': 5, 'fs': 2.232568359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='voxel-F1=0.24 \\nDEF* \\nDEF* \\n', metadata={'x0': 308.1600036621094, 'x1': 477.97381591796875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 169.81381225585938, 'height': 11.162811279296875, 'nlines': 3, 'fs': 3.7209370930989585, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='voxel-F1=0.47 \\nDEF* \\n', metadata={'x0': 308.1600036621094, 'x1': 409.5738220214844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 101.413818359375, 'height': 11.63507080078125, 'nlines': 2, 'fs': 5.817535400390625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='DEF* (high \\nquality) \\n', metadata={'x0': 451.44000244140625, 'x1': 501.4791259765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 50.03912353515625, 'height': 24.362823486328125, 'nlines': 2, 'fs': 12.181411743164062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='voxel-F1=0.19 \\nDEF* \\n', metadata={'x0': 308.1600036621094, 'x1': 409.5738220214844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 101.413818359375, 'height': 11.39508056640625, 'nlines': 2, 'fs': 5.697540283203125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='DEF* (lower \\nquality) \\n', metadata={'x0': 451.44000244140625, 'x1': 506.51910400390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 55.0791015625, 'height': 24.602874755859375, 'nlines': 2, 'fs': 12.301437377929688, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Microtubules \\nDeePiCt \\n2 \\n4 \\n1 #$%&\\'()*+,-./\" VPP \\nHeLa cell \\n', metadata={'x0': 75.5999984741211, 'x1': 493.09381103515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 417.49381256103516, 'height': 16.314453125, 'nlines': 6, 'fs': 2.7190755208333335, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Actin \\nDeePiCt \\n3 \\n8 \\n0 #$%&\\'()*+,-*,\" VPP \\nHeLa cell \\n', metadata={'x0': 75.5999984741211, 'x1': 493.09381103515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 417.49381256103516, 'height': 16.31439208984375, 'nlines': 6, 'fs': 2.7190653483072915, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 218.75999450683594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 146.75999450683594, 'height': 41.50592041015625, 'nlines': 4, 'fs': 10.376480102539062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Table 2: Specifications of three 3D-CNN training and prediction rounds performed for \\nribosome ground truth construction. Each of the rounds in step 2 of the RIBO ground truth construction \\nconsisted of training 3 different 3D CNNs with default parameters except the IF number (Supplementary Note \\n1). The first training was performed on manually curated TM results of the corresponding dataset (VPP or DEF). \\nFor VPP, in the second round, the 3D CNN were trained on aggregated TM results and first round predictions; \\nwhereas the third CNN round was trained on all previous aggregated results. For DEF, the second and third round \\nof 3D CNNs corresponded to the VPP 3D CNNs of first and second rounds, respectively. \\n \\n', metadata={'x0': 72.0, 'x1': 525.96435546875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96435546875, 'height': 104.97050476074219, 'nlines': 8, 'fs': 13.121313095092773, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='traini\\nng \\nround \\n', metadata={'x0': 84.95999908447266, 'x1': 113.63999938964844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 28.68000030517578, 'height': 37.802764892578125, 'nlines': 3, 'fs': 12.600921630859375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training \\ndataset \\n', metadata={'x0': 121.19999694824219, 'x1': 161.6399688720703, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 40.439971923828125, 'height': 24.362762451171875, 'nlines': 2, 'fs': 12.181381225585938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Prediction \\ndataset \\n', metadata={'x0': 187.9199981689453, 'x1': 235.07931518554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 47.15931701660156, 'height': 24.362762451171875, 'nlines': 2, 'fs': 12.181381225585938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training set \\nannotations \\n', metadata={'x0': 262.79998779296875, 'x1': 317.40032958984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 54.600341796875, 'height': 24.362762451171875, 'nlines': 2, 'fs': 12.181381225585938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1 \\nVPP \\nVPP \\nTM annotations \\nIF=4, 8, and 32 \\nvpp_output_rou\\nnd1 \\n', metadata={'x0': 84.95999908447266, 'x1': 520.3195190429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 435.3595199584961, 'height': 24.602813720703125, 'nlines': 7, 'fs': 3.5146876743861606, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2 \\nVPP \\nVPP \\nTM annotations + \\nvpp_output_round1 \\n', metadata={'x0': 84.95999908447266, 'x1': 344.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 259.3199996948242, 'height': 24.602813720703125, 'nlines': 5, 'fs': 4.920562744140625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nvpp_output_rou\\nnd2 \\n', metadata={'x0': 361.44000244140625, 'x1': 520.3195190429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 158.8795166015625, 'height': 24.602813720703125, 'nlines': 3, 'fs': 8.200937906901041, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3 \\nVPP \\nVPP \\nTM \\nannotations \\n+ \\n', metadata={'x0': 84.95999908447266, 'x1': 353.8788146972656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.91881561279297, 'height': 11.162811279296875, 'nlines': 6, 'fs': 1.8604685465494792, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nvpp_output_rou\\nnd3 \\n', metadata={'x0': 361.44000244140625, 'x1': 520.3195190429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 158.8795166015625, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1 \\nDEF \\nDEF \\nTM annotations \\nIF=4, 8, and 32 \\ndef_output_roun\\nd1 \\n', metadata={'x0': 84.95999908447266, 'x1': 523.1993408203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.23934173583984, 'height': 24.362823486328125, 'nlines': 7, 'fs': 3.480403355189732, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2 \\nDEF \\nDEF \\nTM \\nannotations \\n+ \\n', metadata={'x0': 84.95999908447266, 'x1': 353.8788146972656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.91881561279297, 'height': 11.162811279296875, 'nlines': 6, 'fs': 1.8604685465494792, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='def_output_round1 \\n', metadata={'x0': 262.79998779296875, 'x1': 341.8799743652344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 79.07998657226562, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\ndef_output_roun\\nd2 \\n', metadata={'x0': 361.44000244140625, 'x1': 523.1993408203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 161.75933837890625, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3 \\nVPP \\nDEF \\nTM \\nannotations \\n+ \\n', metadata={'x0': 84.95999908447266, 'x1': 353.8788146972656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.91881561279297, 'height': 11.162811279296875, 'nlines': 6, 'fs': 1.8604685465494792, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\ndef_output_roun\\nd3 \\n', metadata={'x0': 361.44000244140625, 'x1': 523.1993408203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 161.75933837890625, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\n \\nSupplementary Table 3: Specifications of 3D-CNN training and prediction rounds performed for FAS \\nground truth construction. Step 2 in FAS ground truth construction consisted of a single round of 3D CNN \\ntraining, performed separately per dataset type (VPP and DEF). The round consisted of 3 networks with default \\nparameters (Supplementary Note 1) except for the IF number, set to IF=4, 8, and 32. The networks were trained \\non an initial incomplete manual picking. \\n \\n', metadata={'x0': 72.0, 'x1': 525.9617919921875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9617919921875, 'height': 106.09832763671875, 'nlines': 8, 'fs': 13.262290954589844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training dataset \\nPrediction dataset \\nTraining \\nset \\n', metadata={'x0': 84.0, 'x1': 349.0801086425781, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.0801086425781, 'height': 11.162841796875, 'nlines': 4, 'fs': 2.79071044921875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='VPP \\nVPP \\nInitial \\nmanual \\n', metadata={'x0': 84.0, 'x1': 349.0800476074219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.0800476074219, 'height': 11.162841796875, 'nlines': 4, 'fs': 2.79071044921875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nfas_vpp_output_round\\n1 \\n', metadata={'x0': 356.6400146484375, 'x1': 522.47998046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 165.8399658203125, 'height': 24.60284423828125, 'nlines': 3, 'fs': 8.200948079427084, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='DEF \\nDEF \\nInitial \\nmanual \\n', metadata={'x0': 84.0, 'x1': 349.0800476074219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.0800476074219, 'height': 11.162841796875, 'nlines': 4, 'fs': 2.79071044921875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nfas_def_output_round\\n1 \\n', metadata={'x0': 356.6400146484375, 'x1': 520.3198852539062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 163.67987060546875, 'height': 24.60284423828125, 'nlines': 3, 'fs': 8.200948079427084, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Note 2 \\n', metadata={'x0': 230.16000366210938, 'x1': 368.82000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 138.66000366210938, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Considerations for structural analysis following CNN predictions under \\n', metadata={'x0': 82.80000305175781, 'x1': 516.0829467773438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.28294372558594, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='different imaging conditions \\n', metadata={'x0': 212.16000366210938, 'x1': 386.82000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 174.66000366210938, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cryo-ET datasets of S. pombe utilized in this study were derived from cryo-FIB lamellae of wild-type, \\nnative, yeast cells. 20 tomograms used for comprehensive annotations were acquired with a dose-\\nsymmetric tilt scheme6 employing similar acquisition parameters (Extended Data Tables 4-6, Online \\nMethods), with the only difference being imaging with traditional defocus-only (DEF) or, in addition, \\nthe usage of a VPP. All RIBO and FAS particles annotated in the two datasets (DEF and VPP) were \\nsubjected to subtomogram analysis including CTF correction utilizing CTF fitting and model creation \\nin Warp, and 3D refinements, hierarchical, and focused 3D classifications in RELION. During this \\nstructural analysis we made several observations which will be discussed in the following.  \\n  \\nAs described previously7,8,9, acquiring tomography data with a VPP emphasizes low frequencies. \\nPronounced low frequencies lead to improved image contrast which can facilitate data mining. With \\nour method, more particles were detected in VPP than in DEF ground truth annotations of FAS \\ncomplexes, which represent more challenging targets for data mining due to their low abundance and \\nhollow structural signature. The increased SNR in VPP was also advantageous in hierarchical \\nclassifications of ribosomes (Supplementary Fig. 2), while no difference was apparent for FAS likely \\ndue to the low particle numbers (Supplementary Fig. 3). For VPP, all ribosomes clustered in defined \\naverages which confirmed the high performance of DeePiCt and enabled the classification of a subset \\nof 60S large subunits (Supplementary Fig. 2). In comparison, most DEF ribosomes from ground truth \\nannotations or DeePiCt predictions clustered in poorly defined classes and no 60S large subunits could \\nbe separated even when classification results were improved by optimized particle poses (coordinates \\nand orientations) after multi-particle refinement in M10 (Supplementary Fig. 4, 5). Also, in \\nclassifications focused on an area close to the head of the ribosomal small subunit and at the peptide \\nexit tunnel the additional head density was already apparent in 2D slices of the respective classes in the \\nVPP data (Supplementary Fig. 6 and 7, respectively). Similarly, ribosomes close to ER and \\nmitochondria that clustered into classes with adjacent membrane densities are more pronounced in VPP \\n(Supplementary Fig. 8 and 9, respectively). \\n  \\nHowever, fine structural details that provide insights into functional configurations of molecular \\ncomplexes are lost or cannot be recovered in the final reconstructions of VPP tomography data9. This \\nhas been suggested to be caused by inaccurate weighing9 and a signal loss at high spatial frequencies11. \\nThe preservation of high-resolution information in DEF in comparison to VPP becomes especially \\napparent when looking at fine structural details in the subtomogram averages. Densities fitting ACPs of \\nFAS (Extended data Fig. 8 d-e), P-site tRNAs (Extended data Fig. 8 j-k) inside the 80S ribosome \\nand densities connecting ribosomes to an adjacent mitochondrion membrane (Supplementary Fig. 9) \\ncould only be resolved in DEF data. \\n  \\nWe conclude that the usage of a VPP in tomography acquisition facilitated data mining including the \\ndemonstrated domain generalization of applying DeePiCt models trained in VPP to DEF, and improved \\n3D classifications. This makes it a valuable tool that ultimately provides benefits at the exploratory \\nphase of structural analysis, helping the design of experiments and analysis pipelines that can eventually \\nbe performed on DEF data from which higher resolution reconstructions can be obtained. \\n', metadata={'x0': 72.0, 'x1': 526.2098999023438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20989990234375, 'height': 593.9859619140625, 'nlines': 41, 'fs': 14.487462485708841, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 2 | Hierarchical 3D classification of cytosolic ribosomes in VPP ground truth. 2D slices \\nthrough 3D class averages of iteration 25 are displayed. The highlighted class (cyan box) shows the 60S large \\nribosomal subunit class. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9614868164062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96148681640625, 'height': 79.93829345703125, 'nlines': 6, 'fs': 13.323048909505209, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 3 | 3D classification of VPP and DEF ground truth, and DeePiCt prediction in DEF \\ndataset for FAS. a-c. 2D slices through 3D-refined subtomogram average of FAS using D3 symmetry recovered \\nfrom VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. 3D classifications separated in all 3 \\ndatasets into two classes of which one is better structurally defined than the other. 3D refinements for DEF were \\nperformed in M. \\n \\n', metadata={'x0': 72.0, 'x1': 525.9619140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9619140625, 'height': 90.36279296875, 'nlines': 7, 'fs': 12.908970424107142, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 4 | Hierarchical 3D classification of cytosolic ribosomes in DEF ground truth starting \\nfrom M-refined alignments. 2D slices through 3D class averages of iteration 25 are displayed. The highlighted \\nclass (yellow box) was further refined in M, resulting in a well aligned class with a resolution of 9.3 Å. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.96142578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96142578125, 'height': 78.49832153320312, 'nlines': 6, 'fs': 13.083053588867188, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 5 | Hierarchical 3D classification of cytosolic ribosomes from DeePiCt predictions in \\nDEF data starting from M-refined alignments. 2D slices through 3D class averages of iteration 25 are \\ndisplayed. The highlighted class (yellow box) was further refined in M resulting in a well-aligned class with a \\nresolution of 9.4 Å. The right side of the classification representing the vast majority of ribosomes does not \\nconverge into a well defined map. \\n \\n', metadata={'x0': 72.0, 'x1': 525.9690551757812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96905517578125, 'height': 90.60284423828125, 'nlines': 7, 'fs': 12.943263462611608, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 6 | 3D classifications and refinements of cryo-EM densities in proximity to the head of \\nthe 40S small ribosomal subunit in VPP gt, DEF gt, and DeePiCt predictions in DEF datasets. a-c. 2D slices \\nthrough 3D-refined subtomogram average of all cytosolic ribosomes from VPP ground truth (gt), DEF gt or \\nDeePiCt predictions on DEF datasets. Focused 3D classifications (blue sphere mask displayed in a) separated in \\nall 3 datasets one class (dark blue) with an additional density close to the head of the 40S small ribosomal subunit. \\n3D refinements for DEF were performed in M. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9642333984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9642333984375, 'height': 117.21051025390625, 'nlines': 10, 'fs': 11.721051025390626, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 7 | 3D classifications and refinements of ribosomes at the exit tunnel in VPP gt, DEF \\ngt, and DeePiCt predictions in DEF datasets. a-c. 2D slices through 3D-refined subtomogram average of all \\nribosomes from VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. Focused 3D classifications \\n(cyan sphere mask displayed in a) separated in all 3 datasets one class (cyan) with an additional exit tunnel density. \\n3D refinements for DEF were performed in M. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9629516601562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96295166015625, 'height': 103.56280517578125, 'nlines': 9, 'fs': 11.506978352864584, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 8 | 3D classifications and refinements of ER-bound ribosomes in VPP gt, DEF gt, and \\nDeePiCt predictions in DEF datasets. a-c. 2D slices through 3D-refined subtomogram average of ribosomes in \\n25 nm distance to the ER recovered from VPP ground truth (gt), DEF gt or DeePiCt predictions on DEF datasets. \\nFocused 3D classifications (cyan sphere mask displayed in Supplementary Fig. 7a) separated in all 3 datasets one \\nclass (orange) with membrane density. 3D refinements for DEF were performed in M. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.968017578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.968017578125, 'height': 105.0028076171875, 'nlines': 9, 'fs': 11.666978624131945, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 9 | 3D classifications and refinements of mitochondria-bound ribosomes in VPP gt, \\nDEF gt, and DeePiCt predictions in DEF datasets. a-c. 2D slices through 3D-refined subtomogram average of \\nribosomes in 25 nm distance to the mitochondria recovered from VPP ground truth (gt), DEF gt or DeePiCt \\nprediction on DEF datasets. Focused 3D classifications (cyan sphere mask displayed in Supplementary Fig. 7a) \\nseparated in all 3 datasets one class (green) with membrane density. 3D refinements for DEF were performed in \\nM. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9649658203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9649658203125, 'height': 118.44284057617188, 'nlines': 10, 'fs': 11.844284057617188, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. \\nRonneberger, O., Fischer, P. & Brox, T. U-Net: Convolutional Networks for Biomedical Image \\n', metadata={'x0': 72.0, 'x1': 516.6036987304688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.60369873046875, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Navab, N., Hornegger, J., Wells, W. M. & Frangi, A. F.) vol. 9351 234–241 (Springer \\n', metadata={'x0': 93.83999633789062, 'x1': 475.8011169433594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 381.96112060546875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='International Publishing, 2015). \\n', metadata={'x0': 93.83999633789062, 'x1': 236.51998901367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.67999267578125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. \\nZeiler, M. D., Krishnan, D., Taylor, G. W. & Fergus, R. Deconvolutional networks. in 2010 \\n', metadata={'x0': 72.0, 'x1': 500.7604064941406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 428.7604064941406, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IEEE Computer Society Conference on Computer Vision and Pattern Recognition 2528–2535 \\n', metadata={'x0': 93.83999633789062, 'x1': 509.6404113769531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 415.8004150390625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(IEEE, 2010). doi:10.1109/CVPR.2010.5539957. \\n', metadata={'x0': 93.83999633789062, 'x1': 314.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 220.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. \\nMilletari, F., Navab, N. & Ahmadi, S.-A. V-Net: Fully Convolutional Neural Networks for \\n', metadata={'x0': 72.0, 'x1': 496.683349609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 424.683349609375, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Volumetric Medical Image Segmentation. in 2016 Fourth International Conference on 3D Vision \\n', metadata={'x0': 93.83999633789062, 'x1': 526.2052001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 432.3652038574219, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(3DV) 565–571 (IEEE, 2016). doi:10.1109/3DV.2016.79. \\n', metadata={'x0': 93.83999633789062, 'x1': 350.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 256.9200134277344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. \\nKingma, D. P. & Ba, J. Adam: A Method for Stochastic Optimization. \\n', metadata={'x0': 72.0, 'x1': 406.2078857421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 334.2078857421875, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='https://arxiv.org/abs/1412.6980 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 269.4000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 175.56002807617188, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='5. \\nvan der Walt, S. et al. scikit-image: image processing in Python. PeerJ 2, e453 (2014). \\n', metadata={'x0': 72.0, 'x1': 478.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 406.20001220703125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. \\nHagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-\\n', metadata={'x0': 72.0, 'x1': 520.1563720703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.1563720703125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 \\n', metadata={'x0': 93.83999633789062, 'x1': 502.200439453125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 408.3604431152344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 128.75999450683594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 34.91999816894531, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='7. \\nDanev, R., Buijsse, B., Khoshouei, M., Plitzko, J. M. & Baumeister, W. Volta potential phase \\n', metadata={'x0': 72.0, 'x1': 509.1627197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 437.1627197265625, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='plate for in-focus phase contrast transmission electron microscopy. Proc Natl Acad Sci USA 111, \\n', metadata={'x0': 93.83999633789062, 'x1': 523.5602416992188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 429.7202453613281, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='15635–15640 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 191.87998962402344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 98.03999328613281, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='8. \\nDanev, R., Tegunov, D. & Baumeister, W. Using the Volta phase plate with defocus for cryo-EM \\n', metadata={'x0': 72.0, 'x1': 526.2044067382812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20440673828125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='9. \\nTuroňová, B. et al. Benchmarking tomographic acquisition schemes for high-resolution \\n', metadata={'x0': 72.0, 'x1': 482.2804260253906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 410.2804260253906, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='structural biology. Nat. Commun. 11, 876 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 314.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 220.68002319335938, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='10. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM \\n', metadata={'x0': 72.0, 'x1': 490.4444274902344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.4444274902344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='refinement with M visualizes ribosome-antibiotic complex at 3.5 Å in cells. Nat. Methods 18, \\n', metadata={'x0': 93.83999633789062, 'x1': 507.4797668457031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 413.6397705078125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='186–193 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 169.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='11. Buijsse, B., Trompenaars, P., Altin, V., Danev, R. & Glaeser, R. M. Spectral DQE of the Volta \\n', metadata={'x0': 72.0, 'x1': 514.6878662109375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 442.6878662109375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='phase plate. Ultramicroscopy 218, 113079 (2020). \\navailable under aCC-BY-NC-ND 4.0 International license.\\nwas not certified by peer review) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity. It is made \\nbioRxiv preprint doi: https://doi.org/10.1101/2022.04.12.488077; this version posted April 13, 2022. The copyright holder for this preprint (which\\n', metadata={'x0': 51.65239334106445, 'x1': 556.3480224609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 504.69562911987305, 'height': 31.30805206298828, 'nlines': 4, 'fs': 7.82701301574707, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66})]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from langchain.document_loaders.pdf import BasePDFLoader\n",
    "from langchain.document_loaders.base import BaseBlobParser\n",
    "from typing import Optional, List, Iterator, Mapping, Any, Dict\n",
    "from langchain.docstore.document import Document\n",
    "from langchain.document_loaders.blob_loaders import Blob\n",
    "\n",
    "import re\n",
    "\n",
    "import dataclasses\n",
    "\n",
    "@dataclasses.dataclass\n",
    "class PyMuPDFBlock:\n",
    "    \"\"\"A block of text occuring in a PDF file.\"\"\"\n",
    "\n",
    "    page: int\n",
    "\n",
    "    x0:float\n",
    "    x1:float\n",
    "    width:float\n",
    "\n",
    "    y0:float\n",
    "    y1:float\n",
    "    height:float\n",
    "    \n",
    "    nlines:int\n",
    "    font_size_proxy:float\n",
    "\n",
    "    text:str\n",
    "    t:str\n",
    "\n",
    "    def __init__(self, p:int, x0:float, y0:float, x1:float, y1:float, text:str):\n",
    "        self.x0 = x0\n",
    "        self.y0 = y0\n",
    "        self.x1 = x1\n",
    "        self.y1 = y1\n",
    "        self.page = p\n",
    "        \n",
    "        self.text = text\n",
    "        self.width = x1 - x0\n",
    "        self.height = y1 - y0\n",
    "        self.nlines = text.count('\\n')\n",
    "        if self.nlines == 0:\n",
    "            self.nlines = 1 \n",
    "        self.font_size_proxy = (self.height / self.nlines)\n",
    "        self.t = re.sub('\\-\\n','',self.text)\n",
    "        self.t = re.sub('\\n',' ',self.text)\n",
    "        self.t = re.sub('\\s+',' ',self.text)\n",
    "        \n",
    "    def __repr__(self):\n",
    "        return f\"PyMuPDFBlock(page={self.page}, x0={self.x0}, x1={self.x1}, y0={self.y0}, y1={self.y1}, text={self.text})\"\n",
    "    \n",
    "    def __str__(self): \n",
    "        return f\"PyMuPDFBlock(page={self.page}, x0={self.x0}, x1={self.x1}, y0={self.y0}, y1={self.y1}, text={self.text})\"\n",
    "    \n",
    "    def __eq__(self, other):\n",
    "        if self.page == other.page and self.x0 == other.x0 and self.x1 == other.x1 and self.y0 == other.y0 and self.y1 == other.y1:\n",
    "            return True\n",
    "        else:\n",
    "            return False\n",
    "\n",
    "class PyMuPDFBlockLoader(BasePDFLoader):\n",
    "    \"\"\"Load `PDF` files using `PyMuPDF`.\"\"\"\n",
    "\n",
    "    def __init__(self, file_path: str) -> None:\n",
    "        \"\"\"Initialize with a file path.\"\"\"\n",
    "        try:\n",
    "            import fitz  # noqa:F401\n",
    "        except ImportError:\n",
    "            raise ImportError(\n",
    "                \"`PyMuPDF` package not found, please install it with \"\n",
    "                \"`pip install pymupdf`\"\n",
    "            )\n",
    "\n",
    "        super().__init__(file_path)\n",
    "\n",
    "    def load(self, **kwargs: Optional[Any]) -> List[Document]:\n",
    "        \"\"\"Load file.\"\"\"\n",
    "\n",
    "        parser = PyMuPDFBlockParser(text_kwargs=kwargs)\n",
    "        blob = Blob.from_path(self.file_path)\n",
    "        blocks = parser.parse(blob)\n",
    "    \n",
    "        return blocks\n",
    "    \n",
    "class PyMuPDFBlockParser(BaseBlobParser):\n",
    "    \"\"\"Parse `PDF` using `PyMuPDF`.\"\"\"\n",
    "\n",
    "    def __init__(self, text_kwargs: Optional[Mapping[str, Any]] = None) -> None:\n",
    "        \"\"\"Initialize the parser.\n",
    "\n",
    "        Args:\n",
    "            text_kwargs: Keyword arguments to pass to ``fitz.Page.get_text()``.\n",
    "        \"\"\"\n",
    "        self.text_kwargs = text_kwargs or {}\n",
    "\n",
    "    def lazy_parse(self, blob: Blob) -> Iterator[Document]:\n",
    "        \"\"\"Lazily parse the blob.\"\"\"\n",
    "        import fitz\n",
    "\n",
    "        with blob.as_bytes_io() as file_path:\n",
    "\n",
    "            doc = fitz.open(file_path)  # open document\n",
    "\n",
    "            blocks = []\n",
    "            text_dict = {}\n",
    "            \n",
    "            for page in doc:\n",
    "                for (x0, y0, x1, y1, text, bn, bt) in page.get_textpage().extractBLOCKS():\n",
    "                    b = PyMuPDFBlock(page.number, x0, y0, x1, y1, text)\n",
    "                    blocks.append(b)\n",
    "                    if text_dict.get(text):\n",
    "                        text_dict[text].append(b)\n",
    "                    else:\n",
    "                        text_dict[text] = [b]\n",
    "\n",
    "            # remove blocks that are repeated\n",
    "            #for t in text_dict.keys():\n",
    "            #    if len(text_dict[t]) > 1:\n",
    "            #        for b in text_dict[t]:\n",
    "            #            blocks.remove(b)\n",
    "            \n",
    "            # remove blocks that are urls on their own\n",
    "            # remove any blocks that are one word long\n",
    "            #to_remove = []\n",
    "            #for b in blocks:\n",
    "            #    m1 = re.search('^\\s*http://[A-Za-z1-9\\.\\/\\-\\_]+\\s*$', b.text)\n",
    "            #    m2 = re.search('^\\s*\\S+\\s*$', b.text)\n",
    "            #    if m1 or m2:\n",
    "            #       to_remove.append(b)\n",
    "\n",
    "            for b in blocks:\n",
    "                yield Document(\n",
    "                    page_content=b.text,\n",
    "                    metadata=dict({\n",
    "                        \"x0\": b.x0, \"x1\": b.x1, \n",
    "                        \"y0\": y0, \"y1\": y1, \n",
    "                        \"width\": b.width, \"height\": b.height,\n",
    "                        \"nlines\": b.nlines,\n",
    "                        \"fs\": b.font_size_proxy,\n",
    "                        \"file_path\": blob.source,\n",
    "                        \"page\": page.number,\n",
    "                        \"total_pages\": len(doc),\n",
    "                    }))\n",
    "\n",
    "\n",
    "\n",
    "loader = PyMuPDFBlockLoader(d+p)\n",
    "blocks = loader.load()\n",
    "blocks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(page_content='Convolutional networks for supervised mining of molecular patterns within \\ncellular context \\n \\n', metadata={'x0': 72.0, 'x1': 543.6431884765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 471.6431884765625, 'height': 58.50422668457031, 'nlines': 3, 'fs': 19.50140889485677, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Irene de Teresa1,2*, Sara K. Goetz1,3*, Alexander Mattausch1,4, Frosina Stojanovska1,3,8, Christian E. \\nZimmerli1,5, Mauricio Toro-Nahuelpan1,6, Dorothy W.C. Cheng1,3,7, Fergus Tollervey1,3, Constantin Pape7,9, \\nMartin Beck1,5, Anna Kreshuk7, Julia Mahamid1,7+ and Judith Zaugg1,8+. \\n \\n', metadata={'x0': 72.0, 'x1': 542.76025390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76025390625, 'height': 56.59019470214844, 'nlines': 4, 'fs': 14.14754867553711, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Structural and Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, \\n', metadata={'x0': 90.0, 'x1': 542.759765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.759765625, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Germany. \\n', metadata={'x0': 108.0, 'x1': 154.4399871826172, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 46.43998718261719, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. Current address: Computer Science and Artificial Intelligence Lab, ENGIE Lab Crigen, Stains, \\n', metadata={'x0': 90.0, 'x1': 542.764404296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.764404296875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='France. \\n', metadata={'x0': 108.0, 'x1': 143.39999389648438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 35.399993896484375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Collaboration for Joint PhD Degree between EMBL and Heidelberg University, Faculty of \\n', metadata={'x0': 90.0, 'x1': 542.7587890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.7587890625, 'height': 12.382278442382812, 'nlines': 1, 'fs': 12.382278442382812, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Biosciences, Heidelberg, Germany. \\n', metadata={'x0': 108.0, 'x1': 266.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 158.760009765625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. Institute of Pharmacy and Molecular Biotechnology, Heidelberg University, Heidelberg, Germany. \\n5. Current address: Max Planck Institute of Biophysics, Department of Molecular Sociology, \\n', metadata={'x0': 90.0, 'x1': 542.7619018554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.76190185546875, 'height': 26.782318115234375, 'nlines': 2, 'fs': 13.391159057617188, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Frankfurt, Germany. \\n', metadata={'x0': 108.0, 'x1': 201.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 93.47999572753906, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. Current address: Santiago GmbH & Co. KG, Willich, Germany. \\n7. Cell Biology and Biophysics Unit, European Molecular Biology Laboratory, Heidelberg, Germany. \\n8. Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany. \\n9. Institute for Computer Science, Universität Göttingen \\n', metadata={'x0': 90.0, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.760009765625, 'height': 55.822296142578125, 'nlines': 4, 'fs': 13.955574035644531, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\n* These authors contributed equally to this work. \\n+ correspondence to: julia.mahamid@embl.de, judith.zaugg@embl.de  \\n \\n \\nAbstract. \\n', metadata={'x0': 72.0, 'x1': 386.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 314.52001953125, 'height': 89.42849731445312, 'nlines': 6, 'fs': 14.904749552408854, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cryo-electron tomograms capture a wealth of structural information on the molecular constituents of cells \\nand tissues. We present DeePiCt (Deep Picker in Context), an open-source deep-learning framework for \\nsupervised structure segmentation and macromolecular complex localization in cellular cryo-electron \\ntomography. To train and benchmark DeePiCt on experimental data, we comprehensively annotated 20 \\ntomograms of Schizosaccharomyces pombe for ribosomes, fatty acid synthases, membranes, nuclear pore \\ncomplexes, organelles and cytosol. By comparing our method to state-of-the-art approaches on this dataset, \\nwe show its unique ability to identify low-abundance and low-density complexes. We use DeePiCt to study \\ncompositionally-distinct subpopulations of cellular ribosomes, with emphasis on their contextual \\nassociation with mitochondria and the endoplasmic reticulum. Finally, by applying pre-trained networks to \\na HeLa cell dataset, we demonstrate that DeePiCt achieves high-quality predictions in unseen datasets from \\ndifferent biological species in a matter of minutes. The comprehensively annotated experimental data and \\npre-trained networks are provided for immediate exploitation by the community.  \\n \\n', metadata={'x0': 72.0, 'x1': 542.7642822265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7642822265625, 'height': 186.9459228515625, 'nlines': 13, 'fs': 14.380455603966347, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Main. \\n', metadata={'x0': 72.0, 'x1': 111.29999542236328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 39.29999542236328, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Introduction \\nCryo-Electron Tomography (cryo-ET) produces three-dimensional (3D) snapshots of cellular landscapes at \\nmolecular resolution, making it possible to investigate the structural and functional states of \\nmacromolecular complexes in their native environment, and to unveil how different macromolecular \\npopulations interact with other cellular structures1–8. With improved instrumentation, sample preparation \\nprotocols, and automation, high-quality in-cell cryo-ET data are rapidly being generated, opening the \\npossibility to conduct high-throughput studies9–12. However, due to the complex and crowded nature of the \\nintracellular milieu, together with limitations arising from cryo-ET image acquisition (low signal-to-noise \\nratio (SNR) and incomplete angular sampling) data mining of 3D cryo-ET volumes remains a major \\nbottleneck13,14. Data mining in terms of reliable identification of a relatively homogenous set of \\nmacromolecular complexes constitutes a fundamental prerequisite for structural analysis15,13,14. \\n \\nA range of available semi-automated methods for segmentation of cellular structures and localization of \\nmacromolecular complexes (from here on, particles) in cryo-ET datasets are broadly classified as template-\\nbased and template-free approaches16. Traditional Template Matching (TM)17 is a commonly applied \\ncomputational approach and is based on a point-wise numerical computation of a similarity coefficient \\n(cross-correlation) to a known template of the complex in question. TM is accurate in the localization of \\nribosomes and cytoskeletal filaments, but fails at identifying smaller or less dense particles18, and is \\ncomputationally intensive. On the other hand, current template-free methods implementing classical image \\nprocessing approaches are purposely designed for and thus limited to specific molecular shapes, or to \\nspecial configurations in which particles are associated with large cellular structures such as membranes or \\nmicrotubules16,19,20. These methods together with TM require manual inspection and are therefore laborious \\nand time-consuming. The advent of deep learning methods, and particularly convolutional neural networks \\n(CNNs)21–23, has enabled developing more generally applicable and automated approaches to accelerate the \\ntasks of segmentation and localization in cryo-ET. The first of such methods was a two-dimensional (2D) \\nCNN for semantic segmentation of ribosomes and large structures, such as cellular organelles or \\nmembranes24. However, its 2D nature makes it less suitable for particle localization, where probing the 3D \\nstructure becomes beneficial. More recently, DeepFinder, a fully supervised method based on the U-Net \\narchitecture25 for multi-class semantic segmentation, has positioned itself as the state-of-the-art in \\nautomated particle localization in both simulated and real cryo-ET datasets26–29. Remaining limitations \\ninclude the identification of less prevalent particles, as well as the interpretation of the obtained predictions \\nwithin their cellular context. Furthermore, the field lacks publicly available non-synthetic expert-annotated \\ncryo-ET data to train new models and to benchmark the available methods.  \\n \\nHere we present DeePiCt (Deep Picker in Context), our open-source software which synergizes supervised \\nconvolutional networks for segmentation of cellular compartments (organelles or cytosol) and structures \\n(membranes or cytoskeletal filaments), and localization of particles. We generated a set of comprehensively \\nannotated tomograms of wild-type S. pombe for training and benchmarking our method, which we make \\npublicly available to overcome the critical limitations arising from the absence of publicly available \\nannotated experimental datasets. From here on, we refer to the term network as the deep learning algorithm \\nitself and model as the algorithm once already trained. We provide DeePiCt models, trained on this dataset \\nwhich show high data-mining performance and can be readily applied across species and datasets.  \\n', metadata={'x0': 72.0, 'x1': 542.7666015625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7666015625, 'height': 614.1817016601562, 'nlines': 42, 'fs': 14.623373849051339, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Results \\n \\nDeePiCt: A deep-learning approach for automated compartment segmentation and particle \\nlocalization  \\nDeePiCt is based on deep learning technologies. It combines a 2D CNN for segmentation of cellular \\ncompartments and a 3D CNN for particle localization, membrane, and cytoskeletal filament annotations \\n(Fig. 1). This synergy allows for more precise particle picking and interpretation of their cellular context. \\nThe 2D and 3D CNN are adapted from the original U-Net architecture25 (Fig. 1a, Supplementary Note 1). \\nSince cellular compartments are easily recognizable in 2D image data, the compartment segmentation \\nnetwork operates in 2D, which has the advantage of requiring little training data. The 2D CNN employs a \\nfixed depth (D) of 5 (4 max-pooling layers) and 16 initial filters (IF) (Fig. 1a). Our implementation of the \\n3D CNN allows multi-label learning (Supplementary Note 1) and adjustable architectural parameters D, \\nnumber of IF, a batch normalization layer (BN), and the dropout parameter in the encoder and decoder \\npaths (ED and DD, respectively) (Fig. 1a). These parameters can be set according to the amount and quality \\nof training data, and the shape complexity, size and abundance of the particle of interest as suggested in \\nSupplementary Table 1. In general, larger particles benefit from larger values of D to increase the \\nnetwork’s receptive field, and particles with a low-density print (low SNR with respect to the surrounding \\ncontext) require larger IF values. The remaining optional layers, BN and dropout, are well known \\ntechniques in computer vision to avoid overfitting30,31. \\n \\nFor training, each network requires tomograms and corresponding 3D binary masks, which represent the \\nsegmentation of the structures of interest, e.g. organelle segmentations for the 2D U-Net and spheres \\nrepresenting particles for the 3D U-Net (Fig. 1b.1). The raw input tomograms are optionally preprocessed \\nusing an amplitude spectrum equalization filter (Fig. 1b.2, Extended Data Fig. 1) to enhance image \\ncontrast by matching the amplitude spectrum of the input tomogram to a preselected target amplitude \\nspectrum distribution (Extended Data Fig. 1). Both CNNs use the Adam optimizer algorithm and the Dice \\nLoss function for training (Supplementary Note 1). In the 2D network, tiles are randomly flipped and \\nrotated in 90-degree increments during training to improve generalization. For the 3D CNN, we \\nimplemented a data augmentation strategy by applying a number of optional random transformations to the \\ninput images (Supplementary Note 1). In our experience, more than 700 annotated instances are required \\nfor particle learning in the 3D CNNs, while about 5 tomograms are sufficient for membrane segmentation. \\nIn the case of 2D CNNs, 6 fully segmented tomograms are enough for training networks identifying either \\na cytosol class or a single class including all cellular organelles. \\n \\nFor predicting, the trained networks receive as input unseen tomograms pre-processed in the same way as \\nthe training data, and output 3D probability maps that are subsequently post-processed (Fig. 1b.3). \\nAutomated post-processing for the 2D network includes the combination of predicted 2D slices into a 3D \\nvolume, application of a 1D Gaussian filter along the z-axis for smoothing the predictions across the z \\nslices, and thresholding (user-definable, default=0.75) (Extended Data Fig. 2a-c). The 3D CNN post-\\nprocessing entails the generation of a predicted segmentation. To that end, the 3D CNN output is \\nthresholded by a probability value set by the user and then clustered. The clustered output can be integrated \\nwith contextual information from a binary map representing a tomographic region (e.g. the cytosol \\nsegmentation from the 2D CNN). The mode of integration can be chosen among three different options: \\nintersection, contact, or colocalization, depending on the users’ specific application (Fig. 1b.4, \\n', metadata={'x0': 72.0, 'x1': 543.0001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 471.0001220703125, 'height': 645.6217041015625, 'nlines': 44, 'fs': 14.673220547762783, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Note 1, Supplementary Fig. 1). In the case of particle localization, the clusters’ centroids \\nare used to obtain a list of the particle coordinates; their orientations and structural features can then be \\nobtained by subtomogram analysis in external softwares (e. g. Warp32, M33, RELION34, Dynamo35, \\nPyTom36 or EMAN237). For more details on the method, we refer to Supplementary Note 1 and our github \\nrepository (Code and data availability). \\n \\nGeneration of comprehensive ground-truth annotations in S. pombe \\n', metadata={'x0': 72.0, 'x1': 542.7622680664062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76226806640625, 'height': 100.75360107421875, 'nlines': 7, 'fs': 14.39337158203125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The ultimate requirement for developing and benchmarking any deep neural network is the existence of a \\nground truth (gt) non-synthetic dataset. Since this was not available at the outset of this study, we first \\ncreated a comprehensive annotation of cellular features in wild-type S. pombe representing diverse \\nstructures, sizes and abundances. \\n \\nIn cryo-ET data, the complete annotation of a given cellular structure or particle species is a challenging \\nprocess, even for large compartments or complexes with a clear structural signature like the ribosome. To \\nachieve the most comprehensive annotation possible, we devised an iterative workflow to mine patterns in \\nthree different datasets (Online Methods). In a set of 10 tomograms acquired by combining defocus and \\nVolta potential phase plate (VPP38,39) and a second set of 10 tomograms using defocus only (DEF), we \\nannotated ribosomes (RIBO), fatty acid synthases (FAS), membranes, organelles and cytoplasm  (Fig. 2, \\nExtended Data Tables 1-3). For the nuclear pore complex (NPC), given its structural flexibility, low \\nabundance (on average 3 per tomogram), and large size, a third dataset denoted by DEF* was used40. It \\nconsists of 127 tomograms featuring a total of 2,830 NPC subunits (~354 NPCs) to ensure enough training \\ndata and is composed of two subsets: one of higher and one of lower quality classified based on lamella \\nthickness and tilt-series alignment error. A detailed description of the ground truth construction is available \\nin the Online Methods.  \\n \\nThe resulting ground truth RIBO annotation in VPP consisted of 25,311 particles. An initial TM annotation \\ncontributed 61.6% (1,559 particles per tomogram (ppt), on average) of the final ground truth, whereas \\niterative rounds of DeePiCt on the incomplete ground truth contributed to a total of additional 21.36% (541 \\nppt on average) and final manual annotations to 17.05% (431 ppt on average) (Fig. 2j, Extended Data Fig. \\n3a,e, Extended Data Table 2). \\n \\nThe final ground truth annotation of FAS in the 10 VPP datasets amounts to 731 particles, from which \\n58.96% (43 ppt on average) constituted the initial annotations from manual picking, as TM failed. The \\ncombined predictions of DeePiCt on the incomplete ground truth contributed to 21.88% (16 ppt on average). \\nFor FAS we performed only one round of DeePiCt using the initial manual annotations, since we observed \\na high false positive rate in subsequent rounds, which is likely a consequence of the natural lower abundance \\nof FAS in the tomograms and its low imprint density. The final manual picking added a significant fraction \\nof 19% (14 ppt on average) (Extended Data Fig. 3c,e, Extended Data Table 3). \\n \\nThe final ground truth RIBO annotations on the DEF dataset sum a total of 25,901 particles across the 10 \\ntomograms, from which only 19.23% (498 ppt on average) were included in the initial TM annotation (step \\n1), 19.39% (502 ppt on average) were a contribution of DeePiCt on incomplete ground truth, and 61.36% \\n(1,590 ppt on average) were manually identified (Fig. 2k-m, Extended Data Fig. 3b,e), highlighting the \\nvalue of our carefully curated annotations. In the case of FAS, the ground truth annotations across the 10 \\n', metadata={'x0': 72.0, 'x1': 542.7744140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7744140625, 'height': 535.9059448242188, 'nlines': 37, 'fs': 14.483944454708615, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"tomograms amount to 366 particles, from which 48.63% (18 ppt on average) came from the initial manual \\nannotations (as TM failed), 36.88% (14 ppt on average) from DeePiCt trained on incomplete ground truth, \\nand 14.48% (5 ppt on average) from the final manual picking (Fig. 2k-m, Extended data Fig. 3d,f). \\nOverall, a final round of manual picking for RIBO ground truth annotation was notably more crucial in \\nDEF compared to VPP.  \\n \\nTotal numbers of RIBO in both acquisition types are comparable, while for FAS fewer particles were \\ndetected in DEF likely due to the lower SNR despite the application of an equalization filter (Extended \\nData Fig. 3, Extended Data Table 2-3, Supplementary Note 2). FAS annotations were similarly \\ndistributed among the ground truth construction steps in the two acquisition conditions (Extended Data \\nFig. 3). Comparison to previous proteomics studies41,42 shows that our annotations of fully assembled FAS \\nin cryo-electron tomograms is in agreement with levels of FAS-𝛼 and FAS-𝛽 quantified by means of mass \\nspectrometry, while the abundance of cytosolic ribosomal proteins detected by mass spectrometry are \\nunderestimated (Extended Data Fig. 4).  \\n \\nTo assess the quality of the RIBO and FAS particle annotations in the VPP and DEF datasets, we processed \\nthe tomograms in Warp and performed structural 3D classification and refinement of the subtomograms in \\nRELION (Extended Data Tables 4-5, Supplementary Note. 2, Supplementary Fig. 2-4). Subtomogram \\naverages of RIBO particles detected only by DeePiCt ('tp new'), represent 80S ribosomes, consistent with \\nmaps of all cytosolic RIBO particles for both VPP and DEF datasets (Fig. 2l-m, blue and gray, \\nrespectively). 3D classifications performed in both data types for FAS demonstrated that the 3D CNN \\ntogether with manual annotations were capable of recovering these challenging shell-like structures \\nindependent of the data acquisition type (Supplementary Note 2, Supplementary Fig. 3).  \\n \\nIn conclusion, using DeePiCt, we provide comprehensive annotations for large macromolecular complexes \\nand cellular structures in S. pombe cryo-electron tomograms acquired with VPP and DEF, which set the \\nground for benchmarking the method’s performance and for future developments in particle detection \\napproaches. \\n \\nPerformance analysis and hyperparameter tuning of the DeePiCt workflow \\n \\nPerformance analysis of 2D CNN in VPP \\nFor the 2D CNN, basic hyperparameter tuning had close to no effect and a fixed architecture produced \\nsufficiently good segmentations. Hence, no extensive hyper-parameter tuning was performed. The 2D \\nCNN's performance was evaluated on 10 manually annotated ground-truth VPP tomograms in a 5-fold \\ncross-validation (CV). For each CV fold, two tomograms were split off from the training data to evaluate \\nmodel performance during training. Two binary segmentation tasks were evaluated: segmentation of all \\norganelles, i.e. all membrane-enclosed organelles and the nucleoplasm, and segmentation of the cytosol. \\nThe results show that the 2D CNN achieves high areas under the precision-recall curve (AUPRC, \\nSupplementary Note 1), with a median AUPRC of 0.98 for cytosol and 0.92 for organelles (Fig. 3a, \\nExtended Data Fig. 5). \\n \\nThe 2D CNN showed an average false positive rate (FPR) of 0.054 for cytosol and 0.024 for organelle \\nsegmentation, with the post-processing step providing marginal improvements (reducing FPR to 0.049 for \\n\", metadata={'x0': 72.0, 'x1': 542.7688598632812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76885986328125, 'height': 639.345947265625, 'nlines': 44, 'fs': 14.530589710582387, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cytosol and 0.021 for organelles, Extended Data Fig. 2). In addition to the two segmentation tasks \\ndescribed here, the 2D CNN has the potential to segment individual organelle types when provided with \\nsufficient amounts of training data (Extended Data Table 1). Operating in 2D rather than 3D results in \\nonly little memory requirement (<1GB at the default batch size) and short training time (~15 minutes to \\ntrain on ten tomograms using a Nvidia V100 GPU; Online Methods). \\n \\nHyperparameter tuning for the 3D CNN \\nThe 3D CNN is built on a flexible architecture for which we performed hyperparameter tuning. We \\ncharacterized the effect of varying the adjustable hyperparameters (Online Methods) on the DeePiCt \\nworkflow using a CV approach for RIBO, FAS and membranes in the VPP dataset, and for NPC in the \\nDEF* dataset.  \\n \\nIn each case we combined the 3D CNN segmentation of the target structure with an appropriate region \\nmask (Supplementary Fig. 1, Supplementary Table 1) and determined the 3D CNN hyperparameter \\ncombination that optimizes the task of localization or segmentation (Extended Data Fig. 6, \\nSupplementary Table 1). Our results show that the best hyperparameter selection for the 3D CNN is \\nstructure-dependent, and not only related to the size of the complex or to the receptive field of the network \\n(a function of the depth parameter D), but also to the particle’s density, symmetry and abundance, among \\nothers. Therefore, the architecture flexibility of our implementation of the 3D CNN is essential for the \\naccurate segmentation of a diverse set of biological structures. In particular, BN shortens the learning speed \\n(and notably improves the performance results in the case of NPCs and FAS). Dropout layers and a number \\nof data augmentation strategies (Gaussian noise, salt-and-pepper noise, rotations, elastic transformations, \\nsee Supplementary Note 1) did not improve performance. \\n \\nPerformance of DeePiCt in the same-domain setting \\nOnce the 3D CNN architectures which optimized the tasks of particle localization (for FAS and RIBO) and \\nstructure segmentation (membranes and NPC) were determined, we analyzed the performance of the \\nDeePiCt workflow in the same-domain setting (i.e. training and testing in the same domain). The 3-fold \\nCV tests in VPP (Fig. 3b, Extended Data Fig. 7a-c), showed that it achieved a performance F1 score \\nbetween 0.68-0.80 for RIBO (median F1 of 0.79), 0.21-0.64 for FAS (median F1 of 0.46) and 0.58-0.90 for \\nmembranes (median voxel-F1 of 0.71). The results in DEF* for NPC segmentation (Fig. 3c) show that, \\neven if the network is trained on the whole DEF* dataset (23% of which is high quality data, Online \\nMethods), the performance of the method is clearly different in the high- versus lower-quality DEF* \\ntomograms: with a median voxel-F1 of 0.47 in high-quality data, while a median voxel-F1 of 0.19 in lower-\\nquality DEF* tomograms. \\n \\nComparison of DeePiCt to state-of-the-art particle localization tools \\nTo benchmark DeePiCt in the particle localization task, we compared it to TM and DeepFinder in the tasks \\nof RIBO and FAS in the VPP dataset (Fig. 3d,e). Following the suggestions by the authors28, DeepFinder \\nwas trained in a multi-class fashion, where both particles were learnt simultaneously by a single network. \\nFor the training, we used the same 3-fold CV splitting of the VPP datasets as for DeePiCt. In RIBO \\nlocalization, DeepFinder achieved a median F1 of 0.83, comparable to DeePiCt. For FAS localization, \\nDeepFinder achieved a median F1 of 0.11 (significantly below DeePiCt, t-test p-value of 0.007; Fig. 3d). \\nIt is worth mentioning that single-class DeepFinder networks tested for FAS localization completely failed \\n', metadata={'x0': 72.0, 'x1': 542.7683715820312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76837158203125, 'height': 639.10595703125, 'nlines': 44, 'fs': 14.525135387073863, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='by not localizing any particles. DeePiCt needs ~17h for training vs ~3h for DeepFinder, while predicting \\nand post processing is equally fast for both (~500 clusters/minute) (Online Methods). Furthermore, \\nDeePiCt’s trained networks (models) for all structures mentioned in this work are open source and publicly \\navailable (Code and data availability).  \\n \\nFinally, the analysis of TM performance was done with the raw output with respect to the top 2000 peaks \\nin the case of RIBO and to the top 1000 peaks in the case of FAS. TM also takes several hours (s. parameters \\nin Online Methods) and our results confirm that TM has lower performance than both deep-learning \\nmethods for the localization of RIBO and FAS, completely failing at the latter (Fig. 3e).  \\n \\nDomain generalization of DeePiCt across image acquisition conditions \\nThe DEF data is characterized by a lower SNR and lower contrast in the image, as compared to VPP \\n(Supplementary Note 2). This makes the task of particle localization and structure segmentation more \\nchallenging, but has the advantage that the resulting subtomogram averages generally provide better \\nresolutions43. The comprehensive ground truth construction in DEF enabled us to study the generalization \\npotential of the DeePiCt workflow across image acquisition conditions, from VPP to DEF data. To that \\nend, we predicted cytosol, organelles, membranes, FAS and RIBO in 10 DEF tomograms, with networks \\ntrained in the yeast VPP data (Fig. 3f-g).  \\n \\nCompared to segmentation of VPP data, the 2D CNN shows slightly poorer performance on DEF data for \\ncytosol (Fig. 3a,f). By employing amplitude spectrum equalization for the 2D CNN, the domain \\ngeneralization capabilities greatly improved for both cytosol (from initial median AUPRC=0.82 to median \\nAUPRC=0.95) and organelles (from AUPRC=0.42 to 0.74) (Fig. 3f, Extended Data Fig. 5). For DeePiCt, \\nthe performance scores in this domain generalization setting also showed a decay with respect to the same \\ndomain setting: with a median F1 of 0.59 for RIBO, a median F1 of 0.15 for FAS, and a median voxel-F1 \\nof 0.41 for membranes (Fig. 3g). Importantly, as in the case of the 2D CNN, the spectrum equalization filter \\nin the preprocessing step improved the domain generalization results (Extended Data Fig. 7d-f). \\nFurthermore, we confirmed that using the cytosol predictions as a region mask to clean particle annotations \\nalso improved the performances (Extended Data Fig. 7g-h). An example of DeePict segmentation outcome \\nin the cross-domain setting is shown in Fig. 4a-b and visually resembles the ground truth annotation \\n(Extended Data Fig. 8a). The DeePiCt predictions and the ground truth annotations enabled us to analyze \\nthe DEF tomograms in the same way as we had done for the VPP data, by downstream 3D structural \\nclassification and refinement (Fig. 5).  \\n \\nDeePiCt predictions result in high quality subtomogram averages  \\nIn the case of FAS in the DEF dataset, fewer particles were predicted than in the ground truth (Extended \\nData Tables 5, 6). Nevertheless, the subtomogram average of the detected particles revealed the two half \\ndomes of the barrel-shaped type I FAS with applied D3 symmetry, consistent with the ground truth average \\nand published structures of this fungal 2.6-megadalton heterododecameric (α6β6) complex from other yeast \\nspecies (Fig. 4c-d, Extended Data Fig. 8b-e). The phosphopantetheine transferase (PPT) domains required \\nfor activation of FAS could be observed along the equatorial plane of the complex. In addition, three \\nequatorial densities were resolved that could not be assigned based on published structures. Three densities \\ninside each half dome connected to the central α wheel and in close proximity to the ketosynthase (KS) fit \\nwith the acyl carrier protein (ACP) of Saccharomyces cerevisiae44 (Fig. 4d, PDB: 2uv8) and Pichia \\n', metadata={'x0': 72.0, 'x1': 542.7681274414062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76812744140625, 'height': 640.3059692382812, 'nlines': 44, 'fs': 14.55240839177912, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='pastoris45 (Extended Data Fig. 8c, EMD-12139). The ACP shuttles the growing acyl-chain between the \\ndifferent catalytic sites, and its localization has been suggested in connection with the activity of the whole \\nmulti-enzyme complex44,46–49. Here, S. pombe FAS complexes within FIB-lamellae prepared from \\nexponentially growing cells revealed a specific ACP site, which had not been observed by cryo-ET and \\nthus in its native state before (Fig. 4d, Extended Data Fig. 8, Supplementary Note 2). \\n \\nFor ribosomes, the particle averages and numbers purely derived from DeePiCt predictions are comparable \\nto the ground truth annotations in DEF (Extended Data Tables 5, 6; Supplementary Note 2, \\nSupplementary Fig. 4, 5) complementing the performance analysis results described above. 3D \\nrefinements of these lamella-derived ribosomal particles from both ground truth and DeePiCt resulted in \\nsubtomogram averages with the nominal resolutions of 11 Å and 15 Å after multi-particle refinement in \\nM33, respectively (Fig. 4e, Extended Data Fig. 8f, Extended Data Tables 5, 6). Hierarchical 3D \\nclassification revealed in both datasets a well-aligned class that was further refined in M to subnanometer \\nresolutions of 9.3 Å and 9.4 Å for ground truth and DeePiCt predictions, respectively (Fig. 4f-h, Extended \\nData Fig. 8i-k, Extended Data Tables 5, 6, Supplementary Note 2, Supplementary Fig. 4, 5). This \\nallowed the identification of tRNA occupying the P-site of the peptidyl transferase center (PTC) as well as \\nthe L1 stalk facing the E-site (Fig. 4g-h, Extended Data Fig. 8j-k).  \\n \\nDeePiCt-predicted ribosomes reveal functional sub-populations \\nThe large number of particles that DeePiCt localized in DEF in a high throughput manner allows for \\nexamining subpopulations of macromolecular complexes that are functionally distinct by downstream \\nsubtomogram analysis. Further classification of all DeePiCt-predicted ribosomes focused on the head of \\nthe 40S small subunit revealed a subset with additional densities close to the head and at the exit tunnel \\n(Fig. 4i, Supplementary Fig. 6). This class was also detected in VPP and in DEF ground truth datasets \\n(Extended Data Tables 4-6). In the DEF ground truth dataset, densities for P- and E-site tRNAs and the \\nL1 stalk facing inwards are resolved (Extended Data Fig. 9a-e). The ribosome-bound ATPase eEF3 from \\nS. cerevisiae50 fitted well into the additional head density (CC 0.8972, EMD-12062, Fig. 5i, Extended Data \\nFig. 9 a-c). During translation, this eukaryotic elongation factor facilitates binding of a new tRNA to the \\nA-site of the ribosome via the ternary aminoacyl-tRNA–eEF1A–GTP complex50,51. In addition, unassigned \\ndensity at the ribosomal exit tunnel is observable for this class. Focused classification of all DeePiCt-\\npredicted ribosomes at this location revealed a subtomogram average fitting the S. cerevisiae ribosome52 \\n(CC 0.9657, EMD-1667) with the rRNA expansion segment ES27L in a particular configuration53 (CC \\n0.7938, PDB-3izd) connecting to an additional density close to the ribosomal exit tunnel (Fig. 4j, \\nSupplementary Fig. 7). ES27L plays a role in translation fidelity and recruits enzymes to the ribosomal \\nexit tunnel, such as the methionine aminopeptidase (MetAP) that co-translationally processes the nascent \\npeptide chain54,55. The nuclear export factor Arx1, which is released during cytosolic ribosomal 60S \\nmaturation in S. cerevisiae56, and its human homologue Ebp1, a translation regulator57,58, similarly bind at \\nlocations of the observed extra density (Fig. 4j, Extended Data Fig. 9h). These binding factors recruit the \\nflexible rRNA scaffold ES27L and cover the ribosomal exit tunnel with their MetAP-like folds. This \\nstructural class was also detected in DEF gt and VPP gt datasets (Extended Data Fig. 9f-j, Extended Data \\nTables 4-5).  \\n \\nThus, DeePiCt predictions combined with structural analysis revealed functional subpopulations of \\nribosomes. Mapping them back into the tomograms revealed their 3D spatial distributions in relation to \\n', metadata={'x0': 72.0, 'x1': 542.766845703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.766845703125, 'height': 638.5902099609375, 'nlines': 44, 'fs': 14.51341386274858, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"other macromolecules such as FAS and organelles in the cellular context (Fig. 4b). \\n \\nDeePiCt's combination of organelle segmentation and high-throughput particle localization \\nenables structural insights into ribosome-mitochondria association  \\nOne advantage of DeePiCt over state-of-the-art tools is that it allows studying particle populations within \\ntheir cellular context, by recovering particles in close proximity to specific organelles selected from \\npredictions by the 2D CNN. We recovered cytosolic ribosomes within 25 nm distance to predicted \\nendoplasmic reticulum (ER) and mitochondria from 7 and 3 DEF tomograms, respectively. Using \\nsubtomogram analyses for both subsets, two classes could be separated, one with and one without a \\nmembrane density (Supplementary Fig. 8-9). In the first class, ribosomes associated with the membrane \\nin a specific orientation, whereas they were randomly oriented in the second class. The latter class likely \\narises from the highly crowded nature of the S. pombe cytoplasm (Fig. 4k-l), which in the case of DeePiCt \\npredictions in DEF data, is also affected by imperfect organelle segmentations (Fig. 4b). ER-bound \\nribosomes faced the membrane with the ribosomal exit tunnel in agreement with published structures from \\nother species such as S. cerevisiae1,52,59 (Extended Data Fig. 9 k-n, Extended Data Table 6). \\nMitochondria-bound ribosomes, which posed a particular challenge to structural analysis in previous \\nstudies59,60, were found to interact with the membrane at an angular offset of around 35° in comparison to \\nthe ER-bound ribosomes (Fig. 4m-n, Extended Data Fig. 9o-r, Extended Data Table 6). Interestingly, a \\ndensity connecting the ribosome to the mitochondrial membrane was detected at the bottom of the large \\nsubunit in close proximity to the small subunit, but not to the ribosomal exit tunnel. It possibly represents \\nthe ribosome nascent chain complex (NAC) in contact with the mitochondrion receptor OM1461–63 which \\nhas yet to be structurally described.  \\n \\nIn summary, ribosomes close to mitochondria and ER exhibit different interfaces with the respective \\nmembranes, facilitating specific protein nascent chain membrane insertion or transfer into the \\norganelle2,52,62. These results highlight the power of DeePiCt for high-throughput particle localization and \\ncryo-electron tomogram segmentation to rapidly gain new biological insights. \\n \\nTrained networks can be readily applied to other species \\nAs a demonstration of the domain generalization potential of our workflow across species, we predicted \\nsegmentations of ribosomes, membranes and cytoskeleton structures (actin filaments and microtubules) in \\na published VPP HeLa cell dataset1 (Fig. 5). The application of the DeePiCt pipeline employed a manually \\ngenerated mask using Amira64 of the cytoplasmic volume, excluding the nucleus. For the evaluation of our \\nresults, we computed the voxel-F1 score in comparison to the publicly available annotations (EMD-\\n119921). For ribosomes and membranes, we used the ribosome- and membrane-networks trained on the 10 \\nVPP-tomograms from S. pombe described in previous sections. This resulted in a voxel-F1 of 0.55 for \\nribosomes and a voxel-F1 of 0.18 for membranes (Extended Data Fig. 10a). However, the available expert \\nmembrane segmentation for the dataset covered only ER membranes, which makes the voxel-F1 value less \\nmeaningful. Visual inspection of the predicted membrane segmentation revealed a good fit (Fig. 5). \\n \\nAdditionally we generated models for microtubules prediction, initially training a network for simultaneous \\nactin and microtubule segmentations in 4 tomograms of cryo-FIB milled Human retinal pigment epithelial-\\n1 (RPE-1) cells65. Due to the high preferential orientation of the cytoskeletal filaments in this data exhibiting \\nstress fibers, the performance for microtubules’ segmentation was low resulting in a voxel-F1 score of 0.26 \\n\", metadata={'x0': 72.0, 'x1': 542.9998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.9998779296875, 'height': 640.3059692382812, 'nlines': 44, 'fs': 14.55240839177912, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(Extended Data Fig. 10a). We therefore trained a second 3D CNN for microtubule segmentation on 11 \\ntomograms from lamella of cryo-FIB-milled, dissociated C. elegans cells, containing 890 microtubules in \\na wide range of orientations. This new microtubule segmentation network achieved a voxel-F1 score of \\n0.83 (Extended Data Fig. 10a), exemplifying the importance of a training set with a high number of \\nstructures, as well as orientations to mitigate the effect of the missing wedge on the prediction performance.  \\n \\nFinally, we trained a dedicated network for predicting actin-filaments on 5 manually curated tomograms, 2 \\nfrom RPE-1 cells and 3 from Mouse embryonic fibroblasts (MEF) 3T3 cells, that contained approximately \\n3,740 actin filaments (Online Methods). The actin filaments network shows a low voxel-F1 score of 0.10 \\nin the HeLa cell tomogram (Fig. 5, Extended Data Fig. 10a), which is likely caused by the finer structure \\nof actin in comparison to microtubules, the minimal set of orientations sampled in the small amount of \\ntraining data, and to the fact that most training filaments are arranged in bundles, probably causing the CNN \\nto learn this superstructure rather than individual filaments (Extended Data Fig. 10b). \\n \\nOverall, these results show that the prediction of ribosomes and microtubules, representing large \\nmacromolecular complexes, is especially well-preserved across species and that the availability of diverse \\nand well-annotated training datasets are crucial for good performance of DeePiCt. The results shown in \\nFig. 5 highlight the use of trained networks from DeePiCt to segment novel datasets spanning different \\nspecies.  \\n \\nDiscussion \\n', metadata={'x0': 72.0, 'x1': 545.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 473.6400146484375, 'height': 307.5884704589844, 'nlines': 21, 'fs': 14.647070021856399, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Our DeePiCt workflow facilitates accurate and fast localization of diverse structures in cryo-ET of intact \\ncells. The demonstrated high performance and the flexibility of the 3D CNN architecture offer the user a \\nreliable tool for pattern recognition. This enabled us to detect particle species, such as FAS, that are low in \\nabundance and have a less dense structural signature in comparison to ribosomes. The integration of \\nstructure segmentation (predicted by the 3D CNN) with the contextual information excludes false positives \\nin the 3D localization of particles and structures segmentation task, and harnesses the cellular context to \\ncarry out spatial studies focused on regions of biological interest. This enabled us to investigate the 3D \\nspatial distribution of macromolecules such as ribosomes in proximity to specific organelles (e.g. \\nmitochondria) and to obtain structural insights with functional implications. \\n \\nHere, we provide the first publicly available experimental cryo-ET dataset of 20 S. pombe tomograms under \\ntwo microscopy acquisition settings (VPP and DEF), together with a high-quality comprehensive \\nannotation of RIBO and FAS, membranes, organelle and cytosol segmentations. This constitutes the first \\ngold-standard dataset in the field that is large enough for unbiased model training, which will enable \\nbenchmarking of current methods, and spur the development of future computational tools for unbiased \\ndata mining in cryo-ET data.  \\n \\nSubtomogram averaging of the annotated particles from either ground truth or DeePiCt predictions resulted \\nin the first density maps of the S. pombe ribosome and fatty acid synthase. We observed differences in the \\nanalysis of subtomograms derived from either VPP and DEF tomograms, despite both being derived from \\nwild-type S. pombe cryo-FIB lamellae (Supplementary Note 2). Due to the preservation of high frequency \\nfeatures in DEF data, a subset of well-aligned ribosomes could be identified and refined to subnanometer \\n', metadata={'x0': 72.0, 'x1': 542.7723388671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7723388671875, 'height': 317.5059509277344, 'nlines': 22, 'fs': 14.43208867853338, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"resolution densities from cryo-FIB lamellae. The obtained maps provide detailed structural insights; for \\nexample, the P-site tRNA occupying the PTC, which together with the elongation factor eEF3 detected \\nclose to the head of the 40S subunit in a subset of ribosomes, suggest active translation. Further \\nclassifications close to the ribosomal exit tunnel revealed ES27L in a specific configuration providing a \\nplatform for other factors to interact with the exit tunnel and a potentially protruding nascent chain. By \\nutilizing the contextual information of the DeePiCt predictions, differences in relative membrane \\norientation of mitochondria- and ER-bound S. pombe ribosomes were detected. The observed connecting \\ndensity between ribosomes and mitochondria likely highlights the not yet structurally characterized \\nribosome-bound NAC in contact with OM-1461–63  facilitating protein import into mitochondria60,61.  \\n \\nThe analysis of DeePiCt's performance confirmed that data quality is an important factor in its predictive \\nability, as demonstrated in the case of NPC predictions in the DEF* dataset, which we classified into high \\nand lower quality subsets based on lamella thickness and tilt-series alignment error. As the NPC has a high \\ndegree of structural flexibility on the subunit and diameter level inside cells40 it is a more challenging target \\nthan for example ribosomes and thus emphasizes the importance of higher quality data to achieve sufficient \\nperformance. The introduction of a pre-processing equalizing filter is also an important element of the \\npipeline that improves both the learning process during the training of 3D segmentation networks for \\nparticles with less dense print than the ribosome (e.g. FAS), and especially the generalization power across \\ndomains, including different microscopy acquisition conditions. Thus, amplitude spectrum equalization \\nallows localizing macromolecules across different data acquisition types. In the case of the 2D network, \\nalthough pre-processing did not improve model performance on the same-domain inference, it led to cross-\\ndomain improvements when training on VPP data and inferring on DEF data, or when training on both data \\ntypes combined to segment organelles and cytosol. More elaborate tasks, such as prediction of individual \\norganelle types, will likely require more training data or training of a dedicated 3D CNN with a tailored \\nnetwork architecture. \\n \\nOur workflow allows easy adaptation to the segmentations of other structures, as demonstrated by the \\napplication of cytoskeleton segmentation networks. The networks show high-quality performance for \\nmicrotubules in the HeLa cell dataset after training on data with broad orientation sampling of the filaments. \\nActin predictions revealed a low F1 score and therefore likely require more training data sampling different \\norientations of the structural features to improve performance. Altogether, the application of multiple \\nsegmentation networks to the HeLa cell dataset revealed that DeePiCt trained on datasets from different \\nmicroscopes, species, and conditions lead to reasonably good results when the quality of data is high. \\nAlthough more in-depth analysis is needed to study the specific limitations for the applicability of DeePiCt \\nnetworks on other datasets, the results presented here constitute the first step towards conducting large-\\nscale quantitative analyses for structural biology using cryo-ET on cells from different laboratories and \\nopen source datasets. In this sense, the generated ground truth annotations in this study are a starting point \\nto provide the community with a resource to improve and further develop cryo-ET object segmentation and \\ndetection tools, which will ultimately enable further exploration of particles in their cellular context. \\nTogether with the trained networks and their demonstrated applicability to other species, as well as the \\nflexibility of the DeePiCt workflow, the software harbors great potential for quantitative cryo-ET studies \\nin the future. \\n \\n\", metadata={'x0': 72.0, 'x1': 542.77099609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77099609375, 'height': 623.2659301757812, 'nlines': 43, 'fs': 14.494556515715843, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Acknowledgments \\n', metadata={'x0': 72.0, 'x1': 188.33999633789062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 116.33999633789062, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='We thank Jenia Zagoriy and Wim Hagen for technical support, EMBL IT and especially Thomas Hoffman \\nfor computational support. I.D.T. and M.T.N were supported by a fellowship from the EMBL \\nInterdisciplinary Postdoctoral Program (EI3POD) under Marie Skłodowska-Curie Actions COFUND. J.M. \\nacknowledges funding from the EMBL and the European Research Council starting grant (3DCellPhase-\\n760067). \\n', metadata={'x0': 72.0, 'x1': 542.7700805664062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77008056640625, 'height': 70.30595397949219, 'nlines': 5, 'fs': 14.061190795898437, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Author contributions \\n', metadata={'x0': 72.0, 'x1': 201.05999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 129.05999755859375, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='I.D.T., J.M. and J.Z. conceived the study. I.D.T. implemented the 3D CNN code, co-produced ground truth \\nannotations, carried out numerical experiments to evaluate performance. S.K.G. prepared S. pombe cryo-\\nFIB lamellae and acquired cryo-electron tomograms, co-produced ground truth annotations, carried out \\nsubtomogram averaging and structural analysis. A.M. implemented the 2D CNN and spectrum equalization \\nfilter, produced ground truth segmentations and conducted performance analysis of the 2D CNN. F.S. \\ncarried out training and prediction with 3D CNN networks, implemented code and created Colab Notebooks \\nfor prediction. C.Z. generated data and provided NPC annotations under the supervision of M.B. M.T.N. \\ngenerated RPE1 and MEF 3T3 data and together with D.W.C.C. produced ground truth and trained \\nnetworks for actin segmentation. F.T. generated data, ground truth and trained networks for microtubule \\nsegmentation. C.P. and A.K. provided valuable input for the design and generation of the CNNs. I.D.T., \\nS.K.G., A.M., J.M. and J.Z. wrote the manuscript with input from all authors. \\n', metadata={'x0': 72.0, 'x1': 542.771240234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.771240234375, 'height': 157.66595458984375, 'nlines': 11, 'fs': 14.333268599076705, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Code and data availability \\n', metadata={'x0': 72.0, 'x1': 233.70001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 161.70001220703125, 'height': 20.9967041015625, 'nlines': 1, 'fs': 20.9967041015625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The DeePiCt code is assembled as two Snakemake pipelines (2D CNN and 3D CNN); the 3D CNN is \\nimplemented in the Pytorch framework while the 2D CNN is implemented in Keras, both in Python 3. The \\ncode, trained models, and link to the Google Colab notebook are available in the github repository \\nhttps://github.com/ZauggGroup/DeePiCt.   \\n', metadata={'x0': 72.0, 'x1': 542.7699584960938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76995849609375, 'height': 55.90594482421875, 'nlines': 4, 'fs': 13.976486206054688, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Raw tilt series, tomograms, ground truth coordinates and segmentations are available via EMPIAR \\naccession codes EMPIAR-10988 (S. pombe) and EMPIAR-10989 (RPE1). Subtomogram averages for S. \\npombe VPP and DEF ground truth annotations are available on EMDB (VPP gt: EMD-14404, EMD-14405, \\nEMD-14406, EMD-14408, EMD-14409, EMD-14410, EMD-14411; DEF gt: EMD-14412, EMD-14413, \\nEMD-14415, EMD-14417, EMD-14418, EMD-14419, EMD-14420; DEF DeePiCt predicted: EMD-\\n14422, EMD-14423, EMD-14424, EMD-14425, EMD-14426).  \\n', metadata={'x0': 72.0, 'x1': 542.7652587890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7652587890625, 'height': 84.9459228515625, 'nlines': 6, 'fs': 14.15765380859375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Structural comparisons were performed with S. cerevisiae FAS (PDB: 2uv844), P. pastoris FAS (EMD-\\n1213945), eEF3 from S. cerevisiae (EMD-1206250), the S. cerevisiae ribosome (EMD-166752) with the \\nrRNA expansion segment ES27L in a particular configuration (PDB-3izd53), the nuclear export factor Arx1 \\nbound to the 60S large ribosomal subunit S. cerevisiae (EMD-216956), the human Ebp1 (EMD-106857), S. \\ncerevisiae ribosomes derived from extracted ER (EMD-376459), and the ER-bound HeLa ribosomes (EMD-\\n80561). The large subunit (LSU, 60S) of a published S. cerevisiae 80S ribosome map (EMD-322866) and \\nthe S. cerevisiae FAS map (EMD-162346) were used as references for TM in S. pombe tomograms. The \\nHeLa cell dataset is available via accession code EMD-119921. \\n', metadata={'x0': 72.0, 'x1': 542.76025390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76025390625, 'height': 115.15020751953125, 'nlines': 8, 'fs': 14.393775939941406, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Competing interests \\n', metadata={'x0': 72.0, 'x1': 196.25999450683594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 124.25999450683594, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The authors declare no competing interests. \\n', metadata={'x0': 72.0, 'x1': 266.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 194.52001953125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Additional information \\n', metadata={'x0': 72.0, 'x1': 213.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 141.47999572753906, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Information is available for this paper. \\n', metadata={'x0': 72.0, 'x1': 313.32000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 241.32000732421875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figures \\n', metadata={'x0': 72.0, 'x1': 119.39999389648438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 47.399993896484375, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 1 | DeePiCt 2D and 3D CNN architecture implemented in an automated workflow combining \\ncompartment segmentations and particle localizations in cryo-ET data. a. The CNN U-Net architecture: the 2D \\nnetwork performs all tensor operations on the two-dimensional spatial coordinates, with D=5 and IF=16; the 3D \\nnetwork performs tensor operations on the three spatial dimensions, and the architectural hyperparameters in red can \\nbe set by the user. b. The DeePiCt pipeline is used to train and predict various structural features in cryo-electron \\ntomograms. b.1 The DeePiCt pipeline consists of two independent CNNs: a 2D network for compartment \\nsegmentation and a 3D network for particle localization. b.2 Trained networks are applied to input tomograms which \\ncan be pre-processed with a spectrum-matching filter to improve image contrast. The example 2D tomographic slice \\nvisualizes the cytoplasm with the endoplasmic reticulum (ER), vacuoles (V), nucleus and extracellular space (ECS). \\nb.3 DeePict raw predictions for cytosol, membranes, ribosomes and fatty acid synthase (FAS) are post-processed by \\nthresholding, cluster size and centroid fitting. b.4 The outputs of the two networks can be combined to include the \\ncellular context by intersecting particle predictions with cytosol masks (top), selecting particles (NPC, green) in \\ncontact with specific membranes (nuclear envelope, middle), and to identify particles (e.g. ribosomes, orange) \\nassociated with specific organelles (e.g. ER). \\n \\n \\n \\n \\n \\n  \\n', metadata={'x0': 72.0, 'x1': 542.527587890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.527587890625, 'height': 183.00283813476562, 'nlines': 20, 'fs': 9.15014190673828, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 2 | Iterative comprehensive annotation of ground truth for non-synthetic data. The three columns (left to \\nright) show: the annotation process for RIBO (a, d, g), FAS (b, e, h), membranes (c, f). a-f Show the cumulative \\npredictions of 3 rounds of DeePiCt for RIBO and one round for FAS as cross-section overlaid on a tomographic slice \\n(a-c) and in 3D view (d-f). Particles are classified as true positives recovered from the initial annotation (tp recovered; \\nyellow for RIBO; green for FAS), new true positives provided by DeePiCt predictions (tp new; blue), and the \\ncorresponding false positives (fp; red). g, h. 3D views of the resulting ground truth, including false negatives (fn) \\ndistinguishing unrecovered fn from the initial annotations (pink) and final round of manual picking (salmon). i. \\nCombined ground truth annotation of RIBO, FAS, membranes, and NPC (green cylindrical mask, manual annotation). \\nj-k. Relative contributions of the DeePiCt rounds for RIBO (j) and FAS (k) identification are plotted across 10 VPP \\nand 10 DEF S. pombe tomograms: contribution of step 1 (yellow: TM and manual annotation for RIBO; green: manual \\nannotation for FAS; salmon: initial annotations not detected by DeePiCt), step 2 (blue); and step 3 (pink bars). For \\nFAS the initial annotation was not based on TM, and instead was purely manual. l, m. Subtomogram averages of all \\nRIBO, exclusively DeePiCt-detected (tp new; blue) and all FAS particles in VPP and DEF ground truth, respectively. \\n \\n', metadata={'x0': 72.0, 'x1': 542.5265502929688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.52655029296875, 'height': 184.17047119140625, 'nlines': 14, 'fs': 13.155033656529017, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 3 | DeePiCt performance, cross-domain generalization and comparison with other methods. a. 2D CNN \\nperformance results for organelle and cytosol segmentation under a 5-fold CV scheme, when training and testing \\ndatasets are both in the same domain (VPP). The median AUPRC scores are indicated (black dotted lines). b. \\nPerformance of DeePiCt for the same-domain setting. The RIBO localization, FAS localization, and membrane \\nsegmentation tasks are shown (left to right), under a 3-fold CV scheme. In each case, the corresponding architectures \\nof the 3D CNN were optimized by hyperparameter tuning (Supplementary Table 1, Extended Data Fig. 6). The \\nmedian F1 score for RIBOS and FAS, and a median voxel-F1 for membranes are indicated (black dotted lines). c. \\nSame-domain NPC segmentation results using a 3-fold CV scheme. From left to right: performance in all DEF* \\ntomograms, high quality DEF* tomograms, and lower quality DEF* tomograms. d-e. DeepFinder (d) and TM (e) \\nparticle localization results for RIBO and FAS localization. The median F1 values are indicated in each case (black \\ndashed line), along with corresponding values of DeePiCt’s performance for comparison (red dashed line). d. \\nDeepFinder results are based on the same 3-fold CV scheme as in b, by training a multiclass DeepFinder network that \\nsimultaneously segments FAS and RIBO. e. TM results are shown for the VPP tomograms. f-g. Results of the cross-\\ndomain generalization for the 2D CNN (f) and DeePiCt (g) (training on 10 VPP tomograms, testing on 10 DEF \\ntomograms). Dashed lines and their numerical labels indicate median performance values either of the corresponding \\nplot (black), or the corresponding experiment performed in the same domain with DeePiCt (red), or in the same domain \\nwith the 2D CNN (green).  \\n \\n', metadata={'x0': 72.0, 'x1': 542.5325317382812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.53253173828125, 'height': 237.21044921875, 'nlines': 18, 'fs': 13.178358289930555, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='19 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='20 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 4 | DeePiCt enables exploration of macromolecular complexes in their cellular context. a. 2D slice of a \\nDEF S. pombe cytoplasm tomogram. Mitochondrion (M), vesicle (V), the ER and the cell wall (CW) are detectable in \\nthe raw (bottom left) and with improved contrast after amplitude spectrum equalization (top right). b. DeePiCt \\npredictions generated with models trained in VPP data. Organelles (gray), membranes (purple), FAS (pink, c, d), \\nribosomes (yellow, e-h) and subsets classified in RELION (head density, dark blue, i; exit tunnel density, bright blue, \\nj), and within 25 nm to mitochondria (mito-bound, green, m) and the ER (ER-bound, orange, n). c. FAS subtomogram \\naverage (pink) fits the S. cerevisiae structure (cyan) including PPT domains. An extra density cannot be assigned. d. \\nCross-section of c close to the alpha-wheel with three densities fitting ACPs (white asterisks). e. Subtomogram average \\nof all ribosomes from 10 DEF tomograms. f. Well-aligned ribosome subset detected by hierarchical 3D classification \\nin RELION and refined in M. g, h. Slices through (f) reveal the PTC with a P-site tRNA and the L1 stalk facing the \\nE-site. i. Ribosome subclass with additional densities (white arrowheads) close to the head of the small ribosomal \\nsubunit which fits eEF3 (red), and close to the ribosomal exit tunnel. j. Ribosomes classified for a density below the \\nribosomal exit tunnel (white arrowhead). The S. cerevisiae ribosome with ES27L in a particular configuration (left, \\npurple) connects to the additional exit-tunnel density which fits Arx1 bound to the 60S pre-ribosome (middle and \\nright, purple). k, l. Different z slices of the tomogram in a (white dashed boxes) show ribosomes bound to the ER (k) \\nand a mitochondrion (l). m. ER-bound ribosomes from 7 tomograms show a connection of the peptide exit tunnel of \\nthe large subunit to the membrane density. n. Mitochondria-bound ribosomes from 3 tomograms show a linker \\nconnecting the large subunit, at a site close to the small subunit, to the membrane density. Overlay of the ribosomes \\nin m and n shows different interfaces with the respective organelle membranes.  \\n \\n', metadata={'x0': 72.0, 'x1': 542.5247802734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.5247802734375, 'height': 249.45050048828125, 'nlines': 20, 'fs': 12.472525024414063, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='21 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Figure 5 | DeePiCt’s generalization across species. A dataset depicting a HeLa cell nuclear periphery1 is segmented \\nby applying four independently trained DeePiCt networks. The results show the segmentation of actin filaments (red) \\ntrained on RPE-1 and MEF 3T3 tomograms, microtubules (MTs, cyan) trained on C. elegans tomograms, and cytosolic \\nribosomes (yellow) and membranes (purple) trained on S. pombe tomograms.  \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 542.521728515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.521728515625, 'height': 83.20498657226562, 'nlines': 7, 'fs': 11.886426653180804, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='22 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Online Methods \\n', metadata={'x0': 72.0, 'x1': 172.74000549316406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 100.74000549316406, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Yeast cell culture \\nS. pombe K972 Sp h- wt haploid was kindly provided by C. Haering, originally from P. Nurse. Cells were \\nrecovered from frozen stock by streaking on YE5S agar plates (YES Broth, Formedium, 20 g agarose/L) \\nand incubated at 30 °C for 1-3 days. Colonies were re-streaked on fresh YES agar plates and incubated 1-\\n3 days at 30 °C. Single colonies were inoculated in 5 mL of YES medium (YES Broth, Formedium, \\nPCM0302, FM0618/8573) and grown at 30 °C, 170 rpm overnight (NCU-Shaker mini, Benchmark). On \\nthe next day, cultures were grown to their log phase at OD600 of 0.5 - 0.6 and diluted beforehand in YES if \\nnecessary.  \\n  \\nVitrification of yeast cells \\nA Leica EM GP (Leica Microsystems) was utilized to vitrify yeast cells at liquid nitrogen temperature. \\nYeast cells were either diluted to OD600 of 0.2-0.4 in YES medium or, following a prior wash step, in PBS \\ncontaining 5 % or 10 % BSA as cryoprotectant. In the chamber, 4 µl were directly applied to TEM grids \\n(Quantifoil R1/2, Cu 200 mesh, holey carbon or SiO2 film), which were glow discharged on both sides for \\n45 s (Pelco Easy glow) immediately before usage. Blotting from the back side of the support was performed \\nfor 1-2 s at 22 °C and 99% humidity. Cells were plunge-frozen in liquid ethane cooled by liquid nitrogen \\nand transferred into grid boxes until further usage. \\n  \\nCryo-FIB of S. pombe \\nTEM grids containing vitrified yeast cells were clipped into an autogrid with a cut-out enabling cryo-\\nfocused ion beam (cryo-FIB) milling at shallow angles67. Mounted on a 45° pre-tilt shuttle, grids were \\ntransferred into an Aquilos Dual beam microscope (ThermoFisher Scientific). Prior to cryo-FIB milling at \\nliquid nitrogen temperature, cells were sputter-coated with platinum for 10-15 s (1 kV, 10 mA, 10 Pa). \\nSubsequently, a layer of organometallic platinum (reservoir at 28°C) was applied by opening the gas \\ninjection system (GIS) for 8 sec at a stage Z position of 3-4 mm below the coincidence point. In three \\nindependent sessions, three grids with 5 lamellae each were prepared at a milling angle of 15°. \\nAgglomerations of several cells were micromachined in three steps of rough milling to a thickness of 5 µm \\nat 1 nA ion beam current, 3 µm at 0.5 nA and 1 µm at 0.1 nA. Milling progress was visually monitored \\nbetween each milling step with the scanning electron microscope beam (SEM, 10 kV, 50 pA). Fine milling \\nwas performed at 50 pA to a target thickness of 200 nm. To render the lamellae conductive for TEM \\nimaging, they were sputtered with platinum for 5 s (1 kV, 10 mA, 10 Pa) and transferred into cryo boxes. \\n  \\nCryo-electron tomography of S. pombe \\nAutogrids containing lamellae were loaded into a Titan Krios (Thermo Fisher Scientific) such that the axis \\nof the pre-tilt introduced by FIB milling was aligned perpendicular to the tilt axis of the microscope68. Cryo-\\nelectron tomography acquisition parameters are summarized in Extended Data Tables 4-5. Tomograms \\nwere acquired at 300 kV on a K2 Summit direct detection camera (Gatan) operating in dose fractionation \\nmode and utilizing a Quantum post-column energy filter operated at zero-loss (Gatan). Magnification of \\n42,000 (EFTEM) with a calibrated pixel size of 3.37 Å was used for the ground truth VPP and DEF datasets \\nand 3.45 Å for NPC data. Up to 14 tilt series were collected on a single lamella in low dose mode using \\nSerialEM69. Starting from the lamella pre-tilt, images were acquired at 2.0 - 4.0 µm underfocus, in 2° \\nincrements within a range of + 58° to - 40° using a dose-symmetric tilt scheme70. The total dose was up to \\n', metadata={'x0': 72.0, 'x1': 542.7710571289062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77105712890625, 'height': 612.7059936523438, 'nlines': 42, 'fs': 14.588237944103422, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='23 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='120 e-/Å2 with a constant electron dose per tilt image. For the ground truth data, a set of 10 tilt-series were \\neither collected with a 70 µm objective aperture or a Volta potential phase plate (VPP, Thermo Fisher \\nScientific39) with prior conditioning for 5 min. NPC data was collected as described in previous work71,40, \\nbut at 1.5 - 4.5 µm underfocus, with 3° increments and an effective tilt range of + 50° to - 50°.  \\n \\nTomogram reconstruction \\nTilt movie frames were aligned using a SerialEM plugin. Tilt series were filtered according to the \\naccumulated \\nelectron \\ndose \\nby \\nFourier \\ncropping \\nusing \\nthe \\nmtffilter \\nfunction \\nin \\netomo \\n', metadata={'x0': 72.0, 'x1': 542.770263671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.770263671875, 'height': 114.91020202636719, 'nlines': 19, 'fs': 6.0479053698088, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(IMOD/BETA4.10.1273), and sorted by tilt angle using a python script. Four times binned tilt images (movie \\nsums) were aligned in etomo (IMOD/4.9.4) software package73 using patch tracking (residual error 0.291 - \\n0.569 px) and tomograms were reconstructed via weighted back projection. Tomogram thicknesses \\nmeasured in 3dmod ranged between 80 - 310 nm. \\n', metadata={'x0': 72.0, 'x1': 542.7698364257812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76983642578125, 'height': 56.83024597167969, 'nlines': 4, 'fs': 14.207561492919922, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ground truth annotation for organelles, cytoplasm, and membranes in VPP and DEF \\n', metadata={'x0': 72.0, 'x1': 475.32000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 403.32000732421875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular compartment segmentation was performed for both the VPP and DEF datasets. The annotations \\ninclude 10 different organelle classes (mitochondria, vesicle, tube, ER, nuclear envelope, nucleus, vacuole, \\nlipid droplet, Golgi apparatus, vesicular body, see Extended Data Table 1) and cytosol annotations, each \\nidentified through a unique numerical label to allow for selection of specific subsets of compartments.. \\nSegmentations of the 10 VPP tomograms were performed manually in Amira64. These annotations were \\nused to train a 2D CNN and to predict the initial segmentations in 10 DEF tomograms, which were first \\npre-processed with the spectrum equalization filter. These predictions were then manually corrected to \\nobtain high-quality ground truth segmentations in DEF (Extended Data Table 1).  \\n', metadata={'x0': 72.0, 'x1': 542.7678833007812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76788330078125, 'height': 114.2259521484375, 'nlines': 8, 'fs': 14.278244018554688, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=\"Membrane annotations were performed on the 10 VPP tomograms and 5 DEF tomograms. Initially, 5 VPP \\ntomograms were annotated using Amira to manually segment every 2 to 3 slices, which were subsequently \\ninterpolated by means of Amira's interpolation tool. These 5 VPP tomograms were then used to train a \\nmembrane segmentation 3D CNN, whose predictions on the remaining 5 VPP tomograms as well as in 5 \\nDEF tomograms, were manually corrected in Amira.  \\n\", metadata={'x0': 72.0, 'x1': 542.7700805664062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.77008056640625, 'height': 70.54595947265625, 'nlines': 5, 'fs': 14.10919189453125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ground truth particle annotation in VPP \\n', metadata={'x0': 72.0, 'x1': 267.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 195.48001098632812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='RIBO and FAS were localized in 4-times binned tomograms (13.48 Å pixel size) in an iterative workflow. \\nTM (RIBO) or manual annotations (FAS) (step 1, Extended Data Tables 2-3) were used for training 3D \\nCNNs (step 2). CNN predictions were masked with a segmentation of the cytosol. Step 2 was repeated \\nseveral times and each training round trained on the combined predictions of step 1 and the preceding round. \\nEach round in step 2 consisted of three simultaneously trained networks sharing the same set of default \\nhyperparameters (Supplementary Tables 2-3), except for the number of IF=4, 8, and 32, to provide a \\ncumulative prediction less overfitted to the incomplete training data. The cumulative total predictions from \\nsteps 1 and 2 were then complemented by a round of visual inspection to eliminate false positives and \\nmanual picking to recover remaining false negatives (step 3, Extended Data Tables 2-3). Finally, the \\nparticle lists were cleaned for duplicates by applying elliptic distance constraints to the coordinates \\n(Supplementary Note 1, Extended Data Tables 2-3). \\n \\nIn particular, for RIBO in the VPP dataset, the initial annotation (step 1) was performed as a combination \\nof two sub-steps: step 1a employed TM performed with the pyTOM toolbox36. In detail, a 3D cross-\\n', metadata={'x0': 72.0, 'x1': 542.7723388671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7723388671875, 'height': 201.34600830078125, 'nlines': 14, 'fs': 14.381857735770089, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='24 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='correlation search over 1,944 Euler angle combinations was conducted using the large subunit (LSU, 60S) \\nof a published S. cerevisiae 80S ribosome map (EMD-322866) scaled to the corresponding pixel size, and a \\nspherical mask with a diameter of 337 Å applied. The obtained coordinates corresponding to the top 2000-\\n3000 highest cross correlation scores were manually revised in Gaussian filtered tomograms (sigma = 3) \\nwith tom_chooser (pyTOM toolbox36). This was followed by step 1b, which consisted of a non-exhaustive \\ncomplementary manual annotation of particles missed by TM (Extended Data Tables 2-3). These initial \\nribosome coordinates served as input for step 2: three training rounds of CNNs were successively applied, \\nas a global visual inspection confirmed that the false positive rate was low, while the fraction of true \\npositives recovered was substantial. The resulting CNN predictions were manually revised in tom_chooser \\nas described above. In step 3, remaining particles that were not detected by either TM or CNN were \\nmanually localized in either spectrum-matched or Gaussian filtered tomograms (sigma = 3) for up to 3 \\nrounds in EMAN2 spt2_boxer74. \\n  \\nFAS in the VPP dataset was localized in step 1 manually in EMAN2, as TM with a published S. cerevisiae \\nFAS46 map (EMD-1623) as reference failed (Extended Data Tables 3). In step 2 we performed only one \\nround of training with these initial annotations and the resulting predictions were revised in EMAN2. \\nAdditional FAS particles were localized manually.  \\n', metadata={'x0': 72.0, 'x1': 542.7679443359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7679443359375, 'height': 245.02593994140625, 'nlines': 17, 'fs': 14.413290584788603, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ground truth particle annotation in DEF \\n', metadata={'x0': 72.0, 'x1': 268.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 196.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='In the DEF dataset, the annotation procedure was the same as the one described for the VPP dataset, except \\nthat: 1) for RIBO the initial annotations relied on manually cleaned TM results and for FAS on incomplete \\nmanual picking, and 2) in step 2 of the RIBO ground truth construction, the first of the 3 rounds of CNN \\npredictions used the DEF training data (from the TM results), while for the remaining 2 rounds the \\npredictions were obtained using the models from the first and second rounds of RIBO VPP ground truth \\nconstruction (trained only on VPP data, Supplementary Table 2). The visual inspection in step 2 was \\nperformed over the combined predictions of both the previously mentioned VPP networks and a round of \\nDEF networks (IF=4, 8, 32) based on DEF TM results. After the final manual picking for missing particles \\n(not present in either the initial TM results or the manually cleaned network predictions), we achieved \\ncomprehensive RIBO and FAS annotations (Extended Data Tables 2-3).  \\n \\nComparison of cryo-ET-derived particle numbers with proteomics \\nCopy numbers of ribosomes and FAS per cell were calculated with the ground truth annotations for an S. \\npombe cell with the assumption of 30 % cytosolic volume75 of a total of 150 µm3 average cell volume76,77. \\nIt was also considered in the case of FAS that one fully assembled FAS complex is constituted by each six \\nalpha and beta subunits. \\n \\nNPC manual localization \\nNPCs were manually localized as described previously40. In brief, coordinates and initial orientations of \\n354 NPCs were manually determined in 127 4-times binned, SIRT-like filtered73 defocus-only tomograms, \\nas described earlier71. Two points located at opposing inner and outer nuclear membrane (INM-ONM) \\nfusion points were manually picked for each NPC using IMOD. A third point was then chosen located at \\nthe INM-ONM fusion ring such that all three points define the horizontal plane of the NPC particle. The \\nmean coordinates between the first two points were then used to assign the center of the NPC. The initial \\norientation of the NPCs was defined based on the normal vector to the defined NPC plane. A fourth point \\n', metadata={'x0': 72.0, 'x1': 542.7698974609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7698974609375, 'height': 361.4259948730469, 'nlines': 25, 'fs': 14.457039794921876, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='25 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='was chosen in the cytoplasmic side of the horizontal plane to assign the correct initial orientation and \\nprevent false upside-down assignment of initial orientations. The annotated NPC data was divided based \\non quality criteria: 38 tomograms of quality 1 have a thickness below 300 nm, thus display optimal image \\ncontrast, and a tilt-series alignment residual error below 0.7 px. Quality 0 was assigned to 89 tomograms \\nwith 300 - 395 nm thickness and a residual error of 0.7 - 5.0 for tilt series alignments. \\n \\nVoxel-level representation of ground truth \\nAs the final product of our ground truth generation process, we obtain voxel-based masks for membranes, \\ncellular organelles, cytoplasm, and particles (RIBO, FAS, NPC). For RIBO and FAS, the lists of coordinates \\nobtained were used to paste spherical masks (with radii of 10.78 nm for RIBO and 13.48 nm for FAS) \\ncentered on each of these points using a python script (Code and data availability). For the NPCs in DEF*, \\na subunit mask obtained by prior 3D averaging in novaSTA78, was pasted at each of the 8 subunit locations. \\nThe RIBO, FAS, membrane, organelle and cytosol masks in DEF and VPP along with the list of coordinates \\nand respective tomograms are used in the subsequent training and performance analysis and are available \\non EMPIAR (Code and data availability).  \\n \\nCytoskeletal filaments segmentation \\nMicrotubule and actin networks were trained on tomograms of Human retinal pigment epithelial-1 (RPE-\\n1) cells previously described65. Segmentations of the individual cytoskeletal elements performed using the \\nfilament tracing function in Amira79,80, followed by manual curation. Trained networks were applied to a \\ntomogram depicting the nuclear periphery of a HeLa cell and compared against the corresponding \\nsegmentations (EMD-11992). \\n', metadata={'x0': 72.0, 'x1': 542.771240234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.771240234375, 'height': 317.7459411621094, 'nlines': 22, 'fs': 14.442997325550426, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Subtomogram analysis for ribosomes and FAS \\nContrast transfer function (CTF) estimations, generation of 3D CTF models and subtomogram \\nreconstructions were performed in Warp32. CTFs were first estimated in the sums of raw tilt movies and \\nsubsequently in the tilt series taking the tilt angles into account. Subtomograms containing ribosomes and \\ntheir respective CTF models were reconstructed in volumes of 140³ pixels with a pixel size of 3.3702 Å \\nand a particle diameter of 350 Å. Initial alignments of the subtomograms were performed in RELION \\nversion 3.0.734 in 25 iterations of 3D classification, with the S. cerevisiae 80S ribosome (EMD-3228, low-\\npass filtered to 60 Å) as reference to generate an initial single class average. 3D refinements were performed \\nwith the resulting average as a reference. In the case of DEF data, this RELION-refined average was further \\nrefined in M33 to optimize particle poses, images and volumes warping to model non-linear deformations. \\nParticles were re-extracted and hierarchical 3D classifications (25 iterations each) were performed in \\nRELION. In the case of VPP data, hierarchical 3D classifications were performed directly after 3D \\nrefinements in RELION. Focused classifications were performed with binary masks indicated in the \\nrespective figures. Masks were generated from spheres produced in Matlab (MathWorks) in a cubic volume \\nof 50 pixels per side and a radius of 20 pixels, scaled to a pixel size of 3.3702 Å and converted into binary \\nmasks resulting in a radius of around 130 Å using RELION, and placed relative to the area of interest using \\nChimera81. \\n \\nFAS particles and CTF models were reconstructed in cubic volumes of 160 pixels per side with a pixel size \\nof 3.3702 Å and a particle diameter of 400 Å. Initial alignments were performed in RELION in 25 iterations \\nof 3D classification into a single class, and followingly refined with the 3D refinement option and applying \\n', metadata={'x0': 72.0, 'x1': 542.7735595703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7735595703125, 'height': 303.345947265625, 'nlines': 21, 'fs': 14.445045107886905, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='26 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='D3 symmetry using the S. cerevisiae FAS map (EMD-162346, low-pass filtered to 60 Å) as reference. \\nHierarchical 3D classifications (25 iterations each) were performed either after 3D refinements in RELION \\n(VPP data) or after subtomogram re-extraction using the ribosome-optimized image and volume models in \\nM (described above). \\n \\nFinal subtomogram averages of each particle class were obtained by 3D refinement and post-processing, \\nincluding filtering to their respective resolutions determined by Fourier Shell Correlation (FSC) of two \\nindependently refined half maps at a cut-off of 0.143. Details of particle numbers and resolutions for each \\nsubtomogram average are summarized in Extended Data Tables 4-6. Visualization and calculation of \\ncross-correlations (CC) between different maps and models, was performed with the UCSF ChimeraX \\npackage82.  \\n \\nCNN Pre- and post-processing \\nPreprocessing \\nTomograms are normalized to obtain uniform mean of 0 and variance of 1 in the frequency domain, prior \\nto training. Additionally, the spectrum equalization filter is applied by matching the amplitude spectrum of \\nthe tomogram that is to be transformed to the target spectrum of one manually selected high-contrast VPP \\ntomogram (Tomogram TS_001, Extended Data Fig. 1b). Extraction of spectra amplitudes is done using \\nfast Fourier transform (FFT) followed by radially averaging amplitudes across the frequency domain. If the \\ntarget tomogram’s Nyquist frequency is lower than the input tomogram’s, the target spectrum is padded \\nwith zeros to match the input spectrum’s size. Next, an equalization vector is created by dividing entry-\\nwise the target spectrum by the respective input spectrum. The resulting equalization vector is converted \\ninto a rotational kernel and multiplied by the input tomograms in the frequency domain in combination with \\na sigmoidal-shaped low-pass filter to eliminate high frequency noise. After back transformation, the \\ntomogram exhibits a similar contrast to the target tomogram (Extended Data Fig. 1). Tomogram intensities \\nare additionally normalized to obtain uniform mean and variance. For the 2D CNN, tomograms and training \\nsegmentations are processed slice-wise into 2D tiles with a fixed size of 288x288 pixels (256x256 pixels \\nand 16 pixels padding on each side). For the 3D CNN, tomograms are by default split into cubic patches of \\n64x64x64 voxels, and 12 voxels overlap in each dimension.  \\n \\nPost-processing (2D CNN) \\nThe post-processing steps for the 2D network are focused on assembling the per-slice prediction into a 3D \\nsegmentation. To that aim, predicted tile segmentations are cropped on each side by 48 pixels to reduce \\nartifacts around the edges, followed by reassembly into 3D stacks, with remaining overlapping areas being \\naveraged. A 1D Gaussian filter is successively applied along the z axis in order to reduce single-slice false \\npositives (Extended Data Fig. 2). \\n \\nPost-processing (3D CNN) \\nAfter reassembly of the individual 64x64x64 voxel patches into the probability map resulting from the 3D \\nCNN, the post processing step includes the clustering of the thresholded map (usually at threshold value of \\n0.5). According to user preferences, only clusters that lie within or close to a given mask consisting of an \\norganelle or cytosol segmentation outputted by the 2D network, and whose size lies within a given range \\nof numbers of voxels, are considered for the final prediction map (Supplementary Note 1). This ‘cleaning’ \\nprocedure allows adding a priori information on the size of the particles in question. Additionally, for \\n', metadata={'x0': 72.0, 'x1': 542.76904296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.76904296875, 'height': 638.5902099609375, 'nlines': 44, 'fs': 14.51341386274858, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='27 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='macromolecular complex localization, the user has the option to export a list of coordinates of cluster-\\ncentroids, which represent the particles’ locations prediction. \\n \\nPrecision-Recall and Performance Analysis \\nCross-validation approach \\nThe analysis of performance for RIBO and FAS localization, as well as membrane segmentation, were done \\nusing a 3-fold CV scheme, where 8 out of the 10 VPP tomograms were used for training, and the remaining \\n2 VPP tomograms were used for testing. In the test for domain generalization, the same three networks \\n(trained on the 8 VPP tomograms) were applied to the 10 DEF tomograms. \\n \\nThe analysis of performance in the NPC segmentation task was done in the dedicated DEF* of 127 \\ntomograms. Importantly, in order to apply the DeePiCt pipeline, we employed a predicted segmentation of \\nthe nuclear envelope as the region mask in the contact mode during post-processing of the NPC prediction \\n(Supplementary Note 1). The NPC prediction was studied under a 3-fold CV scheme, where each split \\nconsisted of roughly 42 tomograms. The nuclear envelope prediction was achieved through an \\nindependently trained 3D CNN on 18 tomograms belonging to the DEF* dataset for which a manual \\nannotation of the nuclear envelope was available, and which were uniformly split across the 3 CV subsets.  \\n \\nFor organelle and cytosol segmentation evaluation, model performance was evaluated on the voxel-level \\nfor the post-processed 2D CNN predictions produced by each CV fold’s individually trained model, using \\n5000 voxels picked randomly from each tomogram of the CV fold’s respective test set. Precision and recall \\n(Supplementary Note 1) were computed at threshold values varying from 0 to 1 on the picked voxels to \\ncompute the area under the precision-recall curves (AURPC) for each tomogram.  \\n \\nEvaluation metrics \\nIn the case of particles (RIBO and FAS), the localization evaluation was made directly from the predicted \\nlists containing their centroids’ voxel coordinates. In the case of DeePict, the lists of particles’ coordinates \\nare sorted according to a decreasing score (cluster size) value. For both RIBO and FAS, we defined a true \\npositive when a predicted coordinate matched a ground-truth particle coordinate within a tolerance radius \\nof 130 Å (Supplementary Note 1). The F1 score, which is calculated as the harmonic mean of the precision \\nand the recall, provides a measure of the quality of the predictions. We chose this measure for easier \\ncomparison to previous studies for particle localization methods in the field of cryo-ET26,27,29,83. Other \\nmetrics such as the area under the precision recall curve (AUPRC) generated by increasing length of \\nparticles’ list were disregarded as the ordering of the list (a function of cluster size) does not necessarily \\nreflect the method’s confidence on positive classification. \\n \\nThe evaluation of the segmentation task is of a totally different nature, as the shape of the object should be \\ntaken into account voxel-wise. For the 2D CNN predictions of organelles and cytosol, we used the AUPRC \\nscore with varying threshold over the network’s output map (Supplementary Note 1). For Deepict’s \\npredictions of NPC and membrane segmentation, we instead used the voxel-based F1 score \\n(Supplementary Note 1), to maintain hegemony with DeePiCt particle prediction evaluations. \\nHyperparameter tuning of DeePiCt and performance assessment \\nThe previously described cross-validation schemes and evaluation metrics were not only used for the final \\nassessment of methods’ performance (Fig. 3), but to study the effect of the different hyperparameters that \\n', metadata={'x0': 72.0, 'x1': 545.4000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 473.4000244140625, 'height': 637.6659545898438, 'nlines': 44, 'fs': 14.492408058860086, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='28 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='could be set by the user in the case of the 3D CNN (IF, D, ED, DD, and BN). The experiments’ starting \\npoint was always the default hyperparameter combination 𝐷 = 2, 𝐼𝐹 = 4, 𝐵𝑁 = 0, 𝐸𝐷 = 0, 𝐷𝐷 = 0 \\n(Supplementary Note 1). In all cases, we tested first for different values of D, while keeping the rest fixed. \\nThen, we fixed D=D*, where D* denotes the value for which best performance was achieved and repeated \\nthe same process for IF. We continued with each of the remaining parameters, BN, ED and DD. The results \\nare reported in Extended Data Fig. 6. They allowed us to define the best hyperparameter combination for \\neach of the structures studied, which are summarized in Supplementary Table 1. \\n \\nIn each case we combined the 3D CNN segmentation of the target structure with an appropriate region \\nmask (Supplementary Note 1, Supplementary Table 1): in the RIBO and FAS localization task, we used \\na cytosol prediction from the 2D CNN in intersection mode to eliminate false positives in organelles and \\nthe extracellular space; for membrane segmentation we used again the predicted cytosol from the 2D CNN, \\nthis time in contact mode to eliminate clusters that are in the extracellular space or within organelles; for \\nNPCs we used a 3D CNN prediction of the nuclear envelope in contact mode to eliminate false NPC \\nsegmentations far from the nuclear boundary. For RIBO, FAS and membrane, both 3D CNN (for target \\nstructure prediction) and 2D CNN (for region mask prediction) in DeePiCt employed a 3-fold CV in the \\nVPP dataset. In each fold, 8 tomograms were used for training and model selection while the remaining 2 \\ntomograms were used for testing (Extended Data Fig. 6a-c). The NPC predictions, which we treat as \\nsegmentation tasks, were evaluated in the DEF* independent data selected for that purpose \\n(Supplementary Note 1 and Extended Data Fig. 6d-f). The region mask of predicted nuclear envelope \\nsegmentation was generated by a fixed 3D CNN trained on 18 of the 127 DEF* tomograms. These 18 \\ntomograms were evenly distributed among the 3 splits of the DEF* dataset used for the corresponding 3-\\nfold CV scheme employed for the hyperparameter tuning analysis of NPC segmentation by DeePiCt \\n(Extended Data Fig. 6d-f). We used a 3-fold CV where the total 127 tomograms were split into 3 random \\nsubsets with roughly the same number of tomograms each. Each fold consists of two such sets as training \\ndata and the remaining one for testing. Throughout our experiments, high-quality DEF* tomograms \\n(defined based on lamella thickness and tomographic alignment residual error as described above) \\nconsistently showed higher performance for the NPC segmentation task than the low-quality DEF* \\ntomograms, regardless of architectural variations (Extended Data Fig. 6d-f). This hints at the importance \\nof high data-quality in segmentation and localization tasks. \\n \\nFor the evaluation of the particle localization task, we compared the lists of coordinates of the particles in \\nour ground truth data with the predicted coordinate lists. For both RIBO and FAS, we considered a true \\npositive when coordinates coincide within a tolerance radius of 10 voxels (corresponding to approximately \\n130 Å). In this way, we computed the recall (the proportion of ground truth particles that were recovered \\nby the method) and the precision (the proportion of predicted particles that were true positives) and reported \\nthe F1 score, which is the harmonic mean of precision and recall (Supplementary Note 1, Supplementary \\nTable 1). For the structure segmentation task (such as the membrane or the NPC segmentation evaluation), \\nwe compared the ground truth masks with the DeePiCt predicted post-processed segmentation by \\ncalculating their voxel-wise precision and recall, and reporting the corresponding voxel-based F1 (voxel-\\nF1) score, also known as Sørensen-Dice coefficient (Supplementary Note 1).  \\n \\nAll experiments for the 3D CNN of the DeePiCt pipeline and DeepFinder were performed using NVIDIA \\n2080 Ti GPU, Cuda 10.0, Python 3 and Pytorch 1.3.1. For the 2D CNN, training was conducted using an \\n', metadata={'x0': 72.0, 'x1': 542.7711181640625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7711181640625, 'height': 637.9059448242188, 'nlines': 44, 'fs': 14.497862382368607, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='29 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='NVIDIA 2080 Ti GPU and an NVIDIA V100S GPU used for performance evaluation, using CUDA 10.0, \\nPython 3 and Keras 2.3.1 with a tensorflow 2.0.0 backend. Detailed lists of parameters used for the 2D and \\n3D CNN are available alongside DeePiCt’s source code (Code and data availability). \\n \\n \\n', metadata={'x0': 72.0, 'x1': 542.7667236328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 470.7667236328125, 'height': 55.90592956542969, 'nlines': 5, 'fs': 11.181185913085937, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='30 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. \\nMahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science \\n', metadata={'x0': 72.0, 'x1': 529.5599975585938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 457.55999755859375, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='351, 969–972 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 191.87998962402344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 98.03999328613281, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. \\nPfeffer, S. et al. Dissecting the molecular organization of the translocon-associated protein complex. \\n', metadata={'x0': 72.0, 'x1': 538.9215698242188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.92156982421875, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Nat. Commun. 8, 14516 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 236.51998901367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.67999267578125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. \\nAlbert, S. et al. Proteasomes tether to two distinct sites at the nuclear pore complex. Proc Natl Acad \\n', metadata={'x0': 72.0, 'x1': 536.760986328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 464.760986328125, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Sci USA 114, 13726–13731 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 253.0800018310547, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 159.24000549316406, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. \\nBäuerlein, F. J. B. et al. In situ architecture and cellular interactions of polyq inclusions. Cell 171, \\n', metadata={'x0': 72.0, 'x1': 527.6397705078125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 455.6397705078125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='179-187.e10 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 186.59999084472656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 92.75999450683594, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='5. \\nGuo, Q. et al. In Situ Structure of Neuronal C9orf72 Poly-GA Aggregates Reveals Proteasome \\n', metadata={'x0': 72.0, 'x1': 513.7218017578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.7218017578125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Recruitment. Cell 172, 696-705.e12 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 289.55999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 195.72000122070312, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. \\nAlbert, S. et al. Direct visualization of degradation microcompartments at the ER membrane. Proc \\n', metadata={'x0': 72.0, 'x1': 530.0406494140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.0406494140625, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Natl Acad Sci USA 117, 1069–1080 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 289.08001708984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 195.24002075195312, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='7. \\nWilfling, F. et al. A Selective Autophagy Pathway for Phase-Separated Endocytic Protein Deposits. \\n', metadata={'x0': 72.0, 'x1': 536.040771484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 464.040771484375, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Mol. Cell 80, 764-778.e7 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 242.0399932861328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 148.1999969482422, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='8. \\nWatanabe, R. et al. The In Situ Structure of Parkinson’s Disease-Linked LRRK2. Cell 182, 1508-\\n', metadata={'x0': 72.0, 'x1': 523.7564086914062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.75640869140625, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1518.e16 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 171.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 78.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='9. \\nBöhning, J. & Bharat, T. A. M. Towards high-throughput in situ structural biology using electron \\n', metadata={'x0': 72.0, 'x1': 525.005126953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.005126953125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cryotomography. Prog. Biophys. Mol. Biol. 160, 97–103 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 380.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 286.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='10. Weis, F. & Hagen, W. J. H. Combining high throughput and high quality for cryo-electron \\n', metadata={'x0': 72.0, 'x1': 494.76055908203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 422.76055908203125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='microscopy data collection. Acta Crystallogr. D Struct. Biol. 76, 724–728 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 455.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 361.8000183105469, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='11. Klumpe, S. et al. A modular platform for automated cryo-FIB workflows. eLife 10, (2021). \\n', metadata={'x0': 72.0, 'x1': 497.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 425.6400146484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='12. Zachs, T. et al. Fully automated, sequential focused ion beam milling for cryo-electron tomography. \\n', metadata={'x0': 72.0, 'x1': 537.720703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 465.720703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='eLife 9, (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 164.51998901367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 70.67999267578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='13. Pyle, E. & Zanetti, G. Current data processing strategies for cryo-electron tomography and \\n', metadata={'x0': 72.0, 'x1': 495.48193359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 423.48193359375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='subtomogram averaging. Biochem. J. 478, 1827–1845 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 369.2400207519531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 275.4000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='31 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='14. Lučič, V., Rigort, A. & Baumeister, W. Cryo-electron tomography: the challenge of doing structural \\n', metadata={'x0': 72.0, 'x1': 538.2041625976562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.20416259765625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='biology in situ. J. Cell Biol. 202, 407–419 (2013). \\n', metadata={'x0': 93.83999633789062, 'x1': 316.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 222.36001586914062, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='15. Wan, W. & Briggs, J. A. G. Cryo-Electron Tomography and Subtomogram Averaging. Meth. \\n', metadata={'x0': 72.0, 'x1': 507.9606018066406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 435.9606018066406, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Enzymol. 579, 329–367 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 235.3199920654297, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 141.47999572753906, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='16. Wu, X., Zeng, X., Zhu, Z., Gao, X. & Xu, M. Template-Based and Template-Free Approaches in \\n', metadata={'x0': 72.0, 'x1': 524.28173828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.28173828125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular Cryo-Electron Tomography Structural Pattern Mining. in Computational Biology (ed. Husi, \\n', metadata={'x0': 93.83999633789062, 'x1': 537.9610595703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.1210632324219, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='H.) (Codon Publications, 2019). doi:10.15586/computationalbiology.2019.ch11. \\n', metadata={'x0': 93.83999633789062, 'x1': 449.8800048828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 356.0400085449219, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='17. Bohm, J. et al. Toward detecting and identifying macromolecules in a cellular context: template \\n', metadata={'x0': 72.0, 'x1': 518.7681274414062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 446.76812744140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='matching applied to electron tomograms. Proc Natl Acad Sci USA 97, 14245–14250 (2000). \\n', metadata={'x0': 93.83999633789062, 'x1': 501.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 408.1200256347656, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='18. Bäuerlein, F. J. B. & Baumeister, W. Towards visual proteomics at high resolution. J. Mol. Biol. \\n', metadata={'x0': 72.0, 'x1': 520.9199829101562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.91998291015625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='433, 167187 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 186.36000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 92.52000427246094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='19. Xu, M., Beck, M. & Alber, F. Template-free detection of macromolecular complexes in cryo \\n', metadata={'x0': 72.0, 'x1': 505.3250732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.3250732421875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='electron tomograms. Bioinformatics 27, i69-76 (2011). \\n', metadata={'x0': 93.83999633789062, 'x1': 337.8000183105469, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 243.96002197265625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='20. Martinez-Sanchez, A. et al. Template-free detection and classification of membrane-bound \\n', metadata={'x0': 72.0, 'x1': 497.39996337890625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 425.39996337890625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='complexes in cryo-electron tomograms. Nat. Methods 17, 209–216 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 425.8800048828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 332.0400085449219, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='21. Krizhevsky, A., Sutskever, I. & Hinton, G. E. ImageNet classification with deep convolutional \\n', metadata={'x0': 72.0, 'x1': 513.4891967773438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.48919677734375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='neural networks. Commun. ACM 60, 84–90 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 322.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 228.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='22. LeCun, Y. Deep learning & convolutional networks. in 2015 IEEE Hot Chips 27 Symposium (HCS) \\n', metadata={'x0': 72.0, 'x1': 535.5599975585938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 463.55999755859375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1–95 (IEEE, 2015). doi:10.1109/HOTCHIPS.2015.7477328. \\n', metadata={'x0': 93.83999633789062, 'x1': 363.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 270.1200256347656, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='23. LeCun, Y., Bengio, Y. & Hinton, G. Deep learning. Nature 521, 436–444 (2015). \\n', metadata={'x0': 72.0, 'x1': 455.6400146484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 383.6400146484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='24. Chen, M. et al. Convolutional neural networks for automated annotation of cellular cryo-electron \\n', metadata={'x0': 72.0, 'x1': 523.8001708984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.8001708984375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomograms. Nat. Methods 14, 983–985 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 303.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 209.6400146484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='25. Ronneberger, O., Fischer, P. & Brox, T. U-Net: Convolutional Networks for Biomedical Image \\n', metadata={'x0': 72.0, 'x1': 516.6036987304688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.60369873046875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Navab, N., Hornegger, J., Wells, W. M. & Frangi, A. F.) vol. 9351 234–241 (Springer International \\n', metadata={'x0': 93.83999633789062, 'x1': 534.6023559570312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 440.7623596191406, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Publishing, 2015). \\n', metadata={'x0': 93.83999633789062, 'x1': 177.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 83.63999938964844, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='32 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='26. Gubbins, I. et al. Classification in Cryo-Electron Tomograms. \\n', metadata={'x0': 72.0, 'x1': 369.4820861816406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 297.4820861816406, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='https://diglib.eg.org/handle/10.2312/3dor20191061. \\n', metadata={'x0': 93.83999633789062, 'x1': 324.6000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 230.760009765625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='27. Moebel, E., Martinez, A., Larivière, D. & Ortiz, J. 3D ConvNet improves macromolecule \\n', metadata={'x0': 72.0, 'x1': 490.6821594238281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.6821594238281, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='localization in 3D cellular cryo-electron tomograms. (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 362.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.44000244140625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='28. Moebel, E. et al. Deep learning improves macromolecule identification in 3D cellular cryo-electron \\n', metadata={'x0': 72.0, 'x1': 534.840576171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 462.840576171875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomograms. Nat. Methods 18, 1386–1394 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 314.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 220.68002319335938, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='29. Gubins, I. et al. SHREC 2021: Classification in Cryo-electron Tomograms. The Eurographics \\n', metadata={'x0': 72.0, 'x1': 510.6011657714844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.6011657714844, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Association (2021) doi:10.2312/3dor.20211307. \\n', metadata={'x0': 93.83999633789062, 'x1': 308.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 214.44000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='30. Srivastava, N., Hinton, G., Krizhevsky, A., Sutskever, I. & Salakhutdinov, R. Dropout: A Simple \\n', metadata={'x0': 72.0, 'x1': 524.2863159179688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.28631591796875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Way to Prevent Neural Networks from Overfitting. J Mach Learn Res 15, (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 456.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 362.2799987792969, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='31. Ioffe, S. Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate \\n', metadata={'x0': 72.0, 'x1': 538.2088623046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.2088623046875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Shift. in vol. 37 (Association for Computing Machinery, 2015). \\n', metadata={'x0': 93.83999633789062, 'x1': 375.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 281.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='32. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Nat. \\n', metadata={'x0': 72.0, 'x1': 533.3994140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.3994140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Methods 16, 1146–1152 (2019). \\n', metadata={'x0': 93.83999633789062, 'x1': 237.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 144.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='33. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement \\n', metadata={'x0': 72.0, 'x1': 540.8409423828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 468.8409423828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='with M visualizes ribosome-antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 533.1600341796875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 439.3200378417969, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='34. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in \\n', metadata={'x0': 72.0, 'x1': 511.0824890136719, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 439.0824890136719, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='RELION-3. eLife 7, (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 219.47999572753906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 125.63999938964844, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='35. Castaño-Díez, D. The Dynamo package for tomography and subtomogram averaging: components \\n', metadata={'x0': 72.0, 'x1': 530.2884521484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.2884521484375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='for MATLAB, GPU computing and EC2 Amazon Web Services. Acta Crystallogr. D Struct. Biol. \\n', metadata={'x0': 93.83999633789062, 'x1': 528.8400268554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 435.0000305175781, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='73, 478–487 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 186.36000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 92.52000427246094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='36. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron \\n', metadata={'x0': 72.0, 'x1': 536.5205688476562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 464.52056884765625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomograms and subtomogram analysis. J. Struct. Biol. 178, 177–188 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 432.8399963378906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 339.0, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='37. Chen, M. et al. A complete data processing workflow for cryo-ET and subtomogram averaging. Nat. \\n', metadata={'x0': 72.0, 'x1': 539.880615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 467.880615234375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Methods 16, 1161–1168 (2019). \\n', metadata={'x0': 93.83999633789062, 'x1': 237.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 144.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='38. Danev, R., Buijsse, B., Khoshouei, M., Plitzko, J. M. & Baumeister, W. Volta potential phase plate \\n', metadata={'x0': 72.0, 'x1': 533.4031372070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.40313720703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='33 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='for in-focus phase contrast transmission electron microscopy. Proc Natl Acad Sci USA 111, 15635–\\n', metadata={'x0': 93.83999633789062, 'x1': 532.3200073242188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.4800109863281, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='15640 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 159.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 65.16000366210938, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='39. Danev, R., Tegunov, D. & Baumeister, W. Using the Volta phase plate with defocus for cryo-EM \\n', metadata={'x0': 72.0, 'x1': 526.2044067382812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20440673828125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='40. Zimmerli, C. E. et al. Nuclear pores dilate and constrict in cellulo. Science 374, eabd9776 (2021). \\n', metadata={'x0': 72.0, 'x1': 526.4400024414062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.44000244140625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='41. Carpy, A. et al. Absolute proteome and phosphoproteome dynamics during the cell cycle of \\n', metadata={'x0': 72.0, 'x1': 500.2835998535156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 428.2835998535156, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='42. Marguerat, S. et al. Quantitative analysis of fission yeast transcriptomes and proteomes in \\n', metadata={'x0': 72.0, 'x1': 492.12591552734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 420.12591552734375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='proliferating and quiescent cells. Cell 151, 671–683 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 359.4000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.5600280761719, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='43. Turoňová, B. et al. Benchmarking tomographic acquisition schemes for high-resolution structural \\n', metadata={'x0': 72.0, 'x1': 526.4415283203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.4415283203125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='biology. Nat. Commun. 11, 876 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 270.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 176.27999877929688, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='44. Leibundgut, M., Jenni, S., Frick, C. & Ban, N. Structural basis for substrate delivery by acyl carrier \\n', metadata={'x0': 72.0, 'x1': 533.8854370117188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.88543701171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='protein in the yeast fatty acid synthase. Science 316, 288–290 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 402.3600158691406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 308.52001953125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='45. Snowden, J. S. et al. Structural insight into Pichia pastoris fatty acid synthase. Sci. Rep. 11, 9773 \\n', metadata={'x0': 72.0, 'x1': 522.6004638671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.6004638671875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='46. Gipson, P. et al. Direct structural insight into the substrate-shuttling mechanism of yeast fatty acid \\n', metadata={'x0': 72.0, 'x1': 528.8413696289062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 456.84136962890625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='synthase by electron cryomicroscopy. Proc Natl Acad Sci USA 107, 9164–9169 (2010). \\n', metadata={'x0': 93.83999633789062, 'x1': 482.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 388.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='47. Singh, K. et al. Discovery of a regulatory subunit of the yeast fatty acid synthase. Cell 180, 1130-\\n', metadata={'x0': 72.0, 'x1': 523.5164184570312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.51641845703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1143.e20 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 171.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 78.1199951171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='48. Kastritis, P. L. et al. Capturing protein communities by structural proteomics in a thermophilic \\n', metadata={'x0': 72.0, 'x1': 513.0034790039062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.00347900390625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='eukaryote. Mol. Syst. Biol. 13, 936 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 283.55999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 189.72000122070312, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='49. Jenni, S. et al. Structure of fungal fatty acid synthase and implications for iterative substrate \\n', metadata={'x0': 72.0, 'x1': 501.48443603515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 429.48443603515625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='shuttling. Science 316, 254–261 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 272.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 178.44000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='50. Ranjan, N. et al. Yeast translation elongation factor eEF3 promotes late stages of tRNA \\n', metadata={'x0': 72.0, 'x1': 482.0445861816406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 410.0445861816406, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='translocation. EMBO J. 40, e106449 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 291.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 197.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='51. Andersen, C. B. F. et al. Structure of eEF3 and the mechanism of transfer RNA release from the E-\\n', metadata={'x0': 72.0, 'x1': 530.7163696289062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.71636962890625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='34 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='site. Nature 443, 663–668 (2006). \\n', metadata={'x0': 93.83999633789062, 'x1': 245.87998962402344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 152.0399932861328, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='52. Becker, T. et al. Structure of monomeric yeast and mammalian Sec61 complexes interacting with the \\n', metadata={'x0': 72.0, 'x1': 541.56298828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 469.56298828125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='53. Armache, J.-P. et al. Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome \\n', metadata={'x0': 72.0, 'x1': 532.6869506835938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 460.68695068359375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='at 5.5-A resolution. Proc Natl Acad Sci USA 107, 19748–19753 (2010). \\n', metadata={'x0': 93.83999633789062, 'x1': 412.44000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 318.6000061035156, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='54. Shankar, V. et al. rRNA expansion segment 27Lb modulates the factor recruitment capacity of the \\n', metadata={'x0': 72.0, 'x1': 529.5655517578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 457.5655517578125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='yeast ribosome and shapes the proteome. Nucleic Acids Res. 48, 3244–3256 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 465.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 371.6400146484375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='55. Fujii, K., Susanto, T. T., Saurabh, S. & Barna, M. Decoding the function of expansion segments in \\n', metadata={'x0': 72.0, 'x1': 531.0042114257812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 459.00421142578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='ribosomes. Mol. Cell 72, 1013-1020.e6 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 303.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 209.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='56. Greber, B. J. et al. Insertion of the Biogenesis Factor Rei1 Probes the Ribosomal Tunnel during 60S \\n', metadata={'x0': 72.0, 'x1': 537.2453002929688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 465.24530029296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Maturation. Cell 164, 91–102 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 261.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 168.12002563476562, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='57. Wild, K. et al. MetAP-like Ebp1 occupies the human ribosomal tunnel exit and recruits flexible \\n', metadata={'x0': 72.0, 'x1': 517.323486328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 445.323486328125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='rRNA expansion segments. Nat. Commun. 11, 776 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 354.6000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 260.760009765625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='58. Kowalinski, E. et al. The crystal structure of Ebp1 reveals a methionine aminopeptidase fold as \\n', metadata={'x0': 72.0, 'x1': 516.1268920898438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.12689208984375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='binding platform for multiple interactions. FEBS Lett. 581, 4450–4454 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 442.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 348.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='59. Gold, V. A., Chroscicki, P., Bragoszewski, P. & Chacinska, A. Visualization of cytosolic ribosomes \\n', metadata={'x0': 72.0, 'x1': 537.2483520507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 465.24835205078125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='on the surface of mitochondria by electron cryo-tomography. EMBO Rep. 18, 1786–1800 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 524.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 430.9200134277344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='60. Avendaño-Monsalve, M. C., Ponce-Rojas, J. C. & Funes, S. From cytosol to mitochondria: the \\n', metadata={'x0': 72.0, 'x1': 513.9610595703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 441.9610595703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='beginning of a protein journey. Biol. Chem. 401, 645–661 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 386.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 292.44000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='61. Tucker, K. & Park, E. Cryo-EM structure of the mitochondrial protein-import channel TOM \\n', metadata={'x0': 72.0, 'x1': 502.9219970703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 430.9219970703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='complex at near-atomic resolution. Nat. Struct. Mol. Biol. 26, 1158–1166 (2019). \\n', metadata={'x0': 93.83999633789062, 'x1': 453.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 359.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='62. Lesnik, C., Cohen, Y., Atir-Lande, A., Schuldiner, M. & Arava, Y. OM14 is a mitochondrial \\n', metadata={'x0': 72.0, 'x1': 505.087158203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.087158203125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='receptor for cytosolic ribosomes that supports co-translational import into mitochondria. Nat. \\n', metadata={'x0': 93.83999633789062, 'x1': 506.5205383300781, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 412.6805419921875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Commun. 5, 5711 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 209.63999938964844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 115.80000305175781, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='63. George, R., Walsh, P., Beddoe, T. & Lithgow, T. The nascent polypeptide-associated complex \\n', metadata={'x0': 72.0, 'x1': 512.760986328125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 440.760986328125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(NAC) promotes interaction of ribosomes with the mitochondrial surface in vivo. FEBS Lett. 516, \\n', metadata={'x0': 93.83999633789062, 'x1': 527.4002685546875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.5602722167969, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='35 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='213–216 (2002). \\n', metadata={'x0': 93.83999633789062, 'x1': 169.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='64. Stalling, D., Westerhoff, M. & Hege, H.-C. amira: A Highly Interactive System for Visual Data \\n', metadata={'x0': 72.0, 'x1': 517.5670776367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 445.56707763671875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Analysis. in Visualization Handbook 749–767 (Elsevier, 2005). doi:10.1016/B978-012387582-\\n', metadata={'x0': 93.83999633789062, 'x1': 512.7164306640625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.8764343261719, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2/50040-X. \\n', metadata={'x0': 93.83999633789062, 'x1': 147.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 53.160003662109375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='65. Vignaud, T. et al. Stress fibres are embedded in a contractile cortical network. Nat. Mater. 20, 410–\\n', metadata={'x0': 72.0, 'x1': 533.0400390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 461.0400390625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='420 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 147.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 54.1199951171875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='66. Bharat, T. A. M. & Scheres, S. H. W. Resolving macromolecular structures from electron cryo-\\n', metadata={'x0': 72.0, 'x1': 514.3964233398438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 442.39642333984375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016). \\n', metadata={'x0': 93.83999633789062, 'x1': 524.280029296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 430.4400329589844, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='67. Wagner, F. R. et al. Preparing samples from whole cells using focused-ion-beam milling for cryo-\\n', metadata={'x0': 72.0, 'x1': 525.9163818359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9163818359375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='electron tomography. Nat. Protoc. 15, 2041–2070 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 351.2400207519531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 257.4000244140625, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='68. Schaffer, M. et al. Optimized cryo-focused ion beam sample preparation aimed at in situ structural \\n', metadata={'x0': 72.0, 'x1': 530.5262451171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 458.5262451171875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='studies of membrane proteins. J. Struct. Biol. 197, 73–82 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 382.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 288.3600158691406, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='69. Mastronarde, D. N. Advanced data acquisition from electron microscopes with serialem. Microsc. \\n', metadata={'x0': 72.0, 'x1': 527.8800048828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 455.8800048828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Microanal. 24, 864–865 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 238.4399871826172, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 144.59999084472656, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='70. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-\\n', metadata={'x0': 72.0, 'x1': 520.1563720703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.1563720703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 537.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 443.1600036621094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='71. Allegretti, M. et al. In-cell architecture of the nuclear pore and snapshots of its turnover. Nature 586, \\n', metadata={'x0': 72.0, 'x1': 540.3598022460938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 468.35980224609375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='796–800 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 169.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='73. Mastronarde, D. N. & Held, S. R. Automated tilt series alignment and tomographic reconstruction in \\n', metadata={'x0': 72.0, 'x1': 539.6473999023438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 467.64739990234375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IMOD. J. Struct. Biol. 197, 102–113 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 292.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 198.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='74. Tang, G. et al. EMAN2: an extensible image processing suite for electron microscopy. J. Struct. \\n', metadata={'x0': 72.0, 'x1': 520.200927734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.200927734375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Biol. 157, 38–46 (2007). \\n', metadata={'x0': 93.83999633789062, 'x1': 204.83999633789062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 111.0, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='75. Wu, J.-Q. & Pollard, T. D. Counting cytokinesis proteins globally and locally in fission yeast. \\n', metadata={'x0': 72.0, 'x1': 509.4070129394531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 437.4070129394531, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Science 310, 310–314 (2005). \\n', metadata={'x0': 93.83999633789062, 'x1': 228.36000061035156, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 134.52000427246094, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='76. Mitchison, J. M. The growth of single cells. Exp. Cell Res. 13, 244–262 (1957). \\n', metadata={'x0': 72.0, 'x1': 447.0, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 375.0, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='77. Nurse, P. Genetic control of cell size at cell division in yeast. Nature 256, 547–551 (1975). \\n', metadata={'x0': 72.0, 'x1': 496.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 424.6800231933594, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='36 \\n', metadata={'x0': 528.9600219726562, 'x1': 542.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 13.79998779296875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='78. Turonova. turonova/novaSTA: novaSTA. Zenodo (2020) doi:10.5281/zenodo.3973623. \\n', metadata={'x0': 72.0, 'x1': 482.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 410.0400085449219, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='79. Rigort, A. et al. Automated segmentation of electron tomograms for a quantitative description of \\n', metadata={'x0': 72.0, 'x1': 522.36474609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.36474609375, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='actin filament networks. J. Struct. Biol. 177, 135–144 (2012). \\n', metadata={'x0': 93.83999633789062, 'x1': 367.08001708984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 273.2400207519531, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='80. Redemann, S. et al. The segmentation of microtubules in electron tomograms using Amira. Methods \\n', metadata={'x0': 72.0, 'x1': 538.6806640625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 466.6806640625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Mol. Biol. 1136, 261–278 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 244.4399871826172, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 150.59999084472656, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='81. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. \\n', metadata={'x0': 72.0, 'x1': 535.3240966796875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 463.3240966796875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='J. Comput. Chem. 25, 1605–1612 (2004). \\n', metadata={'x0': 93.83999633789062, 'x1': 279.2400207519531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 185.4000244140625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='82. Goddard, T. D. et al. UCSF ChimeraX: Meeting modern challenges in visualization and analysis. \\n', metadata={'x0': 72.0, 'x1': 525.0051879882812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.00518798828125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Protein Sci. 27, 14–25 (2018). \\n', metadata={'x0': 93.83999633789062, 'x1': 230.0399932861328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 136.1999969482422, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='83. Moebel, E. et al. Deep Learning Improves Macromolecules Localization and Identification in 3D \\n', metadata={'x0': 72.0, 'x1': 524.5264892578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 452.5264892578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular Cryo-Electron Tomograms. BioRxiv (2020) doi:10.1101/2020.04.15.042747. \\n', metadata={'x0': 93.83999633789062, 'x1': 472.6800231933594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 378.84002685546875, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 1 | Pre-processing of input data by spectrum equalization. a. Input DEF tomogram with \\nlow signal-to-noise ratio. b. Target VPP tomogram with high signal-to-noise ratio. c. Output showing the \\ntomogram in a after spectrum matching. d. Amplitude spectra of target, input and output tomograms, as well as \\nper-frequency (1/Å) scaling factors of the rotational equalization kernel (y-axis cropped). \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9611206054688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96112060546875, 'height': 63.269989013671875, 'nlines': 6, 'fs': 10.544998168945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='  \\n', metadata={'x0': 72.0, 'x1': 78.41999816894531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 6.4199981689453125, 'height': 17.807342529296875, 'nlines': 1, 'fs': 17.807342529296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 2 | 2D CNN organelle and cytosol segmentations. Isosurface view of organelle ground \\ntruth annotations (a), model output prior to (b) and after 3D post-processing (c). d-f. Organelle (d) and cytosol \\n(e) predictions of the tomogram in Fig. 1 b. The 2D CNN’s segmentations (red) are overlaid with the manually \\ncreated ground truth (blue), with overlapping regions (i.e. true positives) in white. Ground truth annotations in the \\ntop and bottom left corners in e (yellow arrowheads) are artifacts resulting from cross-slice interpolation in the \\nmanual annotation process. f. 3D post-processing improves performance of the 2D CNN as exemplified in (a-c). \\n', metadata={'x0': 72.0, 'x1': 525.960205078125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.960205078125, 'height': 81.86553955078125, 'nlines': 7, 'fs': 11.695077078683036, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 3 | Details of the construction of ground truth for RIBO and FAS in VPP and DEF \\ntomograms. a-b. Contributions of the steps applied during the construction of RIBO ground truth in the VPP (a) \\nand DEF (b) datasets. Upper subplots show the absolute number of particles per tomogram (ppt): true positives \\nfrom the initial annotations that were recovered in step 2 (yellow), newly identified true positives in step 2 (blue), \\nparticles that were in the initial annotation but that were not recovered in step 2 (false negatives from initial \\nannotation; pink), unidentified particles from any of the two first steps, which were manually picked in step 3 \\n(fuchsia), and total false negatives (red) in step 2. Lower subplots show the associated relative contribution, as a \\npercentage of the ground truth per tomogram (ground truth panel) or as percentage of the predictions per tomogram \\nin step 2 (prediction panel). c-d. Equivalent plots for FAS ground truth construction, where the only difference is \\nthat the recovered true positives originate from an initial manual annotation (green), as opposed to RIBO where \\nwe used TM additionally. e-f. Summary of the relative contributions in the RIBO (e) and FAS (f) ground truth \\nacross the 10 VPP and 10 DEF tomograms. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9671020507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96710205078125, 'height': 162.81048583984375, 'nlines': 14, 'fs': 11.629320417131696, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 4 | S. pombe ribosome and FAS complex abundances in cryo-electron tomograms in \\ncomparison to proteomics analysis. In this study (cryo-ET, left), fully assembled ribosomes (a) and FAS (b) \\nwere iteratively annotated in 20 ground truth cryo-electron tomograms and cytosolic concentrations calculated \\nusing the cytosol segmentations. Copy numbers per cell were calculated for an S. pombe cell with the assumption \\nof 30 % cytosolic volume1 of a total of 150 µm3 average cell volume2,3. a. With cryo-ET, an average of 671,303 \\n土 96,708 ribosomes/cell were annotated. Ribosome counts from individual tomograms are represented as \\nindividual points (red). Proteomics data of individual ribosomal proteins were derived from Marguerat et al.4 \\n(yellow) and Carpy et al.5 (purple) and resulted in average of 97,795 土 99,298 and 343,511 土 244,226 \\nribosomes/cell, respectively. b. Each measurement displayed in the plot corresponds to 6 times fully assembled \\nFAS counts per tomogram (the complex is constituted by six alpha and beta subunits). With cryo-ET, an average \\nof 106,282 土 86,247 FAS subunits/cell were observed. Proteomics data of individual FAS proteins (subunit alpha \\nin purple, beta in yellow) were derived from Marguerat et al.4 and Carpy et al.5 and resulted in average of 67,035 \\n土 12,600 and 202,263 土 47,086 FAS subunits/cell, respectively. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9640502929688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96405029296875, 'height': 204.4659423828125, 'nlines': 17, 'fs': 12.027408375459558, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 5 | Effects of spectrum equalization on cross-validation and performance evaluation \\nacross domains of the 2D CNN. a-f. Performance was evaluated for 2D networks in three different scenarios \\n(training on VPP in cross-validation, cross-domain generalization from VPP to DEF, training on VPP and DEF in \\ncross validation), for either organelles or cytoplasm, without (-EQ) and with (+EQ) spectrum equalization. Areas \\nunder the precision-recall curves (AUPRCs) were computed by randomly picking 1000 voxels from each \\ntomogram in test set (for the two cross-validation scenarios) or from the DEF tomograms (for the cross-domain \\ntasks), while restricting evaluation to z-slices with any positive label. AUPRCs were computed after 3D post-\\nprocessing. The baselines are defined as the fraction of positive labels within those z-slices and averaged across \\ntest tomograms. g-h. Precision, recall and F1-score of organelle (g) and cytosol (h) segmentation depending on \\nprobability threshold used. Scores were computed on the cross-validation results for the spectrum-equalized VPP \\ntomograms. \\n', metadata={'x0': 72.0, 'x1': 525.9631958007812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96319580078125, 'height': 138.73834228515625, 'nlines': 12, 'fs': 11.561528523763021, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 6 | Hyperparameter tuning and performance analysis of 3D CNN for RIBO, FAS, \\nmembranes and NPCs. a-c. CV analysis for the performance of RIBO, FAS and membranes, respectively. In \\nthese experiments we follow an incremental search: from left to right, the starting subplot corresponds to the \\ndefault configuration of the 3D CNN, D=2, BN=0, and (ED, DD)=(0,0), except for variable IF. The best value of \\nIF (denoted by IF*) is fixed in the subsequent subplot to the right, where D is varied over 1, 2, and 3. The next \\nsubplot to the right compares the effect of not applying and applying the batch normalization layer (BN=0 and \\nBN=1, respectively), under the best IF=IF* and D=D* from previous subplots. Finally, the right most subplot \\nvaries the encoder- and decoder-dropout parameters (ED, DD), with fixed best IF=IF*, D=D*, and BN=BN* from \\nprevious subplots. d-f. Results of hyperparameter exploration for NPC; in all DEF* tomograms (left), in the subset \\nof high-quality DEF* tomograms (middle), and in the subset of lower quality DEF* tomograms (right). d. Shows \\nthe variations in IF, when D=2, BN=0, and (ED, DD) = (0,0); e. Variations in IF, when D=3, BN=0, and (ED, \\nDD) = (0,0); f. Variations in IF, when D=3, BN=1, and (ED, DD) = (0,0). The summary of best hyperparameters’ \\ncombination is provided in Supplementary Table 1. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9760131835938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.97601318359375, 'height': 160.84197998046875, 'nlines': 15, 'fs': 10.722798665364584, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 7 | Effects of spectrum equalization and cytosol masking on DeePiCt performance. a-c \\nPerformance of the DeePiCt pipeline within the VPP domain for RIBO, FAS and membranes, without (-EQ) and \\nwith (+EQ) spectrum equalization pre-processing. The F1 score distribution shows that although for RIBO and \\nmembranes the spectrum equalization does not have a clear positive effect, for FAS it brings better results with a \\nshift in median F1 score from 0.25 to 0.46. d-f. The same networks were used to test performance of the DeePiCt \\npipeline across the VPP and DEF domains (training on VPP and testing in DEF) for RIBO, FAS and membranes \\nwithout (-EQ) and with (+EQ) spectrum equalization pre-processing. The F1 score distribution shows that while \\nfor ribosomes the spectrum equalization does not bring benefits, for FAS and membrane prediction across \\ndomains, its use brings a positive median F1 shift from 0.03 to 0.15 and from 0.16 to 0.41, respectively. g-h. \\nEffects of employing a segmentation of cytosol as a region mask (+ mask) or not (- mask) during DeePiCt’s post-\\nprocessing, for the localization of RIBO and FAS. The plots show the results under a 3-fold CV in VPP data, \\nwhere the cytosol segmentation was obtained by a 2D CNN. g. RIBO localization does not show a significant \\nimprovement in F1 score when the cytosol masking is used. h. FAS results show a stronger difference in F1 score, \\nwith a shift of median F1 from 0.10 to 0.46. \\n', metadata={'x0': 72.0, 'x1': 525.9652099609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9652099609375, 'height': 173.29833984375, 'nlines': 15, 'fs': 11.55322265625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 8 | Ground truth annotations and 3D refinement of ribosomes and FAS in DEF \\ntomograms. a. Complete annotation of ribosomes (yellow, orange, green, dark and light blue; sub-populations \\ndescribed in (f) and in Extended Data Figure 9), FAS (pink), membranes (purple) and organelles (gray) in the \\ntomogram depicted in Fig. 4. Ribosome subclasses in the context of specific organelles were identified by a \\nmaximal distance of 25 nm to mitochondria (mito-bound, green) or ER (ER-bound, orange). Focused classification \\non densities close to the head of the 40S small ribosomal subunit or in proximity to the exit tunnel recovered \\nribosome subsets with head (dark blue) and exit tunnel density (cyan). b-e. FAS subtomogram averages (pink) fit \\nof the S. cerevisiae X-ray structure (b, cyan) and the single particle cryo-EM density of P. pastoris (c, cyan) \\nincluding the PPT domains. An extra density cannot be assigned. The cross section views close to the alpha-wheel \\nreveal three densities fitting three and one ACPs of S. cerevisiae and P. pastoris, respectively (white asterisks, \\nbottom). d. Overlay with FAS from VPP ground truth (gt) tomograms (gray) shows overall similarity, but the \\nACPs could not be resolved (white asterisks, bottom). e. FAS predicted by DeePiCt (gray) matches the DEF gt \\naverage with three ACP densities close to the alpha-wheel. f-h. Subtomogram average of all ribosomes (yellow) \\nmatches the DeePiCt- predicted (g, gray) and VPP-derived (h, gray) ribosome densities. i-k. Well-aligned \\nribosomes (yellow) detected by hierarchical 3D classification in RELION and refined in M (Supplementary Figure \\n3). j-k. Slicing through the average depicted in (i) at different axes reveals the PTC with a P-site tRNA and the \\nL1 stalk facing the E-site. \\n', metadata={'x0': 72.0, 'x1': 525.963134765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.963134765625, 'height': 206.73049926757812, 'nlines': 18, 'fs': 11.485027737087673, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Fig. 9 | Comparison of ribosome subtomogram averages in DEF ground truth (gt). a-e. \\nSubtomogram average of ribosomes (blue) with electron density in proximity to the head of the 40S small subunit \\n(white arrowhead) which fits the ribosome-bound eEF36 (CC 0.8972, red). b. Cross section of (a) showing the \\nPTC with P- and E-site tRNAs and the L1 stalk facing inwards. c. Zoom into the additional head density fitting \\neEF3. d-e. Overlay with the same ribosome class from DeePiCt predictions in DEF (d, gray) and from VPP gt \\ndata (e, gray). f. Ribosome subset (cyan) with electron density in proximity to the exit tunnel (white arrowhead). \\ng. Overlay with the S. cerevisiae ribosome7 (CC 0.9651, purple) and a specific conformation of the expansion \\nsegment ES27L8 (CC 0.7806, purple). h. Zoom into the exit tunnel area in (g) highlights the additional density \\nconnected to ES27L that colocalizes with for example the human Ebp19 (gray, lower panel, CC 0.8495, EMD-\\n1068). i-j. Overlay with the same ribosome class from DeePiCt predictions in DEF (i, gray) and from VPP gt data \\n(j, gray). k-n. Overlay of ribosomes within 25 nm distance to the ER (orange) with ER-bound ribosomes from \\nDeePiCt predictions in DEF (k, gray), from VPP gt data (l, gray), with the published density of S. cerevisiae \\nribosomes derived from extracted ER10 (k, gray, CC 0.8972, EMD-3764), and the ER-bound HeLa ribosome11 (n, \\ngray, CC 0.9105, EMD-8056). All densities show the same interface between ribosome and ER membrane. o-p. \\nOverlay of ribosomes within 25 nm distance to mitochondria (green) with mito-bound ribosomes from DeePiCt \\npredictions in DEF (o, gray), and VPP gt data (p, gray). All densities show the same interface between ribosome \\nand mitochondria membrane. q-r. Comparisons between ER-bound (orange) and mitochondria-bound (mito-\\nbound, green) subtomogram averages in DEF gt (q) and VPP gt (r) data reveal an angular offset of the membrane \\ninterfaces of 35° and 33°, respectively. \\n', metadata={'x0': 72.0, 'x1': 525.9635620117188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96356201171875, 'height': 218.0428466796875, 'nlines': 19, 'fs': 11.475939298930921, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Fig. 10 | Domain generalization of different cellular structures and species. a. Performance \\nresults of five DeePiCt networks trained on a variety of cell types for the segmentation of different cellular \\nstructures used for predictions in a HeLa cell dataset11. Left: plot of performance scores when employing the \\nspectrum equalization (+EQ) filter in both the training and on the HeLa cell data. The results correspond to \\nsegmentations of actin filaments and microtubules (MT) (trained on 4 RPE1 tomograms; pink), RIBO (trained on \\n10 VPP S. pombe tomograms; yellow-green), membranes (trained on 10 S. pombe VPP tomograms; aquamarine), \\nMT (trained on 11 C. elegans tomograms; blue) and actin filaments (trained on 2 RPE1 and 3 MEF 3T3 tomograms \\nwith actin filaments at different orientations; light purple). Right: performance for RIBO and membrane without \\nemploying the equalization filter (-EQ) in the HeLa cell dataset. b. RPE1 training data examples (left and middle, \\ntomograms on top, ground truth annotations at the bottom) and predictions (right) of the actin filament and MT \\nmodels in the HeLa cell dataset. The training examples show 2 of the RPE1 tomograms12 used for this model. The \\nperformance of actin and microtubule predictions in the HeLa cell dataset is poor (voxel-F1=0.09 for actin and \\n0.26 for MT; a left) despite the high-quality training data (left). This is likely due to the high unidirectional \\norientation of the filaments in the training data (bundles in stress fibers).  \\n', metadata={'x0': 72.0, 'x1': 525.9666748046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9666748046875, 'height': 172.20281982421875, 'nlines': 15, 'fs': 11.48018798828125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. \\nWu, J.-Q. & Pollard, T. D. Counting cytokinesis proteins globally and locally in fission yeast. Science 310, \\n', metadata={'x0': 72.0, 'x1': 522.8399658203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.8399658203125, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='310–314 (2005). \\n', metadata={'x0': 91.91999816894531, 'x1': 161.16000366210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 69.24000549316406, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. \\nMitchison, J. M. The growth of single cells. Exp. Cell Res. 13, 244–262 (1957). \\n', metadata={'x0': 72.0, 'x1': 412.91998291015625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 340.91998291015625, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. \\nNurse, P. Genetic control of cell size at cell division in yeast. Nature 256, 547–551 (1975). \\n', metadata={'x0': 72.0, 'x1': 458.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 386.2799987792969, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. \\nMarguerat, S. et al. Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and \\n', metadata={'x0': 72.0, 'x1': 523.3216552734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 451.3216552734375, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='quiescent cells. Cell 151, 671–683 (2012). \\n', metadata={'x0': 91.91999816894531, 'x1': 263.8799743652344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 171.95997619628906, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='5. \\nCarpy, A. et al. Absolute proteome and phosphoproteome dynamics during the cell cycle of \\n', metadata={'x0': 72.0, 'x1': 461.64208984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 389.64208984375, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. \\nRanjan, N. et al. Yeast translation elongation factor eEF3 promotes late stages of tRNA translocation. \\n', metadata={'x0': 72.0, 'x1': 501.7206726074219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 429.7206726074219, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='EMBO J. 40, e106449 (2021). \\n', metadata={'x0': 91.91999816894531, 'x1': 215.16000366210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 123.24000549316406, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='7. \\nBecker, T. et al. Structure of monomeric yeast and mammalian Sec61 complexes interacting with the \\n', metadata={'x0': 72.0, 'x1': 499.0824890136719, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 427.0824890136719, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='8. \\nArmache, J.-P. et al. Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome at 5.5-\\n', metadata={'x0': 72.0, 'x1': 516.7166748046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.7166748046875, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='A resolution. Proc Natl Acad Sci USA 107, 19748–19753 (2010). \\n', metadata={'x0': 91.91999816894531, 'x1': 356.03997802734375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 264.11997985839844, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='9. \\nWild, K. et al. MetAP-like Ebp1 occupies the human ribosomal tunnel exit and recruits flexible rRNA \\n', metadata={'x0': 72.0, 'x1': 503.8804016113281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 431.8804016113281, 'height': 11.162811279296875, 'nlines': 2, 'fs': 5.5814056396484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='expansion segments. Nat. Commun. 11, 776 (2020). \\n', metadata={'x0': 91.91999816894531, 'x1': 301.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 209.87998962402344, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='10. Gold, V. A., Chroscicki, P., Bragoszewski, P. & Chacinska, A. Visualization of cytosolic ribosomes on the \\n', metadata={'x0': 72.0, 'x1': 522.1256103515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 450.1256103515625, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='surface of mitochondria by electron cryo-tomography. EMBO Rep. 18, 1786–1800 (2017). \\n', metadata={'x0': 91.91999816894531, 'x1': 456.5999755859375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 364.6799774169922, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='11. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science 351, \\n', metadata={'x0': 72.0, 'x1': 508.2000427246094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2000427246094, 'height': 11.162841796875, 'nlines': 1, 'fs': 11.162841796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='969–972 (2016). \\n', metadata={'x0': 91.91999816894531, 'x1': 161.16000366210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 69.24000549316406, 'height': 11.162841796875, 'nlines': 1, 'fs': 11.162841796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='12. Vignaud, T. et al. Stress fibres are embedded in a contractile cortical network. Nat. Mater. 20, 410–420 \\n', metadata={'x0': 72.0, 'x1': 508.44012451171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.44012451171875, 'height': 11.162841796875, 'nlines': 1, 'fs': 11.162841796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Information Guide \\n', metadata={'x0': 118.55999755859375, 'x1': 483.1200256347656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 364.5600280761719, 'height': 28.704376220703125, 'nlines': 1, 'fs': 28.704376220703125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nExtended Data Figures.docx \\n10 Figures supporting main text. \\n', metadata={'x0': 72.0, 'x1': 218.0399932861328, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 146.0399932861328, 'height': 41.26591491699219, 'nlines': 3, 'fs': 13.755304972330729, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Material.docx \\nContents:  \\n', metadata={'x0': 72.0, 'x1': 215.1599884033203, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 143.1599884033203, 'height': 26.865890502929688, 'nlines': 2, 'fs': 13.432945251464844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Supplementary Note 1: Architectural description of 2D and 3D CNNs, post processing steps \\n', metadata={'x0': 90.0, 'x1': 516.8466186523438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 426.84661865234375, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='and evaluation metrics. Includes Supplementary Figure 1. \\n', metadata={'x0': 108.0, 'x1': 364.9200134277344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 256.9200134277344, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Supplementary Tables:  \\n', metadata={'x0': 90.0, 'x1': 215.1599884033203, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 125.15998840332031, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='○ Supplementary Table 1: Selected hyperparameters of DeePiCt CNNs for different \\n', metadata={'x0': 126.0, 'x1': 506.76580810546875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 380.76580810546875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cellular features. \\n', metadata={'x0': 144.0, 'x1': 219.95999145507812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='○ Supplementary Table 2: Specifications of 3D-CNN training and prediction rounds \\n', metadata={'x0': 126.0, 'x1': 508.2021484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.2021484375, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='performed for FAS ground truth construction. \\n', metadata={'x0': 144.0, 'x1': 348.3600158691406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 204.36001586914062, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='○ Supplementary Table 3: Hyperparameter tuning experiment settings. \\n', metadata={'x0': 126.0, 'x1': 448.9200134277344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 322.9200134277344, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Supplementary Note 2: Considerations for structural analysis following CNN predictions \\n', metadata={'x0': 90.0, 'x1': 502.924072265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 412.924072265625, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='under different image acquisition conditions. Includes Supplementary Figures 2-9 detailing \\nsubtomogram hierarchical classification results. \\n', metadata={'x0': 108.0, 'x1': 512.2811889648438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 404.28118896484375, 'height': 26.865936279296875, 'nlines': 2, 'fs': 13.432968139648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Extended Data Tables.xlsx \\nContents: Excel file with the following 6 tables as separate sheets: \\n', metadata={'x0': 72.0, 'x1': 366.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 294.1199951171875, 'height': 26.625946044921875, 'nlines': 2, 'fs': 13.312973022460938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='● Table 1: Organelle content per tomogram in DEF and VPP datasets. \\n● Table 2: RIBO gt construction. \\n● Table 3: FAS gt construction. \\n● Table 4: VPP gt data acquisition, refinement statistics. \\n● Table 5: DEF gt data acquisition, refinement statistics. \\n● Table 6: Refinement statistics for subtomograms from DeePiCt in DEF data. \\n', metadata={'x0': 90.0, 'x1': 447.7200012207031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 357.7200012207031, 'height': 85.10226440429688, 'nlines': 6, 'fs': 14.183710734049479, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Note 1  \\n', metadata={'x0': 230.16000366210938, 'x1': 372.6600036621094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.5, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Architectural description of 2D and 3D CNN, post processing steps and \\n', metadata={'x0': 82.55999755859375, 'x1': 516.5902099609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 434.03021240234375, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='evaluation metrics \\n', metadata={'x0': 242.39999389648438, 'x1': 356.5799865722656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 114.17999267578125, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The general architecture of the 3D and 2D CNN is based also on the U-Net architecture1 and consists \\nof two consecutive symmetric paths: an encoder and a decoder path connected by skip-connections and \\na transitional base-layer (Fig. 1a), where:  \\n \\n(a) The encoder path is a sequence of alternated convolutional blocks and max-pooling layers. In turn, \\nthe convolutional blocks are a series of convolutions (kernel size 𝑘\" = \"3, stride 𝑠 = 1 and zero padding \\n𝑝\" = \"2) followed by a rectified linear unit (ReLU). In the case of the 3D CNN, optional batch \\nnormalization (BN) and dropout layers can be alternated (Fig. 1a). The max-pooling layer has fixed \\nwindow size 𝑤 = 2 and stride 𝑠 = 2. \\nIn our implementations, the initial convolutional block has a variable number of initial filters (IF) in its \\nfirst convolutional layer, which must be set by the user. In all cases, subsequent convolutional blocks \\nduplicate the number of feature maps from the previous level. The number of downsampling layers plus \\n1, is referred to as the network’s depth (D), which is also a hyperparameter to be set by the user. It \\ndetermines the size of the receptive field (𝑅𝐹) of the network (𝑅𝐹\" = \"2 ∗ 𝐷). For the 2D CNN, the \\ndepth is a fixed value 𝐷 = 5. \\n \\n(b) The base of the CNNs is the transitional convolutional layer between the encoder and decoder paths.  \\n \\n(c) The decoder path is symmetric with respect to the encoder path, and is a sequence of alternated \\nconvolutional and upsampling (via transposed convolution) layers2, with kernel size 𝑘 = 3, stride 𝑠 =\\n1 and padding 𝑝 = 1. For the 3D CNN, analogous optional layers can be alternated between \\nconvolutional and upsampling blocks (BN and dropout). The encoder and decoder paths’ associated \\nparameters are denoted throughout the text as ED and DD, respectively. \\nEssential to the U-Net architecture are the skip-connections, which consist of concatenating the feature \\nmaps from the encoder to the decoder path at every level. The last activation layer is in our case a \\nsigmoid function, compatible with our choice of the Dice Loss function3. By definition, this is a multi-\\nlabel approach, where single voxels may belong to multiple semantic classes, and which frees the user \\nfrom defining additional weighting parameters to compensate for class imbalance necessary for other \\nloss function choices such as categorical cross-entropy. In our experience, the use of Dice Loss provided \\nbetter qualitative results in all cases. \\nAll models were trained using Adam Optimiser4. Application of these networks to unseen tomograms \\nrequires scaling of their pixel size to match (at least approximately) that of the training data. \\n \\nData Augmentation \\nFor the 2D CNN, tiles are randomly flipped and rotated in 90-degree increments during training to \\nimprove generalization.  \\n \\nAlthough it did not provide improved results in our hands (not shown), data augmentation for training \\nthe 3D CNN is available and optional. The number of times that each of the original training examples \\nis augmented is given by a parameter 𝑛01 that can be set by the user. Each augmentation includes 4 \\n', metadata={'x0': 72.0, 'x1': 529.0800170898438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 457.08001708984375, 'height': 585.4017333984375, 'nlines': 40, 'fs': 14.635043334960937, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='different types of random volumetric transformations (applied to the original image and the labels, \\nexcept in the case of noise), that we describe in what follows. Let A in 𝑅2×2×2 be a single channel \\ncubic image of side-length 𝑛 > 0. Then:  \\n \\n1. Additive Gaussian noise. Given a predefined modulation factor 𝜖\" > \"0 , the Additive Gaussian \\nnoise transformation 𝐺8 is defined by 𝐺8(𝐴)\"= \"𝐴\" + \"𝜖𝐵, where 𝐵>?@ ~ 𝑁(0, 1) are independent and \\nidentically distributed random variables (i.i.d.r.v.), and 𝑁(𝜇, 𝜎E) denotes a normally distributed \\nGaussian random variable with mean 𝜇 and variance 𝜎E. \\n \\n2. Salt-and-pepper noise. Given user-defined modulation factor 𝑎 > 0 and a probability rate 1 > 𝑝\" >\\n\"0 , the salt-and-pepper noise transformation is defined by 𝑆H,I(𝐴)\"= \"𝐴\" + \"𝑎 ∗ 𝐵, where 𝐵>?@ ~ \\n𝑈(𝑝, [−1, 1]) are i.i.d.r.v., and 𝑈(𝑝, 𝐼) denotes a uniformly distributed random variable in a real interval \\n𝐼. \\n \\n3. Elastic transformation. Given coordinate entries of A that are located at a coarse regular grid of \\nthickness 𝑛O > 1 are displaced by a random vector of i.i.d.r.v. uniform random variables. To achieve a \\nsmooth displacement vector field for all the points of the image 𝐴 (i.e. on the fine grid), we generate the \\nelastically transformed image as the interpolated vector field (via polynomial interpolation of order 𝑘 >\\n0, defined by the user). \\n \\n4. Random rotation. Given a probability of rotation 𝑝 ∈ (0, 1) and angle range 𝛼 ∈ (0, 180), the image \\nA is rotated with a probability 𝑝 with and angle in [−𝛼, 𝛼], with respect to the 𝑧 −axis.  \\n \\n3D CNN default hyperparameters \\nA default, starting point set of hyperparameters values for the 3D CNN for this study: 𝐷 = 2, 𝐼𝐹 = 4, \\n𝐵𝑁 = 0, 𝐸𝐷 = 0, 𝐷𝐷 = 0, and 𝑛01 = 0. We used them to later, through testing variations on them, \\nstudy the optimal combinations of hyperparameters in the different segmentation and localization tasks \\n(Supplementary Table 1). \\n \\n2D CNN post-processing  \\nPost-processing of the 2D CNN includes the reassembly of predicted tile segmentations, by cropping \\n48 px on each side to reduce artifacts around the edges. Areas of sets of tiles still overlapping are \\naveraged after cropping. In order to remove inter-slide discontinuities, a Gaussian filter with parameter \\n𝜎 > 0 (default 𝜎 = 5) is then applied along the z axis.  \\n \\n3D CNN post-processing  \\n', metadata={'x0': 72.0, 'x1': 526.2054443359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.2054443359375, 'height': 533.7135620117188, 'nlines': 36, 'fs': 14.825376722547743, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Thresholding of the final probability map outputted by the last layer of the neural network \\n', metadata={'x0': 90.0, 'x1': 526.2054443359375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2054443359375, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(default threshold is the unbiased value 0.5, but is set by the user in the pipeline’s configuration \\nfile). \\n', metadata={'x0': 108.0, 'x1': 526.2009887695312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.20098876953125, 'height': 26.865966796875, 'nlines': 2, 'fs': 13.4329833984375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. Clustering of the resulting thresholded map via the connected component labeling label \\n', metadata={'x0': 90.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.20001220703125, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='algorithm, using the function morphology.label of the scikit-image python library5.  \\n', metadata={'x0': 108.0, 'x1': 477.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 369.96002197265625, 'height': 13.15020751953125, 'nlines': 1, 'fs': 13.15020751953125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Integration with other segmentation maps, e.g. with the output of the 2D CNN. Importantly, a \\n', metadata={'x0': 90.0, 'x1': 526.2005004882812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.20050048828125, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='subsequent selection of clusters is made with respect to a region_mask provided by the user, \\nwhich consists of an auxiliary binary image defining a tomographic region of interest (Fig 1b). \\nWe then allow 3 types of options (intersection, contact, or colocalization) for employing the \\nregion_mask through the corresponding contact_mode parameter, as follows: \\n', metadata={'x0': 108.0, 'x1': 526.2008666992188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.20086669921875, 'height': 55.906005859375, 'nlines': 4, 'fs': 13.97650146484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='a. contact_mode: \\nintersection \\n', metadata={'x0': 126.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 400.20001220703125, 'height': 12.38226318359375, 'nlines': 2, 'fs': 6.191131591796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The region_mask is used to directly mask the output of point 2, by considering only the \\noverlapping region, e.g. including particles that are within an organelle’s mask. \\n', metadata={'x0': 144.0, 'x1': 526.2008666992188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.20086669921875, 'height': 26.625900268554688, 'nlines': 2, 'fs': 13.312950134277344, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='b. contact_mode: \\ncontact \\n', metadata={'x0': 126.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 400.20001220703125, 'height': 12.38226318359375, 'nlines': 2, 'fs': 6.191131591796875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Only clusters that have voxels in common with the region of interest are kept, e.g. \\nselecting NPC on the nuclear envelope. \\n', metadata={'x0': 144.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.20001220703125, 'height': 26.865890502929688, 'nlines': 2, 'fs': 13.432945251464844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='c. contact_mode: colocalization  \\n', metadata={'x0': 126.0, 'x1': 279.96002197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 153.96002197265625, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Only clusters whose centroid is located at a given distance (colocalization_radius) \\nfrom the region of interest are kept, e.g. particles within a distance to an organelle. \\n', metadata={'x0': 144.0, 'x1': 526.1995239257812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 382.19952392578125, 'height': 26.865890502929688, 'nlines': 2, 'fs': 13.432945251464844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='At this step, only clusters within the size range [𝑚, 𝑀] are kept, where 0 ≤ 𝑚\" < \"𝑀\" ≤ \"∞ are \\ndefined by the user in the configuration file. \\n \\n', metadata={'x0': 108.0, 'x1': 526.2000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2000732421875, 'height': 42.158233642578125, 'nlines': 3, 'fs': 14.052744547526041, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Fig. 1 | 3D CNN post-processing steps. The clustered 3D CNN output (yellow) and a \\nregion mask (white) are combined according to the three contact modes. Cluster centroids are represented \\nby the red dots.  \\n \\n', metadata={'x0': 108.0, 'x1': 525.962646484375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 417.962646484375, 'height': 51.9305419921875, 'nlines': 4, 'fs': 12.982635498046875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. If the calculate_motl parameter is set to True, a list of the clusters’ centroids resulting from \\n', metadata={'x0': 90.0, 'x1': 526.2010498046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2010498046875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='previous steps is saved as a 4-columns comma separated values (csv) file, where the first \\ncolumn is the score (cluster size), and the three remaining columns indicate the (𝑥, 𝑦, 𝑧) centroid \\nvoxel coordinates. The list is sorted according to decreasing-score. \\n', metadata={'x0': 108.0, 'x1': 526.2003784179688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.20037841796875, 'height': 41.50592041015625, 'nlines': 3, 'fs': 13.835306803385416, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSegmentation evaluation \\nGiven a 3D cellular segmentation prediction 𝐼H]^_ (resulting either from the 2D or the 3D CNN), its \\nquality can be voxel-wise evaluated, by comparison to a ground truth binary image 𝐼`a, according to \\ndifferent metrics: \\n  \\n', metadata={'x0': 72.0, 'x1': 526.20263671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20263671875, 'height': 88.30596923828125, 'nlines': 6, 'fs': 14.717661539713541, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Voxel-based precision (P) is defined by: \\n', metadata={'x0': 90.0, 'x1': 288.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 198.1199951171875, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑃\" =\\n', metadata={'x0': 283.44000244140625, 'x1': 304.16766357421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 20.7276611328125, 'height': 12.942893981933594, 'nlines': 1, 'fs': 12.942893981933594, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='|cdefg\"| , \\n', metadata={'x0': 312.9599914550781, 'x1': 350.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 37.800018310546875, 'height': 18.477554321289062, 'nlines': 1, 'fs': 18.477554321289062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='  \\nwhere the bars || indicate the number of entries different from 0. Thus, P is the proportion of \\nvoxels in the predicted segmentation that truly belong to the wanted structure. \\n', metadata={'x0': 108.0, 'x1': 526.1981201171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.1981201171875, 'height': 41.74595642089844, 'nlines': 3, 'fs': 13.915318806966146, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. Voxel-based recall (R) is defined by: \\n', metadata={'x0': 90.0, 'x1': 273.4800109863281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 183.48001098632812, 'height': 12.38226318359375, 'nlines': 1, 'fs': 12.38226318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑅\" =\\n', metadata={'x0': 283.20001220703125, 'x1': 304.16766357421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 20.9676513671875, 'height': 12.942886352539062, 'nlines': 1, 'fs': 12.942886352539062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='\"|chi|\\n, \\n', metadata={'x0': 316.55999755859375, 'x1': 350.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 34.20001220703125, 'height': 18.477554321289062, 'nlines': 2, 'fs': 9.238777160644531, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='thus indicating the proportion of voxels of the ground truth segmentation that were correctly \\nrecovered by the prediction. \\n', metadata={'x0': 108.0, 'x1': 516.6055297851562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 408.60552978515625, 'height': 26.865951538085938, 'nlines': 2, 'fs': 13.432975769042969, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Area \\nunder \\nthe \\nprecision-recall \\ncurve \\n(AUPRC).  \\n', metadata={'x0': 90.0, 'x1': 528.8400268554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.84002685546875, 'height': 12.38226318359375, 'nlines': 6, 'fs': 2.0637105305989585, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Since 𝐼H]^_ is dependent on the threshold value 0 ≤ 𝑡ℎ𝑟 ≤ \"1 applied to the probability map, \\nboth \\nprecision \\nand \\nrecall \\ndepend \\nin \\nii \\nas \\nwell, \\n𝑃 = 𝑃no] \\nand \\n𝑅 = 𝑅no].  \\n', metadata={'x0': 108.0, 'x1': 528.8400268554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 420.84002685546875, 'height': 30.489715576171875, 'nlines': 13, 'fs': 2.345362736628606, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='The area under the precision-recall curve (AUPRC) is the area under the parametrized curve \\n(𝑃no], 𝑅no]) is a widely used tool to summarize the power of the method. \\n', metadata={'x0': 108.0, 'x1': 526.2052001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2052001953125, 'height': 28.63751220703125, 'nlines': 2, 'fs': 14.318756103515625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. Voxel-based \\nF1-score \\n(also \\nknown \\nas \\nSørensen-Dice \\ncoefficient). \\n', metadata={'x0': 90.0, 'x1': 526.2000122070312, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.20001220703125, 'height': 12.382293701171875, 'nlines': 7, 'fs': 1.7688991001674108, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='A common metric to calculate the performance of the method by combining the P and R values \\nis \\nsimply \\nthrough \\nthe \\nF1 \\nscore, \\ndefined \\nby \\ntheir \\nharmonic \\nmean: \\n', metadata={'x0': 108.0, 'x1': 526.2020263671875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2020263671875, 'height': 26.865966796875, 'nlines': 12, 'fs': 2.23883056640625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝐹p \"=\"\\n', metadata={'x0': 288.0, 'x1': 316.1111755371094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 28.111175537109375, 'height': 15.102874755859375, 'nlines': 1, 'fs': 15.102874755859375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='E∗q∗r\\n', metadata={'x0': 317.5199890136719, 'x1': 340.4852294921875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 22.965240478515625, 'height': 9.285125732421875, 'nlines': 1, 'fs': 9.285125732421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='q\"sr . \\n', metadata={'x0': 319.9200134277344, 'x1': 346.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 26.279998779296875, 'height': 17.745941162109375, 'nlines': 1, 'fs': 17.745941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Notice that here no account of the dependence with respect to the 𝑡ℎ𝑟 value is used. \\n \\n', metadata={'x0': 108.0, 'x1': 478.9200134277344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 370.9200134277344, 'height': 27.518218994140625, 'nlines': 2, 'fs': 13.759109497070312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Particle detection evaluation \\nIn the case of particles as discrete objects that are small enough to be defined by their coordinates, the \\nevaluation of the object detection task requires the following definitions: \\n', metadata={'x0': 72.0, 'x1': 526.2066040039062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20660400390625, 'height': 42.601409912109375, 'nlines': 3, 'fs': 14.200469970703125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. Given a sorted list of predicted particles’ centroids, one by one (in that order) they are either \\n', metadata={'x0': 90.0, 'x1': 526.1964721679688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.19647216796875, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='classified as false positives (fp) or true positives (tp). \\n', metadata={'x0': 108.0, 'x1': 343.8000183105469, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 235.80001831054688, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. A predicted particle coordinate is considered a true positive if and only if given a pre-defined \\n', metadata={'x0': 90.0, 'x1': 526.2000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.2000732421875, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='tolerance radius r (𝑟 = 10 vox for both RIBO and FAS), there exists a ground-truth coordinate \\nnot previously matched to any other predicted particle. \\n', metadata={'x0': 108.0, 'x1': 526.2012939453125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.2012939453125, 'height': 27.518218994140625, 'nlines': 2, 'fs': 13.759109497070312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. Then the coordinate-based precision (p), recall (r), F1 score and area under the precision-recall \\n', metadata={'x0': 90.0, 'x1': 526.221923828125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 436.221923828125, 'height': 12.382293701171875, 'nlines': 1, 'fs': 12.382293701171875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='curve (AUPRC) are defined by: \\n', metadata={'x0': 108.0, 'x1': 250.67999267578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.67999267578125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑝\" =\\n', metadata={'x0': 108.0, 'x1': 128.00765991210938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 20.007659912109375, 'height': 12.94293212890625, 'nlines': 1, 'fs': 12.94293212890625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='nH\"s\"tH, 𝑟\" =\"\\n', metadata={'x0': 131.0399932861328, 'x1': 187.71116638183594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 56.671173095703125, 'height': 17.76971435546875, 'nlines': 1, 'fs': 17.76971435546875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='nH\"s\"t2, 𝐹p \"=\\n', metadata={'x0': 189.36000061035156, 'x1': 247.52764892578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 58.16764831542969, 'height': 17.76971435546875, 'nlines': 1, 'fs': 17.76971435546875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='E∗H∗]\\n', metadata={'x0': 250.55999755859375, 'x1': 272.2720031738281, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 21.712005615234375, 'height': 9.28515625, 'nlines': 1, 'fs': 9.28515625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='H\"s\"] \", and AUPRC is the area under the curve defined by the \\n', metadata={'x0': 252.0, 'x1': 526.2023315429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 274.20233154296875, 'height': 18.462890625, 'nlines': 1, 'fs': 18.462890625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(𝑝, 𝑟) points when parametrized with respect to the cluster number (from 0 to all). \\n \\nMerging several lists for ground truth construction \\nTo account for possible duplicates when merging lists of particle coordinates from different methods \\n(e.g. manually-, TM- and CNN-originated lists), we integrate them by restricting the distance between \\ncentroid coordinates, i.e. with the constraint 𝑑(𝑝, 𝑞) > 1, where the elliptic distance 𝑑(𝑝, 𝑞) is defined \\nby: \\n \\n', metadata={'x0': 72.0, 'x1': 526.2056884765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.2056884765625, 'height': 116.3182373046875, 'nlines': 8, 'fs': 14.539779663085938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='𝑑(𝑝, 𝑞)E \"=\\n', metadata={'x0': 154.55999755859375, 'x1': 205.7676544189453, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 51.20765686035156, 'height': 17.02288818359375, 'nlines': 1, 'fs': 17.02288818359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='wx\"(𝑝p\"−\"𝑞p)E +\"\\n', metadata={'x0': 208.8000030517578, 'x1': 290.1911926269531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 81.39118957519531, 'height': 21.8497314453125, 'nlines': 1, 'fs': 21.8497314453125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='yx (𝑝E\"−\"𝑞E)E \"+\"\\n', metadata={'x0': 291.3599853515625, 'x1': 372.9911804199219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 81.63119506835938, 'height': 21.8497314453125, 'nlines': 1, 'fs': 21.8497314453125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ox (𝑝z\"−\"𝑞z)E,  \\n', metadata={'x0': 374.1600036621094, 'x1': 446.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 72.1199951171875, 'height': 18.8851318359375, 'nlines': 1, 'fs': 18.8851318359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nfor 𝑎 = 𝑏 = 9, 𝑐 = 15 , and where 𝑝 = (𝑝p, 𝑝E,𝑝z) and 𝑞 = (𝑞p, 𝑞E, 𝑞z). \\nThe choice of the elliptic distance coefficients takes on account both the effect of elongation along the \\nz-axis in the cryo-ET images and the size of the particle in voxels of the tomogram (1 vox = 13.48 Å).  \\n \\n \\n', metadata={'x0': 72.0, 'x1': 526.6799926757812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.67999267578125, 'height': 70.78594970703125, 'nlines': 6, 'fs': 11.797658284505209, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Tables \\n', metadata={'x0': 230.39999389648438, 'x1': 368.82000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 138.42001342773438, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Table 1: Selected hyperparameters of DeePiCt for varying cellular structures. Performances \\nwere calculated by cross-validation (CV) within the same domain, or in the case of microtubules and actin \\nnetworks, in evaluations for generalization across cellular species with respect to expert annotations in the HeLa \\n', metadata={'x0': 72.0, 'x1': 525.96533203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96533203125, 'height': 51.858306884765625, 'nlines': 4, 'fs': 12.964576721191406, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='cell tomogram (!\"!). DeePiCt networks were applied to the test datasets with the following region mask and contact \\n', metadata={'x0': 72.0, 'x1': 525.9603881835938, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96038818359375, 'height': 11.506546020507812, 'nlines': 1, 'fs': 11.506546020507812, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='mode combinations: cytosol and intersection for ribosome and FAS; cytosol and contact for membranes; nuclear \\nenvelope and contact for NPCs; cytosolic volume and intersection for microtubules and actin filaments.  \\n \\n', metadata={'x0': 72.0, 'x1': 525.9619750976562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96197509765625, 'height': 37.562835693359375, 'nlines': 3, 'fs': 12.520945231119791, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cellular \\nfeature \\nMethod \\n', metadata={'x0': 75.5999984741211, 'x1': 176.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 101.1600112915039, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Depth \\n(D) \\n', metadata={'x0': 188.8800048828125, 'x1': 217.5520477294922, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 28.672042846679688, 'height': 24.362823486328125, 'nlines': 2, 'fs': 12.181411743164062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Initial \\nfilters \\n(IF) \\nBN \\n', metadata={'x0': 229.44000244140625, 'x1': 293.1600036621094, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 63.720001220703125, 'height': 37.562835693359375, 'nlines': 4, 'fs': 9.390708923339844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Median \\nPerformance \\n', metadata={'x0': 308.1600036621094, 'x1': 366.1199951171875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 57.959991455078125, 'height': 24.362823486328125, 'nlines': 2, 'fs': 12.181411743164062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training \\ndataset \\nTesting dataset \\n', metadata={'x0': 383.0400085449219, 'x1': 518.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 135.72000122070312, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cytoplasm \\n2D CNN \\n5 \\n16 \\n1 AUPRC=0.97 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.635101318359375, 'nlines': 7, 'fs': 1.6621573311941964, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Organelles \\n2D CNN \\n5 \\n16 \\n1 AUPRC=0.92 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.395111083984375, 'nlines': 7, 'fs': 1.6278730119977678, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Ribosomes \\nDeePiCt \\n2 \\n4 \\n1 F1=0.79 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.162811279296875, 'nlines': 7, 'fs': 1.5946873256138392, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='FAS \\nDeePiCt \\n2 \\n16 \\n1 F1=0.46 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.162811279296875, 'nlines': 7, 'fs': 1.5946873256138392, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Membranes \\nDeePiCt \\n2 \\n4 \\n1 voxel-F1=0.79 \\nVPP \\nVPP \\n', metadata={'x0': 75.5999984741211, 'x1': 472.21380615234375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 396.61380767822266, 'height': 11.162811279296875, 'nlines': 7, 'fs': 1.5946873256138392, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='NPCs \\nDeePiCt \\n3 \\n16 \\n1 \\n', metadata={'x0': 75.5999984741211, 'x1': 306.8399963378906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 231.23999786376953, 'height': 11.162841796875, 'nlines': 5, 'fs': 2.232568359375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='voxel-F1=0.24 \\nDEF* \\nDEF* \\n', metadata={'x0': 308.1600036621094, 'x1': 477.97381591796875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 169.81381225585938, 'height': 11.162811279296875, 'nlines': 3, 'fs': 3.7209370930989585, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='voxel-F1=0.47 \\nDEF* \\n', metadata={'x0': 308.1600036621094, 'x1': 409.5738220214844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 101.413818359375, 'height': 11.63507080078125, 'nlines': 2, 'fs': 5.817535400390625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='DEF* (high \\nquality) \\n', metadata={'x0': 451.44000244140625, 'x1': 501.4791259765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 50.03912353515625, 'height': 24.362823486328125, 'nlines': 2, 'fs': 12.181411743164062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='voxel-F1=0.19 \\nDEF* \\n', metadata={'x0': 308.1600036621094, 'x1': 409.5738220214844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 101.413818359375, 'height': 11.39508056640625, 'nlines': 2, 'fs': 5.697540283203125, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='DEF* (lower \\nquality) \\n', metadata={'x0': 451.44000244140625, 'x1': 506.51910400390625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 55.0791015625, 'height': 24.602874755859375, 'nlines': 2, 'fs': 12.301437377929688, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Microtubules \\nDeePiCt \\n2 \\n4 \\n1 #$%&\\'()*+,-./\" VPP \\nHeLa cell \\n', metadata={'x0': 75.5999984741211, 'x1': 493.09381103515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 417.49381256103516, 'height': 16.314453125, 'nlines': 6, 'fs': 2.7190755208333335, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Actin \\nDeePiCt \\n3 \\n8 \\n0 #$%&\\'()*+,-*,\" VPP \\nHeLa cell \\n', metadata={'x0': 75.5999984741211, 'x1': 493.09381103515625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 417.49381256103516, 'height': 16.31439208984375, 'nlines': 6, 'fs': 2.7190653483072915, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 218.75999450683594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 146.75999450683594, 'height': 41.50592041015625, 'nlines': 4, 'fs': 10.376480102539062, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Table 2: Specifications of three 3D-CNN training and prediction rounds performed for \\nribosome ground truth construction. Each of the rounds in step 2 of the RIBO ground truth construction \\nconsisted of training 3 different 3D CNNs with default parameters except the IF number (Supplementary Note \\n1). The first training was performed on manually curated TM results of the corresponding dataset (VPP or DEF). \\nFor VPP, in the second round, the 3D CNN were trained on aggregated TM results and first round predictions; \\nwhereas the third CNN round was trained on all previous aggregated results. For DEF, the second and third round \\nof 3D CNNs corresponded to the VPP 3D CNNs of first and second rounds, respectively. \\n \\n', metadata={'x0': 72.0, 'x1': 525.96435546875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96435546875, 'height': 104.97050476074219, 'nlines': 8, 'fs': 13.121313095092773, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='traini\\nng \\nround \\n', metadata={'x0': 84.95999908447266, 'x1': 113.63999938964844, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 28.68000030517578, 'height': 37.802764892578125, 'nlines': 3, 'fs': 12.600921630859375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training \\ndataset \\n', metadata={'x0': 121.19999694824219, 'x1': 161.6399688720703, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 40.439971923828125, 'height': 24.362762451171875, 'nlines': 2, 'fs': 12.181381225585938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Prediction \\ndataset \\n', metadata={'x0': 187.9199981689453, 'x1': 235.07931518554688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 47.15931701660156, 'height': 24.362762451171875, 'nlines': 2, 'fs': 12.181381225585938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training set \\nannotations \\n', metadata={'x0': 262.79998779296875, 'x1': 317.40032958984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 54.600341796875, 'height': 24.362762451171875, 'nlines': 2, 'fs': 12.181381225585938, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1 \\nVPP \\nVPP \\nTM annotations \\nIF=4, 8, and 32 \\nvpp_output_rou\\nnd1 \\n', metadata={'x0': 84.95999908447266, 'x1': 520.3195190429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 435.3595199584961, 'height': 24.602813720703125, 'nlines': 7, 'fs': 3.5146876743861606, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2 \\nVPP \\nVPP \\nTM annotations + \\nvpp_output_round1 \\n', metadata={'x0': 84.95999908447266, 'x1': 344.2799987792969, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 259.3199996948242, 'height': 24.602813720703125, 'nlines': 5, 'fs': 4.920562744140625, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nvpp_output_rou\\nnd2 \\n', metadata={'x0': 361.44000244140625, 'x1': 520.3195190429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 158.8795166015625, 'height': 24.602813720703125, 'nlines': 3, 'fs': 8.200937906901041, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3 \\nVPP \\nVPP \\nTM \\nannotations \\n+ \\n', metadata={'x0': 84.95999908447266, 'x1': 353.8788146972656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.91881561279297, 'height': 11.162811279296875, 'nlines': 6, 'fs': 1.8604685465494792, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nvpp_output_rou\\nnd3 \\n', metadata={'x0': 361.44000244140625, 'x1': 520.3195190429688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 158.8795166015625, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1 \\nDEF \\nDEF \\nTM annotations \\nIF=4, 8, and 32 \\ndef_output_roun\\nd1 \\n', metadata={'x0': 84.95999908447266, 'x1': 523.1993408203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 438.23934173583984, 'height': 24.362823486328125, 'nlines': 7, 'fs': 3.480403355189732, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2 \\nDEF \\nDEF \\nTM \\nannotations \\n+ \\n', metadata={'x0': 84.95999908447266, 'x1': 353.8788146972656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.91881561279297, 'height': 11.162811279296875, 'nlines': 6, 'fs': 1.8604685465494792, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='def_output_round1 \\n', metadata={'x0': 262.79998779296875, 'x1': 341.8799743652344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 79.07998657226562, 'height': 11.162811279296875, 'nlines': 1, 'fs': 11.162811279296875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\ndef_output_roun\\nd2 \\n', metadata={'x0': 361.44000244140625, 'x1': 523.1993408203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 161.75933837890625, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3 \\nVPP \\nDEF \\nTM \\nannotations \\n+ \\n', metadata={'x0': 84.95999908447266, 'x1': 353.8788146972656, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 268.91881561279297, 'height': 11.162811279296875, 'nlines': 6, 'fs': 1.8604685465494792, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\ndef_output_roun\\nd3 \\n', metadata={'x0': 361.44000244140625, 'x1': 523.1993408203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 161.75933837890625, 'height': 24.362823486328125, 'nlines': 3, 'fs': 8.120941162109375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\n \\nSupplementary Table 3: Specifications of 3D-CNN training and prediction rounds performed for FAS \\nground truth construction. Step 2 in FAS ground truth construction consisted of a single round of 3D CNN \\ntraining, performed separately per dataset type (VPP and DEF). The round consisted of 3 networks with default \\nparameters (Supplementary Note 1) except for the IF number, set to IF=4, 8, and 32. The networks were trained \\non an initial incomplete manual picking. \\n \\n', metadata={'x0': 72.0, 'x1': 525.9617919921875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9617919921875, 'height': 106.09832763671875, 'nlines': 8, 'fs': 13.262290954589844, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Training dataset \\nPrediction dataset \\nTraining \\nset \\n', metadata={'x0': 84.0, 'x1': 349.0801086425781, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.0801086425781, 'height': 11.162841796875, 'nlines': 4, 'fs': 2.79071044921875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='VPP \\nVPP \\nInitial \\nmanual \\n', metadata={'x0': 84.0, 'x1': 349.0800476074219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.0800476074219, 'height': 11.162841796875, 'nlines': 4, 'fs': 2.79071044921875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nfas_vpp_output_round\\n1 \\n', metadata={'x0': 356.6400146484375, 'x1': 522.47998046875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 165.8399658203125, 'height': 24.60284423828125, 'nlines': 3, 'fs': 8.200948079427084, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='DEF \\nDEF \\nInitial \\nmanual \\n', metadata={'x0': 84.0, 'x1': 349.0800476074219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 265.0800476074219, 'height': 11.162841796875, 'nlines': 4, 'fs': 2.79071044921875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IF=4, 8, and 32 \\nfas_def_output_round\\n1 \\n', metadata={'x0': 356.6400146484375, 'x1': 520.3198852539062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 163.67987060546875, 'height': 24.60284423828125, 'nlines': 3, 'fs': 8.200948079427084, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Supplementary Note 2 \\n', metadata={'x0': 230.16000366210938, 'x1': 368.82000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 138.66000366210938, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Considerations for structural analysis following CNN predictions under \\n', metadata={'x0': 82.80000305175781, 'x1': 516.0829467773438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 433.28294372558594, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='different imaging conditions \\n', metadata={'x0': 212.16000366210938, 'x1': 386.82000732421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 174.66000366210938, 'height': 16.744216918945312, 'nlines': 1, 'fs': 16.744216918945312, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Cryo-ET datasets of S. pombe utilized in this study were derived from cryo-FIB lamellae of wild-type, \\nnative, yeast cells. 20 tomograms used for comprehensive annotations were acquired with a dose-\\nsymmetric tilt scheme6 employing similar acquisition parameters (Extended Data Tables 4-6, Online \\nMethods), with the only difference being imaging with traditional defocus-only (DEF) or, in addition, \\nthe usage of a VPP. All RIBO and FAS particles annotated in the two datasets (DEF and VPP) were \\nsubjected to subtomogram analysis including CTF correction utilizing CTF fitting and model creation \\nin Warp, and 3D refinements, hierarchical, and focused 3D classifications in RELION. During this \\nstructural analysis we made several observations which will be discussed in the following.  \\n  \\nAs described previously7,8,9, acquiring tomography data with a VPP emphasizes low frequencies. \\nPronounced low frequencies lead to improved image contrast which can facilitate data mining. With \\nour method, more particles were detected in VPP than in DEF ground truth annotations of FAS \\ncomplexes, which represent more challenging targets for data mining due to their low abundance and \\nhollow structural signature. The increased SNR in VPP was also advantageous in hierarchical \\nclassifications of ribosomes (Supplementary Fig. 2), while no difference was apparent for FAS likely \\ndue to the low particle numbers (Supplementary Fig. 3). For VPP, all ribosomes clustered in defined \\naverages which confirmed the high performance of DeePiCt and enabled the classification of a subset \\nof 60S large subunits (Supplementary Fig. 2). In comparison, most DEF ribosomes from ground truth \\nannotations or DeePiCt predictions clustered in poorly defined classes and no 60S large subunits could \\nbe separated even when classification results were improved by optimized particle poses (coordinates \\nand orientations) after multi-particle refinement in M10 (Supplementary Fig. 4, 5). Also, in \\nclassifications focused on an area close to the head of the ribosomal small subunit and at the peptide \\nexit tunnel the additional head density was already apparent in 2D slices of the respective classes in the \\nVPP data (Supplementary Fig. 6 and 7, respectively). Similarly, ribosomes close to ER and \\nmitochondria that clustered into classes with adjacent membrane densities are more pronounced in VPP \\n(Supplementary Fig. 8 and 9, respectively). \\n  \\nHowever, fine structural details that provide insights into functional configurations of molecular \\ncomplexes are lost or cannot be recovered in the final reconstructions of VPP tomography data9. This \\nhas been suggested to be caused by inaccurate weighing9 and a signal loss at high spatial frequencies11. \\nThe preservation of high-resolution information in DEF in comparison to VPP becomes especially \\napparent when looking at fine structural details in the subtomogram averages. Densities fitting ACPs of \\nFAS (Extended data Fig. 8 d-e), P-site tRNAs (Extended data Fig. 8 j-k) inside the 80S ribosome \\nand densities connecting ribosomes to an adjacent mitochondrion membrane (Supplementary Fig. 9) \\ncould only be resolved in DEF data. \\n  \\nWe conclude that the usage of a VPP in tomography acquisition facilitated data mining including the \\ndemonstrated domain generalization of applying DeePiCt models trained in VPP to DEF, and improved \\n3D classifications. This makes it a valuable tool that ultimately provides benefits at the exploratory \\nphase of structural analysis, helping the design of experiments and analysis pipelines that can eventually \\nbe performed on DEF data from which higher resolution reconstructions can be obtained. \\n', metadata={'x0': 72.0, 'x1': 526.2098999023438, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20989990234375, 'height': 593.9859619140625, 'nlines': 41, 'fs': 14.487462485708841, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 2 | Hierarchical 3D classification of cytosolic ribosomes in VPP ground truth. 2D slices \\nthrough 3D class averages of iteration 25 are displayed. The highlighted class (cyan box) shows the 60S large \\nribosomal subunit class. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9614868164062, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96148681640625, 'height': 79.93829345703125, 'nlines': 6, 'fs': 13.323048909505209, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 3 | 3D classification of VPP and DEF ground truth, and DeePiCt prediction in DEF \\ndataset for FAS. a-c. 2D slices through 3D-refined subtomogram average of FAS using D3 symmetry recovered \\nfrom VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. 3D classifications separated in all 3 \\ndatasets into two classes of which one is better structurally defined than the other. 3D refinements for DEF were \\nperformed in M. \\n \\n', metadata={'x0': 72.0, 'x1': 525.9619140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9619140625, 'height': 90.36279296875, 'nlines': 7, 'fs': 12.908970424107142, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 4 | Hierarchical 3D classification of cytosolic ribosomes in DEF ground truth starting \\nfrom M-refined alignments. 2D slices through 3D class averages of iteration 25 are displayed. The highlighted \\nclass (yellow box) was further refined in M, resulting in a well aligned class with a resolution of 9.3 Å. \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.96142578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96142578125, 'height': 78.49832153320312, 'nlines': 6, 'fs': 13.083053588867188, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 5 | Hierarchical 3D classification of cytosolic ribosomes from DeePiCt predictions in \\nDEF data starting from M-refined alignments. 2D slices through 3D class averages of iteration 25 are \\ndisplayed. The highlighted class (yellow box) was further refined in M resulting in a well-aligned class with a \\nresolution of 9.4 Å. The right side of the classification representing the vast majority of ribosomes does not \\nconverge into a well defined map. \\n \\n', metadata={'x0': 72.0, 'x1': 525.9690551757812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96905517578125, 'height': 90.60284423828125, 'nlines': 7, 'fs': 12.943263462611608, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 6 | 3D classifications and refinements of cryo-EM densities in proximity to the head of \\nthe 40S small ribosomal subunit in VPP gt, DEF gt, and DeePiCt predictions in DEF datasets. a-c. 2D slices \\nthrough 3D-refined subtomogram average of all cytosolic ribosomes from VPP ground truth (gt), DEF gt or \\nDeePiCt predictions on DEF datasets. Focused 3D classifications (blue sphere mask displayed in a) separated in \\nall 3 datasets one class (dark blue) with an additional density close to the head of the 40S small ribosomal subunit. \\n3D refinements for DEF were performed in M. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9642333984375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9642333984375, 'height': 117.21051025390625, 'nlines': 10, 'fs': 11.721051025390626, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 7 | 3D classifications and refinements of ribosomes at the exit tunnel in VPP gt, DEF \\ngt, and DeePiCt predictions in DEF datasets. a-c. 2D slices through 3D-refined subtomogram average of all \\nribosomes from VPP ground truth (gt), DEF gt or DeePiCt prediction on DEF datasets. Focused 3D classifications \\n(cyan sphere mask displayed in a) separated in all 3 datasets one class (cyan) with an additional exit tunnel density. \\n3D refinements for DEF were performed in M. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9629516601562, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.96295166015625, 'height': 103.56280517578125, 'nlines': 9, 'fs': 11.506978352864584, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 8 | 3D classifications and refinements of ER-bound ribosomes in VPP gt, DEF gt, and \\nDeePiCt predictions in DEF datasets. a-c. 2D slices through 3D-refined subtomogram average of ribosomes in \\n25 nm distance to the ER recovered from VPP ground truth (gt), DEF gt or DeePiCt predictions on DEF datasets. \\nFocused 3D classifications (cyan sphere mask displayed in Supplementary Fig. 7a) separated in all 3 datasets one \\nclass (orange) with membrane density. 3D refinements for DEF were performed in M. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.968017578125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.968017578125, 'height': 105.0028076171875, 'nlines': 9, 'fs': 11.666978624131945, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content=' \\nSupplementary Fig. 9 | 3D classifications and refinements of mitochondria-bound ribosomes in VPP gt, \\nDEF gt, and DeePiCt predictions in DEF datasets. a-c. 2D slices through 3D-refined subtomogram average of \\nribosomes in 25 nm distance to the mitochondria recovered from VPP ground truth (gt), DEF gt or DeePiCt \\nprediction on DEF datasets. Focused 3D classifications (cyan sphere mask displayed in Supplementary Fig. 7a) \\nseparated in all 3 datasets one class (green) with membrane density. 3D refinements for DEF were performed in \\nM. \\n \\n \\n \\n', metadata={'x0': 72.0, 'x1': 525.9649658203125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 453.9649658203125, 'height': 118.44284057617188, 'nlines': 10, 'fs': 11.844284057617188, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='1. \\nRonneberger, O., Fischer, P. & Brox, T. U-Net: Convolutional Networks for Biomedical Image \\n', metadata={'x0': 72.0, 'x1': 516.6036987304688, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 444.60369873046875, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Navab, N., Hornegger, J., Wells, W. M. & Frangi, A. F.) vol. 9351 234–241 (Springer \\n', metadata={'x0': 93.83999633789062, 'x1': 475.8011169433594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 381.96112060546875, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='International Publishing, 2015). \\n', metadata={'x0': 93.83999633789062, 'x1': 236.51998901367188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 142.67999267578125, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='2. \\nZeiler, M. D., Krishnan, D., Taylor, G. W. & Fergus, R. Deconvolutional networks. in 2010 \\n', metadata={'x0': 72.0, 'x1': 500.7604064941406, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 428.7604064941406, 'height': 12.225936889648438, 'nlines': 2, 'fs': 6.112968444824219, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='IEEE Computer Society Conference on Computer Vision and Pattern Recognition 2528–2535 \\n', metadata={'x0': 93.83999633789062, 'x1': 509.6404113769531, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 415.8004150390625, 'height': 12.225936889648438, 'nlines': 1, 'fs': 12.225936889648438, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(IEEE, 2010). doi:10.1109/CVPR.2010.5539957. \\n', metadata={'x0': 93.83999633789062, 'x1': 314.0400085449219, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 220.20001220703125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='3. \\nMilletari, F., Navab, N. & Ahmadi, S.-A. V-Net: Fully Convolutional Neural Networks for \\n', metadata={'x0': 72.0, 'x1': 496.683349609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 424.683349609375, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='Volumetric Medical Image Segmentation. in 2016 Fourth International Conference on 3D Vision \\n', metadata={'x0': 93.83999633789062, 'x1': 526.2052001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 432.3652038574219, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(3DV) 565–571 (IEEE, 2016). doi:10.1109/3DV.2016.79. \\n', metadata={'x0': 93.83999633789062, 'x1': 350.760009765625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 256.9200134277344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='4. \\nKingma, D. P. & Ba, J. Adam: A Method for Stochastic Optimization. \\n', metadata={'x0': 72.0, 'x1': 406.2078857421875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 334.2078857421875, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='https://arxiv.org/abs/1412.6980 (2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 269.4000244140625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 175.56002807617188, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='5. \\nvan der Walt, S. et al. scikit-image: image processing in Python. PeerJ 2, e453 (2014). \\n', metadata={'x0': 72.0, 'x1': 478.20001220703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 406.20001220703125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='6. \\nHagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-\\n', metadata={'x0': 72.0, 'x1': 520.1563720703125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 448.1563720703125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 \\n', metadata={'x0': 93.83999633789062, 'x1': 502.200439453125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 408.3604431152344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='(2017). \\n', metadata={'x0': 93.83999633789062, 'x1': 128.75999450683594, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 34.91999816894531, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='7. \\nDanev, R., Buijsse, B., Khoshouei, M., Plitzko, J. M. & Baumeister, W. Volta potential phase \\n', metadata={'x0': 72.0, 'x1': 509.1627197265625, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 437.1627197265625, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='plate for in-focus phase contrast transmission electron microscopy. Proc Natl Acad Sci USA 111, \\n', metadata={'x0': 93.83999633789062, 'x1': 523.5602416992188, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 429.7202453613281, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='15635–15640 (2014). \\n', metadata={'x0': 93.83999633789062, 'x1': 191.87998962402344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 98.03999328613281, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='8. \\nDanev, R., Tegunov, D. & Baumeister, W. Using the Volta phase plate with defocus for cryo-EM \\n', metadata={'x0': 72.0, 'x1': 526.2044067382812, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 454.20440673828125, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='9. \\nTuroňová, B. et al. Benchmarking tomographic acquisition schemes for high-resolution \\n', metadata={'x0': 72.0, 'x1': 482.2804260253906, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 410.2804260253906, 'height': 12.2259521484375, 'nlines': 2, 'fs': 6.11297607421875, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='structural biology. Nat. Commun. 11, 876 (2020). \\n', metadata={'x0': 93.83999633789062, 'x1': 314.52001953125, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 220.68002319335938, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='10. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM \\n', metadata={'x0': 72.0, 'x1': 490.4444274902344, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 418.4444274902344, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='refinement with M visualizes ribosome-antibiotic complex at 3.5 Å in cells. Nat. Methods 18, \\n', metadata={'x0': 93.83999633789062, 'x1': 507.4797668457031, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 413.6397705078125, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='186–193 (2021). \\n', metadata={'x0': 93.83999633789062, 'x1': 169.79998779296875, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 75.95999145507812, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='11. Buijsse, B., Trompenaars, P., Altin, V., Danev, R. & Glaeser, R. M. Spectral DQE of the Volta \\n', metadata={'x0': 72.0, 'x1': 514.6878662109375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 442.6878662109375, 'height': 12.2259521484375, 'nlines': 1, 'fs': 12.2259521484375, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66}),\n",
       " Document(page_content='phase plate. Ultramicroscopy 218, 113079 (2020). \\navailable under aCC-BY-NC-ND 4.0 International license.\\nwas not certified by peer review) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity. It is made \\nbioRxiv preprint doi: https://doi.org/10.1101/2022.04.12.488077; this version posted April 13, 2022. The copyright holder for this preprint (which\\n', metadata={'x0': 51.65239334106445, 'x1': 556.3480224609375, 'y0': 53.400001525878906, 'y1': 84.70805358886719, 'width': 504.69562911987305, 'height': 31.30805206298828, 'nlines': 4, 'fs': 7.82701301574707, 'file_path': '/Users/gburns/Documents/2023H2/cryoet_portal_rocrates/reading/cryoet_papers/de_teresa-2022-10.1101-2022.04.12.488077.pdf', 'page': 65, 'total_pages': 66})]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "l2 = PyMuPDFBlockLoader(d2+f2)\n",
    "blocks = l2.load()\n",
    "\n",
    "blocks\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Starting analysis with Langchain"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "llama_model_loader: loaded meta data with 19 key-value pairs and 723 tensors from /Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf (version GGUF V2 (latest))\n",
      "llama_model_loader: - tensor    0:                token_embd.weight q5_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - tensor    1:           blk.0.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    2:            blk.0.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    3:            blk.0.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    4:              blk.0.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    5:            blk.0.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    6:              blk.0.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor    7:         blk.0.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    8:              blk.0.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    9:              blk.0.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   10:           blk.1.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   11:            blk.1.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   12:            blk.1.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   13:              blk.1.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   14:            blk.1.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   15:              blk.1.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   16:         blk.1.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   17:              blk.1.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   18:              blk.1.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   19:           blk.2.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   20:            blk.2.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   21:            blk.2.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   22:              blk.2.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   23:            blk.2.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   24:              blk.2.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   25:         blk.2.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   26:              blk.2.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   27:              blk.2.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   28:           blk.3.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   29:            blk.3.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   30:            blk.3.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   31:              blk.3.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   32:            blk.3.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   33:              blk.3.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   34:         blk.3.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   35:              blk.3.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   36:              blk.3.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   37:           blk.4.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   38:            blk.4.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   39:            blk.4.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   40:              blk.4.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   41:            blk.4.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   42:              blk.4.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   43:         blk.4.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   44:              blk.4.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   45:              blk.4.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   46:            blk.5.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   47:              blk.5.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   48:         blk.5.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   49:              blk.5.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   50:              blk.5.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   51:          blk.10.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   52:           blk.10.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   53:           blk.10.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   54:             blk.10.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   55:           blk.10.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   56:             blk.10.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   57:        blk.10.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   58:             blk.10.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   59:             blk.10.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   60:             blk.11.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   61:        blk.11.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   62:             blk.11.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   63:             blk.11.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   64:           blk.5.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   65:            blk.5.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   66:              blk.5.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   67:            blk.5.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   68:           blk.6.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   69:            blk.6.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   70:            blk.6.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   71:              blk.6.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   72:            blk.6.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   73:              blk.6.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   74:         blk.6.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   75:              blk.6.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   76:              blk.6.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   77:           blk.7.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   78:            blk.7.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   79:            blk.7.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   80:              blk.7.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   81:            blk.7.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   82:              blk.7.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   83:         blk.7.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   84:              blk.7.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   85:              blk.7.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   86:           blk.8.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   87:            blk.8.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   88:            blk.8.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   89:              blk.8.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   90:            blk.8.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   91:              blk.8.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   92:         blk.8.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   93:              blk.8.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   94:              blk.8.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   95:           blk.9.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   96:            blk.9.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   97:            blk.9.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   98:              blk.9.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   99:            blk.9.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  100:              blk.9.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  101:         blk.9.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  102:              blk.9.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  103:              blk.9.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  104:          blk.11.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  105:           blk.11.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  106:           blk.11.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  107:             blk.11.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  108:           blk.11.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  109:          blk.12.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  110:           blk.12.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  111:           blk.12.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  112:             blk.12.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  113:           blk.12.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  114:             blk.12.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  115:        blk.12.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  116:             blk.12.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  117:             blk.12.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  118:          blk.13.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  119:           blk.13.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  120:           blk.13.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  121:             blk.13.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  122:           blk.13.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  123:             blk.13.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  124:        blk.13.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  125:             blk.13.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  126:             blk.13.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  127:          blk.14.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  128:           blk.14.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  129:           blk.14.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  130:             blk.14.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  131:           blk.14.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  132:             blk.14.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  133:        blk.14.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  134:             blk.14.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  135:             blk.14.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  136:          blk.15.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  137:           blk.15.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  138:           blk.15.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  139:             blk.15.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  140:           blk.15.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  141:             blk.15.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  142:        blk.15.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  143:             blk.15.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  144:             blk.15.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  145:          blk.16.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  146:           blk.16.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  147:           blk.16.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  148:             blk.16.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  149:           blk.16.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  150:             blk.16.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  151:        blk.16.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  152:             blk.16.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  153:             blk.16.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  154:          blk.17.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  155:           blk.17.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  156:           blk.17.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  157:             blk.17.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  158:           blk.17.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  159:             blk.17.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  160:        blk.17.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  161:             blk.17.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  162:             blk.17.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  163:          blk.18.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  164:           blk.18.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  165:           blk.18.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  166:             blk.18.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  167:           blk.18.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  168:             blk.18.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  169:        blk.18.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  170:             blk.18.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  171:             blk.18.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  172:          blk.19.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  173:           blk.19.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  174:           blk.19.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  175:             blk.19.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  176:           blk.19.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  177:             blk.19.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  178:        blk.19.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  179:             blk.19.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  180:             blk.19.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  181:          blk.20.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  182:           blk.20.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  183:           blk.20.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  184:             blk.20.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  185:           blk.20.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  186:             blk.20.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  187:        blk.20.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  188:             blk.20.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  189:             blk.20.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  190:          blk.21.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  191:           blk.21.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  192:           blk.21.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  193:             blk.21.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  194:           blk.21.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  195:             blk.21.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  196:        blk.21.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  197:             blk.21.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  198:             blk.21.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  199:           blk.22.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  200:             blk.22.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  201:             blk.22.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  202:        blk.22.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  203:             blk.22.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  204:             blk.22.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  205:          blk.22.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  206:           blk.22.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  207:           blk.22.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  208:          blk.23.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  209:           blk.23.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  210:           blk.23.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  211:             blk.23.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  212:           blk.23.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  213:             blk.23.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  214:        blk.23.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  215:             blk.23.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  216:             blk.23.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  217:          blk.24.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  218:           blk.24.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  219:           blk.24.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  220:             blk.24.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  221:           blk.24.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  222:             blk.24.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  223:        blk.24.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  224:             blk.24.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  225:             blk.24.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  226:          blk.25.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  227:           blk.25.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  228:           blk.25.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  229:             blk.25.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  230:           blk.25.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  231:             blk.25.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  232:        blk.25.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  233:             blk.25.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  234:             blk.25.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  235:          blk.26.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  236:           blk.26.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  237:           blk.26.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  238:             blk.26.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  239:           blk.26.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  240:             blk.26.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  241:        blk.26.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  242:             blk.26.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  243:             blk.26.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  244:          blk.27.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  245:           blk.27.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  246:           blk.27.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  247:             blk.27.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  248:           blk.27.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  249:             blk.27.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  250:        blk.27.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  251:             blk.27.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  252:             blk.27.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  253:           blk.28.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  254:             blk.28.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  255:        blk.28.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  256:             blk.28.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  257:             blk.28.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  258:          blk.28.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  259:           blk.28.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  260:             blk.28.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  261:           blk.28.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  262:          blk.29.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  263:           blk.29.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  264:           blk.29.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  265:             blk.29.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  266:           blk.29.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  267:             blk.29.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  268:        blk.29.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  269:             blk.29.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  270:             blk.29.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  271:          blk.30.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  272:           blk.30.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  273:           blk.30.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  274:             blk.30.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  275:           blk.30.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  276:             blk.30.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  277:        blk.30.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  278:             blk.30.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  279:             blk.30.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  280:          blk.31.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  281:           blk.31.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  282:           blk.31.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  283:             blk.31.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  284:           blk.31.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  285:             blk.31.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  286:        blk.31.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  287:             blk.31.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  288:             blk.31.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  289:          blk.32.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  290:           blk.32.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  291:           blk.32.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  292:             blk.32.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  293:           blk.32.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  294:             blk.32.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  295:        blk.32.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  296:             blk.32.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  297:             blk.32.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  298:          blk.33.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  299:           blk.33.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  300:           blk.33.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  301:             blk.33.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  302:           blk.33.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  303:             blk.33.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  304:        blk.33.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  305:             blk.33.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  306:             blk.33.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  307:             blk.34.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  308:        blk.34.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  309:             blk.34.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  310:             blk.34.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  311:          blk.34.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  312:           blk.34.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  313:           blk.34.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  314:             blk.34.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  315:           blk.34.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  316:          blk.35.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  317:           blk.35.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  318:           blk.35.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  319:             blk.35.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  320:           blk.35.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  321:             blk.35.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  322:        blk.35.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  323:             blk.35.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  324:             blk.35.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  325:          blk.36.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  326:           blk.36.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  327:           blk.36.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  328:             blk.36.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  329:           blk.36.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  330:             blk.36.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  331:        blk.36.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  332:             blk.36.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  333:             blk.36.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  334:          blk.37.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  335:           blk.37.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  336:           blk.37.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  337:             blk.37.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  338:           blk.37.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  339:             blk.37.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  340:        blk.37.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  341:             blk.37.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  342:             blk.37.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  343:          blk.38.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  344:           blk.38.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  345:           blk.38.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  346:             blk.38.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  347:           blk.38.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  348:             blk.38.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  349:        blk.38.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  350:             blk.38.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  351:             blk.38.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  352:          blk.39.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  353:           blk.39.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  354:           blk.39.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  355:             blk.39.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  356:           blk.39.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  357:             blk.39.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  358:        blk.39.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  359:             blk.39.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  360:             blk.39.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  361:          blk.40.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  362:           blk.40.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  363:           blk.40.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  364:             blk.40.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  365:           blk.40.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  366:             blk.40.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  367:        blk.40.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  368:             blk.40.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  369:             blk.40.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  370:          blk.41.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  371:           blk.41.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  372:           blk.41.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  373:             blk.41.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  374:           blk.41.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  375:             blk.41.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  376:        blk.41.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  377:             blk.41.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  378:             blk.41.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  379:          blk.42.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  380:           blk.42.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  381:           blk.42.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  382:             blk.42.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  383:           blk.42.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  384:             blk.42.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  385:        blk.42.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  386:             blk.42.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  387:             blk.42.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  388:          blk.43.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  389:           blk.43.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  390:           blk.43.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  391:             blk.43.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  392:           blk.43.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  393:             blk.43.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  394:        blk.43.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  395:             blk.43.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  396:             blk.43.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  397:          blk.44.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  398:           blk.44.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  399:           blk.44.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  400:             blk.44.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  401:           blk.44.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  402:             blk.44.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  403:        blk.44.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  404:             blk.44.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  405:             blk.44.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  406:           blk.45.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  407:             blk.45.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  408:             blk.45.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  409:        blk.45.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  410:             blk.45.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  411:             blk.45.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  412:          blk.45.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  413:           blk.45.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  414:           blk.45.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  415:          blk.46.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  416:           blk.46.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  417:           blk.46.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  418:             blk.46.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  419:           blk.46.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  420:             blk.46.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  421:        blk.46.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  422:             blk.46.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  423:             blk.46.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  424:          blk.47.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  425:           blk.47.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  426:           blk.47.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  427:             blk.47.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  428:           blk.47.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  429:             blk.47.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  430:        blk.47.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  431:             blk.47.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  432:             blk.47.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  433:          blk.48.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  434:           blk.48.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  435:           blk.48.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  436:             blk.48.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  437:           blk.48.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  438:             blk.48.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  439:        blk.48.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  440:             blk.48.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  441:             blk.48.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  442:          blk.49.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  443:           blk.49.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  444:           blk.49.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  445:             blk.49.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  446:           blk.49.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  447:             blk.49.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  448:        blk.49.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  449:             blk.49.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  450:             blk.49.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  451:          blk.50.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  452:           blk.50.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  453:           blk.50.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  454:             blk.50.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  455:           blk.50.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  456:             blk.50.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  457:        blk.50.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  458:             blk.50.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  459:             blk.50.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  460:           blk.51.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  461:             blk.51.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  462:        blk.51.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  463:             blk.51.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  464:             blk.51.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  465:          blk.51.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  466:           blk.51.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  467:             blk.51.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  468:           blk.51.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  469:          blk.52.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  470:           blk.52.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  471:           blk.52.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  472:             blk.52.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  473:           blk.52.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  474:             blk.52.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  475:        blk.52.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  476:             blk.52.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  477:             blk.52.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  478:          blk.53.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  479:           blk.53.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  480:           blk.53.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  481:             blk.53.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  482:           blk.53.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  483:             blk.53.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  484:        blk.53.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  485:             blk.53.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  486:             blk.53.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  487:          blk.54.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  488:           blk.54.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  489:           blk.54.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  490:             blk.54.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  491:           blk.54.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  492:             blk.54.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  493:        blk.54.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  494:             blk.54.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  495:             blk.54.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  496:          blk.55.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  497:           blk.55.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  498:           blk.55.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  499:             blk.55.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  500:           blk.55.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  501:             blk.55.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  502:        blk.55.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  503:             blk.55.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  504:             blk.55.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  505:          blk.56.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  506:           blk.56.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  507:           blk.56.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  508:             blk.56.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  509:           blk.56.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  510:             blk.56.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  511:        blk.56.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  512:             blk.56.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  513:             blk.56.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  514:             blk.57.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  515:        blk.57.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  516:             blk.57.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  517:             blk.57.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  518:          blk.57.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  519:           blk.57.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  520:           blk.57.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  521:             blk.57.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  522:           blk.57.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  523:          blk.58.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  524:           blk.58.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  525:           blk.58.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  526:             blk.58.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  527:           blk.58.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  528:             blk.58.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  529:        blk.58.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  530:             blk.58.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  531:             blk.58.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  532:          blk.59.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  533:           blk.59.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  534:           blk.59.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  535:             blk.59.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  536:           blk.59.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  537:             blk.59.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  538:        blk.59.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  539:             blk.59.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  540:             blk.59.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  541:          blk.60.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  542:           blk.60.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  543:           blk.60.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  544:             blk.60.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  545:           blk.60.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  546:             blk.60.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  547:        blk.60.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  548:             blk.60.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  549:             blk.60.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  550:          blk.61.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  551:           blk.61.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  552:           blk.61.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  553:             blk.61.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  554:           blk.61.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  555:             blk.61.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  556:        blk.61.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  557:             blk.61.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  558:             blk.61.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  559:          blk.62.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  560:           blk.62.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  561:           blk.62.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  562:             blk.62.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  563:           blk.62.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  564:             blk.62.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  565:        blk.62.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  566:             blk.62.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  567:             blk.62.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  568:          blk.63.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  569:           blk.63.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  570:           blk.63.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  571:             blk.63.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  572:           blk.63.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  573:             blk.63.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  574:        blk.63.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  575:             blk.63.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  576:             blk.63.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  577:          blk.64.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  578:           blk.64.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  579:           blk.64.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  580:             blk.64.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  581:           blk.64.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  582:             blk.64.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  583:        blk.64.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  584:             blk.64.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  585:             blk.64.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  586:          blk.65.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  587:           blk.65.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  588:           blk.65.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  589:             blk.65.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  590:           blk.65.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  591:             blk.65.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  592:        blk.65.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  593:             blk.65.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  594:             blk.65.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  595:          blk.66.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  596:           blk.66.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  597:           blk.66.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  598:             blk.66.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  599:           blk.66.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  600:             blk.66.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  601:        blk.66.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  602:             blk.66.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  603:             blk.66.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  604:          blk.67.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  605:           blk.67.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  606:           blk.67.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  607:             blk.67.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  608:           blk.67.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  609:             blk.67.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  610:        blk.67.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  611:             blk.67.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  612:             blk.67.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  613:           blk.68.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  614:             blk.68.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  615:             blk.68.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  616:        blk.68.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  617:             blk.68.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  618:             blk.68.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  619:          blk.68.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  620:           blk.68.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  621:           blk.68.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  622:          blk.69.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  623:           blk.69.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  624:           blk.69.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  625:             blk.69.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  626:           blk.69.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  627:             blk.69.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  628:        blk.69.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  629:             blk.69.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  630:             blk.69.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  631:          blk.70.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  632:           blk.70.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  633:           blk.70.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  634:             blk.70.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  635:           blk.70.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  636:             blk.70.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  637:        blk.70.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  638:             blk.70.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  639:             blk.70.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  640:          blk.71.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  641:           blk.71.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  642:           blk.71.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  643:             blk.71.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  644:           blk.71.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  645:             blk.71.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  646:        blk.71.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  647:             blk.71.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  648:             blk.71.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  649:          blk.72.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  650:           blk.72.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  651:           blk.72.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  652:             blk.72.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  653:           blk.72.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  654:             blk.72.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  655:        blk.72.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  656:             blk.72.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  657:             blk.72.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  658:          blk.73.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  659:           blk.73.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  660:           blk.73.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  661:             blk.73.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  662:           blk.73.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  663:             blk.73.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  664:        blk.73.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  665:             blk.73.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  666:             blk.73.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  667:           blk.74.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  668:             blk.74.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  669:        blk.74.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  670:             blk.74.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  671:             blk.74.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  672:          blk.74.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  673:           blk.74.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  674:             blk.74.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  675:           blk.74.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  676:          blk.75.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  677:           blk.75.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  678:           blk.75.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  679:             blk.75.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  680:           blk.75.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  681:             blk.75.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  682:        blk.75.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  683:             blk.75.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  684:             blk.75.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  685:          blk.76.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  686:           blk.76.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  687:           blk.76.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  688:             blk.76.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  689:           blk.76.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  690:             blk.76.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  691:        blk.76.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  692:             blk.76.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  693:             blk.76.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  694:          blk.77.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  695:           blk.77.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  696:           blk.77.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  697:             blk.77.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  698:           blk.77.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  699:             blk.77.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  700:        blk.77.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  701:             blk.77.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  702:             blk.77.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  703:          blk.78.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  704:           blk.78.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  705:           blk.78.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  706:             blk.78.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  707:           blk.78.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  708:             blk.78.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  709:        blk.78.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  710:             blk.78.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  711:             blk.78.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  712:          blk.79.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  713:           blk.79.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  714:           blk.79.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  715:             blk.79.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  716:           blk.79.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  717:             blk.79.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  718:        blk.79.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  719:             blk.79.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  720:             blk.79.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  721:               output_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  722:                    output.weight q6_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - kv   0:                       general.architecture str     \n",
      "llama_model_loader: - kv   1:                               general.name str     \n",
      "llama_model_loader: - kv   2:                       llama.context_length u32     \n",
      "llama_model_loader: - kv   3:                     llama.embedding_length u32     \n",
      "llama_model_loader: - kv   4:                          llama.block_count u32     \n",
      "llama_model_loader: - kv   5:                  llama.feed_forward_length u32     \n",
      "llama_model_loader: - kv   6:                 llama.rope.dimension_count u32     \n",
      "llama_model_loader: - kv   7:                 llama.attention.head_count u32     \n",
      "llama_model_loader: - kv   8:              llama.attention.head_count_kv u32     \n",
      "llama_model_loader: - kv   9:     llama.attention.layer_norm_rms_epsilon f32     \n",
      "llama_model_loader: - kv  10:                          general.file_type u32     \n",
      "llama_model_loader: - kv  11:                       tokenizer.ggml.model str     \n",
      "llama_model_loader: - kv  12:                      tokenizer.ggml.tokens arr     \n",
      "llama_model_loader: - kv  13:                      tokenizer.ggml.scores arr     \n",
      "llama_model_loader: - kv  14:                  tokenizer.ggml.token_type arr     \n",
      "llama_model_loader: - kv  15:                tokenizer.ggml.bos_token_id u32     \n",
      "llama_model_loader: - kv  16:                tokenizer.ggml.eos_token_id u32     \n",
      "llama_model_loader: - kv  17:            tokenizer.ggml.unknown_token_id u32     \n",
      "llama_model_loader: - kv  18:               general.quantization_version u32     \n",
      "llama_model_loader: - type  f32:  161 tensors\n",
      "llama_model_loader: - type q5_K:  481 tensors\n",
      "llama_model_loader: - type q6_K:   81 tensors\n",
      "llm_load_print_meta: format         = GGUF V2 (latest)\n",
      "llm_load_print_meta: arch           = llama\n",
      "llm_load_print_meta: vocab type     = SPM\n",
      "llm_load_print_meta: n_vocab        = 32000\n",
      "llm_load_print_meta: n_merges       = 0\n",
      "llm_load_print_meta: n_ctx_train    = 4096\n",
      "llm_load_print_meta: n_ctx          = 4096\n",
      "llm_load_print_meta: n_embd         = 8192\n",
      "llm_load_print_meta: n_head         = 64\n",
      "llm_load_print_meta: n_head_kv      = 8\n",
      "llm_load_print_meta: n_layer        = 80\n",
      "llm_load_print_meta: n_rot          = 128\n",
      "llm_load_print_meta: n_gqa          = 8\n",
      "llm_load_print_meta: f_norm_eps     = 1.0e-05\n",
      "llm_load_print_meta: f_norm_rms_eps = 1.0e-05\n",
      "llm_load_print_meta: n_ff           = 28672\n",
      "llm_load_print_meta: freq_base      = 10000.0\n",
      "llm_load_print_meta: freq_scale     = 1\n",
      "llm_load_print_meta: model type     = 70B\n",
      "llm_load_print_meta: model ftype    = mostly Q5_K - Medium\n",
      "llm_load_print_meta: model size     = 68.98 B\n",
      "llm_load_print_meta: general.name   = LLaMA v2\n",
      "llm_load_print_meta: BOS token = 1 '<s>'\n",
      "llm_load_print_meta: EOS token = 2 '</s>'\n",
      "llm_load_print_meta: UNK token = 0 '<unk>'\n",
      "llm_load_print_meta: LF token  = 13 '<0x0A>'\n",
      "llm_load_tensors: ggml ctx size =    0.23 MB\n",
      "llm_load_tensors: mem required  = 46494.72 MB (+ 1280.00 MB per state)\n",
      "....................................................................................................\n",
      "llama_new_context_with_model: kv self size  = 1280.00 MB\n",
      "ggml_metal_init: allocating\n",
      "ggml_metal_init: found device: Apple M2 Max\n",
      "ggml_metal_init: picking default device: Apple M2 Max\n",
      "ggml_metal_init: loading '/Users/gburns/miniconda3/envs/alhazen/lib/python3.11/site-packages/llama_cpp/ggml-metal.metal'\n",
      "ggml_metal_init: loaded kernel_add                            0x1203b6a00 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_add_row                        0x1203b8500 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul                            0x1203b7960 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_row                        0x1203b8c00 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_scale                          0x1203b9370 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_silu                           0x1203b99b0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_relu                           0x1203b7bc0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_gelu                           0x1203ba610 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_soft_max                       0x1203b7e20 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_soft_max_4                     0x1203bb380 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_diag_mask_inf                  0x1203bba10 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_diag_mask_inf_8                0x1203bc8a0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_f16                   0x1203bac50 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_0                  0x1203bcfa0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_1                  0x1203bd660 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q8_0                  0x1203be450 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q2_K                  0x1203beaf0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q3_K                  0x1203bf230 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q4_K                  0x1203bf8d0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q5_K                  0x1203c0010 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_get_rows_q6_K                  0x1203c0720 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_rms_norm                       0x1203c0e40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_norm                           0x1203c1550 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32                0x1203c1ff0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32_1row           0x1203c27a0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_f16_f32_l4             0x1203c30e0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_0_f32               0x1203c38a0 | th_max =  896 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_1_f32               0x1203c41f0 | th_max =  896 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q8_0_f32               0x1203c4a30 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q2_K_f32               0x1203c54d0 | th_max =  640 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q3_K_f32               0x1203c5d10 | th_max =  576 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q4_K_f32               0x1203c6530 | th_max =  576 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q5_K_f32               0x1203c6cf0 | th_max =  640 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mat_q6_K_f32               0x1203c7510 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_f16_f32                 0x1203c7d40 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_0_f32                0x1203c8500 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q8_0_f32                0x1203c8c90 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_1_f32                0x1203c9490 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q2_K_f32                0x1203c9c90 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q3_K_f32                0x1203ca490 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q4_K_f32                0x1203cac90 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q5_K_f32                0x1203cb490 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_mul_mm_q6_K_f32                0x1203cbc90 | th_max =  768 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_rope                           0x1203cc210 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_alibi_f32                      0x1203ccc40 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f32_f16                    0x1203cd4f0 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f32_f32                    0x1203ce080 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: loaded kernel_cpy_f16_f16                    0x1203cec10 | th_max = 1024 | th_width =   32\n",
      "ggml_metal_init: hasUnifiedMemory              = true\n",
      "ggml_metal_init: recommendedMaxWorkingSetSize  = 73728.00 MB\n",
      "ggml_metal_init: maxTransferRate               = built-in GPU\n",
      "llama_new_context_with_model: compute buffer total size =  565.47 MB\n",
      "llama_new_context_with_model: max tensor size =   205.08 MB\n",
      "ggml_metal_add_buffer: allocated 'data            ' buffer, size = 46495.22 MB, (46495.72 / 73728.00)\n",
      "ggml_metal_add_buffer: allocated 'eval            ' buffer, size =     1.48 MB, (46497.20 / 73728.00)\n",
      "ggml_metal_add_buffer: allocated 'kv              ' buffer, size =  1282.00 MB, (47779.20 / 73728.00)\n",
      "AVX = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 0 | NEON = 1 | ARM_FMA = 1 | F16C = 0 | FP16_VA = 1 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 0 | SSSE3 = 0 | VSX = 0 | \n",
      "ggml_metal_add_buffer: allocated 'alloc           ' buffer, size =   564.02 MB, (48343.22 / 73728.00)\n"
     ]
    }
   ],
   "source": [
    "from langchain.llms import LlamaCpp\n",
    "from langchain import PromptTemplate, LLMChain\n",
    "from langchain.callbacks.manager import CallbackManager\n",
    "from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler\n",
    "\n",
    "# Callbacks support token-wise streaming\n",
    "callback_manager = CallbackManager([StreamingStdOutCallbackHandler()])\n",
    "\n",
    "n_gpu_layers = 1  # Change this value based on your model and your GPU VRAM pool.\n",
    "n_batch = 512  # Should be between 1 and n_ctx, consider the amount of VRAM in your GPU.\n",
    "\n",
    "# Make sure the model path is correct for your system!\n",
    "llm = LlamaCpp(\n",
    "    model_path=\"/Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf\",\n",
    "    n_ctx=4096,\n",
    "    n_gpu_layers=n_gpu_layers,\n",
    "    n_batch=n_batch,\n",
    "    callback_manager=callback_manager,\n",
    "    verbose=True, # Verbose is required to pass to the callback manager\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "from __future__ import annotations\n",
    "\n",
    "from abc import abstractmethod\n",
    "from typing import List\n",
    "import re\n",
    "\n",
    "from langchain.schema import BaseOutputParser, OutputParserException\n",
    "\n",
    "import json\n",
    "from json import JSONDecodeError\n",
    "from typing import Any, List\n",
    "\n",
    "class JsonEnclosedByTextOutputParser(BaseOutputParser[Any]):\n",
    "    \"\"\"Parse the output of an LLM call to a JSON object.\"\"\"\n",
    "\n",
    "    def parse(self, text: str) -> Any:\n",
    "        text = text.strip()\n",
    "        m = re.search('.*([\\[\\{](.|\\n)*[\\}\\]]).*', text, flags=re.M)\n",
    "        if m:\n",
    "            text1 = m.group(1)\n",
    "            try:\n",
    "                return json.loads(text1)\n",
    "            except JSONDecodeError as e:\n",
    "                return None\n",
    "                #raise OutputParserException(f\"Invalid json output: {text1} derived from {text}\") from e\n",
    "        else: \n",
    "            raise OutputParserException(f\"Could not find json-formatted data in: {text}\")\n",
    "\n",
    "    @property\n",
    "    def _type(self) -> str:\n",
    "        return \"json_enclosed_by_text_output_parser\"\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain import PromptTemplate\n",
    "from langchain.output_parsers import CommaSeparatedListOutputParser\n",
    "\n",
    "output_parser = JsonEnclosedByTextOutputParser()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>PARAGRAPH_ID</th>\n",
       "      <th>TAG</th>\n",
       "      <th>TAG_TREE</th>\n",
       "      <th>OFFSET</th>\n",
       "      <th>LENGTH</th>\n",
       "      <th>FIG_REF</th>\n",
       "      <th>PLAIN_TEXT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>3</td>\n",
       "      <td>p</td>\n",
       "      <td>Introduction</td>\n",
       "      <td>3282</td>\n",
       "      <td>1006</td>\n",
       "      <td></td>\n",
       "      <td>Cryo-Electron Tomography (...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>4</td>\n",
       "      <td>p</td>\n",
       "      <td>Introduction</td>\n",
       "      <td>4288</td>\n",
       "      <td>2166</td>\n",
       "      <td></td>\n",
       "      <td>A range of available semi-...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>5</td>\n",
       "      <td>p</td>\n",
       "      <td>Introduction</td>\n",
       "      <td>6454</td>\n",
       "      <td>825</td>\n",
       "      <td></td>\n",
       "      <td>Here we present DeePiCt (D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>8</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>7384</td>\n",
       "      <td>1500</td>\n",
       "      <td></td>\n",
       "      <td>DeePiCt is based on deep l...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>11</td>\n",
       "      <td>p</td>\n",
       "      <td>Results</td>\n",
       "      <td>9081</td>\n",
       "      <td>1328</td>\n",
       "      <td>fig1</td>\n",
       "      <td>a. The CNN U-Net architect...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>122</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>60254</td>\n",
       "      <td>921</td>\n",
       "      <td></td>\n",
       "      <td>Our DeePiCt workflow facil...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>123</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>61175</td>\n",
       "      <td>535</td>\n",
       "      <td></td>\n",
       "      <td>Here, we provide the first...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>124</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>61710</td>\n",
       "      <td>1434</td>\n",
       "      <td></td>\n",
       "      <td>Subtomogram averaging of t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>125</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>63144</td>\n",
       "      <td>1503</td>\n",
       "      <td></td>\n",
       "      <td>The analysis of DeePiCt's ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>10.1101/2022.04.12.488077</td>\n",
       "      <td>126</td>\n",
       "      <td>p</td>\n",
       "      <td>Discussion</td>\n",
       "      <td>64647</td>\n",
       "      <td>1598</td>\n",
       "      <td></td>\n",
       "      <td>Our workflow allows easy a...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>62 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          PMID  PARAGRAPH_ID TAG      TAG_TREE  OFFSET  \\\n",
       "3    10.1101/2022.04.12.488077             3   p  Introduction    3282   \n",
       "4    10.1101/2022.04.12.488077             4   p  Introduction    4288   \n",
       "5    10.1101/2022.04.12.488077             5   p  Introduction    6454   \n",
       "8    10.1101/2022.04.12.488077             8   p       Results    7384   \n",
       "11   10.1101/2022.04.12.488077            11   p       Results    9081   \n",
       "..                         ...           ...  ..           ...     ...   \n",
       "122  10.1101/2022.04.12.488077           122   p    Discussion   60254   \n",
       "123  10.1101/2022.04.12.488077           123   p    Discussion   61175   \n",
       "124  10.1101/2022.04.12.488077           124   p    Discussion   61710   \n",
       "125  10.1101/2022.04.12.488077           125   p    Discussion   63144   \n",
       "126  10.1101/2022.04.12.488077           126   p    Discussion   64647   \n",
       "\n",
       "     LENGTH FIG_REF                     PLAIN_TEXT  \n",
       "3      1006          Cryo-Electron Tomography (...  \n",
       "4      2166          A range of available semi-...  \n",
       "5       825          Here we present DeePiCt (D...  \n",
       "8      1500          DeePiCt is based on deep l...  \n",
       "11     1328    fig1  a. The CNN U-Net architect...  \n",
       "..      ...     ...                            ...  \n",
       "122     921          Our DeePiCt workflow facil...  \n",
       "123     535          Here, we provide the first...  \n",
       "124    1434          Subtomogram averaging of t...  \n",
       "125    1503          The analysis of DeePiCt's ...  \n",
       "126    1598          Our workflow allows easy a...  \n",
       "\n",
       "[62 rows x 8 columns]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#pd.set_option('display.max_rows', None)\n",
    "#pd.set_option('display.max_colwidth', 0)\n",
    "pd.set_option('display.max_rows', 10)\n",
    "pd.set_option('display.max_colwidth', 30)\n",
    "df.replace(r'\\n',  ' ', inplace=True) \n",
    "df['PLAIN_TEXT'] = df['PLAIN_TEXT'].str.wrap(50)\n",
    "df_p = df[df.TAG=='p']\n",
    "df_p\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'PromptTemplate' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb Cell 20\u001b[0m line \u001b[0;36m2\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m t3 \u001b[39m=\u001b[39m \u001b[39m\"\"\"\u001b[39m\u001b[39m[INST]\u001b[39m\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=1'>2</a>\u001b[0m \u001b[39m<<SYS>>\u001b[39m\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=2'>3</a>\u001b[0m \u001b[39mYour job is to carefully read an excerpt from a scientific research article and perform the task described below. \u001b[39m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=19'>20</a>\u001b[0m \u001b[39m[INST]\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=20'>21</a>\u001b[0m \u001b[39m\"\"\"\u001b[39m\n\u001b[0;32m---> <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=21'>22</a>\u001b[0m p3 \u001b[39m=\u001b[39m PromptTemplate(\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=22'>23</a>\u001b[0m     template\u001b[39m=\u001b[39mt3,\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=23'>24</a>\u001b[0m     input_variables\u001b[39m=\u001b[39m[\u001b[39m\"\u001b[39m\u001b[39mexcerpt\u001b[39m\u001b[39m\"\u001b[39m],\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X65sZmlsZQ%3D%3D?line=24'>25</a>\u001b[0m )\n",
      "\u001b[0;31mNameError\u001b[0m: name 'PromptTemplate' is not defined"
     ]
    }
   ],
   "source": [
    "t3 = \"\"\"[INST]\n",
    "<<SYS>>\n",
    "Your job is to carefully read an excerpt from a scientific research article and perform the task described below. \n",
    "The text of the excerpt will be delimited with triple backticks. \n",
    "\n",
    "Be precise in your answers. \n",
    "<</SYS>>\n",
    "\n",
    "1. Split the text of the excerpt into separate sentences.\n",
    "\n",
    "2. Identify if there are any sentences in the excerpt that describe the process of annotating a dataset obtained using electron microsopy. \n",
    "If there no such sentences, return an empty list.\n",
    "\n",
    "3. If there are any sentences that describe how annotations were obtained, write a short summary of the methods used. \n",
    "If there no such sentences, respond with \"No methods described.\"\n",
    "\n",
    "In addition, explain your reasoning briefly in one or two sentences.\n",
    "\n",
    "excerpt: '''{excerpt}'''\n",
    "[INST]\n",
    "\"\"\"\n",
    "p3 = PromptTemplate(\n",
    "    template=t3,\n",
    "    input_variables=[\"excerpt\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "22"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(df[df.PLAIN_TEXT.str.contains('[Aa]nnotation')])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "clear out\n"
     ]
    }
   ],
   "source": [
    "#for i, row in df[df.PLAIN_TEXT.str.contains('[Aa]nnotation')].iterrows():\n",
    "#    prompt = p3.format(excerpt=row.PLAIN_TEXT)\n",
    "#    output = llm(prompt)\n",
    "print('clear out')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'DataFrame' object has no attribute 'ID_PAPER'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m/var/folders/pl/z8kd7hxd3qv9_0f1_ndg3hxm0000gq/T/ipykernel_40766/681896153.py\u001b[0m in \u001b[0;36m?\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;31m# check\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdf\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdf\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mID_PAPER\u001b[0m\u001b[0;34m==\u001b[0m\u001b[0;34m\"28945706\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m \u001b[0ma\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mABSTRACT\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvalues\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0mt\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mTITLE\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvalues\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0mprompt\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mp2\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mformat\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtitle\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mt\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mabstract\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/site-packages/pandas/core/generic.py\u001b[0m in \u001b[0;36m?\u001b[0;34m(self, name)\u001b[0m\n\u001b[1;32m   6198\u001b[0m             \u001b[0;32mand\u001b[0m \u001b[0mname\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_accessors\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   6199\u001b[0m             \u001b[0;32mand\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_info_axis\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_can_hold_identifiers_and_holds_name\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   6200\u001b[0m         ):\n\u001b[1;32m   6201\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 6202\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0mobject\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m__getattribute__\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mname\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m: 'DataFrame' object has no attribute 'ID_PAPER'"
     ]
    }
   ],
   "source": [
    "# check\n",
    "r = df[df.ID_PAPER==\"28945706\"]\n",
    "a = r.ABSTRACT.values[0]\n",
    "t = r.TITLE.values[0]\n",
    "prompt = p2.format(title=t, abstract=a)\n",
    "print(prompt)\n",
    "output = llm(p3.format(title=t, abstract=a))\n",
    "#m = output_parser.parse(output)\n",
    "\n",
    "print('\\n\\n\\n'+output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "llama_model_loader: loaded meta data with 19 key-value pairs and 723 tensors from /Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf (version GGUF V2 (latest))\n",
      "llama_model_loader: - tensor    0:                token_embd.weight q5_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - tensor    1:           blk.0.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    2:            blk.0.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    3:            blk.0.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    4:              blk.0.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor    5:            blk.0.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor    6:              blk.0.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor    7:         blk.0.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    8:              blk.0.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor    9:              blk.0.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   10:           blk.1.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   11:            blk.1.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   12:            blk.1.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   13:              blk.1.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   14:            blk.1.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   15:              blk.1.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   16:         blk.1.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   17:              blk.1.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   18:              blk.1.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   19:           blk.2.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   20:            blk.2.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   21:            blk.2.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   22:              blk.2.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   23:            blk.2.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   24:              blk.2.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   25:         blk.2.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   26:              blk.2.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   27:              blk.2.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   28:           blk.3.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   29:            blk.3.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   30:            blk.3.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   31:              blk.3.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   32:            blk.3.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   33:              blk.3.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   34:         blk.3.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   35:              blk.3.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   36:              blk.3.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   37:           blk.4.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   38:            blk.4.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   39:            blk.4.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   40:              blk.4.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   41:            blk.4.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   42:              blk.4.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   43:         blk.4.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   44:              blk.4.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   45:              blk.4.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   46:            blk.5.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   47:              blk.5.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   48:         blk.5.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   49:              blk.5.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   50:              blk.5.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   51:          blk.10.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   52:           blk.10.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   53:           blk.10.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   54:             blk.10.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   55:           blk.10.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   56:             blk.10.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   57:        blk.10.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   58:             blk.10.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   59:             blk.10.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   60:             blk.11.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   61:        blk.11.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   62:             blk.11.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   63:             blk.11.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   64:           blk.5.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   65:            blk.5.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   66:              blk.5.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   67:            blk.5.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   68:           blk.6.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   69:            blk.6.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   70:            blk.6.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   71:              blk.6.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   72:            blk.6.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   73:              blk.6.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   74:         blk.6.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   75:              blk.6.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   76:              blk.6.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   77:           blk.7.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   78:            blk.7.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   79:            blk.7.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   80:              blk.7.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   81:            blk.7.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   82:              blk.7.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   83:         blk.7.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   84:              blk.7.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   85:              blk.7.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   86:           blk.8.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   87:            blk.8.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   88:            blk.8.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   89:              blk.8.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   90:            blk.8.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   91:              blk.8.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   92:         blk.8.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   93:              blk.8.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   94:              blk.8.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor   95:           blk.9.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor   96:            blk.9.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor   97:            blk.9.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   98:              blk.9.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor   99:            blk.9.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  100:              blk.9.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  101:         blk.9.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  102:              blk.9.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  103:              blk.9.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  104:          blk.11.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  105:           blk.11.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  106:           blk.11.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  107:             blk.11.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  108:           blk.11.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  109:          blk.12.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  110:           blk.12.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  111:           blk.12.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  112:             blk.12.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  113:           blk.12.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  114:             blk.12.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  115:        blk.12.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  116:             blk.12.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  117:             blk.12.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  118:          blk.13.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  119:           blk.13.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  120:           blk.13.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  121:             blk.13.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  122:           blk.13.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  123:             blk.13.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  124:        blk.13.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  125:             blk.13.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  126:             blk.13.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  127:          blk.14.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  128:           blk.14.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  129:           blk.14.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  130:             blk.14.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  131:           blk.14.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  132:             blk.14.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  133:        blk.14.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  134:             blk.14.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  135:             blk.14.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  136:          blk.15.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  137:           blk.15.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  138:           blk.15.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  139:             blk.15.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  140:           blk.15.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  141:             blk.15.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  142:        blk.15.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  143:             blk.15.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  144:             blk.15.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  145:          blk.16.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  146:           blk.16.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  147:           blk.16.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  148:             blk.16.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  149:           blk.16.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  150:             blk.16.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  151:        blk.16.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  152:             blk.16.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  153:             blk.16.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  154:          blk.17.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  155:           blk.17.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  156:           blk.17.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  157:             blk.17.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  158:           blk.17.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  159:             blk.17.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  160:        blk.17.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  161:             blk.17.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  162:             blk.17.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  163:          blk.18.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  164:           blk.18.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  165:           blk.18.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  166:             blk.18.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  167:           blk.18.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  168:             blk.18.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  169:        blk.18.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  170:             blk.18.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  171:             blk.18.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  172:          blk.19.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  173:           blk.19.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  174:           blk.19.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  175:             blk.19.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  176:           blk.19.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  177:             blk.19.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  178:        blk.19.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  179:             blk.19.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  180:             blk.19.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  181:          blk.20.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  182:           blk.20.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  183:           blk.20.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  184:             blk.20.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  185:           blk.20.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  186:             blk.20.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  187:        blk.20.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  188:             blk.20.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  189:             blk.20.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  190:          blk.21.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  191:           blk.21.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  192:           blk.21.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  193:             blk.21.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  194:           blk.21.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  195:             blk.21.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  196:        blk.21.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  197:             blk.21.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  198:             blk.21.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  199:           blk.22.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  200:             blk.22.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  201:             blk.22.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  202:        blk.22.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  203:             blk.22.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  204:             blk.22.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  205:          blk.22.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  206:           blk.22.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  207:           blk.22.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  208:          blk.23.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  209:           blk.23.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  210:           blk.23.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  211:             blk.23.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  212:           blk.23.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  213:             blk.23.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  214:        blk.23.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  215:             blk.23.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  216:             blk.23.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  217:          blk.24.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  218:           blk.24.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  219:           blk.24.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  220:             blk.24.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  221:           blk.24.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  222:             blk.24.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  223:        blk.24.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  224:             blk.24.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  225:             blk.24.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  226:          blk.25.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  227:           blk.25.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  228:           blk.25.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  229:             blk.25.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  230:           blk.25.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  231:             blk.25.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  232:        blk.25.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  233:             blk.25.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  234:             blk.25.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  235:          blk.26.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  236:           blk.26.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  237:           blk.26.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  238:             blk.26.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  239:           blk.26.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  240:             blk.26.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  241:        blk.26.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  242:             blk.26.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  243:             blk.26.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  244:          blk.27.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  245:           blk.27.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  246:           blk.27.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  247:             blk.27.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  248:           blk.27.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  249:             blk.27.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  250:        blk.27.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  251:             blk.27.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  252:             blk.27.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  253:           blk.28.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  254:             blk.28.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  255:        blk.28.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  256:             blk.28.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  257:             blk.28.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  258:          blk.28.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  259:           blk.28.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  260:             blk.28.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  261:           blk.28.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  262:          blk.29.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  263:           blk.29.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  264:           blk.29.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  265:             blk.29.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  266:           blk.29.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  267:             blk.29.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  268:        blk.29.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  269:             blk.29.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  270:             blk.29.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  271:          blk.30.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  272:           blk.30.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  273:           blk.30.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  274:             blk.30.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  275:           blk.30.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  276:             blk.30.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  277:        blk.30.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  278:             blk.30.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  279:             blk.30.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  280:          blk.31.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  281:           blk.31.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  282:           blk.31.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  283:             blk.31.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  284:           blk.31.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  285:             blk.31.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  286:        blk.31.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  287:             blk.31.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  288:             blk.31.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  289:          blk.32.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  290:           blk.32.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  291:           blk.32.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  292:             blk.32.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  293:           blk.32.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  294:             blk.32.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  295:        blk.32.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  296:             blk.32.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  297:             blk.32.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  298:          blk.33.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  299:           blk.33.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  300:           blk.33.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  301:             blk.33.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  302:           blk.33.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  303:             blk.33.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  304:        blk.33.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  305:             blk.33.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  306:             blk.33.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  307:             blk.34.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  308:        blk.34.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  309:             blk.34.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  310:             blk.34.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  311:          blk.34.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  312:           blk.34.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  313:           blk.34.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  314:             blk.34.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  315:           blk.34.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  316:          blk.35.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  317:           blk.35.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  318:           blk.35.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  319:             blk.35.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  320:           blk.35.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  321:             blk.35.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  322:        blk.35.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  323:             blk.35.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  324:             blk.35.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  325:          blk.36.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  326:           blk.36.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  327:           blk.36.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  328:             blk.36.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  329:           blk.36.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  330:             blk.36.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  331:        blk.36.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  332:             blk.36.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  333:             blk.36.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  334:          blk.37.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  335:           blk.37.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  336:           blk.37.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  337:             blk.37.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  338:           blk.37.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  339:             blk.37.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  340:        blk.37.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  341:             blk.37.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  342:             blk.37.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  343:          blk.38.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  344:           blk.38.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  345:           blk.38.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  346:             blk.38.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  347:           blk.38.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  348:             blk.38.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  349:        blk.38.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  350:             blk.38.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  351:             blk.38.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  352:          blk.39.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  353:           blk.39.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  354:           blk.39.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  355:             blk.39.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  356:           blk.39.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  357:             blk.39.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  358:        blk.39.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  359:             blk.39.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  360:             blk.39.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  361:          blk.40.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  362:           blk.40.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  363:           blk.40.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  364:             blk.40.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  365:           blk.40.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  366:             blk.40.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  367:        blk.40.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  368:             blk.40.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  369:             blk.40.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  370:          blk.41.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  371:           blk.41.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  372:           blk.41.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  373:             blk.41.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  374:           blk.41.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  375:             blk.41.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  376:        blk.41.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  377:             blk.41.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  378:             blk.41.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  379:          blk.42.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  380:           blk.42.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  381:           blk.42.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  382:             blk.42.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  383:           blk.42.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  384:             blk.42.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  385:        blk.42.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  386:             blk.42.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  387:             blk.42.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  388:          blk.43.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  389:           blk.43.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  390:           blk.43.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  391:             blk.43.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  392:           blk.43.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  393:             blk.43.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  394:        blk.43.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  395:             blk.43.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  396:             blk.43.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  397:          blk.44.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  398:           blk.44.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  399:           blk.44.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  400:             blk.44.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  401:           blk.44.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  402:             blk.44.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  403:        blk.44.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  404:             blk.44.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  405:             blk.44.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  406:           blk.45.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  407:             blk.45.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  408:             blk.45.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  409:        blk.45.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  410:             blk.45.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  411:             blk.45.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  412:          blk.45.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  413:           blk.45.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  414:           blk.45.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  415:          blk.46.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  416:           blk.46.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  417:           blk.46.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  418:             blk.46.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  419:           blk.46.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  420:             blk.46.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  421:        blk.46.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  422:             blk.46.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  423:             blk.46.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  424:          blk.47.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  425:           blk.47.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  426:           blk.47.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  427:             blk.47.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  428:           blk.47.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  429:             blk.47.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  430:        blk.47.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  431:             blk.47.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  432:             blk.47.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  433:          blk.48.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  434:           blk.48.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  435:           blk.48.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  436:             blk.48.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  437:           blk.48.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  438:             blk.48.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  439:        blk.48.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  440:             blk.48.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  441:             blk.48.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  442:          blk.49.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  443:           blk.49.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  444:           blk.49.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  445:             blk.49.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  446:           blk.49.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  447:             blk.49.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  448:        blk.49.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  449:             blk.49.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  450:             blk.49.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  451:          blk.50.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  452:           blk.50.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  453:           blk.50.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  454:             blk.50.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  455:           blk.50.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  456:             blk.50.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  457:        blk.50.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  458:             blk.50.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  459:             blk.50.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  460:           blk.51.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  461:             blk.51.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  462:        blk.51.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  463:             blk.51.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  464:             blk.51.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  465:          blk.51.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  466:           blk.51.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  467:             blk.51.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  468:           blk.51.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  469:          blk.52.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  470:           blk.52.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  471:           blk.52.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  472:             blk.52.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  473:           blk.52.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  474:             blk.52.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  475:        blk.52.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  476:             blk.52.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  477:             blk.52.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  478:          blk.53.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  479:           blk.53.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  480:           blk.53.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  481:             blk.53.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  482:           blk.53.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  483:             blk.53.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  484:        blk.53.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  485:             blk.53.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  486:             blk.53.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  487:          blk.54.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  488:           blk.54.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  489:           blk.54.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  490:             blk.54.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  491:           blk.54.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  492:             blk.54.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  493:        blk.54.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  494:             blk.54.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  495:             blk.54.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  496:          blk.55.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  497:           blk.55.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  498:           blk.55.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  499:             blk.55.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  500:           blk.55.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  501:             blk.55.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  502:        blk.55.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  503:             blk.55.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  504:             blk.55.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  505:          blk.56.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  506:           blk.56.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  507:           blk.56.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  508:             blk.56.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  509:           blk.56.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  510:             blk.56.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  511:        blk.56.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  512:             blk.56.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  513:             blk.56.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  514:             blk.57.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  515:        blk.57.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  516:             blk.57.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  517:             blk.57.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  518:          blk.57.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  519:           blk.57.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  520:           blk.57.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  521:             blk.57.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  522:           blk.57.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  523:          blk.58.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  524:           blk.58.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  525:           blk.58.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  526:             blk.58.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  527:           blk.58.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  528:             blk.58.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  529:        blk.58.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  530:             blk.58.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  531:             blk.58.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  532:          blk.59.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  533:           blk.59.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  534:           blk.59.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  535:             blk.59.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  536:           blk.59.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  537:             blk.59.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  538:        blk.59.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  539:             blk.59.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  540:             blk.59.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  541:          blk.60.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  542:           blk.60.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  543:           blk.60.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  544:             blk.60.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  545:           blk.60.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  546:             blk.60.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  547:        blk.60.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  548:             blk.60.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  549:             blk.60.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  550:          blk.61.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  551:           blk.61.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  552:           blk.61.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  553:             blk.61.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  554:           blk.61.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  555:             blk.61.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  556:        blk.61.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  557:             blk.61.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  558:             blk.61.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  559:          blk.62.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  560:           blk.62.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  561:           blk.62.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  562:             blk.62.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  563:           blk.62.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  564:             blk.62.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  565:        blk.62.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  566:             blk.62.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  567:             blk.62.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  568:          blk.63.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  569:           blk.63.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  570:           blk.63.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  571:             blk.63.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  572:           blk.63.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  573:             blk.63.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  574:        blk.63.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  575:             blk.63.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  576:             blk.63.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  577:          blk.64.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  578:           blk.64.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  579:           blk.64.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  580:             blk.64.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  581:           blk.64.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  582:             blk.64.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  583:        blk.64.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  584:             blk.64.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  585:             blk.64.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  586:          blk.65.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  587:           blk.65.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  588:           blk.65.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  589:             blk.65.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  590:           blk.65.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  591:             blk.65.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  592:        blk.65.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  593:             blk.65.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  594:             blk.65.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  595:          blk.66.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  596:           blk.66.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  597:           blk.66.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  598:             blk.66.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  599:           blk.66.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  600:             blk.66.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  601:        blk.66.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  602:             blk.66.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  603:             blk.66.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  604:          blk.67.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  605:           blk.67.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  606:           blk.67.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  607:             blk.67.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  608:           blk.67.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  609:             blk.67.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  610:        blk.67.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  611:             blk.67.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  612:             blk.67.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  613:           blk.68.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  614:             blk.68.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  615:             blk.68.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  616:        blk.68.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  617:             blk.68.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  618:             blk.68.attn_v.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  619:          blk.68.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  620:           blk.68.ffn_down.weight q5_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  621:           blk.68.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  622:          blk.69.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  623:           blk.69.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  624:           blk.69.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  625:             blk.69.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  626:           blk.69.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  627:             blk.69.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  628:        blk.69.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  629:             blk.69.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  630:             blk.69.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  631:          blk.70.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  632:           blk.70.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  633:           blk.70.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  634:             blk.70.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  635:           blk.70.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  636:             blk.70.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  637:        blk.70.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  638:             blk.70.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  639:             blk.70.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  640:          blk.71.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  641:           blk.71.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  642:           blk.71.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  643:             blk.71.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  644:           blk.71.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  645:             blk.71.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  646:        blk.71.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  647:             blk.71.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  648:             blk.71.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  649:          blk.72.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  650:           blk.72.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  651:           blk.72.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  652:             blk.72.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  653:           blk.72.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  654:             blk.72.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  655:        blk.72.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  656:             blk.72.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  657:             blk.72.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  658:          blk.73.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  659:           blk.73.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  660:           blk.73.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  661:             blk.73.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  662:           blk.73.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  663:             blk.73.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  664:        blk.73.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  665:             blk.73.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  666:             blk.73.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  667:           blk.74.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  668:             blk.74.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  669:        blk.74.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  670:             blk.74.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  671:             blk.74.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  672:          blk.74.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  673:           blk.74.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  674:             blk.74.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  675:           blk.74.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  676:          blk.75.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  677:           blk.75.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  678:           blk.75.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  679:             blk.75.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  680:           blk.75.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  681:             blk.75.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  682:        blk.75.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  683:             blk.75.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  684:             blk.75.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  685:          blk.76.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  686:           blk.76.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  687:           blk.76.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  688:             blk.76.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  689:           blk.76.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  690:             blk.76.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  691:        blk.76.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  692:             blk.76.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  693:             blk.76.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  694:          blk.77.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  695:           blk.77.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  696:           blk.77.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  697:             blk.77.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  698:           blk.77.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  699:             blk.77.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  700:        blk.77.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  701:             blk.77.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  702:             blk.77.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  703:          blk.78.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  704:           blk.78.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  705:           blk.78.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  706:             blk.78.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  707:           blk.78.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  708:             blk.78.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  709:        blk.78.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  710:             blk.78.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  711:             blk.78.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  712:          blk.79.attn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  713:           blk.79.ffn_down.weight q6_K     [ 28672,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  714:           blk.79.ffn_gate.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  715:             blk.79.ffn_up.weight q5_K     [  8192, 28672,     1,     1 ]\n",
      "llama_model_loader: - tensor  716:           blk.79.ffn_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  717:             blk.79.attn_k.weight q5_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  718:        blk.79.attn_output.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  719:             blk.79.attn_q.weight q5_K     [  8192,  8192,     1,     1 ]\n",
      "llama_model_loader: - tensor  720:             blk.79.attn_v.weight q6_K     [  8192,  1024,     1,     1 ]\n",
      "llama_model_loader: - tensor  721:               output_norm.weight f32      [  8192,     1,     1,     1 ]\n",
      "llama_model_loader: - tensor  722:                    output.weight q6_K     [  8192, 32000,     1,     1 ]\n",
      "llama_model_loader: - kv   0:                       general.architecture str     \n",
      "llama_model_loader: - kv   1:                               general.name str     \n",
      "llama_model_loader: - kv   2:                       llama.context_length u32     \n",
      "llama_model_loader: - kv   3:                     llama.embedding_length u32     \n",
      "llama_model_loader: - kv   4:                          llama.block_count u32     \n",
      "llama_model_loader: - kv   5:                  llama.feed_forward_length u32     \n",
      "llama_model_loader: - kv   6:                 llama.rope.dimension_count u32     \n",
      "llama_model_loader: - kv   7:                 llama.attention.head_count u32     \n",
      "llama_model_loader: - kv   8:              llama.attention.head_count_kv u32     \n",
      "llama_model_loader: - kv   9:     llama.attention.layer_norm_rms_epsilon f32     \n",
      "llama_model_loader: - kv  10:                          general.file_type u32     \n",
      "llama_model_loader: - kv  11:                       tokenizer.ggml.model str     \n",
      "llama_model_loader: - kv  12:                      tokenizer.ggml.tokens arr     \n",
      "llama_model_loader: - kv  13:                      tokenizer.ggml.scores arr     \n",
      "llama_model_loader: - kv  14:                  tokenizer.ggml.token_type arr     \n",
      "llama_model_loader: - kv  15:                tokenizer.ggml.bos_token_id u32     \n",
      "llama_model_loader: - kv  16:                tokenizer.ggml.eos_token_id u32     \n",
      "llama_model_loader: - kv  17:            tokenizer.ggml.unknown_token_id u32     \n",
      "llama_model_loader: - kv  18:               general.quantization_version u32     \n",
      "llama_model_loader: - type  f32:  161 tensors\n",
      "llama_model_loader: - type q5_K:  481 tensors\n",
      "llama_model_loader: - type q6_K:   81 tensors\n",
      "llm_load_print_meta: format         = GGUF V2 (latest)\n",
      "llm_load_print_meta: arch           = llama\n",
      "llm_load_print_meta: vocab type     = SPM\n",
      "llm_load_print_meta: n_vocab        = 32000\n",
      "llm_load_print_meta: n_merges       = 0\n",
      "llm_load_print_meta: n_ctx_train    = 4096\n",
      "llm_load_print_meta: n_ctx          = 512\n",
      "llm_load_print_meta: n_embd         = 8192\n",
      "llm_load_print_meta: n_head         = 64\n",
      "llm_load_print_meta: n_head_kv      = 8\n",
      "llm_load_print_meta: n_layer        = 80\n",
      "llm_load_print_meta: n_rot          = 128\n",
      "llm_load_print_meta: n_gqa          = 8\n",
      "llm_load_print_meta: f_norm_eps     = 1.0e-05\n",
      "llm_load_print_meta: f_norm_rms_eps = 1.0e-05\n",
      "llm_load_print_meta: n_ff           = 28672\n",
      "llm_load_print_meta: freq_base      = 10000.0\n",
      "llm_load_print_meta: freq_scale     = 1\n",
      "llm_load_print_meta: model type     = 70B\n",
      "llm_load_print_meta: model ftype    = mostly Q5_K - Medium\n",
      "llm_load_print_meta: model size     = 68.98 B\n",
      "llm_load_print_meta: general.name   = LLaMA v2\n",
      "llm_load_print_meta: BOS token = 1 '<s>'\n",
      "llm_load_print_meta: EOS token = 2 '</s>'\n",
      "llm_load_print_meta: UNK token = 0 '<unk>'\n",
      "llm_load_print_meta: LF token  = 13 '<0x0A>'\n",
      "llm_load_tensors: ggml ctx size =    0.23 MB\n",
      "llm_load_tensors: mem required  = 46494.72 MB (+  320.00 MB per state)\n",
      "....................................................................................................\n",
      "llama_new_context_with_model: kv self size  =  320.00 MB\n",
      "llama_new_context_with_model: compute buffer total size =  145.47 MB\n",
      "AVX = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 0 | NEON = 1 | ARM_FMA = 1 | F16C = 0 | FP16_VA = 1 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 0 | SSSE3 = 0 | VSX = 0 | \n"
     ]
    }
   ],
   "source": [
    "\n",
    "os.environ[\"WEAVIATE_URL\"] = 'http://localhost:8080/'\n",
    "\n",
    "import weaviate\n",
    "from langchain.vectorstores import Weaviate\n",
    "from langchain.embeddings.llamacpp import LlamaCppEmbeddings\n",
    "\n",
    "client = weaviate.Client(url=os.environ[\"WEAVIATE_URL\"])\n",
    "docs = [\n",
    "    Document(page_content=\"foo\", metadata={\"id\": 1}),\n",
    "]\n",
    "embeddings = LlamaCppEmbeddings(model_path=\"/Users/gburns/Documents/Coding/ChatGPT_etc/LLMs/llama-2-70b-chat.Q5_K_M.gguf\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "client = weaviate.Client(os.environ[\"WEAVIATE_URL\"])\n",
    "\n",
    "w = Weaviate.from_documents(blocks, embeddings, weaviate_url=os.environ[\"WEAVIATE_URL\"], by_text=False, index_name=\"MyIndex\")\n",
    "w.similarity_search_by_text(query=\"annotations\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "module 'sqlalchemy' has no attribute 'similarity_search_by_text'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[1;32m/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb Cell 25\u001b[0m line \u001b[0;36m1\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/gburns/Documents/Coding/ChatGPT_etc/alzhazen/nb_scratchpad/95_full_text_reading.ipynb#X34sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m db\u001b[39m.\u001b[39msimilarity_search_by_text(query\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mannotations\u001b[39m\u001b[39m\"\u001b[39m)\n",
      "\u001b[0;31mAttributeError\u001b[0m: module 'sqlalchemy' has no attribute 'similarity_search_by_text'"
     ]
    }
   ],
   "source": [
    "db.similarity_search_by_text(query=\"annotations\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "python3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
