{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Cell Biology Database.  \n",
    "\n",
    "> Applying AI to understand trends of research."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Preliminaries\n",
    "\n",
    "Here we set up libraries and methods to create and query the local Postgres database we will be using to store our information from the Alhazen tools and agent"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from alhazen.aliases import *\n",
    "from alhazen.core import lookup_chat_models\n",
    "from alhazen.agent import AlhazenAgent\n",
    "from alhazen.tools.basic import *\n",
    "from alhazen.tools.paperqa_emulation_tool import *\n",
    "from alhazen.toolkit import *\n",
    "\n",
    "from alhazen.utils.ceifns_db import Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, list_databases\n",
    "from alhazen.utils.searchEngineUtils import *\n",
    "\n",
    "from datetime import datetime\n",
    "\n",
    "from importlib_resources import files\n",
    "import os\n",
    "import pandas as pd\n",
    "\n",
    "from sqlalchemy import func, text\n",
    "\n",
    "from time import time\n",
    "from tqdm import tqdm\n",
    "\n",
    "from transformers import pipeline, AutoModel, AutoTokenizer\n",
    "from transformers import pipeline, AutoModel, AutoTokenizer\n",
    "import torch\n",
    "import os\n",
    "import torch\n",
    "from transformers import pipeline\n",
    "\n",
    "from langchain.schema.runnable import RunnableParallel, RunnablePassthrough, RunnableLambda\n",
    "from operator import itemgetter\n",
    "from langchain.chat_models import ChatOllama\n",
    "from langchain.schema import get_buffer_string, OutputParserException, format_document\n",
    "from langchain.callbacks.tracers import ConsoleCallbackHandler\n",
    "from langchain_core.output_parsers import StrOutputParser, JsonOutputParser\n",
    "from langchain.prompts import ChatPromptTemplate, PromptTemplate\n",
    "from alhazen.utils.output_parsers import JsonEnclosedByTextOutputParser\n",
    "\n",
    "from langchain.schema import format_document\n",
    "from langchain_core.messages import AIMessage, HumanMessage, get_buffer_string\n",
    "from langchain_core.runnables import RunnableParallel"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Remember to set environmental variables for this code:\n",
    "\n",
    "* `ALHAZEN_DB_NAME` - the name of the Postgres database you are storing information into\n",
    "* `LOCAL_FILE_PATH` - the location on disk where you save files for your digital library, downloaded models or other data.   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "if os.environ.get('LOCAL_FILE_PATH') is None: \n",
    "    raise Exception('Where are you storing your local literature database?')\n",
    "if os.path.exists(os.environ['LOCAL_FILE_PATH']) is False:\n",
    "    os.makedirs(os.environ['LOCAL_FILE_PATH'])    \n",
    "\n",
    "loc = os.environ['LOCAL_FILE_PATH']\n",
    "db_name = 'cell_biology'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Run this command to destroy your current database \n",
    "\n",
    "**USE WITH CAUTION**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "drop_ceifns_database(os.environ['ALHAZEN_DB_NAME'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Run this command to create a new, empty database. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "create_ceifns_database(db_name)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This command lists all the tools the Alhazen agent system has access to"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "21d872844fcd47f5be0ff4c8cdce0c47",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       ".gitattributes:   0%|          | 0.00/1.52k [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "5198691168544971878edb7dc29ea321",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "1_Pooling/config.json:   0%|          | 0.00/191 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "ac395c87eda24db1a82ee17aea7f0ff7",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "README.md:   0%|          | 0.00/94.6k [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "bfc44cc17189494d9783bfeff9dbafef",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "config.json:   0%|          | 0.00/779 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "201a96f09afe4909a86ba488610c061e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "config_sentence_transformers.json:   0%|          | 0.00/124 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "8fe198c26a934c1a943e91a31a4c2db3",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "model.safetensors:   0%|          | 0.00/1.34G [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "e9125c2cf308401e85352499aaccec07",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "model.onnx:   0%|          | 0.00/1.34G [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "fff2d456a3f74b4ea2536792eddb6db3",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "pytorch_model.bin:   0%|          | 0.00/1.34G [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "87a4f9f4c77e465aa2501ed8a2b2c492",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "sentence_bert_config.json:   0%|          | 0.00/52.0 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "714c97d247ce4953b93206318472722f",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "special_tokens_map.json:   0%|          | 0.00/125 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "aff1595fb0874e75913b81e4ec7fd739",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "tokenizer.json:   0%|          | 0.00/711k [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "9d585786c6b14df2b5a6e545635d004c",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "tokenizer_config.json:   0%|          | 0.00/366 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "325f41d639be480899ebe601c15eb6ce",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "vocab.txt:   0%|          | 0.00/232k [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "a000861170f649938c9bb494be407398",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "modules.json:   0%|          | 0.00/349 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "0it [00:00, ?it/s]\n",
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py:293: LangChainPendingDeprecationWarning: Please use JSONB instead of JSON for metadata. This change will allow for more efficient querying that involves filtering based on metadata.Please note that filtering operators have been changed when using JSOB metadata to be prefixed with a $ sign to avoid name collisions with columns. If you're using an existing database, you will need to create adb migration for your metadata column to be JSONB and update your queries to use the new operators. \n",
      "  warn_deprecated(\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Indexing 0 documents\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_core/_api/deprecation.py:119: LangChainDeprecationWarning: The class `ChatOpenAI` was deprecated in LangChain 0.0.10 and will be removed in 0.2.0. An updated version of the class exists in the langchain-openai package and should be used instead. To use it run `pip install -U langchain-openai` and import as `from langchain_openai import ChatOpenAI`.\n",
      "  warn_deprecated(\n",
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_groq/chat_models.py:145: UserWarning: WARNING! stop is not default parameter.\n",
      "                    stop was transferred to model_kwargs.\n",
      "                    Please confirm that stop is what you intended.\n",
      "  warnings.warn(\n",
      "0it [00:00, ?it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Indexing 0 documents\n",
      "AGENT TOOLS\n",
      "\tAddCollectionFromEPMCTool\n",
      "\tAddAuthorsToCollectionTool\n",
      "\tDescribeCollectionCompositionTool\n",
      "\tDeleteCollectionTool\n",
      "\tRetrieveFullTextTool\n",
      "\tRetrieveFullTextToolForACollection\n",
      "\tMetadataExtraction_EverythingEverywhere_Tool\n",
      "\tMetadataExtraction_MethodsSectionOnly_Tool\n",
      "\tSimpleExtractionWithRAGTool\n",
      "\tPaperQAEmulationTool\n",
      "\tProcotolEntitiesExtractionTool\n",
      "\tCheckExpressionTool\n",
      "\tTitleAbstractClassifier_OneDocAtATime_Tool\n",
      "\tTitleAbstractDiscourseMappingTool\n",
      "\tTitleAbstractExtraction_OneDocAtATime_Tool\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py:293: LangChainPendingDeprecationWarning: Please use JSONB instead of JSON for metadata. This change will allow for more efficient querying that involves filtering based on metadata.Please note that filtering operators have been changed when using JSOB metadata to be prefixed with a $ sign to avoid name collisions with columns. If you're using an existing database, you will need to create adb migration for your metadata column to be JSONB and update your queries to use the new operators. \n",
      "  warn_deprecated(\n"
     ]
    }
   ],
   "source": [
    "ldb = Ceifns_LiteratureDb(loc=loc, name=db_name)\n",
    "\n",
    "llms = lookup_chat_models()\n",
    "llm_dbrx = llms.get('gpt4_1106')\n",
    "\n",
    "cb = AlhazenAgent(db_name=db_name, agent_llm=llm_dbrx, tool_llm=llm_dbrx)\n",
    "print('AGENT TOOLS')\n",
    "for t in cb.tk.get_tools():\n",
    "    print('\\t'+type(t).__name__)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Build paper collections\n",
    "\n",
    "This section will build a literature collection across each of the diseases in the Rare As One Cohorts for cycle 1 and 2. \n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "What diseases are we querying the literature for?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#\n",
    "# Note - create a new corpus for collaborative discussions with CellXGene team (in particular Maximillian L.)\n",
    "#\n",
    "addEMPCCollection_tool = [t for t in cb.tk.get_tools() if isinstance(t, AddCollectionFromEPMCTool)][0]\n",
    "id = 'SEC61B + Unfolded Protein Response'\n",
    "name = 'Question: What is the connection between SEC61B and the unfolded protein response?'\n",
    "query = 'What is the connection between SEC61B and the unfolded protein response?'\n",
    "addEMPCCollection_tool.run(tool_input={'id': id, 'name':name, 'query':query, 'full_text':False})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "addEMPCCollection_tool = [t for t in cb.tk.get_tools() if isinstance(t, AddCollectionFromEPMCTool)][0]\n",
    "id = 'SEC61B + Unfolded Protein Response'\n",
    "name = 'Question: What is the connection between SEC61B and the unfolded protein response?'\n",
    "query = 'What is the connection between SEC61B and the unfolded protein response?'\n",
    "addEMPCCollection_tool.run(tool_input={'id': id, 'name':name, 'query':query, 'full_text':False})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#\n",
    "# Note - create a new corpus for collaborative discussions with CellXGene team (in particular Maximillian L.)\n",
    "#\n",
    "retrieveFullTextTool_tool = [t for t in cb.tk.get_tools() if isinstance(t, RetrieveFullTextToolForACollection)][0]\n",
    "retrieveFullTextTool_tool.run(tool_input={'collection_id': id})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py:293: LangChainPendingDeprecationWarning: Please use JSONB instead of JSON for metadata. This change will allow for more efficient querying that involves filtering based on metadata.Please note that filtering operators have been changed when using JSOB metadata to be prefixed with a $ sign to avoid name collisions with columns. If you're using an existing database, you will need to create adb migration for your metadata column to be JSONB and update your queries to use the new operators. \n",
      "  warn_deprecated(\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 2:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 2:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 3:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 3:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 4:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 4:chain:RunnableSequence > 5:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 4:chain:RunnableSequence > 5:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 4:chain:RunnableSequence > 7:chain:_combine_documents] Entering Chain run with input:\n",
      "\u001b[0m[inputs]\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 4:chain:RunnableSequence > 7:chain:_combine_documents] [1ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"DOI\\\": \\\"doi:10.3389/fphys.2022.991149\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\"}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"DOI\\\": \\\"doi:10.3390/ijms221910358\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\"}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"DOI\\\": \\\"doi:10.1038/sigtrans.2017.2\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\"}, {\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"DOI\\\": \\\"doi:10.1016/j.cell.2016.11.048\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\"}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"DOI\\\": \\\"doi:10.1074/mcp.ra120.002228\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\"}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"DOI\\\": \\\"doi:10.1128/jvi.00507-23\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\"}, {\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"DOI\\\": \\\"doi:10.1371/journal.pone.0175970\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\"}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41467-023-43283-2\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\"}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"DOI\\\": \\\"doi:10.1002/2211-5463.13605\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\"}, {\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"DOI\\\": \\\"doi:10.3390/cells8080915\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\"}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"DOI\\\": \\\"doi:10.1134/s0026893321030080\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\"}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"DOI\\\": \\\"doi:10.1186/s13062-021-00311-9\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\"}, {\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41419-018-0406-3\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\"}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41583-020-00407-7\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\"}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"DOI\\\": \\\"doi:10.1002/ueg2.12126\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\"}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context> > 4:chain:RunnableSequence] [326ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"DOI\\\": \\\"doi:10.3389/fphys.2022.991149\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\"}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"DOI\\\": \\\"doi:10.3390/ijms221910358\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\"}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"DOI\\\": \\\"doi:10.1038/sigtrans.2017.2\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\"}, {\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"DOI\\\": \\\"doi:10.1016/j.cell.2016.11.048\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\"}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"DOI\\\": \\\"doi:10.1074/mcp.ra120.002228\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\"}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"DOI\\\": \\\"doi:10.1128/jvi.00507-23\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\"}, {\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"DOI\\\": \\\"doi:10.1371/journal.pone.0175970\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\"}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41467-023-43283-2\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\"}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"DOI\\\": \\\"doi:10.1002/2211-5463.13605\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\"}, {\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"DOI\\\": \\\"doi:10.3390/cells8080915\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\"}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"DOI\\\": \\\"doi:10.1134/s0026893321030080\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\"}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"DOI\\\": \\\"doi:10.1186/s13062-021-00311-9\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\"}, {\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41419-018-0406-3\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\"}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41583-020-00407-7\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\"}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"DOI\\\": \\\"doi:10.1002/ueg2.12126\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\"}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableParallel<k,question,context>] [332ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"DOI\\\": \\\"doi:10.3389/fphys.2022.991149\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\"}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"DOI\\\": \\\"doi:10.3390/ijms221910358\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\"}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"DOI\\\": \\\"doi:10.1038/sigtrans.2017.2\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\"}, {\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"DOI\\\": \\\"doi:10.1016/j.cell.2016.11.048\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\"}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"DOI\\\": \\\"doi:10.1074/mcp.ra120.002228\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\"}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"DOI\\\": \\\"doi:10.1128/jvi.00507-23\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\"}, {\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"DOI\\\": \\\"doi:10.1371/journal.pone.0175970\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\"}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41467-023-43283-2\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\"}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"DOI\\\": \\\"doi:10.1002/2211-5463.13605\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\"}, {\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"DOI\\\": \\\"doi:10.3390/cells8080915\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\"}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"DOI\\\": \\\"doi:10.1134/s0026893321030080\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\"}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"DOI\\\": \\\"doi:10.1186/s13062-021-00311-9\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\"}, {\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41419-018-0406-3\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\"}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"DOI\\\": \\\"doi:10.1038/s41583-020-00407-7\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\"}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"DOI\\\": \\\"doi:10.1002/ueg2.12126\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\"}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 1000\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] [1ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence > 10:prompt:ChatPromptTemplate] Entering Prompt run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence > 10:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:\n",
      "\u001b[0m[outputs]\n",
      "\u001b[32;1m\u001b[1;3m[llm/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence > 11:llm:ChatOpenAI] Entering LLM run with input:\n",
      "\u001b[0m{\n",
      "  \"prompts\": [\n",
      "    \"System: Answer in a direct and concise tone. Your audience is an expert, so be highly specific. If there are ambiguous terms or acronyms, first define them.\\nHuman: Read through the following JSON encoding of 5 research articles: \\nEach document has three attributes: (A) an integer ID for the paper, (B) a CITATION string containing the authors, publication year, title and publication location, and the (C) CONTENT field with the title and abstract of the paper.  \\n```CONTEXT:[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]```\\nIf CONTEXT is provided, generate a JSON list of summaries of each article in order to help answer the following question:\\nQuestion: Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\\nDo NOT directly answer the question, instead summarize the text from each paper's CONTENT to provide evidence to help answer the question.  Do not include information that does not appear directly in the CONTENT of each paper. Focus on specific details, including numbers, equations, or specific quotes.  Reply \\\"Not applicable\\\" if text is irrelevant.  Restrict each summary to 1000 words.  Also, provide a score from 1-10 indicating relevance to question. Do not explain your score. \\nWrite this answer as JSON formatted output. Provide a list of 5 dict objects with the following fields: ID, SUMMARY, RELEVANCE SCORE. \\nDo not provide additional explanation for the answer. Do not include any other response other than a JSON object.\"\n",
      "  ]\n",
      "}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[36;1m\u001b[1;3m[llm/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence > 11:llm:ChatOpenAI] [18.17s] Exiting LLM run with output:\n",
      "\u001b[0m{\n",
      "  \"generations\": [\n",
      "    [\n",
      "      {\n",
      "        \"text\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"The Sec61 complex is acknowledged for its role in intracellular Ca2+ homeostasis, functioning as a major Ca2+ leak pathway in the ER. The paper discusses the Sec61 complex's structure, its regulation as a Ca2+ leak channel, and its pharmacology. It also highlights the association of Sec61 mutations with conditions such as plasma cell deficiency and congenital neutropenia and examines the relevance of the Sec61 complex as a Ca2+ leak channel in various pathophysiological and pathological contexts, including ER stress, apoptosis, ischemia-reperfusion, type 2 diabetes, and cancer.\\\",\\n    \\\"RELEVANCE SCORE\\\": 5\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"This study used bimolecular luminescence complementation to detect protein-protein interactions in living cells, focusing on the ER protein translocase. It confirmed intra-complex interactions such as Sec61-Sec61, and inter-complex interactions involving Sec61-TRAP, Sec61-Sec63, and Sec61-Sec63. The research demonstrated the use of synthetic peptide complementation for expression profiling and a method for protease-mediated reporter degradation to show cytosolic localization of reporter complementation.\\\",\\n    \\\"RELEVANCE SCORE\\\": 3\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"The Sec61 complex, along with the Sec62/Sec63 complex, is integral to the translocation of polypeptides into the ER. Sec62 is implicated in ER-phagy during ER stress recovery, and the Sec61 channel functions as a passive ER calcium leak channel. Mutations and overexpression of SEC genes have been associated with diseases such as kidney and liver diseases, diabetes, and cancer. Recent studies have indicated the potential of Sec62 as a biomarker and therapeutic target in various cancers. The article reviews the function of Sec proteins in protein translocation and their emerging roles in disease pathogenesis and treatment.\\\",\\n    \\\"RELEVANCE SCORE\\\": 7\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "        \"generation_info\": {\n",
      "          \"finish_reason\": \"stop\",\n",
      "          \"logprobs\": null\n",
      "        },\n",
      "        \"type\": \"ChatGeneration\",\n",
      "        \"message\": {\n",
      "          \"lc\": 1,\n",
      "          \"type\": \"constructor\",\n",
      "          \"id\": [\n",
      "            \"langchain\",\n",
      "            \"schema\",\n",
      "            \"messages\",\n",
      "            \"AIMessage\"\n",
      "          ],\n",
      "          \"kwargs\": {\n",
      "            \"content\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"The Sec61 complex is acknowledged for its role in intracellular Ca2+ homeostasis, functioning as a major Ca2+ leak pathway in the ER. The paper discusses the Sec61 complex's structure, its regulation as a Ca2+ leak channel, and its pharmacology. It also highlights the association of Sec61 mutations with conditions such as plasma cell deficiency and congenital neutropenia and examines the relevance of the Sec61 complex as a Ca2+ leak channel in various pathophysiological and pathological contexts, including ER stress, apoptosis, ischemia-reperfusion, type 2 diabetes, and cancer.\\\",\\n    \\\"RELEVANCE SCORE\\\": 5\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"This study used bimolecular luminescence complementation to detect protein-protein interactions in living cells, focusing on the ER protein translocase. It confirmed intra-complex interactions such as Sec61-Sec61, and inter-complex interactions involving Sec61-TRAP, Sec61-Sec63, and Sec61-Sec63. The research demonstrated the use of synthetic peptide complementation for expression profiling and a method for protease-mediated reporter degradation to show cytosolic localization of reporter complementation.\\\",\\n    \\\"RELEVANCE SCORE\\\": 3\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"The Sec61 complex, along with the Sec62/Sec63 complex, is integral to the translocation of polypeptides into the ER. Sec62 is implicated in ER-phagy during ER stress recovery, and the Sec61 channel functions as a passive ER calcium leak channel. Mutations and overexpression of SEC genes have been associated with diseases such as kidney and liver diseases, diabetes, and cancer. Recent studies have indicated the potential of Sec62 as a biomarker and therapeutic target in various cancers. The article reviews the function of Sec proteins in protein translocation and their emerging roles in disease pathogenesis and treatment.\\\",\\n    \\\"RELEVANCE SCORE\\\": 7\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "            \"additional_kwargs\": {},\n",
      "            \"tool_calls\": [],\n",
      "            \"invalid_tool_calls\": []\n",
      "          }\n",
      "        }\n",
      "      }\n",
      "    ]\n",
      "  ],\n",
      "  \"llm_output\": {\n",
      "    \"token_usage\": {\n",
      "      \"completion_tokens\": 503,\n",
      "      \"prompt_tokens\": 1489,\n",
      "      \"total_tokens\": 1992\n",
      "    },\n",
      "    \"model_name\": \"gpt-4-1106-preview\",\n",
      "    \"system_fingerprint\": null\n",
      "  },\n",
      "  \"run\": null\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence > 12:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:\n",
      "\u001b[0m[inputs]\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence > 12:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The Sec61 complex is acknowledged for its role in intracellular Ca2+ homeostasis, functioning as a major Ca2+ leak pathway in the ER. The paper discusses the Sec61 complex's structure, its regulation as a Ca2+ leak channel, and its pharmacology. It also highlights the association of Sec61 mutations with conditions such as plasma cell deficiency and congenital neutropenia and examines the relevance of the Sec61 complex as a Ca2+ leak channel in various pathophysiological and pathological contexts, including ER stress, apoptosis, ischemia-reperfusion, type 2 diabetes, and cancer.\",\n",
      "      \"RELEVANCE SCORE\": 5\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"This study used bimolecular luminescence complementation to detect protein-protein interactions in living cells, focusing on the ER protein translocase. It confirmed intra-complex interactions such as Sec61-Sec61, and inter-complex interactions involving Sec61-TRAP, Sec61-Sec63, and Sec61-Sec63. The research demonstrated the use of synthetic peptide complementation for expression profiling and a method for protease-mediated reporter degradation to show cytosolic localization of reporter complementation.\",\n",
      "      \"RELEVANCE SCORE\": 3\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The Sec61 complex, along with the Sec62/Sec63 complex, is integral to the translocation of polypeptides into the ER. Sec62 is implicated in ER-phagy during ER stress recovery, and the Sec61 channel functions as a passive ER calcium leak channel. Mutations and overexpression of SEC genes have been associated with diseases such as kidney and liver diseases, diabetes, and cancer. Recent studies have indicated the potential of Sec62 as a biomarker and therapeutic target in various cancers. The article reviews the function of Sec proteins in protein translocation and their emerging roles in disease pathogenesis and treatment.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableSequence] [18.17s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The Sec61 complex is acknowledged for its role in intracellular Ca2+ homeostasis, functioning as a major Ca2+ leak pathway in the ER. The paper discusses the Sec61 complex's structure, its regulation as a Ca2+ leak channel, and its pharmacology. It also highlights the association of Sec61 mutations with conditions such as plasma cell deficiency and congenital neutropenia and examines the relevance of the Sec61 complex as a Ca2+ leak channel in various pathophysiological and pathological contexts, including ER stress, apoptosis, ischemia-reperfusion, type 2 diabetes, and cancer.\",\n",
      "      \"RELEVANCE SCORE\": 5\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"This study used bimolecular luminescence complementation to detect protein-protein interactions in living cells, focusing on the ER protein translocase. It confirmed intra-complex interactions such as Sec61-Sec61, and inter-complex interactions involving Sec61-TRAP, Sec61-Sec63, and Sec61-Sec63. The research demonstrated the use of synthetic peptide complementation for expression profiling and a method for protease-mediated reporter degradation to show cytosolic localization of reporter complementation.\",\n",
      "      \"RELEVANCE SCORE\": 3\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The Sec61 complex, along with the Sec62/Sec63 complex, is integral to the translocation of polypeptides into the ER. Sec62 is implicated in ER-phagy during ER stress recovery, and the Sec61 channel functions as a passive ER calcium leak channel. Mutations and overexpression of SEC genes have been associated with diseases such as kidney and liver diseases, diabetes, and cancer. Recent studies have indicated the potential of Sec62 as a biomarker and therapeutic target in various cancers. The article reviews the function of Sec proteins in protein translocation and their emerging roles in disease pathogenesis and treatment.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] [18.17s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The Sec61 complex is acknowledged for its role in intracellular Ca2+ homeostasis, functioning as a major Ca2+ leak pathway in the ER. The paper discusses the Sec61 complex's structure, its regulation as a Ca2+ leak channel, and its pharmacology. It also highlights the association of Sec61 mutations with conditions such as plasma cell deficiency and congenital neutropenia and examines the relevance of the Sec61 complex as a Ca2+ leak channel in various pathophysiological and pathological contexts, including ER stress, apoptosis, ischemia-reperfusion, type 2 diabetes, and cancer.\",\n",
      "      \"RELEVANCE SCORE\": 5\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"This study used bimolecular luminescence complementation to detect protein-protein interactions in living cells, focusing on the ER protein translocase. It confirmed intra-complex interactions such as Sec61-Sec61, and inter-complex interactions involving Sec61-TRAP, Sec61-Sec63, and Sec61-Sec63. The research demonstrated the use of synthetic peptide complementation for expression profiling and a method for protease-mediated reporter degradation to show cytosolic localization of reporter complementation.\",\n",
      "      \"RELEVANCE SCORE\": 3\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The Sec61 complex, along with the Sec62/Sec63 complex, is integral to the translocation of polypeptides into the ER. Sec62 is implicated in ER-phagy during ER stress recovery, and the Sec61 channel functions as a passive ER calcium leak channel. Mutations and overexpression of SEC genes have been associated with diseases such as kidney and liver diseases, diabetes, and cancer. Recent studies have indicated the potential of Sec62 as a biomarker and therapeutic target in various cancers. The article reviews the function of Sec proteins in protein translocation and their emerging roles in disease pathogenesis and treatment.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence] [18.17s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The Sec61 complex is acknowledged for its role in intracellular Ca2+ homeostasis, functioning as a major Ca2+ leak pathway in the ER. The paper discusses the Sec61 complex's structure, its regulation as a Ca2+ leak channel, and its pharmacology. It also highlights the association of Sec61 mutations with conditions such as plasma cell deficiency and congenital neutropenia and examines the relevance of the Sec61 complex as a Ca2+ leak channel in various pathophysiological and pathological contexts, including ER stress, apoptosis, ischemia-reperfusion, type 2 diabetes, and cancer.\",\n",
      "      \"RELEVANCE SCORE\": 5\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"This study used bimolecular luminescence complementation to detect protein-protein interactions in living cells, focusing on the ER protein translocase. It confirmed intra-complex interactions such as Sec61-Sec61, and inter-complex interactions involving Sec61-TRAP, Sec61-Sec63, and Sec61-Sec63. The research demonstrated the use of synthetic peptide complementation for expression profiling and a method for protease-mediated reporter degradation to show cytosolic localization of reporter complementation.\",\n",
      "      \"RELEVANCE SCORE\": 3\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The Sec61 complex, along with the Sec62/Sec63 complex, is integral to the translocation of polypeptides into the ER. Sec62 is implicated in ER-phagy during ER stress recovery, and the Sec61 channel functions as a passive ER calcium leak channel. Mutations and overexpression of SEC genes have been associated with diseases such as kidney and liver diseases, diabetes, and cancer. Recent studies have indicated the potential of Sec62 as a biomarker and therapeutic target in various cancers. The article reviews the function of Sec proteins in protein translocation and their emerging roles in disease pathogenesis and treatment.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\\n\\\\nThe heterotrimeric Sec61 protein complex forms the functional core of the so-called translocon that forms an aqueous channel in the endoplasmic reticulum (ER). The primary role of the Sec61 complex is to allow protein import in the ER during translation. Surprisingly, a completely different function in intracellular Ca<sup>2+</sup> homeostasis has emerged for the Sec61 complex, and the latter is now accepted as one of the major Ca<sup>2+</sup>-leak pathways of the ER. In this review, we first discuss the structure of the Sec61 complex and focus on the pharmacology and regulation of the Sec61 complex as a Ca<sup>2+</sup>-leak channel. Subsequently, we will pay particular attention to pathologies that are linked to Sec61 mutations, such as plasma cell deficiency and congenital neutropenia. Finally, we will explore the relevance of the Sec61 complex as a Ca<sup>2+</sup>-leak channel in various pathophysiological (ER stress, apoptosis, ischemia-reperfusion) and pathological (type 2 diabetes, cancer) settings.\\\", \\\"CITATION\\\": \\\"Parys JB, Van Coppenolle F. (2022) Sec61 complex/translocon: The role of an atypical ER Ca<sup>2+</sup>-leak channel in health and disease.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\\n\\\\nVarious landmark studies have revealed structures and functions of the Sec61/SecY complex in all domains of live demonstrating the conserved nature of this ancestral protein translocase. While the bacterial homolog of the Sec61 complex resides in the plasma membrane, the eukaryotic counterpart manages the transfer of precursor proteins into or across the membrane of the endoplasmic reticulum (ER). Sec61 complexes are accompanied by a set of dynamically recruited auxiliary proteins assisting the transport of certain precursor polypeptides. TRAP and Sec62/Sec63 are two auxiliary protein complexes in mammalian cells that have been characterized by structural and biochemical methods. Using these ER membrane protein complexes for our proof-of-concept study, we aimed to detect interactions of membrane proteins in living mammalian cells under physiological conditions. Bimolecular luminescence complementation and competition was used to demonstrate multiple protein-protein interactions of different topological layouts. In addition to the interaction of the soluble catalytic and regulatory subunits of the cytosolic protein kinase A, we detected interactions of ER membrane proteins that either belong to the same multimeric protein complex (intra-complex interactions: Sec61\\\\u03b1-Sec61\\\\u03b2, TRAP\\\\u03b1-TRAP\\\\u03b2) or protein complexes in juxtaposition (inter-complex interactions: Sec61\\\\u03b1-TRAP\\\\u03b1, Sec61\\\\u03b1-Sec63, and Sec61\\\\u03b2-Sec63). In the process, we established further control elements like synthetic peptide complementation for expression profiling of fusion constructs and protease-mediated reporter degradation demonstrating the cytosolic localization of a reporter complementation. Ease of use and flexibility of the approach presented here will spur further research regarding the dynamics of protein-protein interactions in response to changing cellular conditions in living cells.\\\", \\\"CITATION\\\": \\\"Sicking M, Jung M, Lang S. (2021) Lights, Camera, Interaction: Studying Protein-Protein Interactions of the ER Protein Translocase in Living Cells.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\\n\\\\nThe heterotrimeric Sec61 complex and the dimeric Sec62/Sec63 complex are located in the membrane of the human endoplasmic reticulum (ER) and play a central role in translocation of nascent and newly synthesized precursor polypeptides into the ER. This process involves targeting of the precursors to the membrane and opening of the polypeptide conducting Sec61 channel for translocation. Apart from this central role in the intracellular transport of polypeptides, several studies of the last decade uncovered additional functions of Sec proteins in intracellular signaling: Sec62 can induce ER-phagy in the process of recovery of cells from ER stress and the Sec61 channel can also act as a passive ER calcium leak channel. Furthermore, mutations, amplifications and an overexpression of the <i>SEC</i> genes were linked to various diseases including kidney and liver diseases, diabetes and human cancer. Studies of the last decade could not only elucidate the functional role of Sec proteins in the pathogenesis of these diseases, but also demonstrate a relevance of Sec62 as a prognostic and predictive biomarker in head and neck cancer, prostate and lung cancer including a basis for new therapeutic strategies. In this article, we review the current understanding of protein transport across the ER membrane as central function of Sec proteins and further focus on recent studies that gave first insights into the functional role and therapeutic relevance of Sec61, Sec62 and Sec63 in human diseases.\\\", \\\"CITATION\\\": \\\"Linxweiler M, Schick B, Zimmermann R. (2017) Let's talk about Secs: Sec61, Sec62 and Sec63 in signal transduction, oncology and personalized medicine.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 1000\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] [2ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] Entering Prompt run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:\n",
      "\u001b[0m[outputs]\n",
      "\u001b[32;1m\u001b[1;3m[llm/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] Entering LLM run with input:\n",
      "\u001b[0m{\n",
      "  \"prompts\": [\n",
      "    \"System: Answer in a direct and concise tone. Your audience is an expert, so be highly specific. If there are ambiguous terms or acronyms, first define them.\\nHuman: Read through the following JSON encoding of 5 research articles: \\nEach document has three attributes: (A) an integer ID for the paper, (B) a CITATION string containing the authors, publication year, title and publication location, and the (C) CONTENT field with the title and abstract of the paper.  \\n```CONTEXT:[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]```\\nIf CONTEXT is provided, generate a JSON list of summaries of each article in order to help answer the following question:\\nQuestion: Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\\nDo NOT directly answer the question, instead summarize the text from each paper's CONTENT to provide evidence to help answer the question.  Do not include information that does not appear directly in the CONTENT of each paper. Focus on specific details, including numbers, equations, or specific quotes.  Reply \\\"Not applicable\\\" if text is irrelevant.  Restrict each summary to 1000 words.  Also, provide a score from 1-10 indicating relevance to question. Do not explain your score. \\nWrite this answer as JSON formatted output. Provide a list of 5 dict objects with the following fields: ID, SUMMARY, RELEVANCE SCORE. \\nDo not provide additional explanation for the answer. Do not include any other response other than a JSON object.\"\n",
      "  ]\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[llm/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] [13.97s] Exiting LLM run with output:\n",
      "\u001b[0m{\n",
      "  \"generations\": [\n",
      "    [\n",
      "      {\n",
      "        \"text\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"Perturb-seq bridges the gap between the number of perturbations examined and the complexity of phenotypes measured by combining single-cell RNA-seq with CRISPR-mediated perturbations. Applied to the mammalian unfolded protein response (UPR), it identified genes whose repression disrupts ER homeostasis and enabled functional clustering of \\\\u223c100 hits. Single-cell analysis revealed bifurcated UPR branch activation and differential branch activation across hits, including a feedback loop between the translocon and IRE1\\\\u03b1.\\\",\\n    \\\"RELEVANCE SCORE\\\": 9\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"Quantitative proximity proteomics was used to identify the neighboring protein networks for SEC61\\\\u03b2, RPN1, SEC62, and LRRC59, which are involved in ER-localized translation. Biotin labeling time course studies suggested the ER membrane has discrete protein interaction domains. SEC61\\\\u03b2 and RPN1 are associated with the translocon network, while SEC62 and LRRC59 have divergent interactomes. LRRC59 is linked to mRNA translation regulation on the ER, and silencing LRRC59 reduces ER translation levels by about 50%.\\\",\\n    \\\"RELEVANCE SCORE\\\": 8\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "        \"generation_info\": {\n",
      "          \"finish_reason\": \"stop\",\n",
      "          \"logprobs\": null\n",
      "        },\n",
      "        \"type\": \"ChatGeneration\",\n",
      "        \"message\": {\n",
      "          \"lc\": 1,\n",
      "          \"type\": \"constructor\",\n",
      "          \"id\": [\n",
      "            \"langchain\",\n",
      "            \"schema\",\n",
      "            \"messages\",\n",
      "            \"AIMessage\"\n",
      "          ],\n",
      "          \"kwargs\": {\n",
      "            \"content\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"Perturb-seq bridges the gap between the number of perturbations examined and the complexity of phenotypes measured by combining single-cell RNA-seq with CRISPR-mediated perturbations. Applied to the mammalian unfolded protein response (UPR), it identified genes whose repression disrupts ER homeostasis and enabled functional clustering of \\\\u223c100 hits. Single-cell analysis revealed bifurcated UPR branch activation and differential branch activation across hits, including a feedback loop between the translocon and IRE1\\\\u03b1.\\\",\\n    \\\"RELEVANCE SCORE\\\": 9\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"Quantitative proximity proteomics was used to identify the neighboring protein networks for SEC61\\\\u03b2, RPN1, SEC62, and LRRC59, which are involved in ER-localized translation. Biotin labeling time course studies suggested the ER membrane has discrete protein interaction domains. SEC61\\\\u03b2 and RPN1 are associated with the translocon network, while SEC62 and LRRC59 have divergent interactomes. LRRC59 is linked to mRNA translation regulation on the ER, and silencing LRRC59 reduces ER translation levels by about 50%.\\\",\\n    \\\"RELEVANCE SCORE\\\": 8\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "            \"additional_kwargs\": {},\n",
      "            \"tool_calls\": [],\n",
      "            \"invalid_tool_calls\": []\n",
      "          }\n",
      "        }\n",
      "      }\n",
      "    ]\n",
      "  ],\n",
      "  \"llm_output\": {\n",
      "    \"token_usage\": {\n",
      "      \"completion_tokens\": 381,\n",
      "      \"prompt_tokens\": 1507,\n",
      "      \"total_tokens\": 1888\n",
      "    },\n",
      "    \"model_name\": \"gpt-4-1106-preview\",\n",
      "    \"system_fingerprint\": null\n",
      "  },\n",
      "  \"run\": null\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 11:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:\n",
      "\u001b[0m[inputs]\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 11:parser:JsonEnclosedByTextOutputParser] [0ms] Exiting Parser run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"Perturb-seq bridges the gap between the number of perturbations examined and the complexity of phenotypes measured by combining single-cell RNA-seq with CRISPR-mediated perturbations. Applied to the mammalian unfolded protein response (UPR), it identified genes whose repression disrupts ER homeostasis and enabled functional clustering of 100 hits. Single-cell analysis revealed bifurcated UPR branch activation and differential branch activation across hits, including a feedback loop between the translocon and IRE1.\",\n",
      "      \"RELEVANCE SCORE\": 9\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Quantitative proximity proteomics was used to identify the neighboring protein networks for SEC61, RPN1, SEC62, and LRRC59, which are involved in ER-localized translation. Biotin labeling time course studies suggested the ER membrane has discrete protein interaction domains. SEC61 and RPN1 are associated with the translocon network, while SEC62 and LRRC59 have divergent interactomes. LRRC59 is linked to mRNA translation regulation on the ER, and silencing LRRC59 reduces ER translation levels by about 50%.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] [13.97s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"Perturb-seq bridges the gap between the number of perturbations examined and the complexity of phenotypes measured by combining single-cell RNA-seq with CRISPR-mediated perturbations. Applied to the mammalian unfolded protein response (UPR), it identified genes whose repression disrupts ER homeostasis and enabled functional clustering of 100 hits. Single-cell analysis revealed bifurcated UPR branch activation and differential branch activation across hits, including a feedback loop between the translocon and IRE1.\",\n",
      "      \"RELEVANCE SCORE\": 9\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Quantitative proximity proteomics was used to identify the neighboring protein networks for SEC61, RPN1, SEC62, and LRRC59, which are involved in ER-localized translation. Biotin labeling time course studies suggested the ER membrane has discrete protein interaction domains. SEC61 and RPN1 are associated with the translocon network, while SEC62 and LRRC59 have divergent interactomes. LRRC59 is linked to mRNA translation regulation on the ER, and silencing LRRC59 reduces ER translation levels by about 50%.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] [13.97s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"Perturb-seq bridges the gap between the number of perturbations examined and the complexity of phenotypes measured by combining single-cell RNA-seq with CRISPR-mediated perturbations. Applied to the mammalian unfolded protein response (UPR), it identified genes whose repression disrupts ER homeostasis and enabled functional clustering of 100 hits. Single-cell analysis revealed bifurcated UPR branch activation and differential branch activation across hits, including a feedback loop between the translocon and IRE1.\",\n",
      "      \"RELEVANCE SCORE\": 9\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Quantitative proximity proteomics was used to identify the neighboring protein networks for SEC61, RPN1, SEC62, and LRRC59, which are involved in ER-localized translation. Biotin labeling time course studies suggested the ER membrane has discrete protein interaction domains. SEC61 and RPN1 are associated with the translocon network, while SEC62 and LRRC59 have divergent interactomes. LRRC59 is linked to mRNA translation regulation on the ER, and silencing LRRC59 reduces ER translation levels by about 50%.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence] [13.98s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"Perturb-seq bridges the gap between the number of perturbations examined and the complexity of phenotypes measured by combining single-cell RNA-seq with CRISPR-mediated perturbations. Applied to the mammalian unfolded protein response (UPR), it identified genes whose repression disrupts ER homeostasis and enabled functional clustering of 100 hits. Single-cell analysis revealed bifurcated UPR branch activation and differential branch activation across hits, including a feedback loop between the translocon and IRE1.\",\n",
      "      \"RELEVANCE SCORE\": 9\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Quantitative proximity proteomics was used to identify the neighboring protein networks for SEC61, RPN1, SEC62, and LRRC59, which are involved in ER-localized translation. Biotin labeling time course studies suggested the ER membrane has discrete protein interaction domains. SEC61 and RPN1 are associated with the translocon network, while SEC62 and LRRC59 have divergent interactomes. LRRC59 is linked to mRNA translation regulation on the ER, and silencing LRRC59 reduces ER translation levels by about 50%.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\\n\\\\nFunctional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting \\\\u223c100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\\\u00a0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1\\\\u03b1. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.\\\", \\\"CITATION\\\": \\\"Adamson B, Norman TM, Jost M, Cho MY, Nu\\\\u00f1ez JK, Chen Y, Villalta JE, Gilbert LA, Horlbeck MA, Hein MY, Pak RA, Gray AN, Gross CA, Dixit A, Parnas O, Regev A, Weissman JS. (2016) A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\\n\\\\nProtein synthesis on the endoplasmic reticulum (ER) requires the dynamic coordination of numerous cellular components. Together, resident ER membrane proteins, cytoplasmic translation factors, and both integral membrane and cytosolic RNA-binding proteins operate in concert with membrane-associated ribosomes to facilitate ER-localized translation. Little is known, however, regarding the spatial organization of ER-localized translation. This question is of growing significance as it is now known that ER-bound ribosomes contribute to secretory, integral membrane, and cytosolic protein synthesis alike. To explore this question, we utilized quantitative proximity proteomics to identify neighboring protein networks for the candidate ribosome interactors SEC61\\\\u03b2 (subunit of the protein translocase), RPN1 (oligosaccharyltransferase subunit), SEC62 (translocation integral membrane protein), and LRRC59 (ribosome binding integral membrane protein). Biotin labeling time course studies of the four BioID reporters revealed distinct labeling patterns that intensified but only modestly diversified as a function of labeling time, suggesting that the ER membrane is organized into discrete protein interaction domains. Whereas SEC61\\\\u03b2 and RPN1 reporters identified translocon-associated networks, SEC62 and LRRC59 reporters revealed divergent protein interactomes. Notably, the SEC62 interactome is enriched in redox-linked proteins and ER luminal chaperones, with the latter likely representing proximity to an ER luminal chaperone reflux pathway. In contrast, the LRRC59 interactome is highly enriched in SRP pathway components, translation factors, and ER-localized RNA-binding proteins, uncovering a functional link between LRRC59 and mRNA translation regulation. Importantly, analysis of the LRRC59 interactome by native immunoprecipitation identified similar protein and functional enrichments. Moreover, [<sup>35</sup>S]-methionine incorporation assays revealed that siRNA silencing of LRRC59 expression reduced steady state translation levels on the ER by ca. 50%, and also impacted steady state translation levels in the cytosol compartment. Collectively, these data reveal a functional domain organization for the ER and identify a key role for LRRC59 in the organization and regulation of local translation.\\\", \\\"CITATION\\\": \\\"Hannigan MM, Hoffman AM, Thompson JW, Zheng T, Nicchitta CV. (2020) Quantitative Proteomics Links the LRRC59 Interactome to mRNA Translation on the ER Membrane.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\\n\\\\n<h4>Importance</h4>Generation of virus-host protein-protein interactions (PPIs) maps may provide clues to uncover SARS-CoV-2-hijacked cellular processes. However, these PPIs maps were created by expressing each viral protein singularly, which does not reflect the life situation in which certain viral proteins synergistically interact with host proteins. Our results reveal the host-viral protein-protein interactome of SARS-CoV-2 NSP3, NSP4, and NSP6 expressed individually or in combination. Furthermore, REEP5/TRAM1 complex interacts with NSP3 at ROs and promotes viral replication. The significance of our research is identifying virus-host interactions that may be targeted for therapeutic intervention.\\\", \\\"CITATION\\\": \\\"Li J, Gui Q, Liang F-X, Sall J, Zhang Q, Duan Y, Dhabaria A, Askenazi M, Ueberheide B, Stapleford KA, Pagano M. (2023) The REEP5/TRAM1 complex binds SARS-CoV-2 NSP3 and promotes virus replication.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 1000\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 5\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] [3ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] Entering Prompt run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:\n",
      "\u001b[0m[outputs]\n",
      "\u001b[32;1m\u001b[1;3m[llm/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] Entering LLM run with input:\n",
      "\u001b[0m{\n",
      "  \"prompts\": [\n",
      "    \"System: Answer in a direct and concise tone. Your audience is an expert, so be highly specific. If there are ambiguous terms or acronyms, first define them.\\nHuman: Read through the following JSON encoding of 5 research articles: \\nEach document has three attributes: (A) an integer ID for the paper, (B) a CITATION string containing the authors, publication year, title and publication location, and the (C) CONTENT field with the title and abstract of the paper.  \\n```CONTEXT:[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]```\\nIf CONTEXT is provided, generate a JSON list of summaries of each article in order to help answer the following question:\\nQuestion: Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\\nDo NOT directly answer the question, instead summarize the text from each paper's CONTENT to provide evidence to help answer the question.  Do not include information that does not appear directly in the CONTENT of each paper. Focus on specific details, including numbers, equations, or specific quotes.  Reply \\\"Not applicable\\\" if text is irrelevant.  Restrict each summary to 1000 words.  Also, provide a score from 1-10 indicating relevance to question. Do not explain your score. \\nWrite this answer as JSON formatted output. Provide a list of 5 dict objects with the following fields: ID, SUMMARY, RELEVANCE SCORE. \\nDo not provide additional explanation for the answer. Do not include any other response other than a JSON object.\"\n",
      "  ]\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[llm/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] [14.91s] Exiting LLM run with output:\n",
      "\u001b[0m{\n",
      "  \"generations\": [\n",
      "    [\n",
      "      {\n",
      "        \"text\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"The paper discusses the role of mutant uromodulin expression in altering the homeostasis of the endoplasmic reticulum (ER) and activating the unfolded protein response (UPR). Mutations in UMOD cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), with symptoms including hyperuricemia, gout, and renal function decline. The study used transcriptional profiling and biochemical characterization in cellular models to find that mutant uromodulin accumulates in the ER, but does not affect cell viability or proliferation. It identified 109 differentially expressed genes and indicated upregulation of ER chaperones and protein disulfide isomerases. Pathway enrichment analysis suggested that mutant uromodulin impacts ER function and protein homeostasis. The IRE1 branch of the UPR is particularly induced, as evidenced by increased splicing of XBP1 mRNA. The study also observed an increased interaction of mutant uromodulin with ER chaperones Bip, calnexin, and PDI, and partial degradation of the mutant protein by the proteasome through ER-associated degradation.\\\",\\n    \\\"RELEVANCE SCORE\\\": 8\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "        \"generation_info\": {\n",
      "          \"finish_reason\": \"stop\",\n",
      "          \"logprobs\": null\n",
      "        },\n",
      "        \"type\": \"ChatGeneration\",\n",
      "        \"message\": {\n",
      "          \"lc\": 1,\n",
      "          \"type\": \"constructor\",\n",
      "          \"id\": [\n",
      "            \"langchain\",\n",
      "            \"schema\",\n",
      "            \"messages\",\n",
      "            \"AIMessage\"\n",
      "          ],\n",
      "          \"kwargs\": {\n",
      "            \"content\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"The paper discusses the role of mutant uromodulin expression in altering the homeostasis of the endoplasmic reticulum (ER) and activating the unfolded protein response (UPR). Mutations in UMOD cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), with symptoms including hyperuricemia, gout, and renal function decline. The study used transcriptional profiling and biochemical characterization in cellular models to find that mutant uromodulin accumulates in the ER, but does not affect cell viability or proliferation. It identified 109 differentially expressed genes and indicated upregulation of ER chaperones and protein disulfide isomerases. Pathway enrichment analysis suggested that mutant uromodulin impacts ER function and protein homeostasis. The IRE1 branch of the UPR is particularly induced, as evidenced by increased splicing of XBP1 mRNA. The study also observed an increased interaction of mutant uromodulin with ER chaperones Bip, calnexin, and PDI, and partial degradation of the mutant protein by the proteasome through ER-associated degradation.\\\",\\n    \\\"RELEVANCE SCORE\\\": 8\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "            \"additional_kwargs\": {},\n",
      "            \"tool_calls\": [],\n",
      "            \"invalid_tool_calls\": []\n",
      "          }\n",
      "        }\n",
      "      }\n",
      "    ]\n",
      "  ],\n",
      "  \"llm_output\": {\n",
      "    \"token_usage\": {\n",
      "      \"completion_tokens\": 390,\n",
      "      \"prompt_tokens\": 1592,\n",
      "      \"total_tokens\": 1982\n",
      "    },\n",
      "    \"model_name\": \"gpt-4-1106-preview\",\n",
      "    \"system_fingerprint\": null\n",
      "  },\n",
      "  \"run\": null\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 11:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:\n",
      "\u001b[0m[inputs]\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 11:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The paper discusses the role of mutant uromodulin expression in altering the homeostasis of the endoplasmic reticulum (ER) and activating the unfolded protein response (UPR). Mutations in UMOD cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), with symptoms including hyperuricemia, gout, and renal function decline. The study used transcriptional profiling and biochemical characterization in cellular models to find that mutant uromodulin accumulates in the ER, but does not affect cell viability or proliferation. It identified 109 differentially expressed genes and indicated upregulation of ER chaperones and protein disulfide isomerases. Pathway enrichment analysis suggested that mutant uromodulin impacts ER function and protein homeostasis. The IRE1 branch of the UPR is particularly induced, as evidenced by increased splicing of XBP1 mRNA. The study also observed an increased interaction of mutant uromodulin with ER chaperones Bip, calnexin, and PDI, and partial degradation of the mutant protein by the proteasome through ER-associated degradation.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] [14.91s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The paper discusses the role of mutant uromodulin expression in altering the homeostasis of the endoplasmic reticulum (ER) and activating the unfolded protein response (UPR). Mutations in UMOD cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), with symptoms including hyperuricemia, gout, and renal function decline. The study used transcriptional profiling and biochemical characterization in cellular models to find that mutant uromodulin accumulates in the ER, but does not affect cell viability or proliferation. It identified 109 differentially expressed genes and indicated upregulation of ER chaperones and protein disulfide isomerases. Pathway enrichment analysis suggested that mutant uromodulin impacts ER function and protein homeostasis. The IRE1 branch of the UPR is particularly induced, as evidenced by increased splicing of XBP1 mRNA. The study also observed an increased interaction of mutant uromodulin with ER chaperones Bip, calnexin, and PDI, and partial degradation of the mutant protein by the proteasome through ER-associated degradation.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] [14.91s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The paper discusses the role of mutant uromodulin expression in altering the homeostasis of the endoplasmic reticulum (ER) and activating the unfolded protein response (UPR). Mutations in UMOD cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), with symptoms including hyperuricemia, gout, and renal function decline. The study used transcriptional profiling and biochemical characterization in cellular models to find that mutant uromodulin accumulates in the ER, but does not affect cell viability or proliferation. It identified 109 differentially expressed genes and indicated upregulation of ER chaperones and protein disulfide isomerases. Pathway enrichment analysis suggested that mutant uromodulin impacts ER function and protein homeostasis. The IRE1 branch of the UPR is particularly induced, as evidenced by increased splicing of XBP1 mRNA. The study also observed an increased interaction of mutant uromodulin with ER chaperones Bip, calnexin, and PDI, and partial degradation of the mutant protein by the proteasome through ER-associated degradation.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence] [14.92s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The paper discusses the role of mutant uromodulin expression in altering the homeostasis of the endoplasmic reticulum (ER) and activating the unfolded protein response (UPR). Mutations in UMOD cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), with symptoms including hyperuricemia, gout, and renal function decline. The study used transcriptional profiling and biochemical characterization in cellular models to find that mutant uromodulin accumulates in the ER, but does not affect cell viability or proliferation. It identified 109 differentially expressed genes and indicated upregulation of ER chaperones and protein disulfide isomerases. Pathway enrichment analysis suggested that mutant uromodulin impacts ER function and protein homeostasis. The IRE1 branch of the UPR is particularly induced, as evidenced by increased splicing of XBP1 mRNA. The study also observed an increased interaction of mutant uromodulin with ER chaperones Bip, calnexin, and PDI, and partial degradation of the mutant protein by the proteasome through ER-associated degradation.\",\n",
      "      \"RELEVANCE SCORE\": 8\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\\n\\\\nUromodulin is the most abundant urinary protein in physiological conditions. It is exclusively produced by renal epithelial cells lining the thick ascending limb of Henle's loop (TAL) and it plays key roles in kidney function and disease. Mutations in UMOD, the gene encoding uromodulin, cause autosomal dominant tubulointerstitial kidney disease uromodulin-related (ADTKD-UMOD), characterised by hyperuricemia, gout and progressive loss of renal function. While the primary effect of UMOD mutations, retention in the endoplasmic reticulum (ER), is well established, its downstream effects are still largely unknown. To gain insight into ADTKD-UMOD pathogenesis, we performed transcriptional profiling and biochemical characterisation of cellular models (immortalised mouse TAL cells) of robust expression of wild type or mutant GFP-tagged uromodulin. In this model mutant uromodulin accumulation in the ER does not impact on cell viability and proliferation. Transcriptional profiling identified 109 genes that are differentially expressed in mutant cells relative to wild type ones. Up-regulated genes include several ER resident chaperones and protein disulphide isomerases. Consistently, pathway enrichment analysis indicates that mutant uromodulin expression affects ER function and protein homeostasis. Interestingly, mutant uromodulin expression induces the Unfolded Protein Response (UPR), and specifically the IRE1 branch, as shown by an increased splicing of XBP1. Consistent with UPR induction, we show increased interaction of mutant uromodulin with ER chaperones Bip, calnexin and PDI. Using metabolic labelling, we also demonstrate that while autophagy plays no role, mutant protein is partially degraded by the proteasome through ER-associated degradation. Our work demonstrates that ER stress could play a central role in ADTKD-UMOD pathogenesis. This sets the bases for future work to develop novel therapeutic strategies through modulation of ER homeostasis and associated protein degradation pathways.\\\", \\\"CITATION\\\": \\\"Schaeffer C, Merella S, Pasqualetto E, Lazarevic D, Rampoldi L. (2017) Mutant uromodulin expression leads to altered homeostasis of the endoplasmic reticulum and activates the unfolded protein response.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\\n\\\\nInterferon-gamma (IFN-\\\\u03b3) signaling is necessary for the proinflammatory activation of macrophages but IFN-\\\\u03b3-independent pathways, for which the initiating stimuli and downstream mechanisms are lesser known, also contribute. Here we identify, by high-content screening, SEPTIN2 (SEPT2) as a negative regulation of IFN-\\\\u03b3-independent macrophage autoactivation. Mechanistically, endoplasmic reticulum (ER) stress induces the expression of SEPT2, which balances the competition between acetylation and ubiquitination of heat shock protein 5 at position Lysine 327, thereby alleviating ER stress and constraining M1-like polarization and proinflammatory cytokine release. Disruption of this negative feedback regulation leads to the accumulation of unfolded proteins, resulting in accelerated M1-like polarization, excessive inflammation and tissue damage. Our study thus uncovers an IFN-\\\\u03b3-independent macrophage proinflammatory autoactivation pathway and suggests that SEPT2 may play a role in the prevention or resolution of inflammation during infection.\\\", \\\"CITATION\\\": \\\"Fu B, Xiong Y, Sha Z, Xue W, Xu B, Tan S, Guo D, Lin F, Wang L, Ji J, Luo Y, Lin X, Wu H. (2023) SEPTIN2 suppresses an IFN-\\\\u03b3-independent, proinflammatory macrophage activation pathway.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\\n\\\\nIntracellular organelles carry out many of their functions by engaging in extensive interorganellar communication through specialized membrane contact sites (MCSs) formed where two organelles tether to each other or to the plasma membrane (PM) without fusing. In recent years, these ubiquitous membrane structures have emerged as central signaling hubs that control a multitude of cellular pathways, ranging from lipid metabolism/transport to the exchange of metabolites and ions (i.e., Ca<sup>2+</sup> ), and general organellar biogenesis. The functional crosstalk between juxtaposed membranes at MCSs relies on a defined composite of proteins and lipids that populate these microdomains in a dynamic fashion. This is particularly important in the nervous system, where alterations in the composition of MCSs have been shown to affect their functions and have been implicated in the pathogenesis of neurodegenerative diseases. In this review, we focus on the MCSs that are formed by the tethering of the endoplasmic reticulum (ER) to the mitochondria, the ER to the endo-lysosomes and the mitochondria to the lysosomes. We highlight how glycosphingolipids that are aberrantly processed/degraded and accumulate ectopically in intracellular membranes and the PM change the topology of MCSs, disrupting signaling pathways that lead to neuronal demise and neurodegeneration. In particular, we focus on neurodegenerative lysosomal storage diseases linked to altered glycosphingolipid catabolism.\\\", \\\"CITATION\\\": \\\"Weesner JA, Annunziata I, van de Vlekkert D, d'Azzo A. (2023) Glycosphingolipids within membrane contact sites influence their function as signaling hubs in neurodegenerative diseases.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 1000\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] [3ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] Entering Prompt run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:\n",
      "\u001b[0m[outputs]\n",
      "\u001b[32;1m\u001b[1;3m[llm/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] Entering LLM run with input:\n",
      "\u001b[0m{\n",
      "  \"prompts\": [\n",
      "    \"System: Answer in a direct and concise tone. Your audience is an expert, so be highly specific. If there are ambiguous terms or acronyms, first define them.\\nHuman: Read through the following JSON encoding of 5 research articles: \\nEach document has three attributes: (A) an integer ID for the paper, (B) a CITATION string containing the authors, publication year, title and publication location, and the (C) CONTENT field with the title and abstract of the paper.  \\n```CONTEXT:[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]```\\nIf CONTEXT is provided, generate a JSON list of summaries of each article in order to help answer the following question:\\nQuestion: Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\\nDo NOT directly answer the question, instead summarize the text from each paper's CONTENT to provide evidence to help answer the question.  Do not include information that does not appear directly in the CONTENT of each paper. Focus on specific details, including numbers, equations, or specific quotes.  Reply \\\"Not applicable\\\" if text is irrelevant.  Restrict each summary to 1000 words.  Also, provide a score from 1-10 indicating relevance to question. Do not explain your score. \\nWrite this answer as JSON formatted output. Provide a list of 5 dict objects with the following fields: ID, SUMMARY, RELEVANCE SCORE. \\nDo not provide additional explanation for the answer. Do not include any other response other than a JSON object.\"\n",
      "  ]\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] [2ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[llm/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] [17.75s] Exiting LLM run with output:\n",
      "\u001b[0m{\n",
      "  \"generations\": [\n",
      "    [\n",
      "      {\n",
      "        \"text\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"The article discusses the nuclear envelope associated endosomes (NAE) pathway, which delivers cell surface receptors such as EGFR from the cell surface to the nucleus. This process involves transport, docking, and fusion with the outer membrane of the nuclear envelope, followed by transport through the nuclear pore complex. The article mentions involvement of the SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins, and the Sec61 translocon in this pathway. It also addresses the role of nuclear EGFR in regulating gene transcription, cell proliferation, and DNA damage repair, and its correlation with poor prognosis in various cancers.\\\",\\n    \\\"RELEVANCE SCORE\\\": 7\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"The paper reviews the spatial organization of protein biosynthesis in eukaryotic cells, discussing classical and new concepts of mRNA targeting, transport, and localized translation. It highlights the selective translation of mRNA in proximity to various organelles, including endosomes and the role of RNA-binding proteins in forming RNP particles for mRNA transport. The paper also addresses the concept of 'translation factories' and their significance in protein folding and interaction. While it discusses the importance of mRNA compartmentalization in protein biosynthesis regulation, it does not specifically mention the unfolded protein response or Sec61B.\\\",\\n    \\\"RELEVANCE SCORE\\\": 2\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"The study focuses on Rab32, a small GTPase that influences the composition of the mitochondria-associated membrane (MAM) and regulates mitochondrial membrane dynamics through its effector dynamin-related protein 1 (Drp1). Rab32 is reported to induce autophagy and the study identifies the effector reticulon 3L (RTN3L) as a Rab32 effector, which mediates autophagic degradation of a subset of MERC proteins. The article introduces the concept of 'MAM-phagy', a type of selective autophagy that targets mitochondrial-proximal ER membranes, but does not directly discuss the unfolded protein response or SEC61B.\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "        \"generation_info\": {\n",
      "          \"finish_reason\": \"stop\",\n",
      "          \"logprobs\": null\n",
      "        },\n",
      "        \"type\": \"ChatGeneration\",\n",
      "        \"message\": {\n",
      "          \"lc\": 1,\n",
      "          \"type\": \"constructor\",\n",
      "          \"id\": [\n",
      "            \"langchain\",\n",
      "            \"schema\",\n",
      "            \"messages\",\n",
      "            \"AIMessage\"\n",
      "          ],\n",
      "          \"kwargs\": {\n",
      "            \"content\": \"```json\\n[\\n  {\\n    \\\"ID\\\": 1,\\n    \\\"SUMMARY\\\": \\\"The article discusses the nuclear envelope associated endosomes (NAE) pathway, which delivers cell surface receptors such as EGFR from the cell surface to the nucleus. This process involves transport, docking, and fusion with the outer membrane of the nuclear envelope, followed by transport through the nuclear pore complex. The article mentions involvement of the SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins, and the Sec61 translocon in this pathway. It also addresses the role of nuclear EGFR in regulating gene transcription, cell proliferation, and DNA damage repair, and its correlation with poor prognosis in various cancers.\\\",\\n    \\\"RELEVANCE SCORE\\\": 7\\n  },\\n  {\\n    \\\"ID\\\": 2,\\n    \\\"SUMMARY\\\": \\\"The paper reviews the spatial organization of protein biosynthesis in eukaryotic cells, discussing classical and new concepts of mRNA targeting, transport, and localized translation. It highlights the selective translation of mRNA in proximity to various organelles, including endosomes and the role of RNA-binding proteins in forming RNP particles for mRNA transport. The paper also addresses the concept of 'translation factories' and their significance in protein folding and interaction. While it discusses the importance of mRNA compartmentalization in protein biosynthesis regulation, it does not specifically mention the unfolded protein response or Sec61B.\\\",\\n    \\\"RELEVANCE SCORE\\\": 2\\n  },\\n  {\\n    \\\"ID\\\": 3,\\n    \\\"SUMMARY\\\": \\\"The study focuses on Rab32, a small GTPase that influences the composition of the mitochondria-associated membrane (MAM) and regulates mitochondrial membrane dynamics through its effector dynamin-related protein 1 (Drp1). Rab32 is reported to induce autophagy and the study identifies the effector reticulon 3L (RTN3L) as a Rab32 effector, which mediates autophagic degradation of a subset of MERC proteins. The article introduces the concept of 'MAM-phagy', a type of selective autophagy that targets mitochondrial-proximal ER membranes, but does not directly discuss the unfolded protein response or SEC61B.\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 4,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  },\\n  {\\n    \\\"ID\\\": 5,\\n    \\\"SUMMARY\\\": \\\"Not applicable\\\",\\n    \\\"RELEVANCE SCORE\\\": 1\\n  }\\n]\\n```\",\n",
      "            \"additional_kwargs\": {},\n",
      "            \"tool_calls\": [],\n",
      "            \"invalid_tool_calls\": []\n",
      "          }\n",
      "        }\n",
      "      }\n",
      "    ]\n",
      "  ],\n",
      "  \"llm_output\": {\n",
      "    \"token_usage\": {\n",
      "      \"completion_tokens\": 527,\n",
      "      \"prompt_tokens\": 1579,\n",
      "      \"total_tokens\": 2106\n",
      "    },\n",
      "    \"model_name\": \"gpt-4-1106-preview\",\n",
      "    \"system_fingerprint\": null\n",
      "  },\n",
      "  \"run\": null\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 11:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:\n",
      "\u001b[0m[inputs]\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 11:parser:JsonEnclosedByTextOutputParser] [1ms] Exiting Parser run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The article discusses the nuclear envelope associated endosomes (NAE) pathway, which delivers cell surface receptors such as EGFR from the cell surface to the nucleus. This process involves transport, docking, and fusion with the outer membrane of the nuclear envelope, followed by transport through the nuclear pore complex. The article mentions involvement of the SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins, and the Sec61 translocon in this pathway. It also addresses the role of nuclear EGFR in regulating gene transcription, cell proliferation, and DNA damage repair, and its correlation with poor prognosis in various cancers.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"The paper reviews the spatial organization of protein biosynthesis in eukaryotic cells, discussing classical and new concepts of mRNA targeting, transport, and localized translation. It highlights the selective translation of mRNA in proximity to various organelles, including endosomes and the role of RNA-binding proteins in forming RNP particles for mRNA transport. The paper also addresses the concept of 'translation factories' and their significance in protein folding and interaction. While it discusses the importance of mRNA compartmentalization in protein biosynthesis regulation, it does not specifically mention the unfolded protein response or Sec61B.\",\n",
      "      \"RELEVANCE SCORE\": 2\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The study focuses on Rab32, a small GTPase that influences the composition of the mitochondria-associated membrane (MAM) and regulates mitochondrial membrane dynamics through its effector dynamin-related protein 1 (Drp1). Rab32 is reported to induce autophagy and the study identifies the effector reticulon 3L (RTN3L) as a Rab32 effector, which mediates autophagic degradation of a subset of MERC proteins. The article introduces the concept of 'MAM-phagy', a type of selective autophagy that targets mitochondrial-proximal ER membranes, but does not directly discuss the unfolded protein response or SEC61B.\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] [17.75s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The article discusses the nuclear envelope associated endosomes (NAE) pathway, which delivers cell surface receptors such as EGFR from the cell surface to the nucleus. This process involves transport, docking, and fusion with the outer membrane of the nuclear envelope, followed by transport through the nuclear pore complex. The article mentions involvement of the SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins, and the Sec61 translocon in this pathway. It also addresses the role of nuclear EGFR in regulating gene transcription, cell proliferation, and DNA damage repair, and its correlation with poor prognosis in various cancers.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"The paper reviews the spatial organization of protein biosynthesis in eukaryotic cells, discussing classical and new concepts of mRNA targeting, transport, and localized translation. It highlights the selective translation of mRNA in proximity to various organelles, including endosomes and the role of RNA-binding proteins in forming RNP particles for mRNA transport. The paper also addresses the concept of 'translation factories' and their significance in protein folding and interaction. While it discusses the importance of mRNA compartmentalization in protein biosynthesis regulation, it does not specifically mention the unfolded protein response or Sec61B.\",\n",
      "      \"RELEVANCE SCORE\": 2\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The study focuses on Rab32, a small GTPase that influences the composition of the mitochondria-associated membrane (MAM) and regulates mitochondrial membrane dynamics through its effector dynamin-related protein 1 (Drp1). Rab32 is reported to induce autophagy and the study identifies the effector reticulon 3L (RTN3L) as a Rab32 effector, which mediates autophagic degradation of a subset of MERC proteins. The article introduces the concept of 'MAM-phagy', a type of selective autophagy that targets mitochondrial-proximal ER membranes, but does not directly discuss the unfolded protein response or SEC61B.\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] [17.76s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The article discusses the nuclear envelope associated endosomes (NAE) pathway, which delivers cell surface receptors such as EGFR from the cell surface to the nucleus. This process involves transport, docking, and fusion with the outer membrane of the nuclear envelope, followed by transport through the nuclear pore complex. The article mentions involvement of the SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins, and the Sec61 translocon in this pathway. It also addresses the role of nuclear EGFR in regulating gene transcription, cell proliferation, and DNA damage repair, and its correlation with poor prognosis in various cancers.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"The paper reviews the spatial organization of protein biosynthesis in eukaryotic cells, discussing classical and new concepts of mRNA targeting, transport, and localized translation. It highlights the selective translation of mRNA in proximity to various organelles, including endosomes and the role of RNA-binding proteins in forming RNP particles for mRNA transport. The paper also addresses the concept of 'translation factories' and their significance in protein folding and interaction. While it discusses the importance of mRNA compartmentalization in protein biosynthesis regulation, it does not specifically mention the unfolded protein response or Sec61B.\",\n",
      "      \"RELEVANCE SCORE\": 2\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The study focuses on Rab32, a small GTPase that influences the composition of the mitochondria-associated membrane (MAM) and regulates mitochondrial membrane dynamics through its effector dynamin-related protein 1 (Drp1). Rab32 is reported to induce autophagy and the study identifies the effector reticulon 3L (RTN3L) as a Rab32 effector, which mediates autophagic degradation of a subset of MERC proteins. The article introduces the concept of 'MAM-phagy', a type of selective autophagy that targets mitochondrial-proximal ER membranes, but does not directly discuss the unfolded protein response or SEC61B.\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence] [17.76s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"summary\": [\n",
      "    {\n",
      "      \"ID\": 1,\n",
      "      \"SUMMARY\": \"The article discusses the nuclear envelope associated endosomes (NAE) pathway, which delivers cell surface receptors such as EGFR from the cell surface to the nucleus. This process involves transport, docking, and fusion with the outer membrane of the nuclear envelope, followed by transport through the nuclear pore complex. The article mentions involvement of the SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins, and the Sec61 translocon in this pathway. It also addresses the role of nuclear EGFR in regulating gene transcription, cell proliferation, and DNA damage repair, and its correlation with poor prognosis in various cancers.\",\n",
      "      \"RELEVANCE SCORE\": 7\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 2,\n",
      "      \"SUMMARY\": \"The paper reviews the spatial organization of protein biosynthesis in eukaryotic cells, discussing classical and new concepts of mRNA targeting, transport, and localized translation. It highlights the selective translation of mRNA in proximity to various organelles, including endosomes and the role of RNA-binding proteins in forming RNP particles for mRNA transport. The paper also addresses the concept of 'translation factories' and their significance in protein folding and interaction. While it discusses the importance of mRNA compartmentalization in protein biosynthesis regulation, it does not specifically mention the unfolded protein response or Sec61B.\",\n",
      "      \"RELEVANCE SCORE\": 2\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 3,\n",
      "      \"SUMMARY\": \"The study focuses on Rab32, a small GTPase that influences the composition of the mitochondria-associated membrane (MAM) and regulates mitochondrial membrane dynamics through its effector dynamin-related protein 1 (Drp1). Rab32 is reported to induce autophagy and the study identifies the effector reticulon 3L (RTN3L) as a Rab32 effector, which mediates autophagic degradation of a subset of MERC proteins. The article introduces the concept of 'MAM-phagy', a type of selective autophagy that targets mitochondrial-proximal ER membranes, but does not directly discuss the unfolded protein response or SEC61B.\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 4,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    },\n",
      "    {\n",
      "      \"ID\": 5,\n",
      "      \"SUMMARY\": \"Not applicable\",\n",
      "      \"RELEVANCE SCORE\": 1\n",
      "    }\n",
      "  ],\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\\n\\\\nVarious growth factors and full-length cell surface receptors such as EGFR are translocated from the cell surface to the nucleoplasm, baffling cell biologists to the mechanisms and functions of this process. Elevated levels of nuclear EGFR correlate with poor prognosis in various cancers. In recent years, nuclear EGFR has been implicated in regulating gene transcription, cell proliferation and DNA damage repair. Different models have been proposed to explain how the receptors are transported into the nucleus. However, a clear consensus has yet to be reached. Recently, we described the nuclear envelope associated endosomes (NAE) pathway, which delivers EGFR from the cell surface to the nucleus. This pathway involves transport, docking and fusion of NAEs with the outer membrane of the nuclear envelope. EGFR is then presumed to be transported through the nuclear pore complex, extracted from membranes and solubilised. The SUN1/2 nuclear envelope proteins, Importin-beta, nuclear pore complex proteins and the Sec61 translocon have been implicated in the process. While this framework can explain the cell surface to nucleus traffic of EGFR and other cell surface receptors, it raises several questions that we consider in this review, together with implications for health and disease.\\\", \\\"CITATION\\\": \\\"Shah P, Chaumet A, Royle SJ, Bard FA. (2019) The NAE Pathway: Autobahn to the Nucleus for Cell Surface Receptors.\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\\n\\\\nSpatial organization of protein biosynthesis in the eukaryotic cell has been studied for more than fifty years, thus many facts have already been included in textbooks. According to the classical view, mRNA transcripts encoding secreted and transmembrane proteins are translated by ribosomes associated with endoplasmic reticulum membranes, while soluble cytoplasmic proteins are synthesized on free polysomes. However, in the last few years, new data has emerged, revealing selective translation of mRNA on mitochondria and plastids, in proximity to peroxisomes and endosomes, in various granules and at the cytoskeleton (actin network, vimentin intermediate filaments, microtubules and centrosomes). There are also long-standing debates about the possibility of protein synthesis in the nucleus. Localized translation can be determined by targeting signals in the synthesized protein, nucleotide sequences in the mRNA itself, or both. With RNA-binding proteins, many transcripts can be assembled into specific RNA condensates and form RNP particles, which may be transported by molecular motors to the sites of active translation, form granules and provoke liquid-liquid phase separation in the cytoplasm, both under normal conditions and during cell stress. The translation of some mRNAs occurs in specialized \\\\\\\"translation factories,\\\\\\\" assemblysomes, transperons and other structures necessary for the correct folding of proteins, interaction with functional partners and formation of oligomeric complexes. Intracellular localization of mRNA has a significant impact on the efficiency of its translation and presumably determines its response to cellular stress. Compartmentalization of mRNAs and the translation machinery also plays an important role in viral infections. Many viruses provoke the formation of specific intracellular structures, virus factories, for the production of their proteins. Here we review the current concepts of the molecular mechanisms of transport, selective localization and local translation of cellular and viral mRNAs, their effects on protein targeting and topogenesis, and on the regulation of protein biosynthesis in different compartments of the eukaryotic cell. Special attention is paid to new systems biology approaches, providing new cues to the study of localized translation.\\\", \\\"CITATION\\\": \\\"Lashkevich KA, Dmitriev SE. (2021) mRNA Targeting, Transport and Local Translation in Eukaryotic Cells: From the Classical View to a Diversity of New Concepts.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\\n\\\\n<h4>Background</h4>Rab32 is a small GTPase associated with multiple organelles but is particularly enriched at the endoplasmic reticulum (ER). Here, it controls targeting to mitochondria-ER contacts (MERCs), thus influencing composition of the mitochondria-associated membrane (MAM). Moreover, Rab32 regulates mitochondrial membrane dynamics via its effector dynamin-related protein 1 (Drp1). Rab32 has also been reported to induce autophagy, an essential pathway targeting intracellular components for their degradation. However, no autophagy-specific effectors have been identified for Rab32. Similarly, the identity of the intracellular membrane targeted by this small GTPase and the type of autophagy it induces are not known yet.<h4>Results</h4>To investigate the target of autophagic degradation mediated by Rab32, we tested a large panel of organellar proteins. We found that a subset of MERC proteins, including the thioredoxin-related transmembrane protein TMX1, are specifically targeted for degradation in a Rab32-dependent manner. We also identified the long isoform of reticulon-3 (RTN3L), a known ER-phagy receptor, as a Rab32 effector.<h4>Conclusions</h4>Rab32 promotes degradation of mitochondrial-proximal ER membranes through autophagy with the help of RTN3L. We propose to call this type of selective autophagy \\\\\\\"MAM-phagy\\\\\\\".\\\", \\\"CITATION\\\": \\\"Herrera-Cruz MS, Yap MC, Tahbaz N, Phillips K, Thomas L, Thomas G, Simmen T. (2021) Rab32 uses its effector reticulon 3L to trigger autophagic degradation of mitochondria-associated membrane (MAM) proteins.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 3:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 5\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 4:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 5:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": 1000\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"k\": 5,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context> > 6:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:chain:RunnableParallel<k,question,summary_length,context>] [3ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context>] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] Entering Prompt run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 9:prompt:ChatPromptTemplate] [0ms] Exiting Prompt run with output:\n",
      "\u001b[0m[outputs]\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"k\": 5,\n",
      "  \"question\": \"Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\",\n",
      "  \"summary_length\": 1000,\n",
      "  \"context\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 9:chain:RunnableLambda] [0ms] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"output\": \"[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[llm/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 7:chain:RunnableParallel<summary,context> > 8:chain:RunnableSequence > 10:llm:ChatOpenAI] Entering LLM run with input:\n",
      "\u001b[0m{\n",
      "  \"prompts\": [\n",
      "    \"System: Answer in a direct and concise tone. Your audience is an expert, so be highly specific. If there are ambiguous terms or acronyms, first define them.\\nHuman: Read through the following JSON encoding of 5 research articles: \\nEach document has three attributes: (A) an integer ID for the paper, (B) a CITATION string containing the authors, publication year, title and publication location, and the (C) CONTENT field with the title and abstract of the paper.  \\n```CONTEXT:[{\\\"CONTENT\\\": \\\"Wolfram syndrome: MAMs' connection?\\\\n\\\\nWolfram syndrome (WS) is a rare neurodegenerative disease, the main pathological hallmarks of which associate with diabetes, optic atrophy, and deafness. Other symptoms may be identified in some but not all patients. Prognosis is poor, with death occurring around 35 years of age. To date, no treatment is available. WS was first described as a mitochondriopathy. However, the localization of the protein on the endoplasmic reticulum (ER) membrane challenged this hypothesis. ER contacts mitochondria to ensure effective Ca<sup>2+</sup> transfer, lipids transfer, and apoptosis within stabilized and functionalized microdomains, termed \\\\\\\"mitochondria-associated ER membranes\\\\\\\" (MAMs). Two types of WS are characterized so far and Wolfram syndrome type 2 is due to mutation in CISD2, a protein mostly expressed in MAMs. The aim of the present review is to collect evidences showing that WS is indeed a mitochondriopathy, with established MAM dysfunction, and thus share commonalities with several neurodegenerative diseases, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis, as well as metabolic diseases, such as diabetes.\\\", \\\"CITATION\\\": \\\"Delprat B, Maurice T, Delettre C. (2018) Wolfram syndrome: MAMs' connection?\\\", \\\"ID\\\": 1}, {\\\"CONTENT\\\": \\\"The functional organization of axonal mRNA transport and translation.\\\\n\\\\nAxons extend for tremendously long distances from the neuronal soma and make use of localized mRNA translation to rapidly respond to different extracellular stimuli and physiological states. The locally synthesized proteins support many different functions in both developing and mature axons, raising questions about the mechanisms by which local translation is organized to ensure the appropriate responses to specific stimuli. Publications over the past few years have uncovered new mechanisms for regulating the axonal transport and localized translation of mRNAs, with several of these pathways converging on the regulation of cohorts of functionally related mRNAs - known as RNA regulons - that drive axon growth, axon guidance, injury responses, axon survival and even axonal mitochondrial function. Recent advances point to these different regulatory pathways as organizing platforms that allow the axon's proteome to be modulated to meet its physiological needs.\\\", \\\"CITATION\\\": \\\"Dalla Costa I, Buchanan CN, Zdradzinski MD, Sahoo PK, Smith TP, Thames E, Kar AN, Twiss JL. (2020) The functional organization of axonal mRNA transport and translation.\\\", \\\"ID\\\": 2}, {\\\"CONTENT\\\": \\\"Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\\n\\\\n<h4>Background</h4>Polycystic liver diseases (PLDs) are genetic inherited disorders characterized by the progressive growth of numerous intrahepatic biliary cysts, which are the main cause of morbidity. Previous studies revealed that cystic cholangiocytes are characterized by endoplasmic reticulum stress and aberrant posttranslational modification (PTM) of proteins, in particular hyper-SUMOylation, that promote PLD pathobiology. Protein NEDDylation is a newly characterized PTM that modulates a plethora of biological processes and its dysregulation is associated with the development and progression of several human diseases. However, the role of NEDDylation in PLD remains elusive.<h4>Objective</h4>To explore the role of protein NEDDylation in PLD and its potential therapeutic regulatory value.<h4>Methods</h4>Levels and functional effects of NEDDylation, including response to Pevonedistat (first-in-class selective inhibitor of the NEDDylation E1 enzyme NAE), were assessed in vitro, in vivo, and/or in patients with PLD. NEDDylated protein\\\\u00a0levels in normal and cystic human cholangiocytes were assessed by immunoprecipitation, and the proteomic profile was further analyzed by mass spectrometry.<h4>Results and conclusion</h4>The genes involved in the NEDDylation pathway were found overexpressed (mRNA) in polycystic human and rat liver tissue, as well as in cystic cholangiocytes in culture, compared to controls. Elevated levels of NEDDylated proteins were further confirmed in cystic cholangiocytes in vitro, which diminished under Pevonedistat incubation. Pevonedistat promoted apoptotic cell death and reduced proliferation in cystic cholangiocytes in vitro. Comparative proteomic profiling of NEDD8-immunoprecipitated proteins between normal and cystic cholangiocytes in culture reported candidate proteins involved in cystogenesis, mostly associated with protein biogenesis and quality control. All these data indicate that cystic cholangiocytes display increased protein NEDDylation, contributing to cell survival and proliferation, ultimately supporting hepatic cystogenesis. Targeting of protein hyper-NEDDylation in cystic cholangiocytes inhibits cystogenesis in experimental models, representing a novel therapeutic opportunity in PLD.\\\", \\\"CITATION\\\": \\\"Lee-Law PY, Olaizola P, Caballero-Camino FJ, Izquierdo-Sanchez L, Rodrigues PM, Perugorria MJ, Azkargorta M, Elortza F, Martinez-Chantar ML, Aspichueta P, Marzioni M, Bujanda L, Drenth JPH, Banales JM. (2021) Inhibition of NAE-dependent protein hyper-NEDDylation in cystic cholangiocytes halts cystogenesis in experimental models of polycystic liver disease.\\\", \\\"ID\\\": 3}]```\\nIf CONTEXT is provided, generate a JSON list of summaries of each article in order to help answer the following question:\\nQuestion: Write an essay to answer the question: \\\"What is the connection between SEC61B and the unfolded protein response?\\nDo NOT directly answer the question, instead summarize the text from each paper's CONTENT to provide evidence to help answer the question.  Do not include information that does not appear directly in the CONTENT of each paper. Focus on specific details, including numbers, equations, or specific quotes.  Reply \\\"Not applicable\\\" if text is irrelevant.  Restrict each summary to 1000 words.  Also, provide a score from 1-10 indicating relevance to question. Do not explain your score. \\nWrite this answer as JSON formatted output. Provide a list of 5 dict objects with the following fields: ID, SUMMARY, RELEVANCE SCORE. \\nDo not provide additional explanation for the answer. Do not include any other response other than a JSON object.\"\n",
      "  ]\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "#\n",
    "# Note - Run this as an example question based on available data. \n",
    "#\n",
    "PaperQA_tool = [t for t in cb.tk.get_tools() if isinstance(t, PaperQAEmulationTool)][0]\n",
    "question = 'Write an essay to answer the question: \"What is the connection between SEC61B and the unfolded protein response?'\n",
    "PaperQA_tool.run(tool_input={'question': question})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Crazy Bug\n",
    "\n",
    "Running the `skc_id=83` query below takes 6 minutes and `skc_id=79` takes less than 1 sec. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.session.rollback()\n",
    "ldb.session.execute(text(\n",
    "    '''SELECT DISTINCT skc.name, ske.id, ske.content, ske.publication_date as pub_date, ske.type as pub_type, emb.embedding, skf.content \n",
    "    FROM langchain_pg_embedding as emb, \n",
    "        \"ScientificKnowledgeExpression\" as ske,\n",
    "        \"ScientificKnowledgeCollection_has_members\" as skc_hm, \n",
    "        \"ScientificKnowledgeCollection\" as skc, \n",
    "        \"ScientificKnowledgeFragment\" as skf\n",
    "    WHERE emb.cmetadata->>'i_type' = 'CitationRecord' AND\n",
    "        emb.cmetadata->>'e_id' = ske.id AND \n",
    "        emb.cmetadata->>'f_id' = skf.id AND\n",
    "        skc_hm.\"ScientificKnowledgeCollection_id\" = skc.id AND\n",
    "        ske.id = skc_hm.has_members_id AND (skc.id='79')\n",
    "    ORDER BY pub_date DESC;''')).fetchall()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.session.rollback()\n",
    "ldb.session.execute(text(\n",
    "    '''SELECT DISTINCT skc.name, ske.id, ske.content, ske.publication_date as pub_date, ske.type as pub_type, emb.embedding, skf.content \n",
    "    FROM langchain_pg_embedding as emb, \n",
    "        \"ScientificKnowledgeExpression\" as ske,\n",
    "        \"ScientificKnowledgeCollection_has_members\" as skc_hm, \n",
    "        \"ScientificKnowledgeCollection\" as skc, \n",
    "        \"ScientificKnowledgeFragment\" as skf\n",
    "    WHERE emb.cmetadata->>'i_type' = 'CitationRecord' AND\n",
    "        emb.cmetadata->>'e_id' = ske.id AND \n",
    "        emb.cmetadata->>'f_id' = skf.id AND\n",
    "        skc_hm.\"ScientificKnowledgeCollection_id\" = skc.id AND\n",
    "        ske.id = skc_hm.has_members_id AND (skc.id='83')\n",
    "    ORDER BY pub_date DESC;''')).fetchall()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "q = ldb.session.query(SKE).distinct(SKE.id)\n",
    "print(q.count())\n",
    "ldb.embed_expression_list(q.all())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cb.agent_executor.invoke({'input':'Get full text copies of all papers in the collection with id=\"0\".'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "q = ldb.session.query(SKC.id, SKC.name, func.count(SKC_HM.has_members_id)) \\\n",
    "    .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "    .group_by(SKC.id, SKC.name) \\\n",
    "    .order_by(SKC.id.cast(Integer))\n",
    "corpora_df = pd.DataFrame(q.all(), columns=['Corpus ID', 'Corpus Name', 'Paper Count'])\n",
    "\n",
    "paper_count = ldb.session.query(func.count(SKE.id)).first()\n",
    "print('Count of all papers in database: %d'%(paper_count[0]))\n",
    "\n",
    "corpora_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "q3 = ldb.session.query(N) \\\n",
    "        .filter(N.type == 'NoteAboutFragment') \n",
    "\n",
    "for n in q3.all():\n",
    "    n_content = json.loads(n.content)\n",
    "    print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')\n",
    "    print(n.id)\n",
    "    print(n_content.get('response')) \n",
    "    print(n_content.get('data')) \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "skes = ldb.session.query(SKE).all()\n",
    "#ldb.embed_expression_list(skes)\n",
    "print(len(skes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.session.rollback()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ft_retriever  = [t for t in tk.get_tools() if isinstance(t, RetrieveFullTextTool)][0]\n",
    "\n",
    "for i, c in corpora_df.iterrows():\n",
    "    if c['Corpus ID'] != '81':\n",
    "        continue\n",
    "    print(c['Corpus Name'])\n",
    "    ft_count = 0\n",
    "    no_ft_count = 0\n",
    "    doi_list = [e.id for e in ldb.list_expressions(collection_id=c['Corpus ID'])]\n",
    "    for doi in doi_list:\n",
    "        d2 = doi.replace('doi:', '')\n",
    "        path = loc+db_name+'/ft/'\n",
    "        nxml_file_path = path+'/'+d2+'.nxml'\n",
    "        pdf_file_path = path+'/'+d2+'.pdf'\n",
    "        html_file_path = path+'/'+d2+'.html'\n",
    "        if os.path.exists(nxml_file_path) or  \\\n",
    "                os.path.exists(pdf_file_path) or \\\n",
    "                os.path.exists(html_file_path):\n",
    "            ft_count += 1\n",
    "        try: \n",
    "            no_ft_count += 1\n",
    "            #print('\\t'+doi)\n",
    "            ft_retriever.run(tool_input={'paper_id': doi})\n",
    "        except Exception as e:\n",
    "            print(e)\n",
    "    print(ft_count)\n",
    "    print(no_ft_count)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "q = ldb.session.query(SKE.id, SKI.id, SKI.type, SKF.id, SKF.type, SKF.offset, SKF.content) \\\n",
    "    .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "    .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "    .filter(SKE.id==SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "    .filter(SKE_HR.has_representation_id==SKI.id) \\\n",
    "    .filter(SKI.id==SKI_HP.ScientificKnowledgeItem_id) \\\n",
    "    .filter(SKI_HP.has_part_id==SKF.id) \\\n",
    "    .filter(SKE_HR.has_representation_id==SKI.id) \\\n",
    "    .filter(SKF.type=='section') \\\n",
    "    .filter(SKI.type.like('%FullText')) \\\n",
    "    .order_by(SKE.id, SKF.offset)\n",
    "items_df = pd.DataFrame(q.all(), columns=['doi', 'item_id', 'item_type', 'fragment_id', 'fragment_type', 'offset', 'content'])\n",
    "\n",
    "items_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Index the abstracts and run some simple semantic queries\n",
    "\n",
    "Here we index each paper's title and abstract to build a simple question / answer interface."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.session.rollback()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i, c in tqdm(corpora_df.iterrows()):\n",
    "    if c['Corpus ID'] != '81':\n",
    "        continue\n",
    "    expressions = ldb.list_expressions(collection_id=c['Corpus ID'])    \n",
    "    ldb.embed_expression_list(expressions)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "question = 'What is known about genetics underlying Stiff Person Syndrome?'\n",
    "\n",
    "ldb.query_vectorindex(question, k=10, collection_name='ScienceKnowledgeItem_FullText')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## ATTEMPTING TO RECONSTRUCT PAPER-QA PIPELINE IN OUR SYSTEM.\n",
    "\n",
    "1. Embed paper sections + question\n",
    "2. Given the question, summarize the retrieved paper sections relative to the question\n",
    "3. Score and select relevant passages\n",
    "4. Put summaries into prompt\n",
    "5. Generate answer with prompt\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    " \n",
    "os.environ['PGVECTOR_CONNECTION_STRING'] = \"postgresql+psycopg2:///\"+ldb.name\n",
    "vectorstore = PGVector.from_existing_index(\n",
    "        embedding = ldb.embed_model, \n",
    "        collection_name = 'ScienceKnowledgeItem') \n",
    "retriever = vectorstore.as_retriever(search_kwargs={'k':15, 'filter': {'skc_ids': 81}})\n",
    "#retriever = vectorstore.as_retriever()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "retriever.invoke(question)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "#from paperqa.prompts import summary_prompt as paperqa_summary_prompt, qa_prompt as paperqa_qa_prompt, select_paper_prompt, citation_prompt, default_system_prompt\n",
    "\n",
    "hum_p = '''First, read through the following JSON encoding of {k} research articles: \n",
    "\n",
    "Each document has three attributes: (A) a digital object identifier ('DOI') code, (B) a CITATION string containing the authors, publication year, title and publication location, and the (C) CONTENT field with the title and abstract of the paper.  \n",
    "\n",
    "```json:{context}```\n",
    "\n",
    "Then, generate a JSON list of summaries of each article in order to help answer the following question:\n",
    "\n",
    "Question: {question}\n",
    "\n",
    "Do NOT directly answer the question, instead summarize to give evidence to help answer the question. \n",
    "Focus on specific details, including numbers, equations, or specific quotes. \n",
    "Reply \"Not applicable\" if text is irrelevant. \n",
    "Restrict each summary to {summary_length} words. \n",
    "Also, provide a score from 1-10 indicating relevance to question. Do not explain your score. \n",
    "\n",
    "Write this answer as JSON formatted output. Provide a list of {k} dict objects with the following fields: DOI, SUMMARY, RELEVANCE SCORE. \n",
    "\n",
    "Do not provide additional explanation for the answer.\n",
    "Do not include any other response other than a JSON object.\n",
    "'''\n",
    "sys_p = '''Answer in a direct and concise tone. Your audience is an expert, so be highly specific. If there are ambiguous terms or acronyms, first define them.'''\n",
    "\n",
    "DEFAULT_DOCUMENT_PROMPT = PromptTemplate.from_template(template=\"'DOI': '{ske_id}', CITATION: '{citation}', CONTENT:'{page_content}'\")\n",
    "def combine_documents(\n",
    "    docs, document_prompt=DEFAULT_DOCUMENT_PROMPT, document_separator=\"},{\\n\"\n",
    "):\n",
    "    doc_strings = [format_document(doc, document_prompt) for doc in docs]\n",
    "    return '[{'+document_separator.join(doc_strings)+'}]'\n",
    "\n",
    "template = ChatPromptTemplate.from_messages([\n",
    "            (\"system\", sys_p),\n",
    "            (\"human\", hum_p)])\n",
    "\n",
    "qa_chain = (\n",
    "    RunnableParallel({\n",
    "        \"k\": itemgetter(\"k\"),\n",
    "        \"question\": itemgetter(\"question\"),\n",
    "        \"summary_length\": itemgetter(\"summary_length\"),\n",
    "        \"context\": itemgetter(\"question\") | retriever | combine_documents,\n",
    "    })\n",
    "    | {\n",
    "        \"summary\": template | ChatOllama(model='mixtral') | JsonEnclosedByTextOutputParser(),\n",
    "        \"context\": itemgetter(\"context\"),\n",
    "    }\n",
    ")\n",
    "\n",
    "input = {'question': question, 'summary_length': 1000, 'k':5}    \n",
    "out = qa_chain.invoke(input, config={'callbacks': [ConsoleCallbackHandler()]})\n",
    "print(json.dumps(out, indent=4))\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Discourse Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "model_path = '/Users/gully.burns/Documents/2024H1/models/discourse_tagger'\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"dmis-lab/biobert-v1.1\", \n",
    "                                          truncation=True, \n",
    "                                          max_length=512)\n",
    "labels = ['BACKGROUND', 'OBJECTIVE', 'METHODS', 'RESULTS', 'CONCLUSIONS']\n",
    "lookup = {'LABEL_%d'%(i):l for i, l in enumerate(labels)}\n",
    "model = AutoModel.from_pretrained(model_path)\n",
    "model.eval()\n",
    "\n",
    "classifier = pipeline(\"text-classification\", \n",
    "                      model = model_path, \n",
    "                      tokenizer=tokenizer, \n",
    "                      truncation=True,\n",
    "                      batch_size=8,\n",
    "                      device='mps')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Try an out-of-the-box classifier on the data for discourse tagging.\n",
    "\n",
    "\n",
    "ldb.session.rollback()\n",
    "one_year_ago = (datetime.now() - timedelta(days=1*365))\n",
    "\n",
    "q = ldb.session.query(SKE, SKF) \\\n",
    "    .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "    .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "    .filter(SKE.id==SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "    .filter(SKE_HR.has_representation_id==SKI.id) \\\n",
    "    .filter(SKI.id==SKI_HP.ScientificKnowledgeItem_id) \\\n",
    "    .filter(SKI_HP.has_part_id==SKF.id) \\\n",
    "    .filter(SKE_HR.has_representation_id==SKI.id) \\\n",
    "    .filter(SKI.type == 'CitationRecord' ) \\\n",
    "    .order_by(SKE.id)\n",
    "\n",
    "#   .filter(SKC.name == 'The Stiff Person Syndrome' ) \\\n",
    "#   .filter(SKE.publication_date >= one_year_ago) \\\n",
    "\n",
    "s_list = []\n",
    "for e, f in q.all():\n",
    "    for i, s in enumerate(ldb.sent_detector.tokenize(f.content)):\n",
    "        s_list.append([e.id, f.id, i, s])\n",
    "sent_df = pd.DataFrame(s_list, columns=['doi', 'f_id', 's_id', 'text'])\n",
    "sent_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# Predict multipe texts on single CPU and time the inference duration\n",
    "start = time()\n",
    "\n",
    "df = sent_df\n",
    "\n",
    "preds = classifier([row.text for i, row in df.iterrows()])\n",
    "pred_df = pd.DataFrame(preds)\n",
    "df['label'] = [lookup[row.label] for i, row in pred_df.iterrows()]\n",
    "df['score'] = [row.score for i, row in pred_df.iterrows()]\n",
    "\n",
    "end = time()\n",
    "\n",
    "print('Prediction time:', str(timedelta(seconds=end-start)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.session.rollback()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Generate fragment sentences and add them as Notes\n",
    "ldb.session.rollback()\n",
    "for i, row in df.iterrows():\n",
    "    f_q = ldb.session.query(SKF).filter(SKF.id == row.f_id).first()\n",
    "    i_q = ldb.session.query(SKI).filter(SKI.id == row.f_id.split('.')[0]).first()\n",
    "    o = i_q.content.find(row.text)\n",
    "    l = len(row.text)\n",
    "    sentence_fragment = ScientificKnowledgeFragment(id=f_q.id+'.'+str(row.s_id), \\\n",
    "                                                    content=row.text, \\\n",
    "                                                    offset=o, \\\n",
    "                                                    length=l, \\\n",
    "                                                    type='sentence')\n",
    "    i_q.has_part.append(sentence_fragment)\n",
    "    note_content = {'discourse_label': row.label, 'score': row.score}\n",
    "    n = Note(id=f_q.id+'.'+str(row.s_id)+'.discourse_type',\n",
    "             content=json.dumps(note_content, indent=4),\n",
    "             format='json',\n",
    "             type='NoteAboutFragment')\n",
    "    sentence_fragment.has_notes.append(n)\n",
    "    ldb.session.flush()\n",
    "ldb.session.commit()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Running DRSM Classifiers."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "model_path = '/Users/gully.burns/Documents/2024H1/models/drsm_classifier'\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"dmis-lab/biobert-v1.1\", \n",
    "                                          truncation=True, \n",
    "                                          max_length=512)\n",
    "labels = ['BACKGROUND', 'OBJECTIVE', 'METHODS', 'RESULTS', 'CONCLUSIONS']\n",
    "lookup = {'LABEL_%d'%(i):l for i, l in enumerate(labels)}\n",
    "model = AutoModel.from_pretrained(model_path)\n",
    "model.eval()\n",
    "\n",
    "classifier = pipeline(\"text-classification\", \n",
    "                      model = model_path, \n",
    "                      tokenizer=tokenizer, \n",
    "                      truncation=True,\n",
    "                      batch_size=8,\n",
    "                      device='mps')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Topic Modeling over the corpus. \n",
    "\n",
    "What are the main topics being discussed in each paper?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "model_path = '/Users/gully.burns/Documents/2024H1/models/drsm_classifier'\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"dmis-lab/biobert-v1.1\", \n",
    "                                          truncation=True, \n",
    "                                          max_length=512)\n",
    "labels = ['BACKGROUND', 'OBJECTIVE', 'METHODS', 'RESULTS', 'CONCLUSIONS']\n",
    "lookup = {'LABEL_%d'%(i):l for i, l in enumerate(labels)}\n",
    "model = AutoModel.from_pretrained(model_path)\n",
    "model.eval()\n",
    "\n",
    "classifier = pipeline(\"text-classification\", \n",
    "                      model = model_path, \n",
    "                      tokenizer=tokenizer, \n",
    "                      truncation=True,\n",
    "                      batch_size=8,\n",
    "                      device='mps')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Search for and download Full Text Papers.\n",
    "\n",
    "Can we search for all Stiff Person Syndrome papers published in the last 10 years?\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.session.rollback()\n",
    "\n",
    "ten_years_ago = (datetime.now() - timedelta(days=10*365))\n",
    "print(ten_years_ago)\n",
    "\n",
    "q = ldb.session.query(func.extract('year', SKE.publication_date.cast(Date)), func.count(SKE.id) ) \\\n",
    "    .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "    .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "    .filter(SKE.publication_date >= ten_years_ago) \\\n",
    "    .filter(SKC.name == 'The Stiff Person Syndrome' ) \\\n",
    "    .group_by(func.extract('year', SKE.publication_date.cast(Date))) \\\n",
    "    .order_by(func.extract('year', SKE.publication_date.cast(Date)))\n",
    "sps_pubcount_df = pd.DataFrame(q.all(), columns=['doi', 'date'])\n",
    "sps_pubcount_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run PaperQA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cb.agent_executor.invoke({'input':'Write a short essay on \"What connections between primary ciliary diskinesia and primary cilia have been studied?\" based on the collection with ID=\"70\".'})"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "alhazen",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
